<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2987344/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association study for age-related hearing impairment in the Saami</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2987344/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="A genome-wide association study for age-related hearing impairment in the Saami"><meta name="citation_authors" content="Lut Van Laer, Jeroen R Huyghe, Samuli Hannula, Els Van Eyken, Dietrich A Stephan, Elina Mäki-Torkko, Pekka Aikio, Erik Fransen, Alana Lysholm-Bernacchi, Martti Sorri, Matthew J Huentelman, Guy Van Camp"><meta name="citation_date" content="June 2010"><meta name="citation_issue" content="6"><meta name="citation_volume" content="18"><meta name="citation_firstpage" content="685"><meta name="citation_doi" content="10.1038/ejhg.2009.234"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2987344/?report=abstract"><meta name="citation_pmid" content="20068591"><meta name="DC.Title" content="A genome-wide association study for age-related hearing impairment in the Saami"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Lut Van Laer"><meta name="DC.Contributor" content="Jeroen R Huyghe"><meta name="DC.Contributor" content="Samuli Hannula"><meta name="DC.Contributor" content="Els Van Eyken"><meta name="DC.Contributor" content="Dietrich A Stephan"><meta name="DC.Contributor" content="Elina Mäki-Torkko"><meta name="DC.Contributor" content="Pekka Aikio"><meta name="DC.Contributor" content="Erik Fransen"><meta name="DC.Contributor" content="Alana Lysholm-Bernacchi"><meta name="DC.Contributor" content="Martti Sorri"><meta name="DC.Contributor" content="Matthew J Huentelman"><meta name="DC.Contributor" content="Guy Van Camp"><meta name="DC.Date" content="2010 Jun"><meta name="DC.Identifier" content="10.1038/ejhg.2009.234"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association study for age-related hearing impairment in the Saami"><meta property="og:type" content="article"><meta property="og:description" content="This study aimed at contributing to the elucidation of the genetic basis of age-related hearing impairment (ARHI), a common multifactorial disease with an important genetic contribution as demonstrated by heritability studies. We conducted a genome-wide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2987344/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2987344/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2987344/pdf/ejhg2009234a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE882895DD5045810000000003460232.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2987344%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/191823/">v.18(6); 2010 Jun</a></li><li class="accid">PMC2987344</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2010 Jun; 18(6): 685–693. </span></div><div><span class="fm-vol-iss-date">Published online 2010 Jan 13. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2009.234" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2009.234</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2987344</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20068591">20068591</a></div></div></div></div><h1 class="content-title">A genome-wide association study for age-related hearing impairment in the Saami</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Van%20Laer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398723814896" co-class="co-affbox">Lut Van Laer</a>,<sup>1,</sup><sup>5</sup> <a href="/pubmed/?term=Huyghe%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398769562752" co-class="co-affbox">Jeroen R Huyghe</a>,<sup>1,</sup><sup>5</sup> <a href="/pubmed/?term=Hannula%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398731611024" co-class="co-affbox">Samuli Hannula</a>,<sup>2</sup> <a href="/pubmed/?term=Van%20Eyken%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398731608800" co-class="co-affbox">Els Van Eyken</a>,<sup>1</sup> <a href="/pubmed/?term=Stephan%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398731606576" co-class="co-affbox">Dietrich A Stephan</a>,<sup>3</sup> <a href="/pubmed/?term=M%26%23x000e4%3Bki-Torkko%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398730752624" co-class="co-affbox">Elina Mäki-Torkko</a>,<sup>2</sup> <a href="/pubmed/?term=Aikio%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398730750400" co-class="co-affbox">Pekka Aikio</a>,<sup>4</sup> <a href="/pubmed/?term=Fransen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398769669184" co-class="co-affbox">Erik Fransen</a>,<sup>1</sup> <a href="/pubmed/?term=Lysholm-Bernacchi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398769666960" co-class="co-affbox">Alana Lysholm-Bernacchi</a>,<sup>3,</sup><sup>6</sup> <a href="/pubmed/?term=Sorri%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398734910640" co-class="co-affbox">Martti Sorri</a>,<sup>2</sup> <a href="/pubmed/?term=Huentelman%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398734908416" co-class="co-affbox">Matthew J Huentelman</a>,<sup>3</sup> and  <a href="/pubmed/?term=Van%20Camp%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591" class="affpopup" co-rid="_co_idm140398720060528" co-class="co-affbox">Guy Van Camp</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140398723814896"><h3 class="no_margin">Lut Van Laer</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20Laer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Lut Van Laer</a></div></div><div id="_co_idm140398769562752"><h3 class="no_margin">Jeroen R Huyghe</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Huyghe%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Jeroen R Huyghe</a></div></div><div id="_co_idm140398731611024"><h3 class="no_margin">Samuli Hannula</h3><p><sup>2</sup>Department of Otorhinolaryngology, University of Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Hannula%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Samuli Hannula</a></div></div><div id="_co_idm140398731608800"><h3 class="no_margin">Els Van Eyken</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20Eyken%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Els Van Eyken</a></div></div><div id="_co_idm140398731606576"><h3 class="no_margin">Dietrich A Stephan</h3><p><sup>3</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Stephan%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Dietrich A Stephan</a></div></div><div id="_co_idm140398730752624"><h3 class="no_margin">Elina Mäki-Torkko</h3><p><sup>2</sup>Department of Otorhinolaryngology, University of Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000e4%3Bki-Torkko%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Elina Mäki-Torkko</a></div></div><div id="_co_idm140398730750400"><h3 class="no_margin">Pekka Aikio</h3><p><sup>4</sup>Thule Institute, University of Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Aikio%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Pekka Aikio</a></div></div><div id="_co_idm140398769669184"><h3 class="no_margin">Erik Fransen</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Fransen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Erik Fransen</a></div></div><div id="_co_idm140398769666960"><h3 class="no_margin">Alana Lysholm-Bernacchi</h3><p><sup>3</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Lysholm-Bernacchi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Alana Lysholm-Bernacchi</a></div></div><div id="_co_idm140398734910640"><h3 class="no_margin">Martti Sorri</h3><p><sup>2</sup>Department of Otorhinolaryngology, University of Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Sorri%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Martti Sorri</a></div></div><div id="_co_idm140398734908416"><h3 class="no_margin">Matthew J Huentelman</h3><p><sup>3</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Huentelman%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Matthew J Huentelman</a></div></div><div id="_co_idm140398720060528"><h3 class="no_margin">Guy Van Camp</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20Camp%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20068591">Guy Van Camp</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140398734906192_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140398734906192_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140398734906192_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140398734906192_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Medical Genetics, University of Antwerp, Antwerp, Belgium</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Otorhinolaryngology, University of Oulu, Oulu, Finland</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Thule Institute, University of Oulu, Oulu, Finland</div><div id="caf1"><sup>*</sup>Department of Medical Genetics, University of Antwerp, Campus Drie Eiken, Universiteitsplein 1, Antwerp B-2610, Belgium. Tel: +32 3 275 9762; Fax: +32 3 275 9722; E-mail: <a href="mailto:dev@null" data-email="eb.ca.au@pmacnav.yug" class="oemail">eb.ca.au@pmacnav.yug</a></div><div id="note1"><sup>5</sup>These authors contributed equally to this work.</div><div id="note2"><sup>6</sup>In loving memory.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140398734906192_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Feb 23; Revised 2009 Sep 15; Accepted 2009 Dec 1.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140398734906192_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2010 Macmillan Publishers Limited</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2987344/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="4" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 1.</strong></div><div><a href="/pmc/articles/PMC2987344/bin/ejhg2009234x1.pdf" data-ga-action="click_feat_suppl">ejhg2009234x1.pdf</a><span style="color:gray"> (27K)</span></div><div class="small guid">GUID:&nbsp;C5CB5FC7-2E52-43D1-9FED-0DBE8781BD24</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 2.</strong></div><div><a href="/pmc/articles/PMC2987344/bin/ejhg2009234x2.pdf" data-ga-action="click_feat_suppl">ejhg2009234x2.pdf</a><span style="color:gray"> (148K)</span></div><div class="small guid">GUID:&nbsp;B5CF3A2C-404E-4CA1-B22C-96E04F130A21</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 3.</strong></div><div><a href="/pmc/articles/PMC2987344/bin/ejhg2009234x3.tif" data-ga-action="click_feat_suppl">ejhg2009234x3.tif</a><span style="color:gray"> (4.5M)</span></div><div class="small guid">GUID:&nbsp;F246DF6E-B8D9-45E5-9D0C-4DFFC7E17ED5</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure Legends.</strong></div><div><a href="/pmc/articles/PMC2987344/bin/ejhg2009234x4.doc" data-ga-action="click_feat_suppl">ejhg2009234x4.doc</a><span style="color:gray"> (37K)</span></div><div class="small guid">GUID:&nbsp;A76A0EE2-4CC5-41B0-91F4-77FCE6218F31</div></div></dd></dl></div><div id="idm140398730714336" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140398730714336title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">This study aimed at contributing to the elucidation of the genetic basis of age-related hearing impairment (ARHI), a common multifactorial disease with an important genetic contribution as demonstrated by heritability studies. We conducted a genome-wide association study (GWAS) in the Finnish Saami, a small, ancient, genetically isolated population without evidence of demographic expansion. The choice of this study population was motivated by its anticipated higher extent of LD, potentially offering a substantial power advantage for association mapping. DNA samples and audiometric measurements were collected from 352 Finnish Saami individuals, aged between 50 and 75 years. To reduce the burden of multiple testing, we applied principal component (PC) analysis to the multivariate audiometric phenotype. The first three PCs captured 80% of the variation in hearing thresholds, while maintaining biologically important audiometric features. All subjects were genotyped with the Affymetrix 100 K chip. To account for multiple levels of relatedness among subjects, as well as for population stratification, association testing was performed using a mixed model. We summarised the top-ranking association signals for the three traits under study. The top-ranked SNP, rs457717 (<em>P</em>-value 3.55 × 10<sup>−7</sup>), was associated with PC3 and was localised in an intron of the IQ motif-containing GTPase-activating-like protein (<em>IQGAP2</em>). Intriguingly, the SNP rs161927 (<em>P</em>-value 0.000149), seventh-ranked for PC1, was positioned immediately downstream from the metabotropic glutamate receptor-7 gene (<em>GRM7</em>). As a previous GWAS of a European and Finnish sample set already suggested a role for <em>GRM7</em> in ARHI, this study provides further evidence for the involvement of this gene.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Saami, isolated population, mixed model, genome-wide association study, age-related hearing impairment, presbycusis</span></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p4" class="p p-first">Age-related hearing impairment (ARHI), or presbycusis, is the most frequent sensory disability among the elderly.<sup><a href="#bib1" rid="bib1" class=" bibr popnode">1</a>, <a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921676">2</a></sup> The involvement of genetic factors in the development of this multifactorial disorder was demonstrated by several heritability studies.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921663">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921657">4</a>, <a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921705">5</a></sup> Only recently have the first studies aiming at the identification of these genetic factors been conducted. Hitherto, three linkage studies have been performed,<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921690">6</a>, <a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921708">7</a>, <a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921716">8</a></sup> but only one region on chromosome 8q24.13–q24.22 reached genome-wide significance.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921685">8</a></sup> In addition, several association studies based on a candidate gene approach and two genome-wide association studies (GWAS)<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921694">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921698">9</a></sup> have been conducted. Early candidate gene-based association studies are reviewed by Van Eyken <em>et al.</em><sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921674">10</a></sup> Recently, based on a candidate gene approach, we reported associations between ARHI and variants in <em>GRHL2</em>,<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921680">11</a></sup> encoding a transcription factor that is also involved in monogenic hearing loss. In addition, based on a pooling GWAS, we reported association with variants in the glutamate receptor-7 gene (<em>GRM7</em>),<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921706">9</a></sup> encoding a metabotropic glutamate receptor assumed to modulate hair cell excitability and synaptic efficacy. Acquired mitochondrial mutations have also been proposed as a cause of ARHI in humans. These include both the ‘common' 4977-bp mitochondrial DNA deletion associated with ageing<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921712">12</a></sup> and accumulation of many different mitochondrial mutations.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921677">13</a></sup></p><p id="__p5">Despite these first series of studies, the genetic aetiology of ARHI remains elusive. In this paper, we present the results of a GWAS for ARHI in the Finnish Saami population. The Saami, previously known as Lapps, live across Northern Scandinavia and the Kola Peninsula. Traditionally, Saami have been fishers, hunters and reindeer herders. A minority of the Saami continue to be engaged in these traditional economic activities, whereas the majority has adopted other occupations and a more westernised lifestyle. Among European populations, the Saami are considered a population genetic ‘outlier'. Despite the fact that both the Saami and Finnish languages belong to the Uralic language group, there is strong genetic differentiation between the Saami and Finns.<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921702">14</a></sup> Several genetic studies have been conducted to shed light on the genetic origin of the Saami.<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921709">15</a>, <a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921689">16</a>, <a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921717">17</a>, <a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921679">18</a></sup> These studies suggested that the Saami are descendants of Europeans who, several thousand years ago, reached northern Fennoscandia from both a western and an eastern route and subsequently got admixed. They also found a small East Asian contribution to the gene pool.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921700">18</a>, <a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921691">19</a></sup> Several genetic studies have further suggested that the extent of linkage disequilibrium (LD) in the Saami is dramatically higher compared to that of outbred populations.<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921667">20</a>, <a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921681">21</a>, <a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921701">22</a>, <a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921665">23</a></sup> Higher levels of LD are observed in other population isolates as well.<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921656">24</a>, <a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921670">25</a></sup> Most of the genetic studies of genetically isolated populations were conducted involving young founder populations, in which the high extent of LD can be attributed to a founder effect. However, in ancient, small, genetically isolated populations without evidence of demographic expansion, high levels of LD are believed to have been generated by genetic drift.<sup><a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921704">26</a>, <a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921714">27</a></sup> The Saami represent such a population. Therefore, the choice of this study population was based on its anticipated statistical power advantage for association mapping, offering better genomic coverage for a given set of markers, as compared to outbred populations.</p><p id="__p6" class="p p-last">Owing to the expected higher levels of LD in the Saami population and based on what was known about the structure of the human genome at the time this study was designed, we anticipated that the Affymetrix 100 K array pair would entail sufficient genomic coverage for identification of putative ARHI-susceptibility genes.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Subjects</h3><p id="__p7" class="p p-first-last">Saami subjects, aged between 50 and 75 years, were enrolled through the public population register from four municipalities in Northern Finland (Ivalo, Utsjoki, Vuotso and Enontekiö) in three stages. In the first stage, geographical criteria were applied; only subjects from an area with a high probability of finding Saami individuals were selected. In the second stage, putative study subjects were invited based on an expert evaluation of Saami communities. Finally, the Saami identity of the subject in question was confirmed through a discussion involving the subject and the researcher. Except for the age range that was used, the ARHI selection criteria were as described previously.<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921684">28</a></sup> Briefly, all volunteers completed an extended questionnaire detailing medical history and exposure to environmental factors putatively influencing hearing. Subjects suffering from one or more medical pathologies from a standardised list of conditions that could potentially affect hearing ability were excluded. All subjects who passed this initial screening underwent an otoscopic investigation before pure-tone audiometry. Air conduction thresholds were measured at 0.125, 0.25, 0.5, 1, 2, 3, 4, 6 and 8 kHz, and bone conduction at 0.5, 1, 2 and 4 kHz for all participating volunteers. Audiological exclusion criteria were an air-bone gap of more than 15 dB averaged over 0.5, 1 and 2 kHz in at least one ear or asymmetrical hearing impairment with a difference in air conduction thresholds exceeding 20 dB in at least two frequencies between 0.5, 1 and 2 kHz. Written informed consent was obtained from all volunteers. The study was approved by the Finnish National Advisory Board on Health Care Ethics and by the ethical committees or the appropriate local institutional review boards at all participating institutions.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Description of the phenotype</h3><p id="__p8" class="p p-first">For each subject, only hearing thresholds from the best ear, defined as the ear with the lowest average threshold over all measured frequencies, were used in the analyses. Supplementary Figure 1 shows that hearing thresholds increase with age and that, for the higher frequencies, thresholds for men are higher and display greater variability (i.e., there is heteroscedasticity) than those for females. We therefore adjusted hearing thresholds for age and sex. The effect of age was removed by regressing log-transformed threshold values (log<sub>10</sub>(20+threshold)) for each of the measured frequencies on age, age squared and age cubed. To resolve the heteroscedasticity problem and to correct for sex, this adjustment for age was carried out for males and females separately, after which the residuals of each regression were scaled and subsequently combined. Note that adding the age and sex covariates in the model for association testing does not deal with this problem satisfactorily.</p><p id="__p9">To deal with the multivariate audiometric phenotype, we applied principal component (PC) analysis as a data-reduction technique. Performing association tests for each frequency separately would have resulted in a large number of tests, worsening the multiple-testing problem. In addition, such analysis would have completely ignored the correlation structure of the audiometric data, that is, the shape of the audiogram. The latter would also apply when simple pure-tone averages (PTAs), which are more widely used in hearing research, would have been analysed. Our first PC (PC1) was highly correlated with the PTA over all frequencies, adjusted for sex, age, age square and age cubed (Spearman correlation=0.96). Spearman correlations between PC1 and sex- and age-adjusted PTAs for the high (2–8 kHz) and low frequencies (0.125–1 kHz) were 0.88 and 0.61, respectively. The association testing results for PC1, therefore, would not differ much from those that would be obtained when PTAs would have been analysed.</p><p id="__p10" class="p p-last"><a href="/pmc/articles/PMC2987344/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> lists the eigenvalues and eigenvectors for the first three PCs, PC1 to PC3. The results of separate calculations for males and females yielded similar values, justifying data pooling (results not shown). Each of the PCs provides independent bits of information. The first three PCs were retained for the analysis as they captured approximately 80% of the total variation. The coefficients of the eigenvectors (loadings) are directly proportional to the correlations between the transformed hearing thresholds corrected for age and gender, and each of the PCs (see, for example, Johnson and Wichern<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup>). PC1 is clearly a ‘size' variable, providing an overall measure of a subject's hearing ability. PC2 and PC3 are ‘shape' variables. PC2 contrasts low with high frequencies and is a measure of the slope of the audiogram. PC3 contrasts the middle frequencies with the lower and higher frequencies and can be considered a measure of the concavity of an audiogram (<a href="/pmc/articles/PMC2987344/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC2987344/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>). These phenotypes are biologically meaningful<sup><a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322803052">30</a>, <a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322803047">31</a></sup> and we have previously shown that they have significant heritabilities.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921682">8</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC2987344/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398732297520" class="figure"><a href="/pmc/articles/PMC2987344/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2987344_ejhg2009234f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg2009234f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2987344/bin/ejhg2009234f1.jpg"></a></div><div id="largeobj_idm140398732297520" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2987344/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC2987344/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p29">Comparison of typical audiograms between subjects with high and low values for PC1, PC2 and PC3. For each of the three traits, three subjects from the lower and upper 10% extremes of the distribution of the trait are shown. The audiograms for both ears are given, to illustrate the extent of intra-subject variability. The left and right ears correspond to the filled and open circles, respectively. For PC1, on average, hearing loss is much more pronounced in individuals from the higher extremes (top row) as compared with those from the lower extremes (second row from the top). Subjects with low values for PC2 (fourth row from the top) tend to have a much more sloping audiogram, compared with those with high values for PC2 (third row from the top). Subjects with high values for PC3 (second row from the bottom) tend to have a more concave audiogram, compared with those in the lower extreme for PC3 (bottom row). Note that the displayed audiometric data have not been adjusted for sex and age.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Eigenvectors and eigenvalues for the first three PCs calculated from age- and sex-corrected hearing thresholds</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398732294816" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Principal component (PC)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>PC1</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>PC2</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>PC3</em></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Eigenvalue</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.421</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.790</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.904</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">% of variation explained<sup><a href="#t1-fn1" rid="t1-fn1" class=" fn">a</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.491</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.199</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.100</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td></tr><tr><td colspan="4" charoff="50" rowspan="1" valign="top" align="left"><em>Coefficients of eigenvectors (loadings)</em><sup><a href="#t1-fn2" rid="t1-fn2" class=" fn">b</a></sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res125</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.208</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.460</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.527</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res250</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.288</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.470</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.287</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res500</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.333</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.382</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.186</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res1000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.348</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.186</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.481</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res2000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.373</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.020</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.408</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res3000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.393</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.237</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.109</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res4000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.358</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.324</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.088</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res6000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.356</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.328</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.276</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> res8000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.305</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.345</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.335</td></tr></tbody></table></div><div id="largeobj_idm140398732294816" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2987344/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1-fn1"><sup>a</sup>Eigenvalue for the PC divided by the sum of all eigenvalues, which corresponds to the total number of variables in the original dataset (9).</div><div id="t1-fn2"><sup>b</sup>The correlations between the PC and the age- and sex-corrected, transformed hearing thresholds for each frequency can be obtained by multiplying each coefficient by the square root of its corresponding eigenvalue.</div></div></div></div><div id="__sec5" class="sec"><h3 id="__sec5title">Genotyping and quality control of data</h3><p id="__p11" class="p p-first-last">Genomic DNA was extracted using standard protocols and diluted to 50 ng/<em>μ</em>l. The quality of the genomic DNA was assessed by spectrophotometric analysis and gel electrophoresis. Each sample was genotyped with the Affymetrix GeneChip Human Mapping 100 K array pair (Santa Clara, CA, USA), as described in the GeneChip Human Mapping 100 K Assay Manual. Genotype calling was performed using the BRLMM algorithm of the Affymetrix GeneChip Genotyping Analysis Software (GTYPE) version 4.1. Quality control of data was performed based on the following criteria: a sample call rate of &gt;94%, a marker call rate of &gt;90%, a Hardy–Weinberg equilibrium <em>P</em>-value of &gt;0.001 and a minor allele frequency of &gt;5%. In total, 352 samples from the Saami subjects were genotyped with the Affymetrix 100 K array. One subject was excluded because of low sample call rate. The total genotyping rate in the remaining samples was 0.9931. One additional sample was removed owing to an unintentional duplication event, and three further subjects were excluded owing to missing phenotypic information. After data quality control, 347 subjects and 83 381 SNPs were used for statistical analysis. Genotyping for SNP rs11928865 was performed using a Taqman assay (Applied Biosystems, Foster City, CA, USA) with a Lightcycler480 real-time PCR machine (Roche, Basel, Switzerland).</p></div><div id="__sec6" class="sec sec-last"><h3 id="__sec6title">Association testing</h3><p id="__p12" class="p p-first-last">Estimates of genome-wide pairwise identity-by-descent sharing, provided by PLINK,<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921697">32</a></sup> revealed a substantial degree of undocumented relatedness among the subjects. This was also reflected in an excess of SNPs that were not in Hardy–Weinberg equilibrium. The results of a linear regression for each of the three quantitative traits would, therefore, not be trustworthy as dependent data lead to underestimation of the standard errors for the regression coefficients, yielding <em>P</em>-values that are too small. In addition, population stratification may confound the results of the analysis. To account for the dependency of the data, as well as for population stratification, we analysed the data using a mixed model.<sup><a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921718">33</a></sup> With this approach, the covariance matrix of the phenotype is modelled as: </p><div class="disp-formula" id="equ1"><div class="f"><img src="/pmc/articles/PMC2987344/bin/ejhg2009234e1.jpg" class="eq-image" alt="equation image" title="equation image"></div></div><p> where <em>y</em> is the column vector with the phenotypes of the 347 subjects, <em>K</em> is a matrix with kinship coefficients estimated based on the genome-wide data, <em>I</em> is an identity matrix and <em>σ</em><sub><em>g</em></sub><sup>2</sup> and <em>σ</em><sub><em>e</em></sub><sup>2</sup> are the genetic and the residual variances, respectively. This was implemented using the MIXED procedure of SAS 9.1.3 within SAS macro, which was based on that used by Yu <em>et al.</em><sup><a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921668">33</a></sup> The 2<em>K</em> matrix with the proportion of alleles shared identical-by-descent was estimated using PLINK.<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921672">32</a></sup> The model also corrected for origin. As samples originated from four municipalities, three dummy variables were included as covariates in the model. An additive genetic model was assumed for each SNP. As it was computationally infeasible to run this analysis for the entire set of SNPs, for each trait only the top 4000 SNPs (approximately 5% of the total number of SNPs passing quality control) resulting from an initial screening based on linear regression analysis (naive analysis) were considered for the mixed model analysis. To verify that no true association signals were lost by using this analytical strategy, we first examined how well the <em>P</em>-values of the naive analysis correlated with those of the mixed model analysis. For this purpose, we analysed 8000 randomly selected SNPs meeting the quality control standards, representing approximately 10% of the SNPs that passed the quality control, with the mixed model. Comparison of <em>P</em>-values between naive analysis and mixed-model results showed that no SNPs with highly significant <em>P</em>-values in the mixed-model analysis had a non-significant <em>P</em>-value in the naive analysis. A reciprocal relationship would be attributed to inflated test statistics. Therefore, for the data at hand, by analysing the top 5% (4000 SNPs) of a naive linear regression analysis, the chance of missing any SNP that would not be significant in the naive analysis, but highly significant in the mixed-model analysis seems very low (Supplementary Figure 2). Quantile–quantile plots in Supplementary Figure 2 show that residual population stratification or cryptic relatedness is of minor concern. Genomic control inflation factors varied from 1.02 to 1.08. The graph on the right of each panel in Supplementary Figure 2 further shows that <em>P</em>-values obtained from the naive analysis tend to be too low. For example, for PC1, the most extreme −log<sub>10</sub>(<em>P</em>-value) from the naive analysis is 4, whereas it is 3.5 for the mixed model.</p></div></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title">Results</h2><p id="__p13" class="p p-first">After quality control, 347 subjects and 83 381 SNPs were used for association testing. The quantitative traits tested were the first three PCs calculated from the log-transformed hearing threshold values, adjusted for age and sex. To account for the multiple levels of relatedness present in our data and potential population stratification, the associations between SNPs and the traits were tested using a mixed-model approach, based upon genome-wide kinship estimates. Owing to computational constraints, only the top 4000 SNPs (approximately 5% of the total number of SNPs passing quality control) resulting from an initial screening based on a linear regression analysis were used in the mixed-model analysis. In the Materials and methods section, we have outlined that, using this strategy, it is unlikely that strong association signals will be lost. For each of the three PCs, the top 25 SNPs resulting from this analysis are listed in <a href="/pmc/articles/PMC2987344/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Tables 2</span></a>, <a href="/pmc/articles/PMC2987344/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,3,</span></span><span>3</span></a>, <a href="/pmc/articles/PMC2987344/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,4</span></span><span>4</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Top 25 SNPs resulting from the mixed-model analysis for PC1</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398730235168" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Chr.</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Position</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Alleles</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Minor allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Genotypes</em><sup><a href="#t2-fn2" rid="t2-fn2" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P HWE</em><sup><a href="#t2-fn3" rid="t2-fn3" class=" fn">b</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P Mixed</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Gene region</em><sup><a href="#t2-fn4" rid="t2-fn4" class=" fn">c</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Pos. Rel. gene</em><sup><a href="#t2-fn5" rid="t2-fn5" class=" fn">d</a></sup></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1413042</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">38 600 755</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.228916</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">24/104/204</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0432</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>2.91E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Similar to RIKEN cDNA 8030451K01 (LOC387921)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1016111</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14 121 677</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.367521</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">49/160/142</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.7308</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>5.86E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">CD83 antigen precursor (cell-surface protein HB15)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1826882</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">19 989 036</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.113506</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1/77/270</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1024</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>9.41E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs854273</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">69 016 062</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.247863</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18/138/195</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3898</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000115</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">DEP domain containing-1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2825601</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">19 766 620</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.336182</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">39/158/154</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9054</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000134</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1988145</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">92 258 480</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.228228</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">25/102/206</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0195</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000144</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">COUP transcription factor 1 (COUP-TF1)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9311746</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">60 015 052</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.381766</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">50/168/133</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8221</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000144</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Bis(5′-adenosyl)-triphosphatase (EC 3.6.1.29)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs161927</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7 813 242</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.065407</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3/39/302</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1663</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000149</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Metabotropic glutamate receptor-7 precursor (GRM7)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1358520</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">181 148 326</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.195157</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9/119/223</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1738</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000187</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Ubiquitin-conjugating enzyme E2 E3 (EC 6.3.2.19)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2148418</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">38 564 233</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.215714</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">20/111/219</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2675</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000194</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Similar to RIKEN cDNA 8030451K01 (LOC387921)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1661609</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">52 720 759</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.150000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7/91/252</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8359</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000208</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs572619</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">115 738 853</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.361823</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">45/164/142</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9080</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000211</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">BUD13 homologue</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10510777</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">55 212 778</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.472934</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">76/180/95</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.6684</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000227</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Calcium channel, voltage-dependent, alpha-2/delta-3 subunit</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2182406</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">129 456 655</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.133903</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8/78/265</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.4864</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000231</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Transmembrane protein-12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9301169</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">106 353 066</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.250712</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">21/134/196</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8870</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000234</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1388654</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">68 985 603</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.295714</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">26/155/169</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3039</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000249</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">DEP domain containing-1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs3751591</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">49 394 002</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.148415</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14/75/258</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0100</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000250</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Cytochrome P450 19A1 (EC 1.14.14.1)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7620394</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">55 206 368</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.304843</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">33/148/170</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9003</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000253</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Leucine-rich repeats and transmembrane domains-1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10503920</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">32 548 231</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.274929</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">34/125/192</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0454</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000259</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Neuregulin-1, sensory and motor neuron-derived factor isoform</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10497334</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">168 549 782</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.081197</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1/55/295</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.7146</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000266</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">STE20/SPS1-related, proline–alanine-rich protein kinase (EC 2.7.11.1)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7357631</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">99 568 141</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.497151</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">87/175/89</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000282</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Forkhead box protein-E1 (thyroid transcription factor-2)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2055439</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">75 511 014</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.403458</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">61/158/128</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3166</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000292</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Synaptic vesicle glycoprotein-2C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10515629</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">148 875 068</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.196581</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">15/108/228</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.6132</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000303</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Casein kinase-I isoform-alpha (EC 2.7.11.1)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2037344</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">121 088 918</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.348665</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">39/157/141</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.7191</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000339</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">DEP domain containing-6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs3912622</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">102 626 683</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.111748</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8/62/279</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0570</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000355</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">DYNC2H1 protein</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr></tbody></table></div><div id="largeobj_idm140398730235168" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2987344/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t2-fn1"><p id="__p32" class="p p-first-last">Abbreviations: Chr., chromosome; MAF, minor allele frequency; P HWE, <em>P</em>-value resulting after Hardy–Weinberg equilibrium testing; P Mixed, <em>P</em>-value resulting after mixed-model analysis (indicated in bold); Pos. Rel. Gene, position of the SNP relative to the gene listed in the previous column.</p></div><div id="t2-fn2"><sup>a</sup>The number of homozygotes for the minor allele/the number of heterozygotes/the number of homozygotes for the major allele.</div><div id="t2-fn3"><sup>b</sup>The Hardy–Weinberg equilibrium <em>P</em>-value should be larger than 0.001 for the SNP genotyping to be valid.</div><div id="t2-fn4"><sup>c</sup>‘/' indicates that this SNP was not located within a gene region.</div><div id="t2-fn5"><sup>d</sup>‘Upstream' or ‘Downstream' was based on the Affymetrix annotation.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Top 25 SNPs resulting from the mixed model analysis for PC2</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398768148464" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Chr.</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Position</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Alleles</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Minor allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Genotypes</em><sup><a href="#t3-fn2" rid="t3-fn2" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P HWE</em><sup><a href="#t3-fn3" rid="t3-fn3" class=" fn">b</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P mixed</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Gene region</em><sup><a href="#t3-fn4" rid="t3-fn4" class=" fn">c</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Pos. Rel. gene</em><sup><a href="#t3-fn5" rid="t3-fn5" class=" fn">d</a></sup></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10499138</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">128 533 875</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.253561</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">30/118/203</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0473</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>1.70E−06</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Receptor-type tyrosine-prot phosphatase kappa precursor (EC 3.1.3.48)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10491923</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">76 139 000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.365672</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">49/147/139</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3460</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>2.79E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Nuclear receptor ROR-beta (nuclear receptor RZR-beta)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10499136</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">128 419 828</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.296296</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">36/136/179</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2002</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>3.58E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Receptor-type tyrosine-prot phosphatase-kappa precursor (EC 3.1.3.48)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9308868</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">103 727 852</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.268895</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">25/135/184</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>6.45E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">U3 small nucleolar RNA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1555167</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">62 465 026</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.398860</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">57/166/128</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8239</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>9.62E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">KH dom-containing, RNA-binding signal transduction-associated prot-2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9294311</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">62 470 112</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.398860</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">57/166/128</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8239</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>9.62E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">KH dom-containing, RNA-binding, signal transduction-associated prot-2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9294314</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">62 470 480</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.398860</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">57/166/128</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8239</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>9.62E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">KH dom-containing, RNA-binding, signal transduction-associated prot-2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10485162</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">128 639 684</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.145299</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12/78/261</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0530</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000122</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Receptor-type tyrosine-prot phosphatase-kappa precursor (EC 3.1.3.48)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs514878</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">73 705 460</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.058405</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2/37/312</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3294</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000132</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Ectoderm-neural cortex 1 protein (ENC-1)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10520939</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">27 970 143</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.082813</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5/43/272</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0516</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000149</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">U6 spliceosomal RNA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1201494</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">62 547 842</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.398571</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">56/167/127</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9116</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000165</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">KH dom-containing, RNA-binding, signal transduction-associated prot-2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs304408</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1 095 249</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.245588</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">28/111/201</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0288</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000175</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Putative C18 or f2 variant-3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs768166</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8 087 556</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.228198</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18/121/205</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000175</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Transcription initiation factor TFIID subunit-3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs304409</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1 096 342</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.246356</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">25/119/199</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2439</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000180</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Pituitary adenylate cyclase-activating polypeptide precursor (PACAP)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4936417</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">117 409 046</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.200000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">20/100/230</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0460</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000202</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Eukaryotic-type signal recognition particle RNA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4386936</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">135 859 109</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.219373</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">15/124/212</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.6414</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000221</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Zinc finger prot 406 (Protein ZFAT)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs211004</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">161 577 376</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.370640</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">53/149/142</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2033</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000222</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Gamma-aminobutyric-acid receptor gamma-2 subunit precursor</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs522914</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">62 592 457</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.400285</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">57/167/127</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9115</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000224</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">KH dom-containing, RNA-binding, signal transduction-associated prot-2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs565795</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">62 597 708</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.400285</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">57/167/127</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9115</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000224</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">KH dom-containing, RNA-binding, signal transduction-associated prot-2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs265360</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">129 579 569</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.272727</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">33/120/188</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0404</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000247</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Laminin-alpha-2 chain precursor (laminin M-chain)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9312176</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">67 882 411</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.456522</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">74/167/104</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.6647</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000254</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Similar to interferon-induced transmembrane prot-3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1544044</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">20</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14 459 168</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.465318</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">72/178/96</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.5892</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000257</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">OTTHUMP00000030317</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10493937</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">101 115 238</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.472934</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">85/162/104</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1650</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000267</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Exostosin-like-2 (EC 2.4.1.223)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1358815</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">57 471 407</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.464183</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">73/178/98</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.6680</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000274</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">DNA primase large subunit (EC 2.7.7.-)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6489314</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1 627 431</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A/C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.341642</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">47/139/155</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0916</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000300</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Adiponectin receptor prot-2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr></tbody></table></div><div id="largeobj_idm140398768148464" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2987344/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t3-fn1"><p id="__p37" class="p p-first-last">Abbreviations: Chr., chromosome; dom, domain; MAF, minor allele frequency; P HWE, <em>P</em>-value resulting after Hardy–Weinberg equilibrium testing; P Mixed, <em>P</em>-value resulting mixed-model analysis (indicated in bold); Pos. Rel. Gene, position of the SNP relative to the gene listed in the previous column; prot, protein.</p></div><div id="t3-fn2"><sup>a</sup>The number of homozygotes for the minor allele/the number of heterozygotes/the number of homozygotes for the major allele.</div><div id="t3-fn3"><sup>b</sup>The Hardy–Weinberg equilibrium <em>P</em>-value should be larger than 0.001 for the SNP genotyping to be valid.</div><div id="t3-fn4"><sup>c</sup>‘/' indicates that this SNP was not located within a gene region.</div><div id="t3-fn5"><sup>d</sup>‘Upstream' or ‘Downstream' was based on the Affymetrix annotation.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Top 25 SNPs resulting from the mixed model analysis for PC3</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398768278960" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Chr.</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Position</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Alleles</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Minor allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Genotypes</em><sup><a href="#t4-fn2" rid="t4-fn2" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P HWE</em><sup><a href="#t4-fn3" rid="t4-fn3" class=" fn">b</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P mixed</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Gene region</em><sup><a href="#t4-fn4" rid="t4-fn4" class=" fn">c</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Pos. Rel. gene</em><sup><a href="#t4-fn5" rid="t4-fn5" class=" fn">d</a></sup></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs457717</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">75 956 728</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.251429</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">26/124/200</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2589</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>3.55E−07</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Ras GTPase-activating-like prot IQGAP2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1697845</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">75 958 260</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.293447</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">37/132/182</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0935</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>1.63E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Ras GTPase-activating-like prot IQGAP2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs989414</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">151 552 224</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.061960</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3/37/307</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1326</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>3.72E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">CDNA FLJ45645 fis, clone CTONG2003517</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9310194</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">70 713 297</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.334770</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">35/163/150</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3994</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>4.96E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Microphthalmia-associated transcription factor</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10514104</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">76 843 891</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.113506</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4/71/273</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>5.08E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7913244</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10 224 371</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.102857</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5/62/283</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3902</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>5.69E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2299641</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">17 397 566</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.200000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14/112/224</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>7.90E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">ATP-binding cassette transporter subfamily-C member-8</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10494078</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">108 157 301</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.472303</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">72/180/91</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3860</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>9.16E−05</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Prot vav-3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4636135</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">103 406 504</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.246418</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">19/134/196</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.6650</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000100</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Reelin prec (EC 3.4.21.-)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6539455</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">107 744 032</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.115607</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9/62/275</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0310</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000101</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Prot phosphatase Slingshot homologue-1 (EC 3.1.3.48)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10497072</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">151 581 781</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.058405</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1/39/311</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000135</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Rho-related GTP-binding prot RhoE (Rho family GTPase-3)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4664883</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">151 583 181</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.058405</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1/39/311</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000135</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">CDNA FLJ45645 fis, clone CTONG2003517</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1500237</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">98 643 065</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.205128</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">17/110/224</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.5116</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000145</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Y RNA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6433104</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">151 559 112</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.058908</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1/39/308</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000151</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Rho-related GTP-binding prot RhoE (Rho family GTPase-3)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10483426</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">32 670 589</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.452991</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">79/160/112</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1326</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000163</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Neuronal PAS domain-containing prot-3 (neuronal PAS3)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1940484</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">98 447 082</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.475714</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">82/169/99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.5920</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000164</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9301099</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">105 483 574</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.169516</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14/91/246</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1321</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000183</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2527756</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5 390 145</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.188571</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10/112/228</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.4856</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000199</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10512077</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">80 741 480</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.417379</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">67/159/125</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2269</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000201</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs42064</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">95 557 193</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.319088</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">42/140/169</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1399</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000210</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Cytoplasmic dynein-1 intermediate chain-1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1363556</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">144 886 649</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.498457</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">90/143/91</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0351</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000216</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">7SK RNA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1260764</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">64 316 010</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">G/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.176638</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12/100/239</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.7133</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000221</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Tyrosine-prot kinase transmembrane receptor ROR1 prec (EC 2.7.10.1)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1260772</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">64 313 964</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">A/G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.176638</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12/100/239</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.7133</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000221</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Tyrosine-prot kinase transmembrane receptor ROR1 prec (EC 2.7.10.1)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1559776</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">55 746 920</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.150997</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7/92/252</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8356</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000236</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">GRINL1A combined prot isoforms-1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4809590</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">20</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">44 518 234</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">C/T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.102564</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7/58/286</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0732</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><strong>0.000253</strong></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">/</td></tr></tbody></table></div><div id="largeobj_idm140398768278960" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2987344/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t4-fn1"><p id="__p42" class="p p-first-last">Abbreviations: Chr., chromosome; MAF, minor allele frequency; P HWE, <em>P</em>-value resulting after Hardy–Weinberg equilibrium testing; P Mixed, <em>P</em>-value resulting after mixed-model analysis (indicated in bold); Pos. Rel. Gene, position of the SNP relative to the gene listed in the previous column; prot, protein; prec, precursor.</p></div><div id="t4-fn2"><sup>a</sup>The number of homozygotes for the minor allele/the number of heterozygotes/the number of homozygotes for the major allele.</div><div id="t4-fn3"><sup>b</sup>The Hardy–Weinberg equilibrium <em>P</em>-value should be larger than 0.001 for the SNP genotyping to be valid.</div><div id="t4-fn4"><sup>c</sup>‘/' indicates that this SNP was not located within a gene region.</div><div id="t4-fn5"><sup>d</sup>‘Upstream' or ‘Downstream' was based on the Affymetrix annotation.</div><div id="t4-fn6"><p id="__p47" class="p p-first-last"><em>P</em>-values are indicated in bold.</p></div></div></div><p id="__p14">If, in agreement with other GWAS,<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921660">34</a></sup> we used a <em>P</em>-value cut-off of 5 × 10<sup>−7</sup> to declare genome-wide significance, the top-ranked SNP for PC3 (rs457717; <a href="/pmc/articles/PMC2987344/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>) survived this significance threshold (<em>P</em>-value=3.55 × 10<sup>−7</sup>). This SNP is located in an intron of the IQ motif-containing GTPase-activating-like protein (<em>IQGAP2</em>). The second-ranked SNP (rs1697845) is also positioned in <em>IQGAP2</em>, and is highly correlated with the first (<em>D</em>′=1; <em>r</em><sup>2</sup>=0.965), indicating that this signal is not a genotyping artefact.</p><p id="__p15">For the other two traits, PC1 and PC2, none of the SNPs reached this level of significance (<a href="/pmc/articles/PMC2987344/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC2987344/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3).</span></span><span>3</span></a>). Intriguingly, however, the eighth-ranked SNP for PC1 (rs161927; <a href="/pmc/articles/PMC2987344/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>; <em>P</em>-value 0.000149) was positioned 40 kb downstream from the metabotropic <em>GRM7</em> gene (Supplementary Figure 3). We have previously reported an association of a common variant in the <em>GRM7</em> gene, rs11928865, with hearing ability based on a pooling-based GWAS of a European sample set.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921696">9</a></sup> Therefore, we also genotyped this SNP in the Saami population and analysed these data using the mixed-model approach. This resulted in a <em>P</em>-value of 0.045. However, the A-allele was associated with an elevated hearing threshold in the Saami sample set, while in the European sample set, the associated allele was T.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921710">9</a></sup> One of the two top-scoring <em>GRM7</em> SNPs resulting from a GWAS of a Finnish sample set,<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921671">9</a></sup> rs779701, was present on the 100 K Affymetrix array. However, in the Saami sample set, this SNP did not result in a significant association (<em>P</em>-value of 0.735). The LD pattern within the <em>GRM7</em> region deduced for the Saami (Supplementary Figure 3D) was similar to that observed in the HapMap for the European population (Supplementary Figure 3C). No obvious biological meaning related to the inner ear could be deduced for the gene regions of the seven higher-ranked SNPs for PC1.</p><p id="__p16" class="p p-last">For PC2, no strong biological candidates were present among the top 25 SNPs (<a href="/pmc/articles/PMC2987344/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>). The presence of six SNPs from the same gene (KH domain-containing, RNA-binding, signal transduction-associated protein-2; <em>KHDRBS2</em>) in <a href="/pmc/articles/PMC2987344/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a> is due to the fact that all of these SNPs reside in the same region of high LD.</p></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Discussion</h2><p id="__p17" class="p p-first">We conducted a genome-wide association scan for ARHI traits derived from audiometric data in the genetically isolated Finnish Saami population. We summarised the information contained in each audiogram by the first three components obtained through PC analysis. Previously, we have shown that these traits have significant heritabilities.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921683">8</a></sup> Using these audiometric measures, we retain biologically important information on the shape of the audiogram that would be lost when simple PTAs or individual frequencies are analysed. Indeed, premortem-derived audiometric patterns have been correlated to pathophysiological findings in human temporal bones.<sup><a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322803051">30</a>, <a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322803048">31</a></sup> Nelson <em>et al</em><sup><a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921699">35</a></sup> studied individuals with downward-sloping audiograms and reported that the severity of hearing loss, based on audiometric thresholds, was highly associated with a decrease in stria vascularis volume and degeneration of outer and inner hair cell and ganglion cell populations. The slope of the audiogram was significantly associated with the extent of ganglion cell degeneration.</p><p id="__p18">Our association scan resulted in two SNPs in or near genes that are reasonable biological candidates and that are candidates for further research. The top-ranked SNP, rs457717 (<em>P</em>-value 3.55 × 10<sup>−7</sup>), that reached genome-wide significance, was associated with PC3 and was localised in an intron of the IQ motif-containing, GTPase-activating-like protein (<em>IQGAP2</em>). IQGAP2 is a member of the Ras superfamily of GTPases. This class of proteins regulates a wide variety of cellular signalling pathways. IQGAP2 is expressed inside the cochlea<sup><a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921678">36</a></sup> and has been implicated in cadherin-mediated cell adhesion.<sup><a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921662">37</a></sup> As cadherin-23 (CDH23) plays an important role in the sensory hair cells,<sup><a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921664">38</a></sup> and mutations in <em>CDH23</em> are responsible for syndromic and non-syndromic autosomal recessive deafness in humans<sup><a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921687">39</a>, <a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921688">40</a></sup> and age-related hearing loss in inbred mouse strains,<sup><a href="#bib41" rid="bib41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322803049">41</a></sup> this warrants further research. Intriguingly, the SNP rs161927 (<em>P</em>-value 0.000149), that ranked eighth for PC1, was positioned immediately downstream from the metabotropic <em>GRM7</em> gene. A previous GWAS of a European and a Finnish sample set, which was based on an age- and gender-corrected <em>Z</em>-score highly correlated with our PC1,<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921659">9</a>, <a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921658">11</a>, <a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921713">28</a>, <a href="#bib42" rid="bib42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921711">42</a></sup> already suggested a role for <em>GRM7</em> in ARHI.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921673">9</a></sup> <em>GRM7</em> encodes a metabotropic glutamate receptor that can be activated by -glutamate, the primary excitatory neurotransmitter in the cochlear hair cell. GRM7 expression was observed in the inner and outer hair cells, and in the spiral ganglion nerve cell bodies of the inner ear.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921661">9</a></sup> We have hypothesised that certain <em>GRM7</em> variants may cause increased susceptibility to glutamate excitotoxicity, which may consequently lead to the development of ARHI. However, variants that were significant in our previous study did not replicate in our current study. The LD structure within the <em>GRM7</em> region indicates that these are independent signals. These results, therefore, are consistent with allelic heterogeneity, which is a plausible scenario given the likely absence of any selective pressure for this disorder. Independent studies are needed to investigate the role of common variants in <em>GRM7</em> in ARHI.</p><p id="__p19">This study was conducted involving the genetically isolated Saami population. A number of studies provide evidence that the extent of LD is much higher in the Saami as compared to that in other European populations.<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921695">20</a>, <a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921715">21</a>, <a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921655">22</a>, <a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921693">23</a></sup> This would provide an important power advantage, as higher LD leads to better genomic coverage. It has been proposed that association studies based on a map of modest marker density within a population isolate such as the Saami would provide a first rough identification of the region associated with a particular complex disease. This first demarcation of the region should then preferably be refined by association studies of an outbred population using a denser map.<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921707">22</a></sup> We have published elsewhere the results of an elaborate genome-wide evaluation of the relative power for GWAS in the Saami.<sup><a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322803050">43</a></sup></p><p id="__p20">A possible limitation of this study is the limited sample size. Recent studies indicate that several thousand subjects need to be recruited in order to obtain sufficient power to detect the effect sizes typically expected for complex phenotypes (with each contributing variant explaining as little as 0.1–1.3% of the variation),<sup><a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921703">44</a>, <a href="#bib45" rid="bib45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921675">45</a></sup> although successful GWAS have been conducted using only 96 cases and 50 controls.<sup><a href="#bib46" rid="bib46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921669">46</a></sup> Obviously, large sample sizes can never be recruited in small isolated populations such as the Saami. A specific difficulty of working with isolated populations became apparent in the current study. Owing to the fact that isolated populations are often derived from a limited number of founders and that inbreeding is sometimes inevitable, an increased relatedness is observed within such populations. As a consequence, classical statistical methods that assume unrelatedness, cannot be applied to analyse an association study performed in a population isolate. In our exploratory phase, we investigated several approaches, including a naive linear regression analysis and a PC-based correction (EIGENSTRAT<sup><a href="#bib47" rid="bib47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921686">47</a></sup>). However, for the latter approach, the basic assumptions were violated and the inbreeding and relatedness present in our Saami sample created spurious axes of variation.<sup><a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921666">48</a></sup> Mixed models offered a solution. In this approach, we estimated the genetic relationships between the study subjects based on genome-wide data and subsequently accounted for them in the analysis.<sup><a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201921692">33</a></sup></p><p id="__p21" class="p p-last">In conclusion, we present the results of a first GWAS performed in the genetically isolated Saami population. We identified <em>IQGAP2</em> as a candidate ARHI gene. Independent replication is needed to confirm this lead. In addition, we found further evidence implicating <em>GRM7</em> in ARHI.</p></div><div id="idm140398731039616" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140398731039616title">Acknowledgments</h2><div class="sec"><p id="__p22">We express our most sincere gratitude to all the Saami volunteers who participated in this study. JRH thanks Gael Pressoir and Jianming Yu from the Ed Buckler Lab for providing help with the SAS macro. This study was funded by the European Community (5th Framework project QLRT-2001-00331), the University of Antwerp (TOP project), the Research Foundation – Flanders (FWO grant G.0163.09), and the State of Arizona. JRH is a fellow of the Research Foundation – Flanders (FWO).</p></div></div><div id="idm140398770030352" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140398770030352title">Notes</h2><p id="__p24">The authors declare no conflict of interest.</p></div><div id="idm140398770031136" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140398770031136title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140398770031008"></p><p id="__p23" class="p p-first-last">Supplementary Information accompanies the paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Figure 1</h4><div class="sup-box half_rhythm" id="idm140398804483872"><a href="/pmc/articles/PMC2987344/bin/ejhg2009234x1.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(27K, pdf)</sup></div></div><div class="sec suppmat" id="xob2"><h4>Supplementary Figure 2</h4><div class="sup-box half_rhythm" id="idm140398732312000"><a href="/pmc/articles/PMC2987344/bin/ejhg2009234x2.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(148K, pdf)</sup></div></div><div class="sec suppmat" id="xob3"><h4>Supplementary Figure 3</h4><div class="sup-box half_rhythm" id="idm140398732308224"><a href="/pmc/articles/PMC2987344/bin/ejhg2009234x3.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(4.5M, tif)</sup></div></div><div class="sec suppmat" id="xob4"><h4>Supplementary Figure Legends</h4><div class="sup-box half_rhythm" id="idm140398730392400"><a href="/pmc/articles/PMC2987344/bin/ejhg2009234x4.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(37K, doc)</sup></div></div></div><div id="idm140398770029728" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140398770029728title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Davis A. <span class="ref-title">Prevalence of hearing impairment</span>in Davis A (ed): <span class="ref-journal">Hearing in Adults</span> London: Whurr Publishers Ltd; 199443–321. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Hearing+in+Adults&amp;author=A+Davis&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Gates GA, Mills JH. Presbycusis. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">366</span>:1111–1120.</span> [<a href="/pubmed/16182900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Presbycusis&amp;author=GA+Gates&amp;author=JH+Mills&amp;volume=366&amp;publication_year=2005&amp;pages=1111-1120&amp;pmid=16182900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Christensen K, Frederiksen H, Hoffman HJ. Genetic and environmental influences on self-reported reduced hearing in the old and oldest old. <span><span class="ref-journal">J Am Geriatr Soc. </span>2001;<span class="ref-vol">49</span>:1512–1517.</span> [<a href="/pubmed/11890591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Geriatr+Soc&amp;title=Genetic+and+environmental+influences+on+self-reported+reduced+hearing+in+the+old+and+oldest+old&amp;author=K+Christensen&amp;author=H+Frederiksen&amp;author=HJ+Hoffman&amp;volume=49&amp;publication_year=2001&amp;pages=1512-1517&amp;pmid=11890591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Gates GA, Couropmitree NN, Myers RH. Genetic associations in age-related hearing thresholds. <span><span class="ref-journal">Arch Otolaryngol Head Neck Surg. </span>1999;<span class="ref-vol">125</span>:654–659.</span> [<a href="/pubmed/10367922" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Otolaryngol+Head+Neck+Surg&amp;title=Genetic+associations+in+age-related+hearing+thresholds&amp;author=GA+Gates&amp;author=NN+Couropmitree&amp;author=RH+Myers&amp;volume=125&amp;publication_year=1999&amp;pages=654-659&amp;pmid=10367922&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Karlsson KK, Harris JR, Svartengren M. Description and primary results from an audiometric study of male twins. <span><span class="ref-journal">Ear Hear. </span>1997;<span class="ref-vol">18</span>:114–120.</span> [<a href="/pubmed/9099560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ear+Hear&amp;title=Description+and+primary+results+from+an+audiometric+study+of+male+twins&amp;author=KK+Karlsson&amp;author=JR+Harris&amp;author=M+Svartengren&amp;volume=18&amp;publication_year=1997&amp;pages=114-120&amp;pmid=9099560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">DeStefano AL, Gates GA, Heard-Costa N, Myers RH, Baldwin CT. Genomewide linkage analysis to presbycusis in the Framingham Heart Study. <span><span class="ref-journal">Arch Otolaryngol Head Neck Surg. </span>2003;<span class="ref-vol">129</span>:285–289.</span> [<a href="/pubmed/12622536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Otolaryngol+Head+Neck+Surg&amp;title=Genomewide+linkage+analysis+to+presbycusis+in+the+Framingham+Heart+Study&amp;author=AL+DeStefano&amp;author=GA+Gates&amp;author=N+Heard-Costa&amp;author=RH+Myers&amp;author=CT+Baldwin&amp;volume=129&amp;publication_year=2003&amp;pages=285-289&amp;pmid=12622536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Garringer HJ, Pankratz ND, Nichols WC, Reed T. Hearing impairment susceptibility in elderly men and the DFNA18 locus. <span><span class="ref-journal">Arch Otolaryngol Head Neck Surg. </span>2006;<span class="ref-vol">132</span>:506–510.</span> [<a href="/pubmed/16702566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Otolaryngol+Head+Neck+Surg&amp;title=Hearing+impairment+susceptibility+in+elderly+men+and+the+DFNA18+locus&amp;author=HJ+Garringer&amp;author=ND+Pankratz&amp;author=WC+Nichols&amp;author=T+Reed&amp;volume=132&amp;publication_year=2006&amp;pages=506-510&amp;pmid=16702566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Huyghe JR, Van Laer L, Hendrickx JJ, et al.  Genome-wide SNP-based linkage scan identifies a locus on 8q24 for an age-related hearing impairment trait. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:401–407.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2556434/">PMC free article</a>]</span> [<a href="/pubmed/18760390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+SNP-based+linkage+scan+identifies+a+locus+on+8q24+for+an+age-related+hearing+impairment+trait&amp;author=JR+Huyghe&amp;author=L+Van+Laer&amp;author=JJ+Hendrickx&amp;volume=83&amp;publication_year=2008&amp;pages=401-407&amp;pmid=18760390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">Friedman RA, Van Laer L, Huentelman MJ, et al.  GRM7 variants confer susceptibility to age-related hearing impairment. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:785–796.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2638831/">PMC free article</a>]</span> [<a href="/pubmed/19047183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=GRM7+variants+confer+susceptibility+to+age-related+hearing+impairment&amp;author=RA+Friedman&amp;author=L+Van+Laer&amp;author=MJ+Huentelman&amp;volume=18&amp;publication_year=2009&amp;pages=785-796&amp;pmid=19047183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Van Eyken E, Van Camp G, Van Laer L. The complexity of age-related hearing impairment: contributing environmental and genetic factors. <span><span class="ref-journal">Audiol Neurootol. </span>2007;<span class="ref-vol">12</span>:345–358.</span> [<a href="/pubmed/17664866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Audiol+Neurootol&amp;title=The+complexity+of+age-related+hearing+impairment:+contributing+environmental+and+genetic+factors&amp;author=E+Van+Eyken&amp;author=G+Van+Camp&amp;author=L+Van+Laer&amp;volume=12&amp;publication_year=2007&amp;pages=345-358&amp;pmid=17664866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Van Laer L, Van Eyken E, Fransen E, et al.  The grainyhead like 2 gene (GRHL2), alias TFCP2L3, is associated with age-related hearing impairment. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:159–169.</span> [<a href="/pubmed/17921507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+grainyhead+like+2+gene+(GRHL2),+alias+TFCP2L3,+is+associated+with+age-related+hearing+impairment&amp;author=L+Van+Laer&amp;author=E+Van+Eyken&amp;author=E+Fransen&amp;volume=17&amp;publication_year=2008&amp;pages=159-169&amp;pmid=17921507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Bai U, Seidman MD, Hinojosa R, Quirk WS. Mitochondrial DNA deletions associated with aging and possibly presbycusis: a human archival temporal bone study. <span><span class="ref-journal">Am J Otol. </span>1997;<span class="ref-vol">18</span>:449–453.</span> [<a href="/pubmed/9233484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Otol&amp;title=Mitochondrial+DNA+deletions+associated+with+aging+and+possibly+presbycusis:+a+human+archival+temporal+bone+study&amp;author=U+Bai&amp;author=MD+Seidman&amp;author=R+Hinojosa&amp;author=WS+Quirk&amp;volume=18&amp;publication_year=1997&amp;pages=449-453&amp;pmid=9233484&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Fischel-Ghodsian N, Bykhovskaya Y, Taylor K, et al.  Temporal bone analysis of patients with presbycusis reveals high frequency of mitochondrial mutations. <span><span class="ref-journal">Hear Res. </span>1997;<span class="ref-vol">110</span>:147–154.</span> [<a href="/pubmed/9282897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hear+Res&amp;title=Temporal+bone+analysis+of+patients+with+presbycusis+reveals+high+frequency+of+mitochondrial+mutations&amp;author=N+Fischel-Ghodsian&amp;author=Y+Bykhovskaya&amp;author=K+Taylor&amp;volume=110&amp;publication_year=1997&amp;pages=147-154&amp;pmid=9282897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Cavalli-Sforza LL, Piazza A. Human genomic diversity in Europe: a summary of recent research and prospects for the future. <span><span class="ref-journal">Eur J Hum Genet. </span>1993;<span class="ref-vol">1</span>:3–18.</span> [<a href="/pubmed/7520820" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Human+genomic+diversity+in+Europe:+a+summary+of+recent+research+and+prospects+for+the+future&amp;author=LL+Cavalli-Sforza&amp;author=A+Piazza&amp;volume=1&amp;publication_year=1993&amp;pages=3-18&amp;pmid=7520820&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Lahermo P, Sajantila A, Sistonen P, et al.  The genetic relationship between the Finns and the Finnish Saami (Lapps): analysis of nuclear DNA and mtDNA. <span><span class="ref-journal">Am J Hum Genet. </span>1996;<span class="ref-vol">58</span>:1309–1322.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1915079/">PMC free article</a>]</span> [<a href="/pubmed/8651309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=The+genetic+relationship+between+the+Finns+and+the+Finnish+Saami+(Lapps):+analysis+of+nuclear+DNA+and+mtDNA&amp;author=P+Lahermo&amp;author=A+Sajantila&amp;author=P+Sistonen&amp;volume=58&amp;publication_year=1996&amp;pages=1309-1322&amp;pmid=8651309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib16"><span class="mixed-citation">Lahermo P, Savontaus ML, Sistonen P, et al.  Y chromosomal polymorphisms reveal founding lineages in the Finns and the Saami. <span><span class="ref-journal">Eur J Hum Genet. </span>1999;<span class="ref-vol">7</span>:447–458.</span> [<a href="/pubmed/10352935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Y+chromosomal+polymorphisms+reveal+founding+lineages+in+the+Finns+and+the+Saami&amp;author=P+Lahermo&amp;author=ML+Savontaus&amp;author=P+Sistonen&amp;volume=7&amp;publication_year=1999&amp;pages=447-458&amp;pmid=10352935&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Sajantila A, Lahermo P, Anttinen T, et al.  Genes and languages in Europe: an analysis of mitochondrial lineages. <span><span class="ref-journal">Genome Res. </span>1995;<span class="ref-vol">5</span>:42–52.</span> [<a href="/pubmed/8717054" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Genes+and+languages+in+Europe:+an+analysis+of+mitochondrial+lineages&amp;author=A+Sajantila&amp;author=P+Lahermo&amp;author=T+Anttinen&amp;volume=5&amp;publication_year=1995&amp;pages=42-52&amp;pmid=8717054&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">Tambets K, Rootsi S, Kivisild T, et al.  The western and eastern roots of the Saami – the story of genetic ‘outliers' told by mitochondrial DNA and Y chromosomes. <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">74</span>:661–682.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181943/">PMC free article</a>]</span> [<a href="/pubmed/15024688" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=The+western+and+eastern+roots+of+the+Saami+–+the+story+of+genetic+‘outliers'+told+by+mitochondrial+DNA+and+Y+chromosomes&amp;author=K+Tambets&amp;author=S+Rootsi&amp;author=T+Kivisild&amp;volume=74&amp;publication_year=2004&amp;pages=661-682&amp;pmid=15024688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Ingman M, Gyllensten U. A recent genetic link between Saami and the Volga–Ural region of Russia. <span><span class="ref-journal">Eur J Hum Genet. </span>2007;<span class="ref-vol">15</span>:115–120.</span> [<a href="/pubmed/16985502" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=A+recent+genetic+link+between+Saami+and+the+Volga–Ural+region+of+Russia&amp;author=M+Ingman&amp;author=U+Gyllensten&amp;volume=15&amp;publication_year=2007&amp;pages=115-120&amp;pmid=16985502&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Johansson A, Vavruch-Nilsson V, Cox DR, Frazer KA, Gyllensten U. Evaluation of the SNP tagging approach in an independent population sample – array-based SNP discovery in Saami. <span><span class="ref-journal">Hum Genet. </span>2007;<span class="ref-vol">122</span>:141–150.</span> [<a href="/pubmed/17554563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Evaluation+of+the+SNP+tagging+approach+in+an+independent+population+sample+–+array-based+SNP+discovery+in+Saami&amp;author=A+Johansson&amp;author=V+Vavruch-Nilsson&amp;author=DR+Cox&amp;author=KA+Frazer&amp;author=U+Gyllensten&amp;volume=122&amp;publication_year=2007&amp;pages=141-150&amp;pmid=17554563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib21"><span class="mixed-citation">Johansson A, Vavruch-Nilsson V, Edin-Liljegren A, Sjolander P, Gyllensten U. Linkage disequilibrium between microsatellite markers in the Swedish Saami relative to a worldwide selection of populations. <span><span class="ref-journal">Hum Genet. </span>2005;<span class="ref-vol">116</span>:105–113.</span> [<a href="/pubmed/15549393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Linkage+disequilibrium+between+microsatellite+markers+in+the+Swedish+Saami+relative+to+a+worldwide+selection+of+populations&amp;author=A+Johansson&amp;author=V+Vavruch-Nilsson&amp;author=A+Edin-Liljegren&amp;author=P+Sjolander&amp;author=U+Gyllensten&amp;volume=116&amp;publication_year=2005&amp;pages=105-113&amp;pmid=15549393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib22"><span class="mixed-citation">Kaessmann H, Zollner S, Gustafsson AC, et al.  Extensive linkage disequilibrium in small human populations in Eurasia. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>:673–685.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384945/">PMC free article</a>]</span> [<a href="/pubmed/11813132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Extensive+linkage+disequilibrium+in+small+human+populations+in+Eurasia&amp;author=H+Kaessmann&amp;author=S+Zollner&amp;author=AC+Gustafsson&amp;volume=70&amp;publication_year=2002&amp;pages=673-685&amp;pmid=11813132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib23"><span class="mixed-citation">Laan M, Paabo S. Demographic history and linkage disequilibrium in human populations. <span><span class="ref-journal">Nat Genet. </span>1997;<span class="ref-vol">17</span>:435–438.</span> [<a href="/pubmed/9398845" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Demographic+history+and+linkage+disequilibrium+in+human+populations&amp;author=M+Laan&amp;author=S+Paabo&amp;volume=17&amp;publication_year=1997&amp;pages=435-438&amp;pmid=9398845&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib24"><span class="mixed-citation">Bonnen PE, Pe'er I, Plenge RM, et al.  Evaluating potential for whole-genome studies in Kosrae, an isolated population in Micronesia. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:214–217.</span> [<a href="/pubmed/16429162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Evaluating+potential+for+whole-genome+studies+in+Kosrae,+an+isolated+population+in+Micronesia&amp;author=PE+Bonnen&amp;author=I+Pe'er&amp;author=RM+Plenge&amp;volume=38&amp;publication_year=2006&amp;pages=214-217&amp;pmid=16429162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib25"><span class="mixed-citation">Service S, DeYoung J, Karayiorgou M, et al.  Magnitude and distribution of linkage disequilibrium in population isolates and implications for genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:556–560.</span> [<a href="/pubmed/16582909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Magnitude+and+distribution+of+linkage+disequilibrium+in+population+isolates+and+implications+for+genome-wide+association+studies&amp;author=S+Service&amp;author=J+DeYoung&amp;author=M+Karayiorgou&amp;volume=38&amp;publication_year=2006&amp;pages=556-560&amp;pmid=16582909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib26"><span class="mixed-citation">Slatkin M. Linkage disequilibrium in growing and stable populations. <span><span class="ref-journal">Genetics. </span>1994;<span class="ref-vol">137</span>:331–336.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1205949/">PMC free article</a>]</span> [<a href="/pubmed/8056320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Linkage+disequilibrium+in+growing+and+stable+populations&amp;author=M+Slatkin&amp;volume=137&amp;publication_year=1994&amp;pages=331-336&amp;pmid=8056320&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib27"><span class="mixed-citation">Terwilliger JD, Zollner S, Laan M, Paabo S. Mapping genes through the use of linkage disequilibrium generated by genetic drift: ‘drift mapping' in small populations with no demographic expansion. <span><span class="ref-journal">Hum Hered. </span>1998;<span class="ref-vol">48</span>:138–154.</span> [<a href="/pubmed/9618061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Mapping+genes+through+the+use+of+linkage+disequilibrium+generated+by+genetic+drift:+‘drift+mapping'+in+small+populations+with+no+demographic+expansion&amp;author=JD+Terwilliger&amp;author=S+Zollner&amp;author=M+Laan&amp;author=S+Paabo&amp;volume=48&amp;publication_year=1998&amp;pages=138-154&amp;pmid=9618061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib28"><span class="mixed-citation">Van Eyken E, Van Laer L, Fransen E, et al.  KCNQ4: a gene for age-related hearing impairment. <span><span class="ref-journal">Hum Mutat. </span>2006;<span class="ref-vol">27</span>:1007–1016.</span> [<a href="/pubmed/16917933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=KCNQ4:+a+gene+for+age-related+hearing+impairment&amp;author=E+Van+Eyken&amp;author=L+Van+Laer&amp;author=E+Fransen&amp;volume=27&amp;publication_year=2006&amp;pages=1007-1016&amp;pmid=16917933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib29"><span class="mixed-citation">Johnson RA, Wichern DW.  <span class="ref-journal">Applied Multivariate Statistical Analysis.</span> New Jersey: Prentice Hall; 2002.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Applied+Multivariate+Statistical+Analysis&amp;author=RA+Johnson&amp;author=DW+Wichern&amp;publication_year=2002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib30"><span class="mixed-citation">Nelson EG, Hinojosa R. Presbycusis: a human temporal bone study of individuals with downward sloping audiometric patterns of hearing loss and review of the literature. <span><span class="ref-journal">Laryngoscope. </span>2006;<span class="ref-vol">116</span>:1–12.</span> [<a href="/pubmed/16946668" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Laryngoscope&amp;title=Presbycusis:+a+human+temporal+bone+study+of+individuals+with+downward+sloping+audiometric+patterns+of+hearing+loss+and+review+of+the+literature&amp;author=EG+Nelson&amp;author=R+Hinojosa&amp;volume=116&amp;publication_year=2006&amp;pages=1-12&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib31"><span class="mixed-citation">Schuknecht HF, Gacek MR. Cochlear pathology in presbycusis. <span><span class="ref-journal">Ann Otol Rhinol Laryngol. </span>1993;<span class="ref-vol">102</span>:1–16.</span> [<a href="/pubmed/8420477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Otol+Rhinol+Laryngol&amp;title=Cochlear+pathology+in+presbycusis&amp;author=HF+Schuknecht&amp;author=MR+Gacek&amp;volume=102&amp;publication_year=1993&amp;pages=1-16&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib32"><span class="mixed-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib33"><span class="mixed-citation">Yu J, Pressoir G, Briggs WH, et al.  A unified mixed-model method for association mapping that accounts for multiple levels of relatedness. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:203–208.</span> [<a href="/pubmed/16380716" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+unified+mixed-model+method+for+association+mapping+that+accounts+for+multiple+levels+of+relatedness&amp;author=J+Yu&amp;author=G+Pressoir&amp;author=WH+Briggs&amp;volume=38&amp;publication_year=2006&amp;pages=203-208&amp;pmid=16380716&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib34"><span class="mixed-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib35"><span class="mixed-citation">Nelson EG, Hinojosa R. Presbycusis: a human temporal bone study of individuals with flat audiometric patterns of hearing loss using a new method to quantify stria vascularis volume. <span><span class="ref-journal">Laryngoscope. </span>2003;<span class="ref-vol">113</span>:1672–1686.</span> [<a href="/pubmed/14520091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Laryngoscope&amp;title=Presbycusis:+a+human+temporal+bone+study+of+individuals+with+flat+audiometric+patterns+of+hearing+loss+using+a+new+method+to+quantify+stria+vascularis+volume&amp;author=EG+Nelson&amp;author=R+Hinojosa&amp;volume=113&amp;publication_year=2003&amp;pages=1672-1686&amp;pmid=14520091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib36"><span class="mixed-citation">Robertson NG, Khetarpal U, Gutierrez-Espeleta GA, Bieber FR, Morton CC. Isolation of novel and known genes from a human fetal cochlear cDNA library using subtractive hybridization and differential screening. <span><span class="ref-journal">Genomics. </span>1994;<span class="ref-vol">23</span>:42–50.</span> [<a href="/pubmed/7829101" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Isolation+of+novel+and+known+genes+from+a+human+fetal+cochlear+cDNA+library+using+subtractive+hybridization+and+differential+screening&amp;author=NG+Robertson&amp;author=U+Khetarpal&amp;author=GA+Gutierrez-Espeleta&amp;author=FR+Bieber&amp;author=CC+Morton&amp;volume=23&amp;publication_year=1994&amp;pages=42-50&amp;pmid=7829101&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib37"><span class="mixed-citation">Yamashiro S, Abe H, Mabuchi I. IQGAP2 is required for the cadherin-mediated cell-to-cell adhesion in <em>Xenopus laevis</em> embryos. <span><span class="ref-journal">Dev Biol. </span>2007;<span class="ref-vol">308</span>:485–493.</span> [<a href="/pubmed/17617398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Biol&amp;title=IQGAP2+is+required+for+the+cadherin-mediated+cell-to-cell+adhesion+in+Xenopus+laevis+embryos&amp;author=S+Yamashiro&amp;author=H+Abe&amp;author=I+Mabuchi&amp;volume=308&amp;publication_year=2007&amp;pages=485-493&amp;pmid=17617398&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib38"><span class="mixed-citation">Kazmierczak P, Sakaguchi H, Tokita J, et al.  Cadherin 23 and protocadherin 15 interact to form tip-link filaments in sensory hair cells. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:87–91.</span> [<a href="/pubmed/17805295" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Cadherin+23+and+protocadherin+15+interact+to+form+tip-link+filaments+in+sensory+hair+cells&amp;author=P+Kazmierczak&amp;author=H+Sakaguchi&amp;author=J+Tokita&amp;volume=449&amp;publication_year=2007&amp;pages=87-91&amp;pmid=17805295&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib39"><span class="mixed-citation">Bolz H, von Brederlow B, Ramirez A, et al.  Mutation of CDH23, encoding a new member of the cadherin gene family, causes Usher syndrome type 1D. <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">27</span>:108–112.</span> [<a href="/pubmed/11138009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mutation+of+CDH23,+encoding+a+new+member+of+the+cadherin+gene+family,+causes+Usher+syndrome+type+1D&amp;author=H+Bolz&amp;author=B+von+Brederlow&amp;author=A+Ramirez&amp;volume=27&amp;publication_year=2001&amp;pages=108-112&amp;pmid=11138009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib40"><span class="mixed-citation">Bork JM, Peters LM, Riazuddin S, et al.  Usher syndrome 1D and nonsyndromic autosomal recessive deafness DFNB12 are caused by allelic mutations of the novel cadherin-like gene CDH23. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">68</span>:26–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1234923/">PMC free article</a>]</span> [<a href="/pubmed/11090341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Usher+syndrome+1D+and+nonsyndromic+autosomal+recessive+deafness+DFNB12+are+caused+by+allelic+mutations+of+the+novel+cadherin-like+gene+CDH23&amp;author=JM+Bork&amp;author=LM+Peters&amp;author=S+Riazuddin&amp;volume=68&amp;publication_year=2001&amp;pages=26-37&amp;pmid=11090341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib41"><span class="mixed-citation">Noben-Trauth K, Zheng QY, Johnson KR. Association of cadherin 23 with polygenic inheritance and genetic modification of sensorineural hearing loss. <span><span class="ref-journal">Nat Genet. </span>2003;<span class="ref-vol">35</span>:21–23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2864026/">PMC free article</a>]</span> [<a href="/pubmed/12910270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+of+cadherin+23+with+polygenic+inheritance+and+genetic+modification+of+sensorineural+hearing+loss&amp;author=K+Noben-Trauth&amp;author=QY+Zheng&amp;author=KR+Johnson&amp;volume=35&amp;publication_year=2003&amp;pages=21-23&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib42"><span class="mixed-citation">Van Eyken E, Van Camp G, Fransen E, et al.  The <em>N</em>-acetyltransferase 2 polymorphism NAT2*6A is a causative factor for age related hearing impairment. <span><span class="ref-journal">J Med Genet. </span>2007;<span class="ref-vol">44</span>:570–578.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2597944/">PMC free article</a>]</span> [<a href="/pubmed/17513527" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=The+N-acetyltransferase+2+polymorphism+NAT2*6A+is+a+causative+factor+for+age+related+hearing+impairment&amp;author=E+Van+Eyken&amp;author=G+Van+Camp&amp;author=E+Fransen&amp;volume=44&amp;publication_year=2007&amp;pages=570-578&amp;pmid=17513527&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib43"><span class="mixed-citation">Huyghe JR, Fransen E, Hannula S, et al.  <span class="ref-title">Genome-wide SNP analysis reveals no gain in power for association studies of common variants in the Finnish Saami </span><span class="ref-journal">Eur J Hum Genet</span> 2009. e-pub ahead of print]. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2987317/">PMC free article</a>]</span> [<a href="/pubmed/19935831" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></li><li id="bib44"><span class="mixed-citation">Purcell S, Cherny SS, Sham PC. Genetic Power Calculator: design of linkage and association genetic mapping studies of complex traits. <span><span class="ref-journal">Bioinformatics. </span>2003;<span class="ref-vol">19</span>:149–150.</span> [<a href="/pubmed/12499305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genetic+Power+Calculator:+design+of+linkage+and+association+genetic+mapping+studies+of+complex+traits&amp;author=S+Purcell&amp;author=SS+Cherny&amp;author=PC+Sham&amp;volume=19&amp;publication_year=2003&amp;pages=149-150&amp;pmid=12499305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib45"><span class="mixed-citation">Weedon MN, Lango H, Lindgren CM, et al.  Genome-wide association analysis identifies 20 loci that influence adult height. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:575–583.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681221/">PMC free article</a>]</span> [<a href="/pubmed/18391952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analysis+identifies+20+loci+that+influence+adult+height&amp;author=MN+Weedon&amp;author=H+Lango&amp;author=CM+Lindgren&amp;volume=40&amp;publication_year=2008&amp;pages=575-583&amp;pmid=18391952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib46"><span class="mixed-citation">Klein RJ, Zeiss C, Chew EY, et al.  Complement factor H polymorphism in age-related macular degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:385–389.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1512523/">PMC free article</a>]</span> [<a href="/pubmed/15761122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+polymorphism+in+age-related+macular+degeneration&amp;author=RJ+Klein&amp;author=C+Zeiss&amp;author=EY+Chew&amp;volume=308&amp;publication_year=2005&amp;pages=385-389&amp;pmid=15761122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib47"><span class="mixed-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib48"><span class="mixed-citation">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2987344&amp;issue-id=191823&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140398730714336title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec7title">Results</a></li><li><a href="#__sec8title">Discussion</a></li><li><a href="#idm140398731039616title">Acknowledgments</a></li><li><a href="#idm140398770030352title">Notes</a></li><li><a href="#idm140398770031136title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm140398770029728title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2987344/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2987344/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2987344/pdf/ejhg2009234a.pdf">PDF (398K)</a></li> | <li><a href="#" data-citationid="PMC2987344" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2987344%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2987344%2F&amp;text=A%20genome-wide%20association%20study%20for%20age-related%20hearing%20impairment%20in%20the%20Saami" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2987344%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20068591" data-aiid="2987344" data-aid="2987344" data-iid="191823" data-domainid="1300" data-domain="ejhg" data-accid="PMC2987344" data-md5="c590810cccc327d23df09a0d1228da81"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:38:32-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE882895DD5045810000000003460232&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2987344%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2987344/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>