<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2664150/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A two-stage genome-wide association study of sporadic amyotrophic lateral sclerosis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2664150/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="A two-stage genome-wide association study of sporadic amyotrophic lateral sclerosis"><meta name="citation_authors" content="Adriano Chiò, Jennifer C. Schymick, Gabriella Restagno, Sonja W. Scholz, Federica Lombardo, Shiao-Lin Lai, Gabriele Mora, Hon-Chung Fung, Angela Britton, Sampath Arepalli, J. Raphael Gibbs, Michael Nalls, Stephen Berger, Lydia Coulter Kwee, Eugene Z. Oddone, Jinhui Ding, Cynthia Crews, Ian Rafferty, Nicole Washecka, Dena Hernandez, Luigi Ferrucci, Stefania Bandinelli, Jack Guralnik, Fabio Macciardi, Federica Torri, Sara Lupoli, Stephen J. Chanock, Gilles Thomas, David J. Hunter, Christian Gieger, H. Erich Wichmann, Andrea Calvo, Roberto Mutani, Stefania Battistini, Fabio Giannini, Claudia Caponnetto, Giovanni Luigi Mancardi, Vincenzo La Bella, Francesca Valentino, Maria Rosaria Monsurrò, Gioacchino Tedeschi, Kalliopi Marinou, Mario Sabatelli, Amelia Conte, Jessica Mandrioli, Patrizia Sola, Fabrizio Salvi, Ilaria Bartolomei, Gabriele Siciliano, Cecilia Carlesi, Richard W. Orrell, Kevin Talbot, Zachary Simmons, James Connor, Erik P. Pioro, Travis Dunkley, Dietrich A. Stephan, Dalia Kasperaviciute, Elizabeth M. Fisher, Sibylle Jabonka, Michael Sendtner, Marcus Beck, Lucie Bruijn, Jeffrey Rothstein, Silke Schmidt, Andrew Singleton, John Hardy, Bryan J. Traynor"><meta name="citation_date" content="15 April 2009"><meta name="citation_issue" content="8"><meta name="citation_volume" content="18"><meta name="citation_firstpage" content="1524"><meta name="citation_doi" content="10.1093/hmg/ddp059"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2664150/?report=abstract"><meta name="citation_pmid" content="19193627"><meta name="DC.Title" content="A two-stage genome-wide association study of sporadic amyotrophic lateral sclerosis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Adriano Chiò"><meta name="DC.Contributor" content="Jennifer C. Schymick"><meta name="DC.Contributor" content="Gabriella Restagno"><meta name="DC.Contributor" content="Sonja W. Scholz"><meta name="DC.Contributor" content="Federica Lombardo"><meta name="DC.Contributor" content="Shiao-Lin Lai"><meta name="DC.Contributor" content="Gabriele Mora"><meta name="DC.Contributor" content="Hon-Chung Fung"><meta name="DC.Contributor" content="Angela Britton"><meta name="DC.Contributor" content="Sampath Arepalli"><meta name="DC.Contributor" content="J. Raphael Gibbs"><meta name="DC.Contributor" content="Michael Nalls"><meta name="DC.Contributor" content="Stephen Berger"><meta name="DC.Contributor" content="Lydia Coulter Kwee"><meta name="DC.Contributor" content="Eugene Z. Oddone"><meta name="DC.Contributor" content="Jinhui Ding"><meta name="DC.Contributor" content="Cynthia Crews"><meta name="DC.Contributor" content="Ian Rafferty"><meta name="DC.Contributor" content="Nicole Washecka"><meta name="DC.Contributor" content="Dena Hernandez"><meta name="DC.Contributor" content="Luigi Ferrucci"><meta name="DC.Contributor" content="Stefania Bandinelli"><meta name="DC.Contributor" content="Jack Guralnik"><meta name="DC.Contributor" content="Fabio Macciardi"><meta name="DC.Contributor" content="Federica Torri"><meta name="DC.Contributor" content="Sara Lupoli"><meta name="DC.Contributor" content="Stephen J. Chanock"><meta name="DC.Contributor" content="Gilles Thomas"><meta name="DC.Contributor" content="David J. Hunter"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="H. Erich Wichmann"><meta name="DC.Contributor" content="Andrea Calvo"><meta name="DC.Contributor" content="Roberto Mutani"><meta name="DC.Contributor" content="Stefania Battistini"><meta name="DC.Contributor" content="Fabio Giannini"><meta name="DC.Contributor" content="Claudia Caponnetto"><meta name="DC.Contributor" content="Giovanni Luigi Mancardi"><meta name="DC.Contributor" content="Vincenzo La Bella"><meta name="DC.Contributor" content="Francesca Valentino"><meta name="DC.Contributor" content="Maria Rosaria Monsurrò"><meta name="DC.Contributor" content="Gioacchino Tedeschi"><meta name="DC.Contributor" content="Kalliopi Marinou"><meta name="DC.Contributor" content="Mario Sabatelli"><meta name="DC.Contributor" content="Amelia Conte"><meta name="DC.Contributor" content="Jessica Mandrioli"><meta name="DC.Contributor" content="Patrizia Sola"><meta name="DC.Contributor" content="Fabrizio Salvi"><meta name="DC.Contributor" content="Ilaria Bartolomei"><meta name="DC.Contributor" content="Gabriele Siciliano"><meta name="DC.Contributor" content="Cecilia Carlesi"><meta name="DC.Contributor" content="Richard W. Orrell"><meta name="DC.Contributor" content="Kevin Talbot"><meta name="DC.Contributor" content="Zachary Simmons"><meta name="DC.Contributor" content="James Connor"><meta name="DC.Contributor" content="Erik P. Pioro"><meta name="DC.Contributor" content="Travis Dunkley"><meta name="DC.Contributor" content="Dietrich A. Stephan"><meta name="DC.Contributor" content="Dalia Kasperaviciute"><meta name="DC.Contributor" content="Elizabeth M. Fisher"><meta name="DC.Contributor" content="Sibylle Jabonka"><meta name="DC.Contributor" content="Michael Sendtner"><meta name="DC.Contributor" content="Marcus Beck"><meta name="DC.Contributor" content="Lucie Bruijn"><meta name="DC.Contributor" content="Jeffrey Rothstein"><meta name="DC.Contributor" content="Silke Schmidt"><meta name="DC.Contributor" content="Andrew Singleton"><meta name="DC.Contributor" content="John Hardy"><meta name="DC.Contributor" content="Bryan J. Traynor"><meta name="DC.Date" content="2009 Apr 15"><meta name="DC.Identifier" content="10.1093/hmg/ddp059"><meta name="DC.Language" content="en"><meta property="og:title" content="A two-stage genome-wide association study of sporadic amyotrophic lateral sclerosis"><meta property="og:type" content="article"><meta property="og:description" content="The cause of sporadic amyotrophic lateral sclerosis (ALS) is largely unknown, but genetic factors are thought to play a significant role in determining susceptibility to motor neuron degeneration. To identify genetic variants altering risk of ALS, we ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2664150/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2664150/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2664150/pdf/ddp059.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89B42ADD40EF810000000005AC032D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2664150%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="nuccore">Nucleotide</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC2664150</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2009 Apr 15; 18(8): 1524–1532. </span></div><div><span class="fm-vol-iss-date">Published online 2009 Feb 4. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddp059" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddp059</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2664150</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19193627">19193627</a></div></div></div></div><h1 class="content-title">A two-stage genome-wide association study of sporadic amyotrophic lateral sclerosis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Chi%26%23x000f2%3B%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949321568" co-class="co-affbox">Adriano Chiò</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Schymick%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187982385536" co-class="co-affbox">Jennifer C. Schymick</a>,<sup>2,</sup><sup>3,</sup><sup>†</sup> <a href="/pubmed/?term=Restagno%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187948338320" co-class="co-affbox">Gabriella Restagno</a>,<sup>4,</sup><sup>†</sup> <a href="/pubmed/?term=Scholz%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941668080" co-class="co-affbox">Sonja W. Scholz</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Lombardo%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941404720" co-class="co-affbox">Federica Lombardo</a>,<sup>4</sup> <a href="/pubmed/?term=Lai%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188015413312" co-class="co-affbox">Shiao-Lin Lai</a>,<sup>5,</sup><sup>7</sup> <a href="/pubmed/?term=Mora%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188017890400" co-class="co-affbox">Gabriele Mora</a>,<sup>8</sup> <a href="/pubmed/?term=Fung%20HC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188017888176" co-class="co-affbox">Hon-Chung Fung</a>,<sup>2,</sup><sup>7</sup> <a href="/pubmed/?term=Britton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187982331312" co-class="co-affbox">Angela Britton</a>,<sup>5</sup> <a href="/pubmed/?term=Arepalli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941506112" co-class="co-affbox">Sampath Arepalli</a>,<sup>5</sup> <a href="/pubmed/?term=Gibbs%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941503888" co-class="co-affbox">J. Raphael Gibbs</a>,<sup>6,</sup><sup>9</sup> <a href="/pubmed/?term=Nalls%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187934992912" co-class="co-affbox">Michael Nalls</a>,<sup>5</sup> <a href="/pubmed/?term=Berger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187934990688" co-class="co-affbox">Stephen Berger</a>,<sup>2</sup> <a href="/pubmed/?term=Kwee%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187946717616" co-class="co-affbox">Lydia Coulter Kwee</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Oddone%20EZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187946714656" co-class="co-affbox">Eugene Z. Oddone</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Ding%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949344032" co-class="co-affbox">Jinhui Ding</a>,<sup>9</sup> <a href="/pubmed/?term=Crews%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941512432" co-class="co-affbox">Cynthia Crews</a>,<sup>2</sup> <a href="/pubmed/?term=Rafferty%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941510208" co-class="co-affbox">Ian Rafferty</a>,<sup>2</sup> <a href="/pubmed/?term=Washecka%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187948488256" co-class="co-affbox">Nicole Washecka</a>,<sup>2</sup> <a href="/pubmed/?term=Hernandez%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187948486032" co-class="co-affbox">Dena Hernandez</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187946691952" co-class="co-affbox">Luigi Ferrucci</a>,<sup>13</sup> <a href="/pubmed/?term=Bandinelli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187946689728" co-class="co-affbox">Stefania Bandinelli</a>,<sup>14</sup> <a href="/pubmed/?term=Guralnik%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188018569776" co-class="co-affbox">Jack Guralnik</a>,<sup>15</sup> <a href="/pubmed/?term=Macciardi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188018567552" co-class="co-affbox">Fabio Macciardi</a>,<sup>16</sup> <a href="/pubmed/?term=Torri%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188017865184" co-class="co-affbox">Federica Torri</a>,<sup>16</sup> <a href="/pubmed/?term=Lupoli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188017862960" co-class="co-affbox">Sara Lupoli</a>,<sup>17</sup> <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941720416" co-class="co-affbox">Stephen J. Chanock</a>,<sup>18</sup> <a href="/pubmed/?term=Thomas%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941718192" co-class="co-affbox">Gilles Thomas</a>,<sup>18</sup> <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949596528" co-class="co-affbox">David J. Hunter</a>,<sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949593568" co-class="co-affbox">Christian Gieger</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187948154944" co-class="co-affbox">H. Erich Wichmann</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Calvo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949589472" co-class="co-affbox">Andrea Calvo</a>,<sup>1</sup> <a href="/pubmed/?term=Mutani%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949587248" co-class="co-affbox">Roberto Mutani</a>,<sup>1</sup> <a href="/pubmed/?term=Battistini%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187944015312" co-class="co-affbox">Stefania Battistini</a>,<sup>22</sup> <a href="/pubmed/?term=Giannini%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187944013088" co-class="co-affbox">Fabio Giannini</a>,<sup>22</sup> <a href="/pubmed/?term=Caponnetto%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187945619072" co-class="co-affbox">Claudia Caponnetto</a>,<sup>23</sup> <a href="/pubmed/?term=Mancardi%20GL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187945616848" co-class="co-affbox">Giovanni Luigi Mancardi</a>,<sup>23</sup> <a href="/pubmed/?term=La%20Bella%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187945614624" co-class="co-affbox">Vincenzo La Bella</a>,<sup>24</sup> <a href="/pubmed/?term=Valentino%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188018178672" co-class="co-affbox">Francesca Valentino</a>,<sup>24</sup> <a href="/pubmed/?term=Monsurr%26%23x000f2%3B%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188018176448" co-class="co-affbox">Maria Rosaria Monsurrò</a>,<sup>25</sup> <a href="/pubmed/?term=Tedeschi%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188015188480" co-class="co-affbox">Gioacchino Tedeschi</a>,<sup>25</sup> <a href="/pubmed/?term=Marinou%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188015186256" co-class="co-affbox">Kalliopi Marinou</a>,<sup>8</sup> <a href="/pubmed/?term=Sabatelli%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187946229392" co-class="co-affbox">Mario Sabatelli</a>,<sup>26</sup> <a href="/pubmed/?term=Conte%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187946227168" co-class="co-affbox">Amelia Conte</a>,<sup>26</sup> <a href="/pubmed/?term=Mandrioli%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949630064" co-class="co-affbox">Jessica Mandrioli</a>,<sup>27</sup> <a href="/pubmed/?term=Sola%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949627840" co-class="co-affbox">Patrizia Sola</a>,<sup>27</sup> <a href="/pubmed/?term=Salvi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187948236016" co-class="co-affbox">Fabrizio Salvi</a>,<sup>28</sup> <a href="/pubmed/?term=Bartolomei%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187948233792" co-class="co-affbox">Ilaria Bartolomei</a>,<sup>28</sup> <a href="/pubmed/?term=Siciliano%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187940481104" co-class="co-affbox">Gabriele Siciliano</a>,<sup>29</sup> <a href="/pubmed/?term=Carlesi%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187940478880" co-class="co-affbox">Cecilia Carlesi</a>,<sup>29</sup> <a href="/pubmed/?term=Orrell%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941516016" co-class="co-affbox">Richard W. Orrell</a>,<sup>30</sup> <a href="/pubmed/?term=Talbot%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941513792" co-class="co-affbox">Kevin Talbot</a>,<sup>3</sup> <a href="/pubmed/?term=Simmons%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188019606880" co-class="co-affbox">Zachary Simmons</a>,<sup>31</sup> <a href="/pubmed/?term=Connor%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188019604656" co-class="co-affbox">James Connor</a>,<sup>32</sup> <a href="/pubmed/?term=Pioro%20EP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187948007968" co-class="co-affbox">Erik P. Pioro</a>,<sup>33</sup> <a href="/pubmed/?term=Dunkley%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187948005744" co-class="co-affbox">Travis Dunkley</a>,<sup>34</sup> <a href="/pubmed/?term=Stephan%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941613808" co-class="co-affbox">Dietrich A. Stephan</a>,<sup>34</sup> <a href="/pubmed/?term=Kasperaviciute%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941611584" co-class="co-affbox">Dalia Kasperaviciute</a>,<sup>35</sup> <a href="/pubmed/?term=Fisher%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188017921664" co-class="co-affbox">Elizabeth M. Fisher</a>,<sup>35</sup> <a href="/pubmed/?term=Jabonka%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140188017919440" co-class="co-affbox">Sibylle Jabonka</a>,<sup>36</sup> <a href="/pubmed/?term=Sendtner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941788016" co-class="co-affbox">Michael Sendtner</a>,<sup>36</sup> <a href="/pubmed/?term=Beck%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187941785792" co-class="co-affbox">Marcus Beck</a>,<sup>36</sup> <a href="/pubmed/?term=Bruijn%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187942947728" co-class="co-affbox">Lucie Bruijn</a>,<sup>37</sup> <a href="/pubmed/?term=Rothstein%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187942945504" co-class="co-affbox">Jeffrey Rothstein</a>,<sup>38</sup> <a href="/pubmed/?term=Schmidt%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187942943280" co-class="co-affbox">Silke Schmidt</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Singleton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949732176" co-class="co-affbox">Andrew Singleton</a>,<sup>5</sup> <a href="/pubmed/?term=Hardy%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949410192" co-class="co-affbox">John Hardy</a>,<sup>2,</sup><sup>6</sup> and  <a href="/pubmed/?term=Traynor%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627" class="affpopup" co-rid="_co_idm140187949407232" co-class="co-affbox">Bryan J. Traynor</a><sup>2,</sup><sup>38,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140187949321568"><h3 class="no_margin">Adriano Chiò</h3><p><sup>1</sup>Department of Neuroscience, University of Turin, Turin, Italy</p><div>Find articles by <a href="/pubmed/?term=Chi%26%23x000f2%3B%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Adriano Chiò</a></div></div><div id="_co_idm140187982385536"><h3 class="no_margin">Jennifer C. Schymick</h3><p><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><p><sup>3</sup>Department of Physiology, Anatomy and Genetics, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Schymick%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Jennifer C. Schymick</a></div></div><div id="_co_idm140187948338320"><h3 class="no_margin">Gabriella Restagno</h3><p><sup>4</sup>Molecular Genetics Unit, Department of Clinical Pathology, A.S.O. O.I.R.M.-S.Anna, Turin, Italy</p><div>Find articles by <a href="/pubmed/?term=Restagno%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Gabriella Restagno</a></div></div><div id="_co_idm140187941668080"><h3 class="no_margin">Sonja W. Scholz</h3><p><sup>5</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><p><sup>6</sup>Department of Molecular Neuroscience and Reta Lila Weston Institute of Neurological Studies, Institute of Neurology, Queen Square, London, UK</p><div>Find articles by <a href="/pubmed/?term=Scholz%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Sonja W. Scholz</a></div></div><div id="_co_idm140187941404720"><h3 class="no_margin">Federica Lombardo</h3><p><sup>4</sup>Molecular Genetics Unit, Department of Clinical Pathology, A.S.O. O.I.R.M.-S.Anna, Turin, Italy</p><div>Find articles by <a href="/pubmed/?term=Lombardo%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Federica Lombardo</a></div></div><div id="_co_idm140188015413312"><h3 class="no_margin">Shiao-Lin Lai</h3><p><sup>5</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><p><sup>7</sup>Department of Neurology, Chang Gung Memorial Hospital and College of Medicine, Taiwan, Republic of China</p><div>Find articles by <a href="/pubmed/?term=Lai%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Shiao-Lin Lai</a></div></div><div id="_co_idm140188017890400"><h3 class="no_margin">Gabriele Mora</h3><p><sup>8</sup>Salvatore Maugeri Foundation, Lissone, Italy</p><div>Find articles by <a href="/pubmed/?term=Mora%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Gabriele Mora</a></div></div><div id="_co_idm140188017888176"><h3 class="no_margin">Hon-Chung Fung</h3><p><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><p><sup>7</sup>Department of Neurology, Chang Gung Memorial Hospital and College of Medicine, Taiwan, Republic of China</p><div>Find articles by <a href="/pubmed/?term=Fung%20HC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Hon-Chung Fung</a></div></div><div id="_co_idm140187982331312"><h3 class="no_margin">Angela Britton</h3><p><sup>5</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Britton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Angela Britton</a></div></div><div id="_co_idm140187941506112"><h3 class="no_margin">Sampath Arepalli</h3><p><sup>5</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Arepalli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Sampath Arepalli</a></div></div><div id="_co_idm140187941503888"><h3 class="no_margin">J. Raphael Gibbs</h3><p><sup>6</sup>Department of Molecular Neuroscience and Reta Lila Weston Institute of Neurological Studies, Institute of Neurology, Queen Square, London, UK</p><p><sup>9</sup>Computational Biology Core, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Gibbs%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">J. Raphael Gibbs</a></div></div><div id="_co_idm140187934992912"><h3 class="no_margin">Michael Nalls</h3><p><sup>5</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Nalls%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Michael Nalls</a></div></div><div id="_co_idm140187934990688"><h3 class="no_margin">Stephen Berger</h3><p><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Berger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Stephen Berger</a></div></div><div id="_co_idm140187946717616"><h3 class="no_margin">Lydia Coulter Kwee</h3><p><sup>10</sup>Center for Human Genetics, Duke University Medical Center, Durham, NC, USA</p><p><sup>11</sup>Department of Medicine, Duke University Medical Center, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Kwee%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Lydia Coulter Kwee</a></div></div><div id="_co_idm140187946714656"><h3 class="no_margin">Eugene Z. Oddone</h3><p><sup>11</sup>Department of Medicine, Duke University Medical Center, Durham, NC, USA</p><p><sup>12</sup>Epidemiology Research and Information Center, Durham VAMC, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Oddone%20EZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Eugene Z. Oddone</a></div></div><div id="_co_idm140187949344032"><h3 class="no_margin">Jinhui Ding</h3><p><sup>9</sup>Computational Biology Core, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Ding%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Jinhui Ding</a></div></div><div id="_co_idm140187941512432"><h3 class="no_margin">Cynthia Crews</h3><p><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Crews%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Cynthia Crews</a></div></div><div id="_co_idm140187941510208"><h3 class="no_margin">Ian Rafferty</h3><p><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Rafferty%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Ian Rafferty</a></div></div><div id="_co_idm140187948488256"><h3 class="no_margin">Nicole Washecka</h3><p><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Washecka%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Nicole Washecka</a></div></div><div id="_co_idm140187948486032"><h3 class="no_margin">Dena Hernandez</h3><p><sup>5</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><p><sup>6</sup>Department of Molecular Neuroscience and Reta Lila Weston Institute of Neurological Studies, Institute of Neurology, Queen Square, London, UK</p><div>Find articles by <a href="/pubmed/?term=Hernandez%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Dena Hernandez</a></div></div><div id="_co_idm140187946691952"><h3 class="no_margin">Luigi Ferrucci</h3><p><sup>13</sup>Longitudinal Studies Section, Clinical Research Branch, National Institute on Aging, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Luigi Ferrucci</a></div></div><div id="_co_idm140187946689728"><h3 class="no_margin">Stefania Bandinelli</h3><p><sup>14</sup>Geriatric Unit, Azienda Sanitaria di Firenze, Florence, Italy</p><div>Find articles by <a href="/pubmed/?term=Bandinelli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Stefania Bandinelli</a></div></div><div id="_co_idm140188018569776"><h3 class="no_margin">Jack Guralnik</h3><p><sup>15</sup>Laboratory of Epidemiology, Demography and Biometry, National Institute on Aging, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Guralnik%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Jack Guralnik</a></div></div><div id="_co_idm140188018567552"><h3 class="no_margin">Fabio Macciardi</h3><p><sup>16</sup>Department of Science and Biomedical Technology, University of Milan, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Macciardi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Fabio Macciardi</a></div></div><div id="_co_idm140188017865184"><h3 class="no_margin">Federica Torri</h3><p><sup>16</sup>Department of Science and Biomedical Technology, University of Milan, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Torri%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Federica Torri</a></div></div><div id="_co_idm140188017862960"><h3 class="no_margin">Sara Lupoli</h3><p><sup>17</sup>INSPE, San Raffaele Scientific Institute, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Lupoli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Sara Lupoli</a></div></div><div id="_co_idm140187941720416"><h3 class="no_margin">Stephen J. Chanock</h3><p><sup>18</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Stephen J. Chanock</a></div></div><div id="_co_idm140187941718192"><h3 class="no_margin">Gilles Thomas</h3><p><sup>18</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Thomas%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Gilles Thomas</a></div></div><div id="_co_idm140187949596528"><h3 class="no_margin">David J. Hunter</h3><p><sup>18</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, NIH, Bethesda, MD, USA</p><p><sup>19</sup>Department of Epidemiology, Harvard School of Public Health, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">David J. Hunter</a></div></div><div id="_co_idm140187949593568"><h3 class="no_margin">Christian Gieger</h3><p><sup>20</sup>Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg/Munich, Germany</p><p><sup>21</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Christian Gieger</a></div></div><div id="_co_idm140187948154944"><h3 class="no_margin">H. Erich Wichmann</h3><p><sup>20</sup>Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg/Munich, Germany</p><p><sup>21</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">H. Erich Wichmann</a></div></div><div id="_co_idm140187949589472"><h3 class="no_margin">Andrea Calvo</h3><p><sup>1</sup>Department of Neuroscience, University of Turin, Turin, Italy</p><div>Find articles by <a href="/pubmed/?term=Calvo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Andrea Calvo</a></div></div><div id="_co_idm140187949587248"><h3 class="no_margin">Roberto Mutani</h3><p><sup>1</sup>Department of Neuroscience, University of Turin, Turin, Italy</p><div>Find articles by <a href="/pubmed/?term=Mutani%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Roberto Mutani</a></div></div><div id="_co_idm140187944015312"><h3 class="no_margin">Stefania Battistini</h3><p><sup>22</sup>Department of Neuroscience, Neurology Section, University of Siena, Siena, Italy</p><div>Find articles by <a href="/pubmed/?term=Battistini%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Stefania Battistini</a></div></div><div id="_co_idm140187944013088"><h3 class="no_margin">Fabio Giannini</h3><p><sup>22</sup>Department of Neuroscience, Neurology Section, University of Siena, Siena, Italy</p><div>Find articles by <a href="/pubmed/?term=Giannini%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Fabio Giannini</a></div></div><div id="_co_idm140187945619072"><h3 class="no_margin">Claudia Caponnetto</h3><p><sup>23</sup>Department of Neuroscience, Ophthalmology and Genetics, University of Genoa, Genoa, Italy</p><div>Find articles by <a href="/pubmed/?term=Caponnetto%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Claudia Caponnetto</a></div></div><div id="_co_idm140187945616848"><h3 class="no_margin">Giovanni Luigi Mancardi</h3><p><sup>23</sup>Department of Neuroscience, Ophthalmology and Genetics, University of Genoa, Genoa, Italy</p><div>Find articles by <a href="/pubmed/?term=Mancardi%20GL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Giovanni Luigi Mancardi</a></div></div><div id="_co_idm140187945614624"><h3 class="no_margin">Vincenzo La Bella</h3><p><sup>24</sup>Department of Clinical Neurosciences, University of Palermo, Sicily, Italy</p><div>Find articles by <a href="/pubmed/?term=La%20Bella%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Vincenzo La Bella</a></div></div><div id="_co_idm140188018178672"><h3 class="no_margin">Francesca Valentino</h3><p><sup>24</sup>Department of Clinical Neurosciences, University of Palermo, Sicily, Italy</p><div>Find articles by <a href="/pubmed/?term=Valentino%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Francesca Valentino</a></div></div><div id="_co_idm140188018176448"><h3 class="no_margin">Maria Rosaria Monsurrò</h3><p><sup>25</sup>Department of Neurological Sciences, Second University of Naples, Caserta, Italy</p><div>Find articles by <a href="/pubmed/?term=Monsurr%26%23x000f2%3B%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Maria Rosaria Monsurrò</a></div></div><div id="_co_idm140188015188480"><h3 class="no_margin">Gioacchino Tedeschi</h3><p><sup>25</sup>Department of Neurological Sciences, Second University of Naples, Caserta, Italy</p><div>Find articles by <a href="/pubmed/?term=Tedeschi%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Gioacchino Tedeschi</a></div></div><div id="_co_idm140188015186256"><h3 class="no_margin">Kalliopi Marinou</h3><p><sup>8</sup>Salvatore Maugeri Foundation, Lissone, Italy</p><div>Find articles by <a href="/pubmed/?term=Marinou%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Kalliopi Marinou</a></div></div><div id="_co_idm140187946229392"><h3 class="no_margin">Mario Sabatelli</h3><p><sup>26</sup>Neurological Institute, Catholic University and I.CO.M.M. Association for ALS Research, Rome, Italy</p><div>Find articles by <a href="/pubmed/?term=Sabatelli%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Mario Sabatelli</a></div></div><div id="_co_idm140187946227168"><h3 class="no_margin">Amelia Conte</h3><p><sup>26</sup>Neurological Institute, Catholic University and I.CO.M.M. Association for ALS Research, Rome, Italy</p><div>Find articles by <a href="/pubmed/?term=Conte%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Amelia Conte</a></div></div><div id="_co_idm140187949630064"><h3 class="no_margin">Jessica Mandrioli</h3><p><sup>27</sup>Department of Neuroscience, S. Agostino- Estense Hospital and University of Modena, Modena, Italy</p><div>Find articles by <a href="/pubmed/?term=Mandrioli%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Jessica Mandrioli</a></div></div><div id="_co_idm140187949627840"><h3 class="no_margin">Patrizia Sola</h3><p><sup>27</sup>Department of Neuroscience, S. Agostino- Estense Hospital and University of Modena, Modena, Italy</p><div>Find articles by <a href="/pubmed/?term=Sola%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Patrizia Sola</a></div></div><div id="_co_idm140187948236016"><h3 class="no_margin">Fabrizio Salvi</h3><p><sup>28</sup>Department of Neurology, Center for Diagnosis and Cure of Rare Diseases, Bellaria Hospital, Bologna, Italy</p><div>Find articles by <a href="/pubmed/?term=Salvi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Fabrizio Salvi</a></div></div><div id="_co_idm140187948233792"><h3 class="no_margin">Ilaria Bartolomei</h3><p><sup>28</sup>Department of Neurology, Center for Diagnosis and Cure of Rare Diseases, Bellaria Hospital, Bologna, Italy</p><div>Find articles by <a href="/pubmed/?term=Bartolomei%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Ilaria Bartolomei</a></div></div><div id="_co_idm140187940481104"><h3 class="no_margin">Gabriele Siciliano</h3><p><sup>29</sup>Department of Neuroscience, University of Pisa, Pisa, Italy</p><div>Find articles by <a href="/pubmed/?term=Siciliano%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Gabriele Siciliano</a></div></div><div id="_co_idm140187940478880"><h3 class="no_margin">Cecilia Carlesi</h3><p><sup>29</sup>Department of Neuroscience, University of Pisa, Pisa, Italy</p><div>Find articles by <a href="/pubmed/?term=Carlesi%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Cecilia Carlesi</a></div></div><div id="_co_idm140187941516016"><h3 class="no_margin">Richard W. Orrell</h3><p><sup>30</sup>University Department of Clinical Neurosciences, Institute of Neurology, University College London, London</p><div>Find articles by <a href="/pubmed/?term=Orrell%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Richard W. Orrell</a></div></div><div id="_co_idm140187941513792"><h3 class="no_margin">Kevin Talbot</h3><p><sup>3</sup>Department of Physiology, Anatomy and Genetics, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Talbot%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Kevin Talbot</a></div></div><div id="_co_idm140188019606880"><h3 class="no_margin">Zachary Simmons</h3><p><sup>31</sup>Department of Neurology, Penn State College of Medicine, Hershey, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Simmons%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Zachary Simmons</a></div></div><div id="_co_idm140188019604656"><h3 class="no_margin">James Connor</h3><p><sup>32</sup>Department of Neurosurgery, Penn State College of Medicine, Hershey, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Connor%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">James Connor</a></div></div><div id="_co_idm140187948007968"><h3 class="no_margin">Erik P. Pioro</h3><p><sup>33</sup>Department of Neurology, Cleveland Clinic, Cleveland, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Pioro%20EP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Erik P. Pioro</a></div></div><div id="_co_idm140187948005744"><h3 class="no_margin">Travis Dunkley</h3><p><sup>34</sup>Neurogenomics Division, Translational Genomics Institute (TGEN), Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Dunkley%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Travis Dunkley</a></div></div><div id="_co_idm140187941613808"><h3 class="no_margin">Dietrich A. Stephan</h3><p><sup>34</sup>Neurogenomics Division, Translational Genomics Institute (TGEN), Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Stephan%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Dietrich A. Stephan</a></div></div><div id="_co_idm140187941611584"><h3 class="no_margin">Dalia Kasperaviciute</h3><p><sup>35</sup>Department of Neurodegenerative Disease, UCL Institute of Neurology, Queen Square, London, UK</p><div>Find articles by <a href="/pubmed/?term=Kasperaviciute%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Dalia Kasperaviciute</a></div></div><div id="_co_idm140188017921664"><h3 class="no_margin">Elizabeth M. Fisher</h3><p><sup>35</sup>Department of Neurodegenerative Disease, UCL Institute of Neurology, Queen Square, London, UK</p><div>Find articles by <a href="/pubmed/?term=Fisher%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Elizabeth M. Fisher</a></div></div><div id="_co_idm140188017919440"><h3 class="no_margin">Sibylle Jabonka</h3><p><sup>36</sup>Institute of Clinical Neurobiology, University of Wuerzburg, Wuerzburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Jabonka%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Sibylle Jabonka</a></div></div><div id="_co_idm140187941788016"><h3 class="no_margin">Michael Sendtner</h3><p><sup>36</sup>Institute of Clinical Neurobiology, University of Wuerzburg, Wuerzburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Sendtner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Michael Sendtner</a></div></div><div id="_co_idm140187941785792"><h3 class="no_margin">Marcus Beck</h3><p><sup>36</sup>Institute of Clinical Neurobiology, University of Wuerzburg, Wuerzburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Beck%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Marcus Beck</a></div></div><div id="_co_idm140187942947728"><h3 class="no_margin">Lucie Bruijn</h3><p><sup>37</sup>The ALS Association, Palm Harbor, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Bruijn%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Lucie Bruijn</a></div></div><div id="_co_idm140187942945504"><h3 class="no_margin">Jeffrey Rothstein</h3><p><sup>38</sup>Department of Neurology, Johns Hopkins University, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Rothstein%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Jeffrey Rothstein</a></div></div><div id="_co_idm140187942943280"><h3 class="no_margin">Silke Schmidt</h3><p><sup>10</sup>Center for Human Genetics, Duke University Medical Center, Durham, NC, USA</p><p><sup>11</sup>Department of Medicine, Duke University Medical Center, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Schmidt%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Silke Schmidt</a></div></div><div id="_co_idm140187949732176"><h3 class="no_margin">Andrew Singleton</h3><p><sup>5</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Singleton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Andrew Singleton</a></div></div><div id="_co_idm140187949410192"><h3 class="no_margin">John Hardy</h3><p><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><p><sup>6</sup>Department of Molecular Neuroscience and Reta Lila Weston Institute of Neurological Studies, Institute of Neurology, Queen Square, London, UK</p><div>Find articles by <a href="/pubmed/?term=Hardy%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">John Hardy</a></div></div><div id="_co_idm140187949407232"><h3 class="no_margin">Bryan J. Traynor</h3><p><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</p><p><sup>38</sup>Department of Neurology, Johns Hopkins University, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Traynor%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19193627">Bryan J. Traynor</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140187942977408_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140187942977408_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140187942977408_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140187942977408_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Neuroscience, University of Turin, Turin, Italy</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Physiology, Anatomy and Genetics, University of Oxford, Oxford, UK</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Molecular Genetics Unit, Department of Clinical Pathology, A.S.O. O.I.R.M.-S.Anna, Turin, Italy</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Department of Molecular Neuroscience and Reta Lila Weston Institute of Neurological Studies, Institute of Neurology, Queen Square, London, UK</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Neurology, Chang Gung Memorial Hospital and College of Medicine, Taiwan, Republic of China</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Salvatore Maugeri Foundation, Lissone, Italy</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Computational Biology Core, Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, MD, USA</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Center for Human Genetics, Duke University Medical Center, Durham, NC, USA</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Department of Medicine, Duke University Medical Center, Durham, NC, USA</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Epidemiology Research and Information Center, Durham VAMC, Durham, NC, USA</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Longitudinal Studies Section, Clinical Research Branch, National Institute on Aging, Baltimore, MD, USA</div><div class="fm-affl" id="af14" lang="en"><sup>14</sup>Geriatric Unit, Azienda Sanitaria di Firenze, Florence, Italy</div><div class="fm-affl" id="af15" lang="en"><sup>15</sup>Laboratory of Epidemiology, Demography and Biometry, National Institute on Aging, Bethesda, MD, USA</div><div class="fm-affl" id="af16" lang="en"><sup>16</sup>Department of Science and Biomedical Technology, University of Milan, Milan, Italy</div><div class="fm-affl" id="af17" lang="en"><sup>17</sup>INSPE, San Raffaele Scientific Institute, Milan, Italy</div><div class="fm-affl" id="af18" lang="en"><sup>18</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, NIH, Bethesda, MD, USA</div><div class="fm-affl" id="af19" lang="en"><sup>19</sup>Department of Epidemiology, Harvard School of Public Health, Boston, MA, USA</div><div class="fm-affl" id="af20" lang="en"><sup>20</sup>Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg/Munich, Germany</div><div class="fm-affl" id="af21" lang="en"><sup>21</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany</div><div class="fm-affl" id="af22" lang="en"><sup>22</sup>Department of Neuroscience, Neurology Section, University of Siena, Siena, Italy</div><div class="fm-affl" id="af23" lang="en"><sup>23</sup>Department of Neuroscience, Ophthalmology and Genetics, University of Genoa, Genoa, Italy</div><div class="fm-affl" id="af24" lang="en"><sup>24</sup>Department of Clinical Neurosciences, University of Palermo, Sicily, Italy</div><div class="fm-affl" id="af25" lang="en"><sup>25</sup>Department of Neurological Sciences, Second University of Naples, Caserta, Italy</div><div class="fm-affl" id="af26" lang="en"><sup>26</sup>Neurological Institute, Catholic University and I.CO.M.M. Association for ALS Research, Rome, Italy</div><div class="fm-affl" id="af27" lang="en"><sup>27</sup>Department of Neuroscience, S. Agostino- Estense Hospital and University of Modena, Modena, Italy</div><div class="fm-affl" id="af28" lang="en"><sup>28</sup>Department of Neurology, Center for Diagnosis and Cure of Rare Diseases, Bellaria Hospital, Bologna, Italy</div><div class="fm-affl" id="af29" lang="en"><sup>29</sup>Department of Neuroscience, University of Pisa, Pisa, Italy</div><div class="fm-affl" id="af30" lang="en"><sup>30</sup>University Department of Clinical Neurosciences, Institute of Neurology, University College London, London</div><div class="fm-affl" id="af31" lang="en"><sup>31</sup>Department of Neurology, Penn State College of Medicine, Hershey, PA, USA</div><div class="fm-affl" id="af32" lang="en"><sup>32</sup>Department of Neurosurgery, Penn State College of Medicine, Hershey, PA, USA</div><div class="fm-affl" id="af33" lang="en"><sup>33</sup>Department of Neurology, Cleveland Clinic, Cleveland, OH, USA</div><div class="fm-affl" id="af34" lang="en"><sup>34</sup>Neurogenomics Division, Translational Genomics Institute (TGEN), Phoenix, AZ, USA</div><div class="fm-affl" id="af35" lang="en"><sup>35</sup>Department of Neurodegenerative Disease, UCL Institute of Neurology, Queen Square, London, UK</div><div class="fm-affl" id="af36" lang="en"><sup>36</sup>Institute of Clinical Neurobiology, University of Wuerzburg, Wuerzburg, Germany</div><div class="fm-affl" id="af37" lang="en"><sup>37</sup>The ALS Association, Palm Harbor, FL, USA</div><div class="fm-affl" id="af38" lang="en"><sup>38</sup>Department of Neurology, Johns Hopkins University, Baltimore, MD, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: National Institutes of Health, Building 35, Room 1A/1000, 35 Convent Drive, Bethesda, MD 20892-3720, USA. Tel: +1 3014517606; Fax: +1 3014517295; Email: <a href="mailto:dev@null" data-email="vog.hin.liam@bronyart" class="oemail">vog.hin.liam@bronyart</a></div><div id="FN1"><sup>†</sup>The authors wish it to be known that, in their opinion, the first three authors should be regarded as joint First Authors.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140187942977408_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 Dec 2; Accepted 2009 Feb 2.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140187942977408_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2009. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oxfordjournals.org</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2664150/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
[Supplementary Data]
</div>

<div><a href="/pmc/articles/PMC2664150/bin/ddp059_index.html" data-ga-action="click_feat_suppl">ddp059_index.html</a><span style="color:gray"> (932 bytes)</span></div><div class="small guid">GUID:&nbsp;03CDCD49-4EDB-4AC9-8C7E-28130B6E86B1</div>
<div><a href="/pmc/articles/PMC2664150/bin/ddp059_1.pdf" data-ga-action="click_feat_suppl">ddp059_1.pdf</a><span style="color:gray"> (2.4M)</span></div><div class="small guid">GUID:&nbsp;13911AD9-60DC-467F-8754-B2835A192ABE</div>
<div><a href="/pmc/articles/PMC2664150/bin/ddp059_ddp059_Table_3.xls" data-ga-action="click_feat_suppl">ddp059_ddp059_Table_3.xls</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID:&nbsp;C3332BD5-5280-4145-B7B3-C541A8CA4E02</div>
<div><a href="/pmc/articles/PMC2664150/bin/ddp059_ddp059_Table_4.xls" data-ga-action="click_feat_suppl">ddp059_ddp059_Table_4.xls</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;671D6803-A0E9-4248-9BE9-E7E0E3DE09F0</div>
</div></dd></dl></div><div id="idm140187943101984" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140187943101984title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">The cause of sporadic amyotrophic lateral sclerosis (ALS) is largely unknown, but genetic factors are thought to play a significant role in determining susceptibility to motor neuron degeneration. To identify genetic variants altering risk of ALS, we undertook a two-stage genome-wide association study (GWAS): we followed our initial GWAS of 545 066 SNPs in 553 individuals with ALS and 2338 controls by testing the 7600 most associated SNPs from the first stage in three independent cohorts consisting of 2160 cases and 3008 controls. None of the SNPs selected for replication exceeded the Bonferroni threshold for significance. The two most significantly associated SNPs, rs2708909 and rs2708851 [odds ratio (OR) = 1.17 and 1.18, and <em>P</em>-values = 6.98 × 10<sup>−7</sup> and 1.16 × 10<sup>−6</sup>], were located on chromosome 7p13.3 within a 175 kb linkage disequilibrium block containing the <em>SUNC1</em>, <em>HUS1</em> and <em>C7orf57</em> genes. These associations did not achieve genome-wide significance in the original cohort and failed to replicate in an additional independent cohort of 989 US cases and 327 controls (OR = 1.18 and 1.19, <em>P</em>-values = 0.08 and 0.06, respectively). Thus, we chose to cautiously interpret our data as hypothesis-generating requiring additional confirmation, especially as all previously reported loci for ALS have failed to replicate successfully. Indeed, the three loci (<em>FGGY</em>, <em>ITPR2</em> and <em>DPP6</em>) identified in previous GWAS of sporadic ALS were not significantly associated with disease in our study. Our findings suggest that ALS is more genetically and clinically heterogeneous than previously recognized. Genotype data from our study have been made available online to facilitate such future endeavors.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first">Amyotrophic lateral sclerosis (ALS) is a rare and devastating neurodegenerative disease that predominantly affects motor neurons leading to progressive paralysis, and ultimately death from respiratory failure within 3–5 years of symptom onset. Approximately 5% of ALS is familial in nature, whereas the remaining 95% occurs sporadically throughout the community (<a href="#DDP059C1" rid="DDP059C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747901">1</a>). Although the genetic causes of many monogenic familial forms of ALS have been described (<a href="#DDP059C2" rid="DDP059C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747932">2</a>), the etiology of sporadic ALS is largely unknown. Familial aggregation studies, twin studies and epidemiological observations have suggested a substantial genetic contribution to disease risk (<a href="#DDP059C3" rid="DDP059C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747895">3</a>,<a href="#DDP059C4" rid="DDP059C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747934">4</a>). Recently, genome-wide association studies (GWAS) have putatively identified variants with moderate effects on the risk of developing ALS in the 1p32.1 region (<em>FGGY</em>) (<a href="#DDP059C5" rid="DDP059C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747939">5</a>), in the 12p11 region (<em>ITPR2</em>) (<a href="#DDP059C6" rid="DDP059C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747913">6</a>) and in the 7q36.2 region (<em>DPP6</em>) (<a href="#DDP059C7" rid="DDP059C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747903">7</a>,<a href="#DDP059C8" rid="DDP059C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747926">8</a>). However, these loci require replication in independent cohorts to confirm disease association, and, at most, account for only a fraction of the elevated risk of developing ALS, suggesting that additional genetic factors exist.</p><p id="__p4" class="p p-last">We conducted a two-stage GWAS to search for common variants with moderate risk (<a href="#DDP059C9" rid="DDP059C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747904">9</a>,<a href="#DDP059C10" rid="DDP059C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747893">10</a>). For the first stage, we used 555 352 SNPs that extract information on 91% of common autosomal SNPs identified in the European populations based on the HapMap data [CEU, <em>r</em><sup>2</sup> &gt; 0.8, minor allele frequency (MAF) &gt; 5%] (<a href="#DDP059C10" rid="DDP059C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747947">10</a>,<a href="#DDP059C11" rid="DDP059C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747919">11</a>). These SNPs were genotyped in two independent cohorts of European origin consisting of 553 ALS cases and 2338 controls. For the second stage, we analyzed the 7600 SNPs that were most associated with the altered risk of disease in the initial genome-wide scan in an additional 2160 cases and 3008 controls. The large number of SNPs and samples genotyped in the second stage provided sufficient power to follow up on regions with moderate association in the initial genome-wide scan (threshold <em>P</em>-value for follow-up study &lt;0.005).</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p5" class="p p-first">We conducted the initial genome-wide scan in a case-control cohort of 553 ALS cases and 2338 neurologically normal control of European ancestry. In the second stage, we genotyped 7600 of the most associated SNPs from the first stage in three additional replication cohorts totaling 2160 ALS cases, and compared this with data for the same 7600 SNPs in three control cohorts totaling 3008 samples. After quality control procedures, 6758 SNPs were available for analysis in a final combined stage 1 and stage 2 cohort of 2289 cases and 4532 controls. These SNPs covered 3152 distinct chromosomal regions defined by a maximal distance between two SNPs of &lt;100 kb; 1745 regions contained only one SNP and 40 regions contained 10 or more SNPs. Of these regions, 94 had at least one SNP with an observed <em>P</em>-value &lt;10<sup>−3</sup> (Fig.&nbsp;<a href="/pmc/articles/PMC2664150/figure/DDP059F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP059F1" rid-ob="ob-DDP059F1" co-legend-rid="lgnd_DDP059F1"><span>1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDP059F1" co-legend-rid="lgnd_DDP059F1"><a href="/pmc/articles/PMC2664150/figure/DDP059F1/" target="figure" rid-figpopup="DDP059F1" rid-ob="ob-DDP059F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140187939943744" class="figure"><a href="/pmc/articles/PMC2664150/figure/DDP059F1/" target="figure" rid-figpopup="DDP059F1" rid-ob="ob-DDP059F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2664150_ddp05901.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddp05901.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2664150/bin/ddp05901.jpg"></a></div><div id="largeobj_idm140187939943744" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2664150/figure/DDP059F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDP059F1"><div><a class="figpopup" href="/pmc/articles/PMC2664150/figure/DDP059F1/" target="figure" rid-figpopup="DDP059F1" rid-ob="ob-DDP059F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p6">Association analysis of combined joints analysis in two-stage GWAS of ALS (<em>n</em> = 2289 ALS cases and 4532 controls) based on the logistic regression model correcting for age, gender and population. SNPs listed in Table&nbsp;<a href="/pmc/articles/PMC2664150/table/DDP059TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP059TB1" rid-ob="ob-DDP059TB1" co-legend-rid=""><span>1</span></a> are represented by red dots.</p></div></div></div><p id="__p7">None of the SNPs tested in this study clearly achieved genome-wide significance after correction for multiple testing (see <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a> for association results of all 6758 tested SNPs). The SNPs with the lowest <em>P</em>-values identified by our two-stage GWAS were located on chromosome 7p12.3 (Table&nbsp;<a href="/pmc/articles/PMC2664150/table/DDP059TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP059TB1" rid-ob="ob-DDP059TB1" co-legend-rid=""><span>1</span></a>), a region which has not been previously linked to the pathogenesis of ALS. The SNPs were located within a 175 kb linkage disequilibrium (LD) block containing three genes, <em>SUNC1</em>, <em>HUS1</em> and <em>C7orf57</em>. The strongest signal was observed for rs2708909 located in the third intron of the gene <em>SUNC1</em> (<em>P</em>-value = 6.98 × 10<sup>−7</sup> in combined analysis, <a id="__tag_380292426" class="tag_hotlink" href="/nuccore/NM_152782.3" ref="/nuccore/NM_152782.3"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_152782.3","term_id":"71834867","term_text":"NM_152782.3"}}</span>NM_152782.3</a>), which encodes the ‘SAD1 and UNC84 domain containing 1’ protein. The second SNP, rs2708851, was in complete LD with rs2708909 (<em>r</em><sup>2</sup> = 0.97; Fig.&nbsp;<a href="/pmc/articles/PMC2664150/figure/DDP059F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP059F2" rid-ob="ob-DDP059F2" co-legend-rid="lgnd_DDP059F2"><span>2</span></a>), and was located 22 kb upstream of <em>SUNC1</em> within intron 4 of <em>C7orf57</em> (<a id="__tag_380292425" class="tag_hotlink" href="/nuccore/NM_001100159.1" ref="/nuccore/NM_001100159.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001100159.1","term_id":"154091006","term_text":"NM_001100159.1"}}</span>NM_001100159.1</a>). These SNPs did not exceed the threshold for genome-wide significance in the overall cohort, were only marginally associated with ALS risk when analyzed in the individual North American and Italian cohorts (<em>P</em>-values for rs2708909 = 5.40 × 10<sup>−5</sup> and 0.0006, respectively), and were not associated in the German dataset (<em>P</em>-value for rs2708909 = 0.503), probably reflecting the smaller size of this cohort.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDP059F2" co-legend-rid="lgnd_DDP059F2"><a href="/pmc/articles/PMC2664150/figure/DDP059F2/" target="figure" rid-figpopup="DDP059F2" rid-ob="ob-DDP059F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140187948023456" class="figure"><a href="/pmc/articles/PMC2664150/figure/DDP059F2/" target="figure" rid-figpopup="DDP059F2" rid-ob="ob-DDP059F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2664150_ddp05902.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddp05902.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2664150/bin/ddp05902.jpg"></a></div><div id="largeobj_idm140187948023456" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2664150/figure/DDP059F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDP059F2"><div><a class="figpopup" href="/pmc/articles/PMC2664150/figure/DDP059F2/" target="figure" rid-figpopup="DDP059F2" rid-ob="ob-DDP059F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p8">Location of the association signal and pairwise linkage disequilibrium (LD) surrounding the most associated SNPs on chromosome 7p12.3. LD pattern is depicted using stage 1 US data. Association signals are shown for all SNPs genotyped in (<strong>A</strong>) stage 1 US samples (blue squares, <em>n</em> = 1065); (<strong>B</strong>) stage 1 Italian samples (green triangles, <em>n</em> = 1456); (<strong>C</strong>) stage 2 all populations (orange circles, <em>n</em> = 4300), and the combined dataset (red diamonds, <em>n</em> = 6821). The most associated SNPs, rs2708909 and rs2708851, lie in or near gene <em>SUNC1</em>, and are in almost complete LD (<em>D</em>′ = 0.981, <em>r</em><sup>2</sup> = 0.959 based on stage 1 US data). Plots were produced using the snp.plotter package within R version 2.6.1.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP059TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p9">SNPs with a <em>P</em>-value &lt;10<sup>−6</sup> based on the logistic regression model of 2289 ALS cases and 4532 controls</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140187948020224" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">rs2708909</th><th rowspan="1" colspan="1" align="left">rs2708851</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Chromosomal location</td><td rowspan="1" colspan="1">7p12.3</td><td rowspan="1" colspan="1">7p12.3</td></tr><tr><td rowspan="1" colspan="1">Position</td><td rowspan="1" colspan="1">48,018,204</td><td rowspan="1" colspan="1">48,052,327</td></tr><tr><td rowspan="1" colspan="1">Nearest gene (SNP type)</td><td rowspan="1" colspan="1"><em>SUNC1</em> (intronic)</td><td rowspan="1" colspan="1"><em>SUNC1</em> (intergenic)</td></tr><tr><td rowspan="1" colspan="1">Risk allele</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td></tr><tr><td rowspan="1" colspan="1">Minor allele</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1">Major allele</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td></tr><tr><td rowspan="1" colspan="1">Minor allele frequency (cases)</td><td rowspan="1" colspan="1">0.449</td><td rowspan="1" colspan="1">0.451</td></tr><tr><td rowspan="1" colspan="1">Minor allele frequency (controls)</td><td rowspan="1" colspan="1">0.496</td><td rowspan="1" colspan="1">0.497</td></tr><tr><td rowspan="1" colspan="1">Allelic odds ratio (95% CI)</td><td rowspan="1" colspan="1">1.17 (1.11–1.23)</td><td rowspan="1" colspan="1">1.17 (1.11–1.23)</td></tr><tr><td rowspan="1" colspan="1">Population attributable risk (95% CI)</td><td rowspan="1" colspan="1">0.08 (0.05–0.12)</td><td rowspan="1" colspan="1">0.08 (0.05–0.12)</td></tr><tr><td colspan="3" rowspan="1"><em>All samples</em></td></tr><tr><td rowspan="1" colspan="1"> <em>P</em>-value</td><td rowspan="1" colspan="1">6.98 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">1.16 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (cases)</td><td rowspan="1" colspan="1">0.551</td><td rowspan="1" colspan="1">0.549</td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (controls)</td><td rowspan="1" colspan="1">0.504</td><td rowspan="1" colspan="1">0.503</td></tr><tr><td rowspan="1" colspan="1"> Inter-SNP LD (<em>r</em><sup>2</sup>)</td><td rowspan="1" colspan="1">0.949</td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="3" rowspan="1"><em>North American samples</em></td></tr><tr><td rowspan="1" colspan="1"> <em>P</em>-value</td><td rowspan="1" colspan="1">5.40 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">2.61 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (cases)</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.54</td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (controls)</td><td rowspan="1" colspan="1">0.488</td><td rowspan="1" colspan="1">0.512</td></tr><tr><td rowspan="1" colspan="1"> Inter-SNP LD (<em>r</em><sup>2</sup>)</td><td rowspan="1" colspan="1">0.956</td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="3" rowspan="1"><em>Italian samples</em></td></tr><tr><td rowspan="1" colspan="1"> <em>P</em>-value</td><td rowspan="1" colspan="1">0.0006</td><td rowspan="1" colspan="1">0.0011</td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (cases)</td><td rowspan="1" colspan="1">0.590</td><td rowspan="1" colspan="1">0.589</td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (controls)</td><td rowspan="1" colspan="1">0.548</td><td rowspan="1" colspan="1">0.548</td></tr><tr><td rowspan="1" colspan="1">Inter-SNP LD (<em>r</em><sup>2</sup>)</td><td rowspan="1" colspan="1">0.927</td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="3" rowspan="1"><em>German samples</em></td></tr><tr><td rowspan="1" colspan="1"> <em>P</em>-value</td><td rowspan="1" colspan="1">0.503</td><td rowspan="1" colspan="1">0.611</td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (cases)</td><td rowspan="1" colspan="1">0.505</td><td rowspan="1" colspan="1">0.50</td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (controls)</td><td rowspan="1" colspan="1">0.489</td><td rowspan="1" colspan="1">0.488</td></tr><tr><td colspan="3" rowspan="1"><em>National Registry of Veterans with ALS</em></td></tr><tr><td rowspan="1" colspan="1"> <em>P</em>-value</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.06</td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (cases)</td><td rowspan="1" colspan="1">0.499</td><td rowspan="1" colspan="1">0.499</td></tr><tr><td rowspan="1" colspan="1"> Risk allele frequency (controls)</td><td rowspan="1" colspan="1">0.457</td><td rowspan="1" colspan="1">0.456</td></tr><tr><td rowspan="1" colspan="1"> Inter-SNP LD (<em>r</em><sup>2</sup>)</td><td rowspan="1" colspan="1">0.961</td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm140187948020224" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2664150/table/DDP059TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140187945480816"><p id="__p10" class="p p-first-last">Position was based on NCBI Build 36.3; allele frequencies, odds ratios [with 95% confidence intervals (CI)] and population attributable risk were calculated using the combined stage 1 and stage 2 data; risk allele, the allele with higher frequency in cases compared with controls; minor allele, the allele with the higher frequency in controls; major allele, the allele with higher frequency in controls; <em>P</em>-values for <em>all samples</em> were based on the logistic regression model corrected for age, gender and population, and were based on the logistic regression model corrected for age and gender for the individual populations; all samples, 2289 ALS cases and 4532 controls included in the combined joint analysis; North American samples, 869 ALS cases and 2858 controls; Italian samples, 871 Italian ALS cases and 1190 Italian controls; German samples consisted of 549 ALS cases and 484 controls; National Registry of Veterans with ALS samples consisted of 989 ALS cases and 327 controls; inter-SNP LD (<em>r</em><sup>2</sup>), linkage disequilibrium between rs2708909 and rs2708851 measured as <em>r</em><sup>2</sup> (Haploview 4.1) (<a href="#DDP059C36" rid="DDP059C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747946">36</a>).</p></div></div></div><p id="__p11">To further test the association with increased risk of disease, we genotyped rs2708851 and rs2708909 in a dataset of 989 US cases with ALS or other motor neuron diseases (MNDs) and 327 neurologically normal US controls, all of whom were of non-Hispanic Caucasian ethnicity and had previously served in the US military (<a href="#DDP059C12" rid="DDP059C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747945">12</a>,<a href="#DDP059C13" rid="DDP059C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747890">13</a>). This sample set represented and independent sample, as none of these samples were included in the initial genome-wide stage or in the replication stage. Rs2708909 and rs2708851 failed to reach significance [<em>P</em>-value = 0.08 for rs2708909 and 0.06 for rs2708851 based on a logistic regression model correcting for age at onset and gender, odds ratio (OR) = 1.176 and 1.189, respectively], though the sample size was underpowered to detect moderate effect alleles (power to detect OR of 1.17 for a MAF of 0.45 = 41.1% at <em>P</em>-value of 0.05). The results are very similar when only patients diagnosed with definite or probable ALS were analyzed as cases (<em>P</em>-value for rs2708909 = 0.09; <em>P</em>-value for rs2708851 = 0.06). Furthermore, no evidence for association with the previously implicated SNPs in <em>ITPR2</em> and <em>DPP6</em> was found in this dataset (rs109260404: <em>P</em>-value = 0.62; rs2306677: <em>P</em>-value = 0.77) (<a href="#DDP059C14" rid="DDP059C14" class=" bibr popnode">14</a>).</p><p id="__p12">Rs2708909 and rs2708851 lie within a 175 kb region of LD (multiallelic <em>D</em>′&gt;0.8) on chromosome 7p12.3. Using our stage 1 datasets, we found that the HapMap CEU, the US and the Italian populations share an almost identical haplotype structure across this region (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>), and determined that seven SNPs (rs6955251, rs2686821, rs2686831, rs2708909, rs2708851, rs2307252 and rs2708912) account for 85% of the variation across the 175 kb region at an <em>r</em><sup>2</sup> &gt; 0.5. The first five of these markers had been genotyped as part of our stage 1 and stage 2 datasets. To investigate whether other SNPs in the same region were more significantly associated with altered risk of developing ALS, we analyzed genotype data for the two additional SNPs rs2307252 and rs2708912 for all stage 1 cases and controls (based on previous whole genome data, <em>n</em> = 2521), stage 2 controls (based on previous whole genome data, <em>n</em> = 2548) and 216 stage 2 US cases (based on additional sequencing data). Neither rs2307252 nor rs2708912 achieved genome-wide significance (<em>P</em>-values = 0.47 and 0.16) based on this cohort of 753 cases and 4532 controls. Next, we applied imputation to our stage 1 data using MACH version 1.0, but none of the untyped SNPs in the region of 7p12.3 provided stronger evidence of association compared with rs2708909 and rs2708851 (Fig.&nbsp;<a href="/pmc/articles/PMC2664150/figure/DDP059F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP059F2" rid-ob="ob-DDP059F2" co-legend-rid="lgnd_DDP059F2"><span>2</span></a>).</p><p id="__p13">Our two-stage GWAS identified several additional loci with <em>P</em>-values less than 10<sup>−3</sup> representing hypotheses that may merit additional follow-up studies (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>) (<a href="#DDP059C9" rid="DDP059C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747906">9</a>,<a href="#DDP059C10" rid="DDP059C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747930">10</a>). The three loci (<em>FGGY</em>, <em>ITPR2</em> and <em>DPP6</em>) identified in previous GWAS of sporadic ALS (<a href="#DDP059C5" rid="DDP059C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747897">5</a>–<a href="#DDP059C8" rid="DDP059C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747938">8</a>) did not alter risk of developing disease in either the combined case-control cohort or in the three individual populations examined in our study (Table&nbsp;<a href="/pmc/articles/PMC2664150/table/DDP059TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP059TB2" rid-ob="ob-DDP059TB2" co-legend-rid=""><span>2</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP059TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p14"><em>P</em>-values from the two-stage GWAS for SNPs that have been previously associated with increased risk of developing ALS (<a href="#DDP059C5" rid="DDP059C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747936">5</a>–<a href="#DDP059C7" rid="DDP059C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747925">7</a>)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140187945799728" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="left">Risk allele (frequency affected)</th><th colspan="4" rowspan="1" align="left"><em>P</em>-value<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">All samples</th><th rowspan="1" colspan="1" align="left">US samples</th><th rowspan="1" colspan="1" align="left">Italian samples</th><th rowspan="1" colspan="1" align="left">German samples</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs6700125</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>FGGY</em></td><td rowspan="1" colspan="1">T (0.36)</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.55</td><td rowspan="1" colspan="1">0.35</td></tr><tr><td rowspan="1" colspan="1">rs2306677</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1"><em>ITPR2</em></td><td rowspan="1" colspan="1">T (0.11)</td><td rowspan="1" colspan="1">0.74</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.97</td></tr><tr><td rowspan="1" colspan="1">rs10260404</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1"><em>DPP6</em></td><td rowspan="1" colspan="1">C (0.39)</td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">0.28</td></tr></tbody></table></div><div id="largeobj_idm140187945799728" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2664150/table/DDP059TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140187945846288"><p id="__p15" class="p p-first-last">Chr, chromosome; risk allele, the allele with higher frequency in cases compared with controls (<a href="#DDP059C5" rid="DDP059C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747915">5</a>–<a href="#DDP059C7" rid="DDP059C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747892">7</a>); frequency affected, frequency of risk allele in 2289 ALS cases included in the current study; <em>P</em>-values for <em>all samples</em> were based on the logistic regression model corrected for age, gender and population, and were based on the logistic regression model corrected for age and gender for the individual populations; all samples, 2289 ALS cases and 4532 controls included in the combined joint analysis; North American samples, 869 ALS cases and 2858 controls; Italian samples, 871 Italian ALS cases and 1190 Italian controls; German samples consisted of 549 ALS cases and 484 controls.</p></div></div></div><p id="__p16" class="p p-last">Raw sample-level genotype data from the initial GWAS study (North American ALS cases, North American controls, Italian ALS cases and Italian controls from the Piemonte/Turin region) are available for download through the dbGAP portal (<a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000006.v1.p1">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000006.v1.p1</a>). Individual genotype data for the CGEMS dataset are available for registered users through the CGEMS portal (<a href="http://cgems.cancer.gov/data">http://cgems.cancer.gov/data</a>.), whereas individual genotype data for the KORA cohort may be requested (<a href="http://epi.gsf.de/kora-gen/index_e.php">http://epi.gsf.de/kora-gen/index_e.php</a>).</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p17" class="p p-first">Here we present the results of our two-stage GWAS involving 2713 cases and 5346 controls. This analysis was corrected based on the 531 661 autosomal SNPs genotyped in the initial whole genome scan, rather than the smaller number of SNPs followed up in the replication stage, as we wanted to be as conservative as possible. Our study represents the largest GWAS published to date in ALS, and the first to be sufficiently powered to reliably detect moderately associated SNPs in this relatively rare, fatal neurodegenerative disease.</p><p id="__p18">Our study did not identify any SNPs that clearly exceeded the standard threshold for genome-wide significance (i.e. &lt;10<sup>−7</sup>), and there is little or no overlap with the results of previously published studies (<a href="#DDP059C5" rid="DDP059C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747924">5</a>–<a href="#DDP059C8" rid="DDP059C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747943">8</a>). This contrasts with GWAS in other neurological diseases such as multiple sclerosis, where the most associated SNP in the HLA-DRA locus had a <em>P</em>-value &lt; 10<sup>−80</sup> (<a href="#DDP059C15" rid="DDP059C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747935">15</a>). One possible explanation for this lack of success may be that ALS is a more genetically and clinically heterogeneous disease than previously appreciated, which would significantly limit the power of GWAS. The identification of multiple different familial ALS genes, each involving disparate biological pathways, supports this notion (<a href="#DDP059C16" rid="DDP059C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747907">16</a>). Alternatively, causative genetic factors may increase the risk of motor neuron degeneration by only a small amount (i.e. OR &lt; 1.2) meaning that even larger GWAS involving 5000–10 000 will be required to reliably identify loci (<a href="#DDP059C17" rid="DDP059C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747927">17</a>). Finally, we compensated for the relatively small number of cases in the first stage of our study by selecting several thousand SNPs for detailed follow-up. Although this approach is likely to be adequate for identifying alleles of moderate effects (i.e. OR of &gt; 1.4), mild effect alleles could easily have failed to reach the threshold for inclusion in the replication stage.</p><p id="__p19">Our study also failed to replicate the three loci (<em>FGGY</em>, <em>ITPR2</em> and <em>DPP6</em>) that had been previously published as been associated with increased risk of sporadic ALS (<a href="#DDP059C5" rid="DDP059C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747896">5</a>–<a href="#DDP059C8" rid="DDP059C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747916">8</a>). This finding agrees with data from the National Registry of Veterans with ALS, which also failed to replicate these loci in a cohort of 989 cases and 327 controls (<a href="#DDP059C14" rid="DDP059C14" class=" bibr popnode">14</a>). There are several possible explanations for this finding. First, the lack of replication of these loci in the current study may be explained by the small number of SNPs selected from the initial genome-wide scans of the Dutch and TGEN studies for follow-up to confirm disease association. In the Dutch study of 461 cases and 450 controls, the 200 most associated SNPs were brought forward to the replication stages (<a href="#DDP059C6" rid="DDP059C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747942">6</a>), whereas the TGEN study used a DNA pooling methodology involving 386 North American sporadic ALS cases and 542 controls to select 192 SNPs for individual-level genotyping (<a href="#DDP059C5" rid="DDP059C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747928">5</a>). The several hundred thousand tests performed as part of any GWAS make it more likely that the most associated SNPs in the initial genome-wide scan represent false positive associations arising by chance (‘winner's curse’). Indeed, previous two-stage GWAS studies have repeatedly shown that truly causative SNPs are often not ranked in the top 1000 SNPs in the initial genome-wide scan (<a href="#DDP059C10" rid="DDP059C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747923">10</a>), which led us to select a large number of SNPs for replication in our stage 2 analysis. Another possible explanation for the lack of replication of the <em>FGGY</em>, <em>ITPR2</em> and <em>DPP6</em> is that the initial Dutch and TGEN studies identified markers that are not in strong LD with the causal variant, leading to a false refutation in our study that was based on different populations (<a href="#DDP059C9" rid="DDP059C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747950">9</a>).</p><p id="__p20">The chromosome 7 risk variants putatively identified as hypotheses by our study were not associated with disease when analyzed in the individual German population included in the study. Although population-to-population variation in causative genes has been postulated for ALS (<a href="#DDP059C18" rid="DDP059C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747905">18</a>,<a href="#DDP059C19" rid="DDP059C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747899">19</a>), our findings are more likely to reflect the smaller number of the samples from the individual populations included in the study, and the consequent loss in power to detect moderate effect loci: the smallest German cohort of 549 cases and 484 controls had only 16.4% power to detect the <em>SUNC1</em> locus (assuming an OR of 1.18 and a MAF of 0.45), whereas the larger North American dataset of 3727 samples had 55.1% power under the same parameters. Indeed, the putative association of rs2708909 and rs2708851 with ALS is only apparent in the combined analysis of 2289 cases and 4532 controls (power to detect <em>SUNC1</em> locus = 94.8%), emphasizing the necessity of using several thousand samples to detect variants that only moderately increase risk of developing sporadic ALS (<a href="#DDP059C20" rid="DDP059C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747922">20</a>).</p><p id="__p21">Even if we assume that the chromosome 7 variants are truly associated with ALS, we are left with the problem of determining which gene within this LD block is responsible for increased risk of disease. The location of the variant with the most significant <em>P</em>-value within the intron of <em>SUNC1</em> would suggest that this gene is the most likely candidate. Indeed, <em>SUNC1</em> encodes a 40.5 kDa nuclear envelope protein ‘Sad1 and UNC84 domain containing 1’ (<a href="#DDP059C21" rid="DDP059C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747891">21</a>), and mutations in nuclear envelope proteins underlie a variety of neuromuscular diseases including Charcot-Marie-Tooth disease, type 2B1 (<a href="#DDP059C22" rid="DDP059C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747917">22</a>) and spastin-associated hereditary spastic paraplegia (<a href="#DDP059C23" rid="DDP059C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747944">23</a>). However, these biological hypotheses should be interpreted cautiously: although the gene lying closest to an associated SNP is generally considered to be the prime suspect in disease pathogenesis, a number of alternative pathogenic mechanisms must be considered: our own studies have shown that the associated SNP may ‘tag’ the true causative variant residing many kilobases distant in another gene; the associated SNP could affect expression of <em>cis</em> genes up to 100 Kb distant, or could act in <em>trans</em> to alter gene expression on other chromosomes (<a href="#DDP059C24" rid="DDP059C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747911">24</a>); alternatively, the SNP could alter the function or tissue-specific expression of a previously unidentified microRNA or genetic element. Furthermore, despite the large number of samples analyzed in our study, replication of the locus in independent cohorts remains a necessity (<a href="#DDP059C9" rid="DDP059C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747929">9</a>). The two SNPs reported in the current study did not achieve significant association with disease in a separate cohort of 221 Irish ALS cases and 211 neurologically normal controls, though the small size of this cohort precludes firm conclusions being drawn from this data (Irish data was not included in the current study as necessary covariates were not available from the investigators associated with the study) (<a href="#DDP059C7" rid="DDP059C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747949">7</a>). Public release of raw genotype data is helpful in this regard, as it reduces the expense of future whole genome association studies and allows researchers to have greater confidence in the results of their association studies by increasing sample size and power to accurately detect causative loci (<a href="#DDP059C25" rid="DDP059C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747921">25</a>). Our initial public release of data established a powerful, unique resource for the ALS research community (<a href="#DDP059C25" rid="DDP059C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747910">25</a>), and this data has been incorporated into all other ALS GWAS published to date (<a href="#DDP059C5" rid="DDP059C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747940">5</a>–<a href="#DDP059C8" rid="DDP059C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747941">8</a>). Coincident with publication, we have augmented this initiative with data from all 2713 ALS cases genotyped in the current study.</p><p id="__p22" class="p p-last">In summary, we present the results of our two-stage genome-wide association in a large cohort of sporadic ALS patients. None of the studied loci clearly achieved genome-wide level of significance, and none of the previously published loci were significantly associated with disease in our study. Though the data supporting an association of the chromosome 7p12.3 variants are suggestive, we chose to interpret these results cautiously as loci previously reported to be associated with increased risk of developing ALS have uniformly failed to replicate (<a href="#DDP059C26" rid="DDP059C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747902">26</a>). Thus, these variants should be considered as hypothesis-generating that require additional replication to confirm or refute their veracity. The current lack of success of GWAS in sporadic ALS may indicate that the disease is more heterogeneous than previously recognized, and highlights the fact that even larger sample numbers will be required to definitively dissect the genetics underlying this fatal neurodegenerative disease.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Initial genome-wide scans</h3><p id="__p23" class="p p-first">We used HumanHap550 version one BeadChips (Illumina Inc., San Diego, CA, USA) to genotype 555 352 SNPs in (i) 276 North American ALS cases and 828 North American neurologically normal control samples obtained from the NINDS Neurogenetics repository at the Coriell Institute for Medical Research (NJ, USA). The initial part of this scan has been previously published, and the raw genotype data made publicly available (<a href="#DDP059C25" rid="DDP059C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747914">25</a>); and (ii) 277 Italian ALS cases and 263 Italian control samples collected within the Turin/Piemonte area of Northern Italy. We also genotyped 561 466 SNPs in an additional cohort of 1247 control samples obtained from the InChianti study, a representative population-based cohort of older persons living in the Chianti geographic area (Tuscany, Central Italy) (<a href="#DDP059C27" rid="DDP059C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747920">27</a>), using the HumanHap550 version three BeadChip (<a href="#DDP059C28" rid="DDP059C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747948">28</a>). Analysis was confined to the 545 066 SNPs that are common to versions one and three HumanHap550 BeadChips. Demographics and clinical features of the case and control cohorts are shown in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>. All patients included in the initial and follow-up stages of the study had been diagnosed with probable, clinically probable, laboratory supported or definite sporadic ALS accorded to the El Escorial criteria (<a href="#DDP059C29" rid="DDP059C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747908">29</a>). Only unrelated, non-Hispanic, white Caucasian individuals of European descent were included in the study.</p><p id="__p24" class="p">Standard quality control procedures (e.g. exclusion of samples with low call rates, mismatch between gender according to phenotype data and gender defined by genotype, non-European ancestry, cryptic relatedness and incomplete phenotype data; and exclusion of SNPs with low call rates, Hardy-Weinberg equilibrium <em>P</em>-value &lt; 0.001 and non-random missingness) were applied to the data (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). After applying these filters, the cohorts included in the initial genome-wide scans consisted of (i) 271 North American sporadic ALS cases and 794 North American control individuals and (ii) 266 Italian sporadic ALS cases and 1190 Italian control samples. A total of 474,554 SNPs were available for association testing in the North American cohort, and 466 131 SNPs in the Italian cohort. Q-Q plots were prepared using R version 2.6.1 (2007, The R Foundation for Statistical Computing) based on genomically controlled association data (Fig.&nbsp;<a href="/pmc/articles/PMC2664150/figure/DDP059F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP059F3" rid-ob="ob-DDP059F3" co-legend-rid="lgnd_DDP059F3"><span>3</span></a>). Genomic inflation factor <em>λ</em> for the US cohort was 1.002 and for the Italian cohort was 1.147. The most significantly associated SNPs identified in the genome-wide scans of the North American and Italian cohorts are listed in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDP059F3" co-legend-rid="lgnd_DDP059F3"><a href="/pmc/articles/PMC2664150/figure/DDP059F3/" target="figure" rid-figpopup="DDP059F3" rid-ob="ob-DDP059F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140187948296096" class="figure"><a href="/pmc/articles/PMC2664150/figure/DDP059F3/" target="figure" rid-figpopup="DDP059F3" rid-ob="ob-DDP059F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2664150_ddp05903.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddp05903.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2664150/bin/ddp05903.jpg"></a></div><div id="largeobj_idm140187948296096" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2664150/figure/DDP059F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDP059F3"><div><a class="figpopup" href="/pmc/articles/PMC2664150/figure/DDP059F3/" target="figure" rid-figpopup="DDP059F3" rid-ob="ob-DDP059F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p25">Q-Q plot based on (<strong>A</strong>) genomic-controlled Cochran-Armitage Trend test <em>P</em>-values for 474 554 SNPs analyzed in the stage 1 271 North American ALS cases and 794 North American controls (genomic inflation factor <em>λ</em> = 1.002); (<strong>B</strong>) genomic-controlled Cochran-Armitage Trend test <em>P</em>-values for 466 131 SNPs analyzed in the stage 1 266 Italian ALS cases and 1190 Italian controls (genomic inflation factor <em>λ</em> = 1.147) and (<strong>C</strong>) logistic regression <em>P</em>-values for 6758 SNPs analyzed in the stage 2 1752 ALS cases and 2548 controls.</p></div></div></div></div><div id="s4b" class="sec"><h3 id="s4btitle">Replication stage</h3><p id="__p26" class="p p-first">A total of 7600 SNPs were selected on the basis of single-SNP association tests in the initial genome-wide scans. Of these, 2533 were selected as they were the most significantly associated SNPs in the North American cohort based on the Cochran-Armitage test; 2533 were the most significant SNPs in the Italian cohort and 2534 SNPs were chosen based on their disease association in the combined Italian and North American GWAS cohorts. These SNPs were genotyped using a custom-designed iSelect Infinium assay (Illumina) in three cohorts of sporadic ALS patients of European background totaling 2160 cases. The case dataset was compared with data for the same SNPs obtained from 3008 neurologically normal control individuals. The cases consisted of 963 North American sporadic ALS patients, 631 Italian sporadic ALS patients and 566 German individuals diagnosed with sporadic ALS. Population control data was obtained from: the Cancer Genetic Markers of Susceptibility study (CGEMS; <em>n</em> = 2243 North American control samples) (<a href="#DDP059C30" rid="DDP059C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747894">30</a>); European Network for Genetic-Epidemiological Studies (HYPERGENES, <em>n</em> = 275 Italian control samples) and the Cooperative Health Research in the Region of Augsburg study (KORA; <em>n</em> = 490 German control samples) (<a href="#DDP059C31" rid="DDP059C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747900">31</a>). These studies were approved by the appropriate institutional review boards.</p><p id="__p27">After applying quality control filters to the replication data, 6758 SNPs were suitable for analysis in the replication cohort consisting of 1752 cases and 2548 controls (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). Association analysis was carried out in two ways: for each population separately, and for the stage 1 and stage 2 replication data combined (<a href="#DDP059C32" rid="DDP059C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747898">32</a>). The combined stage 1 and stage 2 samples consisted of 2289 ALS cases and 4532 controls, which yielded 94.8% power to detect loci with an OR of 1.17 and a MAF of 0.45 under the log-additive model assuming a two-sided <em>α</em> of 0.005 (threshold <em>P</em>-value for selection of SNPs for follow-up). The PLINK toolset (<a href="#DDP059C33" rid="DDP059C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747931">33</a>) was used to test for association using logistic regression, adjusted for age, gender and population. This approach retains power to detect recessive or overdominant alleles at the cost of a small decrease in power relative to the Cochrane-Armitage trend test (<a href="#DDP059C34" rid="DDP059C34" class=" bibr popnode">34</a>) for the detection of alleles with multiplicative effect (<a href="#DDP059C10" rid="DDP059C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747909">10</a>). Bonferroni correction for multiple testing yielded a threshold <em>P</em>-value of 10<sup>−7</sup> (<em>α</em> = 0.05/531 661 autosomal SNPs genotyped in stage 1) (<a href="#DDP059C35" rid="DDP059C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747937">35</a>).</p><p id="__p28" class="p p-last">Detailed descriptions of sample collection methodology and the quality control filters used in this study are available in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Methods</a>.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Additional replication dataset</h3><p id="__p29" class="p p-first-last">Rs2708851 and rs2708909 were genotyped by TaqMan assays (ABI) in an additional dataset of 989 non-Hispanic Caucasian individuals diagnosed with ALS or other MNDs and 327 neurologically normal control samples. The case samples have been collected by the National Registry of Veterans with ALS (<a href="#DDP059C12" rid="DDP059C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747912">12</a>), and the control samples by the GENEVA study (<a href="#DDP059C13" rid="DDP059C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747918">13</a>). Of these 989 patients, 663 (67%) were diagnosed with definite or probable ALS by El Escorial criteria, 79 (8%) had possible ALS, 158 (16%) had progressive muscular atrophy and the remaining 89 (9%) had primary lateral sclerosis or progressive bulbar palsy. These samples had not been genotyped in either the initial genome-wide stage or in the replication stage.</p></div><div id="s4d" class="sec sec-last"><h3 id="s4dtitle">Haplotype block determination and imputation</h3><p id="__p30" class="p p-first-last">Haploview v4.1 was used for assessment of LD (<a href="#DDP059C36" rid="DDP059C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167747933">36</a>). Haplotype blocks were defined using the Haploview v4.1 solid spine of LD method (<em>D</em>′ &gt; 0.8). SNPs selected to fine-map the haplotype block on chromosome 7p12.3 were genotyped by sequence analysis. PCRs were amplified using primers designed using Primer3-web v0.3.0 (<a href="http://frodo.wi.mit.edu">http://frodo.wi.mit.edu</a>) and FastStart PCR MasterMix polymerase (Roche Diagnostics Corp., IN, USA) were sequenced using BigDye terminator v3.1 sequencing chemistry, and run on an ABI3730xl (Applied Biosystems, CA, USA) genetic analyzer as per manufacturer's instructions. The sequences were analyzed with Sequencher software, version 4.2 (Genecodes, VA, USA). MACH version 1.0 software (<a href="http://www.sph.umich.edu/csg/abecasis/MACH/download/">http://www.sph.umich.edu/csg/abecasis/MACH/download/</a>) (<a href="#DDP059C37" rid="DDP059C37" class=" bibr popnode">37</a>) was used to estimate haplotypes, and map crossover and error rates using 100 iterations of the Markov chain Monte Carlo algorithm in subsets of 250 random samples from the stage 1 US and Italian cohorts. These estimates were combined with haplotypes from the HapMap CEU samples to infer genotypes in the region of interest for both cohorts using maximum likelihood estimates of genotypes present in HapMap samples (<a href="http://www.hapmap.org">www.hapmap.org</a>, release 21), but not in the Illumina data. After genotype imputation, the maximum likelihood genotypes for the stage 1 US and Italian cohorts were merged. Analyses were rerun, excluding imputed SNPs with <em>r</em><sup>2</sup> values &lt;0.30 between imputed and known genotypes, and posterior probability averages &lt;0.80 for the most likely genotype imputed.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p31" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp059/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p32" class="p p-first-last">This work was supported by the Intramural Research Program of the National Institute on Aging (project Z01 AG000949-02), the National Institute of Neurological Disorders and Stroke, and the National Institute of Mental Health; the ALS Association; the Packard Center for ALS Research at Johns Hopkins; Istituto Superiore di Sanità (grant number 2005-10 to FL); PF A15 Regione Piemonte (to GR); and the Medical Research Council (to EF and DK). The MONICA/KORA Augsburg studies were financed by the Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany and supported by grants from the German Federal Ministry of Education and Research (BMBF). Part of this work was financed by the German National Genome Research Network (NGFN). The KORA research was supported within the Munich Center of Health Sciences (MC Health) as part of LMUinnovativ. We gratefully acknowledge support for the GENEVA study (‘Genes and Environmental Exposures in Veterans with ALS’) from the National Institutes of Health (grant number ES013244) and the ALS Association (grant number 1230). The National Registry of Veterans with ALS (grant number CSP #500A) and its DNA bank (grant number CSP #478) were supported by the Department of Veterans Affairs Cooperative Studies Program (CSP).</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>[Supplementary Data]</strong>

<div class="sup-box half_rhythm" id="idm140187946150944"><a href="/pmc/articles/PMC2664150/bin/ddp059_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>



</div></div><div id="idm140187946434912" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140187946434912title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p33">We would like to thank Katrina Gwinn, Larry Refolo and Roderick Corriveau of Coriell, The Italian Football League (FIGC), and Augustin Luna. DNA panels from the NINDS Human Genetics Resource Center DNA and Cell Line Repository (<a href="http://ccr.coriell.org/ninds">http://ccr.coriell.org/ninds</a>) were used in this study, as well as clinical data. The submitters that contributed samples are acknowledged in detailed descriptions of each panel: NDPT002, NDPT006, NDPT008, NDPT011 to NDPT013, NDPT019 to NDPT030. We also thank the ALS Research Group and the patients who submitted their samples to the NINDS Repository and to other investigators.</p><p id="__p34"><em>Conflict of Interest statement</em>. L.B. is Vice-President for Research, ALS Association and J.R. is Medical Director of the Packard Center for ALS Research. Other authors report no conflict of interest.</p></div></div><div id="idm140187945894448" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140187945894448title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDP059C1">1. <span class="citation">Chiò A., Traynor B.J., Lombardo F., Fimognari M., Calvo A., Ghiglione P., Mutani R., Restagno G. Prevalence of SOD1 mutations in the Italian ALS population. <span><span class="ref-journal">Neurology. </span>2008;<span class="ref-vol">70</span>:533–537.</span> [<a href="/pubmed/18268245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Prevalence+of+SOD1+mutations+in+the+Italian+ALS+population&amp;author=A.+Chiò&amp;author=B.J.+Traynor&amp;author=F.+Lombardo&amp;author=M.+Fimognari&amp;author=A.+Calvo&amp;volume=70&amp;publication_year=2008&amp;pages=533-537&amp;pmid=18268245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C2">2. <span class="citation">Valdmanis P.N., Rouleau G.A. Genetics of familial amyotrophic lateral sclerosis. <span><span class="ref-journal">Neurology. </span>2008;<span class="ref-vol">70</span>:144–152.</span> [<a href="/pubmed/18180444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Genetics+of+familial+amyotrophic+lateral+sclerosis&amp;author=P.N.+Valdmanis&amp;author=G.A.+Rouleau&amp;volume=70&amp;publication_year=2008&amp;pages=144-152&amp;pmid=18180444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C3">3. <span class="citation">Graham A.J., Macdonald A.M., Hawkes C.H. British motor neuron disease twin study. <span><span class="ref-journal">J. Neurol. Neurosurg. Psychiatry. </span>1997;<span class="ref-vol">62</span>:562–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1074137/">PMC free article</a>]</span> [<a href="/pubmed/9219739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurol.+Neurosurg.+Psychiatry&amp;title=British+motor+neuron+disease+twin+study&amp;author=A.J.+Graham&amp;author=A.M.+Macdonald&amp;author=C.H.+Hawkes&amp;volume=62&amp;publication_year=1997&amp;pages=562-569&amp;pmid=9219739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C4">4. <span class="citation">Majoor-Krakauer D., Ottman R., Johnson W.G., Rowland L.P. Familial aggregation of amyotrophic lateral sclerosis, dementia, and Parkinson's disease: evidence of shared genetic susceptibility. <span><span class="ref-journal">Neurology. </span>1994;<span class="ref-vol">44</span>:1872–1877.</span> [<a href="/pubmed/7936240" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Familial+aggregation+of+amyotrophic+lateral+sclerosis,+dementia,+and+Parkinson's+disease:+evidence+of+shared+genetic+susceptibility&amp;author=D.+Majoor-Krakauer&amp;author=R.+Ottman&amp;author=W.G.+Johnson&amp;author=L.P.+Rowland&amp;volume=44&amp;publication_year=1994&amp;pages=1872-1877&amp;pmid=7936240&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C5">5. <span class="citation">Dunckley T., Huentelman M.J., Craig D.W., Pearson J.V., Szelinger S., Joshipura K., Halperin R.F., Stamper C., Jensen K.R., Letizia D., et al.  Whole-genome analysis of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">N. Engl. J. Med. </span>2007;<span class="ref-vol">357</span>:775–788.</span> [<a href="/pubmed/17671248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Whole-genome+analysis+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=T.+Dunckley&amp;author=M.J.+Huentelman&amp;author=D.W.+Craig&amp;author=J.V.+Pearson&amp;author=S.+Szelinger&amp;volume=357&amp;publication_year=2007&amp;pages=775-788&amp;pmid=17671248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C6">6. <span class="citation">van Es M.A., Van Vught P.W., Blauw H.M., Franke L., Saris C.G., Andersen P.M., Van Den B.L., de Jong S.W., van't S.R., Birve A., et al.  ITPR2 as a susceptibility gene in sporadic amyotrophic lateral sclerosis: a genome-wide association study. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:869–877.</span> [<a href="/pubmed/17827064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol.&amp;title=ITPR2+as+a+susceptibility+gene+in+sporadic+amyotrophic+lateral+sclerosis:+a+genome-wide+association+study&amp;author=M.A.+van+Es&amp;author=P.W.+Van+Vught&amp;author=H.M.+Blauw&amp;author=L.+Franke&amp;author=C.G.+Saris&amp;volume=6&amp;publication_year=2007&amp;pages=869-877&amp;pmid=17827064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C7">7. <span class="citation">Cronin S., Berger S., Ding J., Schymick J.C., Washecka N., Hernandez D.G., Greenway M.J., Bradley D.G., Traynor B.J., Hardiman O. A genome-wide association study of sporadic ALS in a homogenous Irish population. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:768–774.</span> [<a href="/pubmed/18057069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+genome-wide+association+study+of+sporadic+ALS+in+a+homogenous+Irish+population&amp;author=S.+Cronin&amp;author=S.+Berger&amp;author=J.+Ding&amp;author=J.C.+Schymick&amp;author=N.+Washecka&amp;volume=17&amp;publication_year=2008&amp;pages=768-774&amp;pmid=18057069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C8">8. <span class="citation">van Es M.A., Van Vught P.W., Blauw H.M., Franke L., Saris C.G., Van Den B.L., de Jong S.W., de J.V., Baas F., van't Slot R., et al.  Genetic variation in DPP6 is associated with susceptibility to amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:29–31.</span> [<a href="/pubmed/18084291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+variation+in+DPP6+is+associated+with+susceptibility+to+amyotrophic+lateral+sclerosis&amp;author=M.A.+van+Es&amp;author=P.W.+Van+Vught&amp;author=H.M.+Blauw&amp;author=L.+Franke&amp;author=C.G.+Saris&amp;volume=40&amp;publication_year=2008&amp;pages=29-31&amp;pmid=18084291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C9">9. <span class="citation">Chanock S.J., Manolio T., Boehnke M., Boerwinkle E., Hunter D.J., Thomas G., Hirschhorn J.N., Abecasis G., Altshuler D., Bailey-Wilson J.E., et al.  Replicating genotype-phenotype associations. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:655–660.</span> [<a href="/pubmed/17554299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Replicating+genotype-phenotype+associations&amp;author=S.J.+Chanock&amp;author=T.+Manolio&amp;author=M.+Boehnke&amp;author=E.+Boerwinkle&amp;author=D.J.+Hunter&amp;volume=447&amp;publication_year=2007&amp;pages=655-660&amp;pmid=17554299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C10">10. <span class="citation">Thomas G., Jacobs K.B., Yeager M., Kraft P., Wacholder S., Orr N., Yu K., Chatterjee N., Welch R., Hutchinson A., et al.  Multiple loci identified in a genome-wide association study of prostate cancer. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:310–315.</span> [<a href="/pubmed/18264096" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Multiple+loci+identified+in+a+genome-wide+association+study+of+prostate+cancer&amp;author=G.+Thomas&amp;author=K.B.+Jacobs&amp;author=M.+Yeager&amp;author=P.+Kraft&amp;author=S.+Wacholder&amp;volume=40&amp;publication_year=2008&amp;pages=310-315&amp;pmid=18264096&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C11">11. <span class="citation">Frazer K.A., Ballinger D.G., Cox D.R., Hinds D.A., Stuve L.L., Gibbs R.A., Belmont J.W., Boudreau A., Hardenbol P., Leal S.M., et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=K.A.+Frazer&amp;author=D.G.+Ballinger&amp;author=D.R.+Cox&amp;author=D.A.+Hinds&amp;author=L.L.+Stuve&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C12">12. <span class="citation">Allen K.D., Kasarskis E.J., Bedlack R.S., Rozear M.P., Morgenlander J.C., Sabet A., Sams L., Lindquist J.H., Harrelson M.L., Coffman C.J., et al.  The National Registry of Veterans with amyotrophic lateral sclerosis. <span><span class="ref-journal">Neuroepidemiology. </span>2008;<span class="ref-vol">30</span>:180–190.</span> [<a href="/pubmed/18421218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroepidemiology&amp;title=The+National+Registry+of+Veterans+with+amyotrophic+lateral+sclerosis&amp;author=K.D.+Allen&amp;author=E.J.+Kasarskis&amp;author=R.S.+Bedlack&amp;author=M.P.+Rozear&amp;author=J.C.+Morgenlander&amp;volume=30&amp;publication_year=2008&amp;pages=180-190&amp;pmid=18421218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C13">13. <span class="citation">Schmidt S., Allen K.D., Loiacono V.T., Norman B., Stanwyck C.L., Nord K.M., Williams C.D., Kasarskis E.J., Kamel F., McGuire V., et al.  Genes and environmental exposures in veterans with amyotrophic lateral sclerosis: the GENEVA study. Rationale, study design and demographic characteristics. <span><span class="ref-journal">Neuroepidemiology. </span>2008;<span class="ref-vol">30</span>:191–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2645711/">PMC free article</a>]</span> [<a href="/pubmed/18421219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroepidemiology&amp;title=Genes+and+environmental+exposures+in+veterans+with+amyotrophic+lateral+sclerosis:+the+GENEVA+study.+Rationale,+study+design+and+demographic+characteristics&amp;author=S.+Schmidt&amp;author=K.D.+Allen&amp;author=V.T.+Loiacono&amp;author=B.+Norman&amp;author=C.L.+Stanwyck&amp;volume=30&amp;publication_year=2008&amp;pages=191-204&amp;pmid=18421219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C14">14. <span class="citation">Schmidt S., Allen K.D., Rimmler J., Loiacono V., Stanwyck C., Williams C., Munger H., Hauser M., Oddone E. Association of sporadic ALS with candidate genes and environmental risk factors: The Genes and Environmental Exposures in Veterans with ALS (GENEVA) Study. Abstract presented at 58th Annual Meeting, ASHG.2008.  <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Schmidt+S.+Allen+K.D.+Rimmler+J.+Loiacono+V.+Stanwyck+C.+Williams+C.+Munger+H.+Hauser+M.+Oddone+E.+Association+of+sporadic+ALS+with+candidate+genes+and+environmental+risk+factors:+The+Genes+and+Environmental+Exposures+in+Veterans+with+ALS+(GENEVA)+Study+2008+Abstract+presented+at+58th+Annual+Meeting,+ASHG+" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C15">15. <span class="citation">Hafler D.A., Compston A., Sawcer S., Lander E.S., Daly M.J., De Jager P.L., de Bakker P.I., Gabriel S.B., Mirel D.B., Ivinson A.J., et al.  Risk alleles for multiple sclerosis identified by a genomewide study. <span><span class="ref-journal">N. Engl. J. Med. </span>2007;<span class="ref-vol">357</span>:851–862.</span> [<a href="/pubmed/17660530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study&amp;author=D.A.+Hafler&amp;author=A.+Compston&amp;author=S.+Sawcer&amp;author=E.S.+Lander&amp;author=M.J.+Daly&amp;volume=357&amp;publication_year=2007&amp;pages=851-862&amp;pmid=17660530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C16">16. <span class="citation">Pasinelli P., Brown R.H. Molecular biology of amyotrophic lateral sclerosis: insights from genetics. <span><span class="ref-journal">Nat. Rev. Neurosci. </span>2006;<span class="ref-vol">7</span>:710–723.</span> [<a href="/pubmed/16924260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Neurosci.&amp;title=Molecular+biology+of+amyotrophic+lateral+sclerosis:+insights+from+genetics&amp;author=P.+Pasinelli&amp;author=R.H.+Brown&amp;volume=7&amp;publication_year=2006&amp;pages=710-723&amp;pmid=16924260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C17">17. <span class="citation">Schymick J.C., Talbot K., Traynor B.J. Genetics of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Hum. Mol. Genet. </span>2007;<span class="ref-vol">16</span>(Spec No. 2):R233–R242.</span> [<a href="/pubmed/17911166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genetics+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=J.C.+Schymick&amp;author=K.+Talbot&amp;author=B.J.+Traynor&amp;volume=16&amp;issue=Spec+No.+2&amp;publication_year=2007&amp;pages=R233-R242&amp;pmid=17911166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C18">18. <span class="citation">Greenway M.J., Andersen P.M., Russ C., Ennis S., Cashman S., Donaghy C., Patterson V., Swingler R., Kieran D., Prehn J., et al.  ANG mutations segregate with familial and ‘sporadic’ amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:411–413.</span> [<a href="/pubmed/16501576" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=ANG+mutations+segregate+with+familial+and+‘sporadic’+amyotrophic+lateral+sclerosis&amp;author=M.J.+Greenway&amp;author=P.M.+Andersen&amp;author=C.+Russ&amp;author=S.+Ennis&amp;author=S.+Cashman&amp;volume=38&amp;publication_year=2006&amp;pages=411-413&amp;pmid=16501576&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C19">19. <span class="citation">Lambrechts D., Storkebaum E., Morimoto M., Del Favero J., Desmet F., Marklund S.L., Wyns S., Thijs V., Andersson J., van M.I., et al.  VEGF is a modifier of amyotrophic lateral sclerosis in mice and humans and protects motoneurons against ischemic death. <span><span class="ref-journal">Nat. Genet. </span>2003;<span class="ref-vol">34</span>:383–394.</span> [<a href="/pubmed/12847526" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=VEGF+is+a+modifier+of+amyotrophic+lateral+sclerosis+in+mice+and+humans+and+protects+motoneurons+against+ischemic+death&amp;author=D.+Lambrechts&amp;author=E.+Storkebaum&amp;author=M.+Morimoto&amp;author=J.+Del+Favero&amp;author=F.+Desmet&amp;volume=34&amp;publication_year=2003&amp;pages=383-394&amp;pmid=12847526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C20">20. <span class="citation">Wang W.Y., Barratt B.J., Clayton D.G., Todd J.A. Genome-wide association studies: theoretical and practical concerns. <span><span class="ref-journal">Nat. Rev. Genet. </span>2005;<span class="ref-vol">6</span>:109–118.</span> [<a href="/pubmed/15716907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genome-wide+association+studies:+theoretical+and+practical+concerns&amp;author=W.Y.+Wang&amp;author=B.J.+Barratt&amp;author=D.G.+Clayton&amp;author=J.A.+Todd&amp;volume=6&amp;publication_year=2005&amp;pages=109-118&amp;pmid=15716907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C21">21. <span class="citation">Raff J.W. The missing (L) UNC? <span><span class="ref-journal">Curr. Biol. </span>1999;<span class="ref-vol">9</span>:R708–R710.</span> [<a href="/pubmed/10508607" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Biol.&amp;title=The+missing+(L)+UNC?&amp;author=J.W.+Raff&amp;volume=9&amp;publication_year=1999&amp;pages=R708-R710&amp;pmid=10508607&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C22">22. <span class="citation">Sandre-Giovannoli A., Chaouch M., Kozlov S., Vallat J.M., Tazir M., Kassouri N., Szepetowski P., Hammadouche T., Vandenberghe A., Stewart C.L., et al.  Homozygous defects in LMNA, encoding lamin A/C nuclear-envelope proteins, cause autosomal recessive axonal neuropathy in human (Charcot-Marie-Tooth disorder type 2) and mouse. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">70</span>:726–736.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384949/">PMC free article</a>]</span> [<a href="/pubmed/11799477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Homozygous+defects+in+LMNA,+encoding+lamin+A/C+nuclear-envelope+proteins,+cause+autosomal+recessive+axonal+neuropathy+in+human+(Charcot-Marie-Tooth+disorder+type+2)+and+mouse&amp;author=A.+Sandre-Giovannoli&amp;author=M.+Chaouch&amp;author=S.+Kozlov&amp;author=J.M.+Vallat&amp;author=M.+Tazir&amp;volume=70&amp;publication_year=2002&amp;pages=726-736&amp;pmid=11799477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C23">23. <span class="citation">Hazan J., Fonknechten N., Mavel D., Paternotte C., Samson D., Artiguenave F., Davoine C.S., Cruaud C., Durr A., Wincker P., et al.  Spastin, a new AAA protein, is altered in the most frequent form of autosomal dominant spastic paraplegia. <span><span class="ref-journal">Nat. Genet. </span>1999;<span class="ref-vol">23</span>:296–303.</span> [<a href="/pubmed/10610178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Spastin,+a+new+AAA+protein,+is+altered+in+the+most+frequent+form+of+autosomal+dominant+spastic+paraplegia&amp;author=J.+Hazan&amp;author=N.+Fonknechten&amp;author=D.+Mavel&amp;author=C.+Paternotte&amp;author=D.+Samson&amp;volume=23&amp;publication_year=1999&amp;pages=296-303&amp;pmid=10610178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C24">24. <span class="citation">Myers A.J., Gibbs J.R., Webster J.A., Rohrer K., Zhao A., Marlowe L., Kaleem M., Leung D., Bryden L., Nath P., et al.  A survey of genetic human cortical gene expression. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1494–1499.</span> [<a href="/pubmed/17982457" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+survey+of+genetic+human+cortical+gene+expression&amp;author=A.J.+Myers&amp;author=J.R.+Gibbs&amp;author=J.A.+Webster&amp;author=K.+Rohrer&amp;author=A.+Zhao&amp;volume=39&amp;publication_year=2007&amp;pages=1494-1499&amp;pmid=17982457&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C25">25. <span class="citation">Schymick J.C., Scholz S.W., Fung H.C., Britton A., Arepalli S., Gibbs J.R., Lombardo F., Matarin M., Kasperaviciute D., Crews C., et al.  Genome-wide genotyping in amyotrophic lateral sclerosis and neurologically normal controls. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:322–328.</span> [<a href="/pubmed/17362836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol.&amp;title=Genome-wide+genotyping+in+amyotrophic+lateral+sclerosis+and+neurologically+normal+controls&amp;author=J.C.+Schymick&amp;author=S.W.+Scholz&amp;author=H.C.+Fung&amp;author=A.+Britton&amp;author=S.+Arepalli&amp;volume=6&amp;publication_year=2007&amp;pages=322-328&amp;pmid=17362836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C26">26. <span class="citation">Garber K. Genetics. The elusive ALS genes. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">319</span>:20.</span> [<a href="/pubmed/18174402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetics.+The+elusive+ALS+genes&amp;author=K.+Garber&amp;volume=319&amp;publication_year=2008&amp;pages=20&amp;pmid=18174402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C27">27. <span class="citation">Ferrucci L., Bandinelli S., Benvenuti E., Di Iorio A., Macchi C., Harris T.B., Guralnik J.M. Subsystems contributing to the decline in ability to walk: bridging the gap between epidemiology and geriatric practice in the InCHIANTI study. <span><span class="ref-journal">J. Am. Geriatr. Soc. </span>2000;<span class="ref-vol">48</span>:1618–1625.</span> [<a href="/pubmed/11129752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Geriatr.+Soc.&amp;title=Subsystems+contributing+to+the+decline+in+ability+to+walk:+bridging+the+gap+between+epidemiology+and+geriatric+practice+in+the+InCHIANTI+study&amp;author=L.+Ferrucci&amp;author=S.+Bandinelli&amp;author=E.+Benvenuti&amp;author=A.+Di+Iorio&amp;author=C.+Macchi&amp;volume=48&amp;publication_year=2000&amp;pages=1618-1625&amp;pmid=11129752&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C28">28. <span class="citation">Melzer D., Perry J.R., Hernandez D., Corsi A.M., Stevens K., Rafferty I., Lauretani F., Murray A., Gibbs J.R., Paolisso G., et al.  A genome-wide association study identifies protein quantitative trait loci (pQTLs) <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2362067/">PMC free article</a>]</span> [<a href="/pubmed/18464913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+genome-wide+association+study+identifies+protein+quantitative+trait+loci+(pQTLs)&amp;author=D.+Melzer&amp;author=J.R.+Perry&amp;author=D.+Hernandez&amp;author=A.M.+Corsi&amp;author=K.+Stevens&amp;volume=4&amp;publication_year=2008&amp;pages=e1000072&amp;pmid=18464913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C29">29. <span class="citation">Brooks B.R., Miller R.G., Swash M., Munsat T.L. El Escorial revisited: revised criteria for the diagnosis of amyotrophic lateral sclerosis. <span><span class="ref-journal">Amyotroph. Lateral Scler. Other Motor Neuron Disord. </span>2000;<span class="ref-vol">1</span>:293–299.</span> [<a href="/pubmed/11464847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Amyotroph.+Lateral+Scler.+Other+Motor+Neuron+Disord.&amp;title=El+Escorial+revisited:+revised+criteria+for+the+diagnosis+of+amyotrophic+lateral+sclerosis&amp;author=B.R.+Brooks&amp;author=R.G.+Miller&amp;author=M.+Swash&amp;author=T.L.+Munsat&amp;volume=1&amp;publication_year=2000&amp;pages=293-299&amp;pmid=11464847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C30">30. <span class="citation">Yeager M., Orr N., Hayes R.B., Jacobs K.B., Kraft P., Wacholder S., Minichiello M.J., Fearnhead P., Yu K., Chatterjee N., et al.  Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:645–649.</span> [<a href="/pubmed/17401363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+of+prostate+cancer+identifies+a+second+risk+locus+at+8q24&amp;author=M.+Yeager&amp;author=N.+Orr&amp;author=R.B.+Hayes&amp;author=K.B.+Jacobs&amp;author=P.+Kraft&amp;volume=39&amp;publication_year=2007&amp;pages=645-649&amp;pmid=17401363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C31">31. <span class="citation">Wichmann H.E., Gieger C., Illig T. KORA-gen – resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>(suppl. 1):S26–S30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen+–+resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=H.E.+Wichmann&amp;author=C.+Gieger&amp;author=T.+Illig&amp;volume=67&amp;issue=suppl.+1&amp;publication_year=2005&amp;pages=S26-S30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C32">32. <span class="citation">Skol A.D., Scott L.J., Abecasis G.R., Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:209–213.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=A.D.+Skol&amp;author=L.J.+Scott&amp;author=G.R.+Abecasis&amp;author=M.+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C33">33. <span class="citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C34">34. <span class="citation">Margolin B.H.  Test for trend in proportions. In: Klotz S., Johnson N.L., editors. <span class="ref-journal">Encyclopedia of Statistical Sciences.</span> New York: John Wiley &amp; Sons, Inc.; 1988. pp. 334–336. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Encyclopedia+of+Statistical+Sciences&amp;author=B.H.+Margolin&amp;publication_year=1988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C35">35. <span class="citation">Rice T.K., Schork N.J., Rao D.C. Methods for handling multiple testing. <span><span class="ref-journal">Adv. Genet. </span>2008;<span class="ref-vol">60</span>:293–308.</span> [<a href="/pubmed/18358325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Genet.&amp;title=Methods+for+handling+multiple+testing&amp;author=T.K.+Rice&amp;author=N.J.+Schork&amp;author=D.C.+Rao&amp;volume=60&amp;publication_year=2008&amp;pages=293-308&amp;pmid=18358325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C36">36. <span class="citation">Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=J.C.+Barrett&amp;author=B.+Fry&amp;author=J.+Maller&amp;author=M.J.+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP059C37">37. <span class="citation">Li Y., Abecasis G.R. Rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2664150&amp;issue-id=177678&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140187943101984title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140187946434912title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140187945894448title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2664150/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2664150/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2664150/pdf/ddp059.pdf">PDF (285K)</a></li> | <li><a href="#" data-citationid="PMC2664150" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2664150%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2664150%2F&amp;text=A%20two-stage%20genome-wide%20association%20study%20of%20sporadic%20amyotrophic%20lateral%20sclerosis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2664150%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19193627" data-aiid="2664150" data-aid="2664150" data-iid="177678" data-domainid="740" data-domain="hmg" data-accid="PMC2664150" data-md5="0d5e275c536479f8a0772fb77768f2ca"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T11:17:21-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89B42ADD40EF810000000005AC032D&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2664150%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2664150/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>