<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pnas"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2678639/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association and meta-analysis of bipolar disorder in individuals of European ancestry</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2678639/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="Genome-wide association and meta-analysis of bipolar disorder in individuals of European ancestry"><meta name="citation_authors" content="Laura J. Scott, Pierandrea Muglia, Xiangyang Q. Kong, Weihua Guan, Matthew Flickinger, Ruchi Upmanyu, Federica Tozzi, Jun Z. Li, Margit Burmeister, Devin Absher, Robert C. Thompson, Clyde Francks, Fan Meng, Athos Antoniades, Audrey M. Southwick, Alan F. Schatzberg, William E. Bunney, Jack D. Barchas, Edward G. Jones, Richard Day, Keith Matthews, Peter McGuffin, John S. Strauss, James L. Kennedy, Lefkos Middleton, Allen D. Roses, Stanley J. Watson, John B. Vincent, Richard M. Myers, Ann E. Farmer, Huda Akil, Daniel K. Burns, Michael Boehnke"><meta name="citation_date" content="5 May 2009"><meta name="citation_issue" content="18"><meta name="citation_volume" content="106"><meta name="citation_firstpage" content="7501"><meta name="citation_doi" content="10.1073/pnas.0813386106"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2678639/?report=abstract"><meta name="citation_pmid" content="19416921"><meta name="DC.Title" content="Genome-wide association and meta-analysis of bipolar disorder in individuals of European ancestry"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="Laura J. Scott"><meta name="DC.Contributor" content="Pierandrea Muglia"><meta name="DC.Contributor" content="Xiangyang Q. Kong"><meta name="DC.Contributor" content="Weihua Guan"><meta name="DC.Contributor" content="Matthew Flickinger"><meta name="DC.Contributor" content="Ruchi Upmanyu"><meta name="DC.Contributor" content="Federica Tozzi"><meta name="DC.Contributor" content="Jun Z. Li"><meta name="DC.Contributor" content="Margit Burmeister"><meta name="DC.Contributor" content="Devin Absher"><meta name="DC.Contributor" content="Robert C. Thompson"><meta name="DC.Contributor" content="Clyde Francks"><meta name="DC.Contributor" content="Fan Meng"><meta name="DC.Contributor" content="Athos Antoniades"><meta name="DC.Contributor" content="Audrey M. Southwick"><meta name="DC.Contributor" content="Alan F. Schatzberg"><meta name="DC.Contributor" content="William E. Bunney"><meta name="DC.Contributor" content="Jack D. Barchas"><meta name="DC.Contributor" content="Edward G. Jones"><meta name="DC.Contributor" content="Richard Day"><meta name="DC.Contributor" content="Keith Matthews"><meta name="DC.Contributor" content="Peter McGuffin"><meta name="DC.Contributor" content="John S. Strauss"><meta name="DC.Contributor" content="James L. Kennedy"><meta name="DC.Contributor" content="Lefkos Middleton"><meta name="DC.Contributor" content="Allen D. Roses"><meta name="DC.Contributor" content="Stanley J. Watson"><meta name="DC.Contributor" content="John B. Vincent"><meta name="DC.Contributor" content="Richard M. Myers"><meta name="DC.Contributor" content="Ann E. Farmer"><meta name="DC.Contributor" content="Huda Akil"><meta name="DC.Contributor" content="Daniel K. Burns"><meta name="DC.Contributor" content="Michael Boehnke"><meta name="DC.Date" content="2009 May 5"><meta name="DC.Identifier" content="10.1073/pnas.0813386106"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association and meta-analysis of bipolar disorder in individuals of European ancestry"><meta property="og:type" content="article"><meta property="og:description" content="Bipolar disorder (BP) is a disabling and often life-threatening disorder that affects â‰ˆ1% of the population worldwide. To identify genetic variants that increase the risk of BP, we genotyped on the Illumina HumanHap550 Beadchip 2,076 bipolar cases ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2678639/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2678639/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2678639/pdf/zpq7501.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A82A0DD4EF2F100000000071E04C6.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2678639%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/178666/">v.106(18); 2009 May 5</a></li><li class="accid">PMC2678639</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="306,2,479,64" alt="PNAS" title="PNAS" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="59,34,142,50" alt="Info for Authors" title="Info for Authors" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="143,34,200,50" alt="Subscriptions" title="Subscriptions" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="201,34,239,50" alt="About" title="About" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,34,57,50" alt="This Article" title="This Article" href="//dx.doi.org/10.1073/pnas.0813386106" ref="reftype=publisher&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Proc Natl Acad Sci U S A</a></span>. 2009 May 5; 106(18): 7501â€“7506. </span></div><div><span class="fm-vol-iss-date">Published online 2009 Apr 28. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1073%2Fpnas.0813386106" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.0813386106</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2678639</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19416921">19416921</a></div></div></div><div class="fm-doctopic no_margin">Genetics</div></div><h1 class="content-title">Genome-wide association and meta-analysis of bipolar disorder in individuals of European ancestry</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Scott%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570818264384" co-class="co-affbox">Laura J. Scott</a>,<sup>a,</sup><sup>1</sup> <a href="/pubmed/?term=Muglia%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570778817120" co-class="co-affbox">Pierandrea Muglia</a>,<sup>b,</sup><sup>c,</sup><sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Kong%20XQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782919760" co-class="co-affbox">Xiangyang Q. Kong</a>,<sup>d</sup> <a href="/pubmed/?term=Guan%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570851757600" co-class="co-affbox">Weihua Guan</a>,<sup>a</sup> <a href="/pubmed/?term=Flickinger%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570850287024" co-class="co-affbox">Matthew Flickinger</a>,<sup>a</sup> <a href="/pubmed/?term=Upmanyu%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570777900720" co-class="co-affbox">Ruchi Upmanyu</a>,<sup>e</sup> <a href="/pubmed/?term=Tozzi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570779434688" co-class="co-affbox">Federica Tozzi</a>,<sup>b</sup> <a href="/pubmed/?term=Li%20JZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570784406848" co-class="co-affbox">Jun Z. Li</a>,<sup>f,</sup><sup>3</sup> <a href="/pubmed/?term=Burmeister%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570778713184" co-class="co-affbox">Margit Burmeister</a>,<sup>g,</sup><sup>h</sup> <a href="/pubmed/?term=Absher%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570776931952" co-class="co-affbox">Devin Absher</a>,<sup>f,</sup><sup>4</sup> <a href="/pubmed/?term=Thompson%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570776928736" co-class="co-affbox">Robert C. Thompson</a>,<sup>h</sup> <a href="/pubmed/?term=Francks%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570816863392" co-class="co-affbox">Clyde Francks</a>,<sup>b</sup> <a href="/pubmed/?term=Meng%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570816861040" co-class="co-affbox">Fan Meng</a>,<sup>h</sup> <a href="/pubmed/?term=Antoniades%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570816858688" co-class="co-affbox">Athos Antoniades</a>,<sup>b</sup> <a href="/pubmed/?term=Southwick%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782423968" co-class="co-affbox">Audrey M. Southwick</a>,<sup>f</sup> <a href="/pubmed/?term=Schatzberg%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782421616" co-class="co-affbox">Alan F. Schatzberg</a>,<sup>i</sup> <a href="/pubmed/?term=Bunney%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782419264" co-class="co-affbox">William E. Bunney</a>,<sup>j</sup> <a href="/pubmed/?term=Barchas%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782416912" co-class="co-affbox">Jack D. Barchas</a>,<sup>k</sup> <a href="/pubmed/?term=Jones%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570785105168" co-class="co-affbox">Edward G. Jones</a>,<sup>l,</sup><sup>2</sup> <a href="/pubmed/?term=Day%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570785101952" co-class="co-affbox">Richard Day</a>,<sup>m</sup> <a href="/pubmed/?term=Matthews%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570778079536" co-class="co-affbox">Keith Matthews</a>,<sup>m</sup> <a href="/pubmed/?term=McGuffin%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570778077184" co-class="co-affbox">Peter McGuffin</a>,<sup>n</sup> <a href="/pubmed/?term=Strauss%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570778074832" co-class="co-affbox">John S. Strauss</a>,<sup>c</sup> <a href="/pubmed/?term=Kennedy%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782628704" co-class="co-affbox">James L. Kennedy</a>,<sup>c</sup> <a href="/pubmed/?term=Middleton%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782626464" co-class="co-affbox">Lefkos Middleton</a>,<sup>o</sup> <a href="/pubmed/?term=Roses%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782624112" co-class="co-affbox">Allen D. Roses</a>,<sup>p</sup> <a href="/pubmed/?term=Watson%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782621760" co-class="co-affbox">Stanley J. Watson</a>,<sup>h</sup> <a href="/pubmed/?term=Vincent%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570775956656" co-class="co-affbox">John B. Vincent</a>,<sup>c</sup> <a href="/pubmed/?term=Myers%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570775954304" co-class="co-affbox">Richard M. Myers</a>,<sup>f,</sup><sup>4</sup> <a href="/pubmed/?term=Farmer%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782724912" co-class="co-affbox">Ann E. Farmer</a>,<sup>n</sup> <a href="/pubmed/?term=Akil%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782722560" co-class="co-affbox">Huda Akil</a>,<sup>h</sup> <a href="/pubmed/?term=Burns%20DK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782720208" co-class="co-affbox">Daniel K. Burns</a>,<sup>e</sup> and  <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921" class="affpopup" co-rid="_co_idm140570782757152" co-class="co-affbox">Michael Boehnke</a><sup>a,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140570818264384"><h3 class="no_margin">Laura J. Scott</h3><p><sup>a</sup>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, </p><div>Find articles by <a href="/pubmed/?term=Scott%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Laura J. Scott</a></div></div><div id="_co_idm140570778817120"><h3 class="no_margin">Pierandrea Muglia</h3><p><sup>b</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, 37135 Verona, Italy; </p><p><sup>c</sup>Centre for Addiction and Mental Health, Department of Psychiatry, University of Toronto, ON, Canada M5T 1W7; </p><div>Find articles by <a href="/pubmed/?term=Muglia%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Pierandrea Muglia</a></div></div><div id="_co_idm140570782919760"><h3 class="no_margin">Xiangyang Q. Kong</h3><p><sup>d</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, Research Triangle Park, NC 27709; </p><div>Find articles by <a href="/pubmed/?term=Kong%20XQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Xiangyang Q. Kong</a></div></div><div id="_co_idm140570851757600"><h3 class="no_margin">Weihua Guan</h3><p><sup>a</sup>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, </p><div>Find articles by <a href="/pubmed/?term=Guan%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Weihua Guan</a></div></div><div id="_co_idm140570850287024"><h3 class="no_margin">Matthew Flickinger</h3><p><sup>a</sup>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, </p><div>Find articles by <a href="/pubmed/?term=Flickinger%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Matthew Flickinger</a></div></div><div id="_co_idm140570777900720"><h3 class="no_margin">Ruchi Upmanyu</h3><p><sup>e</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, Harlow CM19 5AW, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Upmanyu%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Ruchi Upmanyu</a></div></div><div id="_co_idm140570779434688"><h3 class="no_margin">Federica Tozzi</h3><p><sup>b</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, 37135 Verona, Italy; </p><div>Find articles by <a href="/pubmed/?term=Tozzi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Federica Tozzi</a></div></div><div id="_co_idm140570784406848"><h3 class="no_margin">Jun Z. Li</h3><p><sup>f</sup>Department of Genetics, Stanford Human Genome Center, Stanford University, Stanford, CA 94305; </p><div>Find articles by <a href="/pubmed/?term=Li%20JZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Jun Z. Li</a></div></div><div id="_co_idm140570778713184"><h3 class="no_margin">Margit Burmeister</h3><p><sup>g</sup>Departments of Human Genetics and Psychiatry, and </p><p><sup>h</sup>Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann Arbor, MI 48109; </p><div>Find articles by <a href="/pubmed/?term=Burmeister%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Margit Burmeister</a></div></div><div id="_co_idm140570776931952"><h3 class="no_margin">Devin Absher</h3><p><sup>f</sup>Department of Genetics, Stanford Human Genome Center, Stanford University, Stanford, CA 94305; </p><div>Find articles by <a href="/pubmed/?term=Absher%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Devin Absher</a></div></div><div id="_co_idm140570776928736"><h3 class="no_margin">Robert C. Thompson</h3><p><sup>h</sup>Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann Arbor, MI 48109; </p><div>Find articles by <a href="/pubmed/?term=Thompson%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Robert C. Thompson</a></div></div><div id="_co_idm140570816863392"><h3 class="no_margin">Clyde Francks</h3><p><sup>b</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, 37135 Verona, Italy; </p><div>Find articles by <a href="/pubmed/?term=Francks%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Clyde Francks</a></div></div><div id="_co_idm140570816861040"><h3 class="no_margin">Fan Meng</h3><p><sup>h</sup>Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann Arbor, MI 48109; </p><div>Find articles by <a href="/pubmed/?term=Meng%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Fan Meng</a></div></div><div id="_co_idm140570816858688"><h3 class="no_margin">Athos Antoniades</h3><p><sup>b</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, 37135 Verona, Italy; </p><div>Find articles by <a href="/pubmed/?term=Antoniades%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Athos Antoniades</a></div></div><div id="_co_idm140570782423968"><h3 class="no_margin">Audrey M. Southwick</h3><p><sup>f</sup>Department of Genetics, Stanford Human Genome Center, Stanford University, Stanford, CA 94305; </p><div>Find articles by <a href="/pubmed/?term=Southwick%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Audrey M. Southwick</a></div></div><div id="_co_idm140570782421616"><h3 class="no_margin">Alan F. Schatzberg</h3><p><sup>i</sup>Psychiatry and Behavioral Science, Stanford University School of Medicine, Palo Alto, CA 94305; </p><div>Find articles by <a href="/pubmed/?term=Schatzberg%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Alan F. Schatzberg</a></div></div><div id="_co_idm140570782419264"><h3 class="no_margin">William E. Bunney</h3><p><sup>j</sup>Department of Psychiatry and Human Behavior, University of California, Irvine, CA 92697; </p><div>Find articles by <a href="/pubmed/?term=Bunney%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">William E. Bunney</a></div></div><div id="_co_idm140570782416912"><h3 class="no_margin">Jack D. Barchas</h3><p><sup>k</sup>Department of Psychiatry, Weill Medical College, Cornell University, New York, NY 10065; </p><div>Find articles by <a href="/pubmed/?term=Barchas%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Jack D. Barchas</a></div></div><div id="_co_idm140570785105168"><h3 class="no_margin">Edward G. Jones</h3><p><sup>l</sup>Center for Neuroscience, University of California, Davis, CA 95618; </p><div>Find articles by <a href="/pubmed/?term=Jones%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Edward G. Jones</a></div></div><div id="_co_idm140570785101952"><h3 class="no_margin">Richard Day</h3><p><sup>m</sup>Centre for Neuroscience, Division of Medical Sciences, University of Dundee, Dundee DD1 9SY, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Day%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Richard Day</a></div></div><div id="_co_idm140570778079536"><h3 class="no_margin">Keith Matthews</h3><p><sup>m</sup>Centre for Neuroscience, Division of Medical Sciences, University of Dundee, Dundee DD1 9SY, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Matthews%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Keith Matthews</a></div></div><div id="_co_idm140570778077184"><h3 class="no_margin">Peter McGuffin</h3><p><sup>n</sup>Medical Research Council Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King's College London, DeCrespigny Park, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=McGuffin%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Peter McGuffin</a></div></div><div id="_co_idm140570778074832"><h3 class="no_margin">John S. Strauss</h3><p><sup>c</sup>Centre for Addiction and Mental Health, Department of Psychiatry, University of Toronto, ON, Canada M5T 1W7; </p><div>Find articles by <a href="/pubmed/?term=Strauss%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">John S. Strauss</a></div></div><div id="_co_idm140570782628704"><h3 class="no_margin">James L. Kennedy</h3><p><sup>c</sup>Centre for Addiction and Mental Health, Department of Psychiatry, University of Toronto, ON, Canada M5T 1W7; </p><div>Find articles by <a href="/pubmed/?term=Kennedy%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">James L. Kennedy</a></div></div><div id="_co_idm140570782626464"><h3 class="no_margin">Lefkos Middleton</h3><p><sup>o</sup>Division of Neurosciences and Mental Health, Imperial College, London SW7 2AZ, United Kingdom; and </p><div>Find articles by <a href="/pubmed/?term=Middleton%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Lefkos Middleton</a></div></div><div id="_co_idm140570782624112"><h3 class="no_margin">Allen D. Roses</h3><p><sup>p</sup>Duke Drug Discovery Institute, Duke University Medical Center, Durham, NC 27708</p><div>Find articles by <a href="/pubmed/?term=Roses%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Allen D. Roses</a></div></div><div id="_co_idm140570782621760"><h3 class="no_margin">Stanley J. Watson</h3><p><sup>h</sup>Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann Arbor, MI 48109; </p><div>Find articles by <a href="/pubmed/?term=Watson%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Stanley J. Watson</a></div></div><div id="_co_idm140570775956656"><h3 class="no_margin">John B. Vincent</h3><p><sup>c</sup>Centre for Addiction and Mental Health, Department of Psychiatry, University of Toronto, ON, Canada M5T 1W7; </p><div>Find articles by <a href="/pubmed/?term=Vincent%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">John B. Vincent</a></div></div><div id="_co_idm140570775954304"><h3 class="no_margin">Richard M. Myers</h3><p><sup>f</sup>Department of Genetics, Stanford Human Genome Center, Stanford University, Stanford, CA 94305; </p><div>Find articles by <a href="/pubmed/?term=Myers%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Richard M. Myers</a></div></div><div id="_co_idm140570782724912"><h3 class="no_margin">Ann E. Farmer</h3><p><sup>n</sup>Medical Research Council Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King's College London, DeCrespigny Park, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Farmer%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Ann E. Farmer</a></div></div><div id="_co_idm140570782722560"><h3 class="no_margin">Huda Akil</h3><p><sup>h</sup>Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann Arbor, MI 48109; </p><div>Find articles by <a href="/pubmed/?term=Akil%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Huda Akil</a></div></div><div id="_co_idm140570782720208"><h3 class="no_margin">Daniel K. Burns</h3><p><sup>e</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, Harlow CM19 5AW, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Burns%20DK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Daniel K. Burns</a></div></div><div id="_co_idm140570782757152"><h3 class="no_margin">Michael Boehnke</h3><p><sup>a</sup>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, </p><div>Find articles by <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19416921">Michael Boehnke</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140570780936976_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140570780936976_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140570780936976_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140570780936976_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, </div><div class="fm-affl" id="aff2" lang="en"><sup>g</sup>Departments of Human Genetics and Psychiatry, and </div><div class="fm-affl" id="aff3" lang="en"><sup>h</sup>Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann Arbor, MI 48109; </div><div class="fm-affl" id="aff4" lang="en"><sup>b</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, 37135 Verona, Italy; </div><div class="fm-affl" id="aff5" lang="en"><sup>c</sup>Centre for Addiction and Mental Health, Department of Psychiatry, University of Toronto, ON, Canada M5T 1W7; </div><div class="fm-affl" id="aff6" lang="en"><sup>d</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, Research Triangle Park, NC 27709; </div><div class="fm-affl" id="aff7" lang="en"><sup>e</sup>Genetics Division, Drug Discovery, GlaxoSmithKline Research and Development, Harlow CM19 5AW, United Kingdom; </div><div class="fm-affl" id="aff8" lang="en"><sup>f</sup>Department of Genetics, Stanford Human Genome Center, Stanford University, Stanford, CA 94305; </div><div class="fm-affl" id="aff9" lang="en"><sup>i</sup>Psychiatry and Behavioral Science, Stanford University School of Medicine, Palo Alto, CA 94305; </div><div class="fm-affl" id="aff10" lang="en"><sup>j</sup>Department of Psychiatry and Human Behavior, University of California, Irvine, CA 92697; </div><div class="fm-affl" id="aff11" lang="en"><sup>k</sup>Department of Psychiatry, Weill Medical College, Cornell University, New York, NY 10065; </div><div class="fm-affl" id="aff12" lang="en"><sup>l</sup>Center for Neuroscience, University of California, Davis, CA 95618; </div><div class="fm-affl" id="aff13" lang="en"><sup>m</sup>Centre for Neuroscience, Division of Medical Sciences, University of Dundee, Dundee DD1 9SY, United Kingdom; </div><div class="fm-affl" id="aff14" lang="en"><sup>n</sup>Medical Research Council Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King's College London, DeCrespigny Park, London SE5 8AF, United Kingdom; </div><div class="fm-affl" id="aff15" lang="en"><sup>o</sup>Division of Neurosciences and Mental Health, Imperial College, London SW7 2AZ, United Kingdom; and </div><div class="fm-affl" id="aff16" lang="en"><sup>p</sup>Duke Drug Discovery Institute, Duke University Medical Center, Durham, NC 27708</div><div id="cor1"><sup>2</sup>To whom correspondence should be addressed. E-mail: <a href="mailto:dev@null" data-email="moc.ksg@ailgum.2.aerdnareip" class="oemail">moc.ksg@ailgum.2.aerdnareip</a>, <a href="mailto:dev@null" data-email="ude.sivadcu@senoje" class="oemail">ude.sivadcu@senoje</a>, or <a href="mailto:dev@null" data-email="ude.hcimu@eknheob" class="oemail">ude.hcimu@eknheob</a></div><div><p id="__p1" class="p p-first-last">Contributed by Edward G. Jones, December 31, 2008</p>.</div><div><p id="__p2" class="p p-first-last">Author contributions: L.J.S., J.Z.L., M. Burmeister, A.F.S., W.E.B., J.D.B., E.G.J., P. McGuffin, J.L.K., L.M., A.D.R., S.J.W., J.B.V., R.M.M., A.E.F., H.A., D.K.B., and M. Boehnke designed research; P. Muglia, J.Z.L., D.A., R.C.T., C.F., F.M., A.A., A.M.S., R.D., K.M., and J.S.S. performed research; L.J.S., X.Q.K., W.G., M.F., R.U., F.T., J.Z.L., and D.A. analyzed data; and L.J.S., P. Muglia, X.Q.K., W.G., R.U., F.T., M. Burmeister, C.F., H.A., and M. Boehnke wrote the paper.</p></div><div id="FN1"><sup>1</sup>L.J.S. and P.M. contributed equally to this work</div><div id="FN2"><sup>3</sup>Present address: Department of Human Genetics, University of Michigan, Ann Arbor, MI 48109.</div><div id="FN3"><sup>4</sup>Present address: HudsonAlpha Institute for Biotechnology, Huntsville, AL 35806.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140570780936976_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 Dec 3</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140570780936976_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> Â© 2009 by The National Academy of Sciences of the USA</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2678639/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supporting Information
            </div>
            
            <div><a href="/pmc/articles/PMC2678639/bin/supp_106_18_7501__index.html" data-ga-action="click_feat_suppl">supp_106_18_7501__index.html</a><span style="color:gray"> (491 bytes)</span></div><div class="small guid">GUID:&nbsp;7D50694F-FCEE-4793-A835-4EB6B913384D</div>
            <div><a href="/pmc/articles/PMC2678639/bin/0813386106_0813386106SI.pdf" data-ga-action="click_feat_suppl">0813386106_0813386106SI.pdf</a><span style="color:gray"> (2.7M)</span></div><div class="small guid">GUID:&nbsp;020D7ABE-D36A-468F-AA36-4A793515B6F2</div>
         </div></dd></dl></div><div id="idm140570782102272" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140570782102272title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p id="__p6" class="p p-first-last">Bipolar disorder (BP) is a disabling and often life-threatening disorder that affects â‰ˆ1% of the population worldwide. To identify genetic variants that increase the risk of BP, we genotyped on the Illumina HumanHap550 Beadchip 2,076 bipolar cases and 1,676 controls of European ancestry from the National Institute of Mental Health Human Genetics Initiative Repository, and the Prechter Repository and samples collected in London, Toronto, and Dundee. We imputed SNP genotypes and tested for SNP-BP association in each sample and then performed meta-analysis across samples. The strongest association <em>P</em> value for this 2-study meta-analysis was 2.4 Ã— 10<sup>âˆ’6</sup>. We next imputed SNP genotypes and tested for SNP-BP association based on the publicly available Affymetrix 500K genotype data from the Wellcome Trust Case Control Consortium for 1,868 BP cases and a reference set of 12,831 individuals. A 3-study meta-analysis of 3,683 nonoverlapping cases and 14,507 extended controls on &gt;2.3 M genotyped and imputed SNPs resulted in 3 chromosomal regions with association <em>P</em> â‰ˆ 10<sup>âˆ’7</sup>: 1p31.1 (no known genes), 3p21 (&gt;25 known genes), and 5q15 (<em>MCTP1</em>). The most strongly associated nonsynonymous SNP rs1042779 (OR = 1.19, <em>P</em> = 1.8 Ã— 10<sup>âˆ’7</sup>) is in the <em>ITIH1</em> gene on chromosome 3, with other strongly associated nonsynonymous SNPs in <em>GNL3</em>, <em>NEK4</em>, and <em>ITIH3</em>. Thus, these chromosomal regions harbor genes implicated in cell cycle, neurogenesis, neuroplasticity, and neurosignaling. In addition, we replicated the reported <em>ANK3</em> association results for SNP rs10994336 in the nonoverlapping GSK sample (OR = 1.37, <em>P</em> = 0.042). Although these results are promising, analysis of additional samples will be required to confirm that variant(s) in these regions influence BP risk.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genetics, genome-wide association study</span></div></div><div id="idm140570778807856" class="tsec sec headless whole_rhythm"><p id="__p7" class="p p-first">Bipolar disorder (BP) is characterized by dramatic mood changes, with individuals experiencing alternating episodes of depression and mania interspersed with periods of normal function. BP is chronic, severely disabling, and life-threatening, with increased risk of suicide and estimated lifetime prevalence of â‰ˆ1% (<a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102690">1</a>).</p><p id="__p8">BP has a substantial genetic component. Monozygotic twin concordance rate estimates range from 45 to 70% and sibling recurrence risk estimates from 5 to 10 (<a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102641">2</a>). Nonetheless, the underlying genetic and neurobiological triggers of BP remain elusive. Numerous linkage and candidate gene studies have sought to identify BP linked regions and associated genes, but no loci have been convincingly identified. Several groups have recently reported results of BP genome-wide association studies (GWAS), using pooled (<a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102647">3</a>) or individually genotyped (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102669">4</a><a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_353930258"></a>â€“<a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102653">6</a>) samples; these studies identified SNPs in <em>CACNA1C</em> (alpha 1C subunit of the L-type voltage-gated calcium channel), <em>ANK3</em> (ankyrin 3), and <em>DGKH</em> (diacylglycerol kinase, eta) as potentially associated with BP.</p><p id="__p9" class="p">To test for SNP-BP association in additional well-characterized samples, we analyzed data for &gt;2.3 million genotyped and imputed SNPs on &gt;3,700 individuals in 2 sample sets: (<em>i</em>) 1,177 bipolar I (BP I) cases and 772 controls from the National Institutes of Mental Health Genetic Initiative Repository and the Prechter Repository [National Institute of Mental Health (NIMH)/Pritzker], and (<em>ii</em>) 899 BP cases and 904 controls from London and Dundee, U.K., and Toronto, Canada whose collection was sponsored by GlaxoSmithKline Research and Development (GSK). We analyzed the NIMH/Pritzker and GSK GWAS samples separately and performed a 2-study meta-analysis. We then imputed and analyzed the publicly available Affymetrix 500K genotype data from the Wellcome Trust Case Control Consortium (WTCCC) (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102662">4</a>) for 1,868 BP cases and an expanded reference set of 12,831 individuals. The expanded reference set comprised a blood donor sample and 6 non-BP disease case groups. After removing from the GSK London sample 261 cases that overlapped with the WTCCC sample, we performed a 3-study meta-analysis of 3,683 cases and 14,507 extended reference set individuals. We identified 3 regions that harbored SNPs with association <em>P</em> â‰ˆ 10<sup>âˆ’7</sup>.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title" style="text-transform: uppercase;">Results</h2><p id="__p10" class="p p-first">For the NIMH/Pritzker GWAS, 1,177 BP I cases (473 sibling pairs and 231 unrelated individuals) from the NIMH and Prechter Repositories and 772 controls from the NIMH Repository (<a href="/pmc/articles/PMC2678639/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) were genotyped on the Illumina HumanHap 550K chip; 512,844 autosomal SNPs passed QC (see Methods) and had minor allele frequency (MAF)â‰¥1%. For the GSK GWAS, 899 BP cases and 904 controls were genotyped (<a href="/pmc/articles/PMC2678639/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>); 512,508 autosomal SNPs passed QC and had MAF â‰¥1%. We carried out genotype imputation, using these data together with estimated haplotypes for the HapMap CEU (Utah residents with ancestry from northern and western Europe) samples, resulting in 2,473,048 (NIMH/Pritzker) or 2,465,069 (GSK) genotyped or imputed autosomal SNPs with MAF â‰¥1%. Previous work has shown good concordance of imputed and experimental genotypes (<a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102665">7</a><a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_249671899"></a>â€“<a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102670">9</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p11">NIMH/Pritzker, GSK (complete and reduced sample), and WTCCC bipolar case and control characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570778455440" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" align="left">Study sample</th><th rowspan="2" colspan="1" align="center"><em>n</em></th><th colspan="2" rowspan="1" align="center">Age at recruitment (yrs)<hr></th><th rowspan="2" colspan="1" align="center">Female, %</th><th rowspan="2" colspan="1" align="center">Diagnosis, % BP I</th></tr><tr><th rowspan="1" colspan="1" align="center">Mean (SD)</th><th rowspan="1" colspan="1" align="center">Range</th></tr></thead><tbody valign="top"><tr><td colspan="6" rowspan="1" align="left">NIMH/Pritzker</td></tr><tr><td rowspan="1" colspan="1" align="left">â€ƒâ€ƒâ€ƒâ€ƒCases</td><td rowspan="1" colspan="1" align="center">1,177</td><td rowspan="1" colspan="1" align="center">42.2 (12.6)</td><td rowspan="1" colspan="1" align="center">14â€“88</td><td rowspan="1" colspan="1" align="center">62.8</td><td rowspan="1" colspan="1" align="center">100</td></tr><tr><td rowspan="1" colspan="1" align="left">â€ƒâ€ƒâ€ƒâ€ƒControls</td><td rowspan="1" colspan="1" align="center">772</td><td rowspan="1" colspan="1" align="center">42.2 (13.4)</td><td rowspan="1" colspan="1" align="center">20â€“69</td><td rowspan="1" colspan="1" align="center">50.3</td><td rowspan="1" colspan="1" align="center">â€”</td></tr><tr><td colspan="6" rowspan="1" align="left">GSK</td></tr><tr><td colspan="6" rowspan="1" align="left">â€ƒâ€ƒâ€ƒâ€ƒCases</td></tr><tr><td rowspan="1" colspan="1" align="left">â€ƒâ€ƒâ€ƒâ€ƒâ€ƒâ€ƒâ€ƒâ€ƒComplete sample</td><td rowspan="1" colspan="1" align="center">899</td><td rowspan="1" colspan="1" align="center">47.1 (12.2)</td><td rowspan="1" colspan="1" align="center">18â€“84</td><td rowspan="1" colspan="1" align="center">64.2</td><td rowspan="1" colspan="1" align="center">90.6</td></tr><tr><td rowspan="1" colspan="1" align="left">â€ƒâ€ƒâ€ƒâ€ƒâ€ƒâ€ƒâ€ƒâ€ƒ<em>Reduced sample</em></td><td rowspan="1" colspan="1" align="center"><em>638</em></td><td rowspan="1" colspan="1" align="center"><em>46.8 (12.3)</em></td><td rowspan="1" colspan="1" align="center"><em>18â€“84</em></td><td rowspan="1" colspan="1" align="center"><em>64.9</em></td><td rowspan="1" colspan="1" align="center"><em>88.6</em></td></tr><tr><td rowspan="1" colspan="1" align="left">â€ƒâ€ƒâ€ƒâ€ƒControls</td><td rowspan="1" colspan="1" align="center">904</td><td rowspan="1" colspan="1" align="center">39.5 (16.3)</td><td rowspan="1" colspan="1" align="center">18â€“89</td><td rowspan="1" colspan="1" align="center">58.6</td><td rowspan="1" colspan="1" align="center">â€”</td></tr><tr><td colspan="6" rowspan="1" align="left">WTCCC</td></tr><tr><td rowspan="1" colspan="1" align="left">â€ƒâ€ƒâ€ƒâ€ƒCases</td><td rowspan="1" colspan="1" align="center">1,868</td><td rowspan="1" colspan="1" align="center">40â€“49 <sup>*</sup></td><td rowspan="1" colspan="1" align="center">&lt;40 to &gt;70</td><td rowspan="1" colspan="1" align="center">63</td><td rowspan="1" colspan="1" align="center">71</td></tr><tr><td rowspan="1" colspan="1" align="left">â€ƒâ€ƒâ€ƒâ€ƒControls</td><td rowspan="1" colspan="1" align="center">12,831</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">51</td><td rowspan="1" colspan="1" align="center">â€”</td></tr></tbody></table></div><div id="largeobj_idm140570778455440" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2678639/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140570850180400"><p id="__p12" class="p p-first-last">GSK reduced sample (italicized): Excluding 261 BP cases also present in WTCCC sample.</p></div><div id="TF1-1"><p id="__p13" class="p p-first-last">*Median age category.</p></div></div></div><p id="__p14">We tested for SNP-BP association under an additive genetic model, using the observed allele count for genotyped SNPs and estimated allele dosage for imputed SNPs. For both studies, we included as covariates principal components (PCs) based on the genotype data to help correct for potential population stratification; for GSK, we also included study site as a covariate. Genomic control values (<a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102674">10</a>) for genotyped and imputed SNPs were 1.03 and 1.03 in the NIMH/Pritzker sample and 1.02 and 1.03 in the GSK sample (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1 <em>Aâ€“D</em></a>). After applying genomic control to each set of results, we combined results between samples, using a fixed effects meta-analysis, with resulting genomic control value 1.01 (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S2<em>A</em></a>). No SNP in either study or in the 2-study meta-analysis attained <em>P</em> &lt; 5 Ã— 10<sup>âˆ’8</sup>, corresponding to genome-wide significance of 0.05 assuming the equivalent of 1 million independent tests (<a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102686">11</a>) (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S3<em>A</em></a>). We identified 2 regions with SNP association <em>P</em> &lt; 10<sup>âˆ’6</sup> (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1A</a>): rs12998006 (<em>P</em> = 7.6 Ã— 10<sup>âˆ’7</sup>, OR = 1.34) on chromosome 2 near <em>CPS1</em> (carbamoyl-phosphate synthetase 1 isoform a) (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF4" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S4<em>A</em></a>) and rs2813164 (<em>P</em> = 8.3 Ã— 10<sup>âˆ’7</sup>, OR = 1.31) on chromosome 1 between <em>NEK7</em> (never in mitosis-related gene 7) and <em>ATP6V1G3</em> (ATPase, H+ transporting, lysosomal, V1 subunit G3) (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF4" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S4<em>B</em></a>).</p><p id="__p15">To increase power to detect BP-associated loci, we obtained genotype data from the WTCCC (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102687">4</a>). These data included 1,868 BP cases and an extended reference set of 12,831 individuals comprised of the National Blood Service (NBS) controls and individuals with coronary artery disease, Crohn's disease, hypertension, rheumatoid arthritis, type 1 diabetes, or type 2 diabetes (<a href="/pmc/articles/PMC2678639/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Given the low population prevalence of BP (1%), use of an unscreened expanded reference set should result in little loss of power to detect BP association. These samples had been genotyped on the Affymetrix 500K chip; 397,653 autosomal SNPs passed WTCCC QC criteria and had MAF â‰¥1%. We carried out imputation as before, resulting in 2,431,899 genotyped or imputed SNPs that passed QC criteria and had MAF â‰¥1%.</p><p id="__p16">To assess whether the non-BP disease cases could reasonably serve as part of an extended reference set for our BP association study, we first compared the non-BP disease cases to the NBS controls. We observed no strong SNP associations except in the HLA region. After removal of genotypes for autoimmune disease cases (Crohn's disease, rheumatoid arthritis, type 1 diabetes) from the analysis of the HLA region, the genomic control value for non-BP cases vs. NBS controls was 1.03.</p><p id="__p17">We tested for SNP-BP association in the WTCCC BP case and extended reference set under an additive model with the genotype-based PCs as covariates. Genomic control values for genotyped and imputed SNPs were both 1.15 (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1 <em>G</em> and <em>H</em></a>). Analysis without PCs resulted in genomic control values of 1.18 and 1.17, suggesting that differences between cases and controls were only partially captured by the PCs, consistent with observations by WTCCC (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102679">4</a>) investigators.</p><p id="__p18">We dropped 261 cases present in both the WTCCC and GSK samples from the GSK sample (<a href="/pmc/articles/PMC2678639/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1 <em>E</em> and <em>F</em></a>), reanalyzed the remaining GSK samples, and applied genomic control to results from each of the 3 studies. We then performed a 3-study meta-analysis of 3,683 cases and 14,507 controls genotyped or imputed for 2,366,197 autosomal SNPs. The meta-analysis genomic control value was 1.04 (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figs. S2<em>B</em></a> and <a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">S3<em>B</em></a>). No SNP in the 3-study meta-analysis reached genome-wide significance at <em>P</em> &lt; 5 Ã— 10<sup>âˆ’8</sup>.</p><p id="__p19">In the 3-study meta-analysis, 53 SNPs from 3 independent regions had <em>P</em> &lt; 10<sup>âˆ’6</sup> [<a href="/pmc/articles/PMC2678639/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>(top regional SNP) and <a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Tables S2</a> and S3]. On chromosome 5q15, we observed strongest association evidence in the genome with SNP rs17418283 (OR = 1.21, <em>P</em> = 1.3 Ã— 10<sup>âˆ’7</sup>) located in an intron of <em>MCTP1</em> (multiple C2 domains, transmembrane 1 isoform) (<a href="/pmc/articles/PMC2678639/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a><em>A</em>). We observed a second signal â‰ˆ400 kb away in the first intron of <em>MCTP1</em> at rs153291 (OR = 1.13, <em>P</em> = 2.7 Ã— 10<sup>âˆ’4</sup>). Inclusion of rs17418283 as a covariate in the analysis did not change the evidence for association for rs153291 (OR = 1.12, <em>P</em> = 2.8 Ã— 10<sup>âˆ’4</sup>) (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF5" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S5</a>). On chromosome 3p21, we observed strongest association evidence (OR = 1.19, <em>P</em> = 1.8 Ã— 10<sup>âˆ’7</sup>) with rs1042779 in a large LD block from 52.2 to 53.2 Mb (<a href="/pmc/articles/PMC2678639/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a><em>B</em>). rs1042779 (Arg595Gln) and the nearby rs678 (Glu585Val) (OR = 1.19, <em>P</em> = 2.5 Ã— 10<sup>âˆ’7</sup>) are nonsynonymous SNPs in <em>ITIH1</em> (inter-alpha trypsin inhibitor, heavy chain 1). 33 SNPs in this region showed strong evidence for association (<em>P</em> &lt; 10<sup>âˆ’6</sup>). On chromosome 1p32.1, we observed strongest association evidence with rs472913 (OR = 1.18, <em>P</em> = 2.0 Ã— 10<sup>âˆ’7</sup>) â‰ˆ500 kb from the closest known gene, <em>NF1A</em> (nuclear factor 1 A-type) (<a href="/pmc/articles/PMC2678639/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a><em>C</em>). In all 3 regions, the large WTCCC case/extended reference set showed the strongest association evidence, but there was no significant evidence of heterogeneity among the 3 studies (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a>). On chromosomes 1 and 3, inclusion of the most strongly associated SNP as a covariate in the analysis diminished the evidence for association to <em>P</em> &gt; 10<sup>âˆ’2</sup> for other SNPs in the region; these conditional analysis results are consistent with chromosome 1 and 3 association signals that reflect 1 BP-predisposing variant, or multiple BP-predisposing variants in high LD (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF5" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S5</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p20">NIMH/Pritzker, GSK (reduced sample), and WTCCC bipolar meta-analysis association results: loci with <em>P</em> &lt; 10<sup>âˆ’6</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570784773168" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="center">Chr</th><th rowspan="2" colspan="1" align="center">Position<sup>*</sup> bp</th><th rowspan="2" colspan="1" align="center">Nearest gene(s)</th><th rowspan="2" colspan="1" align="center">Risk/nonrisk allele</th><th rowspan="2" colspan="1" align="center">Control risk allele freq</th><th colspan="2" rowspan="1" align="center">NIMH/Pritzker<hr></th><th colspan="2" rowspan="1" align="center">GSK<hr></th><th colspan="2" rowspan="1" align="center">WTCCC<hr></th><th colspan="2" rowspan="1" align="center">Meta-analysis<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center">OR 95% CI</th><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center">OR 95% CI</th><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em></th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">rs472913</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">60,807,579</td><td rowspan="1" colspan="1" align="center">â€”</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.50</td><td rowspan="1" colspan="1" align="center"><em>1.12 (0.97â€“1.29)</em></td><td rowspan="1" colspan="1" align="center"><em>0.11</em></td><td rowspan="1" colspan="1" align="center"><em>1.17 (1.00â€“1.36)</em></td><td rowspan="1" colspan="1" align="center"><em>0.051</em></td><td rowspan="1" colspan="1" align="center"><em>1.20 (1.11â€“1.28)</em></td><td rowspan="1" colspan="1" align="center"><em>6.3</em> Ã— <em>10</em><sup>âˆ’7</sup></td><td rowspan="1" colspan="1" align="center">1.18 (1.11â€“1.25)</td><td rowspan="1" colspan="1" align="center">2.0 Ã— 10<sup>âˆ’7</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1042779</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">52,796,051</td><td rowspan="1" colspan="1" align="center"><em>NEK4; ITIH1</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.63</td><td rowspan="1" colspan="1" align="center">1.20 (1.04â€“1.38)</td><td rowspan="1" colspan="1" align="center">0.015</td><td rowspan="1" colspan="1" align="center">1.31 (1.11â€“1.54)</td><td rowspan="1" colspan="1" align="center">0.0012</td><td rowspan="1" colspan="1" align="center"><em>1.16 (1.07â€“1.25)</em></td><td rowspan="1" colspan="1" align="center"><em>0.00012</em></td><td rowspan="1" colspan="1" align="center">1.19 (1.11â€“1.27)</td><td rowspan="1" colspan="1" align="center">1.8 Ã— 10<sup>âˆ’7</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs17418283</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">94,180,344</td><td rowspan="1" colspan="1" align="center"><em>MCTP1</em></td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.28</td><td rowspan="1" colspan="1" align="center"><em>1.09 (0.93â€“1.28)</em></td><td rowspan="1" colspan="1" align="center"><em>0.31</em></td><td rowspan="1" colspan="1" align="center"><em>1.19 (1.01â€“1.41)</em></td><td rowspan="1" colspan="1" align="center"><em>0.038</em></td><td rowspan="1" colspan="1" align="center"><em>1.25 (1.15â€“1.36)</em></td><td rowspan="1" colspan="1" align="center"><em>9.7</em> Ã— <em>10</em><sup>âˆ’8</sup></td><td rowspan="1" colspan="1" align="center">1.21 (1.13â€“1.30)</td><td rowspan="1" colspan="1" align="center">1.3 Ã— 10<sup>âˆ’7</sup></td></tr></tbody></table></div><div id="largeobj_idm140570784773168" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2678639/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140570784845888"><p id="__p21" class="p p-first-last">GSK reduced sample: Excluding 261 BP cases also present in WTCCC sample. Analyzed using an additive genetic model; italics indicate analysis of an imputed SNP.</p></div><div id="TF2-1"><p id="__p22" class="p p-first-last">*NCBI Build 35-bp position.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2678639/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570850057728" class="figure"><a href="/pmc/articles/PMC2678639/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2678639_zpq9990968330001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq9990968330001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2678639/bin/zpq9990968330001.jpg"></a></div><div id="largeobj_idm140570850057728" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2678639/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2678639/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p23">Plot of âˆ’log<sub>10</sub> <em>P</em> values for NIMH/Pritzker, GSK, and WTCCC BP association meta-analysis for chromosomal regions with <em>P</em> &lt; 10<sup>âˆ’6</sup>. refFLAT annotated genes are shown in <em>Aâ€“C Lower</em>. SNPs genotyped in all 3 samples are denoted by a thick black circle. Stronger red intersity indicates higher <em>r</em><sup>2</sup> with the most significant SNP (purple diamond).</p></div></div></div><p id="__p24">To assess the sensitivity of our results to adjustment for PCs, we reanalyzed data from each study without PCs (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST5" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S4</a>). Meta-analysis <em>P</em> values for the top SNPs with and without PC adjustment varied by up to approximately 1 order of magnitude (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST8" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S5</a>). The most notable change was that evidence for our most strongly associated nonsynonymous SNP, rs1042779 on chromosome 3, became stronger (OR = 1.20, <em>P</em> = 2.7 Ã— 10<sup>âˆ’8</sup> vs. OR = 1.19, <em>P</em> = 1.8 Ã— 10<sup>âˆ’7</sup>), indicating there may have been some population stratification at this locus.</p><p id="__p25">We next considered the impact on our top results of data imputation. On chromosome 3, the most strongly associated SNP was genotyped in NIMH/Pritzker and GSK; on chromosomes 5 and 1 the most strongly associated SNP was imputed in all 3 studies. Imputation quality of the most strongly associated SNP from each region was high (estimated <em>r</em><sup>2</sup> &gt; 0.8) (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST5" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S4</a>); previous work demonstrates that results based on imputed data of this quality generally agree well with corresponding results based on direct genotyping (refs. <a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102656">7</a> and <a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_249671900">8</a> and Y. Li, C. J. Willer, J. Ding, P. Scheet, and G. R. Abecasis, personal communication). Further, in the chromosome 1 region, SNP rs1461356 (OR = 1.79, <em>P</em> = 3.3 Ã— 10<sup>âˆ’7</sup>) was directly genotyped in all 3 studies, as was rs2289247 (OR = 1.16, <em>P</em> = 1.7 Ã— 10<sup>âˆ’6</sup>) in the chromosome 3 region. On chromosome 5, no completely genotyped SNP had <em>P</em> &lt; 10<sup>âˆ’4</sup>.</p><p id="__p26">Because of the strong WTCCC contribution to the 3 top results, we further evaluated the use of the WTCCC extended reference set in our analysis. We tested for association between the WTCCC BP cases and the much smaller NBS-only control sample. With this much-reduced sample, we saw more modest evidence of association for each SNP, although similar effect sizes for the chromosome 1 and 3 SNPs: rs472913 (OR = 1.17, <em>P</em> = 0.0020 vs. OR = 1.20, <em>P</em> = 6.3 Ã— 10<sup>âˆ’7</sup> in the BP cases vs. extended reference set), rs1042779 (OR = 1.17, <em>P</em> = 0.0024 vs. OR = 1.16, <em>P</em> = 0.00012), and rs17418283 (OR = 1.11, <em>P</em> = 0.076 vs. OR = 1.25, <em>P</em> = 9.7 Ã— 10<sup>âˆ’8</sup>) (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST7" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S4<em>C</em></a>). We also tested for allele frequency differences between the NBS controls and the non-BP cases. SNPs rs472913 (<em>P</em> = 0.75) and rs1042779 (<em>P</em> = 0.56) on chromosomes 1 and 3 showed no evidence of a difference. For rs17418283, the allele frequencies in the BP cases, NBS controls, and non-BP cases were 0.31, 0.29, and 0.27, and the allele frequencies in the NBS and non-BP cases were significantly different (<em>P</em> = 0.0067), but this difference was considerably less significant than that observed in the BP/extended reference set analysis (<em>P</em> = 9.7 Ã— 10<sup>âˆ’8</sup>).</p><p id="__p27" class="p p-last">These data quality and robustness analyses suggest that the evidence for BP association remains strong for the chromosome 1 and 3 regions and less so for the chromosome 5 region, although the presence of a second signal in the chromosome 5 region strengthens our interest in that region.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title" style="text-transform: uppercase;">Discussion</h2><p id="__p28" class="p p-first">We carried out genome-wide association analyses of 2 new and 1 previously published (WTCCC) BP GWAS, and then performed metaanalyses of the 2 new studies with and without the WTCCC study. The sample size of the 3-study meta-analysis was 3,683 cases and 14,507 controls. No SNP reached genome-wide significance. For the most strongly associated SNP from each of the 7 regions with <em>P</em> &lt; 10<sup>âˆ’5</sup> in the 2-study meta-analysis, none had <em>P</em> &lt; 0.1 in the WTCCC data or &lt;0.0001 in the 3-study meta-analysis (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1<em>B</em></a>). However, 6 of these 7 associations went in the same direction in the WTCCC data.</p><p id="__p29">In the 3-study meta-analysis, we identified 3 regions with <em>P</em> â‰ˆ 10<sup>âˆ’7</sup>: 1p31.1, 3p21, and 5q15. The most strongly associated SNP genome-wide is rs17418283, located in an intron of <em>MCTP1</em> on chromosome 5. <em>MCTP1</em> is an extensively spliced, highly conserved membrane protein that binds Ca<sup>2+</sup> with high affinity in the absence of phospholipids (<a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102676">12</a>) and is highly expressed in the brain (<a href="http://symatlas.gnf.org/SymAtlas/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://symatlas.gnf.org/SymAtlas/</a>). This gene may have 2 independent association signals, because SNPs in the first intron of <em>MCTP1</em> are also moderately and independently associated with BP. Our finding is intriguing because Ferreira et al. (<a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102654">6</a>) recently implicated another Ca<sup>2+</sup>-related gene, the L-type calcium channel subunit gene, <em>CACNA1C</em>, in BP (<em>P</em> = 7.0 Ã— 10<sup>âˆ’8</sup>). The chromosome 5 region also contains <em>ANKRD32</em> (ankyrin repeat domain 32), which encodes an uncharacterized ankyrin-repeat-containing protein. There is no known functional relationship between <em>ANKRD32</em> and the <em>ANK3</em> gene implicated by Ferreira et al. (<a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102692">6</a>).</p><p id="__p30">We observed the strongest nonsynonymous SNP-BP association genome-wide for rs1042779 on chromosome 3 at 52.8 Mb. There are 10 genes in the most strongly associated 243 kb region, and &gt;25 genes in the â‰ˆ1 Mb larger region bounded by flanking recombination hot spots (<a href="/pmc/articles/PMC2678639/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a><em>B</em>). rs1042779 is an Arg595Gln SNP in <em>ITIH1</em>. We also observed association with the nonsynonymous Glu585Val SNP rs678 in the same exon of <em>ITIH1 (r</em><sup>2</sup> = 0.96 for rs678 and rs1042779 in HapMap CEU). Four additional nonsynonymous SNPs in this region had <em>P</em> &lt; 10<sup>âˆ’5</sup>: rs2289247 (Val367Met) and rs11177 Arg27Gln in <em>GNL3</em> (guanine nucleotide binding like-3<em>)</em>, rs1029871 (Pro225Ala) in <em>NEK4</em>, and rs3617 (Gln315Lys) in <em>ITIH3</em>. For the Glu585Val and Pro225Ala variants, Glu and Pro are the conserved alleles in mammals, and the Val and Ala alleles are functionally nonconservative changes. <em>ITIH1</em> encodes a serine protease inhibitor highly expressed in liver (<a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102649">13</a>). The family of inter-alpha trypsin inhibitors is thought to have anti-proteolytic activities and to play an anti-inflammatory role (<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102648">14</a>). <em>GNL3</em> encodes nucleostemin, which was isolated from rat CNS, is expressed in the nucleolus of stem cells, and is thought to be a critical regulator of the cell cycle. Expression of <em>GNL3</em> rapidly declines upon neuronal cell differentiation, and both over- and under-expression lead to decreased stem cell proliferation in the CNS (<a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102682">15</a>). Aberrant regulation of nucleostemin would be consistent with the neurotrophic hypothesis of mood disorders, which posits that stem-cell proliferative potential in the brain modulates BP risk (<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102675">16</a>). This association signal is 11 Mb proximal to a BP linkage peak (LOD = 2.01) reported for the families that are the source of the NIMH/Pritzker BP cases analyzed here (<a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102658">17</a>).</p><p id="__p31">On chromosome 1 at 60.8 Mb, there is no well annotated gene within 500 kb of the most strongly associated SNP, rs472913. rs472913 is in moderately high LD with rs2989476 (<em>r</em><sup>2</sup> = 0.74), the most strongly associated BP SNP in the WTCCC BP case/extended reference set analysis (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102657">4</a>). This SNP is in an intron of a possible transcript annotated by Unigene as P3 NTera2D1 teratocarcinoma, and defined by multiple expressed sequence tags, including one expressed in brain.</p><p id="__p32">After the 3 loci with the strongest associations on chromosomes 1, 3, and 5, there were additional notable observations. On chromosome 2, we identified rs13409348 (OR = 1.20, <em>P</em> = 2.7 Ã— 10<sup>âˆ’6</sup>) (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a>) located in intron 4 of the gene <em>CTNNA2</em> (encoding alpha N catenin 2). <em>CTNNA2</em> is expressed almost exclusively in distinct neuronal populations in primates (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102673">18</a>). alpha N-catenin is thought to be a key regulator of the stability of synaptic contacts and the motility of dendritic spines, a key aspect of neuronal plasticity (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102642">19</a>). Its deficiency in mice causes axon migration defects (<a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102681">20</a>) and an abnormal startle response (<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102659">21</a>), a murine behavior indicative of dysregulation of sensorimotor gating that is often considered an endophenotype of psychosis in humans (<a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102655">22</a>). The gene <em>LRRTM1</em> (Leucine-rich repeat transmembrane neuronal 1) is located in an intron of <em>CTNNA2</em>. We observed associations of a haplotype near <em>LRRTM1</em> with schizophrenia (<em>P</em> = 0.0014 in 1,002 families) and handedness (<em>P</em> = 0.00002) (<a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210757105">23</a>). This risk haplotype is best tagged by the rs1446109 A allele, which is â‰ˆ1 Mb from rs13409348 and shows modest evidence of association in our meta-analysis (A risk allele, OR = 1.08, <em>P</em> = 0.08). We also identified rs2537859 (OR = 1.16, <em>P</em> = 4.2 Ã— 10<sup>âˆ’6</sup>) (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a>) on chromosome 4, 42 kb upstream of <em>KIT</em> (v-kit Hardy-Zuckerman 4 feline sarcoma viral)<em>. KIT</em> encodes a cytokine receptor of the tyrosine kinase family, expressed in multiple cells including neurons (<a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102688">24</a>). <em>KIT</em> binds stem cell factor and plays a role in cell survival, proliferation, and differentiation (<a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102660">25</a>). Nonsynonymous mutations and deletions in <em>KIT</em> are associated with multiple diseases and <em>KIT</em> overexpression in the brain can induce gliomas (<a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102666">26</a>).</p><p id="__p33">In the 2- and 3-study meta-analyses we identified a region on chromosome 1 at â‰ˆ195 Mb with 2 strong but distinct association signals â‰ˆ300 kb apart and separated by a strong recombination hotspot. rs2813164 (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>) emerged in the 2-study meta-analysis and is located between <em>NEK7</em> <a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF4" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF4</a>and <em>ATP6V1G3</em> (Fig. S4<em>B</em>). rs12568099 emerged in the 3-study meta-analysis (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a>) and is located 25 kb downstream of <em>PTPRC</em> (<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=SF4" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S4<em>C</em></a>); there also was evidence for association around <em>NEK7</em>. Within the region, <em>ATP6V1G3</em> is of particular interest as it encodes a component of vacuolar proton-pumping ATPase involved in organelle and synaptic vesicle acidification (<a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102644">27</a>).</p><p id="__p34">Previous genome-wide association studies and subsequent metaanalyses of BP have identified SNPs that reached genome-wide significance or had support across multiple studies; for a description of the overlap between our sample and those of other studies, see <a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Materials and Methods</em></a>. The Ferreira et al. (<a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102650">6</a>) meta-analysis identified SNPs with strong evidence of association in the regions of <em>CACNA1C</em> (rs1006737, OR = 1.18, <em>P</em> = 7.0 Ã— 10<sup>âˆ’8</sup>) and <em>ANK3</em> (rs10994336, OR = 1.45, <em>P</em> = 9.1 Ã— 10<sup>âˆ’9</sup>). Schulze et al. (<a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321379385">28</a>) reported support for association with <em>ANK3</em> in a German sample (rs10994336, OR = 1.70, <em>P</em> = 0.0001) and stronger evidence from meta-analysis with NIMH case and control samples (OR = 1.70, <em>P</em> = 1.7 Ã— 10<sup>âˆ’5</sup>). In the GSK samples that did not overlap the sample of Ferreira et al. (<a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102693">6</a>), we observed modest replication of rs10994336 (OR = 1.37, <em>P</em> = 0.042) and slightly stronger evidence when we included the NIMH samples overlapping those of Shultze et al., but not Ferreria et al. (OR = 1.39, <em>P</em> = 0.012); a fixed-effects meta-analysis of the results of Ferreria et al., the German Schulze et al. sample, and the nonoverlapping GSK and NIMH samples, yielded compelling evidence for association (OR = 1.47, <em>P</em> = 1.1 Ã— 10<sup>âˆ’10</sup>). The <em>ANK3</em>-encoded protein, Ankyrin G, is a cytoskeleton-binding molecule located at the Nodes of Ranvier and involved in the assembly of a variety of membrane proteins, including voltage-gated sodium (<a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102683">29</a>) and potassium (<a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_802037609">30</a>) channels and cell adhesion molecules (reviewed in ref. <a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102668">31</a>). We did not observe association with rs1006737 (OR = 1.01, <em>P</em> = 0.81). The Baum et al. (<a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102684">3</a>) meta-analysis identified a strongly associated SNP in <em>DGKH</em> (rs1012053, OR = 1.59, <em>P</em> = 1.5 Ã— 10<sup>âˆ’8</sup>); we did not observe association in the independent GSK sample (OR = 1.01, <em>P</em> = 0.76). The lack of confirmation of association in our independent NIMH/Pritzker and GSK samples for these loci may reflect the need for larger sample sizes to detect small effects, differences in sample inclusion criteria including heterogeneity in BP cases, or false positive results in the original studies.</p><p id="__p35">Our GWAS results and others reported to date suggest there are few, if any, common SNP variants with large effects on BP risk. BP differs from complex diseases such as macular degeneration (<a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102643">32</a>), Crohn's disease (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102671">4</a>), and type 1 diabetes (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102661">4</a>), which are influenced by one or more loci of large effect. Type 2 diabetes appears intermediate in that it has a single locus (<em>TCF7L2</em>) of moderate effect together with loci of substantially smaller effects requiring large samples for detection (<a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102667">33</a>). BP may be due to a combination of many common variants with weak effects, potentially including those identified above, and rarer variants with a range of effect sizes. For common variants, detection and verification will require larger GWA and follow-up samples. The strong familial component of BP and the lack of common variants of large effect suggest that rarer SNP variants along with other types of genetic differences such as copy number variations (CNVs) or epigenetic modifications may play a substantial role. Sequencing of BP cases and controls will help unravel the role of rare SNPs and copy number variants. Detection of BP risk variants anywhere in the allele frequency spectrum has the potential to lead to greater understanding of the biology that underlies BP, the sources of heterogeneity among BP cases, and to the subsequent development of new BP treatments.</p><p id="__p36" class="p p-last">In summary, we have identified chromosomal regions and specific genes that may harbor variants that contribute to BP risk and merit further study. Several of the genes encoded in these chromosomal regions are consistent with the notion that subtle differences in neural development and/or neuroplasticity may underlie mood disorders. Further work is required to determine whether these loci contain causal variants that could be more strongly associated with BP risk. If so, it would be of great interest to study the neural localization, expression, and regulation of these genes in human and animal brain, particularly in circuits relevant to mood disorders.</p></div><div id="__sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec3title" style="text-transform: uppercase;">Methods</h2><div id="__sec4" class="sec sec-first"><p></p><h3 id="__sec4title" class="inline">Sample Ascertainment and Selection. </h3><p id="__p37" class="p p-first">Subjects for all participating studies gave informed consent and study protocols were approved by the relevant institutional review board or ethics committee.</p><div id="__sec5" class="sec"><p></p><h4 id="__sec5title" class="inline">NIMH/Pritzker. </h4><p id="__p38" class="p p-first">We selected samples for previously-studied individuals from 2 sources: 1,130 Bipolar I (BP I) cases and 772 controls from the NIMH Human Genetics Initiative Repository (<a href="http://zork.wustl.edu/nimh/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://zork.wustl.edu/nimh/</a>) and 47 BP I cases from the University of Michigan Prechter Repository. NIMH Repository cases were collected as part of a multicenter study of individuals with BP and their families (<a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102685">34</a>â€“<a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102663">35</a>). Cases were diagnosed according to DMS-III or DSM-IV criteria, using the Diagnostic Interview for Genetic Studies (DIGS) (<a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321379386">36</a>) (<em>n</em> = 1,081) or Family Interview for Genetic Studies (FIGS) (<a href="#B37" rid="B37" class=" bibr popnode">37</a>) and/or medical record review (<em>n</em> = 67); we excluded cases with low confidence diagnoses. From each available non-Ashkenazi European-origin family, when possible we selected 2 BP I siblings including the proband if available (<em>n</em> = 946 individuals in 473 sibling pairs); otherwise we selected a single BP I case (<em>n</em> = 184).</p><p id="__p39">We selected an additional 47 non-Ashkenazi European-origin BP I cases who met DSM-IV criteria from the University of Michigan Prechter Bipolar Genetics Repository. Subjects were recruited using flyers distributed at the University of Michigan Depression Center and on the Center website and interviewed using the DIGS.</p><p id="__p40">We selected as controls 772 non-Ashkenazi European-origin individuals aged 20â€“70 years from the NIMH Human Genetics Initiative Repository who reported they had not been diagnosed with or treated for BP or schizophrenia, and had not heard voices that others could not hear. We excluded individuals with suspected major depression based on answers to questions related to depressive mood.</p><p id="__p41" class="p p-last">To minimize population stratification, we selected individuals reporting primarily European ancestry and matched NIMH controls to NIMH cases based on self-reported ancestry in the DIGS.</p></div><div id="__sec6" class="sec"><p></p><h4 id="__sec6title" class="inline">GSK. </h4><p id="__p42" class="p p-first-last">We selected 899 BP cases (814 BP I and 85 BP II) and 904 controls from subjects recruited at 3 study sites: the Institute of Psychiatry (IOP) in London, U.K. (483 cases and 462 controls); the Centre for Addiction and Mental Health in Toronto, Canada (334 cases and 257 controls); and the University of Dundee, U.K. (82 cases and 185 controls). Cases were recruited through advertisements in hospitals, clinics, primary care physician offices, and patient support groups, were â‰¥18 years of age at interview, and reported Caucasian ethnicity. They were interviewed using the Schedules for Clinical Assessment in Neuropsychiatry (SCAN). BP diagnoses were established according to DSM-IV or ICD-10 criteria, using the computerized algorithm (CATEGO) for the SCAN2.1 interview (WHO) (<a href="#B38" rid="B38" class=" bibr popnode">38</a>). Cases were excluded if they received a diagnosis of i.v. drug dependency or reported i.v. drug use or if they had mood incongruent psychotic symptoms or if manic episodes only occurred in conjunction with or as a result of alcohol, substance abuse, substance dependence, medical illnesses, or medications. 261 cases recruited at the IOP were present in the WTCCC BP sample. When we performed a 3-study meta-analysis, we excluded these overlapping cases from the GSK sample and used 638 cases and 904 controls as the GSK analysis sample (<a href="/pmc/articles/PMC2678639/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and<a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=ST1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"> Table S1</a>). Controls were â‰¥18 years of age, reported Caucasian ethnicity, and denied the presence of any psychiatric disorders in a questionnaire.</p></div><div id="__sec7" class="sec sec-last"><p></p><h4 id="__sec7title" class="inline">WTCCC. </h4><p id="__p43" class="p p-first-last">We accessed phenotype and GWAS genotype data on 1,868 BP cases and 12,831 additional individuals provided by the WTCCC (<a href="http://www.wtccc.org.uk/info/access_to_data_samples.shtml" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.wtccc.org.uk/info/access_to_data_samples.shtml</a>). All samples were ascertained in the U.K., reported European ancestry, and clustered with the WTCCC in multidimensional scaling of the WTCCC and the 270 HapMap samples. BP and related phenotypes were diagnosed using Research Diagnostic Criteria (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102652">4</a>). 71% of cases were diagnosed as BP I, the remainder as schizoaffective disorder bipolar type (15%), BP II (9%), or manic disorder (5%). For our primary WTCCC control group, we included 12,831 individuals unscreened for BP: 1,458 National Blood Service controls together with a total of 11,373 individuals from the coronary artery disease (<em>n</em> = 1,926), Crohn's disease (<em>n</em> = 1,748), hypertension (<em>n</em> = 1,952), rheumatoid arthritis (<em>n</em> = 1,860), type 1 diabetes (<em>n</em> = 1,963), and type 2 diabetes (<em>n</em> = 1,924) groups. We could not include the WTCCC 1958 Birth Cohort data owing to access constraints.</p></div></div><div id="__sec8" class="sec"><p></p><h3 id="__sec8title" class="inline">Sample Genotyping and Quality Control (QC). </h3><div id="__sec9" class="sec sec-first"><p></p><h4 id="__sec9title" class="inline">NIMH/Pritzker. </h4><p id="__p44" class="p p-first">NIMH and Prechter samples were genotyped on the Illumina HumanHap550 Beadchip at Stanford University and the University of Michigan following manufacturer recommendations. Phase1 (phase 2) comprised 999 (1,122) samples, including 29 (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102645">19</a>) duplicate samples and 15 parent-offspring trios (1 parent-offspring duo,1 parent-offspring trio, 8 parent-offspring quartets). In each phase, we first called genotypes using the Illumina default cluster file based on the HapMap CEU samples and excluded samples with success rate &lt;98.0%. Genotypes were then reclustered based on our own genotyped samples; for each SNP, we retained the genotypes from the clustering method yielding higher genotyping success rate. Nine individuals with inconsistent reported and genotype-based sex were dropped as were 6 samples to eliminate cryptic relatedness. We removed 6 individuals with evidence for ancestry that differed from the rest of the sample (see below).</p><p id="__p45" class="p p-last">Of the 541,327 genotyped autosomal SNPs, we excluded 12,592 because of (<em>i</em>) total non-Mendelian and duplicate errors &gt;2; (<em>ii</em>) data inconsistent with Hardy-Weinberg equilibrium (<em>P</em> &lt; 10<sup>âˆ’5</sup>) in unrelated cases and controls; or (<em>iii</em>) SNP call rate &lt;95% in either phase. Genotyping success rate (99.9%), duplicate genotype concordance (99.992%), and parent-child Mendelian consistency rates (99.93%) were high. There was no evidence for differential quality parameters between phases 1 and 2. Among the 528,735 SNPs that passed QC, 512,844 had MAF â‰¥1% in the combined sample and were used in association analysis.</p></div><div id="__sec10" class="sec"><p></p><h4 id="__sec10title" class="inline">GSK. </h4><p id="__p46" class="p p-first-last">DNAs for 1,896 GSK individuals, including duplicated samples for 109 individuals, were genotyped on the Illumina HumanHap550 Beadchip at Illumina following manufacturer recommendations. Genotype calls were generated as described in ref. <a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102677">39</a> with minor modifications. Initial calls were made using the Illumina CEU cluster file. 61 samples from 43 individuals with call rate &lt;95% were dropped as were 11 individuals with inconsistent reported and genotype-based gender, 13 individuals to eliminate cryptic relatedness, and 26 outlier individuals in the stratification analysis (see below). SNPs with call frequency &lt;99% were reclustered based on the GSK samples. 20,172 SNPs with (a) call frequency &lt;95%; (b) call frequency 95â€“98% and cluster separation score &lt;0.3, or heterozygote excess frequency &gt;0.1 or less than âˆ’0.1; or (c) call frequency &gt;98% and cluster separation &lt;0.25 or heterozygote excess frequency &gt;0.3 or less than âˆ’0.3 were dropped. Of the 521,990 SNPs that passed QC, 512,508 (full sample) and 512,668 (reduced sample) had MAFâ‰¥0.01 and were used in association analysis. Duplicate genotype concordance was 99.99%.</p></div><div id="__sec11" class="sec sec-last"><p></p><h4 id="__sec11title" class="inline">WTCCC. </h4><p id="__p47" class="p p-first-last">Samples were genotyped on the Affymetrix GeneChip 500K Mapping Array Set as described in ref. <a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102678">4</a>. 397,653 SNPs passed WTCCC QC and had MAFâ‰¥0.01.</p></div></div><div id="__sec12" class="sec"><p></p><h3 id="__sec12title" class="inline">Genotype Imputation. </h3><p id="__p48" class="p p-first">For each study, we imputed genotypes for up to â‰ˆ2 million autosomal SNPs with MAFâ‰¥0.01, using the genotypes from the Illumina HumanHap550 Beadchip (NIMH/Pritzker, GSK) or the Affymetrix GeneChip 500K Mapping Array Set (WTCCC) and phased chromosomes for the 60 HapMap CEU founders. Genotypes were imputed using a Hidden Markov model as programmed in MACH (<a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102651">9</a>) for SNPs not present in the genotyping platform or whose genotype data failed QC. We imputed WTCCC and Pritzker to Build 35 and GSK to Build 36. We retained imputed SNPs with estimated <em>r</em><sup>2</sup> &gt; 0.3 and imputed MAFâ‰¥0.01. The numbers of imputed SNPs used in analysis were 1,960,204, 1,952,561 (1,952,849), and 2,034,246 for NIMH/Pritzker, GSK full sample (reduced sample), and WTCCC, respectively.</p><p id="__p49" class="p p-last">The <a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Materials Materials and Methods</em></a> contains an expanded version of the following sections.</p></div><div id="__sec13" class="sec"><p></p><h3 id="__sec13title" class="inline">Assessment of Stratification. </h3><p id="__p50" class="p p-first-last">In each set of study samples we performed principal components (PC) analysis based on a subset of the sample genotypes (<a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102691">40</a>) in unrelated individuals. We excluded individuals in the NIMH/Pritzker and GSK samples with PC &gt;6 SD from the mean of one or more of the top 10 PCs; to mirror the analysis used by the WTCCC (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102639">4</a>), we did not exclude WTCCC samples.</p></div><div id="__sec14" class="sec"><p></p><h3 id="__sec14title" class="inline">GWA Analysis. </h3><p id="__p51" class="p p-first-last">We eliminated 694 SNPs with allele frequency differences &gt;0.2 for any pair of studies. We analyzed the observed allele counts or imputed allele dosages, using logistic regression assuming an additive genetic model, with genotype-based PCs and study site (GSK only) as covariates, and then repeated the analysis without PCs. In the NIMH/Pritzker sample, we used a sandwich estimator (<a href="#B41" rid="B41" class=" bibr popnode">41</a>) to adjust the estimated variances. For the WTCCC sample, we compared the BP cases to the extended reference set as our primary analysis, except in the HLA region on chromosome 6 from 27.2 to 34.0 Mb where we excluded the 5,571 autoimmune disease cases (type 1 diabetes, Crohn's disease, rheumatoid arthritis) from the GWA analysis.</p></div><div id="__sec15" class="sec sec-last"><p></p><h3 id="__sec15title" class="inline">Meta-Analysis of GWA Samples. </h3><p id="__p52" class="p p-first-last">We performed a fixed effects meta-analysis, using the OR and 95% confidence intervals to combine the association evidence from the study-specific GWA analyses. We used association results for experimentally derived genotypes when available, and for imputed genotypes otherwise. 2,366,197 autosomal SNPs passed QC and had MAFâ‰¥0.01 in all 3 samples. We adjusted for the genomic control values in each study separately for genotyped and imputed SNPs by increasing the standard error of the OR estimate to correspond to the genomic control <em>P</em> value. Evidence for heterogeneity between ORs was assessed using Cochrans's Q statistic and <em>I</em><sup>2</sup> (<a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178102640">9</a>).</p></div></div><div id="__sec16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec16title" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supporting Information:</strong>
            
            <div class="sup-box half_rhythm" id="idm140570816797808"><a href="/pmc/articles/PMC2678639/bin/supp_106_18_7501__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140570776864768" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140570776864768title" style="text-transform: uppercase;">Acknowledgments.</h2><div class="sec"><p id="__p55">We thank the participants who donated their time and DNA to make this study possible; Elzbieta Sliwerska for assistance with genotyping; Terry Gliedt, Peggy White, and Randy Pruim for assistance with data management and manuscript preparation; members of the National Institutes of Mental Health Human Genetics Initiative (see <a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Acknowledgments</em></a>) and the University of Michigan Prechter Bipolar DNA Repository for generously providing phenotype data and DNA samples; the staff at the recruiting sites in London, Toronto, and Dundee, and at GlaxoSmithKline for contributions to recruitment and study management; and the other members of the Pritzker Neuropsychiatric Disorders Research Consortium (PNDRC) for their contribution to the research. This study makes use of data generated by the Wellcome Trust Case Control Consortium (WTCCC) (see <a href="http://www.pnas.org/cgi/data/0813386106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Acknowledgments</em></a>). Many of the authors (L.J.S., W.G., M.F., J.Z.L., M. Burmeister, D.A., R.C.T., F.M., A.F.S., W.E. B., J.D.B., E.G.J., S.J.W, R.M.M., H.A., M. Boehnke) are members of the PNDRC. This work was supported by the Pritzker Neuropsychiatric Disorders Research Fund, L.L.C.</p></div></div><div id="idm140570776868160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140570776868160title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140570776868032"> </p><p id="__p53" class="p p-first-last">Conflict of interest statement: P. Muglia, X.Q.K., F.T., C.F., A.A., and D.K.B. are, or were, full-time employees of GlaxoSmithKline when this article was written. GlaxoSmithKline sponsored the bipolar collection at the London, Toronto, and Dundee sites. R.D., K.M., P. McGuffin, J.S.S., J.L.K., L.M., J.B.V., and A.E.F. worked at those sites</p><p></p><p class="fn sec" id="idm140570776866816"></p><p id="__p54" class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/cgi/content/full/0813386106/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/cgi/content/full/0813386106/DCSupplemental</a>.</p><p></p></div></div><div id="idm140570777162352" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140570777162352title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Merikangas KR, et al.  Lifetime and 12-month prevalence of bipolar spectrum disorder in the National Comorbidity Survey replication. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2007;<span class="ref-vol">64</span>:543â€“552.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1931566/">PMC free article</a>]</span> [<a href="/pubmed/17485606" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Lifetime+and+12-month+prevalence+of+bipolar+spectrum+disorder+in+the+National+Comorbidity+Survey+replication&amp;author=KR+Merikangas&amp;volume=64&amp;publication_year=2007&amp;pages=543-552&amp;pmid=17485606&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Craddock N, Forty L. Genetics of affective (mood) disorders. <span><span class="ref-journal">Eur J Hum Genet. </span>2006;<span class="ref-vol">14</span>:660â€“668.</span> [<a href="/pubmed/16721402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Genetics+of+affective+(mood)+disorders&amp;author=N+Craddock&amp;author=L+Forty&amp;volume=14&amp;publication_year=2006&amp;pages=660-668&amp;pmid=16721402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Baum AE, et al.  A genome-wide association study implicates diacylglycerol kinase eta (<em>DGKH</em>) and several other genes in the etiology of bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2008;<span class="ref-vol">13</span>:197â€“207.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2527618/">PMC free article</a>]</span> [<a href="/pubmed/17486107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=A+genome-wide+association+study+implicates+diacylglycerol+kinase+eta+(DGKH)+and+several+other+genes+in+the+etiology+of+bipolar+disorder&amp;author=AE+Baum&amp;volume=13&amp;publication_year=2008&amp;pages=197-207&amp;pmid=17486107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661â€“678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Sklar P, et al.  Whole-genome association study of bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2008;<span class="ref-vol">13</span>:558â€“569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3777816/">PMC free article</a>]</span> [<a href="/pubmed/18317468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Whole-genome+association+study+of+bipolar+disorder&amp;author=P+Sklar&amp;volume=13&amp;publication_year=2008&amp;pages=558-569&amp;pmid=18317468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Ferreira MA, et al.  Collaborative genome-wide association analysis supports a role for <em>ANK3</em> and <em>CACNA1C</em> in bipolar disorder. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1056â€“1058.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2703780/">PMC free article</a>]</span> [<a href="/pubmed/18711365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Collaborative+genome-wide+association+analysis+supports+a+role+for+ANK3+and+CACNA1C+in+bipolar+disorder&amp;author=MA+Ferreira&amp;volume=40&amp;publication_year=2008&amp;pages=1056-1058&amp;pmid=18711365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906â€“913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Scott LJ, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341â€“1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Higgins JP, Thompson SG, Deeks JJ, Altman DG. Measuring inconsistency in meta-analyses. <span><span class="ref-journal">Br Med J. </span>2003;<span class="ref-vol">327</span>:557â€“560.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC192859/">PMC free article</a>]</span> [<a href="/pubmed/12958120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+Med+J&amp;title=Measuring+inconsistency+in+meta-analyses&amp;author=JP+Higgins&amp;author=SG+Thompson&amp;author=JJ+Deeks&amp;author=DG+Altman&amp;volume=327&amp;publication_year=2003&amp;pages=557-560&amp;pmid=12958120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997â€“1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Hoggart J, Clark TG, DeIorio M, Whittaker JC, Balding DJ. Genome-wide significance for dense SNP and resequencing data. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:179â€“185.</span> [<a href="/pubmed/18200594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Genome-wide+significance+for+dense+SNP+and+resequencing+data&amp;author=J+Hoggart&amp;author=TG+Clark&amp;author=M+DeIorio&amp;author=JC+Whittaker&amp;author=DJ+Balding&amp;volume=32&amp;publication_year=2008&amp;pages=179-185&amp;pmid=18200594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Shin OH, Han W, Wang Y, SÃ¼dhof TC. Evolutionarily conserved multiple C2 domain proteins with two transmembrane regions (MCTPs) and unusual Ca2+ binding properties. <span><span class="ref-journal">J Biol Chem. </span>2005;<span class="ref-vol">280</span>:1641â€“1651.</span> [<a href="/pubmed/15528213" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Evolutionarily+conserved+multiple+C2+domain+proteins+with+two+transmembrane+regions+(MCTPs)+and+unusual+Ca2++binding+properties&amp;author=OH+Shin&amp;author=W+Han&amp;author=Y+Wang&amp;author=TC+SÃ¼dhof&amp;volume=280&amp;publication_year=2005&amp;pages=1641-1651&amp;pmid=15528213&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Bost F, et al.  Isolation and characterization of the human inter-alpha-trypsin inhibitor heavy-chain H1 gene. <span><span class="ref-journal">Eur J Biochem. </span>1993;<span class="ref-vol">218</span>:283â€“291.</span> [<a href="/pubmed/7505744" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Biochem&amp;title=Isolation+and+characterization+of+the+human+inter-alpha-trypsin+inhibitor+heavy-chain+H1+gene&amp;author=F+Bost&amp;volume=218&amp;publication_year=1993&amp;pages=283-291&amp;pmid=7505744&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Kobayashi H. Endogenous anti-inflammatory substances, inter-alpha-inhibitor and bikunin. <span><span class="ref-journal">Biol Chem. </span>2006;<span class="ref-vol">387</span>:1545â€“1549.</span> [<a href="/pubmed/17132099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Chem&amp;title=Endogenous+anti-inflammatory+substances,+inter-alpha-inhibitor+and+bikunin&amp;author=H+Kobayashi&amp;volume=387&amp;publication_year=2006&amp;pages=1545-1549&amp;pmid=17132099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Tsai RY, McKay RD. A nucleolar mechanism controlling cell proliferation in stem cells and cancer cells. <span><span class="ref-journal">Genes Dev. </span>2002;<span class="ref-vol">16</span>:2991â€“3003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC187487/">PMC free article</a>]</span> [<a href="/pubmed/12464630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=A+nucleolar+mechanism+controlling+cell+proliferation+in+stem+cells+and+cancer+cells&amp;author=RY+Tsai&amp;author=RD+McKay&amp;volume=16&amp;publication_year=2002&amp;pages=2991-3003&amp;pmid=12464630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Duman RS, Monteggia LM. A neurotrophic model for stress-related mood disorders. <span><span class="ref-journal">Biol Psychiatry. </span>2006;<span class="ref-vol">59</span>:1116â€“1127.</span> [<a href="/pubmed/16631126" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=A+neurotrophic+model+for+stress-related+mood+disorders&amp;author=RS+Duman&amp;author=LM+Monteggia&amp;volume=59&amp;publication_year=2006&amp;pages=1116-1127&amp;pmid=16631126&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">Kelsoe JR, et al.  A genome survey indicates a possible susceptibility locus for bipolar disorder on chromosome 22. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2001;<span class="ref-vol">98</span>:585â€“590.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC14631/">PMC free article</a>]</span> [<a href="/pubmed/11149935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=A+genome+survey+indicates+a+possible+susceptibility+locus+for+bipolar+disorder+on+chromosome+22&amp;author=JR+Kelsoe&amp;volume=98&amp;publication_year=2001&amp;pages=585-590&amp;pmid=11149935&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Smith A, Bourdeau I, Wang J, Bondy CA. Expression of Catenin family members CTNNA1, CTNNA2, CTNNB1 and JUP in the primate prefrontal cortex and hippocampus. <span><span class="ref-journal">Brain Res Mol Brain Res. </span>2005;<span class="ref-vol">135</span>:225â€“231.</span> [<a href="/pubmed/15857685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res+Mol+Brain+Res&amp;title=Expression+of+Catenin+family+members+CTNNA1,+CTNNA2,+CTNNB1+and+JUP+in+the+primate+prefrontal+cortex+and+hippocampus&amp;author=A+Smith&amp;author=I+Bourdeau&amp;author=J+Wang&amp;author=CA+Bondy&amp;volume=135&amp;publication_year=2005&amp;pages=225-231&amp;pmid=15857685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Abe K, Chisaka O, Van Roy F, Takeichi M. Stability of dendritic spines and synaptic contacts is controlled by alpha N-catenin. <span><span class="ref-journal">Nat Neurosci. </span>2004;<span class="ref-vol">7</span>:357â€“363.</span> [<a href="/pubmed/15034585" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Stability+of+dendritic+spines+and+synaptic+contacts+is+controlled+by+alpha+N-catenin&amp;author=K+Abe&amp;author=O+Chisaka&amp;author=F+Van+Roy&amp;author=M+Takeichi&amp;volume=7&amp;publication_year=2004&amp;pages=357-363&amp;pmid=15034585&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">Uemura M, Takeichi M. Alpha N-catenin deficiency causes defects in axon migration and nuclear organization in restricted regions of the mouse brain. <span><span class="ref-journal">Dev Dyn. </span>2006;<span class="ref-vol">235</span>:2559â€“2566.</span> [<a href="/pubmed/16691566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Dyn&amp;title=Alpha+N-catenin+deficiency+causes+defects+in+axon+migration+and+nuclear+organization+in+restricted+regions+of+the+mouse+brain&amp;author=M+Uemura&amp;author=M+Takeichi&amp;volume=235&amp;publication_year=2006&amp;pages=2559-2566&amp;pmid=16691566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">Park C, Falls W, Finger JH, Longo-Guess CM, Ackerman SL. Deletion in Catna2, encoding alpha N-catenin, causes cerebellar and hippocampal lamination defects and impaired startle modulation. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">31</span>:279â€“284.</span> [<a href="/pubmed/12089526" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Deletion+in+Catna2,+encoding+alpha+N-catenin,+causes+cerebellar+and+hippocampal+lamination+defects+and+impaired+startle+modulation&amp;author=C+Park&amp;author=W+Falls&amp;author=JH+Finger&amp;author=CM+Longo-Guess&amp;author=SL+Ackerman&amp;volume=31&amp;publication_year=2002&amp;pages=279-284&amp;pmid=12089526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">Seong E, Seasholtz AF, Burmeister M. Mouse models for psychiatric disorders. <span><span class="ref-journal">Trends Genet. </span>2002;<span class="ref-vol">18</span>:643â€“650.</span> [<a href="/pubmed/12446150" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=Mouse+models+for+psychiatric+disorders&amp;author=E+Seong&amp;author=AF+Seasholtz&amp;author=M+Burmeister&amp;volume=18&amp;publication_year=2002&amp;pages=643-650&amp;pmid=12446150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Francks C, et al.  LRRTM1 on chromosome 2p12 is a maternally suppressed gene that is associated paternally with handedness and schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2007;<span class="ref-vol">12</span>:1129â€“1139.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2990633/">PMC free article</a>]</span> [<a href="/pubmed/17667961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=LRRTM1+on+chromosome+2p12+is+a+maternally+suppressed+gene+that+is+associated+paternally+with+handedness+and+schizophrenia&amp;author=C+Francks&amp;volume=12&amp;publication_year=2007&amp;pages=1129-1139&amp;pmid=17667961&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Zhang SC, Fedoroff S. Cellular localization of stem cell factor and c-kit receptor in the mouse nervous system. <span><span class="ref-journal">J Neurosci Res. </span>1997;<span class="ref-vol">47</span>:1â€“15.</span> [<a href="/pubmed/8981233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci+Res&amp;title=Cellular+localization+of+stem+cell+factor+and+c-kit+receptor+in+the+mouse+nervous+system&amp;author=SC+Zhang&amp;author=S+Fedoroff&amp;volume=47&amp;publication_year=1997&amp;pages=1-15&amp;pmid=8981233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Jensen BM, Akin C, Gilfillan AM. Pharmacological targeting of the KIT growth factor receptor: A therapeutic consideration for mast cell disorders. <span><span class="ref-journal">Br J Pharmacol. </span>2008;<span class="ref-vol">154</span>:1572â€“1582.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2518463/">PMC free article</a>]</span> [<a href="/pubmed/18500355" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Pharmacol&amp;title=Pharmacological+targeting+of+the+KIT+growth+factor+receptor:+A+therapeutic+consideration+for+mast+cell+disorders&amp;author=BM+Jensen&amp;author=C+Akin&amp;author=AM+Gilfillan&amp;volume=154&amp;publication_year=2008&amp;pages=1572-1582&amp;pmid=18500355&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">Blom T, et al.  KIT overexpression induces proliferation in astrocytes in an imatinib-responsive manner and associates with proliferation index in gliomas. <span><span class="ref-journal">Int J Cancer. </span>2008;<span class="ref-vol">123</span>:793â€“800.</span> [<a href="/pubmed/18506689" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=KIT+overexpression+induces+proliferation+in+astrocytes+in+an+imatinib-responsive+manner+and+associates+with+proliferation+index+in+gliomas&amp;author=T+Blom&amp;volume=123&amp;publication_year=2008&amp;pages=793-800&amp;pmid=18506689&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Beyenbach KW, Wieczorek H. The V-type H+ ATPase: Molecular structure andfunction, physiological roles and regulation. <span><span class="ref-journal">J Exp Biol. </span>2006;<span class="ref-vol">209</span>:577â€“589.</span> [<a href="/pubmed/16449553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Biol&amp;title=The+V-type+H++ATPase:+Molecular+structure+andfunction,+physiological+roles+and+regulation&amp;author=KW+Beyenbach&amp;author=H+Wieczorek&amp;volume=209&amp;publication_year=2006&amp;pages=577-589&amp;pmid=16449553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">Schulze TG, et al.  Two variants in Ankyrin 3 (<em>ANK3</em>) are independent genetic risk factors for bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2008 doi: 10.1038/mp.2008.134.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2793269/">PMC free article</a>]</span> [<a href="/pubmed/19088739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fmp.2008.134" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Two+variants+in+Ankyrin+3+(ANK3)+are+independent+genetic+risk+factors+for+bipolar+disorder&amp;author=TG+Schulze&amp;publication_year=2008&amp;doi=10.1038/mp.2008.134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Malhotra JD, Kazen-Gillespie K, Hortsch M, Isom LL. Sodium channel beta subunits mediate homophilic cell adhesion and recruit ankyrin to points of cellâ€“cell contact. <span><span class="ref-journal">J Biol Chem. </span>2000;<span class="ref-vol">275</span>:11383â€“11388.</span> [<a href="/pubmed/10753953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Sodium+channel+beta+subunits+mediate+homophilic+cell+adhesion+and+recruit+ankyrin+to+points+of+cellâ€“cell+contact&amp;author=JD+Malhotra&amp;author=K+Kazen-Gillespie&amp;author=M+Hortsch&amp;author=LL+Isom&amp;volume=275&amp;publication_year=2000&amp;pages=11383-11388&amp;pmid=10753953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">Pan Z, et al.  A common ankyrin-G-based mechanism retains KCNQ and NaV channels at electrically active domains of the axon. <span><span class="ref-journal">J Neurosci. </span>2006;<span class="ref-vol">26</span>:2599â€“2613.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6675151/">PMC free article</a>]</span> [<a href="/pubmed/16525039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=A+common+ankyrin-G-based+mechanism+retains+KCNQ+and+NaV+channels+at+electrically+active+domains+of+the+axon&amp;author=Z+Pan&amp;volume=26&amp;publication_year=2006&amp;pages=2599-2613&amp;pmid=16525039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">Bennett V, Chen L. Ankyrins and cellular targeting of diverse membrane proteins to physiological sites. <span><span class="ref-journal">Curr Opin Cell Biol. </span>2001;<span class="ref-vol">13</span>:61â€“67.</span> [<a href="/pubmed/11163135" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Cell+Biol&amp;title=Ankyrins+and+cellular+targeting+of+diverse+membrane+proteins+to+physiological+sites&amp;author=V+Bennett&amp;author=L+Chen&amp;volume=13&amp;publication_year=2001&amp;pages=61-67&amp;pmid=11163135&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="element-citation">Klein RJ, et al.  Complement factor H polymorphism in age-related macular degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:385â€“389.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1512523/">PMC free article</a>]</span> [<a href="/pubmed/15761122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+polymorphism+in+age-related+macular+degeneration&amp;author=RJ+Klein&amp;volume=308&amp;publication_year=2005&amp;pages=385-389&amp;pmid=15761122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="element-citation">Zeggini, et al.  Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:638â€“645.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=+Zeggini&amp;volume=40&amp;publication_year=2008&amp;pages=638-645&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="element-citation">Nurnberger JI, Jr, et al.  Genomic survey of bipolar illness in the NIMH genetics initiative pedigrees: A preliminary report. <span><span class="ref-journal">Am J Med Genet. </span>1997;<span class="ref-vol">74</span>:227â€“237.</span> [<a href="/pubmed/9184304" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Genomic+survey+of+bipolar+illness+in+the+NIMH+genetics+initiative+pedigrees:+A+preliminary+report&amp;author=JI+Nurnberger&amp;volume=74&amp;publication_year=1997&amp;pages=227-237&amp;pmid=9184304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="element-citation">Dick DM, et al.  Genomewide linkage analyses of bipolar disorder: A new sample of 250 pedigrees from the National Institute of Mental Health Genetics Initiative. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">73</span>:107â€“114.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180573/">PMC free article</a>]</span> [<a href="/pubmed/12772088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genomewide+linkage+analyses+of+bipolar+disorder:+A+new+sample+of+250+pedigrees+from+the+National+Institute+of+Mental+Health+Genetics+Initiative&amp;author=DM+Dick&amp;volume=73&amp;publication_year=2003&amp;pages=107-114&amp;pmid=12772088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="element-citation">Nurnberger JI, Jr, et al.  Diagnostic interview for genetic studies. Rationale, unique features, and training. <span><span class="ref-journal">NIMH Gen Initiative Arch Gen Psych. </span>1994;<span class="ref-vol">51</span>:849â€“859.</span> [<a href="/pubmed/7944874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NIMH+Gen+Initiative+Arch+Gen+Psych&amp;title=Diagnostic+interview+for+genetic+studies.+Rationale,+unique+features,+and+training&amp;author=JI+Nurnberger&amp;volume=51&amp;publication_year=1994&amp;pages=849-859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="element-citation">Maxwell ME.  <span class="ref-journal">Family Interview for Genetic Studies (FIGS): A Manual for FIGS, Clinical Neurogenetics Branch, Intramural Research Program, National Institute of Mental Health, Bethesda, Maryland.</span> Bethesda, MD: NIMH; 1992.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Family+Interview+for+Genetic+Studies+(FIGS):+A+Manual+for+FIGS,+Clinical+Neurogenetics+Branch,+Intramural+Research+Program,+National+Institute+of+Mental+Health,+Bethesda,+Maryland&amp;author=ME+Maxwell&amp;publication_year=1992&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="element-citation">Celik C.  Geneva: WHO; 2003. Computer assisted personal interviewing application for the Schedules for Clinical Assessment in Neuropsychiatry Version 2.1 and diagnostic algorithms for WHO ICD 10 chapter V DCR and for statistical manual IV. Release I. Ed. 1.0.3.5. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Celik+C+Computer+assisted+personal+interviewing+application+for+the+Schedules+for+Clinical+Assessment+in+Neuropsychiatry+Version+2.1+and+diagnostic+algorithms+for+WHO+ICD+10+chapter+V+DCR+and+for+statistical+manual+IV+2003+Geneva+WHO+Release+I.+Ed.+1.0.3.5+" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="element-citation">Fellay J, et al.  A whole-genome association study of major determinants for host control of HIV-1. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>:944â€“947.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1991296/">PMC free article</a>]</span> [<a href="/pubmed/17641165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+whole-genome+association+study+of+major+determinants+for+host+control+of+HIV-1&amp;author=J+Fellay&amp;volume=317&amp;publication_year=2007&amp;pages=944-947&amp;pmid=17641165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904â€“909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="element-citation">White H. Maximum likelihood estimation of misspecified model. <span><span class="ref-journal">Econometrica. </span>1982;<span class="ref-vol">50</span>:1â€“25.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Econometrica&amp;title=Maximum+likelihood+estimation+of+misspecified+model&amp;author=H+White&amp;volume=50&amp;publication_year=1982&amp;pages=1-25&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2678639&amp;issue-id=178666&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140570782102272title">Abstract</a></li><li><a href="#__sec1title">Results</a></li><li><a href="#__sec2title">Discussion</a></li><li><a href="#__sec3title">Methods</a></li><li><a href="#__sec16title">Supplementary Material</a></li><li><a href="#idm140570776864768title">Acknowledgments.</a></li><li><a href="#idm140570776868160title">Footnotes</a></li><li><a href="#idm140570777162352title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2678639/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2678639/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2678639/pdf/zpq7501.pdf">PDF (423K)</a></li> | <li><a href="#" data-citationid="PMC2678639" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2678639%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2678639%2F&amp;text=Genome-wide%20association%20and%20meta-analysis%20of%20bipolar%20disorder%20in%20individuals%20of%20European%20ancestry" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2678639%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19416921" data-aiid="2678639" data-aid="2678639" data-iid="178666" data-domainid="2" data-domain="pnas" data-accid="PMC2678639" data-md5="37e2e859ae2c1fb15b73da05f8cf9c8e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:37:26-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A82A0DD4EF2F100000000071E04C6&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2678639%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2678639/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>