<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="genores"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2963812/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Radiation pharmacogenomics: A genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2963812/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Genome Research"><meta name="citation_title" content="Radiation pharmacogenomics: A genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines"><meta name="citation_authors" content="Nifang Niu, Yuxin Qin, Brooke L. Fridley, Junmei Hou, Krishna R. Kalari, Minjia Zhu, Tse-Yu Wu, Gregory D. Jenkins, Anthony Batzler, Liewei Wang"><meta name="citation_date" content="November 2010"><meta name="citation_issue" content="11"><meta name="citation_volume" content="20"><meta name="citation_firstpage" content="1482"><meta name="citation_doi" content="10.1101/gr.107672.110"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2963812/?report=abstract"><meta name="citation_pmid" content="20923822"><meta name="DC.Title" content="Radiation pharmacogenomics: A genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"><meta name="DC.Contributor" content="Nifang Niu"><meta name="DC.Contributor" content="Yuxin Qin"><meta name="DC.Contributor" content="Brooke L. Fridley"><meta name="DC.Contributor" content="Junmei Hou"><meta name="DC.Contributor" content="Krishna R. Kalari"><meta name="DC.Contributor" content="Minjia Zhu"><meta name="DC.Contributor" content="Tse-Yu Wu"><meta name="DC.Contributor" content="Gregory D. Jenkins"><meta name="DC.Contributor" content="Anthony Batzler"><meta name="DC.Contributor" content="Liewei Wang"><meta name="DC.Date" content="2010 Nov"><meta name="DC.Identifier" content="10.1101/gr.107672.110"><meta name="DC.Language" content="en"><meta property="og:title" content="Radiation pharmacogenomics: A genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines"><meta property="og:type" content="article"><meta property="og:description" content="Radiation therapy is used to treat half of all cancer patients. Response to radiation therapy varies widely among patients. Therefore, we performed a genome-wide association study (GWAS) to identify biomarkers to help predict radiation response using ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2963812/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-genores.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2963812/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2963812/pdf/1482.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B6C88DD513FC10000000005DB03BC.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2963812%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="clinvar">ClinVar</option><option value="nlmcatalog" class="last">NLM Catalog</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/102/">Genome Res</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/191029/">v.20(11); 2010 Nov</a></li><li class="accid">PMC2963812</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-genores.gif" alt="Logo of genores" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,0,500,57" alt="Genome Research" title="Genome Research" href="http://genome.cshlp.org/" ref="reftype=publisher&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,58,94,75" alt="CSHL Press" title="CSHL Press" href="http://www.cshlpress.com/" ref="reftype=publisher&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="95,58,196,75" alt="Journal Home" title="Journal Home" href="http://genome.cshlp.org/" ref="reftype=publisher&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="197,58,298,75" alt="Subscriptions" title="Subscriptions" href="http://genome.cshlp.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="299,58,390,75" alt="eTOC Alerts" title="eTOC Alerts" href="http://genome.cshlp.org/site/misc/etoc.xhtml" ref="reftype=publisher&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="391,58,500,75" alt="BioSupplyNet" title="BioSupplyNet" href="http://www.biosupplynet.com/" ref="reftype=publisher&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Genome Res</a></span>. 2010 Nov; 20(11): 1482–1492. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1101%2Fgr.107672.110" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1101/gr.107672.110</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2963812</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20923822">20923822</a></div></div></div></div><h1 class="content-title">Radiation pharmacogenomics: A genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Niu%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386671457648" co-class="co-affbox">Nifang Niu</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Qin%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386680411104" co-class="co-affbox">Yuxin Qin</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Fridley%20BL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386679473312" co-class="co-affbox">Brooke L. Fridley</a>,<sup>2</sup> <a href="/pubmed/?term=Hou%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386713194768" co-class="co-affbox">Junmei Hou</a>,<sup>1</sup> <a href="/pubmed/?term=Kalari%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386713192544" co-class="co-affbox">Krishna R. Kalari</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Zhu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386751266592" co-class="co-affbox">Minjia Zhu</a>,<sup>1,</sup><sup>4</sup> <a href="/pubmed/?term=Wu%20TY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386751263632" co-class="co-affbox">Tse-Yu Wu</a>,<sup>1</sup> <a href="/pubmed/?term=Jenkins%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386670042192" co-class="co-affbox">Gregory D. Jenkins</a>,<sup>2</sup> <a href="/pubmed/?term=Batzler%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386670039968" co-class="co-affbox">Anthony Batzler</a>,<sup>2</sup> and  <a href="/pubmed/?term=Wang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822" class="affpopup" co-rid="_co_idm140386678215392" co-class="co-affbox">Liewei Wang</a><sup>1,</sup><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140386671457648"><h3 class="no_margin">Nifang Niu</h3><p><sup>1</sup> Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, Minnesota 55905, USA;</p><div>Find articles by <a href="/pubmed/?term=Niu%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Nifang Niu</a></div></div><div id="_co_idm140386680411104"><h3 class="no_margin">Yuxin Qin</h3><p><sup>1</sup> Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, Minnesota 55905, USA;</p><div>Find articles by <a href="/pubmed/?term=Qin%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Yuxin Qin</a></div></div><div id="_co_idm140386679473312"><h3 class="no_margin">Brooke L. Fridley</h3><p><sup>2</sup> Department of Health Sciences Research, Mayo Clinic, Rochester, Minnesota 55905, USA</p><div>Find articles by <a href="/pubmed/?term=Fridley%20BL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Brooke L. Fridley</a></div></div><div id="_co_idm140386713194768"><h3 class="no_margin">Junmei Hou</h3><p><sup>1</sup> Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, Minnesota 55905, USA;</p><div>Find articles by <a href="/pubmed/?term=Hou%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Junmei Hou</a></div></div><div id="_co_idm140386713192544"><h3 class="no_margin">Krishna R. Kalari</h3><p><sup>1</sup> Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, Minnesota 55905, USA;</p><p><sup>2</sup> Department of Health Sciences Research, Mayo Clinic, Rochester, Minnesota 55905, USA</p><div>Find articles by <a href="/pubmed/?term=Kalari%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Krishna R. Kalari</a></div></div><div id="_co_idm140386751266592"><h3 class="no_margin">Minjia Zhu</h3><p><sup>1</sup> Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, Minnesota 55905, USA;</p><div>Find articles by <a href="/pubmed/?term=Zhu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Minjia Zhu</a></div></div><div id="_co_idm140386751263632"><h3 class="no_margin">Tse-Yu Wu</h3><p><sup>1</sup> Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, Minnesota 55905, USA;</p><div>Find articles by <a href="/pubmed/?term=Wu%20TY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Tse-Yu Wu</a></div></div><div id="_co_idm140386670042192"><h3 class="no_margin">Gregory D. Jenkins</h3><p><sup>2</sup> Department of Health Sciences Research, Mayo Clinic, Rochester, Minnesota 55905, USA</p><div>Find articles by <a href="/pubmed/?term=Jenkins%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Gregory D. Jenkins</a></div></div><div id="_co_idm140386670039968"><h3 class="no_margin">Anthony Batzler</h3><p><sup>2</sup> Department of Health Sciences Research, Mayo Clinic, Rochester, Minnesota 55905, USA</p><div>Find articles by <a href="/pubmed/?term=Batzler%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Anthony Batzler</a></div></div><div id="_co_idm140386678215392"><h3 class="no_margin">Liewei Wang</h3><p><sup>1</sup> Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, Minnesota 55905, USA;</p><div>Find articles by <a href="/pubmed/?term=Wang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20923822">Liewei Wang</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140386677947264_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140386677947264_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140386677947264_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140386677947264_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup> Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, Minnesota 55905, USA;</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup> Department of Health Sciences Research, Mayo Clinic, Rochester, Minnesota 55905, USA</div><div id="fn1"><sup>3</sup>These authors contributed equally to this work.</div><div id="fn2"><sup>4</sup>Present address: Department of Physiology, West China School of Preclinical and Forensic Medicine, Sichuan University, Chendu 610041, China.</div><div id="cor1"><sup>5</sup>Corresponding author.E-mail <a href="mailto:dev@null" data-email="ude.oyam@ieweil.gnaw" class="oemail">ude.oyam@ieweil.gnaw</a>; fax (507) 284-4455.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140386677947264_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Mar 12; Accepted 2010 Jul 15.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140386677947264_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2010 by Cold Spring Harbor Laboratory Press</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2963812/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140386676570144" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140386676570144title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">Radiation therapy is used to treat half of all cancer patients. Response to radiation therapy varies widely among patients. Therefore, we performed a genome-wide association study (GWAS) to identify biomarkers to help predict radiation response using 277 ethnically defined human lymphoblastoid cell lines (LCLs). Basal gene expression levels and 1.3 million genome-wide single nucleotide polymorphism (SNP) markers from both Affymetrix and Illumina platforms were assayed for all 277 human LCLs. MTS [3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium] assays for radiation cytotoxicity were also performed to obtain area under the curve (AUC) as a radiation response phenotype for use in the association studies. Functional validation of candidate genes, selected from an integrated analysis that used SNP, expression, and AUC data, was performed with multiple cancer cell lines using specific siRNA knockdown, followed by MTS and colony-forming assays. A total of 27 loci, each containing at least two SNPs within 50 kb with <em>P</em>-values less than 10<sup>−4</sup> were associated with radiation AUC. A total of 270 expression probe sets were associated with radiation AUC with <em>P</em> &lt; 10<sup>−3</sup>. The integrated analysis identified 50 SNPs in 14 of the 27 loci that were associated with both AUC and the expression of 39 genes, which were also associated with radiation AUC (<em>P</em> &lt; 10<sup>−3</sup>). Functional validation using siRNA knockdown in multiple tumor cell lines showed that <em>C13orf34</em>, <em>MAD2L1</em>, <em>PLK4</em>, <em>TPD52</em>, and <em>DEPDC1B</em> each significantly altered radiation sensitivity in at least two cancer cell lines. Studies performed with LCLs can help to identify novel biomarkers that might contribute to variation in response to radiation therapy and enhance our understanding of mechanisms underlying that variation.</p></div></div><div id="idm140386670004720" class="tsec sec headless whole_rhythm"><p id="__p4" class="p p-first">Radiation is widely used in the treatment of cancer. Half of all cancer patients are treated with radiation, either alone or in combination with other forms of therapy (<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951384">Burnet et al. 2006</a>). However, response to radiation therapy varies widely, ranging from complete eradication of the tumor to severe short- and long-term treatment-dependent adverse reactions in normal tissue (<a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951385">Bentzen 2006</a>; <a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951336">Gerber and Chan 2008</a>). Several clinical factors are known to influence radiation response, including radiation dose, volume, and fraction, but it is also known that genetic inheritance can play an important role in variation in radiation response (<a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951354">Barnett et al. 2009</a>). Many genotyping studies have been performed in an attempt to identify biomarkers for the prediction of radiation response by studying candidate genes known to be involved in radiation effect (<a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951353">Chistiakov et al. 2008</a>; <a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951320">Andreassen and Alsner 2009</a>; <a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951335">Popanda et al. 2009</a>; <a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951365">Pugh et al. 2009</a>). In addition, expression quantitative trait locus (eQTL) studies performed using post-radiation gene expression profiles have identified single nucleotide polymorphisms (SNPs) associated with radiation response through their influence on gene expression (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951341">Correa and Cheung 2004</a>; <a href="#B50" rid="B50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206340664">Smirnov et al. 2009</a>). The results of all of these studies suggest that genetic variation plays an important role in individual variation in radiation response.</p><p id="__p5">In this study, we have used approaches previously applied in pharmacogenomic studies involving genome-wide basal gene expression profiles plus genome-wide SNPs for 277 human lymphoblastoid cell lines (LCLs) to identify SNPs/genes that might contribute to variation in radiation response and also to understand the biology underlying the association by performing functional studies of the candidates that we identified. Specifically, we obtained basal gene expression data for all 277 cell lines using Affymetrix U133 plus 2.0 Gene Chips, as well as genome-wide SNP data using Illumina HumanHap 550K and 510S BeadChips together with publicly available Affymetrix SNP Array 6.0 SNP data, resulting in a total of over 1.3 million SNPs per cell line for analysis. We next performed radiation cytotoxicity assays with the same LCLs to obtain the radiation area under the curve (AUC) as an in vitro radiation response phenotype. We then performed a genome-wide association study using the 1.3 million SNPs, basal gene expression array data, and AUC as a radiation response phenotype. We identified 27 loci, defined as at least two SNPs within 50 kb having <em>P</em>-values &lt; 10<sup>−4</sup>, which were associated with radiation AUC. We also identified genes with expression levels that were associated with radiation AUC with <em>P</em> &lt; 10<sup>−3</sup>. By then performing an integrated analysis of SNPs, gene expression, and radiation AUC, we selected a total of 23 candidate genes to perform siRNA knockdown with multiple tumor cell lines, followed by MTS and colony-forming functional validation assays to identify genes that influenced radiation exposure sensitivity.</p><p id="__p6" class="p">In summary, the series of experiments described subsequently represents the application of genome-wide expression and SNP data from a cell line-based model system to identify genes associated with radiation sensitivity. Those genes, selected based on the association of radiation AUC with SNPs and/or expression, were then validated functionally. Since our major purpose was to identify genes that might provide novel insights into the biology of variation in response to radiation treatment to insure the largest set of genes for validation, the initial selection of genes to be followed up was based on a liberal significance threshold and not the standard genome-wide significance level of 10<sup>−7</sup>. The functional validation involved siRNA gene knockdown performed with cancer cell lines, followed by MTS cytotoxicity and colony-forming assays. The goals were to identify and functionally validate biomarkers for radiation response and to identify novel mechanisms that might contribute to radiation sensitivity.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Results</h2><div id="__sec2" class="sec sec-first"><h3 id="__sec2title">Radiation cytotoxicity</h3><p id="__p7" class="p p-first">Radiation cytotoxicity studies were performed to determine the range of variation in radiation AUC among the individual cell lines studied. <a href="/pmc/articles/PMC2963812/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a> shows representative radiation cytotoxicity data for a set of cell lines. AUC values differed significantly among the three racial groups studied, with cells from Han Chinese–American (HCA) subjects appearing to be more sensitive to radiation than were those of the cells from Caucasian–American (CA) subjects (<em>P</em> = 0.007; <a href="/pmc/articles/PMC2963812/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1B</span></a>). Gender did not have a significant effect on AUC (<em>P</em> = 0.125; <a href="/pmc/articles/PMC2963812/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1C</span></a>). The distribution of AUC values was skewed, with a median AUC value of 3.17. However, the distribution of values for the natural-log (ln)-transformed AUC was nearly unimodal and symmetric (i.e., “normal” in shape) with a mean value of 1.27 and a range (±2 SD of the mean) of 0.28 to 2.26 (<a href="/pmc/articles/PMC2963812/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1D</span></a>). We also performed a biological replication study for the cytotoxicity data. Specifically, cytotoxicity assays were repeated 1 yr after the initial studies for 19 randomly selected cell lines. The results obtained at the two different times were significantly correlated (Pearson correlation <em>r</em> = 0.51, <em>P</em> = 0.026).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2963812/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140386672194192" class="figure"><a href="/pmc/articles/PMC2963812/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2963812_1482fig1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1482fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2963812/bin/1482fig1.jpg"></a></div><div id="largeobj_idm140386672194192" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2963812/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2963812/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p8">Radiation cytotoxicity. (<em>A</em>) Representative radiation cytotoxicity dose-response curves. Two cell lines from each of the three ethnic groups studied were selected to illustrate a range of radiation cytotoxicity “dose response” curves. Squares indicate African-Americans (AA), triangles Caucasian-Americans (CA), and circles indicate Han Chinese–Americans (HCA). The <em>x</em>-axis indicates radiation dose, and the <em>y</em>-axis indicates the surviving fraction after radiation exposure. (<em>B</em>) Relationship of ethnic group to radiation AUC. (<em>C</em>) Gender effect on radiation AUC. (<em>D</em>) Frequency distribution histogram of natural-log (ln) AUC values for 277 cell lines.</p></div></div></div></div><div id="__sec3" class="sec"><h3 id="__sec3title">Correlation between expression and AUC</h3><p id="__p9" class="p p-first">We next performed correlation analysis for the association of expression array and radiation AUC data to identify genes with expression levels that might be associated with radiation AUC (<a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>). The association analysis identified 50 expression probe sets that were associated with radiation AUC with <em>P</em>-values &lt; 10<sup>−4</sup> (<em>Q</em>-values &lt; 0.096), and 270 individual probe sets with <em>P</em>-values &lt; 10<sup>−3</sup> (<em>Q</em>-values &lt; 0.182). These 270 expression probe sets represented 211 annotated genes, but only one of the probe sets remained significant after Bonferroni correction for multiple testing (<em>P</em> &lt; 0.0002). The top 20 probe sets are listed in <a href="/pmc/articles/PMC2963812/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, and the entire list of 270 expression probe sets is found in Supplemental Table 1.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p10">The top 20 expression probe sets associated with radiation cytotoxicity (AUC values)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140386750296496" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2963812_1482tbl1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1482tbl1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2963812/bin/1482tbl1.jpg"></a></div><div id="largeobj_idm140386750296496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2963812/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140386750295280"><p id="__p11" class="p p-first-last"><sup>a</sup><em>R</em>-values represent correlation coefficients for the association.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140386713169504" class="figure"><a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2963812_1482fig2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1482fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2963812/bin/1482fig2.jpg"></a></div><div id="largeobj_idm140386713169504" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2963812/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2963812/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p12">Genome-wide associations with radiation AUC. (<em>A</em>) Association of basal expression with radiation AUC for 277 cell lines. The <em>y</em>-axis represents the –log<sub>10</sub>(<em>P</em>-value) for the association of individual expression array probe sets. Expression probe sets are plotted on the <em>x</em>-axis based on the chromosomal locations of their genes. If genes had more than one probe set, the one with the lowest <em>P</em>-value was plotted. A <em>P</em>-value of 10<sup>−3</sup> is highlighted with a red line. (<em>B</em>) Genome-wide SNP association with radiation AUC for 277 cell lines. The <em>y</em>-axis represents −log<sub>10</sub>(<em>P</em>-value) for the association of each SNP with radiation AUC. SNPs are plotted on the <em>x</em>-axis based on their chromosomal locations. A <em>P</em>-value of 10<sup>−4</sup> is highlighted with a red line. (<em>C</em>) This panel shows the most significant locus on chromosome 8 that was associated with radiation AUC. Blue diamonds indicate SNPs observed by genotyping, while red triangles indicate imputed SNPs. The <em>y</em>-axis represents –log<sub>10</sub>(<em>P</em>-value) for the association of each SNP with radiation AUC, and the <em>x</em>-axis represents the location of the SNP on chromosome 8.</p></div></div></div></div><div id="__sec4" class="sec"><h3 id="__sec4title">Genome-wide SNP association with radiation AUC</h3><p id="__p13" class="p p-first">We also performed an analysis of the association of genome-wide SNPs with radiation AUC (<a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>). A total of 561,298 SNPs on the Illumina 550K SNP array and 493,750 SNPs on the Illumina 510S SNP array had been genotyped using DNA from each of these 277 cell lines. We also had access to publically available Affymetrix 6.0 SNP array data. We performed quality control (QC) for all of these SNPs prior to performing the statistical analysis. Specifically, for data obtained with the Illumina 550K array, we removed 12,261 SNPs that had call rates &lt;95%, 32,550 SNPs with minor allele frequencies (MAFs) &lt;5%, and 4676 SNPs that deviated from Hardy-Weinberg equilibrium (HWE), using a stringent threshold of <em>P</em> &lt; 0.001. Therefore, 511,811 Illumina 550K SNPs were used in the genome-wide SNP analysis. A similar approach was used for the QC analysis of SNPs on the Illumina 510S platform. After removing 10,353 SNPs with call rates &lt;95%, 147,027 SNPs with a MAF &lt;5%, and 3805 SNPs that deviated from HWE (<em>P</em> &lt; 0.001), a total of 332,565 Illumina 510S SNPs remained for analysis. For the publicly available Affymetrix 6.0 SNPs, we first removed SNPs that had already been genotyped using the Illumina platforms, resulting in 643,600 unique SNPs on the Affymetrix 6.0 SNP array for which we performed the QC analysis. After removing 26,140 SNPs with call rates &lt;95%, 107,275 SNPs with a MAF &lt;5%, and 5763 that deviated from HWE (<em>P</em> &lt; 0.001), 504,422 remained. Therefore, after combining data from the two platforms, 1,348,798 SNPs were available for use in the analysis (<a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>).</p><p id="__p14">The <em>P</em>-value for the most significant SNP, Affymetrix marker SNP_A-8538282 (rs7000734), was 3.82 × 10<sup>−7</sup> (<em>r</em> = 0.309, MAF = 0.081). The top 16 SNPs, all with <em>P</em>-values &lt; 10<sup>−5</sup> are listed in <a href="/pmc/articles/PMC2963812/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, and the 151 SNPs that were associated with AUC with <em>P</em>-values &lt; 10<sup>−4</sup> are listed in Supplemental Table 2. These 151 SNPs were located within or close to 99 unique genes on the basis of the annotation of genome build 36.3. Among the 151 top SNPs, three were within coding regions, 45 within introns, and 36 and 68 were within 5′- or 3′-untranslated regions (UTRs) or flanking regions, respectively.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p15">The top 16 SNPs that were associated with radiation AUC with <em>P</em>-values &lt; 10<sup>−5</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140386680859328" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2963812_1482tbl2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1482tbl2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2963812/bin/1482tbl2.jpg"></a></div><div id="largeobj_idm140386680859328" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2963812/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140386680858112"><p id="__p16" class="p p-first-last"><sup>a</sup><em>R</em>-values represent correlation coefficients for the association.</p></div></div></div><p id="__p17" class="p p-last">For our integrated analyses of SNPs with both expression array and cytotoxicity data, we began with 1335 SNPs that had <em>P</em>-values &lt; 10<sup>−3</sup> for association with radiation AUC. As stated previously, we used these less stringent criteria to identify more possible candidates that, even though they did not reach genome-wide significance, might have functional impact on radiation sensitivity. When we determined the locations of these 1335 SNPs, we were able to identify 27 regions/loci that contained at least two SNPs with <em>P</em>-values &lt; 10<sup>−4</sup> within 50 kb in each of these regions (a total of 175 SNPs). For purposes of this analysis, we defined these regions as a locus or a “SNP peak region.” All 27 of these loci, and the SNPs within each locus, are listed in Supplemental Table 3. We focused on these loci for the integrated analyses. The most significant locus, locus 8C, mapped to chromosome 8 (<a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>; Supplemental Table 3). This region had six SNPs within 50 kb with <em>P</em>-values &lt; 10<sup>−4</sup>, including the SNP with the lowest <em>P</em>-value, 3.82 × 10<sup>−7</sup>. The gene closest to this region encoded <em>PLEKHF2</em>. All six of the SNPs were in tight linkage disequilibrium (LD), with <em>r</em><sup>2</sup> values that ranged from 0.7 to 1.0 in Caucasian-Americans (CAs), with slightly lower values in the African-American (AA) subjects (Supplemental Fig. 1). The allele frequencies for these SNPs were also higher in CA and AA subjects (MAF ≥ 0.065) than in Han Chinese–American (HCA) subjects (MAF ≤ 0.021). We also imputed SNPs not on the genotyping platforms, using the HapMap populations as the reference, that were located 200 kb up or downstream from SNP_A-8538282, the most significant SNP on chromosome 8 (see <a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>). That area contained 222 “observed SNPs” that were on the combined Illumina and Affymetrix platforms, plus 104 SNP markers that we imputed (<a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>). None of the imputed SNPs was more significant than the genotyped SNP_A-8538282.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Integrated SNP, basal expression, and radiation AUC analyses</h3><p id="__p18" class="p p-first">The effect of genetic variation on radiation-induced cytotoxicity might result, in part, from the regulation of gene expression. Previous studies performed with lymphoblastoid cell lines have shown that post-radiation gene expression is highly influenced by DNA sequence variation (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951338">Correa and Cheung 2004</a>; <a href="#B50" rid="B50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206340665">Smirnov et al. 2009</a>). However, few studies have focused on basal gene expression levels and their possible relationship to radiation response, i.e., on information that might be used to predict response. Therefore, we also performed an “integrated analysis” that included data for SNPs, basal expression, and radiation AUC. Specifically, we identified 175 SNPs with <em>P</em>-values &lt; 10<sup>−3</sup> that mapped to the 27 loci that we had identified (Supplemental Table 3), and then used data for the 54,000 basal expression array probe sets on the Affymetrix U133 Plus 2.0 platform to identify SNPs within those loci that might be associated with basal gene expression, in either a <em>cis</em> or <em>trans</em> relationship.</p><p id="__p19" class="p p-last">Specifically, we observed 2432 SNP expression associations for the 175 SNPs with <em>P</em>-values &lt; 10<sup>−4</sup>. We then correlated those 2432 expression probe sets with radiation AUC and identified probe sets with <em>P</em>-values &lt; 10<sup>−3</sup> for association with radiation AUC. We selected a less-stringent <em>P</em>-value cutoff to capture as much information as possible, realizing that many of the associations would be false-positives. This integrated analysis, moving from loci to SNP markers to expression, identified 50 unique SNPs located in 14 of the 27 loci that were associated with data for 47 probe sets, which represented 39 unique annotated genes, i.e., basal expression of these genes was associated with radiation AUC with <em>P</em> &lt; 10<sup>−3</sup> (Supplemental Table 4). None of the SNPs were in <em>cis</em>-regulatory regions, defined as 5 Mb on either side of the gene identified. These 50 unique SNPs mapped to eight different chromosomes, with at least two SNPs on each of those chromosomes. The four most significant loci or “SNP peak regions” mapped to chromosomes 1, 4, 5, and 8, respectively, and contained the <em>LRRN2</em>, <em>IL19</em>, <em>KCNK1</em>, <em>LDB2</em>, <em>NCRNA00290</em> (also known as <em>LOC728081</em>), <em>DEPDC1B</em>, <em>PPIGP1</em> (also known as <em>LOC100131033</em>), and <em>PLEKHF2</em> genes (Supplemental Table 4). The SNPs near <em>PLEKHF2</em> within the locus on chromosome 8 (<a href="/pmc/articles/PMC2963812/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>) were particularly striking since they were associated with variation in the expression of six annotated genes, and variation in the expression of those genes was, in turn, significantly associated with radiation AUC. The chromosome 1 locus contained the largest number of SNPs, 19, that were associated with radiation AUC, with <em>P</em>-values that ranged from 10<sup>−3</sup> to 10<sup>−4</sup>. Those 19 SNPs were associated with the expression of 12 annotated genes that were also associated with radiation AUC with <em>P</em>-values that ranged from 10<sup>−3</sup> to 10<sup>−4</sup>. Six annotated genes were associated with the locus on chromosome 8 that contained the most significant six linked SNPs (<em>P</em>-values &lt; 10<sup>−4</sup>), and those six genes were associated with radiation AUC, with <em>P</em>-values &lt; 10<sup>−3</sup>. The chromosomes 4 and 5 loci included seven and six SNPs, respectively, that were associated with radiation AUC with <em>P</em>-values that ranged from 10<sup>−3</sup> to 10<sup>−4</sup>, and those SNPs were associated with the expression of 11 and four unique annotated genes, respectively, with <em>P</em>-values that ranged from 10<sup>−4</sup> to 10<sup>−7</sup>. Expression levels for those 15 genes were also associated with radiation AUC, with <em>P</em>-values &lt; 10<sup>−3</sup>.</p></div><div id="__sec6" class="sec sec-last"><h3 id="__sec6title">Functional validation of candidate genes in tumor cell lines</h3><p id="__p20" class="p p-first">Our initial association experiments were performed with human LCLs. Since nongenetic factors might confound the results of these association studies, and since gene regulation is tissue specific (<a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951362">Dimas et al. 2009</a>), we next turned to human tumor cell lines, two pancreatic cancer cell lines, MIA-PaCa2 (TP53 mutant), and HupT3 (TP53 mutant), and one cervical cancer cell line, HeLa, which has <em>TP53</em> wild-type (WT) sequence, but for which the function of TP53 is disrupted by the expression of human papillomaviruse (HPV) E6 (<a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951344">DeFilippis et al. 2003</a>), to functionally validate association results obtained with LCLs. These functional experiments involved siRNA knockdown, followed by MTS cytotoxicity assays and, subsequently, colony-forming assays. We chose these three cancer cell lines because of their relative sensitivity to radiation after testing with MTS assays.</p><p id="__p21">Based on our analysis of the 54,000 basal expression probe sets, 1.3 million SNPs and radiation AUC data, as well as an evaluation of their biological function, we selected 23 candidate genes identified as described in the preceding paragraphs for siRNA screening performed with multiple tumor cell lines. We choose these genes based on their potential biological significance, as well as their expression levels in LCL, which has to be greater than 50 after GCRMA normalization. These genes were chosen based on the following criteria: (1) genes with at least one expression array probe set that had a <em>P</em>-value &lt; 5 × 10<sup>−4</sup> or at least two probe sets with <em>P</em>-value &lt; 10<sup>−3</sup> for association with radiation AUC, which identified 96 genes expressed in the LCLs. We selected 18 “less studied” genes in terms of their relationship to radiation response from 96 genes for functional studies, since most of them were known to relate to pathways that are important for radiation response, such as DNA repair, cell cycle, cell survival, and apoptosis pathways; (2) genes containing SNPs found within a locus associated with radiation AUC (<em>P</em> &lt; 10<sup>−4</sup>), which identified 10 genes out of the 27 SNP loci (29 genes) expressed in the LCLs. We selected four genes from the four most significant SNP loci; (3) genes for which expression was associated with both AUC and SNPs (<em>P</em> &lt; 10<sup>−3</sup> for AUC, and <em>P</em> &lt; 10<sup>−4</sup> for SNPs, i.e., the integrated analysis), which identified 30 genes expressed in the LCLs. We selected seven genes for further functional studies, of which three genes were associated with the three most significant SNP loci, and the other four genes were associated with either at least one SNP with a <em>P</em>-value &lt; 10<sup>−5</sup> or at least three SNPs with <em>P</em>-values &lt; 10<sup>−4</sup>. This selection process resulted in 23 genes after removing redundant genes among different selection categories. This overall selection strategy is depicted graphically in <a href="/pmc/articles/PMC2963812/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2963812/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140386677553360" class="figure"><a href="/pmc/articles/PMC2963812/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2963812_1482fig3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1482fig3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2963812/bin/1482fig3.jpg"></a></div><div id="largeobj_idm140386677553360" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2963812/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2963812/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><p id="__p22">Schematic diagram of the strategy used to select genes for functional validation. Genome-wide association studies for radiation AUC were performed with 1.3 million SNPs or 54,000 expression probe sets. A SNP locus–Expression–AUC integrated analysis was performed using loci that contained at least two SNPs associated with radiation AUC with <em>P</em>-values &lt;10<sup>−4</sup>, 54,000 expression probe sets and radiation AUC associations were used to identify SNPs associated with radiation AUC through their influence on gene expression (SNP-expression <em>P</em>-value &lt; 10<sup>−4</sup>, expression-AUC <em>P</em>-value &lt; 10<sup>−3</sup>). We excluded genes that had expression levels less than 50 and have previously been known to be involved in pathways related to radiation sensitivity. Finally, 23 candidate genes were selected for functional validation studies performed with multiple cancer cell lines.</p></div></div></div><p id="__p23">For functional validation, we used either two unvalidated siRNAs or one validated siRNA based on QIAGEN data to knockdown each of the 23 candidate genes. If two siRNAs were available, we defined “significance” as a gene with a significant change in apparent AUC for both siRNAs in comparison with a control siRNA. In order to screen candidates efficiently, MTS assays were used with all three tumor cell lines for all 23 of the genes selected for study, although we understood the limitations of short-term cytotoxicity assays for the prediction of radiation survival and that we might miss some associations using this assay (<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951350">Carmichael et al. 1987</a>; <a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951321">Anoopkumar-Dukie et al. 2005</a>). Knockdown of seven genes had a significant effect on radiation sensitivity in one cell line, three genes were positive for two cell lines, and knockdown of two genes significantly altered radiation sensitivity in all three cell lines (<a href="/pmc/articles/PMC2963812/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>; <a href="/pmc/articles/PMC2963812/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). We then selected five genes for further study for which knockdown with specific siRNAs significantly altered radiation sensitivity in at least two cancer cell lines, specifically <em>C13orf34</em>, <em>MAD2L1</em>, <em>PLK4</em>, <em>TPD52</em>, and <em>TTF1</em> (<a href="/pmc/articles/PMC2963812/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>; <a href="/pmc/articles/PMC2963812/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). We also included <em>DEPDC1B</em>, even though this gene only showed an effect of knockdown on radiation sensitivity in HeLa cells, since two SNPs located 2.5 and 8.4 kb downstream of this gene (rs4326096 and rs2409791) were significantly correlated, in the cell-line model system, with the expression of <em>C13orf34</em> (<em>P</em> = 9.97 × 10<sup>−5</sup> and 4.86 × 10<sup>−5</sup>, respectively) (Supplemental Table 4), a gene that displayed a functional effect on radiation-induced cytotoxicity in all three cancer cell lines (<a href="/pmc/articles/PMC2963812/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>; <a href="/pmc/articles/PMC2963812/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). These two SNPs were highly linked (<em>r</em><sup>2</sup> &gt; 0.9).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p24">Candidate genes selected for siRNA screening, with MTS assay results as well as colony-forming assay results when appropriate</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140386670997520" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2963812_1482tbl3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1482tbl3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2963812/bin/1482tbl3.jpg"></a></div><div id="largeobj_idm140386670997520" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2963812/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140386670996304"><p id="__p25" class="p p-first-last">Twenty-three genes selected for siRNA screening are listed.</p></div><div id="idm140386670995680"><p id="__p26" class="p p-first-last"><sup>a</sup>“Yes” indicates individual candidate genes with the <em>P</em>-value listed.</p></div><div id="idm140386670994336"><p id="__p27" class="p p-first-last"><sup>b</sup>“Yes” indicates that knockdown of the gene altered radiation cytotoxicity (AUC values) when compared with control siRNA.</p></div><div id="idm140386670993296"><p id="__p28" class="p p-first-last"><sup>c</sup>NP, This assay was not performed; NC, no colonies formed for that cell line after knockdown with a specific siRNA.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2963812/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140386674710896" class="figure"><a href="/pmc/articles/PMC2963812/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2963812_1482fig4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1482fig4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2963812/bin/1482fig4.jpg"></a></div><div id="largeobj_idm140386674710896" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2963812/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2963812/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p id="__p29">siRNA screening of candidate genes by MTS assay in multiple cancer cell lines. Data are shown for five of the top 23 candidate genes that were studied functionally in HupT3, MIA-PaCa2, and HeLa cancer cell lines by MTS assay after siRNA knockdown performed with two “unvalidated” or one “validated” siRNA when available. (Blue) Data for nontransfected cells; (red) negative control siRNA; (green, light blue) data for specific siRNAs. “Significance” was defined as a gene with a significant change in apparent AUC in comparison with control siRNA and was indicated by the <em>P</em>-value. If two siRNAs were available, “significance” was defined as a gene with significant changes for both siRNAs, and P<sub>1</sub> and P<sub>2</sub> represented the <em>P</em>-values for siRNA 1 and siRNA 2, respectively. At least three independent experiments were performed in triplicate. Error bar, SEM of at least three independent experiments. (<em>A</em>) Candidate gene symbols. (<em>B</em>) qRT-PCR. The <em>y</em>-axis indicates relative gene expression after siRNA knockdown when compared with “all star negative” siRNA. (<em>C</em>) MTS assays. The <em>x</em>-axis indicates the radiation dose, and the <em>y</em>-axis indicates the surviving fraction after exposure to radiation.</p></div></div></div><p id="__p30">As the next step in the analysis, and to further confirm results obtained with the MTS assay, we also performed colony-forming assays for these same six genes in the MIA-PaCa2, HupT3, and HeLa cells used to perform the MTS assays, plus a lung-cancer cell line, A549, since radiation is commonly used to treat lung cancer. As shown graphically in <a href="/pmc/articles/PMC2963812/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>, knockdown of <em>C13orf34</em>, <em>DEPDC1B</em>, and <em>TPD52</em> desensitized all four cell lines to radiation treatment. <em>MAD2L1</em> knockdown had a significant impact on radiation effect in HeLa and HupT3, but not in the A549 cell line. However, knockdown of <em>MAD2L1</em> resulted in lack of colony formation in MIA-PaCa2 cells in the absence of radiation treatment, indicating that this gene might be essential for cell proliferation. Knockdown of <em>PLK4</em> only desensitized radiation response in HupT3 cells, consistent with the MTS assay results, and <em>TTF1</em> knockdown did not alter radiation sensitivity in any of the four cell lines (data not shown). These results indicated that this surrogate MTS assay can provide valid leads for identifying genes that might play a role in radiation sensitivity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC2963812/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140386681562384" class="figure"><a href="/pmc/articles/PMC2963812/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2963812_1482fig5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1482fig5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2963812/bin/1482fig5.jpg"></a></div><div id="largeobj_idm140386681562384" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2963812/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC2963812/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><p id="__p31">siRNA screening of candidate genes by colony-forming assay performed with multiple cancer cell lines. Data for five of the 23 top candidate genes were selected for colony-forming assays using HupT3, MIA-PaCa2, HeLa, and A549 cancer cell lines. qRT-PCR was also performed to determine knockdown efficiency. At least three independent experiments were performed in triplicate. Error bar, SEM of at least three independent experiments. Significance was defined by <em>P</em>-values. (<em>A</em>) Candidate gene symbols. (<em>B</em>) qRT-PCR to assess expression levels for each candidate gene after knockdown in each cell line. (<em>C</em>) Colony-forming assays performed with HupT3, MIA-PaCa2, HeLa, and A549 cancer cell lines. The <em>x</em>-axis indicates the radiation dose, and the <em>y</em>-axis indicates the surviving fraction after radiation exposure.</p></div></div></div><p id="__p32" class="p p-last">Taken together, through the initial screening studies in LCLs and further functional validation in multiple tumor cell lines, we found that the expression of five genes, <em>C13orf34</em>, <em>MAD2L1</em>, <em>PLK4</em>, <em>TPD52</em>, <em>DEPDC1B</em>, were involved in radiation-induced response. We computed the percentage of variation in the AUC of LCLs that was explained by the variation of these five genes' expression (with and without inclusion of these two SNPs, rs4326096 and rs2409791, close to <em>DEPDC1B</em>), which were 11.4% and 16.1%.</p></div></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title">Discussion</h2><p id="__p33" class="p p-first">Radiation therapy plays an important role in the treatment of cancer. However, response to radiation therapy varies widely. The therapeutic efficacy of radiation is determined mainly by total dose, but the use of high doses has been limited because of the severity of side effects in normal tissue. It is estimated that nearly 80% of interindividual variation in normal tissue response to radiation might be due to genetic factors (<a href="#B53" rid="B53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951368">Turesson et al. 1996</a>). Radiation therapy also has a relatively narrow therapeutic index (<a href="#B52" rid="B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951381">Turesson 1990</a>; <a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951376">Bentzen et al. 2008</a>). Therefore, understanding the biology underlying the variation in response to radiation might help us to maximize radiation efficacy in the tumor, while minimizing side effects in normal tissues. Several radio-genetic studies have already shown that genetic polymorphisms in genes within known radiation response pathways are significantly associated with radiosensitivity. Those pathways include endogenous oxidative stress defense, inflammatory response, cytokine activity related to fibrosis, DNA damage signaling, cell cycle control, and DNA repair (<a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951387">Chistiakov et al. 2008</a>; <a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951334">Andreassen and Alsner 2009</a>; <a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951360">Popanda et al. 2009</a>; <a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951355">Pugh et al. 2009</a>). These previous studies raised the possibility that genetic variation might contribute to individual variation in radiation response. In the present study, we performed a genome-wide association analysis using 277 LCLs for which we had 1.3 million SNPs, basal gene expression data, and an ionizing radiation cytotoxicity AUC phenotype to identify genes that might be responsible for radiation sensitivity. LCLs have been used successfully in many pharmacogenomic studies to identify and understand genetic variation associated with variation in drug response phenotypes (<a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951351">Huang et al. 2008</a>; <a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951383">Li et al. 2008</a>, <a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951366">2009</a>; <a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951322">Bleibel et al. 2009</a>). In addition, in vitro lymphocyte radiosensitivity has been found to be significantly correlated with clinical response to radiation therapy, although biological differences obviously exist between lymphocytes and LCLs (<a href="#B55" rid="B55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951386">West et al. 2001</a>; <a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951363">Dikomey et al. 2003</a>).</p><p id="__p34">Based on the analysis of SNP, basal gene expression and radiation cytotoxicity (AUC) data, our study yielded a total of 240 candidate genes, including 211 identified as a result of either expression versus AUC associations (<em>P</em> &lt; 10<sup>−3</sup>) or an integrated analysis that included SNP markers, expression, and radiation AUC data. We also identified 29 genes based on the association of SNPs with AUC (<em>P</em> &lt; 10<sup>−4</sup>). Although only one probe set was statistically significant after Bonferroni correction, and none of the SNPs remained significant after Bonferroni correction, which might be due to our limited sample size used for our GWA analyses, when we performed Ingenuity Pathway analysis for these 240 genes, the top three networks all involved “cell death” and centered around NFKB, PI3K/Akt and MAPK/ERK as “network hubs” (Supplemental Fig. 2). Similarly, the NFKB pathway has been also found to be involved in the response to a higher dose of radiation (10 Gy) in TP53-null or mutant lymphoblast cell lines (<a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951333">Lu et al. 2010</a>). Many candidate genes identified during our study have been reported to have altered levels of expression in response to radiation exposure in the NCI-60 cell lines or in lymphoblastoid cell lines, especially TP53-dependent genes such as <em>CDK1</em>, <em>PHLDA3</em>, and <em>PTPRC</em> (<a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951367">Amundson et al. 1999</a>, <a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951326">2003</a>, <a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951389">2005</a>, <a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951348">2008</a>; <a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951349">Jen and Cheung 2005</a>). In addition, genes such as <em>MEF2B</em>, <em>NRF1</em>, <em>PHPT1</em>, <em>ZMAT3</em>, <em>CHEK1</em>, and <em>GALR3</em> that are up- or down-regulated by ionizing radiation exposure, were also found to be associated with radiation AUC from our study (<a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951343">Amundson et al. 2004</a>; <a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951337">Jen and Cheung 2005</a>; <a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951359">Dressman et al. 2007</a>; <a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951358">Paul and Amundson 2008</a>; <a href="#B43" rid="B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951328">Rzeszowska-Wolny et al. 2009</a>; <a href="#B56" rid="B56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951329">Westbury et al. 2009</a>). These results suggested that our association study performed with 277 human lymphoblastoid cell lines was capable of generating biologically relevant candidates for follow-up study.</p><p id="__p35">The LCL system used in our initial screening studies, like all model systems, has limitations. EBV transformation might cause chromosomal instability and cellular changes in the LCLs (<a href="#B48" rid="B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951319">Sie et al. 2009</a>), and variation of the response of these cells might be influenced by nongenetic factors such as cell growth rate or baseline ATP levels (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951342">Choy et al. 2008</a>). In addition, the regulation of gene expression is tissue specific (<a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951374">Dimas et al. 2009</a>; <a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951378">Kwan et al. 2009</a>). Therefore, to validate the initial association results obtained using LCLs, we selected 23 top candidate genes (see <a href="/pmc/articles/PMC2963812/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>; <a href="/pmc/articles/PMC2963812/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>) to perform functional validation studies using siRNA knockdown performed with MIA-PaCa2, HupT3, Hela, and A549 cancer cell lines (<a href="/pmc/articles/PMC2963812/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figs. 4</span></a>, <a href="/pmc/articles/PMC2963812/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,5).</span></span><span>5</span></a>). We removed genes that were not highly expressed in the LCLs, as well as genes that are involved in pathways such as DNA repair, cell cycle, cell survival, and apoptosis, which are already known to influence radiation sensitivity. We understood that these selection criteria were arbitrary, but they provided a way to narrow down the candidates for functional follow up. Five genes were shown to alter the radiation response with MTS and colony-forming assays (<a href="/pmc/articles/PMC2963812/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figs. 4</span></a>, <a href="/pmc/articles/PMC2963812/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,5),</span></span><span>5</span></a>), and all five genes contributed ∼11% of variation in AUC observed. These results suggested that, like many complex traits, response to radiation also involves multiple genes in different pathways, and, indeed, many of the genes at the top rank in our association study have been shown to play important roles in pathways related to radiation response. However, this is not unique. Even with clinically established pharmacogenomic examples such as <em>CYP2C19</em> *<em>2</em>, only 12% of the variation in platelet aggregation can be explained by this variant (<a href="#B47" rid="B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337500447">Shuldiner et al. 2009</a>). Although the five final genes contributed 11% of the total variation, they also provided us with new insights into novel mechanisms in radiation response. For example, among the genes for which we performed colony-forming assays (<a href="/pmc/articles/PMC2963812/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>), two altered radiation sensitivity in all four cancer cell lines tested. The first, <em>C13orf34</em>, mapped to chromosome 13q22, and has been implicated as a putative breast cancer susceptibility locus (<a href="#B41" rid="B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951375">Rozenblum et al. 2002</a>). This portion of the genome is also a common site for somatic deletions in many malignant tumors (<a href="#B41" rid="B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951323">Rozenblum et al. 2002</a>). As a cell cycle protein, C13orf34 enhances the initial activation of Polo-like kinase 1 (Plk1) in an Aurora A-dependent manner at the G<sub>2</sub>/M transition, and facilitates G<sub>2</sub>/M transition (<a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951339">Macurek et al. 2008</a>, <a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951361">2009</a>; <a href="#B45" rid="B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951369">Seki et al. 2008a</a>,<a href="#B46" rid="B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951324">b</a>; <a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951379">Pomerening 2009</a>). However, no previous study had linked C13orf34 to the DNA damage response pathway or to radiation sensitivity. The fact that we found that <em>C13orf34</em> expression was correlated with radiation cytotoxicity and that knockdown of C13orf34 desensitized cancer cells to radiation exposure suggests a potential role for <em>C13orf34</em> in the DNA damage response pathway, in addition to its role as a cell cycle regulator. This hypothesis will need to be tested further in future experiments.</p><p id="__p36">The second gene that influenced radiation response in all four cell lines was <em>TPD52</em>, also known as “Tumor protein D52.” TPD52 is a coiled-coil motif bearing protein that plays a role in cell proliferation, apoptosis, vesicle trafficking, tumorigenesis, and metastasis (<a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951347">Lewis et al. 2007</a>; <a href="#B54" rid="B54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951382">Ummanni et al. 2008</a>). TPD52 is highly overexpressed in multiple cancers as a result of gene amplification (<a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951346">Balleine et al. 2000</a>; <a href="#B42" rid="B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951356">Rubin et al. 2004</a>; <a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951325">Byrne et al. 2005</a>; <a href="#B59" rid="B59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951340">Zhu et al. 2007</a>; <a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951372">Petrova et al. 2008</a>). In breast cancer patients, TPD52 overexpression was correlated with lymphocyte radiosensitivity (<a href="#B49" rid="B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951364">Sims et al. 2007</a>), an observation consistent with our association study in LCLs and our functional validation experiments performed with tumor cell lines.</p><p id="__p37" class="p p-last">In summary, although variation in radiation response might result from multiple gene effects and, possibly, gene-environment interactions, our results provide important insight into novel genes and mechanisms that may contribute to variation in response to radiation therapy.</p></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Methods</h2><div id="__sec9" class="sec sec-first"><h3 id="__sec9title">Cell lines</h3><p id="__p38" class="p p-first">EBV-transformed LCLs from 93 African-American (AA), 89 Caucasian-American (CA), and 95 Han Chinese–American (HCA) unrelated healthy subjects (sample sets HD100AA, HD100CAU, HD100CHI) were purchased from the Coriell Cell Repository. These samples had been collected and anonymized by NIGMS, and all subjects had provided written consent for their experimental use. This study was reviewed and approved by the Mayo Clinic Institutional Review Board. Human pancreatic cancer MIA-PaCa2 and HupT3 cell lines were gifts from Daniel D. Billadeau (Mayo Clinic, Rochester, MN). Human cervical cancer HeLa and non-small-cell lung cancer A549 cell lines (ATCC no. CCL-185) were obtained from the ATCC.</p><p id="__p39" class="p p-last">LCLs were cultured in RPMI 1640 medium (Mediatech) supplemented with 15% heat-inactivated Fetal Bovine Serum (FBS) (Mediatech). According to the protocol from the Coriell website (<a href="http://ccr.coriell.org/sections/support/global/lymphoblastoid.aspx?pgid=213" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ccr.coriell.org/sections/support/global/lymphoblastoid.aspx?pgid=213</a>), LCLs were maintained at a density of 2–8 × 10<sup>5</sup> cells/mL, and were split or refed with fresh medium every 3 d, depending on the growth status of each cell line. HeLa and MIA-PaCa2 cell lines were cultured in DMEM medium containing 10% FBS. HupT3 and A549 cell lines were grown in RPMI 1640 medium with 10% FBS.</p></div><div id="__sec10" class="sec"><h3 id="__sec10title">Cell proliferation assay</h3><p id="__p40" class="p p-first-last">Cell proliferation assays were performed in triplicate at each radiation dose. Specifically, 100 μL of cells (5 × 10<sup>5</sup> cells/mL) were plated into 96-well plates (Corning) (<a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951377">Li et al. 2008</a>) and were treated with ionizing radiation at 0, 0.156, 0.3125, 0.625, 1.25, 2.5, 5, 10, and 20 Gy using <sup>137</sup>Cesium gamma-rays (J.L. Shepherd and Associates Mark I Model 25 Irradiator). After incubation for 3 d, 20 μL of CellTiter 96 AQueous Non-Radioactive Cell Proliferation Assay solution (Promega Corporation) was added to each well. Plates were read in a Safire2 plate reader (Tecan AG). Nineteen LCLs were selected randomly, and the MTS assay was repeated ∼1 yr after the initial assay. Cytotoxicity for human tumor cell lines was determined in a similar fashion, except the cells were incubated overnight before radiation treatment at 0, 0.25, 0.5, 1, 2.5, 5, 10, and 20 Gy, followed by MTS and colony-forming assays.</p></div><div id="__sec11" class="sec"><h3 id="__sec11title">Expression array assays</h3><p id="__p41" class="p p-first-last">Total RNA was extracted from each of the cell lines using Qiagen RNeasy Mini kits (QIAGEN, Inc.). RNA quality was tested using an Agilent 2100 Bioanalyzer, followed by hybridization to Affymetrix U133 Plus 2.0 Gene-Chips. A total of 54,613 probe sets were used in the analyses. Expression array data were obtained for all of the cell lines, 166 of which had been used in previous studies (<a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951352">Li et al. 2008</a>, <a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951331">2009</a>) plus 111 new cell lines that were used in the present study.</p></div><div id="__sec12" class="sec"><h3 id="__sec12title">Genome-wide SNP analysis</h3><p id="__p42" class="p p-first-last">DNA from all of the LCLs was genotyped using Illumina HumanHap 550K and 510S BeadChips, which assayed 561,298 and 493,750 SNPs, respectively. Genotyping was performed in the Genotype Shared Resource (GSR) at the Mayo Clinic. We also obtained publicly available Affymetrix SNP Array 6.0 Chip SNP data for the same cell lines, which involved 643,600 SNPs unique to the Affymetrix SNP array. The genotyping data from 166 of 277 LCLs using Illumina HumanHap 550K BeadChips had been used in a previous study (<a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951345">Li et al. 2009</a>). SNPs that deviated from Hardy-Weinberg Equilibrium (HWE), based on the minimum <em>P</em>-value from an exact test for HWE (<a href="#B57" rid="B57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951330">Wigginton et al. 2005</a>) and the stratified test for HWE (<a href="#B44" rid="B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951357">Schaid et al. 2006</a>) (<em>P</em>-values &lt; 0.001); SNPs with call rates &lt;95%; or SNPs with MAFs &lt;5% were removed from the analysis.</p></div><div id="__sec13" class="sec"><h3 id="__sec13title">Transient transfection and RNA interference</h3><p id="__p43" class="p p-first-last">siRNAs for the candidate genes and “all star negative control” siRNA were purchased from QIAGEN. The human HeLa cervical cancer cell line and the human pancreatic cancer cell lines, MIA-PaCa2 and HupT3, were used for the siRNA knockdown experiments. Reverse transfection was performed in 96-well plates. Specifically, ∼3000–4000 cells were mixed with 0.1 μL of lipofectamine RNAiMAX reagent (Invitrogen) and 10 nM siRNA for these experiments.</p></div><div id="__sec14" class="sec"><h3 id="__sec14title">Colony-forming assays</h3><p id="__p44" class="p p-first-last">“All star negative control” and specific siRNAs were transfected into MIA-PaCa2, HupT3, HeLa, and A549 cell lines. Twenty-four hours later, ∼400–1000 cells were plated in triplicate in 6-well plates. After exposure to radiation, cells were incubated for up to 7 d. Colonies in each well were fixed with methanol and stained with crystal violet. All colonies were counted visually or by the use of Quantity One (Bio-Rad).</p></div><div id="__sec15" class="sec"><h3 id="__sec15title">Real-time quantitative reverse transcription-PCR (qRT-PCR)</h3><p id="__p45" class="p p-first-last">Total RNA was isolated from cultured cells transfected with control or specific siRNAs with the Qiagen RNeasy kit (QIAGEN, Inc.), followed by qRT-PCR performed with the one-step, Brilliant SYBR Green qRT-PCR master mix kit (Stratagene). Specifically, primers purchased from QIAGEN were used to perform qRT-PCR using the Stratagene Mx3005P Real-Time PCR detection system (Stratagene). All experiments were performed in triplicate with beta-actin as an internal control. Reverse transcribed Universal Human reference RNA (Stratagene) was used to generate a standard curve. Control reactions lacked RNA template.</p></div><div id="__sec16" class="sec sec-last"><h3 id="__sec16title">Statistical methods</h3><p id="__p46" class="p p-first">The radiation cytotoxicity phenotype, AUC, was calculated based on a logistic model. Three different logistic functions (a four parameter logistic model, a three parameter logistic model with a fixed asymptote at 0%, and a three parameter logistic model with a fixed asymptote at 100%) were used to fit the data with the R package “drc” (<a href="http://cran.r-project.org/web/packages/drc/drc.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cran.r-project.org/web/packages/drc/drc.pdf</a>). The best model fit (i.e., lowest mean square error) from the three logistic models was used to determine the cytotoxicity AUC phenotype. The AUC phenotype was determined by numerically computing the area under the estimated dose-response curve, from dose 0 to 20 Gy. AUC values were then natural-log transformed. Expression array data were normalized on a log<sub>2</sub> scale using GCRMA (<a href="#B58" rid="B58" class=" bibr popnode">Wu et al. 2004</a>). The normalized expression data and log transformed AUC values were then regressed on gender. Since the LCLs were from multiple races/ethnic groups, before completing the genetic association analysis, population stratification was assessed using the method developed by <a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951371">Price et al. (2006)</a>. This approach uses an eigen analysis for detecting and adjusting for population stratification, in which the eigen analysis was performed within each of the three racial groups. Using the top five eigenvectors within each race, the individual genotypes were adjusted using the model <strong><em>G<sub>ij</sub></em> = <em>α<sub>j</sub></em> + <em>γ<sub>kj</sub></em> + <em>ɛ<sub>ij</sub></em></strong>, with <strong><em>G<sub>ij</sub></em></strong> representing the genotype for the <em>i</em>th cell line in racial group <em>j</em> (<em>j</em> = 1, 2, 3), <strong><em>α<sub>j</sub></em></strong> the race effect for race <em>j</em>, and <strong><em>γ<sub>kj</sub></em></strong> the <em>k</em>th eigenvector (<em>k</em> = 1, 2, 3, 4, 5) effect for race <em>j</em>. Similarly, for genetic analysis of SNPs with AUC or expression, the log transformed AUC values and normalized log2 expression values were also adjusted for race using the five eigenvectors, in addition to gender.</p><p id="__p47">Association analyses of expression-AUC, expression-SNP, and SNP-AUC were then completed using Pearson correlations and the adjusted AUC, SNP, and expression data. False Discovery <em>Q</em>-values (<a href="#B51" rid="B51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951332">Storey and Tibshirani 2003</a>) were also computed for each test. Pairwise LD was estimated using <em>r</em><sup>2</sup> (<em>r</em>-squared) statistics and was displayed graphically using the Haploview software (<a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951327">Barrett et al. 2005</a>). Genes and SNPs were annotated using NCBI Build 36.3. The pathway analysis, for the top genes, was performed using Ingenuity Pathway Analysis (IPA; Ingenuity Systems).</p><p id="__p48">To integrate the genotype, expression, and drug cytotoxicity data, we first identified loci containing at least two SNPs associated with AUC with <em>P</em>-values &lt; 10<sup>−4</sup> within 50 kb of each other and identified all SNPs located within the specific loci with <em>P</em>-values &lt; 0.001. Next, we determined which expression probe sets were associated with these SNPs (<em>P</em>-values &lt; 10<sup>−4</sup>). Finally, to determine whether the expression probe sets associated with these SNPs were also associated with radiation AUC values, we identified which expression probe sets were associated with radiation AUC (<em>P</em>-values &lt; 0.001). A similar approach has been used successfully to detect novel candidate genes for functional follow-up (<a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_199951370">Li et al. 2009</a>).</p><p id="__p49">For the most significant locus on chromosome 8, SNPs were imputed for a region 200 kb in length on either side of the most significant SNP. Imputation was performed using MACH 1.0 (<a href="#B29" rid="B29" class=" bibr popnode">Li and Abecasis 2006</a>), with HapMap data (<a href="http://hapmap.ncbi.nlm.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hapmap.ncbi.nlm.nih.gov/</a>) as the reference panel. Specifically, SNPs for AA were imputed using both CEU and YRI data, SNPs for CA were imputed based on CEU data, and SNP markers for HCA were imputed based on the CHB and JPT data. In addition, the percent of variation in the drug response phenotype explained by a set of genes was estimated using <em>r</em><sup>2</sup> from a multivariable linear regression model. Lastly, significance of the AUC values between negative control siRNA and gene-specific siRNA was determined by a two-tailed unpaired <em>t</em>-test.</p><p id="__p50" class="p p-last">To determine association between SNPs and expression using the data sets generated from our 300 human lymphoblastoid cell lines, visit <a href="http://pgxcms.mayo.edu/AR" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pgxcms.mayo.edu/AR</a>.</p></div></div><div id="idm140386748198416" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140386748198416title">Acknowledgments</h2><div class="sec"><p id="__p51">This work was supported in part by National Institutes of Health (NIH) grants K22 CA130828, R01 CA138461, and U01 GM61388 (The Pharmacogenetics Research Network), an ASPET-Astellas Award, and a PhRMA Foundation “Center of Excellence in Clinical Pharmacology” Award.</p></div></div><div id="idm140386680837520" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140386680837520title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140386680837392"></p><p id="__p52" class="p p-first-last">[Supplemental material is available online at <a href="http://www.genome.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.org</a>. The microarray data and SNP data from this study have been submitted to the NCBI Gene Expression Omnibus (<a href="http://www.ncbi.nlm.nih.gov/geo" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/geo</a>) under SuperSeries accession no. <a id="__tag_199951390" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE24277" ref="/geo/query/acc.cgi?acc=GSE24277"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE24277","term_id":"24277"}}</span>GSE24277</a>.]</p><p></p><p class="fn sec" id="idm140386680833808"></p><p id="__p53" class="p p-first-last">Article published online before print. Article and publication date are at <a href="http://www.genome.org/cgi/doi/10.1101/gr.107672.110" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.org/cgi/doi/10.1101/gr.107672.110</a>.</p><p></p></div></div><div id="idm140386672382208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140386672382208title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="first-line-outdent"><li id="B1"><span class="mixed-citation">Amundson SA, Bittner M, Chen Y, Trent J, Meltzer P, Fornace AJ Jr 1999.  <span class="ref-title">Fluorescent cDNA microarray hybridization reveals complexity and heterogeneity of cellular genotoxic stress responses</span>. <span class="ref-journal">Oncogene</span> <span class="ref-vol">18</span>: 3666–3672 [<a href="/pubmed/10380890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Fluorescent+cDNA+microarray+hybridization+reveals+complexity+and+heterogeneity+of+cellular+genotoxic+stress+responses&amp;author=SA+Amundson&amp;author=M+Bittner&amp;author=Y+Chen&amp;author=J+Trent&amp;author=P+Meltzer&amp;volume=18&amp;publication_year=1999&amp;pages=3666-3672&amp;pmid=10380890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B2"><span class="mixed-citation">Amundson SA, Lee RA, Koch-Paiz CA, Bittner ML, Meltzer P, Trent JM, Fornace AJ Jr 2003.  <span class="ref-title">Differential responses of stress genes to low dose-rate gamma irradiation</span>. <span class="ref-journal">Mol Cancer Res</span> <span class="ref-vol">1</span>: 445–452 [<a href="/pubmed/12692264" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cancer+Res&amp;title=Differential+responses+of+stress+genes+to+low+dose-rate+gamma+irradiation&amp;author=SA+Amundson&amp;author=RA+Lee&amp;author=CA+Koch-Paiz&amp;author=ML+Bittner&amp;author=P+Meltzer&amp;volume=1&amp;publication_year=2003&amp;pages=445-452&amp;pmid=12692264&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B3"><span class="mixed-citation">Amundson SA, Grace MB, McLeland CB, Epperly MW, Yeager A, Zhan Q, Greenberger JS, Fornace AJ Jr 2004.  <span class="ref-title">Human in vivo radiation-induced biomarkers: Gene expression changes in radiotherapy patients</span>. <span class="ref-journal">Cancer Res</span> <span class="ref-vol">64</span>: 6368–6371 [<a href="/pubmed/15374940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Human+in+vivo+radiation-induced+biomarkers:+Gene+expression+changes+in+radiotherapy+patients&amp;author=SA+Amundson&amp;author=MB+Grace&amp;author=CB+McLeland&amp;author=MW+Epperly&amp;author=A+Yeager&amp;volume=64&amp;publication_year=2004&amp;pages=6368-6371&amp;pmid=15374940&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B4"><span class="mixed-citation">Amundson SA, Do KT, Vinikoor L, Koch-Paiz CA, Bittner ML, Trent JM, Meltzer P, Fornace AJ Jr 2005.  <span class="ref-title">Stress-specific signatures: Expression profiling of p53 wild-type and -null human cells</span>. <span class="ref-journal">Oncogene</span> <span class="ref-vol">24</span>: 4572–4579 [<a href="/pubmed/15824734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Stress-specific+signatures:+Expression+profiling+of+p53+wild-type+and+-null+human+cells&amp;author=SA+Amundson&amp;author=KT+Do&amp;author=L+Vinikoor&amp;author=CA+Koch-Paiz&amp;author=ML+Bittner&amp;volume=24&amp;publication_year=2005&amp;pages=4572-4579&amp;pmid=15824734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B5"><span class="mixed-citation">Amundson SA, Do KT, Vinikoor LC, Lee RA, Koch-Paiz CA, Ahn J, Reimers M, Chen Y, Scudiero DA, Weinstein JN, et al. 2008.  <span class="ref-title">Integrating global gene expression and radiation survival parameters across the 60 cell lines of the National Cancer Institute Anticancer Drug Screen</span>. <span class="ref-journal">Cancer Res</span> <span class="ref-vol">68</span>: 415–424 [<a href="/pubmed/18199535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Integrating+global+gene+expression+and+radiation+survival+parameters+across+the+60+cell+lines+of+the+National+Cancer+Institute+Anticancer+Drug+Screen&amp;author=SA+Amundson&amp;author=KT+Do&amp;author=LC+Vinikoor&amp;author=RA+Lee&amp;author=CA+Koch-Paiz&amp;volume=68&amp;publication_year=2008&amp;pages=415-424&amp;pmid=18199535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B6"><span class="mixed-citation">Andreassen CN, Alsner J 2009.  <span class="ref-title">Genetic variants and normal tissue toxicity after radiotherapy: A systematic review</span>. <span class="ref-journal">Radiother Oncol</span> <span class="ref-vol">92</span>: 299–309 [<a href="/pubmed/19683821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Radiother+Oncol&amp;title=Genetic+variants+and+normal+tissue+toxicity+after+radiotherapy:+A+systematic+review&amp;author=CN+Andreassen&amp;author=J+Alsner&amp;volume=92&amp;publication_year=2009&amp;pages=299-309&amp;pmid=19683821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B7"><span class="mixed-citation">Anoopkumar-Dukie S, Carey JB, Conere T, O'Sullivan E, van Pelt FN, Allshire A 2005.  <span class="ref-title">Resazurin assay of radiation response in cultured cells</span>. <span class="ref-journal">Br J Radiol</span> <span class="ref-vol">78</span>: 945–947 [<a href="/pubmed/16177019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Radiol&amp;title=Resazurin+assay+of+radiation+response+in+cultured+cells&amp;author=S+Anoopkumar-Dukie&amp;author=JB+Carey&amp;author=T+Conere&amp;author=E+O'Sullivan&amp;author=FN+van+Pelt&amp;volume=78&amp;publication_year=2005&amp;pages=945-947&amp;pmid=16177019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B8"><span class="mixed-citation">Balleine RL, Fejzo MS, Sathasivam P, Basset P, Clarke CL, Byrne JA 2000.  <span class="ref-title">The hD52 (TPD52) gene is a candidate target gene for events resulting in increased 8q21 copy number in human breast carcinoma</span>. <span class="ref-journal">Genes Chromosomes Cancer</span> <span class="ref-vol">29</span>: 48–57 [<a href="/pubmed/10918393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Chromosomes+Cancer&amp;title=The+hD52+(TPD52)+gene+is+a+candidate+target+gene+for+events+resulting+in+increased+8q21+copy+number+in+human+breast+carcinoma&amp;author=RL+Balleine&amp;author=MS+Fejzo&amp;author=P+Sathasivam&amp;author=P+Basset&amp;author=CL+Clarke&amp;volume=29&amp;publication_year=2000&amp;pages=48-57&amp;pmid=10918393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B9"><span class="mixed-citation">Barnett GC, West CM, Dunning AM, Elliott RM, Coles CE, Pharoah PD, Burnet NG 2009.  <span class="ref-title">Normal tissue reactions to radiotherapy: Towards tailoring treatment dose by genotype</span>. <span class="ref-journal">Nat Rev Cancer</span> <span class="ref-vol">9</span>: 134–142 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2670578/">PMC free article</a>]</span> [<a href="/pubmed/19148183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Normal+tissue+reactions+to+radiotherapy:+Towards+tailoring+treatment+dose+by+genotype&amp;author=GC+Barnett&amp;author=CM+West&amp;author=AM+Dunning&amp;author=RM+Elliott&amp;author=CE+Coles&amp;volume=9&amp;publication_year=2009&amp;pages=134-142&amp;pmid=19148183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B10"><span class="mixed-citation">Barrett JC, Fry B, Maller J, Daly MJ 2005.  <span class="ref-title">Haploview: Analysis and visualization of LD and haplotype maps</span>. <span class="ref-journal">Bioinformatics</span> <span class="ref-vol">21</span>: 263–265 [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+Analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B11"><span class="mixed-citation">Bentzen SM 2006.  <span class="ref-title">Preventing or reducing late side effects of radiation therapy: Radiobiology meets molecular pathology</span>. <span class="ref-journal">Nat Rev Cancer</span> <span class="ref-vol">6</span>: 702–713 [<a href="/pubmed/16929324" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Preventing+or+reducing+late+side+effects+of+radiation+therapy:+Radiobiology+meets+molecular+pathology&amp;author=SM+Bentzen&amp;volume=6&amp;publication_year=2006&amp;pages=702-713&amp;pmid=16929324&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B12"><span class="mixed-citation">Bentzen SM, Agrawal RK, Aird EG, Barrett JM, Barrett-Lee PJ, Bliss JM, Brown J, Dewar JA, Dobbs HJ, Haviland JS, et al. 2008.  <span class="ref-title">The UK Standardisation of Breast Radiotherapy (START) Trial A of radiotherapy hypofractionation for treatment of early breast cancer: A randomised trial</span>. <span class="ref-journal">Lancet Oncol</span> <span class="ref-vol">9</span>: 331–341 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2323709/">PMC free article</a>]</span> [<a href="/pubmed/18356109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Oncol&amp;title=The+UK+Standardisation+of+Breast+Radiotherapy+(START)+Trial+A+of+radiotherapy+hypofractionation+for+treatment+of+early+breast+cancer:+A+randomised+trial&amp;author=SM+Bentzen&amp;author=RK+Agrawal&amp;author=EG+Aird&amp;author=JM+Barrett&amp;author=PJ+Barrett-Lee&amp;volume=9&amp;publication_year=2008&amp;pages=331-341&amp;pmid=18356109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B13"><span class="mixed-citation">Bleibel WK, Duan S, Huang RS, Kistner EO, Shukla SJ, Wu X, Badner JA, Dolan ME 2009.  <span class="ref-title">Identification of genomic regions contributing to etoposide-induced cytotoxicity</span>. <span class="ref-journal">Hum Genet</span> <span class="ref-vol">125</span>: 173–180 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2714550/">PMC free article</a>]</span> [<a href="/pubmed/19089452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Identification+of+genomic+regions+contributing+to+etoposide-induced+cytotoxicity&amp;author=WK+Bleibel&amp;author=S+Duan&amp;author=RS+Huang&amp;author=EO+Kistner&amp;author=SJ+Shukla&amp;volume=125&amp;publication_year=2009&amp;pages=173-180&amp;pmid=19089452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B14"><span class="mixed-citation">Burnet NG, Elliott RM, Dunning A, West CM 2006.  <span class="ref-title">Radiosensitivity, radiogenomics and RAPPER</span>. <span class="ref-journal">Clin Oncol (R Coll Radiol)</span> <span class="ref-vol">18</span>: 525–528 [<a href="/pubmed/16969982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Oncol+(R+Coll+Radiol)&amp;title=Radiosensitivity,+radiogenomics+and+RAPPER&amp;author=NG+Burnet&amp;author=RM+Elliott&amp;author=A+Dunning&amp;author=CM+West&amp;volume=18&amp;publication_year=2006&amp;pages=525-528&amp;pmid=16969982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B15"><span class="mixed-citation">Byrne JA, Balleine RL, Schoenberg Fejzo M, Mercieca J, Chiew YE, Livnat Y, St Heaps L, Peters GB, Byth K, Karlan BY, et al. 2005.  <span class="ref-title">Tumor protein D52 (TPD52) is overexpressed and a gene amplification target in ovarian cancer</span>. <span class="ref-journal">Int J Cancer</span> <span class="ref-vol">117</span>: 1049–1054 [<a href="/pubmed/15986428" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=Tumor+protein+D52+(TPD52)+is+overexpressed+and+a+gene+amplification+target+in+ovarian+cancer&amp;author=JA+Byrne&amp;author=RL+Balleine&amp;author=M+Schoenberg+Fejzo&amp;author=J+Mercieca&amp;author=YE+Chiew&amp;volume=117&amp;publication_year=2005&amp;pages=1049-1054&amp;pmid=15986428&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B16"><span class="mixed-citation">Carmichael J, DeGraff WG, Gazdar AF, Minna JD, Mitchell JB 1987.  <span class="ref-title">Evaluation of a tetrazolium-based semiautomated colorimetric assay: Assessment of radiosensitivity</span>. <span class="ref-journal">Cancer Res</span> <span class="ref-vol">47</span>: 943–946 [<a href="/pubmed/3802101" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Evaluation+of+a+tetrazolium-based+semiautomated+colorimetric+assay:+Assessment+of+radiosensitivity&amp;author=J+Carmichael&amp;author=WG+DeGraff&amp;author=AF+Gazdar&amp;author=JD+Minna&amp;author=JB+Mitchell&amp;volume=47&amp;publication_year=1987&amp;pages=943-946&amp;pmid=3802101&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B17"><span class="mixed-citation">Chistiakov DA, Voronova NV, Chistiakov PA 2008.  <span class="ref-title">Genetic variations in DNA repair genes, radiosensitivity to cancer and susceptibility to acute tissue reactions in radiotherapy-treated cancer patients</span>. <span class="ref-journal">Acta Oncol</span> <span class="ref-vol">47</span>: 809–824 [<a href="/pubmed/18568480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Oncol&amp;title=Genetic+variations+in+DNA+repair+genes,+radiosensitivity+to+cancer+and+susceptibility+to+acute+tissue+reactions+in+radiotherapy-treated+cancer+patients&amp;author=DA+Chistiakov&amp;author=NV+Voronova&amp;author=PA+Chistiakov&amp;volume=47&amp;publication_year=2008&amp;pages=809-824&amp;pmid=18568480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B18"><span class="mixed-citation">Choy E, Yelensky R, Bonakdar S, Plenge RM, Saxena R, De Jager PL, Shaw SY, Wolfish CS, Slavik JM, Cotsapas C, et al. 2008.  <span class="ref-title">Genetic analysis of human traits in vitro: Drug response and gene expression in lymphoblastoid cell lines</span>. <span class="ref-journal">PLoS Genet</span> <span class="ref-vol">4</span>: e1000287 doi: 10.1371/journal.pgen.1000287 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2583954/">PMC free article</a>]</span> [<a href="/pubmed/19043577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetic+analysis+of+human+traits+in+vitro:+Drug+response+and+gene+expression+in+lymphoblastoid+cell+lines&amp;author=E+Choy&amp;author=R+Yelensky&amp;author=S+Bonakdar&amp;author=RM+Plenge&amp;author=R+Saxena&amp;volume=4&amp;publication_year=2008&amp;pages=e1000287&amp;pmid=19043577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B19"><span class="mixed-citation">Correa CR, Cheung VG 2004.  <span class="ref-title">Genetic variation in radiation-induced expression phenotypes</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">75</span>: 885–890 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182118/">PMC free article</a>]</span> [<a href="/pubmed/15359380" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+variation+in+radiation-induced+expression+phenotypes&amp;author=CR+Correa&amp;author=VG+Cheung&amp;volume=75&amp;publication_year=2004&amp;pages=885-890&amp;pmid=15359380&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B20"><span class="mixed-citation">DeFilippis RA, Goodwin EC, Wu L, DiMaio D 2003.  <span class="ref-title">Endogenous human papillomavirus E6 and E7 proteins differentially regulate proliferation, senescence, and apoptosis in HeLa cervical carcinoma cells</span>. <span class="ref-journal">J Virol</span> <span class="ref-vol">77</span>: 1551–1563 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC140828/">PMC free article</a>]</span> [<a href="/pubmed/12502868" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Virol&amp;title=Endogenous+human+papillomavirus+E6+and+E7+proteins+differentially+regulate+proliferation,+senescence,+and+apoptosis+in+HeLa+cervical+carcinoma+cells&amp;author=RA+DeFilippis&amp;author=EC+Goodwin&amp;author=L+Wu&amp;author=D+DiMaio&amp;volume=77&amp;publication_year=2003&amp;pages=1551-1563&amp;pmid=12502868&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B21"><span class="mixed-citation">Dikomey E, Borgmann K, Peacock J, Jung H 2003.  <span class="ref-title">Why recent studies relating normal tissue response to individual radiosensitivity might have failed and how new studies should be performed</span>. <span class="ref-journal">Int J Radiat Oncol Biol Phys</span> <span class="ref-vol">56</span>: 1194–1200 [<a href="/pubmed/12829159" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Radiat+Oncol+Biol+Phys&amp;title=Why+recent+studies+relating+normal+tissue+response+to+individual+radiosensitivity+might+have+failed+and+how+new+studies+should+be+performed&amp;author=E+Dikomey&amp;author=K+Borgmann&amp;author=J+Peacock&amp;author=H+Jung&amp;volume=56&amp;publication_year=2003&amp;pages=1194-1200&amp;pmid=12829159&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B22"><span class="mixed-citation">Dimas AS, Deutsch S, Stranger BE, Montgomery SB, Borel C, Attar-Cohen H, Ingle C, Beazley C, Gutierrez Arcelus M, Sekowska M, et al. 2009.  <span class="ref-title">Common regulatory variation impacts gene expression in a cell type-dependent manner</span>. <span class="ref-journal">Science</span> <span class="ref-vol">325</span>: 1246–1250 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2867218/">PMC free article</a>]</span> [<a href="/pubmed/19644074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common+regulatory+variation+impacts+gene+expression+in+a+cell+type-dependent+manner&amp;author=AS+Dimas&amp;author=S+Deutsch&amp;author=BE+Stranger&amp;author=SB+Montgomery&amp;author=C+Borel&amp;volume=325&amp;publication_year=2009&amp;pages=1246-1250&amp;pmid=19644074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B23"><span class="mixed-citation">Dressman HK, Muramoto GG, Chao NJ, Meadows S, Marshall D, Ginsburg GS, Nevins JR, Chute JP 2007.  <span class="ref-title">Gene expression signatures that predict radiation exposure in mice and humans</span>. <span class="ref-journal">PLoS Med</span> <span class="ref-vol">4</span>: e106 doi: 10.1371/journal.pone.0001912 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1845155/">PMC free article</a>]</span> [<a href="/pubmed/17407386" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med&amp;title=Gene+expression+signatures+that+predict+radiation+exposure+in+mice+and+humans&amp;author=HK+Dressman&amp;author=GG+Muramoto&amp;author=NJ+Chao&amp;author=S+Meadows&amp;author=D+Marshall&amp;volume=4&amp;publication_year=2007&amp;pages=e106&amp;pmid=17407386&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B24"><span class="mixed-citation">Gerber DE, Chan TA 2008.  <span class="ref-title">Recent advances in radiation therapy</span>. <span class="ref-journal">Am Fam Physician</span> <span class="ref-vol">78</span>: 1254–1262 [<a href="/pubmed/19069018" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+Fam+Physician&amp;title=Recent+advances+in+radiation+therapy&amp;author=DE+Gerber&amp;author=TA+Chan&amp;volume=78&amp;publication_year=2008&amp;pages=1254-1262&amp;pmid=19069018&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B25"><span class="mixed-citation">Huang RS, Duan S, Kistner EO, Hartford CM, Dolan ME 2008.  <span class="ref-title">Genetic variants associated with carboplatin-induced cytotoxicity in cell lines derived from Africans</span>. <span class="ref-journal">Mol Cancer Ther</span> <span class="ref-vol">7</span>: 3038–3046 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2743011/">PMC free article</a>]</span> [<a href="/pubmed/18765826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cancer+Ther&amp;title=Genetic+variants+associated+with+carboplatin-induced+cytotoxicity+in+cell+lines+derived+from+Africans&amp;author=RS+Huang&amp;author=S+Duan&amp;author=EO+Kistner&amp;author=CM+Hartford&amp;author=ME+Dolan&amp;volume=7&amp;publication_year=2008&amp;pages=3038-3046&amp;pmid=18765826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B26"><span class="mixed-citation">Jen KY, Cheung VG 2005.  <span class="ref-title">Identification of novel p53 target genes in ionizing radiation response</span>. <span class="ref-journal">Cancer Res</span> <span class="ref-vol">65</span>: 7666–7673 [<a href="/pubmed/16140933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Identification+of+novel+p53+target+genes+in+ionizing+radiation+response&amp;author=KY+Jen&amp;author=VG+Cheung&amp;volume=65&amp;publication_year=2005&amp;pages=7666-7673&amp;pmid=16140933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B27"><span class="mixed-citation">Kwan T, Grundberg E, Koka V, Ge B, Lam KC, Dias C, Kindmark A, Mallmin H, Ljunggren O, Rivadeneira F, et al. 2009.  <span class="ref-title">Tissue effect on genetic control of transcript isoform variation</span>. <span class="ref-journal">PLoS Genet</span> <span class="ref-vol">5</span>: e1000608 doi: 10.1371/journal.pgen.1000608 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719916/">PMC free article</a>]</span> [<a href="/pubmed/19680542" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Tissue+effect+on+genetic+control+of+transcript+isoform+variation&amp;author=T+Kwan&amp;author=E+Grundberg&amp;author=V+Koka&amp;author=B+Ge&amp;author=KC+Lam&amp;volume=5&amp;publication_year=2009&amp;pages=e1000608&amp;pmid=19680542&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B28"><span class="mixed-citation">Lewis JD, Payton LA, Whitford JG, Byrne JA, Smith DI, Yang L, Bright RK 2007.  <span class="ref-title">Induction of tumorigenesis and metastasis by the murine orthologue of tumor protein D52</span>. <span class="ref-journal">Mol Cancer Res</span> <span class="ref-vol">5</span>: 133–144 [<a href="/pubmed/17314271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cancer+Res&amp;title=Induction+of+tumorigenesis+and+metastasis+by+the+murine+orthologue+of+tumor+protein+D52&amp;author=JD+Lewis&amp;author=LA+Payton&amp;author=JG+Whitford&amp;author=JA+Byrne&amp;author=DI+Smith&amp;volume=5&amp;publication_year=2007&amp;pages=133-144&amp;pmid=17314271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B29"><span class="mixed-citation">Li Y, Abecasis GR 2006.  <span class="ref-title">Mach 1.0: Rapid haplotype reconstruction and missing genotype inference</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">S79</span>: 2290 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Mach+1.0:+Rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B30"><span class="mixed-citation">Li L, Fridley B, Kalari K, Jenkins G, Batzler A, Safgren S, Hildebrandt M, Ames M, Schaid D, Wang L 2008.  <span class="ref-title">Gemcitabine and cytosine arabinoside cytotoxicity: Association with lymphoblastoid cell expression</span>. <span class="ref-journal">Cancer Res</span> <span class="ref-vol">68</span>: 7050–7058 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2562356/">PMC free article</a>]</span> [<a href="/pubmed/18757419" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Gemcitabine+and+cytosine+arabinoside+cytotoxicity:+Association+with+lymphoblastoid+cell+expression&amp;author=L+Li&amp;author=B+Fridley&amp;author=K+Kalari&amp;author=G+Jenkins&amp;author=A+Batzler&amp;volume=68&amp;publication_year=2008&amp;pages=7050-7058&amp;pmid=18757419&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B31"><span class="mixed-citation">Li L, Fridley BL, Kalari K, Jenkins G, Batzler A, Weinshilboum RM, Wang L 2009.  <span class="ref-title">Gemcitabine and arabinosylcytosin pharmacogenomics: Genome-wide association and drug response biomarkers</span>. <span class="ref-journal">PLoS ONE</span> <span class="ref-vol">4</span>: e7765 doi: 10.1371/journal.pone.0007765 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2770319/">PMC free article</a>]</span> [<a href="/pubmed/19898621" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Gemcitabine+and+arabinosylcytosin+pharmacogenomics:+Genome-wide+association+and+drug+response+biomarkers&amp;author=L+Li&amp;author=BL+Fridley&amp;author=K+Kalari&amp;author=G+Jenkins&amp;author=A+Batzler&amp;volume=4&amp;publication_year=2009&amp;pages=e7765&amp;pmid=19898621&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B32"><span class="mixed-citation">Lu TP, Lai LC, Lin BI, Chen LH, Hsiao TH, Liber HL, Cook JA, Mitchell JB, Tsai MH, Chuang EY 2010.  <span class="ref-title">Distinct signaling pathways after higher or lower doses of radiation in three closely related human lymphoblast cell lines</span>. <span class="ref-journal">Int J Radiat Oncol Biol Phys</span> <span class="ref-vol">76</span>: 212–219 [<a href="/pubmed/20005454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Radiat+Oncol+Biol+Phys&amp;title=Distinct+signaling+pathways+after+higher+or+lower+doses+of+radiation+in+three+closely+related+human+lymphoblast+cell+lines&amp;author=TP+Lu&amp;author=LC+Lai&amp;author=BI+Lin&amp;author=LH+Chen&amp;author=TH+Hsiao&amp;volume=76&amp;publication_year=2010&amp;pages=212-219&amp;pmid=20005454&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B33"><span class="mixed-citation">Macurek L, Lindqvist A, Lim D, Lampson MA, Klompmaker R, Freire R, Clouin C, Taylor SS, Yaffe MB, Medema RH 2008.  <span class="ref-title">Polo-like kinase-1 is activated by aurora A to promote checkpoint recovery</span>. <span class="ref-journal">Nature</span> <span class="ref-vol">455</span>: 119–123 [<a href="/pubmed/18615013" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Polo-like+kinase-1+is+activated+by+aurora+A+to+promote+checkpoint+recovery&amp;author=L+Macurek&amp;author=A+Lindqvist&amp;author=D+Lim&amp;author=MA+Lampson&amp;author=R+Klompmaker&amp;volume=455&amp;publication_year=2008&amp;pages=119-123&amp;pmid=18615013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B34"><span class="mixed-citation">Macurek L, Lindqvist A, Medema RH 2009.  <span class="ref-title">Aurora-A and hBora join the game of Polo</span>. <span class="ref-journal">Cancer Res</span> <span class="ref-vol">69</span>: 4555–4558 [<a href="/pubmed/19487276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Aurora-A+and+hBora+join+the+game+of+Polo&amp;author=L+Macurek&amp;author=A+Lindqvist&amp;author=RH+Medema&amp;volume=69&amp;publication_year=2009&amp;pages=4555-4558&amp;pmid=19487276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B35"><span class="mixed-citation">Paul S, Amundson SA 2008.  <span class="ref-title">Development of gene expression signatures for practical radiation biodosimetry</span>. <span class="ref-journal">Int J Radiat Oncol Biol Phys</span> <span class="ref-vol">71</span>: 1236–1244 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2478639/">PMC free article</a>]</span> [<a href="/pubmed/18572087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Radiat+Oncol+Biol+Phys&amp;title=Development+of+gene+expression+signatures+for+practical+radiation+biodosimetry&amp;author=S+Paul&amp;author=SA+Amundson&amp;volume=71&amp;publication_year=2008&amp;pages=1236-1244&amp;pmid=18572087&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B36"><span class="mixed-citation">Petrova DT, Asif AR, Armstrong VW, Dimova I, Toshev S, Yaramov N, Oellerich M, Toncheva D 2008.  <span class="ref-title">Expression of chloride intracellular channel protein 1 (CLIC1) and tumor protein D52 (TPD52) as potential biomarkers for colorectal cancer</span>. <span class="ref-journal">Clin Biochem</span> <span class="ref-vol">41</span>: 1224–1236 [<a href="/pubmed/18710659" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Biochem&amp;title=Expression+of+chloride+intracellular+channel+protein+1+(CLIC1)+and+tumor+protein+D52+(TPD52)+as+potential+biomarkers+for+colorectal+cancer&amp;author=DT+Petrova&amp;author=AR+Asif&amp;author=VW+Armstrong&amp;author=I+Dimova&amp;author=S+Toshev&amp;volume=41&amp;publication_year=2008&amp;pages=1224-1236&amp;pmid=18710659&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B37"><span class="mixed-citation">Pomerening JR 2009.  <span class="ref-title">Positive-feedback loops in cell cycle progression</span>. <span class="ref-journal">FEBS Lett</span> <span class="ref-vol">583</span>: 3388–3396 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2784177/">PMC free article</a>]</span> [<a href="/pubmed/19818353" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=Positive-feedback+loops+in+cell+cycle+progression&amp;author=JR+Pomerening&amp;volume=583&amp;publication_year=2009&amp;pages=3388-3396&amp;pmid=19818353&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B38"><span class="mixed-citation">Popanda O, Marquardt JU, Chang-Claude J, Schmezer P 2009.  <span class="ref-title">Genetic variation in normal tissue toxicity induced by ionizing radiation</span>. <span class="ref-journal">Mutat Res</span> <span class="ref-vol">667</span>: 58–69 [<a href="/pubmed/19022265" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mutat+Res&amp;title=Genetic+variation+in+normal+tissue+toxicity+induced+by+ionizing+radiation&amp;author=O+Popanda&amp;author=JU+Marquardt&amp;author=J+Chang-Claude&amp;author=P+Schmezer&amp;volume=667&amp;publication_year=2009&amp;pages=58-69&amp;pmid=19022265&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B39"><span class="mixed-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D 2006.  <span class="ref-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <span class="ref-journal">Nat Genet</span> <span class="ref-vol">38</span>: 904–909 [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B40"><span class="mixed-citation">Pugh TJ, Keyes M, Barclay L, Delaney A, Krzywinski M, Thomas D, Novik K, Yang C, Agranovich A, McKenzie M, et al. 2009.  <span class="ref-title">Sequence variant discovery in DNA repair genes from radiosensitive and radiotolerant prostate brachytherapy patients</span>. <span class="ref-journal">Clin Cancer Res</span> <span class="ref-vol">15</span>: 5008–5016 [<a href="/pubmed/19638463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Cancer+Res&amp;title=Sequence+variant+discovery+in+DNA+repair+genes+from+radiosensitive+and+radiotolerant+prostate+brachytherapy+patients&amp;author=TJ+Pugh&amp;author=M+Keyes&amp;author=L+Barclay&amp;author=A+Delaney&amp;author=M+Krzywinski&amp;volume=15&amp;publication_year=2009&amp;pages=5008-5016&amp;pmid=19638463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B41"><span class="mixed-citation">Rozenblum E, Vahteristo P, Sandberg T, Bergthorsson JT, Syrjakoski K, Weaver D, Haraldsson K, Johannsdottir HK, Vehmanen P, Nigam S, et al. 2002.  <span class="ref-title">A genomic map of a 6-Mb region at 13q21-q22 implicated in cancer development: Identification and characterization of candidate genes</span>. <span class="ref-journal">Hum Genet</span> <span class="ref-vol">110</span>: 111–121 [<a href="/pubmed/11935316" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=A+genomic+map+of+a+6-Mb+region+at+13q21-q22+implicated+in+cancer+development:+Identification+and+characterization+of+candidate+genes&amp;author=E+Rozenblum&amp;author=P+Vahteristo&amp;author=T+Sandberg&amp;author=JT+Bergthorsson&amp;author=K+Syrjakoski&amp;volume=110&amp;publication_year=2002&amp;pages=111-121&amp;pmid=11935316&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B42"><span class="mixed-citation">Rubin MA, Varambally S, Beroukhim R, Tomlins SA, Rhodes DR, Paris PL, Hofer MD, Storz-Schweizer M, Kuefer R, Fletcher JA, et al. 2004.  <span class="ref-title">Overexpression, amplification, and androgen regulation of TPD52 in prostate cancer</span>. <span class="ref-journal">Cancer Res</span> <span class="ref-vol">64</span>: 3814–3822 [<a href="/pubmed/15172988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Overexpression,+amplification,+and+androgen+regulation+of+TPD52+in+prostate+cancer&amp;author=MA+Rubin&amp;author=S+Varambally&amp;author=R+Beroukhim&amp;author=SA+Tomlins&amp;author=DR+Rhodes&amp;volume=64&amp;publication_year=2004&amp;pages=3814-3822&amp;pmid=15172988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B43"><span class="mixed-citation">Rzeszowska-Wolny J, Herok R, Widel M, Hancock R 2009.  <span class="ref-title">X-irradiation and bystander effects induce similar changes of transcript profiles in most functional pathways in human melanoma cells</span>. <span class="ref-journal">DNA Repair</span> <span class="ref-vol">8</span>: 732–738 [<a href="/pubmed/19272842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair&amp;title=X-irradiation+and+bystander+effects+induce+similar+changes+of+transcript+profiles+in+most+functional+pathways+in+human+melanoma+cells&amp;author=J+Rzeszowska-Wolny&amp;author=R+Herok&amp;author=M+Widel&amp;author=R+Hancock&amp;volume=8&amp;publication_year=2009&amp;pages=732-738&amp;pmid=19272842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B44"><span class="mixed-citation">Schaid DJ, Batzler AJ, Jenkins GD, Hildebrandt MA 2006.  <span class="ref-title">Exact tests of Hardy-Weinberg equilibrium and homogeneity of disequilibrium across strata</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">79</span>: 1071–1080 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1698709/">PMC free article</a>]</span> [<a href="/pubmed/17186465" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Exact+tests+of+Hardy-Weinberg+equilibrium+and+homogeneity+of+disequilibrium+across+strata&amp;author=DJ+Schaid&amp;author=AJ+Batzler&amp;author=GD+Jenkins&amp;author=MA+Hildebrandt&amp;volume=79&amp;publication_year=2006&amp;pages=1071-1080&amp;pmid=17186465&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B45"><span class="mixed-citation">Seki A, Coppinger JA, Du H, Jang CY, Yates JR III, Fang G 2008a.  <span class="ref-title">Plk1- and beta-TrCP-dependent degradation of Bora controls mitotic progression</span>. <span class="ref-journal">J Cell Biol</span> <span class="ref-vol">181</span>: 65–78 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2287288/">PMC free article</a>]</span> [<a href="/pubmed/18378770" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=Plk1-+and+beta-TrCP-dependent+degradation+of+Bora+controls+mitotic+progression&amp;author=A+Seki&amp;author=JA+Coppinger&amp;author=H+Du&amp;author=CY+Jang&amp;author=JR+Yates&amp;volume=181&amp;publication_year=2008a&amp;pages=65-78&amp;pmid=18378770&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B46"><span class="mixed-citation">Seki A, Coppinger JA, Jang CY, Yates JR, Fang G 2008b.  <span class="ref-title">Bora and the kinase Aurora a cooperatively activate the kinase Plk1 and control mitotic entry</span>. <span class="ref-journal">Science</span> <span class="ref-vol">320</span>: 1655–1658 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2834883/">PMC free article</a>]</span> [<a href="/pubmed/18566290" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Bora+and+the+kinase+Aurora+a+cooperatively+activate+the+kinase+Plk1+and+control+mitotic+entry&amp;author=A+Seki&amp;author=JA+Coppinger&amp;author=CY+Jang&amp;author=JR+Yates&amp;author=G+Fang&amp;volume=320&amp;publication_year=2008b&amp;pages=1655-1658&amp;pmid=18566290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B47"><span class="mixed-citation">Shuldiner AR, O'Connell JR, Bliden KP, Gandhi A, Ryan K, Horenstein RB, Damcott CM, Pakyz R, Tantry US, Gibson Q, et al. 2009.  <span class="ref-title">Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy</span>. <span class="ref-journal">JAMA</span> <span class="ref-vol">302</span>: 849–857 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3641569/">PMC free article</a>]</span> [<a href="/pubmed/19706858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Association+of+cytochrome+P450+2C19+genotype+with+the+antiplatelet+effect+and+clinical+efficacy+of+clopidogrel+therapy&amp;author=AR+Shuldiner&amp;author=JR+O'Connell&amp;author=KP+Bliden&amp;author=A+Gandhi&amp;author=K+Ryan&amp;volume=302&amp;publication_year=2009&amp;pages=849-857&amp;pmid=19706858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B48"><span class="mixed-citation">Sie L, Loong S, Tan EK 2009.  <span class="ref-title">Utility of lymphoblastoid cell lines</span>. <span class="ref-journal">J Neurosci Res</span> <span class="ref-vol">87</span>: 1953–1959 [<a href="/pubmed/19224581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci+Res&amp;title=Utility+of+lymphoblastoid+cell+lines&amp;author=L+Sie&amp;author=S+Loong&amp;author=EK+Tan&amp;volume=87&amp;publication_year=2009&amp;pages=1953-1959&amp;pmid=19224581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B49"><span class="mixed-citation">Sims AH, Finnon P, Miller CJ, Bouffler SD, Howell A, Scott D, Clarke RB 2007.  <span class="ref-title">TPD52 and NFKB1 gene expression levels correlate with G2 chromosomal radiosensitivity in lymphocytes of women with and at risk of hereditary breast cancer</span>. <span class="ref-journal">Int J Radiat Biol</span> <span class="ref-vol">83</span>: 409–420 [<a href="/pubmed/17487680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Radiat+Biol&amp;title=TPD52+and+NFKB1+gene+expression+levels+correlate+with+G2+chromosomal+radiosensitivity+in+lymphocytes+of+women+with+and+at+risk+of+hereditary+breast+cancer&amp;author=AH+Sims&amp;author=P+Finnon&amp;author=CJ+Miller&amp;author=SD+Bouffler&amp;author=A+Howell&amp;volume=83&amp;publication_year=2007&amp;pages=409-420&amp;pmid=17487680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B50"><span class="mixed-citation">Smirnov DA, Morley M, Shin E, Spielman RS, Cheung VG 2009.  <span class="ref-title">Genetic analysis of radiation-induced changes in human gene expression</span>. <span class="ref-journal">Nature</span> <span class="ref-vol">459</span>: 587–591 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3005325/">PMC free article</a>]</span> [<a href="/pubmed/19349959" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+analysis+of+radiation-induced+changes+in+human+gene+expression&amp;author=DA+Smirnov&amp;author=M+Morley&amp;author=E+Shin&amp;author=RS+Spielman&amp;author=VG+Cheung&amp;volume=459&amp;publication_year=2009&amp;pages=587-591&amp;pmid=19349959&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B51"><span class="mixed-citation">Storey JD, Tibshirani R 2003.  <span class="ref-title">Statistical significance for genomewide studies</span>. <span class="ref-journal">Proc Natl Acad Sci</span> <span class="ref-vol">100</span>: 9440–9445 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC170937/">PMC free article</a>]</span> [<a href="/pubmed/12883005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=Statistical+significance+for+genomewide+studies&amp;author=JD+Storey&amp;author=R+Tibshirani&amp;volume=100&amp;publication_year=2003&amp;pages=9440-9445&amp;pmid=12883005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B52"><span class="mixed-citation">Turesson I 1990.  <span class="ref-title">Individual variation and dose dependency in the progression rate of skin telangiectasia</span>. <span class="ref-journal">Int J Radiat Oncol Biol Phys</span> <span class="ref-vol">19</span>: 1569–1574 [<a href="/pubmed/2262383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Radiat+Oncol+Biol+Phys&amp;title=Individual+variation+and+dose+dependency+in+the+progression+rate+of+skin+telangiectasia&amp;author=I+Turesson&amp;volume=19&amp;publication_year=1990&amp;pages=1569-1574&amp;pmid=2262383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B53"><span class="mixed-citation">Turesson I, Nyman J, Holmberg E, Oden A 1996.  <span class="ref-title">Prognostic factors for acute and late skin reactions in radiotherapy patients</span>. <span class="ref-journal">Int J Radiat Oncol Biol Phys</span> <span class="ref-vol">36</span>: 1065–1075 [<a href="/pubmed/8985028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Radiat+Oncol+Biol+Phys&amp;title=Prognostic+factors+for+acute+and+late+skin+reactions+in+radiotherapy+patients&amp;author=I+Turesson&amp;author=J+Nyman&amp;author=E+Holmberg&amp;author=A+Oden&amp;volume=36&amp;publication_year=1996&amp;pages=1065-1075&amp;pmid=8985028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B54"><span class="mixed-citation">Ummanni R, Teller S, Junker H, Zimmermann U, Venz S, Scharf C, Giebel J, Walther R 2008.  <span class="ref-title">Altered expression of tumor protein D52 regulates apoptosis and migration of prostate cancer cells</span>. <span class="ref-journal">FEBS J</span> <span class="ref-vol">275</span>: 5703–5713 [<a href="/pubmed/18959755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+J&amp;title=Altered+expression+of+tumor+protein+D52+regulates+apoptosis+and+migration+of+prostate+cancer+cells&amp;author=R+Ummanni&amp;author=S+Teller&amp;author=H+Junker&amp;author=U+Zimmermann&amp;author=S+Venz&amp;volume=275&amp;publication_year=2008&amp;pages=5703-5713&amp;pmid=18959755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B55"><span class="mixed-citation">West CM, Davidson SE, Elyan SA, Valentine H, Roberts SA, Swindell R, Hunter RD 2001.  <span class="ref-title">Lymphocyte radiosensitivity is a significant prognostic factor for morbidity in carcinoma of the cervix</span>. <span class="ref-journal">Int J Radiat Oncol Biol Phys</span> <span class="ref-vol">51</span>: 10–15 [<a href="/pubmed/11516845" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Radiat+Oncol+Biol+Phys&amp;title=Lymphocyte+radiosensitivity+is+a+significant+prognostic+factor+for+morbidity+in+carcinoma+of+the+cervix&amp;author=CM+West&amp;author=SE+Davidson&amp;author=SA+Elyan&amp;author=H+Valentine&amp;author=SA+Roberts&amp;volume=51&amp;publication_year=2001&amp;pages=10-15&amp;pmid=11516845&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B56"><span class="mixed-citation">Westbury CB, Reis-Filho JS, Dexter T, Mahler-Araujo B, Fenwick K, Iravani M, Grigoriadis A, Parry S, Robertson D, Mackay A, et al. 2009.  <span class="ref-title">Genome-wide transcriptomic profiling of microdissected human breast tissue reveals differential expression of KIT (c-Kit, CD117) and oestrogen receptor-alpha (ERα) in response to therapeutic radiation</span>. <span class="ref-journal">J Pathol</span> <span class="ref-vol">219</span>: 131–140 [<a href="/pubmed/19562735" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pathol&amp;title=Genome-wide+transcriptomic+profiling+of+microdissected+human+breast+tissue+reveals+differential+expression+of+KIT+(c-Kit,+CD117)+and+oestrogen+receptor-alpha+(ERα)+in+response+to+therapeutic+radiation&amp;author=CB+Westbury&amp;author=JS+Reis-Filho&amp;author=T+Dexter&amp;author=B+Mahler-Araujo&amp;author=K+Fenwick&amp;volume=219&amp;publication_year=2009&amp;pages=131-140&amp;pmid=19562735&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B57"><span class="mixed-citation">Wigginton JE, Cutler DJ, Abecasis GR 2005.  <span class="ref-title">A note on exact tests of Hardy-Weinberg equilibrium</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">76</span>: 887–893 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1199378/">PMC free article</a>]</span> [<a href="/pubmed/15789306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+note+on+exact+tests+of+Hardy-Weinberg+equilibrium&amp;author=JE+Wigginton&amp;author=DJ+Cutler&amp;author=GR+Abecasis&amp;volume=76&amp;publication_year=2005&amp;pages=887-893&amp;pmid=15789306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B58"><span class="mixed-citation">Wu ZJ, Irizarry RA, Gentleman R, Martinez-Murillo F, Spencer F 2004.  <span class="ref-title">A model-based background adjustment for oligonucleotide expression arrays</span>. <span class="ref-journal">J Am Stat Assoc</span> <span class="ref-vol">99</span>: 909–917 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Stat+Assoc&amp;title=A+model-based+background+adjustment+for+oligonucleotide+expression+arrays&amp;author=ZJ+Wu&amp;author=RA+Irizarry&amp;author=R+Gentleman&amp;author=F+Martinez-Murillo&amp;author=F+Spencer&amp;volume=99&amp;publication_year=2004&amp;pages=909-917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B59"><span class="mixed-citation">Zhu H, Lam DC, Han KC, Tin VP, Suen WS, Wang E, Lam WK, Cai WW, Chung LP, Wong MP 2007.  <span class="ref-title">High resolution analysis of genomic aberrations by metaphase and array comparative genomic hybridization identifies candidate tumour genes in lung cancer cell lines</span>. <span class="ref-journal">Cancer Lett</span> <span class="ref-vol">245</span>: 303–314 [<a href="/pubmed/16517066" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Lett&amp;title=High+resolution+analysis+of+genomic+aberrations+by+metaphase+and+array+comparative+genomic+hybridization+identifies+candidate+tumour+genes+in+lung+cancer+cell+lines&amp;author=H+Zhu&amp;author=DC+Lam&amp;author=KC+Han&amp;author=VP+Tin&amp;author=WS+Suen&amp;volume=245&amp;publication_year=2007&amp;pages=303-314&amp;pmid=16517066&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2963812&amp;issue-id=191029&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140386676570144title">Abstract</a></li><li><a href="#__sec1title">Results</a></li><li><a href="#__sec7title">Discussion</a></li><li><a href="#__sec8title">Methods</a></li><li><a href="#idm140386748198416title">Acknowledgments</a></li><li><a href="#idm140386680837520title">Footnotes</a></li><li><a href="#idm140386672382208title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Genome Research</span> are provided here courtesy of <strong>Cold Spring Harbor Laboratory Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2963812/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2963812/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2963812/pdf/1482.pdf">PDF (935K)</a></li> | <li><a href="#" data-citationid="PMC2963812" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2963812%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2963812%2F&amp;text=Radiation%20pharmacogenomics%3A%20A%20genome-wide%20association%20approach%20to%20identify%20radiation%20response%20biomarkers%20using%20human%20lymphoblastoid%20cell%20lines" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2963812%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20923822" data-aiid="2963812" data-aid="2963812" data-iid="191029" data-domainid="102" data-domain="genores" data-accid="PMC2963812" data-md5="d2f258faf6a03630dd5156e03cf7b543"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:02:54-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B6C88DD513FC10000000005DB03BC&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2963812%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=genores&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2963812/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>