<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2860887/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of homocysteine levels in Filipinos provides evidence for CPS1 in women and a stronger MTHFR effect in young adults</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2860887/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study of homocysteine levels in Filipinos provides evidence for CPS1 in women and a stronger MTHFR effect in young adults"><meta name="citation_authors" content="Leslie A. Lange, Damien C. Croteau-Chonka, Amanda F. Marvelle, Li Qin, Kyle J. Gaulton, Christopher W. Kuzawa, Thomas W. McDade, Yunfei Wang, Yun Li, Shawn Levy, Judith B. Borja, Ethan M. Lange, Linda S. Adair, Karen L. Mohlke"><meta name="citation_date" content="15 May 2010"><meta name="citation_issue" content="10"><meta name="citation_volume" content="19"><meta name="citation_firstpage" content="2050"><meta name="citation_doi" content="10.1093/hmg/ddq062"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2860887/?report=abstract"><meta name="citation_pmid" content="20154341"><meta name="DC.Title" content="Genome-wide association study of homocysteine levels in Filipinos provides evidence for CPS1 in women and a stronger MTHFR effect in young adults"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Leslie A. Lange"><meta name="DC.Contributor" content="Damien C. Croteau-Chonka"><meta name="DC.Contributor" content="Amanda F. Marvelle"><meta name="DC.Contributor" content="Li Qin"><meta name="DC.Contributor" content="Kyle J. Gaulton"><meta name="DC.Contributor" content="Christopher W. Kuzawa"><meta name="DC.Contributor" content="Thomas W. McDade"><meta name="DC.Contributor" content="Yunfei Wang"><meta name="DC.Contributor" content="Yun Li"><meta name="DC.Contributor" content="Shawn Levy"><meta name="DC.Contributor" content="Judith B. Borja"><meta name="DC.Contributor" content="Ethan M. Lange"><meta name="DC.Contributor" content="Linda S. Adair"><meta name="DC.Contributor" content="Karen L. Mohlke"><meta name="DC.Date" content="2010 May 15"><meta name="DC.Identifier" content="10.1093/hmg/ddq062"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of homocysteine levels in Filipinos provides evidence for CPS1 in women and a stronger MTHFR effect in young adults"><meta property="og:type" content="article"><meta property="og:description" content="Plasma homocysteine (Hcy) level is associated with cardiovascular disease and may play an etiologic role in vascular damage, a precursor for atherosclerosis. We performed a genome-wide association study for Hcy in 1786 unrelated Filipino women from the ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2860887/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2860887/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2860887/pdf/ddq062.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88BCF2DD50B7C100000000002C001B.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2860887%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC2860887</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2010 May 15; 19(10): 2050–2058. </span></div><div><span class="fm-vol-iss-date">Published online 2010 Feb 13. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddq062" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddq062</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2860887</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20154341">20154341</a></div></div></div></div><h1 class="content-title">Genome-wide association study of homocysteine levels in Filipinos provides evidence for <em>CPS1</em> in women and a stronger <em>MTHFR</em> effect in young adults</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Lange%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221844040464" co-class="co-affbox">Leslie A. Lange</a>,<sup>1</sup> <a href="/pubmed/?term=Croteau-Chonka%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221847410256" co-class="co-affbox">Damien C. Croteau-Chonka</a>,<sup>1</sup> <a href="/pubmed/?term=Marvelle%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221844052016" co-class="co-affbox">Amanda F. Marvelle</a>,<sup>1</sup> <a href="/pubmed/?term=Qin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221838284864" co-class="co-affbox">Li Qin</a>,<sup>1</sup> <a href="/pubmed/?term=Gaulton%20KJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221838282640" co-class="co-affbox">Kyle J. Gaulton</a>,<sup>1</sup> <a href="/pubmed/?term=Kuzawa%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221840919824" co-class="co-affbox">Christopher W. Kuzawa</a>,<sup>4</sup> <a href="/pubmed/?term=McDade%20TW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221841493888" co-class="co-affbox">Thomas W. McDade</a>,<sup>4</sup> <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221841491664" co-class="co-affbox">Yunfei Wang</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221839280832" co-class="co-affbox">Yun Li</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Levy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221840092416" co-class="co-affbox">Shawn Levy</a>,<sup>5</sup> <a href="/pubmed/?term=Borja%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221842653072" co-class="co-affbox">Judith B. Borja</a>,<sup>6</sup> <a href="/pubmed/?term=Lange%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221842650848" co-class="co-affbox">Ethan M. Lange</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Adair%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221836228720" co-class="co-affbox">Linda S. Adair</a>,<sup>3</sup> and  <a href="/pubmed/?term=Mohlke%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341" class="affpopup" co-rid="_co_idm140221842523824" co-class="co-affbox">Karen L. Mohlke</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140221844040464"><h3 class="no_margin">Leslie A. Lange</h3><p><sup>1</sup>Department of Genetics,</p><div>Find articles by <a href="/pubmed/?term=Lange%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Leslie A. Lange</a></div></div><div id="_co_idm140221847410256"><h3 class="no_margin">Damien C. Croteau-Chonka</h3><p><sup>1</sup>Department of Genetics,</p><div>Find articles by <a href="/pubmed/?term=Croteau-Chonka%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Damien C. Croteau-Chonka</a></div></div><div id="_co_idm140221844052016"><h3 class="no_margin">Amanda F. Marvelle</h3><p><sup>1</sup>Department of Genetics,</p><div>Find articles by <a href="/pubmed/?term=Marvelle%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Amanda F. Marvelle</a></div></div><div id="_co_idm140221838284864"><h3 class="no_margin">Li Qin</h3><p><sup>1</sup>Department of Genetics,</p><div>Find articles by <a href="/pubmed/?term=Qin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Li Qin</a></div></div><div id="_co_idm140221838282640"><h3 class="no_margin">Kyle J. Gaulton</h3><p><sup>1</sup>Department of Genetics,</p><div>Find articles by <a href="/pubmed/?term=Gaulton%20KJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Kyle J. Gaulton</a></div></div><div id="_co_idm140221840919824"><h3 class="no_margin">Christopher W. Kuzawa</h3><p><sup>4</sup>Department of Anthropology, Institute for Policy Research, Northwestern University, Evanston, IL 60208, USA,</p><div>Find articles by <a href="/pubmed/?term=Kuzawa%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Christopher W. Kuzawa</a></div></div><div id="_co_idm140221841493888"><h3 class="no_margin">Thomas W. McDade</h3><p><sup>4</sup>Department of Anthropology, Institute for Policy Research, Northwestern University, Evanston, IL 60208, USA,</p><div>Find articles by <a href="/pubmed/?term=McDade%20TW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Thomas W. McDade</a></div></div><div id="_co_idm140221841491664"><h3 class="no_margin">Yunfei Wang</h3><p><sup>1</sup>Department of Genetics,</p><p><sup>2</sup>Department of Biostatistics,  and</p><div>Find articles by <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Yunfei Wang</a></div></div><div id="_co_idm140221839280832"><h3 class="no_margin">Yun Li</h3><p><sup>1</sup>Department of Genetics,</p><p><sup>2</sup>Department of Biostatistics,  and</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Yun Li</a></div></div><div id="_co_idm140221840092416"><h3 class="no_margin">Shawn Levy</h3><p><sup>5</sup>Department of Biomedical Informatics, Vanderbilt University, Nashville, TN 37232, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Levy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Shawn Levy</a></div></div><div id="_co_idm140221842653072"><h3 class="no_margin">Judith B. Borja</h3><p><sup>6</sup>Office of Population Studies Foundation, University of San Carlos, Cebu City 6000, Philippines</p><div>Find articles by <a href="/pubmed/?term=Borja%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Judith B. Borja</a></div></div><div id="_co_idm140221842650848"><h3 class="no_margin">Ethan M. Lange</h3><p><sup>1</sup>Department of Genetics,</p><p><sup>2</sup>Department of Biostatistics,  and</p><div>Find articles by <a href="/pubmed/?term=Lange%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Ethan M. Lange</a></div></div><div id="_co_idm140221836228720"><h3 class="no_margin">Linda S. Adair</h3><p><sup>3</sup>Department of Nutrition, University of North Carolina, Chapel Hill, NC 27599, USA,</p><div>Find articles by <a href="/pubmed/?term=Adair%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Linda S. Adair</a></div></div><div id="_co_idm140221842523824"><h3 class="no_margin">Karen L. Mohlke</h3><p><sup>1</sup>Department of Genetics,</p><div>Find articles by <a href="/pubmed/?term=Mohlke%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20154341">Karen L. Mohlke</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140221846959792_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140221846959792_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140221846959792_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140221846959792_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Genetics,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Biostatistics,  and</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Nutrition, University of North Carolina, Chapel Hill, NC 27599, USA,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Anthropology, Institute for Policy Research, Northwestern University, Evanston, IL 60208, USA,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Biomedical Informatics, Vanderbilt University, Nashville, TN 37232, USA,  and</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Office of Population Studies Foundation, University of San Carlos, Cebu City 6000, Philippines</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Department of Genetics, University of North Carolina, 5096 Genetic Medicine, Chapel Hill, NC 27599-7264, USA. Tel: +1 9199662913; Fax: +1 9198430291; Email: <a href="mailto:dev@null" data-email="ude.cnu.dem@eklhom" class="oemail">ude.cnu.dem@eklhom</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140221846959792_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Nov 12; Revised 2010 Jan 21; Accepted 2010 Feb 11.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140221846959792_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2010. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oxfordjournals.org</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2860887/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               [Supplementary Data]
            </div>
            
            <div><a href="/pmc/articles/PMC2860887/bin/ddq062_index.html" data-ga-action="click_feat_suppl">ddq062_index.html</a><span style="color:gray"> (839 bytes)</span></div><div class="small guid">GUID:&nbsp;A273FD86-10CF-4CEC-9142-18E2FE62AFD2</div>
            <div><a href="/pmc/articles/PMC2860887/bin/ddq062_1.pdf" data-ga-action="click_feat_suppl">ddq062_1.pdf</a><span style="color:gray"> (96K)</span></div><div class="small guid">GUID:&nbsp;C6B73324-8AB7-42EF-849B-8758A4C7CCB1</div>
         </div></dd></dl></div><div id="idm140221843337024" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140221843337024title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Plasma homocysteine (Hcy) level is associated with cardiovascular disease and may play an etiologic role in vascular damage, a precursor for atherosclerosis. We performed a genome-wide association study for Hcy in 1786 unrelated Filipino women from the Cebu Longitudinal Health and Nutrition Survey (CLHNS). The most strongly associated single-nucleotide polymorphism (SNP) (rs7422339, <em>P</em> = 4.7 × 10<sup>−13</sup>) encodes Thr1405Asn in the gene <em>CPS1</em> and explained 3.0% of variation in the Hcy level. The widely studied <em>MTHFR</em> C677T SNP (rs1801133) was also highly significant (<em>P</em> = 8.7 × 10<sup>−10</sup>) and explained 1.6% of the trait variation. We also genotyped these two SNPs in 1679 CLHNS young adult offspring. The <em>MTHFR</em> C677T SNP was strongly associated with Hcy (<em>P</em> = 1.9 × 10<sup>−26</sup>) and explained ∼5.1% of the variation in the offspring. In contrast, the <em>CPS1</em> variant was significant only in females (<em>P</em> = 0.11 in all; <em>P</em> = 0.0087 in females). Combined analysis of all samples confirmed that the <em>MTHFR</em> variant was more strongly associated with Hcy in the offspring (interaction <em>P</em> = 1.2 × 10<sup>−5</sup>). Furthermore, although there was evidence for a positive synergistic effect between the <em>CPS1</em> and <em>MTHFR</em> SNPs in the offspring (interaction <em>P</em> = 0.0046), there was no significant evidence for an interaction in the mothers (<em>P</em> = 0.55). These data confirm a recent finding that <em>CPS1</em> is a locus influencing Hcy levels in women and suggest that genetic effects on Hcy may differ across developmental stages.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Homocysteine (Hcy) is a sulfur-containing amino acid produced in the metabolism of the essential amino acid methionine, which occurs through folic acid and vitamin B<sub>6</sub>- and vitamin B<sub>12</sub>-dependent pathways (<a href="#DDQ062C1" rid="DDQ062C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138881">1</a>). Increased blood plasma Hcy levels have been shown to be a risk factor for cardiovascular disease (CVD) and may play an etiologic role in vascular damage by promoting oxidative stress, systemic inflammation and endothelial dysfunction (<a href="#DDQ062C2" rid="DDQ062C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138883">2</a>). Numerous epidemiologic studies have reported that elevated Hcy is associated with both subclinical and clinical CVD, and evidence supports that the increased risk is independent of other atherosclerosis risk factors (<a href="#DDQ062C3" rid="DDQ062C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138882">3</a>–<a href="#DDQ062C8" rid="DDQ062C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138872">8</a>).</p><p id="__p3">Genetic factors play a substantial role in Hcy levels, but whether the same variants influence Hcy across populations is unknown. Heritability estimates range from 47–70% across several genetic and environmental backgrounds (<a href="#DDQ062C9" rid="DDQ062C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_335635055">9</a>–<a href="#DDQ062C11" rid="DDQ062C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138887">11</a>). The most well-studied gene containing variants associated with Hcy levels is methylene tetrahydrofolate reductase (<em>MTHFR</em>), which encodes a key enzyme in folic acid metabolism and was first identified as the causal gene for three rare autosomal homocysteinurias (<a href="#DDQ062C12" rid="DDQ062C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138868">12</a>). A meta-analysis of the <em>MTHFR</em> C677T variant (rs1801133) in 33 studies representing 35 populations showed a mean difference in serum Hcy between the TT and CC genotypes of 2.7 mmol/L (2.1–3.4 mmol/L) and between the CT and CC genotypes of 0.29 mmol/L (0.20–0.39 mmol/L) (<a href="#DDQ062C8" rid="DDQ062C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138879">8</a>). More recently, genome-wide association (GWA) studies for Hcy levels in populations of European descent identified novel loci (<a href="#DDQ062C13" rid="DDQ062C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138864">13</a>,<a href="#DDQ062C14" rid="DDQ062C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138865">14</a>).</p><p id="__p4" class="p p-last">To further clarify the genetic contributions to normal variation in Hcy in a population of primarily non-European descent, we performed a GWA study of the Hcy level in mothers from the Cebu Longitudinal Health and Nutrition Survey (CLHNS), using single-nucleotide polymorphism (SNP) data on 1786 women from Cebu, the Philippines. We also genotyped selected SNPs and tested them for association with the Hcy level in their young adult offspring to further evaluate genetic effects in males and across a wider age range.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">GWA results in CLHNS mothers</h3><p id="__p5" class="p p-first">In 1786 CLHNS mothers with complete genotype and phenotype data (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>), we observed 22 SNPs significantly associated with natural log-transformed Hcy (log-Hcy) levels (<em>P</em> &lt; 5 × 10<sup>−8</sup>), all located within or near the carbamoyl phosphate synthetase 1 (<em>CPS1</em>) gene on chromosome 2 (Fig.&nbsp;<a href="/pmc/articles/PMC2860887/figure/DDQ062F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ062F1" rid-ob="ob-DDQ062F1" co-legend-rid="lgnd_DDQ062F1"><span>1</span></a>). The most strongly associated SNP (rs7422339, <em>P</em> = 4.7 × 10<sup>−13</sup>) was imputed, and encodes a Thr1405Asn substitution. Tests of association for the other <em>CPS1</em> SNPs conditional on rs7422339 suggested a single <em>CPS1</em> signal. The most significant SNP outside the <em>CPS1</em> region was rs6722617 (<em>P</em> = 1.9 × 10<sup>−7</sup>), an imputed SNP located on chromosome 2q11.2 near lysozyme G-like 1 (<em>LYG1</em>) and thioredoxin domain containing 9 (<em>TXNDC9</em>), neither of which has been previously implicated as having effects on Hcy or other CVD-related traits. Several <em>MTHFR</em> SNPs showed modest evidence of association (best <em>P</em> = 2.7 × 10<sup>−4</sup> for rs13306561). However, the widely studied <em>MTHFR</em> C677T variant (rs1801133) was not directly tested as part of the genome-wide panel, and the imputation quality score did not meet the minimum requirement for inclusion in analysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ062F1" co-legend-rid="lgnd_DDQ062F1"><a href="/pmc/articles/PMC2860887/figure/DDQ062F1/" target="figure" rid-figpopup="DDQ062F1" rid-ob="ob-DDQ062F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140221842668288" class="figure"><a href="/pmc/articles/PMC2860887/figure/DDQ062F1/" target="figure" rid-figpopup="DDQ062F1" rid-ob="ob-DDQ062F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2860887_ddq06201.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq06201.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2860887/bin/ddq06201.jpg"></a></div><div id="largeobj_idm140221842668288" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2860887/figure/DDQ062F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ062F1"><div><a class="figpopup" href="/pmc/articles/PMC2860887/figure/DDQ062F1/" target="figure" rid-figpopup="DDQ062F1" rid-ob="ob-DDQ062F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p6">GWA results for log-Hcy levels in 1786 CLHNS mothers. Result for directly genotyped SNP rs1801133 (<em>MTHFR</em>) is included. Loci described in the text are annotated with gene symbols.</p></div></div></div><p id="__p7" class="p">Four analyses showed little-to-no evidence for genetic heterogeneity among CLHNS subjects: principal component (PC) analysis of population substructure (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>), a minimal effect of adjusting for PCs (data not shown), the observed genomic control inflation factor (λ<sub>GC</sub>) of 1.0 and a quantile–quantile plot of GWA <em>P</em>-values (Fig.&nbsp;<a href="/pmc/articles/PMC2860887/figure/DDQ062F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ062F2" rid-ob="ob-DDQ062F2" co-legend-rid="lgnd_DDQ062F2"><span>2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ062F2" co-legend-rid="lgnd_DDQ062F2"><a href="/pmc/articles/PMC2860887/figure/DDQ062F2/" target="figure" rid-figpopup="DDQ062F2" rid-ob="ob-DDQ062F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140221845158512" class="figure"><a href="/pmc/articles/PMC2860887/figure/DDQ062F2/" target="figure" rid-figpopup="DDQ062F2" rid-ob="ob-DDQ062F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2860887_ddq06202.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq06202.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2860887/bin/ddq06202.jpg"></a></div><div id="largeobj_idm140221845158512" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2860887/figure/DDQ062F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ062F2"><div><a class="figpopup" href="/pmc/articles/PMC2860887/figure/DDQ062F2/" target="figure" rid-figpopup="DDQ062F2" rid-ob="ob-DDQ062F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p8">Quantile–quantile plot of SNP association with log-Hcy in CLHNS mothers. Black closed circles indicate all SNPs and dark-gray closed circles exclude all SNPs in 1 Mb windows centered on each previously reported SNP associated with log-Hcy (Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB2" rid-ob="ob-DDQ062TB2" co-legend-rid=""><span>2</span></a>). A solid black line indicates the uniform expectation and the gray dashed lines indicate the 95% prediction interval.</p></div></div></div></div><div id="s2b" class="sec"><h3 id="s2btitle">Follow-up genotyping</h3><p id="__p9" class="p p-first">The most significant variants in the <em>CPS1</em> and <em>LYG1-TXNDC9</em> regions were both imputed SNPs. To confirm association in the mothers and to test these variants for association with log-Hcy in the offspring (<em>n</em> = 1679; 54.8% male, <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>), we directly genotyped rs7422339 and rs6722617. We also genotyped the <em>MTHFR</em> SNP rs1801133 in both the mothers and the offspring.</p><p id="__p10">The test of association between log-Hcy and the directly genotyped <em>CPS1</em> SNP rs7422339 remained significant in the CLHNS mothers (<em>P</em> = 5.3 × 10<sup>−12</sup>, Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB1" rid-ob="ob-DDQ062TB1" co-legend-rid=""><span>1</span></a>). The estimated increase in log-Hcy for each additional A allele was 0.071 [standard error (SE) = 0.009], and the proportion of variability in the log-Hcy level explained by genotype was 3.0%. This SNP was not significantly associated with log-Hcy levels (<em>P</em> = 0.11) in the pooled CLHNS offspring sample (Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB1" rid-ob="ob-DDQ062TB1" co-legend-rid=""><span>1</span></a>). However, evidence for association was present in the female-only subset (<em>n</em> = 759; <em>P</em> = 0.0087), with the direction of effect the same as observed in the mothers (increase of 0.040 in mean log-Hcy for each additional A allele). A formal test for a gender-by-genotype interaction in the offspring-only sample was not significant (<em>P</em> = 0.095), but the statistical power to detect such an interaction was only 54% for the effect size observed. The test for interaction between rs7422339 genotype and generation (mothers versus offspring) was significant (<em>P</em> = 0.0004). There was no significant evidence for an interaction between age and genotype in the mothers (<em>P</em> &gt; 0.10).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ062TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p11">Evidence of log-Hcy association with <em>CPS1</em> rs7422339 and <em>MTHFR</em> C677T</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140221845050320" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left"><em>n</em></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value for additive effect<sup>a</sup></th><th rowspan="1" colspan="1" align="left">β-Coefficient (SE)<sup>a</sup></th><th rowspan="1" colspan="1" align="left"><em>R</em><sup>2</sup> (%)</th><th colspan="3" rowspan="1" align="left">Genotypic least-square means for Hcy<hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>CPS1</em> rs7422339</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CC</td><td rowspan="1" colspan="1">CA</td><td rowspan="1" colspan="1">AA</td></tr><tr><td rowspan="1" colspan="1">Mothers</td><td rowspan="1" colspan="1">1746</td><td rowspan="1" colspan="1">5.3 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">0.071 (0.009)</td><td rowspan="1" colspan="1">3.0</td><td rowspan="1" colspan="1">11.02</td><td rowspan="1" colspan="1">11.92</td><td rowspan="1" colspan="1">12.46</td></tr><tr><td rowspan="1" colspan="1">Female offspring</td><td rowspan="1" colspan="1">759</td><td rowspan="1" colspan="1">0.0087</td><td rowspan="1" colspan="1">0.040 (0.015)</td><td rowspan="1" colspan="1">0.9</td><td rowspan="1" colspan="1">10.46</td><td rowspan="1" colspan="1">10.98</td><td rowspan="1" colspan="1">11.60</td></tr><tr><td rowspan="1" colspan="1">Male offspring</td><td rowspan="1" colspan="1">920</td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">0.001 (0.017)</td><td rowspan="1" colspan="1">&lt;0.1</td><td rowspan="1" colspan="1">14.36</td><td rowspan="1" colspan="1">14.50</td><td rowspan="1" colspan="1">13.78</td></tr><tr><td rowspan="1" colspan="1">All offspring</td><td rowspan="1" colspan="1">1679</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.021 (0.012)</td><td rowspan="1" colspan="1">0.2</td><td rowspan="1" colspan="1">12.58</td><td rowspan="1" colspan="1">12.91</td><td rowspan="1" colspan="1">12.79</td></tr><tr><td rowspan="1" colspan="1">Combined mothers + offspring</td><td rowspan="1" colspan="1">3425</td><td rowspan="1" colspan="1">5.4 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.050 (0.008)</td><td rowspan="1" colspan="1">1.1</td><td rowspan="1" colspan="1">11.80</td><td rowspan="1" colspan="1">12.43</td><td rowspan="1" colspan="1">12.79</td></tr><tr><td rowspan="1" colspan="1"><em>MTHFR</em> C677T</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CC</td><td rowspan="1" colspan="1">CT</td><td rowspan="1" colspan="1">TT</td></tr><tr><td rowspan="1" colspan="1">Mothers</td><td rowspan="1" colspan="1">1769</td><td rowspan="1" colspan="1">8.7<em>×</em>10<sup>−10</sup></td><td rowspan="1" colspan="1">0.083 (0.015)</td><td rowspan="1" colspan="1">1.6</td><td rowspan="1" colspan="1">11.25</td><td rowspan="1" colspan="1">11.94</td><td rowspan="1" colspan="1">22.17</td></tr><tr><td rowspan="1" colspan="1">Female offspring</td><td rowspan="1" colspan="1">753</td><td rowspan="1" colspan="1">3.6 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.136 (0.022)</td><td rowspan="1" colspan="1">5.1</td><td rowspan="1" colspan="1">10.42</td><td rowspan="1" colspan="1">11.46</td><td rowspan="1" colspan="1">21.43</td></tr><tr><td rowspan="1" colspan="1">Male offspring</td><td rowspan="1" colspan="1">923</td><td rowspan="1" colspan="1">1.1 × 10<sup>−16</sup></td><td rowspan="1" colspan="1">0.199 (0.023)</td><td rowspan="1" colspan="1">7.3</td><td rowspan="1" colspan="1">13.81</td><td rowspan="1" colspan="1">16.38</td><td rowspan="1" colspan="1">35.11</td></tr><tr><td rowspan="1" colspan="1">All offspring</td><td rowspan="1" colspan="1">1676</td><td rowspan="1" colspan="1">1.9 × 10<sup>−26</sup></td><td rowspan="1" colspan="1">0.171 (0.016)</td><td rowspan="1" colspan="1">6.3</td><td rowspan="1" colspan="1">12.28</td><td rowspan="1" colspan="1">14.22</td><td rowspan="1" colspan="1">29.13</td></tr><tr><td rowspan="1" colspan="1">Combined mothers + offspring</td><td rowspan="1" colspan="1">3445</td><td rowspan="1" colspan="1">1.3 × 10<sup>−29</sup></td><td rowspan="1" colspan="1">0.129 (0.012)</td><td rowspan="1" colspan="1">3.3</td><td rowspan="1" colspan="1">11.80</td><td rowspan="1" colspan="1">12.44</td><td rowspan="1" colspan="1">12.86</td></tr></tbody></table></div><div id="largeobj_idm140221845050320" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2860887/table/DDQ062TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140221839465920"><p id="__p12" class="p p-first-last">Results of directly genotyped SNPs are shown.</p></div><div id="idm140221839465296"><p id="__p13" class="p p-first-last"><sup>a</sup>For tests of association and effect estimation, natural log-Hcy was used. β-Coefficients and SEs are reported for an additive effect.</p></div></div></div><p id="__p14">Evidence of log-Hcy association for the directly genotyped <em>LYG1-TXNDC9</em> SNP rs6722617 (<em>P</em> = 1.2 × 10<sup>−7</sup>) was similar to the result based on its imputed genotype. The estimated increase for log-Hcy was 0.62 (SE = 0.23) for each additional C allele. No evidence for association between rs6722617 and log-Hcy was observed in the offspring-only sample (<em>P</em> = 0.58).</p><p id="__p15">The directly genotyped <em>MTHFR</em> C677T SNP showed convincing evidence of association (<em>P</em> = 8.7 × 10<sup>−10</sup>) and explained 1.6% of the proportion of variability in log-Hcy in the CLHNS mothers (Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB1" rid-ob="ob-DDQ062TB1" co-legend-rid=""><span>1</span></a>). The estimated increase in log-Hcy for each additional copy of the T allele was 0.083 (SE = 0.015). This SNP was even more strongly associated with log-Hcy levels in the gender-combined offspring sample (<em>P</em> = 1.9 × 10<sup>−26</sup>, Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB1" rid-ob="ob-DDQ062TB1" co-legend-rid=""><span>1</span></a>). Each additional copy of the T allele was associated with an increase in log-Hcy of 0.171 (SE = 0.016), and the proportion of variability in the log-Hcy level explained by this locus was 5.1%. The gender-stratified results in the CLHNS offspring indicated that the <em>MTHFR</em> C677T association appeared slightly stronger in males (<em>P</em> = 1.1 × 10<sup>−16</sup>; β = 0.199; <em>R</em><sup>2</sup> = 7.3%) than females (<em>P</em> = 3.6 × 10<sup>−10</sup>; β = 0.136; <em>R</em><sup>2</sup> = 5.1%), and there was marginal evidence for a gender-by-genotype interaction (<em>P</em> = 0.053). The combined study of mothers and offspring for <em>MTHFR</em> C677T resulted in a smaller <em>P</em>-value (<em>P</em> = 1.3 × 10<sup>−29</sup>) than when analyses were restricted to the offspring alone. Evidence for an interaction between genotype and generation (<em>P</em> = 1.2 × 10<sup>−5</sup>) suggests that the increase in mean log-Hcy levels associated with each additional T allele is greater in the offspring cohort than in the mothers, although this result may be somewhat confounded by the marginal evidence for a gender interaction in the offspring. We did not observe evidence for an interaction between age and genotype in the mothers (<em>P</em> &gt; 0.10). The <em>MTHFR</em> C677T genotypic means in the CLHNS mothers and offspring are most consistent with a recessive model for the T allele. The sample sizes for the 677T homozygotes (<em>n</em> = 7 for mothers and <em>n</em> = 16 for offspring), however, were too small to test this model.</p><p id="__p16" class="p p-last">Additional covariate adjustment for estimated folate, vitamin B<sub>6</sub> and vitamin B<sub>12</sub> intake levels in our linear models did not appreciably change any tests of genotype association or genetic effect size estimates for <em>CPS1</em> rs7422339 or <em>MTHFR</em> C677T in either the CLHNS mother or offspring cohorts (data not shown). Furthermore, we found no evidence of interaction between <em>MTHFR</em> C677T genotype and dietary intake levels of folate (data not shown).</p></div><div id="s2c" class="sec"><h3 id="s2ctitle">Replication of previously identified loci</h3><p id="__p17" class="p p-first-last">To investigate whether this Filipino sample replicated evidence for loci reported (<em>P</em> &lt; 1 × 10<sup>−6</sup>) in a GWA scan for Hcy in white American women (<a href="#DDQ062C13" rid="DDQ062C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138888">13</a>), we defined positive evidence for replication as <em>P</em> &lt; 0.10, with an effect size in the same direction as observed for the original study (i.e. one-sided test with α = 0.05). In addition to <em>MTHFR</em> and <em>CPS1</em>, which showed stronger estimated effects in the CLHNS, association signals were replicated with similar effect sizes in <em>NOX4</em>, <em>CHMPIA-DPEP1</em> and <em>CENPQ-MUT</em>, but not <em>AKAP13</em>, nor a second signal near <em>MTHFR</em> (Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB2" rid-ob="ob-DDQ062TB2" co-legend-rid=""><span>2</span></a>). For the reported <em>CBS</em> signal, neither the SNP nor a suitable proxy was available in the CLHNS samples.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ062TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p18">Evidence of association in CLHNS mothers at previously reported GWA loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140221877948032" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Nearby gene</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Allele</th><th colspan="3" rowspan="1" align="left">Identifying sample<hr></th><th colspan="3" rowspan="1" align="left">CLHNS mothers<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Allele frequency</th><th rowspan="1" colspan="1" align="left">β<sup>a</sup> ± SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Allele frequency</th><th rowspan="1" colspan="1" align="left">β<sup>a</sup> ± SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs7422339</td><td rowspan="1" colspan="1"><em>CPS1</em></td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">211 248 752</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">0.027 ± 0.004</td><td rowspan="1" colspan="1">1.9 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.076 ± 0.011</td><td rowspan="1" colspan="1">4.7 × 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1">rs1801133<sup>b</sup></td><td rowspan="1" colspan="1"><em>MTHFR</em></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">11 778 965</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.048 ± 0.004</td><td rowspan="1" colspan="1">8.1 × 10<sup>−35</sup></td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.083 ± 0.015</td><td rowspan="1" colspan="1">8.7 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">rs11018628</td><td rowspan="1" colspan="1"><em>NOX4</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">88 846 159</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">−0.050 ± 0.007</td><td rowspan="1" colspan="1">9.6 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">−0.059 ± 0.015</td><td rowspan="1" colspan="1">9.8 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">rs460879</td><td rowspan="1" colspan="1"><em>CHMP1A</em></td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">88 240 390</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">0.026 ± 0.004</td><td rowspan="1" colspan="1">3.8 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">0.53</td><td rowspan="1" colspan="1">0.024 ± 0.008</td><td rowspan="1" colspan="1">0.0039</td></tr><tr><td rowspan="1" colspan="1">rs4267943</td><td rowspan="1" colspan="1"><em>CENPQ</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">49 547 764</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">0.024 ± 0.004</td><td rowspan="1" colspan="1">2.0 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.018 ± 0.009</td><td rowspan="1" colspan="1">0.050</td></tr><tr><td rowspan="1" colspan="1">rs10986018</td><td rowspan="1" colspan="1"><em>GPR51</em></td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">98 202 891</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">−0.06 ± 0.01</td><td rowspan="1" colspan="1">2.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">&lt;0.01</td><td rowspan="1" colspan="1">−0.120 ± 0.092</td><td rowspan="1" colspan="1">0.19</td></tr><tr><td rowspan="1" colspan="1">rs2061821</td><td rowspan="1" colspan="1"><em>AKAP13</em></td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">83 923 658</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.38</td><td rowspan="1" colspan="1">0.019 ± 0.004</td><td rowspan="1" colspan="1">5.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">0.002 ± 0.008</td><td rowspan="1" colspan="1">0.79</td></tr><tr><td rowspan="1" colspan="1">rs1999594</td><td rowspan="1" colspan="1"><em>MTHFR</em></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">11 893 482</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">0.04 ± 0.01</td><td rowspan="1" colspan="1">6.2 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">−0.002 ± 0.009</td><td rowspan="1" colspan="1">0.87</td></tr><tr><td rowspan="1" colspan="1">rs6586282</td><td rowspan="1" colspan="1"><em>CBS</em></td><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">43 351 566</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">−0.030 ± 0.005</td><td rowspan="1" colspan="1">3.2 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">—</td></tr></tbody></table></div><div id="largeobj_idm140221877948032" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2860887/table/DDQ062TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140221845134000"><p id="__p19" class="p p-first-last">SNPs previously reported to be associated with Hcy in GWA studies are from reference (<a href="#DDQ062C13" rid="DDQ062C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138896">13</a>) except rs10986018 and rs1999594, which are from reference (<a href="#DDQ062C14" rid="DDQ062C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138898">14</a>). Chromosomal positions are Build 36. SNPs are arranged in ascending order according to CLHNS <em>P</em>-value.</p></div><div id="idm140221843225664"><p id="__p20" class="p p-first-last"><sup>a</sup>β-Coefficient for additive effect with respect to the reference allele.</p></div><div id="idm140221846763024"><p id="__p21" class="p p-first-last"><sup>b</sup>SNP rs1801133 was poorly imputed, instead the directly genotyped results are reported here. The following SNPs were imputed (imputation quality score): rs7422339 (0.76), rs11018628 (&gt;0.99), rs460879 (0.96), rs10986018 (0.65), rs2061821 (0.97), rs1999594 (0.83). SNP rs10986018 was not analyzed in the GWA analysis due to MAF (&lt;0.01). <em>CBS</em> SNP rs6586282 was neither genotyped nor successfully imputed.</p></div></div></div></div><div id="s2d" class="sec"><h3 id="s2dtitle">Multi-locus association results</h3><p id="__p22" class="p p-first-last">We examined additive and interaction effects for SNPs at the <em>CPS1</em> and <em>MTHFR</em> loci in the CLHNS mothers. The cumulative proportion of variability explained by rs7422339 and rs1801133 was 4.7% in the CLHNS mothers, and there was no evidence for an interaction between the two loci (<em>P</em> = 0.30). However in the offspring sample, the test of interaction between the loci was statistically significant (<em>P</em> = 0.0046). The cumulative proportion of variance of log-Hcy explained by both loci and their interaction in the offspring sample was 5.8%. We also estimated the cumulative proportion of log-Hcy variability explained by five loci reported previously (<a href="#DDQ062C13" rid="DDQ062C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138867">13</a>) that had at least nominal evidence of significance in the CLHNS mothers (<em>CPS1</em> rs7422339, <em>MTHFR</em> rs1801133, <em>NOX4</em> rs11018628, <em>CHMPIA-DPEP1</em> rs460879 and <em>CENPQ-MUT</em> rs4267943) to be 5.9%.</p></div><div id="s2e" class="sec"><h3 id="s2etitle">Association of <em>CPS1</em> rs7422339 genotype with other CVD-related traits</h3><p id="__p23" class="p p-first-last">To characterize the <em>CPS1</em> gene with respect to associations with CVD-related phenotypes, we further evaluated other possible effects of the rs7422339 SNP and additional traits of total cholesterol, high-density lipoprotein cholesterol (HDL-C), low-density lipoprotein cholesterol (LDL-C), triglycerides, C-reactive protein (CRP), fasting glucose and body mass index (BMI) in the CLHNS mothers (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). The A allele of rs7422339 was nominally associated with higher LDL-C (<em>P</em> = 0.0010) and higher total cholesterol (<em>P</em> = 0.0094). After adjustment for Hcy, the <em>P</em>-value for LDL-C was 0.021, suggesting that rs7422339 may have some effect on LDL-C independent of Hcy. No evidence for association was observed between rs7422339 and total cholesterol after adjustment for Hcy (<em>P</em> = 0.11).</p></div><div id="s2f" class="sec sec-last"><h3 id="s2ftitle">Discussion</h3><p id="__p24" class="p p-first">We observed strong evidence for association between <em>CPS1</em> variation and log-Hcy level in this GWA study of 1786 Filipino mothers from the CLHNS cohort. The most significantly associated SNP (<em>P</em> = 4.7 × 10<sup>−13</sup>) encodes Thr1405Asn in CPS1 and explains ∼3% of the variability in log-Hcy levels in this cohort. This finding confirms a recent report by the Women's Genome Health Study (WGHS), which observed significant evidence for an association between Hcy levels and rs7422339 (<em>P</em> = 1.9 × 10<sup>−11</sup>) in a sample of 13 974 Caucasian women. The protein encoded by <em>CPS1</em> catalyzes the first committed step of the hepatic urea cycle, and rs7422339 has been previously reported to be associated with nitric oxide level and vascular smooth muscle activity, necrotizing enterocolitis, pulmonary hypertension and, also in the WGHS, fibrinogen level (<a href="#DDQ062C15" rid="DDQ062C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138897">15</a>–<a href="#DDQ062C18" rid="DDQ062C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138876">18</a>). Consistent with our evidence of association in the female but not male CLHNS offspring, the WGHS did not observe replication evidence for <em>CPS1</em> in samples from the Precocious Coronary Artery Disease Study (PROCARDIS), a subset of 840 post-myocardial infarction patients from several European countries, until they performed a gender-stratified analysis (male <em>n</em> = 658, <em>P</em> &gt; 0.05; female <em>n</em> = 161, <em>P</em> = 0.0009). Although the <em>P</em>-value for gender-by-genotype interaction in the CLHNS offspring was not statistically significant (<em>P</em> = 0.095), together these results suggest that the effects of this gene may be stronger in women than in men.</p><p id="__p25">The CLHNS mothers also showed evidence for association with the widely studied <em>MTHFR</em> C677T SNP (rs1801133), which explained an estimated 1.6% of the variation in log-Hcy levels. Follow-up analysis of the CLHNS young adult offspring (age 20–22 years) suggests that <em>MTHFR</em> C677T has a much stronger effect in this younger generation, with an estimated proportion of variance explained by genotype of 6.3%, which is very large for a single SNP for a complex phenotype. We also observed marginal evidence (<em>P</em> = 0.053) for a gender-by-genotype interaction in the offspring, suggesting that, in contrast to <em>CPS1</em>, there may be a stronger effect of this SNP in males than in females. These findings are consistent with several reports that have suggested that the effect of <em>MTHFR</em> C677T is stronger in males than in females (<a href="#DDQ062C10" rid="DDQ062C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138891">10</a>,<a href="#DDQ062C19" rid="DDQ062C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138871">19</a>–<a href="#DDQ062C21" rid="DDQ062C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138875">21</a>) and that the importance of <em>MTHFR</em> C677T genotype decreases with increasing age (<a href="#DDQ062C22" rid="DDQ062C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138870">22</a>), although no evidence for gender differences was reported in the PROCARDIS replication sample (<a href="#DDQ062C13" rid="DDQ062C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138873">13</a>). Hcy levels have been shown to be consistently higher in males than in females after childhood (<a href="#DDQ062C23" rid="DDQ062C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138869">23</a>,<a href="#DDQ062C24" rid="DDQ062C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138895">24</a>), and estrogen lowers plasma Hcy (<a href="#DDQ062C25" rid="DDQ062C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138866">25</a>). In our study, CHLNS male offspring had significantly higher mean Hcy levels (<em>P</em> &lt; 0.0001) than female offspring despite higher mean dietary intake of folate, vitamin B<sub>6</sub> and vitamin B<sub>12</sub> (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>), suggesting that any increase in the effect of <em>MTHFR</em> C677T genotype in CLHNS male offspring compared with female offspring is not due to lower folate intake.</p><p id="__p26">Evidence for differences in genotype effects between the CLHNS mothers and offspring was also detected at <em>CPS1</em>. In contrast to <em>MTHFR</em>, the observed effect of <em>CPS1</em> was stronger in the mothers than in the offspring (Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB1" rid-ob="ob-DDQ062TB1" co-legend-rid=""><span>1</span></a>). The difference in effects for <em>CPS1</em> could be due to bias based on the identification of this locus in the mothers and only following up the most significant results in the offspring, although this bias is mitigated by the report of the <em>CPS1</em> association in the WGHS. The differences in genetic effect strength between CLHNS mothers and their offspring could also be attributed to age or other factors. We observed no evidence for a <em>CPS1</em>–age interaction in the mothers, where there is an age range of 34 years (data not shown). Nonetheless, there remains the possibility that the difference may not be a linear effect with age, but rather a consequence of different life stages or environmental exposures between younger and older adults.</p><p id="__p27">Effect sizes appear substantially different between CLHNS and WGHS women at both the <em>CPS1</em> and <em>MTHFR</em> loci. Overall mean levels of Hcy were the same (CLHNS mean = 11.4, WGHS mean = 11.4), but CLHNS mothers’ Hcy levels had less variability [CLHNS standard deviation (SD) = 3.0, WGHS SD = 4.7]. <em>CPS1</em> SNP rs7422339 was estimated to explain a greater proportion of the variability in log-Hcy levels in the CLHNS mothers (<em>R</em><sup>2</sup> = 3.0%) than in the WGHS samples (<em>R</em><sup>2</sup> = 0.3%) (<a href="#DDQ062C13" rid="DDQ062C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138884">13</a>) despite a lower frequency (0.24 versus 0.31) of the minor allele in CLHNS mothers. Similarly, <em>MTHFR</em> C677T explained a greater proportion of the variability in log-Hcy levels in CLHNS mothers than in WGHS women (CLHNS <em>R</em><sup>2</sup> = 1.6%; WGHS <em>R</em><sup>2</sup> = 0.95%) despite a lower estimated frequency of the minor allele (0.22 versus 0.33). At both loci, a significantly higher estimated increase of log-Hcy levels per each additional copy of the minor allele was observed in CLHNS mothers compared with WGHS women (see Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB2" rid-ob="ob-DDQ062TB2" co-legend-rid=""><span>2</span></a> for effect size estimates and their associated SEs for both studies). The estimated proportions of variability explained by <em>CPS1</em> SNP rs7422339 (<em>R</em><sup>2</sup> = 0.9%) and <em>MTHFR</em> C677T (<em>R</em><sup>2</sup> = 5.1%) in CLHNS female offspring were also higher than in WGHS women, although the estimated increase in log-Hcy levels per minor allele for rs7422339 in CLHNS female offspring was less than the estimated increase observed in WGHS women. The higher estimated proportion of variation explained by rs7422339 in CLHNS female offspring can be attributed to the lower overall variability of Hcy levels in CLHNS female offspring (SD = 3.0) compared with WGHS.</p><p id="__p28">Because environmental factors, such as dietary intake of folate, vitamin B<sub>6</sub> and vitamin B<sub>12</sub>, and indicators of estrogen levels in women are predictive of the Hcy level, it is possible that differences across populations could play a role in how genes influence Hcy. Both CLHNS and WGHS Hcy measurements were made in environments in which folate and vitamin B<sub>12</sub> fortification was not widely available (WGHS measurements were made prior to 1998, when fortification of refined wheat products became mainstream in the USA). Adjustment for folate, vitamin B<sub>6</sub> and vitamin B<sub>12</sub> did not appreciably change genetic association test results in either CLHNS sample. We also found no evidence for any interactions between either <em>CPS1</em> rs7422339 or <em>MTHFR</em> C677T genotype and folate levels to suggest their genotypic effects are stronger in those with lower reported intake levels of folate. Other important differences to consider when comparing results from CLHNS mothers with other studies include substantially lower use of hormone replacement therapy (&lt;1%), which was widely reported in the WGHS (44.9%), and low oral contraceptive use in CLHNS (&lt;2%).</p><p id="__p29">Three of the four other loci identified in the WGHS for which SNP genotypes were available in the CLHNS also showed evidence of association (<em>P</em> &lt; 0.05, Table&nbsp;<a href="/pmc/articles/PMC2860887/table/DDQ062TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ062TB2" rid-ob="ob-DDQ062TB2" co-legend-rid=""><span>2</span></a>). Evidence for the <em>NOX4</em> locus (rs11018628) was relatively strong in the CLHNS mothers (<em>P</em> = 9.8 × 10<sup>−5</sup>), and we observed modest evidence for rs460879 (<em>P</em> = 0.0039) at <em>CHMP1A-DPEP1</em> and rs4267943 (<em>P</em> = 0.050) at <em>CENPQ-MUT</em>, but not for rs2061821 at <em>AKAP13</em> (<em>P</em> = 0.79). The direction of estimated effects in the CLHNS mothers was consistent with those observed in the WGHS for the <em>NOX4</em>, <em>CHMP1A-DPEP1</em> and <em>CENPQ-MUT</em> SNPs. We did not observe evidence for the replication of rs10986018 at <em>GPR51</em> reported in a meta-analysis by Hazra <em>et al</em>. (<a href="#DDQ062C14" rid="DDQ062C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138885">14</a>). The <em>GPR51</em> C allele frequency, however, was much smaller in the CLHNS (&lt;0.01) than in the European-ancestry based sample examined in that study.</p><p id="__p30">Finally, we did not replicate an observed association for rs1999594 near <em>MTHFR</em> (<a href="#DDQ062C13" rid="DDQ062C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138878">13</a>,<a href="#DDQ062C14" rid="DDQ062C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138874">14</a>), for which a joint analysis including <em>MTHFR</em> SNPs rs1801133 and rs1999594 showed that both SNPs were significant (<a href="#DDQ062C14" rid="DDQ062C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138893">14</a>). The frequency of the rs1999594 minor allele was lower in CLHNS mothers compared with the previous reports; however, there was no evidence of a main effect for this locus (β = −0.002) in CLHNS mothers. The different results for rs1999594 in CLHNS may be influenced by weaker linkage disequilibrium (LD) between rs1801133 and rs1999594 (HapMap CHB + JPT <em>R</em><sup>2</sup> = 0.00, <em>D</em>′ = 0.23; HapMap CEU <em>R</em><sup>2</sup> = 0.26, D′ = 0.91).</p><p id="__p31" class="p p-last">This study represents the first GWA results for Hcy in an Asian population. Our findings suggest that at least some of the main genetic factors influencing plasma Hcy levels are likely shared across populations of European and Asian descent, despite different genetic backgrounds and environmental and dietary exposures. These results also suggest that the effects of genotype on plasma Hcy levels may differ across gender and life stages and should motivate future studies to take these factors into consideration.</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">MATERIALS AND METHODS</h2><div id="s3a" class="sec sec-first"><h3 id="s3atitle">Study population and phenotypes</h3><p id="__p32" class="p p-first">Data and samples come from the CLHNS, a community-based birth cohort study that originally enrolled 3327 pregnant women in 1983–84 (3080 singleton live births), and has since followed them and their offspring to the present (<a href="#DDQ062C26" rid="DDQ062C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138889">26</a>). In 2005, 1895 healthy Filipino mothers and 1775 of their offspring remained in the study and had DNA and measurement of biomarkers available.</p><p id="__p33">Trained field staff conducted in-home interviews and collected anthropometric measurements and comprehensive environmental data at each visit (data available online at <a href="http://www.cpc.unc.edu/projects/cebu/">http://www.cpc.unc.edu/projects/cebu/</a>). Blood samples, which were used for biomarker measurement and DNA extraction, were obtained in 2005. Plasma Hcy was measured using the automated Bayer<sup>®</sup> ADVIA Centaur chemiluminescent Hcy assay, as described (<a href="#DDQ062C27" rid="DDQ062C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138894">27</a>).</p><p id="__p34">Dietary intake data were collected via 24 h recall by trained study personnel during the 2005 visit. Intake was measured for 2 days, and the mean was used in the analysis. Nutrient intakes of folate, vitamin B<sub>6</sub> and vitamin B<sub>12</sub> were estimated using the Philippines Food Composition Tables (<a href="#DDQ062C28" rid="DDQ062C28" class=" bibr popnode">28</a>,<a href="#DDQ062C29" rid="DDQ062C29" class=" bibr popnode">29</a>).</p><p id="__p35" class="p p-last">Sample characteristics for the CLHNS mothers and offspring by gender are presented in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S1 and S2</a>. Informed consent was obtained from all CLHNS subjects, and the study protocol was approved by the University of North Carolina Institutional Review Board for the Protection of Human Subjects.</p></div><div id="s3b" class="sec"><h3 id="s3btitle">GWA SNP genotyping</h3><p id="__p36" class="p p-first">SNP genotyping was performed with the Affymetrix Genome-Wide Human SNP Array 5.0 at the Vanderbilt Microarray Shared Resource at Vanderbilt University Medical Center, Nashville, TN, USA, using the standard protocol recommended by the manufacturer. Genotype calling was performed using Birdseed (version 2). Genotyping was attempted on the 1895 unique CLHNS mothers samples, 40 CLHNS duplicates and 5 HapMap CEPH trios. Ten CLHNS samples failed genotyping, and four CLHNS samples were removed after genotyping for a call rate &lt;97%. The final call rate across the 1881 remaining samples was 99.6%.</p><p id="__p37" class="p p-last">We applied SNP quality control checks in PLINK v1.02 on the 1881 CLHNS samples that were successfully genotyped (<a href="#DDQ062C30" rid="DDQ062C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138877">30</a>). Of the initial 424 670 SNPs, we discarded 13 287 SNPs due to poor mapping, call rates &lt;90% and/or deviation from Hardy–Weinberg equilibrium (<em>P</em> &lt; 10<sup>−6</sup>). An additional 164 SNPs were dropped due to three or more discrepancies between genotypes in 40 duplicate pairs and/or Mendelian inheritance errors in five CEPH trios. Another 3113 SNPs were dropped due to three or more genotype discrepancies with HapMap genotypes.</p></div><div id="s3c" class="sec"><h3 id="s3ctitle">Additional SNP genotyping</h3><p id="__p38" class="p p-first-last">Follow-up genotyping was performed for three SNPs (rs7422339, rs1801133 and rs6722617) in the CLHNS mothers and offspring using TaqMan allelic discrimination (Applied Biosystems, Foster City, CA, USA). The SNPs rs7422339 and rs6722617 were genotyped to confirm results for the genetic regions with the strongest evidence for association with Hcy at which the most significant SNP in the region was imputed (see below), and to test for evidence of association in the CLHNS offspring. <em>MTHFR</em> C677T (rs1801133) was genotyped because it had a low imputation quality score (<em>R</em><sup>2</sup> &lt; 0.3, where <em>R</em> is the estimated correlation between the imputed genotype calls and the true unknown genotype for the SNP) in the CLHNS mothers and was an important <em>a priori</em> candidate SNP. The genotype success rate for all SNPs was &gt;98% and no discrepancies were observed among duplicate samples (discrepancy rate &lt;0.3%).</p></div><div id="s3d" class="sec"><h3 id="s3dtitle">Population relatedness</h3><p id="__p39" class="p p-first-last">We used identity-by-descent and identity-by-state estimates calculated in PLINK in combination with our prior knowledge of relationships between CLHNS participants to eliminate 81 estimated first-degree relatives. Our final sample for GWA analysis consisted of 1786 CLHNS women with available genotypes, Hcy levels and covariates from the 2005 survey.</p></div><div id="s3e" class="sec"><h3 id="s3etitle">SNP imputation</h3><p id="__p40" class="p p-first-last">Using a hidden Markov model algorithm implemented in MACH software version 1.0 (available online at <a href="http://www.sph.umich.edu/csg/abecasis/mach/">http://www.sph.umich.edu/csg/abecasis/mach/</a>), genotype imputation was conducted using 352 264 directly genotyped SNPs that were polymorphic in both the 60 CEU founders and the 89 combined CHB + JPT HapMap samples (<a href="#DDQ062C31" rid="DDQ062C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195810650">31</a>). We pooled haplotypes from phased chromosomes in these two HapMap populations to better capture the LD structure in our CLHNS samples. We have shown previously that the CHB HapMap population is a reasonable proxy to the CLHNS cohort (<a href="#DDQ062C32" rid="DDQ062C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138886">32</a>). For each subject, the genotype for each SNP was reported as a dosage value (a continuous number between 0 and 2) that reflects the expected number of copies of an arbitrary allele at that SNP conditional on the directly observed genotypes in both the subject and a representative sample of 200 unrelated CLHNS subjects and the phased haplotype assignments in the CEU and CHB + JPT HapMap samples. Imputation yielded genotype data for 1 878 188 new SNPs, and the 352 264 directly genotyped SNPs were also assigned imputed genotypes. We used <em>R</em><sup>2</sup> ≥ 0.3 as a quality control threshold to discard 150 177 low-quality imputed SNPs, and also excluded 30 351 SNPs with estimated minor allele frequencies (MAF) ≤ 0.01. Finally, discrete dosage values of 0, 1 or 2 were assigned for 23 750 directly genotyped SNPs non-polymorphic in either HapMap sample (and therefore not imputed) but with MAF &gt; 0.01 in the CLHNS samples. In total, 2 073 674 SNPs were tested for association with the Hcy level.</p></div><div id="s3f" class="sec"><h3 id="s3ftitle">Accounting for population substructure</h3><p id="__p41" class="p p-first-last">We constructed PCs using the software EIGENSOFT to capture population substructure among our CHLNS subjects (<a href="#DDQ062C33" rid="DDQ062C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138890">33</a>). We identified a set of 13 972 independent SNPs (estimated <em>R</em><sup>2</sup> &lt; 0.005 between all pairs of SNPs within 1 Mb) with observed MAF &gt; 0.05. The PCs were constructed on the basis of these 13 972 SNPs and 1571 CHLNS subjects with estimated genome-wide pair-wise identity-by-descent &lt;0.1. The PCs for the remaining 228 subjects were subsequently calculated using the parameter coefficients obtained from the first 1571 subjects. The corresponding eigenvalues for the individual PCs were plotted (data not shown) and ‘the elbow’ of the eigenvalue plot was used to define which PCs to be included as covariates in association analyses. A plot of the first versus the second PC is shown in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S1</a>. In addition, we tested for the association between each of the first 10 PCs and the Hcy level to confirm that any important ancestry explanatory PC was included. The first three PCs were included as covariates in the linear regression models. Finally, we examined the calculated genomic control value to evaluate for systematic <em>P</em>-value inflation due to cryptic relatedness between samples not accounted for by the analytic adjustment for PCs in the regression analyses (<a href="#DDQ062C34" rid="DDQ062C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138899">34</a>).</p></div><div id="s3g" class="sec"><h3 id="s3gtitle">GWA association analysis</h3><p id="__p42" class="p p-first-last">Hcy level was natural log-transformed (log-Hcy) to satisfy the model assumption of normally distributed residuals after adjustment for the covariates of age, number of previous pregnancies (categorized as 0–4, 5–10, ≥11), menopausal status and the first three PCs from the population substructure analysis (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). Array Studio software version 3.1 was used to perform the initial GWA analyses (Array Studio and all other Omicsoft products or service names are registered trademarks or trademarks of Omicsoft Corporation, Research Triangle Park, NC, USA). Multiple linear regression was used to test for the association between log-Hcy and each SNP, with adjustment for covariates. Genotype was modeled as an additive effect, with the genotype dosage values used as the primary predictor of interest. A 1 degree-of-freedom likelihood ratio test was used to assess statistical significance. A significance threshold of <em>P</em> &lt; 5 × 10<sup>−8</sup> was used to define a single result as genome-wide significant. Quantile–quantile plots of observed versus expected –log<sub>10</sub>(<em>P</em>-values) were constructed to help assess any cumulative inflation or deflation of statistical significance estimates compared with expectation under the null hypothesis. To compare effect sizes with one study that did not report SEs (<a href="#DDQ062C13" rid="DDQ062C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_118138880">13</a>), SE estimates for the additive genetic effect β-coefficients were derived from the reported estimated β-coefficients and their corresponding Wald test <em>P</em>-values.</p></div><div id="s3h" class="sec"><h3 id="s3htitle">Follow-up association analysis on CLHNS mothers and offspring</h3><p id="__p43" class="p p-first">Linear regression models were analyzed in the software package SAS version 9.3 (SAS Inc., Cary, NC, USA) to test the genotyped SNPs rs7422339, rs6722617 and rs1801133 for association with log-Hcy separately in the mother and the offspring samples. In the offspring sample, we adjusted only for gender, because the ages of all offspring were within 2 years of each other, menopause status was not relevant and the number of previous pregnancies and PCs for population stratification were not available. Genotype was modeled as an additive effect. Partial correlations were calculated for each locus to estimate the proportion of variation explained by genotype. In the offspring sample, we further performed analyses stratified by gender and tests of gender-by-genotype interaction to assess whether results were similar for males and females. We also tested for evidence of a genotype-by-genotype interaction effect between <em>CPS1</em> rs7422339 and <em>MTHFR</em> C677T on the Hcy level.</p><p id="__p44" class="p p-last">We used general linear mixed models to evaluate genotype effects on the combined sample of mothers and offspring while accounting for the correlation of Hcy levels between mother–child pairs due to shared environmental and genetic exposures. Within this setting, we also conducted tests of interaction to assess whether the effects for each locus were similar in mothers and offspring. A significance threshold of <em>P</em> &lt; 0.05 was used for tests of interaction. We repeated the mother–offspring-stratified and -combined analyses with additional adjustment for estimated dietary folate, vitamin B<sub>6</sub> and vitamin B<sub>12</sub> intake to assess whether adjustment for these factors appreciably changed genetic association results. Finally, we tested for interactions between genotype and folate levels to assess whether there were stronger genetic effects in subjects with lower reported folate levels.</p></div><div id="s3i" class="sec"><h3 id="s3ititle">Conditional analysis for <em>CPS1</em> SNPs</h3><p id="__p45" class="p p-first-last">To evaluate whether any of the other 21 <em>CPS1</em> SNPs with <em>P</em> &lt; 5 × 10<sup>−8</sup> were independently associated with log-Hcy levels in the CLHNS mothers after accounting for the strongest associated SNP (rs7422339), we performed association analyses for the <em>CPS1</em> SNPs conditioning on rs7422339 genotype. Using SAS, we examined tests of additive genotype effects for each of the 21 SNPs in individual linear regression models with the same covariates as above plus the rs7422339 genotype.</p></div><div id="s3j" class="sec sec-last"><h3 id="s3jtitle">Association of <em>CPS1</em> SNP rs7422339 with additional traits</h3><p id="__p46" class="p p-first-last">We tested the <em>CPS1</em> SNP rs7422339 for association with several additional biomarkers associated with CVD: total cholesterol, HDL-C, LDL-C, triglycerides, CRP, fasting glucose and BMI. Outcome traits were first natural log-transformed and adjusted for the same covariates as for the Hcy analysis. Any associations that were found to be nominally significant (<em>P</em> &lt; 0.05) were further tested adding the Hcy level to the model to evaluate whether the association was independent of the phenotype's correlation with Hcy.</p></div></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">SUPPLEMENTARY MATERIAL</h2><p id="__p47" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq062/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">FUNDING</h2><p id="__p48" class="p p-first-last">This work was supported by National Institutes of Health grants (<span>DK078150, TW05596, HL085144</span>), pilot funds (<span>RR20649, ES10126 and DK56350</span>) and training grants (<span>T32 GM007092</span>; to D.C.C.-C., <span>T32 HL69768</span> to A.F.M.). K.L.M. is a Pew Scholar in the Biological Sciences.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>[Supplementary Data]</strong>
            
            <div class="sup-box half_rhythm" id="idm140221838217824"><a href="/pmc/articles/PMC2860887/bin/ddq062_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140221836268080" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140221836268080title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p49">We thank the Office of Population Studies Foundation research and data collection teams and Melissa Daniels for nutrient intake estimates.</p><p id="__p50"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140221836266288" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140221836266288title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDQ062C1">1. <span class="element-citation">Wierzbicki A.S. Homocysteine and cardiovascular disease: a review of the evidence. <span><span class="ref-journal">Diab. Vasc. Dis. Res. </span>2007;<span class="ref-vol">4</span>:143–150.</span> [<a href="/pubmed/17654449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diab.+Vasc.+Dis.+Res.&amp;title=Homocysteine+and+cardiovascular+disease:+a+review+of+the+evidence&amp;author=A.S.+Wierzbicki&amp;volume=4&amp;publication_year=2007&amp;pages=143-150&amp;pmid=17654449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C2">2. <span class="element-citation">Maxwell S.R. Coronary artery disease – free radical damage, antioxidant protection and the role of homocysteine. <span><span class="ref-journal">Basic Res. Cardiol. </span>2000;<span class="ref-vol">95</span>(Suppl. 1):I65–I71.</span> [<a href="/pubmed/11192356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Basic+Res.+Cardiol.&amp;title=Coronary+artery+disease+–+free+radical+damage,+antioxidant+protection+and+the+role+of+homocysteine&amp;author=S.R.+Maxwell&amp;volume=95&amp;issue=Suppl.+1&amp;publication_year=2000&amp;pages=I65-I71&amp;pmid=11192356&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C3">3. <span class="element-citation">Bots M.L., Launer L.J., Lindemans J., Hofman A., Grobbee D.E. Homocysteine, atherosclerosis and prevalent cardiovascular disease in the elderly: the Rotterdam Study. <span><span class="ref-journal">J. Intern. Med. </span>1997;<span class="ref-vol">242</span>:339–347.</span> [<a href="/pubmed/9366813" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Intern.+Med.&amp;title=Homocysteine,+atherosclerosis+and+prevalent+cardiovascular+disease+in+the+elderly:+the+Rotterdam+Study&amp;author=M.L.+Bots&amp;author=L.J.+Launer&amp;author=J.+Lindemans&amp;author=A.+Hofman&amp;author=D.E.+Grobbee&amp;volume=242&amp;publication_year=1997&amp;pages=339-347&amp;pmid=9366813&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C4">4. <span class="element-citation">Fortin L.J., Genest J., Jr Measurement of homocyst(e)ine in the prediction of arteriosclerosis. <span><span class="ref-journal">Clin. Biochem. </span>1995;<span class="ref-vol">28</span>:155–162.</span> [<a href="/pubmed/7628074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Biochem.&amp;title=Measurement+of+homocyst(e)ine+in+the+prediction+of+arteriosclerosis&amp;author=L.J.+Fortin&amp;author=J.+Genest&amp;volume=28&amp;publication_year=1995&amp;pages=155-162&amp;pmid=7628074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C5">5. <span class="element-citation">Nygard O., Vollset S.E., Refsum H., Stensvold I., Tverdal A., Nordrehaug J.E., Ueland M., Kvale G. Total plasma homocysteine and cardiovascular risk profile. The Hordaland Homocysteine Study. <span><span class="ref-journal">JAMA. </span>1995;<span class="ref-vol">274</span>:1526–1533.</span> [<a href="/pubmed/7474221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Total+plasma+homocysteine+and+cardiovascular+risk+profile.+The+Hordaland+Homocysteine+Study&amp;author=O.+Nygard&amp;author=S.E.+Vollset&amp;author=H.+Refsum&amp;author=I.+Stensvold&amp;author=A.+Tverdal&amp;volume=274&amp;publication_year=1995&amp;pages=1526-1533&amp;pmid=7474221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C6">6. <span class="element-citation">Perry I.J., Refsum H., Morris R.W., Ebrahim S.B., Ueland P.M., Shaper A.G. Prospective study of serum total homocysteine concentration and risk of stroke in middle-aged British men. <span><span class="ref-journal">Lancet. </span>1995;<span class="ref-vol">346</span>:1395–1398.</span> [<a href="/pubmed/7475822" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Prospective+study+of+serum+total+homocysteine+concentration+and+risk+of+stroke+in+middle-aged+British+men&amp;author=I.J.+Perry&amp;author=H.+Refsum&amp;author=R.W.+Morris&amp;author=S.B.+Ebrahim&amp;author=P.M.+Ueland&amp;volume=346&amp;publication_year=1995&amp;pages=1395-1398&amp;pmid=7475822&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C7">7. <span class="element-citation">Christen W.G., Ajani U.A., Glynn R.J., Hennekens C.H. Blood levels of homocysteine and increased risks of cardiovascular disease: causal or casual? <span><span class="ref-journal">Arch. Intern. Med. </span>2000;<span class="ref-vol">160</span>:422–434.</span> [<a href="/pubmed/10695683" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Intern.+Med.&amp;title=Blood+levels+of+homocysteine+and+increased+risks+of+cardiovascular+disease:+causal+or+casual?&amp;author=W.G.+Christen&amp;author=U.A.+Ajani&amp;author=R.J.+Glynn&amp;author=C.H.+Hennekens&amp;volume=160&amp;publication_year=2000&amp;pages=422-434&amp;pmid=10695683&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C8">8. <span class="element-citation">Wald D.S., Law M., Morris J.K. The dose–response relation between serum homocysteine and cardiovascular disease: implications for treatment and screening. <span><span class="ref-journal">Eur. J. Cardiovasc. Prev. Rehabil. </span>2004;<span class="ref-vol">11</span>:250–253.</span> [<a href="/pubmed/15179109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Cardiovasc.+Prev.+Rehabil.&amp;title=The+dose–response+relation+between+serum+homocysteine+and+cardiovascular+disease:+implications+for+treatment+and+screening&amp;author=D.S.+Wald&amp;author=M.+Law&amp;author=J.K.+Morris&amp;volume=11&amp;publication_year=2004&amp;pages=250-253&amp;pmid=15179109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C9">9. <span class="element-citation">Siva A., De Lange M., Clayton D., Monteith S., Spector T., Brown M.J. The heritability of plasma homocysteine, and the influence of genetic variation in the homocysteine methylation pathway. <span><span class="ref-journal">Q. J. Med. </span>2007;<span class="ref-vol">100</span>:495–499.</span> [<a href="/pubmed/17636160" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Q.+J.+Med.&amp;title=The+heritability+of+plasma+homocysteine,+and+the+influence+of+genetic+variation+in+the+homocysteine+methylation+pathway&amp;author=A.+Siva&amp;author=M.+De+Lange&amp;author=D.+Clayton&amp;author=S.+Monteith&amp;author=T.+Spector&amp;volume=100&amp;publication_year=2007&amp;pages=495-499&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C10">10. <span class="element-citation">Jee S.H., Song K.S., Shim W.H., Kim H.K., Suh I., Park J.Y., Won S.Y., Beaty T.H. Major gene evidence after MTHFR-segregation analysis of serum homocysteine in families of patients undergoing coronary arteriography. <span><span class="ref-journal">Hum. Genet. </span>2002;<span class="ref-vol">111</span>:128–135.</span> [<a href="/pubmed/12189485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Major+gene+evidence+after+MTHFR-segregation+analysis+of+serum+homocysteine+in+families+of+patients+undergoing+coronary+arteriography&amp;author=S.H.+Jee&amp;author=K.S.+Song&amp;author=W.H.+Shim&amp;author=H.K.+Kim&amp;author=I.+Suh&amp;volume=111&amp;publication_year=2002&amp;pages=128-135&amp;pmid=12189485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C11">11. <span class="element-citation">Kullo I.J., Ding K., Boerwinkle E., Turner S.T., Mosley T.H., Jr, Kardia S.L., de Andrade M. Novel genomic loci influencing plasma homocysteine levels. <span><span class="ref-journal">Stroke. </span>2006;<span class="ref-vol">37</span>:1703–1709.</span> [<a href="/pubmed/16741189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Novel+genomic+loci+influencing+plasma+homocysteine+levels&amp;author=I.J.+Kullo&amp;author=K.+Ding&amp;author=E.+Boerwinkle&amp;author=S.T.+Turner&amp;author=T.H.+Mosley&amp;volume=37&amp;publication_year=2006&amp;pages=1703-1709&amp;pmid=16741189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C12">12. <span class="element-citation">Wilcken D.E., Wang X.L., Wilcken B. Methylenetetrahydrofolate reductase (MTHFR) mutation, homocyst(e)ine, and coronary artery disease. <span><span class="ref-journal">Circulation. </span>1997;<span class="ref-vol">96</span>:2738–2740.</span> [<a href="/pubmed/9355925" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Methylenetetrahydrofolate+reductase+(MTHFR)+mutation,+homocyst(e)ine,+and+coronary+artery+disease&amp;author=D.E.+Wilcken&amp;author=X.L.+Wang&amp;author=B.+Wilcken&amp;volume=96&amp;publication_year=1997&amp;pages=2738-2740&amp;pmid=9355925&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C13">13. <span class="element-citation">Paré G., Chasman D.I., Parker A.N., Robert R.Y., Zee R.R.Y., Mälarstig A., Seedorf U., Collins R., Watkins H., Hamsten A., Miletich J.P., Ridker P.M. Novel associations of CPS1, MUT, NOX4 and DPEP1 with plasma homocysteine in a healthy population: a genome-wide evaluation of 13,974 participants in the Women's Genome Health Study. <span><span class="ref-journal">Circ. Cardiovasc. Genet. </span>2009;<span class="ref-vol">2</span>:142–150.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745176/">PMC free article</a>]</span> [<a href="/pubmed/20031578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet.&amp;title=Novel+associations+of+CPS1,+MUT,+NOX4+and+DPEP1+with+plasma+homocysteine+in+a+healthy+population:+a+genome-wide+evaluation+of+13,974+participants+in+the+Women's+Genome+Health+Study&amp;author=G.+Paré&amp;author=D.I.+Chasman&amp;author=A.N.+Parker&amp;author=R.Y.+Robert&amp;author=R.R.Y.+Zee&amp;volume=2&amp;publication_year=2009&amp;pages=142-150&amp;pmid=20031578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C14">14. <span class="element-citation">Hazra A., Kraft P., Lazarus R., Chen C., Chanock S.J., Jacques P., Selhub J., Hunter D.J. Genome-wide significant predictors of metabolites in the one-carbon metabolism pathway. <span><span class="ref-journal">Hum. Mol. Genet. </span>2009;<span class="ref-vol">18</span>:4677–4687.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2773275/">PMC free article</a>]</span> [<a href="/pubmed/19744961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+significant+predictors+of+metabolites+in+the+one-carbon+metabolism+pathway&amp;author=A.+Hazra&amp;author=P.+Kraft&amp;author=R.+Lazarus&amp;author=C.+Chen&amp;author=S.J.+Chanock&amp;volume=18&amp;publication_year=2009&amp;pages=4677-4687&amp;pmid=19744961&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C15">15. <span class="element-citation">Summar M.L., Hall L., Christman B., Barr F., Smith H., Kallianpur A., Brown N., Yadav M., Willis A., Eeds A., et al.  Environmentally determined genetic expression: clinical correlates with molecular variants of carbamyl phosphate synthetase I. <span><span class="ref-journal">Mol. Genet. Metab. </span>2004;<span class="ref-vol">81</span>(Suppl. 1):S12–S19.</span> [<a href="/pubmed/15050969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Genet.+Metab.&amp;title=Environmentally+determined+genetic+expression:+clinical+correlates+with+molecular+variants+of+carbamyl+phosphate+synthetase+I&amp;author=M.L.+Summar&amp;author=L.+Hall&amp;author=B.+Christman&amp;author=F.+Barr&amp;author=H.+Smith&amp;volume=81&amp;issue=Suppl.+1&amp;publication_year=2004&amp;pages=S12-S19&amp;pmid=15050969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C16">16. <span class="element-citation">Moonen R.M., Paulussen A.D., Souren N.Y., Kessels A.G., Rubio-Gozalbo M.E., Villamor E. Carbamoyl phosphate synthetase polymorphisms as a risk factor for necrotizing enterocolitis. <span><span class="ref-journal">Pediatr. Res. </span>2007;<span class="ref-vol">62</span>:188–190.</span> [<a href="/pubmed/17597649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Res.&amp;title=Carbamoyl+phosphate+synthetase+polymorphisms+as+a+risk+factor+for+necrotizing+enterocolitis&amp;author=R.M.+Moonen&amp;author=A.D.+Paulussen&amp;author=N.Y.+Souren&amp;author=A.G.+Kessels&amp;author=M.E.+Rubio-Gozalbo&amp;volume=62&amp;publication_year=2007&amp;pages=188-190&amp;pmid=17597649&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C17">17. <span class="element-citation">Canter J.A., Summar M.L., Smith H.B., Rice G.D., Hall L.D., Ritchie M.D., Motsinger A.A., Christian K.G., Drinkwater D.C., Jr, Scholl F.G., et al.  Genetic variation in the mitochondrial enzyme carbamyl-phosphate synthetase I predisposes children to increased pulmonary artery pressure following surgical repair of congenital heart defects: a validated genetic association study. <span><span class="ref-journal">Mitochondrion. </span>2007;<span class="ref-vol">7</span>:204–210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1929167/">PMC free article</a>]</span> [<a href="/pubmed/17188582" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mitochondrion&amp;title=Genetic+variation+in+the+mitochondrial+enzyme+carbamyl-phosphate+synthetase+I+predisposes+children+to+increased+pulmonary+artery+pressure+following+surgical+repair+of+congenital+heart+defects:+a+validated+genetic+association+study&amp;author=J.A.+Canter&amp;author=M.L.+Summar&amp;author=H.B.+Smith&amp;author=G.D.+Rice&amp;author=L.D.+Hall&amp;volume=7&amp;publication_year=2007&amp;pages=204-210&amp;pmid=17188582&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C18">18. <span class="element-citation">Danik J.S., Pare G., Chasman D.I., Zee R.Y.L., Kwiatkowski D.J., Parker A., Miletich J.P., Ridker P.M. Novel loci, including those related to Crohn disease, psoriasis, and inflammation, identified in a genome-wide association of fibrinogen in 17,686 women. <span><span class="ref-journal">Circ. Cardiovasc. Genet. </span>2009;<span class="ref-vol">2</span>:134–141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2749513/">PMC free article</a>]</span> [<a href="/pubmed/20031577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet.&amp;title=Novel+loci,+including+those+related+to+Crohn+disease,+psoriasis,+and+inflammation,+identified+in+a+genome-wide+association+of+fibrinogen+in+17,686+women&amp;author=J.S.+Danik&amp;author=G.+Pare&amp;author=D.I.+Chasman&amp;author=R.Y.L.+Zee&amp;author=D.J.+Kwiatkowski&amp;volume=2&amp;publication_year=2009&amp;pages=134-141&amp;pmid=20031577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C19">19. <span class="element-citation">Chango A., Potier De Courcy G., Boisson F., Guilland J.C., Barbe F., Perrin M.O., Christides J.P., Rabhi K., Pfister M., Galan P., et al.  5,10-methylenetetrahydrofolate reductase common mutations, folate status and plasma homocysteine in healthy French adults of the Supplementation en Vitamines et Mineraux Antioxydants (SU.VI.MAX) cohort. <span><span class="ref-journal">Br. J. Nutr. </span>2000;<span class="ref-vol">84</span>:891–896.</span> [<a href="/pubmed/11177206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Nutr.&amp;title=5,10-methylenetetrahydrofolate+reductase+common+mutations,+folate+status+and+plasma+homocysteine+in+healthy+French+adults+of+the+Supplementation+en+Vitamines+et+Mineraux+Antioxydants+(SU.VI.MAX)+cohort&amp;author=A.+Chango&amp;author=G.+Potier+De+Courcy&amp;author=F.+Boisson&amp;author=J.C.+Guilland&amp;author=F.+Barbe&amp;volume=84&amp;publication_year=2000&amp;pages=891-896&amp;pmid=11177206&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C20">20. <span class="element-citation">Dekou V., Whincup P., Papacosta O., Ebrahim S., Lennon L., Ueland P.M., Refsum H., Humphries S.E., Gudnason V. The effect of the C677T and A1298C polymorphisms in the methylenetetrahydrofolate reductase gene on homocysteine levels in elderly men and women from the British regional heart study. <span><span class="ref-journal">Atherosclerosis. </span>2001;<span class="ref-vol">154</span>:659–666.</span> [<a href="/pubmed/11257267" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=The+effect+of+the+C677T+and+A1298C+polymorphisms+in+the+methylenetetrahydrofolate+reductase+gene+on+homocysteine+levels+in+elderly+men+and+women+from+the+British+regional+heart+study&amp;author=V.+Dekou&amp;author=P.+Whincup&amp;author=O.+Papacosta&amp;author=S.+Ebrahim&amp;author=L.+Lennon&amp;volume=154&amp;publication_year=2001&amp;pages=659-666&amp;pmid=11257267&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C21">21. <span class="element-citation">Papoutsakis C., Yiannakouris N., Manios Y., Papaconstantinou E., Magkos F., Schulpis K.H., Zampelas A., Matalas A.L. The effect of MTHFR(C677T) genotype on plasma homocysteine concentrations in healthy children is influenced by gender. <span><span class="ref-journal">Eur. J. Clin. Nutr. </span>2006;<span class="ref-vol">60</span>:155–162.</span> [<a href="/pubmed/16234842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Clin.+Nutr.&amp;title=The+effect+of+MTHFR(C677T)+genotype+on+plasma+homocysteine+concentrations+in+healthy+children+is+influenced+by+gender&amp;author=C.+Papoutsakis&amp;author=N.+Yiannakouris&amp;author=Y.+Manios&amp;author=E.+Papaconstantinou&amp;author=F.+Magkos&amp;volume=60&amp;publication_year=2006&amp;pages=155-162&amp;pmid=16234842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C22">22. <span class="element-citation">Husemoen L.L., Thomsen T.F., Fenger M., Jorgensen H.L., Jorgensen T. Contribution of thermolabile methylenetetrahydrofolate reductase variant to total plasma homocysteine levels in healthy men and women. Inter99 (2) <span><span class="ref-journal">Genet. Epidemiol. </span>2003;<span class="ref-vol">24</span>:322–330.</span> [<a href="/pubmed/12687650" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Contribution+of+thermolabile+methylenetetrahydrofolate+reductase+variant+to+total+plasma+homocysteine+levels+in+healthy+men+and+women.+Inter99+(2)&amp;author=L.L.+Husemoen&amp;author=T.F.+Thomsen&amp;author=M.+Fenger&amp;author=H.L.+Jorgensen&amp;author=T.+Jorgensen&amp;volume=24&amp;publication_year=2003&amp;pages=322-330&amp;pmid=12687650&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C23">23. <span class="element-citation">Selhub J., Jacques P.F., Rosenberg I.H., Rogers G., Bowman B.A., Gunter E.W., Wright J.D., Johnson C.L. Serum total homocysteine concentrations in the third National Health and Nutrition Examination Survey (1991–1994): population reference ranges and contribution of vitamin status to high serum concentrations. <span><span class="ref-journal">Ann. Intern. Med. </span>1999;<span class="ref-vol">131</span>:331–339.</span> [<a href="/pubmed/10475885" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Intern.+Med.&amp;title=Serum+total+homocysteine+concentrations+in+the+third+National+Health+and+Nutrition+Examination+Survey+(1991–1994):+population+reference+ranges+and+contribution+of+vitamin+status+to+high+serum+concentrations&amp;author=J.+Selhub&amp;author=P.F.+Jacques&amp;author=I.H.+Rosenberg&amp;author=G.+Rogers&amp;author=B.A.+Bowman&amp;volume=131&amp;publication_year=1999&amp;pages=331-339&amp;pmid=10475885&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C24">24. <span class="element-citation">Bjorke Monsen A.L., Ueland P.M. Homocysteine and methylmalonic acid in diagnosis and risk assessment from infancy to adolescence. <span><span class="ref-journal">Am. J. Clin. Nutr. </span>2003;<span class="ref-vol">78</span>:7–21.</span> [<a href="/pubmed/12816766" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Clin.+Nutr.&amp;title=Homocysteine+and+methylmalonic+acid+in+diagnosis+and+risk+assessment+from+infancy+to+adolescence&amp;author=A.L.+Bjorke+Monsen&amp;author=P.M.+Ueland&amp;volume=78&amp;publication_year=2003&amp;pages=7-21&amp;pmid=12816766&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C25">25. <span class="element-citation">Morris M.S., Jacques P.F., Selhub J., Rosenberg I.H. Total homocysteine and estrogen status indicators in the Third National Health and Nutrition Examination Survey. <span><span class="ref-journal">Am. J. Epidemiol. </span>2000;<span class="ref-vol">152</span>:140–148.</span> [<a href="/pubmed/10909951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Total+homocysteine+and+estrogen+status+indicators+in+the+Third+National+Health+and+Nutrition+Examination+Survey&amp;author=M.S.+Morris&amp;author=P.F.+Jacques&amp;author=J.+Selhub&amp;author=I.H.+Rosenberg&amp;volume=152&amp;publication_year=2000&amp;pages=140-148&amp;pmid=10909951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C26">26. <span class="element-citation">Adair L.S., Kuzawa C.W., Borja J. Maternal energy stores and diet composition during pregnancy program adolescent blood pressure. <span><span class="ref-journal">Circulation. </span>2001;<span class="ref-vol">104</span>:1034–1039.</span> [<a href="/pubmed/11524398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Maternal+energy+stores+and+diet+composition+during+pregnancy+program+adolescent+blood+pressure&amp;author=L.S.+Adair&amp;author=C.W.+Kuzawa&amp;author=J.+Borja&amp;volume=104&amp;publication_year=2001&amp;pages=1034-1039&amp;pmid=11524398&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C27">27. <span class="element-citation">Tewari P.C., Zhang B., Bluestein B.I. Analytical and clinical evaluation of the Bayer ADVIA Centaur homocysteine assay. <span><span class="ref-journal">Clin. Chim. Acta. </span>2004;<span class="ref-vol">342</span>:171–178.</span> [<a href="/pubmed/15026278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chim.+Acta&amp;title=Analytical+and+clinical+evaluation+of+the+Bayer+ADVIA+Centaur+homocysteine+assay&amp;author=P.C.+Tewari&amp;author=B.+Zhang&amp;author=B.I.+Bluestein&amp;volume=342&amp;publication_year=2004&amp;pages=171-178&amp;pmid=15026278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C28">28. <span class="element-citation">Daniels M.C.  <span class="ref-journal">Dietary Diversity as a Measure of the Micronutrient Adequacy Results from Metropolitan Cebu, Philippines Site.</span> Washington, DC: Food and Nutrition Technical Assistance II Project, Academy for Educational Development; 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Dietary+Diversity+as+a+Measure+of+the+Micronutrient+Adequacy+Results+from+Metropolitan+Cebu,+Philippines+Site&amp;author=M.C.+Daniels&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C29">29. <span class="element-citation">Food and Nutrition Research Institute. <span class="ref-journal">Food Composition Table and Menu Eval.</span> Manila, Philippines: Department of Science and Technology; 2000.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Food+Composition+Table+and+Menu+Eval.&amp;publication_year=2000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C30">30. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C31">31. <span class="element-citation">Li Y., Willer C., Sanna S., Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu. Rev. Genomics Hum. Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genomics+Hum.+Genet.&amp;title=Genotype+imputation&amp;author=Y.+Li&amp;author=C.+Willer&amp;author=S.+Sanna&amp;author=G.+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C32">32. <span class="element-citation">Marvelle A.F., Lange L.A., Qin L., Wang Y., Lange E.M., Adair L.S., Mohlke K.L. Comparison of ENCODE region SNPs between Cebu Filipino and Asian HapMap samples. <span><span class="ref-journal">J. Hum. Genet. </span>2007;<span class="ref-vol">52</span>:729–737.</span> [<a href="/pubmed/17636361" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet.&amp;title=Comparison+of+ENCODE+region+SNPs+between+Cebu+Filipino+and+Asian+HapMap+samples&amp;author=A.F.+Marvelle&amp;author=L.A.+Lange&amp;author=L.+Qin&amp;author=Y.+Wang&amp;author=E.M.+Lange&amp;volume=52&amp;publication_year=2007&amp;pages=729-737&amp;pmid=17636361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C33">33. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ062C34">34. <span class="element-citation">Devlin B., Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2860887&amp;issue-id=186685&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140221843337024title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">MATERIALS AND METHODS</a></li><li><a href="#s4title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s5title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140221836268080title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140221836266288title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2860887/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2860887/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2860887/pdf/ddq062.pdf">PDF (174K)</a></li> | <li><a href="#" data-citationid="PMC2860887" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2860887%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2860887%2F&amp;text=Genome-wide%20association%20study%20of%20homocysteine%20levels%20in%20Filipinos%20provides%20evidence%20for%20CPS1%20in%20women%20and%20a%20stronger%20MTHFR%20effect%20in%20young%20adults" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2860887%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20154341" data-aiid="2860887" data-aid="2860887" data-iid="186685" data-domainid="740" data-domain="hmg" data-accid="PMC2860887" data-md5="cbc08365149f9af63045f3ac7b58d20f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:47:52-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88BCF2DD50B7C100000000002C001B&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2860887%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2860887/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>