<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2556434/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide SNP-Based Linkage Scan Identifies a Locus on 8q24 for an Age-Related Hearing Impairment Trait</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2556434/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Genome-wide SNP-Based Linkage Scan Identifies a Locus on 8q24 for an Age-Related Hearing Impairment Trait"><meta name="citation_authors" content="Jeroen R. Huyghe, Lut Van Laer, Jan-Jaap Hendrickx, Erik Fransen, Kelly Demeester, Vedat Topsakal, Sylvia Kunst, Minna Manninen, Mona Jensen, Amanda Bonaconsa, Manuela Mazzoli, Manuela Baur, Samuli Hannula, Elina Mäki-Torkko, Angeles Espeso, Els Van Eyken, Antonia Flaquer, Christian Becker, Dafydd Stephens, Martti Sorri, Eva Orzan, Michael Bille, Agnete Parving, Ilmari Pyykkö, Cor W.R.J. Cremers, Hannie Kremer, Paul H. Van de Heyning, Thomas F. Wienker, Peter Nürnberg, Markus Pfister, Guy Van Camp"><meta name="citation_date" content="12 September 2008"><meta name="citation_issue" content="3"><meta name="citation_volume" content="83"><meta name="citation_firstpage" content="401"><meta name="citation_doi" content="10.1016/j.ajhg.2008.08.002"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2556434/?report=abstract"><meta name="citation_pmid" content="18760390"><meta name="DC.Title" content="Genome-wide SNP-Based Linkage Scan Identifies a Locus on 8q24 for an Age-Related Hearing Impairment Trait"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Jeroen R. Huyghe"><meta name="DC.Contributor" content="Lut Van Laer"><meta name="DC.Contributor" content="Jan-Jaap Hendrickx"><meta name="DC.Contributor" content="Erik Fransen"><meta name="DC.Contributor" content="Kelly Demeester"><meta name="DC.Contributor" content="Vedat Topsakal"><meta name="DC.Contributor" content="Sylvia Kunst"><meta name="DC.Contributor" content="Minna Manninen"><meta name="DC.Contributor" content="Mona Jensen"><meta name="DC.Contributor" content="Amanda Bonaconsa"><meta name="DC.Contributor" content="Manuela Mazzoli"><meta name="DC.Contributor" content="Manuela Baur"><meta name="DC.Contributor" content="Samuli Hannula"><meta name="DC.Contributor" content="Elina Mäki-Torkko"><meta name="DC.Contributor" content="Angeles Espeso"><meta name="DC.Contributor" content="Els Van Eyken"><meta name="DC.Contributor" content="Antonia Flaquer"><meta name="DC.Contributor" content="Christian Becker"><meta name="DC.Contributor" content="Dafydd Stephens"><meta name="DC.Contributor" content="Martti Sorri"><meta name="DC.Contributor" content="Eva Orzan"><meta name="DC.Contributor" content="Michael Bille"><meta name="DC.Contributor" content="Agnete Parving"><meta name="DC.Contributor" content="Ilmari Pyykkö"><meta name="DC.Contributor" content="Cor W.R.J. Cremers"><meta name="DC.Contributor" content="Hannie Kremer"><meta name="DC.Contributor" content="Paul H. Van de Heyning"><meta name="DC.Contributor" content="Thomas F. Wienker"><meta name="DC.Contributor" content="Peter Nürnberg"><meta name="DC.Contributor" content="Markus Pfister"><meta name="DC.Contributor" content="Guy Van Camp"><meta name="DC.Date" content="2008 Sep 12"><meta name="DC.Identifier" content="10.1016/j.ajhg.2008.08.002"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide SNP-Based Linkage Scan Identifies a Locus on 8q24 for an Age-Related Hearing Impairment Trait"><meta property="og:type" content="article"><meta property="og:description" content="Age-related hearing impairment (ARHI), or presbycusis, is a very common multifactorial disorder. Despite the knowledge that genetics play an important role in the etiology of human ARHI as revealed by heritability studies, to date, its precise genetic ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2556434/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2556434/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2556434/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CC479DD3F80F100000000095705CB.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2556434%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books">Books</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/172506/">v.83(3); 2008 Sep 12</a></li><li class="accid">PMC2556434</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2008 Sep 12; 83(3): 401–407. </span></div><div><span class="fm-vol-iss-date">Published online 2008 Sep 5. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2008.08.002" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2008.08.002</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2556434</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18760390">18760390</a></div></div></div></div><h1 class="content-title">Genome-wide SNP-Based Linkage Scan Identifies a Locus on 8q24 for an Age-Related Hearing Impairment Trait</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Huyghe%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064641429088" co-class="co-affbox">Jeroen R. Huyghe</a>,<sup>1,</sup><sup>14</sup> <a href="/pubmed/?term=Van%20Laer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064682194432" co-class="co-affbox">Lut Van Laer</a>,<sup>1,</sup><sup>14</sup> <a href="/pubmed/?term=Hendrickx%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064688424960" co-class="co-affbox">Jan-Jaap Hendrickx</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Fransen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064684925696" co-class="co-affbox">Erik Fransen</a>,<sup>1</sup> <a href="/pubmed/?term=Demeester%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064683747888" co-class="co-affbox">Kelly Demeester</a>,<sup>2</sup> <a href="/pubmed/?term=Topsakal%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064688527008" co-class="co-affbox">Vedat Topsakal</a>,<sup>2</sup> <a href="/pubmed/?term=Kunst%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064722256944" co-class="co-affbox">Sylvia Kunst</a>,<sup>3</sup> <a href="/pubmed/?term=Manninen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064687313824" co-class="co-affbox">Minna Manninen</a>,<sup>4</sup> <a href="/pubmed/?term=Jensen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064680152800" co-class="co-affbox">Mona Jensen</a>,<sup>5</sup> <a href="/pubmed/?term=Bonaconsa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064686112656" co-class="co-affbox">Amanda Bonaconsa</a>,<sup>6</sup> <a href="/pubmed/?term=Mazzoli%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064680143584" co-class="co-affbox">Manuela Mazzoli</a>,<sup>6</sup> <a href="/pubmed/?term=Baur%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064678230432" co-class="co-affbox">Manuela Baur</a>,<sup>7</sup> <a href="/pubmed/?term=Hannula%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064684173968" co-class="co-affbox">Samuli Hannula</a>,<sup>8</sup> <a href="/pubmed/?term=M%26%23x000e4%3Bki-Torkko%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064687988640" co-class="co-affbox">Elina Mäki-Torkko</a>,<sup>8</sup> <a href="/pubmed/?term=Espeso%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064680136528" co-class="co-affbox">Angeles Espeso</a>,<sup>9</sup> <a href="/pubmed/?term=Van%20Eyken%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064681244368" co-class="co-affbox">Els Van Eyken</a>,<sup>1</sup> <a href="/pubmed/?term=Flaquer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064687613904" co-class="co-affbox">Antonia Flaquer</a>,<sup>10</sup> <a href="/pubmed/?term=Becker%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064687227232" co-class="co-affbox">Christian Becker</a>,<sup>11</sup> <a href="/pubmed/?term=Stephens%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064758516912" co-class="co-affbox">Dafydd Stephens</a>,<sup>9</sup> <a href="/pubmed/?term=Sorri%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064677934320" co-class="co-affbox">Martti Sorri</a>,<sup>8</sup> <a href="/pubmed/?term=Orzan%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064684122736" co-class="co-affbox">Eva Orzan</a>,<sup>6</sup> <a href="/pubmed/?term=Bille%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064755991008" co-class="co-affbox">Michael Bille</a>,<sup>5</sup> <a href="/pubmed/?term=Parving%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064685245824" co-class="co-affbox">Agnete Parving</a>,<sup>5</sup> <a href="/pubmed/?term=Pyykk%26%23x000f6%3B%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064684776368" co-class="co-affbox">Ilmari Pyykkö</a>,<sup>4</sup> <a href="/pubmed/?term=Cremers%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064663443984" co-class="co-affbox">Cor W.R.J. Cremers</a>,<sup>3</sup> <a href="/pubmed/?term=Kremer%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064676544448" co-class="co-affbox">Hannie Kremer</a>,<sup>3,</sup><sup>13</sup> <a href="/pubmed/?term=Van%20de%20Heyning%20PH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064662360736" co-class="co-affbox">Paul H. Van de Heyning</a>,<sup>2</sup> <a href="/pubmed/?term=Wienker%20TF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064677540800" co-class="co-affbox">Thomas F. Wienker</a>,<sup>10</sup> <a href="/pubmed/?term=N%26%23x000fc%3Brnberg%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064664713056" co-class="co-affbox">Peter Nürnberg</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Pfister%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064755986752" co-class="co-affbox">Markus Pfister</a>,<sup>7</sup> and  <a href="/pubmed/?term=Van%20Camp%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390" class="affpopup" co-rid="_co_idm140064667406960" co-class="co-affbox">Guy Van Camp</a><sup>1,</sup><sup>∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140064641429088"><h3 class="no_margin">Jeroen R. Huyghe</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, B-2610 Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Huyghe%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Jeroen R. Huyghe</a></div></div><div id="_co_idm140064682194432"><h3 class="no_margin">Lut Van Laer</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, B-2610 Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20Laer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Lut Van Laer</a></div></div><div id="_co_idm140064688424960"><h3 class="no_margin">Jan-Jaap Hendrickx</h3><p><sup>2</sup>Department of Otorhinolaryngology, University Hospital of Antwerp, B-2650 Antwerp, Belgium</p><p><sup>3</sup>Department of Otorhinolaryngology, Radboud University Nijmegen Medical Centre, NL-6500 HB Nijmegen, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Hendrickx%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Jan-Jaap Hendrickx</a></div></div><div id="_co_idm140064684925696"><h3 class="no_margin">Erik Fransen</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, B-2610 Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Fransen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Erik Fransen</a></div></div><div id="_co_idm140064683747888"><h3 class="no_margin">Kelly Demeester</h3><p><sup>2</sup>Department of Otorhinolaryngology, University Hospital of Antwerp, B-2650 Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Demeester%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Kelly Demeester</a></div></div><div id="_co_idm140064688527008"><h3 class="no_margin">Vedat Topsakal</h3><p><sup>2</sup>Department of Otorhinolaryngology, University Hospital of Antwerp, B-2650 Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Topsakal%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Vedat Topsakal</a></div></div><div id="_co_idm140064722256944"><h3 class="no_margin">Sylvia Kunst</h3><p><sup>3</sup>Department of Otorhinolaryngology, Radboud University Nijmegen Medical Centre, NL-6500 HB Nijmegen, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Kunst%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Sylvia Kunst</a></div></div><div id="_co_idm140064687313824"><h3 class="no_margin">Minna Manninen</h3><p><sup>4</sup>Department of Otorhinolaryngology, University of Tampere, FIN-33014 Tampere, Finland</p><div>Find articles by <a href="/pubmed/?term=Manninen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Minna Manninen</a></div></div><div id="_co_idm140064680152800"><h3 class="no_margin">Mona Jensen</h3><p><sup>5</sup>Department of Audiology, Bispebjerg Hospital, DK-2400 Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=Jensen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Mona Jensen</a></div></div><div id="_co_idm140064686112656"><h3 class="no_margin">Amanda Bonaconsa</h3><p><sup>6</sup>Department of Oto-surgery, University Hospital Padova, I-35128 Padova, Italy</p><div>Find articles by <a href="/pubmed/?term=Bonaconsa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Amanda Bonaconsa</a></div></div><div id="_co_idm140064680143584"><h3 class="no_margin">Manuela Mazzoli</h3><p><sup>6</sup>Department of Oto-surgery, University Hospital Padova, I-35128 Padova, Italy</p><div>Find articles by <a href="/pubmed/?term=Mazzoli%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Manuela Mazzoli</a></div></div><div id="_co_idm140064678230432"><h3 class="no_margin">Manuela Baur</h3><p><sup>7</sup>Department of Otorhinolaryngology, University of Tübingen, D-72074 Tübingen, Germany</p><div>Find articles by <a href="/pubmed/?term=Baur%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Manuela Baur</a></div></div><div id="_co_idm140064684173968"><h3 class="no_margin">Samuli Hannula</h3><p><sup>8</sup>Department of Otorhinolaryngology, University of Oulu, FIN-90220 Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Hannula%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Samuli Hannula</a></div></div><div id="_co_idm140064687988640"><h3 class="no_margin">Elina Mäki-Torkko</h3><p><sup>8</sup>Department of Otorhinolaryngology, University of Oulu, FIN-90220 Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000e4%3Bki-Torkko%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Elina Mäki-Torkko</a></div></div><div id="_co_idm140064680136528"><h3 class="no_margin">Angeles Espeso</h3><p><sup>9</sup>Welsh Hearing Institute, Cardiff University, Cardiff CF4 4XW, UK</p><div>Find articles by <a href="/pubmed/?term=Espeso%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Angeles Espeso</a></div></div><div id="_co_idm140064681244368"><h3 class="no_margin">Els Van Eyken</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, B-2610 Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20Eyken%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Els Van Eyken</a></div></div><div id="_co_idm140064687613904"><h3 class="no_margin">Antonia Flaquer</h3><p><sup>10</sup>Institute of Medical Biometry, Informatics and Epidemiology, University of Bonn, D-53105 Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=Flaquer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Antonia Flaquer</a></div></div><div id="_co_idm140064687227232"><h3 class="no_margin">Christian Becker</h3><p><sup>11</sup>Cologne Center for Genomics (CCG) and Institute for Genetics, University of Cologne, D-50674 Cologne, Germany</p><div>Find articles by <a href="/pubmed/?term=Becker%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Christian Becker</a></div></div><div id="_co_idm140064758516912"><h3 class="no_margin">Dafydd Stephens</h3><p><sup>9</sup>Welsh Hearing Institute, Cardiff University, Cardiff CF4 4XW, UK</p><div>Find articles by <a href="/pubmed/?term=Stephens%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Dafydd Stephens</a></div></div><div id="_co_idm140064677934320"><h3 class="no_margin">Martti Sorri</h3><p><sup>8</sup>Department of Otorhinolaryngology, University of Oulu, FIN-90220 Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Sorri%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Martti Sorri</a></div></div><div id="_co_idm140064684122736"><h3 class="no_margin">Eva Orzan</h3><p><sup>6</sup>Department of Oto-surgery, University Hospital Padova, I-35128 Padova, Italy</p><div>Find articles by <a href="/pubmed/?term=Orzan%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Eva Orzan</a></div></div><div id="_co_idm140064755991008"><h3 class="no_margin">Michael Bille</h3><p><sup>5</sup>Department of Audiology, Bispebjerg Hospital, DK-2400 Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=Bille%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Michael Bille</a></div></div><div id="_co_idm140064685245824"><h3 class="no_margin">Agnete Parving</h3><p><sup>5</sup>Department of Audiology, Bispebjerg Hospital, DK-2400 Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=Parving%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Agnete Parving</a></div></div><div id="_co_idm140064684776368"><h3 class="no_margin">Ilmari Pyykkö</h3><p><sup>4</sup>Department of Otorhinolaryngology, University of Tampere, FIN-33014 Tampere, Finland</p><div>Find articles by <a href="/pubmed/?term=Pyykk%26%23x000f6%3B%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Ilmari Pyykkö</a></div></div><div id="_co_idm140064663443984"><h3 class="no_margin">Cor W.R.J. Cremers</h3><p><sup>3</sup>Department of Otorhinolaryngology, Radboud University Nijmegen Medical Centre, NL-6500 HB Nijmegen, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Cremers%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Cor W.R.J. Cremers</a></div></div><div id="_co_idm140064676544448"><h3 class="no_margin">Hannie Kremer</h3><p><sup>3</sup>Department of Otorhinolaryngology, Radboud University Nijmegen Medical Centre, NL-6500 HB Nijmegen, the Netherlands</p><p><sup>13</sup>Nijmegen Centre for Molecular Life Sciences, NL-6500 HB Nijmegen, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Kremer%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Hannie Kremer</a></div></div><div id="_co_idm140064662360736"><h3 class="no_margin">Paul H. Van de Heyning</h3><p><sup>2</sup>Department of Otorhinolaryngology, University Hospital of Antwerp, B-2650 Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20de%20Heyning%20PH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Paul H. Van de Heyning</a></div></div><div id="_co_idm140064677540800"><h3 class="no_margin">Thomas F. Wienker</h3><p><sup>10</sup>Institute of Medical Biometry, Informatics and Epidemiology, University of Bonn, D-53105 Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=Wienker%20TF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Thomas F. Wienker</a></div></div><div id="_co_idm140064664713056"><h3 class="no_margin">Peter Nürnberg</h3><p><sup>11</sup>Cologne Center for Genomics (CCG) and Institute for Genetics, University of Cologne, D-50674 Cologne, Germany</p><p><sup>12</sup>Cologne Excellence Cluster on Cellular Stress Responses in Aging-associated Diseases (CECAD), University of Cologne, D-50674 Cologne, Germany</p><div>Find articles by <a href="/pubmed/?term=N%26%23x000fc%3Brnberg%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Peter Nürnberg</a></div></div><div id="_co_idm140064755986752"><h3 class="no_margin">Markus Pfister</h3><p><sup>7</sup>Department of Otorhinolaryngology, University of Tübingen, D-72074 Tübingen, Germany</p><div>Find articles by <a href="/pubmed/?term=Pfister%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Markus Pfister</a></div></div><div id="_co_idm140064667406960"><h3 class="no_margin">Guy Van Camp</h3><p><sup>1</sup>Department of Medical Genetics, University of Antwerp, B-2610 Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20Camp%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18760390">Guy Van Camp</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140064667165312_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140064667165312_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140064667165312_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140064667165312_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Medical Genetics, University of Antwerp, B-2610 Antwerp, Belgium</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Otorhinolaryngology, University Hospital of Antwerp, B-2650 Antwerp, Belgium</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Department of Otorhinolaryngology, Radboud University Nijmegen Medical Centre, NL-6500 HB Nijmegen, the Netherlands</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Otorhinolaryngology, University of Tampere, FIN-33014 Tampere, Finland</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Audiology, Bispebjerg Hospital, DK-2400 Copenhagen, Denmark</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Oto-surgery, University Hospital Padova, I-35128 Padova, Italy</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Otorhinolaryngology, University of Tübingen, D-72074 Tübingen, Germany</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Department of Otorhinolaryngology, University of Oulu, FIN-90220 Oulu, Finland</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Welsh Hearing Institute, Cardiff University, Cardiff CF4 4XW, UK</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Institute of Medical Biometry, Informatics and Epidemiology, University of Bonn, D-53105 Bonn, Germany</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Cologne Center for Genomics (CCG) and Institute for Genetics, University of Cologne, D-50674 Cologne, Germany</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Cologne Excellence Cluster on Cellular Stress Responses in Aging-associated Diseases (CECAD), University of Cologne, D-50674 Cologne, Germany</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Nijmegen Centre for Molecular Life Sciences, NL-6500 HB Nijmegen, The Netherlands</div><div><span class="contrib-email" id="idm140064667406960">Guy Van Camp: <a href="mailto:dev@null" data-email="eb.ca.au@pmacnav.yug" class="oemail">eb.ca.au@pmacnav.yug</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="eb.ca.au@pmacnav.yug" class="oemail">eb.ca.au@pmacnav.yug</a></div><div id="fn1"><sup>14</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140064667165312_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 May 22; Revised 2008 Aug 7; Accepted 2008 Aug 7.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140064667165312_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2008 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div><div class="license half_rhythm">This document may be redistributed and reused, subject to <a href="http://www.elsevier.com/wps/find/authorsview.authors/supplementalterms1.0" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">certain conditions</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2556434/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Six Figures</div><div><a href="/pmc/articles/PMC2556434/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (730K)</span></div><div class="small guid">GUID:&nbsp;913A4B59-1607-4D6A-9AD9-CC253A3124B9</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S1. Family-Specific Contributions to the Multipoint LOD Scores for PC3 at the Positions of the SNPs beneath the Significant Linkage Peak on Chromosome 8</div><div><a href="/pmc/articles/PMC2556434/bin/mmc2.xls" data-ga-action="click_feat_suppl">mmc2.xls</a><span style="color:gray"> (146K)</span></div><div class="small guid">GUID:&nbsp;A9A1BD87-48B3-460B-BB71-82D386CA4534</div></div></dd></dl></div><div id="idm140064683238032" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140064683238032title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Age-related hearing impairment (ARHI), or presbycusis, is a very common multifactorial disorder. Despite the knowledge that genetics play an important role in the etiology of human ARHI as revealed by heritability studies, to date, its precise genetic determinants remain elusive. Here we report the results of a cross-sectional family-based genetic study employing audiometric data. By using principal component analysis, we were able to reduce the dimensionality of this multivariate phenotype while capturing most of the variation and retaining biologically important features of the audiograms. We conducted a genome-wide association as well as a linkage scan with high-density SNP microarrays. Because of the presence of genetic population substructure, association testing was stratified after which evidence was combined by meta-analysis. No association signals reaching genome-wide significance were detected. Linkage analysis identified a linkage peak on 8q24.13-q24.22 for a trait correlated to audiogram shape. The signal reached genome-wide significance, as assessed by simulations. This finding represents the first locus for an ARHI trait.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Main Text</h2><p id="__p3" class="p p-first">Age-related hearing impairment (ARHI), or presbycusis, is the progressive bilaterally symmetric deterioration of hearing ability that occurs with aging. Both studies of the cochlea in many animal species and histopathological studies of human temporal bones have shown that stria vascularis volumes, spiral ganglion cell, inner hair cell and outer hair cell populations, as well as many other cochlear cell types and structures show age-related degeneration.<a href="#bib1" rid="bib1 bib2 bib3" class=" bibr popnode">1–3</a></p><p id="__p4">Environmental risk factors for human ARHI have been studied extensively. Among these are noise exposure, smoking, ototoxic medication, and cardiovascular disease and its risk factors.<a href="#bib1" rid="bib1 bib4 bib5" class=" bibr popnode">1,4,5</a> Only very recently investigators have started to elucidate the genetic architecture of human ARHI. Familial aggregation of ARHI is well established now. Heritability estimates vary between 0.25 and 0.75, depending, among other factors, on study design (families versus twins), age range of the study population, and the phenotype studied.<a href="#bib6" rid="bib6 bib7 bib8 bib9" class=" bibr popnode">6–9</a></p><p id="__p5">To date, two linkage studies for ARHI have been published.<a href="#bib10" rid="bib10 bib11" class=" bibr popnode">10,11</a> These studies resulted in a number of weakly suggestive linkage peaks. A number of candidate gene-based association studies has been carried out, which are reviewed in Van Eyken et&nbsp;al.<a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690633">4</a> Recently, by using a candidate gene approach, we reported associations between hearing ability and variants in the <em>GRHL2</em> (MIM <a href="/omim/608576" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608576</a>) gene.<a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690647">12</a></p><p id="__p6">In this paper, we present the results of a cross-sectional family-based genetic study employing audiometric data. With high-density SNP data, a genome-wide linkage as well as an association scan were performed. Principal component analysis was used to reduce the dimensionality of the multivariate phenotype, while capturing most of the information contained in the audiometric data. This study reports a genome-wide significant linkage to an ARHI trait.</p><p id="__p7">DNA samples were collected from 1126 subjects of European descent from 204 large sibships by 9 centers in 7 European countries. Neither families nor subjects were ascertained based on phenotype. Samples were collected via population registries or via audiological consultations. The study was approved by the ethics committees or the appropriate local institutional review boards at all participating universities or hospitals. Informed consent was obtained from all study participants. All subjects underwent an otoscopic investigation and completed an extended questionnaire detailing medical history and exposure to environmental factors. Subjects with medical conditions or pathologies that could potentially affect hearing ability were excluded. Air-conduction pure-tone hearing thresholds were measured at 0.25, 0.5, 1, 2, 4, and 8 kilohertz (kHz). Bone conduction was tested at 0.5, 1, 2, and 4 kHz. Audiological exclusion criteria were an air-bone gap of more than 15 dB (dB) averaged over 0.5, 1, and 2 kHz in one or both ears or asymmetrical hearing loss with a difference in air-conduction thresholds exceeding 20 dB in at least 2 frequencies between 0.5, 1, and 2 kHz. After genotypic data cleaning, phenotypic data (air-conduction pure-tone hearing thresholds) were available for 955 subjects. This sample contained 430 men (45%) and 525 women (55%) with a median age of 61 (ranging from 49 to 76). The median number of phenotyped sibs per sibship was 5 with a maximum of 10. All subjects were genotyped with the Affymetrix 250K <em>Sty</em> chip (238,000 SNPs). To aid in determining identity-by-descent status, these subjects included 126 unphenotyped sibs and parents. Genotype calling was done with the BRLMM algorithm of Affymetrix.</p><p id="__p8">Motivated by the assumption that unilateral hearing loss primarily reflects environmental effects, for each subject, hearing thresholds for the better-hearing ear were selected for the analysis, based on the average over all frequencies. Individual hearing thresholds <em>Y</em> were transformed as follows: log<sub>10</sub>(20 + <em>Y</em>). Mean collecting center, age, age squared, and age cubed effects were regressed out. This was carried out for males and females separately. To deal with the significant heteroscedasticity that was present mainly in the high frequencies, residuals were scaled. Subsequently, classical principal component analysis was conducted on the combined centered and scaled residuals. In addition, the principal components of heritability as proposed by Ott and Rabinowitz<a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690634">13</a> were calculated. These analyses were carried out with R and SAS version 9.1.3. The calculation of the principal components of heritability entails solving a generalized eigenproblem that was solved numerically with the GENEIG subroutine of SAS. Variance components model-based heritability calculations were carried out with the MIXED procedure in SAS and heritability was estimated as twice the intraclass correlation.<a href="#bib14" rid="bib14" class=" bibr popnode">14</a></p><p id="__p9">Quality control was carried out with the PLINK toolset.<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690636">15</a> Six subjects were excluded because of poor sample call rate (&lt;94%). The average genotyping rate of the remaining samples was 99.13%. Nonpositioned SNPs (74) and SNPs with more than 10% missingness (1740) were removed from the analysis. Familial relationships were checked with GRR<a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690627">16</a> with 3000 randomly selected SNPs. Fourteen subjects were removed because of relationship inconsistencies. There were four pairs of monozygous twins in the data. For each of these pairs, the subject with the lowest sample call rate was removed from the analysis. Concordance rates for the four pairs varied from 99.71% to 99.81%. One subject was removed from the analysis because it had a large negative inbreeding coefficient, which is indicative of sample contamination. Five subjects were excluded because they no longer had relatives in the sample set. A further four subjects were removed because no sex was specified. A sex check identified six discrepancies. For these subjects, sex was imputed based on the SNP data.</p><p id="__p10">Multidimensional scaling on a genetic distance matrix in which each entry corresponded to 1 minus the genome-wide mean proportion of alleles shared identical-by-state (IBS) revealed a family with possible non-European ancestry that was excluded from further analyses (<a href="#app3" rid="app3" class=" sec">Figure&nbsp;S1</a> available online). These steps left 1081 genotyped subjects from 200 sibships amenable to analysis.</p><p id="__p11">Multidimensional scaling revealed a clear separation between samples from Western European countries (WEU) and from Finland (FIN) and a south-north axis within WEU (<a href="#app3" rid="app3" class=" sec">Figure&nbsp;S1</a>). Therefore, it was decided to conduct Hardy-Weinberg equilibrium (HWE) testing for samples from WEU and from FIN separately. One individual per sibship was randomly selected and an exact test<a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690628">17</a> was used on this subsample of unrelated individuals. This process was repeated 10 times and the geometric mean of the resulting 10 p values was regarded as the final p value for HWE tests for each SNP.</p><p id="__p12">SNPs with minor allele frequencies (MAF) &gt; 5% and HWE p values &gt; 0.001 in both WEU and FIN were used in the association analysis (169,154 SNPs). For linkage analysis, a subset of informative SNPs was used. Based on the Marshfield genetic map, chromosomes were divided into segments 0.1 cM in length and for each segment the first SNP with MAF &gt; 15% and HWE p value &gt; 0.05 in both WEU and FIN was selected. This yielded a map of 26,879 SNPs with an average marker spacing of ∼0.14 cM. This map density should be sufficient to extract most of the inheritance information<a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690642">18</a> and approaches the upper limit of what current linkage analysis software can handle. To assess the impact of the choice of SNPs on the final results, analyses were repeated with two other map densities. A second set of 15,839 SNPs with an average marker spacing of ∼0.24 cM was analyzed (4250 SNPs overlapped with the previous set), as well as a third set for which every other marker of the second set was selected yielding a map of 7,723 SNPs with an average spacing of ∼0.49 cM (autosomes only). The results for the three different maps were virtually identical (see <a href="#app3" rid="app3" class=" sec">Figure&nbsp;S2</a> for PC3). The multipoint entropy-based measure of marker informativeness reported for each analysis position by MERLIN<a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690638">19</a> was high throughout the genome, generally being well above 90% (<a href="#app3" rid="app3" class=" sec">Figure&nbsp;S2</a>).</p><p id="__p13">Possible genotyping errors were detected by looking for unlikely double recombinants via MERLIN. During this process, linkage disequilibrium (LD) was modeled via the approach of Abecasis and Wigginton<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690626">20</a> that groups markers in LD into clusters. As recommended, markers with <em>r</em><sup>2</sup> &gt; 0.1 were grouped into clusters. The percentage of genotypes identified as erroneous was 0.29%. These genotypes were set to missing.</p><p id="__p14">Quantitative trait linkage analysis was carried out with the regression-based procedure of Sham et&nbsp;al.<a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690632">21</a> that has been implemented in MERLIN-REGRESS.<a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690640">21</a> Because the presence of LD may result in upward biases in multipoint linkage analysis, the analysis was carried out both ignoring any LD between markers and via the approach of Abecasis and Wigginton where SNPs with <em>r</em><sup>2</sup> &gt; 0.1 were organized into clusters.<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690630">20</a> Results obtained with both approaches were virtually identical (<a href="#app3" rid="app3" class=" sec">Figure&nbsp;S3</a>). To deal with potential outliers, quantile normalization was applied to trait values. The X chromosome was analyzed separately with a variance-components model in MINX, the X-specific version of MERLIN.</p><p id="__p15">To determine the empirical genome-wide significance level, we conducted simulations. With MERLIN, we generated 1000 simulated data sets under the null hypothesis. In each simulation run, data were generated for WEU and FIN separately. The data for WEU and FIN were subsequently combined and analyzed via MERLIN-REGRESS. The empirical false positive rate for a LOD score <em>L</em> was calculated with the formula (<em>r</em> + 1)/1001, where <em>r</em> is the number of times a simulation run had one or more LOD scores greater than <em>L</em>.<a href="#bib22" rid="bib22 bib23" class=" bibr popnode">22,23</a> Because this task was very computer intensive, we used the sparsest map and ignored LD.</p><p id="__p16">Quantitative trait association testing was performed with MERLIN, which implements a variance component model that models background polygenic effects.<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690646">24</a> To deal with potential outliers, quantile normalization was used throughout. The X chromosome was analyzed separately with MINX. Throughout all calculations, an additive model was assumed. To avoid spurious associations resulting from population stratification, association was tested for WEU (751 subjects) and FIN (204 subjects) samples separately. Results were subsequently combined by meta-analysis. To assess the presence of residual population stratification within the WEU and FIN samples, the resulting empirical p value distributions were inspected by quantile-quantile plots and by calculating genomic control inflation factors.<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690625">25</a> To combine evidence for association in WEU and FIN, we used the meta-analytic approach described in Sanna et&nbsp;al.<a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690648">26</a> In agreement with other studies, we took a p value cutoff of 5×10<sup>−7</sup> to declare genome-wide significance.<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690629">27</a> We further explored the data by carrying out an analysis stratified by gender. The annotation software tool WGAViewer<a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690644">28</a> was used to aid in interpreting the association results.</p><p id="__p17">Hearing thresholds were corrected for age, gender, and collection center, and classical principal component analysis (PCA) was performed on the scaled residuals. The resulting eigenvalues, the loadings (coefficients of the eigenvectors), and heritability estimates for the first three classical principal components (PCs) are given in <a href="/pmc/articles/PMC2556434/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>. The eigenvalue for a PC corresponds to the proportion of the total variability explained by that PC. The first three PCs capture 87.7% of the variation. PC1 is clearly a “size” variable, an overall measure of one's hearing ability. PC2 and PC3 are audiogram shape variables: PC2 contrasts low with high frequencies, whereas PC3 contrasts the middle with the low and the high frequencies. <a href="/pmc/articles/PMC2556434/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a> shows how these traits relate to the audiograms. Based on nonstandard likelihood ratio tests, there was significant familial aggregation for all traits (all p values &lt; 0.001). Heritability was highest for PC1 (66.3%) and lowest for PC2 (27.2%).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC2556434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140064685542576" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC2556434/bin/gr1.jpg"></div></a><div id="largeobj_idm140064685542576" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2556434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC2556434/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC2556434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p39">Comparison of Audiograms between Subjects with High and Low Values for PC1, PC2, and PC3</p><p id="__p40">Three subjects were randomly selected from the lower and upper 10% extremes of the distribution for each of the three traits. To show the extent of intrasubject variability, audiograms for both ears are given. Left and right ears correspond to filled and open circles, respectively. For PC1, on average, hearing loss is much less pronounced in individuals from higher extremes (top row) compared to the lower extremes (second row from the top). Subjects with low values for PC2 (fourth row from the top) tend to have a much more sloping audiogram, compared to subjects with high values for PC2 (third row from the top). Subjects with high values for PC3 (second row from the bottom) tend to have a more concave audiogram, compared to subjects in the lower extreme for PC3 (bottom row).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p43">Eigenvalues, Coefficients of the Eigenvectors, and Heritabilities of the First Three Classical Principal Components and of the First Three Principal Components that Maximize the Heritability</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140064722456848" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Trait</th><th rowspan="1" colspan="1">PC1</th><th rowspan="1" colspan="1">PC2</th><th rowspan="1" colspan="1">PC3</th><th rowspan="1" colspan="1">PCH1</th><th rowspan="1" colspan="1">PCH2</th><th rowspan="1" colspan="1">PCH3</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Eigenvalue</td><td rowspan="1" colspan="1" align="center">0.603</td><td rowspan="1" colspan="1" align="center">0.174</td><td rowspan="1" colspan="1" align="center">0.100</td><td rowspan="1" colspan="1" align="center">0.524</td><td rowspan="1" colspan="1" align="center">0.241</td><td rowspan="1" colspan="1" align="center">0.155</td></tr><tr><td colspan="7" rowspan="1"><hr></td></tr><tr><td colspan="7" rowspan="1">Coefficients Eigenvector</td></tr><tr><td colspan="7" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">0.25 kHz</td><td rowspan="1" colspan="1" align="center">−0.397</td><td rowspan="1" colspan="1" align="center">0.397</td><td rowspan="1" colspan="1" align="center">0.551</td><td rowspan="1" colspan="1" align="center">0.590</td><td rowspan="1" colspan="1" align="center">0.915</td><td rowspan="1" colspan="1" align="center">0.497</td></tr><tr><td rowspan="1" colspan="1">0.5 kHz</td><td rowspan="1" colspan="1" align="center">−0.448</td><td rowspan="1" colspan="1" align="center">0.384</td><td rowspan="1" colspan="1" align="center">0.126</td><td rowspan="1" colspan="1" align="center">0.198</td><td rowspan="1" colspan="1" align="center">0.193</td><td rowspan="1" colspan="1" align="center">−1.267</td></tr><tr><td rowspan="1" colspan="1">1 kHz</td><td rowspan="1" colspan="1" align="center">−0.451</td><td rowspan="1" colspan="1" align="center">0.241</td><td rowspan="1" colspan="1" align="center">−0.354</td><td rowspan="1" colspan="1" align="center">0.238</td><td rowspan="1" colspan="1" align="center">−0.406</td><td rowspan="1" colspan="1" align="center">0.096</td></tr><tr><td rowspan="1" colspan="1">2 kHz</td><td rowspan="1" colspan="1" align="center">−0.440</td><td rowspan="1" colspan="1" align="center">−0.123</td><td rowspan="1" colspan="1" align="center">−0.531</td><td rowspan="1" colspan="1" align="center">0.381</td><td rowspan="1" colspan="1" align="center">−0.845</td><td rowspan="1" colspan="1" align="center">0.251</td></tr><tr><td rowspan="1" colspan="1">4 kHz</td><td rowspan="1" colspan="1" align="center">−0.378</td><td rowspan="1" colspan="1" align="center">−0.510</td><td rowspan="1" colspan="1" align="center">−0.118</td><td rowspan="1" colspan="1" align="center">0.108</td><td rowspan="1" colspan="1" align="center">−0.241</td><td rowspan="1" colspan="1" align="center">0.142</td></tr><tr><td rowspan="1" colspan="1">8 kHz</td><td rowspan="1" colspan="1" align="center">−0.319</td><td rowspan="1" colspan="1" align="center">−0.602</td><td rowspan="1" colspan="1" align="center">0.510</td><td rowspan="1" colspan="1" align="center">−0.046</td><td rowspan="1" colspan="1" align="center">0.451</td><td rowspan="1" colspan="1" align="center">0.779</td></tr><tr><td colspan="7" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Heritability</td><td rowspan="1" colspan="1" align="center">0.663</td><td rowspan="1" colspan="1" align="center">0.272</td><td rowspan="1" colspan="1" align="center">0.375</td><td rowspan="1" colspan="1" align="center">0.703</td><td rowspan="1" colspan="1" align="center">0.391</td><td rowspan="1" colspan="1" align="center">0.269</td></tr></tbody></table></div><div id="largeobj_idm140064722456848" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2556434/table/tbl1/?report=objectonly">Open in a separate window</a></div></div><p id="__p18">Principal components of heritability (PCHs) were calculated as well because these may offer a significant power advantage if the individual variables comprised in the multivariate phenotype vector are correlated.<a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690637">13</a> The results for the first three PCHs are given in <a href="/pmc/articles/PMC2556434/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>. Compared to the classical PCs, the PCHs have a less clear-cut biological interpretation. <a href="/pmc/articles/PMC2556434/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a> shows the correlations between the PCs and the PCHs. It can be seen that PC1 is strongly correlated with PCH1 (Spearman correlation of 0.95), PC2 is moderately correlated with PCH3 (0.79), and PC3 is strongly correlated with PCH2 (0.95). This was also reflected in the linkage results: LOD score curves were virtually identical between correlated PCs (not shown). We therefore pursued only the classical PCs because these have a more straightforward interpretation. In addition, they allow us to compare results with future association studies based on unrelated subjects, because PCHs cannot be calculated for unrelated subjects.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p44">Spearman Rank-Order Correlations between the First Three Classical Principal Components and the Principal Components of Heritability</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140064687600736" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">PCH1</th><th rowspan="1" colspan="1">PCH2</th><th rowspan="1" colspan="1">PCH3</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">PC1</td><td rowspan="1" colspan="1" align="center">−0.953</td><td rowspan="1" colspan="1" align="center">0.113</td><td rowspan="1" colspan="1" align="center">−0.171</td></tr><tr><td rowspan="1" colspan="1">PC2</td><td rowspan="1" colspan="1" align="center">0.203</td><td rowspan="1" colspan="1" align="center">0.264</td><td rowspan="1" colspan="1" align="center">−0.790</td></tr><tr><td rowspan="1" colspan="1">PC3</td><td rowspan="1" colspan="1" align="center">−0.013</td><td rowspan="1" colspan="1" align="center">0.947</td><td rowspan="1" colspan="1" align="center">0.219</td></tr></tbody></table></div><div id="largeobj_idm140064687600736" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2556434/table/tbl2/?report=objectonly">Open in a separate window</a></div></div><p id="__p19">Quantitative trait linkage analysis was carried out for the first three PCs via the reverse Haseman-Elston method of Sham et&nbsp;al.<a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690635">21</a> The only trait for which multipoint LOD scores greater than 3 were observed was PC3 (<a href="#app3" rid="app3" class=" sec">Figure&nbsp;S4</a>; <a href="#app3" rid="app3" class=" sec">Figure&nbsp;S3</a> shows the results for PC1 and PC2). Based on the sparsest map used, on chromosome 8 a multipoint LOD score of 4.23 was attained at the position of SNP rs4512366 (physical position 125528900; pointwise p value = 5×10<sup>−6</sup>). On chromosome 9, a multipoint LOD score of 3.13 was attained at the position of SNP rs10814227 (physical position 35320608; pointwise p value = 7×10<sup>−5</sup>).</p><p id="__p20">We conducted 1000 simulations to assess the empirical genome-wide significance level of these linkage peaks. For the data at hand, in order to reach genome-wide significance at the 5% level, a LOD score should exceed 3.55 (see <a href="#app3" rid="app3" class=" sec">Figure&nbsp;S5</a>). The maximal LOD score for chromosome 8 reached genome-wide significance (p value = 0.0170). The peak on chromosome 9 did not (p value = 0.1039). The region reaching significance on chromosome 8 spanned approximately 7 Mb, roughly between SNPs rs3765212 and rs4601326. This corresponds to cytogenetic chromosome bands 8q24.13-q24.22 (UCSC Genome Browser).</p><p id="__p21">We further explored the linkage peak for PC3 on chromosome 8. <a href="/pmc/articles/PMC2556434/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a> shows multipoint and singlepoint LOD scores together with informativities for chromosome 8. The singlepoint analysis revealed that three SNPs have high singlepoint LOD scores of 5.26, 5.24, and 4.75, respectively. We repeated the analysis, leaving out these three SNPs. The results of the multipoint analysis were barely affected (maximal multipoint LOD score of 4.22). Inspecting SNP informativities is not very enlightening here and informativity of individual SNPs is low. A likely explanation for the three high singlepoint LOD scores is that the informativity of the corresponding SNPs in the (unknown) subset of pedigrees in which a locus on 8q is truly linked is high compared to surrounding markers. <a href="#app3" rid="app3" class=" sec">Table S1</a> provides additional information on each family and family-specific contributions to the multipoint LOD score at the positions of the SNPs beneath the linkage peak. The latter are based on a variance components analysis that yielded results that were strikingly similar to the regression-based analysis (see <a href="#app3" rid="app3" class=" sec">Figure&nbsp;S6</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC2556434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140064676398064" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC2556434/bin/gr2.jpg"></div></a><div id="largeobj_idm140064676398064" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2556434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC2556434/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC2556434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p41">Chromosome 8 Multipoint and Singlepoint LOD Scores for PC3 together with Marker Informativities</p><p id="__p42">Gray and red dots represent singlepoint marker informativities and LOD scores, respectively. Gray and blue solid lines represent multipoint informativities and LOD scores, respectively.</p></div></div></div><p id="__p22">Quantitative trait association testing was performed with a variance component model to account for background polygenic effects. Genomic control inflation factors<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690639">25</a> varied from 1.011 to 1.035, suggesting that residual population stratification is of minor concern. No association signals meeting genome-wide significance were observed for any of the three traits, neither in the analyses stratified by origin and gender, nor in the meta-analysis (see <a href="#app3" rid="app3" class=" sec">Figure&nbsp;S7</a> for plots of meta-analysis −log<sub>10</sub>(p values) for the three traits). <a href="#app3" rid="app3" class=" sec">Figure&nbsp;S4</a> shows for PC3 the results of the linkage analysis superimposed on those of the association analysis, together with the positions of the 45 known nonsyndromic hearing-loss genes. We further scrutinized the region beneath the significant as well as the suggestive linkage peak for clustering of semi-independent association signals with p values &lt; 10<sup>−3</sup> in a gene region, taking into account LD, and for hits in or near plausible candidate genes. We found no evidence for association with PC3 in this region.</p><p id="__p23">We found a genome-wide significant linkage signal on 8q24.13-q24.22 for PC3, a trait related to audiogram shape that was obtained via principal component analysis. These results suggest that one or several rare variants that have a moderate to large effect on audiogram shape in the elderly reside in this broad chromosomal region.</p><p id="__p24">To date, two linkage studies for ARHI have been published. Garringer et&nbsp;al.<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690631">11</a> reported suggestive linkage (LOD score = 2.5) in the 3q22 region based on a sample of male dizygotic twins concordant for self-reported hearing loss. The study of DeStefano et&nbsp;al.<a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690643">10</a> used extended pedigrees from the Framingham cohort. Based upon the average pure-tone hearing thresholds for the middle and low frequencies (after correction for age and gender), several regions of suggestive linkage were found, the highest of which reached a LOD score of 2.1 on 11q13. Because the phenotypes analyzed in the two linkage studies cited above correlate with PC1 (unpublished results), suggestive linkage peaks from these studies were compared with our results. We did not find any evidence in favor of true linkage signals.</p><p id="__p25">Whereas linkage analysis is the preferred strategy for identifying rare genetic variants with large effects, genome-wide association (GWA) analysis is the method of choice for identifying common genetic variants with only small effects. The absence of any genome-wide significant association signals in this study may be due to a variety of factors, which may act in concert. The Affymetrix 250K <em>Sty</em> chip has low genomic coverage and this presents the major limitation of our association study. The modest sample size presents a further limitation. In the light of what we now know about typical effect sizes for complex diseases and traits, power to detect any causative variants in this association study is probably very low.<a href="#bib29" rid="bib29 bib30 bib31" class=" bibr popnode">29–31</a> Furthermore, because it is plausible that the culprit variants for ARHI are not manifesting themselves during reproductive age, the resulting absence of any selective pressure could have easily led to the propagation of many harmful variants in human populations by genetic drift or genetic hitchhiking. Because the samples studied have different ethnicities, substantial genetic and allelic heterogeneity may wash out any effect. Synergistic gene-gene and gene-environment interactions may further dilute the effect and reduce power. Last, but not least, the precision of hearing threshold measurements is low (5 dB). Therefore, despite the substantial role of genetics in the etiology of ARHI as indicated by heritability studies, the signal-to-noise ratio may be too low to pick up any signal of the individual contributing variants.</p><p id="__p26">Based on a candidate gene approach, we previously reported associations between variants in the <em>GRHL2</em> gene, a known monogenic nonsyndromic hearing loss gene, and hearing ability.<a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690645">12</a> In this study, subjects were assigned case or control status based on whether they belonged to the upper or lower extreme of the distribution of the average of the high-frequency hearing thresholds, corrected for age and gender. Because this variable is correlated to PC1 (unpublished results), we looked for evidence of replication. The top hit from the candidate gene study (rs10955255) was not among the SNPs analyzed in the present study. We scrutinized association signals in the region of this SNP but failed to find any evidence in favor of replication. However, the maximum <em>r</em><sup>2</sup> between the currently tested SNPs and rs10955255 was only 0.44.</p><p id="__p27">The present study differs from previous genetic studies for ARHI in an important way. Previous studies were based on either self-reported hearing loss or simple averages of age- and gender-corrected hearing thresholds for different frequencies. By using such strategies, many biologically relevant features of the audiometric data will be lost because completely different audiometric shapes may lead to similar phenotypic values when thresholds are averaged over frequencies. By using principal component analysis, we obtained two measures for audiogram shape, PC2 and PC3, which measure slope and concavity, respectively. A number of studies in which human temporal bones were compared to premortem hearing tests have shown that audiometric patterns correlate with pathophysiology.<a href="#bib2" rid="bib2 bib32" class=" bibr popnode">2,32</a> Nelson et&nbsp;al.<a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221690641">2</a> studied individuals with downward sloping audiometric patterns and reported that the severity of hearing loss, based on audiometric thresholds, was highly associated with the degeneration of stria vascularis volumes and outer and inner hair cell and ganglion cell populations. The slope of the audiogram was associated with the extent of ganglion cell degeneration. Therefore, it is plausible that PC2 and PC3 contain critically important clues to elucidate the molecular etiology of ARHI.</p><p id="__p28" class="p p-last">The significant heritability estimates for the three traits studied suggest that the role of genetics in the etiology of ARHI is substantial. In view of the fact that a plethora of genes plays a role in the hearing system, ARHI probably will turn out to be a very heterogeneous disorder. For nonsyndromic monogenic hearing loss only, 45 genes have been identified (Hereditary Hearing Loss webpage). Because of the great difficulty of separating the age effect from harmful environmental effects, a big challenge remains the collection of high-quality data. In this respect, the collection of cohort data should be heavily encouraged because the study of individual longitudinal profiles may yield important insights that could lead to the discovery of different genetic subtypes of ARHI.</p></div><div id="idm140064677869104" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140064677869104title">Acknowledgments</h2><div class="sec"><p id="__p38">We thank all the families and the staff from all the collecting centers for their cooperation. J.R.H. wishes to thank Gonçalo Abecasis for kindly providing help with MERLIN and for providing valuable suggestions for the analysis. This work was financially supported by the European Community (5th Framework project QLRT-2001-00331), the University of Antwerp (TOP project), and the German Federal Ministry of Education and Research through the National Genome Research Network (grant 01GR0416 to P.N.).</p></div></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Six Figures:</strong><div class="sup-box half_rhythm" id="idm140064676262336"><a href="/pmc/articles/PMC2556434/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(730K, pdf)</sup></div></div><div class="sec suppmat" id="mmc2"><!--caption a9--><strong>Table S1. Family-Specific Contributions to the Multipoint LOD Scores for PC3 at the Positions of the SNPs beneath the Significant Linkage Peak on Chromosome 8:</strong><div class="sup-box half_rhythm" id="idm140064685362464"><a href="/pmc/articles/PMC2556434/bin/mmc2.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(146K, xls)</sup></div></div><p></p></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p30" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li><p id="__p31">GRR, <a href="http://www.sph.umich.edu/csg/abecasis/GRR/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/GRR/</a></p></li><li><p id="__p32">Hereditary Hearing Loss webpage, <a href="http://webh01.ua.ac.be/hhh/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://webh01.ua.ac.be/hhh/</a></p></li><li><p id="__p33">MERLIN, <a href="http://www.sph.umich.edu/csg/abecasis/Merlin/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Merlin/index.html</a> (for MERLIN-REGRESS, MERLIN, and MINX)</p></li><li><p id="__p34">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.ncbi.nlm.nih.gov/Omim/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/Omim/</a></p></li><li><p id="__p35">PLINK, <a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a></p></li><li><p id="__p36">R, <a href="http://www.R-project.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.R-project.org</a></p></li><li><p id="__p37">UCSC Genome Browser, <a href="http://genome.ucsc.edu/cgi-bin/hgGateway" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.ucsc.edu/cgi-bin/hgGateway</a></p></li></ul><p></p></div><div id="idm140064680837456" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140064680837456title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Gates G.A., Mills J.H. Presbycusis. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">366</span>:1111–1120.</span> [<a href="/pubmed/16182900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Presbycusis&amp;author=G.A.+Gates&amp;author=J.H.+Mills&amp;volume=366&amp;publication_year=2005&amp;pages=1111-1120&amp;pmid=16182900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Nelson E.G., Hinojosa R. Presbycusis: A human temporal bone study of individuals with downward sloping audiometric patterns of hearing loss and review of the literature. <span><span class="ref-journal">Laryngoscope. </span>2006;<span class="ref-vol">116</span>:1–12.</span> [<a href="/pubmed/16946668" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Laryngoscope&amp;title=Presbycusis:+A+human+temporal+bone+study+of+individuals+with+downward+sloping+audiometric+patterns+of+hearing+loss+and+review+of+the+literature&amp;author=E.G.+Nelson&amp;author=R.+Hinojosa&amp;volume=116&amp;publication_year=2006&amp;pages=1-12&amp;pmid=16946668&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">Ohlemiller K.K. Age-related hearing loss: The status of Schuknecht's typology. <span><span class="ref-journal">Curr. Opin. Otolaryngol. Head Neck Surg. </span>2004;<span class="ref-vol">12</span>:439–443.</span> [<a href="/pubmed/15377958" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Otolaryngol.+Head+Neck+Surg.&amp;title=Age-related+hearing+loss:+The+status+of+Schuknecht's+typology&amp;author=K.K.+Ohlemiller&amp;volume=12&amp;publication_year=2004&amp;pages=439-443&amp;pmid=15377958&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Van Eyken E., Van Camp G., Van Laer L. The complexity of age-related hearing impairment: Contributing environmental and genetic factors. <span><span class="ref-journal">Audiol. Neurootol. </span>2007;<span class="ref-vol">12</span>:345–358.</span> [<a href="/pubmed/17664866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Audiol.+Neurootol.&amp;title=The+complexity+of+age-related+hearing+impairment:+Contributing+environmental+and+genetic+factors&amp;author=E.+Van+Eyken&amp;author=G.+Van+Camp&amp;author=L.+Van+Laer&amp;volume=12&amp;publication_year=2007&amp;pages=345-358&amp;pmid=17664866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Fransen E., Topsakal V., Hendrickx J.J., Van Laer L., Huyghe J.R., Van Eyken E., Lemkens N., Hannula S., Maki-Torkko E., Jensen M. Occupational noise, smoking, and a high body mass index are risk factors for age-related hearing impairment and moderate alcohol consumption is protective: A European population-based multicenter study. <span><span class="ref-journal">J. Assoc. Res. Otolaryngol. </span>2008;<span class="ref-vol">9</span>:264–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2492985/">PMC free article</a>]</span> [<a href="/pubmed/18543032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Assoc.+Res.+Otolaryngol.&amp;title=Occupational+noise,+smoking,+and+a+high+body+mass+index+are+risk+factors+for+age-related+hearing+impairment+and+moderate+alcohol+consumption+is+protective:+A+European+population-based+multicenter+study&amp;author=E.+Fransen&amp;author=V.+Topsakal&amp;author=J.J.+Hendrickx&amp;author=L.+Van+Laer&amp;author=J.R.+Huyghe&amp;volume=9&amp;publication_year=2008&amp;pages=264-276&amp;pmid=18543032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">Karlsson K.K., Harris J.R., Svartengren M. Description and primary results from an audiometric study of male twins. <span><span class="ref-journal">Ear Hear. </span>1997;<span class="ref-vol">18</span>:114–120.</span> [<a href="/pubmed/9099560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ear+Hear.&amp;title=Description+and+primary+results+from+an+audiometric+study+of+male+twins&amp;author=K.K.+Karlsson&amp;author=J.R.+Harris&amp;author=M.+Svartengren&amp;volume=18&amp;publication_year=1997&amp;pages=114-120&amp;pmid=9099560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">Gates G.A., Couropmitree N.N., Myers R.H. Genetic associations in age-related hearing thresholds. <span><span class="ref-journal">Arch. Otolaryngol. Head Neck Surg. </span>1999;<span class="ref-vol">125</span>:654–659.</span> [<a href="/pubmed/10367922" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Otolaryngol.+Head+Neck+Surg.&amp;title=Genetic+associations+in+age-related+hearing+thresholds&amp;author=G.A.+Gates&amp;author=N.N.+Couropmitree&amp;author=R.H.+Myers&amp;volume=125&amp;publication_year=1999&amp;pages=654-659&amp;pmid=10367922&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">Christensen K., Frederiksen H., Hoffman H.J. Genetic and environmental influences on self-reported reduced hearing in the old and oldest old. <span><span class="ref-journal">J. Am. Geriatr. Soc. </span>2001;<span class="ref-vol">49</span>:1512–1517.</span> [<a href="/pubmed/11890591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Geriatr.+Soc.&amp;title=Genetic+and+environmental+influences+on+self-reported+reduced+hearing+in+the+old+and+oldest+old&amp;author=K.+Christensen&amp;author=H.+Frederiksen&amp;author=H.J.+Hoffman&amp;volume=49&amp;publication_year=2001&amp;pages=1512-1517&amp;pmid=11890591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Viljanen A., Era P., Kaprio J., Pyykko I., Koskenvuo M., Rantanen T. Genetic and environmental influences on hearing in older women. <span><span class="ref-journal">J. Gerontol. A Biol. Sci. Med. Sci. </span>2007;<span class="ref-vol">62</span>:447–452.</span> [<a href="/pubmed/17452741" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Gerontol.+A+Biol.+Sci.+Med.+Sci.&amp;title=Genetic+and+environmental+influences+on+hearing+in+older+women&amp;author=A.+Viljanen&amp;author=P.+Era&amp;author=J.+Kaprio&amp;author=I.+Pyykko&amp;author=M.+Koskenvuo&amp;volume=62&amp;publication_year=2007&amp;pages=447-452&amp;pmid=17452741&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">DeStefano A.L., Gates G.A., Heard-Costa N., Myers R.H., Baldwin C.T. Genomewide linkage analysis to presbycusis in the Framingham Heart Study. <span><span class="ref-journal">Arch. Otolaryngol. Head Neck Surg. </span>2003;<span class="ref-vol">129</span>:285–289.</span> [<a href="/pubmed/12622536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Otolaryngol.+Head+Neck+Surg.&amp;title=Genomewide+linkage+analysis+to+presbycusis+in+the+Framingham+Heart+Study&amp;author=A.L.+DeStefano&amp;author=G.A.+Gates&amp;author=N.+Heard-Costa&amp;author=R.H.+Myers&amp;author=C.T.+Baldwin&amp;volume=129&amp;publication_year=2003&amp;pages=285-289&amp;pmid=12622536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">Garringer H.J., Pankratz N.D., Nichols W.C., Reed T. Hearing impairment susceptibility in elderly men and the DFNA18 locus. <span><span class="ref-journal">Arch. Otolaryngol. Head Neck Surg. </span>2006;<span class="ref-vol">132</span>:506–510.</span> [<a href="/pubmed/16702566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Otolaryngol.+Head+Neck+Surg.&amp;title=Hearing+impairment+susceptibility+in+elderly+men+and+the+DFNA18+locus&amp;author=H.J.+Garringer&amp;author=N.D.+Pankratz&amp;author=W.C.+Nichols&amp;author=T.+Reed&amp;volume=132&amp;publication_year=2006&amp;pages=506-510&amp;pmid=16702566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Van Laer L., Van Eyken E., Fransen E., Huyghe J.R., Topsakal V., Hendrickx J.J., Hannula S., Maki-Torkko E., Jensen M., Demeester K. The grainyhead like 2 gene (GRHL2), alias TFCP2L3, is associated with age-related hearing impairment. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:159–169.</span> [<a href="/pubmed/17921507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=The+grainyhead+like+2+gene+(GRHL2),+alias+TFCP2L3,+is+associated+with+age-related+hearing+impairment&amp;author=L.+Van+Laer&amp;author=E.+Van+Eyken&amp;author=E.+Fransen&amp;author=J.R.+Huyghe&amp;author=V.+Topsakal&amp;volume=17&amp;publication_year=2008&amp;pages=159-169&amp;pmid=17921507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Ott J., Rabinowitz D. A principal-components approach based on heritability for combining phenotype information. <span><span class="ref-journal">Hum. Hered. </span>1999;<span class="ref-vol">49</span>:106–111.</span> [<a href="/pubmed/10077732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered.&amp;title=A+principal-components+approach+based+on+heritability+for+combining+phenotype+information&amp;author=J.+Ott&amp;author=D.+Rabinowitz&amp;volume=49&amp;publication_year=1999&amp;pages=106-111&amp;pmid=10077732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Falconer D., Mackay T.F.C.  Longman; Harlow: 1997. Introduction to Quantitative Genetics. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Falconer+D.+Mackay+T.F.C.+Introduction+to+Quantitative+Genetics+1997+Longman+Harlow+" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J. PLINK: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Abecasis G.R., Cherny S.S., Cookson W.O., Cardon L.R. GRR: Graphical representation of relationship errors. <span><span class="ref-journal">Bioinformatics. </span>2001;<span class="ref-vol">17</span>:742–743.</span> [<a href="/pubmed/11524377" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GRR:+Graphical+representation+of+relationship+errors&amp;author=G.R.+Abecasis&amp;author=S.S.+Cherny&amp;author=W.O.+Cookson&amp;author=L.R.+Cardon&amp;volume=17&amp;publication_year=2001&amp;pages=742-743&amp;pmid=11524377&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Wigginton J.E., Cutler D.J., Abecasis G.R. A note on exact tests of Hardy-Weinberg equilibrium. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2005;<span class="ref-vol">76</span>:887–893.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1199378/">PMC free article</a>]</span> [<a href="/pubmed/15789306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+note+on+exact+tests+of+Hardy-Weinberg+equilibrium&amp;author=J.E.+Wigginton&amp;author=D.J.+Cutler&amp;author=G.R.+Abecasis&amp;volume=76&amp;publication_year=2005&amp;pages=887-893&amp;pmid=15789306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Evans D.M., Cardon L.R. Guidelines for genotyping in genomewide linkage studies: single-nucleotide-polymorphism maps versus microsatellite maps. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2004;<span class="ref-vol">75</span>:687–692.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182056/">PMC free article</a>]</span> [<a href="/pubmed/15311375" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Guidelines+for+genotyping+in+genomewide+linkage+studies:+single-nucleotide-polymorphism+maps+versus+microsatellite+maps&amp;author=D.M.+Evans&amp;author=L.R.+Cardon&amp;volume=75&amp;publication_year=2004&amp;pages=687-692&amp;pmid=15311375&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Abecasis G.R., Cherny S.S., Cookson W.O., Cardon L.R. Merlin–Rapid analysis of dense genetic maps using sparse gene flow trees. <span><span class="ref-journal">Nat. Genet. </span>2002;<span class="ref-vol">30</span>:97–101.</span> [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Merlin–Rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=G.R.+Abecasis&amp;author=S.S.+Cherny&amp;author=W.O.+Cookson&amp;author=L.R.+Cardon&amp;volume=30&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Abecasis G.R., Wigginton J.E. Handling marker-marker linkage disequilibrium: Pedigree analysis with clustered markers. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2005;<span class="ref-vol">77</span>:754–767.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1271385/">PMC free article</a>]</span> [<a href="/pubmed/16252236" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Handling+marker-marker+linkage+disequilibrium:+Pedigree+analysis+with+clustered+markers&amp;author=G.R.+Abecasis&amp;author=J.E.+Wigginton&amp;volume=77&amp;publication_year=2005&amp;pages=754-767&amp;pmid=16252236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">Sham P.C., Purcell S., Cherny S.S., Abecasis G.R. Powerful regression-based quantitative-trait linkage analysis of general pedigrees. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">71</span>:238–253.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379157/">PMC free article</a>]</span> [<a href="/pubmed/12111667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Powerful+regression-based+quantitative-trait+linkage+analysis+of+general+pedigrees&amp;author=P.C.+Sham&amp;author=S.+Purcell&amp;author=S.S.+Cherny&amp;author=G.R.+Abecasis&amp;volume=71&amp;publication_year=2002&amp;pages=238-253&amp;pmid=12111667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">North B.V., Curtis D., Sham P.C. A note on the calculation of empirical P values from Monte Carlo procedures. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">71</span>:439–441.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379178/">PMC free article</a>]</span> [<a href="/pubmed/12111669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+note+on+the+calculation+of+empirical+P+values+from+Monte+Carlo+procedures&amp;author=B.V.+North&amp;author=D.+Curtis&amp;author=P.C.+Sham&amp;volume=71&amp;publication_year=2002&amp;pages=439-441&amp;pmid=12111669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">North B.V., Curtis D., Sham P.C. A note on calculation of empirical P values from Monte Carlo procedure. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2003;<span class="ref-vol">72</span>:498–499.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379244/">PMC free article</a>]</span> [<a href="/pubmed/12596795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+note+on+calculation+of+empirical+P+values+from+Monte+Carlo+procedure&amp;author=B.V.+North&amp;author=D.+Curtis&amp;author=P.C.+Sham&amp;volume=72&amp;publication_year=2003&amp;pages=498-499&amp;pmid=12596795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Chen W.M., Abecasis G.R. Family-based association tests for genomewide association scans. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:913–926.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265659/">PMC free article</a>]</span> [<a href="/pubmed/17924335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Family-based+association+tests+for+genomewide+association+scans&amp;author=W.M.+Chen&amp;author=G.R.+Abecasis&amp;volume=81&amp;publication_year=2007&amp;pages=913-926&amp;pmid=17924335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Devlin B., Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation">Sanna S., Jackson A.U., Nagaraja R., Willer C.J., Chen W.M., Bonnycastle L.L., Shen H., Timpson N., Lettre G., Usala G. Common variants in the GDF5-UQCC region are associated with variation in human height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:198–203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914680/">PMC free article</a>]</span> [<a href="/pubmed/18193045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+in+the+GDF5-UQCC+region+are+associated+with+variation+in+human+height&amp;author=S.+Sanna&amp;author=A.U.+Jackson&amp;author=R.+Nagaraja&amp;author=C.J.+Willer&amp;author=W.M.+Chen&amp;volume=40&amp;publication_year=2008&amp;pages=198-203&amp;pmid=18193045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation">The Welcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation">Ge D., Zhang K., Need A.C., Martin O., Fellay J., Urban T.J., Telenti A., Goldstein D.B. WGAViewer: Software for genomic annotation of whole genome association studies. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:640–643.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2279251/">PMC free article</a>]</span> [<a href="/pubmed/18256235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=WGAViewer:+Software+for+genomic+annotation+of+whole+genome+association+studies&amp;author=D.+Ge&amp;author=K.+Zhang&amp;author=A.C.+Need&amp;author=O.+Martin&amp;author=J.+Fellay&amp;volume=18&amp;publication_year=2008&amp;pages=640-643&amp;pmid=18256235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation">Ioannidis J.P., Trikalinos T.A., Khoury M.J. Implications of small effect sizes of individual genetic variants on the design and interpretation of genetic association studies of complex diseases. <span><span class="ref-journal">Am. J. Epidemiol. </span>2006;<span class="ref-vol">164</span>:609–614.</span> [<a href="/pubmed/16893921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Implications+of+small+effect+sizes+of+individual+genetic+variants+on+the+design+and+interpretation+of+genetic+association+studies+of+complex+diseases&amp;author=J.P.+Ioannidis&amp;author=T.A.+Trikalinos&amp;author=M.J.+Khoury&amp;volume=164&amp;publication_year=2006&amp;pages=609-614&amp;pmid=16893921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation">Altshuler D., Daly M. Guilt beyond a reasonable doubt. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:813–815.</span> [<a href="/pubmed/17597768" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Guilt+beyond+a+reasonable+doubt&amp;author=D.+Altshuler&amp;author=M.+Daly&amp;volume=39&amp;publication_year=2007&amp;pages=813-815&amp;pmid=17597768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation">Moonesinghe R., Khoury M.J., Liu T., Ioannidis J.P. Required sample size and nonreplicability thresholds for heterogeneous genetic associations. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2008;<span class="ref-vol">105</span>:617–622.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2206585/">PMC free article</a>]</span> [<a href="/pubmed/18174335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Required+sample+size+and+nonreplicability+thresholds+for+heterogeneous+genetic+associations&amp;author=R.+Moonesinghe&amp;author=M.J.+Khoury&amp;author=T.+Liu&amp;author=J.P.+Ioannidis&amp;volume=105&amp;publication_year=2008&amp;pages=617-622&amp;pmid=18174335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation">Schuknecht H.F., Gacek M.R. Cochlear pathology in presbycusis. <span><span class="ref-journal">Ann. Otol. Rhinol. Laryngol. </span>1993;<span class="ref-vol">102</span>:1–16.</span> [<a href="/pubmed/8420477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Otol.+Rhinol.+Laryngol.&amp;title=Cochlear+pathology+in+presbycusis&amp;author=H.F.+Schuknecht&amp;author=M.R.+Gacek&amp;volume=102&amp;publication_year=1993&amp;pages=1-16&amp;pmid=8420477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2556434&amp;issue-id=172506&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140064683238032title">Abstract</a></li><li><a href="#sec1title">Main Text</a></li><li><a href="#idm140064677869104title">Acknowledgments</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#idm140064680837456title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2556434/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2556434/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2556434/pdf/main.pdf">PDF (495K)</a></li> | <li><a href="#" data-citationid="PMC2556434" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2556434%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2556434%2F&amp;text=Genome-wide%20SNP-Based%20Linkage%20Scan%20Identifies%20a%20Locus%20on%208q24%20for%20an%20Age-Related%20Hearing%20Impairment%20Trait" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2556434%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18760390" data-aiid="2556434" data-aid="2556434" data-iid="172506" data-domainid="203" data-domain="ajhg" data-accid="PMC2556434" data-md5="267c5bbd8b8dd1135b7aea9b35049186"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T10:00:20-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CC479DD3F80F100000000095705CB&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2556434%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2556434/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>