<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="molmed"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2581605/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome wide association for nicotine dependence and smoking cessation success in NIH research volunteers</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2581605/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Molecular Medicine"><meta name="citation_title" content="Genome wide association for nicotine dependence and smoking cessation success in NIH research volunteers"><meta name="citation_authors" content="Tomas Drgon, Ivan Montoya, Catherine Johnson, Qing-Rong Liu, Donna Walther, Dean Hamer, George R Uhl"><meta name="citation_date" content="Jan-Feb 2009"><meta name="citation_issue" content="1-2"><meta name="citation_volume" content="15"><meta name="citation_firstpage" content="21"><meta name="citation_doi" content="10.2119/molmed.2008.00096"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2581605/?report=abstract"><meta name="citation_pmid" content="19009022"><meta name="DC.Title" content="Genome wide association for nicotine dependence and smoking cessation success in NIH research volunteers"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="The Feinstein Institute for Medical Research"><meta name="DC.Contributor" content="Tomas Drgon"><meta name="DC.Contributor" content="Ivan Montoya"><meta name="DC.Contributor" content="Catherine Johnson"><meta name="DC.Contributor" content="Qing-Rong Liu"><meta name="DC.Contributor" content="Donna Walther"><meta name="DC.Contributor" content="Dean Hamer"><meta name="DC.Contributor" content="George R Uhl"><meta name="DC.Date" content="Jan-Feb 2009"><meta name="DC.Identifier" content="10.2119/molmed.2008.00096"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome wide association for nicotine dependence and smoking cessation success in NIH research volunteers"><meta property="og:type" content="article"><meta property="og:description" content="Phenotypes related to both nicotine dependence and ability to successfully quit smoking display substantial heritabilities in classical and molecular genetic studies. Twin studies suggest that some of the genetics for dependence overlaps with genetic ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2581605/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-molmed.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2581605/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2581605/pdf/mol-15-21.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B58D0DD405C710000000004950295.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2581605%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/325/">Mol Med</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/175111/">v.15(1-2); Jan-Feb 2009</a></li><li class="accid">PMC2581605</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-molmed.png" alt="Logo of molmed" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://molmed.biomedcentral.com/" target="pmc_ext" ref="reftype=publisher&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Mol Med</a></span>. 2009 Jan-Feb; 15(1-2): 21–27. </span></div><div><span class="fm-vol-iss-date">Published online 2008 Nov 9. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.2119%2Fmolmed.2008.00096" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.2119/molmed.2008.00096</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2581605</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19009022">19009022</a></div></div></div></div><h1 class="content-title">Genome wide association for nicotine dependence and smoking cessation success in NIH research volunteers</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Drgon%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022" class="affpopup" co-rid="_co_idm140587018539936" co-class="co-affbox">Tomas Drgon</a>,<sup>1</sup> <a href="/pubmed/?term=Montoya%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022" class="affpopup" co-rid="_co_idm140587017833264" co-class="co-affbox">Ivan Montoya</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Johnson%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022" class="affpopup" co-rid="_co_idm140586981478880" co-class="co-affbox">Catherine Johnson</a>,<sup>1</sup> <a href="/pubmed/?term=Liu%20QR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022" class="affpopup" co-rid="_co_idm140586983010176" co-class="co-affbox">Qing-Rong Liu</a>,<sup>1</sup> <a href="/pubmed/?term=Walther%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022" class="affpopup" co-rid="_co_idm140586985384288" co-class="co-affbox">Donna Walther</a>,<sup>1</sup> <a href="/pubmed/?term=Hamer%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022" class="affpopup" co-rid="_co_idm140586985382416" co-class="co-affbox">Dean Hamer</a>,<sup>3</sup> and  <a href="/pubmed/?term=Uhl%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022" class="affpopup" co-rid="_co_idm140586981196144" co-class="co-affbox">George R Uhl</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140587018539936"><h3 class="no_margin">Tomas Drgon</h3><p><sup>1</sup> Molecular Neurobiology Branch, NIH-IRP (NIDA), Suite 3510, 333 Cassell Drive Baltimore, Maryland 21224</p><div>Find articles by <a href="/pubmed/?term=Drgon%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022">Tomas Drgon</a></div></div><div id="_co_idm140587017833264"><h3 class="no_margin">Ivan Montoya</h3><p><sup>1</sup> Molecular Neurobiology Branch, NIH-IRP (NIDA), Suite 3510, 333 Cassell Drive Baltimore, Maryland 21224</p><p><sup>2</sup> Division of Pharmacotherapies and Medical Consequences of Drug Abuse, NIDA, NIH, Rockville MD</p><div>Find articles by <a href="/pubmed/?term=Montoya%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022">Ivan Montoya</a></div></div><div id="_co_idm140586981478880"><h3 class="no_margin">Catherine Johnson</h3><p><sup>1</sup> Molecular Neurobiology Branch, NIH-IRP (NIDA), Suite 3510, 333 Cassell Drive Baltimore, Maryland 21224</p><div>Find articles by <a href="/pubmed/?term=Johnson%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022">Catherine Johnson</a></div></div><div id="_co_idm140586983010176"><h3 class="no_margin">Qing-Rong Liu</h3><p><sup>1</sup> Molecular Neurobiology Branch, NIH-IRP (NIDA), Suite 3510, 333 Cassell Drive Baltimore, Maryland 21224</p><div>Find articles by <a href="/pubmed/?term=Liu%20QR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022">Qing-Rong Liu</a></div></div><div id="_co_idm140586985384288"><h3 class="no_margin">Donna Walther</h3><p><sup>1</sup> Molecular Neurobiology Branch, NIH-IRP (NIDA), Suite 3510, 333 Cassell Drive Baltimore, Maryland 21224</p><div>Find articles by <a href="/pubmed/?term=Walther%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022">Donna Walther</a></div></div><div id="_co_idm140586985382416"><h3 class="no_margin">Dean Hamer</h3><p><sup>3</sup> Section on Gene Structure and Regulation, Laboratory of Biochemistry, NIH IRP (NCI) Bethesda MD 20892</p><div>Find articles by <a href="/pubmed/?term=Hamer%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022">Dean Hamer</a></div></div><div id="_co_idm140586981196144"><h3 class="no_margin">George R Uhl</h3><p><sup>1</sup> Molecular Neurobiology Branch, NIH-IRP (NIDA), Suite 3510, 333 Cassell Drive Baltimore, Maryland 21224</p><div>Find articles by <a href="/pubmed/?term=Uhl%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19009022">George R Uhl</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140586977301952_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140586977301952_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140586977301952_ai" style="display:none"><div class="fm-affl" id="af1-drgon" lang="en"><sup>1</sup> Molecular Neurobiology Branch, NIH-IRP (NIDA), Suite 3510, 333 Cassell Drive Baltimore, Maryland 21224</div><div class="fm-affl" id="af2-drgon" lang="en"><sup>2</sup> Division of Pharmacotherapies and Medical Consequences of Drug Abuse, NIDA, NIH, Rockville MD</div><div class="fm-affl" id="af3-drgon" lang="en"><sup>3</sup> Section on Gene Structure and Regulation, Laboratory of Biochemistry, NIH IRP (NCI) Bethesda MD 20892</div><div id="c1-drgon">*To whom correspondence should be addressed at: Molecular Neurobiology, Box 5180, Baltimore, MD 21224, phone:(443)740-2799, fax:(443)740-2122, email: <a href="mailto:dev@null" data-email="vog.hin.adin.artni@lhug" class="oemail">vog.hin.adin.artni@lhug</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140586977301952_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2581605/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC2581605/bin/08_096_Drgon_Suppl.pdf" data-ga-action="click_feat_suppl">08_096_Drgon_Suppl.pdf</a><span style="color:gray"> (312K)</span></div><div class="small guid">GUID:&nbsp;F79796CB-6E88-4622-A8B7-47A4F2489F00</div></div></dd></dl></div><div id="idm140586986079568" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140586986079568title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Phenotypes related to both nicotine dependence and ability to successfully quit smoking display substantial heritabilities in classical and molecular genetic studies. Twin studies suggest that some of the genetics for dependence overlaps with genetic components of ability to quit, but that much does not. Initial genome wide association (GWA) studies have demonstrated haplotypes that distinguish nicotine-dependent from nondependent smokers. These haplotypes overlap partially with those that distinguish successful quitters from individuals who were not able to quit smoking in clinical trials for smoking cessation. We now report novel genome wide association results from NIH research volunteers who report smoking histories, symptoms of nicotine dependence and ability to successfully quit smoking outside the context of a clinical trial. These results buttress data from several prior GWA studies. They support the idea that previously-reported genetics of smoking cessation success in clinical trial participants may also apply to smokers who are more or less able to initiate and sustain abstinence outside of clinical trial settings.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Smoking Cessation, Nicotine, Nicotine Dependence, Genetic Susceptibility, DNA Microarrays</span></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">Vulnerability to dependence on an addictive substance is a complex trait with strong genetic influences that are well documented by data from family, adoption and twin studies (<a href="#b1-drgon" rid="b1-drgon" class=" bibr popnode">1</a>–<a href="#b4-drgon" rid="b4-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728075">4</a>). Twin studies support the view that much of the heritable influence on vulnerability to dependence on addictive substances from different pharmacological classes (<em>eg</em> nicotine and stimulants) is shared (<a href="#b2-drgon" rid="b2-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728063">2</a>, <a href="#b3-drgon" rid="b3-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728058">3</a>, <a href="#b5-drgon" rid="b5-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728044">5</a>), adding to interest in genes that might predispose to dependence on multiple substances, such as nicotine and other drugs. For smoking, substantial data also document heritabilities of nicotine dependence as defined using criteria from either the DSM (diagnostic and statistical manual) or Fagerstrom test for nicotine dependence (FTND)(<a href="#b6-drgon" rid="b6-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728073">6</a>–<a href="#b9-drgon" rid="b9-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728072">9</a>).</p><p id="__p3">Twin study data also documents that ability to successfully abstain from smoking displays substantial heritability (<a href="#b10-drgon" rid="b10-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728070">10</a>, <a href="#b11-drgon" rid="b11-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728062">11</a>). These data suggest that some, but not most, of these genetic influences are shared with the genetics of developing dependence on nicotine.</p><p id="__p4">Genome wide association (GWA) approaches of increasing sophistication have been developed and used to identify the specific genes and genomic variants that predispose to phenotypes related to smoking and to smoking cessation, as well as to dependence on other addictive substances (<a href="#b12-drgon" rid="b12-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728060">12</a>–<a href="#b17-drgon" rid="b17-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728078">17</a>). Comparisons between data from 1) smokers who are dependent on cigarettes based on FTND scores <em>vs</em> 2) smokers who display little evidence of dependence by Fagerstrom criteria have produced repeated associations of moderate size within a chromosome 15 cluster of nicotinic receptor genes (<a href="#b15-drgon" rid="b15-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728077">15</a>–<a href="#b18-drgon" rid="b18-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728067">18</a>). Partial data from one of these studies that also reported modest associations at a number of additional loci is available for comparative analyses (<a href="#b15-drgon" rid="b15-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728074">15</a>).</p><p id="__p5">We have recently reported genome wide association data for success in quitting smoking in each of three independent samples of carefully-monitored individuals who tried to quit smoking in the context of clinical trials that used placebo, nicotine replacement or bupropion treatments (<a href="#b14-drgon" rid="b14-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728050">14</a>). However, these trials involved highly-selected smokers who were required to fulfill many entrance criteria. Individuals whose data was analyzed were required to participate in detailed and lengthy follow-ups. Such selection criteria might have rendered them unrepresentative of smokers in the general population who seek to quit smoking. Ability to replicate at least many of these results in individuals who quit in community settings would add to our confidence in their generalizability outside of the somewhat artificial settings of a clinical trial.</p><p id="__p6" class="p p-last">Hamer and colleagues have assembled a research volunteer population at and around the NIH clinical center in Bethesda Maryland (<a href="#b19-drgon" rid="b19-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728083">19</a>). These individuals provided data on their histories of smoking and success at quitting smoking. We now report 500k SNP genome wide association studies of these “Hamer et al” samples. We have targeted these analyses to seek overlaps between nominally-positive results from this sample and positive results from other genome wide association results that include: 1) “Bierut et al”: 38,000 SNP genome wide association dataset for heavy smokers who display FTND nicotine dependence <em>vs</em> smokers who not display FTND dependence (<a href="#b15-drgon" rid="b15-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728082">15</a>), 2) “Uhl et al” 500-600,000 SNP genome wide association datasets for smokers who succeeded <em>vs</em> those who did not succeed in at least two of three clinical trials of smoking cessation (<a href="#b14-drgon" rid="b14-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728046">14</a>) and 3) “Liu/Drgon et al” 600,000 - 1M SNP genome wide association datasets for individuals who were dependent on at least one illegal addictive substance in comparison to control individuals with little or no peak lifetime use of any addictive substance, including tobacco (<a href="#b20-drgon" rid="b20-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728045">20</a>). We discuss the significance of the substantial overlaps in the data that we report, as well as the limitations of the samples and datasets. These data buttress previous GWA results for FTND nicotine dependence. They also document, for the first time, overlaps between the molecular genetics of smoking cessation in clinical trial <em>vs</em> smoking cessation in community-based research volunteer samples who quit outside of the context of a clinical trial.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and Methods</h2><div id="__sec3" class="sec sec-first"><p></p><h3 id="__sec3title" class="inline">Samples </h3><p id="__p7" class="p p-first-last">Subjects were recruited at the NIH clinical center for protocols based on cancer risk behaviors and personality as described (<a href="#b19-drgon" rid="b19-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728069">19</a>). Four hundred eighty individuals from the largest self- reported ethnicity in this sample, European-Americans, were divided into: 1) 13 pools (n = 20 in each) of individuals who never smoked extensively or became nicotine dependent, 2) 6 pools containing DNA from 120 individuals who reported lifetime nicotine dependence and current smoking when interviewed and 3) 5 pools containing DNA from 100 individuals who reported having been nicotine dependent at sometime in their lives but who achieved abstinence. As previously reported, these three groups thus described, respectively: 1) smoking less than 101 cigarettes in their lives, 2) starting smoking at age 17 +/− 4, smoking for 18 +/− 13years, consuming 20 +/− 13 cigarettes/day and continuing to smoke when interviewed, and 3) starting smoking at 17 +/− 3 years of age, smoking an average of 20 +/− 13 cigarettes/day for 13 +/− 11 years and subsequently maintaining abstinence for 16 +/− 12 years by the time of interviews (<a href="#b19-drgon" rid="b19-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728064">19</a>).</p></div><div id="__sec4" class="sec sec-last"><p></p><h3 id="__sec4title" class="inline">DNA preparation and assessment of allelic frequencies </h3><p id="__p8" class="p p-first">DNA was prepared from blood or cell lines (<a href="#b21-drgon" rid="b21-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728080">21</a>–<a href="#b23-drgon" rid="b23-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728079">23</a>) and carefully quantitated. DNAs from groups of 20 individuals of the same phenotype were combined. Pooled genotyping 1) reduced costs and 2) allowed us to assess high densities of genotypes in these subjects while providing no threat of loss of genetic confidentiality to these individual research volunteers. Hybridization probes were prepared with precautions to avoid contamination, as described (Affymetrix assays 500k (<a href="#b20-drgon" rid="b20-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728052">20</a>)). One hundred and fifty ng of pooled DNA was digested using <em>Sty</em>I or <em>Nsp</em>I, ligated to appropriate adaptors and amplified using a GeneAmp PCR System 9700 (Applied Biosystems, Foster City, CA) with 3 min 94<sup>o</sup>C, 30 cycles of 30 sec 94<sup>o</sup>C, 45 sec 60<sup>o</sup>C, 15 sec at 68<sup>o</sup>C and a final 7 min 68<sup>o</sup>C extension. PCR products were purified (MinEluteTM 96 UF kits, Qiagen, Valencia, CA) and quantitated. Forty μg of PCR product was digested for 35 min at 37<sup>o</sup>C with 0.04 unit/μl DNase I to produce 30–100 bp fragments which were end-labeled using terminal deoxynucleotidyl transferase and biotinylated dideoxynucleotides and hybridized to the appropriate 500k array (<em>Sty I</em> or <em>Nsp I</em> arrays) (Affymetrix, Santa Clara, CA). Arrays were stained and washed as described (Affymetrix Genechip Mapping Assay Manual) using immunopure strepavidin (Pierce, Milwaukee, WI), biotinylated antistreptavidin antibody (Vector Labs, Burlingame, CA) and R-phycoerythrin strepavidin (Molecular Probes, Eugene, OR). Arrays were scanned and fluorescence intensities quantitated using an Affymetrix array scanner as described (<a href="#b20-drgon" rid="b20-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728057">20</a>). Chromosomal positions for each SNP were sought using NCBI (Build 36.1) and NETAFFYX (AffymetrixSanta Clara, CA) data. Allele fr equencies for each SNP in each DNA pool were assessed based on hybridization intensity signals from four arrays, allowing assessment of hybridization to the 12 “perfect match” cells on each array that are complementary to the PCR products from alleles “A” and “B” for each diallelic SNP on sense and antisense strands. We eliminated: i) SNPs on sex chromosomes, to provide greater power based on inclusion of both males and females and iii) SNPs whose chromosomal positions could not be adequately determined.</p><p id="__p9">Each array was analyzed as described (<a href="#b20-drgon" rid="b20-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728053">20</a>), with background values subtracted, normalization to the highest values noted on the array, averaging of the hybridization intensities from the array cells that corresponded to the perfect match “A” and “B” cells, calculation of “A/B ratios” by dividing average normalized A values by average normalized B values, arctangent transformations to aid combination of data from arrays hybridized and scanned on different days, and determination of the average arctan value for each SNP from the 4 replicate arrays. A “t” statistic for the differences between abusers and controls was generated as described (<a href="#b20-drgon" rid="b20-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728076">20</a>) for each SNP. We focused on SNPs that displayed t statistics with p &lt; 0.05 – p &lt; 0.01 for differences between: 1) nicotine-dependent <em>vs</em> control individuals who never smoked more than 101 cigarettes and 2) nicotine-dependent current smokers <em>vs</em> individuals who were successful in maintaining abstinence. We sought evidence for clustering of these SNPs by focusing on chromosomal regions in which at least three - four of these outlier SNPs, assessed by at least two array types, lay within 25 kb of each other. We term these clustered, nominally-positive SNPs “clustered positive SNPs”, and focus our analyses on regions in which they lie. We utilized different criteria for p values and numbers of SNP/cluster in these different analyses so that the methods would be as nearly identical as possible to those used for previously-reported analyses of the appropriate comparison “Uhl et al, Liu/Drgon et al and Beirut et al” datasets.</p><p id="__p10">To seek additional support for the clusters of nominally- positive SNPs from the current dataset, we sought convergence between data from these clustered nominally-positive SNPs and clustered nominally-positive SNPs, determined in the same way, from: 1) Beirut et al: 38,000 SNP genome wide association studies of light <em>vs</em> heavy smokers (<a href="#b15-drgon" rid="b15-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728065">15</a>), 2) Uhl et al: 500-600,000 SNP GWA studies of individuals who were successful <em>vs</em> unsuccessful in quitting (<a href="#b14-drgon" rid="b14-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728048">14</a>), and Liu/Drgon et al: 600,000 - 1M SNP GWA studies of dependence on illegal addictive substances (<a href="#b20-drgon" rid="b20-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728066">20</a>). To provide insights into some of the genes likely to harbor variants that contribute to individual differences in addiction vulnerability, we sought candidate genes that were identified by overlapping clusters of positive SNPs from each of these samples.</p><p id="__p11">We compare observed results to those expected by chance using 10,000 Monte Carlo simulation trials, as described (<a href="#b20-drgon" rid="b20-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728061">20</a>). For each trial, a randomly-selected set of SNPs from the current dataset was assessed to see if it provided results equal to or greater than the results that we actually observed. The number of trials for which the randomly-selected SNPs displayed (<em>at least</em>) the same features displayed by the observed results was then tallied to generate an empirical p value. These simulations thus correct for the number of repeated comparisons made in these analyses, an important consideration in evaluating these GWA datasets.</p><p id="__p12" class="p p-last">To assess the power of our current approach, we use current sample sizes and standard deviations, the program PS v2.1.31 (<a href="#b24-drgon" rid="b24-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728047">24</a>, <a href="#b25-drgon" rid="b25-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728071">25</a>) and α = 0.05. To provide controls for the possibilities that abuser-control differences observed herein were due to a) occult ethnic/racial allele frequency differences or b) noisy assays, we assessed the overlap between the results obtained here and the SNPs that displayed the largest a) allele frequency differences between African-American <em>vs</em> European-American control individuals and b) the largest assay “noise”.</p></div></div><div id="__sec5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec5title">Results</h2><p id="__p13" class="p p-first">We assessed allele frequencies in multiple pools of DNA from smokers who smoked when assessed, former smokers who quit prior to assessment, and individuals who never smoked significant numbers of cigarettes. There was modest variability among replicate arrays that assessed the same pool. Standard error of the mean (SEM) values were 0.039. For variability among the different pools that assessed the same phenotype, SEM was 0.024. For the smoker <em>vs</em> non-smoker comparison, these samples and these estimates of variance thus provided 0.99, 0.98, 0.87 and 0.54 power to detect allele frequency differences of 12.5%, 10%, 7.5% and 5%, respectively. For the continuing smokers <em>vs</em> successful quitters comparison, the samples provided 0.97, 0.89, 0.66 and 0.33 power to detect differences of the same sizes. The allele frequency differences used in these power calculations are comparable to the actual mean allele frequency differences for the nominally-significant SNPs for smoker <em>vs</em> nonsmoker and continuing smokers <em>vs</em> successful quitters in these studies (4 and 7%, respectively).</p><div id="__sec6" class="sec"><p></p><h3 id="__sec6title" class="inline">“Smoker vs Non-smoker” </h3><p id="__p14" class="p p-first-last">When we compared nicotine-dependent individuals from these samples to nondependent individuals who smoked fewer than 101 cigarettes in their lives, 27,803 SNPs displayed allele frequency differences with nominal p &lt; 0.05. Of these, 3,563 nominally-positive SNPs fell into 665 clusters of at least four SNPs in which each nominally-positive SNP was separated from at least one other nominally positive SNP by ≤ 25 kb. In each of these clusters, we also required that SNPs assayed on both array types display nominally positive results. These results identified 289 genes (<a href="/pmc/articles/PMC2581605/table/t3-drgon/" target="table" class="fig-table-link figpopup" rid-figpopup="t3-drgon" rid-ob="ob-t3-drgon" co-legend-rid=""><span>Table S1</span></a>). None of 10,000 Monte Carlo simulation trials identified clustering that was this significant (thus p &lt; 0.0001). For the 5,570 SNPs that fell into 1,334 clusters of at least 3 SNPs, Monte Carlo p values were &lt; 0.0001.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t3-drgon"><h3>Supplementary Data Table S1</h3><!--caption a7--><div class="caption"><p id="__p31">Dependence Genes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140586977584864" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">GENE</th><th rowspan="1" colspan="1" align="center">description</th><th rowspan="1" colspan="1" align="center">ch</th><th rowspan="1" colspan="1" align="center">bp</th><th rowspan="1" colspan="1" align="center">clustered SNPs</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">A2BP1</td><td rowspan="1" colspan="1" align="left">ataxin 2-binding protein 1</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">6009133</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">ABI1P</td><td rowspan="1" colspan="1" align="left">abl-interactor 1 pseudogene</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">55083079</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ACCN1</td><td rowspan="1" colspan="1" align="left">amiloride-sensitive cation channel 1, neuronal (degenerin)</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">28364218</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">ACSS1</td><td rowspan="1" colspan="1" align="left">acyl-CoA synthetase short-chain family member 1</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">24934873</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ADAM12</td><td rowspan="1" colspan="1" align="left">ADAM metallopeptidase domain 12 (meltrin alpha)</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">127693415</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">ADAMTS18</td><td rowspan="1" colspan="1" align="left">ADAM metallopeptidase with thrombospondin type 1 motif, 18</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">75873929</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">ADAMTS3</td><td rowspan="1" colspan="1" align="left">ADAM metallopeptidase with thrombospondin type 1 motif, 3</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">73365551</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ADD1</td><td rowspan="1" colspan="1" align="left">adducin 1 (alpha)</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">2815382</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ADH1C</td><td rowspan="1" colspan="1" align="left">alcohol dehydrogenase 1C (class I), gamma polypeptide</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">100476672</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">ADHFE1</td><td rowspan="1" colspan="1" align="left">alcohol dehydrogenase, iron containing, 1</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">67507288</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">AKAP13</td><td rowspan="1" colspan="1" align="left">A kinase (PRKA) anchor protein 13</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">83724875</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">AKR1CL2</td><td rowspan="1" colspan="1" align="left">aldo-keto reductase family 1, member C-like 2</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">4858402</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ALDH9A1</td><td rowspan="1" colspan="1" align="left">aldehyde dehydrogenase 9 family, member A1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">163898077</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ALDOAP2</td><td rowspan="1" colspan="1" align="left">aldolase A, fructose-bisphosphate pseudogene 2</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">127345434</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ANK2</td><td rowspan="1" colspan="1" align="left">ankyrin 2, neuronal</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">114190319</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">ANKRD15</td><td rowspan="1" colspan="1" align="left">ankyrin repeat domain 15</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">494703</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">ANKRD38</td><td rowspan="1" colspan="1" align="left">ankyrin repeat domain 38</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">62475409</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">AQP9</td><td rowspan="1" colspan="1" align="left">aquaporin 9</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">56217771</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">ARHGAP26</td><td rowspan="1" colspan="1" align="left">Rho GTPase activating protein 26</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">142130476</td><td rowspan="1" colspan="1" align="center">12</td></tr><tr><td rowspan="1" colspan="1" align="left">ARHGAP28</td><td rowspan="1" colspan="1" align="left">Rho GTPase activating protein 28</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">6824484</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ARNTL2</td><td rowspan="1" colspan="1" align="left">aryl hydrocarbon receptor nuclear translocator-like 2</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">27377255</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">ARSB</td><td rowspan="1" colspan="1" align="left">arylsulfatase B</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">78108788</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ASB13</td><td rowspan="1" colspan="1" align="left">ankyrin repeat and SOCS box-containing 13</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">5720837</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">ATF6</td><td rowspan="1" colspan="1" align="left">activating transcription factor 6</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">160002708</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ATRN</td><td rowspan="1" colspan="1" align="left">attractin</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">3399676</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">ATRNL1</td><td rowspan="1" colspan="1" align="left">attractin-like 1</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">116843114</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">B3GNTL1</td><td rowspan="1" colspan="1" align="left">UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase-like 1</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">78494956</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">BTBD11</td><td rowspan="1" colspan="1" align="left">BTB (POZ) domain containing 11</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">106236327</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">C10orf11</td><td rowspan="1" colspan="1" align="left">chromosome 10 open reading frame 11</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">77212525</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">C11orf74</td><td rowspan="1" colspan="1" align="left">chromosome 11 open reading frame 74</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">36572669</td><td rowspan="1" colspan="1" align="center">8</td></tr><tr><td rowspan="1" colspan="1" align="left">C4orf13</td><td rowspan="1" colspan="1" align="left">chromosome 4 open reading frame 13</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">147395627</td><td rowspan="1" colspan="1" align="center">8</td></tr><tr><td rowspan="1" colspan="1" align="left">C4orf22</td><td rowspan="1" colspan="1" align="left">chromosome 4 open reading frame 22</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">81475898</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">C6orf162</td><td rowspan="1" colspan="1" align="left">chromosome 6 open reading frame 162</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">88103446</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">C6orf71</td><td rowspan="1" colspan="1" align="left">chromosome 6 open reading frame 71</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">150731721</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">C7</td><td rowspan="1" colspan="1" align="left">complement component 7</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">40945356</td><td rowspan="1" colspan="1" align="center">10</td></tr><tr><td rowspan="1" colspan="1" align="left">C8A</td><td rowspan="1" colspan="1" align="left">complement component 8, alpha polypeptide</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">57093065</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CACNB2</td><td rowspan="1" colspan="1" align="left">calcium channel, voltage-dependent, beta 2 subunit</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">18469612</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">CAND1</td><td rowspan="1" colspan="1" align="left">cullin-associated and neddylation-dissociated 1</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">65949416</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">CAPN13</td><td rowspan="1" colspan="1" align="left">calpain 13</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">30799141</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CARD11</td><td rowspan="1" colspan="1" align="left">caspase recruitment domain family, member 11</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">2912308</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CASQ2</td><td rowspan="1" colspan="1" align="left">calsequestrin 2 (cardiac muscle)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">116044166</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CCDC3</td><td rowspan="1" colspan="1" align="left">coiled-coil domain containing 3</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">12978631</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CCM2</td><td rowspan="1" colspan="1" align="left">cerebral cavernous malformation 2</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">45006312</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CCRL2</td><td rowspan="1" colspan="1" align="left">chemokine (C-C motif) receptor-like 2</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">46423725</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CD38</td><td rowspan="1" colspan="1" align="left">CD38 molecule</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">15389029</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CDC25B</td><td rowspan="1" colspan="1" align="left">cell division cycle 25B</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">3724401</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CDC42BPA</td><td rowspan="1" colspan="1" align="left">CDC42 binding protein kinase alpha (DMPK-like)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">225246676</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CDCA2</td><td rowspan="1" colspan="1" align="left">cell division cycle associated 2</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">25372428</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CDH13</td><td rowspan="1" colspan="1" align="left">cadherin 13, H-cadherin (heart)</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">81218079</td><td rowspan="1" colspan="1" align="center">14</td></tr><tr><td rowspan="1" colspan="1" align="left">CDH7</td><td rowspan="1" colspan="1" align="left">cadherin 7, type 2</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">61568468</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CDKN2B</td><td rowspan="1" colspan="1" align="left">cyclin-dependent kinase inhibitor 2B (p15, inhibits CDK4)</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">21992902</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CDYL2</td><td rowspan="1" colspan="1" align="left">chromodomain protein, Y-like 2</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">79195179</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">CEP70</td><td rowspan="1" colspan="1" align="left">centrosomal protein 70kDa</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">139695876</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CHIA</td><td rowspan="1" colspan="1" align="left">chitinase, acidic</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">111635007</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CHST11</td><td rowspan="1" colspan="1" align="left">carbohydrate (chondroitin 4) sulfotransferase 11</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">103370614</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">CLEC12A</td><td rowspan="1" colspan="1" align="left">C-type lectin domain family 12, member A</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">10015281</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CLSTN2</td><td rowspan="1" colspan="1" align="left">calsyntenin 2</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">141136897</td><td rowspan="1" colspan="1" align="center">10</td></tr><tr><td rowspan="1" colspan="1" align="left">CNOT4</td><td rowspan="1" colspan="1" align="left">CCR4-NOT transcription complex, subunit 4</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">134697089</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CNTN6</td><td rowspan="1" colspan="1" align="left">contactin 6</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">1109629</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">CNTNAP2</td><td rowspan="1" colspan="1" align="left">contactin associated protein-like 2</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">145444386</td><td rowspan="1" colspan="1" align="center">21</td></tr><tr><td rowspan="1" colspan="1" align="left">CNTNAP5</td><td rowspan="1" colspan="1" align="left">contactin associated protein-like 5</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">124499334</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">COL14A1</td><td rowspan="1" colspan="1" align="left">collagen, type XIV, alpha 1 (undulin)</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">121206533</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">COL4A2</td><td rowspan="1" colspan="1" align="left">collagen, type IV, alpha 2</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">109757632</td><td rowspan="1" colspan="1" align="center">13</td></tr><tr><td rowspan="1" colspan="1" align="left">COMMD1</td><td rowspan="1" colspan="1" align="left">copper metabolism (Murr1) domain containing 1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">61986307</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CPLX4</td><td rowspan="1" colspan="1" align="left">complexin 4</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">55113634</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CPN2</td><td rowspan="1" colspan="1" align="left">carboxypeptidase N, polypeptide 2, 83kD</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">195542189</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">CPVL</td><td rowspan="1" colspan="1" align="left">carboxypeptidase, vitellogenic-like</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">29001772</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CREB5</td><td rowspan="1" colspan="1" align="left">cAMP responsive element binding protein 5</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">28305465</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">CSMD1</td><td rowspan="1" colspan="1" align="left">CUB and Sushi multiple domains 1</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">2782789</td><td rowspan="1" colspan="1" align="center">14</td></tr><tr><td rowspan="1" colspan="1" align="left">CSMD2</td><td rowspan="1" colspan="1" align="left">CUB and Sushi multiple domains 2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">33752196</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">CSRP3</td><td rowspan="1" colspan="1" align="left">cysteine and glycine-rich protein 3 (cardiac LIM protein)</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">19160154</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CTNNA3</td><td rowspan="1" colspan="1" align="left">catenin (cadherin-associated protein), alpha 3 catenin (cadherin-associated protein),</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">67349937</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CTNND2</td><td rowspan="1" colspan="1" align="left">delta 2 (neural plakophilin-related arm-repeat protein)</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">11024952</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">CUBN</td><td rowspan="1" colspan="1" align="left">cubilin (intrinsic factor-cobalamin receptor)</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">16906534</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">DAAM1</td><td rowspan="1" colspan="1" align="left">dishevelled associated activator of morphogenesis 1</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">58725152</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">DAB1</td><td rowspan="1" colspan="1" align="left">disabled homolog 1 (Drosophila)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">57236167</td><td rowspan="1" colspan="1" align="center">10</td></tr><tr><td rowspan="1" colspan="1" align="left">DAD1</td><td rowspan="1" colspan="1" align="left">defender against cell death 1</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">22103647</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">DCAMKL1</td><td rowspan="1" colspan="1" align="left">doublecortin and CaM kinase-like 1</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">35243478</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">DCTN4</td><td rowspan="1" colspan="1" align="left">dynactin 4 (p62)</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">150068502</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">DDEF1</td><td rowspan="1" colspan="1" align="left">development and differentiation enhancing factor 1</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">131133535</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">DEFA4</td><td rowspan="1" colspan="1" align="left">defensin, alpha 4, corticostatin</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">6780755</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">DEFA6</td><td rowspan="1" colspan="1" align="left">defensin, alpha 6, Paneth cell-specific</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">6769629</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">DEFB127</td><td rowspan="1" colspan="1" align="left">defensin, beta 127</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">86186</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">DERA</td><td rowspan="1" colspan="1" align="left">2-deoxyribose-5-phosphate aldolase homolog (C. elegans)</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">15955545</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">DNAHL1</td><td rowspan="1" colspan="1" align="left">dynein, axonemal, heavy like 1</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">74051432</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">DNAPTP6</td><td rowspan="1" colspan="1" align="left">DNA polymerase-transactivated protein 6</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">200879041</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">DNMT3B</td><td rowspan="1" colspan="1" align="left">DNA (cytosine-5-)-methyltransferase 3 beta</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">30813852</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">DOCK5</td><td rowspan="1" colspan="1" align="left">dedicator of cytokinesis is 5</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">25098204</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">EBF</td><td rowspan="1" colspan="1" align="left">early B-cell factor</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">158058006</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">EMR1</td><td rowspan="1" colspan="1" align="left">egf-like module containing, mucin-like, hormone receptor-like 1</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="right">6838582</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">ENTPD4</td><td rowspan="1" colspan="1" align="left">ectonucleoside triphosphate diphosphohydrolase 4</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">23346075</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">EPHA7</td><td rowspan="1" colspan="1" align="left">EPH receptor A7 excision repair cross-complementing rodent repair deficiency,</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">94007860</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">ERCC4</td><td rowspan="1" colspan="1" align="left">complementation group 4</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">13921524</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">F3</td><td rowspan="1" colspan="1" align="left">coagulation factor III (thromboplastin, tissue factor)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">94767461</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">F5</td><td rowspan="1" colspan="1" align="left">coagulation factor V (proaccelerin, labile factor)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">167750033</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FABP2</td><td rowspan="1" colspan="1" align="left">fatty acid binding protein 2, intestinal</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">120457853</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FAIM</td><td rowspan="1" colspan="1" align="left">Fas apoptotic inhibitory molecule</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">139810232</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FAM120B</td><td rowspan="1" colspan="1" align="left">family with sequence similarity 120B</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">170457769</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FAM19A4</td><td rowspan="1" colspan="1" align="left">family with sequence similarity 19 (chemokine (C-C motif)-like), member A4</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">68863607</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">FAM57A</td><td rowspan="1" colspan="1" align="left">family with sequence similarity 57, member A</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">582597</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FARS2</td><td rowspan="1" colspan="1" align="left">phenylalanine-tRNA synthetase 2 (mitochondrial)</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">5206726</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FBXO31</td><td rowspan="1" colspan="1" align="left">F-box protein 31</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">85920443</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">FHIT</td><td rowspan="1" colspan="1" align="left">fragile histidine triad gene</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">59710076</td><td rowspan="1" colspan="1" align="center">15</td></tr><tr><td rowspan="1" colspan="1" align="left">FHOD3</td><td rowspan="1" colspan="1" align="left">formin homology 2 domain containing 3</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">32131700</td><td rowspan="1" colspan="1" align="center">8</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ10159</td><td rowspan="1" colspan="1" align="left">hypothetical protein FLJ10159</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">107918261</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ16008</td><td rowspan="1" colspan="1" align="left">FLJ16008 protein</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">127657882</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ16124</td><td rowspan="1" colspan="1" align="left">FLJ16124 protein</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">65517349</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ34503</td><td rowspan="1" colspan="1" align="left">hypothetical protein FLJ34503</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">114331995</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ37307</td><td rowspan="1" colspan="1" align="left">hypothetical protein FLJ37307</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">51288888</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ44379</td><td rowspan="1" colspan="1" align="left">similar to S-100 protein, alpha chain</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">63613384</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ45872</td><td rowspan="1" colspan="1" align="left">FLJ45872 protein</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">138890869</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FREM2</td><td rowspan="1" colspan="1" align="left">FRAS1 related extracellular matrix protein 2</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">38159482</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FRMD3</td><td rowspan="1" colspan="1" align="left">FERM domain containing 3</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">85052120</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FRY</td><td rowspan="1" colspan="1" align="left">furry homolog (Drosophila)</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">31503437</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">FUCA2</td><td rowspan="1" colspan="1" align="left">fucosidase, alpha-L- 2, plasma</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">143857935</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GABRA1</td><td rowspan="1" colspan="1" align="left">gamma-aminobutyric acid (GABA) A receptor, alpha 1</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">161206983</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GABRG3</td><td rowspan="1" colspan="1" align="left">gamma-aminobutyric acid (GABA) A receptor, gamma 3</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">25399391</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">GDAP1L1</td><td rowspan="1" colspan="1" align="left">ganglioside-induced differentiation-associated protein 1-like 1</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">42309322</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">GEMIN4</td><td rowspan="1" colspan="1" align="left">gem (nuclear organelle) associated protein 4</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">594608</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">GJB7</td><td rowspan="1" colspan="1" align="left">gap junction protein, beta 7</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">88049417</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GLG1</td><td rowspan="1" colspan="1" align="left">golgi apparatus protein 1</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">73043357</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GOLT1A</td><td rowspan="1" colspan="1" align="left">golgi transport 1 homolog A (S. cerevisiae)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">202433911</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GPC5</td><td rowspan="1" colspan="1" align="left">glypican 5</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">90848930</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">GPR135</td><td rowspan="1" colspan="1" align="left">G protein-coupled receptor 135</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">58965491</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GRIK2</td><td rowspan="1" colspan="1" align="left">glutamate receptor, ionotropic, kainate 2</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">101953675</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GRIK4</td><td rowspan="1" colspan="1" align="left">glutamate receptor, ionotropic, kainate 4</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">120036238</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">GRIN2A</td><td rowspan="1" colspan="1" align="left">glutamate receptor, ionotropic, N-methyl D-aspartate 2A</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">9762923</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GRM3</td><td rowspan="1" colspan="1" align="left">glutamate receptor, metabotropic 3</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">86111166</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">HDAC9</td><td rowspan="1" colspan="1" align="left">histone deacetylase 9</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">18501894</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">HECA</td><td rowspan="1" colspan="1" align="left">headcase homolog (Drosophila)</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">139497942</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">HMGCLL1</td><td rowspan="1" colspan="1" align="left">3-hydroxymethyl-3-methylglutaryl-Coenzyme A lyase-like 1</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">55407132</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">HPCAL1</td><td rowspan="1" colspan="1" align="left">hippocalcin-like 1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">10360491</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">HPSE2</td><td rowspan="1" colspan="1" align="left">heparanase 2</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">100208867</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">HSD17B2</td><td rowspan="1" colspan="1" align="left">hydroxysteroid (17-beta) dehydrogenase 2</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">80626364</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">HTR3B</td><td rowspan="1" colspan="1" align="left">5-hydroxytryptamine (serotonin) receptor 3B</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">113280799</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">IFT52</td><td rowspan="1" colspan="1" align="left">intraflagellar transport 52 homolog (Chlamydomonas)</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">41652993</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">IGSF11</td><td rowspan="1" colspan="1" align="left">immunoglobulin superfamily, member 11</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">120102167</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">IL18RAP</td><td rowspan="1" colspan="1" align="left">interleukin 18 receptor accessory protein</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">102401686</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">IL1F5</td><td rowspan="1" colspan="1" align="left">interleukin 1 family, member 5 (delta)</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">113532686</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">IL1F8</td><td rowspan="1" colspan="1" align="left">interleukin 1 family, member 8 (eta)</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">113496139</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">IL1RAP</td><td rowspan="1" colspan="1" align="left">interleukin 1 receptor accessory protein</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">191714585</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">IMMP2L</td><td rowspan="1" colspan="1" align="left">IMP2 inner mitochondrial membrane peptidase-like (S. cerevisiae)</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">110090346</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">IMPA2</td><td rowspan="1" colspan="1" align="left">inositol(myo)-1(or 4)-monophosphatase 2</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">11971455</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">JMJD2C</td><td rowspan="1" colspan="1" align="left">jumonji domain containing 2C</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">6748083</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">KC6</td><td rowspan="1" colspan="1" align="left">keratoconus gene 6</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">37314233</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">KCNIP4</td><td rowspan="1" colspan="1" align="left">Kv channel interacting protein 4</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">20339337</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">KCNJ6</td><td rowspan="1" colspan="1" align="left">potassium inwardly-rectifying channel, subfamily J, member 6</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="right">37918655</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">KCNK10</td><td rowspan="1" colspan="1" align="left">potassium channel, subfamily K, member 10 potassium large conductance calcium-activated channel,</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">87720993</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">KCNMA1</td><td rowspan="1" colspan="1" align="left">subfamily M, alpha member 1</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">78299366</td><td rowspan="1" colspan="1" align="center">11</td></tr><tr><td rowspan="1" colspan="1" align="left">KCNQ5</td><td rowspan="1" colspan="1" align="left">potassium voltage-gated channel, KQT-like subfamily, member 5</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">73388556</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">KCTD10</td><td rowspan="1" colspan="1" align="left">potassium channel tetramerisation domain containing 10</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">108370843</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">KCTD9</td><td rowspan="1" colspan="1" align="left">potassium channel tetramerisation domain containing 9</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">25341280</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">KIAA1576</td><td rowspan="1" colspan="1" align="left">KIAA1576 protein</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">76379984</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">KISS1</td><td rowspan="1" colspan="1" align="left">KiSS-1 metastasis-suppressor</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">202426092</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">KLHL25</td><td rowspan="1" colspan="1" align="left">kelch-like 25 (Drosophila)</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">84103561</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">LAMC1</td><td rowspan="1" colspan="1" align="left">laminin, gamma 1 (formerly LAMB2)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">181259176</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">LGI2</td><td rowspan="1" colspan="1" align="left">leucine-rich repeat LGI family, member 2</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">24612488</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">LIN7A</td><td rowspan="1" colspan="1" align="left">lin-7 homolog A (C. elegans)</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">79715302</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">LPA</td><td rowspan="1" colspan="1" align="left">lipoprotein, Lp(a)</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">160872505</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">LPAL2</td><td rowspan="1" colspan="1" align="left">lipoprotein, Lp(a)-like 2</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">160807831</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">LPP</td><td rowspan="1" colspan="1" align="left">LIM domain containing preferred translocation in partner lipoma</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">189413415</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">LRP1B</td><td rowspan="1" colspan="1" align="left">low density lipoprotein-related protein 1B (deleted in tumors)</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">140705466</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">LRRC15</td><td rowspan="1" colspan="1" align="left">leucine rich repeat containing 15</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">195557273</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">LRRC4C</td><td rowspan="1" colspan="1" align="left">leucine rich repeat containing 4C</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">40092329</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">LRRC8D</td><td rowspan="1" colspan="1" align="left">leucine rich repeat containing 8 family, member D</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">90060101</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">LRRN3</td><td rowspan="1" colspan="1" align="left">leucine rich repeat neuronal 3</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">110518419</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">LRRN6C</td><td rowspan="1" colspan="1" align="left">leucine rich repeat neuronal 6C</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">27938528</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">LSM6</td><td rowspan="1" colspan="1" align="left">LSM6 homolog, U6 small nuclear RNA associated (S. cerevisiae)</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">147316343</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">MAML3</td><td rowspan="1" colspan="1" align="left">mastermind-like 3 (Drosophila)</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">140856995</td><td rowspan="1" colspan="1" align="center">10</td></tr><tr><td rowspan="1" colspan="1" align="left">MAN1C1</td><td rowspan="1" colspan="1" align="left">mannosidase, alpha, class 1C, member 1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">25816546</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">MAP2</td><td rowspan="1" colspan="1" align="left">microtubule-associated protein 2</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">209997016</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">MAP3K7IP2</td><td rowspan="1" colspan="1" align="left">mitogen-activated protein kinase kinase kinase 7 interacting protein 2</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">149680756</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">MAPK4</td><td rowspan="1" colspan="1" align="left">mitogen-activated protein kinase 4</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">46340482</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">MGST3</td><td rowspan="1" colspan="1" align="left">microsomal glutathione S-transferase 3</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">163867074</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">MIRN101-2</td><td rowspan="1" colspan="1" align="left">microRNA 101-2</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">4840296</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">MOV10L1</td><td rowspan="1" colspan="1" align="left">Mov10l1, Moloney leukemia virus 10-like 1, homolog (mouse)</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="right">48870621</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">MPHOSPH6</td><td rowspan="1" colspan="1" align="left">M-phase phosphoprotein 6</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">80739272</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">MRPS33P2</td><td rowspan="1" colspan="1" align="left">mitochondrial ribosomal protein S33 pseudogene 2 membrane-spanning 4-domains, subfamily A, member 3</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">37684800</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">MS4A3</td><td rowspan="1" colspan="1" align="left">(hematopoietic cell-specific)</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">59580677</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">MTMR7</td><td rowspan="1" colspan="1" align="left">myotubularin related protein 7</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">17199910</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">MYBL2</td><td rowspan="1" colspan="1" align="left">v-myb myeloblastosis viral oncogene homolog (avian)-like 2</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">41729123</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">MYLK</td><td rowspan="1" colspan="1" align="left">myosin, light polypeptide kinase</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">124813833</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">MYO1G</td><td rowspan="1" colspan="1" align="left">myosin IG</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">44968786</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">MYO5A</td><td rowspan="1" colspan="1" align="left">myosin VA (heavy polypeptide 12, myoxin)</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">50392602</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">NAV2</td><td rowspan="1" colspan="1" align="left">neuron navigator 2</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">19691488</td><td rowspan="1" colspan="1" align="center">8</td></tr><tr><td rowspan="1" colspan="1" align="left">NELL1</td><td rowspan="1" colspan="1" align="left">NEL-like 1 (chicken)</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">20647712</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">NKAIN2</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">124166768</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">NLGN1</td><td rowspan="1" colspan="1" align="left">neuroligin 1</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">174598938</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">NME7</td><td rowspan="1" colspan="1" align="left">non-metastatic cells 7, protein expressed in (nucleoside-diphosphate kinase)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">167368393</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">OR51A10P</td><td rowspan="1" colspan="1" align="left">olfactory receptor, family 51, subfamily A, member 10 pseudogene</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">5446310</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">OR51I2</td><td rowspan="1" colspan="1" align="left">olfactory receptor, family 51, subfamily I, member 2</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">5431295</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">OR51N1P</td><td rowspan="1" colspan="1" align="left">olfactory receptor, family 51, subfamily N, member 1 pseudogene</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">4764560</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">OR52Y1P</td><td rowspan="1" colspan="1" align="left">olfactory receptor, family 52, subfamily Y, member 1 pseudogene</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">4771575</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">OR9Q1</td><td rowspan="1" colspan="1" align="left">olfactory receptor, family 9, subfamily Q, member 1</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">57547929</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PALLD</td><td rowspan="1" colspan="1" align="left">palladin, cytoskeletal associated protein</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">169654792</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">PAPPA2</td><td rowspan="1" colspan="1" align="left">pappalysin 2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">174698930</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">PARD3B</td><td rowspan="1" colspan="1" align="left">par-3 partitioning defective 3 homolog B</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">205118761</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PBX1</td><td rowspan="1" colspan="1" align="left">pre-B-cell leukemia transcription factor 1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">162795561</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PDE1C</td><td rowspan="1" colspan="1" align="left">phosphodiesterase 1C, calmodulin-dependent 70kDa</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">31795772</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">PDE5A</td><td rowspan="1" colspan="1" align="left">phosphodiesterase 5A, cGMP-specific</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">120634998</td><td rowspan="1" colspan="1" align="center">14</td></tr><tr><td rowspan="1" colspan="1" align="left">PDE8B</td><td rowspan="1" colspan="1" align="left">phosphodiesterase 8B</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">76542462</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PDE9A</td><td rowspan="1" colspan="1" align="left">phosphodiesterase 9A</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="right">42946931</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">PEBP4</td><td rowspan="1" colspan="1" align="left">phosphatidylethanolamine-binding protein 4</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">22626713</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PEPD</td><td rowspan="1" colspan="1" align="left">peptidase D</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="right">38569699</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">PEX3</td><td rowspan="1" colspan="1" align="left">peroxisomal biogenesis factor 3</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">143813810</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PIG38</td><td rowspan="1" colspan="1" align="left">proliferation-inducing protein 38</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">42685704</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">PINX1</td><td rowspan="1" colspan="1" align="left">PIN2-interacting protein 1</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">10659883</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">PKP4</td><td rowspan="1" colspan="1" align="left">plakophilin 4</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">159021722</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">PLB1</td><td rowspan="1" colspan="1" align="left">phospholipase B1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">28572486</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PLCB1</td><td rowspan="1" colspan="1" align="left">phospholipase C, beta 1 (phosphoinositide-specific)</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">8061296</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PLEKHA6</td><td rowspan="1" colspan="1" align="left">pleckstrin homology domain containing, family A member 6</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">202454604</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">PLXNC1</td><td rowspan="1" colspan="1" align="left">plexin C1</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">93066630</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PNOC</td><td rowspan="1" colspan="1" align="left">prepronociceptin</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">28230568</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">POLE</td><td rowspan="1" colspan="1" align="left">polymerase (DNA directed), epsilon</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">131710421</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">POLR1A</td><td rowspan="1" colspan="1" align="left">polymerase (RNA) I polypeptide A, 194kDa</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">86107891</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PPAPDC1A</td><td rowspan="1" colspan="1" align="left">phosphatidic acid phosphatase type 2 domain containing 1A</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">122206456</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PPARGC1B</td><td rowspan="1" colspan="1" align="left">peroxisome proliferative activated receptor, gamma, coactivator 1, beta</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">149090057</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">PPFIBP1</td><td rowspan="1" colspan="1" align="left">PTPRF interacting protein, binding protein 1 (liprin beta 1)</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">27568312</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PREX1</td><td rowspan="1" colspan="1" align="left">phosphatidylinositol 3,4,5-trisphosphate-dependent RAC exchanger 1</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">46674200</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">PTK2B</td><td rowspan="1" colspan="1" align="left">PTK2B protein tyrosine kinase 2 beta</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">27224916</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">PTN</td><td rowspan="1" colspan="1" align="left">pleiotrophin (heparin binding growth factor 8, neurite growth-promoting factor 1)</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">136562628</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PTPN14</td><td rowspan="1" colspan="1" align="left">protein tyrosine phosphatase, non-receptor type 14</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">212597634</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PTPRG</td><td rowspan="1" colspan="1" align="left">protein tyrosine phosphatase, receptor type, G</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">61522285</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PTPRM</td><td rowspan="1" colspan="1" align="left">protein tyrosine phosphatase, receptor type, M</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">7557817</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">PTPRN2</td><td rowspan="1" colspan="1" align="left">protein tyrosine phosphatase, receptor type, N polypeptide 2</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">157024516</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PZP</td><td rowspan="1" colspan="1" align="left">pregnancy-zone protein</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">9192703</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">RAD51L1</td><td rowspan="1" colspan="1" align="left">RAD51-like 1 (S. cerevisiae)</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">67356262</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">RAG2</td><td rowspan="1" colspan="1" align="left">recombination activating gene 2</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">36570069</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">RCL1</td><td rowspan="1" colspan="1" align="left">RNA terminal phosphate cyclase-like 1</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">4782937</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">RFXDC1</td><td rowspan="1" colspan="1" align="left">regulatory factor X domain containing 1</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">117305069</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">RGS7</td><td rowspan="1" colspan="1" align="left">regulator of G-protein signalling 7</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">239005494</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">RIN2</td><td rowspan="1" colspan="1" align="left">Ras and Rab interactor 2</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">19818210</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">RNF111</td><td rowspan="1" colspan="1" align="left">ring finger protein 111</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">57067157</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ROBO2</td><td rowspan="1" colspan="1" align="left">roundabout, axon guidance receptor, homolog 2 (Drosophila)</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">77172621</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">RORA</td><td rowspan="1" colspan="1" align="left">RAR-related orphan receptor A</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">58576755</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">RPS7P1</td><td rowspan="1" colspan="1" align="left">ribosomal protein S7 pseudogene 1</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">23818967</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">RUNX2</td><td rowspan="1" colspan="1" align="left">runt-related transcription factor 2</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">45404032</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">RYR3</td><td rowspan="1" colspan="1" align="left">ryanodine receptor 3</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">31390469</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">S100A10</td><td rowspan="1" colspan="1" align="left">S100 calcium binding protein A10 (annexin II ligand, calpactin I, light polypeptide (p11))</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">150222018</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">SART3</td><td rowspan="1" colspan="1" align="left">squamous cell carcinoma antigen recognised by T cells 3</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">107440487</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SCEL</td><td rowspan="1" colspan="1" align="left">sciellin</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">77007860</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SCHIP1</td><td rowspan="1" colspan="1" align="left">schwannomin interacting protein 1</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">160474238</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">SCYL1BP1</td><td rowspan="1" colspan="1" align="left">SCY1-like 1 binding protein 1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">168767894</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SDHB</td><td rowspan="1" colspan="1" align="left">succinate dehydrogenase complex, subunit B, iron sulfur (Ip)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">17217840</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SELE</td><td rowspan="1" colspan="1" align="left">selectin E (endothelial adhesion molecule 1)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">167958406</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">SERPINA12</td><td rowspan="1" colspan="1" align="left">serpin peptidase inhibitor, clade A (alpha-1 antiproteinase, antitrypsin), member 12</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">94023373</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">SGCZ</td><td rowspan="1" colspan="1" align="left">sarcoglycan zeta</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">13991744</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">SGK2</td><td rowspan="1" colspan="1" align="left">serum/glucocorticoid regulated kinase 2</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">41621100</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SH3GL2</td><td rowspan="1" colspan="1" align="left">SH3-domain GRB2-like 2</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">17569194</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">SLC11A2</td><td rowspan="1" colspan="1" align="left">solute carrier family 11 (proton-coupled divalent metal ion transporters), member 2</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">49666044</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">SLC13A2</td><td rowspan="1" colspan="1" align="left">solute carrier family 13 (sodium-dependent dicarboxylate transporter), member 2</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">23824791</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SLC24A2</td><td rowspan="1" colspan="1" align="left">solute carrier family 24 (sodium/potassium/calcium exchanger), member 2</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">19505978</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">SLC2A9</td><td rowspan="1" colspan="1" align="left">solute carrier family 2 (facilitated glucose transporter), member 9</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">9436946</td><td rowspan="1" colspan="1" align="center">15</td></tr><tr><td rowspan="1" colspan="1" align="left">SLC39A6</td><td rowspan="1" colspan="1" align="left">solute carrier family 39 (zinc transporter), member 6</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">31943197</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">SLC9A8</td><td rowspan="1" colspan="1" align="left">solute carrier family 9 (sodium/hydrogen exchanger), member 8</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="right">47862657</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">SLIT3</td><td rowspan="1" colspan="1" align="left">slit homolog 3 (Drosophila)</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">168025857</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">SNORA9</td><td rowspan="1" colspan="1" align="left">small nucleolar RNA, H/ACA box 9</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">44991502</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SOCS5</td><td rowspan="1" colspan="1" align="left">suppressor of cytokine signaling 5</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">46779603</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">SORCS2</td><td rowspan="1" colspan="1" align="left">sortilin-related VPS10 domain containing receptor 2</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">7245373</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">SPOCK1</td><td rowspan="1" colspan="1" align="left">sparc/osteonectin, cwcv and kazal-like domains proteoglycan (testican) 1</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">136338886</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">SPTB</td><td rowspan="1" colspan="1" align="left">spectrin, beta, erythrocytic (includes spherocytosis, clinical type I)</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">64282754</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">SPTBN1</td><td rowspan="1" colspan="1" align="left">spectrin, beta, non-erythrocytic 1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">54601684</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">SSBP3</td><td rowspan="1" colspan="1" align="left">single stranded DNA binding protein 3 ST6 (alpha-N-acetyl-neuraminyl-2,3-beta-galactosyl-1,3)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">54464783</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ST6GALNAC3</td><td rowspan="1" colspan="1" align="left">-N-acetylgalactosaminide alpha-2,6-sialyltransferase 3</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">76312992</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">STARD13</td><td rowspan="1" colspan="1" align="left">START domain containing 13</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">32575307</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">STS-1</td><td rowspan="1" colspan="1" align="left">Cbl-interacting protein Sts-1</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">122031640</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">SV2C</td><td rowspan="1" colspan="1" align="left">synaptic vesicle glycoprotein 2C</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">75415061</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">TBC1D1</td><td rowspan="1" colspan="1" align="left">TBC1 (tre-2/USP6, BUB2, cdc16) domain family, member 1</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">37569115</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">TDRD9</td><td rowspan="1" colspan="1" align="left">tudor domain containing 9</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">103506687</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">THSD4</td><td rowspan="1" colspan="1" align="left">thrombospondin, type I, domain containing 4</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">69220842</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">TM9SF3</td><td rowspan="1" colspan="1" align="left">transmembrane 9 superfamily member 3</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">98270420</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">TMEM132C</td><td rowspan="1" colspan="1" align="left">transmembrane protein 132C</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">127318855</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">TMEM23</td><td rowspan="1" colspan="1" align="left">transmembrane protein 23</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">51735351</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">TRB@</td><td rowspan="1" colspan="1" align="left">T cell receptor beta locus</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">141645314</td><td rowspan="1" colspan="1" align="center">11</td></tr><tr><td rowspan="1" colspan="1" align="left">TRBV20-1</td><td rowspan="1" colspan="1" align="left">T cell receptor beta variable 20-1</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">142015233</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">TRBV21-1</td><td rowspan="1" colspan="1" align="left">T cell receptor beta variable 21-1</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">142025416</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">TRBV22</td><td rowspan="1" colspan="1" align="left">T cell receptor beta variable 22</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">142030237</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">TRPM8</td><td rowspan="1" colspan="1" align="left">transient receptor potential cation channel, subfamily M, member 8</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">234490782</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">TSPAN14</td><td rowspan="1" colspan="1" align="left">tetraspanin 14</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">82204047</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">TSPAN9</td><td rowspan="1" colspan="1" align="left">tetraspanin 9</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">3056818</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">UCK2</td><td rowspan="1" colspan="1" align="left">uridine-cytidine kinase 2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">164063514</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">UNC5C</td><td rowspan="1" colspan="1" align="left">unc-5 homolog C (C. elegans)</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">96308712</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">VANGL1</td><td rowspan="1" colspan="1" align="left">vang-like 1 (van gogh, Drosophila)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">115986097</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">VAV3</td><td rowspan="1" colspan="1" align="left">vav 3 oncogene</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">107915310</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">WDR27</td><td rowspan="1" colspan="1" align="left">WD repeat domain 27</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">169599232</td><td rowspan="1" colspan="1" align="center">8</td></tr><tr><td rowspan="1" colspan="1" align="left">WISP1</td><td rowspan="1" colspan="1" align="left">WNT1 inducible signaling pathway protein 1</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">134272494</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ZBTB16</td><td rowspan="1" colspan="1" align="left">zinc finger and BTB domain containing 16</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">113435641</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">ZNF248</td><td rowspan="1" colspan="1" align="left">zinc finger protein 248</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">38157905</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">ZNF609</td><td rowspan="1" colspan="1" align="left">zinc finger protein 609</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">62578672</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ZNF96</td><td rowspan="1" colspan="1" align="left">zinc finger protein 96</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">28454973</td><td rowspan="1" colspan="1" align="center">4</td></tr></tbody></table></div><div id="largeobj_idm140586977584864" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2581605/table/t3-drgon/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec7" class="sec"><p></p><h3 id="__sec7title" class="inline">Current smokers vs Quitters </h3><p id="__p15" class="p p-first-last">There was also significant clustering of nominally-positive SNPs in comparisons of current smokers <em>vs</em> those who had successfully quit. There were 5,408 SNPs that displayed allele frequencies differences with nominal p &lt; 0.01. Nine hundred fourteen of these nominally-positive SNPs fell into 239 clusters of at least three nominally-positive SNPs separated from each other by &lt; 100 kb. In each of these clusters, we again required that SNPs assayed on both array types display nominally positive results. These results identified 67 genes (<a href="/pmc/articles/PMC2581605/table/t4-drgon/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-drgon" rid-ob="ob-t4-drgon" co-legend-rid=""><span>Table S2</span></a>). Monte Carlo p values for this degree of clustering were p &lt; 0.0001.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t4-drgon"><h3>Supplementary Data Table S2</h3><!--caption a7--><div class="caption"><p id="__p32">Abstinence Genes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140586974417824" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">GENE</th><th rowspan="1" colspan="1" align="center">description</th><th rowspan="1" colspan="1" align="center">ch</th><th rowspan="1" colspan="1" align="center">bp</th><th rowspan="1" colspan="1" align="center">clustered SNPs</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">A2BP1</td><td rowspan="1" colspan="1" align="left">ataxin 2-binding protein 1</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">6009133</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">ACADSB</td><td rowspan="1" colspan="1" align="left">acyl-Coenzyme A dehydrogenase, short/branched chain 1-acylglycerol-3-phosphate O-acyltransferase 5</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">124758524</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">AGPAT5</td><td rowspan="1" colspan="1" align="left">(lysophosphatidic acid acyltransferase, epsilon)</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">6553286</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">ANKFY1</td><td rowspan="1" colspan="1" align="left">ankyrin repeat and FYVE domain containing 1</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">4013950</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">ATP10B</td><td rowspan="1" colspan="1" align="left">ATPase, Class V, type 10B</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">159922705</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">C8orf13</td><td rowspan="1" colspan="1" align="left">chromosome 8 open reading frame 13</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">11316382</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CACHD1</td><td rowspan="1" colspan="1" align="left">cache domain containing 1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">64744186</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CALB2</td><td rowspan="1" colspan="1" align="left">calbindin 2, 29kDa (calretinin)</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">69950127</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CDH6</td><td rowspan="1" colspan="1" align="left">cadherin 6, type 2, K-cadherin (fetal kidney)</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">31229553</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CLEC2D</td><td rowspan="1" colspan="1" align="left">C-type lectin domain family 2, member D</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">9713576</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CSMD1</td><td rowspan="1" colspan="1" align="left">CUB and Sushi multiple domains 1</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">2782789</td><td rowspan="1" colspan="1" align="center">10</td></tr><tr><td rowspan="1" colspan="1" align="left">CSMD3</td><td rowspan="1" colspan="1" align="left">CUB and Sushi multiple domains 3</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">113304333</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CTNNA3</td><td rowspan="1" colspan="1" align="left">catenin (cadherin-associated protein), alpha 3</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">67349937</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">DKFZp761B107</td><td rowspan="1" colspan="1" align="left">hypothetical protein DKFZp761B107</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">24437679</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">DPP10</td><td rowspan="1" colspan="1" align="left">dipeptidyl-peptidase 10</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">115636202</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">DSCAM</td><td rowspan="1" colspan="1" align="left">Down syndrome cell adhesion molecule</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="right">40306213</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">ELL</td><td rowspan="1" colspan="1" align="left">elongation factor RNA polymerase II</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="right">18414473</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">EXT1</td><td rowspan="1" colspan="1" align="left">exostoses (multiple) 1</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">118880783</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">FARS2</td><td rowspan="1" colspan="1" align="left">phenylalanine-tRNA synthetase 2 (mitochondrial)</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">5206726</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">FHIT</td><td rowspan="1" colspan="1" align="left">fragile histidine triad gene</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">59710076</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ21986</td><td rowspan="1" colspan="1" align="left">hypothetical protein FLJ21986</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">120416032</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ32110</td><td rowspan="1" colspan="1" align="left">hypothetical protein FLJ32110</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">88226689</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">GPR176</td><td rowspan="1" colspan="1" align="left">G protein-coupled receptor 176</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">37880223</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">GPR39</td><td rowspan="1" colspan="1" align="left">G protein-coupled receptor 39</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">132891086</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">GRM7</td><td rowspan="1" colspan="1" align="left">glutamate receptor, metabotropic 7</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">6877927</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">IGL@</td><td rowspan="1" colspan="1" align="left">immunoglobulin lambda locus</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="right">20710659</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">INADL</td><td rowspan="1" colspan="1" align="left">InaD-like (Drosophila)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">61980737</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">KALRN</td><td rowspan="1" colspan="1" align="left">kalirin, RhoGEF kinase</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">125296275</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">KIAA1600</td><td rowspan="1" colspan="1" align="left">KIAA1600</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">116571773</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">KIF1B</td><td rowspan="1" colspan="1" align="left">kinesin family member 1B</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">10193418</td><td rowspan="1" colspan="1" align="center">8</td></tr><tr><td rowspan="1" colspan="1" align="left">KSR2</td><td rowspan="1" colspan="1" align="left">kinase suppressor of ras 2</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">116389387</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">LAPTM4B</td><td rowspan="1" colspan="1" align="left">lysosomal associated protein transmembrane 4 beta</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">98856985</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">LPP</td><td rowspan="1" colspan="1" align="left">LIM domain containing preferred translocation partner in lipoma</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">189413415</td><td rowspan="1" colspan="1" align="center">9</td></tr><tr><td rowspan="1" colspan="1" align="left">LRRN6C</td><td rowspan="1" colspan="1" align="left">leucine rich repeat neuronal 6C</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="right">27938528</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">LRRTM3</td><td rowspan="1" colspan="1" align="left">leucine rich repeat transmembrane neuronal 3</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">68355819</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">MAML2</td><td rowspan="1" colspan="1" align="left">mastermind-like 2 (Drosophila)</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">95351088</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">MGC34646</td><td rowspan="1" colspan="1" align="left">hypothetical protein MGC34646</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">62363104</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">MUC19</td><td rowspan="1" colspan="1" align="left">mucin 19, oligomeric</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">39073517</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">MYO5A</td><td rowspan="1" colspan="1" align="left">myosin VA (heavy polypeptide 12, myoxin)</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">50392602</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">NCAM1</td><td rowspan="1" colspan="1" align="left">neural cell adhesion molecule 1</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">112337368</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">NDST3</td><td rowspan="1" colspan="1" align="left">N-deacetylase/N-sulfotransferase (heparan glucosaminyl) 3</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">119174948</td><td rowspan="1" colspan="1" align="center">8</td></tr><tr><td rowspan="1" colspan="1" align="left">NRXN1</td><td rowspan="1" colspan="1" align="left">neurexin 1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">50000992</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">OR2D3</td><td rowspan="1" colspan="1" align="left">olfactory receptor, family 2, subfamily D, member 3</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">6898809</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">OR8B2</td><td rowspan="1" colspan="1" align="left">olfactory receptor, family 8, subfamily B, member 2</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">123757508</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">OR8C1P</td><td rowspan="1" colspan="1" align="left">olfactory receptor, family 8, subfamily C, member 1 pseudogene</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">123752047</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">PAN3</td><td rowspan="1" colspan="1" align="left">PAN3 polyA specific ribonuclease subunit homolog (S. cerevisiae)</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">27646417</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">PCDH15</td><td rowspan="1" colspan="1" align="left">protocadherin 15</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">55250866</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">PHACTR1</td><td rowspan="1" colspan="1" align="left">phosphatase and actin regulator 1</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">12825819</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">PIGN</td><td rowspan="1" colspan="1" align="left">phosphatidylinositol glycan anchor biosynthesis, class N</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">57862437</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">PLA2G4A</td><td rowspan="1" colspan="1" align="left">phospholipase A2, group IVA (cytosolic, calcium-dependent)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">185064728</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">PTGER3</td><td rowspan="1" colspan="1" align="left">prostaglandin E receptor 3 (subtype EP3)</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">71090624</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">RARB</td><td rowspan="1" colspan="1" align="left">retinoic acid receptor, beta</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">25444758</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">RARS</td><td rowspan="1" colspan="1" align="left">arginyl-tRNA synthetase</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">167846041</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">RBM35A</td><td rowspan="1" colspan="1" align="left">RNA binding motif protein 35A</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">95722576</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">ROR1</td><td rowspan="1" colspan="1" align="left">receptor tyrosine kinase-like orphan receptor 1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">64012302</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">RYR3</td><td rowspan="1" colspan="1" align="left">ryanodine receptor 3</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">31390469</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SLC14A2</td><td rowspan="1" colspan="1" align="left">solute carrier family 14 (urea transporter), member 2</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">41448764</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">SLC2A12</td><td rowspan="1" colspan="1" align="left">solute carrier family 2 (facilitated glucose transporter), member 12</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">134350412</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SLIT3</td><td rowspan="1" colspan="1" align="left">slit homolog 3 (Drosophila)</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">168025857</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SUHW2</td><td rowspan="1" colspan="1" align="left">suppressor of hairy wing homolog 2 (Drosophila)</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="right">21168767</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">TAS2R44</td><td rowspan="1" colspan="1" align="left">taste receptor, type 2, member 44</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">11074274</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">TAS2R48</td><td rowspan="1" colspan="1" align="left">taste receptor, type 2, member 48</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">11065538</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">TECTA</td><td rowspan="1" colspan="1" align="left">tectorin alpha</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="right">120478585</td><td rowspan="1" colspan="1" align="center">7</td></tr><tr><td rowspan="1" colspan="1" align="left">UBE2G1</td><td rowspan="1" colspan="1" align="left">ubiquitin-conjugating enzyme E2G 1 (UBC7 homolog, yeast)</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">4119261</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">WWOX</td><td rowspan="1" colspan="1" align="left">WW domain containing oxidoreductase</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">76691052</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">XKR4</td><td rowspan="1" colspan="1" align="left">XK, Kell blood group complex subunit-related family, member 4</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">56177571</td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">ZZEF1</td><td rowspan="1" colspan="1" align="left">zinc finger, ZZ-type with EF-hand domain 1</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">3854488</td><td rowspan="1" colspan="1" align="center">3</td></tr></tbody></table></div><div id="largeobj_idm140586974417824" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2581605/table/t4-drgon/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec8" class="sec"><p></p><h3 id="__sec8title" class="inline">Overlap with data from Beirut et al and Liu/Drgon et al </h3><p id="__p16" class="p p-first-last">These data for clustered, nominally positive SNPs from the current dataset provide significant overlap with genes identified by other relevant datasets. The clusters of four nominally-positive SNPs identified by comparisons between dependent and nondependent subjects provide highly significant overlap with the subset of 38,000 SNPs that were identified as nominally-significant by Beirut and colleagues in comparisons of dependent smokers to nondependent smokers (Monte Carlo p &lt; 0.0001). Overlaps with genes identified in 600,000 – 1M GWA for dependence on at least one illegal substance <em>vs</em> ethnically-matched control individuals provided p values reached statistical significance (p = 0.047). These overlaps identify 30 genes (<a href="/pmc/articles/PMC2581605/table/t1-drgon/" target="table" class="fig-table-link figpopup" rid-figpopup="t1-drgon" rid-ob="ob-t1-drgon" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t1-drgon"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p29">Nicotine dependence genes. Genes that contain overlapping clusters of nominally positive SNPs in comparisons of nicotine dependent <em>vs</em> individuals who report use of &lt; 100 cigarettes in their lives (Hamer samples) as well as Liu/Drgon et al and/or Beirut et al comparions. Columns list gene description, chromosome, basepairs for gene’s start, number of nominally-positive SNPs that lay in clusters within the gene’s exons and in 10 kb genomic flanking regions, and the numbers of clustered, nominally-positive SNPs that fall within overlapping clusters within the same gene from Liu/Drgon et al 600k (MNB I) and 1M (MNB II) data from NIDA research volunteers dependent on at least one illegal substance and from Bierut et al 38k dataset (<a href="#b15-drgon" rid="b15-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728051">15</a>) are listed in the last three columns. All of the genes that are identified by clustered nominally positive data from the current dataset are presented in <a href="/pmc/articles/PMC2581605/table/t3-drgon/" target="table" class="fig-table-link figpopup" rid-figpopup="t3-drgon" rid-ob="ob-t3-drgon" co-legend-rid=""><span>Table S1</span></a>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140586971545024" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th colspan="4" rowspan="1" align="center">clustered SNPs</th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th colspan="2" rowspan="1" align="center">MNB</th><th rowspan="1" colspan="1" align="center"></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left">description</th><th rowspan="1" colspan="1" align="center">ch</th><th rowspan="1" colspan="1" align="center">bp</th><th rowspan="1" colspan="1" align="center">Hamer</th><th rowspan="1" colspan="1" align="center">I</th><th rowspan="1" colspan="1" align="center">II</th><th rowspan="1" colspan="1" align="center">Bierut</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">A2BP1</td><td rowspan="1" colspan="1" align="left">ataxin 2-binding protein 1</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="right">6009133</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">13</td></tr><tr><td rowspan="1" colspan="1" align="left">C10orf11</td><td rowspan="1" colspan="1" align="left">chromosome 10 open reading frame 11</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">77212525</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">1</td></tr><tr><td rowspan="1" colspan="1" align="left">CAPN13</td><td rowspan="1" colspan="1" align="left">calpain 13</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">30799141</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">2</td></tr><tr><td rowspan="1" colspan="1" align="left">CDH13</td><td rowspan="1" colspan="1" align="left">cadherin 13</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="right">81218079</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">2</td></tr><tr><td rowspan="1" colspan="1" align="left">CHL1</td><td rowspan="1" colspan="1" align="left">close homolog of L1</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">213650</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">CNTN6</td><td rowspan="1" colspan="1" align="left">contactin 6</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">1109629</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left">CSMD1</td><td rowspan="1" colspan="1" align="left">CUB and Sushi multiple domains 1</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">2782789</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">12</td></tr><tr><td rowspan="1" colspan="1" align="left">CSMD2</td><td rowspan="1" colspan="1" align="left">CUB and Sushi multiple domains 2</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">33752196</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">CSRP3</td><td rowspan="1" colspan="1" align="left">cysteine and glycine-rich protein 3</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">19160154</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">CTNNA3</td><td rowspan="1" colspan="1" align="left">catenin alpha 3</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">67349937</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">21</td></tr><tr><td rowspan="1" colspan="1" align="left">CTNND2</td><td rowspan="1" colspan="1" align="left">catenin delta 2</td><td rowspan="1" colspan="1" align="right">5</td><td rowspan="1" colspan="1" align="right">11024952</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">1</td></tr><tr><td rowspan="1" colspan="1" align="left">CXCL14</td><td rowspan="1" colspan="1" align="left">chemokine (C-X-C motif) ligand 14</td><td rowspan="1" colspan="1" align="right">5</td><td rowspan="1" colspan="1" align="right">134934274</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">DDEF1</td><td rowspan="1" colspan="1" align="left">development and differentiation enhancing factor 1</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">131133535</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">FHIT</td><td rowspan="1" colspan="1" align="left">fragile histidine triad gene</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">59710076</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">2</td></tr><tr><td rowspan="1" colspan="1" align="left">FHOD3</td><td rowspan="1" colspan="1" align="left">formin homology 2 domain containing 3</td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">32131700</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">FLJ45872</td><td rowspan="1" colspan="1" align="left">FLJ45872 protein</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">138890869</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">GRIK2</td><td rowspan="1" colspan="1" align="left">glutamate receptor, ionotropic, kainate 2</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">101953675</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">1</td></tr><tr><td rowspan="1" colspan="1" align="left">GRM3</td><td rowspan="1" colspan="1" align="left">glutamate receptor, metabotropic 3</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">86111166</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">1</td></tr><tr><td rowspan="1" colspan="1" align="left">HPSE2</td><td rowspan="1" colspan="1" align="left">heparanase 2</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">100208867</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">1</td></tr><tr><td rowspan="1" colspan="1" align="left">LRP1B</td><td rowspan="1" colspan="1" align="left">low density lipoprotein-related protein 1B</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">140705466</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">LRRN6C</td><td rowspan="1" colspan="1" align="left">leucine rich repeat neuronal 6C</td><td rowspan="1" colspan="1" align="right">9</td><td rowspan="1" colspan="1" align="right">27938528</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">18</td></tr><tr><td rowspan="1" colspan="1" align="left">NME7</td><td rowspan="1" colspan="1" align="left">nucleoside-diphosphate kinase</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">167368393</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">PARD3B</td><td rowspan="1" colspan="1" align="left">par-3 partitioning defective 3 homolog B</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">205118761</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">PBX1</td><td rowspan="1" colspan="1" align="left">pre-B-cell leukemia transcription factor 1</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">162795561</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">PDE1C</td><td rowspan="1" colspan="1" align="left">phosphodiesterase 1C</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">31795772</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">PTPRM</td><td rowspan="1" colspan="1" align="left">receptor type protein tyrosine phosphatase M</td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">7557817</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">RAD51L1</td><td rowspan="1" colspan="1" align="left">RAD51-like 1</td><td rowspan="1" colspan="1" align="right">14</td><td rowspan="1" colspan="1" align="right">67356262</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">RYR3</td><td rowspan="1" colspan="1" align="left">ryanodine receptor 3</td><td rowspan="1" colspan="1" align="right">15</td><td rowspan="1" colspan="1" align="right">31390469</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left">SGCZ</td><td rowspan="1" colspan="1" align="left">sarcoglycan zeta</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">13991744</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">THSD4</td><td rowspan="1" colspan="1" align="left">thrombospondin type I dom 4</td><td rowspan="1" colspan="1" align="right">15</td><td rowspan="1" colspan="1" align="right">69220842</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">_</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140586971545024" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2581605/table/t1-drgon/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec9" class="sec"><p></p><h3 id="__sec9title" class="inline">Overlap with data from Uhl et al </h3><p id="__p17" class="p p-first-last">The clusters of three nominally positive SNPs from the data that compare current to former smokers also overlap significantly with the clustered, nominally-positive results from two of the three recently-reported samples of individuals who are successful <em>vs</em> those who are unsuccessful in quitting smoking in clinical trials. Comparisons between these data and results reported for samples from Rose and colleagues and from Niarua, David and colleagues (Samples II and III in (<a href="#b14-drgon" rid="b14-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728056">14</a>)) each displayed significant Monte Carlo p values of ≤ 0.0001. Comparison with similarly analyzed samples from Lerman and coworkers, however, provided only a trend toward significance (p = 0.1). Genes identified by clusters of nominally positive SNPs in current and prior studies of ability to successfully quit smoking include ataxin 2-binding protein 1 (3 clustered nominally positive SNPs in the current dataset); CUB and Sushi multiple domains (10 SNPs), Down syndrome cell adhesion molecule (3 SNPs), protocadherin 15 (3 SNPs) and the retinoic acid receptor β (3 SNPs) (<a href="/pmc/articles/PMC2581605/table/t2-drgon/" target="table" class="fig-table-link figpopup" rid-figpopup="t2-drgon" rid-ob="ob-t2-drgon" co-legend-rid=""><span>Table 2</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t2-drgon"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p30">Quit success genes. Genes that contain overlapping clusters of nominally positive SNPs in comparisons of 1) nicotine dependent individuals who report continuing smoking when interviewed <em>vs</em> those who report prolonged abstinence and 2) clusters of nominally positive SNPs from studies of at least two samples of successful <em>vs</em> unsuccessful quitters in clinical trials recently reported by Uhl et al (<a href="#b14-drgon" rid="b14-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728049">14</a>). Columns list gene description, chromosome, basepairs for gene’s start, number of nominally-positive SNPs that lay in clusters within the gene’s exons and in 10 kb genomic flanking regions in the current dataset. All genes identified by clustered, nominally positive data from the current dataset are presented in <a href="/pmc/articles/PMC2581605/table/t4-drgon/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-drgon" rid-ob="ob-t4-drgon" co-legend-rid=""><span>Table S2</span></a>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140586980623504" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center">gene</th><th rowspan="1" colspan="1" align="center">ch</th><th rowspan="1" colspan="1" align="center">bp</th><th rowspan="1" colspan="1" align="center">Clustered SNPS</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><strong>A2BP1</strong></td><td rowspan="1" colspan="1" align="left">ataxin 2-binding protein 1</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="center">6009133</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>CSMD1</strong></td><td rowspan="1" colspan="1" align="left">CUB and Sushi multiple domains 1</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="center">2782789</td><td rowspan="1" colspan="1" align="center">10</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>DSCAM</strong></td><td rowspan="1" colspan="1" align="left">Down syndrome cell adhesion molecule</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="center">40306213</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>PCDH15</strong></td><td rowspan="1" colspan="1" align="left">protocadherin 15</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="center">55250866</td><td rowspan="1" colspan="1" align="center">3</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>RARB</strong></td><td rowspan="1" colspan="1" align="left">retinoic acid receptor β</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="center">25444758</td><td rowspan="1" colspan="1" align="center">3</td></tr></tbody></table></div><div id="largeobj_idm140586980623504" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2581605/table/t2-drgon/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec10" class="sec sec-last"><p></p><h3 id="__sec10title" class="inline">Possible alternative explanations for observed results </h3><p id="__p18" class="p p-first">We would anticipate the observed, highly significant clustering of SNPs that display nominally-positive results if many of these reproducibly-positive SNPs lay near and were in linkage disequilibrium with functional allelic variants that distinguished substance-dependent subjects from control subjects. We would not anticipate this degree of clustering if the results were due to chance. The Monte Carlo p values noted here are thus likely to receive contributions from both the extent of linkage disequilibrium among the clustered, nominally-positive SNPs and the extent of linkage disequilibrium between these SNPs and the functional haplotype(s) that lead to the association with substance dependence.</p><p id="__p19" class="p p-last">Neither control for occult stratification nor assay variability provide convincing alternative explanations for most of the data obtained here. We examined the overlap between the 3,563 and 914 clustered positive SNPs from nicotine dependence and quit success analyses, respectively and the 2.5% of the SNPs for which the noise in validating studies was highest as well as the 2.5% of SNPs that displayed the largest differences between Baltimore African-American <em>vs</em> European American control individuals. We found 57 and 90, respectively, <em>vs</em> 89 and 89 expected by chance for dependence analyses. We found 23 and 15, respectively, <em>vs</em> 29 and 29 expected by chance for the quit success analyses.</p></div></div><div id="__sec11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title">Discussion</h2><p id="__p20" class="p p-first">The current results: 1) provide independent support for GWA results from larger samples of individuals studied for other substance dependence phenotypes, 2) provide independent support for GWA results for studies of smoking cessation and 3) provide a control for one of the potential confounding features of previously-reported smoking cessation GWA samples. The possibility that genetic results from members of any sample of research volunteers might not represent the genetics of members of the general population is ever present. In studies of addiction molecular genetics, however, features that are both 1) heritable and 2) differentially present in substance dependent individuals might, <em>a priori</em>, be considered to be especially likely to provide confounding influences (<a href="#b26-drgon" rid="b26-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728081">26</a>).</p><p id="__p21">Our present observations that findings from a community-based sample of research volunteers converge with findings from two larger research volunteer samples that were ascertained in different fashions are thus reassuring. They support the idea that many of the molecular genetic findings that have been previously reported are not due simply to the methods used for ascertainment and definition of “cases” and “controls”. However, it is important to note that this overall conclusion does not exclude contributions related to sampling to the observations in some particular genes.</p><p id="__p22">The observations comparing individuals who have smoked less than 101 cigarettes with a group of individuals who are currently nicotine-dependent or were nicotine dependent in the past fit with many, but not all, aspects of the GWA dataset available from the work of Beirut and colleagues (<a href="#b15-drgon" rid="b15-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728055">15</a>). The overall fit between these datasets provides mutual support for a number of the genes identified herein. However, we note that we identified no positive associations for SNPs in the chromosome 15 nicotinic acetylcholine receptor gene cluster. This gene cluster was previously identified by comparisons of heavy smokers with FTND dependence <em>vs</em> smokers who were not nicotine dependent by FTND criteria but who were required to have smoked more than 100 cigarettes in their lives. Almost 25% of the members of this “control” comparison group thus displayed nicotine dependence based on DSM criteria. The current results are consistent with those predicted from recent publications from Thorgerison and colleagues, in which SNP allele frequencies in the nicotine receptor chromosome 15 acetylcholine receptor gene cluster were similar in nonsmokers and heavy smokers (<a href="#b16-drgon" rid="b16-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728059">16</a>). The significant differences that emerged in their data when both groups were compared to individuals who smoked more than 100 cigarettes in their lives but did not progress to heavy smoking and FTND nicotine dependence fit with the present data. Non-dependent individuals who smoked at low and steady levels would fit into neither the nicotine dependent nor the control group in our current study.</p><p id="__p23">It is also important to keep a number of limitations in mind in considering the present results. 1) The preplanned approach used here demands that multiple nominally-positive SNPs from each sample tag the same genomic region that lies within a gene. Requirements that nominally positive SNPs from the current dataset come from each of the two 500k array types add a technical control. Monte Carlo approaches that do not require specification of underlying distributions can readily judge the degree to which all of the observations made here could be due to chance. Nevertheless, there have been no unanimous criteria for declaring “replication” or “convergence” for GWA studies, a consideration worth considering in evaluating the current results. 2) These NIH research volunteer samples are of modest size. Power calculations that document the modest power in European-Americans samples reveal even more modest power for the smaller numbers of individuals of other racial/ethnic ancestries. We have thus not analyzed samples with non-European ancestries. 3) Modest power limits interpretation of negative data and restricts inferences about genes that are not identified in these samples. 4) There is significant confidence in the overall set of convergent positive results reported here. Tests for the significance of individual genes, however, provide much more modest levels of statistical assurance. 5) Focus on data from autosomal chromosomes here allows us to combine data from male and female subjects, but misses potentially important contributions from sex chromosomes. 6) The pooling approach that we use here provides excellent correlations between individually-genotyped and pooled allele frequency assessments in validation experiments. This approach has allowed us to use these samples without adding additional confidentiality burdens. Nevertheless, estimates of allele frequencies based on pooled data represent approximations of “true” allele frequency differences that might be determined by error-free individual genotyping of each participant. 7) There is no indication that the overall positive results reported here are based on the SNPs whose assays provide more noise, and no indication that occult stratification on racial/ethnic lines contributed overall to the results that we obtain here. However, we cannot totally exclude contributions of occult stratification that cannot be detected by these overall screens to findings in specific genes. 8) The convergence of data from smoking GWA with data derived from studies of individuals with addictions to substances in several different pharmacological classes supports the idea that many allelic variants enhance vulnerability to many addictions; the more modest statistical significance of this overlap fits with the idea that there is also likely to be some nicotine-selective genetics as well. 9) We have no convincing explanation for the strong fit of our current quit success molecular genetic results with data from two of the three previously-reported samples, but not with the third sample. 10) We focus on identification of genes. Although associations away from annotated genes can also provide interesting results, the genes that we identify in the present work provide a number of interesting views of addiction. These data reinforce our observations that many of these genes are likely to contribute to brain differences that are reflected in the mnemonic aspects of addiction, and that some of them also provide tempting targets for anti-addiction therapeutics. We discuss these ideas in more detail elsewhere [26].</p><p id="__p24">The functional classes into which the genes identified here are interesting, and too complex to be described in detail here. We remain impressed by identification of a number of genes whose products are involved in “cell adhesion” cell to cell communication processes that are likely to be key to proper establishment and plasticity of neuronal connections. CSMD1 CUB and Sushi multiple domains 1, DSCAM, Down syndrome cell adhesion molecule and PCDH15 protocadherin 15 comprise 3 of the 5 “quit success” genes identified in Table II while CDH13 cadherin 13, CHL1 close homolog of L1, CNTN6 contactin 6, CSMD1, CSMD2 CUB and Sushi multiple domains 2, CTNNA3 catenin alpha 3, CTNND2 catenin delta 2, FHIT fragile histidine triad gene, HPSE2 heparanase 2, LRP1B low density lipoprotein-related protein 1B, LRRN6C leucine rich repeat neuronal 6C, PTPRM receptor type protein tyrosine phosphatase M, SGCZ sarcoglycan zeta and THSD4 thrombospondin type I domain 4 comprise 14 of the 30 genes identified in Table I. Each of these genes encodes a protein implicated in cell adhesion and/or extracellular matrix activities that are crucial to synapse formation and maintenance.</p><p id="__p25">The significant convergence between the “quit success” molecular genetics identified here and data from two of the three currently-reported samples supports the idea that genetic contributions to the ability of individuals to stop smoking in the community, often with modest pharmacological and behavioral support, are likely to overlap significantly with genetic contributions to the ability to quit smoking in research settings that frequently manifest substantially greater levels of pharmacological and behavioral support. These findings are thus reassuring for future attempts to use genotype- based data to match smokers with the types and/or intensities of treatments that provide the best and most cost effective smoking cessation opportunities for them in community settings. They also fit with the data from twin studies, since none of the twin studies of smoking cessation success was carried out in a clinical trial setting (<a href="#b10-drgon" rid="b10-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728054">10</a>, <a href="#b11-drgon" rid="b11-drgon" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_621728068">11</a>).</p><p id="__p26" class="p p-last">It appears likely that initial molecular genetic influences on therapeutic responses in other diseases may well be carried out using DNA from participants in clinical trials, since the intensive patient/subject monitoring that is often built into such trials provides reassurance that clinical outcomes are well documented in these datasets. However, the greatest utility of the information from genetic studies of clinical trial participants will come when there is additional data that can assure us that the results can be generalized to more real world settings and to individuals who would be less likely to be enrolled in a clinical trial. The findings presented here thus support ongoing processes for comparing GWA datasets from more intensively-studied and meticulously selected individuals with those from community based samples. For addictions, as for many complex disorders, such data provides an increasingly rich basis for improved understanding and for development of personalized prevention and treatment strategies.</p></div><div id="__sec12" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Supplemental Data</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><div class="sup-box half_rhythm" id="idm140586979187728"><a href="/pmc/articles/PMC2581605/bin/08_096_Drgon_Suppl.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(312K, pdf)</sup></div></div></div><div id="idm140586979186128" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140586979186128title">Acknowledgments</h2><div class="sec"><p id="__p28">We are grateful for thoughtful advice and discussion from Drs J Rose, and to fruitful collaborations with Drs Rose, Lerman, Niaura and David. This research was supported financially by the NIH Intramural Research Programs, NIDA, NIMH and NCI, DHSS.</p></div></div><div id="idm140586978167328" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140586978167328title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="b1-drgon">1. <span class="element-citation">Uhl GR, Elmer GI, Labuda MC, Pickens RW.  Genetic influences in drug abuse. In: Gloom FE, Kupfer DJ, editors. <span class="ref-journal">Psychopharmacology: The Fourth Generation of Progress.</span> Raven Press; New York: 1995. pp. 1793–2783. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Psychopharmacology:+The+Fourth+Generation+of+Progress&amp;author=GR+Uhl&amp;author=GI+Elmer&amp;author=MC+Labuda&amp;author=RW+Pickens&amp;publication_year=1995&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b2-drgon">2. <span class="element-citation">Tsuang MT, Lyons MJ, Meyer JM, et al.  Co-occurrence of abuse of different drugs in men: the role of drug-specific and shared vulnerabilities. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1998;<span class="ref-vol">55</span>:967–972.</span> [<a href="/pubmed/9819064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Co-occurrence+of+abuse+of+different+drugs+in+men:+the+role+of+drug-specific+and+shared+vulnerabilities&amp;author=MT+Tsuang&amp;author=MJ+Lyons&amp;author=JM+Meyer&amp;volume=55&amp;publication_year=1998&amp;pages=967-972&amp;pmid=9819064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b3-drgon">3. <span class="element-citation">Karkowski LM, Prescott CA, Kendler KS. Multivariate assessment of factors influencing illicit substance use in twins from female-female pairs. <span><span class="ref-journal">Am J Med Genet. </span>2000;<span class="ref-vol">96</span>:665–670.</span> [<a href="/pubmed/11054775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Multivariate+assessment+of+factors+influencing+illicit+substance+use+in+twins+from+female-female+pairs&amp;author=LM+Karkowski&amp;author=CA+Prescott&amp;author=KS+Kendler&amp;volume=96&amp;publication_year=2000&amp;pages=665-670&amp;pmid=11054775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b4-drgon">4. <span class="element-citation">True WR, Heath AC, Scherrer JF, et al.  Interrelationship of genetic and environmental influences on conduct disorder and alcohol and marijuana dependence symptoms. <span><span class="ref-journal">Am J Med Genet. </span>1999;<span class="ref-vol">88</span>:391–397.</span> [<a href="/pubmed/10402507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Interrelationship+of+genetic+and+environmental+influences+on+conduct+disorder+and+alcohol+and+marijuana+dependence+symptoms&amp;author=WR+True&amp;author=AC+Heath&amp;author=JF+Scherrer&amp;volume=88&amp;publication_year=1999&amp;pages=391-397&amp;pmid=10402507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b5-drgon">5. <span class="element-citation">Kendler KS, Karkowski LM, Neale MC, Prescott CA. Illicit psychoactive substance use, heavy use, abuse, and dependence in a US population-based sample of male twins. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2000;<span class="ref-vol">57</span>:261–269.</span> [<a href="/pubmed/10711912" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Illicit+psychoactive+substance+use,+heavy+use,+abuse,+and+dependence+in+a+US+population-based+sample+of+male+twins&amp;author=KS+Kendler&amp;author=LM+Karkowski&amp;author=MC+Neale&amp;author=CA+Prescott&amp;volume=57&amp;publication_year=2000&amp;pages=261-269&amp;pmid=10711912&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b6-drgon">6. <span class="element-citation">Fagerstrom KO. Measuring degree of physical dependence to tobacco smoking with reference to individualization of treatment. <span><span class="ref-journal">Addict Behav. </span>1978;<span class="ref-vol">3</span>:235–241.</span> [<a href="/pubmed/735910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Addict+Behav&amp;title=Measuring+degree+of+physical+dependence+to+tobacco+smoking+with+reference+to+individualization+of+treatment&amp;author=KO+Fagerstrom&amp;volume=3&amp;publication_year=1978&amp;pages=235-241&amp;pmid=735910&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b7-drgon">7. <span class="element-citation">Fagerstrom KO, Schneider NG. Measuring nicotine dependence: a review of the Fagerstrom Tolerance Questionnaire. <span><span class="ref-journal">J Behav Med. </span>1989;<span class="ref-vol">12</span>:159–182.</span> [<a href="/pubmed/2668531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Behav+Med&amp;title=Measuring+nicotine+dependence:+a+review+of+the+Fagerstrom+Tolerance+Questionnaire&amp;author=KO+Fagerstrom&amp;author=NG+Schneider&amp;volume=12&amp;publication_year=1989&amp;pages=159-182&amp;pmid=2668531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b8-drgon">8. <span class="element-citation">Pomerleau CS, Carton SM, Lutzke ML, Flessland KA, Pomerleau OF. Reliability of the Fagerstrom Tolerance Questionnaire and the Fagerstrom Test for Nicotine Dependence. <span><span class="ref-journal">Addict Behav. </span>1994;<span class="ref-vol">19</span>:33–39.</span> [<a href="/pubmed/8197891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Addict+Behav&amp;title=Reliability+of+the+Fagerstrom+Tolerance+Questionnaire+and+the+Fagerstrom+Test+for+Nicotine+Dependence&amp;author=CS+Pomerleau&amp;author=SM+Carton&amp;author=ML+Lutzke&amp;author=KA+Flessland&amp;author=OF+Pomerleau&amp;volume=19&amp;publication_year=1994&amp;pages=33-39&amp;pmid=8197891&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b9-drgon">9. <span class="element-citation">Pomerleau CS, Majchrzak MJ, Pomerleau OF. Nicotine dependence and the Fagerstrom Tolerance Questionnaire: a brief review. <span><span class="ref-journal">J Subst Abuse. </span>1989;<span class="ref-vol">1</span>:471–477.</span> [<a href="/pubmed/2485293" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Subst+Abuse&amp;title=Nicotine+dependence+and+the+Fagerstrom+Tolerance+Questionnaire:+a+brief+review&amp;author=CS+Pomerleau&amp;author=MJ+Majchrzak&amp;author=OF+Pomerleau&amp;volume=1&amp;publication_year=1989&amp;pages=471-477&amp;pmid=2485293&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b10-drgon">10. <span class="element-citation">Broms U, Silventoinen K, Madden PA, Heath AC, Kaprio J. Genetic architecture of smoking behavior: a study of Finnish adult twins. <span><span class="ref-journal">Twin Res Hum Genet. </span>2006;<span class="ref-vol">9</span>:64–72.</span> [<a href="/pubmed/16611469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Hum+Genet&amp;title=Genetic+architecture+of+smoking+behavior:+a+study+of+Finnish+adult+twins&amp;author=U+Broms&amp;author=K+Silventoinen&amp;author=PA+Madden&amp;author=AC+Heath&amp;author=J+Kaprio&amp;volume=9&amp;publication_year=2006&amp;pages=64-72&amp;pmid=16611469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b11-drgon">11. <span class="element-citation">Lessov CN, Martin NG, Statham DJ, et al.  Defining nicotine dependence for genetic research: evidence from Australian twins. <span><span class="ref-journal">Psychol Med. </span>2004;<span class="ref-vol">34</span>:865–879.</span> [<a href="/pubmed/15500307" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychol+Med&amp;title=Defining+nicotine+dependence+for+genetic+research:+evidence+from+Australian+twins&amp;author=CN+Lessov&amp;author=NG+Martin&amp;author=DJ+Statham&amp;volume=34&amp;publication_year=2004&amp;pages=865-879&amp;pmid=15500307&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b12-drgon">12. <span class="element-citation">Uhl GR, Drgon T, Liu QR, et al.  Genome-wide association for methamphetamine dependence: convergent results from 2 samples. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2008;<span class="ref-vol">65</span>:345–355.</span> [<a href="/pubmed/18316681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Genome-wide+association+for+methamphetamine+dependence:+convergent+results+from+2+samples&amp;author=GR+Uhl&amp;author=T+Drgon&amp;author=QR+Liu&amp;volume=65&amp;publication_year=2008&amp;pages=345-355&amp;pmid=18316681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b13-drgon">13. <span class="element-citation">Uhl GR, Liu QR, Drgon T, Johnson C, Walther D, Rose JE. Molecular genetics of nicotine dependence and abstinence: whole genome association using 520,000 SNPs. <span><span class="ref-journal">BMC Genet. </span>2007;<span class="ref-vol">8</span>:10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1853105/">PMC free article</a>]</span> [<a href="/pubmed/17407593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=Molecular+genetics+of+nicotine+dependence+and+abstinence:+whole+genome+association+using+520,000+SNPs&amp;author=GR+Uhl&amp;author=QR+Liu&amp;author=T+Drgon&amp;author=C+Johnson&amp;author=D+Walther&amp;volume=8&amp;publication_year=2007&amp;pages=10&amp;pmid=17407593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b14-drgon">14. <span class="element-citation">Uhl GR, Liu QR, Drgon T, et al.  Molecular genetics of successful smoking cessation: convergent genome-wide association study results. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2008;<span class="ref-vol">65</span>:683–693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2430596/">PMC free article</a>]</span> [<a href="/pubmed/18519826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Molecular+genetics+of+successful+smoking+cessation:+convergent+genome-wide+association+study+results&amp;author=GR+Uhl&amp;author=QR+Liu&amp;author=T+Drgon&amp;volume=65&amp;publication_year=2008&amp;pages=683-693&amp;pmid=18519826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b15-drgon">15. <span class="element-citation">Bierut LJ, Madden PA, Breslau N, et al.  Novel genes identified in a high-density genome wide association study for nicotine dependence. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>:24–35.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2278047/">PMC free article</a>]</span> [<a href="/pubmed/17158188" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Novel+genes+identified+in+a+high-density+genome+wide+association+study+for+nicotine+dependence&amp;author=LJ+Bierut&amp;author=PA+Madden&amp;author=N+Breslau&amp;volume=16&amp;publication_year=2007&amp;pages=24-35&amp;pmid=17158188&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b16-drgon">16. <span class="element-citation">Thorgeirsson TE, Geller F, Sulem P, et al.  A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">452</span>:638–642.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4539558/">PMC free article</a>]</span> [<a href="/pubmed/18385739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+variant+associated+with+nicotine+dependence,+lung+cancer+and+peripheral+arterial+disease&amp;author=TE+Thorgeirsson&amp;author=F+Geller&amp;author=P+Sulem&amp;volume=452&amp;publication_year=2008&amp;pages=638-642&amp;pmid=18385739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b17-drgon">17. <span class="element-citation">Amos CI, Wu X, Broderick P, et al.  Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:616–622.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2713680/">PMC free article</a>]</span> [<a href="/pubmed/18385676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+locus+for+lung+cancer+at+15q25.1&amp;author=CI+Amos&amp;author=X+Wu&amp;author=P+Broderick&amp;volume=40&amp;publication_year=2008&amp;pages=616-622&amp;pmid=18385676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b18-drgon">18. <span class="element-citation">Saccone SF, Hinrichs AL, Saccone NL, et al.  Cholinergic nicotinic receptor genes implicated in a nicotine dependence association study targeting 348 candidate genes with 3713 SNPs. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>:36–49.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2270437/">PMC free article</a>]</span> [<a href="/pubmed/17135278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Cholinergic+nicotinic+receptor+genes+implicated+in+a+nicotine+dependence+association+study+targeting+348+candidate+genes+with+3713+SNPs&amp;author=SF+Saccone&amp;author=AL+Hinrichs&amp;author=NL+Saccone&amp;volume=16&amp;publication_year=2007&amp;pages=36-49&amp;pmid=17135278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b19-drgon">19. <span class="element-citation">Lueders KK, Hu S, McHugh L, Myakishev MV, Sirota LA, Hamer DH. Genetic and functional analysis of single nucleotide polymorphisms in the beta2-neuronal nicotinic acetylcholine receptor gene (CHRNB2) <span><span class="ref-journal">Nicotine Tob Res. </span>2002;<span class="ref-vol">4</span>:115–125.</span> [<a href="/pubmed/11906688" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nicotine+Tob+Res&amp;title=Genetic+and+functional+analysis+of+single+nucleotide+polymorphisms+in+the+beta2-neuronal+nicotinic+acetylcholine+receptor+gene+(CHRNB2)&amp;author=KK+Lueders&amp;author=S+Hu&amp;author=L+McHugh&amp;author=MV+Myakishev&amp;author=LA+Sirota&amp;volume=4&amp;publication_year=2002&amp;pages=115-125&amp;pmid=11906688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b20-drgon">20. <span class="element-citation">Liu QR, Drgon T, Johnson C, Walther D, Hess J, Uhl GR. Addiction molecular genetics: 639,401 SNP whole genome association identifies many "cell adhesion" genes. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2006;<span class="ref-vol">141B</span>:918–925.</span> [<a href="/pubmed/17099884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Addiction+molecular+genetics:+639,401+SNP+whole+genome+association+identifies+many+&quot;cell+adhesion&quot;+genes&amp;author=QR+Liu&amp;author=T+Drgon&amp;author=C+Johnson&amp;author=D+Walther&amp;author=J+Hess&amp;volume=141B&amp;publication_year=2006&amp;pages=918-925&amp;pmid=17099884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b21-drgon">21. <span class="element-citation">Uhl GR, Liu QR, Walther D, Hess J, Naiman D. Polysubstance abuse-vulnerability genes: genome scans for association, using 1,004 subjects and 1,494 single-nucleotide polymorphisms. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">69</span>:1290–1300.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235541/">PMC free article</a>]</span> [<a href="/pubmed/11704927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Polysubstance+abuse-vulnerability+genes:+genome+scans+for+association,+using+1,004+subjects+and+1,494+single-nucleotide+polymorphisms&amp;author=GR+Uhl&amp;author=QR+Liu&amp;author=D+Walther&amp;author=J+Hess&amp;author=D+Naiman&amp;volume=69&amp;publication_year=2001&amp;pages=1290-1300&amp;pmid=11704927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b22-drgon">22. <span class="element-citation">Smith SS, O'Hara BF, Persico AM, et al.  Genetic vulnerability to drug abuse. The D2 dopamine receptor Taq I B1 restriction fragment length polymorphism appears more frequently in polysubstance abusers. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1992;<span class="ref-vol">49</span>:723–727.</span> [<a href="/pubmed/1355337" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Genetic+vulnerability+to+drug+abuse.+The+D2+dopamine+receptor+Taq+I+B1+restriction+fragment+length+polymorphism+appears+more+frequently+in+polysubstance+abusers&amp;author=SS+Smith&amp;author=BF+O'Hara&amp;author=AM+Persico&amp;volume=49&amp;publication_year=1992&amp;pages=723-727&amp;pmid=1355337&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b23-drgon">23. <span class="element-citation">Persico AM, Bird G, Gabbay FH, Uhl GR. D2 dopamine receptor gene TaqI A1 and B1 restriction fragment length polymorphisms: enhanced frequencies in psychostimulant-preferring polysubstance abusers. <span><span class="ref-journal">Biol Psychiatry. </span>1996;<span class="ref-vol">40</span>:776–784.</span> [<a href="/pubmed/8894071" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=D2+dopamine+receptor+gene+TaqI+A1+and+B1+restriction+fragment+length+polymorphisms:+enhanced+frequencies+in+psychostimulant-preferring+polysubstance+abusers&amp;author=AM+Persico&amp;author=G+Bird&amp;author=FH+Gabbay&amp;author=GR+Uhl&amp;volume=40&amp;publication_year=1996&amp;pages=776-784&amp;pmid=8894071&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b24-drgon">24. <span class="element-citation">Dupont WD, Plummer WD., Jr Power and sample size calculations. A review and computer program. <span><span class="ref-journal">Control Clin Trials. </span>1990;<span class="ref-vol">11</span>:116–128.</span> [<a href="/pubmed/2161310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Control+Clin+Trials&amp;title=Power+and+sample+size+calculations.+A+review+and+computer+program&amp;author=WD+Dupont&amp;author=WD+Plummer&amp;volume=11&amp;publication_year=1990&amp;pages=116-128&amp;pmid=2161310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b25-drgon">25. <span class="element-citation">Dupont WD, Plummer WD., Jr Power and sample size calculations for studies involving linear regression. <span><span class="ref-journal">Control Clin Trials. </span>1998;<span class="ref-vol">19</span>:589–601.</span> [<a href="/pubmed/9875838" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Control+Clin+Trials&amp;title=Power+and+sample+size+calculations+for+studies+involving+linear+regression&amp;author=WD+Dupont&amp;author=WD+Plummer&amp;volume=19&amp;publication_year=1998&amp;pages=589-601&amp;pmid=9875838&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b26-drgon">26. <span class="element-citation">Uhl GR, Drgon T, Johnson C, et al.  Molecular genetics of addiction and related heritable phenotypes: genome wide association approaches identify “connectivity constellation” and drug target genes with pleiotropic effects. <span><span class="ref-journal">Ann NYAS. </span>2008</span> (in press) <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3922196/">PMC free article</a>]</span> [<a href="/pubmed/18991966" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+NYAS&amp;title=Molecular+genetics+of+addiction+and+related+heritable+phenotypes:+genome+wide+association+approaches+identify+“connectivity+constellation”+and+drug+target+genes+with+pleiotropic+effects&amp;author=GR+Uhl&amp;author=T+Drgon&amp;author=C+Johnson&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2581605&amp;issue-id=175111&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140586986079568title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and Methods</a></li><li><a href="#__sec5title">Results</a></li><li><a href="#__sec11title">Discussion</a></li><li><a href="#__sec12title">Supplemental Data</a></li><li><a href="#idm140586979186128title">Acknowledgments</a></li><li><a href="#idm140586978167328title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Molecular Medicine</span> are provided here courtesy of <strong>The Feinstein Institute for Medical Research at North Shore LIJ</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2581605/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2581605/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2581605/pdf/mol-15-21.pdf">PDF (76K)</a></li> | <li><a href="#" data-citationid="PMC2581605" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2581605%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2581605%2F&amp;text=Genome%20wide%20association%20for%20nicotine%20dependence%20and%20smoking%20cessation%20success%20in%20NIH%20research%20volunteers" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2581605%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19009022" data-aiid="2615804" data-aid="2581605" data-iid="175111" data-domainid="325" data-domain="molmed" data-accid="PMC2581605" data-md5="2ca6617b10b64321f5e91c78f88758f8"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T10:33:40-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B58D0DD405C710000000004950295&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2581605%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=molmed&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2581605/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>