<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3857947/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association study and biological pathway analysis of epilepsy prognosis in a prospective cohort of newly treated epilepsy</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3857947/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="A genome-wide association study and biological pathway analysis of epilepsy prognosis in a prospective cohort of newly treated epilepsy"><meta name="citation_authors" content="Doug Speed, Clive Hoggart, Slave Petrovski, Ioanna Tachmazidou, Alison Coffey, Andrea Jorgensen, Hariklia Eleftherohorinou, Maria De Iorio, Marian Todaro, Tisham De, David Smith, Philip E. Smith, Margaret Jackson, Paul Cooper, Mark Kellett, Stephen Howell, Mark Newton, Raju Yerra, Meng Tan, Chris French, Markus Reuber, Graeme E. Sills, David Chadwick, Munir Pirmohamed, David Bentley, Ingrid Scheffer, Samuel Berkovic, David Balding, Aarno Palotie, Anthony Marson, Terence J. O'Brien, Michael R. Johnson"><meta name="citation_date" content="1 January 2014"><meta name="citation_issue" content="1"><meta name="citation_volume" content="23"><meta name="citation_firstpage" content="247"><meta name="citation_doi" content="10.1093/hmg/ddt403"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3857947/?report=abstract"><meta name="citation_pmid" content="23962720"><meta name="DC.Title" content="A genome-wide association study and biological pathway analysis of epilepsy prognosis in a prospective cohort of newly treated epilepsy"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Doug Speed"><meta name="DC.Contributor" content="Clive Hoggart"><meta name="DC.Contributor" content="Slave Petrovski"><meta name="DC.Contributor" content="Ioanna Tachmazidou"><meta name="DC.Contributor" content="Alison Coffey"><meta name="DC.Contributor" content="Andrea Jorgensen"><meta name="DC.Contributor" content="Hariklia Eleftherohorinou"><meta name="DC.Contributor" content="Maria De Iorio"><meta name="DC.Contributor" content="Marian Todaro"><meta name="DC.Contributor" content="Tisham De"><meta name="DC.Contributor" content="David Smith"><meta name="DC.Contributor" content="Philip E. Smith"><meta name="DC.Contributor" content="Margaret Jackson"><meta name="DC.Contributor" content="Paul Cooper"><meta name="DC.Contributor" content="Mark Kellett"><meta name="DC.Contributor" content="Stephen Howell"><meta name="DC.Contributor" content="Mark Newton"><meta name="DC.Contributor" content="Raju Yerra"><meta name="DC.Contributor" content="Meng Tan"><meta name="DC.Contributor" content="Chris French"><meta name="DC.Contributor" content="Markus Reuber"><meta name="DC.Contributor" content="Graeme E. Sills"><meta name="DC.Contributor" content="David Chadwick"><meta name="DC.Contributor" content="Munir Pirmohamed"><meta name="DC.Contributor" content="David Bentley"><meta name="DC.Contributor" content="Ingrid Scheffer"><meta name="DC.Contributor" content="Samuel Berkovic"><meta name="DC.Contributor" content="David Balding"><meta name="DC.Contributor" content="Aarno Palotie"><meta name="DC.Contributor" content="Anthony Marson"><meta name="DC.Contributor" content="Terence J. O'Brien"><meta name="DC.Contributor" content="Michael R. Johnson"><meta name="DC.Date" content="2014 Jan 1"><meta name="DC.Identifier" content="10.1093/hmg/ddt403"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association study and biological pathway analysis of epilepsy prognosis in a prospective cohort of newly treated epilepsy"><meta property="og:type" content="article"><meta property="og:description" content="We present the analysis of a prospective multicentre study to investigate genetic effects on the prognosis of newly treated epilepsy. Patients with a new clinical diagnosis of epilepsy requiring medication were recruited and followed up prospectively. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3857947/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3857947/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3857947/pdf/ddt403.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C46D1DD55B7510000000007DC04BB.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3857947%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3857947</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2014 Jan 1; 23(1): 247–258. </span></div><div><span class="fm-vol-iss-date">Published online 2013 Aug 19. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddt403" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddt403</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3857947</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23962720">23962720</a></div></div></div></div><h1 class="content-title">A genome-wide association study and biological pathway analysis of epilepsy prognosis in a prospective cohort of newly treated epilepsy</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Speed%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715888467232" co-class="co-affbox">Doug Speed</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Hoggart%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715889730464" co-class="co-affbox">Clive Hoggart</a>,<sup>2,</sup><sup>†</sup> <a href="/pubmed/?term=Petrovski%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715889728416" co-class="co-affbox">Slave Petrovski</a>,<sup>3,</sup><sup>4,</sup><sup>†</sup> <a href="/pubmed/?term=Tachmazidou%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715890675056" co-class="co-affbox">Ioanna Tachmazidou</a>,<sup>5,</sup><sup>†</sup> <a href="/pubmed/?term=Coffey%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715890672448" co-class="co-affbox">Alison Coffey</a>,<sup>5</sup> <a href="/pubmed/?term=Jorgensen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715891861520" co-class="co-affbox">Andrea Jorgensen</a>,<sup>6</sup> <a href="/pubmed/?term=Eleftherohorinou%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715901378048" co-class="co-affbox">Hariklia Eleftherohorinou</a>,<sup>2</sup> <a href="/pubmed/?term=De%20Iorio%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715901376064" co-class="co-affbox">Maria De Iorio</a>,<sup>7</sup> <a href="/pubmed/?term=Todaro%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715889894288" co-class="co-affbox">Marian Todaro</a>,<sup>3</sup> <a href="/pubmed/?term=De%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715900569472" co-class="co-affbox">Tisham De</a>,<sup>7</sup> <a href="/pubmed/?term=Smith%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715900567488" co-class="co-affbox">David Smith</a>,<sup>8</sup> <a href="/pubmed/?term=Smith%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715898821904" co-class="co-affbox">Philip E. Smith</a>,<sup>9</sup> <a href="/pubmed/?term=Jackson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715898819920" co-class="co-affbox">Margaret Jackson</a>,<sup>10</sup> <a href="/pubmed/?term=Cooper%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715896596432" co-class="co-affbox">Paul Cooper</a>,<sup>11</sup> <a href="/pubmed/?term=Kellett%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715894622672" co-class="co-affbox">Mark Kellett</a>,<sup>12</sup> <a href="/pubmed/?term=Howell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715894620688" co-class="co-affbox">Stephen Howell</a>,<sup>13</sup> <a href="/pubmed/?term=Newton%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715891131632" co-class="co-affbox">Mark Newton</a>,<sup>14</sup> <a href="/pubmed/?term=Yerra%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715970135392" co-class="co-affbox">Raju Yerra</a>,<sup>3</sup> <a href="/pubmed/?term=Tan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715970133408" co-class="co-affbox">Meng Tan</a>,<sup>3</sup> <a href="/pubmed/?term=French%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715889651248" co-class="co-affbox">Chris French</a>,<sup>3</sup> <a href="/pubmed/?term=Reuber%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715895579952" co-class="co-affbox">Markus Reuber</a>,<sup>15</sup> <a href="/pubmed/?term=Sills%20GE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715895577968" co-class="co-affbox">Graeme E. Sills</a>,<sup>16</sup> <a href="/pubmed/?term=Chadwick%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715891112128" co-class="co-affbox">David Chadwick</a>,<sup>8</sup> <a href="/pubmed/?term=Pirmohamed%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715972292192" co-class="co-affbox">Munir Pirmohamed</a>,<sup>16</sup> <a href="/pubmed/?term=Bentley%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715972290208" co-class="co-affbox">David Bentley</a>,<sup>17</sup> <a href="/pubmed/?term=Scheffer%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715887791696" co-class="co-affbox">Ingrid Scheffer</a>,<sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Berkovic%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715892465968" co-class="co-affbox">Samuel Berkovic</a>,<sup>14</sup> <a href="/pubmed/?term=Balding%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715892463984" co-class="co-affbox">David Balding</a>,<sup>1</sup> <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715889744560" co-class="co-affbox">Aarno Palotie</a>,<sup>5,</sup><sup>20,</sup><sup>21,</sup><sup>22,</sup><sup>23,</sup><sup>‡</sup> <a href="/pubmed/?term=Marson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715894522864" co-class="co-affbox">Anthony Marson</a>,<sup>16,</sup><sup>‡</sup> <a href="/pubmed/?term=O'Brien%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715890890480" co-class="co-affbox">Terence J. O'Brien</a>,<sup>3,</sup><sup>‡</sup> and  <a href="/pubmed/?term=Johnson%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720" class="affpopup" co-rid="_co_idm140715903199648" co-class="co-affbox">Michael R. Johnson</a><sup>24,</sup><sup>‡</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140715888467232"><h3 class="no_margin">Doug Speed</h3><p><sup>1</sup>UCL Genetics Institute, University College London, WC1E 6BT, UK,</p><div>Find articles by <a href="/pubmed/?term=Speed%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Doug Speed</a></div></div><div id="_co_idm140715889730464"><h3 class="no_margin">Clive Hoggart</h3><p><sup>2</sup>Department of Genomics of Common Disease, Imperial College London, W2 1PG, UK,</p><div>Find articles by <a href="/pubmed/?term=Hoggart%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Clive Hoggart</a></div></div><div id="_co_idm140715889728416"><h3 class="no_margin">Slave Petrovski</h3><p><sup>3</sup>The Departments of Medicine and Neurology, The Royal Melbourne Hospital, The University of Melbourne, Victoria 3050, Australia,</p><p><sup>4</sup>The Center for Human Genome Variation, Duke University School of Medicine, Durham, NC 27708, USA,</p><div>Find articles by <a href="/pubmed/?term=Petrovski%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Slave Petrovski</a></div></div><div id="_co_idm140715890675056"><h3 class="no_margin">Ioanna Tachmazidou</h3><p><sup>5</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK,</p><div>Find articles by <a href="/pubmed/?term=Tachmazidou%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Ioanna Tachmazidou</a></div></div><div id="_co_idm140715890672448"><h3 class="no_margin">Alison Coffey</h3><p><sup>5</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK,</p><div>Find articles by <a href="/pubmed/?term=Coffey%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Alison Coffey</a></div></div><div id="_co_idm140715891861520"><h3 class="no_margin">Andrea Jorgensen</h3><p><sup>6</sup>Department of Biostatistics, University of Liverpool, Liverpool L69 3GS, UK,</p><div>Find articles by <a href="/pubmed/?term=Jorgensen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Andrea Jorgensen</a></div></div><div id="_co_idm140715901378048"><h3 class="no_margin">Hariklia Eleftherohorinou</h3><p><sup>2</sup>Department of Genomics of Common Disease, Imperial College London, W2 1PG, UK,</p><div>Find articles by <a href="/pubmed/?term=Eleftherohorinou%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Hariklia Eleftherohorinou</a></div></div><div id="_co_idm140715901376064"><h3 class="no_margin">Maria De Iorio</h3><p><sup>7</sup>Department of Epidemiology and Biostatistics, Imperial College London, W2 1NY, UK,</p><div>Find articles by <a href="/pubmed/?term=De%20Iorio%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Maria De Iorio</a></div></div><div id="_co_idm140715889894288"><h3 class="no_margin">Marian Todaro</h3><p><sup>3</sup>The Departments of Medicine and Neurology, The Royal Melbourne Hospital, The University of Melbourne, Victoria 3050, Australia,</p><div>Find articles by <a href="/pubmed/?term=Todaro%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Marian Todaro</a></div></div><div id="_co_idm140715900569472"><h3 class="no_margin">Tisham De</h3><p><sup>7</sup>Department of Epidemiology and Biostatistics, Imperial College London, W2 1NY, UK,</p><div>Find articles by <a href="/pubmed/?term=De%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Tisham De</a></div></div><div id="_co_idm140715900567488"><h3 class="no_margin">David Smith</h3><p><sup>8</sup>The Walton Centre NHS Foundation Trust, Liverpool L9 7LJ, UK,</p><div>Find articles by <a href="/pubmed/?term=Smith%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">David Smith</a></div></div><div id="_co_idm140715898821904"><h3 class="no_margin">Philip E. Smith</h3><p><sup>9</sup>Department of Neurology, Alan Richens Epilepsy Unit, University Hospital of Wales, Cardiff CF14 4XW, UK,</p><div>Find articles by <a href="/pubmed/?term=Smith%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Philip E. Smith</a></div></div><div id="_co_idm140715898819920"><h3 class="no_margin">Margaret Jackson</h3><p><sup>10</sup>Newcastle General Hospital, The Newcastle Upon Tyne Hospitals NHS Trust, Newcastle NE7 7DN, UK,</p><div>Find articles by <a href="/pubmed/?term=Jackson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Margaret Jackson</a></div></div><div id="_co_idm140715896596432"><h3 class="no_margin">Paul Cooper</h3><p><sup>11</sup>Salford Royal NHS Foundation Trust, Salford M6 8HD, UK,</p><div>Find articles by <a href="/pubmed/?term=Cooper%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Paul Cooper</a></div></div><div id="_co_idm140715894622672"><h3 class="no_margin">Mark Kellett</h3><p><sup>12</sup>Royal Bolton and Hope Hospitals NHS Trust, Bolton BL4 0JR, UK,</p><div>Find articles by <a href="/pubmed/?term=Kellett%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Mark Kellett</a></div></div><div id="_co_idm140715894620688"><h3 class="no_margin">Stephen Howell</h3><p><sup>13</sup>Sheffield Teaching Hospital Foundation NHS Trust, Sheffield S10 2JF, UK,</p><div>Find articles by <a href="/pubmed/?term=Howell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Stephen Howell</a></div></div><div id="_co_idm140715891131632"><h3 class="no_margin">Mark Newton</h3><p><sup>14</sup>The Epilepsy Research Centre, The Departments of Medicine and Neurology, Austin Health, The University of Melbourne, Heidelberg 3084, Australia,</p><div>Find articles by <a href="/pubmed/?term=Newton%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Mark Newton</a></div></div><div id="_co_idm140715970135392"><h3 class="no_margin">Raju Yerra</h3><p><sup>3</sup>The Departments of Medicine and Neurology, The Royal Melbourne Hospital, The University of Melbourne, Victoria 3050, Australia,</p><div>Find articles by <a href="/pubmed/?term=Yerra%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Raju Yerra</a></div></div><div id="_co_idm140715970133408"><h3 class="no_margin">Meng Tan</h3><p><sup>3</sup>The Departments of Medicine and Neurology, The Royal Melbourne Hospital, The University of Melbourne, Victoria 3050, Australia,</p><div>Find articles by <a href="/pubmed/?term=Tan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Meng Tan</a></div></div><div id="_co_idm140715889651248"><h3 class="no_margin">Chris French</h3><p><sup>3</sup>The Departments of Medicine and Neurology, The Royal Melbourne Hospital, The University of Melbourne, Victoria 3050, Australia,</p><div>Find articles by <a href="/pubmed/?term=French%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Chris French</a></div></div><div id="_co_idm140715895579952"><h3 class="no_margin">Markus Reuber</h3><p><sup>15</sup>Academic Neurology Unit, Royal Hallamshire Hospital, Sheffield S10 2JF, UK,</p><div>Find articles by <a href="/pubmed/?term=Reuber%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Markus Reuber</a></div></div><div id="_co_idm140715895577968"><h3 class="no_margin">Graeme E. Sills</h3><p><sup>16</sup>Department of Molecular and Clinical Pharmacology, University of Liverpool, L69 3BX, UK,</p><div>Find articles by <a href="/pubmed/?term=Sills%20GE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Graeme E. Sills</a></div></div><div id="_co_idm140715891112128"><h3 class="no_margin">David Chadwick</h3><p><sup>8</sup>The Walton Centre NHS Foundation Trust, Liverpool L9 7LJ, UK,</p><div>Find articles by <a href="/pubmed/?term=Chadwick%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">David Chadwick</a></div></div><div id="_co_idm140715972292192"><h3 class="no_margin">Munir Pirmohamed</h3><p><sup>16</sup>Department of Molecular and Clinical Pharmacology, University of Liverpool, L69 3BX, UK,</p><div>Find articles by <a href="/pubmed/?term=Pirmohamed%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Munir Pirmohamed</a></div></div><div id="_co_idm140715972290208"><h3 class="no_margin">David Bentley</h3><p><sup>17</sup>Illumina Cambridge Ltd., Chesterford Research Park, Cambridge CB10 1XL, UK,</p><div>Find articles by <a href="/pubmed/?term=Bentley%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">David Bentley</a></div></div><div id="_co_idm140715887791696"><h3 class="no_margin">Ingrid Scheffer</h3><p><sup>18</sup>The Florey Institute, Department of Medicine, Austin Health, Victoria 3084, Australia,</p><p><sup>19</sup>Department of Paediatrics, Royal Children's Hospital, The University of Melbourne, Victoria 3010, Australia,</p><div>Find articles by <a href="/pubmed/?term=Scheffer%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Ingrid Scheffer</a></div></div><div id="_co_idm140715892465968"><h3 class="no_margin">Samuel Berkovic</h3><p><sup>14</sup>The Epilepsy Research Centre, The Departments of Medicine and Neurology, Austin Health, The University of Melbourne, Heidelberg 3084, Australia,</p><div>Find articles by <a href="/pubmed/?term=Berkovic%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Samuel Berkovic</a></div></div><div id="_co_idm140715892463984"><h3 class="no_margin">David Balding</h3><p><sup>1</sup>UCL Genetics Institute, University College London, WC1E 6BT, UK,</p><div>Find articles by <a href="/pubmed/?term=Balding%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">David Balding</a></div></div><div id="_co_idm140715889744560"><h3 class="no_margin">Aarno Palotie</h3><p><sup>5</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK,</p><p><sup>20</sup>Institute for Molecular Medicine Finland (FIMM),  and,</p><p><sup>21</sup>Department of Medical Genetics, University of Helsinki, FI-00014, Finland,</p><p><sup>22</sup>The Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA,</p><p><sup>23</sup>University Central Hospital, Helsinki, FI-00290, Finland,  and</p><div>Find articles by <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Aarno Palotie</a></div></div><div id="_co_idm140715894522864"><h3 class="no_margin">Anthony Marson</h3><p><sup>16</sup>Department of Molecular and Clinical Pharmacology, University of Liverpool, L69 3BX, UK,</p><div>Find articles by <a href="/pubmed/?term=Marson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Anthony Marson</a></div></div><div id="_co_idm140715890890480"><h3 class="no_margin">Terence J. O'Brien</h3><p><sup>3</sup>The Departments of Medicine and Neurology, The Royal Melbourne Hospital, The University of Melbourne, Victoria 3050, Australia,</p><div>Find articles by <a href="/pubmed/?term=O'Brien%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Terence J. O'Brien</a></div></div><div id="_co_idm140715903199648"><h3 class="no_margin">Michael R. Johnson</h3><p><sup>24</sup>Division of Brain Science, Imperial College London, W6 8RF, UK</p><div>Find articles by <a href="/pubmed/?term=Johnson%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23962720">Michael R. Johnson</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140715896985328_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140715896985328_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140715896985328_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140715896985328_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>UCL Genetics Institute, University College London, WC1E 6BT, UK,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Genomics of Common Disease, Imperial College London, W2 1PG, UK,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>The Departments of Medicine and Neurology, The Royal Melbourne Hospital, The University of Melbourne, Victoria 3050, Australia,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>The Center for Human Genome Variation, Duke University School of Medicine, Durham, NC 27708, USA,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, UK,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Department of Biostatistics, University of Liverpool, Liverpool L69 3GS, UK,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Epidemiology and Biostatistics, Imperial College London, W2 1NY, UK,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>The Walton Centre NHS Foundation Trust, Liverpool L9 7LJ, UK,</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Department of Neurology, Alan Richens Epilepsy Unit, University Hospital of Wales, Cardiff CF14 4XW, UK,</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Newcastle General Hospital, The Newcastle Upon Tyne Hospitals NHS Trust, Newcastle NE7 7DN, UK,</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Salford Royal NHS Foundation Trust, Salford M6 8HD, UK,</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Royal Bolton and Hope Hospitals NHS Trust, Bolton BL4 0JR, UK,</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Sheffield Teaching Hospital Foundation NHS Trust, Sheffield S10 2JF, UK,</div><div class="fm-affl" id="af14" lang="en"><sup>14</sup>The Epilepsy Research Centre, The Departments of Medicine and Neurology, Austin Health, The University of Melbourne, Heidelberg 3084, Australia,</div><div class="fm-affl" id="af15" lang="en"><sup>15</sup>Academic Neurology Unit, Royal Hallamshire Hospital, Sheffield S10 2JF, UK,</div><div class="fm-affl" id="af16" lang="en"><sup>16</sup>Department of Molecular and Clinical Pharmacology, University of Liverpool, L69 3BX, UK,</div><div class="fm-affl" id="af17" lang="en"><sup>17</sup>Illumina Cambridge Ltd., Chesterford Research Park, Cambridge CB10 1XL, UK,</div><div class="fm-affl" id="af18" lang="en"><sup>18</sup>The Florey Institute, Department of Medicine, Austin Health, Victoria 3084, Australia,</div><div class="fm-affl" id="af19" lang="en"><sup>19</sup>Department of Paediatrics, Royal Children's Hospital, The University of Melbourne, Victoria 3010, Australia,</div><div class="fm-affl" id="af20" lang="en"><sup>20</sup>Institute for Molecular Medicine Finland (FIMM),  and,</div><div class="fm-affl" id="af21" lang="en"><sup>21</sup>Department of Medical Genetics, University of Helsinki, FI-00014, Finland,</div><div class="fm-affl" id="af22" lang="en"><sup>22</sup>The Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA,</div><div class="fm-affl" id="af23" lang="en"><sup>23</sup>University Central Hospital, Helsinki, FI-00290, Finland,  and</div><div class="fm-affl" id="af24" lang="en"><sup>24</sup>Division of Brain Science, Imperial College London, W6 8RF, UK</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Centre for Clinical Translation, Division of Brain Sciences, Imperial College London, Charing Cross Hospital Campus, London W6 8RF, UK. Tel: +44 02033111194; Fax: +44 02033117487; Email: <a href="mailto:dev@null" data-email="ku.ca.lairepmi@nosnhoj.m" class="oemail">ku.ca.lairepmi@nosnhoj.m</a></div><div id="AN1"><sup>†</sup>These authors contributed equally: D.S., C.H., S.P. and I.T.</div><div id="AN2"><sup>‡</sup>These authors are the joint senior authors: T.J.O'B., A.M., A.P. and M.R.J.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140715896985328_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2013 Feb 23; Revised 2013 Jul 29; Accepted 2013 Aug 12.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140715896985328_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2013. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3857947/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3857947/bin/supp_23_1_247__index.html" data-ga-action="click_feat_suppl">supp_23_1_247__index.html</a><span style="color:gray"> (1.6K)</span></div><div class="small guid">GUID:&nbsp;2093150E-028E-4829-93F9-4DBFF07E031F</div>
<div><a href="/pmc/articles/PMC3857947/bin/supp_ddt403_ddt403supp.doc" data-ga-action="click_feat_suppl">supp_ddt403_ddt403supp.doc</a><span style="color:gray"> (1.7M)</span></div><div class="small guid">GUID:&nbsp;DEB62084-98B3-4A76-A1FB-8DCEE2858F52</div>
<div><a href="/pmc/articles/PMC3857947/bin/supp_ddt403_ddt403supp_table1.doc" data-ga-action="click_feat_suppl">supp_ddt403_ddt403supp_table1.doc</a><span style="color:gray"> (32K)</span></div><div class="small guid">GUID:&nbsp;BF3F1026-E65E-4CC9-823F-8A9B082DB1FC</div>
<div><a href="/pmc/articles/PMC3857947/bin/supp_ddt403_ddt403supp_table2.xls" data-ga-action="click_feat_suppl">supp_ddt403_ddt403supp_table2.xls</a><span style="color:gray"> (318K)</span></div><div class="small guid">GUID:&nbsp;D378944F-9337-4AEC-9822-DE10C708DF75</div>
<div><a href="/pmc/articles/PMC3857947/bin/supp_ddt403_ddt403supp_table3.xls" data-ga-action="click_feat_suppl">supp_ddt403_ddt403supp_table3.xls</a><span style="color:gray"> (111K)</span></div><div class="small guid">GUID:&nbsp;F9EC346E-1145-4D3F-930E-BDF485246AE3</div>
<div><a href="/pmc/articles/PMC3857947/bin/supp_ddt403_ddt403supp_table4.xls" data-ga-action="click_feat_suppl">supp_ddt403_ddt403supp_table4.xls</a><span style="color:gray"> (47K)</span></div><div class="small guid">GUID:&nbsp;A4B2C15E-9FD8-4EE3-AE7B-3F694FE0D639</div>
<div><a href="/pmc/articles/PMC3857947/bin/supp_ddt403_ddt403supp_table5.doc" data-ga-action="click_feat_suppl">supp_ddt403_ddt403supp_table5.doc</a><span style="color:gray"> (47K)</span></div><div class="small guid">GUID:&nbsp;CCEC5DA4-21BE-4B34-AFE4-6C5EB9065AFD</div>
<div><a href="/pmc/articles/PMC3857947/bin/supp_ddt403_ddt403supp_table6.xls" data-ga-action="click_feat_suppl">supp_ddt403_ddt403supp_table6.xls</a><span style="color:gray"> (28K)</span></div><div class="small guid">GUID:&nbsp;5F7361C6-CBB0-446C-BD2F-64E9CF77411F</div>
</div></dd></dl></div><div id="idm140715891874128" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140715891874128title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">We present the analysis of a prospective multicentre study to investigate genetic effects on the prognosis of newly treated epilepsy. Patients with a new clinical diagnosis of epilepsy requiring medication were recruited and followed up prospectively. The clinical outcome was defined as freedom from seizures for a minimum of 12 months in accordance with the consensus statement from the International League Against Epilepsy (ILAE). Genetic effects on remission of seizures after starting treatment were analysed with and without adjustment for significant clinical prognostic factors, and the results from each cohort were combined using a fixed-effects meta-analysis. After quality control (QC), we analysed 889 newly treated epilepsy patients using 472 450 genotyped and 6.9 × 10<sup>6</sup> imputed single-nucleotide polymorphisms. Suggestive evidence for association (defined as <em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−7</sup>) with remission of seizures after starting treatment was observed at three loci: 6p12.2 (rs492146, <em>P</em><sub>meta</sub> = 2.1 × 10<sup>−7</sup>, OR[G] = 0.57), 9p23 (rs72700966, <em>P</em><sub>meta</sub> = 3.1 × 10<sup>−7</sup>, OR[C] = 2.70) and 15q13.2 (rs143536437, <em>P</em><sub>meta</sub> = 3.2 × 10<sup>−7</sup>, OR[C] = 1.92). Genes of biological interest at these loci include <em>PTPRD</em> and <em>ARHGAP11B</em> (encoding functions implicated in neuronal development) and <em>GSTA4</em> (a phase II biotransformation enzyme). Pathway analysis using two independent methods implicated a number of pathways in the prognosis of epilepsy, including KEGG categories ‘calcium signaling pathway’ and ‘phosphatidylinositol signaling pathway’. Through a series of power curves, we conclude that it is unlikely any single common variant explains &gt;4.4% of the variation in the outcome of newly treated epilepsy.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p4" class="p p-first">In clinical epidemiology, prognosis refers to the future course and outcome of a disease. A notable aspect of the epilepsies is their highly variable prognosis, even among individuals with the same seizure types and epilepsy syndrome. Approximately 60% of people with epilepsy achieve long-term remission of seizures very shortly after starting antiepileptic drug (AED) treatment, while 20–30% have a chronic disorder without ever experiencing significant periods of remission (<a href="#DDT403C1" rid="DDT403C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013746">1</a>). Seizure control is an important factor in minimizing the risk of death from epilepsy, and remission of seizures is associated with improvements in quality of life (<a href="#DDT403C2" rid="DDT403C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013729">2</a>). Therefore, a key issue for clinical practice in epilepsy, and the development of new therapeutic approaches, is the extent to which genetic variation contributes to variation in treatment response. While a great deal is becoming known about genetic susceptibility to epilepsy (<a href="#DDT403C3" rid="DDT403C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013728">3</a>), very little is known about genetic influences on the prognosis of epilepsy, and to date, genetic effects on epilepsy prognosis are unexplored at a genome-wide level.</p><p id="__p5">For most pharmacogenetic research, attempts to identify genetic factors governing individual response to treatment are founded on a usually untested assumption—namely that genetically determined individual responses exist (<a href="#DDT403C4" rid="DDT403C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013738">4</a>). For epilepsy however, the clinical observation that therapeutic response to the first AED predicts response to subsequent AEDs (<a href="#DDT403C5" rid="DDT403C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013695">5</a>) supports the presence of individual effects on broad treatment response. Twin studies suggest that such individual effects on the outcome of treated epilepsy are mediated, at least in part, by epilepsy genetic susceptibility factors (<a href="#DDT403C6" rid="DDT403C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013701">6</a>).</p><p id="__p6">From a mechanistic point of view, genetic effects on the prognosis of newly treated epilepsy can be envisaged to operate on a number of levels including effects on inherent disease severity (<a href="#DDT403C7" rid="DDT403C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013741">7</a>), or via pharmacodynamic (PD) or pharmacokinetic (PK) mechanisms of pharmacological effect (<a href="#DDT403C8" rid="DDT403C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013698">8</a>) (Fig.&nbsp;<a href="/pmc/articles/PMC3857947/figure/DDT403F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT403F1" rid-ob="ob-DDT403F1" co-legend-rid="lgnd_DDT403F1"><span>1</span></a>). Determining the precise mechanism of effect for any given genetic association with epilepsy prognosis requires downstream experimental investigation, but from an epidemiological perspective, genetic associations with epilepsy prognosis via any mechanism may have important implications for the development of new therapeutic approaches and could contribute increased precision to prediction of AED response.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT403F1" co-legend-rid="lgnd_DDT403F1"><a href="/pmc/articles/PMC3857947/figure/DDT403F1/" target="figure" rid-figpopup="DDT403F1" rid-ob="ob-DDT403F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715889598848" class="figure"><a href="/pmc/articles/PMC3857947/figure/DDT403F1/" target="figure" rid-figpopup="DDT403F1" rid-ob="ob-DDT403F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3857947_ddt40301.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddt40301.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3857947/bin/ddt40301.jpg"></a></div><div id="largeobj_idm140715889598848" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3857947/figure/DDT403F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT403F1"><div><a class="figpopup" href="/pmc/articles/PMC3857947/figure/DDT403F1/" target="figure" rid-figpopup="DDT403F1" rid-ob="ob-DDT403F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p7">Mechanistic pathways for prognosis of newly treated epilepsy. PK, pharmacokinetic; PD, pharmacodynamic.</p></div></div></div><p id="__p8">Another important consideration in the investigation of genetic effects on the prognosis of epilepsy is whether the study should be conducted in the retrospective case–control setting or using a prospective cohort design. While the retrospective case–control design has been the standard approach for disease susceptibility genome-wide association studies (GWASs), in the study of disease prognosis, the prospective cohort design confers a number of important advantages. These include the ability to characterize clinical risk factors before treatment is initiated (mitigating concerns regarding the retrospective ascertainment of exposure), improved accuracy of measurement of clinical exposure and outcome, the ability to minimize bias in the selection of cases and controls, improved understanding of gene-exposure interactions and improved accuracy of predictive modelling (<a href="#DDT403C9" rid="DDT403C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013707">9</a>,<a href="#DDT403C10" rid="DDT403C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013706">10</a>). These strengths led us to adopt a prospective cohort design for our study despite its disadvantages in terms of time duration and cost compared with the retrospective case–control design.</p><p id="__p9" class="p p-last">Here, we report the first GWAS of prognosis of epilepsy using two independent, prospective cohorts of newly treated epilepsy. We report single SNP association <em>P-</em>values for each cohort and total evidence from a meta-analysis of the two cohorts. In addition, we sought evidence that particular classes of biological pathways are associated with epilepsy prognosis, which is an important next step in translating GWAS information to knowledge of disease processes underlying prognosis of epilepsy, as well as the development of future multi-genic predictors for use in clinical settings.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p10" class="p p-first">Patients with a new clinical diagnosis of epilepsy requiring medical treatment were recruited to independent prospective cohorts of newly treated epilepsy in the UK and Australia. The UK cohort consisted of 916 subjects who participated in the Standard and New AED (SANAD) trial (<a href="#DDT403C11" rid="DDT403C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013697">11</a>,<a href="#DDT403C12" rid="DDT403C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013721">12</a>). The Australian (AUS) cohort consisted of 380 subjects recruited from epilepsy clinics at two hospitals in Australia; the Royal Melbourne Hospital and the Austin Hospital in Victoria. The distribution of clinical characteristics for all subjects and for those included in the GWAS is detailed in Table&nbsp;<a href="/pmc/articles/PMC3857947/table/DDT403TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT403TB1" rid-ob="ob-DDT403TB1" co-legend-rid=""><span>1</span></a>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT403TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p11">Baseline patient demographics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715900052992" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">&nbsp;</th><th colspan="2" rowspan="1" align="left">UK cohort<hr></th><th colspan="2" rowspan="1" align="left">AUS cohort<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">ALL</th><th rowspan="1" colspan="1" align="left">GWAS</th><th rowspan="1" colspan="1" align="left">ALL</th><th rowspan="1" colspan="1" align="left">GWAS</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Total patients</td><td rowspan="1" colspan="1">916</td><td rowspan="1" colspan="1">654</td><td rowspan="1" colspan="1">380</td><td rowspan="1" colspan="1">235</td></tr><tr><td colspan="5" rowspan="1">One-year remission</td></tr><tr><td rowspan="1" colspan="1"> Yes</td><td rowspan="1" colspan="1">562 (61.4)</td><td rowspan="1" colspan="1">436 (66.7)</td><td rowspan="1" colspan="1">244 (64.2)</td><td rowspan="1" colspan="1">188 (80.0)</td></tr><tr><td rowspan="1" colspan="1"> No</td><td rowspan="1" colspan="1">342 (37.3)</td><td rowspan="1" colspan="1">218 (33.3)</td><td rowspan="1" colspan="1">96 (25.3)</td><td rowspan="1" colspan="1">47 (20.0)</td></tr><tr><td rowspan="1" colspan="1"> Not available (exclude)</td><td rowspan="1" colspan="1">12 (1.3)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">40 (10.5)</td><td rowspan="1" colspan="1">0</td></tr><tr><td colspan="5" rowspan="1">Gender</td></tr><tr><td rowspan="1" colspan="1"> Male</td><td rowspan="1" colspan="1">499 (54.5)</td><td rowspan="1" colspan="1">346 (52.9)</td><td rowspan="1" colspan="1">210 (55.3)</td><td rowspan="1" colspan="1">120 (51.1)</td></tr><tr><td rowspan="1" colspan="1"> Female</td><td rowspan="1" colspan="1">417 (45.5)</td><td rowspan="1" colspan="1">308 (47.1)</td><td rowspan="1" colspan="1">170 (44.7)</td><td rowspan="1" colspan="1">115 (48.9)</td></tr><tr><td rowspan="1" colspan="1"> Age at treatment in years, mean (IQR)</td><td rowspan="1" colspan="1">39 (23–53)</td><td rowspan="1" colspan="1">39 (22–53)</td><td rowspan="1" colspan="1">41 (25–55)</td><td rowspan="1" colspan="1">43 (26–56)</td></tr><tr><td colspan="5" rowspan="1">Neurological impairment</td></tr><tr><td rowspan="1" colspan="1"> Yes</td><td rowspan="1" colspan="1">69 (7.5)</td><td rowspan="1" colspan="1">37 (5.7)</td><td rowspan="1" colspan="1">9 (2.4)</td><td rowspan="1" colspan="1">7 (3.0)</td></tr><tr><td rowspan="1" colspan="1"> No</td><td rowspan="1" colspan="1">847 (92.5)</td><td rowspan="1" colspan="1">617 (94.3)</td><td rowspan="1" colspan="1">264 (69.5)</td><td rowspan="1" colspan="1">228 (97.0)</td></tr><tr><td rowspan="1" colspan="1"> Not available (exclude)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">107 (28.2)</td><td rowspan="1" colspan="1">0</td></tr><tr><td colspan="5" rowspan="1">Number of seizures ever before treatment</td></tr><tr><td rowspan="1" colspan="1"> One or not available (exclude)</td><td rowspan="1" colspan="1">2 (0.2)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">53 (13.9)</td><td rowspan="1" colspan="1">0</td></tr><tr><td rowspan="1" colspan="1"> 2</td><td rowspan="1" colspan="1">113 (12.3)</td><td rowspan="1" colspan="1">93 (14.2)</td><td rowspan="1" colspan="1">88 (23.2)</td><td rowspan="1" colspan="1">64 (27.2)</td></tr><tr><td rowspan="1" colspan="1"> 3</td><td rowspan="1" colspan="1">88 (9.6)</td><td rowspan="1" colspan="1">71 (10.9)</td><td rowspan="1" colspan="1">40 (10.5)</td><td rowspan="1" colspan="1">31 (13.2)</td></tr><tr><td rowspan="1" colspan="1"> 4</td><td rowspan="1" colspan="1">67 (7.3)</td><td rowspan="1" colspan="1">57 (8.7)</td><td rowspan="1" colspan="1">25 (6.6)</td><td rowspan="1" colspan="1">16 (6.8)</td></tr><tr><td rowspan="1" colspan="1"> 5</td><td rowspan="1" colspan="1">35 (3.8)</td><td rowspan="1" colspan="1">26 (4.0)</td><td rowspan="1" colspan="1">12 (3.2)</td><td rowspan="1" colspan="1">10 (4.3)</td></tr><tr><td rowspan="1" colspan="1"> &gt;5</td><td rowspan="1" colspan="1">611 (66.7)</td><td rowspan="1" colspan="1">407 (62.2)</td><td rowspan="1" colspan="1">162 (42.6)</td><td rowspan="1" colspan="1">114 (48.5)</td></tr><tr><td colspan="5" rowspan="1">Epilepsy type</td></tr><tr><td rowspan="1" colspan="1"> Generalized</td><td rowspan="1" colspan="1">140 (15.3)</td><td rowspan="1" colspan="1">107 (16.4)</td><td rowspan="1" colspan="1">73 (19.2)</td><td rowspan="1" colspan="1">41 (17.4)</td></tr><tr><td rowspan="1" colspan="1"> Focal</td><td rowspan="1" colspan="1">647 (70.6)</td><td rowspan="1" colspan="1">455 (69.6)</td><td rowspan="1" colspan="1">273 (71.8)</td><td rowspan="1" colspan="1">185 (78.7)</td></tr><tr><td rowspan="1" colspan="1"> Unclassified</td><td rowspan="1" colspan="1">125 (13.6)</td><td rowspan="1" colspan="1">92 (14.1)</td><td rowspan="1" colspan="1">18 (4.7)</td><td rowspan="1" colspan="1">9 (3·8)</td></tr><tr><td rowspan="1" colspan="1"> Not available</td><td rowspan="1" colspan="1">4 (0·4)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">16 (4.2)</td><td rowspan="1" colspan="1">0</td></tr><tr><td colspan="5" rowspan="1">EEG results</td></tr><tr><td rowspan="1" colspan="1"> Normal/non-specific abnormality</td><td rowspan="1" colspan="1">606 (66.2)</td><td rowspan="1" colspan="1">431 (65.9)</td><td rowspan="1" colspan="1">229 (60.3)</td><td rowspan="1" colspan="1">152 (64.7)</td></tr><tr><td rowspan="1" colspan="1"> Epileptiform abnormality</td><td rowspan="1" colspan="1">238 (26.0)</td><td rowspan="1" colspan="1">174 (26.6)</td><td rowspan="1" colspan="1">142 (37.4)</td><td rowspan="1" colspan="1">81 (34.5)</td></tr><tr><td rowspan="1" colspan="1"> Not done</td><td rowspan="1" colspan="1">72 (7.9)</td><td rowspan="1" colspan="1">49 (7.5)</td><td rowspan="1" colspan="1">9 (2.4)</td><td rowspan="1" colspan="1">2 (0.9)</td></tr><tr><td colspan="5" rowspan="1">CT/MRI results</td></tr><tr><td rowspan="1" colspan="1"> Normal</td><td rowspan="1" colspan="1">519 (56.3)</td><td rowspan="1" colspan="1">381 (58.2)</td><td rowspan="1" colspan="1">279 (73.4)</td><td rowspan="1" colspan="1">185 (78.2)</td></tr><tr><td rowspan="1" colspan="1"> Abnormal</td><td rowspan="1" colspan="1">186 (20.3)</td><td rowspan="1" colspan="1">119 (18.2)</td><td rowspan="1" colspan="1">84 (22.1)</td><td rowspan="1" colspan="1">46 (20.2)</td></tr><tr><td rowspan="1" colspan="1"> Not done</td><td rowspan="1" colspan="1">211 (23.0)</td><td rowspan="1" colspan="1">154 (23.5)</td><td rowspan="1" colspan="1">17 (4.5)</td><td rowspan="1" colspan="1">4</td></tr><tr><td colspan="5" rowspan="1">Initial AED treatment</td></tr><tr><td rowspan="1" colspan="1">  CBZ</td><td rowspan="1" colspan="1">159 (17.4)</td><td rowspan="1" colspan="1">107 (16.4)</td><td rowspan="1" colspan="1">172 (45.3)</td><td rowspan="1" colspan="1">111 (47.2)</td></tr><tr><td rowspan="1" colspan="1"> Gabapentin (GBP)</td><td rowspan="1" colspan="1">156 (17.0)</td><td rowspan="1" colspan="1">105 (16.1)</td><td rowspan="1" colspan="1">1 (0.3)</td><td rowspan="1" colspan="1">0</td></tr><tr><td rowspan="1" colspan="1"> Levetiracetam (LEV)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">25 (6·6)</td><td rowspan="1" colspan="1">18 (7·7)</td></tr><tr><td rowspan="1" colspan="1"> Lamotrigine (LTG)</td><td rowspan="1" colspan="1">208 (22.7)</td><td rowspan="1" colspan="1">152 (23.2)</td><td rowspan="1" colspan="1">21 (5.5)</td><td rowspan="1" colspan="1">16 (6.8)</td></tr><tr><td rowspan="1" colspan="1"> Phenytoin (PHT)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">14 (3.7)</td><td rowspan="1" colspan="1">8 (3.4)</td></tr><tr><td rowspan="1" colspan="1"> Zonisamide (ZNS)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">4 (1.1)</td><td rowspan="1" colspan="1">3 (1.3)</td></tr><tr><td rowspan="1" colspan="1"> Oxcarbazepine (OXC)</td><td rowspan="1" colspan="1">99 (10.8)</td><td rowspan="1" colspan="1">70 (10.7)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td></tr><tr><td rowspan="1" colspan="1"> Topiramate (TPM)</td><td rowspan="1" colspan="1">225 (24.6)</td><td rowspan="1" colspan="1">165 (25.2)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td></tr><tr><td rowspan="1" colspan="1"> Sodium valproate (VPS)</td><td rowspan="1" colspan="1">69 (7.5)</td><td rowspan="1" colspan="1">55 (8.4)</td><td rowspan="1" colspan="1">141 (37.1)</td><td rowspan="1" colspan="1">79 (33.6)</td></tr><tr><td rowspan="1" colspan="1"> Not available</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">2 (0.5)</td><td rowspan="1" colspan="1">0</td></tr></tbody></table></div><div id="largeobj_idm140715900052992" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3857947/table/DDT403TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140715902467632"><p id="__p12" class="p p-first-last">Values in the table are actual number with percentages in brackets<em>.</em> ALL refers to all patients recruited to the study, and GWAS to those patients included in the genome-wide association study.</p></div></div></div><p id="__p13">The recent consensus statement from the International League Against Epilepsy (ILAE) proposes that treatment success in epilepsy should be defined as freedom from seizures for a minimum of 12 months, as this outcome is consistently associated with improved quality of life (<a href="#DDT403C13" rid="DDT403C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013718">13</a>). Therefore, patients achieving 12-month (365 days or longer) remission of seizures were defined as “responders”, and patients failing to achieve 12-month remission were defined as “non-responders”. Patients followed for &lt;1 year were excluded from the study. A potential difficulty with this outcome is that it is not possible to know whether non-responders might become responders if followed for long enough. However, based on the empirical distribution of time to 12-month remission for patients achieving 12-month remission (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>), we observed that the likelihood of remission falls sharply with time, such that for all patients in the study who achieved 12-month remission, 90% did so within 2.03 years of starting AED therapy. As the median follow-up of non-responders in our study was 2·4 years (IQR 1.7–3.6 years), the number of patients misclassified due to inadequate follow-up is expected to be small.</p><p id="__p14">To date, no validated genetic association with epilepsy prognosis has been reported. However, a number of clinical factors such as the number of seizures pre-treatment and to a lesser extent the results of electroencephalographic (EEG) and brain-imaging investigations have been shown to be associated with chance of remission of seizures after starting treatment (<a href="#DDT403C14" rid="DDT403C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013747">14</a>). We reasoned that if genetic factors influence epilepsy prognosis via clinical factors associated with prognosis, then adjustment for clinical prognostic factors may result in no evidence for genetic association, as in the case of the metabolic trait fasting glucose and the <em>FTO</em> gene, where there is no evidence for association when adjusting for the body mass index (<a href="#DDT403C15" rid="DDT403C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013737">15</a>). Conversely, failure to adjust for clinical prognostic factors may reduce the power to detect genetic associations. A comprehensive analysis of genetic influences on remission of seizures after starting AED treatment therefore requires statistical analysis with and without adjustment for clinical factors informative for epilepsy prognosis.</p><p id="__p15">We identified clinical factors informative for epilepsy prognosis using a univariate logistic regression model. Univariate odds ratios (ORs) for association of clinical factors with 12-month remission of seizures in the UK and AUS cohorts are shown in Table&nbsp;<a href="/pmc/articles/PMC3857947/table/DDT403TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT403TB2" rid-ob="ob-DDT403TB2" co-legend-rid=""><span>2</span></a>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT403TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p16">Univariate ORs and 95% CIs for 12-month remission of seizures in newly treated epilepsy</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715893831280" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">&nbsp;</th><th colspan="2" rowspan="1" align="left">UK cohort (ALL)<hr></th><th colspan="2" rowspan="1" align="left">AUS cohort (ALL)<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P-</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Age at treatment (per ten years)</td><td rowspan="1" colspan="1">1.12 (1.03–1.22)</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.08 (0.91–1.27)</td><td rowspan="1" colspan="1">0.39</td></tr><tr><td rowspan="1" colspan="1">Sex (female)</td><td rowspan="1" colspan="1">1.35 (0.98–1.87)</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">1.24 (0.65–2.35)</td><td rowspan="1" colspan="1">0.52</td></tr><tr><td colspan="5" rowspan="1">Epilepsy type (focal)</td></tr><tr><td rowspan="1" colspan="1"> Generalized</td><td rowspan="1" colspan="1">1.73 (1.05–2.85)</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.66 (0.65–4.22)</td><td rowspan="1" colspan="1">0.29</td></tr><tr><td rowspan="1" colspan="1"> Unclassified</td><td rowspan="1" colspan="1">1.73 (1.07–2.78)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">(0-Inf)</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1">Neurological impairment (no)</td><td rowspan="1" colspan="1">0.45 (0.23–0.87)</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.52 (0.18–12.9)</td><td rowspan="1" colspan="1">0.70</td></tr><tr><td colspan="5" rowspan="1">Number seizures before treatment (&gt;5)</td></tr><tr><td rowspan="1" colspan="1"> 2</td><td rowspan="1" colspan="1">3.20 (1.83–5.60)</td><td rowspan="1" colspan="1">0.00005</td><td rowspan="1" colspan="1">2.78 (1.14–6.77)</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1"> 3</td><td rowspan="1" colspan="1">3.12 (1.66–5.88)</td><td rowspan="1" colspan="1">0.0004</td><td rowspan="1" colspan="1">1.42 (0.53–3.81)</td><td rowspan="1" colspan="1">0.48</td></tr><tr><td rowspan="1" colspan="1"> 4</td><td rowspan="1" colspan="1">1.38 (0.77–2.47)</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">0.71 (0.24–2.34)</td><td rowspan="1" colspan="1">0.62</td></tr><tr><td rowspan="1" colspan="1"> 5</td><td rowspan="1" colspan="1">3.79 (1.28–11.19)</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">(0-Inf)</td><td rowspan="1" colspan="1">0.98</td></tr><tr><td colspan="5" rowspan="1">EEG results (normal)</td></tr><tr><td rowspan="1" colspan="1"> Epileptiform abnormality</td><td rowspan="1" colspan="1">1.07 (0.73–1.56)</td><td rowspan="1" colspan="1">0.72</td><td rowspan="1" colspan="1">1.12 (0.57–2.24)</td><td rowspan="1" colspan="1">0.73</td></tr><tr><td rowspan="1" colspan="1"> Not done</td><td rowspan="1" colspan="1">0.46 (0.25–0.83)</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.26 (0.02–4.21)</td><td rowspan="1" colspan="1">0.34</td></tr><tr><td colspan="5" rowspan="1">CT/MR results (normal)</td></tr><tr><td rowspan="1" colspan="1"> Abnormal</td><td rowspan="1" colspan="1">0.88 (0.60–1.30)</td><td rowspan="1" colspan="1">0.51</td><td rowspan="1" colspan="1">0.42 (0.20–0.86)</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1"> Not done</td><td rowspan="1" colspan="1">1.28 (0.82–2.01)</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">0.60 (0.06–6.00)</td><td rowspan="1" colspan="1">0.67</td></tr><tr><td colspan="5" rowspan="1">AED treatment (LTG)</td></tr><tr><td rowspan="1" colspan="1"> OXC</td><td rowspan="1" colspan="1">0.75 (0.41–1.34)</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td></tr><tr><td rowspan="1" colspan="1"> VPS/VPA</td><td rowspan="1" colspan="1">1.64 (0.79–3.38)</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">1.17 (0.29–4.70)</td><td rowspan="1" colspan="1">0.82</td></tr><tr><td rowspan="1" colspan="1"> CBZ</td><td rowspan="1" colspan="1">0.94 (0.56–1.59)</td><td rowspan="1" colspan="1">0.82</td><td rowspan="1" colspan="1">0.68 (0.18–2.58)</td><td rowspan="1" colspan="1">0.57</td></tr><tr><td rowspan="1" colspan="1"> TPM</td><td rowspan="1" colspan="1">1.10 (0.68–1.77)</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td></tr><tr><td rowspan="1" colspan="1"> GBP</td><td rowspan="1" colspan="1">0.54 (0.32–0.90)</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td></tr><tr><td rowspan="1" colspan="1"> LEV</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">3.92 (0.36–42.2)</td><td rowspan="1" colspan="1">0.26</td></tr><tr><td rowspan="1" colspan="1"> PHT</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">0.69 (0.09–5.29)</td><td rowspan="1" colspan="1">0.72</td></tr><tr><td rowspan="1" colspan="1"> ZNS</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">(0-Inf)</td><td rowspan="1" colspan="1">0.99</td></tr></tbody></table></div><div id="largeobj_idm140715893831280" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3857947/table/DDT403TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140715902340528"><p id="__p17" class="p p-first-last">For categorical covariates, ORs are relative to the reference state provided in brackets. In three cases, the number of occurrences of a state was too few to permit meaningful estimation of OR (shown as 0-Inf). Covariate states nominally significant in the UK cohort were included in the adjusted GWAS.</p></div></div></div><p id="__p18">Clinical prognostic factors chosen for inclusion in the GWAS based on the significance at <em>P</em> &lt; 0.05 in the UK cohort were as follows: age at starting treatment, number of seizures before treatment, EEG result, epilepsy type, presence or absence of neurological impairment and treatment with gabapentin (GBP).</p><p id="__p19">After sample and genotyping quality control (QC, Materials and Methods), a total of 889 newly treated epilepsy patients followed for at least 1 year and for whom remission status and complete clinical covariate information was available were included in the analysis. The duration of follow-up is reported in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>. There was no difference in proportions of responders and non-responders among patients included or excluded in the GWAS (<em>P</em> = 0.91, 0.75 and 1.0 for UK, AUS and combined, respectively).</p><p id="__p20">Association analysis of 12-month remission of seizures was performed for the UK and AUS cohorts separately, and the results from each cohort were combined using a fixed-effect meta-analysis (Materials and Methods). The genomic inflation factors for the two pairs of GWAS and their meta-analyses were UK<sub>Adjusted</sub> = 1.01, UK<sub>Unadjusted</sub> = 1.00, AUS<sub>Adjusted</sub> = 1.06, AUS<sub>Unadjusted</sub> = 1.01, Meta<sub>Adjusted</sub> = 0.98, Meta<sub>Unadjusted</sub> = 0.98. Quantile–quantile (QQ) plots of the expected versus observed <em>P</em>-value distributions for these are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S3</a>. Manhattan plots of –log<sub>10</sub>-transformed <em>P-</em>values from the meta-analyses with and without adjustment for significant clinical prognostic factors are shown in Figure&nbsp;<a href="/pmc/articles/PMC3857947/figure/DDT403F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT403F2" rid-ob="ob-DDT403F2" co-legend-rid="lgnd_DDT403F2"><span>2</span></a>.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT403F2" co-legend-rid="lgnd_DDT403F2"><a href="/pmc/articles/PMC3857947/figure/DDT403F2/" target="figure" rid-figpopup="DDT403F2" rid-ob="ob-DDT403F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715888055024" class="figure"><a href="/pmc/articles/PMC3857947/figure/DDT403F2/" target="figure" rid-figpopup="DDT403F2" rid-ob="ob-DDT403F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3857947_ddt40302.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddt40302.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3857947/bin/ddt40302.jpg"></a></div><div id="largeobj_idm140715888055024" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3857947/figure/DDT403F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT403F2"><div><a class="figpopup" href="/pmc/articles/PMC3857947/figure/DDT403F2/" target="figure" rid-figpopup="DDT403F2" rid-ob="ob-DDT403F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p21">Plot of –log10 transformed <em>P</em>-values of SNP associations with 12-month remission of seizures from the meta-analysis of the UK and AUS cohorts. Top, unadjusted for significant clinical prognostic factors; bottom, adjusted for significant clinical prognostic factors. Coloured dots correspond to genotyped SNPs and grey dots imputed SNPs. The dashed horizontal line marks a <em>P</em>-value significance threshold of 5.0 × 10<sup>−7</sup>.</p></div></div></div><p id="__p22"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>, reports all SNPs with associated <em>P</em>-values &lt;1.0 × 10<sup>−4</sup> from the meta-analyses. No variant achieved genome-wide significance (defined as <em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−8</sup>). Two loci (indexed by rs492146 and rs72700966) showed suggestive evidence (defined as <em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−7</sup>) for association in the unadjusted analysis, and a third locus (indexed by rs143536437) showed suggestive evidence for association in the adjusted analysis (Table&nbsp;<a href="/pmc/articles/PMC3857947/table/DDT403TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT403TB3" rid-ob="ob-DDT403TB3" co-legend-rid=""><span>3</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT403TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p23">Loci associated with remission of seizures in newly treated epilepsy at <em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−7</sup> (with and without adjusting for clinical prognostic factors)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715890040400" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Index SNP</th><th rowspan="2" colspan="1" align="left">Imp</th><th rowspan="2" colspan="1" align="left">Gene</th><th rowspan="2" colspan="1" align="left">Chr</th><th rowspan="2" colspan="1" align="left">Position (BP)</th><th rowspan="2" colspan="1" align="left">Test allele</th><th rowspan="2" colspan="1" align="left">Ref allele</th><th rowspan="2" colspan="1" align="left">Test allele freq. (UK)</th><th rowspan="2" colspan="1" align="left">Test allele freq. (AUS)</th><th colspan="4" rowspan="1" align="left">Adjusted for clinical prognostic factors<hr></th><th colspan="4" rowspan="1" align="left">Unadjusted for clinical prognostic factors<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">UK <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">AUS <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Meta <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">OR (meta)</th><th rowspan="1" colspan="1" align="left">UK <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">AUS <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Meta <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">OR (meta)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs492146</td><td rowspan="1" colspan="1">Yes<sup>a,b</sup></td><td rowspan="1" colspan="1"><em>GSTA4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">52835895</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">2.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.007</td><td rowspan="1" colspan="1">7.9 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.56</td><td rowspan="1" colspan="1">5.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">2.1 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.57</td></tr><tr><td rowspan="1" colspan="1">rs72700966</td><td rowspan="1" colspan="1">Yes</td><td rowspan="1" colspan="1"><em>PTPRD</em></td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">10505224</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">3.7 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1">9.1 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">2.8</td><td rowspan="1" colspan="1">8.2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">3.1 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">2.7</td></tr><tr><td rowspan="1" colspan="1">rs143536437</td><td rowspan="1" colspan="1">Yes<sup>b</sup></td><td rowspan="1" colspan="1"><em>ARHGAP11B</em></td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">30898332</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.72</td><td rowspan="1" colspan="1">0.74</td><td rowspan="1" colspan="1">4.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">3.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">1.9</td><td rowspan="1" colspan="1">2.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.035</td><td rowspan="1" colspan="1">2.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">1.73</td></tr></tbody></table></div><div id="largeobj_idm140715890040400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3857947/table/DDT403TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140715891034592"><p id="__p24" class="p p-first-last">SNP, single nucleotide polymorphism; Chr, chromosome; freq, frequency; OR, odds ratio; bp, base-pair; Imp, imputed.</p></div><div id="idm140715891033840"><p id="__p25" class="p p-first-last"><sup>a</sup>Additional genotyped SNPs also exceeded the 5·0 × 10<sup>−7</sup> significant threshold.</p></div><div id="idm140715902974144"><p id="__p26" class="p p-first-last"><sup>b</sup>Association present in both the UK and AUS cohorts independently.</p></div><div id="idm140715902973264"><p id="__p27" class="p p-first-last">MAF for responders and non-responders for each cohort (UK and AUS) for these SNPs and all SNPs with <em>P</em> &lt; 1.0 × 10<sup>−4</sup> are given in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>.</p></div></div></div><p id="__p28">A further eight previously unreported loci were tentatively (defined as <em>P</em> &lt; 1.0 × 10<sup>−5</sup>) associated with epilepsy prognosis (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). The full list of SNP IDs and associated <em>P</em>-values are available online at <a href="https://www.gwascentral.org/">https://www.gwascentral.org/</a>.</p><p id="__p29">The two loci indexed by rs492146 and rs143536437 were independently associated in both the UK and AUS cohorts, fulfilling the Wellcome Trust Case Control Consortium 2 (WTCCC2) criteria for ‘strong’ evidence for association (<a href="#DDT403C16" rid="DDT403C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013696">16</a>), while rs72700966 was only significant in the UK cohort (although the observed direction of effect was the same in AUS as that in the UK study).</p><p id="__p30">All the three variants indexed by SNPs with <em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−7</sup> exhibited similar effect sizes in the adjusted and unadjusted analyses, suggesting that the associations were not mediated by the identified clinical prognostic factors. Regional association plots for the three loci showing strongest association with remission of seizures are shown in Figure&nbsp;<a href="/pmc/articles/PMC3857947/figure/DDT403F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT403F3" rid-ob="ob-DDT403F3" co-legend-rid="lgnd_DDT403F3"><span>3</span></a>.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT403F3" co-legend-rid="lgnd_DDT403F3"><a href="/pmc/articles/PMC3857947/figure/DDT403F3/" target="figure" rid-figpopup="DDT403F3" rid-ob="ob-DDT403F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715895550848" class="figure"><a href="/pmc/articles/PMC3857947/figure/DDT403F3/" target="figure" rid-figpopup="DDT403F3" rid-ob="ob-DDT403F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3857947_ddt40303.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddt40303.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3857947/bin/ddt40303.jpg"></a></div><div id="largeobj_idm140715895550848" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3857947/figure/DDT403F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT403F3"><div><a class="figpopup" href="/pmc/articles/PMC3857947/figure/DDT403F3/" target="figure" rid-figpopup="DDT403F3" rid-ob="ob-DDT403F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p31">Regional plots for loci associated with remission of seizures at <em>P</em> &lt; 5.0 × 10<sup>−7</sup>. SNPs are represented by diamonds and plotted by –log10 transformed <em>P</em>-value and genomic position. Estimated recombination rates are shown by the blue peaks, and gene annotations are indicated by green arrows. Plots are for the meta-analysis results adjusted (<em>ARHGAP11B</em>) and unadjusted (<em>GSTA4</em>, <em>PTPRD</em>) for clinical prognostic factors.</p></div></div></div><p id="__p32">In order to provide an approximate guide to the credibility of the three suggestive associations, we calculated a posterior probability of association (PPA) as described in Stephens and Balding (<a href="#DDT403C17" rid="DDT403C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013703">17</a>). The PPAs were calculated using the combined genotypic data for UK and AUS, without clinical covariates, assuming a normal prior for effect sizes and a 4:1 weighting of additive and general models. We assigned a prior probability of association of 10<sup>−4</sup>, corresponding to an assumption that ∼300 kb is tightly linked with a variant associated with remission of seizures in newly treated epilepsy. The PPAs obtained were 0.72 for rs492146, 0.16 for rs72700966 and 0.18 for rs143536437, suggesting that the first of these is more likely than not a true association, while the other two are less secure but have a non-negligible probability and are worthy of further consideration.</p><p id="__p33">Genes of biological interest at the three loci with index SNPs associated with epilepsy prognosis at <em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−7</sup> are considered briefly below:</p><p id="__p34">6p12.2 (rs492146): <em>GSTA4</em> encodes glutathione <em>S</em>-transferase (GST) alpha 4. GSTs are a superfamily of phase-II drug-metabolizing enzymes. The alpha class of GSTs encode enzymes with glutathione peroxidase activity that function in the detoxification of lipid peroxidation products and are implicated in the protection of neurons following injury (<a href="#DDT403C18" rid="DDT403C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013742">18</a>).</p><p id="__p35">9p23 (rs72700966): <em>PTPRD</em> encodes protein tyrosine phosphatase receptor type D. <em>PTPRD</em> is implicated in the regulation of synapse development and function (<a href="#DDT403C19" rid="DDT403C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013694">19</a>). Rare structural variants in <em>PTPRD</em> have been associated with attention deficit disorder (<a href="#DDT403C20" rid="DDT403C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013702">20</a>) and autism (<a href="#DDT403C21" rid="DDT403C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013739">21</a>) and ptprd−/− mice exhibit impaired spatial learning and enhanced long-term potentiation (<a href="#DDT403C22" rid="DDT403C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013724">22</a>). Variants in the 5′ UTR of PTPRD are associated with the neurological disorder restless legs syndrome (<a href="#DDT403C23" rid="DDT403C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013715">23</a>).</p><p id="__p36">15q13.2 (rs143536437): <em>ARHGAP11B</em> (alias <em>FAM7B1</em>) encodes rho GTPase activating protein 11B. Rho GTPases play an essential role in neuronal development (<a href="#DDT403C24" rid="DDT403C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013734">24</a>). <em>ARHGAP11B</em> is one of the seven genes (the others being <em>MTMR15</em>, <em>MTMR10</em>, <em>TRPM1</em>, <em>KLF13</em>, <em>OTUD7A</em> and <em>CHRNA7</em>) deleted in 15q13.3 microdeletion syndrome associated with mental retardation and refractory epilepsy (<a href="#DDT403C25" rid="DDT403C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013713">25</a>), and rare <em>ARHGAP11B</em> deletions are observed in autism spectrum disorder (<a href="#DDT403C26" rid="DDT403C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013725">26</a>).</p><p id="__p37">A focussed examination of 280 genes involved in drug absorption, distribution, metabolism and excretion (ADME) defined according to the PharmaADME database (<a href="#DDT403C27" rid="DDT403C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013730">27</a>), as well as ADME genes for AEDs (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>), revealed no significant association between these genes and prognosis of newly treated epilepsy other than <em>GSTA4</em>. Similarly, we observed no significant association between the outcome of newly treated epilepsy and the human leukocyte antigen genes or genes associated with epilepsy susceptibility by GWASs (<a href="#DDT403C28" rid="DDT403C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013704">28</a>,<a href="#DDT403C29" rid="DDT403C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013700">29</a>) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>).</p><p id="__p38">For the three loci indexed by SNPs below the <em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−7</sup> threshold, we investigated whether there was an interaction between the genotype and the treatment type by conditioning on each AED in turn (see Materials and Methods and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S4</a>). We first performed the analysis according to the initial AED treatment, but since patients may change AED during their course of treatment we repeated the analysis for the 355 patients who made no change in AED during the study. After adjusting for epilepsy type, we observed a nominally significant interaction between rs72700966 (<em>PTPRD</em>) and oxcarbazepine (<em>P</em> = 7.0 × 10<sup>−4</sup>) and between rs492146 (<em>GSTA4</em>) and topiramate (<em>P</em> = 0.04). However, only rs72700966 survived correction for multiple tests (<em>P</em> = 0.035), and the interaction results should be further interpreted with caution, given the small numbers in each treatment category.</p><p id="__p39">For all SNPs associated with prognosis of epilepsy at <em>P</em> &lt; 1.0 × 10<sup>−4</sup>, we tested whether the observed allele frequencies in UK and AUS cohorts were significantly different from those in the 381 European samples of the 1000 Genomes project (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). The results were not significant for rs72700966 (<em>P</em> = 0.53), and only nominally significant for rs492146 and rs143536437 (<em>P</em> = 0.01 and 0.02 respectively), providing little evidence that these variants may be epilepsy susceptibility SNPs. In keeping with this conclusion, none of the top three loci showed evidence for being epilepsy susceptibility SNPs in GWAS reporting loci associated with epilepsy at genome-wide significance (<a href="#DDT403C28" rid="DDT403C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013740">28</a>,<a href="#DDT403C29" rid="DDT403C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013714">29</a>).</p><p id="__p40">We estimated the power of our study to detect genetic associations with prognosis of newly treated epilepsy following the methodology of Bacanu <em>et al.</em> (<a href="#DDT403C30" rid="DDT403C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013712">30</a>), modified for an additive test. Our power calculations (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S5</a>) indicated that we had 80% power to detect causal variants at genome-wide significance (<em>P</em> &lt; 5.0 × 10<sup>−8</sup>) which individually explained ≥4.4% of the variance of outcome of newly treated epilepsy, and 50% power for variants explaining ≥3.3% of the variance. The findings from our GWAS therefore suggest that there are unlikely to exist common variants that individually have a strong influence on a patient's likelihood of achieving remission after starting AED treatment.</p><p id="__p41">Our results do not exclude a model in which epilepsy prognosis is a polygenic trait of multiple common variants of small effect. Under such a model, sets of genes representing causal biological pathways may be enriched among genes with moderate association <em>P</em>-values. To date, pathway analysis of prognosis of epilepsy has been unexplored, and pathway analysis is an under-utilized approach to analysing treatment response phenotypes in general (<a href="#DDT403C31" rid="DDT403C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013736">31</a>). Yet such an approach might inform biological processes underling disease prognosis, and act as a starting point for the development of multi-SNP predictors of outcome. We therefore sought evidence from the GWAS that particular classes of biological pathways are associated with epilepsy prognosis using two independent pathway methods, ALIGATOR (<a href="#DDT403C32" rid="DDT403C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013711">32</a>,<a href="#DDT403C33" rid="DDT403C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013744">33</a>) and pathMaster (<a href="#DDT403C34" rid="DDT403C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013723">34</a>) (Materials and Methods). ALIGATOR and pathMaster methods can be considered complementary; rather than testing for enrichment of significant genes within pathways as ALIGATOR does, pathMaster tests the absolute association between a pathway and an outcome by aggregating all information within a pathway into a single test statistic. Kyoto Encyclopedia of Genes and Genomes (KEGG) and Gene Ontology (GO) pathways associated with epilepsy prognosis by both ALIGATOR and pathMaster are reported in Table&nbsp;<a href="/pmc/articles/PMC3857947/table/DDT403TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT403TB4" rid-ob="ob-DDT403TB4" co-legend-rid=""><span>4</span></a>, with the full results from either method given in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>. GWAS <em>P</em>-value thresholds for ALIGATOR were chosen in accordance with the summary statistics for ALIGATOR (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT403TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p42">Functional categories significantly enriched for genes associated with prognosis of newly treated epilepsy by both ALIGATOR and pathMaster</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715890939216" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">GWAS <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Category ID</th><th rowspan="1" colspan="1" align="left">Pathway length</th><th rowspan="1" colspan="1" align="left">Expected overlap</th><th rowspan="1" colspan="1" align="left">Observed overlap</th><th rowspan="1" colspan="1" align="left">ALIGATOR <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">pathMaster <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Biological function</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">0.0001</td><td rowspan="1" colspan="1">hsa04020</td><td rowspan="1" colspan="1">174</td><td rowspan="1" colspan="1">0.7</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">0.0052</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">Calcium signalling pathway</td></tr><tr><td rowspan="1" colspan="1">0.0001</td><td rowspan="1" colspan="1">hsa04070</td><td rowspan="1" colspan="1">78</td><td rowspan="1" colspan="1">0.3</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">0.043</td><td rowspan="1" colspan="1">Phosphatidylinositol signalling system</td></tr><tr><td rowspan="1" colspan="1">0.001</td><td rowspan="1" colspan="1">GO:0008270</td><td rowspan="1" colspan="1">209</td><td rowspan="1" colspan="1">5.3</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">0.0002</td><td rowspan="1" colspan="1">0.021</td><td rowspan="1" colspan="1">Zinc ion binding</td></tr><tr><td rowspan="1" colspan="1">0.001</td><td rowspan="1" colspan="1">GO:0046870</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">0.2</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">0.0002</td><td rowspan="1" colspan="1">0.045</td><td rowspan="1" colspan="1">Cadmium ion binding</td></tr><tr><td rowspan="1" colspan="1">0.0001</td><td rowspan="1" colspan="1">GO:0030262</td><td rowspan="1" colspan="1">27</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">0.001</td><td rowspan="1" colspan="1">0.001</td><td rowspan="1" colspan="1">Apoptotic nuclear change</td></tr></tbody></table></div><div id="largeobj_idm140715890939216" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3857947/table/DDT403TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140715899354912"><p id="__p43" class="p p-first-last">ALIGATOR and pathMaster <em>P</em>-value = pathway enrichment <em>P</em>-value.</p></div><div id="idm140715895234064"><p id="__p44" class="p p-first-last">Proposed biological functions were provided by the Gene Ontology (prefixed by GO) and KEGG databases (prefixed by hsa).</p></div><div id="idm140715895233424"><p id="__p45" class="p p-first-last">Pathway length, expected overlap and observed overlap refer to ALIGATOR statistics.</p></div></div></div><p id="__p46">Taken together, our results suggest that prognosis of newly treated epilepsy is potentially influenced by multiple genetic factors. In such situations, heritability analysis, when applied to distantly related individuals in a GWAS, can be used to estimate the total phenotypic variance of a trait by considering the variance explained by a linear mixed-effects model involving all SNPs (<a href="#DDT403C35" rid="DDT403C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013727">35</a>,<a href="#DDT403C36" rid="DDT403C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013717">36</a>). Using the software LDAK (<a href="#DDT403C36" rid="DDT403C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013705">36</a>), we computed kinship matrices for each cohort, both with and without adjustment for linkage disequilibrium, then using GCTA (<a href="#DDT403C35" rid="DDT403C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013709">35</a>) we estimated variance explained, both with and without adjustment for clinical prognostic factors. Unfortunately, the standard error of the heritability estimates were too high (a minimum of 50% for both UK and AUS cohorts) to provide reliable estimates of heritability.</p><p id="__p47" class="p p-last">Finally, for the three loci indexed by SNPs with <em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−7</sup>, we examined whether the SNPs could be tagging copy number variation (CNV) using cnvHap (<a href="#DDT403C37" rid="DDT403C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013749">37</a>). Several rare CNVs were identified in genes at or within 20 kb of the index SNPs (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>), none of which were associated with remission of seizures at significance ≤5%.</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p48" class="p p-first">To date, the focus of genetic efforts in treatment response to epilepsy has been on candidate gene studies. However, the biological mechanisms by which AEDs act are poorly understood, constraining candidate gene investigations to the existing knowledge base. In contrast, the genome-wide association method offers a hypothesis-free approach to systematically investigate genetic effects. Yet despite the advantages of the genome-wide approach in pharmacogenetics, fewer than 5% of GWASs catalogued by the US National Human Genome Research Institute are studies of treatment response (<a href="http://www.genome.gov/gwastudies/">http://www.genome.gov/gwastudies/</a>), perhaps reflecting the difficulties of studying prognosis as opposed to susceptibility.</p><p id="__p49">Here, we report the genome-wide association and biological pathway analysis of prognosis of newly treated epilepsy. Uniform baseline clinical characteristics were collected on all patients at study entry and seizure outcomes were measured prospectively. This prospective design overcomes many of the problematic aspects of analysing disease prognosis in the retrospective case–control setting, and permits an unbiased incorporation of clinical prognostic factors in the statistical analysis.</p><p id="__p50">How to define responders and non-responders is a key issue in any study of treatment response. We chose 12-month remission of seizures as the clinical outcome in accordance with the recent consensus statement from the ILAE and because this epilepsy outcome is consistently associated with improved quality of life (and in many countries, including the UK and Australia, is the minimum seizure-free period that allows a patient with epilepsy to drive legally). While this definition is not without limitations, international acceptance of an outcome definition is essential to facilitate replication and future meta-analysis, and external clinical validity is important for developing clinically relevant predictive models of response.</p><p id="__p51">We considered that the inclusion of clinical covariates informative for epilepsy prognosis could either help or hinder the detection of genetic effects on prognosis, depending on whether the genetic factors acted via one or more intermediate clinical prognostic factor. Therefore, we performed the analyses both with and without adjustment for clinical prognostic factors. This approach has the advantage of being agnostic about whether including clinical prognostic factors in the model can increase or decrease the power to detect genetic effects on outcome.</p><p id="__p52">In our study, no single SNP achieved the WTCCC2 cut-off for genome-wide significance (<em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−8</sup>), although suggestive evidence for association (<em>P</em><sub>meta</sub> &lt; 5.0 × 10<sup>−7</sup>) was observed for three loci, and calculation of posterior probabilities of association suggested that rs492146 is more likely than not a true association. Although our moderate sample size does limit our ability to detect variants of small effect, we have sufficient power to conclude it is unlikely that any single common variant explains &gt;4.4% of the variance of outcome of newly treated epilepsy.</p><p id="__p53">The finding that no single common variant has a major influence on the chance of remission of seizures in newly treated epilepsy is important, since it has been suggested that drug response phenotypes might be mediated by higher effect size common variants due to little negative evolutionary pressure on drug response variants. While large effect sized SNPs have been reported for hypersensitive drug reactions (e.g. <a href="#DDT403C38" rid="DDT403C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013710">38</a>), our study suggests that this may not be the case for drug efficacy phenotypes, where there is a more complex interplay between inherent disease severity and pharmacological effectiveness. Our results suggest that for epilepsy, the genetic architecture of treatment response more closely aligns with complex traits than expected, and so would be improved by the inclusion of additional samples. This conclusion is supported by the large standard error associated with our estimates of heritability, but to our knowledge, the cohorts reported here are the only genotyped prospective cohorts of newly treated epilepsy currently available worldwide.</p><p id="__p54">As a result of these insights, we reasoned that the analysis of sets of genes representing biological pathways may have greater power to detect genetic effects on the outcome of newly treated epilepsy. We used two methods of pathway analysis (ALIGATOR and pathMaster) that could be considered complimentary; ALIGATOR tests for enrichment of significant genes within pathways whereas pathMaster aggregates all information within pathways. A number of candidate pathways informative for epilepsy outcome were identified in both ALIGATOR and pathMaster analyses, including KEGG categories ‘calcium signaling pathway’ and ‘phosphatidylinositol signaling pathway,’ which may warrant further investigation.</p><p id="__p55" class="p p-last">In conclusion, the findings from our GWAS represent a first step in the comprehensive analysis of genetic effects on the prognosis of newly treated epilepsy. Our results suggest a limited role for common variants of strong effect and prompt efforts directed at increasing the sample size through additional prospective cohorts of newly treated epilepsy, the development of geneset analyses and exploring the role of rare variant effects in epilepsy outcome.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Patients</h3><p id="__p56" class="p p-first-last">Epilepsy patients were recruited to UK and Australian prospective cohorts of newly treated epilepsy. The UK cohort consisted of 916 patients who participated in the Standard and New AED (SANAD) trial (<a href="#DDT403C11" rid="DDT403C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013716">11</a>,<a href="#DDT403C12" rid="DDT403C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013722">12</a>). SANAD was a randomized, controlled trial consisting of two treatment arms. Arm A included patients for whom carbamazepine (CBZ) was considered the first-line treatment, most of whom had focal epilepsy. Patients in Arm A were randomly assigned to receive CBZ, GBP, lamotrigine, oxcarbazepine or topiramate. Arm B included patients for whom sodium valproate was considered the first-line treatment, most of whom had generalized epilepsy. Patients in Arm B were randomly assigned to receive sodium valproate, topiramate or lamotrigine. Inclusion criteria for the study were (i) epilepsy patients aged ≥5 years, (ii) two or more spontaneous seizures requiring AED treatment, (iii) not previously treated with AED, (iv) monotherapy considered the most appropriate treatment option and (v) willing to provide consent. Exclusion criteria were (i) provoked seizures (e.g. alcohol), (ii) acute symptomatic seizures (e.g. acute brain injury) and (iii) progressive neurological disease (e.g. brain tumour). Patients were classified according to clinician's judgement and classification of epilepsy and seizure outcomes were re-assessed at final data entry. Database checks highlighting inconsistencies were queried with the investigator. The Australian (AUS) cohort consisted of 380 treatment naïve patients prospectively recruited from epilepsy clinics at two hospitals in Australia: the Royal Melbourne Hospital and the Austin Hospital in Victoria. Inclusion and exclusion criteria for the AUS cohort were identical to the UK cohort, except that the AUS study excluded patients &lt;10 years and the choice of AED was determined by physician's preference.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Clinical covariates</h3><p id="__p57" class="p p-first-last">Baseline clinical covariates were gender, age at starting treatment, cranial computed tomography (CT) or magnetic resonance imaging (MRI) result, EEG result, total number of seizures pre-treatment, type of epilepsy, neurological impairment (defined as localizing neurological signs resulting in functional impairment) and initial AED. CT and MRI scans were classified as abnormal, not done or normal/non-specific abnormality. EEGs were classified as epileptiform abnormality (defined as focal or generalized spike or spike and slow wave activity), not done or normal/non-specific abnormality. Seizure types and epilepsy syndromes were classified according to the ILAE Classification (<a href="#DDT403C39" rid="DDT403C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013733">39</a>). Epilepsy type was classified as focal, generalized or unclassified (unclassified where there was uncertainty between focal or generalized onset epilepsy). The UK cohort, which was more than twice the size of AUS, was chosen as the discovery cohort for the purpose of selecting clinical prognostic factors for inclusion in the GWAS; clinical factors which showed association with 12-month remission of seizures in a univariate logistic regression model at <em>P</em> &lt; 0.05 were included.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Outcome definition</h3><p id="__p58" class="p p-first-last">Epilepsy patients achieving 12-month (365 days or longer) remission of seizures were defined as “responders”, and patients failing to achieve 12-month remission were defined as “non-responders”. Patients followed for &lt;1 year were excluded from the study.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Sample and genotyping QC</h3><p id="__p59" class="p p-first">The UK samples were genotyped at the Wellcome Trust Sanger Institute on Illumina 660. QC of samples was based on the following criteria, with inclusion/exclusion thresholds for each determined empirically: samples were removed if they displayed heterozygosity outside the interval [0.281,0.299] (28 samples failed), sample call rate &lt;0.98 (11 additional samples failed), gender discordance (3 additional samples failed), pairwise relatedness &gt;0.9 (i.e. accidental duplicates, in which case the lowest quality sample was excluded) (28 samples failed). The presence of highly related individuals can cause confounding in association studies, so a second filtering was then applied to ensure that no pair had estimated relatedness &gt;0.1, a threshold set just below that expected for first cousins (12 samples removed). Principal component analysis (PCA) was performed using a subset of SNPs in approximate linkage disequilibrium in order to identify ancestry outliers: individuals with extreme values on principal component axes 1 or 2 were removed (two samples) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>). Of the remaining 822 samples, the UK GWAS was performed using the 654 treatment naïve patients observed for at least 1 year after starting AED therapy and for whom the remission status and complete clinical covariate information was available (436 responders and 218 non-responders).</p><p id="__p60" class="p p-last">The AUS cohort of 380 samples was genotyped on the same platform and at the same institution (but at a different time) as the UK cohort, and genotype QC followed the same procedures: heterozygosity outside the interval [0.291, 0.309] (10 samples failed), call rate &lt;0.98 (8 additional samples failed), accidental duplication (2 additional samples failed), pairwise relatedness &gt;0.1 (7 samples failed) and PCA (4 samples removed). For the remaining 349 samples, remission status and complete clinical covariate information were available for 235 treatment naïve patients followed for at least 1 year (188 responders and 47 non-responders) who were included in the AUS GWAS.</p></div><div id="s4e" class="sec"><h3 id="s4etitle">SNP imputation</h3><p id="__p61" class="p p-first-last">The Illumina 660 chip interrogates genotype values for 594 398 SNPs. This number was first reduced to 540 497 for the UK cohort and 533 985 for AUS, by applying (QC) thresholds based on minor allele frequency (MAF &gt; 0.01), call rate (CR &gt; 0.95) and a <em>P</em>-value from a test for Hardy-Weinberg Equilibrium (HWE &gt; 10<sup>−6</sup>). We imputed against the 1000 Genome reference panel using IMPUTE2 (<a href="#DDT403C40" rid="DDT403C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013708">40</a>) (first dividing the genome into approximately 5 Mb regions) resulting in (expected) genotypic values for ∼40 M SNPs. We then performed SNP QC for a second time, keeping only SNPs with (expected) MAF &gt; 0.01, (expected) CR &gt; 0.95, and INFO &gt; 0.8 (the latter an imputation quality score calculated by the IMPUTE2 algorithm); 6 923 995 SNPs passed these criteria in both cohorts.</p></div><div id="s4f" class="sec"><h3 id="s4ftitle">GWAS analysis</h3><p id="__p62" class="p p-first"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S2</a>, presents plots of the first two principal component axes for the UK and AUS cohorts separately, and when combined. When compared against samples from the HapMap project, it is evident that the UK and AUS cohorts are predominantly of Caucasian ancestry, but also that AUS appears to be more heterogeneous than the UK cohort. For this reason, we decided to analyse each cohort separately, then combine the two sets of results by meta-analysis.</p><p id="__p63">Association analyses were performed using a logistic regression model; letting <em>p</em> denote the probability of becoming a responder, the model supposes logit(<em>p</em>) = log(<em>p</em>/1 − <em>p</em>) = <em>μ</em> + <em>Xβ</em> + <em>Cλ</em> + <em>e</em>, where <em>μ</em> reflects the baseline odds, <em>X</em> represents the genotype of the SNP under examination (with effect size <em>β</em>) and <em>C</em> denotes the covariates (with effect sizes <em>λ</em>). Each SNP's genotypes were coded under an additive model, which assumes that the SNP's effect on the log odds is determined by the count of the alternative allele (0, 1 or 2). For imputed SNPs, <em>X</em> could be non-integer, equal to the expected allele count provided by the imputation; however, it is advised to perform the analysis using the expected values rather than replace them with, say, the most likely genotype value (<a href="#DDT403C41" rid="DDT403C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013735">41</a>). <em>C</em> includes covariates representative of population structure, obtained through PCA of the genotype matrix; the leading axes with eigenvalues significant at 5% using the Tracy–Widom test were included (four for UK, three for AUS). The regression analysis was performed using the logistic option of PLINK (<a href="#DDT403C42" rid="DDT403C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013748">42</a>). For both cohorts, the analysis was performed once with significant clinical prognostic factors included in C (“adjusted”), and once without (“unadjusted”). The effect size estimates for each SNP from the UK and AUS GWASs were corrected for genomic inflation (<a href="#DDT403C43" rid="DDT403C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013693">43</a>) and combined using a fixed-effect meta-analysis, weighting the effect size estimates from each study by their standard deviation using the PLINK option—meta-analysis. Regional association plots (“Broad Plots”) for the most strongly associated regions were prepared using R code provided by the Broad Institute (<a href="#DDT403C44" rid="DDT403C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013699">44</a>).</p><p id="__p64" class="p p-last">To examine evidence for an interaction between the genotype and the treatment type, for the top SNPs from the meta-analysis we enlarged the logistic regression model to include a drug-specific effect size. For example, when considering the possible effect of CBZ, we included β_CBZ to allow for an SNP's effect to be different across patients administered CBZ relative to those on other treatments. We considered each of the AEDs in turn, computing a <em>P</em>-value based on whether β_DRUG was significantly non-zero. Because patients may change AED during the course of the study, we repeated this analyses restricted to the 355 patients in the study whose AED remained unchanged.</p></div><div id="s4g" class="sec"><h3 id="s4gtitle">Pathway analysis</h3><p id="__p65" class="p p-first-last">We used two independent methods, ALIGATOR (<a href="#DDT403C32" rid="DDT403C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013720">32</a>,<a href="#DDT403C33" rid="DDT403C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013745">33</a>) and pathMaster (<a href="#DDT403C34" rid="DDT403C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013743">34</a>), to test the results of the meta-analysis for over-representation of biological pathways obtained from GO (downloaded from <a href="http://www.geneontology.org/">http://www.geneontology.org/</a>, restricting to pathways containing between 5 and 600 genes) and KEGG. ALIGATOR corrects for varying numbers of SNPs per gene and multiple overlapping functional pathways. To apply ALIGATOR, it is necessary to specify a GWAS <em>P</em>-value threshold; each pathway is scored by counting the number of its genes that contain one or more SNPs with <em>P</em>-value below this threshold. This score is then tested for significance by permutation. The choice of GWAS <em>P</em>-value threshold is arbitrary, since it depends on the sample size and the distribution of genetic effect sizes which is usually unknown; the most informative threshold will therefore balance confidence that the identified pathways have a true causal relationship with the phenotype and not missing any genuine pathway associations. As a pragmatic solution to the problem of choosing a <em>P</em>-value threshold ALIGATOR recommends exploring a range of thresholds to determine which gives the most significant increase in overrepresented functional categories. We therefore considered <em>P</em>-value thresholds at 0.01, 0.001 and 0.0001. Analyses were undertaken before and after adjusting for clinical prognostic factors. All ALIGATOR analyses used 5000 simulated replicate gene lists and 2000 simulated replicate studies. PathMaster (<a href="#DDT403C34" rid="DDT403C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013732">34</a>) evaluates the overall genetic contribution of a given pathway via a cumulative trend test statistic; this is the sum of the Armitage trend test statistic over all of the SNPs in the pathway. The null distribution of the pathway statistic is estimated by a skew normal or gamma distribution; the distribution chosen is determined by the Kolmogorov–Smirnof test statistic. The parameters of the chosen distribution are estimated from 100 random permutations of case–control labels. The null distribution is estimated separately for each pathway (see (<a href="#DDT403C42" rid="DDT403C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013726">42</a>) for details). Analyses were undertaken before adjusting for clinical prognostic factors. For both ALIGATOR and pathMaster, SNPs were assigned to genes if they were within gene boundaries or 20 kb either side.</p></div><div id="s4h" class="sec"><h3 id="s4htitle">Copy number variation (CNV) analysis</h3><p id="__p66" class="p p-first-last">For loci indexed by SNPs below the <em>P</em> &lt; 5.0 × 10<sup>−7</sup> threshold, we examined whether the SNPs were tagging CNV using cnvHap (<a href="#DDT403C37" rid="DDT403C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013719">37</a>). cnvHap is an integrative multi-platform haplotype-based method which uses population distribution of allele frequency to train its haplotype hidden Markov model and has been shown to be more accurate than other methods in calling CNVs from SNP data. We extracted Log R Ratio (LRR) and B allele frequency (BAF) from the intensity files and corrected for the GC content and long-range autocorrelation. CNV calls were generated in cnvHap using LRR, BAF and Illumina platform-specific parameters and any potential CNV call was visually inspected. We searched each gene using a 50 Kb window, testing each CNV discovered for association with 12-month remission using a logistic regression framework with population genotype derived principal components and age and gender included as covariates.</p></div><div id="s4i" class="sec"><h3 id="s4ititle">Power plots</h3><p id="__p67" class="p p-first-last">Power plots were estimated following the methodology of Bacanu <em>et al</em>. (<a href="#DDT403C30" rid="DDT403C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_345013731">30</a>) modified for an additive test. Power was calculated based on the significance threshold of 5.0 × 10<sup>−8</sup> (genome-wide significance), given the number of responders and non-responders in our meta-analysis, for MAFs between 1 and 50%.</p></div><div id="s4j" class="sec sec-last"><h3 id="s4jtitle">Ethics</h3><p id="__p68" class="p p-first-last">This study was conducted under MREC 02/8/45. Consents were obtained according to the Declaration of Helsinki (BMJ 1991; 302: 1194).</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p69" class="p p-first"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt403/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p><p id="__p70" class="p p-last"><em>Conflict of Interest statement</em>. None declared.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p71" class="p p-first-last">This work was funded by the Wellcome Trust (<span>WT066056</span> to M.R.J.), The NIHR
Biomedical Research Centres Scheme (<span>P31753</span> to M.R.J.), Department of Health NHS Chair of Pharmacogenetics (to M.P.), The Medical Research Council (REF: 25105 to D.B. and M.R.J.), The Royal Melbourne Hospital Foundation Lottery Grants (REF: <span>604955</span> to S.P.) and The RMH Neuroscience Foundation (to T.J.O'B.). M.R.J. conceived the UK study and coordinated the UK DNA collection. T.J.O'B. conceived the Australian study and coordinated the Australian DNA collection. M.R.J., D.C., M.P. and D.B. obtained the funding to collect the UK DNA samples. T.J.O'B. obtained the funding to collect Australian samples. M.R.J., D.C., M.P., D.B., A.P., A.M., S.P. and T.J.O'B. obtained the funding to genotype the samples. Genomic data were analysed by D.S., C.H., I.T., S.P., A.J., A.P., M.D., T.D. and M.R.J. DNA samples and phenotypic information was collected and analysed by S.P., A.J., M.T., D.S., P.E.S., P.C., M.K., S.H., M.R., G.E.S., D.C., M.P., I.S., S.B., A.M., I.S., S.B., M.N., R.Y., M.T., C.F. and M.R.J. D.S., C.H., S.P. and M.R.J. drafted the manuscript. All authors reviewed and edited the manuscript. M.R.J. confirms that he has full access to all the data in the study and has final responsibility for the decision to submit for publication.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140715902865152"><a href="/pmc/articles/PMC3857947/bin/supp_23_1_247__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>







</div></div><div id="idm140715889423904" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140715889423904title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p72">We thank the clinical and non-clinical staff who supported the UK DNA collection: Alwaidh M (Whiston Hospital, St Helens and Knowsley Hospitals NHS Trust), Andrews J (New Cross Hospital), Appleton RE (Alder Hey Hospital, Royal Liverpool Children's NHS Trust), Bell S (Craigavon Area Hospital), Bucknall J (Homerton Hospital), Cleland P (Sunderland Royal Hospital), Cock H (St Georges Hospital), Cockerell OC (Royal London Hospital), Corston R (New Cross Hospital), Cramp C (Princess Royal Telford), Crawford P (York Hospitals NHS Trust), Dafalla BEA (Calderdale &amp; Huddersfield NHS Trust), Thompson C (Calderdale &amp; Huddersfield NHS Trust), Doherty C (Imperial College London), Doran M (The Walton Centre for Neurology and Neurosurgery), Duncan S (Hope Hospital), Esmonde TFG (Royal Victoria, Belfast), Goulding P (S James's University Hospital, The Leeds Teaching Hospitals NHS Trust), Gurtin D (Homerton Hospital), Harrington B (Wexham Maelor), Hinde F (Princess Royal Telford), Howell S (Doncaster and Bassetlaw Hospital), Hughes A (Arrowe Park Hospital, Wirral Hospitals NHS Trust), Hulme A (Hope Hospital), Kindley AD (Royal Aberdeen Children's Hospital), Lawden M (Leicester Royal Infirmery), Litherland G (Whiston Hospital, St Helens and Knowsley Hospitals NHS Trust), MacDonald S (Ninewells Hospital, Dundee), McLean B (Royal Cornwall Hospital), Middleton C (Imperial College London), Minchom P (Wexham Maelor), Newton R (Manchester Royal Infirmary), Nicholl D (Queen Elizabeth, Birmingham), Owens G (Wexham Maelor), Parret M (Royal Cornwall Hospital), Reuber M (Royal Hallamshire Hospital, Sheffield &amp; Chesterfield &amp; North Derbyshire Royal Hospital), Roberts R (Ninewells Hospital, Dundee), Sharrack B (Lincoln County Hospital), Silver N (North Cheshire Hospitals NHS Trust), Sood R (Homerton Hospital), Stewart J (Arrowe Park Hospital, Wirral Hospitals NHS Trust), Tidswell P (Blackburn Royal Infirmary), Von Oertzen T (St Georges Hospital), Waite P (Lincoln County Hospital), Weishmann U (The Walton Centre for Neurology and Neurosurgery) and White K (Ninewells Hospital, Dundee).</p></div></div><div id="idm140715889420880" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140715889420880title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDT403C1">1. <span class="element-citation">Annegers J.F., Hauser W.A., Elverback L.R. Remission of seizures and relapse in patients with epilepsy. <span><span class="ref-journal">Epilepsia. </span>1979;<span class="ref-vol">20</span>:729–737.</span> [<a href="/pubmed/499118" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epilepsia&amp;title=Remission+of+seizures+and+relapse+in+patients+with+epilepsy&amp;author=J.F.+Annegers&amp;author=W.A.+Hauser&amp;author=L.R.+Elverback&amp;volume=20&amp;publication_year=1979&amp;pages=729-737&amp;pmid=499118&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C2">2. <span class="element-citation">Jacoby A., Lane S., Marson A., Baker G.A. MESS Study Group. Relationship of clinical and quality of life trajectories following the onset of seizures: findings from the UK MESS study. <span><span class="ref-journal">Epilepsia. </span>2011;<span class="ref-vol">52</span>:965–974.</span> [<a href="/pubmed/21320116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epilepsia&amp;title=Relationship+of+clinical+and+quality+of+life+trajectories+following+the+onset+of+seizures:+findings+from+the+UK+MESS+study&amp;author=A.+Jacoby&amp;author=S.+Lane&amp;author=A.+Marson&amp;author=G.A.+Baker&amp;volume=52&amp;publication_year=2011&amp;pages=965-974&amp;pmid=21320116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C3">3. <span class="element-citation">Hildebrand M.S., Dahl H.H., Damiano J.A., Smith R.J., Scheffer I.E., Berkovic S.F. Recent advances in the molecular genetics of epilepsy. <span><span class="ref-journal">J. Med. Genet. </span>2013;<span class="ref-vol">50</span>:271–279.</span> [<a href="/pubmed/23468209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet.&amp;title=Recent+advances+in+the+molecular+genetics+of+epilepsy&amp;author=M.S.+Hildebrand&amp;author=H.H.+Dahl&amp;author=J.A.+Damiano&amp;author=R.J.+Smith&amp;author=I.E.+Scheffer&amp;volume=50&amp;publication_year=2013&amp;pages=271-279&amp;pmid=23468209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C4">4. <span class="element-citation">Senn S. Individual response to treatment: is it a valid assumption. <span><span class="ref-journal">BMJ. </span>2004;<span class="ref-vol">329</span>:966–968.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC524113/">PMC free article</a>]</span> [<a href="/pubmed/15499115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Individual+response+to+treatment:+is+it+a+valid+assumption&amp;author=S.+Senn&amp;volume=329&amp;publication_year=2004&amp;pages=966-968&amp;pmid=15499115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C5">5. <span class="element-citation">Kwan P., Brodie M.J. Early identification of refractory epilepsy. <span><span class="ref-journal">N. Engl. J. Med. </span>2000;<span class="ref-vol">342</span>:314–319.</span> [<a href="/pubmed/10660394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Early+identification+of+refractory+epilepsy&amp;author=P.+Kwan&amp;author=M.J.+Brodie&amp;volume=342&amp;publication_year=2000&amp;pages=314-319&amp;pmid=10660394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C6">6. <span class="element-citation">Johnson M.R., Milne R.L., Torn-Broers Y., Hopper J.L., Scheffer I.E., Berkovic S.F. A twin study of genetic influences on epilepsy outcome. <span><span class="ref-journal">Twin Res. </span>2003;<span class="ref-vol">6</span>:140–146.</span> [<a href="/pubmed/12724000" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res.&amp;title=A+twin+study+of+genetic+influences+on+epilepsy+outcome&amp;author=M.R.+Johnson&amp;author=R.L.+Milne&amp;author=Y.+Torn-Broers&amp;author=J.L.+Hopper&amp;author=I.E.+Scheffer&amp;volume=6&amp;publication_year=2003&amp;pages=140-146&amp;pmid=12724000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C7">7. <span class="element-citation">Rogawski M.A., Johnson M.R. Intrinsic severity as a determinant of antiepilepsy drug refractoriness. <span><span class="ref-journal">Epilepsy Curr. </span>2008;<span class="ref-vol">8</span>:127–130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2566613/">PMC free article</a>]</span> [<a href="/pubmed/18852835" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epilepsy+Curr.&amp;title=Intrinsic+severity+as+a+determinant+of+antiepilepsy+drug+refractoriness&amp;author=M.A.+Rogawski&amp;author=M.R.+Johnson&amp;volume=8&amp;publication_year=2008&amp;pages=127-130&amp;pmid=18852835&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C8">8. <span class="element-citation">Kwan P., Schachter S.C., Brodie M.J. Drug resistant epilepsy. <span><span class="ref-journal">N. Engl. J. Med. </span>2011;<span class="ref-vol">365</span>:919–926.</span> [<a href="/pubmed/21899452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Drug+resistant+epilepsy&amp;author=P.+Kwan&amp;author=S.C.+Schachter&amp;author=M.J.+Brodie&amp;volume=365&amp;publication_year=2011&amp;pages=919-926&amp;pmid=21899452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C9">9. <span class="element-citation">Manolio T.A., Bailey-Wilson J.E., Collins F.S. Genes, environment and the value of prospective cohort studies. <span><span class="ref-journal">Nat. Rev. Genet. </span>2006;<span class="ref-vol">7</span>:812–820.</span> [<a href="/pubmed/16983377" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genes,+environment+and+the+value+of+prospective+cohort+studies&amp;author=T.A.+Manolio&amp;author=J.E.+Bailey-Wilson&amp;author=F.S.+Collins&amp;volume=7&amp;publication_year=2006&amp;pages=812-820&amp;pmid=16983377&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C10">10. <span class="element-citation">Leschziner G., Jorgensen A., Andrew T., Pirmohamed M., Williamson P., Marson A.G., Coffey A.J., Middleditch C., Rogers J., Bentley D.R., et al.  Clinical factors and <em>ABCB1</em> polymorphisms in prediction of antiepileptic drug response: a prospective cohort study. <span><span class="ref-journal">Lancet. Neurol. </span>2006;<span class="ref-vol">5</span>:668–676.</span> [<a href="/pubmed/16857572" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet.+Neurol.&amp;title=Clinical+factors+and+ABCB1+polymorphisms+in+prediction+of+antiepileptic+drug+response:+a+prospective+cohort+study&amp;author=G.+Leschziner&amp;author=A.+Jorgensen&amp;author=T.+Andrew&amp;author=M.+Pirmohamed&amp;author=P.+Williamson&amp;volume=5&amp;publication_year=2006&amp;pages=668-676&amp;pmid=16857572&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C11">11. <span class="element-citation">Marson A.G., Al-Kharusi A.M., Alwaidh M., Appleton R., Baker G.A., Chadwick D.W., Cramp C., Cockerell O.C., Cooper P.N., Doughty J., et al.  The SANAD study of effectiveness of carbamazepine, gabapentin, lamotrigine, oxcarbazepine, or topiramate for treatment of partial epilepsy: an unblinded randomised controlled trial. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">369</span>:1000–1015.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2080688/">PMC free article</a>]</span> [<a href="/pubmed/17382827" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=The+SANAD+study+of+effectiveness+of+carbamazepine,+gabapentin,+lamotrigine,+oxcarbazepine,+or+topiramate+for+treatment+of+partial+epilepsy:+an+unblinded+randomised+controlled+trial&amp;author=A.G.+Marson&amp;author=A.M.+Al-Kharusi&amp;author=M.+Alwaidh&amp;author=R.+Appleton&amp;author=G.A.+Baker&amp;volume=369&amp;publication_year=2007&amp;pages=1000-1015&amp;pmid=17382827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C12">12. <span class="element-citation">Marson A.G., Al-Kharusi A.M., Alwaidh M., Appleton R., Baker G.A., Chadwick D.W., Cramp C., Cockerell O.C., Cooper P.N., Doughty J., et al.  The SANAD study of effectiveness of valproate, lamotrigine, or topiramate for generalised and unclassifiable epilepsy: an unblinded randomised controlled trial. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">369</span>:1016–1026.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2039891/">PMC free article</a>]</span> [<a href="/pubmed/17382828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=The+SANAD+study+of+effectiveness+of+valproate,+lamotrigine,+or+topiramate+for+generalised+and+unclassifiable+epilepsy:+an+unblinded+randomised+controlled+trial&amp;author=A.G.+Marson&amp;author=A.M.+Al-Kharusi&amp;author=M.+Alwaidh&amp;author=R.+Appleton&amp;author=G.A.+Baker&amp;volume=369&amp;publication_year=2007&amp;pages=1016-1026&amp;pmid=17382828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C13">13. <span class="element-citation">Kwan P., Arzimanoglou P., Berg A.T., Brodie M.J., Hauser A.W., Mathern G., Moshe S.L., Perucca E.L., Wiebe S., French J. Definition of drug resistant epilepsy: consensus proposal by the ad hoc Task Force of the ILAE Commission on Therapeutic Strategies. <span><span class="ref-journal">Epilepsia. </span>2010;<span class="ref-vol">51</span>:1069–1077.</span> [<a href="/pubmed/19889013" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epilepsia&amp;title=Definition+of+drug+resistant+epilepsy:+consensus+proposal+by+the+ad+hoc+Task+Force+of+the+ILAE+Commission+on+Therapeutic+Strategies&amp;author=P.+Kwan&amp;author=P.+Arzimanoglou&amp;author=A.T.+Berg&amp;author=M.J.+Brodie&amp;author=A.W.+Hauser&amp;volume=51&amp;publication_year=2010&amp;pages=1069-1077&amp;pmid=19889013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C14">14. <span class="element-citation">MacDonald B.K., Johnson A.L., Goodridge D.M., Cockerell O.C., Sander J.W., Shorvon S.D. Factors predicting prognosis of epilepsy after presentation with seizures. <span><span class="ref-journal">Ann. Neurol. </span>2000;<span class="ref-vol">48</span>:833–841.</span> [<a href="/pubmed/11117539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Neurol.&amp;title=Factors+predicting+prognosis+of+epilepsy+after+presentation+with+seizures&amp;author=B.K.+MacDonald&amp;author=A.L.+Johnson&amp;author=D.M.+Goodridge&amp;author=O.C.+Cockerell&amp;author=J.W.+Sander&amp;volume=48&amp;publication_year=2000&amp;pages=833-841&amp;pmid=11117539&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C15">15. <span class="element-citation">Freathy R.M., Timpson N.J., Lawlor D.A., Pouta A., Ben-Shlomo Y., Ruokonen A., Ebrahim S., Shields B., Zeggini E., Weedon M.N., et al.  Common variation in the FTO gene alters diabetes-related metabolic traits to the extent expected given its effect on BMI. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:1419–1426.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3073395/">PMC free article</a>]</span> [<a href="/pubmed/18346983" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Common+variation+in+the+FTO+gene+alters+diabetes-related+metabolic+traits+to+the+extent+expected+given+its+effect+on+BMI&amp;author=R.M.+Freathy&amp;author=N.J.+Timpson&amp;author=D.A.+Lawlor&amp;author=A.+Pouta&amp;author=Y.+Ben-Shlomo&amp;volume=57&amp;publication_year=2008&amp;pages=1419-1426&amp;pmid=18346983&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C16">16. <span class="element-citation">The International Multiple Sclerosis Genetics Consortium &amp; The Wellcome Trust Case Control Consortium 2. Genetic risk and a primary role for cell-mediated immune mechanisms in multiple sclerosis. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">476</span>:214–219.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3182531/">PMC free article</a>]</span> [<a href="/pubmed/21833088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+risk+and+a+primary+role+for+cell-mediated+immune+mechanisms+in+multiple+sclerosis&amp;volume=476&amp;publication_year=2011&amp;pages=214-219&amp;pmid=21833088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C17">17. <span class="element-citation">Stephens M., Balding D. Bayesian statistical methods for genetic association studies. <span><span class="ref-journal">Nat. Rev. Genet. </span>2009;<span class="ref-vol">10</span>:681–690.</span> [<a href="/pubmed/19763151" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Bayesian+statistical+methods+for+genetic+association+studies&amp;author=M.+Stephens&amp;author=D.+Balding&amp;volume=10&amp;publication_year=2009&amp;pages=681-690&amp;pmid=19763151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C18">18. <span class="element-citation">Al Nimer F., Strom M., Lindblom R., Aeinehband S., Bellander B.M., Nyengaard J.R., Lidman O., Piehl F. Naturally occurring variation in the glutathione-S-transferase 4 gene determines neurodegeneration after traumatic brain injury. <span><span class="ref-journal">Antioxid. Redox. Signal. </span>2013;<span class="ref-vol">18</span>:784–794.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3555113/">PMC free article</a>]</span> [<a href="/pubmed/22881716" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antioxid.+Redox.+Signal.&amp;title=Naturally+occurring+variation+in+the+glutathione-S-transferase+4+gene+determines+neurodegeneration+after+traumatic+brain+injury&amp;author=F.+Al+Nimer&amp;author=M.+Strom&amp;author=R.+Lindblom&amp;author=S.+Aeinehband&amp;author=B.M.+Bellander&amp;volume=18&amp;publication_year=2013&amp;pages=784-794&amp;pmid=22881716&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C19">19. <span class="element-citation">Takahashi H., Katayama K., Sohya K., Miyamoto H., Prasad T., Matsumoto Y., Ota M., Yasuda H., Tsumoto T., Aruga J., Craig A.M. Selective control of inhibitory synapse development by Slitrk3-PTPdelta trans-synaptic interaction. <span><span class="ref-journal">Nat. Neurosci. </span>2012;<span class="ref-vol">13</span>:389–398.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3288805/">PMC free article</a>]</span> [<a href="/pubmed/22286174" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Neurosci.&amp;title=Selective+control+of+inhibitory+synapse+development+by+Slitrk3-PTPdelta+trans-synaptic+interaction&amp;author=H.+Takahashi&amp;author=K.+Katayama&amp;author=K.+Sohya&amp;author=H.+Miyamoto&amp;author=T.+Prasad&amp;volume=13&amp;publication_year=2012&amp;pages=389-398&amp;pmid=22286174&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C20">20. <span class="element-citation">Elia J., Gai X., Xie H.M., Perin J.C., Geiger E., Glessner J.T., D'arcy M., de Berardinis R., Frackelton E., Kim C., et al.  Rare structural variants found in attention-deficit hyperactivity disorder are preferentially associated with neurodevelopmental genes. <span><span class="ref-journal">Mol. Psychiatry. </span>2010;<span class="ref-vol">15</span>:637–646.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2877197/">PMC free article</a>]</span> [<a href="/pubmed/19546859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=Rare+structural+variants+found+in+attention-deficit+hyperactivity+disorder+are+preferentially+associated+with+neurodevelopmental+genes&amp;author=J.+Elia&amp;author=X.+Gai&amp;author=H.M.+Xie&amp;author=J.C.+Perin&amp;author=E.+Geiger&amp;volume=15&amp;publication_year=2010&amp;pages=637-646&amp;pmid=19546859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C21">21. <span class="element-citation">Pinto D., Pagnamenta A.T., Klei L., Anney R., Merico D., Regan R., Conroy J., Magalhaes R.R., Correia C., Abrahams B.S., et al.  Functional impact of global rare copy number variation in autism spectrum disorders. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:368–372.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3021798/">PMC free article</a>]</span> [<a href="/pubmed/20531469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Functional+impact+of+global+rare+copy+number+variation+in+autism+spectrum+disorders&amp;author=D.+Pinto&amp;author=A.T.+Pagnamenta&amp;author=L.+Klei&amp;author=R.+Anney&amp;author=D.+Merico&amp;volume=466&amp;publication_year=2010&amp;pages=368-372&amp;pmid=20531469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C22">22. <span class="element-citation">Uetani N., Kato K., Ogura H., Mizuno K., Kawano K., Mikoshiba K., Yakura H., Asano M., Iwakura Y. Impaired learning with enhanced hippocampal long-term potentiation in PTPδ-deficient mice. <span><span class="ref-journal">EMBO J. </span>2000;<span class="ref-vol">19</span>:2775–2785.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC203365/">PMC free article</a>]</span> [<a href="/pubmed/10856223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J.&amp;title=Impaired+learning+with+enhanced+hippocampal+long-term+potentiation+in+PTPδ-deficient+mice&amp;author=N.+Uetani&amp;author=K.+Kato&amp;author=H.+Ogura&amp;author=K.+Mizuno&amp;author=K.+Kawano&amp;volume=19&amp;publication_year=2000&amp;pages=2775-2785&amp;pmid=10856223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C23">23. <span class="element-citation">Schormair B., Kemlink D., Roeske D., Eckstein G., Xiong L., Lichtner P., Ripke S., Trenkwalkder C., Zimprich A., Stiasny-Kolster K., et al.  PTPRD (protein tyrosine phosphatase receptor type delta) is associated with restless legs syndrome. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:946–948.</span> [<a href="/pubmed/18660810" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=PTPRD+(protein+tyrosine+phosphatase+receptor+type+delta)+is+associated+with+restless+legs+syndrome&amp;author=B.+Schormair&amp;author=D.+Kemlink&amp;author=D.+Roeske&amp;author=G.+Eckstein&amp;author=L.+Xiong&amp;volume=40&amp;publication_year=2008&amp;pages=946-948&amp;pmid=18660810&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C24">24. <span class="element-citation">Govek E.E., Newey S.E., Van Aelst L. The role of the Rho GTPases in neuronal development. <span><span class="ref-journal">Genes Dev. </span>2005;<span class="ref-vol">19</span>:1–49.</span> [<a href="/pubmed/15630019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=The+role+of+the+Rho+GTPases+in+neuronal+development&amp;author=E.E.+Govek&amp;author=S.E.+Newey&amp;author=L.+Van+Aelst&amp;volume=19&amp;publication_year=2005&amp;pages=1-49&amp;pmid=15630019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C25">25. <span class="element-citation">Sharp A.J., Mefford H.C., Li K., Baker C., Skinner C., Stevenson R.E., Schroer R.J., Novara F., De Gregori M., Ciccone R., et al.  A recurrent 15q13.3 microdeletion syndrome associated with mental retardation and seizures. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:322–328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365467/">PMC free article</a>]</span> [<a href="/pubmed/18278044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+recurrent+15q13.3+microdeletion+syndrome+associated+with+mental+retardation+and+seizures&amp;author=A.J.+Sharp&amp;author=H.C.+Mefford&amp;author=K.+Li&amp;author=C.+Baker&amp;author=C.+Skinner&amp;volume=40&amp;publication_year=2008&amp;pages=322-328&amp;pmid=18278044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C26">26. <span class="element-citation">Leblond C.S., Heinrich J., Delorme R., Proepper C., Betancur C., Huguet G., Konyukh M., Chaste P., Ey E., Rastam M., Anckarsater H., et al.  Genetic and functional analysis of SHANK2 mutations suggest a multiple hit model of autism spectrum disorders. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002521.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3276563/">PMC free article</a>]</span> [<a href="/pubmed/22346768" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genetic+and+functional+analysis+of+SHANK2+mutations+suggest+a+multiple+hit+model+of+autism+spectrum+disorders&amp;author=C.S.+Leblond&amp;author=J.+Heinrich&amp;author=R.+Delorme&amp;author=C.+Proepper&amp;author=C.+Betancur&amp;volume=8&amp;publication_year=2012&amp;pages=e1002521&amp;pmid=22346768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C27">27. <span class="element-citation">Li J., Zhang L., Zhou H., Stoneking M., Tang K. Global patterns of genetic diversity and signals of natural selection for human ADME genes. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:528–540.</span> [<a href="/pubmed/21081654" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Global+patterns+of+genetic+diversity+and+signals+of+natural+selection+for+human+ADME+genes&amp;author=J.+Li&amp;author=L.+Zhang&amp;author=H.+Zhou&amp;author=M.+Stoneking&amp;author=K.+Tang&amp;volume=20&amp;publication_year=2011&amp;pages=528-540&amp;pmid=21081654&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C28">28. <span class="element-citation">Gou Y., Baum L.W., Sham P.C., Wong V., Ng P.W., Lui C.H.T., Sin N.C., Tsoi T.H., Tang C.S.M., Kwan J.S.H., et al.  Two-stage genome-wide association study identifies variants in <em>CAMSAP1L1</em> as suceptibility loci for epilepsy in Chinese. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:1184–1189.</span> [<a href="/pubmed/22116939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Two-stage+genome-wide+association+study+identifies+variants+in+CAMSAP1L1+as+suceptibility+loci+for+epilepsy+in+Chinese&amp;author=Y.+Gou&amp;author=L.W.+Baum&amp;author=P.C.+Sham&amp;author=V.+Wong&amp;author=P.W.+Ng&amp;volume=21&amp;publication_year=2012&amp;pages=1184-1189&amp;pmid=22116939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C29">29. <span class="element-citation">Steffans M., Leu C., Ruppert A-K., Zara F., Striano P., Robbiano A., Capovilla G., Tinuper A., et al.  EPICURE Consortium., EMINet Consortium. Genome-wide association analysis of genetic generalized epilepsies implicates susceptibility loci at 1q43, 2p16.1, 2q22.3 and 17q21.32. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:5359–5372.</span> [<a href="/pubmed/22949513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+analysis+of+genetic+generalized+epilepsies+implicates+susceptibility+loci+at+1q43,+2p16.1,+2q22.3+and+17q21.32&amp;author=M.+Steffans&amp;author=C.+Leu&amp;author=A-K.+Ruppert&amp;author=F.+Zara&amp;author=P.+Striano&amp;volume=21&amp;publication_year=2012&amp;pages=5359-5372&amp;pmid=22949513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C30">30. <span class="element-citation">Bacanu S-A., Whittaker J.C., Nelson M.R. How informative is a negative finding in a small pharmacogenetic study? <span><span class="ref-journal">Pharmacogenomics J. </span>2012;<span class="ref-vol">12</span>:93–95.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3321504/">PMC free article</a>]</span> [<a href="/pubmed/22158332" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics+J.&amp;title=How+informative+is+a+negative+finding+in+a+small+pharmacogenetic+study?&amp;author=S-A.+Bacanu&amp;author=J.C.+Whittaker&amp;author=M.R.+Nelson&amp;volume=12&amp;publication_year=2012&amp;pages=93-95&amp;pmid=22158332&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C31">31. <span class="element-citation">Zhou K., Pearson E.R. Insights from genome-wide association studies of drug response. <span><span class="ref-journal">Annu. Rev. Pharmacol. Toxicol. </span>2012;<span class="ref-vol">53</span>:299–310.</span> [<a href="/pubmed/23072379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Pharmacol.+Toxicol.&amp;title=Insights+from+genome-wide+association+studies+of+drug+response&amp;author=K.+Zhou&amp;author=E.R.+Pearson&amp;volume=53&amp;publication_year=2012&amp;pages=299-310&amp;pmid=23072379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C32">32. <span class="element-citation">Holmans P., Green E.K., Pahwa J.S., Ferreira M.A.R., Purcell S.M., Sklar P. Owen M.J., O'Donovan M.C., Craddock N. Wellcome Trust Case-Control Consortium. Gene ontology analysis of GWA study data sets provides insights into the biology of bipolar disorder. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">85</span>:13–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2706963/">PMC free article</a>]</span> [<a href="/pubmed/19539887" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Gene+ontology+analysis+of+GWA+study+data+sets+provides+insights+into+the+biology+of+bipolar+disorder&amp;author=P.+Holmans&amp;author=E.K.+Green&amp;author=J.S.+Pahwa&amp;author=M.A.R.+Ferreira&amp;author=S.M.+Purcell&amp;volume=85&amp;publication_year=2009&amp;pages=13-24&amp;pmid=19539887&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C33">33. <span class="element-citation">Holmans P., Moskvina V., Jones L., Sharma M. Vedernikov A., Buchel F., Sadd M., Bras J.M., Bettella F., et al.  The International Parkinson's Disease Genomics Consortium (IPDGC). A pathway-based analysis provides additional support for an immune-related genetic susceptibility to Parkinson's disease. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">22</span>:1039–1049.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3561909/">PMC free article</a>]</span> [<a href="/pubmed/23223016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+pathway-based+analysis+provides+additional+support+for+an+immune-related+genetic+susceptibility+to+Parkinson's+disease&amp;author=P.+Holmans&amp;author=V.+Moskvina&amp;author=L.+Jones&amp;author=M.+Sharma&amp;author=A.+Vedernikov&amp;volume=22&amp;publication_year=2012&amp;pages=1039-1049&amp;pmid=23223016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C34">34. <span class="element-citation">Eleftherohorinou H., Hoggart C.J., Wright V.J., Levin M., Coin L.J. Pathway-driven gene stability selection of two rheumatoid arthritis GWAS identifies and validates new susceptibility genes in receptor mediated signaling pathways. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:3494–3506.</span> [<a href="/pubmed/21653640" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Pathway-driven+gene+stability+selection+of+two+rheumatoid+arthritis+GWAS+identifies+and+validates+new+susceptibility+genes+in+receptor+mediated+signaling+pathways&amp;author=H.+Eleftherohorinou&amp;author=C.J.+Hoggart&amp;author=V.J.+Wright&amp;author=M.+Levin&amp;author=L.J.+Coin&amp;volume=20&amp;publication_year=2011&amp;pages=3494-3506&amp;pmid=21653640&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C35">35. <span class="element-citation">Jang J., Lee S.H., Goddard M.E., Visscher P.M. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Jang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C36">36. <span class="element-citation">Speed D., Hemani G., Johnson M.R., Balding D.J. Improved heritability estimation from genome-wide SNPs. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">91</span>:1011–1021.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3516604/">PMC free article</a>]</span> [<a href="/pubmed/23217325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Improved+heritability+estimation+from+genome-wide+SNPs&amp;author=D.+Speed&amp;author=G.+Hemani&amp;author=M.R.+Johnson&amp;author=D.J.+Balding&amp;volume=91&amp;publication_year=2012&amp;pages=1011-1021&amp;pmid=23217325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C37">37. <span class="element-citation">Coin L.J., Asher J.E., Walters R.G., Moustafa J.S., de Smith A.J., Sladek R., Balding D.J., Froguel P., Blakemore A.I. cnvHap: an integrative population and haplotype-based multiplatform model of SNPs and CNVs. <span><span class="ref-journal">Nat. Methods. </span>2010;<span class="ref-vol">7</span>:541–546.</span> [<a href="/pubmed/20512141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=cnvHap:+an+integrative+population+and+haplotype-based+multiplatform+model+of+SNPs+and+CNVs&amp;author=L.J.+Coin&amp;author=J.E.+Asher&amp;author=R.G.+Walters&amp;author=J.S.+Moustafa&amp;author=A.J.+de+Smith&amp;volume=7&amp;publication_year=2010&amp;pages=541-546&amp;pmid=20512141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C38">38. <span class="element-citation">McCormack M., Alfirevic A., Bourgeois S., Farrell J.F., Kasperaviciute D., Carrington M.N., Sills G., Marson A., de Bakker P.I., Chinthapalli K., et al.  HLA-A*3101 and carbamazepine-induced hypersensitivity reactions in Europeans. <span><span class="ref-journal">N. Engl. J. Med. </span>2011;<span class="ref-vol">364</span>:1134–1143.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3113609/">PMC free article</a>]</span> [<a href="/pubmed/21428769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=HLA-A*3101+and+carbamazepine-induced+hypersensitivity+reactions+in+Europeans&amp;author=M.+McCormack&amp;author=A.+Alfirevic&amp;author=S.+Bourgeois&amp;author=J.F.+Farrell&amp;author=D.+Kasperaviciute&amp;volume=364&amp;publication_year=2011&amp;pages=1134-1143&amp;pmid=21428769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C39">39. <span class="element-citation">Berg A.T., Berkovic S.F., Brodie M.J., Buchhalter J., Cross J.H., van Emde Boas W., Engel J., French J., Glauser T.A., Mathern G.W. Revised terminology and concepts for organization of seizures and epilepsies: report of the ILAE Commission on Classification and Terminology, 2005–2009. <span><span class="ref-journal">Epilepsia. </span>2009;<span class="ref-vol">51</span>:676–685.</span> [<a href="/pubmed/20196795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epilepsia&amp;title=Revised+terminology+and+concepts+for+organization+of+seizures+and+epilepsies:+report+of+the+ILAE+Commission+on+Classification+and+Terminology,+2005–2009&amp;author=A.T.+Berg&amp;author=S.F.+Berkovic&amp;author=M.J.+Brodie&amp;author=J.+Buchhalter&amp;author=J.H.+Cross&amp;volume=51&amp;publication_year=2009&amp;pages=676-685&amp;pmid=20196795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C40">40. <span class="element-citation">Howie B.N., Donnelly P., Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=B.N.+Howie&amp;author=P.+Donnelly&amp;author=J.+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C41">41. <span class="element-citation">Marchini J., Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat. Rev. Genet. </span>2010;<span class="ref-vol">11</span>:499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J.+Marchini&amp;author=B.+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C42">42. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A.R., Bender D., Maller J., Skiar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: a toolset for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+toolset+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.R.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C43">43. <span class="element-citation">Devlin B., Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT403C44">44. <span class="element-citation">Diabetes Genetics Initiative of Broad Institute of Harvard and MIT., Lund University., and Novartis Institutes of BioMedical Research. Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3857947&amp;issue-id=230861&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140715891874128title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140715889423904title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140715889420880title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3857947/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3857947/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3857947/pdf/ddt403.pdf">PDF (645K)</a></li> | <li><a href="#" data-citationid="PMC3857947" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3857947%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3857947%2F&amp;text=A%20genome-wide%20association%20study%20and%20biological%20pathway%20analysis%20of%20epilepsy%20prognosis%20in%20a%20prospective%20cohort%20of%20newly%20treated%20epilepsy" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3857947%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23962720" data-aiid="3857947" data-aid="3857947" data-iid="230861" data-domainid="740" data-domain="hmg" data-accid="PMC3857947" data-md5="000a75d9fb246d0cd4a1ca15c5d17f7f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T11:12:15-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C46D1DD55B7510000000007DC04BB&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3857947%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3857947/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>