<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3773904/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genomewide perspective of genetic variation in human metabolism</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3773904/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="A genomewide perspective of genetic variation in human metabolism"><meta name="citation_authors" content="Thomas Illig, Christian Gieger, Guangju Zhai, Werner Römisch-Margl, Rui Wang-Sattler, Cornelia Prehn, Elisabeth Altmaier, Gabi Kastenmüller, Bernet S. Kato, Hans-Werner Mewes, Thomas Meitinger, Martin Hrabé de Angelis, Florian Kronenberg, Nicole Soranzo, H.-Erich Wichmann, Tim D. Spector, Jerzy Adamski, Karsten Suhre"><meta name="citation_date" content="February 2010"><meta name="citation_issue" content="2"><meta name="citation_volume" content="42"><meta name="citation_firstpage" content="137"><meta name="citation_doi" content="10.1038/ng.507"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3773904/?report=abstract"><meta name="citation_pmid" content="20037589"><meta name="DC.Title" content="A genomewide perspective of genetic variation in human metabolism"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Thomas Illig"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Guangju Zhai"><meta name="DC.Contributor" content="Werner Römisch-Margl"><meta name="DC.Contributor" content="Rui Wang-Sattler"><meta name="DC.Contributor" content="Cornelia Prehn"><meta name="DC.Contributor" content="Elisabeth Altmaier"><meta name="DC.Contributor" content="Gabi Kastenmüller"><meta name="DC.Contributor" content="Bernet S. Kato"><meta name="DC.Contributor" content="Hans-Werner Mewes"><meta name="DC.Contributor" content="Thomas Meitinger"><meta name="DC.Contributor" content="Martin Hrabé de Angelis"><meta name="DC.Contributor" content="Florian Kronenberg"><meta name="DC.Contributor" content="Nicole Soranzo"><meta name="DC.Contributor" content="H.-Erich Wichmann"><meta name="DC.Contributor" content="Tim D. Spector"><meta name="DC.Contributor" content="Jerzy Adamski"><meta name="DC.Contributor" content="Karsten Suhre"><meta name="DC.Date" content="2010 Feb"><meta name="DC.Identifier" content="10.1038/ng.507"><meta name="DC.Language" content="en"><meta property="og:title" content="A genomewide perspective of genetic variation in human metabolism"><meta property="og:type" content="article"><meta property="og:description" content="Serum metabolite concentrations provide a direct readout of biological processes in the human body, and are associated with disorders such as cardiovascular and metabolic diseases. Here we present a genome-wide association study with 163 metabolic traits ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3773904/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3773904/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3773904/pdf/emss-54332.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89106CDD4FB1610000000008D305B8.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3773904%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC3773904</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2013 Sep 16.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20037589" target="pmc_ext" ref="reftype=publisher&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2010 Feb; 42(2): 137–141. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Dec 27. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.507" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.507</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3773904</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS54332</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20037589">20037589</a></div></div></div></div><h1 class="content-title">A genomewide perspective of genetic variation in human metabolism</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400172213968" co-class="co-affbox">Thomas Illig</a>,<sup>1,</sup><sup>+</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400180857392" co-class="co-affbox">Christian Gieger</a>,<sup>1,</sup><sup>+</sup> <a href="/pubmed/?term=Zhai%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400172144704" co-class="co-affbox">Guangju Zhai</a>,<sup>2</sup> <a href="/pubmed/?term=R%26%23x000f6%3Bmisch-Margl%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400172142720" co-class="co-affbox">Werner Römisch-Margl</a>,<sup>3</sup> <a href="/pubmed/?term=Wang-Sattler%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400172140736" co-class="co-affbox">Rui Wang-Sattler</a>,<sup>1</sup> <a href="/pubmed/?term=Prehn%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400210052144" co-class="co-affbox">Cornelia Prehn</a>,<sup>4</sup> <a href="/pubmed/?term=Altmaier%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400210050160" co-class="co-affbox">Elisabeth Altmaier</a>,<sup>3,</sup><sup>5</sup> <a href="/pubmed/?term=Kastenm%26%23x000fc%3Bller%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400172286384" co-class="co-affbox">Gabi Kastenmüller</a>,<sup>3</sup> <a href="/pubmed/?term=Kato%20BS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400172284400" co-class="co-affbox">Bernet S. Kato</a>,<sup>2</sup> <a href="/pubmed/?term=Mewes%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400172282416" co-class="co-affbox">Hans-Werner Mewes</a>,<sup>3,</sup><sup>6</sup> <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400177018544" co-class="co-affbox">Thomas Meitinger</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=de%20Angelis%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400177015824" co-class="co-affbox">Martin Hrabé de Angelis</a>,<sup>4,</sup><sup>9</sup> <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400177397712" co-class="co-affbox">Florian Kronenberg</a>,<sup>10</sup> <a href="/pubmed/?term=Soranzo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400177395728" co-class="co-affbox">Nicole Soranzo</a>,<sup>11,</sup><sup>2</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400247530464" co-class="co-affbox">H.-Erich Wichmann</a>,<sup>1,</sup><sup>12</sup> <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400247527744" co-class="co-affbox">Tim D. Spector</a>,<sup>2</sup> <a href="/pubmed/?term=Adamski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400247525760" co-class="co-affbox">Jerzy Adamski</a>,<sup>4,</sup><sup>9</sup> and  <a href="/pubmed/?term=Suhre%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589" class="affpopup" co-rid="_co_idm140400210559088" co-class="co-affbox">Karsten Suhre</a><sup>3,</sup><sup>5,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140400172213968"><h3 class="no_margin">Thomas Illig</h3><p><sup>1</sup>Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Thomas Illig</a></div></div><div id="_co_idm140400180857392"><h3 class="no_margin">Christian Gieger</h3><p><sup>1</sup>Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Christian Gieger</a></div></div><div id="_co_idm140400172144704"><h3 class="no_margin">Guangju Zhai</h3><p><sup>2</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Zhai%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Guangju Zhai</a></div></div><div id="_co_idm140400172142720"><h3 class="no_margin">Werner Römisch-Margl</h3><p><sup>3</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=R%26%23x000f6%3Bmisch-Margl%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Werner Römisch-Margl</a></div></div><div id="_co_idm140400172140736"><h3 class="no_margin">Rui Wang-Sattler</h3><p><sup>1</sup>Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Wang-Sattler%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Rui Wang-Sattler</a></div></div><div id="_co_idm140400210052144"><h3 class="no_margin">Cornelia Prehn</h3><p><sup>4</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Prehn%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Cornelia Prehn</a></div></div><div id="_co_idm140400210050160"><h3 class="no_margin">Elisabeth Altmaier</h3><p><sup>3</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>5</sup>Faculty of Biology, Ludwig-Maximilians-Universität, Planegg-Martinsried, Germany </p><div>Find articles by <a href="/pubmed/?term=Altmaier%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Elisabeth Altmaier</a></div></div><div id="_co_idm140400172286384"><h3 class="no_margin">Gabi Kastenmüller</h3><p><sup>3</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Kastenm%26%23x000fc%3Bller%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Gabi Kastenmüller</a></div></div><div id="_co_idm140400172284400"><h3 class="no_margin">Bernet S. Kato</h3><p><sup>2</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Kato%20BS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Bernet S. Kato</a></div></div><div id="_co_idm140400172282416"><h3 class="no_margin">Hans-Werner Mewes</h3><p><sup>3</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>6</sup>Department of Genome-oriented Bioinformatics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </p><div>Find articles by <a href="/pubmed/?term=Mewes%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Hans-Werner Mewes</a></div></div><div id="_co_idm140400177018544"><h3 class="no_margin">Thomas Meitinger</h3><p><sup>7</sup>Institute of Human Genetics, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>8</sup>Institute of Human Genetics, Klinikum rechts der Isar, Technische Universität München, Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Thomas Meitinger</a></div></div><div id="_co_idm140400177015824"><h3 class="no_margin">Martin Hrabé de Angelis</h3><p><sup>4</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>9</sup>Institute of Experimental Genetics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </p><div>Find articles by <a href="/pubmed/?term=de%20Angelis%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Martin Hrabé de Angelis</a></div></div><div id="_co_idm140400177397712"><h3 class="no_margin">Florian Kronenberg</h3><p><sup>10</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Innsbruck Medical University, Innsbruck, Austria </p><div>Find articles by <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Florian Kronenberg</a></div></div><div id="_co_idm140400177395728"><h3 class="no_margin">Nicole Soranzo</h3><p><sup>2</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><p><sup>11</sup>The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton UK </p><div>Find articles by <a href="/pubmed/?term=Soranzo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Nicole Soranzo</a></div></div><div id="_co_idm140400247530464"><h3 class="no_margin">H.-Erich Wichmann</h3><p><sup>1</sup>Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>12</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität and Klinikum Grosshadern, Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">H.-Erich Wichmann</a></div></div><div id="_co_idm140400247527744"><h3 class="no_margin">Tim D. Spector</h3><p><sup>2</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Tim D. Spector</a></div></div><div id="_co_idm140400247525760"><h3 class="no_margin">Jerzy Adamski</h3><p><sup>4</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>9</sup>Institute of Experimental Genetics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </p><div>Find articles by <a href="/pubmed/?term=Adamski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Jerzy Adamski</a></div></div><div id="_co_idm140400210559088"><h3 class="no_margin">Karsten Suhre</h3><p><sup>3</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>5</sup>Faculty of Biology, Ludwig-Maximilians-Universität, Planegg-Martinsried, Germany </p><div>Find articles by <a href="/pubmed/?term=Suhre%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20037589">Karsten Suhre</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140400179461824_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140400179461824_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140400179461824_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Institute of Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Faculty of Biology, Ludwig-Maximilians-Universität, Planegg-Martinsried, Germany </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Genome-oriented Bioinformatics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Institute of Human Genetics, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Institute of Human Genetics, Klinikum rechts der Isar, Technische Universität München, Munich, Germany </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Institute of Experimental Genetics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Innsbruck Medical University, Innsbruck, Austria </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton UK </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität and Klinikum Grosshadern, Munich, Germany </div><div id="CR1"><sup>*</sup>To whom correspondence should be addressed. <a href="mailto:dev@null" data-email="ed.nehcneum-ztlohmleh@erhus.netsrak" class="oemail">ed.nehcneum-ztlohmleh@erhus.netsrak</a></div><div id="FN1"><strong>AUTHOR CONTRIBUTION</strong><p id="P2" class="p">Designed the experiment: TI, TDS, JA, KS</p><p id="P3" class="p">Contributed genetics data/analysis from the KORA study: TI, CG, TM, HEW</p><p id="P4" class="p">Contributed genetics data/analysis from the TwinsUK study: GZ, BSK, NS, TS</p><p id="P5" class="p">Contributed to the metabolomics experiments: WRM, RWS, CP, GK, HWM, MHA, TDS, JA, KS</p><p id="P6" class="p">Analyzed the data: CG, GZ, EA, KS</p><p id="P7" class="p">Wrote the manuscript: CG, FK, NS, KS</p><p id="P8" class="p p-last">All authors contributed their critical reviews of the manuscript in its preparation.</p></div><div id="FN3">+These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140400179461824_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20037589" target="pmc_ext" ref="reftype=publisher&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3773904/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 1.</strong></div><div><a href="/pmc/articles/PMC3773904/bin/NIHMS54332-supplement-Supplementary_Table_1.xls" data-ga-action="click_feat_suppl">NIHMS54332-supplement-Supplementary_Table_1.xls</a><span style="color:gray"> (2.3M)</span></div><div class="small guid">GUID:&nbsp;A8A317BB-E8D9-4639-B3E1-EBE876A1DCFC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 2.</strong></div><div><a href="/pmc/articles/PMC3773904/bin/NIHMS54332-supplement-Supplementary_Table_2.xls" data-ga-action="click_feat_suppl">NIHMS54332-supplement-Supplementary_Table_2.xls</a><span style="color:gray"> (328K)</span></div><div class="small guid">GUID:&nbsp;D24C3C82-2B07-4CFE-A74E-54D6FF15DBB1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Text and Figures.</strong></div><div><a href="/pmc/articles/PMC3773904/bin/NIHMS54332-supplement-Supplementary_Text_and_Figures.pdf" data-ga-action="click_feat_suppl">NIHMS54332-supplement-Supplementary_Text_and_Figures.pdf</a><span style="color:gray"> (849K)</span></div><div class="small guid">GUID:&nbsp;6A51901A-0699-4C6A-B21C-40F6B47258D5</div></div></dd></dl></div><div id="idm140400176789456" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140400176789456title">Abstract</h2><!--article-meta--><div><p id="P9" class="p p-first-last">Serum metabolite concentrations provide a direct readout of biological processes in the human body, and are associated with disorders such as cardiovascular and metabolic diseases. Here we present a genome-wide association study with 163 metabolic traits using 1809 participants from the KORA population, followed up in the TwinsUK cohort with 422 participants. In eight out of nine replicated loci (<em>FADS1</em>, <em>ELOVL2</em>, <em>ACADS</em>, <em>ACADM</em>, <em>ACADL</em>, <em>SPTLC3</em>, <em>ETFDH</em>, <em>SLC16A9</em>) the genetic variant is located in or near enzyme or solute carrier coding genes, where the associating metabolic traits match the proteins’ function. Many of these loci are located in rate limiting steps of important enzymatic reactions. Use of metabolite concentration ratios as proxies for enzymatic reaction rates reduces the variance and yields robust statistical associations with p-values between 3×10<sup>−24</sup> and 6.5×10<sup>−179</sup>. These loci explained 5.6% to 36.3% of the observed variance. For several loci, associations with clinically relevant parameters have previously been reported.</p></div></div><div id="idm140400179694384" class="tsec sec headless whole_rhythm"><p id="P10" class="p p-first">We have previously identified frequent genetic polymorphisms with large effects sizes that alter an individual’s metabolic capacities <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918455">1</a></sup>. In that study we described genetic variants in metabolism-related genes that lead to specific and clearly differentiated metabolic phenotypes, which we call “genetically determined metabotypes”. We argue that knowledge of these genetically determined metabotypes in human populations is key to identify the contributions and interactions of genetic and environmental factors in the etiology of complex diseases, providing a new paradigm for the study of gene-environment interactions. However, our original genome-wide association (GWA) study was limited in power due to its modest number of participants (n=284). In an effort to identify new major genetically determined metabotypes of biomedical relevance, we conducted a GWA study with metabolic traits in human serum in a much larger population using eight times the number of subjects.</p><p id="P11">Genotyping of the KORA samples was performed using the <em>Affymetrix 6.0 GeneChip array</em> and of TwinsUK using the <em>Illumina Hap317K chip</em>. Fasting serum concentrations of 163 metabolites, covering a biologically relevant panel of amino acids, sugars, acylcarnitines, and phospholipids, were determined by electrospray ionization tandem mass spectrometry (ESI-MS/MS) using the <em>Biocrates AbsoluteIDQ</em>™ targeted metabolomics technology. A full list of the measured metabolites and abbreviations used in this paper and their biological role is presented in the Online Methods section. Motivated by our previous finding that use of metabolite concentration ratios as proxies for enzymatic reaction rates reduces the variance and yields robust statistical associations <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918434">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918440">2</a></sup>, we tested all of the 163 metabolite concentrations and also all possible metabolite concentration ratios (163*162 = 26,406 traits) with a linear additive model for association with all SNPs that passed our selection criteria. The corresponding estimated genome-wide significance level after correction for testing 517,480 SNPs (MAF &gt; 10%) and 26,406 multiple metabolic trait combinations is p = 3.64×10<sup>−12</sup> (see Online Methods). This hypothesis-free approach allows the genetics to highlight pairs of metabolites that are more likely to be coupled either biochemically or physiologically.</p><p id="P12">We applied a two-step discovery design in the KORA F4 population, followed by a replication in the TwinsUK cohort. Starting with a first discovery step based on samples of 1,029 male and female individuals of Southern German origin from the KORA F4 population, we selected all loci with p-values of association &lt;10<sup>−7</sup> for metabolite concentrations and &lt;10<sup>−9</sup> for concentration ratios in a genome-wide association screen (<a href="/pmc/articles/PMC3773904/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). 32 loci satisfy these criteria. One SNP for each locus was then tested in a second step in an independent sample of 780 participants selected from the remaining KORA F4 population. Identical genotyping and metabolomics techniques as in the first step were used. The metabolomics and genotyping experiments for this second step were conducted independently and after completion of the initial study at several months interval. Using data from all 1,809 KORA individuals, joint p-values of association were computed. Although this approach is less well powered than a full genome-wide joint analysis, it reflects the historical way in which we selected SNPs for follow-up. The top joint p-values from all 1,809 KORA samples are presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> and a full list is available on request.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3773904/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140400180177040" class="figure"><a href="/pmc/articles/PMC3773904/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3773904_emss-54332-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-54332-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3773904/bin/emss-54332-f0001.jpg"></a></div><div id="largeobj_idm140400180177040" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3773904/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3773904/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Manhattan plot of the strength of association with metabolite concentrations (top; data points with p&lt;10<sup>−7</sup> are plotted in red) and concentration ratios (bottom; data points with p&lt;10<sup>−9</sup> are plotted in red); based on association with 1029 samples (step 1 of discovery stage); for each SNP only the metabolic trait with the lowest p-value of association is shown; multiple dots thus indicate that several SNPs support the association at that locus (data provided as <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p></div></div></div><p id="P13">We identified fifteen loci where the strength of association (indicated by decreasing p-values) increases when additional data is added and selected only these for further investigation. All fifteen loci display genome-wide significant p-values of association that are smaller than 3.64×10<sup>−12</sup> in this joint analysis (<a href="/pmc/articles/PMC3773904/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; local association plots, boxplots by genotype, and QQ-plots are presented in <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Fig. 1-3</a>). In a third step, as a replication in an independent population, we used metabolomics data measured on our platform from serum samples of 422 female participants of the TwinsUK cohort. Nine of the fifteen loci are replicated (p&lt;0.05) after Bonferroni correction for 15 tests. Five loci displayed signals of association with similar effect size estimates but were above this significance threshold and should at present be considered as non-replicated. However, as we show in the discussion of the individual loci, most of these five “suggestive loci” are supported by biological evidence (<em>CPS1</em>, <em>SCD</em>, <em>SLC22A4</em>, <em>PHGDH</em>) and two of them represent indirect replications of previous studies (<em>CPS1</em> and <em>SLC22A4</em>). Note also that the TwinsUK study had only limited power due to its smaller sample size. Moreover, all but one SNP were imputed, and the signal to noise ratio in the TwinsUK metabolomics data was about 20% higher than in KORA. We therefore believe that these five suggestive loci are worthy of analysis in further studies and report relevant biological information on these loci in the <a href="#SD3" rid="SD3" class=" supplementary-material">supplementary material</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Summary of the 15 loci that were identified in the in the KORA population. Loci that were then replicated in the TwinsUK cohort (p&lt;0.05after correction for 15 tests) are indicated by a ‘*’. Beta’ is the estimation of the effect per allele copy, normalized by the mean of the metabolic trait. Eta<sup>2</sup> is the proportion of the variance that can be explained by the genotype in the linear model. Gene variants are reported on the positive strand of NCBI build 36. Additional information on the 50 strongest associating metabolic traits for each locus is provided as <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>. Additional p-values computed using log-scaled metabolite concentration ratios and also from the first discovery step alone are given in <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Table 3</a>. Abbreviations for metabolite names are as follows: phophatidylcholine (PC), acyl-acyl-PC (aa), acyl-alkyl-PC (ae), lyso-PC (a), sphingomyelin (SM), acylcarnitine (C), composition of fatty acid side chains (x:y, x=number of carbons, y=number of double bonds), with decarboxyl group (DC), with hydroxyl group (OH), with methyl group (M). A full list of metabolites determined in this study is provided in the Online Methods section.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140400245557824" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">rs-number</th><th rowspan="1" colspan="1" valign="middle" align="center">SNP type</th><th rowspan="1" colspan="1" valign="middle" align="center">Locus</th><th rowspan="1" colspan="1" valign="middle" align="center">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">Coded/<br>noncoded<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="center">Position</th><th rowspan="1" colspan="1" valign="middle" align="center">Minor<br>allele<br>frequency</th><th rowspan="1" colspan="1" valign="middle" align="center">Strongest<br>association</th><th rowspan="1" colspan="1" valign="middle" align="center">N<br>KORA</th><th rowspan="1" colspan="1" valign="middle" align="center">beta’<br>KORA</th><th rowspan="1" colspan="1" valign="middle" align="center">eta<sup>2</sup><br>KORA</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value<br>KORA</th><th rowspan="1" colspan="1" valign="middle" align="center">N<br>TwinsUK</th><th rowspan="1" colspan="1" valign="middle" align="center">beta’<br>TwinsUK</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value<br>TwinsUK</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs174547</td><td rowspan="1" colspan="1" valign="middle" align="center">intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">61,327,359</td><td rowspan="1" colspan="1" valign="middle" align="center">30.4%</td><td rowspan="1" colspan="1" valign="middle" align="center">PC aa C36:3 /<br>PC aa C36:4</td><td rowspan="1" colspan="1" valign="middle" align="center">1806</td><td rowspan="1" colspan="1" valign="middle" align="center">0.151</td><td rowspan="1" colspan="1" valign="middle" align="center">36.3%</td><td rowspan="1" colspan="1" valign="middle" align="center">6.5×10<sup>−179</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">422</td><td rowspan="1" colspan="1" valign="middle" align="center">0.156</td><td rowspan="1" colspan="1" valign="middle" align="center">2.0×10<sup>−26</sup> *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2014355</td><td rowspan="1" colspan="1" valign="middle" align="center">intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ACADS</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">12</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">119,659,907</td><td rowspan="1" colspan="1" valign="middle" align="center">27.7%</td><td rowspan="1" colspan="1" valign="middle" align="center">C3 / C4</td><td rowspan="1" colspan="1" valign="middle" align="center">1790</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.218</td><td rowspan="1" colspan="1" valign="middle" align="center">21.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">5.1×10<sup>−96</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">416</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.229</td><td rowspan="1" colspan="1" valign="middle" align="center">2.1×10<sup>−24</sup> *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs211718</td><td rowspan="1" colspan="1" valign="middle" align="center">upstream</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ACADM</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">C/T</td><td rowspan="1" colspan="1" valign="middle" align="center">75,879,263</td><td rowspan="1" colspan="1" valign="middle" align="center">30.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">C12 / C10</td><td rowspan="1" colspan="1" valign="middle" align="center">1804</td><td rowspan="1" colspan="1" valign="middle" align="center">0.120</td><td rowspan="1" colspan="1" valign="middle" align="center">14.6%</td><td rowspan="1" colspan="1" valign="middle" align="center">1.3×10<sup>−63</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">299</td><td rowspan="1" colspan="1" valign="middle" align="center">0.133</td><td rowspan="1" colspan="1" valign="middle" align="center">2.8×10<sup>−11</sup> *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2286963</td><td rowspan="1" colspan="1" valign="middle" align="center">coding</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ACADL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">T/G</td><td rowspan="1" colspan="1" valign="middle" align="center">210,768,295</td><td rowspan="1" colspan="1" valign="middle" align="center">36.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">C9 / C10:2</td><td rowspan="1" colspan="1" valign="middle" align="center">1806</td><td rowspan="1" colspan="1" valign="middle" align="center">0.219</td><td rowspan="1" colspan="1" valign="middle" align="center">13.8%</td><td rowspan="1" colspan="1" valign="middle" align="center">3.1×10<sup>−60</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">421</td><td rowspan="1" colspan="1" valign="middle" align="center">0.312</td><td rowspan="1" colspan="1" valign="middle" align="center">2.4×10<sup>−19</sup> *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs9393903</td><td rowspan="1" colspan="1" valign="middle" align="center">intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ELOVL2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">G/A</td><td rowspan="1" colspan="1" valign="middle" align="center">11,150,895</td><td rowspan="1" colspan="1" valign="middle" align="center">24.6%</td><td rowspan="1" colspan="1" valign="middle" align="center">PC aa C40:3 /<br>PC aa C42:5</td><td rowspan="1" colspan="1" valign="middle" align="center">1803</td><td rowspan="1" colspan="1" valign="middle" align="center">0.087</td><td rowspan="1" colspan="1" valign="middle" align="center">9.8%</td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−42</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">419</td><td rowspan="1" colspan="1" valign="middle" align="center">0.076</td><td rowspan="1" colspan="1" valign="middle" align="center">4.0×10<sup>−5</sup> *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2216405</td><td rowspan="1" colspan="1" valign="middle" align="center">downstream</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CPS1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">A/G</td><td rowspan="1" colspan="1" valign="middle" align="center">211,325,139</td><td rowspan="1" colspan="1" valign="middle" align="center">18.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">Glycine /<br>PC ae C38:2</td><td rowspan="1" colspan="1" valign="middle" align="center">1792</td><td rowspan="1" colspan="1" valign="middle" align="center">0.129</td><td rowspan="1" colspan="1" valign="middle" align="center">7.1%</td><td rowspan="1" colspan="1" valign="middle" align="center">1.9×10−30</td><td rowspan="1" colspan="1" valign="middle" align="center">420</td><td rowspan="1" colspan="1" valign="middle" align="center">0.094</td><td rowspan="1" colspan="1" valign="middle" align="center">0.014</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs7156144</td><td rowspan="1" colspan="1" valign="middle" align="center">upstream</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PLEKHH1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">14</td><td rowspan="1" colspan="1" valign="middle" align="center">G/A</td><td rowspan="1" colspan="1" valign="middle" align="center">67,049,466</td><td rowspan="1" colspan="1" valign="middle" align="center">41.4%</td><td rowspan="1" colspan="1" valign="middle" align="center">PC ae C32:1 /<br>PC ae C34:1</td><td rowspan="1" colspan="1" valign="middle" align="center">1799</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.042</td><td rowspan="1" colspan="1" valign="middle" align="center">6.6%</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−28</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">405</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.023</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0024 *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs11158519</td><td rowspan="1" colspan="1" valign="middle" align="center">intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SYNE2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">14</td><td rowspan="1" colspan="1" valign="middle" align="center">G/A</td><td rowspan="1" colspan="1" valign="middle" align="center">63,434,338</td><td rowspan="1" colspan="1" valign="middle" align="center">14.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">PC ae C38:1 /<br>PC aa C28:1</td><td rowspan="1" colspan="1" valign="middle" align="center">1763</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.083</td><td rowspan="1" colspan="1" valign="middle" align="center">6.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">1.5×10<sup>−27</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">394</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.098</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0075</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs168622</td><td rowspan="1" colspan="1" valign="middle" align="center">upstream</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SPTLC3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">20</td><td rowspan="1" colspan="1" valign="middle" align="center">G/T</td><td rowspan="1" colspan="1" valign="middle" align="center">12,914,089</td><td rowspan="1" colspan="1" valign="middle" align="center">37.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">SM (OH) C24:1 /<br>SM C16:0</td><td rowspan="1" colspan="1" valign="middle" align="center">1796</td><td rowspan="1" colspan="1" valign="middle" align="center">0.061</td><td rowspan="1" colspan="1" valign="middle" align="center">5.8%</td><td rowspan="1" colspan="1" valign="middle" align="center">5.2×10<sup>−25</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">421</td><td rowspan="1" colspan="1" valign="middle" align="center">0.061</td><td rowspan="1" colspan="1" valign="middle" align="center">2.9×10<sup>−4</sup> *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs8396</td><td rowspan="1" colspan="1" valign="middle" align="center">downstream</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ETFDH</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">159,850,267</td><td rowspan="1" colspan="1" valign="middle" align="center">29.8%</td><td rowspan="1" colspan="1" valign="middle" align="center">C14:1-OH / C10</td><td rowspan="1" colspan="1" valign="middle" align="center">1778</td><td rowspan="1" colspan="1" valign="middle" align="center">0.102</td><td rowspan="1" colspan="1" valign="middle" align="center">5.6%</td><td rowspan="1" colspan="1" valign="middle" align="center">3.5×10<sup>−24</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">421</td><td rowspan="1" colspan="1" valign="middle" align="center">0.114</td><td rowspan="1" colspan="1" valign="middle" align="center">3.1×10<sup>−7</sup> *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs7094971</td><td rowspan="1" colspan="1" valign="middle" align="center">intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SLC16A9</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">A/G</td><td rowspan="1" colspan="1" valign="middle" align="center">61,119,570</td><td rowspan="1" colspan="1" valign="middle" align="center">13.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">C0</td><td rowspan="1" colspan="1" valign="middle" align="center">1786</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.091</td><td rowspan="1" colspan="1" valign="middle" align="center">4.6%</td><td rowspan="1" colspan="1" valign="middle" align="center">3.8×10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">421</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.089</td><td rowspan="1" colspan="1" valign="middle" align="center">9.3×10<sup>−4</sup> *</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2046813</td><td rowspan="1" colspan="1" valign="middle" align="center">upstream</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ACSL1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">186,006,153</td><td rowspan="1" colspan="1" valign="middle" align="center">32.2%</td><td rowspan="1" colspan="1" valign="middle" align="center">PC ae C44:5 /<br>PC ae C42:5</td><td rowspan="1" colspan="1" valign="middle" align="center">1804</td><td rowspan="1" colspan="1" valign="middle" align="center">0.033</td><td rowspan="1" colspan="1" valign="middle" align="center">4.1%</td><td rowspan="1" colspan="1" valign="middle" align="center">3.6×10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">409</td><td rowspan="1" colspan="1" valign="middle" align="center">0.002</td><td rowspan="1" colspan="1" valign="middle" align="center">0.91</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs603424</td><td rowspan="1" colspan="1" valign="middle" align="center">upstream</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SCD</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">G/A</td><td rowspan="1" colspan="1" valign="middle" align="center">102,065,469</td><td rowspan="1" colspan="1" valign="middle" align="center">19.4%</td><td rowspan="1" colspan="1" valign="middle" align="center">C14 / C16:1</td><td rowspan="1" colspan="1" valign="middle" align="center">1805</td><td rowspan="1" colspan="1" valign="middle" align="center">0.054</td><td rowspan="1" colspan="1" valign="middle" align="center">4.0%</td><td rowspan="1" colspan="1" valign="middle" align="center">1.5×10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">422</td><td rowspan="1" colspan="1" valign="middle" align="center">0.023</td><td rowspan="1" colspan="1" valign="middle" align="center">0.10</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs272889</td><td rowspan="1" colspan="1" valign="middle" align="center">intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SLC22A4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">G/A</td><td rowspan="1" colspan="1" valign="middle" align="center">131,693,277</td><td rowspan="1" colspan="1" valign="middle" align="center">38.5%</td><td rowspan="1" colspan="1" valign="middle" align="center">Valine / C5</td><td rowspan="1" colspan="1" valign="middle" align="center">1809</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.075</td><td rowspan="1" colspan="1" valign="middle" align="center">3.3%</td><td rowspan="1" colspan="1" valign="middle" align="center">7.9×10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">422</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.052</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0098</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs541503</td><td rowspan="1" colspan="1" valign="middle" align="center">upstream</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PHGDH</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">120,009,820</td><td rowspan="1" colspan="1" valign="middle" align="center">37.9%</td><td rowspan="1" colspan="1" valign="middle" align="center">Ornithine /<br>Serine</td><td rowspan="1" colspan="1" valign="middle" align="center">1809</td><td rowspan="1" colspan="1" valign="middle" align="center">0.058</td><td rowspan="1" colspan="1" valign="middle" align="center">2.7%</td><td rowspan="1" colspan="1" valign="middle" align="center">3.0×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">419</td><td rowspan="1" colspan="1" valign="middle" align="center">0.029</td><td rowspan="1" colspan="1" valign="middle" align="center">0.087</td></tr></tbody></table></div><div id="largeobj_idm140400245557824" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3773904/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P14">When the functional roles of the genes in these loci are considered, we can draw the most comprehensive to date view of genetic variation in human metabolism. The connections between these genes in the human metabolism are outlined in <a href="/pmc/articles/PMC3773904/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>. As each locus and association has its particularities and may be of different relevance to the individual reader, we provide a full discussion of all loci from their specific perspectives as a Supplementary Note, while focusing here on the overall picture. <a href="/pmc/articles/PMC3773904/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> summarizes the relevant information from that discussion for all loci. For eight of nine fully replicated genetic polymorphisms and also four of the five suggestive loci, the genetic variant is located in or near enzyme or solute carrier coding genes, where the associating metabolic traits match the proteins’ function. Many of these polymorphisms are located in rate limiting steps of important enzymatic reactions. Four loci replicate associations from previous studies (<em>FADS1</em>, <em>ACADS</em>, <em>ACADM</em>, <em>ELOVL2</em>), including our own. Two of the suggestive loci replicate previous studies on related metabolic traits (<em>SLC22A4</em> and <em>CPS1</em>). For some loci, new hypotheses on the genes’ function can be derived from the associating metabolite pattern (e.g. <em>SLC16A9</em> and <em>PLEKHH1</em>). For three loci, an association with clinical endpoints has previously been reported (<em>SLC22A4</em> with Crohn’s disease, <em>FADS1</em> with hyperactivity and cholesterol/triglyceride levels, <em>ACADS</em> as susceptibility locus for ethylmalonic aciduria).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3773904/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140400171195552" class="figure"><a href="/pmc/articles/PMC3773904/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3773904_emss-54332-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-54332-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3773904/bin/emss-54332-f0002.jpg"></a></div><div id="largeobj_idm140400171195552" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3773904/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3773904/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">A systemic view of genetic variation in human metabolism as identified by this study. Eight of nine fully replicated genetic polymorphisms (beige) and also four of five suggestive loci (grey) are located in or near enzyme-coding genes that are central to the different processes in human lipid metabolism, that is, the beta-oxidation (<em>ACADS</em>, <em>ACADM</em>, <em>ACADL</em>), the polyunsaturated fatty acid biosynthesis (<em>FADS1</em>, <em>ELOVL2</em>), the fatty acid synthesis (<em>SCD</em>), the breakdown of fats and proteins to energy (<em>ETFDH</em>), and the biosynthesis of phospholipids (<em>SPTLC3</em>). Two SNPs are located in or near genes coding for carrier proteins (<em>SLC22A4</em>, <em>SLC16A9</em>) and two SNPs involve enzymes that are related to amino acid metabolism (<em>PHGDH</em>, <em>CPS1</em>). Only for two genetic variants does the attribution of a metabolic function remain elusive (<em>PLEKHH1</em>, <em>SYNE2</em>). For each locus the most strongly associating single metabolite is indicated in red.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Summary of findings for the loci discussed in this paper; top: fully replicated loci, middle: loci from discovery phase that were not replicated in TwinsUK; bottom: already described risk loci that associate here with metabolic traits but that do not reach the genome-wide significance level. Metabolites with the lowest p-values of association for these loci with respect to metabolite concentrations are reported; note that concentration ratios with the lowest p-values of association are given in <a href="/pmc/articles/PMC3773904/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and data for the 50 strongest associations is provided as <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>. Loci that have already been published elsewhere are indicated as replication. The term “indirect replication” is used when the study in question did not measure the same, but related metabolites. Commonly used but non-standard gene names are mentioned in parentheses.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140400180681024" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">Locus</th><th rowspan="1" colspan="1" valign="middle" align="center">Gene function</th><th rowspan="1" colspan="1" valign="middle" align="center">Metabolites</th><th rowspan="1" colspan="1" valign="middle" align="center">Notes</th><th rowspan="1" colspan="1" valign="middle" align="center">Reference</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>Fully replicated loci</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">fatty acid desaturase 1</td><td rowspan="1" colspan="1" valign="middle" align="center">lysoPC a C20:4,<br>PC aa C38:4, PC aa C36:4</td><td rowspan="1" colspan="1" valign="middle" align="center">Risk locus for high cholesterol <sup><a href="#R6" rid="R6" class=" bibr popnode">6</a>-<a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>; linked to hyperactivity <sup><a href="#R23" rid="R23" class=" bibr popnode">23</a></sup>;<br>covariate in association breastfeeding-IQ <sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup>; C20:4 fatty acids are<br>products of <em>FADS1</em>.</td><td rowspan="1" colspan="1" valign="middle" align="center">Replication of KORA <sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup> and indirect<br>replication of INCHIANTI <sup><a href="#R25" rid="R25" class=" bibr popnode">25</a></sup>, who<br>measured selected PUFAs</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ACADS</em>
<br>
<em>(SCAD)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">acyl-Coenzyme A<br>dehydrogenase,<br>C-2 to C-3 short chain</td><td rowspan="1" colspan="1" valign="middle" align="center">C4</td><td rowspan="1" colspan="1" valign="middle" align="center">Predisposition allele that can cause <em>ACADS</em> deficiency if<br>additional factors are present <sup><a href="#R26" rid="R26" class=" bibr popnode">26</a></sup>; C4 fatty acids are substrates of<br><em>ACADS</em>.</td><td rowspan="1" colspan="1" valign="middle" align="center">Replication of KORA <sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup> and a target<br>gene study in a US population <sup><a href="#R27" rid="R27" class=" bibr popnode">27</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ACADM</em>
<br>
<em>(MCAD)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">acyl-Coenzyme A<br>dehydrogenase,<br>C-4 to C-12 straight chain</td><td rowspan="1" colspan="1" valign="middle" align="center">C6<sup>*</sup>, C8, C10, C10:1</td><td rowspan="1" colspan="1" valign="middle" align="center">C4 to C12 fatty acids are substrates of <em>ACADM</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Replication of KORA <sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ACADL</em>
<br>
<em>(LCAD)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">acyl-Coenzyme A<br>dehydrogenase,<br>long chain</td><td rowspan="1" colspan="1" valign="middle" align="center">C9</td><td rowspan="1" colspan="1" valign="middle" align="center">Coding SNP; patented in a method for diagnosing the risk of<br>thermo-labile phenotype in influenza-associated<br>encephalopathy <sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup>; long chain fatty acids (incl. C9) are substrates<br>of <em>ACADL</em>.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ELOVL2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">elongation of very long chain<br>fatty acids<br>(FEN1/Elo2, SUR4/Elo3,<br>yeast)-like 2</td><td rowspan="1" colspan="1" valign="middle" align="center">PC aa C42:5, PC aa C42:6</td><td rowspan="1" colspan="1" valign="middle" align="center">PUFA products and/or substrates of <em>ELOVL2</em> are incorporated<br>into phosphatidylcholines, such as PC aa C42:5 and PC aa C42:6.</td><td rowspan="1" colspan="1" valign="middle" align="center">Indirect replication of INCHIANTI <sup><a href="#R25" rid="R25" class=" bibr popnode">25</a></sup>,<br>who measured selected PUFAs</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PLEKHH1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">pleckstrin homology domain<br>containing,<br>family H (with MyTH4<br>domain) member 1</td><td rowspan="1" colspan="1" valign="middle" align="center">PC ae C36:5, PC ae C32:2</td><td rowspan="1" colspan="1" valign="middle" align="center">Gene function unknown; our results suggest that this locus may<br>be functionally related to the metabolism of plasmalogens, such<br>as PC ae C36:5 and PC ae C32:2.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SPTLC3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">serine palmitoyltransferase,<br>long chain base subunit 3</td><td rowspan="1" colspan="1" valign="middle" align="center">SM (OH) C24:l</td><td rowspan="1" colspan="1" valign="middle" align="center">The SPT complex catalyses the rate limiting step of SM<br>biosynthesis; <em>SPTLC3</em> is the regulatory subunit; association with<br>ratios between sphingolipids indicates that this polymorphism<br>may modify the cellular mechanism to adjust SPT activity to<br>tissue-specific requirements of sphingolipid synthesis.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ETFDH</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">electron-transferring-<br>flavoprotein<br>dehydrogenase</td><td rowspan="1" colspan="1" valign="middle" align="center">C10, C8</td><td rowspan="1" colspan="1" valign="middle" align="center">Breakdown of fats and proteins to energy (beta-oxidation of<br>substrates such as C8 and C10 fatty acids); genetic variant in this<br>locus also involves changes in acylcarnitine hydroxylation and<br>carboxylation.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SLC16A9</em>
<br>
<em>(MCT9)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">solute carrier family 16,<br>member 9<br>(monocarboxylic acid<br>transporter 9)</td><td rowspan="1" colspan="1" valign="middle" align="center">C0</td><td rowspan="1" colspan="1" valign="middle" align="center">Hypothesis: carnitine (C0) is a monocarboxylic acid and may<br>therefore be the substrate of this “orphan” transporter.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td colspan="5" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<strong>“Suggestive loci“ from discovery stage that were not replicated in TwinsUK</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CPS1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">carbamoyl-phosphate<br>synthetase 1,<br>mitochondrial</td><td rowspan="1" colspan="1" valign="middle" align="center">Glycine</td><td rowspan="1" colspan="1" valign="middle" align="center">Locus associates with glycine to arginine and glutamine ratios;<br>these amino acids are linked to carbamoyl phosphate in the urea<br>cycle.</td><td rowspan="1" colspan="1" valign="middle" align="center">Indirect replication of Paré et al.<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup>,<br>who show an association with the<br>related metabolite homocysteine.</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SYNE2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">spectrin repeat containing,<br>nuclear envelope 2</td><td rowspan="1" colspan="1" valign="middle" align="center">PC aa C28:l</td><td rowspan="1" colspan="1" valign="middle" align="center">Gene function unknown; causative SNP may possibly be linked<br>to a modification in the nearby sphingosine-l-phosphate<br>phosphatase 1 (SGPP1) activity.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SCD</em>
<br>
<em>(FADS5)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">stearoyl-CoA desaturase<br>(delta-9-desaturase)</td><td rowspan="1" colspan="1" valign="middle" align="center">lysoPC a C16:l</td><td rowspan="1" colspan="1" valign="middle" align="center">Fatty acid C16:l is the direct product of <em>SCD</em>; <em>SCD</em> activity is<br>linked to adiposity and plasma lipid profiles <sup><a href="#R29" rid="R29" class=" bibr popnode">29</a></sup>; C16:l<br>palmitoleate regulates insulin signaling and glucose<br>metabolism <sup><a href="#R30" rid="R30" class=" bibr popnode">30</a></sup></td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SLC22A4</em>
<br>
<em>(OCTN1)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">solute carrier family 22<br>(organic cation/<br>ergothioneine transporter),<br>member 4</td><td rowspan="1" colspan="1" valign="middle" align="center">C5</td><td rowspan="1" colspan="1" valign="middle" align="center">Locus associates with Crohn’s disease; association with C5<br>carnitine suggests role of modified transport of this metabolite<br>in disease etiology; the causative SNP may also be located in<br>neighboring <em>SLC22A5</em> (<em>OCTN2</em>) gene <sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup>.</td><td rowspan="1" colspan="1" valign="middle" align="center">Indirect replication of Peltekova et<br>.al<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup>, who show that this genetic<br>variation at this locus associates with<br>carnitine concentrations<br>(Fig. 3a in Peltekova et al. <sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup>);</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PHGDH</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">phosphoglycerate<br>dehydrogenase</td><td rowspan="1" colspan="1" valign="middle" align="center">Serine</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>PHGDH</em> catalyses the rate limiting step of serine biosynthesis;<br><em>PHGDH</em> deficiency leads to congenital microcephaly, seizures,<br>and severe psychomotor retardation <sup><a href="#R32" rid="R32" class=" bibr popnode">32</a></sup>.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td colspan="5" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<strong>Already described risk loci that associate here with metabolic traits above the genome-wide significance level</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APO-</em>
<br>
<em>cluster</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">apolipoprotein</td><td rowspan="1" colspan="1" valign="middle" align="center">PC aa C36:2/PC aa C38:1<br>(p=1.8×10<sup>−11</sup>)</td><td rowspan="1" colspan="1" valign="middle" align="center">SNP rs964184 in the <em>APOA1-APOC3-APOA4-APOA5</em> cluster<br>strongly associates with blood triglyceride levels (p&lt;10<sup>−60</sup>) <sup><a href="#R4" rid="R4" class=" bibr popnode">4</a></sup><br></td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GCKR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">glucokinase (hexokinase 4)<br>regulator</td><td rowspan="1" colspan="1" valign="middle" align="center">PC ae C34:2/PC aa C32:2<br>(p=3.2×10<sup>−8</sup>)</td><td rowspan="1" colspan="1" valign="middle" align="center">SNP rs1260326 (P446L polymorphism) in <em>GCKR</em> inversely<br>modulates fasting glucose (p=8×10<sup>−13</sup>) and triglyceride (p=1×10<sup>−4</sup>)<br>levels and reduces type 2 diabetes risk in the DESIR prospective<br>general French population <sup><a href="#R10" rid="R10" class=" bibr popnode">10</a></sup></td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MTNR1B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">melatonin receptor 1B</td><td rowspan="1" colspan="1" valign="middle" align="center">tryptophan/phenylalanine<br>(p=5.7×10<sup>−6</sup>)</td><td rowspan="1" colspan="1" valign="middle" align="center">SNP rs10830963 in the melatonin-receptor (<em>MTNR1B</em>) associates<br>with fasting glucose <sup><a href="#R11" rid="R11" class=" bibr popnode">11</a></sup>; phenylalanine is a precursor of<br>melatonin.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm140400180681024" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3773904/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>Note that C6 cannot be discerned from C4:1-DC by the tandem mass spectrometry technique used here.</div></div></div><p id="P15">For several other loci, loss of function of the corresponding gene leads to severe disorders (e.g. <em>ACADM</em>, <em>ACADL</em>, <em>ETFDH</em>) indicating that the genetic variants we identify here or variants in linkage disequilibrium may induce a related but likely more moderate phenotype. This is in line with findings of a recent GWA study on kidney function which identified <em>UMOD</em> to be associated with glomerular filtration rate <sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918431">3</a></sup>. Rare mutations in the <em>UMOD</em> gene are known to be the cause of monogenic autosomal dominant kidney diseases. Common mutations in the same gene region can be the cause of disease-related phenotypes of less severity on the population level. As discussed in Gieger et al. <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918454">1</a></sup>, a ratio between the concentrations of two metabolites that are linked to a substrate/product pair of some enzymatic reaction may constitute an approximation of the conversion rate of that reaction. From the effect size of the association (beta’ in the linear model), one can therefore derive the per-allele difference in metabolic capacities of an individual with respect to the considered enzymatic reaction. For instance, beta’ of the association of rs211718 (<em>ACADM</em>) with C12/C10 is 0.12. Assuming an additive-per-copy effect (see <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Fig. 2</a>), this implies that individuals who are homozygotes of the major allele of <em>ACADM</em> burn fatty acids with a chain length of 12 carbons about 24% faster than carriers of two copies of the minor allele. Similar arguments hold for the other loci (beta’ for all loci is reported in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>).</p><p id="P16">The SNPs identified in this study can now be used for the identification of true positives in GWA studies with clinical parameters. As an example, in our previous study <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918442">1</a></sup> we suggested <em>FADS1</em> to be a risk locus for perturbed blood lipid parameters. This was supported by the observed association with different phospholipids and the fact that two published GWA studies with lipid levels reported p-values of association for the <em>FADS1</em> locus with LDL, HDL, and total cholesterol levels that ranged between 1.89×10<sup>−4</sup> and 6.07×10<sup>−5 </sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918438">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630685235">5</a></sup>. These associations had not been included in the list of potential candidates for replication in those studies, as their p-values taken alone were not sufficiently small in the context of a “classical” GWA study. Three more well powered GWA studies with lipid parameters have only recently confirmed this prediction <sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918447">6</a>-<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918443">8</a></sup>, thereby proving that a combination of a GWA study using metabolomic phenotypic traits with data from previous GWA studies can identify new candidate SNPs associated to known phenotypes of clinical relevance. To facilitate further studies we provide a list of all high scoring associations of this study as supplemental data set for a similar use by other consortia (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p><p id="P17">The association data presented in this study can be used to learn more about already known risk loci. Using the catalog of published GWA studies <sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup> we identified three such loci that also associate with metabolic traits in our study (<a href="/pmc/articles/PMC3773904/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). For instance, SNP rs964184 in the apolipoprotein <em>APOA1-APOC3-APOA4-APOA5</em> cluster strongly associates with blood triglyceride levels (p&lt;10<sup>−60</sup>) <sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918445">4</a></sup>. We find that the same SNP associates with ratios between different phosphatidylcholines (e.g. PC aa C36:2/PC aa 38:1, p=1.8×10<sup>−10</sup>), which are biochemically connected to triglycerides by the intermediary of only a few enzymatic reaction steps. SNP rs1260326 (P446L polymorphism) in the glucose kinase regulator protein (<em>GCKR</em>) inversely modulates fasting glucose (p=8×10<sup>−13</sup>) and triglyceride levels (p=1×10<sup>−4</sup>) and reduces type 2 diabetes risk in the DESIR prospective general French population <sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918439">10</a></sup>. This locus associates with different ratios between plasmalogens and phosphatidylcholines (e.g. PC ae C34:2/PC aa C32:2, p=3.2×10<sup>−8</sup>), thereby providing new avenues for further investigation on the functional background of this association. SNP rs10830963 in the melatonin-receptor (<em>MTNR1B</em>) associates with fasting glucose <sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918453">11</a></sup>. The same SNP associates in this study with tryptophan/phenylalanine ratios (p=5.7×10<sup>−6</sup>). This is of particular interest since phenylalanine is a precursor of melatonin, indicating a functional relationship between this pathway and the regulation of glucose homeostasis. We expect the list of loci with parallel association of clinically relevant parameters and metabolic traits to grow as new GWA studies will become available, and therefore provide our association data as supplemental material for such use.</p><p id="P18">The SNPs identified in this study can now be used in clinical studies for association with response to drug treatment. One published example is a common polymorphism in the dihydropyrimidine dehydrogenase (<em>DPYP</em>) gene that associates strongly with fluoropyrimidine-related toxicity in cancer patients <sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918451">12</a></sup>. Carriers of this variant could benefit from individual dose adjustment of the fluoropyrimidine drug or alternate therapies. It is now possible to use the here identified SNPs in association studies with phenotypes that are specific to a disease, such as the development of particular complications during the course of a disease or treatment. One published example is a SNP in carnitine palmitoyltransferase II (<em>CPT-II</em>) that is a predisposing factor for influenza-associated encephalopathy (thermo-labile phenotype) <sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918433">13</a></sup>. Indeed, the SNP that we identified here in the <em>ACADL</em> gene has been patented by others <sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> along with the <em>CPT-II</em> polymorphism as a method of diagnosing risk of a thermo-labile phenotype.</p><p id="P19" class="p">In summary, this study allowed us to draw a systemic perspective of the genetic variation that is found in human metabolism. In contrast to most GWA studies with clinically relevant endpoints, it appears that for metabolic traits most of the associations are linked to genetic variants in genes with a matching metabolic function (<a href="/pmc/articles/PMC3773904/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). The use of metabolite concentration ratios demonstrates a pronounced sharpening of the association with dramatically decreased p-values when compared to an analysis of single metabolites. Moreover, as we show with examples (<em>FADS1</em>, <em>APO</em>-cluster, <em>MTNR1B</em>, <em>GCKR</em>), it allows the deriving of new functional information from GWA studies with clinically relevant endpoints, and this based on a relatively small number of samples. Our study shows the exciting potential of metabolomics to unravel the genetics of human metabolism. The presented genome-wide perspective of genetic variation in human metabolism will continue to improve as more extensive metabolite panels become available for use at a genome-wide scale, including additional studies with nutritional challenges of the participants. We believe that the introduction of metabolomics into the field of molecular epidemiology provides a new and more hypothesis-driven approach to GWA studies.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">ONLINE METHODS</h2><p id="P20" class="p p-first">This study is based on genotyping and metabolic profiling efforts in the German KORA and the British TwinsUK population for which we report the essentials here.</p><div id="S2" class="sec"><h3 id="S2title">Study population</h3><p id="P21" class="p p-first-last">The KORA S4 survey, an independent population-based sample from the general population living in the region of Augsburg, Southern Germany, was conducted in 1999/2001. The standardized examinations applied in the survey (4261 participants, response 67%) have been described in detail <sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918444">15</a> and references therein</sup>. A total of 3,080 subjects participated in a follow-up examination of S4 in 2006-08 (KORA F4), comprising individuals who, at that time, were aged 32–81 years. Informed consent has been given. The study has been approved by the local ethical committee. For the first genome-wide screening step, 1048 blood samples of KORA F4 participants were metabolically characterized. For 1029 samples of this group genome-wide genotype data were also available (509 males, 520 females). In a second step, 972 samples were metabolically characterized in an independent experimental batch. For 780 of these samples genome-wide genotype data were available (374 males, 406 females). For the joint analysis in KORA, metabolomics and genotype data for a total of 1809 individuals were available. No evidence of population stratification has been found in multiple published analyses using the KORA cohort.</p></div><div id="S3" class="sec"><h3 id="S3title">Genotyping and imputation</h3><p id="P22" class="p p-first-last">In KORA F4 genotyping was done using the Affymetrix 6.0 GeneChip array. Analysis in the discovery stage was performed solely on genotyped SNPs. As is current standard for GWA studies, we excluded all X-chromosome-linked SNPs owing to the following reasons: (i) the X chromosome has to be treated differently from the autosomes; (ii) it cannot be predicted which allele is active, (iii) testing males separately results in different sample sizes and power. Imputation of SNPs in the HapMap CEU population was performed using IMPUTE <em><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918448">16</a></sup></em> for use in the regional association plots. In the discovery stage we limited our analysis to SNPs with a moderate to high minor allele frequency (MAF&gt;10%), high genotyping quality (call rate &gt;95%), and with respect to Hardy-Weinberg equilibrium (p(HWE)&gt;0.001). A total of 517,480 SNPs satisfy all of these criteria.</p></div><div id="S4" class="sec"><h3 id="S4title">Blood sampling</h3><p id="P23" class="p p-first-last">Blood samples for metabolic analysis were collected between 2006 and 2008 in parallel with the KORA F4 examinations. To avoid variation due to circadian rhythm, blood was drawn in the morning between 8 and 10 am after a period of overnight fasting. Material was drawn into serum gel tubes, gently inverted two times, followed by 30 min resting at room temperature to obtain complete coagulation. The material was then centrifuged for 10 minutes (2750g; 15°C). Serum was aliquoted and kept for max. 6 hours at 4°C, after which it was deep frozen to −80°C until analysis.</p></div><div id="S5" class="sec"><h3 id="S5title">Metabolite measurements</h3><p id="P24" class="p p-first-last">Liquid handling of serum samples (100 μl) was performed with Hamilton Star (Hamilton Bonaduz AG, Bonaduz, Switzerland) robot and prepared for quantification using the <em>AbsoluteIDQ</em> kit (BIOCRATES Life Sciences AG, Innsbruck, Austria). Sample analyses were done on API 4000 Q TRAP LC/MS/MS System (Applied Biosystems, Darmstadt, Germany) equipped with a Schimadzu Prominence LC20AD pump and a SIL-20AC auto sampler. The complete analytical process was performed using the <em>MetIQ</em>™ software package, which is an integral part of the Absolute<em>IDQ</em>™ kit. We did not apply any data correction, nor were any data points removed. The experimental metabolomics measurement technique is described in detail by patent US 2007/0004044 (accessible online at <a href="http://www.freepatentsonline.com/20070004044.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.freepatentsonline.com/20070004044.html</a>) and in the manufacturer’s manuals. A summary of the method can be found in <sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918450">17</a>,<a href="#R18" rid="R18" class=" bibr popnode">18</a></sup> and a comprehensive overview of the field and the related technologies is given in the review paper by Wenk <sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918437">19</a></sup>. Briefly, a targeted profiling scheme is used to quantitatively screen for known small molecule metabolites using multiple reaction monitoring, neutral loss and precursor ion scans. Quantification of the metabolites of the biological sample is achieved by reference to appropriate internal standards. The method has been proven to be in conformance with 21CFR (Code of Federal Regulations) Part 11, which implies proof of reproducibility within a given error range (see <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Table 4</a> for data). It has been applied in different academic and industrial applications <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918449">1</a>,<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918452">20</a>,<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918436">21</a></sup>. Concentrations of all analyzed metabolites are reported in μM.</p></div><div id="S6" class="sec"><h3 id="S6title">Metabolite panel</h3><p id="P25" class="p p-first-last">In total, 163 different metabolites were detected (<a href="/pmc/articles/PMC3773904/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> in Online Methods). The metabolomics dataset contains 14 amino acids, hexose (H1), free carnitine (C0), 40 acylcarnitines (C<em>x:y</em>), hydroxylacylcarnitines (C(OH)<em>x:y</em>), and dicarboxylacylcarnitines (C<em>x:y</em>-DC), 15 sphingomyelins (SM<em>x:y</em>) and N-hydroxylacyloylsphingosyl-phosphocholine (SM (OH)<em>x:y</em>), 77 phosphatidylcholines (PC, <em>aa</em>=diacyl, <em>ae</em>=acyl-alkyl) and 15 lysophosphatidylcholines. Lipid side chain composition is abbreviated as C<em>x:y</em>, where <em>x</em> denotes the number of carbons in the side chain and <em>y</em> the number of double bonds. E.g. “PC ae C33:1” denotes an acyl-alkyl phosphatidylcholine with 33 carbons in the two fatty acid side chains and a single double bond in one of them. Full biochemical names are provided in <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Table 4</a>. The precise position of the double bonds and the distribution of the carbon atoms in different fatty acid side chains cannot be determined with this technology. In some cases, the mapping of metabolite names to individual masses can be ambiguous. For example, stereo-chemical differences are not always discernible, neither are isobaric fragments. In such cases, possible alternative assignments are indicated.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p6">Metabolite concentrations determined using the <em>Biocrates AbsolutelDQ</em><em>™</em> kit. Full biochemical names are provided in <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Table 4</a>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140400179232400" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">Metabolite class</th><th rowspan="1" colspan="1" valign="middle" align="center">N</th><th rowspan="1" colspan="1" valign="middle" align="center">Metabolite name or abbreviation</th><th rowspan="1" colspan="1" valign="middle" align="center">Biological relevance<br>(selected examples)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">Amino acids</td><td rowspan="1" colspan="1" valign="middle" align="center">14</td><td rowspan="1" colspan="1" valign="middle" align="center">Arginine, Glutamine, Glycine, Histidine, Methionine, Ornithine, Phenylalanine, Proline,<br>Serine, Threonine,<br>Tryptophan, Tyrosine, Valine, (Iso)Leucine</td><td rowspan="1" colspan="1" valign="middle" align="center">Amino acid metabolism, urea-cycle, activity of<br>gluconeogenesis and glycolysis, insulin<br>sensitivity, neurotransmitter metabolism,<br>oxidative stress</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Sum of hexoses</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">H1</td><td rowspan="1" colspan="1" valign="middle" align="center">Carbohydrate metabolism</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Carnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">C0</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Acylcarnitines</td><td rowspan="1" colspan="1" valign="middle" align="center">26</td><td rowspan="1" colspan="1" valign="middle" align="center">C2, C3, C3:1, C4, C4:1, C5, C5:1, C6 (or C4:1-DC), C6:1, C8, C8:1, C9, C10, C10:1, C10:2,<br>C12, C12:1, C14, C14:1, C14:2,<br>C16, C16:1, C16:2, C18, C18:1, C18:2</td><td rowspan="2" colspan="1" valign="middle" align="center">Energy metabolism, fatty acid transport and<br>, mitochondrial fatty acid oxidation, ketosis,<br>oxidative stress, mitochondrial membrane<br>damage</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Hydroxy- and dicarboxy-<br>acylcarnitines</td><td rowspan="1" colspan="1" valign="middle" align="center">14</td><td rowspan="1" colspan="1" valign="middle" align="center">C3-OH, C4-OH (or C3-DC), C5-DC (or C6-OH), C5-OH (or C3-DC-M),<br>C5:1-DC, C5-M-DC, C7-DC, C12-DC, C14:1-OH, C14:2-OH,<br>C16:1-OH, C16:2-OH, C16-OH, C18:1-OH ,</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Sphingomyelins</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">SM C16:0, SM C16:1, SM C18:0, SM C18:1, SM C20:2, SM C22:3,<br>SM C24:0, SM C24:1, SM C26:0, SM C26:1</td><td rowspan="2" colspan="1" valign="middle" align="center">Signaling cascades, membrane damage (e.g.<br>neurodegeneration)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Hydroxysphingomyelins</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">SM (OH) C14:1, SM (OH) C16:1, SM (OH) C22:1,<br>SM (OH) C22:2, SM (OH) C24:1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Diacyl-phosphatidyl-<br>cholines</td><td rowspan="1" colspan="1" valign="middle" align="center">38</td><td rowspan="1" colspan="1" valign="middle" align="center">PC aa C24:0/C26:0/C28:1/C30:0/C30:2/C32:0/C32:1/C32:2/C32:3/<br>C34:1/C34:2/C34:3/C34:4/C36:0/C36:1/C36:2/C36:3/C36:4/C36:5/<br>C36:6/C38:0/C38:1/C38:3/C38:4/C38:5/C38:6/C40:1/C40:2/C40:3/<br>C40:4/C40:5/C40:6/C42:0/C42:1/C42:2/C42:4/C42:5/C42:6</td><td rowspan="2" colspan="1" valign="middle" align="center">Dyslipidemia, membrane composition and<br>damage, fatty acid profile, activity of<br>desaturases</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Acyl-alkyl-phosphatidyl-<br>cholines</td><td rowspan="1" colspan="1" valign="middle" align="center">39</td><td rowspan="1" colspan="1" valign="middle" align="center">PC ae C30:0/C30:1/C30:2/C32:1/C32:2/C34:0/C34:1/C34:2/C34:3/C36:0/<br>C36:1/C36:2/C36:3/C36:4/C36:5/C38:0/C38:1/C38:2/C38:3/C38:4/<br>C38:5/C38:6/C40:0/C40:1/C40:2/C40:3/C40:4/C40:5/C40:6/C42:0/ C42:1/C42:2/C42:3/C42:4/C42:5/C44:3/C44:4/C44:5/C44:6</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">Lyso-phosphatidyl-<br>cholines</td><td rowspan="1" colspan="1" valign="middle" align="center">15</td><td rowspan="1" colspan="1" valign="middle" align="center">lysoPC a C6:0/C14:0/C16:0/C16:1/C17:0/C18:0/C18:1/C18:2/C20:3/<br>C20:4/C24:0/C26:0/C26:1/C28:0/C28:1</td><td rowspan="1" colspan="1" valign="middle" align="center">Degradation of phospholipids, membrane<br>damage, signaling cascades, fatty acid profile</td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">total</td><td rowspan="1" colspan="1" valign="middle" align="center">163</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140400179232400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3773904/table/T3/?report=objectonly">Open in a separate window</a></div></div></div><div id="S7" class="sec"><h3 id="S7title">Statistical analysis</h3><p id="P26" class="p p-first-last">In the statistical analysis only SNPs with a minor allele frequency of at least 10% were included in order to avoid spurious associations due to small numbers. Additive genetic models assuming a trend per copy of the minor allele were used to specify the association between genotype categories and each of the 163 metabolite concentrations as well as all possible metabolite concentration ratios (163*162 = 26.406 traits). No further adjustment was performed. The linear regression algorithm implemented in the statistical analysis system R (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>) was used in the genome wide association study and SPSS for Windows (Version 17.0, Chicago: SPSS Inc.) was used for statistical analysis on a case-by-case level. Motivated by our previous observation that the use of ratios may lead to a strong reduction in the overall variance and a corresponding improvement in the p-values of association <sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918432">20</a></sup>, we also computed all possible pairs of metabolite concentration ratios for those cases and used those ratios as quantitative traits. We present in the main text the results for the untransformed ratios for ease of interpretation. We used a conservative estimate of a genome-wide significance level (using a Bonferroni correction) based on a nominal level of 0.05 is 5.93×10<sup>−10</sup> (0.05 / (163*517,480). To estimate whether deviation from normality of metabolite ratios may have biased our results we tested associations for both untransformed and log-scaled ratios, not detecting significant differences (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Table 3</a>). The reported p-values of step 1 of the discovery stage were not corrected for genomic inflation, as the genomic control inflation factors (lambda) are small, ranging from 1.00 to 1.03, both in KORA and in TwinsUK.</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">Replication in the TwinsUK study</h3><p id="P27" class="p p-first-last">The TwinsUK cohort (KCL, <a href="http://www.twinsuk.ac.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.twinsuk.ac.uk</a>) is an adult twin British registry. These unselected twins were recruited from the general population through national media campaigns in the UK and shown to be comparable to age-matched population singletons in terms of disease-related and lifestyle characteristics <sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918441">22</a></sup>. Ethics approval was obtained from the Guy’s and St. Thomas’ Hospital Ethics Committee. Written informed consent was obtained from every participant to the study. A total of 2277 individuals of European ancestry (1073 singletons and 602 dizygotic twins (DZs)) from the TwinsUK registry were genotyped using the Illumina Hap317K chip (Illumina, San Diego, USA). We applied a strict quality control at both individual and SNP levels. We excluded 51 individuals due to non-European ancestry and 3366 SNPs due to MAF &lt;1%, call rate &lt; 95% if MAF &lt; 5% or call rate &lt;99% if MAF &gt;5%, or P<sub>HWE</sub> &lt;1×10<sup>−4</sup> . After the quality control, 305,811 autosomal SNPs available on 2226 individuals (1046 singletons and 590 DZs) were available and used for imputation. The imputation was carried out using the IMPUTE software <sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332918435">16</a></sup>. NCBI build 36 was used for strand reference. 2.5 million autosomal imputed SNPs data are made available on these 2226 individuals. 422 unrelated individuals from these genotyped were selected for the metabolomics assay. For the TwinsUK study, blood samples were taken after at least 6 hours fasting. The samples were immediately inverted three times, followed by 40 min resting at 4°C to obtain complete coagulation. The samples were then centrifuged for 10 minutes at 3,000RPM. Serum is removed from the centrifuged brown-topped tubes as the top, yellow, clear layer of liquid. Aliquot in 4 × 1.5mls skirted microcentrifuge tubes was then stored in a −45°C freezer until sampling. Metabolite measurements were performed using the same metabolomics platform and following an identical protocol as for the KORA study at the Genome Analysis Centre of the Helmholtz Zentrum München. For the purpose of the replication for the KORA study, the data on the metabolites, the ratios of the concentration, and the genotypes were extracted from the available database, and the association between the concentration of the metabolites or the ratios and the corresponding SNPs were tested using a linear regression model in STATA version 10 (StatCorp LP, College Station, TX, USA).</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Table 1</h4><div class="sup-box half_rhythm" id="d36e787"><a href="/pmc/articles/PMC3773904/bin/NIHMS54332-supplement-Supplementary_Table_1.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.3M, xls)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supplementary Table 2</h4><div class="sup-box half_rhythm" id="d36e791"><a href="/pmc/articles/PMC3773904/bin/NIHMS54332-supplement-Supplementary_Table_2.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(328K, xls)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Supplementary Text and Figures</h4><div class="sup-box half_rhythm" id="d36e795"><a href="/pmc/articles/PMC3773904/bin/NIHMS54332-supplement-Supplementary_Text_and_Figures.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(849K, pdf)</sup></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p1">The KORA research platform (KORA: <em>Kooperative Gesundheitsforschung in der Region Augsburg</em>) and the MONICA Augsburg studies (Monitoring trends and determinants on cardiovascular diseases) were initiated and financed by the <em>Helmholtz Zentrum München</em> - <em>National Research Center for Environmental Health</em>, which is funded by the German Federal Ministry of Education, Science, Research and Technology and by the State of Bavaria. Part of this work was financed by the German National Genome Research Network (NGFNPlus: 01GS0823) and by grants from the “Genomics of Lipid-associated Disorders – GOLD” of the “Austrian Genome Research Programme GEN-AU”. Computing resources have been made available by the Leibniz Supercomputing Centre of the Bavarian Academy of Sciences and Humanities (HLRB project h1231) and the DEISA Extreme Computing Initiative (project PHAGEDA). Part of this research was supported within the Munich Center of Health Sciences (MC Health) as part of LMUinnovativ. The TwinsUK study was funded by the Wellcome Trust; European Community’s Seventh Framework Programme (FP7/2007-2013)/grant agreement HEALTH-F2-2008-201865-GEFOS and (FP7/2007-2013), ENGAGE project grant agreement HEALTH-F4-2007-201413 and the FP-5 GenomEUtwin Project (QLG2-CT-2002-01254). The study also receives support from the Dept of Health via the National Institute for Health Research (NIHR) comprehensive Biomedical Research Centre award to Guy’s &amp; St Thomas’ NHS Foundation Trust in partnership with King’s College London. TDS is an NIHR senior Investigator. The project also received support from a <em>Biotechnology and Biological Sciences Research Council</em> (BBSRC) project grant (G20234) .The authors acknowledge the funding and support of the National Eye Institute via an NIH/CIDR genotyping project (PI: Terri Young). We gratefully acknowledge the contributions of P. Lichtner, G. Eckstein, Guido Fischer, T. Strom and all other members of the Helmholtz Zentrum München genotyping staff in generating the SNP dataset, those of Tamara Halex and Arsin Sabunchi to the metabolomics measurements, as well as the contribution of all members of field staffs who were involved in the planning and conduct of the MONICA/KORA Augsburg studies. The KORA group consists of H.E. Wichmann (speaker), A. Peters, C. Meisinger, T. Illig, R. Holle, J. John and their co-workers who are responsible for the design and conduct of the KORA studies. For TwinsUK we thank the staff from the Genotyping Facilities at the Wellcome Trust Sanger Institute for sample preparation, Quality Control and Genotyping led by Leena Peltonen and Panos Deloukas; Le Centre National de Génotypage, France, led by Mark Lathrop, for genotyping; Duke University, North Carolina, USA, led by David Goldstein, for genotyping; and the Finnish Institute of Molecular Medicine, Finnish Genome Center, University of Helsinki, led by Aarno Palotie. Genotyping was also performed by CIDR as part of an NEI/NIH project grant. Finally, we express our appreciation to all participants of the KORA and the TwinsUK studies for donating their blood and time.</p></div></div><div id="idm140400176947504" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140400176947504title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P28" class="p p-first"><strong>DISCLOSURE STATEMENTS</strong></p><p id="P29" class="p p-last">The authors have no competing interests to disclose.</p><p></p></div></div><div id="idm140400171224256" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140400171224256title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Gieger C, et al.  Genetics meets metabolomics: a genome-wide association study of metabolite profiles in human serum. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2581785/">PMC free article</a>]</span> [<a href="/pubmed/19043545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetics+meets+metabolomics:+a+genome-wide+association+study+of+metabolite+profiles+in+human+serum&amp;author=C+Gieger&amp;volume=4&amp;publication_year=2008&amp;pages=e1000282&amp;pmid=19043545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Altmaier E, et al.  Bioinformatics analysis of targeted metabolomics--uncovering old and new tales of diabetic mice under medication. <span><span class="ref-journal">Endocrinology. </span>2008;<span class="ref-vol">149</span>:3478–89.</span> [<a href="/pubmed/18372322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocrinology&amp;title=Bioinformatics+analysis+of+targeted+metabolomics--uncovering+old+and+new+tales+of+diabetic+mice+under+medication&amp;author=E+Altmaier&amp;volume=149&amp;publication_year=2008&amp;pages=3478-89&amp;pmid=18372322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Kottgen A, et al.  Multiple loci associated with indices of renal function and chronic kidney disease. <span><span class="ref-journal">Nat Genet. </span>2009</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039280/">PMC free article</a>]</span> [<a href="/pubmed/19430482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+loci+associated+with+indices+of+renal+function+and+chronic+kidney+disease&amp;author=A+Kottgen&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Kathiresan S, et al.  Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:189–97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682493/">PMC free article</a>]</span> [<a href="/pubmed/18193044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Six+new+loci+associated+with+blood+low-density+lipoprotein+cholesterol,+high-density+lipoprotein+cholesterol+or+triglycerides+in+humans&amp;author=S+Kathiresan&amp;volume=40&amp;publication_year=2008&amp;pages=189-97&amp;pmid=18193044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Willer CJ, et al.  Newly identified loci that influence lipid concentrations and risk of coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:161–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;author=CJ+Willer&amp;volume=40&amp;publication_year=2008&amp;pages=161-9&amp;pmid=18193043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Aulchenko YS, et al.  Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:47–55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687074/">PMC free article</a>]</span> [<a href="/pubmed/19060911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Loci+influencing+lipid+levels+and+coronary+heart+disease+risk+in+16+European+population+cohorts&amp;author=YS+Aulchenko&amp;volume=41&amp;publication_year=2009&amp;pages=47-55&amp;pmid=19060911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Kathiresan S, et al.  Common variants at 30 loci contribute to polygenic dyslipidemia. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2881676/">PMC free article</a>]</span> [<a href="/pubmed/19060906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+30+loci+contribute+to+polygenic+dyslipidemia&amp;author=S+Kathiresan&amp;volume=41&amp;publication_year=2009&amp;pages=56-65&amp;pmid=19060906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Sabatti C, et al.  Genome-wide association analysis of metabolic traits in a birth cohort from a founder population. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:35–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687077/">PMC free article</a>]</span> [<a href="/pubmed/19060910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analysis+of+metabolic+traits+in+a+birth+cohort+from+a+founder+population&amp;author=C+Sabatti&amp;volume=41&amp;publication_year=2009&amp;pages=35-46&amp;pmid=19060910&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Hindorff LA, Junkins HA, Mehta JP, Manolio TA.  [accessed 14 April 2009];<span><span class="ref-journal">A Catalog of Published Genome-Wide Association Studies. </span></span>  <a href="http://www.genome.gov/26525384" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.genome.gov/26525384</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Vaxillaire M, et al.  The common P446L polymorphism in GCKR inversely modulates fasting glucose and triglyceride levels and reduces type 2 diabetes risk in the DESIR prospective general French population. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:2253–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2494697/">PMC free article</a>]</span> [<a href="/pubmed/18556336" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=The+common+P446L+polymorphism+in+GCKR+inversely+modulates+fasting+glucose+and+triglyceride+levels+and+reduces+type+2+diabetes+risk+in+the+DESIR+prospective+general+French+population&amp;author=M+Vaxillaire&amp;volume=57&amp;publication_year=2008&amp;pages=2253-7&amp;pmid=18556336&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Prokopenko I, et al.  Variants in MTNR1B influence fasting glucose levels. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:77–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682768/">PMC free article</a>]</span> [<a href="/pubmed/19060907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variants+in+MTNR1B+influence+fasting+glucose+levels&amp;author=I+Prokopenko&amp;volume=41&amp;publication_year=2009&amp;pages=77-81&amp;pmid=19060907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Gross E, et al.  Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-related toxicity in cancer patients. <span><span class="ref-journal">PLoS ONE. </span>2008;<span class="ref-vol">3</span>:e4003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2602733/">PMC free article</a>]</span> [<a href="/pubmed/19104657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Strong+association+of+a+common+dihydropyrimidine+dehydrogenase+gene+polymorphism+with+fluoropyrimidine-related+toxicity+in+cancer+patients&amp;author=E+Gross&amp;volume=3&amp;publication_year=2008&amp;pages=e4003&amp;pmid=19104657&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Chen Y, et al.  Thermolabile phenotype of carnitine palmitoyltransferase II variations as a predisposing factor for influenza-associated encephalopathy. <span><span class="ref-journal">FEBS Lett. </span>2005;<span class="ref-vol">579</span>:2040–4.</span> [<a href="/pubmed/15811315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=Thermolabile+phenotype+of+carnitine+palmitoyltransferase+II+variations+as+a+predisposing+factor+for+influenza-associated+encephalopathy&amp;author=Y+Chen&amp;volume=579&amp;publication_year=2005&amp;pages=2040-4&amp;pmid=15811315&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">KIDO H, KINOSHITA M, MIZUGUCHI H, TAKAHASHI N.  <span class="ref-journal">METHOD OF DIAGNOSING THE RISK OF THERMOLABILE PHENOTYPE DISEASES BY USING GENE.</span> OTSUKA PHARMACEUTICAL CO., LTD. &amp; The University of Tokushima; 2007.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=METHOD+OF+DIAGNOSING+THE+RISK+OF+THERMOLABILE+PHENOTYPE+DISEASES+BY+USING+GENE&amp;author=H+KIDO&amp;author=M+KINOSHITA&amp;author=H+MIZUGUCHI&amp;author=N+TAKAHASHI&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Wichmann HE, Gieger C, Illig T. KORA-gen--resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>(Suppl 1):S26–30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen--resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=HE+Wichmann&amp;author=C+Gieger&amp;author=T+Illig&amp;volume=67&amp;issue=Suppl+1&amp;publication_year=2005&amp;pages=S26-30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–13.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-13&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Weinberger KM. Metabolomics in diagnosing metabolic diseases. <span><span class="ref-journal">Ther Umsch. </span>2008;<span class="ref-vol">65</span>:487–91.</span> [<a href="/pubmed/18791962" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ther+Umsch&amp;title=Metabolomics+in+diagnosing+metabolic+diseases&amp;author=KM+Weinberger&amp;volume=65&amp;publication_year=2008&amp;pages=487-91&amp;pmid=18791962&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Weinberger KM, Graber A. Using Comprehensive Metabolomics to Identify Novel Biomarkers. <span><span class="ref-journal">Screening Trends in Drug Discovery. </span>2005;<span class="ref-vol">6</span>:42–45.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Screening+Trends+in+Drug+Discovery&amp;title=Using+Comprehensive+Metabolomics+to+Identify+Novel+Biomarkers&amp;author=KM+Weinberger&amp;author=A+Graber&amp;volume=6&amp;publication_year=2005&amp;pages=42-45&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Wenk MR. The emerging field of lipidomics. <span><span class="ref-journal">Nat Rev Drug Discov. </span>2005;<span class="ref-vol">4</span>:594–610.</span> [<a href="/pubmed/16052242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Drug+Discov&amp;title=The+emerging+field+of+lipidomics&amp;author=MR+Wenk&amp;volume=4&amp;publication_year=2005&amp;pages=594-610&amp;pmid=16052242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Altmaier E, et al.  Bioinformatics analysis of targeted metabolomics - uncovering old and new tales of diabetic mice under medication. <span><span class="ref-journal">Endocrinology. </span>2008;<span class="ref-vol">149</span>:34783489.</span> [<a href="/pubmed/18372322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocrinology&amp;title=Bioinformatics+analysis+of+targeted+metabolomics+-+uncovering+old+and+new+tales+of+diabetic+mice+under+medication&amp;author=E+Altmaier&amp;volume=149&amp;publication_year=2008&amp;pages=34783489&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Wang-Sattler R, et al.  Metabolic profiling reveals distinct variations linked to nicotine consumption in humans--first results from the KORA study. <span><span class="ref-journal">PLoS ONE. </span>2008;<span class="ref-vol">3</span>:e3863.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2588343/">PMC free article</a>]</span> [<a href="/pubmed/19057651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Metabolic+profiling+reveals+distinct+variations+linked+to+nicotine+consumption+in+humans--first+results+from+the+KORA+study&amp;author=R+Wang-Sattler&amp;volume=3&amp;publication_year=2008&amp;pages=e3863&amp;pmid=19057651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Andrew T, et al.  Are twins and singletons comparable? A study of disease-related and lifestyle characteristics in adult women. <span><span class="ref-journal">Twin Res. </span>2001;<span class="ref-vol">4</span>:464–77.</span> [<a href="/pubmed/11780939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res&amp;title=Are+twins+and+singletons+comparable?+A+study+of+disease-related+and+lifestyle+characteristics+in+adult+women&amp;author=T+Andrew&amp;volume=4&amp;publication_year=2001&amp;pages=464-77&amp;pmid=11780939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Brookes KJ, Chen W, Xu X, Taylor E, Asherson P. Association of fatty acid desaturase genes with attention-deficit/hyperactivity disorder. <span><span class="ref-journal">Biol Psychiatry. </span>2006;<span class="ref-vol">60</span>:1053–61.</span> [<a href="/pubmed/16893529" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Association+of+fatty+acid+desaturase+genes+with+attention-deficit/hyperactivity+disorder&amp;author=KJ+Brookes&amp;author=W+Chen&amp;author=X+Xu&amp;author=E+Taylor&amp;author=P+Asherson&amp;volume=60&amp;publication_year=2006&amp;pages=1053-61&amp;pmid=16893529&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Caspi A, et al.  Moderation of breastfeeding effects on the IQ by genetic variation in fatty acid metabolism. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2007;<span class="ref-vol">104</span>:18860–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2141867/">PMC free article</a>]</span> [<a href="/pubmed/17984066" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Moderation+of+breastfeeding+effects+on+the+IQ+by+genetic+variation+in+fatty+acid+metabolism&amp;author=A+Caspi&amp;volume=104&amp;publication_year=2007&amp;pages=18860-5&amp;pmid=17984066&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Tanaka T, et al.  Genome-wide association study of plasma polyunsaturated fatty acids in the InCHIANTI Study. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000338.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2613033/">PMC free article</a>]</span> [<a href="/pubmed/19148276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+of+plasma+polyunsaturated+fatty+acids+in+the+InCHIANTI+Study&amp;author=T+Tanaka&amp;volume=5&amp;publication_year=2009&amp;pages=e1000338&amp;pmid=19148276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Gregersen N, et al.  Identification of four new mutations in the short-chain acyl-CoA dehydrogenase (SCAD) gene in two patients: one of the variant alleles, 511C--&gt;T, is present at an unexpectedly high frequency in the general population, as was the case for 625G--&gt;A, together conferring susceptibility to ethylmalonic aciduria. <span><span class="ref-journal">Hum Mol Genet. </span>1998;<span class="ref-vol">7</span>:619–27.</span> [<a href="/pubmed/9499414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Identification+of+four+new+mutations+in+the+short-chain+acyl-CoA+dehydrogenase+(SCAD)+gene+in+two+patients:+one+of+the+variant+alleles,+511C-->T,+is+present+at+an+unexpectedly+high+frequency+in+the+general+population,+as+was+the+case+for+625G-->A,+together+conferring+susceptibility+to+ethylmalonic+aciduria&amp;author=N+Gregersen&amp;volume=7&amp;publication_year=1998&amp;pages=619-27&amp;pmid=9499414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Nagan N, et al.  The frequency of short-chain acyl-CoA dehydrogenase gene variants in the US population and correlation with the C(4)-acylcarnitine concentration in newborn blood spots. <span><span class="ref-journal">Mol Genet Metab. </span>2003;<span class="ref-vol">78</span>:239–46.</span> [<a href="/pubmed/12706374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Genet+Metab&amp;title=The+frequency+of+short-chain+acyl-CoA+dehydrogenase+gene+variants+in+the+US+population+and+correlation+with+the+C(4)-acylcarnitine+concentration+in+newborn+blood+spots&amp;author=N+Nagan&amp;volume=78&amp;publication_year=2003&amp;pages=239-46&amp;pmid=12706374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Pare G, et al.  Novel Associations of CPS1, MUT, NOX4, and DPEP1 With Plasma Homocysteine in a Healthy Population: A Genome-Wide Evaluation of 13 974 Participants in the Women’s Genome Health Study. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2009;<span class="ref-vol">2</span>:142–150.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745176/">PMC free article</a>]</span> [<a href="/pubmed/20031578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Novel+Associations+of+CPS1,+MUT,+NOX4,+and+DPEP1+With+Plasma+Homocysteine+in+a+Healthy+Population:+A+Genome-Wide+Evaluation+of+13+974+Participants+in+the+Women’s+Genome+Health+Study&amp;author=G+Pare&amp;volume=2&amp;publication_year=2009&amp;pages=142-150&amp;pmid=20031578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Zhou YE, Egeland GM, Meltzer SJ, Kubow S. The association of desaturase 9 and plasma fatty acid composition with insulin resistance-associated factors in female adolescents. <span><span class="ref-journal">Metabolism. </span>2009;<span class="ref-vol">58</span>:158–66.</span> [<a href="/pubmed/19154947" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolism&amp;title=The+association+of+desaturase+9+and+plasma+fatty+acid+composition+with+insulin+resistance-associated+factors+in+female+adolescents&amp;author=YE+Zhou&amp;author=GM+Egeland&amp;author=SJ+Meltzer&amp;author=S+Kubow&amp;volume=58&amp;publication_year=2009&amp;pages=158-66&amp;pmid=19154947&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Cao H, et al.  Identification of a lipokine, a lipid hormone linking adipose tissue to systemic metabolism. <span><span class="ref-journal">Cell. </span>2008;<span class="ref-vol">134</span>:933–44.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2728618/">PMC free article</a>]</span> [<a href="/pubmed/18805087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Identification+of+a+lipokine,+a+lipid+hormone+linking+adipose+tissue+to+systemic+metabolism&amp;author=H+Cao&amp;volume=134&amp;publication_year=2008&amp;pages=933-44&amp;pmid=18805087&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Peltekova VD, et al.  Functional variants of OCTN cation transporter genes are associated with Crohn disease. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:471–5.</span> [<a href="/pubmed/15107849" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Functional+variants+of+OCTN+cation+transporter+genes+are+associated+with+Crohn+disease&amp;author=VD+Peltekova&amp;volume=36&amp;publication_year=2004&amp;pages=471-5&amp;pmid=15107849&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Pind S, et al.  V490M, a common mutation in 3-phosphoglycerate dehydrogenase deficiency, causes enzyme deficiency by decreasing the yield of mature enzyme. <span><span class="ref-journal">J Biol Chem. </span>2002;<span class="ref-vol">277</span>:7136–43.</span> [<a href="/pubmed/11751922" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=V490M,+a+common+mutation+in+3-phosphoglycerate+dehydrogenase+deficiency,+causes+enzyme+deficiency+by+decreasing+the+yield+of+mature+enzyme&amp;author=S+Pind&amp;volume=277&amp;publication_year=2002&amp;pages=7136-43&amp;pmid=11751922&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3773904&amp;issue-id=227484&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140400176789456title">Abstract</a></li><li><a href="#S1title">ONLINE METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S9title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140400176947504title">Footnotes</a></li><li><a href="#idm140400171224256title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3773904/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3773904/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3773904/pdf/emss-54332.pdf">PDF (482K)</a></li> | <li><a href="#" data-citationid="PMC3773904" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3773904%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3773904%2F&amp;text=A%20genomewide%20perspective%20of%20genetic%20variation%20in%20human%20metabolism" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3773904%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20037589" data-aiid="3773904" data-aid="3773904" data-iid="227484" data-domainid="358" data-domain="wtpa" data-accid="PMC3773904" data-md5="129183b203ad93003c630c733dbcca96"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:34:29-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89106CDD4FB1610000000008D305B8&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3773904%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3773904/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>