<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3511984/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Association and Population Genetic Analysis of C-Reactive Protein in African American and Hispanic American Women</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3511984/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Genome-wide Association and Population Genetic Analysis of C-Reactive Protein in African American and Hispanic American Women"><meta name="citation_authors" content="Alex&nbsp;P. Reiner, Sandra Beleza, Nora Franceschini, Paul&nbsp;L. Auer, Jennifer&nbsp;G. Robinson, Charles Kooperberg, Ulrike Peters, Hua Tang"><meta name="citation_date" content="7 September 2012"><meta name="citation_issue" content="3"><meta name="citation_volume" content="91"><meta name="citation_firstpage" content="502"><meta name="citation_doi" content="10.1016/j.ajhg.2012.07.023"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3511984/?report=abstract"><meta name="citation_pmid" content="22939635"><meta name="DC.Title" content="Genome-wide Association and Population Genetic Analysis of C-Reactive Protein in African American and Hispanic American Women"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Alex&nbsp;P. Reiner"><meta name="DC.Contributor" content="Sandra Beleza"><meta name="DC.Contributor" content="Nora Franceschini"><meta name="DC.Contributor" content="Paul&nbsp;L. Auer"><meta name="DC.Contributor" content="Jennifer&nbsp;G. Robinson"><meta name="DC.Contributor" content="Charles Kooperberg"><meta name="DC.Contributor" content="Ulrike Peters"><meta name="DC.Contributor" content="Hua Tang"><meta name="DC.Date" content="2012 Sep 7"><meta name="DC.Identifier" content="10.1016/j.ajhg.2012.07.023"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Association and Population Genetic Analysis of C-Reactive Protein in African American and Hispanic American Women"><meta property="og:type" content="article"><meta property="og:description" content="C-reactive protein (CRP) is a systemic inflammation marker that predicts future cardiovascular risk. CRP levels are higher in African Americans and Hispanic Americans than in European Americans, but the genetic determinants of CRP in these admixed United ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3511984/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3511984/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3511984/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88B5C9DD54D171000000000075004B.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3511984%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books">Books</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/216885/">v.91(3); 2012 Sep 7</a></li><li class="accid">PMC3511984</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2012 Sep 7; 91(3): 502–512. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2012.07.023" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2012.07.023</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3511984</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22939635">22939635</a></div></div></div></div><h1 class="content-title">Genome-wide Association and Population Genetic Analysis of C-Reactive Protein in African American and Hispanic American Women</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Reiner%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635" class="affpopup" co-rid="_co_idm140434089905680" co-class="co-affbox">Alex&nbsp;P. Reiner</a>,<sup>1,</sup><sup>2,</sup><sup>∗</sup> <a href="/pubmed/?term=Beleza%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635" class="affpopup" co-rid="_co_idm140434089903808" co-class="co-affbox">Sandra Beleza</a>,<sup>3</sup> <a href="/pubmed/?term=Franceschini%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635" class="affpopup" co-rid="_co_idm140434089901824" co-class="co-affbox">Nora Franceschini</a>,<sup>4</sup> <a href="/pubmed/?term=Auer%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635" class="affpopup" co-rid="_co_idm140434097885328" co-class="co-affbox">Paul&nbsp;L. Auer</a>,<sup>1</sup> <a href="/pubmed/?term=Robinson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635" class="affpopup" co-rid="_co_idm140434097883344" co-class="co-affbox">Jennifer&nbsp;G. Robinson</a>,<sup>5</sup> <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635" class="affpopup" co-rid="_co_idm140434097881360" co-class="co-affbox">Charles Kooperberg</a>,<sup>1</sup> <a href="/pubmed/?term=Peters%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635" class="affpopup" co-rid="_co_idm140434093965680" co-class="co-affbox">Ulrike Peters</a>,<sup>1</sup> and  <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635" class="affpopup" co-rid="_co_idm140434093963696" co-class="co-affbox">Hua Tang</a><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140434089905680"><h3 class="no_margin">Alex&nbsp;P. Reiner</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p><p><sup>2</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635">Alex&nbsp;P. Reiner</a></div></div><div id="_co_idm140434089903808"><h3 class="no_margin">Sandra Beleza</h3><p><sup>3</sup>Departments of Statistics and Genetics, Stanford University, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Beleza%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635">Sandra Beleza</a></div></div><div id="_co_idm140434089901824"><h3 class="no_margin">Nora Franceschini</h3><p><sup>4</sup>Gillings School of Global Public Health, University of North Carolina, Chapel Hill, NC 27599, USA</p><div>Find articles by <a href="/pubmed/?term=Franceschini%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635">Nora Franceschini</a></div></div><div id="_co_idm140434097885328"><h3 class="no_margin">Paul&nbsp;L. Auer</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p><div>Find articles by <a href="/pubmed/?term=Auer%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635">Paul&nbsp;L. Auer</a></div></div><div id="_co_idm140434097883344"><h3 class="no_margin">Jennifer&nbsp;G. Robinson</h3><p><sup>5</sup>Department of Epidemiology and Medicine, University of Iowa, Iowa City, IA&nbsp;52242, USA</p><div>Find articles by <a href="/pubmed/?term=Robinson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635">Jennifer&nbsp;G. Robinson</a></div></div><div id="_co_idm140434097881360"><h3 class="no_margin">Charles Kooperberg</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p><div>Find articles by <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635">Charles Kooperberg</a></div></div><div id="_co_idm140434093965680"><h3 class="no_margin">Ulrike Peters</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p><div>Find articles by <a href="/pubmed/?term=Peters%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635">Ulrike Peters</a></div></div><div id="_co_idm140434093963696"><h3 class="no_margin">Hua Tang</h3><p><sup>3</sup>Departments of Statistics and Genetics, Stanford University, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22939635">Hua Tang</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140434092027344_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140434092027344_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140434092027344_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140434092027344_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Departments of Statistics and Genetics, Stanford University, Stanford, CA 94305, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Gillings School of Global Public Health, University of North Carolina, Chapel Hill, NC 27599, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Epidemiology and Medicine, University of Iowa, Iowa City, IA&nbsp;52242, USA</div><div><span class="contrib-email" id="idm140434089905680">Alex&nbsp;P. Reiner: <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@renierpa" class="oemail">ude.notgnihsaw.u@renierpa</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@renierpa" class="oemail">ude.notgnihsaw.u@renierpa</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140434092027344_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Jun 10; Revised 2012 Jul 8; Accepted 2012 Jul 26.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140434092027344_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2012 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3511984/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Figures S1–S4 and Table S1</div><div><a href="/pmc/articles/PMC3511984/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (495K)</span></div><div class="small guid">GUID:&nbsp;42506308-C925-4B80-A200-3E391A57AF98</div></div></dd></dl></div><div id="idm140434092733232" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140434092733232title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">C-reactive protein (CRP) is a systemic inflammation marker that predicts future cardiovascular risk. CRP levels are higher in African Americans and Hispanic Americans than in European Americans, but the genetic determinants of CRP in these admixed United States minority populations are largely unknown. We performed genome-wide association studies (GWASs) of 8,280 African American (AA) and 3,548 Hispanic American (HA) postmenopausal women from the Women's Health Initiative SNP Health Association Resource. We discovered and validated a CRP-associated variant of triggering receptors expressed by myeloid cells 2 (<em>TREM2</em>) in chromosomal region 6p21 (p = 10<sup>−10</sup>). The <em>TREM2</em> variant associated with higher CRP is common in Africa but rare in other ancestral populations. In AA women, the <em>CRP</em> region in 1q23 contained a strong admixture association signal (p = 10<sup>−17</sup>), which appears to be related to several independent CRP-associated alleles; the strongest of these is present only in African ancestral populations and is associated with higher CRP. Of the other genomic loci previously associated with CRP through GWASs of European populations, most loci (<em>LEPR</em>, <em>IL1RN</em>, <em>IL6R</em>, <em>GCKR</em>, <em>NLRP3</em>, <em>HNF1A</em>, <em>HNF4A</em>, and <em>APOC1</em>) showed consistent patterns of association with CRP in AA and HA women. In summary, we&nbsp;have identified a common <em>TREM2</em> variant associated with CRP in United States minority populations. The genetic architecture underlying the CRP phenotype in AA women is complex and involves genetic variants shared across populations, as well as variants specific to&nbsp;populations of African descent.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p2" class="p p-first">C-reactive protein (CRP) is a systemic inflammation marker that predicts future cardiovascular risk.<a href="#bib1" rid="bib1 bib2 bib3" class=" bibr popnode">1–3</a> Plasma CRP levels increase with age and are higher in women than in men.<a href="#bib4" rid="bib4 bib5 bib6 bib7 bib8 bib9" class=" bibr popnode">4–9</a> African American (AA) and Hispanic American (HA) individuals have higher concentrations of CRP than do European American (EA) individuals.<a href="#bib4" rid="bib4 bib5 bib6 bib7 bib8 bib9" class=" bibr popnode">4–9</a> Chronic low-grade inflammation might contribute to increased cardiovascular disease risk and other health disparities among these minority United States populations.<a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477947">10</a> CRP levels are also correlated with obesity and other metabolic and atherosclerotic risk factors.<a href="#bib11" rid="bib11 bib12 bib13" class=" bibr popnode">11–13</a> The correlation between CRP and adiposity does not appear to completely explain the higher CRP levels in populations of African descent.<a href="#bib12" rid="bib12 bib13 bib14 bib15" class=" bibr popnode">12–15</a> Therefore, other factors, both genetic and nongenetic, most likely contribute to CRP differences between racial and ethnic groups.</p><p id="__p3">Family and twin studies suggest that additive genetic factors account for 25%–40% of the variance in plasma CRP levels in Europeans<a href="#bib16" rid="bib16 bib17 bib18" class=" bibr popnode">16–18</a> and 45% in AA individuals.<a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_342005128">19</a> Recent large genome-wide association studies (GWASs) conducted on Europeans and Japanese have identified approximately 20 CRP-associated loci that together explain ∼5% of the variance.<a href="#bib20" rid="bib20 bib21 bib22 bib23" class=" bibr popnode">20–23</a> Many of these genes involve pathways related to either innate immunity (these are <em>CRP</em> [MIM <a href="/omim/123260" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">123260</a>], <em>IL6R</em> [MIM <a href="/omim/147880" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147880</a>], <em>IL6</em> [MIM <a href="/omim/147620" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147620</a>], <em>NLRP3</em> [MIM <a href="/omim/606416" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606416</a>], <em>IL1RN</em> [MIM <a href="/omim/147679" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147679</a>], and <em>IRF1</em> [MIM <a href="/omim/147575" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147575</a>]) or lipid and glucose metabolism (these are <em>APOC1</em> [MIM <a href="/omim/107710" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107710</a>], <em>HNF1A</em> [MIM <a href="/omim/142410" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">142410</a>], <em>LEPR</em> [MIM <a href="/omim/601007" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601007</a>], <em>GCKR</em> [MIM <a href="/omim/600842" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600842</a>], <em>HNF4A</em> [MIM <a href="/omim/600281" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600281</a>], <em>RORA</em> [MIM <a href="/omim/600825" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600825</a>], <em>PPP1R3B</em> [MIM <a href="/omim/610541" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">610541</a>]).</p><p id="__p4" class="p p-last">No large CRP GWAS performed to date has included participants of African or Hispanic ancestry. Recently, a GWAS was performed on ∼800 AA individuals, but no new loci were discovered.<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477955">24</a> Using genotype and phenotype data from the Women's Health Initiative SNP Health Association Resource (WHI-SHARe), we therefore undertook a GWAS and local-ancestry analysis of 8,280 AA participants and a GWAS of 3,548 HA participants. Our goals were to identify new and possibly ancestry-specific genetic variants associated with CRP in AA and HA women and to assess the genetic architecture of CRP association in these women at inflammatory and metabolism loci previously identified in CRP GWASs of European and Asian populations.</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Subjects and Methods</h2><p id="__p5" class="p p-first">Our overall approach can be summarized as follows. (1) The GWAS discovery stage consisted of 8,280 WHI-SHARe AA samples and 3,548 HA samples and also included admixture mapping of the AA population. Because of issues related to heterogeneous ethnicity within Latinos,<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477952">25</a> the lack of availability of an adequately-sized Native American reference panel (which impacts estimation accuracy), and the smaller Women's Health Initiative (WHI) HA sample size (compared to the larger AA sample), we did not perform admixture mapping in the HA sample. (2) Validation of the CRP-associated locus, as well as functional validation with expression quantitative trait locus (eQTL) analysis, was performed for independent cohorts including <em>C</em>andidate Gene <em>A</em>ssociation <em>Re</em>source (CARe) AA individuals (n = 6,300) and whites from the Atherosclerosis Risk in Communities (ARIC) study (n = 5,656). (3) Assessment of transferability of other previously published loci associated with CRP was performed with the WHI-SHARe AA and HA samples.</p><div id="sec2.1" class="sec"><h3 id="sec2.1title">Study Subjects</h3><p id="__p6" class="p p-first-last">Recruitment for the WHI began in 1993 and ended in 1998 and was conducted by 40 clinical centers in 24 states and the District of Columbia.<a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477970">26</a> At the time of enrollment, all women were required to be between 50 and 79 years old, postmenopausal, and expected to remain in the area for at least 3 years. Enrollment of racial or ethnic minority groups proportionate to the total minority population of women between 50 and 79 years of age was a high priority of the WHI. Ten of the 40 WHI clinical centers with expertise and access to specific minority groups (American Indian, black, Asian American or Pacific Islander, and Hispanic) were selected to serve as minority recruitment sites. At the end of the recruitment period, 161,808 women had joined the WHI, and about 17% represented racial or ethnic minority groups. Clinical information was collected by self-report and physical examination. Certified staff obtained fasting blood samples at the baseline clinic visit by venipuncture and deposited these samples into tubes containing ethylenediaminetetraacetic acid. There were 12,151 self-identified AA and 5,469 self identified HA WHI subjects who had consented to genetic research and were eligible for WHI-SHARe. Because of budget constraints, we randomly selected a subsample of 12,157 of these women: 8,515 (70.1%) AA and 3,642 (66.6%) HA women. DNA was extracted by the Specimen Processing Laboratory at the Fred Hutchinson Cancer Research Center from specimens that were collected at the time of enrollment. All participants provided written informed consent as approved by local human subjects committees.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">CRP Measurement</h3><p id="__p7" class="p p-first-last">High-sensitivity CRP was measured in serum from a total of 11,962 WHI-SHARe participants on a Roche Modular P Chemistry analyzer at the University of Minnesota with the use of a latex-particle enhanced immunoturbidimetric assay kit (Roche Diagnostics, Indianapolis, IN) The results were reported in mg/l. Median CRP was 3.50 and the interquartile range was 1.58–7.40. The assay coefficient of variation was 2.3%. All CRP values were natural-log transformed for normalizing the distributions of the phenotypic data.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Genotyping and Quality Control</h3><p id="__p8" class="p p-first-last">Genome-wide genotyping was performed at Affymetrix on the Affymetrix 6.0 array with 2&nbsp;μg of DNA at a concentration of 100&nbsp;ng/μl. Approximately 1% (n = 99) of samples could not be genotyped. We excluded samples that had a call rate below 95% (n = 16), that were duplicates of subjects other than monozygotic twins (n = 34), or that had a Y chromosome (n = 1). SNPs on the Y&nbsp;chromosome and Affymetrix quality-control (QC) probes were excluded from analysis (n = 3,280). We also flagged SNPs that had call rates, calculated separately for AA and HA individuals, &lt;&nbsp;95% and concordance rates &lt; 98%, leaving 871,309 unflagged SNPs. After exclusions, there were 12,007 unique subjects (8,421 AA and 3,586 HA), and there was an average call rate of 99.8% over the unflagged SNPs. We analyzed 188 pairs of blind duplicate samples. The overall concordance rate was 99.8% (the ranges were 94.5%–100% over samples, 98.3%–100% over samples with call rates &gt; 98%, and 98.1%–100% over unflagged SNPs).</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">Genotype Imputation</h3><p id="__p9" class="p p-first-last">Imputation was carried out with MaCH.<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477949">27</a> After some more stringent filtering, 829,370 genotyped SNPs were used for imputation. For the imputation in AA samples, we used 240 HapMap 2 (release 22) phased haplotypes from the CEU (Utah residents with ancestry from northern and western Europe from the CEPH collection) and YRI (Yoruba in Ibadan, Nigeria) reference panels and were left with a total of 2,203,609 SNPs. To augment the size of the reference panel and aid imputation accuracy in the remaining individuals in the study,<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477948">27</a> we estimated parameters on a subset of 200 WHI AA subjects and then imputed all WHI AA subjects. For 2,190,779 SNPs, we obtained imputations with a minor allele frequency (MAF) &gt; 1% and an estimated R<sup>2</sup> &gt; 0.3. On a small test sample (2% of the markers on three chromosomes), the average R<sup>2</sup> was 0.88 and the allelic discordance rate was 2.3%. For imputation in HA samples, we used a larger reference panel of HapMap III CEU + MEX (Mexican ancestry in Los Angeles, California) + YRI samples for a total of 1,387,466 SNPs (MAF &gt; 1%), of which 1,368,178 SNPs met the quality threshold of R<sup>2</sup> &gt; 0.3.</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Relatedness</h3><p id="__p10" class="p p-first-last">We used a random subset of 100,000 SNPs from autosomal chromosomes and the method-of-moments approach to a three-parameter identity-by-descent (IBD) model<a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477953">28</a> to compute IBD coefficients between all pairs of 12,008 subjects. On the basis of these coefficients, we identified pairs of parents and offspring (22 pairs and 2 trios), monozygotic twins (five pairs), and siblings (192 pairs and 5 trios). A more thorough confirmatory analysis with a pairwise kinship-coefficient estimator,<a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477969">29</a> as well as the aforementioned method-of-moments approach, was performed separately for AA and HA individuals, and it validated these relationships and identified half siblings (73 pairs). In CRP association analyses, we included one individual from each relative pair, typically the one with the higher call rate.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">Admixture Estimation and Principal-Component Analysis</h3><p id="__p11" class="p p-first-last">Separately for the AA and the HA samples, we computed eigenvectors by using Eigenstrat<a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477950">30</a> at 178,101 markers that were in common between our samples and the reference panels. We also determined individual ancestral proportions by using Frappe<a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477958">31</a> at 656,852 autosomal markers. For both of these calculations, we included 475 publically available samples from ancestral populations (YRI, CEU, East Asians, and Native Americans).<a href="#bib32" rid="bib32 bib33" class=" bibr popnode">32,33</a> On the basis of the Frappe estimates, we identified 56 subjects who were self-identified as AA but who appeared to have less than 10% African ancestry. We also flagged one participant with questionable estimates of both ancestry and relatedness. Because we cannot exclude the possibility of either a sample mishandling or a data-entry error, we flagged these 57 samples for exclusion.</p></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">Local-Ancestry Estimation</h3><p id="__p12" class="p p-first-last">For each AA individual in the sample, locus-specific ancestry (the probability of whether an individual has 0, 1, or 2 alleles of African ancestry at each locus) was estimated with the program SABER+, an extension of the model described in Tang et&nbsp;al.<a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477951">34</a> In brief, SABER+ uses a graphical-model approach to adaptively capture local haplotype structure within each ancestral population and thereby more accurately accounts for background linkage disequilibrium (LD). In simulation studies, this method compares favorably to existing methods in terms of the accuracy and computational efficiency; its error rate is less than 1.5%. In the current analysis, phased haplotype data from the HapMap3 CEU and YRI individuals were augmented as the reference panels. In simulation studies, SABER+ achieves accuracy similar to that of HapMix; based on analysis of simulated and real AA genotypes, the correlation between local-ancestry estimates produced by the two methods is very high.<a href="#bib35" rid="bib35" class=" bibr popnode">35</a></p></div><div id="sec2.8" class="sec"><h3 id="sec2.8title">GWAS Data Analysis</h3><p id="__p13" class="p p-first">GWASs for ln-transformed CRP (excluding the WHI-SHARe study participants with raw CRP values &gt; 100&nbsp;mg/dl) were performed under an additive genetic model with the use of covariate-adjusted linear regression implemented in either PLINK v.1.07<a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477971">28</a> or MACH2QTL v.1.08.<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477956">27</a> Analyses were conducted separately for self-identified AA and HA individuals. For direct or imputed genotypes, allelic dosage at each SNP was used as the independent variable, adjusted for primary covariates of age and geographic region. In sensitivity analyses, we additionally adjusted for body mass index (BMI), hormone therapy, diabetes, and hypertension. To adjust for population stratification and global admixture, we also incorporated the first four principal components (PCs) as covariates in the regression models. Eigenvectors 5–10 showed no evidence of association with CRP levels. To maintain an overall type 1 error rate of 5%, we used a threshold of α = 2.5&nbsp;× 10<sup>−8</sup> to declare genome-wide statistical significance. This threshold has been suggested for African ancestry populations on the basis of estimates of approximately two million independent common-variant tests in African genomes.<a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477966">36</a></p><p id="__p14" class="p p-last">To further assess the potential existence of multiple, independent variants influencing a trait at the same locus (allelic heterogeneity), we repeated regression analyses, conditional on the most strongly associated (index) SNP in that region. To assess in the WHI-SHARe AA and HA individuals trait-associated loci previously reported in Europeans or Japanese, we evaluated the AA and HA results (including consistency of direction of effect) for each index SNP in the regions reported and assessed statistical significance by a simple Bonferroni adjustment based on the total number of SNPs assessed with a two-sided hypothesis test.</p></div><div id="sec2.9" class="sec"><h3 id="sec2.9title">Admixture Mapping</h3><p id="__p15" class="p p-first-last">Using the local-ancestry estimates, we performed an admixture-mapping analysis to detect variants that are present at different frequencies among the European and African ancestral populations. Adjusting for age and genome-wide ancestry proportions by using the first four PCs, we regressed ln-CRP on locus-specific ancestry. We performed the admixture scan with and without additional adjustment for BMI. The critical value for genome-wide significance level of admixture mapping is substantially lower than that for the genotype test because the recent history of admixture gives rise to extensive correlation in local ancestry. On the basis of previous theoretical analysis and simulation results, a nominal p value of 7&nbsp;× 10<sup>−6</sup> yielded a genome-wide type I error of 0.05.<a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477961">34</a></p></div><div id="sec2.10" class="sec"><h3 id="sec2.10title">Validation of New CRP Association Signals in Independent Samples</h3><p id="__p16" class="p p-first-last">Replication of novel association findings in WHI AA individuals was performed with GWAS data from 3,548 HA women from WHI-SHARe, 6,300 AA men and women from the CARe consortium, and 5,656 whites from the ARIC study. Both genotyped and imputed SNP data were available in the CARe AA, WHI-SHARe HA, and ARIC white samples. We used a p value of &lt; 0.05 to declare statistical significance in the replication samples. We conducted meta-analyses combining data from our discovery and replication cohorts by using inverse-variance weighted fixed-effects models to combine beta coefficients and standard errors from study-level regression results for each SNP to derive effect estimates and a combined p value. Meta-analyses were implemented in the software METAL.<a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477974">37</a></p></div><div id="sec2.11" class="sec sec-last"><h3 id="sec2.11title">Analysis of eQTLs</h3><p id="__p17" class="p p-first-last">We assessed the functional effects of newly identified CRP-associated SNPs on gene expression by querying a publically accessible database containing genome-wide analysis of 22,305 genes expressed in peripheral-blood monocytes of 1,467 European individuals.<a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477977">38</a></p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">Assessment of Global and Local Ancestry in Relation to CRP in Admixed Populations</h3><p id="__p18" class="p p-first">WHI participant characteristics are summarized by self-reported ethnicity in <a href="/pmc/articles/PMC3511984/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>. Among AA individuals, mean estimates of genome-wide ancestry proportions were 76% African, 21% European, &lt;2% Asian, and &lt;2% Native American. Among HA participants, the mean estimates were 60% European, 32% Native American, 6% African, and &lt;2% East Asian. European ancestry was negatively correlated with age-adjusted CRP in both AA (p&nbsp;&lt; 10&nbsp;× 10<sup>−5</sup>) and HA (p = 0.001) individuals; the correlation with European ancestry was attenuated upon adjustment for BMI (p = 0.3 in AA samples and p = 0.05 in HA samples). In HA individuals, Native American ancestry was associated with higher age-adjusted CRP (p = 0.0001) and East Asian ancestry was associated with lower age-adjusted CRP (p = 0.007); these associations were largely robust to BMI adjustment (p = 0.001 for Native American ancestry and p = 0.005 for East Asian ancestry).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p56">Characteristics of WHI-SHARe AA and HA Participants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140434094504752" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Study</strong></th><th rowspan="1" colspan="1"><strong>AA Women</strong></th><th rowspan="1" colspan="1"><strong>HA Women</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Sample size</td><td rowspan="1" colspan="1">8,280</td><td rowspan="1" colspan="1">3,548</td></tr><tr><td rowspan="1" colspan="1">Age (SD)</td><td rowspan="1" colspan="1">61.6 years (7.0)</td><td rowspan="1" colspan="1">60.2 years (6.7)</td></tr><tr><td rowspan="1" colspan="1">Female</td><td rowspan="1" colspan="1">100%</td><td rowspan="1" colspan="1">100%</td></tr><tr><td rowspan="1" colspan="1">BMI (SD)</td><td rowspan="1" colspan="1">31.0&nbsp;kg/m2 (6.7)</td><td rowspan="1" colspan="1">28.9&nbsp;kg/m2 (5.6)</td></tr><tr><td rowspan="1" colspan="1">Current smoker</td><td rowspan="1" colspan="1">11.6%</td><td rowspan="1" colspan="1">6.8%</td></tr><tr><td rowspan="1" colspan="1">Diabetes</td><td rowspan="1" colspan="1">13.9%</td><td rowspan="1" colspan="1">8.3%</td></tr><tr><td rowspan="1" colspan="1">Hypertension</td><td rowspan="1" colspan="1">55.1%</td><td rowspan="1" colspan="1">29.3%</td></tr><tr><td rowspan="1" colspan="1">High cholesterol</td><td rowspan="1" colspan="1">15.6%</td><td rowspan="1" colspan="1">13.7%</td></tr><tr><td rowspan="1" colspan="1">Current hormone use</td><td rowspan="1" colspan="1">53.5%</td><td rowspan="1" colspan="1">61.2%</td></tr><tr><td rowspan="1" colspan="1">Median CRP (SD)</td><td rowspan="1" colspan="1">3.68&nbsp;mg/l (7.85)</td><td rowspan="1" colspan="1">3.16&nbsp;mg/l (7.22)</td></tr><tr><td colspan="3" rowspan="1"><hr></td></tr><tr><td colspan="3" rowspan="1"><strong>Admixture Proportions</strong></td></tr><tr><td colspan="3" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">African (SD)</td><td rowspan="1" colspan="1">75.9% (15.3)</td><td rowspan="1" colspan="1">6.2% (11.9)</td></tr><tr><td rowspan="1" colspan="1">European (SD)</td><td rowspan="1" colspan="1">20.7% (15.2)</td><td rowspan="1" colspan="1">60.1% (20.2)</td></tr><tr><td rowspan="1" colspan="1">East Asian (SD)</td><td rowspan="1" colspan="1">2.2% (2.7)</td><td rowspan="1" colspan="1">1.8% (5.3)</td></tr><tr><td rowspan="1" colspan="1">Native American (SD)</td><td rowspan="1" colspan="1">1.2% (1.8)</td><td rowspan="1" colspan="1">31.9% (20.1)</td></tr></tbody></table></div><div id="largeobj_idm140434094504752" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3511984/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140434095615600"><p id="__p57" class="p p-first-last">The following abbreviations are used: AA, African American; HA, Hispanic American; SD, standard deviation; and BMI, body mass index.</p></div></div></div><p id="__p19" class="p p-last">In the genome-wide admixture scan, we observed for CRP a strong admixture signal spanning a 15 Mb region of chromosome 1q (151.5–166.7 Mb; GRCh37/hg19) and found the peak association (p = 1.45&nbsp;× 10<sup>−17</sup>) at <em>CRP</em> (159.7 Mb) (<a href="/pmc/articles/PMC3511984/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>). In this region, local African ancestry is associated with higher CRP. A second, albeit much weaker, ancestry-association peak is present on chromosome 13q at 108.2&nbsp;Mb (rs9514697–rs9514698), which is located within an intron of a large gene designated <em>FAM155A</em> (family with sequence similarity 155, member A). The function of <em>FAM155A</em> is unknown, and there are&nbsp;no other biologically plausible candidate genes in the&nbsp;region. The p value was borderline significant (6.6&nbsp;×&nbsp;10<sup>−6</sup>) and was only present in the BMI-adjusted analysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3511984/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140434096501504" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC3511984/bin/gr1.jpg"></div></a><div id="largeobj_idm140434096501504" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3511984/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC3511984/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3511984/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p50">Admixture Mapping Scan of CRP in WHI AA Women</p><p id="__p51">Admixture mapping −log<sub>10</sub> p values are shown on the <em>y</em> axis, and SNPs are ordered by chromosomal position on the <em>x</em> axis. The dotted line represents a genome-wide p value significance level less than approximately 7&nbsp;× 10<sup>−6</sup> (under the assumption of 7,000 independent tests). There is one strongly genome-wide significant peak centered around <em>CRP</em> on chromosome 1, indicating that local African ancestry in this region is associated with CRP.</p></div></div></div></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">GWAS of CRP in AA and HA Individuals</h3><p id="__p20" class="p p-first-last">The results of the GWAS for CRP in AA and HA individuals under an additive genetic model are shown in <a href="/pmc/articles/PMC3511984/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figures 2</span></a> and <a href="/pmc/articles/PMC3511984/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3.</span></span><span>3</span></a>. Quantile-quantile plots are shown in <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>, available online. The lead or index SNPs associated with the CRP phenotype in each genomic region are summarized in <a href="/pmc/articles/PMC3511984/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC3511984/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3,</span></span><span>3</span></a>, and LD association plots for each region are shown in <a href="#app2" rid="app2" class=" sec">Figures S2 and S3</a>. In AA participants, the strongest GWAS signal includes <em>CRP</em>, corresponding to the 1q23 peak observed in the CRP admixture scan. Four other regions reached genome-wide significance in the GWAS: 2q13 (<em>IL1F10-IL1RN</em>), 6p21 (<em>TREM2</em> [MIM <a href="/omim/605086" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605086</a>]), 12q24 (<em>HNF1A</em>), and 19q13 (<em>TOMM40-APOE</em> [MIM <a href="/omim/608061" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608061</a>]). In HA individuals, three regions reached genome-wide significance: 1p31 (<em>LEPR</em>), 1q23 (<em>CRP</em>), and 12q24 (<em>HNF1A</em>). Repeating the GWAS with additional covariate adjustments for BMI, hormone therapy, diabetes, and hypertension did not appreciably alter the results.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC3511984/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140434093925392" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC3511984/bin/gr2.jpg"></div></a><div id="largeobj_idm140434093925392" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3511984/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC3511984/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC3511984/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p52">Manhattan Plot of CRP in WHI AA Women</p><p id="__p53">Gene regions containing genome-wide significant SNPs (p&nbsp;&lt; 2.5&nbsp;× 10<sup>−8</sup>) are highlighted in green.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC3511984/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140434094507328" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" src="/pmc/articles/PMC3511984/bin/gr3.jpg"></div></a><div id="largeobj_idm140434094507328" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3511984/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a target="object" href="/pmc/articles/PMC3511984/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC3511984/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p54">Manhattan Plot of CRP in WHI HA Women</p><p id="__p55">Gene regions containing genome-wide significant SNPs (p &lt; 2.5&nbsp;× 10<sup>−8</sup>) are highlighted in green.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p58">Genome-wide Significant SNPs for CRP in AA Women</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140434095613104" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Chromosomal Region</strong></th><th rowspan="1" colspan="1"><strong>Number of SNPS with p &lt; 2.5&nbsp;× 10</strong><sup><strong>−8</strong></sup></th><th rowspan="1" colspan="1"><strong>Top SNP in Region</strong></th><th rowspan="1" colspan="1"><strong>Position Hg18</strong></th><th rowspan="1" colspan="1"><strong>Candidate Gene</strong></th><th rowspan="1" colspan="1"><strong>Minor/Major Allele</strong></th><th rowspan="1" colspan="1"><strong>Minor Allele Frequency</strong></th><th rowspan="1" colspan="1"><strong>Effect Size (Standard Error)</strong></th><th rowspan="1" colspan="1"><strong>p Value</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">1q23</td><td rowspan="1" colspan="1">131</td><td rowspan="1" colspan="1">rs16827466</td><td rowspan="1" colspan="1">159,649,700</td><td rowspan="1" colspan="1"><em>CRP</em>, <em>APCS</em>, <em>DARC</em>, <em>FCER1A</em>, <em>DUSP23</em>, <em>OR10J1</em>, <em>OR10J5</em>, <em>OR10J3</em>, <em>OLFML2B</em>, <em>IFI16</em>, <em>FCRL6</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.178</td><td rowspan="1" colspan="1">0.4199 (0.023)</td><td rowspan="1" colspan="1">4.149&nbsp;× 10<sup>−73</sup></td></tr><tr><td rowspan="1" colspan="1">2q13</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs6734238</td><td rowspan="1" colspan="1">113,557,501</td><td rowspan="1" colspan="1"><em>IL1F10</em>, <em>IL1RN</em></td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.446</td><td rowspan="1" colspan="1">−0.108 (0.018)</td><td rowspan="1" colspan="1">9.3&nbsp;× 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>6p21</strong></td><td rowspan="1" colspan="1"><strong>1</strong></td><td rowspan="1" colspan="1"><strong>rs7748513</strong></td><td rowspan="1" colspan="1"><strong>41,235,950</strong></td><td rowspan="1" colspan="1"><strong><em>TREM2</em></strong></td><td rowspan="1" colspan="1"><strong>G/A</strong></td><td rowspan="1" colspan="1"><strong>0.432</strong></td><td rowspan="1" colspan="1"><strong>0.106 (0.018)</strong></td><td rowspan="1" colspan="1"><strong>8.39&nbsp;× 10</strong><sup>−<strong>9</strong></sup></td></tr><tr><td rowspan="1" colspan="1">12q24</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">rs7979473</td><td rowspan="1" colspan="1">119,904,643</td><td rowspan="1" colspan="1"><em>HNF1A</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.443</td><td rowspan="1" colspan="1">−0.119 (0.019)</td><td rowspan="1" colspan="1">1.45&nbsp;× 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">19q13</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs1160985</td><td rowspan="1" colspan="1">50,095,252</td><td rowspan="1" colspan="1"><em>TOMM40</em></td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.366</td><td rowspan="1" colspan="1">−0.1343</td><td rowspan="1" colspan="1">4.19&nbsp;× 10<sup>−13</sup></td></tr></tbody></table></div><div id="largeobj_idm140434095613104" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3511984/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140434167072608"><p id="__p59" class="p p-first-last">For each locus, the lead SNP with the smallest p values among the genotyped or imputed SNPs is indicated. Effect size represents the effect of a minor allele on natural-log-transformed CRP. Positions of the SNPs were derived from dbSNP build 136. Novel CRP associations are shown in bold.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p60">Genome-wide Significant SNPs for CRP in HA Women</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140434167069952" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Chromosomal Region</strong></th><th rowspan="1" colspan="1"><strong>Number of SNPs with p &lt; 5&nbsp;× 10</strong><sup><strong>−8</strong></sup></th><th rowspan="1" colspan="1"><strong>Top SNP in Region</strong></th><th rowspan="1" colspan="1"><strong>Position Hg18</strong></th><th rowspan="1" colspan="1"><strong>Candidate Gene</strong></th><th rowspan="1" colspan="1"><strong>Minor/Major Allele</strong></th><th rowspan="1" colspan="1"><strong>Minor Allele Frequency</strong></th><th rowspan="1" colspan="1"><strong>Effect Size (Standard Error)</strong></th><th rowspan="1" colspan="1"><strong>p Value</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">1p31</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">rs1805096</td><td rowspan="1" colspan="1">65,874,845</td><td rowspan="1" colspan="1"><em>LEPR</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.451</td><td rowspan="1" colspan="1">−0.113 (0.025)</td><td rowspan="1" colspan="1">2.2&nbsp;× 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1">1q23</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">rs7553007</td><td rowspan="1" colspan="1">157,965,173</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.344</td><td rowspan="1" colspan="1">0.129 (0.026)</td><td rowspan="1" colspan="1">1.2&nbsp;× 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1">12q24</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs2259816</td><td rowspan="1" colspan="1">119,919,970</td><td rowspan="1" colspan="1"><em>HNF1A</em>, <em>OASL</em>, <em>C12orf43</em></td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1">0.379</td><td rowspan="1" colspan="1">0.141 (0.025)</td><td rowspan="1" colspan="1">3.0&nbsp;× 10<sup>−10</sup></td></tr></tbody></table></div><div id="largeobj_idm140434167069952" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3511984/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140434093209024"><p id="__p61" class="p p-first-last">For each locus, the lead SNP with the smallest p values among the genotyped or imputed SNPs is indicated. Effect size represents the effect of a minor allele on natural-log-transformed CRP. Positions of the SNPs were derived from dbSNP build 136.</p></div></div></div></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">Assessment of Allelic Heterogeneity at the <em>CRP</em> Locus</h3><p id="__p21" class="p p-first">The magnitude and breadth of the CRP association signal spanning chromosomal region 1q23 in both the admixture scan and GWAS suggest the possibility of allelic heterogeneity at this locus. We therefore performed a series of conditional analyses in which we initially adjusted for the index SNP rs16827466 and then successively adjusted the CRP association signal for the most strongly associated SNP remaining at each step. As shown in <a href="/pmc/articles/PMC3511984/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>, there was evidence of at least three independent CRP-associated alleles. The African-ancestry-specific rs16827466&nbsp;T allele is associated with higher CRP and appears to account for ∼50% of the local-ancestry association signal in this region. rs7553007 and rs2808634 are associated with lower CRP, and the CRP-lowering alleles at each SNP are more common in European populations than in African populations. After a simultaneous adjustment for rs16827466, rs7553007, and rs2808634, the p value for association between local ancestry and CRP is 0.047. Therefore, these three CRP variants appear to account for much, but perhaps not all, of local-ancestry association signal in chromosomal region 1q23.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p62">Association between <em>CRP</em> SNPs or Local Ancestry and CRP Phenotype in WHI AA Individuals with and without Simultaneous Adjustment for the Remaining CRP SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140434093205648" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1"><strong>SNP</strong></th><th rowspan="2" colspan="1"><strong>Position Hg18</strong></th><th rowspan="2" colspan="1"><strong>Minor/Major Allele</strong></th><th rowspan="2" colspan="1"><strong>Minor Allele Frequency</strong></th><th rowspan="2" colspan="1"><strong>Effect Allele Frequency in HapMap YRI/CEU/CHB Samples</strong></th><th colspan="2" rowspan="1"><strong>Without Simultaneous Adjustment (Age and PCs Only)</strong><hr></th><th colspan="2" rowspan="1"><strong>With Simultaneous SNP Adjustment</strong><hr></th></tr><tr><th rowspan="1" colspan="1"><strong>Effect Size (Standard Error)</strong></th><th rowspan="1" colspan="1"><strong>p Value</strong></th><th rowspan="1" colspan="1"><strong>Effect Size (Standard Error)</strong></th><th rowspan="1" colspan="1"><strong>p Value</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs16827466</td><td rowspan="1" colspan="1">157,916,324</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.178</td><td rowspan="1" colspan="1">0.20/0/0</td><td rowspan="1" colspan="1">0.4199 (0.023)</td><td rowspan="1" colspan="1">4.149&nbsp;× 10<sup>−73</sup></td><td rowspan="1" colspan="1">0.324 (0.025)</td><td rowspan="1" colspan="1">2.35&nbsp;× 10<sup>−39</sup></td></tr><tr><td rowspan="1" colspan="1">rs7553007</td><td rowspan="1" colspan="1">157,965,173</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.228</td><td rowspan="1" colspan="1">0.20/0.34/0.556</td><td rowspan="1" colspan="1">−0.272 (0.021)</td><td rowspan="1" colspan="1">1.1&nbsp;× 10<sup>−37</sup></td><td rowspan="1" colspan="1">−0.229 (0.022)</td><td rowspan="1" colspan="1">2.82&nbsp;× 10<sup>−24</sup></td></tr><tr><td rowspan="1" colspan="1">rs2808634</td><td rowspan="1" colspan="1">157,959,197</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.156</td><td rowspan="1" colspan="1">0.137/0.305/0.220</td><td rowspan="1" colspan="1">−0.153 (0.024)</td><td rowspan="1" colspan="1">3.9&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1">−0.143 (0.026)</td><td rowspan="1" colspan="1">2.28&nbsp;× 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">Local ancestry</td><td rowspan="1" colspan="1">157,916,324</td><td rowspan="1" colspan="1">European/African</td><td rowspan="1" colspan="1">0.228</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">−0.174 (0.024)</td><td rowspan="1" colspan="1">1.45&nbsp;× 10<sup>−17</sup></td><td rowspan="1" colspan="1">−0.048 (0.024)</td><td rowspan="1" colspan="1">0.047</td></tr></tbody></table></div><div id="largeobj_idm140434093205648" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3511984/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140434098155760"><p id="__p63" class="p p-first-last">The following abbreviations are used: YRI, Yoruba in Ibadan, Nigeria; CEU, Utah residents with ancestry from northern and western Europe from the CEPH collection; CHB, Han Chinese in Beijing, China; PC, principal component; and N/A, not available.</p></div></div></div><p id="__p22">Interestingly, local African ancestry in chromosomal region 1q23 is also strongly associated with lower white blood cell count (WBC) (another inflammation-sensitive phenotype) in African Americans as a result of a common African-ancestry-specific null <em>DARC</em> (MIM <a href="/omim/613665" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">613665</a>) variant, FY<sup>∗</sup>O (rs2814778). In a recent GWAS from Sardinia,<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477964">20</a>
<em>DARC</em> rs3845624 was reported to be independently associated with CRP. In the initial WHI AA GWAS, rs3845624 (MAF = 0.15; imputation R<sup>2</sup> = 0.94) was nominally associated with CRP (p = 2&nbsp;× 10<sup>−7</sup>). However, in the conditional GWAS (with an adjustment for the three independently associated <em>CRP</em> variants rs16827466, rs7553007, and rs2808634), the p value for this SNP was 0.30. Because of the correlation between CRP and WBC (which accounts for ∼5% of CRP phenotypic variation), we repeated the CRP association analysis in AA individuals by adjusting for WBC. The FY<sup>∗</sup>O allele at <em>DARC</em> showed statistically significant association (when we adjusted for both the three <em>CRP</em> SNPs and WBC; p = 2.4&nbsp;× 10<sup>−30</sup>). However, upon adjustment for local ancestry at CRP, the association disappeared. We emphasize that these results do not exclude the <em>DARC</em> locus as a functional locus but rather are constrained by the population genetic structure. The apparent association between the FY<sup>∗</sup>O allele and CRP level might represent functional involvement of DARC, or FY<sup>∗</sup>O might simply tag additional undiscovered CRP-associated variants in chromosomal region 1q23. In AA individuals, admixture generates strong LD between <em>DARC</em> and <em>CRP</em>; therefore, it is difficult to statistically disentangle the contribution of the two loci.</p><p id="__p23" class="p p-last">To assess allelic heterogeneity at additional CRP-associated loci, we repeated the GWAS in AA individuals by conditioning on all eight independent, genome-wide significant CRP-associated SNPs (three from <em>CRP</em>, one from <em>LEPR</em>, one from <em>IL1RN</em>, one from <em>TREM2</em>, one from <em>HNF1A</em>, and one from <em>APOE</em>). There were no additional association signals.</p></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Discovery and Validation of the CRP-Associated Locus in Chromosomal Region 6p21</h3><p id="__p24" class="p p-first">Of the six genomic regions significantly associated with CRP in either AA or HA individuals or both, only chromosomal region 6p21 has not been previously associated with CRP in any population. The CRP signal in this region is located within a cluster of genes that encode the triggering receptors expressed by myeloid cells (TREM) family of leukocyte receptors, which are involved in the regulation of proinflammatory cytokines and chemokine production by neutrophils, monocytes, macrophages, and dendritic cells. The index SNP rs7748513 is located within the second intron of <em>TREM2</em>. The rs7748513 variant G allele associated with higher CRP is common in AA individuals (MAF = 0.43) but is much less common in other populations (MAF &lt; 0.05 in HapMap European and Asian populations). Additional adjustment for local African ancestry did&nbsp;not alter the association between <em>TREM2</em> rs7748513 and CRP (beta = 0.102 ± 0.020; p = 3.8&nbsp;× 10<sup>−7</sup>). When&nbsp;stratified by local ancestry and adjusted for global ancestry, the association between rs7748513 and CRP was present in the 7,722 AA individuals who carried two (beta = 0.100 ± 0.0232; p &lt; 0.0001) or one (beta = 0.122 ± 0.041; p &lt; 0.0001) African chromosomes at the <em>TREM2</em> locus, but not in the 557 AA individuals who carried two European chromosomes (beta = −0.222 ± 0.190; p = 0.25). However, not surprisingly, the MAF of rs7748513 is considerably lower (0.04) among those women carrying two European chromosomes at this locus; therefore, there is limited power for detecting an association in this stratum.</p><p id="__p25">No other SNPs in the <em>TREM</em> cluster reached genome-wide significance in AA individuals. However, another African-ancestry-specific coding variant of <em>TREM2</em>, rs2234253 (MAF = 0.11), was associated with 0.12 ± 0.023 higher ln-CRP (p = 3&nbsp;× 10<sup>−5</sup>). Within <em>TREM2</em> exon 2, rs2234253 encodes a p.Thr96Lys substitution that is predicted to be deleterious.<a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477979">39</a> The r<sup>2</sup> between rs2234253 and rs7748513 in WHI-SHARe AA participants was 0.41. Upon adjustment for rs7748513, the CRP association with rs2234253 was greatly attenuated but still nominally significant (beta = 0.063 ± 0.030; p = 0.04). Haplotype analysis showed that the minor allele of rs2234253 (the p.Lys96 variant) is only present in combination with the African G variant of rs7748513; thus, these two SNPs define three common <em>TREM2</em> haplotypes that are associated with lower (A/Thr), intermediate (G/Thr), and higher (G/Lys) CRP (<a href="/pmc/articles/PMC3511984/table/tbl5/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl5" rid-ob="ob-tbl5" co-legend-rid=""><span>Table 5</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p64">Association between <em>TREM2</em> Haplotypes and CRP in WHI-SHARe AA Women</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140434094092576" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Haplotype</strong></th><th rowspan="1" colspan="1"><strong>rs7748513</strong></th><th rowspan="1" colspan="1"><strong>rs2234253</strong></th><th rowspan="1" colspan="1"><strong>Frequency</strong></th><th rowspan="1" colspan="1"><strong>Beta [95% CI]</strong></th><th rowspan="1" colspan="1"><strong>p Value</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">H1</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.564</td><td rowspan="1" colspan="1">referent</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">H2</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.319</td><td rowspan="1" colspan="1">0.0642 [0.026, 0.102]</td><td rowspan="1" colspan="1">0.00098</td></tr><tr><td rowspan="1" colspan="1">H3</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.116</td><td rowspan="1" colspan="1">0.1107 [0.056, 0.166]</td><td rowspan="1" colspan="1">7.86&nbsp;× 10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm140434094092576" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3511984/table/tbl5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140434098957696"><p id="__p65" class="p p-first-last">The overall p value for <em>TREM2</em> haplotype association with CRP = 6.97&nbsp;× 10<sup>−9</sup>. The following abbreviation is used: CI, confidence interval.</p></div></div></div><p id="__p26">To validate the newly discovered <em>TREM2</em> CRP association in an independent sample of AA individuals, we performed association analysis for rs7748513 and rs2234253 in 6,272 AA men and women from the CARe GWAS consortium. rs7748513 met the significance threshold for association (0.044 ± 0.021 increased ln-CRP; p = 0.037), whereas rs2234253 did not (0.053 ± 0.031 increased ln-CRP; p&nbsp;=&nbsp;0.09). When stratified by sex, the rs7748513 association was present among the 3,787 AA women (beta = 0.070; p = 0.011), but not among the 2,488 AA men (beta = 0.0013; p = 0.97). The p value for interaction between <em>TREM2</em> rs7748513 genotype and sex in CARe AA individuals was 0.06. Because the association between the <em>TREM2</em> genotype and CRP appears to be confined to women, we meta-analyzed the WHI-SHARe AA discovery results with the CARe AA validation results in women only. The combined discovery and validation p values were 1&nbsp;× 10<sup>−10</sup> for rs7748513 and 3&nbsp;× 10<sup>−6</sup> for rs2234253.</p><p id="__p27">We further assessed the association between <em>TREM2</em> and CRP in the 3,548 WHI-SHARe HA women, in whom the frequency of the rs7748513&nbsp;G allele was 0.08. In HA women, the rs7748513&nbsp;G allele was associated with 0.095 natural-log units higher CRP (p = 0.035), whereas the rs2234253 coding variant showed a nonsignificant trend in the same direction (beta = 0.168; p = 0.12). In 5,656 white ARIC study participants, there was no evidence of association between CRP and rs7748513 (MAF = 0.04; p&nbsp;= 0.96). When stratified by sex, the results for rs7748513 were somewhat stronger, though not statistically significant, in white women (0.0614 higher ln-CRP in 3,046 white women [p = 0.18]) compared to men (0.00177 higher ln-CRP in 2,694 white men [p = 0.97]).</p><p id="__p28">Given that TREM-2 has been associated with autoimmune and inflammatory diseases<a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477973">40</a> and that autoimmune diseases (MIM <a href="/omim/109100" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">109100</a>) are more prevalent in women, we stratified the WHI-SHARe AA results by autoimmune-disease status. In the 2,505 women reporting a history of autoimmune disease, the rs7748513&nbsp;G allele was associated with 0.132 natural-log units higher CRP (p = 3.8&nbsp;× 10<sup>−4</sup>). In the 5,732 women without autoimmune disease, the rs7748513&nbsp;G allele was associated with 0.083 natural-log units higher CRP (p = 2.2&nbsp;× 10<sup>−5</sup>). The p value for interaction was 0.31.</p><p id="__p29" class="p p-last">To assess the effect of the <em>TREM2</em> rs7748513&nbsp;G allele on gene expression, we queried a human monocyte eQTL database derived from 1,467 European subjects.<a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477946">38</a> Monocytes are one of the white blood cell subtypes that express the TREM receptors on their surface. We did not find evidence for a <em>cis</em>-eQTL in <em>TREM2</em>. Nonetheless, a <em>cis</em>-eQTL was observed in this region of chromosome 6 for association with monocyte mRNA levels of <em>TREML4</em> (MIM <a href="/omim/614664" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">614664</a>) (p = 8.43&nbsp;× 10<sup>−6</sup>), which is located 68 kb downstream of rs7748513 and is between <em>TREM2</em> and <em>TREM1</em>. No additional significant <em>cis</em>- or <em>trans</em>-effects were&nbsp;observed for <em>TREM2</em> rs7748513 with the use of a p&nbsp;threshold&nbsp;of &lt;10<sup>−5</sup>.</p></div><div id="sec3.5" class="sec sec-last"><h3 id="sec3.5title">Assessment of Other Previously Discovered WBC-Associated Loci in AA Individuals</h3><p id="__p30" class="p p-first-last">Eighteen CRP loci have been identified previously in European or Japanese GWASs.<a href="#bib20" rid="bib20 bib21 bib22 bib23" class=" bibr popnode">20–23</a>
<a href="#app2" rid="app2" class=" sec">Table S1</a> shows the association results of the originally reported index SNP from these loci in our WHI-SHARe AA and HA populations. In AA individuals, seven loci met the strict Bonferroni-corrected significance threshold (p &lt; 0.0028), whereas 12 of 16 loci tested (75%) had p &lt; 0.05 and the same direction of effect. Notably, in several instances (<em>HNF1A</em>, <em>IL6R</em>, <em>HNF4A</em>, <em>RORA</em>, <em>SALL1</em> [MIM <a href="/omim/602218" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602218</a>], <em>PSMG1</em> [MIM <a href="/omim/605296" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605296</a>], and <em>BCL7B</em> [MIM <a href="/omim/605846" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605846</a>]), the allele associated with higher CRP is more common among Africans than among Europeans. In HA individuals, six loci met the Bonferroni significance criteria for replication, whereas 9 of 13 (69%) were nominally associated with CRP at p &lt; 0.05 and showed consistent direction of effect. Eight (<em>CRP</em>, <em>APOC1</em>, <em>LEPR</em>, <em>IL6R</em>, <em>GCKR</em>, <em>NLRP3</em>, <em>IL1F10</em>, and <em>HNF4A</em>) of 12 loci (67%) showed consistency of CRP association across white, AA, and HA individuals.</p></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p31" class="p p-first">In the largest reported GWAS of CRP in United States minority populations, we have identified and validated in chromosomal region 6p21 an African <em>TREM2</em> variant associated with higher CRP in AA women. We also demonstrate in AA individuals a strong admixture association signal due to several polymorphisms (located within the <em>CRP</em> region) that are independently associated with CRP. As previously reported, the most significant of these is specific to African ancestral populations and is associated with higher CRP. Most loci (<em>CRP</em>, <em>LEPR</em>, <em>IL1RN</em>, <em>IL6R</em>, <em>GCKR</em>, <em>NLRP3</em>, <em>HNF1A</em>, <em>HNF4A</em>, and <em>APOC1</em>) previously associated with CRP through GWASs of European populations showed consistent patterns of association with CRP in AA and HA individuals. Of the CRP loci identified through GWASs of European, Japanese, or AA&nbsp;individuals, the index SNPs at <em>CRP</em>, <em>IL6R</em>, <em>HNF1A</em>, <em>GCKR</em>, <em>TREM2</em>, <em>RORA</em>, and <em>HNF4A</em> showed differences in allele frequency between ancestral populations; the African ancestral allele generally was associated with higher CRP.</p><div id="sec4.1" class="sec"><h3 id="sec4.1title"><em>TREM2</em> Is a Genetic Locus Associated with CRP in AA Women</h3><p id="__p32" class="p p-first-last">TREMs belong to a family of membrane-bound and soluble receptors encoded by a gene cluster centromeric to the HLA region on human chromosome 6.<a href="#bib40" rid="bib40 bib41" class=" bibr popnode">40,41</a> TREM-1 activates neutrophils and monocytes in response to bacterial and fungal pathogens by signaling through the adaptor protein DAP12, triggering secretion of proinflammatory chemokines and cytokines, neutrophil degranulation, and phagocytosis. TREM-2 is present on macrophages, microglia, and monocyte-derived dendritic cells and is thought to negatively regulate macrophage cytokine production and autoimmunity. TREM-2 deficiency in humans (Nasu-Hakola syndrome [MIM <a href="/omim/221770" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">221770</a>]) is characterized by bone cysts and CNS demyelination, which leads to dementia.<a href="#bib42" rid="bib42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477965">42</a> TREM-1 and TREM-2 are also expressed constitutively by hepatic macrophages and endothelial cells, and they might have an important role in the inflammatory response to bacterial infection.<a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477975">43</a> Given the influence of gender and ethnicity on CRP, it is noteworthy that the <em>TREM2</em> variant associated with higher CRP is common only among African ancestral populations and appears to be confined to women only. Why this association would be sex specific is unclear. All WHI-SHARe participants were postmenopausal, and there was no evidence of interaction between exogenous sex-hormone use and the <em>TREM2</em> genotype on CRP (data not shown). Further evaluation of the <em>TREM2</em> genotype in larger samples of AA men and women is needed for confirming the gender specificity of the CRP effect.</p></div><div id="sec4.2" class="sec"><h3 id="sec4.2title">Allelic Heterogeneity at the <em>CRP</em> Locus</h3><p id="__p33" class="p p-first-last">Allelic heterogeneity at the <em>CRP</em> locus has been previously demonstrated in single-cohort studies of EA and AA adults.<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477978">15</a> Of the common <em>CRP</em> SNPs identified by resequencing, several in the <em>CRP</em> promoter region are strongly associated with CRP levels and are correlated with the independently associated <em>CRP</em> index SNPs observed in WHI-SHARe. The CRP-increasing allele of rs16827466 is in LD with a common African <em>CRP</em> promoter haplotype tagged by rs3093058 (CRP-790&nbsp;T allele), associated with&nbsp;increased basal <em>CRP</em> transcriptional activity in human&nbsp;hepatocytes.<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477962">15</a> By contrast, rs7553007 is in LD with a haplotype tagged by rs1205, located in the <em>CRP</em> 3′ flanking region (CRP-3872). The same CRP-lowering haplotype tagged by rs1205 has been associated with lower CRP in AA and HA individuals<a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477972">44</a> and with lower promoter activity.<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477954">15</a> In Europeans, rs1205 is in perfect LD (r<sup>2</sup> = 1.0) with the index SNP rs2794520 at the 1q23 <em>CRP</em> locus previously reported in European GWASs.<a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477963">21</a> The CRP-lowering allele of rs1205 (or rs287994520) is less common in African populations than in Europeans and Hispanics (see <a href="#app2" rid="app2" class=" sec">Table&nbsp;S1</a>). A third CRP-lowering SNP rs1800947 (CRP-2667) is a synonymous coding variant<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477959">15</a> that is present in European, Asian, and Hispanic, but not African, populations.<a href="#bib15" rid="bib15 bib44" class=" bibr popnode">15,44</a> Thus, interpopulation differences in CRP might be attributable to allele-frequency differences at several <em>CRP</em> SNPs, in which the allele associated with lower CRP levels is more frequent in EA or HA individuals than in AA individuals.</p></div><div id="sec4.3" class="sec"><h3 id="sec4.3title">Population Genetic Analysis of the 1q23 Region Containing <em>CRP</em> and <em>DARC</em></h3><p id="__p34" class="p p-first">The admixture mapping evidence in chromosomal region 1q23 suggests that this region harbors CRP-associated alleles, which occur at disparate frequencies between the European and African ancestral populations. Indeed, the CRP-increasing variant rs16827466 occurs almost exclusively in Africans, whereas alleles at rs16827466, rs7553007, and rs2808634 also exhibit frequency difference between the HapMap CEU and YRI populations. It is therefore reasonable to ask whether this pattern of divergence is influenced by evolutionary forces other than random drift.</p><p id="__p35"><em>CRP</em> is located ∼500 kb downstream of <em>DARC</em>, a locus that represents a target of natural selection.<a href="#bib45" rid="bib45 bib46" class=" bibr popnode">45,46</a> In Africa, homozygosity for a null allele (FY<sup>∗</sup>O or rs2814778) is known to confer resistance to vivax malaria; the null allele is essentially fixed in sub-Saharan African populations but is virtually absent in other parts of the world. Could some of the <em>CRP</em> alleles have reached higher frequency in Africans by hitchhiking on the FY<sup>∗</sup>O haplotypes? Contradicting this hypothesis, we found that LD between <em>CRP</em> and <em>DARC</em> is low in both HapMap CEU and YRI populations (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>). In contrast to FY<sup>∗</sup>O, both rs7553007 and rs2808634 are polymorphic in HapMap CEU, YRI, East Asian, and other populations, suggesting that these variations predate the adaptive selection of the FY<sup>∗</sup>O allele ∼10,000 years ago.<a href="#bib47" rid="bib47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477976">47</a> These observations suggest that evolution at the <em>CRP</em> locus is most likely independent of that at the <em>DARC</em> locus. Although methods based on homozygosity or Fst do not present unambiguous signature of selection in <em>CRP</em> in any continental population, the possibility of adaptive evolution cannot be excluded. This is because most existing methods for detecting signature of selection focus on classic sweeps, in which a beneficial mutation is rapidly driven toward fixation.<a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_342005129">48</a> Variation in CRP shows a more complex pattern: multiple variants influence the phenotype. Additionally, the complex population genetic structure within Africa and the admixing history of AA individuals require selection studies for carefully modeling the demographic history.</p><p id="__p36" class="p p-last">Although the strong correlation between CRP and obesity accounts for some of the differences in CRP across populations,<a href="#bib12" rid="bib12 bib15" class=" bibr popnode">12,15</a> genetic factors also appear to contribute to the interethnic phenotypic differences. Intriguingly, almost all CRP-increasing variants occur at a higher frequency in African populations than they do in European populations (see <a href="#app2" rid="app2" class=" sec">Table S1</a>).This is consistent with a model in which standing variation is subjected to population-specific directional selection: CRP-increasing variants become adaptive in Africans while CRP-decreasing variants spread in Europeans.</p></div><div id="sec4.4" class="sec sec-last"><h3 id="sec4.4title">CRP Genetic Loci and the Role of Inflammation in Cardiovascular and Metabolic Disease</h3><p id="__p37" class="p p-first">Of the approximately 20 loci associated with CRP, many of these genes involve pathways related to either innate immunity (e.g., <em>CRP</em>, <em>IL6R</em>, <em>IL6</em>, <em>NLRP3</em>, <em>IL1RN</em>, <em>IRF1</em>, and <em>TREM2</em>) or lipid and glucose metabolism (e.g., <em>APOC1</em>, <em>HNF1A</em>, <em>LEPR</em>, <em>GCKR</em>, <em>HNF4A</em>, <em>RORA</em>, and <em>PTPN2</em>), highlighting the connection between inflammation and metabolic dysregulation. Interethnic differences in CRP and the underlying genetic architecture of CRP have potential implications for cardiovascular-risk stratification, prediction, and treatment, such as statin eligibility. AA individuals have higher CVD risk than do non-Hispanic whites despite higher high-density lipoprotein (HDL) cholesterol and lower triglyceride levels.<a href="#bib49" rid="bib49 bib50 bib51" class=" bibr popnode">49–51</a> Chronic low-grade inflammation might contribute to increased cardiovascular-disease risk and other health disparities among these minority United States populations.<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477967">11</a> The metabolic syndrome paradoxically has a lower prevalence in AA individuals than in EA individuals, but there is a greater difference in CRP among AA individuals with and without the metabolic syndrome than there is among EA individuals with and without metabolic syndrome.<a href="#bib52" rid="bib52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477968">52</a></p><p id="__p38" class="p p-last">CRP biosynthesis is largely under the control of IL-6,&nbsp;which together with IL-1β activates transcription factors that induce <em>CRP</em> transcription. The IL-6-receptor p.Asp358Ala polymorphism encoded by <em>IL6R</em> rs8192284 is known to dampen the inflammatory effect of IL-6 and&nbsp;lower CRP and has been associated with a lower risk of coronary heart disease (CHD).<a href="#bib53" rid="bib53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287477980">53</a> These results suggest not only a causal role of IL-6-receptor signaling in CHD pathogenesis but also that IL-6-receptor blockade with agents such as tocilizumab might provide a novel therapeutic approach to CHD prevention. Notably, the cardioprotective IL-6-receptor p.Asp358Ala variant is considerably less common among AA individuals than it is among whites and HA individuals (<a href="#app2" rid="app2" class=" sec">Table S1</a>). Therefore, IL-6-receptor blockade might represent a particularly important therapeutic target for CHD prevention in AA individuals.</p></div></div><div id="idm140434093018592" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140434093018592title">Acknowledgments</h2><div class="sec"><p id="__p49">The Women's Health Initiative (WHI) program is funded by the National Heart, Lung, and Blood Institute, National Institutes of Health, and the United States Department of Health and Human Services through contracts HHSN268201100046C, HHSN268201100001C, HHSN268201100002C, HHSN268201100003C, HHSN268201100004C, and HHSN271201100004C. The authors thank the WHI investigators and staff for their dedication and the study participants for making the program possible. A listing of WHI&nbsp;investigators can be found at <a href="http://www.whiscience.org/publications/WHI_investigators_shortlist_2010-2015.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.whiscience.org/publications/WHI_investigators_shortlist_2010-2015.pdf</a>.</p></div></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Figures S1–S4 and Table S1:</strong><div class="sup-box half_rhythm" id="idm140434092852096"><a href="/pmc/articles/PMC3511984/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(495K, pdf)</sup></div></div><p></p></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p40" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u0010"><p id="__p41">Frappe, <a href="http://med.stanford.edu/tanglab/software/frappe.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://med.stanford.edu/tanglab/software/frappe.html</a></p></li><li id="u0015"><p id="__p42">International HapMap Project, <a href="http://hapmap.ncbi.nlm.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hapmap.ncbi.nlm.nih.gov/</a></p></li><li id="u0020"><p id="__p43">MaCH, <a href="http://www.sph.umich.edu/csg/yli/mach/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/yli/mach/</a></p></li><li id="u0025"><p id="__p44">Monocyte eQTL database, <a href="http://genecanvas.ecgene.net/uploads/ForReview/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genecanvas.ecgene.net/uploads/ForReview/</a></p></li><li id="u0030"><p id="__p45">NHLBI CARe Consortium, <a href="http://www.broadinstitute.org/gen_analysis/care/index.php/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/gen_analysis/care/index.php/</a></p></li><li id="u0035"><p id="__p46">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.omim.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org</a></p></li><li id="u0040"><p id="__p47">PLINK, <a href="http://pngu.mgh.harvard.edu/%7Epurcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/plink/</a></p></li><li id="u0045"><p id="__p48">USCS Genome Browser, <a href="http://genome.ucsc.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.ucsc.edu/</a></p></li></ul><p></p></div><div id="idm140434089976224" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140434089976224title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Pearson T.A., Mensah G.A., Alexander R.W., Anderson J.L., Cannon R.O., 3rd, Criqui M., Fadl Y.Y., Fortmann S.P., Hong Y., Myers G.L., Centers for Disease Control and Prevention. American Heart Association  Markers of inflammation and cardiovascular disease: Application to clinical and public health practice: A statement for healthcare professionals from the Centers for Disease Control and Prevention and the American Heart Association. <span><span class="ref-journal">Circulation. </span>2003;<span class="ref-vol">107</span>:499–511.</span> [<a href="/pubmed/12551878" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Markers+of+inflammation+and+cardiovascular+disease:+Application+to+clinical+and+public+health+practice:+A+statement+for+healthcare+professionals+from+the+Centers+for+Disease+Control+and+Prevention+and+the+American+Heart+Association&amp;author=T.A.+Pearson&amp;author=G.A.+Mensah&amp;author=R.W.+Alexander&amp;author=J.L.+Anderson&amp;author=R.O.+Cannon&amp;volume=107&amp;publication_year=2003&amp;pages=499-511&amp;pmid=12551878&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Ridker P.M., Buring J.E., Rifai N., Cook N.R. Development and validation of improved algorithms for the assessment of global cardiovascular risk in women: the Reynolds Risk Score. <span><span class="ref-journal">JAMA. </span>2007;<span class="ref-vol">297</span>:611–619.</span> [<a href="/pubmed/17299196" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Development+and+validation+of+improved+algorithms+for+the+assessment+of+global+cardiovascular+risk+in+women:+the+Reynolds+Risk+Score&amp;author=P.M.+Ridker&amp;author=J.E.+Buring&amp;author=N.+Rifai&amp;author=N.R.+Cook&amp;volume=297&amp;publication_year=2007&amp;pages=611-619&amp;pmid=17299196&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">Ridker P.M., Paynter N.P., Rifai N., Gaziano J.M., Cook N.R. C-reactive protein and parental history improve global cardiovascular risk prediction: The Reynolds Risk Score for men. <span><span class="ref-journal">Circulation. </span>2008;<span class="ref-vol">118</span>:2243–2251.</span> 4p, 2251. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2752381/">PMC free article</a>]</span> [<a href="/pubmed/18997194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=C-reactive+protein+and+parental+history+improve+global+cardiovascular+risk+prediction:+The+Reynolds+Risk+Score+for+men&amp;author=P.M.+Ridker&amp;author=N.P.+Paynter&amp;author=N.+Rifai&amp;author=J.M.+Gaziano&amp;author=N.R.+Cook&amp;volume=118&amp;publication_year=2008&amp;pages=2243-2251&amp;pmid=18997194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Ford E.S., Giles W.H., Myers G.L., Rifai N., Ridker P.M., Mannino D.M. C-reactive protein concentration distribution among US children and young adults: Findings from the National Health and Nutrition Examination Survey, 1999-2000. <span><span class="ref-journal">Clin. Chem. </span>2003;<span class="ref-vol">49</span>:1353–1357.</span> [<a href="/pubmed/12881452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=C-reactive+protein+concentration+distribution+among+US+children+and+young+adults:+Findings+from+the+National+Health+and+Nutrition+Examination+Survey,+1999-2000&amp;author=E.S.+Ford&amp;author=W.H.+Giles&amp;author=G.L.+Myers&amp;author=N.+Rifai&amp;author=P.M.+Ridker&amp;volume=49&amp;publication_year=2003&amp;pages=1353-1357&amp;pmid=12881452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Albert M.A., Ridker P.M. C-reactive protein as a risk predictor: do race/ethnicity and gender make a difference? <span><span class="ref-journal">Circulation. </span>2006;<span class="ref-vol">114</span>:e67–e74.</span> [<a href="/pubmed/16880331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=C-reactive+protein+as+a+risk+predictor:+do+race/ethnicity+and+gender+make+a+difference?&amp;author=M.A.+Albert&amp;author=P.M.+Ridker&amp;volume=114&amp;publication_year=2006&amp;pages=e67-e74&amp;pmid=16880331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">Lakoski S.G., Cushman M., Criqui M., Rundek T., Blumenthal R.S., D'Agostino R.B., Jr., Herrington D.M. Gender and C-reactive protein: Data from the Multiethnic Study of Atherosclerosis (MESA) cohort. <span><span class="ref-journal">Am. Heart J. </span>2006;<span class="ref-vol">152</span>:593–598.</span> [<a href="/pubmed/16923436" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+Heart+J.&amp;title=Gender+and+C-reactive+protein:+Data+from+the+Multiethnic+Study+of+Atherosclerosis+(MESA)+cohort&amp;author=S.G.+Lakoski&amp;author=M.+Cushman&amp;author=M.+Criqui&amp;author=T.+Rundek&amp;author=R.S.+Blumenthal&amp;volume=152&amp;publication_year=2006&amp;pages=593-598&amp;pmid=16923436&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">Khera A., McGuire D.K., Murphy S.A., Stanek H.G., Das S.R., Vongpatanasin W., Wians F.H., Jr., Grundy S.M., de Lemos J.A. Race and gender differences in C-reactive protein levels. <span><span class="ref-journal">J. Am. Coll. Cardiol. </span>2005;<span class="ref-vol">46</span>:464–469.</span> [<a href="/pubmed/16053959" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol.&amp;title=Race+and+gender+differences+in+C-reactive+protein+levels&amp;author=A.+Khera&amp;author=D.K.+McGuire&amp;author=S.A.+Murphy&amp;author=H.G.+Stanek&amp;author=S.R.+Das&amp;volume=46&amp;publication_year=2005&amp;pages=464-469&amp;pmid=16053959&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">Cushman M., McClure L.A., Howard V.J., Jenny N.S., Lakoski S.G., Howard G. Implications of increased C-reactive protein for cardiovascular risk stratification in black and white men and women in the US. <span><span class="ref-journal">Clin. Chem. </span>2009;<span class="ref-vol">55</span>:1627–1636.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2810186/">PMC free article</a>]</span> [<a href="/pubmed/19643839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Implications+of+increased+C-reactive+protein+for+cardiovascular+risk+stratification+in+black+and+white+men+and+women+in+the+US&amp;author=M.+Cushman&amp;author=L.A.+McClure&amp;author=V.J.+Howard&amp;author=N.S.+Jenny&amp;author=S.G.+Lakoski&amp;volume=55&amp;publication_year=2009&amp;pages=1627-1636&amp;pmid=19643839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Albert M.A., Glynn R.J., Buring J., Ridker P.M. C-reactive protein levels among women of various ethnic groups living in the United States (from the Women's Health Study) <span><span class="ref-journal">Am. J. Cardiol. </span>2004;<span class="ref-vol">93</span>:1238–1242.</span> [<a href="/pubmed/15135696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol.&amp;title=C-reactive+protein+levels+among+women+of+various+ethnic+groups+living+in+the+United+States+(from+the+Women's+Health+Study)&amp;author=M.A.+Albert&amp;author=R.J.+Glynn&amp;author=J.+Buring&amp;author=P.M.+Ridker&amp;volume=93&amp;publication_year=2004&amp;pages=1238-1242&amp;pmid=15135696&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">Mensah G.A., Mokdad A.H., Ford E.S., Greenlund K.J., Croft J.B. State of disparities in cardiovascular health in the United States. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">111</span>:1233–1241.</span> [<a href="/pubmed/15769763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=State+of+disparities+in+cardiovascular+health+in+the+United+States&amp;author=G.A.+Mensah&amp;author=A.H.+Mokdad&amp;author=E.S.+Ford&amp;author=K.J.+Greenlund&amp;author=J.B.+Croft&amp;volume=111&amp;publication_year=2005&amp;pages=1233-1241&amp;pmid=15769763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">Miller M., Zhan M., Havas S. High attributable risk of elevated C-reactive protein level to conventional coronary heart disease risk factors: The Third National Health and Nutrition Examination Survey. <span><span class="ref-journal">Arch. Intern. Med. </span>2005;<span class="ref-vol">165</span>:2063–2068.</span> [<a href="/pubmed/16216995" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Intern.+Med.&amp;title=High+attributable+risk+of+elevated+C-reactive+protein+level+to+conventional+coronary+heart+disease+risk+factors:+The+Third+National+Health+and+Nutrition+Examination+Survey&amp;author=M.+Miller&amp;author=M.+Zhan&amp;author=S.+Havas&amp;volume=165&amp;publication_year=2005&amp;pages=2063-2068&amp;pmid=16216995&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Wee C.C., Mukamal K.J., Huang A., Davis R.B., McCarthy E.P., Mittleman M.A. Obesity and C-reactive protein levels among white, black, and hispanic US adults. <span><span class="ref-journal">Obesity (Silver Spring) </span>2008;<span class="ref-vol">16</span>:875–880.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2848449/">PMC free article</a>]</span> [<a href="/pubmed/18379563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obesity+(Silver+Spring)&amp;title=Obesity+and+C-reactive+protein+levels+among+white,+black,+and+hispanic+US+adults&amp;author=C.C.+Wee&amp;author=K.J.+Mukamal&amp;author=A.+Huang&amp;author=R.B.+Davis&amp;author=E.P.+McCarthy&amp;volume=16&amp;publication_year=2008&amp;pages=875-880&amp;pmid=18379563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Shah T., Newcombe P., Smeeth L., Addo J., Casas J.P., Whittaker J., Miller M.A., Tinworth L., Jeffery S., Strazzullo P. Ancestry as a determinant of mean population C-reactive protein values: Implications for cardiovascular risk prediction. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2010;<span class="ref-vol">3</span>:436–444.</span> [<a href="/pubmed/20876875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Ancestry+as+a+determinant+of+mean+population+C-reactive+protein+values:+Implications+for+cardiovascular+risk+prediction&amp;author=T.+Shah&amp;author=P.+Newcombe&amp;author=L.+Smeeth&amp;author=J.+Addo&amp;author=J.P.+Casas&amp;volume=3&amp;publication_year=2010&amp;pages=436-444&amp;pmid=20876875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Rodriguez F., Peralta C.A., Green A.R., López L. Comparison of C-reactive protein levels in less versus more acculturated Hispanic adults in the United States (from the National Health and Nutrition Examination Survey 1999-2008) <span><span class="ref-journal">Am. J. Cardiol. </span>2012;<span class="ref-vol">109</span>:665–669.</span> [<a href="/pubmed/22169128" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol.&amp;title=Comparison+of+C-reactive+protein+levels+in+less+versus+more+acculturated+Hispanic+adults+in+the+United+States+(from+the+National+Health+and+Nutrition+Examination+Survey+1999-2008)&amp;author=F.+Rodriguez&amp;author=C.A.+Peralta&amp;author=A.R.+Green&amp;author=L.+López&amp;volume=109&amp;publication_year=2012&amp;pages=665-669&amp;pmid=22169128&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Carlson C.S., Aldred S.F., Lee P.K., Tracy R.P., Schwartz S.M., Rieder M., Liu K., Williams O.D., Iribarren C., Lewis E.C. Polymorphisms within the C-reactive protein (CRP) promoter region are associated with plasma CRP levels. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2005;<span class="ref-vol">77</span>:64–77.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226195/">PMC free article</a>]</span> [<a href="/pubmed/15897982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Polymorphisms+within+the+C-reactive+protein+(CRP)+promoter+region+are+associated+with+plasma+CRP+levels&amp;author=C.S.+Carlson&amp;author=S.F.+Aldred&amp;author=P.K.+Lee&amp;author=R.P.+Tracy&amp;author=S.M.+Schwartz&amp;volume=77&amp;publication_year=2005&amp;pages=64-77&amp;pmid=15897982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Retterstol L., Eikvar L., Berg K. A twin study of C-Reactive Protein compared to other risk factors for coronary heart disease. <span><span class="ref-journal">Atherosclerosis. </span>2003;<span class="ref-vol">169</span>:279–282.</span> [<a href="/pubmed/12921979" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=A+twin+study+of+C-Reactive+Protein+compared+to+other+risk+factors+for+coronary+heart+disease&amp;author=L.+Retterstol&amp;author=L.+Eikvar&amp;author=K.+Berg&amp;volume=169&amp;publication_year=2003&amp;pages=279-282&amp;pmid=12921979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Pankow J.S., Folsom A.R., Cushman M., Borecki I.B., Hopkins P.N., Eckfeldt J.H., Tracy R.P. Familial and genetic determinants of systemic markers of inflammation: The NHLBI family heart study. <span><span class="ref-journal">Atherosclerosis. </span>2001;<span class="ref-vol">154</span>:681–689.</span> [<a href="/pubmed/11257270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Familial+and+genetic+determinants+of+systemic+markers+of+inflammation:+The+NHLBI+family+heart+study&amp;author=J.S.+Pankow&amp;author=A.R.+Folsom&amp;author=M.+Cushman&amp;author=I.B.+Borecki&amp;author=P.N.+Hopkins&amp;volume=154&amp;publication_year=2001&amp;pages=681-689&amp;pmid=11257270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Dupuis J., Larson M.G., Vasan R.S., Massaro J.M., Wilson P.W., Lipinska I., Corey D., Vita J.A., Keaney J.F., Jr., Benjamin E.J. Genome scan of systemic biomarkers of vascular inflammation in the Framingham Heart Study: evidence for susceptibility loci on 1q. <span><span class="ref-journal">Atherosclerosis. </span>2005;<span class="ref-vol">182</span>:307–314.</span> [<a href="/pubmed/16159603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Genome+scan+of+systemic+biomarkers+of+vascular+inflammation+in+the+Framingham+Heart+Study:+evidence+for+susceptibility+loci+on+1q&amp;author=J.+Dupuis&amp;author=M.G.+Larson&amp;author=R.S.+Vasan&amp;author=J.M.+Massaro&amp;author=P.W.+Wilson&amp;volume=182&amp;publication_year=2005&amp;pages=307-314&amp;pmid=16159603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Fox E.R., Benjamin E.J., Sarpong D.F., Rotimi C.N., Wilson J.G., Steffes M.W., Chen G., Adeyemo A., Taylor J.K., Samdarshi T.E., Taylor H.A., Jr. Epidemiology, heritability, and genetic linkage of C-reactive protein in African Americans (from the Jackson Heart Study) <span><span class="ref-journal">Am. J. Cardiol. </span>2008;<span class="ref-vol">102</span>:835–841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3733442/">PMC free article</a>]</span> [<a href="/pubmed/18805107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol.&amp;title=Epidemiology,+heritability,+and+genetic+linkage+of+C-reactive+protein+in+African+Americans+(from+the+Jackson+Heart+Study)&amp;author=E.R.+Fox&amp;author=E.J.+Benjamin&amp;author=D.F.+Sarpong&amp;author=C.N.+Rotimi&amp;author=J.G.+Wilson&amp;volume=102&amp;publication_year=2008&amp;pages=835-841&amp;pmid=18805107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Naitza S., Porcu E., Steri M., Taub D.D., Mulas A., Xiao X., Strait J., Dei M., Lai S., Busonero F. A genome-wide association scan on the levels of markers of inflammation in Sardinians reveals associations that underpin its complex regulation. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002480.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3266885/">PMC free article</a>]</span> [<a href="/pubmed/22291609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+genome-wide+association+scan+on+the+levels+of+markers+of+inflammation+in+Sardinians+reveals+associations+that+underpin+its+complex+regulation&amp;author=S.+Naitza&amp;author=E.+Porcu&amp;author=M.+Steri&amp;author=D.D.+Taub&amp;author=A.+Mulas&amp;volume=8&amp;publication_year=2012&amp;pages=e1002480&amp;pmid=22291609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">Dehghan A., Dupuis J., Barbalic M., Bis J.C., Eiriksdottir G., Lu C., Pellikka N., Wallaschofski H., Kettunen J., Henneman P. Meta-analysis of genome-wide association studies in &gt;80 000 subjects identifies multiple loci for C-reactive protein levels. <span><span class="ref-journal">Circulation. </span>2011;<span class="ref-vol">123</span>:731–738.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3147232/">PMC free article</a>]</span> [<a href="/pubmed/21300955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Meta-analysis+of+genome-wide+association+studies+in+>80+000+subjects+identifies+multiple+loci+for+C-reactive+protein+levels&amp;author=A.+Dehghan&amp;author=J.+Dupuis&amp;author=M.+Barbalic&amp;author=J.C.+Bis&amp;author=G.+Eiriksdottir&amp;volume=123&amp;publication_year=2011&amp;pages=731-738&amp;pmid=21300955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">Okada Y., Takahashi A., Ohmiya H., Kumasaka N., Kamatani Y., Hosono N., Tsunoda T., Matsuda K., Tanaka T., Kubo M. Genome-wide association study for C-reactive protein levels identified pleiotropic associations in the IL6 locus. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:1224–1231.</span> [<a href="/pubmed/21196492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+study+for+C-reactive+protein+levels+identified+pleiotropic+associations+in+the+IL6+locus&amp;author=Y.+Okada&amp;author=A.+Takahashi&amp;author=H.+Ohmiya&amp;author=N.+Kumasaka&amp;author=Y.+Kamatani&amp;volume=20&amp;publication_year=2011&amp;pages=1224-1231&amp;pmid=21196492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">Ridker P.M., Pare G., Parker A., Zee R.Y., Danik J.S., Buring J.E., Kwiatkowski D., Cook N.R., Miletich J.P., Chasman D.I. Loci related to metabolic-syndrome pathways including LEPR,HNF1A, IL6R, and GCKR associate with plasma C-reactive protein: The Women's Genome Health Study. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:1185–1192.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427311/">PMC free article</a>]</span> [<a href="/pubmed/18439548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Loci+related+to+metabolic-syndrome+pathways+including+LEPR,HNF1A,+IL6R,+and+GCKR+associate+with+plasma+C-reactive+protein:+The+Women's+Genome+Health+Study&amp;author=P.M.+Ridker&amp;author=G.+Pare&amp;author=A.+Parker&amp;author=R.Y.+Zee&amp;author=J.S.+Danik&amp;volume=82&amp;publication_year=2008&amp;pages=1185-1192&amp;pmid=18439548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Doumatey A.P., Chen G., Tekola Ayele F., Zhou J., Erdos M., Shriner D., Huang H., Adeleye J., Balogun W., Fasanmade O. C-reactive protein (CRP) promoter polymorphisms influence circulating CRP levels in a genome-wide association study of African Americans. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:3063–3072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3373247/">PMC free article</a>]</span> [<a href="/pubmed/22492993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=C-reactive+protein+(CRP)+promoter+polymorphisms+influence+circulating+CRP+levels+in+a+genome-wide+association+study+of+African+Americans&amp;author=A.P.+Doumatey&amp;author=G.+Chen&amp;author=F.+Tekola+Ayele&amp;author=J.+Zhou&amp;author=M.+Erdos&amp;volume=21&amp;publication_year=2012&amp;pages=3063-3072&amp;pmid=22492993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Johnson N.A., Coram M.A., Shriver M.D., Romieu I., Barsh G.S., London S.J., Tang H. Ancestral components of admixed genomes in a Mexican cohort. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002410.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3240599/">PMC free article</a>]</span> [<a href="/pubmed/22194699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Ancestral+components+of+admixed+genomes+in+a+Mexican+cohort&amp;author=N.A.+Johnson&amp;author=M.A.+Coram&amp;author=M.D.+Shriver&amp;author=I.+Romieu&amp;author=G.S.+Barsh&amp;volume=7&amp;publication_year=2011&amp;pages=e1002410&amp;pmid=22194699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation">Hays J., Hunt J.R., Hubbell F.A., Anderson G.L., Limacher M., Allen C., Rossouw J.E. The Women's Health Initiative recruitment methods and results. <span><span class="ref-journal">Ann. Epidemiol. </span>2003;<span class="ref-vol">13</span>(9, Suppl):S18–S77.</span> [<a href="/pubmed/14575939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Epidemiol.&amp;title=The+Women's+Health+Initiative+recruitment+methods+and+results&amp;author=J.+Hays&amp;author=J.R.+Hunt&amp;author=F.A.+Hubbell&amp;author=G.L.+Anderson&amp;author=M.+Limacher&amp;volume=13&amp;issue=9,+Suppl&amp;publication_year=2003&amp;pages=S18-S77&amp;pmid=14575939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation">Li Y., Willer C.J., Ding J., Scheet P., Abecasis G.R. MaCH: Using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=MaCH:+Using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y.+Li&amp;author=C.J.+Willer&amp;author=J.+Ding&amp;author=P.+Scheet&amp;author=G.R.+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation">Thornton T., McPeek M.S. ROADTRIPS: Case-control association testing with partially or completely unknown population and pedigree structure. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2010;<span class="ref-vol">86</span>:172–184.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820184/">PMC free article</a>]</span> [<a href="/pubmed/20137780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=ROADTRIPS:+Case-control+association+testing+with+partially+or+completely+unknown+population+and+pedigree+structure&amp;author=T.+Thornton&amp;author=M.S.+McPeek&amp;volume=86&amp;publication_year=2010&amp;pages=172-184&amp;pmid=20137780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation">Tang H., Peng J., Wang P., Risch N.J. Estimation of individual admixture: Analytical and study design considerations. <span><span class="ref-journal">Genet. Epidemiol. </span>2005;<span class="ref-vol">28</span>:289–301.</span> [<a href="/pubmed/15712363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Estimation+of+individual+admixture:+Analytical+and+study+design+considerations&amp;author=H.+Tang&amp;author=J.+Peng&amp;author=P.+Wang&amp;author=N.J.+Risch&amp;volume=28&amp;publication_year=2005&amp;pages=289-301&amp;pmid=15712363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation">Cann H.M., de Toma C., Cazes L., Legrand M.F., Morel V., Piouffre L., Bodmer J., Bodmer W.F., Bonne-Tamir B., Cambon-Thomsen A. A human genome diversity cell line panel. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">296</span>:261–262.</span> [<a href="/pubmed/11954565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+human+genome+diversity+cell+line+panel&amp;author=H.M.+Cann&amp;author=C.+de+Toma&amp;author=L.+Cazes&amp;author=M.F.+Legrand&amp;author=V.+Morel&amp;volume=296&amp;publication_year=2002&amp;pages=261-262&amp;pmid=11954565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation">International HapMap Consortium  The International HapMap Project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+International+HapMap+Project&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation">Tang H., Coram M., Wang P., Zhu X., Risch N. Reconstructing genetic ancestry blocks in admixed individuals. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">79</span>:1–12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1474129/">PMC free article</a>]</span> [<a href="/pubmed/16773560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Reconstructing+genetic+ancestry+blocks+in+admixed+individuals&amp;author=H.+Tang&amp;author=M.+Coram&amp;author=P.+Wang&amp;author=X.+Zhu&amp;author=N.+Risch&amp;volume=79&amp;publication_year=2006&amp;pages=1-12&amp;pmid=16773560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="mixed-citation">Johnson, N.A. (2011). Efficient models and algorithms for problems in genomics. PhD thesis, Stanford University, Stanford, CA.</span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation">Pe'er I., Yelensky R., Altshuler D., Daly M.J. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I.+Pe'er&amp;author=R.+Yelensky&amp;author=D.+Altshuler&amp;author=M.J.+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation">Willer C.J., Li Y., Abecasis G.R. METAL: Fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+Fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation">Zeller T., Wild P., Szymczak S., Rotival M., Schillert A., Castagne R., Maouche S., Germain M., Lackner K., Rossmann H. Genetics and beyond—the transcriptome of human monocytes and disease susceptibility. <span><span class="ref-journal">PLoS ONE. </span>2010;<span class="ref-vol">5</span>:e10693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872668/">PMC free article</a>]</span> [<a href="/pubmed/20502693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Genetics+and+beyond—the+transcriptome+of+human+monocytes+and+disease+susceptibility&amp;author=T.+Zeller&amp;author=P.+Wild&amp;author=S.+Szymczak&amp;author=M.+Rotival&amp;author=A.+Schillert&amp;volume=5&amp;publication_year=2010&amp;pages=e10693&amp;pmid=20502693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation">González-Pérez A., López-Bigas N. Improving the assessment of the outcome of nonsynonymous SNVs with a consensus deleteriousness score, Condel. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:440–449.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3071923/">PMC free article</a>]</span> [<a href="/pubmed/21457909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Improving+the+assessment+of+the+outcome+of+nonsynonymous+SNVs+with+a+consensus+deleteriousness+score,+Condel&amp;author=A.+González-Pérez&amp;author=N.+López-Bigas&amp;volume=88&amp;publication_year=2011&amp;pages=440-449&amp;pmid=21457909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation">Ford J.W., McVicar D.W. TREM and TREM-like receptors in inflammation and disease. <span><span class="ref-journal">Curr. Opin. Immunol. </span>2009;<span class="ref-vol">21</span>:38–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2723941/">PMC free article</a>]</span> [<a href="/pubmed/19230638" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Immunol.&amp;title=TREM+and+TREM-like+receptors+in+inflammation+and+disease&amp;author=J.W.+Ford&amp;author=D.W.+McVicar&amp;volume=21&amp;publication_year=2009&amp;pages=38-46&amp;pmid=19230638&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation">Allcock R.J., Barrow A.D., Forbes S., Beck S., Trowsdale J. The human TREM gene cluster at 6p21.1 encodes both activating and inhibitory single IgV domain receptors and includes NKp44. <span><span class="ref-journal">Eur. J. Immunol. </span>2003;<span class="ref-vol">33</span>:567–577.</span> [<a href="/pubmed/12645956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Immunol.&amp;title=The+human+TREM+gene+cluster+at+6p21.1+encodes+both+activating+and+inhibitory+single+IgV+domain+receptors+and+includes+NKp44&amp;author=R.J.+Allcock&amp;author=A.D.+Barrow&amp;author=S.+Forbes&amp;author=S.+Beck&amp;author=J.+Trowsdale&amp;volume=33&amp;publication_year=2003&amp;pages=567-577&amp;pmid=12645956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation">Paloneva J., Manninen T., Christman G., Hovanes K., Mandelin J., Adolfsson R., Bianchin M., Bird T., Miranda R., Salmaggi A. Mutations in two genes encoding different subunits of a receptor signaling complex result in an identical disease phenotype. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">71</span>:656–662.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379202/">PMC free article</a>]</span> [<a href="/pubmed/12080485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Mutations+in+two+genes+encoding+different+subunits+of+a+receptor+signaling+complex+result+in+an+identical+disease+phenotype&amp;author=J.+Paloneva&amp;author=T.+Manninen&amp;author=G.+Christman&amp;author=K.+Hovanes&amp;author=J.+Mandelin&amp;volume=71&amp;publication_year=2002&amp;pages=656-662&amp;pmid=12080485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation">Sharif O., Knapp S. From expression to signaling: Roles of TREM-1 and TREM-2 in innate immunity and bacterial infection. <span><span class="ref-journal">Immunobiology. </span>2008;<span class="ref-vol">213</span>:701–713.</span> [<a href="/pubmed/18926286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunobiology&amp;title=From+expression+to+signaling:+Roles+of+TREM-1+and+TREM-2+in+innate+immunity+and+bacterial+infection&amp;author=O.+Sharif&amp;author=S.+Knapp&amp;volume=213&amp;publication_year=2008&amp;pages=701-713&amp;pmid=18926286&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation">Crawford D.C., Sanders C.L., Qin X., Smith J.D., Shephard C., Wong M., Witrak L., Rieder M.J., Nickerson D.A. Genetic variation is associated with C-reactive protein levels in the Third National Health and Nutrition Examination Survey. <span><span class="ref-journal">Circulation. </span>2006;<span class="ref-vol">114</span>:2458–2465.</span> [<a href="/pubmed/17101857" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Genetic+variation+is+associated+with+C-reactive+protein+levels+in+the+Third+National+Health+and+Nutrition+Examination+Survey&amp;author=D.C.+Crawford&amp;author=C.L.+Sanders&amp;author=X.+Qin&amp;author=J.D.+Smith&amp;author=C.+Shephard&amp;volume=114&amp;publication_year=2006&amp;pages=2458-2465&amp;pmid=17101857&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation">Livingstone F.B. The Duffy blood groups, vivax malaria, and malaria selection in human populations: A review. <span><span class="ref-journal">Hum. Biol. </span>1984;<span class="ref-vol">56</span>:413–425.</span> [<a href="/pubmed/6386656" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Biol.&amp;title=The+Duffy+blood+groups,+vivax+malaria,+and+malaria+selection+in+human+populations:+A+review&amp;author=F.B.+Livingstone&amp;volume=56&amp;publication_year=1984&amp;pages=413-425&amp;pmid=6386656&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation">Hamblin M.T., Thompson E.E., Di Rienzo A. Complex signatures of natural selection at the Duffy blood group locus. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">70</span>:369–383.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC419988/">PMC free article</a>]</span> [<a href="/pubmed/11753822" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Complex+signatures+of+natural+selection+at+the+Duffy+blood+group+locus&amp;author=M.T.+Hamblin&amp;author=E.E.+Thompson&amp;author=A.+Di+Rienzo&amp;volume=70&amp;publication_year=2002&amp;pages=369-383&amp;pmid=11753822&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation">Hamblin M.T., Di Rienzo A. Detection of the signature of natural selection in humans: evidence from the Duffy blood group locus. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2000;<span class="ref-vol">66</span>:1669–1679.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1378024/">PMC free article</a>]</span> [<a href="/pubmed/10762551" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Detection+of+the+signature+of+natural+selection+in+humans:+evidence+from+the+Duffy+blood+group+locus&amp;author=M.T.+Hamblin&amp;author=A.+Di+Rienzo&amp;volume=66&amp;publication_year=2000&amp;pages=1669-1679&amp;pmid=10762551&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation">Hernandez R.D., Kelley J.L., Elyashiv E., Melton S.C., Auton A., McVean G., Sella G., Przeworski M., 1000 Genomes Project  Classic selective sweeps were rare in recent human evolution. <span><span class="ref-journal">Science. </span>2011;<span class="ref-vol">331</span>:920–924.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3669691/">PMC free article</a>]</span> [<a href="/pubmed/21330547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Classic+selective+sweeps+were+rare+in+recent+human+evolution&amp;author=R.D.+Hernandez&amp;author=J.L.+Kelley&amp;author=E.+Elyashiv&amp;author=S.C.+Melton&amp;author=A.+Auton&amp;volume=331&amp;publication_year=2011&amp;pages=920-924&amp;pmid=21330547&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation">Roger V.L., Go A.S., Lloyd-Jones D.M., Benjamin E.J., Berry J.D., Borden W.B., Bravata D.M., Dai S., Ford E.S., Fox C.S., American Heart Association Statistics Committee and Stroke Statistics Subcommittee  Heart disease and stroke statistics—2012 update: A report from the American Heart Association. <span><span class="ref-journal">Circulation. </span>2012;<span class="ref-vol">125</span>:e2–e220.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4440543/">PMC free article</a>]</span> [<a href="/pubmed/22179539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Heart+disease+and+stroke+statistics—2012+update:+A+report+from+the+American+Heart+Association&amp;author=V.L.+Roger&amp;author=A.S.+Go&amp;author=D.M.+Lloyd-Jones&amp;author=E.J.+Benjamin&amp;author=J.D.+Berry&amp;volume=125&amp;publication_year=2012&amp;pages=e2-e220&amp;pmid=22179539&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation">Chambless L.E., Folsom A.R., Sharrett A.R., Sorlie P., Couper D., Szklo M., Nieto F.J. Coronary heart disease risk prediction in the Atherosclerosis Risk in Communities (ARIC) study. <span><span class="ref-journal">J. Clin. Epidemiol. </span>2003;<span class="ref-vol">56</span>:880–890.</span> [<a href="/pubmed/14505774" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Epidemiol.&amp;title=Coronary+heart+disease+risk+prediction+in+the+Atherosclerosis+Risk+in+Communities+(ARIC)+study&amp;author=L.E.+Chambless&amp;author=A.R.+Folsom&amp;author=A.R.+Sharrett&amp;author=P.+Sorlie&amp;author=D.+Couper&amp;volume=56&amp;publication_year=2003&amp;pages=880-890&amp;pmid=14505774&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation">Chambless L.E., Heiss G., Shahar E., Earp M.J., Toole J. Prediction of ischemic stroke risk in the Atherosclerosis Risk in Communities Study. <span><span class="ref-journal">Am. J. Epidemiol. </span>2004;<span class="ref-vol">160</span>:259–269.</span> [<a href="/pubmed/15257999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Prediction+of+ischemic+stroke+risk+in+the+Atherosclerosis+Risk+in+Communities+Study&amp;author=L.E.+Chambless&amp;author=G.+Heiss&amp;author=E.+Shahar&amp;author=M.J.+Earp&amp;author=J.+Toole&amp;volume=160&amp;publication_year=2004&amp;pages=259-269&amp;pmid=15257999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation">DeBoer M.D., Gurka M.J., Sumner A.E. Diagnosis of the metabolic syndrome is associated with disproportionately high levels of high-sensitivity C-reactive protein in non-Hispanic black adolescents: An analysis of NHANES 1999-2008. <span><span class="ref-journal">Diabetes Care. </span>2011;<span class="ref-vol">34</span>:734–740.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3041218/">PMC free article</a>]</span> [<a href="/pubmed/21285387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes+Care&amp;title=Diagnosis+of+the+metabolic+syndrome+is+associated+with+disproportionately+high+levels+of+high-sensitivity+C-reactive+protein+in+non-Hispanic+black+adolescents:+An+analysis+of+NHANES+1999-2008&amp;author=M.D.+DeBoer&amp;author=M.J.+Gurka&amp;author=A.E.+Sumner&amp;volume=34&amp;publication_year=2011&amp;pages=734-740&amp;pmid=21285387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation">Hingorani A.D., Casas J.P., Interleukin-6 Receptor Mendelian Randomisation Analysis (IL6R MR) Consortium  The interleukin-6 receptor as a target for prevention of coronary heart disease: A mendelian randomisation analysis. <span><span class="ref-journal">Lancet. </span>2012;<span class="ref-vol">379</span>:1214–1224.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3316968/">PMC free article</a>]</span> [<a href="/pubmed/22421340" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=The+interleukin-6+receptor+as+a+target+for+prevention+of+coronary+heart+disease:+A+mendelian+randomisation+analysis&amp;author=A.D.+Hingorani&amp;author=J.P.+Casas&amp;volume=379&amp;publication_year=2012&amp;pages=1214-1224&amp;pmid=22421340&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3511984&amp;issue-id=216885&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140434092733232title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Subjects and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#idm140434093018592title">Acknowledgments</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#idm140434089976224title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3511984/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3511984/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3511984/pdf/main.pdf">PDF (742K)</a></li> | <li><a href="#" data-citationid="PMC3511984" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3511984%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3511984%2F&amp;text=Genome-wide%20Association%20and%20Population%20Genetic%20Analysis%20of%20C-Reactive%20Protein%20in%20African%20American%20and%20Hispanic%20American%20Women" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3511984%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22939635" data-aiid="3511984" data-aid="3511984" data-iid="216885" data-domainid="203" data-domain="ajhg" data-accid="PMC3511984" data-md5="113305e08082f52444b5cdbd29419d81"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:28:56-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88B5C9DD54D171000000000075004B&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3511984%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3511984/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>