<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3675231/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Characterization of Shared and Distinct Genetic Components that Influence Blood Lipid Levels in Ethnically Diverse Human Populations</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3675231/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Genome-wide Characterization of Shared and Distinct Genetic Components that Influence Blood Lipid Levels in Ethnically Diverse Human Populations"><meta name="citation_authors" content="Marc&nbsp;A. Coram, Qing Duan, Thomas&nbsp;J. Hoffmann, Timothy Thornton, Joshua&nbsp;W. Knowles, Nicholas&nbsp;A. Johnson, Heather&nbsp;M. Ochs-Balcom, Timothy&nbsp;A. Donlon, Lisa&nbsp;W. Martin, Charles&nbsp;B. Eaton, Jennifer&nbsp;G. Robinson, Neil&nbsp;J. Risch, Xiaofeng Zhu, Charles Kooperberg, Yun Li, Alex&nbsp;P. Reiner, Hua Tang"><meta name="citation_date" content="6 June 2013"><meta name="citation_issue" content="6"><meta name="citation_volume" content="92"><meta name="citation_firstpage" content="904"><meta name="citation_doi" content="10.1016/j.ajhg.2013.04.025"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3675231/?report=abstract"><meta name="citation_pmid" content="23726366"><meta name="DC.Title" content="Genome-wide Characterization of Shared and Distinct Genetic Components that Influence Blood Lipid Levels in Ethnically Diverse Human Populations"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Marc&nbsp;A. Coram"><meta name="DC.Contributor" content="Qing Duan"><meta name="DC.Contributor" content="Thomas&nbsp;J. Hoffmann"><meta name="DC.Contributor" content="Timothy Thornton"><meta name="DC.Contributor" content="Joshua&nbsp;W. Knowles"><meta name="DC.Contributor" content="Nicholas&nbsp;A. Johnson"><meta name="DC.Contributor" content="Heather&nbsp;M. Ochs-Balcom"><meta name="DC.Contributor" content="Timothy&nbsp;A. Donlon"><meta name="DC.Contributor" content="Lisa&nbsp;W. Martin"><meta name="DC.Contributor" content="Charles&nbsp;B. Eaton"><meta name="DC.Contributor" content="Jennifer&nbsp;G. Robinson"><meta name="DC.Contributor" content="Neil&nbsp;J. Risch"><meta name="DC.Contributor" content="Xiaofeng Zhu"><meta name="DC.Contributor" content="Charles Kooperberg"><meta name="DC.Contributor" content="Yun Li"><meta name="DC.Contributor" content="Alex&nbsp;P. Reiner"><meta name="DC.Contributor" content="Hua Tang"><meta name="DC.Date" content="2013 Jun 6"><meta name="DC.Identifier" content="10.1016/j.ajhg.2013.04.025"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Characterization of Shared and Distinct Genetic Components that Influence Blood Lipid Levels in Ethnically Diverse Human Populations"><meta property="og:type" content="article"><meta property="og:description" content="Blood lipid concentrations are heritable risk factors associated with atherosclerosis and cardiovascular diseases. Lipid traits exhibit considerable variation among populations of distinct ancestral origin as well as between individuals within a population. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3675231/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3675231/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3675231/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88D885DD5598010000000006DA0449.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3675231%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/223736/">v.92(6); 2013 Jun 6</a></li><li class="accid">PMC3675231</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2013 Jun 6; 92(6): 904–916. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2013.04.025" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2013.04.025</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3675231</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23726366">23726366</a></div></div></div></div><h1 class="content-title">Genome-wide Characterization of Shared and Distinct Genetic Components that Influence Blood Lipid Levels in Ethnically Diverse Human Populations</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Coram%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190200480880" co-class="co-affbox">Marc&nbsp;A. Coram</a>,<sup>1</sup> <a href="/pubmed/?term=Duan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190201288704" co-class="co-affbox">Qing Duan</a>,<sup>2</sup> <a href="/pubmed/?term=Hoffmann%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164116320" co-class="co-affbox">Thomas&nbsp;J. Hoffmann</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Thornton%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164114272" co-class="co-affbox">Timothy Thornton</a>,<sup>5</sup> <a href="/pubmed/?term=Knowles%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164112736" co-class="co-affbox">Joshua&nbsp;W. Knowles</a>,<sup>6</sup> <a href="/pubmed/?term=Johnson%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164111088" co-class="co-affbox">Nicholas&nbsp;A. Johnson</a>,<sup>7,</sup><sup>21</sup> <a href="/pubmed/?term=Ochs-Balcom%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164108368" co-class="co-affbox">Heather&nbsp;M. Ochs-Balcom</a>,<sup>8</sup> <a href="/pubmed/?term=Donlon%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164106384" co-class="co-affbox">Timothy&nbsp;A. Donlon</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Martin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164103664" co-class="co-affbox">Lisa&nbsp;W. Martin</a>,<sup>11</sup> <a href="/pubmed/?term=Eaton%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164101680" co-class="co-affbox">Charles&nbsp;B. Eaton</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Robinson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164098960" co-class="co-affbox">Jennifer&nbsp;G. Robinson</a>,<sup>14</sup> <a href="/pubmed/?term=Risch%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164979232" co-class="co-affbox">Neil&nbsp;J. Risch</a>,<sup>3,</sup><sup>4,</sup><sup>15</sup> <a href="/pubmed/?term=Zhu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164975776" co-class="co-affbox">Xiaofeng Zhu</a>,<sup>16</sup> <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164973792" co-class="co-affbox">Charles Kooperberg</a>,<sup>17</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164971808" co-class="co-affbox">Yun Li</a>,<sup>2,</sup><sup>18</sup> <a href="/pubmed/?term=Reiner%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164969088" co-class="co-affbox">Alex&nbsp;P. Reiner</a>,<sup>19,</sup><sup>∗</sup> and  <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366" class="affpopup" co-rid="_co_idm140190164966112" co-class="co-affbox">Hua Tang</a><sup>20,</sup><sup>∗∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140190200480880"><h3 class="no_margin">Marc&nbsp;A. Coram</h3><p><sup>1</sup>Department of Health Research and Policy, Stanford University School of Medicine, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Coram%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Marc&nbsp;A. Coram</a></div></div><div id="_co_idm140190201288704"><h3 class="no_margin">Qing Duan</h3><p><sup>2</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA</p><div>Find articles by <a href="/pubmed/?term=Duan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Qing Duan</a></div></div><div id="_co_idm140190164116320"><h3 class="no_margin">Thomas&nbsp;J. Hoffmann</h3><p><sup>3</sup>Institute for Human Genetics, University of California, San Francisco, San Francisco, CA 94143, USA</p><p><sup>4</sup>Department of Epidemiology and Biostatistics, University of California, San Francisco, San Francisco, CA 94107, USA</p><div>Find articles by <a href="/pubmed/?term=Hoffmann%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Thomas&nbsp;J. Hoffmann</a></div></div><div id="_co_idm140190164114272"><h3 class="no_margin">Timothy Thornton</h3><p><sup>5</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Thornton%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Timothy Thornton</a></div></div><div id="_co_idm140190164112736"><h3 class="no_margin">Joshua&nbsp;W. Knowles</h3><p><sup>6</sup>Division of Cardiovascular Medicine and Stanford Cardiovascular Institute, Stanford University School of Medicine, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Knowles%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Joshua&nbsp;W. Knowles</a></div></div><div id="_co_idm140190164111088"><h3 class="no_margin">Nicholas&nbsp;A. Johnson</h3><p><sup>7</sup>Department of Statistics, Stanford University, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Johnson%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Nicholas&nbsp;A. Johnson</a></div></div><div id="_co_idm140190164108368"><h3 class="no_margin">Heather&nbsp;M. Ochs-Balcom</h3><p><sup>8</sup>Department of Social and Preventive Medicine, University at Buffalo, Buffalo, NY 14214, USA</p><div>Find articles by <a href="/pubmed/?term=Ochs-Balcom%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Heather&nbsp;M. Ochs-Balcom</a></div></div><div id="_co_idm140190164106384"><h3 class="no_margin">Timothy&nbsp;A. Donlon</h3><p><sup>9</sup>Kuakini Medical Center, Honolulu, HI 96817, USA</p><p><sup>10</sup>Departments of Cell &amp; Molecular Biology and Pathology, John A. Burns School of Medicine, University of Hawaii, Honolulu, HI 96813, USA</p><div>Find articles by <a href="/pubmed/?term=Donlon%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Timothy&nbsp;A. Donlon</a></div></div><div id="_co_idm140190164103664"><h3 class="no_margin">Lisa&nbsp;W. Martin</h3><p><sup>11</sup>Division of Cardiology, George Washington University School of Medicine and Health Sciences, Washington, DC 20037, USA</p><div>Find articles by <a href="/pubmed/?term=Martin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Lisa&nbsp;W. Martin</a></div></div><div id="_co_idm140190164101680"><h3 class="no_margin">Charles&nbsp;B. Eaton</h3><p><sup>12</sup>Department of Family Medicine, Alpert Medical School of Brown University, Providence, RI 02860, USA</p><p><sup>13</sup>Department of Epidemiology, Program in Public Health, Brown University, Providence, RI 02860, USA</p><div>Find articles by <a href="/pubmed/?term=Eaton%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Charles&nbsp;B. Eaton</a></div></div><div id="_co_idm140190164098960"><h3 class="no_margin">Jennifer&nbsp;G. Robinson</h3><p><sup>14</sup>Department of Epidemiology and Medicine, University of Iowa, Iowa City, IA 52242, USA</p><div>Find articles by <a href="/pubmed/?term=Robinson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Jennifer&nbsp;G. Robinson</a></div></div><div id="_co_idm140190164979232"><h3 class="no_margin">Neil&nbsp;J. Risch</h3><p><sup>3</sup>Institute for Human Genetics, University of California, San Francisco, San Francisco, CA 94143, USA</p><p><sup>4</sup>Department of Epidemiology and Biostatistics, University of California, San Francisco, San Francisco, CA 94107, USA</p><p><sup>15</sup>Division of Research, Kaiser Permanente, Oakland, CA 94612, USA</p><div>Find articles by <a href="/pubmed/?term=Risch%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Neil&nbsp;J. Risch</a></div></div><div id="_co_idm140190164975776"><h3 class="no_margin">Xiaofeng Zhu</h3><p><sup>16</sup>Department of Epidemiology and Biostatistics, Case Western Reserve University School of Medicine, Cleveland, OH 44106, USA</p><div>Find articles by <a href="/pubmed/?term=Zhu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Xiaofeng Zhu</a></div></div><div id="_co_idm140190164973792"><h3 class="no_margin">Charles Kooperberg</h3><p><sup>17</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p><div>Find articles by <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Charles Kooperberg</a></div></div><div id="_co_idm140190164971808"><h3 class="no_margin">Yun Li</h3><p><sup>2</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA</p><p><sup>18</sup>Department of Biostatistics, University of North Carolina, Chapel Hill, NC 27599, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Yun Li</a></div></div><div id="_co_idm140190164969088"><h3 class="no_margin">Alex&nbsp;P. Reiner</h3><p><sup>19</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Alex&nbsp;P. Reiner</a></div></div><div id="_co_idm140190164966112"><h3 class="no_margin">Hua Tang</h3><p><sup>20</sup>Department of Genetics, Stanford University School of Medicine, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23726366">Hua Tang</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140190201088160_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140190201088160_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140190201088160_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140190201088160_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Health Research and Policy, Stanford University School of Medicine, Stanford, CA 94305, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Institute for Human Genetics, University of California, San Francisco, San Francisco, CA 94143, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Epidemiology and Biostatistics, University of California, San Francisco, San Francisco, CA 94107, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Division of Cardiovascular Medicine and Stanford Cardiovascular Institute, Stanford University School of Medicine, Stanford, CA 94305, USA</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Statistics, Stanford University, Stanford, CA 94305, USA</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Department of Social and Preventive Medicine, University at Buffalo, Buffalo, NY 14214, USA</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Kuakini Medical Center, Honolulu, HI 96817, USA</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Departments of Cell &amp; Molecular Biology and Pathology, John A. Burns School of Medicine, University of Hawaii, Honolulu, HI 96813, USA</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Division of Cardiology, George Washington University School of Medicine and Health Sciences, Washington, DC 20037, USA</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Department of Family Medicine, Alpert Medical School of Brown University, Providence, RI 02860, USA</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Department of Epidemiology, Program in Public Health, Brown University, Providence, RI 02860, USA</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Department of Epidemiology and Medicine, University of Iowa, Iowa City, IA 52242, USA</div><div class="fm-affl" id="aff15" lang="en"><sup>15</sup>Division of Research, Kaiser Permanente, Oakland, CA 94612, USA</div><div class="fm-affl" id="aff16" lang="en"><sup>16</sup>Department of Epidemiology and Biostatistics, Case Western Reserve University School of Medicine, Cleveland, OH 44106, USA</div><div class="fm-affl" id="aff17" lang="en"><sup>17</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</div><div class="fm-affl" id="aff18" lang="en"><sup>18</sup>Department of Biostatistics, University of North Carolina, Chapel Hill, NC 27599, USA</div><div class="fm-affl" id="aff19" lang="en"><sup>19</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</div><div class="fm-affl" id="aff20" lang="en"><sup>20</sup>Department of Genetics, Stanford University School of Medicine, Stanford, CA 94305, USA</div><div><span class="contrib-email" id="idm140190164969088">Alex&nbsp;P. Reiner: <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@renierpa" class="oemail">ude.notgnihsaw.u@renierpa</a>; </span> <span class="contrib-email" id="idm140190164966112">Hua Tang: <a href="mailto:dev@null" data-email="ude.drofnats@gnatauh" class="oemail">ude.drofnats@gnatauh</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@renierpa" class="oemail">ude.notgnihsaw.u@renierpa</a></div><div id="cor2"><sup>∗∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ude.drofnats@gnatauh" class="oemail">ude.drofnats@gnatauh</a></div><div id="fn1"><sup>21</sup>Present address: Google Inc., Mountain View, CA 94043, USA</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140190201088160_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Dec 15; Revised 2013 Jan 18; Accepted 2013 Apr 26.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140190201088160_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2013 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3675231/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Figures S1–S6 and Tables S1–S4</div><div><a href="/pmc/articles/PMC3675231/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (286K)</span></div><div class="small guid">GUID:&nbsp;6471B23A-317A-432E-A50A-3C4C1C986530</div></div></dd></dl></div><div id="idm140190164924336" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140190164924336title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Blood lipid concentrations are heritable risk factors associated with atherosclerosis and cardiovascular diseases. Lipid traits exhibit considerable variation among populations of distinct ancestral origin as well as between individuals within a population. We performed association analyses to identify genetic loci influencing lipid concentrations in African American and Hispanic American women in the Women’s Health Initiative SNP Health Association Resource. We validated one African-specific high-density lipoprotein cholesterol locus at <em>CD36</em> as well as 14 known lipid loci that have been previously implicated in studies of European populations. Moreover, we demonstrate striking similarities in genetic architecture (loci influencing the trait, direction and magnitude of genetic effects, and proportions of phenotypic variation explained) of lipid traits across populations. In particular, we found that a disproportionate fraction of lipid variation in African Americans and Hispanic Americans can be attributed to genomic loci exhibiting statistical evidence of association in Europeans, even though the precise genes and variants remain unknown. At the same time, we found substantial allelic heterogeneity within shared loci, characterized both by population-specific rare variants and variants shared among multiple populations that occur at disparate frequencies. The allelic heterogeneity emphasizes the importance of including diverse populations in future genetic association studies of complex traits such as lipids; furthermore, the overlap in lipid loci across populations of diverse ancestral origin argues that additional knowledge can be gleaned from multiple populations.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p3" class="p p-first">Plasma concentrations of lipoproteins (low-density lipoprotein [LDL] cholesterol, high-density lipoprotein [HDL] cholesterol, and triglycerides [TG] are heritable risk factors for atherosclerosis and cardiovascular diseases (CVDs).<sup><a href="#bib1" rid="bib1 bib2" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1,2</a></sup> These lipid concentrations vary substantially between individuals as well as between populations.<sup><a href="#bib3" rid="bib3" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3</a></sup> For example, mean TG levels are highest among Hispanic American populations; despite their higher CVD mortality, African Americans tend to have higher HDL levels and lower TG levels compared to whites.<sup><a href="#bib4" rid="bib4 bib5 bib6 bib7" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">4–7</a></sup> The heritability estimates of blood lipids vary across studies and by population but are consistently high: 40%–80% for both LDL and HDL and 30%–50% for TG.<sup><a href="#bib8" rid="bib8 bib9 bib10" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">8–10</a></sup> A recent meta-analysis of more than 100,000 individuals of European ancestry identified&nbsp;95 loci significantly associated with blood lipids.<sup><a href="#bib11" rid="bib11" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11</a></sup> Follow-up studies indicated that a majority of these loci are “ethnically transferrable,” in the sense that they show statistical association and consistent direction of genetic effects in non-European populations.<sup><a href="#bib3" rid="bib3 bib11 bib12" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3,11,12</a></sup> Specifically, of the 36 LDL loci, 44 HDL loci, and 30 TG loci examined in African American participants, 33, 37, and 24 loci, respectively, showed the same direction of association as in the European cohorts; even higher “replication” rates were observed in East Asians, South Asians, and Hispanics, suggesting a shared genetic contribution to lipid variability among human populations. However, our understanding of the ethnically shared and distinct components of the genetic architecture that underlie blood lipid concentrations is incomplete. Nonreplication could arise as a result of ethnic-specific causal variants, different linkage disequilibrium (LD) patterns surrounding the same causal variants, interaction effects with a distinct genetic background, or simply lack of statistical power. Furthermore, the genetic architecture underlying a trait depends not only on the direction and magnitude of genetic effects, but also on factors such as frequencies of the risk variants. In other words, replication of variants across populations does not automatically imply that these variants constitute a substantial genetic component that explains phenotype variation in all populations.</p><p id="__p4" class="p p-last">Characterizing the shared and distinct genetic components that underlie complex traits among human populations has significant public health impact. Such an understanding regarding lipids levels can provide valuable insights on the genetic contribution to ethnic health disparities related to atherosclerosis and CVD across different ethnicities. However, a genome-wide assessment of the relative contribution of panethnic and population-specific genetic components to any complex trait is analytically challenging because our knowledge of the genetic architecture within each population is limited. Here we present a series of genome-wide analyses that consider various aspects of the genetic architecture of blood lipid levels, using genotype and phenotype data from the Women’s Health Initiative SNP Health Association Resource (WHI-SHARe), which includes 8,153 African American (AA) and 3,587 Hispanic American (HA) participants. By applying a mixed effects model that makes use of the full reference GWAS (instead of just the top SNPs), we demonstrate that a large overlapping genetic component influences individual variation of lipid levels across these populations, including many loci that do not reach genome-wide significance level in any population to date. At the same time, genotype association and admixture mapping analyses reveal important population-specific lipid loci and variants. Our findings argue that an effective approach for elucidating the genetic architecture of complex traits is to combine evidence across multiple populations. The analytic approach we use is applicable to studies of other heritable complex traits.</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Material and Methods</h2><p id="__p5" class="p p-first">Our analytic approach consists of three components. (1) A discovery GWAS in a cohort of 8,153 WHI-SHARe AA samples and 3,587 HA samples. Validation analysis of variants not previously implicated in a large European GWAS was performed in an independent cohort of 7,138 AAs in the Candidate Gene Association Resource (CARe). (2) Genome-wide assessment of transethnic overlap in the genetic architecture underlying lipid variation was performed with the WHI-SHARe AA and HA cohorts by examining statistical significance, allelic effect, and proportion of variance explained by subsets of the genome. (3) Characterization of genetic variation that could account for population-level lipid differences by admixture mapping in WHI-SHARe AAs and conditional analyses in regions showing admixture signal (i.e., association between local ancestry and trait).</p><div id="sec2.1" class="sec"><h3 id="sec2.1title">Study Subjects</h3><p id="__p6" class="p p-first-last">The WHI is a U.S.-wide study focusing on common health issues in postmenopausal women. A total of 161,808 postmenopausal women aged 50–79 years old were recruited, including 12,151 self-identified AAs and 5,469 self-identified HAs. Fasting blood samples were collected at the baseline clinic visit by venipuncture. Clinical information was collected by self-report and physical examination. All participants provided written informed consent as approved by local Human Subjects Committees. Details of the study design and cohort characteristics have been previously described.<sup><a href="#bib13" rid="bib13" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">13</a></sup> An independent cohort of 7,138 AA participants from the NHLBI Candidate-gene Association Resource (CARe) Study was used to validate those SNP-trait associations identified in WHI, which have not been reported in previous GWASs.<sup><a href="#bib11" rid="bib11" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11</a></sup></p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">Genotyping QC and Biomarker Measurement</h3><div id="sec2.2.1" class="sec sec-first"><p></p><h4 id="sec2.2.1title" class="inline">Genotyping and QC </h4><p id="__p7" class="p p-first-last">A cohort of 8,515 self-identified AA and 3,642 self-identified HA participants from WHI, who had consented to genetic research, were selected for WHI SHARe (n = 12,157) and genotyped on the Affymetrix 6.0 array. Genotype quality control criteria included call rate, concordance rates for blinded and unblinded duplicates, and sex discrepancy. Furthermore, individuals whose genetic ancestries differ from self-reported ethnicities and one individual from each close relative pair were excluded. In total, 11,740 individuals passed all genotype and sample QC criteria (8,153 AA, 3,587 HA).<sup><a href="#bib14" rid="bib14 bib15" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">14,15</a></sup> Details of the QC procedures have been described in previous WHI-SHARe studies.<sup><a href="#bib16" rid="bib16 bib17" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16,17</a></sup> Sample sizes for each stage of the study are displayed in <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a> available online; demographic and lipid trait variables are summarized in <a href="#app2" rid="app2" class=" sec">Table S1</a>.</p></div><div id="sec2.2.2" class="sec"><p></p><h4 id="sec2.2.2title" class="inline">Lipid Measurements </h4><p id="__p8" class="p p-first-last">HDL, LDL, and TG measurements were performed at the University of Minnesota by standard biochemical methods on the Roche Modular P Chemistry analyzer (Roche Diagnostics): HDL was measured in serum by the HDL-C plus third generation direct method; TG was measured in serum by Triglyceride GB reagent, and total cholesterol (TC) was measured in serum by a cholesterol oxidase method. The accuracy and precision of the lipid assays were regularly monitored with the CDC/NHLBI Lipid Standardization Program to control for any potential drift over time. LDL was calculated in serum specimens having a TG value &lt; 400&nbsp;mg/dl according to the formula of Friedewald et&nbsp;al.<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup> Based on the LDL-lowering effects of statins, we estimated the pretreatment LDL value for individuals on lipid-lowering medication by dividing treated LDL values by 0.75. All analyses described here are based on analyzing the pretreatment LDL, although the results are largely consistent when treated LDL values were analyzed. For all association analyses, LDL values greater than 300&nbsp;mg/dl and TG values greater than 650 md/dl were excluded. TG values were log-transformed. HDL values were also log-transformed to better satisfy the normality assumption for the linear and linear mixed effect models, although post hoc analyses indicated that the transformed and untransformed trait values yielded qualitatively similar results, and in particular the identical set of genome-wide significant loci.</p></div><div id="sec2.2.3" class="sec sec-last"><p></p><h4 id="sec2.2.3title" class="inline">Genotype Imputation </h4><p id="__p9" class="p p-first-last">Imputed genotypes were examined in regions where at least one genotyped SNP achieved a p &lt; 10<sup>−6</sup> and in regions showing ancestry association in WHI AAs (p &lt; 7&nbsp;× 10<sup>−6</sup>). GWAS data in the WHI AAs were prephased via MaCH, with options “–states 200” and “–rounds 50.”<sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup> The imputation reference panel was derived from the 2012-02-14 release of the 1000 Genomes Project, which included 246 Africans/African Americans, 379 Europeans, 181 Americans, and 286 Asians, using minimac with default parameter&nbsp;settings.<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup> We note that genome-wide imputation to 1000 Genomes Project data is in fact available for WHI-SHARe. However, we chose not to test all imputed markers at the beginning of the study based on a power consideration. The rationale is that, for untyped risk variants with common frequencies, with high probability, there would be a typed variant in moderate LD in the vicinity reaching the relaxed threshold of p &lt; 10<sup>−6</sup>. Additionally, this relaxed threshold can identify regions harboring untyped rare variants, which are well tagged by a SNP on the array. On the other hand, for regions harboring very rare variants that are not tagged by any markers on the array, our sample size is probably underpowered even if we tested the imputed genotype. For this reason, we did not expect an exhaustive scan of all imputed SNPs to be fruitful.</p></div></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Population Structure and Ancestry Estimation</h3><div id="sec2.3.1" class="sec sec-first"><p></p><h4 id="sec2.3.1title" class="inline">Population Structure and Genome-level Ancestry </h4><p id="__p10" class="p p-first-last">Principal component analysis (PCA) was performed for AAs and HAs combined, using Eigenstrat<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup> at 178,101 markers that were in common between our samples and the reference panels. We also determined individual ancestral proportions by using Frappe from 656,852 autosomal markers.<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> For both of these calculations we included 475 publicly available samples from ancestral populations (YRI and CEU from HapMap and East Asian and Native Americans from the Human Genome Diversity Project).<sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup></p></div><div id="sec2.3.2" class="sec sec-last"><p></p><h4 id="sec2.3.2title" class="inline">Local Ancestry Estimation </h4><p id="__p11" class="p p-first-last">For each AA individual in the sample, locus-specific ancestry (probabilities of whether an individual has 0, 1, or 2 alleles of African ancestry at each locus) was estimated with program SABER+, an extension of the SABER algorithm.<sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup> In brief, SABER+ uses a graphical model approach to adaptively capture local haplotype structure within each ancestral population, and thereby more accurately accounts for background linkage disequilibrium (LD). In the current analysis, phased haplotype data from the HapMap3 CEU and YRI individuals were augmented as the reference panels.&nbsp;In simulation studies, SABER+ has an error rate of less than 2%, similar in accuracy to another commonly used method, HapMix.<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup> Based on analysis of simulated and real AA genotypes, the correlation between local-ancestry estimates produced by the two methods is greater than 0.98 (N.A.J., unpublished data).</p></div></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">GWAS Analysis</h3><p id="__p12" class="p p-first-last">The overall GWAS strategy was as follows: we initially tested genotyped SNPs in WHI AAs and HAs separately, with α = 5&nbsp;× 10<sup>−8</sup> as the threshold for genome-wide significance. In regions showing suggestive evidence (at least one SNP with p &lt; 10<sup>−6</sup>), imputed genotypes were tested. To validate SNPs not previously associated with lipid traits, we combined results from WHI AAs and CARe AAs by using the program METAL to perform sample-size weighted meta-analysis.<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup> Association analysis in CARe used individual-level genotype data and corrected for both PC1 and subcohort membership. Genome-wide association (GWA) analysis was performed under an additive genetic model using linear regression adjusted for covariates and implemented in PLINK v.1.05.<sup><a href="#bib14" rid="bib14" class=" bibr popnode">14</a></sup> Analyses were conducted separately for AAs and HAs. To correct for population stratification, the genome-wide European ancestry proportions, which have a correlation of 0.99 with PC1, were adjusted as covariates for AAs (PC2–PC10 showed no evidence of association with any of the lipid traits). For HAs, the first four principal components were adjusted as covariates (PC5–PC10 showed no evidence of association with any of the lipid traits). Age, age<sup>2</sup>, BMI, and smoking history were included as covariates for all lipid traits; additionally, fasting status was adjusted for LDL. The same regression model was used to test the imputed allelic dosage at each SNP, via MACH2QTL.<sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup> The same threshold of 5&nbsp;× 10<sup>−8</sup> was used to declare genome-wide statistical significance for the imputed genotypes, although we note that the significant regions identified remained the same when a more stringent threshold of 2.5&nbsp;× 10<sup>−8</sup> was adopted.<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup></p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Comparison of p Values and Genetic Effects between Populations</h3><p id="__p13" class="p p-first">We characterized the shared genetic architecture between Europeans, AAs, and HAs by enrichment in statistical significance (p values), correlation in estimated genetic effects, and the proportions of variance explained by subsets of genomes. To delineate the enrichment in statistical significance, we asked whether SNPs showing suggestive evidence in Europeans (defined by p&nbsp;&lt;&nbsp;10<sup>−5</sup>) tend to have small p values in AAs and HAs. The p values in Europeans were taken from one of the largest lipid meta-analysis to date.<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> The p values for WHI AAs and HAs were obtained from GWAS based on the single-marker additive model.</p><p id="__p14" class="p p-last">To compare genetic effects across populations, we again defined candidate lipid-associated SNPs based on the p values in the European GWAS (p &lt; 10<sup>−5</sup>) and computed correlation coefficients between the estimates in WHI AAs and HAs. Because all WHI participants were genotyped with the same platform and all lipid biomarkers were measured in a consistent manner, this analysis avoids potential artifacts resulting from nonoverlapping SNPs, allele flipping, and assay batch effects. Furthermore, because the SNPs were chosen based on the p values in a GWAS whose participants do not overlap with WHI, the estimated genetic effects in AAs and HAs do not suffer from the typical bias due to selecting the most significant SNPs (i.e., winner’s curse).<sup><a href="#bib28" rid="bib28" class=" bibr popnode">28</a></sup></p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">Estimation of Phenotypic Variance Explained by Subsets of Genomes</h3><p id="__p15" class="p p-first">We developed a method, termed population overlap in genetic architecture (POGA), which tests the hypothesis that loci identified in one population explain a large proportion of the phenotypic variance in other populations. POGA is an extension of the mixed effects model of a polygenic trait, introduced by Yang and Visscher.<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup> Under this approach, we prioritized the genomes into regions based on association evidence in Europeans and estimated the phenotypic variance explained in AAs and HAs, using a mixed effects model. We reasoned that if the genetic architecture overlaps between populations, loci showing the strongest evidence of association in one population would account for a disproportionate amount of phenotypic variation in another population.</p><div id="sec2.6.1" class="sec"><p></p><h4 id="sec2.6.1title" class="inline">POGA Algorithm </h4><p id="__p16" class="p p-first-last">For each SNP genotyped in WHI, we assigned a priority score (denoted by <em>u</em><sub><em>m</em></sub>) as the smallest p values within a 20 kb neighborhood based on the European GWAS.<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> We then grouped the genome into 22 nested subsets, such that the first subset included regions around SNPs that reached genome-wide significance in the European GWAS (p &lt; 5&nbsp;× 10<sup>−8</sup>); the second subset included SNPs with the top 1% priority scores; the third and subsequent subsets incrementally included the 5%, 10%, … 100% of the markers according to <em>u</em><sub><em>m</em></sub>. For each subset s = 1,…,22, we computed the proportions of phenotypic variance explained with program GCTA (v.1.02) and the AI-REML algorithm.<sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup> The model is:</p><div class="disp-formula" id="ufd1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" altimg="si1.gif" overflow="scroll"><mrow><msub><mi>Y</mi><mi>i</mi></msub><mo>=</mo><mi>α</mi><mo>+</mo><mi>β</mi><msub><mi>Q</mi><mi>i</mi></msub><mo>+</mo><msubsup><mi>G</mi><mi>i</mi><mi>s</mi></msubsup><mo>+</mo><msubsup><mi>H</mi><mi>i</mi><mi>s</mi></msubsup><mo>+</mo><msub><mi>ε</mi><mi>i</mi></msub><mo>,</mo></mrow></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M1" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="msub" id="MathJax-Span-4"><span style="display: inline-block; position: relative; width: 0.9em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.76em, 4.131em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.182em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.581em;"><span class="mi" id="MathJax-Span-6" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mi" id="MathJax-Span-8" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">α</span><span class="mo" id="MathJax-Span-9" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mi" id="MathJax-Span-10" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span class="msub" id="MathJax-Span-11"><span style="display: inline-block; position: relative; width: 1.11em; height: 0px;"><span style="position: absolute; clip: rect(3.118em, 1000.74em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-12" style="font-family: MathJax_Math; font-style: italic;">Q</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.791em;"><span class="mi" id="MathJax-Span-13" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-14" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msubsup" id="MathJax-Span-15" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.193em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.76em, 4.154em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-16" style="font-family: MathJax_Math; font-style: italic;">G</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.51em, 1000.41em, 4.139em, -1000em); top: -4.329em; left: 0.786em;"><span class="mi" id="MathJax-Span-17" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.355em, 1000.32em, 4.14em, -1000em); top: -3.674em; left: 0.786em;"><span class="mi" id="MathJax-Span-18" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-19" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msubsup" id="MathJax-Span-20" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.362em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-21" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.51em, 1000.41em, 4.139em, -1000em); top: -4.329em; left: 0.955em;"><span class="mi" id="MathJax-Span-22" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.355em, 1000.32em, 4.14em, -1000em); top: -3.674em; left: 0.831em;"><span class="mi" id="MathJax-Span-23" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-24" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-25" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.785em; height: 0px;"><span style="position: absolute; clip: rect(3.37em, 1000.43em, 4.154em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-26" style="font-family: MathJax_Math; font-style: italic;">ε</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.466em;"><span class="mi" id="MathJax-Span-27" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-28" style="font-family: MathJax_Main;">,</span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" altimg="si1.gif" overflow="scroll"><mrow><msub><mi>Y</mi><mi>i</mi></msub><mo>=</mo><mi>α</mi><mo>+</mo><mi>β</mi><msub><mi>Q</mi><mi>i</mi></msub><mo>+</mo><msubsup><mi>G</mi><mi>i</mi><mi>s</mi></msubsup><mo>+</mo><msubsup><mi>H</mi><mi>i</mi><mi>s</mi></msubsup><mo>+</mo><msub><mi>ε</mi><mi>i</mi></msub><mo>,</mo></mrow></math></script></div></div><p>where Q are the fixed effect covariates (age and PCs) and <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" altimg="si2.gif" overflow="scroll"><mrow><msubsup><mi>G</mi><mi>i</mi><mi>s</mi></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-2-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" altimg="si2.gif" overflow="scroll"><mrow><msubsup><mi>G</mi><mi>i</mi><mi>s</mi></msubsup></mrow></math></script> and <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" altimg="si3.gif" overflow="scroll"><mrow><msubsup><mi>H</mi><mi>i</mi><mi>s</mi></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-3-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" altimg="si3.gif" overflow="scroll"><mrow><msubsup><mi>H</mi><mi>i</mi><mi>s</mi></msubsup></mrow></math></script> are the random effects representing the genetic effects resulting from markers included and excluded by set <em>s</em> respectively. The inclusion of <em>H</em><sup><em>s</em></sup> in the model is necessary because markers in set <em>s</em> can be in LD with flanking regions not in the set, and thus a model that includes <em>G</em><sup><em>s</em></sup> alone tends to overestimate the variance explained just by the genomic regions.</p></div><div id="sec2.6.2" class="sec"><p></p><h4 id="sec2.6.2title" class="inline">Genetic Relationship Matrix </h4><p id="__p17" class="p p-first-last">In a standard application of GCTA, a genetic relationship matrix (GRM) is computed as the covariance matrix of the standardized genotype matrix, <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" altimg="si4.gif" overflow="scroll"><mrow><msub><mi>w</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>g</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mo>−</mo><mn>2</mn><msub><mi>p</mi><mi>m</mi></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mrow><mn>2</mn><msub><mi>p</mi><mi>m</mi></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mi>p</mi><mi>m</mi></msub></mrow><mo>)</mo></mrow></mrow></msqrt></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-4-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" altimg="si4.gif" overflow="scroll"><mrow><msub><mi>w</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>g</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mo>−</mo><mn>2</mn><msub><mi>p</mi><mi>m</mi></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mrow><mn>2</mn><msub><mi>p</mi><mi>m</mi></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mi>p</mi><mi>m</mi></msub></mrow><mo>)</mo></mrow></mrow></msqrt></mrow></math></script>, where <em>g</em><sub><em>i</em><em>m</em></sub>&nbsp;∈&nbsp;(0,&nbsp;1,&nbsp;2) represents the original genotype and <em>p</em><sub><em>m</em></sub> represents the allele frequencies. We have previously shown that relationship coefficients estimated this way can be biased in an admixed population, and we have developed a <em>reap</em> estimator, which does not suffer from such bias.<sup><a href="#bib31" rid="bib31" class=" bibr popnode">31</a></sup> In brief, the <em>reap</em> estimator standardizes the genotype by an “admixture-adjusted” allele frequency: <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" altimg="si6.gif" overflow="scroll"><mrow><msub><mi>w</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>g</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mo>−</mo><mn>2</mn><msub><mi>p</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mrow><mn>2</mn><msub><mi>p</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mi>p</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow></msqrt></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-5-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" altimg="si6.gif" overflow="scroll"><mrow><msub><mi>w</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>g</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mo>−</mo><mn>2</mn><msub><mi>p</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mrow><mn>2</mn><msub><mi>p</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mi>p</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow></msqrt></mrow></math></script> where <em>p</em><sub><em>im</em></sub> is the expected allele frequencies given the individual’s genome-wide ancestry proportions. For a given subset of markers, <em>s</em>, we computed the GRM by using the <em>reap</em> estimator, for markers included in the subset (<em>G</em><sup><em>s</em></sup>) and not included in the subset (<em>H</em><sup><em>s</em></sup>), and supplied these two GRM into GCTA. For each subset, denote the proportions of variance attributed to <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M7" altimg="si2.gif" overflow="scroll"><mrow><msubsup><mi>G</mi><mi>i</mi><mi>s</mi></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-6-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M7" altimg="si2.gif" overflow="scroll"><mrow><msubsup><mi>G</mi><mi>i</mi><mi>s</mi></msubsup></mrow></math></script> as <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M8" altimg="si7.gif" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>s</mi><mn>2</mn></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-7-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-7"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M8" altimg="si7.gif" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>s</mi><mn>2</mn></msubsup></mrow></math></script>. As an informal test of significance, we compared the estimated <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M9" altimg="si7.gif" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>s</mi><mn>2</mn></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-8-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-8"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M9" altimg="si7.gif" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>s</mi><mn>2</mn></msubsup></mrow></math></script> with a simple null model, under which we expect that the phenotypic variance explained by a randomly selected region is proportional to its coverage of the genome. Thus, a set of regions that includes <em>x</em>% of the genome would explain <em>h</em><sup>2</sup><em>x</em>% of the phenotypic variance, where <em>h</em><sup>2</sup> is the variance explained by the entire genome.</p></div><div id="sec2.6.3" class="sec sec-last"><p></p><h4 id="sec2.6.3title" class="inline">A Scenario with No Overlap in Genetic Architecture </h4><p id="__p18" class="p p-first-last">To verify that the expected <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M10" altimg="si7.gif" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>s</mi><mn>2</mn></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-9-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-9"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M10" altimg="si7.gif" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>s</mi><mn>2</mn></msubsup></mrow></math></script>, for a set of randomly selected regions, is proportional to the proportion of the genome included, we performed the following permutation experiment. First, the European p value list was permuted in a way that largely preserves the correlation structure between neighboring SNPs. The p values were sorted by chromosome and base pair location. The 22 autosomes were concatenated, circularized, and then cut at a random location. The resulting list of p values was then mapped to the original SNP positions, starting with the p-term of chromosome 1 and ending with the last SNP position on chromosome 22. This permuted list of p values were then used as if they were p&nbsp;values from a European GWAS study, and the genomes were regrouped based on these permuted p values. GCTA analyses were repeated and <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M11" altimg="si7.gif" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>s</mi><mn>2</mn></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-10-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-10"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M11" altimg="si7.gif" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>s</mi><mn>2</mn></msubsup></mrow></math></script> estimated for these randomly partitioned genomes. This permutation procedure was performed for each HDL, LDL, and TG p value list and used to estimate the proportion of LDL variance explained.</p></div></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">Admixture Mapping</h3><p id="__p19" class="p p-first-last">With the estimated local ancestry, we performed an admixture mapping analysis in AAs to detect variants present at different frequencies among the European and African ancestral populations. In this analysis, we regressed lipid levels on locus-specific ancestry, adjusting for the same covariates as in the GWAS. The critical value for genome-wide significance level of admixture mapping is substantially lower than for the genotype test, because the recent admixing history gives rise to extensive correlation in local ancestry. Based on previous theoretical analysis and simulation results, a nominal p value of 7&nbsp;× 10<sup>−6</sup> yielded a genome-wide type I error of 0.05.<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> We chose not to perform admixture mapping in the HA sample because we expect such an analysis is severely underpowered as a result of the lack of availability of an appropriate Native American reference panel (which impacts estimation accuracy) and the smaller HA sample size (compared to the AA sample).</p></div><div id="sec2.8" class="sec sec-last"><h3 id="sec2.8title">Conditional Analysis under Admixture Mapping Peaks</h3><p id="__p20" class="p p-first-last">Genetic regions showing significant association with local ancestry tend to be broad because of the recent admixing history in AAs. To refine these admixture mapping regions and in an effort to reveal genes or variants that contribute to population-level trait differences, we performed conditional analyses with all typed and imputed SNPs in each region. We reason that variants that “explain” an admixture mapping peak should meet two criteria. First, these variants should show suggestive association with the trait conditioning on the genome-wide ancestry; and second, these variants should substantially reduce the local ancestry-trait association. To test the first criterion, we required the variants to have GWAS p &lt; 10<sup>−5</sup>. In genes where multiple rare variants (defined by an allele frequency of less than 1% in AAs) show association with blood lipids, we also evaluated gene-based association by using either dosage (sum of rare alleles) or an indicator defined as dosage &gt; 0. To assess the second criterion, we used a joint regression model that includes both local ancestry and SNP genotype in addition to all covariates adjusted in the GWAS and required that the p value for local ancestry to be attenuated (i.e., less significant) by at least 100-fold compared to that in the model without the SNP genotype. In regions where multiple SNPs or genes meet both criteria, we performed a step-wise regression to nominate a set of SNPs that may jointly explain the local-ancestry association.</p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">Genome-wide Association Analysis</h3><p id="__p21" class="p p-first">Genotype association analyses in AAs identified seven, five, and four loci significantly associated with HDL, LDL, and TG, respectively (<a href="/pmc/articles/PMC3675231/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> and <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>). For HDL, these included <em>CD36</em> (MIM <a href="/omim/173510" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">173510</a>), <em>PPP1R3B</em> (MIM <a href="/omim/610541" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">610541</a>), <em>LPL</em> (MIM <a href="/omim/609708" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609708</a>), <em>CETP</em> (MIM <a href="/omim/118470" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">118470</a>), <em>LOC55908,</em> the <em>APOA/APOC</em> gene cluster (<em>APOA1</em>, <em>APOC3</em>, <em>APOA4</em>, <em>APOA5</em> [MIM <a href="/omim/107680" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107680</a>, MIM <a href="/omim/107720" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107720</a>, MIM <a href="/omim/107690" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107690</a>, MIM <a href="/omim/606368" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606368</a>]), and an intergenic locus on 21q22; for LDL, <em>PCSK9</em> (MIM <a href="/omim/607786" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607786</a>), <em>APOB</em> (MIM <a href="/omim/107730" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107730</a>), <em>ABCG8</em> (MIM <a href="/omim/605460" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605460</a>), <em>APOE</em> (MIM <a href="/omim/107741" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107741</a>), and <em>LDLR</em> (MIM <a href="/omim/606945" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606945</a>); and for TG, <em>LPL</em>, <em>APOA/APOC</em>, <em>APOC1</em> (MIM <a href="/omim/107710" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107710</a>), and <em>GCKR</em> (MIM <a href="/omim/600842" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600842</a>). The inflation factor for genomic control in AAs was 1.061, 1.057, and 1.046 for HDL, LDL, and TG, respectively, indicating adequate adjustment of population stratification.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p66">Loci Associated with Lipids Traits in WHI AAs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190198638400" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Chr</strong></th><th rowspan="1" colspan="1"><strong>Index SNP in the Region</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>Pos (hg18)</strong></th><th rowspan="1" colspan="1"><strong>Minor/Major Allele</strong></th><th rowspan="1" colspan="1"><strong>Minor Allele Frequency</strong></th><th rowspan="1" colspan="1"><strong>Candidate Gene</strong></th><th rowspan="1" colspan="1"><strong>MIM</strong></th><th rowspan="1" colspan="1"><strong>Beta</strong><sup>b</sup></th><th rowspan="1" colspan="1"><strong>p Value</strong></th><th rowspan="1" colspan="1"><strong>Trait</strong></th></tr></thead><tbody><tr><td colspan="10" rowspan="1"><strong>Genotype Association</strong></td></tr><tr><td colspan="10" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs17111684</td><td rowspan="1" colspan="1" align="center">55398136</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1" align="center">0.120</td><td rowspan="1" colspan="1"><em>PCSK9</em></td><td rowspan="1" colspan="1"><a href="/omim/607786" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607786</a></td><td rowspan="1" colspan="1" align="center">−9.01</td><td rowspan="1" colspan="1" align="center">2.40&nbsp;× 10<sup>−17</sup></td><td rowspan="1" colspan="1">LDL</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs12713956</td><td rowspan="1" colspan="1" align="center">21095010</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1" align="center">0.183</td><td rowspan="1" colspan="1"><em>APOB</em></td><td rowspan="1" colspan="1"><a href="/omim/107730" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107730</a></td><td rowspan="1" colspan="1" align="center">−4.86</td><td rowspan="1" colspan="1" align="center">3.74&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1">LDL</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs4665972<sup>c</sup></td><td rowspan="1" colspan="1" align="center">27451601</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.123</td><td rowspan="1" colspan="1"><em>GCKR</em></td><td rowspan="1" colspan="1"><a href="/omim/600842" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600842</a></td><td rowspan="1" colspan="1" align="center">0.065</td><td rowspan="1" colspan="1" align="center">1.05&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1">TG</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs4245791</td><td rowspan="1" colspan="1" align="center">43927935</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1" align="center">0.143</td><td rowspan="1" colspan="1"><em>ABCG8</em></td><td rowspan="1" colspan="1"><a href="/omim/605460" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605460</a></td><td rowspan="1" colspan="1" align="center">5.97</td><td rowspan="1" colspan="1" align="center">1.24&nbsp;× 10<sup>−09</sup></td><td rowspan="1" colspan="1">LDL</td></tr><tr><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">rs2366858</td><td rowspan="1" colspan="1" align="center">80178558</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1" align="center">0.173</td><td rowspan="1" colspan="1"><em>CD36</em></td><td rowspan="1" colspan="1"><a href="/omim/173510" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">173510</a></td><td rowspan="1" colspan="1" align="center">0.0325</td><td rowspan="1" colspan="1" align="center">5.59&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs1461729</td><td rowspan="1" colspan="1" align="center">9224652</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.116</td><td rowspan="1" colspan="1"><em>PPP1R3B</em></td><td rowspan="1" colspan="1"><a href="/omim/610541" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">610541</a></td><td rowspan="1" colspan="1" align="center">−0.0355</td><td rowspan="1" colspan="1" align="center">7.39&nbsp;× 10<sup>−09</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs326</td><td rowspan="1" colspan="1" align="center">19863719</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.469</td><td rowspan="1" colspan="1"><em>LPL</em></td><td rowspan="1" colspan="1"><a href="/omim/609708" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609708</a></td><td rowspan="1" colspan="1" align="center">0.0221</td><td rowspan="1" colspan="1" align="center">1.23&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs326</td><td rowspan="1" colspan="1" align="center">19863719</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.469</td><td rowspan="1" colspan="1"><em>LPL</em></td><td rowspan="1" colspan="1"><a href="/omim/609708" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609708</a></td><td rowspan="1" colspan="1" align="center">−0.0410</td><td rowspan="1" colspan="1" align="center">1.02&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1">TG</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs6589566</td><td rowspan="1" colspan="1" align="center">116157633</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1" align="center">0.0176</td><td rowspan="1" colspan="1"><em>APOA/APOC</em></td><td rowspan="1" colspan="1"><a href="/omim/107680" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107680</a>, <a href="/omim/107720" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107720</a>, <a href="/omim/107690" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107690</a>, <a href="/omim/606368" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606368</a></td><td rowspan="1" colspan="1" align="center">0.2066</td><td rowspan="1" colspan="1" align="center">4.99&nbsp;× 10<sup>−14</sup></td><td rowspan="1" colspan="1">TG</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">chr11: 116,799,496<sup>c</sup></td><td rowspan="1" colspan="1" align="center">116304706</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1" align="center">0.0016</td><td rowspan="1" colspan="1"><em>APOA/APOC</em></td><td rowspan="1" colspan="1"><a href="/omim/107680" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107680</a>, <a href="/omim/107720" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107720</a>, <a href="/omim/107690" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107690</a>, <a href="/omim/606368" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606368</a></td><td rowspan="1" colspan="1" align="center">0.409</td><td rowspan="1" colspan="1" align="center">1.08&nbsp;× 10<sup>−12</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs247617</td><td rowspan="1" colspan="1" align="center">55548217</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1" align="center">0.262</td><td rowspan="1" colspan="1"><em>CETP</em></td><td rowspan="1" colspan="1"><a href="/omim/118470" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">118470</a></td><td rowspan="1" colspan="1" align="center">0.0619</td><td rowspan="1" colspan="1" align="center">1.48&nbsp;× 10<sup>−44</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs17249141<sup>c</sup></td><td rowspan="1" colspan="1" align="center">11061008</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.0126</td><td rowspan="1" colspan="1"><em>LDLR</em></td><td rowspan="1" colspan="1"><a href="/omim/606945" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606945</a></td><td rowspan="1" colspan="1" align="center">−32.93</td><td rowspan="1" colspan="1" align="center">2.43&nbsp;× 10<sup>−17</sup></td><td rowspan="1" colspan="1">LDL</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs12979813</td><td rowspan="1" colspan="1" align="center">11203703</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.495</td><td rowspan="1" colspan="1"><em>LOC55908</em></td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1" align="center">−0.0235</td><td rowspan="1" colspan="1" align="center">1.99&nbsp;× 10<sup>−09</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs1160985</td><td rowspan="1" colspan="1" align="center">50095252</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1" align="center">0.365</td><td rowspan="1" colspan="1"><em>APOE</em></td><td rowspan="1" colspan="1"><a href="/omim/107741" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107741</a></td><td rowspan="1" colspan="1" align="center">6.772</td><td rowspan="1" colspan="1" align="center">1.87&nbsp;× 10<sup>−21</sup></td><td rowspan="1" colspan="1">LDL</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs12721054</td><td rowspan="1" colspan="1" align="center">50114427</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1" align="center">0.1137</td><td rowspan="1" colspan="1"><em>APOC1</em></td><td rowspan="1" colspan="1"><a href="/omim/107710" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107710</a></td><td rowspan="1" colspan="1" align="center">−0.101</td><td rowspan="1" colspan="1" align="center">2.86&nbsp;× 10<sup>−19</sup></td><td rowspan="1" colspan="1">TG</td></tr><tr><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">rs13046373</td><td rowspan="1" colspan="1" align="center">30982361</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1" align="center">0.391</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">−0.0226</td><td rowspan="1" colspan="1" align="center">2.26&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td colspan="10" rowspan="1"><hr></td></tr><tr><td colspan="10" rowspan="1"><strong>Admixture Mapping</strong><hr></td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs10818782</td><td rowspan="1" colspan="1">98.5–101.3 Mb</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0508</td><td rowspan="1" colspan="1" align="center">5.57&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs11217785</td><td rowspan="1" colspan="1">118.6–122.1 Mb</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>UBASH3B</em></td><td rowspan="1" colspan="1"><a href="/omim/609201" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609201</a></td><td rowspan="1" colspan="1" align="center">0.0526</td><td rowspan="1" colspan="1" align="center">2.82&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs1889209</td><td rowspan="1" colspan="1">54.8–55.5 Mb</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>PCSK9</em></td><td rowspan="1" colspan="1"><a href="/omim/607786" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607786</a></td><td rowspan="1" colspan="1" align="center">−8.50</td><td rowspan="1" colspan="1" align="center">2.11&nbsp;× 10<sup>−06</sup></td><td rowspan="1" colspan="1">LDL</td></tr></tbody></table></div><div id="largeobj_idm140190198638400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3675231/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140190198406352"><p id="__p67" class="p p-first-last">n = 7,917; 7,861; and 7,918 for HDL, LDL, and TG, respectively.</p></div></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>SNPs with the lowest p value at a locus.</div></div><div class="tblwrap-foot"><div id="tblfn2"><sup>b</sup>For genotype association, the direction of the regression coefficient represents the effect of each extra minor allele. For admixture mapping, the direction of the regression coefficient represents the effect of an additional African-derived allele.</div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>c</sup>Loci where no genotyped SNP reaches genome-wide significance (5&nbsp;× 10<sup>−8</sup>) but at least one genotyped SNP reaches a p &lt; 10<sup>−6</sup> and at least one imputed SNP reaches p &lt; 5&nbsp;× 10<sup>−8</sup> (see <a href="#sec2" rid="sec2" class=" sec">Methods</a>).</div></div></div><p id="__p22">With the exception of <em>CD36</em> and the locus on 21q22, all other loci are in proximity to previously implicated regions in GWASs in Europeans.<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> CD36 is a scavenger receptor that binds long-chain fatty acids and lipoproteins; genetic variants in <em>CD36</em> have been associated with protection from various components of the metabolic syndrome (MetS) in AAs,<sup><a href="#bib33" rid="bib33 bib34" class=" bibr popnode">33,34</a></sup> and CD36-deficient individuals have been observed to have higher HDL compared to controls.<sup><a href="#bib35" rid="bib35" class=" bibr popnode">35</a></sup> Recently, association between a nonsynonymous SNP in <em>CD36</em> (rs3211938, p.Tyr325Ter) and HDL was reported in a meta-analysis in CARe.<sup><a href="#bib12" rid="bib12" class=" bibr popnode">12</a></sup> The same SNP achieved a p &lt; 7&nbsp;× 10<sup>−9</sup> in WHI and p &lt; 1.45&nbsp;× 10<sup>−19</sup> in the meta-analysis that included AA participants in WHI and CARe. The HDL-increasing allele (G) occurs at a frequency of 0.28 in YRI and is essentially absent in Europeans and East Asians. At 21q22, two SNPs reached genome-wide significance in WHI AAs for HDL. This region has not been previously implicated in lipid genetics and was not replicated in CARe (meta p = 4.10&nbsp;× 10<sup>−6</sup>); therefore, we did not pursue this locus further in this study.</p><p id="__p23" class="p p-last">In the WHI HA cohort, <em>APOA/APOC</em> and <em>CETP</em> reached genome-wide significant association with HDL and <em>GCKR</em>, <em>LPL</em>, and <em>APOA/APOC</em> were found significantly associated with TG. The number of HA participants is much lower compared to AA participants in WHI; therefore, we expect, a priori, that the power of GWAS is much lower in HAs (<a href="/pmc/articles/PMC3675231/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a> and <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p71">Genome-wide Significant Regions in WHI HAs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190198396496" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Chr</strong></th><th rowspan="1" colspan="1"><strong>Index SNP in the Region</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>Pos (hg 18)</strong></th><th rowspan="1" colspan="1"><strong>Minor/Major Allele</strong></th><th rowspan="1" colspan="1"><strong>Minor Allele Frequency</strong></th><th rowspan="1" colspan="1"><strong>Candidate Gene</strong></th><th rowspan="1" colspan="1"><strong>MIM</strong></th><th rowspan="1" colspan="1"><strong>Beta</strong><sup>b</sup></th><th rowspan="1" colspan="1"><strong>p Value</strong></th><th rowspan="1" colspan="1"><strong>Trait</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1" align="center">27594741</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1" align="center">0.358</td><td rowspan="1" colspan="1"><em>GCKR</em></td><td rowspan="1" colspan="1"><a href="/omim/600842" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600842</a></td><td rowspan="1" colspan="1" align="center">0.0688</td><td rowspan="1" colspan="1" align="center">7.35&nbsp;× 10<sup>−09</sup></td><td rowspan="1" colspan="1">TG</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs17410962</td><td rowspan="1" colspan="1" align="center">19892360</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1" align="center">0.112</td><td rowspan="1" colspan="1"><em>LPL</em></td><td rowspan="1" colspan="1"><a href="/omim/609708" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609708</a></td><td rowspan="1" colspan="1" align="center">−0.1064</td><td rowspan="1" colspan="1" align="center">7.35&nbsp;× 10<sup>−09</sup></td><td rowspan="1" colspan="1">TG</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs964184</td><td rowspan="1" colspan="1" align="center">116154127</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1" align="center">0.248</td><td rowspan="1" colspan="1"><em>APOA/APOC</em></td><td rowspan="1" colspan="1"><a href="/omim/107680" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107680</a>, <a href="/omim/107720" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107720</a>, <a href="/omim/107690" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107690</a>, <a href="/omim/606368" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606368</a></td><td rowspan="1" colspan="1" align="center">−0.0459</td><td rowspan="1" colspan="1" align="center">2.81&nbsp;× 10<sup>−12</sup></td><td rowspan="1" colspan="1">HDL</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs964184</td><td rowspan="1" colspan="1" align="center">116154127</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1" align="center">0.248</td><td rowspan="1" colspan="1"><em>APOA/APOC</em></td><td rowspan="1" colspan="1"><a href="/omim/107680" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107680</a>, <a href="/omim/107720" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107720</a>, <a href="/omim/107690" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107690</a>, <a href="/omim/606368" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606368</a></td><td rowspan="1" colspan="1" align="center">0.1567</td><td rowspan="1" colspan="1" align="center">3.66&nbsp;× 10<sup>−33</sup></td><td rowspan="1" colspan="1">TG</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs247617</td><td rowspan="1" colspan="1" align="center">55548217</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1" align="center">0.298</td><td rowspan="1" colspan="1"><em>CETP</em></td><td rowspan="1" colspan="1"><a href="/omim/118470" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">118470</a></td><td rowspan="1" colspan="1" align="center">0.0509</td><td rowspan="1" colspan="1" align="center">3.48&nbsp;× 10<sup>−16</sup></td><td rowspan="1" colspan="1">HDL</td></tr></tbody></table></div><div id="largeobj_idm140190198396496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3675231/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140190198320464"><p id="__p72" class="p p-first-last">n = 3,506; 3,425; and 3,506 for HDL, LDL, and TG, respectively.</p></div></div><div class="tblwrap-foot"><div id="tblfn4"><sup>a</sup>SNPs with the lowest p value at a locus.</div></div><div class="tblwrap-foot"><div id="tblfn5"><sup>b</sup>The direction of the regression coefficient represents the effect of each extra minor allele.</div></div></div></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">Genetic Variants Associated with Lipid Traits Overlap across Ethnicities and Have Correlated Allelic Effects</h3><p id="__p24" class="p p-first">Previous studies of traits such as lipids, height, BMI, and coronary heart disease have found that genetic risk factors identified in populations of European descent often show consistent direction of effect in non-European populations.<sup><a href="#bib3" rid="bib3" class=" bibr popnode">3</a></sup> In WHI, a majority of lipid loci reaching the genome-wide significance level in AAs and all loci found in HAs overlapped with loci discovered in European populations.<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> Furthermore, we observed a strong enrichment of small p values in the WHI cohorts among those SNPs showing significance or suggestive evidence of association in the European GWAS (p &lt; 10<sup>−5</sup>) (<a href="/pmc/articles/PMC3675231/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>); as a comparison, the p value distribution for all SNPs appears roughly uniform (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>). This suggests that, with&nbsp;a sufficient sample size, a large fraction of loci influencing lipid traits in Europeans would probably reach statistical significance in AAs and HAs.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3675231/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190198661440" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC3675231/bin/gr1.jpg"></div></a><div id="largeobj_idm140190198661440" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3675231/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC3675231/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3675231/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p51">Enrichment for Small p Values among SNPs that Are Significantly or Suggestively Associated in European GWASs</p><p id="__p52">(A and B) High-density lipoprotein (HDL) cholesterol in African Americans (AAs) (A) and in Hispanic Americans (HAs) (B).</p><p id="__p53">(C and D) Low-density lipoprotein (LDL) cholesterol in AAs (C) and in HAs (D).</p><p id="__p54">(E and F) Triglycerides (TG) in AAs (E) and in HAs (F).</p></div></div></div><p id="__p25" class="p p-last">Although previous studies have focused on consistency of the direction of genetic effects between populations, it is desirable to assess whether the risk variants have genetic effects of similar magnitude because this quantity plays an important role in individual disease risk prediction. Furthermore, the correlation in allelic effects is a more informative test because it also considers the strength of correlation. A caveat in this analysis is that, in order to avoid bias resulting from regression to the mean (or winner’s curse), the effects should be estimated from a sample that is independent of the sample used to select the risk variants. Therefore, we defined lipid-associated SNPs based on the results from the largest European GWAS to date<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> and compared the estimated genetic effects per allele, or allelic effect, in WHI AAs and HAs. We examined SNPs with p ≤ 10<sup>−5</sup> in Europeans for HDL, LDL, and TG, respectively. For HDL, 992 SNPs were considered and the correlation of the estimated allelic effects in AAs and HAs was 0.498; for LDL, 786 SNPs were considered and the correlation was 0.429; for TG, 810 SNPs were considered and the correlation was 0.602 (<a href="/pmc/articles/PMC3675231/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figures 2</span></a>A–2C). As a negative control, we repeated this analysis on SNPs with p values greater than 0.01 in Europeans. As expected, estimated allelic effects in AAs and HAs were essentially uncorrelated (0.0113, 0.00241, and 0.00519 for HDL, LDL, and TG, respectively). Because some SNPs included in this analysis may be in LD, the observed correlation between AAs and HAs may be overestimated. To eliminate this possibility, <a href="/pmc/articles/PMC3675231/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>D displays the estimated genetic effects in HAs versus AAs for 92 index SNPs from distinct loci implicated in the European GWAS for HDL, LDL, and TG. The Pearson correlation between the estimated effects was 0.69 (p = 1.11&nbsp;× 10<sup>−14</sup>), despite the fact that only 34 and 40 of the 92 SNPs had nominal p values less than 0.05 in AAs and HAs, respectively. These results suggest that the degree of overlap in risk loci between populations exceeds the set of SNPs that replicate on the basis of a predefined p value or significance threshold; therefore, combining association evidence across populations is likely to increase the overall efficiency of the ability to identify new trait loci.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC3675231/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190198652208" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC3675231/bin/gr2.jpg"></div></a><div id="largeobj_idm140190198652208" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3675231/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC3675231/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC3675231/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p55">Genetic Effects of Candidate Lipid Variants Are Correlated between African Americans and Hispanic Americans</p><p id="__p56">(A–C) SNPs are those with p &lt; 10<sup>−5</sup> in a European GWAS. The <em>x</em> axis represents the estimated allelic effects in African Americans (AAs) and the <em>y</em> axis represents the estimated allelic effects of the corresponding SNPs in Hispanic Americans (HAs).</p><p id="__p57">(A) High-density lipoprotein (HDL) cholesterol.</p><p id="__p58">(B) Low-density lipoprotein (LDL) cholesterol.</p><p id="__p59">(C) Triglycerides (TG).</p><p id="__p60">(D) Genetic effects of 92 SNPs representing&nbsp;independent loci; SNPs are the best surrogate index SNPs on Affy 6.0 arrays defined by Telosvich et&nbsp;al.<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> The estimated allelic effects in (D) are in the unit of standard deviation of the phenotype.</p></div></div></div></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">Quantifying “Overlapping Heritability”</h3><p id="__p26" class="p p-first">We next asked whether loci associated in European populations contribute substantially to the genetic architecture of lipid traits in AAs, in the sense of phenotypic variance explained. The approach we developed, termed population overlap in genetic architecture (POGA), is based on an extension of the mixed effects model of a polygenic trait, introduced by Yang and Visscher.<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup> The rationale of POGA is intuitive: if the “important” risk loci overlap between populations, then we expect that the loci showing the strongest evidence of association in one population will account for a substantial proportion of the phenotypic variation in another population. This approach has two features. First it makes use of the complete list of p values from a reference GWAS instead of just the top SNPs; and second, this method defines “overlap” more broadly to accommodate allelic heterogeneity and unknown LD patterns.</p><p id="__p27" class="p p-last"><a href="/pmc/articles/PMC3675231/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a> displays the proportion of phenotypic variance in AAs explained by <em>G</em><sup><em>s</em></sup> when the markers in <em>s</em> expand from regions that met genome-wide significance levels in Europeans to the entire genome, and will be referred to as a POGA plot. The entire genome explains 28.5%, 30.5%, and 18.8% of the additive variance for HDL, LDL, and TG, respectively, of which nearly one-third can be attributed to the top 1% of the genome showing strongest evidence of association in Europeans and more than half can be attributed to the top 10% of the genome. The excess variance explained by the top regions is statistically significant in the sense that the 95% confidence intervals (with the standard error estimated by GCTA) do not include the expected values under the null model. In HAs, too, genomic regions showing association in Europeans account for disproportionate phenotypic variance, although the uncertainties associated with the estimates are greater because of the smaller sample size (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S5</a>). In contrast, when we divided the genomic regions according to a permuted list of p values, the proportion of variance increased roughly linearly as a proportion of the genome included; not a single point significantly deviated from the expectation under the null model in the POGA plot (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC3675231/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190198641296" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" src="/pmc/articles/PMC3675231/bin/gr3.jpg"></div></a><div id="largeobj_idm140190198641296" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3675231/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a target="object" href="/pmc/articles/PMC3675231/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC3675231/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p61">Overlap in Genetic Architecture in AAs</p><p id="__p62">Genomic regions are ranked by the SNP with the strongest association evidence in a European GWAS. Proportions of the phenotypic variance explained in AAs (<em>y</em> axis) by the top <em>x</em>% of the genome (<em>x</em> axis) are estimated with a mixed effect model. Red points indicates the 95% confidence interval excludes the expected value (magenta line) under the null model that <em>x</em>% randomly selected genome explains <em>h</em><sup>2</sup><em>x</em>% of phenotypic variance, where <em>h</em><sup>2</sup> is the variance explained by the entire genome. When the null cannot be excluded, the point estimates are drawn with gray points.</p><p id="__p63">(A) High-density lipoprotein (HDL) cholesterol.</p><p id="__p64">(B) Low-density lipoprotein (LDL) cholesterol.</p><p id="__p65">(C) Triglycerides (TG).</p></div></div></div></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Ancestry Association and Admixture Mapping in AAs</h3><p id="__p28" class="p p-first">We next investigated factors that underlie ethnic differences in lipid levels. Adjusting for the same set of covariates as used in the AA GWAS, genome-wide African ancestry was associated with increased HDL (β = 0.10, p&nbsp;= 4.26&nbsp;× 10<sup>−7</sup>) and decreased TG (β = −0.33, p &lt; 10<sup>−16</sup>) and was not significantly associated with LDL (p&nbsp;= 0.178). This pattern is consistent with a previous analysis of an independent AA cohort in the Family Blood Pressure Program.<sup><a href="#bib36" rid="bib36 bib37" class=" bibr popnode">36,37</a></sup> The phenotypic variance attributed to genome-wide ancestry is very low: less than 0.3% for HDL and less than 1% for TG. Comparing HAs and AAs in WHI, HAs have significantly lower HDL (β = −0.10, p&nbsp;&lt; 2&nbsp;× 10<sup>−16</sup>), lower LDL (β = −3.56, p = 5.08&nbsp;× 10<sup>−5</sup>), and higher TG (β = 0.37, p &lt; 2&nbsp;× 10<sup>−16</sup>).</p><p id="__p29">Admixture mapping is an effective approach for identifying loci with strong effects and disparate allele frequencies between ancestral populations. Because the recent admixing history induces strong correlation in admixture mapping test statistics, we adopted a genome-wide significance threshold of 7&nbsp;× 10<sup>−6</sup>.<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> In the genome-wide admixture scan, local African ancestry at 9q22 (p = 2.82&nbsp;× 10<sup>−7</sup>) and at 11q23 (p = 5.57&nbsp;× 10<sup>−7</sup>) was associated with increased HDL (<a href="/pmc/articles/PMC3675231/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>, <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S6</a>). Together, the ancestry at these two loci explains 0.6% of variation in HDL, and genome-wide ancestry was no longer significantly associated with the trait upon adjusting for local ancestry at the two loci. At 1p32, local African ancestry was associated with decreased LDL (p = 2.11&nbsp;× 10<sup>−6</sup>), explaining &lt;0.3% of phenotypic variation. Somewhat surprisingly, no locus reached the genome-wide significance level for TG, despite the strong correlation between the genome-wide African ancestry and TG.</p><p id="__p30" class="p p-last">Association between local ancestry and a trait can arise because the region harbors population-specific risk variants or because it harbors risk variants shared among populations that occur at disparate allele frequencies. Thus, regions showing local ancestry association are excellent candidates for identifying the genetic basis that underlies trait differences between ethnicities. Below we describe further fine-scale characterization of each admixture-mapping peak. These analyses aim to identify genes or variants that are associated with lipids and explain the local ancestry-trait association. We note a caveat that, because admixture generates extensive linkage disequilibrium, these markers may simply be in LD with the true causal variants.</p></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">Admixture Association with LDL at 1q32</h3><p id="__p31" class="p p-first-last">In the region of the 1q32 admixture signal for LDL (UCSC Genome Browser hg18: 54.8–55.5 Mb), genotype association analyses revealed three low-frequency nonsynonymous variants in <em>PCSK9</em> associated with decreased LDL (rs28362286 [p.Cys679Ter], rs28362263 [p.Ala443Thr], and rs28362261 [p.Asn425Ser]); the first two variants were associated with decreased LDL in another AA cohort,<sup><a href="#bib38" rid="bib38" class=" bibr popnode">38</a></sup> and the third was validated in CARe (meta-analysis p = 1.43&nbsp;× 10<sup>−10</sup>). Conditioning on the dosage or carrier status of these three variants substantially reduced the local ancestry association (p = 1.15&nbsp;× 10<sup>−2</sup>). Curiously, we note that a nonsynonymous variant in <em>PCSK9</em>, rs505151 (p.Gly670Glu), was associated with LDL, but the allelic effect was opposite to the local ancestry effect. The minor allele (G) at this SNP was associated with higher LDL (p&nbsp;= 1.09&nbsp;× 10<sup>−6</sup> in WHI, 4.31&nbsp;× 10<sup>−09</sup> in CARe, and 2.42&nbsp;× 10<sup>−12</sup> in Europeans<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup>); this allele occurs in YRI at a frequency of 0.336 but at a much lower frequency of 0.031 in Europeans. Indeed, a joint analysis that included local ancestry and rs505151 showed stronger LDL association with both variables (<a href="#app2" rid="app2" class=" sec">Table S2</a>).</p></div><div id="sec3.6" class="sec"><h3 id="sec3.6title">Admixture Association with HDL at 9q22</h3><p id="__p32" class="p p-first-last">Local African ancestry at 9q22 (UCSC Genome Browser hg18: 98.5–101.4 Mb) was associated with increased HDL. From a biological standpoint, the nearest gene with a plausible role in lipid metabolism is <em>CORO2A</em> (MIM <a href="/omim/602159" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602159</a>), which encodes coronin 2A. Coronin 2A (CORO2A) is a nuclear receptor corepressor (NCoR) exchange factor that interacts with cholesterol-sensing liver X receptor to derepress inflammatory genes in macrophages.<sup><a href="#bib39" rid="bib39 bib40" class=" bibr popnode">39,40</a></sup> The peak local ancestry association is 5 Mb away from <em>ABCA1</em> (MIM <a href="/omim/600046" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600046</a>), a known HDL locus identified in Europeans. The ancestry association signal vanished upon conditioning on three SNPs: rs751800 in <em>CORO2A</em>, rs1537960 in <em>GABBR2</em> (MIM <a href="/omim/607340" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607340</a>), and rs4149310 in <em>ABCA1</em> (<a href="#app2" rid="app2" class=" sec">Table S3</a>). At rs1537960 and rs751800, the HDL-decreasing alleles are essentially absent in YRI, whereas the HDL-decreasing allele at rs4149310 occurs in both CEU and YRI but at a much higher frequency in CEU. Intriguingly, previous transcriptomic studies have identified rs751800, a SNP in the 3′ UTR of <em>CORO2A</em>, as a <em>cis</em>-eQTL in monocytes;<sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup> based on data generated from the ENCODE project, this SNP is categorized as likely to affect transcription factor binding (category 1b in RegulomeDB v.1.0).<sup><a href="#bib42" rid="bib42" class=" bibr popnode">42</a></sup> On the other hand, rs751800 was only weakly associated with HDL in the CARe cohort (p = 0.05) and showed no association in Europeans.<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> We note that rs4149310 also shows suggestive association in European populations (p = 8.46&nbsp;× 10<sup>−6</sup>); in contrast, the <em>ABCA1</em> SNP with the strongest HDL association in Europeans, rs1883025, had similar frequencies in YRI and CEU and was not significantly associated with HDL in AAs (p = 0.215). Thus, rs1883025 cannot explain the HDL-ancestry association. In summary, the ancestry-HDL association on 9q22 can be attributed to variants in <em>CORO2A</em>, <em>GABBR2</em>, and <em>ABCA1</em>. Although <em>ABCA1</em> appears to influence HDL in both Europeans and AAs, the risk variants are either distinct in the two populations or have very different allele frequencies.</p></div><div id="sec3.7" class="sec sec-last"><h3 id="sec3.7title">Admixture Association with HDL at 11q23</h3><p id="__p33" class="p p-first-last">Local African ancestry at chromosomal region 11q23 (UCSC Genome Browser hg18: 118.6–122.1 Mb) was also associated with increased HDL. The region is flanked at either end by two HDL loci identified in Europeans, <em>UBASH3B</em> (MIM <a href="/omim/609201" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609201</a>) and the <em>APOA/APOC</em> gene cluster. At the <em>APOA/APOC</em> locus, 13 SNPs were associated with HDL (p &lt; 5&nbsp;× 10<sup>−8</sup>) in WHI AAs. The minor alleles were each associated with decreased HDL and are all very rare (&lt;1% in AAs). Collectively, the number of rare variants carried by WHI AA individuals was positively correlated with African ancestry (p = 0.02) and strongly correlated with HDL (p = 6.78&nbsp;× 10<sup>−14</sup>). We performed conditional analysis of the local-ancestry-HDL association, adjusting for either single markers, combined dosage across all markers, or a 0/1 variable indicating whether an individual harbors any of the rare variants, but none of these variables explained the ancestry peak (<a href="#app2" rid="app2" class=" sec">Table S4</a>). At <em>UBASH3B</em>, the most significant SNP in Europeans (rs7115089) and the SNP showing the strongest association in AAs (rs7107934 [p = 2.49&nbsp;× 10<sup>−7</sup>]), both occur at similar frequencies in YRI and CEU and thus cannot explain the local ancestry association. No other variants were found in the 2 Mb region around <em>UBASH3B</em> that could explain the HDL admixture association signal. Because the local ancestry association at 11q23 cannot be attributed to variants in <em>APOA/APOC</em> or <em>UBASH3B</em>, the region probably harbors unrecognized HDL-influencing loci. A plausible candidate is <em>ABCG4</em> (MIM <a href="/omim/607784" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607784</a>), which has been shown to promote cholesterol efflux to HDL-like particles.<sup><a href="#bib43" rid="bib43 bib44" class=" bibr popnode">43,44</a></sup> However, single marker association analysis in <em>ABCG4</em> did not identify significant genotype-HDL association.</p></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p34" class="p p-first">In this study, we have demonstrated a substantial overlap in genes that contribute to the variation in plasma lipid levels among human populations. At the same time, substantial allelic heterogeneity has been observed within the shared loci, which contributes to the ethnic variation in lipid levels.</p><div id="sec4.1" class="sec"><h3 id="sec4.1title">A Shared Genetic Basis of Lipid Phenotypes among Human Populations</h3><p id="__p35" class="p p-first-last">Previous studies have found that variants associated with lipid traits in Europeans can be “replicated” in non-European populations by using criteria of relaxed statistical stringency and consistent direction of genetic effects.<sup><a href="#bib3" rid="bib3 bib11" class=" bibr popnode">3,11</a></sup> By using several approaches, we demonstrate evidence for even greater shared genetic components beyond the known replicated loci. First, we find a strong enrichment of small p values, in both AAs and HAs, among SNPs that show suggestive evidence of association in Europeans (p&nbsp;&lt; 10<sup>−5</sup>) (<a href="/pmc/articles/PMC3675231/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>). With an increased sample size, the fraction of variants that can be replicated in non-European populations is likely to increase. Second, despite some well-established examples in which a genetic variant confers striking population-specific risk,<sup><a href="#bib45" rid="bib45 bib46" class=" bibr popnode">45,46</a></sup> the estimated genetic effects in AAs and HAs, at loci chosen based on the association evidence in Europeans, show strong correlations in both direction and magnitude (<a href="/pmc/articles/PMC3675231/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>). This is even more striking considering the moderate sample sizes of AAs and HAs; thus the interethnic correlation in SNP effects will probably be strengthened with larger sample size and reduced sampling errors. Finally, POGA plots demonstrate that the portion of the human genome that contributes importantly to lipid variation in Europeans also contributes substantially to phenotypic variation in non-Europeans: for the majority of lipid traits we examined in AAs and HAs, more than half of the additive phenotypic variance explained by the entire genome can be attributed to the 10% of the genome showing the strongest evidence of association in Europeans (<a href="/pmc/articles/PMC3675231/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figures 3</span></a> and <a href="#app2" rid="app2" class=" sec">S5</a>). This last approach is particularly useful because it defines an overlapping genetic architecture more broadly by taking into account the possibility that, although a locus influences the trait in multiple populations, the precise variants may not be identical. Furthermore, a variant discovered in a GWAS may simply be a tagging SNP for a functional variant; hence, even when a causal variant is shared between populations, association with the tagging SNP may not replicate across populations because of different LD patterns. Estimating the proportion of phenotypic variation that is explained by a part of the genome accommodates both true allelic heterogeneity and population-specific LD patterns and has an intuitive interpretation as the contribution by a set of candidate loci to phenotypic variance. Again, the degree of overlap in genetic basis represents a lower bound because of the limited sample sizes.</p></div><div id="sec4.2" class="sec"><h3 id="sec4.2title">Genetic Factors that Underlie Lipid Trait Differences between Populations</h3><p id="__p36" class="p p-first">Although the distribution of plasma-lipid concentration overlaps considerably between populations, these traits also vary between populations. Because shared ancestry can confound genetic and nongenetic risk factors, it is difficult to quantify the genetic and nongenetic contribution to population differences in traits such as lipid levels. However, genetic association and admixture mapping analyses identified a number of genetic factors that underlie the ethnic differences. Furthermore, these examples illustrate that both common and rare variants can contribute to population differences. As expected, rare variants that contribute to population differences tend to be population specific, exemplified by multiple African-specific variants in <em>PCSK9</em> (associated with LDL) and in <em>APOA/APOC</em> (associated with HDL). However, we also identified common variants that underlie population differences: the African-specific <em>CD36</em> variant, Tyr325Ter, has a frequency of 0.28 in YRI; the HDL-increasing allele at rs4149310 in <em>ABCA1</em> has a frequency of 0.75 and 0.14 in YRI and CEU, respectively, and explains a large proportion of the ancestry association at 9q22 with HDL (<a href="#app2" rid="app2" class=" sec">Table S2</a>). It is also interesting to note that <em>PCSK9</em>, <em>APOA/APOC</em>, and <em>ABCA1</em> all play a role in lipid genetics in both Europeans and AAs, yet the precise alleles within each locus differ between the populations. This supports a view that lipid loci are largely shared among populations, but the allelic structure within a locus has been shaped by population history and thus can exhibit considerable heterogeneity. The presence of allelic heterogeneity has also been demonstrated in a recent transethnic study of lipid trait via the Metabochip.<sup><a href="#bib47" rid="bib47" class=" bibr popnode">47</a></sup></p><p id="__p37" class="p p-last">Admixture mapping has successfully detected loci that contribute to phenotypic diversity between populations. In most admixture mapping examples, where the genetic factors that give rise to the ancestry association are known, the ancestry-phenotype association can be largely attributed to a single variant with disparate allele frequencies between populations. Examples include <em>SLC24A5</em> (MIM <a href="/omim/609802" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609802</a>) for skin pigmentation, <em>DARC</em> (MIM <a href="/omim/613665" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">613665</a>) null allele for white blood cells, and <em>APOL1</em> (MIM <a href="/omim/603743" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">603743</a>) for kidney diseases; the striking genetic differentiation at the implicated loci signifies selective sweep.<sup><a href="#bib48" rid="bib48 bib49 bib50" class=" bibr popnode">48–50</a></sup> In contrast, of the three loci showing ancestry-trait association for lipid levels (<em>1p32</em>, <em>9q22</em>, and <em>11q23</em>), none could be attributed to single common variants. Instead, two or more variants explain the ancestry association in an additive fashion in each region. Furthermore, in at least one instance (<em>PCSK9</em>), we find African-specific coding variants that are associated with either increased or decreased LDL. Our interpretation of these observations is that the evolutionary dynamics of lipid traits in humans are influenced by a combination of forces and are in contrast to the strong directional adaptation that diversifies, for example, skin pigmentation. Rather, the many low-frequency coding variants may have been subjected to weak purifying selection,<sup><a href="#bib51" rid="bib51" class=" bibr popnode">51</a></sup> whereas the more common variants may have been driven to disparate population allele frequencies by other mechanisms, including balancing selection and random drift.<sup><a href="#bib52" rid="bib52" class=" bibr popnode">52</a></sup></p></div><div id="sec4.3" class="sec sec-last"><h3 id="sec4.3title">Implication for Future GWASs</h3><p id="__p38" class="p p-first-last">Racial and ethnic minorities constitute a growing proportion of the US population and suffer disproportionately higher rates of CVD.<sup><a href="#bib53" rid="bib53" class=" bibr popnode">53</a></sup> The abundance of population-specific variants that underlie lipid traits highlights the importance of including individuals of diverse ethnic background in future GWASs. At the same time, the identification of a large fraction of ethnically shared trait loci suggests that further insights into the genetic mechanisms that underlie lipid traits can be gained by continued study of multiple ethnicities simultaneously. Based on published GWASs that focus on populations of European decent, a majority of complex traits and diseases are influenced by a large number of loci, each conferring a modest risk. Studies with sample size exceeding 100,000 are nonetheless underpowered, as indicated by the substantial fraction of “missing heritability.''<sup><a href="#bib54" rid="bib54" class=" bibr popnode">54</a></sup> Thus, the scarcity of cohorts that represent AAs, HAs, and other minority populations poses a challenge for understanding the genetic basis of complex traits in these populations. Take coronary heart disease (CAD) as an example: the CARDIoGRAM GWAS cohort numbered 82,000 Europeans,<sup><a href="#bib55" rid="bib55" class=" bibr popnode">55</a></sup> compared to the largest published AA cohort of 8,000.<sup><a href="#bib56" rid="bib56" class=" bibr popnode">56</a></sup> Moreover, an increasing proportion of the world’s populations do not fall into conventional ethnic categories (e.g. individuals with mixed African and South Asian ancestry). In contrast to studying each ethnic population separately, a multiethnic approach that integrates evidence across populations will probably be more efficient, both for gene discovery and for individual risk prediction. One analytic approach has been proposed and takes the form of a “transethnic meta-analysis;” it is desirable to extend this approach to accommodate allelic heterogeneity and differential LD patterns.<sup><a href="#bib57" rid="bib57 bib58" class=" bibr popnode">57,58</a></sup> The ability to harness information across multiple populations will provide a more complete portrait of the genetic bases that underlie complex traits in human species, which will in turn allow all people to benefit from the new paradigm of personalized medicine.</p></div></div><div id="idm140190198668896" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140190198668896title">Acknowledgments</h2><div class="sec"><p id="__p50">This study was supported in part by the National Institutes of&nbsp;Health grants GM073059 (to H.T.), K01 CA148958 (to T.T.), and R25 CA112355 (to T.J.H.). The Women’s Health Initiative&nbsp;(WHI) program is funded by the National Heart, Lung, and&nbsp;Blood&nbsp;Institute, National Institutes of Health, and the&nbsp;United&nbsp;States Department of Health and Human Services through contracts HHSN268201100046C, HHSN2682011 00001C, HHSN268201100002C, HHSN268201100003C, HHSN2 68201100004C, and HHSN271201100004C. The authors thank the WHI investigators and staff for their dedication and the study participants for making the program possible. A listing of WHI investigators can be found at <a href="https://cleo.whi.org/researchers/Documents%2520%2520Write%2520a%2520Paper/WHI%2520Investigator%2520Long%2520List.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://cleo.whi.org/researchers/Documents%20%20Write%20a%20Paper/WHI%20Investigator%20Long%20List.pdf</a>. We thank two anonymous reviewers for their helpful comments.</p></div></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Figures S1–S6 and Tables S1–S4:</strong><div class="sup-box half_rhythm" id="idm140190198691696"><a href="/pmc/articles/PMC3675231/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(286K, pdf)</sup></div></div><p></p></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p40" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u0010"><p id="__p41">1000 Genomes, <a href="http://browser.1000genomes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://browser.1000genomes.org</a></p></li><li id="u0015"><p id="__p42">GCTA, <a href="http://www.complextraitgenomics.com/software/gcta/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.complextraitgenomics.com/software/gcta/</a></p></li><li id="u0020"><p id="__p43">Human Genome Diversity Project, <a href="http://hagsc.org/hgdp/files.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hagsc.org/hgdp/files.html</a></p></li><li id="u0025"><p id="__p44">Imputation Reference Panel, <a href="http://www.sph.umich.edu/csg/yli/mach/download/1000G.2012-02-14.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/yli/mach/download/1000G.2012-02-14.html</a></p></li><li id="u0030"><p id="__p45">International HapMap Project, <a href="http://hapmap.ncbi.nlm.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hapmap.ncbi.nlm.nih.gov/</a></p></li><li id="u0035"><p id="__p46">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.omim.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org/</a></p></li><li id="u0040"><p id="__p47">RegulomeDB, <a href="http://RegulomeDB.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://RegulomeDB.org/</a></p></li><li id="u0045"><p id="__p48">SABER+, <a href="http://med.stanford.edu/tanglab/software/saber.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://med.stanford.edu/tanglab/software/saber.html</a></p></li><li id="u0050"><p id="__p49">UCSC Genome Browser, <a href="http://genome.ucsc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.ucsc.edu</a></p></li></ul><p></p></div><div id="idm140190199620160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140190199620160title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Namboodiri K.K., Kaplan E.B., Heuch I., Elston R.C., Green P.P., Rao D.C., Laskarzewski P., Glueck C.J., Rifkind B.M. The Collaborative Lipid Research Clinics Family Study: biological and cultural determinants of familial resemblance for plasma lipids and lipoproteins. <span><span class="ref-journal">Genet. Epidemiol. </span>1985;<span class="ref-vol">2</span>:227–254.</span> [<a href="/pubmed/4054601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=The+Collaborative+Lipid+Research+Clinics+Family+Study:+biological+and+cultural+determinants+of+familial+resemblance+for+plasma+lipids+and+lipoproteins&amp;author=K.K.+Namboodiri&amp;author=E.B.+Kaplan&amp;author=I.+Heuch&amp;author=R.C.+Elston&amp;author=P.P.+Green&amp;volume=2&amp;publication_year=1985&amp;pages=227-254&amp;pmid=4054601&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Kannel W.B., Dawber T.R., Kagan A., Revotskie N., Stokes J., 3rd Factors of risk in the development of coronary heart disease—six year follow-up experience. The Framingham Study. <span><span class="ref-journal">Ann. Intern. Med. </span>1961;<span class="ref-vol">55</span>:33–50.</span> [<a href="/pubmed/13751193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Intern.+Med.&amp;title=Factors+of+risk+in+the+development+of+coronary+heart+disease—six+year+follow-up+experience.+The+Framingham+Study&amp;author=W.B.+Kannel&amp;author=T.R.+Dawber&amp;author=A.+Kagan&amp;author=N.+Revotskie&amp;author=J.+Stokes&amp;volume=55&amp;publication_year=1961&amp;pages=33-50&amp;pmid=13751193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Dumitrescu L., Carty C.L., Taylor K., Schumacher F.R., Hindorff L.A., Ambite J.L., Anderson G., Best L.G., Brown-Gentry K., Bůžková P. Genetic determinants of lipid traits in diverse populations from the population architecture using genomics and epidemiology (PAGE) study. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002138.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128106/">PMC free article</a>]</span> [<a href="/pubmed/21738485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genetic+determinants+of+lipid+traits+in+diverse+populations+from+the+population+architecture+using+genomics+and+epidemiology+(PAGE)+study&amp;author=L.+Dumitrescu&amp;author=C.L.+Carty&amp;author=K.+Taylor&amp;author=F.R.+Schumacher&amp;author=L.A.+Hindorff&amp;volume=7&amp;publication_year=2011&amp;pages=e1002138&amp;pmid=21738485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">LaRosa J.C., Brown C.D. Cardiovascular risk factors in minorities. <span><span class="ref-journal">Am. J. Med. </span>2005;<span class="ref-vol">118</span>:1314–1322.</span> [<a href="/pubmed/16378771" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.&amp;title=Cardiovascular+risk+factors+in+minorities&amp;author=J.C.+LaRosa&amp;author=C.D.+Brown&amp;volume=118&amp;publication_year=2005&amp;pages=1314-1322&amp;pmid=16378771&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Metcalf P.A., Sharrett A.R., Folsom A.R., Duncan B.B., Patsch W., Hutchinson R.G., Szklo M., Davis C.E., Tyroler H.A. African American-white differences in lipids, lipoproteins, and apolipoproteins, by educational attainment, among middle-aged adults: the Atherosclerosis Risk in Communities Study. <span><span class="ref-journal">Am. J. Epidemiol. </span>1998;<span class="ref-vol">148</span>:750–760.</span> [<a href="/pubmed/9786230" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=African+American-white+differences+in+lipids,+lipoproteins,+and+apolipoproteins,+by+educational+attainment,+among+middle-aged+adults:+the+Atherosclerosis+Risk+in+Communities+Study&amp;author=P.A.+Metcalf&amp;author=A.R.+Sharrett&amp;author=A.R.+Folsom&amp;author=B.B.+Duncan&amp;author=W.+Patsch&amp;volume=148&amp;publication_year=1998&amp;pages=750-760&amp;pmid=9786230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Gartside P.S., Khoury P., Glueck C.J. Determinants of high-density lipoprotein cholesterol in blacks and whites: the second National Health and Nutrition Examination Survey. <span><span class="ref-journal">Am. Heart J. </span>1984;<span class="ref-vol">108</span>:641–653.</span> [<a href="/pubmed/6475735" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+Heart+J.&amp;title=Determinants+of+high-density+lipoprotein+cholesterol+in+blacks+and+whites:+the+second+National+Health+and+Nutrition+Examination+Survey&amp;author=P.S.+Gartside&amp;author=P.+Khoury&amp;author=C.J.+Glueck&amp;volume=108&amp;publication_year=1984&amp;pages=641-653&amp;pmid=6475735&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Wei M., Mitchell B.D., Haffner S.M., Stern M.P. Effects of cigarette smoking, diabetes, high cholesterol, and hypertension on all-cause mortality and cardiovascular disease mortality in Mexican Americans. The San Antonio Heart Study. <span><span class="ref-journal">Am. J. Epidemiol. </span>1996;<span class="ref-vol">144</span>:1058–1065.</span> [<a href="/pubmed/8942437" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Effects+of+cigarette+smoking,+diabetes,+high+cholesterol,+and+hypertension+on+all-cause+mortality+and+cardiovascular+disease+mortality+in+Mexican+Americans.+The+San+Antonio+Heart+Study&amp;author=M.+Wei&amp;author=B.D.+Mitchell&amp;author=S.M.+Haffner&amp;author=M.P.+Stern&amp;volume=144&amp;publication_year=1996&amp;pages=1058-1065&amp;pmid=8942437&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Beekman M., Heijmans B.T., Martin N.G., Pedersen N.L., Whitfield J.B., DeFaire U., van Baal G.C., Snieder H., Vogler G.P., Slagboom P.E., Boomsma D.I. Heritabilities of apolipoprotein and lipid levels in three countries. <span><span class="ref-journal">Twin Res. </span>2002;<span class="ref-vol">5</span>:87–97.</span> [<a href="/pubmed/11931686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res.&amp;title=Heritabilities+of+apolipoprotein+and+lipid+levels+in+three+countries&amp;author=M.+Beekman&amp;author=B.T.+Heijmans&amp;author=N.G.+Martin&amp;author=N.L.+Pedersen&amp;author=J.B.+Whitfield&amp;volume=5&amp;publication_year=2002&amp;pages=87-97&amp;pmid=11931686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">Weiss L.A., Pan L., Abney M., Ober C. The sex-specific genetic architecture of quantitative traits in humans. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:218–222.</span> [<a href="/pubmed/16429159" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=The+sex-specific+genetic+architecture+of+quantitative+traits+in+humans&amp;author=L.A.+Weiss&amp;author=L.+Pan&amp;author=M.+Abney&amp;author=C.+Ober&amp;volume=38&amp;publication_year=2006&amp;pages=218-222&amp;pmid=16429159&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Luo B.F., Du L., Li J.X., Pan B.Y., Xu J.M., Chen J., Yin X.Y., Ren Y., Zhang F. Heritability of metabolic syndrome traits among healthy younger adults: a population based study in China. <span><span class="ref-journal">J.&nbsp;Med. Genet. </span>2010;<span class="ref-vol">47</span>:415–420.</span> [<a href="/pubmed/19755428" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Med.+Genet.&amp;title=Heritability+of+metabolic+syndrome+traits+among+healthy+younger+adults:+a+population+based+study+in+China&amp;author=B.F.+Luo&amp;author=L.+Du&amp;author=J.X.+Li&amp;author=B.Y.+Pan&amp;author=J.M.+Xu&amp;volume=47&amp;publication_year=2010&amp;pages=415-420&amp;pmid=19755428&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Teslovich T.M., Musunuru K., Smith A.V., Edmondson A.C., Stylianou I.M., Koseki M., Pirruccello J.P., Ripatti S., Chasman D.I., Willer C.J. Biological, clinical and population relevance of 95 loci for blood lipids. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:707–713.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039276/">PMC free article</a>]</span> [<a href="/pubmed/20686565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological,+clinical+and+population+relevance+of+95+loci+for+blood+lipids&amp;author=T.M.+Teslovich&amp;author=K.+Musunuru&amp;author=A.V.+Smith&amp;author=A.C.+Edmondson&amp;author=I.M.+Stylianou&amp;volume=466&amp;publication_year=2010&amp;pages=707-713&amp;pmid=20686565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Elbers C.C., Guo Y., Tragante V., van Iperen E.P., Lanktree M.B., Castillo B.A., Chen F., Yanek L.R., Wojczynski M.K., Li Y.R. Gene-centric meta-analysis of lipid traits in African, East Asian and Hispanic populations. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e50198.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3517599/">PMC free article</a>]</span> [<a href="/pubmed/23236364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Gene-centric+meta-analysis+of+lipid+traits+in+African,+East+Asian+and+Hispanic+populations&amp;author=C.C.+Elbers&amp;author=Y.+Guo&amp;author=V.+Tragante&amp;author=E.P.+van+Iperen&amp;author=M.B.+Lanktree&amp;volume=7&amp;publication_year=2012&amp;pages=e50198&amp;pmid=23236364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Hays J., Hunt J.R., Hubbell F.A., Anderson G.L., Limacher M., Allen C., Rossouw J.E. The Women’s Health Initiative recruitment methods and results. <span><span class="ref-journal">Ann. Epidemiol. </span>2003;<span class="ref-vol">13</span>(9, Suppl):S18–S77.</span> [<a href="/pubmed/14575939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Epidemiol.&amp;title=The+Women’s+Health+Initiative+recruitment+methods+and+results&amp;author=J.+Hays&amp;author=J.R.+Hunt&amp;author=F.A.+Hubbell&amp;author=G.L.+Anderson&amp;author=M.+Limacher&amp;volume=13&amp;issue=9,+Suppl&amp;publication_year=2003&amp;pages=S18-S77&amp;pmid=14575939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">Thornton T., McPeek M.S. ROADTRIPS: case-control association testing with partially or completely unknown population and pedigree structure. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2010;<span class="ref-vol">86</span>:172–184.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820184/">PMC free article</a>]</span> [<a href="/pubmed/20137780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=ROADTRIPS:+case-control+association+testing+with+partially+or+completely+unknown+population+and+pedigree+structure&amp;author=T.+Thornton&amp;author=M.S.+McPeek&amp;volume=86&amp;publication_year=2010&amp;pages=172-184&amp;pmid=20137780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Reiner A.P., Beleza S., Franceschini N., Auer P.L., Robinson J.G., Kooperberg C., Peters U., Tang H. Genome-wide association and population genetic analysis of C-reactive protein in African American and Hispanic American women. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">91</span>:502–512.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3511984/">PMC free article</a>]</span> [<a href="/pubmed/22939635" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Genome-wide+association+and+population+genetic+analysis+of+C-reactive+protein+in+African+American+and+Hispanic+American+women&amp;author=A.P.+Reiner&amp;author=S.+Beleza&amp;author=N.+Franceschini&amp;author=P.L.+Auer&amp;author=J.G.+Robinson&amp;volume=91&amp;publication_year=2012&amp;pages=502-512&amp;pmid=22939635&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Carty C.L., Johnson N.A., Hutter C.M., Reiner A.P., Peters U., Tang H., Kooperberg C. Genome-wide association study of body height in African Americans: the Women’s Health Initiative SNP Health Association Resource (SHARe) <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:711–720.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3259012/">PMC free article</a>]</span> [<a href="/pubmed/22021425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+study+of+body+height+in+African+Americans:+the+Women’s+Health+Initiative+SNP+Health+Association+Resource+(SHARe)&amp;author=C.L.+Carty&amp;author=N.A.+Johnson&amp;author=C.M.+Hutter&amp;author=A.P.+Reiner&amp;author=U.+Peters&amp;volume=21&amp;publication_year=2012&amp;pages=711-720&amp;pmid=22021425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">Friedewald W.T., Levy R.I., Fredrickson D.S. Estimation of the concentration of low-density lipoprotein cholesterol in plasma, without use of the preparative ultracentrifuge. <span><span class="ref-journal">Clin. Chem. </span>1972;<span class="ref-vol">18</span>:499–502.</span> [<a href="/pubmed/4337382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Estimation+of+the+concentration+of+low-density+lipoprotein+cholesterol+in+plasma,+without+use+of+the+preparative+ultracentrifuge&amp;author=W.T.+Friedewald&amp;author=R.I.+Levy&amp;author=D.S.+Fredrickson&amp;volume=18&amp;publication_year=1972&amp;pages=499-502&amp;pmid=4337382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Li Y., Willer C.J., Ding J., Scheet P., Abecasis G.R. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y.+Li&amp;author=C.J.+Willer&amp;author=J.+Ding&amp;author=P.+Scheet&amp;author=G.R.+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G.R. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;author=C.+Fuchsberger&amp;author=M.+Stephens&amp;author=J.+Marchini&amp;author=G.R.+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Tang H., Peng J., Wang P., Risch N.J. Estimation of individual admixture: analytical and study design considerations. <span><span class="ref-journal">Genet. Epidemiol. </span>2005;<span class="ref-vol">28</span>:289–301.</span> [<a href="/pubmed/15712363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Estimation+of+individual+admixture:+analytical+and+study+design+considerations&amp;author=H.+Tang&amp;author=J.+Peng&amp;author=P.+Wang&amp;author=N.J.+Risch&amp;volume=28&amp;publication_year=2005&amp;pages=289-301&amp;pmid=15712363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Li J.Z., Absher D.M., Tang H., Southwick A.M., Casto A.M., Ramachandran S., Cann H.M., Barsh G.S., Feldman M., Cavalli-Sforza L.L., Myers R.M. Worldwide human relationships inferred from genome-wide patterns of variation. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">319</span>:1100–1104.</span> [<a href="/pubmed/18292342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Worldwide+human+relationships+inferred+from+genome-wide+patterns+of+variation&amp;author=J.Z.+Li&amp;author=D.M.+Absher&amp;author=H.+Tang&amp;author=A.M.+Southwick&amp;author=A.M.+Casto&amp;volume=319&amp;publication_year=2008&amp;pages=1100-1104&amp;pmid=18292342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Tang H., Coram M., Wang P., Zhu X., Risch N. Reconstructing genetic ancestry blocks in admixed individuals. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">79</span>:1–12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1474129/">PMC free article</a>]</span> [<a href="/pubmed/16773560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Reconstructing+genetic+ancestry+blocks+in+admixed+individuals&amp;author=H.+Tang&amp;author=M.+Coram&amp;author=P.+Wang&amp;author=X.+Zhu&amp;author=N.+Risch&amp;volume=79&amp;publication_year=2006&amp;pages=1-12&amp;pmid=16773560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Price A.L., Tandon A., Patterson N., Barnes K.C., Rafaels N., Ruczinski I., Beaty T.H., Mathias R., Reich D., Myers S. Sensitive detection of chromosomal segments of distinct ancestry in admixed populations. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000519.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689842/">PMC free article</a>]</span> [<a href="/pubmed/19543370" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Sensitive+detection+of+chromosomal+segments+of+distinct+ancestry+in+admixed+populations&amp;author=A.L.+Price&amp;author=A.+Tandon&amp;author=N.+Patterson&amp;author=K.C.+Barnes&amp;author=N.+Rafaels&amp;volume=5&amp;publication_year=2009&amp;pages=e1000519&amp;pmid=19543370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">Pe’er I., Yelensky R., Altshuler D., Daly M.J. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I.+Pe’er&amp;author=R.+Yelensky&amp;author=D.+Altshuler&amp;author=M.J.+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Zollner S., Pritchard J.K. Overcoming the winner’s curse: estimating penetrance parameters from case-control data. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:605–615.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852705/">PMC free article</a>]</span> [<a href="/pubmed/17357068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Overcoming+the+winner’s+curse:+estimating+penetrance+parameters+from+case-control+data&amp;author=S.+Zollner&amp;author=J.K.+Pritchard&amp;volume=80&amp;publication_year=2007&amp;pages=605-615&amp;pmid=17357068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">Yang J., Benyamin B., McEvoy B.P., Gordon S., Henders A.K., Nyholt D.R., Madden P.A., Heath A.C., Martin N.G., Montgomery G.W. Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J.+Yang&amp;author=B.+Benyamin&amp;author=B.P.+McEvoy&amp;author=S.+Gordon&amp;author=A.K.+Henders&amp;volume=42&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Yang J., Lee S.H., Goddard M.E., Visscher P.M. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref31">Thornton T., Tang H., Hoffmann T.J., Ochs-Balcom H.M., Caan B.J., Risch N. Estimating kinship in admixed populations. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">91</span>:122–138.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397261/">PMC free article</a>]</span> [<a href="/pubmed/22748210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Estimating+kinship+in+admixed+populations&amp;author=T.+Thornton&amp;author=H.+Tang&amp;author=T.J.+Hoffmann&amp;author=H.M.+Ochs-Balcom&amp;author=B.J.+Caan&amp;volume=91&amp;publication_year=2012&amp;pages=122-138&amp;pmid=22748210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref32">Tang H., Siegmund D.O., Johnson N.A., Romieu I., London S.J. Joint testing of genotype and ancestry association in admixed families. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:783–791.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3103820/">PMC free article</a>]</span> [<a href="/pubmed/21031451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Joint+testing+of+genotype+and+ancestry+association+in+admixed+families&amp;author=H.+Tang&amp;author=D.O.+Siegmund&amp;author=N.A.+Johnson&amp;author=I.+Romieu&amp;author=S.J.+London&amp;volume=34&amp;publication_year=2010&amp;pages=783-791&amp;pmid=21031451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref33">Love-Gregory L., Sherva R., Sun L., Wasson J., Schappe T., Doria A., Rao D.C., Hunt S.C., Klein S., Neuman R.J. Variants in the CD36 gene associate with the metabolic syndrome and high-density lipoprotein cholesterol. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:1695–1704.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2655228/">PMC free article</a>]</span> [<a href="/pubmed/18305138" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Variants+in+the+CD36+gene+associate+with+the+metabolic+syndrome+and+high-density+lipoprotein+cholesterol&amp;author=L.+Love-Gregory&amp;author=R.+Sherva&amp;author=L.+Sun&amp;author=J.+Wasson&amp;author=T.+Schappe&amp;volume=17&amp;publication_year=2008&amp;pages=1695-1704&amp;pmid=18305138&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref34">Love-Gregory L., Sherva R., Schappe T., Qi J.S., McCrea J., Klein S., Connelly M.A., Abumrad N.A. Common CD36 SNPs reduce protein expression and may contribute to a protective atherogenic profile. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:193–201.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3000679/">PMC free article</a>]</span> [<a href="/pubmed/20935172" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Common+CD36+SNPs+reduce+protein+expression+and+may+contribute+to+a+protective+atherogenic+profile&amp;author=L.+Love-Gregory&amp;author=R.+Sherva&amp;author=T.+Schappe&amp;author=J.S.+Qi&amp;author=J.+McCrea&amp;volume=20&amp;publication_year=2011&amp;pages=193-201&amp;pmid=20935172&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref35">Furuhashi M., Ura N., Nakata T., Shimamoto K. Insulin sensitivity and lipid metabolism in human CD36 deficiency. <span><span class="ref-journal">Diabetes Care. </span>2003;<span class="ref-vol">26</span>:471–474.</span> [<a href="/pubmed/12547883" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes+Care&amp;title=Insulin+sensitivity+and+lipid+metabolism+in+human+CD36+deficiency&amp;author=M.+Furuhashi&amp;author=N.+Ura&amp;author=T.+Nakata&amp;author=K.+Shimamoto&amp;volume=26&amp;publication_year=2003&amp;pages=471-474&amp;pmid=12547883&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref36">Basu A., Tang H., Lewis C.E., North K., Curb J.D., Quertermous T., Mosley T.H., Boerwinkle E., Zhu X., Risch N.J. Admixture mapping of quantitative trait loci for blood lipids in African-Americans. <span><span class="ref-journal">Hum. Mol. Genet. </span>2009;<span class="ref-vol">18</span>:2091–2098.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722229/">PMC free article</a>]</span> [<a href="/pubmed/19304782" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Admixture+mapping+of+quantitative+trait+loci+for+blood+lipids+in+African-Americans&amp;author=A.+Basu&amp;author=H.+Tang&amp;author=C.E.+Lewis&amp;author=K.+North&amp;author=J.D.+Curb&amp;volume=18&amp;publication_year=2009&amp;pages=2091-2098&amp;pmid=19304782&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation" id="sref37">Deo R.C., Reich D., Tandon A., Akylbekova E., Patterson N., Waliszewska A., Kathiresan S., Sarpong D., Taylor H.A., Jr., Wilson J.G. Genetic differences between the determinants of lipid profile phenotypes in African and European Americans: the Jackson Heart Study. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000342.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2613537/">PMC free article</a>]</span> [<a href="/pubmed/19148283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genetic+differences+between+the+determinants+of+lipid+profile+phenotypes+in+African+and+European+Americans:+the+Jackson+Heart+Study&amp;author=R.C.+Deo&amp;author=D.+Reich&amp;author=A.+Tandon&amp;author=E.+Akylbekova&amp;author=N.+Patterson&amp;volume=5&amp;publication_year=2009&amp;pages=e1000342&amp;pmid=19148283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation" id="sref38">Huang C.C., Fornage M., Lloyd-Jones D.M., Wei G.S., Boerwinkle E., Liu K. Longitudinal association of PCSK9 sequence variations with low-density lipoprotein cholesterol levels: the Coronary Artery Risk Development in Young Adults Study. <span><span class="ref-journal">Circ. Cardiovasc. Genet. </span>2009;<span class="ref-vol">2</span>:354–361.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2810147/">PMC free article</a>]</span> [<a href="/pubmed/20031607" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet.&amp;title=Longitudinal+association+of+PCSK9+sequence+variations+with+low-density+lipoprotein+cholesterol+levels:+the+Coronary+Artery+Risk+Development+in+Young+Adults+Study&amp;author=C.C.+Huang&amp;author=M.+Fornage&amp;author=D.M.+Lloyd-Jones&amp;author=G.S.+Wei&amp;author=E.+Boerwinkle&amp;volume=2&amp;publication_year=2009&amp;pages=354-361&amp;pmid=20031607&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation" id="sref39">Huang W., Ghisletti S., Saijo K., Gandhi M., Aouadi M., Tesz G.J., Zhang D.X., Yao J., Czech M.P., Goode B.L. Coronin 2A mediates actin-dependent de-repression of inflammatory response genes. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">470</span>:414–418.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3464905/">PMC free article</a>]</span> [<a href="/pubmed/21331046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Coronin+2A+mediates+actin-dependent+de-repression+of+inflammatory+response+genes&amp;author=W.+Huang&amp;author=S.+Ghisletti&amp;author=K.+Saijo&amp;author=M.+Gandhi&amp;author=M.+Aouadi&amp;volume=470&amp;publication_year=2011&amp;pages=414-418&amp;pmid=21331046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation" id="sref40">Zelcer N., Tontonoz P. Liver X receptors as integrators of metabolic and inflammatory signaling. <span><span class="ref-journal">J.&nbsp;Clin. Invest. </span>2006;<span class="ref-vol">116</span>:607–614.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1386115/">PMC free article</a>]</span> [<a href="/pubmed/16511593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Clin.+Invest.&amp;title=Liver+X+receptors+as+integrators+of+metabolic+and+inflammatory+signaling&amp;author=N.+Zelcer&amp;author=P.+Tontonoz&amp;volume=116&amp;publication_year=2006&amp;pages=607-614&amp;pmid=16511593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation" id="sref41">Zeller T., Wild P., Szymczak S., Rotival M., Schillert A., Castagne R., Maouche S., Germain M., Lackner K., Rossmann H. Genetics and beyond—the transcriptome of human monocytes and disease susceptibility. <span><span class="ref-journal">PLoS ONE. </span>2010;<span class="ref-vol">5</span>:e10693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872668/">PMC free article</a>]</span> [<a href="/pubmed/20502693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Genetics+and+beyond—the+transcriptome+of+human+monocytes+and+disease+susceptibility&amp;author=T.+Zeller&amp;author=P.+Wild&amp;author=S.+Szymczak&amp;author=M.+Rotival&amp;author=A.+Schillert&amp;volume=5&amp;publication_year=2010&amp;pages=e10693&amp;pmid=20502693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation" id="sref42">Boyle A.P., Hong E.L., Hariharan M., Cheng Y., Schaub M.A., Kasowski M., Karczewski K.J., Park J., Hitz B.C., Weng S. Annotation of functional variation in personal genomes using RegulomeDB. <span><span class="ref-journal">Genome Res. </span>2012;<span class="ref-vol">22</span>:1790–1797.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431494/">PMC free article</a>]</span> [<a href="/pubmed/22955989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Annotation+of+functional+variation+in+personal+genomes+using+RegulomeDB&amp;author=A.P.+Boyle&amp;author=E.L.+Hong&amp;author=M.+Hariharan&amp;author=Y.+Cheng&amp;author=M.A.+Schaub&amp;volume=22&amp;publication_year=2012&amp;pages=1790-1797&amp;pmid=22955989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation" id="sref43">Wang N., Lan D., Chen W., Matsuura F., Tall A.R. ATP-binding cassette transporters G1 and G4 mediate cellular cholesterol efflux to high-density lipoproteins. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2004;<span class="ref-vol">101</span>:9774–9779.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC470750/">PMC free article</a>]</span> [<a href="/pubmed/15210959" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=ATP-binding+cassette+transporters+G1+and+G4+mediate+cellular+cholesterol+efflux+to+high-density+lipoproteins&amp;author=N.+Wang&amp;author=D.+Lan&amp;author=W.+Chen&amp;author=F.+Matsuura&amp;author=A.R.+Tall&amp;volume=101&amp;publication_year=2004&amp;pages=9774-9779&amp;pmid=15210959&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation" id="sref44">Wang N., Yvan-Charvet L., Lütjohann D., Mulder M., Vanmierlo T., Kim T.W., Tall A.R. ATP-binding cassette transporters G1 and G4 mediate cholesterol and desmosterol efflux to HDL and regulate sterol accumulation in the brain. <span><span class="ref-journal">FASEB J. </span>2008;<span class="ref-vol">22</span>:1073–1082.</span> [<a href="/pubmed/18039927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J.&amp;title=ATP-binding+cassette+transporters+G1+and+G4+mediate+cholesterol+and+desmosterol+efflux+to+HDL+and+regulate+sterol+accumulation+in+the+brain&amp;author=N.+Wang&amp;author=L.+Yvan-Charvet&amp;author=D.+Lütjohann&amp;author=M.+Mulder&amp;author=T.+Vanmierlo&amp;volume=22&amp;publication_year=2008&amp;pages=1073-1082&amp;pmid=18039927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation" id="sref45">Farrer L.A., Cupples L.A., Haines J.L., Hyman B., Kukull W.A., Mayeux R., Myers R.H., Pericak-Vance M.A., Risch N., van Duijn C.M., APOE and Alzheimer Disease Meta Analysis Consortium  Effects of age, sex, and ethnicity on the association between apolipoprotein E genotype and Alzheimer disease. A meta-analysis. <span><span class="ref-journal">JAMA. </span>1997;<span class="ref-vol">278</span>:1349–1356.</span> [<a href="/pubmed/9343467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Effects+of+age,+sex,+and+ethnicity+on+the+association+between+apolipoprotein+E+genotype+and+Alzheimer+disease.+A+meta-analysis&amp;author=L.A.+Farrer&amp;author=L.A.+Cupples&amp;author=J.L.+Haines&amp;author=B.+Hyman&amp;author=W.A.+Kukull&amp;volume=278&amp;publication_year=1997&amp;pages=1349-1356&amp;pmid=9343467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation" id="sref46">Helgadottir A., Manolescu A., Helgason A., Thorleifsson G., Thorsteinsdottir U., Gudbjartsson D.F., Gretarsdottir S., Magnusson K.P., Gudmundsson G., Hicks A. A variant of the gene encoding leukotriene A4 hydrolase confers ethnicity-specific risk of myocardial infarction. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:68–74.</span> [<a href="/pubmed/16282974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+variant+of+the+gene+encoding+leukotriene+A4+hydrolase+confers+ethnicity-specific+risk+of+myocardial+infarction&amp;author=A.+Helgadottir&amp;author=A.+Manolescu&amp;author=A.+Helgason&amp;author=G.+Thorleifsson&amp;author=U.+Thorsteinsdottir&amp;volume=38&amp;publication_year=2006&amp;pages=68-74&amp;pmid=16282974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation" id="sref47">Wu Y., Waite L.L., Jackson A.U., Sheu W.H., Buyske S., Absher D., Arnett D.K., Boerwinkle E., Bonnycastle L.L., Carty C.L. Trans-ethnic fine-mapping of lipid loci identifies population-specific signals and allelic heterogeneity that increases the trait variance explained. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003379.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605054/">PMC free article</a>]</span> [<a href="/pubmed/23555291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Trans-ethnic+fine-mapping+of+lipid+loci+identifies+population-specific+signals+and+allelic+heterogeneity+that+increases+the+trait+variance+explained&amp;author=Y.+Wu&amp;author=L.L.+Waite&amp;author=A.U.+Jackson&amp;author=W.H.+Sheu&amp;author=S.+Buyske&amp;volume=9&amp;publication_year=2013&amp;pages=e1003379&amp;pmid=23555291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation" id="sref48">Reiner A.P., Lettre G., Nalls M.A., Ganesh S.K., Mathias R., Austin M.A., Dean E., Arepalli S., Britton A., Chen Z. Genome-wide association study of white blood cell count in 16,388 African Americans: the continental origins and genetic epidemiology network (COGENT) <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128101/">PMC free article</a>]</span> [<a href="/pubmed/21738479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+study+of+white+blood+cell+count+in+16,388+African+Americans:+the+continental+origins+and+genetic+epidemiology+network+(COGENT)&amp;author=A.P.+Reiner&amp;author=G.+Lettre&amp;author=M.A.+Nalls&amp;author=S.K.+Ganesh&amp;author=R.+Mathias&amp;volume=7&amp;publication_year=2011&amp;pages=e1002108&amp;pmid=21738479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation" id="sref49">Lamason R.L., Mohideen M.A., Mest J.R., Wong A.C., Norton H.L., Aros M.C., Jurynec M.J., Mao X., Humphreville V.R., Humbert J.E. SLC24A5, a putative cation exchanger, affects pigmentation in zebrafish and humans. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">310</span>:1782–1786.</span> [<a href="/pubmed/16357253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=SLC24A5,+a+putative+cation+exchanger,+affects+pigmentation+in+zebrafish+and+humans&amp;author=R.L.+Lamason&amp;author=M.A.+Mohideen&amp;author=J.R.+Mest&amp;author=A.C.+Wong&amp;author=H.L.+Norton&amp;volume=310&amp;publication_year=2005&amp;pages=1782-1786&amp;pmid=16357253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation" id="sref50">Genovese G., Friedman D.J., Ross M.D., Lecordier L., Uzureau P., Freedman B.I., Bowden D.W., Langefeld C.D., Oleksyk T.K., Uscinski Knob A.L. Association of trypanolytic ApoL1 variants with kidney disease in African Americans. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">329</span>:841–845.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2980843/">PMC free article</a>]</span> [<a href="/pubmed/20647424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Association+of+trypanolytic+ApoL1+variants+with+kidney+disease+in+African+Americans&amp;author=G.+Genovese&amp;author=D.J.+Friedman&amp;author=M.D.+Ross&amp;author=L.+Lecordier&amp;author=P.+Uzureau&amp;volume=329&amp;publication_year=2010&amp;pages=841-845&amp;pmid=20647424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation" id="sref51">Tennessen J.A., Bigham A.W., O’Connor T.D., Fu W., Kenny E.E., Gravel S., McGee S., Do R., Liu X., Jun G., Broad GO. Seattle GO. NHLBI Exome Sequencing Project  Evolution and functional impact of rare coding variation from deep sequencing of human exomes. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">337</span>:64–69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3708544/">PMC free article</a>]</span> [<a href="/pubmed/22604720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Evolution+and+functional+impact+of+rare+coding+variation+from+deep+sequencing+of+human+exomes&amp;author=J.A.+Tennessen&amp;author=A.W.+Bigham&amp;author=T.D.+O’Connor&amp;author=W.+Fu&amp;author=E.E.+Kenny&amp;volume=337&amp;publication_year=2012&amp;pages=64-69&amp;pmid=22604720&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation" id="sref52">Charlesworth B., Nordborg M., Charlesworth D. The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations. <span><span class="ref-journal">Genet. Res. </span>1997;<span class="ref-vol">70</span>:155–174.</span> [<a href="/pubmed/9449192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Res.&amp;title=The+effects+of+local+selection,+balanced+polymorphism+and+background+selection+on+equilibrium+patterns+of+genetic+diversity+in+subdivided+populations&amp;author=B.+Charlesworth&amp;author=M.+Nordborg&amp;author=D.+Charlesworth&amp;volume=70&amp;publication_year=1997&amp;pages=155-174&amp;pmid=9449192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation" id="sref53">Mensah G.A., Mokdad A.H., Ford E.S., Greenlund K.J., Croft J.B. State of disparities in cardiovascular health in the United States. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">111</span>:1233–1241.</span> [<a href="/pubmed/15769763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=State+of+disparities+in+cardiovascular+health+in+the+United+States&amp;author=G.A.+Mensah&amp;author=A.H.+Mokdad&amp;author=E.S.+Ford&amp;author=K.J.+Greenlund&amp;author=J.B.+Croft&amp;volume=111&amp;publication_year=2005&amp;pages=1233-1241&amp;pmid=15769763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation" id="sref54">Manolio T.A., Collins F.S., Cox N.J., Goldstein D.B., Hindorff L.A., Hunter D.J., McCarthy M.I., Ramos E.M., Cardon L.R., Chakravarti A. Finding the missing heritability of complex diseases. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>:747–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=T.A.+Manolio&amp;author=F.S.+Collins&amp;author=N.J.+Cox&amp;author=D.B.+Goldstein&amp;author=L.A.+Hindorff&amp;volume=461&amp;publication_year=2009&amp;pages=747-753&amp;pmid=19812666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib55">55. <span class="element-citation" id="sref55">Schunkert H., König I.R., Kathiresan S., Reilly M.P., Assimes T.L., Holm H., Preuss M., Stewart A.F., Barbalic M., Gieger C., Cardiogenics. CARDIoGRAM Consortium  Large-scale association analysis identifies 13 new susceptibility loci for coronary artery disease. <span><span class="ref-journal">Nat. Genet. </span>2011;<span class="ref-vol">43</span>:333–338.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3119261/">PMC free article</a>]</span> [<a href="/pubmed/21378990" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Large-scale+association+analysis+identifies+13+new+susceptibility+loci+for+coronary+artery+disease&amp;author=H.+Schunkert&amp;author=I.R.+König&amp;author=S.+Kathiresan&amp;author=M.P.+Reilly&amp;author=T.L.+Assimes&amp;volume=43&amp;publication_year=2011&amp;pages=333-338&amp;pmid=21378990&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib56">56. <span class="element-citation" id="sref56">Lettre G., Palmer C.D., Young T., Ejebe K.G., Allayee H., Benjamin E.J., Bennett F., Bowden D.W., Chakravarti A., Dreisbach A. Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1001300.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037413/">PMC free article</a>]</span> [<a href="/pubmed/21347282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+study+of+coronary+heart+disease+and+its+risk+factors+in+8,090+African+Americans:+the+NHLBI+CARe+Project&amp;author=G.+Lettre&amp;author=C.D.+Palmer&amp;author=T.+Young&amp;author=K.G.+Ejebe&amp;author=H.+Allayee&amp;volume=7&amp;publication_year=2011&amp;pages=e1001300&amp;pmid=21347282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib57">57. <span class="element-citation" id="sref57">Morris A.P. Transethnic meta-analysis of genomewide association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2011;<span class="ref-vol">35</span>:809–822.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460225/">PMC free article</a>]</span> [<a href="/pubmed/22125221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Transethnic+meta-analysis+of+genomewide+association+studies&amp;author=A.P.+Morris&amp;volume=35&amp;publication_year=2011&amp;pages=809-822&amp;pmid=22125221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib58">58. <span class="element-citation" id="sref58">Dastani Z., Hivert M.F., Timpson N., Perry J.R., Yuan X., Scott R.A., Henneman P., Heid I.M., Kizer J.R., Lyytikäinen L.P., DIAGRAM+ Consortium. MAGIC Consortium. GLGC Investigators. MuTHER Consortium. DIAGRAM Consortium. GIANT Consortium. Global B Pgen Consortium. Procardis Consortium. MAGIC investigators. GLGC Consortium  Novel loci for adiponectin levels and their influence on type 2 diabetes and metabolic traits: a multi-ethnic meta-analysis of 45,891 individuals. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002607.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3315470/">PMC free article</a>]</span> [<a href="/pubmed/22479202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Novel+loci+for+adiponectin+levels+and+their+influence+on+type+2+diabetes+and+metabolic+traits:+a+multi-ethnic+meta-analysis+of+45,891+individuals&amp;author=Z.+Dastani&amp;author=M.F.+Hivert&amp;author=N.+Timpson&amp;author=J.R.+Perry&amp;author=X.+Yuan&amp;volume=8&amp;publication_year=2012&amp;pages=e1002607&amp;pmid=22479202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140190164924336title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Material and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#idm140190198668896title">Acknowledgments</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#idm140190199620160title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3675231/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3675231/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3675231/pdf/main.pdf">PDF (339K)</a></li> | <li><a href="#" data-citationid="PMC3675231" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3675231%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3675231%2F&amp;text=Genome-wide%20Characterization%20of%20Shared%20and%20Distinct%20Genetic%20Components%20that%20Influence%20Blood%20Lipid%20Levels%20in%20Ethnically%20Diverse%20Human%20Populations" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3675231%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23726366" data-aiid="3675231" data-aid="3675231" data-iid="223736" data-domainid="203" data-domain="ajhg" data-accid="PMC3675231" data-md5="a9b2377ffef34b71dc9fee711b34a900">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input name="absid" id="absid" value="3675231" type="hidden"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input class="button_apply" name="addtocollection" value="1" type="submit"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-6" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-6" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/22939635" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association and population genetic analysis of C-reactive protein in African American and Hispanic American women.</a><span class="source">[Am J Hum Genet. 2012]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Reiner AP, Beleza S, Franceschini N, Auer PL, Robinson JG, Kooperberg C, Peters U, Tang H. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Am J Hum Genet. 2012 Sep 7; 91(3):502-12. Epub 2012 Aug 30.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/22994408" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Transferability and fine mapping of genome-wide associated loci for lipids in African Americans.</a><span class="source">[BMC Med Genet. 2012]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Adeyemo A, Bentley AR, Meilleur KG, Doumatey AP, Chen G, Zhou J, Shriner D, Huang H, Herbert A, Gerry NP, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">BMC Med Genet. 2012 Sep 21; 13:88. Epub 2012 Sep 21.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/29025430" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genetic architecture of lipid traits in the Hispanic community health study/study of Latinos.</a><span class="source">[Lipids Health Dis. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Graff M, Emery LS, Justice AE, Parra E, Below JE, Palmer ND, Gao C, Duan Q, Valladares-Salgado A, Cruz M, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Lipids Health Dis. 2017 Oct 12; 16(1):200. Epub 2017 Oct 12.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/27007718" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-Wide Studies of Type 2 Diabetes and Lipid Traits in Hispanics.</a><span class="source">[Curr Diab Rep. 2016]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Below JE, Parra EJ. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Curr Diab Rep. 2016 May; 16(5):41. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/29448166" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Insights from population-based analyses of plasma lipids across the allele frequency spectrum.</a><span class="source">[Curr Opin Genet Dev. 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-11" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Peloso GM, Natarajan P. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Curr Opin Genet Dev. 2018 Jun; 50:1-6. Epub 2018 Feb 13.</em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=23726366&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=23726366&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-12" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-12" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6817526/" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Harmonizing Genetic Ancestry and Self-identified Race/Ethnicity in Genome-wide Association Studies</a><span class="source">[American Journal of Human Gene...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Fang H, Hui Q, Lynch J, Honerlaw J, Assimes TL, Huang J, Vujkovic M, Damrauer SM, Pyarajan S, Gaziano JM, DuVall SL, O’Donnell CJ, Cho K, Chang KM, Wilson PW, Tsao PS, the VA Million Veteran ProgramGazianoJ. MichaelJMRamoniRachelRBreelingJimJChangKyong-MiKMHuangGrantGMuralidharSumitraSO’DonnellChristopher J.CJTsaoPhilip S.PSMuralidharSumitraSMoserJenniferJWhitbourneStacey B.SBBrewerJessica V.JVConcatoJohnJWarrenStuartSArgyresDean P.DPStephensBradyBBrophyMary T.MTHumphriesDonald E.DEDoNhanNShayanShahpoorSNguyenXuan-Mai T.XMPyarajanSaijuSChoKellyKHauserElizabethESunYanYZhaoHongyuHWilsonPeterPMcArdleRachelRDellitaliaLouisLHarleyJohnJWhittleJeffreyJBeckhamJeanJWellsJohnJGutierrezSalvadorSGibsonGretchenGKaminskyLaurenceLVillarealGerardoGKinlayScottSXuJunzheJHamnerMarkMHaddockKathlyn SueKSBhushanSujataSIruvantiPranPGodschalkMichaelMBallasZuhairZBufordMalcolmMMastoridesStephenSKleinJonJRatcliffeNoraNFlorezHermesHSwannAlanAMurdochMaureenMSriramPeruvembaPYehShing ShingSSWashburnRonaldRJhalaDarshanaDAguayoSamuelSCohenDavidDSharmaSatishSCallaghanJohnJOurslerKris AnnKAWhooleyMaryMAhujaSunilSGutierrezAmparoASchifmanRonaldRGrecoJenniferJRauchmanMichaelMServatiusRichardROehlertMaryMWallbomAgnesAFernandoRonaldRMorganTimothyTStapleyToddTShermanScottSAndersonGwenevereGSonelElifEBoykoEdwardEMeyerLaurenceLGuptaSamirSFayadJosephJHungAdrianaALichyJackJHurleyRobinRRobeyBrooksBStrikerRobertR, Sun YV, Tang H. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">American Journal of Human Genetics. 2019 Sep 26; 105(4): 763-772</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6659770/" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">CD 36: Focus on Epigenetic and Post-Transcriptional Regulation</a><span class="source">[Frontiers in Genetics. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Niculite CM, Enciu AM, Hinescu ME. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Frontiers in Genetics. 2019 Jul 19; 10: 680</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6656333/" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Relevance of Multi-Omics Studies in Cardiovascular Diseases</a><span class="source">[Frontiers in Cardiovascular Me...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Leon-Mimila P, Wang J, Huertas-Vazquez A. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Frontiers in Cardiovascular Medicine. 2019 Jul 17; 6: 91</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6535090/" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Identification of 22 novel loci associated with urinary biomarkers of albumin, sodium, and potassium excretion</a><span class="source">[Kidney international. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Zanetti D, Rao A, Gustafsson S, Assimes TL, Montgomery SB, Ingelsson E. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Kidney international. 2019 Mar 12; 95(5): 1197-1208</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6440667/" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Metabolic Syndrome Prediction Using Machine Learning Models with Genetic and Clinical Information from a Nonobese Healthy Population</a><span class="source"></span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-17" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Choe EK, Rhee H, Lee S, Shin E, Oh SW, Lee JE, Choi SH. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Genomics &amp; Informatics. 2018 Dec 28; 16(4): e31</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-18" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-18" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gene&amp;IdsFromResult=3675231" ref="reftype=Gene&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Gene&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Gene<div class="brieflinkpopdesc">Gene links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gene_nucleotide&amp;IdsFromResult=3675231" ref="reftype=Gene%20(nucleotide)&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene (nucleotide)</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Gene (nucleotide)<div class="brieflinkpopdesc">Records in Gene identified from shared sequence links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=geoprofiles&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_geoprofiles&amp;IdsFromResult=3675231" ref="reftype=GEO%20Profiles&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GEO Profiles</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">GEO Profiles<div class="brieflinkpopdesc">Related GEO records</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=homologene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_homologene&amp;IdsFromResult=3675231" ref="reftype=HomoloGene&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|HomoloGene&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">HomoloGene</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-22" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">HomoloGene<div class="brieflinkpopdesc">HomoloGene links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=3675231" ref="reftype=MedGen&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-23" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=nuccore&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_nuccore&amp;IdsFromResult=3675231" ref="reftype=Nucleotide&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Nucleotide&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Nucleotide</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-24" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Nucleotide<div class="brieflinkpopdesc">Published Nucleotide sequences</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/23726366/" ref="reftype=PubMed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-25" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=3675231" ref="reftype=Taxonomy&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-26" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-27" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-27" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5dd5625675f66d018f4483b0">Genome-wide Characterization of Shared and Distinct Genetic Components that Infl...</a><div class="ralinkpop offscreen_noflow">Genome-wide Characterization of Shared and Distinct Genetic Components that Influence Blood Lipid Levels in Ethnically Diverse Human Populations<div class="brieflinkpopdesc">American Journal of Human Genetics. 2013 Jun 6; 92(6)904</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=5dd5623275f66d018f3c63e9">GWAS of 126,559 Individuals Identifies Genetic Variants Associated with Educatio...</a><div class="ralinkpop offscreen_noflow">GWAS of 126,559 Individuals Identifies Genetic Variants Associated with Educational Attainment<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2013 Jun 21; 340(6139)1467</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=5dd56215dde0893a8b58840c">Genome-wide association study identifies loci affecting blood copper, selenium a...</a><div class="ralinkpop offscreen_noflow">Genome-wide association study identifies loci affecting blood copper, selenium and zinc<div class="brieflinkpopdesc">Human Molecular Genetics. 2013 Oct 1; 22(19)3998</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=5dd561f839a66eb03d2eace3">Genome-wide association study identifies a region on chromosome 11q14.3 associat...</a><div class="ralinkpop offscreen_noflow">Genome-wide association study identifies a region on chromosome 11q14.3 associated with late rectal bleeding following radiation therapy for prostate cancer<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2013 Jun; 107(3)372</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=5dd561bcdde0893a8b48d330">Genome-wide analysis of BMI in adolescents and young adults reveals additional i...</a><div class="ralinkpop offscreen_noflow">Genome-wide analysis of BMI in adolescents and young adults reveals additional insight into the effects of genetic loci over the life course<div class="brieflinkpopdesc">Human Molecular Genetics. 2013 Sep 1; 22(17)3597</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-bib1 bib2" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib3" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">3. <span class="element-citation" id="sref3">Dumitrescu L., Carty C.L., Taylor K., Schumacher F.R., Hindorff L.A., Ambite J.L., Anderson G., Best L.G., Brown-Gentry K., Bůžková P. Genetic determinants of lipid traits in diverse populations from the population architecture using genomics and epidemiology (PAGE) study. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002138.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128106/">PMC free article</a>]</span> [<a href="/pubmed/21738485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genetic+determinants+of+lipid+traits+in+diverse+populations+from+the+population+architecture+using+genomics+and+epidemiology+(PAGE)+study&amp;author=L.+Dumitrescu&amp;author=C.L.+Carty&amp;author=K.+Taylor&amp;author=F.R.+Schumacher&amp;author=L.A.+Hindorff&amp;volume=7&amp;publication_year=2011&amp;pages=e1002138&amp;pmid=21738485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib3">Ref list</a>]</div><div id="body-link-popper-bib4 bib5 bib6 bib7" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib8 bib9 bib10" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib11" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">11. <span class="element-citation" id="sref11">Teslovich T.M., Musunuru K., Smith A.V., Edmondson A.C., Stylianou I.M., Koseki M., Pirruccello J.P., Ripatti S., Chasman D.I., Willer C.J. Biological, clinical and population relevance of 95 loci for blood lipids. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:707–713.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039276/">PMC free article</a>]</span> [<a href="/pubmed/20686565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological,+clinical+and+population+relevance+of+95+loci+for+blood+lipids&amp;author=T.M.+Teslovich&amp;author=K.+Musunuru&amp;author=A.V.+Smith&amp;author=A.C.+Edmondson&amp;author=I.M.+Stylianou&amp;volume=466&amp;publication_year=2010&amp;pages=707-713&amp;pmid=20686565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib11">Ref list</a>]</div><div id="body-link-popper-bib3 bib11 bib12" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib13" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">13. <span class="element-citation" id="sref13">Hays J., Hunt J.R., Hubbell F.A., Anderson G.L., Limacher M., Allen C., Rossouw J.E. The Women’s Health Initiative recruitment methods and results. <span><span class="ref-journal">Ann. Epidemiol. </span>2003;<span class="ref-vol">13</span>(9, Suppl):S18–S77.</span> [<a href="/pubmed/14575939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Epidemiol.&amp;title=The+Women’s+Health+Initiative+recruitment+methods+and+results&amp;author=J.+Hays&amp;author=J.R.+Hunt&amp;author=F.A.+Hubbell&amp;author=G.L.+Anderson&amp;author=M.+Limacher&amp;volume=13&amp;issue=9,+Suppl&amp;publication_year=2003&amp;pages=S18-S77&amp;pmid=14575939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3675231&amp;issue-id=223736&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib13">Ref list</a>]</div><div id="body-link-popper-bib14 bib15" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib16 bib17" class="body-link-popper" style="display: none"></div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:57:10-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88D885DD5598010000000006DA0449&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3675231%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3675231/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Main, sans-serif;"></div></div></body></html>