<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3373247/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>C-reactive protein (CRP) promoter polymorphisms influence circulating CRP levels in a genome-wide association study of African Americans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3373247/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="C-reactive protein (CRP) promoter polymorphisms influence circulating CRP levels in a genome-wide association study of African Americans"><meta name="citation_authors" content="Ayo P. Doumatey, Guanjie Chen, Fasil Tekola Ayele, Jie Zhou, Michael Erdos, Daniel Shriner, Hanxia Huang, Jokotade Adeleye, Williams Balogun, Olufemi Fasanmade, Thomas Johnson, Johnnie Oli, Godfrey Okafor, Albert Amoah, Benjamin A. Eghan, Kofi Agyenim-Boateng, Joseph Acheampong, Clement Adebamowo, Norman P. Gerry, Michael F. Christman, Adebowale Adeyemo, Charles N. Rotimi"><meta name="citation_date" content="1 July 2012"><meta name="citation_issue" content="13"><meta name="citation_volume" content="21"><meta name="citation_firstpage" content="3063"><meta name="citation_doi" content="10.1093/hmg/dds133"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3373247/?report=abstract"><meta name="citation_pmid" content="22492993"><meta name="DC.Title" content="C-reactive protein (CRP) promoter polymorphisms influence circulating CRP levels in a genome-wide association study of African Americans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Ayo P. Doumatey"><meta name="DC.Contributor" content="Guanjie Chen"><meta name="DC.Contributor" content="Fasil Tekola Ayele"><meta name="DC.Contributor" content="Jie Zhou"><meta name="DC.Contributor" content="Michael Erdos"><meta name="DC.Contributor" content="Daniel Shriner"><meta name="DC.Contributor" content="Hanxia Huang"><meta name="DC.Contributor" content="Jokotade Adeleye"><meta name="DC.Contributor" content="Williams Balogun"><meta name="DC.Contributor" content="Olufemi Fasanmade"><meta name="DC.Contributor" content="Thomas Johnson"><meta name="DC.Contributor" content="Johnnie Oli"><meta name="DC.Contributor" content="Godfrey Okafor"><meta name="DC.Contributor" content="Albert Amoah"><meta name="DC.Contributor" content="Benjamin A. Eghan"><meta name="DC.Contributor" content="Kofi Agyenim-Boateng"><meta name="DC.Contributor" content="Joseph Acheampong"><meta name="DC.Contributor" content="Clement Adebamowo"><meta name="DC.Contributor" content="Norman P. Gerry"><meta name="DC.Contributor" content="Michael F. Christman"><meta name="DC.Contributor" content="Adebowale Adeyemo"><meta name="DC.Contributor" content="Charles N. Rotimi"><meta name="DC.Date" content="2012 Jul 1"><meta name="DC.Identifier" content="10.1093/hmg/dds133"><meta name="DC.Language" content="en"><meta property="og:title" content="C-reactive protein (CRP) promoter polymorphisms influence circulating CRP levels in a genome-wide association study of African Americans"><meta property="og:type" content="article"><meta property="og:description" content="C-reactive protein (CRP) is an acute phase reactant protein produced primarily by the liver. Circulating CRP levels are influenced by genetic and non-genetic factors, including infection and obesity. Genome-wide association studies (GWAS) provide an unbiased ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3373247/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3373247/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3373247/pdf/dds133.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE897A07DD537C010000000007A4054C.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3373247%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3373247</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2012 Jul 1; 21(13): 3063–3072. </span></div><div><span class="fm-vol-iss-date">Published online 2012 Apr 5. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fdds133" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/dds133</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3373247</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22492993">22492993</a></div></div></div></div><h1 class="content-title">C-reactive protein (<em>CRP</em>) promoter polymorphisms influence circulating CRP levels in a genome-wide association study of African Americans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Doumatey%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192504930224" co-class="co-affbox">Ayo P. Doumatey</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Chen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192510362304" co-class="co-affbox">Guanjie Chen</a>,<sup>1</sup> <a href="/pubmed/?term=Tekola%20Ayele%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192511267104" co-class="co-affbox">Fasil Tekola Ayele</a>,<sup>1</sup> <a href="/pubmed/?term=Zhou%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192507338528" co-class="co-affbox">Jie Zhou</a>,<sup>1</sup> <a href="/pubmed/?term=Erdos%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192507336544" co-class="co-affbox">Michael Erdos</a>,<sup>2</sup> <a href="/pubmed/?term=Shriner%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192507824944" co-class="co-affbox">Daniel Shriner</a>,<sup>1</sup> <a href="/pubmed/?term=Huang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192509025072" co-class="co-affbox">Hanxia Huang</a>,<sup>1</sup> <a href="/pubmed/?term=Adeleye%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192509023088" co-class="co-affbox">Jokotade Adeleye</a>,<sup>3</sup> <a href="/pubmed/?term=Balogun%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192546884000" co-class="co-affbox">Williams Balogun</a>,<sup>3</sup> <a href="/pubmed/?term=Fasanmade%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192546882016" co-class="co-affbox">Olufemi Fasanmade</a>,<sup>4</sup> <a href="/pubmed/?term=Johnson%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192582305360" co-class="co-affbox">Thomas Johnson</a>,<sup>4</sup> <a href="/pubmed/?term=Oli%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192509707296" co-class="co-affbox">Johnnie Oli</a>,<sup>5</sup> <a href="/pubmed/?term=Okafor%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192509705312" co-class="co-affbox">Godfrey Okafor</a>,<sup>5</sup> <a href="/pubmed/?term=Amoah%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192548132576" co-class="co-affbox">Albert Amoah</a>,<sup>6</sup> <a href="/pubmed/?term=Eghan%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192548130592" co-class="co-affbox">Benjamin A. Eghan</a>,<sup>7</sup> <a href="/pubmed/?term=Agyenim-Boateng%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192510358816" co-class="co-affbox">Kofi Agyenim-Boateng</a>,<sup>7</sup> <a href="/pubmed/?term=Acheampong%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192510356832" co-class="co-affbox">Joseph Acheampong</a>,<sup>7</sup> <a href="/pubmed/?term=Adebamowo%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192504642848" co-class="co-affbox">Clement Adebamowo</a>,<sup>3,</sup><sup>8</sup> <a href="/pubmed/?term=Gerry%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192584259488" co-class="co-affbox">Norman P. Gerry</a>,<sup>9</sup> <a href="/pubmed/?term=Christman%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192584257504" co-class="co-affbox">Michael F. Christman</a>,<sup>9</sup> <a href="/pubmed/?term=Adeyemo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192511676064" co-class="co-affbox">Adebowale Adeyemo</a>,<sup>1</sup> and  <a href="/pubmed/?term=Rotimi%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993" class="affpopup" co-rid="_co_idm140192545862448" co-class="co-affbox">Charles N. Rotimi</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140192504930224"><h3 class="no_margin">Ayo P. Doumatey</h3><p><sup>1</sup>Center for Research on Genomics and Global Health,  and</p><div>Find articles by <a href="/pubmed/?term=Doumatey%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Ayo P. Doumatey</a></div></div><div id="_co_idm140192510362304"><h3 class="no_margin">Guanjie Chen</h3><p><sup>1</sup>Center for Research on Genomics and Global Health,  and</p><div>Find articles by <a href="/pubmed/?term=Chen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Guanjie Chen</a></div></div><div id="_co_idm140192511267104"><h3 class="no_margin">Fasil Tekola Ayele</h3><p><sup>1</sup>Center for Research on Genomics and Global Health,  and</p><div>Find articles by <a href="/pubmed/?term=Tekola%20Ayele%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Fasil Tekola Ayele</a></div></div><div id="_co_idm140192507338528"><h3 class="no_margin">Jie Zhou</h3><p><sup>1</sup>Center for Research on Genomics and Global Health,  and</p><div>Find articles by <a href="/pubmed/?term=Zhou%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Jie Zhou</a></div></div><div id="_co_idm140192507336544"><h3 class="no_margin">Michael Erdos</h3><p><sup>2</sup>Genome Technology Branch, Molecular Genetics Section, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD, USA,</p><div>Find articles by <a href="/pubmed/?term=Erdos%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Michael Erdos</a></div></div><div id="_co_idm140192507824944"><h3 class="no_margin">Daniel Shriner</h3><p><sup>1</sup>Center for Research on Genomics and Global Health,  and</p><div>Find articles by <a href="/pubmed/?term=Shriner%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Daniel Shriner</a></div></div><div id="_co_idm140192509025072"><h3 class="no_margin">Hanxia Huang</h3><p><sup>1</sup>Center for Research on Genomics and Global Health,  and</p><div>Find articles by <a href="/pubmed/?term=Huang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Hanxia Huang</a></div></div><div id="_co_idm140192509023088"><h3 class="no_margin">Jokotade Adeleye</h3><p><sup>3</sup>University of Ibadan, Ibadan, Nigeria,</p><div>Find articles by <a href="/pubmed/?term=Adeleye%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Jokotade Adeleye</a></div></div><div id="_co_idm140192546884000"><h3 class="no_margin">Williams Balogun</h3><p><sup>3</sup>University of Ibadan, Ibadan, Nigeria,</p><div>Find articles by <a href="/pubmed/?term=Balogun%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Williams Balogun</a></div></div><div id="_co_idm140192546882016"><h3 class="no_margin">Olufemi Fasanmade</h3><p><sup>4</sup>University of Lagos, Lagos, Nigeria,</p><div>Find articles by <a href="/pubmed/?term=Fasanmade%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Olufemi Fasanmade</a></div></div><div id="_co_idm140192582305360"><h3 class="no_margin">Thomas Johnson</h3><p><sup>4</sup>University of Lagos, Lagos, Nigeria,</p><div>Find articles by <a href="/pubmed/?term=Johnson%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Thomas Johnson</a></div></div><div id="_co_idm140192509707296"><h3 class="no_margin">Johnnie Oli</h3><p><sup>5</sup>University of Nigeria Teaching Hospital, Enugu, Nigeria,</p><div>Find articles by <a href="/pubmed/?term=Oli%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Johnnie Oli</a></div></div><div id="_co_idm140192509705312"><h3 class="no_margin">Godfrey Okafor</h3><p><sup>5</sup>University of Nigeria Teaching Hospital, Enugu, Nigeria,</p><div>Find articles by <a href="/pubmed/?term=Okafor%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Godfrey Okafor</a></div></div><div id="_co_idm140192548132576"><h3 class="no_margin">Albert Amoah</h3><p><sup>6</sup>University of Ghana Medical School, Accra, Ghana,</p><div>Find articles by <a href="/pubmed/?term=Amoah%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Albert Amoah</a></div></div><div id="_co_idm140192548130592"><h3 class="no_margin">Benjamin A. Eghan</h3><p><sup>7</sup>University of Science and Technology, Kumasi, Ghana,</p><div>Find articles by <a href="/pubmed/?term=Eghan%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Benjamin A. Eghan</a></div></div><div id="_co_idm140192510358816"><h3 class="no_margin">Kofi Agyenim-Boateng</h3><p><sup>7</sup>University of Science and Technology, Kumasi, Ghana,</p><div>Find articles by <a href="/pubmed/?term=Agyenim-Boateng%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Kofi Agyenim-Boateng</a></div></div><div id="_co_idm140192510356832"><h3 class="no_margin">Joseph Acheampong</h3><p><sup>7</sup>University of Science and Technology, Kumasi, Ghana,</p><div>Find articles by <a href="/pubmed/?term=Acheampong%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Joseph Acheampong</a></div></div><div id="_co_idm140192504642848"><h3 class="no_margin">Clement Adebamowo</h3><p><sup>3</sup>University of Ibadan, Ibadan, Nigeria,</p><p><sup>8</sup>Institute of Human Virology, School of Medicine, University of Maryland, Baltimore, MD, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Adebamowo%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Clement Adebamowo</a></div></div><div id="_co_idm140192584259488"><h3 class="no_margin">Norman P. Gerry</h3><p><sup>9</sup>Coriell Institute for Medical Research, Camden, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Gerry%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Norman P. Gerry</a></div></div><div id="_co_idm140192584257504"><h3 class="no_margin">Michael F. Christman</h3><p><sup>9</sup>Coriell Institute for Medical Research, Camden, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Christman%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Michael F. Christman</a></div></div><div id="_co_idm140192511676064"><h3 class="no_margin">Adebowale Adeyemo</h3><p><sup>1</sup>Center for Research on Genomics and Global Health,  and</p><div>Find articles by <a href="/pubmed/?term=Adeyemo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Adebowale Adeyemo</a></div></div><div id="_co_idm140192545862448"><h3 class="no_margin">Charles N. Rotimi</h3><p><sup>1</sup>Center for Research on Genomics and Global Health,  and</p><div>Find articles by <a href="/pubmed/?term=Rotimi%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22492993">Charles N. Rotimi</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140192509785008_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140192509785008_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140192509785008_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140192509785008_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Center for Research on Genomics and Global Health,  and</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Genome Technology Branch, Molecular Genetics Section, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD, USA,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>University of Ibadan, Ibadan, Nigeria,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>University of Lagos, Lagos, Nigeria,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>University of Nigeria Teaching Hospital, Enugu, Nigeria,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>University of Ghana Medical School, Accra, Ghana,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>University of Science and Technology, Kumasi, Ghana,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Institute of Human Virology, School of Medicine, University of Maryland, Baltimore, MD, USA,  and</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Coriell Institute for Medical Research, Camden, NJ, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Center for Research on Genomics and Global Health, NHGRI/NIH, 12 South Dr. Bldg12A-4047, Bethesda, MD 20852, USA. Tel: +1 3014510195; Fax: +1 3014515426; Email: <a href="mailto:dev@null" data-email="vog.hin.liam@ayetamuod" class="oemail">vog.hin.liam@ayetamuod</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140192509785008_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Feb 1; Revised 2012 Mar 16; Accepted 2012 Apr 2.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140192509785008_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> Published by Oxford University Press 2012.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3373247/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3373247/bin/supp_21_13_3063__index.html" data-ga-action="click_feat_suppl">supp_21_13_3063__index.html</a><span style="color:gray"> (1.1K)</span></div><div class="small guid">GUID:&nbsp;EE1D1800-9059-4D1C-8484-1BF0301CD594</div>
<div><a href="/pmc/articles/PMC3373247/bin/supp_dds133_dds133supp.rtf" data-ga-action="click_feat_suppl">supp_dds133_dds133supp.rtf</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;31B32A00-46B4-4C6F-BED4-FCF12C482A9A</div>
</div></dd></dl></div><div id="idm140192509915552" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140192509915552title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">C-reactive protein (CRP) is an acute phase reactant protein produced primarily by the liver. Circulating CRP levels are influenced by genetic and non-genetic factors, including infection and obesity. Genome-wide association studies (GWAS) provide an unbiased approach towards identifying loci influencing CRP levels. None of the six GWAS for CRP levels has been conducted in an African ancestry population. The present study aims to: (i) identify genetic variants that influence serum CRP in African Americans (AA) using a genome-wide association approach and replicate these findings in West Africans (WA), (ii) assess transferability of major signals for CRP reported in European ancestry populations (EA) to AA and (iii) use the weak linkage disequilibrium (LD) structure characteristic of African ancestry populations to fine-map the previously reported <em>CRP</em> locus. The discovery cohort comprised 837 unrelated AA, with the replication of significant single-nucleotide polymorphisms (SNPs) assessed in 486 WA. The association analysis was conducted with 2 366 856 genotyped and imputed SNPs under an additive genetic model with adjustment for appropriate covariates. Genome-wide and replication significances were set at <em>P</em> &lt; 5 × 10<sup>−8</sup> and <em>P</em> &lt; 0.05, respectively. Ten SNPs in&nbsp;(<em>CRP pseudogene-1</em>) <em>CRPP1</em> and <em>CRP</em> genes were associated with serum CRP (<em>P</em> = 2.4 × 10<sup>−09</sup> to 4.3 × 10<sup>−11</sup>). All but one of the top-scoring SNPs associated with CRP in AA were successfully replicated in WA. <em>CRP</em> signals previously identified in EA samples were transferable to AAs, and we were able to fine-map this signal, reducing the region of interest from the 25 kb of LD around the locus in the HapMap CEU sample to only 8 kb in our AA sample.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">C-reactive protein (CRP), a protein of the pentaxin family, is produced mainly by the liver and is involved in several host defense mechanisms. Its levels are raised in the acute phase response to tissue injury, infection and other inflammatory stimuli (<a href="#DDS133C1" rid="DDS133C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939323">1</a>). In cell culture, its induction is mainly under the control of interleukin-6 (IL-6) (<a href="#DDS133C2" rid="DDS133C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939345">2</a>–<a href="#DDS133C4" rid="DDS133C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939348">4</a>). However, <em>in vivo</em> studies have shown that its induction is controlled by a complex network of cytokines, some of which are IL-6 independent (<a href="#DDS133C4" rid="DDS133C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939297">4</a>,<a href="#DDS133C5" rid="DDS133C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939317">5</a>). The role of CRP in chronic complex disease is gaining increasing recognition. For example, CRP has been shown to be elevated in obese, hypertensive and diabetic subjects (<a href="#DDS133C6" rid="DDS133C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360255303">6</a>,<a href="#DDS133C7" rid="DDS133C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939300">7</a>). Furthermore, high basal levels of serum CRP predict future cardiovascular disease (CVD) and metabolic abnormalities (<a href="#DDS133C4" rid="DDS133C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939329">4</a>,<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939336">8</a>,<a href="#DDS133C9" rid="DDS133C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298503604">9</a>). Consequently, it is used in clinical settings along with the lipids profile to evaluate the risk of developing CVD (<a href="#DDS133C4" rid="DDS133C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939305">4</a>,<a href="#DDS133C10" rid="DDS133C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_807141020">10</a>).</p><p id="__p3" class="p p-last">The gene encoding CRP is located on chromosome 1 (1q21–q23) and a number of studies have shown associations between polymorphisms at <em>CRP</em> locus and serum CRP levels (<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939299">8</a>,<a href="#DDS133C11" rid="DDS133C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939306">11</a>–<a href="#DDS133C13" rid="DDS133C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939315">13</a>). However, only a small fraction of the reported heritability, estimated at 30–50% (<a href="#DDS133C14" rid="DDS133C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939346">14</a>), is explained by these associations. To date, six genome-wide association studies (GWAS) of CRP have been published and they are all in populations of European ancestry (EA) or Asian ancestry (<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939343">8</a>,<a href="#DDS133C14" rid="DDS133C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939303">14</a>–<a href="#DDS133C18" rid="DDS133C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298503607">18</a>). All six studies confirmed previously reported associations between <em>CRP</em> polymorphisms and serum CRP levels while reporting the new loci: <em>APOE</em>, <em>HNF1A</em> (<em>hepatocyte nuclear factor 1 homeobox A</em>), <em>IL6R</em> (<em>interleukin-6 receptor</em>), <em>LEPR</em> (<em>leptin receptor</em>) and <em>IL6</em>. Recently, Dehghan <em>et al</em>. (<a href="#DDS133C16" rid="DDS133C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939295">16</a>) reported additional loci [such as <em>NLRP3</em> (<em>NLR family, pyrin domain containing 3</em>) and <em>IL1F10</em>] that may be involved in chronic inflammation. Thus far, there is no GWAS of CRP in a population of African ancestry. Notably, a number of studies have reported that CRP levels vary with ethnicity. African Americans (AA) have been shown to have the highest baseline CRP level when compared with other ethnic groups in the USA (<a href="#DDS133C6" rid="DDS133C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360255302">6</a>,<a href="#DDS133C19" rid="DDS133C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939316">19</a>). In view of the well-known differences between ancestral populations with respect to haplotype blocks, linkage disequilibrium (LD) patterns and allele frequencies (<a href="#DDS133C11" rid="DDS133C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939333">11</a>), conducting GWAS in multiple populations has been strongly advocated (<a href="#DDS133C20" rid="DDS133C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939290">20</a>). In the present study, we report the first GWAS for CRP in AA enrolled from the Washington DC metropolitan area and replicated the top-scoring single-nucleotide polymorphisms (SNPs) in an independent sample of 486 West Africans (WA). Second, we attempted to replicate previously reported significant GWAS findings. Finally, we use the weak LD structures characteristic of African ancestry populations to fine-map the reported <em>CRP</em> loci in EA.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p4" class="p p-first">The sample included 837 unrelated AA subjects (340 men and 497 women); 15.4% of the subjects had type 2 diabetes (T2D). The anthropometric and clinical characteristics of the study subjects are given in Table&nbsp;<a href="/pmc/articles/PMC3373247/table/DDS133TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS133TB1" rid-ob="ob-DDS133TB1" co-legend-rid=""><span>1</span></a>. Men and women were similar in age with a mean age of 47 years. Women were heavier (31.5 versus 29.2 kg/m<sup>2</sup>, <em>P</em> &lt; 0.0001) and had higher circulating high-sensitivity C-reactive protein (hsCRP) levels (0.63 versus 0.50 mg/dl, <em>P</em> = 0.15), but this was not statistically significant.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS133TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p5">Anthropometric and clinical characteristics of study subjects</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140192545178176" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Parameters</th><th rowspan="1" colspan="1" align="left">Male</th><th rowspan="1" colspan="1" align="left">Female</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>n</em></td><td rowspan="1" colspan="1">340</td><td rowspan="1" colspan="1">497</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1">47.7 ± 13</td><td rowspan="1" colspan="1">48.6 ± 14.6</td><td rowspan="1" colspan="1">0.37</td></tr><tr><td rowspan="1" colspan="1">BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1">29.2 ± 7.7</td><td rowspan="1" colspan="1">31.5 ± 9.1</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1">Type 2 diabetic (%)</td><td rowspan="1" colspan="1">12.9</td><td rowspan="1" colspan="1">17.1</td><td rowspan="1" colspan="1">0.1</td></tr><tr><td rowspan="1" colspan="1">Smoking (%)</td><td rowspan="1" colspan="1">58.7</td><td rowspan="1" colspan="1">41.7</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1">Hypertensive (%)</td><td rowspan="1" colspan="1">49.7</td><td rowspan="1" colspan="1">47.5</td><td rowspan="1" colspan="1">0.56</td></tr><tr><td rowspan="1" colspan="1">SBP (mmHg)</td><td rowspan="1" colspan="1">131 ± 21.7</td><td rowspan="1" colspan="1">128.5 ± 22.4</td><td rowspan="1" colspan="1">0.12</td></tr><tr><td rowspan="1" colspan="1">DBP (mmHg)</td><td rowspan="1" colspan="1">82.8 ± 14.1</td><td rowspan="1" colspan="1">79 ± 12.1</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1">Cholesterol (mg/dl)</td><td rowspan="1" colspan="1">191.2 ± 42</td><td rowspan="1" colspan="1">199 ± 42.8</td><td rowspan="1" colspan="1">0.012</td></tr><tr><td rowspan="1" colspan="1">HDL cholesterol (mg/dl)</td><td rowspan="1" colspan="1">50.1 ± 16.1</td><td rowspan="1" colspan="1">53.8 ± 16.8</td><td rowspan="1" colspan="1">0.0014</td></tr><tr><td rowspan="1" colspan="1">LDL cholesterol (mg/dl)</td><td rowspan="1" colspan="1">115.7 ± 38.3</td><td rowspan="1" colspan="1">120.6 ± 39</td><td rowspan="1" colspan="1">0.073</td></tr><tr><td rowspan="1" colspan="1">Triglycerides (mg/dl)</td><td rowspan="1" colspan="1">107.4 ± 59.6</td><td rowspan="1" colspan="1">105.7 ± 85.2</td><td rowspan="1" colspan="1">0.74</td></tr><tr><td colspan="4" rowspan="1">HsCRP</td></tr><tr><td rowspan="1" colspan="1"> Mean (mg/dl)</td><td rowspan="1" colspan="1">0.50 ± 1.47</td><td rowspan="1" colspan="1">0.63 ± 1.0</td><td rowspan="1" colspan="1">0.15</td></tr><tr><td rowspan="1" colspan="1"> Geometric mean (mg/dl)</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm140192545178176" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3373247/table/DDS133TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140192508897136"><p id="__p6" class="p p-first-last">Values are mean ± SD except for hypertension, type 2 diabetes and smoking which are frequencies. Mean values were compared by Student's <em>t</em>-test and the frequencies by the χ<sup>2</sup> test.</p></div></div></div><p id="__p7"><em>P</em>-values for the associations of each SNP with serum hsCRP levels according to chromosome number and position are shown on the Manhattan plot (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). The top 50 SNPs associated with serum hsCRP adjusting for age, sex, body mass index (BMI), T2D and the first two principal components (PCs) of genotypes are presented in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>. Additional adjustment for hypertension and smoking status (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>) did not change the association results.</p><p id="__p8">In this sample of AA, 10 SNPs were associated with hsCRP at a genome-wide level of significance (<em>P</em> &lt; 5 × 10<sup>–8</sup>) (Table&nbsp;<a href="/pmc/articles/PMC3373247/table/DDS133TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS133TB2" rid-ob="ob-DDS133TB2" co-legend-rid=""><span>2</span></a>, Fig.&nbsp;<a href="/pmc/articles/PMC3373247/figure/DDS133F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDS133F1" rid-ob="ob-DDS133F1" co-legend-rid="lgnd_DDS133F1"><span>1</span></a>). All 10 SNPs were clustered in and near the <em>CRP</em> locus. Two of these SNPs (rs3093058, <em>P</em> = 7.6 × 10<sup>–10</sup> and rs9628671, <em>P</em> = 1.3 × 10<sup>–10</sup>) are upstream of the <em>CRP</em> locus in the promoter region; the other eight SNPs, including the strongest association signal in this study, were located downstream from the <em>CRP</em> locus in the <em>CRP pseudogene-1</em> (<em>CRPP1</em>) (Fig.&nbsp;<a href="/pmc/articles/PMC3373247/figure/DDS133F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDS133F1" rid-ob="ob-DDS133F1" co-legend-rid="lgnd_DDS133F1"><span>1</span></a>). Three other SNPs (rs2027471, rs1341665, rs7553007) at the <em>CRP</em> locus resulted in <em>P</em>-values&nbsp;&lt;10<sup>–4</sup> (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). To investigate whether the association signals can be solely explained by the leading SNP (rs3093058) in the <em>CRP</em> locus, we conditioned on this SNP and repeated the association analyses. There was a significant drop in the strength of association of the other <em>CRPP1</em>/<em>CRP</em> SNPs, suggesting that the leading SNP is driving most of the association. For example, after adjusting for the lead SNP, the most significant <em>P</em>-value in the <em>CRPP1/CRP</em> region was 0.02 (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). We noted a number of SNPs with suggestive association (<em>P</em>-value ∼10<sup>–6</sup>) with hsCRP levels in this sample (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). These SNPs are clustered in three distinct chromosomal regions: seven on chromosome 2 in <em>low-density lipoprotein receptor-related protein 1B</em> (<em>LRP1B</em>; Gene ID 53353); six on chromosome 5 near <em>mannosidase-alpha class 2 A member 1</em> (<em>MAN2A1</em>; Gene ID 4124) and five on chromosome 16 near (<em>excision repair cross-complementing rodent repair deficiency, complementation group</em><em>-4</em>) <em>ERCC4</em> (Gene ID 2072).

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS133TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p9">SNPs associated with ln(hsCRP) at a genome-wide significance level in African Americans</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140192511199088" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chromosome</th><th rowspan="1" colspan="1" align="left">Position<sup>a</sup></th><th rowspan="1" colspan="1" align="left">Type</th><th rowspan="1" colspan="1" align="left">CA<sup>b</sup> (CAF)<sup>c</sup></th><th rowspan="1" colspan="1" align="left">Closest gene</th><th rowspan="1" colspan="1" align="left">Distance to gene (bp)</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P-</em>value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs16827466<sup>d</sup></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159649700</td><td rowspan="1" colspan="1">INTERGENIC</td><td rowspan="1" colspan="1">T (0.19)</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">25 073</td><td rowspan="1" colspan="1">0.47 (0.07)</td><td rowspan="1" colspan="1">4.3 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs726640</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159655518</td><td rowspan="1" colspan="1">INTERGENIC</td><td rowspan="1" colspan="1">A (0.17)</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">19 255</td><td rowspan="1" colspan="1">0.50 (0.08)</td><td rowspan="1" colspan="1">5.0 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs7531832</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159654216</td><td rowspan="1" colspan="1">INTERGENIC</td><td rowspan="1" colspan="1">G (0.17)</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">20 557</td><td rowspan="1" colspan="1">0.50 (0.08)</td><td rowspan="1" colspan="1">5.9 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs16842525</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159664624</td><td rowspan="1" colspan="1">INTERGENIC</td><td rowspan="1" colspan="1">C (0.19)</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">10 149</td><td rowspan="1" colspan="1">0.45 (0.07)</td><td rowspan="1" colspan="1">6.8 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">rs1341666</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159667059</td><td rowspan="1" colspan="1">INTERGENIC</td><td rowspan="1" colspan="1">G (0.19)</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">7714</td><td rowspan="1" colspan="1">0.44 (0.07)</td><td rowspan="1" colspan="1">6.8 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">rs3093058</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159685315</td><td rowspan="1" colspan="1">UPSTREAM</td><td rowspan="1" colspan="1">A (0.21)</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">−936</td><td rowspan="1" colspan="1">0.43 (0.07)</td><td rowspan="1" colspan="1">7.6 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">rs6667499</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159677654</td><td rowspan="1" colspan="1">UPSTREAM</td><td rowspan="1" colspan="1">A (0.21)</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">−2257</td><td rowspan="1" colspan="1">0.42 (0.07)</td><td rowspan="1" colspan="1">9.4 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">rs9628671</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159687972</td><td rowspan="1" colspan="1">UPSTREAM</td><td rowspan="1" colspan="1">T (0.21)</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">−3593</td><td rowspan="1" colspan="1">0.42 (0.07)</td><td rowspan="1" colspan="1">1.3 × 10<sup>−09</sup></td></tr><tr><td rowspan="1" colspan="1">rs16842520</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159657850</td><td rowspan="1" colspan="1">INTERGENIC</td><td rowspan="1" colspan="1">G (0.18)</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">16 923</td><td rowspan="1" colspan="1">0.45 (0.07)</td><td rowspan="1" colspan="1">1.8 × 10<sup>−09</sup></td></tr><tr><td rowspan="1" colspan="1">rs12239267</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159670928</td><td rowspan="1" colspan="1">DOWNSTREAM</td><td rowspan="1" colspan="1">T (0.18)</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">3845</td><td rowspan="1" colspan="1">0.44 (0.07)</td><td rowspan="1" colspan="1">2.4 × 10<sup>−09</sup></td></tr></tbody></table></div><div id="largeobj_idm140192511199088" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3373247/table/DDS133TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140192548798608"><p id="__p10" class="p p-first-last">CRPP1, CRP pseudogene1.</p></div><div id="idm140192548797984"><p id="__p11" class="p p-first-last"><sup>a</sup>Position of the SNPs was derived from dbSNP genome build 37.</p></div><div id="idm140192548797104"><p id="__p12" class="p p-first-last"><sup>b</sup>CA, coded allele.</p></div><div id="idm140192548796224"><p id="__p13" class="p p-first-last"><sup>c</sup>CAF, coded allele frequency.</p></div><div id="idm140192548795344"><p id="__p14" class="p p-first-last"><sup>d</sup>rs16827466 has been merged into rs10494326.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDS133F1" co-legend-rid="lgnd_DDS133F1"><a href="/pmc/articles/PMC3373247/figure/DDS133F1/" target="figure" rid-figpopup="DDS133F1" rid-ob="ob-DDS133F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140192510856368" class="figure"><a href="/pmc/articles/PMC3373247/figure/DDS133F1/" target="figure" rid-figpopup="DDS133F1" rid-ob="ob-DDS133F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3373247_dds13301.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is dds13301.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3373247/bin/dds13301.jpg"></a></div><div id="largeobj_idm140192510856368" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3373247/figure/DDS133F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDS133F1"><div><a class="figpopup" href="/pmc/articles/PMC3373247/figure/DDS133F1/" target="figure" rid-figpopup="DDS133F1" rid-ob="ob-DDS133F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p15">Regional plots of top-scoring SNPs identified in the GWAS and linkage disequilibrium (LD) plot in African Americans. (<strong>A</strong>) SNPs are plotted by chromosomal position (in megabases, according to NCBI build 36, dbSNP build 126) on the <em>x</em>-axis against association (−10log<sub>10</sub>
<em>P</em>-value) with hsCRP on the <em>y</em>-axis. Light blue lines show recombination rates taken from HapMap Phase II data to reflect local LD structure around the associated SNP. The lower panel shows gene annotations taken from the UCSC genome browser. The purple diamond represents one of the SNP that shows the strongest association in the plotted region. The LD of other SNPs and the best ranking SNP in the plotted region is shown in a scale from minimum (blue) to maximum (red). (<strong>B</strong>) The LD plot at the <em>CRP/CRPP1</em> loci in African Americans.</p></div></div></div><p id="__p16">The characteristics of 486 unrelated subjects in the WA replication sample are summarized in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>. Overall, the mean age in this cohort was 49 years and, like the discovery sample, women were heavier (by ∼3.6 BMI units, <em>P</em> &lt; 0.0001) and had slightly higher circulating CRP than men but this was not statistically significant (<em>P</em> = 0.45). Considering the 10 SNPs significantly and 1 SNP marginally associated with CRP in the discovery sample, 10 passed our quality control filters and were included in this analysis. The results of the replication conducted under an additive model with adjustment for age, gender, BMI and T2D are shown in Table&nbsp;<a href="/pmc/articles/PMC3373247/table/DDS133TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS133TB3" rid-ob="ob-DDS133TB3" co-legend-rid=""><span>3</span></a>. Of the 10 SNPs that were successfully genotyped in this cohort, nine were significantly associated with CRP at a <em>P</em>-value of &lt;0.05 among WA. The combined analysis (meta-analysis) showed that all nine SNPs were significantly associated with CRP and with the same direction of effect in both samples. The test of heterogeneity showed some heterogeneity between the individual data sets, especially marked for rs16827466 that showed no association in the replication data set.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS133TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p17">Replication of selected SNPs in a cohort of West Africans and meta-analysis results in the discovery and replication data sets</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140192504167840" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Chromosome</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Genes</th><th colspan="3" rowspan="1" align="left">Africa America Diabetes Mellitus study (<em>n</em>=486)<hr></th><th colspan="3" rowspan="1" align="left">Howard University Family Study (<em>n</em>=837)<hr></th><th colspan="6" rowspan="1" align="left">Meta-analysis (<em>n</em>=1323)<hr></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">CA<sup>a</sup> (CAF)<sup>b</sup></th><th rowspan="1" colspan="1" align="left"><em>P-</em>value</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left">CA<sup>a</sup> (CAF)<sup>b</sup></th><th rowspan="1" colspan="1" align="left"><em>P-</em>value</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P-</em>value</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left">Dir.</th><th rowspan="1" colspan="1" align="left">P(Het)*</th><th rowspan="1" colspan="1" align="left"><em>I</em><sup>2</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value**</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs16827466<sup>c</sup></td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">T (0.48)</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">0.05 (0.07)</td><td rowspan="1" colspan="1">T (0.19)</td><td rowspan="1" colspan="1">4.3 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.47 (0.07)</td><td rowspan="1" colspan="1">9.6 × 10<sup>−08</sup></td><td rowspan="1" colspan="1">0.27 (0.05)</td><td rowspan="1" colspan="1">++</td><td rowspan="1" colspan="1">4.3 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">94</td><td rowspan="1" colspan="1">1.0 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs7531832</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">G (0.22)</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">0.27 (0.09)</td><td rowspan="1" colspan="1">G (0.17)</td><td rowspan="1" colspan="1">5.9 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.50 (0.08)</td><td rowspan="1" colspan="1">2.8 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">−0.40 (0.06)</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">5.9 × 10<sup>−02</sup></td><td rowspan="1" colspan="1">72.1</td><td rowspan="1" colspan="1">0.37</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs726640</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">A (0.21)</td><td rowspan="1" colspan="1">0.0008</td><td rowspan="1" colspan="1">0.34 (0.10)</td><td rowspan="1" colspan="1">A (0.17)</td><td rowspan="1" colspan="1">5.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.50 (0.08)</td><td rowspan="1" colspan="1">1.8 × 10<sup>−13</sup></td><td rowspan="1" colspan="1">0.44 (0.06)</td><td rowspan="1" colspan="1">++</td><td rowspan="1" colspan="1">2.1 × 10<sup>−01</sup></td><td rowspan="1" colspan="1">36.3</td><td rowspan="1" colspan="1">0.47</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs16842525</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">C (0.20)</td><td rowspan="1" colspan="1">0.002</td><td rowspan="1" colspan="1">0.31 (0.10)</td><td rowspan="1" colspan="1">C (0.19)</td><td rowspan="1" colspan="1">6.8 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.44 (0.07)</td><td rowspan="1" colspan="1">5.9 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">−0.40 (0.06)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">2.9 × 10<sup>−01</sup></td><td rowspan="1" colspan="1">11.7</td><td rowspan="1" colspan="1">0.86</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs1341666</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">G (0.21)</td><td rowspan="1" colspan="1">0.002</td><td rowspan="1" colspan="1">0.31 (0.10)</td><td rowspan="1" colspan="1">G (0.19)</td><td rowspan="1" colspan="1">6.8 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.44 (0.07)</td><td rowspan="1" colspan="1">7.6 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">-0.40 (0.06)</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">2.6 × 10<sup>−01</sup></td><td rowspan="1" colspan="1">20.9</td><td rowspan="1" colspan="1">0.72</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs12239267</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">T (0.21)</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">0.28 (0.10)</td><td rowspan="1" colspan="1">T (0.18)</td><td rowspan="1" colspan="1">2.4 × 10<sup>−09</sup></td><td rowspan="1" colspan="1">0.44 (0.07)</td><td rowspan="1" colspan="1">6.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.39 (0.06)</td><td rowspan="1" colspan="1">++</td><td rowspan="1" colspan="1">2.2 × 10<sup>−01</sup></td><td rowspan="1" colspan="1">34.9</td><td rowspan="1" colspan="1">0.59</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs6667499</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">A (0.21)</td><td rowspan="1" colspan="1">0.002</td><td rowspan="1" colspan="1">0.31 (0.10)</td><td rowspan="1" colspan="1">A (0.21)</td><td rowspan="1" colspan="1">9.4 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.42 (0.07)</td><td rowspan="1" colspan="1">7.0 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">0.38 (0.06)</td><td rowspan="1" colspan="1">++</td><td rowspan="1" colspan="1">3.5 × 10<sup>−01</sup></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs3093058</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">A (0.21)</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">0.29 (0.10)</td><td rowspan="1" colspan="1">A (0.21)</td><td rowspan="1" colspan="1">7.6 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.43 (0.07)</td><td rowspan="1" colspan="1">1.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.39 (0.06)</td><td rowspan="1" colspan="1">++</td><td rowspan="1" colspan="1">2.7 × 10<sup>−01</sup></td><td rowspan="1" colspan="1">19.2</td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs9628671</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">T (0.23)</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">0.26 (0.10)</td><td rowspan="1" colspan="1">T (0.21)</td><td rowspan="1" colspan="1">1.3 × 10<sup>−09</sup></td><td rowspan="1" colspan="1">0.42 (0.07)</td><td rowspan="1" colspan="1">4.5 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.38 (0.06)</td><td rowspan="1" colspan="1">++</td><td rowspan="1" colspan="1">1.9 × 10<sup>−01</sup></td><td rowspan="1" colspan="1">42.5</td><td rowspan="1" colspan="1">0.73</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs16827462</td><td rowspan="1" colspan="1"><em>CRPP1</em></td><td rowspan="1" colspan="1">A (0.22)</td><td rowspan="1" colspan="1">0.001</td><td rowspan="1" colspan="1">0.33 (0.10)</td><td rowspan="1" colspan="1">A (0.19)</td><td rowspan="1" colspan="1">9.8 × 10<sup>−08</sup></td><td rowspan="1" colspan="1">0.40 (0.07)</td><td rowspan="1" colspan="1">4.2 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.37 (0.06)</td><td rowspan="1" colspan="1">++</td><td rowspan="1" colspan="1">5.8 × 10<sup>−01</sup></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0.60</td></tr></tbody></table></div><div id="largeobj_idm140192504167840" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3373247/table/DDS133TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140192508079008"><p id="__p18" class="p p-first-last"><sup>a</sup>CA, coded allele.</p></div><div id="idm140192548703840"><p id="__p19" class="p p-first-last"><sup>b</sup>CAF, coded allele frequency.</p></div><div id="idm140192548702960"><p id="__p20" class="p p-first-last"><sup>c</sup>rs16827466 has been merged into rs10494326.</p></div><div id="idm140192548702080"><p id="__p21" class="p p-first-last">*<em>P</em>-value for heterogeneity test.</p></div><div id="idm140192548700832"><p id="__p22" class="p p-first-last">**<em>P</em>-value for the test of allele frequency difference between WA and AA.</p></div></div></div><p id="__p23">In the <em>in silico</em> replication study of the published CRP GWAS findings in populations of EA and Indian Asian ancestry, we successfully replicated the <em>CRP</em> locus (rs7553007, <em>P</em> = 2.9 × 10<sup>–5</sup> and rs11265260, <em>P</em> = 0.03) in AA (Table&nbsp;<a href="/pmc/articles/PMC3373247/table/DDS133TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS133TB4" rid-ob="ob-DDS133TB4" co-legend-rid=""><span>4</span></a>). In parallel, we used an LD-based replication strategy to seek additional signals that were not directly replicated in our study. By searching all the SNPs in LD (<em>r</em><sup>2</sup>≥ 0.3) in the HapMap CEU with each of the reported SNP in a 500 kb window around the SNP of interest, we found 11 additional SNPs significantly associated with hsCRP at the <em>CRP</em> locus (Table&nbsp;<a href="/pmc/articles/PMC3373247/table/DDS133TB5/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS133TB5" rid-ob="ob-DDS133TB5" co-legend-rid=""><span>5</span></a>) and one SNP at the <em>LEPR</em> locus. Because some of the previously reported SNPs were discovered using a cohort comprising Europeans and Asian Indians, a search using HapMap (Gujarati Indians in Houston) GIH LD data found six <em>CRP</em> SNPs significantly associated with hsCRP in our study as well as one SNP in <em>LEPR</em> (rs3828033) (Table&nbsp;<a href="/pmc/articles/PMC3373247/table/DDS133TB5/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS133TB5" rid-ob="ob-DDS133TB5" co-legend-rid=""><span>5</span></a>).

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS133TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p24">Direct replication of SNPs previously reported to be associated with CRP in African Americans</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140192504498976" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Reference</th><th rowspan="1" colspan="1" align="left">Populations</th><th rowspan="1" colspan="1" align="left">SNPs</th><th rowspan="1" colspan="1" align="left">CA</th><th rowspan="1" colspan="1" align="left">Locus</th><th rowspan="1" colspan="1" align="left">Gene name</th><th rowspan="1" colspan="1" align="left">Reported <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Observed <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Reported beta</th><th rowspan="1" colspan="1" align="left">Observed beta</th><th rowspan="1" colspan="1" align="left">CA (CAF) in HUFS</th><th rowspan="1" colspan="1" align="left">CA (CAF) in CEU HapMap</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">(<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939322">8</a>)</td><td rowspan="1" colspan="1">Europeans and Indian Asians</td><td rowspan="1" colspan="1">rs7553007</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1q23.2</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">8 × 10<sup>−44</sup></td><td rowspan="1" colspan="1">2.9 × 10<sup>−05<sup>a</sup></sup></td><td rowspan="1" colspan="1">−0.21</td><td rowspan="1" colspan="1">−1.3<sup>b</sup></td><td rowspan="1" colspan="1">A (0.22)</td><td rowspan="1" colspan="1">A (0.33)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C14" rid="DDS133C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939286">14</a>)</td><td rowspan="1" colspan="1">Self-reported European ancestry</td><td rowspan="1" colspan="1">rs11265260</td><td rowspan="1" colspan="1">?</td><td rowspan="1" colspan="1">1q23.2</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">7 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.03<sup>a</sup></td><td rowspan="1" colspan="1">NR</td><td rowspan="1" colspan="1">1.04<sup>b</sup></td><td rowspan="1" colspan="1">G (0.08)</td><td rowspan="1" colspan="1">G (0.05)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939325">15</a>)</td><td rowspan="1" colspan="1">Women of European ancestry</td><td rowspan="1" colspan="1">rs3091244<sup>c</sup></td><td rowspan="1" colspan="1">?</td><td rowspan="1" colspan="1">1q23.2</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">6 × 10<sup>−28</sup></td><td rowspan="1" colspan="1">0.03<sup>a</sup></td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">T (0.26)</td><td rowspan="1" colspan="1">T (0.33)<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939309">8</a>)</td><td rowspan="1" colspan="1">Europeans and Indian Asians</td><td rowspan="1" colspan="1">rs6700896</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1p31.3</td><td rowspan="1" colspan="1"><em>LEPR</em></td><td rowspan="1" colspan="1">3 × 10<sup>−14</sup></td><td rowspan="1" colspan="1">0.80<sup>e</sup></td><td rowspan="1" colspan="1">−0.15</td><td rowspan="1" colspan="1">−0.04<sup>b</sup></td><td rowspan="1" colspan="1">T (0.45)</td><td rowspan="1" colspan="1">T (0.35)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939334">15</a>)</td><td rowspan="1" colspan="1">Women of European ancestry</td><td rowspan="1" colspan="1">rs1892534</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1p31.3</td><td rowspan="1" colspan="1"><em>LEPR</em></td><td rowspan="1" colspan="1">7 × 10<sup>−21</sup></td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1">−0.17</td><td rowspan="1" colspan="1">−0.004</td><td rowspan="1" colspan="1">A (0.45)</td><td rowspan="1" colspan="1">A (0.34)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939282">8</a>)</td><td rowspan="1" colspan="1">Europeans and Indian Asians</td><td rowspan="1" colspan="1">rs4537545</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1q21.3</td><td rowspan="1" colspan="1"><em>IL-6R</em></td><td rowspan="1" colspan="1">2 × 10<sup>−14</sup></td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">−0.30<sup>b</sup></td><td rowspan="1" colspan="1">C (0.40)</td><td rowspan="1" colspan="1">T (0.37)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939319">15</a>)</td><td rowspan="1" colspan="1">Women of European ancestry</td><td rowspan="1" colspan="1">rs8192284<sup>f</sup></td><td rowspan="1" colspan="1">?</td><td rowspan="1" colspan="1">1q23.3</td><td rowspan="1" colspan="1"><em>IL-6R</em></td><td rowspan="1" colspan="1">2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.41<sup>e</sup></td><td rowspan="1" colspan="1">−0.10</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">C (0.12)</td><td rowspan="1" colspan="1">C (0.32)<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939293">15</a>)</td><td rowspan="1" colspan="1">Women of European ancestry</td><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">2p23.3</td><td rowspan="1" colspan="1"><em>GCKR</em></td><td rowspan="1" colspan="1">7 × 10<sup>−15</sup></td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">A(0.16)</td><td rowspan="1" colspan="1">A (0.39)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C14" rid="DDS133C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939283">14</a>)</td><td rowspan="1" colspan="1">Self-reported European ancestry</td><td rowspan="1" colspan="1">rs1169310</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">12q24.31</td><td rowspan="1" colspan="1"><em>HNF1A</em></td><td rowspan="1" colspan="1">2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.89<sup>e</sup></td><td rowspan="1" colspan="1">NR</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">A (0.14)</td><td rowspan="1" colspan="1">A (0.38)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939340">15</a>)</td><td rowspan="1" colspan="1">Women of European ancestry</td><td rowspan="1" colspan="1">rs7310409</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">12q24.31</td><td rowspan="1" colspan="1"><em>HNF1A</em></td><td rowspan="1" colspan="1">7 × 10<sup>−17</sup></td><td rowspan="1" colspan="1">0.52<sup>e</sup></td><td rowspan="1" colspan="1">−0.15</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">A (0.31)</td><td rowspan="1" colspan="1">A (0.42)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939318">8</a>)</td><td rowspan="1" colspan="1">Europeans and Indian Asians</td><td rowspan="1" colspan="1">rs1183910</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">12q24.31</td><td rowspan="1" colspan="1"><em>HNF1A</em></td><td rowspan="1" colspan="1">1 × 10<sup>−30</sup></td><td rowspan="1" colspan="1">0.71<sup>e</sup></td><td rowspan="1" colspan="1">−0.14</td><td rowspan="1" colspan="1">0.13<sup>b</sup></td><td rowspan="1" colspan="1">T (0.12)</td><td rowspan="1" colspan="1">T (0.29)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939298">15</a>)</td><td rowspan="1" colspan="1">Women of European ancestry</td><td rowspan="1" colspan="1">rs10778213</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">12q23.2</td><td rowspan="1" colspan="1">Unknown</td><td rowspan="1" colspan="1">1 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.70</td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">A (0.27)</td><td rowspan="1" colspan="1">G (0.51)</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C14" rid="DDS133C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939344">14</a>)</td><td rowspan="1" colspan="1">Self-reported European ancestry</td><td rowspan="1" colspan="1">rs2075650</td><td rowspan="1" colspan="1">?</td><td rowspan="1" colspan="1">19q13.32</td><td rowspan="1" colspan="1"><em>APOE</em></td><td rowspan="1" colspan="1">1 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NR</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">—</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939342">15</a>)</td><td rowspan="1" colspan="1">Women of European ancestry</td><td rowspan="1" colspan="1">rs769449</td><td rowspan="1" colspan="1">?</td><td rowspan="1" colspan="1">19q13.32</td><td rowspan="1" colspan="1"><em>APOE</em></td><td rowspan="1" colspan="1">9 × 10<sup>−21</sup></td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">−0.26</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">—</td></tr><tr><td rowspan="1" colspan="1">(<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939328">8</a>)</td><td rowspan="1" colspan="1">Europeans and Indian Asians</td><td rowspan="1" colspan="1">rs4420638</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">19q13.32</td><td rowspan="1" colspan="1"><em>APO</em> complex</td><td rowspan="1" colspan="1">5 × 10<sup>−27</sup></td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">−0.22</td><td rowspan="1" colspan="1">−0.21</td><td rowspan="1" colspan="1">G (0.21)</td><td rowspan="1" colspan="1">G (0.18)</td></tr></tbody></table></div><div id="largeobj_idm140192504498976" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3373247/table/DDS133TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140192508811664"><p id="__p25" class="p p-first-last">NA, SNP not available in HUFS; NR, not reported; ?, risk allele was not reported.</p></div><div id="idm140192549508608"><p id="__p26" class="p p-first-last"><sup>a</sup>SNP replicated in HUFS.</p></div><div id="idm140192549507664"><p id="__p27" class="p p-first-last"><sup>b</sup>Observed beta was converted to adhere to uniformity in units and log transformation across studies. Our beta was multiplied by a factor of 4.35 (∼10/2.3) to convert it from mg/dl to mg/l and from natural log to log10.</p></div><div id="idm140192549506448"><p id="__p28" class="p p-first-last"><sup>c</sup>SNP was not available in Affymetrix data set but was genotyped in HUFS on Sequenom platform.</p></div><div id="idm140192549505392"><p id="__p29" class="p p-first-last"><sup>d</sup>Frequency from 1000 Genome.</p></div><div id="idm140192511661568"><p id="__p30" class="p p-first-last"><sup>e</sup>Imputed SNP.</p></div><div id="idm140192511660736"><p id="__p31" class="p p-first-last"><sup>f</sup>rs8192284 has been merged into rs2228145.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS133TB5"><h3>Table&nbsp;5.</h3><!--caption a7--><div class="caption"><p id="__p32">SNPs associated with hsCRP levels in African Americans using the LD-based replication method</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140192508309344" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNPs<sup>a</sup></th><th rowspan="1" colspan="1" align="left">Locus</th><th rowspan="1" colspan="1" align="left">Gene name</th><th rowspan="1" colspan="1" align="left">Reported <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">SNPs in LD</th><th rowspan="1" colspan="1" align="left">Beta<sub>AA</sub>/beta <sub>EA</sub></th><th rowspan="1" colspan="1" align="left">Uncorrected <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Corrected <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">df</th></tr></thead><tbody><tr><td colspan="9" rowspan="1">Using HapMap CEU LD</td></tr><tr><td rowspan="1" colspan="1"> rs7553007<sup>b</sup></td><td rowspan="1" colspan="1">1q23.2</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">8 × 10<sup>−44</sup></td><td rowspan="1" colspan="1">rs2592887</td><td rowspan="1" colspan="1">−0.25/−0.16</td><td rowspan="1" colspan="1">4.5 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">7.1 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs1470515</td><td rowspan="1" colspan="1">−0.24/NA</td><td rowspan="1" colspan="1">1.9 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">3.2 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2808624</td><td rowspan="1" colspan="1">−0.28/NA</td><td rowspan="1" colspan="1">1.4 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">2.1 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs11265257</td><td rowspan="1" colspan="1">−0.28/NA</td><td rowspan="1" colspan="1">1.5 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">2.4 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs876537</td><td rowspan="1" colspan="1">−0.28/NA</td><td rowspan="1" colspan="1">1.9 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">3.0 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2808628</td><td rowspan="1" colspan="1">−0.27/NA</td><td rowspan="1" colspan="1">1.4 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">2.2 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2808629</td><td rowspan="1" colspan="1">−0.28/NA</td><td rowspan="1" colspan="1">1.7 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">2.6 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2794520</td><td rowspan="1" colspan="1">−0.28/NA</td><td rowspan="1" colspan="1">6.4 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">1.0 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs1205</td><td rowspan="1" colspan="1">−0.29/−0.21</td><td rowspan="1" colspan="1">9.5 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">1.5 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2027471<sup>c</sup></td><td rowspan="1" colspan="1">−0.30/−0.20</td><td rowspan="1" colspan="1">2.3 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">3.7 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs1341665<sup>c</sup></td><td rowspan="1" colspan="1">−0.30/NA</td><td rowspan="1" colspan="1">2.3 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">3.7 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">1.58</td></tr><tr><td rowspan="1" colspan="1">rs1892534</td><td rowspan="1" colspan="1">1p31.3</td><td rowspan="1" colspan="1"><em>LEPR</em></td><td rowspan="1" colspan="1">7 × 10<sup>−21</sup></td><td rowspan="1" colspan="1">rs4291477</td><td rowspan="1" colspan="1">−0.20/NA</td><td rowspan="1" colspan="1">4.1 × 10<sup>−03</sup></td><td rowspan="1" colspan="1">7.6 × 10<sup>−03</sup></td><td rowspan="1" colspan="1">1.87</td></tr><tr><td rowspan="1" colspan="1">rs6700896<sup>b</sup></td><td rowspan="1" colspan="1">1p31.3</td><td rowspan="1" colspan="1"><em>LEPR</em></td><td rowspan="1" colspan="1">3 × 10<sup>−14</sup></td><td rowspan="1" colspan="1">rs4291477</td><td rowspan="1" colspan="1">−0.20/NA</td><td rowspan="1" colspan="1">4.1 × 10<sup>−03</sup></td><td rowspan="1" colspan="1">7.7 × 10<sup>−03</sup></td><td rowspan="1" colspan="1">1.89</td></tr><tr><td colspan="9" rowspan="1">Using HapMap GIH LD</td></tr><tr><td rowspan="1" colspan="1"> rs7553007<sup>b</sup></td><td rowspan="1" colspan="1">1q23.2</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1">8 × 10<sup>−44</sup></td><td rowspan="1" colspan="1">rs2592887</td><td rowspan="1" colspan="1">−0.25/−0.16</td><td rowspan="1" colspan="1">4.5 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">8.5 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">1.87</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs1572970</td><td rowspan="1" colspan="1">−0.23/NA</td><td rowspan="1" colspan="1">1.9 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">3.5 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.87</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs876537</td><td rowspan="1" colspan="1">−0.27/NA</td><td rowspan="1" colspan="1">1.9 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">3.6 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.87</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2808629</td><td rowspan="1" colspan="1">−0.28/NA</td><td rowspan="1" colspan="1">1.7 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">3.1 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.87</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2794520</td><td rowspan="1" colspan="1">−0.28/NA</td><td rowspan="1" colspan="1">6.4 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">1.2 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.87</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs1205</td><td rowspan="1" colspan="1">−0.29/−0.21</td><td rowspan="1" colspan="1">9.5 × 10<sup>−05</sup></td><td rowspan="1" colspan="1">1.78 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">1.87</td></tr><tr><td rowspan="1" colspan="1">rs6700896<sup>b</sup></td><td rowspan="1" colspan="1">1p31.3</td><td rowspan="1" colspan="1"><em>LEPR</em></td><td rowspan="1" colspan="1">3 × 10<sup>−14</sup></td><td rowspan="1" colspan="1">rs3828033</td><td rowspan="1" colspan="1">−0.14/NA</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">2.16</td></tr></tbody></table></div><div id="largeobj_idm140192508309344" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3373247/table/DDS133TB5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140192509831072"><p id="__p33" class="p p-first-last">Beta <sub>EA,</sub> beta in EA are taken from reference (<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939324">15</a>); beta<sub>AA<strong>,</strong></sub> beta in AA; NA; not available; df, degree of freedom.</p></div><div id="idm140192505076096"><p id="__p34" class="p p-first-last"><sup>a</sup>Only SNPs that showed association with CRP levels are reported.</p></div><div id="idm140192505075120"><p id="__p35" class="p p-first-last"><sup>b</sup>SNPs were among the top 100 SNPs associated with hsCRP in initial analysis.</p></div><div id="idm140192504668256"><p id="__p36" class="p p-first-last"><sup>c</sup>SNPs reported in Europeans and Asian Indians.</p></div></div></div><p id="__p37" class="p p-last">Finally, we attempted to refine the <em>CRP</em> signal taking advantage of the weak LD structure in our AA sample (Fig.&nbsp;<a href="/pmc/articles/PMC3373247/figure/DDS133F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDS133F2" rid-ob="ob-DDS133F2" co-legend-rid="lgnd_DDS133F2"><span>2</span></a>). We generated LD plots centered on one of the SNPs (rs7553005) associated with the CRP level across populations and examined the plots in an EA population (HapMap CEU) as well as in our study. LD around rs7553005 extends an interval of 25 kb in HapMap CEU and contains all the SNPs previously associated with CRP in EA but was restricted to 8 kb in our AA and contained two SNPs (rs7553007, rs11265260) of the three major SNPs associated with CRP in EA.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDS133F2" co-legend-rid="lgnd_DDS133F2"><a href="/pmc/articles/PMC3373247/figure/DDS133F2/" target="figure" rid-figpopup="DDS133F2" rid-ob="ob-DDS133F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140192510631392" class="figure"><a href="/pmc/articles/PMC3373247/figure/DDS133F2/" target="figure" rid-figpopup="DDS133F2" rid-ob="ob-DDS133F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3373247_dds13302.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is dds13302.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3373247/bin/dds13302.jpg"></a></div><div id="largeobj_idm140192510631392" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3373247/figure/DDS133F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDS133F2"><div><a class="figpopup" href="/pmc/articles/PMC3373247/figure/DDS133F2/" target="figure" rid-figpopup="DDS133F2" rid-ob="ob-DDS133F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p38">Comparison of linkage disequilibrium in the HapMap Phase II CEU, YRI data and our HUFS sample at <em>CRP</em> locus. Circled SNPs are the top-ranking SNPs associated with CRP levels in populations of European ancestry. Arrow indicates the index SNP (rs7553007) used to build the LD plot in the region. Arrow indicates the position of the index SNP in our data (rs3093058).</p></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p39" class="p p-first">CRP is a well-known acute phase reactant in the inflammatory process whose levels are elevated not just in infection, but also in chronic non-communicable diseases such as CVD (<a href="#DDS133C4" rid="DDS133C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939321">4</a>). Most studies of the genetic variants influencing circulating CRP levels have used a candidate gene approach with only a handful using a genome-wide strategy. In the present study, we conducted the first genome-wide association analysis of CRP levels in AA. In common with the previous GWAS in populations of EA and Asian ancestry (<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939330">8</a>,<a href="#DDS133C14" rid="DDS133C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939289">14</a>–<a href="#DDS133C18" rid="DDS133C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298503605">18</a>), we found that the strongest associations were with SNPs located within and around the <em>CRP</em> locus. Similar to findings in Europeans, the SNPs with the lowest <em>P</em>-values in this study of AA (rs3093058, rs9628671) are located in the <em>CRP</em> promoter region; however, the risk alleles for these two SNPs are very rare in populations of EA as demonstrated in the HapMap CEU reference and NHANES data (<a href="#DDS133C21" rid="DDS133C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939284">21</a>). Two previous candidate gene studies conducted in AA—Carlson <em>et al</em>. (<a href="#DDS133C22" rid="DDS133C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939312">22</a>) and Crawford <em>et al</em>. (<a href="#DDS133C21" rid="DDS133C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939287">21</a>)—confirmed rs3093058 as being one of three <em>CRP</em> SNPs contributing to ethnic differences in CRP levels. The index SNP (rs3093058) associated with CRP in our study is different from those previously reported in EA (<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939338">8</a>,<a href="#DDS133C14" rid="DDS133C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939313">14</a>–<a href="#DDS133C16" rid="DDS133C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939335">16</a>) but tagging the same signal as shown by the LD plots of EA and AA (Fig.&nbsp;<a href="/pmc/articles/PMC3373247/figure/DDS133F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDS133F2" rid-ob="ob-DDS133F2" co-legend-rid="lgnd_DDS133F2"><span>2</span></a>). The difference seen across populations in the reported SNPs associated with a phenotype is not uncommon, especially if one of the populations has an African ancestry background that is characterized by shorter LD blocks (<a href="#DDS133C23" rid="DDS133C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939341">23</a>). These findings illustrate an important issue often observed in genetic studies of admixed populations such as AA. The reported genetic variants associated with a phenotype may be population-specific due to differences in the population's ancestral background, haplotype structure and allele frequencies as demonstrated by our study and by re-sequencing of the <em>CRP</em> locus in several populations (<a href="#DDS133C24" rid="DDS133C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939310">24</a>).</p><p id="__p40">The confirmation or replication of reported signals is often more successful in populations with a similar ancestry as the original discovery population (<a href="#DDS133C25" rid="DDS133C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939331">25</a>). Therefore, we chose to replicate our findings in WA and successfully replicated all but one SNP (rs16827466) with directionally consistent effect size estimates. The lack of replication of rs16827466 in WA may be due to heterogeneity in both allele frequencies and effect size (as shown in Table&nbsp;<a href="/pmc/articles/PMC3373247/table/DDS133TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS133TB3" rid-ob="ob-DDS133TB3" co-legend-rid=""><span>3</span></a>). Although heterogeneity between individual studies can affect replication efforts, the robustness of the <em>CRP</em> association is quite remarkable, seeing that our study successfully replicated three previously reported CRP-associated SNPs (rs3091244, rs11265260 and rs7553007) despite the sharp differences in allele frequency (especially rs3091244 and rs7553007) between EA populations and our study sample. Differences in allele frequency between populations can easily obscure associations and it has been estimated that a change in minor allele frequency (MAF) &lt;0.1 can result in a drop in power to replicate from 80% to as low as 20% (<a href="#DDS133C26" rid="DDS133C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939307">26</a>). In this study, we declared replication of a locus based on an adjusted <em>P</em>-value (&lt;0.05) and consistent effect size direction. It should be noted that the weak LD structure seen in our AA samples is not always an advantage when doing ‘exact replication’ because the large LD block containing all the signals in EA was segmented into smaller LD blocks in AA. On the other hand, this weak LD in African ancestry populations is often an advantage in following up an association signal because it could result in considerable refinement of the original locus, as has been successfully demonstrated with loci associated with height, uric acid and bilirubin (<a href="#DDS133C23" rid="DDS133C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939349">23</a>,<a href="#DDS133C27" rid="DDS133C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939301">27</a>,<a href="#DDS133C28" rid="DDS133C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298503603">28</a>). In the present study, we were able to refine the localization of the <em>CRP</em> signal by successfully fine-mapping the 25 kb interval of LD around the <em>CRP</em> locus in the HapMap CEU sample by more than 2-fold to 8 kb in our AA sample (assuming a single causal variant in the region common to both populations). However, it should be noted that fine mapping may be complicated by multiple causal variants in the same region (some of which may be population-specific). Future studies using sequence-based approaches would probably provide examples of such situations.</p><p id="__p41">The present study did not find novel loci, which is not surprising given the limited power of our small discovery panel for detecting weak effect sizes. On the other hand, using an LD-based replication strategy, we successfully replicated additional SNPs in the <em>LEPR</em> gene previously reported to be associated with CRP levels by two independent studies in Europeans (<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939292">8</a>,<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939288">15</a>,<a href="#DDS133C16" rid="DDS133C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939296">16</a>). In addition to <em>CRP</em> and <em>LEPR</em>, other genes have been associated with CRP levels in Europeans including <em>HNF1A</em> (gene encoding a hepatic transcriptional regulator), <em>IL6R</em>, <em>APO</em> complex and <em>GCKR</em> (gene encoding a hepatic and pancreatic glucokinase regulator protein). However, none of these was replicated in the present study. It should be noted that the associated SNPs vary considerably in frequency between populations. For example, rs1183910 (T) in <em>HNF1A</em> has an MAF of 0.292 in HapMap CEU in contrast to 0.092 and 0.048 in the HapMap (Yoruba in Ibadan) YRI and AA from Perlegen samples, respectively. This, apart from sample size issues, is a source of heterogeneity between studies.</p><p id="__p42">Besides the commonly reported loci, we found suggestive associations between serum CRP levels and <em>LRP1B</em>, <em>MAN2A1</em>, <em>ERCC4</em>. Although the precise biological mechanisms by which these genes could influence CRP levels are currently unclear, there is indirect evidence that they play a role in CRP regulation through processes that are mediated by obesity. For example, <em>ERCC4</em>, a gene involved in nucleotide excision repair, has been associated with abdominal obesity in the GWAS of metabolic syndrome in Indian Asian men (<a href="#DDS133C29" rid="DDS133C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939326">29</a>) and <em>LRP1B</em>, which encodes an LRP1B, is associated with BMI in a meta-analysis (conducted by GWAS) of 249 796 individuals of EA (<a href="#DDS133C30" rid="DDS133C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939280">30</a>).</p><p id="__p43" class="p p-last">In conclusion, we have conducted a genome-wide association analysis of CRP levels in AA and confirmed that the genetic variants with the strongest effects are all within and around the <em>CRP</em> locus. We also fine-mapped the <em>CRP</em> signal to an 8 kb interval. Although the power to detect weak effect sizes was limited by the sample size, we were able to successfully replicate a secondary locus, the <em>LEPR</em> gene. As additional studies of AA and other ethnicities are conducted, meta-analysis would facilitate the discovery of more loci regulating circulating CRP levels.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><p id="__p44" class="p p-first">The participants used in this study were drawn from a larger population-based study of AA from the Washington, DC metropolitan area. This study is described in detail elsewhere (<a href="#DDS133C31" rid="DDS133C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939308">31</a>). Briefly, the goal of the study was to enroll and examine families as well as unrelated AA to study the genetics of common complex traits. Participants were not ascertained based on any phenotype. During clinic visits, participants provided written informed consent and completed a set of questionnaires. Anthropometric and blood pressure measurements were taken and blood samples were drawn for measurements of several biochemical parameters. All individuals included in this GWAS for CRP were unrelated. The study protocol was approved by Howard University Internal Review Board (IRB).</p><p id="__p45">HsCRP was measured using a clinical auto analyzer (COBAS Integra 400 plus). This is a particle-enhanced turbidimetric assay. Human CRP is made to react with latex particles coated with monoclonal anti-CRP antibodies and the precipitate is determined turbidimetrically at 552 nm. The measuring range for this assay is 0.01–2 mg/dl.</p><p id="__p46">Genome-wide genotyping was performed using the Affymetrix<sup>®</sup> Genome-Wide Human SNP Array 6.0. DNA samples were prepared and hybridized following the manufacturer's instructions. After processing, chips were scanned and genotype calls were made using the Birdseed 2 Algorithm. All samples used in the analysis achieved a chip-wide call rate of ≥95%. Individual SNPs were excluded if they had a call rate of &lt;95% across all individuals, an MAF of ≤0.01 or had a Hardy–Weinberg equilibrium test <em>P</em>-value of &lt;1 × 10<sup>−3</sup>. The average call rate for this set of SNPs in these individuals was 99.55%. The concordance of blind duplicates was 99.74%. A check for population stratification or structure was sought by conducting non-parametric clustering of genotypes as previously described (<a href="#DDS133C32" rid="DDS133C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298503606">32</a>). The genomic inflation factor (<em>λ</em>) of 0.9961 and the quantile–quantile plot (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>) showed minimal evidence of population stratification. To adjust for any potential residual population stratification and admixture, all association analyses were adjusted for the first two PCs of the genotypes computed using EIGENSOFT (<a href="#DDS133C33" rid="DDS133C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939291">33</a>).To increase genomic coverage, imputation was performed using the MACH algorithm and with the HapMap reference populations as previously described (<a href="#DDS133C23" rid="DDS133C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939314">23</a>). After quality control filters were applied, ∼1.6 million successfully imputed SNPs were available, bringing the total number of SNPs to 2 366 856 genotyped and imputed autosomal SNPs. CRP values were natural log-transformed to assure model assumptions of normally distributed residuals. Genome-wide association analyses for ln(CRP) were conducted using multiple linear regression models as implemented in PLINK v1.07. Analysis was done under an additive genetic model, with one model adjusting for age, sex, BMI, T2D and the first two PCs of genotypes, whereas a second model adjusted for these covariates as well as smoking status and hypertension. An association <em>P</em>-value of &lt;5 × 10<sup>−8</sup> was considered genome-wide significant.</p><p id="__p47">The replication in WA of our top-scoring GWAS findings was conducted in an independent set of unrelated subjects enrolled as part of the Africa America Diabetes Mellitus (AADM) study, described fully elsewhere (<a href="#DDS133C34" rid="DDS133C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939327">34</a>,<a href="#DDS133C35" rid="DDS133C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939332">35</a>). The AADM study is an international collaboration between American and African investigators to study the genetic epidemiology of T2D in WA, with the participants coming from two WA countries (Nigeria and Ghana). The selected SNPs were genotyped using Sequenom Homogenous MassEXTEND or iPLEX Gold SBE assays at the National Human Genome Research Institute (NHGRI). The quality control filters previously described for the discovery sample were applied to this data set. The association between each SNP and CRP level was assessed using similar regression models under an additive genetic model controlling for the same covariates as in the discovery sample; replication significance was set at a <em>P</em>-value of &lt;0.05. The meta-analysis of the discovery and replication cohorts was conducted using the METAL software (<a href="#DDS133C36" rid="DDS133C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939302">36</a>). The evidence of association was combined in two-sided <em>P</em>-values using a fixed effects model. The sample size of each study was used as weight and the sign of the beta value was used as the direction of the association of the minor allele. The heterogeneity test was performed to evaluate between-study heterogeneity and a <em>P</em>-value of χ<sup>2</sup> (<em>P</em> &lt; 0.05) was used to declare significance (<a href="#DDS133C37" rid="DDS133C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939281">37</a>).</p><p id="__p48" class="p p-last">To conduct <em>in silico</em> replication of previously published GWAS CRP signals (<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939320">8</a>,<a href="#DDS133C14" rid="DDS133C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939294">14</a>,<a href="#DDS133C15" rid="DDS133C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939285">15</a>) in AA [Howard University Family Study, (HUFS)], we used a two-step strategy. Of the 15 commonly reported SNPs, 12 were available (either genotyped or imputed) in this study and we directly checked their association statistics and <em>P</em>-values in our AA data set. As a second step, we used LD to find additional markers in LD with the reported SNPs. We queried a 500 kb window centered on the reported SNP for the SNPs in LD (<em>r</em>² ≥ 0.3) with the query SNP as previously described by Ramos <em>et al</em>. (<a href="#DDS133C38" rid="DDS133C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939337">38</a>) and then obtained association statistics for the resulting SNPs. The appropriate HapMap reference population for the source GWAS (CEU for European, GIH for Indian Asian) was used to determine LD (<a href="#DDS133C8" rid="DDS133C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274939347">8</a>). To correct for multiple testing of correlated test statistics for SNPs in LD, the number of independent tests was estimated by computing the number of effective degrees of freedom (df) using the spectrally decomposed covariance matrix (<a href="#DDS133C39" rid="DDS133C39" class=" bibr popnode">39</a>). <em>P</em>-values were adjusted for the number of independent tests in the region. <em>P</em>-values for replication were considered significant if the adjusted <em>P</em> &lt; 0.05.</p></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p49" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds133/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p50" class="p p-first-last">This work was supported in part by the Intramural Research Program of the Center for Research on Genomics and Global Health (CRGGH). The CRGGH is supported by funds from the Office of the Director, National Institute of Diabetes and Digestive and Kidney diseases (NIDDK) and National Human Genome Research Institute (NHGRI) at National Institutes of Health (<span>Z01HG200362</span>). The Howard University Family Study was supported by National Institutes of Health grants (S06GM008016-320107 to C.R., S06GM008016-380111 to A.A.). The Howard University General Clinical Research Center was supported by National Institutes of Health (grant <span>2M01RR010284</span>).</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140192503959520"><a href="/pmc/articles/PMC3373247/bin/supp_21_13_3063__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140192506881008" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140192506881008title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p51">We thank the participants of the study, for which enrolment was carried out at the Howard University General Clinical Research Center. The contents of this publication are solely the responsibility of the authors and do not necessarily represent the official view of the National Institutes of Health. Genotyping support was provided by the Coriell Institute for Medical Research.</p><p id="__p52"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140192506878512" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140192506878512title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDS133C1">1. <span class="element-citation">Clyne B., Olshaker J.S. The C-reactive protein. <span><span class="ref-journal">J. Emerg. Med. </span>1999;<span class="ref-vol">17</span>:1019–1025.</span> [<a href="/pubmed/10595891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Emerg.+Med.&amp;title=The+C-reactive+protein&amp;author=B.+Clyne&amp;author=J.S.+Olshaker&amp;volume=17&amp;publication_year=1999&amp;pages=1019-1025&amp;pmid=10595891&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C2">2. <span class="element-citation">Ganapathi M.K., May L.T., Schultz D., Brabenec A., Weinstein J., Sehgal P.B., Kushner I. Role of interleukin-6 in regulating synthesis of C-reactive protein and serum amyloid A in human hepatoma cell lines. <span><span class="ref-journal">Biochem. Biophys. Res. Commun. </span>1988;<span class="ref-vol">157</span>:271–277.</span> [<a href="/pubmed/2461709" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Biophys.+Res.+Commun.&amp;title=Role+of+interleukin-6+in+regulating+synthesis+of+C-reactive+protein+and+serum+amyloid+A+in+human+hepatoma+cell+lines&amp;author=M.K.+Ganapathi&amp;author=L.T.+May&amp;author=D.+Schultz&amp;author=A.+Brabenec&amp;author=J.+Weinstein&amp;volume=157&amp;publication_year=1988&amp;pages=271-277&amp;pmid=2461709&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C3">3. <span class="element-citation">Castell J.V., Gomez-Lechon M.J., David M., Andus T., Geiger T., Trullenque R., Fabra R., Heinrich P.C. Interleukin-6 is the major regulator of acute phase protein synthesis in adult human hepatocytes. <span><span class="ref-journal">FEBS Lett. </span>1989;<span class="ref-vol">242</span>:237–239.</span> [<a href="/pubmed/2464504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett.&amp;title=Interleukin-6+is+the+major+regulator+of+acute+phase+protein+synthesis+in+adult+human+hepatocytes&amp;author=J.V.+Castell&amp;author=M.J.+Gomez-Lechon&amp;author=M.+David&amp;author=T.+Andus&amp;author=T.+Geiger&amp;volume=242&amp;publication_year=1989&amp;pages=237-239&amp;pmid=2464504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C4">4. <span class="element-citation">Ridker P.M. C-reactive protein: eighty years from discovery to emergence as a major risk marker for cardiovascular disease. <span><span class="ref-journal">Clin. Chem. </span>2009;<span class="ref-vol">55</span>:209–215.</span> [<a href="/pubmed/19095723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=C-reactive+protein:+eighty+years+from+discovery+to+emergence+as+a+major+risk+marker+for+cardiovascular+disease&amp;author=P.M.+Ridker&amp;volume=55&amp;publication_year=2009&amp;pages=209-215&amp;pmid=19095723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C5">5. <span class="element-citation">Weinhold B., Ruther U. Interleukin-6-dependent and -independent regulation of the human C-reactive protein gene. <span><span class="ref-journal">Biochem. J. </span>1997;<span class="ref-vol">327</span>:425–429.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1218811/">PMC free article</a>]</span> [<a href="/pubmed/9359411" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J.&amp;title=Interleukin-6-dependent+and+-independent+regulation+of+the+human+C-reactive+protein+gene&amp;author=B.+Weinhold&amp;author=U.+Ruther&amp;volume=327&amp;publication_year=1997&amp;pages=425-429&amp;pmid=9359411&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C6">6. <span class="element-citation">Fox E.R., Benjamin E.J., Sarpong D.F., Rotimi C.N., Wilson J.G., Steffes M.W., Chen G., Adeyemo A., Taylor J.K., Samdarshi T.E., et al.  Epidemiology, heritability, and genetic linkage of C-reactive protein in African Americans (from the Jackson Heart Study) <span><span class="ref-journal">Am. J. Cardiol. </span>2008;<span class="ref-vol">102</span>:835–841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3733442/">PMC free article</a>]</span> [<a href="/pubmed/18805107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol.&amp;title=Epidemiology,+heritability,+and+genetic+linkage+of+C-reactive+protein+in+African+Americans+(from+the+Jackson+Heart+Study)&amp;author=E.R.+Fox&amp;author=E.J.+Benjamin&amp;author=D.F.+Sarpong&amp;author=C.N.+Rotimi&amp;author=J.G.+Wilson&amp;volume=102&amp;publication_year=2008&amp;pages=835-841&amp;pmid=18805107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C7">7. <span class="element-citation">Florez H., Castillo-Florez S., Mendez A., Casanova-Romero P., Larreal-Urdaneta C., Lee D., Goldberg R. C-reactive protein is elevated in obese patients with the metabolic syndrome. <span><span class="ref-journal">Diabetes Res. Clin. Pract. </span>2006;<span class="ref-vol">71</span>:92–100.</span> [<a href="/pubmed/16002176" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes+Res.+Clin.+Pract.&amp;title=C-reactive+protein+is+elevated+in+obese+patients+with+the+metabolic+syndrome&amp;author=H.+Florez&amp;author=S.+Castillo-Florez&amp;author=A.+Mendez&amp;author=P.+Casanova-Romero&amp;author=C.+Larreal-Urdaneta&amp;volume=71&amp;publication_year=2006&amp;pages=92-100&amp;pmid=16002176&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C8">8. <span class="element-citation">Elliott P., Chambers J.C., Zhang W., Clarke R., Hopewell J.C., Peden J.F., Erdmann J., Braund P., Engert J.C., Bennett D., et al.  Genetic Loci associated with C-reactive protein levels and risk of coronary heart disease. <span><span class="ref-journal">JAMA. </span>2009;<span class="ref-vol">302</span>:37–48.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2803020/">PMC free article</a>]</span> [<a href="/pubmed/19567438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Genetic+Loci+associated+with+C-reactive+protein+levels+and+risk+of+coronary+heart+disease&amp;author=P.+Elliott&amp;author=J.C.+Chambers&amp;author=W.+Zhang&amp;author=R.+Clarke&amp;author=J.C.+Hopewell&amp;volume=302&amp;publication_year=2009&amp;pages=37-48&amp;pmid=19567438&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C9">9. <span class="element-citation">Ridker P.M., Cushman M., Stampfer M.J., Tracy R.P., Hennekens C.H. Inflammation, aspirin, and the risk of cardiovascular disease in apparently healthy men. <span><span class="ref-journal">N. Eng. J. Med. </span>1997;<span class="ref-vol">336</span>:973–979.</span> [<a href="/pubmed/9077376" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Eng.+J.+Med.&amp;title=Inflammation,+aspirin,+and+the+risk+of+cardiovascular+disease+in+apparently+healthy+men&amp;author=P.M.+Ridker&amp;author=M.+Cushman&amp;author=M.J.+Stampfer&amp;author=R.P.+Tracy&amp;author=C.H.+Hennekens&amp;volume=336&amp;publication_year=1997&amp;pages=973-979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C10">10. <span class="element-citation">Yeh E.T. High-sensitivity C-reactive protein as a risk assessment tool for cardiovascular disease. <span><span class="ref-journal">Clin. Cardiol. </span>2005;<span class="ref-vol">28</span>:408–412.</span> [<a href="/pubmed/16250263" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cardiol.&amp;title=High-sensitivity+C-reactive+protein+as+a+risk+assessment+tool+for+cardiovascular+disease&amp;author=E.T.+Yeh&amp;volume=28&amp;publication_year=2005&amp;pages=408-412&amp;pmid=16250263&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C11">11. <span class="element-citation">Lee C.C., You N.C., Song Y., Hsu Y.H., Manson J., Nathan L., Tinker L., Liu S. Relation of genetic variation in the gene coding for C-reactive protein with its plasma protein concentrations: findings from the Women's Health Initiative Observational Cohort. <span><span class="ref-journal">Clin. Chem. </span>2009;<span class="ref-vol">55</span>:351–360.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2856608/">PMC free article</a>]</span> [<a href="/pubmed/19095725" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Relation+of+genetic+variation+in+the+gene+coding+for+C-reactive+protein+with+its+plasma+protein+concentrations:+findings+from+the+Women's+Health+Initiative+Observational+Cohort&amp;author=C.C.+Lee&amp;author=N.C.+You&amp;author=Y.+Song&amp;author=Y.H.+Hsu&amp;author=J.+Manson&amp;volume=55&amp;publication_year=2009&amp;pages=351-360&amp;pmid=19095725&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C12">12. <span class="element-citation">Dehghan A., Kardys I., de Maat M.P., Uitterlinden A.G., Sijbrands E.J., Bootsma A.H., Stijnen T., Hofman A., Schram M.T., Witteman J.C. Genetic variation, C-reactive protein levels, and incidence of diabetes. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:872–878.</span> [<a href="/pubmed/17327459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Genetic+variation,+C-reactive+protein+levels,+and+incidence+of+diabetes&amp;author=A.+Dehghan&amp;author=I.+Kardys&amp;author=M.P.+de+Maat&amp;author=A.G.+Uitterlinden&amp;author=E.J.+Sijbrands&amp;volume=56&amp;publication_year=2007&amp;pages=872-878&amp;pmid=17327459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C13">13. <span class="element-citation">Hage F.G., Szalai A.J. C-reactive protein gene polymorphisms, C-reactive protein blood levels, and cardiovascular disease risk. <span><span class="ref-journal">J. Am. Coll. Cardiol. </span>2007;<span class="ref-vol">50</span>:1115–1122.</span> [<a href="/pubmed/17868801" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol.&amp;title=C-reactive+protein+gene+polymorphisms,+C-reactive+protein+blood+levels,+and+cardiovascular+disease+risk&amp;author=F.G.+Hage&amp;author=A.J.+Szalai&amp;volume=50&amp;publication_year=2007&amp;pages=1115-1122&amp;pmid=17868801&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C14">14. <span class="element-citation">Reiner A.P., Barber M.J., Guan Y., Ridker P.M., Lange L.A., Chasman D.I., Walston J.D., Cooper G.M., Jenny N.S., Rieder M.J., et al.  Polymorphisms of the HNF1A gene encoding hepatocyte nuclear factor-1 alpha are associated with C-reactive protein. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:1193–1201.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427318/">PMC free article</a>]</span> [<a href="/pubmed/18439552" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Polymorphisms+of+the+HNF1A+gene+encoding+hepatocyte+nuclear+factor-1+alpha+are+associated+with+C-reactive+protein&amp;author=A.P.+Reiner&amp;author=M.J.+Barber&amp;author=Y.+Guan&amp;author=P.M.+Ridker&amp;author=L.A.+Lange&amp;volume=82&amp;publication_year=2008&amp;pages=1193-1201&amp;pmid=18439552&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C15">15. <span class="element-citation">Ridker P.M., Pare G., Parker A., Zee R.Y., Danik J.S., Buring J.E., Kwiatkowski D., Cook N.R., Miletich J.P., Chasman D.I. Loci related to metabolic-syndrome pathways including LEPR,HNF1A, IL6R, and GCKR associate with plasma C-reactive protein: the Women's Genome Health Study. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:1185–1192.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427311/">PMC free article</a>]</span> [<a href="/pubmed/18439548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Loci+related+to+metabolic-syndrome+pathways+including+LEPR,HNF1A,+IL6R,+and+GCKR+associate+with+plasma+C-reactive+protein:+the+Women's+Genome+Health+Study&amp;author=P.M.+Ridker&amp;author=G.+Pare&amp;author=A.+Parker&amp;author=R.Y.+Zee&amp;author=J.S.+Danik&amp;volume=82&amp;publication_year=2008&amp;pages=1185-1192&amp;pmid=18439548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C16">16. <span class="element-citation">Dehghan A., Dupuis J., Barbalic M., Bis J.C., Eiriksdottir G., Lu C., Pellikka N., Wallaschofski H., Kettunen J., Henneman P., et al.  Meta-analysis of genome-wide association studies in &gt;80 000 subjects identifies multiple loci for C-reactive protein levels. <span><span class="ref-journal">Circulation. </span>2011;<span class="ref-vol">123</span>:731–738.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3147232/">PMC free article</a>]</span> [<a href="/pubmed/21300955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Meta-analysis+of+genome-wide+association+studies+in+>80+000+subjects+identifies+multiple+loci+for+C-reactive+protein+levels&amp;author=A.+Dehghan&amp;author=J.+Dupuis&amp;author=M.+Barbalic&amp;author=J.C.+Bis&amp;author=G.+Eiriksdottir&amp;volume=123&amp;publication_year=2011&amp;pages=731-738&amp;pmid=21300955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C17">17. <span class="element-citation">Okada Y., Takahashi A., Ohmiya H., Kumasaka N., Kamatani Y., Hosono N., Tsunoda T., Matsuda K., Tanaka T., Kubo M., et al.  Genome-wide association study for C-reactive protein levels identified pleiotropic associations in the IL6 locus. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:1224–1231.</span> [<a href="/pubmed/21196492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+study+for+C-reactive+protein+levels+identified+pleiotropic+associations+in+the+IL6+locus&amp;author=Y.+Okada&amp;author=A.+Takahashi&amp;author=H.+Ohmiya&amp;author=N.+Kumasaka&amp;author=Y.+Kamatani&amp;volume=20&amp;publication_year=2011&amp;pages=1224-1231&amp;pmid=21196492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C18">18. <span class="element-citation">Wu Y., McDade T.W., Kuzawa C.W., Borja J., Li Y., Adair L.S., Mohlke K.L., Lange L.A. Genome-wide Association with C-Reactive Protein Levels in CLHNS: evidence for the CRP and HNF1A loci and their interaction with exposure to a pathogenic environment. <span><span class="ref-journal">Inflammation. </span>2011</span> 10.1007/s10753-011-9348-y. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3193904/">PMC free article</a>]</span> [<a href="/pubmed/21647738" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Inflammation&amp;title=Genome-wide+Association+with+C-Reactive+Protein+Levels+in+CLHNS:+evidence+for+the+CRP+and+HNF1A+loci+and+their+interaction+with+exposure+to+a+pathogenic+environment&amp;author=Y.+Wu&amp;author=T.W.+McDade&amp;author=C.W.+Kuzawa&amp;author=J.+Borja&amp;author=Y.+Li&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C19">19. <span class="element-citation">Patel D.A., Srinivasan S.R., Xu J.H., Li S., Chen W., Berenson G.S. Distribution and metabolic syndrome correlates of plasma C-reactive protein in biracial (black-white) younger adults: the Bogalusa Heart Study. <span><span class="ref-journal">Metab. Clin. Exp. </span>2006;<span class="ref-vol">55</span>:699–705.</span> [<a href="/pubmed/16713426" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metab.+Clin.+Exp.&amp;title=Distribution+and+metabolic+syndrome+correlates+of+plasma+C-reactive+protein+in+biracial+(black-white)+younger+adults:+the+Bogalusa+Heart+Study&amp;author=D.A.+Patel&amp;author=S.R.+Srinivasan&amp;author=J.H.+Xu&amp;author=S.+Li&amp;author=W.+Chen&amp;volume=55&amp;publication_year=2006&amp;pages=699-705&amp;pmid=16713426&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C20">20. <span class="element-citation">Rosenberg N.A., Huang L., Jewett E.M., Szpiech Z.A., Jankovic I., Boehnke M. Genome-wide association studies in diverse populations. <span><span class="ref-journal">Nat. Rev. Genet. </span>2010;<span class="ref-vol">11</span>:356–366.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079573/">PMC free article</a>]</span> [<a href="/pubmed/20395969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genome-wide+association+studies+in+diverse+populations&amp;author=N.A.+Rosenberg&amp;author=L.+Huang&amp;author=E.M.+Jewett&amp;author=Z.A.+Szpiech&amp;author=I.+Jankovic&amp;volume=11&amp;publication_year=2010&amp;pages=356-366&amp;pmid=20395969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C21">21. <span class="element-citation">Crawford D.C., Sanders C.L., Qin X., Smith J.D., Shephard C., Wong M., Witrak L., Rieder M.J., Nickerson D.A. Genetic variation is associated with C-reactive protein levels in the Third National Health and Nutrition Examination Survey. <span><span class="ref-journal">Circulation. </span>2006;<span class="ref-vol">114</span>:2458–2465.</span> [<a href="/pubmed/17101857" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Genetic+variation+is+associated+with+C-reactive+protein+levels+in+the+Third+National+Health+and+Nutrition+Examination+Survey&amp;author=D.C.+Crawford&amp;author=C.L.+Sanders&amp;author=X.+Qin&amp;author=J.D.+Smith&amp;author=C.+Shephard&amp;volume=114&amp;publication_year=2006&amp;pages=2458-2465&amp;pmid=17101857&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C22">22. <span class="element-citation">Carlson C.S., Aldred S.F., Lee P.K., Tracy R.P., Schwartz S.M., Rieder M., Liu K., Williams O.D., Iribarren C., Lewis E.C., et al.  Polymorphisms within the C-reactive protein (CRP) promoter region are associated with plasma CRP levels. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2005;<span class="ref-vol">77</span>:64–77.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226195/">PMC free article</a>]</span> [<a href="/pubmed/15897982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Polymorphisms+within+the+C-reactive+protein+(CRP)+promoter+region+are+associated+with+plasma+CRP+levels&amp;author=C.S.+Carlson&amp;author=S.F.+Aldred&amp;author=P.K.+Lee&amp;author=R.P.+Tracy&amp;author=S.M.+Schwartz&amp;volume=77&amp;publication_year=2005&amp;pages=64-77&amp;pmid=15897982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C23">23. <span class="element-citation">Shriner D., Adeyemo A., Gerry N.P., Herbert A., Chen G., Doumatey A., Huang H., Zhou J., Christman M.F., Rotimi C.N. Transferability and fine-mapping of genome-wide associated loci for adult height across human populations. <span><span class="ref-journal">PLoS ONE. </span>2009;<span class="ref-vol">4</span>:e8398.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2792725/">PMC free article</a>]</span> [<a href="/pubmed/20027299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Transferability+and+fine-mapping+of+genome-wide+associated+loci+for+adult+height+across+human+populations&amp;author=D.+Shriner&amp;author=A.+Adeyemo&amp;author=N.P.+Gerry&amp;author=A.+Herbert&amp;author=G.+Chen&amp;volume=4&amp;publication_year=2009&amp;pages=e8398&amp;pmid=20027299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C24">24. <span class="element-citation">Crawford D.C., Yi Q., Smith J.D., Shephard C., Wong M., Witrak L., Livingston R.J., Rieder M.J., Nickerson D.A. Allelic spectrum of the natural variation in CRP. <span><span class="ref-journal">Hum. Genet. </span>2006;<span class="ref-vol">119</span>:496–504.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1449912/">PMC free article</a>]</span> [<a href="/pubmed/16550411" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Allelic+spectrum+of+the+natural+variation+in+CRP&amp;author=D.C.+Crawford&amp;author=Q.+Yi&amp;author=J.D.+Smith&amp;author=C.+Shephard&amp;author=M.+Wong&amp;volume=119&amp;publication_year=2006&amp;pages=496-504&amp;pmid=16550411&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C25">25. <span class="element-citation">Chanock S.J., Manolio T., Boehnke M., Boerwinkle E., Hunter D.J., Thomas G., Hirschhorn J.N., Abecasis G., Altshuler D., Bailey-Wilson J.E., et al.  Replicating genotype-phenotype associations. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:655–660.</span> [<a href="/pubmed/17554299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Replicating+genotype-phenotype+associations&amp;author=S.J.+Chanock&amp;author=T.+Manolio&amp;author=M.+Boehnke&amp;author=E.+Boerwinkle&amp;author=D.J.+Hunter&amp;volume=447&amp;publication_year=2007&amp;pages=655-660&amp;pmid=17554299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C26">26. <span class="element-citation">Greene C.S., Penrod N.M., Williams S.M., Moore J.H. Failure to replicate a genetic association may provide important clues about genetic architecture. <span><span class="ref-journal">PLoS ONE. </span>2009;<span class="ref-vol">4</span>:e5639.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2685469/">PMC free article</a>]</span> [<a href="/pubmed/19503614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Failure+to+replicate+a+genetic+association+may+provide+important+clues+about+genetic+architecture&amp;author=C.S.+Greene&amp;author=N.M.+Penrod&amp;author=S.M.+Williams&amp;author=J.H.+Moore&amp;volume=4&amp;publication_year=2009&amp;pages=e5639&amp;pmid=19503614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C27">27. <span class="element-citation">Charles B.A., Shriner D., Doumatey A., Chen G., Zhou J., Huang H., Herbert A., Gerry N.P., Christman M.F., Adeyemo A., et al.  A genome-wide association study of serum uric acid in African Americans. <span><span class="ref-journal">BMC Med. Genomics. </span>2011;<span class="ref-vol">4</span>:17.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3045279/">PMC free article</a>]</span> [<a href="/pubmed/21294900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genomics&amp;title=A+genome-wide+association+study+of+serum+uric+acid+in+African+Americans&amp;author=B.A.+Charles&amp;author=D.+Shriner&amp;author=A.+Doumatey&amp;author=G.+Chen&amp;author=J.+Zhou&amp;volume=4&amp;publication_year=2011&amp;pages=17&amp;pmid=21294900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C28">28. <span class="element-citation">Chen G., Ramos E., Adeyemo A., Shriner D., Zhou J., Doumatey A.P., Huang H., Erdos M.R., Gerry N.P., Herbert A., et al.  UGT1A1 is a major locus influencing bilirubin levels in African Americans. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2011</span> 10.1038/ejhg.2011.206. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3306855/">PMC free article</a>]</span> [<a href="/pubmed/22085899" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=UGT1A1+is+a+major+locus+influencing+bilirubin+levels+in+African+Americans&amp;author=G.+Chen&amp;author=E.+Ramos&amp;author=A.+Adeyemo&amp;author=D.+Shriner&amp;author=J.+Zhou&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C29">29. <span class="element-citation">Zabaneh D., Balding D.J. A genome-wide association study of the metabolic syndrome in Indian Asian men. <span><span class="ref-journal">PLoS ONE. </span>2010;<span class="ref-vol">5</span>:e11961.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2915922/">PMC free article</a>]</span> [<a href="/pubmed/20694148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=A+genome-wide+association+study+of+the+metabolic+syndrome+in+Indian+Asian+men&amp;author=D.+Zabaneh&amp;author=D.J.+Balding&amp;volume=5&amp;publication_year=2010&amp;pages=e11961&amp;pmid=20694148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C30">30. <span class="element-citation">Speliotes E.K., Willer C.J., Berndt S.I., Monda K.L., Thorleifsson G., Jackson A.U., Allen H.L., Lindgren C.M., Luan J.A., Magi R., et al.  Association analyses of 249,796 individuals reveal 18 new loci associated with body mass index. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:937–948.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014648/">PMC free article</a>]</span> [<a href="/pubmed/20935630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Association+analyses+of+249,796+individuals+reveal+18+new+loci+associated+with+body+mass+index&amp;author=E.K.+Speliotes&amp;author=C.J.+Willer&amp;author=S.I.+Berndt&amp;author=K.L.+Monda&amp;author=G.+Thorleifsson&amp;volume=42&amp;publication_year=2010&amp;pages=937-948&amp;pmid=20935630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C31">31. <span class="element-citation">Adeyemo A., Gerry N., Chen G., Herbert A., Doumatey A., Huang H., Zhou J., Lashley K., Chen Y., Christman M., et al.  A genome-wide association study of hypertension and blood pressure in African Americans. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000564.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2702100/">PMC free article</a>]</span> [<a href="/pubmed/19609347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+genome-wide+association+study+of+hypertension+and+blood+pressure+in+African+Americans&amp;author=A.+Adeyemo&amp;author=N.+Gerry&amp;author=G.+Chen&amp;author=A.+Herbert&amp;author=A.+Doumatey&amp;volume=5&amp;publication_year=2009&amp;pages=e1000564&amp;pmid=19609347&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C32">32. <span class="element-citation">Gao X., Starmer J.D. AWclust: point-and-click software for non-parametric population structure analysis. <span><span class="ref-journal">BMC Bioinf. </span>2008;<span class="ref-vol">9</span>:77.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253519/">PMC free article</a>]</span> [<a href="/pubmed/18237431" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf.&amp;title=AWclust:+point-and-click+software+for+non-parametric+population+structure+analysis&amp;author=X.+Gao&amp;author=J.D.+Starmer&amp;volume=9&amp;publication_year=2008&amp;pages=77&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C33">33. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C34">34. <span class="element-citation">Rotimi C.N., Dunston G.M., Berg K., Akinsete O., Amoah A., Owusu S., Acheampong J., Boateng K., Oli J., Okafor G., et al.  In search of susceptibility genes for type 2 diabetes in West Africa: the design and results of the first phase of the AADM study. <span><span class="ref-journal">Ann. Epidemiol. </span>2001;<span class="ref-vol">11</span>:51–58.</span> [<a href="/pubmed/11164120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Epidemiol.&amp;title=In+search+of+susceptibility+genes+for+type+2+diabetes+in+West+Africa:+the+design+and+results+of+the+first+phase+of+the+AADM+study&amp;author=C.N.+Rotimi&amp;author=G.M.+Dunston&amp;author=K.+Berg&amp;author=O.+Akinsete&amp;author=A.+Amoah&amp;volume=11&amp;publication_year=2001&amp;pages=51-58&amp;pmid=11164120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C35">35. <span class="element-citation">Rotimi C.N., Chen G., Adeyemo A.A., Furbert-Harris P., Parish-Gause D., Zhou J., Berg K., Adegoke O., Amoah A., Owusu S., et al.  A genome-wide search for type 2 diabetes susceptibility genes in West Africans: the Africa America Diabetes Mellitus (AADM) Study. <span><span class="ref-journal">Diabetes. </span>2004;<span class="ref-vol">53</span>:838–841.</span> [<a href="/pubmed/14988271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+genome-wide+search+for+type+2+diabetes+susceptibility+genes+in+West+Africans:+the+Africa+America+Diabetes+Mellitus+(AADM)+Study&amp;author=C.N.+Rotimi&amp;author=G.+Chen&amp;author=A.A.+Adeyemo&amp;author=P.+Furbert-Harris&amp;author=D.+Parish-Gause&amp;volume=53&amp;publication_year=2004&amp;pages=838-841&amp;pmid=14988271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C36">36. <span class="element-citation">Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C37">37. <span class="element-citation">Higgins J.P., Thompson S.G., Deeks J.J., Altman D.G. Measuring inconsistency in meta-analyses. <span><span class="ref-journal">BMJ. </span>2003;<span class="ref-vol">327</span>:557–560.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC192859/">PMC free article</a>]</span> [<a href="/pubmed/12958120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Measuring+inconsistency+in+meta-analyses&amp;author=J.P.+Higgins&amp;author=S.G.+Thompson&amp;author=J.J.+Deeks&amp;author=D.G.+Altman&amp;volume=327&amp;publication_year=2003&amp;pages=557-560&amp;pmid=12958120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C38">38. <span class="element-citation">Ramos E., Chen G., Shriner D., Doumatey A., Gerry N.P., Herbert A., Huang H., Zhou J., Christman M.F., Adeyemo A., et al.  Replication of genome-wide association studies (GWAS) loci for fasting plasma glucose in African-Americans. <span><span class="ref-journal">Diabetologia. </span>2011;<span class="ref-vol">54</span>:783–788.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3052446/">PMC free article</a>]</span> [<a href="/pubmed/21188353" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Replication+of+genome-wide+association+studies+(GWAS)+loci+for+fasting+plasma+glucose+in+African-Americans&amp;author=E.+Ramos&amp;author=G.+Chen&amp;author=D.+Shriner&amp;author=A.+Doumatey&amp;author=N.P.+Gerry&amp;volume=54&amp;publication_year=2011&amp;pages=783-788&amp;pmid=21188353&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS133C39">39. <span class="element-citation">Bretherton C.S., Widmann M., Dymnikov V.P., Wallace J.M., Bladé I. The effective number of spatial degrees of freedom of a time-varying field. <span><span class="ref-journal">J. Clim. </span>1999;<span class="ref-vol">12</span>:1990–2009.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clim.&amp;title=The+effective+number+of+spatial+degrees+of+freedom+of+a+time-varying+field&amp;author=C.S.+Bretherton&amp;author=M.+Widmann&amp;author=V.P.+Dymnikov&amp;author=J.M.+Wallace&amp;author=I.+Bladé&amp;volume=12&amp;publication_year=1999&amp;pages=1990-2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3373247&amp;issue-id=210999&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140192509915552title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140192506881008title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140192506878512title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3373247/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3373247/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3373247/pdf/dds133.pdf">PDF (354K)</a></li> | <li><a href="#" data-citationid="PMC3373247" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3373247%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3373247%2F&amp;text=C-reactive%20protein%20(CRP)%20promoter%20polymorphisms%20influence%20circulating%20CRP%20levels%20in%20a%20genome-wide%20association%20study%20of%20African%20Americans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3373247%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22492993" data-aiid="3373247" data-aid="3373247" data-iid="210999" data-domainid="740" data-domain="hmg" data-accid="PMC3373247" data-md5="081523c5a921deddc82a7a9afa0207b5"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:43:53-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE897A07DD537C010000000007A4054C&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3373247%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3373247/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>