<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3736322/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association study identifies susceptibility loci for non-syndromic sagittal craniosynostosis near BMP2 and within BBS9</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3736322/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="A genome-wide association study identifies susceptibility loci for non-syndromic sagittal craniosynostosis near BMP2 and within BBS9"><meta name="citation_authors" content="Cristina M. Justice, Garima Yagnik, Yoonhee Kim, Inga Peter, Ethylin Wang Jabs, Monica Erazo, Xiaoqian Ye, Edmond Ainehsazan, Lisong Shi, Michael L. Cunningham, Virginia Kimonis, Tony Roscioli, Steven A. Wall, Andrew O.M. Wilkie, Joan Stoler, Joan T. Richtsmeier, Yann Heuzé, Pedro A. Sanchez-Lara, Michael F. Buckley, Charlotte M. Druschel, James L. Mills, Michele Caggana, Paul A. Romitti, Denise M. Kay, Craig Senders, Peter J. Taub, Ophir D. Klein, James Boggan, Marike Zwienenberg-Lee, Cyril Naydenov, Jinoh Kim, Alexander F. Wilson, Simeon A. Boyadjiev"><meta name="citation_date" content="December 2012"><meta name="citation_issue" content="12"><meta name="citation_volume" content="44"><meta name="citation_firstpage" content="1360"><meta name="citation_doi" content="10.1038/ng.2463"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3736322/?report=abstract"><meta name="citation_pmid" content="23160099"><meta name="DC.Title" content="A genome-wide association study identifies susceptibility loci for non-syndromic sagittal craniosynostosis near BMP2 and within BBS9"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Cristina M. Justice"><meta name="DC.Contributor" content="Garima Yagnik"><meta name="DC.Contributor" content="Yoonhee Kim"><meta name="DC.Contributor" content="Inga Peter"><meta name="DC.Contributor" content="Ethylin Wang Jabs"><meta name="DC.Contributor" content="Monica Erazo"><meta name="DC.Contributor" content="Xiaoqian Ye"><meta name="DC.Contributor" content="Edmond Ainehsazan"><meta name="DC.Contributor" content="Lisong Shi"><meta name="DC.Contributor" content="Michael L. Cunningham"><meta name="DC.Contributor" content="Virginia Kimonis"><meta name="DC.Contributor" content="Tony Roscioli"><meta name="DC.Contributor" content="Steven A. Wall"><meta name="DC.Contributor" content="Andrew O.M. Wilkie"><meta name="DC.Contributor" content="Joan Stoler"><meta name="DC.Contributor" content="Joan T. Richtsmeier"><meta name="DC.Contributor" content="Yann Heuzé"><meta name="DC.Contributor" content="Pedro A. Sanchez-Lara"><meta name="DC.Contributor" content="Michael F. Buckley"><meta name="DC.Contributor" content="Charlotte M. Druschel"><meta name="DC.Contributor" content="James L. Mills"><meta name="DC.Contributor" content="Michele Caggana"><meta name="DC.Contributor" content="Paul A. Romitti"><meta name="DC.Contributor" content="Denise M. Kay"><meta name="DC.Contributor" content="Craig Senders"><meta name="DC.Contributor" content="Peter J. Taub"><meta name="DC.Contributor" content="Ophir D. Klein"><meta name="DC.Contributor" content="James Boggan"><meta name="DC.Contributor" content="Marike Zwienenberg-Lee"><meta name="DC.Contributor" content="Cyril Naydenov"><meta name="DC.Contributor" content="Jinoh Kim"><meta name="DC.Contributor" content="Alexander F. Wilson"><meta name="DC.Contributor" content="Simeon A. Boyadjiev"><meta name="DC.Date" content="2012 Dec"><meta name="DC.Identifier" content="10.1038/ng.2463"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association study identifies susceptibility loci for non-syndromic sagittal craniosynostosis near BMP2 and within BBS9"><meta property="og:type" content="article"><meta property="og:description" content="Sagittal craniosynostosis is the most common form of craniosynostosis, affecting approximately one of 5,000 newborns. We conducted the first genome-wide association study (GWAS) for non-syndromic sagittal craniosynostosis (sNSC) using 130 non-Hispanic ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3736322/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3736322/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3736322/pdf/emss-52898.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CC83EDD54AB0100000000061203F9.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3736322%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="protein" class="last">Protein</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC3736322</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2013 Aug 7.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23160099" target="pmc_ext" ref="reftype=publisher&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2012 Dec; 44(12): 1360–1364. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Nov 18. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.2463" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.2463</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3736322</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS52898</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23160099">23160099</a></div></div></div></div><h1 class="content-title">A genome-wide association study identifies susceptibility loci for non-syndromic sagittal craniosynostosis near <em>BMP2</em> and within <em>BBS9</em></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Justice%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712022076896" co-class="co-affbox">Cristina M. Justice</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Yagnik%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712019716608" co-class="co-affbox">Garima Yagnik</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Kim%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712096948592" co-class="co-affbox">Yoonhee Kim</a>,<sup>1</sup> <a href="/pubmed/?term=Peter%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712096946608" co-class="co-affbox">Inga Peter</a>,<sup>3</sup> <a href="/pubmed/?term=Jabs%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712096944624" co-class="co-affbox">Ethylin Wang Jabs</a>,<sup>3</sup> <a href="/pubmed/?term=Erazo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712096942640" co-class="co-affbox">Monica Erazo</a>,<sup>3</sup> <a href="/pubmed/?term=Ye%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712096940656" co-class="co-affbox">Xiaoqian Ye</a>,<sup>3</sup> <a href="/pubmed/?term=Ainehsazan%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712096938672" co-class="co-affbox">Edmond Ainehsazan</a>,<sup>3</sup> <a href="/pubmed/?term=Shi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061173936" co-class="co-affbox">Lisong Shi</a>,<sup>3</sup> <a href="/pubmed/?term=Cunningham%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061171952" co-class="co-affbox">Michael L. Cunningham</a>,<sup>4</sup> <a href="/pubmed/?term=Kimonis%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061169968" co-class="co-affbox">Virginia Kimonis</a>,<sup>5</sup> <a href="/pubmed/?term=Roscioli%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061167984" co-class="co-affbox">Tony Roscioli</a>,<sup>6</sup> <a href="/pubmed/?term=Wall%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061166000" co-class="co-affbox">Steven A. Wall</a>,<sup>7</sup> <a href="/pubmed/?term=Wilkie%20AO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061164016" co-class="co-affbox">Andrew O.M. Wilkie</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Stoler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061161296" co-class="co-affbox">Joan Stoler</a>,<sup>9</sup> <a href="/pubmed/?term=Richtsmeier%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061159312" co-class="co-affbox">Joan T. Richtsmeier</a>,<sup>10</sup> <a href="/pubmed/?term=Heuz%26%23x000e9%3B%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061157328" co-class="co-affbox">Yann Heuzé</a>,<sup>10</sup> <a href="/pubmed/?term=Sanchez-Lara%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061155344" co-class="co-affbox">Pedro A. Sanchez-Lara</a>,<sup>11</sup> <a href="/pubmed/?term=Buckley%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061153360" co-class="co-affbox">Michael F. Buckley</a>,<sup>12</sup> <a href="/pubmed/?term=Druschel%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712061151376" co-class="co-affbox">Charlotte M. Druschel</a>,<sup>13</sup> <a href="/pubmed/?term=Mills%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023651440" co-class="co-affbox">James L. Mills</a>,<sup>14</sup> <a href="/pubmed/?term=Caggana%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023649456" co-class="co-affbox">Michele Caggana</a>,<sup>15</sup> <a href="/pubmed/?term=Romitti%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023647472" co-class="co-affbox">Paul A. Romitti</a>,<sup>16</sup> <a href="/pubmed/?term=Kay%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023645488" co-class="co-affbox">Denise M. Kay</a>,<sup>15</sup> <a href="/pubmed/?term=Senders%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023643504" co-class="co-affbox">Craig Senders</a>,<sup>17</sup> <a href="/pubmed/?term=Taub%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023641520" co-class="co-affbox">Peter J. Taub</a>,<sup>18</sup> <a href="/pubmed/?term=Klein%20OD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023639536" co-class="co-affbox">Ophir D. Klein</a>,<sup>19,</sup><sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Boggan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023636080" co-class="co-affbox">James Boggan</a>,<sup>22</sup> <a href="/pubmed/?term=Zwienenberg-Lee%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023634096" co-class="co-affbox">Marike Zwienenberg-Lee</a>,<sup>22</sup> <a href="/pubmed/?term=Naydenov%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023632112" co-class="co-affbox">Cyril Naydenov</a>,<sup>23</sup> <a href="/pubmed/?term=Kim%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712023630128" co-class="co-affbox">Jinoh Kim</a>,<sup>2</sup> <a href="/pubmed/?term=Wilson%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712026484272" co-class="co-affbox">Alexander F. Wilson</a>,<sup>1</sup> and  <a href="/pubmed/?term=Boyadjiev%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099" class="affpopup" co-rid="_co_idm140712026482288" co-class="co-affbox">Simeon A. Boyadjiev</a><sup>2,</sup><sup>#</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140712022076896"><h3 class="no_margin">Cristina M. Justice</h3><p><sup>1</sup>Genometrics Section, Inherited Disease Research Branch, Division of Intramural Research, NHGRI, NIH, Baltimore, MD </p><div>Find articles by <a href="/pubmed/?term=Justice%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Cristina M. Justice</a></div></div><div id="_co_idm140712019716608"><h3 class="no_margin">Garima Yagnik</h3><p><sup>2</sup>Section of Genetics, Department of Pediatrics, University of California Davis, Sacramento, CA </p><div>Find articles by <a href="/pubmed/?term=Yagnik%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Garima Yagnik</a></div></div><div id="_co_idm140712096948592"><h3 class="no_margin">Yoonhee Kim</h3><p><sup>1</sup>Genometrics Section, Inherited Disease Research Branch, Division of Intramural Research, NHGRI, NIH, Baltimore, MD </p><div>Find articles by <a href="/pubmed/?term=Kim%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Yoonhee Kim</a></div></div><div id="_co_idm140712096946608"><h3 class="no_margin">Inga Peter</h3><p><sup>3</sup>Department of Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Peter%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Inga Peter</a></div></div><div id="_co_idm140712096944624"><h3 class="no_margin">Ethylin Wang Jabs</h3><p><sup>3</sup>Department of Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Jabs%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Ethylin Wang Jabs</a></div></div><div id="_co_idm140712096942640"><h3 class="no_margin">Monica Erazo</h3><p><sup>3</sup>Department of Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Erazo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Monica Erazo</a></div></div><div id="_co_idm140712096940656"><h3 class="no_margin">Xiaoqian Ye</h3><p><sup>3</sup>Department of Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Ye%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Xiaoqian Ye</a></div></div><div id="_co_idm140712096938672"><h3 class="no_margin">Edmond Ainehsazan</h3><p><sup>3</sup>Department of Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Ainehsazan%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Edmond Ainehsazan</a></div></div><div id="_co_idm140712061173936"><h3 class="no_margin">Lisong Shi</h3><p><sup>3</sup>Department of Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Shi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Lisong Shi</a></div></div><div id="_co_idm140712061171952"><h3 class="no_margin">Michael L. Cunningham</h3><p><sup>4</sup>Department of Pediatrics, Division of Craniofacial Medicine, University of Washington and Seattle Children’s Research Institute, Seattle, WA </p><div>Find articles by <a href="/pubmed/?term=Cunningham%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Michael L. Cunningham</a></div></div><div id="_co_idm140712061169968"><h3 class="no_margin">Virginia Kimonis</h3><p><sup>5</sup>Division of Genetics, Department of Pediatrics, University of California Irvine, Irvine, CA </p><div>Find articles by <a href="/pubmed/?term=Kimonis%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Virginia Kimonis</a></div></div><div id="_co_idm140712061167984"><h3 class="no_margin">Tony Roscioli</h3><p><sup>6</sup>School of Women’s and Children’s Health, Sydney Children’s Hospital, University of New South Wales, Sydney, Australia </p><div>Find articles by <a href="/pubmed/?term=Roscioli%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Tony Roscioli</a></div></div><div id="_co_idm140712061166000"><h3 class="no_margin">Steven A. Wall</h3><p><sup>7</sup>Craniofacial Unit, Oxford University Hospitals NHS Trust, John Radcliffe Hospital, Oxford OX3 9DU, UK </p><div>Find articles by <a href="/pubmed/?term=Wall%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Steven A. Wall</a></div></div><div id="_co_idm140712061164016"><h3 class="no_margin">Andrew O.M. Wilkie</h3><p><sup>7</sup>Craniofacial Unit, Oxford University Hospitals NHS Trust, John Radcliffe Hospital, Oxford OX3 9DU, UK </p><p><sup>8</sup>Weatherall Institute of Molecular Medicine, University of Oxford, John Radcliffe Hospital, Oxford OX3 9DS, UK </p><div>Find articles by <a href="/pubmed/?term=Wilkie%20AO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Andrew O.M. Wilkie</a></div></div><div id="_co_idm140712061161296"><h3 class="no_margin">Joan Stoler</h3><p><sup>9</sup>Division of Genetics, Children’s Hospital Boston, Harvard University, Boston, MA </p><div>Find articles by <a href="/pubmed/?term=Stoler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Joan Stoler</a></div></div><div id="_co_idm140712061159312"><h3 class="no_margin">Joan T. Richtsmeier</h3><p><sup>10</sup>Department of Anthropology, Pennsylvania State University, University Park, PA </p><div>Find articles by <a href="/pubmed/?term=Richtsmeier%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Joan T. Richtsmeier</a></div></div><div id="_co_idm140712061157328"><h3 class="no_margin">Yann Heuzé</h3><p><sup>10</sup>Department of Anthropology, Pennsylvania State University, University Park, PA </p><div>Find articles by <a href="/pubmed/?term=Heuz%26%23x000e9%3B%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Yann Heuzé</a></div></div><div id="_co_idm140712061155344"><h3 class="no_margin">Pedro A. Sanchez-Lara</h3><p><sup>11</sup>Division of Genetics, Department of Pediatrics, University of South California, Los Angeles, CA </p><div>Find articles by <a href="/pubmed/?term=Sanchez-Lara%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Pedro A. Sanchez-Lara</a></div></div><div id="_co_idm140712061153360"><h3 class="no_margin">Michael F. Buckley</h3><p><sup>12</sup>Department of Haematology and Genetics, SEALS, Sydney, Australia </p><div>Find articles by <a href="/pubmed/?term=Buckley%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Michael F. Buckley</a></div></div><div id="_co_idm140712061151376"><h3 class="no_margin">Charlotte M. Druschel</h3><p><sup>13</sup>Congenital Malformations Registry, New York State Department of Health, Albany, NY </p><div>Find articles by <a href="/pubmed/?term=Druschel%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Charlotte M. Druschel</a></div></div><div id="_co_idm140712023651440"><h3 class="no_margin">James L. Mills</h3><p><sup>14</sup>Division of Epidemiology, Statistics, and Prevention Research, Eunice Kennedy Shriver National Institute of Child Health and Human Development, National Institutes of Health, Department of Health and Human Services, Bethesda, MD </p><div>Find articles by <a href="/pubmed/?term=Mills%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">James L. Mills</a></div></div><div id="_co_idm140712023649456"><h3 class="no_margin">Michele Caggana</h3><p><sup>15</sup>Division of Genetics, Wadsworth Center, New York State Department of Health, Albany, NY </p><div>Find articles by <a href="/pubmed/?term=Caggana%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Michele Caggana</a></div></div><div id="_co_idm140712023647472"><h3 class="no_margin">Paul A. Romitti</h3><p><sup>16</sup>Department of Epidemiology, College of Public Health, The University of Iowa, Iowa City, IA </p><div>Find articles by <a href="/pubmed/?term=Romitti%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Paul A. Romitti</a></div></div><div id="_co_idm140712023645488"><h3 class="no_margin">Denise M. Kay</h3><p><sup>15</sup>Division of Genetics, Wadsworth Center, New York State Department of Health, Albany, NY </p><div>Find articles by <a href="/pubmed/?term=Kay%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Denise M. Kay</a></div></div><div id="_co_idm140712023643504"><h3 class="no_margin">Craig Senders</h3><p><sup>17</sup>Department of Otolaryngology, University of California Davis, Sacramento, CA </p><div>Find articles by <a href="/pubmed/?term=Senders%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Craig Senders</a></div></div><div id="_co_idm140712023641520"><h3 class="no_margin">Peter J. Taub</h3><p><sup>18</sup>Division of Plastic and Reconstructive Surgery, Kravis Children’s Hospital, Mount Sinai Medical Center, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Taub%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Peter J. Taub</a></div></div><div id="_co_idm140712023639536"><h3 class="no_margin">Ophir D. Klein</h3><p><sup>19</sup>Department of Orofacial Sciences, University of California San Francisco, San Francisco CA </p><p><sup>20</sup>Department of Pediatrics, University of California San Francisco, San Francisco CA </p><p><sup>21</sup>Program in Craniofacial and Mesenchymal Biology, University of California San Francisco, San Francisco, CA </p><div>Find articles by <a href="/pubmed/?term=Klein%20OD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Ophir D. Klein</a></div></div><div id="_co_idm140712023636080"><h3 class="no_margin">James Boggan</h3><p><sup>22</sup>Department of Neurological Surgery, University of California Davis, Sacramento, CA </p><div>Find articles by <a href="/pubmed/?term=Boggan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">James Boggan</a></div></div><div id="_co_idm140712023634096"><h3 class="no_margin">Marike Zwienenberg-Lee</h3><p><sup>22</sup>Department of Neurological Surgery, University of California Davis, Sacramento, CA </p><div>Find articles by <a href="/pubmed/?term=Zwienenberg-Lee%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Marike Zwienenberg-Lee</a></div></div><div id="_co_idm140712023632112"><h3 class="no_margin">Cyril Naydenov</h3><p><sup>23</sup>Department of Chemistry and Biochemistry, Medical University, Sofia, Bulgaria </p><div>Find articles by <a href="/pubmed/?term=Naydenov%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Cyril Naydenov</a></div></div><div id="_co_idm140712023630128"><h3 class="no_margin">Jinoh Kim</h3><p><sup>2</sup>Section of Genetics, Department of Pediatrics, University of California Davis, Sacramento, CA </p><div>Find articles by <a href="/pubmed/?term=Kim%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Jinoh Kim</a></div></div><div id="_co_idm140712026484272"><h3 class="no_margin">Alexander F. Wilson</h3><p><sup>1</sup>Genometrics Section, Inherited Disease Research Branch, Division of Intramural Research, NHGRI, NIH, Baltimore, MD </p><div>Find articles by <a href="/pubmed/?term=Wilson%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Alexander F. Wilson</a></div></div><div id="_co_idm140712026482288"><h3 class="no_margin">Simeon A. Boyadjiev</h3><p><sup>2</sup>Section of Genetics, Department of Pediatrics, University of California Davis, Sacramento, CA </p><div>Find articles by <a href="/pubmed/?term=Boyadjiev%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23160099">Simeon A. Boyadjiev</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140712061873776_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140712061873776_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140712061873776_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Genometrics Section, Inherited Disease Research Branch, Division of Intramural Research, NHGRI, NIH, Baltimore, MD </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Section of Genetics, Department of Pediatrics, University of California Davis, Sacramento, CA </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, NY </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Pediatrics, Division of Craniofacial Medicine, University of Washington and Seattle Children’s Research Institute, Seattle, WA </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Division of Genetics, Department of Pediatrics, University of California Irvine, Irvine, CA </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>School of Women’s and Children’s Health, Sydney Children’s Hospital, University of New South Wales, Sydney, Australia </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Craniofacial Unit, Oxford University Hospitals NHS Trust, John Radcliffe Hospital, Oxford OX3 9DU, UK </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Weatherall Institute of Molecular Medicine, University of Oxford, John Radcliffe Hospital, Oxford OX3 9DS, UK </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Division of Genetics, Children’s Hospital Boston, Harvard University, Boston, MA </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Anthropology, Pennsylvania State University, University Park, PA </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Division of Genetics, Department of Pediatrics, University of South California, Los Angeles, CA </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Haematology and Genetics, SEALS, Sydney, Australia </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Congenital Malformations Registry, New York State Department of Health, Albany, NY </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Division of Epidemiology, Statistics, and Prevention Research, Eunice Kennedy Shriver National Institute of Child Health and Human Development, National Institutes of Health, Department of Health and Human Services, Bethesda, MD </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Division of Genetics, Wadsworth Center, New York State Department of Health, Albany, NY </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Epidemiology, College of Public Health, The University of Iowa, Iowa City, IA </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Otolaryngology, University of California Davis, Sacramento, CA </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Division of Plastic and Reconstructive Surgery, Kravis Children’s Hospital, Mount Sinai Medical Center, New York, NY </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Orofacial Sciences, University of California San Francisco, San Francisco CA </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Department of Pediatrics, University of California San Francisco, San Francisco CA </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Program in Craniofacial and Mesenchymal Biology, University of California San Francisco, San Francisco, CA </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Neurological Surgery, University of California Davis, Sacramento, CA </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Department of Chemistry and Biochemistry, Medical University, Sofia, Bulgaria </div><div id="FN1"><sup>*</sup>Authors with equal contribution</div><div id="CR1"><sup>#</sup>Corresponding author (<a href="mailto:dev@null" data-email="ude.sivadcu.cmdcu@dyob.noemis" class="oemail">ude.sivadcu.cmdcu@dyob.noemis</a>) </div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140712061873776_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23160099" target="pmc_ext" ref="reftype=publisher&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3736322/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SuppInfo.</strong></div><div><a href="/pmc/articles/PMC3736322/bin/NIHMS52898-supplement-SuppInfo.pdf" data-ga-action="click_feat_suppl">NIHMS52898-supplement-SuppInfo.pdf</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;44EB6321-54A1-4BF4-8D09-CAF5ECB7A51A</div></div></dd></dl></div><div id="idm140712026236688" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140712026236688title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">Sagittal craniosynostosis is the most common form of craniosynostosis, affecting approximately one of 5,000 newborns. We conducted the first genome-wide association study (GWAS) for non-syndromic sagittal craniosynostosis (sNSC) using 130 non-Hispanic white (NHW) case-parent trios. Robust associations were observed in a 120 kb region downstream of <em>BMP2</em>, flanked by rs1884302 (<em>P</em> = 1.13 × 10<sup>−14</sup>; odds ratio [OR] = 4.58) and rs6140226 (<em>P</em> = 3.40 × 10<sup>−11</sup>; OR = 0.24) and within a 167 kb region of <em>BBS9</em> between rs10262453 (<em>P</em> = 1.61 × 10<sup>−10</sup>; OR=0.19) and rs17724206 (<em>P</em> = 1.50 × 10<sup>−8</sup>; OR = 0.22). We replicated the associations to both loci [rs1884302 (<em>P</em> = 4.39 × 10<sup>−31</sup>); rs10262453 (<em>P</em> = 3.50 × 10<sup>−14</sup>)] in an independent NHW population of 172 unrelated sNSC probands and 548 controls. Both <em>BMP2</em> and <em>BBS9</em> are genes with a role in skeletal development warranting functional studies to further understand the etiology of sNSC.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genome-wide association study, non-syndromic sagittal craniosynostosis, <em>BMP2</em>, <em>BBS9</em>, meta-analysis, nonsyndromic</span></div></div><div id="idm140712024636288" class="tsec sec headless whole_rhythm"><p id="P3" class="p p-first">Skull development is a complex process that involves ongoing interaction between the bones of the skull and cranial soft tissues. The cranial vault is comprised of intramembranous bones joined by sutures of dense fibrous tissue that accommodate the growing brain. Bone is added at these sutures during growth, and the skull eventually ossifies completely. Craniosynostosis (CS), the premature closure of one or more of the cranial vault sutures is a common congenital anomaly. Approximately 80% of CS occurs as an isolated anomaly, non-syndromic craniosynostosis (NSC), without major associated malformations (<a href="#R1" rid="R1" class=" bibr popnode">1</a>). Rare mutations in the <em>FGFR2, TWIST1</em>, <em>FREM1, LRIT3, EFNA4,</em> and <em>RUNX2</em> duplications have been reported in a minor fraction of NSC cases (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724009">2</a>-<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724014">9</a>). The remainder (approximately 20%) of CS cases are syndromic, occurring with one or more additional major malformations, due to single-gene mutations in one of at least eight genes (<em>FGFR1, FGFR2, FGFR3, TWIST1, EFNB1, POR, MSX2</em> and <em>RAB23</em>) primarily involving the coronal sutures (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723989">10</a>-<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724031">12</a>).</p><p id="P4">The most common type of NSC is synostosis of the sagittal suture (sNSC), accounting for 40 to 58% of all cases (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724021">13</a>-<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723997">14</a>). The etiology of sNSC is not well understood, although the published literature suggests that it is a multifactorial condition, where both genetic and environmental factors play a role, as indicated by a higher rate of concordance in monozygotic as compared to dizygotic twins (30% vs. 0% in sNSC) (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724028">15</a>), increased male to female ratio (3:1) (<a href="#R1" rid="R1" class=" bibr popnode">1</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724003">16</a>), and a high recurrence risk in affected families (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724015">17</a>). Several environmental factors have also been associated with sNSC, including parity, prematurity, intrauterine constraint, and maternal tobacco or nitrosatable drug use (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724002">18</a>-<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724033">21</a>).</p><p id="P5">In an attempt to identify susceptibility loci for sNSC, we recruited 201 case-parent trios and 13 nuclear families with at least one individual affected with sNSC (68% non-Hispanic white [NHW], 32% mixed ethnicity families). These participants were enrolled and evaluated through the collaborative effort of the International Craniosynostosis Consortium (<a href="http://genetics.ucdmc.ucdavis.edu/icc.cfm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">genetics.ucdmc.ucdavis.edu/icc.cfm</a>). Samples were genotyped on the Illumina 1M Human Omni1-Quad array. Final criteria for the current analyses were based on strict sNSC phenotype (no additional synostoses, congenital anomalies or developmental delay present in the proband or affected sibling) and NHW ethnicity as determined by self-reported ancestry and confirmed by principal-component analysis (PCA) using EIGENSOFT 3.0 (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723988">22</a>). Application of these criteria reduced our discovery population to 130 case-parent trios.</p><p id="P6">After stringent quality control procedures, we retained genotypes for 915,307 SNPs (914,402 autosomal and 905 pseudo-autosomal) which were analyzed using the transmission disequilibrium test (TDT) as implemented in PLINK v1.07 (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724006">23</a>). This resulted in the identification of 21 SNPs , 18 on 20p12.3 and 3 on 7p14.3, which reached the genome-wide significance threshold of P &lt; 5 × 10<sup>−8</sup> (<a href="/pmc/articles/PMC3736322/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). The 18 chromosome 20 SNPs were in two LD blocks (92 kb and 19 kb, on chromosome 20p12.3; <a href="/pmc/articles/PMC3736322/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>, <a href="/pmc/articles/PMC3736322/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), with the most significant SNP, rs1884302 (<em>P</em> = 1.13 × 10<sup>−14</sup>), located approximately 345 kb downstream of <em>BMP2.</em> Conditional analysis using extended TDT (ETDT) in UNPHASED (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724010">24</a>) confirmed that rs1884302 is leading these signals (data not shown). Except for a predicted mRNA, <em><a id="__tag_809267518" class="tag_hotlink" href="/nuccore/BC038533" ref="/nuccore/BC038533"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"BC038533","term_id":"23959007"}}</span>BC038533</a></em>, no other known genes, CNVs, transcripts, miRNA or regulatory elements were identified in this region. However, at least four highly conserved genome regions with multiple predicted sites for transcription factor binding were identified by bioinformatic analysis in the vicinity of rs1884302 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>). Exonic resequencing of <em>BMP2</em> in our cohort did not identify any variants with obvious deleterious effects. The three SNPs on chromosome 7p14.3 span a region of 167 kb within <em>BBS9</em> introns 4 and 15 (<a href="/pmc/articles/PMC3736322/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>). RT-PCR sequencing of cDNA from eight sNSC calvarial osteoblast cell lines discovered six <em>BBS9</em> variants, but three were determined to be previously reported SNPs and all were benign based on bioinformatic protein prediction tools and conservation analysis (see online Methods). We also searched for two-way gene-gene interactions using the 100 most significant TDT GWAS SNPs using a conditional logistic regression model (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724011">25</a>). No results were significant after correcting for multiple testing (1247 tests, Bonferroni <em>P</em> ≤ 4 × 10<sup>−5</sup>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3736322/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140712021892304" class="figure"><a href="/pmc/articles/PMC3736322/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3736322_emss-52898-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-52898-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3736322/bin/emss-52898-f0001.jpg"></a></div><div id="largeobj_idm140712021892304" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3736322/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3736322/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Manhattan plot of <em>P</em>-values obtained from the genome-wide TDT of 130 trios (N=914,402)</strong><p id="__p2">The x-axis corresponds to the genomic position of the autosomes, while the y-axis is the −log10 of the P-value. Horizontal dashed line corresponds to the genome-wide significance threshold of P ≤ 5 × 10<sup>−8</sup>.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3736322/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140712021884624" class="figure"><a href="/pmc/articles/PMC3736322/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3736322_emss-52898-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-52898-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3736322/bin/emss-52898-f0002.jpg"></a></div><div id="largeobj_idm140712021884624" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3736322/figure/F2/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140712021883776" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3736322_emss-52898-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-52898-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3736322/bin/emss-52898-f0003.jpg"></a></div><div id="largeobj_idm140712021883776" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3736322/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3736322/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional association plots for associations with sNSC at genome-wide significance</strong><p id="__p3">Upper panel: <em>P</em>-values (-log10) of the GWAS (solid circle), replication (triangle) and meta-analysis (asterisk) on the y-axis are plotted against the genomic positions of each SNP on the x-axis on <strong>(a)</strong> chromosome 20 and on <strong>(b)</strong> chromosome 7. Horizontal dashed line corresponds to the genome-wide significance threshold of <em>P</em> ≤ 5 × 10<sup>−8</sup>. Genes in region are depicted below (not to scale). The LD (<em>r<sup>2</sup></em>) between the lead SNP and other SNPs are indicated in different colors. The blue lines indicate the recombination rates in cM per Mb using HapMap controls. Lower panel: LD plots (based on <em>r<sup>2</sup></em> value) using genotyped subjects from the GWAS study.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">GWAS, replication study and meta-analysis results for most significant SNP on chromosomes 20 and 7</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140712026934448" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="2" rowspan="1" valign="middle" align="left">GWAS</th><th colspan="2" rowspan="1" valign="middle" align="left">Replication</th><th colspan="3" rowspan="1" valign="middle" align="left">Meta-analysis</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="left">Alleles<sup><sup>a</sup></sup></th><th rowspan="1" colspan="1" valign="middle" align="left">RAF<sup><sup>b</sup></sup></th><th rowspan="1" colspan="1" valign="middle" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>P</em>
<sub>het</sub>
<sup><sup>c</sup></sup>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">20</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1884302</td><td rowspan="1" colspan="1" valign="middle" align="left"><span class="underline">C</span>/T</td><td rowspan="1" colspan="1" valign="middle" align="left">0.3</td><td rowspan="1" colspan="1" valign="middle" align="left">4.58 (2.99–6.97)</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">4.30 (3.33–5.56)</td><td rowspan="1" colspan="1" valign="middle" align="left">4.4 × 10<sup>−31</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">4.38 (3.51–5.45)</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1 × 10<sup>−39</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">0.81</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">7</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10262453</td><td rowspan="1" colspan="1" valign="middle" align="left">C/<span class="underline">A</span></td><td rowspan="1" colspan="1" valign="middle" align="left">0.69</td><td rowspan="1" colspan="1" valign="middle" align="left">0.20 (0.11–0.34)</td><td rowspan="1" colspan="1" valign="middle" align="left">1.6 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">0.26 (0.18–0.37)</td><td rowspan="1" colspan="1" valign="middle" align="left">3.5 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">0.24 (0.17–0.32)</td><td rowspan="1" colspan="1" valign="middle" align="left">5.6 × 10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">0.44</td></tr></tbody></table></div><div id="largeobj_idm140712026934448" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3736322/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Minor allele/major allele.</div><div id="TFN2"><sup>b</sup>Risk allele frequency (RAF) calculated from the replication study controls.</div><div id="TFN3"><sup>c</sup><em>P</em> value for heterogeneity between the discovery and replication cohorts. Underlined bases are overtransmitted alleles. Chr., chromosome.</div></div></div><p id="P7">Considering the male predominance of sNSC, we performed a separate TDT analysis for 104 NHW trios with a male proband. The 21 SNPs on chromosomes 20 and 7 remained genome-wide significant, with an additional pseudo-autosomal marker reaching a low, but not genome–wide statistically significant <em>P</em>-value (rs2522623, <em>P</em> = 5.75 × 10<sup>−5</sup>) and thus did not meet our criteria for replication. This marker is located in protocadherin 11 X-linked (<em>PCDH11X</em>), which to date has not been reported to play a functional role in sNSC. Analysis of a larger cohort may demonstrate a stronger association to this locus.</p><p id="P8">From a total of 46 autosomal SNPs that showed significant and suggestive associations with <em>P</em> &lt; 1 × 10<sup>−5</sup>, 25 SNPs were prioritized for the replication study (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). In order to confirm the observed signals, we genotyped a NHW replication population of 172 unrelated cases with sNSC and 548 unaffected controls. We considered our data as a replication when the direction of effect of the alleles surpassing nominal significance was consistent between the discovery and replication datasets. All seven genotyped markers on chromosome 20 and four out of five markers on 7p14.3, with one SNP failing genotyping, were successfully replicated with meta-analysis <em>P-</em>values &lt;10<sup>−9</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). There was little or no evidence of heterogeneity of the odds ratio in the two different sample sets for the eleven replicated SNPs. Additionally, genotypic TDT (gTDT) analysis indicated the effect of SNPs was consistent with an additive model (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). We detected that, in most cases, transmitted minor alleles conferred a 2.36-4.38 fold risk of sNSC at the chromosome 20 markers, whereas the chromosome 7 transmitted minor alleles were associated with a decreased risk of sNSC (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). We were unable to replicate other signals from the GWAS TDT with <em>P</em> &lt; 1 × 10<sup>−5</sup>. These associations could be false positives or were missed due to differences between the study design and statistical methods used for the discovery and replication stages. We also relied on self-reported maternal ethnicity instead of ancestry informative markers for the replication cohort, which limited our ability to precisely assign ethnicity to the case-control study. The effect of gene-environment interaction or the relatively small sample of the replication cohort may also have affected the findings.</p><p id="P9">Imputation on borderline significant regions on chromosomes 3, 7, 8, 15, 22 was performed using BEAGLE v3.3.2 (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724018">26</a>) to identify additional genome-wide significant SNPs. Fifty-two intronic SNPs in <em>DLG1</em> reached genome-wide significance (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>), but we previously failed to replicate this association in our case/control sample (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). The lack of replication of a common variant may be due to our small sample size, differences in study design between our GWAS and replication datasets, or rare variants that contribute to the risk. Imputation was carried out using the 1000Genomes data for the reference panel, which does include some sequence variants with minor allele frequencies &lt; 1%. Given that sNSC has a prevalence of 1 in 5000, we suspect that rare functional variants not identified by 1000Genomes may be involved and sequencing of the associated region will be necessary. No statistically significant signals were observed after the imputation at the other loci.</p><p id="P10">Using these modestly-sized populations we discovered and replicated common genetic variants in at least two loci that contribute to sNSC. The chromosome 20 locus is located in the vicinity of the <em>BMP2</em> gene that plays a role in skeletal development. BMP2 is a member of the TGFβ superfamily that activates SMAD1, 5, and 8, and together with other osteogenic transcription factors such as RUNX2, ultimately regulating osteoblast development (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723990">27</a>).</p><p id="P11">An interesting observation in humans and animal models is that mutations with opposite effects in the same gene result in craniosynostosis (accelerated ossification) or parietal foramina/large fontanelles (deficient ossification) phenotypes. This holds true for mutations in at least 4 genes: <em>MSX2</em> (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724008">28</a>-<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724017">30</a>), <em>TWIST1</em> (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723998">31</a>-<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724016">32</a>), <em>RUNX2</em> (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723995">6</a>-<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724032">7</a>, <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724026">33</a>) and <em>Nell1</em> (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723987">34</a>-<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723996">35</a>) Consistent with this, gain-of-function mutations in <em>ALX4</em>, a gene associated with parietal foramina, have been identified in sNSC probands by our group (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723994">36</a>). Similarly, a variant in <em>BMP2</em>, p.S37A, has been associated with osteoporosis (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724019">37</a>): thus, if p.S37A results in loss-of-function, one can speculate that gain-of-function variants causing overexpression of <em>BMP2</em> would result in a phenotype of accelerated ossification and potentially CS. Additionally of interest, a limb specific regulatory element located 110 kb downstream of <em>BMP2</em> is mutated in brachydactyly type A2 (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724025">38</a>). Therefore, we predict that a skull-specific <em>BMP2</em> regulatory element is located within or near the association region. Also, non-coding SNPs can affect gene expression variation of one or many genes. Even though none of our top hits were found in the eQTL databases, this may be due to the fact that no human eQTL studies have been conducted in the skull suture tissues.</p><p id="P12">The GWAS and the replication analysis also showed evidence for association on chromosome 7p14 within an intronic region of <em>BBS9.</em> BBS9 is a component of the multiprotein complex BBSome which plays critical roles in intraflagellar transport, or moving cargo molecules in and out of cilia. Loss-of-function mutations in <em>BBS9</em> have been found in patients with Bardet-Biedl syndrome (BBS) (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724007">39</a>). BBS arises from defects of primary cilia that function as a platform for many important signaling events involving at least PDGFRα, hedgehog, epidermal growth factor, and 5-HT<sub>6</sub> serotonin receptor (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724005">40</a>-<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723993">44</a>). Although BBS does not present with suture phenotypes, it presents with a skeletal phenotype (polydactyly) and orofacial anomalies (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724001">39</a>). It is noteworthy that aberrant signaling of PDGFRα and epidermal growth factor or hedgehog has been implicated in craniofacial development (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724000">45</a>-<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724030">49</a>). In addition, it has been shown that FGF-signaling regulates cilia length and function during development of zebrafish and <em>Xenopus</em> (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723992">50</a>). Thus, it is possible that CS-causing FGFR gain-of-function mutations also work via aberrant assembly and/or signaling of cilia. It remains to be seen whether gain-of-function mutations or overexpression of <em>BBS9</em> would influence suture fusion. Three of the genes in the vicinity of the association on chromosome 7 (<em>RP9P</em>, <em>KBTBD2</em>, <em>FKBP9</em>) are pseudogenes and the remaining two, <em>RP9</em> and <em>NT5C3</em>, have no known role in skeletal development and cause retinitis pigmentosa and hemolytic anemia, respectively (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724024">51</a>-<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723991">52</a>). This region is located approximately 14 Mb telomeric of <em>TWIST1</em> and thus is unlikely to impact its function.</p><p id="P13">Our findings implicate variants within or near genes with relevant function in skeletal development, suggesting a major genetic effect for sNSC. A larger sample size will be needed to detect additional susceptibility genes with moderate genetic effects. The lack of Mendelian segregation suggests a more complex pattern of inheritance for sNSC. The causative variants may be of low frequency and in linkage disequilibrium with the common alleles associated with sNSC in this study. Another possibility is that one or more environmental factors interacting with genetic variants contribute to sNSC. Less likely, but still possible are oligogenic inheritance, gene-gene interactions, imprinting, epigenetic changes, or incomplete penetrance of the variants.</p><p id="P14" class="p">Biologically plausible candidate genes, <em>BMP2</em> and <em>BBS9</em>, both involved in skeletal development, were identified in our study. Functional studies of these genes, such as RNA sequencing and possibly tiling arrays of the regions to look for non-coding or microRNAs, are warranted. The potential contribution of these loci to other types of NSC is worthy of investigation. Our results represent the first major step toward deciphering the genetic etiology of the sNSC.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Methods</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Subjects</h3><p id="P15" class="p p-first">Informed consents were obtained from all patients and/or their parents. This study was approved by the Institutional Review Boards of the University of California, Davis, and other participating institutions and was conducted in accordance with institutional guidelines. Craniosynostosis (CS) was documented by computerized tomography of the skull and/or the protocol of the surgical correction. All patients with sagittal non-syndromic craniosynostosis (sNSC) were clinically assessed by a clinical geneticist and those with synostosis of additional sutures, associated extracranial congenital anomalies or developmental delays were excluded. Patients for both the initial GWAS study and the replication study were recruited within the USA.</p><p id="P16" class="p p-last">The case-control replication sample was comprised of archived, residual newborn blood spots from non-Hispanic white (NHW) infants diagnosed with sNSC (n=172) and control infants without a congenital malformation (n=548). Case infants were delivered by New York State residents from January 1, 1998 – December 31, 2005 and identified from the New York State Congenital Malformations Registry, a population-based birth defects surveillance program. Control infants were a random sample of live births in New York State delivered during the same time frame with the control-to-case ratio of approximately 3:1. Records for cases and controls were linked to records of the New York State Newborn Screening Program to obtain the respective blood spots and provided anonymously to study investigators. Use of the blood spots was approved by the Institutional Review Board of the New York State Department of Health and Mount Sinai School of Medicine.</p></div><div id="S3" class="sec"><h3 id="S3title">SNP genotyping and quality control in GWAS</h3><p id="P17" class="p p-first-last">Genomic DNA was extracted from whole blood or oral samples (468 whole blood, 139 mouth wash, 2 buccal and 24 Oragene). A total of 680 samples, including 16 blind duplicates and 13 HapMap controls (11 Utah residents of European ancestry, CEU and 2 Yoruban controls, YRI) were genotyped on the Illumina 1M Human Omni1-Quad array at the Center for Inherited Disease Research (<a href="http://www.cidr.jhmi.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cidr.jhmi.edu</a>). Genotypes for 1,006,422 SNPs and 133,997 CNV probes were released. The missing data rate was 0.39%, the blind duplicate rate was 99.99%, and the HapMap sample concordance rate was 99.7%. To check for cryptic relatedness and population stratification, 92,867 independent SNPs were selected based on a pair-wise inter-marker linkage disequilibrium r2 &lt; 0.4, minor allele frequency (MAF) &gt; 0.1 and genotype call rates &gt; 0.99. Pairwise identity-by-descent (IBD) calculations were performed using PLINK v1.0.7 (<a href="#R53" rid="R53" class=" bibr popnode">53</a>) and inconsistent samples were dropped, reducing the sample size to 646 individuals (213 trios). To confirm the self-reported ancestry of samples, principal components analysis (PCA) was performed using EIGENSOFT 3.0 (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724023">22</a>)]. A total of 145 families (451 individuals) were found to cluster with the 11 HapMap CEU controls, with no other well-defined clusters noted. A strict sNSC phenotype (no additional synostosis, congenital anomalies or developmental delay present in proband or affected sibling) further reduced our number of NHW trios to 130. SNPs were excluded from analysis if the call rate was less than 98%, they were discordant in more than one duplicate, had a Mendelian inconsistency in more than one trio, were missing in more than 10% of the samples, had a MAF of 0.0, and/or had a Hardy-Weinberg equilibrium P-value ≤ 1 × 10-5 with no heterozygotes. This reduced the number of SNPs available for analysis to 914,402 autosomal SNPs and 905 pseudo-autosomal SNPs.</p></div><div id="S4" class="sec"><h3 id="S4title">SNP selection and quality control in replication study</h3><p id="P18" class="p p-first-last">Genomic DNA was extracted from 3 mm dried blood spot punches using a sodium hydroxide precipitation protocol described previously (<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724012">54</a>). SNPs from the discovery stage were selected if P-value for association was ≤ 1.0 × 10-5 in the discovery stage. When multiple SNPs from the same region passed the P-value cutoff for replication, SNPs with the lowest P-value and/or lowest pairwise LD were selected. Using this screening process, from 46 autosomal SNPs with P-values ≤ 1.0 × 10-5, we selected to genotype 25 SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>) that tagged all 11 significant regions of interest (1 region each on 10 different chromosomes and 2 regions on chromosome 7). SNPs for the replication phase were genotyped using the ABI TaqMan allelic discrimination assay according to the manufacturer’s protocol (Applied Biosystems). SNP genotypes were called and discriminated using software S.D.S v3.4.4. Genotyping was conducted in duplicates and the concordance rate was 100%. Case and control samples were randomly distributed across the DNA plates. Disease status was unknown at the time of the genotyping experiments. Of the 25 SNPs from the original discovery phase that were selected for replication, two failed multiple TaqMan assay attempts even when conditions were modified (chr.7: rs17724206, failed and chr.22: rs4634034, call rate &lt;90%). For the remaining SNPs, the genotype call rates were &gt;95%.</p></div><div id="S5" class="sec"><h3 id="S5title">Sequencing</h3><p id="P19" class="p p-first">BMP2 sequencing was carried out on both exons for 149 sagittal NSC probands, of which 102 were part of the 130 proband cohort used in the initial discovery phase. Primer sequences for BMP2 are available in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6A</a>. Three variants were identified. Two of them, c.109T&gt;G (p.S37A) and c.570A&gt;T (p.R190S), were previously reported (rs2273073 and rs23576, respectively) and the allelic frequencies for the CEU population (obtained from HapMap Rel. 28, Phase II + III) was not significantly different than that found in our sequencing sample. The other SNP, c.393A&gt;T (p.R131S) was novel, present in the proband’s father and characterized as non-damaging based on bioinformatic predictions from SIFT (Sorting Tolerant from Intolerant, Fred Hutchinson Cancer Research Center, Seattle, web software, <a href="http://blocks.fhcrc.org/sift/SIFT.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://blocks.fhcrc.org/sift/SIFT.html</a>) and PPH (PolyPhen, Bork Group &amp; Sunyaev Lab, Harvard, web software, <a href="http://genetics.bwh.harvard.edu/pph" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genetics.bwh.harvard.edu/pph</a>).</p><p id="P20">Reverse transcriptase-PCR using control fetal calvarial osteoblast cDNA was attempted with all possible exonic primer combinations for predicted mRNA <a id="__tag_809267519" class="tag_hotlink" href="/nuccore/BC038533" ref="/nuccore/BC038533"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"BC038533","term_id":"23959007"}}</span>BC038533</a>, but did not produce PCR amplicons. <em>BBS9</em> sequencing was performed on cDNA from eight NSC fetal calvarial osteoblast lines. cDNA was created via RT-PCR (using Invitrogen SuperScript protocol and kit) on RNA extracted from available patients. <em>BBS9</em> sequencing primers are available in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6B</a>. Six variants were identified; three (c.1029A&gt;G G343G, c.2100G&gt;A L700L and c.1729A&gt;C I577L) were determined to be previously reported (rs25195153, rs115809567 and rs34209904 respectively) while the remaining three (BBS9_3′UTR +108 T&gt;A, BBS93′UTR +115 C&gt;T and c.1284C&gt;T) were not found in any searched databases (1000Genomes, HapMap, NCBI or MapBack). All six variants were indicated as non-damaging or benign by SIFT and PPH respectively.</p><p id="P21" class="p p-last">ClustalW (<a href="http://www.ebi.ac.uk/Tools/msa/clustalw2/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ebi.ac.uk/Tools/msa/clustalw2/</a>) alignment of BBS9 protein sequence for conservation analysis was performed in twelve available species - <a id="__tag_809267514" class="tag_hotlink" href="/protein/NP_001028776.1" ref="/protein/NP_001028776.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001028776.1","term_id":"75905801"}}</span>NP_001028776.1</a> (<em>Homo sapiens</em>), <a id="__tag_809267511" class="tag_hotlink" href="/protein/NP_848502.1" ref="/protein/NP_848502.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_848502.1","term_id":"189339274"}}</span>NP_848502.1</a> (<em>Mus musculus</em>), XP_002664838.1 (<em>Danio rerio</em>), XP_235942.4 (<em>Rattus norvegicus</em>), <a id="__tag_809267515" class="tag_hotlink" href="/protein/NP_001179782.1" ref="/protein/NP_001179782.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001179782.1","term_id":"300797295"}}</span>NP_001179782.1</a> (<em>Bos taurus</em>), XP_418843.3 (<em>Gallus gallus</em>), <a id="__tag_809267513" class="tag_hotlink" href="/protein/NP_001016821.1" ref="/protein/NP_001016821.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001016821.1","term_id":"62857447"}}</span>NP_001016821.1</a> (<em>Xenopus tropicalis</em>), <a id="__tag_809267512" class="tag_hotlink" href="/protein/XP_002957514.1" ref="/protein/XP_002957514.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"XP_002957514.1","term_id":"302851984"}}</span>XP_002957514.1</a> (<em>Volvox carteri f. nagariensis</em>), XP_001107028.1 (<em>Macaca mulatta</em>), XP_852259.1 (<em>Canis lupis familiaris),</em> XP_001168457.1 (<em>Pan troglodytes</em>), <a id="__tag_809267510" class="tag_hotlink" href="/protein/NP_001087412.1" ref="/protein/NP_001087412.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001087412.1","term_id":"148231376"}}</span>NP_001087412.1</a> (<em>Xenopus laevis</em>). Parental DNA was not available for sequencing analysis.</p></div><div id="S6" class="sec"><h3 id="S6title">Imputation</h3><p id="P22" class="p p-first-last">Ungenotyped markers 50 kb upstream and downstream from borderline significant regions from the TDT GWAS (chromosomes 3, 7, 8, 15 and 22) were imputed for the discovery sample using a trio design as implemented in BEAGLE v3.3.2 (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724020">26</a>). A reference panel of 379 European individuals (GBR, FIN, IBS, CEU and TWI) from the 1000 Genomes Project database was used. We first pre-phased the trio GWAS data and then imputed SNPs using a phased reference panel. There were 58, 19, 75, 37, and 30 genotyped SNPs and 1074, 234, 613, 276, and 431 imputed variants on chromosomes 3, 7, 8, 15, and 22 respectively. We carried out TDT tests on imputed variants of high quality (r<sup>2</sup> ≥ 0.3).</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">Statistical analysis</h3><p id="P23" class="p p-first">GWAS association tests were carried out using TDT analysis as implemented in PLINK v1.07 (<a href="#R53" rid="R53" class=" bibr popnode">53</a>). Manhattan plots were generated using the R script, and LD plots and r2 measures were obtained using HaploView v4.06 (<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724013">55</a>). All SNP positioning information was gathered from human genome version hg18 (March 2006; NCBI36).</p><p id="P24">Association mapping in the replication panel was performed by the <em>χ</em>2 test for allelic association using SAS/Genetics software version 9.2 (SAS Institute, Inc., Cary, NC). Direction of effect of markers surpassing nominal significance in the replication dataset was compared between both the discovery and replication datasets and three markers had opposite effects (chr.6: rs4716412; chr.7: rs7457112; and chr.16: rs4522429).</p><p id="P25">We searched for 2-way gene-gene interactions using the 100 most significant SNPs from the 130 NHW trio TDT analysis. We performed a genotypic TDT (gTDT) for the interaction of each pair of SNPs, excluding results from SNP pairs on the same chromosome due to long range LD (1247 tests). For each case-parents trio, we generated 15 pseudo-controls matched to each proband as a function of parental genotypes of the two considered SNPs. Pseudo-controls for each case are made of one of the two-locus genotypes that could have been, but were not transmitted to the case. Using these matched case and 15 pseudo-controls, we fitted a conditional logistic regression model (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724029">25</a>) to test for epistatic interactions that incorporate additive and dominance effects at the two loci. The ‘trio’ R package v1.4.24 was used to perform these tests (<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724022">56</a>).</p><p id="P26" class="p p-last">Conditional analysis was performed to identify additional SNPs potentially masked by the primary significant SNP using the conditional extended transmission disequilibrium test (ETDT) for trio data (<a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724027">57</a>) as implemented in UNPHASED v3.0.9 (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327723999">24</a>). Meta-analysis was performed by combining the trait-marker associations of the discovery TDT and replication case-control study using a fixed-effects model after testing for inter-study heterogeneity (<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_327724004">58</a>).</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>SuppInfo</h4><div class="sup-box half_rhythm" id="d36e1030"><a href="/pmc/articles/PMC3736322/bin/NIHMS52898-supplement-SuppInfo.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.1M, pdf)</sup></div></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Acknowledgements</h2><div class="sec"><p id="__p1">The authors are highly indebted to all families who contributed to this study. S.A.B. is partially funded through a Children’s Miracle Network Endowed Chair and through grants K23 DE00462, R03 DE016342, and R01 DE016886 from NIDCR/NIH and M01-RR00052 from NCRR/NIH and fully supported by Zlatka, Anton and Alec Boyadjiev. Partial funding was also obtained from grants to E.W.J. (CDC, 5 R01 DD000350), M.L.C. (R01 DE018227), A.O.M.W. (Wellcome Trust; #093329), P.A.R. (Centers for Disease Control and Prevention; #5U01DD000492), J.K. (National Institute Of Dental &amp; Craniofacial Research, NIH; # R21DE022419), J.L.M. (Intramural Research Program of the National Institutes of Health, Eunice Kennedy Shriver National Institute of Child Health and Human Development ; IRP # HHSN267200703431C; NICHD # N01-DK-7-3431), P.A.S-L. (Robert Wood Johnson Foundation; #3R37DE012711-13S1 and CHLA-USC Child Health Research Career Development Program; #NIH K12-HD05954), J.T.R. (NIDCR/NIH and the American Recovery and Reinvestment Act: #R01 DE018500 and 3R01 DE018500-02S1) and I.P. (National Center for Advancing Translational Sciences, NIH; #UL1TR000067). This project was also supported, in part, by the Division of Intramural Research Program of the National Human Genome Research Institute, National Institutes of Health (C.M.J., Y.K. and A.F.W.). Genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract number HHSN268200782096C. We are indebted to Bridget Wilson, Nisha Issac, Christopher Nauta, Erica Goude, Elijah Cherkez, Linda Peters, and Jeanette Harrison for patient recruitment, Corrine Boehm and Alan Scott for coordination of discovery phase genotyping, Christina Stevens, Armineh Stoner, Jia Lie Liu, Adam Gearhart, April Atkins and Emily McGrath for bench work, Douglas Mortlock for bioinformatic analysis and informative discussion and John Graham, Jonathan Bernstein, Jeffrey Marsh, Jayesh Panchal, Travis Tollefson and Maria Passos-Bueno for contributing clinical information and biospecimens for this project.</p></div></div><div id="idm140712061903376" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140712061903376title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P27" class="p p-first"><strong>Author Contributions</strong> C. M. J. – Co-first author with G. Y., performed statistical analysis, analyzed data and wrote the manuscript</p><p id="P28" class="p">G. Y. – Co-first author with C. M. J., performed experiments, analyzed data and wrote the manuscript</p><p id="P29" class="p">Y. K. – Performed statistical analyses, analyzed the data and contributed to writing of manuscript.</p><p id="P30" class="p">I. P. – Performed statistical analyses, analyzed the data and contributed to writing of manuscript</p><p id="P31" class="p">E. W. J. – Helped design the experiments, contributed materials and reagents and contributed to editing of manuscript</p><p id="P32" class="p">M. E. – Performed experiments</p><p id="P33" class="p">X. Y. – Performed experiments</p><p id="P34" class="p">E. A. – Performed experiments</p><p id="P35" class="p">L.S. – Performed experiments</p><p id="P36" class="p">M. L. C. – Analyzed the data and contributed materials and reagents</p><p id="P37" class="p">V. K. – Analyzed the data and contributed materials and reagents.</p><p id="P38" class="p">T. R. – Analyzed the data</p><p id="P39" class="p">S. A. W. – Contributed materials and reagents</p><p id="P40" class="p">A. O. M. W. – Analyzed the data and contributed to editing of manuscript</p><p id="P41" class="p">J. S. – Analyzed the data and contributed materials and reagents</p><p id="P42" class="p">J. T. R. – Analyzed the data and contributed to writing of manuscript.</p><p id="P43" class="p">Y. H. – Analyzed the data and contributed to writing of manuscript.</p><p id="P44" class="p">P. A. S-L. – Analyzed the data and contributed materials and reagents</p><p id="P45" class="p">M. F. B. – Analyzed the data, contributed materials and reagents, contributed to editing of the manuscript</p><p id="P46" class="p">C. M. D. – Contributed materials and reagents and helped with experimental design</p><p id="P47" class="p">J. L. M. – Contributed materials and reagents and helped with experimental design</p><p id="P48" class="p">M. C. – Contributed materials and reagents and helped with experimental design</p><p id="P49" class="p">P. A. R. – Contributed materials and reagents, helped with experimental design and contributed to writing of manuscript</p><p id="P50" class="p">D. M. K. – Contributed materials and reagents, helped with experimental design, and helped with editing of the manuscript</p><p id="P51" class="p">C. S. – Contributed materials and reagents and helped with experimental design</p><p id="P52" class="p">P. J. T. – Contributed materials and reagents, helped with experimental design, and helped with editing of the manuscript</p><p id="P53" class="p">O. D. K. – Contributed materials and reagents and helped with experimental design</p><p id="P54" class="p">J. B. – Contributed materials and reagents and helped with experimental design</p><p id="P55" class="p">M. Z-L. – Contributed materials and reagents and helped with experimental design</p><p id="P56" class="p">C. N. – Contributed materials and reagentsand helped with experimental design</p><p id="P57" class="p">J. K. – Supervised research and helped analyze data</p><p id="P58" class="p">A. F. W. – Supervised research, analyzed the data, and helped with writing of manuscript</p><p id="P59" class="p p-last">S. A. B – Principal investigator, designed and supervised research, recruited and evaluated participants, analyzed the data, and contributed to writing of manuscript</p><p></p></div></div><div id="idm140712026683472" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140712026683472title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Cohen MM.  <span class="ref-journal">Craniosynostosis: Diagnosis, Evaluation, and Management.</span> Oxford University Press; New York: 2000.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Craniosynostosis:+Diagnosis,+Evaluation,+and+Management&amp;author=MM+Cohen&amp;publication_year=2000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Johnson D, et al.  A novel mutation, Ala315Ser, in FGFR2: a gene-environment interaction leading to craniosynostosis? <span><span class="ref-journal">Eur J Hum Genet. </span>2000;<span class="ref-vol">8</span>(8):571–577.</span> [<a href="/pubmed/10951518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=A+novel+mutation,+Ala315Ser,+in+FGFR2:+a+gene-environment+interaction+leading+to+craniosynostosis?&amp;author=D+Johnson&amp;volume=8&amp;issue=8&amp;publication_year=2000&amp;pages=571-577&amp;pmid=10951518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Seto ML, et al.  Isolated sagittal and coronal craniosynostosis associated with TWIST box mutations. <span><span class="ref-journal">Am J Med Genet A. </span>2007;<span class="ref-vol">143</span>(7):678–86.</span> [<a href="/pubmed/17343269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Isolated+sagittal+and+coronal+craniosynostosis+associated+with+TWIST+box+mutations&amp;author=ML+Seto&amp;volume=143&amp;issue=7&amp;publication_year=2007&amp;pages=678-86&amp;pmid=17343269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Weber I, et al.  Molecular analyisis of 74 patients with craniosynostosis. <span><span class="ref-journal">Eur J Hum Genet. </span>2001;<span class="ref-vol">9</span>(Sup.1):P0409, 179.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Molecular+analyisis+of+74+patients+with+craniosynostosis&amp;author=I+Weber&amp;volume=9&amp;issue=Sup.1&amp;publication_year=2001&amp;pages=P0409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Merrill AE, et al.  Cell mixing at a neural crest-mesoderm boundary and deficient ephrin-Eph signaling in the pathogenesis of craniosynostosis. <span><span class="ref-journal">Hum Mol Genet. </span>2006;<span class="ref-vol">15</span>(8):1319–28.</span> [<a href="/pubmed/16540516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Cell+mixing+at+a+neural+crest-mesoderm+boundary+and+deficient+ephrin-Eph+signaling+in+the+pathogenesis+of+craniosynostosis&amp;author=AE+Merrill&amp;volume=15&amp;issue=8&amp;publication_year=2006&amp;pages=1319-28&amp;pmid=16540516&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Wilkie AO, et al.  Clinical dividends from the molecular genetic diagnosis of craniosynostosis. <span><span class="ref-journal">Am J Med Genet A. </span>2007;<span class="ref-vol">143A</span>(16):1941–9.</span> [<a href="/pubmed/17621648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Clinical+dividends+from+the+molecular+genetic+diagnosis+of+craniosynostosis&amp;author=AO+Wilkie&amp;volume=143A&amp;issue=16&amp;publication_year=2007&amp;pages=1941-9&amp;pmid=17621648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Mefford HC, et al.  Copy number variation analysis in single-suture craniosynostosis: multiple rare variants including RUNX2 duplication in two cousins with metopic craniosynostosis. <span><span class="ref-journal">Am J Med Genet A. </span>2010;<span class="ref-vol">152A</span>(9):2203–10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3104131/">PMC free article</a>]</span> [<a href="/pubmed/20683987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Copy+number+variation+analysis+in+single-suture+craniosynostosis:+multiple+rare+variants+including+RUNX2+duplication+in+two+cousins+with+metopic+craniosynostosis&amp;author=HC+Mefford&amp;volume=152A&amp;issue=9&amp;publication_year=2010&amp;pages=2203-10&amp;pmid=20683987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Vissers LE, et al.  Heterozygous mutations of FREM1 are associated with an increased risk of isolated metopic craniosynostosis in humans and mice. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>(9):e1002278.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3169541/">PMC free article</a>]</span> [<a href="/pubmed/21931569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Heterozygous+mutations+of+FREM1+are+associated+with+an+increased+risk+of+isolated+metopic+craniosynostosis+in+humans+and+mice&amp;author=LE+Vissers&amp;volume=7&amp;issue=9&amp;publication_year=2011&amp;pages=e1002278&amp;pmid=21931569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Kim S-D, et al.  Leucine-rich repeat, immunoglobulin-like and transmembrane domain 3 (LRIT3) is a modulator of FGFR1. <span><span class="ref-journal">Federation of European Biochemical Societies. </span>2012;<span class="ref-vol">586</span>(10):1516–1521.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3372856/">PMC free article</a>]</span> [<a href="/pubmed/22673519" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Federation+of+European+Biochemical+Societies&amp;title=Leucine-rich+repeat,+immunoglobulin-like+and+transmembrane+domain+3+(LRIT3)+is+a+modulator+of+FGFR1&amp;author=S-D+Kim&amp;volume=586&amp;issue=10&amp;publication_year=2012&amp;pages=1516-1521&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Melville H, et al.  Genetic basis of potential therapeutic strategies for craniosynostosis. <span><span class="ref-journal">Am J Med Genet A. </span>2010;<span class="ref-vol">152A</span>(12):3007–15.</span> [<a href="/pubmed/21082653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Genetic+basis+of+potential+therapeutic+strategies+for+craniosynostosis&amp;author=H+Melville&amp;volume=152A&amp;issue=12&amp;publication_year=2010&amp;pages=3007-15&amp;pmid=21082653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Wilkie AO, et al.  Prevalence and complications of single-gene and chromosomal disorders in craniosynostosis. <span><span class="ref-journal">Pediatrics. </span>2010;<span class="ref-vol">126</span>(2):e391–400.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3535761/">PMC free article</a>]</span> [<a href="/pubmed/20643727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Prevalence+and+complications+of+single-gene+and+chromosomal+disorders+in+craniosynostosis&amp;author=AO+Wilkie&amp;volume=126&amp;issue=2&amp;publication_year=2010&amp;pages=e391-400&amp;pmid=20643727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Passos-Bueno MR, et al.  Genetics of craniosynostosis: genes, syndromes, mutations and genotype-phenotype correlations. <span><span class="ref-journal">Front Oral Biol. </span>2008;<span class="ref-vol">12</span>:107–43.</span> [<a href="/pubmed/18391498" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Oral+Biol&amp;title=Genetics+of+craniosynostosis:+genes,+syndromes,+mutations+and+genotype-phenotype+correlations&amp;author=MR+Passos-Bueno&amp;volume=12&amp;publication_year=2008&amp;pages=107-43&amp;pmid=18391498&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Kimonis V, et al.  Genetics of craniosynostosis. <span><span class="ref-journal">Semin Pediatr Neurol. </span>2007;<span class="ref-vol">14</span>(3):150–61.</span> [<a href="/pubmed/17980312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin+Pediatr+Neurol&amp;title=Genetics+of+craniosynostosis&amp;author=V+Kimonis&amp;volume=14&amp;issue=3&amp;publication_year=2007&amp;pages=150-61&amp;pmid=17980312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Kolar JC. An epidemiological study of nonsyndromal craniosynostoses. <span><span class="ref-journal">J Craniofac Surg. </span>2011;<span class="ref-vol">22</span>(1):47–9.</span> [<a href="/pubmed/21187784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Craniofac+Surg&amp;title=An+epidemiological+study+of+nonsyndromal+craniosynostoses&amp;author=JC+Kolar&amp;volume=22&amp;issue=1&amp;publication_year=2011&amp;pages=47-9&amp;pmid=21187784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Lajeunie E, et al.  Genetic considerations in nonsyndromic midline craniosynostoses: a study of twins and their families. <span><span class="ref-journal">J Neurosurg. </span>2005;<span class="ref-vol">103</span>(4 Suppl):353–6.</span> [<a href="/pubmed/16270687" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosurg&amp;title=Genetic+considerations+in+nonsyndromic+midline+craniosynostoses:+a+study+of+twins+and+their+families&amp;author=E+Lajeunie&amp;volume=103&amp;issue=4+Suppl&amp;publication_year=2005&amp;pages=353-6&amp;pmid=16270687&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Lajeunie E, et al.  Syndromal and nonsyndromal primary trigonocephaly: analysis of a series of 237 patients. <span><span class="ref-journal">Am J Med Genet. </span>1998;<span class="ref-vol">75</span>(2):211–215.</span> [<a href="/pubmed/9450889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Syndromal+and+nonsyndromal+primary+trigonocephaly:+analysis+of+a+series+of+237+patients&amp;author=E+Lajeunie&amp;volume=75&amp;issue=2&amp;publication_year=1998&amp;pages=211-215&amp;pmid=9450889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Lajeunie E, et al.  Genetic study of scaphocephaly. <span><span class="ref-journal">Am J Med Genet. </span>1996;<span class="ref-vol">62</span>(3):282–5.</span> [<a href="/pubmed/8882788" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Genetic+study+of+scaphocephaly&amp;author=E+Lajeunie&amp;volume=62&amp;issue=3&amp;publication_year=1996&amp;pages=282-5&amp;pmid=8882788&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Sanchez-Lara PA, et al.  Fetal constraint as a potential risk factor for craniosynostosis. <span><span class="ref-journal">Am J Med Genet A. </span>2010;<span class="ref-vol">152A</span>(2):394–400.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2815148/">PMC free article</a>]</span> [<a href="/pubmed/20101684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Fetal+constraint+as+a+potential+risk+factor+for+craniosynostosis&amp;author=PA+Sanchez-Lara&amp;volume=152A&amp;issue=2&amp;publication_year=2010&amp;pages=394-400&amp;pmid=20101684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Zeiger JS, et al.  Genetic and environmental risk factors for sagittal craniosynostosis. <span><span class="ref-journal">J Craniofac Surg. </span>2002;<span class="ref-vol">13</span>(5):602–6.</span> [<a href="/pubmed/12218784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Craniofac+Surg&amp;title=Genetic+and+environmental+risk+factors+for+sagittal+craniosynostosis&amp;author=JS+Zeiger&amp;volume=13&amp;issue=5&amp;publication_year=2002&amp;pages=602-6&amp;pmid=12218784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Gardner JS, et al.  Maternal exposure to prescription and non-prescription pharmaceuticals or drugs of abuse and risk of craniosynostosis. <span><span class="ref-journal">Int J Epidemiol. </span>1998;<span class="ref-vol">27</span>(1):64–7.</span> [<a href="/pubmed/9563695" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Maternal+exposure+to+prescription+and+non-prescription+pharmaceuticals+or+drugs+of+abuse+and+risk+of+craniosynostosis&amp;author=JS+Gardner&amp;volume=27&amp;issue=1&amp;publication_year=1998&amp;pages=64-7&amp;pmid=9563695&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Kallen K. Maternal smoking and craniosynostosis. <span><span class="ref-journal">Teratology. </span>1999;<span class="ref-vol">60</span>(3):146–150.</span> [<a href="/pubmed/10471899" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Teratology&amp;title=Maternal+smoking+and+craniosynostosis&amp;author=K+Kallen&amp;volume=60&amp;issue=3&amp;publication_year=1999&amp;pages=146-150&amp;pmid=10471899&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>(12):e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;issue=12&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Dudbridge F. Pedigree disequilibrium tests for multilocus haplotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2003;<span class="ref-vol">25</span>(2):115–21.</span> [<a href="/pubmed/12916020" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Pedigree+disequilibrium+tests+for+multilocus+haplotypes&amp;author=F+Dudbridge&amp;volume=25&amp;issue=2&amp;publication_year=2003&amp;pages=115-21&amp;pmid=12916020&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Cordell HJ. Epistasis: what it means, what it doesn’t mean, and statistical methods to detect it in humans. <span><span class="ref-journal">Hum Mol Genet. </span>2002;<span class="ref-vol">11</span>(20):2463–8.</span> [<a href="/pubmed/12351582" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Epistasis:+what+it+means,+what+it+doesn’t+mean,+and+statistical+methods+to+detect+it+in+humans&amp;author=HJ+Cordell&amp;volume=11&amp;issue=20&amp;publication_year=2002&amp;pages=2463-8&amp;pmid=12351582&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Browning BL, Browning SR. A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>(2):210–23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668004/">PMC free article</a>]</span> [<a href="/pubmed/19200528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+unified+approach+to+genotype+imputation+and+haplotype-phase+inference+for+large+data+sets+of+trios+and+unrelated+individuals&amp;author=BL+Browning&amp;author=SR+Browning&amp;volume=84&amp;issue=2&amp;publication_year=2009&amp;pages=210-23&amp;pmid=19200528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Rosen V. BMP2 signaling in bone development and repair. <span><span class="ref-journal">Cytokine Growth Factor Rev. </span>2009;<span class="ref-vol">20</span>(5-6):475–80.</span> [<a href="/pubmed/19892583" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine+Growth+Factor+Rev&amp;title=BMP2+signaling+in+bone+development+and+repair&amp;author=V+Rosen&amp;volume=20&amp;issue=5-6&amp;publication_year=2009&amp;pages=475-80&amp;pmid=19892583&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Jabs E, et al.  A mutation in the homeodomain of the human MSX2 gene in a family affected with autosomal dominant craniosynostosis. <span><span class="ref-journal">Cell. </span>1993;<span class="ref-vol">75</span>:443–450.</span> [<a href="/pubmed/8106171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+mutation+in+the+homeodomain+of+the+human+MSX2+gene+in+a+family+affected+with+autosomal+dominant+craniosynostosis&amp;author=E+Jabs&amp;volume=75&amp;publication_year=1993&amp;pages=443-450&amp;pmid=8106171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Spruijt L, et al.  A novel mutation in the MSX2 gene in a family with foramina parietalia permagna (FPP) <span><span class="ref-journal">Am J Med Genet A. </span>2005;<span class="ref-vol">139</span>(1):45–7.</span> [<a href="/pubmed/16222674" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=A+novel+mutation+in+the+MSX2+gene+in+a+family+with+foramina+parietalia+permagna+(FPP)&amp;author=L+Spruijt&amp;volume=139&amp;issue=1&amp;publication_year=2005&amp;pages=45-7&amp;pmid=16222674&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Wilkie A, et al.  Functional haploinsufficiency of the human homeobox gene MSX2 causes defects in skull ossification. <span><span class="ref-journal">Nat Genet. </span>2000;<span class="ref-vol">24</span>(4):330–331.</span> [<a href="/pubmed/10742103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Functional+haploinsufficiency+of+the+human+homeobox+gene+MSX2+causes+defects+in+skull+ossification&amp;author=A+Wilkie&amp;volume=24&amp;issue=4&amp;publication_year=2000&amp;pages=330-331&amp;pmid=10742087&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Howard T, et al.  Mutations in TWIST, a basic helix-loop-helix transcription factor, in Saethre-Chotzen syndrome. <span><span class="ref-journal">Nat Genet. </span>1997;<span class="ref-vol">15</span>(1):36–41.</span> [<a href="/pubmed/8988166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mutations+in+TWIST,+a+basic+helix-loop-helix+transcription+factor,+in+Saethre-Chotzen+syndrome&amp;author=T+Howard&amp;volume=15&amp;issue=1&amp;publication_year=1997&amp;pages=36-41&amp;pmid=8988166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Stankiewicz P, et al.  Phenotypic findings due to trisomy 7p15.3-pter including the TWIST locus. <span><span class="ref-journal">Am J Med Genet. </span>2001;<span class="ref-vol">103</span>(1):56–62.</span> [<a href="/pubmed/11562935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Phenotypic+findings+due+to+trisomy+7p15.3-pter+including+the+TWIST+locus&amp;author=P+Stankiewicz&amp;volume=103&amp;issue=1&amp;publication_year=2001&amp;pages=56-62&amp;pmid=11562935&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Mundlos S, et al.  Mutations involving the transcription factor CBFA1 cause cleidocranial dysplasia. <span><span class="ref-journal">Cell. </span>1997;<span class="ref-vol">89</span>:773–779.</span> [<a href="/pubmed/9182765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Mutations+involving+the+transcription+factor+CBFA1+cause+cleidocranial+dysplasia&amp;author=S+Mundlos&amp;volume=89&amp;publication_year=1997&amp;pages=773-779&amp;pmid=9182765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Desai J, et al.  Nell1-deficient mice have reduced expression of extracellular matrix proteins causing cranial and vertebral defects. <span><span class="ref-journal">Hum Mol Genet. </span>2006;<span class="ref-vol">15</span>(8):1329–41.</span> [<a href="/pubmed/16537572" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Nell1-deficient+mice+have+reduced+expression+of+extracellular+matrix+proteins+causing+cranial+and+vertebral+defects&amp;author=J+Desai&amp;volume=15&amp;issue=8&amp;publication_year=2006&amp;pages=1329-41&amp;pmid=16537572&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Zhang X, et al.  The role of NELL-1, a growth factor associated with craniosynostosis, in promoting bone regeneration. <span><span class="ref-journal">J Dent Res. </span>2010;<span class="ref-vol">89</span>(9):865–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2959101/">PMC free article</a>]</span> [<a href="/pubmed/20647499" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Dent+Res&amp;title=The+role+of+NELL-1,+a+growth+factor+associated+with+craniosynostosis,+in+promoting+bone+regeneration&amp;author=X+Zhang&amp;volume=89&amp;issue=9&amp;publication_year=2010&amp;pages=865-78&amp;pmid=20647499&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Yagnik G, et al.  ALX4 gain-of-function mutations in nonsyndromic craniosynostosis. <span><span class="ref-journal">Hum Mutat. </span>2012</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3495992/">PMC free article</a>]</span> [<a href="/pubmed/22829454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=ALX4+gain-of-function+mutations+in+nonsyndromic+craniosynostosis&amp;author=G+Yagnik&amp;publication_year=2012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Styrkarsdottir U, et al.  Linkage of osteoporosis to chromosome 20p12 and association to BMP2. <span><span class="ref-journal">PLoS Biol. </span>2003;<span class="ref-vol">1</span>(3):E69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC270020/">PMC free article</a>]</span> [<a href="/pubmed/14691541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Linkage+of+osteoporosis+to+chromosome+20p12+and+association+to+BMP2&amp;author=U+Styrkarsdottir&amp;volume=1&amp;issue=3&amp;publication_year=2003&amp;pages=E69&amp;pmid=14691541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Dathe K, et al.  Duplications involving a conserved regulatory element downstream of BMP2 are associated with brachydactyly type A2. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>(4):483–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2667973/">PMC free article</a>]</span> [<a href="/pubmed/19327734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Duplications+involving+a+conserved+regulatory+element+downstream+of+BMP2+are+associated+with+brachydactyly+type+A2&amp;author=K+Dathe&amp;volume=84&amp;issue=4&amp;publication_year=2009&amp;pages=483-92&amp;pmid=19327734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Tobin JL, Beales PL. Bardet-Biedl syndrome: beyond the cilium. <span><span class="ref-journal">Pediatr Nephrol. </span>2007;<span class="ref-vol">22</span>(7):926–36.</span> [<a href="/pubmed/17357787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr+Nephrol&amp;title=Bardet-Biedl+syndrome:+beyond+the+cilium&amp;author=JL+Tobin&amp;author=PL+Beales&amp;volume=22&amp;issue=7&amp;publication_year=2007&amp;pages=926-36&amp;pmid=17357787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Marshall WF, Nonaka S. Cilia: tuning in to the cell’s antenna. <span><span class="ref-journal">Curr Biol. </span>2006;<span class="ref-vol">16</span>(15):R604–14.</span> [<a href="/pubmed/16890522" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;title=Cilia:+tuning+in+to+the+cell’s+antenna&amp;author=WF+Marshall&amp;author=S+Nonaka&amp;volume=16&amp;issue=15&amp;publication_year=2006&amp;pages=R604-14&amp;pmid=16890522&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Schneider L, et al.  PDGFRalphaalpha signaling is regulated through the primary cilium in fibroblasts. <span><span class="ref-journal">Curr Biol. </span>2005;<span class="ref-vol">15</span>(20):1861–6.</span> [<a href="/pubmed/16243034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;title=PDGFRalphaalpha+signaling+is+regulated+through+the+primary+cilium+in+fibroblasts&amp;author=L+Schneider&amp;volume=15&amp;issue=20&amp;publication_year=2005&amp;pages=1861-6&amp;pmid=16243034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Haycraft CJ, et al.  Gli2 and Gli3 localize to cilia and require the intraflagellar transport protein polaris for processing and function. <span><span class="ref-journal">PLoS Genet. </span>2005;<span class="ref-vol">1</span>(4):e53.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1270009/">PMC free article</a>]</span> [<a href="/pubmed/16254602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Gli2+and+Gli3+localize+to+cilia+and+require+the+intraflagellar+transport+protein+polaris+for+processing+and+function&amp;author=CJ+Haycraft&amp;volume=1&amp;issue=4&amp;publication_year=2005&amp;pages=e53&amp;pmid=16254602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Ma R, et al.  PKD2 functions as an epidermal growth factor-activated plasma membrane channel. <span><span class="ref-journal">Mol Cell Biol. </span>2005;<span class="ref-vol">25</span>(18):8285–98.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1234340/">PMC free article</a>]</span> [<a href="/pubmed/16135816" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=PKD2+functions+as+an+epidermal+growth+factor-activated+plasma+membrane+channel&amp;author=R+Ma&amp;volume=25&amp;issue=18&amp;publication_year=2005&amp;pages=8285-98&amp;pmid=16135816&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Brailov I, et al.  Localization of 5-HT(6) receptors at the plasma membrane of neuronal cilia in the rat brain. <span><span class="ref-journal">Brain Res. </span>2000;<span class="ref-vol">872</span>(1-2):271–5.</span> [<a href="/pubmed/10924708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res&amp;title=Localization+of+5-HT(6)+receptors+at+the+plasma+membrane+of+neuronal+cilia+in+the+rat+brain&amp;author=I+Brailov&amp;volume=872&amp;issue=1-2&amp;publication_year=2000&amp;pages=271-5&amp;pmid=10924708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Jenkins D, et al.  RAB23 mutations in Carpenter syndrome imply an unexpected role for hedgehog signaling in cranial-suture development and obesity. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>(6):1162–70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1867103/">PMC free article</a>]</span> [<a href="/pubmed/17503333" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=RAB23+mutations+in+Carpenter+syndrome+imply+an+unexpected+role+for+hedgehog+signaling+in+cranial-suture+development+and+obesity&amp;author=D+Jenkins&amp;volume=80&amp;issue=6&amp;publication_year=2007&amp;pages=1162-70&amp;pmid=17503333&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Ehlen HW, Buelens LA, Vortkamp A. Hedgehog signaling in skeletal development. <span><span class="ref-journal">Birth Defects Res C Embryo Today. </span>2006;<span class="ref-vol">78</span>(3):267–79.</span> [<a href="/pubmed/17061262" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res+C+Embryo+Today&amp;title=Hedgehog+signaling+in+skeletal+development&amp;author=HW+Ehlen&amp;author=LA+Buelens&amp;author=A+Vortkamp&amp;volume=78&amp;issue=3&amp;publication_year=2006&amp;pages=267-79&amp;pmid=17061262&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Miettinen PJ, et al.  Epidermal growth factor receptor function is necessary for normal craniofacial development and palate closure. <span><span class="ref-journal">Nat Genet. </span>1999;<span class="ref-vol">22</span>(1):69–73.</span> [<a href="/pubmed/10319864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Epidermal+growth+factor+receptor+function+is+necessary+for+normal+craniofacial+development+and+palate+closure&amp;author=PJ+Miettinen&amp;volume=22&amp;issue=1&amp;publication_year=1999&amp;pages=69-73&amp;pmid=10319864&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Ding H, et al.  A specific requirement for PDGF-C in palate formation and PDGFR-alpha signaling. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>(10):1111–6.</span> [<a href="/pubmed/15361870" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+specific+requirement+for+PDGF-C+in+palate+formation+and+PDGFR-alpha+signaling&amp;author=H+Ding&amp;volume=36&amp;issue=10&amp;publication_year=2004&amp;pages=1111-6&amp;pmid=15361870&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Choi DS, et al.  5-HT2B receptor-mediated serotonin morphogenetic functions in mouse cranial neural crest and myocardiac cells. <span><span class="ref-journal">Development. </span>1997;<span class="ref-vol">124</span>(9):1745–55.</span> [<a href="/pubmed/9165122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=5-HT2B+receptor-mediated+serotonin+morphogenetic+functions+in+mouse+cranial+neural+crest+and+myocardiac+cells&amp;author=DS+Choi&amp;volume=124&amp;issue=9&amp;publication_year=1997&amp;pages=1745-55&amp;pmid=9165122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Neugebauer JM, et al.  FGF signalling during embryo development regulates cilia length in diverse epithelia. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">458</span>(7238):651–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2688717/">PMC free article</a>]</span> [<a href="/pubmed/19242413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=FGF+signalling+during+embryo+development+regulates+cilia+length+in+diverse+epithelia&amp;author=JM+Neugebauer&amp;volume=458&amp;issue=7238&amp;publication_year=2009&amp;pages=651-4&amp;pmid=19242413&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Keen TJ, et al.  Mutations in a protein target of the Pim-1 kinase associated with the RP9 form of autosomal dominant retinitis pigmentosa. <span><span class="ref-journal">Eur J Hum Genet. </span>2002;<span class="ref-vol">10</span>(4):245–9.</span> [<a href="/pubmed/12032732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Mutations+in+a+protein+target+of+the+Pim-1+kinase+associated+with+the+RP9+form+of+autosomal+dominant+retinitis+pigmentosa&amp;author=TJ+Keen&amp;volume=10&amp;issue=4&amp;publication_year=2002&amp;pages=245-9&amp;pmid=12032732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Marinaki AM, et al.  Genetic basis of hemolytic anemia caused by pyrimidine 5′ nucleotidase deficiency. <span><span class="ref-journal">Blood. </span>2001;<span class="ref-vol">97</span>(11):3327–32.</span> [<a href="/pubmed/11369620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genetic+basis+of+hemolytic+anemia+caused+by+pyrimidine+5′+nucleotidase+deficiency&amp;author=AM+Marinaki&amp;volume=97&amp;issue=11&amp;publication_year=2001&amp;pages=3327-32&amp;pmid=11369620&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">McDonald-McGinn D, et al.  Molecular characterization of Pfeiffer syndrome: implications for prognosis and genetic counseling. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">73</span>(5):212, A253.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Molecular+characterization+of+Pfeiffer+syndrome:+implications+for+prognosis+and+genetic+counseling&amp;author=D+McDonald-McGinn&amp;volume=73&amp;issue=5&amp;publication_year=2003&amp;pages=212&amp;pmid=12865991&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Mills JL, et al.  Folate and vitamin B12-related genes and risk for omphalocele. <span><span class="ref-journal">Hum Genet. </span>2012;<span class="ref-vol">131</span>(5):739–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3374579/">PMC free article</a>]</span> [<a href="/pubmed/22116453" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Folate+and+vitamin+B12-related+genes+and+risk+for+omphalocele&amp;author=JL+Mills&amp;volume=131&amp;issue=5&amp;publication_year=2012&amp;pages=739-46&amp;pmid=22116453&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Barrett JC, et al.  Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>(2):263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;volume=21&amp;issue=2&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Schwender H, et al.  Rapid Testing of SNPs and Gene-Environment Interactions in Case-Parent Trio Data Based on Exact Analytic Parameter Estimation. <span><span class="ref-journal">Biometrics. </span>2011</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3387527/">PMC free article</a>]</span> [<a href="/pubmed/22150644" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Rapid+Testing+of+SNPs+and+Gene-Environment+Interactions+in+Case-Parent+Trio+Data+Based+on+Exact+Analytic+Parameter+Estimation&amp;author=H+Schwender&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Koeleman BP, et al.  Adaptation of the extended transmission/disequilibrium test to distinguish disease associations of multiple loci: the Conditional Extended Transmission/Disequilibrium Test. <span><span class="ref-journal">Annals of Human Genetics. </span>2000;<span class="ref-vol">64</span>(Pt 3):207–13.</span> [<a href="/pubmed/11246472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+Human+Genetics&amp;title=Adaptation+of+the+extended+transmission/disequilibrium+test+to+distinguish+disease+associations+of+multiple+loci:+the+Conditional+Extended+Transmission/Disequilibrium+Test&amp;author=BP+Koeleman&amp;volume=64&amp;issue=Pt+3&amp;publication_year=2000&amp;pages=207-13&amp;pmid=11246472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Kazeem GR, Farrall M. Integrating case-control and TDT studies. <span><span class="ref-journal">Annals of Human Genetics. </span>2005;<span class="ref-vol">69</span>:329–335.</span> [<a href="/pubmed/15845037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+Human+Genetics&amp;title=Integrating+case-control+and+TDT+studies&amp;author=GR+Kazeem&amp;author=M+Farrall&amp;volume=69&amp;publication_year=2005&amp;pages=329-335&amp;pmid=15845037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3736322&amp;issue-id=226059&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140712026236688title">Abstract</a></li><li><a href="#S1title">Methods</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S8title">Acknowledgements</a></li><li><a href="#idm140712061903376title">Footnotes</a></li><li><a href="#idm140712026683472title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3736322/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3736322/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3736322/pdf/emss-52898.pdf">PDF (2.2M)</a></li> | <li><a href="#" data-citationid="PMC3736322" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3736322%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3736322%2F&amp;text=A%20genome-wide%20association%20study%20identifies%20susceptibility%20loci%20for%20non-syndromic%20sagittal%20craniosynostosis%20near%20BMP2%20and%20within%20BBS9" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3736322%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23160099" data-aiid="3736322" data-aid="3736322" data-iid="226059" data-domainid="358" data-domain="wtpa" data-accid="PMC3736322" data-md5="6f67f0c71ff8dce7c53d212589c928f2"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:52:26-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CC83EDD54AB0100000000061203F9&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3736322%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3736322/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>