<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3743946/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Generalization of Variants Identified by Genome-wide Association Studies for Electrocardiographic Traits in African Americans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3743946/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Annals of human genetics"><meta name="citation_title" content="Generalization of Variants Identified by Genome-wide Association Studies for Electrocardiographic Traits in African Americans"><meta name="citation_authors" content="Janina M. Jeff, Marylyn D. Ritchie, Joshua C. Denny, Abel N. Kho, Andrea H. Ramirez, David Crosslin, Loren Armstrong, Melissa A. Basford, Wendy A. Wolf, Jennifer A. Pacheco, Rex L. Chisholm, Dan M. Roden, M. Geoffrey Hayes, Dana C. Crawford"><meta name="citation_date" content="July 2013"><meta name="citation_issue" content="4"><meta name="citation_volume" content="77"><meta name="citation_firstpage" content="321"><meta name="citation_doi" content="10.1111/ahg.12023"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3743946/?report=abstract"><meta name="citation_pmid" content="23534349"><meta name="DC.Title" content="Generalization of Variants Identified by Genome-wide Association Studies for Electrocardiographic Traits in African Americans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Janina M. Jeff"><meta name="DC.Contributor" content="Marylyn D. Ritchie"><meta name="DC.Contributor" content="Joshua C. Denny"><meta name="DC.Contributor" content="Abel N. Kho"><meta name="DC.Contributor" content="Andrea H. Ramirez"><meta name="DC.Contributor" content="David Crosslin"><meta name="DC.Contributor" content="Loren Armstrong"><meta name="DC.Contributor" content="Melissa A. Basford"><meta name="DC.Contributor" content="Wendy A. Wolf"><meta name="DC.Contributor" content="Jennifer A. Pacheco"><meta name="DC.Contributor" content="Rex L. Chisholm"><meta name="DC.Contributor" content="Dan M. Roden"><meta name="DC.Contributor" content="M. Geoffrey Hayes"><meta name="DC.Contributor" content="Dana C. Crawford"><meta name="DC.Date" content="2013 Jul"><meta name="DC.Identifier" content="10.1111/ahg.12023"><meta name="DC.Language" content="en"><meta property="og:title" content="Generalization of Variants Identified by Genome-wide Association Studies for Electrocardiographic Traits in African Americans"><meta property="og:type" content="article"><meta property="og:description" content="Electrocardiographic (ECG) measurements vary by ancestry. Genome-wide association studies (GWAS) have identified loci that contribute to ECG measurements; however most are performed in Europeans collected from population-based cohorts or surveys. The ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3743946/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3743946/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3743946/pdf/nihms463822.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C217BDD54ED51000000000A140681.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3743946%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="nlmcatalog" class="last">NLM Catalog</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3743946</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Ann Hum Genet</a></span>. Author manuscript; available in PMC 2014 Jul 30.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23534349" target="pmc_ext" ref="reftype=publisher&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Ann Hum Genet. 2013 Jul; 77(4): 321–332. </span></a></div><span class="fm-vol-iss-date">Published online 2013 Mar 28. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1111%2Fahg.12023" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1111/ahg.12023</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3743946</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS463822</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23534349">23534349</a></div></div></div></div><h1 class="content-title">Generalization of Variants Identified by Genome-wide Association Studies for Electrocardiographic Traits in African Americans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Jeff%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322329988688" co-class="co-affbox">Janina M. Jeff</a>,<sup>1</sup> <a href="/pubmed/?term=Ritchie%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322332231040" co-class="co-affbox">Marylyn D. Ritchie</a>,<sup>8</sup> <a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322331768896" co-class="co-affbox">Joshua C. Denny</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Kho%20AN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322331766176" co-class="co-affbox">Abel N. Kho</a>,<sup>4</sup> <a href="/pubmed/?term=Ramirez%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322331764192" co-class="co-affbox">Andrea H. Ramirez</a>,<sup>2</sup> <a href="/pubmed/?term=Crosslin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322331762208" co-class="co-affbox">David Crosslin</a>,<sup>9</sup> <a href="/pubmed/?term=Armstrong%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322331760224" co-class="co-affbox">Loren Armstrong</a>,<sup>5</sup> <a href="/pubmed/?term=Basford%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322331758240" co-class="co-affbox">Melissa A. Basford</a>,<sup>6</sup> <a href="/pubmed/?term=Wolf%20WA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322400066832" co-class="co-affbox">Wendy A. Wolf</a>,<sup>7</sup> <a href="/pubmed/?term=Pacheco%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322400064848" co-class="co-affbox">Jennifer A. Pacheco</a>,<sup>7</sup> <a href="/pubmed/?term=Chisholm%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322400062864" co-class="co-affbox">Rex L. Chisholm</a>,<sup>7</sup> <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322400060880" co-class="co-affbox">Dan M. Roden</a>,<sup>2,</sup><sup>6</sup> <a href="/pubmed/?term=Hayes%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322400058160" co-class="co-affbox">M. Geoffrey Hayes</a>,<sup>5</sup> and  <a href="/pubmed/?term=Crawford%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349" class="affpopup" co-rid="_co_idm140322400056176" co-class="co-affbox">Dana C. Crawford</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140322329988688"><h3 class="no_margin">Janina M. Jeff</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University, Nashville, TN</p><div>Find articles by <a href="/pubmed/?term=Jeff%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Janina M. Jeff</a></div></div><div id="_co_idm140322332231040"><h3 class="no_margin">Marylyn D. Ritchie</h3><p><sup>8</sup>Department of Pharmacology, Vanderbilt University, Nashville, TN</p><div>Find articles by <a href="/pubmed/?term=Ritchie%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Marylyn D. Ritchie</a></div></div><div id="_co_idm140322331768896"><h3 class="no_margin">Joshua C. Denny</h3><p><sup>2</sup>Department of Medicine, Division of Clinical Pharmacology, Vanderbilt University, Nashville, TN</p><p><sup>3</sup>Department of Biomedical Informatics, Vanderbilt University, Nashville, TN</p><div>Find articles by <a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Joshua C. Denny</a></div></div><div id="_co_idm140322331766176"><h3 class="no_margin">Abel N. Kho</h3><p><sup>4</sup>Division of General Internal Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</p><div>Find articles by <a href="/pubmed/?term=Kho%20AN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Abel N. Kho</a></div></div><div id="_co_idm140322331764192"><h3 class="no_margin">Andrea H. Ramirez</h3><p><sup>2</sup>Department of Medicine, Division of Clinical Pharmacology, Vanderbilt University, Nashville, TN</p><div>Find articles by <a href="/pubmed/?term=Ramirez%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Andrea H. Ramirez</a></div></div><div id="_co_idm140322331762208"><h3 class="no_margin">David Crosslin</h3><p><sup>9</sup>University of Washington Medical Center, Seattle Washington</p><div>Find articles by <a href="/pubmed/?term=Crosslin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">David Crosslin</a></div></div><div id="_co_idm140322331760224"><h3 class="no_margin">Loren Armstrong</h3><p><sup>5</sup>Division of Endocrinology, Metabolism, and Molecular Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</p><div>Find articles by <a href="/pubmed/?term=Armstrong%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Loren Armstrong</a></div></div><div id="_co_idm140322331758240"><h3 class="no_margin">Melissa A. Basford</h3><p><sup>6</sup>Office of Personalized Medicine, Vanderbilt University, Nashville, TN</p><div>Find articles by <a href="/pubmed/?term=Basford%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Melissa A. Basford</a></div></div><div id="_co_idm140322400066832"><h3 class="no_margin">Wendy A. Wolf</h3><p><sup>7</sup>Center for Genetic Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</p><div>Find articles by <a href="/pubmed/?term=Wolf%20WA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Wendy A. Wolf</a></div></div><div id="_co_idm140322400064848"><h3 class="no_margin">Jennifer A. Pacheco</h3><p><sup>7</sup>Center for Genetic Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</p><div>Find articles by <a href="/pubmed/?term=Pacheco%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Jennifer A. Pacheco</a></div></div><div id="_co_idm140322400062864"><h3 class="no_margin">Rex L. Chisholm</h3><p><sup>7</sup>Center for Genetic Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</p><div>Find articles by <a href="/pubmed/?term=Chisholm%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Rex L. Chisholm</a></div></div><div id="_co_idm140322400060880"><h3 class="no_margin">Dan M. Roden</h3><p><sup>2</sup>Department of Medicine, Division of Clinical Pharmacology, Vanderbilt University, Nashville, TN</p><p><sup>6</sup>Office of Personalized Medicine, Vanderbilt University, Nashville, TN</p><div>Find articles by <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Dan M. Roden</a></div></div><div id="_co_idm140322400058160"><h3 class="no_margin">M. Geoffrey Hayes</h3><p><sup>5</sup>Division of Endocrinology, Metabolism, and Molecular Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</p><div>Find articles by <a href="/pubmed/?term=Hayes%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">M. Geoffrey Hayes</a></div></div><div id="_co_idm140322400056176"><h3 class="no_margin">Dana C. Crawford</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University, Nashville, TN</p><div>Find articles by <a href="/pubmed/?term=Crawford%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23534349">Dana C. Crawford</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140322331156768_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140322331156768_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140322331156768_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Center for Human Genetics Research, Vanderbilt University, Nashville, TN</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Medicine, Division of Clinical Pharmacology, Vanderbilt University, Nashville, TN</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Biomedical Informatics, Vanderbilt University, Nashville, TN</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Division of General Internal Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Division of Endocrinology, Metabolism, and Molecular Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Office of Personalized Medicine, Vanderbilt University, Nashville, TN</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Center for Genetic Medicine, Feinberg School of Medicine, Northwestern University, Chicago, IL</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Pharmacology, Vanderbilt University, Nashville, TN</div><div class="fm-affl" id="A9" lang="en"><sup>8</sup>Department of Biochemistry and Molecular Biology, Penn State University, University Park, PA</div><div class="fm-affl" id="A10" lang="en"><sup>9</sup>University of Washington Medical Center, Seattle Washington</div><div id="FN1">Corresponding Author: Janina M. Jeff, PhD, Center for Human Genetics Research, Vanderbilt University, 2215 Garland Avenue, 515G Light Hall, Nashville, TN 37232-0700, Telephone: (615) 343-8613, Fax: (615) 322-8619, <a href="mailto:dev@null" data-email="ude.tlibrednav@ffej.m.aninaj" class="oemail">ude.tlibrednav@ffej.m.aninaj</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140322331156768_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23534349" target="pmc_ext" ref="reftype=publisher&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Ann Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3743946/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Data: </strong><strong>Supplementary Table 1. <em>SNP-trait associations significant in African Americans (p&lt;0.05) but trend in the opposite direction compared with previously identified association.</em></strong> We compared association results in African Americans to the association results from the original study accounting for the coded allele. This table only represents associations that are statistically significant in African Americans (p&lt;0.05) but have discordant effect sizes (expressed as a beta coefficient) compared to the original study. Abbreviations: CAF= coded allele frequency.<p><strong>Supplementary Figure 1<em>. Genome-wide association results for heart rate in African Americans (n= 455).</em> (A) Manhattan plot of GWAS results for heart rate.</strong> Association results are plotted for each maker on the x-axis across the genome, and the −log of the p-value is plotted on the y axis. <strong>(B)Locus Zoom Plot for the most significant association.</strong> We plotted the local association results around 100kb of the most significant SNP rs1015003 from our GWAS of heart rate. The −log of the p-value is plotted on the y-axis, and each SNP is plotted by location on the x-axis. Pairwise linkage disequilibrium with the index SNP, measured by r<sup>2</sup>, is indicated by color. <strong>(C) Locus Zoom Plot for the <em>SERPINI1</em> region</strong>. We plotted the local association results around 100kb of the most significant SNP (rs13090836) region in the <em>SERPINI1</em> region.</p><p><strong>Supplementary Figure 2. <em>Comparison of SCN5A/SCN10A Locus Zoom plots between African Americans (top panel) and European Americans from Denny et al (</em></strong><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip">Denny et al. 2010</a><strong><em>) (bottom panel).</em></strong> Using locus specific plots, we compared the index SNP (indicated at the top of each plot) identified in African Americans to previously identified association results in European Americans from VGER.</p><p><strong>Supplementary Table 2. <em>Effect size and p-values for GWAS-identified associations with a consistent direction of effect in African Americans compared to European descent populations.</em></strong> We compared the association results from GWAS-identified variants to African American association results for SNPs that have a consistent direction of effect but did not meet our significance threshold (p&lt;0.05) for all ECG traits.</p><p><strong>Supplementary Table 3. <em>Coded Allele Frequencies for GWAS-identified SNPs in African Americans compared to the original study population.</em></strong> We calculated allele frequencies for all GWAS-identified SNPs in African Americans after accounting for the coded allele.</p><p><strong>Supplementary Table 4. <em>Replication of previous associations for PR interval in VGER/NUgene African Americans.</em></strong> We tested previously reported associations with PR interval in African Americans from Smith et al. for replication in our study population (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip">Smith et al. 2011</a>). Using the same coded allele as the original study, we performed tests of association for all reported SNPs genotyped in our study population using an additive genetic model. Abbreviations: CAF= coded allele frequency</p><p><strong>Supplementary Table 5. <em>Replication of previous associations for QT interval in VGER/NUgene African Americans.</em></strong> We tested previously reported associations with QT interval in African Americans from Avery et al. for replication in our study population (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip">Avery et al. 2012</a>). Using the same coded allele as the original study, we performed tests of association for all reported SNPs that were also genotyped in our study population using an additive genetic model. Abbreviations: CAF= coded allele frequency, *= Novel association in African Americans for PR interval reported by Avery et al</p><p><strong>Supplementary Figure 3. <em>Linkage Disequilibrium plots around the SCN10A region in YRI HapMap samples compared to CEU samples.</em></strong> Pairwise linkage disequilibrium was calculated for all SNPs with a MAF of &gt;0.05 within 100kb of the SCN10A gene using Seattle SNPs Genome Server (<a href="http://gvs.gs.washington.edu/GVS/index.jsp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gvs.gs.washington.edu/GVS/index.jsp</a>).</p><p><strong>Supplementary Figure 4. <em>Linkage Disequilibrium plots in the NOS1AP gene in YRI HapMap samples compared to CEU samples.</em></strong> Pairwise linkage disequilibrium was calculated for all SNPs with a MAF of &gt;0.05 within the <em>NOS1AP</em> gene using Seattle SNPs Genome Server (<a href="http://gvs.gs.washington.edu/GVS/index.jsp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gvs.gs.washington.edu/GVS/index.jsp</a>).</p></div><div><a href="/pmc/articles/PMC3743946/bin/NIHMS463822-supplement-Supplementary_Data.docx" data-ga-action="click_feat_suppl">NIHMS463822-supplement-Supplementary_Data.docx</a><span style="color:gray"> (7.7M)</span></div><div class="small guid">GUID:&nbsp;D3BEEE00-3D52-4FED-A3A5-308651C887C8</div></div></dd></dl></div><div id="idm140322333196976" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140322333196976title">SUMMARY</h2><!--article-meta--><div><p id="P2" class="p p-first-last">Electrocardiographic (ECG) measurements vary by ancestry. Genome-wide association studies (GWAS) have identified loci that contribute to ECG measurements; however most are performed in Europeans collected from population-based cohorts or surveys. The strongest associations reported are in <em>NOS1AP</em> with QT interval and <em>SCN10A</em> with PR and QRS durations. The extent to which these associations can be generalized to African Americans has yet to be determined. Using electronic medical records, PR and QT intervals, QRS duration, and heart rate were determined in 455 African Americans as part of the Vanderbilt Genome-Electronic Records Project and Northwestern University NUgene Project. We tested for an association between these ECG traits and &gt;930K SNPs. We identified a total 36 novel associations with PR interval, QRS duration, QT interval, and heart rate at p&lt; 1.0 ×10<sup>−6</sup>. Using published GWAS data, we compared our results with those previously identified in other populations. Five associations originally identified in other populations generalized with respect to statistical significance and direction of effect. A total of 43 associations have a consistent direction of effect with European and/or Asian populations. This work provides a catalogue of generalized versus non-generalized associations, a necessary step in prioritizing GWAS-identified regions for further fine-mapping in diverse populations.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Electrocardiography, African Americans, GWAS, Generalization, Electronic Medical Records</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P3" class="p p-first">Candidate gene studies have identified several genes that affect cardiac depolarization and conduction measured by ECG traits. One set of candidates consists of genes that encode voltage gated potassium and sodium channels, which also play an important role in the pathophysiology of long QT syndrome (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492013">Lai et al. 1994</a>; <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491950">Splawski et al. 2002</a>; <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491993">Wang et al. 1996</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491954">Wang et al. 1995a</a>; <a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491991">Wang et al. 1995b</a>). These associations have been confirmed by molecular genetic studies, and, in the past five years, GWAS have also identified these associations with various ECG phenotypes in European and Asian-descent populations (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491992">Chambers et al. 2010</a>; <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492000">Newton-Cheh et al. 2009</a>; <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491980">Smith et al. 2009</a>). These GWAS have not only implicated common variation in congenital long QT syndrome genes, but also have identified associations between genes not previously implicated in cardiac electrophysiology and ECG traits, such as <em>NOS1AP</em> with QT interval (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491997">Arking et al. 2006</a>; <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491970">Post et al. 2007</a>). Genome-wide association studies for ECG traits have also revealed pleiotropy: for example, variants in the cardiac sodium channel gene <em>SCN5A,</em> has been associated with across multiple ECG traits (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491960">Holm et al. 2010</a>; <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491953">Jeff et al. 2011</a>; <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491967">Marroni et al. 2009</a>; <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491987">Newton-Cheh et al. 2007</a>; <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491969">Smith et al. 2009</a>). Variants in <em>SCN10A,</em> a gene previously known to play a role in the action potentials in nociceptive nerve fibers (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491973">Abrahamsen et al. 2008</a>; <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491947">Akopian et al. 1999</a>; <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491946">Zimmermann et al. 2007</a>), have been identified by GWAS for an association with PR and QRS interval (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491989">Chambers et al. 2010</a>; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491959">Denny et al. 2010</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491971">Holm et al. 2010</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492014">Pfeufer et al. 2010</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491962">Sotoodehnia et al. 2010</a>).</p><p id="P4">As of October 2011, there were 127 unique trait-SNP associations reported in the National Human Genome Research Institute’s (NHGRI) GWAS catalog for electrocardiographic (ECG) traits (<a href="http://www.genome.gov/gwastudies/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/gwastudies/</a>). Most of these studies were limited to European (118 associations) or Asian (9 associations)-descent populations, and it is unclear if these GWAS-identified variants generalize across populations, particularly the extent to which these associations generalize to African Americans. There is considerable variability in ECG traits across populations. African Americans have a shorter QRS duration, QTc interval, and a longer PR interval compared to Europeans (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491985">Ramirez et al. 2011</a>). From a genetic standpoint, there are also significant differences in allele frequencies and linkage disequilibrium patterns between Africans and Europeans. For example, rs7626962 (S1103Y) in <em>SCN5A</em> is rare in European and Asian-descent populations but is common in African Americans and has been linked to arrhythmia susceptibility (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491956">Jeff et al. 2011</a>; <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491979">Splawski et al. 2002</a>). Expanding genetic association studies to diverse populations has been of recent interest to many investigators in the field. To date there has been one GWAS published for ECG traits in African Americans on PR interval and one fine-mapping study for QT interval in African Americans (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491958">Avery et al. 2012</a>; <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491963">Smith et al. 2011</a>). Missing from the current literature is a comprehensive study of multiple ECG traits that explain the generalizability or lack thereof across populations. Furthermore, to date there are no GWAS reported for heart rate in African Americans.</p><p id="P5" class="p p-last">Assessing the utility of electronic medical records (EMRs) systems coupled to DNA repositories as a tool for genome science is one of the primary objectives for the National Human Genome Research Institute’s electronic MEdical Records and GEnomics (eMERGE) Network (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491951">McCarty et al. 2011</a>). Studies from eMERGE, including those on PR interval, have demonstrated that EMR-based genetic studies replicate existing findings and discover new ones (Crosslin et al. 2011; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492002">Denny et al. 2010</a>; <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491949">Denny et al. 2011</a>; <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491964">Turner et al. 2011</a>). Here we performed a GWAS of ECG measurements to identify novel genetic associations and describe the extent to which previous associations generalize in African Americans within the eMERGE network. We also examine reasons for non-generalization of associations in our African American study population.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">MATERIALS AND METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Study population</h3><p id="P6" class="p p-first">African American subjects were collected from the Vanderbilt or Northwestern University biobanks. The Vanderbilt Genome-Electronic Records (VGER) accesses BioVU, the Vanderbilt biorepository of DNA extracted from blood collected for routine clinical care linked to de-identified electronic medical records (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491983">Roden et al. 2008</a>). BioVU is a subset of the larger repository of de-identified EMRs known as the synthetic derivative. The Northwestern biobank, NUgene, combines DNA samples from consented participants with enrollment questionnaire and longitudinal data from the EMR (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491968">McCarty et al. 2011</a>). Both biobanks were approved by Institutional Review Boards at their respective sites.</p><p id="P7" class="p p-last">Study population demographics and characteristics are described in <a href="/pmc/articles/PMC3743946/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Individuals with a normal ECG without evidence of cardiac disease before or within one month following the ECG, without concurrent use of medications that interfere with QRS duration, and who did not have abnormal electrolyte values at the time of the ECG were included. Using natural language processing combined with EMR (structured database queries) to query (specific to VGER), we excluded individuals with any of the following before or within one month of the ECG: indication of heart failure, arrhythmia, cardiomyopathy, cardiac conduction defect, or myocardial ischemia/infarct based on clinical notes, unstructured text, billing codes, and labs. VGER subjects were African American as indicated by observer reported ancestry, which is highly concordant with genetic ancestry (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491982">Dumitrescu et al. 2010</a>). African American ancestry was self-reported for Northwestern subjects (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492008">McCarty et al. 2011</a>), which is also known to be highly concordant with genetic ancestry. All ECGs had normal Bazett’s corrected QT intervals (&lt;450ms), heart rates (between 50–100 bpm), and QRS duration (65–120 ms).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Descriptive statistics of the study population</strong><p id="__p5">Means and standard deviations were calculated in the final study population for all covariates and ECG traits unless otherwise noted.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322320686672" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="3" rowspan="1" valign="middle" align="center">Study Population (n= 455)</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Variable</th><th rowspan="1" colspan="1" valign="middle" align="center">Mean /%</th><th rowspan="1" colspan="1" valign="middle" align="center">SD</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">Female</td><td rowspan="1" colspan="1" valign="top" align="center">77%</td><td rowspan="1" colspan="1" valign="top" align="center">--</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Age (yr)</td><td rowspan="1" colspan="1" valign="top" align="center">46</td><td rowspan="1" colspan="1" valign="top" align="center">15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PR duration (msec)</td><td rowspan="1" colspan="1" valign="top" align="center">159</td><td rowspan="1" colspan="1" valign="top" align="center">21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">QRS duration (msec)</td><td rowspan="1" colspan="1" valign="top" align="center">82</td><td rowspan="1" colspan="1" valign="top" align="center">8</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">QTc duration (msec)</td><td rowspan="1" colspan="1" valign="top" align="center">410</td><td rowspan="1" colspan="1" valign="top" align="center">21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Heart Rate (msec)</td><td rowspan="1" colspan="1" valign="top" align="center">74</td><td rowspan="1" colspan="1" valign="top" align="center">11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">On QT drug</td><td rowspan="1" colspan="1" valign="top" align="center">10%</td><td rowspan="1" colspan="1" valign="top" align="center">--</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">On PR drug</td><td rowspan="1" colspan="1" valign="top" align="center">21%</td><td rowspan="1" colspan="1" valign="top" align="center">--</td></tr></tbody></table></div><div id="largeobj_idm140322320686672" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3743946/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="S4" class="sec"><h3 id="S4title">Genotyping</h3><p id="P8" class="p p-first-last">Genotyping for the eMERGE network was performed by the Center for Inherited Disease Research (CIDR) and the Broad Institute. All individuals that met the inclusion criteria (n = 501) were genotyped for &gt;1.1 million SNPs using the Illumina 1M BeadChip at the Broad Institute. Data were cleaned by the eMERGE QC pipeline (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491975">Zuvich et al. 2011</a>). Individuals with cryptic relatedness, ancestry inconsistent with observer- or self-reported ancestry, anomalous X-chromosome heterozygosity or poor genotyping efficiency were removed from further analysis (n = 46). All markers that were intensity only probes that had technical failure, minor allele frequency ≤ 0.05, genotyping efficiency &lt;99%, discordant calls with duplicates, and Mendelian errors &gt;0 were removed. Tests of association were performed with and without SNP that deviated from Hardy Weinberg Equilibrium (p-value &lt;1.0 × 10 <sup>−4</sup>), and consistent findings were observed for both SNP sets (data not shown).</p></div><div id="S5" class="sec"><h3 id="S5title">Statistical methods</h3><p id="P9" class="p p-first-last">Greater than 930K SNPs from the Illumina 1M BeadChip were tested for an association with PR interval, QRS duration, QTc interval, and heart rate using linear regression assuming an additive genetic model. Tests of association were performed unadjusted and adjusted for age, sex, PR/QT drug usage (for PR and QT interval only), and principal components (PCs; two to four PCs depending on the trait) using PLINK (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491965">Purcell et al. 2007</a>). All results were plotted using Synthesis View, Haploview, or Locus Zoom (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491957">Barrett et al. 2005</a>; <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491996">Pendergrass et al. 2010</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491961">Pruim et al. 2010</a>). Power calculations were performed using QUANTO (<a href="#R12" rid="R12" class=" bibr popnode">Gauderman &amp; Morrison 2006</a>) assuming genetic effect sizes based on the original published GWAS. The fixation index F<sub>ST</sub>, a measure of population differentiation, was calculated using the Weir and Cockerham algorithm (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_761984663">Weir &amp; Cockerham 1984</a>). We calculated F<sub>ST</sub> between European descent populations and our African American study population using the Platform for the Analysis, Translation, and Organization of large-scale data (PLATO) (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492010">Grady et al. 2010</a>). Pair-wise linkage disequilibrium was calculated (r<sup>2</sup>) around the <em>SCN5A/SCN10A</em> and <em>NOS1AP</em> regions using the SeattleSNPs Genome Variation Server (gvs.gs.washington.edu/).</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Generalization</h3><p id="P10" class="p p-first-last">Using the NHGRI GWAS catalog, we identified SNPs associated with PR interval, QTc interval, QRS duration, and heart rate at significance threshold of p&lt;10<sup>−6</sup> from European or Asian -descent populations. We then examined the significance level and direction of effect for the same SNP-trait pair in this African American dataset.</p></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">RESULTS</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Discovery</h3><p id="P11" class="p p-first-last">None of the SNPs tested for an association met genome-wide significance (p&lt;5.0×10<sup>−8</sup>) with any ECG trait. Although we did not detect associations at genome-wide significance, we were able to detect novel associations at p&lt;10<sup>−6</sup> (<a href="/pmc/articles/PMC3743946/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). Nine common variants were associated with increased heart rate, representing five independent disease loci. Three SNPs were in complete linkage disequilibrium with each other (r<sup>2</sup>= 1.0) in <em>SERPINI1</em> on chromosome 3. Aside from SNPs associated with heart rate, nine associations at p&lt;10<sup>−6</sup> were observed for QT interval, QRS duration, and PR interval, separately (27 associations total). Collectively, these potentially novel associations represent 17 candidate genes throughout the genome associated with these traits. None of these associations have been reported by previously published GWAS for ECG traits in any population.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Most significant genome-wide association study (GWAS) results in African Americans for heart rate, QT interval, QRS duration, and PR interval</strong><p id="__p6">We performed single SNP tests of association between &gt;930 SNPs and ECG traits using linear regression assuming an additive genetic model in African Americans (n=455). All tests of association were adjusted by age, sex, and principal components. Associations at p ≤ 10<sup>−6</sup> are shown, and for each association, chromosome, rs number, nearest gene, coded allele, beta coefficient, and p-value are given.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322333089040" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="6" rowspan="1" valign="middle" align="center">Heart Rate</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">CHR</th><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">LOCATION</th><th rowspan="1" colspan="1" valign="top" align="center">CODED ALLELE</th><th rowspan="1" colspan="1" valign="top" align="center">BETA</th><th rowspan="1" colspan="1" valign="top" align="center">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">rs1015003</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">3.94</td><td rowspan="1" colspan="1" valign="top" align="center">1.21E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">rs6468401</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">3.93</td><td rowspan="1" colspan="1" valign="top" align="center">1.55E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">rs816475</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">3.69</td><td rowspan="1" colspan="1" valign="top" align="center">2.83E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs13090836</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SERPINI1</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">3.58</td><td rowspan="1" colspan="1" valign="top" align="center">6.86E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs9815034</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SERPINI1</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">3.59</td><td rowspan="1" colspan="1" valign="top" align="center">7.26E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs1473511</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SERPINI1</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">3.59</td><td rowspan="1" colspan="1" valign="top" align="center">7.26E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">rs12824981</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TMEM132D</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">4.56</td><td rowspan="1" colspan="1" valign="top" align="center">8.90E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">rs35061590</td><td rowspan="1" colspan="1" valign="top" align="center"><em>DAB2IP</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">16.03</td><td rowspan="1" colspan="1" valign="top" align="center">9.45E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">rs13290547</td><td rowspan="1" colspan="1" valign="top" align="center"><em>DAB2IP</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">16.03</td><td rowspan="1" colspan="1" valign="top" align="center">9.45E-06</td></tr><tr><td colspan="6" rowspan="1" valign="middle" align="center"><strong>QT Interval</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">CHR</td><td rowspan="1" colspan="1" valign="top" align="center">SNP</td><td rowspan="1" colspan="1" valign="top" align="center">LOCATION</td><td rowspan="1" colspan="1" valign="top" align="center">CODED ALLELE</td><td rowspan="1" colspan="1" valign="top" align="center">BETA</td><td rowspan="1" colspan="1" valign="top" align="center">P</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">rs6894385</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">−9.92</td><td rowspan="1" colspan="1" valign="top" align="center">1.03E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">rs9342616</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">−6.92</td><td rowspan="1" colspan="1" valign="top" align="center">1.04E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">rs12666280</td><td rowspan="1" colspan="1" valign="top" align="center"><em>DPP6</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">8.93</td><td rowspan="1" colspan="1" valign="top" align="center">1.66E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">rs237450</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6.86</td><td rowspan="1" colspan="1" valign="top" align="center">5.36E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">rs6819013</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6.06</td><td rowspan="1" colspan="1" valign="top" align="center">5.50E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">rs4698433</td><td rowspan="1" colspan="1" valign="top" align="center"><em>FGFBP2</em> (near 5′ region)</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">7.23</td><td rowspan="1" colspan="1" valign="top" align="center">5.53E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">rs1483012</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LDB2</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">6.02</td><td rowspan="1" colspan="1" valign="top" align="center">7.19E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">rs8045405</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">−11.38</td><td rowspan="1" colspan="1" valign="top" align="center">9.44E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">rs17237606</td><td rowspan="1" colspan="1" valign="top" align="center"><em>UNC13C</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">−8.91</td><td rowspan="1" colspan="1" valign="top" align="center">9.92E-06</td></tr><tr><td colspan="6" rowspan="1" valign="middle" align="center"><strong>QRS Duration</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">CHR</td><td rowspan="1" colspan="1" valign="top" align="center">SNP</td><td rowspan="1" colspan="1" valign="top" align="center">LOCATION</td><td rowspan="1" colspan="1" valign="top" align="center">CODED ALLELE</td><td rowspan="1" colspan="1" valign="top" align="center">BETA</td><td rowspan="1" colspan="1" valign="top" align="center">P</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">rs1867082</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">2.82</td><td rowspan="1" colspan="1" valign="top" align="center">1.46E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">rs504008</td><td rowspan="1" colspan="1" valign="top" align="center"><em>NKAIN2</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">2.70</td><td rowspan="1" colspan="1" valign="top" align="center">2.58E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">rs6861497</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">2.34</td><td rowspan="1" colspan="1" valign="top" align="center">4.73E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs308309</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">4.39</td><td rowspan="1" colspan="1" valign="top" align="center">4.95E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">rs12194062</td><td rowspan="1" colspan="1" valign="top" align="center"><em>C6orf190</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">5.16</td><td rowspan="1" colspan="1" valign="top" align="center">5.10E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">rs6820368</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ADH6</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">9.90</td><td rowspan="1" colspan="1" valign="top" align="center">5.57E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">rs10784762</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">−2.38</td><td rowspan="1" colspan="1" valign="top" align="center">6.31E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">rs17444745</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">−6.40</td><td rowspan="1" colspan="1" valign="top" align="center">6.96E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">rs13375391</td><td rowspan="1" colspan="1" valign="top" align="center"><em>HMCN1</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">5.82</td><td rowspan="1" colspan="1" valign="top" align="center">9.37E-06</td></tr><tr><td colspan="6" rowspan="1" valign="middle" align="center"><strong>PR Interval</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">CHR</td><td rowspan="1" colspan="1" valign="top" align="center">SNP</td><td rowspan="1" colspan="1" valign="top" align="center">LOCATION</td><td rowspan="1" colspan="1" valign="top" align="center">CODED ALLELE</td><td rowspan="1" colspan="1" valign="top" align="center">BETA</td><td rowspan="1" colspan="1" valign="top" align="center">P</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs1994318</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MICAL2</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">−6.79</td><td rowspan="1" colspan="1" valign="top" align="center">1.52E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">rs10447419</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">−8.65</td><td rowspan="1" colspan="1" valign="top" align="center">1.95E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs16926523</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYO3A</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">9.55</td><td rowspan="1" colspan="1" valign="top" align="center">4.23E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs7604827</td><td rowspan="1" colspan="1" valign="top" align="center"><em>VWC2L</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">6.54</td><td rowspan="1" colspan="1" valign="top" align="center">4.52E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs3733017</td><td rowspan="1" colspan="1" valign="top" align="center"><em>BCL6</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">−10.07</td><td rowspan="1" colspan="1" valign="top" align="center">5.55E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">rs746265</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">−6.27</td><td rowspan="1" colspan="1" valign="top" align="center">7.92E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs1524976</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MAGI1</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">8.00</td><td rowspan="1" colspan="1" valign="top" align="center">7.96E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">rs3103778</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MFSD2</em> (intron)</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">6.30</td><td rowspan="1" colspan="1" valign="top" align="center">8.67E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">rs12595668</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">−6.26</td><td rowspan="1" colspan="1" valign="top" align="center">9.01E-06</td></tr></tbody></table></div><div id="largeobj_idm140322333089040" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3743946/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S9" class="sec"><h3 id="S9title">Generalization</h3><p id="P12" class="p p-first">In European-descent populations, 118 GWAS-identified SNPs have been reported for ECG traits in the NHGRI GWAS catalog as of October 2011. We examined the reported European-descent SNP-trait association to determine if the association “generalized” to African Americans. We considered an association generalized if the observed association was significant at a liberal significance threshold (p&lt;0.05) and had a direction of effect consistent with the original study. Of the 118 SNPs identified in the NHGRI GWAS catalog, 92 were also directly genotyped in our study population: 26, 14, 19, and 33 SNPs were previously associated with heart rate, PR interval, QRS duration, and QT interval, respectively.</p><p id="P13">Overall, there were five SNP-trait associations that generalized to African descent populations with respect to both level of significance (p&lt;0.05) and direction of effect: for heart rate rs4352210 (intergenic, β= −1.6, p= 0.03), for QTc interval, rs1112795 (<em>SCN5A,</em> β = − 4.77; p=0.05) and rs4725982 (<em>KCNH2,</em> β = 4.79; p=0.03), and for QRS duration, rs6795970 (<em>SCN10A,</em> β = 2.43; p=0.01) and rs11710077 (<em>SCN5A,</em> β= 2.42; p=0.004) (<a href="/pmc/articles/PMC3743946/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). No SNPs generalized for PR interval. There were six additional SNP-trait associations from the GWAS catalog that were significant (p-value &lt;0.05) but trended in the opposite direction in African Americans compared with European-descent populations (after accounting for the coded allele): four for PR interval, three for heart rate, and one for QT interval (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Association results for genome-wide association study (GWAS)-identified associations that generalize in African Americans</strong><p id="__p7">We compared GWAS-identified association results originally identified in European or Asian-descent populations to 455 African Americans from VGER and NUgene in the eMERGE network. We declared a generalization as having a consistent direction of genetic effect (expressed as a beta coefficient) and p-value &lt;0.05. Abbreviations: CAF= coded allele frequency.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322330600208" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="center">Trait</th><th rowspan="2" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="2" colspan="1" valign="bottom" align="center">Coded Allele</th><th rowspan="2" colspan="1" valign="bottom" align="center">Gene</th><th colspan="4" rowspan="1" valign="bottom" align="center">Original Study</th><th colspan="3" rowspan="1" valign="bottom" align="center">African Americans</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Population</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">CAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">CAF</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">QT interval</td><td rowspan="1" colspan="1" valign="top" align="center">rs11129795</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="center">European</td><td rowspan="1" colspan="1" valign="top" align="center">5.0E-14</td><td rowspan="1" colspan="1" valign="top" align="center">−1.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">−4.77</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">QT interval</td><td rowspan="1" colspan="1" valign="top" align="center">rs4725982</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KCNH2</em></td><td rowspan="1" colspan="1" valign="top" align="center">European and Asian-descent</td><td rowspan="1" colspan="1" valign="top" align="center">5.0E-16</td><td rowspan="1" colspan="1" valign="top" align="center">1.58</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">4.79</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">QRS duration</td><td rowspan="1" colspan="1" valign="top" align="center">rs6795970</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">European and Asian-descent</td><td rowspan="1" colspan="1" valign="top" align="center">4.0E-09</td><td rowspan="1" colspan="1" valign="top" align="center">5.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td><td rowspan="1" colspan="1" valign="top" align="center">2.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">QRS duration</td><td rowspan="1" colspan="1" valign="top" align="center">rs11710077</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="center">European</td><td rowspan="1" colspan="1" valign="top" align="center">1.0E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">2.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Heart rate</td><td rowspan="1" colspan="1" valign="top" align="center">rs4352210</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">European</td><td rowspan="1" colspan="1" valign="top" align="center">2.00E-06</td><td rowspan="1" colspan="1" valign="top" align="center">−0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">−1.6</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td></tr></tbody></table></div><div id="largeobj_idm140322330600208" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3743946/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P14">An additional 40 SNP-trait associations failed to achieve the liberal threshold of significance (p&lt;0.05) in African Americans but did have consistent directions of effect compared with European-descent populations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>, <a href="/pmc/articles/PMC3743946/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Based solely on direction of effect, approximately 50% and 52% of the associations tested for PR and QT intervals, respectively, have a consistent direction of effect in African Americans compared with European descent populations. For QRS duration, 37% of the associations tested generalized in African Americans while 44% generalized for heart rate. Collectively, 44 out of 92 (47%) SNP-trait associations had a consistent direction effect regardless of significance in African Americans compared to previously identified associations in European-descent populations. The remaining 48/92 (52%) SNP-trait associations identified in European-descent populations did not generalize to African Americans with respect to either level of significance or direction of effect.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3743946/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322365855760" class="figure"><a href="/pmc/articles/PMC3743946/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3743946_nihms463822f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms463822f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3743946/bin/nihms463822f1.jpg"></a></div><div id="largeobj_idm140322365855760" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3743946/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3743946/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>(A–D). SNP-trait association comparison between European or Asian -descent and African American populations across ECG traits</strong><p id="__p4">Each SNP was tested for an association with each ECG trait assuming an additive genetic model adjusted for age and sex. P-values are −log10 transformed along the y-axis and corresponding location for each SNP is located on the x-axis. Each point represents a p-value for each population indicated by color (see legend). The direction of the arrows corresponds to the direction of the effect (measured by beta coefficient). The significance threshold is indicated by the red bar at p= 0.05. The bottom panel displays the minor allele frequency comparisons for both populations for each SNP.</p></div></div></div><p id="P15" class="p p-last">There were ten SNPs reported in the NHGRI catalog that are associated with ECG traits in Asian -descent populations. Of these, six SNPs were also genotyped in our study population. None of these SNPs generalized in African Americans with respect to significance and direction of effect, although two of these associations had a consistent direction of effect compared with the original study population.</p></div><div id="S10" class="sec"><h3 id="S10title">Power</h3><p id="P16" class="p p-first-last">SNP-trait associations that failed to achieve significance or failed to generalize to African Americans may represent lack of power or true differences in allelic architecture and/or linkage disequilibrium patterns between the two populations. To help distinguish between these two possibilities, we calculated the power to detect the mostly European-reported associations in this African American sample. Overall, 17 SNP-trait tests of association were adequately powered (&gt;80%). Among the adequately powered tests of association, almost all (16/17) SNP-trait tests of association failed to generalize in African Americans with respect to level of significance anddirection of effect (<a href="/pmc/articles/PMC3743946/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Association results for adequately powered genome-wide association study (GWAS)-identified SNPs that do not generalize in African Americans</strong><p id="__p8">Using the effect size, mean, and standard deviations reported in the published study, we calculated our power to detect the same effect given our sample size and minor allele frequency in African Americans. CAF= coded allele frequency.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322328378496" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="center">SNP</th><th rowspan="2" colspan="1" valign="middle" align="center">Trait</th><th rowspan="2" colspan="1" valign="middle" align="center">Gene</th><th rowspan="2" colspan="1" valign="middle" align="center">Coded Allele</th><th colspan="3" rowspan="1" valign="middle" align="center">Original Study</th><th colspan="4" rowspan="1" valign="middle" align="center">African Americans</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">CAF</th><th rowspan="1" colspan="1" valign="middle" align="center">P-value</th><th rowspan="1" colspan="1" valign="middle" align="center">Beta</th><th rowspan="1" colspan="1" valign="middle" align="center">CAF</th><th rowspan="1" colspan="1" valign="middle" align="center">P-value</th><th rowspan="1" colspan="1" valign="middle" align="center">Beta</th><th rowspan="1" colspan="1" valign="middle" align="center">Power</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10494366</td><td rowspan="1" colspan="1" valign="top" align="center">QT Interval</td><td rowspan="1" colspan="1" valign="top" align="center"><em>NOS1AP</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">5.00E-22</td><td rowspan="1" colspan="1" valign="top" align="center">12.2</td><td rowspan="1" colspan="1" valign="top" align="center">0.60</td><td rowspan="1" colspan="1" valign="top" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11129795</td><td rowspan="1" colspan="1" valign="top" align="center">QRS duration</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.77</td><td rowspan="1" colspan="1" valign="top" align="center">5.00E-10</td><td rowspan="1" colspan="1" valign="top" align="center">−8.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td><td rowspan="1" colspan="1" valign="top" align="center">0.73</td><td rowspan="1" colspan="1" valign="top" align="center">−0.25</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12296050</td><td rowspan="1" colspan="1" valign="top" align="center">QT Interval</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KCNQ1</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">1.00E-11</td><td rowspan="1" colspan="1" valign="top" align="center">13.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">0.52</td><td rowspan="1" colspan="1" valign="top" align="center">−1.20</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3807989</td><td rowspan="1" colspan="1" valign="top" align="center">PR interval</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CAV1</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td><td rowspan="1" colspan="1" valign="top" align="center">7.00E-13</td><td rowspan="1" colspan="1" valign="top" align="center">6.40</td><td rowspan="1" colspan="1" valign="top" align="center">0.62</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.95</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3825214</td><td rowspan="1" colspan="1" valign="top" align="center">QRS duration</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TBX5</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">3.00E-13</td><td rowspan="1" colspan="1" valign="top" align="center">7.35</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.52</td><td rowspan="1" colspan="1" valign="top" align="center">−0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3825214</td><td rowspan="1" colspan="1" valign="top" align="center">PR interval</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TBX5</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.00E-07</td><td rowspan="1" colspan="1" valign="top" align="center">5.88</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.79</td><td rowspan="1" colspan="1" valign="top" align="center">−0.40</td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1321311</td><td rowspan="1" colspan="1" valign="top" align="center">QRS duration</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CDKN1A</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">3.00E-10</td><td rowspan="1" colspan="1" valign="top" align="center">6.52</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.77</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs174547</td><td rowspan="1" colspan="1" valign="top" align="center">Heart Rate</td><td rowspan="1" colspan="1" valign="top" align="center"><em>FADS1</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">2.00E-09</td><td rowspan="1" colspan="1" valign="top" align="center">6.20</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">1.028</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs223116</td><td rowspan="1" colspan="1" valign="top" align="center">Heart Rate</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYH7, NDNG</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">3.00E-08</td><td rowspan="1" colspan="1" valign="top" align="center">7.40</td><td rowspan="1" colspan="1" valign="top" align="center">0.76</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">−0.635</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2461751</td><td rowspan="1" colspan="1" valign="top" align="center">PR interval</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">8.00E-06</td><td rowspan="1" colspan="1" valign="top" align="center">4.54</td><td rowspan="1" colspan="1" valign="top" align="center">0.68</td><td rowspan="1" colspan="1" valign="top" align="center">0.83</td><td rowspan="1" colspan="1" valign="top" align="center">−0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.80</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs281868</td><td rowspan="1" colspan="1" valign="top" align="center">Heart Rate</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SLC35F1</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">4.00E-10</td><td rowspan="1" colspan="1" valign="top" align="center">6.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs314370</td><td rowspan="1" colspan="1" valign="top" align="center">Heart Rate</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SLC12A9</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">6.00E-10</td><td rowspan="1" colspan="1" valign="top" align="center">7.60</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">1.58</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs365990</td><td rowspan="1" colspan="1" valign="top" align="center">HR</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYH6</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">7.00E-06</td><td rowspan="1" colspan="1" valign="top" align="center">5.25</td><td rowspan="1" colspan="1" valign="top" align="center">0.61</td><td rowspan="1" colspan="1" valign="top" align="center">0.61</td><td rowspan="1" colspan="1" valign="top" align="center">−0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3807375</td><td rowspan="1" colspan="1" valign="top" align="center">QT Interval</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KCNH2</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">5.00E-11</td><td rowspan="1" colspan="1" valign="top" align="center">11.95</td><td rowspan="1" colspan="1" valign="top" align="center">0.69</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">1.95</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs452036</td><td rowspan="1" colspan="1" valign="top" align="center">Heart Rate</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYH6</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">4.00E-14</td><td rowspan="1" colspan="1" valign="top" align="center">7.80</td><td rowspan="1" colspan="1" valign="top" align="center">0.61</td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">−0.68</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7660702</td><td rowspan="1" colspan="1" valign="top" align="center">PR interval</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ARHGAP24</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.74</td><td rowspan="1" colspan="1" valign="top" align="center">3.00E-17</td><td rowspan="1" colspan="1" valign="top" align="center">8.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.60</td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr></tbody></table></div><div id="largeobj_idm140322328378496" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3743946/table/T4/?report=objectonly">Open in a separate window</a></div></div></div><div id="S11" class="sec sec-last"><h3 id="S11title">Allele Frequencies</h3><p id="P17" class="p p-first-last">We calculated the coded allele frequencies for previously identified SNPs (from European descent populations) in African Americans (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). To measure population differences, we calculated F<sub>ST</sub> for all SNPs previously identified by GWAS in European descent populations that report sample sizes and minor allele frequencies (85 SNPs). There were 20 SNPs with an F<sub>ST</sub> value &gt;0.15 (ranging from 0.15 to 0.85), which is indicative of significant population differentiation at these loci. We observed the largest F<sub>ST</sub> value (F= 0.85) for rs789852 located in the intron region of the <em>TMEMFF</em> gene located on chromosome 3. This SNP was previously associated with QT interval in Europeans (β= 0.25, p=7.0 × 10<sup>−7</sup> ) (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492005">Marroni et al. 2009</a>). This association did not generalize in African Americans and trended in the opposite direction compared to Europeans (β = −0.95, p =0.60).</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">DISCUSSION</h2><p id="P18" class="p p-first">Here we identified novel associations with heart rate, QT interval, QRS duration, and PR interval. We also characterized previously reported associations in our African American study population derived in a clinical based setting. To our knowledge we are the first to perform a genome-wide association study for heart rate in African Americans and to characterize GWAS-identified variants in European and Asian-descent populations to African Americans for heart rate, QT interval, and QRS duration.</p><div id="S13" class="sec"><h3 id="S13title">Discovery</h3><p id="P19" class="p p-first">We have identified 36 novel associations in 17 candidate genes with the four ECG traits tested at p&lt; 1.0 × 10 <sup>−6</sup>. While none of our associations met genome-wide significance, reporting these associations can provide insight to novel regions in the genome that effect ECG trait variability yet to be explored in African populations. Interestingly we were able to detect a novel region on chromosome 3 with three SNPs that are in high LD and are associated with increased heart rate in <em>SERPINI1</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1C</a>). The <em>SERPINI1</em> gene encodes serine peptidase inhibitors secreted by axons in the brain and has been associated with familial encephalopathy (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491948">Davis et al. 1999</a>).</p><p id="P20" class="p p-last">We also identified a region on chromosome 9 within the <em>DAB2IP</em> gene encompassing two SNPs that were associated with increased heart rate. This gene encodes a GTPase-activating protein and is an inhibitor of cell growth and survival. <em>DAB2IP</em> has been identified by a GWAS for an association with prostate cancer in European and African descent populations (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_483618066">Gretarsdottir et al. 2010</a>). Recently, a GWAS of individuals from Iceland and the Netherlands identified an association with <em>DAB2IP</em> with abdominal aortic aneurysms and other vascular diseases including early onset myocardial infarction (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_483618065">Gretarsdottir et al. 2010</a>).</p></div><div id="S14" class="sec"><h3 id="S14title">Generalization</h3><p id="P21" class="p p-first">Four of the five generalizations we identified in African Americans are located in voltage-gated channels (<a href="/pmc/articles/PMC3743946/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). For PR interval, we compared the <em>SCN5A/SCN10</em> region association results in African Americans with previously identified European Americans (also ascertained from VGER; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>)(<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491999">Denny et al. 2010</a>). Though not significant, the index SNP identified in Europeans was different compared to African Americans. This is likely due to differences in linkage disequilibrium patterns between the two populations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 2 and 3</a>).</p><p id="P22">For QRS duration, two of these variants are within the <em>SCN5A/SCN10A</em> region and generalize in African Americans (<a href="/pmc/articles/PMC3743946/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). <em>SCN10A</em> variant rs97595970 originally associated with PR interval has been previously reported to have pleiotropic effects with QRS duration (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491978">Chambers et al. 2010</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491966">Holm et al. 2010</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491972">Sotoodehnia et al. 2010</a>). This association for QRS duration generalized to African Americans (β = 2.43, p= 0.01). There were a total of three associations that generalized in African Americans for QT interval, two of which are in voltage gated channels <em>KCNH2</em> and <em>SCN5A</em> (<a href="/pmc/articles/PMC3743946/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The third association, rs12143842, is located in <em>NOS1AP</em>, which has been consistently associated with prolonged QT interval (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491990">Arking et al. 2006</a>). The effect of this variant accounts for more 1.5% of the trait variability in Europeans (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492012">Arking et al. 2006</a>) and 1.3% in our African Americans. For PR interval, two previously identified associations in the <em>SCN10A</em> gene have consistent directions of effect with African Americans (<a href="/pmc/articles/PMC3743946/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Non-synonymous variant rs6795970 (in the <em>SCN10A</em> gene) has been recently reported to have an association with increased PR interval in Asian and European-descent populations (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491994">Holm et al. 2010</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491984">Pfeufer et al. 2010</a>). In African Americans the magnitude of this effect (β= 3.67) is consistent with Europeans (β= 5.17) (<a href="/pmc/articles/PMC3743946/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>).</p><p id="P23" class="p p-last">There were 48 associations that did not generalize to African Americans with respect to direction of effect and statistical significance. GWAS associations identified in European descent populations may not generalize in African Americans for several reasons. More often than not, this is due to discordant minor allele frequency differences and linkage disequilibrium (LD) patterns, both of which impact statistical power. On average for SNPs that do not generalize, the MAF in the original population was significantly different compared to the MAF in our study population (p ≤ 0.01). The MAF was lower in African Americans compared to the original study population for 60% of the SNPs that do not generalize (<a href="/pmc/articles/PMC3743946/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Low minor allele frequency is directly correlated with power in genetic association studies. We also noticed this correlation in our analysis; 13/79 inadequately powered tests of association had a minor allele frequency &lt; 0.09.</p></div><div id="S15" class="sec"><h3 id="S15title">Replication of published GWAS in African Americans</h3><p id="P24" class="p p-first">As previously mentioned, there have been one recent GWAS and one fine-mapping study performed in African Americans for ECG traits (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491995">Avery et al. 2012</a>; <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492001">Smith et al. 2011</a>). The first GWAS in African Americans for PR interval described an association between <em>SCN5A</em> intronic rs3922844 and increased PR interval (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491952">Smith et al. 2011</a>). This association was also identified in an independent candidate gene study of <em>SCN5A</em> in African Americans (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491986">Jeff et al. 2011</a>). In the present study, rs3922844 was associated with PR interval at p = 0.01 and a consistent direction of effect (β = 3.06) compared with the published reports (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). To date rs3922844 has only been identified in African descent populations and thus was not included in this generalization analysis. Although not statistically significant, we observed similar results for the other SNPs published by <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492007">Smith et al (2011)</a> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Despite the lack of power, these data collectively suggest and confirm previous reports that <em>SCN5A</em> has pleiotropic effects and is associated with multiple ECG traits in African Americans (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491981">Jeff et al. 2011</a>).</p><p id="P25" class="p p-last">In the second study, GWAS regions identified for the QT interval in European and Asian populations were further fine-mapped in &gt;8,000 African Americans (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492006">Avery et al. 2012</a>). Approximately 40% of the associations reported in European or Asian populations generalized in the African American study population, which is consistent with this present study’s generalization results, despite differences in sample size. Three novel associations with QT interval specific to African Americans were also reported; of these, two of the reported SNPs were not on the Illumina 1M BeadChip used for this present study, and one SNP (rs12061601) did not replicate in our African American study population (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). The fine-mapping effort also highlighted differences in the index SNP associated with QT interval between African Americans and Europeans. We compared the index SNPs reported in African Americans from the fine-mapping study to our study population (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). Of the fifteen index SNPs reported, only six SNPs were directly genotyped in our study of which only two, rs1805120 and rs735951, trended towards significance (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>).</p></div><div id="S16" class="sec"><h3 id="S16title">The Effects of Linkage Disequilibrium</h3><p id="P26" class="p p-first">Linkage disequilibrium (LD) is often used in genetic association studies to select tagSNPs for genotyping that represent a region in the genome. While this method is cost effective, identifying an association with a tagSNP does not necessarily identify the true functional variant given that the un-assayed functional variant is most likely in LD with the assayed variant. TagSNPs are also population specific; therefore, associations identified in one population may not necessary generalize to another population if only the index variant is genotyped. Our observations in this African American population compared with European descent populations for ECG trait associations are consistent with known properties of tagSNPs. That is, there are differences in LD between African Americans and Europeans for known loci associated with PR interval, QRS duration, and QT interval. For the <em>SCN10A/SCN5A</em> and <em>NOS1AP</em> regions, we calculated pair-wise LD (r<sup>2</sup>) for each SNP pair. As expected, there is less linkage disequilibrium in African descent populations compared to European populations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 3 and 4</a>) (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491955">Rosenberg et al. 2010</a>). This lack of LD could account for the non-generalization of associations in African Americans originally identified in European descent populations.</p><p id="P27" class="p p-last">LD is often described as a “double-edged” sword: strong LD can be advantageous to identifying associations, but is it not useful when searching for the functional variants. Using populations with low amounts of LD can help fine map the functional variant (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492004">Rosenberg et al. 2010</a>). Therefore, index associations that robustly associate with ECG traits in European descent populations but do not generalize to African descent populations could be prioritized for fine-mapping studies to identify the functional variant. Also, further large discovery and fine-mapping studies should be performed in African descent populations to identify additional ECG-trait associated loci not found in other populations.</p></div><div id="S17" class="sec sec-last"><h3 id="S17title">Limitations and Strengths</h3><p id="P28" class="p p-first">There were several factors that limited our ability to detect novel associations at the GWAS level (at p&lt; 1.0 × 10 <sup>−8</sup>) with ECG traits. Our study population compared to recently published GWAS for ECG traits is relatively small and underpowered. Specifically, we were underpowered (&lt;80%) to detect effects that explain less than 10% of the trait variability at p&lt;1.0 × 10 <sup>−8</sup> even for common variants (MAF &gt;0.05). Power also limited our ability to generalize European/Asian identified associations to African Americans. As previously mentioned power is directly correlated to allele frequencies. Most GWAS fixed-content products are biased to common variation and based on LD patterns of European populations (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491976">Spencer et al. 2009</a>). Therefore, the SNPs previously identified may only be common in Europeans and may be rare in other populations such as African Americans, limiting our power to generalize these variants. It is important to note the power calculations we report in our generalization analysis were limited to the effect sizes detected in the original study. These effect sizes could be subject to the “winner’s curse” and may over estimate the true effect size (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492011">Xiao &amp; Boehnke 2009</a>; <a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491977">Zhong &amp; Prentice 2010</a>). Using these potentially overestimated effect sizes in our power calculations could consequently over estimate our power to detect these effects.</p><p id="P29">Another limitation to both the discovery and the generalization analyses was our liberal significance threshold. For discovery, a conservative Bonferroni threshold to correct for multiple testing is the generally accepted 5.0×10<sup>−8</sup>. Given our small sample size and this conservative threshold, we had sufficient power to detect large genetic effect sizes; however, these effects are not realistic given previously reported candidate gene and GWAS findings for these traits. To address this limitation, we arbitrarily choose a significance threshold of 1.0×10<sup>−6</sup> to identify possible real associations that would be missed by the conservative significance threshold. For generalization, we declared a generalization as having a consistent direction of effect and meeting a liberal significance threshold of p&lt;0.05. Given that the generalization analysis was limited to only a subset of loci that was previously identified, a genome-wide correction for multiple testing was not necessary.</p><p id="P30">Here we confirm the ability to use electronic medical records (EMRs) linked to DNA to identify genotype-phenotype relationships. We identified several potentially novel associations across the genome for heart rate, PR interval, QT interval, and QRS duration. Additionally, we generalized several associations identified by GWAS in European and Asian -descent populations to African Americans. Most importantly, we extensively characterized European and Asian identified associations in African Americans by reporting population specific information such as allele frequency and linkage disequilibrium differences, which may be important in both research and clinic-based settings.</p><p id="P31" class="p p-last">We successfully assessed the generalizability of European identified genetic associations for ECG traits in African Americans derived from an electronic medical record. Additionally, we report novel associations for ECG traits in African Americans. Replication of these findings in an independent African American population is needed to confirm these associations. Indeed, in the future, replication may be possible within the eMERGE network given that the network has expanded with respect to study sites and sample size. Most of the associations identified in European and Asian -descent populations failed to generalize due to power (as a consequence of sample size and low minor allele frequencies). However, there were several adequately powered associations that did not generalize to African Americans. These data suggest that the functional variant has yet to be identified for these traits by GWAS and highlight the need for future fine mapping studies in a large African-descent population.</p></div></div><div id="S18" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Data</h4><p id="P32"><strong>Supplementary Table 1. <em>SNP-trait associations significant in African Americans (p&lt;0.05) but trend in the opposite direction compared with previously identified association.</em></strong> We compared association results in African Americans to the association results from the original study accounting for the coded allele. This table only represents associations that are statistically significant in African Americans (p&lt;0.05) but have discordant effect sizes (expressed as a beta coefficient) compared to the original study. Abbreviations: CAF= coded allele frequency.</p><p id="P33"><strong>Supplementary Figure 1<em>. Genome-wide association results for heart rate in African Americans (n= 455).</em> (A) Manhattan plot of GWAS results for heart rate.</strong> Association results are plotted for each maker on the x-axis across the genome, and the −log of the p-value is plotted on the y axis. <strong>(B)Locus Zoom Plot for the most significant association.</strong> We plotted the local association results around 100kb of the most significant SNP rs1015003 from our GWAS of heart rate. The −log of the p-value is plotted on the y-axis, and each SNP is plotted by location on the x-axis. Pairwise linkage disequilibrium with the index SNP, measured by r<sup>2</sup>, is indicated by color. <strong>(C) Locus Zoom Plot for the <em>SERPINI1</em> region</strong>. We plotted the local association results around 100kb of the most significant SNP (rs13090836) region in the <em>SERPINI1</em> region.</p><p id="P34"><strong>Supplementary Figure 2. <em>Comparison of SCN5A/SCN10A Locus Zoom plots between African Americans (top panel) and European Americans from Denny et al (</em></strong><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492009">Denny et al. 2010</a><strong><em>) (bottom panel).</em></strong> Using locus specific plots, we compared the index SNP (indicated at the top of each plot) identified in African Americans to previously identified association results in European Americans from VGER.</p><p id="P35"><strong>Supplementary Table 2. <em>Effect size and p-values for GWAS-identified associations with a consistent direction of effect in African Americans compared to European descent populations.</em></strong> We compared the association results from GWAS-identified variants to African American association results for SNPs that have a consistent direction of effect but did not meet our significance threshold (p&lt;0.05) for all ECG traits.</p><p id="P36"><strong>Supplementary Table 3. <em>Coded Allele Frequencies for GWAS-identified SNPs in African Americans compared to the original study population.</em></strong> We calculated allele frequencies for all GWAS-identified SNPs in African Americans after accounting for the coded allele.</p><p id="P37"><strong>Supplementary Table 4. <em>Replication of previous associations for PR interval in VGER/NUgene African Americans.</em></strong> We tested previously reported associations with PR interval in African Americans from Smith et al. for replication in our study population (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491988">Smith et al. 2011</a>). Using the same coded allele as the original study, we performed tests of association for all reported SNPs genotyped in our study population using an additive genetic model. Abbreviations: CAF= coded allele frequency</p><p id="P38"><strong>Supplementary Table 5. <em>Replication of previous associations for QT interval in VGER/NUgene African Americans.</em></strong> We tested previously reported associations with QT interval in African Americans from Avery et al. for replication in our study population (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392491974">Avery et al. 2012</a>). Using the same coded allele as the original study, we performed tests of association for all reported SNPs that were also genotyped in our study population using an additive genetic model. Abbreviations: CAF= coded allele frequency, *= Novel association in African Americans for PR interval reported by Avery et al</p><p id="P39"><strong>Supplementary Figure 3. <em>Linkage Disequilibrium plots around the SCN10A region in YRI HapMap samples compared to CEU samples.</em></strong> Pairwise linkage disequilibrium was calculated for all SNPs with a MAF of &gt;0.05 within 100kb of the SCN10A gene using Seattle SNPs Genome Server (<a href="http://gvs.gs.washington.edu/GVS/index.jsp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gvs.gs.washington.edu/GVS/index.jsp</a>).</p><p id="P40"><strong>Supplementary Figure 4. <em>Linkage Disequilibrium plots in the NOS1AP gene in YRI HapMap samples compared to CEU samples.</em></strong> Pairwise linkage disequilibrium was calculated for all SNPs with a MAF of &gt;0.05 within the <em>NOS1AP</em> gene using Seattle SNPs Genome Server (<a href="http://gvs.gs.washington.edu/GVS/index.jsp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gvs.gs.washington.edu/GVS/index.jsp</a>).</p><div class="sup-box half_rhythm" id="d36e905"><a href="/pmc/articles/PMC3743946/bin/NIHMS463822-supplement-Supplementary_Data.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(7.7M, docx)</sup></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Acknowledgments</h2><div class="sec"><p id="__p1">We would like to thank the Vanderbilt University Center for Human Genetics Research, Computational Genomics Core (CGC) for their analytical and computational support. We also thank the team of clinicians and medical personnel for selecting study participants and collecting the data. This work was supported by NIH U01HG004609 (Northwestern University as part of eMERGE) and U01HG04603 (Vanderbilt University as part of eMERGE, also serving as the Administrative Coordinating Center). A portion of the dataset(s) used for the analyses described were obtained from Vanderbilt University Medical Center<sup>1</sup>s BioVU which is supported by institutional funding and by the Vanderbilt CTSA grant UL1 TR000445 from NCATS/NIH.</p></div></div><div id="idm140322332124160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140322332124160title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p2" class="p p-first"><a href="#SD1" rid="SD1" class=" supplementary-material">SUPPORTING INFORMATION</a></p><p id="__p3" class="p p-last">Additional <a href="#SD1" rid="SD1" class=" supplementary-material">supporting information</a> for this article may be found in the online version of this article:</p><p></p></div></div><div id="idm140322329453120" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140322329453120title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Abrahamsen B, Zhao J, Asante CO, Cendan CM, Marsh S, Martinez-Barbera JP, Nassar MA, Dickenson AH, Wood JN. The cell and molecular basis of mechanical, cold, and inflammatory pain. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">321</span>:702–705.</span> [<a href="/pubmed/18669863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+cell+and+molecular+basis+of+mechanical,+cold,+and+inflammatory+pain&amp;author=B+Abrahamsen&amp;author=J+Zhao&amp;author=CO+Asante&amp;author=CM+Cendan&amp;author=S+Marsh&amp;volume=321&amp;publication_year=2008&amp;pages=702-705&amp;pmid=18669863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Akopian AN, Souslova V, England S, Okuse K, Ogata N, Ure J, Smith A, Kerr BJ, McMahon SB, Boyce S, Hill R, Stanfa LC, Dickenson AH, Wood JN. The tetrodotoxin-resistant sodium channel SNS has a specialized function in pain pathways. <span><span class="ref-journal">Nat Neurosci. </span>1999;<span class="ref-vol">2</span>:541–548.</span> [<a href="/pubmed/10448219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=The+tetrodotoxin-resistant+sodium+channel+SNS+has+a+specialized+function+in+pain+pathways&amp;author=AN+Akopian&amp;author=V+Souslova&amp;author=S+England&amp;author=K+Okuse&amp;author=N+Ogata&amp;volume=2&amp;publication_year=1999&amp;pages=541-548&amp;pmid=10448219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Arking DE, Pfeufer A, Post W, Kao WH, Newton-Cheh C, Ikeda M, West K, Kashuk C, Akyol M, Perz S, Jalilzadeh S, Illig T, Gieger C, Guo CY, Larson MG, Wichmann HE, Marban E, O’Donnell CJ, Hirschhorn JN, Kaab S, Spooner PM, Meitinger T, Chakravarti A. A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:644–651.</span> [<a href="/pubmed/16648850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+genetic+variant+in+the+NOS1+regulator+NOS1AP+modulates+cardiac+repolarization&amp;author=DE+Arking&amp;author=A+Pfeufer&amp;author=W+Post&amp;author=WH+Kao&amp;author=C+Newton-Cheh&amp;volume=38&amp;publication_year=2006&amp;pages=644-651&amp;pmid=16648850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Avery CL, Sethupathy P, Buyske S, He Q, Lin DY, Arking DE, Carty CL, Duggan D, Fesinmeyer MD, Hindorff LA, Jeff JM, Klein L, Patton KK, Peters U, Shohet RV, Sotoodehnia N, Young AM, Kooperberg C, Haiman CA, Mohlke KL, Whitsel EA, North KE. Fine-Mapping and Initial Characterization of QT Interval Loci in African Americans. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002870.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3415454/">PMC free article</a>]</span> [<a href="/pubmed/22912591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Fine-Mapping+and+Initial+Characterization+of+QT+Interval+Loci+in+African+Americans&amp;author=CL+Avery&amp;author=P+Sethupathy&amp;author=S+Buyske&amp;author=Q+He&amp;author=DY+Lin&amp;volume=8&amp;publication_year=2012&amp;pages=e1002870&amp;pmid=22912591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Chambers JC, Zhao J, Terracciano CM, Bezzina CR, Zhang W, Kaba R, Navaratnarajah M, Lotlikar A, Sehmi JS, Kooner MK, Deng G, Siedlecka U, Parasramka S, El-Hamamsy I, Wass MN, Dekker LR, de Jong JS, Sternberg MJ, McKenna W, Severs NJ, de SR, Wilde AA, Anand P, Yacoub M, Scott J, Elliott P, Wood JN, Kooner JS. Genetic variation in SCN10A influences cardiac conduction. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:149–152.</span> [<a href="/pubmed/20062061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+SCN10A+influences+cardiac+conduction&amp;author=JC+Chambers&amp;author=J+Zhao&amp;author=CM+Terracciano&amp;author=CR+Bezzina&amp;author=W+Zhang&amp;volume=42&amp;publication_year=2010&amp;pages=149-152&amp;pmid=20062061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Crosslin DR, McDavid A, Weston N, Nelson SC, Zheng X, Hart E, de AM, Kullo IJ, McCarty CA, Doheny KF, Pugh E, Kho A, Hayes MG, Pretel S, Saip A, Ritchie MD, Crawford DC, Crane PK, Newton K, Li R, Mirel DB, Crenshaw A, Larson EB, Carlson CS, Jarvik GP. Genetic variants associated with the white blood cell count in 13,923 subjects in the eMERGE Network. <span><span class="ref-journal">Hum Genet. </span>2012;<span class="ref-vol">131</span>:639–52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3640990/">PMC free article</a>]</span> [<a href="/pubmed/22037903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Genetic+variants+associated+with+the+white+blood+cell+count+in+13,923+subjects+in+the+eMERGE+Network&amp;author=DR+Crosslin&amp;author=A+McDavid&amp;author=N+Weston&amp;author=SC+Nelson&amp;author=X+Zheng&amp;volume=131&amp;publication_year=2012&amp;pages=639-52&amp;pmid=22037903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Davis RL, Holohan PD, Shrimpton AE, Tatum AH, Daucher J, Collins GH, Todd R, Bradshaw C, Kent P, Feiglin D, Rosenbaum A, Yerby MS, Shaw CM, Lacbawan F, Lawrence DA. Familial encephalopathy with neuroserpin inclusion bodies. <span><span class="ref-journal">Am J Pathol. </span>1999;<span class="ref-vol">155</span>:1901–1913.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3277299/">PMC free article</a>]</span> [<a href="/pubmed/10595921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pathol&amp;title=Familial+encephalopathy+with+neuroserpin+inclusion+bodies&amp;author=RL+Davis&amp;author=PD+Holohan&amp;author=AE+Shrimpton&amp;author=AH+Tatum&amp;author=J+Daucher&amp;volume=155&amp;publication_year=1999&amp;pages=1901-1913&amp;pmid=10595921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Denny JC, Crawford DC, Ritchie MD, Bielinski SJ, Basford MA, Bradford Y, Chai HS, Bastarache L, Zuvich R, Peissig P, Carrell D, Ramirez AH, Pathak J, Wilke RA, Rasmussen L, Wang X, Pacheco JA, Kho AN, Hayes MG, Weston N, Matsumoto M, Kopp PA, Newton KM, Jarvik GP, Li R, Manolio TA, Kullo IJ, Chute CG, Chisholm RL, Larson EB, McCarty CA, Masys DR, Roden DM, de AM. Variants near FOXE1 are associated with hypothyroidism and other thyroid conditions: using electronic medical records for genome- and phenome-wide studies. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">89</span>:529–542.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3188836/">PMC free article</a>]</span> [<a href="/pubmed/21981779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Variants+near+FOXE1+are+associated+with+hypothyroidism+and+other+thyroid+conditions:+using+electronic+medical+records+for+genome-+and+phenome-wide+studies&amp;author=JC+Denny&amp;author=DC+Crawford&amp;author=MD+Ritchie&amp;author=SJ+Bielinski&amp;author=MA+Basford&amp;volume=89&amp;publication_year=2011&amp;pages=529-542&amp;pmid=21981779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Denny JC, Ritchie MD, Crawford DC, Schildcrout JS, Ramirez AH, Pulley JM, Basford MA, Masys DR, Haines JL, Roden DM. Identification of genomic predictors of atrioventricular conduction: using electronic medical records as a tool for genome science. <span><span class="ref-journal">Circulation. </span>2010;<span class="ref-vol">122</span>:2016–2021.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2991609/">PMC free article</a>]</span> [<a href="/pubmed/21041692" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Identification+of+genomic+predictors+of+atrioventricular+conduction:+using+electronic+medical+records+as+a+tool+for+genome+science&amp;author=JC+Denny&amp;author=MD+Ritchie&amp;author=DC+Crawford&amp;author=JS+Schildcrout&amp;author=AH+Ramirez&amp;volume=122&amp;publication_year=2010&amp;pages=2016-2021&amp;pmid=21041692&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Dumitrescu L, Ritchie MD, Brown-Gentry K, Pulley JM, Basford M, Denny JC, Oksenberg JR, Roden DM, Haines JL, Crawford DC. Assessing the accuracy of observer-reported ancestry in a biorepository linked to electronic medical records. <span><span class="ref-journal">Genet Med. </span>2010;<span class="ref-vol">12</span>:648–50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2952033/">PMC free article</a>]</span> [<a href="/pubmed/20733501" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Med&amp;title=Assessing+the+accuracy+of+observer-reported+ancestry+in+a+biorepository+linked+to+electronic+medical+records&amp;author=L+Dumitrescu&amp;author=MD+Ritchie&amp;author=K+Brown-Gentry&amp;author=JM+Pulley&amp;author=M+Basford&amp;volume=12&amp;publication_year=2010&amp;pages=648-50&amp;pmid=20733501&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Gauderman WJ, Morrison JM. <span><span class="ref-journal">QUANTO 1.1: A computer program for power and sample size calculations for genetic-epidemiology studies. </span>2006</span> Ref Type: Computer Program. Web address:  <a href="http://hydra.usc.edu/gxe/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://hydra.usc.edu/gxe/</a></span></li><li id="R13"><span class="element-citation">Grady BJ, Torstenson E, Dudek SM, Giles J, Sexton D, Ritchie MD. Finding unique filter sets in plato: a precursor to efficient interaction analysis in gwas data. <span><span class="ref-journal">Pac Symp Biocomput. </span>2010:315–326.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2903053/">PMC free article</a>]</span> [<a href="/pubmed/19908384" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pac+Symp+Biocomput&amp;title=Finding+unique+filter+sets+in+plato:+a+precursor+to+efficient+interaction+analysis+in+gwas+data&amp;author=BJ+Grady&amp;author=E+Torstenson&amp;author=SM+Dudek&amp;author=J+Giles&amp;author=D+Sexton&amp;publication_year=2010&amp;pages=315-326&amp;pmid=19908384&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Gretarsdottir S, Baas AF, Thorleifsson G, Holm H, den HM, de Vries JP, Kranendonk SE, Zeebregts CJ, van Sterkenburg SM, Geelkerken RH, van Rij AM, Williams MJ, Boll AP, Kostic JP, Jonasdottir A, Jonasdottir A, Walters GB, Masson G, Sulem P, Saemundsdottir J, Mouy M, Magnusson KP, Tromp G, Elmore JR, Sakalihasan N, Limet R, Defraigne JO, Ferrell RE, Ronkainen A, Ruigrok YM, Wijmenga C, Grobbee DE, Shah SH, Granger CB, Quyyumi AA, Vaccarino V, Patel RS, Zafari AM, Levey AI, Austin H, Girelli D, Pignatti PF, Olivieri O, Martinelli N, Malerba G, Trabetti E, Becker LC, Becker DM, Reilly MP, Rader DJ, Mueller T, Dieplinger B, Haltmayer M, Urbonavicius S, Lindblad B, Gottsater A, Gaetani E, Pola R, Wells P, Rodger M, Forgie M, Langlois N, Corral J, Vicente V, Fontcuberta J, Espana F, Grarup N, Jorgensen T, Witte DR, Hansen T, Pedersen O, Aben KK, de GJ, Holewijn S, Folkersen L, Franco-Cereceda A, Eriksson P, Collier DA, Stefansson H, Steinthorsdottir V, Rafnar T, Valdimarsson EM, Magnadottir HB, Sveinbjornsdottir S, Olafsson I, Magnusson MK, Palmason R, Haraldsdottir V, Andersen K, Onundarson PT, Thorgeirsson G, Kiemeney LA, Powell JT, Carey DJ, Kuivaniemi H, Lindholt JS, Jones GT, Kong A, Blankensteijn JD, Matthiasson SE, Thorsteinsdottir U, Stefansson K. Genome-wide association study identifies a sequence variant within the DAB2IP gene conferring susceptibility to abdominal aortic aneurysm. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:692–697.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4157066/">PMC free article</a>]</span> [<a href="/pubmed/20622881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+sequence+variant+within+the+DAB2IP+gene+conferring+susceptibility+to+abdominal+aortic+aneurysm&amp;author=S+Gretarsdottir&amp;author=AF+Baas&amp;author=G+Thorleifsson&amp;author=H+Holm&amp;author=HM+den&amp;volume=42&amp;publication_year=2010&amp;pages=692-697&amp;pmid=20622881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njolstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Lochen ML, Kong A, Thorsteinsdottir U, Stefansson K. Several common variants modulate heart rate, PR interval and QRS duration. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:117–122.</span> [<a href="/pubmed/20062063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Several+common+variants+modulate+heart+rate,+PR+interval+and+QRS+duration&amp;author=H+Holm&amp;author=DF+Gudbjartsson&amp;author=DO+Arnar&amp;author=G+Thorleifsson&amp;author=G+Thorgeirsson&amp;volume=42&amp;publication_year=2010&amp;pages=117-122&amp;pmid=20062063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Jeff JM, Brown-Gentry K, Buxbaum SG, Sarpong DF, Taylor HA, George AL, Jr, Roden DM, Crawford DC. SCN5A Variation is Associated with Electrocardiographic Traits in the Jackson Heart Study. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2011;<span class="ref-vol">4</span>:139–144.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080430/">PMC free article</a>]</span> [<a href="/pubmed/21325150" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=SCN5A+Variation+is+Associated+with+Electrocardiographic+Traits+in+the+Jackson+Heart+Study&amp;author=JM+Jeff&amp;author=K+Brown-Gentry&amp;author=SG+Buxbaum&amp;author=DF+Sarpong&amp;author=HA+Taylor&amp;volume=4&amp;publication_year=2011&amp;pages=139-144&amp;pmid=21325150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Lai LP, Deng CL, Moss AJ, Kass RS, Liang CS. Polymorphism of the gene encoding a human minimal potassium ion channel (minK) <span><span class="ref-journal">Gene. </span>1994;<span class="ref-vol">151</span>:339–340.</span> [<a href="/pubmed/7828904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Polymorphism+of+the+gene+encoding+a+human+minimal+potassium+ion+channel+(minK)&amp;author=LP+Lai&amp;author=CL+Deng&amp;author=AJ+Moss&amp;author=RS+Kass&amp;author=CS+Liang&amp;volume=151&amp;publication_year=1994&amp;pages=339-340&amp;pmid=7828904&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Marroni F, Pfeufer A, Aulchenko YS, Franklin CS, Isaacs A, Pichler I, Wild SH, Oostra BA, Wright AF, Campbell H, Witteman JC, Kaab S, Hicks AA, Gyllensten U, Rudan I, Meitinger T, Pattaro C, van Duijn CM, Wilson JF, Pramstaller PP. A genome-wide association scan of RR and QT interval duration in 3 European genetically isolated populations: the EUROSPAN project. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2009;<span class="ref-vol">2</span>:322–328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2760953/">PMC free article</a>]</span> [<a href="/pubmed/20031603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=A+genome-wide+association+scan+of+RR+and+QT+interval+duration+in+3+European+genetically+isolated+populations:+the+EUROSPAN+project&amp;author=F+Marroni&amp;author=A+Pfeufer&amp;author=YS+Aulchenko&amp;author=CS+Franklin&amp;author=A+Isaacs&amp;volume=2&amp;publication_year=2009&amp;pages=322-328&amp;pmid=20031603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">McCarty CA, Chisholm RL, Chute CG, Kullo IJ, Jarvik GP, Larson EB, Li R, Masys DR, Ritchie MD, Roden DM, Struewing JP, Wolf WA. The eMERGE Network: a consortium of biorepositories linked to electronic medical records data for conducting genomic studies. <span><span class="ref-journal">BMC Med Genomics. </span>2011;<span class="ref-vol">4</span>:13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3038887/">PMC free article</a>]</span> [<a href="/pubmed/21269473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genomics&amp;title=The+eMERGE+Network:+a+consortium+of+biorepositories+linked+to+electronic+medical+records+data+for+conducting+genomic+studies&amp;author=CA+McCarty&amp;author=RL+Chisholm&amp;author=CG+Chute&amp;author=IJ+Kullo&amp;author=GP+Jarvik&amp;volume=4&amp;publication_year=2011&amp;pages=13&amp;pmid=21269473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Newton-Cheh C, Eijgelsheim M, Rice KM, de Bakker PI, Yin X, Estrada K, Bis JC, Marciante K, Rivadeneira F, Noseworthy PA, Sotoodehnia N, Smith NL, Rotter JI, Kors JA, Witteman JC, Hofman A, Heckbert SR, O’Donnell CJ, Uitterlinden AG, Psaty BM, Lumley T, Larson MG, Stricker BH. Common variants at ten loci influence QT interval duration in the QTGEN Study. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:399–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701449/">PMC free article</a>]</span> [<a href="/pubmed/19305408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+ten+loci+influence+QT+interval+duration+in+the+QTGEN+Study&amp;author=C+Newton-Cheh&amp;author=M+Eijgelsheim&amp;author=KM+Rice&amp;author=PI+de+Bakker&amp;author=X+Yin&amp;volume=41&amp;publication_year=2009&amp;pages=399-406&amp;pmid=19305408&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Newton-Cheh C, Guo CY, Wang TJ, O’Donnell CJ, Levy D, Larson MG. Genome-wide association study of electrocardiographic and heart rate variability traits: the Framingham Heart Study. <span><span class="ref-journal">BMC Med Genet. </span>2007;<span class="ref-vol">8</span>(Suppl 1):S7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995612/">PMC free article</a>]</span> [<a href="/pubmed/17903306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Genome-wide+association+study+of+electrocardiographic+and+heart+rate+variability+traits:+the+Framingham+Heart+Study&amp;author=C+Newton-Cheh&amp;author=CY+Guo&amp;author=TJ+Wang&amp;author=CJ+O’Donnell&amp;author=D+Levy&amp;volume=8&amp;issue=Suppl+1&amp;publication_year=2007&amp;pages=S7&amp;pmid=17903306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Pendergrass SA, Dudek S, Crawford DCC, Ritchie M. Synthesis-View: visualization and interpretation of SNP association results for multi-cohort, multi-phenotype data and meta-analysis. <span><span class="ref-journal">BioData Mining. </span>2010;<span class="ref-vol">3</span>:10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3012023/">PMC free article</a>]</span> [<a href="/pubmed/21162740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BioData+Mining&amp;title=Synthesis-View:+visualization+and+interpretation+of+SNP+association+results+for+multi-cohort,+multi-phenotype+data+and+meta-analysis&amp;author=SA+Pendergrass&amp;author=S+Dudek&amp;author=DCC+Crawford&amp;author=M+Ritchie&amp;volume=3&amp;publication_year=2010&amp;pages=10&amp;pmid=21162740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Pfeufer A, van NC, Marciante KD, Arking DE, Larson MG, Smith AV, Tarasov KV, Muller M, Sotoodehnia N, Sinner MF, Verwoert GC, Li M, Kao WH, Kottgen A, Coresh J, Bis JC, Psaty BM, Rice K, Rotter JI, Rivadeneira F, Hofman A, Kors JA, Stricker BH, Uitterlinden AG, van Duijn CM, Beckmann BM, Sauter W, Gieger C, Lubitz SA, Newton-Cheh C, Wang TJ, Magnani JW, Schnabel RB, Chung MK, Barnard J, Smith JD, Van Wagoner DR, Vasan RS, Aspelund T, Eiriksdottir G, Harris TB, Launer LJ, Najjar SS, Lakatta E, Schlessinger D, Uda M, Abecasis GR, Muller-Myhsok B, Ehret GB, Boerwinkle E, Chakravarti A, Soliman EZ, Lunetta KL, Perz S, Wichmann HE, Meitinger T, Levy D, Gudnason V, Ellinor PT, Sanna S, Kaab S, Witteman JC, Alonso A, Benjamin EJ, Heckbert SR. Genome-wide association study of PR interval. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:153–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850197/">PMC free article</a>]</span> [<a href="/pubmed/20062060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+PR+interval&amp;author=A+Pfeufer&amp;author=NC+van&amp;author=KD+Marciante&amp;author=DE+Arking&amp;author=MG+Larson&amp;volume=42&amp;publication_year=2010&amp;pages=153-159&amp;pmid=20062060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Post W, Shen H, Damcott C, Arking DE, Kao WH, Sack PA, Ryan KA, Chakravarti A, Mitchell BD, Shuldiner AR. Associations between genetic variants in the NOS1AP (CAPON) gene and cardiac repolarization in the old order Amish. <span><span class="ref-journal">Hum Hered. </span>2007;<span class="ref-vol">64</span>:214–219.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2880727/">PMC free article</a>]</span> [<a href="/pubmed/17565224" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Associations+between+genetic+variants+in+the+NOS1AP+(CAPON)+gene+and+cardiac+repolarization+in+the+old+order+Amish&amp;author=W+Post&amp;author=H+Shen&amp;author=C+Damcott&amp;author=DE+Arking&amp;author=WH+Kao&amp;volume=64&amp;publication_year=2007&amp;pages=214-219&amp;pmid=17565224&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, Boehnke M, Abecasis GR, Willer CJ. LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Ramirez AH, Schildcrout JS, Blakemore DL, Masys DR, Pulley JM, Basford MA, Roden DM, Denny JC. Modulators of normal electrocardiographic intervals identified in a large electronic medical record. <span><span class="ref-journal">Heart Rhythm. </span>2011;<span class="ref-vol">8</span>:271–277.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3057498/">PMC free article</a>]</span> [<a href="/pubmed/21044898" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Heart+Rhythm&amp;title=Modulators+of+normal+electrocardiographic+intervals+identified+in+a+large+electronic+medical+record&amp;author=AH+Ramirez&amp;author=JS+Schildcrout&amp;author=DL+Blakemore&amp;author=DR+Masys&amp;author=JM+Pulley&amp;volume=8&amp;publication_year=2011&amp;pages=271-277&amp;pmid=21044898&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Roden DM, Pulley JM, Basford MA, Bernard GR, Clayton EW, Balser JR, Masys DR. Development of a large-scale de-identified DNA biobank to enable personalized medicine. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2008;<span class="ref-vol">84</span>(3):362–369.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3763939/">PMC free article</a>]</span> [<a href="/pubmed/18500243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Development+of+a+large-scale+de-identified+DNA+biobank+to+enable+personalized+medicine&amp;author=DM+Roden&amp;author=JM+Pulley&amp;author=MA+Basford&amp;author=GR+Bernard&amp;author=EW+Clayton&amp;volume=84&amp;issue=3&amp;publication_year=2008&amp;pages=362-369&amp;pmid=18500243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Rosenberg NA, Huang L, Jewett EM, Szpiech ZA, Jankovic I, Boehnke M. Genome-wide association studies in diverse populations. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>(5):356–366.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079573/">PMC free article</a>]</span> [<a href="/pubmed/20395969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+in+diverse+populations&amp;author=NA+Rosenberg&amp;author=L+Huang&amp;author=EM+Jewett&amp;author=ZA+Szpiech&amp;author=I+Jankovic&amp;volume=11&amp;issue=5&amp;publication_year=2010&amp;pages=356-366&amp;pmid=20395969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Smith JG, Lowe JK, Kovvali S, Maller JB, Salit J, Daly MJ, Stoffel M, Altshuler DM, Friedman JM, Breslow JL, Newton-Cheh C. Genome-wide association study of electrocardiographic conduction measures in an isolated founder population: Kosrae. <span><span class="ref-journal">Heart Rhythm. </span>2009;<span class="ref-vol">6</span>(5):634–641.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2673462/">PMC free article</a>]</span> [<a href="/pubmed/19389651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Heart+Rhythm&amp;title=Genome-wide+association+study+of+electrocardiographic+conduction+measures+in+an+isolated+founder+population:+Kosrae&amp;author=JG+Smith&amp;author=JK+Lowe&amp;author=S+Kovvali&amp;author=JB+Maller&amp;author=J+Salit&amp;volume=6&amp;issue=5&amp;publication_year=2009&amp;pages=634-641&amp;pmid=19389651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Smith JG, Magnani JW, Palmer C, Meng YA, Soliman EZ, Musani SK, Kerr KF, Schnabel RB, Lubitz SA, Sotoodehnia N, Redline S, Pfeufer A, Muller M, Evans DS, Nalls MA, Liu Y, Newman AB, Zonderman AB, Evans MK, Deo R, Ellinor PT, Paltoo DN, Newton-Cheh C, Benjamin EJ, Mehra R, Alonso A, Heckbert SR, Fox ER. Genome-wide association studies of the PR interval in African Americans. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>(2):e1001304.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037415/">PMC free article</a>]</span> [<a href="/pubmed/21347284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+studies+of+the+PR+interval+in+African+Americans&amp;author=JG+Smith&amp;author=JW+Magnani&amp;author=C+Palmer&amp;author=YA+Meng&amp;author=EZ+Soliman&amp;volume=7&amp;issue=2&amp;publication_year=2011&amp;pages=e1001304&amp;pmid=21347284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Sotoodehnia N, Isaacs A, de Bakker PI, Dorr M, Newton-Cheh C, Nolte IM, van der HP, Muller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Kottgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo LI, Estrada K, Volker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Volzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van HG, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O’Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te MG, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kaab S, Arking DE. Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>(12):1068–1076.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3338195/">PMC free article</a>]</span> [<a href="/pubmed/21076409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+in+22+loci+are+associated+with+QRS+duration+and+cardiac+ventricular+conduction&amp;author=N+Sotoodehnia&amp;author=A+Isaacs&amp;author=PI+de+Bakker&amp;author=M+Dorr&amp;author=C+Newton-Cheh&amp;volume=42&amp;issue=12&amp;publication_year=2010&amp;pages=1068-1076&amp;pmid=21076409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Spencer CC, Su Z, Donnelly P, Marchini J. Designing genome-wide association studies: sample size, power, imputation, and the choice of genotyping chip. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>(5):e1000477.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2688469/">PMC free article</a>]</span> [<a href="/pubmed/19492015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Designing+genome-wide+association+studies:+sample+size,+power,+imputation,+and+the+choice+of+genotyping+chip&amp;author=CC+Spencer&amp;author=Z+Su&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;issue=5&amp;publication_year=2009&amp;pages=e1000477&amp;pmid=19492015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Splawski I, Timothy KW, Tateyama M, Clancy CE, Malhotra A, Beggs AH, Cappuccio FP, Sagnella GA, Kass RS, Keating MT. Variant of SCN5A sodium channel implicated in risk of cardiac arrhythmia. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">297</span>(5585):1333–1336.</span> [<a href="/pubmed/12193783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Variant+of+SCN5A+sodium+channel+implicated+in+risk+of+cardiac+arrhythmia&amp;author=I+Splawski&amp;author=KW+Timothy&amp;author=M+Tateyama&amp;author=CE+Clancy&amp;author=A+Malhotra&amp;volume=297&amp;issue=5585&amp;publication_year=2002&amp;pages=1333-1336&amp;pmid=12193783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Turner SD, Berg RL, Linneman JG, Peissig PL, Crawford DC, Denny JC, Roden DM, McCarty CA, Ritchie MD, Wilke RA. Knowledge-driven multi-locus analysis reveals gene-gene interactions influencing HDL cholesterol level in two independent EMR-linked biobanks. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>(5):e19586.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3092760/">PMC free article</a>]</span> [<a href="/pubmed/21589926" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Knowledge-driven+multi-locus+analysis+reveals+gene-gene+interactions+influencing+HDL+cholesterol+level+in+two+independent+EMR-linked+biobanks&amp;author=SD+Turner&amp;author=RL+Berg&amp;author=JG+Linneman&amp;author=PL+Peissig&amp;author=DC+Crawford&amp;volume=6&amp;issue=5&amp;publication_year=2011&amp;pages=e19586&amp;pmid=21589926&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Wang Q, Curran ME, Splawski I, Burn TC, Millholland JM, VanRaay TJ, Shen J, Timothy KW, Vincent GM, de JT, Schwartz PJ, Toubin JA, Moss AJ, Atkinson DL, Landes GM, Connors TD, Keating MT. Positional cloning of a novel potassium channel gene: KVLQT1 mutations cause cardiac arrhythmias. <span><span class="ref-journal">Nat Genet. </span>1996;<span class="ref-vol">12</span>(1):17–23.</span> [<a href="/pubmed/8528244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Positional+cloning+of+a+novel+potassium+channel+gene:+KVLQT1+mutations+cause+cardiac+arrhythmias&amp;author=Q+Wang&amp;author=ME+Curran&amp;author=I+Splawski&amp;author=TC+Burn&amp;author=JM+Millholland&amp;volume=12&amp;issue=1&amp;publication_year=1996&amp;pages=17-23&amp;pmid=8528244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Wang Q, Shen J, Li Z, Timothy K, Vincent GM, Priori SG, Schwartz PJ, Keating MT. Cardiac sodium channel mutations in patients with long QT syndrome, an inherited cardiac arrhythmia. <span><span class="ref-journal">Hum Mol Genet. </span>1995a;<span class="ref-vol">4</span>(9):1603–1607.</span> [<a href="/pubmed/8541846" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Cardiac+sodium+channel+mutations+in+patients+with+long+QT+syndrome,+an+inherited+cardiac+arrhythmia&amp;author=Q+Wang&amp;author=J+Shen&amp;author=Z+Li&amp;author=K+Timothy&amp;author=GM+Vincent&amp;volume=4&amp;issue=9&amp;publication_year=1995a&amp;pages=1603-1607&amp;pmid=8541846&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Wang Q, Shen J, Splawski I, Atkinson D, Li Z, Robinson JL, Moss AJ, Towbin JA, Keating MT. SCN5A mutations associated with an inherited cardiac arrhythmia, long QT syndrome. <span><span class="ref-journal">Cell. </span>1995b;<span class="ref-vol">80</span>(5):805–811.</span> [<a href="/pubmed/7889574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=SCN5A+mutations+associated+with+an+inherited+cardiac+arrhythmia,+long+QT+syndrome&amp;author=Q+Wang&amp;author=J+Shen&amp;author=I+Splawski&amp;author=D+Atkinson&amp;author=Z+Li&amp;volume=80&amp;issue=5&amp;publication_year=1995b&amp;pages=805-811&amp;pmid=7889574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Weir BS, Cockerham CC. Estimating <em>F</em>- Statistics for the Analysis of Population Structure. <span><span class="ref-journal">Evolution. </span>1984;<span class="ref-vol">38</span>(6):1358–1370.</span> [<a href="/pubmed/28563791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Estimating+F-+Statistics+for+the+Analysis+of+Population+Structure&amp;author=BS+Weir&amp;author=CC+Cockerham&amp;volume=38&amp;issue=6&amp;publication_year=1984&amp;pages=1358-1370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Xiao R, Boehnke M. Quantifying and correcting for the winner’s curse in genetic association studies. <span><span class="ref-journal">Genet Epidemiol. </span>2009;<span class="ref-vol">33</span>(5):453–462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2706290/">PMC free article</a>]</span> [<a href="/pubmed/19140131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Quantifying+and+correcting+for+the+winner’s+curse+in+genetic+association+studies&amp;author=R+Xiao&amp;author=M+Boehnke&amp;volume=33&amp;issue=5&amp;publication_year=2009&amp;pages=453-462&amp;pmid=19140131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">Zhong H, Prentice RL. Correcting “winner’s curse” in odds ratios from genomewide association findings for major complex human diseases. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>(1):78–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2796696/">PMC free article</a>]</span> [<a href="/pubmed/19639606" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Correcting+“winner’s+curse”+in+odds+ratios+from+genomewide+association+findings+for+major+complex+human+diseases&amp;author=H+Zhong&amp;author=RL+Prentice&amp;volume=34&amp;issue=1&amp;publication_year=2010&amp;pages=78-91&amp;pmid=19639606&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Zimmermann K, Leffler A, Babes A, Cendan CM, Carr RW, Kobayashi J, Nau C, Wood JN, Reeh PW. Sensory neuron sodium channel Nav1.8 is essential for pain at low temperatures. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>(7146):855–858.</span> [<a href="/pubmed/17568746" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Sensory+neuron+sodium+channel+Nav1.8+is+essential+for+pain+at+low+temperatures&amp;author=K+Zimmermann&amp;author=A+Leffler&amp;author=A+Babes&amp;author=CM+Cendan&amp;author=RW+Carr&amp;volume=447&amp;issue=7146&amp;publication_year=2007&amp;pages=855-858&amp;pmid=17568746&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Zuvich RL, Armstrong LL, Bielinski SJ, Bradford Y, Carlson CS, Crawford DC, Crenshaw AT, de AM, Doheny KF, Haines JL, Hayes MG, Jarvik GP, Jiang L, Kullo IJ, Li R, Ling H, Manolio TA, Matsumoto ME, McCarty CA, McDavid AN, Mirel DB, Olson LM, Paschall JE, Pugh EW, Rasmussen LV, Rasmussen-Torvik LJ, Turner SD, Wilke RA, Ritchie MD. Pitfalls of merging GWAS data: lessons learned in the eMERGE network and quality control procedures to maintain high data quality. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>(8):887–898.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3592376/">PMC free article</a>]</span> [<a href="/pubmed/22125226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Pitfalls+of+merging+GWAS+data:+lessons+learned+in+the+eMERGE+network+and+quality+control+procedures+to+maintain+high+data+quality&amp;author=RL+Zuvich&amp;author=LL+Armstrong&amp;author=SJ+Bielinski&amp;author=Y+Bradford&amp;author=CS+Carlson&amp;volume=35&amp;issue=8&amp;publication_year=2011&amp;pages=887-898&amp;pmid=22125226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3743946&amp;issue-id=226309&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140322333196976title">SUMMARY</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">MATERIALS AND METHODS</a></li><li><a href="#S7title">RESULTS</a></li><li><a href="#S12title">DISCUSSION</a></li><li><a href="#S18title">Supplementary Material</a></li><li><a href="#S19title">Acknowledgments</a></li><li><a href="#idm140322332124160title">Footnotes</a></li><li><a href="#idm140322329453120title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3743946/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3743946/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3743946/pdf/nihms463822.pdf">PDF (375K)</a></li> | <li><a href="#" data-citationid="PMC3743946" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3743946%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3743946%2F&amp;text=Generalization%20of%20Variants%20Identified%20by%20Genome-wide%20Association%20Studies%20for%20Electrocardiographic%20Traits%20in%20African%20Americans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3743946%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23534349" data-aiid="3743946" data-aid="3743946" data-iid="226309" data-domainid="319" data-domain="nihpa" data-accid="PMC3743946" data-md5="3cbe1d4dad02372518ef35e557a7b233"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:32:20-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C217BDD54ED51000000000A140681&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3743946%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3743946/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>