<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3419811/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association study of venous thromboembolism identifies risk variants in chromosomes 1q24.2 and 9q</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3419811/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of thrombosis and haemostasis : JTH"><meta name="citation_title" content="A genome-wide association study of venous thromboembolism identifies risk variants in chromosomes 1q24.2 and 9q"><meta name="citation_authors" content="John A. Heit, Sebastian M. Armasu, Yan W. Asmann, Julie M. Cunningham, Martha E. Matsumoto, Tanya M. Petterson, Mariza de Andrade"><meta name="citation_date" content="August 2012"><meta name="citation_issue" content="8"><meta name="citation_volume" content="10"><meta name="citation_firstpage" content="1521"><meta name="citation_doi" content="10.1111/j.1538-7836.2012.04810.x"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3419811/?report=abstract"><meta name="citation_pmid" content="22672568"><meta name="DC.Title" content="A genome-wide association study of venous thromboembolism identifies risk variants in chromosomes 1q24.2 and 9q"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="John A. Heit"><meta name="DC.Contributor" content="Sebastian M. Armasu"><meta name="DC.Contributor" content="Yan W. Asmann"><meta name="DC.Contributor" content="Julie M. Cunningham"><meta name="DC.Contributor" content="Martha E. Matsumoto"><meta name="DC.Contributor" content="Tanya M. Petterson"><meta name="DC.Contributor" content="Mariza de Andrade"><meta name="DC.Date" content="2012 Aug"><meta name="DC.Identifier" content="10.1111/j.1538-7836.2012.04810.x"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association study of venous thromboembolism identifies risk variants in chromosomes 1q24.2 and 9q"><meta property="og:type" content="article"><meta property="og:description" content="To identify venous thromboembolism (VTE) disease-susceptibility genes.We performed in silico genome wide association (GWAS) analyses using genotype data imputed to ~2.5 million single nucleotide polymorphisms (SNPs) from adults with objectively-diagnosed ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3419811/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3419811/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3419811/pdf/nihms383857.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A3BC8DD53CC710000000007950546.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419811%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gds" class="last">GEO DataSets</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3419811</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Thromb Haemost</a></span>. Author manuscript; available in PMC 2013 Aug 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22672568" target="pmc_ext" ref="reftype=publisher&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">J Thromb Haemost. 2012 Aug; 10(8): 1521–1531. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1111%2Fj.1538-7836.2012.04810.x" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1111/j.1538-7836.2012.04810.x</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3419811</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS383857</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22672568">22672568</a></div></div></div></div><h1 class="content-title">A genome-wide association study of venous thromboembolism identifies risk variants in chromosomes 1q24.2 and 9q</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Heit%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22672568">John A. Heit</a>, M.D., <a href="/pubmed/?term=Armasu%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22672568">Sebastian M. Armasu</a>, M.S., <a href="/pubmed/?term=Asmann%20YW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22672568">Yan W. Asmann</a>, Ph.D., <a href="/pubmed/?term=Cunningham%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22672568">Julie M. Cunningham</a>, Ph.D., <a href="/pubmed/?term=Matsumoto%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22672568">Martha E. Matsumoto</a>, B.S., <a href="/pubmed/?term=Petterson%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22672568">Tanya M. Petterson</a>, M.S., and  <a href="/pubmed/?term=de%20Andrade%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22672568">Mariza de Andrade</a>, Ph.D.</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140106501645120_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140106501645120_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140106501645120_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">Division of Cardiovascular Diseases (JAH), Department of Internal Medicine; the Division of Biomedical Statistics and Informatics (SMA, YWA, MEM, TMP, MdA), Department of Health Sciences Research; and the Division of Experimental Pathology (JMC), Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN, USA</div><div id="FN1">Address Correspondence to: John A. Heit, M.D., Stabile 6-Hematology Research, Mayo Clinic, 200 First Street, SW, Rochester, MN 55905, 507-284-4634 (voice), 507-266-9302 (fax), <a href="mailto:dev@null" data-email="ude.oyam@nhoj.tieh" class="oemail">ude.oyam@nhoj.tieh</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140106501645120_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22672568" target="pmc_ext" ref="reftype=publisher&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">J Thromb Haemost</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3419811/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental.</strong></div><div><a href="/pmc/articles/PMC3419811/bin/NIHMS383857-supplement-Supplemental.doc" data-ga-action="click_feat_suppl">NIHMS383857-supplement-Supplemental.doc</a><span style="color:gray"> (2.3M)</span></div><div class="small guid">GUID:&nbsp;20F2D2E1-11A3-4EBD-A863-E5096021A85B</div></div></dd></dl></div><div id="idm140106493957056" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140106493957056title">Summary</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Objectives</h3><p id="P1" class="p p-first-last">To identify venous thromboembolism (<strong>VTE</strong>) disease-susceptibility genes.</p></div><div id="S2" class="sec"><h3 id="S2title">Patients/Methods</h3><p id="P2" class="p p-first-last">We performed <em>in silico</em> genome wide association (<strong>GWAS</strong>) analyses using genotype data imputed to ~2.5 million single nucleotide polymorphisms (<strong>SNPs</strong>) from adults with objectively-diagnosed VTE (n=1503), and controls frequency-matched on age and sex (n=1459; discovery population). SNPs exceeding genome-wide significance were replicated in a separate population (VTE cases, n=1407; controls, n=1418). Genes associated with VTE were resequenced.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">Seven SNPs exceeded genome-wide significance (<em>P</em> &lt; 5 × 10<sup>-8</sup>); four on chromosome 1q24.2 (<em>F5</em> rs6025 [Factor V Leiden], <em>BLZF1</em> rs7538157, <em>NME7</em> rs16861990 and <em>SLC19A2</em> rs2038024) and three on chromosome 9q34.2 (<em>ABO</em> rs2519093 [<em>ABO</em> intron 1], rs495828, rs8176719 [ABO blood type O allele]). The replication study confirmed a significant association of <em>F5, NME7</em>, and <em>ABO</em> with VTE. However, <em>F5</em> was the main signal on 1q24.2 as only <em>ABO</em> SNPs remained significantly associated with VTE after adjusting for <em>F5</em> rs6025. This 1q24.2 region was shown to be inherited as a haplotype block. <em>ABO</em> resequencing identified 15 novel single nucleotide variations (<strong>SNV</strong>) in <em>ABO</em> intron 6 and the <em>ABO</em> 3’ UTR that were strongly associated with VTE (<em>P</em> &lt; 10<sup>-4</sup>) and belonged to three distinct linkage disequilibrium (LD) blocks; none were in LD with <em>ABO</em> rs8176719 or rs2519093. Our sample size provided 80% power to detect odds ratios=2.0 and 1.51 for minor allele frequencies=0.05 and 0.5, respectively (α=1 × 10<sup>-8</sup>; 1% VTE prevalence).</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">Aside from <em>F5</em> rs6025, <em>ABO</em> rs8176719 and rs2519093, and <em>F2</em> rs1799963, additional common and high VTE-risk SNPs among whites are unlikely.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">venous thromboembolism, deep vein thrombosis, pulmonary embolism, genetics, genome-wide scan, epidemiology</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first-last">Venous thromboembolism, consisting of deep vein thrombosis (<strong>DVT</strong>) and its complication, pulmonary embolism (<strong>PE</strong>), is a common and potentially fatal disease. The overall annual VTE incidence among USA whites is 108 per 100,000 person-years; incidence rates increase exponentially with age for both men and women, and for both DVT and PE.[<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909735">1</a>] For almost one-quarter of PE patients, the initial clinical presentation is sudden death.[<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909770">2</a>] Family and twin studies suggest that VTE is highly heritable (<em>h<sup>2</sup></em>=0.62) and likely results from multigenic action as well as environmental exposures.[<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909740">3</a>-<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909767">5</a>] Several genetic disorders or mutations that affect the regulation of hemostasis have been associated with VTE in small case-control studies or observed in high VTE-risk pedigrees.[<a href="#R6" rid="R6" class=" bibr popnode">6</a>] These include mutations that impair the anticoagulant pathway, up-regulate (e.g., prothrombin G20210A) or impair downregulation of the procoagulant pathway (e.g., Factor V Leiden), downregulate fibrinolysis, and up-regulate innate immunity. In a SNP-based candidate gene, case-control study, we tested genetic variation within all known important components of the anticoagulant, procoagulant, fibrinolytic and innate immunity pathways for an independent association with VTE and found one or more SNPs within <em>ABO, F2, F5, F11, KLKB1, SELP</em> and <em>SCUBE1</em> were significantly associated with VTE, including Factor V Leiden (<em>F5</em> rs6025 T [minor] allele), prothrombin G20210A, ABO non-O blood type (<em>ABO</em> rs8176719, odds ratio [<strong>OR</strong>]=1.47, <em>P</em>=5.68×10<sup>-12</sup>), and a novel association with <em>ABO</em> rs2519093 (OR=1.68, <em>P</em>=8.08×10<sup>-16</sup>) that was independent of blood type.[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909737">7</a>] A GWAS confirmed associations of VTE with Factor V Leiden and with <em>ABO</em> loci (blood groups non-O and A2),[<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909772">8</a>] and a second GWAS with a larger sample size and more dense genome-wide scan genotype data found associations of VTE with the <em>F11</em> and <em>FGG</em> loci.[<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909790">9</a>] We hypothesized that genes within other pathways also are VTE disease-susceptibility genes. To test this hypothesis, we performed a genome-wide association study on the same VTE case-control (discovery) population used for our candidate gene study, and replicated all SNPs exceeding genome-wide significance in a separate VTE case-control (replication) population. Finally, we performed gene resequencing in order to identify potentially “causative” genetic variation in LD with haplotype-tagged SNPs that were associated with VTE.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Materials and Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Discovery population</h3><p id="P6" class="p p-first-last">The discovery study population consisted 1503 VTE cases and 1459 controls (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>) from a candidate gene (<strong>CG</strong>) study, as previously described.[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909779">7</a>] A subset of the CG study, 1270 VTE cases and 1302 controls, who consented to sending their leukocyte genomic DNA outside of the Mayo Clinic, and newly recruited cases and controls, have genome-wide scan (<strong>GWS</strong>) genotype data. The Mayo Clinic IRB approved the study protocol as well as the data sharing protocol administered through Database for Genotypes and Phenotypes (<strong>dbGAP</strong>).</p></div><div id="S8" class="sec"><h3 id="S8title">Genotyping</h3><div id="S9" class="sec sec-first"><p></p><h4 id="S9title" class="inline">Candidate Gene (CG) study </h4><p id="P7" class="p p-first-last">Leukocyte genomic DNA was extracted, quantified and diluted to the appropriate concentration for Illumina Infinium iSelect genotyping within the Genotyping Core Laboratory of the Mayo Clinic Advanced GenomicsTechnology Center, as previously described.[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909752">7</a>] Genotyping controls included 2% sample replicates and a CEPH trio for quality control. In addition, case and control DNA sample addresses were randomly assigned across both the 96-well plate as well as the 12-address iSelect BeadChip, ensuring approximately equal numbers of case and control DNA samples by each stratum to avoid potential plate and chip effects, respectively. Genotyping results from high-quality control DNA (SNP call rate ≥95%) were used to generate a cluster algorithm. Of the 14,612 SNPs submitted to Illumina from 764 genes within the anticoagulant, procoagulant, fibrinolytic and innate immunity pathways, 1,585 SNPs (covering 1,100 LD bins) failed manufacture, leaving 13,027 successfully genotyped SNPs. Four additional SNPs were custom genotyped on TaqMan, leaving 13,031 SNPs for analysis.</p></div><div id="S10" class="sec sec-last"><p></p><h4 id="S10title" class="inline">Genome-wide Scan (GWS) study </h4><p id="P8" class="p p-first-last">The Center for Inherited Disease Research (<strong>CIDR</strong>), one of two genotyping centers supported by the Gene, Environment Association Studies Consortium (<strong>GENEVA</strong>) consortium,[<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909733">10</a>] genotyped DNA samples using the Illumina Human660-Quad v.1_A BeadChip (Illumina) and DNA from a subset of CG cases and controls that consented to sending their DNA outside Mayo. Similar to the CG study, case and control DNA sample addresses were randomly assigned across 96-well plates provided by CIDR while ensuring roughly equal percentages of cases and controls within each plate. Genotype clusters for each SNP were determined using the Illumina BeadStudio Module (version 3.3.7), and combined intensity data from 96.6% of samples were used to define clusters and call genotypes. Overall, 99.9% of samples attempted (2,597 of 2,600 total) passed quality-control standards. Genotypes were not called if the quality score from BeadStudio was &lt;0.15. Both the mean SNP call rates and the mean sample call rates were 99.9%. Genotypes were released for 560,816 SNPs (99.88% of those attempted). Genotypes were not released for autosomal SNPs with call rates &lt;85%, &gt;1 HapMap replicate error, &gt;9% difference in call rate between genders, &lt;0.2 cluster separation or &gt;11% difference in heterozygote frequency. Duplicate samples from both HapMap and the study were included on each plate. Four HapMap trios (parents and child) were included to detect Mendelian inheritance errors (i.e., whether the alleles are incorrectly passed from each parent to the child). Reproducibility rates in the raw data were 99.99% among 32 duplicated study subjects. The genotype concordance in the 134 HapMap control samples was 99.8%.</p></div></div><div id="S11" class="sec"><h3 id="S11title">Quality-control steps</h3><p id="P9" class="p p-first-last">For both CG and GWS studies, four main categories of quality-control flags were set for autosomal SNPs: (i) unacceptably high rates of missing genotype calls (&gt;5%), (ii) low minor allele frequency (<strong>MAF</strong>) (&lt;0.005), (iii) unacceptably high rates of Mendelian errors (&gt;1) and (iv) deviation from Hardy-Weinberg equilibrium (<strong>HWE</strong>, <em>P</em>&lt;E-05). For the CG study, of the SNPs perfoming well (12,477/13,031) 181 SNPs were excluded due a MAF&lt;0.005 (n=127) or a call rate &lt;0.95 (n=54), leaving 12,296 SNPs (covering 10,456 bins) for the association analysis. For the GWS study, we flagged 1,965 SNPs with &gt;5% missing genotypes. A total of 1,683 (1,115/1,135) SNPs were flagged overall (cases/controls) for HWE deviation at <em>P</em>&lt;E-05.</p></div><div id="S12" class="sec"><h3 id="S12title">Population stratification</h3><p id="P10" class="p p-first-last">To test for “cryptic relatedness” in our population, we conducted a population structure analysis on all participating subjects to document genetic diversity among our population. Using 494 Ancestry Informative Markers (<strong>AIMS</strong>) available on both our candidate gene and GWA studies,[<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909753">11</a>] we ran STRUCTURE on 2962 participants.[<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909780">12</a>] The triangle plot provided a graphical representation of genetic structure of our participants plus 209 unrelated individuals from HapMap phase II populations (Yorubans [<strong>YRI</strong>]; European-Americans from the CEPH collection [<strong>CEU</strong>]; Chinese from Beijing [<strong>CHB</strong>]; and Japanese from Tokyo [<strong>JPT</strong>]), giving a clear sense of how our participants fall among the HapMap reference populations (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 1</a>).</p></div><div id="S13" class="sec"><h3 id="S13title">Replication population</h3><p id="P11" class="p p-first-last">SNPs exceeding genome-wide significance in the discovery population and not in LD within a gene were genotyped in a separate replication population of 2,825 individuals (VTE cases, n=1407; controls, n=1418; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>). Replication study leukocyte genomic DNA samples were largely from Olmsted County, MN, USA residents with objectively-diagnosed incident VTE over the 45-year period, 1966-2010, and previously-matched Olmsted County resident controls, as previously described.[<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909755">13</a>-<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909727">15</a>] The replication case and control sample size was augmented by leukocyte DNA from non-Olmsted County Mayo Clinic patients identified as described for the CG study population.[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909766">7</a>]</p></div><div id="S14" class="sec"><h3 id="S14title">DNA resequencing</h3><p id="P12" class="p p-first">We selected a subset of the discovery VTE cases and controls for resequencing to take advantage of the joint configuration of two <em>ABO</em> SNPs independently associated with VTE, rs8176719 (<em>ABO</em> exon 6 deletion determining type O blood group) and rs2519093 (<em>ABO</em> intron 1 tag SNP).[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909739">7</a>] We randomly sampled 82 cases and 14 controls within 3 of the 9 potential allele frequency cells (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 8</a>). The rs8176719 alleles are --/-- (deletion is the major allele), --/G, and G/G (G is the minor allele). The rs2519093 alleles are G/G (G is the major allele), A/G, and A/A (A is the minor allele). For each SNP, the genotypes are represented as 0, 1, or 2 copies of the minor allele. We represented the joint allelic configuration of the two SNPs with the number of copies of the rs8176719 minor allele given first as 0/0 (both with 0 copies of the minor allele), 0/1, 0/2 (0 copies of the rs8176719 minor allele), 1/0 (1 copy of the minor rs8176719 allele), 1/1, and 1/2, and 2/0 (2 copies of the rs8176719 SNP minor allele and 0 copies of the rs2519093 minor allele), 2/1, and 2/2. From <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 8</a>, one observes discrepancies between cases and controls at the 0/0, 1/1 and 2/2 combinations. We randomly sampled from these three combinations, taking one third of the case series. For each SNP, we had 28 cases with 0/0 copies of the minor allele, 27 cases with 1/1 copies of the minor allele; and 27 cases with the combination of 2/2 copies of the minor allele. We compared these 82 cases with 14 controls that do not have any of these combinations.</p><p id="P13">Capture of the target genomic regions was performed using the Agilent custom eArray. The capturing probes (baits) of 120 bases in length were designed based on the paired-end sequencing protocol with a tiling frequency of 3x. The standard repeat masked regions were avoided based on the definition in the UCSC genome database. The repeat regions are mostly in intronic regions of the genes.</p><p id="P14">Sequencing was performed using Illumina’s HiSeq 2000 sequencer. Twelve samples were multiplexed in each lane of the 8-lane flow cell, and a total of 96 samples (82 VTE cases and 14 controls) were sequenced. The read qualities were examined by FastQC (<a href="http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc</a>) which generates QC matrix from the FASTQ files including per-base sequence qualities, per-sequence quality scores, per-base nucleotide content, and sequence duplication levels. The FastQC tool also provides warnings for parameters failing to pass QC thresholds. The paired-end 100-base reads were aligned to human genome build 36 using Burrows Wheeler Aligner (BWA)[<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909731">16</a>] allowing 4% of editing distance within a seed length of 32. If the sequence duplication levels failed to pass the FastQC threshold, the duplicated reads were removed using the SAMtools’s rmdup method.[<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909732">17</a>] The BWA alignment was then cleaned up and improved using the Genome Analysis ToolKit (<strong>GATK</strong>)[<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909786">18</a>] by recalibration and local re-alignments. SNVs were called using SNVMix[<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909784">19</a>] with a cut-off probability score of 0.8 based on our preliminary testing using a HapMap CEPH subject sequenced by the 1000 genome project (data not shown), and INDELs were called by GATK with default parameters setting.</p><p id="P15">The read depths of each of the A, C, G, T bases at each variant position, as well as the average mapping quality score were provided by curating the BAM pile-up files using SAMtools.[<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909782">17</a>] If an identified SNV was a known variant from dbSNP or 1000 Genomes Project, the allele frequencies of CEU, YRI, and CHB/JPT populations from HapMap and 1000 Genome Project were provided. Both SNVs and INDELs were annotated by batch submission to the SeattleSeq server, and for SNVs additional annotations were acquired using a locally or cloud installed SIFT.[<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909751">20</a>] The SNVs or INDELs within a user defined distance (default: 5 bases) to exon-intron boundaries were flagged as potential splice variants and the corresponding transcript IDs are provided.</p><p id="P16" class="p p-last">Information was reported on genes hosting SNVs and INDELs, including (i) the KEGG pathway(s) (<a href="http://www.genome.jp/kegg" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.jp/kegg</a>) to which the gene belongs; and (ii) tissue expression specificity of the gene.</p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Statistical methods</h3><p id="P17" class="p p-first">GWS (Illumina 660Q; 557,112 SNPs) and CG (n=764 genes relevant to the anticoagulant, procoagulant, fibrinolytic and innate immunity pathways; n=12,551 SNPs) genotypes from the discovery population VTE cases (objectively-diagnosed; no cancer, catheter or antiphospholipid antibodies; n=1503) and controls (frequency-matched on case age, gender, race, MI/stroke status; n= 1459) were merged and imputed to ~2.5 million SNPs (i.e., <em>in silico</em> genotypes) with MACH using HapMap Phase II (60 CEU).[<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298479367">21</a>] All genetic coordinates in tables and figures refer to HapMap release 22 build 36. In regions where no candidate gene genotypes were available, only subjects with GWS data were used for the imputation.</p><p id="P18">The primary outcome was VTE status, a binary measure. The covariates were age at interview or blood sample collection, sex, stroke/MI status and USA state of residence (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). We tested for an association between each SNP and VTE using unconditional logistic regression, adjusting for age, sex, stroke/MI status and USA state of residence using PLINK v 1.07.[<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909745">22</a>] A similar analysis was performed for the replication study and the covariates were age at interview or blood sample collection, sex and USA state of residence (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>). Haplotype analyses were performed on the 25 chromosome 1q.24.2 SNPs most significantly associated with VTE, considering 25- and 10-SNP-sliding-windows, as implemented in PLINK v 1.07.[<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909785">22</a>] Novel <em>ABO</em> SNVs identified by resequencing were tested for an association with VTE using age-, sex-adjusted unconditional logistic regression and Fisher’s Exact Test. Population attributable risk (<strong>PAR</strong>) was estimated for F5 rs6025 (Factor V Leiden), F2 rs1799963 (prothrombin G20210A) and ABO blood type haplotype, as previously performed.[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909749">7</a>]</p><p id="P19" class="p p-last">VTE prevalence was estimated using the resources of the Rochester Epidemiology Project which has mapped Olmsted County, MN, USA residency for over 500,000 people from 1/1/1966 to 12/31/2010. All Olmsted County residents with objectively-diagnosed, incident VTE over the 40-year period, 1966-2005, have been followed forward in time until death or last follow-up. VTE point prevalence was recorded for 1/1/2006 using these data and adjusted to the USA white population data from the 2000 Census.</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Results</h2><div id="S17" class="sec sec-first"><h3 id="S17title">Discovery <em>in silico</em> GWAS results</h3><p id="P20" class="p p-first">Among the 1503 VTE cases and 1459 controls in the discovery population, the age and sex distributions were similar between cases and controls; however we found a significant difference between VTE cases and controls for prior stroke or myocardial infarction (<strong>stroke/MI</strong>); and USA state of residence (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). From the population stratification analysis, 98.64% of samples were classified as European and 1.36% as “other” ancestry (includes individuals of mixed ancestry; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 1</a>). Since age, sex, prior stroke or myocardial infarction have been shown or suggested to be risk factors for VTE,[<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909764">1</a>] and due to the significant difference between VTE cases and controls regarding USA state of residence, we adjusted for these four characteristics in the logistic regression analyses.</p><p id="P21" class="p p-last">After adjusting for age, sex, stroke/MI and USA state of residence, 39 SNPs exceeded genome-wide significance (<em>P</em>≤5 × 10<sup>-8</sup>; <a href="/pmc/articles/PMC3419811/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>); <em>F2</em> rs1799963 (prothrombin G20210A) was borderline significant (OR=2.46, <em>P</em>=1.7 × 10<sup>-6</sup>). The quantile-quantile plot of the expected vs. observed genome-wide −log(<em>P</em>-values) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>) shows the impact of chromosome regions 1q24.2 and 9q on VTE risk. <a href="/pmc/articles/PMC3419811/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> shows the association results for significant SNPs not in LD within a gene (<em>F5</em> rs6025, <em>BLZF1</em> rs7538157, <em>NME7</em> rs16861990, <em>SLC19A2</em> rs2038024, and <em>ABO</em> rs2519093, rs8176719 [ABO blood type non-O] and rs495828).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3419811/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106464767888" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms383857f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms383857f1.jpg" src="/pmc/articles/PMC3419811/bin/nihms383857f1.jpg"></div></a><div id="largeobj_idm140106464767888" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3419811/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3419811/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3419811/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p7">A. Manhattan plot of –log<sub>10</sub>(<em>P</em>-values) from the case-control association analysis on the merged/imputed VTE data assuming an additive genetic model. The horizontal line at 1.99 × 10<sup>-08</sup> represents the Bonferroni correction. Figure 1B. Location and linkage disequilibrium for SNPs on chromosome 1q24.2 surrounding <em>F5</em>. The black diamond corresponds to <em>F5</em> 6025 (Factor V Leiden) the most significant SNP. Figure 1C. Location and linkage disequilibrium for SNPs on chromosome 9q surrounding <em>ABO</em>. The black diamond corresponds to the most significant SNP, rs495828. For Figures 1B and 1C, the linkage disequilibrium measure, r<sup>2</sup>, between the most significant SNP and any other SNP in the region is symbol coded; the square, star, triangle, circle, and cross symbols correspond to r<sup>2</sup> &gt; 0.95, &gt; 0.8 and ≤ 0.95, &gt; 0.5 and ≤ 0.8, &gt; 0.2 and ≤ 0.5, and ≤ 0.2, respectively.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p11">Odds ratio and 95% confidence intervals, minor allele, minor allele frequency (MAF) and imputation quality for single nucleotide polymorphisms (SNPs) showing genome-wide significance, adjusted for age, sex, stroke/myocardial infarction status and USA state of residence</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106456585136" class="xtable"><table frame="hsides" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="center">Gene</th><th rowspan="1" colspan="1" align="center">Chromosome</th><th rowspan="1" colspan="1" align="center">Position<sup>†</sup></th><th rowspan="1" colspan="1" align="center">Minor Allele</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center">Imputed (r<sup>2</sup>)/Genotyped</th></tr></thead><tbody><tr><td colspan="9" rowspan="1" align="left"><span class="underline">SNPs in genes previously reported</span></td></tr><tr><td rowspan="1" colspan="1" align="left">rs6025</td><td rowspan="1" colspan="1" align="center"><em>F5</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">167785673</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.063</td><td rowspan="1" colspan="1" align="center">3.57 (2.76, 4.60)</td><td rowspan="1" colspan="1" align="center">1.68E-22</td><td rowspan="1" colspan="1" align="center">Yes (0.97)/Yes</td></tr><tr><td rowspan="1" colspan="1" align="left">rs8176719</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">135122729</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.419</td><td rowspan="1" colspan="1" align="center">1.47 (1.32, 1.64)</td><td rowspan="1" colspan="1" align="center">5.68E-12</td><td rowspan="1" colspan="1" align="center">No/Yes<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2519093</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">135131691</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.243</td><td rowspan="1" colspan="1" align="center">1.69 (1.48, 1.91)</td><td rowspan="1" colspan="1" align="center">8.08E-16</td><td rowspan="1" colspan="1" align="center">No/Yes<sup>‡</sup></td></tr><tr><td colspan="9" rowspan="1" align="left"><span class="underline">SNPs in genes not previously reported</span></td></tr><tr><td rowspan="1" colspan="1" align="left">rs7538157</td><td rowspan="1" colspan="1" align="center"><em>BLZF1</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">167618168</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.062</td><td rowspan="1" colspan="1" align="center">2.69 (2.09, 3.45)</td><td rowspan="1" colspan="1" align="center">1.04E-14</td><td rowspan="1" colspan="1" align="center">Yes (0.901)/No</td></tr><tr><td rowspan="1" colspan="1" align="left">rs16861990</td><td rowspan="1" colspan="1" align="center"><em>NME7</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">167401751</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.099</td><td rowspan="1" colspan="1" align="center">2.02 (1.66, 2.45)</td><td rowspan="1" colspan="1" align="center">1.69E-12</td><td rowspan="1" colspan="1" align="center">Yes (0.932)/Yes<sup>§</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2038024</td><td rowspan="1" colspan="1" align="center"><em>SLC19A2</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">167722606</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.177</td><td rowspan="1" colspan="1" align="center">1.53 (1.32, 1.78)</td><td rowspan="1" colspan="1" align="center">1.12E-08</td><td rowspan="1" colspan="1" align="center">Yes (0.893)/Yes<sup>§</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs495828</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">135144688</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.272</td><td rowspan="1" colspan="1" align="center">1.65 (1.46, 1.86)</td><td rowspan="1" colspan="1" align="center">2.96E-16</td><td rowspan="1" colspan="1" align="center">Yes (0.988)/Yes<sup>§</sup></td></tr></tbody></table></div><div id="largeobj_idm140106456585136" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3419811/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>†</sup>NCBI build 36 of the human genome</div><div id="TFN2"><sup>‡</sup>Genotype available in all subjects through the candidate gene data set.</div><div id="TFN3"><sup>§</sup>Genotype available in only 2570 subjects (87% of our combined samples) through the GWA data set</div></div></div></div><div id="S18" class="sec"><h3 id="S18title">Replication and combined results findings</h3><p id="P22" class="p p-first">In the separate replication population consisting of 1407 VTE cases and 1418 controls, no significant difference was found between age, sex, and USA state of residence (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>). Because <em>BLZF1</em> rs7538157 was in high LD with <em>F5, SLC19A2</em> and <em>ATP1B1</em>, the <em>BLZF1</em> SNP (rs7538157) was replaced by <em>ATP1B1</em> rs12061601 (<a href="/pmc/articles/PMC3419811/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>). After adjusting for age, sex and USA state of residence, <em>F5</em> rs6025 (<em>P</em>=1.4 × 10<sup>-12</sup>), <em>NME7</em> rs16861990 (<em>P</em>=4.9 × 10<sup>-9</sup>), and <em>ABO</em> rs2519093 (<em>P</em>=1.2 × 10<sup>-17</sup>), rs495828 (<em>P</em>=2.4 × 10<sup>-17</sup>) and rs8176719 (<em>P</em>=5.7 × 10<sup>-16</sup>), were significantly associated with VTE, while ATP1B1 rs12061601 (P=0.02), <em>F2</em> rs1799963 (<em>P</em>=0.03) and <em>SLC19A2</em> rs2038024 (<em>P</em>=0.09) were marginally associated (<a href="/pmc/articles/PMC3419811/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2a</span></a>). All replicated SNP ORs were in the same direction and of similar magnitude as that of the discovery population.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p15">a. Replication odds ratio and 95% confidence intervals, minor allele, minor allele frequency (MAF; overall and by case/control status) using 2,825 subjects (1407 cases and 1418 controls), adjusted for age, sex and USA state of residence</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106535986704" class="xtable"><table frame="hsides" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="center">Gene</th><th rowspan="1" colspan="1" align="center">Chromosome</th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">MAF - cases</th><th rowspan="1" colspan="1" align="center">MAF - controls</th><th rowspan="1" colspan="1" align="center">OR (95%CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs6025</td><td rowspan="1" colspan="1" align="center"><em>F5</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2543</td><td rowspan="1" colspan="1" align="center">0.060</td><td rowspan="1" colspan="1" align="center">0.086</td><td rowspan="1" colspan="1" align="center">0.035</td><td rowspan="1" colspan="1" align="center">2.56 (1.97, 3.32)</td><td rowspan="1" colspan="1" align="left">1.40E-12</td></tr><tr><td rowspan="1" colspan="1" align="left">rs12061601</td><td rowspan="1" colspan="1" align="center"><em>ATP1B1</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2521</td><td rowspan="1" colspan="1" align="center">0.139</td><td rowspan="1" colspan="1" align="center">0.151</td><td rowspan="1" colspan="1" align="center">0.126</td><td rowspan="1" colspan="1" align="center">1.21 (1.03, 1.43)</td><td rowspan="1" colspan="1" align="left">0.02</td></tr><tr><td rowspan="1" colspan="1" align="left">rs16861990</td><td rowspan="1" colspan="1" align="center"><em>NME7</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2559</td><td rowspan="1" colspan="1" align="center">0.097</td><td rowspan="1" colspan="1" align="center">0.122</td><td rowspan="1" colspan="1" align="center">0.071</td><td rowspan="1" colspan="1" align="center">1.79 (1.47, 2.18)</td><td rowspan="1" colspan="1" align="left">4.89E-09</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2038024</td><td rowspan="1" colspan="1" align="center"><em>SLC19A2</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2603</td><td rowspan="1" colspan="1" align="center">0.172</td><td rowspan="1" colspan="1" align="center">0.182</td><td rowspan="1" colspan="1" align="center">0.162</td><td rowspan="1" colspan="1" align="center">1.13 (0.98, 1.31)</td><td rowspan="1" colspan="1" align="left">0.09</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2519093</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">2599</td><td rowspan="1" colspan="1" align="center">0.235</td><td rowspan="1" colspan="1" align="center">0.289</td><td rowspan="1" colspan="1" align="center">0.182</td><td rowspan="1" colspan="1" align="center">1.80 (1.58, 2.07)</td><td rowspan="1" colspan="1" align="left">1.15E-17</td></tr><tr><td rowspan="1" colspan="1" align="left">rs495828</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">2627</td><td rowspan="1" colspan="1" align="center">0.261</td><td rowspan="1" colspan="1" align="center">0.317</td><td rowspan="1" colspan="1" align="center">0.205</td><td rowspan="1" colspan="1" align="center">1.73 (1.52, 1.96)</td><td rowspan="1" colspan="1" align="left">2.38E-17</td></tr><tr><td rowspan="1" colspan="1" align="left">rs8176719</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">2532</td><td rowspan="1" colspan="1" align="center">0.416</td><td rowspan="1" colspan="1" align="center">0.476</td><td rowspan="1" colspan="1" align="center">0.357</td><td rowspan="1" colspan="1" align="center">1.63 (1.45, 1.84)</td><td rowspan="1" colspan="1" align="left">5.62E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1799963</td><td rowspan="1" colspan="1" align="center"><em>F2</em></td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">2620</td><td rowspan="1" colspan="1" align="center">0.019</td><td rowspan="1" colspan="1" align="center">0.024</td><td rowspan="1" colspan="1" align="center">0.015</td><td rowspan="1" colspan="1" align="center">1.69 (1.06, 2.41)</td><td rowspan="1" colspan="1" align="left">0.03</td></tr></tbody></table></div><div id="largeobj_idm140106535986704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3419811/table/T2/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106460234096" class="xtable"><table frame="below" rules="all" class="rendered small default_table"><tbody><tr><td colspan="8" rowspan="1" align="left"><strong>b. Replication odds ratio and 95% confidence intervals using 2,825 subjects (1407 cases and 1418 controls), adjusted for age, sex, USA state of residence and <em>F5</em> rs6025 T allele (Factor V Leiden)</strong></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>SNP</strong></td><td rowspan="1" colspan="1" align="center"><strong>Gene</strong></td><td rowspan="1" colspan="1" align="center"><strong>N</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF-Cases</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF-Controls</strong></td><td rowspan="1" colspan="1" align="center"><strong>OR (95%CI)</strong></td><td rowspan="1" colspan="1" align="left"><strong><em>P</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left">rs12061601</td><td rowspan="1" colspan="1" align="center"><em>ATP1B1</em></td><td rowspan="1" colspan="1" align="center">2477</td><td rowspan="1" colspan="1" align="center">0.137</td><td rowspan="1" colspan="1" align="center">0.150</td><td rowspan="1" colspan="1" align="center">0.125</td><td rowspan="1" colspan="1" align="center">0.92 (0.76, 1.10)</td><td rowspan="1" colspan="1" align="left">0.35</td></tr><tr><td rowspan="1" colspan="1" align="left">rs16861990</td><td rowspan="1" colspan="1" align="center"><em>NME7</em></td><td rowspan="1" colspan="1" align="center">2505</td><td rowspan="1" colspan="1" align="center">0.097</td><td rowspan="1" colspan="1" align="center">0.123</td><td rowspan="1" colspan="1" align="center">0.071</td><td rowspan="1" colspan="1" align="center">1.17 (0.89, 1.54)</td><td rowspan="1" colspan="1" align="left">0.25</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2038024</td><td rowspan="1" colspan="1" align="center"><em>SLC19A2</em></td><td rowspan="1" colspan="1" align="center">2474</td><td rowspan="1" colspan="1" align="center">0.170</td><td rowspan="1" colspan="1" align="center">0.178</td><td rowspan="1" colspan="1" align="center">0.163</td><td rowspan="1" colspan="1" align="center">0.77 (0.65, 0.92)</td><td rowspan="1" colspan="1" align="left">0.004</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2519093</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">2474</td><td rowspan="1" colspan="1" align="center">0.235</td><td rowspan="1" colspan="1" align="center">0.289</td><td rowspan="1" colspan="1" align="center">0.181</td><td rowspan="1" colspan="1" align="center">1.85 (1.61, 2.13)</td><td rowspan="1" colspan="1" align="left">1.37E-17</td></tr><tr><td rowspan="1" colspan="1" align="left">rs495828</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">2501</td><td rowspan="1" colspan="1" align="center">0.261</td><td rowspan="1" colspan="1" align="center">0.317</td><td rowspan="1" colspan="1" align="center">0.205</td><td rowspan="1" colspan="1" align="center">1.76 (1.54, 2.01)</td><td rowspan="1" colspan="1" align="left">3.60E-17</td></tr><tr><td rowspan="1" colspan="1" align="left">rs8176719</td><td rowspan="1" colspan="1" align="center"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">2477</td><td rowspan="1" colspan="1" align="center">0.416</td><td rowspan="1" colspan="1" align="center">0.474</td><td rowspan="1" colspan="1" align="center">0.359</td><td rowspan="1" colspan="1" align="center">1.58 (1.40, 1.78)</td><td rowspan="1" colspan="1" align="left">9.75E-14</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1799963</td><td rowspan="1" colspan="1" align="center"><em>F2</em></td><td rowspan="1" colspan="1" align="center">2485</td><td rowspan="1" colspan="1" align="center">0.019</td><td rowspan="1" colspan="1" align="center">0.023</td><td rowspan="1" colspan="1" align="center">0.015</td><td rowspan="1" colspan="1" align="center">1.71 (1.12, 2.63)</td><td rowspan="1" colspan="1" align="left">0.014</td></tr></tbody></table></div><div id="largeobj_idm140106460234096" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3419811/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P23">These SNPs cluster in two genomic regions located on chromosome 1q24.2 (<a href="/pmc/articles/PMC3419811/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>) and on chromosome 9q (<a href="/pmc/articles/PMC3419811/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1C</span></a>). Chromosome 1q24.2 and 9q LD patterns observed in controls are shown in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figures 3 and 4</a>, and the ORs and respective 95% confidence intervals for an association of SNPs in these two genomics regions with VTE are depicted in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figures 5 and 6</a>. Since <em>ATP1B1, NME7, BLZF1</em> and <em>SLC19A2</em> are in close proximity to <em>F5</em>, we repeated the association analysis including the <em>F5</em> rs6025 T (minor) allele (Factor V Leiden) as a covariate and found that only the <em>ABO</em> SNPs remained significantly associated with VTE at the genome-wide level. Similarly, in the replication study only the <em>ABO</em> SNPs and <em>SLC19A2</em> rs2038024 remained significantly associated with VTE after adjusting for <em>F5</em> rs6025 T. Although <em>SLC19A2</em> rs2038024 was marginally associated with VTE, the OR was in the opposite direction from that of the discovery population, suggesting a spurious association (<a href="/pmc/articles/PMC3419811/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2b</span></a>).</p><p id="P24" class="p p-last">Receiver Operator Characteristic areas under the curve (<strong>AUC</strong>), calculated individually and jointly using all significant SNPs, were not different for covariates (age, sex, stroke/MI and USA state of residence) plus <em>F5</em> rs6025 T (AUC=0.653) compared to covariates plus <em>F5</em> rs6025 T, <em>ATP1B1</em> rs12061601, <em>NME7</em> rs16861990 and <em>SLC19A2</em> rs2038024 (AUC=0.653), while addition of <em>ABO</em> SNPs did increase the AUC (AUC=0.675; <a href="/pmc/articles/PMC3419811/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3419811/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106499679776" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms383857f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms383857f2.jpg" src="/pmc/articles/PMC3419811/bin/nihms383857f2.jpg"></div></a><div id="largeobj_idm140106499679776" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3419811/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3419811/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3419811/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p8">Receiver Operator Characteristic (<strong>ROC</strong>) Curve Analyses of Chromosome 1q24.2 and <em>ABO</em> SNPs</p></div></div></div></div><div id="S19" class="sec"><h3 id="S19title">Haplotype analysis results</h3><p id="P25" class="p p-first-last">Haplotype analysis of the 25 chromosome 1q.24.2 SNPs (including <em>ATP1B1, NME7, BLZF1, C1orf114, SLC19A2, F5</em> and <em>SELP</em>; <a href="/pmc/articles/PMC3419811/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a>) most significantly associated with VTE showed that only the two haplotypes containing <em>F5</em> rs6025 T were significantly associated with VTE (<a href="/pmc/articles/PMC3419811/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Together, these data suggest that this chromosome 1q24.2 region is inherited as a haplotype block of ~0.52mb. Among all <em>F5</em> rs6025 T carriers (N=369; 280 cases and 89 controls; 186 males and 183 females), 235 (63.7%; 177 cases and 58 controls) carriers inherited at least one of these two 1q24.2 haplotypes. The haplotype containing the <em>F5</em> rs6025 T and the <em>SELP</em> rs3917862 G (minor) alleles slightly increased the association of the haplotype with VTE (OR=3.59) compared to the haplotype containing the <em>F5</em> rs6025 T and the <em>SELP</em> rs3917862 A alleles (OR=3.16, <a href="/pmc/articles/PMC3419811/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). However, a 10-SNP-sliding-window with nine SNPs in <em>F5</em> (including the <em>F5</em> rs6025 T allele) and one SNP in <em>SELP</em> (SNP rs3917862 G [minor] allele) considerably increased the association of this haplotype with VTE (OR=4.55, <em>P</em>=2.07 × 10<sup>-13</sup>) compared to the association for the haplotype containing the <em>F5</em> rs6025 T and <em>SELP</em> rs3917862 A alleles (OR=3.09, <em>P</em>=1.83 × 10<sup>-11</sup>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 5</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p16">Haplotype analysis of the 25 most significant chromosome 1q24.2 SNPs adjusted for age, sex, stroke/myocardial infarction and USA state of residence.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106451853104" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Haplotype<sup>†</sup><sup>‡</sup><sup>§</sup></th><th rowspan="1" colspan="1" align="left">F</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left">STAT</th><th rowspan="1" colspan="1" align="left"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">TATAGGTAGAGCTATGCAGCATG<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.136</td><td rowspan="1" colspan="1" align="left">0.80</td><td rowspan="1" colspan="1" align="left">7.65</td><td rowspan="1" colspan="1" align="left">0.006</td></tr><tr><td rowspan="1" colspan="1" align="left">TATAGGTAGAGCTATGCAGCAGG<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.447</td><td rowspan="1" colspan="1" align="left">0.92</td><td rowspan="1" colspan="1" align="left">2.02</td><td rowspan="1" colspan="1" align="left">0.16</td></tr><tr><td rowspan="1" colspan="1" align="left">TATAAGTAGAGCTATGCAGCATG<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.0795</td><td rowspan="1" colspan="1" align="left">0.82</td><td rowspan="1" colspan="1" align="left">3.68</td><td rowspan="1" colspan="1" align="left">0.06</td></tr><tr><td rowspan="1" colspan="1" align="left">TATAAGTAGAGCTATGCAGCAGG<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.0273</td><td rowspan="1" colspan="1" align="left">0.83</td><td rowspan="1" colspan="1" align="left">1.04</td><td rowspan="1" colspan="1" align="left">0.31</td></tr><tr><td rowspan="1" colspan="1" align="left">CCCCAACCAGAACTACGGATCGA<strong>T</strong><span class="underline">G</span></td><td rowspan="1" colspan="1" align="left">0.0185</td><td rowspan="1" colspan="1" align="left">3.59</td><td rowspan="1" colspan="1" align="left">28.9</td><td rowspan="1" colspan="1" align="left">7.46E-08</td></tr><tr><td rowspan="1" colspan="1" align="left">TATAAGTCAAAACTACGGATCGA<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.0127</td><td rowspan="1" colspan="1" align="left">1.04</td><td rowspan="1" colspan="1" align="left">0.0313</td><td rowspan="1" colspan="1" align="left">0.86</td></tr><tr><td rowspan="1" colspan="1" align="left">CCCCAACCAGAACTACGGATCGA<strong>T</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.0227</td><td rowspan="1" colspan="1" align="left">3.16</td><td rowspan="1" colspan="1" align="left">29.2</td><td rowspan="1" colspan="1" align="left">6.55E-08</td></tr><tr><td rowspan="1" colspan="1" align="left">CATAGGTAGAGCTATGCAGCAGG<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.029</td><td rowspan="1" colspan="1" align="left">0.97</td><td rowspan="1" colspan="1" align="left">0.0266</td><td rowspan="1" colspan="1" align="left">0.87</td></tr><tr><td rowspan="1" colspan="1" align="left">TATAGGTAGAGCTATGCAGCAGG<strong>C</strong><span class="underline">G</span></td><td rowspan="1" colspan="1" align="left">0.0211</td><td rowspan="1" colspan="1" align="left">1.07</td><td rowspan="1" colspan="1" align="left">0.109</td><td rowspan="1" colspan="1" align="left">0.74</td></tr><tr><td rowspan="1" colspan="1" align="left">TATAAGTCAAGCTATGCAGCATG<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.0196</td><td rowspan="1" colspan="1" align="left">0.89</td><td rowspan="1" colspan="1" align="left">0.385</td><td rowspan="1" colspan="1" align="left">0.54</td></tr><tr><td rowspan="1" colspan="1" align="left">TCCAAACAGAGCTATGCAGCAGG<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.0129</td><td rowspan="1" colspan="1" align="left">0.73</td><td rowspan="1" colspan="1" align="left">1.51</td><td rowspan="1" colspan="1" align="left">0.22</td></tr><tr><td rowspan="1" colspan="1" align="left">TATAAGTCGAGCTATGCAGCAGG<strong>C</strong><span class="underline">A</span></td><td rowspan="1" colspan="1" align="left">0.0162</td><td rowspan="1" colspan="1" align="left">1.27</td><td rowspan="1" colspan="1" align="left">1.15</td><td rowspan="1" colspan="1" align="left">0.28</td></tr></tbody></table></div><div id="largeobj_idm140106451853104" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3419811/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><sup>†</sup>The first SNP of the haplotype is <em>ATP1B1</em> rs12061601, and the last SNP of the haplotype is <em>SELP</em> rs3917862</div><div id="TFN5"><sup>‡</sup>Allele in bold is <em>F5</em> rs6025 (T is the minor allele [Factor V Leiden])</div><div id="TFN6"><sup>§</sup>Underlined allele is <em>SELP</em> rs3917862 (G is the minor allele)</div></div></div></div><div id="S20" class="sec sec-last"><h3 id="S20title">Resequencing results</h3><p id="P26" class="p p-first">To further evaluate <em>ABO</em> sequence variation for an association with VTE, we performed <em>ABO</em> resequencing, including 10 Kb of the flanking regions, using Illumina HiSeq 2000. Excluding the intronic repeat regions, 98% of the targeted area (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 7</a>) was sequenced with &gt; 20X coverage in 96 samples (82 VTE cases and 14 controls). On average, ~35-40 SNVs and ~5-10 insertion-deletions (<strong>INDELs</strong>) were detected in each sample. Fifteen novel SNVs in <em>ABO</em> intron 6 and the <em>ABO</em> 3’ UTR were associated with VTE (all <em>P</em>&lt;10<sup>-04</sup>) and belonged to three distinctive LD blocks; none were in LD with <em>ABO</em> rs8176719 (blood type O allele) or rs2519093 although <em>ABO</em> rs8176746 (blood type A allele) and rs8176747 (blood type B allele) were in moderate LD (r<sup>2</sup>=0.56-0.65) with novel SNPs in block 3 (<a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). Although <em>ABO</em> rs8176746 and rs8176747 did not reach genome wide significance in the primary analysis, a secondary analysis (adjusted for age, sex, stroke/MI and USA state of residence) of the derivation population that considered ABO blood type (either haplotype/genotype or phenotype; VTE cases, n=1459; controls, n=1422) found strong associations of ABO blood types A (OR=1.64; 95%CI: 1.41, 1.91; <em>P</em>=7.83 × 10<sup>-11</sup>) and O (OR=0.52; 95%CI: 0.44, 0.60; P=2.40 × 10<sup>-16</sup>) with VTE, but no association with blood types B (OR=1.15; 95%CI: 0.91, 1.44; <em>P</em>=0.25) or AB (OR=1.22; 95%CI: 0.87, 1.70; <em>P</em>=0.25; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 7</a>). Finally, a significant, protective, frame-shifting single base (G) deletion of one of three guanines at <em>ABO</em> chr9:135120878-80 was found within exon 7. This deletion likely corresponds to <em>ABO</em> rs8176750 (ABO blood-type A2).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106463692048" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms383857f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms383857f3.jpg" src="/pmc/articles/PMC3419811/bin/nihms383857f3.jpg"></div></a><div id="largeobj_idm140106463692048" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC3419811/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3419811/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p9">Linkage disequilibrium (<strong>LD</strong>) between novel <em>ABO</em> SNVs identified by resequencing, and <em>ABO</em> rs8176474 (blood type A allele), rs8176746 (blood type B allele), rs8176719 (blood type O allele) and rs2519093 (<em>ABO</em> intron 1), among 82 VTE cases and 14 controls selected based on <em>ABO</em> rs8176719 and rs2519093 genotype (see Methods). The numbers inside each square and the square shading indicate the degree of LD between SNPs and/or SNVs (i.e., black squares with no number indicate complete LD [r<sup>2</sup>=1.0]; intensity of the gray shading indicates intermediate values between complete LD [r<sup>2</sup>=1.0] and no LD [r<sup>2</sup>=0.0], and white squares with 0 number indicate no LD).</p></div></div></div><p id="P27" class="p p-last">The individual, joint and group population attributable risk (<strong>PAR</strong>) were calculated for the risk genotypes for Factor V Leiden (<em>F5</em> rs6025), prothrombin G20210A (<em>F2</em> rs1799963), and ABO blood type haplotype or genotype (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 8</a>). The highest PAR value was from the ABO blood type non-O, followed by ABO blood type A, Factor V Leiden, and prothrombin G20210A. The PAR values were very similar between the joint and group estimation methods, with highest joint or group PAR (PAR=0.47) for ABO blood type non-O, <em>F5</em> rs6025 and <em>F2</em> rs1799963).</p></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Discussion</h2><p id="P28" class="p p-first">Our most notable finding was the strong association of VTE with three novel and distinctive <em>ABO</em> LD blocks that were independent of ABO type O blood type. <em>ABO</em> encodes for glycosyltransferases responsible for post translational glycosylation of procoagulant FVIII and von Willebrand Factor (<strong>VWF</strong>).[<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909781">23</a>] ABO O blood group phenotype and genotype (i.e., <em>ABO</em> rs8176719) is associated with a decreased risk for VTE,[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909789">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909729">8</a>, <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909765">24</a>] presumably due to decreased plasma levels of FVIII and VWF.[<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909773">25</a>, <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909761">26</a>] <em>ABO</em>-mediated glycosylation protects VWF from proteolysis by ADAMTS13[<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909777">27</a>] such that individuals with type O blood group (homozygous exon 6 <em>ABO</em> rs8176719 deletion) and hypoglycosylated VWF have ~25% lower plasma FVIII and VWF levels.[<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909759">24</a>, <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909768">28</a>]</p><p id="P29">We previously identified an independent association of an <em>ABO</em> intron 1 SNP (rs2519093) with VTE (OR=1.68, <em>P</em>=8.08×10<sup>-16</sup>).[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909728">7</a>] This intronic region does not affect RNA splicing or harbor any suppressor RNA elements, nor is it close to any other <em>ABO</em> variants with known function, including variants in <em>ABO</em> exons 6 and 7 that have been variably associated with VTE risk.[<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909744">29</a>-<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909757">33</a>] Of the 15 novel <em>ABO</em> SNVs found on resequencing, none were in LD with this <em>ABO</em> intron 1 SNP (<a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). Consequently, the findings of this study expand on our previous observation and suggest that this <em>ABO</em> intron 1 region along with an LD block in <em>ABO</em> intron 6 and two LD blocks in the <em>ABO</em> 3’ UTR play important roles in <em>ABO</em> expression and associated VTE risk.</p><p id="P30">In a secondary analysis, the ABO blood type A haplotype was strongly associated with VTE (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 7</a>) as previously described.[<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909754">31</a>-<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909758">34</a>] In the limited and select sample (n=96) of our discovery population who underwent <em>ABO</em> resequencing, the ABO A allele was in moderate LD (r<sup>2</sup>=0.56-0.65) with SNVs in <em>ABO</em> LD block 3 (<a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>) which potentially could account for the association of these novel block 3 SNVs with VTE. The association of VTE with novel SNVs in <em>ABO</em> LD block 2 was independent of blood type A (rs8176747) and O (rs8176719) alleles, and <em>ABO</em> rs2519093 (<a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). <em>ABO</em> LD block 2 may be important for gene regulation due to a potential open chromatin region as reflected by enrichment for multiple epi-genetic markers in the region,(ENCODE: <a href="http://www.genome.gov/10005107" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/10005107</a>) a transcription factor binding site (ChIP-seq, ENCODE: <a href="http://www.genome.gov/10005107" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/10005107</a>), and evolutionary conservation (UCSC phatCons[<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909741">35</a>] and phylo[<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909774">36</a>] algorithms; <a href="/pmc/articles/PMC3419811/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). In addition, strong evidence from dbEST supports the middle LD block as also lying within a novel transcript, probably an extension of the 3’ of <em>ABO</em> (dbEST: <a href="http://www.ncbi.nlm.nih.gov/dbEST/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/dbEST/</a>; <a href="/pmc/articles/PMC3419811/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). <em>ABO</em> SNV_135116452 was not part of any <em>ABO</em> LD block but it was in high LD with SNVs from LD block 3, and in moderate (r<sup>2</sup>=0.54) LD with the ABO blood type A allele (rs8176747; <a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). Thus, the association of SNV_135116452 with VTE may also be due to blood type A. Lastly, the SNVs in <em>ABO</em> LD block 1 are in low LD with all the other SNVs and SNPs (<a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). The associations of SNVs in LD block 1 with VTE are independent of blood type A (r<sup>2</sup>~0.27-0.44) and O (r<sup>2</sup>~0-0.02) alleles, and <em>ABO</em> rs2519093 (r<sup>2</sup>~0-0.04; <a href="/pmc/articles/PMC3419811/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3419811/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106500642608" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms383857f4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms383857f4.jpg" src="/pmc/articles/PMC3419811/bin/nihms383857f4.jpg"></div></a><div id="largeobj_idm140106500642608" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3419811/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a target="object" href="/pmc/articles/PMC3419811/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3419811/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p10">Potential functional roles of <em>ABO</em> SNVs identified on resequencing, by LD block or deletion.</p></div></div></div><p id="P31">Our <em>ABO</em> resequencing data also support the previously reported association of VTE with ABO blood type A2 (<em>ABO</em> rs8176750).[<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909756">8</a>, <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909788">31</a>] Although we did not directly genotype <em>ABO</em> rs8176750 in either our CG or GWAS studies, one directly-genotyped <em>ABO</em> SNP (rs8176704) in strong LD with rs8176750 (r<sup>2</sup>≥0.8) was associated with a significantly reduced risk of VTE (OR=0.78; 95%CI: 0.63, 0.97, <em>P</em>=0.02) that was of similar magnitude to that reported for ABO blood type A2.[<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909748">8</a>, <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909736">31</a>] The <em>ABO</em> rs8176750 deletion results in a frame-shift that abrogates the wild-type stop codon within exon 7 and likely leads to an elongated transcript (<a href="/pmc/articles/PMC3419811/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>) and an absent or dysfunctional protein similar to blood type O.</p><p id="P32">Our second notable finding was the discovery that Factor V Leiden is inherited as a haplotype block that spans seven genes on chromosome 1q24.2, including <em>ATP1B1, NME7, BLZF1, C1orf114, SLC19A2, F5</em> and <em>SELP. F5</em> encodes for plasma procoagulant Factor(<strong>F</strong>) V; FV accelerates activation of prothrombin to thrombin by catalyzing assembly of the prothrombinase complex. FV inactivation is key for down-regulating procoagulant activity. Activated Protein C (<strong>APC</strong>), a serine protease, inactivates FV via cleavage at arginine[R]306, R506 and R679; initial APC cleavage at R506 is required for rapid FV inactivation by APC cleavage at R306 and R679.[<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909775">37</a>] <em>F5</em> rs6025 encodes for substitution of glutamine for R506.[<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909783">38</a>, <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909726">39</a>] Factor V Leiden carriers have normal FV procoagulant activity but impaired FV down-regulation due to “resistance” to cleavage/inactivation by APC.[<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909776">40</a>, <a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909769">41</a>] Factor V Leiden is a founder mutation occurring about 30,000 years ago,[<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909760">42</a>] and is associated with ~3.5-fold increased risk for VTE (lifetime VTE penetrance ~5%).[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909762">7</a>, <a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909763">38</a>, <a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909746">43</a>]</p><p id="P33">Our third finding was that the tag SNP <em>SELP</em> rs3917862 compounds the risk of VTE among Factor V Leiden carriers. <em>SELP</em> encodes for P-Selectin, a membrane glycoprotein with pro-inflammatory activity that mediates the interaction between activated platelets, leukocytes, immune cells and endothelial cells.[<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909771">44</a>, <a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909750">45</a>] Cell-surface P-selectin expression and soluble P-selectin concentration are partly controlled by <em>SELP</em> genetic variation.[<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909738">46</a>, <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909778">47</a>] Increased soluble P-selectin plasma levels are associated with incident and recurrent VTE.[<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909734">48</a>, <a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909730">49</a>] We previously showed that <em>SELP</em> rs3917862 was associated with a 1.6-fold increased risk of VTE at a false discovery rate q = 0.006.[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909742">7</a>] <em>SELP</em> is located immediately upstream from <em>F5</em> (<a href="/pmc/articles/PMC3419811/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>), and associations between <em>SELP</em> haplotypes and VTE have been attributed to high LD with <em>F5</em> rs6025 T.[<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909747">50</a>] In our previous report, while <em>SELP</em> rs3917862 was not in high LD with <em>F5</em> rs6025 T (r<sup>2</sup>=0.12, D’=0.37), the association of rs3917862 with VTE was no longer significant (<em>P</em>=0.3) after controlling for <em>F5</em> rs6025 T.[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909787">7</a>] Our new analyses indicate that the haplotype containing the <em>F5</em> rs6025 T and the <em>SELP</em> rs3917862 G (minor) alleles increases the association of the haplotype with VTE (OR=3.59) compared to the haplotype containing the <em>F5</em> rs6025 T and the <em>SELP</em> rs3917862 A alleles (OR=3.16, <a href="/pmc/articles/PMC3419811/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Furthermore, the association of the haplotype with VTE increases when considering different numbers of SNPs in the sliding window approach (OR=4.55 for 10-SNP-sliding-window with nine SNPs in <em>F5</em> and one in <em>SELP</em>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 5</a>). Based on these statistical analyses, we conclude that the <em>SELP</em> rs3917862 T allele compounds the risk of VTE associated with <em>F5</em> rs6025 T.</p><p id="P34">Although we found associations of VTE with <em>ATP1B1, NME7, BLZF1</em>, and <em>SLC19A2</em>, these associations disappeared in both the discovery and replication populations after controlling for <em>F5</em> rs6025 T (Factor V Leiden). <em>ATP1B1</em> encodes for a Na+/K+ ATPase important in endothelial cell and platelet function, <em>NME7</em> encodes for a nucleoside-diphosphate kinase important in erythropoiesis and CD4 T-cell activation, <em>BLZF1</em> (<em>JEM-1</em>) encodes for basic leucine zipper nuclear factor 1 which is important for the response of acute promyelocytic leukemia to retinoic acid, and <em>SLC19A2</em> encodes for a thiamine transporter protein (THTR-1) implicated in thiamine-responsive megaloblastic anemia. While the products of these genes are potentially biologically plausible in the causal pathway to VTE, none have been previously associated with VTE. Moreover, ROC curve analyses showed that SNPs within these four genes added no additional information over that provided by patient characteristics and <em>F5</em> rs6025 T alone. Thus, we believe it is unlikely that these genes are VTE disease-susceptibility genes.</p><p id="P35" class="p p-last">Our sample size provided 80% power to detect odds ratios of 2.0 and 1.51 for minor allele frequencies (<strong>MAF</strong>)=0.05 and 0.5, respectively (α=5 × 10<sup>-8</sup>) based on a 1% prevalence of VTE among USA whites for the year 2000.[<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_274909743">51</a>] While single studies or individual-level data meta-analyses with a larger sample size and/or more dense genome-wide scan genotype data may identify additional SNPs that associate with VTE, we believe that aside from <em>F5</em> rs6025, <em>ABO</em> rs8176719 and rs2519093, and <em>F2</em> rs1799963, additional common (i.e., MAF&gt;0.05) and high VTE-risk (i.e., OR&gt;2.0) SNPs among persons of non-Hispanic European ancestry are unlikely.</p></div><div id="S22" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD2"><h4>Supplemental</h4><div class="sup-box half_rhythm" id="d35e1392"><a href="/pmc/articles/PMC3419811/bin/NIHMS383857-supplement-Supplemental.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.3M, doc)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Acknowledgments</h2><div class="sec"><p id="__p1">We thank C.E. Regnier, J.L. Alkhamis, L.M. Heimer, R.M. Weatherly, R.A. Mueller, D. Tines, A, Xue, R.A. Miller, J.J. Larson, E.N. Jeavons and A.F. Beauseigneur for their excellent technical assistance. All were compensated as part of their regular duties.</p><p id="__p2">We sincerely thank all participants in this study. This project was part of the Gene, Environment Association Studies Consortium (<strong>GENEVA</strong>) funded by the USA National Human Genome Research Institute (<strong>NHGRI</strong>) to enhance communication and collaboration among investigators conducting genome-wide studies for a variety of complex diseases. Our group benefited greatly from the work and efforts of the entire consortium, especially the work by the Coordinating Center (directed by B. Weir and C. Laurie of the University of Washington) in data cleaning (D.R. Crosslin) and preparation of these cases and controls for submission to the Database for Genotypes and Phenotypes (dbGAP; S. Bennett). We also acknowledge the leadership of T. Manolio of NHGRI. Genotyping services were provided by the Center for Inherited Disease Research (<strong>CIDR</strong>), funded through a federal contract from the USA National Institutes of Health (<strong>NIH</strong>) to Johns Hopkins University.</p><p id="__p3"><strong>Funding</strong></p><p id="__p4">This work was supported by the National Institutes of Health, National Heart, Lung and Blood Institute [grant numbers HL66216, HL83141], National Human Genome Research Institute [grant number HG04735], and National Cancer Institute [grant number CA92153], U.S. Public Health Service; the Centers for Disease Control and Prevention [grant number DD000235]; the Mayo Clinic Center for Individualized Medicine; and by Mayo Foundation.</p></div></div><div id="idm140106457984784" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140106457984784title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"> </p><p id="__p5" class="p p-first"><strong>Conflict of Interest Statement</strong></p><p id="__p6" class="p p-last">The authors have no conflict of interest to declare.</p><p></p></div></div><div id="idm140106464174192" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140106464174192title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Heit JA. The epidemiology of venous thromboembolism in the community. <span><span class="ref-journal">Arterioscler Thromb Vasc Biol. </span>2008;<span class="ref-vol">28</span>:370–2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2873781/">PMC free article</a>]</span> [<a href="/pubmed/18296591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=The+epidemiology+of+venous+thromboembolism+in+the+community&amp;author=JA+Heit&amp;volume=28&amp;publication_year=2008&amp;pages=370-2&amp;pmid=18296591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Heit JA, Silverstein MD, Mohr DN, Petterson TM, O’Fallon WM, Melton LJI. Predictors of survival after deep vein thrombosis and pulmonary embolism: a population-based, cohort study. <span><span class="ref-journal">Arch Intern Med. </span>1999;<span class="ref-vol">159</span>:445–53.</span> [<a href="/pubmed/10074952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Intern+Med&amp;title=Predictors+of+survival+after+deep+vein+thrombosis+and+pulmonary+embolism:+a+population-based,+cohort+study&amp;author=JA+Heit&amp;author=MD+Silverstein&amp;author=DN+Mohr&amp;author=TM+Petterson&amp;author=WM+O’Fallon&amp;volume=159&amp;publication_year=1999&amp;pages=445-53&amp;pmid=10074952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Souto JC, Almasy L, Borrell M, Blanco-Vaca F, Mateo J, Soria JM, Coll I, Felices R, Stone W, Fontcuberta J, Blangero J. Genetic susceptibility to thrombosis and its relationship to physiological risk factors: the GAIT study. Genetic Analysis of Idiopathic Thrombophilia. <span><span class="ref-journal">Am J Hum Genet. </span>2000;<span class="ref-vol">67</span>:1452–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1287922/">PMC free article</a>]</span> [<a href="/pubmed/11038326" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+susceptibility+to+thrombosis+and+its+relationship+to+physiological+risk+factors:+the+GAIT+study.+Genetic+Analysis+of+Idiopathic+Thrombophilia&amp;author=JC+Souto&amp;author=L+Almasy&amp;author=M+Borrell&amp;author=F+Blanco-Vaca&amp;author=J+Mateo&amp;volume=67&amp;publication_year=2000&amp;pages=1452-9&amp;pmid=11038326&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Heit JA, Phelps MA, Ward SA, Slusser JP, Petterson TM, De Andrade M. Familial segregation of venous thromboembolism. <span><span class="ref-journal">J Thromb Haemost. </span>2004;<span class="ref-vol">2</span>:731–6.</span> [<a href="/pubmed/15099278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=Familial+segregation+of+venous+thromboembolism&amp;author=JA+Heit&amp;author=MA+Phelps&amp;author=SA+Ward&amp;author=JP+Slusser&amp;author=TM+Petterson&amp;volume=2&amp;publication_year=2004&amp;pages=731-6&amp;pmid=15099278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Larsen TB, Sorensen HT, Skytthe A, Johnsen SP, Vaupel JW, Christensen K. Major genetic susceptibility for venous thromboembolism in men: a study of Danish twins. <span><span class="ref-journal">Epidemiology. </span>2003;<span class="ref-vol">14</span>:328–32.</span> [<a href="/pubmed/12859034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epidemiology&amp;title=Major+genetic+susceptibility+for+venous+thromboembolism+in+men:+a+study+of+Danish+twins&amp;author=TB+Larsen&amp;author=HT+Sorensen&amp;author=A+Skytthe&amp;author=SP+Johnsen&amp;author=JW+Vaupel&amp;volume=14&amp;publication_year=2003&amp;pages=328-32&amp;pmid=12859034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Heit JA.  Thrombophilia: Clinical and laboratory assessment and managment. In: Kitchens C, Alving B, Kessler C, editors. <span class="ref-journal">Consultative Hemostasis and Thrombosis.</span> W. B. Saunders; 2007. pp. 213–44. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Consultative+Hemostasis+and+Thrombosis&amp;author=JA+Heit&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Heit JA, Cunningham JM, Petterson TM, Armasu SM, Rider DN, DE Andrade M. Genetic variation within the anticoagulant, procoagulant, fibrinolytic and innate immunity pathways as risk factors for venous thromboembolism. <span><span class="ref-journal">Journal of Thrombosis and Haemostasis. </span>2011;<span class="ref-vol">9</span>:1133–42.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3110632/">PMC free article</a>]</span> [<a href="/pubmed/21463476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Thrombosis+and+Haemostasis&amp;title=Genetic+variation+within+the+anticoagulant,+procoagulant,+fibrinolytic+and+innate+immunity+pathways+as+risk+factors+for+venous+thromboembolism&amp;author=JA+Heit&amp;author=JM+Cunningham&amp;author=TM+Petterson&amp;author=SM+Armasu&amp;author=DN+Rider&amp;volume=9&amp;publication_year=2011&amp;pages=1133-42&amp;pmid=21463476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Tregouet DA, Heath S, Saut N, Biron-Andreani C, Schved JF, Pernod G, Galan P, Drouet L, Zelenika D, Juhan-Vague I, Alessi MC, Tiret L, Lathrop M, Emmerich J, Morange PE. Common susceptibility alleles are unlikely to contribute as strongly as the FV and ABO loci to VTE risk: results from a GWAS approach. <span><span class="ref-journal">Blood. </span>2009;<span class="ref-vol">113</span>:5298–303.</span> [<a href="/pubmed/19278955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Common+susceptibility+alleles+are+unlikely+to+contribute+as+strongly+as+the+FV+and+ABO+loci+to+VTE+risk:+results+from+a+GWAS+approach&amp;author=DA+Tregouet&amp;author=S+Heath&amp;author=N+Saut&amp;author=C+Biron-Andreani&amp;author=JF+Schved&amp;volume=113&amp;publication_year=2009&amp;pages=5298-303&amp;pmid=19278955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Germain M, Saut N, Greliche N, Dina C, Lambert JC, Perret C, Cohen W, Oudot-Mellakh T, Antoni G, Alessi MC, Zelenika D, Cambien F, Tiret L, Bertrand M, Dupuy AM, Letenneur L, Lathrop M, Emmerich J, Amouyel P, Tregouet DA, Morange PE. Genetics of venous thrombosis: insights from a new genome wide association study. <span><span class="ref-journal">Plos ONE. </span>2011;<span class="ref-vol">6</span>:e25581. doi: 10.1371/journal.pone.0025581.</span>PONE-D-11-09932 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3181335/">PMC free article</a>]</span> [<a href="/pubmed/21980494" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pone.0025581" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plos+ONE&amp;title=Genetics+of+venous+thrombosis:+insights+from+a+new+genome+wide+association+study&amp;author=M+Germain&amp;author=N+Saut&amp;author=N+Greliche&amp;author=C+Dina&amp;author=JC+Lambert&amp;volume=6&amp;publication_year=2011&amp;pages=e25581&amp;pmid=21980494&amp;doi=10.1371/journal.pone.0025581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Cornelis MC, Agrawal A, Cole JW, Hansel NN, Barnes KC, Beaty TH, Bennett SN, Bierut LJ, Boerwinkle E, Doheny KF, Feenstra B, Feingold E, Fornage M, Haiman CA, Harris EL, Hayes MG, Heit JA, Hu FB, Kang JH, Laurie CC, Ling H, Manolio TA, Marazita ML, Mathias RA, Mirel DB, Paschall J, Pasquale LR, Pugh EW, Rice JP, Udren J, van Dam RM, Wang X, Wiggs JL, Williams K, Yu K. The Gene, Environment Association Studies consortium (GENEVA): maximizing the knowledge obtained from GWAS by collaboration across studies of multiple conditions. <span><span class="ref-journal">Genetic Epidemiology. </span>2010;<span class="ref-vol">34</span>:364–72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2860056/">PMC free article</a>]</span> [<a href="/pubmed/20091798" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetic+Epidemiology&amp;title=The+Gene,+Environment+Association+Studies+consortium+(GENEVA):+maximizing+the+knowledge+obtained+from+GWAS+by+collaboration+across+studies+of+multiple+conditions&amp;author=MC+Cornelis&amp;author=A+Agrawal&amp;author=JW+Cole&amp;author=NN+Hansel&amp;author=KC+Barnes&amp;volume=34&amp;publication_year=2010&amp;pages=364-72&amp;pmid=20091798&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Seldin MF, Shigeta R, Villoslada P, Selmi C, Tuomilehto J, Silva G, Belmont JW, Klareskog L, Gregersen PK. European population substructure: clustering of northern and southern populations. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e143.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1564423/">PMC free article</a>]</span> [<a href="/pubmed/17044734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=European+population+substructure:+clustering+of+northern+and+southern+populations&amp;author=MF+Seldin&amp;author=R+Shigeta&amp;author=P+Villoslada&amp;author=C+Selmi&amp;author=J+Tuomilehto&amp;volume=2&amp;publication_year=2006&amp;pages=e143&amp;pmid=17044734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–59.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-59&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Heit JA, Silverstein MD, Mohr DN, Petterson TM, O’Fallon WM, Melton LJI. Risk factors for deep vein thrombosis and pulmonary embolism: a population-based case-control study. <span><span class="ref-journal">Arch Intern Med. </span>2000;<span class="ref-vol">160</span>:809–15.</span> [<a href="/pubmed/10737280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Intern+Med&amp;title=Risk+factors+for+deep+vein+thrombosis+and+pulmonary+embolism:+a+population-based+case-control+study&amp;author=JA+Heit&amp;author=MD+Silverstein&amp;author=DN+Mohr&amp;author=TM+Petterson&amp;author=WM+O’Fallon&amp;volume=160&amp;publication_year=2000&amp;pages=809-15&amp;pmid=10737280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Heit JA, Petterson TM, Owen WG, Burke JP, DE Andrade M, Melton LI. Thrombomodulin gene polymorphisms or haplotypes as potential risk factors for venous thromboembolism: a population-based case-control study. <span><span class="ref-journal">J Thromb Haemost. </span>2005;<span class="ref-vol">3</span>:710–7.</span> [<a href="/pubmed/15842356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=Thrombomodulin+gene+polymorphisms+or+haplotypes+as+potential+risk+factors+for+venous+thromboembolism:+a+population-based+case-control+study&amp;author=JA+Heit&amp;author=TM+Petterson&amp;author=WG+Owen&amp;author=JP+Burke&amp;author=M+DE+Andrade&amp;volume=3&amp;publication_year=2005&amp;pages=710-7&amp;pmid=15842356&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Heit JA, Leibson CL, Ashrani AA, Petterson TM, Bailey KR, Melton LJ., III Is diabetes mellitus an independent risk factor for venous thromboembolism?: a population-based case-control study. <span><span class="ref-journal">Arterioscler Thromb Vasc Biol. </span>2009;<span class="ref-vol">29</span>:1399–405.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2735343/">PMC free article</a>]</span> [<a href="/pubmed/19542020" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=Is+diabetes+mellitus+an+independent+risk+factor+for+venous+thromboembolism?:+a+population-based+case-control+study&amp;author=JA+Heit&amp;author=CL+Leibson&amp;author=AA+Ashrani&amp;author=TM+Petterson&amp;author=KR+Bailey&amp;volume=29&amp;publication_year=2009&amp;pages=1399-405&amp;pmid=19542020&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. <span><span class="ref-journal">Bioinformatics. </span>2009;<span class="ref-vol">25</span>:1754–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2705234/">PMC free article</a>]</span> [<a href="/pubmed/19451168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform&amp;author=H+Li&amp;author=R+Durbin&amp;volume=25&amp;publication_year=2009&amp;pages=1754-60&amp;pmid=19451168&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R. The Sequence Alignment/Map format and SAMtools. <span><span class="ref-journal">Bioinformatics. </span>2009;<span class="ref-vol">25</span>:2078–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2723002/">PMC free article</a>]</span> [<a href="/pubmed/19505943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The+Sequence+Alignment/Map+format+and+SAMtools&amp;author=H+Li&amp;author=B+Handsaker&amp;author=A+Wysoker&amp;author=T+Fennell&amp;author=J+Ruan&amp;volume=25&amp;publication_year=2009&amp;pages=2078-9&amp;pmid=19505943&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, Garimella K, Altshuler D, Gabriel S, Daly M, DePristo MA. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. <span><span class="ref-journal">Genome Res. </span>2010;<span class="ref-vol">20</span>:1297–303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928508/">PMC free article</a>]</span> [<a href="/pubmed/20644199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=The+Genome+Analysis+Toolkit:+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data&amp;author=A+McKenna&amp;author=M+Hanna&amp;author=E+Banks&amp;author=A+Sivachenko&amp;author=K+Cibulskis&amp;volume=20&amp;publication_year=2010&amp;pages=1297-303&amp;pmid=20644199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Goya R, Sun MG, Morin RD, Leung G, Ha G, Wiegand KC, Senz J, Crisan A, Marra MA, Hirst M, Huntsman D, Murphy KP, Aparicio S, Shah SP. SNVMix: predicting single nucleotide variants from next-generation sequencing of tumors. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:730–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832826/">PMC free article</a>]</span> [<a href="/pubmed/20130035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNVMix:+predicting+single+nucleotide+variants+from+next-generation+sequencing+of+tumors&amp;author=R+Goya&amp;author=MG+Sun&amp;author=RD+Morin&amp;author=G+Leung&amp;author=G+Ha&amp;volume=26&amp;publication_year=2010&amp;pages=730-6&amp;pmid=20130035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. <span><span class="ref-journal">Nat Protoc. </span>2009;<span class="ref-vol">4</span>:1073–81.</span> [<a href="/pubmed/19561590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Predicting+the+effects+of+coding+non-synonymous+variants+on+protein+function+using+the+SIFT+algorithm&amp;author=P+Kumar&amp;author=S+Henikoff&amp;author=PC+Ng&amp;volume=4&amp;publication_year=2009&amp;pages=1073-81&amp;pmid=19561590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annual Review of Genomics &amp; Human Genetics. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annual+Review+of+Genomics+&amp;+Human+Genetics&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Matsui T, Titani K, Mizuochi T. Structures of the asparagine-linked oligosaccharide chains of human von Willebrand factor. Occurrence of blood group A, B, and H(O) structures. <span><span class="ref-journal">J Biol Chem. </span>1992;<span class="ref-vol">267</span>:8723–31.</span> [<a href="/pubmed/1577715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Structures+of+the+asparagine-linked+oligosaccharide+chains+of+human+von+Willebrand+factor.+Occurrence+of+blood+group+A,+B,+and+H(O)+structures&amp;author=T+Matsui&amp;author=K+Titani&amp;author=T+Mizuochi&amp;volume=267&amp;publication_year=1992&amp;pages=8723-31&amp;pmid=1577715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Ohira T, Cushman M, Tsai MY, Zhang Y, Heckbert SR, Zakai NA, Rosamond WD, Folsom AR. ABO blood group, other risk factors and incidence of venous thromboembolism: the Longitudinal Investigation of Thromboembolism Etiology (LITE) <span><span class="ref-journal">J Thromb Haemost. </span>2007;<span class="ref-vol">5</span>:1455–61.</span> [<a href="/pubmed/17425663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=ABO+blood+group,+other+risk+factors+and+incidence+of+venous+thromboembolism:+the+Longitudinal+Investigation+of+Thromboembolism+Etiology+(LITE)&amp;author=T+Ohira&amp;author=M+Cushman&amp;author=MY+Tsai&amp;author=Y+Zhang&amp;author=SR+Heckbert&amp;volume=5&amp;publication_year=2007&amp;pages=1455-61&amp;pmid=17425663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Kamphuisen PW, Eikenboom JC, Rosendaal FR, Koster T, Blann AD, Vos HL, Bertina RM. High factor VIII antigen levels increase the risk of venous thrombosis but are not associated with polymorphisms in the von Willebrand factor and factor VIII gene. <span><span class="ref-journal">Br J Haematol. </span>2001;<span class="ref-vol">115</span>:156–8.</span> [<a href="/pubmed/11722428" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=High+factor+VIII+antigen+levels+increase+the+risk+of+venous+thrombosis+but+are+not+associated+with+polymorphisms+in+the+von+Willebrand+factor+and+factor+VIII+gene&amp;author=PW+Kamphuisen&amp;author=JC+Eikenboom&amp;author=FR+Rosendaal&amp;author=T+Koster&amp;author=AD+Blann&amp;volume=115&amp;publication_year=2001&amp;pages=156-8&amp;pmid=11722428&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Tsai AW, Cushman M, Rosamond WD, Heckbert SR, Tracy RP, Aleksic N, Folsom AR. Coagulation factors, inflammation markers, and venous thromboembolism: the longitudinal investigation of thromboembolism etiology (LITE) <span><span class="ref-journal">Am J Med. </span>2002;<span class="ref-vol">113</span>:636–42.</span> [<a href="/pubmed/12505113" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med&amp;title=Coagulation+factors,+inflammation+markers,+and+venous+thromboembolism:+the+longitudinal+investigation+of+thromboembolism+etiology+(LITE)&amp;author=AW+Tsai&amp;author=M+Cushman&amp;author=WD+Rosamond&amp;author=SR+Heckbert&amp;author=RP+Tracy&amp;volume=113&amp;publication_year=2002&amp;pages=636-42&amp;pmid=12505113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">McGrath RT, McKinnon TA, Byrne B, O’Kennedy R, Terraube V, McRae E, Preston RJ, Laffan MA, O’Donnell JS. Expression of terminal alpha2-6-linked sialic acid on von Willebrand factor specifically enhances proteolysis by ADAMTS13. <span><span class="ref-journal">Blood. </span>2010;<span class="ref-vol">115</span>:2666–73.</span> [<a href="/pubmed/19965639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Expression+of+terminal+alpha2-6-linked+sialic+acid+on+von+Willebrand+factor+specifically+enhances+proteolysis+by+ADAMTS13&amp;author=RT+McGrath&amp;author=TA+McKinnon&amp;author=B+Byrne&amp;author=R+O’Kennedy&amp;author=V+Terraube&amp;volume=115&amp;publication_year=2010&amp;pages=2666-73&amp;pmid=19965639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Smith NL, Chen MH, Dehghan A, Strachan DP, Basu S, Soranzo N, Hayward C, Rudan I, Sabater-Lleal M, Bis JC, de Maat MP, Rumley A, Kong X, Yang Q, Williams FM, Vitart V, Campbell H, Malarstig A, Wiggins KL, Van Duijn CM, McArdle WL, Pankow JS, Johnson AD, Silveira A, McKnight B, Uitterlinden AG, Aleksic N, Meigs JB, Peters A, Koenig W, Cushman M, Kathiresan S, Rotter JI, Bovill EG, Hofman A, Boerwinkle E, Tofler GH, Peden JF, Psaty BM, Leebeek F, Folsom AR, Larson MG, Spector TD, Wright AF, Wilson JF, Hamsten A, Lumley T, Witteman JC, Tang W, O’Donnell CJ. Novel associations of multiple genetic loci with plasma levels of factor VII, factor VIII, and von Willebrand factor: The CHARGE (Cohorts for Heart and Aging Research in Genome Epidemiology) Consortium. <span><span class="ref-journal">Circulation. </span>2010;<span class="ref-vol">121</span>:1382–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2861278/">PMC free article</a>]</span> [<a href="/pubmed/20231535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Novel+associations+of+multiple+genetic+loci+with+plasma+levels+of+factor+VII,+factor+VIII,+and+von+Willebrand+factor:+The+CHARGE+(Cohorts+for+Heart+and+Aging+Research+in+Genome+Epidemiology)+Consortium&amp;author=NL+Smith&amp;author=MH+Chen&amp;author=A+Dehghan&amp;author=DP+Strachan&amp;author=S+Basu&amp;volume=121&amp;publication_year=2010&amp;pages=1382-92&amp;pmid=20231535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Wu O, Bayoumi N, Vickers MA, Clark P. ABO(H) blood groups and vascular disease: a systematic review and meta-analysis. <span><span class="ref-journal">J Thromb Haemost. </span>2008;<span class="ref-vol">6</span>:62–9.</span> [<a href="/pubmed/17973651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=ABO(H)+blood+groups+and+vascular+disease:+a+systematic+review+and+meta-analysis&amp;author=O+Wu&amp;author=N+Bayoumi&amp;author=MA+Vickers&amp;author=P+Clark&amp;volume=6&amp;publication_year=2008&amp;pages=62-9&amp;pmid=17973651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Mercier B, Oger E, Le Gal G, Mottier D, Ferec C. Phenotypic but not allelic ABO blood group association with risk of venous thrombosis. <span><span class="ref-journal">Thromb Haemost. </span>2005;<span class="ref-vol">93</span>:388–9.</span> [<a href="/pubmed/15711762" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb+Haemost&amp;title=Phenotypic+but+not+allelic+ABO+blood+group+association+with+risk+of+venous+thrombosis&amp;author=B+Mercier&amp;author=E+Oger&amp;author=G+Le+Gal&amp;author=D+Mottier&amp;author=C+Ferec&amp;volume=93&amp;publication_year=2005&amp;pages=388-9&amp;pmid=15711762&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Morelli VM, De Visser MC, Vos HL, Bertina RM, Rosendaal FR. ABO blood group genotypes and the risk of venous thrombosis: effect of factor V Leiden. <span><span class="ref-journal">J Thromb Haemost. </span>2005;<span class="ref-vol">3</span>:183–5.</span> [<a href="/pubmed/15634288" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=ABO+blood+group+genotypes+and+the+risk+of+venous+thrombosis:+effect+of+factor+V+Leiden&amp;author=VM+Morelli&amp;author=MC+De+Visser&amp;author=HL+Vos&amp;author=RM+Bertina&amp;author=FR+Rosendaal&amp;volume=3&amp;publication_year=2005&amp;pages=183-5&amp;pmid=15634288&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Tirado I, Mateo J, Soria JM, Oliver A, Martinez-Sanchez E, Vallve C, Borrell M, Urrutia T, Fontcuberta J. The ABO blood group genotype and factor VIII levels as independent risk factors for venous thromboembolism. <span><span class="ref-journal">Thromb Haemost. </span>2005;<span class="ref-vol">93</span>:468–74.</span> [<a href="/pubmed/15735796" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb+Haemost&amp;title=The+ABO+blood+group+genotype+and+factor+VIII+levels+as+independent+risk+factors+for+venous+thromboembolism&amp;author=I+Tirado&amp;author=J+Mateo&amp;author=JM+Soria&amp;author=A+Oliver&amp;author=E+Martinez-Sanchez&amp;volume=93&amp;publication_year=2005&amp;pages=468-74&amp;pmid=15735796&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Paiva SG, Sabino AP, Carvalho MG, Ribeiro DD, Gomes KB, Santos MS, Oliveira MS, Lages GG, Dusse LM, Fernandes AP. Polymorphisms in exons 6 and 7 of the ABO locus and their association with venous thrombosis in young Brazilian patients. <span><span class="ref-journal">Blood Coagul Fibrinolysis. </span>2009;<span class="ref-vol">20</span>:122–8.</span> [<a href="/pubmed/19786939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+Coagul+Fibrinolysis&amp;title=Polymorphisms+in+exons+6+and+7+of+the+ABO+locus+and+their+association+with+venous+thrombosis+in+young+Brazilian+patients&amp;author=SG+Paiva&amp;author=AP+Sabino&amp;author=MG+Carvalho&amp;author=DD+Ribeiro&amp;author=KB+Gomes&amp;volume=20&amp;publication_year=2009&amp;pages=122-8&amp;pmid=19786939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Wiggins KL, Smith NL, Glazer NL, Rosendaal FR, Heckbert SR, Psaty BM, Rice KM, Lumley T. ABO genotype and risk of thrombotic events and hemorrhagic stroke. <span><span class="ref-journal">J Thromb Haemost. </span>2009;<span class="ref-vol">7</span>:263–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2946410/">PMC free article</a>]</span> [<a href="/pubmed/19036074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=ABO+genotype+and+risk+of+thrombotic+events+and+hemorrhagic+stroke&amp;author=KL+Wiggins&amp;author=NL+Smith&amp;author=NL+Glazer&amp;author=FR+Rosendaal&amp;author=SR+Heckbert&amp;volume=7&amp;publication_year=2009&amp;pages=263-9&amp;pmid=19036074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Felsenstein J, Churchill GA. A Hidden Markov Model approach to variation among sites in rate of evolution. <span><span class="ref-journal">Mol Biol Evol. </span>1996;<span class="ref-vol">13</span>:93–104.</span> [<a href="/pubmed/8583911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=A+Hidden+Markov+Model+approach+to+variation+among+sites+in+rate+of+evolution&amp;author=J+Felsenstein&amp;author=GA+Churchill&amp;volume=13&amp;publication_year=1996&amp;pages=93-104&amp;pmid=8583911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Cooper GM, Stone EA, Asimenos G, Green ED, Batzoglou S, Sidow A. Distribution and intensity of constraint in mammalian genomic sequence. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:901–13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1172034/">PMC free article</a>]</span> [<a href="/pubmed/15965027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Distribution+and+intensity+of+constraint+in+mammalian+genomic+sequence&amp;author=GM+Cooper&amp;author=EA+Stone&amp;author=G+Asimenos&amp;author=ED+Green&amp;author=S+Batzoglou&amp;volume=15&amp;publication_year=2005&amp;pages=901-13&amp;pmid=15965027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Kalafatis M, Bertina RM, Rand MD, Mann KG. Characterization of the molecular defect in factor VR506Q. <span><span class="ref-journal">J Biol Chem. </span>1995;<span class="ref-vol">270</span>:4053–7.</span> [<a href="/pubmed/7876154" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Characterization+of+the+molecular+defect+in+factor+VR506Q&amp;author=M+Kalafatis&amp;author=RM+Bertina&amp;author=MD+Rand&amp;author=KG+Mann&amp;volume=270&amp;publication_year=1995&amp;pages=4053-7&amp;pmid=7876154&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Juul K, Tybjaerg-Hansen A, Schnohr P, Nordestgaard BG. Factor V Leiden and the risk for venous thromboembolism in the adult Danish population. <span><span class="ref-journal">Ann Intern Med. </span>2004;<span class="ref-vol">140</span>:330–7.</span> [<a href="/pubmed/14996674" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Intern+Med&amp;title=Factor+V+Leiden+and+the+risk+for+venous+thromboembolism+in+the+adult+Danish+population&amp;author=K+Juul&amp;author=A+Tybjaerg-Hansen&amp;author=P+Schnohr&amp;author=BG+Nordestgaard&amp;volume=140&amp;publication_year=2004&amp;pages=330-7&amp;pmid=14996674&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Bertina RM, Koeleman BP, Koster T, Rosendaal FR, Dirven RJ, de Ronde H, van der Velden PA, Reitsma PH. Mutation in blood coagulation factor V associated with resistance to activated protein C. <span><span class="ref-journal">Nature. </span>1994;<span class="ref-vol">369</span>:64–7.</span> [<a href="/pubmed/8164741" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mutation+in+blood+coagulation+factor+V+associated+with+resistance+to+activated+protein+C&amp;author=RM+Bertina&amp;author=BP+Koeleman&amp;author=T+Koster&amp;author=FR+Rosendaal&amp;author=RJ+Dirven&amp;volume=369&amp;publication_year=1994&amp;pages=64-7&amp;pmid=8164741&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Dahlback B, Carlsson M, Svensson PJ. Familial thrombophilia due to a previously unrecognized mechanism characterized by poor anticoagulant response to activated protein C: prediction of a cofactor to activated protein C. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>1993;<span class="ref-vol">90</span>:1004–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC45799/">PMC free article</a>]</span> [<a href="/pubmed/8430067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Familial+thrombophilia+due+to+a+previously+unrecognized+mechanism+characterized+by+poor+anticoagulant+response+to+activated+protein+C:+prediction+of+a+cofactor+to+activated+protein+C&amp;author=B+Dahlback&amp;author=M+Carlsson&amp;author=PJ+Svensson&amp;volume=90&amp;publication_year=1993&amp;pages=1004-8&amp;pmid=8430067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Martinelli I, Bottasso B, Duca F, Faioni E, Mannucci PM. Heightened thrombin generation in individuals with resistance to activated protein C. <span><span class="ref-journal">Thromb Haemost. </span>1996;<span class="ref-vol">75</span>:703–5.</span> [<a href="/pubmed/8725708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb+Haemost&amp;title=Heightened+thrombin+generation+in+individuals+with+resistance+to+activated+protein+C&amp;author=I+Martinelli&amp;author=B+Bottasso&amp;author=F+Duca&amp;author=E+Faioni&amp;author=PM+Mannucci&amp;volume=75&amp;publication_year=1996&amp;pages=703-5&amp;pmid=8725708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Zivelin A, Griffin JH, Xu X, Pabinger I, Samama M, Conard J, Brenner B, Eldor A, Seligsohn U. A single genetic origin for a common Caucasian risk factor for venous thrombosis. <span><span class="ref-journal">Blood. </span>1997;<span class="ref-vol">89</span>:397–402.</span> [<a href="/pubmed/9002940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=A+single+genetic+origin+for+a+common+Caucasian+risk+factor+for+venous+thrombosis&amp;author=A+Zivelin&amp;author=JH+Griffin&amp;author=X+Xu&amp;author=I+Pabinger&amp;author=M+Samama&amp;volume=89&amp;publication_year=1997&amp;pages=397-402&amp;pmid=9002940&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Heit JA, Sobell JL, Li H, Sommer SS. The incidence of venous thromboembolism among Factor V Leiden carriers: a community-based cohort study. <span><span class="ref-journal">Journal of Thrombosis and Haemostasis. </span>2005;<span class="ref-vol">3</span>:305–11.</span> [<a href="/pubmed/15670037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Thrombosis+and+Haemostasis&amp;title=The+incidence+of+venous+thromboembolism+among+Factor+V+Leiden+carriers:+a+community-based+cohort+study&amp;author=JA+Heit&amp;author=JL+Sobell&amp;author=H+Li&amp;author=SS+Sommer&amp;volume=3&amp;publication_year=2005&amp;pages=305-11&amp;pmid=15670037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Andre P. P-selectin in haemostasis. <span><span class="ref-journal">British Journal of Haematology. </span>2004;<span class="ref-vol">126</span>:298–306.</span> [<a href="/pubmed/15257701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=British+Journal+of+Haematology&amp;title=P-selectin+in+haemostasis&amp;author=P+Andre&amp;volume=126&amp;publication_year=2004&amp;pages=298-306&amp;pmid=15257701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Nurden AT. Platelets, inflammation and tissue regeneration. <span><span class="ref-journal">Thrombosis &amp; Haemostasis. </span>2011;<span class="ref-vol">105</span>(Suppl 1):S13–33.</span> [<a href="/pubmed/21479340" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thrombosis+&amp;+Haemostasis&amp;title=Platelets,+inflammation+and+tissue+regeneration&amp;author=AT+Nurden&amp;volume=105&amp;issue=Suppl+1&amp;publication_year=2011&amp;pages=S13-33&amp;pmid=21479340&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Volcik KA, Catellier D, Folsom AR, Matijevic N, Wasserman B, Boerwinkle E. SELP and SELPLG genetic variation is associated with cell surface measures of SELP and SELPLG: the Atherosclerosis Risk in Communities Carotid MRI Study. <span><span class="ref-journal">Clinical Chemistry. </span>2009;<span class="ref-vol">55</span>:1076–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2812411/">PMC free article</a>]</span> [<a href="/pubmed/19395438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Chemistry&amp;title=SELP+and+SELPLG+genetic+variation+is+associated+with+cell+surface+measures+of+SELP+and+SELPLG:+the+Atherosclerosis+Risk+in+Communities+Carotid+MRI+Study&amp;author=KA+Volcik&amp;author=D+Catellier&amp;author=AR+Folsom&amp;author=N+Matijevic&amp;author=B+Wasserman&amp;volume=55&amp;publication_year=2009&amp;pages=1076-82&amp;pmid=19395438&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Marteau JB, Lambert D, Herbeth B, Marie B, Droesch S, Tregouet DA, Visvikis-Siest S. P-selectin polymorphisms’ influences on P-selectin serum concentrations and on their familial correlation: the STANISLAS family study. <span><span class="ref-journal">Journal of Thrombosis and Haemostasis. </span>2008;<span class="ref-vol">6</span>:920–7.</span> [<a href="/pubmed/18363816" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Thrombosis+and+Haemostasis&amp;title=P-selectin+polymorphisms’+influences+on+P-selectin+serum+concentrations+and+on+their+familial+correlation:+the+STANISLAS+family+study&amp;author=JB+Marteau&amp;author=D+Lambert&amp;author=B+Herbeth&amp;author=B+Marie&amp;author=S+Droesch&amp;volume=6&amp;publication_year=2008&amp;pages=920-7&amp;pmid=18363816&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Blann AD, Noteboom WM, Rosendaal FR. Increased soluble P-selectin levels following deep venous thrombosis: cause or effect? <span><span class="ref-journal">British Journal of Haematology. </span>2000;<span class="ref-vol">108</span>:191–3.</span> [<a href="/pubmed/10651744" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=British+Journal+of+Haematology&amp;title=Increased+soluble+P-selectin+levels+following+deep+venous+thrombosis:+cause+or+effect?&amp;author=AD+Blann&amp;author=WM+Noteboom&amp;author=FR+Rosendaal&amp;volume=108&amp;publication_year=2000&amp;pages=191-3&amp;pmid=10651744&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Kyrle PA, Hron G, Eichinger S, Wagner O. Circulating P-selectin and the risk of recurrent venous thromboembolism. <span><span class="ref-journal">Thrombosis &amp; Haemostasis. </span>2007;<span class="ref-vol">97</span>:880–3.</span> [<a href="/pubmed/17549288" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thrombosis+&amp;+Haemostasis&amp;title=Circulating+P-selectin+and+the+risk+of+recurrent+venous+thromboembolism&amp;author=PA+Kyrle&amp;author=G+Hron&amp;author=S+Eichinger&amp;author=O+Wagner&amp;volume=97&amp;publication_year=2007&amp;pages=880-3&amp;pmid=17549288&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Uitte de Willige S, De Visser MC, Vos HL, Houwing-Duistermaat JJ, Rosendaal FR, Bertina RM. Selectin haplotypes and the risk of venous thrombosis: influence of linkage disequilibrium with the factor V Leiden mutation. <span><span class="ref-journal">Journal of Thrombosis and Haemostasis. </span>2008;<span class="ref-vol">6</span>:478–85.</span> [<a href="/pubmed/18182036" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Thrombosis+and+Haemostasis&amp;title=Selectin+haplotypes+and+the+risk+of+venous+thrombosis:+influence+of+linkage+disequilibrium+with+the+factor+V+Leiden+mutation&amp;author=S+Uitte+de+Willige&amp;author=MC+De+Visser&amp;author=HL+Vos&amp;author=JJ+Houwing-Duistermaat&amp;author=FR+Rosendaal&amp;volume=6&amp;publication_year=2008&amp;pages=478-85&amp;pmid=18182036&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Skol A, Scott LJ, Abecasis G, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nature Genetics. </span>2006;<span class="ref-vol">38</span>:209–13.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=A+Skol&amp;author=LJ+Scott&amp;author=G+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-13&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419811&amp;issue-id=213050&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140106493957056title">Summary</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Materials and Methods</a></li><li><a href="#S16title">Results</a></li><li><a href="#S21title">Discussion</a></li><li><a href="#S22title">Supplementary Material</a></li><li><a href="#S23title">Acknowledgments</a></li><li><a href="#idm140106457984784title">Footnotes</a></li><li><a href="#idm140106464174192title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3419811/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3419811/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3419811/pdf/nihms383857.pdf">PDF (1.0M)</a></li> | <li><a href="#" data-citationid="PMC3419811" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419811%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419811%2F&amp;text=A%20genome-wide%20association%20study%20of%20venous%20thromboembolism%20identifies%20risk%20variants%20in%20chromosomes%201q24.2%20and%209q" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419811%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22672568" data-aiid="3419811" data-aid="3419811" data-iid="213050" data-domainid="319" data-domain="nihpa" data-accid="PMC3419811" data-md5="8a1534cb8b0f29b307f65b5f74a3c36f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:03:01-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A3BC8DD53CC710000000007950546&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419811%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3419811/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>