<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="sleep"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3369232/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A Genome-Wide Association Study of Caffeine-Related Sleep Disturbance: Confirmation of a Role for a Common Variant in the Adenosine Receptor</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3369232/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Sleep"><meta name="citation_title" content="A Genome-Wide Association Study of Caffeine-Related Sleep Disturbance: Confirmation of a Role for a Common Variant in the Adenosine Receptor"><meta name="citation_authors" content="Enda M. Byrne, Julie Johnson, Allan F. McRae, Dale R. Nyholt, Sarah E. Medland, Philip R. Gehrman, Andrew C. Heath, Pamela A.F. Madden, Grant W. Montgomery, Georgia Chenevix-Trench, Nicholas G. Martin"><meta name="citation_issue" content="7"><meta name="citation_volume" content="35"><meta name="citation_firstpage" content="967"><meta name="citation_doi" content="10.5665/sleep.1962"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3369232/?report=abstract"><meta name="citation_pmid" content="22754043"><meta name="DC.Title" content="A Genome-Wide Association Study of Caffeine-Related Sleep Disturbance: Confirmation of a Role for a Common Variant in the Adenosine Receptor"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Enda M. Byrne"><meta name="DC.Contributor" content="Julie Johnson"><meta name="DC.Contributor" content="Allan F. McRae"><meta name="DC.Contributor" content="Dale R. Nyholt"><meta name="DC.Contributor" content="Sarah E. Medland"><meta name="DC.Contributor" content="Philip R. Gehrman"><meta name="DC.Contributor" content="Andrew C. Heath"><meta name="DC.Contributor" content="Pamela A.F. Madden"><meta name="DC.Contributor" content="Grant W. Montgomery"><meta name="DC.Contributor" content="Georgia Chenevix-Trench"><meta name="DC.Contributor" content="Nicholas G. Martin"><meta name="DC.Identifier" content="10.5665/sleep.1962"><meta name="DC.Language" content="en"><meta property="og:title" content="A Genome-Wide Association Study of Caffeine-Related Sleep Disturbance: Confirmation of a Role for a Common Variant in the Adenosine Receptor"><meta property="og:type" content="article"><meta property="og:description" content="To identify common genetic variants that predispose to caffeine-induced insomnia and to test whether genes whose expression changes in the presence of caffeine are enriched for association with caffeine-induced insomnia.A hypothesis-free, genome-wide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3369232/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-sleep.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3369232/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3369232/pdf/aasm.35.7.967.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CB806DD53BCF10000000009D6069B.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369232%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gene">Gene</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/369/">Sleep</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/210807/">v.35(7); </a></li><li class="accid">PMC3369232</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-sleep.gif" alt="Logo of sleep" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://academic.oup.com/sleep/" target="pmc_ext" ref="reftype=publisher&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Sleep</a></span>. 2012 Jul 1; 35(7): 967–975. </span></div><div><span class="fm-vol-iss-date">Published online 2012 Jul 1. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.5665%2Fsleep.1962" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.5665/sleep.1962</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3369232</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22754043">22754043</a></div></div></div></div><h1 class="content-title">A Genome-Wide Association Study of Caffeine-Related Sleep Disturbance: Confirmation of a Role for a Common Variant in the Adenosine Receptor</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Byrne%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523452463056" co-class="co-affbox">Enda M. Byrne</a>, PhD,<sup>1,</sup><sup>2</sup><sup></sup> <a href="/pubmed/?term=Johnson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523378867968" co-class="co-affbox">Julie Johnson</a>, BSc,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=McRae%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523451742624" co-class="co-affbox">Allan F. McRae</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523378875312" co-class="co-affbox">Dale R. Nyholt</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Medland%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523381465072" co-class="co-affbox">Sarah E. Medland</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Gehrman%20PR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523455142208" co-class="co-affbox">Philip R. Gehrman</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523414297488" co-class="co-affbox">Andrew C. Heath</a>, DPhil,<sup>4</sup> <a href="/pubmed/?term=Madden%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523379205632" co-class="co-affbox">Pamela A.F. Madden</a>, PhD,<sup>4</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523382257248" co-class="co-affbox">Grant W. Montgomery</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Chenevix-Trench%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523377415856" co-class="co-affbox">Georgia Chenevix-Trench</a>, PhD,<sup>1</sup> and  <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043" class="affpopup" co-rid="_co_idm140523375904016" co-class="co-affbox">Nicholas G. Martin</a>, PhD<sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140523452463056"><h3 class="no_margin">Enda M. Byrne</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</p><p><sup>2</sup>University of Queensland, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Byrne%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Enda M. Byrne</a></div></div><div id="_co_idm140523378867968"><h3 class="no_margin">Julie Johnson</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</p><p><sup>2</sup>University of Queensland, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Johnson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Julie Johnson</a></div></div><div id="_co_idm140523451742624"><h3 class="no_margin">Allan F. McRae</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=McRae%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Allan F. McRae</a></div></div><div id="_co_idm140523378875312"><h3 class="no_margin">Dale R. Nyholt</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Dale R. Nyholt</a></div></div><div id="_co_idm140523381465072"><h3 class="no_margin">Sarah E. Medland</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Medland%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Sarah E. Medland</a></div></div><div id="_co_idm140523455142208"><h3 class="no_margin">Philip R. Gehrman</h3><p><sup>3</sup>Behavioral Sleep Medicine Program, Department of Psychiatry and Penn Sleep Center, University of Pennsylvania, Philadelphia</p><div>Find articles by <a href="/pubmed/?term=Gehrman%20PR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Philip R. Gehrman</a></div></div><div id="_co_idm140523414297488"><h3 class="no_margin">Andrew C. Heath</h3><p><sup>4</sup>Department of Psychiatry, Washington University School of Medicine, St Louis, Missouri</p><div>Find articles by <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Andrew C. Heath</a></div></div><div id="_co_idm140523379205632"><h3 class="no_margin">Pamela A.F. Madden</h3><p><sup>4</sup>Department of Psychiatry, Washington University School of Medicine, St Louis, Missouri</p><div>Find articles by <a href="/pubmed/?term=Madden%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Pamela A.F. Madden</a></div></div><div id="_co_idm140523382257248"><h3 class="no_margin">Grant W. Montgomery</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Grant W. Montgomery</a></div></div><div id="_co_idm140523377415856"><h3 class="no_margin">Georgia Chenevix-Trench</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Chenevix-Trench%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Georgia Chenevix-Trench</a></div></div><div id="_co_idm140523375904016"><h3 class="no_margin">Nicholas G. Martin</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22754043">Nicholas G. Martin</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140523382121024_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140523382121024_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140523382121024_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140523382121024_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Queensland Institute of Medical Research, Brisbane, Australia</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>University of Queensland, Brisbane, Australia</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Behavioral Sleep Medicine Program, Department of Psychiatry and Penn Sleep Center, University of Pennsylvania, Philadelphia</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Psychiatry, Washington University School of Medicine, St Louis, Missouri</div><div id="cor1">Address correspondence to: Enda M. Byrne, Psychiatric Genetics, Queensland Institute of Medical Research, 300 Herston Road, Herston, QLD 4006, Brisbane, Australia, Fax: +61 7 3362 0101, <a href="mailto:dev@null" data-email="ua.ude.rmiq@enryb.adne" class="oemail">ua.ude.rmiq@enryb.adne</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140523382121024_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Sep; Revised 2012 Jan; Accepted 2012 Mar.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140523382121024_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2012 Associated Professional Sleep Societies, LLC.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>See commentary "<a href="/pmc/articles/PMC3368971/">“No Thanks, Coffee Keeps Me Awake”: Individual Caffeine Sensitivity Depends on <em>ADORA2A</em> Genotype</a>"  on&nbsp;page&nbsp;899.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC3369232/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140523455537344" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140523455537344title">Abstract</h2><!--article-meta--><div><div id="__sec1" class="sec sec-first"><h3 id="__sec1title">Objectives:</h3><p id="__p1" class="p p-first-last">To identify common genetic variants that predispose to caffeine-induced insomnia and to test whether genes whose expression changes in the presence of caffeine are enriched for association with caffeine-induced insomnia.</p></div><div id="__sec2" class="sec"><h3 id="__sec2title">Design:</h3><p id="__p2" class="p p-first-last">A hypothesis-free, genome-wide association study.</p></div><div id="__sec3" class="sec"><h3 id="__sec3title">Setting:</h3><p id="__p3" class="p p-first-last">Community-based sample of Australian twins from the Australian Twin Registry.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Participants:</h3><p id="__p4" class="p p-first-last">After removal of individuals who said that they do not drink coffee, a total of 2,402 individuals from 1,470 families in the Australian Twin Registry provided both phenotype and genotype information.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Measurements and Results:</h3><p id="__p5" class="p p-first-last">A dichotomized scale based on whether participants reported ever or never experiencing caffeine-induced insomnia. A factor score based on responses to a number of questions regarding normal sleep habits was included as a covariate in the analysis. More than 2 million common single nucleotide polymorphisms (SNPs) were tested for association with caffeine-induced insomnia. No SNPs reached the genome-wide significance threshold. In the analysis that did not include the insomnia factor score as a covariate, the most significant SNP identified was an intronic SNP in the <em>PRIMA1</em> gene (P = 1.4 × 10<sup>−6</sup>, odds ratio = 0.68 [0.53 – 0.89]). An intergenic SNP near the <em>GBP4</em> gene on chromosome 1 was the most significant upon inclusion of the insomnia factor score into the model (P = 1.9 × 10<sup>−6</sup>, odds ratio = 0.70 [0.62 – 0.78]). A previously identified association with a polymorphism in the <em>ADORA2A</em> gene was replicated.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Conclusions:</h3><p id="__p6" class="p p-first-last">Several genes have been identified in the study as potentially influencing caffeine-induced insomnia. They will require replication in another sample. The results may have implications for understanding the biologic mechanisms underlying insomnia.</p></div><div id="__sec7" class="sec sec-last"><h3 id="__sec7title">Citation:</h3><p id="__p7" class="p p-first-last">Byrne EM; Johnson J; McRae AF; Nyholt DR; Medland SE; Gehrman PR; Heath AC; Madden PAF; Montgomery GW; Chenevix-Trench G; Martin NG. A genome-wide association study of caffeine-related sleep disturbance: confirmation of a role for a common variant in the adenosine receptor. <em>SLEEP</em> 2012;35(7):967-975.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Caffeine, insomnia, genetics</span></div></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">INTRODUCTION</h2><p id="__p8" class="p p-first">Caffeine is the most widely used psychoactive drug in the world, being used habitually by more than 80% of adults worldwide.<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723102">1</a></sup> Many different types of food and beverages contain caffeine, and caffeine consumption and withdrawal is known to cause many side effects. One of the most common side effects of caffeine consumption is sleep loss. The ability to stay awake longer is one of the major reasons for people to consume caffeine, whereas sleep loss is also a major reason for some not to consume caffeine.<sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723107">2</a></sup> Caffeine can also help to alleviate negative mood effects brought on by lack of sleep.<sup><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723103">3</a></sup> Negative effects of caffeine on sleep have been consistently found, including shortened total sleep time, prolonged sleep latency, and changes in the duration of light and deep sleep.<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723067">4</a></sup> In general, coffee consumption is lower in good sleepers than in poor sleepers and one study has shown that coffee consumed close to bedtime caused delayed onset of sleep, independent of total caffeine consumption throughout the day.<sup><a href="#B5" rid="B5" class=" bibr popnode">5</a></sup></p><p id="__p9">Caffeine is thought to be metabolized mainly in the liver by the cytochrome p450 CYP1A2, and metabolites are broken down further by the enzymes CYP2A6 and NAT2.<sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723094">6</a><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723080"></a>–<a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723104">8</a></sup>
Evidence suggests that caffeine mediates its effects by acting as a competitive inhibitor at the adenosine receptors<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723092">1</a></sup> and the A<sub>2A</sub> receptor is hypothesized to be the main target for caffeine in the brain. Consistent with the mechanism of caffeine action on the A<sub>2A</sub> receptor, a polymorphism in the <em>ADORA2A</em> gene has been associated with caffeine consumption.<sup><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723088">9</a></sup></p><p id="__p10">The adenosine pathway is known to affect general sleep,<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723074">10</a>,<a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_808215216">11</a></sup> and caffeine's antagonism of this pathway is thought to explain its effects on sleep. Hence, differential sensitivity to caffeine may explain individual differences in caffeine-related sleep loss. It has been previously shown in our sample that caffeine-related sleep loss is heritable and there are genetic effects independent of those on general insomnia,<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723101">12</a></sup> and a polymorphism in the <em>ADORA2A</em> gene is known to explain some of the variation in sleep loss after caffeine consumption.<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723073">4</a></sup> This finding is supported by mouse models, in which animals lacking A<sub>2A</sub> receptors did not experience caffeine-induced wakefulness.<sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723075">13</a></sup></p><p id="__p11" class="p p-last">So far, efforts to find genetic variants that contribute to variation in caffeine-related sleep effects have been confined to candidate gene studies and this approach has had some success,<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723093">4</a></sup> whereas the only known linkage scan that was performed on the same sample used in the current study was less successful.<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723100">12</a></sup> Genome-wide association studies (GWAS) allow testing of several million common variants across the genome and require no prior hypothesis about potential association between a gene or variant and the phenotype of interest. GWAS have been successful in identifying variants that influence complex traits,<sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723077">14</a></sup> and in many cases, those variants have been part of biologic pathways that had never previously been implicated in those traits. Here, we carry out GWAS for caffeine-related sleep disruption in an attempt to identify additional variants that influence this trait, and to replicate the previous association finding in the <em>ADORA2A</em> gene. In addition, we carry out a gene-based test to identify genes that show evidence of association at multiple single nucleotide polymorphisms (SNPs). We combine the results of GWAS with gene expression data from cells that have been treated with caffeine, to test whether genes regulated by caffeine are more likely to show evidence of association than would be expected by chance. Finally, we use pathway analysis to test whether there is enrichment of association of genes in known biologic pathways.</p></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">METHODS</h2><div id="__sec10" class="sec sec-first"><h3 id="__sec10title">Participants</h3><p id="__p12" class="p p-first-last">Between 1980 and 1982, a health and lifestyle questionnaire was administered by mail to 5,867 complete pairs of twins who had been registered with the Australian Twin Registry. Responses were received from a total of 7,526 individuals (2,746 males and 4,780 females) and they had a mean age of 34.5 yr (standard deviation = 14.3). After removal of individuals who said that they do not drink tea or coffee, a total of 2,402 individuals from 1,470 families provided both phenotype and genotype information. Phenotypic and genotypic data collection was approved by the Queensland Institute of Medical Research Ethics Committee and informed consent was obtained from all participants.</p></div><div id="__sec11" class="sec"><h3 id="__sec11title">Sleep Disturbance Measures</h3><p id="__p13" class="p p-first">As part of the questionnaire, respondents were asked a number of questions about their sleep habits and how often they experience sleep disturbance. These included questions about the overall quality of sleep, variability in quality, depth of sleep, difficulty initiating sleep, anxious insomnia, depressed insomnia, and frequency of night waking. Principal components analysis applied to this dataset previously showed that the sleep measures assessed in the questionnaire - sleep quality, variability of quality, initial insomnia, sleep latency, night-time waking, anxious insomnia, and depressed insomnia - appear to load strongly on a general sleep disturbance component that measures general insomnia.<sup><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723076">15</a></sup> In the current analysis, principal components analysis was performed on the same variables to derive an overall score for insomnia for each individual. The analysis supported a single factor loading on these variables. Further information on the sleep disturbance measures in the questionnaire can be found in several studies.<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723105">12</a>,<a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723064">15</a>,<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723066">16</a></sup></p><p id="__p14">Participants were also asked the following question:</p><p id="__p15">If you were to drink COFFEE in the evening would it stop you from getting to sleep at night?</p><p id="__p16">1) Always 2) Usually 3) Sometimes 4) Never</p><p id="__p17" class="p">Given the low endorsement of the first two categories – Always (3.1%) and Usually (6.0%) – they were pooled with those who were sometimes affected. This divided the sample into those who were ever affected and those who were never affected. After pooling, 678 individuals (32.1%) were classified as having ever suffered from caffeine-related sleep disturbance, whereas 1,432 individuals (67.8%) had never experienced sleep disturbance. The breakdown of the sample by zygosity is given in <a href="/pmc/articles/PMC3369232/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p18">Breakdown of the sample by zygosity</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140523375293296" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t01.jpg" title="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t01.jpg" src="/pmc/articles/PMC3369232/bin/aasm.35.7.967.t01.jpg"></div><div id="largeobj_idm140523375293296" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3369232/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec12" class="sec"><h3 id="__sec12title">Genotyping</h3><p id="__p19" class="p p-first">Genotype information was collected as part of a number of genotyping projects undertaken by the genetic epidemiology group at Queensland Institute of Medical Research. DNA samples were collected in accordance with standard protocols and submitted to different genotype centers using different single nucleotide polymorphism platforms (Illumina 317K, IlluminaHumanCNV370- Quadv3, and Illumina Human 610-Quad, Illumina Inc. San Diego, CA). A standard quality control procedure was used for all of the genotyping projects prior to imputation. A detailed description of the quality control steps and procedure for detection of ancestry outliers is given elsewhere.<sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723089">17</a></sup> A set of 274,604 SNPs that were common to all of the genotyping chips were used for imputation, which was performed using the Markov chain-based haplotyper (MACH) program.<sup><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723065">18</a></sup> The imputation process uses information on the haplotype structure in the human genome from the HapMap project (Release22, Build36) to impute nongenotyped SNPs in the sample. The process that transfers pieces of DNA from one chromosome to another and thus shuffles the combinations, known as recombination, occurs nonrandomly throughout the genome. Therefore, segments of chromosomes are always transmitted as a unit and thus knowledge of the genotype at one allele provides information on the genotypes at other alleles in the same segment of chromosome. The imputation process uses the information from genotyped SNPs to infer genotypes at nongenotyped SNPs. This process therefore allows for combining of genotype information from different SNP platforms as the missing SNPs on each platform can be imputed.</p><p id="__p20" class="p p-last">The imputed SNPs were screened further for mendelian errors, minor allele frequency, and missingness. Only SNPs with an imputation quality score (r<sup>2</sup> – the squared correlation between the estimated genotype scores and true genotypes) greater than 0.3 were retained, which resulted in a total number of 2,380,486 SNPs.</p></div><div id="__sec13" class="sec"><h3 id="__sec13title">Genome-wide Association Study</h3><p id="__p21" class="p p-first">Two separate genome-wide association analyses were performed. In both cases generalized estimating equations<sup><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723084">19</a></sup> were used to test for association. This method allows inclusion of family members in case-control analysis by adjusting the expected SNP variance due to familial relationships using a kinship matrix. The generalized estimating equations method was implemented in the freely available Generalized Disequilibrium Test software.<sup><a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723070">20</a></sup> In the first analysis, only age and sex were included as covariates. In the second analysis, the insomnia factor score was included as a covariate in addition to age and sex. The second test was performed to account for the variance in caffeine-induced insomnia caused by genetic variants that increase risk of insomnia independent of caffeine consumption. In both cases, an allelic test of association was used, which assumes an additive mode of action for each SNP and ignores dominance effects.</p><p id="__p22">We used the threshold of 7.2 × 10<sup>−8</sup> derived using simulations in a Caucasian population by Dudbridge and Gusnanto<sup><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723098">21</a></sup> as the genome-wide significance level.</p><p id="__p23" class="p p-last">Post-GWAS analysis and annotation was carried out using the program WGAViewer.<sup><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723081">22</a></sup> Because of the highly correlated nature of the results from imputed data, some regions will have many SNPs with similar P values. The clumping algorithm in PLINK<sup><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723099">23</a></sup> was used to filter results and find the most significant independent signals. SNPs with pairwise r<sup>2</sup> &lt; 0.5 (a measure of linkage disequilibrium between SNPs) were considered to be independent signals.</p></div><div id="__sec14" class="sec"><h3 id="__sec14title">Gene-Based Test</h3><p id="__p24" class="p p-first-last">To determine whether there are any genes that harbor an excess of SNPs with small P values, a gene-based test of association was performed.<sup><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723090">24</a></sup> The test uses the P values from the single SNP association analysis and computes an overall gene-based test statistic by aggregating the individual SNP effects in each gene, accounting for the number of SNPs in each gene, and the correlation between them due to linkage disequilibrium (non-independence of SNPs). The association results from the analysis that included the insomnia factor score were used for computing the gene-based P values. A total of 17,681 genes were included in the gene-based analysis.</p></div><div id="__sec15" class="sec"><h3 id="__sec15title">Gene Expression Analysis</h3><p id="__p25" class="p p-first">Caffeine treatment was performed on three different cell types: (1) lymphoblastoid cell lines established from the blood of 24 females from hereditary non-<em>BRCA1/2</em> breast cancer families recruited into the Kathleen Cuningham Foundation for Research into Breast Cancer (kConFab); (2) the colon cancer cell line, HT29; and (3) a cell line established from a breast to bone metastasis tumor. Experiments were performed in triplicate for each sample. Optimization experiments determined cell number and caffeine concentration for each cell type. Full details of the methods are provided in the study by Amin et al.<sup><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_299294645">25</a></sup> Total RNA was extracted from both caffeine-treated and untreated control cells using the RNeasy RNA extraction kit (Qiagen, Hilden, Germany). A total of 156 samples (750 ng complementary RNA per sample) were hybridized to HumanHT-12 Expression BeadChips (Illumina Inc, San Diego, CA, USA) using all manufacturer's reagents for washing, detecting, and scanning as per the whole-genome gene expression direct hybridization assay protocol (Illumina Inc, San Diego, CA, USA).</p><p id="__p26">The HumanHT-12 Expression BeadChips (Illumina Inc, San Diego, CA, USA) contains 48,803 probes that cover more than 25,000 annotated genes. Expression data were collated and quality checked in the Illumina BeadStudio and then imported into GeneSpring V10.0 (Agilent Technologies, Santa Clara, CA, USA). Data were quantile normalized to the baseline of the median of all samples and then filtered using an Illumina detection score of 0.95 or greater in at least one sample. To identify differentially expressed genes between each caffeine-treated sample to its own untreated control sample, a linear model was implemented using R and the LInear Mixed Models for MicroArray data (LIMMA)<sup><a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_299294643">26</a></sup> package. Data were adjusted for multiple testing with a false discovery rate of 5%. Two criteria were used to select the set of relevant genes. First, a set of genes with a log odds &gt; (B-statistic) was selected, followed by a hierarchical search of genes based on log-fold changes.</p><p id="__p27" class="p p-last">Gene lists were then imported into GeneSpring V10.0 (Agilent Technologies) for data visualization and to determine overlapping genes between different cell types.</p></div><div id="__sec16" class="sec"><h3 id="__sec16title">Enrichment Analysis</h3><p id="__p28" class="p p-first-last">A rank-test was used to test for enrichment of genes whose expression level changed after treatment with caffeine. A total of 647 genes were differentially expressed in all three cell types and had at least one SNP typed for the gene-based test. To test whether this gene set was enriched in the gene-based test, an empirical distribution of mean ranks was generated by randomly drawing 647 genes from the list in the gene-based test, and calculating the mean position on the list. This was repeated 1 million times, and a P value was calculated by testing how many of the randomly drawn gene sets had a lower mean than the set of differentially expressed genes. In addition, the same test was carried out to compare whether genes that showed the greatest log-fold change either up or down in expression in the lymphoblastoid cell lines were enriched in the analysis.</p></div><div id="__sec17" class="sec sec-last"><h3 id="__sec17title">Pathway Analysis</h3><p id="__p29" class="p p-first-last">To test whether there was an enrichment of associations in genes that act in the same biologic pathway or genes that have strongly related functions, all genes with a P value less than 0.05 from the gene-based test were included in a pathway analysis in the Ingenuity Pathway analysis software (Ingenuity Systems Release 6.0, Ingenuity Systems Inc, Redwood City, CA, USA). The Ingenuity program collates information from published research articles regarding the structure, function, localization, and interactions of proteins and molecules and assigns them to functional and canonical pathways. This permits testing for enrichment of a particular pathway that may be relevant to the trait of interest. The Fisher exact test is used initially to test whether a particular pathway is overrepresented, and the Benjamini-Hochberg method is used to correct the P values for multiple testing. A corrected P &lt; 0.05 was considered to be significant.</p></div></div><div id="__sec18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec18title">RESULTS</h2><p id="__p30" class="p p-first">The quantile-quantile plots of the observed versus expected –log<sub>10</sub>(P) from the two (noninsomnia adjusted and insomnia adjusted) association analyses are presented in <a href="/pmc/articles/PMC3369232/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>. No SNPs passed the genome-wide significance threshold (P &lt; 7.2 × 10<sup>−8</sup>) and there was no evidence of population stratification as evidenced by the genomic control <em>λ</em> (the median <em>χ</em><sup>2</sup> association statistic divided by the median expected under the null hypothesis of no association) being 0.99 for both analyses. Manhattan plots for both analyses are provided in <a href="/pmc/articles/PMC3369232/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3369232/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140523373481600" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967a.jpg" title="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967a.jpg" src="/pmc/articles/PMC3369232/bin/aasm.35.7.967a.jpg"></div></a><div id="largeobj_idm140523373481600" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3369232/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3369232/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3369232/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p31"><strong>(A)</strong> Quantile-quantile (Q-Q) plot for analysis with age and sex as covariates only. <strong>(B)</strong> Q-Q plot for analysis with insomnia factor score included as a covariate.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3369232/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140523369548672" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967b.jpg" title="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967b.jpg" src="/pmc/articles/PMC3369232/bin/aasm.35.7.967b.jpg"></div></a><div id="largeobj_idm140523369548672" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3369232/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3369232/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3369232/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p32">Manhattan plots for the 2 association analyses.</p></div></div></div><p id="__p33"><a href="/pmc/articles/PMC3369232/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> shows the most significant SNPs for the association analysis that was not adjusted for the general insomnia factor score. Many of the most significant associations are with SNPs that are found in intergenic regions some distance from the nearest gene, whereas others are found in or near genes that have not been well annotated and about whom little is known. The most significant SNP – rs6573232 (P = 1.4 × 10<sup>−6</sup>) – is found in an intron of the <em>PRIMA1</em> gene that codes for a proline-rich protein whose function is to organize acetylcholinesterase into tetramers and anchor it to the membrane of neurons.<sup><a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723095">27</a></sup> The gene has been shown to be upregulated during the neuronal differentiation process,<sup><a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723079">28</a></sup> and as such, may represent a good candidate to influence caffeine-induced insomnia.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p34">Annotated list of the most highly associated independent SNPs (P &lt; 10<sup>−5</sup>) for the association analysis that was unadjusted for general insomnia</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140523377416592" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t02.jpg" title="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t02.jpg" src="/pmc/articles/PMC3369232/bin/aasm.35.7.967.t02.jpg"></div><div id="largeobj_idm140523377416592" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3369232/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="__p35">A set of intronic SNPs in the <em>MTUS2</em> gene also shows evidence of association (P = 4.0 × 10<sup>−6</sup>) with caffeine-induced insomnia (<a href="/pmc/articles/PMC3369232/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The <em>MTUS2</em> gene encodes the microtubule-associated tumor suppressor candidate 2. The gene is also known as <em>CAZIP</em> and <em>TIP150</em> and is known to be involved in development of the nervous system.<sup><a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723096">29</a></sup></p><p id="__p36">Of the well-annotated genes, perhaps the most interesting is <em>MTNR1B</em>, which codes for one of two melatonin receptors. rs10830964, a SNP found downstream of this gene, shows strong evidence for association (P = 5.2 × 10<sup>−6</sup>, minor allele frequency (MAF) = 0.12, odds ratio (OR) = 0.62 [0.50 – 0.77]).</p><p id="__p37">Among the other genes that harbor SNPs showing evidence of association are <em>NEDD4L</em>, a gene that is known to regulate and interact with ion channels<sup><a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723108">30</a>,<a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723087">31</a></sup> and that has previously been associated with hypertension,<sup><a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723109">32</a></sup> and <em>P2RY2</em>, a gene that codes for a G protein coupled receptor that is known to bind adenosine and control Ca<sup>2+</sup> influx in cells.</p><p id="__p38">The most strongly associated SNPs from the analysis that included the general insomnia factor score as a covariate are shown in <a href="/pmc/articles/PMC3369232/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>. The most significant SNP (rs521704, P = 1.9 × 10<sup>−6</sup>, MAF = 0.45, OR = 0.70 [0.62 – 0.78]) is an intergenic SNP, the nearest gene being <em>GBP4</em>, which codes for guanylate binding protein 4. <em>GBP4</em> is part of a cluster of guanylate-binding genes on chromosome 1 that are induced by the immune system and hydrolyze guanosine triphosphate to guanosine diphosphate.<sup><a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723085">33</a></sup> Another SNP – rs12407812 - located nearby in <em>GBP1</em>, was also among the most significant variants, and the <em>GBP</em> genes ranked at or near the top of the gene-based test (<a href="/pmc/articles/PMC3369232/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). There is no evidence to suggest that these genes also act on adenosine, but adenosine and guanine have a similar structure. Like caffeine, both molecules are purines, and it is possible that proteins that bind to guanine may be inhibited by caffeine occupying the active site.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p39">Annotated list of the most highly associated independent SNPs (P &lt; 10<sup>−5</sup>) for the association analysis that included the general insomnia factor score as a covariate</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140523380940640" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t03.jpg" title="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t03.jpg" src="/pmc/articles/PMC3369232/bin/aasm.35.7.967.t03.jpg"></div><div id="largeobj_idm140523380940640" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3369232/table/T3/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p40">List of genes with P &lt; 10<sup>−3</sup> for gene-based association test<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140523413761296" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t04.jpg" title="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t04.jpg" src="/pmc/articles/PMC3369232/bin/aasm.35.7.967.t04.jpg"></div><div id="largeobj_idm140523413761296" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3369232/table/T4/?report=objectonly">Open in a separate window</a></div></div><p id="__p41" class="p">Other SNPs that rank among the most associated include SNPs located in introns of the <em>LPHN2</em> and <em>NEBL</em> genes on chromosomes 1 and 10, respectively. <em>LPHN2</em> encodes a G protein-coupled receptor that is expressed in brain that is the primary receptor for alpha-latrotoxin.<sup><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723106">34</a></sup> This gene has been implicated in breast cancer.<sup><a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723069">35</a></sup>
<em>NEBL</em> encodes a nebulin-like protein that is expressed primarily in cardiac tissue.</p><div id="__sec19" class="sec"><h3 id="__sec19title">Testing of Previously Reported Association with rs5751876</h3><p id="__p42" class="p p-first-last">A previous study identified rs5751876, a SNP in the adenosine receptor gene <em>ADORA2A</em>, as being associated with individual sensitivity to caffeine's effects on sleep (P = 0.03).<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723083">4</a></sup> It was noted that those with C/C genotype were more likely to be sensitive to caffeine's effects on sleep. The C allele is the major allele with a frequency of 59%, so those carrying the minor allele were less likely to be sensitive. Because this SNP was not included in the HapMap project and was also not included on the genotyping chips used here, we could not directly try to replicate those results. However, based on data from the 1000 Genomes Project (<a href="http://www.1000genomes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.1000genomes.org</a>), we identified a number of SNPs (rs5760423, rs5760425, rs3761422, MAF = 0.38) in the <em>ADORA2A</em> gene in perfect linkage disequilibrium (LD) with the previously identified variant that were imputed in our dataset. We were able to replicate the previous findings, with the LD proxy SNPs having P = 0.019 with an OR of 0.62 (0.50 – 0.78) for the minor allele. From the OR it can be seen that the minor allele decreases the likelihood of experiencing insomnia after coffee consumption, replicating the findings of Retey et al.<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723082">4</a></sup> A further SNP, rs4822498, that is in very strong LD (r<sup>2</sup> = 0.99) with the previously associated SNP showed the strongest evidence for association of all the SNPs within 50 kb either side of the <em>ADORA2A</em> gene (P = 0.0091, OR = 0.61 [0.50 – 0.76]). This SNP is located in an open reading frame that is downstream of the <em>ADORA2A</em> gene on chromosome 22 and is encoded on the opposite strand (<em>C22orf45</em>). A test for nonadditive effects (dominance and recessive) was performed for the most significant <em>ADORA2A</em> SNP (rs4822498) and for one of the SNPs in perfect LD with the previously identified SNP, but no significant effects were found. In addition, a test for interaction effects between rs4822498 and rs5760423 was performed, but there was no evidence of an interaction (P = 0.98). Furthermore, an association scan of the region that conditioned upon the SNP rs4822498 failed to identify any other significant SNPs in the ADORA2A region.</p></div><div id="__sec20" class="sec"><h3 id="__sec20title">Gene-Based Test</h3><p id="__p43" class="p p-first">The most associated genes from the Versatile Gene-based Association Study (VEGAS) gene-based test are provided in <a href="/pmc/articles/PMC3369232/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>. As a comparison, the results for a number of candidate genes for caffeine-related insomnia are shown in gray. The <em>ADORA2A</em> gene ranked 527<sup>th</sup> on the list and had a P value of 0.028.</p><p id="__p44">A total of 647 autosomal genes were found to be differentially expressed in all 3 cell types and also analyzed in the gene-based test. The highest ranking of these, at 43<sup>rd</sup>, is <em>PDCD4</em> (P = 0.0025), a gene that is thought to play a role in programmed cell death. This gene is a tumor suppressor gene that is downregulated in a number of different forms of cancer.<sup><a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723068">36</a>,<a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723072">37</a></sup> There was no evidence to suggest that the differentially expressed genes are more highly associated than would be expected by chance (P = 0.21).</p><p id="__p45" class="p">Interestingly, the second highest-ranked gene in the gene-based test, <em>GBP1</em>, was found to be differentially expressed in two of the cell types (lymphoblastoid cell lines and bone metastasis), but not the other. After relaxing the constraints for inclusion in the rank-test to those genes that are differentially expressed in at least two genes (n = 2,038), there is still no evidence for enrichment of association (P = 0.16). A similar analysis of the most highly changed genes as measured by log-fold change in expression levels also showed no evidence for enrichment. <a href="/pmc/articles/PMC3369232/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a> lists genes ranked in the top 50 most significant genes in the gene-based test and that are differentially expressed in at least two of the cell types.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p46">List of genes ranked in the top 50 most significant in the gene-based test and that were found to be differentially expressed in at least 2 of the cell types</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140523382445280" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t05.jpg" title="An external file that holds a picture, illustration, etc.
Object name is aasm.35.7.967.t05.jpg" src="/pmc/articles/PMC3369232/bin/aasm.35.7.967.t05.jpg"></div><div id="largeobj_idm140523382445280" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3369232/table/T5/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec21" class="sec sec-last"><h3 id="__sec21title">Pathway Analysis</h3><p id="__p47" class="p p-first-last">A total of 628 genes had P &lt; 0.05 from the gene-based test and could be assigned a function or pathway in the Ingenuity program. After accounting for multiple testing, no functional or canonical pathways showed evidence for enrichment. The most strongly significant biologic function was nucleic acid metabolism (P = 1.6 × 10<sup>−4</sup>). The nucleic acid metabolism function is further subdivided into more specific categories, many of which relate specifically to adenosine metabolism. The role of <em>ADORA2A</em> in mediating the signaling effects of adenosine were mentioned previously, but there are several other genes involved in adenosine metabolism that show evidence for association. The genes <em>FXYD2</em> and <em>EPHB6</em> are both involved in increasing binding affinity of adenosine triphosphatase (ATPase) enzymes for ATP, thus increasing ATP metabolism, whereas <em>ADK</em> phosphorylates adenosine to adenosine monophosphate and <em>ADAT1</em> is known to increase deamination of adenosine. Additionally, <em>PAPSS1</em> is involved in the synthesis of phosphoadenosine. Other genes in the nucleic acid such as <em>RRM2</em> fall in to the nucleic acid metabolism category, but are not adenosine-specific. Given the evidence to suggest the involvement of adenosine in both general sleep and in caffeine sensitivity, it is likely not a coincidence that the most significantly enriched pathway for association includes several genes involved in adenosine metabolism. Despite the lack of experiment-wise significance, variants in these genes are good candidates for association with caffeine-attributed insomnia.</p></div></div><div id="__sec22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec22title">DISCUSSION</h2><p id="__p48" class="p p-first">We have undertaken a genome-wide association study of caffeine-related sleep disturbance. Previous analysis of the dataset used here demonstrated that this trait is heritable and is influenced by genetic factors that are independent of those influencing general insomnia.<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723091">12</a></sup> The advantage of the GWAS approach is that it requires no prior hypothesis and in many cases, variants in genes that were never previously thought to be associated with a trait have been discovered, elucidating biologic pathways that have an effect on the phenotype of interest. Due to the large number of tests being carried out, stringent significance levels are used to avoid false-positive associations, and as such, large sample sizes are required to detect variants with small effects on the phenotype. In this study, we have combined gene expression data from cells treated with caffeine with GWAS data in an attempt to gain more insight on caffeine metabolism in cells and differences in within-population effects of caffeine.</p><p id="__p49">We have identified a number of genes that show evidence of association with caffeine-induced insomnia. The most significant association when analyzing the caffeine-related insomnia without adjusting for general insomnia was rs6575353 (P = 1.6 × 10<sup>−6</sup>) located in an intron of the <em>PRIMA1</em> gene on chromosome 14. When general insomnia was included in the analysis, the most significant SNP was rs521704 (P = 1.9 × 10<sup>−6</sup>) located approximately 15 kb upstream of the <em>GBP4</em> gene on chromosome 1. These associations did not reach genome-wide significance, but these SNPs can be targeted for replication in another sample.</p><p id="__p50">Another potential target for replication is rs10830964, located near the <em>MTNR1B</em> gene, which encodes a subunit of the melatonin receptor that is expressed primarily in the retina and brain. The SNP was among the highest ranked in the analysis that excluded general insomnia (P = 5.2 × 10<sup>−6</sup>). Melatonin is a hormone produced by the pineal gland in the brain in response to light and dark signals from the retina. It is an important point of control of circadian rhythms and is at its highest concentration in the middle of the night and lowest during the day. Studies have shown that prolonged release of melatonin can help to cure insomnia in older people,<sup><a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723078">38</a></sup> and a drug called ramelteon, which acts as an agonist of the melatonin receptor B1, has been approved in clinical trials to treat insomnia (<a href="http://www.fda.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.fda.gov</a>). Many other supplements that are designed to improve sleep quality contain slow-release melatonin. Polymorphisms in or near this gene are good candidates to contribute to susceptibility to both caffeine-related and general insomnia. Common genetic variants in the <em>MTNR1B</em> gene have also been found to influence fasting glucose levels and type 2 diabetes in multiple populations.<sup><a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723097">39</a>,<a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262723086">40</a></sup> The SNP remained nominally significant after including the insomnia factor score as a covariate (P = 0.0003), indicating that its effect may act on caffeine-induced insomnia rather than on general insomnia.</p><p id="__p51">In addition, we were able to replicate the previous findings of an association between a polymorphism in the <em>ADORA2A</em> gene rs5751876 and sleep disruption after caffeine consumption. Although the SNP was not typed in our sample, we were able to use information from the 1000 Genomes Project to identify proxy SNPs in complete LD with the candidate SNP. This result is notable in that is one of very few associations from the candidate gene era in behavioral genetics to replicate in the GWAS era. It also highlights the difficulty in isolating the true causal variant in genetic association studies. The association signal identified in the initial study may in fact be due to correlation with other variants (potentially variants not tested for in GWAS) in the same vicinity. But the <em>ADORA2A</em> gene should be targeted for future sequencing and functional studies in the context of caffeine consumption, caffeine-induced insomnia and general insomnia. These studies may help to identify the true causal variant in this region of the genome.</p><p id="__p52">There are several factors that may have affected the power of our study. First, for the sleep disturbance measures, we have assigned each individual to one of two categories based on whether they had ever or never experienced sleep disturbance after drinking coffee. This was due to the very small numbers of people who always or usually experienced sleep disturbance after drinking coffee in the evening. So that the assumptions made when performing the association test are not violated, the trait was dichotomized. This leads to a loss of power as individuals who experience more severe caffeine-induced insomnia may be enriched for risk alleles relative to those who experience only minor insomnia and this variance is lost. In addition, we are relying on self-report data for all of the variables analyzed. These data may be influenced by perceptual biases. It is known that people with insomnia often underestimate how much sleep they receive. To this end, studies of sleep loss often use polysomnography or actigraphy to get an objective measure of sleep loss. These methods provide greater accuracy of measurement, but they are much more time-consuming and less conducive to the large sample sizes required for GWAS. It is worth noting, however, that the association in <em>ADORA2A</em> that was initially discovered in a sample that used polysomnography to assess caffeine-induced insomnia was replicated in our sample that relied on self-report data on the same phenotype, which suggests that self-report data, while imperfect, is a somewhat reliable measure of the true phenotype and may be of particular use in large-scale genetic studies.</p><p id="__p53">From the gene expression analysis, more than 600 genes that showed differential expression in each of three separate cell types were identified. All experiments were performed in triplicate, providing strong statistical support for the genes identified. On the basis that much of the genetic control of gene expression is by <em>cis</em>-acting elements, we tested to determine whether there was an enrichment of associations in SNPs in the differentially expressed genes. There is no evidence that these genes are enriched for associations; however, we have identified several genes and SNPs therein that show strong evidence for association. These genes represent good targets for replication in another sample. In addition, several of the top genes for both phenotypes show differential expression in at least two of the cell types, and so are good candidate genes for playing a role in caffeine sensitivity.</p><p id="__p54">Analysis of gene expression data can give insight into the genetic underpinnings of phenotypic variation and, in the case of caffeine, the downstream changes in expression that occur are not well understood. The data analyzed here help to elucidate some of those changes. Understanding these changes in gene expression will aid in understanding the biologic pathways underlying sleep regulation in humans, and will have implications for insomnia and how it can be treated. The gene expression results should be interpreted with caution for reasons outlined elsewhere.<sup><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_299294644">25</a></sup> However, the results from gene expression experiments can aid in the interpretation of results from GWAS studies by providing more evidence for a role for specific genes that show evidence of association with the phenotype of interest. This is particularly useful given that in many cases, the sample sizes required to detect genome-wide significant effects are not achievable.</p><p id="__p55" class="p p-last">Pathway analysis represents another method of analyzing GWAS data and can offer insights into the genetic architecture of phenotypes. Given that most of the variants found in GWAS so far have had a small effect on the phenotype, it is likely many genes of small effect account for most of the heritability of complex traits. Many of these variants could potentially be found in the same pathway, with minor disruptions in several points in a pathway adding up to a substantial contribution to the observed variance. One of the most significantly enriched pathways in the association analysis was nucleic acid metabolism, with many of the genes specifically involved in adenosine metabolism. This is not inconsistent with the hypothesized mode of action of caffeine in the brain, and the general pathways involved in sleep.</p></div><div id="__sec23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec23title">DISCLOSURE STATEMENT</h2><p id="__p56" class="p p-first-last">This was not an industry supported study. Dr. Madden is largely supported by NIH grants. The other authors have indicated no financial conflicts of interest.</p></div><div id="idm140523449541120" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140523449541120title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p58">The authors thank the twins and their families for their participation. They also thank Dixie Statham, Ann Eldridge, Marlene Grace, Kerrie McAloney, Lisa Bowdler, Steven Crooks, Peter Visscher and Naomi Wray. Funding was provided by the Australian National Health and Medical Research Council (241944, 339462, 389927, 389875, 389891, 389892, 389938, 442915, 442981, 496739, 552485, 552498, 613608), the Australian Research Council (A7960034, A79906588, A79801419, DP0770096, DP0212016, DP0343921), the FP-5 GenomEUtwin Project (QLG2-CT-2002-01254), and the U.S. National Institutes of Health (NIH grants AA07535, AA10248, AA13320, AA13321, AA13326, AA14041, MH66206). E.M.B is supported by the Australian National Health and Medical Research Council grant 613608. G.W.M, D.R.N and S.E.M. are supported by the National Health and Medical Research Council (NHMRC) Fellowship Scheme. A portion of the genotyping on which this study was based (Illumina 370K scans on 4300 individuals) was carried out at the Center for Inherited Disease Research, Baltimore (CIDR), through an access award to the authors’ late colleague Dr. Richard Todd (Psychiatry, Washington University School of Medicine, St Louis). Statistical analyses were carried out on the Genetic Cluster Computer, which is financially supported by the Netherlands Scientific Organization (NWO 480-05-003).</p></div></div><div id="idm140523367689792" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140523367689792title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140523367689664"></p><p id="__p57" class="p p-first-last">A commentary on this article appears in this issue on page 899.</p><p></p></div></div><div id="idm140523449540240" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140523449540240title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Fredholm BB, Battig K, Holmen J, Nehlig A, Zvartau EE. Actions of caffeine in the brain with special reference to factors that contribute to its widespread use. <span><span class="ref-journal">Pharmacol Rev. </span>1999;<span class="ref-vol">51</span>:83–133.</span> [<a href="/pubmed/10049999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol+Rev&amp;title=Actions+of+caffeine+in+the+brain+with+special+reference+to+factors+that+contribute+to+its+widespread+use&amp;author=BB+Fredholm&amp;author=K+Battig&amp;author=J+Holmen&amp;author=A+Nehlig&amp;author=EE+Zvartau&amp;volume=51&amp;publication_year=1999&amp;pages=83-133&amp;pmid=10049999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Soroko S, Chang J, Barrett-Connor E. Reasons for changing caffeinated coffee consumption: the Rancho Bernardo Study. <span><span class="ref-journal">J Am Coll Nutr. </span>1996;<span class="ref-vol">15</span>:97–101.</span> [<a href="/pubmed/8632123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Nutr&amp;title=Reasons+for+changing+caffeinated+coffee+consumption:+the+Rancho+Bernardo+Study&amp;author=S+Soroko&amp;author=J+Chang&amp;author=E+Barrett-Connor&amp;volume=15&amp;publication_year=1996&amp;pages=97-101&amp;pmid=8632123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Penetar D, McCann U, Thorne D, et al.  Caffeine reversal of sleep deprivation effects on alertness and mood. <span><span class="ref-journal">Psychopharmacology (Berl) </span>1993;<span class="ref-vol">112</span>:359–65.</span> [<a href="/pubmed/7871042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychopharmacology+(Berl)&amp;title=Caffeine+reversal+of+sleep+deprivation+effects+on+alertness+and+mood&amp;author=D+Penetar&amp;author=U+McCann&amp;author=D+Thorne&amp;volume=112&amp;publication_year=1993&amp;pages=359-65&amp;pmid=7871042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Retey JV, Adam M, Khatami R, et al.  A genetic variation in the adenosine A2A receptor gene (ADORA2A) contributes to individual sensitivity to caffeine effects on sleep. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2007;<span class="ref-vol">81</span>:692–8.</span> [<a href="/pubmed/17329997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=A+genetic+variation+in+the+adenosine+A2A+receptor+gene+(ADORA2A)+contributes+to+individual+sensitivity+to+caffeine+effects+on+sleep&amp;author=JV+Retey&amp;author=M+Adam&amp;author=R+Khatami&amp;volume=81&amp;publication_year=2007&amp;pages=692-8&amp;pmid=17329997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Pantelios G, Lack L, James JE. Caffeine consumption and sleep. <span><span class="ref-journal">Sleep. </span>1989;<span class="ref-vol">18</span>:65.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sleep&amp;title=Caffeine+consumption+and+sleep&amp;author=G+Pantelios&amp;author=L+Lack&amp;author=JE+James&amp;volume=18&amp;publication_year=1989&amp;pages=65&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Higdon JV, Frei B. Coffee and health: a review of recent human research. <span><span class="ref-journal">Crit Rev Food Sci Nutr. </span>2006;<span class="ref-vol">46</span>:101–23.</span> [<a href="/pubmed/16507475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit+Rev+Food+Sci+Nutr&amp;title=Coffee+and+health:+a+review+of+recent+human+research&amp;author=JV+Higdon&amp;author=B+Frei&amp;volume=46&amp;publication_year=2006&amp;pages=101-23&amp;pmid=16507475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Krul C, Hageman G. Analysis of urinary caffeine metabolites to assess biotransformation enzyme activities by reversed-phase high-performance liquid chromatography. <span><span class="ref-journal">J Chromatogr B Biomed Sci Appl. </span>1998;<span class="ref-vol">709</span>:27–34.</span> [<a href="/pubmed/9653923" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chromatogr+B+Biomed+Sci+Appl&amp;title=Analysis+of+urinary+caffeine+metabolites+to+assess+biotransformation+enzyme+activities+by+reversed-phase+high-performance+liquid+chromatography&amp;author=C+Krul&amp;author=G+Hageman&amp;volume=709&amp;publication_year=1998&amp;pages=27-34&amp;pmid=9653923&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Crews HM, Olivier L, Wilson LA. Urinary biomarkers for assessing dietary exposure to caffeine. <span><span class="ref-journal">Food Addit Contam. </span>2001;<span class="ref-vol">18</span>:1075–87.</span> [<a href="/pubmed/11761118" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Food+Addit+Contam&amp;title=Urinary+biomarkers+for+assessing+dietary+exposure+to+caffeine&amp;author=HM+Crews&amp;author=L+Olivier&amp;author=LA+Wilson&amp;volume=18&amp;publication_year=2001&amp;pages=1075-87&amp;pmid=11761118&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Cornelis MC, El-Sohemy A, Campos H. Genetic polymorphism of the adenosine A2A receptor is associated with habitual caffeine consumption. <span><span class="ref-journal">Am J Clin Nutr. </span>2007;<span class="ref-vol">86</span>:240–4.</span> [<a href="/pubmed/17616786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Clin+Nutr&amp;title=Genetic+polymorphism+of+the+adenosine+A2A+receptor+is+associated+with+habitual+caffeine+consumption&amp;author=MC+Cornelis&amp;author=A+El-Sohemy&amp;author=H+Campos&amp;volume=86&amp;publication_year=2007&amp;pages=240-4&amp;pmid=17616786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Retey JV, Adam M, Honegger E, et al.  A functional genetic variation of adenosine deaminase affects the duration and intensity of deep sleep in humans. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2005;<span class="ref-vol">102</span>:15676–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1266101/">PMC free article</a>]</span> [<a href="/pubmed/16221767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=A+functional+genetic+variation+of+adenosine+deaminase+affects+the+duration+and+intensity+of+deep+sleep+in+humans&amp;author=JV+Retey&amp;author=M+Adam&amp;author=E+Honegger&amp;volume=102&amp;publication_year=2005&amp;pages=15676-81&amp;pmid=16221767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Retey JV, Adam M, Gottselig JM, et al.  Adenosinergic mechanisms contribute to individual differences in sleep deprivation-induced changes in neurobehavioral function and brain rhythmic activity. <span><span class="ref-journal">J Neurosci. </span>2006;<span class="ref-vol">26</span>:10472–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6674679/">PMC free article</a>]</span> [<a href="/pubmed/17035531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Adenosinergic+mechanisms+contribute+to+individual+differences+in+sleep+deprivation-induced+changes+in+neurobehavioral+function+and+brain+rhythmic+activity&amp;author=JV+Retey&amp;author=M+Adam&amp;author=JM+Gottselig&amp;volume=26&amp;publication_year=2006&amp;pages=10472-9&amp;pmid=17035531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Luciano M, Zhu G, Kirk KM, et al.  “No thanks, it keeps me awake”: the genetics of coffee-attributed sleep disturbance. <span><span class="ref-journal">Sleep. </span>2007;<span class="ref-vol">30</span>:1378–86.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2266272/">PMC free article</a>]</span> [<a href="/pubmed/17969472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sleep&amp;title=“No+thanks,+it+keeps+me+awake”:+the+genetics+of+coffee-attributed+sleep+disturbance&amp;author=M+Luciano&amp;author=G+Zhu&amp;author=KM+Kirk&amp;volume=30&amp;publication_year=2007&amp;pages=1378-86&amp;pmid=17969472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Huang ZL, Qu WM, Eguchi N, et al.  Adenosine A2A, but not A1, receptors mediate the arousal effect of caffeine. <span><span class="ref-journal">Nat Neurosci. </span>2005;<span class="ref-vol">8</span>:858–9.</span> [<a href="/pubmed/15965471" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Adenosine+A2A,+but+not+A1,+receptors+mediate+the+arousal+effect+of+caffeine&amp;author=ZL+Huang&amp;author=WM+Qu&amp;author=N+Eguchi&amp;volume=8&amp;publication_year=2005&amp;pages=858-9&amp;pmid=15965471&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">The Wellcome Trust Case-Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-78&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Heath AC, Kendler KS, Eaves LJ, Martin NG. Evidence for genetic influences on sleep disturbance and sleep pattern in twins. <span><span class="ref-journal">Sleep. </span>1990;<span class="ref-vol">13</span>:318–35.</span> [<a href="/pubmed/2267475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sleep&amp;title=Evidence+for+genetic+influences+on+sleep+disturbance+and+sleep+pattern+in+twins&amp;author=AC+Heath&amp;author=KS+Kendler&amp;author=LJ+Eaves&amp;author=NG+Martin&amp;volume=13&amp;publication_year=1990&amp;pages=318-35&amp;pmid=2267475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Heath AC, Eaves LJ, Kirk KM, Martin NG. Effects of lifestyle, personality, symptoms of anxiety and depression, and genetic predisposition on subjective sleep disturbance and sleep pattern. <span><span class="ref-journal">Twin Res. </span>1998;<span class="ref-vol">1</span>:176–88.</span> [<a href="/pubmed/10100809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res&amp;title=Effects+of+lifestyle,+personality,+symptoms+of+anxiety+and+depression,+and+genetic+predisposition+on+subjective+sleep+disturbance+and+sleep+pattern&amp;author=AC+Heath&amp;author=LJ+Eaves&amp;author=KM+Kirk&amp;author=NG+Martin&amp;volume=1&amp;publication_year=1998&amp;pages=176-88&amp;pmid=10100809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">Medland SE, Nyholt DR, Painter JN, et al.  Common variants in the trichohyalin gene are associated with straight hair in Europeans. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">85</span>:750–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775823/">PMC free article</a>]</span> [<a href="/pubmed/19896111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Common+variants+in+the+trichohyalin+gene+are+associated+with+straight+hair+in+Europeans&amp;author=SE+Medland&amp;author=DR+Nyholt&amp;author=JN+Painter&amp;volume=85&amp;publication_year=2009&amp;pages=750-5&amp;pmid=19896111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Zeger SL, Liang KY. Longitudinal data analysis for discrete and continuous outcomes. <span><span class="ref-journal">Biometrics. </span>1986;<span class="ref-vol">42</span>:121–30.</span> [<a href="/pubmed/3719049" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Longitudinal+data+analysis+for+discrete+and+continuous+outcomes&amp;author=SL+Zeger&amp;author=KY+Liang&amp;volume=42&amp;publication_year=1986&amp;pages=121-30&amp;pmid=3719049&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">Chen WM, Manichaikul A, Rich SS. A generalized family-based association test for dichotomous traits. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">85</span>:364–76.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2771538/">PMC free article</a>]</span> [<a href="/pubmed/19732865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+generalized+family-based+association+test+for+dichotomous+traits&amp;author=WM+Chen&amp;author=A+Manichaikul&amp;author=SS+Rich&amp;volume=85&amp;publication_year=2009&amp;pages=364-76&amp;pmid=19732865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">Dudbridge F, Gusnanto A. Estimation of significance thresholds for genomewide association scans. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:227–34.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2573032/">PMC free article</a>]</span> [<a href="/pubmed/18300295" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+significance+thresholds+for+genomewide+association+scans&amp;author=F+Dudbridge&amp;author=A+Gusnanto&amp;volume=32&amp;publication_year=2008&amp;pages=227-34&amp;pmid=18300295&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">Ge D, Zhang K, Need AC, et al.  WGAViewer: software for genomic annotation of whole genome association studies. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:640–3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2279251/">PMC free article</a>]</span> [<a href="/pubmed/18256235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=WGAViewer:+software+for+genomic+annotation+of+whole+genome+association+studies&amp;author=D+Ge&amp;author=K+Zhang&amp;author=AC+Need&amp;volume=18&amp;publication_year=2008&amp;pages=640-3&amp;pmid=18256235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Liu JZ, McRae AF, Nyholt DR, et al.  A versatile gene-based test for genome-wide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:139–45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896770/">PMC free article</a>]</span> [<a href="/pubmed/20598278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+versatile+gene-based+test+for+genome-wide+association+studies&amp;author=JZ+Liu&amp;author=AF+McRae&amp;author=DR+Nyholt&amp;volume=87&amp;publication_year=2010&amp;pages=139-45&amp;pmid=20598278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Amin N, Byrne E, Johnson J, et al.  Genome-wide association analysis of coffee drinking suggests association with CYP1A1/CYP1A2 and NRCAM. <span><span class="ref-journal">Mol Psychiatry. </span>2011</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3482684/">PMC free article</a>]</span> [<a href="/pubmed/21876539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Genome-wide+association+analysis+of+coffee+drinking+suggests+association+with+CYP1A1/CYP1A2+and+NRCAM&amp;author=N+Amin&amp;author=E+Byrne&amp;author=J+Johnson&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">Smyth GK. Linear models and empirical bayes methods for assessing differential expression in microarray experiments. <span><span class="ref-journal">Stat Appl Genet Mol Biol. </span>2004;<span class="ref-vol">3</span></span> Article3. [<a href="/pubmed/16646809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Appl+Genet+Mol+Biol&amp;title=Linear+models+and+empirical+bayes+methods+for+assessing+differential+expression+in+microarray+experiments&amp;author=GK+Smyth&amp;volume=3&amp;publication_year=2004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Perrier AL, Massoulie J, Krejci E. PRiMA: the membrane anchor of acetylcholinesterase in the brain. <span><span class="ref-journal">Neuron. </span>2002;<span class="ref-vol">33</span>:275–85.</span> [<a href="/pubmed/11804574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=PRiMA:+the+membrane+anchor+of+acetylcholinesterase+in+the+brain&amp;author=AL+Perrier&amp;author=J+Massoulie&amp;author=E+Krejci&amp;volume=33&amp;publication_year=2002&amp;pages=275-85&amp;pmid=11804574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">Xie HQ, Choi RC, Leung KW, Chen VP, Chu GK, Tsim KW. Transcriptional regulation of proline-rich membrane anchor (PRiMA) of globular form acetylcholinesterase in neuron: an inductive effect of neuron differentiation. <span><span class="ref-journal">Brain Res. </span>2009;<span class="ref-vol">1265</span>:13–23.</span> [<a href="/pubmed/19368807" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res&amp;title=Transcriptional+regulation+of+proline-rich+membrane+anchor+(PRiMA)+of+globular+form+acetylcholinesterase+in+neuron:+an+inductive+effect+of+neuron+differentiation&amp;author=HQ+Xie&amp;author=RC+Choi&amp;author=KW+Leung&amp;author=VP+Chen&amp;author=GK+Chu&amp;volume=1265&amp;publication_year=2009&amp;pages=13-23&amp;pmid=19368807&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Du Puy L, Beqqali A, Monshouwer-Kloots J, Haagsman HP, Roelen BA, Passier R. CAZIP, a novel protein expressed in the developing heart and nervous system. <span><span class="ref-journal">Dev Dyn. </span>2009;<span class="ref-vol">238</span>:2903–11.</span> [<a href="/pubmed/19806667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Dyn&amp;title=CAZIP,+a+novel+protein+expressed+in+the+developing+heart+and+nervous+system&amp;author=L+Du+Puy&amp;author=A+Beqqali&amp;author=J+Monshouwer-Kloots&amp;author=HP+Haagsman&amp;author=BA+Roelen&amp;volume=238&amp;publication_year=2009&amp;pages=2903-11&amp;pmid=19806667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">He Y, Hryciw DH, Carroll ML, et al.  The ubiquitin-protein ligase Nedd4-2 differentially interacts with and regulates members of the Tweety family of chloride ion channels. <span><span class="ref-journal">J Biol Chem. </span>2008;<span class="ref-vol">283</span>:24000–10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3259790/">PMC free article</a>]</span> [<a href="/pubmed/18577513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+ubiquitin-protein+ligase+Nedd4-2+differentially+interacts+with+and+regulates+members+of+the+Tweety+family+of+chloride+ion+channels&amp;author=Y+He&amp;author=DH+Hryciw&amp;author=ML+Carroll&amp;volume=283&amp;publication_year=2008&amp;pages=24000-10&amp;pmid=18577513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">Hallows KR, Bhalla V, Oyster NM, et al.  Phosphopeptide screen uncovers novel phosphorylation sites of Nedd4-2 that potentiate its inhibition of the epithelial Na+ channel. <span><span class="ref-journal">J Biol Chem. </span>2010;<span class="ref-vol">285</span>:21671–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2898378/">PMC free article</a>]</span> [<a href="/pubmed/20466724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Phosphopeptide+screen+uncovers+novel+phosphorylation+sites+of+Nedd4-2+that+potentiate+its+inhibition+of+the+epithelial+Na++channel&amp;author=KR+Hallows&amp;author=V+Bhalla&amp;author=NM+Oyster&amp;volume=285&amp;publication_year=2010&amp;pages=21671-8&amp;pmid=20466724&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="element-citation">Luo F, Wang Y, Wang X, Sun K, Zhou X, Hui R. A functional variant of NEDD4L is associated with hypertension, antihypertensive response, and orthostatic hypotension. <span><span class="ref-journal">Hypertension. </span>2009;<span class="ref-vol">54</span>:796–801.</span> [<a href="/pubmed/19635985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=A+functional+variant+of+NEDD4L+is+associated+with+hypertension,+antihypertensive+response,+and+orthostatic+hypotension&amp;author=F+Luo&amp;author=Y+Wang&amp;author=X+Wang&amp;author=K+Sun&amp;author=X+Zhou&amp;volume=54&amp;publication_year=2009&amp;pages=796-801&amp;pmid=19635985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="element-citation">Vestal DJ. The guanylate-binding proteins (GBPs): proinflammatory cytokine-induced members of the dynamin superfamily with unique GTPase activity. <span><span class="ref-journal">J Interferon Cytokine Res. </span>2005;<span class="ref-vol">25</span>:435–43.</span> [<a href="/pubmed/16108726" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Interferon+Cytokine+Res&amp;title=The+guanylate-binding+proteins+(GBPs):+proinflammatory+cytokine-induced+members+of+the+dynamin+superfamily+with+unique+GTPase+activity&amp;author=DJ+Vestal&amp;volume=25&amp;publication_year=2005&amp;pages=435-43&amp;pmid=16108726&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="element-citation">Kreienkamp HJ, Zitzer H, Gundelfinger ED, Richter D, Bockers TM. The calcium-independent receptor for alpha-latrotoxin from human and rodent brains interacts with members of the ProSAP/SSTRIP/Shank family of multidomain proteins. <span><span class="ref-journal">J Biol Chem. </span>2000;<span class="ref-vol">275</span>:32387–90.</span> [<a href="/pubmed/10964907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+calcium-independent+receptor+for+alpha-latrotoxin+from+human+and+rodent+brains+interacts+with+members+of+the+ProSAP/SSTRIP/Shank+family+of+multidomain+proteins&amp;author=HJ+Kreienkamp&amp;author=H+Zitzer&amp;author=ED+Gundelfinger&amp;author=D+Richter&amp;author=TM+Bockers&amp;volume=275&amp;publication_year=2000&amp;pages=32387-90&amp;pmid=10964907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="element-citation">White GR, Varley JM, Heighway J. Isolation and characterization of a human homologue of the latrophilin gene from a region of 1p31.1 implicated in breast cancer. <span><span class="ref-journal">Oncogene. </span>1998;<span class="ref-vol">17</span>:3513–9.</span> [<a href="/pubmed/10030676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Isolation+and+characterization+of+a+human+homologue+of+the+latrophilin+gene+from+a+region+of+1p31.1+implicated+in+breast+cancer&amp;author=GR+White&amp;author=JM+Varley&amp;author=J+Heighway&amp;volume=17&amp;publication_year=1998&amp;pages=3513-9&amp;pmid=10030676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="element-citation">Allgayer H. Pdcd4, a colon cancer prognostic that is regulated by a microRNA. <span><span class="ref-journal">Crit Rev Oncol Hematol. </span><span class="ref-vol">73</span>:185–91.</span> [<a href="/pubmed/19836969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit+Rev+Oncol+Hematol&amp;title=Pdcd4,+a+colon+cancer+prognostic+that+is+regulated+by+a+microRNA&amp;author=H+Allgayer&amp;volume=73&amp;pages=185-91&amp;pmid=19836969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="element-citation">Wei NA, Liu SS, Leung TH, et al.  Loss of programmed cell death 4 (Pdcd4) associates with the progression of ovarian cancer. <span><span class="ref-journal">Mol Cancer. </span>2009;<span class="ref-vol">8</span>:70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745358/">PMC free article</a>]</span> [<a href="/pubmed/19728867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cancer&amp;title=Loss+of+programmed+cell+death+4+(Pdcd4)+associates+with+the+progression+of+ovarian+cancer&amp;author=NA+Wei&amp;author=SS+Liu&amp;author=TH+Leung&amp;volume=8&amp;publication_year=2009&amp;pages=70&amp;pmid=19728867&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="element-citation">Wade AG, Ford I, Crawford G, et al.  Efficacy of prolonged release melatonin in insomnia patients aged 55-80 years: quality of sleep and next-day alertness outcomes. <span><span class="ref-journal">Curr Med Res Opin. </span>2007;<span class="ref-vol">23</span>:2597–605.</span> [<a href="/pubmed/17875243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Med+Res+Opin&amp;title=Efficacy+of+prolonged+release+melatonin+in+insomnia+patients+aged+55-80+years:+quality+of+sleep+and+next-day+alertness+outcomes&amp;author=AG+Wade&amp;author=I+Ford&amp;author=G+Crawford&amp;volume=23&amp;publication_year=2007&amp;pages=2597-605&amp;pmid=17875243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="element-citation">Bouatia-Naji N, Bonnefond A, Cavalcanti-Proenca C, et al.  A variant near MTNR1B is associated with increased fasting plasma glucose levels and type 2 diabetes risk. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:89–94.</span> [<a href="/pubmed/19060909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+variant+near+MTNR1B+is+associated+with+increased+fasting+plasma+glucose+levels+and+type+2+diabetes+risk&amp;author=N+Bouatia-Naji&amp;author=A+Bonnefond&amp;author=C+Cavalcanti-Proenca&amp;volume=41&amp;publication_year=2009&amp;pages=89-94&amp;pmid=19060909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="element-citation">Ronn T, Wen J, Yang Z, et al.  A common variant in MTNR1B, encoding melatonin receptor 1B, is associated with type 2 diabetes and fasting plasma glucose in Han Chinese individuals. <span><span class="ref-journal">Diabetologia. </span>2009;<span class="ref-vol">52</span>:830–3.</span> [<a href="/pubmed/19241057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=A+common+variant+in+MTNR1B,+encoding+melatonin+receptor+1B,+is+associated+with+type+2+diabetes+and+fasting+plasma+glucose+in+Han+Chinese+individuals&amp;author=T+Ronn&amp;author=J+Wen&amp;author=Z+Yang&amp;volume=52&amp;publication_year=2009&amp;pages=830-3&amp;pmid=19241057&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369232&amp;issue-id=210807&amp;journal-id=369&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140523455537344title">Abstract</a></li><li><a href="#__sec8title">INTRODUCTION</a></li><li><a href="#__sec9title">METHODS</a></li><li><a href="#__sec18title">RESULTS</a></li><li><a href="#__sec22title">DISCUSSION</a></li><li><a href="#__sec23title">DISCLOSURE STATEMENT</a></li><li><a href="#idm140523449541120title">ACKNOWLEDGMENTS</a></li><li><a href="#idm140523367689792title">Footnotes</a></li><li><a href="#idm140523449540240title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3369232/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3369232/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3369232/pdf/aasm.35.7.967.pdf">PDF (844K)</a></li> | <li><a href="#" data-citationid="PMC3369232" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369232%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369232%2F&amp;text=A%20Genome-Wide%20Association%20Study%20of%20Caffeine-Related%20Sleep%20Disturbance%3A%20Confirmation%20of%20a%20Role%20for%20a%20Common%20Variant%20in%20the%20Adenosine%20Receptor" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369232%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22754043" data-aiid="3369232" data-aid="3369232" data-iid="210807" data-domainid="369" data-domain="sleep" data-accid="PMC3369232" data-md5="378a2b286ce74df3c11c2cb2112aa560"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:13:05-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CB806DD53BCF10000000009D6069B&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369232%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=sleep&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3369232/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>