<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3818640/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818640/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Diabetologia"><meta name="citation_title" content="Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas"><meta name="citation_authors" content="E. J. Parra, J. E. Below, S. Krithika, A. Valladares, J. L. Barta, N. J. Cox, C. L. Hanis, N. Wacher, J. Garcia-Mena, P. Hu, M. D. Shriver, Diabetes Genetics Replication and Meta-analysis (DIAGRAM) Consortium, J. Kumate, P. M. McKeigue, J. Escobedo, M. Cruz"><meta name="citation_date" content="August 2011"><meta name="citation_issue" content="8"><meta name="citation_volume" content="54"><meta name="citation_doi" content="10.1007/s00125-011-2172-y"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3818640/?report=abstract"><meta name="citation_pmid" content="21573907"><meta name="DC.Title" content="Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="E. J. Parra"><meta name="DC.Contributor" content="J. E. Below"><meta name="DC.Contributor" content="S. Krithika"><meta name="DC.Contributor" content="A. Valladares"><meta name="DC.Contributor" content="J. L. Barta"><meta name="DC.Contributor" content="N. J. Cox"><meta name="DC.Contributor" content="C. L. Hanis"><meta name="DC.Contributor" content="N. Wacher"><meta name="DC.Contributor" content="J. Garcia-Mena"><meta name="DC.Contributor" content="P. Hu"><meta name="DC.Contributor" content="M. D. Shriver"><meta name="DC.Contributor" content="Diabetes Genetics Replication and Meta-analysis (DIAGRAM) Consortium"><meta name="DC.Contributor" content="J. Kumate"><meta name="DC.Contributor" content="P. M. McKeigue"><meta name="DC.Contributor" content="J. Escobedo"><meta name="DC.Contributor" content="M. Cruz"><meta name="DC.Date" content="2011 Aug"><meta name="DC.Identifier" content="10.1007/s00125-011-2172-y"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas"><meta property="og:type" content="article"><meta property="og:description" content="We report a genome-wide association study of type 2 diabetes in an admixed sample from Mexico City and describe the results of a meta-analysis of this study and another genome-wide scan in a Mexican-American sample from Starr County, TX, USA. The top ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818640/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3818640/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3818640/pdf/nihms-472351.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89ED29DD5238410000000005D703BD.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3818640%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="protein" class="last">Protein</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3818640</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Diabetologia</a></span>. Author manuscript; available in PMC 2013 Nov 6.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21573907" target="pmc_ext" ref="reftype=publisher&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Diabetologia. 2011 Aug; 54(8): 10.1007/s00125-011-2172-y. </span></a></div><span class="fm-vol-iss-date">Published online 2011 May 15. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00125-011-2172-y" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00125-011-2172-y</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3818640</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS472351</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21573907">21573907</a></div></div></div></div><h1 class="content-title">Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Parra%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892015872" co-class="co-affbox">E. J. Parra</a>, <a href="/pubmed/?term=Below%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055922290144" co-class="co-affbox">J. E. Below</a>, <a href="/pubmed/?term=Krithika%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055931178112" co-class="co-affbox">S. Krithika</a>, <a href="/pubmed/?term=Valladares%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892014112" co-class="co-affbox">A. Valladares</a>, <a href="/pubmed/?term=Barta%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892012176" co-class="co-affbox">J. L. Barta</a>, <a href="/pubmed/?term=Cox%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892010288" co-class="co-affbox">N. J. Cox</a>, <a href="/pubmed/?term=Hanis%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892008432" co-class="co-affbox">C. L. Hanis</a>, <a href="/pubmed/?term=Wacher%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892006544" co-class="co-affbox">N. Wacher</a>, <a href="/pubmed/?term=Garcia-Mena%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892274048" co-class="co-affbox">J. Garcia-Mena</a>, <a href="/pubmed/?term=Hu%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892272304" co-class="co-affbox">P. Hu</a>, <a href="/pubmed/?term=Shriver%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892270560" co-class="co-affbox">M. D. Shriver</a>, Diabetes Genetics Replication and Meta-analysis (DIAGRAM) Consortium, <a href="/pubmed/?term=Kumate%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892268192" co-class="co-affbox">J. Kumate</a>, <a href="/pubmed/?term=McKeigue%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892266448" co-class="co-affbox">P. M. McKeigue</a>, <a href="/pubmed/?term=Escobedo%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892264704" co-class="co-affbox">J. Escobedo</a>, and  <a href="/pubmed/?term=Cruz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907" class="affpopup" co-rid="_co_idm140055892262816" co-class="co-affbox">M. Cruz</a></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140055892015872"><h3 class="no_margin">E. J. Parra</h3><p>Department of Anthropology, University of Toronto at Mississauga, 3359 Mississauga Road North, Mississauga, ON, Canada L5L 1C6</p><div>Find articles by <a href="/pubmed/?term=Parra%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">E. J. Parra</a></div></div><div id="_co_idm140055922290144"><h3 class="no_margin">J. E. Below</h3><p>Department of Medicine, Section of Genetic Medicine, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Below%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">J. E. Below</a></div></div><div id="_co_idm140055931178112"><h3 class="no_margin">S. Krithika</h3><p>Department of Anthropology, University of Toronto at Mississauga, 3359 Mississauga Road North, Mississauga, ON, Canada L5L 1C6</p><div>Find articles by <a href="/pubmed/?term=Krithika%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">S. Krithika</a></div></div><div id="_co_idm140055892014112"><h3 class="no_margin">A. Valladares</h3><p>Unidad de Investigacion Medica en Bioquimica, Hospital de Especialidades, Centro Medico Nacional Siglo XXI, IMSS, Av. Cuauhtemoc 330, Col. Doctores, C.P. 06720, Mexico City, Mexico</p><div>Find articles by <a href="/pubmed/?term=Valladares%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">A. Valladares</a></div></div><div id="_co_idm140055892012176"><h3 class="no_margin">J. L. Barta</h3><p>Department of Anthropology, University of Toronto at Mississauga, 3359 Mississauga Road North, Mississauga, ON, Canada L5L 1C6</p><div>Find articles by <a href="/pubmed/?term=Barta%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">J. L. Barta</a></div></div><div id="_co_idm140055892010288"><h3 class="no_margin">N. J. Cox</h3><p>Department of Medicine, Section of Genetic Medicine, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Cox%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">N. J. Cox</a></div></div><div id="_co_idm140055892008432"><h3 class="no_margin">C. L. Hanis</h3><p>Human Genetics Center, School of Public Health, The University of Texas, Health Science Center at Houston, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Hanis%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">C. L. Hanis</a></div></div><div id="_co_idm140055892006544"><h3 class="no_margin">N. Wacher</h3><p>Unidad de Investigacion Medica en Epidemiologia Clinica, Hospital de Especialidades, Centro Medico Siglo XXI, IMSS, Mexico City, Mexico</p><div>Find articles by <a href="/pubmed/?term=Wacher%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">N. Wacher</a></div></div><div id="_co_idm140055892274048"><h3 class="no_margin">J. Garcia-Mena</h3><p>Departamento de Genetica y Biologia Molecular, Cinvestav-IPN, Mexico City, Mexico</p><div>Find articles by <a href="/pubmed/?term=Garcia-Mena%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">J. Garcia-Mena</a></div></div><div id="_co_idm140055892272304"><h3 class="no_margin">P. Hu</h3><p>The Centre for Applied Genomics, The Hospital for Sick Children, Toronto, ON, Canada</p><div>Find articles by <a href="/pubmed/?term=Hu%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">P. Hu</a></div></div><div id="_co_idm140055892270560"><h3 class="no_margin">M. D. Shriver</h3><p>Department of Anthropology, Penn State University, University Park, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Shriver%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">M. D. Shriver</a></div></div><div id="_co_idm140055892268192"><h3 class="no_margin">J. Kumate</h3><p>Fundacion IMSS, Mexico City, Mexico</p><div>Find articles by <a href="/pubmed/?term=Kumate%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">J. Kumate</a></div></div><div id="_co_idm140055892266448"><h3 class="no_margin">P. M. McKeigue</h3><p>Public Health Sciences, University of Edinburgh Medical School, Edinburgh, Scotland, UK</p><div>Find articles by <a href="/pubmed/?term=McKeigue%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">P. M. McKeigue</a></div></div><div id="_co_idm140055892264704"><h3 class="no_margin">J. Escobedo</h3><p>Unidad de Investigacion en Epidemiologia Clinica, Hospital General Regional 1, Dr Carlos McGregor, IMSS, Mexico City, Mexico</p><div>Find articles by <a href="/pubmed/?term=Escobedo%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">J. Escobedo</a></div></div><div id="_co_idm140055892262816"><h3 class="no_margin">M. Cruz</h3><p>Unidad de Investigacion Medica en Bioquimica, Hospital de Especialidades, Centro Medico Nacional Siglo XXI, IMSS, Av. Cuauhtemoc 330, Col. Doctores, C.P. 06720, Mexico City, Mexico</p><div>Find articles by <a href="/pubmed/?term=Cruz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21573907">M. Cruz</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140055925614192_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140055925614192_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140055925614192_ai" style="display:none"><div><span class="fm-affl">E. J. Parra, </span><span class="fm-affl" id="A1" lang="en"> Department of Anthropology, University of Toronto at Mississauga, 3359 Mississauga Road North, Mississauga, ON, Canada L5L 1C6;</span></div><div><a href="#idm140055925614192aff-info">Contributor Information</a>.</div><div id="CR1"> E. J. Parra <a href="mailto:dev@null" data-email="ac.otnorotu@arrap.nabetse" class="oemail">ac.otnorotu@arrap.nabetse</a>; M. Cruz <a href="mailto:dev@null" data-email="moc.oohay@lzurcm" class="oemail">moc.oohay@lzurcm</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140055925614192_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21573907" target="pmc_ext" ref="reftype=publisher&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Diabetologia</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3818640/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="7" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>ESM1.</strong></div><div><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM1.pdf" data-ga-action="click_feat_suppl">NIHMS472351-supplement-ESM1.pdf</a><span style="color:gray"> (29K)</span></div><div class="small guid">GUID:&nbsp;CF51B8DA-4AEB-4959-A4BA-B2618D0CE090</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>ESM2.</strong></div><div><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM2.pdf" data-ga-action="click_feat_suppl">NIHMS472351-supplement-ESM2.pdf</a><span style="color:gray"> (29K)</span></div><div class="small guid">GUID:&nbsp;56E97F72-5346-4678-AEF6-71719A41220F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>ESM3.</strong></div><div><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM3.pdf" data-ga-action="click_feat_suppl">NIHMS472351-supplement-ESM3.pdf</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;42088CEC-AF06-48C7-B3B1-BC355145C04B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>ESM4.</strong></div><div><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM4.pdf" data-ga-action="click_feat_suppl">NIHMS472351-supplement-ESM4.pdf</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;37B6BC1C-7237-4F97-816E-45C3CFFF41BF</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>ESM5.</strong></div><div><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM5.pdf" data-ga-action="click_feat_suppl">NIHMS472351-supplement-ESM5.pdf</a><span style="color:gray"> (33K)</span></div><div class="small guid">GUID:&nbsp;4BE9B59A-8D45-4562-B3E2-8175635A0DE1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>ESM6.</strong></div><div><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM6.pdf" data-ga-action="click_feat_suppl">NIHMS472351-supplement-ESM6.pdf</a><span style="color:gray"> (35K)</span></div><div class="small guid">GUID:&nbsp;DB28B965-FCD2-4E3F-8AEE-36C7F43B7AB6</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>ESM7.</strong></div><div><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM7.pdf" data-ga-action="click_feat_suppl">NIHMS472351-supplement-ESM7.pdf</a><span style="color:gray"> (37K)</span></div><div class="small guid">GUID:&nbsp;C85954AF-EFCC-4B81-B77E-5F3DDD5EF4BC</div></div></dd></dl></div><div id="idm140055922896192" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140055922896192title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Aims/hypothesis</h3><p id="P1" class="p p-first-last">We report a genome-wide association study of type 2 diabetes in an admixed sample from Mexico City and describe the results of a meta-analysis of this study and another genome-wide scan in a Mexican-American sample from Starr County, TX, USA. The top signals observed in this meta-analysis were followed up in the Diabetes Genetics Replication and Meta-analysis Consortium (DIAGRAM) and DIAGRAM+ datasets.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">We analysed 967 cases and 343 normoglycaemic controls. The samples were genotyped with the Affymetrix Genome-wide Human SNP array 5.0. Associations of genotyped and imputed markers with type 2 diabetes were tested using a missing data likelihood score test. A fixed-effects meta-analysis including 1,804 cases and 780 normoglycaemic controls was carried out by weighting the effect estimates by their inverse variances.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">In the meta-analysis of the two Hispanic studies, markers showing suggestive associations (<em>p</em>&lt;10<sup>−5</sup>) were identified in two known diabetes genes, <em>HNF1A</em> and <em>KCNQ1</em>, as well as in several additional regions. Meta-analysis of the two Hispanic studies and the recent DIAGRAM+ dataset identified genome-wide significant signals (<em>p</em>&lt;5×10<sup>−8</sup>) within or near the genes <em>HNF1A</em> and <em>CDKN2A/CDKN2B</em>, as well as suggestive associations in three additional regions, <em>IGF2BP2, KCNQ1</em> and the previously unreported <em>C14orf70</em>.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions/interpretation</h3><p id="P4" class="p p-first-last">We observed numerous regions with suggestive associations with type 2 diabetes. Some of these signals correspond to regions described in previous studies. However, many of these regions could not be replicated in the DIAGRAM datasets. It is critical to carry out additional studies in Hispanic and American Indian populations, which have a high prevalence of type 2 diabetes.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Genome-wide association, Hispanics, Meta-analysis, Type 2 diabetes</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first">Recent genome-wide association (GWA) studies have dramatically increased the number of known type 2 diabetes susceptibility loci. To date, approximately 40 loci have been confirmed using this approach [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_339327611">1</a>, <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_339327629">2</a>]. However, most have been identified in populations of European ancestry, and the number of GWA studies in non-European populations has been very limited [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_339327648">3</a>–<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_339327613">8</a>]. It is critical to expand this type of study to other population groups because GWA scans in European populations may fail to identify some type 2 diabetes susceptibility variants in some situations, e.g. when: (1) the low-risk or high-risk alleles are nearly fixed in Europeans; (2) linkage disequilibrium (LD) with nearby tag single nucleotide polymorphisms (SNPs) is too weak in Europeans for the effects of risk alleles to be detected; or (3) there is too much allelic heterogeneity for associations with any single tag SNP to be detected. The identification of susceptibility variants within the gene <em>KCNQ1</em> in two Japanese studies is an excellent example. These variants were not identified in any of the previous, larger GWA studies in European populations, because the high frequencies observed for the risk alleles in Europe resulted in a dramatic reduction in statistical power [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_339327615">9</a>]. Therefore, GWA studies in non-European populations are necessary in order to increase our understanding of susceptibility loci and aetiological pathways involved in type 2 diabetes.</p><p id="P6" class="p p-last">We report here the results of the first GWA study of type 2 diabetes in an admixed sample from Mexico. We also describe the results of a meta-analysis of our sample and another genome-wide scan in a Mexican-American sample from Starr County, Texas (see also Below et al. [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_339327618">10</a>] in this edition of Diabetologia). Finally, we followed up some of the signals observed in the meta-analysis of the two Hispanic studies using the Diabetes Genetics Replication and Meta-analysis Consortium (DIAGRAM) dataset.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Study participants</h3><p id="P7" class="p p-first-last">A total of 1,334 individuals participated in this study, 989 cases and 345 normoglycaemic controls. The cases were recruited in Mexico City, with selection based on type 2 diabetes age at diagnosis, preference being given to participants aged &lt;55 years at diagnosis. Diabetes diagnosis was done according to the American Diabetes Association criteria, i.e. fasting plasma glucose ≥7.0 mmol/l or OGTT ≥11.1 mmol/l for the 2 h sample. The controls were 345 healthy blood donors (175 men, 170 women) from Mexico City without a family history of type 2 diabetes and with glucose concentrations of &lt;7.8 mmol/l after an OGTT. After quality control, 24 participants (22 cases, two controls) were eliminated from the sample for different reasons (see below), so the final sample used in the statistical analysis comprised 967 cases (664 women, 303 men) and 343 controls (169 women, 174 men). The average age of diagnosis of type 2 diabetes in the final sample of cases was 43.7 years (range 20–60). The average age of the final sample of controls was 50.7 years (range 42–68). Data on BMI were available for cases and controls. Informed consent was obtained from each participant, and the research was approved by the ethical research boards of the Medical Center ‘Siglo XXI’ and the University of Toronto.</p></div><div id="S8" class="sec"><h3 id="S8title">Genotyping</h3><p id="P8" class="p p-first-last">Genotyping was carried out in the microarray analysis facility located in the Centre for Applied Genomics (Toronto, ON, Canada), using the Affymetrix Genome-wide Human SNP array 5.0 (Affymetrix, Santa Clara, CA, USA) and following standard protocols. Genotype calling was done with the Affymetrix Power Tools software package, including the full sample set and using two genotyping algorithms, the BRLMM-P and Birdseed algorithms (<a href="http://www.affymetrix.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.affymetrix.com</a>, accessed 1 December 2010). To minimise genotyping errors, the PLINK version 1.06 (<a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink/</a>, accessed 1 December 2010) [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_339327610">11</a>] was used to merge the genotype results obtained with both algorithms, using the consensus call mode. The concordance for autosomal SNPs between the genotype calls obtained with the BRLMM-P and the Birdseed algorithms was 99.71%. The missing genotype rate in the sample of cases was 0.59% and the missing genotype rate in the sample of controls was 0.79%. We genotyped six individuals (four cases, two controls) in duplicate and the concordance rate was 99.88%.</p></div><div id="S9" class="sec"><h3 id="S9title">Quality control</h3><p id="P9" class="p p-first-last">On the basis of a preliminary analysis using 430,321 autosomal markers, one sample was excluded from the dataset due to high missing genotype rates (&gt;5%) and 23 samples due to cryptic relatedness (<em>n</em>=20) or sample duplication (<em>n</em>=3). For pairs of individuals with identity by descent estimates &gt;0.2, we removed the sample with the highest proportion of missing genotypes. From the initial list of autosomal markers, we additionally removed SNPs based on the following criteria: (1) minor allele frequency &lt;5%; (2) Hardy–Weinberg <em>p</em> values &lt;0.0001 in the control group; and (3) missingness &gt;5% in the cases and the controls. After filtering, the final number of autosomal markers was 315,658. All the quality control procedures were completed using PLINK.</p></div><div id="S10" class="sec"><h3 id="S10title">Inferring individual admixture proportions</h3><p id="P10" class="p p-first-last">Individual admixture proportions were estimated with the program ADMIXMAP (<a href="http://homepages.ed.ac.uk/pmckeigu/admixmap/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://homepages.ed.ac.uk/pmckeigu/admixmap/index.html</a>, accessed 1 December 2010) using a genome-wide panel of 3,500 ancestry informative markers. This panel of ancestry informative markers was selected using genome-wide data available for European, West African and American Indian populations [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_339327645">12</a>].</p></div><div id="S11" class="sec"><h3 id="S11title">Population stratification</h3><p id="P11" class="p p-first-last">To evaluate population stratification in the sample from Mexico City (e.g. potential differences in ancestry proportions between cases and controls), we used EIGENSOFT (<a href="http://genepath.med.harvard.edu/~reich" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genepath.med.harvard.edu/~reich</a>, accessed 1 December 2010) to perform a principal component analysis of the Mexico City sample, after pruning markers showing short-range and long-range LD at a threshold of <em>r</em><sup>2</sup>=0.2. The first axis explains the majority of the variation in the sample (the eigenvalue for the first axis was 17.6 vs less than 2 for the remaining axes). The scores of the first axis show a very strong correlation with the American Indian admixture proportions estimated with ADMIXMAP (<em>R</em><sup>2</sup>=0.991) and were significantly associated with type 2 diabetes. For this reason, the scores of the first axis were incorporated as covariates in the association tests.</p></div><div id="S12" class="sec"><h3 id="S12title">Imputation</h3><p id="P12" class="p p-first-last">We imputed untyped genotypes with Impute version 2 (<a href="https://mathgen.stats.ox.ac.uk/impute/impute.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute.html</a>, accessed 1 December 2010), using the HapMap Phase 2 combined sample as a phased reference sample and the HapMap Phase 3 Mexican-American sample as an unphased reference sample. To evaluate the performance of the imputation, we randomly masked 5% of the markers (<em>n</em>=1,046) directly genotyped on chromosome 12 and compared the imputed genotypes with the Affymetrix genome-wide Human SNP array 5.0 genotype calls. For this analysis, imputed genotypes with probabilities &lt;0.9 were set as missing data. Of the imputed genotypes, 85.7% had probabilities higher than 0.9 and the concordance rate observed between the Affymetrix 5.0 genotypes and the imputed genotypes was 99.4%. In an independent analysis, we masked 100 markers on chromosomes 4, 9, 16 and 20 from the inference files used for imputation and compared the imputed genotypes with the original genotypes obtained with the Affymetrix 5.0 microchip. This analysis resulted in 74.4% of imputed genotypes with probabilities higher than 0.9 and a concordance rate of 99.0%.</p></div><div id="S13" class="sec"><h3 id="S13title">Association tests</h3><p id="P13" class="p p-first-last">Associations of the genetic markers with type 2 diabetes were tested with SNPTEST version 2 (<a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a>, accessed 1 December 2010) using the frequentist association tests implemented in the program, based on an additive model. To control for genotype uncertainty, we used the missing data likelihood score test (the score method). The statistical model incorporated sex, age and ancestry, and alternatively, sex, age, ancestry and BMI. In this case, ancestry corresponds to the coordinates of the first principal component of the Mexico City sample (see above, Population stratification).</p></div><div id="S14" class="sec"><h3 id="S14title">Meta-analysis</h3><p id="P14" class="p p-first-last">We used META (<a href="http://www.stats.ox.ac.uk/~jsliu/meta.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/~jsliu/meta.html</a>, accessed 1 December, 2010) to carry out an inverse variance fixed-effects meta-analysis of the Mexico City and the Starr County studies (for more details about the Starr County GWAs, see Below et al. [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918924">10</a>] in this edition of Diabetologia). GWAMA (<a href="http://www.well.ox.ac.uk/gwama" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.well.ox.ac.uk/gwama</a>, accessed 1 December, 2010) was used to carry out an inverse variance fixed-effects meta-analysis of the top signals observed in the Hispanic studies and the 2008 DIAGRAM and DIAGRAM+ datasets.</p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Statistical power analysis</h3><p id="P15" class="p p-first-last">A figure depicting the statistical power of the meta-analysis is available in the companion paper by Below et al. [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918927">10</a>].</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Results</h2><div id="S17" class="sec sec-first"><h3 id="S17title">Exploring admixture proportions and population stratification</h3><p id="P16" class="p p-first-last">Analysis with ADMIXMAP indicated that the sample from Mexico City is an admixed sample with substantial American Indian and European contributions, and very limited West African influence. The average American Indian ancestry in the sample was estimated to be 63%, the average European ancestry 34% and the average West African ancestry 3%. <a href="/pmc/articles/PMC3818640/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> shows the individual admixture proportions in the Mexico City sample, as well as the average admixture proportions for the case and control groups. There was considerable variation in admixture proportions in the American Indian–European axis. Additionally, a substantial difference in American Indian admixture proportions was observed in the case and control groups (66% vs 57%, respectively). The effect of population stratification (difference in admixture proportions between cases and controls) is clearly evident in the Quantile–Quantile (QQ) plot comparing <em>p</em> values of the score test conditioning on sex and age with the values expected under the null model of no association (<a href="#SD1" rid="SD1" class=" supplementary-material">electronic supplementary material [ESM] Fig. 1</a>). Under this model, there was a strong departure of the observed and expected <em>p</em> values. In contrast, the score test conditioning on sex, age and ancestry produced a QQ plot with a good fit between observed and expected <em>p</em> values (<a href="#SD2" rid="SD2" class=" supplementary-material">ESM Fig. 2</a>). Therefore, inclusion of ancestry in the model corrected for the effects of stratification in this sample.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3818640/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140055892715600" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-472351-f0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-472351-f0001.jpg" src="/pmc/articles/PMC3818640/bin/nihms-472351-f0001.jpg"></div></a><div id="largeobj_idm140055892715600" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3818640/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3818640/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3818640/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p7">Triangular representation of admixture proportions in the sample from Mexico City, estimated with ADMIXMAP. Individual admixture proportions are indicated by squares. Star, average admixture proportions in the case group; diamond, average admixture proportions in the control group</p></div></div></div></div><div id="S18" class="sec"><h3 id="S18title">Statistical analysis, Affymetrix 5.0 and imputed markers</h3><p id="P17" class="p p-first-last">The statistical analysis was carried out for approximately 2 million common markers (~300,000 directly genotyped and ~1.7 million imputed markers), using the missing likelihood score test implemented in SNPTEST version 2. The Manhattan plots summarising the results of the analyses and controlling for sex, age and ancestry, and sex, age, ancestry and BMI are presented as <a href="#SD3" rid="SD3" class=" supplementary-material">ESM Figs 3</a> and <a href="#SD4" rid="SD4" class=" supplementary-material">4</a>. <a href="#SD5" rid="SD5" class=" supplementary-material">ESM Tables 1</a> and <a href="#SD6" rid="SD6" class=" supplementary-material">2</a> show the top signals (<em>p</em>&lt;10<sup>−5</sup>) observed in the analysis with sex, age and ancestry, and sex, age, ancestry and BMI as covariates, respectively, annotated using WGA Viewer [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918923">13</a>].</p></div><div id="S19" class="sec"><h3 id="S19title">Meta-analysis of two type 2 diabetes GWA studies in Hispanics</h3><p id="P18" class="p p-first-last">We performed a fixed-effects meta-analysis of two GWA studies carried out in individuals of Mexican ancestry, i.e. our Mexico City study (967 type 2 diabetes cases, 343 normoglycaemic controls) and a study in Starr County, Texas, USA (837 cases, 437 normoglycaemic controls). <a href="/pmc/articles/PMC3818640/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> lists the markers showing suggestive associations (<em>p</em>&lt;10<sup>−5</sup>) in the meta-analysis without and with BMI as a covariate. Only markers that were nominally significant (<em>p</em>&lt;0.05) in both studies are included, with only markers with the lowest <em>p</em> values in each region being reported. The meta-analysis was carried out with META, setting the information content exclusion threshold for the imputed markers (score information) at 0.5. The lowest <em>p</em> values were observed for two variants within and near the gene HNF1A on chromosome 12 (<em>p</em>=3.5×10<sup>−7</sup> lowest <em>p</em> value). Eight additional regions with SNPs showing values of <em>p</em>&lt;10<sup>−5</sup> were identified in the meta-analysis, corresponding to the genes <em>RP11-354K1.1</em> on chromosome 1, <em>LSAMP</em> on chromosome 3, <em>FGF12</em> on chromosome 3, <em>CSMD1</em> on chromosome 8, <em>NIPAL2</em> on chromosome 8, <em>CIT</em> on chromosome 12 and <em>DEPDC5</em> on chromosome 22, as well as an expressed sequence tag located on chromosome 9. In the analysis incorporating BMI as a covariate, the markers located within or near the genes <em>LSAMP</em> and <em>NIPAL2</em> were not among the top signals, and there were several new regions with values of <em>p</em>&lt;10<sup>−5</sup> in this analysis, including the genes <em>EPHB2</em> on chromosome 1, <em>ANK2</em> on chromosome 4, <em>MCPH1</em> on chromosome 8 and <em>KCNQ1</em> on chromosome 11.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p8">Top signals observed in the meta-analysis of the Mexico City and Starr County studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140055892713024" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em> value</th><th rowspan="1" colspan="1" valign="middle" align="left">Chr</th><th rowspan="1" colspan="1" valign="middle" align="left">Pos</th><th rowspan="1" colspan="1" valign="middle" align="left">Type</th><th rowspan="1" colspan="1" valign="middle" align="left">Closest gene</th><th rowspan="1" colspan="1" valign="middle" align="center">NEA</th><th rowspan="1" colspan="1" valign="middle" align="center">EA</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">OR_95L</th><th rowspan="1" colspan="1" valign="middle" align="center">OR_95U</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em>-SC</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em>-MC</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">Without BMI</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7516189<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">7.5×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">211828134</td><td rowspan="1" colspan="1" valign="middle" align="left">Upstream</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>RP11-354K1.1</em><sup>d</sup>,<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.46</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">1.72</td><td rowspan="1" colspan="1" valign="middle" align="center">1.9×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">7.6×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs938911<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">8.6×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">3</td><td rowspan="1" colspan="1" valign="middle" align="left">116420202</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LSAMP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">2.05</td><td rowspan="1" colspan="1" valign="middle" align="center">1.50</td><td rowspan="1" colspan="1" valign="middle" align="center">2.82</td><td rowspan="1" colspan="1" valign="middle" align="center">7.9×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">3.1×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs9831045<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.4×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">3</td><td rowspan="1" colspan="1" valign="middle" align="left">192475259</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>FGF12</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.41</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22</td><td rowspan="1" colspan="1" valign="middle" align="center">1.62</td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.9×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs9773092<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">9.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="left">3216025</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CSMD1</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.42</td><td rowspan="1" colspan="1" valign="middle" align="center">1.23</td><td rowspan="1" colspan="1" valign="middle" align="center">1.63</td><td rowspan="1" colspan="1" valign="middle" align="center">2.6×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">7.2×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs896416<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">7.7×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="left">99199066</td><td rowspan="1" colspan="1" valign="middle" align="left">Downstream</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NIPAL2</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.37</td><td rowspan="1" colspan="1" valign="middle" align="center">1.19</td><td rowspan="1" colspan="1" valign="middle" align="center">1.57</td><td rowspan="1" colspan="1" valign="middle" align="center">4.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">3.6×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10809674<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">5.6×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="left">12015741</td><td rowspan="1" colspan="1" valign="middle" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>RP11-74C3.1</em>
<sup>d</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.47</td><td rowspan="1" colspan="1" valign="middle" align="center">1.25</td><td rowspan="1" colspan="1" valign="middle" align="center">1.74</td><td rowspan="1" colspan="1" valign="middle" align="center">5.9×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.6×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs202983<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">8.9×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="left">120162346</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CIT</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.52</td><td rowspan="1" colspan="1" valign="middle" align="center">1.26</td><td rowspan="1" colspan="1" valign="middle" align="center">1.82</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.4×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7305618<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.8×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="left">121402932</td><td rowspan="1" colspan="1" valign="middle" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>HNF1A</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.69</td><td rowspan="1" colspan="1" valign="middle" align="center">1.38</td><td rowspan="1" colspan="1" valign="middle" align="center">2.06</td><td rowspan="1" colspan="1" valign="middle" align="center">7.5×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">9.9×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs5998144<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">4.5×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">22</td><td rowspan="1" colspan="1" valign="middle" align="left">32248244</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>DEPDC5</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.56</td><td rowspan="1" colspan="1" valign="middle" align="center">1.29</td><td rowspan="1" colspan="1" valign="middle" align="center">1.89</td><td rowspan="1" colspan="1" valign="middle" align="center">3.2×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.6×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">With BMI</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10465543<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">9.40×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">23134218</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>EPHB2</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.52</td><td rowspan="1" colspan="1" valign="middle" align="center">1.26</td><td rowspan="1" colspan="1" valign="middle" align="center">1.83</td><td rowspan="1" colspan="1" valign="middle" align="center">1.2×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.4×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7516189<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">5.73×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">211828134</td><td rowspan="1" colspan="1" valign="middle" align="left">Upstream</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>RP11-354K1.1</em><sup>e</sup>,<sup>d</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.48</td><td rowspan="1" colspan="1" valign="middle" align="center">1.25</td><td rowspan="1" colspan="1" valign="middle" align="center">1.75</td><td rowspan="1" colspan="1" valign="middle" align="center">1.1×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.2×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs9831045<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">9.16×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">3</td><td rowspan="1" colspan="1" valign="middle" align="left">192475259</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>FGF12</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.39</td><td rowspan="1" colspan="1" valign="middle" align="center">1.20</td><td rowspan="1" colspan="1" valign="middle" align="center">1.61</td><td rowspan="1" colspan="1" valign="middle" align="center">7.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.9×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs413019<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">6.71×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">4</td><td rowspan="1" colspan="1" valign="middle" align="left">114079127</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>ANK2</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.41</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21</td><td rowspan="1" colspan="1" valign="middle" align="center">1.64</td><td rowspan="1" colspan="1" valign="middle" align="center">2.6×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">7.9×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs9773092<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">6.11×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="left">3216025</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CSMD1</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.43</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">1.65</td><td rowspan="1" colspan="1" valign="middle" align="center">2.1×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">5.7×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4841333<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">3.70×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="left">6453786</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>MCPH1</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.50</td><td rowspan="1" colspan="1" valign="middle" align="center">1.26</td><td rowspan="1" colspan="1" valign="middle" align="center">1.78</td><td rowspan="1" colspan="1" valign="middle" align="center">3.2×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.7×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10809674<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.93×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="left">12015741</td><td rowspan="1" colspan="1" valign="middle" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>RP11-74C3.1</em><sup>e</sup>,<sup>d</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.52</td><td rowspan="1" colspan="1" valign="middle" align="center">1.28</td><td rowspan="1" colspan="1" valign="middle" align="center">1.80</td><td rowspan="1" colspan="1" valign="middle" align="center">2.7×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.9×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2237892<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">6.75×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="left">2839751</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>KCNQ1</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.47</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">1.74</td><td rowspan="1" colspan="1" valign="middle" align="center">5.0×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">3.4×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs202983<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.71×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="left">120162346</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CIT</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.57</td><td rowspan="1" colspan="1" valign="middle" align="center">1.30</td><td rowspan="1" colspan="1" valign="middle" align="center">1.90</td><td rowspan="1" colspan="1" valign="middle" align="center">8.3×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">8.6×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7305618<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">3.91×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="left">121402932</td><td rowspan="1" colspan="1" valign="middle" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>HNF1A</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.69</td><td rowspan="1" colspan="1" valign="middle" align="center">1.38</td><td rowspan="1" colspan="1" valign="middle" align="center">2.08</td><td rowspan="1" colspan="1" valign="middle" align="center">7.0×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.6×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs5998144<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.68×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">22</td><td rowspan="1" colspan="1" valign="middle" align="left">32248244</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>DEPDC5</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.61</td><td rowspan="1" colspan="1" valign="middle" align="center">1.32</td><td rowspan="1" colspan="1" valign="middle" align="center">1.96</td><td rowspan="1" colspan="1" valign="middle" align="center">2.1×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">9.3×10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140055892713024" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3818640/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P32" class="p p-first-last">The <em>p</em> values of the meta-analysis are based on the results of the Mexico City and Starr County score tests; none of the markers showed evidence of heterogeneity of effects between studies (Cochran’s Q, <em>p</em>&gt;0.1)</p></div><div id="TFN2"><p id="P33" class="p p-first-last">All imputed markers had score information &gt;0.8, except rs938911 and rs7305618, which had score information &gt;0.7</p></div><div id="TFN3"><sup>a</sup>Directly genotyped in SC and MC samples;</div><div id="TFN4"><sup>b</sup>directly genotyped in SC and imputed in MC;</div><div id="TFN5"><sup>c</sup>imputed in SC and MC samples;</div><div id="TFN6"><sup>d</sup>ENCODE Genecode gene annotations;</div><div id="TFN7"><sup>e</sup>More than one marker in this region showed a <em>p</em> value of <em>p</em>&lt;10<sup>−5</sup>; only the marker with the lowest <em>p</em> value is shown Chr Chromosome; Pos Position; NEA Non-effect allele; EA Effect allele; OR 95L Lower 95% confidence limit of odds ratio; OR 95U Upper 95% confidence limit of odds ratio; <em>p</em>-MC <em>p</em> value in Mexico City; <em>p</em>-SC <em>p</em> value in Starr County</div></div></div></div><div id="S20" class="sec sec-last"><h3 id="S20title">Follow-up of the top signals of the Hispanic meta-analysis in the DIAGRAM dataset</h3><p id="P19" class="p p-first">The signals observed in the samples of Mexican ancestry were further explored by meta-analysis using the 2008 DIAGRAM consortium data-set [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918934">14</a>], which included three GWA studies in individuals of European ancestry (Wellcome Trust Case–Control Consortium [WTCCC], Diabetes Genetics Initiative [DGI] and the Finland–United States Investigation of NIDDM Genetics [FUSION] studies). <a href="/pmc/articles/PMC3818640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> describes the markers with global values of <em>p</em>&lt;10<sup>−4</sup> that show the same direction of effect in the Hispanic and European studies. The list includes markers in five regions that have been identified in previous GWA studies: <em>IGF2BP2</em> on chromosome 3 (lowest <em>p</em> value <em>p</em>=9.5×10<sup>−6</sup>), <em>PTPRD</em> on chromosome 9 (lowest <em>p</em> value <em>p</em>=4.4×10<sup>−7</sup>), a region near <em>CDKN2A/CDKN2B</em> on chromosome 9 (lowest <em>p</em> value <em>p</em>=9.6×10<sup>−10</sup>), <em>KCNQ1</em> on chromosome 11 (lowest <em>p</em> value <em>p</em>=5.6×10<sup>−6</sup>) and <em>HNF1A</em> on chromosome 12 (<em>p</em>=4.4×10<sup>−5</sup>). There was also one SNP in a region not previously identified in GWA studies, <em>C14orf70</em> on chromosome 14 (<em>p</em>=4.0×10<sup>−6</sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p9">List of markers with combined <em>p</em> values of <em>p</em>&lt;10<sup>−4</sup> in the Hispanic and 2008 DIAGRAM dataset</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140055965847248" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="left">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">NEA</th><th rowspan="1" colspan="1" valign="middle" align="center">EA</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em>-META</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">OR 95L</th><th rowspan="1" colspan="1" valign="middle" align="center">OR 95U</th><th rowspan="1" colspan="1" valign="middle" align="left">Type</th><th rowspan="1" colspan="1" valign="middle" align="left">Closest gene</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em>-DIAG</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em>-Hisp</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1374910</td><td rowspan="1" colspan="1" valign="middle" align="left">3</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">9.5×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.30</td><td rowspan="1" colspan="1" valign="middle" align="center">1.16</td><td rowspan="1" colspan="1" valign="middle" align="center">1.45</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>IGF2BP2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">6.1×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">9.6×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1544056</td><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">4.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.18</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11</td><td rowspan="1" colspan="1" valign="middle" align="center">1.26</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>PTPRD</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.2×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1333051</td><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">9.6×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.32</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21</td><td rowspan="1" colspan="1" valign="middle" align="center">1.44</td><td rowspan="1" colspan="1" valign="middle" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CDKN2A/CDKN2B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">3.0×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2237892</td><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">5.6×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.26</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14</td><td rowspan="1" colspan="1" valign="middle" align="center">1.40</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>KCNQ1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.4×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7305618</td><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">4.4×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.16</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>HNF1A</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.5×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.8×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs730570</td><td rowspan="1" colspan="1" valign="middle" align="left">14</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">4.0×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.21</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12</td><td rowspan="1" colspan="1" valign="middle" align="center">1.31</td><td rowspan="1" colspan="1" valign="middle" align="left">Downstream</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>C14orf70</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">9.7×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">7.0×10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm140055965847248" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3818640/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN8"><p id="P39" class="p p-first-last">Only markers that show the same direction of effect in the Hispanic and European studies are included</p></div><div id="TFN9"><p id="P40" class="p p-first-last">Chr Chromosome; NEA Non-effect allele; EA Effect allele; <em>p</em>-META <em>p</em> value Hispanic/2008 DIAGRAM meta-analysis; OR Odds ratio; OR 95L Lower 95% confidence limit of odds ratio; OR 95U Upper 95% confidence limit of odds ratio; <em>p</em>-DIAG <em>p</em> value 2008 DIAGRAM dataset; <em>p</em>-Hisp</p></div><div id="TFN10"><p id="P41" class="p p-first-last"><em>p</em> value Hispanic meta-analysis</p></div></div></div><p id="P20">Recently, the DIAGRAM consortium expanded their meta-analysis to include eight GWA studies in individuals of European ancestry (DIAGRAM+) [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918952">1</a>]. We did not have access to the full DIAGRAM+ dataset, but we were able to follow up the signals observed in our meta-analysis of the Hispanic (Mexico City and Starr County) and 2008 DIAGRAM studies in this expanded dataset. <a href="/pmc/articles/PMC3818640/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> reports the results of this analysis. Two of the previously described regions achieved genome-wide significance (<em>p</em>&lt;5×10<sup>−8</sup>): <em>CDKN2A/CDKN2B</em> (<em>p</em>=5.7×10<sup>−10</sup>) and <em>HNF1A</em> (<em>p</em>=1.9×10<sup>−8</sup>). <em>IGF2BP2, KCNQ1</em> and <em>C14orf70</em> also showed suggestive associations with type 2 diabetes (<em>p</em>=1.3×10<sup>−7</sup>, <em>p</em>=3.8×10<sup>−6</sup> and <em>p</em>=8.0×10<sup>−6</sup>, respectively). However, the <em>p</em> value for <em>PTPRD</em> decreased substantially (<em>p</em>=0.047), driven by the changes observed in the 2008 DIAGRAM and the DIAGRAM+ datasets.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p10">Follow-up of markers listed in <a href="/pmc/articles/PMC3818640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> in the recent DIAGRAM+ dataset</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140055929566080" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="left">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">NEA</th><th rowspan="1" colspan="1" valign="middle" align="center">EA</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em>-META</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">OR 95L</th><th rowspan="1" colspan="1" valign="middle" align="center">OR 95U</th><th rowspan="1" colspan="1" valign="middle" align="left">Type</th><th rowspan="1" colspan="1" valign="middle" align="left">Closest gene</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em>-DIAG+</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>p</em>-Hisp</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1374910</td><td rowspan="1" colspan="1" valign="middle" align="left">3</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.3×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15</td><td rowspan="1" colspan="1" valign="middle" align="center">1.34</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>IGF2BP2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4.1×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">9.6×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1544056</td><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">4.7×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.05</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>PTPRD</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4.5×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.2×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1333051</td><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">5.7×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15</td><td rowspan="1" colspan="1" valign="middle" align="center">1.30</td><td rowspan="1" colspan="1" valign="middle" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CDKN2A/CDKN2B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">7.1×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2237892</td><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">3.8×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.20</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11</td><td rowspan="1" colspan="1" valign="middle" align="center">1.29</td><td rowspan="1" colspan="1" valign="middle" align="left">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>KCNQ1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2.7×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7305618</td><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.9×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.14</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09</td><td rowspan="1" colspan="1" valign="middle" align="center">1.20</td><td rowspan="1" colspan="1" valign="middle" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>HNF1A</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">6.0×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.8×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs730570</td><td rowspan="1" colspan="1" valign="middle" align="left">14</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">8.0×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.14</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21</td><td rowspan="1" colspan="1" valign="middle" align="left">Downstream</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>C14orf70</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">6.6×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">7.0×10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm140055929566080" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3818640/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN11"><p id="P42" class="p p-first-last">Chr Chromosome; NEA Non-effect allele; EA Effect allele; <em>p</em>-META <em>p</em> value Hispanic/DIAGRAM+ meta-analysis; OR Odds ratio; OR 95L Lower 95% confidence limit of odds ratio; OR 95U Upper 95% confidence limit of odds ratio; <em>p</em>-DIAG+ <em>p</em> value DIAGRAM+ dataset; <em>p</em>-Hisp <em>p</em> value Hispanic meta-analysis</p></div></div></div><p id="P21" class="p p-last">Of the top signals (<em>p</em>&lt;10<sup>−5</sup>) observed in the meta-analysis of the Mexico City and Starr County studies (<a href="/pmc/articles/PMC3818640/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), the <em>HNF1A</em> and <em>KCNQ1</em> regions were confirmed in the European dataset. Also, the marker rs10465543 within the <em>EPHB2</em> gene was nominally significant in the DIAGRAM+ dataset (<em>p</em>=2.5×10<sup>−2</sup>). None of the remaining regions (we followed up only the markers with the lowest <em>p</em> value in each region) were nominally significant in the DIAGRAM+ dataset.</p></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Discussion</h2><p id="P22" class="p p-first">We carried out a GWA study of type 2 diabetes in a sample from Mexico City. We observed strong evidence of population stratification due to variation in admixture proportions within the sample (<a href="/pmc/articles/PMC3818640/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). These results are in agreement with our previous research, in which we had described an association between American Indian ancestry and type 2 diabetes in a sample from Mexico City [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918946">15</a>]. However, it is important to note that socioeconomic status was inversely correlated with diabetes risk and American Indian ancestry in that study. A strong association between socioeconomic status and genetic ancestry has also been reported in other admixed samples in the Americas [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918945">16</a>]. Unfortunately, we do not have information on socioeconomic status for all the samples included in the present analysis. However, an analysis of the subset of samples for which we have information on education showed, in agreement with our previous research, that American Indian ancestry was inversely correlated with education (data not shown). Thus, some of the association between ancestry and disease risk observed in our sample could have been mediated by the effects of socioeconomic status. Irrespective of the factors responsible for the strong population stratification observed in the sample from Mexico City, inclusion of ancestry in the statistical model corrects for the effects of population stratification (<a href="#SD1" rid="SD1" class=" supplementary-material">ESM Figs 1</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">2</a>). Our study emphasises the need to include ancestry in GWA studies in admixed populations, in order to control for confounding. The comparison of our sample from Mexico City with other samples of Mexican ancestry from the US (Mexican–Americans from Los Angeles and Starr County) highlights the large variation in admixture proportions (and also in the extent of population stratification) that may be present in populations of Mexican ancestry living in different geographic areas. For example, based on a genome-wide panel of markers, we estimated that the average American Indian contribution to the Mexican–American HapMap and Starr County samples is substantially lower than that observed in Mexico City (approximately 20% and 30% lower, respectively). Additionally, in contrast to the strong stratification observed in Mexico City, the scores of the top axes of the principal component analysis were not significantly associated with type 2 diabetes in the Starr County sample.</p><p id="P23">In our statistical analysis of the Mexico City sample, we identified several regions showing suggestive associations with type 2 diabetes (<a href="#SD5" rid="SD5" class=" supplementary-material">ESM Tables 1</a> and <a href="#SD6" rid="SD6" class=" supplementary-material">2</a>). We also performed a meta-analysis of two type 2 diabetes GWA studies in Hispanics (Mexico City and Starr County, Texas, USA), further exploring the SNPs that showed nominal significance and similar direction of effect in those analyses in samples of European ancestry (2008 DIAGRAM and DIAGRAM+ datasets). Two of the top signals (<em>p</em>&lt;10<sup>−5</sup>) observed in the Mexico City/Starr County meta-analysis correspond to regions identified in previous GWA studies: <em>HNF1A</em> (hepatocyte nuclear factor 1-alpha) and <em>KCNQ1</em> (potassium voltage-gated channel, KQT-like subfamily, member 1).</p><p id="P24">Variants within and near <em>HNF1A</em> showed the lowest <em>p</em> values in our Hispanic meta-analysis. Functional mutations in this gene underlie MODY3, (MIM number 600496), a familial form of non-insulin-dependent diabetes mellitus with early age of onset and autosomal dominant inheritance. Additionally, a marker near <em>HNF1A</em> reached genome-wide significance in the joint analysis of stage 1 and 2 data in the recent DIAGRAM+ study (rs7957197, <em>p</em>=2.4×10<sup>−8</sup>) [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918948">1</a>]. This meta-analysis in European samples added <em>HNF1A</em> to an increasing list of loci involved in monogenic/syndromic forms of diabetes and in the more common, multifactorial type 2 diabetes, which currently includes <em>GCK</em>, <em>HNF1B</em>, <em>PPARG</em>, <em>WFS1</em> and <em>KCNJ11</em> [<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918954">2</a>]. The SNP with the lowest <em>p</em> value in our Hispanic meta-analysis (rs7305618) (<a href="/pmc/articles/PMC3818640/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) also showed suggestive <em>p</em> values in the DIAGRAM+ study (<em>p</em>=6.0×10<sup>−6</sup>) and achieved genome-wide significance in the combined Hispanic/DIAGRAM+ meta-analysis (<em>p</em>=1.9×10<sup>−8</sup>, OR 1.14) (<a href="/pmc/articles/PMC3818640/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The sentinel SNP reported in the DIAGRAM+ study (rs7957197) is located less than 60 kilobases apart from rs7305618 on chromosome 12. Unfortunately, data for rs7957197 were not available for the Starr County sample. However, these two SNPs are in relatively strong LD in the HapMap European sample (<em>r</em><sup>2</sup>=0.54, D’=0.923) and they show similar <em>p</em> values in the Mexico City sample (rs7957197 <em>p</em>=0.0012; rs7305618 <em>p</em>=0.001), suggesting that the Hispanic and European studies may be capturing the same signal.</p><p id="P25">Polymorphisms within the gene <em>KCNQ1</em> were originally associated with susceptibility to type 2 diabetes in two relatively small GWA studies in Japanese samples [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918937">3</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918933">4</a>] and have been confirmed in many subsequent studies, including two recent GWA studies [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918931">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918930">8</a>]. Interestingly, one of the intronic variants identified in our Hispanic meta-analysis, rs2237892 (<em>p</em>=4.4×10<sup>−5</sup>) is the same variant described in one of the original Japanese studies [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918929">3</a>]. This SNP also had a nominally significant <em>p</em> value in the DIAGRAM+ dataset (<em>p</em>=2.7×10<sup>−3</sup>), while the <em>p</em> value of the Hispanic/DIAGRAM+ meta-analysis was 3.8×10<sup>−6</sup> (OR 1.20) (<a href="/pmc/articles/PMC3818640/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Similarly to the situation described in East Asians, the frequency of the rs2237892 risk allele (C allele) is substantially higher in Europeans than in American Indian populations (data not shown). Importantly, it should be noted that there is now evidence of two independent signals in <em>KCNQ1</em>. The recent DIAGRAM+ study identified a genome-wide signal in this gene (lead SNP, rs231362, <em>p</em>=2.8×10<sup>−13</sup>), which is independent of the markers identified in East Asian scans [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918947">1</a>]. The SNPs rs231362 and rs2237892 are not correlated in the European HapMap (<em>r</em><sup>2</sup>=0.019), East Asian HapMap (<em>r</em><sup>2</sup>=0.016) or the Mexico City samples (<em>r</em><sup>2</sup>=0.001). The marker rs231362 is not significant in the Mexico City sample (<em>p</em>=0.195), and the <em>p</em> value in the Starr County study was <em>p</em>=0.040 (meta-analysis <em>p</em>=0.016).</p><p id="P26">Other loci identified in the Hispanic/DIAGRAM+ meta-analysis and reported in previous GWA studies are <em>CDKN2A/CDKN2B</em> (cyclin-dependent kinase inhibitor 2A and 2B) on chromosome 9 and <em>IGF2BP2</em> (insulin-like growth factor 2 mRNA binding protein 2) on chromosome 3. Several GWA studies in European and Japanese populations have identified genome-wide associations between markers near <em>CDKN2A/CDKN2B</em> and type 2 diabetes [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918935">7</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918926">14</a>, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918953">17</a>–<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918950">19</a>]. In our Hispanic meta-analysis, the SNP rs1333051 had a <em>p</em> value of 2.3×10<sup>−4</sup>. The <em>p</em> value in DIAGRAM+ for this marker was <em>p</em>=7.1×10<sup>−8</sup> and the combined Hispanic/DIAGRAM+<em>p</em> value reached genome-wide significant levels (<em>p</em>=5.7×10<sup>−10</sup>, OR 1.22). In the Mexico City sample, rs1333051 is in strong LD with the lead SNPs reported in Japanese and European studies, e.g. rs2383208 [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918939">7</a>] and rs10811661 [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918949">19</a>] (<em>r</em><sup>2</sup>&gt;0.75, D’&gt;0.95). SNPs within the <em>IGF2BP2</em> gene have also been associated with type 2 diabetes in many GWA studies [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918928">1</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918944">4</a>, <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918943">7</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918955">14</a>, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918938">17</a>–<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918921">19</a>]. The intronic variant rs1374910 located within this gene showed suggestive evidence of association in the Hispanic/DIAGRAM+ meta-analysis (<em>p</em>=1.3×10<sup>−7</sup>, OR 1.24). In the Mexico City sample, rs1374910 showed weak LD with the sentinel SNPs reported in previous studies, e.g. rs1470579 [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918932">1</a>], rs6769511 [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918956">4</a>] and rs4402960 [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918942">17</a>–<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918925">19</a>] (<em>r</em><sup>2</sup>&lt;0.10, D’&lt;0.45), while the <em>p</em> value for rs1374910 (<em>p</em>=9.6×10<sup>−5</sup>) in the Hispanic meta-analysis was substantially lower than those observed for the other three markers (<em>p</em>=0.01 approx.). The opposite is the case for the DIAGRAM+ study (rs1470579, <em>p</em>=2×10<sup>−9</sup> and rs1374910, <em>p</em>=4.1×10<sup>−5</sup>). It would be interesting to explore this region in more detail in additional populations.</p><p id="P27">Finally, we observed another region on chromosome 14 showing suggestive association with type 2 diabetes in the Hispanic/DIAGRAM+ meta-analysis. The SNP rs730570, which is very close to <em>C14orf70</em> (chromosome 14 open reading frame 70), showed a combined <em>p</em> value of <em>p</em>=8.0×10<sup>−6</sup> in this analysis (OR 1.14) (<a href="/pmc/articles/PMC3818640/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). To our knowledge, this region has not been reported in previous GWA studies. It and other markers in this region should therefore be followed up in additional studies in order to confirm its potential involvement in type 2 diabetes risk.</p><p id="P28">Many of the top signals observed in our meta-analysis of the Mexico City and Starr County studies could not be confirmed in the DIAGRAM+ dataset. The markers identified near or within the genes RP11-354K1.1, LSAMP, FGF12, ANK2, CSMD1, MCPH1, NIPAL2, SLC5A8, CIT, CHRNA7 and <em>DEPDC5</em>, as well as the intergenic region on chromosome 9 (<a href="/pmc/articles/PMC3818640/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), were not nominally significant in the DIAGRAM+ meta-analysis. Similarly, one of the loci initially identified in the Hispanic/2008 DIAGRAM meta-analysis (<em>PTPRD</em>) could not be confirmed in the DIAGRAM+ dataset. It is possible that some of these signals are just false positives. Alternatively, these discrepancies may be due to population differences between the Hispanic and DIAGRAM datasets. The two samples of Mexican ancestry show a substantial American Indian component (in fact, the sample from Mexico City has primarily American Indian ancestry). It is crucial to follow up these regions in additional Hispanic and/or American Indian samples in order to clarify these issues. Interestingly, some of the genes mentioned above have been previously associated with type 2 diabetes or diabetes-related traits in previous studies, and/or are relevant from the functional point of view. For example, the gene <em>PTPRD</em> (protein tyrosine phosphatase, receptor type, D) has been previously associated with type 2 diabetes in a recent GWA study in Han Chinese [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918941">8</a>]. However, the variant identified in the Chinese study (rs17584499) was not significant in our meta-analysis (<em>p</em>=0.517). A recent study exploring the genetic determinants of metabolic syndrome in the Framingham Heart Study (Offspring Cohort, Exam 7) identified an association between <em>CSMD1</em> (CUB and Sushi multiple domains 1) and metabolic syndrome using two different approaches [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918922">20</a>]. Finally, recent studies have highlighted the potential role of the product of <em>ANK2</em>, ankyrin-B, in metabolic regulation [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918957">21</a>, <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918936">22</a>]. Kline et al. [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918951">21</a>] demonstrated that the C-terminal domain of Kir6.2, a critical component of K<sub>ATP</sub> channels in many cells including pancreatic cells, contains a C-terminal ankyrin-B-binding motif that is important in K<sub>ATP</sub> channel trafficking and membrane metabolic regulation. Healy et al. [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543918940">22</a>] reported that ankyrin-B is involved in parasympathetic potentiation of glucose-stimulated insulin secretion, and that the R1788W mutation of <em>ANK2</em> impaired its function in pancreatic islets and is associated with type 2 diabetes in individuals of European descent and Hispanics.</p><p id="P29" class="p p-last">In summary, we report the results of a GWA study of type 2 diabetes in an admixed sample from Mexico City and of a meta-analysis combining this study with another genome-wide survey in a Mexican-American sample from Starr County, Texas, USA. It is critical to carry out additional GWA studies in Hispanic and American Indian populations, which have a high prevalence of type 2 diabetes. Most of the recent advances in our knowledge of the genetic architecture of type 2 diabetes have been driven primarily by GWA studies in European populations, which have sampled tens of thousands of individuals. Further advancement will require a similar effort in other population groups in order to obtain a more complete understanding of the common genetic factors involved in type 2 diabetes risk.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>ESM1</h4><div class="sup-box half_rhythm" id="d36e1140"><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(29K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>ESM2</h4><div class="sup-box half_rhythm" id="d36e1144"><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(29K, pdf)</sup></div></div><div class="sec suppmat" id="SD3"><h4>ESM3</h4><div class="sup-box half_rhythm" id="d36e1148"><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM3.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12K, pdf)</sup></div></div><div class="sec suppmat" id="SD4"><h4>ESM4</h4><div class="sup-box half_rhythm" id="d36e1152"><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM4.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12K, pdf)</sup></div></div><div class="sec suppmat" id="SD5"><h4>ESM5</h4><div class="sup-box half_rhythm" id="d36e1156"><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM5.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(33K, pdf)</sup></div></div><div class="sec suppmat" id="SD6"><h4>ESM6</h4><div class="sup-box half_rhythm" id="d36e1160"><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM6.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(35K, pdf)</sup></div></div><div class="sec suppmat" id="SD7"><h4>ESM7</h4><div class="sup-box half_rhythm" id="d36e1164"><a href="/pmc/articles/PMC3818640/bin/NIHMS472351-supplement-ESM7.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(37K, pdf)</sup></div></div></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Acknowledgements</h2><div class="sec"><p id="__p1">We thank all those individuals who participated in the study. In Canada this work was supported by the Canadian Institutes of Health Research, the Banting and Best Diabetes Centre, the Canada Foundation for Innovation and The Ontario Innovation Trust. In Mexico this work was supported by the following grants: CONACYT SALUD-2005-C02-14412, Proyectos Estrategicos, Apoyo Financiero Fundacion IMSS and Fundacion Gonzalo Rio Arronte I, and by A. P. Mexico and Apoyos a la Vinculacion del Cinvestav con el Sector Salud 2006. M. Cruz is a recipient of a Fundación IMSS Scholarship, Mexico. We thank the members of the DIAGRAM consortium for kindly sharing the summary data of their meta-analysis.</p></div></div><div id="idm140055964200656" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140055964200656title">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td>DIAGRAM</td><td>Diabetes Genetics Replication and Meta-analysis Consortium</td></tr><tr><td>GWA</td><td>Genome-wide association</td></tr><tr><td>LD</td><td>Linkage disequilibrium</td></tr><tr><td>QQ</td><td>Quantile-Quantile</td></tr><tr><td>SNP</td><td>Single nucleotide polymorphism</td></tr></tbody></table></div></div></div><div id="idm140055964203456" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140055964203456title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P30" class="p p-first-last"><strong>Duality of interest</strong> The authors declare that there is no duality of interest associated with this manuscript.</p><p></p><p class="fn sec" id="FN2"></p><p id="P31" class="p p-first-last"><strong>Electronic supplementary material</strong> The online version of this article (doi:10.1007/s00125-011-2172-y) contains supplementary material, including a full list of members of the DIAGRAM Consortium, which is available to authorised users.</p><p></p></div></div><div id="idm140055925614192aff-info" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140055925614192aff-infotitle">Contributor Information</h2><p><span class="fm-affl">E. J. Parra, </span><span class="fm-affl" id="A1" lang="en"> Department of Anthropology, University of Toronto at Mississauga, 3359 Mississauga Road North, Mississauga, ON, Canada L5L 1C6.</span></p><p><span class="fm-affl">J. E. Below, </span><span class="fm-affl" id="A2" lang="en"> Department of Medicine, Section of Genetic Medicine, University of Chicago, Chicago, IL, USA.</span></p><p><span class="fm-affl">S. Krithika, </span><span class="fm-affl" id="A3" lang="en"> Department of Anthropology, University of Toronto at Mississauga, 3359 Mississauga Road North, Mississauga, ON, Canada L5L 1C6.</span></p><p><span class="fm-affl">A. Valladares, </span><span class="fm-affl" id="A4" lang="en"> Unidad de Investigacion Medica en Bioquimica, Hospital de Especialidades, Centro Medico Nacional Siglo XXI, IMSS, Av. Cuauhtemoc 330, Col. Doctores, C.P. 06720, Mexico City, Mexico.</span></p><p><span class="fm-affl">J. L. Barta, </span><span class="fm-affl" id="A5" lang="en"> Department of Anthropology, University of Toronto at Mississauga, 3359 Mississauga Road North, Mississauga, ON, Canada L5L 1C6.</span></p><p><span class="fm-affl">N. J. Cox, </span><span class="fm-affl" id="A6" lang="en"> Department of Medicine, Section of Genetic Medicine, University of Chicago, Chicago, IL, USA.</span></p><p><span class="fm-affl">C. L. Hanis, </span><span class="fm-affl" id="A7" lang="en"> Human Genetics Center, School of Public Health, The University of Texas, Health Science Center at Houston, Houston, TX, USA.</span></p><p><span class="fm-affl">N. Wacher, </span><span class="fm-affl" id="A8" lang="en"> Unidad de Investigacion Medica en Epidemiologia Clinica, Hospital de Especialidades, Centro Medico Siglo XXI, IMSS, Mexico City, Mexico.</span></p><p><span class="fm-affl">J. Garcia-Mena, </span><span class="fm-affl" id="A9" lang="en"> Departamento de Genetica y Biologia Molecular, Cinvestav-IPN, Mexico City, Mexico.</span></p><p><span class="fm-affl">P. Hu, </span><span class="fm-affl" id="A10" lang="en"> The Centre for Applied Genomics, The Hospital for Sick Children, Toronto, ON, Canada.</span></p><p><span class="fm-affl">M. D. Shriver, </span><span class="fm-affl" id="A11" lang="en"> Department of Anthropology, Penn State University, University Park, PA, USA.</span></p><p><span class="fm-affl">J. Kumate, </span><span class="fm-affl" id="A12" lang="en"> Fundacion IMSS, Mexico City, Mexico.</span></p><p><span class="fm-affl">P. M. McKeigue, </span><span class="fm-affl" id="A13" lang="en"> Public Health Sciences, University of Edinburgh Medical School, Edinburgh, Scotland, UK.</span></p><p><span class="fm-affl">J. Escobedo, </span><span class="fm-affl" id="A14" lang="en"> Unidad de Investigacion en Epidemiologia Clinica, Hospital General Regional 1, Dr Carlos McGregor, IMSS, Mexico City, Mexico.</span></p><p><span class="fm-affl">M. Cruz, </span><span class="fm-affl" id="A15" lang="en"> Unidad de Investigacion Medica en Bioquimica, Hospital de Especialidades, Centro Medico Nacional Siglo XXI, IMSS, Av. Cuauhtemoc 330, Col. Doctores, C.P. 06720, Mexico City, Mexico.</span></p></div><div id="idm140055964194880" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140055964194880title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Voight BF, Scott LJ, Steinthorsdottir V, et al.  Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:579–589.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080658/">PMC free article</a>]</span> [<a href="/pubmed/20581827" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Twelve+type+2+diabetes+susceptibility+loci+identified+through+large-scale+association+analysis&amp;author=BF+Voight&amp;author=LJ+Scott&amp;author=V+Steinthorsdottir&amp;volume=42&amp;publication_year=2010&amp;pages=579-589&amp;pmid=20581827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">McCarthy MI. Genomics, type 2 diabetes, and obesity. <span><span class="ref-journal">N Engl J Med. </span>2010;<span class="ref-vol">363</span>:2339–2350.</span> [<a href="/pubmed/21142536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genomics,+type+2+diabetes,+and+obesity&amp;author=MI+McCarthy&amp;volume=363&amp;publication_year=2010&amp;pages=2339-2350&amp;pmid=21142536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Yasuda K, Miyake K, Horikawa Y, et al.  Variants in KCNQ1 are associated with susceptibility to type 2 diabetes mellitus. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1092–1097.</span> [<a href="/pubmed/18711367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variants+in+KCNQ1+are+associated+with+susceptibility+to+type+2+diabetes+mellitus&amp;author=K+Yasuda&amp;author=K+Miyake&amp;author=Y+Horikawa&amp;volume=40&amp;publication_year=2008&amp;pages=1092-1097&amp;pmid=18711367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Unoki H, Takahashi A, Kawaguchi T, et al.  SNPs in KCNQ1 are associated with susceptibility to type 2 diabetes in East Asian and European populations. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1098–1102.</span> [<a href="/pubmed/18711366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=SNPs+in+KCNQ1+are+associated+with+susceptibility+to+type+2+diabetes+in+East+Asian+and+European+populations&amp;author=H+Unoki&amp;author=A+Takahashi&amp;author=T+Kawaguchi&amp;volume=40&amp;publication_year=2008&amp;pages=1098-1102&amp;pmid=18711366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Hayes MG, Pluzhnikov A, Miyake K, et al.  Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3033–3044.</span> [<a href="/pubmed/17846124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+type+2+diabetes+genes+in+Mexican+Americans+through+genome-wide+association+studies&amp;author=MG+Hayes&amp;author=A+Pluzhnikov&amp;author=K+Miyake&amp;volume=56&amp;publication_year=2007&amp;pages=3033-3044&amp;pmid=17846124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Hanson RL, Bogardus C, Duggan D, et al.  A search for variants associated with young-onset type 2 diabetes in American Indians in a 100 K genotyping array. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3045–3052.</span> [<a href="/pubmed/17846125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+search+for+variants+associated+with+young-onset+type+2+diabetes+in+American+Indians+in+a+100+K+genotyping+array&amp;author=RL+Hanson&amp;author=C+Bogardus&amp;author=D+Duggan&amp;volume=56&amp;publication_year=2007&amp;pages=3045-3052&amp;pmid=17846125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Takeuchi F, Serizawa M, Yamamoto K, et al.  Confirmation of multiple risk loci and genetic impacts by a genome-wide association study of type 2 diabetes in the Japanese population. <span><span class="ref-journal">Diabetes. </span>2009;<span class="ref-vol">58</span>:1690–1699.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2699880/">PMC free article</a>]</span> [<a href="/pubmed/19401414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Confirmation+of+multiple+risk+loci+and+genetic+impacts+by+a+genome-wide+association+study+of+type+2+diabetes+in+the+Japanese+population&amp;author=F+Takeuchi&amp;author=M+Serizawa&amp;author=K+Yamamoto&amp;volume=58&amp;publication_year=2009&amp;pages=1690-1699&amp;pmid=19401414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Tsai FJ, Yang CF, Chen CC, et al.  A genome-wide association study identifies susceptibility variants for type 2 diabetes in Han Chinese. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000847.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2824763/">PMC free article</a>]</span> [<a href="/pubmed/20174558" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+identifies+susceptibility+variants+for+type+2+diabetes+in+Han+Chinese&amp;author=FJ+Tsai&amp;author=CF+Yang&amp;author=CC+Chen&amp;volume=6&amp;publication_year=2010&amp;pages=e1000847&amp;pmid=20174558&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">McCarthy MI. Casting a wider net for diabetes susceptibility genes. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1039–1040.</span> [<a href="/pubmed/19165915" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Casting+a+wider+net+for+diabetes+susceptibility+genes&amp;author=MI+McCarthy&amp;volume=40&amp;publication_year=2008&amp;pages=1039-1040&amp;pmid=19165915&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Below JE, Gamazon ER, Morrison JV, et al.  Genome-wide association and meta-analysis in populations from Starr County, Texas and Mexico City identify type 2 diabetes susceptibility loci and enrichment for eQTLs in top signals. <span><span class="ref-journal">Diabetologia. </span>2011</span> doi:10.1007/s00125-011-2188-3. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3761075/">PMC free article</a>]</span> [<a href="/pubmed/21647700" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Genome-wide+association+and+meta-analysis+in+populations+from+Starr+County,+Texas+and+Mexico+City+identify+type+2+diabetes+susceptibility+loci+and+enrichment+for+eQTLs+in+top+signals&amp;author=JE+Below&amp;author=ER+Gamazon&amp;author=JV+Morrison&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a toolset for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+toolset+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Mao X, Bigham AW, Mei R, et al.  A genomewide admixture mapping panel for Hispanic/Latino populations. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:1171–1178.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1867104/">PMC free article</a>]</span> [<a href="/pubmed/17503334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genomewide+admixture+mapping+panel+for+Hispanic/Latino+populations&amp;author=X+Mao&amp;author=AW+Bigham&amp;author=R+Mei&amp;volume=80&amp;publication_year=2007&amp;pages=1171-1178&amp;pmid=17503334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Ge D, Zhang D, Need AC, et al.  WGAViewer: software for genomic annotation of whole genome association studies. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:640–643.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2279251/">PMC free article</a>]</span> [<a href="/pubmed/18256235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=WGAViewer:+software+for+genomic+annotation+of+whole+genome+association+studies&amp;author=D+Ge&amp;author=D+Zhang&amp;author=AC+Need&amp;volume=18&amp;publication_year=2008&amp;pages=640-643&amp;pmid=18256235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Zeggini E, Scott LJ, Saxena R, et al.  Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:638–645.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=LJ+Scott&amp;author=R+Saxena&amp;volume=40&amp;publication_year=2008&amp;pages=638-645&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Martinez-Marignac VL, Valladares A, Cameron E, et al.  Admixture in Mexico City: implications for admixture mapping of type 2 diabetes genetic risk factors. <span><span class="ref-journal">Hum Genet. </span>2007;<span class="ref-vol">120</span>:807–819.</span> [<a href="/pubmed/17066296" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Admixture+in+Mexico+City:+implications+for+admixture+mapping+of+type+2+diabetes+genetic+risk+factors&amp;author=VL+Martinez-Marignac&amp;author=A+Valladares&amp;author=E+Cameron&amp;volume=120&amp;publication_year=2007&amp;pages=807-819&amp;pmid=17066296&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Florez JC, Price AL, Campbell D, et al.  Strong association of socioeconomic status with genetic ancestry in Latinos: implications for admixture studies of type 2 diabetes. <span><span class="ref-journal">Diabetologia. </span>2009;<span class="ref-vol">52</span>:1528–1536.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3113605/">PMC free article</a>]</span> [<a href="/pubmed/19526211" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Strong+association+of+socioeconomic+status+with+genetic+ancestry+in+Latinos:+implications+for+admixture+studies+of+type+2+diabetes&amp;author=JC+Florez&amp;author=AL+Price&amp;author=D+Campbell&amp;volume=52&amp;publication_year=2009&amp;pages=1528-1536&amp;pmid=19526211&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Diabetes Genetics Initiative of Broad Institute of Harvard and MIT. Lund University. Novartis Institutes of BioMedical Research et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Zeggini E, Weedon MN, Lindgren CM, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=MN+Weedon&amp;author=CM+Lindgren&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Scott LJ, Mohlke KL, Bonnycastle LL, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;author=KL+Mohlke&amp;author=LL+Bonnycastle&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Nock NL, Wang X, Thompson CL, et al.  Defining genetic determinants of the metabolic syndrome in the Framingham Heart Study using association and structural equation modeling methods. <span><span class="ref-journal">BMC Proc. </span>2009;<span class="ref-vol">3</span>(Suppl 7):S50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2795950/">PMC free article</a>]</span> [<a href="/pubmed/20018043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Proc&amp;title=Defining+genetic+determinants+of+the+metabolic+syndrome+in+the+Framingham+Heart+Study+using+association+and+structural+equation+modeling+methods&amp;author=NL+Nock&amp;author=X+Wang&amp;author=CL+Thompson&amp;volume=3&amp;issue=Suppl+7&amp;publication_year=2009&amp;pages=S50&amp;pmid=20018043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Kline CF, Kurata HT, Hund TJ, et al.  Dual role of K ATP channel C-terminal motif in membrane targeting and metabolic regulation. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009;<span class="ref-vol">106</span>:16669–16674.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2757796/">PMC free article</a>]</span> [<a href="/pubmed/19805355" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Dual+role+of+K+ATP+channel+C-terminal+motif+in+membrane+targeting+and+metabolic+regulation&amp;author=CF+Kline&amp;author=HT+Kurata&amp;author=TJ+Hund&amp;volume=106&amp;publication_year=2009&amp;pages=16669-16674&amp;pmid=19805355&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Healy JA, Nilsson KR, Hohmeier HE, et al.  Cholinergic augmentation of insulin release requires ankyrin-B. <span><span class="ref-journal">Sci Signal. </span>2010;<span class="ref-vol">3</span>:ra19.</span> [<a href="/pubmed/20234002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Signal&amp;title=Cholinergic+augmentation+of+insulin+release+requires+ankyrin-B&amp;author=JA+Healy&amp;author=KR+Nilsson&amp;author=HE+Hohmeier&amp;volume=3&amp;publication_year=2010&amp;pages=ra19&amp;pmid=20234002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3818640&amp;issue-id=229346&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140055922896192title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Methods</a></li><li><a href="#S16title">Results</a></li><li><a href="#S21title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S22title">Acknowledgements</a></li><li><a href="#idm140055964200656title">Abbreviations</a></li><li><a href="#idm140055964203456title">Footnotes</a></li><li><a href="#idm140055925614192aff-infotitle">Contributor Information</a></li><li><a href="#idm140055964194880title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3818640/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3818640/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3818640/pdf/nihms-472351.pdf">PDF (75K)</a></li> | <li><a href="#" data-citationid="PMC3818640" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3818640%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3818640%2F&amp;text=Genome-wide%20association%20study%20of%20type%202%20diabetes%20in%20a%20sample%20from%20Mexico%20City%20and%20a%20meta-analysis%20of%20a%20Mexican-American%20sample%20from%20Starr%20County%2C%20Texas" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3818640%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21573907" data-aiid="3818640" data-aid="3818640" data-iid="229346" data-domainid="319" data-domain="nihpa" data-accid="PMC3818640" data-md5="10540c63fa2f086204cc32d2449b2305"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:04:27-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89ED29DD5238410000000005D703BD&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3818640%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3818640/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>