<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3761075/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3761075/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Diabetologia"><meta name="citation_title" content="Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals"><meta name="citation_authors" content="J. E. Below, E. R. Gamazon, J. V. Morrison, A. Konkashbaev, A. Pluzhnikov, P. M. McKeigue, E. J. Parra, S. C. Elbein, D. M. Hallman, D. L. Nicolae, G. I. Bell, M. Cruz, N. J. Cox, C. L. Hanis"><meta name="citation_date" content="August 2011"><meta name="citation_issue" content="8"><meta name="citation_volume" content="54"><meta name="citation_firstpage" content="2047"><meta name="citation_doi" content="10.1007/s00125-011-2188-3"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3761075/?report=abstract"><meta name="citation_pmid" content="21647700"><meta name="DC.Title" content="Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="J. E. Below"><meta name="DC.Contributor" content="E. R. Gamazon"><meta name="DC.Contributor" content="J. V. Morrison"><meta name="DC.Contributor" content="A. Konkashbaev"><meta name="DC.Contributor" content="A. Pluzhnikov"><meta name="DC.Contributor" content="P. M. McKeigue"><meta name="DC.Contributor" content="E. J. Parra"><meta name="DC.Contributor" content="S. C. Elbein"><meta name="DC.Contributor" content="D. M. Hallman"><meta name="DC.Contributor" content="D. L. Nicolae"><meta name="DC.Contributor" content="G. I. Bell"><meta name="DC.Contributor" content="M. Cruz"><meta name="DC.Contributor" content="N. J. Cox"><meta name="DC.Contributor" content="C. L. Hanis"><meta name="DC.Date" content="2011 Aug"><meta name="DC.Identifier" content="10.1007/s00125-011-2188-3"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals"><meta property="og:type" content="article"><meta property="og:description" content="We conducted genome-wide association studies (GWASs) and expression quantitative trait loci (eQTL) analyses to identify and characterise risk loci for type 2 diabetes in Mexican-Americans from Starr County, TX, USA.Using 1.8 million directly interrogated ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3761075/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3761075/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3761075/pdf/nihms471860.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE898D78DD51E4D1000000000A07067F.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3761075%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3761075</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Diabetologia</a></span>. Author manuscript; available in PMC 2013 Sep 4.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21647700" target="pmc_ext" ref="reftype=publisher&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Diabetologia. 2011 Aug; 54(8): 2047–2055. </span></a></div><span class="fm-vol-iss-date">Published online 2011 Jun 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00125-011-2188-3" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00125-011-2188-3</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3761075</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS471860</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21647700">21647700</a></div></div></div></div><h1 class="content-title">Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Below%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627907853136" co-class="co-affbox">J. E. Below</a>, <a href="/pubmed/?term=Gamazon%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627900129120" co-class="co-affbox">E. R. Gamazon</a>, <a href="/pubmed/?term=Morrison%20JV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627895582944" co-class="co-affbox">J. V. Morrison</a>, <a href="/pubmed/?term=Konkashbaev%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627895581408" co-class="co-affbox">A. Konkashbaev</a>, <a href="/pubmed/?term=Pluzhnikov%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627903866464" co-class="co-affbox">A. Pluzhnikov</a>, <a href="/pubmed/?term=McKeigue%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627903864592" co-class="co-affbox">P. M. McKeigue</a>, <a href="/pubmed/?term=Parra%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627907383776" co-class="co-affbox">E. J. Parra</a>, <a href="/pubmed/?term=Elbein%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627905322720" co-class="co-affbox">S. C. Elbein</a>, <a href="/pubmed/?term=Hallman%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627905320848" co-class="co-affbox">D. M. Hallman</a>, <a href="/pubmed/?term=Nicolae%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627905627648" co-class="co-affbox">D. L. Nicolae</a>, <a href="/pubmed/?term=Bell%20GI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627899536096" co-class="co-affbox">G. I. Bell</a>, <a href="/pubmed/?term=Cruz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627899534208" co-class="co-affbox">M. Cruz</a>, <a href="/pubmed/?term=Cox%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627939609376" co-class="co-affbox">N. J. Cox</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup> and  <a href="/pubmed/?term=Hanis%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700" class="affpopup" co-rid="_co_idm139627904519456" co-class="co-affbox">C. L. Hanis</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139627907853136"><h3 class="no_margin">J. E. Below</h3><p>Department of Human Genetics, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Below%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">J. E. Below</a></div></div><div id="_co_idm139627900129120"><h3 class="no_margin">E. R. Gamazon</h3><p>Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Gamazon%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">E. R. Gamazon</a></div></div><div id="_co_idm139627895582944"><h3 class="no_margin">J. V. Morrison</h3><p>Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Morrison%20JV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">J. V. Morrison</a></div></div><div id="_co_idm139627895581408"><h3 class="no_margin">A. Konkashbaev</h3><p>Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Konkashbaev%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">A. Konkashbaev</a></div></div><div id="_co_idm139627903866464"><h3 class="no_margin">A. Pluzhnikov</h3><p>Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Pluzhnikov%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">A. Pluzhnikov</a></div></div><div id="_co_idm139627903864592"><h3 class="no_margin">P. M. McKeigue</h3><p>Public Health Sciences Section, Division of Community Health Sciences, University of Edinburgh Medical School, Edinburgh, UK</p><div>Find articles by <a href="/pubmed/?term=McKeigue%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">P. M. McKeigue</a></div></div><div id="_co_idm139627907383776"><h3 class="no_margin">E. J. Parra</h3><p>Department of Anthropology, University of Toronto, Mississauga, ON, Canada</p><div>Find articles by <a href="/pubmed/?term=Parra%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">E. J. Parra</a></div></div><div id="_co_idm139627905322720"><h3 class="no_margin">S. C. Elbein</h3><p>Section on Endocrinology and Metabolism, Wake Forest University School of Medicine, Winston-Salem, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Elbein%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">S. C. Elbein</a></div></div><div id="_co_idm139627905320848"><h3 class="no_margin">D. M. Hallman</h3><p>Human Genetics Center, University of Texas Health Science Center at Houston, P.O. Box 20186, Houston, TX 77225, USA</p><div>Find articles by <a href="/pubmed/?term=Hallman%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">D. M. Hallman</a></div></div><div id="_co_idm139627905627648"><h3 class="no_margin">D. L. Nicolae</h3><p>Department of Human Genetics, University of Chicago, Chicago, IL, USA. Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA. Department of Statistics, University of Chicago, Chicago, IL, USA. Department of Medicine, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Nicolae%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">D. L. Nicolae</a></div></div><div id="_co_idm139627899536096"><h3 class="no_margin">G. I. Bell</h3><p>Department of Human Genetics, University of Chicago, Chicago, IL, USA. Department of Medicine, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Bell%20GI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">G. I. Bell</a></div></div><div id="_co_idm139627899534208"><h3 class="no_margin">M. Cruz</h3><p>Unidad de Investigacion Medica en Bioquimica, Hospital de Especialidades, Centro Medico ‘Siglo XXI’, IMSS, Mexico City, Mexico</p><div>Find articles by <a href="/pubmed/?term=Cruz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">M. Cruz</a></div></div><div id="_co_idm139627939609376"><h3 class="no_margin">N. J. Cox</h3><p>Department of Human Genetics, University of Chicago, Chicago, IL, USA. Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA. Department of Medicine, University of Chicago, Chicago, IL, USA. Section of Genetic Medicine, University of Chicago, KCBD 3220, 900 E 57th Street, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Cox%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">N. J. Cox</a></div></div><div id="_co_idm139627904519456"><h3 class="no_margin">C. L. Hanis</h3><p>Human Genetics Center, University of Texas Health Science Center at Houston, P.O. Box 20186, Houston, TX 77225, USA</p><div>Find articles by <a href="/pubmed/?term=Hanis%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21647700">C. L. Hanis</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139627942078880_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139627942078880_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139627942078880_ai" style="display:none"><div><span class="fm-affl">J. E. Below, </span><span class="fm-affl" id="A1" lang="en"> Department of Human Genetics, University of Chicago, Chicago, IL, USA;</span></div><div><a href="#idm139627942078880aff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div><span class="contrib-email" id="idm139627939609376">N. J. Cox: <a href="mailto:dev@null" data-email="ude.ogacihcu.dsb.enicidem@xocn" class="oemail">ude.ogacihcu.dsb.enicidem@xocn</a>; </span> <span class="contrib-email" id="idm139627904519456">C. L. Hanis: <a href="mailto:dev@null" data-email="ude.cmt.htu@sinaH.L.giarC" class="oemail">ude.cmt.htu@sinaH.L.giarC</a></span> </div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139627942078880_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21647700" target="pmc_ext" ref="reftype=publisher&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Diabetologia</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3761075/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="8" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SD1.</strong></div><div><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD1.xlsx" data-ga-action="click_feat_suppl">NIHMS471860-supplement-SD1.xlsx</a><span style="color:gray"> (15K)</span></div><div class="small guid">GUID:&nbsp;AEB12A0A-4257-4383-AAAE-8540474F3DB9</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SD2.</strong></div><div><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD2.xlsx" data-ga-action="click_feat_suppl">NIHMS471860-supplement-SD2.xlsx</a><span style="color:gray"> (21K)</span></div><div class="small guid">GUID:&nbsp;DFD17887-53BD-4425-A7D8-A20E8A75B662</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SD3.</strong></div><div><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD3.pdf" data-ga-action="click_feat_suppl">NIHMS471860-supplement-SD3.pdf</a><span style="color:gray"> (168K)</span></div><div class="small guid">GUID:&nbsp;21EA1A68-1462-4FF4-88C4-6BE085E0AB10</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SD4.</strong></div><div><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD4.pdf" data-ga-action="click_feat_suppl">NIHMS471860-supplement-SD4.pdf</a><span style="color:gray"> (161K)</span></div><div class="small guid">GUID:&nbsp;25785513-52D6-4FFD-9345-05EDE2498BAA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SD5.</strong></div><div><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD5.pdf" data-ga-action="click_feat_suppl">NIHMS471860-supplement-SD5.pdf</a><span style="color:gray"> (165K)</span></div><div class="small guid">GUID:&nbsp;87FE42CB-68E0-424C-937A-8CA4153C01F7</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SD6.</strong></div><div><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD6.pdf" data-ga-action="click_feat_suppl">NIHMS471860-supplement-SD6.pdf</a><span style="color:gray"> (131K)</span></div><div class="small guid">GUID:&nbsp;1A21B6E2-BAAF-4320-B54E-816BF1069E9C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SD7.</strong></div><div><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD7.pdf" data-ga-action="click_feat_suppl">NIHMS471860-supplement-SD7.pdf</a><span style="color:gray"> (112K)</span></div><div class="small guid">GUID:&nbsp;6DB24537-EB45-42F5-B22F-B3CBE1680C39</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SD8.</strong></div><div><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD8.pdf" data-ga-action="click_feat_suppl">NIHMS471860-supplement-SD8.pdf</a><span style="color:gray"> (145K)</span></div><div class="small guid">GUID:&nbsp;F5F96903-2657-4682-9460-339CFB8DC2A7</div></div></dd></dl></div><div id="idm139627900848240" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139627900848240title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Aims/hypothesis</h3><p id="P1" class="p p-first-last">We conducted genome-wide association studies (GWASs) and expression quantitative trait loci (eQTL) analyses to identify and characterise risk loci for type 2 diabetes in Mexican-Americans from Starr County, TX, USA.</p></div><div id="S2" class="sec"><h3 id="S2title">Method</h3><p id="P2" class="p p-first-last">Using 1.8 million directly interrogated and imputed genotypes in 837 unrelated type 2 diabetes cases and 436 normoglycaemic controls, we conducted Armitage trend tests. To improve power in this population with high disease rates, we also performed ordinal regression including an intermediate class with impaired fasting glucose and/or glucose tolerance. These analyses were followed by meta-analysis with a study of 967 type 2 diabetes cases and 343 normoglycaemic controls from Mexico City, Mexico.</p></div><div id="S3" class="sec"><h3 id="S3title">Result</h3><p id="P3" class="p p-first-last">The top signals (unadjusted <em>p</em> value &lt;1×10<sup>−5</sup>) included 49 single nucleotide polymorphisms (SNPs) in eight gene regions (<em>PER3</em>, <em>PARD3B</em>, <em>EPHA4</em>, <em>TOMM7</em>, <em>PTPRD</em>, <em>HNT</em> [also known as <em>RREB1</em>], <em>LOC729993</em> and <em>IL34</em>) and six intergenic regions. Among these was a missense polymorphism (rs10462020; Gly639Val) in the clock gene <em>PER3</em>, a system recently implicated in diabetes. We also report a second signal (minimum <em>p</em> value 1.52× 10<sup>−6</sup>) within <em>PTPRD</em>, independent of the previously implicated SNP, in a population of Han Chinese. Top meta-analysis signals included known regions <em>HNF1A</em> and <em>KCNQ1</em>. Annotation of top association signals in both studies revealed a marked excess of <em>trans</em>-acting eQTL in both adipose and muscle tissues.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions/Interpretation</h3><p id="P4" class="p p-first-last">In the largest study of type 2 diabetes in Mexican populations to date, we identified modest associations of novel and previously reported SNPs. In addition, in our top signals we report significant excess of SNPs that predict transcript levels in muscle and adipose tissues.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Expression quantitative trait loci, Genome-wide association study, Meta-analysis, Mexican-American, Type 2 diabetes</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first-last">Genome-wide association studies (GWASs) of type 2 diabetes in large samples of recent European descent have identified a number of susceptibility loci; it is widely accepted that, to further understand the genetic aetiology of type 2 diabetes, it will be necessary to replicate these findings and identify new loci using other populations. Mexican-Americans have disproportionately high rates of type 2 diabetes. In order to identify genetic variants contributing to type 2 diabetes susceptibility in Mexican-Americans we conducted both binary trait and ordinal GWA on 1618 individuals from Starr County with diabetes (cases), impaired glucose tolerance (IGT) or impaired fasting glucose (IFG), or normal fasting glucose and glucose tolerance (controls), using nearly 2 million directly interrogated and imputed markers. These analyses were combined in meta-analyses with the results from GWASs of admixed Mexicans from Mexico City [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133751">1</a>]. This sample is described in detail in the companion paper. These cohorts comprise the largest type 2 diabetes analysis in Mexican and Mexican-American samples to date. In addition, we followed our association and meta-analyses by annotating SNPs with expression quantitative trait loci (eQTL) information from lymphoblastoid cell lines (LCLs) as well as adipose and muscle tissues, to characterise possible functional roles for SNPs associated with type 2 diabetes.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Study population and phenotype assessment</h3><p id="P6" class="p p-first">Using the 1979 National Diabetes Data Group criteria [<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133739">2</a>], cases (<em>n</em>=990) and controls (<em>n</em>=990) were selected from genetic and epidemiological resources developed in the Mexican-American population in Starr County, TX, USA (these also meet definitions set out in the 1997 ADA criteria [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133742">3</a>]). Of the 931 successfully genotyped case samples, we selected 837 unrelated individuals for our analyses.</p><p id="P7">A representative type 2 diabetes control sample was selected from a systematic survey of the primary population centres of Starr County conducted from 2002 to 2006. Blocks were randomly selected within primary population centres in Starr County, and all individuals in each household on selected blocks were enumerated. A random individual aged 20 years and above with no history of diabetes was selected from each household for a physical evaluation including an oral glucose tolerance test with a 75 g glucose load. From 1,345 oral glucose tolerance tests, 350 individuals were classified as having diabetes based on their fasting or 2 h post-load glucose, and were excluded from consideration as controls.</p><p id="P8" class="p p-last">From the remaining 995 individuals, 990 were selected for genotyping, and after genotype quality control and pruning related samples, the control sample included 781 unrelated Starr County residents. Of these, 345 individuals had IGT (defined as a blood glucose level of 7.8–11.0 mmol/l after a 2 h oral glucose tolerance test) or impaired fasting glucose (defined as fasting blood glucose level of 5.6–6.9 mmol/l after overnight fast) and were included in the IGT/IFG category as an intermediate phenotype in the ordinal analysis. <a href="/pmc/articles/PMC3761075/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> summarises phenotypes measured in the case, IGT/IFG and control groups.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p24">Characteristics of the study population</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139627906990160" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Characteristic</th><th rowspan="1" colspan="1" valign="bottom" align="left">Cases</th><th rowspan="1" colspan="1" valign="bottom" align="left">IFG/IGT</th><th rowspan="1" colspan="1" valign="bottom" align="left">Controls</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>n</em></td><td rowspan="1" colspan="1" valign="top" align="left">837</td><td rowspan="1" colspan="1" valign="top" align="left">345</td><td rowspan="1" colspan="1" valign="top" align="left">436</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Sex (<em>n</em>, female)</td><td rowspan="1" colspan="1" valign="top" align="left">504</td><td rowspan="1" colspan="1" valign="top" align="left">259</td><td rowspan="1" colspan="1" valign="top" align="left">299</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age (years) (age at diagnosis in cases)</td><td rowspan="1" colspan="1" valign="top" align="left">46.7±10.9</td><td rowspan="1" colspan="1" valign="top" align="left">40.9±10.5</td><td rowspan="1" colspan="1" valign="top" align="left">37.6±9.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">31.8±6.4</td><td rowspan="1" colspan="1" valign="top" align="left">31.0±5.8</td><td rowspan="1" colspan="1" valign="top" align="left">29.5±6.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HbA<sub>1c</sub> (%)</td><td rowspan="1" colspan="1" valign="top" align="left">10.9±3.7</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td></tr></tbody></table></div><div id="largeobj_idm139627906990160" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3761075/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p25" class="p p-first-last">Data are mean ± SD, unless otherwise indicated</p></div><div id="TFN2"><p id="__p26" class="p p-first-last">NA, not available</p></div></div></div></div><div id="S8" class="sec"><h3 id="S8title">Genotyping and quality control</h3><p id="P9" class="p p-first">Genomic DNA was isolated from whole blood of 1,980 unique individuals from Starr County, and quantified by picogreen. The Affymetrix Genome-Wide SNPArray 6.0 assay was performed according to manufacturer protocols (Affymetrix, Santa Clara, CA, USA) at the Center for Inherited Disease Research. Case, IGT/ IFG, and control groups were randomised to plates. Genotypes were called for 1,890 samples passing quality control (QC) using both the Affymetrix supported Birdseed v2 [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133737">4</a>] calling algorithm as well as the corrected robust linear model with maximum likelihood classification (CRLMM) [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133740">5</a>] and only calls in complete agreement between the two algorithms were used in analyses.</p><p id="P10">No samples uniformly failed contrast QC (CQC) and skewness QC (SQC) [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133762">6</a>] in the Affymetrix enzyme-specific subsets as well as the full set of enzyme-specific SNPs. Plate effects were examined [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133735">6</a>] and three plates were identified as having an excess of SNPs with low plate-association <em>p</em> values (132, 413, and 1,080 SNPs with <em>p</em> value &gt; 1×10<sup>−7</sup> on these plates). The plate showing the most severe plate effects was comprised of samples that had failed genotyping previously and had been repeated. Since no systematic bias was apparent from the battery of quality measures and no single set of individuals consistently clustered together in a genotype class in the cluster plots of the most plate-biased SNPs (rather poor clustering and miss-calling were frequently observed in these cluster plots), we chose to recall the data with CRLMM without removing any individuals. Only autosomal genotypes with perfect match calls in both the Birdseed and CRLMM datasets (‘consensus calls’) were carried forward, resulting in a modest reduction in overall call rate (from 0.997 to 0.996 across SNPs and samples). Plate effect analysis was repeated using the consensus calls; the number of SNPs with plate effect <em>p</em> values &lt;1 × 10<sup>−7</sup> for the three plates noted above are 14, 111 and 77, respectively. That a modest plate effect remains in the data may reduce our power to detect true phenotype associations, but the lack of evidence of differential bias between cases and controls suggests that the bias will not increase the false positive rate. We annotated all SNPs with their quality scores from the QC phase, and checked for flags, such as strong association with a plate, in all top signals.</p><p id="P11">Using PLINK (<a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink/</a>) [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133753">7</a>], SNP and individual level call rates were calculated as well as sex incompatibilities (one mismatch identified and removed using markers on the X and Y chromosomes). All individuals passed per sample missingness of &lt;0.1 and 7124 SNPs were removed as a result of missingness &gt;0.1. Identical-by-descent (IBD) estimations were calculated in PLINK and an ‘unrelated’ cohort of 1,618 samples was created with a proportion of pairwise sharing IBD less than 0.28, weighting inclusion of cases and higher call rate. Individual and plate level heterozygosity was estimated in PLINK and no outliers observed. SNPs with minor allele frequency (MAF) &lt;0.01 were removed. Departures from Hardy–Weinberg equilibrium were calculated and flagged in the full sample, cases, and controls.</p><p id="P12" class="p p-last">Major ancestry group components for the unrelated admixed sample in a SNP set pruned of local and long-distance linkage disequilibrium (LD) were calculated in EIGENSTRAT [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133745">8</a>] (<a href="/pmc/articles/PMC3761075/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>); however, neither of the first two principal components were found to be significantly correlated with type 2 diabetes (mean regression <em>p</em> values were 0.18, 0.39, with Pearson correlation coefficients between covariate and phenotype of 0.005 and 0.021, respectively), similar to our previous findings [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133755">9</a>]. Although we did not see significant correlation of type 2 diabetes and European admixture in the Starr County sample, strong correlation was observed in the Mexico City study (<a href="/pmc/articles/PMC3761075/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a>). To prevent excessive false positive rates, all analyses in the Mexico City sample controlled for the distribution along the primary axis of variation. For additional details of covariates in this sample please refer to the companion paper [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_368393112">10</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3761075/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139627897807200" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms471860f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms471860f1.jpg" src="/pmc/articles/PMC3761075/bin/nihms471860f1.jpg"></div></a><div id="largeobj_idm139627897807200" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3761075/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3761075/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3761075/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p20">PCA analysis of (<strong>a</strong>) study population from Starr County, TX, USA, and (<strong>b</strong>) study population from Mexico City. Cases are shown in red, IGT/IFG samples are in green, gluconormal controls are in blue. Both populations analysed together with a number of ancestral populations are shown in (<strong>c</strong>)</p></div></div></div></div><div id="S9" class="sec"><h3 id="S9title">Imputation</h3><p id="P13" class="p p-first-last">We performed imputation in MACH 1.0 [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133730">11</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133760">12</a>] from consensus call genotypes with MAF &gt;0.01 in 1,618 Starr County Mexican-Americans to the full panel of CEU (Centre d’Etude du Polymorphisme [Utah residents with northern and western European ancestry]), YRI (Yoruba in Ibadan, Nigeria), and JPT (Japanese in Tokyo, Japan) + CHB (Han Chinese in Beijing, China) HapMap2 phased polymorphic SNPs from release 22 [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133734">13</a>]. In total 137,532 SNPs with an estimate of the squared correlation between imputed and true genotypes, <em>r<sup>2</sup></em>≤0.7 (as estimated by MACH) were removed. An additional 2,616 SNPs with MAF &lt;0.01 were removed, resulting in 1,829,586 genotyped or imputed markers for analyses.</p></div><div id="S10" class="sec"><h3 id="S10title">Type 2 diabetes association analyses</h3><p id="P14" class="p p-first-last">We used SNPtest [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133759">14</a>] to calculate the Armitage trend test using covariates and the option score, which specifies a missing-data likelihood test. Additionally, the missing-data likelihood test was extended to a three-category (cases, IGT/IFG, and controls) ordered logistic regression using the same covariates, and implemented in R [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133744">15</a>]. We tested the significance of the correlation of covariates in PLINK. In Starr County, 56% of people aged 70 years and older have diabetes, suggesting that substantial numbers of those with non-normal or even normal glucose tolerance will ultimately develop diabetes. Therefore, to generate a more meaningful covariate for age, we used the age at diagnosis (dxage) for cases and the age at enrolment for the controls. In the case–control analysis as well as in the data including the IGT/IFG category, age/ dxage, sex and BMI were significantly correlated with phenotype (mean regression <em>p</em> values across directly interrogated SNPs were &lt;1×10<sup>−34</sup>, &lt;1×10<sup>−4</sup> and &lt;1× 10<sup>−3</sup>, with Pearson correlation coefficients between covariate and ordinal phenotype of 0.36, 0.08 and 0.15 respectively).</p></div><div id="S11" class="sec"><h3 id="S11title">Meta-analysis</h3><p id="P15" class="p p-first-last">Type 2 diabetes association results from SNPtest in Mexicans from Mexico City were combined with results from the associations in the Starr County sample using SNPtest’s companion program META [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133756">16</a>]. We performed principal component analysis (PCA) in EIGENSTRAT on a local and long-distance LD-pruned SNP set in the Mexico City sample as well as together with parental populations, to check homogeneity between the samples (<a href="/pmc/articles/PMC3761075/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b, c</span></a>). Population substructure of the samples within the context of HapMap CEU, YRI, CHB + JPT, and Mexican-American samples from Los Angeles, as well as samples from Bolivians, Peruvians, and Nahua, Mayan, and Spanish populations [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133728">17</a>], was largely overlapping, suggesting the samples are derived from similar parental populations; the Starr County sample exhibited more European ancestry. The score method in SNPtest generates a relative information measure on the scale of (0,1) of the observed statistical information from the imputation for the estimate of allele frequency of the SNPs, and was used to assess confidence in the meta-analysis <em>p</em> values. Reported <em>p</em> values have a combined score statistic ≥0.5, MAF&gt;0.05 and <em>p</em> value &lt;0.05 in each study, and a meta-analysis <em>p</em> value &lt;0.001 showing the same direction of effect.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">eQTL analysis</h3><p id="P16" class="p p-first">The top type 2 diabetes association signals from SNPtest and from the three-category ordered logistic regression were evaluated for their effect on transcript levels in LCLs, muscle, and adipose tissue. The set of eQTLs in lymphocytes among the type 2 diabetes associated SNPs were retrieved from SCAN [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133738">18</a>], which holds the results of eQTL mapping of HapMap SNPs from 90 CEU and 90 YRI samples, to transcriptional expression evaluated in LCLs from these individuals by the Affymetrix GeneChip Human Exon 1.0 ST Array (Affymetrix). For the analyses of muscle and adipose eQTLs, samples were chosen from among 184 study participants for availability of both tissues, and are described in Sharma et al. [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133741">19</a>]. Sixty-two study participants at the tails of the distribution of insulin sensitivity (<em>p</em> value &lt;1×10<sup>−5</sup>) adjusted for age, sex, BMI and per cent fat were genotyped on the Illumina 1M platform (Illumina, San Diego, CA, USA) using standard Illumina protocols. Transcript levels were assayed using Human Whole Genome 4×44 arrays (Agilent Technologies, Santa Clara, CA, USA). eQTL mapping in each tissue was done by testing each marker for the additive effect of allele dosage on normalised probe-level expression intensity. In our annotations, the best proxy SNPs were determined for those type 2 diabetes associated SNPs not directly typed on the Illumina platform used in the eQTL analyses of muscle and adipose tissues.</p><p id="P17" class="p p-last">Using SNP eQTL annotations in LCLs, and adipose and muscle tissues, we conducted simulations to test for an enrichment of eQTLs (<em>p</em> value &lt;1 × 10<sup>−4</sup>) among the top signals (<em>p</em> value &lt;1×10<sup>−3</sup>) associated with type 2 diabetes. We generated 1,000 random SNP sets, each of the same size (<em>n</em>=722) as the original list of LD-pruned top signals (<em>r</em><sup>2</sup>&gt;0.30), containing variants matched on MAF distribution, sampled without replacement from the set of typed SNPs on the Illumina 1M platform; MAF matching was done by drawing from the platform SNPs, which had been grouped into discrete MAF bins, each spanning 5% of the allele frequency spectrum. These simulations (<em>n</em> = 1,000) yield an empirical <em>p</em> value, calculated as the proportion of simulations in which the number of eQTLs exceeds the observed number, <em>Q</em>, in the list of top type 2 diabetes signals.</p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Results</h2><div id="S14" class="sec sec-first-last"><h3 id="S14title">Association and meta-analyses</h3><p id="P18" class="p p-first">All analyses were performed using the combined age/dxage measurement and sex as covariates and association tests were conducted with and without correcting for BMI. We estimated the inflation factors (the ratio of the trimmed means of the observed and expected values), <em>λ</em><sub>age/dxage+sex</sub>=1.034 and <em>λ</em><sub>age/dxage+sex+BMI</sub>=1.026 in the case–control analysis and 1.038 and 1.023 (respectively) in the three-category ordered logistic regression. The Manhattan and quantile–quantile (QQ) plots summarising results from the ordinal logistic regression without BMI as a covariate are presented in <a href="/pmc/articles/PMC3761075/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a, b</span></a> (others are in ESM <a href="/pmc/articles/PMC3761075/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figs 1</span></a>–<a href="/pmc/articles/PMC3761075/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>3</span></a>). Forty-nine high quality SNPs, with an uncorrected <em>p</em> value &lt;1 × 10<sup>−5</sup> in at least one analysis, fell within 14 genomic regions, and are listed, with <em>p</em> values from the analysis, in <a href="/pmc/articles/PMC3761075/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and depicted in <a href="/pmc/articles/PMC3761075/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a> (<em>p</em> values for these markers in each analysis can be found in ESM <a href="/pmc/articles/PMC3761075/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). These top signals fall within eight genic regions: <em>PER3</em>, <em>PARD3B</em>, <em>EPHA4</em>, <em>TOMM7</em>, <em>PTPRD</em>, <em>HNT</em> (also known as <em>RREB1</em>), <em>LOC729993</em> (also known as <em>SHISA9</em>), <em>IL34</em>, and six intergenic regions. In general, the SNPs within these genic and intergenic regions represent single blocks of LD (<em>r</em><sup>2</sup> &gt; 0.7). Aside from the SNPs in <em>PARD3B</em> and <em>EPHA4</em>, which have MAF of about 0.05, these signals are driven by common variation (MAF &gt; 0.1) (<a href="/pmc/articles/PMC3761075/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We calculated heterogeneity in META (<a href="http://www.stats.ox.ac.uk/~jsliu/meta.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.stats.ox.ac.uk/~jsliu/meta.html</a>); we observed no excess of heterogeneity genome-wide and all Starr County top signals have heterogeneity <em>p</em> values &gt;1 × 10<sup>−4</sup> (<a href="/pmc/articles/PMC3761075/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133754">20</a>]. Results for SNPs in LD with previously reported associated type 2 diabetes SNPs from the National Human Genome Research Institute (NHGRI) catalogue [<a href="#R21" rid="R21" class=" bibr popnode">21</a>], Diabetes Genetics Replication and Meta-analysis Consortium (expanded dataset) (DIAGRAM+) [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133752">22</a>], as well as top signals identified in the previous GWAS in Starr County [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133746">9</a>], were examined in detail (ESM <a href="/pmc/articles/PMC3761075/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We also examined our results for variation within <em>CAPN10</em>, previously reported as a type 2 diabetes gene in Mexican-Americans, but key SNPs in the associated <em>CAPN10</em> haplotypes were neither directly interrogated nor imputed.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3761075/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139627905894272" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms471860f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms471860f2.jpg" src="/pmc/articles/PMC3761075/bin/nihms471860f2.jpg"></div></a><div id="largeobj_idm139627905894272" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3761075/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3761075/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3761075/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p21">Manhattan plot of −log<sub>10</sub> (<em>p</em> values) at 1.8 million autosomal SNPs (<strong>a</strong>) and QQ plot of observed vs expected <em>p</em> values (<strong>b</strong>) of results of ordinal regression analysis on type 2 diabetes adjusted for age/dxage and sex</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3761075/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139627903422768" class="figure"><a href="/pmc/articles/PMC3761075/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3761075_nihms471860f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms471860f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3761075/bin/nihms471860f3.jpg"></a></div><div id="largeobj_idm139627903422768" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3761075/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3761075/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p22">Simulated distribution of eQTL counts for SNPs with allele frequencies matched to signals with <em>p</em> value &lt;1 × 10<sup>−3</sup> from the ordinal regression on type 2 diabetes adjusted for age/dxage and sex in the Starr County study population. The black dot represents the actual eQTL count observed in SNPs with <em>p</em> value &lt;1 × 10<sup>−3</sup> in this analysis. Results are shown for transcription prediction in adipose (<strong>a</strong>) and muscle (<strong>b</strong>) tissue</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p27">Summary of top regions from GWAS of type 2 diabetes in Starr County Mexican-Americans</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139627939860640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="left">SNP</th><th rowspan="2" colspan="1" valign="top" align="left">Chr</th><th rowspan="2" colspan="1" valign="top" align="left">Pos</th><th rowspan="2" colspan="1" valign="top" align="left">Risk allele</th><th rowspan="2" colspan="1" valign="top" align="left">Non-risk allele</th><th colspan="7" rowspan="1" valign="bottom" align="left">Ordinal analysis, age- and sex-adjusted
<hr></th><th rowspan="2" colspan="1" valign="top" align="center">Meta-analysis <em>p</em> value</th><th rowspan="2" colspan="1" valign="top" align="left">Gene</th><th rowspan="2" colspan="1" valign="top" align="left">Location</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>p</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">Freq (all)</th><th rowspan="1" colspan="1" valign="top" align="center">Freq (cases)</th><th rowspan="1" colspan="1" valign="top" align="left">Freq (IFG/IGT)</th><th rowspan="1" colspan="1" valign="top" align="left">Freq (controls)</th><th rowspan="1" colspan="1" valign="top" align="left">Imputation r<sup>2</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10462020</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">7803270</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.43 (1.27–1.59)</td><td rowspan="1" colspan="1" valign="top" align="center">8.39×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.83</td><td rowspan="1" colspan="1" valign="top" align="center">0.85</td><td rowspan="1" colspan="1" valign="top" align="left">0.8</td><td rowspan="1" colspan="1" valign="top" align="left">0.8</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="center">8.57×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>PER3</em></td><td rowspan="1" colspan="1" valign="top" align="left">Missense</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6728803</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">66001970</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.38 (1.26–1.51)</td><td rowspan="1" colspan="1" valign="top" align="center">6.80×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.52</td><td rowspan="1" colspan="1" valign="top" align="center">0.56</td><td rowspan="1" colspan="1" valign="top" align="left">0.5</td><td rowspan="1" colspan="1" valign="top" align="left">0.48</td><td rowspan="1" colspan="1" valign="top" align="left">0.83</td><td rowspan="1" colspan="1" valign="top" align="center">9.16×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>KRT18P33- LOC729348</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs849230</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">205969274</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.83 (1.57–2.1)</td><td rowspan="1" colspan="1" valign="top" align="center">6.86×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">0.95</td><td rowspan="1" colspan="1" valign="top" align="left">0.92</td><td rowspan="1" colspan="1" valign="top" align="left">0.91</td><td rowspan="1" colspan="1" valign="top" align="left">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PARD3B</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1879976</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">149826207</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.34 (1.21–1.47)</td><td rowspan="1" colspan="1" valign="top" align="center">8.91×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.59</td><td rowspan="1" colspan="1" valign="top" align="center">0.61</td><td rowspan="1" colspan="1" valign="top" align="left">0.59</td><td rowspan="1" colspan="1" valign="top" align="left">0.54</td><td rowspan="1" colspan="1" valign="top" align="left">0.9</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC344741- RPL38P1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2240727</td><td rowspan="1" colspan="1" valign="top" align="left">7</td><td rowspan="1" colspan="1" valign="top" align="left">22819037</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (1.24–1.5)</td><td rowspan="1" colspan="1" valign="top" align="center">2.79×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="left">0.29</td><td rowspan="1" colspan="1" valign="top" align="left">0.31</td><td rowspan="1" colspan="1" valign="top" align="left">0.9</td><td rowspan="1" colspan="1" valign="top" align="center">7.72×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TOMM7</em></td><td rowspan="1" colspan="1" valign="top" align="left">3′ UTR</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4766119</td><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">3419017</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.65 (1.44–1.85)</td><td rowspan="1" colspan="1" valign="top" align="center">2.65×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="left">0.85</td><td rowspan="1" colspan="1" valign="top" align="left">0.84</td><td rowspan="1" colspan="1" valign="top" align="left">0.72</td><td rowspan="1" colspan="1" valign="top" align="center">1.25×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC100128253- LOC100129223</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs149228</td><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">13211975</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.43 (1.28–1.57)</td><td rowspan="1" colspan="1" valign="top" align="center">1.49×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">0.2</td><td rowspan="1" colspan="1" valign="top" align="left">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">1.61×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC729993</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4522658</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">134203633</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.28 (1.17–1.4)</td><td rowspan="1" colspan="1" valign="top" align="center">3.40×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">0.46</td><td rowspan="1" colspan="1" valign="top" align="left">0.42</td><td rowspan="1" colspan="1" valign="top" align="left">0.95</td><td rowspan="1" colspan="1" valign="top" align="center">1.27×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>NAP5-MGAT5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16862811</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">222109330</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.8 (1.53–2.08)</td><td rowspan="1" colspan="1" valign="top" align="center">2.65×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.95</td><td rowspan="1" colspan="1" valign="top" align="left">0.95</td><td rowspan="1" colspan="1" valign="top" align="left">0.91</td><td rowspan="1" colspan="1" valign="top" align="left">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">4.12×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>EPHA4</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10192201</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">237468147</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">1.26 (1.13–1.4)</td><td rowspan="1" colspan="1" valign="top" align="center">4.44×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="left">0.67</td><td rowspan="1" colspan="1" valign="top" align="left">0.58</td><td rowspan="1" colspan="1" valign="top" align="left">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">1.54×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC100128709- LOC93463</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs649891</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">10420602</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.29 (1.16–1.43)</td><td rowspan="1" colspan="1" valign="top" align="center">1.69×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="left">0.44</td><td rowspan="1" colspan="1" valign="top" align="left">0.35</td><td rowspan="1" colspan="1" valign="top" align="left">0.78</td><td rowspan="1" colspan="1" valign="top" align="center">5.80×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>PTPRD</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3099797</td><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">131626596</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.3 (1.18–1.43)</td><td rowspan="1" colspan="1" valign="top" align="center">4.60×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.67</td><td rowspan="1" colspan="1" valign="top" align="left">0.63</td><td rowspan="1" colspan="1" valign="top" align="left">0.58</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="center">3.38×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>HNT</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8033124</td><td rowspan="1" colspan="1" valign="top" align="left">15</td><td rowspan="1" colspan="1" valign="top" align="left">89712478</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.3 (1.16–1.44)</td><td rowspan="1" colspan="1" valign="top" align="center">1.83×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="left">0.44</td><td rowspan="1" colspan="1" valign="top" align="left">0.37</td><td rowspan="1" colspan="1" valign="top" align="left">0.74</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC100128403- LOC441732</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3813905</td><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">69238351</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.41 (1.25–1.57)</td><td rowspan="1" colspan="1" valign="top" align="center">3.35×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td><td rowspan="1" colspan="1" valign="top" align="center">0.77</td><td rowspan="1" colspan="1" valign="top" align="left">0.73</td><td rowspan="1" colspan="1" valign="top" align="left">0.73</td><td rowspan="1" colspan="1" valign="top" align="left">0.73</td><td rowspan="1" colspan="1" valign="top" align="center">9.73×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>IL34</em></td><td rowspan="1" colspan="1" valign="top" align="left">5′ UTR</td></tr></tbody></table></div><div id="largeobj_idm139627939860640" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3761075/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="__p28" class="p p-first-last">Representative SNPs from each genomic region with <em>p</em> values below 1 × 10<sup>−5</sup> in at least one analysis (regression on the ordinal or binary phenotype, with/without covariate BMI). Reported statistics are derived from the ordinal regression on type 2 diabetes adjusted for age/dxage and sex. Reported meta-analysis <em>p</em> values are derived from the binary case–control analysis combined with the analogous analysis in the Mexico City dataset. Any evidence for association is driven solely by the Starr County study, as all <em>p</em> values in the Mexico City sample for these SNPs were &gt;0.15 (SNPs in the table without high quality Mexico City genotypes are labelled with a dash); <em>p</em> values for heterogeneity are greater than 1×10<sup>−4</sup> for all meta-analysed SNPs.</p></div><div id="TFN4"><p id="__p29" class="p p-first-last">Chr, chromosome; Freq, frequency; Pos, position; UTR, untranslated region</p></div></div></div><p id="P19">The top meta-analysis signals fell within and near the genes <em>CSMD1</em> and <em>HNF1A</em>; additional signals implicated <em>LSAMP</em>, <em>FGF12</em>, <em>RP11-354K1.1</em>, <em>KCNQ1</em>, <em>ANK2</em>, <em>NIPAL2</em>, <em>RP11-74C3.1</em>, <em>CIT</em>, <em>C22orf30</em> (also known as <em>PRR14L</em>), <em>EPHB2</em>, <em>MCPH1</em> and <em>DEPDC5.</em> Follow-up meta-analysis of top signals from the combined Mexican/Mexican-American studies with the recent DIAGRAM+ dataset identified genome-wide significant signals (<em>p</em> value &lt;5×10<sup>−8</sup>) in or near susceptibility genes <em>HNF1A</em> and <em>CDKN2A/CDKN2B</em>, and suggestive evidence for <em>IGF2BP2</em>, <em>KCNQ1</em> and the previously unreported <em>C14orf70.</em> For additional details of the meta-analyses of these samples and follow-up of top signals in DIAGRAM+, please refer to the companion paper.</p><p id="P20" class="p p-last">In addition to physical proximity, we considered the roles that SNPs in the top signals play in transcript prediction. Annotation of LD-pruned top signals (<em>p</em> value &lt;1×10<sup>−3</sup>, <em>r</em><sup>2</sup>&lt;0.3) from the ordinal regression analysis in Starr County, with sex and age/dxage as covariates, identified 258 SNPs predicting transcript levels in LCLs, 289 in muscle, and 359 in adipose tissue. The mean number of frequency matched SNPs that predict transcript levels in LCLs in 1,000 simulations was 252 (SD 12.17), suggesting no enrichment. However, the maximum number of muscle and adipose eQTLs observed in 1,000 simulations was 173 and 193, respectively. The distributions of eQTL counts established by simulations, as well as the actual observed counts in muscle and adipose tissues, are depicted in <a href="/pmc/articles/PMC3761075/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3a, b</span></a> (see ESM <a href="/pmc/articles/PMC3761075/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a> for LCLs). Notably, nearly all of the enrichment in muscle and adipose tissue is driven by <em>trans</em>-acting eQTLs; only three of the eQTLs observed in muscle, and eight in adipose tissue, were <em>cis</em>-acting. In addition, we replicated the observed enrichment in muscle and adipose tissues in the top signals (defined as described above) from the analysis in the Mexico City sample (<a href="/pmc/articles/PMC3761075/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4a, b</span></a>). A number of SNPs showing association for type 2 diabetes in our studies predicted gene expression across multiple tissues; 14 SNPs predict expression in all three tissues tested (ESM Fig. 5).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3761075/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139627902726080" class="figure"><a href="/pmc/articles/PMC3761075/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3761075_nihms471860f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms471860f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3761075/bin/nihms471860f4.jpg"></a></div><div id="largeobj_idm139627902726080" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3761075/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3761075/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><p id="__p23">Simulated distribution of eQTL counts for SNPs with allele frequencies matched to signals with <em>p</em> value &lt;1 × 10<sup>−3</sup> from the ordinal regression on type 2 diabetes adjusted for age/dxage and sex in the Mexico City study population. The black dot represents the actual eQTL count observed in SNPs with <em>p</em> value &lt;1 × 10<sup>−3</sup> in this analysis. Results are shown for transcription prediction in adipose (<strong>a</strong>) and muscle (<strong>b</strong>) tissue</p></div></div></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Discussion</h2><p id="P21" class="p p-first">As studies of genetic association in type 2 diabetes are pursued in non-European populations, it is important to recognise that, in populations with markedly increased disease rates, there may be room to improve on the case–control study design. In this case, we favour an approach that uses more information and recognises the staging of the disease, enabling us to recover a substantial number of individuals. In general, the results of the ordinal and binary analyses are very similar, but including the intermediate class allows us to include more of the individuals in the sample, and should, therefore, provide a more complete picture of the association of genetic variation with the spectrum of phenotypes related to type 2 diabetes. Among the top signals in the ordinal analysis is a functional missense polymorphism in the clock gene <em>PER3.</em> Substantial research has implicated circadian clock function in metabolic diseases, coronary function and obesity [<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133748">23</a>–<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133729">25</a>], but this signal is not replicated in the Mexico City sample (<em>p</em> value ~0.78).</p><p id="P22">Another top signal from the Starr County sample, as well as in the meta-analysis with Mexico City, implicates a previously reported type 2 diabetes risk gene <em>PTPRD. PTPRD</em>, protein tyrosine phosphatase receptor type delta gene [<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133749">26</a>], was recently identified by GWAS in Han Chinese populations [<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133747">26</a>]. Although the SNPs are both intronic, our result is 1.6 Mb downstream and represents an independent signal from within the gene (the SNPs are in linkage equilibrium in HapMap CHB + JPT as well as CEU samples). The role of <em>PTPRD</em> in type 2 diabetes is not yet understood; however, when ablated in mice, genes in this subfamily induce defects in glucose homeostasis and insulin sensitivity [<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133750">27</a>, <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133732">28</a>].</p><p id="P23">There is little literature suggesting roles for <em>IL34</em>, <em>TOMM7</em>, <em>HNT</em> or <em>PARD3B</em>, nor the intergenic regions, in diabetes; however, in HapMap Europeans, LD extends from several of these top regions to include candidate genes for type 2 diabetes. For example, two intergenic SNPs on chromosome 12 are in high LD with <em>PRMT8</em> (<em>r</em><sup>2</sup>=0.764), a gene known to influence lipid levels [<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133733">29</a>]. Yet, LD with a type 2 diabetes-related gene alone is not sufficient evidence of a SNP’s functional role. To assess functional consequences of associated variants, we explored our findings in the context of gene regulation.</p><p id="P24">Previous studies have identified enrichment of eQTLs in trait associated SNPs [<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133736">30</a>]. A recent study into the functional role of type 2 diabetes associated SNPs found an excess of eQTLs among signals from the Wellcome Trust Case Control Consortium and the Diabetes Genetics Initiative in muscle and adipose [<a href="#R31" rid="R31" class=" bibr popnode">31</a>, <a href="#R32" rid="R32" class=" bibr popnode">32</a>]. The enrichment in these cases was driven almost exclusively by <em>trans</em>-acting eQTLs. This enrichment was robust to thresholds of eQTL <em>p</em> value (1 × 10<sup>−6</sup> was also considered and a similar level of enrichment is still observed) and threshold of significance of type 2 diabetes association (top 1,000 and top 10,000 signals were analysed and enrichment was seen in both). We find similar enrichment in both the Starr County and Mexico City datasets. The lack of enrichment of eQTLs in LCLs, despite significant enrichment of eQTLs from muscle and adipose tissue, is likely to be attributable to the fact that LCLs as a cell type are not as relevant as muscle and adipose tissue to type 2 diabetes risk, and only a proportion of eQTLs are shared across tissues [<a href="#R31" rid="R31" class=" bibr popnode">31</a>, <a href="#R32" rid="R32" class=" bibr popnode">32</a>]. The number of SNPs predicting transcript level in both muscle and adipose tissue is markedly higher (178) than those overlapping LCLs (33), suggesting concordance of regulatory action in muscle and adipose tissues in a large proportion of top type 2 diabetes associated SNPs in our study. Previous studies of eQTLs in LCLs in top type 1 diabetes associated loci have found significant enrichment [<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133757">30</a>], suggesting that lymphoblasts are a relevant tissue for type 1 diabetes; this is reassuring given the role of autoimmunity in the disease.</p><p id="P25">Of the 53 previously reported SNPs considered (drawn from the NHGRI catalogue, DIAGRAM+, and Hayes et al. [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133743">9</a>], and interrogated or represented by proxy SNP in the Starr County analyses) we identified seven markers with <em>p</em> value &lt;0.05 and odds ratios in the same direction. These markers are in or near <em>IGF2BP2</em>, <em>WFS1</em>, <em>JAZF1</em>, <em>OR13D1-OR13D3P</em>, <em>KCNQ1</em>, <em>VPS33B</em>, and <em>EPB41L3-LOC645355</em>. An additional 20 SNPs do not reach marginal significance in our dataset, but do show odds ratios in the same direction. Seventeen SNPs have previously reported confidence intervals for the odds ratios which overlap those observed in the Starr County dataset (all data shown in ESM <a href="/pmc/articles/PMC3761075/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P26">As in all GWASs of type 2 diabetes published to date, we lack power to detect the kinds of small effect we now know are likely to contribute to disease risk (see ESM Fig. 6). However, the absence of support for a number of established type 2 diabetes genes is notable in this dataset. Of particular interest is the lack of association of SNPs within <em>TCF7L2</em>, the gene with the strongest associations with type 2 diabetes observed in Europeans. While the allele frequency of the variant most strongly associated in Europeans is lower in Asian populations, estimates of the odds ratios were quite similar. In contrast, the frequency of the risk allele is similar in Mexican-Americans and Europeans, but the odds ratios are lower in both the Mexican-American and Mexico City datasets; in a fixed effects meta-analysis of SNP rs7903146 in our data and that of Voight et al. [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133761">22</a>], the heterogeneity <em>p</em> value is 0.05. Our confidence intervals for rs7903146 also overlap, but just barely, according to DIAGRAM estimates.</p><p id="P27">The winner’s curse, the tendency of a biased overestimate of effect in the first study to report an association and be successfully published, may explain the somewhat higher overestimates of effect in the top signals in the Starr County and Mexico City studies [<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_331133731">33</a>]. In addition, ascertainment can contribute to these effects. For example, in Starr County, patients from families with two or more affected members were specifically targeted for the GWAS, which improves power, but leads to estimates of effect size that are larger than would be estimated in an unselected case series. In general, however, lack of replication is likely to be due to the fact that all studies of type 2 diabetes to date are under-powered to detect small effects, and therefore only detect association signals from some (probably different) subset of the contributing variants.</p><p id="P28" class="p p-last">To identify, confirm, and characterise additional genetic risk factors for type 2 diabetes it is likely that we will need to continue studies in non-European populations, to explore new study designs and methods of analysis in high-risk groups, and to enhance GWAS with eQTL and other functional studies.</p></div><div id="S16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>SD1</h4><div class="sup-box half_rhythm" id="d36e873"><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD1.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(15K, xlsx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>SD2</h4><div class="sup-box half_rhythm" id="d36e877"><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(21K, xlsx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>SD3</h4><div class="sup-box half_rhythm" id="d36e881"><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD3.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(168K, pdf)</sup></div></div><div class="sec suppmat" id="SD4"><h4>SD4</h4><div class="sup-box half_rhythm" id="d36e885"><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD4.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(161K, pdf)</sup></div></div><div class="sec suppmat" id="SD5"><h4>SD5</h4><div class="sup-box half_rhythm" id="d36e889"><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD5.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(165K, pdf)</sup></div></div><div class="sec suppmat" id="SD6"><h4>SD6</h4><div class="sup-box half_rhythm" id="d36e893"><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD6.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(131K, pdf)</sup></div></div><div class="sec suppmat" id="SD7"><h4>SD7</h4><div class="sup-box half_rhythm" id="d36e897"><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD7.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(112K, pdf)</sup></div></div><div class="sec suppmat" id="SD8"><h4>SD8</h4><div class="sup-box half_rhythm" id="d36e901"><a href="/pmc/articles/PMC3761075/bin/NIHMS471860-supplement-SD8.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(145K, pdf)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p1">Supported in part by <a id="__tag_811191669" class="tag_hotlink" href="/nuccore/DK073451" ref="/nuccore/DK073451"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DK073451","term_id":"187395899","term_text":"DK073451"}}</span>DK073451</a>, <a id="__tag_811191670" class="tag_hotlink" href="/nuccore/HL084715" ref="/nuccore/HL084715"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL084715","term_id":"1051655123","term_text":"HL084715"}}</span>HL084715</a>, and DK20595, the FOFOI IMSS-2004/014, 2005/2/I/363, 2006/1B/1/002, Fundacion IMSS A. C., and Fundacion Gonzalo Rio Arronte I. A. P. Mexico. Also supported in part by National Institutes of Health grant T32GM007197 and grant CTSA T32 UL1 RR024999. M. Cruz is a recipient of a Fundación IMSS Scholarship, Mexico. We also gratefully acknowledge the willing participation of the residents of Starr County and the efforts of the field staff in Starr County responsible for the collection of these data. Genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract number HHSN268200782096C.</p></div></div><div id="GL" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="GLtitle">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">CQC</td><td>Contrast quality control</td></tr><tr><td id="G2">CRLMM</td><td>Corrected robust linear model with maximum likelihood classification</td></tr><tr><td id="G3">DIAGRAM+</td><td>Diabetes Genetics Replication and Meta-analysis Consortium (expanded dataset)</td></tr><tr><td id="G4">eQTL</td><td>Expression quantitative trait loci</td></tr><tr><td id="G5">GWAS</td><td>Genome-wide association study</td></tr><tr><td id="G6">IBD</td><td>Identical-by-descent</td></tr><tr><td id="G7">IFG</td><td>Impaired fasting glucose</td></tr><tr><td id="G8">IGT</td><td>Impaired glucose tolerance</td></tr><tr><td id="G9">LCL</td><td>Lymphoblastoid cell lines</td></tr><tr><td id="G10">LD</td><td>Linkage disequilibrium</td></tr><tr><td id="G11">MAF</td><td>Minor allele frequency</td></tr><tr><td id="G12">NHGRI</td><td>National Human Genome Research Institute</td></tr><tr><td id="G13">PCA</td><td>Principal component analysis</td></tr><tr><td id="G14">QC</td><td>Quality control</td></tr><tr><td id="G15">SNP</td><td>Single nucleotide polymorphism</td></tr><tr><td id="G16">SQC</td><td>Skewness quality control</td></tr></tbody></table></div></div></div><div id="idm139627905146288" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139627905146288title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="__p2" class="p p-first-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Electronic supplementary material</a> The online version of this article (doi:10.1007/s00125-011-2188-3) contains <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary material</a>, which is available to authorised users.</p><p></p><p class="fn sec" id="FN2"> </p><p id="__p3" class="p p-first-last"><strong>Duality of interest</strong> The authors declare that there is no duality of interest associated with this manuscript.</p><p></p></div></div><div id="idm139627942078880aff-info" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139627942078880aff-infotitle">Contributor Information</h2><p><span class="fm-affl">J. E. Below, </span><span class="fm-affl" id="A1" lang="en"> Department of Human Genetics, University of Chicago, Chicago, IL, USA.</span></p><p><span class="fm-affl">E. R. Gamazon, </span><span class="fm-affl" id="A2" lang="en"> Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA.</span></p><p><span class="fm-affl">J. V. Morrison, </span><span class="fm-affl" id="A3" lang="en"> Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA.</span></p><p><span class="fm-affl">A. Konkashbaev, </span><span class="fm-affl" id="A4" lang="en"> Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA.</span></p><p><span class="fm-affl">A. Pluzhnikov, </span><span class="fm-affl" id="A5" lang="en"> Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA.</span></p><p><span class="fm-affl">P. M. McKeigue, </span><span class="fm-affl" id="A6" lang="en"> Public Health Sciences Section, Division of Community Health Sciences, University of Edinburgh Medical School, Edinburgh, UK.</span></p><p><span class="fm-affl">E. J. Parra, </span><span class="fm-affl" id="A7" lang="en"> Department of Anthropology, University of Toronto, Mississauga, ON, Canada.</span></p><p><span class="fm-affl">S. C. Elbein, </span><span class="fm-affl" id="A8" lang="en"> Section on Endocrinology and Metabolism, Wake Forest University School of Medicine, Winston-Salem, NC, USA.</span></p><p><span class="fm-affl">D. M. Hallman, </span><span class="fm-affl" id="A9" lang="en"> Human Genetics Center, University of Texas Health Science Center at Houston, P.O. Box 20186, Houston, TX 77225, USA.</span></p><p><span class="fm-affl">D. L. Nicolae, </span><span class="fm-affl" id="A10" lang="en"> Department of Human Genetics, University of Chicago, Chicago, IL, USA. Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA. Department of Statistics, University of Chicago, Chicago, IL, USA. Department of Medicine, University of Chicago, Chicago, IL, USA.</span></p><p><span class="fm-affl">G. I. Bell, </span><span class="fm-affl" id="A11" lang="en"> Department of Human Genetics, University of Chicago, Chicago, IL, USA. Department of Medicine, University of Chicago, Chicago, IL, USA.</span></p><p><span class="fm-affl">M. Cruz, </span><span class="fm-affl" id="A12" lang="en"> Unidad de Investigacion Medica en Bioquimica, Hospital de Especialidades, Centro Medico ‘Siglo XXI’, IMSS, Mexico City, Mexico.</span></p><p><span class="fm-affl">N. J. Cox, </span><span class="fm-affl" id="A13" lang="en"> Department of Human Genetics, University of Chicago, Chicago, IL, USA. Section of Genetic Medicine, KCBD 3220, University of Chicago, 900 E 57th Street, Chicago, IL 60637, USA. Department of Medicine, University of Chicago, Chicago, IL, USA. Section of Genetic Medicine, University of Chicago, KCBD 3220, 900 E 57th Street, Chicago, IL 60637, USA.</span></p><p><span class="fm-affl">C. L. Hanis, </span><span class="fm-affl" id="A14" lang="en"> Human Genetics Center, University of Texas Health Science Center at Houston, P.O. Box 20186, Houston, TX 77225, USA.</span></p></div><div id="idm139627903348224" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139627903348224title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Martinez-Marignac VL, Valladares A, Cameron E, et al.  Admixture in Mexico City: implications for admixture mapping of type 2 diabetes genetic risk factors. <span><span class="ref-journal">Hum Genet. </span>2007;<span class="ref-vol">120</span>:807–819.</span> [<a href="/pubmed/17066296" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Admixture+in+Mexico+City:+implications+for+admixture+mapping+of+type+2+diabetes+genetic+risk+factors&amp;author=VL+Martinez-Marignac&amp;author=A+Valladares&amp;author=E+Cameron&amp;volume=120&amp;publication_year=2007&amp;pages=807-819&amp;pmid=17066296&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">National Diabetes Data Group. Classification and diagnosis of diabetes mellitus and other categories of glucose intolerance. <span><span class="ref-journal">Diabetes. </span>1979;<span class="ref-vol">28</span>:1039–1057.</span> [<a href="/pubmed/510803" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Classification+and+diagnosis+of+diabetes+mellitus+and+other+categories+of+glucose+intolerance&amp;volume=28&amp;publication_year=1979&amp;pages=1039-1057&amp;pmid=510803&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">The Expert Committee on the Diagnosis and Classification of Diabetes Mellitus . Report of the Expert Committee on the Diagnosis and Classification of Diabetes Mellitus. <span><span class="ref-journal">Diabetes Care. </span>1997;<span class="ref-vol">20</span>:1183–1197.</span> [<a href="/pubmed/9203460" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes+Care&amp;title=Report+of+the+Expert+Committee+on+the+Diagnosis+and+Classification+of+Diabetes+Mellitus&amp;volume=20&amp;publication_year=1997&amp;pages=1183-1197&amp;pmid=9203460&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Korn JM, Kuruvilla FG, McCarroll SA, et al.  Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1253–1260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=JM+Korn&amp;author=FG+Kuruvilla&amp;author=SA+McCarroll&amp;volume=40&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Carvalho B, Bengtsson H, Speed TP, Irizarry RA. Exploration, normalization, and genotype calls of high-density oligonucleotide SNP array data. <span><span class="ref-journal">Biostatistics. </span>2007;<span class="ref-vol">8</span>:485–499.</span> [<a href="/pubmed/17189563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biostatistics&amp;title=Exploration,+normalization,+and+genotype+calls+of+high-density+oligonucleotide+SNP+array+data&amp;author=B+Carvalho&amp;author=H+Bengtsson&amp;author=TP+Speed&amp;author=RA+Irizarry&amp;volume=8&amp;publication_year=2007&amp;pages=485-499&amp;pmid=17189563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Pluzhnikov A, Below JE, Konkashbaev A, et al.  Spoiling the whole bunch: quality control aimed at preserving the integrity of high-throughput genotyping. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:123–128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896766/">PMC free article</a>]</span> [<a href="/pubmed/20598280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Spoiling+the+whole+bunch:+quality+control+aimed+at+preserving+the+integrity+of+high-throughput+genotyping&amp;author=A+Pluzhnikov&amp;author=JE+Below&amp;author=A+Konkashbaev&amp;volume=87&amp;publication_year=2010&amp;pages=123-128&amp;pmid=20598280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Hayes MG, Pluzhnikov A, Miyake K, et al.  Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3033–3044.</span> [<a href="/pubmed/17846124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+type+2+diabetes+genes+in+Mexican+Americans+through+genome-wide+association+studies&amp;author=MG+Hayes&amp;author=A+Pluzhnikov&amp;author=K+Miyake&amp;volume=56&amp;publication_year=2007&amp;pages=3033-3044&amp;pmid=17846124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Parra EJ, Below JE, Krithika S, et al.  Genome-wide association study of type 2 diabetes in a sample from Mexico City and meta-analysis with a Mexican American sample from Starr County, TX. <span><span class="ref-journal">Diabetologia. </span>2011 doi: 10.1007/s00125-011-2172-y.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3818640/">PMC free article</a>]</span> [<a href="/pubmed/21573907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1007%2Fs00125-011-2172-y" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Genome-wide+association+study+of+type+2+diabetes+in+a+sample+from+Mexico+City+and+meta-analysis+with+a+Mexican+American+sample+from+Starr+County,+TX&amp;author=EJ+Parra&amp;author=JE+Below&amp;author=S+Krithika&amp;publication_year=2011&amp;doi=10.1007/s00125-011-2172-y&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2006;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;author=P+Scheet&amp;author=GR+Abecasis&amp;volume=34&amp;publication_year=2006&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Huang L, Li Y, Singleton AB, et al.  Genotype-imputation accuracy across worldwide human populations. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:235–250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668016/">PMC free article</a>]</span> [<a href="/pubmed/19215730" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genotype-imputation+accuracy+across+worldwide+human+populations&amp;author=L+Huang&amp;author=Y+Li&amp;author=AB+Singleton&amp;volume=84&amp;publication_year=2009&amp;pages=235-250&amp;pmid=19215730&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Valenta Z, Pitha J, Poledne R. Proportional odds logistic regression—effective means of dealing with limited uncertainty in dichotomizing clinical outcomes. <span><span class="ref-journal">Stat Med. </span>2006;<span class="ref-vol">25</span>:4227–4234.</span> [<a href="/pubmed/16929469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Proportional+odds+logistic+regression—effective+means+of+dealing+with+limited+uncertainty+in+dichotomizing+clinical+outcomes&amp;author=Z+Valenta&amp;author=J+Pitha&amp;author=R+Poledne&amp;volume=25&amp;publication_year=2006&amp;pages=4227-4234&amp;pmid=16929469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Mao X, Bigham AW, Mei R, et al.  A genomewide admixture mapping panel for Hispanic/Latino populations. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:1171–1178.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1867104/">PMC free article</a>]</span> [<a href="/pubmed/17503334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genomewide+admixture+mapping+panel+for+Hispanic/Latino+populations&amp;author=X+Mao&amp;author=AW+Bigham&amp;author=R+Mei&amp;volume=80&amp;publication_year=2007&amp;pages=1171-1178&amp;pmid=17503334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Gamazon ER, Zhang W, Konkashbaev A, et al.  SCAN: SNP and copy number annotation. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:259–262.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852202/">PMC free article</a>]</span> [<a href="/pubmed/19933162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SCAN:+SNP+and+copy+number+annotation&amp;author=ER+Gamazon&amp;author=W+Zhang&amp;author=A+Konkashbaev&amp;volume=26&amp;publication_year=2010&amp;pages=259-262&amp;pmid=19933162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Sharma NK, Das SK, Mondal AK, et al.  Endoplasmic reticulum stress markers are associated with obesity in nondiabetic subjects. <span><span class="ref-journal">J Clin Endocrinol Metab. </span>2008;<span class="ref-vol">93</span>:4532–4541.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2582561/">PMC free article</a>]</span> [<a href="/pubmed/18728164" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Endocrinol+Metab&amp;title=Endoplasmic+reticulum+stress+markers+are+associated+with+obesity+in+nondiabetic+subjects&amp;author=NK+Sharma&amp;author=SK+Das&amp;author=AK+Mondal&amp;volume=93&amp;publication_year=2008&amp;pages=4532-4541&amp;pmid=18728164&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Zeggini E, Scott LJ, Saxena R, et al.  Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:638–645.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=LJ+Scott&amp;author=R+Saxena&amp;volume=40&amp;publication_year=2008&amp;pages=638-645&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Hindorff LA, Junkins HA, Hall PN, Mehta JP, Manolio TA.  [Accessed 6 April 2011];<span><span class="ref-journal">A catalog of published genome-wide association studies. </span>2011 </span>  <a href="http://www.genome.gov/gwastudies" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.genome.gov/gwastudies</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Voight BF, Scott LJ, Steinthorsdottir V, et al.  Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:579–589.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080658/">PMC free article</a>]</span> [<a href="/pubmed/20581827" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Twelve+type+2+diabetes+susceptibility+loci+identified+through+large-scale+association+analysis&amp;author=BF+Voight&amp;author=LJ+Scott&amp;author=V+Steinthorsdottir&amp;volume=42&amp;publication_year=2010&amp;pages=579-589&amp;pmid=20581827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Young ME, Wilson CR, Razeghi P, Guthrie PH, Taegtmeyer H. Alterations of the circadian clock in the heart by streptozotocin-induced diabetes. <span><span class="ref-journal">J Mol Cell Cardiol. </span>2002;<span class="ref-vol">34</span>:223–231.</span> [<a href="/pubmed/11851361" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Cell+Cardiol&amp;title=Alterations+of+the+circadian+clock+in+the+heart+by+streptozotocin-induced+diabetes&amp;author=ME+Young&amp;author=CR+Wilson&amp;author=P+Razeghi&amp;author=PH+Guthrie&amp;author=H+Taegtmeyer&amp;volume=34&amp;publication_year=2002&amp;pages=223-231&amp;pmid=11851361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Maury E, Ramsey KM, Bass J. Circadian rhythms and metabolic syndrome: from experimental genetics to human disease. <span><span class="ref-journal">Circ Res. </span>2010;<span class="ref-vol">106</span>:447–462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2837358/">PMC free article</a>]</span> [<a href="/pubmed/20167942" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Circadian+rhythms+and+metabolic+syndrome:+from+experimental+genetics+to+human+disease&amp;author=E+Maury&amp;author=KM+Ramsey&amp;author=J+Bass&amp;volume=106&amp;publication_year=2010&amp;pages=447-462&amp;pmid=20167942&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Marcheva B, Ramsey KM, Buhr ED, et al.  Disruption of the clock components CLOCK and BMAL1 leads to hypoinsulinaemia and diabetes. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:627–631.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2920067/">PMC free article</a>]</span> [<a href="/pubmed/20562852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Disruption+of+the+clock+components+CLOCK+and+BMAL1+leads+to+hypoinsulinaemia+and+diabetes&amp;author=B+Marcheva&amp;author=KM+Ramsey&amp;author=ED+Buhr&amp;volume=466&amp;publication_year=2010&amp;pages=627-631&amp;pmid=20562852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Tsai FJ, Yang CF, Chen CC, et al.  A genome-wide association study identifies susceptibility variants for type 2 diabetes in Han Chinese. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000847.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2824763/">PMC free article</a>]</span> [<a href="/pubmed/20174558" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+identifies+susceptibility+variants+for+type+2+diabetes+in+Han+Chinese&amp;author=FJ+Tsai&amp;author=CF+Yang&amp;author=CC+Chen&amp;volume=6&amp;publication_year=2010&amp;pages=e1000847&amp;pmid=20174558&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Ren JM, Li PM, Zhang WR, et al.  Transgenic mice deficient in the LAR protein-tyrosine phosphatase exhibit profound defects in glucose homeostasis. <span><span class="ref-journal">Diabetes. </span>1998;<span class="ref-vol">47</span>:493–497.</span> [<a href="/pubmed/9519761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Transgenic+mice+deficient+in+the+LAR+protein-tyrosine+phosphatase+exhibit+profound+defects+in+glucose+homeostasis&amp;author=JM+Ren&amp;author=PM+Li&amp;author=WR+Zhang&amp;volume=47&amp;publication_year=1998&amp;pages=493-497&amp;pmid=9519761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Chagnon MJ, Elchebly M, Uetani N, et al.  Altered glucose homeostasis in mice lacking the receptor protein tyrosine phosphatase sigma. <span><span class="ref-journal">Can J Physiol Pharmacol. </span>2006;<span class="ref-vol">84</span>:755–763.</span> [<a href="/pubmed/16998539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Can+J+Physiol+Pharmacol&amp;title=Altered+glucose+homeostasis+in+mice+lacking+the+receptor+protein+tyrosine+phosphatase+sigma&amp;author=MJ+Chagnon&amp;author=M+Elchebly&amp;author=N+Uetani&amp;volume=84&amp;publication_year=2006&amp;pages=755-763&amp;pmid=16998539&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Aulchenko YS, Ripatti S, Lindqvist I, et al.  Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:47–55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687074/">PMC free article</a>]</span> [<a href="/pubmed/19060911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Loci+influencing+lipid+levels+and+coronary+heart+disease+risk+in+16+European+population+cohorts&amp;author=YS+Aulchenko&amp;author=S+Ripatti&amp;author=I+Lindqvist&amp;volume=41&amp;publication_year=2009&amp;pages=47-55&amp;pmid=19060911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Nicolae DL, Gamazon E, Zhang W, Duan S, Dolan ME, Cox NJ. Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000888.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2848547/">PMC free article</a>]</span> [<a href="/pubmed/20369019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Trait-associated+SNPs+are+more+likely+to+be+eQTLs:+annotation+to+enhance+discovery+from+GWAS&amp;author=DL+Nicolae&amp;author=E+Gamazon&amp;author=W+Zhang&amp;author=S+Duan&amp;author=ME+Dolan&amp;volume=6&amp;publication_year=2010&amp;pages=e1000888&amp;pmid=20369019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Elbein SC, Gamazon E, Kern PA, Rasouli N, Tikhomirov A, Cox NJ. <span><span class="ref-journal">Mapping genes near type 2 diabetes (T2D) associated loci as expression quantitative trait loci (eQTL) in human adipose and muscle. </span>2010</span> ADA Abstract no. 205-OR.  <a href="http://professional.diabetes.org/Abstracts_Display.aspx?TYP=1&amp;CID=79149" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://professional.diabetes.org/Abstracts_Display.aspx?TYP=1&amp;CID=79149</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Elbein SC, Gamazon ER, Das SK, Rasouli N, Kern PA, Cox NJ. High proportion of transcripts associated with insulin sensitivity in fat and muscle are associated with expression quantitative trait loci (eQTL) <span><span class="ref-journal">American Society of Human Genetics Abstract. </span>2010</span> (program no. 195).  <a href="http://www.ashg.org/cgi-bin/2010/ashg10s?author=Elbein&amp;sort=ptimes&amp;sbutton=Detail&amp;absno=20639&amp;sid=559883" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.ashg.org/cgi-bin/2010/ashg10s?author=Elbein&amp;sort=ptimes&amp;sbutton=Detail&amp;absno=20639&amp;sid=559883</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Goring HH, Terwilliger JD, Blangero J. Large upward bias in estimation of locus-specific effects from genomewide scans. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">69</span>:1357–1369.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235546/">PMC free article</a>]</span> [<a href="/pubmed/11593451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Large+upward+bias+in+estimation+of+locus-specific+effects+from+genomewide+scans&amp;author=HH+Goring&amp;author=JD+Terwilliger&amp;author=J+Blangero&amp;volume=69&amp;publication_year=2001&amp;pages=1357-1369&amp;pmid=11593451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3761075&amp;issue-id=227053&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139627900848240title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Methods</a></li><li><a href="#S13title">Results</a></li><li><a href="#S15title">Discussion</a></li><li><a href="#S16title">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#GLtitle">Abbreviations</a></li><li><a href="#idm139627905146288title">Footnotes</a></li><li><a href="#idm139627942078880aff-infotitle">Contributor Information</a></li><li><a href="#idm139627903348224title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3761075/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3761075/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3761075/pdf/nihms471860.pdf">PDF (849K)</a></li> | <li><a href="#" data-citationid="PMC3761075" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3761075%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3761075%2F&amp;text=Genome-wide%20association%20and%20meta-analysis%20in%20populations%20from%20Starr%20County%2C%20Texas%2C%20and%20Mexico%20City%20identify%20type%202%20diabetes%20susceptibility%20loci%20and%20enrichment%20for%20expression%20quantitative%20trait%20loci%20in%20top%20signals" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3761075%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21647700" data-aiid="3761075" data-aid="3761075" data-iid="227053" data-domainid="319" data-domain="nihpa" data-accid="PMC3761075" data-md5="3d59a2a6cda2019e3214f95e645793bf"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:09:30-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE898D78DD51E4D1000000000A07067F&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3761075%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3761075/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>