<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3724337/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study in a Swedish population yields support for greater CNV and MHC involvement in schizophrenia compared with bipolar disorder</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3724337/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Molecular psychiatry"><meta name="citation_title" content="Genome-wide association study in a Swedish population yields support for greater CNV and MHC involvement in schizophrenia compared with bipolar disorder"><meta name="citation_authors" content="SE Bergen, CT O’Dushlaine, S Ripke, PH Lee, DM Ruderfer, S Akterin, JL Moran, KD Chambert, RE Handsaker, L Backlund, U Ösby, S McCarroll, M Landen, EM Scolnick, PKE Magnusson, P Lichtenstein, CM Hultman, SM Purcell, P Sklar, PF Sullivan"><meta name="citation_date" content="September 2012"><meta name="citation_issue" content="9"><meta name="citation_volume" content="17"><meta name="citation_firstpage" content="880"><meta name="citation_doi" content="10.1038/mp.2012.73"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3724337/?report=abstract"><meta name="citation_pmid" content="22688191"><meta name="DC.Title" content="Genome-wide association study in a Swedish population yields support for greater CNV and MHC involvement in schizophrenia compared with bipolar disorder"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="SE Bergen"><meta name="DC.Contributor" content="CT O’Dushlaine"><meta name="DC.Contributor" content="S Ripke"><meta name="DC.Contributor" content="PH Lee"><meta name="DC.Contributor" content="DM Ruderfer"><meta name="DC.Contributor" content="S Akterin"><meta name="DC.Contributor" content="JL Moran"><meta name="DC.Contributor" content="KD Chambert"><meta name="DC.Contributor" content="RE Handsaker"><meta name="DC.Contributor" content="L Backlund"><meta name="DC.Contributor" content="U Ösby"><meta name="DC.Contributor" content="S McCarroll"><meta name="DC.Contributor" content="M Landen"><meta name="DC.Contributor" content="EM Scolnick"><meta name="DC.Contributor" content="PKE Magnusson"><meta name="DC.Contributor" content="P Lichtenstein"><meta name="DC.Contributor" content="CM Hultman"><meta name="DC.Contributor" content="SM Purcell"><meta name="DC.Contributor" content="P Sklar"><meta name="DC.Contributor" content="PF Sullivan"><meta name="DC.Date" content="2012 Sep"><meta name="DC.Identifier" content="10.1038/mp.2012.73"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study in a Swedish population yields support for greater CNV and MHC involvement in schizophrenia compared with bipolar disorder"><meta property="og:type" content="article"><meta property="og:description" content="Schizophrenia (SCZ) and bipolar disorder (BD) are highly heritable psychiatric disorders with overlapping susceptibility loci and symptomatology. We conducted a genome-wide association study (GWAS) of these disorders in a large Swedish sample. We report ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3724337/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3724337/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3724337/pdf/nihms-494874.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B3A08DD538B21000000000AC5074C.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3724337%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3724337</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Mol Psychiatry</a></span>. Author manuscript; available in PMC 2013 Jul 26.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22688191" target="pmc_ext" ref="reftype=publisher&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Mol Psychiatry. 2012 Sep; 17(9): 880–886. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Jun 12. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fmp.2012.73" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/mp.2012.73</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3724337</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS494874</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22688191">22688191</a></div></div></div></div><h1 class="content-title">Genome-wide association study in a Swedish population yields support for greater CNV and MHC involvement in schizophrenia compared with bipolar disorder</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Bergen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337872686720" co-class="co-affbox">SE Bergen</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=O%26%23x02019%3BDushlaine%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337881216544" co-class="co-affbox">CT O’Dushlaine</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Ripke%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337876550096" co-class="co-affbox">S Ripke</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Lee%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337876547376" co-class="co-affbox">PH Lee</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Ruderfer%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337878787792" co-class="co-affbox">DM Ruderfer</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Akterin%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337878784336" co-class="co-affbox">S Akterin</a>,<sup>4</sup> <a href="/pubmed/?term=Moran%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337878709024" co-class="co-affbox">JL Moran</a>,<sup>2</sup> <a href="/pubmed/?term=Chambert%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337878707040" co-class="co-affbox">KD Chambert</a>,<sup>2</sup> <a href="/pubmed/?term=Handsaker%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337883204144" co-class="co-affbox">RE Handsaker</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Backlund%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337883201424" co-class="co-affbox">L Backlund</a>,<sup>6</sup> <a href="/pubmed/?term=%26%23x000d6%3Bsby%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337883199440" co-class="co-affbox">U Ösby</a>,<sup>6</sup> <a href="/pubmed/?term=McCarroll%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337883042576" co-class="co-affbox">S McCarroll</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Landen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337883039856" co-class="co-affbox">M Landen</a>,<sup>4,</sup><sup>7</sup> <a href="/pubmed/?term=Scolnick%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337915751696" co-class="co-affbox">EM Scolnick</a>,<sup>2</sup> <a href="/pubmed/?term=Magnusson%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337915749712" co-class="co-affbox">PKE Magnusson</a>,<sup>4</sup> <a href="/pubmed/?term=Lichtenstein%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337915747728" co-class="co-affbox">P Lichtenstein</a>,<sup>4</sup> <a href="/pubmed/?term=Hultman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337878233088" co-class="co-affbox">CM Hultman</a>,<sup>4,</sup><sup>10</sup> <a href="/pubmed/?term=Purcell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337878230368" co-class="co-affbox">SM Purcell</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>8,</sup><sup>10</sup> <a href="/pubmed/?term=Sklar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337876079856" co-class="co-affbox">P Sklar</a>,<sup>1,</sup><sup>2,</sup><sup>8,</sup><sup>10</sup> and  <a href="/pubmed/?term=Sullivan%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191" class="affpopup" co-rid="_co_idm140337871832272" co-class="co-affbox">PF Sullivan</a><sup>9,</sup><sup>10</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140337872686720"><h3 class="no_margin">SE Bergen</h3><p><sup>1</sup>Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><div>Find articles by <a href="/pubmed/?term=Bergen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">SE Bergen</a></div></div><div id="_co_idm140337881216544"><h3 class="no_margin">CT O’Dushlaine</h3><p><sup>1</sup>Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><p><sup>3</sup>Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><div>Find articles by <a href="/pubmed/?term=O%26%23x02019%3BDushlaine%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">CT O’Dushlaine</a></div></div><div id="_co_idm140337876550096"><h3 class="no_margin">S Ripke</h3><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><p><sup>3</sup>Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><div>Find articles by <a href="/pubmed/?term=Ripke%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">S Ripke</a></div></div><div id="_co_idm140337876547376"><h3 class="no_margin">PH Lee</h3><p><sup>1</sup>Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><p><sup>3</sup>Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><div>Find articles by <a href="/pubmed/?term=Lee%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">PH Lee</a></div></div><div id="_co_idm140337878787792"><h3 class="no_margin">DM Ruderfer</h3><p><sup>1</sup>Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><p><sup>3</sup>Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><div>Find articles by <a href="/pubmed/?term=Ruderfer%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">DM Ruderfer</a></div></div><div id="_co_idm140337878784336"><h3 class="no_margin">S Akterin</h3><p><sup>4</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Akterin%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">S Akterin</a></div></div><div id="_co_idm140337878709024"><h3 class="no_margin">JL Moran</h3><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><div>Find articles by <a href="/pubmed/?term=Moran%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">JL Moran</a></div></div><div id="_co_idm140337878707040"><h3 class="no_margin">KD Chambert</h3><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><div>Find articles by <a href="/pubmed/?term=Chambert%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">KD Chambert</a></div></div><div id="_co_idm140337883204144"><h3 class="no_margin">RE Handsaker</h3><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><p><sup>5</sup>Department of Genetics, Harvard Medical School, Boston, MA, USA </p><div>Find articles by <a href="/pubmed/?term=Handsaker%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">RE Handsaker</a></div></div><div id="_co_idm140337883201424"><h3 class="no_margin">L Backlund</h3><p><sup>6</sup>Department of Molecular Medicine and Surgery, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Backlund%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">L Backlund</a></div></div><div id="_co_idm140337883199440"><h3 class="no_margin">U Ösby</h3><p><sup>6</sup>Department of Molecular Medicine and Surgery, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=%26%23x000d6%3Bsby%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">U Ösby</a></div></div><div id="_co_idm140337883042576"><h3 class="no_margin">S McCarroll</h3><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><p><sup>5</sup>Department of Genetics, Harvard Medical School, Boston, MA, USA </p><div>Find articles by <a href="/pubmed/?term=McCarroll%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">S McCarroll</a></div></div><div id="_co_idm140337883039856"><h3 class="no_margin">M Landen</h3><p><sup>4</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><p><sup>7</sup>Institute of Neuroscience and Physiology, University of Gothenburg, Gothenburg, Sweden </p><div>Find articles by <a href="/pubmed/?term=Landen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">M Landen</a></div></div><div id="_co_idm140337915751696"><h3 class="no_margin">EM Scolnick</h3><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><div>Find articles by <a href="/pubmed/?term=Scolnick%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">EM Scolnick</a></div></div><div id="_co_idm140337915749712"><h3 class="no_margin">PKE Magnusson</h3><p><sup>4</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Magnusson%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">PKE Magnusson</a></div></div><div id="_co_idm140337915747728"><h3 class="no_margin">P Lichtenstein</h3><p><sup>4</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Lichtenstein%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">P Lichtenstein</a></div></div><div id="_co_idm140337878233088"><h3 class="no_margin">CM Hultman</h3><p><sup>4</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Hultman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">CM Hultman</a></div></div><div id="_co_idm140337878230368"><h3 class="no_margin">SM Purcell</h3><p><sup>1</sup>Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><p><sup>3</sup>Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><p><sup>8</sup>Division of Psychiatric Genomics, Department of Psychiatry, Mount Sinai School of Medicine, New York, NY, USA </p><div>Find articles by <a href="/pubmed/?term=Purcell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">SM Purcell</a></div></div><div id="_co_idm140337876079856"><h3 class="no_margin">P Sklar</h3><p><sup>1</sup>Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </p><p><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </p><p><sup>8</sup>Division of Psychiatric Genomics, Department of Psychiatry, Mount Sinai School of Medicine, New York, NY, USA </p><div>Find articles by <a href="/pubmed/?term=Sklar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">P Sklar</a></div></div><div id="_co_idm140337871832272"><h3 class="no_margin">PF Sullivan</h3><p><sup>9</sup>Departments of Genetics, Psychiatry and Epidemiology, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA </p><div>Find articles by <a href="/pubmed/?term=Sullivan%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22688191">PF Sullivan</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140337874208512_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140337874208512_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140337874208512_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Genetics, Harvard Medical School, Boston, MA, USA </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Molecular Medicine and Surgery, Karolinska Institutet, Stockholm, Sweden </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Institute of Neuroscience and Physiology, University of Gothenburg, Gothenburg, Sweden </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Division of Psychiatric Genomics, Department of Psychiatry, Mount Sinai School of Medicine, New York, NY, USA </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Departments of Genetics, Psychiatry and Epidemiology, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA </div><div id="CR1">Correspondence: Dr SE Bergen, 7 Cambridge Center, 6055H, Broad Institute, Cambridge, MA 02142, USA. <a href="mailto:dev@null" data-email="moc.liamg@negrebs" class="oemail">moc.liamg@negrebs</a></div><div id="FN1"><sup>10</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140337874208512_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22688191" target="pmc_ext" ref="reftype=publisher&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Mol Psychiatry</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3724337/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC3724337/bin/NIHMS494874-supplement-Supplementary_Information.doc" data-ga-action="click_feat_suppl">NIHMS494874-supplement-Supplementary_Information.doc</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID:&nbsp;FA65CEAA-FD4D-42FF-8E3D-48CAAD0F5979</div></div></dd></dl></div><div id="idm140337879343504" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140337879343504title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">Schizophrenia (SCZ) and bipolar disorder (BD) are highly heritable psychiatric disorders with overlapping susceptibility loci and symptomatology. We conducted a genome-wide association study (GWAS) of these disorders in a large Swedish sample. We report a new and independent case–control analysis of 1507 SCZ cases, 836 BD cases and 2093 controls. No single-nucleotide polymorphisms (SNPs) achieved significance in these new samples; however, combining new and previously reported SCZ samples (2111 SCZ and 2535 controls) revealed a genome-wide significant association in the major histocompatibility complex (MHC) region (rs886424, <em>P</em> = 4.54 × 10<sup>−8</sup>). Imputation using multiple reference panels and meta-analysis with the Psychiatric Genomics Consortium SCZ results underscored the broad, significant association in the MHC region in the full SCZ sample. We evaluated the role of copy number variants (CNVs) in these subjects. As in prior reports, deletions were enriched in SCZ, but not BD cases compared with controls. Singleton deletions were more frequent in both case groups compared with controls (SCZ: <em>P</em> = 0.003, BD: <em>P</em> = 0.013), whereas the largest CNVs (&gt;500 kb) were significantly enriched only in SCZ cases (<em>P</em> = 0.0035). Two CNVs with previously reported SCZ associations were also overrepresented in this SCZ sample: 16p11.2 duplications (<em>P</em> = 0.0035) and 22q11 deletions (<em>P</em> = 0.03). These results reinforce prior reports of significant MHC and CNV associations in SCZ, but not BD.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">schizophrenia, bipolar disorder, genetic, genome-wide association, major histocompatibility complex, copy number variation</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P3" class="p p-first">Schizophrenia (SCZ) and bipolar disorder (BD) are severe and chronic psychiatric disorders.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916486">1</a></sup> A persistent debate in psychiatry has been the extent to which SCZ and BD are distinct.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916480">2</a></sup> Although the classical definitions of these illnesses are relatively discrete,<sup><a href="#R3" rid="R3" class=" bibr popnode">3</a></sup> there are multiple shared features: age of onset in late teens/early adulthood, prevalence (~1%), high heritability (59–64% in the Swedish population) and some symptomatology (e.g., psychosis, chronicity and cognitive deficits).<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916484">4</a>-<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261905">9</a></sup> Genetic overlap between these disorders was validated in a large Swedish study in which the shared genetic contribution was estimated to be &gt;60%.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261902">9</a></sup></p><p id="P4">Genome-wide association studies (GWAS) have discovered robust and replicable risk loci for SCZ. Genetic markers in the major histocompatibility complex (MHC) region and TCF4 have demonstrated replicated associations,<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261907">10</a>-<a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> and the Schizophrenia Psychiatric Genomics Consortium (PGC-SCZ) has recently identified other significant loci at 1p21.3, 2q32.3, 8p23.2, 8q21.3 and 10q24.32–q24.33.</p><p id="P5">GWAS of BD have identified significant associations to <em>DGKH</em>,<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916450">15</a></sup>
<em>ANK3</em><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916464">16</a></sup> and <em>NCAN</em>,<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916449">17</a></sup> and the largest BD GWAS, to date, discovered significant loci in the calcium channel gene <em>CACNA1C</em> and <em>ODZ4</em>.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916473">18</a></sup> In addition, a small number of loci were jointly analyzed by the PGC in SCZ and BD. These analyses demonstrated strong contributions from both disorders to the genome-wide significant results in <em>CACNA1C</em>, <em>ANK3</em> and the <em>ITIH3</em>–<em>ITIH4</em> region, supporting the idea of shared genetic risk.<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916456">18</a></sup></p><p id="P6">In contrast to common genetic variants (single-nucleotide polymorphisms (SNPs)), which have been implicated in these disorders, several large, rare copy number variants (CNVs) have been associated with SCZ, although the associations are less notable for BD. Deletions at 1q21.1, 2p21–16.3, 3q29, 15q11.2, 15q13.3 and 22q11, and duplications at 7q36.3, 16p11.2 and 16p13.11 confer substantial risk for SCZ in a small percentage of cases, but the risk is not specific to SCZ. These CNVs have also been observed at elevated rates in autism, epilepsy and mental retardation.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916446">19</a>-<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916477">22</a></sup> Duplications at 16p11.2 have demonstrated risk for SCZ, autism, and to some extent, BD.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916457">23</a>-<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_506231703">25</a></sup></p><p id="P7" class="p p-last">Few GWAS have examined SCZ and BD concurrently, and this represents the largest combined study of these disorders in a genetically homogeneous population to date. In this unique sample, we conducted association analyses of common SNPs and explored the relationship of CNVs to BD and SCZ. Additional analyses assessing the aggregation of associated loci in biological pathways, explorations of common copy number polymorphisms (CNPs) in SCZ, and comparison of our SCZ association results with the PGC SCZ results through meta-analysis were also performed.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">SUBJECTS AND METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Subjects</h3><div id="S4" class="sec sec-first"><p></p><h4 id="S4title" class="inline">Schizophrenia </h4><p id="P8" class="p p-first-last">Comprehensive analyses of most SCZ cases (<em>N</em> = 1507) and controls (<em>N</em> = 2093) are newly reported here, and some SCZ cases (<em>N</em> = 560) and controls (<em>N</em> = 400) from this study have been reported previously.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261899">10</a>,<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916485">34</a></sup> Additionally, 44 SCZ cases and 42 controls previously removed from analyses owing to Finnish ancestry were reintegrated into the current analyses. All subjects were born in Sweden, and SCZ cases were identified via the Swedish Hospital Discharge Register containing all individuals hospitalized in Sweden since 1973. The Hospital Discharge Register has high agreement with psychiatric diagnoses.<sup><a href="#R26" rid="R26" class=" bibr popnode">26</a>-<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916482">28</a></sup> Diagnoses were established by the attending physician and confirmed in a subset of subjects by medical record review. Cases must have had at least two hospitalizations with an SCZ diagnosis. Control subjects, also selected through registers, were group-matched by age, sex and county of residence, and must not have been hospitalized with a psychiatric diagnosis. All subjects were at least 18 years old and gave written informed consent to participate. The study was approved by the Ethical Committee at Karolinska Institutet.</p></div><div id="S5" class="sec sec-last"><p></p><h4 id="S5title" class="inline">Bipolar disorder </h4><p id="P9" class="p p-first-last">The previously unreported BD subjects (<em>N</em> = 836) were only collected in the most recent recruitment wave through three channels. Some BD cases (<em>N</em> = 256) were identified using the Swedish Hospital Discharge Register with at least two hospitalizations with a BD diagnosis and confirmatory diagnostic review in a subset of subjects.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916489">29</a></sup> Additional subjects were recruited from the Affective Center at St Goran Hospital in Stockholm, Sweden, following physician’s referral for BD (<em>N</em> = 216). The diagnostic instrument used was a Swedish adaptation of the Affective Disorder Evaluation,<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916470">30</a></sup> which includes the affective module of the Structured Clinical Interview for DSM-IV.<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup> A further 571 BD cases were recruited from the Stockholm County catchment area. Diagnoses were made according to the DSM-IV criteria, and cases were not reported previously. The control subjects used were the same as for the SCZ analyses described above. All ascertainment procedures were approved by the Regional Ethical Committees in Sweden.</p></div></div><div id="S6" class="sec"><h3 id="S6title">Genotyping and quality control</h3><p id="P10" class="p p-first">Blood samples were obtained and DNA extracted from whole blood using standard methods at the Karolinska Institutet. Samples were genotyped in four waves (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Most subjects (92.6%) were genotyped with Affymetrix 6.0 chips (Affymetrix, Santa Clara, CA, USA) with the remainder (7.4%) genotyped with the Affymetrix 5.0 chip. All genotyping was conducted at the Broad Institute of Harvard and MIT, and genotypes and CNVs were called using the Birdsuite algorithm.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916445">32</a></sup> The quality control exclusionary measures for subjects were: genotype call rates &lt;95%; ancestry outliers via multidimensional scaling; a randomly selected member of any pair of subjects with high relatedness (<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M1" overflow="scroll"><mrow><mover accent="true"><mi>π</mi><mo>^</mo></mover><mo>&gt;</mo><mn>0.20</mn></mrow></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M1" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mover" id="MathJax-Span-4"><span style="display: inline-block; position: relative; width: 0.591em; height: 0px;"><span style="position: absolute; clip: rect(3.391em, 1000.57em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">π<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.078em, 1000.56em, 3.581em, -1000em); top: -3.986em; left: 0.035em;"><span class="mo" id="MathJax-Span-6" style=""><span style="font-family: MathJax_Size1;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main; padding-left: 0.278em;">&gt;</span><span class="mn" id="MathJax-Span-8" style="font-family: MathJax_Main; padding-left: 0.278em;">0.20</span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M1" overflow="scroll"><mrow><mover accent="true"><mi>π</mi><mo>^</mo></mover><mo>&gt;</mo><mn>0.20</mn></mrow></math></script>); and suspected sample error or contamination. SNPs were excluded for marked departure from Hardy–Weinberg equilibrium (<em>P</em>&lt; 1 × 10<sup>−6</sup>), low minor allele frequencies (&lt;1%), and non-random genotyping failure, inferred from the flanking haplotype background using the PLINK ‘mishap’ test (<em>P</em>&lt; 1 × 10<sup>−10</sup>). Plate-based associations of <em>P</em>&lt; 1 × 10<sup>−6</sup> were taken as evidence of non-random plate failure, based on a comparison of allele frequency of each plate to all others and were removed on a plate-by-plate basis.</p><p id="P11" class="p p-last">Following quality control, the analysis data set consisted of 745 006 SNPs genotyped in 1507 SCZ cases, 836 BD cases and 2093 controls. GWAS results for these samples have not been previously reported. Additional SCZ cases (<em>N</em> = 604) and controls (<em>N</em> = 442) from earlier collection waves of this study<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916460">11</a>,<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916478">34</a></sup> bring the total sample size to 2947 combined cases and 2535 controls.</p></div><div id="S7" class="sec"><h3 id="S7title">Statistical methods</h3><p id="P12" class="p p-first-last">Imputation was conducted using BEAGLE.<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916461">33</a></sup> First, as with the PGC-SCZ data, we imputed our genotypes against autosomal genotype data from HapMap3<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916491">34</a></sup> and retained only SNP dosages imputed with very high confidence (Info &gt;0.8). Second, we imputed MHC Class 1 and 2 SNP and amino acid variants.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916497">35</a></sup> Following standard definitions for human leukocyte antigen (HLA) allele sequences from the EMBL-EBI Immunogenetics HLA Database, we imputed against the 2767 unrelated founder individuals of European descent, with high-resolution MHC genotyping data for 263 HLA alleles, 3852 SNPs and 372 amino acid positions. The genomic window imputed was 29.3–33.9 MB on chromosome 6.</p></div><div id="S8" class="sec"><h3 id="S8title">Association analyses</h3><p id="P13" class="p p-first-last">All association analyses were conducted using logistic regression in PLINK.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916493">36</a></sup> Multidimensional scaling was performed on the entire data set, and each collection wave was analyzed separately using the first four multidimensional scaling components as covariates, to control for population substructure. Results were then combined by meta-analysis in PLINK. Four main association analyses were conducted using the HapMap3 imputed data for: BD and SCZ separately (new subjects), SCZ (full sample), and all SCZ and BD cases combined (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). We used a genome-wide significance threshold of <em>P</em>&lt;5 × 10<sup>−8</sup>.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916453">37</a></sup> Imputed HLA alleles or amino acid dosages were tested for association using a logistic regression model with covariates to correct for population substructure and genotyping batch. For amino acid positions with &gt;2 alleles, we used the omnibus test in the conditional haplotype analysis module in PLINK. Conditional analyses in the MHC region were performed by conditioning on the genotyped SNP with the lowest <em>P</em>-value in the full SCZ results using the condition function in PLINK, which includes the additive effect of the SNP in the association model.</p></div><div id="S9" class="sec"><h3 id="S9title">Meta-analysis with PGC-SCZ</h3><p id="P14" class="p p-first-last">Some Swedish SCZ subjects (<em>N</em> = 558) and controls (<em>N</em> = 396) were among the 17 samples included in the PGC-SCZ report,<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> and we combined the individual non-Swedish PGC sample results with all Swedish SCZ results using the meta-analysis function in PLINK. This yielded <em>P</em>-values and odds ratios (ORs) for ~1.1 million markers in the combined sample of 11 271 cases and 14 601 controls. This is the largest SCZ GWAS meta-analysis yet reported.</p></div><div id="S10" class="sec"><h3 id="S10title">Pathway analysis</h3><p id="P15" class="p p-first-last">Analyses using INRICH software<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916451">38</a></sup> and the set screen test implemented in PLINK<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916499">39</a></sup> were conducted to assess the joint action of common SNPs within pathways. For these tests, genomic regions encompassing the top results as well as surrounding SNPs in linkage disequilibrium were compared against gene pathways. Full methods are in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Material</a>.</p></div><div id="S11" class="sec"><h3 id="S11title">Polygenic scoring</h3><p id="P16" class="p p-first-last">We carried out polygenic scoring analyses<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261903">10</a>,<a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> using the PGC-SCZ stage 1 samples. After excluding Swedish samples, the training data set consisted of 9160 cases and 12 066 controls, and 82 437 SNPs. The testing data set consisted of 2111 SCZ cases or 836 BD cases, and 2535 controls, and quantitative scores were computed for each subject based on the pT (<em>P</em>-value threshold), the proportion of SNPs with <em>P</em>-values &lt; pT in the training data set. For each SNP set defined by pT, we calculated the proportion of variance explained (<em>R</em><sup>2</sup>) by subtracting the Nagelkerke’s <em>R</em><sup>2</sup> attributable to ancestry covariates alone from the <em>R</em><sup>2</sup> for polygenic scores plus covariates. Full methods are described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Material</a>.</p></div><div id="S12" class="sec"><h3 id="S12title">CNV quality control and analyses</h3><p id="P17" class="p p-first">Intensity data from both SNP and structural variation probes were used to identify CNVs based on a hidden Markov model.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916490">32</a></sup> Only subjects who passed SNP quality control filters were considered for CNV analyses. We excluded CNVs with &gt;0.01 frequency, common CNV regions identified in the HapMap samples,<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916479">40</a></sup> and any CNVs spanning large genomic gaps (e.g., centromeres). Analyses were restricted to autosomal CNVs &gt;100 kb, and subjects were removed for having either CNVs totaling &gt;10 Mb or &gt;20 total CNVs. This resulted in 4147 CNVs in 1505 SCZ cases, 834 BD cases and 2087 controls remaining for analysis.</p><p id="P18" class="p p-last">The number of events in cases and controls, the rate per subject, the proportion of cases/controls to have at least one event, the total distance spanned per person and the average event size per person were evaluated for each CNV burden test across a range of frequencies (1, 2–6, or all), and sizes (100–200 kb, 200–500 kb and &gt;500 kb) for duplications and deletions, separately and combined in both case groups. Statistical tests were one-sided, presuming an increased CNV burden in cases, and significance was evaluated using 10 000 permutations. In addition to the number of CNVs, the proportion of genes intersecting these regions in cases versus controls was tested. Large CNVs with previously reported associations with SCZ or BD were specifically examined, and novel loci were explored. For specific CNVs, only deletions and duplications spanning the full CNV region were reported.</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">Copy number polymorphisms</h3><p id="P19" class="p p-first-last">Common CNP deletions 50–260 000 bp in length and present at &gt;1% frequency were identified in the 1000 Genomes data.<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575308710">41</a></sup> These CNPs were in linkage disequilibrium with 40 767 SNPs present in the HapMap3-imputed results for our full SCZ sample and were cross-referenced with these results. When multiple SNPs tagged a CNP, the SNP with the greatest R<sup>2</sup> and lowest <em>P</em>-value was retained.</p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">RESULTS</h2><div id="S15" class="sec sec-first"><h3 id="S15title">SNP association results</h3><p id="P20" class="p p-first">We performed genome-wide association analyses of SCZ and BD, and both disorders considered together using a common set of controls in a large sample from Sweden. Our basic analyses were new independent SCZ samples, all SCZ samples (new and previously reported), BD samples (all new), and SCZ and BD combined. The genomic control, <em>λ</em> (the median observed χ<sup>2</sup>-statistic divided by expectation under the null), showed little evidence for inflation of the test statistics. For SCZ (new and full samples) and SCZ–BD combined <em>λ</em> = 1.04 and for BD, <em>λ</em> = 1.05 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>). Regions of association with P&lt;1 × 10<sup>−5</sup> for the four main analyses are reported in <a href="/pmc/articles/PMC3724337/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The quantile–quantile and Manhattan plots for all analyses can be found in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures S1 and S2</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p2">Regions significant at <em>P</em>&lt;1 × 10<sup>−5</sup> for each of the four main association analyses and genes located in these regions</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337881164672" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">
<em>CHR</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>SNP</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Allele</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>OR</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">P-<em>values</em></th><th rowspan="1" colspan="1" valign="middle" align="center">N</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>Position</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>KB</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>Genes</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>SCZ: new subjects</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs12998068</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.40</td><td rowspan="1" colspan="1" valign="middle" align="center">1.50 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">26</td><td rowspan="1" colspan="1" valign="middle" align="left">154931117–155038981</td><td rowspan="1" colspan="1" valign="middle" align="right">107.9</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>GALNT13</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 17</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11150863</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.78</td><td rowspan="1" colspan="1" valign="middle" align="center">3.49 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">45</td><td rowspan="1" colspan="1" valign="middle" align="left">76071303–76340856</td><td rowspan="1" colspan="1" valign="middle" align="right">269.6</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NPTX1, KIAA1303</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2827775</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.80</td><td rowspan="1" colspan="1" valign="middle" align="center">4.73 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">31</td><td rowspan="1" colspan="1" valign="middle" align="left">23208407–23347097</td><td rowspan="1" colspan="1" valign="middle" align="right">138.7</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 5</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10515677</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.62</td><td rowspan="1" colspan="1" valign="middle" align="center">7.47 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">9</td><td rowspan="1" colspan="1" valign="middle" align="left">151887279–152400647</td><td rowspan="1" colspan="1" valign="middle" align="right">513.4</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs886424</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.71</td><td rowspan="1" colspan="1" valign="middle" align="center">8.80 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">151</td><td rowspan="1" colspan="1" valign="middle" align="left">30404232–31351863</td><td rowspan="1" colspan="1" valign="middle" align="right">947.6</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs9346146</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">9.78 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">18</td><td rowspan="1" colspan="1" valign="middle" align="left">69024600–69115229</td><td rowspan="1" colspan="1" valign="middle" align="right">90.6</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 16</td><td rowspan="1" colspan="1" valign="middle" align="left">rs821470</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.28</td><td rowspan="1" colspan="1" valign="middle" align="center">9.83 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="left">55608655–55623057</td><td rowspan="1" colspan="1" valign="middle" align="right">14.4</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NLRC5</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 22</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4820902</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.78</td><td rowspan="1" colspan="1" valign="middle" align="center">9.98 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">4</td><td rowspan="1" colspan="1" valign="middle" align="left">29468869–29501274</td><td rowspan="1" colspan="1" valign="middle" align="right">32.4</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>OSBP2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>SCZ: full sample</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs886424</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.68</td><td rowspan="1" colspan="1" valign="middle" align="center">4.54 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">158</td><td rowspan="1" colspan="1" valign="middle" align="left">30404232–31380240</td><td rowspan="1" colspan="1" valign="middle" align="right">976.0</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 1</td><td rowspan="1" colspan="1" valign="middle" align="left">rs984402</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.79</td><td rowspan="1" colspan="1" valign="middle" align="center">3.43 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">20</td><td rowspan="1" colspan="1" valign="middle" align="left">239156776–239228271</td><td rowspan="1" colspan="1" valign="middle" align="right">71.5</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>RGS7</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2524005</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.74</td><td rowspan="1" colspan="1" valign="middle" align="center">3.81 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">214</td><td rowspan="1" colspan="1" valign="middle" align="left">29519411–30387109</td><td rowspan="1" colspan="1" valign="middle" align="right">867.7</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 4</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4580657</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.61</td><td rowspan="1" colspan="1" valign="middle" align="center">1.61 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">3</td><td rowspan="1" colspan="1" valign="middle" align="left">102744068–102787481</td><td rowspan="1" colspan="1" valign="middle" align="right">43.4</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3117099</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.79</td><td rowspan="1" colspan="1" valign="middle" align="center">4.12 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">138</td><td rowspan="1" colspan="1" valign="middle" align="left">31986412–32863268</td><td rowspan="1" colspan="1" valign="middle" align="right">876.9</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1409497</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.39</td><td rowspan="1" colspan="1" valign="middle" align="center">5.17 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">22</td><td rowspan="1" colspan="1" valign="middle" align="left">22741893–22887738</td><td rowspan="1" colspan="1" valign="middle" align="right">145.8</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>SGCG, SACS</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3094228</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.31</td><td rowspan="1" colspan="1" valign="middle" align="center">5.96 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">58</td><td rowspan="1" colspan="1" valign="middle" align="left">31420305–31915519</td><td rowspan="1" colspan="1" valign="middle" align="right">495.2</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 5</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4262150</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.81</td><td rowspan="1" colspan="1" valign="middle" align="center">6.34 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">74</td><td rowspan="1" colspan="1" valign="middle" align="left">151771819–152489194</td><td rowspan="1" colspan="1" valign="middle" align="right">717.4</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NMUR2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 9</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7872515</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.73</td><td rowspan="1" colspan="1" valign="middle" align="center">6.65 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">23</td><td rowspan="1" colspan="1" valign="middle" align="left">93572203–93991056</td><td rowspan="1" colspan="1" valign="middle" align="right">418.9</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>SPTLC1, ROR2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs652888</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.29</td><td rowspan="1" colspan="1" valign="middle" align="center">7.28 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">43</td><td rowspan="1" colspan="1" valign="middle" align="left">31573896–32447054</td><td rowspan="1" colspan="1" valign="middle" align="right">873.2</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs13213152</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.42</td><td rowspan="1" colspan="1" valign="middle" align="center">8.43 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">174</td><td rowspan="1" colspan="1" valign="middle" align="left">27960336–28946608</td><td rowspan="1" colspan="1" valign="middle" align="right">986.3</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2235698</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.80</td><td rowspan="1" colspan="1" valign="middle" align="center">8.59 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">11</td><td rowspan="1" colspan="1" valign="middle" align="left">29525796–29663757</td><td rowspan="1" colspan="1" valign="middle" align="right">138.0</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 9</td><td rowspan="1" colspan="1" valign="middle" align="left">rs9411508</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.49</td><td rowspan="1" colspan="1" valign="middle" align="center">8.59 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">1</td><td rowspan="1" colspan="1" valign="middle" align="left">135177441–135177441</td><td rowspan="1" colspan="1" valign="middle" align="right">0.0</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>SURF6</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs12998068</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.32</td><td rowspan="1" colspan="1" valign="middle" align="center">8.60 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">24</td><td rowspan="1" colspan="1" valign="middle" align="left">154931117–155038981</td><td rowspan="1" colspan="1" valign="middle" align="right">107.9</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>GALNT13</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2171250</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.30</td><td rowspan="1" colspan="1" valign="middle" align="center">8.68 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">5</td><td rowspan="1" colspan="1" valign="middle" align="left">153766741–153789744</td><td rowspan="1" colspan="1" valign="middle" align="right">23.0</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs406511</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.41</td><td rowspan="1" colspan="1" valign="middle" align="center">9.07 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">39</td><td rowspan="1" colspan="1" valign="middle" align="left">28999155–29585800</td><td rowspan="1" colspan="1" valign="middle" align="right">586.6</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>BD</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 4</td><td rowspan="1" colspan="1" valign="middle" align="left">rs17746001</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">2.03</td><td rowspan="1" colspan="1" valign="middle" align="center">3.22 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">7</td><td rowspan="1" colspan="1" valign="middle" align="left">180892619–180921485</td><td rowspan="1" colspan="1" valign="middle" align="right">28.9</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 12</td><td rowspan="1" colspan="1" valign="middle" align="left">rs17040430</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.89</td><td rowspan="1" colspan="1" valign="middle" align="center">9.22 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">2</td><td rowspan="1" colspan="1" valign="middle" align="left">107243140–107266950</td><td rowspan="1" colspan="1" valign="middle" align="right">23.8</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CMKLR1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">rs271738</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.32</td><td rowspan="1" colspan="1" valign="middle" align="center">4.81 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">28</td><td rowspan="1" colspan="1" valign="middle" align="left">232723219–232828069</td><td rowspan="1" colspan="1" valign="middle" align="right">104.9</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>IRF2BP2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 8</td><td rowspan="1" colspan="1" valign="middle" align="left">rs289585</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.69</td><td rowspan="1" colspan="1" valign="middle" align="center">5.18 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">5</td><td rowspan="1" colspan="1" valign="middle" align="left">13236908–13304907</td><td rowspan="1" colspan="1" valign="middle" align="right">68.0</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>DLC1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 9</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3011267</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.41</td><td rowspan="1" colspan="1" valign="middle" align="center">9.82 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">4</td><td rowspan="1" colspan="1" valign="middle" align="left">134871014–134890520</td><td rowspan="1" colspan="1" valign="middle" align="right">19.5</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>GTF3C5, GFI1B</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>SCZ and BD combined</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 4</td><td rowspan="1" colspan="1" valign="middle" align="left">rs17746001</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.75</td><td rowspan="1" colspan="1" valign="middle" align="center">2.83 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">11</td><td rowspan="1" colspan="1" valign="middle" align="left">180852651–180921485</td><td rowspan="1" colspan="1" valign="middle" align="right">68.8</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2524005</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.76</td><td rowspan="1" colspan="1" valign="middle" align="center">4.95 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">247</td><td rowspan="1" colspan="1" valign="middle" align="left">29519411–30473427</td><td rowspan="1" colspan="1" valign="middle" align="right">954.0</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 9</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7872515</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.72</td><td rowspan="1" colspan="1" valign="middle" align="center">5.79 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">33</td><td rowspan="1" colspan="1" valign="middle" align="left">93575453–93991056</td><td rowspan="1" colspan="1" valign="middle" align="right">415.6</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>SPTLC1, ROR2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs886424</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.73</td><td rowspan="1" colspan="1" valign="middle" align="center">9.25 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">119</td><td rowspan="1" colspan="1" valign="middle" align="left">30623022–31380240</td><td rowspan="1" colspan="1" valign="middle" align="right">757.2</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 3</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2054399</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.82</td><td rowspan="1" colspan="1" valign="middle" align="center">1.29 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">29</td><td rowspan="1" colspan="1" valign="middle" align="left">179688504–179854731</td><td rowspan="1" colspan="1" valign="middle" align="right">166.2</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>KCNMB2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 16</td><td rowspan="1" colspan="1" valign="middle" align="left">rs821470</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.25</td><td rowspan="1" colspan="1" valign="middle" align="center">2.56 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="left">55608655–55623057</td><td rowspan="1" colspan="1" valign="middle" align="right">14.4</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NLRC5</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3117099</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.80</td><td rowspan="1" colspan="1" valign="middle" align="center">2.70 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">139</td><td rowspan="1" colspan="1" valign="middle" align="left">31986412–32863268</td><td rowspan="1" colspan="1" valign="middle" align="right">876.9</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC, many genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 8</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1484642</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.43</td><td rowspan="1" colspan="1" valign="middle" align="center">3.33 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">24</td><td rowspan="1" colspan="1" valign="middle" align="left">9931186–10039190</td><td rowspan="1" colspan="1" valign="middle" align="right">108.0</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>MSRA</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 10</td><td rowspan="1" colspan="1" valign="middle" align="left">rs181500</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">4.84 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">4</td><td rowspan="1" colspan="1" valign="middle" align="left">119169072–119174309</td><td rowspan="1" colspan="1" valign="middle" align="right">5.2</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 5</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4262150</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.82</td><td rowspan="1" colspan="1" valign="middle" align="center">7.19 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">61</td><td rowspan="1" colspan="1" valign="middle" align="left">151771819–152489194</td><td rowspan="1" colspan="1" valign="middle" align="right">717.4</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NMUR2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 5</td><td rowspan="1" colspan="1" valign="middle" align="left">rs253959</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21</td><td rowspan="1" colspan="1" valign="middle" align="center">8.09 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">96</td><td rowspan="1" colspan="1" valign="middle" align="left">115440636–115779788</td><td rowspan="1" colspan="1" valign="middle" align="right">339.2</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>COMMD10</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> 2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2058710</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">9.39 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="left">206766767–207166124</td><td rowspan="1" colspan="1" valign="middle" align="right">399.4</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>ZDBF2, GPR1, ADAM23</em>
</td></tr></tbody></table></div><div id="largeobj_idm140337881164672" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3724337/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p3">Abbreviations: SCZ, schizophrenia; BD, bipolar disorder; <em>N</em>, number of SNPs in the reported region; CHR, chromosome; SNP, single-nucleotide polymorphism; OR, odds ratio; MHC, major histocompatibility complex.</p><p id="__p4">Position = hg18 coordinates. Genes in these regions or the 20-kb flanking regions were identified using the UCSC Genome Browser.</p></div></div><p id="P21">In the new and previously unreported sample of 1507 SCZ cases and 2093 controls, none of 1 105 620 imputed SNPs were genome-wide significant (<a href="/pmc/articles/PMC3724337/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The most highly associated SNP was in an intron of <em>GALNT13</em> (rs12998068, <em>P</em> = 1.50 × 10<sup>−6</sup>; OR = 1.40). Other noteworthy results include rs11150863 (<em>P</em> = 3.49 × 10<sup>−6</sup>; OR = 0.78) in an intron of <em>RPTOR</em> and many SNPs in the MHC region (top SNP: rs886424, 8.80 × 10<sup>−6</sup>; OR = 0.71).</p><p id="P22">We next combined the new and previously reported SCZ samples ascertained as part of the same study (2111 SCZ cases and 2535 controls). One SNP in the MHC region attained genome-wide significance (rs886424, <em>P</em> = 4.54 × 10<sup>−8</sup>; OR = 0.68). This marker was directly genotyped, and conditioning on this SNP revealed no independent signals in the 10-Mb surrounding region with <em>P</em>&lt;1 × 10<sup>−4</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3</a>). Some evidence of heterogeneity in ORs for this region across collection waves was observed (detailed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Text</a>). After the MHC, the next strongest association signal was from a SNP in RGS7 (rs984402, <em>P</em> = 3.43 × 10<sup>−7</sup>; OR = 0.79). Association testing of 836 BD cases and 2093 controls yielded no genome-wide significant findings. The SNP with the lowest <em>P</em>-value, rs17746001 (<em>P</em> = 3.22 × 10<sup>−7</sup>; OR = 2.03), was in a gene-poor region of chromosome 4.</p><p id="P23" class="p p-last">We next tested for association using SCZ and BD as a combined phenotype in 2949 cases (2111 SCZ and 836 BD) and 2535 controls. The smallest <em>P</em>-value in the combined analysis was also detected for rs17746001 (<em>P</em> = 2.83 × 10<sup>−7</sup>; OR = 1.75) and it was strongly represented in the SCZ results (<em>P</em> = 2.23 × 10<sup>−5</sup>; OR = 1.64). This was closely followed by a large cluster of MHC associations (top SNP: rs2524005, <em>P</em> = 4.95 × 10<sup>−7</sup>; OR = 0.76).</p></div><div id="S16" class="sec"><h3 id="S16title">MHC imputation results</h3><p id="P24" class="p p-first-last">The highly polymorphic MHC region presents challenges to standard imputation methods. An imputation approach specifically tailored for this region was used to refine the signal at this locus through prediction of classical HLA alleles, and their constituent single-nucleotide variants and corresponding amino acids separately. The MHC-specific imputation in the full SCZ sample revealed an additional SNP with genome-wide significance (rs1264353, <em>P</em> = 3.89 × 10<sup>−8</sup>). Two classic HLA alleles were also among the top results (HLA_B*0801, <em>P</em> = 2.36 × 10<sup>−6</sup>, HLA_A*0101, <em>P</em> = 2.83 × 10<sup>−6</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4</a>).</p></div><div id="S17" class="sec"><h3 id="S17title">Meta-analysis with PGC-SCZ</h3><p id="P25" class="p p-first-last">With the PGC samples, we conducted a meta-analysis of 11 271 SCZ cases and 14 601 controls (<a href="/pmc/articles/PMC3724337/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The MHC region manifests the most significant association, and was driven by strong signals in both PGC-SCZ and the Swedish subjects (rs17693963, <em>P</em> = 3.08 × 10<sup>−11</sup>, OR = 1.24). This was followed by a chromosome 10 locus containing <em>CNNM2</em>, <em>C10orf32</em> and other genes (rs11191580, <em>P</em> = 1.73 × 10<sup>−9</sup>, OR = 1.23), and a chromosome 7 locus encompassing MAD1L1 (rs12666575, <em>P</em> = 1.75 × 10<sup>−9</sup>, OR = 0.89), which had not attained genome-wide significance in the PGC-SCZ samples alone. Other regions of convergent and genome-wide association include the NT5C2 locus on chromosome 10 (rs1926034, <em>P</em> = 1.37 × 10<sup>−8</sup>, OR = 0.89) and a chromosome 5 region lacking in annotated genes (rs7709645, <em>P</em> = 3.80 × 10<sup>−8</sup>, OR 1.11), which was not previously noted in the results from either sample.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Results of the meta-analysis of PGC-SCZ minus the Swedish samples and all Swedish SCZ results with <em>P</em>&lt;1 × 10<sup>−6</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337869446896" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>PGC-SCZ</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Swedish sample</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Meta-analysis</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">
<em>CHR</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>SNP</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">P-<em>values</em></th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>OR</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">P-<em>values</em></th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">P-<em>values</em></th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>OR</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>N</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>Position</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">KB</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>Genes in range</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs17693963</td><td rowspan="1" colspan="1" valign="middle" align="center">2.43 ×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22</td><td rowspan="1" colspan="1" valign="middle" align="left">1.78 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.33</td><td rowspan="1" colspan="1" valign="middle" align="center">3.08 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">239</td><td rowspan="1" colspan="1" valign="middle" align="left">27050006-28808331</td><td rowspan="1" colspan="1" valign="middle" align="center">1758</td><td rowspan="1" colspan="1" valign="middle" align="left">MHC</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">10</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11191580</td><td rowspan="1" colspan="1" valign="middle" align="center">2.85 ×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.25</td><td rowspan="1" colspan="1" valign="middle" align="left">0.104</td><td rowspan="1" colspan="1" valign="middle" align="center">1.13</td><td rowspan="1" colspan="1" valign="middle" align="center">1.73 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.23</td><td rowspan="1" colspan="1" valign="middle" align="center">42</td><td rowspan="1" colspan="1" valign="middle" align="left">104277349-104929205</td><td rowspan="1" colspan="1" valign="middle" align="center">652</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>NT5C2, CNNM2</em>, and others</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">7</td><td rowspan="1" colspan="1" valign="middle" align="left">rs12666575</td><td rowspan="1" colspan="1" valign="middle" align="center">2.47 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00161</td><td rowspan="1" colspan="1" valign="middle" align="center">0.87</td><td rowspan="1" colspan="1" valign="middle" align="center">1.75 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center">51</td><td rowspan="1" colspan="1" valign="middle" align="left">1825144-2312641</td><td rowspan="1" colspan="1" valign="middle" align="center">487</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>MAD1L1, SNX8, NUDT1, FTSJ2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">10</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1926034</td><td rowspan="1" colspan="1" valign="middle" align="center">8.17 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0521</td><td rowspan="1" colspan="1" valign="middle" align="center">0.92</td><td rowspan="1" colspan="1" valign="middle" align="center">1.37 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center">52</td><td rowspan="1" colspan="1" valign="middle" align="left">104490649-105378681</td><td rowspan="1" colspan="1" valign="middle" align="center">888</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>NT5C2, CNNM2</em>, and others</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7709645</td><td rowspan="1" colspan="1" valign="middle" align="center">3.65 × 10−6</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00217</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14</td><td rowspan="1" colspan="1" valign="middle" align="center">3.80 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.11</td><td rowspan="1" colspan="1" valign="middle" align="center">40</td><td rowspan="1" colspan="1" valign="middle" align="left">60522272-60879681</td><td rowspan="1" colspan="1" valign="middle" align="center">357</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7004633</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.85</td><td rowspan="1" colspan="1" valign="middle" align="left">0.514</td><td rowspan="1" colspan="1" valign="middle" align="center">0.96</td><td rowspan="1" colspan="1" valign="middle" align="center">6.44 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.87</td><td rowspan="1" colspan="1" valign="middle" align="center">95</td><td rowspan="1" colspan="1" valign="middle" align="left">89257570-89864562</td><td rowspan="1" colspan="1" valign="middle" align="center">607</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>MMP16</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs13025591</td><td rowspan="1" colspan="1" valign="middle" align="center">1.62 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.90</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0159</td><td rowspan="1" colspan="1" valign="middle" align="center">0.90</td><td rowspan="1" colspan="1" valign="middle" align="center">7.74 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.90</td><td rowspan="1" colspan="1" valign="middle" align="center">30</td><td rowspan="1" colspan="1" valign="middle" align="left">236434814-236498899</td><td rowspan="1" colspan="1" valign="middle" align="center">64</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CENTG2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">18</td><td rowspan="1" colspan="1" valign="middle" align="left">rs17597926</td><td rowspan="1" colspan="1" valign="middle" align="center">1.27 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.35</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0677</td><td rowspan="1" colspan="1" valign="middle" align="center">1.31</td><td rowspan="1" colspan="1" valign="middle" align="center">1.01 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.36</td><td rowspan="1" colspan="1" valign="middle" align="center">17</td><td rowspan="1" colspan="1" valign="middle" align="left">51046416-51356936</td><td rowspan="1" colspan="1" valign="middle" align="center">311</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>TCF4</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4765905</td><td rowspan="1" colspan="1" valign="middle" align="center">1.76 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.11</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0314</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">1.60 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.11</td><td rowspan="1" colspan="1" valign="middle" align="center">37</td><td rowspan="1" colspan="1" valign="middle" align="left">2162951-2310725</td><td rowspan="1" colspan="1" valign="middle" align="center">148</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CACNA1C</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">15</td><td rowspan="1" colspan="1" valign="middle" align="left">rs12592967</td><td rowspan="1" colspan="1" valign="middle" align="center">1.51 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0042</td><td rowspan="1" colspan="1" valign="middle" align="center">1.13</td><td rowspan="1" colspan="1" valign="middle" align="center">2.69 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">33</td><td rowspan="1" colspan="1" valign="middle" align="left">59616968-59694773</td><td rowspan="1" colspan="1" valign="middle" align="center">78</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10894294</td><td rowspan="1" colspan="1" valign="middle" align="center">8.61 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.91</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0101</td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center">2.81 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.91</td><td rowspan="1" colspan="1" valign="middle" align="center">23</td><td rowspan="1" colspan="1" valign="middle" align="left">130299565-130400186</td><td rowspan="1" colspan="1" valign="middle" align="center">101</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1198588</td><td rowspan="1" colspan="1" valign="middle" align="center">2.58 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.88</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0621</td><td rowspan="1" colspan="1" valign="middle" align="center">0.91</td><td rowspan="1" colspan="1" valign="middle" align="center">4.71 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center">58</td><td rowspan="1" colspan="1" valign="middle" align="left">98088873-98325420</td><td rowspan="1" colspan="1" valign="middle" align="center">237</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>hsa-mir-137</em>, DPYD</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">rs12071951</td><td rowspan="1" colspan="1" valign="middle" align="center">3.84 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00285</td><td rowspan="1" colspan="1" valign="middle" align="center">0.85</td><td rowspan="1" colspan="1" valign="middle" align="center">5.26 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.88</td><td rowspan="1" colspan="1" valign="middle" align="center">23</td><td rowspan="1" colspan="1" valign="middle" align="left">96883528-97057084</td><td rowspan="1" colspan="1" valign="middle" align="center">174</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>PTBP2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11220082</td><td rowspan="1" colspan="1" valign="middle" align="center">4.64 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0479</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09</td><td rowspan="1" colspan="1" valign="middle" align="center">6.66 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="left">124792166-124951853</td><td rowspan="1" colspan="1" valign="middle" align="center">160</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>PKNOX2, FEZ1, EI24</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">10</td><td rowspan="1" colspan="1" valign="middle" align="left">rs16915157</td><td rowspan="1" colspan="1" valign="middle" align="center">3.72 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.13</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0648</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">6.84 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.13</td><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="left">61885077-62016644</td><td rowspan="1" colspan="1" valign="middle" align="center">132</td><td rowspan="1" colspan="1" valign="middle" align="left">No genes</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10503256</td><td rowspan="1" colspan="1" valign="middle" align="center">1.86 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12</td><td rowspan="1" colspan="1" valign="middle" align="left">0.596</td><td rowspan="1" colspan="1" valign="middle" align="center">1.02</td><td rowspan="1" colspan="1" valign="middle" align="center">8.61 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">31</td><td rowspan="1" colspan="1" valign="middle" align="left">4165199-4228540</td><td rowspan="1" colspan="1" valign="middle" align="center">63</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CSMD1</em>
</td></tr></tbody></table></div><div id="largeobj_idm140337869446896" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3724337/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p6">Abbreviations: SCZ, schizophrenia; BD, bipolar disorder; <em>N</em>, number of SNPs within each associated region; CHR, chromosome; SNP, single-nucleotidepolymorphism; OR, odds ratio; MHC, major histocompatibility complex; PGC-SCZ, Schizophrenia Psychiatric GWAS Consortium; LD, linkage disequilibrium;GWAS, genome-wide association studies.</p><p id="__p7">Only the top MHC clumped region was retained to prevent this large region of high LD from dominating the results. Position = hg18 coordinates.</p></div></div></div><div id="S18" class="sec"><h3 id="S18title">Pathway analysis results</h3><p id="P26" class="p p-first-last">Top PGC-SCZ results did show evidence of enrichment in the SCZ full-sample results (<em>P</em> = 0. 044). The 301 targets of <em>MIR137</em> were not overrepresented in the top results for the SCZ new (<em>P</em> = 0.97) or full samples (<em>P</em> = 0.94). In the synaptic gene tests, the endocytosis-related set was significantly enriched in the full SCZ (<em>P</em> 0.046) and combined SCZ–BD (<em>P</em> = 0.027) results. Calcium channel genes did not show enrichment in the BD or combined SCZ–BD results (<em>P</em> = 1). No single Gene Ontology term was significant in the SCZ–BD results following multiple testing correction; however, more genes contained within Gene Ontology categories demonstrated nominal associations of <em>P</em>&lt;0.05 than expected by chance (<em>P</em> = 0.0055; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>).</p></div><div id="S19" class="sec"><h3 id="S19title">Polygenic scoring results</h3><p id="P27" class="p p-first-last">Using the PGC-SCZ sample minus the Swedish subjects as the discovery set, we demonstrated enrichment of putatively associated SCZ score alleles in both the SCZ and BD target case groups, compared with controls for all discovery pT. In the full Swedish SCZ sample, we were able to predict 3.6% of variance explained (<em>P</em>&lt; 2 × 10<sup>−16</sup> for all pT). A smaller, but still significant, percent of variance explained (1.4%) was evident for the BD sample, supporting evidence for shared genetic risk between these two disorders (<em>P</em>&lt; 5.92 × 10<sup>−4</sup> for all pT). These results corroborate prior reports indicating a polygenic contribution of multiple common SNPs to risk for both SCZ and BD<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261901">10</a>,<a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p></div><div id="S20" class="sec"><h3 id="S20title">CNV analysis results</h3><p id="P28" class="p p-first">We investigated the role of CNVs&gt; 100 kb in our new SCZ and BD samples (<a href="/pmc/articles/PMC3724337/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Overall, deletions were enriched to a greater extent in the SCZ sample than in the BD subjects compared with controls. Deletions observed only once were more common in both case groups compared with controls (SCZ: <em>P</em> = 0.003; BD: <em>P</em> = 0.013), whereas the largest size category CNVs&gt;500 kb, were only significantly enriched in SCZ (<em>P</em> = 0.00048). However, it is of note that the gene count in deletions &gt;500 kb was significantly enriched in both SCZ and BD. Duplications did not show significant overrepresentation for either case group for any size or frequency examined.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p8">CNV burden and gene count results for only the new samples SCZ (<em>N</em> = 1505) versus controls (<em>N</em> = 2087), and BD (<em>N</em> = 834) versus controls.(a) By CNV frequency and type; (b) by CNV size and type</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337874632720" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="2" rowspan="1" valign="middle" align="left">
<em>CNV burden (number)</em>
<hr>
</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="2" rowspan="1" valign="middle" align="left">
<em>CNV burden (gene count)</em>
<hr>
</th><th rowspan="1" colspan="1" valign="middle" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">
<em>CNV type</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>Frequency</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">P-<em>values</em></th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Total</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Case/control ratio</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Controls</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">P-<em>values</em></th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Case/control ratio</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Controls</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>(a) SCZ</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">All</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">0.15</td><td rowspan="1" colspan="1" valign="middle" align="right">998</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08</td><td rowspan="1" colspan="1" valign="middle" align="center">0.269</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0022</td><td rowspan="1" colspan="1" valign="middle" align="center">1.48</td><td rowspan="1" colspan="1" valign="middle" align="center">0.560</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">2-6</td><td rowspan="1" colspan="1" valign="middle" align="left">0.45</td><td rowspan="1" colspan="1" valign="middle" align="right">763</td><td rowspan="1" colspan="1" valign="middle" align="center">1.01</td><td rowspan="1" colspan="1" valign="middle" align="center">0.211</td><td rowspan="1" colspan="1" valign="middle" align="left">0.09</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22</td><td rowspan="1" colspan="1" valign="middle" align="center">0.357</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">All</td><td rowspan="1" colspan="1" valign="middle" align="left">0.10</td><td rowspan="1" colspan="1" valign="middle" align="right">3347</td><td rowspan="1" colspan="1" valign="middle" align="center">1.05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.913</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0005</td><td rowspan="1" colspan="1" valign="middle" align="center">1.33</td><td rowspan="1" colspan="1" valign="middle" align="center">1.629</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">0.003</td><td rowspan="1" colspan="1" valign="middle" align="right">437</td><td rowspan="1" colspan="1" valign="middle" align="center">1.33</td><td rowspan="1" colspan="1" valign="middle" align="center">0.107</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0002</td><td rowspan="1" colspan="1" valign="middle" align="center">3.04</td><td rowspan="1" colspan="1" valign="middle" align="center">0.135</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">2–6</td><td rowspan="1" colspan="1" valign="middle" align="left">0.45</td><td rowspan="1" colspan="1" valign="middle" align="right">250</td><td rowspan="1" colspan="1" valign="middle" align="center">1.05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.068</td><td rowspan="1" colspan="1" valign="middle" align="left">0.12</td><td rowspan="1" colspan="1" valign="middle" align="center">1.52</td><td rowspan="1" colspan="1" valign="middle" align="center">0.069</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">All</td><td rowspan="1" colspan="1" valign="middle" align="left">0.023</td><td rowspan="1" colspan="1" valign="middle" align="right">1244</td><td rowspan="1" colspan="1" valign="middle" align="center">1.13</td><td rowspan="1" colspan="1" valign="middle" align="center">0.328</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0001</td><td rowspan="1" colspan="1" valign="middle" align="center">2.18</td><td rowspan="1" colspan="1" valign="middle" align="center">0.324</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">0.65</td><td rowspan="1" colspan="1" valign="middle" align="right">695</td><td rowspan="1" colspan="1" valign="middle" align="center">0.97</td><td rowspan="1" colspan="1" valign="middle" align="center">0.196</td><td rowspan="1" colspan="1" valign="middle" align="left">0.44</td><td rowspan="1" colspan="1" valign="middle" align="center">1.02</td><td rowspan="1" colspan="1" valign="middle" align="center">0.524</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">2–6</td><td rowspan="1" colspan="1" valign="middle" align="left">0.84</td><td rowspan="1" colspan="1" valign="middle" align="right">426</td><td rowspan="1" colspan="1" valign="middle" align="center">0.91</td><td rowspan="1" colspan="1" valign="middle" align="center">0.123</td><td rowspan="1" colspan="1" valign="middle" align="left">0.31</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">0.250</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">All</td><td rowspan="1" colspan="1" valign="middle" align="left">0.50</td><td rowspan="1" colspan="1" valign="middle" align="right">2103</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00</td><td rowspan="1" colspan="1" valign="middle" align="center">0.585</td><td rowspan="1" colspan="1" valign="middle" align="left">0.10</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12</td><td rowspan="1" colspan="1" valign="middle" align="center">1.305</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>BD</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">0.08</td><td rowspan="1" colspan="1" valign="middle" align="right">817</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14</td><td rowspan="1" colspan="1" valign="middle" align="center">0.269</td><td rowspan="1" colspan="1" valign="middle" align="left">0.032</td><td rowspan="1" colspan="1" valign="middle" align="center">1.37</td><td rowspan="1" colspan="1" valign="middle" align="center">0.560</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">2–6</td><td rowspan="1" colspan="1" valign="middle" align="left">0.60</td><td rowspan="1" colspan="1" valign="middle" align="right">614</td><td rowspan="1" colspan="1" valign="middle" align="center">0.98</td><td rowspan="1" colspan="1" valign="middle" align="center">0.211</td><td rowspan="1" colspan="1" valign="middle" align="left">0.71</td><td rowspan="1" colspan="1" valign="middle" align="center">0.90</td><td rowspan="1" colspan="1" valign="middle" align="center">0.357</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">All</td><td rowspan="1" colspan="1" valign="middle" align="left">0.16</td><td rowspan="1" colspan="1" valign="middle" align="right">2705</td><td rowspan="1" colspan="1" valign="middle" align="center">1.05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.913</td><td rowspan="1" colspan="1" valign="middle" align="left">0.27</td><td rowspan="1" colspan="1" valign="middle" align="center">1.07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.629</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">0.013</td><td rowspan="1" colspan="1" valign="middle" align="right">343</td><td rowspan="1" colspan="1" valign="middle" align="center">1.35</td><td rowspan="1" colspan="1" valign="middle" align="center">0.107</td><td rowspan="1" colspan="1" valign="middle" align="left">0.010</td><td rowspan="1" colspan="1" valign="middle" align="center">2.00</td><td rowspan="1" colspan="1" valign="middle" align="center">0.135</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">2–6</td><td rowspan="1" colspan="1" valign="middle" align="left">0.60</td><td rowspan="1" colspan="1" valign="middle" align="right">198</td><td rowspan="1" colspan="1" valign="middle" align="center">0.99</td><td rowspan="1" colspan="1" valign="middle" align="center">0.068</td><td rowspan="1" colspan="1" valign="middle" align="left">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center">0.50</td><td rowspan="1" colspan="1" valign="middle" align="center">0.069</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">All</td><td rowspan="1" colspan="1" valign="middle" align="left">0.15</td><td rowspan="1" colspan="1" valign="middle" align="right">981</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08</td><td rowspan="1" colspan="1" valign="middle" align="center">0.328</td><td rowspan="1" colspan="1" valign="middle" align="left">0.15</td><td rowspan="1" colspan="1" valign="middle" align="center">1.23</td><td rowspan="1" colspan="1" valign="middle" align="center">0.324</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">0.30</td><td rowspan="1" colspan="1" valign="middle" align="right">582</td><td rowspan="1" colspan="1" valign="middle" align="center">1.06</td><td rowspan="1" colspan="1" valign="middle" align="center">0.196</td><td rowspan="1" colspan="1" valign="middle" align="left">0.33</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08</td><td rowspan="1" colspan="1" valign="middle" align="center">0.524</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">2–6</td><td rowspan="1" colspan="1" valign="middle" align="left">0.87</td><td rowspan="1" colspan="1" valign="middle" align="right">347</td><td rowspan="1" colspan="1" valign="middle" align="center">0.88</td><td rowspan="1" colspan="1" valign="middle" align="center">0.123</td><td rowspan="1" colspan="1" valign="middle" align="left">0.51</td><td rowspan="1" colspan="1" valign="middle" align="center">0.99</td><td rowspan="1" colspan="1" valign="middle" align="center">0.250</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">All</td><td rowspan="1" colspan="1" valign="middle" align="left">0.31</td><td rowspan="1" colspan="1" valign="middle" align="right">1724</td><td rowspan="1" colspan="1" valign="middle" align="center">1.03</td><td rowspan="1" colspan="1" valign="middle" align="center">0.585</td><td rowspan="1" colspan="1" valign="middle" align="left">0.42</td><td rowspan="1" colspan="1" valign="middle" align="center">1.02</td><td rowspan="1" colspan="1" valign="middle" align="center">1.305</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CNV type</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>Size (kb)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">P-<em>values</em></td><td rowspan="1" colspan="1" valign="middle" align="right">
<em>Total</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>Case/control ratio</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>Controls</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">P-<em>values</em></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>Case/control ratio</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>Controls</em>
</td></tr><tr><td colspan="9" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>(b) SCZ</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">100-200</td><td rowspan="1" colspan="1" valign="middle" align="left">0.72</td><td rowspan="1" colspan="1" valign="middle" align="right">1656</td><td rowspan="1" colspan="1" valign="middle" align="center">0.97</td><td rowspan="1" colspan="1" valign="middle" align="center">0.467</td><td rowspan="1" colspan="1" valign="middle" align="left">0.59</td><td rowspan="1" colspan="1" valign="middle" align="center">0.96</td><td rowspan="1" colspan="1" valign="middle" align="center">0.455</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">200-500</td><td rowspan="1" colspan="1" valign="middle" align="left">0.13</td><td rowspan="1" colspan="1" valign="middle" align="right">1201</td><td rowspan="1" colspan="1" valign="middle" align="center">1.07</td><td rowspan="1" colspan="1" valign="middle" align="center">0.324</td><td rowspan="1" colspan="1" valign="middle" align="left">0.49</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00</td><td rowspan="1" colspan="1" valign="middle" align="center">0.617</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">500+</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0048</td><td rowspan="1" colspan="1" valign="middle" align="right">490</td><td rowspan="1" colspan="1" valign="middle" align="center">1.28</td><td rowspan="1" colspan="1" valign="middle" align="center">0.122</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0001</td><td rowspan="1" colspan="1" valign="middle" align="center">2.00</td><td rowspan="1" colspan="1" valign="middle" align="center">0.557</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">100-200</td><td rowspan="1" colspan="1" valign="middle" align="left">0.38</td><td rowspan="1" colspan="1" valign="middle" align="right">731</td><td rowspan="1" colspan="1" valign="middle" align="center">1.03</td><td rowspan="1" colspan="1" valign="middle" align="center">0.201</td><td rowspan="1" colspan="1" valign="middle" align="left">0.28</td><td rowspan="1" colspan="1" valign="middle" align="center">1.28</td><td rowspan="1" colspan="1" valign="middle" align="center">0.127</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">200-500</td><td rowspan="1" colspan="1" valign="middle" align="left">0.056</td><td rowspan="1" colspan="1" valign="middle" align="right">369</td><td rowspan="1" colspan="1" valign="middle" align="center">1.19</td><td rowspan="1" colspan="1" valign="middle" align="center">0.095</td><td rowspan="1" colspan="1" valign="middle" align="left">0.13</td><td rowspan="1" colspan="1" valign="middle" align="center">1.28</td><td rowspan="1" colspan="1" valign="middle" align="center">0.122</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">500+</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0035</td><td rowspan="1" colspan="1" valign="middle" align="right">144</td><td rowspan="1" colspan="1" valign="middle" align="center">1.64</td><td rowspan="1" colspan="1" valign="middle" align="center">0.032</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0001</td><td rowspan="1" colspan="1" valign="middle" align="center">5.19</td><td rowspan="1" colspan="1" valign="middle" align="center">0.075</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">100-200</td><td rowspan="1" colspan="1" valign="middle" align="left">0.87</td><td rowspan="1" colspan="1" valign="middle" align="right">925</td><td rowspan="1" colspan="1" valign="middle" align="center">0.93</td><td rowspan="1" colspan="1" valign="middle" align="center">0.266</td><td rowspan="1" colspan="1" valign="middle" align="left">0.95</td><td rowspan="1" colspan="1" valign="middle" align="center">0.84</td><td rowspan="1" colspan="1" valign="middle" align="center">0.328</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">200–500</td><td rowspan="1" colspan="1" valign="middle" align="left">0.38</td><td rowspan="1" colspan="1" valign="middle" align="right">832</td><td rowspan="1" colspan="1" valign="middle" align="center">1.03</td><td rowspan="1" colspan="1" valign="middle" align="center">0.229</td><td rowspan="1" colspan="1" valign="middle" align="left">0.72</td><td rowspan="1" colspan="1" valign="middle" align="center">0.94</td><td rowspan="1" colspan="1" valign="middle" align="center">0.495</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">500+</td><td rowspan="1" colspan="1" valign="middle" align="left">0.12</td><td rowspan="1" colspan="1" valign="middle" align="right">346</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15</td><td rowspan="1" colspan="1" valign="middle" align="center">0.091</td><td rowspan="1" colspan="1" valign="middle" align="left">0.013</td><td rowspan="1" colspan="1" valign="middle" align="center">1.51</td><td rowspan="1" colspan="1" valign="middle" align="center">0.483</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<em>BD</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">100-200</td><td rowspan="1" colspan="1" valign="middle" align="left">0.21</td><td rowspan="1" colspan="1" valign="middle" align="right">1383</td><td rowspan="1" colspan="1" valign="middle" align="center">1.05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.467</td><td rowspan="1" colspan="1" valign="middle" align="left">0.93</td><td rowspan="1" colspan="1" valign="middle" align="center">0.84</td><td rowspan="1" colspan="1" valign="middle" align="center">0.455</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">200–500</td><td rowspan="1" colspan="1" valign="middle" align="left">0.14</td><td rowspan="1" colspan="1" valign="middle" align="right">971</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09</td><td rowspan="1" colspan="1" valign="middle" align="center">0.324</td><td rowspan="1" colspan="1" valign="middle" align="left">0.57</td><td rowspan="1" colspan="1" valign="middle" align="center">0.97</td><td rowspan="1" colspan="1" valign="middle" align="center">0.617</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All</td><td rowspan="1" colspan="1" valign="middle" align="left">500+</td><td rowspan="1" colspan="1" valign="middle" align="left">0.71</td><td rowspan="1" colspan="1" valign="middle" align="right">351</td><td rowspan="1" colspan="1" valign="middle" align="center">0.94</td><td rowspan="1" colspan="1" valign="middle" align="center">0.122</td><td rowspan="1" colspan="1" valign="middle" align="left">0.08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.35</td><td rowspan="1" colspan="1" valign="middle" align="center">0.557</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">100-200</td><td rowspan="1" colspan="1" valign="middle" align="left">0.43</td><td rowspan="1" colspan="1" valign="middle" align="right">591</td><td rowspan="1" colspan="1" valign="middle" align="center">1.02</td><td rowspan="1" colspan="1" valign="middle" align="center">0.201</td><td rowspan="1" colspan="1" valign="middle" align="left">0.76</td><td rowspan="1" colspan="1" valign="middle" align="center">0.83</td><td rowspan="1" colspan="1" valign="middle" align="center">0.127</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">200–500</td><td rowspan="1" colspan="1" valign="middle" align="left">0.079</td><td rowspan="1" colspan="1" valign="middle" align="right">295</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21</td><td rowspan="1" colspan="1" valign="middle" align="center">0.095</td><td rowspan="1" colspan="1" valign="middle" align="left">0.36</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11</td><td rowspan="1" colspan="1" valign="middle" align="center">0.122</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Deletions</td><td rowspan="1" colspan="1" valign="middle" align="left">500+</td><td rowspan="1" colspan="1" valign="middle" align="left">0.38</td><td rowspan="1" colspan="1" valign="middle" align="right">95</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">0.032</td><td rowspan="1" colspan="1" valign="middle" align="left">0.039</td><td rowspan="1" colspan="1" valign="middle" align="center">2.12</td><td rowspan="1" colspan="1" valign="middle" align="center">0.075</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">100-200</td><td rowspan="1" colspan="1" valign="middle" align="left">0.19</td><td rowspan="1" colspan="1" valign="middle" align="right">792</td><td rowspan="1" colspan="1" valign="middle" align="center">1.07</td><td rowspan="1" colspan="1" valign="middle" align="center">0.266</td><td rowspan="1" colspan="1" valign="middle" align="left">0.91</td><td rowspan="1" colspan="1" valign="middle" align="center">0.85</td><td rowspan="1" colspan="1" valign="middle" align="center">0.328</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">200–500</td><td rowspan="1" colspan="1" valign="middle" align="left">0.36</td><td rowspan="1" colspan="1" valign="middle" align="right">676</td><td rowspan="1" colspan="1" valign="middle" align="center">1.04</td><td rowspan="1" colspan="1" valign="middle" align="center">0.229</td><td rowspan="1" colspan="1" valign="middle" align="left">0.66</td><td rowspan="1" colspan="1" valign="middle" align="center">0.94</td><td rowspan="1" colspan="1" valign="middle" align="center">0.495</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Duplications</td><td rowspan="1" colspan="1" valign="middle" align="left">500+</td><td rowspan="1" colspan="1" valign="middle" align="left">0.81</td><td rowspan="1" colspan="1" valign="middle" align="right">256</td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center">0.091</td><td rowspan="1" colspan="1" valign="middle" align="left">0.18</td><td rowspan="1" colspan="1" valign="middle" align="center">1.23</td><td rowspan="1" colspan="1" valign="middle" align="center">0.483</td></tr></tbody></table></div><div id="largeobj_idm140337874632720" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3724337/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p9">Abbreviations: CNV, copy number variants; SCZ, schizophrenia; BD, bipolar disorder.</p></div></div><p id="P29" class="p p-last">Many large CNVs previously implicated in SCZ, BD, and other psychiatric and neurological conditions were also observed in this sample (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). SCZ subjects possessed significantly more duplications at the 16p11.2 locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 5</a>; nine cases, one control; <em>P</em> = 0.0035) and four SCZ cases had deletions at 22q11, and one control showed an atypical partial deletion (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 6</a>; <em>P</em> = 0.033). A novel, nominal association was also observed for duplications in SCZ and BD combined at 9q34.3 (31 cases, 11 controls; <em>P</em> = 0.0014, genome-wide <em>P</em> = 0.052; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 7</a>). Including the previously reported subjects (six SCZ duplications and one control) strengthened the association (<em>P</em> = 0.0008, genome-wide <em>P</em> = 0.023). However, no enrichment for duplications at this locus was observed in cases in another SCZ sample (International Schizophrenia Consortium, <em>P</em> = 0.37)<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261904">42</a></sup> or a combined SCZ–BD sample (Genetic Association Information Network, <em>P</em> = 0.41).<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916472">43</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916496">11</a></sup></p></div><div id="S21" class="sec sec-last"><h3 id="S21title">Copy number polymorphisms</h3><p id="P30" class="p p-first-last">CNPs represent an additional source of genetic variation that is rarely investigated, but potentially etiologically important. The top SNP in the full sample SCZ analyses, rs886424, (4.54 × 10<sup>−8</sup>), tags two CNPs (chr6: 30 994 409–30 994 549, <em>R</em><sup>2</sup> 0.91; chr6:30 338 498–30 338 570, <em>R</em><sup>2</sup> = 0.80), whereas another SNP in this region (rs3129953; <em>P</em> = 1.20 × 10<sup>−5</sup>) is in perfect linkage disequilibrium with a third CNP (chr6: 32 667 947–32 668 383, <em>R</em><sup>2</sup> = 1). These results indicate that at least three sub-kilobase deletions segregate with SCZ-associated alleles in the MHC (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>).</p></div></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">DISCUSSION</h2><p id="P31" class="p p-first">In a large Swedish sample, we found a genome-wide significant association in the MHC region for SCZ. This signal was attenuated by addition of subjects with BD. Our results are consistent with a growing body of research supporting MHC involvement in SCZ, but not in BD.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261900">10</a>-<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916481">12</a>,<a href="#R14" rid="R14" class=" bibr popnode">14</a>,<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916487">44</a></sup> This is the second most gene-dense region in the genome, and the association encompasses a multi-megabase region. In an effort to localize this signal, we imputed the MHC region using methods specifically tailored to this highly polymorphic region, and revealed nominal associations with HLA-B*0801 and HLA-A*0101 previously observed in the partially overlapping International Schizophrenia Consortium sample.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261908">10</a></sup> Meta-analysis with the PGC-SCZ results also implies a role for the MHC in SCZ, and additional investigations of CNPs uncovered further potentially relevant genetic variation in this region. Association of the MHC region in this genetically similar Swedish SCZ sample provides additional evidence that a false positive due to population stratification is unlikely to underlie this signal.</p><p id="P32">Although the MHC region contained the only genome-wide significant finding, several other loci attained nominal significance and merit further investigation. For the SCZ analysis in the full sample, the regulator of G-protein signaling 7 (RGS7) gene, functioning in inhibition of signal transduction and synaptic vesicle exocytosis, demonstrated the second strongest association. The top SNP for the new SCZ sample analysis falls in the neuronally expressed gene GALNT13, identified as one of the genes driving the significance of the ‘glycan structures biosynthesis 1’ pathway in a prior report.<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916463">45</a></sup> The same SNP was most significant in both the BD and the combined SCZ–BD analyses; however, this region lacks known genes, and association is not supported by the PGC–BD results (rs17746001, <em>P</em> = 0.083).<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916455">18</a></sup> The larger number of SCZ subjects confers greater power for detection of associations compared with the smaller BD sample, and the lack of genome-wide significant results for the latter should be considered in this context.</p><p id="P33">Despite the smaller sample and inherent decrease in power for BD, we generally observed similar (and higher) rates of CNVs in SCZ and BD compared with controls; however, statistical significance was more often attained in the much larger SCZ sample. Direct comparison of the two case groups revealed a significant increase for large CNVs (&gt;500 kb) in the SCZ group (<em>P</em> = 0.007), but not for any other size or frequency classes. This is compatible with the growing literature implicating increased burden of large, rare CNVs in SCZ;<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916471">46</a>,<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_360261906">42</a></sup> however, the role of CNVs in BD has been much less clear. We detected a significant excess of singleton deletions in BD as previously reported,<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916474">47</a></sup> but our results are also consistent with a prior observation of lower CNV rates in BD for other size and frequency categories,<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916492">48</a></sup> or no CNV enrichment in BD.<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_506231701">49</a></sup></p><p id="P34">The most significant specific CNV we observed, 16p11.2, showed more duplication events in SCZ than controls. This is not a novel finding in SCZ,<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916462">23</a>,<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_506231702">25</a></sup> nor is this locus only implicated in SCZ. Most notably, 16p11.2 duplications are one of the few CNVs with some evidence of association with BD,<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916459">23</a></sup> although only one subject in our BD sample had a duplication at this locus. CNVs at this locus have also shown repeated associations with autism as well as occurrences in cases of developmental delay and mental retardation, syringomyelia, and congenital kidney and urinary abnormalities.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916444">23</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916467">24</a>,<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916498">50</a>-<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_325916494">52</a></sup> The novel, nominally associated 160-kb duplications in the combined case group at 9q34.3 encompass one uncharacterized open-reading frame and are 60 kb from the brain-expressed gene <em>KIAA0649</em>, which may regulate cell proliferation.</p><p id="P35" class="p p-last">The relative homogeneity of this Swedish sample is one of its main strengths; however, replication of these findings in other (particularly non-European) samples is important. It is becoming increasingly clear that both rare CNVs and common SNPs contribute to the etiology of SCZ, although for BD, common genetic variation appears more salient. The mechanisms of action by which the genetic associations reported here and elsewhere confer risk for SCZ and BD are yet unknown. Establishing the genetic and environmental background predisposing carriers of shared risk loci to a particular diagnosis is also of critical importance. As genetic associations are identified and validated, their biological ramifications will become increasingly important to understand as we strive toward developing better diagnostic and therapeutic strategies.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="d36e1177"><a href="/pmc/articles/PMC3724337/bin/NIHMS494874-supplement-Supplementary_Information.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.6M, doc)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p1">We are deeply grateful for the participation of all subjects contributing to this research, and to the collection team that worked to recruit them: Emma Flordal-Thelander, Ann-Britt Holmgren, Marie Hallin, Marie Lundin, Ann-Kristin Sundberg, Christina Pettersson, Radja Satgunanthan-Dawoud, Sonja Hassellund, Malin Rådstrom, Birgitta Ohlander, Leila Nyrén, Isabelle Kizling, Louise Frisén, Inger Röhmer, Catharina Lavebratt, Malin Kärn, Martina Wennberg and Agneta Carswärd-Kjellin. We would also like to thank Professor Martin Schalling for facilitating collection of many subjects with BD. Funding support was provided by the Stanley Center for Psychiatric Research, Broad Institute from a grant from Stanley Medical Research Institute, NIMH <a id="__tag_807145311" class="tag_hotlink" href="/nuccore/MH077139" ref="/nuccore/MH077139"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH077139","term_id":"1364874077"}}</span>MH077139</a> (PFS), the Karolinska Institutet, Karolinska University Hospital, the Swedish Research Council, ALF grant from Swedish County Council and Söderström Königska Foundation.</p></div></div><div id="idm140337879382096" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140337879382096title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P36" class="p p-first-last">CONFLICT OF INTEREST
The authors declare no conflict of interest.</p><p></p></div></div><div id="idm140337879381104" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140337879381104title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Collins PY, Patel V, Joestl SS, March D, Insel TR, Daar AS, et al.  Grand challenges in global mental health. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">475</span>:27–30.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173804/">PMC free article</a>]</span> [<a href="/pubmed/21734685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Grand+challenges+in+global+mental+health&amp;author=PY+Collins&amp;author=V+Patel&amp;author=SS+Joestl&amp;author=D+March&amp;author=TR+Insel&amp;volume=475&amp;publication_year=2011&amp;pages=27-30&amp;pmid=21734685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Craddock N, Owen MJ. The Kraepelinian dichotomy - going, going. but still not gone. <span><span class="ref-journal">Br J Psychiatry. </span>2010;<span class="ref-vol">196</span>:92–95.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2815936/">PMC free article</a>]</span> [<a href="/pubmed/20118450" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Psychiatry&amp;title=The+Kraepelinian+dichotomy+-+going,+going.+but+still+not+gone&amp;author=N+Craddock&amp;author=MJ+Owen&amp;volume=196&amp;publication_year=2010&amp;pages=92-95&amp;pmid=20118450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Association AP.  <span class="ref-journal">Diagnostic and Statistical Manual of Mental Disorders.</span> 4th edn Association AP; Washington, DC: 1994.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+and+Statistical+Manual+of+Mental+Disorders&amp;author=AP+Association&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Perlis RH, Miyahara S, Marangell LB, Wisniewski SR, Ostacher M, DelBello MP, et al.  Long-term implications of early onset in bipolar disorder: data from the first 1000 participants in the systematic treatment enhancement program for bipolar disorder (STEP-BD) <span><span class="ref-journal">Biol Psychiatry. </span>2004;<span class="ref-vol">55</span>:875–881.</span> [<a href="/pubmed/15110730" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Long-term+implications+of+early+onset+in+bipolar+disorder:+data+from+the+first+1000+participants+in+the+systematic+treatment+enhancement+program+for+bipolar+disorder+(STEP-BD)&amp;author=RH+Perlis&amp;author=S+Miyahara&amp;author=LB+Marangell&amp;author=SR+Wisniewski&amp;author=M+Ostacher&amp;volume=55&amp;publication_year=2004&amp;pages=875-881&amp;pmid=15110730&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Angermeyer MC, Kuhn L. Gender differences in age at onset of schizophrenia. An overview. <span><span class="ref-journal">Eur Arch Psychiatry Neurol Sci. </span>1988;<span class="ref-vol">237</span>:351–364.</span> [<a href="/pubmed/3053193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+Arch+Psychiatry+Neurol+Sci&amp;title=Gender+differences+in+age+at+onset+of+schizophrenia.+An+overview&amp;author=MC+Angermeyer&amp;author=L+Kuhn&amp;volume=237&amp;publication_year=1988&amp;pages=351-364&amp;pmid=3053193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Saha S, Chant D, Welham J, McGrath J. A systematic review of the prevalence of schizophrenia. <span><span class="ref-journal">PLoS Med. </span>2005;<span class="ref-vol">2</span>:e141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1140952/">PMC free article</a>]</span> [<a href="/pubmed/15916472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med&amp;title=A+systematic+review+of+the+prevalence+of+schizophrenia&amp;author=S+Saha&amp;author=D+Chant&amp;author=J+Welham&amp;author=J+McGrath&amp;volume=2&amp;publication_year=2005&amp;pages=e141&amp;pmid=15916472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Ivleva EI, Morris DW, Moates AF, Suppes T, Thaker GK, Tamminga CA. Genetics and intermediate phenotypes of the schizophrenia–bipolar disorder boundary. <span><span class="ref-journal">Neurosci Biobehav Rev. </span>2010;<span class="ref-vol">34</span>:897–921.</span> [<a href="/pubmed/19954751" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurosci+Biobehav+Rev&amp;title=Genetics+and+intermediate+phenotypes+of+the+schizophrenia–bipolar+disorder+boundary&amp;author=EI+Ivleva&amp;author=DW+Morris&amp;author=AF+Moates&amp;author=T+Suppes&amp;author=GK+Thaker&amp;volume=34&amp;publication_year=2010&amp;pages=897-921&amp;pmid=19954751&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Smoller JW, Finn CT. Family, twin, and adoption studies of bipolar disorder. <span><span class="ref-journal">Am J Med Genet C Semin Med Genet. </span>2003;<span class="ref-vol">123C</span>:48–58.</span> [<a href="/pubmed/14601036" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+C+Semin+Med+Genet&amp;title=Family,+twin,+and+adoption+studies+of+bipolar+disorder&amp;author=JW+Smoller&amp;author=CT+Finn&amp;volume=123C&amp;publication_year=2003&amp;pages=48-58&amp;pmid=14601036&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Lichtenstein P, Yip BH, Bjork C, Pawitan Y, Cannon TD, Sullivan PF, et al.  Common genetic determinants of schizophrenia and bipolar disorder in Swedish families: a population-based study. <span><span class="ref-journal">Lancet. </span>2009;<span class="ref-vol">373</span>:234–239.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3879718/">PMC free article</a>]</span> [<a href="/pubmed/19150704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Common+genetic+determinants+of+schizophrenia+and+bipolar+disorder+in+Swedish+families:+a+population-based+study&amp;author=P+Lichtenstein&amp;author=BH+Yip&amp;author=C+Bjork&amp;author=Y+Pawitan&amp;author=TD+Cannon&amp;volume=373&amp;publication_year=2009&amp;pages=234-239&amp;pmid=19150704&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Purcell SM, Wray NR, Stone JL, Visscher PM, O’Donovan MC, Sullivan PF, et al.  Common polygenic variation contributes to risk of schizophrenia and bipolar disorder. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:748–752.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia+and+bipolar+disorder&amp;author=SM+Purcell&amp;author=NR+Wray&amp;author=JL+Stone&amp;author=PM+Visscher&amp;author=MC+O’Donovan&amp;volume=460&amp;publication_year=2009&amp;pages=748-752&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Shi J, Levinson DF, Duan J, Sanders AR, Zheng Y, Pe’er I, et al.  Common variants on chromosome 6p22.1 are associated with schizophrenia. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:753–757.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775422/">PMC free article</a>]</span> [<a href="/pubmed/19571809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+variants+on+chromosome+6p22.1+are+associated+with+schizophrenia&amp;author=J+Shi&amp;author=DF+Levinson&amp;author=J+Duan&amp;author=AR+Sanders&amp;author=Y+Zheng&amp;volume=460&amp;publication_year=2009&amp;pages=753-757&amp;pmid=19571809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Stefansson H, Ophoff RA, Steinberg S, Andreassen OA, Cichon S, Rujescu D, et al.  Common variants conferring risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:744–747.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3077530/">PMC free article</a>]</span> [<a href="/pubmed/19571808" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+variants+conferring+risk+of+schizophrenia&amp;author=H+Stefansson&amp;author=RA+Ophoff&amp;author=S+Steinberg&amp;author=OA+Andreassen&amp;author=S+Cichon&amp;volume=460&amp;publication_year=2009&amp;pages=744-747&amp;pmid=19571808&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Steinberg S, de Jong S, Andreassen OA, Werge T, Borglum AD, Mors O, et al.  Common variants at VRK2 and TCF4 conferring risk of schizophrenia. <span><span class="ref-journal">Hum Mol Genet. </span>2011;<span class="ref-vol">20</span>:4076–4081.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3298077/">PMC free article</a>]</span> [<a href="/pubmed/21791550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Common+variants+at+VRK2+and+TCF4+conferring+risk+of+schizophrenia&amp;author=S+Steinberg&amp;author=S+de+Jong&amp;author=OA+Andreassen&amp;author=T+Werge&amp;author=AD+Borglum&amp;volume=20&amp;publication_year=2011&amp;pages=4076-4081&amp;pmid=21791550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Ripke S, Sanders AR, Kendler KS, Levinson DF, Sklar P, Holmans PA, et al.  Genome-wide association study identifies five new schizophrenia loci. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:969–976.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+five+new+schizophrenia+loci&amp;author=S+Ripke&amp;author=AR+Sanders&amp;author=KS+Kendler&amp;author=DF+Levinson&amp;author=P+Sklar&amp;volume=43&amp;publication_year=2011&amp;pages=969-976&amp;pmid=21926974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Baum AE, Akula N, Cabanero M, Cardona I, Corona W, Klemens B, et al.  A genome-wide association study implicates diacylglycerol kinase eta (DGKH) and several other genes in the etiology of bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2008;<span class="ref-vol">13</span>:197–207.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2527618/">PMC free article</a>]</span> [<a href="/pubmed/17486107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=A+genome-wide+association+study+implicates+diacylglycerol+kinase+eta+(DGKH)+and+several+other+genes+in+the+etiology+of+bipolar+disorder&amp;author=AE+Baum&amp;author=N+Akula&amp;author=M+Cabanero&amp;author=I+Cardona&amp;author=W+Corona&amp;volume=13&amp;publication_year=2008&amp;pages=197-207&amp;pmid=17486107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Ferreira MA, O’Donovan MC, Meng YA, Jones IR, Ruderfer DM, Jones L, et al.  Collaborative genome-wide association analysis supports a role for ANK3 and CACNA1C in bipolar disorder. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1056–1058.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2703780/">PMC free article</a>]</span> [<a href="/pubmed/18711365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Collaborative+genome-wide+association+analysis+supports+a+role+for+ANK3+and+CACNA1C+in+bipolar+disorder&amp;author=MA+Ferreira&amp;author=MC+O’Donovan&amp;author=YA+Meng&amp;author=IR+Jones&amp;author=DM+Ruderfer&amp;volume=40&amp;publication_year=2008&amp;pages=1056-1058&amp;pmid=18711365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Cichon S, Muhleisen TW, Degenhardt FA, Mattheisen M, Miro X, Strohmaier J, et al.  Genome-wide association study identifies genetic variation in neurocan as a susceptibility factor for bipolar disorder. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:372–381.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059436/">PMC free article</a>]</span> [<a href="/pubmed/21353194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+association+study+identifies+genetic+variation+in+neurocan+as+a+susceptibility+factor+for+bipolar+disorder&amp;author=S+Cichon&amp;author=TW+Muhleisen&amp;author=FA+Degenhardt&amp;author=M+Mattheisen&amp;author=X+Miro&amp;volume=88&amp;publication_year=2011&amp;pages=372-381&amp;pmid=21353194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Sklar P, Ripke S, Scott LJ, Andreassen OA, Cichon S, Craddock N, et al.  Large-scale genome-wide association analysis of bipolar disorder identifies a new susceptibility locus near ODZ4. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:977–983.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3637176/">PMC free article</a>]</span> [<a href="/pubmed/21926972" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Large-scale+genome-wide+association+analysis+of+bipolar+disorder+identifies+a+new+susceptibility+locus+near+ODZ4&amp;author=P+Sklar&amp;author=S+Ripke&amp;author=LJ+Scott&amp;author=OA+Andreassen&amp;author=S+Cichon&amp;volume=43&amp;publication_year=2011&amp;pages=977-983&amp;pmid=21926972&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Guilmatre A, Dubourg C, Mosca AL, Legallic S, Goldenberg A, Drouin-Garraud V, et al.  Recurrent rearrangements in synaptic and neurodevelopmental genes and shared biologic pathways in schizophrenia, autism, and mental retardation. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2009;<span class="ref-vol">66</span>:947–956.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2958844/">PMC free article</a>]</span> [<a href="/pubmed/19736351" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Recurrent+rearrangements+in+synaptic+and+neurodevelopmental+genes+and+shared+biologic+pathways+in+schizophrenia,+autism,+and+mental+retardation&amp;author=A+Guilmatre&amp;author=C+Dubourg&amp;author=AL+Mosca&amp;author=S+Legallic&amp;author=A+Goldenberg&amp;volume=66&amp;publication_year=2009&amp;pages=947-956&amp;pmid=19736351&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Mefford HC, Muhle H, Ostertag P, von Spiczak S, Buysse K, Baker C, et al.  Genome-wide copy number variation in epilepsy: novel susceptibility loci in idiopathic generalized and focal epilepsies. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2873910/">PMC free article</a>]</span> [<a href="/pubmed/20502679" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+copy+number+variation+in+epilepsy:+novel+susceptibility+loci+in+idiopathic+generalized+and+focal+epilepsies&amp;author=HC+Mefford&amp;author=H+Muhle&amp;author=P+Ostertag&amp;author=S+von+Spiczak&amp;author=K+Buysse&amp;volume=6&amp;publication_year=2010&amp;pages=e1000962&amp;pmid=20502679&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Moreno-De-Luca D, Mulle JG, Kaminsky EB, Sanders SJ, Myers SM, Adam MP, et al.  Deletion 17q12 is a recurrent copy number variant that confers high risk of autism and schizophrenia. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:618–630.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2978962/">PMC free article</a>]</span> [<a href="/pubmed/21055719" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Deletion+17q12+is+a+recurrent+copy+number+variant+that+confers+high+risk+of+autism+and+schizophrenia&amp;author=D+Moreno-De-Luca&amp;author=JG+Mulle&amp;author=EB+Kaminsky&amp;author=SJ+Sanders&amp;author=SM+Myers&amp;volume=87&amp;publication_year=2010&amp;pages=618-630&amp;pmid=21055719&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Sebat J, Levy DL, McCarthy SE. Rare structural variants in schizophrenia: one disorder, multiple mutations; one mutation, multiple disorders. <span><span class="ref-journal">Trends Genet. </span>2009;<span class="ref-vol">25</span>:528–535.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3351381/">PMC free article</a>]</span> [<a href="/pubmed/19883952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=Rare+structural+variants+in+schizophrenia:+one+disorder,+multiple+mutations;+one+mutation,+multiple+disorders&amp;author=J+Sebat&amp;author=DL+Levy&amp;author=SE+McCarthy&amp;volume=25&amp;publication_year=2009&amp;pages=528-535&amp;pmid=19883952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">McCarthy SE, Makarov V, Kirov G, Addington AM, McClellan J, Yoon S, et al.  Microduplications of 16p11.2 are associated with schizophrenia. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1223–1227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2951180/">PMC free article</a>]</span> [<a href="/pubmed/19855392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Microduplications+of+16p11.2+are+associated+with+schizophrenia&amp;author=SE+McCarthy&amp;author=V+Makarov&amp;author=G+Kirov&amp;author=AM+Addington&amp;author=J+McClellan&amp;volume=41&amp;publication_year=2009&amp;pages=1223-1227&amp;pmid=19855392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Weiss LA, Shen Y, Korn JM, Arking DE, Miller DT, Fossdal R, et al.  Association between microdeletion and microduplication at 16p11.2 and autism. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:667–675.</span> [<a href="/pubmed/18184952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Association+between+microdeletion+and+microduplication+at+16p11.2+and+autism&amp;author=LA+Weiss&amp;author=Y+Shen&amp;author=JM+Korn&amp;author=DE+Arking&amp;author=DT+Miller&amp;volume=358&amp;publication_year=2008&amp;pages=667-675&amp;pmid=18184952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Levinson DF, Duan J, Oh S, Wang K, Sanders AR, Shi J, et al.  Copy number variants in schizophrenia: confirmation of five previous findings and new evidence for 3q29 microdeletions and VIPR2 duplications. <span><span class="ref-journal">Am J Psychiatry. </span>2011;<span class="ref-vol">168</span>:302–316.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4441324/">PMC free article</a>]</span> [<a href="/pubmed/21285140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Copy+number+variants+in+schizophrenia:+confirmation+of+five+previous+findings+and+new+evidence+for+3q29+microdeletions+and+VIPR2+duplications&amp;author=DF+Levinson&amp;author=J+Duan&amp;author=S+Oh&amp;author=K+Wang&amp;author=AR+Sanders&amp;volume=168&amp;publication_year=2011&amp;pages=302-316&amp;pmid=21285140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Kristiansson EAP, Wistedt B. Validity of the diagnosis of schizophrenia in a psychiatric inpatient register. <span><span class="ref-journal">Nordisk Psykiatrik Tidsskrift. </span>1987;<span class="ref-vol">41</span>:229–234.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nordisk+Psykiatrik+Tidsskrift&amp;title=Validity+of+the+diagnosis+of+schizophrenia+in+a+psychiatric+inpatient+register&amp;author=EAP+Kristiansson&amp;author=B+Wistedt&amp;volume=41&amp;publication_year=1987&amp;pages=229-234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Dalman C, Broms J, Cullberg J, Allebeck P. Young cases of schizophrenia identified in a national inpatient register–are the diagnoses valid? <span><span class="ref-journal">Soc Psychiatry Psychiatr Epidemiol. </span>2002;<span class="ref-vol">37</span>:527–531.</span> [<a href="/pubmed/12395142" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Soc+Psychiatry+Psychiatr+Epidemiol&amp;title=Young+cases+of+schizophrenia+identified+in+a+national+inpatient+register–are+the+diagnoses+valid?&amp;author=C+Dalman&amp;author=J+Broms&amp;author=J+Cullberg&amp;author=P+Allebeck&amp;volume=37&amp;publication_year=2002&amp;pages=527-531&amp;pmid=12395142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Ekholm B, Ekholm A, Adolfsson R, Vares M, Osby U, Sedvall GC, et al.  Evaluation of diagnostic procedures in Swedish patients with schizophrenia and related psychoses. <span><span class="ref-journal">Nord J Psychiatry. </span>2005;<span class="ref-vol">59</span>:457–464.</span> [<a href="/pubmed/16316898" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nord+J+Psychiatry&amp;title=Evaluation+of+diagnostic+procedures+in+Swedish+patients+with+schizophrenia+and+related+psychoses&amp;author=B+Ekholm&amp;author=A+Ekholm&amp;author=R+Adolfsson&amp;author=M+Vares&amp;author=U+Osby&amp;volume=59&amp;publication_year=2005&amp;pages=457-464&amp;pmid=16316898&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Sellgren C, Landen M, Lichtenstein P, Hultman CM, Langstrom N. Validity of bipolar disorder hospital discharge diagnoses: file review and multiple register linkage in Sweden. <span><span class="ref-journal">Acta Psychiatr Scand. </span>2011;<span class="ref-vol">124</span>:447–453.</span> [<a href="/pubmed/21838734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Psychiatr+Scand&amp;title=Validity+of+bipolar+disorder+hospital+discharge+diagnoses:+file+review+and+multiple+register+linkage+in+Sweden&amp;author=C+Sellgren&amp;author=M+Landen&amp;author=P+Lichtenstein&amp;author=CM+Hultman&amp;author=N+Langstrom&amp;volume=124&amp;publication_year=2011&amp;pages=447-453&amp;pmid=21838734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Ryden E, Thase ME, Straht D, Aberg-Wistedt A, Bejerot S, Landen M. A history of childhood attention-deficit hyperactivity disorder (ADHD) impacts clinical outcome in adult bipolar patients regardless of current ADHD. <span><span class="ref-journal">Acta Psychiatr Scand. </span>2009;<span class="ref-vol">120</span>:239–246.</span> [<a href="/pubmed/19426162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Psychiatr+Scand&amp;title=A+history+of+childhood+attention-deficit+hyperactivity+disorder+(ADHD)+impacts+clinical+outcome+in+adult+bipolar+patients+regardless+of+current+ADHD&amp;author=E+Ryden&amp;author=ME+Thase&amp;author=D+Straht&amp;author=A+Aberg-Wistedt&amp;author=S+Bejerot&amp;volume=120&amp;publication_year=2009&amp;pages=239-246&amp;pmid=19426162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">First MBSRL, Gibbon M, Williams JBW.  <span class="ref-journal">Structured Clinical Interview for DSM-IV Axis I Disorders, Clinician Version (SCID-CV)</span> American Psychiatric Press, Inc; Arlington, VA, USA: 1996.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Structured+Clinical+Interview+for+DSM-IV+Axis+I+Disorders,+Clinician+Version+(SCID-CV)&amp;author=MBSRL+First&amp;author=M+Gibbon&amp;author=JBW+Williams&amp;publication_year=1996&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Korn JM, Kuruvilla FG, McCarroll SA, Wysoker A, Nemesh J, Cawley S, et al.  Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1253–1260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=JM+Korn&amp;author=FG+Kuruvilla&amp;author=SA+McCarroll&amp;author=A+Wysoker&amp;author=J+Nemesh&amp;volume=40&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Browning SR, Browning BL. Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:1084–1097.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265661/">PMC free article</a>]</span> [<a href="/pubmed/17924348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Rapid+and+accurate+haplotype+phasing+and+missing-data+inference+for+whole-genome+association+studies+by+use+of+localized+haplotype+clustering&amp;author=SR+Browning&amp;author=BL+Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Altshuler DM, Gibbs RA, Peltonen L, Dermitzakis E, Schaffner SF, Yu F, et al.  Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;author=DM+Altshuler&amp;author=RA+Gibbs&amp;author=L+Peltonen&amp;author=E+Dermitzakis&amp;author=SF+Schaffner&amp;volume=467&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">McMichael AJ, Jones EY. Genetics. First-class control of HIV-1. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">330</span>:1488–1490.</span> [<a href="/pubmed/21148380" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetics.+First-class+control+of+HIV-1&amp;author=AJ+McMichael&amp;author=EY+Jones&amp;volume=330&amp;publication_year=2010&amp;pages=1488-1490&amp;pmid=21148380&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Pe’er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe’er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Lee PH, O’Dushlaine C, Purcell SM. Inrich: interval-based enrichment analysis for genome-wide association studies. <span><span class="ref-journal">Bioinformatics advance online publication. </span>2012 Apr 17;</span> e-pub ahead of print. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3381960/">PMC free article</a>]</span> [<a href="/pubmed/22513993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics+advance+online+publication&amp;title=Inrich:+interval-based+enrichment+analysis+for+genome-wide+association+studies&amp;author=PH+Lee&amp;author=C+O’Dushlaine&amp;author=SM+Purcell&amp;publication_year=2012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Moskvina V, O’Dushlaine C, Purcell S, Craddock N, Holmans P, O’Donovan MC. Evaluation of an approximation method for assessment of overall significance of multiple-dependent tests in a genomewide association study. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:861–866.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3268180/">PMC free article</a>]</span> [<a href="/pubmed/22006681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Evaluation+of+an+approximation+method+for+assessment+of+overall+significance+of+multiple-dependent+tests+in+a+genomewide+association+study&amp;author=V+Moskvina&amp;author=C+O’Dushlaine&amp;author=S+Purcell&amp;author=N+Craddock&amp;author=P+Holmans&amp;volume=35&amp;publication_year=2011&amp;pages=861-866&amp;pmid=22006681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, Andrews TD, et al.  Global variation in copy number in the human genome. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">444</span>:444–454.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669898/">PMC free article</a>]</span> [<a href="/pubmed/17122850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Global+variation+in+copy+number+in+the+human+genome&amp;author=R+Redon&amp;author=S+Ishikawa&amp;author=KR+Fitch&amp;author=L+Feuk&amp;author=GH+Perry&amp;volume=444&amp;publication_year=2006&amp;pages=444-454&amp;pmid=17122850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Handsaker RE, Korn JM, Nemesh J, McCarroll SA. Discovery and genotyping of genome structural polymorphism by sequencing on a population scale. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:269–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5094049/">PMC free article</a>]</span> [<a href="/pubmed/21317889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Discovery+and+genotyping+of+genome+structural+polymorphism+by+sequencing+on+a+population+scale&amp;author=RE+Handsaker&amp;author=JM+Korn&amp;author=J+Nemesh&amp;author=SA+McCarroll&amp;volume=43&amp;publication_year=2011&amp;pages=269-276&amp;pmid=21317889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">International Schizophrenia Consortium  Rare chromosomal deletions and duplications increase risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:237–241.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912847/">PMC free article</a>]</span> [<a href="/pubmed/18668038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Rare+chromosomal+deletions+and+duplications+increase+risk+of+schizophrenia&amp;volume=455&amp;publication_year=2008&amp;pages=237-241&amp;pmid=18668038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Smith EN, Bloss CS, Badner JA, Barrett T, Belmonte PL, Berrettini W, et al.  Genome-wide association study of bipolar disorder in European American and African American individuals. <span><span class="ref-journal">Mol Psychiatry. </span>2009;<span class="ref-vol">14</span>:755–763.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3035981/">PMC free article</a>]</span> [<a href="/pubmed/19488044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Genome-wide+association+study+of+bipolar+disorder+in+European+American+and+African+American+individuals&amp;author=EN+Smith&amp;author=CS+Bloss&amp;author=JA+Badner&amp;author=T+Barrett&amp;author=PL+Belmonte&amp;volume=14&amp;publication_year=2009&amp;pages=755-763&amp;pmid=19488044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Lee KW, Woon PS, Teo YY, Sim K. Genome wide association studies (GWAS) and copy number variation (CNV) studies of the major psychoses: What have we learnt? <span><span class="ref-journal">Neurosci Biobehav Rev. </span>2012;<span class="ref-vol">36</span>:556–571.</span> [<a href="/pubmed/21946175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurosci+Biobehav+Rev&amp;title=Genome+wide+association+studies+(GWAS)+and+copy+number+variation+(CNV)+studies+of+the+major+psychoses:+What+have+we+learnt?&amp;author=KW+Lee&amp;author=PS+Woon&amp;author=YY+Teo&amp;author=K+Sim&amp;volume=36&amp;publication_year=2012&amp;pages=556-571&amp;pmid=21946175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">O’Dushlaine C, Kenny E, Heron E, Donohoe G, Gill M, Morris D, et al.  Molecular pathways involved in neuronal cell adhesion and membrane scaffolding contribute to schizophrenia and bipolar disorder susceptibility. <span><span class="ref-journal">Mol Psychiatry. </span>2011;<span class="ref-vol">16</span>:286–292.</span> [<a href="/pubmed/20157312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Molecular+pathways+involved+in+neuronal+cell+adhesion+and+membrane+scaffolding+contribute+to+schizophrenia+and+bipolar+disorder+susceptibility&amp;author=C+O’Dushlaine&amp;author=E+Kenny&amp;author=E+Heron&amp;author=G+Donohoe&amp;author=M+Gill&amp;volume=16&amp;publication_year=2011&amp;pages=286-292&amp;pmid=20157312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Walsh T, McClellan JM, McCarthy SE, Addington AM, Pierce SB, Cooper GM, et al.  Rare structural variants disrupt multiple genes in neurodevelopmental pathways in schizophrenia. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">320</span>:539–543.</span> [<a href="/pubmed/18369103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Rare+structural+variants+disrupt+multiple+genes+in+neurodevelopmental+pathways+in+schizophrenia&amp;author=T+Walsh&amp;author=JM+McClellan&amp;author=SE+McCarthy&amp;author=AM+Addington&amp;author=SB+Pierce&amp;volume=320&amp;publication_year=2008&amp;pages=539-543&amp;pmid=18369103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Zhang D, Cheng L, Qian Y, Alliey-Rodriguez N, Kelsoe JR, Greenwood T, et al.  Singleton deletions throughout the genome increase risk of bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2009;<span class="ref-vol">14</span>:376–380.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2735188/">PMC free article</a>]</span> [<a href="/pubmed/19114987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Singleton+deletions+throughout+the+genome+increase+risk+of+bipolar+disorder&amp;author=D+Zhang&amp;author=L+Cheng&amp;author=Y+Qian&amp;author=N+Alliey-Rodriguez&amp;author=JR+Kelsoe&amp;volume=14&amp;publication_year=2009&amp;pages=376-380&amp;pmid=19114987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">McQuillin A, Bass N, Anjorin A, Lawrence J, Kandaswamy R, Lydall G, et al.  Analysis of genetic deletions and duplications in the University College London bipolar disorder case control sample. <span><span class="ref-journal">Eur J Hum Genet. </span>2011;<span class="ref-vol">19</span>:588–592.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3083610/">PMC free article</a>]</span> [<a href="/pubmed/21206513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Analysis+of+genetic+deletions+and+duplications+in+the+University+College+London+bipolar+disorder+case+control+sample&amp;author=A+McQuillin&amp;author=N+Bass&amp;author=A+Anjorin&amp;author=J+Lawrence&amp;author=R+Kandaswamy&amp;volume=19&amp;publication_year=2011&amp;pages=588-592&amp;pmid=21206513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Grozeva D, Kirov G, Ivanov D, Jones IR, Jones L, Green EK, et al.  Rare copy number variants: a point of rarity in genetic risk for bipolar disorder and schizophrenia. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2010;<span class="ref-vol">67</span>:318–327.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4476027/">PMC free article</a>]</span> [<a href="/pubmed/20368508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Rare+copy+number+variants:+a+point+of+rarity+in+genetic+risk+for+bipolar+disorder+and+schizophrenia&amp;author=D+Grozeva&amp;author=G+Kirov&amp;author=D+Ivanov&amp;author=IR+Jones&amp;author=L+Jones&amp;volume=67&amp;publication_year=2010&amp;pages=318-327&amp;pmid=20368508&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Shinawi M, Liu P, Kang SH, Shen J, Belmont JW, Scott DA, et al.  Recurrent reciprocal 16p11.2 rearrangements associated with global developmental delay, behavioural problems, dysmorphism, epilepsy, and abnormal head size. <span><span class="ref-journal">J Med Genet. </span>2010;<span class="ref-vol">47</span>:332–341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3158566/">PMC free article</a>]</span> [<a href="/pubmed/19914906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Recurrent+reciprocal+16p11.2+rearrangements+associated+with+global+developmental+delay,+behavioural+problems,+dysmorphism,+epilepsy,+and+abnormal+head+size&amp;author=M+Shinawi&amp;author=P+Liu&amp;author=SH+Kang&amp;author=J+Shen&amp;author=JW+Belmont&amp;volume=47&amp;publication_year=2010&amp;pages=332-341&amp;pmid=19914906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Schaaf CP, Goin-Kochel RP, Nowell KP, Hunter JV, Aleck KA, Cox S, et al.  Expanding the clinical spectrum of the 16p11.2 chromosomal rearrangements: three patients with syringomyelia. <span><span class="ref-journal">Eur J Hum Genet. </span>2011;<span class="ref-vol">19</span>:152–156.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025795/">PMC free article</a>]</span> [<a href="/pubmed/20959866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Expanding+the+clinical+spectrum+of+the+16p11.2+chromosomal+rearrangements:+three+patients+with+syringomyelia&amp;author=CP+Schaaf&amp;author=RP+Goin-Kochel&amp;author=KP+Nowell&amp;author=JV+Hunter&amp;author=KA+Aleck&amp;volume=19&amp;publication_year=2011&amp;pages=152-156&amp;pmid=20959866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Sampson MG, Coughlin CR, II, Kaplan P, Conlin LK, Meyers KE, Zackai EH, et al.  Evidence for a recurrent microdeletion at chromosome 16p11.2 associated with congenital anomalies of the kidney and urinary tract (CAKUT) and Hirschsprung disease. <span><span class="ref-journal">Am J Med Genet A. </span>2010;<span class="ref-vol">152A</span>:2618–2622.</span> [<a href="/pubmed/20799338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Evidence+for+a+recurrent+microdeletion+at+chromosome+16p11.2+associated+with+congenital+anomalies+of+the+kidney+and+urinary+tract+(CAKUT)+and+Hirschsprung+disease&amp;author=MG+Sampson&amp;author=CR+Coughlin&amp;author=P+Kaplan&amp;author=LK+Conlin&amp;author=KE+Meyers&amp;volume=152A&amp;publication_year=2010&amp;pages=2618-2622&amp;pmid=20799338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3724337&amp;issue-id=225606&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140337879343504title">Abstract</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">SUBJECTS AND METHODS</a></li><li><a href="#S14title">RESULTS</a></li><li><a href="#S22title">DISCUSSION</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S23title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140337879382096title">Footnotes</a></li><li><a href="#idm140337879381104title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3724337/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3724337/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3724337/pdf/nihms-494874.pdf">PDF (318K)</a></li> | <li><a href="#" data-citationid="PMC3724337" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3724337%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3724337%2F&amp;text=Genome-wide%20association%20study%20in%20a%20Swedish%20population%20yields%20support%20for%20greater%20CNV%20and%20MHC%20involvement%20in%20schizophrenia%20compared%20with%20bipolar%20disorder" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3724337%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22688191" data-aiid="3724337" data-aid="3724337" data-iid="225606" data-domainid="319" data-domain="nihpa" data-accid="PMC3724337" data-md5="b8948ca8f0a28ce6295021ce096dd717"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:06:11-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B3A08DD538B21000000000AC5074C&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3724337%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3724337/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size1, sans-serif;"></div></div></body></html>