<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3376268/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3376268/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data"><meta name="citation_authors" content="Jie Huang, David Ellinghaus, Andre Franke, Bryan Howie, Yun Li"><meta name="citation_date" content="July 2012"><meta name="citation_issue" content="7"><meta name="citation_volume" content="20"><meta name="citation_firstpage" content="801"><meta name="citation_doi" content="10.1038/ejhg.2012.3"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3376268/?report=abstract"><meta name="citation_pmid" content="22293688"><meta name="DC.Title" content="1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Jie Huang"><meta name="DC.Contributor" content="David Ellinghaus"><meta name="DC.Contributor" content="Andre Franke"><meta name="DC.Contributor" content="Bryan Howie"><meta name="DC.Contributor" content="Yun Li"><meta name="DC.Date" content="2012 Jul"><meta name="DC.Identifier" content="10.1038/ejhg.2012.3"><meta name="DC.Language" content="en"><meta property="og:title" content="1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data"><meta property="og:type" content="article"><meta property="og:description" content="We hypothesize that imputation based on data from the 1000 Genomes Project can identify novel association signals on a genome-wide scale due to the dense marker map and the large number of haplotypes. To test the hypothesis, the Wellcome Trust Case Control ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3376268/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3376268/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3376268/pdf/ejhg20123a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C54F5DD53765100000000051F0388.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3376268%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/211083/">v.20(7); 2012 Jul</a></li><li class="accid">PMC3376268</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2012 Jul; 20(7): 801–805. </span></div><div><span class="fm-vol-iss-date">Published online 2012 Feb 1. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2012.3" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2012.3</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3376268</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22293688">22293688</a></div></div></div></div><h1 class="content-title">1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Huang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688" class="affpopup" co-rid="_co_idm139698344565152" co-class="co-affbox">Jie Huang</a>,<sup>1</sup> <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688" class="affpopup" co-rid="_co_idm139698350222736" co-class="co-affbox">David Ellinghaus</a>,<sup>2</sup> <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688" class="affpopup" co-rid="_co_idm139698344562448" co-class="co-affbox">Andre Franke</a>,<sup>2</sup> <a href="/pubmed/?term=Howie%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688" class="affpopup" co-rid="_co_idm139698344560224" co-class="co-affbox">Bryan Howie</a>,<sup>3</sup> and  <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688" class="affpopup" co-rid="_co_idm139698344558000" co-class="co-affbox">Yun Li</a><sup>4,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139698344565152"><h3 class="no_margin">Jie Huang</h3><p><sup>1</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Huang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688">Jie Huang</a></div></div><div id="_co_idm139698350222736"><h3 class="no_margin">David Ellinghaus</h3><p><sup>2</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688">David Ellinghaus</a></div></div><div id="_co_idm139698344562448"><h3 class="no_margin">Andre Franke</h3><p><sup>2</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688">Andre Franke</a></div></div><div id="_co_idm139698344560224"><h3 class="no_margin">Bryan Howie</h3><p><sup>3</sup>Department of Human Genetics, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Howie%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688">Bryan Howie</a></div></div><div id="_co_idm139698344558000"><h3 class="no_margin">Yun Li</h3><p><sup>4</sup>Department of Genetics, Department of Biostatistics, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22293688">Yun Li</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139698344501968_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139698344501968_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139698344501968_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139698344501968_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, Kiel, Germany</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Department of Human Genetics, University of Chicago, Chicago, IL, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Genetics, Department of Biostatistics, University of North Carolina, Chapel Hill, NC, USA</div><div id="caf1"><sup>*</sup>Department of Genetics, Department of Biostatistics, University of North Carolina Chapel Hill, Chapel Hill, NC, 27599-7264, USA. Tel: +919 843 2832; Fax: +919 843 4682; E-mail: <a href="mailto:dev@null" data-email="ude.cnu.dem@ilnuy" class="oemail">ude.cnu.dem@ilnuy</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139698344501968_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Sep 7; Revised 2011 Dec 30; Accepted 2012 Jan 4.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139698344501968_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2012 Macmillan Publishers Limited</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3376268/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC3376268/bin/ejhg20123x1.doc" data-ga-action="click_feat_suppl">ejhg20123x1.doc</a><span style="color:gray"> (324K)</span></div><div class="small guid">GUID:&nbsp;809A3127-A7B4-4202-BE76-AA839BDA4556</div></div></dd></dl></div><div id="idm139698342457744" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139698342457744title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">We hypothesize that imputation based on data from the 1000 Genomes Project can identify novel association signals on a genome-wide scale due to the dense marker map and the large number of haplotypes. To test the hypothesis, the Wellcome Trust Case Control Consortium (WTCCC) Phase I genotype data were imputed using 1000 genomes as reference (20100804 EUR), and seven case/control association studies were performed using imputed dosages. We observed two ‘missed' disease-associated variants that were undetectable by the original WTCCC analysis, but were reported by later studies after the 2007 WTCCC publication. One is within the <em>IL2RA</em> gene for association with type 1 diabetes and the other in proximity with the <em>CDKN2B</em> gene for association with type 2 diabetes. We also identified two refined associations. One is SNP rs11209026 in exon 9 of <em>IL23R</em> for association with Crohn's disease, which is predicted to be probably damaging by PolyPhen2. The other refined variant is in the <em>CUX2</em> gene region for association with type 1 diabetes, where the newly identified top SNP rs1265564 has an association <em>P</em>-value of 1.68 × 10<sup>−16</sup>. The new lead SNP for the two refined loci provides a more plausible explanation for the disease association. We demonstrated that 1000 Genomes-based imputation could indeed identify both novel (in our case, ‘missed' because they were detected and replicated by studies after 2007) and refined signals. We anticipate the findings derived from this study to provide timely information when individual groups and consortia are beginning to engage in 1000 genomes-based imputation.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genome-wide association study, the 1000 Genomes project, imputation</span></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">It has been four years since the publication of one of the first and largest genome-wise association studies (GWAS), the Wellcome Trust Case Control Consortium (WTCCC) study.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549069">1</a></sup> Although imputation-based analysis was already adopted at that time for refining association signals, its use was limited and few results were reported based on imputations. In one study, an imputed missense mutation in gene <em>GCKR</em> is found to be more strongly associated with triglyceride.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549064">2</a></sup></p><p id="__p3">Imputation significantly increases the statistical power of GWAS and allows meta-analysis of studies genotyped on different platforms.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549057">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549060">4</a>, <a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549044">5</a></sup> Until recently, most imputation work has been using HapMap haplotypes as a reference panel.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549055">6</a></sup> However, the recent publication of the 1000 Genomes Pilot Project and the availability of phased haplotypes from both the pilot and main project bring opportunities for much denser imputations and more extensive analysis.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549037">7</a></sup> The 1000 genomes-based imputation has already been shown to refine association signals and identify underlying genetic variants that are in high linkage disequilibrium (LD) with variants that were included in the genotyping platform.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549053">7</a></sup> However, there have been few reports of novel findings using this latest imputation approach on a genome-wide scale.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329077072">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_364982924">9</a></sup> The Oxford-GSK study<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329077071">8</a></sup> used 1000 genomes-based imputation to refine a single genetic region and successfully identified a SNP that is in the promoter region of a biologically plausible gene with a more significant <em>P</em>-value than that without 1000 genomes imputation. The Sardiana study<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_364982925">9</a></sup> fully utilized the reference panels from HapMap2, HapMap3, and the 1000 genomes, but did not specifically evaluate the power gains from each panel. It also has a smaller discovery sample size (<em>N</em> &lt;1700) compared with the WTCCC study. In contrast, our study is hypothesis driven, based on a flagship study with rich phenotypes and a large sample size. We reason it is very important to confirm that genotype imputation based on the latest 1000 genomes release could identify novel variants on a genome-wide scale, beside refining associations at a regional level, given the large amount of efforts needed for scientists around the globe poised to apply this emerging tool to their scientific investigation.</p><p id="__p4" class="p p-last">We hypothesize that 1000 genomes-based imputation can identify novel variants beyond what could be seen from purely genotyped data or HapMap imputed data, due to the much denser SNP coverage and a much broader representation of reference populations. We test this hypothesis by re-examining the WTCCC Phase I data after imputing the genotype data to the full set of SNPs present in 1000 genomes latest release (version 20100804). We re-run association analysis for the seven traits based on 1000 genomes imputed dosages and highlight novel and refined genetic associations that would have been discovered by the original study should the 1000 genomes reference panel be available back then.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Methods</h2><p id="__p5" class="p p-first">We obtained approval for using the raw genotype data for the original WTCCC data set, and we created one harmonized genotype data set by applying quality controls similar as the original study to the downloaded files (details provided in <a href="#sup1" rid="sup1" class=" supplementary-material">Online Supplementary Section S1</a>). Using this genotype data with embedded disease status, we ran the case/control association tests with PLINK and verified that the results are similar as those reported in the original study with negligible difference.<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549036">10</a></sup> To match the genomic position used by the 1000 genomes reference, we mapped all SNPs to NCBI's build37 (hg19).</p><p id="__p6">We used the MaCH program to first phase the haplotypes and then ran MiniMac for genotype imputation.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549047">11</a></sup> We used the recommended two-step approach and recommended parameters of 20 iterations of the Markov sampler and 200 states. The 1000 genomes reference panel was obtained from the University of Michigan Abecasis lab, version 20100804 (<a href="http://www.sph.umich.edu/csg/abecasis/MACH/download/1000G-2010-08.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MACH/download/1000G-2010-08.html</a>). A total of 566 reference haplotypes of the European ancestry served as the reference panel. We ran a logistic regression analysis based on imputation dosages via MACH2DAT (thus taking imputation uncertainty into account) for each of the seven traits with the shared control without covariate adjustment, a similar statistical analysis approach presented by the original WTCCC study. We included all SNPs with estimated <em>R</em><sup>2</sup> &gt;0.3 and minor allele frequency &gt;0.01 for analysis.</p><p id="__p7">SNPs were considered in the same region if they are within the same gene or are &lt;1 Mb apart. We define ‘novel' for any SNP with association <em>P</em>-value reaching the genome-wide significance level (5 × 10<sup>−8</sup>) in a region not reported in the original study that analyzed genotyped data. Novel variants that were later reported by other studies after the original 2007 WTCCC paper are designated as ‘missed' instead of ‘novel'. We define ‘refined' for any association where there is a reported association in the same region in the original study but the new lead SNP has a <em>P</em>-value more significant even after correcting the number of new SNPs tested. For ‘refined' association, we further require that the lead SNP has either a functional support or is pinpointing to biologically more relevant gene. We used PolyPhen-2 to predict the possible impact of amino-acid substitutions <em>in silico</em>.<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549049">12</a></sup> To evaluate whether the genome-wide significant threshold of 5 × 10<sup>−8</sup> widely used in HapMap2 imputed analysis would be sufficiently conservative in 1000 genomes imputed analysis, we picked tagging SNPs using a greedy algorithm similar to that in <em>ldselect</em><sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549062">13</a></sup> at <em>R</em><sup>2</sup>=0.9 for the SNPs included in our analysis.</p><p id="__p8" class="p p-last">For all genetic loci identified as novel or refined, a 5-Mb region around the lead SNP was re-imputed and then analyzed with an independent imputation program (BEAGLE)<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549056">14</a></sup> and association analysis tool (PLINK)<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549035">10</a></sup> by an independent analyst (DE; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Section S2</a>).</p></div><div id="__sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec3title">Results</h2><p id="__p9" class="p p-first">After SNP quality control and mapping of genomic positions to build37, a total of 389 827 SNPs for 16 179 samples were retained as input genotype data for imputation. After imputation, a total of 6 233 112 SNPs with estimated <em>R</em><sup>2</sup> &gt;0.3 and minor allele frequency &gt;0.01 were used for association analysis. The estimated genomic inflation factor <em>λ</em><sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549061">15</a></sup> for the seven case/control GWAS ranges from 1.04 to 1.09, which is comparable to the original study and indicates low genomic inflation. Association Manhattan plots for all seven analyses are shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1</a>. We compared the signals with those in the original WTCCC study, and highlighted two missed and two refined variants that we identified through this latest imputation method (<a href="/pmc/articles/PMC3376268/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). All four loci were confirmed by an independent analysis using BEAGLE and PLINK (JH, DE).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Novel/missed and refined variants from 1000 genomes imputation-based analysis</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139698343771424" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Type</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Trait</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Lead SNP in present study</em> (<em>association P</em><sup><a href="#t1-fn2" rid="t1-fn2" class=" fn">a</a></sup>)</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Chromosome band</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Gene</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Estimated R</em><sup><em>2</em></sup> <sup><a href="#t1-fn3" rid="t1-fn3" class=" fn">b</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Alleles</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Odds ratio (95</em>% <em>CI)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Lead SNP in original Study (association P, LD</em><sup><a href="#t1-fn4" rid="t1-fn4" class=" fn">c</a></sup>, <em>physical distance, conditional P</em><sup><a href="#t1-fn5" rid="t1-fn5" class=" fn">d</a></sup>)</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Nearest SNP in literature (association P, LD</em><sup><a href="#t1-fn4" rid="t1-fn4" class=" fn">c</a></sup>, <em>physical distance, conditional P</em><sup><a href="#t1-fn5" rid="t1-fn5" class=" fn">d</a></sup>)</th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Missed</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T1D</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs61839660</strong><em>
P</em>=5.1 × 10<sup>−9</sup> (1.8 × 10<sup>−8</sup>)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10p15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>IL2RA</em> intron 7</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.09</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.87 (0.90)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C-&gt;T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.60 (1.44–1.76)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">NA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs12251307</strong><sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup> (<em>P</em>=2.9 × 10<sup>−5</sup>; 
LD=0.74 (0.38) 28.8 kb <em>P</em>con=3.7 × 10<sup>−5</sup>)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Missed</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T2D</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs7018475</strong>
<em>P</em>=2.5 × 10<sup>−8</sup> (6.1 × 10<sup>−8</sup>)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9p21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>CDKN2B</em> -128 kb</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.24</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.79 (0.85)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T-&gt;G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.74 (0.64–0.85)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">NA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs10757282</strong><sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup> (<em>P</em>=2.7 × 10<sup>−3</sup>; 
LD=0.97 (0.41) 3.7 kb <em>P</em>con=3.7 × 10<sup>−7</sup>) <strong>
rs10811661</strong><sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a>, <a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup> (<em>P</em>=7.1 × 10<sup>−4</sup>; 
LD=0.50 (0.02) 3.6 kb 
<em>P</em>con=1.5 × 10<sup>−7</sup>) <em>P</em>con=0.5<sup><a href="#t1-fn6" rid="t1-fn6" class=" fn">e</a></sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Refined</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">CD</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs11209026</strong>
<em>P</em>=4.2 × 10<sup>−21</sup> (1.1 × 10<sup>−17</sup>)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1p31</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>IL23R</em> exon 9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.06</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.90 (0.91)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G-&gt;A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.18 (2.91–3.44)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs11805303</strong> (<em>P</em>=5.8 × 10<sup>−12</sup>; LD=0.57 (0.01) 30.4 kb
<em>P</em>con=7.6 × 10<sup>−17</sup>)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs11465804</strong><sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> (<em>P</em>=7.7 × 10<sup>−20</sup>; LD=1.0 (0.88) 3.4 kb
<em>P</em>con=7.9 × 10<sup>−3</sup>)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Refined</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">T1D</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs1265564</strong>
<em>P</em>=1.0 × 10<sup>−16</sup> (1.1 × 10<sup>−16</sup>)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12q24</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>CUX2</em> intron 4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.48</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.87 (0.92)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A-&gt;C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.69 (0.60–0.78)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs17696736</strong> (<em>P</em>=1.5 × 10<sup>−14</sup>; LD=0.43 (0.18) 
778.4 kb <em>P</em>con=5.3 × 10<sup>−6</sup>)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><strong>rs3184504</strong><sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup> (<em>P</em>=1.4 × 10<sup>−14</sup>; LD=0.74 (0.47) 176.2 kb
<em>P</em>con=1.1 × 10<sup>−4</sup>)</td></tr></tbody></table></div><div id="largeobj_idm139698343771424" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3376268/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1-fn1"><p id="__p23" class="p p-first-last">Abbreviations: CD: Crohns's disease; T1D: type 1 diabetes; T2D: type 2 diabetes.</p></div><div id="t1-fn2"><sup>a</sup>The first value is based on MACH/MACH2DAT and the value in brackets is based on BEAGLE/PLINK.</div><div id="t1-fn3"><sup>b</sup>Imputation quality (estimated <em>R</em><sup>2</sup>) based on MaCH and Beagle (values in brackets).</div><div id="t1-fn4"><sup>c</sup>LD between lead SNP reported in this study and SNP in this column based on the same version of the 1000 genomes data, measured by D-prime (<em>R</em><sup>2</sup>).</div><div id="t1-fn5"><sup>d</sup>Conditional <em>P</em>-value of the lead SNP reported in this study conditioned on SNP in this column.</div><div id="t1-fn6"><sup>e</sup>Conditional <em>P</em>-value of the lead SNP on the two-SNP (rs10757282 and rs10811661) haplotype.</div></div></div><p id="__p10">As shown in the regional plots, the two missed variants would not have been identified with HapMap2-based<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549039">16</a></sup> imputation (shown as red color in <a href="/pmc/articles/PMC3376268/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>). For the refined <em>CUX2</em> region, the best HapMap2 imputed SNP is less significant than the genotyped SNP originally reported. For the refined <em>IL23R</em> region, the best HapMap2 SNP is not exonic and not predicted to have functional consequence.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3376268/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139698343774048" class="figure"><a href="/pmc/articles/PMC3376268/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3376268_ejhg20123f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg20123f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3376268/bin/ejhg20123f1.jpg"></a></div><div id="largeobj_idm139698343774048" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3376268/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3376268/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p22">Regional plots for two novel (missed) and two refined loci. The top two plots are for two novel (‘missed') regions, where highly significant SNPs meet genome-wide significance (<em>P</em>&lt;2.5 × 10<sup>−8</sup>). The bottom two plots are for two refined regions. SNPs are represented by three different colors: black for WTCCC genotyped SNPs, red for HapMap2 imputed SNPs, and green for 1000 genomes imputed SNPs. Chromosome base pair positions (NCBI build 37) are represented on the <em>X</em>-axis. On the <em>Y</em>-axis, statistical significance is expressed as –log<sub>10</sub> of the <em>P</em>-values. The horizontal line marks the <em>P</em>=2.5 × 10<sup>−8</sup> threshold of genome-wide significance.</p></div></div></div><p id="__p11" class="p p-last">A total of 1 915 543 tagging SNPs were picked for the total of 6 233 112 SNPs included in our analysis, at the <em>R</em><sup>2</sup> threshold of 0.9. Therefore, compared with the previous assumption of one million independent loci across the genome,<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549050">17</a></sup>and a genome-wide significance threshold of 5.0 × 10<sup>−8</sup>, we propose a genome-wide threshold of 2.5 × 10<sup>−8</sup> assuming two million independent SNPs. All four SNPs in <a href="/pmc/articles/PMC3376268/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> meet genome-wide significance applying this conservative (because of correlation among the tagging SNPs) threshold.</p></div><div id="__sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title">Discussion</h2><p id="__p12" class="p p-first">This is the first study to comprehensively assess the utility of 1000 genomes-based imputation for identifying novel genetic association signals. We identified two associations that were not reported in the original WTCCC study but later established through other GWAS. One is within gene <em>IL2RA</em> for association with type 1 diabetes (T1D) and the other is 128 kb downstream of gene <em>CDKN2B</em> for association with type 2 diabetes (T2D). No association with phenotypic trait(s) was reported in the original study for these two loci. But as the two lead SNPs are no longer genome-wide significant once conditioning on the SNPs established by independent studies after the publication of the original WTCCC study<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549058">18</a>, <a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549066">19</a>, <a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_433502120">20</a></sup> we view them as ‘missed' rather than ‘novel' loci.</p><p id="__p13">Furthermore, we identified two refined SNP associations: one is SNP rs11209026 in the exon of <em>IL23R</em> gene for association with Crohn's disease. It is predicted to be probably damaging by PolyPhen-2 and has a <em>P</em>-value of 1.41 × 10<sup>−21</sup> compared with the previous best association <em>P</em>-value of 5.85 × 10<sup>−12</sup> (SNP rs11805303 within intron 6 of <em>IL23R</em>, see <a href="/pmc/articles/PMC3376268/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>) from the original WTCCC study. Although the lead SNP rs11209026 is still genome-wide significant after conditioning on the lead genotyped SNP, we consider it a refined signal for two reasons. First two SNPs reside within the same gene and in physical vicinity (30.4 kb apart). Second, the <em>P</em>-value of rs11209026 dropped by more than three orders of magnitude (from 4.2 × 10<sup>−21</sup> to 7.6 × 10<sup>−17</sup>), suggesting that the two signals are partially dependent or tagging the same underlying/untyped causal SNP(s) or haplotype(s). The other refined association is located in the <em>CUX2</em> region for association with T1D, where the newly identified top SNP rs1265564 has an association <em>P</em>-value of 1.68 × 10<sup>−16</sup>. The original WTCCC study reported a best association <em>P</em>-value of 1.51 × 10<sup>−14</sup> (rs17696736) within the gene <em>NAA25</em> (<a href="/pmc/articles/PMC3376268/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). These two SNPs are 780 kb apart, however, the <em>CUX2</em> gene has been shown to directly regulate the expression of NeuroD, a gene that can cause T1D when mutated.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549052">21</a></sup> The lead SNPs for these two refined loci (<em>IL23R</em> for CD and <em>CUX2</em> for T1D) are no longer genome-wide significant after conditioning on the nearest lead SNPs reported in literature to date.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549038">18</a>, <a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549041">22</a></sup></p><p id="__p14">We tend to believe that the two missed loci, namely the <em>IL2RA</em> locus for T1D and <em>CDKN2B</em> locus for T2D, are the same as reported in later studies for two reasons. First, the two missed loci fall in the same region with the SNPs reported in post-2007 literature with physical distance ranging from 3.6 to 28.8 kb. Although the level of LD is largely moderate, given such close physical vicinity, it is hard to rule out the possibility that both our lead SNPs and the SNPs reported in literature are tagging the same (untested) SNP(s) or haplotype(s). Our second reason is that the <em>P</em>-values of our lead SNPs drop by an order of three or four, and all &gt;2.9 × 10<sup>−5</sup> once conditional on the SNPs reported in the post-2007 literature. For the <em>IL2RA</em> locus, more detailed haplotype and fine-mapping analyses would be required to fully appreciate the complex architecture of causal variants in this region.</p><p id="__p15">The independent imputation and association analysis using BEAGLE and PLINK identified the exact same four SNPs showing best association signals in the four regions identified by MaCH and MACH2DAT, confirming our findings that two are novel and two are refined, except that the BEALGE/PLINK generated a <em>P</em>-value for the <em>CDKN2B</em> locus slightly above the 5 × 10<sup>−8</sup> threshold. Both MaCH and BEAGLE have been recommended for practical use because of their user-friendly interface and computational efficiency.<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549043">23</a>, <a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549040">24</a>, <a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549059">25</a>, <a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549070">26</a></sup></p><p id="__p16">We adopted a conservative genome-wide significance threshold 2.5 × 10<sup>−8</sup> to guard against false positives particularly given that we are testing seven phenotypic traits instead of a single one. The fact that we only have genome-wide significant signals from four well-established regions suggests that our conservative threshold fulfilled its purpose. Future studies may gain additional power with more sophisticated methods to control type-I error<sup><a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549071">27</a>, <a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549048">28</a>, <a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549046">29</a>, <a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549042">30</a></sup> or with methods that handle multiple related phenotypes.<sup><a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549073">31</a>, <a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549068">32</a></sup> On the other hand, for four out of the seven traits, our 1000 genomes-based imputation detected nothing novel on top of the original WTCCC study, suggesting that the potential power of imputation is limited by the genetic architecture of the trait(s) of interest and the genomic coverage of the GWAS genotyping panel used.</p><p id="__p17" class="p p-last">We present here an example where 1000 genomes-based imputation identifies both novel and refined signals. By using 1000 genomes-based imputation, we identified two SNPs that are genome-wide significantly associated with two of the seven traits in the WTCCC study, neither discovered in the original study with only genotyped SNPs. The two SNPs ‘missed' from the original analysis serve as positive controls because the two residing regions were both established by other studies after the 2007 WTCCC publication. Our analysis also provided further insights into two regions identified in the original study by identifying SNPs that are either more significant or point to a biologically more plausible gene. Importantly, we had no other signals based on our conservative genome-wide significance threshold, suggesting that we have no inflated false discovery rates. Taken together, our findings suggest that applying 1000 genomes-based imputation to the large number of GWAS data sets existing nowadays has the potential both to identify novel disease-associated genetic variants and to advance our understanding in known regions by examining a much denser set of imputed variants. We believe that larger reference panels continuing to be released by the 1000 Genomes Project will benefit the community even more, by performing single-marker analysis as presented here or rare or structural variant analysis.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549063">11</a>, <a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_263549072">33</a></sup></p></div><div id="idm139698342647584" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139698342647584title">Acknowledgments</h2><div class="sec"><p id="__p18">We thank Prof David P Strachan at the St George's University of London for commenting on an earlier version of this manuscript. We acknowledge the WTCC for making the data available. A portion of this research was conducted using the Linux Cluster for Genetic Analysis (LinGA-II) funded by the Robert Dawson Evans Endowment of the Department of Medicine at Boston University School of Medicine and Boston Medical Center. The effort of DE and AF was supported by the Deutsche Forschungsgemeinschaft (DFG), grant no. FR 2821/2-1, and the German Ministry of Education and Research (BMBF) through the National Genome Research Network (NGFN). This project received infrastructure support through the DFG cluster of excellence ‘Inflammation at Interfaces'. YL is partially supported by the NIH grant R01-HG006292 and 3-R01-CA082659-11S1.</p></div></div><div id="idm139698342644464" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139698342644464title">Notes</h2><p id="__p20">The authors declare no conflict of interest.</p></div><div id="idm139698342646112" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139698342646112title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm139698342645984"></p><p id="__p19" class="p p-first-last"><a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> accompanies the paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="idm139698343782032"><a href="/pmc/articles/PMC3376268/bin/ejhg20123x1.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(324K, doc)</sup></div></div></div><div id="idm139698342643792" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139698342643792title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">WTCCC  Genome-wide association study of 14 000 cases of seven common diseases and 3000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14 000+cases+of+seven+common+diseases+and+3000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Orho-Melander M, Melander O, Guiducci C, et al.  Common missense variant in the glucokinase regulatory protein gene is associated with increased plasma triglyceride and C-reactive protein but lower fasting glucose concentrations. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:3112–3121.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2570409/">PMC free article</a>]</span> [<a href="/pubmed/18678614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Common+missense+variant+in+the+glucokinase+regulatory+protein+gene+is+associated+with+increased+plasma+triglyceride+and+C-reactive+protein+but+lower+fasting+glucose+concentrations&amp;author=M+Orho-Melander&amp;author=O+Melander&amp;author=C+Guiducci&amp;volume=57&amp;publication_year=2008&amp;pages=3112-3121&amp;pmid=18678614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">de Bakker PI, Ferreira MA, Jia X, Neale BM, Raychaudhuri S, Voight BF. Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:R122–R128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=PI+de+Bakker&amp;author=MA+Ferreira&amp;author=X+Jia&amp;author=BM+Neale&amp;author=S+Raychaudhuri&amp;volume=17&amp;publication_year=2008&amp;pages=R122-R128&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Marchini J, Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>:499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Thorisson GA, Smith AV, Krishnan L, Stein LD. The International HapMap Project Web site. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:1592–1593.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1310647/">PMC free article</a>]</span> [<a href="/pubmed/16251469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=The+International+HapMap+Project+Web+site&amp;author=GA+Thorisson&amp;author=AV+Smith&amp;author=L+Krishnan&amp;author=LD+Stein&amp;volume=15&amp;publication_year=2005&amp;pages=1592-1593&amp;pmid=16251469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Durbin RM, Abecasis GR, Altshuler DL, et al.  A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;author=RM+Durbin&amp;author=GR+Abecasis&amp;author=DL+Altshuler&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Liu JZ, Tozzi F, Waterworth DM, et al.  Meta-analysis and imputation refines the association of 15q25 with smoking quantity. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:436–440.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3612983/">PMC free article</a>]</span> [<a href="/pubmed/20418889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+and+imputation+refines+the+association+of+15q25+with+smoking+quantity&amp;author=JZ+Liu&amp;author=F+Tozzi&amp;author=DM+Waterworth&amp;volume=42&amp;publication_year=2010&amp;pages=436-440&amp;pmid=20418889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">Sanna S, Pitzalis M, Zoledziewska M, et al.  Variants within the immunoregulatory CBLB gene are associated with multiple sclerosis. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:495–497.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3786343/">PMC free article</a>]</span> [<a href="/pubmed/20453840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variants+within+the+immunoregulatory+CBLB+gene+are+associated+with+multiple+sclerosis&amp;author=S+Sanna&amp;author=M+Pitzalis&amp;author=M+Zoledziewska&amp;volume=42&amp;publication_year=2010&amp;pages=495-497&amp;pmid=20453840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;author=P+Scheet&amp;author=GR+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Adzhubei IA, Schmidt S, Peshkin L, et al.  A method and server for predicting damaging missense mutations. <span><span class="ref-journal">Nat Methods. </span>2010;<span class="ref-vol">7</span>:248–249.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855889/">PMC free article</a>]</span> [<a href="/pubmed/20354512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+method+and+server+for+predicting+damaging+missense+mutations&amp;author=IA+Adzhubei&amp;author=S+Schmidt&amp;author=L+Peshkin&amp;volume=7&amp;publication_year=2010&amp;pages=248-249&amp;pmid=20354512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Carlson CS, Eberle MA, Rieder MJ, Yi Q, Kruglyak L, Nickerson DA. Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">74</span>:106–120.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181897/">PMC free article</a>]</span> [<a href="/pubmed/14681826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Selecting+a+maximally+informative+set+of+single-nucleotide+polymorphisms+for+association+analyses+using+linkage+disequilibrium&amp;author=CS+Carlson&amp;author=MA+Eberle&amp;author=MJ+Rieder&amp;author=Q+Yi&amp;author=L+Kruglyak&amp;volume=74&amp;publication_year=2004&amp;pages=106-120&amp;pmid=14681826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Browning BL, Browning SR. A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:210–223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668004/">PMC free article</a>]</span> [<a href="/pubmed/19200528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+unified+approach+to+genotype+imputation+and+haplotype-phase+inference+for+large+data+sets+of+trios+and+unrelated+individuals&amp;author=BL+Browning&amp;author=SR+Browning&amp;volume=84&amp;publication_year=2009&amp;pages=210-223&amp;pmid=19200528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib16"><span class="mixed-citation">The International HapMap Consortium  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Pe'er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe'er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">Barrett JC, Clayton DG, Concannon P, et al.  Genome-wide association study and meta-analysis find that over 40 loci affect risk of type 1 diabetes. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:703–707.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2889014/">PMC free article</a>]</span> [<a href="/pubmed/19430480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+and+meta-analysis+find+that+over+40+loci+affect+risk+of+type+1+diabetes&amp;author=JC+Barrett&amp;author=DG+Clayton&amp;author=P+Concannon&amp;volume=41&amp;publication_year=2009&amp;pages=703-707&amp;pmid=19430480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Saxena R, Voight BF, Lyssenko V, et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R+Saxena&amp;author=BF+Voight&amp;author=V+Lyssenko&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Shea J, Agarwala V, Philippakis AA, et al.  Comparing strategies to fine-map the association of common SNPs at chromosome 9p21 with type 2 diabetes and myocardial infarction. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:801–805.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4096898/">PMC free article</a>]</span> [<a href="/pubmed/21775993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Comparing+strategies+to+fine-map+the+association+of+common+SNPs+at+chromosome+9p21+with+type+2+diabetes+and+myocardial+infarction&amp;author=J+Shea&amp;author=V+Agarwala&amp;author=AA+Philippakis&amp;volume=43&amp;publication_year=2011&amp;pages=801-805&amp;pmid=21775993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib21"><span class="mixed-citation">Iulianella A, Sharma M, Durnin M, Vanden Heuvel GB, Trainor PA. Cux2 (Cutl2) integrates neural progenitor development with cell-cycle progression during spinal cord neurogenesis. <span><span class="ref-journal">Development. </span>2008;<span class="ref-vol">135</span>:729–741.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3093760/">PMC free article</a>]</span> [<a href="/pubmed/18223201" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Cux2+(Cutl2)+integrates+neural+progenitor+development+with+cell-cycle+progression+during+spinal+cord+neurogenesis&amp;author=A+Iulianella&amp;author=M+Sharma&amp;author=M+Durnin&amp;author=GB+Vanden+Heuvel&amp;author=PA+Trainor&amp;volume=135&amp;publication_year=2008&amp;pages=729-741&amp;pmid=18223201&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib22"><span class="mixed-citation">Barrett JC, Hansoul S, Nicolae DL, et al.  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn′s disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:955–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn′s+disease&amp;author=JC+Barrett&amp;author=S+Hansoul&amp;author=DL+Nicolae&amp;volume=40&amp;publication_year=2008&amp;pages=955-962&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib23"><span class="mixed-citation">Nothnagel M, Ellinghaus D, Schreiber S, Krawczak M, Franke A. A comprehensive evaluation of SNP genotype imputation. <span><span class="ref-journal">Hum Genet. </span>2009;<span class="ref-vol">125</span>:163–171.</span> [<a href="/pubmed/19089453" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=A+comprehensive+evaluation+of+SNP+genotype+imputation&amp;author=M+Nothnagel&amp;author=D+Ellinghaus&amp;author=S+Schreiber&amp;author=M+Krawczak&amp;author=A+Franke&amp;volume=125&amp;publication_year=2009&amp;pages=163-171&amp;pmid=19089453&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib24"><span class="mixed-citation">Pei YF, Li J, Zhang L, Papasian CJ, Deng HW. Analyses and comparison of accuracy of different genotype imputation methods. <span><span class="ref-journal">PLoS One. </span>2008;<span class="ref-vol">3</span>:e3551.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2569208/">PMC free article</a>]</span> [<a href="/pubmed/18958166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Analyses+and+comparison+of+accuracy+of+different+genotype+imputation+methods&amp;author=YF+Pei&amp;author=J+Li&amp;author=L+Zhang&amp;author=CJ+Papasian&amp;author=HW+Deng&amp;volume=3&amp;publication_year=2008&amp;pages=e3551&amp;pmid=18958166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib25"><span class="mixed-citation">Zheng J, Li Y, Abecasis GR, Scheet P. A comparison of approaches to account for uncertainty in analysis of imputed genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:102–110.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3143715/">PMC free article</a>]</span> [<a href="/pubmed/21254217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=A+comparison+of+approaches+to+account+for+uncertainty+in+analysis+of+imputed+genotypes&amp;author=J+Zheng&amp;author=Y+Li&amp;author=GR+Abecasis&amp;author=P+Scheet&amp;volume=35&amp;publication_year=2011&amp;pages=102-110&amp;pmid=21254217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib26"><span class="mixed-citation">Pei YF, Zhang L, Li J, Deng HW. Analyses and comparison of imputation-based association methods. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>:e10827.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2877082/">PMC free article</a>]</span> [<a href="/pubmed/20520814" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Analyses+and+comparison+of+imputation-based+association+methods&amp;author=YF+Pei&amp;author=L+Zhang&amp;author=J+Li&amp;author=HW+Deng&amp;volume=5&amp;publication_year=2010&amp;pages=e10827&amp;pmid=20520814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib27"><span class="mixed-citation">Nyholt DR. A simple correction for multiple testing for single-nucleotide polymorphisms in linkage disequilibrium with each other. <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">74</span>:765–769.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181954/">PMC free article</a>]</span> [<a href="/pubmed/14997420" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+simple+correction+for+multiple+testing+for+single-nucleotide+polymorphisms+in+linkage+disequilibrium+with+each+other&amp;author=DR+Nyholt&amp;volume=74&amp;publication_year=2004&amp;pages=765-769&amp;pmid=14997420&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib28"><span class="mixed-citation">Conneely KN, Boehnke M. So many correlated tests, so little time! Rapid adjustment of p values for multiple correlated tests. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:1158–1168.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276357/">PMC free article</a>]</span> [<a href="/pubmed/17966093" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=So+many+correlated+tests,+so+little+time!+Rapid+adjustment+of+p+values+for+multiple+correlated+tests&amp;author=KN+Conneely&amp;author=M+Boehnke&amp;volume=81&amp;publication_year=2007&amp;pages=1158-1168&amp;pmid=17966093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib29"><span class="mixed-citation">Gao XY. Multiple testing corrections for imputed SNPs. <span><span class="ref-journal">Genet. Epidemiol. </span>2011;<span class="ref-vol">35</span>:154–158.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3055936/">PMC free article</a>]</span> [<a href="/pubmed/21254223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Multiple+testing+corrections+for+imputed+SNPs&amp;author=XY+Gao&amp;volume=35&amp;publication_year=2011&amp;pages=154-158&amp;pmid=21254223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib30"><span class="mixed-citation">Wen SH, Lu ZS. Factors affecting the effective number of tests in genetic association studies: a comparative study of three PCA-based methods. <span><span class="ref-journal">J Hum Genet. </span>2011;<span class="ref-vol">56</span>:428–435.</span> [<a href="/pubmed/21451529" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hum+Genet&amp;title=Factors+affecting+the+effective+number+of+tests+in+genetic+association+studies:+a+comparative+study+of+three+PCA-based+methods&amp;author=SH+Wen&amp;author=ZS+Lu&amp;volume=56&amp;publication_year=2011&amp;pages=428-435&amp;pmid=21451529&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib31"><span class="mixed-citation">Kullo IJ, de Andrade M, Boerwinkle E, McConnell JP, Kardia SL, Turner ST. Pleiotropic genetic effects contribute to the correlation between HDL cholesterol, triglycerides, and LDL particle size in hypertensive sibships. <span><span class="ref-journal">Am J Hypertens. </span>2005;<span class="ref-vol">18</span>:99–103.</span> [<a href="/pubmed/15691623" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hypertens&amp;title=Pleiotropic+genetic+effects+contribute+to+the+correlation+between+HDL+cholesterol,+triglycerides,+and+LDL+particle+size+in+hypertensive+sibships&amp;author=IJ+Kullo&amp;author=M+de+Andrade&amp;author=E+Boerwinkle&amp;author=JP+McConnell&amp;author=SL+Kardia&amp;volume=18&amp;publication_year=2005&amp;pages=99-103&amp;pmid=15691623&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib32"><span class="mixed-citation">Avery CL, He Q, North KE, et al.  A phenomics-based strategy identifies loci on APOC1, BRAP, and PLCG1 associated with metabolic syndrome phenotype domains. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002322.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3192835/">PMC free article</a>]</span> [<a href="/pubmed/22022282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+phenomics-based+strategy+identifies+loci+on+APOC1,+BRAP,+and+PLCG1+associated+with+metabolic+syndrome+phenotype+domains&amp;author=CL+Avery&amp;author=Q+He&amp;author=KE+North&amp;volume=7&amp;publication_year=2011&amp;pages=e1002322&amp;pmid=22022282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib33"><span class="mixed-citation">Zawistowski M, Gopalakrishnan S, Ding J, Li Y, Grimm S, Zollner S. Extending rare-variant testing strategies: analysis of noncoding sequence and imputed genotypes. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:604–617.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2978957/">PMC free article</a>]</span> [<a href="/pubmed/21070896" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Extending+rare-variant+testing+strategies:+analysis+of+noncoding+sequence+and+imputed+genotypes&amp;author=M+Zawistowski&amp;author=S+Gopalakrishnan&amp;author=J+Ding&amp;author=Y+Li&amp;author=S+Grimm&amp;volume=87&amp;publication_year=2010&amp;pages=604-617&amp;pmid=21070896&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3376268&amp;issue-id=211083&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139698342457744title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Methods</a></li><li><a href="#__sec3title">Results</a></li><li><a href="#__sec4title">Discussion</a></li><li><a href="#idm139698342647584title">Acknowledgments</a></li><li><a href="#idm139698342644464title">Notes</a></li><li><a href="#idm139698342646112title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm139698342643792title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3376268/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3376268/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3376268/pdf/ejhg20123a.pdf">PDF (326K)</a></li> | <li><a href="#" data-citationid="PMC3376268" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3376268%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3376268%2F&amp;text=1000%20Genomes-based%20imputation%20identifies%20novel%20and%20refined%20associations%20for%20the%20Wellcome%20Trust%20Case%20Control%20Consortium%20phase%201%20Data" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3376268%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22293688" data-aiid="3376268" data-aid="3376268" data-iid="211083" data-domainid="1300" data-domain="ejhg" data-accid="PMC3376268" data-md5="f59a0c657ad02e1e3b22234080421da4"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:28:17-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C54F5DD53765100000000051F0388&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3376268%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3376268/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>