<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3756935/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Association Study Identifies Variants Associated with Progression of Liver Fibrosis from HCV Infection</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3756935/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Gastroenterology"><meta name="citation_title" content="Genome-Wide Association Study Identifies Variants Associated with Progression of Liver Fibrosis from HCV Infection"><meta name="citation_authors" content="ETIENNE PATIN, ZOLTÁN KUTALIK, JULIEN GUERGNON, STÉPHANIE BIBERT, BERTRAND NALPAS, EMMANUELLE JOUANGUY, MONA MUNTEANU, LAURENCE BOUSQUET, LAURENT ARGIRO, PHILIPPE HALFON, ANNE BOLAND, BEAT MÜLLHAUPT, DAVID SEMELA, JEAN-FRANÇOIS DUFOUR, MARKUS H. HEIM, DARIUS MORADPOUR, ANDREAS CERNY, RAFFAELE MALINVERNI, HANS HIRSCH, GLADYS MARTINETTI, VIJAYAPRAKASH SUPPIAH, GRAEME STEWART, DAVID R. BOOTH, JACOB GEORGE, JEAN-LAURENT CASANOVA, CHRISTIAN BRÉCHOT, CHARLES M. RICE, ANDREW H. TALAL, IRA M. JACOBSON, MARC BOURLIÈRE, IOANNIS THEODOROU, THIERRY POYNARD, FRANCESCO NEGRO, STANISLAS POL, PIERRE-YVES BOCHUD, LAURENT ABEL"><meta name="citation_date" content="November 2012"><meta name="citation_issue" content="5"><meta name="citation_volume" content="143"><meta name="citation_firstpage" content="1244"><meta name="citation_doi" content="10.1053/j.gastro.2012.07.097"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3756935/?report=abstract"><meta name="citation_pmid" content="22841784"><meta name="DC.Title" content="Genome-Wide Association Study Identifies Variants Associated with Progression of Liver Fibrosis from HCV Infection"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="ETIENNE PATIN"><meta name="DC.Contributor" content="ZOLTÁN KUTALIK"><meta name="DC.Contributor" content="JULIEN GUERGNON"><meta name="DC.Contributor" content="STÉPHANIE BIBERT"><meta name="DC.Contributor" content="BERTRAND NALPAS"><meta name="DC.Contributor" content="EMMANUELLE JOUANGUY"><meta name="DC.Contributor" content="MONA MUNTEANU"><meta name="DC.Contributor" content="LAURENCE BOUSQUET"><meta name="DC.Contributor" content="LAURENT ARGIRO"><meta name="DC.Contributor" content="PHILIPPE HALFON"><meta name="DC.Contributor" content="ANNE BOLAND"><meta name="DC.Contributor" content="BEAT MÜLLHAUPT"><meta name="DC.Contributor" content="DAVID SEMELA"><meta name="DC.Contributor" content="JEAN-FRANÇOIS DUFOUR"><meta name="DC.Contributor" content="MARKUS H. HEIM"><meta name="DC.Contributor" content="DARIUS MORADPOUR"><meta name="DC.Contributor" content="ANDREAS CERNY"><meta name="DC.Contributor" content="RAFFAELE MALINVERNI"><meta name="DC.Contributor" content="HANS HIRSCH"><meta name="DC.Contributor" content="GLADYS MARTINETTI"><meta name="DC.Contributor" content="VIJAYAPRAKASH SUPPIAH"><meta name="DC.Contributor" content="GRAEME STEWART"><meta name="DC.Contributor" content="DAVID R. BOOTH"><meta name="DC.Contributor" content="JACOB GEORGE"><meta name="DC.Contributor" content="JEAN-LAURENT CASANOVA"><meta name="DC.Contributor" content="CHRISTIAN BRÉCHOT"><meta name="DC.Contributor" content="CHARLES M. RICE"><meta name="DC.Contributor" content="ANDREW H. TALAL"><meta name="DC.Contributor" content="IRA M. JACOBSON"><meta name="DC.Contributor" content="MARC BOURLIÈRE"><meta name="DC.Contributor" content="IOANNIS THEODOROU"><meta name="DC.Contributor" content="THIERRY POYNARD"><meta name="DC.Contributor" content="FRANCESCO NEGRO"><meta name="DC.Contributor" content="STANISLAS POL"><meta name="DC.Contributor" content="PIERRE-YVES BOCHUD"><meta name="DC.Contributor" content="LAURENT ABEL"><meta name="DC.Date" content="2012 Nov"><meta name="DC.Identifier" content="10.1053/j.gastro.2012.07.097"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Association Study Identifies Variants Associated with Progression of Liver Fibrosis from HCV Infection"><meta property="og:type" content="article"><meta property="og:description" content="Polymorphisms in IL28B were shown to affect clearance of hepatitis C virus (HCV) infection in genome-wide association (GWA) studies. Only a fraction of patients with chronic HCV infection develop liver fibrosis, a process that might also be affected by ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3756935/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3756935/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3756935/pdf/nihms477303.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BC563DD5421A100000000072404CA.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3756935%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="mesh" class="last">MeSH</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3756935</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Gastroenterology</a></span>. Author manuscript; available in PMC 2013 Nov 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22841784" target="pmc_ext" ref="reftype=publisher&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Gastroenterology. 2012 Nov; 143(5): 1244–52.e1-12. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Jul 27. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1053%2Fj.gastro.2012.07.097" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1053/j.gastro.2012.07.097</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3756935</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS477303</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22841784">22841784</a></div></div></div></div><h1 class="content-title">Genome-Wide Association Study Identifies Variants Associated with Progression of Liver Fibrosis from HCV Infection</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=PATIN%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449170048" co-class="co-affbox">ETIENNE PATIN</a>,<sup>*</sup><sup>‡</sup><sup>§</sup> <a href="/pubmed/?term=KUTALIK%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489838160" co-class="co-affbox">ZOLTÁN KUTALIK</a>,<sup>||</sup><sup>¶</sup> <a href="/pubmed/?term=GUERGNON%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489835440" co-class="co-affbox">JULIEN GUERGNON</a>,<sup>#</sup> <a href="/pubmed/?term=BIBERT%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489833456" co-class="co-affbox">STÉPHANIE BIBERT</a>,<sup>**</sup> <a href="/pubmed/?term=NALPAS%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449166816" co-class="co-affbox">BERTRAND NALPAS</a>,<sup>‡</sup><sup>‡‡</sup> <a href="/pubmed/?term=JOUANGUY%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449164096" co-class="co-affbox">EMMANUELLE JOUANGUY</a>,<sup>*</sup><sup>‡</sup><sup>§§</sup> <a href="/pubmed/?term=MUNTEANU%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449160640" co-class="co-affbox">MONA MUNTEANU</a>,<sup>||||</sup> <a href="/pubmed/?term=BOUSQUET%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449084976" co-class="co-affbox">LAURENCE BOUSQUET</a>,<sup>‡</sup><sup>‡‡</sup> <a href="/pubmed/?term=ARGIRO%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449082256" co-class="co-affbox">LAURENT ARGIRO</a>,<sup>¶¶</sup> <a href="/pubmed/?term=HALFON%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449080272" co-class="co-affbox">PHILIPPE HALFON</a>,<sup>##</sup> <a href="/pubmed/?term=BOLAND%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449078288" co-class="co-affbox">ANNE BOLAND</a>,<sup>***</sup> <a href="/pubmed/?term=M%26%23x000dc%3BLLHAUPT%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449076304" co-class="co-affbox">BEAT MÜLLHAUPT</a>,<sup>‡‡‡</sup> <a href="/pubmed/?term=SEMELA%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449074320" co-class="co-affbox">DAVID SEMELA</a>,<sup>§§§</sup> <a href="/pubmed/?term=DUFOUR%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449072336" co-class="co-affbox">JEAN-FRANÇOIS DUFOUR</a>,<sup>||||||</sup> <a href="/pubmed/?term=HEIM%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449070352" co-class="co-affbox">MARKUS H. HEIM</a>,<sup>¶¶¶</sup> <a href="/pubmed/?term=MORADPOUR%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464449068368" co-class="co-affbox">DARIUS MORADPOUR</a>,<sup>###</sup> <a href="/pubmed/?term=CERNY%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489724448" co-class="co-affbox">ANDREAS CERNY</a>,<sup>****</sup> <a href="/pubmed/?term=MALINVERNI%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489722464" co-class="co-affbox">RAFFAELE MALINVERNI</a>,<sup>‡‡‡‡</sup> <a href="/pubmed/?term=HIRSCH%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489720480" co-class="co-affbox">HANS HIRSCH</a>,<sup>§§§§</sup> <a href="/pubmed/?term=MARTINETTI%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489718496" co-class="co-affbox">GLADYS MARTINETTI</a>,<sup>||||||||</sup> <a href="/pubmed/?term=SUPPIAH%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489716512" co-class="co-affbox">VIJAYAPRAKASH SUPPIAH</a>,<sup>¶¶¶¶</sup><sup>####</sup> <a href="/pubmed/?term=STEWART%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489713792" co-class="co-affbox">GRAEME STEWART</a>,<sup>####</sup> <a href="/pubmed/?term=BOOTH%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489711808" co-class="co-affbox">DAVID R. BOOTH</a>,<sup>####</sup> <a href="/pubmed/?term=GEORGE%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489709824" co-class="co-affbox">JACOB GEORGE</a>,<sup>¶¶¶¶</sup> <a href="/pubmed/?term=CASANOVA%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464489707840" co-class="co-affbox">JEAN-LAURENT CASANOVA</a>,<sup>*</sup><sup>‡</sup><sup>§§</sup> <a href="/pubmed/?term=BR%26%23x000c9%3BCHOT%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445939040" co-class="co-affbox">CHRISTIAN BRÉCHOT</a>,<sup>*****</sup> <a href="/pubmed/?term=RICE%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445937056" co-class="co-affbox">CHARLES M. RICE</a>,<sup>‡‡‡‡‡</sup> <a href="/pubmed/?term=TALAL%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445935072" co-class="co-affbox">ANDREW H. TALAL</a>,<sup>§§§§§</sup> <a href="/pubmed/?term=JACOBSON%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445933088" co-class="co-affbox">IRA M. JACOBSON</a>,<sup>§§§§§</sup> <a href="/pubmed/?term=BOURLI%26%23x000c8%3BRE%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445931104" co-class="co-affbox">MARC BOURLIÈRE</a>,<sup>||||||||||</sup> <a href="/pubmed/?term=THEODOROU%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445929120" co-class="co-affbox">IOANNIS THEODOROU</a>,<sup>#</sup> <a href="/pubmed/?term=POYNARD%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445927136" co-class="co-affbox">THIERRY POYNARD</a>,<sup>¶¶¶¶¶</sup> <a href="/pubmed/?term=NEGRO%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445925152" co-class="co-affbox">FRANCESCO NEGRO</a>,<sup>#####</sup> <a href="/pubmed/?term=POL%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464445923168" co-class="co-affbox">STANISLAS POL</a>,<sup>‡</sup><sup>‡‡</sup> <a href="/pubmed/?term=BOCHUD%20PY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464487171200" co-class="co-affbox">PIERRE-YVES BOCHUD</a>,<sup>**</sup> and  <a href="/pubmed/?term=ABEL%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784" class="affpopup" co-rid="_co_idm140464487169216" co-class="co-affbox">LAURENT ABEL</a><sup>*</sup><sup>‡</sup><sup>§§</sup>, on behalf of the Swiss Hepatitis C Cohort Study Group, the International Hepatitis C Genetics Consortium and the French ANRS HC EP 26 Genoscan Study Group</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140464449170048"><h3 class="no_margin">ETIENNE PATIN</h3><p><sup>*</sup>Laboratory of Human Genetics of Infectious Diseases, Necker Branch, Necker Medical School, Institut National de la Santé et de la Recherche Médicale (INSERM) U980, Paris, France</p><p><sup>‡</sup>University Paris Descartes, Paris, France</p><div>Find articles by <a href="/pubmed/?term=PATIN%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">ETIENNE PATIN</a></div></div><div id="_co_idm140464489838160"><h3 class="no_margin">ZOLTÁN KUTALIK</h3><p><sup>||</sup>Department of Medical Genetics, University of Lausanne, Lausanne, Switzerland</p><p><sup>¶</sup>Swiss Institute of Bioinformatics, Lausanne, Switzerland</p><div>Find articles by <a href="/pubmed/?term=KUTALIK%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">ZOLTÁN KUTALIK</a></div></div><div id="_co_idm140464489835440"><h3 class="no_margin">JULIEN GUERGNON</h3><p><sup>#</sup>Laboratory of Immunity and Infection, INSERM UMR-S 945, UPMC Université Paris 6, Groupe Hospitalier Pitié-Salpêtrière AP-HP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=GUERGNON%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">JULIEN GUERGNON</a></div></div><div id="_co_idm140464489833456"><h3 class="no_margin">STÉPHANIE BIBERT</h3><p><sup>**</sup>Service of Infectious Diseases, Department of Medicine, University Hospital and University of Lausanne, Lausanne, Switzerland</p><div>Find articles by <a href="/pubmed/?term=BIBERT%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">STÉPHANIE BIBERT</a></div></div><div id="_co_idm140464449166816"><h3 class="no_margin">BERTRAND NALPAS</h3><p><sup>‡</sup>University Paris Descartes, Paris, France</p><p><sup>‡‡</sup>Département d’Hépatologie, INSERM U1016, Groupe Hospitalier Cochin-Hôtel Dieu-Broca, Paris, France</p><div>Find articles by <a href="/pubmed/?term=NALPAS%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">BERTRAND NALPAS</a></div></div><div id="_co_idm140464449164096"><h3 class="no_margin">EMMANUELLE JOUANGUY</h3><p><sup>*</sup>Laboratory of Human Genetics of Infectious Diseases, Necker Branch, Necker Medical School, Institut National de la Santé et de la Recherche Médicale (INSERM) U980, Paris, France</p><p><sup>‡</sup>University Paris Descartes, Paris, France</p><p><sup>§§</sup>St; Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, The Rockefeller University, New York, NY, USA</p><div>Find articles by <a href="/pubmed/?term=JOUANGUY%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">EMMANUELLE JOUANGUY</a></div></div><div id="_co_idm140464449160640"><h3 class="no_margin">MONA MUNTEANU</h3><p><sup>||||</sup>Biopredictive, Paris, France</p><div>Find articles by <a href="/pubmed/?term=MUNTEANU%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">MONA MUNTEANU</a></div></div><div id="_co_idm140464449084976"><h3 class="no_margin">LAURENCE BOUSQUET</h3><p><sup>‡</sup>University Paris Descartes, Paris, France</p><p><sup>‡‡</sup>Département d’Hépatologie, INSERM U1016, Groupe Hospitalier Cochin-Hôtel Dieu-Broca, Paris, France</p><div>Find articles by <a href="/pubmed/?term=BOUSQUET%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">LAURENCE BOUSQUET</a></div></div><div id="_co_idm140464449082256"><h3 class="no_margin">LAURENT ARGIRO</h3><p><sup>¶¶</sup>Laboratoire d’Immunologie et de Génétique des Maladies Parasitaires, INSERM-UMR 906/Université de la Méditerranée, Marseilles, France</p><div>Find articles by <a href="/pubmed/?term=ARGIRO%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">LAURENT ARGIRO</a></div></div><div id="_co_idm140464449080272"><h3 class="no_margin">PHILIPPE HALFON</h3><p><sup>##</sup>Laboratoire Alphabio, Hôpital Ambroise Paré, Marseilles, France</p><div>Find articles by <a href="/pubmed/?term=HALFON%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">PHILIPPE HALFON</a></div></div><div id="_co_idm140464449078288"><h3 class="no_margin">ANNE BOLAND</h3><p><sup>***</sup>CEA, Institut de Génomique, Centre National de Génotypage, Evry, France</p><div>Find articles by <a href="/pubmed/?term=BOLAND%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">ANNE BOLAND</a></div></div><div id="_co_idm140464449076304"><h3 class="no_margin">BEAT MÜLLHAUPT</h3><p><sup>‡‡‡</sup>Division of Gastroenterology and Hepatology, University Hospital, Zurich, Switzerland</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000dc%3BLLHAUPT%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">BEAT MÜLLHAUPT</a></div></div><div id="_co_idm140464449074320"><h3 class="no_margin">DAVID SEMELA</h3><p><sup>§§§</sup>Division of Gastroenterology, Canton Hospital, St Gallen, Switzerland</p><div>Find articles by <a href="/pubmed/?term=SEMELA%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">DAVID SEMELA</a></div></div><div id="_co_idm140464449072336"><h3 class="no_margin">JEAN-FRANÇOIS DUFOUR</h3><p><sup>||||||</sup>Hepatology, Department of Clinical Research, University Clinic for visceral Surgery and Medicine, Bern, Switzerland</p><div>Find articles by <a href="/pubmed/?term=DUFOUR%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">JEAN-FRANÇOIS DUFOUR</a></div></div><div id="_co_idm140464449070352"><h3 class="no_margin">MARKUS H. HEIM</h3><p><sup>¶¶¶</sup>Division of Gastroenterology and Hepatology, University Hospital, Basel, Switzerland</p><div>Find articles by <a href="/pubmed/?term=HEIM%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">MARKUS H. HEIM</a></div></div><div id="_co_idm140464449068368"><h3 class="no_margin">DARIUS MORADPOUR</h3><p><sup>###</sup>Centre Hospitalier Universitaire Vaudois, University of Lausanne, Switzerland</p><div>Find articles by <a href="/pubmed/?term=MORADPOUR%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">DARIUS MORADPOUR</a></div></div><div id="_co_idm140464489724448"><h3 class="no_margin">ANDREAS CERNY</h3><p><sup>****</sup>Clinica Moncucco, Lugano, Switzerland</p><div>Find articles by <a href="/pubmed/?term=CERNY%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">ANDREAS CERNY</a></div></div><div id="_co_idm140464489722464"><h3 class="no_margin">RAFFAELE MALINVERNI</h3><p><sup>‡‡‡‡</sup>Pourtalès Hospital, Neuchâtel, Switzerland</p><div>Find articles by <a href="/pubmed/?term=MALINVERNI%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">RAFFAELE MALINVERNI</a></div></div><div id="_co_idm140464489720480"><h3 class="no_margin">HANS HIRSCH</h3><p><sup>§§§§</sup>Institute for Medical Microbiology, University Hospital, Basel, Switzerland</p><div>Find articles by <a href="/pubmed/?term=HIRSCH%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">HANS HIRSCH</a></div></div><div id="_co_idm140464489718496"><h3 class="no_margin">GLADYS MARTINETTI</h3><p><sup>||||||||</sup>Institute of Medical Microbiology, Bellinzona, Switzerland</p><div>Find articles by <a href="/pubmed/?term=MARTINETTI%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">GLADYS MARTINETTI</a></div></div><div id="_co_idm140464489716512"><h3 class="no_margin">VIJAYAPRAKASH SUPPIAH</h3><p><sup>¶¶¶¶</sup>Storr Liver Unit, Westmead Millennium Institute and Westmead Hospital, University of Sydney and University of Sydney Medical Foundation, Sydney, Australia</p><p><sup>####</sup>Institute for Immunology and Allergy Research, Westmead Millennium Institute, University of Sydney, Sydney, Australia</p><div>Find articles by <a href="/pubmed/?term=SUPPIAH%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">VIJAYAPRAKASH SUPPIAH</a></div></div><div id="_co_idm140464489713792"><h3 class="no_margin">GRAEME STEWART</h3><p><sup>####</sup>Institute for Immunology and Allergy Research, Westmead Millennium Institute, University of Sydney, Sydney, Australia</p><div>Find articles by <a href="/pubmed/?term=STEWART%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">GRAEME STEWART</a></div></div><div id="_co_idm140464489711808"><h3 class="no_margin">DAVID R. BOOTH</h3><p><sup>####</sup>Institute for Immunology and Allergy Research, Westmead Millennium Institute, University of Sydney, Sydney, Australia</p><div>Find articles by <a href="/pubmed/?term=BOOTH%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">DAVID R. BOOTH</a></div></div><div id="_co_idm140464489709824"><h3 class="no_margin">JACOB GEORGE</h3><p><sup>¶¶¶¶</sup>Storr Liver Unit, Westmead Millennium Institute and Westmead Hospital, University of Sydney and University of Sydney Medical Foundation, Sydney, Australia</p><div>Find articles by <a href="/pubmed/?term=GEORGE%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">JACOB GEORGE</a></div></div><div id="_co_idm140464489707840"><h3 class="no_margin">JEAN-LAURENT CASANOVA</h3><p><sup>*</sup>Laboratory of Human Genetics of Infectious Diseases, Necker Branch, Necker Medical School, Institut National de la Santé et de la Recherche Médicale (INSERM) U980, Paris, France</p><p><sup>‡</sup>University Paris Descartes, Paris, France</p><p><sup>§§</sup>St; Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, The Rockefeller University, New York, NY, USA</p><div>Find articles by <a href="/pubmed/?term=CASANOVA%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">JEAN-LAURENT CASANOVA</a></div></div><div id="_co_idm140464445939040"><h3 class="no_margin">CHRISTIAN BRÉCHOT</h3><p><sup>*****</sup>INSERM U785, Liver Hepatology Centre, Paul Brousse Hospital, University Paris-XI, Villejuif, France</p><div>Find articles by <a href="/pubmed/?term=BR%26%23x000c9%3BCHOT%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">CHRISTIAN BRÉCHOT</a></div></div><div id="_co_idm140464445937056"><h3 class="no_margin">CHARLES M. RICE</h3><p><sup>‡‡‡‡‡</sup>Center for the Study of Hepatitis C, Rockefeller University, New York, USA</p><div>Find articles by <a href="/pubmed/?term=RICE%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">CHARLES M. RICE</a></div></div><div id="_co_idm140464445935072"><h3 class="no_margin">ANDREW H. TALAL</h3><p><sup>§§§§§</sup>Division of Gastroenterology and Hepatology, Weill Cornell Medical College, New York, USA</p><div>Find articles by <a href="/pubmed/?term=TALAL%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">ANDREW H. TALAL</a></div></div><div id="_co_idm140464445933088"><h3 class="no_margin">IRA M. JACOBSON</h3><p><sup>§§§§§</sup>Division of Gastroenterology and Hepatology, Weill Cornell Medical College, New York, USA</p><div>Find articles by <a href="/pubmed/?term=JACOBSON%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">IRA M. JACOBSON</a></div></div><div id="_co_idm140464445931104"><h3 class="no_margin">MARC BOURLIÈRE</h3><p><sup>||||||||||</sup>Service d’Hépato-gastroentérologie, Hôpital Saint-Joseph, Marseilles, France</p><div>Find articles by <a href="/pubmed/?term=BOURLI%26%23x000c8%3BRE%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">MARC BOURLIÈRE</a></div></div><div id="_co_idm140464445929120"><h3 class="no_margin">IOANNIS THEODOROU</h3><p><sup>#</sup>Laboratory of Immunity and Infection, INSERM UMR-S 945, UPMC Université Paris 6, Groupe Hospitalier Pitié-Salpêtrière AP-HP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=THEODOROU%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">IOANNIS THEODOROU</a></div></div><div id="_co_idm140464445927136"><h3 class="no_margin">THIERRY POYNARD</h3><p><sup>¶¶¶¶¶</sup>Université Pierre et Marie Curie, Service d’Hépato-gastroentérologie, Hôpital Pitié-Salpêtrière (AP-HP), Paris, France</p><div>Find articles by <a href="/pubmed/?term=POYNARD%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">THIERRY POYNARD</a></div></div><div id="_co_idm140464445925152"><h3 class="no_margin">FRANCESCO NEGRO</h3><p><sup>#####</sup>Divisions of Gastroenterology and Hepatology and of Clinical Pathology, University Hospital, Geneva, Switzerland</p><div>Find articles by <a href="/pubmed/?term=NEGRO%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">FRANCESCO NEGRO</a></div></div><div id="_co_idm140464445923168"><h3 class="no_margin">STANISLAS POL</h3><p><sup>‡</sup>University Paris Descartes, Paris, France</p><p><sup>‡‡</sup>Département d’Hépatologie, INSERM U1016, Groupe Hospitalier Cochin-Hôtel Dieu-Broca, Paris, France</p><div>Find articles by <a href="/pubmed/?term=POL%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">STANISLAS POL</a></div></div><div id="_co_idm140464487171200"><h3 class="no_margin">PIERRE-YVES BOCHUD</h3><p><sup>**</sup>Service of Infectious Diseases, Department of Medicine, University Hospital and University of Lausanne, Lausanne, Switzerland</p><div>Find articles by <a href="/pubmed/?term=BOCHUD%20PY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">PIERRE-YVES BOCHUD</a></div></div><div id="_co_idm140464487169216"><h3 class="no_margin">LAURENT ABEL</h3><p><sup>*</sup>Laboratory of Human Genetics of Infectious Diseases, Necker Branch, Necker Medical School, Institut National de la Santé et de la Recherche Médicale (INSERM) U980, Paris, France</p><p><sup>‡</sup>University Paris Descartes, Paris, France</p><p><sup>§§</sup>St; Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, The Rockefeller University, New York, NY, USA</p><div>Find articles by <a href="/pubmed/?term=ABEL%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22841784">LAURENT ABEL</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140464489866064_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140464489866064_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140464489866064_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>*</sup>Laboratory of Human Genetics of Infectious Diseases, Necker Branch, Necker Medical School, Institut National de la Santé et de la Recherche Médicale (INSERM) U980, Paris, France</div><div class="fm-affl" id="A2" lang="en"><sup>‡</sup>University Paris Descartes, Paris, France</div><div class="fm-affl" id="A3" lang="en"><sup>||</sup>Department of Medical Genetics, University of Lausanne, Lausanne, Switzerland</div><div class="fm-affl" id="A4" lang="en"><sup>¶</sup>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div><div class="fm-affl" id="A5" lang="en"><sup>#</sup>Laboratory of Immunity and Infection, INSERM UMR-S 945, UPMC Université Paris 6, Groupe Hospitalier Pitié-Salpêtrière AP-HP, Paris, France</div><div class="fm-affl" id="A6" lang="en"><sup>**</sup>Service of Infectious Diseases, Department of Medicine, University Hospital and University of Lausanne, Lausanne, Switzerland</div><div class="fm-affl" id="A7" lang="en"><sup>‡‡</sup>Département d’Hépatologie, INSERM U1016, Groupe Hospitalier Cochin-Hôtel Dieu-Broca, Paris, France</div><div class="fm-affl" id="A8" lang="en"><sup>§§</sup>St; Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, The Rockefeller University, New York, NY, USA</div><div class="fm-affl" id="A9" lang="en"><sup>||||</sup>Biopredictive, Paris, France</div><div class="fm-affl" id="A10" lang="en"><sup>¶¶</sup>Laboratoire d’Immunologie et de Génétique des Maladies Parasitaires, INSERM-UMR 906/Université de la Méditerranée, Marseilles, France</div><div class="fm-affl" id="A11" lang="en"><sup>##</sup>Laboratoire Alphabio, Hôpital Ambroise Paré, Marseilles, France</div><div class="fm-affl" id="A12" lang="en"><sup>***</sup>CEA, Institut de Génomique, Centre National de Génotypage, Evry, France</div><div class="fm-affl" id="A13" lang="en"><sup>‡‡‡</sup>Division of Gastroenterology and Hepatology, University Hospital, Zurich, Switzerland</div><div class="fm-affl" id="A14" lang="en"><sup>§§§</sup>Division of Gastroenterology, Canton Hospital, St Gallen, Switzerland</div><div class="fm-affl" id="A15" lang="en"><sup>||||||</sup>Hepatology, Department of Clinical Research, University Clinic for visceral Surgery and Medicine, Bern, Switzerland</div><div class="fm-affl" id="A16" lang="en"><sup>¶¶¶</sup>Division of Gastroenterology and Hepatology, University Hospital, Basel, Switzerland</div><div class="fm-affl" id="A17" lang="en"><sup>###</sup>Centre Hospitalier Universitaire Vaudois, University of Lausanne, Switzerland</div><div class="fm-affl" id="A18" lang="en"><sup>****</sup>Clinica Moncucco, Lugano, Switzerland</div><div class="fm-affl" id="A19" lang="en"><sup>‡‡‡‡</sup>Pourtalès Hospital, Neuchâtel, Switzerland</div><div class="fm-affl" id="A20" lang="en"><sup>§§§§</sup>Institute for Medical Microbiology, University Hospital, Basel, Switzerland</div><div class="fm-affl" id="A21" lang="en"><sup>||||||||</sup>Institute of Medical Microbiology, Bellinzona, Switzerland</div><div class="fm-affl" id="A22" lang="en"><sup>¶¶¶¶</sup>Storr Liver Unit, Westmead Millennium Institute and Westmead Hospital, University of Sydney and University of Sydney Medical Foundation, Sydney, Australia</div><div class="fm-affl" id="A23" lang="en"><sup>####</sup>Institute for Immunology and Allergy Research, Westmead Millennium Institute, University of Sydney, Sydney, Australia</div><div class="fm-affl" id="A24" lang="en"><sup>*****</sup>INSERM U785, Liver Hepatology Centre, Paul Brousse Hospital, University Paris-XI, Villejuif, France</div><div class="fm-affl" id="A25" lang="en"><sup>‡‡‡‡‡</sup>Center for the Study of Hepatitis C, Rockefeller University, New York, USA</div><div class="fm-affl" id="A26" lang="en"><sup>§§§§§</sup>Division of Gastroenterology and Hepatology, Weill Cornell Medical College, New York, USA</div><div class="fm-affl" id="A27" lang="en"><sup>||||||||||</sup>Service d’Hépato-gastroentérologie, Hôpital Saint-Joseph, Marseilles, France</div><div class="fm-affl" id="A28" lang="en"><sup>¶¶¶¶¶</sup>Université Pierre et Marie Curie, Service d’Hépato-gastroentérologie, Hôpital Pitié-Salpêtrière (AP-HP), Paris, France</div><div class="fm-affl" id="A29" lang="en"><sup>#####</sup>Divisions of Gastroenterology and Hepatology and of Clinical Pathology, University Hospital, Geneva, Switzerland</div><div id="FN1">Reprint request: Address requests for reprints to: Pierre-Yves Bochud, Infectious Diseases Service, Department of Medicine, Institute of Microbiology, Rue du Bugnon 48, CH-1011 Lausanne CHUV, Switzerland. <a href="mailto:dev@null" data-email="hc.vuhc@duhcob.sevy-erreip" class="oemail">hc.vuhc@duhcob.sevy-erreip</a>; fax: +41 21 314 40 60; or Laurent Abel, Laboratoire de Génétique Humaine des Maladies Infectieuses, Université Paris Descartes-INSERM U980, Faculté de Médecine Necker, 156 rue de Vaugirard, 75015 Paris, France; <a href="mailto:dev@null" data-email="rf.mresni@leba.tnerual" class="oemail">rf.mresni@leba.tnerual</a>, phone: +33 (0)140615689, fax: +33 (0)140615688</div><div id="FN2"><sup>§</sup>Current address: Human Evolutionary Genetics, CNRS URA3012, Institut Pasteur, Paris, France;</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140464489866064_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN5">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22841784" target="pmc_ext" ref="reftype=publisher&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Gastroenterology</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3756935/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplementary data.</strong></div><div><a href="/pmc/articles/PMC3756935/bin/NIHMS477303-supplement-supplementary_data.PDF" data-ga-action="click_feat_suppl">NIHMS477303-supplement-supplementary_data.PDF</a><span style="color:gray"> (462K)</span></div><div class="small guid">GUID:&nbsp;1292410E-79E5-4D98-9DA4-1CEB1F30DBA4</div></div></dd></dl></div><div id="idm140464451798512" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140464451798512title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">BACKGROUND &amp; AIMS</h3><p id="P2" class="p p-first-last">Polymorphisms in <em>IL28B</em> were shown to affect clearance of hepatitis C virus (HCV) infection in genome-wide association (GWA) studies. Only a fraction of patients with chronic HCV infection develop liver fibrosis, a process that might also be affected by genetic factors. We carried out a 2-stage GWA study of liver fibrosis progression related to HCV infection.</p></div><div id="S2" class="sec"><h3 id="S2title">METHODS</h3><p id="P3" class="p p-first-last">We studied well-characterized HCV-infected patients of European descent who had liver biopsies before treatment. We defined various liver fibrosis phenotypes on the basis of Metavir scores, with and without taking the duration of HCV infection into account. Our GWA analyses were conducted on a filtered primary cohort of 1161 patients using 780,650 single nucleotide polymorphisms (SNPs). We genotyped 96 SNPs with <em>P</em>-values&lt;5×10<sup>−5</sup> from an independent replication cohort of 962 patients. We then assessed the most interesting replicated SNPs using DNA samples collected from 219 patients who participated in separate GWA studies of HCV clearance.</p></div><div id="S3" class="sec"><h3 id="S3title">RESULTS</h3><p id="P4" class="p p-first-last">In the combined cohort of 2342 HCV-infected patients, the SNPs rs16851720 (in the total sample) and rs4374383 (in patients that received blood transfusions) were associated with fibrosis progression (<em><sub>Pcombined</sub></em>=8.9×10<sup>−9</sup> and 1.1×10<sup>−9</sup>, respectively). The SNP rs16851720 is located within <em>RNF7</em>, which encodes an antioxidant that protects against apoptosis. The SNP rs4374383, together with another replicated SNP, rs9380516 (<em>P<sub>combined</sub></em>=5.4×10<sup>−7</sup>), were linked to the functionally related genes <em>MERTK</em> and <em>TULP1</em>, which encode factors involved in phagocytosis of apoptotic cells by macrophages.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">CONCLUSIONS</h3><p id="P5" class="p p-first-last">Our GWA study identified several susceptibility loci for HCV-induced liver fibrosis; these were linked to genes that regulate apoptosis. Apoptotic control might therefore be involved in liver fibrosis.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genetic analysis, risk factors, cirrhosis, liver disease</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P6" class="p p-first">End-stage chronic hepatitis C is the leading cause of liver transplantation in developed countries and more than 350,000 people die from HCV-related liver diseases each year.<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup> The natural course of chronic HCV infection is characterized by high levels of inter-individual variation in disease progression.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674020">2</a></sup> Most subjects never develop cirrhosis, but some may develop severe fibrosis in less than 20 years. A number of host demographic and clinical characteristics, as well as viral factors, have been associated with the development of HCV-related liver fibrosis.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674044">2</a>–<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674017">4</a></sup> However, these factors account for only a small proportion of the variability in the rate of liver fibrosis development, which overall remains unpredictable. There is accumulating evidence to suggest that host genetic factors are involved, although these factors remain largely unknown.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674049">2</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674026">5</a></sup> A number of candidate gene approaches have been used in attempts to identify variants influencing the development of liver disease in HCV-infected patients, but most of these studies produced results that were not consistently replicated.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674059">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674009">6</a></sup> Two related studies<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674048">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674041">8</a></sup> investigating ~25,000 putative functional SNPs identified a panel of SNPs predicting the risk of developing cirrhosis; this panel requires validation in prospective studies.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674062">8</a></sup> A recent study of 36 candidate genes, related to the fibrogenesis/fibrolysis process, identified a single cluster of variants of the <em>IFNGR2</em> gene associated with progression to severe fibrosis.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674037">9</a></sup> The results of these studies are interesting, but their approach may have prevented identification of genes strongly associated with liver fibrosis present in parts of the genome not tested.</p><p id="P7" class="p p-last">Genome-wide association (GWA) studies provide a broader and unbiased approach for the discovery of genetic factors involved in disease susceptibility.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674042">10</a></sup> For example, GWA studies identified a SNP cluster in the <em>IL28B</em> gene with a major effect on HCV clearance, either treatment-induced or spontaneous, whereas this gene had never previously been implicated in HCV infection.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674034">11</a></sup> A recent candidate gene study found that <em>IL28B</em> alleles associated with poor HCV clearance had a protective effect against liver inflammation and fibrosis.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_368016586">12</a></sup> No GWA study has yet explored genetic susceptibility to liver fibrosis in patients with chronic HCV infection. Several GWA studies of liver disease-related traits demonstrated a role for a non-synonymous variant of the <em>PNPLA3</em> gene, rs738409 (I148M), in the development of nonalcoholic and alcoholic fatty liver disease and associated disease severity.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674036">13</a>–<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674038">15</a></sup> The same <em>PNPLA3</em> risk allele was recently associated with steatosis and liver fibrosis in patients with chronic HCV infection.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674007">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674035">17</a></sup> In this study, we carried out a two-stage GWA study (primary screen followed by a replication study) in a combined cohort of 2,342 well characterized HCV-infected patients, to identify genetic factors influencing the development of HCV-related liver fibrosis.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Patients &amp; Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Patient Subjects</h3><p id="P8" class="p p-first-last">The sample used for the primary screen combined data from two cohorts of adult patients of European descent from France and Switzerland with chronic HCV infection. We retained only patients who had liver biopsy before treatment. The French cohort (ANRS Genoscan study group) included patients from the hepatology units of several hospitals in Paris and Marseilles; the inclusion criteria applied, including no co-infection by HIV or HBV, have been described elsewhere.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674019">9</a></sup> The Swiss Hepatitis C Cohort Study (SCCS) is a multicenter study of HCV-infected patients enrolled at eight major Swiss hospitals and the affiliated local centers. SCCS patient selection and data collection have also been described elsewhere,<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674025">3</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674033">18</a></sup> and patients with known HIV or active HBV co-infection were excluded for the present study. In total, 1,223 patients (490 from the French cohort and 733 from the SCCS) were eligible for genetic analyses. We studied three additional cohorts of European-descent adult patients with chronic HCV infection and not co-infected with HIV or HBV, with the aim of replicating the principal signals obtained in the primary cohorts. All 962 patients included in these cohorts had biopsy before treatment (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). A first cohort of 64 US patients was recruited at the Weill-Cornell Medical Center in New York, and a second sample of 256 French patients was recruited from different hospitals in Marseilles. The third replication cohort included 642 patients recruited from centers in Australia, Germany, the United Kingdom and Italy, as described elsewhere.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674028">19</a>,<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674053">20</a></sup> Finally, the seven signals showing evidence of true replication in the cohort combining primary and replication cohorts were also tested in an additional independent sample of Australian patients consisting of the primary cohort of a previous GWA study of response to hepatitis C treatment.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674043">19</a></sup> After excluding Australian patients with missing phenotype data, 219 individuals were kept for this analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Clinical risk factors, history of HCV acquisition and of alcohol consumption (assessed using time-line follow back interview) were recorded in the corresponding cohorts through face-to-face interviews conducted by physicians trained in addiction problems. The sampling of all the cohorts was approved by the appropriate institutional review boards, and written informed consent was obtained from all patients.</p></div><div id="S8" class="sec"><h3 id="S8title">Determination of Liver Fibrosis Phenotypes</h3><p id="P9" class="p p-first-last">The stage of liver fibrosis in patients with chronic HCV infection was determined by examination of a liver biopsy specimen obtained before treatment, with quantification according to the Metavir score, on a five-point scale from F0 to F4.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674023">21</a></sup> Fibrosis was assessed by experienced pathologists working in the local expert liver centers corresponding to the place of collection, and all biopsies were considered as interpretable by each local pathologist. Metavir scoring system is validated since many years and highly reproducible for pathologists specialized in liver disease.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674006">22</a></sup> For each patient, we used the information obtained from a single biopsy, referred to as the reference biopsy. For patients who had undergone several biopsies, the reference biopsy was defined as follows: (i) the most recent biopsy obtained in the absence of treatment for patients who had not spontaneously developed fibrosis, i.e. all biopsy results without treatment were F0 or F1; (ii) the biopsy with the highest Metavir score for patients who had developed fibrosis, i.e. at least one biopsy result ≥F2; the earliest biopsy was selected if there were several biopsies with the highest score. We used three different approaches to define liver fibrosis phenotype on the basis of the Metavir score for the reference biopsy. The first of these approaches was a classic case/control approach in which F0-1 patients were considered as controls and F3-4 patients, as cases. Patients with an intermediate Metavir score (F2) were discarded for this analysis, which was thus a <em>binary F0-1/F3-4</em> phenotype analysis. Within this approach, we also used a more extreme definition of the phenotypes, in which we considered only F0 patients as controls and F4 patients as cases (<em>binary F0/F4</em> phenotype analysis). In the second approach, we used survival analysis techniques to take into account the individual duration of infection in the analysis. Failure (F3-4 or only F4) and censored (F0-1 or only F0) events were defined as in the case/control study, and the duration of infection was estimated from the presumed year of HCV acquisition to the year in which the biopsy was carried out. The estimated date of HCV acquisition was obtained using the first reported major event at risk among blood transfusion, drug use (the first year of injecting drug use (IDU) was used as the starting point), accidental needle stick or other invasive medical procedures. In this analysis, the phenotype is denoted <em>duration F0-1/F3-4</em> or <em>duration F0/F4</em>. In the third approach, liver fibrosis progression was considered as a quantitative phenotype, fibrosis progression rate (FPR), corresponding to the ratio of Metavir score to the estimated duration of infection in years (Metavir units per year).<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674015">23</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674032">24</a></sup> Patients with a F2 Metavir score were included in this analysis. Given the distribution of FPR, this phenotype was log-transformed and then inverse normal quantile-transformed in each cohort before statistical analysis. The resulting phenotype, denoted as <em>QTF</em> phenotype, was further standardized such that the differences in mean QTF rates between genotypes for a given SNP could be expressed in standard deviation units, referred to as standardized Metavir units (SMUs). Posterior to the selection of the 96 followed-up SNPs (see next sections), we analyzed our primary data by modeling transition probabilities between Metavir stages using a Markov model, described in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>. This approach was not further considered, because it provided no additional association signals with <em>P</em> values&lt;5×10<sup>−6</sup> in our primary cohort, with respect to analyses using the three other phenotype definitions.</p></div><div id="S9" class="sec"><h3 id="S9title">Genotyping and Imputation Procedures</h3><p id="P10" class="p p-first-last">Details of genotyping methods, quality controls and imputation procedures are provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>. Briefly, the French and Swiss primary cohorts were genotyped for ~350,000 SNPs and ~1,000,000 SNPs, respectively, using Illumina HumanCNV370-Duo and Human1M-Duo beadchips (Illumina, San Diego, USA). Quality-control filtering of SNPs resulted in a total of 325,624 and 912,765 high-quality SNPs, respectively. Genotype imputation was performed in the French primary cohort, using the Swiss cohort as a template. This procedure yielded a total of 780,650 high-quality genotyped or imputed SNPs, which were used for all analyses. Quality-control filtering was also carried out for individuals, and 1,161 filtered individuals were used for GWA analyses (<a href="/pmc/articles/PMC3756935/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The 962 subjects of the replication cohorts (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>) were genotyped for 96 followed-up SNPs by Illumina GoldenGate genotyping with VeraCode technology (Illumina), as well as our French primary cohort for 33 out of the 96 SNPs that were initially imputed in this cohort. Eighty-seven high-quality SNPs were tested for replication. Among the seven SNPs showing evidence for true replication, two SNPs, rs16851720 and rs4374383, were not present in the GWA cohort of 219 Australian patients, genotyped by the Illumina Infinium HumanHap300 or the CNV370-Quad genotyping BeadChips (Illumina). We could impute these two SNPs with a high accuracy of 0.996 and 0.984, respectively (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p19">Clinical and demographic characteristics of the two primary cohorts.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140464445675456" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Covariate</th><th rowspan="1" colspan="1" valign="top" align="center">Category</th><th rowspan="1" colspan="1" valign="top" align="center">French cohort<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Swiss cohort<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Total</th></tr></thead><tbody><tr><td colspan="5" rowspan="1" valign="top" align="left"><em>Sex</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">male</td><td rowspan="1" colspan="1" valign="top" align="center">209 (44.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">433 (62.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">642 (55.3%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">female</td><td rowspan="1" colspan="1" valign="top" align="center">258 (55.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">261 (37.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">519 (44.7%)</td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="5" rowspan="1" valign="top" align="left"><em>Metavir score</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">F0</td><td rowspan="1" colspan="1" valign="top" align="center">41 (8.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">77 (11.1%)</td><td rowspan="1" colspan="1" valign="top" align="center">118 (10.2%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">F1</td><td rowspan="1" colspan="1" valign="top" align="center">244 (52.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">217 (31.3%)</td><td rowspan="1" colspan="1" valign="top" align="center">461 (39.7%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">F2</td><td rowspan="1" colspan="1" valign="top" align="center">20 (4.3%)</td><td rowspan="1" colspan="1" valign="top" align="center">192 (27.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">212 (18.3%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">F3</td><td rowspan="1" colspan="1" valign="top" align="center">93 (19.9%)</td><td rowspan="1" colspan="1" valign="top" align="center">86 (12.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">179 (15.4%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">F4</td><td rowspan="1" colspan="1" valign="top" align="center">69 (14.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">122 (17.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">191 (16.5%)</td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="5" rowspan="1" valign="top" align="left"><em>Alcohol consumption</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Low (&lt;40g/day)</td><td rowspan="1" colspan="1" valign="top" align="center">399 (85.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">542 (78.1%)</td><td rowspan="1" colspan="1" valign="top" align="center">941 (81.1%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">High<sup>b</sup> (&gt;40g/day)</td><td rowspan="1" colspan="1" valign="top" align="center">68 (14.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">132 (19.0%)</td><td rowspan="1" colspan="1" valign="top" align="center">200 (17.2%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Not available (NA)</td><td rowspan="1" colspan="1" valign="top" align="center">0 (0.0%)</td><td rowspan="1" colspan="1" valign="top" align="center">20 (2.9%)</td><td rowspan="1" colspan="1" valign="top" align="center">20 (1.7%)</td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="5" rowspan="1" valign="top" align="left"><em>HCV mode of acquisition</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">IDU<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="center">157 (33.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">290 (41.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">447 (38.5%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Blood transfusion</td><td rowspan="1" colspan="1" valign="top" align="center">205 (43.9%)</td><td rowspan="1" colspan="1" valign="top" align="center">133 (19.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">338 (29.1%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Others<sup>c</sup>/NA</td><td rowspan="1" colspan="1" valign="top" align="center">105 (22.5%)</td><td rowspan="1" colspan="1" valign="top" align="center">271 (39.0%)</td><td rowspan="1" colspan="1" valign="top" align="center">376 (32.4%)</td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="5" rowspan="1" valign="top" align="left"><em>HCV genotype</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">294 (63.0%)</td><td rowspan="1" colspan="1" valign="top" align="center">362 (52.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">656 (56.5%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">41 (8.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">67 (9.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">108 (9.3%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">75 (16.1%)</td><td rowspan="1" colspan="1" valign="top" align="center">193 (27.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">268 (23.1%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">10 (2.1%)</td><td rowspan="1" colspan="1" valign="top" align="center">57 (8.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">67 (5.8%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">47 (10.1%)</td><td rowspan="1" colspan="1" valign="top" align="center">15 (2.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">62 (5.3%)</td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="5" rowspan="1" valign="top" align="left"><em>Age at infection</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Patients with available data</td><td rowspan="1" colspan="1" valign="top" align="center">450 (96.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">614 (88.5%)</td><td rowspan="1" colspan="1" valign="top" align="center">1064 (91.6%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Age at infection &lt; 20 years</td><td rowspan="1" colspan="1" valign="top" align="center">139 (29.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">304 (43.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">443 (38.2%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Age at infection ≥20 years</td><td rowspan="1" colspan="1" valign="top" align="center">311 (66.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">310 (44.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">621 (53.5%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Mean (SD) age at infection (<em>years</em>)</td><td rowspan="1" colspan="1" valign="top" align="center">28.00 (12.93)</td><td rowspan="1" colspan="1" valign="top" align="center">21.20 (10.66)</td><td rowspan="1" colspan="1" valign="top" align="center">24.08 (12.14)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Mean (SD) duration of infection (<em>years</em>)</td><td rowspan="1" colspan="1" valign="top" align="center">20.17 (9.25)</td><td rowspan="1" colspan="1" valign="top" align="center">22.39 (10.61)</td><td rowspan="1" colspan="1" valign="top" align="center">21.45 (10.11)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Mean (SD) FPR (<em>Metavir units/year</em>)</td><td rowspan="1" colspan="1" valign="top" align="center">0.11 (0.16)</td><td rowspan="1" colspan="1" valign="top" align="center">0.11 (0.15)</td><td rowspan="1" colspan="1" valign="top" align="center">0.11 (0.16)</td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> <strong>Total</strong></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">467</td><td rowspan="1" colspan="1" valign="top" align="center">694</td><td rowspan="1" colspan="1" valign="top" align="center">1,161</td></tr></tbody></table></div><div id="largeobj_idm140464445675456" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3756935/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>French patients were included if they were not co-infected with HIV or HBV and had no coexisting chronic liver disease. We also preferentially enrolled patients if (i) they had F0-1 or F3-4 Metavir scores, (ii) their presumed date of HCV acquisition was known and (iii) they had low levels of alcohol consumption. Swiss patients with known HIV or active HBV co-infection were excluded for the purposes of this study. For both cohorts, relevant factors were retrieved from clinical databases, including sex, age at infection, mode of HCV acquisition and HCV genotype.</div><div id="TFN2"><sup>b</sup>This category includes past heavy drinkers.</div><div id="TFN3"><sup>c</sup>IDU stands for injecting drug use. Other modes of HCV acquisition include situations at risk of exposure to blood (e.g. health workers) and invasive protocols (e.g. surgery, tattoo, piercing).</div></div></div></div><div id="S10" class="sec sec-last"><h3 id="S10title">Statistical Analyses</h3><p id="P11" class="p p-first">For liver fibrosis considered <em>as binary</em> phenotypes<em>,</em> we used Fisher’s exact test and logistic regression. For <em>duration</em> phenotypes, we used a Cox model considering estimated age at infection as the starting point and the first biopsy showing severe fibrosis (failure time) or the last biopsy showing an absence of severe fibrosis in the absence of treatment (censored time) as the endpoint. Logistic regression and Cox model analyses were used for stratified and multivariate adjusted analyses, with sex, HCV genotype, age at infection, alcohol consumption and/or mode of HCV acquisition as covariates. For the <em>QTF</em> phenotype, linear regression analysis was performed. All statistical analyses were carried out with procedures implemented in SAS software v.8.2 (SAS Institute, Cary, North Carolina, USA), R software (<a href="http://www.R-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.R-project.org/</a>), Matlab (<a href="http://www.mathworks.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.mathworks.com</a>) and PLINK.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674060">25</a></sup> The power of this study was estimated for the <em>binary F0-1/3–4</em> and the <em>duration F0-1/3–4</em> phenotypes, as described in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>.</p><p id="P12" class="p p-last">As we conducted several analyses, using different phenotypes, genetic models and stratified samples, we estimated the effective number of tests in our study by a method detailed in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>. Briefly, we estimated that the analyses conducted in the whole sample corresponded to a total number of ~3 million effective tests. Adjusted and stratified analyses were performed only on the <em>binary</em> and <em>duration F0-1/F3-4</em> phenotypes, for the 140,797 SNPs giving <em>P</em>-values&lt;0.05 in the analyses conducted on these two phenotypes in the whole sample, taking eight strata into account: age at infection&gt;20 years, age at infection&lt;20 years, male, female, contamination by transfusion, contamination by IDU, HCV genotype=1 and HCV genotype≠1. The number of effective tests for these stratified analyses was estimated at ~1.6 million, giving a total of ~4.9 million independent tests for our entire GWA analysis. We therefore considered <em>P</em>-values&lt;10<sup>−8</sup> (0.05/4.9×10<sup>6</sup>) to indicate genome-wide significance in our study.</p></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Results</h2><div id="S12" class="sec sec-first"><h3 id="S12title">Genome-wide analyses in the primary cohort</h3><p id="P13" class="p p-first">GWA analyses were conducted on a filtered primary cohort of 1,161 HCV-infected patients (<a href="/pmc/articles/PMC3756935/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), with a total of 780,650 SNPs. We first used the <em>binary F0-1/F3-4</em> phenotype of liver fibrosis and assumed an additive genetic model. No significant deviations from expectations were observed on quantile-quantile plots before and after correction for population stratification (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>), attesting to an absence of difference in ancestry between cases and controls (genomic inflation factor <em>λ</em>=1.013). In addition, none of the first 10 principal components of ancestry was associated with case-control status (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a>). We therefore carried out all subsequent analyses without adjustment for population stratification.</p><p id="P14"><a href="/pmc/articles/PMC3756935/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a> shows the Manhattan plot obtained for analysis of the <em>binary F0-1/F3-4</em> phenotype<em>,</em> for the best among the three genetic models tested (additive, recessive, dominant). <em>P</em>-values&lt;10<sup>−6</sup> were obtained for two SNPs on chromosomes 2 and 8 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). When restricting our cohort to the most extreme Metavir scores (<em>binary F0/F4</em> phenotype), a single locus on chromosome 2, tagged by four SNPs in strong linkage disequilibrium (LD), achieved a <em>P</em>-value&lt;10<sup>−6</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3A</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). We then carried out GWA analyses taking into account the estimated duration of infection in 1,064 subjects for whom follow-up information was available. Two SNPs on chromosomes 11 and 18 provided <em>P</em>-values&lt;10<sup>−6</sup> when considering the <em>duration F0-1/F3-4</em> phenotype (<a href="/pmc/articles/PMC3756935/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). When restricting the analysis to the <em>duration F0/F4</em> phenotype, a <em>P</em>-value&lt;10<sup>−6</sup> was obtained for one SNP on chromosome 6 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3B</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Finally, the GWA analysis of the <em>QTF</em> phenotype identified a single SNP on chromosome 3 with a <em>P</em>-value&lt;10<sup>−6</sup> (<a href="/pmc/articles/PMC3756935/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1C</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). For all phenotypes, 45 additional independent signals gave <em>P</em>-values&lt;5×10<sup>−5</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3756935/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140464445687840" class="figure"><a href="/pmc/articles/PMC3756935/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3756935_nihms477303f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms477303f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3756935/bin/nihms477303f1.jpg"></a></div><div id="largeobj_idm140464445687840" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3756935/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3756935/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p17">Manhattan plots of genome-wide analyses of liver fibrosis using (<strong>A</strong>) the <em>binary F0-1/F3-4</em> phenotype (949 F0-1/F3-4 patients), (<strong>B</strong>) the <em>duration F0-1/F3-4</em> phenotype (872 F0-1/F3-4 patients with available duration of infection) and (<strong>C</strong>) the <em>QTF</em> phenotype (1,064 patients with available duration of infection). Larger points correspond to SNPs producing a <em>P</em>-value&lt;10<sup>−6</sup>. All the analyses were performed on 780,650 genotyped or imputed SNPs (Patients &amp; Methods).</p></div></div></div><p id="P15" class="p p-last">All the classical risk factors for fibrosis development available in our sample (i.e. sex, alcohol consumption, HCV genotype, HCV mode of acquisition and age at infection) were significantly and independently associated with fibrosis progression in our sample, in multivariate logistic regression analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Multivariate analyses of our <em>binary</em> and <em>duration F0-1/F3-4</em> phenotypes adjusted for these factors did not substantially change our previous results (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). We also performed adjusted and stratified analyses for the <em>binary</em> and <em>duration F0-1/F3-4</em> phenotypes, using the 140,797 SNPs that achieved a <em>P</em>-value&lt;0.05 in one of these two analyses. These analyses were performed taking into account binarized covariables such as sex (male/female), HCV genotype (1/others) or mode of acquisition (blood transfusion/IDU). In analyses with the <em>duration F0-1/F3-4</em> phenotype, six SNPs gave <em>P</em>-values&lt;10<sup>−6</sup>, including three with <em>P</em>-values&lt;10<sup>−7</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). In total, 38 additional independent signals gave <em>P</em>-values&lt;5×10<sup>−5</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">Replication study</h3><p id="P16" class="p p-first">In the second phase, we genotyped a total of 96 independent SNPs presenting a <em>P</em>-value&lt;5×10<sup>−5</sup> in one of our previous GWA analyses (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>) in a replication cohort of 962 HCV-infected patients (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Eighty-seven of these SNPs satisfying the quality-control filters were tested for association. Evidence for true replication (using the same model as in the primary analysis) at the 0.05 level was obtained for seven SNPs (<a href="/pmc/articles/PMC3756935/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We further tested these seven SNPs in an additional filtered cohort of 219 Australian patients with liver biopsy data (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>), who were previously genotyped for ~310,000 SNPs on Illumina beadchips in a GWA study of response to chronic hepatitis C treatment.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674031">19</a></sup> Out of these seven SNPs, two (rs16851720 and rs4374383) were not present on the arrays, and were successfully imputed in this Australian cohort (Patients &amp; Methods).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p23">Replicated SNPs from our GWA analyses of different phenotypes of liver fibrosis in patients with chronic hepatitis C infection.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140464449340064" class="xtable"><table frame="hsides" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">Closest gene</th><th rowspan="1" colspan="1" valign="middle" align="center">Distance</th><th rowspan="1" colspan="1" valign="middle" align="center">m/M<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">mAF</th><th rowspan="1" colspan="1" valign="middle" align="center">Phenotype</th><th rowspan="1" colspan="1" valign="middle" align="center">Model<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Discovery <em>P</em>-value<sup>b</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Initial combined <em>P</em>-value<sup>b</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Total combined <em>P</em>-value<sup>b</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Hetero-geneity <em>P</em>-value<sup>c</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Genotypes (n)</th><th rowspan="1" colspan="1" valign="middle" align="center">Effect<sup>d</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs16851720</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>RNF7</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">C/A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.19</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>QTF</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Additive</td><td rowspan="1" colspan="1" valign="middle" align="center">4.5×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">6.2×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">8.9×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.37</td><td rowspan="1" colspan="1" valign="middle" align="center">AA (1046)<br>AC (534)<br>CC (56)</td><td rowspan="1" colspan="1" valign="middle" align="center">0<br>−0.23 (−0.31 - −0.15)<br>−0.46 (−0.54 - −0.38)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs4374383</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>MERTK</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">A/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.42</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>Duration F0-1/F3-4</em> in transfused patients</td><td rowspan="1" colspan="1" valign="middle" align="center">Recessive</td><td rowspan="1" colspan="1" valign="middle" align="center">2.7×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.1×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.1×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.81</td><td rowspan="1" colspan="1" valign="middle" align="center">AG/GG (257)<br>AA (62)</td><td rowspan="1" colspan="1" valign="middle" align="center">1<br>0.19 (0.10–0.37)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs9380516</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>TULP1</em></td><td rowspan="1" colspan="1" valign="middle" align="center">21</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.17</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>Duration F0-1/F3-4</em> in male patients</td><td rowspan="1" colspan="1" valign="middle" align="center">Recessive</td><td rowspan="1" colspan="1" valign="middle" align="center">4.0×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">9.6×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">5.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.88</td><td rowspan="1" colspan="1" valign="middle" align="center">TC/CC (602)<br>TT (20)</td><td rowspan="1" colspan="1" valign="middle" align="center">1<br>4.53 (2.78–7.39)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2629751</td><td rowspan="1" colspan="1" valign="middle" align="center">12</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>GLT8D2</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">G/A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.31</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>Binary F0/F4</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Recessive</td><td rowspan="1" colspan="1" valign="middle" align="center">1.6×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.37</td><td rowspan="1" colspan="1" valign="middle" align="center">GA/AA (546)<br>GG (49)</td><td rowspan="1" colspan="1" valign="middle" align="center">1<br>7.10 (2.94–20.74)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs883924</td><td rowspan="1" colspan="1" valign="middle" align="center">9</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>LOC340515</em></td><td rowspan="1" colspan="1" valign="middle" align="center">43</td><td rowspan="1" colspan="1" valign="middle" align="center">A/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.26</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>Duration F0-1/F3-4</em> age at infection &gt; 20y</td><td rowspan="1" colspan="1" valign="middle" align="center">Recessive</td><td rowspan="1" colspan="1" valign="middle" align="center">3.7×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">4.6×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.8×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.15</td><td rowspan="1" colspan="1" valign="middle" align="center">AG/GG (765)<br>AA (68)</td><td rowspan="1" colspan="1" valign="middle" align="center">1<br>2.69 (1.88–3.85)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs7800244</td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>PKD1L1</em></td><td rowspan="1" colspan="1" valign="middle" align="center">34</td><td rowspan="1" colspan="1" valign="middle" align="center">T/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.18</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>Binary F0-1/F3-4</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Dominant</td><td rowspan="1" colspan="1" valign="middle" align="center">1.8×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2.9×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">3.4×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.06</td><td rowspan="1" colspan="1" valign="middle" align="center">GG (1210)<br>TG/TT (602)</td><td rowspan="1" colspan="1" valign="middle" align="center">1<br>0.61 (0.49–0.75)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs6485480</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>ALKBH3</em></td><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.22</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>Binary F0/F4</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Additive</td><td rowspan="1" colspan="1" valign="middle" align="center">2.5×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">5.1×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">3.2×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td><td rowspan="1" colspan="1" valign="middle" align="center">CC (359)<br>TC (201)<br>TT (36)</td><td rowspan="1" colspan="1" valign="middle" align="center">1<br>1.86 (1.41–2.46)<br>3.46 (1.99–6.05)</td></tr></tbody></table></div><div id="largeobj_idm140464449340064" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3756935/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><sup>a</sup>The first allele m refers to the minor allele, and M to the major allele, in the combined cohort. The genetic model is defined on the basis of the minor allele.</div><div id="TFN5"><sup>b</sup>The discovery <em>P</em>-value was obtained in the primary cohort of 1,161 Swiss and French patients; the initial combined <em>P</em>-value was obtained in the cohort of 2,123 patients, including Swiss and French primary cohorts (<a href="/pmc/articles/PMC3756935/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), and our replication cohort (Cornell, Marseilles and International cohorts; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>); the total combined <em>P</em>-value was obtained by combining this cohort of 2,123 patients with the sample of 219 Australian patients (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>) previously genotyped in another GWA study.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674021">19</a></sup></div><div id="TFN6"><sup>c</sup>Heterogeneity was tested among primary, replication and Australian cohorts using the Cochran’s Q-statistics.<sup><a href="#R46" rid="R46" class=" bibr popnode">46</a></sup></div><div id="TFN7"><sup>d</sup>The effect is computed from the total sample of 2,342 patients. Effects correspond to the standardized difference in QTF rates, relative to the first genotype (Patients &amp; Methods), when using the quantitative <em>QTF</em> phenotype, hazard ratios when the <em>duration</em> phenotype is considered, and odds-ratios when the <em>binary</em> phenotype is considered.</div></div></div><p id="P17">The combined analysis of these seven SNPs in the total sample of 2,342 patients identified two SNPs (<a href="/pmc/articles/PMC3756935/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) with <em>P</em>-values below our estimated genome-wide threshold for significance of 10<sup>−8</sup> (see Patients &amp; Methods for details). Tests of heterogeneity across primary and replication cohorts were not significant for either of these SNPs (<a href="/pmc/articles/PMC3756935/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). SNP rs16851720 provided a combined <em>P</em>-value=8.9×10<sup>−9</sup> in the whole sample, in the analysis of the <em>QTF</em> phenotype (additive for C, difference between genotypes of 0.23 SMU (0.15–0.31); <a href="/pmc/articles/PMC3756935/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>). This intronic SNP is located in <em>RNF7</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4A</a>), encoding a redox-inducible antioxidant protein that protects against apoptosis.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674051">26</a></sup> The second SNP was rs4374383, with <em>P</em>-value=1.1×10<sup>−9</sup> in blood-transfused patients in analysis of the <em>duration F0-1/F3-4</em> phenotype (recessive for minor allele A, hazard ratio of developing liver fibrosis (HR) for AA vs.AG/GG=0.18 (0.09–0.36); <a href="/pmc/articles/PMC3756935/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>). We found that this association was independent of viral genotypes (data not shown) although blood-transfused patients are more often infected with HCV genotypes 1 and 2. SNP rs4374383 is intronic in the <em>MERTK</em> gene (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4B</a>), a member of the three TAM receptor tyrosine kinases which are involved in the regulation of inflammatory responses.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674027">27</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3756935/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140464445678112" class="figure"><a href="/pmc/articles/PMC3756935/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3756935_nihms477303f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms477303f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3756935/bin/nihms477303f2.jpg"></a></div><div id="largeobj_idm140464445678112" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3756935/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3756935/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p18">Genotype/phenotype relationships for the four SNPs most associated with liver fibrosis in the final combined cohort. (<strong>A</strong>) Standardized Metavir units (SMU; Patients &amp; Methods) for the three genotypes of SNP rs16851720 (located within <em>RNF7</em>), identified using the <em>QTF</em> phenotype. Bars represent standard errors (SEM) of each mean SMU. Genotype AA was used as the reference (SMU=0). (<strong>B</strong>) Survival curves for SNP rs4374383 (located within <em>MERTK</em>) identified using the <em>duration F0-1/F3-4</em> phenotype, in the subsample of transfused patients. (<strong>C</strong>) Proportions of all HCV-infected patients with Metavir scores of F0 and F4, by genotype at rs2629751 (located within <em>GLT8D2</em>), a replicated SNP identified using the <em>binary F0/F4</em> phenotype. (<strong>D</strong>) Survival curves for SNP rs9380516 (located near <em>TULP1</em>), a replicated SNP identified using the <em>duration F0-1/F3-4</em> phenotype stratified in male patients.</p></div></div></div><p id="P18">Out of the five remaining replicated SNPs, two were improved or unchanged by the addition of the Australian cohort (<a href="/pmc/articles/PMC3756935/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). One was rs2629751 in analysis of the <em>binary F0/F4</em> phenotype (P=1.4×10<sup>−7</sup>; recessive for G, OR=7.10 (2.94–20.74); <a href="/pmc/articles/PMC3756935/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2C</span></a>), which is intronic in the <em>GLT8D2</em> gene. The other signal was identified using the <em>duration F0-1/F3-4</em> phenotype in men at SNP rs9380516 (P=5.4×10<sup>−7</sup>; recessive for T, HR=4.53 (2.78–7.39); <a href="/pmc/articles/PMC3756935/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2D</span></a>), which is located 21 kb downstream from the <em>TULP1</em> gene. The last three signals (SNPs rs883924, rs7800244 and rs6485480) were clearly less significant when the data for the Australian cohort were included in the analysis (<a href="/pmc/articles/PMC3756935/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Consistent with this observation, tests of heterogeneity across cohorts were close to significance (<em>P</em>=0.06) and significant (<em>P</em>=0.01) for rs7800244 and rs6485480, respectively (<a href="/pmc/articles/PMC3756935/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P19" class="p p-last">Finally, we assessed, in our primary cohort, the association of liver fibrosis with a number of variants reported in large association studies (GWA studies or investigations of a large number of genes) to be associated with either liver fibrosis-related phenotypes in non HCV patients or other HCV-related phenotypes (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). Using the same methods as two previous studies partly overlapping with our primary cohort, we found that rs9976971 in <em>IFNGR2</em>
<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674018">9</a></sup> and rs8099917 in <em>IL28B</em>
<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_368016585">12</a></sup> were associated with liver fibrosis, with <em>P</em>-values of 0.009 and 0.018, respectively. We found no significant effect of variants reported to be associated with liver fibrosis in Caucasian patients with chronic HCV infection,<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674064">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674055">8</a></sup> with liver fibrosis in Spanish patients with non-alcoholic fatty liver disease<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674005">28</a></sup> or with the development of hepatocellular carcinoma in HCV-infected Japanese patients.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674022">29</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674047">30</a></sup> However, we found that rs738409 in <em>PNPLA3</em> was associated with liver fibrosis in our primary cohort, in analyses with the <em>binary F0-1/F3-4</em> phenotype (additive model for allele G, <em>P</em>=1.3×10<sup>−4</sup>, OR=1.21 (1.10–1.34)). The non-synonymous <em>PNPLA3</em> variant rs738409 has been shown to be associated with several liver-related traits in GWA studies,<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674046">13</a>–<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674014">15</a></sup> and with liver steatosis and fibrosis in patients chronically infected with HCV in studies based on candidate gene approaches.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674057">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674056">17</a>,<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674054">31</a></sup></p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Discussion</h2><p id="P20" class="p p-first">We report the first GWA study investigating liver fibrosis progression in a large sample of more than 2,300 HCV-infected patients of European descent. For all patients, liver biopsy data were obtained before treatment. We defined several fibrosis phenotypes on the basis of histological findings, because the Metavir system grading is not linear,<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674058">21</a></sup> and because we wished to investigate various and complementary aspects of the fibrosis process. In this context, we also sought to use an additional, more sophisticated approach, involving a Markov model of the probability of transition between Metavir stages (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). This approach provided no additional signals with <em>P</em> values&lt;5×10<sup>−6</sup> for our primary cohort (data not shown). Overall, our analyses covered a large panel of liver fibrosis phenotypes in a well characterized sample of HCV-infected patients, and identified four replicated signals including two that were significant at the genome-wide level in the combined cohort.</p><p id="P21">Gene ontology enrichment analysis of our four most interesting signals revealed significant enrichment for two categories, “photoreceptor outer segment” and “phagocytosis” (<em>P</em>=5×10<sup>−4</sup> and <em>P</em>=5×10<sup>−3</sup>, respectively, after Bonferroni correction; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). This result is explained by two SNPs: rs4374383, our most significant signal, and rs9380516, located within or close to <em>MERTK</em> and <em>TULP1</em>, respectively. Interestingly, rs4374383 is in strong LD (<em>r<sup>2</sup></em>&gt;0.9, measured using HapMap phase II data in the CEU European-descent population) with two <em>MERTK</em> non-synonymous SNPs, rs7604639 and rs3811635, and rs9380516 is in strong LD (<em>r<sup>2</sup></em>=0.9 in CEU) with an intronic SNP of <em>TULP1</em>, rs9296155. Although Mendelian defects of these two genes affect only the retina,<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674050">32</a>–<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674063">34</a></sup> both genes have a broader spectrum of expression, with <em>TULP1</em> expressed in the fetal liver<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674012">35</a></sup> and <em>MERTK</em> expressed in macrophages.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674016">27</a></sup> In addition, the MERTK and TULP1 proteins have been shown to interact during the phagocytosis of apoptotic cells.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674039">36</a></sup> In particular, mice lacking specifically the murine homolog of <em>MERTK</em> display a deficiency in the clearance of apoptotic thymocytes by macrophages.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674013">37</a></sup> Interestingly, the clearance of apoptotic debris by phagocytosis can directly stimulate fibrogenesis.<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674011">38</a>,<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674010">39</a></sup> Hepatic stellate cells (HSC) may play a key role in this process, as they have phagocytic functions, including the NADPH oxidase activity required for the production of reactive oxygen species (ROS).<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674008">40</a></sup> HSC are also found in close proximity to hepatocytes, a key source of apoptotic debris during liver injury.<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674029">41</a></sup> Overall, our results suggest a possible relationship between the clearance of apoptotic cells through phagocytosis and liver fibrosis.</p><p id="P22">The second signal significant at the genome-wide level was that for SNP rs16851720 in the whole sample. The effect of this SNP on fibrosis progression rate (FPR) was clearly additive in our combined cohort (<a href="/pmc/articles/PMC3756935/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>). If FPR means were calculated directly as a function of rs16851720 genotypes, we found that each A risk allele increased the rate of liver fibrosis progression by 0.033 Metavir units/year, corresponding to ~1 Metavir unit over a 30-year period. This SNP is located in the first intron of <em>RNF7</em>, which is also known as <em>SAG</em> (sensitive to apoptosis gene). RNF7 is a cell-protecting molecule that acts as an antioxidant, inhibiting the apoptosis induced by metal ions and ROS,<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674024">26</a></sup> thereby preventing DNA damage.<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674030">42</a></sup> In this context, it is interesting to note that HCV inhibits host DNA damage repair through ROS production.<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674040">43</a></sup> The induction of these processes, including high levels of ROS production in particular, in HCV-infected cells leads to hepatocyte apoptosis and HSC activation, and may also contribute to the development of liver fibrosis<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674061">44</a></sup> through mechanisms described above.</p><p id="P23" class="p p-last">In conclusion, for the whole sample, we found a genome-wide significant effect of the SNP rs16851720 on liver fibrosis, and confirmed the role of the <em>PNPLA3</em> non-synonymous I148M variant. We also found some interesting additional signals, including one genome-wide significant, in more refined subsamples, which require further investigation. Overall, our findings suggest that several variants in a number of genes with rather modest effects (in the whole sample) and/or more complex effects (e.g. limited to a specific subsample) may be involved in genetic susceptibility to liver fibrosis secondary to chronic HCV infection. They also point out the role of apoptosis potentially providing new insights into the mechanisms underlying liver fibrosis development.<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674045">41</a>,<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330674052">45</a></sup></p></div><div id="S15" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supplementary data</h4><div class="sup-box half_rhythm" id="d36e1362"><a href="/pmc/articles/PMC3756935/bin/NIHMS477303-supplement-supplementary_data.PDF" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(462K, PDF)</sup></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Acknowledgments</h2><div class="sec"><p id="__p2">E. P. and Z. K. contributed equally to the work. P.-Y. B. and L. Abel jointly directed the work. The members of the different study groups are listed in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Material</a>. We warmly thank all the study subjects. We thank all members of both branches of the laboratory of Human Genetics of Infectious Diseases for fruitful discussion. We are grateful to Alain Dessein (INSERM-UMR 906/Université de la Méditerranée, Marseilles), Marija Zeremski and Ray Peterson (Division of Gastroenterology and Hepatology, Weill-Cornell Medical College, New-York) for helping to collect the data.</p><p id="__p3"><strong><em>Funding:</em></strong> The SCCS is supported by grants from the Swiss National Science Foundation (3347C0-108782/1), the Swiss Federal Office for Education and Sciences (03.0599) and the European Commission (LSHM-CT-2004-503359; VIRGIL Network of Excellence on Antiviral Drug Resistance). The French cohort and E.P. are supported and sponsored by The National Agency for Research on AIDS and Viral Hepatitis (ANRS) (ANRS Study HC EP 26 Genoscan). E J received funding from The Rockefeller University Center for Clinical and Translational Science Grant Award number UL1RR0241443. C M R, A H T and I M J are supported in part by The Greenberg Medical Research Institute and the Starr Foundation. F N is supported by the Swiss National Science Foundation (314730-130498). P-Y B is supported by the Swiss National Foundation (32003B-127613), the Leenaards foundation and the Santos-Suarez foundation.</p></div></div><div id="GL" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="GLtitle">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">HCV</td><td>hepatitis C virus</td></tr><tr><td id="G2">GWA</td><td>genome-wide association</td></tr><tr><td id="G3">SNP</td><td>single nucleotide polymorphism</td></tr><tr><td id="G4">IDU</td><td>injecting drug use</td></tr><tr><td id="G5">CI</td><td>confidence interval</td></tr><tr><td id="G6">OR</td><td>odds-ratio</td></tr><tr><td id="G7">HR</td><td>hazard ratio</td></tr><tr><td id="G8">FPR</td><td>fibrosis progression rate</td></tr><tr><td id="G9">SMU</td><td>standardized Metabir units</td></tr><tr><td id="G10">LD</td><td>linkage disequilibrium</td></tr></tbody></table></div></div></div><div id="idm140464453586624" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140464453586624title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"> </p><p id="__p4" class="p p-first-last"><strong><em>Conflicts of interest:</em></strong> The authors disclose no conflicts.</p><p></p><div><p id="__p5" class="p p-first-last"><strong><em>Contributions:</em></strong> L. Abel and P.-Y.B., along with B.N., F.N. and S.P., conceived of the study, obtained financial support and were responsible for study design. E.P. and Z.K. performed all statistical analyses and were responsible of data interpretation, with contributions of L. Abel and P.-Y.B. J. Guergnon, S.B., A.B., E.J. and I.T. were responsible for sample processing and performed whole-genome and custom genotyping experiments. B.N., M.M., L.B., L. Argiro, P.A., B.M., D.S., J.-F.D., M.H.H., D.M., A.C., R.M., H.H., G.M., V.S., G.S., D.R.B., J. George, C.B., A.H.T., I.J., M.B., T.P., F.N. and S.P. coordinated and contributed subjects and database phenotype collections as lead investigators for their respective sample collections. E.P. and L. Abel drafted the initial manuscript, along with Z.K. and P.-Y.B. J. George, C.M.R., J.-L.C., I.J., T.P., F.N. and S.P. provided critical revision of the manuscript for important intellectual content. All authors critically reviewed the manuscript revisions and approved the final manuscript.</p></div><p class="fn sec" id="FN5"></p><p id="__p6" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140464454036256" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140464454036256title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">WHO. <span><span class="ref-journal">Hepatitis C. </span>2011 Jun;</span>  <a href="http://wwwwhoint/mediacentre/factsheets/fs164/en/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://wwwwhoint/mediacentre/factsheets/fs164/en/</a> Fact sheet N°164.</span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Missiha SB, Ostrowski M, Heathcote EJ. Disease progression in chronic hepatitis C: modifiable and nonmodifiable factors. <span><span class="ref-journal">Gastroenterology. </span>2008;<span class="ref-vol">134</span>:1699–714.</span> [<a href="/pubmed/18471548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Disease+progression+in+chronic+hepatitis+C:+modifiable+and+nonmodifiable+factors&amp;author=SB+Missiha&amp;author=M+Ostrowski&amp;author=EJ+Heathcote&amp;volume=134&amp;publication_year=2008&amp;pages=1699-714&amp;pmid=18471548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Bochud PY, Cai T, Overbeck K, et al.  Genotype 3 is associated with accelerated fibrosis progression in chronic hepatitis C. <span><span class="ref-journal">J Hepatol. </span>2009;<span class="ref-vol">51</span>:655–66.</span> [<a href="/pubmed/19665246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hepatol&amp;title=Genotype+3+is+associated+with+accelerated+fibrosis+progression+in+chronic+hepatitis+C&amp;author=PY+Bochud&amp;author=T+Cai&amp;author=K+Overbeck&amp;volume=51&amp;publication_year=2009&amp;pages=655-66&amp;pmid=19665246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Probst ADT, Bochud M, Egger M, Negro F, Bochud PY. Role of Hepatitis C virus genotype 3 in liver fibrosis progression – a systematic review and meta-analysis. <span><span class="ref-journal">Journal of Viral Hepatitis. </span>2011;<span class="ref-vol">18</span>:745–59.</span> [<a href="/pubmed/21992794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Viral+Hepatitis&amp;title=Role+of+Hepatitis+C+virus+genotype+3+in+liver+fibrosis+progression+–+a+systematic+review+and+meta-analysis&amp;author=ADT+Probst&amp;author=M+Bochud&amp;author=M+Egger&amp;author=F+Negro&amp;author=PY+Bochud&amp;volume=18&amp;publication_year=2011&amp;pages=745-59&amp;pmid=21992794&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Osterreicher CH, Stickel F, Brenner DA. Genomics of liver fibrosis and cirrhosis. <span><span class="ref-journal">Semin Liver Dis. </span>2007;<span class="ref-vol">27</span>:28–43.</span> [<a href="/pubmed/17295175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin+Liver+Dis&amp;title=Genomics+of+liver+fibrosis+and+cirrhosis&amp;author=CH+Osterreicher&amp;author=F+Stickel&amp;author=DA+Brenner&amp;volume=27&amp;publication_year=2007&amp;pages=28-43&amp;pmid=17295175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Bataller R, North KE, Brenner DA. Genetic polymorphisms and the progression of liver fibrosis: a critical appraisal. <span><span class="ref-journal">Hepatology. </span>2003;<span class="ref-vol">37</span>:493–503.</span> [<a href="/pubmed/12601343" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Genetic+polymorphisms+and+the+progression+of+liver+fibrosis:+a+critical+appraisal&amp;author=R+Bataller&amp;author=KE+North&amp;author=DA+Brenner&amp;volume=37&amp;publication_year=2003&amp;pages=493-503&amp;pmid=12601343&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Huang H, Shiffman ML, Cheung RC, et al.  Identification of two gene variants associated with risk of advanced fibrosis in patients with chronic hepatitis C. <span><span class="ref-journal">Gastroenterology. </span>2006;<span class="ref-vol">130</span>:1679–87.</span> [<a href="/pubmed/16697732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Identification+of+two+gene+variants+associated+with+risk+of+advanced+fibrosis+in+patients+with+chronic+hepatitis+C&amp;author=H+Huang&amp;author=ML+Shiffman&amp;author=RC+Cheung&amp;volume=130&amp;publication_year=2006&amp;pages=1679-87&amp;pmid=16697732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Huang H, Shiffman ML, Friedman S, et al.  A 7 gene signature identifies the risk of developing cirrhosis in patients with chronic hepatitis C. <span><span class="ref-journal">Hepatology. </span>2007;<span class="ref-vol">46</span>:297–306.</span> [<a href="/pubmed/17461418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=A+7+gene+signature+identifies+the+risk+of+developing+cirrhosis+in+patients+with+chronic+hepatitis+C&amp;author=H+Huang&amp;author=ML+Shiffman&amp;author=S+Friedman&amp;volume=46&amp;publication_year=2007&amp;pages=297-306&amp;pmid=17461418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Nalpas B, Lavialle-Meziani R, Plancoulaine S, et al.  Interferon gamma receptor 2 gene variants are associated with liver fibrosis in patients with chronic hepatitis C infection. <span><span class="ref-journal">Gut. </span>2010;<span class="ref-vol">59</span>:1120–6.</span> [<a href="/pubmed/20587546" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Interferon+gamma+receptor+2+gene+variants+are+associated+with+liver+fibrosis+in+patients+with+chronic+hepatitis+C+infection&amp;author=B+Nalpas&amp;author=R+Lavialle-Meziani&amp;author=S+Plancoulaine&amp;volume=59&amp;publication_year=2010&amp;pages=1120-6&amp;pmid=20587546&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Alcais A, Abel L, Casanova JL. Human genetics of infectious diseases: between proof of principle and paradigm. <span><span class="ref-journal">J Clin Invest. </span>2009;<span class="ref-vol">119</span>:2506–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2735901/">PMC free article</a>]</span> [<a href="/pubmed/19729848" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Human+genetics+of+infectious+diseases:+between+proof+of+principle+and+paradigm&amp;author=A+Alcais&amp;author=L+Abel&amp;author=JL+Casanova&amp;volume=119&amp;publication_year=2009&amp;pages=2506-14&amp;pmid=19729848&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Balagopal A, Thomas DL, Thio CL. IL28B and the control of hepatitis C virus infection. <span><span class="ref-journal">Gastroenterology. </span>2010;<span class="ref-vol">139</span>:1865–76.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3072961/">PMC free article</a>]</span> [<a href="/pubmed/20950615" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=IL28B+and+the+control+of+hepatitis+C+virus+infection&amp;author=A+Balagopal&amp;author=DL+Thomas&amp;author=CL+Thio&amp;volume=139&amp;publication_year=2010&amp;pages=1865-76&amp;pmid=20950615&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Bochud PY, Bibert S, Kutalik Z, et al.  IL28B alleles associated with poor hepatitis C virus (HCV) clearance protect against inflammation and fibrosis in patients infected with non-1 HCV genotypes. <span><span class="ref-journal">Hepatology. </span>2011</span> [<a href="/pubmed/22180014" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=IL28B+alleles+associated+with+poor+hepatitis+C+virus+(HCV)+clearance+protect+against+inflammation+and+fibrosis+in+patients+infected+with+non-1+HCV+genotypes&amp;author=PY+Bochud&amp;author=S+Bibert&amp;author=Z+Kutalik&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Chambers JC, Zhang W, Sehmi J, et al.  Genome-wide association study identifies loci influencing concentrations of liver enzymes in plasma. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:1131–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3482372/">PMC free article</a>]</span> [<a href="/pubmed/22001757" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+loci+influencing+concentrations+of+liver+enzymes+in+plasma&amp;author=JC+Chambers&amp;author=W+Zhang&amp;author=J+Sehmi&amp;volume=43&amp;publication_year=2011&amp;pages=1131-8&amp;pmid=22001757&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Romeo S, Kozlitina J, Xing C, et al.  Genetic variation in PNPLA3 confers susceptibility to nonalcoholic fatty liver disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1461–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2597056/">PMC free article</a>]</span> [<a href="/pubmed/18820647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+PNPLA3+confers+susceptibility+to+nonalcoholic+fatty+liver+disease&amp;author=S+Romeo&amp;author=J+Kozlitina&amp;author=C+Xing&amp;volume=40&amp;publication_year=2008&amp;pages=1461-5&amp;pmid=18820647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Yuan X, Waterworth D, Perry JR, et al.  Population-based genome-wide association studies reveal six loci influencing plasma levels of liver enzymes. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:520–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2561937/">PMC free article</a>]</span> [<a href="/pubmed/18940312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Population-based+genome-wide+association+studies+reveal+six+loci+influencing+plasma+levels+of+liver+enzymes&amp;author=X+Yuan&amp;author=D+Waterworth&amp;author=JR+Perry&amp;volume=83&amp;publication_year=2008&amp;pages=520-8&amp;pmid=18940312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Trepo E, Pradat P, Potthoff A, et al.  Impact of patatin-like phospholipase-3 (rs738409 C&gt;G) polymorphism on fibrosis progression and steatosis in chronic hepatitis C. <span><span class="ref-journal">Hepatology. </span>2011;<span class="ref-vol">54</span>:60–9.</span> [<a href="/pubmed/21488075" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Impact+of+patatin-like+phospholipase-3+(rs738409+C>G)+polymorphism+on+fibrosis+progression+and+steatosis+in+chronic+hepatitis+C&amp;author=E+Trepo&amp;author=P+Pradat&amp;author=A+Potthoff&amp;volume=54&amp;publication_year=2011&amp;pages=60-9&amp;pmid=21488075&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Valenti L, Rumi M, Galmozzi E, et al.  Patatin-like phospholipase domain-containing 3 I148M polymorphism, steatosis, and liver damage in chronic hepatitis C. <span><span class="ref-journal">Hepatology. </span>2011;<span class="ref-vol">53</span>:791–9.</span> [<a href="/pubmed/21319195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Patatin-like+phospholipase+domain-containing+3+I148M+polymorphism,+steatosis,+and+liver+damage+in+chronic+hepatitis+C&amp;author=L+Valenti&amp;author=M+Rumi&amp;author=E+Galmozzi&amp;volume=53&amp;publication_year=2011&amp;pages=791-9&amp;pmid=21319195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Prasad L, Spicher VM, Zwahlen M, Rickenbach M, Helbling B, Negro F. Cohort Profile: the Swiss Hepatitis C Cohort Study (SCCS) <span><span class="ref-journal">Int J Epidemiol. </span>2007;<span class="ref-vol">36</span>:731–7.</span> [<a href="/pubmed/17693458" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Cohort+Profile:+the+Swiss+Hepatitis+C+Cohort+Study+(SCCS)&amp;author=L+Prasad&amp;author=VM+Spicher&amp;author=M+Zwahlen&amp;author=M+Rickenbach&amp;author=B+Helbling&amp;volume=36&amp;publication_year=2007&amp;pages=731-7&amp;pmid=17693458&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Suppiah V, Moldovan M, Ahlenstiel G, et al.  IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1100–4.</span> [<a href="/pubmed/19749758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=IL28B+is+associated+with+response+to+chronic+hepatitis+C+interferon-alpha+and+ribavirin+therapy&amp;author=V+Suppiah&amp;author=M+Moldovan&amp;author=G+Ahlenstiel&amp;volume=41&amp;publication_year=2009&amp;pages=1100-4&amp;pmid=19749758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Suppiah V, Gaudieri S, Armstrong NJ, et al.  IL28B, HLA-C, and KIR Variants Additively Predict Response to Therapy in Chronic Hepatitis C Virus Infection in a European Cohort: A Cross-Sectional Study. <span><span class="ref-journal">PLoS Med. </span>2011;<span class="ref-vol">8</span>:e1001092.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3172251/">PMC free article</a>]</span> [<a href="/pubmed/21931540" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med&amp;title=IL28B,+HLA-C,+and+KIR+Variants+Additively+Predict+Response+to+Therapy+in+Chronic+Hepatitis+C+Virus+Infection+in+a+European+Cohort:+A+Cross-Sectional+Study&amp;author=V+Suppiah&amp;author=S+Gaudieri&amp;author=NJ+Armstrong&amp;volume=8&amp;publication_year=2011&amp;pages=e1001092&amp;pmid=21931540&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Bedossa P, Poynard T. An algorithm for the grading of activity in chronic hepatitis C. The METAVIR Cooperative Study Group. <span><span class="ref-journal">Hepatology. </span>1996;<span class="ref-vol">24</span>:289–93.</span> [<a href="/pubmed/8690394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=An+algorithm+for+the+grading+of+activity+in+chronic+hepatitis+C.+The+METAVIR+Cooperative+Study+Group&amp;author=P+Bedossa&amp;author=T+Poynard&amp;volume=24&amp;publication_year=1996&amp;pages=289-93&amp;pmid=8690394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Group TFMCS. Intraobserver and interobserver variations in liver biopsy interpretation in patients with chronic hepatitis C. <span><span class="ref-journal">Hepatology. </span>1994;<span class="ref-vol">20</span>:15–20.</span> [<a href="/pubmed/8020885" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Intraobserver+and+interobserver+variations+in+liver+biopsy+interpretation+in+patients+with+chronic+hepatitis+C&amp;volume=20&amp;publication_year=1994&amp;pages=15-20&amp;pmid=8020885&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Poynard T, Bedossa P, Opolon P. Natural history of liver fibrosis progression in patients with chronic hepatitis C. The OBSVIRC, METAVIR, CLINIVIR, and DOSVIRC groups. <span><span class="ref-journal">Lancet. </span>1997;<span class="ref-vol">349</span>:825–32.</span> [<a href="/pubmed/9121257" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Natural+history+of+liver+fibrosis+progression+in+patients+with+chronic+hepatitis+C.+The+OBSVIRC,+METAVIR,+CLINIVIR,+and+DOSVIRC+groups&amp;author=T+Poynard&amp;author=P+Bedossa&amp;author=P+Opolon&amp;volume=349&amp;publication_year=1997&amp;pages=825-32&amp;pmid=9121257&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Sobesky R, Mathurin P, Charlotte F, et al.  Modeling the impact of interferon alfa treatment on liver fibrosis progression in chronic hepatitis C: a dynamic view. The Multivirc Group. <span><span class="ref-journal">Gastroenterology. </span>1999;<span class="ref-vol">116</span>:378–86.</span> [<a href="/pubmed/9922319" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Modeling+the+impact+of+interferon+alfa+treatment+on+liver+fibrosis+progression+in+chronic+hepatitis+C:+a+dynamic+view.+The+Multivirc+Group&amp;author=R+Sobesky&amp;author=P+Mathurin&amp;author=F+Charlotte&amp;volume=116&amp;publication_year=1999&amp;pages=378-86&amp;pmid=9922319&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Duan H, Wang Y, Aviram M, et al.  SAG, a novel zinc RING finger protein that protects cells from apoptosis induced by redox agents. <span><span class="ref-journal">Mol Cell Biol. </span>1999;<span class="ref-vol">19</span>:3145–55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC84108/">PMC free article</a>]</span> [<a href="/pubmed/10082581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=SAG,+a+novel+zinc+RING+finger+protein+that+protects+cells+from+apoptosis+induced+by+redox+agents&amp;author=H+Duan&amp;author=Y+Wang&amp;author=M+Aviram&amp;volume=19&amp;publication_year=1999&amp;pages=3145-55&amp;pmid=10082581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Rothlin CV, Ghosh S, Zuniga EI, Oldstone MB, Lemke G. TAM receptors are pleiotropic inhibitors of the innate immune response. <span><span class="ref-journal">Cell. </span>2007;<span class="ref-vol">131</span>:1124–36.</span> [<a href="/pubmed/18083102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=TAM+receptors+are+pleiotropic+inhibitors+of+the+innate+immune+response&amp;author=CV+Rothlin&amp;author=S+Ghosh&amp;author=EI+Zuniga&amp;author=MB+Oldstone&amp;author=G+Lemke&amp;volume=131&amp;publication_year=2007&amp;pages=1124-36&amp;pmid=18083102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Chalasani N, Guo X, Loomba R, et al.  Genome-wide association study identifies variants associated with histologic features of nonalcoholic Fatty liver disease. <span><span class="ref-journal">Gastroenterology. </span>2010;<span class="ref-vol">139</span>:1567–76. 76 e1–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2967576/">PMC free article</a>]</span> [<a href="/pubmed/20708005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Genome-wide+association+study+identifies+variants+associated+with+histologic+features+of+nonalcoholic+Fatty+liver+disease&amp;author=N+Chalasani&amp;author=X+Guo&amp;author=R+Loomba&amp;volume=139&amp;publication_year=2010&amp;pages=1567-76&amp;pmid=20708005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Kumar V, Kato N, Urabe Y, et al.  Genome-wide association study identifies a susceptibility locus for HCV-induced hepatocellular carcinoma. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:455–8.</span> [<a href="/pubmed/21499248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+susceptibility+locus+for+HCV-induced+hepatocellular+carcinoma&amp;author=V+Kumar&amp;author=N+Kato&amp;author=Y+Urabe&amp;volume=43&amp;publication_year=2011&amp;pages=455-8&amp;pmid=21499248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Miki D, Ochi H, Hayes CN, et al.  Variation in the DEPDC5 locus is associated with progression to hepatocellular carcinoma in chronic hepatitis C virus carriers. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:797–800.</span> [<a href="/pubmed/21725309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variation+in+the+DEPDC5+locus+is+associated+with+progression+to+hepatocellular+carcinoma+in+chronic+hepatitis+C+virus+carriers&amp;author=D+Miki&amp;author=H+Ochi&amp;author=CN+Hayes&amp;volume=43&amp;publication_year=2011&amp;pages=797-800&amp;pmid=21725309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Cai T, Dufour JF, Muellhaupt B, et al.  Viral genotype-specific role of PNPLA3, PPARG, MTTP, and IL28B in hepatitis C virus-associated steatosis. <span><span class="ref-journal">J Hepatol. </span>2011;<span class="ref-vol">55</span>:529–35.</span> [<a href="/pubmed/21236304" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hepatol&amp;title=Viral+genotype-specific+role+of+PNPLA3,+PPARG,+MTTP,+and+IL28B+in+hepatitis+C+virus-associated+steatosis&amp;author=T+Cai&amp;author=JF+Dufour&amp;author=B+Muellhaupt&amp;volume=55&amp;publication_year=2011&amp;pages=529-35&amp;pmid=21236304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Banerjee P, Kleyn PW, Knowles JA, et al.  TULP1 mutation in two extended Dominican kindreds with autosomal recessive retinitis pigmentosa. <span><span class="ref-journal">Nat Genet. </span>1998;<span class="ref-vol">18</span>:177–9.</span> [<a href="/pubmed/9462751" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=TULP1+mutation+in+two+extended+Dominican+kindreds+with+autosomal+recessive+retinitis+pigmentosa&amp;author=P+Banerjee&amp;author=PW+Kleyn&amp;author=JA+Knowles&amp;volume=18&amp;publication_year=1998&amp;pages=177-9&amp;pmid=9462751&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Gal A, Li Y, Thompson DA, et al.  Mutations in MERTK, the human orthologue of the RCS rat retinal dystrophy gene, cause retinitis pigmentosa. <span><span class="ref-journal">Nat Genet. </span>2000;<span class="ref-vol">26</span>:270–1.</span> [<a href="/pubmed/11062461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mutations+in+MERTK,+the+human+orthologue+of+the+RCS+rat+retinal+dystrophy+gene,+cause+retinitis+pigmentosa&amp;author=A+Gal&amp;author=Y+Li&amp;author=DA+Thompson&amp;volume=26&amp;publication_year=2000&amp;pages=270-1&amp;pmid=11062461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Hagstrom SA, North MA, Nishina PL, Berson EL, Dryja TP. Recessive mutations in the gene encoding the tubby-like protein TULP1 in patients with retinitis pigmentosa. <span><span class="ref-journal">Nat Genet. </span>1998;<span class="ref-vol">18</span>:174–6.</span> [<a href="/pubmed/9462750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Recessive+mutations+in+the+gene+encoding+the+tubby-like+protein+TULP1+in+patients+with+retinitis+pigmentosa&amp;author=SA+Hagstrom&amp;author=MA+North&amp;author=PL+Nishina&amp;author=EL+Berson&amp;author=TP+Dryja&amp;volume=18&amp;publication_year=1998&amp;pages=174-6&amp;pmid=9462750&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Su AI, Wiltshire T, Batalov S, et al.  A gene atlas of the mouse and human protein-encoding transcriptomes. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2004;<span class="ref-vol">101</span>:6062–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC395923/">PMC free article</a>]</span> [<a href="/pubmed/15075390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes&amp;author=AI+Su&amp;author=T+Wiltshire&amp;author=S+Batalov&amp;volume=101&amp;publication_year=2004&amp;pages=6062-7&amp;pmid=15075390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Caberoy NB, Zhou Y, Li W. Tubby and tubby-like protein 1 are new MerTK ligands for phagocytosis. <span><span class="ref-journal">EMBO J. </span>2010;<span class="ref-vol">29</span>:3898–910.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3020645/">PMC free article</a>]</span> [<a href="/pubmed/20978472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Tubby+and+tubby-like+protein+1+are+new+MerTK+ligands+for+phagocytosis&amp;author=NB+Caberoy&amp;author=Y+Zhou&amp;author=W+Li&amp;volume=29&amp;publication_year=2010&amp;pages=3898-910&amp;pmid=20978472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Scott RS, McMahon EJ, Pop SM, et al.  Phagocytosis and clearance of apoptotic cells is mediated by MER. <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">411</span>:207–11.</span> [<a href="/pubmed/11346799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Phagocytosis+and+clearance+of+apoptotic+cells+is+mediated+by+MER&amp;author=RS+Scott&amp;author=EJ+McMahon&amp;author=SM+Pop&amp;volume=411&amp;publication_year=2001&amp;pages=207-11&amp;pmid=11346799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Canbay A, Taimr P, Torok N, Higuchi H, Friedman S, Gores GJ. Apoptotic body engulfment by a human stellate cell line is profibrogenic. <span><span class="ref-journal">Lab Invest. </span>2003;<span class="ref-vol">83</span>:655–63.</span> [<a href="/pubmed/12746475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lab+Invest&amp;title=Apoptotic+body+engulfment+by+a+human+stellate+cell+line+is+profibrogenic&amp;author=A+Canbay&amp;author=P+Taimr&amp;author=N+Torok&amp;author=H+Higuchi&amp;author=S+Friedman&amp;volume=83&amp;publication_year=2003&amp;pages=655-63&amp;pmid=12746475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Lauber K, Bohn E, Krober SM, et al.  Apoptotic cells induce migration of phagocytes via caspase-3-mediated release of a lipid attraction signal. <span><span class="ref-journal">Cell. </span>2003;<span class="ref-vol">113</span>:717–30.</span> [<a href="/pubmed/12809603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Apoptotic+cells+induce+migration+of+phagocytes+via+caspase-3-mediated+release+of+a+lipid+attraction+signal&amp;author=K+Lauber&amp;author=E+Bohn&amp;author=SM+Krober&amp;volume=113&amp;publication_year=2003&amp;pages=717-30&amp;pmid=12809603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Bataller R, Schwabe RF, Choi YH, et al.  NADPH oxidase signal transduces angiotensin II in hepatic stellate cells and is critical in hepatic fibrosis. <span><span class="ref-journal">J Clin Invest. </span>2003;<span class="ref-vol">112</span>:1383–94.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC228420/">PMC free article</a>]</span> [<a href="/pubmed/14597764" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=NADPH+oxidase+signal+transduces+angiotensin+II+in+hepatic+stellate+cells+and+is+critical+in+hepatic+fibrosis&amp;author=R+Bataller&amp;author=RF+Schwabe&amp;author=YH+Choi&amp;volume=112&amp;publication_year=2003&amp;pages=1383-94&amp;pmid=14597764&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Canbay A, Friedman S, Gores GJ. Apoptosis: the nexus of liver injury and fibrosis. <span><span class="ref-journal">Hepatology. </span>2004;<span class="ref-vol">39</span>:273–8.</span> [<a href="/pubmed/14767974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Apoptosis:+the+nexus+of+liver+injury+and+fibrosis&amp;author=A+Canbay&amp;author=S+Friedman&amp;author=GJ+Gores&amp;volume=39&amp;publication_year=2004&amp;pages=273-8&amp;pmid=14767974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Kim SY, Lee JH, Yang ES, Kil IS, Park JW. Human sensitive to apoptosis gene protein inhibits peroxynitrite-induced DNA damage. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2003;<span class="ref-vol">301</span>:671–4.</span> [<a href="/pubmed/12565832" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=Human+sensitive+to+apoptosis+gene+protein+inhibits+peroxynitrite-induced+DNA+damage&amp;author=SY+Kim&amp;author=JH+Lee&amp;author=ES+Yang&amp;author=IS+Kil&amp;author=JW+Park&amp;volume=301&amp;publication_year=2003&amp;pages=671-4&amp;pmid=12565832&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Machida K, McNamara G, Cheng KT, et al.  Hepatitis C virus inhibits DNA damage repair through reactive oxygen and nitrogen species and by interfering with the ATM-NBS1/Mre11/Rad50 DNA repair pathway in monocytes and hepatocytes. <span><span class="ref-journal">J Immunol. </span>2010;<span class="ref-vol">185</span>:6985–98.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3101474/">PMC free article</a>]</span> [<a href="/pubmed/20974981" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Hepatitis+C+virus+inhibits+DNA+damage+repair+through+reactive+oxygen+and+nitrogen+species+and+by+interfering+with+the+ATM-NBS1/Mre11/Rad50+DNA+repair+pathway+in+monocytes+and+hepatocytes&amp;author=K+Machida&amp;author=G+McNamara&amp;author=KT+Cheng&amp;volume=185&amp;publication_year=2010&amp;pages=6985-98&amp;pmid=20974981&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Wang T, Weinman SA. Causes and consequences of mitochondrial reactive oxygen species generation in hepatitis C. <span><span class="ref-journal">J Gastroenterol Hepatol. </span>2006;<span class="ref-vol">21 </span>(Suppl 3):S34–7.</span> [<a href="/pubmed/16958669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Gastroenterol+Hepatol&amp;title=Causes+and+consequences+of+mitochondrial+reactive+oxygen+species+generation+in+hepatitis+C&amp;author=T+Wang&amp;author=SA+Weinman&amp;volume=21+&amp;issue=Suppl+3&amp;publication_year=2006&amp;pages=S34-7&amp;pmid=16958669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Guicciardi ME, Gores GJ. Apoptosis as a mechanism for liver disease progression. <span><span class="ref-journal">Semin Liver Dis. </span>2010;<span class="ref-vol">30</span>:402–10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3071245/">PMC free article</a>]</span> [<a href="/pubmed/20960379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin+Liver+Dis&amp;title=Apoptosis+as+a+mechanism+for+liver+disease+progression&amp;author=ME+Guicciardi&amp;author=GJ+Gores&amp;volume=30&amp;publication_year=2010&amp;pages=402-10&amp;pmid=20960379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Conover WJ.  <span class="ref-journal">Practical Nonparametric Statistics.</span> 3. Wiley; New York, NY USA: 1999.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Practical+Nonparametric+Statistics&amp;author=WJ+Conover&amp;publication_year=1999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3756935&amp;issue-id=226839&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140464451798512title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Patients &amp; Methods</a></li><li><a href="#S11title">Results</a></li><li><a href="#S14title">Discussion</a></li><li><a href="#S15title">Supplementary Material</a></li><li><a href="#S16title">Acknowledgments</a></li><li><a href="#GLtitle">Abbreviations</a></li><li><a href="#idm140464453586624title">Footnotes</a></li><li><a href="#idm140464454036256title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3756935/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3756935/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3756935/pdf/nihms477303.pdf">PDF (903K)</a></li> | <li><a href="#" data-citationid="PMC3756935" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3756935%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3756935%2F&amp;text=Genome-Wide%20Association%20Study%20Identifies%20Variants%20Associated%20with%20Progression%20of%20Liver%20Fibrosis%20from%20HCV%20Infection" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3756935%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22841784" data-aiid="3756935" data-aid="3756935" data-iid="226839" data-domainid="319" data-domain="nihpa" data-accid="PMC3756935" data-md5="2c91ff276b178759b6dc679e9a07e41b"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:19:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BC563DD5421A100000000072404CA&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3756935%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3756935/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>