<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ndt"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3685304/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide significant locus of beta-trace protein, a novel kidney function biomarker, identified in European and African Americans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3685304/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nephrology Dialysis Transplantation"><meta name="citation_title" content="Genome-wide significant locus of beta-trace protein, a novel kidney function biomarker, identified in European and African Americans"><meta name="citation_authors" content="Adrienne Tin, Brad C. Astor, Eric Boerwinkle, Ron C. Hoogeveen, Josef Coresh, W. H. Linda Kao"><meta name="citation_date" content="June 2013"><meta name="citation_issue" content="6"><meta name="citation_volume" content="28"><meta name="citation_firstpage" content="1497"><meta name="citation_doi" content="10.1093/ndt/gfs591"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3685304/?report=abstract"><meta name="citation_pmid" content="23328707"><meta name="DC.Title" content="Genome-wide significant locus of beta-trace protein, a novel kidney function biomarker, identified in European and African Americans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Adrienne Tin"><meta name="DC.Contributor" content="Brad C. Astor"><meta name="DC.Contributor" content="Eric Boerwinkle"><meta name="DC.Contributor" content="Ron C. Hoogeveen"><meta name="DC.Contributor" content="Josef Coresh"><meta name="DC.Contributor" content="W. H. Linda Kao"><meta name="DC.Date" content="2013 Jun"><meta name="DC.Identifier" content="10.1093/ndt/gfs591"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide significant locus of beta-trace protein, a novel kidney function biomarker, identified in European and African Americans"><meta property="og:type" content="article"><meta property="og:description" content="Beta-trace protein (BTP), measured in serum or plasma, has potential as a novel biomarker for kidney function. Little is known about the genes influencing BTP levels.We conducted a genome-wide association study of log-transformed plasma BTP levels in ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3685304/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ndt.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3685304/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3685304/pdf/gfs591.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D3E7BDD54E9C10000000006C4049A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3685304%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Nephrol Dial Transplant%22[journal]" class="navlink">Nephrol Dial Transplant</a></li><li class="accid">PMC3685304</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ndt.png" alt="Logo of ndt" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://ndt.oxfordjournals.org/" target="pmc_ext" ref="reftype=publisher&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nephrol Dial Transplant</a></span>. 2013 Jun; 28(6): 1497–1504. </span></div><div><span class="fm-vol-iss-date">Published online 2013 Jan 16. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fndt%2Fgfs591" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/ndt/gfs591</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3685304</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23328707">23328707</a></div></div></div></div><h1 class="content-title">Genome-wide significant locus of beta-trace protein, a novel kidney function biomarker, identified in European and African Americans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Tin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707" class="affpopup" co-rid="_co_idm140633013214032" co-class="co-affbox">Adrienne Tin</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Astor%20BC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707" class="affpopup" co-rid="_co_idm140633008557728" co-class="co-affbox">Brad C. Astor</a>,<sup>2</sup> <a href="/pubmed/?term=Boerwinkle%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707" class="affpopup" co-rid="_co_idm140633015493440" co-class="co-affbox">Eric Boerwinkle</a>,<sup>3</sup> <a href="/pubmed/?term=Hoogeveen%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707" class="affpopup" co-rid="_co_idm140633009364320" co-class="co-affbox">Ron C. Hoogeveen</a>,<sup>4</sup> <a href="/pubmed/?term=Coresh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707" class="affpopup" co-rid="_co_idm140633005089088" co-class="co-affbox">Josef Coresh</a>,<sup>1</sup> and  <a href="/pubmed/?term=Kao%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707" class="affpopup" co-rid="_co_idm140633013958608" co-class="co-affbox">W. H. Linda Kao</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140633013214032"><h3 class="no_margin">Adrienne Tin</h3><p><sup>1</sup>Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Tin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707">Adrienne Tin</a></div></div><div id="_co_idm140633008557728"><h3 class="no_margin">Brad C. Astor</h3><p><sup>2</sup>University of Wisconsin School of Medicine and Public Health, Madison, WI, USA</p><div>Find articles by <a href="/pubmed/?term=Astor%20BC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707">Brad C. Astor</a></div></div><div id="_co_idm140633015493440"><h3 class="no_margin">Eric Boerwinkle</h3><p><sup>3</sup>Human Genetics Center, University of Texas School of Public Health, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Boerwinkle%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707">Eric Boerwinkle</a></div></div><div id="_co_idm140633009364320"><h3 class="no_margin">Ron C. Hoogeveen</h3><p><sup>4</sup>Baylor College of Medicine, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Hoogeveen%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707">Ron C. Hoogeveen</a></div></div><div id="_co_idm140633005089088"><h3 class="no_margin">Josef Coresh</h3><p><sup>1</sup>Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Coresh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707">Josef Coresh</a></div></div><div id="_co_idm140633013958608"><h3 class="no_margin">W. H. Linda Kao</h3><p><sup>1</sup>Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Kao%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23328707">W. H. Linda Kao</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140633006211920_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140633006211920_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140633006211920_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140633006211920_ai" style="display:none"><div class="fm-affl" id="gfs591_af1" lang="en"><sup>1</sup>Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA</div><div class="fm-affl" id="gfs591_af2" lang="en"><sup>2</sup>University of Wisconsin School of Medicine and Public Health, Madison, WI, USA</div><div class="fm-affl" id="gfs591_af3" lang="en"><sup>3</sup>Human Genetics Center, University of Texas School of Public Health, Houston, TX, USA</div><div class="fm-affl" id="gfs591_af4" lang="en"><sup>4</sup>Baylor College of Medicine, Houston, TX, USA</div><div id="gfs591_cor1"><sup>*</sup><em>Correspondence and offprint requests to:</em> Adrienne Tin; E-mail: <a href="mailto:dev@null" data-email="ude.hpshj@nita" class="oemail">ude.hpshj@nita</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140633006211920_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Aug 30; Accepted 2012 Nov 14.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140633006211920_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2013. Published by Oxford University Press on behalf of ERA-EDTA. All rights reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3685304/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3685304/bin/supp_28_6_1497__index.html" data-ga-action="click_feat_suppl">supp_28_6_1497__index.html</a><span style="color:gray"> (906 bytes)</span></div><div class="small guid">GUID:&nbsp;45BE3D50-EC39-4136-9510-160730F32E74</div>
<div><a href="/pmc/articles/PMC3685304/bin/supp_gfs591_gfs591supp.doc" data-ga-action="click_feat_suppl">supp_gfs591_gfs591supp.doc</a><span style="color:gray"> (833K)</span></div><div class="small guid">GUID:&nbsp;C46EA189-AFEB-4B0C-BB39-11B957C6CCD5</div>
</div></dd></dl></div><div id="idm140633004018432" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140633004018432title">Abstract</h2><!--article-meta--><div><div id="__sec1" class="sec sec-first"><h3 id="__sec1title">Background</h3><p id="__p1" class="p p-first-last">Beta-trace protein (BTP), measured in serum or plasma, has potential as a novel biomarker for kidney function. Little is known about the genes influencing BTP levels.</p></div><div id="__sec2" class="sec"><h3 id="__sec2title">Methods</h3><p id="__p2" class="p p-first-last">We conducted a genome-wide association study of log-transformed plasma BTP levels in 6720 European Americans (EAs) and replicated the significant associations in 1734 African Americans (AAs) from the Atherosclerosis Risk in Communities (ARIC) study.</p></div><div id="__sec3" class="sec"><h3 id="__sec3title">Results</h3><p id="__p3" class="p p-first-last">We identified a genome-wide significant locus in EA upstream of Prostaglandin D2 synthase (<em>PTGDS</em>), the gene encoding BTP. Each copy of the A allele at rs57024841 was associated with 5% higher BTP levels (P = 1.2 × 10<sup>−23</sup>). The association at <em>PTGDS</em> was confirmed in AAs (6% higher BTP for each A allele at rs57024841, P = 1.9 × 10<sup>−7</sup>). The index single nucleotide polymorphisms (SNPs) in EAs and AAs explained ∼1.1% of the log(BTP) variance within each population and explained over 30% of the difference in log(BTP) levels between EAs and AAs. The index SNPs at the <em>PTGDS</em> locus in the two populations were not associated with the estimated glomerular filtration rate (eGFR) or the urine albumin creatinine ratio (P &gt; 0.05). We further tested for the associations of BTP with 16 known loci of the eGFR in EA, and BTP was associated with 3 of 16 tested.</p></div><div id="__sec4" class="sec sec-last"><h3 id="__sec4title">Conclusions</h3><p id="__p4" class="p p-first-last">The identification of a novel BTP-specific (non-renal related) locus and the confirmation of several genetic loci of the eGFR with BTP extend our understanding of the metabolism of BTP and inform its use as a kidney filtration biomarker.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">chronic kidney disease, genome-wide association study, glomerular filtration rate, GWAS, kidney function biomarker</span></div></div><div id="gfs591_s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="gfs591_s1title">INTRODUCTION</h2><p id="__p5" class="p p-first">The glomerular filtration rate (GFR) is the most widely accepted index of kidney function. The direct measure of the GFR using an exogenous molecule is an invasive procedure and often impractical in both clinical and research settings [<a href="#GFS591C1" rid="GFS591C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887375">1</a>]. Therefore, it is often estimated using equations based on endogenous biomarker levels, such as serum creatinine (SCr). Discovery and characterization of novel filtration markers, such as beta-trace protein (BTP) and cystatin C, are important for improving the precision and reducing bias in GFR estimation due to non-GFR factors influencing SCr [<a href="#GFS591C2" rid="GFS591C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887356">2</a>]. However, novel filtration markers will have their own non-GFR factors. Advances in human genetics are enabling an unbiased characterization of common genetic variation influencing any novel filtration biomarker.</p><p id="__p6" class="p p-last">BTP, measured in serum or plasma, has been found to be more sensitive than SCr in detecting a modest decline in the GFR [<a href="#GFS591C3" rid="GFS591C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887349">3</a>–<a href="#GFS591C6" rid="GFS591C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887363">6</a>]. BTP is a low-molecular weight protein (23–29 kDa) that is freely filtered by the kidney. It has been shown to be associated with risk of kidney failure and mortality in participants with chronic kidney disease (CKD) in the Modification of Diet in Renal Disease and African American Study of Kidney Disease and Hypertension studies [<a href="#GFS591C7" rid="GFS591C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887366">7</a>] and with risk of end-stage renal disease (ESRD) in African Americans (AAs) having hypertensive CKD after controlling for measured GFR [<a href="#GFS591C8" rid="GFS591C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887379">8</a>]. Recently, the Atherosclerosis Risk in Communities (ARIC) population-based cohort also found BTP to have stronger associations with ESRD, cardiovascular disease and mortality than SCr estimates of the GFR [<a href="#GFS591C9" rid="GFS591C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_350805289">9</a>]. Little is known about with non-kidney-related factors influencing BTP levels. The primary goal of this study was to identify genetic loci of BTP. As a biomarker of kidney function, BTP may be associated with kidney function loci or loci specific to the regulation of BTP. Our secondary goal was to examine the association between BTP and known eGFR loci. Investigation into the genetic associations of BTP can enhance our understanding of the biological mechanisms underlying BTP metabolism and inform its potential use as a kidney function biomarker. We conducted a genome-wide association study (GWAS) of plasma BTP in a large population-based cohort, tested replication of the genome-wide significant results, determined whether this locus was associated with kidney function and finally tested for the associations of BTP with 16 previously identified GFR loci among individuals of European ancestry [<a href="#GFS591C10" rid="GFS591C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887365">10</a>].</p></div><div id="gfs591_s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="gfs591_s2title">METHODS</h2><div id="gfs591_s2a" class="sec sec-first"><h3 id="gfs591_s2atitle">Study population and phenotype and covariate measures</h3><p id="__p7" class="p p-first">The ARIC study is a prospective observational cohort study of middle-age adults (baseline age between 45 and 64) in four US communities. Details of the study design were reported previously [<a href="#GFS591C11" rid="GFS591C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887384">11</a>]. Briefly, four visits, each ∼3 years apart, were conducted between 1987 and 1998. The baseline sample included 15 792 participants, ∼12 000 European Americans (EAs) and 4000 AAs.</p><p id="__p8">All biomarkers (BTP, serum and urinary creatinine and urinary albumin) for this study were measured from samples collected at visit 4. BTP was measured from plasma samples using nephelometric technology run on the Dade Behring Nephelometer II (BNII) system (sample reliability coefficient: 0.96 in 381 replicates after the removal of 7 outliers with BTP levels &gt;3 SD). A total of 10 557 observations were available (8269 for EA; 2288 for AAs). Combined with the availability of genotype data and the removal of 20 individuals with the self-reported dialysis status or BTP levels &gt;8 SD, the final sample sizes were 6720 in EA and 1734 in AAs (details in <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods section</a>).</p><p id="__p9">SCr was measured using the modified kinetic Jaffe method. eGFRscr was calculated using the CKD-EPI equation [<a href="#GFS591C12" rid="GFS591C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887364">12</a>] after standardizing SCr levels by calibrating to the age- and sex-specific means in the Third National Health and Nutrition Examination Survey (NHANES III) and adding 0.5 [<a href="#GFS591C13" rid="GFS591C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887371">13</a>]. In the calculation of the urine albumin creatinine ratio (UACR), the detectable limit of albumin was determined to be 2 mg/L and that of creatinine was 1 mg/dL. Individuals with non-detectable albumin and creatinine levels were given a value of half of the detectable limit.</p><p id="__p10" class="p p-last">Diabetes was defined as fasting glucose ≥126 mg/dL, non-fasting glucose ≥200 mg/dL, self-reported physician diagnosis of diabetes mellitus or the use of oral hypoglycemic medication or insulin. Hypertension was defined as systolic blood pressure ≥140 mmHg, diastolic blood pressure ≥90 mmHg or the use hypertension treatment medication.</p></div><div id="gfs591_s2b" class="sec sec-last"><h3 id="gfs591_s2btitle">Statistical methods</h3><p id="__p11" class="p p-first">A natural-log transformation was applied to BTP levels because its distribution was right skewed. In the genome-wide and the candidate region association analyses, a linear regression model was used to test for the associations between each SNP and log(BTP) levels controlling for age, gender, center and principal components associated (P &lt; 0.05) with log(BTP). Each SNP assumed an additive genetic effect. To determine the existence of multiple independent signals in each genome-wide significant locus, we conducted linear regression for each SNP adding the index SNP at the locus as an additional covariate. The statistic significant threshold for independent signals was set at 0.05 divided by the number of genotyped SNPs that were relatively independent (variance inflation factor, VIF, &lt;2) within the locus. The VIF was calculated using PLINK v1.07 [<a href="#GFS591C14" rid="GFS591C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887376">14</a>] with a window size of 50 SNPs and defined as 1/(1-R2), where R2 is the multiple correlation coefficient for a SNP being regressed against all other SNPs in a window. The replication P-value threshold in AAs was set at 0.001 (=0.05/45) based on the number of genotyped SNPs with VIF &lt;2 in the imputed region (139.4–140.4 mb on chromosome 9 in build 37 position). Details of genotyping and imputation and principal component generation are reported in the <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods section</a>. The genome-wide and candidate region association analyses in EAs and AAs were performed using ProbABEL [<a href="#GFS591C15" rid="GFS591C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_350805288">15</a>] and PLINK v1.07 [<a href="#GFS591C14" rid="GFS591C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887355">14</a>], respectively. Other analyses were conducted using SAS 9.2 or R.</p><p id="__p12">We tested for the differences in the study participant characteristics between EAs and AAs. (<em>t</em>-tests for age and eGFRscr, Wilcoxon test for BTP levels and Chi-square tests for dichotomous variables). To assess whether the GWAS signals upstream of <em>PTGDS</em> could explain the difference in BTP levels between EAs and AAs, we combined the data of the two cohorts and regressed self-reported race against log(BTP) in three regression models. Model 1 included age, gender and center as covariates. Model 2 added hypertension, diabetes, eGFRscr and common-log-transformed UACR as covariates. Model 3 added one of the index SNPs, rs7019538, which was selected because its association with BTP was similar to the other two index SNPs among EA based on P-value and was the strongest among AAs.</p><p id="__p13" class="p p-last">The regression analyses of the BTP index SNPs at the <em>PTGDS</em> locus against eGFRscr and UACR and the regression analyses of 16 eGFR index SNPs against scaled BTP and eGFRscr in EA were conducted controlling for age, gender, center and significant principal component(s). In the regression analyses of the eGFR index SNPs against BTP, we took the reciprocal of BTP and scaled it to have the same mean as eGFRscr, so that the effect size of the SNPs against BTP could be compared with that against eGFRscr. We refer to this as scaled BTP in this manuscript.</p></div></div><div id="gfs591_s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="gfs591_s3title">RESULTS</h2><p id="__p14" class="p p-first">Table&nbsp;<a href="/pmc/articles/PMC3685304/table/GFS591TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="GFS591TB1" rid-ob="ob-GFS591TB1" co-legend-rid=""><span>1</span></a> presents the basic characteristics of our study populations. Compared with EAs, AAs had lower BTP levels, higher estimated GFR based on SCr and more individuals with albuminuria, hypertension or diabetes.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="GFS591TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p15">Study participant characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140633006435120" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">EAs</th><th rowspan="1" colspan="1" align="left">AAs</th><th rowspan="1" colspan="1" align="left">P-value<sup>a</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Overall, <em>n</em></td><td rowspan="1" colspan="1">6720</td><td rowspan="1" colspan="1">1734</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Age, mean (SD)</td><td rowspan="1" colspan="1">63.1 (5.6)</td><td rowspan="1" colspan="1">61.9 (5.7)</td><td rowspan="1" colspan="1">2.6E-16</td></tr><tr><td rowspan="1" colspan="1">Women, % (<em>n</em>)</td><td rowspan="1" colspan="1">54.8 (3686)</td><td rowspan="1" colspan="1">64.3 (1115)</td><td rowspan="1" colspan="1">1.4E-12</td></tr><tr><td rowspan="1" colspan="1">Hypertension, % (<em>n</em>)</td><td rowspan="1" colspan="1">42.2 (2818)</td><td rowspan="1" colspan="1">67.6 (1166)</td><td rowspan="1" colspan="1">3.5E-79</td></tr><tr><td rowspan="1" colspan="1">Diabetes, % (<em>n</em>)</td><td rowspan="1" colspan="1">13.7 (916)</td><td rowspan="1" colspan="1">27.5 (469)</td><td rowspan="1" colspan="1">5.3E-43</td></tr><tr><td rowspan="1" colspan="1">eGFRscr &lt;60 mL/min/1.73 m<sup>2</sup>, % (<em>n</em>)</td><td rowspan="1" colspan="1">6.6 (437)</td><td rowspan="1" colspan="1">7.15 (124)</td><td rowspan="1" colspan="1">4.4E-01</td></tr><tr><td rowspan="1" colspan="1">eGFRscr, mean (SD)</td><td rowspan="1" colspan="1">82.8 (13.8)</td><td rowspan="1" colspan="1">89.3 (19.2)</td><td rowspan="1" colspan="1">2.9E-39</td></tr><tr><td rowspan="1" colspan="1">BTP (mg/L), median (IQR)</td><td rowspan="1" colspan="1">0.69 (0.60, 0.78)</td><td rowspan="1" colspan="1">0.60 (0.51, 0.69)</td><td rowspan="1" colspan="1">1.6E-110</td></tr><tr><td rowspan="1" colspan="1">Albuminuria<sup>b</sup>, % (<em>n</em>)</td><td rowspan="1" colspan="1">6.6 (440)</td><td rowspan="1" colspan="1">15.0 (258)</td><td rowspan="1" colspan="1">1.9E-29</td></tr><tr><td rowspan="1" colspan="1">CKD<sup>c</sup></td><td rowspan="1" colspan="1">11.8 (773)</td><td rowspan="1" colspan="1">19.1 (328)</td><td rowspan="1" colspan="1">3.4E-15</td></tr></tbody></table></div><div id="largeobj_idm140633006435120" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3685304/table/GFS591TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140633009815152"><p id="__p16" class="p p-first-last">eGFRscr, glomerular filtration rate estimated based on the serum creatinine level using the Chronic Kidney Disease Epidemiology Collaboration equation; CKD, chronic kidney disease; IQR, inter-quartile range.</p></div><div id="idm140633009814304"><p id="__p17" class="p p-first-last"><sup>a</sup>The Chi-square test for categorical variables; <em>t</em>-test for age and eGFRscr; the Wilcoxon test for BTP.</p></div><div id="idm140633009812800"><p id="__p18" class="p p-first-last"><sup>b</sup>Albuminuria defined as urinary albumin creatinine ratio &gt;30.</p></div><div id="idm140633011361680"><p id="__p19" class="p p-first-last"><sup>c</sup>CKD defined as eGFRscr &lt;60 mL/min/1.73 m<sup>2</sup> or the presence of albuminuria.</p></div></div></div><p id="__p20">The GWAS of BTP levels in EAs was initially conducted using imputed dosage based on a HapMap phase 2 reference panel, and it identified 11 SNPs, representing a single locus, on chromosome 9 that achieved genome-wide significance (P &lt; 5 × 10<sup>−8</sup>, <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Tables S1 and S2</a>). This genome-wide significant locus was upstream of <em>PTGDS</em>, the gene encoding BTP (rs7040970, beta = 0.04, % variance explained = 0.7%, P = 1.5 × 10<sup>−17</sup>; Table&nbsp;<a href="/pmc/articles/PMC3685304/table/GFS591TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="GFS591TB2" rid-ob="ob-GFS591TB2" co-legend-rid=""><span>2</span></a>). Figure&nbsp;<a href="/pmc/articles/PMC3685304/figure/GFS591F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="GFS591F1" rid-ob="ob-GFS591F1" co-legend-rid="lgnd_GFS591F1"><span>1</span></a> shows the regional association plot of the <em>PTGDS</em> locus. <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure S1</a> presents a plot of the –log10(P-values) by genomic position, and <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure S2</a> presents the quantile–quantile plot of the GWAS results in the EA samples with a genomic control factor of 1.02. Two other loci showed suggestive significance (P-value &lt; 5 × 10<sup>−6</sup>). One was near <em>MSTN</em> on chromosome 2 (rs16832277, beta = 0.02, P = 1 × 10<sup>−6</sup>, MAF = 0.28, variance explained = 0.3%). The other was at <em>GCKR</em> on chromosome 4 (rs1260326, beta = 0.018, P = 2 × 10<sup>−6</sup>, MAF = 0.41, variance explained = 0.3%), which was a reported eGFR locus [<a href="#GFS591C10" rid="GFS591C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887383">10</a>].</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="GFS591TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p21">Association of log(BTP) with index SNPs at the PTGDS locus in European and African Americans in ARIC</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140633006889280" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Position (b37)</th><th rowspan="1" colspan="1" align="left">Coded/non-coded allele</th><th rowspan="1" colspan="1" align="left">Coded allele frequency</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left">P-value</th></tr></thead><tbody><tr><td colspan="7" rowspan="1">EA HapMap imputation</td></tr><tr><td rowspan="1" colspan="1"> rs7040970</td><td rowspan="1" colspan="1">139 859 013</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">1.5E-17</td></tr><tr><td colspan="7" rowspan="1">EA 1000 Genomes imputation</td></tr><tr><td rowspan="1" colspan="1"> rs57024841</td><td rowspan="1" colspan="1">139 862 633</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">1.2E-23</td></tr><tr><td colspan="7" rowspan="1">AA 1000 Genomes imputation</td></tr><tr><td rowspan="1" colspan="1"> rs57024841</td><td rowspan="1" colspan="1">139 862 633</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.9E-07</td></tr><tr><td rowspan="1" colspan="1"> rs7019538</td><td rowspan="1" colspan="1">139 861 470</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">6.8E-08</td></tr></tbody></table></div><div id="largeobj_idm140633006889280" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3685304/table/GFS591TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140633005556352"><p id="__p22" class="p p-first-last">EA, European American; AA, African American; SE, standard error.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="GFS591F1" co-legend-rid="lgnd_GFS591F1"><a href="/pmc/articles/PMC3685304/figure/GFS591F1/" target="figure" rid-figpopup="GFS591F1" rid-ob="ob-GFS591F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140633010042976" class="figure"><a href="/pmc/articles/PMC3685304/figure/GFS591F1/" target="figure" rid-figpopup="GFS591F1" rid-ob="ob-GFS591F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3685304_gfs59101.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gfs59101.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3685304/bin/gfs59101.jpg"></a></div><div id="largeobj_idm140633010042976" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3685304/figure/GFS591F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_GFS591F1"><div><a class="figpopup" href="/pmc/articles/PMC3685304/figure/GFS591F1/" target="figure" rid-figpopup="GFS591F1" rid-ob="ob-GFS591F1">FIGURE&nbsp;1:</a></div><!--caption a7--><div class="caption"><p id="__p23">The regional association plot of BTP genome-wide significant locus upstream of <em>PTGDS</em> using genotype dosage imputed from HapMap Phase II reference panel. Build 37 genomic positions are shown on the <em>x</em>-axis. The purple diamond marks the index SNP, rs7040970.</p></div></div></div><p id="__p24">We decided to follow-up on the novel genome-wide significant locus at <em>PTGDS</em>. To finemap the region near <em>PTGDS</em>, we subsequently imputed SNPs across this locus (139.4–140.4 mb in build 37 position) in both EAs and AAs based on 1000 Genomes reference panels [August 2010 release for EA and the Phase I (interim) for AAs]. This 139.4–140.4 mb region was selected because the closest recombination peak was ∼400 kb upstream of the GWAS index SNP rs7040970 (<a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure S3</a>) and cis-regulatory elements are unlikely to exist far from the protein-coding gene [<a href="#GFS591C16" rid="GFS591C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887372">16</a>]. Using the 1000 Genomes imputed dosage in EAs, the top SNP was rs57024841 (beta = 0.05, P = 1.2 × 10<sup>−23</sup>; Table&nbsp;<a href="/pmc/articles/PMC3685304/table/GFS591TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="GFS591TB2" rid-ob="ob-GFS591TB2" co-legend-rid=""><span>2</span></a>, <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S3</a>). In the regression analysis controlling for this index SNP (rs57024841), we did not detect other significant signals (all SNPs had P &gt; 2.9 × 10<sup>−3</sup> &gt; 0.05/28, the number of independent genotyped SNPs in this locus in EAs). The two index SNPs from HapMap phase 2 and 1000 Genomes imputed data (rs7040970 and rs57024841, respectively) were in linkage disequilibrium (LD) (<em>r</em><sup>2</sup> of 0.76 in the 1000 Genomes Interim EUR sample and 0.96 in the ARIC EA samples (<a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S4</a>).</p><p id="__p25">The replication analysis in AAs using 1000 Genomes imputed dosage at the <em>PTGDS</em> locus detected 64 significant SNPs with P-value below the pre-specified threshold of 0.001. The top SNP was rs7019538 (beta = 0.06, P = 6.8 × 10<sup>−8</sup>; Table&nbsp;<a href="/pmc/articles/PMC3685304/table/GFS591TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="GFS591TB2" rid-ob="ob-GFS591TB2" co-legend-rid=""><span>2</span></a> and <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S5</a>). After controlling for this top SNP, rs7019538, no additional signals were detected among AAs (P &gt; 1.5 × 10<sup>−3</sup>). The index SNP identified in EA (rs57024841) was also significant in AAs (beta = 0.06, P = 1.9 × 10<sup>−7</sup>; <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S1</a>).</p><p id="__p26">Each of the index SNPs (rs57024841 in EA and rs7019538 in AAs) explained ∼1.1% of the log(BTP) variance in their respective populations (Table&nbsp;<a href="/pmc/articles/PMC3685304/table/GFS591TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="GFS591TB2" rid-ob="ob-GFS591TB2" co-legend-rid=""><span>2</span></a>). The two SNPs were ∼1 kb apart and in high LD within both populations (D' of 1 in 1000 Genomes EUR and AFR populations, <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S4</a>). Figure&nbsp;<a href="/pmc/articles/PMC3685304/figure/GFS591F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="GFS591F2" rid-ob="ob-GFS591F2" co-legend-rid="lgnd_GFS591F2"><span>2</span></a> presents the regional association plots in EAs and AAs with their respective LD plots showing more diffused but largely similar LD pattern in AAs compared with the EA samples. As shown in <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S1</a>, the two index SNPs had comparable beta estimates within each population (0.05 for rs57024841 and 0.04 for rs7019538 in EA and 0.06 for both SNPs in AAs), indicating that they likely represent the same signal(s). Across the two populations, the coding allele of two index SNPs had lower allele frequencies in AAs than in EAs (0.34 in AAs versus 0.58 in EAs for rs57024841, 0.21 in AAs versus 0.51 in EAs for rs7019538).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="GFS591F2" co-legend-rid="lgnd_GFS591F2"><a href="/pmc/articles/PMC3685304/figure/GFS591F2/" target="figure" rid-figpopup="GFS591F2" rid-ob="ob-GFS591F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140633001611136" class="figure"><a href="/pmc/articles/PMC3685304/figure/GFS591F2/" target="figure" rid-figpopup="GFS591F2" rid-ob="ob-GFS591F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3685304_gfs59102.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gfs59102.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3685304/bin/gfs59102.jpg"></a></div><div id="largeobj_idm140633001611136" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3685304/figure/GFS591F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_GFS591F2"><div><a class="figpopup" href="/pmc/articles/PMC3685304/figure/GFS591F2/" target="figure" rid-figpopup="GFS591F2" rid-ob="ob-GFS591F2">FIGURE&nbsp;2:</a></div><!--caption a7--><div class="caption"><p id="__p27">Genetic association and LD patterns at the <em>PTGDS</em> locus using imputed dosage from the 1000 Genomes reference panels in EAs and AAs. The Haploview plots on the right show the D' measures between SNPs in the same region as the regional association plots on the left. The positions of the index SNPs and the <em>PTGDS</em> gene are marked with vertical lines in the Haploview plots.</p></div></div></div><p id="__p28">To determine whether the lower allele frequencies of the coding allele of the index SNPs in AAs could partly account for the lower BTP levels in AAs observed between the ARIC EA and AA participants, we performed a combined analysis of the association between self-reported race and log(BTP) levels. In a model controlled for age, sex and center, self-report ‘black’ race was associated with lower BTP levels (beta = −0.10, equivalent to 10% lower in BTP levels, Table&nbsp;<a href="/pmc/articles/PMC3685304/table/GFS591TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="GFS591TB3" rid-ob="ob-GFS591TB3" co-legend-rid=""><span>3</span></a>). In a model controlled for rs57024841 in addition to age, sex, center, diabetes, hypertension, eGFRscr and UACR, the effect estimate of self-reported ‘black’ race was reduced by 38% (Model 3, beta = −0.035) compared with the model without the SNP (Model 2, beta: −0.056, Table&nbsp;<a href="/pmc/articles/PMC3685304/table/GFS591TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="GFS591TB3" rid-ob="ob-GFS591TB3" co-legend-rid=""><span>3</span></a>). Adjusted for these variables, log BTP was not associated with diabetes and hypertension (P &gt; 0.05) and moderately associated with UACR (beta = 0.06 per 10-fold higher UACR) and, as expected, strongly related to eGFRscr (beta = −0.008/1 mL/min/1.73 m<sup>2</sup>, equivalent to 38% lower BTP levels for 60 mL higher eGFR).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="GFS591TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p29">Association of log(BTP) with self-reported race and rs57024841 in combined race data set</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140633007537600" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Parameter</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left">P-value</th></tr></thead><tbody><tr><td colspan="4" rowspan="1">Model 1: controlled for age, sex, center</td></tr><tr><td rowspan="1" colspan="1"> Race (black)</td><td rowspan="1" colspan="1">−0.100</td><td rowspan="1" colspan="1">0.007</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="4" rowspan="1">Model 2: controlled for age, sex, center, hypertension, diabetes, eGFRscr and UACR</td></tr><tr><td rowspan="1" colspan="1"> Race (black)</td><td rowspan="1" colspan="1">−0.056</td><td rowspan="1" colspan="1">0.0061</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1"> Diabetes</td><td rowspan="1" colspan="1">−0.009</td><td rowspan="1" colspan="1">0.0060</td><td rowspan="1" colspan="1">0.1234</td></tr><tr><td rowspan="1" colspan="1"> Hypertension</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">0.0045</td><td rowspan="1" colspan="1">0.5212</td></tr><tr><td rowspan="1" colspan="1"> eGFRscr</td><td rowspan="1" colspan="1">−0.008</td><td rowspan="1" colspan="1">0.0001</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1"> Log10 (UACR, mg/g)</td><td rowspan="1" colspan="1">0.060</td><td rowspan="1" colspan="1">0.0036</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="4" rowspan="1">Model 3: controlled for age, sex, center, hypertension, diabetes, eGFRscr, UACR and rs7019538</td></tr><tr><td rowspan="1" colspan="1"> Race (black)</td><td rowspan="1" colspan="1">−0.035</td><td rowspan="1" colspan="1">0.0063</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1"> rs57024841</td><td rowspan="1" colspan="1">0.042</td><td rowspan="1" colspan="1">0.0034</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1"> Diabetes</td><td rowspan="1" colspan="1">−0.010</td><td rowspan="1" colspan="1">0.0059</td><td rowspan="1" colspan="1">0.0893</td></tr><tr><td rowspan="1" colspan="1"> Hypertension</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">0.0045</td><td rowspan="1" colspan="1">0.5707</td></tr><tr><td rowspan="1" colspan="1"> eGFRscr (mL/min/1.73 m<sup>2</sup>)</td><td rowspan="1" colspan="1">−0.008</td><td rowspan="1" colspan="1">0.0001</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1"> Log10(UACR, mg/g)</td><td rowspan="1" colspan="1">0.060</td><td rowspan="1" colspan="1">0.0036</td><td rowspan="1" colspan="1">&lt;0.0001</td></tr></tbody></table></div><div id="largeobj_idm140633007537600" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3685304/table/GFS591TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140633014534368"><p id="__p30" class="p p-first-last">eGFRscr, glomerular filtration rate estimated based on the serum creatinine level using the Chronic Kidney Disease Epidemiology Collaboration equation; UACR, urinary albumin-creatinine ratio.</p></div></div></div><p id="__p31">To determine the potential function of the significant SNPs identified from our GWAS, we interrogated gene expression databases for expression SNP (eSNP) associations with <em>PTGDS</em> gene expression. [<a href="#GFS591C17" rid="GFS591C17" class=" bibr popnode">17</a>, <a href="#GFS591C18" rid="GFS591C18" class=" bibr popnode">18</a>] Two eSNPs downstream of <em>PTGDS</em> were reported to be associated with <em>PTGDS</em> expression [<a href="#GFS591C19" rid="GFS591C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887377">19</a>], but did not explain our GWAS signals upstream of <em>PTGDS</em> (<a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S6</a>). Since transcription factor binding may affect gene expression and thus protein levels [<a href="#GFS591C20" rid="GFS591C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887351">20</a>], we also queried the data from the ENCODE project [<a href="#GFS591C21" rid="GFS591C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887367">21</a>] for evidence of transcription factor activities at or near the <em>PTDGS</em> locus and found some evidence of transcription factor binding (<a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure S4</a>).</p><p id="__p32" class="p p-last">To assess whether the <em>PTGDS</em> locus may be associated with kidney function, we tested for the association of the BTP index SNPs at the <em>PTGDS</em> locus against eGFRscr and log-transformed UACR in our EA and AA cohorts and found no evidence of associations (P-value &gt;0.05, <a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S7</a>). Finally, to assess the associations between known eGFR loci [<a href="#GFS591C10" rid="GFS591C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887368">10</a>] and BTP levels, we tested for the association of scaled BTP (for variable definition, see the section ‘Methods’) with the index SNPs at 16 known eGFR loci and compared their association with scaled BTP with their association with eGFRscr. These 16 loci were identified with a total sample size of ∼90 000 [<a href="#GFS591C10" rid="GFS591C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887353">10</a>] from cohorts in the CKDGen consortium. In our study of 6720 individuals, scaled BTP was associated with three loci (<em>GCKR</em>, <em>NAT8</em> and <em>UMOD</em>) at an alpha level of 0.003 (=0.05/16). The eGFR index SNP at the <em>GCKR</em> locus, rs1260326, was the same SNP that attained suggestive significance in the EA GWAS of log(BTP). Even though only three index SNPs reached statistical significance, 13 of the 16 loci had beta estimates in the same direction as the beta estimates originally reported [<a href="#GFS591C10" rid="GFS591C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887361">10</a>] (binomial test P-value = 0.01), an event that is unlikely due to chance. Based on the same P-value threshold of 0.003, eGFRscr was associated with seven loci with all beta estimates in the same direction as those originally reported [<a href="#GFS591C10" rid="GFS591C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887373">10</a>] (<a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table S8</a>).</p></div><div id="gfs591_s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="gfs591_s4title">DISCUSSION</h2><p id="__p33" class="p p-first">We identified a genome-wide significant locus of BTP upstream of <em>PTGDS</em>, the gene encoding BTP. The A allele of rs57024841 had a frequency of 58% in EAs compared with 34% in AAs but was associated with a similarly higher level of BTP in the two populations (5 and 6%, respectively). As a result, the <em>PTGDS</em> locus explained over a third of the difference in log(BTP) levels between EAs and AAs in the ARIC study. It accounted for 1.1% of the log(BTP) variance in both EAs and AAs in our sample, and was not associated with either eGFRscr or UACR. However, 13 of the 16 known eGFR loci showed directionally consistent associations with BTP levels, and three loci were statistically significant after a Bonferroni correction. In addition, BTP levels were highly associated with eGFRscr. After adjustment for eGFRscr, BTP was moderately associated with self-reported race and UACR and not associated with hypertension and diabetes.</p><p id="__p34">BTP belongs to the lipocalin protein family and is also known as lipocalin-type prostaglandin (PG) D synthase (L-PGDS). It catalyzes the isomerization of PGH<sub>2</sub> as an enzyme [<a href="#GFS591C22" rid="GFS591C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887360">22</a>] and was found to have an independent role in inhibiting astrocyte proliferation [<a href="#GFS591C23" rid="GFS591C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887359">23</a>] and protection against oxidative stress in neural cells [<a href="#GFS591C24" rid="GFS591C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887358">24</a>] in functional experiments. It was first isolated in the cerebrospinal fluid [<a href="#GFS591C25" rid="GFS591C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887380">25</a>] and subsequently shown to be expressed in a variety of tissues, including cells in the glomeruli and the Loop of Henle [<a href="#GFS591C26" rid="GFS591C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887352">26</a>]. After BTP is filtered by the glomeruli, it may be partially reabsorbed and degraded by tubular cells [<a href="#GFS591C26" rid="GFS591C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887348">26</a>] and then excreted in the urine [<a href="#GFS591C27" rid="GFS591C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887354">27</a>–<a href="#GFS591C29" rid="GFS591C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887385">29</a>]. Besides kidney function, little is known about the biological mechanism affecting its level in the blood. White <em>et al</em>., using successive 5′ deletions, identified a core promoter within the 5′ flanking sequence of <em>PTDGS</em> and a thyroid hormone response element (TRE) from 2576 to 2562 bp upstream of <em>PTDGS</em>. There were no SNPs within the 14 bp of the TRE in our imputed data. As reported in the ‘Results’ section, our query of a gene expression database did not find SNP associations with <em>PTGDS</em> expression levels that could explain the GWAS signals. The upstream location of the GWAS signals together with evidence of transcription factor binding in this region from the ENCODE project is consistent with the hypothesis of the existence of distal regulatory elements [<a href="#GFS591C30" rid="GFS591C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887381">30</a>–<a href="#GFS591C32" rid="GFS591C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887378">32</a>]. Further studies are needed to elucidate the biological mechanism underlying the SNP associations at the <em>PTGDS</em> locus with BTP levels.</p><p id="__p35">Although BTP has been studied as a serum or plasma biomarker for kidney function in patients with kidney disease [<a href="#GFS591C3" rid="GFS591C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887350">3</a>, <a href="#GFS591C5" rid="GFS591C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887370">5</a>, <a href="#GFS591C6" rid="GFS591C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887374">6</a>, <a href="#GFS591C33" rid="GFS591C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887382">33</a>–<a href="#GFS591C36" rid="GFS591C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319887369">36</a>] and in a community-based cohort [<a href="#GFS591C9" rid="GFS591C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_350805290">9</a>], relatively little is known about the non-kidney factors influencing BTP levels. In our study of population-based European and African American cohorts, we found the median BTP levels to be ∼0.09 mg/L lower in AA than in their EA counterparts. Moreover, we have identified a genetic locus that is associated with 5–6% differences in BTP levels and accounts for about a third of the racial difference. GFR estimating equations usually require a race coefficient to adjust for race differences in biomarker levels unrelated to kidney function. Self-reported race can be imprecise. If genetic information becomes a routine part of the medical chart, our discovery can be used to account for the largest genetic non-kidney-related individual difference in BTP levels. However, it is also useful to note that this genetic influence is only of moderate size (5 in EAs and 6% in AAs), and our unbiased genome-wide scan suggests that common variants of a larger effect on BTP are unlikely to exist. We also showed that after adjustment for eGFRscr, plasma BTP levels were moderately higher at higher UACR levels.</p><p id="__p36">A limitation of this study is that further functional studies and replication are warranted to fully characterize the causal variant(s) in the novel locus we identified. In contrast, a strength of the present study is the robust, consistent association in both EAs and AAs in a population-based cohort. The GWAS panel spanned the whole genome (∼2.5 million imputed SNPs) and the study sample was large with data collected by uniform methods. BTP was measured with high reliability (reliability coefficient of 0.96).</p><p id="__p37" class="p p-last">In conclusion, we discovered a variant upstream of the BTP encoding gene, <em>PTGDS</em>, with a frequency of 58% in EAs, associated with 5% higher levels of BTP and not associated with other measures of kidney function. The frequency of this variant was only 34% in AAs, coding for 6% higher BTP, as a result explaining one-third of the racial difference in log(BTP) levels despite explaining only ∼1.1% of the variance in log(BTP) levels in each ethnic group. Known genetic loci influencing creatinine-based measures of kidney function showed direction consistent associations with BTP levels supporting the role of BTP as a novel filtration marker whose metabolism is known to be different from that of creatinine.</p></div><div id="gfs591_s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="gfs591_s5title">SUPPLEMENTARY DATA</h2><p id="__p38" class="p p-first-last"><a href="http://ndt.oxfordjournals.org/lookup/suppl/doi:10.1093/ndt/gfs591/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary data are available online at http://ndt.oxfordjournals.org</a>.</p></div><div id="gfs591_s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="gfs591_s6title">FUNDING</h2><p id="__p39" class="p p-first-last">This study was supported by the National Institute of Diabetes and Digestive and Kidney Diseases
<span>R01 DK076770–01</span> Siemens Healthcare Diagnostics provided the reagents and loan of a BNII instrument to conduct the BTP assays. The Atherosclerosis Risk in Communities Study is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute contracts (<span>HHSN268201100005C</span>;, <span>HHSN268201100006C</span>;, <span>HHSN268201100007C</span>;, <span>HHSN268201100008C</span>;, <span>HHSN268201100009C</span>;, <span>HHSN268201100010C</span>;, <span>HHSN268201100011C</span>; and <span>HHSN268201100012C</span>;), <span>R01HL087641</span>;, <span>R01HL59367</span>; and <span>R01HL086694</span>;; National Human Genome Research Institute contract <span>U01HG004402</span>;; and National Institutes of Health contract <span>HHSN268200625226C</span>;. Infrastructure was partly supported by Grant Number <span>UL1RR025005</span>, a component of the National Institutes of Health and NIH Roadmap for Medical Research.</p></div><div id="gfs591_s7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="gfs591_s7title">CONFLICT OF INTEREST STATEMENT</h2><p id="__p40" class="p p-first-last">None declared.</p></div><div id="__sec5" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec5title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140633013872496"><a href="/pmc/articles/PMC3685304/bin/supp_28_6_1497__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140633085115632" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140633085115632title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p41">The authors thank the staff and participants of the ARIC study for their important contributions.</p></div></div><div id="idm140633004475440" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140633004475440title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="GFS591C1">1. <span class="element-citation">K/DOQI. K/DOQI clinical practice guidelines for chronic kidney disease: evaluation, classification, and stratification. <span><span class="ref-journal">Am J Kidney Dis. </span>2002;<span class="ref-vol">39</span>(2 Suppl. 1):S1–266.</span> [<a href="/pubmed/11904577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Kidney+Dis&amp;title=K/DOQI+clinical+practice+guidelines+for+chronic+kidney+disease:+evaluation,+classification,+and+stratification&amp;volume=39&amp;issue=2+Suppl.+1&amp;publication_year=2002&amp;pages=S1-266&amp;pmid=11904577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C2">2. <span class="element-citation">Stevens LA, Padala S, Levey AS. Advances in glomerular filtration rate-estimating equations. <span><span class="ref-journal">Curr Opin Nephrol Hypertens. </span>2010;<span class="ref-vol">19</span>:298–307.</span>  <a href="http://dx.doi.org/10.1097/MNH.0b013e32833893e2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1097/MNH.0b013e32833893e2</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3192437/">PMC free article</a>]</span> [<a href="/pubmed/20393287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Nephrol+Hypertens&amp;title=Advances+in+glomerular+filtration+rate-estimating+equations&amp;author=LA+Stevens&amp;author=S+Padala&amp;author=AS+Levey&amp;volume=19&amp;publication_year=2010&amp;pages=298-307&amp;pmid=20393287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C3">3. <span class="element-citation">Priem F, Althaus H, Birnbaum M. Beta-trace protein in serum: a new marker of glomerular filtration rate in the creatinine-blind range. <span><span class="ref-journal">Clin Chem. </span>1999;<span class="ref-vol">45</span>:567–568.</span> [<a href="/pubmed/10102918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Beta-trace+protein+in+serum:+a+new+marker+of+glomerular+filtration+rate+in+the+creatinine-blind+range&amp;author=F+Priem&amp;author=H+Althaus&amp;author=M+Birnbaum&amp;volume=45&amp;publication_year=1999&amp;pages=567-568&amp;pmid=10102918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C4">4. <span class="element-citation">Woitas RP, Stoffel-Wagner B, Poege U. Low-molecular weight proteins as markers for glomerular filtration rate. <span><span class="ref-journal">Clin Chem. </span>2001;<span class="ref-vol">47</span>:2179–2180.</span> [<a href="/pubmed/11719489" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Low-molecular+weight+proteins+as+markers+for+glomerular+filtration+rate&amp;author=RP+Woitas&amp;author=B+Stoffel-Wagner&amp;author=U+Poege&amp;volume=47&amp;publication_year=2001&amp;pages=2179-2180&amp;pmid=11719489&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C5">5. <span class="element-citation">Kobata M, Shimizu A, Rinno H. Beta-trace protein, a new marker of GFR, may predict the early prognostic stages of patients with type 2 diabetic nephropathy. <span><span class="ref-journal">J Clin Lab Anal. </span>2004;<span class="ref-vol">18</span>:237–239.</span>  <a href="http://dx.doi.org/10.1002/jcla.20029" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1002/jcla.20029</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6807906/">PMC free article</a>]</span> [<a href="/pubmed/15202116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Lab+Anal&amp;title=Beta-trace+protein,+a+new+marker+of+GFR,+may+predict+the+early+prognostic+stages+of+patients+with+type+2+diabetic+nephropathy&amp;author=M+Kobata&amp;author=A+Shimizu&amp;author=H+Rinno&amp;volume=18&amp;publication_year=2004&amp;pages=237-239&amp;pmid=15202116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C6">6. <span class="element-citation">Poge U, Gerhardt TM, Stoffel-Wagner B. Beta-trace protein is an alternative marker for glomerular filtration rate in renal transplantation patients. <span><span class="ref-journal">Clin Chem. </span>2005;<span class="ref-vol">51</span>:1531–1533.</span>  <a href="http://dx.doi.org/10.1373/clinchem.2005.048959" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1373/clinchem.2005.048959</a>. [<a href="/pubmed/15951315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Beta-trace+protein+is+an+alternative+marker+for+glomerular+filtration+rate+in+renal+transplantation+patients&amp;author=U+Poge&amp;author=TM+Gerhardt&amp;author=B+Stoffel-Wagner&amp;volume=51&amp;publication_year=2005&amp;pages=1531-1533&amp;pmid=15951315&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C7">7. <span class="element-citation">Tangri N, Inker LA, Tighiouart H. Filtration markers may have prognostic value independent of glomerular filtration rate. <span><span class="ref-journal">J Am Soc Nephrol. </span>2011;<span class="ref-vol">23</span>:351–359.</span>  <a href="http://dx.doi.org/10.1681/ASN.2011070663" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1681/ASN.2011070663</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3269188/">PMC free article</a>]</span> [<a href="/pubmed/22173699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Filtration+markers+may+have+prognostic+value+independent+of+glomerular+filtration+rate&amp;author=N+Tangri&amp;author=LA+Inker&amp;author=H+Tighiouart&amp;volume=23&amp;publication_year=2011&amp;pages=351-359&amp;pmid=22173699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C8">8. <span class="element-citation">Bhavsar NA, Appel LJ, Kusek JW. Comparison of measured GFR, serum creatinine, cystatin C, and beta-trace protein to predict ESRD in African Americans with hypertensive CKD. <span><span class="ref-journal">Am J Kidney Dis. </span>2011;<span class="ref-vol">58</span>:886–893.</span>  <a href="http://dx.doi.org/10.1053/j.ajkd.2011.07.018" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1053/j.ajkd.2011.07.018</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3221777/">PMC free article</a>]</span> [<a href="/pubmed/21944667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Kidney+Dis&amp;title=Comparison+of+measured+GFR,+serum+creatinine,+cystatin+C,+and+beta-trace+protein+to+predict+ESRD+in+African+Americans+with+hypertensive+CKD&amp;author=NA+Bhavsar&amp;author=LJ+Appel&amp;author=JW+Kusek&amp;volume=58&amp;publication_year=2011&amp;pages=886-893&amp;pmid=21944667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C9">9. <span class="element-citation">Astor BC, Shafi T, Hoogeveen RC. Novel markers of kidney function as predictors of ESRD, cardiovascular disease, and mortality in the general population. <span><span class="ref-journal">Am J Kidney Dis. </span>2012;<span class="ref-vol">59</span>:653–662.</span>  <a href="http://dx.doi.org/10.1053/j.ajkd.2011.11.042" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1053/j.ajkd.2011.11.042</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3880682/">PMC free article</a>]</span> [<a href="/pubmed/22305758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Kidney+Dis&amp;title=Novel+markers+of+kidney+function+as+predictors+of+ESRD,+cardiovascular+disease,+and+mortality+in+the+general+population&amp;author=BC+Astor&amp;author=T+Shafi&amp;author=RC+Hoogeveen&amp;volume=59&amp;publication_year=2012&amp;pages=653-662&amp;pmid=22305758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C10">10. <span class="element-citation">Kottgen A, Pattaro C, Boger CA. New loci associated with kidney function and chronic kidney disease. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:376–384.</span>  <a href="http://dx.doi.org/10.1038/ng.568" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.568</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997674/">PMC free article</a>]</span> [<a href="/pubmed/20383146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+loci+associated+with+kidney+function+and+chronic+kidney+disease&amp;author=A+Kottgen&amp;author=C+Pattaro&amp;author=CA+Boger&amp;volume=42&amp;publication_year=2010&amp;pages=376-384&amp;pmid=20383146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C11">11. <span class="element-citation">ARIC. The atherosclerosis risk in communities (ARIC) study: design and objectives. The ARIC investigators. <span><span class="ref-journal">Am J Epidemiol. </span>1989;<span class="ref-vol">129</span>:687–702.</span> [<a href="/pubmed/2646917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=The+atherosclerosis+risk+in+communities+(ARIC)+study:+design+and+objectives.+The+ARIC+investigators&amp;volume=129&amp;publication_year=1989&amp;pages=687-702&amp;pmid=2646917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C12">12. <span class="element-citation">Levey AS, Stevens LA, Schmid CH. A new equation to estimate glomerular filtration rate. <span><span class="ref-journal">Ann Intern Med. </span>2009;<span class="ref-vol">150</span>:604–612.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2763564/">PMC free article</a>]</span> [<a href="/pubmed/19414839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Intern+Med&amp;title=A+new+equation+to+estimate+glomerular+filtration+rate&amp;author=AS+Levey&amp;author=LA+Stevens&amp;author=CH+Schmid&amp;volume=150&amp;publication_year=2009&amp;pages=604-612&amp;pmid=19414839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C13">13. <span class="element-citation">Selvin E, Manzi J, Stevens LA. Calibration of serum creatinine in the National Health and Nutrition Examination Surveys (NHANES) 1988–1994, 1999–2004. <span><span class="ref-journal">Am J Kidney Dis. </span>2007;<span class="ref-vol">50</span>:918–926.</span>  <a href="http://dx.doi.org/10.1053/j.ajkd.2007.08.020" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1053/j.ajkd.2007.08.020</a>. [<a href="/pubmed/18037092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Kidney+Dis&amp;title=Calibration+of+serum+creatinine+in+the+National+Health+and+Nutrition+Examination+Surveys+(NHANES)+1988–1994,+1999–2004&amp;author=E+Selvin&amp;author=J+Manzi&amp;author=LA+Stevens&amp;volume=50&amp;publication_year=2007&amp;pages=918-926&amp;pmid=18037092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C14">14. <span class="element-citation">Purcell S, Neale B, Todd-Brown K. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span>  <a href="http://dx.doi.org/10.1086/519795" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1086/519795</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C15">15. <span class="element-citation">Aulchenko YS, Struchalin MV, van Duijn CM. ProbABEL package for genome-wide association analysis of imputed data. <span><span class="ref-journal">BMC Bioinf. </span>2010;<span class="ref-vol">11</span>:134.</span>  <a href="http://dx.doi.org/10.1186/1471-2105-11-134" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1186/1471-2105-11-134</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2846909/">PMC free article</a>]</span> [<a href="/pubmed/20233392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf&amp;title=ProbABEL+package+for+genome-wide+association+analysis+of+imputed+data&amp;author=YS+Aulchenko&amp;author=MV+Struchalin&amp;author=CM+van+Duijn&amp;volume=11&amp;publication_year=2010&amp;pages=134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C16">16. <span class="element-citation">Gaffney DJ, Veyrieras JB, Degner JF. Dissecting the regulatory architecture of gene expression QTLs. <span><span class="ref-journal">Genome Biol. </span>2012;<span class="ref-vol">13</span>:R7.</span>  <a href="http://dx.doi.org/10.1186/gb-2012-13-1-r7" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1186/gb-2012-13-1-r7</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3334587/">PMC free article</a>]</span> [<a href="/pubmed/22293038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Dissecting+the+regulatory+architecture+of+gene+expression+QTLs&amp;author=DJ+Gaffney&amp;author=JB+Veyrieras&amp;author=JF+Degner&amp;volume=13&amp;publication_year=2012&amp;pages=R7&amp;pmid=22293038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C17">17. <span class="element-citation"> GTEx (Genotype-Tissue Expression) eQTL Database.</span></div><div class="ref-cit-blk half_rhythm" id="GFS591C18">18. <span class="element-citation"> eQTL resources @ the pritchard lab. <a href="http://eqtl.uchicago.edu/Home.html">http://eqtl.uchicago.edu/Home.html</a>.  (15 August 2011, date last accessed)</span></div><div class="ref-cit-blk half_rhythm" id="GFS591C19">19. <span class="element-citation">Zeller T, Wild P, Szymczak S. Genetics and beyond—the transcriptome of human monocytes and disease susceptibility. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>:e10693.</span>  <a href="http://dx.doi.org/10.1371/journal.pone.0010693" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1371/journal.pone.0010693</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872668/">PMC free article</a>]</span> [<a href="/pubmed/20502693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genetics+and+beyond—the+transcriptome+of+human+monocytes+and+disease+susceptibility&amp;author=T+Zeller&amp;author=P+Wild&amp;author=S+Szymczak&amp;volume=5&amp;publication_year=2010&amp;pages=e10693&amp;pmid=20502693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C20">20. <span class="element-citation">Consortium EP, Birney E, Stamatoyannopoulos JA. Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:799–816.</span>  <a href="http://dx.doi.org/10.1038/nature05874" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/nature05874</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2212820/">PMC free article</a>]</span> [<a href="/pubmed/17571346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Identification+and+analysis+of+functional+elements+in+1%+of+the+human+genome+by+the+ENCODE+pilot+project&amp;author=EP+Consortium&amp;author=E+Birney&amp;author=JA+Stamatoyannopoulos&amp;volume=447&amp;publication_year=2007&amp;pages=799-816&amp;pmid=17571346&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C21">21. <span class="element-citation">Raney BJ, Cline MS, Rosenbloom KR. ENCODE whole-genome data in the UCSC genome browser (2011 update) <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>:D871–D875.</span>  <a href="http://dx.doi.org/10.1093/nar/gkq1017" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/nar/gkq1017</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013645/">PMC free article</a>]</span> [<a href="/pubmed/21037257" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=ENCODE+whole-genome+data+in+the+UCSC+genome+browser+(2011+update)&amp;author=BJ+Raney&amp;author=MS+Cline&amp;author=KR+Rosenbloom&amp;volume=39&amp;publication_year=2011&amp;pages=D871-D875&amp;pmid=21037257&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C22">22. <span class="element-citation">Urade Y, Hayaishi O. Biochemical, structural, genetic, physiological, and pathophysiological features of lipocalin-type prostaglandin D synthase. <span><span class="ref-journal">Biochim Biophys Acta. </span>2000;<span class="ref-vol">1482</span>:259–271.</span>  <a href="http://dx.doi.org/10.1016/S0167-4838(00)00161-8" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/S0167-4838(00)00161-8</a>. [<a href="/pubmed/11058767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=Biochemical,+structural,+genetic,+physiological,+and+pathophysiological+features+of+lipocalin-type+prostaglandin+D+synthase&amp;author=Y+Urade&amp;author=O+Hayaishi&amp;volume=1482&amp;publication_year=2000&amp;pages=259-271&amp;pmid=11058767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C23">23. <span class="element-citation">Xin X, Huber A, Meyer P. L-PGDS (betatrace protein) inhibits astrocyte proliferation and mitochondrial ATP production <em>in vitro</em>. <span><span class="ref-journal">J Mol Neurosci. </span>2009;<span class="ref-vol">39</span>:366–371.</span>  <a href="http://dx.doi.org/10.1007/s12031-009-9214-7" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1007/s12031-009-9214-7</a>. [<a href="/pubmed/19598000" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Neurosci&amp;title=L-PGDS+(betatrace+protein)+inhibits+astrocyte+proliferation+and+mitochondrial+ATP+production+in+vitro&amp;author=X+Xin&amp;author=A+Huber&amp;author=P+Meyer&amp;volume=39&amp;publication_year=2009&amp;pages=366-371&amp;pmid=19598000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C24">24. <span class="element-citation">Fukuhara A, Yamada M, Fujimori K. Lipocalin-type prostaglandin D synthase protects against oxidative stress-induced neuronal cell death. <span><span class="ref-journal">Biochem J. </span>2012;<span class="ref-vol">443</span>:75–84.</span>  <a href="http://dx.doi.org/10.1042/BJ20111889" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1042/BJ20111889</a>. [<a href="/pubmed/22248185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Lipocalin-type+prostaglandin+D+synthase+protects+against+oxidative+stress-induced+neuronal+cell+death&amp;author=A+Fukuhara&amp;author=M+Yamada&amp;author=K+Fujimori&amp;volume=443&amp;publication_year=2012&amp;pages=75-84&amp;pmid=22248185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C25">25. <span class="element-citation">Saso L, Leone MG, Sorrentino C. Quantification of prostaglandin D synthetase in cerebrospinal fluid: a potential marker for brain tumor. <span><span class="ref-journal">Biochem Mol Biol Int. </span>1998;<span class="ref-vol">46</span>:643–656.</span> [<a href="/pubmed/9844724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Mol+Biol+Int&amp;title=Quantification+of+prostaglandin+D+synthetase+in+cerebrospinal+fluid:+a+potential+marker+for+brain+tumor&amp;author=L+Saso&amp;author=MG+Leone&amp;author=C+Sorrentino&amp;volume=46&amp;publication_year=1998&amp;pages=643-656&amp;pmid=9844724&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C26">26. <span class="element-citation">Nagata N, Fujimori K, Okazaki I. De novo synthesis, uptake and proteolytic processing of lipocalin-type prostaglandin D synthase, beta-trace, in the kidneys. <span><span class="ref-journal">FEBS J. </span>2009;<span class="ref-vol">276</span>:7146–7158.</span>  <a href="http://dx.doi.org/10.1111/j.1742-4658.2009.07426.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.1742-4658.2009.07426.x</a>. [<a href="/pubmed/19878301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+J&amp;title=De+novo+synthesis,+uptake+and+proteolytic+processing+of+lipocalin-type+prostaglandin+D+synthase,+beta-trace,+in+the+kidneys&amp;author=N+Nagata&amp;author=K+Fujimori&amp;author=I+Okazaki&amp;volume=276&amp;publication_year=2009&amp;pages=7146-7158&amp;pmid=19878301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C27">27. <span class="element-citation">Vynckier LL, Flore KM, Delanghe SE. Urinary beta-trace protein as a new renal tubular marker. <span><span class="ref-journal">Clin Chem. </span>2009;<span class="ref-vol">55</span>:1241–1243.</span>  <a href="http://dx.doi.org/10.1373/clinchem.2008.119727" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1373/clinchem.2008.119727</a>. [<a href="/pubmed/19359535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Urinary+beta-trace+protein+as+a+new+renal+tubular+marker&amp;author=LL+Vynckier&amp;author=KM+Flore&amp;author=SE+Delanghe&amp;volume=55&amp;publication_year=2009&amp;pages=1241-1243&amp;pmid=19359535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C28">28. <span class="element-citation">Donadio C. Serum and urinary markers of early impairment of GFR in chronic kidney disease patients: diagnostic accuracy of urinary beta-trace protein. <span><span class="ref-journal">Am J Physiol Renal Physiol. </span>2010;<span class="ref-vol">299</span>:F1407–F1423.</span>  <a href="http://dx.doi.org/10.1152/ajprenal.00507.2009" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1152/ajprenal.00507.2009</a>. [<a href="/pubmed/20844024" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Renal+Physiol&amp;title=Serum+and+urinary+markers+of+early+impairment+of+GFR+in+chronic+kidney+disease+patients:+diagnostic+accuracy+of+urinary+beta-trace+protein&amp;author=C+Donadio&amp;volume=299&amp;publication_year=2010&amp;pages=F1407-F1423&amp;pmid=20844024&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C29">29. <span class="element-citation">Hoffmann A, Nimtz M, Conradt HS. Molecular characterization of beta-trace protein in human serum and urine: a potential diagnostic marker for renal diseases. <span><span class="ref-journal">Glycobiology. </span>1997;<span class="ref-vol">7</span>:499–506.</span>  <a href="http://dx.doi.org/10.1093/glycob/7.4.499" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/glycob/7.4.499</a>. [<a href="/pubmed/9184830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Molecular+characterization+of+beta-trace+protein+in+human+serum+and+urine:+a+potential+diagnostic+marker+for+renal+diseases&amp;author=A+Hoffmann&amp;author=M+Nimtz&amp;author=HS+Conradt&amp;volume=7&amp;publication_year=1997&amp;pages=499-506&amp;pmid=9184830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C30">30. <span class="element-citation">Bulger M, Groudine M. Enhancers: the abundance and function of regulatory sequences beyond promoters. <span><span class="ref-journal">Dev Biol. </span>2010;<span class="ref-vol">339</span>:250–257.</span>  <a href="http://dx.doi.org/10.1016/j.ydbio.2009.11.035" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.ydbio.2009.11.035</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3060611/">PMC free article</a>]</span> [<a href="/pubmed/20025863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Biol&amp;title=Enhancers:+the+abundance+and+function+of+regulatory+sequences+beyond+promoters&amp;author=M+Bulger&amp;author=M+Groudine&amp;volume=339&amp;publication_year=2010&amp;pages=250-257&amp;pmid=20025863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C31">31. <span class="element-citation">Heintzman ND, Ren B. Finding distal regulatory elements in the human genome. <span><span class="ref-journal">Curr Opin Genet Dev. </span>2009;<span class="ref-vol">19</span>:541–549.</span>  <a href="http://dx.doi.org/10.1016/j.gde.2009.09.006" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.gde.2009.09.006</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3321269/">PMC free article</a>]</span> [<a href="/pubmed/19854636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=Finding+distal+regulatory+elements+in+the+human+genome&amp;author=ND+Heintzman&amp;author=B+Ren&amp;volume=19&amp;publication_year=2009&amp;pages=541-549&amp;pmid=19854636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C32">32. <span class="element-citation">Maston GA, Evans SK, Green MR. Transcriptional regulatory elements in the human genome. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2006;<span class="ref-vol">7</span>:29–59.</span>  <a href="http://dx.doi.org/10.1146/annurev.genom.7.080505.115623" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1146/annurev.genom.7.080505.115623</a>. [<a href="/pubmed/16719718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Transcriptional+regulatory+elements+in+the+human+genome&amp;author=GA+Maston&amp;author=SK+Evans&amp;author=MR+Green&amp;volume=7&amp;publication_year=2006&amp;pages=29-59&amp;pmid=16719718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C33">33. <span class="element-citation">Filler G, Priem F, Lepage N. Beta-trace protein, cystatin C, beta(2)-microglobulin, and creatinine compared for detecting impaired glomerular filtration rates in children. <span><span class="ref-journal">Clin Chem. </span>2002;<span class="ref-vol">48</span>:729–736.</span> [<a href="/pubmed/11978599" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Beta-trace+protein,+cystatin+C,+beta(2)-microglobulin,+and+creatinine+compared+for+detecting+impaired+glomerular+filtration+rates+in+children&amp;author=G+Filler&amp;author=F+Priem&amp;author=N+Lepage&amp;volume=48&amp;publication_year=2002&amp;pages=729-736&amp;pmid=11978599&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C34">34. <span class="element-citation">Solichova P, Novackova L, Ochmanova R. Assessment of serum beta-trace protein (BTP) measurement in the prediction of glomerular filtration rate. Comparison with serum cystatin C. <span><span class="ref-journal">Biomed Pap Med Fac Univ Palacky Olomouc Czech Repub. </span>2006;<span class="ref-vol">150</span>:83–84.</span>  <a href="http://dx.doi.org/10.5507/bp.2006.009" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.5507/bp.2006.009</a>. [<a href="/pubmed/16936905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biomed+Pap+Med+Fac+Univ+Palacky+Olomouc+Czech+Repub&amp;title=Assessment+of+serum+beta-trace+protein+(BTP)+measurement+in+the+prediction+of+glomerular+filtration+rate.+Comparison+with+serum+cystatin+C&amp;author=P+Solichova&amp;author=L+Novackova&amp;author=R+Ochmanova&amp;volume=150&amp;publication_year=2006&amp;pages=83-84&amp;pmid=16936905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C35">35. <span class="element-citation">White CA, Akbari A, Doucette S. Estimating GFR using serum beta trace protein: accuracy and validation in kidney transplant and pediatric populations. <span><span class="ref-journal">Kidney Int. </span>2009;<span class="ref-vol">76</span>:784–791.</span>  <a href="http://dx.doi.org/10.1038/ki.2009.262" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ki.2009.262</a>. [<a href="/pubmed/19625992" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Estimating+GFR+using+serum+beta+trace+protein:+accuracy+and+validation+in+kidney+transplant+and+pediatric+populations&amp;author=CA+White&amp;author=A+Akbari&amp;author=S+Doucette&amp;volume=76&amp;publication_year=2009&amp;pages=784-791&amp;pmid=19625992&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="GFS591C36">36. <span class="element-citation">Spanaus KS, Kollerits B, Ritz E. Serum creatinine, cystatin C, and beta-trace protein in diagnostic staging and predicting progression of primary nondiabetic chronic kidney disease. <span><span class="ref-journal">Clin Chem. </span>2010;<span class="ref-vol">56</span>:740–749.</span>  <a href="http://dx.doi.org/10.1373/clinchem.2009.138826" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1373/clinchem.2009.138826</a>. [<a href="/pubmed/20224047" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Serum+creatinine,+cystatin+C,+and+beta-trace+protein+in+diagnostic+staging+and+predicting+progression+of+primary+nondiabetic+chronic+kidney+disease&amp;author=KS+Spanaus&amp;author=B+Kollerits&amp;author=E+Ritz&amp;volume=56&amp;publication_year=2010&amp;pages=740-749&amp;pmid=20224047&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3685304&amp;issue-id=224130&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140633004018432title">Abstract</a></li><li><a href="#gfs591_s1title">INTRODUCTION</a></li><li><a href="#gfs591_s2title">METHODS</a></li><li><a href="#gfs591_s3title">RESULTS</a></li><li><a href="#gfs591_s4title">DISCUSSION</a></li><li><a href="#gfs591_s5title">SUPPLEMENTARY DATA</a></li><li><a href="#gfs591_s6title">FUNDING</a></li><li><a href="#gfs591_s7title">CONFLICT OF INTEREST STATEMENT</a></li><li><a href="#__sec5title">Supplementary Material</a></li><li><a href="#idm140633085115632title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140633004475440title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Nephrology Dialysis Transplantation</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3685304/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3685304/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3685304/pdf/gfs591.pdf">PDF (330K)</a></li> | <li><a href="#" data-citationid="PMC3685304" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3685304%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3685304%2F&amp;text=Genome-wide%20significant%20locus%20of%20beta-trace%20protein%2C%20a%20novel%20kidney%20function%20biomarker%2C%20identified%20in%20European%20and%20African%20Americans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3685304%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23328707" data-aiid="3685304" data-aid="3685304" data-iid="224130" data-domainid="767" data-domain="ndt" data-accid="PMC3685304" data-md5="560d1cc9df2f09a00bc771ccd80e0ab6"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:12:21-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D3E7BDD54E9C10000000006C4049A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3685304%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ndt&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3685304/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>