<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3517659/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of IL6R and chromosome 11q13.5 as risk loci for asthma</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517659/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Lancet"><meta name="citation_title" content="Identification of IL6R and chromosome 11q13.5 as risk loci for asthma"><meta name="citation_authors" content="Manuel A.R. Ferreira, Melanie C. Matheson, David L. Duffy, Guy B. Marks, Jennie Hui, Peter Le Souëf, Patrick Danoy, Svetlana Baltic, Dale R. Nyholt, Mark Jenkins, Catherine Hayden, Gonneke Willemsen, Wei Ang, Mikko Kuokkanen, John Beilby, Faang Cheah, Eco J. C. de Geus, Adaikalavan Ramasamy, Sailaja Vedantam, Veikko Salomaa, Pamela A. Madden, Andrew C. Heath, John L. Hopper, Peter M. Visscher, Bill Musk, Stephen R. Leeder, Marjo-Riitta Jarvelin, Craig Pennell, Dorret I. Boomsma, Joel Hirschhorn, Haydn Walters, Nicholas G. Martin, Alan James, Graham Jones, Michael J. Abramson, Colin F. Robertson, Shyamali C. Dharmage, Matthew A. Brown, Grant W. Montgomery, Philip J. Thompson"><meta name="citation_date" content="10 September 2011"><meta name="citation_issue" content="9795"><meta name="citation_volume" content="378"><meta name="citation_firstpage" content="1006"><meta name="citation_doi" content="10.1016/S0140-6736(11)60874-X"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3517659/?report=abstract"><meta name="citation_pmid" content="21907864"><meta name="DC.Title" content="Identification of IL6R and chromosome 11q13.5 as risk loci for asthma"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Manuel A.R. Ferreira"><meta name="DC.Contributor" content="Melanie C. Matheson"><meta name="DC.Contributor" content="David L. Duffy"><meta name="DC.Contributor" content="Guy B. Marks"><meta name="DC.Contributor" content="Jennie Hui"><meta name="DC.Contributor" content="Peter Le Souëf"><meta name="DC.Contributor" content="Patrick Danoy"><meta name="DC.Contributor" content="Svetlana Baltic"><meta name="DC.Contributor" content="Dale R. Nyholt"><meta name="DC.Contributor" content="Mark Jenkins"><meta name="DC.Contributor" content="Catherine Hayden"><meta name="DC.Contributor" content="Gonneke Willemsen"><meta name="DC.Contributor" content="Wei Ang"><meta name="DC.Contributor" content="Mikko Kuokkanen"><meta name="DC.Contributor" content="John Beilby"><meta name="DC.Contributor" content="Faang Cheah"><meta name="DC.Contributor" content="Eco J. C. de Geus"><meta name="DC.Contributor" content="Adaikalavan Ramasamy"><meta name="DC.Contributor" content="Sailaja Vedantam"><meta name="DC.Contributor" content="Veikko Salomaa"><meta name="DC.Contributor" content="Pamela A. Madden"><meta name="DC.Contributor" content="Andrew C. Heath"><meta name="DC.Contributor" content="John L. Hopper"><meta name="DC.Contributor" content="Peter M. Visscher"><meta name="DC.Contributor" content="Bill Musk"><meta name="DC.Contributor" content="Stephen R. Leeder"><meta name="DC.Contributor" content="Marjo-Riitta Jarvelin"><meta name="DC.Contributor" content="Craig Pennell"><meta name="DC.Contributor" content="Dorret I. Boomsma"><meta name="DC.Contributor" content="Joel Hirschhorn"><meta name="DC.Contributor" content="Haydn Walters"><meta name="DC.Contributor" content="Nicholas G. Martin"><meta name="DC.Contributor" content="Alan James"><meta name="DC.Contributor" content="Graham Jones"><meta name="DC.Contributor" content="Michael J. Abramson"><meta name="DC.Contributor" content="Colin F. Robertson"><meta name="DC.Contributor" content="Shyamali C. Dharmage"><meta name="DC.Contributor" content="Matthew A. Brown"><meta name="DC.Contributor" content="Grant W. Montgomery"><meta name="DC.Contributor" content="Philip J. Thompson"><meta name="DC.Date" content="2011 Sep 10"><meta name="DC.Identifier" content="10.1016/S0140-6736(11)60874-X"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of IL6R and chromosome 11q13.5 as risk loci for asthma"><meta property="og:type" content="article"><meta property="og:description" content="We aimed to identify novel genetic variants affecting asthma risk, since these might provide novel insights into molecular mechanisms underlying asthma.We performed a genome-wide association study (GWAS) in 2,669 physician-diagnosed asthmatics and 4,528 ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517659/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3517659/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3517659/pdf/nihms-414524.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B6A67DD52E3C10000000003DD0278.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3517659%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3517659</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Lancet</a></span>. Author manuscript; available in PMC 2012 Dec 7.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21907864" target="pmc_ext" ref="reftype=publisher&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Lancet. 2011 Sep 10; 378(9795): 1006–1014. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2FS0140-6736(11)60874-X" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/S0140-6736(11)60874-X</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3517659</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS414524</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21907864">21907864</a></div></div></div></div><h1 class="content-title">Identification of <em>IL6R</em> and chromosome 11q13.5 as risk loci for asthma</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ferreira%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064585754816" co-class="co-affbox">Manuel A.R. Ferreira</a>,<sup>1</sup> <a href="/pubmed/?term=Matheson%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064595269456" co-class="co-affbox">Melanie C. Matheson</a>,<sup>2</sup> <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064595184480" co-class="co-affbox">David L. Duffy</a>,<sup>1</sup> <a href="/pubmed/?term=Marks%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064595182720" co-class="co-affbox">Guy B. Marks</a>,<sup>3</sup> <a href="/pubmed/?term=Hui%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064595180736" co-class="co-affbox">Jennie Hui</a>,<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Le%20Sou%26%23x000eb%3Bf%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064592756624" co-class="co-affbox">Peter Le Souëf</a>,<sup>8</sup> <a href="/pubmed/?term=Danoy%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064594479520" co-class="co-affbox">Patrick Danoy</a>,<sup>9</sup> <a href="/pubmed/?term=Baltic%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064594477536" co-class="co-affbox">Svetlana Baltic</a>,<sup>10</sup> <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064594475552" co-class="co-affbox">Dale R. Nyholt</a>,<sup>1</sup> <a href="/pubmed/?term=Jenkins%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064597903056" co-class="co-affbox">Mark Jenkins</a>,<sup>2</sup> <a href="/pubmed/?term=Hayden%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064597901072" co-class="co-affbox">Catherine Hayden</a>,<sup>8</sup> <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064630146992" co-class="co-affbox">Gonneke Willemsen</a>,<sup>11</sup> <a href="/pubmed/?term=Ang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064630145008" co-class="co-affbox">Wei Ang</a>,<sup>12</sup> <a href="/pubmed/?term=Kuokkanen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064630143024" co-class="co-affbox">Mikko Kuokkanen</a>,<sup>13</sup> <a href="/pubmed/?term=Beilby%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064588320256" co-class="co-affbox">John Beilby</a>,<sup>4,</sup><sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Cheah%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064597606272" co-class="co-affbox">Faang Cheah</a>,<sup>10</sup> <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064597604288" co-class="co-affbox">Eco J. C. de Geus</a>,<sup>11</sup> <a href="/pubmed/?term=Ramasamy%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064597602304" co-class="co-affbox">Adaikalavan Ramasamy</a>,<sup>14</sup> <a href="/pubmed/?term=Vedantam%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064590767184" co-class="co-affbox">Sailaja Vedantam</a>,<sup>15,</sup><sup>16</sup> <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064590764464" co-class="co-affbox">Veikko Salomaa</a>,<sup>13</sup> <a href="/pubmed/?term=Madden%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064592337680" co-class="co-affbox">Pamela A. Madden</a>,<sup>17</sup> <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064592335696" co-class="co-affbox">Andrew C. Heath</a>,<sup>17</sup> <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064592333712" co-class="co-affbox">John L. Hopper</a>,<sup>2</sup> <a href="/pubmed/?term=Visscher%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064594365760" co-class="co-affbox">Peter M. Visscher</a>,<sup>1</sup> <a href="/pubmed/?term=Musk%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064594363776" co-class="co-affbox">Bill Musk</a>,<sup>5,</sup><sup>7,</sup><sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Leeder%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064597016512" co-class="co-affbox">Stephen R. Leeder</a>,<sup>20</sup> <a href="/pubmed/?term=Jarvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064597014528" co-class="co-affbox">Marjo-Riitta Jarvelin</a>,<sup>21</sup> <a href="/pubmed/?term=Pennell%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064595161248" co-class="co-affbox">Craig Pennell</a>,<sup>12</sup> <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064595159264" co-class="co-affbox">Dorret I. Boomsma</a>,<sup>11</sup> <a href="/pubmed/?term=Hirschhorn%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064595157280" co-class="co-affbox">Joel Hirschhorn</a>,<sup>15,</sup><sup>16,</sup><sup>22</sup> <a href="/pubmed/?term=Walters%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064597945440" co-class="co-affbox">Haydn Walters</a>,<sup>23</sup> <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064591617424" co-class="co-affbox">Nicholas G. Martin</a>,<sup>1</sup> <a href="/pubmed/?term=James%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064591615440" co-class="co-affbox">Alan James</a>,<sup>7,</sup><sup>19,</sup><sup>24</sup> <a href="/pubmed/?term=Jones%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064585688576" co-class="co-affbox">Graham Jones</a>,<sup>25</sup> <a href="/pubmed/?term=Abramson%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064585686592" co-class="co-affbox">Michael J. Abramson</a>,<sup>26</sup> <a href="/pubmed/?term=Robertson%20CF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064585684608" co-class="co-affbox">Colin F. Robertson</a>,<sup>27</sup> <a href="/pubmed/?term=Dharmage%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064629493712" co-class="co-affbox">Shyamali C. Dharmage</a>,<sup>2</sup> <a href="/pubmed/?term=Brown%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064629491728" co-class="co-affbox">Matthew A. Brown</a>,<sup>9</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064629489744" co-class="co-affbox">Grant W. Montgomery</a>,<sup>1,</sup><sup>*</sup> and  <a href="/pubmed/?term=Thompson%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864" class="affpopup" co-rid="_co_idm140064590166176" co-class="co-affbox">Philip J. Thompson</a><sup>10,</sup><sup>*</sup>, for the Australian Asthma Genetics Consortium<sup>28</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140064585754816"><h3 class="no_margin">Manuel A.R. Ferreira</h3><p><sup>1</sup>The Queensland Institute of Medical Research, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Ferreira%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Manuel A.R. Ferreira</a></div></div><div id="_co_idm140064595269456"><h3 class="no_margin">Melanie C. Matheson</h3><p><sup>2</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, University of Melbourne, Melbourne, Australia. </p><div>Find articles by <a href="/pubmed/?term=Matheson%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Melanie C. Matheson</a></div></div><div id="_co_idm140064595184480"><h3 class="no_margin">David L. Duffy</h3><p><sup>1</sup>The Queensland Institute of Medical Research, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">David L. Duffy</a></div></div><div id="_co_idm140064595182720"><h3 class="no_margin">Guy B. Marks</h3><p><sup>3</sup>Woolcock Institute of Medical Research, University of Sydney, Sydney, Australia. </p><div>Find articles by <a href="/pubmed/?term=Marks%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Guy B. Marks</a></div></div><div id="_co_idm140064595180736"><h3 class="no_margin">Jennie Hui</h3><p><sup>4</sup>PathWest Laboratory Medicine of Western Australia (WA), Nedlands, Australia. </p><p><sup>5</sup>School of Population Health, The University of WA, Nedlands, Australia. </p><p><sup>6</sup>School of Pathology and Laboratory Medicine, The University of WA, Nedlands, Australia. </p><p><sup>7</sup>Busselton Population Medical Research Foundation, Sir Charles Gairdner Hospital, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Hui%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Jennie Hui</a></div></div><div id="_co_idm140064592756624"><h3 class="no_margin">Peter Le Souëf</h3><p><sup>8</sup>School of Paediatrics and Child Health, Princess Margaret Hospital for Children, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Le%20Sou%26%23x000eb%3Bf%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Peter Le Souëf</a></div></div><div id="_co_idm140064594479520"><h3 class="no_margin">Patrick Danoy</h3><p><sup>9</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Danoy%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Patrick Danoy</a></div></div><div id="_co_idm140064594477536"><h3 class="no_margin">Svetlana Baltic</h3><p><sup>10</sup>Lung Institute of WA and Centre for Asthma, Allergy and Respiratory Research, University of WA, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Baltic%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Svetlana Baltic</a></div></div><div id="_co_idm140064594475552"><h3 class="no_margin">Dale R. Nyholt</h3><p><sup>1</sup>The Queensland Institute of Medical Research, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Dale R. Nyholt</a></div></div><div id="_co_idm140064597903056"><h3 class="no_margin">Mark Jenkins</h3><p><sup>2</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, University of Melbourne, Melbourne, Australia. </p><div>Find articles by <a href="/pubmed/?term=Jenkins%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Mark Jenkins</a></div></div><div id="_co_idm140064597901072"><h3 class="no_margin">Catherine Hayden</h3><p><sup>8</sup>School of Paediatrics and Child Health, Princess Margaret Hospital for Children, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Hayden%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Catherine Hayden</a></div></div><div id="_co_idm140064630146992"><h3 class="no_margin">Gonneke Willemsen</h3><p><sup>11</sup>Netherlands Twin Register, EMGO &amp; NCA Institute, Department of Biological Psychology, VU University, Amsterdam, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Gonneke Willemsen</a></div></div><div id="_co_idm140064630145008"><h3 class="no_margin">Wei Ang</h3><p><sup>12</sup>Telethon Institute for Child Health Research, Centre for Child Health Research, University of WA, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Ang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Wei Ang</a></div></div><div id="_co_idm140064630143024"><h3 class="no_margin">Mikko Kuokkanen</h3><p><sup>13</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Finland. </p><div>Find articles by <a href="/pubmed/?term=Kuokkanen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Mikko Kuokkanen</a></div></div><div id="_co_idm140064588320256"><h3 class="no_margin">John Beilby</h3><p><sup>4</sup>PathWest Laboratory Medicine of Western Australia (WA), Nedlands, Australia. </p><p><sup>6</sup>School of Pathology and Laboratory Medicine, The University of WA, Nedlands, Australia. </p><p><sup>7</sup>Busselton Population Medical Research Foundation, Sir Charles Gairdner Hospital, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Beilby%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">John Beilby</a></div></div><div id="_co_idm140064597606272"><h3 class="no_margin">Faang Cheah</h3><p><sup>10</sup>Lung Institute of WA and Centre for Asthma, Allergy and Respiratory Research, University of WA, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Cheah%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Faang Cheah</a></div></div><div id="_co_idm140064597604288"><h3 class="no_margin">Eco J. C. de Geus</h3><p><sup>11</sup>Netherlands Twin Register, EMGO &amp; NCA Institute, Department of Biological Psychology, VU University, Amsterdam, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Eco J. C. de Geus</a></div></div><div id="_co_idm140064597602304"><h3 class="no_margin">Adaikalavan Ramasamy</h3><p><sup>14</sup>Respiratory Epidemiology and Public Health, Imperial College, London, United Kingdom. </p><div>Find articles by <a href="/pubmed/?term=Ramasamy%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Adaikalavan Ramasamy</a></div></div><div id="_co_idm140064590767184"><h3 class="no_margin">Sailaja Vedantam</h3><p><sup>15</sup>Divisions of Genetics and Endocrinology, Children’s Hospital, Boston, United States. </p><p><sup>16</sup>Broad Institute, Cambridge, United States. </p><div>Find articles by <a href="/pubmed/?term=Vedantam%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Sailaja Vedantam</a></div></div><div id="_co_idm140064590764464"><h3 class="no_margin">Veikko Salomaa</h3><p><sup>13</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Finland. </p><div>Find articles by <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Veikko Salomaa</a></div></div><div id="_co_idm140064592337680"><h3 class="no_margin">Pamela A. Madden</h3><p><sup>17</sup>Washington University School of Medicine, St Louis, United States </p><div>Find articles by <a href="/pubmed/?term=Madden%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Pamela A. Madden</a></div></div><div id="_co_idm140064592335696"><h3 class="no_margin">Andrew C. Heath</h3><p><sup>17</sup>Washington University School of Medicine, St Louis, United States </p><div>Find articles by <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Andrew C. Heath</a></div></div><div id="_co_idm140064592333712"><h3 class="no_margin">John L. Hopper</h3><p><sup>2</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, University of Melbourne, Melbourne, Australia. </p><div>Find articles by <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">John L. Hopper</a></div></div><div id="_co_idm140064594365760"><h3 class="no_margin">Peter M. Visscher</h3><p><sup>1</sup>The Queensland Institute of Medical Research, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Visscher%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Peter M. Visscher</a></div></div><div id="_co_idm140064594363776"><h3 class="no_margin">Bill Musk</h3><p><sup>5</sup>School of Population Health, The University of WA, Nedlands, Australia. </p><p><sup>7</sup>Busselton Population Medical Research Foundation, Sir Charles Gairdner Hospital, Perth, Australia. </p><p><sup>18</sup>Department of Respiratory Medicine, Sir Charles Gairdner Hospital, Perth, Australia </p><p><sup>19</sup>School of Medicine and Pharmacology, University of WA, Nedlands, Australia </p><div>Find articles by <a href="/pubmed/?term=Musk%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Bill Musk</a></div></div><div id="_co_idm140064597016512"><h3 class="no_margin">Stephen R. Leeder</h3><p><sup>20</sup>Australian Health Policy Institute, University of Sydney, Sydney, Australia </p><div>Find articles by <a href="/pubmed/?term=Leeder%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Stephen R. Leeder</a></div></div><div id="_co_idm140064597014528"><h3 class="no_margin">Marjo-Riitta Jarvelin</h3><p><sup>21</sup>Department of Epidemiology and Biostatistics, Imperial College, London, United Kingdom. </p><div>Find articles by <a href="/pubmed/?term=Jarvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Marjo-Riitta Jarvelin</a></div></div><div id="_co_idm140064595161248"><h3 class="no_margin">Craig Pennell</h3><p><sup>12</sup>Telethon Institute for Child Health Research, Centre for Child Health Research, University of WA, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Pennell%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Craig Pennell</a></div></div><div id="_co_idm140064595159264"><h3 class="no_margin">Dorret I. Boomsma</h3><p><sup>11</sup>Netherlands Twin Register, EMGO &amp; NCA Institute, Department of Biological Psychology, VU University, Amsterdam, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Dorret I. Boomsma</a></div></div><div id="_co_idm140064595157280"><h3 class="no_margin">Joel Hirschhorn</h3><p><sup>15</sup>Divisions of Genetics and Endocrinology, Children’s Hospital, Boston, United States. </p><p><sup>16</sup>Broad Institute, Cambridge, United States. </p><p><sup>22</sup>Department of Genetics, Harvard Medical School, Boston, United States. </p><div>Find articles by <a href="/pubmed/?term=Hirschhorn%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Joel Hirschhorn</a></div></div><div id="_co_idm140064597945440"><h3 class="no_margin">Haydn Walters</h3><p><sup>23</sup>Menzies Research Institute, Hobart, Australia. </p><div>Find articles by <a href="/pubmed/?term=Walters%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Haydn Walters</a></div></div><div id="_co_idm140064591617424"><h3 class="no_margin">Nicholas G. Martin</h3><p><sup>1</sup>The Queensland Institute of Medical Research, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Nicholas G. Martin</a></div></div><div id="_co_idm140064591615440"><h3 class="no_margin">Alan James</h3><p><sup>7</sup>Busselton Population Medical Research Foundation, Sir Charles Gairdner Hospital, Perth, Australia. </p><p><sup>19</sup>School of Medicine and Pharmacology, University of WA, Nedlands, Australia </p><p><sup>24</sup>Department of Pulmonary Physiology, West Australian Sleep Disorders Research Institute, Nedlands, Australia </p><div>Find articles by <a href="/pubmed/?term=James%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Alan James</a></div></div><div id="_co_idm140064585688576"><h3 class="no_margin">Graham Jones</h3><p><sup>25</sup>University of Western Sydney, Penrith, Australia </p><div>Find articles by <a href="/pubmed/?term=Jones%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Graham Jones</a></div></div><div id="_co_idm140064585686592"><h3 class="no_margin">Michael J. Abramson</h3><p><sup>26</sup>Department of Epidemiology&amp; Preventive Medicine, Monash University, Melbourne, Australia. </p><div>Find articles by <a href="/pubmed/?term=Abramson%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Michael J. Abramson</a></div></div><div id="_co_idm140064585684608"><h3 class="no_margin">Colin F. Robertson</h3><p><sup>27</sup>Respiratory Medicine, Murdoch Children’s Research Institute, Melbourne, Australia. </p><div>Find articles by <a href="/pubmed/?term=Robertson%20CF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Colin F. Robertson</a></div></div><div id="_co_idm140064629493712"><h3 class="no_margin">Shyamali C. Dharmage</h3><p><sup>2</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, University of Melbourne, Melbourne, Australia. </p><div>Find articles by <a href="/pubmed/?term=Dharmage%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Shyamali C. Dharmage</a></div></div><div id="_co_idm140064629491728"><h3 class="no_margin">Matthew A. Brown</h3><p><sup>9</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Brown%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Matthew A. Brown</a></div></div><div id="_co_idm140064629489744"><h3 class="no_margin">Grant W. Montgomery</h3><p><sup>1</sup>The Queensland Institute of Medical Research, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Grant W. Montgomery</a></div></div><div id="_co_idm140064590166176"><h3 class="no_margin">Philip J. Thompson</h3><p><sup>10</sup>Lung Institute of WA and Centre for Asthma, Allergy and Respiratory Research, University of WA, Perth, Australia. </p><div>Find articles by <a href="/pubmed/?term=Thompson%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21907864">Philip J. Thompson</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140064597543168_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140064597543168_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140064597543168_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>The Queensland Institute of Medical Research, Brisbane, Australia. </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, University of Melbourne, Melbourne, Australia. </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Woolcock Institute of Medical Research, University of Sydney, Sydney, Australia. </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>PathWest Laboratory Medicine of Western Australia (WA), Nedlands, Australia. </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>School of Population Health, The University of WA, Nedlands, Australia. </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>School of Pathology and Laboratory Medicine, The University of WA, Nedlands, Australia. </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Busselton Population Medical Research Foundation, Sir Charles Gairdner Hospital, Perth, Australia. </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>School of Paediatrics and Child Health, Princess Margaret Hospital for Children, Perth, Australia. </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Lung Institute of WA and Centre for Asthma, Allergy and Respiratory Research, University of WA, Perth, Australia. </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Netherlands Twin Register, EMGO &amp; NCA Institute, Department of Biological Psychology, VU University, Amsterdam, The Netherlands. </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Telethon Institute for Child Health Research, Centre for Child Health Research, University of WA, Perth, Australia. </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Finland. </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Respiratory Epidemiology and Public Health, Imperial College, London, United Kingdom. </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Divisions of Genetics and Endocrinology, Children’s Hospital, Boston, United States. </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Broad Institute, Cambridge, United States. </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Washington University School of Medicine, St Louis, United States </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Respiratory Medicine, Sir Charles Gairdner Hospital, Perth, Australia </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>School of Medicine and Pharmacology, University of WA, Nedlands, Australia </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Australian Health Policy Institute, University of Sydney, Sydney, Australia </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Department of Epidemiology and Biostatistics, Imperial College, London, United Kingdom. </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Genetics, Harvard Medical School, Boston, United States. </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Menzies Research Institute, Hobart, Australia. </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Department of Pulmonary Physiology, West Australian Sleep Disorders Research Institute, Nedlands, Australia </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>University of Western Sydney, Penrith, Australia </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Epidemiology&amp; Preventive Medicine, Monash University, Melbourne, Australia. </div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Respiratory Medicine, Murdoch Children’s Research Institute, Melbourne, Australia. </div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Collaborators are listed at the end of the paper. </div><div id="CR1">Correspondence should be addressed to: Dr Manuel A.R. Ferreira The Queensland Institute Of Medical Research 300 Herston Road Herston, QLD 4006 Australia <a href="mailto:dev@null" data-email="ua.ude.rmiq@arierref.leunam" class="oemail">ua.ude.rmiq@arierref.leunam</a> Fax: +61 7 3362 0101</div><div id="FN43"><sup>*</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140064597543168_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21907864" target="pmc_ext" ref="reftype=publisher&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Lancet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3517659/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary-Webappendix.</strong></div><div><a href="/pmc/articles/PMC3517659/bin/NIHMS414524-supplement-Supplementary-Webappendix.doc" data-ga-action="click_feat_suppl">NIHMS414524-supplement-Supplementary-Webappendix.doc</a><span style="color:gray"> (2.9M)</span></div><div class="small guid">GUID:&nbsp;59F058FC-6FCE-44D3-88AA-5CD2681B24CB</div></div></dd></dl></div><div id="idm140064590930720" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140064590930720title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P2" class="p p-first-last">We aimed to identify novel genetic variants affecting asthma risk, since these might provide novel insights into molecular mechanisms underlying asthma.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P3" class="p p-first-last">We performed a genome-wide association study (GWAS) in 2,669 physician-diagnosed asthmatics and 4,528 controls from Australia. Seven loci were prioritised for replication after combining our results with those from the GABRIEL consortium (n=26,475), and these were tested in an additional 25,358 independent samples from four in-silico cohorts. Quantitative multi-SNP scores of genetic load were constructed on the basis of results from the GABRIEL study and tested for association with asthma in our Australian GWAS dataset.</p></div><div id="S3" class="sec"><h3 id="S3title">Findings</h3><p id="P4" class="p p-first-last">Two loci were confirmed to associate with asthma risk in the replication cohorts and reached genome-wide significance in the combined analysis of all available studies (n=57,800): rs4129267 (OR=1.09, combined <em>P</em>=2.4×10<sup>−8</sup>) in the interleukin-6 receptor gene (<em>IL6R)</em> and rs7130588 (OR=1.09, <em>P</em>=1.8×10<sup>−8</sup>) on chromosome 11q13.5 near the leucine-rich repeat containing 32 gene (<em>LRRC32,</em> also known as <em>GARP</em>). The 11q13.5 locus was significantly associated with atopic status among asthmatics (OR = 1.33, <em>P</em> = 7×10<sup>−4</sup>), suggesting that it is a risk factor for allergic but not non-allergic asthma. Multi-SNP association results are consistent with a highly polygenic contribution to asthma risk, including loci with weak effects that may be shared with other immune-related diseases, such as <em>NDFIP1</em>, <em>HLA-B</em>, <em>LPP</em> and <em>BACH2</em>.</p></div><div id="S4" class="sec"><h3 id="S4title">Interpretation</h3><p id="P5" class="p p-first-last">The <em>IL6R</em> association further supports the hypothesis that cytokine signalling dysregulation affects asthma risk, and raises the possibility that an <em>IL6R</em> antagonist (tocilizumab) may be effective to treat the disease, perhaps in a genotype-dependent manner. Results for the 11q13.5 locus suggest that it directly increases the risk of allergic sensitisation which, in turn, increases the risk of subsequent development of asthma. Larger or more functionally focused studies are needed to characterise the many loci with modest effects that remain to be identified for asthma.</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Funding</h3><p id="P6" class="p p-first-last">A full list of funding sources appears at the end of the paper.</p></div></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Introduction</h2><p id="P7" class="p p-first">Eight loci were reported to associate with asthma risk with genome-wide significance, namely the locus containing <em>GSDMB</em>, <em>ORMDL3</em> and <em>GSDMA</em> locus on chromosome 17q21 (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513185">1</a>), <em>PDE4D</em> (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513189">2</a>), <em>DENND1B</em> (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513175">3</a>), the locus containing <em>IL1RL1</em> and <em>IL18R1</em> on chromosome 2q12.1 (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513158">4</a>), <em>HLA-DQ</em>, <em>IL33</em>, <em>IL2RB</em> and <em>SMAD3</em> (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340937">5</a>). Notably, these findings point to a genetically-linked dysregulation of cytokine signalling in asthma, and provide a number of specific targets for the development of novel biological therapies. They also implicate previously unsuspected risk loci, such as the 17q21 region; as our understanding of the biological mechanisms underlying these associations improves, novel insights into the pathophysiology of asthma are likely to emerge.</p><p id="P8">The risk variants identified to date explain only a small fraction of the disease heritability (&lt; 1% each), indicating that many more loci remain to be identified. Because of the proven success of genome-wide association studies (GWAS) to identify common risk variants (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513174">6</a>), further dissection of this uncharacterised component of disease risk through well powered genetic studies represents a unique opportunity to advance our knowledge of the mechanisms that trigger asthma.</p><p id="P9" class="p p-last">In this paper, we describe a series of genetic association analyses carried out to identify novel risk loci for asthma, including (1) a GWAS of physician-diagnosed asthma using data for 7,197 individuals of European descent from Australia; (2) a meta-analysis of these results with findings from 26,475 individuals studied by the GABRIEL consortium (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340952">5</a>); and (3) testing the most significant regions of association in a further 25,358 independent samples.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Methods</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Participants</h3><p id="P10" class="p p-first">We first carried out a GWAS of 7,197 individuals of European ancestry from Australia; throughout this paper, we refer to this analysis as the “Australian GWAS”. Participants were drawn from three cohorts (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 2-6): the Australian Asthma Genetics Consortium (AAGC) cohort (n=1,810); the Busselton Health Study cohort (n=1,230); and the Queensland Institute of Medical Research (QIMR) GWAS cohort (n=4,157). Patients were generally recruited between 1964 and 2010.</p><p id="P11">Of the 2,669 asthmatic patients, 759 (28%) were diagnosed through clinical examination and 1,910 (72%) reported a lifetime physician diagnosis of asthma in epidemiological questionnaires. With respect to disease onset, 1,438 (54%) subjects were classified as having childhood asthma (defined by an age-of-onset at or before age 16 years), 697 (26%) subjects with later onset asthma (age-of-onset after the age of 16 years) and 534 (20%) with unknown age-of-onset. 1,570 (59%) of asthmatics were atopic, as defined by a positive skin prick test (SPT) response to at least one common allergen; 1,444 (54%) had at least one first-degree relative with asthma; and 1,026 (38%) reported lifetime smoking (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 2-3).</p><p id="P12">The 4,528 controls included 2,701 (60%) individuals who were classified as asthma-free based on clinical examination (109 [2%]) or epidemiological questionnaires (2,592 [57%]). The remaining 1,827 (40%) individuals provided no information about their asthma status. As we show in the <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> (pp 7), including this group of asthma-unknown individuals in the analysis as controls improved power to detect a true genetic association. SPT information and lifetime smoking status was unavailable for most controls (3,903 [86%] and 3,822 [84%], respectively; <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 2-3).</p><p id="P13">Overall, the mean age of participants was 39 years (SD=18.5, range 2 to 92) and 3,986 (55%) were women. This dataset includes 4,259 samples that have not been previously included in any asthma GWAS, 1,708 that were included in Ferreira and colleagues, (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513131">7</a>) and 1,230 samples from the Busselton cohort included in the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340943">5</a>).</p><p id="P14">Next, to prioritise loci for replication, results from the Australian GWAS were combined with those published and made publicly available by the GABRIEL consortium (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340958">5</a>). After excluding overlapping samples between the two studies, the meta-analysis was based on results from 12,475 physician-diagnosed asthmatic patients and 19,967 controls.</p><p id="P15" class="p p-last">Lastly, the most significant regions of association were tested for replication in four additional cohorts (3,322 cases and 22,036 controls) that contributed in-silico results for analysis: the Western Australian Pregancy Cohort (Raine) study (654 asthmatic and 621 control patients); the QIMR follow-up cohort (602 asthmatic and 2,206 control patients), consisting of individuals unrelated to those included in the QIMR GWAS cohort; the Netherlands Twin Registry (NTR) study (350 asthmatic and 2,321 control patients); and the Analysis in Population-based Cohorts of Asthma Traits (APCAT) consortium (1,716 asthmatic and 16,888 control patients). The APCAT consortium included six population-based cohorts from Finland and the United States: the Helsinki Birth Cohort (123 asthmatic and 1,533 control patients), Health 2000 (153 asthmatic and 1,841 control patients), Finrisk (160 asthmatic and 1,705 control patients), the Northern Finland Birth Cohort 1966 (364 asthmatic and 3,502 control patients), the Young Finns Study (119 asthmatic and 1,844 control patients) and the Framingham Heart Study (797 asthmatic and 6,463 control patients). The <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> (pp 8-10) contains a detailed description of every cohort. All participants gave written informed consent and the study protocols were reviewed and approved by the appropriate review committees.</p></div><div id="S9" class="sec"><h3 id="S9title">Procedures</h3><p id="P16" class="p p-first">The Australian GWAS included data from 7,197 individuals of whom 5,523 (77%) were genotyped with Illumina 610K (Illumina, San Diego, CA, USA) array 1,674 (23%) with Illumina 370K array as part of four genotyping projects that are summarised in the <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> (pp 11): the AAGC (n=1,810), QIMR_610K (n=2,483), QIMR_370K (n=1,674) and Busselton (n=1,230) projects. The same single-nucleotide polymorphism (SNP) quality control filters were applied to each project individually, including the removal of SNPs with call rate &lt;95%, minor allele frequency (MAF) &lt; 0.01 and Hardy-Weinberg equilibrium test <em>P</em> &lt; 10<sup>−6</sup>. Autosomal SNPs passing quality control were then used to impute 7.8 million variants available from the combined 1000 genomes (60 individuals with northern and western European ancestry from the Centre d’Etude du Polmorphisme Humain collection (CEU), March 2010 release) and HapMap 3 (955 individuals from 11 populations, February 2009 release) reference panels using Impute2 (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513160">8</a>). The AAGC and QIMR_610K datasets were merged before imputation as both were genotyped with the same array, were available concurrently for analysis and had no evidence for systematic allele frequency differences between controls (genomic inflation factor λ = 1.014 for AAGC controls vs. QIMR_610K controls) nor between asthmatic patients (λ = 1.001 for AAGC cases vs. QIMR_610K cases). We nonetheless removed a small subset of 1,104 SNPs (0.2%) that had significant (<em>P</em> &lt; 0.001) allele frequency differences between the two datasets, as these probably indicated technical artifacts. The genomic inflation factor from a case-control association analysis in the resulting AAGC+QIMR_610K dataset (n=4,293) was 1.014, further indicating that batch effects between the AAGC and QIMR_610K datasets did not have a systematic effect on the results.</p><p id="P17" class="p p-last">Imputation and subsequent SNP quality control were carried out as three separate analyses, corresponding to the AAGC+QIMR_610K, QIMR_370K and Busselton datasets. The QIMR_370K and Busselton datasets were imputed separately because the former was genotyped using a different array and because the latter only became available for analysis at a later stage. After imputation, we excluded SNPs with low imputation accuracy (information &lt; 0.3), MAF &lt; 0.01 or Hardy-Weinberg equilibrium test <em>P</em> &lt; 10<sup>−6</sup>. To minimise the impact of potential dataset-specific technical artifacts, we also excluded from analysis SNPs with significant (<em>P</em> &lt; 0.001) allele frequency differences between the three imputation analysis groups (from case-case and control-control comparisons, as described above). After quality control, genotype data for 5.7 million common SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 12) were merged across the analysis datasets of AAGC+QIMR_610K, QIMR_370K and Busselton. All subjects included were confirmed to be unrelated and of European ancestry (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 6) through the analysis of genome-wide allele sharing. Comparable procedures were used for the replication cohorts (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 8-10).</p></div><div id="S10" class="sec"><h3 id="S10title">Statistical analyses</h3><p id="P18" class="p p-first">For the Australian GWAS, individual SNPs were tested for association with lifetime physician-diagnosed asthma using a Cochran-Mantel-Haenszel test as implemented in PLINK (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513153">9</a>), with three strata corresponding to the three imputation analysis groups described in the previous section. This analysis had adequate power (80% for <em>α</em> = 5×10<sup>−8</sup>) to detect loci with a genotype relative risk ranging from 1.23 (MAF = 0.50) to 2.23 (MAF = 0.01) (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 7). For imputed SNPs, we analysed best-guess (i.e. most likely) genotypes. The Breslow-Day test of homogeneity was applied to test for differences in SNP effects between the three groups.</p><p id="P19">To prioritise SNPs for follow-up amongst lower-ranked loci from the Australian GWAS, we performed a fixed-effects meta-analysis of our results (after excluding the 1,230 overlapping samples from Busselton) with those published and made available by the GABRIEL (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340947">5</a>), which were based on 10,365 asthmatic patients and 16,110 controls genotyped with the Illumina 610K array. We restricted our analysis to 421,334 SNPs that were available in all 36 GABRIEL cohorts, had no strong evidence for significant heterogeneous effects between the 36 cohorts (<em>P</em> &gt; 0.01) and which were tested in our study. We used the Cochran’s Q test to identify SNPs with significant heterogeneous effects between our study and the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340939">5</a>). No correction for genomic inflation of test statistics was applied to either set of results before the meta-analysis.</p><p id="P20">In the replication study, SNPs were tested in each cohort separately (logistic regression with cohort-specific adjustments) and then combined by performing a fixed-effects meta-analysis with METAL (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513148">10</a>).</p><p id="P21" class="p p-last">Lastly, we used a recently described (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513169">11</a>) approach for phenotype prediction from genome-wide SNP data to address the hypothesis that hundreds or thousands of loci with individual weak effects contribute to asthma risk. Briefly, we selected groups of SNPs based on their level of association with asthma in the GABRIEL (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340941">5</a>) analysis (for example, SNPs with a <em>P</em> ≤ 5×10<sup>−7</sup>) and created a quantitative score of genetic load based on these SNPs for every individual included in the Australian GWAS, after restricting the sample to 2,082 patients with asthma and 2,211 controls genotyped with the 610K array and that did not overlap with those included in the GABRIEL. This genetic score was calculated as the weighted sum of the number of reference alleles for each genotyped SNP, with the weight corresponding to the effect size (log of the odds ratio [OR]) estimated for that marker in the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340938">5</a>). We then used logistic regression to test the association between a specific genetic score and case-control asthma status in our study. The log of the odds ratio was selected as the weight (instead of, for example, simply counting the number of risk alleles) because it considers both the expected magnitude and direction of effect of an individual SNP. We used the odds ratio estimated in the GABRIEL study because currently that study represents the largest asthma GWAS done so far, hence providing the most accurate estimates of SNP effects available for asthma. We considered seven groups of SNPs based on arbitrary thresholds of significance in the GABRIEL study, from very strict to very liberal: <em>P</em> ≤ 5×10<sup>−7</sup>, 5×10<sup>−7</sup> &lt; <em>P</em> ≤ 10<sup>−4</sup>, 10<sup>−4</sup> &lt; <em>P</em> ≤ 0.01, 0.01 &lt; <em>P</em> ≤ 0.1, 0.1 &lt; <em>P</em> ≤ 0.2, 0.2 &lt; <em>P</em> ≤ 0.5 and 0.5 &lt; <em>P</em> ≤ 1. To facilitate interpretation, we restricted our analysis to directly genotyped SNPs that were in low linkage disequilibrium (<em>r</em><sup>2</sup> &lt; 0.1) with each other, using the clump routine implemented in PLINK (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513177">9</a>). Results were unchanged by including as covariates the first four components obtained from the multidimensional scaling analysis of identity-by-state allele sharing.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Role of the funding source</h3><p id="P22" class="p p-first-last">The sponsors of the study had no role in the study design, data collection, data analysis, data interpretation, or writing of the report. The corresponding author had full access to all the data in the study and had final responsibility for the decision to submit for publication.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Results</h2><p id="P23" class="p p-first">To identify novel SNPs associated with asthma risk, we first performed a GWAS in 2,669 physician-diagnosed asthmatic patients and 4,528 controls from Australia. We recognise that with this sample size, power was only adequate (≥ 80%) at the genome-wide level (<em>α</em> = 5×10<sup>−8</sup>) to detect loci with strong effects (e.g., an OR ≥ 1.25 for an allele frequency of 0.40). The genomic inflation factor confirmed that population substructure or other potential technical artifacts did not have a systematic impact on the results (<em>λ</em>=1.014; Q-Q plot in the <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 13).</p><p id="P24">In the Australian GWAS, five independent SNPs (linkage disequilibrium <em>r</em><sup>2</sup> &lt; 0.1) were associated with asthma risk at a pre-defined cut-off of <em>P</em> ≤ 5×10<sup>−6</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 14). Among these regions were two loci previously reported (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513161">1</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513152">4</a>) to associate with asthma – chromosome 17q21 near <em>GSDMA</em> (rs8071050, OR = 1.27, <em>P</em> = 6.3×10<sup>−10</sup>) and 5q22.1 near <em>WDR36</em> (rs1438673, OR = 0.84, <em>P</em> = 3.2×10<sup>−6</sup>) – and three novel loci: chromosomes 12q24.31 near <em>CABP1</em> (chr12:119562269, OR = 1.85<em>, P</em> = 8.3×10<sup>−7</sup>), 16q24.1 in <em>KCNG4</em> (rs7196274, OR = 0.81, <em>P</em> = 1.3×10<sup>−6</sup>) and 8q22.1 in <em>CDH17</em> (rs11776675, OR = 1.21, <em>P</em> = 2.7×10<sup>−6</sup>).</p><p id="P25">The 12q24.31 locus corresponds to an uncommon variant (chr12:119562269: MAF = 0.02) from the 1000 Genomes Project imputed in our data with modest confidence (imputation information of 0.86 for the 610K array and 0.52 for the 370K array). However, there was no supporting evidence for association from neighbouring markers (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 15) nor from haplotype analyses performed after exclusion of the chr12:119562269 variant (data not shown), suggesting that this association probably represents a technical artifact. The 16q24.1 and 8q22.1 associations were supported by multiple markers (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 15) and so were selected for follow-up in an independent replication panel, as described below.</p><p id="P26">Next, to prioritize additional loci for follow-up, we performed a fixed-effects meta-analysis of our results with those published and made publicly available by the GABRIEL consortium (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340949">5</a>). After exclusion of overlapping samples, the combined analysis was based on results from 12,475 asthmatic patients and 19,967 controls genotyped at 421,334 autosomal SNPs. The genomic inflation factor for this analysis was 1.068 (Q-Q plot in the <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 16); only ~1% of SNPs had a Cochran’s Q test <em>P</em> &lt; 0.01, indicating that SNP effects were largely homogeneous between the two studies.</p><p id="P27">19 independent (<em>r</em><sup>2</sup> &lt; 0.1) SNPs had a meta-analysis association <em>P</em> ≤ 5×10<sup>−6</sup> (listed in the <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a>, pp 17). Of these, 14 were not considered for further analysis because they were not associated with asthma in the Australian GWAS (n=2) or because they were located in regions previously reported to associate with asthma risk (n=12). The remaining five SNPs were nominally significant (<em>P</em> ≤ 0.05) in our dataset and were located in novel regions for asthma; as such, they were also selected for follow-up (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a>, pp 17).</p><p id="P28">Thus, in total, we identified seven putative novel asthma risk loci (<em>P</em> ≤ 5×10<sup>−6</sup>) that we sought to follow-up in an independent panel of 3,322 asthmatics and 22,036 controls previously genotyped as part of four studies (APCAT, RAINE, QIMR and NTR). Power to replicate each individual SNP association at a Bonferroni-corrected threshold (<em>α</em> ≤ 0.05/7=0.007) ranged between 64% and 100%.</p><p id="P29">Follow-up analyses replicated at <em>P</em> ≤ 0.007, the association with one SNP located in the interleukin-6 receptor (<em>IL6R)</em> gene on chromosome 1q21.3 (rs4129267, uncorrected <em>P</em> = 0.0033; <a href="/pmc/articles/PMC3517659/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). In the overall analysis of the discovery and follow-up panels of all the studies (15,797 asthmatic patients and 42,003 controls), this variant was highly associated with asthma risk (OR = 1.09, <em>P</em> = 2.3×10<sup>−8</sup>; <a href="/pmc/articles/PMC3517659/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure1A</span></a> ). A second SNP, rs7130588 on chromosome 11q13.5, replicated less strongly in the follow-up panels (combined <em>P</em> = 0.0328) but reached genome-wide significance in the overall analysis of all samples (OR = 1.09, <em>P</em> = 1.8×10<sup>−8</sup>; <a href="/pmc/articles/PMC3517659/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC3517659/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>). The remaining five regions of association (<em>PCDH20</em>, <em>PRKG1</em>, <em>KCNG4, IGHMBP2</em>, <em>CDH17</em>) were not significantly associated with asthma in the follow-up study (<em>P</em> &gt; 0.05; <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 19-20). Thus, genome-wide association analyses followed by replication identified variants in <em>IL6R</em> and on chromosome 11q13.5 as novel risk loci for asthma.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3517659/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140064589154384" class="figure"><a href="/pmc/articles/PMC3517659/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3517659_nihms-414524-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-414524-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3517659/bin/nihms-414524-f0001.jpg"></a></div><div id="largeobj_idm140064589154384" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3517659/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3517659/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional association plots for chromosomes 1q21.3 and 11q13.5</strong><p id="__p4">Association results (−log<sub>10</sub><em>P</em>-value, <em>y</em>-axis) from a meta-analysis of the Australian GWAS and the GABRIEL (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340945">5</a>) studies for the 1q21.3 (A) and 11q13.5 (B) loci, focusing on a subset of SNPs genotyped in both studies. Results for rs4129267 (<em>P</em> = 2.0×10<sup>−6</sup>) and rs7130588 (<em>P</em> = 1.2×10<sup>−7</sup>) in this analysis are represented by the blue triangle in each plot. The orange triangle corresponds to results for rs4129267 (<em>P</em> = 2.3×10<sup>−8</sup>) and rs7130588 (<em>P</em> = 1.8×10<sup>−8</sup>) in the meta-analysis of our study, the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340946">5</a>) and the follow-up panels (cf. <a href="/pmc/articles/PMC3517659/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), representing 15,797 asthmatic patients and 42,003 controls. The recombination rate (second <em>y</em>-axis) is plotted in light blue and is based on the CEU HapMap population. Exons for every gene are represented by vertical bars. Regional association plots for both loci in the Australian GWAS are presented in the <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> (pp 18). GWAS=genome-wide association studies. SNP=single-nucleotide polymorphism. CEU=individuals from northern and western European ancestry from the Centre d’Etude du Polymorphisme Humain.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">Association results for two variants in IL6R and chromosome 11q13.5 in the discovery, follow-up and overall analyses.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140064596799632" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Panel</th><th rowspan="1" colspan="1" valign="middle" align="center">Cases<br>(n)</th><th rowspan="1" colspan="1" valign="middle" align="center">Controls<br>(n)</th><th colspan="4" rowspan="1" valign="middle" align="center"><em>IL6R</em> (rs4129267, T)</th><th colspan="4" rowspan="1" valign="middle" align="center">11q13.5 (rs7130588, G)</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">MAF</th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">Association<br><em>P</em>-value</th><th rowspan="1" colspan="1" valign="middle" align="center">Heterogeneity test<br><em>P</em>-value <sup>a</sup><br>(<em>k</em>, <em>l</em><sup>2</sup> , 95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF</th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">Association<br><em>P</em>-value</th><th rowspan="1" colspan="1" valign="middle" align="center">Heterogeneity test<br><em>P</em>-value <sup>a</sup><br>(<em>k</em>, <em>l</em><sup>2</sup>, 95% CI)</th></tr></thead><tbody><tr><td colspan="11" rowspan="1" valign="middle" align="left">
<em>Discovery panels</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Australian GWAS <sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">2,110</td><td rowspan="1" colspan="1" valign="middle" align="center">3,857</td><td rowspan="1" colspan="1" valign="middle" align="center">0.41</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09 (1.01-1.18)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0372</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0417 (2; 76, 0-100)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.36</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12 (1.03-1.21)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0111</td><td rowspan="1" colspan="1" valign="middle" align="center">0.3523 (2; 0, 0-100)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> GABRIEL <sup>d</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">10,365</td><td rowspan="1" colspan="1" valign="middle" align="center">16,110</td><td rowspan="1" colspan="1" valign="middle" align="center">0.37</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09 (1.05-1.13)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.9×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.1734 (36; 18, 0-46)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.34</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10 (1.06-1.14)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.2×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.1602 (36; 19, 0-47)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Combined <sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">12,475</td><td rowspan="1" colspan="1" valign="middle" align="center">19,967</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09 (1.06-1.13)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.0×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.1839 (38; 17, 0-44)</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10 (1.07-1.14)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.2×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.1894 (38; 17, 0-44)</td></tr><tr><td colspan="11" rowspan="1" valign="middle" align="left">
<em>Follow-up panels</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> APCAT</td><td rowspan="1" colspan="1" valign="middle" align="center">1,716</td><td rowspan="1" colspan="1" valign="middle" align="center">16,888</td><td rowspan="1" colspan="1" valign="middle" align="center">0.35</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08 (1.00-1.15)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0671</td><td rowspan="1" colspan="1" valign="middle" align="center">0.6792 (3; 0, 0-90)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.31</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08 (1.00-1.16)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0749</td><td rowspan="1" colspan="1" valign="middle" align="center">0.3611 (3; 2, 0-90)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Raine</td><td rowspan="1" colspan="1" valign="middle" align="center">654</td><td rowspan="1" colspan="1" valign="middle" align="center">621</td><td rowspan="1" colspan="1" valign="middle" align="center">0.39</td><td rowspan="1" colspan="1" valign="middle" align="center">1.07 (0.92-1.25)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.4005</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">0.36</td><td rowspan="1" colspan="1" valign="middle" align="center">0.96 (0.82-1.12)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.6013</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> QIMR</td><td rowspan="1" colspan="1" valign="middle" align="center">602</td><td rowspan="1" colspan="1" valign="middle" align="center">2,206</td><td rowspan="1" colspan="1" valign="middle" align="center">0.40</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10 (0.97-1.23)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.1458</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">0.35</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15 (1.02-1.28)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0343</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> NTR</td><td rowspan="1" colspan="1" valign="middle" align="center">350</td><td rowspan="1" colspan="1" valign="middle" align="center">2,321</td><td rowspan="1" colspan="1" valign="middle" align="center">0.39</td><td rowspan="1" colspan="1" valign="middle" align="center">1.16 (1.00-1.32)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0723</td><td rowspan="1" colspan="1" valign="middle" align="center">0.2040 (2; 38, 0-100)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.37</td><td rowspan="1" colspan="1" valign="middle" align="center">1.01 (0.85-1.18)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.8685</td><td rowspan="1" colspan="1" valign="middle" align="center">0.7099 (2; 0, 0-100)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> Combined <sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">3,322</td><td rowspan="1" colspan="1" valign="middle" align="center">22,036</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09 (1.03-1.15)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0033</td><td rowspan="1" colspan="1" valign="middle" align="center">0.7911 (7; 0, 0-71)</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.07 (1.01-1.13)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0328</td><td rowspan="1" colspan="1" valign="middle" align="center">0.4630 (7; 0, 0-71)</td></tr><tr><td colspan="11" rowspan="1" valign="middle" align="left">
<em>Overall</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> All samples <sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">15,797</td><td rowspan="1" colspan="1" valign="middle" align="center">42,003</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09 (1.06-1.12)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.3251 (45; 8, 0-36)</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09 (1.06-1.13)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.8×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.2184 (45; 14, 0-41)</td></tr></tbody></table></div><div id="largeobj_idm140064596799632" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3517659/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P97" class="p p-first-last">Abbreviations: MAF, minor allele frequency. OR, odds ratio. CI, confidence intervals.</p></div><div id="TFN2"><p id="P98" class="p p-first-last">MAF is reported for non-asthmatic individuals; for the GABRIEL (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340950">5</a>), this corresponds to the mean MAF across all 36 individual studies. Both SNPs were directly genotyped in the discovery panels, APCAT (except FHS cohort, imputed with <em>r<sup>2</sup></em> &gt; 0.90), RAINE and QIMR studies; both were imputed with high confidence (information &gt; 0.95) in NTR.</p></div><div id="TFN3"><sup>a</sup>Results for a test of heterogeneity (Breslow-Day test for the Australian GWAS and NTR analyses; Cochran’s Q test for all other analyses) across studies are provided for panels that incorporated multiple cohorts.</div><div id="TFN4"><sup>b</sup>Samples that overlapped with the GABRIEL (n=1,230) were removed in this analysis.</div><div id="TFN5"><sup>c</sup>For the combined analysis, the association <em>P</em>-value corresponds to a fixed-effects meta-analysis of the results from the individual panels. Results remained unchanged under a random-effects model.</div><div id="TFN6"><sup>d</sup>Results from the GABRIEL consortium (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340957">5</a>) were obtained from the website of the Centre National de Génotypage. The MAF of each SNP corresponds to the mean MAF across all 36 individual studies.</div></div></div><p id="P30">To further characterise the association between the <em>IL6R</em> and 11q13.5 variants and asthma risk, we tested each locus for association with nine asthma subphenotypes measured in up to 2,669 asthmatic individuals (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 21). The rs7130588:G predisposing variant on 11q13.5 was more common in atopic asthmatic patients (defined by a positive skin prick test to at least one common allergen) than non-atopic asthmatic patients (G allele frequency 38% vs. 32%, <em>P</em> = 0.0007; <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 21-22). Consistent with this result, there was no evidence for an increased risk of asthma associated with this allele when considering only non-atopic individuals (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 22); however, given the modest sample size for this secondary analysis, these findings require confirmation by independent studies.</p><p id="P31">We also sought to validate the genome-wide significant associations reported by the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340959">5</a>), Sleiman et al. (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513192">3</a>) and Himes et al. (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513162">2</a>). We replicated (same SNP, <em>P</em> ≤ 0.05 and same direction of effect) the association with <em>GSDMB</em> (and <em>GSDMA</em>), <em>IL18R1</em>, <em>IL33</em> and <em>IL2RB</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 23). A consistent but non-significant effect was observed for <em>HLA-DQ</em> and <em>SMAD3</em>. There was no support for an association with <em>PDE4D</em> or <em>DENND1B</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 23-24).</p><p id="P32">Next, we investigated the hypothesis that hundreds or potentially thousands of common variants with weak effects influence asthma risk. A multi-SNP score computed based on data for the ten most associated loci reported in the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340956">5</a>) was significantly associated (<em>P</em> = 8.2×10<sup>−15</sup>) with asthma status in our study (<a href="/pmc/articles/PMC3517659/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Multi-SNP scores based on loci with less remarkable levels of association with asthma in the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340954">5</a>) were also associated with asthma status in our study. For example, a score computed based on 2,520 largely independent SNPs (<em>r</em><sup>2</sup> &lt; 0.1) that had individual <em>P</em>-values between 10<sup>−4</sup> and 0.01 in the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340955">5</a>) significantly associated with asthma case-control status (<em>P</em> = 1.2×10<sup>−4</sup>), indicating that many of these SNPs probably represent genuine asthma risk loci with modest effects. We stress, however, that these results have little clinical relevance, as the multi-SNP scores tested had very poor discriminative ability, with values for the area under the receiver operator characteristic curve (AUC) not exceeding 0.576 (<a href="/pmc/articles/PMC3517659/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3517659/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140064594062976" class="figure"><a href="/pmc/articles/PMC3517659/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3517659_nihms-414524-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-414524-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3517659/bin/nihms-414524-f0002.jpg"></a></div><div id="largeobj_idm140064594062976" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3517659/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3517659/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Multi-SNP prediction of asthma case-control status</strong><p id="__p5">Quantitative scores of genetic load were computed using information for seven non-overlapping groups of independent (<em>r</em><sup>2</sup>&lt;0.1, except rs2305480 and rs3894194 in group 1, <em>r</em><sup>2</sup> 0.5) SNPs identified in the GABRIEL analysis (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340953">5</a>) and tested for association with asthma case-control status in our study with logistic regression. The rightmost vertical bar represents the results obtained when including all seven quantitative scores as predictors in the logistic regression model. The horizontal black line represent a <em>P</em> = 0.05. The AUC, estimated from the logistic regression analysis, is shown above each bar. AUC=area under the receiver operator curve. GWAS=genome-wide association study.</p></div></div></div><p id="P33" class="p p-last">Three confirmed asthma loci are shared with Crohn’s disease, namely <em>ORMDL3</em>, <em>IL1RL1</em> and now chromosome 11q13.5. We postulated that loci with modest effects on asthma risk might also be shared with other inflammatory or immune diseases. To explore this possibility, we identified 356 SNPs listed in the catalog of published GWAS (<a href="#R12" rid="R12" class=" bibr popnode">12</a>) that were previously reported to associate at <em>P</em> ≤ 5×10<sup>−8</sup> with 54 such traits or diseases excluding asthma (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 25). Importantly, this list did not include any SNP located near (&lt;500 kb) the ten GABRIEL loci (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340936">5</a>), <em>PDE4D</em> (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513133">2</a>), <em>DENND1B</em> (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513187">3</a>), <em>IL6R</em> or 11q13.5. After excluding redundant markers (<em>r</em><sup>2</sup> ≥ 0.1), results for 207 largely independent SNPs from the original list were available in the meta-analysis of the Australian GWAS and the GABRIEL (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340948">5</a>), either directly or through a proxy SNP (<em>r</em><sup>2</sup> ≥ 0.8). Of these, 16 (8%) were associated with asthma risk with a <em>P</em> ≤ 0.01 (<a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> pp 26), when only about two were expected at this threshold under the null hypothesis of no association between the 207 SNPs and asthma (Fisher’s exact test <em>P</em> = 0.001). Four SNPs survived a Bonferroni correction for multiple testing (<em>P</em> ≤ 0.00024): rs11167764 near <em>NDFIP1</em> (OR = 1.11<em>, P</em> = 4.6×10<sup>−6</sup>) and rs1847472 in <em>BACH2</em> (OR = 1.07, <em>P</em> = 0.00023), two variants associated with Crohn’s disease (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513141">13</a>); rs2596560 near <em>HLA-B</em> (OR = 0.92<em>, P</em> = 6.5×10<sup>−5</sup>), a proxy SNP (<em>r</em><sup>2</sup> = 1.00) for a variant (rs3134792) reported to associate with psoriasis (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513139">14</a>); and rs13076312 in <em>LPP</em> (OR = 0.93, <em>P</em> = 0.00016), a proxy SNP (<em>r</em><sup>2</sup> = 1.00) for a variant (rs1464510) reported to associate with celiac disease (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513145">15</a>). These variants represent putative novel associations for asthma that require further confirmation and draw attention to molecular pathways that are probably shared between asthma and other inflammatory or immune diseases.</p></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Discussion</h2><p id="P34" class="p p-first">We identified two new loci with genome-wide significant association with asthma risk: rs4129267 in <em>IL6R</em> and rs7130588 on chromosome 11q13.5 (<a href="#BX1" rid="BX1" class=" boxed-text">panel</a>). Multiple lines of evidence suggest that <em>IL6R</em> is indeed the causal gene underlying our observed association with rs4129267. First, rs4129267 is strongly associated with variation in serum concentration of the soluble form of the IL-6 receptor (sIL-6R) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513140">18</a>). Each copy of the rs4129267:T allele increases sIL-6R protein concentration by about 1.4-fold (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513183">18</a>), while it increases the risk of asthma by 1.09-fold based on our analyses. Second, the concentration of sIL-6R is increased in both the serum (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513179">19</a>) and airways (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513178">20</a>) of patients with asthma, and it correlates with Th2 cytokine production in the lung (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513155">20</a>). Lastly, selective blockade of sIL-6R in mice suppressed IL-4, IL-5, and IL-13 production and decreased eosinophil numbers in the lung; on the other hand, blockade of sIL6-R plus the membrane-bound form of the receptor (mIL-6R) caused not only the suppression of Th2 cytokine production but also expansion of CD4+CD25+ Tregs in the lung with increased IL-10 production and suppressive capacity (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513143">20</a>). Thus, together with our results, these data suggest that rs4129267, or a causal variant in linkage disequilibrium with it, increases the risk of developing asthma by up-regulating protein concentrations of sIL-6R or mIL-6R, or both, which in turn contributes to the development and maintenance of a Th2 immune response in the lung. An IL-6R antagonist (tocilizumab) has been approved as an effective biologic drug to treat rheumatoid arthritis (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308938505">21</a>); further studies are warranted to test the hypothesis that tocilizumab might also be effective to treat asthma, particularly for patients with the rs4129267:T risk variant.</p><div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm140064592872656title">Panel: Research in Context</h3></div><div class="sec" id="S15"><h3 id="S15title">Systematic review</h3><p id="__p2" class="p p-first-last">Hundreds of studies have attempted to map genes that contribute to the risk of developing asthma. Most linkage and candidate-gene association studies were underpowered to detect genetic loci with effect sizes now considered realistic for common diseases (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513164">36</a>) and so often the identified regions or genes failed to replicate in independent studies. As a result, prior to 2007, over 100 loci had been implicated in asthma causation (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513129">37</a>), but which, if any, represented genuine risk factors was largely unclear. Since 2007, genome-wide association studies (GWAS) have contributed significantly to change this landscape. In the largest GWAS of asthma published to date, the GABRIEL consortium (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340942">5</a>) identified six loci associated with asthma risk at the genome-wide significance level: <em>IL18R1</em>, <em>HLA-DQ</em>, <em>IL33</em>, <em>SMAD3</em>, the <em>ORMDL3</em> locus, and <em>IL2RB</em>. Given the size of that study, these represent the most convincing genetic risk factors for asthma reported until now. However, each explains only a small fraction of the disease heritability, indicating that many more risk loci remain to be identified. This can be achieved by expanding and combining existing asthma GWAS, as we have done in this study.</p></div><div class="sec" id="S16"><h3 id="S16title">Interpretation</h3><p id="__p3" class="p p-first-last">Our study identifies two additional loci with genome-wide significant association with asthma risk: <em>IL6R</em> and chromosome 11q13.5. The <em>IL6R</em> finding raises the possibility that an approved <em>IL6R</em> antagonist may be effective to treat asthma. We also provide independent support for four of the six GABRIEL loci: <em>IL18R1</em>, <em>IL33</em>, <em>ORMDL3</em> and <em>IL2RB</em>. Taken together, these results support the hypothesis that a genetic dysregulation of cytokine signalling increases disease risk.</p></div></div><p id="P35">The second locus that our analyses implicate in the causation of asthma is represented by rs7130588, which is located on chromosome 11q13.5 near a SNP recently reported to associate with two immune-related diseases, Crohn’s disease (CD) (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513163">22</a>) and atopic dermatitis (AD) (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513151">23</a>). The variant reported to associate with Crohn’s disease and atopic dermatitis (rs7927894) is in complete linkage disequilibrium (LD) (<em>r</em><sup>2</sup> = 1) with rs7130588, and the two predisposing alleles (rs7927894:T and rs7130588:G) occur on the same haplotype. This indicates that the same underlying causal variant is likely to explain the association between this locus and the three diseases. Each copy of the rs7130588:G allele increases the risk of atopic dermatitis by 1.22-fold (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513166">23</a>), Crohn’s disease by 1.16-fold (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513142">22</a>) and asthma in our study by 1.09-fold. The association with atopic dermatitis and our finding that rs7130588:G does not seem to increase asthma risk in non-atopic individuals collectively suggests that this allele directly increases the risk of allergic sensitisation and, if this develops, it increases the risk of subsequently developing asthma. This mechanism is consistent with the epidemiological observations that sensitization, eczema and allergic rhinitis often precede the development of asthma symptoms (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513154">24</a>-<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513146">27</a>). While this work was under review, Marenholz and colleagues (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513188">38</a>) also reported an association between 11q13.5 and asthma, although not at the genome-wide significance level. Results from that study are also consistent with an effect for the 11q13.5 locus that is specific to allergic asthma. The leucine rich repeat containing 32 gene (<em>LRRC32)</em> is a plausible causal candidate in this region, as it is expressed in activated Treg cells (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513165">28</a>), of which numbers and immune suppressive function appear to be impaired in asthma (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513124">29</a>).</p><p id="P36">Our analyses also provided independent evidence for association with four of the eight loci previously reported to associate with asthma risk (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340944">5</a>) at the genome-wide significance level, specifically with the <em>ORMDL3</em> locus, <em>IL18R1</em>, <em>IL33</em>, and <em>IL2RB</em> (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340951">5</a>). The reported SNPs for <em>HLA-DQ</em> and <em>SMAD3</em> had consistent but non-significance evidence for association in our study; these results suggest that they might represent true risk factors for asthma in the population we studied, but have an associated risk that is lower than originally reported (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340960">5</a>). On the other hand, we found no supporting evidence for an association with <em>PDE4D</em> (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513123">2</a>) or <em>DENND1B</em> (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513159">3</a>) variants, despite appropriate power. Larger meta-analyses of available GWAS are in progress and will be able to study these loci in greater detail.</p><p id="P37">We also investigated the hypothesis that the genetic component of asthma risk includes a highly polygenic contribution, with hundreds or potentially thousands of variants that individually explain only a very small fraction of the disease heritability, as suggested for other traits such as schizophrenia (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_388647618">30</a>) and height (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513121">31</a>). Consistent with this hypothesis, we observed that quantitative genetic scores that represented the combined effect of thousands of common SNPs – each individually influencing asthma risk only weakly (e.g., median OR = 1.05 for Group 3 in <a href="/pmc/articles/PMC3517659/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) in the GABRIEL study (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481340940">5</a>) – were significantly associated with asthma case-control status in our study. These results thus suggest that many of these SNPs are either in high LD with true causal variants that are common in the population but influence disease risk weakly, or they are in low LD with causal variants that are rarer but increase disease risk more strongly. The presence of both types of variants is not mutually exclusive; very large sample sizes will be needed to identify these through GWAS with genome-wide significance. The multi-SNP scores tested here, despite being significantly associated with asthma risk, provided low discrimination in disease status (i.e., low sensitivity and specificity) and so currently have little or no diagnostic utility per se, consistent with recent findings (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_388647619">30</a>, <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513180">32</a>, <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513138">33</a>). However, as larger asthma GWAS are conducted, SNP effects will be estimated more precisely, and this will improve the discrimination accuracy of genome-wide SNP scores.</p><p id="P38">Lastly, our analysis of SNPs previously reported to associate with immune or inflammatory diseases identified several loci that may represent genuine asthma risk factors with weak effects. These include variants near plausible functional candidates, such as <em>NDFIP1</em>, which causes severe inflammation of the skin and lung when knocked-out in mice (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513137">34</a>) and <em>BACH2</em>, a B-cell-specific transcription repressor that is a key regulator of antibody response (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513134">35</a>). Further studies are required to confirm these as genuine asthma risk loci.</p><p id="P39">When interpreting results from this study it is important to recognise that we used a broad definition of asthma status, which grouped together not only clinically diagnosed and self-reported physician-diagnosed cases, but also groups of cases that were probably exposed to different environmental risk factors. Similarly, asthma controls included both asthma-free and asthma-unknown individuals, with limited information on atopic status. As a result, our primary association analysis provided improved power to detect risk loci with homogeneous effects across different asthma subtypes but was likely underpowered to detect loci with subtype-specific effects. Furthermore, a large fraction of SNPs discovered by the 1000 Genomes Project were imputed with modest confidence. For these SNPs, power to detect an association with asthma will also have been reduced.</p><p id="P40" class="p p-last">In conclusion, we identified novel variants in <em>IL6R</em> and chromosome 11q13.5 with genome-wide significant association with asthma. The <em>IL6R</em> findings further support the hypothesis that a genetic dysregulation of cytokine signalling increases disease risk and raise the possibility that tocilizumab may be effective to treat asthma, perhaps in a genotype-dependent manner; studies that address this possibility are warranted. At this stage, it is unclear which gene underlies the association with 11q13.5. Given that no specific gene in this region has been directly implicated in allergic disease previously, further characterisation of this region of association is likely to discover novel molecular mechanisms involved in the causality of eczema, atopy and asthma. Lastly, our results are consistent with the contribution of hundreds or potentially thousands of variants with weak effects on asthma risk, which can be identified through larger GWAS as already shown for other diseases (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288513173">13</a>).</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary-Webappendix</h4><div class="sup-box half_rhythm" id="d35e1480"><a href="/pmc/articles/PMC3517659/bin/NIHMS414524-supplement-Supplementary-Webappendix.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.9M, doc)</sup></div></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Acknowledgments</h2><div class="sec"><p id="__p1">The Australian Asthma Genetics Consortium was supported by the Australian National Health and Medical Research Council (613627). Detailed acknowledgements are provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">webappendix</a> (pp 27-29).</p></div></div><div id="idm140064595132880" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140064595132880title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P41" class="p p-first-last">Collaborators of the Australian Asthma Genetics Consortium Désirée Mészáros<sup><sup><a href="#FN2" rid="FN2" class=" fn">1</a></sup></sup>, Mary Roberts<sup><sup><a href="#FN3" rid="FN3" class=" fn">2</a></sup></sup>, Melissa C. Southey<sup><sup><a href="#FN4" rid="FN4" class=" fn">3</a></sup></sup>, Euan R. Tovey<sup><sup><a href="#FN5" rid="FN5" class=" fn">4</a></sup></sup>, Nicole M. Warrington<sup><sup><a href="#FN6" rid="FN6" class=" fn">5</a></sup></sup>, Mathijs Kattenberg<sup><sup><a href="#FN7" rid="FN7" class=" fn">6</a></sup></sup>, Lyle J. Palmer<sup><sup><a href="#FN8" rid="FN8" class=" fn">7</a></sup>,<sup><a href="#FN9" rid="FN9" class=" fn">8</a></sup></sup>, Loren Price<sup><sup><a href="#FN10" rid="FN10" class=" fn">9</a></sup></sup>, Margaret J. Wright<sup><sup><a href="#FN11" rid="FN11" class=" fn">10</a></sup></sup>, Scott D. Gordon<sup><sup><a href="#FN11" rid="FN11" class=" fn">10</a></sup></sup>, Li P. Chung<sup><sup><a href="#FN10" rid="FN10" class=" fn">9</a></sup></sup>, Anjali K. Henders<sup><sup><a href="#FN11" rid="FN11" class=" fn">10</a></sup></sup>, Graham Giles<sup><sup><a href="#FN12" rid="FN12" class=" fn">11</a></sup></sup>, Jouke-Jan Hottenga<sup><sup><a href="#FN7" rid="FN7" class=" fn">6</a></sup></sup>, Paul S. Thomas<sup><sup><a href="#FN13" rid="FN13" class=" fn">12</a></sup></sup>, Suzanna Temple<sup><sup><a href="#FN10" rid="FN10" class=" fn">9</a></sup></sup>, John B. Whitfield<sup><sup><a href="#FN11" rid="FN11" class=" fn">10</a></sup></sup>, Ian Feather<sup><sup><a href="#FN14" rid="FN14" class=" fn">13</a></sup></sup>, Anna-Liisa Hartikainen<sup><sup><a href="#FN15" rid="FN15" class=" fn">14</a></sup></sup>, Tari Haahtela<sup><sup><a href="#FN16" rid="FN16" class=" fn">15</a></sup></sup>, Tarja Laitinen<sup><sup><a href="#FN17" rid="FN17" class=" fn">16</a></sup></sup>, Pekka Jousilahti<sup><sup><a href="#FN18" rid="FN18" class=" fn">17</a></sup></sup>, Johan G. Eriksson<sup><sup><a href="#FN19" rid="FN19" class=" fn">18</a></sup>-<sup><a href="#FN22" rid="FN22" class=" fn">21</a></sup></sup>, Elisabeth Widen<sup><sup><a href="#FN23" rid="FN23" class=" fn">22</a></sup></sup>, Olli T. Raitakari<sup><sup><a href="#FN24" rid="FN24" class=" fn">23</a></sup>,<sup><a href="#FN25" rid="FN25" class=" fn">24</a></sup></sup>, Terho Lehtimäki<sup><sup><a href="#FN26" rid="FN26" class=" fn">25</a></sup>,<sup><a href="#FN27" rid="FN27" class=" fn">26</a></sup></sup>, Mika Kähönen<sup><sup><a href="#FN27" rid="FN27" class=" fn">26</a></sup>,<sup><a href="#FN28" rid="FN28" class=" fn">27</a></sup></sup>, Jorma Viikari<sup><sup><a href="#FN29" rid="FN29" class=" fn">28</a></sup>,<sup><a href="#FN30" rid="FN30" class=" fn">29</a></sup></sup>, Aarno Palotie<sup><sup><a href="#FN23" rid="FN23" class=" fn">22</a></sup>,<sup><a href="#FN31" rid="FN31" class=" fn">30</a></sup>-<sup><a href="#FN33" rid="FN33" class=" fn">32</a></sup></sup>, Zofia K. Gajdos<sup><sup><a href="#FN34" rid="FN34" class=" fn">33</a></sup>-<sup><a href="#FN36" rid="FN36" class=" fn">35</a></sup></sup>, Helen N. Lyon<sup><sup><a href="#FN35" rid="FN35" class=" fn">34</a></sup>,<sup><a href="#FN36" rid="FN36" class=" fn">35</a></sup></sup>, George T. O’Connor<sup><sup><a href="#FN37" rid="FN37" class=" fn">36</a></sup>,<sup><a href="#FN38" rid="FN38" class=" fn">37</a></sup></sup>, Stephen Morrison<sup><sup><a href="#FN39" rid="FN39" class=" fn">38</a></sup></sup>, Peter D. Sly<sup><sup><a href="#FN40" rid="FN40" class=" fn">39</a></sup></sup>, Chalermchai Mitrpant<sup><sup><a href="#FN10" rid="FN10" class=" fn">9</a></sup></sup>, Warwick J. Britton<sup><sup><a href="#FN41" rid="FN41" class=" fn">40</a></sup></sup>, David John<sup><sup><a href="#FN2" rid="FN2" class=" fn">1</a></sup></sup>, Pat G. Holt<sup><sup><a href="#FN6" rid="FN6" class=" fn">5</a></sup></sup>, Andrew S. Kemp<sup><sup><a href="#FN42" rid="FN42" class=" fn">41</a></sup></sup></p><p></p><p class="fn sec" id="FN2"><sup>1</sup>Menzies Research Institute, Hobart, Australia.</p><p class="fn sec" id="FN3"><sup>2</sup>Department of Respiratory Medicine, Royal Children’s Hospital, Parkville, Australia.</p><p class="fn sec" id="FN4"><sup>3</sup>Department of Pathology, The University of Melbourne, Melbourne, Australia.</p><p class="fn sec" id="FN5"><sup>4</sup>Woolcock Institute of Medical Research, University of Sydney, Sydney, Australia.</p><p class="fn sec" id="FN6"><sup>5</sup>Telethon Institute for Child Health Research, Centre for Child Health Research, University of WA, Perth, Australia.</p><p class="fn sec" id="FN7"><sup>6</sup>Netherlands Twin Register, EMGO &amp; NCA Institute, Department of Biological Psychology, VU University, Amsterdam, The Netherlands.</p><p class="fn sec" id="FN8"><sup>7</sup>Genetic Epidemiology and Biostatistics Platform, Ontario Instiute for Cancer Research, Toronto, Canada.</p><p class="fn sec" id="FN9"><sup>8</sup>Samuel Lunenfeld Research Institute, Unviversity of Toronto, Toronto, Canada.</p><p class="fn sec" id="FN10"><sup>9</sup>Lung Institute of WA and Centre for Asthma, Allergy and Respiratory Research, University of WA, Perth, Australia.</p><p class="fn sec" id="FN11"><sup>10</sup>The Queensland Institute of Medical Research, Brisbane, Australia.</p><p class="fn sec" id="FN12"><sup>11</sup>Cancer Epidemiology Centre, The Cancer Council Victoria, Melbourne, Australia.</p><p class="fn sec" id="FN13"><sup>12</sup>Faculty of Medicine, University of New South Wales, Sydney, Australia.</p><p class="fn sec" id="FN14"><sup>13</sup>Gold Coast Hospital, Southport, Australia.</p><p class="fn sec" id="FN15"><sup>14</sup>Department of Clinical Sciences, Obstetrics and Gynecology, Institute of Clinical Medicine, University of Oulu, Oulu, Finland.</p><p class="fn sec" id="FN16"><sup>15</sup>Skin and Allergy Hospital, Helsinki University Hospital, Helsinki, Finland.</p><p class="fn sec" id="FN17"><sup>16</sup>Deptartment of Pulmonary Diseases and Clinical Allergology, Turku University Hospital and University of Turku, Finland.</p><p class="fn sec" id="FN18"><sup>17</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Finland.</p><p class="fn sec" id="FN19"><sup>18</sup>National Institute for Health and Welfare, Finland.</p><p class="fn sec" id="FN20"><sup>19</sup>Department of General Practice and Primary health Care, University of Helsinki, Finland.</p><p class="fn sec" id="FN21"><sup>20</sup>Helsinki University Central Hospital, Unit of General Practice, Helsinki, Finland.</p><p class="fn sec" id="FN22"><sup>21</sup>Folkhalsan Research Centre, Helsinki, Finland.</p><p class="fn sec" id="FN23"><sup>22</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland.</p><p class="fn sec" id="FN24"><sup>23</sup>Research Centre of Applied and Preventive Medicine, University of Turku, Turku, Finland.</p><p class="fn sec" id="FN25"><sup>24</sup>Department of Clinical Physiology, Turku University Hospital, Turku, Finland.</p><p class="fn sec" id="FN26"><sup>25</sup>Department of Clinical Chemistry, University of Tampere, Tampere, Finland.</p><p class="fn sec" id="FN27"><sup>26</sup>Tampere University Hospital, Tampere, Finland.</p><p class="fn sec" id="FN28"><sup>27</sup>Department of Clinical Physiology, University of Tampere, Tampere, Finland.</p><p class="fn sec" id="FN29"><sup>28</sup>Turku University Hospital, Turku, Finland.</p><p class="fn sec" id="FN30"><sup>29</sup>Department of Medicine, University of Turku,Turku, Finland.</p><p class="fn sec" id="FN31"><sup>30</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="fn sec" id="FN32"><sup>31</sup>Program in Medical and Population Genetics and Genetic Analysis Platform, The Broad Institute ofMIT and Harvard, Cambridge, United States.</p><p class="fn sec" id="FN33"><sup>32</sup>Department of Medical Genetics, University of Helsinki and University Central Hospital, Helsinki, Finland.</p><p class="fn sec" id="FN34"><sup>33</sup>Department of Genetics, Harvard Medical School, Boston, United States.</p><p class="fn sec" id="FN35"><sup>34</sup>Divisions of Genetics and Endocrinology, Children’s Hospital, Boston, United States.</p><p class="fn sec" id="FN36"><sup>35</sup>Broad Institute, Cambridge, United States.</p><p class="fn sec" id="FN37"><sup>36</sup>Pulmonary Center, Department of Medicine, Boston University School of Medicine, Boston, United States.</p><p class="fn sec" id="FN38"><sup>37</sup>The National Heart, Lung, and Blood Institute’s Framingham Heart Study, Framingham, United States.</p><p class="fn sec" id="FN39"><sup>38</sup>University of Queensland, Brisbane, Australia.</p><p class="fn sec" id="FN40"><sup>39</sup>Queensland Children’s Medical Research Institute, Brisbane, Australia.</p><p class="fn sec" id="FN41"><sup>40</sup>Centenary Institute and University of Sydney, Camperdown, Australia.</p><p class="fn sec" id="FN42"><sup>41</sup>The Children’s Hospital, Westmead, Sydney, Australia.</p><p class="fn sec" id="FN44"></p><p id="P83" class="p p-first"><strong>Contributors</strong>
<em>Writing group:</em> M.A.R.F, M.C.M., D.L.D., G.B.M., P.LeS, S.D., M.J.A., P.J.T., G.W.M., M.A.B.</p><p id="P84" class="p"><em>Analytic group:</em> M.A.R.F., M.C.M., D.L.D., S.B., W.A., N.M.W., P.D., S.G., M.K., A.R., S.V., J.H., M.R., M.A.B., P.M.V.</p><p id="P85" class="p"><em>Project management:</em> M.A.R.F., M.C.M., D.L.D., G.B.M., J.H., P.LeS, S.D., M.J.A., M.A.B., G.W.M., P.J.T.</p><p id="P86" class="p"><em>Phenotype and genotype collection for QIMR studies</em>: D.L.D., M.A.R.F., D.R.N., S.D.G., M.J.W., A.K.H., J.B.W., P.A.M., A.C.H, G.W.M., N.G.M.</p><p id="P87" class="p"><em>Phenotype and genotype collection for TAHS</em>: D.M., M.C.S., G.G., P.S.T., I.F., S.M., M.J., M.J.A., J.L.H., H.W., M.C.M., S.C.D.</p><p id="P88" class="p"><em>Phenotype and genotype collection for MESCA:</em> C.H., M.R., P.LeS., C.R.</p><p id="P89" class="p"><em>Phenotype and genotype collection for CAPS:</em> G.M., A.J.K., E.R.T., W.J.B., G.J., L.R.S.</p><p id="P90" class="p"><em>Phenotype and genotype collection for RAINE:</em> C.P., P.D.S., P.G.H.</p><p id="P91" class="p"><em>Phenotype and genotype collection for NTR:</em>G.W., E.J.C.deGeus, M.K., JJ.H., D.I.B.</p><p id="P92" class="p"><em>Phenotype and genotype collection for APCAT</em>: V.S., M-R.J., O.T.R., J.V., T.L., M.K., J.E.</p><p id="P93" class="p"><em>Phenotype and genotype collection for Busselton:</em> J.H., J.B., L.J.P., A.J., B.M.,</p><p id="P94" class="p"><em>Genotyping for AAGC:</em> P.D., M.A.B.</p><p id="P95" class="p p-last"><em>Phenotype and genotype collection for LIWA:</em> S.B., F.C., L.P.C., S.T., C.M., B.S., L.P., P.J.T.</p><p></p><p class="fn sec" id="FN45"></p><p id="P96" class="p p-first-last"><strong>Conflicts of interest</strong> MAF owns stock in Illumina, Roche and Complete Genomics. PST’s institution has received consulting fees from GSK as a result of activities outside this work. PST has received lecture fees from GSK and Novartis. GTO has received consulting fees from Novartis and Sunovion, Inc, as a result of work unrelated to this study. GBM was a board member of Novartis; GBM’s institution received fees for lectures and preparation of educational presentations from Boehringer Ingelheim and AstraZeneca. CFR received consulting fees from GSK and Merck as a result of activities outside this work. ASK’s institution received speaker’s fees from Nutricia. All other authors declare that they have no conflicts of interest.</p><p></p></div></div><div id="idm140064590938944" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140064590938944title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Moffatt MF, Kabesch M, Liang L, Dixon AL, Strachan D, Heath S, et al.  Genetic variants regulating ORMDL3 expression contribute to the risk of childhood asthma. <span><span class="ref-journal">Nature. </span>2007 Jul 26;<span class="ref-vol">448</span>(7152):470–3.</span> [<a href="/pubmed/17611496" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+variants+regulating+ORMDL3+expression+contribute+to+the+risk+of+childhood+asthma&amp;author=MF+Moffatt&amp;author=M+Kabesch&amp;author=L+Liang&amp;author=AL+Dixon&amp;author=D+Strachan&amp;volume=448&amp;issue=7152&amp;publication_year=2007&amp;pages=470-3&amp;pmid=17611496&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Himes BE, Hunninghake GM, Baurley JW, Rafaels NM, Sleiman P, Strachan DP, et al.  Genome-wide association analysis identifies PDE4D as an asthma-susceptibility gene. <span><span class="ref-journal">Am J Hum Genet. </span>2009 May;<span class="ref-vol">84</span>(5):581–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681010/">PMC free article</a>]</span> [<a href="/pubmed/19426955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+association+analysis+identifies+PDE4D+as+an+asthma-susceptibility+gene&amp;author=BE+Himes&amp;author=GM+Hunninghake&amp;author=JW+Baurley&amp;author=NM+Rafaels&amp;author=P+Sleiman&amp;volume=84&amp;issue=5&amp;publication_year=2009&amp;pages=581-93&amp;pmid=19426955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Sleiman PM, Flory J, Imielinski M, Bradfield JP, Annaiah K, Willis-Owen SA, et al.  Variants of DENND1B associated with asthma in children. <span><span class="ref-journal">N Engl J Med. </span>2010 Jan 7;<span class="ref-vol">362</span>(1):36–44.</span> [<a href="/pubmed/20032318" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Variants+of+DENND1B+associated+with+asthma+in+children&amp;author=PM+Sleiman&amp;author=J+Flory&amp;author=M+Imielinski&amp;author=JP+Bradfield&amp;author=K+Annaiah&amp;volume=362&amp;issue=1&amp;publication_year=2010&amp;pages=36-44&amp;pmid=20032318&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Gudbjartsson DF, Bjornsdottir US, Halapi E, Helgadottir A, Sulem P, Jonsdottir GM, et al.  Sequence variants affecting eosinophil numbers associate with asthma and myocardial infarction. <span><span class="ref-journal">Nat Genet. </span>2009 Mar;<span class="ref-vol">41</span>(3):342–7.</span> [<a href="/pubmed/19198610" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Sequence+variants+affecting+eosinophil+numbers+associate+with+asthma+and+myocardial+infarction&amp;author=DF+Gudbjartsson&amp;author=US+Bjornsdottir&amp;author=E+Halapi&amp;author=A+Helgadottir&amp;author=P+Sulem&amp;volume=41&amp;issue=3&amp;publication_year=2009&amp;pages=342-7&amp;pmid=19198610&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Moffatt MF, Gut IG, Demenais F, Strachan DP, Bouzigon E, Heath S, et al.  A large-scale, consortium-based genomewide association study of asthma. <span><span class="ref-journal">N Engl J Med. </span>2010 Sep 23;<span class="ref-vol">363</span>(13):1211–21.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4260321/">PMC free article</a>]</span> [<a href="/pubmed/20860503" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=A+large-scale,+consortium-based+genomewide+association+study+of+asthma&amp;author=MF+Moffatt&amp;author=IG+Gut&amp;author=F+Demenais&amp;author=DP+Strachan&amp;author=E+Bouzigon&amp;volume=363&amp;issue=13&amp;publication_year=2010&amp;pages=1211-21&amp;pmid=20860503&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Manolio TA, Brooks LD, Collins FS. A HapMap harvest of insights into the genetics of common disease. <span><span class="ref-journal">J Clin Invest. </span>2008 May;<span class="ref-vol">118</span>(5):1590–605.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2336881/">PMC free article</a>]</span> [<a href="/pubmed/18451988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=A+HapMap+harvest+of+insights+into+the+genetics+of+common+disease&amp;author=TA+Manolio&amp;author=LD+Brooks&amp;author=FS+Collins&amp;volume=118&amp;issue=5&amp;publication_year=2008&amp;pages=1590-605&amp;pmid=18451988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Ferreira MA, McRae AF, Medland SE, Nyholt DR, Gordon SD, Wright MJ, et al.  Association between ORMDL3, IL1RL1 and a deletion on chromosome 17q21 with asthma risk in Australia. <span><span class="ref-journal">Eur J Hum Genet. </span>2011 Apr;<span class="ref-vol">19</span>(4):458–64.</span> Eur J Hum Genet. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3060316/">PMC free article</a>]</span> [<a href="/pubmed/21150878" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Association+between+ORMDL3,+IL1RL1+and+a+deletion+on+chromosome+17q21+with+asthma+risk+in+Australia&amp;author=MA+Ferreira&amp;author=AF+McRae&amp;author=SE+Medland&amp;author=DR+Nyholt&amp;author=SD+Gordon&amp;volume=19&amp;issue=4&amp;publication_year=2011&amp;pages=458-64&amp;pmid=21150878&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009 Jun;<span class="ref-vol">5</span>(6):e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;issue=6&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007 Sep;<span class="ref-vol">81</span>(3):559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010 Sep 1;<span class="ref-vol">26</span>(17):2190–1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;issue=17&amp;publication_year=2010&amp;pages=2190-1&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Wray NR, Goddard ME, Visscher PM. Prediction of individual genetic risk to disease from genome-wide association studies. <span><span class="ref-journal">Genome Res. </span>2007 Oct;<span class="ref-vol">17</span>(10):1520–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1987352/">PMC free article</a>]</span> [<a href="/pubmed/17785532" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Prediction+of+individual+genetic+risk+to+disease+from+genome-wide+association+studies&amp;author=NR+Wray&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=17&amp;issue=10&amp;publication_year=2007&amp;pages=1520-8&amp;pmid=17785532&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Hindorff LA, Junkins HA, Hall PN, Mehta JP, Manolio TA. A Catalog of Published Genome-Wide Association Studies  [Accession date: 15 February 2011]. </span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Franke A, McGovern DP, Barrett JC, Wang K, Radford-Smith GL, Ahmad T, et al.  Genome-wide meta-analysis increases to 71 the number of confirmed Crohn’s disease susceptibility loci. <span><span class="ref-journal">Nat Genet. </span>2010 Dec;<span class="ref-vol">42</span>(12):1118–25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3299551/">PMC free article</a>]</span> [<a href="/pubmed/21102463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+meta-analysis+increases+to+71+the+number+of+confirmed+Crohn’s+disease+susceptibility+loci&amp;author=A+Franke&amp;author=DP+McGovern&amp;author=JC+Barrett&amp;author=K+Wang&amp;author=GL+Radford-Smith&amp;volume=42&amp;issue=12&amp;publication_year=2010&amp;pages=1118-25&amp;pmid=21102463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Capon F, Bijlmakers MJ, Wolf N, Quaranta M, Huffmeier U, Allen M, et al.  Identification of ZNF313/RNF114 as a novel psoriasis susceptibility gene. <span><span class="ref-journal">Hum Mol Genet. </span>2008 Jul;<span class="ref-vol">17</span>(13):1938–45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2900900/">PMC free article</a>]</span> [<a href="/pubmed/18364390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Identification+of+ZNF313/RNF114+as+a+novel+psoriasis+susceptibility+gene&amp;author=F+Capon&amp;author=MJ+Bijlmakers&amp;author=N+Wolf&amp;author=M+Quaranta&amp;author=U+Huffmeier&amp;volume=17&amp;issue=13&amp;publication_year=2008&amp;pages=1938-45&amp;pmid=18364390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Hunt KA, Zhernakova A, Turner G, Heap GA, Franke L, Bruinenberg M, et al.  Newly identified genetic risk variants for celiac disease related to the immune response. <span><span class="ref-journal">Nat Genet. </span>2008 Apr;<span class="ref-vol">40</span>(4):395–402.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2673512/">PMC free article</a>]</span> [<a href="/pubmed/18311140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Newly+identified+genetic+risk+variants+for+celiac+disease+related+to+the+immune+response&amp;author=KA+Hunt&amp;author=A+Zhernakova&amp;author=G+Turner&amp;author=GA+Heap&amp;author=L+Franke&amp;volume=40&amp;issue=4&amp;publication_year=2008&amp;pages=395-402&amp;pmid=18311140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">18. <span class="element-citation">Melzer D, Perry JR, Hernandez D, Corsi AM, Stevens K, Rafferty I, et al.  A genome-wide association study identifies protein quantitative trait loci (pQTLs) <span><span class="ref-journal">PLoS Genet. </span>2008 May;<span class="ref-vol">4</span>(5):e1000072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2362067/">PMC free article</a>]</span> [<a href="/pubmed/18464913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+identifies+protein+quantitative+trait+loci+(pQTLs)&amp;author=D+Melzer&amp;author=JR+Perry&amp;author=D+Hernandez&amp;author=AM+Corsi&amp;author=K+Stevens&amp;volume=4&amp;issue=5&amp;publication_year=2008&amp;pages=e1000072&amp;pmid=18464913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">19. <span class="element-citation">Yokoyama A, Kohno N, Sakai K, Kondo K, Hirasawa Y, Hiwada K. Circulating levels of soluble interleukin-6 receptor in patients with bronchial asthma. <span><span class="ref-journal">Am J Respir Crit Care Med. </span>1997 Nov;<span class="ref-vol">156</span>(5):1688–91.</span> [<a href="/pubmed/9372694" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Respir+Crit+Care+Med&amp;title=Circulating+levels+of+soluble+interleukin-6+receptor+in+patients+with+bronchial+asthma&amp;author=A+Yokoyama&amp;author=N+Kohno&amp;author=K+Sakai&amp;author=K+Kondo&amp;author=Y+Hirasawa&amp;volume=156&amp;issue=5&amp;publication_year=1997&amp;pages=1688-91&amp;pmid=9372694&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">20. <span class="element-citation">Doganci A, Eigenbrod T, Krug N, De Sanctis GT, Hausding M, Erpenbeck VJ, et al.  The IL-6R alpha chain controls lung CD4+CD25+ Treg development and function during allergic airway inflammation in vivo. <span><span class="ref-journal">J Clin Invest. </span>2005 Feb;<span class="ref-vol">115</span>(2):313–25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC544603/">PMC free article</a>]</span> [<a href="/pubmed/15668741" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=The+IL-6R+alpha+chain+controls+lung+CD4+CD25++Treg+development+and+function+during+allergic+airway+inflammation+in+vivo&amp;author=A+Doganci&amp;author=T+Eigenbrod&amp;author=N+Krug&amp;author=GT+De+Sanctis&amp;author=M+Hausding&amp;volume=115&amp;issue=2&amp;publication_year=2005&amp;pages=313-25&amp;pmid=15668741&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">21. <span class="element-citation">Patel AM, Moreland LW. Interleukin-6 inhibition for treatment of rheumatoid arthritis: a review of tocilizumab therapy. <span><span class="ref-journal">Drug Des Devel Ther. </span>2010;<span class="ref-vol">4</span>:263–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2990387/">PMC free article</a>]</span> [<a href="/pubmed/21116333" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Des+Devel+Ther&amp;title=Interleukin-6+inhibition+for+treatment+of+rheumatoid+arthritis:+a+review+of+tocilizumab+therapy&amp;author=AM+Patel&amp;author=LW+Moreland&amp;volume=4&amp;publication_year=2010&amp;pages=263-78&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">22. <span class="element-citation">Barrett JC, Hansoul S, Nicolae DL, Cho JH, Duerr RH, Rioux JD, et al.  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn’s disease. <span><span class="ref-journal">Nat Genet. </span>2008 Aug;<span class="ref-vol">40</span>(8):955–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn’s+disease&amp;author=JC+Barrett&amp;author=S+Hansoul&amp;author=DL+Nicolae&amp;author=JH+Cho&amp;author=RH+Duerr&amp;volume=40&amp;issue=8&amp;publication_year=2008&amp;pages=955-62&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">23. <span class="element-citation">Esparza-Gordillo J, Weidinger S, Folster-Holst R, Bauerfeind A, Ruschendorf F, Patone G, et al.  A common variant on chromosome 11q13 is associated with atopic dermatitis. <span><span class="ref-journal">Nat Genet. </span>2009 May;<span class="ref-vol">41</span>(5):596–601.</span> [<a href="/pubmed/19349984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+variant+on+chromosome+11q13+is+associated+with+atopic+dermatitis&amp;author=J+Esparza-Gordillo&amp;author=S+Weidinger&amp;author=R+Folster-Holst&amp;author=A+Bauerfeind&amp;author=F+Ruschendorf&amp;volume=41&amp;issue=5&amp;publication_year=2009&amp;pages=596-601&amp;pmid=19349984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">24. <span class="element-citation">Schuttelaar ML, Kerkhof M, Jonkman MF, Koppelman GH, Brunekreef B, de Jongste JC, et al.  Filaggrin mutations in the onset of eczema, sensitization, asthma, hay fever and the interaction with cat exposure. <span><span class="ref-journal">Allergy. </span>2009 Dec;<span class="ref-vol">64</span>(12):1758–65.</span> [<a href="/pubmed/19839980" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Allergy&amp;title=Filaggrin+mutations+in+the+onset+of+eczema,+sensitization,+asthma,+hay+fever+and+the+interaction+with+cat+exposure&amp;author=ML+Schuttelaar&amp;author=M+Kerkhof&amp;author=MF+Jonkman&amp;author=GH+Koppelman&amp;author=B+Brunekreef&amp;volume=64&amp;issue=12&amp;publication_year=2009&amp;pages=1758-65&amp;pmid=19839980&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">25. <span class="element-citation">Lowe AJ, Hosking CS, Bennett CM, Carlin JB, Abramson MJ, Hill DJ, et al.  Skin prick test can identify eczematous infants at risk of asthma and allergic rhinitis. <span><span class="ref-journal">Clin Exp Allergy. </span>2007 Nov;<span class="ref-vol">37</span>(11):1624–31.</span> [<a href="/pubmed/17877754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Exp+Allergy&amp;title=Skin+prick+test+can+identify+eczematous+infants+at+risk+of+asthma+and+allergic+rhinitis&amp;author=AJ+Lowe&amp;author=CS+Hosking&amp;author=CM+Bennett&amp;author=JB+Carlin&amp;author=MJ+Abramson&amp;volume=37&amp;issue=11&amp;publication_year=2007&amp;pages=1624-31&amp;pmid=17877754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">26. <span class="element-citation">Burgess JA, Dharmage SC, Byrnes GB, Matheson MC, Gurrin LC, Wharton CL, et al.  Childhood eczema and asthma incidence and persistence: a cohort study from childhood to middle age. <span><span class="ref-journal">J Allergy Clin Immunol. </span>2008 Aug;<span class="ref-vol">122</span>(2):280–5.</span> [<a href="/pubmed/18572229" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Childhood+eczema+and+asthma+incidence+and+persistence:+a+cohort+study+from+childhood+to+middle+age&amp;author=JA+Burgess&amp;author=SC+Dharmage&amp;author=GB+Byrnes&amp;author=MC+Matheson&amp;author=LC+Gurrin&amp;volume=122&amp;issue=2&amp;publication_year=2008&amp;pages=280-5&amp;pmid=18572229&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">27. <span class="element-citation">Lowe AJ, Carlin JB, Bennett CM, Hosking CS, Abramson MJ, Hill DJ, et al.  Do boys do the atopic march while girls dawdle? <span><span class="ref-journal">J Allergy Clin Immunol. </span>2008 May;<span class="ref-vol">121</span>(5):1190–5.</span> [<a href="/pubmed/18313134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Do+boys+do+the+atopic+march+while+girls+dawdle?&amp;author=AJ+Lowe&amp;author=JB+Carlin&amp;author=CM+Bennett&amp;author=CS+Hosking&amp;author=MJ+Abramson&amp;volume=121&amp;issue=5&amp;publication_year=2008&amp;pages=1190-5&amp;pmid=18313134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">28. <span class="element-citation">Wang R, Kozhaya L, Mercer F, Khaitan A, Fujii H, Unutmaz D. Expression of GARP selectively identifies activated human FOXP3+ regulatory T cells. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009 Aug 11;<span class="ref-vol">106</span>(32):13439–44.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2726405/">PMC free article</a>]</span> [<a href="/pubmed/19666573" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Expression+of+GARP+selectively+identifies+activated+human+FOXP3++regulatory+T+cells&amp;author=R+Wang&amp;author=L+Kozhaya&amp;author=F+Mercer&amp;author=A+Khaitan&amp;author=H+Fujii&amp;volume=106&amp;issue=32&amp;publication_year=2009&amp;pages=13439-44&amp;pmid=19666573&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">29. <span class="element-citation">Hartl D, Koller B, Mehlhorn AT, Reinhardt D, Nicolai T, Schendel DJ, et al.  Quantitative and functional impairment of pulmonary CD4+CD25hi regulatory T cells in pediatric asthma. <span><span class="ref-journal">J Allergy Clin Immunol. </span>2007 May;<span class="ref-vol">119</span>(5):1258–66.</span> [<a href="/pubmed/17412402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Quantitative+and+functional+impairment+of+pulmonary+CD4+CD25hi+regulatory+T+cells+in+pediatric+asthma&amp;author=D+Hartl&amp;author=B+Koller&amp;author=AT+Mehlhorn&amp;author=D+Reinhardt&amp;author=T+Nicolai&amp;volume=119&amp;issue=5&amp;publication_year=2007&amp;pages=1258-66&amp;pmid=17412402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">30. <span class="element-citation">Purcell SM, Wray NR, Stone JL, Visscher PM, O’Donovan MC, Sullivan PF, et al.  Common polygenic variation contributes to risk of schizophrenia and bipolar disorder. <span><span class="ref-journal">Nature. </span>2009 Aug 6;<span class="ref-vol">460</span>(7256):748–52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia+and+bipolar+disorder&amp;author=SM+Purcell&amp;author=NR+Wray&amp;author=JL+Stone&amp;author=PM+Visscher&amp;author=MC+O’Donovan&amp;volume=460&amp;issue=7256&amp;publication_year=2009&amp;pages=748-52&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">31. <span class="element-citation">Yang J, Benyamin B, McEvoy BP, Gordon S, Henders AK, Nyholt DR, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010 Jul;<span class="ref-vol">42</span>(7):565–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;author=B+Benyamin&amp;author=BP+McEvoy&amp;author=S+Gordon&amp;author=AK+Henders&amp;volume=42&amp;issue=7&amp;publication_year=2010&amp;pages=565-9&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">32. <span class="element-citation">Evans DM, Visscher PM, Wray NR. Harnessing the information contained within genome-wide association studies to improve individual prediction of complex disease risk. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>(18):3525–31.</span> [<a href="/pubmed/19553258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Harnessing+the+information+contained+within+genome-wide+association+studies+to+improve+individual+prediction+of+complex+disease+risk&amp;author=DM+Evans&amp;author=PM+Visscher&amp;author=NR+Wray&amp;volume=18&amp;issue=18&amp;publication_year=2009&amp;pages=3525-31&amp;pmid=19553258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">33. <span class="element-citation">Paynter NP, Chasman DI, Paré G, Buring JE, Cook NR, Miletich JP, et al.  Association between a literature-based genetic risk score and cardiovascular events in women. <span><span class="ref-journal">JAMA. </span>2010;<span class="ref-vol">303</span>(7):631–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2845522/">PMC free article</a>]</span> [<a href="/pubmed/20159871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Association+between+a+literature-based+genetic+risk+score+and+cardiovascular+events+in+women&amp;author=NP+Paynter&amp;author=DI+Chasman&amp;author=G+Paré&amp;author=JE+Buring&amp;author=NR+Cook&amp;volume=303&amp;issue=7&amp;publication_year=2010&amp;pages=631-7&amp;pmid=20159871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">34. <span class="element-citation">Oliver PM, Cao X, Worthen GS, Shi P, Briones N, MacLeod M, et al.  Ndfip1 protein promotes the function of itch ubiquitin ligase to prevent T cell activation and T helper 2 cell-mediated inflammation. <span><span class="ref-journal">Immunity. </span>2006 Dec;<span class="ref-vol">25</span>(6):929–40.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955961/">PMC free article</a>]</span> [<a href="/pubmed/17137798" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Ndfip1+protein+promotes+the+function+of+itch+ubiquitin+ligase+to+prevent+T+cell+activation+and+T+helper+2+cell-mediated+inflammation&amp;author=PM+Oliver&amp;author=X+Cao&amp;author=GS+Worthen&amp;author=P+Shi&amp;author=N+Briones&amp;volume=25&amp;issue=6&amp;publication_year=2006&amp;pages=929-40&amp;pmid=17137798&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">35. <span class="element-citation">Muto A, Tashiro S, Nakajima O, Hoshino H, Takahashi S, Sakoda E, et al.  The transcriptional programme of antibody class switching involves the repressor Bach2. <span><span class="ref-journal">Nature. </span>2004 Jun 3;<span class="ref-vol">429</span>(6991):566–71.</span> [<a href="/pubmed/15152264" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+transcriptional+programme+of+antibody+class+switching+involves+the+repressor+Bach2&amp;author=A+Muto&amp;author=S+Tashiro&amp;author=O+Nakajima&amp;author=H+Hoshino&amp;author=S+Takahashi&amp;volume=429&amp;issue=6991&amp;publication_year=2004&amp;pages=566-71&amp;pmid=15152264&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">36. <span class="element-citation">Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, et al.  Finding the missing heritability of complex diseases. <span><span class="ref-journal">Nature. </span>2009 Oct 8;<span class="ref-vol">461</span>(7265):747–53.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=TA+Manolio&amp;author=FS+Collins&amp;author=NJ+Cox&amp;author=DB+Goldstein&amp;author=LA+Hindorff&amp;volume=461&amp;issue=7265&amp;publication_year=2009&amp;pages=747-53&amp;pmid=19812666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">37. <span class="element-citation">Ober C, Hoffjan S. Asthma genetics 2006: the long and winding road to gene discovery. <span><span class="ref-journal">Genes Immun. </span>2006 Mar;<span class="ref-vol">7</span>(2):95–100.</span> [<a href="/pubmed/16395390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Asthma+genetics+2006:+the+long+and+winding+road+to+gene+discovery&amp;author=C+Ober&amp;author=S+Hoffjan&amp;volume=7&amp;issue=2&amp;publication_year=2006&amp;pages=95-100&amp;pmid=16395390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">38. <span class="element-citation">Marenholz I, Bauerfeind A, Esparza-Gordillo J, et al.  The eczema risk variant on chromosome 11q13 (rs7927894) in the population-based ALSPAC cohort: a novel susceptibility factor for asthma and hay fever. <span><span class="ref-journal">Hum Mol Genet. </span>2011 Jun;<span class="ref-vol">20</span>(12):2443–49.</span> [<a href="/pubmed/21429916" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+eczema+risk+variant+on+chromosome+11q13+(rs7927894)+in+the+population-based+ALSPAC+cohort:+a+novel+susceptibility+factor+for+asthma+and+hay+fever&amp;author=I+Marenholz&amp;author=A+Bauerfeind&amp;author=J+Esparza-Gordillo&amp;volume=20&amp;issue=12&amp;publication_year=2011&amp;pages=2443-49&amp;pmid=21429916&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3517659&amp;issue-id=217049&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140064590930720title">Abstract</a></li><li><a href="#S6title">Introduction</a></li><li><a href="#S7title">Methods</a></li><li><a href="#S12title">Results</a></li><li><a href="#S13title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S14title">Acknowledgments</a></li><li><a href="#idm140064595132880title">Footnotes</a></li><li><a href="#idm140064590938944title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3517659/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3517659/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3517659/pdf/nihms-414524.pdf">PDF (790K)</a></li> | <li><a href="#" data-citationid="PMC3517659" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3517659%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3517659%2F&amp;text=Identification%20of%20IL6R%20and%20chromosome%2011q13.5%20as%20risk%20loci%20for%20asthma" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3517659%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21907864" data-aiid="3517659" data-aid="3517659" data-iid="217049" data-domainid="319" data-domain="nihpa" data-accid="PMC3517659" data-md5="0e23ee1e3fd96be1706ae1d0895e8080"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:40:53-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B6A67DD52E3C10000000003DD0278&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3517659%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3517659/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>