<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3180858/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Evidence for gene-environment interaction in a genome wide study of isolated, non-syndromic cleft palate</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3180858/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Genetic epidemiology"><meta name="citation_title" content="Evidence for gene-environment interaction in a genome wide study of isolated, non-syndromic cleft palate"><meta name="citation_authors" content="Terri H. Beaty, Ingo Ruczinski, Jeffrey C. Murray, Mary L. Marazita, Ronald G. Munger, Jacqueline B. Hetmanski, Tanda Murray, Richard J. Redett, M. Daniele Fallin, Kung Yee Liang, Tao Wu, Poorav J. Patel, Sheng C. Jin, Tian Xiao Zhang, Holger Schwender, Yah Huei Wu-Chou, Philip K Chen, Samuel S Chong, Felicia Cheah, Vincent Yeow, Xiaoqian Ye, Hong Wang, Shangzhi Huang, Ethylin W. Jabs, Bing Shi, Allen J. Wilcox, Rolv T. Lie, Sun Ha Jee, Kaare Christensen, Kimberley F. Doheny, Elizabeth W. Pugh, Hua Ling, Alan F. Scott"><meta name="citation_date" content="September 2011"><meta name="citation_issue" content="6"><meta name="citation_volume" content="35"><meta name="citation_firstpage" content="469"><meta name="citation_doi" content="10.1002/gepi.20595"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3180858/?report=abstract"><meta name="citation_pmid" content="21618603"><meta name="DC.Title" content="Evidence for gene-environment interaction in a genome wide study of isolated, non-syndromic cleft palate"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Terri H. Beaty"><meta name="DC.Contributor" content="Ingo Ruczinski"><meta name="DC.Contributor" content="Jeffrey C. Murray"><meta name="DC.Contributor" content="Mary L. Marazita"><meta name="DC.Contributor" content="Ronald G. Munger"><meta name="DC.Contributor" content="Jacqueline B. Hetmanski"><meta name="DC.Contributor" content="Tanda Murray"><meta name="DC.Contributor" content="Richard J. Redett"><meta name="DC.Contributor" content="M. Daniele Fallin"><meta name="DC.Contributor" content="Kung Yee Liang"><meta name="DC.Contributor" content="Tao Wu"><meta name="DC.Contributor" content="Poorav J. Patel"><meta name="DC.Contributor" content="Sheng C. Jin"><meta name="DC.Contributor" content="Tian Xiao Zhang"><meta name="DC.Contributor" content="Holger Schwender"><meta name="DC.Contributor" content="Yah Huei Wu-Chou"><meta name="DC.Contributor" content="Philip K Chen"><meta name="DC.Contributor" content="Samuel S Chong"><meta name="DC.Contributor" content="Felicia Cheah"><meta name="DC.Contributor" content="Vincent Yeow"><meta name="DC.Contributor" content="Xiaoqian Ye"><meta name="DC.Contributor" content="Hong Wang"><meta name="DC.Contributor" content="Shangzhi Huang"><meta name="DC.Contributor" content="Ethylin W. Jabs"><meta name="DC.Contributor" content="Bing Shi"><meta name="DC.Contributor" content="Allen J. Wilcox"><meta name="DC.Contributor" content="Rolv T. Lie"><meta name="DC.Contributor" content="Sun Ha Jee"><meta name="DC.Contributor" content="Kaare Christensen"><meta name="DC.Contributor" content="Kimberley F. Doheny"><meta name="DC.Contributor" content="Elizabeth W. Pugh"><meta name="DC.Contributor" content="Hua Ling"><meta name="DC.Contributor" content="Alan F. Scott"><meta name="DC.Date" content="2011 Sep"><meta name="DC.Identifier" content="10.1002/gepi.20595"><meta name="DC.Language" content="en"><meta property="og:title" content="Evidence for gene-environment interaction in a genome wide study of isolated, non-syndromic cleft palate"><meta property="og:type" content="article"><meta property="og:description" content="Non-syndromic cleft palate (CP) is a common birth defect with a complex and heterogeneous etiology involving both genetic and environmental risk factors. We conducted a genome wide association study (GWAS) using 550 case-parent trios, ascertained through ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3180858/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3180858/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3180858/pdf/nihms-297057.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A324ADD5221E10000000007180482.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3180858%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3180858</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Genet Epidemiol</a></span>. Author manuscript; available in PMC 2012 Sep 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21618603" target="pmc_ext" ref="reftype=publisher&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Genet Epidemiol. 2011 Sep; 35(6): 469–478. </span></a></div><span class="fm-vol-iss-date">Published online 2011 May 26. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fgepi.20595" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/gepi.20595</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3180858</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS297057</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21618603">21618603</a></div></div></div></div><h1 class="content-title">Evidence for gene-environment interaction in a genome wide study of isolated, non-syndromic cleft palate</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Beaty%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697016054704" co-class="co-affbox">Terri H. Beaty</a>,<sup>1</sup> <a href="/pubmed/?term=Ruczinski%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697027486608" co-class="co-affbox">Ingo Ruczinski</a>,<sup>1</sup> <a href="/pubmed/?term=Murray%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697028791424" co-class="co-affbox">Jeffrey C. Murray</a>,<sup>2</sup> <a href="/pubmed/?term=Marazita%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697029451440" co-class="co-affbox">Mary L. Marazita</a>,<sup>3</sup> <a href="/pubmed/?term=Munger%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697029449216" co-class="co-affbox">Ronald G. Munger</a>,<sup>4</sup> <a href="/pubmed/?term=Hetmanski%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697016890688" co-class="co-affbox">Jacqueline B. Hetmanski</a>,<sup>1</sup> <a href="/pubmed/?term=Murray%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697016500784" co-class="co-affbox">Tanda Murray</a>,<sup>1</sup> <a href="/pubmed/?term=Redett%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697016498560" co-class="co-affbox">Richard J. Redett</a>,<sup>5</sup> <a href="/pubmed/?term=Fallin%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697010746064" co-class="co-affbox">M. Daniele Fallin</a>,<sup>1</sup> <a href="/pubmed/?term=Liang%20KY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697012197488" co-class="co-affbox">Kung Yee Liang</a>,<sup>1,</sup><sup>6</sup> <a href="/pubmed/?term=Wu%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697029966848" co-class="co-affbox">Tao Wu</a>,<sup>1</sup> <a href="/pubmed/?term=Patel%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697029964624" co-class="co-affbox">Poorav J. Patel</a>,<sup>1</sup> <a href="/pubmed/?term=Jin%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697017215872" co-class="co-affbox">Sheng C. Jin</a>,<sup>1</sup> <a href="/pubmed/?term=Zhang%20TX%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697017771184" co-class="co-affbox">Tian Xiao Zhang</a>,<sup>1</sup> <a href="/pubmed/?term=Schwender%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697031295216" co-class="co-affbox">Holger Schwender</a>,<sup>1</sup> <a href="/pubmed/?term=Wu-Chou%20YH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697031292992" co-class="co-affbox">Yah Huei Wu-Chou</a>,<sup>7</sup> <a href="/pubmed/?term=Chen%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697012463456" co-class="co-affbox">Philip K Chen</a>,<sup>7</sup> <a href="/pubmed/?term=Chong%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697006221120" co-class="co-affbox">Samuel S Chong</a>,<sup>8</sup> <a href="/pubmed/?term=Cheah%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697015968144" co-class="co-affbox">Felicia Cheah</a>,<sup>8</sup> <a href="/pubmed/?term=Yeow%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697015965920" co-class="co-affbox">Vincent Yeow</a>,<sup>9</sup> <a href="/pubmed/?term=Ye%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697016014816" co-class="co-affbox">Xiaoqian Ye</a>,<sup>10,</sup><sup>13</sup> <a href="/pubmed/?term=Wang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697012284336" co-class="co-affbox">Hong Wang</a>,<sup>11</sup> <a href="/pubmed/?term=Huang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697020373440" co-class="co-affbox">Shangzhi Huang</a>,<sup>12</sup> <a href="/pubmed/?term=Jabs%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697020371216" co-class="co-affbox">Ethylin W. Jabs</a>,<sup>5,</sup><sup>13</sup> <a href="/pubmed/?term=Shi%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697017899648" co-class="co-affbox">Bing Shi</a>,<sup>14</sup> <a href="/pubmed/?term=Wilcox%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697029935968" co-class="co-affbox">Allen J. Wilcox</a>,<sup>15</sup> <a href="/pubmed/?term=Lie%20RT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697010720848" co-class="co-affbox">Rolv T. Lie</a>,<sup>16</sup> <a href="/pubmed/?term=Jee%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697010718624" co-class="co-affbox">Sun Ha Jee</a>,<sup>17</sup> <a href="/pubmed/?term=Christensen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697101856912" co-class="co-affbox">Kaare Christensen</a>,<sup>18</sup> <a href="/pubmed/?term=Doheny%20KF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697024486464" co-class="co-affbox">Kimberley F. Doheny</a>,<sup>19</sup> <a href="/pubmed/?term=Pugh%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697026627824" co-class="co-affbox">Elizabeth W. Pugh</a>,<sup>19</sup> <a href="/pubmed/?term=Ling%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697026625712" co-class="co-affbox">Hua Ling</a>,<sup>19</sup> and  <a href="/pubmed/?term=Scott%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603" class="affpopup" co-rid="_co_idm140697005151888" co-class="co-affbox">Alan F. Scott</a><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140697016054704"><h3 class="no_margin">Terri H. Beaty</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Beaty%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Terri H. Beaty</a></div></div><div id="_co_idm140697027486608"><h3 class="no_margin">Ingo Ruczinski</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Ruczinski%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Ingo Ruczinski</a></div></div><div id="_co_idm140697028791424"><h3 class="no_margin">Jeffrey C. Murray</h3><p><sup>2</sup>University of Iowa, Children's Hospital, Iowa City, IA, USA</p><div>Find articles by <a href="/pubmed/?term=Murray%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Jeffrey C. Murray</a></div></div><div id="_co_idm140697029451440"><h3 class="no_margin">Mary L. Marazita</h3><p><sup>3</sup>University of Pittsburgh, Center for Craniofacial and Dental Genetics, School of Dental Medicine, Pittsburgh, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Marazita%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Mary L. Marazita</a></div></div><div id="_co_idm140697029449216"><h3 class="no_margin">Ronald G. Munger</h3><p><sup>4</sup>Utah State University, Logan, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Munger%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Ronald G. Munger</a></div></div><div id="_co_idm140697016890688"><h3 class="no_margin">Jacqueline B. Hetmanski</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Hetmanski%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Jacqueline B. Hetmanski</a></div></div><div id="_co_idm140697016500784"><h3 class="no_margin">Tanda Murray</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Murray%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Tanda Murray</a></div></div><div id="_co_idm140697016498560"><h3 class="no_margin">Richard J. Redett</h3><p><sup>5</sup>Johns Hopkins University, School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Redett%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Richard J. Redett</a></div></div><div id="_co_idm140697010746064"><h3 class="no_margin">M. Daniele Fallin</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Fallin%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">M. Daniele Fallin</a></div></div><div id="_co_idm140697012197488"><h3 class="no_margin">Kung Yee Liang</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><p><sup>6</sup>National Yang-Ming University, Taipei, Taiwan</p><div>Find articles by <a href="/pubmed/?term=Liang%20KY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Kung Yee Liang</a></div></div><div id="_co_idm140697029966848"><h3 class="no_margin">Tao Wu</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Wu%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Tao Wu</a></div></div><div id="_co_idm140697029964624"><h3 class="no_margin">Poorav J. Patel</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Patel%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Poorav J. Patel</a></div></div><div id="_co_idm140697017215872"><h3 class="no_margin">Sheng C. Jin</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Jin%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Sheng C. Jin</a></div></div><div id="_co_idm140697017771184"><h3 class="no_margin">Tian Xiao Zhang</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Zhang%20TX%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Tian Xiao Zhang</a></div></div><div id="_co_idm140697031295216"><h3 class="no_margin">Holger Schwender</h3><p><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Schwender%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Holger Schwender</a></div></div><div id="_co_idm140697031292992"><h3 class="no_margin">Yah Huei Wu-Chou</h3><p><sup>7</sup>Chang Gung Memorial Hospital, Taoyuan, Taiwan</p><div>Find articles by <a href="/pubmed/?term=Wu-Chou%20YH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Yah Huei Wu-Chou</a></div></div><div id="_co_idm140697012463456"><h3 class="no_margin">Philip K Chen</h3><p><sup>7</sup>Chang Gung Memorial Hospital, Taoyuan, Taiwan</p><div>Find articles by <a href="/pubmed/?term=Chen%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Philip K Chen</a></div></div><div id="_co_idm140697006221120"><h3 class="no_margin">Samuel S Chong</h3><p><sup>8</sup>National University of Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Chong%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Samuel S Chong</a></div></div><div id="_co_idm140697015968144"><h3 class="no_margin">Felicia Cheah</h3><p><sup>8</sup>National University of Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Cheah%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Felicia Cheah</a></div></div><div id="_co_idm140697015965920"><h3 class="no_margin">Vincent Yeow</h3><p><sup>9</sup>KK Women's &amp; Children's Hospital, Singapore</p><div>Find articles by <a href="/pubmed/?term=Yeow%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Vincent Yeow</a></div></div><div id="_co_idm140697016014816"><h3 class="no_margin">Xiaoqian Ye</h3><p><sup>10</sup>Wuhan University, School of Stomatology, Wuhan, China</p><p><sup>13</sup>Mount Sinai School of Medicine, New York, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Ye%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Xiaoqian Ye</a></div></div><div id="_co_idm140697012284336"><h3 class="no_margin">Hong Wang</h3><p><sup>11</sup>Peking University Health Science Center, Beijing, China</p><div>Find articles by <a href="/pubmed/?term=Wang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Hong Wang</a></div></div><div id="_co_idm140697020373440"><h3 class="no_margin">Shangzhi Huang</h3><p><sup>12</sup>Peking Union Medical College, Beijing, China</p><div>Find articles by <a href="/pubmed/?term=Huang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Shangzhi Huang</a></div></div><div id="_co_idm140697020371216"><h3 class="no_margin">Ethylin W. Jabs</h3><p><sup>5</sup>Johns Hopkins University, School of Medicine, Baltimore, MD, USA</p><p><sup>13</sup>Mount Sinai School of Medicine, New York, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Jabs%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Ethylin W. Jabs</a></div></div><div id="_co_idm140697017899648"><h3 class="no_margin">Bing Shi</h3><p><sup>14</sup>State Key Laboratory of Oral Disease, West China College of Stomatology, Sichuan University, Chengdu, China</p><div>Find articles by <a href="/pubmed/?term=Shi%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Bing Shi</a></div></div><div id="_co_idm140697029935968"><h3 class="no_margin">Allen J. Wilcox</h3><p><sup>15</sup>NIEHS/NIH, Epidemiology Branch, Durham, North Carolina, USA</p><div>Find articles by <a href="/pubmed/?term=Wilcox%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Allen J. Wilcox</a></div></div><div id="_co_idm140697010720848"><h3 class="no_margin">Rolv T. Lie</h3><p><sup>16</sup>University of Bergen, Bergen, Norway</p><div>Find articles by <a href="/pubmed/?term=Lie%20RT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Rolv T. Lie</a></div></div><div id="_co_idm140697010718624"><h3 class="no_margin">Sun Ha Jee</h3><p><sup>17</sup>Yonsei University, Epidemiology &amp; Health Promotion, Seoul, Korea</p><div>Find articles by <a href="/pubmed/?term=Jee%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Sun Ha Jee</a></div></div><div id="_co_idm140697101856912"><h3 class="no_margin">Kaare Christensen</h3><p><sup>18</sup>University of Southern Denmark, Odense, Denmark</p><div>Find articles by <a href="/pubmed/?term=Christensen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Kaare Christensen</a></div></div><div id="_co_idm140697024486464"><h3 class="no_margin">Kimberley F. Doheny</h3><p><sup>19</sup>Center for Inherited Disease Research, Johns Hopkins University, School of Medicine, Baltimore MD, USA</p><div>Find articles by <a href="/pubmed/?term=Doheny%20KF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Kimberley F. Doheny</a></div></div><div id="_co_idm140697026627824"><h3 class="no_margin">Elizabeth W. Pugh</h3><p><sup>19</sup>Center for Inherited Disease Research, Johns Hopkins University, School of Medicine, Baltimore MD, USA</p><div>Find articles by <a href="/pubmed/?term=Pugh%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Elizabeth W. Pugh</a></div></div><div id="_co_idm140697026625712"><h3 class="no_margin">Hua Ling</h3><p><sup>19</sup>Center for Inherited Disease Research, Johns Hopkins University, School of Medicine, Baltimore MD, USA</p><div>Find articles by <a href="/pubmed/?term=Ling%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Hua Ling</a></div></div><div id="_co_idm140697005151888"><h3 class="no_margin">Alan F. Scott</h3><p><sup>5</sup>Johns Hopkins University, School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Scott%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21618603">Alan F. Scott</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140697017236544_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140697017236544_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140697017236544_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Johns Hopkins University, School of Public Health, Baltimore, MD, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>University of Iowa, Children's Hospital, Iowa City, IA, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>University of Pittsburgh, Center for Craniofacial and Dental Genetics, School of Dental Medicine, Pittsburgh, PA, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Utah State University, Logan, UT, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Johns Hopkins University, School of Medicine, Baltimore, MD, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>National Yang-Ming University, Taipei, Taiwan</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Chang Gung Memorial Hospital, Taoyuan, Taiwan</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>National University of Singapore, Singapore</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>KK Women's &amp; Children's Hospital, Singapore</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Wuhan University, School of Stomatology, Wuhan, China</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Peking University Health Science Center, Beijing, China</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Peking Union Medical College, Beijing, China</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Mount Sinai School of Medicine, New York, NY, USA</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>State Key Laboratory of Oral Disease, West China College of Stomatology, Sichuan University, Chengdu, China</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>NIEHS/NIH, Epidemiology Branch, Durham, North Carolina, USA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>University of Bergen, Bergen, Norway</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Yonsei University, Epidemiology &amp; Health Promotion, Seoul, Korea</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>University of Southern Denmark, Odense, Denmark</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Center for Inherited Disease Research, Johns Hopkins University, School of Medicine, Baltimore MD, USA</div><div id="CR1">Corresponding author: Dr. T.H. Beaty, Dept of Epidemiology, School of Public Health, Johns Hopkins University, 615 N. Wolfe St, Baltimore MD, USA; <a href="mailto:dev@null" data-email="ude.hpshj@ytaebt" class="oemail">ude.hpshj@ytaebt</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140697017236544_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21618603" target="pmc_ext" ref="reftype=publisher&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Genet Epidemiol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3180858/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table S1-S5&amp;Figure S1-S10.</strong></div><div><a href="/pmc/articles/PMC3180858/bin/NIHMS297057-supplement-Supp_Table_S1-S5_Figure_S1-S10.doc" data-ga-action="click_feat_suppl">NIHMS297057-supplement-Supp_Table_S1-S5_Figure_S1-S10.doc</a><span style="color:gray"> (1.7M)</span></div><div class="small guid">GUID:&nbsp;DFDF8B0A-3308-4513-8F0E-B050C00CB0F4</div></div></dd></dl></div><div id="idm140697030957184" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140697030957184title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Non-syndromic cleft palate (CP) is a common birth defect with a complex and heterogeneous etiology involving both genetic and environmental risk factors. We conducted a genome wide association study (GWAS) using 550 case-parent trios, ascertained through a CP case collected in an international consortium. Family based association tests of single nucleotide polymorphisms (SNP) and three common maternal exposures (maternal smoking, alcohol consumption and multivitamin supplementation) were used in a combined 2 df test for gene (G) and gene-environment (G×E) interaction simultaneously, plus a separate 1 df test for G×E interaction alone. Conditional logistic regression models were used to estimate effects on risk to exposed and unexposed children. While no SNP achieved genome wide significance when considered alone, markers in several genes attained or approached genome wide significance when G×E interaction was included. Among these, <em>MLLT3</em> and <em>SMC2</em> on chromosome 9 showed multiple SNPs resulting in increased risk if the mother consumed alcohol during the peri-conceptual period (3 months prior to conception through the first trimester). <em>TBK1</em> on chr. 12 and <em>ZNF236</em> on chr. 18 showed multiple SNPs associated with higher risk of CP in the presence of maternal smoking. Additional evidence of reduced risk due to G×E interaction in the presence of multivitamin supplementation was observed for SNPs in <em>BAALC</em> on chr. 8. These results emphasize the need to consider G×E interaction when searching for genes influencing risk to complex and heterogeneous disorders, such as non-syndromic CP.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P2" class="p p-first">Cleft palate (CP) is a common birth defect where both genetic and environmental components contribute to the etiology. CP has a lower birth prevalence compared to cleft lip with/without cleft palate (CL/P): 1/2500 live births vs. 1/700; but CP shows less variability in birth prevalence across populations compared to CL/P. Almost half of all livebirths with CP occur in infants with another congenital anomaly or some identifiable malformation syndrome [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154118">Genisca et al., 2009</a>]. The current study focuses on isolated, non-syndromic CP (CP) which shows strong familial aggregation; has documented environmental risk factors; and is etiologically heterogeneous.</p><p id="P3">Both population based studies and family studies suggest a strong genetic component to the etiology of non-syndromic CP. Risk of CP among first degree relatives of cases is 56 times greater (95%CI=37–85) than the general population in Norway [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154125">Sivertsen et al., 2008</a>]; and <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154124">Grosen et al. (2009)</a> reported a 15-fold higher risk (95%CI=13–17) in Denmark. Twin and family studies of CP also argue for strong genetic control. In a nationwide study of Danish twins, <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154114">Grosen et al. (2009)</a> reported probandwise concordance rates of 33% for CP among monozygotic twins compared to 7% among dizygotic twins, and the latter was only slightly higher than the 3% recurrence risk seen between full siblings [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154116">Grosen et al., 2009</a>]. Despite this evidence for some genetic component, no single gene model can explain the strong familial aggregation of CP [<a href="#R19" rid="R19" class=" bibr popnode">Marazita, 2002</a>]. Part of this difficulty could reflect locus heterogeneity, i.e. several different genes contribute to the etiology of CP. In the absence of external information to separate CP families into more homogeneous subgroups, however, statistical tests on modest sized samples will always have limited power to detect effects of individual causal genes.</p><p id="P4">Although several genes have been identified for syndromic forms of CP, few have been identified as influencing risk to non-syndromic CP, and none have sufficient evidence to be defined as directly causal. This could reflect the difficulty in amassing sufficiently large samples of cases. Recently, <a href="#R4" rid="R4" class=" bibr popnode">Ghassibe et al. (2010)</a> showed a translocation in a single multiplex CP family with Pierre Robin sequence (which involves CP) included the <em>FAF1</em> gene on chromosome (chr.) 1p. Subsequent analysis of an intronic marker (rs3827730) in <em>FAF1</em> showed evidence of linkage and association in a collection of case-parent trios from several populations (mostly of European ancestry). <a href="#R4" rid="R4" class=" bibr popnode">Ghassibe et al. (2010)</a> also showed evidence of decreased expression in blood samples from a small number of CP patients, although the function of <em>FAF1</em> in craniofacial development remains poorly defined. <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154120">Li et al. (2009)</a> also reported SNP rs7205289 in the microRNA (miRNA-140) region of chr. 16 showed evidence of association in a sample of 557 non-syndromic oral cleft cases (388 CL/P and 169 CP cases) compared to 306 healthy controls from Western China.</p><p id="P5">Part of the difficulty in identifying genes controlling risk to non-syndromic CP could also reflect biological interaction between high risk alleles and exposure to environmental risk factors during early development. There are several recognized environmental risk factors for non-syndromic CP, including maternal smoking and alcohol consumption [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154117">Little et al., 2004</a>; <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154108">DeRoo et al., 2008</a>]. <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320766193">Johnson and Little (2008)</a> reviewed the role of multi-vitamin and folate supplementation on risk to oral clefts. A few candidate gene studies have investigated G×E interaction with maternal smoking, although none used a genome wide approach. Starting with <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154122">Hwang et al. (1995)</a> several CP case-control studies of G×Smoking interaction with an intronic marker in <em>TGFA</em> on chr. 2q gave suggestive evidence, as did some case-parent trio studies [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154128">Maestri et al., 1997</a>]. However, meta-analysis of 5 case-control studies did not reveal compelling evidence for G×Smoking interaction [<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154123">Zeiger et al., 2005</a>]. Recently, evidence of G×E interaction was reported between SNP rs7205289 (in miRNA-140) and maternal passive smoking in an analysis of 162 CP cases and 304 healthy controls [<a href="#R15" rid="R15" class=" bibr popnode">Li et al., 2010</a>]. In this Chinese sample, children whose mother was exposed to passive smoking were 4.7 times more likely to have CP if they carried the high-risk allele.</p><p id="P6" class="p p-last">Considering gene-environment (G×E) interaction in the context of a genome-wide association study (GWAS) can be an important step in identifying genes controlling risk to complex and heterogeneous disorders, such as non-syndromic CP. <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_260568598">Manning et al. (2011)</a> showed how a variety of statistical tests for G×E interaction can be incorporated into GWAS, even meta-analyses of several studies. We conducted a GWAS to identify genes controlling risk to isolated, non-syndromic oral clefts (including CP) under a case-parent trio design in an international consortium [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154113">Beaty et al., 2010</a>]. Here we report on 550 CP trios where tests for G×E interaction with three common maternal exposures (maternal smoking, alcohol consumption and vitamin supplementation) are considered. The genome-wide approach has the advantage of being unbiased in its coverage of the human genome, and the case-parent trio study design has the advantage of minimizing confounding due to population stratification. This provided a unique opportunity to search for genes influencing risk to a common birth defect alone or through interaction with maternal exposures.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Case-parent trios</h3><p id="P7" class="p p-first-last">Case-parent trios were drawn from several groups who formed an international consortium to conduct a genome-wide search for genes influencing risk to oral clefts using a case-parent trio design [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154121">Beaty et al., 2010</a>]. <a href="/pmc/articles/PMC3180858/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a> lists numbers of trios noting the CP proband's gender from each recruitment site and in the total sample. Most cases were ascertained through surgical treatment centers, although population based ascertainment was used in Norway. To minimize potential misclassification of non-syndromic CP, cases were examined by either a clinical geneticist or experienced health care provider to rule out syndromic forms (except in Norway where medical records were reviewed). As often reported, there were more female CP cases (56%) compared to males. Racial ancestry of cases fell into two broad categories: 49% (272 of 550) of CP cases were of European ancestry (including European Americans), 47% (259 of 550) were of Asian ancestry and 3% (19 of 550) were categorized as African ancestry (mostly African Americans) or `other' racial groups (including mixed ancestry).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table I</h3><!--caption a7--><div class="caption"><p id="__p6">Gender of 550 isolated, non-syndromic cleft palate (CP) cases in the international consortium study by recruitment site</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697025080896" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="left">Sites</th><th rowspan="1" colspan="1" valign="top" align="center">Male</th><th rowspan="1" colspan="1" valign="top" align="center">Female</th><th colspan="2" rowspan="1" valign="top" align="center">Total</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">Counts</th><th rowspan="1" colspan="1" valign="top" align="center">Counts</th><th rowspan="1" colspan="1" valign="top" align="center">Counts</th><th rowspan="1" colspan="1" valign="top" align="center">%</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Denmark</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">13</td><td rowspan="1" colspan="1" valign="top" align="center">2%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Norway</td><td rowspan="1" colspan="1" valign="top" align="center">52</td><td rowspan="1" colspan="1" valign="top" align="center">58</td><td rowspan="1" colspan="1" valign="top" align="center">110</td><td rowspan="1" colspan="1" valign="top" align="center">20%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Iowa</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="center">41</td><td rowspan="1" colspan="1" valign="top" align="center">7%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Maryland</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">24</td><td rowspan="1" colspan="1" valign="top" align="center">43</td><td rowspan="1" colspan="1" valign="top" align="center">8%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Pittsburgh</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">3%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Utah</td><td rowspan="1" colspan="1" valign="top" align="center">31</td><td rowspan="1" colspan="1" valign="top" align="center">33</td><td rowspan="1" colspan="1" valign="top" align="center">64</td><td rowspan="1" colspan="1" valign="top" align="center">12%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Singapore</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="center">35</td><td rowspan="1" colspan="1" valign="top" align="center">57</td><td rowspan="1" colspan="1" valign="top" align="center">10%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Taiwan</td><td rowspan="1" colspan="1" valign="top" align="center">29</td><td rowspan="1" colspan="1" valign="top" align="center">50</td><td rowspan="1" colspan="1" valign="top" align="center">79</td><td rowspan="1" colspan="1" valign="top" align="center">14%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Shangdong Prov.</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="center">38</td><td rowspan="1" colspan="1" valign="top" align="center">7%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Hubei Prov.</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">26</td><td rowspan="1" colspan="1" valign="top" align="center">45</td><td rowspan="1" colspan="1" valign="top" align="center">8%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Sichuan Prov.</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="center">40</td><td rowspan="1" colspan="1" valign="top" align="center">7%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Korea</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">1%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Total</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">244 (44%)</td><td rowspan="1" colspan="1" valign="top" align="center">306 (56%)</td><td rowspan="1" colspan="1" valign="top" align="center">550 (100%)</td><td rowspan="1" colspan="1" valign="top" align="center">100%</td></tr></tbody></table></div><div id="largeobj_idm140697025080896" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3180858/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="S4" class="sec"><h3 id="S4title">Genotyping</h3><p id="P8" class="p p-first-last">The Center for Inherited Disease Research (CIDR) genotyped DNA samples using Illumina's 610Quad platform and 99.1% passed CIDR quality control (QC) [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154112">Beaty et al., 2010</a>]. Genotypes on 589,945 SNPs (99.56% of those attempted) were released and then underwent further QC analysis to set up 4 types of flags for each SNP: 1) unacceptably high rates (&gt;5%) of missing genotype calls, 2) low minor allele frequency (MAF&lt;0.01), 3) unacceptably high rates of Mendelian errors (&gt;5%) between parents and child, and 4) significant deviation (p&lt;10<sup>−5</sup>) from Hardy Weinberg equilibrium (HWE) among parents within recruitment site or across populations but within European and Asian groups separately. This QC process flagged 14.6% of all SNPs (mostly for low MAF), leaving ~498K SNPs available.</p></div><div id="S5" class="sec"><h3 id="S5title">Exposure assessment</h3><p id="P9" class="p p-first-last">Maternal exposure was assessed through a structured interview focused on the peri-conceptual period (3 months prior to conception through the first trimester) because palatal development is completed during weeks 8–9 of gestation (often before the pregnancy is recognized). Three maternal exposures were assessed as simple yes/no responses: personal cigarette smoking by the mother, any alcohol consumption, and any use of multi-vitamin supplements (not limited to folate). <a href="/pmc/articles/PMC3180858/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table II</span></a> presents rates of these maternal exposures in the total sample of mothers and stratifying by European, Asian and African/other ancestry groups. Rates of maternal exposures varied considerably across these groups.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table II</h3><!--caption a7--><div class="caption"><p id="__p7">Exposure rates for maternal smoking, multivitamin supplementation and alcohol consumption for the total CP group and for three ancestry groups</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697029436272" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">Ancestry Group</th><th colspan="4" rowspan="1" valign="bottom" align="center">Maternal smoking</th><th colspan="4" rowspan="1" valign="bottom" align="center">Maternal alcohol consumption</th><th colspan="4" rowspan="1" valign="bottom" align="center">Maternal multivitamin supplementation</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Not exposed</th><th rowspan="1" colspan="1" valign="bottom" align="center">Exposed</th><th rowspan="1" colspan="1" valign="bottom" align="center">Unk.</th><th rowspan="1" colspan="1" valign="bottom" align="center">%E</th><th rowspan="1" colspan="1" valign="bottom" align="center">Not exposed</th><th rowspan="1" colspan="1" valign="bottom" align="center">Exposed</th><th rowspan="1" colspan="1" valign="bottom" align="center">Unk.</th><th rowspan="1" colspan="1" valign="bottom" align="center">%E</th><th rowspan="1" colspan="1" valign="bottom" align="center">Not exposed</th><th rowspan="1" colspan="1" valign="bottom" align="center">Exposed</th><th rowspan="1" colspan="1" valign="bottom" align="center">Unk.</th><th rowspan="1" colspan="1" valign="bottom" align="center">%E</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="bottom" align="left">European</td><td rowspan="1" colspan="1" valign="bottom" align="center">198</td><td rowspan="1" colspan="1" valign="bottom" align="center">74</td><td rowspan="1" colspan="1" valign="bottom" align="center">0</td><td rowspan="1" colspan="1" valign="bottom" align="center">27%</td><td rowspan="1" colspan="1" valign="bottom" align="center">162</td><td rowspan="1" colspan="1" valign="bottom" align="center">109</td><td rowspan="1" colspan="1" valign="bottom" align="center">1</td><td rowspan="1" colspan="1" valign="bottom" align="center">40%</td><td rowspan="1" colspan="1" valign="bottom" align="center">88</td><td rowspan="1" colspan="1" valign="bottom" align="center">157</td><td rowspan="1" colspan="1" valign="bottom" align="center">27</td><td rowspan="1" colspan="1" valign="bottom" align="center">58%</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Asian</td><td rowspan="1" colspan="1" valign="bottom" align="center">250</td><td rowspan="1" colspan="1" valign="bottom" align="center">9</td><td rowspan="1" colspan="1" valign="bottom" align="center">0</td><td rowspan="1" colspan="1" valign="bottom" align="center">4%</td><td rowspan="1" colspan="1" valign="bottom" align="center">246</td><td rowspan="1" colspan="1" valign="bottom" align="center">10</td><td rowspan="1" colspan="1" valign="bottom" align="center">3</td><td rowspan="1" colspan="1" valign="bottom" align="center">4%</td><td rowspan="1" colspan="1" valign="bottom" align="center">197</td><td rowspan="1" colspan="1" valign="bottom" align="center">52</td><td rowspan="1" colspan="1" valign="bottom" align="center">10</td><td rowspan="1" colspan="1" valign="bottom" align="center">20%</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Afr/Other</td><td rowspan="1" colspan="1" valign="bottom" align="center">15</td><td rowspan="1" colspan="1" valign="bottom" align="center">3</td><td rowspan="1" colspan="1" valign="bottom" align="center">1</td><td rowspan="1" colspan="1" valign="bottom" align="center">16%</td><td rowspan="1" colspan="1" valign="bottom" align="center">15</td><td rowspan="1" colspan="1" valign="bottom" align="center">3</td><td rowspan="1" colspan="1" valign="bottom" align="center">1</td><td rowspan="1" colspan="1" valign="bottom" align="center">16%</td><td rowspan="1" colspan="1" valign="bottom" align="center">4</td><td rowspan="1" colspan="1" valign="bottom" align="center">13</td><td rowspan="1" colspan="1" valign="bottom" align="center">2</td><td rowspan="1" colspan="1" valign="bottom" align="center">68%</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Total</td><td rowspan="1" colspan="1" valign="bottom" align="center">463</td><td rowspan="1" colspan="1" valign="bottom" align="center">86</td><td rowspan="1" colspan="1" valign="bottom" align="center">1</td><td rowspan="1" colspan="1" valign="bottom" align="center">16%</td><td rowspan="1" colspan="1" valign="bottom" align="center">423</td><td rowspan="1" colspan="1" valign="bottom" align="center">122</td><td rowspan="1" colspan="1" valign="bottom" align="center">5</td><td rowspan="1" colspan="1" valign="bottom" align="center">22%</td><td rowspan="1" colspan="1" valign="bottom" align="center">289</td><td rowspan="1" colspan="1" valign="bottom" align="center">222</td><td rowspan="1" colspan="1" valign="bottom" align="center">39</td><td rowspan="1" colspan="1" valign="bottom" align="center">40%</td></tr></tbody></table></div><div id="largeobj_idm140697029436272" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3180858/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S6" class="sec sec-last"><h3 id="S6title">Statistical analysis</h3><div id="S7" class="sec sec-first"><p></p><h4 id="S7title" class="inline">GWAS </h4><p id="P10" class="p p-first-last">The conventional TDT was used to test individual SNPs in the genome-wide marker panel using PLINK [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154115">Purcell et al., 2007</a>] <a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a>) and PBAT [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154110">Lange et al., 2003</a>] (v3.6; <a href="http://www.biostat.harvard.edu/~clange/default.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.biostat.harvard.edu/~clange/default.htm</a>). Additionally, a total of 14,486 SNPs on the X chromosome were examined using FBAT [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154111">Laird et al., 2000</a>] <a href="http://www.biostat.harvard.edu/fbat/default.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.biostat.harvard.edu/fbat/default.html</a>), which is equivalent to the genotypic TDT under an additive model on independent trios [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154127">Laird and Lange, 2006</a>]. Manhattan plots of −log<sub>10</sub>(p-value) were generated over all autosomal and X-linked markers. Under a Bonferroni correction for ~500K tests, this data set would require a significance level of ≤10<sup>−7</sup> for genome-wide significance. Quantile-quantile (QQ) plots were also generated.</p></div><div id="S8" class="sec sec-last"><p></p><h4 id="S8title" class="inline">GxE interaction </h4><p id="P11" class="p p-first">To test for GxE interaction, we followed a strategy proposed first by <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154129">Lange et al. (2003)</a> for family studies and later by <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154126">Kraft et al. (2007)</a> for case-control studies. The PBAT package was used to compute a combined score test with 2 df test for G and GxE interaction together, and the separate 1 df test for GxE interaction alone. This provides the opportunity to detect genes where the GxE interaction enhances G effects, as well as detecting deviation from predicted effects of G and E (the classical definition of statistical interaction). Autosomal markers were used in the initial screen with tests for SNP effects under an additive model ignoring all exposures, followed by the 2 df test for joint effects of G and GxE interaction and the 1 df test for GxE interaction alone. Any marker with a −log<sub>10</sub>(p-value) for either the 1 df test or the 2 df test &gt;6 (i.e. p-value≤10<sup>−6</sup>) was selected for further analysis.</p><p id="P12" class="p p-last">All genes identified as close to genome wide significant in this PBAT analysis were further examined with a genotypic TDT using conditional logistic regression models. In these models, the observed genotype of the case is compared to genotypes of 3 possible “pseudo-sib” controls in each trio. A conventional conditional logistic regression model specifies the log-odds of being the observed case in the i-th trio as: logit[P(case<sub>i</sub>)]=β<sub>0</sub>+β<sub>G</sub>(G<sub>i</sub>)+ β<sub>GxE</sub>(G<sub>i</sub>*E<sub>i</sub>) where G=0, 1, or 2 reflects the number of high risk alleles in the case/”pseudo-sib” control, and E=0/1 for unexposed and exposed trios, respectively. The regression coefficients (β<sub>G</sub> and β<sub>GxE</sub>, respectively) represent effects of G and GxE interaction on risk. Exposure specific odds ratio of being a CP case for carriers of a high risk allele were calculated; i.e. for unexposed carriers, this becomes OR(CP|G no E)=exp(β<sub>G</sub>), and for exposed carriers, this is OR(CP|G and E)=exp(β<sub>G</sub>+β<sub>GE</sub>). For any gene with several SNPs showing evidence of possible GxE interaction, these estimated OR were plotted along with p-values from the 1 df test from the likelihood ratio test (LRT) for either β<sub>GE</sub> alone or the 2 df combined test of β<sub>G</sub>=β<sub>GE</sub>=0.</p></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">RESULTS</h2><p id="P13" class="p p-first">A conventional search for marginal gene (G) effects influencing risk to CP in the total sample of 550 trios showed no markers achieved significance at a strict genome-wide level (p≤10<sup>−7</sup>). <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figure 1</a> presents a Manhattan plot for all autosomal and X-linked SNPs. There were 7 SNPs with asymptotic p~10<sup>−6</sup>, and two of these mapped to the <em>DSC3</em> gene on chr. 18, which contained several additional SNPs giving p&lt;0.001. Four other genes (<em>TPP2, HTR1B, BCL6</em> and <em>MEST</em>) had a single SNP with p~10<sup>−6</sup>, plus at least one other SNP yielding p&lt;0.001. Thus, while the initial GWAS failed to yield evidence of individual genes controlling risk to CP in these 550 case-parent trios (see QQ plot in <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figure 2</a>), several SNPs did yield suggestive evidence.</p><p id="P14">A genome-wide screen for GxE interaction was carried out using PBAT under the strategy described above, where both the 2 df test for gene (G) and gene-environment (GxE) interaction and the 1 df test for (GxE) interaction alone were examined. This screening process revealed several markers achieving genome-wide significance, especially for the 1 df test of GxE interaction (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figure 3</a>). To further investigate this evidence, <a href="/pmc/articles/PMC3180858/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> presents “double Manhattan plots” to summarize evidence for G and GxE interaction effects for each of the three maternal exposures (<a href="/pmc/articles/PMC3180858/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a> for GxAlcohol; <a href="/pmc/articles/PMC3180858/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -5em;">GxAlcohol;1B</span></span><span>1B</span></a> for GxSmoking and <a href="/pmc/articles/PMC3180858/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and1C</span></span><span>1C</span></a> for GxVitamin). In these plots, the bottom half shows the log<sub>10</sub>(p-value) for the conventional family based test of SNP effects <em>ignoring exposure</em> (where more significant results fall farther below the mid-line). In the top half, the −log<sub>10</sub>(p-values) are presented for each autosomal SNP from both the 2 df test of G and GxE interaction together (red dots) and the 1 df test for GxE interaction alone (blue dots). Only SNPs yielding asymptotic p&lt;0.0001 in either of these two tests were included to minimize clutter. Dashed lines connect p-values from the marginal test ignoring exposures (below mid-line) to those models considering GxE interaction in either the 1 df test or the 2 df test (above mid-line). Here the very strongest signals for G effects ignoring exposures were omitted (i.e. p-value&lt;0.00001 in the conventional TDT) to highlight those SNPs showing evidence of GxE interaction, which favors identifying genes suggesting GxE interaction. We focused on SNPs in genes yielding p&lt;10<sup>−6</sup> in one or another test for GxE interaction for further analysis. As seen in <a href="/pmc/articles/PMC3180858/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a>, 8 markers gave p&lt;10<sup>−6</sup> in the 1 df test for GxAlcohol interaction among autosomal SNPs, including 3 in <em>MLLT3</em> on chr. 9q22.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3180858/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697101746160" class="figure"><a href="/pmc/articles/PMC3180858/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3180858_nihms-297057-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-297057-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3180858/bin/nihms-297057-f0001.jpg"></a></div><div id="largeobj_idm140697101746160" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3180858/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3180858/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Double Manhattan plots for SNP effects ignoring maternal exposures (lower half) and considering G and G×E interaction for three maternal exposures. Blue dots represent –log<sub>10</sub>(p-value) from 1 df test of G×E interaction alone; red dots represent – log<sub>10</sub>(p-value) from 2 df test of G and G×E interaction. Only autosomal SNPs showing p&lt;0.0001 for one of these two tests are displayed. Dashed lines connect SNP showing this level of significance in one test considering G×E interaction with their corresponding p-value when interaction is ignored. Panel A: Results for GxMaternal Alcohol Consumption interaction; Panel B: Results for GxMaternal Smoking interaction; Panel C: Results for GxMaternal Vitamin Use interaction.</p></div></div></div><p id="P15"><a href="/pmc/articles/PMC3180858/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a> lists all genes (including 3 pseudo-genes and 1 open reading frame) showing evidence of GxE interaction for any of the three maternal exposures at p≤10<sup>−6</sup>, along with a total count of SNPs mapping to this gene and the count of additional SNPs yielding p&lt;0.01 in either the 1 df test for GxE alone or the 2 df test for G and GxE interaction together. We dropped 5 genes with one SNP each yielding p&lt;10<sup>−6</sup> in either test for GxE interaction but ≤3 additional SNPs showing p&lt;10<sup>−2</sup> (<em>AGXT2, HMP19, PRDM14, BTN2A</em> and <em>ETV6</em>). While we examined all genes listed in <a href="/pmc/articles/PMC3180858/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a> (including pseudo-genes), here we focus on recognized genes showing evidence of GxE interaction (noted in bold in <a href="/pmc/articles/PMC3180858/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a> and labeled in <a href="/pmc/articles/PMC3180858/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table III</h3><!--caption a7--><div class="caption"><p id="__p8">Genes yielding p-values &lt;10<sup>−6</sup> in either the 2 df test for G and GxE or the 1 df test for GxE with one or more maternal exposures in genome wide screen using PBAT on 550 CP case-parent trios. Total counts of SNPs mapping to genes and numbers of SNPs yielding p&lt;0.01 in either test</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697033287264" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="center">Gene (Chr)</th><th colspan="2" rowspan="1" valign="middle" align="center"># SNPs p&lt;0.01</th><th rowspan="2" colspan="1" valign="middle" align="center">Total # SNPs</th><th rowspan="2" colspan="1" valign="middle" align="center">Exp.</th><th rowspan="2" colspan="1" valign="middle" align="center">Gene (Chr)</th><th colspan="2" rowspan="1" valign="middle" align="center"># SNPs p&lt;0.01</th><th rowspan="2" colspan="1" valign="middle" align="center">Total # SNPs</th><th rowspan="2" colspan="1" valign="middle" align="center">Exp.</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">in 2 df test</th><th rowspan="1" colspan="1" valign="middle" align="center">in 1 df test</th><th rowspan="1" colspan="1" valign="middle" align="center">in 2 df test</th><th rowspan="1" colspan="1" valign="middle" align="center">in 1 df test</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">LOC645762 (4)</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">70</td><td rowspan="1" colspan="1" valign="middle" align="center">Alc</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>ZNF236 (18)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">39</td><td rowspan="1" colspan="1" valign="middle" align="center">Smk</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">c6orf105 (6)</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="center">126</td><td rowspan="1" colspan="1" valign="middle" align="center">Alc</td><td rowspan="1" colspan="1" valign="middle" align="center">ACOXL (2)</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">82</td><td rowspan="1" colspan="1" valign="middle" align="center">Vit</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>MLLT3 (9)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">144</td><td rowspan="1" colspan="1" valign="middle" align="center">Alc</td><td rowspan="1" colspan="1" valign="middle" align="center">LOC391828 (5)</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">52</td><td rowspan="1" colspan="1" valign="middle" align="center">Vit</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>SMC2 (9)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">141</td><td rowspan="1" colspan="1" valign="middle" align="center">Alc</td><td rowspan="1" colspan="1" valign="middle" align="center">LOC392027 (7)</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">214</td><td rowspan="1" colspan="1" valign="middle" align="center">Vit</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">OBSCN (1)</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">27</td><td rowspan="1" colspan="1" valign="middle" align="center">Smk</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>BAALC (8)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="center">61</td><td rowspan="1" colspan="1" valign="middle" align="center">Vit</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>TBK1 (12)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="center">Smk</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140697033287264" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3180858/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P16">When considering maternal alcohol exposure, <em>MLLT3</em> and <em>SMC2</em> on chr. 9 yielded evidence of GxAlcohol interaction. A total of 144 SNPs mapped to <em>MLLT3</em>, and 3 of these gave strong evidence of GxAlcohol interaction in the 1 df test (rs4621895, p=1.9*10<sup>−7</sup>; rs668703, p=6.6*10<sup>−7</sup>; and rs4977433, p=1.7*10<sup>−6</sup>). A cluster of 7 adjacent SNPs showed evidence of G and GxE interaction, even though none were significant when exposure to maternal alcohol was ignored (see column 3 in <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table I</a>). Six of these 7 SNPs yielded nominal significance (p&lt;0.05) in both the 1 df test for GxE interaction alone and in the 2 df test for combined effects of G and GxE interaction (last 2 columns of <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table I</a>). Pairwise linkage disequilibrium (LD) as measured by r<sup>2</sup> in Asian and European parents separately could not account for this pattern alone (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figure 4</a>).</p><p id="P17">Conditional logistic regression models were used to estimate the odds ratio of having CP given the infant carried one risk allele in the absence of exposure [OR(CP|G no E)] and in its presence [OR(CP|G and E)], along with their 95%CI. Estimated OR(CP|G no E) and OR(CP|C and E) for these 7 SNPs in <em>MLLT3</em> are presented in <a href="/pmc/articles/PMC3180858/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a> along with p-values from a LRT for the 1 df test in the conditional logistic regression framework. Here, an additive model was used and the apparent `high-risk allele' became the target allele (which was the minor allele for rs4621895, rs4977433, rs648703 and rs2780841, but the major allele for rs10757142 and rs6475464 -- see <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table I</a>). Estimated OR(CP|G and E) and their 95%CI for a heterozygous child exposed to maternal alcohol consumption were distinctly higher (open circles) compared to a similar unexposed child (solid circles).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3180858/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697031018496" class="figure"><a href="/pmc/articles/PMC3180858/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3180858_nihms-297057-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-297057-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3180858/bin/nihms-297057-f0002.jpg"></a></div><div id="largeobj_idm140697031018496" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3180858/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3180858/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Estimated OR(CP|G no E) and OR(CP|G and E) for maternal alcohol consumption from logistic regression on 550 case-parent trios. P-values from 1 df LRT for G×E interaction are shown along the X axis. Panel A: Estimated OR(CP|G no E) and OR(CP|G and E) for 7 SNPs in <em>MLLT3</em>. Panel B: Estimated OR(CP|G no E) and OR(CP|G and E) for 7 SNPs in <em>SMC2</em>.</p></div></div></div><p id="P18">Although none 141 SNPs mapping to <em>SMC2</em> on chr. 9q31.1 were significant at the p&lt;0.0001 level when maternal exposures were ignored, some did achieve nominal significance. When maternal alcohol consumption was considered, SNP rs1536895 yielded p=1.53*10<sup>−8</sup> in the 1 df test for GxE interaction from PBAT and an adjacent SNP approached genome wide significance (rs10125685, p=9.83*10<sup>−6</sup>). These SNPs identified a region spanning 11kb (and including 6 SNPs) where evidence of GxAlcohol interaction was apparent. When conditional logistic regression models were used to estimate exposure specific ORs, all 6 of these SNPs suggested modest G effects ignoring exposure (see column 3 of <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table II</a> for p-values from the LRT), and 5 of these 6 were also significant in the 2 df test for combined G and GxAlcohol interaction. <a href="/pmc/articles/PMC3180858/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a> shows estimated OR(CP|G no E) and OR(CP|G and E), plus their 95%CI, for these 6 SNPs under an additive model. For 4 separate SNPs, the putative high risk allele was associated with a doubling of risk when the fetus was exposed to maternal alcohol consumption, although some confidence intervals were quite wide.</p><p id="P19">Eighteen SNPs mapped to <em>TBK1</em> on chromosome 12q14.2, but only one was nominally significant when exposure to maternal smoking was ignored (rs2141765; p=0.0095). However, 4 SNPs were significant at p&lt;0.01 in the 2 df test for G and G×Smoking interaction, and 6 were significant at this level in the 1 df test for G×Smoking interaction alone (including rs7969932 with p=7.86*10<sup>−8</sup>), forming a cluster of 9 SNPs spanning 30 kb. In a conditional logistic regression model, 6 of these 9 SNPs were nominally significant in the 2 df test for combined effects of G and G×E interaction, and 5 were significant in the 1 df test for G×E interaction alone (<a href="/pmc/articles/PMC3180858/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a> which shows p-values for both the 1 df and 2 df tests; also <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table III</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3180858/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697027331936" class="figure"><a href="/pmc/articles/PMC3180858/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3180858_nihms-297057-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-297057-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3180858/bin/nihms-297057-f0003.jpg"></a></div><div id="largeobj_idm140697027331936" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3180858/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3180858/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Estimated OR(CP|G no E) and OR(CP|G and E) considering G×E interaction with maternal smoking in logistic regression on 550 case-parent trios. Panel A: Estimated OR(CP|G no E) and OR(CP|G and E) for 9 SNPs in <em>TBK1</em>. P-values from 2 df LRT for G and G×E interaction plus that from 1 df LRT for G×E interaction are shown along the X axis. Panel B: Estimated OR(CP|G no E) and OR(CP|G and E) for 10 SNPs in <em>ZNF236</em>. P-values from 1 df LRT for G×E interaction are shown along the X axis.</p></div></div></div><p id="P20"><em>ZNF236</em> on chromosome 18q22–q23 encompassed 39 SNPs, one of which yielded evidence of influencing risk when exposures were ignored (rs470337, p=0.015). However, rs372075 gave p=6.75*10<sup>−8</sup> in the 1 df test for G×E interaction and rs470563 gave p=6.91*10<sup>−6</sup> in the 2 df test. A block of 10 SNPs (spanning 57 kb) was examined using conditional logistic regression models, and again rs470337 was significant when maternal smoking was ignored (p-value=0.016; see <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table IV</a>). However, when maternal smoking was considered, 7 of these 10 SNPs were significant in the 2 df test for G and G×E interaction combined (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table IV</a>). These SNPs showed no evidence of influencing risk for unexposed infants, i.e. the 95%CI of the OR(CP|G no E) always overlapped the null hypothesis value of one. <a href="/pmc/articles/PMC3180858/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4B</span></a> shows estimated OR(CP|G and E) were distinctly higher for 6 of these SNPs.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3180858/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697065543424" class="figure"><a href="/pmc/articles/PMC3180858/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3180858_nihms-297057-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-297057-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3180858/bin/nihms-297057-f0004.jpg"></a></div><div id="largeobj_idm140697065543424" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3180858/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3180858/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p5">Estimated OR(CP|G no E) and OR(CP|G and E) for 11 SNPs in <em>BAALC</em> considering G×E interaction with maternal vitamin supplementation in logistic regression on 550 CP case-parent trios. P-values from 1 df LRT for G×E interaction are shown along the X axis.</p></div></div></div><p id="P21">Among the 61 SNPs mapping to <em>BAALC</em> on chr. 8q22.3, a block of 11 SNPs (spanning 34kb) yielded one SNP with significant evidence in the 1 df test for G×E interaction (rs6468862; p=2.03*10<sup>−7</sup>), plus 4 additional SNPs yielding nominal significance from PBAT. Only rs6468862 showed strong evidence of influencing risk in conditional logistic regression when maternal exposures were ignored (see column 3 of <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table V</a>). However, when G×E interaction was included in the model, 6 SNPs became significant in either the 1 or 2 df test (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Table V</a>), and the estimated OR(CP|G no E) and OR(CP|G and E) were distinct (see <a href="/pmc/articles/PMC3180858/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>).</p><p id="P22">Among the genes in <a href="/pmc/articles/PMC3180858/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a>, <em>OBSCN</em> on chr. 1q42.13 and ACOXL on chr. 2q13 deserve additional mention. The 27 SNPs mapping to <em>OBSCN</em> included 22 SNPs spanning 114 kb, 18 of which were nominally significant for G effects ignoring maternal exposures. When G×Smoking interaction was included, 17 of these 22 showed significant evidence in the 2 df test for G and G×Smoking interaction combined. Thus, <em>OBSCN</em> may represent “quantitative interaction” where exposure to maternal smoking enhances G effects. Among all the genes considered, only OBSCN showed any evidence of heterogeneity between trios of Asian and European ancestry in formal tests of heterogeneity considering G×Smoking interaction (data not shown). Because exposure to smoking is much lower among Asian mothers, however, it is difficult to confirm the absence of G×Smoking interaction in this group. <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figure 5</a> shows estimated OR(CP|G no E) and OR(CP|G and E) for these 22 SNPs. A total of 126 SNPs mapped to the <em>ACOXL</em> gene, and these included a block of 24 adjacent SNPs (spanning 64 kb) of which 11 were significant in the 1 df test for G×Vitamin interaction (including rs7602030, p=3.13*10<sup>−7</sup>). When conditional logistic regression models were used to estimate OR(CP|G no E) and OR(CP|G and E), 10 SNPs in <em>ACOXL</em> showed significantly lower risk to the child if the mother used multivitamin supplements (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figure 6</a>).</p><p id="P23" class="p p-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figures 7–10</a> illustrate estimated effects as OR(CP|G no E) and OR(CP|G and E) for 3 pseudo-genes (LOC645762 on chr. 4; LOC391828 on chr. 5; and LOC392027 on chr. 7) and for the open reading frame c6orf105 on chr. 6. Each of these putative genes met the criteria used to select genes giving evidence of G×E interaction, but 2 of these 3 pseudo-genes (LOC645762 and LOC391828) have been dropped from the latest version of the human genome (Build 37) and the inferred status of the remaining pseudo-genes makes it difficult to assess their true relevance.</p></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">DISCUSSION</h2><p id="P24" class="p p-first">Our initial GWAS of 550 CP case-parent trios failed to show any markers achieving genome-wide significance after Bonferroni correction (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figures 1 and 2</a>), although <em>DSC3</em> (desmocollin 3; MIM ID *600271) on chr. 18q12.1 showed several SNPs giving suggestive evidence. When markers were screened using family based association tests considering both G and G×E interaction, however, several markers yielded p-values at or near genome-wide significance (<a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Figure 3</a>). Several genes had SNPs showing evidence attaining or approaching genome-wide significance plus blocks of adjacent SNPs yielding additional statistical evidence (<a href="/pmc/articles/PMC3180858/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a>). We estimated OR(CP|G no E) and OR(CP|G and E) for blocks of adjacent SNPs to illustrate how common maternal exposures can alter effects of markers on risk to nonsyndromic CP.</p><p id="P25">While none of the genes listed in <a href="/pmc/articles/PMC3180858/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a> are recognized candidates for oral clefts, further investigation is warranted. <em>MLLT3</em> (myeloid/lymphoid or mixed-lineage leukemia) is a large gene on 9p22 often involved in translocations; although the region of signal represented in <a href="/pmc/articles/PMC3180858/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a> contains only intronic SNPs. The <em>SMC2</em> (structural maintenance of chromosomes 2) gene on 9q31.1 plays a role in mitotic chromosome condensation and DNA repair, and SNPs represented in <a href="/pmc/articles/PMC3180858/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a> are upstream of coding sequences. <em>TBK1</em> (tank-binding kinase 1 on chr. 12q14) is a key signaling molecule in the <em>NF</em>κ<em>B</em> pathway, however, the region of signal shown in <a href="/pmc/articles/PMC3180858/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a> lies outside the gene. <em>ZNF236</em> gene on chr. 18q22–q23 has two alternatively spliced transcripts and is widely expressed; although the region represented in <a href="/pmc/articles/PMC3180858/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3B</span></a> includes mostly intronic SNPs. The <em>BAALC</em> (brain and acute leukemia gene cytoplasmic) gene on chr. 8q22.3 is largely expressed in neural tissues and has several isoforms. The region of signal represented in <a href="/pmc/articles/PMC3180858/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a> lies partially outside this gene, encompassing part of the hypothetical protein coding region <em>FLJ10489</em>. The <em>OBSCN</em> gene (obscurin) on chr 1q42.13 containing several immunoglobulin-like domains, and is located close to <em>WNT3A</em>, recently reported to be associated with non-syndromic CL/P in a Chinese sample [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154109">Yao et al., 2010</a>]. It is important to remember location of a peak signal in a GWAS may lie some distance from any etiologic variant, so these genes showing peak signals may not be functional.</p><p id="P26" class="p">Our GWAS results argue the genetic etiology of non-syndromic CP is distinct from CL/P because none of the genes identified in our analysis of non-syndromic CL/P trios were identified here (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_233154119">Beaty et al., 2010</a>). This difference cannot be attributed to the smaller numbers of CP trios alone, because analysis of 547 CL trios showed clear support for regions such as chr. 8q24 and genes such as <em>IRF6</em>. In this study of CP trios, we only achieved genome-wide significance when G×E interaction was explicitly considered, thus it remains prudent to separate CP from CL/P.</p><div id="S11" class="sec sec-last"><h3 id="S11title">Statistical vs. biological interaction</h3><p id="P27" class="p p-first">Biological interaction is said to exist when both a gene (G) and an environmental factor (E) contribute in the etiology of a disease, and E alters the impact of G on risk (increasing or decreasing risk among carriers of G). Statistical interaction, however, is defined as a detectable deviation from predicted joint effects of G and E under a specific model (typically an additive or a multiplicative model). Thus, statistical interaction may or may not correspond to biological interaction.</p><p id="P28" class="p p-last">Several patterns of statistical interaction are possible [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320766194">Kraft and Hunter, 2005</a>], including “pure interaction” where neither the gene (G) nor the environmental exposure (E) has any detectable effect alone, but when both are present there is a strong impact on risk. More typical is “quantitative interaction” where the effect of one exposure is enhanced by the presence of the second, e.g. G may have a modest effect in the absence of E but a greater effect in its presence. The converse is termed “qualitative interaction”, where the G effects are reversed in the presence of exposure E. These are crude categories and if dose effects are strong, one could envision a transition point where extreme levels of exposure would blur the lines between categories. Power to detect statistical interaction in observational studies is determined by the true type of interaction (pure, quantitative, etc.), but more importantly by the allele frequencies at the unobserved G controlling risk and by the prevalence of exposure to the environmental risk factor E. Ignoring G×E interaction when it exists could easily lead to overlooking the role of genes influencing risk. Without confirmatory data from independent samples, it is difficult to conclude with certainty the suggestive evidence for G×E interaction seen here is important to the etiology of CP. However, replication studies will require adequate numbers of CP cases to provide sufficient statistical power to detect G×E interaction, and this study is the largest collection of CP case-parent trios reported to date. Nonetheless, the present analysis does clearly show ignoring potential G×E interaction will overlook genes controlling risk to a common birth defect of complex and heterogeneous etiology.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp Table S1-S5&amp;Figure S1-S10</h4><div class="sup-box half_rhythm" id="N0x73f0240N0x8328b00"><a href="/pmc/articles/PMC3180858/bin/NIHMS297057-supplement-Supp_Table_S1-S5_Figure_S1-S10.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.7M, doc)</sup></div></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p1">We sincerely thank all of the families at each recruitment site for participating in this international study, and we gratefully acknowledge the invaluable assistance of clinical, field and laboratory staff who contributed to making this work possible. We are particularly grateful to K. Durda and J. L'Heureux of the University of Iowa for assistance with samples and phenotype data: L. Henkle for technical assistance; J. Resick, C. Brandon, and K. Bardi of the University of Pittsburgh for assistance with recruitment; W. Carricato, K. Deeley, and J. Ruff of the University of Pittsburgh for sample handling. We also thank Dr. Christoph Lange for providing guidance in using the PBAT software. Funding to support data collection, genotyping and analysis came from several sources, some to individual investigators (R01-DE-014581, R01-DE-016148, P50-DE-016215, R21-DE-016930, R01-DE-09886, R37-DE-08559, U01-DE-020057, R37-DE-0-8559, U01-DE-020057, R01-DE-012472, R01-DE-0106877) and some to the cleft consortium itself (U01-DE-018993); part of this research was supported by the Intramural Research Program of the NIH, National Institute of Environmental Health Sciences. This project was part of the Gene, Environment Association Studies (GENEVA) Consortium funded by the National Human Genome Research Institute (NHGRI) to enhance communication and collaboration among researchers conducting genome-wide studies of complex diseases. Our group benefited greatly from the work and efforts of the entire consortium, especially the Coordinating Center (directed by B. Weir and C. Laurie of the University of Washington; U01-<a id="__tag_801257094" class="tag_hotlink" href="/nuccore/HG004446" ref="/nuccore/HG004446"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG004446","term_id":"585865293","term_text":"HG004446"}}</span>HG004446</a>) in data cleaning and preparation for submission to the Database for Genotypes and Phenotypes (dbGaP). We also acknowledge the leadership of T. Manolio of NHGRI and E. Harris of National Institute of Dental and Craniofacial Research (NIDCR). Genotyping services were provided by the Center for Inherited Disease Research (CIDR).</p></div></div><div id="idm140697026614256" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140697026614256title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P29" class="p p-first-last">Web Resources: Online Mendelian Inheritance in Man (OMIM), <a href="http://www.ncbi.nlm.nih.gov/Omim/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/Omim/</a></p><p></p></div></div><div id="idm140697020441056" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140697020441056title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Beaty TH, Murray JC, Marazita ML, Munger RG, Ruczinski I, et al.  A genome wide association study of cleft lip with and without cleft palate identifies risk variants near MAFB and ABCA4. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:525–29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2941216/">PMC free article</a>]</span> [<a href="/pubmed/20436469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome+wide+association+study+of+cleft+lip+with+and+without+cleft+palate+identifies+risk+variants+near+MAFB+and+ABCA4&amp;author=TH+Beaty&amp;author=JC+Murray&amp;author=ML+Marazita&amp;author=RG+Munger&amp;author=I+Ruczinski&amp;volume=42&amp;publication_year=2010&amp;pages=525-29&amp;pmid=20436469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">DeRoo LA, Wilcox AJ, Drevon CA, Lie RT. First-trimester maternal alcohol consumption and the risk of infant oral clefts in Norway: a population-based case-control study. <span><span class="ref-journal">Am J Epidemiol. </span>2008;<span class="ref-vol">168</span>:638–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2727199/">PMC free article</a>]</span> [<a href="/pubmed/18667525" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=First-trimester+maternal+alcohol+consumption+and+the+risk+of+infant+oral+clefts+in+Norway:+a+population-based+case-control+study&amp;author=LA+DeRoo&amp;author=AJ+Wilcox&amp;author=CA+Drevon&amp;author=RT+Lie&amp;volume=168&amp;publication_year=2008&amp;pages=638-46&amp;pmid=18667525&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Genisca AE, Frías JL, Broussard CS, Honein MA, Lammer EJ, Moore CA, et al.  National Birth Defects Prevention Study. Orofacial clefts in the National Birth Defects Prevention Study, 1997–2004. <span><span class="ref-journal">Am J Med Genet A. </span>2009;<span class="ref-vol">149A</span>:1149–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3111146/">PMC free article</a>]</span> [<a href="/pubmed/19441124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=National+Birth+Defects+Prevention+Study.+Orofacial+clefts+in+the+National+Birth+Defects+Prevention+Study,+1997–2004&amp;author=AE+Genisca&amp;author=JL+Frías&amp;author=CS+Broussard&amp;author=MA+Honein&amp;author=EJ+Lammer&amp;volume=149A&amp;publication_year=2009&amp;pages=1149-58&amp;pmid=19441124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Ghassibe M, Desmyter L, Langenberg F, Claes O, Boute B, et al.   <span class="ref-journal">Am Hum Genet.</span> 2010. FAF1, the first gene associated with cleft palate. in press. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Am+Hum+Genet&amp;author=M+Ghassibe&amp;author=L+Desmyter&amp;author=F+Langenberg&amp;author=O+Claes&amp;author=B+Boute&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Grosen D, Chevrier C, Skytthe A, Bille C, Molsted K, et al.  A cohort study of recurrence patterns among more than 54,000 relatives of oral cleft cases in Denmark: Support for the multifactorial threshold model of inheritance. <span><span class="ref-journal">J Med Genet. </span>2009;<span class="ref-vol">47</span>:162–68.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2909851/">PMC free article</a>]</span> [<a href="/pubmed/19752161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=A+cohort+study+of+recurrence+patterns+among+more+than+54,000+relatives+of+oral+cleft+cases+in+Denmark:+Support+for+the+multifactorial+threshold+model+of+inheritance&amp;author=D+Grosen&amp;author=C+Chevrier&amp;author=A+Skytthe&amp;author=C+Bille&amp;author=K+Molsted&amp;volume=47&amp;publication_year=2009&amp;pages=162-68&amp;pmid=19752161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Grosen D, Bille C, Petersen I, Skytthe A, Hjelmborg J, et al.  Risk of oral clefts in twins. <span><span class="ref-journal">Epidemiol. </span>2011;<span class="ref-vol">22</span>:313–319.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3089024/">PMC free article</a>]</span> [<a href="/pubmed/21423016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epidemiol&amp;title=Risk+of+oral+clefts+in+twins&amp;author=D+Grosen&amp;author=C+Bille&amp;author=I+Petersen&amp;author=A+Skytthe&amp;author=J+Hjelmborg&amp;volume=22&amp;publication_year=2011&amp;pages=313-319&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Hwang SJ, Beaty TH, Panny SR, Street NA, Joseph JM, et al.  Association study of transforming growth factor alpha (TGFα) TaqI polymorphism and oral clefts: Indication of gene-environment interaction in a population based sample of infants with birth defects. <span><span class="ref-journal">Am J Epidemiol. </span>1995;<span class="ref-vol">141</span>:629–36.</span> [<a href="/pubmed/7702037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Association+study+of+transforming+growth+factor+alpha+(TGFα)+TaqI+polymorphism+and+oral+clefts:+Indication+of+gene-environment+interaction+in+a+population+based+sample+of+infants+with+birth+defects&amp;author=SJ+Hwang&amp;author=TH+Beaty&amp;author=SR+Panny&amp;author=NA+Street&amp;author=JM+Joseph&amp;volume=141&amp;publication_year=1995&amp;pages=629-36&amp;pmid=7702037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Johnson CY, Little J. Folate intake, markers of folate status and oral clefts: is the evidence converging? <span><span class="ref-journal">Intl J Epidemiol. </span>2008;<span class="ref-vol">37</span>:1041–1058.</span> [<a href="/pubmed/18583393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Intl+J+Epidemiol&amp;title=Folate+intake,+markers+of+folate+status+and+oral+clefts:+is+the+evidence+converging?&amp;author=CY+Johnson&amp;author=J+Little&amp;volume=37&amp;publication_year=2008&amp;pages=1041-1058&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Kraft P, Hunter D. Integrating epidemiology and genetic association: the challenge of gene-environment interaction. <span><span class="ref-journal">Phil Trans Soc B. </span>2005;<span class="ref-vol">360</span>:1609–16.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1569522/">PMC free article</a>]</span> [<a href="/pubmed/16096111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phil+Trans+Soc+B&amp;title=Integrating+epidemiology+and+genetic+association:+the+challenge+of+gene-environment+interaction&amp;author=P+Kraft&amp;author=D+Hunter&amp;volume=360&amp;publication_year=2005&amp;pages=1609-16&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Kraft P, Yen YC, Stram DO, Morrison J, Gauderman WJ. Exploiting gene-environment interaction to detect genetic associations. <span><span class="ref-journal">Hum Hered. </span>2007;<span class="ref-vol">63</span>:111–19.</span> [<a href="/pubmed/17283440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Exploiting+gene-environment+interaction+to+detect+genetic+associations&amp;author=P+Kraft&amp;author=YC+Yen&amp;author=DO+Stram&amp;author=J+Morrison&amp;author=WJ+Gauderman&amp;volume=63&amp;publication_year=2007&amp;pages=111-19&amp;pmid=17283440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Laird NM, Horvath S, Xu X. Implementing a unified approach to family-based tests of association. <span><span class="ref-journal">Genet Epidemiol. </span>2000;<span class="ref-vol">19</span>:S36–S42.</span> [<a href="/pubmed/11055368" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Implementing+a+unified+approach+to+family-based+tests+of+association&amp;author=NM+Laird&amp;author=S+Horvath&amp;author=X+Xu&amp;volume=19&amp;publication_year=2000&amp;pages=S36-S42&amp;pmid=11055368&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Laird NM, Lange C. Family-based designs in the age of large-scale gene-association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2006;<span class="ref-vol">7</span>:385–94.</span> [<a href="/pubmed/16619052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Family-based+designs+in+the+age+of+large-scale+gene-association+studies&amp;author=NM+Laird&amp;author=C+Lange&amp;volume=7&amp;publication_year=2006&amp;pages=385-94&amp;pmid=16619052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Lange C, Silverman EK, Xu X, Weiss ST, Laird NM. A multivariate family based association test using generalized estimating equations: FBAT-GEE. <span><span class="ref-journal">Biostatistics. </span>2003;<span class="ref-vol">4</span>:195–206.</span> [<a href="/pubmed/12925516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biostatistics&amp;title=A+multivariate+family+based+association+test+using+generalized+estimating+equations:+FBAT-GEE&amp;author=C+Lange&amp;author=EK+Silverman&amp;author=X+Xu&amp;author=ST+Weiss&amp;author=NM+Laird&amp;volume=4&amp;publication_year=2003&amp;pages=195-206&amp;pmid=12925516&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Li L, Meng T, Jia Z, Zhu G, Shi B. Single nucleotide polymorphism associated with nonsyndromic cleft palate influences the processing of miR-140. <span><span class="ref-journal">Am J Med Genet A. </span>2009;<span class="ref-vol">152A</span>:856–62.</span> [<a href="/pubmed/20358594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Single+nucleotide+polymorphism+associated+with+nonsyndromic+cleft+palate+influences+the+processing+of+miR-140&amp;author=L+Li&amp;author=T+Meng&amp;author=Z+Jia&amp;author=G+Zhu&amp;author=B+Shi&amp;volume=152A&amp;publication_year=2009&amp;pages=856-62&amp;pmid=20358594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Li L, Zhu G, Meng T, Wu J, Shi B.  <span class="ref-journal">Mother passive smoking, a genetic polymorphism in pre-miR-140 and non-syndromic cleft palate: a biological and epidemiological study of gene-environment interaction.</span> 2010.  in press. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Mother+passive+smoking,+a+genetic+polymorphism+in+pre-miR-140+and+non-syndromic+cleft+palate:+a+biological+and+epidemiological+study+of+gene-environment+interaction&amp;author=L+Li&amp;author=G+Zhu&amp;author=T+Meng&amp;author=J+Wu&amp;author=B+Shi&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Little J, Cardy A, Munger RG. Tobacco smoking and oral clefts: a meta-analysis. <span><span class="ref-journal">Bull World Health Organ. </span>2004;<span class="ref-vol">82</span>:213–18.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2585921/">PMC free article</a>]</span> [<a href="/pubmed/15112010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bull+World+Health+Organ&amp;title=Tobacco+smoking+and+oral+clefts:+a+meta-analysis&amp;author=J+Little&amp;author=A+Cardy&amp;author=RG+Munger&amp;volume=82&amp;publication_year=2004&amp;pages=213-18&amp;pmid=15112010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Manning AK, LaValley M, Liu CT, Rice K, An P, Liu Y, Miljkovic I, Rasmussen-Torvik L, Harris TB, Province MA, Borecki IB, Florez JC, Meigs JB, Cupples LA, Dupuis J. Meta-analysis of gene-environment interaction: Joint estimation of SNP and SNPxEnvironment regression coefficients. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:11–18.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3312394/">PMC free article</a>]</span> [<a href="/pubmed/21181894" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Meta-analysis+of+gene-environment+interaction:+Joint+estimation+of+SNP+and+SNPxEnvironment+regression+coefficients&amp;author=AK+Manning&amp;author=M+LaValley&amp;author=CT+Liu&amp;author=K+Rice&amp;author=P+An&amp;volume=35&amp;publication_year=2011&amp;pages=11-18&amp;pmid=21181894&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Maestri NE, Beaty TH, Hetmanski JB, Smith A, McIntosh I, et al.  Application of transmission disequilibrium tests to nonsyndromic oral clefts: Including candidate genes and environmental exposures in the models. <span><span class="ref-journal">Am J Med Genet. </span>1997;<span class="ref-vol">73</span>:337–44.</span> [<a href="/pubmed/9415696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Application+of+transmission+disequilibrium+tests+to+nonsyndromic+oral+clefts:+Including+candidate+genes+and+environmental+exposures+in+the+models&amp;author=NE+Maestri&amp;author=TH+Beaty&amp;author=JB+Hetmanski&amp;author=A+Smith&amp;author=I+McIntosh&amp;volume=73&amp;publication_year=1997&amp;pages=337-44&amp;pmid=9415696&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Marazita ML.  Segregation Analysis. In: Wyszynski D, editor. <span class="ref-journal">Cleft Lip and Palate: From Origin to Treatment.</span> Oxford University Press; Oxford: 2002.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Cleft+Lip+and+Palate:+From+Origin+to+Treatment&amp;author=ML+Marazita&amp;publication_year=2002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, et al.  PLINK: a toolset for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+toolset+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MAR+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Sivertsen A, Wilcox AJ, Skjaerven R, Vindense HA, Abyholm F, et al.  Familial risk of oral clefts by morphological type and severity: population based cohort study of first degree relatives. <span><span class="ref-journal">BMJ. </span>2008;<span class="ref-vol">336</span>:432–34.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2249683/">PMC free article</a>]</span> [<a href="/pubmed/18250102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Familial+risk+of+oral+clefts+by+morphological+type+and+severity:+population+based+cohort+study+of+first+degree+relatives&amp;author=A+Sivertsen&amp;author=AJ+Wilcox&amp;author=R+Skjaerven&amp;author=HA+Vindense&amp;author=F+Abyholm&amp;volume=336&amp;publication_year=2008&amp;pages=432-34&amp;pmid=18250102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Yao T, Yang L, Li PQ, Wu H, Xie HB, et al.  Association of Wnt3A gene variants with non-syndromic cleft lip with or without cleft palate in Chinese population. <span><span class="ref-journal">Arch Oral Biol. </span>2010;<span class="ref-vol">56</span>:73–78.</span> [<a href="/pubmed/20932509" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Oral+Biol&amp;title=Association+of+Wnt3A+gene+variants+with+non-syndromic+cleft+lip+with+or+without+cleft+palate+in+Chinese+population&amp;author=T+Yao&amp;author=L+Yang&amp;author=PQ+Li&amp;author=H+Wu&amp;author=HB+Xie&amp;volume=56&amp;publication_year=2010&amp;pages=73-78&amp;pmid=20932509&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Zeiger JS, Beaty TH, Liang KY. Oral clefts, maternal smoking and TGFA: Meta-analysis of gene-environment interaction. <span><span class="ref-journal">Cleft Palate-Craniofac J. </span>2005;<span class="ref-vol">42</span>:58–63.</span> [<a href="/pubmed/15643916" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cleft+Palate-Craniofac+J&amp;title=Oral+clefts,+maternal+smoking+and+TGFA:+Meta-analysis+of+gene-environment+interaction&amp;author=JS+Zeiger&amp;author=TH+Beaty&amp;author=KY+Liang&amp;volume=42&amp;publication_year=2005&amp;pages=58-63&amp;pmid=15643916&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3180858&amp;issue-id=200400&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140697030957184title">Abstract</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">METHODS</a></li><li><a href="#S9title">RESULTS</a></li><li><a href="#S10title">DISCUSSION</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S12title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140697026614256title">Footnotes</a></li><li><a href="#idm140697020441056title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3180858/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3180858/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3180858/pdf/nihms-297057.pdf">PDF (662K)</a></li> | <li><a href="#" data-citationid="PMC3180858" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3180858%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3180858%2F&amp;text=Evidence%20for%20gene-environment%20interaction%20in%20a%20genome%20wide%20study%20of%20isolated%2C%20non-syndromic%20cleft%20palate" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3180858%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21618603" data-aiid="3180858" data-aid="3180858" data-iid="200400" data-domainid="319" data-domain="nihpa" data-accid="PMC3180858" data-md5="565fb24b2b7cc0543d02fc704484e9e7"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:06:30-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A324ADD5221E10000000007180482&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3180858%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3180858/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>