<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3035981/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>GENOME-WIDE ASSOCIATION STUDY OF BIPOLAR DISORDER IN EUROPEAN AMERICAN AND AFRICAN AMERICAN INDIVIDUALS</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3035981/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Molecular psychiatry"><meta name="citation_title" content="GENOME-WIDE ASSOCIATION STUDY OF BIPOLAR DISORDER IN EUROPEAN AMERICAN AND AFRICAN AMERICAN INDIVIDUALS"><meta name="citation_authors" content="Erin N. Smith, Cinnamon S. Bloss, Judith A. Badner, Thomas Barrett, Pamela L. Belmonte, Wade Berrettini, William Byerley, William Coryell, David Craig, Howard J. Edenberg, Eleazar Eskin, Tatiana Foroud, Elliot Gershon, Tiffany A. Greenwood, Maria Hipolito, Daniel L. Koller, William B. Lawson, Chunyu Liu, Falk Lohoff, Melvin G. McInnis, Francis J. McMahon, Daniel B. Mirel, Caroline Nievergelt, John Nurnberger, Evaristus A. Nwulia, Justin Paschall, James B. Potash, John Rice, Thomas G. Schulze, William Scheftner, Corrie Panganiban, Noah Zaitlen, Peter P. Zandi, Sebastian Zöllner, Nicholas J. Schork, John R. Kelsoe"><meta name="citation_date" content="August 2009"><meta name="citation_issue" content="8"><meta name="citation_volume" content="14"><meta name="citation_firstpage" content="755"><meta name="citation_doi" content="10.1038/mp.2009.43"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3035981/?report=abstract"><meta name="citation_pmid" content="19488044"><meta name="DC.Title" content="GENOME-WIDE ASSOCIATION STUDY OF BIPOLAR DISORDER IN EUROPEAN AMERICAN AND AFRICAN AMERICAN INDIVIDUALS"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Erin N. Smith"><meta name="DC.Contributor" content="Cinnamon S. Bloss"><meta name="DC.Contributor" content="Judith A. Badner"><meta name="DC.Contributor" content="Thomas Barrett"><meta name="DC.Contributor" content="Pamela L. Belmonte"><meta name="DC.Contributor" content="Wade Berrettini"><meta name="DC.Contributor" content="William Byerley"><meta name="DC.Contributor" content="William Coryell"><meta name="DC.Contributor" content="David Craig"><meta name="DC.Contributor" content="Howard J. Edenberg"><meta name="DC.Contributor" content="Eleazar Eskin"><meta name="DC.Contributor" content="Tatiana Foroud"><meta name="DC.Contributor" content="Elliot Gershon"><meta name="DC.Contributor" content="Tiffany A. Greenwood"><meta name="DC.Contributor" content="Maria Hipolito"><meta name="DC.Contributor" content="Daniel L. Koller"><meta name="DC.Contributor" content="William B. Lawson"><meta name="DC.Contributor" content="Chunyu Liu"><meta name="DC.Contributor" content="Falk Lohoff"><meta name="DC.Contributor" content="Melvin G. McInnis"><meta name="DC.Contributor" content="Francis J. McMahon"><meta name="DC.Contributor" content="Daniel B. Mirel"><meta name="DC.Contributor" content="Caroline Nievergelt"><meta name="DC.Contributor" content="John Nurnberger"><meta name="DC.Contributor" content="Evaristus A. Nwulia"><meta name="DC.Contributor" content="Justin Paschall"><meta name="DC.Contributor" content="James B. Potash"><meta name="DC.Contributor" content="John Rice"><meta name="DC.Contributor" content="Thomas G. Schulze"><meta name="DC.Contributor" content="William Scheftner"><meta name="DC.Contributor" content="Corrie Panganiban"><meta name="DC.Contributor" content="Noah Zaitlen"><meta name="DC.Contributor" content="Peter P. Zandi"><meta name="DC.Contributor" content="Sebastian Zöllner"><meta name="DC.Contributor" content="Nicholas J. Schork"><meta name="DC.Contributor" content="John R. Kelsoe"><meta name="DC.Date" content="2009 Aug"><meta name="DC.Identifier" content="10.1038/mp.2009.43"><meta name="DC.Language" content="en"><meta property="og:title" content="GENOME-WIDE ASSOCIATION STUDY OF BIPOLAR DISORDER IN EUROPEAN AMERICAN AND AFRICAN AMERICAN INDIVIDUALS"><meta property="og:type" content="article"><meta property="og:description" content="To identify Bipolar Disorder (BD) genetic susceptibility factors, we conducted two genome-wide association (GWA) studies: one involving a sample of individuals of European ancestry (EA; n = 1,001 cases; n = 1,033 controls) and one involving a sample of ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3035981/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3035981/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3035981/pdf/nihms178631.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C7104DD4F49F10000000005E503B0.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3035981%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3035981</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Mol Psychiatry</a></span>. Author manuscript; available in PMC 2011 Feb 9.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19488044" target="pmc_ext" ref="reftype=publisher&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Mol Psychiatry. 2009 Aug; 14(8): 755–763. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Jun 2. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fmp.2009.43" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/mp.2009.43</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3035981</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS178631</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19488044">19488044</a></div></div></div></div><h1 class="content-title">GENOME-WIDE ASSOCIATION STUDY OF BIPOLAR DISORDER IN EUROPEAN AMERICAN AND AFRICAN AMERICAN INDIVIDUALS</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Smith%20EN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093917353616" co-class="co-affbox">Erin N. Smith</a>,<sup>*,</sup><sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Bloss%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093917350528" co-class="co-affbox">Cinnamon S. Bloss</a>,<sup>*,</sup><sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Badner%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001567280" co-class="co-affbox">Judith A. Badner</a>,<sup>4</sup> <a href="/pubmed/?term=Barrett%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001564912" co-class="co-affbox">Thomas Barrett</a>,<sup>5</sup> <a href="/pubmed/?term=Belmonte%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001562592" co-class="co-affbox">Pamela L. Belmonte</a>,<sup>6</sup> <a href="/pubmed/?term=Berrettini%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001560272" co-class="co-affbox">Wade Berrettini</a>,<sup>7</sup> <a href="/pubmed/?term=Byerley%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001557952" co-class="co-affbox">William Byerley</a>,<sup>8</sup> <a href="/pubmed/?term=Coryell%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001555632" co-class="co-affbox">William Coryell</a>,<sup>9</sup> <a href="/pubmed/?term=Craig%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001553360" co-class="co-affbox">David Craig</a>,<sup>10</sup> <a href="/pubmed/?term=Edenberg%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001551040" co-class="co-affbox">Howard J. Edenberg</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Eskin%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001547792" co-class="co-affbox">Eleazar Eskin</a>,<sup>13</sup> <a href="/pubmed/?term=Foroud%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001545424" co-class="co-affbox">Tatiana Foroud</a>,<sup>12</sup> <a href="/pubmed/?term=Gershon%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001543152" co-class="co-affbox">Elliot Gershon</a>,<sup>4</sup> <a href="/pubmed/?term=Greenwood%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001540880" co-class="co-affbox">Tiffany A. Greenwood</a>,<sup>14</sup> <a href="/pubmed/?term=Hipolito%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001538512" co-class="co-affbox">Maria Hipolito</a>,<sup>15</sup> <a href="/pubmed/?term=Koller%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001536144" co-class="co-affbox">Daniel L. Koller</a>,<sup>16</sup> <a href="/pubmed/?term=Lawson%20WB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001533776" co-class="co-affbox">William B. Lawson</a>,<sup>15</sup> <a href="/pubmed/?term=Liu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001531504" co-class="co-affbox">Chunyu Liu</a>,<sup>4</sup> <a href="/pubmed/?term=Lohoff%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001529184" co-class="co-affbox">Falk Lohoff</a>,<sup>7</sup> <a href="/pubmed/?term=McInnis%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001526912" co-class="co-affbox">Melvin G. McInnis</a>,<sup>17</sup> <a href="/pubmed/?term=McMahon%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001524640" co-class="co-affbox">Francis J. McMahon</a>,<sup>18</sup> <a href="/pubmed/?term=Mirel%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001522320" co-class="co-affbox">Daniel B. Mirel</a>,<sup>19</sup> <a href="/pubmed/?term=Nievergelt%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001519952" co-class="co-affbox">Caroline Nievergelt</a>,<sup>14</sup> <a href="/pubmed/?term=Nurnberger%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001517680" co-class="co-affbox">John Nurnberger</a>,<sup>16</sup> <a href="/pubmed/?term=Nwulia%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140094001515408" co-class="co-affbox">Evaristus A. Nwulia</a>,<sup>15</sup> <a href="/pubmed/?term=Paschall%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999714880" co-class="co-affbox">Justin Paschall</a>,<sup>20</sup> <a href="/pubmed/?term=Potash%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999712512" co-class="co-affbox">James B. Potash</a>,<sup>6</sup> <a href="/pubmed/?term=Rice%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999710240" co-class="co-affbox">John Rice</a>,<sup>21</sup> <a href="/pubmed/?term=Schulze%20TG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999707968" co-class="co-affbox">Thomas G. Schulze</a>,<sup>18</sup> <a href="/pubmed/?term=Scheftner%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999705696" co-class="co-affbox">William Scheftner</a>,<sup>22</sup> <a href="/pubmed/?term=Panganiban%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999703328" co-class="co-affbox">Corrie Panganiban</a>,<sup>10</sup> <a href="/pubmed/?term=Zaitlen%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999701056" co-class="co-affbox">Noah Zaitlen</a>,<sup>13</sup> <a href="/pubmed/?term=Zandi%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999698784" co-class="co-affbox">Peter P. Zandi</a>,<sup>6</sup> <a href="/pubmed/?term=Z%26%23x000f6%3Bllner%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999696512" co-class="co-affbox">Sebastian Zöllner</a>,<sup>17</sup> <a href="/pubmed/?term=Schork%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999694240" co-class="co-affbox">Nicholas J. Schork</a>,<sup>1,</sup><sup>2</sup> and  <a href="/pubmed/?term=Kelsoe%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044" class="affpopup" co-rid="_co_idm140093999691184" co-class="co-affbox">John R. Kelsoe</a><sup>14,</sup><sup>23</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140093917353616"><h3 class="no_margin">Erin N. Smith</h3><p><sup>1</sup>Scripps Genomic Medicine and Scripps Translational Science Institute, La Jolla, CA 92037, USA</p><p><sup>2</sup>Department of Molecular and Experimental Medicine, The Scripps Research Institute, La Jolla, CA 92037, USA</p><div>Find articles by <a href="/pubmed/?term=Smith%20EN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Erin N. Smith</a></div></div><div id="_co_idm140093917350528"><h3 class="no_margin">Cinnamon S. Bloss</h3><p><sup>1</sup>Scripps Genomic Medicine and Scripps Translational Science Institute, La Jolla, CA 92037, USA</p><p><sup>3</sup>Scripps Health, La Jolla, CA 92037, USA</p><div>Find articles by <a href="/pubmed/?term=Bloss%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Cinnamon S. Bloss</a></div></div><div id="_co_idm140094001567280"><h3 class="no_margin">Judith A. Badner</h3><p><sup>4</sup>Department of Psychiatry, University of Chicago, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Badner%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Judith A. Badner</a></div></div><div id="_co_idm140094001564912"><h3 class="no_margin">Thomas Barrett</h3><p><sup>5</sup>Department of Psychiatry, Portland VA Medical Center, Portland, OR, 97239, USA</p><div>Find articles by <a href="/pubmed/?term=Barrett%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Thomas Barrett</a></div></div><div id="_co_idm140094001562592"><h3 class="no_margin">Pamela L. Belmonte</h3><p><sup>6</sup>Department of Psychiatry, Johns Hopkins School of Medicine, Baltimore, MD, 21287, USA</p><div>Find articles by <a href="/pubmed/?term=Belmonte%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Pamela L. Belmonte</a></div></div><div id="_co_idm140094001560272"><h3 class="no_margin">Wade Berrettini</h3><p><sup>7</sup>Department of Psychiatry, University of Pennsylvania, Philadelphia, PA, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Berrettini%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Wade Berrettini</a></div></div><div id="_co_idm140094001557952"><h3 class="no_margin">William Byerley</h3><p><sup>8</sup>Department of Psychiatry, University of California, San Francisco, CA, 94143, USA</p><div>Find articles by <a href="/pubmed/?term=Byerley%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">William Byerley</a></div></div><div id="_co_idm140094001555632"><h3 class="no_margin">William Coryell</h3><p><sup>9</sup>Department of Psychiatry, University of Iowa, Iowa City, IA 52242, USA</p><div>Find articles by <a href="/pubmed/?term=Coryell%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">William Coryell</a></div></div><div id="_co_idm140094001553360"><h3 class="no_margin">David Craig</h3><p><sup>10</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, 85004, USA</p><div>Find articles by <a href="/pubmed/?term=Craig%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">David Craig</a></div></div><div id="_co_idm140094001551040"><h3 class="no_margin">Howard J. Edenberg</h3><p><sup>11</sup>Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, IN, 46202, USA</p><p><sup>12</sup>Department of Medical and Molecular Genetics, Indiana University School of Medicine, Indianapolis, IN, 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Edenberg%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Howard J. Edenberg</a></div></div><div id="_co_idm140094001547792"><h3 class="no_margin">Eleazar Eskin</h3><p><sup>13</sup>Department of Computer Science, University of California, Los Angeles, Los Angeles, CA, 90095, USA</p><div>Find articles by <a href="/pubmed/?term=Eskin%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Eleazar Eskin</a></div></div><div id="_co_idm140094001545424"><h3 class="no_margin">Tatiana Foroud</h3><p><sup>12</sup>Department of Medical and Molecular Genetics, Indiana University School of Medicine, Indianapolis, IN, 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Foroud%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Tatiana Foroud</a></div></div><div id="_co_idm140094001543152"><h3 class="no_margin">Elliot Gershon</h3><p><sup>4</sup>Department of Psychiatry, University of Chicago, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Gershon%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Elliot Gershon</a></div></div><div id="_co_idm140094001540880"><h3 class="no_margin">Tiffany A. Greenwood</h3><p><sup>14</sup>Department of Psychiatry, University of California, San Diego, La Jolla, CA 92093, USA</p><div>Find articles by <a href="/pubmed/?term=Greenwood%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Tiffany A. Greenwood</a></div></div><div id="_co_idm140094001538512"><h3 class="no_margin">Maria Hipolito</h3><p><sup>15</sup>Department of Psychiatry, Howard University, Washington, D.C., 20060, USA</p><div>Find articles by <a href="/pubmed/?term=Hipolito%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Maria Hipolito</a></div></div><div id="_co_idm140094001536144"><h3 class="no_margin">Daniel L. Koller</h3><p><sup>16</sup>Department of Psychiatry, Indiana University School of Medicine, Indianapolis, IN, 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Koller%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Daniel L. Koller</a></div></div><div id="_co_idm140094001533776"><h3 class="no_margin">William B. Lawson</h3><p><sup>15</sup>Department of Psychiatry, Howard University, Washington, D.C., 20060, USA</p><div>Find articles by <a href="/pubmed/?term=Lawson%20WB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">William B. Lawson</a></div></div><div id="_co_idm140094001531504"><h3 class="no_margin">Chunyu Liu</h3><p><sup>4</sup>Department of Psychiatry, University of Chicago, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Chunyu Liu</a></div></div><div id="_co_idm140094001529184"><h3 class="no_margin">Falk Lohoff</h3><p><sup>7</sup>Department of Psychiatry, University of Pennsylvania, Philadelphia, PA, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Lohoff%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Falk Lohoff</a></div></div><div id="_co_idm140094001526912"><h3 class="no_margin">Melvin G. McInnis</h3><p><sup>17</sup>Department of Psychiatry, University of Michigan, Ann Arbor, MI, 48109, USA</p><div>Find articles by <a href="/pubmed/?term=McInnis%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Melvin G. McInnis</a></div></div><div id="_co_idm140094001524640"><h3 class="no_margin">Francis J. McMahon</h3><p><sup>18</sup>Genetic Basis of Mood and Anxiety Disorders Unit, National Institute of Mental Health Intramural Research Program, National Institutes of Health, US Dept of Health and Human Services, Bethesda, MD, 20892, USA</p><div>Find articles by <a href="/pubmed/?term=McMahon%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Francis J. McMahon</a></div></div><div id="_co_idm140094001522320"><h3 class="no_margin">Daniel B. Mirel</h3><p><sup>19</sup>Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, MA, 02142, USA</p><div>Find articles by <a href="/pubmed/?term=Mirel%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Daniel B. Mirel</a></div></div><div id="_co_idm140094001519952"><h3 class="no_margin">Caroline Nievergelt</h3><p><sup>14</sup>Department of Psychiatry, University of California, San Diego, La Jolla, CA 92093, USA</p><div>Find articles by <a href="/pubmed/?term=Nievergelt%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Caroline Nievergelt</a></div></div><div id="_co_idm140094001517680"><h3 class="no_margin">John Nurnberger</h3><p><sup>16</sup>Department of Psychiatry, Indiana University School of Medicine, Indianapolis, IN, 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Nurnberger%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">John Nurnberger</a></div></div><div id="_co_idm140094001515408"><h3 class="no_margin">Evaristus A. Nwulia</h3><p><sup>15</sup>Department of Psychiatry, Howard University, Washington, D.C., 20060, USA</p><div>Find articles by <a href="/pubmed/?term=Nwulia%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Evaristus A. Nwulia</a></div></div><div id="_co_idm140093999714880"><h3 class="no_margin">Justin Paschall</h3><p><sup>20</sup>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Paschall%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Justin Paschall</a></div></div><div id="_co_idm140093999712512"><h3 class="no_margin">James B. Potash</h3><p><sup>6</sup>Department of Psychiatry, Johns Hopkins School of Medicine, Baltimore, MD, 21287, USA</p><div>Find articles by <a href="/pubmed/?term=Potash%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">James B. Potash</a></div></div><div id="_co_idm140093999710240"><h3 class="no_margin">John Rice</h3><p><sup>21</sup>Division of Biostatistics, Washington University, St. Louis, MO, 63130, USA</p><div>Find articles by <a href="/pubmed/?term=Rice%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">John Rice</a></div></div><div id="_co_idm140093999707968"><h3 class="no_margin">Thomas G. Schulze</h3><p><sup>18</sup>Genetic Basis of Mood and Anxiety Disorders Unit, National Institute of Mental Health Intramural Research Program, National Institutes of Health, US Dept of Health and Human Services, Bethesda, MD, 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Schulze%20TG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Thomas G. Schulze</a></div></div><div id="_co_idm140093999705696"><h3 class="no_margin">William Scheftner</h3><p><sup>22</sup>Department of Psychiatry, Rush University, Chicago, IL, 60612, USA</p><div>Find articles by <a href="/pubmed/?term=Scheftner%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">William Scheftner</a></div></div><div id="_co_idm140093999703328"><h3 class="no_margin">Corrie Panganiban</h3><p><sup>10</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, 85004, USA</p><div>Find articles by <a href="/pubmed/?term=Panganiban%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Corrie Panganiban</a></div></div><div id="_co_idm140093999701056"><h3 class="no_margin">Noah Zaitlen</h3><p><sup>13</sup>Department of Computer Science, University of California, Los Angeles, Los Angeles, CA, 90095, USA</p><div>Find articles by <a href="/pubmed/?term=Zaitlen%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Noah Zaitlen</a></div></div><div id="_co_idm140093999698784"><h3 class="no_margin">Peter P. Zandi</h3><p><sup>6</sup>Department of Psychiatry, Johns Hopkins School of Medicine, Baltimore, MD, 21287, USA</p><div>Find articles by <a href="/pubmed/?term=Zandi%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Peter P. Zandi</a></div></div><div id="_co_idm140093999696512"><h3 class="no_margin">Sebastian Zöllner</h3><p><sup>17</sup>Department of Psychiatry, University of Michigan, Ann Arbor, MI, 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Z%26%23x000f6%3Bllner%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Sebastian Zöllner</a></div></div><div id="_co_idm140093999694240"><h3 class="no_margin">Nicholas J. Schork</h3><p><sup>1</sup>Scripps Genomic Medicine and Scripps Translational Science Institute, La Jolla, CA 92037, USA</p><p><sup>2</sup>Department of Molecular and Experimental Medicine, The Scripps Research Institute, La Jolla, CA 92037, USA</p><div>Find articles by <a href="/pubmed/?term=Schork%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">Nicholas J. Schork</a></div></div><div id="_co_idm140093999691184"><h3 class="no_margin">John R. Kelsoe</h3><p><sup>14</sup>Department of Psychiatry, University of California, San Diego, La Jolla, CA 92093, USA</p><p><sup>23</sup>Department of Psychiatry, VA San Diego Healthcare System, La Jolla, CA, 92151, USA</p><div>Find articles by <a href="/pubmed/?term=Kelsoe%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19488044">John R. Kelsoe</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140093919475552_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140093919475552_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140093919475552_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Scripps Genomic Medicine and Scripps Translational Science Institute, La Jolla, CA 92037, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Molecular and Experimental Medicine, The Scripps Research Institute, La Jolla, CA 92037, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Scripps Health, La Jolla, CA 92037, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Psychiatry, University of Chicago, Chicago, IL 60637, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Psychiatry, Portland VA Medical Center, Portland, OR, 97239, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Psychiatry, Johns Hopkins School of Medicine, Baltimore, MD, 21287, USA</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Psychiatry, University of Pennsylvania, Philadelphia, PA, 19104, USA</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Psychiatry, University of California, San Francisco, CA, 94143, USA</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Psychiatry, University of Iowa, Iowa City, IA 52242, USA</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, 85004, USA</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, IN, 46202, USA</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Medical and Molecular Genetics, Indiana University School of Medicine, Indianapolis, IN, 46202, USA</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Computer Science, University of California, Los Angeles, Los Angeles, CA, 90095, USA</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Psychiatry, University of California, San Diego, La Jolla, CA 92093, USA</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Psychiatry, Howard University, Washington, D.C., 20060, USA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Psychiatry, Indiana University School of Medicine, Indianapolis, IN, 46202, USA</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Psychiatry, University of Michigan, Ann Arbor, MI, 48109, USA</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Genetic Basis of Mood and Anxiety Disorders Unit, National Institute of Mental Health Intramural Research Program, National Institutes of Health, US Dept of Health and Human Services, Bethesda, MD, 20892, USA</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, MA, 02142, USA</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892, USA</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Division of Biostatistics, Washington University, St. Louis, MO, 63130, USA</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Psychiatry, Rush University, Chicago, IL, 60612, USA</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Department of Psychiatry, VA San Diego Healthcare System, La Jolla, CA, 92151, USA</div><div id="FN1">Address Correspondence to: Nicholas J. Schork, Ph.D., The Scripps Translational Science Institute, Department of Molecular and Experimental Medicine, The Scripps Research Institute, 10550 North Torrey Pines Avenue, MEM 275, La Jolla, CA 92037, 858-554-5705 (admin), 858-546-9284 (fax), <a href="mailto:dev@null" data-email="ude.sppircs@krohcsn" class="oemail">ude.sppircs@krohcsn</a></div><div id="FN2"><sup>*</sup>These authors contributed equally.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140093919475552_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19488044" target="pmc_ext" ref="reftype=publisher&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Mol Psychiatry</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3035981/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="8" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figure S1: </strong>Supplemental Figure 1 (S1): QQ and multidimensional scaling (MDS) plots for each study population Unadjusted –log(p-values) are shown in black, with MDS (top 4 components) adjustment in red and LAMP adjustment in blue. In the top left hand corner of each plot is the top 2 MDS components for each analysis plotted against each other. Genomic control λ values are shown given the current study size and corrected for a study size of 1000 cases and 1000 controls. In the case where EA and AA individuals are analyzed together, unadjusted λ levels are elevated because of different ratios of cases and controls in the two populations. All individuals shown in the MDS plots in the upper left corners were included in the analyses.</div><div><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Figure_S1.tif" data-ga-action="click_feat_suppl">NIHMS178631-supplement-Figure_S1.tif</a><span style="color:gray"> (5.1M)</span></div><div class="small guid">GUID:&nbsp;F3AC61E0-E8C2-4DAC-AD44-6D65570B1942</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figure S2: </strong>Supplemental Figure 2 (S2): Regions near top hits and areas of interest Regions +/- 250 kb around each SNP listed in Table 1 are shown. P-values are from the analysis where the SNP was identified. Genotyped SNPs are shown as circles, while imputed SNPs are shown as smaller diamonds. The primary SNP of interest is large and colored in black. Other SNPs are colored according to linkage disequilibrium levels with the primary SNP (r2), as calculated from Phase 3 HapMap data using CEU (for EA and EA + AA combined) or YRI (for AA) populations. Recombination rate (HapMap) is shown on the second y-axis in blue. RefSeq genes are shown with all possible exons; arrows indicate transcript direction. In the upper left hand corner of each graph, the genotype intensity plots are shown, with each color indicating the final genotype call (blue and red for homozygotes and purple for the heterozygote).</div><div><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Figure_S2.pdf" data-ga-action="click_feat_suppl">NIHMS178631-supplement-Figure_S2.pdf</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;5CE0B65F-3AE7-4704-B210-D0CB4F13F70D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figure S3: </strong>Supplemental Figure 3 (S3): Top regions characterized in Ferreria et al., +/- 250 kb Each circle represents a SNP, with the first y-axis indicating the p-value for Bipolar Disorder in EA individuals (MDS adjusted) for this study. Genotyped SNPs are indicated with a circle, while imputed SNPs are indicated with diamonds. The most strongly previously associated SNP is indicated by a large black circle. SNPs are colored shades of red depending on their linkage disequilibrium with the most strongly previously associated SNP (r2, calculated from HapMap CEU Phase 3 using Haploview). Recombination rate (HapMap) is shown on the second y-axis in blue. Green horizontal lines indicate haplotype association p-values from a 10 SNP sliding window. Below the plot are SNPs from genotyped SNPs from WTCCC Bipolar Disorder and STEP-BP studies, with p-value indicated by color. RefSeq genes are shown with all possible exons; arrows indicate transcript direction.</div><div><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Figure_S3.pdf" data-ga-action="click_feat_suppl">NIHMS178631-supplement-Figure_S3.pdf</a><span style="color:gray"> (773K)</span></div><div class="small guid">GUID:&nbsp;947F630A-FA2C-4750-AC30-FAAAC442A08F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figure S4: </strong>Supplemental Figure 4 (S4): Comparison of European ancestry estimates Red: HapMap and HGDP with (STRUCTURE, 3503 SNPs on Chr. 1); Green: STRUCTURE and LAMP subset (Hapmap, 3503 SNPs on Chr. 1); Blue: STRUCTURE and LAMP full set (HapMap, 3503 SNPs on Chr. 1 and all markers for LAMP).</div><div><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Figure_S4.tif" data-ga-action="click_feat_suppl">NIHMS178631-supplement-Figure_S4.tif</a><span style="color:gray"> (24M)</span></div><div class="small guid">GUID:&nbsp;76D58C7A-4A5F-456A-9C10-6B2680587D21</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table S5.</strong></div><div><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Table_S5.xls" data-ga-action="click_feat_suppl">NIHMS178631-supplement-Table_S5.xls</a><span style="color:gray"> (46K)</span></div><div class="small guid">GUID:&nbsp;2F7E8FDD-FC08-452B-BDE8-E5BC7AD60D6D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table S6.</strong></div><div><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Table_S6.xls" data-ga-action="click_feat_suppl">NIHMS178631-supplement-Table_S6.xls</a><span style="color:gray"> (52K)</span></div><div class="small guid">GUID:&nbsp;922E7B01-7DA5-4969-B7BD-8B921BDF4FB3</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table S7.</strong></div><div><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Table_S7.xls" data-ga-action="click_feat_suppl">NIHMS178631-supplement-Table_S7.xls</a><span style="color:gray"> (45K)</span></div><div class="small guid">GUID:&nbsp;1A4FDB01-74AF-4295-9749-41279511DE63</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">NIHMS178631-supplement-supplement_1.pdf</a><span style="color:gray"> (138K)</span></div><div class="small guid">GUID:&nbsp;6FE6FAC4-FAB6-441D-8005-72F388927A54</div></div></dd></dl></div><div id="idm140093999647968" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140093999647968title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">To identify Bipolar Disorder (BD) genetic susceptibility factors, we conducted two genome-wide association (GWA) studies: one involving a sample of individuals of European ancestry (EA; n = 1,001 cases; n = 1,033 controls) and one involving a sample of individuals of African ancestry (AA; n = 345 cases; n = 670 controls). For the EA sample, SNPs with strongest statistical evidence for association included rs5907577 in an intergenic region at Xq27.1 (p = 1.6 × 10<sup>-6</sup>) and rs10193871 in <em>NAP5</em> at 2q21.2 (p = 9.8 × 10<sup>-6</sup>). For the AA sample, SNPs with strongest statistical evidence for association included rs2111504 in <em>DPY19L3</em> at 19q13.11 (p = 1.5 × 10<sup>-6</sup>) and rs2769605 in <em>NTRK2</em> at 9q21.33 (p = 4.5 × 10<sup>-5</sup>). We also investigated whether we could provide support for three regions previously associated with BD, and we show that the <em>ANK3</em> region replicates in our sample, along with some support for <em>C15Orf53</em>; other evidence implicates BD candidate genes such as <em>SLITRK2</em>. We also tested the hypothesis that BD susceptibility variants exhibit genetic background-dependent effects; SNPs with the strongest statistical evidence for this included rs11208285 in <em>ROR1</em> at 1p31.3 (p = 1.4 × 10<sup>-6</sup>), rs4657247 in <em>RGS5</em> at 1q23.3 (p = 4.1 × 10<sup>-6</sup>), and rs7078071 in <em>BTBD16</em> at 10q26.13 (p = 4.5 × 10<sup>-6</sup>). This study is the first to conduct GWA of BD in individuals of AA and suggests that genetic variations that contribute to BD may vary as a function of ancestry.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">ANK3, C15Orf53, NAP5, DPY19L3, NTRK2, SLITRK2, ROR1, Bipolar Genome Study, Genetic Information Association Network (GAIN), genetic background, allelic heterogeneity</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P2" class="p p-first">Bipolar disorder (BD) is a paradigmatic complex phenotype with many genetic and non-genetic determinants. BD is characterized by episodes of mania and depression (<a href="#R1" rid="R1" class=" bibr popnode">1</a>). Onset is usually in late adolescence, although BD typically recurs and relapses throughout life. BD affects approximately 1% of the world’s population, and carries a lifetime risk for completed suicide as high as 17%. Family, twin, and adoption studies all support a substantial genetic component in BD (<a href="#R2" rid="R2" class=" bibr popnode">2</a>-<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428534">4</a>). The sibling recurrence risk is between 7 and 10, and heritability is estimated to be about 80%. While this is consistent with a strong genetic component, the identification of specific genetic variations that influence BD susceptibility has been difficult.</p><p id="P3">Although some family studies have suggested that BD has an autosomal dominant genetic determinant, the vast majority of genetic studies suggest that BD has a high level of genetic heterogeneity and a substantial polygenic component (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428514">5</a>, <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428528">6</a>). Linkage studies have identified a number of loci with inconsistent replication. While previous linkage studies were highly divergent (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428537">7</a>), recent meta-analyses of linkage studies have found consistent supportive evidence for linkage to a few potential BD susceptibility loci (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428541">8</a>-<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428521">10</a>), notably 6q, 8q, 13q, and 22q. A number of polymorphisms in a variety of candidate genes have been tested for association with BD, but most of the polymorphisms have shown no statistically compelling associations with BD; those that appear to be associated show odds ratios of 1.1 to 1.3, which is again consistent with a polygenic basis for BD (<a href="#R11" rid="R11" class=" bibr popnode">11</a>).</p><p id="P4">In the last several years the development of genome-wide association (GWA) study designs and analysis methods have made it possible to search for multiple genetic variations underlying a condition like BD without <em>a priori</em> assumptions about the genes or genomic regions that might harbor susceptibility variations (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428523">12</a>). The GWA study approach has revealed a number of genetic variations that are unequivocally associated with traits and diseases (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428539">13</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428531">14</a>). The US National Institutes of Mental Health–sponsored Genetics Initiative for Bipolar Disorder Consortium (Bipolar Consortium) has collected over 3,500 subjects with BD during 1990-2008. A genetic component of the Bipolar Consortium, termed the ‘Bipolar Genome Study (BiGS),’ was initiated in 2006 to conduct a GWA study of BD. The initial BiGS GWA study was funded through the Foundation for the National Institutes of Health Genetic Information Association Network (GAIN) initiative (<a href="http://www.genome.gov/19518664" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/19518664</a>) and we report the results of this GWA study here. Of note, many investigators have access to and have utilized the DNA samples from individuals with BD that were collected as part of the Bipolar Consortium; thus, it is the case that a portion of the samples on which we report here are not entirely independent of previously published BD GWA studies (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428519">15</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352561128">16</a>) and an ongoing BD GWA study (Scott, Muglia, Upmanyu, Guan, Flickinger, Kong, Tozzi, Li, Burmeister, Absher et al., submitted).</p><p id="P5" class="p p-last">We performed a GWA study of BD separately in individuals of European ancestry (EA) and of African ancestry (AA) using a variety of methods to control for population substructure and admixture among the individuals of AA. Our study is the first GWA study of BD involving individuals of AA. In addition to standard single locus analyses within the EA and AA groups, we also assess the extent to which single nucleotide polymorphisms (SNPs) exhibite evidence of genetic background-dependent effects or allelic heterogeneity across the EA and AA individuals. Although more than 190 GWA studies have been performed to date (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428508">12</a>) these studies have virtually all been with EA subjects (NHGRI catalog of published GWA studies: <a href="http://www.genome.gov/gwastudies/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/gwastudies/</a>), raising important questions as to the generalizability of the results to other populations. We assessed the consistency of the SNPs exhibiting the strongest associations with BD in our study with previously published results of BD GWA studies. We tested variation at <em>ANK3</em>, which is a candidate gene that was implicated in an earlier GWA study (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428511">15</a>) and identified as genome-wide significant in a recent collaborative analysis (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321054630">17</a>). Finally, we genotyped a subset of SNPs in the regions exhibiting strongest association in a replication case/control group and tracked the co-inheritance of these SNPs and disease in families.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">MATERIALS AND METHODS</h2><p id="P6" class="p p-first">See <a href="#SD9" rid="SD9" class=" supplementary-material">Supplemental Material</a> for information regarding study subjects and genotyping and quality control. The final number of BD cases was 1,001 EA subjects and 345 AA subjects; control counts were 1,033 EA subjects and 670 AA subjects. The final dataset consisted of 724,067 SNPs in the EA dataset and 840,730 SNPs in the AA dataset. The two datasets had 702,044 SNPs in common, and these were used to perform analyses addressing SNP and SNPxgenetic background interactions in the combined sample.</p><div id="S3" class="sec"><h3 id="S3title">Statistical Analysis</h3><div id="S4" class="sec sec-first"><p></p><h4 id="S4title" class="inline">Primary analyses </h4><p id="P7" class="p p-first-last">All genetic analyses were conducted using PLINK (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428545">18</a>) versions 1.03 and 1.04. Our primary analyses tested the association of each SNP (coded additively) with BD using three distinct methods that control for genetic background heterogeneity (<a href="/pmc/articles/PMC3035981/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>): (1) single locus, contingency table analysis with genomic control adjustment, (2) logistic regression using a LAMP-derived estimate of ancestry as a covariate, and (3) logistic regression using the top 4 MDS dimensions as covariates. Association analyses using these three methods were conducted within the EA and AA samples separately, as well as within the combined sample (see <a href="/pmc/articles/PMC3035981/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). For each analysis, the distribution of expected p-values under the null hypothesis (i.e., no association), and the genomic inflation value (λ) was calculated in PLINK using the adjust command (<a href="#SD2" rid="SD2" class=" supplementary-material">Figure S1</a>). Genomic inflation values are reported as both uncorrected and corrected for a study size of 1,000 cases and 1,000 controls (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428544">19</a>, <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428520">20</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Most strongly associated SNPs using different methods to control for admixture and replication p-values.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093962286656" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="center">Analysis</th><th rowspan="2" colspan="1" valign="middle" align="left">SNP/Allele</th><th rowspan="2" colspan="1" valign="middle" align="left">Location</th><th rowspan="2" colspan="1" valign="middle" align="center">MAF</th><th colspan="2" rowspan="1" align="center">Multidimensional Scaling</th><th colspan="2" rowspan="1" align="center">LAMP-Adjusted</th><th colspan="2" rowspan="1" align="center">Genomic Control</th></tr><tr><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center">p-value</th><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center">p-value</th><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center">p-value</th></tr></thead><tbody><tr><td rowspan="2" colspan="1" valign="middle" align="center"><strong>AA only</strong></td><td rowspan="1" colspan="1" align="left">rs2111504/T<sup>a</sup></td><td rowspan="1" colspan="1" align="left"><em>DPY19L3</em> (19q13.11)</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">1.74</td><td rowspan="1" colspan="1" align="center">1.5 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" align="center">1.73</td><td rowspan="1" colspan="1" align="center">1.7 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" align="center">1.66</td><td rowspan="1" colspan="1" align="center">5.0 × 10<sup>-6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2769605/T<sup>b</sup></td><td rowspan="1" colspan="1" align="left"><em>NTRK2</em> (9q21.33)</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">0.60</td><td rowspan="1" colspan="1" align="center">4.5 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" align="center">0.61</td><td rowspan="1" colspan="1" align="center">4.5 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" align="center">0.63</td><td rowspan="1" colspan="1" align="center">1.5 × 10<sup>-4</sup></td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="center"><strong>EA only</strong></td><td rowspan="1" colspan="1" align="left">rs1825828/C<sup>a</sup>,<sup>c</sup></td><td rowspan="1" colspan="1" align="left">3q11.2</td><td rowspan="1" colspan="1" align="center">0.26</td><td rowspan="1" colspan="1" align="center">0.70</td><td rowspan="1" colspan="1" align="center">7.0 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" align="center">0.70</td><td rowspan="1" colspan="1" align="center">7.6 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" align="center">0.68</td><td rowspan="1" colspan="1" align="center">2.9 × 10<sup>-7</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs5907577/T<sup>a</sup></td><td rowspan="1" colspan="1" align="left">Xq27.1</td><td rowspan="1" colspan="1" align="center">0.31</td><td rowspan="1" colspan="1" align="center">1.48</td><td rowspan="1" colspan="1" align="center">1.6 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" align="center">1.47</td><td rowspan="1" colspan="1" align="center">2.1 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" align="center">1.444</td><td rowspan="1" colspan="1" align="center">4.6 × 10<sup>-6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs10193871/G<sup>b</sup></td><td rowspan="1" colspan="1" align="left"><em>NAP5</em> (2q21.2)</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">0.65</td><td rowspan="1" colspan="1" align="center">9.8 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" align="center">0.65</td><td rowspan="1" colspan="1" align="center">7.1 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" align="center">0.65</td><td rowspan="1" colspan="1" align="center">8.8 × 10<sup>-6</sup></td></tr><tr><td rowspan="1" colspan="1" align="center"><strong>EA/AA combined (main effect)</strong></td><td rowspan="1" colspan="1" align="left">rs4825220/C<sup>a</sup></td><td rowspan="1" colspan="1" align="left">Xq27.1</td><td rowspan="1" colspan="1" align="center">0.37</td><td rowspan="1" colspan="1" align="center">0.71</td><td rowspan="1" colspan="1" align="center">2.6 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" align="center">0.70</td><td rowspan="1" colspan="1" align="center">1.9 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" align="center">0.61</td><td rowspan="1" colspan="1" align="center">0.8496<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1" align="center"><strong>EA Pritzker Non-Overlap</strong></td><td rowspan="1" colspan="1" align="left">rs6046396/G<sup>a</sup></td><td rowspan="1" colspan="1" align="left"><em>RIN2</em> (20p11.23)</td><td rowspan="1" colspan="1" align="center">0.29</td><td rowspan="1" colspan="1" align="center">1.54</td><td rowspan="1" colspan="1" align="center">1.43 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" align="center">1.56</td><td rowspan="1" colspan="1" align="center">6.2 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" align="center">1.57</td><td rowspan="1" colspan="1" align="center">4.93 × 10<sup>-7</sup></td></tr></tbody></table></div><div id="largeobj_idm140093962286656" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3035981/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Overall top hit</div><div id="TFN2"><sup>b</sup>Top hit in region with at least 5 SNPs at P &lt; 1× 10<sup>-4</sup></div><div id="TFN3"><sup>c</sup>Possibly of poor genotype quality (see text and <a href="#SD2" rid="SD2" class=" supplementary-material">Figure S2</a>)</div><div id="TFN4"><sup>d</sup>This p-value likely reflects the imbalance in the ratio of cases and controls in the EA relative to the AA sample.</div></div></div></div><div id="S5" class="sec"><p></p><h4 id="S5title" class="inline">Haplotype tests </h4><p id="P8" class="p p-first-last">Haplotype analyses were performed in the EA population using the sliding window approach in PLINK (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428513">18</a>), and results for 10-SNP windows are highlighted. We used OMNIBUS p-values for the primary analysis and individual haplotype p-values for the analysis of haplotype heterogeneity at <em>ANK3</em>. In these analyses, population structure was not taken into account. For the haplotype heterogeneity analysis at <em>ANK3</em>, the proportion of haplotypes that reached a p-value of p &lt; 0.05 was performed using the 10-SNP window results. For each window, the number of haplotypes with an individual p-value less than 0.05 was counted and a proportion calculated for the window. Smoothing was performed over the results for the rest of the genome, and regions were highlighted that had a smoothed proportion greater than or equal to the maximum found at <em>ANK3</em> (0.33). Regions were distinct and were delineated by the first and last marker with a smoothed value greater than or equal to 0.33.</p></div><div id="S6" class="sec"><p></p><h4 id="S6title" class="inline">Imputation </h4><p id="P9" class="p p-first-last">Imputation was performed for the cleaned EA dataset using MACH v.1.0.16 (<a href="http://www.sph.umich.edu/csg/abecasis/mach/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/mach/index.html</a>) with HapMap rel21 phased haplotypes as a reference (see <a href="#SD9" rid="SD9" class=" supplementary-material">Supplemental Material</a>).</p></div><div id="S7" class="sec"><p></p><h4 id="S7title" class="inline">Genetic background-dependent effects </h4><p id="P10" class="p p-first-last">We assessed genetic background-specific effects, which could be due to interactions with background-specific variation or alternatively, due to different functional variation at the locus of interest. We combined the EA and AA samples and performed logistic regression analyses with adjustment using a LAMP main effect covariate and a LAMP x SNP interaction term (<a href="/pmc/articles/PMC3035981/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We compared the OR across different categories of admixture. We split the AA individuals into high and low admixture categories by the approximate median %CEU of 15%, and calculated the OR and 95% confidence interval within these and the EA group (see <a href="/pmc/articles/PMC3035981/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3035981/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093962293232" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms178631f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms178631f1.jpg" src="/pmc/articles/PMC3035981/bin/nihms178631f1.jpg"></div></a><div id="largeobj_idm140093962293232" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3035981/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3035981/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3035981/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Top genetic background-dependent SNPs</strong><p id="__p2">The region around the top background-dependent SNP. The primary SNP is colored in black. Other SNPs are colored according to linkage disequilibrium levels with the primary SNP (r2) calculated from Phase 3 HapMap data using the CEU ] population. Recombination rate (HapMap) is shown on the second y-axis in blue. RefSeq genes are shown with all possible exons; arrows indicate transcript direction. In the upper left hand corner of each graph, the genotype intensity plots are shown, with each color indicating the final genotype call (blue and red for homozygotes and purple for the heterozygote). B. Odds ratios for the top three background-dependent SNPs within groups showing different degrees of admixture. Low and High admixture groups are all of African Ancestry, but are split into two groups by the approximate median percent CEU. Lines indicate 95% confidence intervals.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p9">SNPs showing strongest genetic background-dependent effects (EA/AA combined sample with interaction term).</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093920071552" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP/Allele</th><th rowspan="1" colspan="1" align="left">Location</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center">SNP x Ancestry Interaction p-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs11208285/A</td><td rowspan="1" colspan="1" align="left"><em>ROR1</em> (1p31.3)</td><td rowspan="1" colspan="1" align="center">0.38</td><td rowspan="1" colspan="1" align="center">2.05</td><td rowspan="1" colspan="1" align="center">1.4 × 10-6</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4657247/T</td><td rowspan="1" colspan="1" align="left"><em>RGS5</em> (1q23.3)</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center">2.11</td><td rowspan="1" colspan="1" align="center">4.1 × 10-6</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7078071/T</td><td rowspan="1" colspan="1" align="left"><em>BTBD16</em> (10q26.13)</td><td rowspan="1" colspan="1" align="center">0.10</td><td rowspan="1" colspan="1" align="center">2.85</td><td rowspan="1" colspan="1" align="center">4.5 × 10-6</td></tr></tbody></table></div><div id="largeobj_idm140093920071552" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3035981/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S8" class="sec"><p></p><h4 id="S8title" class="inline">Power and statistical significance </h4><p id="P11" class="p p-first-last">In the current study, we have employed a significance threshold of 5 × 10<sup>-8</sup>, as this threshold has been commonly used in previous GWA studies. As there were no SNPs that exceeded this threshold, we report the most significant SNP for each sample, as well as regions where there are 5 SNPs within 100 kb of each other all with p &lt; 1 × 10<sup>-4</sup>. In addition, we list all SNPs reported with a p &lt; 10<sup>-4</sup> (<a href="#SD9" rid="SD9" class=" supplementary-material">Tables S3</a>-<a href="#SD6" rid="SD6" class=" supplementary-material">S5</a>). The strongest test of association lies in follow-up replication studies, however, and with the data we have available to us, we have attempted to examine the consistency of our results with previous GWA studies of BD and with our own newly genotyped replication samples (<a href="/pmc/articles/PMC3035981/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p10">Replication genotyping results.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093920048128" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="left">SNP</th><th rowspan="2" colspan="1" valign="middle" align="center">Chromosome</th><th rowspan="2" colspan="1" valign="middle" align="center">Position</th><th rowspan="2" colspan="1" valign="middle" align="center">Gene</th><th rowspan="2" colspan="1" valign="middle" align="center">MAF</th><th colspan="2" rowspan="1" align="center">Family p-value (unaffectedness criteria)</th><th rowspan="2" colspan="1" valign="middle" align="center">Replication case-control p-value</th><th rowspan="2" colspan="1" valign="middle" align="center">Meta-analysis p-value</th></tr><tr><th rowspan="1" colspan="1" align="center">Permissive</th><th rowspan="1" colspan="1" align="center">Strict</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs4671825</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">67762350</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.47</td><td rowspan="1" colspan="1" align="center">0.186</td><td rowspan="1" colspan="1" align="center">0.008</td><td rowspan="1" colspan="1" align="center">0.639</td><td rowspan="1" colspan="1" align="center">0.099</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4671826</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">67762453</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.47</td><td rowspan="1" colspan="1" align="center">0.137</td><td rowspan="1" colspan="1" align="center">0.008</td><td rowspan="1" colspan="1" align="center">0.640</td><td rowspan="1" colspan="1" align="center">0.084</td></tr><tr><td rowspan="1" colspan="1" align="left">rs13358880</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">56845056</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">0.204</td><td rowspan="1" colspan="1" align="center">0.041</td><td rowspan="1" colspan="1" align="center">0.196</td><td rowspan="1" colspan="1" align="center">7.1E-05</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4389663</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">104451583</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">0.166</td><td rowspan="1" colspan="1" align="center">0.047</td><td rowspan="1" colspan="1" align="center">0.918</td><td rowspan="1" colspan="1" align="center">0.130</td></tr><tr><td rowspan="1" colspan="1" align="left">rs10981870</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">115462097</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="center">0.137</td><td rowspan="1" colspan="1" align="center">0.047</td><td rowspan="1" colspan="1" align="center">0.449</td><td rowspan="1" colspan="1" align="center">0.022</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4924281</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">36742939</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.27</td><td rowspan="1" colspan="1" align="center">0.011</td><td rowspan="1" colspan="1" align="center">0.034</td><td rowspan="1" colspan="1" align="center">0.363</td><td rowspan="1" colspan="1" align="center">0.001</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1495186</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">36778748</td><td rowspan="1" colspan="1" align="center"><em>C15ORF53</em></td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">0.015</td><td rowspan="1" colspan="1" align="center">0.043</td><td rowspan="1" colspan="1" align="center">0.045</td><td rowspan="1" colspan="1" align="center">0.009</td></tr><tr><td rowspan="1" colspan="1" align="left">rs903530</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">36786340</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">0.010</td><td rowspan="1" colspan="1" align="center">0.054</td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">0.012</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2643217</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">36791055</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">0.008</td><td rowspan="1" colspan="1" align="center">0.057</td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">0.004</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2012353</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">37591133</td><td rowspan="1" colspan="1" align="center"><em>DPY19L3</em></td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">0.116</td><td rowspan="1" colspan="1" align="center">0.030</td><td rowspan="1" colspan="1" align="center">0.349</td><td rowspan="1" colspan="1" align="center">0.106</td></tr></tbody></table></div><div id="largeobj_idm140093920048128" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3035981/table/T3/?report=objectonly">Open in a separate window</a></div></div></div><div id="S9" class="sec"><p></p><h4 id="S9title" class="inline">Pritzker study overlap and analysis </h4><p id="P12" class="p p-first-last">A concurrent study of BD is being undertaken by the Pritzker Neuropsychiatric Disorders Research Consortium (PNDRC; Scott, Muglia, Upmanyu, Guan, Flickinger, Kong, Tozzi, Li, Burmeister, Absher et al., submitted). There is some overlap between the cases and controls used in our study and those used in the Pritzker study (407 EA cases and 357 EA controls). As an extension of our analyses we excluded these samples and repeated the association tests (see <a href="/pmc/articles/PMC3035981/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) so that the results of our study and the study by the PNDRC could be compared on the basis of independent samples. It is also the case that a portion of the samples on which we report here are not entirely independent of previously published BD GWA studies (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428522">15</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352561131">16</a>). We were not, however, able to perform a similar analysis with the non-overlapping samples from these other studies due to restricted access to the genotype data and time constraints.</p></div><div id="S10" class="sec sec-last"><p></p><h4 id="S10title" class="inline">Comparison of our results with previous GWA studies </h4><p id="P13" class="p p-first-last">Initially, to assess the extent to which our results corresponded with previous GWA studies of BD, we obtained p-values for genotyped markers in the WTCCC (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428518">13</a>) and STEP-BD (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352561130">16</a>) studies; it was the case, however, that most of the top SNPs that we describe were not genotyped directly within these studies. Therefore, we proceeded by investigating the three regions of interest identified in a recent collaborative GWA study (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321054632">17</a>) within the EA sample in our study (<a href="#SD4" rid="SD4" class=" supplementary-material">Figure S3</a>). We evaluated the top SNPs from this study that were directly genotyped in our study, as well as other SNPs in the three regions. We assessed both the direction of associations between the two studies, as well as the strength. For comparison, we include the results from these studies in the plots of regions of interest (<a href="#SD3" rid="SD3" class=" supplementary-material">Figure S2</a>). Although limited, we further analyzed our results relative to one of the single previously published GWA studies (i.e., Baum et al., 2008) (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428540">15</a>), which included about half of the probands and controls used in the present study, in addition to a replication sample that is independent of our sample (i.e., was collected in Germany). We present results in <a href="#SD3" rid="SD3" class=" supplementary-material">Figure S2</a>, but note that further analysis of the replication within each of the BD GWA studies will require specific delination of the overlapping individuals within all the studies and imputation of all genotypes.</p></div></div><div id="S11" class="sec sec-last"><h3 id="S11title">Replication Genotyping</h3><p id="P14" class="p p-first-last">A subset of 85 SNPs from the GWA study was selected for replication genotyping based on several criteria, with a primary focus on the allelic association p-value in the larger EA sample. See <a href="#SD9" rid="SD9" class=" supplementary-material">Supplemental Material</a> for additional information pertaining to replication genotyping.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">RESULTS</h2><div id="S13" class="sec sec-first"><h3 id="S13title">Descriptive Statistics</h3><p id="P15" class="p p-first-last">Demographic variables for EA and AA cases were generally similar (<a href="#SD9" rid="SD9" class=" supplementary-material">Table S1</a>). For the AA population, controls were less likely to be female. In the EA population, gender distributions between cases and controls were similar. Controls were much older than cases in both populations (mean age of cases: 18.0 (AA) and 19.3 (EA) vs. mean age of controls: 45.8 (AA) and 52.2 (EA), which should protect against cryptic disease in the control groups.</p></div><div id="S14" class="sec"><h3 id="S14title">Basic Association Analyses</h3><p id="P16" class="p p-first">We report the top associated SNP in each ancestry group, as well as the top associated SNP in the combined sample (<a href="/pmc/articles/PMC3035981/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; <a href="#SD3" rid="SD3" class=" supplementary-material">Figure S2</a>). As discussed in the Subjects and Methods section, we utilized different methods to control for genetic background heterogeneity and admixture, but generally obtained similar results for all the methods. As shown in <a href="/pmc/articles/PMC3035981/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, the most significant associations were different between the EA and AA subjects. For the EA sample, the most significantly associated SNP was rs1825828 at 3q11.2, p = 7.0 × 10<sup>-7</sup>. However, upon inspection of the genotype intensity plot (see <a href="#SD3" rid="SD3" class=" supplementary-material">Figure S2</a>), rs1825828 appeared to be poorly genotyped. The second-best association was rs5907577 in an intergenic region at Xq27.1, p =1.6 × 10<sup>-6</sup>. For the AA sample, the most significantly associated SNP was rs2111504 in <em>DPY19L3</em> at 19q13.11, p = 1.5 × 10<sup>-6</sup>. In addition, we looked for regions where there were multiple SNPs with low p-values (p &lt; 1 × 10<sup>-4</sup>) with close physical proximity to one another (i.e., within 100kb). We report two regions that contain 5 SNPs that meet this criterion. Among EA, we report <em>NAP5</em> (top SNP is rs10193871, p = 9.8 × 10<sup>-6</sup>), and among AA, we report an intergenic region about 300kb upstream of <em>NTRK2</em> (top SNP is rs2769605, p = 4.5 × 10<sup>-5</sup>). When the EA and AA groups were combined, the most significant SNP was rs4825220 in Xq27.1, p = 2.6 × 10<sup>-7</sup>, which does not lie near a known gene. We also show Q-Q plots for each analysis (<a href="#SD2" rid="SD2" class=" supplementary-material">Figure S1</a>). Within AA and EA samples, overall p-value inflation was low. However, we observed high levels of inflation when we combined the EA and AA datasets. This is due to the difference in case-control ratios between the studies, which artificially induces population stratification. Correcting for admixture using either LAMP or MDS covariates effectively removed this stratification. However, using the genomic control method to adjust the p-values overcorrected the association (<a href="/pmc/articles/PMC3035981/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><p id="P17">We investigated the consistency of our top hits across each of the different groups in our study. <a href="#SD9" rid="SD9" class=" supplementary-material">Table S2</a> shows to what extent the top hits in each individual sample (e.g., in the EA group) showed evidence of association in the other samples (e.g., in the AA group). Top hits within AA were not significant in EA and vice versa, suggesting that variation that contributes to BD may differ between the two ancestral groups.</p><p id="P18" class="p p-last">Imputation was performed using MACH (<a href="http://www.sph.umich.edu/csg/abecasis/mach/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/mach/index.html</a>) on the EA dataset. Imputed SNPs generally supported, and were of similar strength as the observed associations (see <a href="#SD3" rid="SD3" class=" supplementary-material">Figures S2</a> and <a href="#SD4" rid="SD4" class=" supplementary-material">S3</a>). Near EPHA6, however, there were no imputed SNPs that showed association with BP, supporting the argument that the observed association is due to a genotyping error.</p></div><div id="S15" class="sec"><h3 id="S15title">Haplotype Analyses</h3><p id="P19" class="p p-first-last">We performed haplotype-based association tests in the EA population, using the sliding-window approach on the genotyped SNPs in PLINK. In addition to showing 10-SNP haplotype p-values for each region where an individual SNP is highlighted, we also describe the most significant haplotype. This haplotype is located on the X chromosome and is 7 Mb away from the SNP with the highest single locus association strength (10 SNP OMNIBUS p = 1.9 × 10<sup>-11</sup>). This region is about 1Mb downstream of <em>SLITRK2</em> (<a href="#SD3" rid="SD3" class=" supplementary-material">Figure S2</a>).</p></div><div id="S16" class="sec"><h3 id="S16title">Comparison of Our Results with Previous GWA Studies</h3><p id="P20" class="p p-first">A recent collaborative GWA study (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321054631">17</a>) that consisted of 4,387 cases and 6,209 controls pooled from the Wellcome Trust Case Control Consortium (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428538">13</a>) Bipolar Analysis, Sklar et al. (STEP-BD) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352561132">16</a>), and 2,365 new samples highlighted three regions of interest: <em>ANK3</em> (ankyrin G), <em>CACNA1C</em> (alpha 1C subunit of the L-type voltage-gated calcium channel), and a region 3.3kb away from <em>C15ORF53</em> on chromosome 15q14 (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321054633">17</a>). This analysis was restricted to individuals of EA; therefore, we investigated these regions within only the EA sample in our study (<a href="#SD4" rid="SD4" class=" supplementary-material">Figure S3</a>). When we focus on the top SNPs from this study that were directly genotyped in our study, only one of the SNPs reached p &lt; 0.05 (<em>ANK3</em>, rs1938526/G, p = 0.036, OR = 1.31), although the top SNP in the 15q14 region approached p &lt; 0.05 (rs2172835, p = 0.057, OR = 0.88). In both cases, the association was in the same direction and of a similar strength as previously reported. In both of these regions, we saw additional SNPs that showed low (0.01-0.0001) p-values.</p><p id="P21">We thought that this pattern could indicate the presence of allelic heterogeneity, which has also been suggested by a separate study focusing on 2 markers in the region (Schulze, Detera-Wadleigh, Akula, Gupta, Kassem, Steele, Pearl, Strohmaier, Breuer, Schwarz, et al., in press), and which might reflect multiple underlying rare variants. In the presence of allelic heterogeneity, we would expect that multiple haplotypes would show association with BD. Using the 10 SNP window result from the haplotype analysis in the EA population, which provide p-values for each haplotype in addition to the OMNIBUS p-values, we investigated the proportion of haplotypes within each window that had p-values less than 0.05 (<a href="/pmc/articles/PMC3035981/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). In the <em>ANK3</em> region, there were many 10-SNP windows that had a high proportion of low p-value haplotypes (up to 75%, genome-wide average = 4.7%), with 2-5 haplotypes often being implicated. In order to see if this was unexpected in the genome, we smoothed the proportion of significant haplotypes across the genome and looked for other regions that matched or exceeded the maximum smoothed value found in <em>ANK3</em>. We found 10 additional regions covering approximately 1.5 Mb or about 0.05% of the genome that exceeded this level of haplotype heterogeneity, indicating that the proportion of haplotypes that are associated with BD at this locus is relatively high, compared to the rest of the genome (<a href="#SD9" rid="SD9" class=" supplementary-material">Table S3</a>). This provides additional support that the region shows allelic heterogeneity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3035981/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093962289632" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms178631f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms178631f2.jpg" src="/pmc/articles/PMC3035981/bin/nihms178631f2.jpg"></div></a><div id="largeobj_idm140093962289632" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3035981/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3035981/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3035981/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Muliti-haplotype association in the ANK3 region suggests allelic heterogeneity</strong><p id="__p3">Each point summarizes the proportion of haplotypes within a given 10 SNP window. The proportion of the haplotypes that show a p &lt; 0.05 for a given window is plotted on the y-axis. The x-axis is the physical position of the midpoint of the haplotype. The points are colored according to the number of haplotypes that show p &lt; 0.05. The proportion was median smoothed over 101 windows (blue line). The red dotted line indicates the genome-wide mean proportion of haplotypes with a p &lt; 0.05. RefSeq genes are shown below the x-axis.</p></div></div></div><p id="P22" class="p p-last">We further analyzed our results relative to one of the single previously published GWA studies (i.e., Baum et al., 2008) (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428533">15</a>), which included about half of the probands and controls used in the present study, in addition to a replication sample that is independent of our sample (i.e., was collected in Germany). Several of the 88 SNPs with replicated association signals in the Baum et al. study (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428515">15</a>) also show nominal evidence of association in the present study (<a href="#SD9" rid="SD9" class=" supplementary-material">Table S4</a>), including a SNP in <em>ANK3</em> (i.e., rs9804190). The extent to which this can be considered evidence for a consistent finding, however, is limited given the overlap between the samples used in the two studies.</p></div><div id="S17" class="sec"><h3 id="S17title">Genetic Background x SNP Interaction Analysis</h3><p id="P23" class="p p-first-last"><a href="/pmc/articles/PMC3035981/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> depicts evidence of SNP associations with genetic background-dependent effects. As shown, no SNP showed strong genetic background-dependent effects (i.e., no p-values less than 5 × 10<sup>-8</sup>). We do, however, report the top 3 SNPs, which occur in <em>ROR1</em>, <em>RGS5</em>, and <em>BTBD16</em> (all p-values &lt; 4.5 × 10<sup>-6</sup>). We also show the OR across different categories of admixture to depict these results (<a href="/pmc/articles/PMC3035981/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>).</p></div><div id="S18" class="sec"><h3 id="S18title">Comparisons of Different Methods for Determining Ancestry</h3><p id="P24" class="p p-first-last">Based on LAMP ancestry estimates using all autosomal SNPs, the EA set showed less than 1% Yoruban admixture (mean= 0.998, range: 0.944-1, STD = 0.005), whereas the AA set showed almost 19% European admixture (mean = 0.188, range: 0.0-1, STD = 0.124). Furthermore, LAMP ancestry estimates including 3053 EA and AA subjects were significantly correlated with estimates generated with the more traditional STRUCTURE method (r = 0.999, p &lt; 0.001). Including parental allele frequencies of HGDP subjects instead of HapMap subjects did not influence ancestry estimates (r = 1.0, p &lt; 0.001).</p></div><div id="S19" class="sec"><h3 id="S19title">Analysis of Subjects that did not Overlap with the PNDRC</h3><p id="P25" class="p p-first-last">Results of analyses that included the subgroup of EA subjects that did not overlap with the Pritzker Study revealed a different top hit relative to analyses that included the entire sample. Specifically, the most significantly associated SNP in this subsample of EA subjects was rs6046396, which is upstream of <em>RIN2</em> at 20p11. 23, p = 1.43 × 10<sup>-6</sup> (see <a href="/pmc/articles/PMC3035981/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p></div><div id="S20" class="sec sec-last"><h3 id="S20title">Replication Genotyping Analyses</h3><p id="P26" class="p p-first-last">Results from the replication analysis are shown in <a href="/pmc/articles/PMC3035981/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, with SNPs shown from genomic regions that demonstrated association (p &lt; 0.03) in the family-based replication sample. The analysis of the case-control replication cohort did not detect significance below the multiple-testing threshold (p &lt; 0.001) among the 85 SNPs genotyped, although three of the SNPs in the <em>C15ORF53</em> region demonstrated some evidence of association in the case-control cohort (p = 0.03-0.04) as well as in the family sample (p = 0.008-0.015). The preponderance of the association evidence from the family based analysis reported in <a href="/pmc/articles/PMC3035981/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> is derived from transmission of SNP alleles to affected individuals. These transmissions are effectively independent of the population association (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428516">21</a>). The meta-analysis (<a href="/pmc/articles/PMC3035981/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) gave p-values of 0.004-0.01 for SNPs in this region, and reinforced the association evidence for SNP rs13358880 on chromosome 5.</p></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">DISCUSSION</h2><p id="P27" class="p p-first">We conducted two GWA studies, one in a sample of individuals of EA and the second in a sample of individuals of AA. In order to account for possible genetic background differences, we: 1) considered the analysis of each sample separately; 2) estimated ancestry and genetic background diversity from the genetic data and controlled for it in the association studies; and 3) looked for evidence of genetic background x SNP interactions. In order to qualify our results, we also compared them to previous GWA study results investigating BD, performed replication genotyping of our most strongly associated SNPs on an independent cohort, and conducted analysis of only the non-overlapping subjects in our study (with another study’s subjects) in order to tease out independent evidence for associations.</p><p id="P28">Although no single SNP showed significant association after correction for genome-wide testing in either of our populations, some noteworthy associations were observed with BD candidate genes, as well as with genes known to be expressed in human brain. Of particular interest are <em>SLITRK2</em> and <em>NTRK2. SLITRK2</em> is a member of a family of six genes which are widely expressed in neural tissue (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428512">22</a>), producing proteins which are membrane bound. <em>SLITRK2</em> regulates neurite outgrowth <em>in vitro.</em> Thus, <em>SLITRK2</em> is a logical bipolar risk gene. Another member of this gene family, <em>SLITRK1</em>, has been reported to be associated with Tourette’s syndrome (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428517">23</a>). <em>NTRK2</em> (also known as TrkB) is a tyrosine kinase receptor which binds brain derived neurotrophic factor (<em>BDNF</em>) and possibly other neurotrophins, i.e., (for review see 24). <em>NTRK2</em> is a high priority bipolar candidate gene for several reasons. There is abundant evidence from animal models of depression that hippocampal neurogenesis is decreased during the behavioral syndrome, that antidepressants increase neurogenesis, and that <em>BDNF</em> has antidepressant-like properties in these animal models (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428525">25</a>). <em>BDNF</em> expression is increased in animals by treatment with antidepressants or lithium, and <em>BDNF</em> SNPs have repeatedly been implicated in genetic risk for BD, although the effect size is quite limited, e.g. odds ratio of 1.1 (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428509">26</a>-<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428526">29</a>).</p><p id="P29">Our genetic background analysis revealed significant levels of admixture among our AA study subjects. Given this, we explored the comparability of different methods to account for admixture in our analyses and found that three different methods – genomic control adjustment, logistic regression using a LAMP-generated covariate, and logistic regression using MDS-generated covariates – all produced very similar results within the AA population. Regression with either LAMP or MDS based covariates was effective for correcting artificially induced population structure when EA and AA samples were combined, but genomic control overcorrected for this, effectively removing any real association that was not associated with admixture levels.</p><p id="P30">Different putative associations were observed among individuals of EA and individuals of AA when analyzed separately, and analyses assessing the extent to which some SNPs show genetic background-dependent effects highlighted different areas of potential association. We believe that this is the first report emerging from a GWA to explicitly address the dependency of SNP effects on ancestry, admixture, and/or genetic background.</p><p id="P31">We sought to replicate our most strongly associated SNPs in two independent sets of subjects, one family-based, and one case-control. Positive results were seen with rs1495186 in <em>C15ORF53</em> and some additional SNPs in that region. When considered cumulatively <em>via</em> meta-analysis with the primary EA sample, these results demonstrate consistent support for association of SNPs in the <em>C15ORF53</em> region with BD, and provide additional supportive evidence of association in other regions.</p><p id="P32">We also considered the consistency of our results with previous BD GWA studies. There have been three previous GWA studies of BD (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428532">13</a>, <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428527">15</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352561133">16</a>), in addition to a more recent collaborative GWA meta-analysis study (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321054629">17</a>), a subset of which were represented in the previous independent GWA studies. The Ferreira et al. (2008) collaborative GWA meta-analysis study (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321054634">17</a>) identified a region of strong association in <em>ANK3</em>, apparently distinct from that detected in Baum et al. (2008). Ferreira et al. also found new evidence for association at 15q14, which is near <em>C15ORF53</em>, as well as further support for the previously reported <em>CACNA1C</em> gene; these authors concluded that ion channelopathies may be involved in the pathogenesis of BD.</p><p id="P33">In the current study, we found consistent evidence of both previously-reported <em>ANK3</em> findings, and borderline support for replication of a region characterized at 15q14, although we failed to find support for the finding at <em>CACNA1C</em>. In <em>ANK3</em> and at 15q14, multiple SNPs in weak to no linkage disequilibrium with the previously associated SNP showed stronger association. Investigation of haplotype-based associations in our study provides support for allelic heterogeneity in <em>ANK3</em> region. Allelic heterogeneity has the potential to play an important role in genetically influenced disorders, yet can be difficult to detect in population-based samples using common variants, making it a potential explanation of “missing heritability” (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428524">30</a>). Of note, and as we have previously indicated however, the Baum et al. (2008) and Sklar et al. (2008) samples both overlap with the current sample (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_210428529">15</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352561129">16</a>), thus stringent conclusions pertaining to replication are not warranted.</p><p id="P34" class="p p-last">Our GWA study of BD provides some support for previous findings that variation in <em>ANK3</em> and at 15q14 influence BD susceptibility. In addition, regions containing <em>NAP5</em>, <em>NTRK2</em>, <em>SLITRK2</em>, and <em>ROR1</em> are worthy of follow-up studies. As all of these associated SNPs and regions have small effect size, it is likely, however, that the majority of the genetic variations that influence BD remain yet to be discovered.</p></div><div id="S22" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD2"><h4>Figure S1</h4><!--caption a9--><strong>Supplemental Figure 1 (S1): QQ and multidimensional scaling (MDS) plots for each study population: </strong><p id="P35">Unadjusted –log(p-values) are shown in black, with MDS (top 4 components) adjustment in red and LAMP adjustment in blue. In the top left hand corner of each plot is the top 2 MDS components for each analysis plotted against each other. Genomic control λ values are shown given the current study size and corrected for a study size of 1000 cases and 1000 controls. In the case where EA and AA individuals are analyzed together, unadjusted λ levels are elevated because of different ratios of cases and controls in the two populations. All individuals shown in the MDS plots in the upper left corners were included in the analyses.</p><div class="sup-box half_rhythm" id="N0xf714510N0x105b39b0"><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Figure_S1.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(5.1M, tif)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Figure S2</h4><!--caption a9--><strong>Supplemental Figure 2 (S2): Regions near top hits and areas of interest: </strong><p id="P36">Regions +/- 250 kb around each SNP listed in Table 1 are shown. P-values are from the analysis where the SNP was identified. Genotyped SNPs are shown as circles, while imputed SNPs are shown as smaller diamonds. The primary SNP of interest is large and colored in black. Other SNPs are colored according to linkage disequilibrium levels with the primary SNP (r2), as calculated from Phase 3 HapMap data using CEU (for EA and EA + AA combined) or YRI (for AA) populations. Recombination rate (HapMap) is shown on the second y-axis in blue. RefSeq genes are shown with all possible exons; arrows indicate transcript direction. In the upper left hand corner of each graph, the genotype intensity plots are shown, with each color indicating the final genotype call (blue and red for homozygotes and purple for the heterozygote).</p><div class="sup-box half_rhythm" id="N0xf714510N0x105b3ad0"><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Figure_S2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.1M, pdf)</sup></div></div><div class="sec suppmat" id="SD4"><h4>Figure S3</h4><!--caption a9--><strong>Supplemental Figure 3 (S3): Top regions characterized in Ferreria et al., +/- 250 kb: </strong><p id="P37">Each circle represents a SNP, with the first y-axis indicating the p-value for Bipolar Disorder in EA individuals (MDS adjusted) for this study. Genotyped SNPs are indicated with a circle, while imputed SNPs are indicated with diamonds. The most strongly previously associated SNP is indicated by a large black circle. SNPs are colored shades of red depending on their linkage disequilibrium with the most strongly previously associated SNP (r2, calculated from HapMap CEU Phase 3 using Haploview). Recombination rate (HapMap) is shown on the second y-axis in blue. Green horizontal lines indicate haplotype association p-values from a 10 SNP sliding window. Below the plot are SNPs from genotyped SNPs from WTCCC Bipolar Disorder and STEP-BP studies, with p-value indicated by color. RefSeq genes are shown with all possible exons; arrows indicate transcript direction.</p><div class="sup-box half_rhythm" id="N0xf714510N0x105b3bf0"><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Figure_S3.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(773K, pdf)</sup></div></div><div class="sec suppmat" id="SD5"><h4>Figure S4</h4><!--caption a9--><strong>Supplemental Figure 4 (S4): Comparison of European ancestry estimates: </strong><p id="P38">Red: HapMap and HGDP with (STRUCTURE, 3503 SNPs on Chr. 1); Green: STRUCTURE and LAMP subset (Hapmap, 3503 SNPs on Chr. 1); Blue: STRUCTURE and LAMP full set (HapMap, 3503 SNPs on Chr. 1 and all markers for LAMP).</p><div class="sup-box half_rhythm" id="N0xf714510N0x105b3d10"><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Figure_S4.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(24M, tif)</sup></div></div><div class="sec suppmat" id="SD6"><h4>Table S5</h4><div class="sup-box half_rhythm" id="N0xf714510N0x105b3dd0"><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Table_S5.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(46K, xls)</sup></div></div><div class="sec suppmat" id="SD7"><h4>Table S6</h4><div class="sup-box half_rhythm" id="N0xf714510N0x105b3e90"><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Table_S6.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(52K, xls)</sup></div></div><div class="sec suppmat" id="SD8"><h4>Table S7</h4><div class="sup-box half_rhythm" id="N0xf714510N0x105b3f50"><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-Table_S7.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(45K, xls)</sup></div></div><div class="sec suppmat" id="SD9"><div class="sup-box half_rhythm" id="N0xf714510N0x105b4010"><a href="/pmc/articles/PMC3035981/bin/NIHMS178631-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(138K, pdf)</sup></div></div></div><div id="S24" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S24title">Acknowledgments</h2><div class="sec"><p id="__p11">We thank the participants in the study, as without them this work would not have been possible. For best estimate diagnostic work, we thank Vegas Coleman MD, Robert Schweitzer MD, N. Leela Rau MD, and Kelly Rhoadarmer MD. For data management we thank Mariano Erpe and for study coordination Carre Fisher RN. This work was supported by grants from the NIMH and NHGRI to JRK (<a id="__tag_801501828" class="tag_hotlink" href="/nuccore/MH078151" ref="/nuccore/MH078151"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH078151","term_id":"1534898816","term_text":"MH078151"}}</span>MH078151</a>, <a id="__tag_801501827" class="tag_hotlink" href="/nuccore/MH081804" ref="/nuccore/MH081804"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH081804","term_id":"1365466079","term_text":"MH081804"}}</span>MH081804</a>, <a id="__tag_801501826" class="tag_hotlink" href="/nuccore/MH059567" ref="/nuccore/MH059567"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH059567","term_id":"1489134359","term_text":"MH059567"}}</span>MH059567</a> supplement), and by the Genetic Association Information Network (GAIN). This work was additionally supported by the NIMH Intramural Research Program (FJM and TGS). WHB was supported by a grant from the Tzedakah Foundation, a grant from NIH (R01 MH59553) and a grant from Philip and Marcia Cohen. Falk Lohoff was supported by the Daland Fellowship Award from the American Philosophical Society and by NIH grant K08 MH080372. David Craig and Corrie Panganiban would like to acknowledge the Stardust foundation. Follow-up genotyping was performed in the laboratory of HE at Indiana University School of Medicine. This research was also supported, in part, by the Intramural Research Program of the NIH, National Library of Medicine. Additionally, this project was supported by NIH/NCRR Grant Number UL1 RR025774. Its contents are the authors(tm) sole responsibility and do not necessarily represent official NIH views. (ENS, CSB, and NJS).</p><p id="__p12">Data and biomaterials were collected in four projects that participated in the National Institute of Mental Health (NIMH) Bipolar Disorder Genetics Initiative. From 1991-98, the Principal Investigators and Co-Investigators were: Indiana University, Indianapolis, IN, U01 MH46282, John Nurnberger, M.D., Ph.D., Marvin Miller, M.D., and Elizabeth Bowman, M.D.; Washington University, St. Louis, MO, U01 MH46280, Theodore Reich, M.D., Allison Goate, Ph.D., and John Rice, Ph.D.; Johns Hopkins University, Baltimore, MD U01 MH46274, J. Raymond DePaulo, Jr., M.D., Sylvia Simpson, M.D., MPH, and Colin Stine, Ph.D.; NIMH Intramural Research Program, Clinical Neurogenetics Branch, Bethesda, MD, Elliot Gershon, M.D., Diane Kazuba, B.A., and Elizabeth Maxwell, M.S.W.</p><p id="__p13">Data and biomaterials were collected as part of ten projects that participated in the National Institute of Mental Health (NIMH) Bipolar Disorder Genetics Initiative. From 1999-07, the Principal Investigators and Co-Investigators were: Indiana University, Indianapolis, IN, R01 MH59545, John Nurnberger, M.D., Ph.D., Marvin J. Miller, M.D., Elizabeth S. Bowman, M.D., N. Leela Rau, M.D., P. Ryan Moe, M.D., Nalini Samavedy, M.D., Rif El-Mallakh, M.D. (at University of Louisville), Husseini Manji, M.D. (at Wayne State University), Debra A. Glitz, M.D. (at Wayne State University), Eric T. Meyer, M.S., Carrie Smiley, R.N., Tatiana Foroud, Ph.D., Leah Flury, M.S., Danielle M. Dick, Ph.D., Howard Edenberg, Ph.D.; Washington University, St. Louis, MO, R01 MH059534, John Rice, Ph.D, Theodore Reich, M.D., Allison Goate, Ph.D., Laura Bierut, M.D.; Johns Hopkins University, Baltimore, MD, R01 MH59533, Melvin McInnis M.D., J. Raymond DePaulo, Jr., M.D., Dean F. MacKinnon, M.D., Francis M. Mondimore, M.D., James B. Potash, M.D., Peter P. Zandi, Ph.D, Dimitrios Avramopoulos, and Jennifer Payne; University of Pennsylvania, PA, R01 MH59553, Wade Berrettini M.D., Ph.D.; University of California at Irvine, CA, R01 MH60068, William Byerley M.D., and Mark Vawter M.D.; University of Iowa, IA, R01 MH059548, William Coryell M.D., and Raymond Crowe M.D.; University of Chicago, IL, R01 MH59535, Elliot Gershon, M.D., Judith Badner Ph.D., Francis McMahon M.D., Chunyu Liu Ph.D., Alan Sanders M.D., Maria Caserta, Steven Dinwiddie M.D., Tu Nguyen, Donna Harakal; University of California at San Diego, CA, R01 MH59567, John Kelsoe, M.D., Rebecca McKinney, B.A.; Rush University, IL, R01 MH059556, William Scheftner M.D., Howard M. Kravitz, D.O., M.P.H., Diana Marta, B.S., Annette Vaughn-Brown, MSN, RN, and Laurie Bederow, MA; NIMH Intramural Research Program, Bethesda, MD, 1Z01MH002810-01, Francis J. McMahon, M.D., Layla Kassem, PsyD, Sevilla Detera-Wadleigh, Ph.D, Lisa Austin, Ph.D, Dennis L. Murphy, M.D.</p><p id="__p14">JRK is a founder and holds equity in Psynomics, Inc.</p></div></div><div id="idm140093998605744" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140093998605744title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"> </p><p id="__p15" class="p p-first-last">The terms of this arrangement have been reviewed and approved by UCSD in accordance with its conflict of interest policies.</p><p></p></div></div><div id="idm140093919938512" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140093919938512title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Goodwin FK, Jamison KR, Akiskal H, Fawcett J, Ghaemi N, Hammen C, et al.   <span class="ref-journal">Manic-Depressive Illness.</span> 2. Oxford University Press; 2005.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Manic-Depressive+Illness&amp;author=FK+Goodwin&amp;author=KR+Jamison&amp;author=H+Akiskal&amp;author=J+Fawcett&amp;author=N+Ghaemi&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Tsuang MG, Faraone SV.  <span class="ref-journal">The Genetics of Mood Disorders.</span> Johns Hopkins University Press; Baltimore: 1990.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Genetics+of+Mood+Disorders&amp;author=MG+Tsuang&amp;author=SV+Faraone&amp;publication_year=1990&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Gershon ES, Hamovit J, Guroff JJ, Dibble E, Leckman JF, Sceery W, et al.  A family study of schizoaffective, bipolar I, bipolar II, unipolar, and normal control probands. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1982 Oct;<span class="ref-vol">39</span>(10):1157–1167.</span> [<a href="/pubmed/7125846" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=A+family+study+of+schizoaffective,+bipolar+I,+bipolar+II,+unipolar,+and+normal+control+probands&amp;author=ES+Gershon&amp;author=J+Hamovit&amp;author=JJ+Guroff&amp;author=E+Dibble&amp;author=JF+Leckman&amp;volume=39&amp;issue=10&amp;publication_year=1982&amp;pages=1157-1167&amp;pmid=7125846&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Rice J, Reich T, Andreasen NC, Endicott J, Van Eerdewegh M, Fishman R, et al.  The familial transmission of bipolar illness. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1987 May;<span class="ref-vol">44</span>(5):441–447.</span> [<a href="/pubmed/3579495" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=The+familial+transmission+of+bipolar+illness&amp;author=J+Rice&amp;author=T+Reich&amp;author=NC+Andreasen&amp;author=J+Endicott&amp;author=M+Van+Eerdewegh&amp;volume=44&amp;issue=5&amp;publication_year=1987&amp;pages=441-447&amp;pmid=3579495&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Spence MA, Flodman PL, Sadovnick AD, Bailey-Wilson JE, Ameli H, Remick RA. Bipolar disorder: evidence for a major locus. <span><span class="ref-journal">Am J Med Genet. </span>1995 Oct 9;<span class="ref-vol">60</span>(5):370–376.</span> [<a href="/pubmed/8546148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Bipolar+disorder:+evidence+for+a+major+locus&amp;author=MA+Spence&amp;author=PL+Flodman&amp;author=AD+Sadovnick&amp;author=JE+Bailey-Wilson&amp;author=H+Ameli&amp;volume=60&amp;issue=5&amp;publication_year=1995&amp;pages=370-376&amp;pmid=8546148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Craddock N, Khodel V, Van Eerdewegh P, Reich T. Mathematical limits of multilocus models: the genetic transmission of bipolar disorder. <span><span class="ref-journal">Am J Hum Genet. </span>1995 Sep;<span class="ref-vol">57</span>(3):690–702.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1801283/">PMC free article</a>]</span> [<a href="/pubmed/7668299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Mathematical+limits+of+multilocus+models:+the+genetic+transmission+of+bipolar+disorder&amp;author=N+Craddock&amp;author=V+Khodel&amp;author=P+Van+Eerdewegh&amp;author=T+Reich&amp;volume=57&amp;issue=3&amp;publication_year=1995&amp;pages=690-702&amp;pmid=7668299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Risch N, Botstein D. A manic depressive history. <span><span class="ref-journal">Nat Genet. </span>1996 Apr;<span class="ref-vol">12</span>(4):351–353.</span> [<a href="/pubmed/8630482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+manic+depressive+history&amp;author=N+Risch&amp;author=D+Botstein&amp;volume=12&amp;issue=4&amp;publication_year=1996&amp;pages=351-353&amp;pmid=8630482&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Badner JA, Gershon ES. Meta-analysis of whole-genome linkage scans of bipolar disorder and schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2002;<span class="ref-vol">7</span>(4):405–411.</span> [<a href="/pubmed/11986984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Meta-analysis+of+whole-genome+linkage+scans+of+bipolar+disorder+and+schizophrenia&amp;author=JA+Badner&amp;author=ES+Gershon&amp;volume=7&amp;issue=4&amp;publication_year=2002&amp;pages=405-411&amp;pmid=11986984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Segurado R, Detera-Wadleigh SD, Levinson DF, Lewis CM, Gill M, Nurnberger JI, Jr, et al.  Genome scan meta-analysis of schizophrenia and bipolar disorder, part III: Bipolar disorder. <span><span class="ref-journal">Am J Hum Genet. </span>2003 Jul;<span class="ref-vol">73</span>(1):49–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180589/">PMC free article</a>]</span> [<a href="/pubmed/12802785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome+scan+meta-analysis+of+schizophrenia+and+bipolar+disorder,+part+III:+Bipolar+disorder&amp;author=R+Segurado&amp;author=SD+Detera-Wadleigh&amp;author=DF+Levinson&amp;author=CM+Lewis&amp;author=M+Gill&amp;volume=73&amp;issue=1&amp;publication_year=2003&amp;pages=49-62&amp;pmid=12802785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">McQueen MB, Devlin B, Faraone SV, Nimgaonkar VL, Sklar P, Smoller JW, et al.  Combined analysis from eleven linkage studies of bipolar disorder provides strong evidence of susceptibility loci on chromosomes 6q and 8q. <span><span class="ref-journal">Am J Hum Genet. </span>2005 Oct;<span class="ref-vol">77</span>(4):582–595.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1275607/">PMC free article</a>]</span> [<a href="/pubmed/16175504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Combined+analysis+from+eleven+linkage+studies+of+bipolar+disorder+provides+strong+evidence+of+susceptibility+loci+on+chromosomes+6q+and+8q&amp;author=MB+McQueen&amp;author=B+Devlin&amp;author=SV+Faraone&amp;author=VL+Nimgaonkar&amp;author=P+Sklar&amp;volume=77&amp;issue=4&amp;publication_year=2005&amp;pages=582-595&amp;pmid=16175504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Nurnberger JI, Berrettini W, Niculescu AB.  Genetics of psychiatric disorders. In: Fatemi SH, Clayton PJ, editors. <span class="ref-journal">The Medical Basis of Psychiatry.</span> Humana Press; 2008. pp. 487–518. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Medical+Basis+of+Psychiatry&amp;author=JI+Nurnberger&amp;author=W+Berrettini&amp;author=AB+Niculescu&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Manolio TA, Brooks LD, Collins FS. A HapMap harvest of insights into the genetics of common disease. <span><span class="ref-journal">J Clin Invest. </span>2008 May;<span class="ref-vol">118</span>(5):1590–1605.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2336881/">PMC free article</a>]</span> [<a href="/pubmed/18451988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=A+HapMap+harvest+of+insights+into+the+genetics+of+common+disease&amp;author=TA+Manolio&amp;author=LD+Brooks&amp;author=FS+Collins&amp;volume=118&amp;issue=5&amp;publication_year=2008&amp;pages=1590-1605&amp;pmid=18451988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">WTCCC. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007 Jun 7;<span class="ref-vol">447</span>(7145):661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;issue=7145&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Altshuler D, Daly MJ, Lander ES. Genetic mapping in human disease. <span><span class="ref-journal">Science. </span>2008 Nov 7;<span class="ref-vol">322</span>(5903):881–888.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2694957/">PMC free article</a>]</span> [<a href="/pubmed/18988837" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+mapping+in+human+disease&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;author=ES+Lander&amp;volume=322&amp;issue=5903&amp;publication_year=2008&amp;pages=881-888&amp;pmid=18988837&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Baum AE, Akula N, Cabanero M, Cardona I, Corona W, Klemens B, et al.  A genome-wide association study implicates diacylglycerol kinase eta (DGKH) and several other genes in the etiology of bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2008 Feb;<span class="ref-vol">13</span>(2):197–207.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2527618/">PMC free article</a>]</span> [<a href="/pubmed/17486107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=A+genome-wide+association+study+implicates+diacylglycerol+kinase+eta+(DGKH)+and+several+other+genes+in+the+etiology+of+bipolar+disorder&amp;author=AE+Baum&amp;author=N+Akula&amp;author=M+Cabanero&amp;author=I+Cardona&amp;author=W+Corona&amp;volume=13&amp;issue=2&amp;publication_year=2008&amp;pages=197-207&amp;pmid=17486107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Sklar P, Smoller JW, Fan J, Ferreira MA, Perlis RH, Chambert K, et al.  Whole-genome association study of bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2008 Jun;<span class="ref-vol">13</span>(6):558–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3777816/">PMC free article</a>]</span> [<a href="/pubmed/18317468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Whole-genome+association+study+of+bipolar+disorder&amp;author=P+Sklar&amp;author=JW+Smoller&amp;author=J+Fan&amp;author=MA+Ferreira&amp;author=RH+Perlis&amp;volume=13&amp;issue=6&amp;publication_year=2008&amp;pages=558-569&amp;pmid=18317468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Ferreira MA, O’Donovan MC, Meng YA, Jones IR, Ruderfer DM, Jones L, et al.  Collaborative genome-wide association analysis supports a role for ANK3 and CACNA1C in bipolar disorder. <span><span class="ref-journal">Nat Genet. </span>2008 Aug 17;</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2703780/">PMC free article</a>]</span> [<a href="/pubmed/18711365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Collaborative+genome-wide+association+analysis+supports+a+role+for+ANK3+and+CACNA1C+in+bipolar+disorder&amp;author=MA+Ferreira&amp;author=MC+O’Donovan&amp;author=YA+Meng&amp;author=IR+Jones&amp;author=DM+Ruderfer&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007 Sep;<span class="ref-vol">81</span>(3):559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999 Dec;<span class="ref-vol">55</span>(4):997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;issue=4&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Freedman ML, Reich D, Penney KL, McDonald GJ, Mignault AA, Patterson N, et al.  Assessing the impact of population stratification on genetic association studies. <span><span class="ref-journal">Nat Genet. </span>2004 Apr;<span class="ref-vol">36</span>(4):388–393.</span> [<a href="/pubmed/15052270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Assessing+the+impact+of+population+stratification+on+genetic+association+studies&amp;author=ML+Freedman&amp;author=D+Reich&amp;author=KL+Penney&amp;author=GJ+McDonald&amp;author=AA+Mignault&amp;volume=36&amp;issue=4&amp;publication_year=2004&amp;pages=388-393&amp;pmid=15052270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Abecasis GR, Cookson WO, Cardon LR. Pedigree tests of transmission disequilibrium. <span><span class="ref-journal">Eur J Hum Genet. </span>2000 Jul;<span class="ref-vol">8</span>(7):545–551.</span> [<a href="/pubmed/10909856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Pedigree+tests+of+transmission+disequilibrium&amp;author=GR+Abecasis&amp;author=WO+Cookson&amp;author=LR+Cardon&amp;volume=8&amp;issue=7&amp;publication_year=2000&amp;pages=545-551&amp;pmid=10909856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Aruga J, Yokota N, Mikoshiba K. Human SLITRK family genes: genomic organization and expression profiling in normal brain and brain tumor tissue. <span><span class="ref-journal">Gene. </span>2003 Oct 2;<span class="ref-vol">315</span>:87–94.</span> [<a href="/pubmed/14557068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Human+SLITRK+family+genes:+genomic+organization+and+expression+profiling+in+normal+brain+and+brain+tumor+tissue&amp;author=J+Aruga&amp;author=N+Yokota&amp;author=K+Mikoshiba&amp;volume=315&amp;publication_year=2003&amp;pages=87-94&amp;pmid=14557068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Abelson JF, Kwan KY, O’Roak BJ, Baek DY, Stillman AA, Morgan TM, et al.  Sequence variants in SLITRK1 are associated with Tourette’s syndrome. <span><span class="ref-journal">Science. </span>2005 Oct 14;<span class="ref-vol">310</span>(5746):317–320.</span> [<a href="/pubmed/16224024" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Sequence+variants+in+SLITRK1+are+associated+with+Tourette’s+syndrome&amp;author=JF+Abelson&amp;author=KY+Kwan&amp;author=BJ+O’Roak&amp;author=DY+Baek&amp;author=AA+Stillman&amp;volume=310&amp;issue=5746&amp;publication_year=2005&amp;pages=317-320&amp;pmid=16224024&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Reichart LF. Neurotrophin-regulated signalling pathways. <span><span class="ref-journal">Phil Trans R Soc B. </span>2006;(361):1545–1564.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1664664/">PMC free article</a>]</span> [<a href="/pubmed/16939974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phil+Trans+R+Soc+B&amp;title=Neurotrophin-regulated+signalling+pathways&amp;author=LF+Reichart&amp;issue=361&amp;publication_year=2006&amp;pages=1545-1564&amp;pmid=16939974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Schmidt HD, Duman RS. The role of neurotrophic factors in adult hippocampal neurogenesis, antidepressant treatments and animal models of depressive-like behavior. <span><span class="ref-journal">Behav Pharmacol. </span>2007 Sep;<span class="ref-vol">18</span>(5-6):391–418.</span> [<a href="/pubmed/17762509" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behav+Pharmacol&amp;title=The+role+of+neurotrophic+factors+in+adult+hippocampal+neurogenesis,+antidepressant+treatments+and+animal+models+of+depressive-like+behavior&amp;author=HD+Schmidt&amp;author=RS+Duman&amp;volume=18&amp;issue=5-6&amp;publication_year=2007&amp;pages=391-418&amp;pmid=17762509&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Neves-Pereira M, Mundo E, Muglia P, King N, Macciardi F, Kennedy JL. The brain-derived neurotrophic factor gene confers susceptibility to bipolar disorder: evidence from a family-based association study. <span><span class="ref-journal">Am J Hum Genet. </span>2002 Sep;<span class="ref-vol">71</span>(3):651–655.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379201/">PMC free article</a>]</span> [<a href="/pubmed/12161822" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=The+brain-derived+neurotrophic+factor+gene+confers+susceptibility+to+bipolar+disorder:+evidence+from+a+family-based+association+study&amp;author=M+Neves-Pereira&amp;author=E+Mundo&amp;author=P+Muglia&amp;author=N+King&amp;author=F+Macciardi&amp;volume=71&amp;issue=3&amp;publication_year=2002&amp;pages=651-655&amp;pmid=12161822&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Sklar P, Gabriel SB, McInnis MG, Bennett P, Lim YM, Tsan G, et al.  Family-based association study of 76 candidate genes in bipolar disorder: BDNF is a potential risk locus. Brain-derived neutrophic factor. <span><span class="ref-journal">Mol Psychiatry. </span>2002;<span class="ref-vol">7</span>(6):579–593.</span> [<a href="/pubmed/12140781" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Family-based+association+study+of+76+candidate+genes+in+bipolar+disorder:+BDNF+is+a+potential+risk+locus.+Brain-derived+neutrophic+factor&amp;author=P+Sklar&amp;author=SB+Gabriel&amp;author=MG+McInnis&amp;author=P+Bennett&amp;author=YM+Lim&amp;volume=7&amp;issue=6&amp;publication_year=2002&amp;pages=579-593&amp;pmid=12140781&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Lohoff FW, Sander T, Ferraro TN, Dahl JP, Gallinat J, Berrettini WH. Confirmation of association between the Val66Met polymorphism in the brain-derived neurotrophic factor (BDNF) gene and bipolar I disorder. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2005 Nov 5;<span class="ref-vol">139B</span>(1):51–53.</span> [<a href="/pubmed/16152572" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Confirmation+of+association+between+the+Val66Met+polymorphism+in+the+brain-derived+neurotrophic+factor+(BDNF)+gene+and+bipolar+I+disorder&amp;author=FW+Lohoff&amp;author=T+Sander&amp;author=TN+Ferraro&amp;author=JP+Dahl&amp;author=J+Gallinat&amp;volume=139B&amp;issue=1&amp;publication_year=2005&amp;pages=51-53&amp;pmid=16152572&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Liu L, Foroud T, Xuei X, Berrettini W, Byerley W, Coryell W, et al.  Evidence of association between brain-derived neurotrophic factor gene and bipolar disorder. <span><span class="ref-journal">Psychiatr Genet. </span>2008 Dec;<span class="ref-vol">18</span>(6):267–274.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2653694/">PMC free article</a>]</span> [<a href="/pubmed/19018231" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychiatr+Genet&amp;title=Evidence+of+association+between+brain-derived+neurotrophic+factor+gene+and+bipolar+disorder&amp;author=L+Liu&amp;author=T+Foroud&amp;author=X+Xuei&amp;author=W+Berrettini&amp;author=W+Byerley&amp;volume=18&amp;issue=6&amp;publication_year=2008&amp;pages=267-274&amp;pmid=19018231&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Maher B. Personal genomes: The case of the missing heritability. <span><span class="ref-journal">Nature. </span>2008 Nov 6;<span class="ref-vol">456</span>(7218):18–21.</span> [<a href="/pubmed/18987709" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Personal+genomes:+The+case+of+the+missing+heritability&amp;author=B+Maher&amp;volume=456&amp;issue=7218&amp;publication_year=2008&amp;pages=18-21&amp;pmid=18987709&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">APA. <span class="ref-journal">Diagnostic and Statistical Manual of Mental Disorders.</span> 4-Text Revision. Washington DC: 2000.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+and+Statistical+Manual+of+Mental+Disorders&amp;publication_year=2000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Nurnberger JI, Jr, Blehar MC, Kaufmann CA, York-Cooler C, Simpson SG, Harkavy-Friedman J, et al.  Diagnostic interview for genetic studies. Rationale, unique features, and training. NIMH Genetics Initiative. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1994 Nov;<span class="ref-vol">51</span>(11):849–859.</span> discussion 863-844. [<a href="/pubmed/7944874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Diagnostic+interview+for+genetic+studies.+Rationale,+unique+features,+and+training.+NIMH+Genetics+Initiative&amp;author=JI+Nurnberger&amp;author=MC+Blehar&amp;author=CA+Kaufmann&amp;author=C+York-Cooler&amp;author=SG+Simpson&amp;volume=51&amp;issue=11&amp;publication_year=1994&amp;pages=849-859&amp;pmid=7944874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Feighner JP, Robins E, Guze SB, Woodruff RA, Jr, Winokur G, Munoz R. Diagnostic criteria for use in psychiatric research. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1972 Jan;<span class="ref-vol">26</span>(1):57–63.</span> [<a href="/pubmed/5009428" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Diagnostic+criteria+for+use+in+psychiatric+research&amp;author=JP+Feighner&amp;author=E+Robins&amp;author=SB+Guze&amp;author=RA+Woodruff&amp;author=G+Winokur&amp;volume=26&amp;issue=1&amp;publication_year=1972&amp;pages=57-63&amp;pmid=5009428&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Akiskal HS, Pinto O. The evolving bipolar spectrum. Prototypes I, II, III, and IV. <span><span class="ref-journal">Psychiatr Clin North Am. </span>1999 Sep;<span class="ref-vol">22</span>(3):517–534. vii.</span> [<a href="/pubmed/10550853" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychiatr+Clin+North+Am&amp;title=The+evolving+bipolar+spectrum.+Prototypes+I,+II,+III,+and+IV&amp;author=HS+Akiskal&amp;author=O+Pinto&amp;volume=22&amp;issue=3&amp;publication_year=1999&amp;pages=517-534&amp;pmid=10550853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Horne JA, Ostberg O. A self-assessment questionnaire to determine morningness-eveningness in human circadian rhythms. <span><span class="ref-journal">Int J Chronobiol. </span>1976;<span class="ref-vol">4</span>(2):97–110.</span> [<a href="/pubmed/1027738" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Chronobiol&amp;title=A+self-assessment+questionnaire+to+determine+morningness-eveningness+in+human+circadian+rhythms&amp;author=JA+Horne&amp;author=O+Ostberg&amp;volume=4&amp;issue=2&amp;publication_year=1976&amp;pages=97-110&amp;pmid=1027738&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Brown GL, Goodwin FK, Ballenger JC, Goyer PF, Major LF. Aggression in humans correlates with cerebrospinal fluid amine metabolites. <span><span class="ref-journal">Psychiatry Res. </span>1979 Oct;<span class="ref-vol">1</span>(2):131–139.</span> [<a href="/pubmed/95232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychiatry+Res&amp;title=Aggression+in+humans+correlates+with+cerebrospinal+fluid+amine+metabolites&amp;author=GL+Brown&amp;author=FK+Goodwin&amp;author=JC+Ballenger&amp;author=PF+Goyer&amp;author=LF+Major&amp;volume=1&amp;issue=2&amp;publication_year=1979&amp;pages=131-139&amp;pmid=95232&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Cloninger CR, Przybeck TR, Svrakic DM, Wetzel RD.  <span class="ref-journal">The temperament and character inventory (TCI): A guide to its development and use.</span> Center for Psychobiology of Personality; St. Louis: 1994. pp. 1–85. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+temperament+and+character+inventory+(TCI):+A+guide+to+its+development+and+use&amp;author=CR+Cloninger&amp;author=TR+Przybeck&amp;author=DM+Svrakic&amp;author=RD+Wetzel&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Wender EH. Attention-deficit hyperactivity disorders in adolescence. <span><span class="ref-journal">J Dev Behav Pediatr. </span>1995 Jun;<span class="ref-vol">16</span>(3):192–195.</span> [<a href="/pubmed/7560123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Dev+Behav+Pediatr&amp;title=Attention-deficit+hyperactivity+disorders+in+adolescence&amp;author=EH+Wender&amp;volume=16&amp;issue=3&amp;publication_year=1995&amp;pages=192-195&amp;pmid=7560123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Zuckerman M, Kuhlman DM, Joireman J, Teta P, Kraft M. A comparison of three structural models of personality: The big three, the big five, and the alternative five. <span><span class="ref-journal">Journal of Personality and Social Psychology. </span>1993;<span class="ref-vol">65</span>:757–768.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Personality+and+Social+Psychology&amp;title=A+comparison+of+three+structural+models+of+personality:+The+big+three,+the+big+five,+and+the+alternative+five&amp;author=M+Zuckerman&amp;author=DM+Kuhlman&amp;author=J+Joireman&amp;author=P+Teta&amp;author=M+Kraft&amp;volume=65&amp;publication_year=1993&amp;pages=757-768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">McCarroll SA, Kuruvilla FG, Korn JM, Cawley S, Nemesh J, Wysoker A, et al.  Integrated detection and population-genetic analysis of SNPs and copy number variation. <span><span class="ref-journal">Nat Genet. </span>2008 Oct;<span class="ref-vol">40</span>(10):1166–1174.</span> [<a href="/pubmed/18776908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Integrated+detection+and+population-genetic+analysis+of+SNPs+and+copy+number+variation&amp;author=SA+McCarroll&amp;author=FG+Kuruvilla&amp;author=JM+Korn&amp;author=S+Cawley&amp;author=J+Nemesh&amp;volume=40&amp;issue=10&amp;publication_year=2008&amp;pages=1166-1174&amp;pmid=18776908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Korn JM, Kuruvilla FG, McCarroll SA, Wysoker A, Nemesh J, Cawley S, et al.  Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat Genet. </span>2008 Oct;<span class="ref-vol">40</span>(10):1253–1260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=JM+Korn&amp;author=FG+Kuruvilla&amp;author=SA+McCarroll&amp;author=A+Wysoker&amp;author=J+Nemesh&amp;volume=40&amp;issue=10&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Sankararaman S, Sridhar S, Kimmel G, Halperin E. Estimating local ancestry in admixed populations. <span><span class="ref-journal">Am J Hum Genet. </span>2008 Feb;<span class="ref-vol">82</span>(2):290–303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2664993/">PMC free article</a>]</span> [<a href="/pubmed/18252211" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Estimating+local+ancestry+in+admixed+populations&amp;author=S+Sankararaman&amp;author=S+Sridhar&amp;author=G+Kimmel&amp;author=E+Halperin&amp;volume=82&amp;issue=2&amp;publication_year=2008&amp;pages=290-303&amp;pmid=18252211&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000 Jun;<span class="ref-vol">155</span>(2):945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;issue=2&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="citation">Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. <span><span class="ref-journal">Genetics. </span>2003 Aug;<span class="ref-vol">164</span>(4):1567–1587.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1462648/">PMC free article</a>]</span> [<a href="/pubmed/12930761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data:+linked+loci+and+correlated+allele+frequencies&amp;author=D+Falush&amp;author=M+Stephens&amp;author=JK+Pritchard&amp;volume=164&amp;issue=4&amp;publication_year=2003&amp;pages=1567-1587&amp;pmid=12930761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="citation">Li JZ, Absher DM, Tang H, Southwick AM, Casto AM, Ramachandran S, et al.  Worldwide human relationships inferred from genome-wide patterns of variation. <span><span class="ref-journal">Science. </span>2008 Feb 22;<span class="ref-vol">319</span>(5866):1100–1104.</span> [<a href="/pubmed/18292342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Worldwide+human+relationships+inferred+from+genome-wide+patterns+of+variation&amp;author=JZ+Li&amp;author=DM+Absher&amp;author=H+Tang&amp;author=AM+Southwick&amp;author=AM+Casto&amp;volume=319&amp;issue=5866&amp;publication_year=2008&amp;pages=1100-1104&amp;pmid=18292342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="citation">Zeggini E, Scott LJ, Saxena R, Voight BF, Marchini JL, Hu T, et al.  Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2008 May;<span class="ref-vol">40</span>(5):638–645.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=LJ+Scott&amp;author=R+Saxena&amp;author=BF+Voight&amp;author=JL+Marchini&amp;volume=40&amp;issue=5&amp;publication_year=2008&amp;pages=638-645&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3035981&amp;issue-id=194085&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140093999647968title">Abstract</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">MATERIALS AND METHODS</a></li><li><a href="#S12title">RESULTS</a></li><li><a href="#S21title">DISCUSSION</a></li><li><a href="#S22title">Supplementary Material</a></li><li><a href="#S24title">Acknowledgments</a></li><li><a href="#idm140093998605744title">Footnotes</a></li><li><a href="#idm140093919938512title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3035981/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3035981/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3035981/pdf/nihms178631.pdf">PDF (224K)</a></li> | <li><a href="#" data-citationid="PMC3035981" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3035981%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3035981%2F&amp;text=GENOME-WIDE%20ASSOCIATION%20STUDY%20OF%20BIPOLAR%20DISORDER%20IN%20EUROPEAN%20AMERICAN%20AND%20AFRICAN%20AMERICAN%20INDIVIDUALS" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3035981%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19488044" data-aiid="3035981" data-aid="3035981" data-iid="194085" data-domainid="319" data-domain="nihpa" data-accid="PMC3035981" data-md5="160d7b36ef41ebb1cd39605efff71cd7"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:50:02-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C7104DD4F49F10000000005E503B0&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3035981%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3035981/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>