<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jdr"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3383103/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Association Study of Periodontal Pathogen Colonization</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3383103/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of Dental Research"><meta name="citation_title" content="Genome-wide Association Study of Periodontal Pathogen Colonization"><meta name="citation_authors" content="K. Divaris, K.L. Monda, K.E. North, A.F. Olshan, E.M. Lange, K. Moss, S.P. Barros, J.D. Beck, S. Offenbacher"><meta name="citation_date" content="July 2012"><meta name="citation_issue" content="7 Suppl"><meta name="citation_volume" content="91"><meta name="citation_firstpage" content="S21"><meta name="citation_doi" content="10.1177/0022034512447951"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3383103/?report=abstract"><meta name="citation_pmid" content="22699663"><meta name="DC.Title" content="Genome-wide Association Study of Periodontal Pathogen Colonization"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="International Association for Dental Research"><meta name="DC.Contributor" content="K. Divaris"><meta name="DC.Contributor" content="K.L. Monda"><meta name="DC.Contributor" content="K.E. North"><meta name="DC.Contributor" content="A.F. Olshan"><meta name="DC.Contributor" content="E.M. Lange"><meta name="DC.Contributor" content="K. Moss"><meta name="DC.Contributor" content="S.P. Barros"><meta name="DC.Contributor" content="J.D. Beck"><meta name="DC.Contributor" content="S. Offenbacher"><meta name="DC.Date" content="2012 Jul"><meta name="DC.Identifier" content="10.1177/0022034512447951"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Association Study of Periodontal Pathogen Colonization"><meta property="og:type" content="article"><meta property="og:description" content="Pathological shifts of the human microbiome are characteristic of many diseases, including chronic periodontitis. To date, there is limited evidence on host genetic risk loci associated with periodontal pathogen colonization. We conducted a genome-wide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3383103/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jdr.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3383103/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3383103/pdf/10.1177_0022034512447951.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D87ADDD5467C100000000018F0114.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3383103%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22J Dent Res%22[journal]" class="navlink">J Dent Res</a></li><li class="accid">PMC3383103</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jdr.gif" alt="Logo of jdr" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="poly" coords="0,0,500,0,500,55,225,55,225,75,0,75" alt="Home" title="Home" href="http://jdr.sagepub.com/" ref="reftype=publisher&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="226,56,326,75" alt="About" title="About" href="http://www.sagepub.com/journalsProdDesc.nav?prodId=Journal201925" ref="reftype=publisher&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="340,56,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="http://www.sagepub.com/journalsProdDesc.nav?ct_p=manuscriptSubmission&amp;prodId=Journal201925" ref="reftype=publisher&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Dent Res</a></span>. 2012 Jul; 91(7 Suppl): S21–S28. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1177%2F0022034512447951" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1177/0022034512447951</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3383103</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22699663">22699663</a></div></div></div></div><h1 class="content-title">Genome-wide Association Study of Periodontal Pathogen Colonization</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Divaris%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494706427840" co-class="co-affbox">K. Divaris</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Monda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494707181840" co-class="co-affbox">K.L. Monda</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=North%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494707179568" co-class="co-affbox">K.E. North</a>,<sup>2,</sup><sup>4</sup> <a href="/pubmed/?term=Olshan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494707253600" co-class="co-affbox">A.F. Olshan</a>,<sup>2</sup> <a href="/pubmed/?term=Lange%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494777750448" co-class="co-affbox">E.M. Lange</a>,<sup>5,</sup><sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Moss%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494711036064" co-class="co-affbox">K. Moss</a>,<sup>8</sup> <a href="/pubmed/?term=Barros%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494705074128" co-class="co-affbox">S.P. Barros</a>,<sup>9</sup> <a href="/pubmed/?term=Beck%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494707224864" co-class="co-affbox">J.D. Beck</a>,<sup>8</sup> and  <a href="/pubmed/?term=Offenbacher%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663" class="affpopup" co-rid="_co_idm140494707222880" co-class="co-affbox">S. Offenbacher</a><sup>9,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140494706427840"><h3 class="no_margin">K. Divaris</h3><p><sup>1</sup>Department of Pediatric Dentistry, School of Dentistry, University of North Carolina-Chapel Hill, 228 Brauer Hall, CB#7450, NC 27599, USA</p><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina-Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Divaris%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">K. Divaris</a></div></div><div id="_co_idm140494707181840"><h3 class="no_margin">K.L. Monda</h3><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina-Chapel Hill, NC, USA</p><p><sup>3</sup>The Center for Observational Research, Amgen Inc., Thousand Oaks, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Monda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">K.L. Monda</a></div></div><div id="_co_idm140494707179568"><h3 class="no_margin">K.E. North</h3><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina-Chapel Hill, NC, USA</p><p><sup>4</sup>Carolina Center for Genome Sciences, University of North Carolina-Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=North%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">K.E. North</a></div></div><div id="_co_idm140494707253600"><h3 class="no_margin">A.F. Olshan</h3><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina-Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Olshan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">A.F. Olshan</a></div></div><div id="_co_idm140494777750448"><h3 class="no_margin">E.M. Lange</h3><p><sup>5</sup>Department of Genetics, School of Medicine, University of North Carolina-Chapel Hill, NC, USA</p><p><sup>6</sup>Department of Biostatistics, Gillings School of Global Public Health, University of North Carolina-Chapel Hill, NC, USA</p><p><sup>7</sup>Lineberger Comprehensive Cancer Center, University of North Carolina-Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Lange%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">E.M. Lange</a></div></div><div id="_co_idm140494711036064"><h3 class="no_margin">K. Moss</h3><p><sup>8</sup>Department of Dental Ecology, School of Dentistry, University of North Carolina-Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Moss%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">K. Moss</a></div></div><div id="_co_idm140494705074128"><h3 class="no_margin">S.P. Barros</h3><p><sup>9</sup>Department of Periodontology, School of Dentistry, University of North Carolina-Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Barros%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">S.P. Barros</a></div></div><div id="_co_idm140494707224864"><h3 class="no_margin">J.D. Beck</h3><p><sup>8</sup>Department of Dental Ecology, School of Dentistry, University of North Carolina-Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Beck%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">J.D. Beck</a></div></div><div id="_co_idm140494707222880"><h3 class="no_margin">S. Offenbacher</h3><p><sup>9</sup>Department of Periodontology, School of Dentistry, University of North Carolina-Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Offenbacher%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22699663">S. Offenbacher</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140494705443152_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140494705443152_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140494705443152_ai" style="display:none"><div class="fm-affl" id="aff1-0022034512447951" lang="en"><sup>1</sup>Department of Pediatric Dentistry, School of Dentistry, University of North Carolina-Chapel Hill, 228 Brauer Hall, CB#7450, NC 27599, USA</div><div class="fm-affl" id="aff2-0022034512447951" lang="en"><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina-Chapel Hill, NC, USA</div><div class="fm-affl" id="aff3-0022034512447951" lang="en"><sup>3</sup>The Center for Observational Research, Amgen Inc., Thousand Oaks, CA, USA</div><div class="fm-affl" id="aff4-0022034512447951" lang="en"><sup>4</sup>Carolina Center for Genome Sciences, University of North Carolina-Chapel Hill, NC, USA</div><div class="fm-affl" id="aff5-0022034512447951" lang="en"><sup>5</sup>Department of Genetics, School of Medicine, University of North Carolina-Chapel Hill, NC, USA</div><div class="fm-affl" id="aff6-0022034512447951" lang="en"><sup>6</sup>Department of Biostatistics, Gillings School of Global Public Health, University of North Carolina-Chapel Hill, NC, USA</div><div class="fm-affl" id="aff7-0022034512447951" lang="en"><sup>7</sup>Lineberger Comprehensive Cancer Center, University of North Carolina-Chapel Hill, NC, USA</div><div class="fm-affl" id="aff8-0022034512447951" lang="en"><sup>8</sup>Department of Dental Ecology, School of Dentistry, University of North Carolina-Chapel Hill, NC, USA</div><div class="fm-affl" id="aff9-0022034512447951" lang="en"><sup>9</sup>Department of Periodontology, School of Dentistry, University of North Carolina-Chapel Hill, NC, USA</div><div id="corresp1-0022034512447951"><sup>*</sup><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.cnu.yrtsitned@sabneffo" class="oemail">ude.cnu.yrtsitned@sabneffo</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140494705443152_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2012 International &amp; American Associations for Dental Research</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3383103/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Appendix
</div>

<div><a href="/pmc/articles/PMC3383103/bin/supp_91_7-suppl_S21__index.html" data-ga-action="click_feat_suppl">supp_91_7-suppl_S21__index.html</a><span style="color:gray"> (702 bytes)</span></div><div class="small guid">GUID:&nbsp;04F61D22-A7DC-4E47-BE60-220B421D6119</div>
<div><a href="/pmc/articles/PMC3383103/bin/supp_91.7-suppl.S21_10.1177_0022034512447951_Appendix.pdf" data-ga-action="click_feat_suppl">supp_91.7-suppl.S21_10.1177_0022034512447951_Appendix.pdf</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID:&nbsp;371F6C9D-1899-4D21-A779-B1B49DA30643</div>
</div></dd></dl></div><div id="idm140494707217408" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140494707217408title" style="text-transform:uppercase">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Pathological shifts of the human microbiome are characteristic of many diseases, including chronic periodontitis. To date, there is limited evidence on host genetic risk loci associated with periodontal pathogen colonization. We conducted a genome-wide association (GWA) study among 1,020 white participants of the <em>Atherosclerosis Risk in Communities Study</em>, whose periodontal diagnosis ranged from healthy to severe chronic periodontitis, and for whom “checkerboard” DNA-DNA hybridization quantification of 8 periodontal pathogens was performed. We examined 3 traits: “high red” and “high orange” bacterial complexes, and “high” <em>Aggregatibacter actinomycetemcomitans</em> (Aa) colonization. Genotyping was performed on the Affymetrix 6.0 platform. Imputation to 2.5 million markers was based on HapMap II-CEU, and a multiple-test correction was applied (genome-wide threshold of p &lt; 5 × 10<sup>−8</sup>). We detected no genome-wide significant signals. However, 13 loci, including <em>KCNK1, FBXO38, UHRF2, IL33, RUNX2, TRPS1, CAMTA1</em>, and <em>VAMP3</em>, provided suggestive evidence (p &lt; 5 × 10<sup>−6</sup>) of association. All associations reported for “red” and “orange” complex microbiota, but not for Aa, had the same effect direction in a second sample of 123 African-American participants. None of these polymorphisms was associated with periodontitis diagnosis. Investigations replicating these findings may lead to an improved understanding of the complex nature of host-microbiome interactions that characterizes states of health and disease.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">bacteria, infection, chronic periodontitis, genetics, genome-wide association studies, dentistry</span></div></div><div id="section1-0022034512447951" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="section1-0022034512447951title" style="text-transform:uppercase">Introduction</h2><p id="__p2" class="p p-first">The study of the composition and the role of the human microbiome is a rapidly evolving and high-priority research area (<a href="#bibr28-0022034512447951" rid="bibr28-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_338835336">Turnbaugh <em>et al</em>., 2007</a>). A dysbiotic relationship between a host and its microbiota is theorized to be an important component of many morbidities, including Crohn’s disease, obesity, cancer, periodontitis, and others (<a href="#bibr24-0022034512447951" rid="bibr24-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_338835339">Socransky and Haffajee, 2002</a>; <a href="#bibr5-0022034512447951" rid="bibr5-0022034512447951" class=" bibr popnode">Badger <em>et al</em>., 2010</a>; <a href="#bibr1-0022034512447951" rid="bibr1-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_338835337">Ahn <em>et al</em>., 2012</a>). Accordingly, investigations of the oral microbiome are gaining increased attention (<a href="#bibr4-0022034512447951" rid="bibr4-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278364">Avila <em>et al</em>., 2009</a>). To date, over 600 species have been identified in the oral microbiome, and its diversity is greater than was initially theorized (<a href="#bibr22-0022034512447951" rid="bibr22-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_338835338">Paster <em>et al</em>., 2006</a>; <a href="#bibr16-0022034512447951" rid="bibr16-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278342">Keijser <em>et al</em>., 2008</a>). Specific bacterial species that are implicated in chronic periodontitis (CP) have been identified; in general, they are commensal and include Gram-negative anaerobes. Noteworthy, while harboring of periodontal pathogens is virtually universal, only a small proportion of individuals develop severe forms of CP.</p><p id="__p3" class="p p-last">There is a large body of literature in candidate-gene studies investigating the genetics of host inflammatory response, inflammatory mediators, and cytokines in periodontitis (<a href="#bibr29-0022034512447951" rid="bibr29-0022034512447951" class=" bibr popnode">Zhang <em>et al</em>., 2011</a>). The study of the interactions between host genetic factors and oral microbial colonization, termed “periodontal infectogenomics” (<a href="#bibr19-0022034512447951" rid="bibr19-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278357">Nibali <em>et al</em>., 2011</a>), is an important emerging area of research. Recent candidate-gene studies have suggested possible roles of polymorphisms in the <em>IL-6, Fc gamma</em> receptor, <em>IL-1A</em>, and <em>IL-1B</em> (<a href="#bibr17-0022034512447951" rid="bibr17-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278350">Nibali <em>et al</em>., 2007</a>, <a href="#bibr18-0022034512447951" rid="bibr18-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278366">2010</a>, <a href="#bibr19-0022034512447951" rid="bibr19-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278351">2011</a>), whereas an earlier study by Socransky and colleagues (<a href="#bibr27-0022034512447951" rid="bibr27-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278347">Socransky <em>et al</em>., 2000</a>) reported associations between <em>IL-1A</em> and <em>IL-1B</em> polymorphisms and “high” colonization with bacteria of the “red” and “orange complex”. No study to date has carried out a whole-genome investigation for host genetic markers of colonization with periodontal bacteria. To this end, and to add to the knowledge base of infectogenomics, as well as the genetic component of periodontitis, the aim of this study was to investigate susceptibility loci for colonization with subgingival periodontal microbiota using a GWA analysis approach.</p></div><div id="section2-0022034512447951" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="section2-0022034512447951title" style="text-transform:uppercase">Materials &amp; Methods</h2><p id="__p4" class="p p-first">A detailed description of the study population, genotyping and imputation, quality control, and population stratification procedures is included in the online Appendix. In brief, we conducted a GWA study among 1,020 self-reported white participants of the Atherosclerosis Risk in Communities (ARIC) longitudinal cohort investigation (<a href="#bibr2-0022034512447951" rid="bibr2-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278348">The ARIC Investigators, 1989</a>). While ARIC is a study of atherosclerosis, CVD risk factors, and outcomes, a complete oral-dental examination took place between 1996 and 1998 during the fourth ARIC visit. As part of the Dental ARIC, apart from a complete clinical examination, five trained examiners collected subgingival microbial plaque samples (<a href="#bibr8-0022034512447951" rid="bibr8-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278360">Beck <em>et al</em>., 2001</a>) from the mesial surfaces of all first permanent molars in a subset of participants. Participants’ periodontal diagnoses ranged from healthy/mild disease (41%) to severe periodontitis (19%).</p><p id="__p5">The “checkerboard” DNA-DNA hybridization method (<a href="#bibr25-0022034512447951" rid="bibr25-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278358">Socransky <em>et al</em>., 1994</a>) was used on plaque samples to measure the extent of subgingival colonization with 8 periodontal pathogens: <em>Prevotella intermedia</em> [American Type Culture Collection (ATCC) 25611], <em>Campylobacter rectus</em> (ATCC 33238), <em>Fusobacterium nucleatum</em> (ATCC 10953), and <em>Prevotella nigrescens</em> (ATCC 33563) (belonging to the “orange” complex); <em>Porphyromonas gingivalis</em> (ATCC 33277), <em>Tannerella forsythia</em> (ATCC 43037), and <em>Treponema denticola</em> (ATCC 35404) (“red” complex); and <em>Aggregatibacter actinomycetemcomitans</em> (ATCC 43718). In that method, bacterial levels are expressed as counts relative to established microbial standards. Three dichotomous traits of “high” colonization with “red” complex, “orange” complex (<a href="#bibr26-0022034512447951" rid="bibr26-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278369">Socransky <em>et al</em>., 1998</a>), and <em>Aggregatibacter actinomycetemcomitans</em> (<em>Aa</em>) were considered for analytical purposes. The two composite phenotypes were derived by the summation of bacterial counts for each species belonging to the “red” (n = 3) and “orange” (n = 4) groups, as described above. Because <em>Porphyromonas gingivalis</em> is considered the major periodontal pathogen implicated in periodontitis in adult populations, we explored for additional risk loci using its “high” colonization phenotype, defined as above, as a separate trait. Various approaches in defining the bacterial colonization profiles have been previously used, including summations of the absolute microbial counts (<a href="#bibr20-0022034512447951" rid="bibr20-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278362">Offenbacher <em>et al</em>., 2007</a>), tertile categorization (<a href="#bibr10-0022034512447951" rid="bibr10-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278349">Desvarieux <em>et al</em>., 2005</a>), and five-level categorization of log10-transformed counts (<a href="#bibr21-0022034512447951" rid="bibr21-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278346">Papapanou <em>et al</em>., 2000</a>). For the present investigation, we defined a “high” colonization trait as the top quintile (20%) of each trait’s distribution. The rationale for the selection of this phenotype is based on the fact that the “checkerboard” semi-quantitative method has a lower detection threshold of ~10<sup>4</sup> and reduced precision in the lower end of the distribution, whereas individuals with a “high” bacterial colonization profile may be those with reduced or impaired host and at high risk for periodontal tissue destruction.</p><p id="__p6">Participants’ DNA was extracted from blood samples drawn from an antecubital vein, and genotyping was performed with the Affymetrix Genome-Wide Human SNP Array 6.0 chip. The platform offers 906,600 markers for SNPs. Following rigorous quality control procedures, imputation to 2.5 million markers was performed with 669,450 SNPs and MACH v1.0.16 (<a href="http://www.sph.umich.edu/csg/abecasis/MaCH/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MaCH/index.html</a>), based on HapMap Phase II CEU build 36 (Appendix Table 1). For analytical purposes, the dichotomous traits of “high colonization” phenotypes were entered into 3 logistic regression models, assuming multiplicative (log-additive) allelic effects. The logistic regression models included age, sex, examination center, and 10 principal components from the population stratification analysis as covariates. A multiple-comparisons correction was applied, and with 1 million independent tests assumed, the threshold of genome-wide statistical significance was set at p &lt; 5 × 10<sup>−8</sup>. An additional <em>a priori</em> arbitrary threshold of p &lt; 5 × 10<sup>−6</sup> was set to prioritize and investigate loci with suggestive evidence of association. We also compiled a list of the 10,000 SNPs with the lowest p value for each of the 3 traits that we examined. We considered SNPs with minor allele frequency (MAF) 5% or greater.</p><p id="__p7" class="p p-last">In a second step, we used a separate sample of 123 African American (AA) Dental ARIC participants (mean age, 61.5 yrs) to verify the SNP associations that we prioritized from the white sample GWAS. The distribution of the AA participants’ gender, smoking status, DM, and bacterial colonization profile by periodontal diagnosis is presented in Appendix Table 2. We followed identical quality control and analysis methods, including genotyping, population stratification, and genetic modeling. Imputation to 2,653,878 SNPs for the AA sample was based on the YRI sample of HapMap Phase II build 36. All GWA analyses were performed with the ProbABEL software (<a href="#bibr3-0022034512447951" rid="bibr3-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278353">Aulchenko <em>et al</em>., 2010</a>). SNP annotations were performed with WGAViewer ver.1.26l (<a href="#bibr12-0022034512447951" rid="bibr12-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278344">Ge <em>et al</em>., 2008</a>) and Snipper ver. 1.2 (<a href="http://csg.sph.umich.edu/boehnke/snipper/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/boehnke/snipper/</a>), and loci visualizations with LocusZoom ver.1.1 (<a href="#bibr23-0022034512447951" rid="bibr23-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278343">Pruim <em>et al</em>., 2010</a>) and Haploview ver.4.2 (<a href="#bibr6-0022034512447951" rid="bibr6-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278359">Barrett <em>et al</em>., 2005</a>). We used Polyphen-2 (<a href="http://genetics.bwh.harvard.edu/pph2/index.shtml" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genetics.bwh.harvard.edu/pph2/index.shtml</a>) to assess potentially damaging non-synonymous SNPs. Gene reporting was based on the <em>“</em>HUGO Gene Nomenclature” (<a href="http://www.genenames.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.genenames.org</a>).</p></div><div id="section3-0022034512447951" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="section3-0022034512447951title" style="text-transform:uppercase">Results</h2><p id="__p8" class="p p-first">The sample’s descriptive information is presented in <a href="/pmc/articles/PMC3383103/table/table1-0022034512447951/" target="table" class="fig-table-link figpopup" rid-figpopup="table1-0022034512447951" rid-ob="ob-table1-0022034512447951" co-legend-rid=""><span>Table 1</span></a>. Participants had a mean age of 63 yrs and approximately even gender distribution. Forty percent were never-smokers and 12% current smokers. Fourteen percent were diagnosed with DM. “High” bacterial colonization profiles were found in all groups of periodontal diagnosis according to the CDC classification; however, the prevalence of “high red” colonization cases was more than double among participants with severe periodontitis compared with those with mild or no disease. Similar, but less pronounced, associations were noted for “orange” complex and <em>Aa</em>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="table1-0022034512447951"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p9">Distribution of Participants’ Demographic Characteristics, Smoking Status, and Bacterial “High Colonization” (defined as the top quintile of the distribution) Profile (with “Red” complex, “Orange” complex, and <em>Aggregatibacter actinomycetemcomitans</em>) Overall, and Stratified by Periodontal Diagnosis (CDC/AAP classification) among the Dental ARIC Study Self-reported White Participants (n = 1,020)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140494703056688" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="center">Total<hr>
</th><th rowspan="1" colspan="1" align="center">Healthy/Mild<hr>
</th><th rowspan="1" colspan="1" align="center">Moderate<hr>
</th><th rowspan="1" colspan="1" align="center">Severe<hr>
</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="center">n (column %)*</th><th rowspan="1" colspan="1" align="center">n (column %)*</th><th rowspan="1" colspan="1" align="center">n (column %)*</th><th rowspan="1" colspan="1" align="center">n (column %)*</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">n (row %)</td><td rowspan="1" colspan="1">1,020 (100)</td><td rowspan="1" colspan="1">416 (41)</td><td rowspan="1" colspan="1">415 (41)</td><td rowspan="1" colspan="1">189 (19)</td></tr><tr><td rowspan="1" colspan="1">Sex</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> Females</td><td rowspan="1" colspan="1">478 (47)</td><td rowspan="1" colspan="1">246 (59)</td><td rowspan="1" colspan="1">178 (43)</td><td rowspan="1" colspan="1">54 (29)</td></tr><tr><td rowspan="1" colspan="1"> Males</td><td rowspan="1" colspan="1">542 (53)</td><td rowspan="1" colspan="1">170 (41)</td><td rowspan="1" colspan="1">237 (57)</td><td rowspan="1" colspan="1">135 (71)</td></tr><tr><td rowspan="1" colspan="1">Smoking status</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> Never</td><td rowspan="1" colspan="1">410 (41)</td><td rowspan="1" colspan="1">204 (50)</td><td rowspan="1" colspan="1">151 (37)</td><td rowspan="1" colspan="1">55 (30)</td></tr><tr><td rowspan="1" colspan="1"> Former</td><td rowspan="1" colspan="1">468 (47)</td><td rowspan="1" colspan="1">166 (40)</td><td rowspan="1" colspan="1">211 (52)</td><td rowspan="1" colspan="1">91 (50)</td></tr><tr><td rowspan="1" colspan="1"> Current</td><td rowspan="1" colspan="1">121 (12)</td><td rowspan="1" colspan="1">42 (10)</td><td rowspan="1" colspan="1">42 (10)</td><td rowspan="1" colspan="1">37 (20)</td></tr><tr><td rowspan="1" colspan="1">Diabetes mellitus</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> No</td><td rowspan="1" colspan="1">875 (86)</td><td rowspan="1" colspan="1">379 (91)</td><td rowspan="1" colspan="1">353 (85)</td><td rowspan="1" colspan="1">143 (76)</td></tr><tr><td rowspan="1" colspan="1"> Yes</td><td rowspan="1" colspan="1">144 (14)</td><td rowspan="1" colspan="1">37 (9)</td><td rowspan="1" colspan="1">62 (15)</td><td rowspan="1" colspan="1">45 (24)</td></tr><tr><td rowspan="1" colspan="1">Age (yrs; mean, standard deviation)</td><td rowspan="1" colspan="1">63.2 (5.7)</td><td rowspan="1" colspan="1">62.3 (5.5)</td><td rowspan="1" colspan="1">63.6 (5.7)</td><td rowspan="1" colspan="1">64.0 (5.9)</td></tr><tr><td rowspan="1" colspan="1">“High” bacterial colonization (n, % of column)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> “Red” complex</td><td rowspan="1" colspan="1">203 (20)</td><td rowspan="1" colspan="1">58 (14)</td><td rowspan="1" colspan="1">84 (20)</td><td rowspan="1" colspan="1">61 (32)</td></tr><tr><td rowspan="1" colspan="1"> “Orange” complex</td><td rowspan="1" colspan="1">201 (20)</td><td rowspan="1" colspan="1">73 (18)</td><td rowspan="1" colspan="1">72 (17)</td><td rowspan="1" colspan="1">56 (30)</td></tr><tr><td rowspan="1" colspan="1"> 
<em>A. actinomycetemcomitans</em>
</td><td rowspan="1" colspan="1">204 (20)</td><td rowspan="1" colspan="1">75 (18)</td><td rowspan="1" colspan="1">81 (20)</td><td rowspan="1" colspan="1">48 (25)</td></tr></tbody></table></div><div id="largeobj_idm140494703056688" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3383103/table/table1-0022034512447951/?report=objectonly">Open in a separate window</a></div></div><p id="__p10">Of the 2,178,777 examined SNPs, none had p &lt; 5 × 10<sup>−8</sup>. A list of the top 10,000 SNPs ordered in ascending p value (all p &lt; 4.0 × 10<sup>−3</sup>) for each trait is available at: <a href="http://genomewide.net/public/aric/dental/infectogenomics/topSNPs.xls" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genomewide.net/public/aric/dental/infectogenomics/topSNPs.xls</a>. Fifty-three SNPs had p &lt; 5 × 10<sup>−6</sup> and thus were prioritized for further investigation. Lambda inflation factors for 3 traits were: “red”—1.040, “orange”—1.045, and <em>Aa</em>—1.032. The corresponding Q-Q plots are presented in the Appendix (Appendix Fig. 1). Upon inspection of the prioritized SNPs, there were 5 loci that emerged for “red”, 3 loci for “orange”, and 5 loci for <em>Aa</em> colonization (<a href="/pmc/articles/PMC3383103/figure/fig1-0022034512447951/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1-0022034512447951" rid-ob="ob-fig1-0022034512447951" co-legend-rid="lgnd_fig1-0022034512447951"><span>Fig. 1</span></a>). Of those, 1 locus on 1q42 was shared for the “red” complex and <em>Aa</em>. Graphical representations of the genomic areas adjacent to 6 of these loci are presented in <a href="/pmc/articles/PMC3383103/figure/fig2-0022034512447951/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2-0022034512447951" rid-ob="ob-fig2-0022034512447951" co-legend-rid="lgnd_fig2-0022034512447951"><span>Fig. 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1-0022034512447951" co-legend-rid="lgnd_fig1-0022034512447951"><a href="/pmc/articles/PMC3383103/figure/fig1-0022034512447951/" target="figure" rid-figpopup="fig1-0022034512447951" rid-ob="ob-fig1-0022034512447951"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140494701179696" class="figure"><a href="/pmc/articles/PMC3383103/figure/fig1-0022034512447951/" target="figure" rid-figpopup="fig1-0022034512447951" rid-ob="ob-fig1-0022034512447951"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3383103_10.1177_0022034512447951-fig1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 10.1177_0022034512447951-fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3383103/bin/10.1177_0022034512447951-fig1.jpg"></a></div><div id="largeobj_idm140494701179696" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3383103/figure/fig1-0022034512447951/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1-0022034512447951"><div><a class="figpopup" href="/pmc/articles/PMC3383103/figure/fig1-0022034512447951/" target="figure" rid-figpopup="fig1-0022034512447951" rid-ob="ob-fig1-0022034512447951">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p11">Manhattan plots of the GWAS results (–log10 p values of the ~2.5 million examined SNPs arranged by chromosome) for the 3 bacterial pathogen “high” colonization traits among the participants of the Dental Atherosclerosis Risk in Communities Study cohort (n = 1,020). Top panel: “Red” complex. Middle panel: “Orange complex”. Bottom panel: <em>Aggregatibacter actinomycetemcomitans</em>.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2-0022034512447951" co-legend-rid="lgnd_fig2-0022034512447951"><a href="/pmc/articles/PMC3383103/figure/fig2-0022034512447951/" target="figure" rid-figpopup="fig2-0022034512447951" rid-ob="ob-fig2-0022034512447951"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140494707569456" class="figure"><a href="/pmc/articles/PMC3383103/figure/fig2-0022034512447951/" target="figure" rid-figpopup="fig2-0022034512447951" rid-ob="ob-fig2-0022034512447951"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3383103_10.1177_0022034512447951-fig2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 10.1177_0022034512447951-fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3383103/bin/10.1177_0022034512447951-fig2.jpg"></a></div><div id="largeobj_idm140494707569456" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3383103/figure/fig2-0022034512447951/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2-0022034512447951"><div><a class="figpopup" href="/pmc/articles/PMC3383103/figure/fig2-0022034512447951/" target="figure" rid-figpopup="fig2-0022034512447951" rid-ob="ob-fig2-0022034512447951">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p12">Visualization of 6 loci that were marked by suggestive genome-wide association evidence (p &lt; 5 × 10<sup>−6</sup>) for “high” colonization with “red” complex, “orange” complex, and <em>Aggregatibacter actinomycetemcomitans</em> among the participants of the Dental Atherosclerosis Risk in Communities Study cohort (n = 1,020). The vertical axis corresponds to each marker’s associated –log10 p value. The overlaid recombination rate plot and the pairwise linkage disequilibrium values with index SNPs were calculated based on HapMap II – CEU. Rs10043775, rs16924631, rs10010758, and rs12032672 provided association signals for “red” complex colonization; rs1616122 with “orange” complex; and rs11800854 with <em>Aggregatibacter actinomycetemcomitans</em> “high” colonization.</p></div></div></div><p id="__p13">The strongest signal with regard to “red” complex colonization (<a href="/pmc/articles/PMC3383103/table/table2-0022034512447951/" target="table" class="fig-table-link figpopup" rid-figpopup="table2-0022034512447951" rid-ob="ob-table2-0022034512447951" co-legend-rid=""><span>Table 2</span></a>) was produced by rs11800854 in the 1q42 locus [p = 2.8 × 10<sup>−7</sup>; OR = 12.3 (95% CI = 3.7, 41.3); MAF in HapMap-CEU (MAF-CEU): 0.067] in the promoter region (30 Kb upstream) of <em>KCNK1</em> and adjacent to <em>KIAA1804</em>. The common [G] allele showed 3% enrichment among “high colonization cases” for both “red” complex and <em>Aa</em>. Another locus in chromosome 1p22 was marked by rs12032672 (p = 9.6 × 10<sup>−7</sup>), ~500 Kb upstream of <em>PKN2</em>. Rs10043775, in linkage disequilibrium (LD) with multiple markers in the 5q33 locus, represents a missense change in the <em>FBXO38</em> gene (resulting in [Pro]→[Arg] substitution, predicted as “benign” according to PolyPhen-2) and provided the strongest signal in that locus (p = 2.4 × 10<sup>−6</sup>), also adjacent to <em>HTR4</em>. A high LD area in 9p24 including the <em>UHRF2, GLDC, TPD52L3</em>, and <em>IL33</em> genes is marked by rs16924631 [intronic to <em>UHRF2</em>; p = 3.2 × 10<sup>−6</sup>; OR = 2.29 (95% CI = 1.61, 3.24); MAF-CEU: 0.275], of which the [C] risk allele showed almost 10% enrichment among “cases”. An intronic variant (rs10010758) of the <em>TBC1D1</em> gene, adjacent (24 Kb) to <em>PTTG2</em>, provided the strongest signal in the 4p14 locus [p = 3.7 × 10<sup>−6</sup>; OR = 1.91 (95% CI = 1.25, 2.21)]. The exploratory analysis for <em>Pg</em> “high” colonization revealed 3 loci with p &lt; 5 × 10<sup>−6</sup>, including <em>OTOF, C2Orf70, CIB4, DAB2IP, TTLL11</em>, and <em>AKNRD3</em> (Appendix Table 3).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="table2-0022034512447951"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p14">Genome-wide Association Analysis Results of the High Colonization Traits (highest quintile of the distribution <em>vs</em>. the other four; quantified with DNA-DNA “checkerboard” hybridization) for “Red” Complex, “Orange” Complex, and <em>Aggregatibacter actinomycetemcomitans</em>, among the White Participants of the Dental ARIC Study (n = 1,020)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140494707079040" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Locus</th><th rowspan="1" colspan="1" align="center">SNP</th><th rowspan="1" colspan="1" align="center">Position Build36</th><th rowspan="1" colspan="1" align="center">Strand</th><th rowspan="1" colspan="1" align="center">Imputed</th><th rowspan="1" colspan="1" align="center">mja<sup><sup>1</sup></sup>
</th><th rowspan="1" colspan="1" align="center">MAF<sup><sup>2</sup></sup> (HapMap II-CEU)</th><th rowspan="1" colspan="1" align="center">Closest Gene(s) and Position or Distance</th><th rowspan="1" colspan="1" align="center">Risk Allele Frequency Low/High Colonization</th><th rowspan="1" colspan="1" align="center">P value</th><th rowspan="1" colspan="1" align="center">Odds Ratio(95% CI)</th></tr></thead><tbody><tr><td colspan="11" rowspan="1">“Red” complex<hr>
</td></tr><tr><td rowspan="1" colspan="1">1q42</td><td rowspan="1" colspan="1">rs11800854<sup><sup>3</sup></sup>
</td><td rowspan="1" colspan="1">231786607</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">[A] 0.068</td><td rowspan="1" colspan="1">
<em>KCNK1</em> (30Kb)</td><td rowspan="1" colspan="1">[G] 0.947/0.978</td><td rowspan="1" colspan="1">2.8 × 10<sup>−7</sup>
</td><td rowspan="1" colspan="1">12.3 (3.7, 41.3)</td></tr><tr><td rowspan="1" colspan="1">1p22</td><td rowspan="1" colspan="1">rs12032672</td><td rowspan="1" colspan="1">88398224</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">[C] 0.350</td><td rowspan="1" colspan="1">
<em>PKN2</em> (524Kb)</td><td rowspan="1" colspan="1">[C] 0.332/0.446</td><td rowspan="1" colspan="1">9.6 × 10<sup>−7</sup>
</td><td rowspan="1" colspan="1">1.99 (1.50, 2.62)</td></tr><tr><td rowspan="1" colspan="1">5q33</td><td rowspan="1" colspan="1">rs10043775<sup><sup>4</sup></sup>
</td><td rowspan="1" colspan="1">147785313</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">[C] 0.274</td><td rowspan="1" colspan="1">
<em>FBXO38</em> (missense change)<sup><sup>6</sup></sup>/<em>HTR4</em> (19Kb)</td><td rowspan="1" colspan="1">[T] 0.703/0.791</td><td rowspan="1" colspan="1">2.4 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">2.06 (1.51, 2.83)</td></tr><tr><td rowspan="1" colspan="1">9p24</td><td rowspan="1" colspan="1">rs16924631<sup><sup>5</sup></sup>
</td><td rowspan="1" colspan="1">6476308</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">no</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">[C] 0.142</td><td rowspan="1" colspan="1">
<em>UHRF2</em> (non-coding transcript variant) /<em>GLDC</em> (46kb)</td><td rowspan="1" colspan="1">[C] 0.122/0.219</td><td rowspan="1" colspan="1">3.2 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">2.29 (1.61, 3.24)</td></tr><tr><td rowspan="1" colspan="1">4p14</td><td rowspan="1" colspan="1">rs10010758</td><td rowspan="1" colspan="1">37614913</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">no</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">[C] 0.275</td><td rowspan="1" colspan="1">
<em>TBC1D1</em>(intron variant)/ <em>PTTG2</em> (24Kb)<sup><sup>7</sup></sup>
</td><td rowspan="1" colspan="1">[C] 0.291/0.384</td><td rowspan="1" colspan="1">3.7 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">1.91 (1.45, 2.51)</td></tr><tr><td colspan="11" rowspan="1">“Orange” complex<hr>
</td></tr><tr><td rowspan="1" colspan="1">6p21.1</td><td rowspan="1" colspan="1">rs1932040<sup><sup>8</sup></sup>
</td><td rowspan="1" colspan="1">45804766</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">[A] 0.142</td><td rowspan="1" colspan="1">
<em>CLIC5</em> (169Kb)/ <em>RUNX2</em> (178Kb)</td><td rowspan="1" colspan="1">[G] 0.808/0.896</td><td rowspan="1" colspan="1">1.3 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">2.47 (1.67, 3.65)</td></tr><tr><td rowspan="1" colspan="1">8q23</td><td rowspan="1" colspan="1">rs9942773<sup><sup>9</sup></sup>
</td><td rowspan="1" colspan="1">115190203</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">[C] 0.283</td><td rowspan="1" colspan="1">
<em>CSMD3</em> (672Kb)/<em>TRPS1</em> (1.3Mb)</td><td rowspan="1" colspan="1">[A] 0.703/0.803</td><td rowspan="1" colspan="1">1.9 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">2.07 (1.51, 2.82)</td></tr><tr><td rowspan="1" colspan="1">1p36.2</td><td rowspan="1" colspan="1">rs1616122</td><td rowspan="1" colspan="1">7444172</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">no</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">[C] 0.482</td><td rowspan="1" colspan="1">
<em>CAMTA1</em> (intron variant)/<em>VAMP3</em> (310Kb)</td><td rowspan="1" colspan="1">[T] 0.506/0.624</td><td rowspan="1" colspan="1">4.9 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">1.85 (1.41, 2.42)</td></tr><tr><td colspan="11" rowspan="1">
<strong><em>Aggregatibacter actinomycetemcomitans</em></strong>
<hr>
</td></tr><tr><td rowspan="1" colspan="1">14q24</td><td rowspan="1" colspan="1">rs11621969</td><td rowspan="1" colspan="1">74883781</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">[C] 0.167</td><td rowspan="1" colspan="1">
<em>FOS</em> (65Kb)/<em>JDP2</em> (85Kb)</td><td rowspan="1" colspan="1">[T] 0.789/0.885</td><td rowspan="1" colspan="1">9.4 × 10<sup>−7</sup>
</td><td rowspan="1" colspan="1">2.46 (1.68, 3.62)</td></tr><tr><td rowspan="1" colspan="1">5q35</td><td rowspan="1" colspan="1">rs6885116</td><td rowspan="1" colspan="1">167576123</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">[G] 0.084</td><td rowspan="1" colspan="1">
<em>ODZ2</em> (intron variant)/ <em>WWC1</em> (76Kb)</td><td rowspan="1" colspan="1">[G] 0.078/0.169</td><td rowspan="1" colspan="1">1.4 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">2.57 (1.76, 3.74)</td></tr><tr><td rowspan="1" colspan="1">10q23</td><td rowspan="1" colspan="1">rs1970525<sup><sup>10</sup></sup>
</td><td rowspan="1" colspan="1">87624904</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">[C] 0.045</td><td rowspan="1" colspan="1">
<em>GRID1</em> (NMD<sup><sup>12</sup></sup> transcript variant) /<em>MI346</em>/ <em>WAPAL</em> (560Kb)</td><td rowspan="1" colspan="1">[G] 0.054/0.118</td><td rowspan="1" colspan="1">3.8 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">2.89 (1.85, 4.52)</td></tr><tr><td rowspan="1" colspan="1">1q42</td><td rowspan="1" colspan="1">rs11800854<sup><sup>11</sup></sup>
</td><td rowspan="1" colspan="1">231786607</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">[A] 0.067</td><td rowspan="1" colspan="1">
<em>KCNK1</em> (30Kb)/ <em>KIAA1804</em> (199Kb)</td><td rowspan="1" colspan="1">[G] 0.947/0.978</td><td rowspan="1" colspan="1">4.0 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">8.12 (2.73, 24.11)</td></tr><tr><td rowspan="1" colspan="1">2q31</td><td rowspan="1" colspan="1">rs9287989</td><td rowspan="1" colspan="1">176425987</td><td rowspan="1" colspan="1" align="center">+</td><td rowspan="1" colspan="1">yes</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">[T] 0.433</td><td rowspan="1" colspan="1">
<em>KIAA1715</em> (73Kb)/ <em>EVX2</em> (227Kb)/ <em>EXTLP2</em>
<sup><sup>13</sup></sup> (10Kb)</td><td rowspan="1" colspan="1">[C] 0.484/0.605</td><td rowspan="1" colspan="1">4.4 × 10<sup>−6</sup>
</td><td rowspan="1" colspan="1">1.80 (1.39, 2.33)</td></tr></tbody></table></div><div id="largeobj_idm140494707079040" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3383103/table/table2-0022034512447951/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="table-fn1-0022034512447951"><p id="__p15" class="p p-first-last">Single-nucleotide polymorphisms (SNPs) with minor allele frequency (MAF-HapMap II CEU) of ≥ 5% and associated p &lt; 5 × 10<sup>−6</sup>. The SNP with the lowest p value <em>per</em> locus is presented; additional prioritized SNPs in each locus are presented below, along with corresponding R<sup>2</sup> (based on 1000 genomes pilot 1 release) with the top SNPs.</p></div><div id="table-fn2-0022034512447951"><sup>1</sup>Major allele.</div><div id="table-fn3-0022034512447951"><sup>2</sup>Minor allele frequency.</div><div id="table-fn4-0022034512447951"><sup>3</sup>Additional SNPs in locus with p &lt; 5 × 10<sup>−6</sup>: rs6682365 (r<sup>2</sup> = 1.00).</div><div id="table-fn5-0022034512447951"><sup>4</sup>Additional SNP in locus with p &lt; 5 × 10<sup>−6</sup>: rs10068216 (r<sup>2</sup> = 1.00), rs10072051 (r<sup>2</sup> = 1.00), rs17108251 (r<sup>2</sup> = 1.00), rs10044061 (r<sup>2</sup> = 1.00), rs4349707 (r<sup>2</sup> = 1.00), rs10477376 (r<sup>2</sup> = 1.00), rs9325095 (r<sup>2</sup> = 1.00), rs10041283 (r<sup>2</sup> = 1.00), rs9325097 (r<sup>2</sup> = 0.87), rs3734120 (r<sup>2</sup> = 1.00), rs4574533 (r<sup>2</sup> = 0.87), rs4274967 (r<sup>2</sup> = 1.00), rs4274968 (r<sup>2</sup> = 1.00), rs6884076 (r<sup>2</sup> = 1.00), rs9325098 (r<sup>2</sup> = 1.00).</div><div id="table-fn6-0022034512447951"><sup>5</sup>Additional SNP in locus with p &lt; 5 × 10<sup>−6</sup>: rs11795355 (r<sup>2</sup> = 1.00), rs7876000 (r<sup>2</sup> = 1.00), rs10975603 (r<sup>2</sup> = 0.93), rs16924626 (r<sup>2</sup> = 0.93), rs16924624 (r<sup>2</sup> = 1.00), rs10975605 (r<sup>2</sup> = 0.93), rs10115883 (r<sup>2</sup> = 0.93), rs10122116 (r<sup>2</sup> = 0.93).</div><div id="table-fn7-0022034512447951"><sup>6</sup>T&gt;C – Ser&gt;Pro, 35b from the exon boundary.</div><div id="table-fn8-0022034512447951"><sup>7</sup>r<sup>2</sup> = 0.29 with rs6811863, which is a missense change in <em>PTTG2</em>: G&gt;C – [Arg]→[Pro].</div><div id="table-fn9-0022034512447951"><sup>8</sup>Additional SNPs in locus with p &lt; 5 × 10<sup>−6</sup>: rs12525547(r<sup>2</sup> = 0.93), rs9349326 (r<sup>2</sup> = 0.93), rs16873698(r<sup>2</sup> = 0.93).</div><div id="table-fn10-0022034512447951"><sup>9</sup>Additional SNP in locus with p &lt; 5 × 10<sup>−6</sup>: rs10089040 (r<sup>2</sup> = 1.00), rs9942776 (r<sup>2</sup> = 1.00), rs10086149 (r<sup>2</sup> = 1.00), rs7845243 (r<sup>2</sup> = 0.87), rs10105817 (r<sup>2</sup> = 1.00), rs7006291 (r<sup>2</sup> = 1.00), rs11779159 (r<sup>2</sup> = 1.00), rs11783996 (r<sup>2</sup> = 1.00), rs10098056 (r<sup>2</sup> = 1.00), rs7018200 (r<sup>2</sup> = 0.92).</div><div id="table-fn11-0022034512447951"><sup>10</sup>Additional SNPs in locus with p &lt; 5 × 10<sup>−6</sup>: rs4325261 (r<sup>2</sup> = 1.00).</div><div id="table-fn12-0022034512447951"><sup>11</sup>Additional SNP in locus with p &lt; 5 × 10<sup>−6</sup>: rs6682365 (r<sup>2</sup> = 1.00).</div><div id="table-fn13-0022034512447951"><sup>12</sup>Nonsense-mediated decay.</div><div id="table-fn14-0022034512447951"><sup>13</sup>
<em>EXTLP2</em> is a pseudogene.</div></div></div><p id="__p29">The common allele of rs1932040 showed 9% enrichment and provided the strongest association signal with “high orange” bacterial colonization [p = 1.3 × 10<sup>−6</sup>, OR = 2.47 (95% CI = 1.67, 3.65)], marking an intergenic area between <em>RUNX2</em> and <em>CLIC5</em> on the 6p21.1 locus. A low recombination area on 8q23, adjacent to <em>TSPS1</em> (1.3 Mb) and <em>CSMD3</em> (672 Kb), was marked by multiple alleles, of which rs9942773 provided the strongest signal (p = 1.9 × 10<sup>−6</sup>) and 10% enrichment among “cases”. A variant intronic to <em>CAMTA1</em> [rs1616122; p = 4.9 × 10<sup>−6</sup>; OR = 1.85 (95% CI = 1.41, 2.42)] marked the 1p36.2 locus.</p><p id="__p30">The 1q42 locus that was identified for “red” bacteria also provided the third strongest association signal for <em>Aa</em> [rs11800854; p = 4.0 × 10<sup>−6</sup>; OR = 8.12 (95% CI = 2.73, 24.11)]. The common [T] allele of rs11621969 was also nominally associated with high <em>Aa</em> colonization (p = 9.4 × 10<sup>−7</sup>) and was adjacent to <em>FOS</em> and <em>JPD2</em> in 14q24. The rare [G] allele of rs1970525 was more than twice as prevalent among “cases” (0.118 <em>vs</em>. 0.054 among “non-cases”), provided the strongest signal in the 10q23 locus [p = 3.8 × 10<sup>−6</sup>; OR = 2.89 (95% CI = 1.85, 4.52); MAF-CEU: 0.045)], and represents a nonsense-mediated decay transcript variant in the <em>GRID1</em> gene. Rs9287989 is adjacent to <em>KIAA1715</em> (30 Kb) and <em>EVX2</em> (227 Kb) and marked the 2q31 locus (p = 4.4 × 10<sup>−6</sup>). An intronic variant of <em>ODZ2</em> (rs6885116) provided the strongest signal in the locus 5q35 [p = 1.4 × 10<sup>−6</sup>; OR = 2.57 (95% CI = 1.76, 3.74); MAF-CEU: 0.084], showing 9% enrichment among “cases”.</p><p id="__p31" class="p p-last">The examination of our prioritized SNPs in the AA sample revealed that although no p value met conventional replication significance criteria (p &lt; 0.05), all 8 of the effect estimates for the “red” and “orange” complex bacterial colonization traits (Appendix Table 4) had the same direction in the white and AA samples. In contrast, the results for <em>Aa</em> colonization were not consistent between the two samples. None of the prioritized SNPs were associated with periodontitis diagnosis at a nominal significance level (p &lt; 0.05). Further, in Appendix Table 5 we present the effect estimates and associated p value of polymorphisms that were previously reported as associated with periodontal pathogen colonization profiles in studies by Socransky (<a href="#bibr27-0022034512447951" rid="bibr27-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278363">Socransky <em>et al</em>., 2000</a>) and Nibali (<a href="#bibr18-0022034512447951" rid="bibr18-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278361">Nibali <em>et al</em>., 2010</a>, <a href="#bibr19-0022034512447951" rid="bibr19-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278365">2011</a>). Of those, only 2 <em>IL6</em> promoter polymorphisms (rs1800795 and rs1800796) were found to be significantly associated (p &lt; 0.05) with high “red complex” colonization.</p></div><div id="section4-0022034512447951" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="section4-0022034512447951title" style="text-transform:uppercase">Discussion</h2><p id="__p32" class="p p-first">This study is the first report of a genome-wide association analysis investigating risk loci for colonization with pathogenic periodontal bacteria. Although limited by the sample size, this investigation explores a novel phenotype and benefits from a comprehensive quantitative phenotypical characterization. In this investigation, we did not carry out a comprehensive investigation of genetic risk loci for chronic periodontitis, but examined bacterial colonization traits that are implicated in its pathogenesis and progression. We support that an increased understanding of the genetic underpinning of interactions between the host and exogenous or symbiotic bacterial communities has the potential to advance our knowledge not only of periodontitis, but also of other chronic inflammatory and microbiome-related diseases. Several of the risk loci identified in this study may offer promising leads for further exploration and mechanistic studies. For example, a recent experimental study showed that <em>NLRP6</em> knockout mice had altered gut flora and were more susceptible to colitis (<a href="#bibr11-0022034512447951" rid="bibr11-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278355">Elinav <em>et al</em>., 2011</a>).</p><p id="__p33">Acknowledging the limitations of GWAS as a discovery tool, the reported SNP estimates may be overestimates of the true genetic effects (<a href="#bibr13-0022034512447951" rid="bibr13-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278356">Göring <em>et al</em>., 2001</a>), whereas many true association signals with modest effects may not have been prioritized due to insufficient power. In contrast, the “unbiased” and comprehensive consideration of approximately 2.5 million genetic markers is an improvement over candidate-gene studies, which are typically motivated by prior information and biological plausibility (<a href="#bibr15-0022034512447951" rid="bibr15-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278368">Hirschhorn and Daly, 2005</a>). Because it may be difficult to conduct replication studies or a larger GWAS in the near future, we examined our reported associations among a second small sample of AA individuals. While this approach provided compelling evidence for the “red” and “orange” complex results, this was not the case for the <em>Aa</em> colonization results. This may be due to differing genetic susceptibility factors for <em>Aa</em> colonization between the two racial groups, a fact that is supported by well-documented differences in bacterial colonization profiles between ethnic groups (<a href="#bibr7-0022034512447951" rid="bibr7-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278345">Beck <em>et al</em>., 1992</a>; <a href="#bibr14-0022034512447951" rid="bibr14-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278354">Haffajee <em>et al</em>., 2005</a>). Our finding of no association of the identified SNPs with CP diagnosis is in support of both the multifactorial disease pathogenesis and the examination of bacterial colonization as a distinct trait. Although we did not collect information on this domain, it cannot be excluded that some participants might have received anti-inflammatory or antibiotic therapy prior to the oral examination and microbial plaque collection. It is possible that this could have influenced the oral bacterial colonization profile of some participants. However, it must be noted that the requirement of antibiotic prophylaxis prior to dental treatment was a study exclusion criterion.</p><p id="__p34" class="p p-last">Upon replication or validation, these findings have the potential to unveil pathways and mechanisms that direct the host’s symbiosis with healthy microflora that, if altered, may predispose for states of disease (<a href="#bibr9-0022034512447951" rid="bibr9-0022034512447951" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278352">Darveau, 2010</a>). The consideration of specific microbiota as a distinct exposure in investigations of periodontal, oral, and systemic health is consistent with the paradigm of “periodontal medicine” and may provide novel insight into the ‘oral-systemic diseases’ connection. Although the prevention and treatment of periodontitis are obvious goals, the links of CP with other systemic conditions, and the “common theme” of pathogenic ecological shift in other diseases, provide opportunities for even greater impact.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title" style="text-transform:uppercase">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Appendix:</strong>

<div class="sup-box half_rhythm" id="idm140494778953408"><a href="/pmc/articles/PMC3383103/bin/supp_91_7-suppl_S21__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140494702599776" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140494702599776title" style="text-transform:uppercase">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140494702599648"></p><p id="__p35" class="p p-first-last">The Atherosclerosis Risk in Communities Study is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute contracts (<span>HHSN268201100005C</span>;, <span>HHSN268201100006C</span>;, <span>HHSN26820 1100007C</span>;, <span>HHSN268201100008C</span>;, <span>HHSN268201100009C</span>;, <span>HHSN2682 01100010C</span>;, <span>HHSN268201100011C</span>;, and <span>HSN268201100012C</span>;), <span>R01HL087641</span>;, <span>R01HL59367</span>;, and <span>R01HL086694</span>; National Human Genome Research Institute contract <span>U01HG004402</span>; National Institutes of Health
 contract <span>HHSN268200625226C</span>; National Institute of Environmental Health Sciences grant <span>P30ES010126</span>; and National Institute of Dental and Craniofacial Research
 grants <span>R01DE11551</span>;, <span>R01DE021418</span>;. The authors thank the staff and participants of the ARIC study for their important contributions. Infrastructure was partly supported by Grant Number <span>UL1RR025005</span>, a component of the National Institutes of Health
 and NIH Roadmap for Medical Research.</p><p></p><p class="fn sec" id="idm140494708138336"> </p><p id="__p36" class="p p-first-last">The authors declare no potential conflicts of interest with respect to the authorship and/or publication of this article.</p><p></p><p class="fn sec" id="idm140494708137440"></p><p id="__p37" class="p p-first-last">A supplemental appendix to this article is published electronically only at <a href="http://jdr.sagepub.com/supplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://jdr.sagepub.com/supplemental</a>.</p><p></p></div></div><div id="idm140494707072752" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140494707072752title" style="text-transform:uppercase">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="first-line-outdent"><li id="bibr1-0022034512447951"><span class="mixed-citation">
Ahn J, Chen CY, Hayes RB. (2012). <span class="ref-title">Oral microbiome and oral and gastrointestinal cancer risk</span>. <span class="ref-journal">Cancer Causes Control</span>
<span class="ref-vol">23</span>:399-404 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3767140/">PMC free article</a>]</span> [<a href="/pubmed/22271008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Causes+Control&amp;title=Oral+microbiome+and+oral+and+gastrointestinal+cancer+risk&amp;author=J+Ahn&amp;author=CY+Chen&amp;author=RB+Hayes&amp;volume=23&amp;publication_year=2012&amp;pages=399-404&amp;pmid=22271008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr2-0022034512447951"><span class="mixed-citation">
The ARIC investigators (1989). <span class="ref-title">The Atherosclerosis Risk in Communities (ARIC) Study: design and objectives</span>. <span class="ref-journal">Am J Epidemiol</span>
<span class="ref-vol">129</span>:687-702 [<a href="/pubmed/2646917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=The+Atherosclerosis+Risk+in+Communities+(ARIC)+Study:+design+and+objectives&amp;volume=129&amp;publication_year=1989&amp;pages=687-702&amp;pmid=2646917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr3-0022034512447951"><span class="mixed-citation">
Aulchenko YS, Struchalin MV, van Duijn CM. (2010). <span class="ref-title">ProbABEL package for genome-wide association analysis of imputed data</span>. <span class="ref-journal">BMC Bioinformatics</span>
<span class="ref-vol">11</span>:134. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2846909/">PMC free article</a>]</span> [<a href="/pubmed/20233392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=ProbABEL+package+for+genome-wide+association+analysis+of+imputed+data&amp;author=YS+Aulchenko&amp;author=MV+Struchalin&amp;author=CM+van+Duijn&amp;volume=11&amp;publication_year=2010&amp;pages=134&amp;pmid=20233392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr4-0022034512447951"><span class="mixed-citation">
Avila M, Ojcius DM, Yilmaz O. (2009). <span class="ref-title">The oral microbiota: living with a permanent guest</span>. <span class="ref-journal">DNA Cell Biol</span>
<span class="ref-vol">28</span>:405-411 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2768665/">PMC free article</a>]</span> [<a href="/pubmed/19485767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Cell+Biol&amp;title=The+oral+microbiota:+living+with+a+permanent+guest&amp;author=M+Avila&amp;author=DM+Ojcius&amp;author=O+Yilmaz&amp;volume=28&amp;publication_year=2009&amp;pages=405-411&amp;pmid=19485767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr5-0022034512447951"><span class="mixed-citation">
Badger JH, Ng PC, Venter JC. (2010). <span class="ref-title">The human genome, microbiomes, and disease</span>. In: Metagenomics of the human body. Nelson KE, editor. New York, NY, USA: Springer, pp. 1-14 <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=
+Badger+JH+Ng+PC+Venter+JC++(+2010+).++The+human+genome,+microbiomes,+and+disease+.++In+:++Metagenomics+of+the+human+body.+Nelson+KE,+editor.+New+York,+NY,+USA+:+Springer,+pp.++1+-+14+" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr6-0022034512447951"><span class="mixed-citation">
Barrett JC, Fry B, Maller J, Daly MJ. (2005). <span class="ref-title">Haploview: analysis and visualization of LD and haplotype maps</span>. <span class="ref-journal">Bioinformatics</span>
<span class="ref-vol">21</span>:263-265 [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr7-0022034512447951"><span class="mixed-citation">
Beck JD, Koch GG, Zambon JJ, Genco RJ, Tudor GE. (1992). <span class="ref-title">Evaluation of oral bacteria as risk indicators for periodontitis in older adults</span>. <span class="ref-journal">J Periodontol</span>
<span class="ref-vol">63</span>:93-99 [<a href="/pubmed/1313107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Periodontol&amp;title=Evaluation+of+oral+bacteria+as+risk+indicators+for+periodontitis+in+older+adults&amp;author=JD+Beck&amp;author=GG+Koch&amp;author=JJ+Zambon&amp;author=RJ+Genco&amp;author=GE+Tudor&amp;volume=63&amp;publication_year=1992&amp;pages=93-99&amp;pmid=1313107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr8-0022034512447951"><span class="mixed-citation">
Beck JD, Elter JR, Heiss G, Couper D, Mauriello SM, Offenbacher S. (2001). <span class="ref-title">Relationship of periodontal disease to carotid artery intima-media wall thickness: the atherosclerosis risk in communities (ARIC) study</span>. <span class="ref-journal">Arterioscler Thromb Vasc Biol</span>
<span class="ref-vol">21</span>:1816-1822 [<a href="/pubmed/11701471" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=Relationship+of+periodontal+disease+to+carotid+artery+intima-media+wall+thickness:+the+atherosclerosis+risk+in+communities+(ARIC)+study&amp;author=JD+Beck&amp;author=JR+Elter&amp;author=G+Heiss&amp;author=D+Couper&amp;author=SM+Mauriello&amp;volume=21&amp;publication_year=2001&amp;pages=1816-1822&amp;pmid=11701471&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr9-0022034512447951"><span class="mixed-citation">
Darveau RP. (2010). <span class="ref-title">Periodontitis: a polymicrobial disruption of host homeostasis</span>. <span class="ref-journal">Nat Rev Microbiol</span>
<span class="ref-vol">8</span>:481-490 [<a href="/pubmed/20514045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Microbiol&amp;title=Periodontitis:+a+polymicrobial+disruption+of+host+homeostasis&amp;author=RP+Darveau&amp;volume=8&amp;publication_year=2010&amp;pages=481-490&amp;pmid=20514045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr10-0022034512447951"><span class="mixed-citation">
Desvarieux M, Demmer RT, Rundek T, Boden-Albala B, Jacobs DR, Jr, Sacco RL, et al.  (2005). <span class="ref-title">Periodontal microbiota and carotid intima-media thickness: the Oral Infections and Vascular Disease Epidemiology Study (INVEST)</span>. <span class="ref-journal">Circulation</span>
<span class="ref-vol">111</span>:576-582 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2812915/">PMC free article</a>]</span> [<a href="/pubmed/15699278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Periodontal+microbiota+and+carotid+intima-media+thickness:+the+Oral+Infections+and+Vascular+Disease+Epidemiology+Study+(INVEST)&amp;author=M+Desvarieux&amp;author=RT+Demmer&amp;author=T+Rundek&amp;author=B+Boden-Albala&amp;author=DR+Jacobs&amp;volume=111&amp;publication_year=2005&amp;pages=576-582&amp;pmid=15699278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr11-0022034512447951"><span class="mixed-citation">
Elinav E, Strowig T, Kau AL, Henao-Mejia J, Thaiss CA, Booth CJ, et al.  (2011). <span class="ref-title">NLRP6 inflammasome regulates colonic microbial ecology and risk for colitis</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">145</span>:745-757 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3140910/">PMC free article</a>]</span> [<a href="/pubmed/21565393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=NLRP6+inflammasome+regulates+colonic+microbial+ecology+and+risk+for+colitis&amp;author=E+Elinav&amp;author=T+Strowig&amp;author=AL+Kau&amp;author=J+Henao-Mejia&amp;author=CA+Thaiss&amp;volume=145&amp;publication_year=2011&amp;pages=745-757&amp;pmid=21565393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr12-0022034512447951"><span class="mixed-citation">
Ge D, Zhang K, Need AC, Martin O, Fellay J, Urban TJ, et al.  (2008). <span class="ref-title">WGAViewer: software for genomic annotation of whole genome association studies</span>. <span class="ref-journal">Genome Res</span>
<span class="ref-vol">18</span>:640-643 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2279251/">PMC free article</a>]</span> [<a href="/pubmed/18256235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=WGAViewer:+software+for+genomic+annotation+of+whole+genome+association+studies&amp;author=D+Ge&amp;author=K+Zhang&amp;author=AC+Need&amp;author=O+Martin&amp;author=J+Fellay&amp;volume=18&amp;publication_year=2008&amp;pages=640-643&amp;pmid=18256235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr13-0022034512447951"><span class="mixed-citation">
Göring HH, Terwilliger JD, Blangero J. (2001). <span class="ref-title">Large upward bias in estimation of locus-specific effects from genomewide scans</span>. <span class="ref-journal">Am J Hum Genet</span>
<span class="ref-vol">69</span>:1357-1369 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235546/">PMC free article</a>]</span> [<a href="/pubmed/11593451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Large+upward+bias+in+estimation+of+locus-specific+effects+from+genomewide+scans&amp;author=HH+Göring&amp;author=JD+Terwilliger&amp;author=J+Blangero&amp;volume=69&amp;publication_year=2001&amp;pages=1357-1369&amp;pmid=11593451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr14-0022034512447951"><span class="mixed-citation">
Haffajee AD, Japlit M, Bogren A, Kent RL, Jr, Goodson JM, Socransky SS. (2005). <span class="ref-title">Differences in the subgingival microbiota of Swedish and USA subjects who were periodontally healthy or exhibited minimal periodontal disease</span>. <span class="ref-journal">J Clin Periodontol</span>
<span class="ref-vol">32</span>:33-39 [<a href="/pubmed/15642056" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Periodontol&amp;title=Differences+in+the+subgingival+microbiota+of+Swedish+and+USA+subjects+who+were+periodontally+healthy+or+exhibited+minimal+periodontal+disease&amp;author=AD+Haffajee&amp;author=M+Japlit&amp;author=A+Bogren&amp;author=RL+Kent&amp;author=JM+Goodson&amp;volume=32&amp;publication_year=2005&amp;pages=33-39&amp;pmid=15642056&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr15-0022034512447951"><span class="mixed-citation">
Hirschhorn JN, Daly MJ. (2005). <span class="ref-title">Genome-wide association studies for common diseases and complex traits</span>. <span class="ref-journal">Nat Rev Genet</span>
<span class="ref-vol">6</span>:95-108 [<a href="/pubmed/15716906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+for+common+diseases+and+complex+traits&amp;author=JN+Hirschhorn&amp;author=MJ+Daly&amp;volume=6&amp;publication_year=2005&amp;pages=95-108&amp;pmid=15716906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr16-0022034512447951"><span class="mixed-citation">
Keijser BJ, Zaura E, Huse SM, van der Vossen JM, Schuren FH, Montijn RC, et al.  (2008). <span class="ref-title">Pyrosequencing analysis of the oral microflora of healthy adults</span>. <span class="ref-journal">J Dent Res</span>
<span class="ref-vol">87</span>:1016-1,020 [<a href="/pubmed/18946007" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Dent+Res&amp;title=Pyrosequencing+analysis+of+the+oral+microflora+of+healthy+adults&amp;author=BJ+Keijser&amp;author=E+Zaura&amp;author=SM+Huse&amp;author=JM+van+der+Vossen&amp;author=FH+Schuren&amp;volume=87&amp;publication_year=2008&amp;pages=1016-1&amp;pmid=18946007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr17-0022034512447951"><span class="mixed-citation">
Nibali L, Ready DR, Parkar M, Brett PM, Wilson M, Tonetti MS, et al.  (2007). <span class="ref-title">Gene polymorphisms and the prevalence of key periodontal pathogens</span>. <span class="ref-journal">J Dent Res</span>
<span class="ref-vol">86</span>:416-420 [<a href="/pubmed/17452560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Dent+Res&amp;title=Gene+polymorphisms+and+the+prevalence+of+key+periodontal+pathogens&amp;author=L+Nibali&amp;author=DR+Ready&amp;author=M+Parkar&amp;author=PM+Brett&amp;author=M+Wilson&amp;volume=86&amp;publication_year=2007&amp;pages=416-420&amp;pmid=17452560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr18-0022034512447951"><span class="mixed-citation">
Nibali L, Donos N, Farrell S, Ready D, Pratten J, Tu YK, et al.  (2010). <span class="ref-title">Association between interleukin-6 -174 polymorphism and <em>Aggregatibacter actinomycetemcomitans</em> in chronic periodontitis</span>. <span class="ref-journal">J Periodontol</span>
<span class="ref-vol">81</span>:1814-1819 [<a href="/pubmed/20681812" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Periodontol&amp;title=Association+between+interleukin-6+-174+polymorphism+and+Aggregatibacter+actinomycetemcomitans+in+chronic+periodontitis&amp;author=L+Nibali&amp;author=N+Donos&amp;author=S+Farrell&amp;author=D+Ready&amp;author=J+Pratten&amp;volume=81&amp;publication_year=2010&amp;pages=1814-1819&amp;pmid=20681812&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr19-0022034512447951"><span class="mixed-citation">
Nibali L, Madden I, Franch Chillida F, Heitz-Mayfield L, Brett P, Donos N. (2011). <span class="ref-title">IL6-174 genotype associated with <em>Aggregatibacter actinomycetemcomitans</em> in Indians</span>. <span class="ref-journal">Oral Dis</span>
<span class="ref-vol">17</span>:232-237 [<a href="/pubmed/20860759" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oral+Dis&amp;title=IL6-174+genotype+associated+with+Aggregatibacter+actinomycetemcomitans+in+Indians&amp;author=L+Nibali&amp;author=I+Madden&amp;author=F+Franch+Chillida&amp;author=L+Heitz-Mayfield&amp;author=P+Brett&amp;volume=17&amp;publication_year=2011&amp;pages=232-237&amp;pmid=20860759&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr20-0022034512447951"><span class="mixed-citation">
Offenbacher S, Barros SP, Singer RE, Moss K, Williams RC, Beck JD. (2007). <span class="ref-title">Periodontal disease at the biofilm-gingival interface</span>. <span class="ref-journal">J Periodontol</span>
<span class="ref-vol">78</span>:1911-1925 [<a href="/pubmed/18062113" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Periodontol&amp;title=Periodontal+disease+at+the+biofilm-gingival+interface&amp;author=S+Offenbacher&amp;author=SP+Barros&amp;author=RE+Singer&amp;author=K+Moss&amp;author=RC+Williams&amp;volume=78&amp;publication_year=2007&amp;pages=1911-1925&amp;pmid=18062113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr21-0022034512447951"><span class="mixed-citation">
Papapanou PN, Neiderud AM, Papadimitriou A, Sandros J, Dahlén G. (2000). <span class="ref-title">“Checkerboard” assessments of periodontal microbiota and serum antibody responses: a case-control study</span>. <span class="ref-journal">J Periodontol</span>
<span class="ref-vol">71</span>:885-897 [<a href="/pubmed/10914791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Periodontol&amp;title=“Checkerboard”+assessments+of+periodontal+microbiota+and+serum+antibody+responses:+a+case-control+study&amp;author=PN+Papapanou&amp;author=AM+Neiderud&amp;author=A+Papadimitriou&amp;author=J+Sandros&amp;author=G+Dahlén&amp;volume=71&amp;publication_year=2000&amp;pages=885-897&amp;pmid=10914791&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr22-0022034512447951"><span class="mixed-citation">
Paster BJ, Olsen I, Aas JA, Dewhirst FE. (2006). <span class="ref-title">The breadth of bacterial diversity in the human periodontal pocket and other oral sites</span>. <span class="ref-journal">Periodontol</span>
<span class="ref-vol">2000</span>
42: 80-87 [<a href="/pubmed/16930307" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Periodontol&amp;title=The+breadth+of+bacterial+diversity+in+the+human+periodontal+pocket+and+other+oral+sites&amp;author=BJ+Paster&amp;author=I+Olsen&amp;author=JA+Aas&amp;author=FE+Dewhirst&amp;volume=2000&amp;publication_year=2006&amp;pages=42-87&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr23-0022034512447951"><span class="mixed-citation">
Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al.  (2010). <span class="ref-title">LocusZoom: regional visualization of genome-wide association scan results</span>. <span class="ref-journal">Bioinformatics</span>
<span class="ref-vol">26</span>:2336-2337 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr24-0022034512447951"><span class="mixed-citation">
Socransky SS, Haffajee AD. (2002). <span class="ref-title">Dental biofilms: difficult therapeutic targets</span>. <span class="ref-journal">Periodontol</span>
<span class="ref-vol">2000</span>
28: 12-55 [<a href="/pubmed/12013340" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Periodontol&amp;title=Dental+biofilms:+difficult+therapeutic+targets&amp;author=SS+Socransky&amp;author=AD+Haffajee&amp;volume=2000&amp;publication_year=2002&amp;pages=28-55&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr25-0022034512447951"><span class="mixed-citation">
Socransky SS, Smith C, Martin L, Paster BJ, Dewhirst FE, Levin AE. (1994). <span class="ref-title">“Checkerboard” DNA-DNA hybridization</span>. <span class="ref-journal">Biotechniques</span>
<span class="ref-vol">17</span>:788-792 [<a href="/pubmed/7833043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechniques&amp;title=“Checkerboard”+DNA-DNA+hybridization&amp;author=SS+Socransky&amp;author=C+Smith&amp;author=L+Martin&amp;author=BJ+Paster&amp;author=FE+Dewhirst&amp;volume=17&amp;publication_year=1994&amp;pages=788-792&amp;pmid=7833043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr26-0022034512447951"><span class="mixed-citation">
Socransky SS, Haffajee AD, Cugini MA, Smith C, Kent RL., Jr (1998). <span class="ref-title">Microbial complexes in subgingival plaque</span>. <span class="ref-journal">J Clin Periodontol</span>
<span class="ref-vol">25</span>:134-144 [<a href="/pubmed/9495612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Periodontol&amp;title=Microbial+complexes+in+subgingival+plaque&amp;author=SS+Socransky&amp;author=AD+Haffajee&amp;author=MA+Cugini&amp;author=C+Smith&amp;author=RL+Kent&amp;volume=25&amp;publication_year=1998&amp;pages=134-144&amp;pmid=9495612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr27-0022034512447951"><span class="mixed-citation">
Socransky SS, Haffajee AD, Smith C, Duff GW. (2000). <span class="ref-title">Microbiological parameters associated with IL-1 gene polymorphisms in periodontitis patients</span>. <span class="ref-journal">J Clin Periodontol</span>
<span class="ref-vol">27</span>:810-818 [<a href="/pubmed/11073323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Periodontol&amp;title=Microbiological+parameters+associated+with+IL-1+gene+polymorphisms+in+periodontitis+patients&amp;author=SS+Socransky&amp;author=AD+Haffajee&amp;author=C+Smith&amp;author=GW+Duff&amp;volume=27&amp;publication_year=2000&amp;pages=810-818&amp;pmid=11073323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr28-0022034512447951"><span class="mixed-citation">
Turnbaugh PJ, Ley RE, Hamady M, Fraser-Liggett CM, Knight R, Gordon JI. (2007). <span class="ref-title">The human microbiome project</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">449</span>:804-810 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3709439/">PMC free article</a>]</span> [<a href="/pubmed/17943116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+human+microbiome+project&amp;author=PJ+Turnbaugh&amp;author=RE+Ley&amp;author=M+Hamady&amp;author=CM+Fraser-Liggett&amp;author=R+Knight&amp;volume=449&amp;publication_year=2007&amp;pages=804-810&amp;pmid=17943116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bibr29-0022034512447951"><span class="mixed-citation">
Zhang J, Sun X, Xiao L, Xie C, Xuan D, Luo G. (2011). <span class="ref-title">Gene polymorphisms and periodontitis</span>. <span class="ref-journal">Periodontol</span>
<span class="ref-vol">2000</span>
56: 102-124 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Periodontol&amp;title=Gene+polymorphisms+and+periodontitis&amp;author=J+Zhang&amp;author=X+Sun&amp;author=L+Xiao&amp;author=C+Xie&amp;author=D+Xuan&amp;volume=2000&amp;publication_year=2011&amp;pages=56-124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3383103&amp;issue-id=211514&amp;journal-id=1509&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140494707217408title">Abstract</a></li><li><a href="#section1-0022034512447951title">Introduction</a></li><li><a href="#section2-0022034512447951title">Materials &amp; Methods</a></li><li><a href="#section3-0022034512447951title">Results</a></li><li><a href="#section4-0022034512447951title">Discussion</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140494702599776title">Footnotes</a></li><li><a href="#idm140494707072752title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Journal of Dental Research</span> are provided here courtesy of <strong>International and American Associations for Dental Research</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3383103/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3383103/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3383103/pdf/10.1177_0022034512447951.pdf">PDF (2.0M)</a></li> | <li><a href="#" data-citationid="PMC3383103" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3383103%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3383103%2F&amp;text=Genome-wide%20Association%20Study%20of%20Periodontal%20Pathogen%20Colonization" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3383103%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22699663" data-aiid="3383103" data-aid="3383103" data-iid="211514" data-domainid="1509" data-domain="jdr" data-accid="PMC3383103" data-md5="7ec34c070a005efdc65df87702ebca28"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:07:35-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D87ADDD5467C100000000018F0114&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3383103%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jdr&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3383103/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>