<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3419486/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association and linkage study in the Amish detects a novel candidate late-onset Alzheimer disease gene</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3419486/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Annals of human genetics"><meta name="citation_title" content="Genome-wide association and linkage study in the Amish detects a novel candidate late-onset Alzheimer disease gene"><meta name="citation_authors" content="Anna C. Cummings, Lan Jiang, Digna R. Velez Edwards, Jacob L. McCauley, Renee Laux, Lynne L. McFarland, Denise Fuzzell, Clare Knebusch, Laura Caywood, Lori Reinhart-Mercer, Laura Nations, John R. Gilbert, Ioanna Konidari, Michael Tramontana, Michael L. Cuccaro, William K. Scott, Margaret A. Pericak-Vance, Jonathan L. Haines"><meta name="citation_date" content="September 2012"><meta name="citation_issue" content="5"><meta name="citation_volume" content="76"><meta name="citation_firstpage" content="342"><meta name="citation_doi" content="10.1111/j.1469-1809.2012.00721.x"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3419486/?report=abstract"><meta name="citation_pmid" content="22881374"><meta name="DC.Title" content="Genome-wide association and linkage study in the Amish detects a novel candidate late-onset Alzheimer disease gene"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Anna C. Cummings"><meta name="DC.Contributor" content="Lan Jiang"><meta name="DC.Contributor" content="Digna R. Velez Edwards"><meta name="DC.Contributor" content="Jacob L. McCauley"><meta name="DC.Contributor" content="Renee Laux"><meta name="DC.Contributor" content="Lynne L. McFarland"><meta name="DC.Contributor" content="Denise Fuzzell"><meta name="DC.Contributor" content="Clare Knebusch"><meta name="DC.Contributor" content="Laura Caywood"><meta name="DC.Contributor" content="Lori Reinhart-Mercer"><meta name="DC.Contributor" content="Laura Nations"><meta name="DC.Contributor" content="John R. Gilbert"><meta name="DC.Contributor" content="Ioanna Konidari"><meta name="DC.Contributor" content="Michael Tramontana"><meta name="DC.Contributor" content="Michael L. Cuccaro"><meta name="DC.Contributor" content="William K. Scott"><meta name="DC.Contributor" content="Margaret A. Pericak-Vance"><meta name="DC.Contributor" content="Jonathan L. Haines"><meta name="DC.Date" content="2012 Sep"><meta name="DC.Identifier" content="10.1111/j.1469-1809.2012.00721.x"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association and linkage study in the Amish detects a novel candidate late-onset Alzheimer disease gene"><meta property="og:type" content="article"><meta property="og:description" content="To identify novel late-onset Alzheimer disease (LOAD) risk genes, we have analyzed Amish populations of Ohio and Indiana. We performed genome-wide SNP linkage and association studies on 798 individuals (109 with LOAD). We tested association using the ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3419486/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3419486/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3419486/pdf/nihms387665.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B9065DD54A88100000000015900C7.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419486%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3419486</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Ann Hum Genet</a></span>. Author manuscript; available in PMC 2013 Sep 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22881374" target="pmc_ext" ref="reftype=publisher&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Ann Hum Genet. 2012 Sep; 76(5): 342–351. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1111%2Fj.1469-1809.2012.00721.x" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1111/j.1469-1809.2012.00721.x</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3419486</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS387665</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22881374">22881374</a></div></div></div></div><h1 class="content-title">Genome-wide association and linkage study in the Amish detects a novel candidate late-onset Alzheimer disease gene</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Cummings%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628251306912" co-class="co-affbox">Anna C. Cummings</a>,<sup>1</sup> <a href="/pubmed/?term=Jiang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628251304800" co-class="co-affbox">Lan Jiang</a>,<sup>1</sup> <a href="/pubmed/?term=Velez%20Edwards%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628291855248" co-class="co-affbox">Digna R. Velez Edwards</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=McCauley%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628291852288" co-class="co-affbox">Jacob L. McCauley</a>,<sup>3</sup> <a href="/pubmed/?term=Laux%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628251523280" co-class="co-affbox">Renee Laux</a>,<sup>1</sup> <a href="/pubmed/?term=McFarland%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628251521056" co-class="co-affbox">Lynne L. McFarland</a>,<sup>1</sup> <a href="/pubmed/?term=Fuzzell%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628323848304" co-class="co-affbox">Denise Fuzzell</a>,<sup>1</sup> <a href="/pubmed/?term=Knebusch%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628323846080" co-class="co-affbox">Clare Knebusch</a>,<sup>1</sup> <a href="/pubmed/?term=Caywood%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628237057248" co-class="co-affbox">Laura Caywood</a>,<sup>3</sup> <a href="/pubmed/?term=Reinhart-Mercer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628237055024" co-class="co-affbox">Lori Reinhart-Mercer</a>,<sup>3</sup> <a href="/pubmed/?term=Nations%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628250407360" co-class="co-affbox">Laura Nations</a>,<sup>3</sup> <a href="/pubmed/?term=Gilbert%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628250405136" co-class="co-affbox">John R. Gilbert</a>,<sup>3</sup> <a href="/pubmed/?term=Konidari%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628254899664" co-class="co-affbox">Ioanna Konidari</a>,<sup>3</sup> <a href="/pubmed/?term=Tramontana%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628254897440" co-class="co-affbox">Michael Tramontana</a>,<sup>4</sup> <a href="/pubmed/?term=Cuccaro%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628252236160" co-class="co-affbox">Michael L. Cuccaro</a>,<sup>3</sup> <a href="/pubmed/?term=Scott%20WK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628252233936" co-class="co-affbox">William K. Scott</a>,<sup>3</sup> <a href="/pubmed/?term=Pericak-Vance%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628253077056" co-class="co-affbox">Margaret A. Pericak-Vance</a>,<sup>3</sup> and  <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374" class="affpopup" co-rid="_co_idm140628253074832" co-class="co-affbox">Jonathan L. Haines</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140628251306912"><h3 class="no_margin">Anna C. Cummings</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Cummings%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Anna C. Cummings</a></div></div><div id="_co_idm140628251304800"><h3 class="no_margin">Lan Jiang</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Jiang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Lan Jiang</a></div></div><div id="_co_idm140628291855248"><h3 class="no_margin">Digna R. Velez Edwards</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</p><p><sup>2</sup>Vanderbilt Epidemiology Center, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Velez%20Edwards%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Digna R. Velez Edwards</a></div></div><div id="_co_idm140628291852288"><h3 class="no_margin">Jacob L. McCauley</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=McCauley%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Jacob L. McCauley</a></div></div><div id="_co_idm140628251523280"><h3 class="no_margin">Renee Laux</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Laux%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Renee Laux</a></div></div><div id="_co_idm140628251521056"><h3 class="no_margin">Lynne L. McFarland</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=McFarland%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Lynne L. McFarland</a></div></div><div id="_co_idm140628323848304"><h3 class="no_margin">Denise Fuzzell</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Fuzzell%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Denise Fuzzell</a></div></div><div id="_co_idm140628323846080"><h3 class="no_margin">Clare Knebusch</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Knebusch%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Clare Knebusch</a></div></div><div id="_co_idm140628237057248"><h3 class="no_margin">Laura Caywood</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Caywood%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Laura Caywood</a></div></div><div id="_co_idm140628237055024"><h3 class="no_margin">Lori Reinhart-Mercer</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Reinhart-Mercer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Lori Reinhart-Mercer</a></div></div><div id="_co_idm140628250407360"><h3 class="no_margin">Laura Nations</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Nations%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Laura Nations</a></div></div><div id="_co_idm140628250405136"><h3 class="no_margin">John R. Gilbert</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Gilbert%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">John R. Gilbert</a></div></div><div id="_co_idm140628254899664"><h3 class="no_margin">Ioanna Konidari</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Konidari%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Ioanna Konidari</a></div></div><div id="_co_idm140628254897440"><h3 class="no_margin">Michael Tramontana</h3><p><sup>4</sup>School of Medicine, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Tramontana%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Michael Tramontana</a></div></div><div id="_co_idm140628252236160"><h3 class="no_margin">Michael L. Cuccaro</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Cuccaro%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Michael L. Cuccaro</a></div></div><div id="_co_idm140628252233936"><h3 class="no_margin">William K. Scott</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Scott%20WK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">William K. Scott</a></div></div><div id="_co_idm140628253077056"><h3 class="no_margin">Margaret A. Pericak-Vance</h3><p><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Pericak-Vance%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Margaret A. Pericak-Vance</a></div></div><div id="_co_idm140628253074832"><h3 class="no_margin">Jonathan L. Haines</h3><p><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22881374">Jonathan L. Haines</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140628253882160_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140628253882160_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140628253882160_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Vanderbilt Epidemiology Center, Vanderbilt University Medical Center, Nashville, TN, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Dr. John T. Macdonald Foundation Department of Human Genetics and John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>School of Medicine, Vanderbilt University Medical Center, Nashville, TN, USA</div><div id="FN1">Corresponding author: Jonathan L. Haines, Center for Human Genetics Research, Vanderbilt University Medical Center, 519 Light Hall, Nashville, TN, 37232-0700, United States, <a href="mailto:dev@null" data-email="ude.tlibrednav.cm.rghc@nahtanoj" class="oemail">ude.tlibrednav.cm.rghc@nahtanoj</a>, Tel: 615-343-5851, Fax: 615-343-8619</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140628253882160_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22881374" target="pmc_ext" ref="reftype=publisher&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Ann Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3419486/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table S1.</strong></div><div><a href="/pmc/articles/PMC3419486/bin/NIHMS387665-supplement-Supp_Table_S1.doc" data-ga-action="click_feat_suppl">NIHMS387665-supplement-Supp_Table_S1.doc</a><span style="color:gray"> (43K)</span></div><div class="small guid">GUID:&nbsp;30AD643C-185A-4A1F-8196-4EDC65CB3403</div></div></dd></dl></div><div id="idm140628290398944" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140628290398944title">Summary</h2><!--article-meta--><div><p id="P2" class="p p-first-last">To identify novel late-onset Alzheimer disease (LOAD) risk genes, we have analyzed Amish populations of Ohio and Indiana. We performed genome-wide SNP linkage and association studies on 798 individuals (109 with LOAD). We tested association using the Modified Quasi-Likelihood Score (MQLS) test and also performed two-point and multipoint linkage analyses. We found that LOAD was significantly associated with <em>APOE</em> (P=9.0×10-6) in all our ascertainment regions except for the Adams County, Indiana, community (P=0.55). Genome-wide, the most strongly associated SNP was rs12361953 (P=7.92×10-7). A very strong, genome-wide significant multipoint peak (recessive HLOD=6.14, dominant HLOD=6.05) was detected on 2p12. Three additional loci with multipoint HLOD scores &gt;3 were detected on 3q26, 9q31, and 18p11. Converging linkage and association results, the most significantly associated SNP under the 2p12 peak was at rs2974151 (P=1.29×10-4). This SNP is located in <em>CTNNA2</em>, which encodes catenin alpha 2, a neuronal-specific catenin known to have function in the developing brain. These results identify <em>CTNNA2</em> as a novel candidate LOAD gene, and implicate three other regions of the genome as novel LOAD loci. These results underscore the utility of using family-based linkage and association analysis in isolated populations to identify novel loci for traits with complex genetic architecture.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">GWAS, Linkage, founder population, Amish, Alzheimer</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P3" class="p p-first">Late-onset Alzheimer disease (LOAD) is a neurodegenerative disorder causing the majority of dementia cases in the elderly. A complex combination of genetic and environmental components likely determine susceptibility to LOAD (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054495">Bertram et al. 2010</a>). The <em>APOE E4</em> allele is a well-established genetic risk factor for LOAD. Additional risk genes have been difficult to detect and replicate until recent successes using large consortia-derived genome-wide association study (GWAS) datasets, which have added <em>CR1</em>, <em>CLU</em>, <em>PICALM</em>, <em>BIN1</em>, <em>EPHA1</em>, <em>MS4A</em>, <em>CD33</em>, <em>CD2AP</em>, and <em>ABCA7</em> to the list of confirmed LOAD susceptibility genes, each with modest effect (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054485">Harold et al. 2009</a>; <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054507">Hollingworth et al. 2011</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054486">Lambert et al. 2009</a>; <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054510">Naj et al. 2011</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054497">Seshadri et al. 2010</a>).</p><p id="P4">Despite these recent successes the majority of the genetic risk for LOAD remains unknown. The remaining genetic risk may in part lie in additional loci with small effects at the population level, making most datasets underpowered. The use of a genetically isolated founder population, such as the Amish, represents an alternative to the use of large population based consortia-derived datasets in the search for genetic risk factors. In the case of a founder population, the number of disease variants is hypothesized to be fewer, thereby decreasing heterogeneity and increasing power.</p><p id="P5">We have taken this approach to discover at least one novel LOAD risk gene by studying the Amish communities of Holmes County, Ohio, and Adams, Elkhart and LaGrange Counties, Indiana (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054484">Hahs et al. 2006</a>; <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054504">McCauley et al. 2006</a>). These communities are collectively part of a genetically isolated founder population originating from two waves of immigration of Swiss Anabaptists into the U.S in the 1700’s and 1800’s. The first wave of immigration brought the Anabaptists to Pennsylvania. In the early 1800’s some of these immigrants moved to Holmes County, OH (<a href="#R5" rid="R5" class=" bibr popnode">Beachy 2011</a>), while a second wave of immigration from Europe established more Amish communities in Ohio (including Wayne County but not Holmes County) and Indiana (including Adams County) (<a href="#R13" rid="R13" class=" bibr popnode">Hostetler 1993</a>). Starting in 1841, the Elkhart and LaGrange Counties Amish community was founded by Amish families primarily from Somerset County, PA, and from Holmes and Wayne Counties, OH, who were seeking new farmland to settle (<a href="#R4" rid="R4" class=" bibr popnode">Amish Heritage Committee 2009</a>). The Amish marry within their faith, limiting the amount of genetic variation introduced to the population. Not only are the Amish more genetically homogeneous, but because of their strict lifestyle, environmental exposures are also more homogeneous. The Amish have large families and a well-preserved comprehensive family history that can be queried via the Anabaptist Genealogy Database (AGDB) (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054492">Agarwala et al. 1999</a>; <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054488">Agarwala et al. 2003</a>), making the Amish a valuable resource for genetic studies.</p><p id="P6" class="p p-last">Our current study undertook a genome-wide approach, in a population isolate, using complementary linkage and association analyses to further elucidate the complex genetic architecture of LOAD. We utilized linkage analysis to look for sharing of genomic regions among affected individuals, while also using association analysis to look for differences in allele frequencies between affecteds and unaffecteds. We previously performed a genome-wide linkage study using microsatellites genotyped in only a small subset of the individuals included in this study (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054498">Hahs et al 2006</a>). Here we use a much larger dataset with a much denser panel of markers using a genome-wide SNP chip. The results indicate that several novel regions likely harbor LOAD genes in the Amish, underscoring the genetic heterogeneity of this phenotype.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials &amp; Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Subjects</h3><p id="P7" class="p p-first-last">Methods for ascertainment were reviewed and approved by the individual Institutional Review Boards of the respective institutions. Participants were identified from published community directories, referral from other community members or due to close relationship with other participants, as previously described (Edwards et al. 2011). Informed consent was obtained from participants recruited from the Amish communities in Elkhart, LaGrange, and surrounding Indiana counties, and Holmes and surrounding Ohio counties with which we have had established working relationships for over 10 years.</p></div><div id="S4" class="sec"><h3 id="S4title">Clinical Data</h3><p id="P8" class="p p-first-last">For individuals who agreed to participate, demographic, family, and environmental information was collected, informed consent was obtained, and both a functional assessment and the Modified-Mini-Mental State Exam (3MS) were administered (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054509">Teng &amp; Chui 1987</a>; <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054489">Tschanz et al. 2002</a>). Those scoring ≥ 87 on the 3MS were considered cognitively normal and were considered unaffected in our study. Those scoring &lt;87 were re-examined with further tests from the CERAD neuropsychological battery (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054491">Morris et al. 1989</a>). Depression was also evaluated using the geriatric depression scale (GDS). Diagnoses for possible and probable AD were made according to the NINCDS-ADRDA criteria (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054502">McKhann et al. 1984</a>). A yearly consensus case conference was held to confirm all diagnoses.</p></div><div id="S5" class="sec"><h3 id="S5title">Genotyping</h3><p id="P9" class="p p-first">SNPs for <em>APOE</em> were genotyped for 823 individuals (127 with LOAD). To identify the six <em>APOE</em> genotypes determined by the <em>APOE</em> *<em>E2</em>, *<em>E3</em> and *<em>E4</em> alleles, two single nucleotide polymorphisms (SNPs) were assayed using the TaqMan method [Applied Biosystems Inc. (ABI), Foster City, CA, USA]. SNP-specific primers and probes were designed by ABI (TaqMan genotyping assays) and assays were performed according to the manufacturer’s instructions in 5 μl total volumes in 384-well plates. The polymorphisms distinguish the *<em>E2</em> allele from the *<em>E3</em> and *<em>E4</em> alleles at amino acid position 158 (NCBI <em>rs7412</em>) and the *<em>E4</em> allele from the *<em>E2</em> and *<em>E3</em> alleles at amino acid position 112 (NCBI <em>rs429358</em>).</p><p id="P10">Genome-wide genotyping was performed on 830 DNA samples using the Affymetrix 6.0 GeneChip ® Human Mapping 1 million array set (Affymetrix®, Inc Santa Clara, CA). DNA for this project was allocated by the respective DNA banks at both the Hussman Institute of Human Genomics (HIHG) at the University of Miami and the Center for Human Genetics Research (CHGR) at Vanderbilt University. Genomic DNA was quantitated via the ND-8000 spectrophotometer and DNA quality was evaluated via gel electrophoresis. The genomic DNA (250 ng/5ul) samples were processed according to standard Affymetrix procedures for processing of the Affymetrix 6.0 GeneChip assay. The arrays were then scanned using the GeneChip Scanner 3000 7G operated by the Affymetrix® GeneChip® Command Console® (AGCC) software. The data were processed for genotype calling using the Affymetrix® Power Tools (APT) software using the birdseed calling algorithm version 2.0 Affymetrix®, Inc Santa Clara, CA (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054514">Korn et al. 2008</a>).</p><p id="P11">We applied a number of quality control (QC) procedures to both samples and SNPs to ensure the accuracy of our genotype data prior to linkage and association analyses. Specific sample QC included: 1) Each individual DNA sample was examined via agarose to ensure that the sample was of high quality prior to inclusion on the array; 2) CEPH samples were placed across multiple arrays to ensure reproducibility of results across the arrays; 3) Samples with call rates &lt; 95% were re-examined individually to ensure quality of genotypes. 4) Ultimately if the sample call rate remained below 95% after further evaluation, attempts were made to rerun the array with a new DNA sample. If the sample still failed, it was dropped. Nine samples were dropped due to low genotyping efficiency. Three samples were excluded because they did not connect into a pedigree with the rest of the samples, and therefore, relationships of those individuals could not be accounted for. Sixteen samples with questionable gender based on X chromosome heterozygosity rates were eliminated. Three samples appearing to be aberrantly connected in the pedigree based on the genotype data were also excluded.</p><p id="P12" class="p p-last">Specific SNP QC included: 1) Dropping 76,816 SNPs with call rates &lt;98%. 2) Dropping 206,970 SNPs with minor allele frequencies (MAF) ≤0.05. We additionally excluded 7,849 SNPs with a MAF less than 0.05 after adjusting for pedigree relationships using MQLS (see below). Due to the relatedness in this dataset we did not check SNPs for Hardy-Weinberg equilibrium. Following this extensive quality control, 798 samples (109 with LOAD, see <a href="/pmc/articles/PMC3419486/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) and 614,963 SNPs were analyzed. Because <em>APOE</em> genotyping and QC were done separately from genome-wide genotyping and QC, the sample sizes are different and the datasets are mostly, but not completely, overlapping. All 798 samples belong to one 4998-member pedigree with many consanguineous loops. The AGDB provided the pedigree information using an “all common paths” database query with all genotyped individuals (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054506">Agarwala et al 2003</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Genome-wide dataset</strong><p id="__p9">Ages of exam and onset averages and standard deviations were calculated for the 798 samples—Late-onset AD (LOAD) samples, cognitively normal (unaffected) samples, and unclear or unknown samples—which passed QC for genome-wide genotyping.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628254478336" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center">Males</th><th rowspan="1" colspan="1" valign="top" align="center">Females</th><th rowspan="1" colspan="1" valign="top" align="center">Total</th><th rowspan="1" colspan="1" valign="top" align="center">Average Age of Exam (Standard Deviation)</th><th rowspan="1" colspan="1" valign="top" align="center">Average Age of Onset (Standard Deviation)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">LOAD Affected</td><td rowspan="1" colspan="1" valign="top" align="center">43</td><td rowspan="1" colspan="1" valign="top" align="center">66</td><td rowspan="1" colspan="1" valign="top" align="center">109</td><td rowspan="1" colspan="1" valign="top" align="center">83 (7.57)</td><td rowspan="1" colspan="1" valign="top" align="center">79 (6.68)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Cognitively Normal</td><td rowspan="1" colspan="1" valign="top" align="center">192</td><td rowspan="1" colspan="1" valign="top" align="center">258</td><td rowspan="1" colspan="1" valign="top" align="center">450</td><td rowspan="1" colspan="1" valign="top" align="center">78 (7.67)</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Unclear or Unknown</td><td rowspan="1" colspan="1" valign="top" align="center">117</td><td rowspan="1" colspan="1" valign="top" align="center">122</td><td rowspan="1" colspan="1" valign="top" align="center">239</td><td rowspan="1" colspan="1" valign="top" align="center">74 (15.52)</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm140628254478336" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3419486/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="S6" class="sec sec-last"><h3 id="S6title">Statistical Analysis</h3><div id="S7" class="sec sec-first"><p></p><h4 id="S7title" class="inline">Association analysis </h4><p id="P13" class="p p-first-last">We used the Modified Quasi-Likelihood Score (MQLS) test (software version 1.2) to correct for pedigree relationships (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054505">Thornton &amp; McPeek 2007</a>). MQLS is analogous to a χ<sup>2</sup> test, the most common approach for case-control data analysis with a binary trait, but MQLS incorporates kinship coefficients to correct for correlated genotypes of all the pedigree relationships. This test allows all samples to be included without dividing the pedigree. The MQLS test cannot be applied to X chromosome data, which were, therefore, eliminated from analysis. Because we previously found that Adams County has a lower <em>APOE</em>-4 allele frequency than the general population (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054483">Pericak-Vance et al. 1996</a>), we did a stratified association analysis for <em>APOE</em> analyzing Adams County separately from the combined Elkhart, LaGrange, and Holmes Counties. Using the same stratification, we also re-analyzed our most significant SNPs from the GWAS analysis. To test the validity of the MQLS test in our pedigree, we performed simulation studies using this same pedigree structure to assess the type 1 error rate using MQLS for association. Type 1 error rates were not inflated (unpublished data).</p></div><div id="S8" class="sec sec-last"><p></p><h4 id="S8title" class="inline">Linkage analysis </h4><p id="P14" class="p p-first">Because of the large size and substantial consanguinity of the pedigree, we used PedCut (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054511">Liu et al. 2008</a>) to find an optimal set of sub-pedigrees including the maximal number of subjects of interest within a bit-size limit (24 in this study) conducive to linkage analysis. This procedure resulted in 34 sub-pedigrees for analysis with an average of seven genotyped individuals (three genotyped affected) per sub-pedigree. Parametric heterogeneity two-point LOD (HLOD) scores were computed assuming affecteds-only autosomal dominant and recessive models using Merlin (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054499">Abecasis et al. 2002</a>). A disease allele frequency of 10% was used to approximate Alzheimer disease prevalence. For the dominant model penetrances of 0 for no disease allele and 0.0001 for one or two copies of the disease allele, and under the recessive model penetrances of 0 for zero or one disease allele and 0.0001 for two disease alleles were used. Because the underlying genetic model is unknown, we tested both dominant and recessive models to maximize our ability to find a disease locus. SNPs on the X chromosome were analyzed using MINX (Merlin in X). Regions showing evidence for linkage, i.e. containing at least one two-point HLOD ≥ 3.0, were followed up with parametric multipoint linkage analysis (also using Merlin). For the multipoint analyses, SNPs were pruned for linkage disequilibrium (LD) in each region so that all pair-wise r<sup>2</sup> values were &lt; 0.16 between all SNPs (Boyles et al. 2005). The LD from the HapMap CEPH samples (parents only) were used for pruning. Because the HapMap CEPH samples may not be an exact representation of LD in our Amish population, we also tested pruning using the data from this Amish dataset, but linkage results did not change using this approach (data not shown). Because linkage analyses can be biased when breaking larger pedigrees into a series of smaller ones (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054487">Liu et al. 2007</a>; <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054493">Liu et al. 2006</a>), we performed simulation studies assuming no linkage (e.g. null distribution) and using the same large pedigree structure and the same pedigree splitting method. We determined empirical cut-offs for significance in our linkage studies to maintain a nominal type I error rate. We found that after 1000 replications, only 2.5% of the multipoint linkage scans generated a maximum HLOD &gt;3.0 (unpublished data).</p><p id="P15" class="p p-last">All computations were done using either the Center for Human Genetics Research computational cluster or the Advanced Computing Center for Research and Education (ACCRE) cluster at Vanderbilt University.</p></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Results</h2><div id="S10" class="sec sec-first"><h3 id="S10title">APOE</h3><p id="P16" class="p p-first">We found that LOAD was significantly associated with <em>APOE</em> (MQLS <em>P</em>=9.0×10<sup>−6</sup>) in our Amish population except for the Adams County, Indiana, community (MQLS <em>P</em>=0.55). The <em>E</em>4 frequency, adjusted for pedigree relationships, in LOAD individuals in Elkhart, LaGrange, and Holmes Counties was 0.18 for affected individuals compared to 0.11 for unaffected individuals (<a href="/pmc/articles/PMC3419486/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). This compares to an <em>E</em>4 allele frequency of 0.38 in Caucasian AD individuals (0.14 for controls) (alzgene.org). We also saw a progressively younger average age of onset with each additional copy of the <em>E</em>4 allele (<a href="/pmc/articles/PMC3419486/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), consistent with other populations. We did not see evidence for linkage with <em>APOE</em> in our sub-pedigrees (dominant HLOD=0.50, recessive HLOD=0.29).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>MQLS-corrected <em>APOE</em> allele frequencies</strong><p id="__p10"><em>APOE</em> allele frequencies of Late-onset AD (LOAD) affected individuals versus cognitively normal individuals (unaffecteds) were calculated using MQLS to correct for pedigree relationships. Frequencies were calculated in the Adams County individuals separately from Elkhart, LaGrange, and Holmes Counties.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628251236528" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="bottom" align="center"><em>APOE</em> allele frequencies</th></tr><tr><th colspan="4" rowspan="1" valign="bottom" align="center">Elkhart, LaGrange, and Holmes Counties</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>E</em>2</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>E</em>3</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>E</em>4</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">LOAD Affected</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Cognitively Normal</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.82</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="center"><strong>Adams County</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">LOAD Affected</td><td rowspan="1" colspan="1" valign="top" align="center">0.00</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Cognitively Normal</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td></tr></tbody></table></div><div id="largeobj_idm140628251236528" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3419486/table/T2/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Age of onset and number of affected versus unaffected individuals by APOE genotype</strong><p id="__p11">Average ages of onset and standard deviations by <em>APOE</em> genotype and number LOAD affected and unaffected by <em>APOE</em> genotype</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628248820832" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="3" rowspan="1" valign="top" align="center"><em>APOE</em> Genotype</th><th rowspan="1" colspan="1" valign="top" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">4/4</th><th rowspan="1" colspan="1" valign="top" align="center">3/4</th><th rowspan="1" colspan="1" valign="top" align="center">2/4</th><th rowspan="1" colspan="1" valign="top" align="center">3/3</th><th rowspan="1" colspan="1" valign="top" align="center">2/3</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">Average Age of Onset (stdev)</td><td rowspan="1" colspan="1" valign="top" align="center">71 (7.59)</td><td rowspan="1" colspan="1" valign="top" align="center">76 (7.94)</td><td rowspan="1" colspan="1" valign="top" align="center">74 (3.54)</td><td rowspan="1" colspan="1" valign="top" align="center">80 (6.70)</td><td rowspan="1" colspan="1" valign="top" align="center">84 (7.42)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Number LOAD Affected</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">34</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">69</td><td rowspan="1" colspan="1" valign="top" align="center">12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Number Cognitively normal</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">90</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">308</td><td rowspan="1" colspan="1" valign="top" align="center">35</td></tr></tbody></table></div><div id="largeobj_idm140628248820832" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3419486/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P17" class="p p-last">Genome-wide association: In the GWAS, the most significant MQLS P-value (7.92×10<sup>−7</sup>), which did not surpass a Bonferroni-corrected genome-wide significance threshold of 8.13×10<sup>−8</sup>, was at rs12361953 on chromosome 11 in <em>LUZP2</em> (leucine zipper protein 2) (<a href="/pmc/articles/PMC3419486/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>, <a href="/pmc/articles/PMC3419486/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig 1</span></a>). The pedigree-adjusted minor allele frequency was 0.26 for affected individuals versus 0.15 for unaffected individuals. Fourteen additional SNPs had p-values &lt;1.0×10<sup>−5</sup> (<a href="/pmc/articles/PMC3419486/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). According to our simulation analyses, we have &gt;80% power to detect a p-value ≤ 0.005 under an additive model with an odds ratio of 2.0 (data not shown). After stratifying, each of the fifteen top SNPs had a more significant p-value in the non-Adams County dataset. Although some of the SNPs have very different minor allele frequencies in the two strata, the less significant p-values for the Adams County dataset can be explained mostly by the lack of power in that stratum (9 LOAD affected). All SNPs showed the same direction of effect in the two strata except for rs472926, rs12361953, and rs472926 (<a href="#SD1" rid="SD1" class=" supplementary-material">supplemental table 1</a>). These association results did not fall within a megabase of any of the other 9 previously verified LOAD genes (<em>CR1</em>, <em>CLU</em>, <em>PICALM</em>, <em>BIN1</em>, <em>EPHA1</em>, <em>MS4A</em>, <em>CD33</em>, <em>CD2AP</em>, and <em>ABCA7</em>). However, four SNPs (rs10792820, rs11234505, rs10501608, and rs7131120) in <em>PICALM</em> generated nominally significant p-values (P&lt;0.05). Rs11234505 is only ~3.0 kb from rs561655, the most significant SNP published by <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054496">Naj et al (2011)</a>, and rs10501608 is only ~10.5 kb from rs541458 the most significant SNP published by <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054503">Harold et al (2009)</a> and <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054494">Lambert et al (2009)</a>. We also have a nominally significant SNP, rs6591625, in the MS4A10 gene. The SNP is ~0.5 Mb from rs4938933, the most significant SNP published by <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054512">Naj et al (2011)</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3419486/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628253497600" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms387665f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms387665f1.jpg" src="/pmc/articles/PMC3419486/bin/nihms387665f1.jpg"></div></a><div id="largeobj_idm140628253497600" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3419486/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3419486/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3419486/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>MQLS Manhattan plot</strong><p id="__p7">Genome-wide association results were calculated using MQLS for 798 individuals (109 Late-onset Alzheimer disease affected). The lowest P-value (7.92×10<sup>−7</sup>) was calculated on chromosome 11 at rs12361953 which is located in the Leucine zipper protein 2 (<em>LUZP2)</em> gene.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Most significant genome-wide association results</strong><p id="__p12">Top genome-wide association results calculated using MQLS. Minor allele frequencies (MAF) are MQLS-corrected for pedigree relationships. A gene is only listed if SNP falls within specified gene. Megabase pair (Mpb) positions are based on NCBI Build 36.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628255911760" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Chromosome</th><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position (Mbp)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Minor Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Affected MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">Unaffected MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">MQLS <em>P</em>-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">rs4145462</td><td rowspan="1" colspan="1" valign="top" align="center">165.99</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">1.22×10-06</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MPZL1</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs41458646</td><td rowspan="1" colspan="1" valign="top" align="center">23.09</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">8.44×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs41476545</td><td rowspan="1" colspan="1" valign="top" align="center">23.09</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">9.02×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs6738181</td><td rowspan="1" colspan="1" valign="top" align="center">204.84</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">4.97×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs7638995</td><td rowspan="1" colspan="1" valign="top" align="center">69.26</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">1.82×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">rs679974</td><td rowspan="1" colspan="1" valign="top" align="center">105.06</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">8.67×10-06</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ATXN7L1</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">rs11983798</td><td rowspan="1" colspan="1" valign="top" align="center">105.07</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">1.49×10-06</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ATXN7L1</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">rs6468852</td><td rowspan="1" colspan="1" valign="top" align="center">104.05</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">1.06×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">rs9969729</td><td rowspan="1" colspan="1" valign="top" align="center">107.67</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">1.94×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs12361953</td><td rowspan="1" colspan="1" valign="top" align="center">24.57</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">7.92×10-07</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LUZP2</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs472926</td><td rowspan="1" colspan="1" valign="top" align="center">125.41</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">3.28×10-06</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CDON</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs4937314</td><td rowspan="1" colspan="1" valign="top" align="center">127.69</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">7.00×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">rs11848070</td><td rowspan="1" colspan="1" valign="top" align="center">70.58</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">5.64×10-06</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PCNX</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">rs17767225</td><td rowspan="1" colspan="1" valign="top" align="center">70.74</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.2</td><td rowspan="1" colspan="1" valign="top" align="center">7.88×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">rs6085820</td><td rowspan="1" colspan="1" valign="top" align="center">6.93</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">9.31×10-06</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm140628255911760" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3419486/table/T4/?report=objectonly">Open in a separate window</a></div></div></div><div id="S11" class="sec sec-last"><h3 id="S11title">Genome-wide linkage</h3><p id="P18" class="p p-first">In the genome-wide analysis, Forty five regions, among all chromosomes except 17, 21, and X, had at least one two-point HLOD ≥ 3.0 (data not shown). Multipoint linkage analysis for these regions resulted in four regions, one each on chromosomes 2, 3, 9, and 18 with a multipoint peak HLOD &gt; 3 (<a href="/pmc/articles/PMC3419486/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>, <a href="/pmc/articles/PMC3419486/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig 2</span></a>). The highest peak occurred on chromosome 2 with a recessive peak HLOD of 6.14 (90.91 Mbp) and a dominant peak HLOD of 6.05 (81.03 Mbp). The most significant association results within the recessive and dominant ±1-LOD-unit support interval were rs1258411 (<em>P</em>=5.29×10<sup>−2</sup>) and rs2974151 (<em>P</em>=1.29×10<sup>−4</sup>), respectively. Rs1258411 is not located in a gene, but rs2974151 is located in an intron of <em>CTNNA2</em> (catenin, alpha 2). In addition to rs2974151, 10 other SNPs in this gene had <em>P</em>-values &lt;0.05 (data not shown). While this is less than 5% of the analyzed SNPs in <em>CTNNA2</em>, it still warrants attention.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3419486/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628254481680" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms387665f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms387665f2.jpg" src="/pmc/articles/PMC3419486/bin/nihms387665f2.jpg"></div></a><div id="largeobj_idm140628254481680" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3419486/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3419486/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3419486/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Strongest multipoint linkage peaks</strong><p id="__p8">Parametric dominant (blue) and recessive (red) multipoint linkage peaks with HLOD scores &gt;3 were calculated on chromosomes 2 (a), 3 (b), 9 (c), and 18 (d). Red=recessive, Blue=dominant</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Most significant multipoint linkage results</strong><p id="__p13">Parametric dominant (Dom) and recessive (Rec) multipoint maximum heterogeneity (HLOD) scores were calculated using Merlin. Regions are determined by ±1-LOD-unit support intervals.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628228792736" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="center">Dom Peak HLOD</th><th rowspan="1" colspan="1" valign="bottom" align="center">Dom peak alpha</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position (Mbp)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Region</th><th rowspan="1" colspan="1" valign="bottom" align="center">Lowest MQLS p-value in Region</th><th rowspan="1" colspan="1" valign="bottom" align="center">Rec Peak HLOD</th><th rowspan="1" colspan="1" valign="bottom" align="center">Rec peak alpha</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position (Mbp)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Region</th><th rowspan="1" colspan="1" valign="bottom" align="center">Lowest MQLS p-value in Region</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">6.05</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center">81.03</td><td rowspan="1" colspan="1" valign="top" align="center">79.46–82.95</td><td rowspan="1" colspan="1" valign="top" align="center">1.29E-4 (rs2974151)</td><td rowspan="1" colspan="1" valign="top" align="center">6.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">90.81</td><td rowspan="1" colspan="1" valign="top" align="center">87.97–97.46</td><td rowspan="1" colspan="1" valign="top" align="center">5.29E-2 (rs1258411)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">5.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">168.43</td><td rowspan="1" colspan="1" valign="top" align="center">168.06–168.60</td><td rowspan="1" colspan="1" valign="top" align="center">4.00E-2 (rs9812366)</td><td rowspan="1" colspan="1" valign="top" align="center">3.53</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">168.43</td><td rowspan="1" colspan="1" valign="top" align="center">168.06–168.60</td><td rowspan="1" colspan="1" valign="top" align="center">4.00E-2 (rs9812366)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">4.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">107.77</td><td rowspan="1" colspan="1" valign="top" align="center">102.94–110.80</td><td rowspan="1" colspan="1" valign="top" align="center">1.94E-6 (rs9969729)</td><td rowspan="1" colspan="1" valign="top" align="center">3.77</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">101.7</td><td rowspan="1" colspan="1" valign="top" align="center">98.86–109.79</td><td rowspan="1" colspan="1" valign="top" align="center">1.94E-6 (rs9969729)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">3.97</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">8.77</td><td rowspan="1" colspan="1" valign="top" align="center">8.12–9.59</td><td rowspan="1" colspan="1" valign="top" align="center">8.80E-4 (rs632912)</td><td rowspan="1" colspan="1" valign="top" align="center">4.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">8.77</td><td rowspan="1" colspan="1" valign="top" align="center">8.12–9.59</td><td rowspan="1" colspan="1" valign="top" align="center">8.80E-4 (rs632912)</td></tr></tbody></table></div><div id="largeobj_idm140628228792736" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3419486/table/T5/?report=objectonly">Open in a separate window</a></div></div><p id="P19" class="p p-last">The next highest multipoint result was on chromosome 3 with a dominant HLOD of 5.27 and a recessive HLOD of 3.53. The peak for both models is at 168.43 Mbp, and the most significant association result in the ±1-LOD-unit support interval was at rs9812366 (<em>P</em>=4.00×10<sup>−2</sup>), which is intergenic. The linkage peak on chromosome 9 reached an HLOD of 4.44 (107.76 Mbp) under the dominant model and 3.77 (101.7 Mbp) under the recessive model. This peak overlaps with the suggestive linkage peak found in the joint linkage analysis published by <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054501">Hamshere et al (2007)</a>, however this region has not been consistently replicated in other studies. For both models the most significant association result in the ±1-LOD-unit support interval was at rs9969729 (<em>P</em>=1.94×10<sup>−6</sup>), which is intergenic. On chromosome 18 the dominant and recessive results both peaked at 8.77 Mbp with HLOD=3.97 for the dominant model and HLOD=4.43 for the recessive model. The most significant association result in this ±1-LOD-unit support interval was at rs632912 (<em>P</em>=8.80×10<sup>−4</sup>), which is intergenic. None of these regions overlap the linkage peaks found in our previous genome-wide microsatellite linkage study, which used only a subset of the individuals in the current dataset (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054508">Hahs et al 2006</a>). As with our association results, these multipoint peaks did not encompass the previously known LOAD genes.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Discussion</h2><p id="P20" class="p p-first"><em>APOE</em> was clearly associated with dementia in our population; however, it did not explain the majority of affected individuals. In the Adams County communities, there were only 8/74 individuals who carried at least one <em>APOE</em>-<em>E</em>4 allele. In the remaining Amish communities, the <em>APOE</em>-<em>E</em>4 allele was more common, but still less common than in the general population. In addition, the majority of affected individuals (81/127, 64% for all counties; 45/115, 39% for non-Adams counties) did not carry an <em>APOE</em>-<em>E</em>4 allele. The specific deficit of the <em>APOE</em>-<em>E</em>4 allele in Adams County as well as differences in allele frequencies for some of the top GWAS SNPs indicates at least some level of locus heterogeneity underlying LOAD in the Amish population.</p><p id="P21">Additional support for locus heterogeneity arises from the linkage results. Examination of the subpedigree-specific lod scores for the four significant loci indicates that 13 of the 34 subpedigrees generate no lod scores &gt;0.50 for any of the loci, and 14/21 (67%) of the remaining subpedigrees generate lod scores &gt;0.50 for only one of the four loci. In addition, the vast majority of the remaining SNPs across the genome generated HLOD scores with alpha values (proportion of linked pedigrees) &lt;1.0. Finally, the suggestion of locus heterogeneity is consistent with the societal differences across church districts, which can further restrict marriages even within the Amish.</p><p id="P22">Because of the relatedness of individuals in our dataset we could take advantage of both linkage and association approaches to identify potential LOAD loci. In our examination, we found that our most significant association results did not fall under any of the four linkage peaks. However, under the linkage peaks we did see some evidence of association. Within our most significant region of linkage lies <em>CTNNA2</em>, which also had suggestive evidence for association. In addition to the result at rs2974151 (<em>P</em>=1.29×10<sup>−4</sup>), multiple SNPs in <em>CTNNA2</em> had P-values &lt; 0.05, decreasing the likelihood of a false positive association for this gene. However, because of the relatedness in our dataset it was difficult to get an accurate measurement of LD structure to determine if the SNPs in this region were more highly correlated due to a founder effect.</p><p id="P23"><em>CTNNA2</em> encodes the catenin alpha 2 protein, which is a neuronal-specific catenin. Catenins are cadherin-associated proteins and are thought to link cadherins to the cytoskeleton to regulate cell-cell adhesion. Catenin alpha 2 can form complexes with other catenins such as beta-catenin, which interacts with presinilin. Mutations in presinilin lead to destabilization of beta-catenin which potentiates neuronal apoptosis (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054500">Zhang et al. 1998</a>). Catenin alpha 2 is also thought to regulate morphological plasticity of synapses and cerebellar and hippocampal lamination during development in mice (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054513">Park et al. 2002</a>). It also functions in the control of startle modulation in mice (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_269054490">Park et al 2002</a>).</p><p id="P24">It was not completely unexpected to see some discordance between the linkage and association results, as was demonstrated in our <em>APOE</em> results where we saw evidence for association but not for linkage. Because we needed to divide the pedigree to facilitate linkage analysis and because we used an affecteds-only analysis, only a subset of the individuals analyzed in association analysis were analyzed in linkage analysis. The breaking of the pedigree likely reduces the observed genomic sharing between relatives as the tracking of the natural flow of alleles was somewhat disrupted, as we saw when we tested <em>APOE</em> for linkage. Also, the very nature of association analysis versus linkage analysis will provide some different results. Linkage analysis locates shared genomic regions between affected individuals in the same pedigree by testing for co-segregation of a chromosomal segment from a common ancestor. Association using MQLS tests for differences in allele frequencies between affected and unaffected individuals while correcting for the pedigree relationships. Association analysis is more powerful in detecting protective effects as well as smaller effects in the population compared to affecteds-only linkage analysis but is underpowered when sample sizes are small and genetic heterogeneity is present. Conversely, linkage analysis is more suitable for finding large effects in a small number of related individuals and is more robust to genetic heterogeneity.</p><p id="P25" class="p p-last">Our results confirmed the complex genetic architecture of LOAD even in this more homogeneous set of individuals. Multiple genes appeared to be significantly contributing to LOAD risk in the Amish. We replicated the effect of <em>APOE</em>, replicated the evidence for linkage on 9q22, and also found modest evidence for association of both <em>PICALM</em> and <em>MS4A</em> in this population. Most importantly, this unique population allowed us to find additional candidate loci, particularly in the <em>CTNNA2</em> region in which we saw strong evidence for both linkage and association. The role of CTNNA2 in the brain also makes this gene a promising candidate. The <em>CTNNA2</em> region, in addition to other potential risk regions, needs to be more closely examined to identify the underlying responsible variants and their functional consequences.</p></div><div id="S13" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp Table S1</h4><div class="sup-box half_rhythm" id="d35e739"><a href="/pmc/articles/PMC3419486/bin/NIHMS387665-supplement-Supp_Table_S1.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(43K, doc)</sup></div></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Acknowledgments</h2><div class="sec"><p id="__p1">We thank the family participants and community members for graciously agreeing to participate, making this research possible. This study is supported by the National Institutes of Health grants AG019085 (to JLH and MAP-V) and <a id="__tag_786362550" class="tag_hotlink" href="/nuccore/AG019726" ref="/nuccore/AG019726"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG019726","term_id":"6045670","term_text":"AG019726"}}</span>AG019726</a> (to WKS). Some of the samples used in this study were collected while WKS, JRG, and MAP-V were faculty members at Duke University. The authors would like to thank Gene Jackson of Scott &amp; White for his effort and support on this project. Additional work was performed using the Vanderbilt Center for Human Genetics Research Core facilities: the Genetic Studies Ascertainment Core, the DNA Resources Core, and the Computational Genomics Core.</p></div></div><div id="idm140628251353872" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140628251353872title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"> </p><p id="__p2" class="p p-first-last">The authors have no conflict of interest to declare.</p><p></p><p class="fn sec" id="FN3"></p><p id="__p3" class="p p-first">SUPPORTING INFORMATION</p><p id="__p4" class="p">Additional supporting information may be found in the online version of this article:</p><p id="__p5" class="p"><a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a> Most significant genome-wide association results, stratified.</p><p id="__p6" class="p p-last">As a service to our authors and readers, this journal provides supporting information supplied by the authors. Such materials are peer-reviewed and may be re-organised for online delivery, but are not copy-edited or typeset. Technical support issues arising from supporting information (other than missing files) should be addressed to the authors.</p><p></p></div></div><div id="idm140628252715424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140628252715424title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Abecasis GR, Cherny SS, Cookson WO, Cardon LR. Rapid analysis of dense genetic maps using sparse gene flow trees. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">30</span>:97–101.</span> [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=GR+Abecasis&amp;author=SS+Cherny&amp;author=WO+Cookson&amp;author=LR+Cardon&amp;volume=30&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Agarwala R, Biesecker LG, Schäffer AA. Anabaptist genealogy database. <span><span class="ref-journal">Am J Med Genet C Semin Med Genet. </span>2003;<span class="ref-vol">121</span>:32–37.</span> [<a href="/pubmed/12888984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+C+Semin+Med+Genet&amp;title=Anabaptist+genealogy+database&amp;author=R+Agarwala&amp;author=LG+Biesecker&amp;author=AA+Schäffer&amp;volume=121&amp;publication_year=2003&amp;pages=32-37&amp;pmid=12888984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Agarwala R, Biesecker LG, Tomlin JF, Schäffer AA. Towards a complete North American Anabaptist genealogy: A systematic approach to merging partially overlapping genealogy resources. <span><span class="ref-journal">Am J Med Genet. </span>1999;<span class="ref-vol">86</span>:156–161.</span> [<a href="/pubmed/10449652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Towards+a+complete+North+American+Anabaptist+genealogy:+A+systematic+approach+to+merging+partially+overlapping+genealogy+resources&amp;author=R+Agarwala&amp;author=LG+Biesecker&amp;author=JF+Tomlin&amp;author=AA+Schäffer&amp;volume=86&amp;publication_year=1999&amp;pages=156-161&amp;pmid=10449652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Amish Heritage Committee. <span class="ref-journal">Amish and Mennonites in Eastern Elkhart &amp; LaGrange Counties, Indiana 1841–1991.</span> Goshen, Indiana: Amish Heritage Committee; 2009.  2nd printing. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Amish+and+Mennonites+in+Eastern+Elkhart+&amp;+LaGrange+Counties,+Indiana+1841–1991&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Beachy L.  <span class="ref-journal">Unser Leit: The Story of the Amish.</span> Millersburg, OH: Goodly Heritage Books; 2011.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Unser+Leit:+The+Story+of+the+Amish&amp;author=L+Beachy&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Bertram L, Lill CM, Tanzi RE. The genetics of Alzheimer disease: back to the future. <span><span class="ref-journal">Neuron. </span>2010;<span class="ref-vol">68</span>:270–281.</span> [<a href="/pubmed/20955934" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=The+genetics+of+Alzheimer+disease:+back+to+the+future&amp;author=L+Bertram&amp;author=CM+Lill&amp;author=RE+Tanzi&amp;volume=68&amp;publication_year=2010&amp;pages=270-281&amp;pmid=20955934&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Boyles AL, Scott WK, Martin ER, Schmidt S, Li YJ, Ashley-Koch A, Bass MP, Schmidt M, Pericak-Vance MA, Speer MC, Hauser ER. Linkage disequilibrium inflates type I error rates in multipoint linkage analysis when parental genotypes are missing. <span><span class="ref-journal">Hum Hered. </span>2005;<span class="ref-vol">59</span>:220–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1224705/">PMC free article</a>]</span> [<a href="/pubmed/16093727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Linkage+disequilibrium+inflates+type+I+error+rates+in+multipoint+linkage+analysis+when+parental+genotypes+are+missing&amp;author=AL+Boyles&amp;author=WK+Scott&amp;author=ER+Martin&amp;author=S+Schmidt&amp;author=YJ+Li&amp;volume=59&amp;publication_year=2005&amp;pages=220-227&amp;pmid=16093727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Edwards DR, Gilbert JR, Jiang L, Gallins PJ, Caywood L, Creason M, Fuzzell D, Knebusch C, Jackson CE, Pericak-Vance MA, Haines JL, Scott WK. Successful aging shows linkage to chromosomes 6, 7, and 14 in the amish. <span><span class="ref-journal">Ann Hum Genet. </span>2011;<span class="ref-vol">75</span>:516–528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3756593/">PMC free article</a>]</span> [<a href="/pubmed/21668908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Successful+aging+shows+linkage+to+chromosomes+6,+7,+and+14+in+the+amish&amp;author=DR+Edwards&amp;author=JR+Gilbert&amp;author=L+Jiang&amp;author=PJ+Gallins&amp;author=L+Caywood&amp;volume=75&amp;publication_year=2011&amp;pages=516-528&amp;pmid=21668908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Hahs DW, McCauley JL, Crunk AE, McFarland LL, Gaskell PC, Jiang L, Slifer SH, Vance JM, Scott WK, Welsh-Bohmer KA, Johnson SR, Jackson CE, Pericak-Vance MA, Haines JL. A genome-wide linkage analysis of dementia in the Amish. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2006;<span class="ref-vol">141</span>:160–166.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2613780/">PMC free article</a>]</span> [<a href="/pubmed/16389594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=A+genome-wide+linkage+analysis+of+dementia+in+the+Amish&amp;author=DW+Hahs&amp;author=JL+McCauley&amp;author=AE+Crunk&amp;author=LL+McFarland&amp;author=PC+Gaskell&amp;volume=141&amp;publication_year=2006&amp;pages=160-166&amp;pmid=16389594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Hamshere ML, Holmans PA, Avramopoulos D, Bassett SS, Blacker D, Bertram L, Wiener H, Rochberg N, Tanzi RE, Myers A, Wavrant-De Vrièze F, Go R, Fallin D, Lovestone S, Hardy J, Goate A, O’Donovan M, Williams J, Owen MJ. Genome-wide linkage analysis of 723 affected relative pairs with late-onset Alzheimer’s disease. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>:2703–2712.</span> [<a href="/pubmed/17725986" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+linkage+analysis+of+723+affected+relative+pairs+with+late-onset+Alzheimer’s+disease&amp;author=ML+Hamshere&amp;author=PA+Holmans&amp;author=D+Avramopoulos&amp;author=SS+Bassett&amp;author=D+Blacker&amp;volume=16&amp;publication_year=2007&amp;pages=2703-2712&amp;pmid=17725986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Harold D, Abraham R, Hollingworth P, Sims R, Gerrish A, Hamshere ML, Pahwa JS, Moskvina V, Dowzell K, Williams A, Jones N, Thomas C, Stretton A, Morgan AR, Lovestone S, Powell J, Proitsi P, Lupton MK, Brayne C, Rubinsztein DC, Gill M, Lawlor B, Lynch A, Morgan K, Brown KS, Passmore PA, Craig D, McGuinness B, Todd S, Holmes C, Mann D, Smith AD, Love S, Kehoe PG, Hardy J, Mead S, Fox N, Rossor M, Collinge J, Maier W, Jessen F, Schürmann B, van den Bussche H, Heuser I, Kornhuber J, Wiltfang J, Dichgans M, Frölich L, Hampel H, Hüll M, Rujescu D, Goate AM, Kauwe JS, Cruchaga C, Nowotny P, Morris JC, Mayo K, Sleegers K, Bettens K, Engelborghs S, De Deyn PP, Van Broeckhoven C, Livingston G, Bass NJ, Gurling H, McQuillin A, Gwilliam R, Deloukas P, Al-Chalabi A, Shaw CE, Tsolaki M, Singleton AB, Guerreiro R, Mühleisen TW, Nöthen MM, Moebus S, Jöckel KH, Klopp N, Wichmann HE, Carrasquillo MM, Pankratz VS, Younkin SG, Holmans PA, O’Donovan M, Owen MJ, Williams J. Genome-wide association study identifies variants at CLU and PICALM associated with Alzheimer’s disease. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1088–1093.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2845877/">PMC free article</a>]</span> [<a href="/pubmed/19734902" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CLU+and+PICALM+associated+with+Alzheimer’s+disease&amp;author=D+Harold&amp;author=R+Abraham&amp;author=P+Hollingworth&amp;author=R+Sims&amp;author=A+Gerrish&amp;volume=41&amp;publication_year=2009&amp;pages=1088-1093&amp;pmid=19734902&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Hollingworth P, Harold D, Sims R, Gerrish A, Lambert JC, Carrasquillo MM, Abraham R, Hamshere ML, Pahwa JS, Moskvina V, Dowzell K, Jones N, Stretton A, Thomas C, Richards A, Ivanov D, Widdowson C, Chapman J, Lovestone S, Powell J, Proitsi P, Lupton MK, Brayne C, Rubinsztein DC, Gill M, Lawlor B, Lynch A, Brown KS, Passmore PA, Craig D, McGuinness B, Todd S, Holmes C, Mann D, Smith AD, Beaumont H, Warden D, Wilcock G, Love S, Kehoe PG, Hooper NM, Vardy ER, Hardy J, Mead S, Fox NC, Rossor M, Collinge J, Maier W, Jessen F, Rüther E, Schürmann B, Heun R, Kölsch H, van den Bussche H, Heuser I, Kornhuber J, Wiltfang J, Dichgans M, Frölich L, Hampel H, Gallacher J, Hüll M, Rujescu D, Giegling I, Goate AM, Kauwe JS, Cruchaga C, Nowotny P, Morris JC, Mayo K, Sleegers K, Bettens K, Engelborghs S, De Deyn PP, Van Broeckhoven C, Livingston G, Bass NJ, Gurling H, McQuillin A, Gwilliam R, Deloukas P, Al-Chalabi A, Shaw CE, Tsolaki M, Singleton AB, Guerreiro R, Mühleisen TW, Nöthen MM, Moebus S, Jöckel KH, Klopp N, Wichmann HE, Pankratz VS, Sando SB, Aasly JO, Barcikowska M, Wszolek ZK, Dickson DW, Graff-Radford NR, Petersen RC, van Duijn CM, Breteler MM, Ikram MA, DeStefano AL, Fitzpatrick AL, Lopez O, Launer LJ, Seshadri S, Berr C, Campion D, Epelbaum J, Dartigues JF, Tzourio C, Alpérovitch A, Lathrop M, Feulner TM, Friedrich P, Riehle C, Krawczak M, Schreiber S, Mayhaus M, Nicolhaus S, Wagenpfeil S, Steinberg S, Stefansson H, Stefansson K, Snaedal J, Björnsson S, Jonsson PV, Chouraki V, Genier-Boley B, Hiltunen M, Soininen H, Combarros O, Zelenika D, Delepine M, Bullido MJ, Pasquier F, Mateo I, Frank-Garcia A, Porcellini E, Hanon O, Coto E, Alvarez V, Bosco P, Siciliano G, Mancuso M, Panza F, Solfrizzi V, Nacmias B, Sorbi S, Bossù P, Piccardi P, Arosio B, Annoni G, Seripa D, Pilotto A, Scarpini E, Galimberti D, Brice A, Hannequin D, Licastro F, Jones L, Holmans PA, Jonsson T, Riemenschneider M, Morgan K, Younkin SG, Owen MJ, O’Donovan M, Amouyel P, Williams J Alzheimer’s Disease Neuroimaging Initiative; CHARGE consortium; EADI1 consortium. Common variants at ABCA7, MS4A6A/MS4A4E, EPHA1, CD33 and CD2AP are associated with Alzheimer’s disease. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:429–435.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3084173/">PMC free article</a>]</span> [<a href="/pubmed/21460840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+ABCA7,+MS4A6A/MS4A4E,+EPHA1,+CD33+and+CD2AP+are+associated+with+Alzheimer’s+disease&amp;author=P+Hollingworth&amp;author=D+Harold&amp;author=R+Sims&amp;author=A+Gerrish&amp;author=JC+Lambert&amp;volume=43&amp;publication_year=2011&amp;pages=429-435&amp;pmid=21460840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Hostetler J.  <span class="ref-journal">Amish Society.</span> 4. Baltimore, MD: Johns Hopkins University Press; 1993.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Amish+Society&amp;author=J+Hostetler&amp;publication_year=1993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Korn JM, Kuruvilla FG, McCarroll SA, Wysoker A, Nemesh J, Cawley S, Hubbell E, Veitch J, Collins PJ, Darvishi K, Lee C, Nizzari MM, Gabriel SB, Purcell S, Daly MJ, Altshuler D. Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1253–1260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=JM+Korn&amp;author=FG+Kuruvilla&amp;author=SA+McCarroll&amp;author=A+Wysoker&amp;author=J+Nemesh&amp;volume=40&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Lambert JC, Heath S, Even G, Campion D, Sleegers K, Hiltunen M, Combarros O, Zelenika D, Bullido MJ, Tavernier B, Letenneur L, Bettens K, Berr C, Pasquier F, Fiévet N, Barberger-Gateau P, Engelborghs S, De Deyn P, Mateo I, Franck A, Helisalmi S, Porcellini E, Hanon O, de Pancorbo MM, Lendon C, Dufouil C, Jaillard C, Leveillard T, Alvarez V, Bosco P, Mancuso M, Panza F, Nacmias B, Bossù P, Piccardi P, Annoni G, Seripa D, Galimberti D, Hannequin D, Licastro F, Soininen H, Ritchie K, Blanché H, Dartigues JF, Tzourio C, Gut I, Van Broeckhoven C, Alpérovitch A, Lathrop M, Amouyel P. Genome-wide association study identifies variants at CLU and CR1 associated with Alzheimer’s disease. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1094–1099.</span> [<a href="/pubmed/19734903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CLU+and+CR1+associated+with+Alzheimer’s+disease&amp;author=JC+Lambert&amp;author=S+Heath&amp;author=G+Even&amp;author=D+Campion&amp;author=K+Sleegers&amp;volume=41&amp;publication_year=2009&amp;pages=1094-1099&amp;pmid=19734903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Liu F, Arias-Vásquez A, Sleegers K, Aulchenko YS, Kayser M, Sanchez-Juan P, Feng BJ, Bertoli-Avella AM, van Swieten J, Axenovich TI, Heutink P, van Broeckhoven C, Oostra BA, van Duijn CM. A genomewide screen for late-onset Alzheimer disease in a genetically isolated Dutch population. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:17–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950931/">PMC free article</a>]</span> [<a href="/pubmed/17564960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genomewide+screen+for+late-onset+Alzheimer+disease+in+a+genetically+isolated+Dutch+population&amp;author=F+Liu&amp;author=A+Arias-Vásquez&amp;author=K+Sleegers&amp;author=YS+Aulchenko&amp;author=M+Kayser&amp;volume=81&amp;publication_year=2007&amp;pages=17-31&amp;pmid=17564960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Liu F, Elefante S, van Duijn CM, Aulchenko YS. Ignoring distant genealogic loops leads to false-positives in homozygosity mapping. <span><span class="ref-journal">Ann Hum Genet. </span>2006;<span class="ref-vol">70</span>:965–970.</span> [<a href="/pubmed/17044871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Ignoring+distant+genealogic+loops+leads+to+false-positives+in+homozygosity+mapping&amp;author=F+Liu&amp;author=S+Elefante&amp;author=CM+van+Duijn&amp;author=YS+Aulchenko&amp;volume=70&amp;publication_year=2006&amp;pages=965-970&amp;pmid=17044871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Liu F, Kirichenko A, Axenovich TI, van Duijn CM, Aulchenko YS. An approach for cutting large and complex pedigrees for linkage analysis. <span><span class="ref-journal">Eur J Hum Genet. </span>2008;<span class="ref-vol">16</span>:854–860.</span> [<a href="/pubmed/18301450" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=An+approach+for+cutting+large+and+complex+pedigrees+for+linkage+analysis&amp;author=F+Liu&amp;author=A+Kirichenko&amp;author=TI+Axenovich&amp;author=CM+van+Duijn&amp;author=YS+Aulchenko&amp;volume=16&amp;publication_year=2008&amp;pages=854-860&amp;pmid=18301450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">McCauley JL, Hahs DW, Jiang L, Scott WK, Welsh-Bohmer KA, Jackson CE, Vance JM, Pericak-Vance MA, Haines JL. Combinatorial Mismatch Scan (CMS) for loci associated with dementia in the Amish. <span><span class="ref-journal">BMC Med Genet. </span>2006;<span class="ref-vol">7</span>:19.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1448207/">PMC free article</a>]</span> [<a href="/pubmed/16515697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Combinatorial+Mismatch+Scan+(CMS)+for+loci+associated+with+dementia+in+the+Amish&amp;author=JL+McCauley&amp;author=DW+Hahs&amp;author=L+Jiang&amp;author=WK+Scott&amp;author=KA+Welsh-Bohmer&amp;volume=7&amp;publication_year=2006&amp;pages=19&amp;pmid=16515697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">McKhann G, Drachman D, Folstein M, Katzman R, Price D, Stadlan EM. Clinical diagnosis of Alzheimer’s disease: report of the NINCDS-ADRDA Work Group under the auspices of Department of Health and Human Services Task Force on Alzheimer’s Disease. <span><span class="ref-journal">Neurology. </span>1984;<span class="ref-vol">34</span>:939–944.</span> [<a href="/pubmed/6610841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Clinical+diagnosis+of+Alzheimer’s+disease:+report+of+the+NINCDS-ADRDA+Work+Group+under+the+auspices+of+Department+of+Health+and+Human+Services+Task+Force+on+Alzheimer’s+Disease&amp;author=G+McKhann&amp;author=D+Drachman&amp;author=M+Folstein&amp;author=R+Katzman&amp;author=D+Price&amp;volume=34&amp;publication_year=1984&amp;pages=939-944&amp;pmid=6610841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Morris JC, Heyman A, Mohs RC, Hughes JP, van Belle G, Fillenbaum G, Mellits ED, Clark C. The Consortium to Establish a Registry for Alzheimer’s Disease (CERAD) Part I Clinical and neuropsychological assessment of Alzheimer’s disease. <span><span class="ref-journal">Neurology. </span>1989;<span class="ref-vol">39</span>:1159–1165.</span> [<a href="/pubmed/2771064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Part+I+Clinical+and+neuropsychological+assessment+of+Alzheimer’s+disease&amp;author=JC+Morris&amp;author=A+Heyman&amp;author=RC+Mohs&amp;author=JP+Hughes&amp;author=G+van+Belle&amp;volume=39&amp;publication_year=1989&amp;pages=1159-1165&amp;pmid=2771064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Naj AC, Jun G, Beecham GW, Wang LS, Vardarajan BN, Buros J, Gallins PJ, Buxbaum JD, Jarvik GP, Crane PK, Larson EB, Bird TD, Boeve BF, Graff-Radford NR, De Jager PL, Evans D, Schneider JA, Carrasquillo MM, Ertekin-Taner N, Younkin SG, Cruchaga C, Kauwe JS, Nowotny P, Kramer P, Hardy J, Huentelman MJ, Myers AJ, Barmada MM, Demirci FY, Baldwin CT, Green RC, Rogaeva E, St George-Hyslop P, Arnold SE, Barber R, Beach T, Bigio EH, Bowen JD, Boxer A, Burke JR, Cairns NJ, Carlson CS, Carney RM, Carroll SL, Chui HC, Clark DG, Corneveaux J, Cotman CW, Cummings JL, DeCarli C, DeKosky ST, Diaz-Arrastia R, Dick M, Dickson DW, Ellis WG, Faber KM, Fallon KB, Farlow MR, Ferris S, Frosch MP, Galasko DR, Ganguli M, Gearing M, Geschwind DH, Ghetti B, Gilbert JR, Gilman S, Giordani B, Glass JD, Growdon JH, Hamilton RL, Harrell LE, Head E, Honig LS, Hulette CM, Hyman BT, Jicha GA, Jin LW, Johnson N, Karlawish J, Karydas A, Kaye JA, Kim R, Koo EH, Kowall NW, Lah JJ, Levey AI, Lieberman AP, Lopez OL, Mack WJ, Marson DC, Martiniuk F, Mash DC, Masliah E, McCormick WC, McCurry SM, McDavid AN, McKee AC, Mesulam M, Miller BL, Miller CA, Miller JW, Parisi JE, Perl DP, Peskind E, Petersen RC, Poon WW, Quinn JF, Rajbhandary RA, Raskind M, Reisberg B, Ringman JM, Roberson ED, Rosenberg RN, Sano M, Schneider LS, Seeley W, Shelanski ML, Slifer MA, Smith CD, Sonnen JA, Spina S, Stern RA, Tanzi RE, Trojanowski JQ, Troncoso JC, Van Deerlin VM, Vinters HV, Vonsattel JP, Weintraub S, Welsh-Bohmer KA, Williamson J, Woltjer RL, Cantwell LB, Dombroski BA, Beekly D, Lunetta KL, Martin ER, Kamboh MI, Saykin AJ, Reiman EM, Bennett DA, Morris JC, Montine TJ, Goate AM, Blacker D, Tsuang DW, Hakonarson H, Kukull WA, Foroud TM, Haines JL, Mayeux R, Pericak-Vance MA, Farrer LA, Schellenberg GD. Common variants at MS4A4/MS4A6E, CD2AP, CD33 and EPHA1 are associated with late-onset Alzheimer’s disease. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:436–441.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3090745/">PMC free article</a>]</span> [<a href="/pubmed/21460841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+MS4A4/MS4A6E,+CD2AP,+CD33+and+EPHA1+are+associated+with+late-onset+Alzheimer’s+disease&amp;author=AC+Naj&amp;author=G+Jun&amp;author=GW+Beecham&amp;author=LS+Wang&amp;author=BN+Vardarajan&amp;volume=43&amp;publication_year=2011&amp;pages=436-441&amp;pmid=21460841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Park C, Falls W, Finger JH, Longo-Guess CM, Ackerman SL. Deletion in Catna2, encoding alpha N-catenin, causes cerebellar and hippocampal lamination defects and impaired startle modulation. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">31</span>:279–284.</span> [<a href="/pubmed/12089526" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Deletion+in+Catna2,+encoding+alpha+N-catenin,+causes+cerebellar+and+hippocampal+lamination+defects+and+impaired+startle+modulation&amp;author=C+Park&amp;author=W+Falls&amp;author=JH+Finger&amp;author=CM+Longo-Guess&amp;author=SL+Ackerman&amp;volume=31&amp;publication_year=2002&amp;pages=279-284&amp;pmid=12089526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Pericak-Vance MA, Johnson CC, Rimmler JB, Saunders AM, Robinson LC, D’Hondt EG, Jackson CE, Haines JL. Alzheimer’s disease and apolipoprotein E-4 allele in an Amish population. <span><span class="ref-journal">Ann Neurol. </span>1996;<span class="ref-vol">39</span>:700–704.</span> [<a href="/pubmed/8651641" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Alzheimer’s+disease+and+apolipoprotein+E-4+allele+in+an+Amish+population&amp;author=MA+Pericak-Vance&amp;author=CC+Johnson&amp;author=JB+Rimmler&amp;author=AM+Saunders&amp;author=LC+Robinson&amp;volume=39&amp;publication_year=1996&amp;pages=700-704&amp;pmid=8651641&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Seshadri S, Fitzpatrick AL, Ikram MA, DeStefano AL, Gudnason V, Boada M, Bis JC, Smith AV, Carassquillo MM, Lambert JC, Harold D, Schrijvers EM, Ramirez-Lorca R, Debette S, Longstreth WT, Jr, Janssens AC, Pankratz VS, Dartigues JF, Hollingworth P, Aspelund T, Hernandez I, Beiser A, Kuller LH, Koudstaal PJ, Dickson DW, Tzourio C, Abraham R, Antunez C, Du Y, Rotter JI, Aulchenko YS, Harris TB, Petersen RC, Berr C, Owen MJ, Lopez-Arrieta J, Varadarajan BN, Becker JT, Rivadeneira F, Nalls MA, Graff-Radford NR, Campion D, Auerbach S, Rice K, Hofman A, Jonsson PV, Schmidt H, Lathrop M, Mosley TH, Au R, Psaty BM, Uitterlinden AG, Farrer LA, Lumley T, Ruiz A, Williams J, Amouyel P, Younkin SG, Wolf PA, Launer LJ, Lopez OL, van Duijn CM, Breteler MM European Alzheimer’s Disease Initiative Investigators; CHARGE Consortium; GERAD1 Consortium; EADI1 Consortium. Genome-wide analysis of genetic loci associated with Alzheimer disease. <span><span class="ref-journal">Journal of the American Medical Association. </span>2010;<span class="ref-vol">303</span>:1832–1840.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2989531/">PMC free article</a>]</span> [<a href="/pubmed/20460622" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+the+American+Medical+Association&amp;title=Genome-wide+analysis+of+genetic+loci+associated+with+Alzheimer+disease&amp;author=S+Seshadri&amp;author=AL+Fitzpatrick&amp;author=MA+Ikram&amp;author=AL+DeStefano&amp;author=V+Gudnason&amp;volume=303&amp;publication_year=2010&amp;pages=1832-1840&amp;pmid=20460622&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Teng EL, Chui HC. The Modified Mini-Mental State (3MS) examination. <span><span class="ref-journal">J Clin Psychiatry. </span>1987;<span class="ref-vol">48</span>:314–318.</span> [<a href="/pubmed/3611032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Psychiatry&amp;title=The+Modified+Mini-Mental+State+(3MS)+examination&amp;author=EL+Teng&amp;author=HC+Chui&amp;volume=48&amp;publication_year=1987&amp;pages=314-318&amp;pmid=3611032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Thornton T, McPeek MS. Case-control association testing with related individuals: a more powerful quasi-likelihood score test. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:321–337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950805/">PMC free article</a>]</span> [<a href="/pubmed/17668381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Case-control+association+testing+with+related+individuals:+a+more+powerful+quasi-likelihood+score+test&amp;author=T+Thornton&amp;author=MS+McPeek&amp;volume=81&amp;publication_year=2007&amp;pages=321-337&amp;pmid=17668381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Tschanz JT, Welsh-Bohmer KA, Plassman BL, Norton MC, Wyse BW, Breitner JC Cache County Study Group. An adaptation of the modified mini-mental state examination: analysis of demographic influences and normative data: the cache county study. <span><span class="ref-journal">Neuropsychiatry Neuropsychol Behav Neurol. </span>2002;<span class="ref-vol">15</span>:28–38.</span> [<a href="/pubmed/11877549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychiatry+Neuropsychol+Behav+Neurol&amp;title=An+adaptation+of+the+modified+mini-mental+state+examination:+analysis+of+demographic+influences+and+normative+data:+the+cache+county+study&amp;author=JT+Tschanz&amp;author=KA+Welsh-Bohmer&amp;author=BL+Plassman&amp;author=MC+Norton&amp;author=BW+Wyse&amp;volume=15&amp;publication_year=2002&amp;pages=28-38&amp;pmid=11877549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Zhang Z, Hartmann H, Do VM, Abramowski D, Sturchler-Pierrat C, Staufenbiel M, Sommer B, van de Wetering M, Clevers H, Saftig P, De Strooper B, He X, Yankner BA. Destabilization of beta-catenin by mutations in presenilin-1 potentiates neuronal apoptosis. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">395</span>:698–702.</span> [<a href="/pubmed/9790190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Destabilization+of+beta-catenin+by+mutations+in+presenilin-1+potentiates+neuronal+apoptosis&amp;author=Z+Zhang&amp;author=H+Hartmann&amp;author=VM+Do&amp;author=D+Abramowski&amp;author=C+Sturchler-Pierrat&amp;volume=395&amp;publication_year=1998&amp;pages=698-702&amp;pmid=9790190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3419486&amp;issue-id=213039&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140628290398944title">Summary</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials &amp; Methods</a></li><li><a href="#S9title">Results</a></li><li><a href="#S12title">Discussion</a></li><li><a href="#S13title">Supplementary Material</a></li><li><a href="#S14title">Acknowledgments</a></li><li><a href="#idm140628251353872title">Footnotes</a></li><li><a href="#idm140628252715424title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3419486/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3419486/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3419486/pdf/nihms387665.pdf">PDF (494K)</a></li> | <li><a href="#" data-citationid="PMC3419486" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419486%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419486%2F&amp;text=Genome-wide%20association%20and%20linkage%20study%20in%20the%20Amish%20detects%20a%20novel%20candidate%20late-onset%20Alzheimer%20disease%20gene" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419486%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22881374" data-aiid="3419486" data-aid="3419486" data-iid="213039" data-domainid="319" data-domain="nihpa" data-accid="PMC3419486" data-md5="9f3530f63be5428adca6e56450e18f0a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:22:14-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B9065DD54A88100000000015900C7&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3419486%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3419486/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>