<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3869788/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common variants at SCN5A-SCN10A and HEY2 are associated with Brugada syndrome, a rare disease with high risk of sudden cardiac death</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3869788/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Common variants at SCN5A-SCN10A and HEY2 are associated with Brugada syndrome, a rare disease with high risk of sudden cardiac death"><meta name="citation_authors" content="Connie R. Bezzina, Julien Barc, Yuka Mizusawa, Carol Ann Remme, Jean-Baptiste Gourraud, Floriane Simonet, Arie O. Verkerk, Peter J. Schwartz, Lia Crotti, Federica Dagradi, Pascale Guicheney, Véronique Fressart, Antoine Leenhardt, Charles Antzelevitch, Susan Bartkowiak, Eric Schulze-Bahr, Sven Zumhagen, Elijah R. Behr, Rachel Bastiaenen, Jacob Tfelt-Hansen, Morten Salling Olesen, Stefan Kääb, Britt M. Beckmann, Peter Weeke, Hiroshi Watanabe, Naoto Endo, Tohru Minamino, Minoru Horie, Seiko Ohno, Kanae Hasegawa, Naomasa Makita, Akihiko Nogami, Wataru Shimizu, Takeshi Aiba, Philippe Froguel, Beverley Balkau, Olivier Lantieri, Margherita Torchio, Cornelia Wiese, David Weber, Rianne Wolswinkel, Ruben Coronel, Bas J. Boukens, Stéphane Bézieau, Eric Charpentier, Stéphanie Chatel, Aurore Despres, Françoise Gros, Florence Kyndt, Simon Lecointe, Pierre Lindenbaum, Vincent Portero, Jade Violleau, Manfred Gessler, Hanno L. Tan, Dan M. Roden, Vincent M. Christoffels, Hervé Le Marec, Arthur A Wilde, Vincent Probst, Jean-Jacques Schott, Christian Dina, Richard Redon"><meta name="citation_date" content="September 2013"><meta name="citation_issue" content="9"><meta name="citation_volume" content="45"><meta name="citation_firstpage" content="1044"><meta name="citation_doi" content="10.1038/ng.2712"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3869788/?report=abstract"><meta name="citation_pmid" content="23872634"><meta name="DC.Title" content="Common variants at SCN5A-SCN10A and HEY2 are associated with Brugada syndrome, a rare disease with high risk of sudden cardiac death"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Connie R. Bezzina"><meta name="DC.Contributor" content="Julien Barc"><meta name="DC.Contributor" content="Yuka Mizusawa"><meta name="DC.Contributor" content="Carol Ann Remme"><meta name="DC.Contributor" content="Jean-Baptiste Gourraud"><meta name="DC.Contributor" content="Floriane Simonet"><meta name="DC.Contributor" content="Arie O. Verkerk"><meta name="DC.Contributor" content="Peter J. Schwartz"><meta name="DC.Contributor" content="Lia Crotti"><meta name="DC.Contributor" content="Federica Dagradi"><meta name="DC.Contributor" content="Pascale Guicheney"><meta name="DC.Contributor" content="Véronique Fressart"><meta name="DC.Contributor" content="Antoine Leenhardt"><meta name="DC.Contributor" content="Charles Antzelevitch"><meta name="DC.Contributor" content="Susan Bartkowiak"><meta name="DC.Contributor" content="Eric Schulze-Bahr"><meta name="DC.Contributor" content="Sven Zumhagen"><meta name="DC.Contributor" content="Elijah R. Behr"><meta name="DC.Contributor" content="Rachel Bastiaenen"><meta name="DC.Contributor" content="Jacob Tfelt-Hansen"><meta name="DC.Contributor" content="Morten Salling Olesen"><meta name="DC.Contributor" content="Stefan Kääb"><meta name="DC.Contributor" content="Britt M. Beckmann"><meta name="DC.Contributor" content="Peter Weeke"><meta name="DC.Contributor" content="Hiroshi Watanabe"><meta name="DC.Contributor" content="Naoto Endo"><meta name="DC.Contributor" content="Tohru Minamino"><meta name="DC.Contributor" content="Minoru Horie"><meta name="DC.Contributor" content="Seiko Ohno"><meta name="DC.Contributor" content="Kanae Hasegawa"><meta name="DC.Contributor" content="Naomasa Makita"><meta name="DC.Contributor" content="Akihiko Nogami"><meta name="DC.Contributor" content="Wataru Shimizu"><meta name="DC.Contributor" content="Takeshi Aiba"><meta name="DC.Contributor" content="Philippe Froguel"><meta name="DC.Contributor" content="Beverley Balkau"><meta name="DC.Contributor" content="Olivier Lantieri"><meta name="DC.Contributor" content="Margherita Torchio"><meta name="DC.Contributor" content="Cornelia Wiese"><meta name="DC.Contributor" content="David Weber"><meta name="DC.Contributor" content="Rianne Wolswinkel"><meta name="DC.Contributor" content="Ruben Coronel"><meta name="DC.Contributor" content="Bas J. Boukens"><meta name="DC.Contributor" content="Stéphane Bézieau"><meta name="DC.Contributor" content="Eric Charpentier"><meta name="DC.Contributor" content="Stéphanie Chatel"><meta name="DC.Contributor" content="Aurore Despres"><meta name="DC.Contributor" content="Françoise Gros"><meta name="DC.Contributor" content="Florence Kyndt"><meta name="DC.Contributor" content="Simon Lecointe"><meta name="DC.Contributor" content="Pierre Lindenbaum"><meta name="DC.Contributor" content="Vincent Portero"><meta name="DC.Contributor" content="Jade Violleau"><meta name="DC.Contributor" content="Manfred Gessler"><meta name="DC.Contributor" content="Hanno L. Tan"><meta name="DC.Contributor" content="Dan M. Roden"><meta name="DC.Contributor" content="Vincent M. Christoffels"><meta name="DC.Contributor" content="Hervé Le Marec"><meta name="DC.Contributor" content="Arthur A Wilde"><meta name="DC.Contributor" content="Vincent Probst"><meta name="DC.Contributor" content="Jean-Jacques Schott"><meta name="DC.Contributor" content="Christian Dina"><meta name="DC.Contributor" content="Richard Redon"><meta name="DC.Date" content="2013 Sep"><meta name="DC.Identifier" content="10.1038/ng.2712"><meta name="DC.Language" content="en"><meta property="og:title" content="Common variants at SCN5A-SCN10A and HEY2 are associated with Brugada syndrome, a rare disease with high risk of sudden cardiac death"><meta property="og:type" content="article"><meta property="og:description" content="Brugada syndrome is a rare cardiac arrhythmia disorder, causally related to SCN5A mutations in around 20% of cases[–] . Through a genome-wide association study of 312 individuals with Brugada syndrome and 1,115 controls, we detected 2 significant ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3869788/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3869788/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3869788/pdf/nihms528125.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CBF6CDD555CE1000000000B3506E6.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/jax.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3869788%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3869788</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2014 Sep 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23872634" target="pmc_ext" ref="reftype=publisher&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2013 Sep; 45(9): 1044–1049. </span></a></div><span class="fm-vol-iss-date">Published online 2013 Jul 21. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.2712" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.2712</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3869788</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS528125</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23872634">23872634</a></div></div></div></div><h1 class="content-title">Common variants at <em>SCN5A</em>-<em>SCN10A</em> and <em>HEY2</em> are associated with Brugada syndrome, a rare disease with high risk of sudden cardiac death</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Bezzina%20CR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688899710080" co-class="co-affbox">Connie R. Bezzina</a>,<sup>1,</sup><sup>44</sup> <a href="/pubmed/?term=Barc%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898575840" co-class="co-affbox">Julien Barc</a>,<sup>1,</sup><sup>2,</sup><sup>44</sup> <a href="/pubmed/?term=Mizusawa%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688899707968" co-class="co-affbox">Yuka Mizusawa</a>,<sup>1,</sup><sup>44</sup> <a href="/pubmed/?term=Remme%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688899705248" co-class="co-affbox">Carol Ann Remme</a>,<sup>1,</sup><sup>44</sup> <a href="/pubmed/?term=Gourraud%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688899702528" co-class="co-affbox">Jean-Baptiste Gourraud</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>44</sup> <a href="/pubmed/?term=Simonet%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688901225872" co-class="co-affbox">Floriane Simonet</a>,<sup>3,</sup><sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Verkerk%20AO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688901222416" co-class="co-affbox">Arie O. Verkerk</a>,<sup>7</sup> <a href="/pubmed/?term=Schwartz%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688901220432" co-class="co-affbox">Peter J. Schwartz</a>,<sup>8,</sup><sup>9,</sup><sup>10,</sup><sup>11,</sup><sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Crotti%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688901214768" co-class="co-affbox">Lia Crotti</a>,<sup>8,</sup><sup>9,</sup><sup>14</sup> <a href="/pubmed/?term=Dagradi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906438256" co-class="co-affbox">Federica Dagradi</a>,<sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Guicheney%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906435536" co-class="co-affbox">Pascale Guicheney</a>,<sup>15,</sup><sup>16</sup> <a href="/pubmed/?term=Fressart%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906432816" co-class="co-affbox">Véronique Fressart</a>,<sup>15,</sup><sup>16,</sup><sup>17</sup> <a href="/pubmed/?term=Leenhardt%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906429360" co-class="co-affbox">Antoine Leenhardt</a>,<sup>18,</sup><sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Antzelevitch%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906425904" co-class="co-affbox">Charles Antzelevitch</a>,<sup>21</sup> <a href="/pubmed/?term=Bartkowiak%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906423920" co-class="co-affbox">Susan Bartkowiak</a>,<sup>21</sup> <a href="/pubmed/?term=Schulze-Bahr%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688905784304" co-class="co-affbox">Eric Schulze-Bahr</a>,<sup>22</sup> <a href="/pubmed/?term=Zumhagen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688905782320" co-class="co-affbox">Sven Zumhagen</a>,<sup>22</sup> <a href="/pubmed/?term=Behr%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688905780336" co-class="co-affbox">Elijah R. Behr</a>,<sup>23</sup> <a href="/pubmed/?term=Bastiaenen%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688905778352" co-class="co-affbox">Rachel Bastiaenen</a>,<sup>23</sup> <a href="/pubmed/?term=Tfelt-Hansen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688905776368" co-class="co-affbox">Jacob Tfelt-Hansen</a>,<sup>24,</sup><sup>25</sup> <a href="/pubmed/?term=Olesen%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688905773648" co-class="co-affbox">Morten Salling Olesen</a>,<sup>24,</sup><sup>25</sup> <a href="/pubmed/?term=K%26%23x000e4%3B%26%23x000e4%3Bb%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688905770928" co-class="co-affbox">Stefan Kääb</a>,<sup>26,</sup><sup>27</sup> <a href="/pubmed/?term=Beckmann%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688977988864" co-class="co-affbox">Britt M. Beckmann</a>,<sup>26</sup> <a href="/pubmed/?term=Weeke%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688977986880" co-class="co-affbox">Peter Weeke</a>,<sup>28</sup> <a href="/pubmed/?term=Watanabe%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688977984896" co-class="co-affbox">Hiroshi Watanabe</a>,<sup>29</sup> <a href="/pubmed/?term=Endo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688977982912" co-class="co-affbox">Naoto Endo</a>,<sup>30</sup> <a href="/pubmed/?term=Minamino%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688977980928" co-class="co-affbox">Tohru Minamino</a>,<sup>29</sup> <a href="/pubmed/?term=Horie%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688977978944" co-class="co-affbox">Minoru Horie</a>,<sup>31</sup> <a href="/pubmed/?term=Ohno%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688977976960" co-class="co-affbox">Seiko Ohno</a>,<sup>31</sup> <a href="/pubmed/?term=Hasegawa%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898633328" co-class="co-affbox">Kanae Hasegawa</a>,<sup>31</sup> <a href="/pubmed/?term=Makita%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898631344" co-class="co-affbox">Naomasa Makita</a>,<sup>32</sup> <a href="/pubmed/?term=Nogami%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898629360" co-class="co-affbox">Akihiko Nogami</a>,<sup>33</sup> <a href="/pubmed/?term=Shimizu%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898627376" co-class="co-affbox">Wataru Shimizu</a>,<sup>34,</sup><sup>35</sup> <a href="/pubmed/?term=Aiba%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898624656" co-class="co-affbox">Takeshi Aiba</a>,<sup>35</sup> <a href="/pubmed/?term=Froguel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898622672" co-class="co-affbox">Philippe Froguel</a>,<sup>36,</sup><sup>37,</sup><sup>38</sup> <a href="/pubmed/?term=Balkau%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898619216" co-class="co-affbox">Beverley Balkau</a>,<sup>39,</sup><sup>40</sup> <a href="/pubmed/?term=Lantieri%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688908541824" co-class="co-affbox">Olivier Lantieri</a>,<sup>41</sup> <a href="/pubmed/?term=Torchio%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688908539840" co-class="co-affbox">Margherita Torchio</a>,<sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Wiese%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688908537120" co-class="co-affbox">Cornelia Wiese</a>,<sup>42</sup> <a href="/pubmed/?term=Weber%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688908535136" co-class="co-affbox">David Weber</a>,<sup>42</sup> <a href="/pubmed/?term=Wolswinkel%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688908533152" co-class="co-affbox">Rianne Wolswinkel</a>,<sup>1</sup> <a href="/pubmed/?term=Coronel%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688908531168" co-class="co-affbox">Ruben Coronel</a>,<sup>1</sup> <a href="/pubmed/?term=Boukens%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688908529184" co-class="co-affbox">Bas J. Boukens</a>,<sup>1,</sup><sup>7</sup> <a href="/pubmed/?term=B%26%23x000e9%3Bzieau%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688908526464" co-class="co-affbox">Stéphane Bézieau</a>,<sup>43</sup> <a href="/pubmed/?term=Charpentier%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688899890496" co-class="co-affbox">Eric Charpentier</a>,<sup>3,</sup><sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Chatel%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688899887040" co-class="co-affbox">Stéphanie Chatel</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Despres%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688899882848" co-class="co-affbox">Aurore Despres</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Gros%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688899878656" co-class="co-affbox">Françoise Gros</a>,<sup>3,</sup><sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Kyndt%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688907168064" co-class="co-affbox">Florence Kyndt</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>43</sup> <a href="/pubmed/?term=Lecointe%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688907163136" co-class="co-affbox">Simon Lecointe</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Lindenbaum%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688907158944" co-class="co-affbox">Pierre Lindenbaum</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Portero%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688907154752" co-class="co-affbox">Vincent Portero</a>,<sup>3,</sup><sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Violleau%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898615984" co-class="co-affbox">Jade Violleau</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Gessler%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898611792" co-class="co-affbox">Manfred Gessler</a>,<sup>42</sup> <a href="/pubmed/?term=Tan%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898609808" co-class="co-affbox">Hanno L. Tan</a>,<sup>1</sup> <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898607824" co-class="co-affbox">Dan M. Roden</a>,<sup>28</sup> <a href="/pubmed/?term=Christoffels%20VM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898605840" co-class="co-affbox">Vincent M. Christoffels</a>,<sup>7</sup> <a href="/pubmed/?term=Le%20Marec%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688898603856" co-class="co-affbox">Hervé Le Marec</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Wilde%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906578816" co-class="co-affbox">Arthur A Wilde</a>,<sup>1,</sup><sup>44</sup> <a href="/pubmed/?term=Probst%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906576096" co-class="co-affbox">Vincent Probst</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>44</sup> <a href="/pubmed/?term=Schott%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906571168" co-class="co-affbox">Jean-Jacques Schott</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>44</sup> <a href="/pubmed/?term=Dina%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688906566240" co-class="co-affbox">Christian Dina</a>,<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>44</sup> and  <a href="/pubmed/?term=Redon%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634" class="affpopup" co-rid="_co_idm139688905491552" co-class="co-affbox">Richard Redon</a><sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>44</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139688899710080"><h3 class="no_margin">Connie R. Bezzina</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Bezzina%20CR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Connie R. Bezzina</a></div></div><div id="_co_idm139688898575840"><h3 class="no_margin">Julien Barc</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><p><sup>2</sup>ICIN (Netherlands Heart Institute), Utrecht, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Barc%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Julien Barc</a></div></div><div id="_co_idm139688899707968"><h3 class="no_margin">Yuka Mizusawa</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Mizusawa%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Yuka Mizusawa</a></div></div><div id="_co_idm139688899705248"><h3 class="no_margin">Carol Ann Remme</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Remme%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Carol Ann Remme</a></div></div><div id="_co_idm139688899702528"><h3 class="no_margin">Jean-Baptiste Gourraud</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Gourraud%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Jean-Baptiste Gourraud</a></div></div><div id="_co_idm139688901225872"><h3 class="no_margin">Floriane Simonet</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Simonet%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Floriane Simonet</a></div></div><div id="_co_idm139688901222416"><h3 class="no_margin">Arie O. Verkerk</h3><p><sup>7</sup>Department of Anatomy, Embryology and Physiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Verkerk%20AO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Arie O. Verkerk</a></div></div><div id="_co_idm139688901220432"><h3 class="no_margin">Peter J. Schwartz</h3><p><sup>8</sup>Department of Molecular Medicine, University of Pavia, Pavia, Italy</p><p><sup>9</sup>Center for Cardiac Arrhythmias of Genetic Origin, Istituto di Ricovero e Cura a Carattere Scientifico (IRCCS), Istituto Auxologico Italiano, Milan, Italy</p><p><sup>10</sup>Cardiovascular Genetics Laboratory, Hatter Institute for Cardiovascular Research in Africa, Cape Town, South Africa</p><p><sup>11</sup>Department of Medicine, University of Cape Town, Cape Town, South Africa</p><p><sup>12</sup>Department of Medicine, University of Stellenbosch, Stellenbosch, South Africa</p><p><sup>13</sup>Chair of Sudden Death, Department of Family and Community Medicine, College of Medicine, King Saud University, Riyadh, Saudi Arabia</p><div>Find articles by <a href="/pubmed/?term=Schwartz%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Peter J. Schwartz</a></div></div><div id="_co_idm139688901214768"><h3 class="no_margin">Lia Crotti</h3><p><sup>8</sup>Department of Molecular Medicine, University of Pavia, Pavia, Italy</p><p><sup>9</sup>Center for Cardiac Arrhythmias of Genetic Origin, Istituto di Ricovero e Cura a Carattere Scientifico (IRCCS), Istituto Auxologico Italiano, Milan, Italy</p><p><sup>14</sup>Institute of Human Genetics, Helmholtz Center Munich, Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Crotti%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Lia Crotti</a></div></div><div id="_co_idm139688906438256"><h3 class="no_margin">Federica Dagradi</h3><p><sup>8</sup>Department of Molecular Medicine, University of Pavia, Pavia, Italy</p><p><sup>9</sup>Center for Cardiac Arrhythmias of Genetic Origin, Istituto di Ricovero e Cura a Carattere Scientifico (IRCCS), Istituto Auxologico Italiano, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Dagradi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Federica Dagradi</a></div></div><div id="_co_idm139688906435536"><h3 class="no_margin">Pascale Guicheney</h3><p><sup>15</sup>INSERM U956, Faculté de Médecine Pierre et Marie Curie, Site Pitié-Salpêtrière, Paris, France</p><p><sup>16</sup>Université Pierre et Marie Curie Université Paris 06, Unité de Mixte Recherche Scientifique (UMRS) 956, Institut Fédératif de Recherche 14, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Guicheney%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Pascale Guicheney</a></div></div><div id="_co_idm139688906432816"><h3 class="no_margin">Véronique Fressart</h3><p><sup>15</sup>INSERM U956, Faculté de Médecine Pierre et Marie Curie, Site Pitié-Salpêtrière, Paris, France</p><p><sup>16</sup>Université Pierre et Marie Curie Université Paris 06, Unité de Mixte Recherche Scientifique (UMRS) 956, Institut Fédératif de Recherche 14, Paris, France</p><p><sup>17</sup>Assistance Publique–Hôpitaux de Paris (AP-HP), Groupe Hospitalier Pitié-Salpétrière, Service de Biochimie Métabolique, Unité Fonctionnelle Cardiogénétique et Myogénétique Moléculaire et Cellulaire, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Fressart%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Véronique Fressart</a></div></div><div id="_co_idm139688906429360"><h3 class="no_margin">Antoine Leenhardt</h3><p><sup>18</sup>Service de Cardiologie, Centre de Référence des Maladies Cardiaques Héréditaires, Hôpital Bichat, AP-HP, Paris, France</p><p><sup>19</sup>INSERM U698, Paris, France</p><p><sup>20</sup>Université Paris Diderot, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Leenhardt%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Antoine Leenhardt</a></div></div><div id="_co_idm139688906425904"><h3 class="no_margin">Charles Antzelevitch</h3><p><sup>21</sup>Department of Experimental Cardiology, Masonic Medical Research Laboratory, Utica, New York, USA</p><div>Find articles by <a href="/pubmed/?term=Antzelevitch%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Charles Antzelevitch</a></div></div><div id="_co_idm139688906423920"><h3 class="no_margin">Susan Bartkowiak</h3><p><sup>21</sup>Department of Experimental Cardiology, Masonic Medical Research Laboratory, Utica, New York, USA</p><div>Find articles by <a href="/pubmed/?term=Bartkowiak%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Susan Bartkowiak</a></div></div><div id="_co_idm139688905784304"><h3 class="no_margin">Eric Schulze-Bahr</h3><p><sup>22</sup>Institute for Genetics of Heart Diseases (IfGH), Department of Cardiovascular Medicine, University Hospital, Münster, Germany</p><div>Find articles by <a href="/pubmed/?term=Schulze-Bahr%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Eric Schulze-Bahr</a></div></div><div id="_co_idm139688905782320"><h3 class="no_margin">Sven Zumhagen</h3><p><sup>22</sup>Institute for Genetics of Heart Diseases (IfGH), Department of Cardiovascular Medicine, University Hospital, Münster, Germany</p><div>Find articles by <a href="/pubmed/?term=Zumhagen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Sven Zumhagen</a></div></div><div id="_co_idm139688905780336"><h3 class="no_margin">Elijah R. Behr</h3><p><sup>23</sup>Cardiovascular Sciences Research Centre, St George’s University of London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Behr%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Elijah R. Behr</a></div></div><div id="_co_idm139688905778352"><h3 class="no_margin">Rachel Bastiaenen</h3><p><sup>23</sup>Cardiovascular Sciences Research Centre, St George’s University of London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Bastiaenen%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Rachel Bastiaenen</a></div></div><div id="_co_idm139688905776368"><h3 class="no_margin">Jacob Tfelt-Hansen</h3><p><sup>24</sup>Danish National Research Foundation Centre for Cardiac Arrhythmia (DARC), Laboratory of Molecular Cardiology, University of Copenhagen, Copenhagen, Denmark</p><p><sup>25</sup>Department of Cardiology, The Heart Centre, Copenhagen University Hospital Rigshospitalet, Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=Tfelt-Hansen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Jacob Tfelt-Hansen</a></div></div><div id="_co_idm139688905773648"><h3 class="no_margin">Morten Salling Olesen</h3><p><sup>24</sup>Danish National Research Foundation Centre for Cardiac Arrhythmia (DARC), Laboratory of Molecular Cardiology, University of Copenhagen, Copenhagen, Denmark</p><p><sup>25</sup>Department of Cardiology, The Heart Centre, Copenhagen University Hospital Rigshospitalet, Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=Olesen%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Morten Salling Olesen</a></div></div><div id="_co_idm139688905770928"><h3 class="no_margin">Stefan Kääb</h3><p><sup>26</sup>Department of Medicine I, University Hospital Munich, Campus Grosshadern, Ludwig-Maximilians University, Munich, Germany</p><p><sup>27</sup>Deutsches Forschungszentrum für Herz-Kreislauferkrankungen (DZHK), partner site Munich Heart Alliance, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000e4%3B%26%23x000e4%3Bb%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Stefan Kääb</a></div></div><div id="_co_idm139688977988864"><h3 class="no_margin">Britt M. Beckmann</h3><p><sup>26</sup>Department of Medicine I, University Hospital Munich, Campus Grosshadern, Ludwig-Maximilians University, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Beckmann%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Britt M. Beckmann</a></div></div><div id="_co_idm139688977986880"><h3 class="no_margin">Peter Weeke</h3><p><sup>28</sup>Department of Medicine and Pharmacology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Weeke%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Peter Weeke</a></div></div><div id="_co_idm139688977984896"><h3 class="no_margin">Hiroshi Watanabe</h3><p><sup>29</sup>Department of Cardiovascular Biology and Medicine, Niigata University Graduate School of Medical and Dental Sciences, Niigata, Japan</p><div>Find articles by <a href="/pubmed/?term=Watanabe%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Hiroshi Watanabe</a></div></div><div id="_co_idm139688977982912"><h3 class="no_margin">Naoto Endo</h3><p><sup>30</sup>Department of Regenerative and Transplant Medicine, Division of Orthopaedic Surgery, Niigata University Graduate School of Medicine and Dental Science, Niigata, Japan</p><div>Find articles by <a href="/pubmed/?term=Endo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Naoto Endo</a></div></div><div id="_co_idm139688977980928"><h3 class="no_margin">Tohru Minamino</h3><p><sup>29</sup>Department of Cardiovascular Biology and Medicine, Niigata University Graduate School of Medical and Dental Sciences, Niigata, Japan</p><div>Find articles by <a href="/pubmed/?term=Minamino%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Tohru Minamino</a></div></div><div id="_co_idm139688977978944"><h3 class="no_margin">Minoru Horie</h3><p><sup>31</sup>Department of Cardiovascular and Respiratory Medicine, Shiga University of Medical Science, Otsu, Japan</p><div>Find articles by <a href="/pubmed/?term=Horie%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Minoru Horie</a></div></div><div id="_co_idm139688977976960"><h3 class="no_margin">Seiko Ohno</h3><p><sup>31</sup>Department of Cardiovascular and Respiratory Medicine, Shiga University of Medical Science, Otsu, Japan</p><div>Find articles by <a href="/pubmed/?term=Ohno%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Seiko Ohno</a></div></div><div id="_co_idm139688898633328"><h3 class="no_margin">Kanae Hasegawa</h3><p><sup>31</sup>Department of Cardiovascular and Respiratory Medicine, Shiga University of Medical Science, Otsu, Japan</p><div>Find articles by <a href="/pubmed/?term=Hasegawa%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Kanae Hasegawa</a></div></div><div id="_co_idm139688898631344"><h3 class="no_margin">Naomasa Makita</h3><p><sup>32</sup>Department of Molecular Physiology, Graduate School of Biomedical Sciences, Nagasaki University, Nagasaki, Japan</p><div>Find articles by <a href="/pubmed/?term=Makita%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Naomasa Makita</a></div></div><div id="_co_idm139688898629360"><h3 class="no_margin">Akihiko Nogami</h3><p><sup>33</sup>Division of Heart Rhythm Management, Yokohama Rosai Hospital, Yokohama, Japan</p><div>Find articles by <a href="/pubmed/?term=Nogami%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Akihiko Nogami</a></div></div><div id="_co_idm139688898627376"><h3 class="no_margin">Wataru Shimizu</h3><p><sup>34</sup>Department of Cardiovascular Medicine, Nippon Medical School, Tokyo, Japan</p><p><sup>35</sup>Department of Cardiovascular Medicine, National Cerebral and Cardiovascular Center, Division of Arrhythmia and Electrophysiology, Suita, Japan</p><div>Find articles by <a href="/pubmed/?term=Shimizu%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Wataru Shimizu</a></div></div><div id="_co_idm139688898624656"><h3 class="no_margin">Takeshi Aiba</h3><p><sup>35</sup>Department of Cardiovascular Medicine, National Cerebral and Cardiovascular Center, Division of Arrhythmia and Electrophysiology, Suita, Japan</p><div>Find articles by <a href="/pubmed/?term=Aiba%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Takeshi Aiba</a></div></div><div id="_co_idm139688898622672"><h3 class="no_margin">Philippe Froguel</h3><p><sup>36</sup>CNRS UMR 8199, Lille Pasteur Institute, Lille, France</p><p><sup>37</sup>Lille Nord de France University, Lille, France</p><p><sup>38</sup>Department of Genomics of Common Disease, School of Public Health, Imperial College London, Hammersmith Hospital, London, UK</p><div>Find articles by <a href="/pubmed/?term=Froguel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Philippe Froguel</a></div></div><div id="_co_idm139688898619216"><h3 class="no_margin">Beverley Balkau</h3><p><sup>39</sup>INSERM U1018, Centre de Recherche en Epidémiologie et Santé des Populations (CESP), Villejuif, France</p><p><sup>40</sup>Université Paris Sud, UMRS 1018, Villejuif, France</p><div>Find articles by <a href="/pubmed/?term=Balkau%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Beverley Balkau</a></div></div><div id="_co_idm139688908541824"><h3 class="no_margin">Olivier Lantieri</h3><p><sup>41</sup>Institut Inter-Régional pour la Santé (IRSA), La Riche, France</p><div>Find articles by <a href="/pubmed/?term=Lantieri%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Olivier Lantieri</a></div></div><div id="_co_idm139688908539840"><h3 class="no_margin">Margherita Torchio</h3><p><sup>8</sup>Department of Molecular Medicine, University of Pavia, Pavia, Italy</p><p><sup>9</sup>Center for Cardiac Arrhythmias of Genetic Origin, Istituto di Ricovero e Cura a Carattere Scientifico (IRCCS), Istituto Auxologico Italiano, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Torchio%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Margherita Torchio</a></div></div><div id="_co_idm139688908537120"><h3 class="no_margin">Cornelia Wiese</h3><p><sup>42</sup>Developmental Biochemistry, Theodor-Boveri-Institute, Biocenter, University of Würzburg, Würzburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Wiese%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Cornelia Wiese</a></div></div><div id="_co_idm139688908535136"><h3 class="no_margin">David Weber</h3><p><sup>42</sup>Developmental Biochemistry, Theodor-Boveri-Institute, Biocenter, University of Würzburg, Würzburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Weber%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">David Weber</a></div></div><div id="_co_idm139688908533152"><h3 class="no_margin">Rianne Wolswinkel</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Wolswinkel%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Rianne Wolswinkel</a></div></div><div id="_co_idm139688908531168"><h3 class="no_margin">Ruben Coronel</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Coronel%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Ruben Coronel</a></div></div><div id="_co_idm139688908529184"><h3 class="no_margin">Bas J. Boukens</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><p><sup>7</sup>Department of Anatomy, Embryology and Physiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Boukens%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Bas J. Boukens</a></div></div><div id="_co_idm139688908526464"><h3 class="no_margin">Stéphane Bézieau</h3><p><sup>43</sup>CHU Nantes, Service de Génétique Médicale, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=B%26%23x000e9%3Bzieau%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Stéphane Bézieau</a></div></div><div id="_co_idm139688899890496"><h3 class="no_margin">Eric Charpentier</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Charpentier%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Eric Charpentier</a></div></div><div id="_co_idm139688899887040"><h3 class="no_margin">Stéphanie Chatel</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Chatel%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Stéphanie Chatel</a></div></div><div id="_co_idm139688899882848"><h3 class="no_margin">Aurore Despres</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Despres%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Aurore Despres</a></div></div><div id="_co_idm139688899878656"><h3 class="no_margin">Françoise Gros</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Gros%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Françoise Gros</a></div></div><div id="_co_idm139688907168064"><h3 class="no_margin">Florence Kyndt</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><p><sup>43</sup>CHU Nantes, Service de Génétique Médicale, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Kyndt%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Florence Kyndt</a></div></div><div id="_co_idm139688907163136"><h3 class="no_margin">Simon Lecointe</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Lecointe%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Simon Lecointe</a></div></div><div id="_co_idm139688907158944"><h3 class="no_margin">Pierre Lindenbaum</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Lindenbaum%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Pierre Lindenbaum</a></div></div><div id="_co_idm139688907154752"><h3 class="no_margin">Vincent Portero</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Portero%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Vincent Portero</a></div></div><div id="_co_idm139688898615984"><h3 class="no_margin">Jade Violleau</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Violleau%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Jade Violleau</a></div></div><div id="_co_idm139688898611792"><h3 class="no_margin">Manfred Gessler</h3><p><sup>42</sup>Developmental Biochemistry, Theodor-Boveri-Institute, Biocenter, University of Würzburg, Würzburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Gessler%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Manfred Gessler</a></div></div><div id="_co_idm139688898609808"><h3 class="no_margin">Hanno L. Tan</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Tan%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Hanno L. Tan</a></div></div><div id="_co_idm139688898607824"><h3 class="no_margin">Dan M. Roden</h3><p><sup>28</sup>Department of Medicine and Pharmacology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Dan M. Roden</a></div></div><div id="_co_idm139688898605840"><h3 class="no_margin">Vincent M. Christoffels</h3><p><sup>7</sup>Department of Anatomy, Embryology and Physiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Christoffels%20VM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Vincent M. Christoffels</a></div></div><div id="_co_idm139688898603856"><h3 class="no_margin">Hervé Le Marec</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Le%20Marec%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Hervé Le Marec</a></div></div><div id="_co_idm139688906578816"><h3 class="no_margin">Arthur A Wilde</h3><p><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Wilde%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Arthur A Wilde</a></div></div><div id="_co_idm139688906576096"><h3 class="no_margin">Vincent Probst</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Probst%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Vincent Probst</a></div></div><div id="_co_idm139688906571168"><h3 class="no_margin">Jean-Jacques Schott</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Schott%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Jean-Jacques Schott</a></div></div><div id="_co_idm139688906566240"><h3 class="no_margin">Christian Dina</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Dina%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Christian Dina</a></div></div><div id="_co_idm139688905491552"><h3 class="no_margin">Richard Redon</h3><p><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</p><p><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</p><p><sup>5</sup>Université de Nantes, Nantes, France</p><p><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=Redon%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23872634">Richard Redon</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139688906498896_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139688906498896_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139688906498896_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Clinical and Experimental Cardiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>ICIN (Netherlands Heart Institute), Utrecht, The Netherlands</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Institut National de la Santé et de la Recherche Médicale (INSERM) Unité Mixte de Recherche (UMR) 1087, L’Institut du Thorax, Nantes, France</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Centre National de la Recherche Scientifique (CNRS) UMR 6291, Nantes, France</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Université de Nantes, Nantes, France</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Centre Hospitalier Universitaire (CHU) Nantes, L’Institut du Thorax, Service de Cardiologie, Nantes, France</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Anatomy, Embryology and Physiology, Heart Failure Research Center, Academic Medical Center, Amsterdam, The Netherlands</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Molecular Medicine, University of Pavia, Pavia, Italy</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Center for Cardiac Arrhythmias of Genetic Origin, Istituto di Ricovero e Cura a Carattere Scientifico (IRCCS), Istituto Auxologico Italiano, Milan, Italy</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Cardiovascular Genetics Laboratory, Hatter Institute for Cardiovascular Research in Africa, Cape Town, South Africa</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Medicine, University of Cape Town, Cape Town, South Africa</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Medicine, University of Stellenbosch, Stellenbosch, South Africa</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Chair of Sudden Death, Department of Family and Community Medicine, College of Medicine, King Saud University, Riyadh, Saudi Arabia</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Institute of Human Genetics, Helmholtz Center Munich, Neuherberg, Germany</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>INSERM U956, Faculté de Médecine Pierre et Marie Curie, Site Pitié-Salpêtrière, Paris, France</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Université Pierre et Marie Curie Université Paris 06, Unité de Mixte Recherche Scientifique (UMRS) 956, Institut Fédératif de Recherche 14, Paris, France</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Assistance Publique–Hôpitaux de Paris (AP-HP), Groupe Hospitalier Pitié-Salpétrière, Service de Biochimie Métabolique, Unité Fonctionnelle Cardiogénétique et Myogénétique Moléculaire et Cellulaire, Paris, France</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Service de Cardiologie, Centre de Référence des Maladies Cardiaques Héréditaires, Hôpital Bichat, AP-HP, Paris, France</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>INSERM U698, Paris, France</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Université Paris Diderot, Paris, France</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Department of Experimental Cardiology, Masonic Medical Research Laboratory, Utica, New York, USA</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Institute for Genetics of Heart Diseases (IfGH), Department of Cardiovascular Medicine, University Hospital, Münster, Germany</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Cardiovascular Sciences Research Centre, St George’s University of London, London, UK</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Danish National Research Foundation Centre for Cardiac Arrhythmia (DARC), Laboratory of Molecular Cardiology, University of Copenhagen, Copenhagen, Denmark</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Department of Cardiology, The Heart Centre, Copenhagen University Hospital Rigshospitalet, Copenhagen, Denmark</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Medicine I, University Hospital Munich, Campus Grosshadern, Ludwig-Maximilians University, Munich, Germany</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Deutsches Forschungszentrum für Herz-Kreislauferkrankungen (DZHK), partner site Munich Heart Alliance, Munich, Germany</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Department of Medicine and Pharmacology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Department of Cardiovascular Biology and Medicine, Niigata University Graduate School of Medical and Dental Sciences, Niigata, Japan</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Department of Regenerative and Transplant Medicine, Division of Orthopaedic Surgery, Niigata University Graduate School of Medicine and Dental Science, Niigata, Japan</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Department of Cardiovascular and Respiratory Medicine, Shiga University of Medical Science, Otsu, Japan</div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Department of Molecular Physiology, Graduate School of Biomedical Sciences, Nagasaki University, Nagasaki, Japan</div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Division of Heart Rhythm Management, Yokohama Rosai Hospital, Yokohama, Japan</div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Department of Cardiovascular Medicine, Nippon Medical School, Tokyo, Japan</div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>Department of Cardiovascular Medicine, National Cerebral and Cardiovascular Center, Division of Arrhythmia and Electrophysiology, Suita, Japan</div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>CNRS UMR 8199, Lille Pasteur Institute, Lille, France</div><div class="fm-affl" id="A37" lang="en"><sup>37</sup>Lille Nord de France University, Lille, France</div><div class="fm-affl" id="A38" lang="en"><sup>38</sup>Department of Genomics of Common Disease, School of Public Health, Imperial College London, Hammersmith Hospital, London, UK</div><div class="fm-affl" id="A39" lang="en"><sup>39</sup>INSERM U1018, Centre de Recherche en Epidémiologie et Santé des Populations (CESP), Villejuif, France</div><div class="fm-affl" id="A40" lang="en"><sup>40</sup>Université Paris Sud, UMRS 1018, Villejuif, France</div><div class="fm-affl" id="A41" lang="en"><sup>41</sup>Institut Inter-Régional pour la Santé (IRSA), La Riche, France</div><div class="fm-affl" id="A42" lang="en"><sup>42</sup>Developmental Biochemistry, Theodor-Boveri-Institute, Biocenter, University of Würzburg, Würzburg, Germany</div><div class="fm-affl" id="A43" lang="en"><sup>43</sup>CHU Nantes, Service de Génétique Médicale, Nantes, France</div><div id="FN1">Correspondence should be addressed to C.R.B. (<a href="mailto:dev@null" data-email="ln.avu.cma@anizzeb.r.c" class="oemail">ln.avu.cma@anizzeb.r.c</a>) or R.R. (<a href="mailto:dev@null" data-email="rf.mresni@noder.drahcir" class="oemail">rf.mresni@noder.drahcir</a>)</div><div id="FN2"><sup>44</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139688906498896_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23872634" target="pmc_ext" ref="reftype=publisher&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div><strong>This article has been corrected. </strong>See the correction in volume 45 on&nbsp;page&nbsp;1409.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3869788/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Corrigendum.</strong></div><div><a href="/pmc/articles/PMC3869788/bin/NIHMS528125-supplement-Corrigendum.pdf" data-ga-action="click_feat_suppl">NIHMS528125-supplement-Corrigendum.pdf</a><span style="color:gray"> (79K)</span></div><div class="small guid">GUID:&nbsp;443CFC68-A34B-4E50-9DDD-060E785CD507</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>OnlineSuppl.</strong></div><div><a href="/pmc/articles/PMC3869788/bin/NIHMS528125-supplement-OnlineSuppl.pdf" data-ga-action="click_feat_suppl">NIHMS528125-supplement-OnlineSuppl.pdf</a><span style="color:gray"> (5.8M)</span></div><div class="small guid">GUID:&nbsp;6586A774-9DE3-4764-9846-61AE4BCEE1F3</div></div></dd></dl></div><div id="idm139688907878576" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139688907878576title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Brugada syndrome is a rare cardiac arrhythmia disorder, causally related to <em>SCN5A</em> mutations in around 20% of cases<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263885">1</a>–<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263871">3</a></sup>. Through a genome-wide association study of 312 individuals with Brugada syndrome and 1,115 controls, we detected 2 significant association signals at the <em>SCN10A</em> locus (rs10428132) and near the <em>HEY2</em> gene (rs9388451). Independent replication confirmed both signals (meta-analyses: rs10428132, <em>P</em> = 1.0 × 10<sup>−68</sup>; rs9388451, <em>P</em> = 5.1 × 10<sup>−17</sup>) and identified one additional signal in <em>SCN5A</em> (at 3p21; rs11708996, <em>P</em> = 1.0 × 10<sup>−14</sup>). The cumulative effect of the three loci on disease susceptibility was unexpectedly large (<em>P</em><sub>trend</sub> = 6.1 × 10<sup>−81</sup>). The association signals at <em>SCN5A</em>-<em>SCN10A</em> demonstrate that genetic polymorphisms modulating cardiac conduction<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263894">4</a>–<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263877">7</a></sup> can also influence susceptibility to cardiac arrhythmia. The implication of association with <em>HEY2</em>, supported by new evidence that Hey2 regulates cardiac electrical activity, shows that Brugada syndrome may originate from altered transcriptional programming during cardiac development<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263841">8</a></sup>. Altogether, our findings indicate that common genetic variation can have a strong impact on the predisposition to rare diseases.</p></div></div><div id="idm139688905731664" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Sudden cardiac death (SCD) is a leading cause of mortality in Western countries, with an incidence close to 1 per 1,000 individuals per year<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263907">9</a></sup>. SCD results most frequently from ventricular fibrillation in the setting of coronary artery disease<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397606633">10</a></sup>. In 5–10% of cases, however, SCD occurs owing to rare inherited cardiac arrhythmias, which are typically associated with a distinctive electrocardiogram (ECG) pattern in the absence of identifiable structural heart disease<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397606632">10</a></sup>. One such disorder is Brugada syndrome, characterized by ST-segment elevation in right precordial ECG leads<sup>2</sup>. ST-segment elevation may be transient in nature and can be evoked by pharmacological sodium channel blockade. Loss-of-function mutations in <em>SCN5A</em>, encoding the pore-forming subunit of the cardiac sodium channel (Na<sub>v</sub>1.5) at 3p21, have been causally related to the disease in ~20% of cases<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263909">3</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263861">11</a></sup>. However, whether arrhythmias arise as a result of abnormal conduction, repolarization or both is under debate<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263856">12</a></sup>. Mutations in genes other than <em>SCN5A</em> have been found in a small subset of cases, but their involvement in Brugada syndrome remains unclear<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263848">13</a></sup>.</p><p id="P3">Although Brugada syndrome is commonly considered a mendelian disorder with autosomal dominant transmission, studies in families harboring <em>SCN5A</em> mutations have demonstrated low disease penetrance<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263912">14</a>,<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263895">15</a></sup> and, in some instances, absence of the familial <em>SCN5A</em> mutation in some affected family members<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263857">15</a></sup>. Also, many cases are sporadic<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263872">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263913">17</a></sup>, and familial linkage analyses have largely been unsuccessful in uncovering new disease-causing genes. These observations suggest a more complex inheritance model. Identifying new genetic risk factors could assist in further diagnosis, provide new insights into underlying molecular mechanisms and yield new information relevant to the broader problem of SCD.</p><p id="P4">We conducted a genome-wide association study (GWAS) to explore the role of common genetic variants in susceptibility to Brugada syndrome. We established an international consortium enabling the recruitment of 1,114 unrelated, clinically well-defined cases from 13 centers in Europe, the United States and Japan (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Every case had a Brugada syndrome type I ECG, as defined by consensus criteria, either at baseline or after drug challenge<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263899">2</a></sup>, and <em>SCN5A</em> mutation status was systematically assessed.</p><p id="P5">We first conducted a GWAS on 383 cases of self-reported European ancestry and 898 control individuals from western France (cohort Données Epidémiologiques sur le Syndrome d’Insulino-Résistance (D.E.S.I.R.)<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263908">18</a></sup>). Cases and controls were genotyped using Axiom Genome-Wide CEU 1 arrays (Affymetrix). After stringent quality control and the exclusion of rare SNPs, a total of 360,149 markers were available for further analysis (Online Methods). Multidimensional scaling on the combined cases and controls together with reference populations from the 1000 Genomes Project excluded 42 samples of non-European descent (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). Because controls were largely French individuals, whereas cases had a broader geographic origin, we supplemented the control set with individuals from four European populations from the 1000 Genomes Project that best matched the subsets of cases (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). After the exclusion of 29 cases for whom no matching controls were available, 2 homogeneous groups were defined. GWAS analysis was conducted separately on each group, and association data were combined in a meta-analysis, which included in total 312 cases and 1,115 controls (Online Methods).</p><p id="P6">We found an excess of SNPs to be associated with Brugada syndrome compared to the number expected under the null hypothesis of no association (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>). Two genomic regions showed association signals reaching genome-wide significance (<a href="/pmc/articles/PMC3869788/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>). The most significant association was obtained for rs10428132, a SNP located in the <em>SCN10A</em> gene at 3p22 (<em>P</em> = 6.79 × 10<sup>−26</sup>; <a href="/pmc/articles/PMC3869788/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Nine other markers in linkage disequilibrium (LD) with rs10428132 (<em>r</em><sup>2</sup> = 0.20–0.76) also had associations that reached genome-wide significance (<a href="/pmc/articles/PMC3869788/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). We detected another cluster of two SNPs in high LD (<em>r</em><sup>2</sup> = 0.81) at 6q22. The lead SNP at this locus was rs9388451 (<em>P</em> = 8.85 × 10<sup>−10</sup>), located downstream of the <em>HEY2</em> gene (<a href="/pmc/articles/PMC3869788/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1c</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Neither conditional analysis at each associated locus nor GWAS following genome-wide imputation of non-genotyped SNPs uncovered any additional independent association signals at genome-wide significance (Online Methods and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>). We confirmed both associations with similar effect sizes when the GWAS was restricted to 856 D.E.S.I.R. controls and 254 ancestry-matched cases by applying stringent exclusion criteria on the multidimensional scaling results (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3869788/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139688899595136" class="figure"><a href="/pmc/articles/PMC3869788/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3869788_nihms528125f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms528125f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3869788/bin/nihms528125f1.jpg"></a></div><div id="largeobj_idm139688899595136" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3869788/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3869788/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p9">Genome-wide association analysis identifies two susceptibility loci for Brugada syndrome. (<strong>a</strong>) Manhattan plot showing the association of SNPs with Brugada syndrome in a GWAS of 312 cases and 1,115 controls. The red horizontal line marks the threshold for genome-wide significance (<em>P</em> = 5 × 10<sup>−8</sup>). Two loci reached genome-wide significance on chromosomes 3 and 6. (<strong>b</strong>) Association plots for 3q22 (left) and 6q22 (right). Each SNP is plotted with respect to its chromosomal location (<em>x</em> axis) and its association <em>P</em> value (left <em>y</em> axis). SNPs are colored according to their degree of LD (<em>r</em><sup>2</sup>) with the leading variant represented by a purple diamond and labeled. The tall blue spikes indicate the recombination rate (right <em>y</em> axis) in that region of the chromosome. Coordinates are given according to NCBI Build 37.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p12">Discovery and replication studies confirm three alleles on chromosomes 3 and 6 are associated with Brugada syndrome</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139688905374416" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Marker</th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">rs11708996</th><th rowspan="1" colspan="1" valign="bottom" align="center">rs10428132</th><th rowspan="1" colspan="1" valign="bottom" align="center">rs9388451</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Genome position (Build 37)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">Chr. 3: 38633923</td><td rowspan="1" colspan="1" valign="top" align="center">Chr. 3: 38777554</td><td rowspan="1" colspan="1" valign="top" align="center">Chr. 6: 126090377</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Closest gene(s)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>HEY2</em>, <em>NCOA7</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Risk allele</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Protective allele</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">T</td></tr><tr><td rowspan="3" colspan="1" valign="top" align="left">GWAS (312 cases and 1,115 controls; European)<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="left">RAF<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.23/0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.69/0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.65/0.50</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>P</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.70 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.79 × 10<sup>−26</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.85 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">OR (CI)</td><td rowspan="1" colspan="1" valign="top" align="center">1.64 (1.30–2.07)</td><td rowspan="1" colspan="1" valign="top" align="center">3.00 (2.45–3.69)</td><td rowspan="1" colspan="1" valign="top" align="center">1.83 (1.51–2.22)</td></tr><tr><td rowspan="3" colspan="1" valign="top" align="left">Replication 1 (594 cases and 806 controls; European)<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="left">RAF<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.23/0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.65/0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.59/0.50</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>P</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.10 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.66 × 10<sup>−30</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.10 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">OR (CI)</td><td rowspan="1" colspan="1" valign="top" align="center">1.69 (1.39–2.04)</td><td rowspan="1" colspan="1" valign="top" align="center">2.35 (2.03–2.72)</td><td rowspan="1" colspan="1" valign="top" align="center">1.39 (1.19–1.61)</td></tr><tr><td rowspan="3" colspan="1" valign="top" align="left">Replication 2 (208 cases and 1,016 controls; Japanese)<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="left">RAF<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.09/0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.44/0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.72/0.61</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>P</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.63 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.56 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.70 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">OR (CI)</td><td rowspan="1" colspan="1" valign="top" align="center">2.30 (1.53–3.45)</td><td rowspan="1" colspan="1" valign="top" align="center">2.56 (2.05–3.19)</td><td rowspan="1" colspan="1" valign="top" align="center">1.74 (1.37–2.21)</td></tr><tr><td rowspan="5" colspan="1" valign="top" align="left">Meta-analysis</td><td rowspan="1" colspan="1" valign="top" align="left"><em>P</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.02 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.01 × 10<sup>−68</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.14 × 10<sup>−17</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">OR (CI)</td><td rowspan="1" colspan="1" valign="top" align="center">1.73 (1.51–1.99)</td><td rowspan="1" colspan="1" valign="top" align="center">2.55 (2.30–2.84)</td><td rowspan="1" colspan="1" valign="top" align="center">1.58 (1.42–1.75)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">Heterogeneity</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Q</em> = 2.17</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Q</em> = 3.68</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Q</em> = 5.68</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><em>P</em> = 0.338</td><td rowspan="1" colspan="1" valign="top" align="center"><em>P</em> = 0.159</td><td rowspan="1" colspan="1" valign="top" align="center"><em>P</em> = 0.058</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sup>2</sup> = 4%</td><td rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sup>2</sup> = 45%</td><td rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sup>2</sup> = 65%</td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">Cases with symptoms<sup>c</sup> (416 cases and 2,937 controls; meta-analysis)<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>P</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.88 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.15 × 10<sup>−39</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.01 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">OR (CI)</td><td rowspan="1" colspan="1" valign="top" align="center">1.73 (1.42–2.12)</td><td rowspan="1" colspan="1" valign="top" align="center">2.84 (2.43–3.32)</td><td rowspan="1" colspan="1" valign="top" align="center">1.55 (1.32–1.81)</td></tr></tbody></table></div><div id="largeobj_idm139688905374416" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3869788/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Size of case and control sets; sample ancestry.</div><div id="TFN2"><sup>b</sup>Risk allele frequency (RAF) in cases over controls.</div><div id="TFN3"><sup>c</sup>Symptoms included ventricular tachycardia, ventricular fibrillation, syncope and near syncope.</div></div></div><p id="P7">We next considered candidate SNPs identified in previous GWAS on ECG traits<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263902">4</a>–<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263884">7</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263890">19</a>–<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263910">23</a></sup>, using both genotype and imputation data (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). After the removal of redundant SNPs from the same haplotype (<em>r</em><sup>2</sup> &gt; 0.2) and the exclusion of the haplotype containing rs10428132, we observed a significant enrichment in association for the remaining 54 markers (<em>P</em> = 5.0 × 10<sup>−8</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a>). The SNP with the lowest association <em>P</em> value, rs11708996 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>), was located in <em>SCN5A</em> and has previously been associated with variability in PR interval and QRS duration<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263864">6</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263905">7</a></sup>, two ECG parameters that reflect cardiac conduction, a process possibly affected in Brugada syndrome<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263911">12</a></sup>. Although residing next to the strongest association signal at the 3p22 locus, rs11708996 showed no LD with rs10428132 (<em>r</em><sup>2</sup> = 0.06), and its association <em>P</em> value in GWAS data remained unchanged in analysis conditioning on rs10428132, thus suggesting that it represents an independent association (data not shown). Considering the established involvement of <em>SCN5A</em> in Brugada syndrome, we carried this SNP forward to the validation step of the study. We also considered SNPs at loci harboring the 11 other susceptibility genes<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263914">13</a></sup> but found no enrichment in association at statistically significant levels (genomic inflation factor (<em>λ</em><sub>GC</sub>) = 0.98; Online Methods and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 7</a>).</p><p id="P8">We sought to replicate the two genome-wide significant signals (rs10428132 and rs9388451) as well as rs11708996 in an independent case-control set of 594 individuals of European descent with Brugada syndrome and 806 previously genotyped D.E.S.I.R. individuals<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263876">24</a></sup> (Online Methods). All three SNPs showed robust association, with a similar direction of effect as seen in the discovery set, and the signal at rs10428132 reached genome-wide statistical significance in the replication set alone (<a href="/pmc/articles/PMC3869788/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). To seek additional replication and assess associations in other ancestry groups, we tested a third independent case-control set comprising 208 Japanese cases and 1,016 ancestry-matched controls. We again replicated the three associations, with rs10428132 exceeding the genome-wide significance threshold (<a href="/pmc/articles/PMC3869788/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Meta-analysis of the discovery and replication sets yielded highly significant association <em>P</em> values for all three loci (rs10428132, <em>P</em> = 1.01 × 10<sup>−68</sup>; rs11708996, <em>P</em> = 1.02 × 10<sup>−14</sup>; rs9388451, <em>P</em> = 5.14 × 10<sup>−17</sup>), with the corresponding effect sizes ranging from 1.58 to 2.55 (<a href="/pmc/articles/PMC3869788/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Effect sizes were similar when the meta-analysis was restricted to symptomatic individuals, with the association for rs10428132 reaching genome-wide statistical significance (<a href="/pmc/articles/PMC3869788/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><p id="P9">We next assessed the cumulative effect of the three loci on susceptibility to Brugada syndrome (<a href="/pmc/articles/PMC3869788/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). We found that disease risk increased consistently with increasing numbers of carried risk alleles (<em>P</em><sub>trend</sub> = 6.1 × 10<sup>−81</sup>), with the estimated odds ratio (OR) reaching 21.5 in the presence of more than four risk alleles versus less than 2 (<a href="/pmc/articles/PMC3869788/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). Under a multiplicative model, assuming a prevalence of 0.05% for Brugada syndrome<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263916">2</a></sup> and using the ORs and allele frequencies reported in <a href="/pmc/articles/PMC3869788/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, the sibling relative risk attributable to the three lead SNPs was estimated at <em>λ</em><sub>S</sub> = 1.4, with the three loci accounting for 7% of the variance in disease susceptibility (Online Methods). The proportion of the phenotypic variance explained by only these three loci, as well as their cumulative effect, are unexpectedly large in comparison to previously reported effects for common genetic variation on susceptibility to common disease<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263850">25</a></sup>. However, as 1.5% of the European population is expected to carry more than four risk alleles (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>), these three polymorphisms are unlikely to by themselves explain the occurrence of Brugada syndrome and are only associated with a low absolute risk for this rare condition. Furthermore, the ORs reported here were calculated using data from case-control collections and thus may overestimate relative risks.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3869788/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139688905397936" class="figure"><a href="/pmc/articles/PMC3869788/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3869788_nihms528125f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms528125f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3869788/bin/nihms528125f2.jpg"></a></div><div id="largeobj_idm139688905397936" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3869788/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3869788/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p10">Cumulative effect of alleles at the three associated loci on susceptibility to Brugada syndrome. (<strong>a</strong>) Distribution of risk alleles among individuals with Brugada syndrome (white bars) and among control individuals (black bars) from Europe (top) and Japan (bottom). (<strong>b</strong>) ORs calculated according to the number of risk alleles carried. A meta-analysis was performed as described in the Online Methods, using individuals carrying no or one risk allele as the reference. Each black bar represents the log(OR) value (horizontal bar) and the 95% confidence interval (on the log scale; vertical bar).</p></div></div></div><p id="P10">In subsequent analyses comparing SNP allele dosages in subsets of cases, we could not detect any consistent association of risk alleles with symptoms, <em>SCN5A</em> mutation status or Brugada syndrome type I ECG at baseline compared to after drug challenge (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>).</p><p id="P11">Our strongest association (rs10428132) resides in intron 14 of the <em>SCN10A</em> gene, located adjacent to <em>SCN5A</em> on chromosome 3p21–22. The particular haplotype tagged by this SNP, which also contains a nonsynonymous variant affecting <em>SCN10A</em> (rs6795970, <em>r</em><sup>2</sup> = 0.97 with rs10428132), has previously been associated with variability in PR interval and QRS duration in the general population<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263896">4</a>–<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263900">7</a></sup>. <em>SCN10A</em>, which encodes the sodium channel isoform Na<sub>v</sub>1.8, was originally reported to be highly expressed in nociceptive sensory neurons of the dorsal root ganglia and cranial sensory ganglia<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263842">26</a></sup>. Besides being expressed in cardiac neurons<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263869">27</a></sup>, <em>SCN10A</em> was also shown in recent studies to be expressed in the working myocardium<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263844">4</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263845">28</a></sup> and the specialized conduction system<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263901">7</a>,<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263860">29</a></sup>, indicating a possible role for Na<sub>v</sub>1.8 in cardiac electrical function. An independent report implicated the rs6801957 SNP (<em>r</em><sup>2</sup> = 0.97 with rs10428132) as a probable causal variant on the associated haplotype<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263847">30</a></sup>. This SNP, which alters a highly conserved nucleotide within a consensus T-box–binding site, affects TBX5- or TBX3-mediated enhancer activity<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263887">30</a></sup> and is thus expected to affect TBX5/TBX3-regulated expression of <em>SCN5A</em> and <em>SCN10A in vivo</em><sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263855">31</a></sup>. Further studies are required to determine whether the effects of this SNP on conduction and Brugada syndrome are mediated through regulation of <em>SCN5A</em>, <em>SCN10A</em> or both. The other independent association at 3p21–22 (rs11708996) involved another haplotype previously associated with cardiac conduction<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263897">6</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263893">7</a></sup>. For both haplotypes, the PR- and QRS-prolonging allele was associated with disease risk, providing support for the concept that the cardiomyocyte depolarization process has an important role in the pathogenesis of Brugada syndrome<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263868">12</a></sup>.</p><p id="P12">In contrast to both signals at the <em>SCN5A</em>-<em>SCN10A</em> locus, to our knowledge, the signal at 6q22 has never been detected by previous GWAS on ECG traits. The corresponding SNP tagged an LD block that entirely encompasses the <em>HEY2</em> gene and extends into the first intron of <em>NCOA7</em> (also called <em>ERAP140</em>), which encodes a nuclear receptor coactivator that interacts with estrogen receptor α to modulate its activity<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263858">32</a></sup>. <em>NCOA7</em> is predominantly expressed in the brain<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263879">32</a></sup> and has not been implicated in cardiovascular function. In contrast, <em>HEY2</em> (also called <em>HESR2</em>, <em>HRT2</em> and <em>CHF1</em>) encodes a basic helix-loop-helix (bHLH) transcriptional repressor that is expressed in the cardiovascular system<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263881">8</a></sup>.</p><p id="P13">Studies we have conducted in <em>Hey2</em>-targeted mice provide strong support for the role of this gene in Brugada syndrome. In the developing mouse heart, <em>Hey2</em> expression is confined to the (subepicardial) compact myocardium of the ventricle<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263843">33</a></sup>. <em>Hey2</em>-null mice exhibit a spectrum of developmental anomalies, including ventricular wall thinning, abnormal right ventricular morphology and postnatal cardiomyopathic changes<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263875">34</a>–<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263904">37</a></sup>. The expression of <em>Gja5</em> (encoding Cx40), <em>Nppa</em> and <em>Tbx5</em>, normally enriched in the (subendocardial) trabecular component of the ventricle, is expanded into the compact myocardium in <em>Hey2</em>-deficient embryos<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263853">34</a>,<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263878">38</a>,<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263889">39</a></sup>. Because such transmural heterogeneity in expression is similarly well established for Na<sub>v</sub>1.5 (high expression in subendocardium, low expression in subepicardium)<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263892">40</a></sup>, loss of <em>Hey2</em> might also affect the transmural expression gradient of this ion channel implicated in Brugada syndrome<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263854">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263918">3</a></sup>. Indeed, in hearts from homozygous <em>Hey2</em>-null embryos, we observed higher Na<sub>v</sub>1.5 expression in the compact layer than in wild-type hearts, flattening the expression gradient of this channel (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 8a</a>).</p><p id="P14">The functional consequences of Hey2 loss were investigated in adult heterozygous <em>Hey2</em> mice (<em>Hey2</em><sup>+/−</sup>), which have structurally normal hearts (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 9</a>). <em>In vivo</em> surface ECG parameters were unchanged in <em>Hey2</em><sup>+/−</sup> mice (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10</a>). However, conduction velocity was significantly increased in the right ventricular outflow tract (RVOT) of isolated <em>Hey2</em><sup>+/−</sup> hearts (<a href="/pmc/articles/PMC3869788/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3a, b</span></a>), whereas conduction velocity was unaffected in the right and left ventricular free wall (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 11</a>). Action potential upstroke velocity was increased in <em>Hey2</em><sup>+/−</sup> myocytes isolated from the RVOT region (<a href="/pmc/articles/PMC3869788/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3c, d</span></a>), pointing to increased sodium channel function, despite undetectable changes in Na<sub>v</sub>1.5 expression in adult hearts from <em>Hey2</em><sup>+/−</sup> mice in immunohistochemistry analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 8b</a>). Furthermore, the prolonged repolarization parameters observed in these cells suggest an additional regulatory role for Hey2 in repolarizing currents (<a href="/pmc/articles/PMC3869788/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3d</span></a>). Future work should address whether the observed alterations in action potential characteristics and conduction are mediated through ion channel correlates, subtle structural heart alterations or both. Nevertheless, the preferential involvement of the RVOT is in line with ECG manifestations in right precordial leads and concurs with the observation that the RVOT is a common site of origin of ventricular arrhythmias in individuals with Brugada syndrome<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263873">41</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3869788/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139688905377120" class="figure"><a href="/pmc/articles/PMC3869788/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3869788_nihms528125f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms528125f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3869788/bin/nihms528125f3.jpg"></a></div><div id="largeobj_idm139688905377120" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3869788/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3869788/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p11">Increased conduction velocity and sodium channel availability in the RVOT of adult <em>Hey2</em><sup>+/−</sup> mice. (<strong>a</strong>) Representative optical activation maps from isolated wild-type (WT; <em>n</em> = 12) and <em>Hey2</em><sup>+/−</sup> (<em>n</em> = 11) hearts stimulated at the RVOT at a basic cycle length of 120 ms (scale bars, 1 mm). Arrows indicate conduction along the RVOT epicardium through the connection of similar isochrones. Isochrones (0.5-ms intervals) in the RVOT of <em>Hey2</em><sup>+/−</sup> hearts are less crowded compared to wild-type isochromes, indicating faster conduction in the RVOT of <em>Hey2</em><sup>+/−</sup> hearts (as indicated by increased arrow length). (<strong>b</strong>) On average, conduction velocity in the RVOT of <em>Hey2</em><sup>+/−</sup> hearts was significantly increased compared to that in wild-type hears (*<em>P</em> &lt; 0.05; Student’s <em>t</em> test). Error bars, s.e.m. (<strong>c</strong>) Representative action potentials measured in adult wild-type (WT; <em>n</em> = 7) and <em>Hey2</em><sup>+/−</sup> (<em>n</em> = 11) cardiomyocytes isolated from RVOT. (<strong>d</strong>) Average action potential characteristics measured at 4 Hz. RVOT cardiomyocytes from <em>Hey2</em><sup>+/−</sup> mice showed a significant increase in maximal upstroke velocity (dV/dT<sub>max</sub>, a measure of sodium channel availability and a determinant of conduction velocity) and action potential amplitude (APA), indicating increased sodium channel availability (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; Student’s <em>t</em> test). Resting membrane potential (RMP), action potential duration at 20% and 50% repolarization (APD<sub>20</sub> and APD<sub>50</sub>, respectively) were not significantly different in wild-type and <em>Hey2</em><sup>+/−</sup> mice; action potential duration at 90% repolarization (APD<sub>90</sub>) was significantly increased in RVOT cardiomyocytes from <em>Hey2</em><sup>+/−</sup> mice. Results are expressed as mean ± s.e.m.</p></div></div></div><p id="P15" class="p">In conclusion, we have identified Hey2 as a transcriptional regulator of cardiac electrical function involved in the pathogenesis of Brugada syndrome. Furthermore, we provide new evidence that common variants, previously shown to modulate ECG conduction indices, also modulate susceptibility to a rare arrhythmia disorder. Most notably, this study demonstrates that the GWAS paradigm can be successfully applied to a rare disorder, previously considered monogenic, to identify common genetic variants with unexpectedly strong modifier effects.</p><div id="S2" class="sec"><h3 id="S2title">URLs</h3><p id="P16" class="p p-first-last">Affymetrix Power Tools, <a href="http://www.affymetrix.com/partners_programs/programs/developer/tools/powertools.affx" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.affymetrix.com/partners_programs/programs/developer/tools/powertools.affx</a>; GTOOL, <a href="http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html</a>; R statistical package, <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>; 1000 Genomes Project, <a href="http://www.1000genomes.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.1000genomes.org/</a>; 1000 Genomes phase I integrated variant set release, <a href="http://mathgen.stats.ox.ac.uk/impute/data_download_1000G_phase1_integrated.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mathgen.stats.ox.ac.uk/impute/data_download_1000G_phase1_integrated.html</a>.</p></div></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">ONLINE METHODS</h2><div id="S5" class="sec sec-first"><h3 id="S5title">Case and control samples</h3><p id="P18" class="p p-first-last">Individuals with Brugada syndrome, defined by the presence of a type 1 ECG<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263906">2</a></sup>, were recruited from 13 centers in Europe (Nantes, Paris, Amsterdam, Pavia, Copenhagen, Munich, Münster and London), the United States (Utica and Nashville) and Japan (Nagasaki, Shiga and Osaka). Only index cases were included from extended pedigrees. Appropriate medical ethical committee approval was obtained at each participating clinical center. Informed consent was available from all subjects. Clinical data (age at diagnostic ECG, <em>SCN5A</em> mutation status, symptoms and family history of sudden cardiac death) and ECGs were collected centrally and reviewed. A Brugada syndrome type I ECG pattern was defined on the basis of the criteria drawn out at the Second Consensus Conference on Brugada Syndrome<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263903">2</a></sup>, namely, a coved type ST elevation at baseline or after a drug challenge test, in one or more leads in the right precordial leads, including the third and fourth intercostal space. Drug challenge tests were performed according to consensus criteria<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263859">2</a></sup>. Control subjects were drawn from the D.E.S.I.R. cohort<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263886">18</a></sup> for the GWAS and the European replication set and were drawn from the Sado study<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263866">42</a></sup> for the Japanese replication set. No statistical method was used to predetermine sample size.</p></div><div id="S6" class="sec"><h3 id="S6title">GWAS genotyping</h3><p id="P19" class="p p-first-last">SNP genotyping was performed on population-optimized Affymetrix Axiom Genome-Wide CEU 1 array plates following the standard manufacturer’s protocol. Each array contains 567,097 SNPs. Fluorescence intensities were quantified using the Affymetrix GeneTitan Multi-Channel Instrument, and primary analysis was conducted with Affymetrix Power Tools following the manufacturer’s recommendations (see URLs). Genotype calling, a two-dimensional clustering analysis, was performed using the ‘apt’ program. Individuals with genotype call rate of lower than 97% were removed, as were those with fewer than 10,000 markers reporting a heterozygous state (the threshold was determined after visual inspection). Monomorphic SNPs were excluded, as were those with minor allele frequency (MAF) of &lt;10% (<em>n</em> = 175,153), a call rate of &lt;95% (<em>n</em> = 19,986) or Hardy-Weinberg disequilibrium in controls (<em>n</em> = 2,054 with <em>P</em> &lt; 1 × 10<sup>−4</sup> when testing for Hardy-Weinberg equilibrium). Note that Hardy-Weinberg disequilibrium was also tested in demographically homogenous cases to identify very large deviations (<em>P</em> &lt; 1 × 10<sup>−7</sup>). Additional SNPs were excluded for batch effect: such SNPs were defined as those with significant differences in allele frequency in one plate versus all others within cases and within controls only (<em>n</em> = 68) or with unexplained large differences observed in controls versus the 1000 Genomes Project European (non-Finnish) population (<em>n</em> = 9,686).</p></div><div id="S7" class="sec"><h3 id="S7title">Demographic analyses</h3><p id="P20" class="p p-first-last">The ancestry of individuals was assessed using a multidimensional scaling technique, as implemented in PLINK<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263888">43</a></sup>. SNPs were selected for short-range LD independence. Pruning was performed using a two-step procedure to accommodate longer range LD (this is particularly important, as the Axiom Human array is enriched in SNPs in the human leukocyte antigen (HLA) region). In a first step, we applied the threshold <em>r</em><sup>2</sup> &lt; 0.2 within a 20-kb LD block or within 50 SNPs. In a second step, we applied the same threshold within a 10-Mb distance or within 100 SNPs on the pruned data set. We then created an identity-by-state (IBS) matrix including all individuals and applied the multidimensional scaling method (–mds option in PLINK) to retrieve the first five components. Three matrices were estimated using our cases and controls together with all 1000 Genomes Project populations (IC) and all European (except Finnish) populations (E). At each level, we excluded outliers on the first two components using an expectation-maximization-fitted Gaussian mixture clustering method<sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup> implemented in the R package M-CLUST, assuming either three (for IC) or two (for E) clusters and noise. Outlier position was assigned using nearest-neighbor-based classification<sup><a href="#R45" rid="R45" class=" bibr popnode">45</a></sup> (NNclust in R package PrabClus). Outliers were excluded from the analysis, as previously done in GWAS<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263919">46</a></sup>.</p></div><div id="S8" class="sec"><h3 id="S8title">GWAS</h3><p id="P21" class="p p-first-last">Using the clustering algorithm described above, we defined two homogenous groups (A and B). To carry out the genome-wide analysis, each SNP was tested within groups A and B separately, using logistic regression and assuming an additive genetic model with adjustment for the first five components retrieved. No additional covariates were added, as advised<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263863">47</a></sup>. Instead, the results from groups A and B were combined into a meta-analysis using an inverse normal strategy<sup><a href="#R48" rid="R48" class=" bibr popnode">48</a></sup>, whereby the summary <em>P</em> values for each test (and effect direction) are combined into a signed <em>z</em> score that, properly weighted, yields <em>N</em> (μ = 0, σ<sup>2</sup> = 1). Because the number of controls exceeded by far the number of cases in all studies, we used the effective sample size (weighting studies A and B) using METAL software as advised<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263882">49</a></sup>. In addition, we performed a second genome-wide analysis on a homogenous sample of 254 cases and 806 controls of apparent French origin (largest geographically homogenous sample; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>).</p></div><div id="S9" class="sec"><h3 id="S9title">Concordance rate between Axiom and 1000 Genomes Project data</h3><p id="P22" class="p p-first-last">We genotyped 95 HapMap individuals on Affymetrix Axiom Genome-Wide CEU 1 arrays using the same process as described above. We could retrieve the genotypes of 58 of these 95 individuals from the 1000 Genomes Project database. The concordance rate was tested using PLINK (merging mode 7, which compares the common non-missing genotypes). The concordance rate was 99.4% over a total of 20,853,552 genotypes and 100% over the 174 genotypes corresponding to the 3 associated SNPs.</p></div><div id="S10" class="sec"><h3 id="S10title">Genome-wide imputation analysis</h3><p id="P23" class="p p-first-last">Genotyped SNPs in cases and controls were phased using the SHAPE-IT (v.1) program<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263862">50</a></sup>. Imputation of 6.1 million frequent SNPs (MAF &gt; 0.05 in Europeans) was carried out using IMPUTE v2 (ref. <a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263880">51</a>). Chromosome regions were split in chunks of approximately 7 Mb. The reference panel was Phase I integrated variant set release (v3) in NCBI Build 37 (hg19) coordinates (see URLs). For each chromosomal chunk, a set of genetically matched panel individuals was selected, according to the last strategy used by IMPUTE<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263891">52</a></sup>. Imputed SNPs were combined with SNPs extracted from the 1000 Genomes Project data set under the IMPUTE format. We merged both data sets using GTOOL. We applied a logistic regression (additive model) as implemented in SNPTEST<sup><a href="#R45" rid="R45" class=" bibr popnode">45</a></sup> (options -frequentist 1 and -score), using the first five components as covariates. Individuals from the 1000 Genomes Project data set were added for all SNPs either genotyped or imputed.</p></div><div id="S11" class="sec"><h3 id="S11title">Post-analysis quality control</h3><p id="P24" class="p p-first-last">For each significantly associated SNP in a region (called here lead SNP), we visually inspected the cluster graph (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 12a</a>) and the association results of SNPs in LD (locus plot).</p></div><div id="S12" class="sec"><h3 id="S12title">Enrichment analysis</h3><p id="P25" class="p p-first-last">We tested enrichment in moderately associated SNPs grouped by distinctive properties. The first group included SNPs that have been reported to be associated with ECG traits in previous GWAS. We used the list published in ref. <a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263846">53</a> after removal of redundant SNPs (one SNP was selected randomly for each group of SNPs with <em>r</em><sup>2</sup> &gt; 0.2) and exclusion of SNPs located in the same haplotype block as rs10428132. The second list comprised every SNP located within 1-Mb intervals (500 kb upstream and 500 kb downstream) centered on the susceptibility genes for Brugada syndrome listed in ff <sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263849">13</a></sup>. We removed the <em>SCN5A</em> gene, as it would have disproportionately inflated the overall statistics. Quantile-quantile plots were first visually inspected to check for enrichment. The significance of enrichment suggested by the quantile-quantile plot for the first group (ECG-associated SNPs) was formally tested by a simple Fisher combination test.</p></div><div id="S13" class="sec"><h3 id="S13title">Replication genotyping</h3><p id="P26" class="p p-first-last">The three significantly associated SNPs from the GWAS stage (rs10428132, rs9388451 and rs11708996) were typed for the replication steps by TaqMan SNP Genotyping assays (Applied Biosystems) according to the manufacturer’s protocol on a LightCycler 480 Real-Time PCR System (Roche) or an ABI Prism 7900HT Sequence Detection System (Applied Biosystems). Assays C__26860683_10, C___2824356_10 and C__44417794_10 were functionally tested by Applied Biosystems for the SNPs rs10428132, rs9388451 and rs11708996, respectively. Reaction conditions included an initial denaturation step at 95 °C for 10 min, followed by 50 cycles of denaturation at 95 °C for 15 s and annealing at 60 °C for 30 s. Data were analyzed with LightCycler 480 software (release 1.5.0 SP4; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 12b</a>) or by ABI 7900HT Sequence Detection Systems software (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 12c</a>). Outliers were excluded from the analysis. After the first round of genotyping, two samples per SNP genotype cluster were sequenced to verify genotype (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 12b</a>). Furthermore, 15 samples that were previously genotyped on Affymetrix Axiom Genome-Wide CEU 1 arrays were used as control samples: the genotypes obtained by TaqMan assays were perfectly concordant with those generated with Axiom array technology.</p></div><div id="S14" class="sec"><h3 id="S14title">SNP imputation for the control population used in the European replication</h3><p id="P27" class="p p-first-last">The genotypes of the three SNPs tested in replication were imputed using IMPUTE software for 806 D.E.S.I.R. individuals on the basis of genotype data sets previously generated on Illumina Human CNV370-Duo arrays<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263898">24</a>,<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263917">54</a></sup>. We selected a threshold of 0.9 for genotype calling, meaning that the genotypes whose posterior probability exceeded this threshold were called, whereas those where this probability was below the threshold were set as unknown. We genotyped 49 individuals on both Affymetrix and Illumina arrays. Once hard calls were obtained using GTOOL (see URLs) with -Gen mode and –threshold option set to 0.9, concordance rates were calculated using PLINK (merging mode 7). Perfect concordance was observed between genotyped and imputed SNPs (100% concordance rate for 144 genotypes).</p></div><div id="S15" class="sec"><h3 id="S15title">Replications and meta-analyses</h3><p id="P28" class="p p-first-last">The case-control replication study was performed using a logistic regression method that accounts for genotype calling uncertainty. This method, based on missing data theory, allows the unbiased estimation of ORs and confidence intervals and is implemented in SNPTEST (options–method ml). Pooled ORs were obtained by averaging the ORs from all stages (GWAS and European and Japanese replications) and weighted by the inverse of the variance. Heterogeneity was tested using the Cochran’s <em>Q</em> test and was also measured using Higgins’ index<sup><a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263883">55</a></sup>. We generated genetic scores for individuals on the basis of an allelic scoring system involving our three SNPs. These scores were created either through the number of at-risk alleles for the European discovery and Japanese replication populations or the risk allele dosage in the European replication population. Risk allele dosages from the European replication population were collapsed using dosage. The distribution of imputed dosage is shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 13a</a>. Results were similar between imputed dosage (<em>β</em> = 0.62921, <em>σ</em> = 0.05427, <em>P</em> = 4.43 × 10<sup>−31</sup>) and collapsed imputed dosage (<em>β</em> = 0.61778, <em>σ</em> = 0.05386, <em>P</em> = 1.88 × 10<sup>−30</sup>). Moreover, we compared the genetic scores obtained with genotyped versus imputed SNPs for 49 individuals who were genotyped on Axiom Genome-Wide CEU 1 arrays and imputed with the European replication population and observed high correlation between methods (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 13b</a>). Finally, we tested whether a non-additive model (recessive or dominant) might be a better fit for each genome-wide significant SNP. A heterozygote effect was added to the logistic regression analysis along with the linear effect (effect of the number of alternative alleles) in each study and in a meta-analysis. We did not detect any consistent deviation from the additive model (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>).</p></div><div id="S16" class="sec"><h3 id="S16title">Estimation of the genetic score effect by multiple imputation</h3><p id="P29" class="p p-first">Despite this high concordance, we chose to estimate genotype score risk within the Multiple Imputation framework<sup><a href="#R56" rid="R56" class=" bibr popnode">56</a></sup>. Ten data sets were created where each uncertain genotype was replaced by a value simulated under the probability distribution obtained through genetic imputation (IMPUTE output consisting of <em>P</em>(AA), <em>P</em>(AB) and <em>P</em>(BB)). The <em>β</em> value and variance were obtained using standard procedures<sup><a href="#R57" rid="R57" class=" bibr popnode">57</a></sup>. We let <em>m</em> be the number of simulations (called replicates). For each simulation, we carried out a logistic regression (either on score as an ordinal value or on each score versus baseline). The Multiple Imputation effect estimation was calculated as follows: 
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><msub><mover accent="true"><mi>β</mi><mo>¯</mo></mover><mtext>MI</mtext></msub><mo>=</mo><mfrac><mn>1</mn><mi>m</mi></mfrac><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></munderover><msub><mi>β</mi><mi>j</mi></msub></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><msub><mover accent="true"><mi>β</mi><mo>¯</mo></mover><mtext>MI</mtext></msub><mo>=</mo><mfrac><mn>1</mn><mi>m</mi></mfrac><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></munderover><msub><mi>β</mi><mi>j</mi></msub></math></script></div></div><p></p><p id="P30">The variance was calculated as the sum of within-replicate and between-replicate variances</p><div class="disp-formula" id="FD2"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><msubsup><mi>σ</mi><mtext>MI</mtext><mn>2</mn></msubsup><mo>=</mo><mfrac><mn>1</mn><mi>m</mi></mfrac><msubsup><mover accent="true"><mi>σ</mi><mo>⌢</mo></mover><mi>j</mi><mn>2</mn></msubsup><mo>+</mo><mfrac><mn>1</mn><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></mfrac><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></munderover><msup><mrow><mo stretchy="false">(</mo><msub><mover accent="true"><mi>β</mi><mo>¯</mo></mover><mtext>MI</mtext></msub><mo>-</mo><msub><mi>β</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><msubsup><mi>σ</mi><mtext>MI</mtext><mn>2</mn></msubsup><mo>=</mo><mfrac><mn>1</mn><mi>m</mi></mfrac><msubsup><mover accent="true"><mi>σ</mi><mo>⌢</mo></mover><mi>j</mi><mn>2</mn></msubsup><mo>+</mo><mfrac><mn>1</mn><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></mfrac><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></munderover><msup><mrow><mo stretchy="false">(</mo><msub><mover accent="true"><mi>β</mi><mo>¯</mo></mover><mtext>MI</mtext></msub><mo>-</mo><msub><mi>β</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></math></script></div></div><p id="P31" class="p p-last">Confidence intervals were retrieved using the 95% quantile of a Student distribution with a number of degrees of freedom, which is a function of the two components of the variance. We used the ‘glm’ function of the R statistical package (see URLs) to perform logistic regression. R was used to create complete data sets from IMPUTE output.</p></div><div id="S17" class="sec"><h3 id="S17title">Calculation of sibling relative risk and liability-scale variance</h3><p id="P32" class="p p-first">Assuming a multiplicative model (within and between variants), the contribution to the sibling relative risk of a set of <em>N</em> SNPs was calculated using the following formula</p><p id="P33">
</p><div class="disp-formula" id="FD3"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" display="block" overflow="scroll"><msub><mi>λ</mi><mi mathvariant="normal">s</mi></msub><mo>=</mo><msubsup><mi mathvariant="normal">Π</mi><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><msup><mrow><mo>[</mo><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">)</mo><msup><mrow><mo stretchy="false">(</mo><msub><mtext>OR</mtext><mi>j</mi></msub><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow><mrow><mn>2</mn><msup><mrow><mo stretchy="false">[</mo><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">)</mo><mo>+</mo><msub><mi>p</mi><mi>j</mi></msub><msub><mtext>OR</mtext><mi>j</mi></msub><mo stretchy="false">]</mo></mrow><mn>2</mn></msup></mrow></mfrac></mrow><mo>]</mo></mrow><mn>2</mn></msup></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" display="block" overflow="scroll"><msub><mi>λ</mi><mi mathvariant="normal">s</mi></msub><mo>=</mo><msubsup><mi mathvariant="normal">Π</mi><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><msup><mrow><mo>[</mo><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">)</mo><msup><mrow><mo stretchy="false">(</mo><msub><mtext>OR</mtext><mi>j</mi></msub><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow><mrow><mn>2</mn><msup><mrow><mo stretchy="false">[</mo><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">)</mo><mo>+</mo><msub><mi>p</mi><mi>j</mi></msub><msub><mtext>OR</mtext><mi>j</mi></msub><mo stretchy="false">]</mo></mrow><mn>2</mn></msup></mrow></mfrac></mrow><mo>]</mo></mrow><mn>2</mn></msup></math></script></div></div><p> where <em>p<sub>j</sub></em> and OR<em><sub>j</sub></em> denote the RAF and corresponding allelic OR at the <em>j</em>th SNP<sup><a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263915">58</a></sup>. Assuming disease prevalence <em>K</em>, the liability-scale variance<sup><a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263870">59</a></sup> explained by these SNPs was calculated as follows: 
</p><div class="disp-formula" id="FD4"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" display="block" overflow="scroll"><msubsup><mi>h</mi><mi>L</mi><mn>2</mn></msubsup><mo>=</mo><mfrac><mrow><mn>2</mn><mo stretchy="false">[</mo><mi>T</mi><mo>-</mo><msub><mi>T</mi><mn>1</mn></msub><msqrt><mrow><mn>1</mn><mo>-</mo><mo stretchy="false">(</mo><msup><mi>T</mi><mn>2</mn></msup><mo>-</mo><msubsup><mi>T</mi><mn>1</mn><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></msqrt><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><mi>T</mi><mo>/</mo><mi>ω</mi><mo stretchy="false">)</mo><mo stretchy="false">]</mo></mrow><mrow><mi>ω</mi><mo>+</mo><msubsup><mi>T</mi><mn>1</mn><mn>2</mn></msubsup><mo stretchy="false">(</mo><mi>ω</mi><mo>-</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></mfrac></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" display="block" overflow="scroll"><msubsup><mi>h</mi><mi>L</mi><mn>2</mn></msubsup><mo>=</mo><mfrac><mrow><mn>2</mn><mo stretchy="false">[</mo><mi>T</mi><mo>-</mo><msub><mi>T</mi><mn>1</mn></msub><msqrt><mrow><mn>1</mn><mo>-</mo><mo stretchy="false">(</mo><msup><mi>T</mi><mn>2</mn></msup><mo>-</mo><msubsup><mi>T</mi><mn>1</mn><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></msqrt><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><mi>T</mi><mo>/</mo><mi>ω</mi><mo stretchy="false">)</mo><mo stretchy="false">]</mo></mrow><mrow><mi>ω</mi><mo>+</mo><msubsup><mi>T</mi><mn>1</mn><mn>2</mn></msubsup><mo stretchy="false">(</mo><mi>ω</mi><mo>-</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></mfrac></math></script></div></div><p></p><p id="P34" class="p p-last">In the expression, <em>T</em> = <em>ϕ</em><sup>−1</sup> (1 − <em>K</em>), <em>T</em><sub>1</sub> = <em>ϕ</em><sup>−1</sup> (1 − <em>λ</em><sub>S</sub><em>K</em>) and <em>ω</em> = <em>z</em>/<em>K</em>, where <em>z</em> is the height of the standard Gaussian density at <em>T</em>.</p></div><div id="S18" class="sec"><h3 id="S18title">Surface ECG analysis</h3><p id="P35" class="p p-first-last">Mice (male and female, aged 3–5 months) were anesthetized using isoflurane inhalation (0.8–1.0 volume percentage in oxygen), and surface ECGs were recorded from subcutaneous 23-gauge needle electrodes attached to each limb using the Powerlab acquisition system (ADInstruments). ECG traces were signal averaged and analyzed for heart rate (RR interval) and for PR-interval, QRS-interval and QT-interval duration using LabChart7Pro software (ADInstruments). QT intervals were corrected for heart rate using the formula 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" overflow="scroll"><mtext>QTc</mtext><mo>=</mo><mtext>QT</mtext><mo>/</mo><msqrt><mrow><mtext>RR</mtext><mo>/</mo><mn>100</mn></mrow></msqrt></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" overflow="scroll"><mtext>QTc</mtext><mo>=</mo><mtext>QT</mtext><mo>/</mo><msqrt><mrow><mtext>RR</mtext><mo>/</mo><mn>100</mn></mrow></msqrt></math></script> (RR in ms).</p></div><div id="S19" class="sec"><h3 id="S19title">Optical mapping in isolated hearts</h3><p id="P36" class="p p-first-last">Mice were anesthetized by an intraperitoneal injection of pentobarbital, after which the heart was excised, cannulated, incubated in 10 ml Tyrode’s solution containing 15 μM Di-4 ANEPPS and subsequently placed in an optical mapping setup. Hearts were perfused at 37 °C with Tyrode’s solution (128 mM NaCl, 4.7 mM KCl, 1.45 mM CaCl<sub>2</sub>, 0.6 mM MgCl<sub>2</sub>, 27 mM NaHCO<sub>3</sub>, 0.4 mM NaH<sub>2</sub>PO<sub>4</sub> and 11 mM glucose (pH maintained at 7.4 by equilibration with a mixture of 95% O<sub>2</sub> and 5% CO<sub>2</sub>)) containing blebbistatin to prevent movement artifacts. Excitation light was provided by a 5-W power LED (filtered 510 ± 20 nm). Fluorescence (filtered for &gt;610 nm) was transmitted through a tandem lens system on a CMOS sensor (100 × 100 elements; MICAM Ultima). Hearts were paced at a basic cycle length (BCL) of 120 ms (twice the diastolic stimulation threshold) from the center of the right or left ventricle epicardial surface or from the RVOT epicardial surface. Optical action potentials were analyzed with custom software. Local activation was defined as the maximum positive slope of the action potential, and local activation times were used to construct ventricular activation maps. To calculate conduction velocity in longitudinal and transversal directions in the right and left ventricles, the difference in activation time between at least three consecutive electrode terminals separated by a known distance located parallel (longitudinal) or perpendicular (transversal) to the direction of propagation was measured. The direction of propagation was determined using isochronal maps. For c velocity measurements in the RVOT, transverse fiber direction was defined as the slowest conduction velocity.</p></div><div id="S20" class="sec"><h3 id="S20title">Action potential measurements in isolated cardiomyocytes</h3><p id="P37" class="p p-first-last">Cardiomyocytes were isolated by enzymatic dissociation as previously described<sup><a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263851">60</a></sup>. After perfusion of the heart with enzyme solution, the RVOT area at the base of the right ventricle just below the truncus pulmonalis was excised and used during the subsequent isolation process<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263867">43</a></sup>. Quiescent rod-shaped cross-striated cells with a smooth surface were selected for measurements. Action potentials were recorded with the amphotericin-B-perforated patch-clamp using an Axopatch 200B Clamp amplifier (Molecular Devices Corporation). Signals were filtered (low pass, 10 kHz) and digitized (40 kHz), and action potentials were corrected for the estimated change in liquid junction potential. Action potentials were measured at 36 ± 0.2 °C using a modified Tyrode’s solution containing 140 mM NaCl, 5.4 mM KCl, 1.8 mM CaCl<sub>2</sub>, 1.0 mM MgCl<sub>2</sub>, 5.5 mM glucose and 5.0 mM HEPES, pH 7.4 (NaOH). Pipettes (1.5–2.5 MΩ) were filled with solution containing 125 mM potassium gluconate, 20 mM KCl, 10 mM NaCl, 0.22 mM amphotericin-B and 10 mM HEPES, pH 7.2 (KOH). Action potentials were elicited at 4 Hz by 3 ms, 1.2× threshold current pulses through the patch pipette. We analyzed resting membrane potential (RMP), action potential amplitude (APA), maximal upstroke velocity (dV/dt<sub>max</sub>) and action potential duration at 20, 50 and 90% repolarization (APD<sub>20</sub>, APD<sub>50</sub> and APD<sub>90</sub>, respectively). Data from ten consecutive action potentials were averaged. Results are expressed as mean ± s.e.m. Two sets of data were considered significantly different if the <em>P</em> value of the unpaired Student’s <em>t</em> test with Bonferroni correction was &lt;0.05.</p></div><div id="S21" class="sec sec-last"><h3 id="S21title">Other methods</h3><p id="P38" class="p p-first-last">Procedures for immunohistochemistry hybridization on mouse heart sections were performed as described previously<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396263865">40</a></sup>.</p></div></div><div id="S22" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Corrigendum</h4><div class="sup-box half_rhythm" id="d36e2403"><a href="/pmc/articles/PMC3869788/bin/NIHMS528125-supplement-Corrigendum.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(79K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>OnlineSuppl</h4><div class="sup-box half_rhythm" id="d36e2407"><a href="/pmc/articles/PMC3869788/bin/NIHMS528125-supplement-OnlineSuppl.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(5.8M, pdf)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Acknowledgments</h2><div class="sec"><p id="__p2">We thank L. Beekman, C. de Gier-de Vries, B. de Jonge and the Genomic Platform of Nantes (Biogenouest Genomics) for technical support. We are also grateful to the French Clinical Network against Inherited Cardiac Arrhythmias, which includes the University Hospitals of Nantes, Bordeaux, Rennes, Tours, Brest, Strasbourg, La Réunion, Angers and Montpellier. This study was funded by research grants from the Leducq Foundation (CVD-05; Alliance Against Sudden Cardiac Death), the Ministry of Education, Culture, Sports, Science and Technology of Japan (grant-in-aid for the Project in Sado for Total Health, PROST), the French Ministry of Health (PHRC <a id="__tag_788518565" class="tag_hotlink" href="/protein/AOR04070" ref="/protein/AOR04070"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"AOR04070","term_id":"1068002118","term_text":"AOR04070"}}</span>AOR04070</a>, P040411 and PHRCI DGS2001/0248), INSERM (ATIP-Avenir program to R.R.) and the French Regional Council of Pays-de-la-Loire. This research was also supported by the Netherlands Heart Institute (grant 061.02 to C.A.R. and C.R.B.) and the Division for Earth and Life Sciences (ALW; project 836.09.003 to C.A.R.) with financial aid from the Netherlands Organization for Scientific Research (NWO). C.R.B. acknowledges support from the Netherlands Heart Foundation (NHS 2007B202 and 2009B066). J.B. was supported by a research grant from the European Society of Cardiology and the Netherlands Heart Institute (ICIN) and by the French-Dutch Academy through the Van Gogh program. E.S.-B. was supported by the Interdisziplinären Zentrums für Klinische Forschung (IZKF) of the University of Münster and the Collaborative Research Center SFB656. M.G. acknowledges support from the German Research Foundation (DFG-SFB 688 and TP A16). This manuscript is dedicated to the memory of Denis Escande, who founded the Leducq Foundation Network Alliance Against Sudden Cardiac Death.</p></div></div><div id="idm139688906068208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139688906068208title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><div><p id="__p3" class="p p-first"><strong>AUTHOR CONTRIBUTIONS</strong></p>C.R.B., J.-J.S. and R.R. designed the study. Y.M. and J.-B.G. evaluated all ECGs. C.D. coordinated the statistical analyses, which C.D., F.S., P.L. and E.C. carried out. F.G., A.D., S.L. and E.C. performed genotyping for the GWAS. J.B., J.V., V. Portero and K.H. carried out genotyping in the validation sets. A.A.W., H.L.T., H.L.M., V. Probst, F.K., S. Bézieau, S.C., S.K., B.M.B., E.S.-B., S.Z., L.C., P.J.S., F.D., M.T., C.A., S. Bartkowiak, P.G., V.F., A.L., D.M.R., P.W., E.R.B., R.B., J.T.-H., M.S.O., N.M., A.N., M.H., S.O., K.H., W.S. and T.A. recruited subjects and participated in clinical and molecular diagnostics. P.F., B.B., O.L., H.W., T.M. and N.E. provided controls. M.G., D.W. and C.W. provided the mice. C.A.R., A.O.V., B.J.B. and R.W. acquired and analyzed electrophysiological data. V.M.C., C.A.R. and R.W. acquired and analyzed protein expression data. C.R.B., J.B., C.A.R., C.D., J.-J.S., V.M.C., R.C. and R.R. interpreted the data. C.R.B., J.-J.S., V. Probst, D.M.R., A.A.W., S.K., E.S.-B., A.L. and R.R. obtained funding. C.R.B., J.B., C.D. and R.R. drafted the manuscript. All coauthors critically revised the manuscript for intellectual content. C.R.B. and R.R. led the study together.</div><p class="fn sec" id="FN4"></p><p id="__p5" class="p p-first-last">Note: Any <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a> and Source Data files are available in the online version of the paper.</p><p></p><p class="fn sec" id="FN5"> </p><p id="__p6" class="p p-first"><strong>COMPETING FINANCIAL INTERESTS</strong></p><p id="__p7" class="p p-last">The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN6"></p><p id="__p8" class="p p-first-last">Reprints and permissions information is available online at <a href="http://www.nature.com/reprints/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/reprints/index.html</a>.</p><p></p></div></div><div id="idm139688906762288" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139688906762288title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Brugada P, Brugada J. Right bundle branch block, persistent ST segment elevation and sudden cardiac death: a distinct clinical and electrocardiographic syndrome. A multicenter report. <span><span class="ref-journal">J Am Coll Cardiol. </span>1992;<span class="ref-vol">20</span>:1391–1396.</span> [<a href="/pubmed/1309182" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Cardiol&amp;title=Right+bundle+branch+block,+persistent+ST+segment+elevation+and+sudden+cardiac+death:+a+distinct+clinical+and+electrocardiographic+syndrome.+A+multicenter+report&amp;author=P+Brugada&amp;author=J+Brugada&amp;volume=20&amp;publication_year=1992&amp;pages=1391-1396&amp;pmid=1309182&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Antzelevitch C, et al.  Brugada syndrome: Report of the Second Consensus Conference Endorsed by the Heart Rhythm Society and the European Heart Rhythm Association. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">111</span>:659–670.</span> [<a href="/pubmed/15655131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Brugada+syndrome:+Report+of+the+Second+Consensus+Conference+Endorsed+by+the+Heart+Rhythm+Society+and+the+European+Heart+Rhythm+Association&amp;author=C+Antzelevitch&amp;volume=111&amp;publication_year=2005&amp;pages=659-670&amp;pmid=15655131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Kapplinger JD, et al.  An international compendium of mutations in the <em>SCN5A</em>-encoded cardiac sodium channel in patients referred for Brugada syndrome genetic testing. <span><span class="ref-journal">Heart Rhythm. </span>2010;<span class="ref-vol">7</span>:33–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2822446/">PMC free article</a>]</span> [<a href="/pubmed/20129283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Heart+Rhythm&amp;title=An+international+compendium+of+mutations+in+the+SCN5A-encoded+cardiac+sodium+channel+in+patients+referred+for+Brugada+syndrome+genetic+testing&amp;author=JD+Kapplinger&amp;volume=7&amp;publication_year=2010&amp;pages=33-46&amp;pmid=20129283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Chambers JC, et al.  Genetic variation in <em>SCN10A</em> influences cardiac conduction. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:149–152.</span> [<a href="/pubmed/20062061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+SCN10A+influences+cardiac+conduction&amp;author=JC+Chambers&amp;volume=42&amp;publication_year=2010&amp;pages=149-152&amp;pmid=20062061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Holm H, et al.  Several common variants modulate heart rate, PR interval and QRS duration. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:117–122.</span> [<a href="/pubmed/20062063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Several+common+variants+modulate+heart+rate,+PR+interval+and+QRS+duration&amp;author=H+Holm&amp;volume=42&amp;publication_year=2010&amp;pages=117-122&amp;pmid=20062063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Pfeufer A, et al.  Genome-wide association study of PR interval. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:153–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850197/">PMC free article</a>]</span> [<a href="/pubmed/20062060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+PR+interval&amp;author=A+Pfeufer&amp;volume=42&amp;publication_year=2010&amp;pages=153-159&amp;pmid=20062060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Sotoodehnia N, et al.  Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:1068–1076.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3338195/">PMC free article</a>]</span> [<a href="/pubmed/21076409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+in+22+loci+are+associated+with+QRS+duration+and+cardiac+ventricular+conduction&amp;author=N+Sotoodehnia&amp;volume=42&amp;publication_year=2010&amp;pages=1068-1076&amp;pmid=21076409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Leimeister C, Externbrink A, Klamt B, Gessler M. <em>Hey</em> genes: a novel subfamily of hairy- and Enhancer of split related genes specifically expressed during mouse embryogenesis. <span><span class="ref-journal">Mech Dev. </span>1999;<span class="ref-vol">85</span>:173–177.</span> [<a href="/pubmed/10415358" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mech+Dev&amp;title=Hey+genes:+a+novel+subfamily+of+hairy-+and+Enhancer+of+split+related+genes+specifically+expressed+during+mouse+embryogenesis&amp;author=C+Leimeister&amp;author=A+Externbrink&amp;author=B+Klamt&amp;author=M+Gessler&amp;volume=85&amp;publication_year=1999&amp;pages=173-177&amp;pmid=10415358&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Straus SMJM, et al.  The incidence of sudden cardiac death in the general population. <span><span class="ref-journal">J Clin Epidemiol. </span>2004;<span class="ref-vol">57</span>:98–102.</span> [<a href="/pubmed/15019016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Epidemiol&amp;title=The+incidence+of+sudden+cardiac+death+in+the+general+population&amp;author=SMJM+Straus&amp;volume=57&amp;publication_year=2004&amp;pages=98-102&amp;pmid=15019016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Priori SG, et al.  Task Force on Sudden Cardiac Death, European Society of Cardiology, Summary of Recommendations. <span><span class="ref-journal">Europace. </span>2002;<span class="ref-vol">4</span>:3–18.</span> [<a href="/pubmed/12478833" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Europace&amp;title=Task+Force+on+Sudden+Cardiac+Death,+European+Society+of+Cardiology,+Summary+of+Recommendations&amp;author=SG+Priori&amp;volume=4&amp;publication_year=2002&amp;pages=3-18&amp;pmid=11858152&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Chen Q, et al.  Genetic basis and molecular mechanism for idiopathic ventricular fibrillation. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">392</span>:293–296.</span> [<a href="/pubmed/9521325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+basis+and+molecular+mechanism+for+idiopathic+ventricular+fibrillation&amp;author=Q+Chen&amp;volume=392&amp;publication_year=1998&amp;pages=293-296&amp;pmid=9521325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Wilde AAM, et al.  The pathophysiological mechanism underlying Brugada syndrome: depolarization versus repolarization. <span><span class="ref-journal">J Mol Cell Cardiol. </span>2010;<span class="ref-vol">49</span>:543–553.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2932806/">PMC free article</a>]</span> [<a href="/pubmed/20659475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Cell+Cardiol&amp;title=The+pathophysiological+mechanism+underlying+Brugada+syndrome:+depolarization+versus+repolarization&amp;author=AAM+Wilde&amp;volume=49&amp;publication_year=2010&amp;pages=543-553&amp;pmid=20659475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Crotti L, et al.  Spectrum and prevalence of mutations involving BrS1-through BrS12-susceptibility genes in a cohort of unrelated patients referred for Brugada syndrome genetic testing: implications for genetic testing. <span><span class="ref-journal">J Am Coll Cardiol. </span>2012;<span class="ref-vol">60</span>:1410–1418.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3624764/">PMC free article</a>]</span> [<a href="/pubmed/22840528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Cardiol&amp;title=Spectrum+and+prevalence+of+mutations+involving+BrS1-through+BrS12-susceptibility+genes+in+a+cohort+of+unrelated+patients+referred+for+Brugada+syndrome+genetic+testing:+implications+for+genetic+testing&amp;author=L+Crotti&amp;volume=60&amp;publication_year=2012&amp;pages=1410-1418&amp;pmid=22840528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Priori SG, et al.  Clinical and genetic heterogeneity of right bundle branch block and ST-segment elevation syndrome: a prospective evaluation of 52 families. <span><span class="ref-journal">Circulation. </span>2000;<span class="ref-vol">102</span>:2509–2515.</span> [<a href="/pubmed/11076825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Clinical+and+genetic+heterogeneity+of+right+bundle+branch+block+and+ST-segment+elevation+syndrome:+a+prospective+evaluation+of+52+families&amp;author=SG+Priori&amp;volume=102&amp;publication_year=2000&amp;pages=2509-2515&amp;pmid=11076825&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Probst V, et al.  <em>SCN5A</em> mutations and the role of genetic background in the pathophysiology of Brugada syndrome (clinical perspective) <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2009;<span class="ref-vol">2</span>:552–557.</span> [<a href="/pubmed/20031634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=SCN5A+mutations+and+the+role+of+genetic+background+in+the+pathophysiology+of+Brugada+syndrome+(clinical+perspective)&amp;author=V+Probst&amp;volume=2&amp;publication_year=2009&amp;pages=552-557&amp;pmid=20031634&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Schulze-Bahr E, et al.  Sodium channel gene (<em>SCN5A</em>) mutations in 44 index patients with Brugada syndrome: different incidences in familial and sporadic disease. <span><span class="ref-journal">Hum Mutat. </span>2003;<span class="ref-vol">21</span>:651–652.</span> [<a href="/pubmed/14961552" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=Sodium+channel+gene+(SCN5A)+mutations+in+44+index+patients+with+Brugada+syndrome:+different+incidences+in+familial+and+sporadic+disease&amp;author=E+Schulze-Bahr&amp;volume=21&amp;publication_year=2003&amp;pages=651-652&amp;pmid=14961552&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Hermida JS, et al.  Prospective evaluation of the familial prevalence of the Brugada syndrome. <span><span class="ref-journal">Am J Cardiol. </span>2010;<span class="ref-vol">106</span>:1758–1762.</span> [<a href="/pubmed/21126620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Cardiol&amp;title=Prospective+evaluation+of+the+familial+prevalence+of+the+Brugada+syndrome&amp;author=JS+Hermida&amp;volume=106&amp;publication_year=2010&amp;pages=1758-1762&amp;pmid=21126620&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Balkau B. An epidemiologic survey from a network of French Health Examination Centres, (D.E.S.I.R.): epidemiologic data on the insulin resistance syndrome. <span><span class="ref-journal">Rev Epidemiol Sante Publique. </span>1996;<span class="ref-vol">44</span>:373–375.</span> [<a href="/pubmed/8927780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Rev+Epidemiol+Sante+Publique&amp;title=An+epidemiologic+survey+from+a+network+of+French+Health+Examination+Centres,+(D.E.S.I.R.):+epidemiologic+data+on+the+insulin+resistance+syndrome&amp;author=B+Balkau&amp;volume=44&amp;publication_year=1996&amp;pages=373-375&amp;pmid=8927780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Arking DE, et al.  A common genetic variant in the <em>NOS1</em> regulator <em>NOS1AP</em> modulates cardiac repolarization. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:644–651.</span> [<a href="/pubmed/16648850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+genetic+variant+in+the+NOS1+regulator+NOS1AP+modulates+cardiac+repolarization&amp;author=DE+Arking&amp;volume=38&amp;publication_year=2006&amp;pages=644-651&amp;pmid=16648850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Newton-Cheh C, et al.  Common variants at ten loci influence QT interval duration in the QTGEN Study. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:399–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701449/">PMC free article</a>]</span> [<a href="/pubmed/19305408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+ten+loci+influence+QT+interval+duration+in+the+QTGEN+Study&amp;author=C+Newton-Cheh&amp;volume=41&amp;publication_year=2009&amp;pages=399-406&amp;pmid=19305408&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Pfeufer A, et al.  Common variants at ten loci modulate the QT interval duration in the QTSCD Study. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:407–414.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2976045/">PMC free article</a>]</span> [<a href="/pubmed/19305409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+ten+loci+modulate+the+QT+interval+duration+in+the+QTSCD+Study&amp;author=A+Pfeufer&amp;volume=41&amp;publication_year=2009&amp;pages=407-414&amp;pmid=19305409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Eijgelsheim M, et al.  Genome-wide association analysis identifies multiple loci related to resting heart rate. <span><span class="ref-journal">Hum Mol Genet. </span>2010;<span class="ref-vol">19</span>:3885–3894.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3657480/">PMC free article</a>]</span> [<a href="/pubmed/20639392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+analysis+identifies+multiple+loci+related+to+resting+heart+rate&amp;author=M+Eijgelsheim&amp;volume=19&amp;publication_year=2010&amp;pages=3885-3894&amp;pmid=20639392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Marroni F, et al.  A genome-wide association scan of RR and QT interval duration in 3 European genetically isolated populations: the EUROSPAN project. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2009;<span class="ref-vol">2</span>:322–328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2760953/">PMC free article</a>]</span> [<a href="/pubmed/20031603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=A+genome-wide+association+scan+of+RR+and+QT+interval+duration+in+3+European+genetically+isolated+populations:+the+EUROSPAN+project&amp;author=F+Marroni&amp;volume=2&amp;publication_year=2009&amp;pages=322-328&amp;pmid=20031603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Sladek R, et al.  A genome-wide association study identifies novel risk loci for type 2 diabetes. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:881–885.</span> [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes&amp;author=R+Sladek&amp;volume=445&amp;publication_year=2007&amp;pages=881-885&amp;pmid=17293876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Manolio TA, et al.  Finding the missing heritability of complex diseases. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>:747–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=TA+Manolio&amp;volume=461&amp;publication_year=2009&amp;pages=747-753&amp;pmid=19812666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Akopian AN, Sivilotti L, Wood JN. A tetrodotoxin-resistant voltage-gated sodium channel expressed by sensory neurons. <span><span class="ref-journal">Nature. </span>1996;<span class="ref-vol">379</span>:257–262.</span> [<a href="/pubmed/8538791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+tetrodotoxin-resistant+voltage-gated+sodium+channel+expressed+by+sensory+neurons&amp;author=AN+Akopian&amp;author=L+Sivilotti&amp;author=JN+Wood&amp;volume=379&amp;publication_year=1996&amp;pages=257-262&amp;pmid=8538791&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Verkerk AO, et al.  Functional Nav1.8 channels in intracardiac neurons: the link between <em>SCN10A</em> and cardiac electrophysiology. <span><span class="ref-journal">Circ Res. </span>2012;<span class="ref-vol">111</span>:333–343.</span> [<a href="/pubmed/22723301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Functional+Nav1.8+channels+in+intracardiac+neurons:+the+link+between+SCN10A+and+cardiac+electrophysiology&amp;author=AO+Verkerk&amp;volume=111&amp;publication_year=2012&amp;pages=333-343&amp;pmid=22723301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Yang T, et al.  Blocking Scn10a channels in heart reduces late sodium current and is antiarrhythmic. <span><span class="ref-journal">Circ Res. </span>2012;<span class="ref-vol">111</span>:322–332.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3412150/">PMC free article</a>]</span> [<a href="/pubmed/22723299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Blocking+Scn10a+channels+in+heart+reduces+late+sodium+current+and+is+antiarrhythmic&amp;author=T+Yang&amp;volume=111&amp;publication_year=2012&amp;pages=322-332&amp;pmid=22723299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Pallante BA, et al.  Contactin-2 expression in the cardiac Purkinje fiber network. <span><span class="ref-journal">Circ Arrhythm Electrophysiol. </span>2010;<span class="ref-vol">3</span>:186–194.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3068837/">PMC free article</a>]</span> [<a href="/pubmed/20110552" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Arrhythm+Electrophysiol&amp;title=Contactin-2+expression+in+the+cardiac+Purkinje+fiber+network&amp;author=BA+Pallante&amp;volume=3&amp;publication_year=2010&amp;pages=186-194&amp;pmid=20110552&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">van den Boogaard M, et al.  Genetic variation in T-box binding element functionally affects <em>SCN5A</em>/<em>SCN10A</em> enhancer. <span><span class="ref-journal">J Clin Invest. </span>2012;<span class="ref-vol">122</span>:2519–2530.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386824/">PMC free article</a>]</span> [<a href="/pubmed/22706305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Genetic+variation+in+T-box+binding+element+functionally+affects+SCN5A/SCN10A+enhancer&amp;author=M+van+den+Boogaard&amp;volume=122&amp;publication_year=2012&amp;pages=2519-2530&amp;pmid=22706305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Arnolds DE, et al.  TBX5 drives Scn5a expression to regulate cardiac conduction system function. <span><span class="ref-journal">J Clin Invest. </span>2012;<span class="ref-vol">122</span>:2509–2518.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386825/">PMC free article</a>]</span> [<a href="/pubmed/22728936" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=TBX5+drives+Scn5a+expression+to+regulate+cardiac+conduction+system+function&amp;author=DE+Arnolds&amp;volume=122&amp;publication_year=2012&amp;pages=2509-2518&amp;pmid=22728936&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Shao W, Halachmi S, Brown M. ERAP140, a conserved tissue-specific nuclear receptor coactivator. <span><span class="ref-journal">Mol Cell Biol. </span>2002;<span class="ref-vol">22</span>:3358–3372.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC133794/">PMC free article</a>]</span> [<a href="/pubmed/11971969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=ERAP140,+a+conserved+tissue-specific+nuclear+receptor+coactivator&amp;author=W+Shao&amp;author=S+Halachmi&amp;author=M+Brown&amp;volume=22&amp;publication_year=2002&amp;pages=3358-3372&amp;pmid=11971969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Fischer A, Gessler M. <em>Hey</em> genes in cardiovascular development. <span><span class="ref-journal">Trends Cardiovasc Med. </span>2003;<span class="ref-vol">13</span>:221–226.</span> [<a href="/pubmed/12922017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Cardiovasc+Med&amp;title=Hey+genes+in+cardiovascular+development&amp;author=A+Fischer&amp;author=M+Gessler&amp;volume=13&amp;publication_year=2003&amp;pages=221-226&amp;pmid=12922017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Xin M, et al.  Essential roles of the bHLH transcription factor Hrt2 in repression of atrial gene expression and maintenance of postnatal cardiac function. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:7975–7980.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1876557/">PMC free article</a>]</span> [<a href="/pubmed/17468400" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Essential+roles+of+the+bHLH+transcription+factor+Hrt2+in+repression+of+atrial+gene+expression+and+maintenance+of+postnatal+cardiac+function&amp;author=M+Xin&amp;volume=104&amp;publication_year=2007&amp;pages=7975-7980&amp;pmid=17468400&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Kokubo H, et al.  Targeted disruption of <em>hesr2</em> results in atrioventricular valve anomalies that lead to heart dysfunction. <span><span class="ref-journal">Circ Res. </span>2004;<span class="ref-vol">95</span>:540–547.</span> [<a href="/pubmed/15297376" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Targeted+disruption+of+hesr2+results+in+atrioventricular+valve+anomalies+that+lead+to+heart+dysfunction&amp;author=H+Kokubo&amp;volume=95&amp;publication_year=2004&amp;pages=540-547&amp;pmid=15297376&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Gessler M, et al.  Mouse gridlock: no aortic coarctation or deficiency, but fatal cardiac defects in <em>Hey2</em><sup>−/−</sup> mice. <span><span class="ref-journal">Curr Biol. </span>2002;<span class="ref-vol">12</span>:1601–1604.</span> [<a href="/pubmed/12372253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;title=Mouse+gridlock:+no+aortic+coarctation+or+deficiency,+but+fatal+cardiac+defects+in+Hey2−/−+mice&amp;author=M+Gessler&amp;volume=12&amp;publication_year=2002&amp;pages=1601-1604&amp;pmid=12372253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Sakata Y, et al.  Ventricular septal defect and cardiomyopathy in mice lacking the transcription factor CHF1/Hey2. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;<span class="ref-vol">99</span>:16197–16202.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC138588/">PMC free article</a>]</span> [<a href="/pubmed/12454287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Ventricular+septal+defect+and+cardiomyopathy+in+mice+lacking+the+transcription+factor+CHF1/Hey2&amp;author=Y+Sakata&amp;volume=99&amp;publication_year=2002&amp;pages=16197-16202&amp;pmid=12454287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Koibuchi N, Chin MT. CHF1/Hey2 plays a pivotal role in left ventricular maturation through suppression of ectopic atrial gene expression. <span><span class="ref-journal">Circ Res. </span>2007;<span class="ref-vol">100</span>:850–855.</span> [<a href="/pubmed/17332425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=CHF1/Hey2+plays+a+pivotal+role+in+left+ventricular+maturation+through+suppression+of+ectopic+atrial+gene+expression&amp;author=N+Koibuchi&amp;author=MT+Chin&amp;volume=100&amp;publication_year=2007&amp;pages=850-855&amp;pmid=17332425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Fischer A, et al.  Hey basic helix-loop-helix transcription factors are repressors of GATA4 and GATA6 and restrict expression of the GATA target gene <em>ANF</em> in fetal hearts. <span><span class="ref-journal">Mol Cell Biol. </span>2005;<span class="ref-vol">25</span>:8960–8970.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1265774/">PMC free article</a>]</span> [<a href="/pubmed/16199874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Hey+basic+helix-loop-helix+transcription+factors+are+repressors+of+GATA4+and+GATA6+and+restrict+expression+of+the+GATA+target+gene+ANF+in+fetal+hearts&amp;author=A+Fischer&amp;volume=25&amp;publication_year=2005&amp;pages=8960-8970&amp;pmid=16199874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Remme CA, et al.  The cardiac sodium channel displays differential distribution in the conduction system and transmural heterogeneity in the murine ventricular myocardium. <span><span class="ref-journal">Basic Res Cardiol. </span>2009;<span class="ref-vol">104</span>:511–522.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722719/">PMC free article</a>]</span> [<a href="/pubmed/19255801" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Basic+Res+Cardiol&amp;title=The+cardiac+sodium+channel+displays+differential+distribution+in+the+conduction+system+and+transmural+heterogeneity+in+the+murine+ventricular+myocardium&amp;author=CA+Remme&amp;volume=104&amp;publication_year=2009&amp;pages=511-522&amp;pmid=19255801&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Nademanee K, et al.  Prevention of ventricular fibrillation episodes in Brugada syndrome by catheter ablation over the anterior right ventricular outflow tract epicardium. <span><span class="ref-journal">Circulation. </span>2011;<span class="ref-vol">123</span>:1270–1279.</span> [<a href="/pubmed/21403098" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Prevention+of+ventricular+fibrillation+episodes+in+Brugada+syndrome+by+catheter+ablation+over+the+anterior+right+ventricular+outflow+tract+epicardium&amp;author=K+Nademanee&amp;volume=123&amp;publication_year=2011&amp;pages=1270-1279&amp;pmid=21403098&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Sakuma M, et al.  Incidence and outcome of osteoporotic fractures in 2004 in Sado City, Niigata Prefecture, Japan. <span><span class="ref-journal">J Bone Miner Metab. </span>2008;<span class="ref-vol">26</span>:373–378.</span> [<a href="/pubmed/18600404" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Metab&amp;title=Incidence+and+outcome+of+osteoporotic+fractures+in+2004+in+Sado+City,+Niigata+Prefecture,+Japan&amp;author=M+Sakuma&amp;volume=26&amp;publication_year=2008&amp;pages=373-378&amp;pmid=18600404&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Frayley C, Raftery AE.  <span class="ref-journal">MCLUST Version 3 for R: Normal Mixture Modeling and Model-Based Clustering.</span> Department of Statistics, University of Washington; Seattle: 2006.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=MCLUST+Version+3+for+R:+Normal+Mixture+Modeling+and+Model-Based+Clustering&amp;author=C+Frayley&amp;author=AE+Raftery&amp;publication_year=2006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Byers S, Raftery AE. Nearest-neighbor clutter removal for estimating features in spatial point processes. <span><span class="ref-journal">J Am Stat Assoc. </span>1998;<span class="ref-vol">93</span>:577–584.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Stat+Assoc&amp;title=Nearest-neighbor+clutter+removal+for+estimating+features+in+spatial+point+processes&amp;author=S+Byers&amp;author=AE+Raftery&amp;volume=93&amp;publication_year=1998&amp;pages=577-584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Postel-Vinay S, et al.  Common variants near <em>TARDBP</em> and <em>EGR2</em> are associated with susceptibility to Ewing sarcoma. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:323–327.</span> [<a href="/pubmed/22327514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+near+TARDBP+and+EGR2+are+associated+with+susceptibility+to+Ewing+sarcoma&amp;author=S+Postel-Vinay&amp;volume=44&amp;publication_year=2012&amp;pages=323-327&amp;pmid=22327514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Pirinen M, Donnelly P, Spencer CCA. Including known covariates can reduce power to detect genetic effects in case-control studies. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:848–851.</span> [<a href="/pubmed/22820511" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Including+known+covariates+can+reduce+power+to+detect+genetic+effects+in+case-control+studies&amp;author=M+Pirinen&amp;author=P+Donnelly&amp;author=CCA+Spencer&amp;volume=44&amp;publication_year=2012&amp;pages=848-851&amp;pmid=22820511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Stouffer SA, Suchman EA, Devinney LC, Star SA, Williams RM., Jr . <span class="ref-journal">Studies in Social Psychology in World War II.</span> Vol. 1. Princeton University Press; Princeton, NJ: 1949. The American Soldier: Adjustment During Army Life. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Studies+in+Social+Psychology+in+World+War+II&amp;author=SA+Stouffer&amp;author=EA+Suchman&amp;author=LC+Devinney&amp;author=SA+Star&amp;author=RM+Williams&amp;publication_year=1949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Delaneau O, Marchini J, Zagury JF. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>:179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O+Delaneau&amp;author=J+Marchini&amp;author=JF+Zagury&amp;volume=9&amp;publication_year=2012&amp;pages=179-181&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Howie B, Marchini J, Stephens M. Genotype imputation with thousands of genomes. <span><span class="ref-journal">G3. </span>2011;<span class="ref-vol">1</span>:457–470.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3276165/">PMC free article</a>]</span> [<a href="/pubmed/22384356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=G3&amp;title=Genotype+imputation+with+thousands+of+genomes&amp;author=B+Howie&amp;author=J+Marchini&amp;author=M+Stephens&amp;volume=1&amp;publication_year=2011&amp;pages=457-470&amp;pmid=22384356&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Kolder ICRM, Tanck MWT, Bezzina CR. Common genetic variation modulating cardiac ECG parameters and susceptibility to sudden cardiac death. <span><span class="ref-journal">J Mol Cell Cardiol. </span>2012;<span class="ref-vol">52</span>:620–629.</span> [<a href="/pubmed/22248531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Cell+Cardiol&amp;title=Common+genetic+variation+modulating+cardiac+ECG+parameters+and+susceptibility+to+sudden+cardiac+death&amp;author=ICRM+Kolder&amp;author=MWT+Tanck&amp;author=CR+Bezzina&amp;volume=52&amp;publication_year=2012&amp;pages=620-629&amp;pmid=22248531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Meyre D, et al.  Genome-wide association study for early-onset and morbid adult obesity identifies three new risk loci in European populations. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:157–159.</span> [<a href="/pubmed/19151714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+for+early-onset+and+morbid+adult+obesity+identifies+three+new+risk+loci+in+European+populations&amp;author=D+Meyre&amp;volume=41&amp;publication_year=2009&amp;pages=157-159&amp;pmid=19151714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Higgins JPT, Thompson SG. Quantifying heterogeneity in a meta-analysis. <span><span class="ref-journal">Stat Med. </span>2002;<span class="ref-vol">21</span>:1539–1558.</span> [<a href="/pubmed/12111919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Quantifying+heterogeneity+in+a+meta-analysis&amp;author=JPT+Higgins&amp;author=SG+Thompson&amp;volume=21&amp;publication_year=2002&amp;pages=1539-1558&amp;pmid=12111919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Little RJA, Rubin DB.  <span class="ref-journal">Statistical Analysis with Missing Data.</span> Wiley and Sons; New York: 1987.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Statistical+Analysis+with+Missing+Data&amp;author=RJA+Little&amp;author=DB+Rubin&amp;publication_year=1987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Rubin DB. <span><span class="ref-journal">Multiple Imputation for Nonresponse in Surveys. </span>2008</span>  <a href="http://onlinelibrary.wiley.com/book/10.1002/9780470316696" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://onlinelibrary.wiley.com/book/10.1002/9780470316696</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Lin S, Chakravarti A, Cutler DJ. Exhaustive allelic transmission disequilibrium tests as a new approach to genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:1181–1188.</span> [<a href="/pubmed/15502828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Exhaustive+allelic+transmission+disequilibrium+tests+as+a+new+approach+to+genome-wide+association+studies&amp;author=S+Lin&amp;author=A+Chakravarti&amp;author=DJ+Cutler&amp;volume=36&amp;publication_year=2004&amp;pages=1181-1188&amp;pmid=15502828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Wray NR, Yang J, Goddard ME, Visscher PM. The genetic interpretation of area under the ROC curve in genomic profiling. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000864.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2829056/">PMC free article</a>]</span> [<a href="/pubmed/20195508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=The+genetic+interpretation+of+area+under+the+ROC+curve+in+genomic+profiling&amp;author=NR+Wray&amp;author=J+Yang&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=6&amp;publication_year=2010&amp;pages=e1000864&amp;pmid=20195508&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Remme CA, et al.  Overlap syndrome of cardiac sodium channel disease in mice carrying the equivalent mutation of human <em>SCN5A</em>-1795insD. <span><span class="ref-journal">Circulation. </span>2006;<span class="ref-vol">114</span>:2584–2594.</span> [<a href="/pubmed/17145985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Overlap+syndrome+of+cardiac+sodium+channel+disease+in+mice+carrying+the+equivalent+mutation+of+human+SCN5A-1795insD&amp;author=CA+Remme&amp;volume=114&amp;publication_year=2006&amp;pages=2584-2594&amp;pmid=17145985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3869788&amp;issue-id=231361&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139688907878576title">Abstract</a></li><li><a href="#S4title">ONLINE METHODS</a></li><li><a href="#S22title">Supplementary Material</a></li><li><a href="#S23title">Acknowledgments</a></li><li><a href="#idm139688906068208title">Footnotes</a></li><li><a href="#idm139688906762288title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3869788/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3869788/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3869788/pdf/nihms528125.pdf">PDF (621K)</a></li> | <li><a href="#" data-citationid="PMC3869788" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3869788%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3869788%2F&amp;text=Common%20variants%20at%20SCN5A-SCN10A%20and%20HEY2%20are%20associated%20with%20Brugada%20syndrome%2C%20a%20rare%20disease%20with%20high%20risk%20of%20sudden%20cardiac%20death" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3869788%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23872634" data-aiid="3869788" data-aid="3869788" data-iid="231361" data-domainid="319" data-domain="nihpa" data-accid="PMC3869788" data-md5="0f8a082bf4ed91f1cfb859d826f345cd"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T11:06:45-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CBF6CDD555CE1000000000B3506E6&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3869788%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3869788/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>