<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="blood"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3369619/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Evaluation of published single nucleotide polymorphisms associated with acute GVHD</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3369619/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Blood"><meta name="citation_title" content="Evaluation of published single nucleotide polymorphisms associated with acute GVHD"><meta name="citation_authors" content="Jason W. Chien, Xinyi Cindy Zhang, Wenhong Fan, Hongwei Wang, Lue Ping Zhao, Paul J. Martin, Barry E. Storer, Michael Boeckh, Edus H. Warren, John A. Hansen"><meta name="citation_date" content="31 May 2012"><meta name="citation_issue" content="22"><meta name="citation_volume" content="119"><meta name="citation_firstpage" content="5311"><meta name="citation_doi" content="10.1182/blood-2011-09-371153"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3369619/?report=abstract"><meta name="citation_pmid" content="22282500"><meta name="DC.Title" content="Evaluation of published single nucleotide polymorphisms associated with acute GVHD"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="The American Society of Hematology"><meta name="DC.Contributor" content="Jason W. Chien"><meta name="DC.Contributor" content="Xinyi Cindy Zhang"><meta name="DC.Contributor" content="Wenhong Fan"><meta name="DC.Contributor" content="Hongwei Wang"><meta name="DC.Contributor" content="Lue Ping Zhao"><meta name="DC.Contributor" content="Paul J. Martin"><meta name="DC.Contributor" content="Barry E. Storer"><meta name="DC.Contributor" content="Michael Boeckh"><meta name="DC.Contributor" content="Edus H. Warren"><meta name="DC.Contributor" content="John A. Hansen"><meta name="DC.Date" content="2012 May 31"><meta name="DC.Identifier" content="10.1182/blood-2011-09-371153"><meta name="DC.Language" content="en"><meta property="og:title" content="Evaluation of published single nucleotide polymorphisms associated with acute GVHD"><meta property="og:type" content="article"><meta property="og:description" content="Candidate genetic associations with acute GVHD (aGVHD) were evaluated with the use of genotyped and imputed single-nucleotide polymorphism data from genome-wide scans of 1298 allogeneic hematopoietic cell transplantation (HCT) donors and recipients. Of ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3369619/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-blood.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3369619/epub/">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CB806DD53BCF1000000000231016C.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369619%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Blood%22[journal]" class="navlink">Blood</a></li><li class="accid">PMC3369619</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-blood.gif" alt="Logo of blood" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="139,56,200,76" alt="Original Article" title="Original Article" href="http://www.bloodjournal.org/content/119/22/5311" ref="reftype=publisher&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7CArticle%7CFullText" target="pmc_ext"><area shape="rect" coords="0,0,157,57" alt="Blood Journal" title="Blood Journal" href="http://www.bloodjournal.org/" ref="reftype=publisher&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="200,56,258,76" alt="Current Issue" title="Current Issue" href="http://www.bloodjournal.org/content/current" ref="reftype=publisher&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="257,56,311,76" alt="About Blood" title="About Blood" href="http://www.bloodjournal.org/page/about-blood" ref="reftype=publisher&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="309,56,364,77" alt="Submissions" title="Submissions" href="https://submit.bloodjournal.org/" ref="reftype=publisher&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="363,56,420,78" alt="Subscriptions" title="Subscriptions" href="http://www.bloodjournal.org/page/subscriptions" ref="reftype=publisher&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="418,56,476,76" alt="Contact Us" title="Contact Us" href="http://bloodjournal.org" ref="reftype=publisher&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="poly" coords="450,2,431,9,425,27,429,43,437,54,459,55,471,52,483,42,486,29,479,11,466,2,449,2" alt="ASH Homepage" title="ASH Homepage" href="http://www.hematology.org" ref="reftype=publisher&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Blood</a></span>. 2012 May 31; 119(22): 5311–5319. </span></div><div><span class="fm-vol-iss-date">Prepublished online 2012 Jan 26. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1182%2Fblood-2011-09-371153" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1182/blood-2011-09-371153</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3369619</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22282500">22282500</a></div></div></div></div><h1 class="content-title">Evaluation of published single nucleotide polymorphisms associated with acute GVHD</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Chien%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713383294608" co-class="co-affbox">Jason W. Chien</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Zhang%20XC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713377419376" co-class="co-affbox">Xinyi Cindy Zhang</a>,<sup>3</sup> <a href="/pubmed/?term=Fan%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713376017280" co-class="co-affbox">Wenhong Fan</a>,<sup>3</sup> <a href="/pubmed/?term=Wang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713380690560" co-class="co-affbox">Hongwei Wang</a>,<sup>3</sup> <a href="/pubmed/?term=Zhao%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713368196464" co-class="co-affbox">Lue Ping Zhao</a>,<sup>3</sup> <a href="/pubmed/?term=Martin%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713382768512" co-class="co-affbox">Paul J. Martin</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Storer%20BE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713416888528" co-class="co-affbox">Barry E. Storer</a>,<sup>1</sup> <a href="/pubmed/?term=Boeckh%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713379279152" co-class="co-affbox">Michael Boeckh</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Warren%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713368267776" co-class="co-affbox">Edus H. Warren</a>,<sup>1,</sup><sup>2</sup> and  <a href="/pubmed/?term=Hansen%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500" class="affpopup" co-rid="_co_idm139713378955664" co-class="co-affbox">John A. Hansen</a><sup>1,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139713383294608"><h3 class="no_margin">Jason W. Chien</h3><p><sup>1</sup>Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA; </p><p><sup>2</sup>University of Washington School of Medicine, Seattle, WA; and </p><div>Find articles by <a href="/pubmed/?term=Chien%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Jason W. Chien</a></div></div><div id="_co_idm139713377419376"><h3 class="no_margin">Xinyi Cindy Zhang</h3><p><sup>3</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA</p><div>Find articles by <a href="/pubmed/?term=Zhang%20XC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Xinyi Cindy Zhang</a></div></div><div id="_co_idm139713376017280"><h3 class="no_margin">Wenhong Fan</h3><p><sup>3</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA</p><div>Find articles by <a href="/pubmed/?term=Fan%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Wenhong Fan</a></div></div><div id="_co_idm139713380690560"><h3 class="no_margin">Hongwei Wang</h3><p><sup>3</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA</p><div>Find articles by <a href="/pubmed/?term=Wang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Hongwei Wang</a></div></div><div id="_co_idm139713368196464"><h3 class="no_margin">Lue Ping Zhao</h3><p><sup>3</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA</p><div>Find articles by <a href="/pubmed/?term=Zhao%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Lue Ping Zhao</a></div></div><div id="_co_idm139713382768512"><h3 class="no_margin">Paul J. Martin</h3><p><sup>1</sup>Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA; </p><p><sup>2</sup>University of Washington School of Medicine, Seattle, WA; and </p><div>Find articles by <a href="/pubmed/?term=Martin%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Paul J. Martin</a></div></div><div id="_co_idm139713416888528"><h3 class="no_margin">Barry E. Storer</h3><p><sup>1</sup>Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA; </p><div>Find articles by <a href="/pubmed/?term=Storer%20BE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Barry E. Storer</a></div></div><div id="_co_idm139713379279152"><h3 class="no_margin">Michael Boeckh</h3><p><sup>1</sup>Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA; </p><p><sup>2</sup>University of Washington School of Medicine, Seattle, WA; and </p><div>Find articles by <a href="/pubmed/?term=Boeckh%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Michael Boeckh</a></div></div><div id="_co_idm139713368267776"><h3 class="no_margin">Edus H. Warren</h3><p><sup>1</sup>Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA; </p><p><sup>2</sup>University of Washington School of Medicine, Seattle, WA; and </p><div>Find articles by <a href="/pubmed/?term=Warren%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">Edus H. Warren</a></div></div><div id="_co_idm139713378955664"><h3 class="no_margin">John A. Hansen</h3><p><sup>1</sup>Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA; </p><p><sup>2</sup>University of Washington School of Medicine, Seattle, WA; and </p><div>Find articles by <a href="/pubmed/?term=Hansen%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22282500">John A. Hansen</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139713378844016_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139713378844016_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139713378844016_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139713378844016_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA; </div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>University of Washington School of Medicine, Seattle, WA; and </div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139713378844016_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Sep 6; Accepted 2012 Jan 22.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139713378844016_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2012 by The American Society of Hematology</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3369619/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm139713415673904" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139713415673904title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Candidate genetic associations with acute GVHD (aGVHD) were evaluated with the use of genotyped and imputed single-nucleotide polymorphism data from genome-wide scans of 1298 allogeneic hematopoietic cell transplantation (HCT) donors and recipients. Of 40 previously reported candidate SNPs, 6 were successfully genotyped, and 10 were imputed and passed criteria for analysis. Patient and donor genotypes were assessed for association with grades IIb-IV and III-IV aGVHD, stratified by donor type, in univariate and multivariate allelic, recessive and dominant models. Use of imputed genotypes to replicate previous <em>IL10</em> associations was validated. Similar to previous publications, the <em>IL6</em> donor genotype for rs1800795 was associated with a 20%-50% increased risk for grade IIb-IV aGVHD after unrelated HCT in the allelic (adjusted <em>P</em> = .011) and recessive (adjusted <em>P</em> = .0013) models. The donor genotype was associated with a 60% increase in risk for grade III-IV aGVHD after related HCT (adjusted <em>P</em> = .028). Other associations were found for <em>IL2</em>, <em>CTLA4</em>, <em>HPSE</em>, and <em>MTHFR</em> but were inconsistent with original publications. These results illustrate the advantages of using imputed single-nucleotide polymorphism data in genetic analyses and demonstrate the importance of validation in genetic association studies.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">During the past decade, many reports have identified genetic variants such as single-nucleotide polymorphisms (SNPs) and other polymorphisms that influence the risk of acute GVHD (aGVHD) after allogeneic hematopoietic cell transplantation (HCT). Many of these variants regulate the function of immune cells, their receptors, effector molecules, cytokines, and chemokines. Some results have led to the suggestion that pretransplantation assessment of these variants might help to assess the risk of adverse outcomes for each patient, guide the clinical management of patients who are at high risk, and ultimately serve as potential biologic targets for novel therapeutics.</p><p id="__p3" class="p p-last">Although these concepts have been greeted with some enthusiasm, the consistency of results leading to such suggestions has been questioned. Many of the reported results have been difficult to replicate in independent cohorts, and in no published studies have authors comprehensively evaluated these variants in the same cohort simultaneously. We recently had the opportunity to perform genome-wide scans of a cohort of patients and their donors with the use of an array that is informative for &gt; 500 000 SNPs. In conjunction with the vast HapMap resource for SNP genotypes across the entire genome, these data provided a unique opportunity to use either the genotyped or imputed SNP data to determine how many of the previously published associations could be replicated.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Literature search</h3><p id="__p4" class="p p-first-last">We performed a comprehensive PubMed search using the terms “acute GVHD” and “polymorphism” to identify all studies published by April 30, 2011, that reported an association between aGVHD and a genetic polymorphism at an α level &lt; 0.05. Studies that did not meet this threshold were not included. Studies that reported associations with alternative genetic variants such as deletions, microsatellites, or VNTRs were excluded because our SNP genotyping array is not informative for such variants. Because our group previously published associations between SNPs in the <em>IL10</em> and <em>IL10RB</em> genes and aGVHD using a cohort that overlaps with the current cohort,<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758154">1</a>,<a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758123">2</a></sup> we used imputed genotypes for these SNPs as a quality control measure to evaluate whether previously identified associations could be replicated with imputed genotypes.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Study population</h3><p id="__p5" class="p p-first">The source population included 1424 donor-recipient pairs randomly selected from among 3177 patients who received allogeneic HCT after myeloablative conditioning at the Fred Hutchinson Cancer Research Center and Seattle Cancer Care Alliance between 1992 and 2004. All recipient and donor samples were collected before HCT according to approved research protocols. Project-specific institutional research board approval from the Fred Hutchinson Cancer Research Center was obtained for the use of these samples. Recipient and donor demographic information collected at the time of pretransplantation evaluation, including sex, XY karyotype, ABO blood group, and race, was available through the FHCRC Clinical Research Division patient database (“Clinical Research Database”). All clinical data were prospectively collected and retrospectively reviewed.</p><p id="__p6">Clinical characteristics of the patients and aGVHD prevalence data are summarized in <a href="/pmc/articles/PMC3369619/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The cohort included recipients with either HLA-matched related donor (n = 612) or unrelated donors (n = 686) who underwent transplantation for a hematologic malignancy or myelodysplasia. Patients with related HLA-mismatched donors (n = 126) were excluded because of the limited number for meaningful stratified analysis. The pretransplantation disease risk category was determined on the basis of disease type and stage or remission/relapse status and classified as low, intermediate, or high. Patient-donor HLA matching was determined on the basis of genotyping for HLA-A, B, C, HLA-DRB1, and DQB1. All patients received T cell–replete BM or growth factor-mobilized blood cell grafts, and cyclosporine or tacrolimus plus methotrexate or mycophenylate for aGVHD prophylaxis. Conditioning regimens were categorized according to the use of total body irradiation. Peak severity of aGVHD after HCT, defined according to grade (0, I, IIa, IIb, III, IV), was recorded.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p7">Summary demographic and outcome data for the GWAS cohort</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139713379308608" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Variables</th><th rowspan="1" colspan="1" align="center">Patient</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">Number of patients</td><td rowspan="1" colspan="1" align="center">1298</td></tr><tr><td rowspan="1" colspan="1" align="left">Age, y</td><td rowspan="1" colspan="1" align="center">38.2 ± 14</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Sex match (patient/donor), n (%)</strong></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">    Male/male</td><td rowspan="1" colspan="1" align="center">446 (34.3)</td></tr><tr><td rowspan="1" colspan="1" align="left">    Male/female</td><td rowspan="1" colspan="1" align="center">309 (23.8)</td></tr><tr><td rowspan="1" colspan="1" align="left">    Female/male</td><td rowspan="1" colspan="1" align="center">288 (22.2)</td></tr><tr><td rowspan="1" colspan="1" align="left">    Female/female</td><td rowspan="1" colspan="1" align="center">255 (19.7)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Race, n (%)</strong></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">    White</td><td rowspan="1" colspan="1" align="center">1116 (86.0)</td></tr><tr><td rowspan="1" colspan="1" align="left">    Other</td><td rowspan="1" colspan="1" align="center">182 (14.0)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Diagnosis, n (%)</strong></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">    Acute leukemia</td><td rowspan="1" colspan="1" align="center">465 (35.82)</td></tr><tr><td rowspan="1" colspan="1" align="left">    CML</td><td rowspan="1" colspan="1" align="center">486 (37.44)</td></tr><tr><td rowspan="1" colspan="1" align="left">    MDS</td><td rowspan="1" colspan="1" align="center">229 (17.64)</td></tr><tr><td rowspan="1" colspan="1" align="left">    CLL, HD, and NHL</td><td rowspan="1" colspan="1" align="center">79 (6.1)</td></tr><tr><td rowspan="1" colspan="1" align="left">    Multiple myeloma</td><td rowspan="1" colspan="1" align="center">39 (3.0)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Disease risk, n (%)</strong></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">    Low</td><td rowspan="1" colspan="1" align="center">361 (27.8)</td></tr><tr><td rowspan="1" colspan="1" align="left">    Intermediate</td><td rowspan="1" colspan="1" align="center">594 (45.8)</td></tr><tr><td rowspan="1" colspan="1" align="left">    High</td><td rowspan="1" colspan="1" align="center">343 (26.4)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Donor type, n (%)</strong></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">    Unrelated donor</td><td rowspan="1" colspan="1" align="center">686 (52.8)</td></tr><tr><td rowspan="1" colspan="1" align="left">    Matched related donor</td><td rowspan="1" colspan="1" align="center">612 (47.2)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Stem cell source</strong></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">    BM, n (%)</td><td rowspan="1" colspan="1" align="center">921 (71.0)</td></tr><tr><td rowspan="1" colspan="1" align="left">    Peripheral blood</td><td rowspan="1" colspan="1" align="center">377 (29.0)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>TBI dose</strong></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">    None, n (%)</td><td rowspan="1" colspan="1" align="center">531 (40.9)</td></tr><tr><td rowspan="1" colspan="1" align="left">    ≤ 1200 Gy</td><td rowspan="1" colspan="1" align="center">366 (28.2)</td></tr><tr><td rowspan="1" colspan="1" align="left">    &gt; 1200 Gy</td><td rowspan="1" colspan="1" align="center">401 (30.9)</td></tr><tr><td rowspan="1" colspan="1" align="left">aGVHD IIb-IV, n (%)</td><td rowspan="1" colspan="1" align="center">829 (58.5)</td></tr><tr><td rowspan="1" colspan="1" align="left">aGVHD III-IV, n (%)</td><td rowspan="1" colspan="1" align="center">323 (25.0)</td></tr></tbody></table></div><div id="largeobj_idm139713379308608" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3369619/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139713383606800"><p id="__p8" class="p p-first-last">aGVHD indicates acute graft versus host disease; CML, chronic myelogenous leukemia; CLL, chronic lymphocytic leukemia; GWAS, genome-wide association study; HD, Hodgkin's disease; HLA, human leukocyte antigen; MDS, myelodysplastic syndrome; NHL, non-Hodgkin's lymphoma; and TBI, total body irradiation.</p></div></div></div><p id="__p9" class="p p-last">aGVHD was considered as 2 different phenotypes: grades 0-IIa versus grades IIb-IV, and grades 0-II versus grades III-IV. Grade IIa GVHD is a subset of grade II aGVHD that includes cases with upper gastrointestinal symptoms generally documented as aGVHD by biopsy, without diarrhea or with stool volume &lt; 1000 mL/d, without rash or with rash involving &lt; 50% of the body surface, and without liver involvement.<sup><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758114">3</a></sup> Grade IIb aGVHD is a subset of grade II GVHD that includes cases with rash involving &gt; 50% of the body surface (stage III skin disease) or with total serum bilirubin concentration between 2.0 and 2.9 mg/dL (stage I liver disease), with or without stage I gastrointestinal involvement. As previously reported, the overall incidence of grades II-IV GVHD is greater at our center than reported at other centers, primarily because of high sensitivity for the diagnosis of upper gastrointestinal GVHD.<sup><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758116">3</a></sup> The incidence of grades IIb-IV GVHD at our center is similar to the overall incidence of grades II-IV GVHD at other centers.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Sample preparation, genotyping, and imputations</h3><p id="__p10" class="p p-first">DNA specimens originated from blood mononuclear cells or EBV-transformed B-lymphocyte cell lines. Genomic DNA was extracted by a standard salting-out method with the use of the Puregene kit (QIAGEN). The quantity and purity of DNA in each sample was determined by UV absorption in a spectrophotometer (OD<sub>260</sub>) and by OD<sub>260/280</sub> ratio, respectively. After DNA quantitation and initial screening for verification of sample identity, DNA samples were distributed into 96-well plates (92 samples per plate) at a targeted concentration of 50-100 ng/μL. Each plate of 92 samples contained 4 duplicate samples carried forward from a previously prepared plate as a quality control indicator. The plates were then sealed, frozen at −20°C, and shipped on dry ice to the Affymetrix Service Laboratory (ASL) in Santa Clara, CA, for amplification and hybridization.</p><p id="__p11">All assays used the Affymetrix GeneChip Genome-Wide Human SNP Array 5.0, with hybridization and scanning at the ASL. After scanning of hybridized GeneChip arrays, the resulting raw probe intensities were evaluated with the Bayesian Robust Linear Model with Mahalanobis distance classifier (BRLMM) analysis tool adapted for the 5.0 Array (BRLMM-P). Data quality was assessed via 3 different methods: the Affymetrix “QC call rate,” the clustering call rate, and the sex call rate. Once the array data were cleared by internal ASL quality assessment, the genome-wide SNP data were transmitted to FHCRC, where they were subjected to further QC analysis. This analysis included assessment of overall sample quality using the DM and BRLMM-P call rate statistics generated by Affymetrix, adjustment of threshold standards to optimize genotyping accuracy, if necessary, and assessment and resolution of sample identity questions with the use of calculations for estimating genomic similarity between samples. Because sex and ABO type was known for all samples, PCR-based ABO and XY genotyping assays were implemented to determine sample quality and verify sample identity before submitting the DNA samples to the ASL for hybridization and scanning. A further evaluation of sample identity was performed via use of the Affymetrix-supplied genotypes obtained by clustering with the BRLMM-P algorithm for those samples passing the QC call rate threshold. The final SNP-based genotype data generated is referred to as the “GWAS-HCT” dataset.</p><p id="__p12" class="p">The candidate SNP genotype determination algorithm is summarized in <a href="/pmc/articles/PMC3369619/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>. If the original candidate SNP was not genotyped on our array, we evaluated the imputed genotype, which was obtained by imputing all the HapMap release 22 SNPs (combined panel of ethnic groups: CEU [Utah residents with Northern and Western European ancestry from the CEPH collection], YRI [Yoruba in Ibadan], CHB [Han Chinese in Beijing], and JPT [Japanese in Tokyo]) that were not genotyped on the Affymetrix 5.0 array using IMPUTE software (<a href="https://mathgen.stats.ox.ac.uk/impute/impute.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute.html</a>). At each unobserved HapMap SNP locus, 1 or more genotypes may be predicted with a positive posterior probability, which measures the probability of observing that genotype at the imputed locus. We assign the genotype with the maximum posterior probability as the imputed SNP genotype only if the maximum posterior probability exceeded 0.8. All genotyped and imputed SNPs that violated Hardy-Weinberg equilibrium with <em>P</em> &lt; .001, had a minor allele frequency &lt; 0.05, or had a &lt; 90% call rate were excluded from the analyses.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3369619/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139713452376144" class="figure"><a href="/pmc/articles/PMC3369619/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3369619_zh89991287730001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zh89991287730001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3369619/bin/zh89991287730001.jpg"></a></div><div id="largeobj_idm139713452376144" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3369619/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3369619/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p13"><strong>Candidate SNP genotype determination algorithm.</strong> HWE indicates Hardy-Weinberg equilibrium; MAF, minor allele frequency; and SNP, single-nucleotide polymorphism.</p></div></div></div></div><div id="__sec6" class="sec sec-last"><h3 id="__sec6title">Statistical analysis</h3><p id="__p14" class="p p-first">All analyses were performed with Matlab v.2010b (MathWorks). All analyses were stratified by donor type (matched related donor [MRD] vs unrelated donor [URD]) and were examined for recipients and donors in separate univariate and multivariate Cox regression models, censored at the time of death or onset of recurrent malignancy. Multivariate Cox models adjusted for clinical factors recently found to be associated with the risk of aGVHD female donor and male recipient status and total body irradiation.<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758128">4</a></sup> For URDs, an additional in-model adjustment was made for HLA match. We also incorporated as covariates principal components derived from a principal component analysis of population stratification,<sup><a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758126">5</a></sup> where the first 4 components were included as covariates in the final adjusted analysis.</p><p id="__p15" class="p p-last">Each SNP was assessed for allelic and genotypic (recessive and dominant models) modes of transmission, stratified according to donor type (HLA-matched related versus unrelated). For a SNP with a major allele “<em>a</em>” and a minor allele “<em>b</em>,” the recessive model tests the hypothesis that the genotype “<em>bb</em>” is associated with an increased or decreased risk compared with the collective genotypes “<em>ab</em> and “<em>aa</em>.” The dominant model tests the hypothesis that the collective genotypes “<em>bb</em>” and “<em>ab</em>” are associated with an increased or decreased risk compared with the genotype “<em>aa</em>.” The allelic model tests the hypothesis that the minor allele <em>b</em> is associated with an increased or decreased risk compared with the major allele <em>a</em>, and the number of copies of the minor allele is modeled as an additive effect. Because the main goal of this analysis was to replicate published associations with candidate SNPs, a 2-sided <em>P</em> ≤ .05 was selected as the threshold of significance, despite the multiple comparisons made in this analysis.</p></div></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title">Results</h2><div id="__sec8" class="sec sec-first"><h3 id="__sec8title">Candidate gene and SNP selection</h3><p id="__p16" class="p p-first">A search of the PubMed database identified 41 publications that reported associations of donor or recipient genotypes for 40 candidate SNPs in 22 genes with the risk of aGVHD: <em>CD31</em>,<sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758108">6</a>,<a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758084">7</a></sup>
<em>CTLA4</em>,<sup><a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758147">8</a></sup>
<em>ESR1</em>,<sup><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758083">9</a></sup>
<em>FAS</em>,<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758105">10</a></sup>
<em>HMGB1</em>,<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758149">11</a></sup>
<em>HPSE</em>,<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758087">12</a></sup>
<em>HSPA1L</em>,<sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758103">13</a>,<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758102">14</a></sup>
<em>IL1</em>α,<sup><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758077">9</a>,<a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758153">15</a></sup>
<em>IL1</em>β,<sup><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758129">16</a></sup>
<em>IL2</em>,<sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758127">17</a></sup>
<em>IL6</em>,<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758098">10</a>,<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758131">18</a>,<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758092">19</a></sup>
<em>IL10</em>,<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758091">1</a>,<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758140">16</a>,<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758139">18</a>,<a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758099">20</a><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758094"></a><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758134"></a><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758130"></a><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758138"></a>–<a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758109">25</a></sup>
<em>IL10RB</em>,<sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758151">2</a>,<a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758124">24</a></sup>
<em>IL23R</em>,<sup><a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758097">26</a><a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758086"></a>–<a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758152">28</a></sup>
<em>MADCAM1</em>,<sup><a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758080">29</a></sup>
<em>MTHFR</em>,<sup><a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758125">30</a><a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758118"></a><a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758079"></a>–<a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758101">33</a></sup>
<em>NOD2</em>,<sup><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758141">34</a><a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758113"></a><a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758135"></a>–<a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758137">37</a></sup>
<em>RFC1</em>,<sup><a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758143">38</a></sup>
<em>TGF</em>β<em>1</em>,<sup><a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758132">39</a>,<a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758144">40</a></sup>
<em>TNF</em>,<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758089">10</a>,<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758096">16</a>,<a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758078">20</a>,<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758076">21</a>,<a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758145">23</a>,<a href="#B41" rid="B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758119">41</a><a href="#B42" rid="B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758111"></a>–<a href="#B43" rid="B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758107">43</a></sup>
<em>TNFRII</em>,<sup><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758142">23</a>,<a href="#B42" rid="B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758120">42</a></sup> and <em>VEGF</em>α.<sup><a href="#B44" rid="B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758106">44</a></sup> The median cohort size in these studies was 89 (range, 24-536). The median number of genetic variations examined in each study was 3.5 (range, 1-19); in none of the studies did authors adjust for multiple comparisons in the analyses.</p><p id="__p17" class="p">Of the 40 published candidate SNP associations, usable genotype or imputed data were available for 16 (40%; <a href="/pmc/articles/PMC3369619/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Six candidate SNPs were genotyped: <em>CD31</em> (rs1131012), <em>CTLA4</em> (rs3087243), <em>HPSE</em> (rs4364254), <em>IL23R</em> (rs11209026), <em>IL6</em> (rs1800795), and <em>TNF</em> (rs1799964). Ten candidate SNPs were imputed and passed the criteria for analysis: <em>CD31</em> (rs668), <em>IL1</em>β (rs16944), <em>IL2</em> (rs2069762), <em>IL10</em> (rs1800871, rs1800872), <em>MTHFR</em> (rs1801131), <em>RFC1</em> (rs1057807), <em>TNF</em> (rs1800629, rs1800630), and <em>TNFRII</em> (rs1061622). Of the remaining 24 candidate SNPs, 10 candidate SNPs were imputed but did not pass the criteria for analysis: <em>ESR1</em> (rs2234693), <em>FAS</em> (rs1800682), <em>HPSE</em> (rs4693608), <em>IL10</em> (rs1800896), <em>IL10RB</em> (rs2834167), <em>MTHFR</em> (rs1801133), <em>RFC1</em> (rs4975003, rs6833176), and <em>VEGF</em>α (rs699947, rs833061). Fourteen SNPs were not genotyped and could not be imputed: <em>CD31</em> (rs12953), <em>FAS</em> (rs2234767), <em>HMGB1</em> (rs41376448), <em>HSPA1L</em> (rs2075800), <em>MADCAM1</em> (rs2302217), <em>NOD2</em> (rs2066844, rs2066845, rs2066847), <em>TGF</em>β<em>1</em> (rs1800470), <em>TNF</em> (−488, rs1799724, rs361525), <em>VEGF</em>α (rs2010963, rs3025039).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p18">Summary of genotyped and imputed SNP data generated from the GWAS-HCT dataset using the Affymetrix 5.0 Array and the HapMap release 22 SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139713376631536" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="center">Type</th><th rowspan="1" colspan="1" align="center">rs number</th><th rowspan="1" colspan="1" align="center">Chr</th><th rowspan="1" colspan="1" align="center">Alleles<sup>*</sup></th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">Call rate</th><th rowspan="1" colspan="1" align="center">p_HWE</th><th rowspan="1" colspan="1" align="center">Average posterior probability</th><th rowspan="1" colspan="1" align="center">Status</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left"><em>CD31</em></td><td rowspan="1" colspan="1" align="left">Genotyped</td><td rowspan="1" colspan="1" align="left">rs1131012</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.472</td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="center">0.08</td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>CD31</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs668</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">G/C</td><td rowspan="1" colspan="1" align="center">0.494</td><td rowspan="1" colspan="1" align="center">0.966</td><td rowspan="1" colspan="1" align="center">0.7</td><td rowspan="1" colspan="1" align="center">0.978</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>CTLA4</em></td><td rowspan="1" colspan="1" align="left">Genotyped</td><td rowspan="1" colspan="1" align="left">rs3087243</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.446</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0.814</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>ESR1</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs2234693</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.35</td><td rowspan="1" colspan="1" align="center">0.642</td><td rowspan="1" colspan="1" align="center">0.121</td><td rowspan="1" colspan="1" align="center">0.831</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>FAS</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1800682</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.464</td><td rowspan="1" colspan="1" align="center">0.899</td><td rowspan="1" colspan="1" align="center">0.218</td><td rowspan="1" colspan="1" align="center">0.948</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>HPSE</em></td><td rowspan="1" colspan="1" align="left">Genotyped</td><td rowspan="1" colspan="1" align="left">rs4364254</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.296</td><td rowspan="1" colspan="1" align="center">0.999</td><td rowspan="1" colspan="1" align="center">0.349</td><td rowspan="1" colspan="1" align="center">0.999</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>HPSE</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs4693608</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.328</td><td rowspan="1" colspan="1" align="center">0.62</td><td rowspan="1" colspan="1" align="center">0.668</td><td rowspan="1" colspan="1" align="center">0.817</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL10</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1800871</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.238</td><td rowspan="1" colspan="1" align="center">0.951</td><td rowspan="1" colspan="1" align="center">0.352</td><td rowspan="1" colspan="1" align="center">0.972</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL10</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1800872</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.238</td><td rowspan="1" colspan="1" align="center">0.951</td><td rowspan="1" colspan="1" align="center">0.359</td><td rowspan="1" colspan="1" align="center">0.972</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL10</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1800896</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.472</td><td rowspan="1" colspan="1" align="center">0.799</td><td rowspan="1" colspan="1" align="center">0.245</td><td rowspan="1" colspan="1" align="center">0.898</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL10RB</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs2834167</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.162</td><td rowspan="1" colspan="1" align="center">0.353</td><td rowspan="1" colspan="1" align="center">0.027</td><td rowspan="1" colspan="1" align="center">0.722</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL1B</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs16944</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.359</td><td rowspan="1" colspan="1" align="center">0.944</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">0.962</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL2</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs2069762</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.317</td><td rowspan="1" colspan="1" align="center">0.998</td><td rowspan="1" colspan="1" align="center">0.115</td><td rowspan="1" colspan="1" align="center">0.998</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL23R</em></td><td rowspan="1" colspan="1" align="left">Genotyped</td><td rowspan="1" colspan="1" align="left">rs11209026</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.064</td><td rowspan="1" colspan="1" align="center">0.998</td><td rowspan="1" colspan="1" align="center">0.598</td><td rowspan="1" colspan="1" align="center">0.998</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL6</em></td><td rowspan="1" colspan="1" align="left">Genotyped</td><td rowspan="1" colspan="1" align="left">rs1800795</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.375</td><td rowspan="1" colspan="1" align="center">0.998</td><td rowspan="1" colspan="1" align="center">0.006</td><td rowspan="1" colspan="1" align="center">0.998</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>MTHFR</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1801131</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">G/T</td><td rowspan="1" colspan="1" align="center">0.304</td><td rowspan="1" colspan="1" align="center">0.946</td><td rowspan="1" colspan="1" align="center">0.238</td><td rowspan="1" colspan="1" align="center">0.967</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>MTHFR</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1801133</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.182</td><td rowspan="1" colspan="1" align="center">0.449</td><td rowspan="1" colspan="1" align="center">0.221</td><td rowspan="1" colspan="1" align="center">0.77</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>RFC1</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1057807</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.425</td><td rowspan="1" colspan="1" align="center">0.999</td><td rowspan="1" colspan="1" align="center">0.166</td><td rowspan="1" colspan="1" align="center">0.999</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>RFC1</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs4975003</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.441</td><td rowspan="1" colspan="1" align="center">0.893</td><td rowspan="1" colspan="1" align="center">0.274</td><td rowspan="1" colspan="1" align="center">0.932</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>RFC1</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs6844176</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.494</td><td rowspan="1" colspan="1" align="center">0.748</td><td rowspan="1" colspan="1" align="center">0.047</td><td rowspan="1" colspan="1" align="center">0.878</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>TNF</em></td><td rowspan="1" colspan="1" align="left">Genotyped</td><td rowspan="1" colspan="1" align="left">rs1799964</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.205</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0.063</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>TNF</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1800629</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.135</td><td rowspan="1" colspan="1" align="center">0.926</td><td rowspan="1" colspan="1" align="center">0.726</td><td rowspan="1" colspan="1" align="center">0.965</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>TNF</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1800630</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.152</td><td rowspan="1" colspan="1" align="center">0.996</td><td rowspan="1" colspan="1" align="center">0.153</td><td rowspan="1" colspan="1" align="center">0.997</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>TNFRII</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs1061622</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">G/T</td><td rowspan="1" colspan="1" align="center">0.227</td><td rowspan="1" colspan="1" align="center">0.964</td><td rowspan="1" colspan="1" align="center">0.12</td><td rowspan="1" colspan="1" align="center">0.971</td><td rowspan="1" colspan="1" align="left">Pass</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>VEGFA</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs699947</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.114</td><td rowspan="1" colspan="1" align="center">0.141</td><td rowspan="1" colspan="1" align="center">0.244</td><td rowspan="1" colspan="1" align="center">0.633</td><td rowspan="1" colspan="1" align="left">Fail</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>VEGFA</em></td><td rowspan="1" colspan="1" align="left">Imputed</td><td rowspan="1" colspan="1" align="left">rs833061</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.183</td><td rowspan="1" colspan="1" align="center">0.094</td><td rowspan="1" colspan="1" align="center">0.283</td><td rowspan="1" colspan="1" align="center">0.607</td><td rowspan="1" colspan="1" align="left">Fail</td></tr></tbody></table></div><div id="largeobj_idm139713376631536" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3369619/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139713380790880"><p id="__p19" class="p p-first-last">Chr indicates chromosome; HWE, Hardy-Weinberg equilibrium MAF, minor allele frequency; and SNP, single-nucleotide polymorphism.</p></div><div id="TF2-1"><sup>*</sup>The first allele is designated as the minor allele, and the second allele is designated as the major allele.</div></div></div></div><div id="__sec9" class="sec"><h3 id="__sec9title">Validation of previous published <em>IL10</em> SNP associations with aGVHD</h3><p id="__p21" class="p p-first">We previously published associations between aGVHD risk and SNPs on <em>IL10</em> (rs1800896, rs1800871, and rs1800872)<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758133">1</a></sup> and <em>IL10RB</em> (rs2834167)<sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758146">2</a></sup> in a cohort that overlapped significantly with the current cohort. Because these SNPs were not directly genotyped on the array, we evaluated whether the original association could be replicated with the use of imputed genotypes. Although the genotypes of all 4 SNPs were imputable, only the <em>IL10</em> SNPs rs1800871 and rs1800872 passed the selection criteria for analysis (<a href="/pmc/articles/PMC3369619/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). These 2 successfully imputed <em>IL10</em> SNPs are in significant linkage disequilibrium (D′ &gt; 0.9). Among MRD transplantations, the patient's genotypes for both SNPs were significantly associated with a 30% decrease of the risk for grade III-IV aGVHD in the allelic model (<a href="/pmc/articles/PMC3369619/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>; hazard ratio [HR] 0.72, <em>P</em> = .048). These findings are consistent with the findings of our original publication (<a href="/pmc/articles/PMC3369619/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>).<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758088">1</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p22">Summary of significant associations (<em>P</em> &lt; .05) between aGVHD and candidate SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139713382009520" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="center">Phenotype</th><th rowspan="1" colspan="1" align="center">Genome</th><th rowspan="1" colspan="1" align="center">Donor type</th><th rowspan="1" colspan="1" align="center">SNP</th><th rowspan="1" colspan="1" align="center">Alleles</th><th rowspan="1" colspan="1" align="center">MAF<sup>*</sup></th><th rowspan="1" colspan="1" align="center">Model</th><th rowspan="1" colspan="1" align="center">HR</th><th rowspan="1" colspan="1" align="center">Unadjusted CI</th><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center">HR</th><th rowspan="1" colspan="1" align="center">Adjusted CI<sup>†</sup></th><th rowspan="1" colspan="1" align="center"><em>P</em></th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left"><em>IL2</em></td><td rowspan="1" colspan="1" align="left">III-IV</td><td rowspan="1" colspan="1" align="left">DNR</td><td rowspan="1" colspan="1" align="left">URD</td><td rowspan="1" colspan="1" align="left">rs2069762</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1" align="left">(0.96-1.40)</td><td rowspan="1" colspan="1" align="center">.1293</td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="left">(0.94-1.40)</td><td rowspan="1" colspan="1" align="center">.1811</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">1.04</td><td rowspan="1" colspan="1" align="left">(0.70-1.55)</td><td rowspan="1" colspan="1" align="center">.8466</td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="left">(0.64-1.50)</td><td rowspan="1" colspan="1" align="center">.9310</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">1.32</td><td rowspan="1" colspan="1" align="left">(1.00-1.74)</td><td rowspan="1" colspan="1" align="center">.0487</td><td rowspan="1" colspan="1" align="center">1.33</td><td rowspan="1" colspan="1" align="left">(0.99-1.78)</td><td rowspan="1" colspan="1" align="center">.0557</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL6</em></td><td rowspan="1" colspan="1" align="left">IIb-IV</td><td rowspan="1" colspan="1" align="left">DNR</td><td rowspan="1" colspan="1" align="left">URD</td><td rowspan="1" colspan="1" align="left">rs1800795</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">1.21</td><td rowspan="1" colspan="1" align="left">(1.07-1.36)</td><td rowspan="1" colspan="1" align="center">.0028</td><td rowspan="1" colspan="1" align="center">1.19</td><td rowspan="1" colspan="1" align="left">(1.04-1.36)</td><td rowspan="1" colspan="1" align="center">.0106</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">1.48</td><td rowspan="1" colspan="1" align="left">(1.19-1.84)</td><td rowspan="1" colspan="1" align="center">.0004</td><td rowspan="1" colspan="1" align="center">1.44</td><td rowspan="1" colspan="1" align="left">(1.15-1.81)</td><td rowspan="1" colspan="1" align="center">.0013</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">1.17</td><td rowspan="1" colspan="1" align="left">(0.97-1.40)</td><td rowspan="1" colspan="1" align="center">.0966</td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="left">(0.93-1.38)</td><td rowspan="1" colspan="1" align="center">.2231</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL6</em></td><td rowspan="1" colspan="1" align="left">IIb-IV</td><td rowspan="1" colspan="1" align="left">PT</td><td rowspan="1" colspan="1" align="left">MRD</td><td rowspan="1" colspan="1" align="left">rs1800795</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="left">(0.73-1.04)</td><td rowspan="1" colspan="1" align="center">.1362</td><td rowspan="1" colspan="1" align="center">0.8</td><td rowspan="1" colspan="1" align="left">(0.66-0.97)</td><td rowspan="1" colspan="1" align="center">.0240</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="left">(0.57-1.25)</td><td rowspan="1" colspan="1" align="center">.3983</td><td rowspan="1" colspan="1" align="center">0.81</td><td rowspan="1" colspan="1" align="left">(0.54-1.20)</td><td rowspan="1" colspan="1" align="center">.2923</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="left">(0.66-1.06)</td><td rowspan="1" colspan="1" align="center">.1444</td><td rowspan="1" colspan="1" align="center">0.74</td><td rowspan="1" colspan="1" align="left">(0.57-0.95)</td><td rowspan="1" colspan="1" align="center">.0195</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL6</em></td><td rowspan="1" colspan="1" align="left">III-IV</td><td rowspan="1" colspan="1" align="left">DNR</td><td rowspan="1" colspan="1" align="left">MRD</td><td rowspan="1" colspan="1" align="left">rs1800795</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">1.26</td><td rowspan="1" colspan="1" align="left">(0.98-1.63)</td><td rowspan="1" colspan="1" align="center">.0719</td><td rowspan="1" colspan="1" align="center">1.27</td><td rowspan="1" colspan="1" align="left">(0.97-1.67)</td><td rowspan="1" colspan="1" align="center">.0864</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">1.09</td><td rowspan="1" colspan="1" align="left">(0.65-1.82)</td><td rowspan="1" colspan="1" align="center">.7360</td><td rowspan="1" colspan="1" align="center">1.06</td><td rowspan="1" colspan="1" align="left">(0.63-1.79)</td><td rowspan="1" colspan="1" align="center">.8177</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">1.54</td><td rowspan="1" colspan="1" align="left">(1.05-2.26)</td><td rowspan="1" colspan="1" align="center">.0257</td><td rowspan="1" colspan="1" align="center">1.59</td><td rowspan="1" colspan="1" align="left">(1.05-2.41)</td><td rowspan="1" colspan="1" align="center">.0279</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL10</em></td><td rowspan="1" colspan="1" align="left">III-IV</td><td rowspan="1" colspan="1" align="left">PT</td><td rowspan="1" colspan="1" align="left">MRD</td><td rowspan="1" colspan="1" align="left">rs1800872</td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">0.72</td><td rowspan="1" colspan="1" align="left">(0.52-1.00)</td><td rowspan="1" colspan="1" align="center">.0481</td><td rowspan="1" colspan="1" align="center">0.73</td><td rowspan="1" colspan="1" align="left">(0.52-1.04)</td><td rowspan="1" colspan="1" align="center">.0796</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="left">(0.00,Inf)</td><td rowspan="1" colspan="1" align="center">.9817</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="left">(0.00,Inf)</td><td rowspan="1" colspan="1" align="center">.9815</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="left">(0.56-1.19)</td><td rowspan="1" colspan="1" align="center">.2933</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="left">(0.58-1.25)</td><td rowspan="1" colspan="1" align="center">.4035</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL10</em></td><td rowspan="1" colspan="1" align="left">III-IV</td><td rowspan="1" colspan="1" align="left">PT</td><td rowspan="1" colspan="1" align="left">MRD</td><td rowspan="1" colspan="1" align="left">rs1800871</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">0.72</td><td rowspan="1" colspan="1" align="left">(0.52-1.00)</td><td rowspan="1" colspan="1" align="center">.0481</td><td rowspan="1" colspan="1" align="center">0.73</td><td rowspan="1" colspan="1" align="left">(0.52-1.04)</td><td rowspan="1" colspan="1" align="center">.0796</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="left">(0.00,Inf)</td><td rowspan="1" colspan="1" align="center">.9817</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="left">(0.00,Inf)</td><td rowspan="1" colspan="1" align="center">.9815</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="left">(0.56-1.19)</td><td rowspan="1" colspan="1" align="center">.2933</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="left">(0.58-1.25)</td><td rowspan="1" colspan="1" align="center">.4035</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>CTLA4</em></td><td rowspan="1" colspan="1" align="left">III-IV</td><td rowspan="1" colspan="1" align="left">PT</td><td rowspan="1" colspan="1" align="left">URD</td><td rowspan="1" colspan="1" align="left">rs3087243</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">1.17</td><td rowspan="1" colspan="1" align="left">(0.97-1.42)</td><td rowspan="1" colspan="1" align="center">.1042</td><td rowspan="1" colspan="1" align="center">1.23</td><td rowspan="1" colspan="1" align="left">(1.01-1.50)</td><td rowspan="1" colspan="1" align="center">.0374</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">(0.72-1.41)</td><td rowspan="1" colspan="1" align="center">.9782</td><td rowspan="1" colspan="1" align="center">1.05</td><td rowspan="1" colspan="1" align="left">(0.74-1.49)</td><td rowspan="1" colspan="1" align="center">.7870</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">1.5</td><td rowspan="1" colspan="1" align="left">(1.09-2.07)</td><td rowspan="1" colspan="1" align="center">.0138</td><td rowspan="1" colspan="1" align="center">1.65</td><td rowspan="1" colspan="1" align="left">(1.18-2.30)</td><td rowspan="1" colspan="1" align="center">.0035</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>HPSE</em></td><td rowspan="1" colspan="1" align="left">IIb-IV</td><td rowspan="1" colspan="1" align="left">DNR</td><td rowspan="1" colspan="1" align="left">URD</td><td rowspan="1" colspan="1" align="left">rs4364254</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.29</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="left">(0.93-1.22)</td><td rowspan="1" colspan="1" align="center">.3377</td><td rowspan="1" colspan="1" align="center">1.11</td><td rowspan="1" colspan="1" align="left">(0.97-1.27)</td><td rowspan="1" colspan="1" align="center">.1453</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="left">(0.66-1.26)</td><td rowspan="1" colspan="1" align="center">.5736</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="left">(0.68-1.33)</td><td rowspan="1" colspan="1" align="center">.7767</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="left">(0.97-1.37)</td><td rowspan="1" colspan="1" align="center">.1158</td><td rowspan="1" colspan="1" align="center">1.21</td><td rowspan="1" colspan="1" align="left">(1.01-1.45)</td><td rowspan="1" colspan="1" align="center">.0415</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>MTHFR</em></td><td rowspan="1" colspan="1" align="left">III-IV</td><td rowspan="1" colspan="1" align="left">PT</td><td rowspan="1" colspan="1" align="left">MRD</td><td rowspan="1" colspan="1" align="left">rs1801131</td><td rowspan="1" colspan="1" align="center">G/T</td><td rowspan="1" colspan="1" align="center">0.29</td><td rowspan="1" colspan="1" align="left">Allelic</td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="left">(0.67-1.17)</td><td rowspan="1" colspan="1" align="center">.3870</td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="left">(0.63-1.11)</td><td rowspan="1" colspan="1" align="center">.2157</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Recessive</td><td rowspan="1" colspan="1" align="center">1.29</td><td rowspan="1" colspan="1" align="left">(0.76-2.19)</td><td rowspan="1" colspan="1" align="center">.3408</td><td rowspan="1" colspan="1" align="center">1.22</td><td rowspan="1" colspan="1" align="left">(0.71-2.08)</td><td rowspan="1" colspan="1" align="center">.4679</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">Dominant</td><td rowspan="1" colspan="1" align="center">0.71</td><td rowspan="1" colspan="1" align="left">(0.49-1.04)</td><td rowspan="1" colspan="1" align="center">.0764</td><td rowspan="1" colspan="1" align="center">0.66</td><td rowspan="1" colspan="1" align="left">(0.45-0.96)</td><td rowspan="1" colspan="1" align="center">.0309</td></tr></tbody></table></div><div id="largeobj_idm139713382009520" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3369619/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139713362194752"><p id="__p23" class="p p-first-last">Allo indicates allogeneic; CI, confidence interval; DNR, donor, HR, hazard ratio MAF, minor allele frequency; MRD, matched related donor; NR, not reported; PT, patient; SNP, single-nucleotide polymorphism; and URD, unrelated donor.</p></div><div id="TF3-1"><sup>*</sup>The first allele is designated as the minor allele, and the second allele is designated as the major allele.</div><div id="TF3-2"><sup>†</sup>Models adjusted for total body irradiation (yes vs no), sex mismatch, and 4 principal components for population stratification. Matched related donor analyses are also adjusted for HLA match.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p26">Summary of previous publications reporting associations with candidate genes that meet the <em>P</em> &lt; .05 threshold in the current validation study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139713453380656" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="center">Reference</th><th rowspan="1" colspan="1" align="center">Phenotype</th><th rowspan="1" colspan="1" align="center">Genome</th><th rowspan="1" colspan="1" align="center">Donor type</th><th rowspan="1" colspan="1" align="center">SNP</th><th rowspan="1" colspan="1" align="center">Alleles<sup>*</sup></th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">HR</th><th rowspan="1" colspan="1" align="center">Unadjusted CI</th><th rowspan="1" colspan="1" align="center"><em>P</em></th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left"><em>IL2</em></td><td rowspan="1" colspan="1" align="left">MacMillan et al<sup><a href="#B17" rid="B17" class=" bibr popnode">17</a></sup></td><td rowspan="1" colspan="1" align="center">II-IV</td><td rowspan="1" colspan="1" align="center">PT</td><td rowspan="1" colspan="1" align="center">URD</td><td rowspan="1" colspan="1" align="center">rs2069762</td><td rowspan="1" colspan="1" align="center">G/T</td><td rowspan="1" colspan="1" align="center">0.3</td><td rowspan="1" colspan="1" align="center">2.1</td><td rowspan="1" colspan="1" align="center">(1.0-4.5)</td><td rowspan="1" colspan="1" align="center">.05</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL6</em></td><td rowspan="1" colspan="1" align="left">Mullighan et al<sup><a href="#B10" rid="B10" class=" bibr popnode">10</a></sup></td><td rowspan="1" colspan="1" align="center">I-IV</td><td rowspan="1" colspan="1" align="center">DNR</td><td rowspan="1" colspan="1" align="center">MRD</td><td rowspan="1" colspan="1" align="center">rs1800795</td><td rowspan="1" colspan="1" align="center">G/C</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">4.4</td><td rowspan="1" colspan="1" align="center">(1.6-11.7)</td><td rowspan="1" colspan="1" align="center">.001</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL6</em></td><td rowspan="1" colspan="1" align="left">Karabon e tal<sup><a href="#B18" rid="B18" class=" bibr popnode">18</a></sup></td><td rowspan="1" colspan="1" align="center">II-IV</td><td rowspan="1" colspan="1" align="center">DNR</td><td rowspan="1" colspan="1" align="center">NR</td><td rowspan="1" colspan="1" align="center">rs1800795</td><td rowspan="1" colspan="1" align="center">G/C</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">3.4</td><td rowspan="1" colspan="1" align="center">(1.1-13.0)</td><td rowspan="1" colspan="1" align="center">.03</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL6</em></td><td rowspan="1" colspan="1" align="left">Ambruzova et al<sup><a href="#B19" rid="B19" class=" bibr popnode">19</a></sup></td><td rowspan="1" colspan="1" align="center">II-IV</td><td rowspan="1" colspan="1" align="center">PT</td><td rowspan="1" colspan="1" align="center">Allo</td><td rowspan="1" colspan="1" align="center">rs1800795</td><td rowspan="1" colspan="1" align="center">G/C</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">2.2</td><td rowspan="1" colspan="1" align="center">(1.1-4.4)</td><td rowspan="1" colspan="1" align="center">.03</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL10</em></td><td rowspan="1" colspan="1" align="left">Lin et al<sup><a href="#B1" rid="B1" class=" bibr popnode">1</a></sup></td><td rowspan="1" colspan="1" align="center">III-IV</td><td rowspan="1" colspan="1" align="center">PT</td><td rowspan="1" colspan="1" align="center">MRD</td><td rowspan="1" colspan="1" align="center">rs1800871<sup>†</sup></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.3</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left"><em>IL10</em></td><td rowspan="1" colspan="1" align="left">Lin et al<sup><a href="#B1" rid="B1" class=" bibr popnode">1</a></sup></td><td rowspan="1" colspan="1" align="center">III-IV</td><td rowspan="1" colspan="1" align="center">PT</td><td rowspan="1" colspan="1" align="center">MRD</td><td rowspan="1" colspan="1" align="center">rs1800872</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.3</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">(0.2-0.9)</td><td rowspan="1" colspan="1" align="center">.02</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>CTLA4</em></td><td rowspan="1" colspan="1" align="left">Perez-Garcia et al<sup><a href="#B8" rid="B8" class=" bibr popnode">8</a></sup></td><td rowspan="1" colspan="1" align="center">II-IV</td><td rowspan="1" colspan="1" align="center">DNR</td><td rowspan="1" colspan="1" align="center">MRD</td><td rowspan="1" colspan="1" align="center">rs3087243</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">1.5</td><td rowspan="1" colspan="1" align="center">(1.0-2.3)</td><td rowspan="1" colspan="1" align="center">.03</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>HPSE</em></td><td rowspan="1" colspan="1" align="left">Ostrovsky et al<sup><a href="#B12" rid="B12" class=" bibr popnode">12</a></sup></td><td rowspan="1" colspan="1" align="center">II-IV and III-IV</td><td rowspan="1" colspan="1" align="center">PT</td><td rowspan="1" colspan="1" align="center">Allo</td><td rowspan="1" colspan="1" align="center">rs4364254</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">NR</td><td rowspan="1" colspan="1" align="center">NR</td><td rowspan="1" colspan="1" align="center">&lt; .01</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>MTHFR</em></td><td rowspan="1" colspan="1" align="left">Robien et al<sup><a href="#B30" rid="B30" class=" bibr popnode">30</a></sup></td><td rowspan="1" colspan="1" align="center">I-IV</td><td rowspan="1" colspan="1" align="center">PT</td><td rowspan="1" colspan="1" align="center">Allo</td><td rowspan="1" colspan="1" align="center">rs1801131</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">3.6</td><td rowspan="1" colspan="1" align="center">(1.0-12.7)</td><td rowspan="1" colspan="1" align="center">&lt; .01</td></tr></tbody></table></div><div id="largeobj_idm139713453380656" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3369619/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139713382931152"><p id="__p27" class="p p-first-last">Allo indicates allogeneic; CI, confidence interval; DNR, donor; HR, hazard ratio; MAF, minor allele frequency; MRD, matched related donor; NR, not reported; PT, patient; SNP, single-nucleotide polymorphism; and URD, unrelated donor.</p></div><div id="TF4-1"><sup>*</sup>The first allele is designated as the minor allele, and the second allele is designated as the major allele.</div><div id="TF4-2"><sup>†</sup>Assessed as component of a haplotype.</div></div></div></div><div id="__sec10" class="sec sec-last"><h3 id="__sec10title">Associations between <em>IL6</em>, <em>IL2</em>, <em>CTLA4</em>, <em>HPSE</em>, and <em>MTHFR</em> SNPs and aGVHD</h3><p id="__p30" class="p p-first">Results for the remaining 14 evaluable candidate SNPs are summarized in <a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>A-D. These analyses revealed that rs1800795 in <em>IL6</em>, rs2069762 in <em>IL2</em>, rs3087243 in <em>CTLA4</em>, rs4364254 in <em>HPSE</em>, and rs1801131 in <em>MTHFR</em> had unadjusted or adjusted models that met the <em>P</em> &lt; .05 threshold (<a href="/pmc/articles/PMC3369619/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The previously published associations for these SNPs are summarized in <a href="/pmc/articles/PMC3369619/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a> for comparison. We report in the next 3 paragraphs the details of these associations in our cohort.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139713366008448" class="figure"><a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3369619_zh89991287730002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zh89991287730002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3369619/bin/zh89991287730002.jpg"></a></div><div id="largeobj_idm139713366008448" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3369619/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3369619/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p31"><strong>Forest plots for unadjusted and adjusted analyses of aGVHD and candidate SNPs.</strong> Forest plots for unadjusted (squares) and adjusted (circles) analyses of aGVHD and candidate SNPs among MRD (A) and URD (B) HCT. Hazard ratios are represented on the x-axis in logarithmic scale. Positive values indicate an increased risk for aGVHD, and negative values indicate a decreased risk for aGVHD. Gene/rs numbers in bold and closed squares and circles indicate the association exceeded the <em>P</em> = .05 threshold.</p></div></div></div><p id="__p32">Of all the SNPs that met the <em>P</em> &lt; .05 threshold, the associations with rs1800795 in <em>IL6</em> were most consistently observed throughout the genetic models. In the unadjusted and adjusted URD analyses, the donor genotype for rs1800795 was associated with a 20% to 50% increase in the risk for grade IIb-IV aGVHD in the allelic (unadjusted <em>P</em> = .003; adjusted <em>P</em> = .011) and recessive (unadjusted <em>P</em> &lt; .001; adjusted <em>P</em> = .001) models (<a href="/pmc/articles/PMC3369619/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). For MRD HCT, the donor genotype for rs1800795 was associated with a 60% increase in risk for grade III-IV aGVHD in both dominant unadjusted (<em>P</em> = .026) and adjusted (<em>P</em> = .028) models. We also found the patient C allele for rs1800795 in <em>IL6</em> was associated with a 20% to 26% decrease in risk of grade IIb-IV aGVHD but only in the adjusted MRD allelic (<em>P</em> = .024) and dominant (<em>P</em> = .020) models (<a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>A).</p><p id="__p33">We found that the <em>IL2</em> polymorphism rs2069762 in the donor genotype was associated in the dominant genetic mode with a 1.3-fold increase in risk of grade III-IV aGVHD after URD HCT (<em>P</em> = .049; <a href="/pmc/articles/PMC3369619/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>B). The <em>P</em> value shifted slightly to .056 in multivariate analysis, but the point estimate remained unchanged at 1.3. In both unadjusted and adjusted analyses, the patient genotypes in our cohort indicated a consistent shift toward slightly greater risks of grade IIb-IV and III-IV aGVHD after URD HCT in the allelic and recessive models, but the <em>P</em> values did not exceed the .05 threshold. For <em>CTLA4</em>, the donor rs3087243 genotype was associated with an increased risk for grade III-IV aGVHD after URD HCT in both the unadjusted (HR 1.5, <em>P</em> = .014) and adjusted (HR 1.65, <em>P</em> = .004) analyses (<a href="/pmc/articles/PMC3369619/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>B). However, unlike the published association, the point estimates in our analysis of the donor rs3087243 genotype in MRD HCT suggested a reduced risk for grade IIb-IV aGVHD in all models, with no appreciable association with risk for grade III-IV aGVHD (<a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>A).</p><p id="__p34" class="p p-last">In the recently published <em>HSPE</em> analysis, the patient-donor disparity for SNP rs4693608 and rs4364254 genotypes was examined.<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758085">12</a></sup> Because the imputed rs4693608 genotype did not meet our quality control threshold, we were not able to perform a disparity analysis, and therefore we restricted our analysis to rs4364254, which was directly genotyped. The donor genotype for rs4364254 was significantly associated with a 1.2-fold increase (<em>P</em> = .042) in risk for grade IIb-IV aGVHD after URD HCT in the dominant multivariate model (<a href="/pmc/articles/PMC3369619/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>B). For the <em>MTHFR</em> gene, we identified an association between rs1801131 and grade III-IV aGVHD. But unlike the original publication, the patient genotype was associated with a 35% decrease in risk (<em>P</em> = .031) in the dominant model after MRD HCT (<a href="/pmc/articles/PMC3369619/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="/pmc/articles/PMC3369619/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>A).</p></div></div><div id="__sec11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title">Discussion</h2><p id="__p35" class="p p-first">We conducted an independent replication analysis of 14 candidate SNP genotypes previously documented in the published literature to be associated with the risk of aGVHD. To our knowledge, this is the first attempt to replicate such a large number of published genetic associations with the use of genotyped and imputed genome wide array data generated from a relatively large cohort of HCT patients and donors. Our analytical approach and the results of these analyses raise several key questions regarding candidate gene genetic association studies, some of which are of particular importance to the field of HCT.</p><p id="__p36">The idea of an agnostic genome-wide approach toward discovery of risk variants is particularly challenging for HCT because of the cost of comprehensively screening both the patient and donor genomes and the need to accumulate a sample size of at least 5000 transplantations (10 000 total samples from patients and donors) to attain &gt; 80% power for detecting significant associations with phenotype of &gt; 30% frequency and odds ratios ranging from 1.5 to 1.8 for SNPs with minor allele frequencies &gt; 20%.<sup><a href="#B45" rid="B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758150">45</a></sup> As we await the assembly of such a cohort, a bridging approach might be to use genome wide data from an intermediate sized cohort to conduct candidate gene studies, using both genotyped and imputed data.</p><p id="__p37">The imputation approach, which has been widely adopted in genetic association studies,<sup><a href="#B46" rid="B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758090">46</a></sup> facilitates the use of actual genotypes for a limited set of SNPs to access information for a much larger number of SNPs across the entire genome. Our validation analysis of the imputed genotypes for rs1800871 and rs1800872 in <em>IL10</em> demonstrate that imputed genotype data can be robust and correlates well with actual genotype data. This approach opens the possibility of evaluating candidate genes in pathways of high biologic relevance. Although this approach is limited in scope, it is financially and statistically more realistic for the limited sample size of most HCT cohorts. However, this approach is restricted by the quantity of usable imputed data, which depends on the extent and quality of the genome-wide SNP data generated by the commercial array. In our study, the SNP data generated by the Affymetrix 5.0 array only had a success rate of 40% for useable high-quality actual and imputed genotypes for the specified candidate SNPs, thus highlighting the limitation of this approach, at least with this array.</p><p id="__p38">Our <em>IL6</em> results suggest that the association with the <em>IL6</em> SNP is likely worthy of more in-depth analysis. We found significant associations with the <em>IL6</em> SNP genotype in both the patient and donor genomes, as well as with both aGVHD phenotypes. Associations with the donor and patient <em>IL6</em> genotypes have been previously reported, all with risks that are similar in magnitude in comparison with our findings.<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758122">10</a>,<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758115">18</a>,<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758093">19</a></sup> In particular, the association of patient genotype has been validated under our multivariate analysis, in terms of all factors such as genome, donor type, and association model. However, the association of donor genotype is only arguably validated because the original findings were discrepant with regards to the assignment of the minor allele. Although both studies identified the association to be with the donor's G allele, Mullighan et al<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758104">10</a></sup> assigned the G allele as the major allele and Karabon et al<sup><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758082">18</a></sup> assigned the G allele as the minor allele. The complementary nature of the C and G alleles, in conjunction with their similar allele frequencies, may have resulted in these differences in minor allele assignment in the genotyping process. However, because the minor allele's frequency is close to 50% in both study populations and this appears to vary among different ethnic groups in the HapMap database, it is possible that because of regional ethnic/racial differences in previous and current study populations, the minor allele frequencies may vary to the point where the minor allele assignment can differ between studies.</p><p id="__p39">To judge whether these <em>IL6</em> findings are significant, the totality of the <em>IL6</em> association data should be considered. An association with polymorphisms in the promoter region of <em>IL6</em> is biologically plausible. SNP rs1800795 is located at position −174 in the 5′ flanking region of the <em>IL6</em> gene.<sup><a href="#B47" rid="B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758095">47</a></sup> This SNP has been documented to result in up-regulation of <em>IL6</em> gene transcription and higher circulating IL6 levels.<sup><a href="#B47" rid="B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758121">47</a></sup> This SNP has also been reported to influence the risk for many other phenotypes, including juvenile arthritis,<sup><a href="#B47" rid="B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758148">47</a></sup> type 2 diabetes,<sup><a href="#B48" rid="B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758117">48</a></sup> bronchiolitis obliterans syndrome after lung transplantation,<sup><a href="#B49" rid="B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758081">49</a></sup> and kidney allograft survival.<sup><a href="#B50" rid="B50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758112">50</a></sup> Multiple studies have documented increased plasma or serum concentrations of IL6 in patients with aGVHD.<sup><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758110">18</a></sup> IL6 is a pleiotropic interleukin that in the context of an allogeneic HCT functions as both an acute phase mediator of inflammation and also as a critical factor in the differentiation of regulatory and Th17 effector cells. These effects may be mediated through variable <em>IL6</em> expression encoded by both the host and donor genomes. It is unclear why donor cells genetically predisposed to lower circulating IL6 levels will increase the risk for aGVHD. It is feasible that a patient predisposed to chronically elevated IL6 levels (the phenotype associated with the major allele) may experience down regulation of the <em>IL6</em> receptor and thereby reduce their responsiveness to IL6 and risk for aGCHD. This hypothesis requires further investigation.</p><p id="__p40">The results for <em>IL2</em>, <em>CTLA4</em>, <em>HPSE</em>, and <em>MTHFR</em> indicate that the definition of “replication” is worthy of additional discussion in the context of genetic association studies in HCT. For each of these genes, we were able to identify an association with the candidate SNP that met the <em>P</em> &lt; .05 threshold. However, certain aspects for each association were inconsistent with the original published findings. In the strictest sense, these associations did not represent replications of the original results. For <em>IL2</em>, our association was with the donor genome, which was inconsistent with the patient genome association in the original publication. For <em>CTLA4</em>, our association was observed with URD HCT and not with MRD HCT, which was the transplantation type for which the original association was documented.</p><p id="__p41">For <em>HSPE</em>, we identified a SNP association with aGVHD, but this association was not reported as statistically significant in the original study. In fact, Ostrovsky et al reported that rs4364254 trended toward a lower cumulative risk for grades II-IV and III-IV aGVHD in the recessive model.<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758136">12</a></sup> Unfortunately, we were not able to impute the genotypes for rs4693608, the key <em>HSPE</em> SNP that was associated with aGVHD risk. For <em>MTHFR</em>, we observed an association with the same genome and donor type, but the observed effect was opposite to that of the original publication. At best, the associations we observed can be considered as new discoveries; at worst, they represent false-positive associations. Thus, our results cannot be used alone to judge whether a previously published association is valid. Instead, our results should be combined with previously published association studies to judge the likelihood that an association remains worthy of additional investigation.</p><p id="__p42">If our <em>IL6</em> findings are regarded as the only true replication of previously reported findings, then the replication rate in our analysis is only 7%. This low rate of replication is consistent with the observed replication rate of 3.6% in a comprehensive review of 166 candidate SNPs with putative associations that had been evaluated 3 or more times in the published literature.<sup><a href="#B51" rid="B51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_262758100">51</a></sup> Guidelines for replicating genotype-phenotype associations are now well defined. These guidelines indicate that the likelihood of replicating a genetic association depends on many variables that can be related to the study population, genetic approach, phenotype definition and statistical power. We believe our results are valid because we took every measure to remain faithful to these guidelines.</p><p id="__p43">Although we are unable to ensure that the ethnic background of our cohort is comparable with those of the published studies, we incorporated the principal component approach to account for the potential confounding effects of population stratification. Because there is little that we can do to control for center-to-center differences in clinical practice, we attempted to minimize imbalance in clinical risk factors by stratifying our analyses according to donor type, which provides some degree of standardization regarding treatment regimens. We also attempted to minimize center-specific differences in phenotype ascertainment by analyzing each SNP with 2 aGVHD phenotypes, increasing the likelihood that our aGVHD phenotypes approximate those described at other centers. Finally, our cohort was larger than all of the previously studied cohorts, and hence was at least equally powered to detect the original associations.</p><p id="__p44" class="p p-last">In summary, our study demonstrates the advantages and disadvantages of a novel approach via the use of imputed SNP data in genetic analyses and sets high standards for the conduct and reporting conventions of genetic association studies in the HCT population. If HCT investigators apply these standards uniformly, their findings may be more easily interpreted and replicated by other investigators in this field, thereby significantly improving the credibility of future genetic association reports. This has the potential to increase the likelihood that non-HLA genetic studies can contribute meaningfully to advancing the current understanding of the biology of aGVHD and improve our management of this disease.</p></div><div id="idm139713364493472" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139713364493472title">Acknowledgments</h2><div class="sec"><p id="__p47">This work was supported by National Institutes of Health grants AI33484, <a id="__tag_802298490" class="tag_hotlink" href="/nuccore/AI149213" ref="/nuccore/AI149213"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AI149213","term_id":"3677682","term_text":"AI149213"}}</span>AI149213</a>, <a id="__tag_802298489" class="tag_hotlink" href="/nuccore/CA015704" ref="/nuccore/CA015704"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA015704","term_id":"24293048","term_text":"CA015704"}}</span>CA015704</a>, CA18029, <a id="__tag_802298488" class="tag_hotlink" href="/nuccore/HL087690" ref="/nuccore/HL087690"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL087690","term_id":"1051658099","term_text":"HL087690"}}</span>HL087690</a>, <a id="__tag_802298492" class="tag_hotlink" href="/nuccore/HL088201" ref="/nuccore/HL088201"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL088201","term_id":"1051658610","term_text":"HL088201"}}</span>HL088201</a>, <a id="__tag_802298491" class="tag_hotlink" href="/nuccore/HL094260" ref="/nuccore/HL094260"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL094260","term_id":"1051664669","term_text":"HL094260"}}</span>HL094260</a>, <a id="__tag_802298487" class="tag_hotlink" href="/nuccore/HL105914" ref="/nuccore/HL105914"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL105914","term_id":"1051678241","term_text":"HL105914"}}</span>HL105914</a>, and K23HL69860.</p></div></div><div id="idm139713371273296" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139713371273296title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm139713371273168"></p><p id="__p45" class="p p-first-last">There is an Inside <em>Blood</em> commentary on this article in this issue.</p><p></p><p class="fn sec" id="idm139713364494704"></p><p id="__p46" class="p p-first-last">The publication costs of this article were defrayed in part by page charge payment. Therefore, and solely to indicate this fact, this article is hereby marked “advertisement” in accordance with 18 USC section 1734.</p><p></p></div></div><div id="__sec12" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Authorship</h2><!--/article/back/sec/--><p id="__p48" class="p p-first">Contribution: J.W.C., L.P.Z., P.J.M., B.E.S., M.B., E.H.W., and J.A.H. were responsible for the study concept and design; X.C.Z., W.H.F., H.W., L.P.Z., and B.E.S. were responsible for data acquisition, statistical analysis, imputation, and informatics analyses; and J.W.C., X.C.Z., L.P.Z., P.J.M., B.E.S., M.B., E.H.W., and J.A.H. were responsible for data interpretation and writing the manuscript.</p><p id="__p49">Conflict-of-interest disclosure: The authors declare no competing financial interests.</p><p id="__p50" class="p p-last">Correspondence: Jason W. Chien, MD, MSc, Fred Hutchinson Cancer Research Center, 1100 Fairview Ave N, Suite D5-280, Seattle, WA 98109-1024; e-mail: <a href="mailto:dev@null" data-email="gro.crchf@neihcj" class="oemail">gro.crchf@neihcj</a>.</p></div><div id="idm139713416371616" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139713416371616title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Lin MT, Storer B, Martin PJ, et al.  Relation of an interleukin-10 promoter polymorphism to graft-versus-host disease and survival after hematopoietic-cell transplantation. <span><span class="ref-journal">N Engl J Med. </span>2003;<span class="ref-vol">349</span>(23):2201–2210.</span> [<a href="/pubmed/14657427" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Relation+of+an+interleukin-10+promoter+polymorphism+to+graft-versus-host+disease+and+survival+after+hematopoietic-cell+transplantation.&amp;author=MT+Lin&amp;author=B+Storer&amp;author=PJ+Martin&amp;volume=349&amp;issue=23&amp;publication_year=2003&amp;pages=2201-2210&amp;pmid=14657427&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Lin MT, Storer B, Martin PJ, et al.  Genetic variation in the IL-10 pathway modulates severity of acute graft-versus-host disease following hematopoietic cell transplantation: synergism between IL-10 genotype of patient and IL-10 receptor beta genotype of donor. <span><span class="ref-journal">Blood. </span>2005;<span class="ref-vol">106</span>(12):3995–4001.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1895107/">PMC free article</a>]</span> [<a href="/pubmed/16109775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genetic+variation+in+the+IL-10+pathway+modulates+severity+of+acute+graft-versus-host+disease+following+hematopoietic+cell+transplantation:+synergism+between+IL-10+genotype+of+patient+and+IL-10+receptor+beta+genotype+of+donor.&amp;author=MT+Lin&amp;author=B+Storer&amp;author=PJ+Martin&amp;volume=106&amp;issue=12&amp;publication_year=2005&amp;pages=3995-4001&amp;pmid=16109775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Martin PJ, McDonald GB, Sanders JE, et al.  Increasingly frequent diagnosis of acute gastrointestinal graft-versus-host disease after allogeneic hematopoietic cell transplantation. <span><span class="ref-journal">Biol Blood Marrow Transplant. </span>2004;<span class="ref-vol">10</span>(5):320–327.</span> [<a href="/pubmed/15111931" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Blood+Marrow+Transplant&amp;title=Increasingly+frequent+diagnosis+of+acute+gastrointestinal+graft-versus-host+disease+after+allogeneic+hematopoietic+cell+transplantation.&amp;author=PJ+Martin&amp;author=GB+McDonald&amp;author=JE+Sanders&amp;volume=10&amp;issue=5&amp;publication_year=2004&amp;pages=320-327&amp;pmid=15111931&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Flowers ME, Inamoto Y, Carpenter PA, et al.  Comparative analysis of risk factors for acute graft-versus-host disease and for chronic graft-versus-host disease according to National Institutes of Health consensus criteria. <span><span class="ref-journal">Blood. </span>2011;<span class="ref-vol">117</span>(11):3214–3219.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3062319/">PMC free article</a>]</span> [<a href="/pubmed/21263156" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Comparative+analysis+of+risk+factors+for+acute+graft-versus-host+disease+and+for+chronic+graft-versus-host+disease+according+to+National+Institutes+of+Health+consensus+criteria.&amp;author=ME+Flowers&amp;author=Y+Inamoto&amp;author=PA+Carpenter&amp;volume=117&amp;issue=11&amp;publication_year=2011&amp;pages=3214-3219&amp;pmid=21263156&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>(8):904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies.&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;issue=8&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Cavanagh G, Chapman CE, Carter V, Dickinson AM, Middleton PG. Donor CD31 genotype impacts on transplant complications after human leukocyte antigen-matched sibling allogeneic bone marrow transplantation. <span><span class="ref-journal">Transplantation. </span>2005;<span class="ref-vol">79</span>(5):602–605.</span> [<a href="/pubmed/15753851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=Donor+CD31+genotype+impacts+on+transplant+complications+after+human+leukocyte+antigen-matched+sibling+allogeneic+bone+marrow+transplantation.&amp;author=G+Cavanagh&amp;author=CE+Chapman&amp;author=V+Carter&amp;author=AM+Dickinson&amp;author=PG+Middleton&amp;volume=79&amp;issue=5&amp;publication_year=2005&amp;pages=602-605&amp;pmid=15753851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Goodman RS, Ewing J, Evans PC, et al.  Donor CD31 genotype and its association with acute graft-versus-host disease in HLA identical sibling stem cell transplantation. <span><span class="ref-journal">Bone Marrow Transplant. </span>2005;<span class="ref-vol">36</span>(2):151–156.</span> [<a href="/pubmed/15908974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Donor+CD31+genotype+and+its+association+with+acute+graft-versus-host+disease+in+HLA+identical+sibling+stem+cell+transplantation.&amp;author=RS+Goodman&amp;author=J+Ewing&amp;author=PC+Evans&amp;volume=36&amp;issue=2&amp;publication_year=2005&amp;pages=151-156&amp;pmid=15908974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Perez-Garcia A, De la Camara R, Roman-Gomez J, et al.  CTLA-4 polymorphisms and clinical outcome after allogeneic stem cell transplantation from HLA-identical sibling donors. <span><span class="ref-journal">Blood. </span>2007;<span class="ref-vol">110</span>(1):461–467.</span> [<a href="/pubmed/17384200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=CTLA-4+polymorphisms+and+clinical+outcome+after+allogeneic+stem+cell+transplantation+from+HLA-identical+sibling+donors.&amp;author=A+Perez-Garcia&amp;author=R+De+la+Camara&amp;author=J+Roman-Gomez&amp;volume=110&amp;issue=1&amp;publication_year=2007&amp;pages=461-467&amp;pmid=17384200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Middleton PG, Norden J, Cullup H, et al.  Oestrogen receptor alpha gene polymorphism associates with occurrence of graft-versus-host disease and reduced survival in HLA-matched sib-allo BMT. <span><span class="ref-journal">Bone Marrow Transplant. </span>2003;<span class="ref-vol">32</span>(1):41–47.</span> [<a href="/pubmed/12815477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Oestrogen+receptor+alpha+gene+polymorphism+associates+with+occurrence+of+graft-versus-host+disease+and+reduced+survival+in+HLA-matched+sib-allo+BMT.&amp;author=PG+Middleton&amp;author=J+Norden&amp;author=H+Cullup&amp;volume=32&amp;issue=1&amp;publication_year=2003&amp;pages=41-47&amp;pmid=12815477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Mullighan C, Heatley S, Doherty K, et al.  Non-HLA immunogenetic polymorphisms and the risk of complications after allogeneic hemopoietic stem-cell transplantation. <span><span class="ref-journal">Transplantation. </span>2004;<span class="ref-vol">77</span>(4):587–596.</span> [<a href="/pubmed/15084940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=Non-HLA+immunogenetic+polymorphisms+and+the+risk+of+complications+after+allogeneic+hemopoietic+stem-cell+transplantation.&amp;author=C+Mullighan&amp;author=S+Heatley&amp;author=K+Doherty&amp;volume=77&amp;issue=4&amp;publication_year=2004&amp;pages=587-596&amp;pmid=15084940&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Kornblit B, Masmas T, Petersen SL, et al.  Association of HMGB1 polymorphisms with outcome after allogeneic hematopoietic cell transplantation. <span><span class="ref-journal">Biol Blood Marrow Transplant. </span>2010;<span class="ref-vol">16</span>(2):239–252.</span> [<a href="/pubmed/19819342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Blood+Marrow+Transplant&amp;title=Association+of+HMGB1+polymorphisms+with+outcome+after+allogeneic+hematopoietic+cell+transplantation.&amp;author=B+Kornblit&amp;author=T+Masmas&amp;author=SL+Petersen&amp;volume=16&amp;issue=2&amp;publication_year=2010&amp;pages=239-252&amp;pmid=19819342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Ostrovsky O, Shimoni A, Rand A, Vlodavsky I, Nagler A. Genetic variations in the heparanase gene (HPSE) associate with increased risk of GVHD following allogeneic stem cell transplantation: effect of discrepancy between recipients and donors. <span><span class="ref-journal">Blood. </span>2010;<span class="ref-vol">115</span>(11):2319–2328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2844015/">PMC free article</a>]</span> [<a href="/pubmed/20075159" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genetic+variations+in+the+heparanase+gene+(HPSE)+associate+with+increased+risk+of+GVHD+following+allogeneic+stem+cell+transplantation:+effect+of+discrepancy+between+recipients+and+donors.&amp;author=O+Ostrovsky&amp;author=A+Shimoni&amp;author=A+Rand&amp;author=I+Vlodavsky&amp;author=A+Nagler&amp;volume=115&amp;issue=11&amp;publication_year=2010&amp;pages=2319-2328&amp;pmid=20075159&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Bogunia-Kubik K, Uklejewska A, Dickinson A, Jarvis M, Lange A. HSP70-hom gene polymorphism as a prognostic marker of graft-versus-host disease. <span><span class="ref-journal">Transplantation. </span>2006;<span class="ref-vol">82</span>(8):1116–1117.</span> [<a href="/pubmed/17060867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=HSP70-hom+gene+polymorphism+as+a+prognostic+marker+of+graft-versus-host+disease.&amp;author=K+Bogunia-Kubik&amp;author=A+Uklejewska&amp;author=A+Dickinson&amp;author=M+Jarvis&amp;author=A+Lange&amp;volume=82&amp;issue=8&amp;publication_year=2006&amp;pages=1116-1117&amp;pmid=17060867&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Bogunia-Kubik K, Lange A. HSP70-hom gene polymorphism in allogeneic hematopoietic stem-cell transplant recipients correlates with the development of acute graft-versus-host disease. <span><span class="ref-journal">Transplantation. </span>2005;<span class="ref-vol">79</span>(7):815–820.</span> [<a href="/pubmed/15818324" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=HSP70-hom+gene+polymorphism+in+allogeneic+hematopoietic+stem-cell+transplant+recipients+correlates+with+the+development+of+acute+graft-versus-host+disease.&amp;author=K+Bogunia-Kubik&amp;author=A+Lange&amp;volume=79&amp;issue=7&amp;publication_year=2005&amp;pages=815-820&amp;pmid=15818324&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Cullup H, Dickinson AM, Cavet J, Jackson GH, Middleton PG. Polymorphisms of interleukin-1alpha constitute independent risk factors for chronic graft-versus-host disease after allogeneic bone marrow transplantation. <span><span class="ref-journal">Br J Haematol. </span>2003;<span class="ref-vol">122</span>(5):778–787.</span> [<a href="/pubmed/12930389" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Polymorphisms+of+interleukin-1alpha+constitute+independent+risk+factors+for+chronic+graft-versus-host+disease+after+allogeneic+bone+marrow+transplantation.&amp;author=H+Cullup&amp;author=AM+Dickinson&amp;author=J+Cavet&amp;author=GH+Jackson&amp;author=PG+Middleton&amp;volume=122&amp;issue=5&amp;publication_year=2003&amp;pages=778-787&amp;pmid=12930389&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Bertinetto FE, Dall'Omo AM, Mazzola GA, et al.  Role of non-HLA genetic polymorphisms in graft-versus-host disease after haematopoietic stem cell transplantation. <span><span class="ref-journal">Int J Immunogenet. </span>2006;<span class="ref-vol">33</span>(5):375–384.</span> [<a href="/pubmed/16984283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Immunogenet&amp;title=Role+of+non-HLA+genetic+polymorphisms+in+graft-versus-host+disease+after+haematopoietic+stem+cell+transplantation.&amp;author=FE+Bertinetto&amp;author=AM+Dall'Omo&amp;author=GA+Mazzola&amp;volume=33&amp;issue=5&amp;publication_year=2006&amp;pages=375-384&amp;pmid=16984283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">MacMillan ML, Radloff GA, Kiffmeyer WR, DeFor TE, Weisdorf DJ, Davies SM. High-producer interleukin-2 genotype increases risk for acute graft-versus-host disease after unrelated donor bone marrow transplantation. <span><span class="ref-journal">Transplantation. </span>2003;<span class="ref-vol">76</span>(12):1758–1762.</span> [<a href="/pubmed/14688528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=High-producer+interleukin-2+genotype+increases+risk+for+acute+graft-versus-host+disease+after+unrelated+donor+bone+marrow+transplantation.&amp;author=ML+MacMillan&amp;author=GA+Radloff&amp;author=WR+Kiffmeyer&amp;author=TE+DeFor&amp;author=DJ+Weisdorf&amp;volume=76&amp;issue=12&amp;publication_year=2003&amp;pages=1758-1762&amp;pmid=14688528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Karabon L, Wysoczanska B, Bogunia-Kubik K, Suchnicki K, Lange A. IL-6 and IL-10 promoter gene polymorphisms of patients and donors of allogeneic sibling hematopoietic stem cell transplants associate with the risk of acute graft-versus-host disease. <span><span class="ref-journal">Hum Immunol. </span>2005;<span class="ref-vol">66</span>(6):700–710.</span> [<a href="/pubmed/15993715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Immunol&amp;title=IL-6+and+IL-10+promoter+gene+polymorphisms+of+patients+and+donors+of+allogeneic+sibling+hematopoietic+stem+cell+transplants+associate+with+the+risk+of+acute+graft-versus-host+disease.&amp;author=L+Karabon&amp;author=B+Wysoczanska&amp;author=K+Bogunia-Kubik&amp;author=K+Suchnicki&amp;author=A+Lange&amp;volume=66&amp;issue=6&amp;publication_year=2005&amp;pages=700-710&amp;pmid=15993715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Ambruzova Z, Mrazek F, Raida L, et al.  Association of IL6 and CCL2 gene polymorphisms with the outcome of allogeneic haematopoietic stem cell transplantation. <span><span class="ref-journal">Bone Marrow Transplant. </span>2009;<span class="ref-vol">44</span>(4):227–235.</span> [<a href="/pubmed/19234509" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Association+of+IL6+and+CCL2+gene+polymorphisms+with+the+outcome+of+allogeneic+haematopoietic+stem+cell+transplantation.&amp;author=Z+Ambruzova&amp;author=F+Mrazek&amp;author=L+Raida&amp;volume=44&amp;issue=4&amp;publication_year=2009&amp;pages=227-235&amp;pmid=19234509&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">Nordlander A, Uzunel M, Mattsson J, Remberger M. The TNFd4 allele is correlated to moderate-to-severe acute graft-versus-host disease after allogeneic stem cell transplantation. <span><span class="ref-journal">Br J Haematol. </span>2002;<span class="ref-vol">119</span>(4):1133–1136.</span> [<a href="/pubmed/12472598" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=The+TNFd4+allele+is+correlated+to+moderate-to-severe+acute+graft-versus-host+disease+after+allogeneic+stem+cell+transplantation.&amp;author=A+Nordlander&amp;author=M+Uzunel&amp;author=J+Mattsson&amp;author=M+Remberger&amp;volume=119&amp;issue=4&amp;publication_year=2002&amp;pages=1133-1136&amp;pmid=12472598&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">Cavet J, Middleton PG, Segall M, Noreen H, Davies SM, Dickinson AM. Recipient tumor necrosis factor-alpha and interleukin-10 gene polymorphisms associate with early mortality and acute graft-versus-host disease severity in HLA-matched sibling bone marrow transplants. <span><span class="ref-journal">Blood. </span>1999;<span class="ref-vol">94</span>(11):3941–3946.</span> [<a href="/pubmed/10572111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Recipient+tumor+necrosis+factor-alpha+and+interleukin-10+gene+polymorphisms+associate+with+early+mortality+and+acute+graft-versus-host+disease+severity+in+HLA-matched+sibling+bone+marrow+transplants.&amp;author=J+Cavet&amp;author=PG+Middleton&amp;author=M+Segall&amp;author=H+Noreen&amp;author=SM+Davies&amp;volume=94&amp;issue=11&amp;publication_year=1999&amp;pages=3941-3946&amp;pmid=10572111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">Middleton PG, Taylor PR, Jackson G, Proctor SJ, Dickinson AM. Cytokine gene polymorphisms associating with severe acute graft-versus-host disease in HLA-identical sibling transplants. <span><span class="ref-journal">Blood. </span>1998;<span class="ref-vol">92</span>(10):3943–3948.</span> [<a href="/pubmed/9808588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Cytokine+gene+polymorphisms+associating+with+severe+acute+graft-versus-host+disease+in+HLA-identical+sibling+transplants.&amp;author=PG+Middleton&amp;author=PR+Taylor&amp;author=G+Jackson&amp;author=SJ+Proctor&amp;author=AM+Dickinson&amp;volume=92&amp;issue=10&amp;publication_year=1998&amp;pages=3943-3948&amp;pmid=9808588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Stark GL, Dickinson AM, Jackson GH, Taylor PR, Proctor SJ, Middleton PG. Tumour necrosis factor receptor type II 196M/R genotype correlates with circulating soluble receptor levels in normal subjects and with graft-versus-host disease after sibling allogeneic bone marrow transplantation. <span><span class="ref-journal">Transplantation. </span>2003;<span class="ref-vol">76</span>(12):1742–1749.</span> [<a href="/pubmed/14688526" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=Tumour+necrosis+factor+receptor+type+II+196M/R+genotype+correlates+with+circulating+soluble+receptor+levels+in+normal+subjects+and+with+graft-versus-host+disease+after+sibling+allogeneic+bone+marrow+transplantation.&amp;author=GL+Stark&amp;author=AM+Dickinson&amp;author=GH+Jackson&amp;author=PR+Taylor&amp;author=SJ+Proctor&amp;volume=76&amp;issue=12&amp;publication_year=2003&amp;pages=1742-1749&amp;pmid=14688526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Sivula J, Turpeinen H, Volin L, Partanen J. Association of IL-10 and IL-10Rbeta gene polymorphisms with graft-versus-host disease after haematopoietic stem cell transplantation from an HLA-identical sibling donor. <span><span class="ref-journal">BMC Immunol. </span>2009;<span class="ref-vol">10</span>:24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2685414/">PMC free article</a>]</span> [<a href="/pubmed/19409109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Immunol&amp;title=Association+of+IL-10+and+IL-10Rbeta+gene+polymorphisms+with+graft-versus-host+disease+after+haematopoietic+stem+cell+transplantation+from+an+HLA-identical+sibling+donor.&amp;author=J+Sivula&amp;author=H+Turpeinen&amp;author=L+Volin&amp;author=J+Partanen&amp;volume=10&amp;publication_year=2009&amp;pages=24&amp;pmid=19409109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Socié G, Loiseau P, Tamouza R, et al.  Both genetic and clinical factors predict the development of graft-versus-host disease after allogeneic hematopoietic stem cell transplantation. <span><span class="ref-journal">Transplantation. </span>2001;<span class="ref-vol">72</span>(4):699–706.</span> [<a href="/pubmed/11544434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=Both+genetic+and+clinical+factors+predict+the+development+of+graft-versus-host+disease+after+allogeneic+hematopoietic+stem+cell+transplantation.&amp;author=G+Socié&amp;author=P+Loiseau&amp;author=R+Tamouza&amp;volume=72&amp;issue=4&amp;publication_year=2001&amp;pages=699-706&amp;pmid=11544434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">Elmaagacli AH, Koldehoff M, Landt O, Beelen DW. Relation of an interleukin-23 receptor gene polymorphism to graft-versus-host disease after hematopoietic-cell transplantation. <span><span class="ref-journal">Bone Marrow Transplant. </span>2008;<span class="ref-vol">41</span>(9):821–826.</span> [<a href="/pubmed/18209723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Relation+of+an+interleukin-23+receptor+gene+polymorphism+to+graft-versus-host+disease+after+hematopoietic-cell+transplantation.&amp;author=AH+Elmaagacli&amp;author=M+Koldehoff&amp;author=O+Landt&amp;author=DW+Beelen&amp;volume=41&amp;issue=9&amp;publication_year=2008&amp;pages=821-826&amp;pmid=18209723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Gruhn B, Intek J, Pfaffendorf N, et al.  Polymorphism of interleukin-23 receptor gene but not of NOD2/CARD15 is associated with graft-versus-host disease after hematopoietic stem cell transplantation in children. <span><span class="ref-journal">Biol Blood Marrow Transplant. </span>2009;<span class="ref-vol">15</span>(12):1571–1577.</span> [<a href="/pubmed/19896081" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Blood+Marrow+Transplant&amp;title=Polymorphism+of+interleukin-23+receptor+gene+but+not+of+NOD2/CARD15+is+associated+with+graft-versus-host+disease+after+hematopoietic+stem+cell+transplantation+in+children.&amp;author=B+Gruhn&amp;author=J+Intek&amp;author=N+Pfaffendorf&amp;volume=15&amp;issue=12&amp;publication_year=2009&amp;pages=1571-1577&amp;pmid=19896081&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">Wermke M, Maiwald S, Schmelz R, et al.  Genetic variations of interleukin-23R (1143A&gt;G) and BPI (A645G), but not of NOD2, are associated with acute graft-versus-host disease after allogeneic transplantation. <span><span class="ref-journal">Biol Blood Marrow Transplant. </span>2010;<span class="ref-vol">16</span>(12):1718–1727.</span> [<a href="/pubmed/20541026" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Blood+Marrow+Transplant&amp;title=Genetic+variations+of+interleukin-23R+(1143A>G)+and+BPI+(A645G),+but+not+of+NOD2,+are+associated+with+acute+graft-versus-host+disease+after+allogeneic+transplantation.&amp;author=M+Wermke&amp;author=S+Maiwald&amp;author=R+Schmelz&amp;volume=16&amp;issue=12&amp;publication_year=2010&amp;pages=1718-1727&amp;pmid=20541026&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Ambruzova Z, Mrazek F, Raida L, et al.  Possible impact of MADCAM1 gene single nucleotide polymorphisms to the outcome of allogeneic hematopoietic stem cell transplantation. <span><span class="ref-journal">Hum Immunol. </span>2009;<span class="ref-vol">70</span>(6):457–460.</span> [<a href="/pubmed/19286444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Immunol&amp;title=Possible+impact+of+MADCAM1+gene+single+nucleotide+polymorphisms+to+the+outcome+of+allogeneic+hematopoietic+stem+cell+transplantation.&amp;author=Z+Ambruzova&amp;author=F+Mrazek&amp;author=L+Raida&amp;volume=70&amp;issue=6&amp;publication_year=2009&amp;pages=457-460&amp;pmid=19286444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">Robien K, Bigler J, Yasui Y, et al.  Methylenetetrahydrofolate reductase and thymidylate synthase genotypes and risk of acute graft-versus-host disease following hematopoietic cell transplantation for chronic myelogenous leukemia. <span><span class="ref-journal">Biol Blood Marrow Transplant. </span>2006;<span class="ref-vol">12</span>(9):973–980.</span> [<a href="/pubmed/16920564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Blood+Marrow+Transplant&amp;title=Methylenetetrahydrofolate+reductase+and+thymidylate+synthase+genotypes+and+risk+of+acute+graft-versus-host+disease+following+hematopoietic+cell+transplantation+for+chronic+myelogenous+leukemia.&amp;author=K+Robien&amp;author=J+Bigler&amp;author=Y+Yasui&amp;volume=12&amp;issue=9&amp;publication_year=2006&amp;pages=973-980&amp;pmid=16920564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">Sugimoto K, Murata M, Onizuka M, et al.  Decreased risk of acute graft-versus-host disease following allogeneic hematopoietic stem cell transplantation in patients with the 5,10-methylenetetrahydrofolate reductase 677TT genotype. <span><span class="ref-journal">Int J Hematol. </span>2008;<span class="ref-vol">87</span>(5):451–458.</span> [<a href="/pubmed/18365141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Hematol&amp;title=Decreased+risk+of+acute+graft-versus-host+disease+following+allogeneic+hematopoietic+stem+cell+transplantation+in+patients+with+the+5,10-methylenetetrahydrofolate+reductase+677TT+genotype.&amp;author=K+Sugimoto&amp;author=M+Murata&amp;author=M+Onizuka&amp;volume=87&amp;issue=5&amp;publication_year=2008&amp;pages=451-458&amp;pmid=18365141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="element-citation">Rocha V, Porcher R, Fernandes JF, et al.  Association of drug metabolism gene polymorphisms with toxicities, graft-versus-host disease and survival after HLA-identical sibling hematopoietic stem cell transplantation for patients with leukemia. <span><span class="ref-journal">Leukemia. </span>2009;<span class="ref-vol">23</span>(3):545–556.</span> [<a href="/pubmed/19005482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Association+of+drug+metabolism+gene+polymorphisms+with+toxicities,+graft-versus-host+disease+and+survival+after+HLA-identical+sibling+hematopoietic+stem+cell+transplantation+for+patients+with+leukemia.&amp;author=V+Rocha&amp;author=R+Porcher&amp;author=JF+Fernandes&amp;volume=23&amp;issue=3&amp;publication_year=2009&amp;pages=545-556&amp;pmid=19005482&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="element-citation">Murphy N, Diviney M, Szer J, et al.  Donor methylenetetrahydrofolate reductase genotype is associated with graft-versus-host disease in hematopoietic stem cell transplant patients treated with methotrexate. <span><span class="ref-journal">Bone Marrow Transplant. </span>2006;<span class="ref-vol">37</span>(8):773–779.</span> [<a href="/pubmed/16518429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Donor+methylenetetrahydrofolate+reductase+genotype+is+associated+with+graft-versus-host+disease+in+hematopoietic+stem+cell+transplant+patients+treated+with+methotrexate.&amp;author=N+Murphy&amp;author=M+Diviney&amp;author=J+Szer&amp;volume=37&amp;issue=8&amp;publication_year=2006&amp;pages=773-779&amp;pmid=16518429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="element-citation">Holler E, Rogler G, Brenmoehl J, et al.  The role of genetic variants of NOD2/CARD15, a receptor of the innate immune system, in GvHD and complications following related and unrelated donor haematopoietic stem cell transplantation. <span><span class="ref-journal">Int J Immunogenet. </span>2008;<span class="ref-vol">35</span>(4-5):381–384.</span> [<a href="/pubmed/18976442" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Immunogenet&amp;title=The+role+of+genetic+variants+of+NOD2/CARD15,+a+receptor+of+the+innate+immune+system,+in+GvHD+and+complications+following+related+and+unrelated+donor+haematopoietic+stem+cell+transplantation.&amp;author=E+Holler&amp;author=G+Rogler&amp;author=J+Brenmoehl&amp;volume=35&amp;issue=4-5&amp;publication_year=2008&amp;pages=381-384&amp;pmid=18976442&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="element-citation">Holler E, Rogler G, Brenmoehl J, et al.  Prognostic significance of NOD2/CARD15 variants in HLA-identical sibling hematopoietic stem cell transplantation: effect on long-term outcome is confirmed in 2 independent cohorts and may be modulated by the type of gastrointestinal decontamination. <span><span class="ref-journal">Blood. </span>2006;<span class="ref-vol">107</span>(10):4189–4193.</span> [<a href="/pubmed/16424393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Prognostic+significance+of+NOD2/CARD15+variants+in+HLA-identical+sibling+hematopoietic+stem+cell+transplantation:+effect+on+long-term+outcome+is+confirmed+in+2+independent+cohorts+and+may+be+modulated+by+the+type+of+gastrointestinal+decontamination.&amp;author=E+Holler&amp;author=G+Rogler&amp;author=J+Brenmoehl&amp;volume=107&amp;issue=10&amp;publication_year=2006&amp;pages=4189-4193&amp;pmid=16424393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="element-citation">Holler E, Rogler G, Herfarth H, et al.  Both donor and recipient NOD2/CARD15 mutations associate with transplant-related mortality and GvHD following allogeneic stem cell transplantation. <span><span class="ref-journal">Blood. </span>2004;<span class="ref-vol">104</span>(3):889–894.</span> [<a href="/pubmed/15090455" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Both+donor+and+recipient+NOD2/CARD15+mutations+associate+with+transplant-related+mortality+and+GvHD+following+allogeneic+stem+cell+transplantation.&amp;author=E+Holler&amp;author=G+Rogler&amp;author=H+Herfarth&amp;volume=104&amp;issue=3&amp;publication_year=2004&amp;pages=889-894&amp;pmid=15090455&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="element-citation">van der Velden WJ, Blijlevens NM, Maas FM, et al.  NOD2 polymorphisms predict severe acute graft-versus-host and treatment-related mortality in T-cell-depleted haematopoietic stem cell transplantation. <span><span class="ref-journal">Bone Marrow Transplant. </span>2009;<span class="ref-vol">44</span>(4):243–248.</span> [<a href="/pubmed/19219079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=NOD2+polymorphisms+predict+severe+acute+graft-versus-host+and+treatment-related+mortality+in+T-cell-depleted+haematopoietic+stem+cell+transplantation.&amp;author=WJ+van+der+Velden&amp;author=NM+Blijlevens&amp;author=FM+Maas&amp;volume=44&amp;issue=4&amp;publication_year=2009&amp;pages=243-248&amp;pmid=19219079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="element-citation">Arora M, Lindgren B, Basu S, et al.  Polymorphisms in the base excision repair pathway and graft-versus-host disease. <span><span class="ref-journal">Leukemia. </span>2010;<span class="ref-vol">24</span>(8):1470–1475.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2921004/">PMC free article</a>]</span> [<a href="/pubmed/20574454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Polymorphisms+in+the+base+excision+repair+pathway+and+graft-versus-host+disease.&amp;author=M+Arora&amp;author=B+Lindgren&amp;author=S+Basu&amp;volume=24&amp;issue=8&amp;publication_year=2010&amp;pages=1470-1475&amp;pmid=20574454&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="element-citation">Tambur AR, Yaniv I, Stein J, et al.  Cytokine gene polymorphism in patients with graft-versus-host disease. <span><span class="ref-journal">Transplant Proc. </span>2001;<span class="ref-vol">33</span>(1-2):502–503.</span> [<a href="/pubmed/11266928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplant+Proc&amp;title=Cytokine+gene+polymorphism+in+patients+with+graft-versus-host+disease.&amp;author=AR+Tambur&amp;author=I+Yaniv&amp;author=J+Stein&amp;volume=33&amp;issue=1-2&amp;publication_year=2001&amp;pages=502-503&amp;pmid=11266928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="element-citation">Hattori H, Matsuzaki A, Suminoe A, et al.  Polymorphisms of transforming growth factor-beta1 and transforming growth factor-beta1 type II receptor genes are associated with acute graft-versus-host disease in children with HLA-matched sibling bone marrow transplantation. <span><span class="ref-journal">Bone Marrow Transplant. </span>2002;<span class="ref-vol">30</span>(10):665–671.</span> [<a href="/pubmed/12420205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Polymorphisms+of+transforming+growth+factor-beta1+and+transforming+growth+factor-beta1+type+II+receptor+genes+are+associated+with+acute+graft-versus-host+disease+in+children+with+HLA-matched+sibling+bone+marrow+transplantation.&amp;author=H+Hattori&amp;author=A+Matsuzaki&amp;author=A+Suminoe&amp;volume=30&amp;issue=10&amp;publication_year=2002&amp;pages=665-671&amp;pmid=12420205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="element-citation">Takahashi H, Furukawa T, Hashimoto S, et al.  Contribution of TNF-alpha and IL-10 gene polymorphisms to graft-versus-host disease following allo-hematopoietic stem cell transplantation. <span><span class="ref-journal">Bone Marrow Transplant. </span>2000;<span class="ref-vol">26</span>(12):1317–1323.</span> [<a href="/pubmed/11223972" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Contribution+of+TNF-alpha+and+IL-10+gene+polymorphisms+to+graft-versus-host+disease+following+allo-hematopoietic+stem+cell+transplantation.&amp;author=H+Takahashi&amp;author=T+Furukawa&amp;author=S+Hashimoto&amp;volume=26&amp;issue=12&amp;publication_year=2000&amp;pages=1317-1323&amp;pmid=11223972&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="element-citation">Ishikawa Y, Kashiwase K, Akaza T, et al.  Polymorphisms in TNFA and TNFR2 affect outcome of unrelated bone marrow transplantation. <span><span class="ref-journal">Bone Marrow Transplant. </span>2002;<span class="ref-vol">29</span>(7):569–575.</span> [<a href="/pubmed/11979305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Polymorphisms+in+TNFA+and+TNFR2+affect+outcome+of+unrelated+bone+marrow+transplantation.&amp;author=Y+Ishikawa&amp;author=K+Kashiwase&amp;author=T+Akaza&amp;volume=29&amp;issue=7&amp;publication_year=2002&amp;pages=569-575&amp;pmid=11979305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="element-citation">Cavet J, Dickinson AM, Norden J, Taylor PR, Jackson GH, Middleton PG. Interferon-gamma and interleukin-6 gene polymorphisms associate with graft-versus-host disease in HLA-matched sibling bone marrow transplantation. <span><span class="ref-journal">Blood. </span>2001;<span class="ref-vol">98</span>(5):1594–1600.</span> [<a href="/pubmed/11520812" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Interferon-gamma+and+interleukin-6+gene+polymorphisms+associate+with+graft-versus-host+disease+in+HLA-matched+sibling+bone+marrow+transplantation.&amp;author=J+Cavet&amp;author=AM+Dickinson&amp;author=J+Norden&amp;author=PR+Taylor&amp;author=GH+Jackson&amp;volume=98&amp;issue=5&amp;publication_year=2001&amp;pages=1594-1600&amp;pmid=11520812&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="element-citation">Kim DH, Lee NY, Lee MH, Sohn SK. Vascular endothelial growth factor gene polymorphisms may predict the risk of acute graft-versus-host disease following allogeneic transplantation: preventive effect of vascular endothelial growth factor gene on acute graft-versus-host disease. <span><span class="ref-journal">Biol Blood Marrow Transplant. </span>2008;<span class="ref-vol">14</span>(12):1408–1416.</span> [<a href="/pubmed/19041064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Blood+Marrow+Transplant&amp;title=Vascular+endothelial+growth+factor+gene+polymorphisms+may+predict+the+risk+of+acute+graft-versus-host+disease+following+allogeneic+transplantation:+preventive+effect+of+vascular+endothelial+growth+factor+gene+on+acute+graft-versus-host+disease.&amp;author=DH+Kim&amp;author=NY+Lee&amp;author=MH+Lee&amp;author=SK+Sohn&amp;volume=14&amp;issue=12&amp;publication_year=2008&amp;pages=1408-1416&amp;pmid=19041064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="element-citation">Hansen JA, Chien JW, Warren EH, Zhao LP, Martin PJ. Defining genetic risk for graft-versus-host disease and mortality following allogeneic hematopoietic stem cell transplantation. <span><span class="ref-journal">Curr Opin Hematol. </span>2010;<span class="ref-vol">17</span>(6):483–492.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3177530/">PMC free article</a>]</span> [<a href="/pubmed/20827186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Hematol&amp;title=Defining+genetic+risk+for+graft-versus-host+disease+and+mortality+following+allogeneic+hematopoietic+stem+cell+transplantation.&amp;author=JA+Hansen&amp;author=JW+Chien&amp;author=EH+Warren&amp;author=LP+Zhao&amp;author=PJ+Martin&amp;volume=17&amp;issue=6&amp;publication_year=2010&amp;pages=483-492&amp;pmid=20827186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46">46. <span class="element-citation">Marchini J, Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>(7):499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genotype+imputation+for+genome-wide+association+studies.&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;issue=7&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47">47. <span class="element-citation">Fishman D, Faulds G, Jeffery R, et al.  The effect of novel polymorphisms in the interleukin-6 (IL-6) gene on IL-6 transcription and plasma IL-6 levels, and an association with systemic-onset juvenile chronic arthritis. <span><span class="ref-journal">J Clin Invest. </span>1998;<span class="ref-vol">102</span>(7):1369–1376.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC508984/">PMC free article</a>]</span> [<a href="/pubmed/9769329" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=The+effect+of+novel+polymorphisms+in+the+interleukin-6+(IL-6)+gene+on+IL-6+transcription+and+plasma+IL-6+levels,+and+an+association+with+systemic-onset+juvenile+chronic+arthritis.&amp;author=D+Fishman&amp;author=G+Faulds&amp;author=R+Jeffery&amp;volume=102&amp;issue=7&amp;publication_year=1998&amp;pages=1369-1376&amp;pmid=9769329&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48">48. <span class="element-citation">Illig T, Bongardt F, Schopfer A, et al.  Significant association of the interleukin-6 gene polymorphisms C-174G and A-598G with type 2 diabetes. <span><span class="ref-journal">J Clin Endocr Metab. </span>2004;<span class="ref-vol">89</span>(10):5053–5058.</span> [<a href="/pubmed/15472205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Endocr+Metab&amp;title=Significant+association+of+the+interleukin-6+gene+polymorphisms+C-174G+and+A-598G+with+type+2+diabetes.&amp;author=T+Illig&amp;author=F+Bongardt&amp;author=A+Schopfer&amp;volume=89&amp;issue=10&amp;publication_year=2004&amp;pages=5053-5058&amp;pmid=15472205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B49">49. <span class="element-citation">Lu KC, Jaramillo A, Lecha RL, et al.  Interleukin-6 and interferon-gamma gene polymorphisms in the development of bronchiolitis obliterans syndrome after lung transplantation. <span><span class="ref-journal">Transplantation. </span>2002;<span class="ref-vol">74</span>(9):1297–1302.</span> [<a href="/pubmed/12451269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=Interleukin-6+and+interferon-gamma+gene+polymorphisms+in+the+development+of+bronchiolitis+obliterans+syndrome+after+lung+transplantation.&amp;author=KC+Lu&amp;author=A+Jaramillo&amp;author=RL+Lecha&amp;volume=74&amp;issue=9&amp;publication_year=2002&amp;pages=1297-1302&amp;pmid=12451269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B50">50. <span class="element-citation">Muller-Steinhardt M, Hartel C, Muller B, Kirchner H, Fricke L. The interleukin-6-174promoter polymorphism is associated with long-term kidney allograft survival. <span><span class="ref-journal">Kidney Int. </span>2002;<span class="ref-vol">62</span>(5):1824–1827.</span> [<a href="/pubmed/12371985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=The+interleukin-6-174promoter+polymorphism+is+associated+with+long-term+kidney+allograft+survival.&amp;author=M+Muller-Steinhardt&amp;author=C+Hartel&amp;author=B+Muller&amp;author=H+Kirchner&amp;author=L+Fricke&amp;volume=62&amp;issue=5&amp;publication_year=2002&amp;pages=1824-1827&amp;pmid=12371985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B51">51. <span class="element-citation">Hirschhorn JN, Lohmueller K, Byrne E, Hirschhorn K. A comprehensive review of genetic association studies. <span><span class="ref-journal">Genet Med. </span>2002;<span class="ref-vol">4</span>(2):45–61.</span> [<a href="/pubmed/11882781" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Med&amp;title=A+comprehensive+review+of+genetic+association+studies.&amp;author=JN+Hirschhorn&amp;author=K+Lohmueller&amp;author=E+Byrne&amp;author=K+Hirschhorn&amp;volume=4&amp;issue=2&amp;publication_year=2002&amp;pages=45-61&amp;pmid=11882781&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3369619&amp;issue-id=210848&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm139713415673904title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Methods</a></li><li><a href="#__sec7title">Results</a></li><li><a href="#__sec11title">Discussion</a></li><li><a href="#idm139713364493472title">Acknowledgments</a></li><li><a href="#idm139713371273296title">Footnotes</a></li><li><a href="#__sec12title">Authorship</a></li><li><a href="#idm139713416371616title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Blood</span> are provided here courtesy of <strong>The American Society of Hematology</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3369619/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3369619/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3369619/?report=printable">Printer Friendly</a></li> | <li><a href="#" data-citationid="PMC3369619" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369619%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369619%2F&amp;text=Evaluation%20of%20published%20single%20nucleotide%20polymorphisms%20associated%20with%20acute%20GVHD" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369619%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22282500" data-aiid="3369619" data-aid="3369619" data-iid="210848" data-domainid="446" data-domain="blood" data-accid="PMC3369619" data-md5="0c546d158eb79e3bc0a6d7f533758956"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:26:37-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CB806DD53BCF1000000000231016C&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3369619%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=blood&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3369619/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>