<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3493132/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3493132/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer"><meta name="citation_authors" content="David J. Hunter, Peter Kraft, Kevin B. Jacobs, David G. Cox, Meredith Yeager, Susan E. Hankinson, Sholom Wacholder, Zhaoming Wang, Robert Welch, Amy Hutchinson, Kai Yu, Nilanjan Chatterjee, Nick Orr, Walter C. Willett, Graham A. Colditz, Regina G. Ziegler, Christine D. Berg, Saundra S. Buys, Catherine A. McCarty, Heather Spencer Feigelson, Eugenia E. Calle, Michael J. Thun, Richard B. Hayes, Margaret Tucker, Daniela S. Gerhard, Joseph F. Fraumeni, Jr., Robert N. Hoover, Gilles Thomas, Stephen J Chanock"><meta name="citation_date" content="July 2007"><meta name="citation_issue" content="7"><meta name="citation_volume" content="39"><meta name="citation_firstpage" content="870"><meta name="citation_doi" content="10.1038/ng2075"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3493132/?report=abstract"><meta name="citation_pmid" content="17529973"><meta name="DC.Title" content="A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="David J. Hunter"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="Kevin B. Jacobs"><meta name="DC.Contributor" content="David G. Cox"><meta name="DC.Contributor" content="Meredith Yeager"><meta name="DC.Contributor" content="Susan E. Hankinson"><meta name="DC.Contributor" content="Sholom Wacholder"><meta name="DC.Contributor" content="Zhaoming Wang"><meta name="DC.Contributor" content="Robert Welch"><meta name="DC.Contributor" content="Amy Hutchinson"><meta name="DC.Contributor" content="Kai Yu"><meta name="DC.Contributor" content="Nilanjan Chatterjee"><meta name="DC.Contributor" content="Nick Orr"><meta name="DC.Contributor" content="Walter C. Willett"><meta name="DC.Contributor" content="Graham A. Colditz"><meta name="DC.Contributor" content="Regina G. Ziegler"><meta name="DC.Contributor" content="Christine D. Berg"><meta name="DC.Contributor" content="Saundra S. Buys"><meta name="DC.Contributor" content="Catherine A. McCarty"><meta name="DC.Contributor" content="Heather Spencer Feigelson"><meta name="DC.Contributor" content="Eugenia E. Calle"><meta name="DC.Contributor" content="Michael J. Thun"><meta name="DC.Contributor" content="Richard B. Hayes"><meta name="DC.Contributor" content="Margaret Tucker"><meta name="DC.Contributor" content="Daniela S. Gerhard"><meta name="DC.Contributor" content="Joseph F. Fraumeni, Jr."><meta name="DC.Contributor" content="Robert N. Hoover"><meta name="DC.Contributor" content="Gilles Thomas"><meta name="DC.Contributor" content="Stephen J Chanock"><meta name="DC.Date" content="2007 Jul"><meta name="DC.Identifier" content="10.1038/ng2075"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer"><meta property="og:type" content="article"><meta property="og:description" content="We conducted a genome-wide association study (GWAS) of breast cancer by genotyping 528,173 single nucleotide polymorphisms (SNPs) in 1,145 cases of invasive breast cancer among postmenopausal white women, and 1,142 controls. We identified a set of four ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3493132/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3493132/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3493132/pdf/nihms415304.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88108DDD3E993100000000050C02DC.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3493132%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3493132</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2012 Nov 8.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17529973" target="pmc_ext" ref="reftype=publisher&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2007 Jul; 39(7): 870–874. </span></a></div><span class="fm-vol-iss-date">Published online 2007 May 27. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng2075" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng2075</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3493132</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS415304</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/17529973">17529973</a></div></div></div></div><h1 class="content-title">A genome-wide association study identifies alleles in <em>FGFR2</em> associated with risk of sporadic postmenopausal breast cancer</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365851079056" co-class="co-affbox">David J. Hunter</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>6</sup> <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365851074976" co-class="co-affbox">Peter Kraft</a>,<sup>2</sup> <a href="/pubmed/?term=Jacobs%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365842071232" co-class="co-affbox">Kevin B. Jacobs</a>,<sup>4</sup> <a href="/pubmed/?term=Cox%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365842069248" co-class="co-affbox">David G. Cox</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Yeager%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365842066528" co-class="co-affbox">Meredith Yeager</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Hankinson%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365847467200" co-class="co-affbox">Susan E. Hankinson</a>,<sup>1</sup> <a href="/pubmed/?term=Wacholder%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365847465216" co-class="co-affbox">Sholom Wacholder</a>,<sup>6</sup> <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365847463232" co-class="co-affbox">Zhaoming Wang</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Welch%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365844669184" co-class="co-affbox">Robert Welch</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Hutchinson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365844666464" co-class="co-affbox">Amy Hutchinson</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Yu%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365844663744" co-class="co-affbox">Kai Yu</a>,<sup>6</sup> <a href="/pubmed/?term=Chatterjee%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365839600928" co-class="co-affbox">Nilanjan Chatterjee</a>,<sup>6</sup> <a href="/pubmed/?term=Orr%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365839598944" co-class="co-affbox">Nick Orr</a>,<sup>7</sup> <a href="/pubmed/?term=Willett%20WC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365839596960" co-class="co-affbox">Walter C. Willett</a>,<sup>1,</sup><sup>8</sup> <a href="/pubmed/?term=Colditz%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365847138448" co-class="co-affbox">Graham A. Colditz</a>,<sup>9</sup> <a href="/pubmed/?term=Ziegler%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365847136464" co-class="co-affbox">Regina G. Ziegler</a>,<sup>6</sup> <a href="/pubmed/?term=Berg%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365847134480" co-class="co-affbox">Christine D. Berg</a>,<sup>10</sup> <a href="/pubmed/?term=Buys%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365848132352" co-class="co-affbox">Saundra S. Buys</a>,<sup>11</sup> <a href="/pubmed/?term=McCarty%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365848130368" co-class="co-affbox">Catherine A. McCarty</a>,<sup>12</sup> <a href="/pubmed/?term=Feigelson%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365848128384" co-class="co-affbox">Heather Spencer Feigelson</a>,<sup>13</sup> <a href="/pubmed/?term=Calle%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365841009568" co-class="co-affbox">Eugenia E. Calle</a>,<sup>13</sup> <a href="/pubmed/?term=Thun%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365841007584" co-class="co-affbox">Michael J. Thun</a>,<sup>13</sup> <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365841005600" co-class="co-affbox">Richard B. Hayes</a>,<sup>6</sup> <a href="/pubmed/?term=Tucker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365844898848" co-class="co-affbox">Margaret Tucker</a>,<sup>6</sup> <a href="/pubmed/?term=Gerhard%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365844896976" co-class="co-affbox">Daniela S. Gerhard</a>,<sup>14</sup> <a href="/pubmed/?term=Fraumeni%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365844894992" co-class="co-affbox">Joseph F. Fraumeni, Jr.</a>,<sup>6</sup> <a href="/pubmed/?term=Hoover%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365844892752" co-class="co-affbox">Robert N. Hoover</a>,<sup>6</sup> <a href="/pubmed/?term=Thomas%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365847184576" co-class="co-affbox">Gilles Thomas</a>,<sup>6</sup> and  <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973" class="affpopup" co-rid="_co_idm140365847182592" co-class="co-affbox">Stephen J Chanock</a><sup>6,</sup><sup>7</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140365851079056"><h3 class="no_margin">David J. Hunter</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</p><p><sup>2</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts</p><p><sup>3</sup>Broad Institute of Harvard and MIT</p><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">David J. Hunter</a></div></div><div id="_co_idm140365851074976"><h3 class="no_margin">Peter Kraft</h3><p><sup>2</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts</p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Peter Kraft</a></div></div><div id="_co_idm140365842071232"><h3 class="no_margin">Kevin B. Jacobs</h3><p><sup>4</sup>Bioinformed Consulting Services, Gaithersburg, MD</p><div>Find articles by <a href="/pubmed/?term=Jacobs%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Kevin B. Jacobs</a></div></div><div id="_co_idm140365842069248"><h3 class="no_margin">David G. Cox</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</p><p><sup>2</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts</p><div>Find articles by <a href="/pubmed/?term=Cox%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">David G. Cox</a></div></div><div id="_co_idm140365842066528"><h3 class="no_margin">Meredith Yeager</h3><p><sup>5</sup>SAIC-Frederick, NCI-FCRDC, Frederick, MD</p><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Yeager%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Meredith Yeager</a></div></div><div id="_co_idm140365847467200"><h3 class="no_margin">Susan E. Hankinson</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</p><div>Find articles by <a href="/pubmed/?term=Hankinson%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Susan E. Hankinson</a></div></div><div id="_co_idm140365847465216"><h3 class="no_margin">Sholom Wacholder</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Wacholder%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Sholom Wacholder</a></div></div><div id="_co_idm140365847463232"><h3 class="no_margin">Zhaoming Wang</h3><p><sup>5</sup>SAIC-Frederick, NCI-FCRDC, Frederick, MD</p><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Zhaoming Wang</a></div></div><div id="_co_idm140365844669184"><h3 class="no_margin">Robert Welch</h3><p><sup>5</sup>SAIC-Frederick, NCI-FCRDC, Frederick, MD</p><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Welch%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Robert Welch</a></div></div><div id="_co_idm140365844666464"><h3 class="no_margin">Amy Hutchinson</h3><p><sup>5</sup>SAIC-Frederick, NCI-FCRDC, Frederick, MD</p><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Hutchinson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Amy Hutchinson</a></div></div><div id="_co_idm140365844663744"><h3 class="no_margin">Kai Yu</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Yu%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Kai Yu</a></div></div><div id="_co_idm140365839600928"><h3 class="no_margin">Nilanjan Chatterjee</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Chatterjee%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Nilanjan Chatterjee</a></div></div><div id="_co_idm140365839598944"><h3 class="no_margin">Nick Orr</h3><p><sup>7</sup>Pediatric Oncology Branch, Center for Cancer Research, NCI, NIH, DHHS</p><div>Find articles by <a href="/pubmed/?term=Orr%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Nick Orr</a></div></div><div id="_co_idm140365839596960"><h3 class="no_margin">Walter C. Willett</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</p><p><sup>8</sup>Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts</p><div>Find articles by <a href="/pubmed/?term=Willett%20WC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Walter C. Willett</a></div></div><div id="_co_idm140365847138448"><h3 class="no_margin">Graham A. Colditz</h3><p><sup>9</sup>Washington University School of Medicine, St. Louis, MO</p><div>Find articles by <a href="/pubmed/?term=Colditz%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Graham A. Colditz</a></div></div><div id="_co_idm140365847136464"><h3 class="no_margin">Regina G. Ziegler</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Ziegler%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Regina G. Ziegler</a></div></div><div id="_co_idm140365847134480"><h3 class="no_margin">Christine D. Berg</h3><p><sup>10</sup>Division of Cancer Prevention, NCI, NIH, DHHS</p><div>Find articles by <a href="/pubmed/?term=Berg%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Christine D. Berg</a></div></div><div id="_co_idm140365848132352"><h3 class="no_margin">Saundra S. Buys</h3><p><sup>11</sup>Department of Internal Medicine, University of Utah, Salt Lake City, UT</p><div>Find articles by <a href="/pubmed/?term=Buys%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Saundra S. Buys</a></div></div><div id="_co_idm140365848130368"><h3 class="no_margin">Catherine A. McCarty</h3><p><sup>12</sup>The Center for Human Genetics, Marshfield Clinic Research Foundation, Marshfield, WI</p><div>Find articles by <a href="/pubmed/?term=McCarty%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Catherine A. McCarty</a></div></div><div id="_co_idm140365848128384"><h3 class="no_margin">Heather Spencer Feigelson</h3><p><sup>13</sup>Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</p><div>Find articles by <a href="/pubmed/?term=Feigelson%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Heather Spencer Feigelson</a></div></div><div id="_co_idm140365841009568"><h3 class="no_margin">Eugenia E. Calle</h3><p><sup>13</sup>Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</p><div>Find articles by <a href="/pubmed/?term=Calle%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Eugenia E. Calle</a></div></div><div id="_co_idm140365841007584"><h3 class="no_margin">Michael J. Thun</h3><p><sup>13</sup>Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</p><div>Find articles by <a href="/pubmed/?term=Thun%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Michael J. Thun</a></div></div><div id="_co_idm140365841005600"><h3 class="no_margin">Richard B. Hayes</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Richard B. Hayes</a></div></div><div id="_co_idm140365844898848"><h3 class="no_margin">Margaret Tucker</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Tucker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Margaret Tucker</a></div></div><div id="_co_idm140365844896976"><h3 class="no_margin">Daniela S. Gerhard</h3><p><sup>14</sup>Office of Cancer Genomics, NCI, NIH, DHHS</p><div>Find articles by <a href="/pubmed/?term=Gerhard%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Daniela S. Gerhard</a></div></div><div id="_co_idm140365844894992"><h3 class="no_margin">Joseph F. Fraumeni, Jr.</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Fraumeni%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Joseph F. Fraumeni, Jr.</a></div></div><div id="_co_idm140365844892752"><h3 class="no_margin">Robert N. Hoover</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Hoover%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Robert N. Hoover</a></div></div><div id="_co_idm140365847184576"><h3 class="no_margin">Gilles Thomas</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><div>Find articles by <a href="/pubmed/?term=Thomas%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Gilles Thomas</a></div></div><div id="_co_idm140365847182592"><h3 class="no_margin">Stephen J Chanock</h3><p><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</p><p><sup>7</sup>Pediatric Oncology Branch, Center for Cancer Research, NCI, NIH, DHHS</p><div>Find articles by <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17529973">Stephen J Chanock</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140365851052656_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140365851052656_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140365851052656_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Broad Institute of Harvard and MIT</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Bioinformed Consulting Services, Gaithersburg, MD</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>SAIC-Frederick, NCI-FCRDC, Frederick, MD</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Pediatric Oncology Branch, Center for Cancer Research, NCI, NIH, DHHS</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Washington University School of Medicine, St. Louis, MO</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Division of Cancer Prevention, NCI, NIH, DHHS</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Internal Medicine, University of Utah, Salt Lake City, UT</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>The Center for Human Genetics, Marshfield Clinic Research Foundation, Marshfield, WI</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Office of Cancer Genomics, NCI, NIH, DHHS</div><div id="cor1"><sup>*</sup>Correspondence to: David J. Hunter, 181 Longwood Ave, Boston, MA 02115, Telephone: 617-525-2755, Fax: 617-525-2008, <a href="mailto:dev@null" data-email="ude.dravrah.hpsh@retnuhd" class="oemail">ude.dravrah.hpsh@retnuhd</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140365851052656_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17529973" target="pmc_ext" ref="reftype=publisher&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3493132/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm140365842488288" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140365842488288title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">We conducted a genome-wide association study (GWAS) of breast cancer by genotyping 528,173 single nucleotide polymorphisms (SNPs) in 1,145 cases of invasive breast cancer among postmenopausal white women, and 1,142 controls. We identified a set of four SNPs in intron 2 of <em>FGFR2,</em> a tyrosine kinase receptor previously shown to be amplified and/or over-expressed in some breast cancers, as highly associated with breast cancer and we confirmed this association in 1,776 cases and 2,072 controls from three additional studies. In both association testing and ancestral recombination graph analysis, <em>FGFR2</em> haplotypes were associated with risk of breast cancer. Across the four studies the association with all four SNPs was highly statistically significant (P<sub>trend</sub> for the most strongly associated SNP, rs1219648 = 1.1 × 10<sup>−10</sup>; population attributable risk = 16%). Four SNPs at other chromosomal loci most strongly associated with breast cancer in the initial GWAS were not associated with risk in the three replication studies. Our summary results from the GWAS are freely available online in a form that should speed the identification of additional loci conferring risk.</p></div></div><div id="idm140365844493520" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Family history is an established risk factor for breast cancer, yet estimates of the inherited component of the disease are uncertain<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980718">1</a></sup>. Investigation of multiple-case families segregating breast cancer with Mendelian pattern inheritance led to the identification of the tumor suppressor genes <em>BRCA1</em> and <em>BRCA2</em> that account for a substantial proportion of early-onset breast cancer, but a much smaller proportion of late-onset disease<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980714">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980705">3</a></sup>. Most late-onset cases occur in the absence of a first-degree family history of breast cancer and are often called “sporadic” cases. Previously, family-based studies have been the primary focus of study in the search for genetic determinants, but with new technologies that enable analysis of hundreds of thousands of SNPs, together with new insights into the structure of genomic variation in the human genome, it is now possible to scan across the genome in an agnostic manner in search of common genetic variants associated with disease risk<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980711">4</a></sup>. One strategy for conducting a GWAS is to analyze early-onset cases, often enriched with cases with a positive family history of the disease, in order to maximize the opportunity to detect inherited causal variants. Already, two such studies, one of diabetes and one of breast cancer have successfully identified and replicated common genetic variants<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980704">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373896172">6</a></sup>. Alternatively, GWAS analysis of older subjects may identify common genetic variants associated with sporadic disease, as has been successfully demonstrated for prostate cancer<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303859306">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303859309">8</a></sup>.</p><p id="P3">We initially genotyped 1183 cases of postmenopausal invasive breast cancer and 1185 individually-matched controls from the Nurses’ Health Study (NHS) cohort, using the Illumina HumanHap500 array, as part of the National Cancer Institute CGEMS (<span class="underline">C</span>ancer <span class="underline">G</span>enetic <span class="underline">M</span>arkers of <span class="underline">S</span>usceptibility) Project. Cases were identified from a consecutive series of postmenopausal women, unselected for any other characteristics, who were among the 32,826 cohort members who gave a blood sample in 1989–1990 and had not been previously diagnosed with breast cancer, but who were subsequently diagnosed prior to June 1, 2004. Controls were women who were not diagnosed with breast cancer during follow-up, were matched to cases on year of birth and post-menopausal hormone use at blood draw, and were postmenopausal. All cases and controls were self-described Caucasians. Fifty-nine (30 cases and 29 controls) samples were removed from the analysis because of completion rates less than 90% for the 528,173 SNPs that passed quality control. An additional 18 (5 cases and 13 controls) were removed because of unclear identity (or possible contamination) and 4 (3 cases and 1 control) were removed from the analysis due to evidence of intercontinental admixture (Supplemental Figure 1). Thus, the GWAS analysis was performed on 1,145 cases and 1,142 controls.</p><p id="P4">We analyzed each locus in a logistic regression model using a two-degree of freedom score test with indicator variables for heterozygous and homozygote carriers of the variant allele (for rare variants we collapsed heterozygote and homozygote carrier categories, see Methods). The distribution of the observed P values shows no suggestion of distortion due to population stratification or other sources of bias or distortion of Type I error rates (Supplemental Figures 2a,2b). When we adjusted for matching factors and the top three principal components from an analysis of genetic covariance, the overall distribution of p-values did not significantly change (Komolgorov-Smirnov p=0.34). All loci with unadjusted p&lt;2×10<sup>−5</sup> maintained this level of significance after adjustment. This suggests that these associations are not due to population stratification.</p><p id="P5">The GWAS identified several genomic locations as potentially associated with breast cancer (<a href="/pmc/articles/PMC3493132/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig 1</span></a>). Of 528,173 SNP’s tested, two of the most significant P values (rs1219648 and rs2420946, <a href="/pmc/articles/PMC3493132/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) were in intron 2 (<a href="/pmc/articles/PMC3493132/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig 2</span></a>) of the gene <em>FGFR2</em>, a tyrosine kinase receptor previously shown to be important in mammary gland development and neoplasia<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980721">9</a></sup>; an additional two SNPs in <em>FGFR2</em> (rs11200014 and rs2981579) were among the 16 most extreme P values from the unadjusted analysis. Modeling all pairwise combinations of the four SNPs and their interactions, as well as haplotypes of the four SNPs, suggested that all four were similar with respect to their association with breast cancer risk, consistent with the very high degree of linkage disequilibrium (each pairwise D’ &gt;0.95 and r<sup>2</sup>&gt;0.84). None of the other 31 SNPs at the <em>FGFR2</em> locus were in strong linkage disequilibrium (r<sup>2</sup>) with the SNPs in intron 2 and none were associated with breast cancer risk (<a href="/pmc/articles/PMC3493132/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig 2</span></a>). In silica determination of haplotypes (see Methods) indicated 4 common haplotypes, with the AAGT haplotype the most common risk haplotype, present in 43.58% of case chromosomes and 36.87% of control chromosomes in the Nurses’ Health Study (<a href="/pmc/articles/PMC3493132/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3493132/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365849719184" class="figure"><a href="/pmc/articles/PMC3493132/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3493132_nihms415304f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms415304f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3493132/bin/nihms415304f1.jpg"></a></div><div id="largeobj_idm140365849719184" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3493132/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3493132/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Summary of genome-wide association study results by chromosome</strong><p id="P40">Association with breast cancer was determined for 528,173 SNPs among 1,145 cases of postmenopausal breast cancer, and 1,142 controls. The x axis represents position on each chromosome from pter (left) to qter (right) ; the y axis shows the P value on a logarithmic scale. Only P values &lt;10<sup>−2</sup> are displayed.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3493132/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365843794400" class="figure"><a href="/pmc/articles/PMC3493132/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3493132_nihms415304f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms415304f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3493132/bin/nihms415304f2.jpg"></a></div><div id="largeobj_idm140365843794400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3493132/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3493132/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Association Analysis of SNPs Across the FGFR2 gene</strong><p id="P41"><em>Upper Panel.</em> P-values for association testing drawn from the genome wide association scan covering the FGFR2 gene and 100 Kb 5’ upstream. The presented analysis is based on the two degrees of freedom test corrected for age and the three first principal components of population stratification (see Supplemental materials and Methods).</p><p id="P42"><em>Lower Panel</em>. Estimates of the squared correlation coefficient, r<sup>2</sup>, were calculated for each pairwise comparison of SNPs. Log<sub>10</sub> r<sup>2</sup> was color coded according to the scale shown on the left. The 4 black diamonds indicate the 4 SNPs most strongly associated with breast cancer risk.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P43">The six SNPs with the smallest P values of the 528,173 tested among 1,145 cases of postmenopausal invasive breast cancer and 1,142 controls (full results available at <a href="http://cgems.cancer.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgems.cancer.gov</a>).</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365843790784" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP ID</th><th rowspan="1" colspan="1" align="left">χ<sup>2</sup><sup>*</sup></th><th rowspan="1" colspan="1" align="left">P<sup>*</sup></th><th rowspan="1" colspan="1" align="left">OR<sub>het</sub><sup>*</sup></th><th rowspan="1" colspan="1" align="left">OR<sub>homo</sub><sup>*</sup></th><th rowspan="1" colspan="1" align="left">Chromosome</th><th rowspan="1" colspan="1" align="left">Gene</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">1. rs10510126</td><td rowspan="1" colspan="1" align="left">25.92</td><td rowspan="1" colspan="1" align="left">0.0000024</td><td rowspan="1" colspan="1" align="left">0.59</td><td rowspan="1" colspan="1" align="left">0.59</td><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">2. rs12505080</td><td rowspan="1" colspan="1" align="left">23.45</td><td rowspan="1" colspan="1" align="left">0.0000081</td><td rowspan="1" colspan="1" align="left">1.22</td><td rowspan="1" colspan="1" align="left">0.51</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;4</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">3. rs17157903</td><td rowspan="1" colspan="1" align="left">23.28</td><td rowspan="1" colspan="1" align="left">0.0000088</td><td rowspan="1" colspan="1" align="left">1.60</td><td rowspan="1" colspan="1" align="left">0.77</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;7</td><td rowspan="1" colspan="1" align="left">RELN</td></tr><tr><td rowspan="1" colspan="1" align="left">4. rs1219648</td><td rowspan="1" colspan="1" align="left">22.63</td><td rowspan="1" colspan="1" align="left">0.000012</td><td rowspan="1" colspan="1" align="left">1.23</td><td rowspan="1" colspan="1" align="left">1.80</td><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left">FGFR2</td></tr><tr><td rowspan="1" colspan="1" align="left">5. rs7696175</td><td rowspan="1" colspan="1" align="left">22.40</td><td rowspan="1" colspan="1" align="left">0.000013</td><td rowspan="1" colspan="1" align="left">1.39</td><td rowspan="1" colspan="1" align="left">0.86</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;4</td><td rowspan="1" colspan="1" align="left">TLR1,TLR6</td></tr><tr><td rowspan="1" colspan="1" align="left">6. rs2420946</td><td rowspan="1" colspan="1" align="left">22.17</td><td rowspan="1" colspan="1" align="left">0.000015</td><td rowspan="1" colspan="1" align="left">1.24</td><td rowspan="1" colspan="1" align="left">1.79</td><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left">FGFR2</td></tr></tbody></table></div><div id="largeobj_idm140365843790784" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3493132/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>From analyses adjusting for age, matching factors (see Methods), and three eigenvectors of the principal components identified by Eigenstrat. P value obtained by a score test with 2df.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P46">Haplotypes for 4 SNPs in intron 2 of <em>FGFR2</em> (rs11200014, rs2981579, rs1219648 rs2420946) and breast cancer risk, in the individual studies and pooled across studies.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365849002496" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="7" rowspan="1" align="left">Nurses Health Study</th></tr><tr><th rowspan="1" colspan="1" align="left">Haplotype</th><th rowspan="1" colspan="1" align="right">cases</th><th rowspan="1" colspan="1" align="right">controls</th><th rowspan="1" colspan="1" align="right">OR</th><th rowspan="1" colspan="1" align="right">95%</th><th rowspan="1" colspan="1" align="right">CI</th><th rowspan="1" colspan="1" align="right">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">G-G-A-C</td><td rowspan="1" colspan="1" align="right">1195</td><td rowspan="1" colspan="1" align="right">1348</td><td rowspan="1" colspan="1" align="right">1.0</td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-T</td><td rowspan="1" colspan="1" align="right">998</td><td rowspan="1" colspan="1" align="right">842</td><td rowspan="1" colspan="1" align="right">1.33</td><td rowspan="1" colspan="1" align="right">1.18</td><td rowspan="1" colspan="1" align="right">1.50</td><td rowspan="1" colspan="1" align="right">3×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-A-C</td><td rowspan="1" colspan="1" align="right">40</td><td rowspan="1" colspan="1" align="right">47</td><td rowspan="1" colspan="1" align="right">0.96</td><td rowspan="1" colspan="1" align="right">0.62</td><td rowspan="1" colspan="1" align="right">1.48</td><td rowspan="1" colspan="1" align="right">0.85</td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-C</td><td rowspan="1" colspan="1" align="right">24</td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="right">1.40</td><td rowspan="1" colspan="1" align="right">0.76</td><td rowspan="1" colspan="1" align="right">2.57</td><td rowspan="1" colspan="1" align="right">0.28</td></tr><tr><td rowspan="1" colspan="1" align="left">Rare &lt;1%</td><td rowspan="1" colspan="1" align="right">33</td><td rowspan="1" colspan="1" align="right">28</td><td rowspan="1" colspan="1" align="right">1.36</td><td rowspan="1" colspan="1" align="right">0.81</td><td rowspan="1" colspan="1" align="right">2.29</td><td rowspan="1" colspan="1" align="right">0.24</td></tr></tbody></table></div><div id="largeobj_idm140365849002496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3493132/table/T3/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365848425312" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="7" rowspan="1" align="left">Nurses Health Study 2</th></tr><tr><th rowspan="1" colspan="1" align="left">Haplotype</th><th rowspan="1" colspan="1" align="right">cases</th><th rowspan="1" colspan="1" align="right">controls</th><th rowspan="1" colspan="1" align="right">OR</th><th rowspan="1" colspan="1" align="right">95%</th><th rowspan="1" colspan="1" align="right">CI</th><th rowspan="1" colspan="1" align="right">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">G-G-A-C</td><td rowspan="1" colspan="1" align="right">295</td><td rowspan="1" colspan="1" align="right">667</td><td rowspan="1" colspan="1" align="right">1.0</td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-T</td><td rowspan="1" colspan="1" align="right">276</td><td rowspan="1" colspan="1" align="right">474</td><td rowspan="1" colspan="1" align="right">1.34</td><td rowspan="1" colspan="1" align="right">1.09</td><td rowspan="1" colspan="1" align="right">1.65</td><td rowspan="1" colspan="1" align="right">0.0061</td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-A-C</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="right">0.76</td><td rowspan="1" colspan="1" align="right">0.30</td><td rowspan="1" colspan="1" align="right">1.90</td><td rowspan="1" colspan="1" align="right">0.55</td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-C</td><td rowspan="1" colspan="1" align="right">13</td><td rowspan="1" colspan="1" align="right">20</td><td rowspan="1" colspan="1" align="right">1.50</td><td rowspan="1" colspan="1" align="right">0.70</td><td rowspan="1" colspan="1" align="right">3.21</td><td rowspan="1" colspan="1" align="right">0.30</td></tr><tr><td rowspan="1" colspan="1" align="left">Rare &lt;1%</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="right">1.96</td><td rowspan="1" colspan="1" align="right">0.90</td><td rowspan="1" colspan="1" align="right">4.28</td><td rowspan="1" colspan="1" align="right">0.09</td></tr></tbody></table></div><div id="largeobj_idm140365848425312" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3493132/table/T3/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365843836000" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="7" rowspan="1" align="left">PLCO Study</th></tr><tr><th rowspan="1" colspan="1" align="left">Haplotype</th><th rowspan="1" colspan="1" align="right">cases</th><th rowspan="1" colspan="1" align="right">controls</th><th rowspan="1" colspan="1" align="right">OR</th><th rowspan="1" colspan="1" align="right">95%</th><th rowspan="1" colspan="1" align="right">CI</th><th rowspan="1" colspan="1" align="right">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">G-G-A-C</td><td rowspan="1" colspan="1" align="right">994</td><td rowspan="1" colspan="1" align="right">1140</td><td rowspan="1" colspan="1" align="right">1.0</td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-T</td><td rowspan="1" colspan="1" align="right">795</td><td rowspan="1" colspan="1" align="right">728</td><td rowspan="1" colspan="1" align="right">1.13</td><td rowspan="1" colspan="1" align="right">0.99</td><td rowspan="1" colspan="1" align="right">1.29</td><td rowspan="1" colspan="1" align="right">0.064</td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-A-C</td><td rowspan="1" colspan="1" align="right">32</td><td rowspan="1" colspan="1" align="right">24</td><td rowspan="1" colspan="1" align="right">1.44</td><td rowspan="1" colspan="1" align="right">0.83</td><td rowspan="1" colspan="1" align="right">2.47</td><td rowspan="1" colspan="1" align="right">0.19</td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-C</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">24</td><td rowspan="1" colspan="1" align="right">0.47</td><td rowspan="1" colspan="1" align="right">0.23</td><td rowspan="1" colspan="1" align="right">0.97</td><td rowspan="1" colspan="1" align="right">0.041</td></tr><tr><td rowspan="1" colspan="1" align="left">Rare &lt;1%</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="right">34</td><td rowspan="1" colspan="1" align="right">0.63</td><td rowspan="1" colspan="1" align="right">0.36</td><td rowspan="1" colspan="1" align="right">1.10</td><td rowspan="1" colspan="1" align="right">0.11</td></tr></tbody></table></div><div id="largeobj_idm140365843836000" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3493132/table/T3/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365849299008" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="7" rowspan="1" align="left">ACS CPS-II</th></tr><tr><th rowspan="1" colspan="1" align="left">Haplotype</th><th rowspan="1" colspan="1" align="right">cases</th><th rowspan="1" colspan="1" align="right">controls</th><th rowspan="1" colspan="1" align="right">OR</th><th rowspan="1" colspan="1" align="right">95%</th><th rowspan="1" colspan="1" align="right">CI</th><th rowspan="1" colspan="1" align="right">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">G-G-A-C</td><td rowspan="1" colspan="1" align="right">583</td><td rowspan="1" colspan="1" align="right">664</td><td rowspan="1" colspan="1" align="right">1.0</td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-T</td><td rowspan="1" colspan="1" align="right">482</td><td rowspan="1" colspan="1" align="right">406</td><td rowspan="1" colspan="1" align="right">1.38</td><td rowspan="1" colspan="1" align="right">1.16</td><td rowspan="1" colspan="1" align="right">1.65</td><td rowspan="1" colspan="1" align="right">0.00040</td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-A-C</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="right">22</td><td rowspan="1" colspan="1" align="right">1.07</td><td rowspan="1" colspan="1" align="right">0.59</td><td rowspan="1" colspan="1" align="right">1.94</td><td rowspan="1" colspan="1" align="right">0.82</td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-C</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">0.67</td><td rowspan="1" colspan="1" align="right">0.23</td><td rowspan="1" colspan="1" align="right">1.89</td><td rowspan="1" colspan="1" align="right">0.45</td></tr><tr><td rowspan="1" colspan="1" align="left">Rare &lt;1%</td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">1.98</td><td rowspan="1" colspan="1" align="right">0.91</td><td rowspan="1" colspan="1" align="right">4.31</td><td rowspan="1" colspan="1" align="right">0.084</td></tr></tbody></table></div><div id="largeobj_idm140365849299008" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3493132/table/T3/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365850416448" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="7" rowspan="1" align="left">Pooled across studies</th></tr><tr><th rowspan="1" colspan="1" align="left">haplotype</th><th rowspan="1" colspan="1" align="right">cases</th><th rowspan="1" colspan="1" align="right">controls</th><th rowspan="1" colspan="1" align="right">OR</th><th rowspan="1" colspan="1" align="right">95%</th><th rowspan="1" colspan="1" align="right">CI</th><th rowspan="1" colspan="1" align="right">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">G-G-A-C</td><td rowspan="1" colspan="1" align="right">3068</td><td rowspan="1" colspan="1" align="right">3718</td><td rowspan="1" colspan="1" align="right">1.0</td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-T</td><td rowspan="1" colspan="1" align="right">2551</td><td rowspan="1" colspan="1" align="right">2450</td><td rowspan="1" colspan="1" align="right">1.26</td><td rowspan="1" colspan="1" align="right">1.17</td><td rowspan="1" colspan="1" align="right">1.35</td><td rowspan="1" colspan="1" align="right">6×10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-A-C</td><td rowspan="1" colspan="1" align="right">102</td><td rowspan="1" colspan="1" align="right">114</td><td rowspan="1" colspan="1" align="right">1.09</td><td rowspan="1" colspan="1" align="right">0.83</td><td rowspan="1" colspan="1" align="right">1.43</td><td rowspan="1" colspan="1" align="right">0.55</td></tr><tr><td rowspan="1" colspan="1" align="left">A-A-G-C</td><td rowspan="1" colspan="1" align="right">54</td><td rowspan="1" colspan="1" align="right">74</td><td rowspan="1" colspan="1" align="right">0.88</td><td rowspan="1" colspan="1" align="right">0.61</td><td rowspan="1" colspan="1" align="right">1.27</td><td rowspan="1" colspan="1" align="right">0.50</td></tr><tr><td rowspan="1" colspan="1" align="left">Rare &lt;1%</td><td rowspan="1" colspan="1" align="right">107</td><td rowspan="1" colspan="1" align="right">151</td><td rowspan="1" colspan="1" align="right">0.87</td><td rowspan="1" colspan="1" align="right">0.68</td><td rowspan="1" colspan="1" align="right">1.12</td><td rowspan="1" colspan="1" align="right">0.28</td></tr></tbody></table></div><div id="largeobj_idm140365850416448" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3493132/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P6">To further explore the association signal observed on <em>FGFR2</em> in the NHS, we performed analyses using inferred ancestral recombination graphs (ARGs). This involves estimating a simple approximation to the distribution of possible genealogies relating the haplotypes of the cases and controls. Using the Margarita program<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980708">10</a></sup>, we inferred ARGs for 81-SNP haplotypes spanning the <em>FGFR2</em> gene and its flanking regions (from position 123225862 to position 123471190 on NCBI build35, as shown in Supplemental Figure 3). For every ARG, a putative risk mutation was placed on the marginal genealogy at each SNP position by maximizing the association between the mutation and disease status. We evaluated the significance of this observed association using a maximum of 10<sup>6</sup> permutations on the phenotypes. Supplemental Figure 3 shows that the permutation p-value was consistently higher than 0.05 over the entire region with the exception of the 20 Kb segment located between 123.32 Mb and 123.34 Mb in intron 2 of <em>FGFR2</em>. In this segment, all marginal trees demonstrated association with a P value lower than 3 ×10<sup>−3</sup>. The permutation p-values for four notable SNPs were all smaller than 2×10−5, and the frequency of the inferred mutation was similar for all four SNPs. (Supplemental Table 1). These results suggest that there is a single risk locus in the <em>FGFR2</em> region.</p><p id="P7">For the six most significant SNPs in the GWAS (two in <em>FGFR2</em>, four at other loci), and for the two additional SNPs that appeared to define the <em>FGFR2</em> risk haplotype, we attempted to replicate the initial associations in the GWAS in an additional 1,776 cases and 2,072 controls from breast cancer case-control studies nested in three prospective cohorts: the Nurses’ Health Study 2 (NHS2), the Prostate, Lung, Colorectal, and Ovary Cancer Screening Trial (PLCO) Cohort, and the American Cancer Society Cancer Prevention Study-II (CPS-II) (<a href="/pmc/articles/PMC3493132/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). For the SNP in <em>FGFR2</em> most strongly associated with breast cancer in the GWAS (rs1219648), the pooled P value across all four studies = 4.2 × 10<sup>−10</sup> for the 2 d.f. model, and 1.1 × 10<sup>−10</sup> for the Cochran-Armitage test for trend. Both P values are lower than a threshold for genome-wide significance based on the conservative Bonferroni correction for 528,173 tests with a nominal α=0.05. There was no statistical evidence of heterogeneity of the genotype-specific odds ratios across the studies. The pooled estimate of the odds ratios across studies, compared to wild-type homozygotes, was 1.20 (95% CI 1.07 – 1.34) for heterozygotes, and 1.64 (95%CI 1.42–1.90) for homozygote variants. Across the four studies the SNP with the strongest association (rs1219648) was associated with a Population Attributable Risk (PAR) of 16%<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980703">11</a></sup>. In each of the three replication studies, and in all studies combined, the AAGT haplotype was the only common haplotype significantly associated with risk of breast cancer (<a href="/pmc/articles/PMC3493132/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The associations were not significantly different across categories of age at diagnosis of breast cancer. In the NHS2, a study involving mainly premenopausal women, the associations were equivalent to those in the other three studies comprised of postmenopausal cases. None of the 4 SNPs at other chromosomal loci was associated with increased risk in the pooled replication studies.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P45">Association of rs1219648 in the Nurses Health Study, Nurses’ Health Study 2, the PLCO study, the ACS CPS-II study, and pooled across studies.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365843390624" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="left">Study Population<br>(N cases/N controls)</th><th colspan="2" rowspan="1" valign="top" align="left">Allele Frequency</th><th rowspan="2" colspan="1" valign="top" align="left">OR<sub>het</sub><br>(95% CI)</th><th rowspan="2" colspan="1" valign="top" align="left">OR<sub>homo</sub><br>(95% CI)</th><th rowspan="2" colspan="1" valign="top" align="left">P<sub>trend</sub></th></tr><tr><th rowspan="1" colspan="1" align="left">Cases<br>(%)</th><th rowspan="1" colspan="1" align="left">Controls<br>(%)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">NHS (1,145/1,141)</td><td rowspan="1" colspan="1" valign="top" align="left">45.54</td><td rowspan="1" colspan="1" valign="top" align="left">38.47</td><td rowspan="1" colspan="1" valign="top" align="left">1.24<br>(1.04–1.50)</td><td rowspan="1" colspan="1" valign="top" align="left">1.81<br>(1.43–2.31)</td><td rowspan="1" colspan="1" valign="top" align="left">2.0 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">NHS2 (302/594)</td><td rowspan="1" colspan="1" valign="top" align="left">48.18</td><td rowspan="1" colspan="1" valign="top" align="left">40.57</td><td rowspan="1" colspan="1" valign="top" align="left">1.29<br>(0.95–1.75)</td><td rowspan="1" colspan="1" valign="top" align="left">1.93<br>(1.31–2.86)</td><td rowspan="1" colspan="1" valign="top" align="left">0.002</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">PLCO (919/922)</td><td rowspan="1" colspan="1" valign="top" align="left">44.50</td><td rowspan="1" colspan="1" valign="top" align="left">41.49</td><td rowspan="1" colspan="1" valign="top" align="left">1.06<br>(0.86–1.30)</td><td rowspan="1" colspan="1" valign="top" align="left">1.22<br>(0.94–1.58)</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ACS CPS-II (555/556)</td><td rowspan="1" colspan="1" valign="top" align="left">44.95</td><td rowspan="1" colspan="1" valign="top" align="left">37.41</td><td rowspan="1" colspan="1" valign="top" align="left">1.32<br>(1.02–1.72)</td><td rowspan="1" colspan="1" valign="top" align="left">2.06<br>(1.42–2.97)</td><td rowspan="1" colspan="1" valign="top" align="left">0.0002</td></tr><tr><td colspan="2" rowspan="1" valign="top" align="left">Pooled estimates (2,921/3,213)</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">1.20<br>(1.07–1.34)</td><td rowspan="1" colspan="1" valign="top" align="left">1.64<br>(1.42–1.90)</td><td rowspan="1" colspan="1" valign="top" align="left">1.1 × 10<sup>−10</sup></td></tr></tbody></table></div><div id="largeobj_idm140365843390624" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3493132/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P8"><em>FGFR2</em> is a tumor suppressor gene that is amplified and over-expressed in breast cancer<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980709">9</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980720">12</a></sup>. Furthermore, alternatively spliced variants of this gene result in differential signal transduction and transformation of mammary epithelial cell lines. Further work is needed to identify the causal variant at this locus.</p><p id="P9">In a large, three-stage GWAS of breast cancer using the Perlegen platform as the initial genome scan, Easton et al. identified SNPs in <em>FGFR2</em> as the strongest of their reported associations<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373896174">6</a></sup>. These SNPs were also in intron 2; the association was originally detected with rs2981582, which has an r<sup>2</sup> of 1.0 with rs1219648 and rs2420946, 0.97 with rs2981579, and 0.96 with rs11200014 in the HapMap CEU samples, which indicates that we have detected essentially the same association. Substantial resequencing in intron 2 did not discover any SNPs with a more obvious probability of being functionally related to breast cancer risk<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373896175">6</a></sup>. Easton et al. used genotypes from 390 breast cancer cases under age 60, selected to have a strong family history or bilaterality of breast cancer. Our study focused on later-onset, “sporadic” cases. Such consecutive series of cases may be more easily obtained for many diseases, and more generalizable to the most common forms of the disease, so it is reassuring that the use of unselected cases resulted in the identification of the same principal locus.</p><p id="P10" class="p">We focused on the most highly statistically significant associations from our GWAS, identifying variants in FGFR2 as reproducibly associated with breast cancer. Since a subset of true associations would be weakly associated with outcome in any given GWAS, large-scale replication is necessary for confirmation, and some true associations may be missed if they are not carried forward into replication studies<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup>. Multi-stage designs in which potentially associated SNPs from the first stage are carried into additional studies are an economical and scientifically sound approach to cope with the present cost of high throughput genotyping<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980716">14</a></sup>. In this regard, the precomputed rankings and P values for all the SNPs included in the GWAS conducted in the NHS are freely available from our website (<a href="http://cgems.cancer.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgems.cancer.gov</a>) for others to use in subsequent studies of women with breast cancer.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Nurses’ Health Study nested case-control study</h3><p id="P11" class="p p-first">The Nurses’ Health Study (NHS) is a longitudinal study of 121,700 women enrolled in 1976. The CGEMS nested case-control study is derived from 32,826 participants who provided a blood sample between 1989 and 1990 and were free of diagnosed breast cancer at blood collection and followed for incident disease until June 1, 2004. Cancer follow-up in the NHS was conducted by personal mailings and searches of the National Death Index. It is estimated that the percentage of true cancers in the cohort captured by this system is greater than 98%<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980717">15</a></sup>. Permission was requested from all participants diagnosed with cancer to review medical records to confirm the diagnoses and obtain additional information on tumor histology, staging, and other characteristics. All study participants who were menopausal at blood draw with a confirmed diagnosis of invasive breast cancer and had sufficient blood sample available for DNA extraction at the time of case and control selection were included as cases in the CGEMS project. Controls were not diagnosed with breast cancer during follow-up, and were matched to cases based on age at diagnosis, blood collection variables (time of day, season, and year of blood collection, as well as recent (&lt;3 months) use of postmenopausal hormones), ethnicity (all cases and controls are self-reported Caucasians), and menopausal status (all cases were postmenopausal at diagnosis). Menopausal status was defined according to self-reported information on whether a woman’s regular periods had ceased<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980715">16</a></sup>.</p><p id="P12" class="p p-last">Informed consent was obtained from all participants. The study was approved by the Institutional Review Board of the Brigham and Women’s Hospital, Boston, MA, USA.</p></div><div id="S3" class="sec"><h3 id="S3title">Nurses’ Health Study II</h3><p id="P13" class="p p-first-last">The NHS2 began in 1989 when 116,671 female registered nurses (aged 25 to 42) from 14 U.S. states were enrolled<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980706">17</a></sup>. Blood samples were collected from 29,611 NHS2 participants in 1996–1999. Between blood collection and June 1, 2003 incident breast cancer cases (n=317) were identified via self-report, and confirmed by medical records. To increase power two controls for each case (n=634) were then selected for each case matched case based on year of follow-up, month and year of blood draw, and fasting status at blood draw, as well as age, menopausal status, and ethnicity. Pre-menopausal cases were additionally matched to controls based on luteal day (days before start of next menstrual cycle) to permit analyses of plasma hormone levels.</p></div><div id="S4" class="sec"><h3 id="S4title">PLCO Breast Cancer Study</h3><p id="P14" class="p p-first-last">The Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial (PLCO), a randomized two-arm trial to determine if screening reduced the mortality from these cancers, enrolled during 1993–2001 155,000 men and women, aged 55–74, in 10 U.S. centers<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303859307">18</a></sup>. At study entry, demographic, medical, and lifestyle information and a blood sample were requested from the subjects assigned to the screening arm, including 39,000 women. Incident cancers are ascertained by annual mailed questionnaires (94% completion rate for living Trial participants), as well as searches of the National Death Index. For all reported cancers, medical records were requested from the appropriate hospitals to confirm diagnoses. Of the 37,800 women completing the baseline questionnaire and at least one follow up questionnaire, 83% also provided a blood sample and informed consent. Included in this nested case-control study of breast cancer are all 927 non-Hispanic white women with incident breast cancer reported by June 30, 2005, no previous history of breast cancer, a baseline questionnaire and blood sample, and informed consent. Also selected for this study were 927 non-Hispanic white controls, frequency matched to cases on age at randomization (5-year categories) and year of randomization (two categories), with no report of breast cancer as of June 30, 2005, a baseline questionnaire and blood sample, and informed consent. Because the youngest women in the cohort were 55 y at entry, practically all of the breast cancers diagnosed were postmenopausal.</p></div><div id="S5" class="sec"><h3 id="S5title">The American Cancer Society Cancer Prevention Study II Nutrition Cohort (CPS-II)</h3><p id="P15" class="p p-first">The American Cancer Society Cancer Prevention Study II Nutrition Cohort (CPS-II) was established in 1992; the cohort includes over 86,000 men and 97,000 women from 21 U.S. states who completed a mailed questionnaire in 1992<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980722">19</a></sup>. At baseline, the cohort was 97% white and the median age of participants was 63 (range: 40–92). Starting in 1997, follow-up questionnaires have been sent to surviving cohort members every other year to update exposure information and to ascertain occurrence of new cases of cancer; a &gt;90% response rate has been achieved for each follow-up questionnaire. Incident cancers are verified through medical records, state cancer registries, or death certificates. From 1998 – 2001, blood samples were collected from a subgroup of 39,376 cohort members.</p><p id="P16" class="p p-last">Cases of postmenopausal breast cancer were frequency matched to controls on single year of age, ethnicity, date of sample collection and menopausal status (all postmenopausal). All controls were selected from individuals who were cancer-free (except for non-melanoma skin cancer) at the beginning of the interval preceding the diagnosis of each case. Similarly, cases were not eligible if they had a history of another cancer other than non-melanoma skin cancer prior to their diagnosis of breast cancer. A total of 555 non-Hispanic white cases of invasive breast cancer and 556 non-Hispanic white controls were included in this analysis.</p></div><div id="S6" class="sec"><h3 id="S6title">Genotyping and quality control for NHS</h3><p id="P17" class="p p-first">DNA samples were received from the NHS biorepository and visually inspected for adequate fluid in individual tubes. Three measurements of quantification were performed according to the standard procedures at the Core Genotyping Facility of the National Cancer Institute<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303859308">7</a></sup>. These include pico-green analysis, optical density spectrophotometry and real time PCR (<a href="http://cgf.nci.nih.gov/dnaquant.cfm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgf.nci.nih.gov/dnaquant.cfm</a>). Samples were also analyzed for 15 short tandem repeats and the Amelogenin marker in the Identifiler™ Assay (ABI, Foster City, CA). All samples advanced to genotype analysis completed no less than 13 of the 15 micro-satellite markers.</p><p id="P18">After final review and sample handling, a total of 1188 cases DNAs, and 1183 controls DNAs were selected for genotyping in CGEMS. Ninety three DNAs were aliquoted twice and five DNAs were aliquoted three times, resulting in the addition of 103 redundant DNAs from the NHS used for quality control. Finally, 23 external QC control DNAs were added. Thus, genotyping was attempted on a total of 2,494 DNA samples.</p><p id="P19" class="p p-last">Genotyping of the CGEMS Breast Cancer Study was performed at the NCI Core Genotyping Facility using the Sentrix® HumanHap550 genotyping assay according to a protocol designated by the manufacturer.</p></div><div id="S7" class="sec"><h3 id="S7title">Initial Assessment of sample completion rates</h3><p id="P20" class="p p-first-last">A total of 555,352 SNP genotype assays were attempted on the 2,494 DNA samples using the Illumina HumanHap550 chip. Whenever the completion rate for a sample was below 90%, the sample was assayed a second time. Samples that did not meet the 90% completion threshold after a second attempt were excluded from further analysis. Fifty-nine samples from NHS (30 cases and 29 controls) were excluded from further analysis based on these criteria, which left 2,435 DNAs for the subsequent analyses</p></div><div id="S8" class="sec"><h3 id="S8title">Assessment of SNP call rates</h3><p id="P21" class="p p-first-last">A total of 8,706 SNPs (~1.57% overall) failed to provide accurate genotype results due to either no calls or low call rates (&lt;90%). Further quality control analysis was performed on the remaining 546,646 SNPs. An additional 18,473 SNPs with an observed low MAF (&lt;1%) were dropped from the association analysis; thus 528,173 SNPs (95.1%), were maintained in the subsequent analyses.</p></div><div id="S9" class="sec"><h3 id="S9title">Summary of completion rate for NHS samples</h3><p id="P22" class="p p-first-last">
<!--table ft1--><!--table-wrap mode="anchored" t5--></p><div class="table-wrap table anchored whole_rhythm" id="T4"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140365843447536" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Sample Completion rate for</th><th rowspan="1" colspan="1" align="left">528,173 SNPs<br>(retained)</th><th rowspan="1" colspan="1" align="left">546,646 SNPs<br>(attempted)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Scan 1 study</td><td rowspan="1" colspan="1" align="left">99.754 %</td><td rowspan="1" colspan="1" align="left">95.754 %</td></tr><tr><td rowspan="1" colspan="1" align="left">Scan 1 case</td><td rowspan="1" colspan="1" align="left">99.756 %</td><td rowspan="1" colspan="1" align="left">95.704 %</td></tr><tr><td rowspan="1" colspan="1" align="left">Scan1 control</td><td rowspan="1" colspan="1" align="left">99.773 %</td><td rowspan="1" colspan="1" align="left">95.799 %</td></tr></tbody></table></div><div id="largeobj_idm140365843447536" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3493132/table/T4/?report=objectonly">Open in a separate window</a></div></div><p>
The genotyping of the 516,383 SNPs with high call rate on the 2,412 NHS DNAs with high completion rate generated 1.26 billion genotype calls. For this set of SNPs and samples, the percentage of missing data was less than 1%.</p></div><div id="S10" class="sec"><h3 id="S10title">Concordance rate</h3><p id="P23" class="p p-first-last">The genotype concordance rate for SNP assays was evaluated using the 93 pairs of known NHS duplicated DNAs. These pairs of DNAs were separate aliquots from the same DNA preparation; all met quality control criteria required for the other DNAs, thereby, providing reliable data for comparison. Analysis of the discrepancies within these pairs of DNA revealed similar results to the CEPH DNA duplicates reported in the prostate cancer CGEMS genome-wide association scan. An average concordance rate of 99.985% was observed (50,820,003 concordant genotype calls out of 50,827,468 comparisons). No SNP or DNA was removed from the search for association as a result of this analysis.</p></div><div id="S11" class="sec"><h3 id="S11title">Deviation from Hardy–Weinberg proportions in control DNAs</h3><p id="P24" class="p p-first-last">Genotype data for all autosomal SNPs were tested for deviation from Hardy-Weinberg proportions. The analysis was conducted in the NHS control group. Significant deviations were observed for 28,710 SNPs (5.57% of 515,302 SNPs) at the level of P = 0.005 and for 2843 SNPs (0.55%) at P = 0.001. However, none of these SNPs were excluded from analysis since the tests for association applied to such data are valid in the presence of departure from Hardy-Weinberg proportions, although with potentially reduced power when these deviations are due to systematic genotyping errors with equal effects among cases and controls.</p></div><div id="S12" class="sec"><h3 id="S12title">Final sample selection for association analysis</h3><p id="P25" class="p p-first-last">For all DNAs the frequency of heterozygote loci on the X chromosome was compatible with a female origin. Eighteen DNA samples (5 cases, 13 controls) had unclear identity as they could not be mapped back unambiguously to previous genotype results from these samples and were excluded. Subsequent inspection of the genotype concordance rate between pairs of DNAs did not disclose unexpected duplicates. Finally, based on two analyses with two independent sets of 7,050 and 7,061 randomly selected SNPs with very low linkage disequilibrium (r<sup>2</sup>&lt;0.01) using the STRUCTURE program<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980719">20</a></sup>. Four subjects (3 cases, 1 control) were estimated to be of admixed origin with greater than 15% of either Asian or West African ancestry (described in detail below). These 4 subjects were also removed from subsequent analyses. Thus, the search for associations was performed on a final set of 2,287 unique subjects including 1,145 cases and 1,142 controls.</p></div><div id="S13" class="sec"><h3 id="S13title">Statistical analysis</h3><p id="P26" class="p p-first">For the initial scan in the Nurses’ Health Study, we analyzed each locus using logistic regression. When the rare homozygote genotype was observed more than fifteen times, we regressed disease status on indicator variables for heterozygous and homozygote carriers of the variant allele. Otherwise, the rare homozygote genotype was collapsed with the heterozygote genotype, and we regressed disease status on variant-allele carrier status. This aggregation of genotypes was performed for 64,589 SNPs.</p><p id="P27">We calculated unadjusted score tests for genetic association as well as two adjusted score tests. The first adjusted test controlled for age categories (ages &lt;55, 55–59, 60–64, 65–69, 70–74, and &gt;74) and hormone replacement therapy use. The second controlled for age, hormone replacement therapy use, and three eigenvectors from the principal component analysis of genetic covariance (see below). The latter were included in the logistic regression as continuous covariates.</p><p id="P28">The importance of a second SNP conditional on a given SNP was assessed using nested likelihood ratio tests, comparing the logistic regression model with genotypic indicator variables for the first SNP only to the model with indicator variables for each multi-locus genotype. None of the four FGFR2 SNPs showed any evidence of association with risk of breast cancer after adjusting for any of the other three FGFR2 SNPs.</p><p id="P29" class="p p-last">For the four FGFR2 SNPs, haplotype frequencies and expected haplotype counts for each individual were estimated using a simple expectation-maximization algorithm (implemented in SAS PROC HAPLOTYPE). Haplotype association analyses were performed using the expectation-substitution technique<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980707">21</a></sup>.</p></div><div id="S14" class="sec"><h3 id="S14title">Assessment of population stratification</h3><p id="P30" class="p p-first">Two independent sets of 7,050 and 7,061 SNPs with very low linkage disequilibrium (r<sup>2</sup>&lt;0.01) were analyzed using the STRUCTURE<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980710">20</a></sup> program to determine if subjects have an admixed origin greater than 15% of either Asian or West African ancestry (based on HapMap II data)<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980713">22</a></sup></p><p id="P31" class="p p-last">The pooled case and control DNAs were analyzed using a set of 14,111 SNPs with very low pair-wise linkage disequilibrium (r2 &lt; 0.01) using the procedure described by Price et al.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980712">23</a></sup>. Testing for significance using the Tracy-Widom statistics<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303859310">24</a></sup> revealed 4 significant principal components at the level of p &lt; 0.05. Inspection of the distribution of the DNAs in the space defined by these components revealed little difference between cases and controls. Nevertheless, statistically borderline significant differences in this distribution for local groups observed in the space defined by the first three components led us to retain these components in the statistical analysis. No difference was observed with the 4th and higher components, which were not retained in the analysis.</p></div><div id="S15" class="sec"><h3 id="S15title">Genotyping in replication studies</h3><p id="P32" class="p p-first-last">The same TaqMan assays developed for rs10510126, rs1219648, rs17157903, rs2420946, rs7696175, rs12505080, rs11200014 and rs2981579 were performed at the NCI Core Genotyping Facility and at the DF/HCC Polymorphism Detection Core (Primers and probe sequences available on request).</p></div><div id="S16" class="sec"><h3 id="S16title">Statistical analysis of replication studies</h3><p id="P33" class="p p-first-last">For the individual replication studies, we used unconditional logistic regression to fit codominant and additive genetic risk models. For pooled analyses of multiple studies, we used unconditional logistic regression with separate baseline odds parameters for each study. We also adjusted for age in five year intervals. Effect modification by age at diagnosis (comparing 65+ years versus &lt;65, or 55+ versus &lt;55) and by menopausal status at diagnosis was assessed using nested logistic regression models. For these analyses, controls' ages were set to the ages at diagnosis of the matched cases (NHS, NHS2, ACS) or age at censoring (PLCO). We calculated Population Attributable Risk (PAR) using the method of Bruzzi et al<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278980702">11</a></sup>.</p></div><div id="S17" class="sec"><h3 id="S17title">ARG analysis with 81 SNPs after 10<sup>6</sup> Permutations</h3><p id="P34" class="p p-first-last">In order to identify the at-risk haplotypes, the most likely pair of haplotypes present in each case and control was inferred using PHASE and this information was used to generate one hundred ARG. On each ARG, the position of the putative functional mutation that best explains the distribution of cases and controls identifies the haplotypes that harbor the new functional allele. This allele is declared at-risk if its frequency is higher in the cases than in the controls. The new allele was observed deleterious in 46% of the ARG and therefore this analysis was not conclusive. The ARG analysis was however more successful in reproducibly differentiating between the at-risk and protective haplotypes. (Supplemental Figure 3).</p></div><div id="S18" class="sec sec-last"><h3 id="S18title">Identification of protective and at-risk haplotypes for the <em>FGFR2</em> susceptibility locus</h3><p id="P35" class="p p-first-last">The most likely pair of haplotypes present in each case and control of the NHS was inferred using PHASE and an ARG analysis was performed to infer the presence of either the protective or of the at-risk allele for each haplotype. For marginal trees at all 4 positions with the lowest permutation p values, the same four of 8 haplotypes were inferred with a probability higher than 97% to carry the at-risk allele (Supplementary Table 1). Similarly, three haplotypes were predicted to carry the protective allele with the same confidence. These two groups of haplotypes systematically differ at 4 of the 9 SNPs positions. One rare haplotype, haplotype-8, with a population frequency of 2%, possibly results from a recombination between one haplotype of each group. Its centromeric region contains 2 alleles specific to the protective group of haplotypes as its telomeric region contains 2 alleles specific to the at-risk group of haplotypes. Unfortunately, the ARG analysis is unable to ambiguously determine whether this rare haplotype carries the at-risk or the protective allele. We note however that haplotype-8 may correspond to haplotype 2111111 of supplementary table 3 of Easton et al<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373896173">6</a></sup>.The latter is shown to be associated with a lower risk of breast cancer. Confirmation of the status of this haplotype would suggest that the functional polymorphisms should lie centromeric (i.e. on the 3’side) to rs12119648. Results of the ARG analysis are shown for all haplotypes with frequencies higher than 1% defined by the following 9 contiguous SNPs : rs3750817, rs11200014, rs2981579 ,rs17542768, rs1219648, rs1219643, rs17102287, rs2420946, rs1047111. The alleles are color coded : red is the ancestral allele, blue is the new allele and black is when this information is unclear. The probability of a haplotype to carry the at-risk allele is evaluated as the ratio of the number of times it is predicted to carry the at-risk allele over the total number of prediction made for this haplotypes for all marginal trees at the indicated position.</p></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="P36">The Nurses’ Health Studies are supported by NIH grants CA 65725, CA87969, CA49449, CA67262, CA50385 and 5UO1CA098233. The authors thank Barbara Egan, Lori Egan, Helena Judge Ellis, Hardeep Ranu, and Pati Soule for assistance, and the participants in the Nurses’ Health Studies.</p><p id="P37">The PLCO study is supported by the Intramural Research Program of the Division of Cancer Epidemiology and Genetics and contracts from the Division of Cancer Prevention, National Cancer Institute, NIH, DHHS. The authors thank Dr Philip Prorok, Division of Cancer Prevention, National Cancer Institute; the Screening Center investigators and staff of the Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial (PLCO); Mr. Tom Riley, Mr. Craig Williams, and staff, Information Management Services, Inc.; Ms. Barbara O’Brien and staff, Westat, Inc.; and Dr. Bill Kopp, Mr. Tim Sheehy, and staff, SAIC-Frederick. Most importantly, we acknowledge the study participants for their contributions to making this study possible.</p><p id="P38">The ACS study is supported by UO1 CA098710. We thank Cari Lichtman for data management and the participants on the CPS-II.</p><p id="P39">We thank Mark Minichiello for providing the Margarita program and fruitful discussions.</p></div></div><div id="idm140365845892656" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140365845892656title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Pharoah PD, et al.  Polygenic susceptibility to breast cancer and implications for prevention. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">31</span>:33–36.</span> [<a href="/pubmed/11984562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Polygenic+susceptibility+to+breast+cancer+and+implications+for+prevention&amp;author=PD+Pharoah&amp;volume=31&amp;publication_year=2002&amp;pages=33-36&amp;pmid=11984562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Antoniou A, et al.  Average risks of breast and ovarian cancer associated with BRCA1 or BRCA2 mutations detected in case Series unselected for family history: a combined analysis of 22 studies. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">72</span>:1117–1130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180265/">PMC free article</a>]</span> [<a href="/pubmed/12677558" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Average+risks+of+breast+and+ovarian+cancer+associated+with+BRCA1+or+BRCA2+mutations+detected+in+case+Series+unselected+for+family+history:+a+combined+analysis+of+22+studies&amp;author=A+Antoniou&amp;volume=72&amp;publication_year=2003&amp;pages=1117-1130&amp;pmid=12677558&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Risch HA, et al.  Population BRCA1 and BRCA2 mutation frequencies and cancer penetrances: a kin-cohort study in Ontario, Canada. <span><span class="ref-journal">J Natl Cancer Inst. </span>2006;<span class="ref-vol">98</span>:1694–1706.</span> [<a href="/pubmed/17148771" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Population+BRCA1+and+BRCA2+mutation+frequencies+and+cancer+penetrances:+a+kin-cohort+study+in+Ontario,+Canada&amp;author=HA+Risch&amp;volume=98&amp;publication_year=2006&amp;pages=1694-1706&amp;pmid=17148771&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Hirschhorn JN, Daly MJ. Genome-wide association studies for common diseases and complex traits. <span><span class="ref-journal">Nat Rev Genet. </span>2005;<span class="ref-vol">6</span>:95–108.</span> [<a href="/pubmed/15716906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+for+common+diseases+and+complex+traits&amp;author=JN+Hirschhorn&amp;author=MJ+Daly&amp;volume=6&amp;publication_year=2005&amp;pages=95-108&amp;pmid=15716906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Sladek R, et al.  A genome-wide association study identifies novel risk loci for type 2 diabetes. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:881–885.</span> [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes&amp;author=R+Sladek&amp;volume=445&amp;publication_year=2007&amp;pages=881-885&amp;pmid=17293876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Easton D, et al.  A genome-wide association study identifies multiple breast cancer susceptibility loci. <span><span class="ref-journal">Nature. </span></span> <em>(in press)</em>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2714974/">PMC free article</a>]</span> [<a href="/pubmed/17529967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+multiple+breast+cancer+susceptibility+loci&amp;author=D+Easton&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Daugherty SE, et al.  RNASEL Arg462Gln polymorphism and prostate cancer in PLCO. <span><span class="ref-journal">Prostate. </span>2007</span> [<a href="/pubmed/17407163" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prostate&amp;title=RNASEL+Arg462Gln+polymorphism+and+prostate+cancer+in+PLCO&amp;author=SE+Daugherty&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Gudmundsson J, et al.  Genome-wide association study identifies a second prostate cancer susceptibility variant at 8q24. <span><span class="ref-journal">Nat Genet. </span>2007</span> [<a href="/pubmed/17401366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+second+prostate+cancer+susceptibility+variant+at+8q24&amp;author=J+Gudmundsson&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Grose R, Dickson C. Fibroblast growth factor signaling in tumorigenesis. <span><span class="ref-journal">Cytokine Growth Factor Rev. </span>2005;<span class="ref-vol">16</span>:179–186.</span> [<a href="/pubmed/15863033" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine+Growth+Factor+Rev&amp;title=Fibroblast+growth+factor+signaling+in+tumorigenesis&amp;author=R+Grose&amp;author=C+Dickson&amp;volume=16&amp;publication_year=2005&amp;pages=179-186&amp;pmid=15863033&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Minichiello MJ, Durbin R. Mapping trait loci by use of inferred ancestral recombination graphs. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">79</span>:910–922.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1698562/">PMC free article</a>]</span> [<a href="/pubmed/17033967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Mapping+trait+loci+by+use+of+inferred+ancestral+recombination+graphs&amp;author=MJ+Minichiello&amp;author=R+Durbin&amp;volume=79&amp;publication_year=2006&amp;pages=910-922&amp;pmid=17033967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Bruzzi P, Green SB, Byar DP, Brinton LA, Schairer C. Estimating the population attributable risk for multiple risk factors using case-control data. <span><span class="ref-journal">Am J Epidemiol. </span>1985;<span class="ref-vol">122</span>:904–914.</span> [<a href="/pubmed/4050778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Estimating+the+population+attributable+risk+for+multiple+risk+factors+using+case-control+data&amp;author=P+Bruzzi&amp;author=SB+Green&amp;author=DP+Byar&amp;author=LA+Brinton&amp;author=C+Schairer&amp;volume=122&amp;publication_year=1985&amp;pages=904-914&amp;pmid=4050778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Moffa AB, Ethier SP. Differential signal transduction of alternatively spliced FGFR2 variants expressed in human mammary epithelial cells. <span><span class="ref-journal">J Cell Physiol. </span>2007;<span class="ref-vol">210</span>:720–731.</span> [<a href="/pubmed/17133345" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Physiol&amp;title=Differential+signal+transduction+of+alternatively+spliced+FGFR2+variants+expressed+in+human+mammary+epithelial+cells&amp;author=AB+Moffa&amp;author=SP+Ethier&amp;volume=210&amp;publication_year=2007&amp;pages=720-731&amp;pmid=17133345&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Chanock S, et al.  What constitutes Replication of a Genotype-Phenotpe Association? Summary of an NCI-NHGRI Working Group. <span><span class="ref-journal">Nature. </span></span> <em>(in press)</em>. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=What+constitutes+Replication+of+a+Genotype-Phenotpe+Association?+Summary+of+an+NCI-NHGRI+Working+Group&amp;author=S+Chanock&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Wang H, Thomas DC, Pe'er I, Stram DO. Optimal two-stage genotyping designs for genome-wide association scans. <span><span class="ref-journal">Genet Epidemiol. </span>2006;<span class="ref-vol">30</span>:356–368.</span> [<a href="/pubmed/16607626" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Optimal+two-stage+genotyping+designs+for+genome-wide+association+scans&amp;author=H+Wang&amp;author=DC+Thomas&amp;author=I+Pe'er&amp;author=DO+Stram&amp;volume=30&amp;publication_year=2006&amp;pages=356-368&amp;pmid=16607626&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Tworoger SS, Eliassen AH, Sluss P, Hankinson SE. A prospective study of plasma prolactin concentrations and risk of premenopausal and postmenopausal breast cancer. <span><span class="ref-journal">J Clin Oncol. </span>2007;<span class="ref-vol">25</span>:1482–1498.</span> [<a href="/pubmed/17372279" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=A+prospective+study+of+plasma+prolactin+concentrations+and+risk+of+premenopausal+and+postmenopausal+breast+cancer&amp;author=SS+Tworoger&amp;author=AH+Eliassen&amp;author=P+Sluss&amp;author=SE+Hankinson&amp;volume=25&amp;publication_year=2007&amp;pages=1482-1498&amp;pmid=17372279&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Colditz GA, Rosner B. Cumulative risk of breast cancer to age 70 years according to risk factor status: data from the Nurses' Health Study. <span><span class="ref-journal">Am J Epidemiol. </span>2000;<span class="ref-vol">152</span>:950–964.</span> [<a href="/pubmed/11092437" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Cumulative+risk+of+breast+cancer+to+age+70+years+according+to+risk+factor+status:+data+from+the+Nurses'+Health+Study&amp;author=GA+Colditz&amp;author=B+Rosner&amp;volume=152&amp;publication_year=2000&amp;pages=950-964&amp;pmid=11092437&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Eliassen AH, Tworoger SS, Mantzoros CS, Pollak MN, Hankinson SE. Circulating insulin and c-peptide levels and risk of breast cancer among predominately premenopausal women. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2007;<span class="ref-vol">16</span>:161–164.</span> [<a href="/pubmed/17220346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Circulating+insulin+and+c-peptide+levels+and+risk+of+breast+cancer+among+predominately+premenopausal+women&amp;author=AH+Eliassen&amp;author=SS+Tworoger&amp;author=CS+Mantzoros&amp;author=MN+Pollak&amp;author=SE+Hankinson&amp;volume=16&amp;publication_year=2007&amp;pages=161-164&amp;pmid=17220346&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Hayes RB, et al.  Etiologic and early marker studies in the prostate, lung, colorectal and ovarian (PLCO) cancer screening trial. <span><span class="ref-journal">Controlled Clinical Trials. </span>2000;<span class="ref-vol">21</span>:341S–355S.</span> [<a href="/pubmed/11189687" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Controlled+Clinical+Trials&amp;title=Etiologic+and+early+marker+studies+in+the+prostate,+lung,+colorectal+and+ovarian+(PLCO)+cancer+screening+trial&amp;author=RB+Hayes&amp;volume=21&amp;publication_year=2000&amp;pages=341S-355S&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Stevens VL, Rodriguez C, Pavluck AL, Thun MJ, Calle EE. Association of polymorphisms in the paraoxonase 1 gene with breast cancer incidence in the CPS-II Nutrition Cohort. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2006;<span class="ref-vol">15</span>:1226–1228.</span> [<a href="/pubmed/16775186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Association+of+polymorphisms+in+the+paraoxonase+1+gene+with+breast+cancer+incidence+in+the+CPS-II+Nutrition+Cohort&amp;author=VL+Stevens&amp;author=C+Rodriguez&amp;author=AL+Pavluck&amp;author=MJ+Thun&amp;author=EE+Calle&amp;volume=15&amp;publication_year=2006&amp;pages=1226-1228&amp;pmid=16775186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. <span><span class="ref-journal">Genetics. </span>2003;<span class="ref-vol">164</span>:1567–1587.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1462648/">PMC free article</a>]</span> [<a href="/pubmed/12930761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data:+linked+loci+and+correlated+allele+frequencies&amp;author=D+Falush&amp;author=M+Stephens&amp;author=JK+Pritchard&amp;volume=164&amp;publication_year=2003&amp;pages=1567-1587&amp;pmid=12930761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Kraft P, Cox DG, Paynter RA, Hunter D, De Vivo I. Accounting for haplotype uncertainty in matched association studies: a comparison of simple and flexible techniques. <span><span class="ref-journal">Genet Epidemiol. </span>2005;<span class="ref-vol">28</span>:261–272.</span> [<a href="/pubmed/15637718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Accounting+for+haplotype+uncertainty+in+matched+association+studies:+a+comparison+of+simple+and+flexible+techniques&amp;author=P+Kraft&amp;author=DG+Cox&amp;author=RA+Paynter&amp;author=D+Hunter&amp;author=I+De+Vivo&amp;volume=28&amp;publication_year=2005&amp;pages=261-272&amp;pmid=15637718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Patterson N, et al.  Population Structure and Eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:2076–2093.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+Structure+and+Eigenanalysis&amp;author=N+Patterson&amp;volume=2&amp;publication_year=2006&amp;pages=2076-2093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3493132&amp;issue-id=216159&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140365842488288title">Abstract</a></li><li><a href="#S1title">METHODS</a></li><li><a href="#S19title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140365845892656title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3493132/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3493132/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3493132/pdf/nihms415304.pdf">PDF (1.7M)</a></li> | <li><a href="#" data-citationid="PMC3493132" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3493132%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3493132%2F&amp;text=A%20genome-wide%20association%20study%20identifies%20alleles%20in%20FGFR2%20associated%20with%20risk%20of%20sporadic%20postmenopausal%20breast%20cancer" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3493132%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="17529973" data-aiid="3493132" data-aid="3493132" data-iid="216159" data-domainid="319" data-domain="nihpa" data-accid="PMC3493132" data-md5="3deb3fc7118e6a3d75df2101a5039f27"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T08:36:35-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88108DDD3E993100000000050C02DC&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3493132%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3493132/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>