<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3640413/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Interaction between ERAP1 and HLA-B27 in ankylosing spondylitis implicates peptide handling in the mechanism for HLA-B27 in disease susceptibility</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3640413/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Interaction between ERAP1 and HLA-B27 in ankylosing spondylitis implicates peptide handling in the mechanism for HLA-B27 in disease susceptibility"><meta name="citation_authors" content="David M Evans, Chris C A Spencer, Jennifer J Pointon, Zhan Su, David Harvey, Grazyna Kochan, Udo Oppermann, Alexander Dilthey, Matti Pirinen, Millicent A Stone, Louise Appleton, Loukas Moutsianas, Stephen Leslie, Tom Wordsworth, Tony J Kenna, Tugce Karaderi, Gethin P Thomas, Michael M Ward, Michael H Weisman, Claire Farrar, Linda A Bradbury, Patrick Danoy, Robert D Inman, Walter Maksymowych, Dafna Gladman, Proton Rahman, Spondyloarthritis Research Consortium of Canada (SPARCC), Ann Morgan, Helena Marzo-Ortega, Paul Bowness, Karl Gaffney, J S Hill Gaston, Malcolm Smith, Jacome Bruges-Armas, Ana-Rita Couto, Rosa Sorrentino, Fabiana Paladini, Manuel A Ferreira, Huji Xu, Yu Liu, Lei Jiang, Carlos Lopez-Larrea, Roberto Díaz-Peña, Antonio López-Vázquez, Tetyana Zayats, Gavin Band, Céline Bellenguez, Hannah Blackburn, Jenefer M Blackwell, Elvira Bramon, Suzannah J Bumpstead, Juan P Casas, Aiden Corvin, Nicholas Craddock, Panos Deloukas, Serge Dronov, Audrey Duncanson, Sarah Edkins, Colin Freeman, Matthew Gillman, Emma Gray, Rhian Gwilliam, Naomi Hammond, Sarah E Hunt, Janusz Jankowski, Alagurevathi Jayakumar, Cordelia Langford, Jennifer Liddle, Hugh S Markus, Christopher G Mathew, Owen T McCann, Mark I McCarthy, Colin N A Palmer, Leena Peltonen, Robert Plomin, Simon C Potter, Anna Rautanen, Radhi Ravindrarajah, Michelle Ricketts, Nilesh Samani, Stephen J Sawcer, Amy Strange, Richard C Trembath, Ananth C Viswanathan, Matthew Waller, Paul Weston, Pamela Whittaker, Sara Widaa, Nicholas W Wood, Gilean McVean, John D Reveille, B Paul Wordsworth, Matthew A Brown, Peter Donnelly, The Australo-Anglo-American Spondyloarthritis Consortium (TASC), the Wellcome Trust Case Control Consortium 2 (WTCCC2)"><meta name="citation_issue" content="8"><meta name="citation_volume" content="43"><meta name="citation_firstpage" content="761"><meta name="citation_doi" content="10.1038/ng.873"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3640413/?report=abstract"><meta name="citation_pmid" content="21743469"><meta name="DC.Title" content="Interaction between ERAP1 and HLA-B27 in ankylosing spondylitis implicates peptide handling in the mechanism for HLA-B27 in disease susceptibility"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="David M Evans"><meta name="DC.Contributor" content="Chris C A Spencer"><meta name="DC.Contributor" content="Jennifer J Pointon"><meta name="DC.Contributor" content="Zhan Su"><meta name="DC.Contributor" content="David Harvey"><meta name="DC.Contributor" content="Grazyna Kochan"><meta name="DC.Contributor" content="Udo Oppermann"><meta name="DC.Contributor" content="Alexander Dilthey"><meta name="DC.Contributor" content="Matti Pirinen"><meta name="DC.Contributor" content="Millicent A Stone"><meta name="DC.Contributor" content="Louise Appleton"><meta name="DC.Contributor" content="Loukas Moutsianas"><meta name="DC.Contributor" content="Stephen Leslie"><meta name="DC.Contributor" content="Tom Wordsworth"><meta name="DC.Contributor" content="Tony J Kenna"><meta name="DC.Contributor" content="Tugce Karaderi"><meta name="DC.Contributor" content="Gethin P Thomas"><meta name="DC.Contributor" content="Michael M Ward"><meta name="DC.Contributor" content="Michael H Weisman"><meta name="DC.Contributor" content="Claire Farrar"><meta name="DC.Contributor" content="Linda A Bradbury"><meta name="DC.Contributor" content="Patrick Danoy"><meta name="DC.Contributor" content="Robert D Inman"><meta name="DC.Contributor" content="Walter Maksymowych"><meta name="DC.Contributor" content="Dafna Gladman"><meta name="DC.Contributor" content="Proton Rahman"><meta name="DC.Contributor" content="Spondyloarthritis Research Consortium of Canada (SPARCC)"><meta name="DC.Contributor" content="Ann Morgan"><meta name="DC.Contributor" content="Helena Marzo-Ortega"><meta name="DC.Contributor" content="Paul Bowness"><meta name="DC.Contributor" content="Karl Gaffney"><meta name="DC.Contributor" content="J S Hill Gaston"><meta name="DC.Contributor" content="Malcolm Smith"><meta name="DC.Contributor" content="Jacome Bruges-Armas"><meta name="DC.Contributor" content="Ana-Rita Couto"><meta name="DC.Contributor" content="Rosa Sorrentino"><meta name="DC.Contributor" content="Fabiana Paladini"><meta name="DC.Contributor" content="Manuel A Ferreira"><meta name="DC.Contributor" content="Huji Xu"><meta name="DC.Contributor" content="Yu Liu"><meta name="DC.Contributor" content="Lei Jiang"><meta name="DC.Contributor" content="Carlos Lopez-Larrea"><meta name="DC.Contributor" content="Roberto Díaz-Peña"><meta name="DC.Contributor" content="Antonio López-Vázquez"><meta name="DC.Contributor" content="Tetyana Zayats"><meta name="DC.Contributor" content="Gavin Band"><meta name="DC.Contributor" content="Céline Bellenguez"><meta name="DC.Contributor" content="Hannah Blackburn"><meta name="DC.Contributor" content="Jenefer M Blackwell"><meta name="DC.Contributor" content="Elvira Bramon"><meta name="DC.Contributor" content="Suzannah J Bumpstead"><meta name="DC.Contributor" content="Juan P Casas"><meta name="DC.Contributor" content="Aiden Corvin"><meta name="DC.Contributor" content="Nicholas Craddock"><meta name="DC.Contributor" content="Panos Deloukas"><meta name="DC.Contributor" content="Serge Dronov"><meta name="DC.Contributor" content="Audrey Duncanson"><meta name="DC.Contributor" content="Sarah Edkins"><meta name="DC.Contributor" content="Colin Freeman"><meta name="DC.Contributor" content="Matthew Gillman"><meta name="DC.Contributor" content="Emma Gray"><meta name="DC.Contributor" content="Rhian Gwilliam"><meta name="DC.Contributor" content="Naomi Hammond"><meta name="DC.Contributor" content="Sarah E Hunt"><meta name="DC.Contributor" content="Janusz Jankowski"><meta name="DC.Contributor" content="Alagurevathi Jayakumar"><meta name="DC.Contributor" content="Cordelia Langford"><meta name="DC.Contributor" content="Jennifer Liddle"><meta name="DC.Contributor" content="Hugh S Markus"><meta name="DC.Contributor" content="Christopher G Mathew"><meta name="DC.Contributor" content="Owen T McCann"><meta name="DC.Contributor" content="Mark I McCarthy"><meta name="DC.Contributor" content="Colin N A Palmer"><meta name="DC.Contributor" content="Leena Peltonen"><meta name="DC.Contributor" content="Robert Plomin"><meta name="DC.Contributor" content="Simon C Potter"><meta name="DC.Contributor" content="Anna Rautanen"><meta name="DC.Contributor" content="Radhi Ravindrarajah"><meta name="DC.Contributor" content="Michelle Ricketts"><meta name="DC.Contributor" content="Nilesh Samani"><meta name="DC.Contributor" content="Stephen J Sawcer"><meta name="DC.Contributor" content="Amy Strange"><meta name="DC.Contributor" content="Richard C Trembath"><meta name="DC.Contributor" content="Ananth C Viswanathan"><meta name="DC.Contributor" content="Matthew Waller"><meta name="DC.Contributor" content="Paul Weston"><meta name="DC.Contributor" content="Pamela Whittaker"><meta name="DC.Contributor" content="Sara Widaa"><meta name="DC.Contributor" content="Nicholas W Wood"><meta name="DC.Contributor" content="Gilean McVean"><meta name="DC.Contributor" content="John D Reveille"><meta name="DC.Contributor" content="B Paul Wordsworth"><meta name="DC.Contributor" content="Matthew A Brown"><meta name="DC.Contributor" content="Peter Donnelly"><meta name="DC.Contributor" content="The Australo-Anglo-American Spondyloarthritis Consortium (TASC)"><meta name="DC.Contributor" content="the Wellcome Trust Case Control Consortium 2 (WTCCC2)"><meta name="DC.Identifier" content="10.1038/ng.873"><meta name="DC.Language" content="en"><meta property="og:title" content="Interaction between ERAP1 and HLA-B27 in ankylosing spondylitis implicates peptide handling in the mechanism for HLA-B27 in disease susceptibility"><meta property="og:type" content="article"><meta property="og:description" content="Ankylosing spondylitis is a common form of inflammatory arthritis predominantly affecting the spine and pelvis that occurs in approximately 5 out of 1,000 adults of European descent. Here we report the identification of three variants in the RUNX3, LTBRTNFRSF1A ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3640413/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3640413/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3640413/pdf/nihms-405164.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D5F7DDD5298D100000000043C02D6.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3640413%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="nuccore">Nucleotide</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3640413</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2013 May 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21743469" target="pmc_ext" ref="reftype=publisher&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2011 Jul 10; 43(8): 761–767. </span></a></div><span class="fm-vol-iss-date">Published online 2011 Jul 10. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.873" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.873</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3640413</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS405164</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21743469">21743469</a></div></div></div></div><h1 class="content-title">Interaction between <em>ERAP1</em> and HLA-B27 in ankylosing spondylitis implicates peptide handling in the mechanism for HLA-B27 in disease susceptibility</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Evans%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960900967520" co-class="co-affbox">David M Evans</a>,<sup>1,</sup><sup>43</sup> <a href="/pubmed/?term=Spencer%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960879311120" co-class="co-affbox">Chris C A Spencer</a>,<sup>2,</sup><sup>43</sup> <a href="/pubmed/?term=Pointon%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960887393552" co-class="co-affbox">Jennifer J Pointon</a>,<sup>3</sup> <a href="/pubmed/?term=Su%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960891298464" co-class="co-affbox">Zhan Su</a>,<sup>2</sup> <a href="/pubmed/?term=Harvey%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960888225872" co-class="co-affbox">David Harvey</a>,<sup>3</sup> <a href="/pubmed/?term=Kochan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960877375680" co-class="co-affbox">Grazyna Kochan</a>,<sup>4</sup> <a href="/pubmed/?term=Oppermann%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960899033680" co-class="co-affbox">Udo Oppermann</a>,<sup>4</sup> <a href="/pubmed/?term=Dilthey%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960888033280" co-class="co-affbox">Alexander Dilthey</a>,<sup>1</sup> <a href="/pubmed/?term=Pirinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960878661136" co-class="co-affbox">Matti Pirinen</a>,<sup>1</sup> <a href="/pubmed/?term=Stone%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960883543168" co-class="co-affbox">Millicent A Stone</a>,<sup>5</sup> <a href="/pubmed/?term=Appleton%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960881686304" co-class="co-affbox">Louise Appleton</a>,<sup>3</sup> <a href="/pubmed/?term=Moutsianas%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960898643664" co-class="co-affbox">Loukas Moutsianas</a>,<sup>6</sup> <a href="/pubmed/?term=Leslie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960885082592" co-class="co-affbox">Stephen Leslie</a>,<sup>7</sup> <a href="/pubmed/?term=Wordsworth%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960895631264" co-class="co-affbox">Tom Wordsworth</a>,<sup>3</sup> <a href="/pubmed/?term=Kenna%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960889633168" co-class="co-affbox">Tony J Kenna</a>,<sup>8</sup> <a href="/pubmed/?term=Karaderi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960879505216" co-class="co-affbox">Tugce Karaderi</a>,<sup>3</sup> <a href="/pubmed/?term=Thomas%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960898782144" co-class="co-affbox">Gethin P Thomas</a>,<sup>8</sup> <a href="/pubmed/?term=Ward%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960993316816" co-class="co-affbox">Michael M Ward</a>,<sup>9</sup> <a href="/pubmed/?term=Weisman%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960992849920" co-class="co-affbox">Michael H Weisman</a>,<sup>10</sup> <a href="/pubmed/?term=Farrar%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960992522560" co-class="co-affbox">Claire Farrar</a>,<sup>3</sup> <a href="/pubmed/?term=Bradbury%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139961003733456" co-class="co-affbox">Linda A Bradbury</a>,<sup>8</sup> <a href="/pubmed/?term=Danoy%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960981511424" co-class="co-affbox">Patrick Danoy</a>,<sup>8</sup> <a href="/pubmed/?term=Inman%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139961003905280" co-class="co-affbox">Robert D Inman</a>,<sup>11</sup> <a href="/pubmed/?term=Maksymowych%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960988554048" co-class="co-affbox">Walter Maksymowych</a>,<sup>12</sup> <a href="/pubmed/?term=Gladman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960995608672" co-class="co-affbox">Dafna Gladman</a>,<sup>11</sup> <a href="/pubmed/?term=Rahman%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960896209328" co-class="co-affbox">Proton Rahman</a>,<sup>13</sup> Spondyloarthritis Research Consortium of Canada (SPARCC), <a href="/pubmed/?term=Morgan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960997368208" co-class="co-affbox">Ann Morgan</a>,<sup>14</sup> <a href="/pubmed/?term=Marzo-Ortega%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139961073311248" co-class="co-affbox">Helena Marzo-Ortega</a>,<sup>14</sup> <a href="/pubmed/?term=Bowness%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960991313776" co-class="co-affbox">Paul Bowness</a>,<sup>3</sup> <a href="/pubmed/?term=Gaffney%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960902249616" co-class="co-affbox">Karl Gaffney</a>,<sup>15</sup> <a href="/pubmed/?term=Gaston%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960991211680" co-class="co-affbox">J S Hill Gaston</a>,<sup>16</sup> <a href="/pubmed/?term=Smith%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960992590800" co-class="co-affbox">Malcolm Smith</a>,<sup>17</sup> <a href="/pubmed/?term=Bruges-Armas%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960877356176" co-class="co-affbox">Jacome Bruges-Armas</a>,<sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Couto%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960998509680" co-class="co-affbox">Ana-Rita Couto</a>,<sup>18</sup> <a href="/pubmed/?term=Sorrentino%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960888633408" co-class="co-affbox">Rosa Sorrentino</a>,<sup>20</sup> <a href="/pubmed/?term=Paladini%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960982630528" co-class="co-affbox">Fabiana Paladini</a>,<sup>20</sup> <a href="/pubmed/?term=Ferreira%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139961002648560" co-class="co-affbox">Manuel A Ferreira</a>,<sup>21</sup> <a href="/pubmed/?term=Xu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960902225040" co-class="co-affbox">Huji Xu</a>,<sup>22</sup> <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960988100992" co-class="co-affbox">Yu Liu</a>,<sup>22</sup> <a href="/pubmed/?term=Jiang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960881170608" co-class="co-affbox">Lei Jiang</a>,<sup>22</sup> <a href="/pubmed/?term=Lopez-Larrea%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960883700880" co-class="co-affbox">Carlos Lopez-Larrea</a>,<sup>23</sup> <a href="/pubmed/?term=D%26%23x000ed%3Baz-Pe%26%23x000f1%3Ba%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960889635088" co-class="co-affbox">Roberto Díaz-Peña</a>,<sup>23</sup> <a href="/pubmed/?term=L%26%23x000f3%3Bpez-V%26%23x000e1%3Bzquez%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960889093248" co-class="co-affbox">Antonio López-Vázquez</a>,<sup>23</sup> <a href="/pubmed/?term=Zayats%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960994710352" co-class="co-affbox">Tetyana Zayats</a>,<sup>1</sup> <a href="/pubmed/?term=Band%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960896497184" co-class="co-affbox">Gavin Band</a>,<sup>2</sup> <a href="/pubmed/?term=Bellenguez%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960884207728" co-class="co-affbox">Céline Bellenguez</a>,<sup>2</sup> <a href="/pubmed/?term=Blackburn%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960901690720" co-class="co-affbox">Hannah Blackburn</a>,<sup>23</sup> <a href="/pubmed/?term=Blackwell%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960898352608" co-class="co-affbox">Jenefer M Blackwell</a>,<sup>25,</sup><sup>26</sup> <a href="/pubmed/?term=Bramon%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960991457296" co-class="co-affbox">Elvira Bramon</a>,<sup>24</sup> <a href="/pubmed/?term=Bumpstead%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960992570576" co-class="co-affbox">Suzannah J Bumpstead</a>,<sup>24</sup> <a href="/pubmed/?term=Casas%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139961076287936" co-class="co-affbox">Juan P Casas</a>,<sup>27</sup> <a href="/pubmed/?term=Corvin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960998021232" co-class="co-affbox">Aiden Corvin</a>,<sup>28</sup> <a href="/pubmed/?term=Craddock%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960896091728" co-class="co-affbox">Nicholas Craddock</a>,<sup>29</sup> <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960892908384" co-class="co-affbox">Panos Deloukas</a>,<sup>24</sup> <a href="/pubmed/?term=Dronov%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960897144208" co-class="co-affbox">Serge Dronov</a>,<sup>24</sup> <a href="/pubmed/?term=Duncanson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960883908528" co-class="co-affbox">Audrey Duncanson</a>,<sup>30</sup> <a href="/pubmed/?term=Edkins%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960880789120" co-class="co-affbox">Sarah Edkins</a>,<sup>24</sup> <a href="/pubmed/?term=Freeman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960897427552" co-class="co-affbox">Colin Freeman</a>,<sup>2</sup> <a href="/pubmed/?term=Gillman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960887971872" co-class="co-affbox">Matthew Gillman</a>,<sup>24</sup> <a href="/pubmed/?term=Gray%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960997354272" co-class="co-affbox">Emma Gray</a>,<sup>24</sup> <a href="/pubmed/?term=Gwilliam%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960882935104" co-class="co-affbox">Rhian Gwilliam</a>,<sup>24</sup> <a href="/pubmed/?term=Hammond%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960891527120" co-class="co-affbox">Naomi Hammond</a>,<sup>24</sup> <a href="/pubmed/?term=Hunt%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960985429040" co-class="co-affbox">Sarah E Hunt</a>,<sup>24</sup> <a href="/pubmed/?term=Jankowski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960891068048" co-class="co-affbox">Janusz Jankowski</a>,<sup>31</sup> <a href="/pubmed/?term=Jayakumar%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960898458912" co-class="co-affbox">Alagurevathi Jayakumar</a>,<sup>24</sup> <a href="/pubmed/?term=Langford%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960883137808" co-class="co-affbox">Cordelia Langford</a>,<sup>24</sup> <a href="/pubmed/?term=Liddle%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960894135952" co-class="co-affbox">Jennifer Liddle</a>,<sup>24</sup> <a href="/pubmed/?term=Markus%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960892313520" co-class="co-affbox">Hugh S Markus</a>,<sup>32</sup> <a href="/pubmed/?term=Mathew%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960882066224" co-class="co-affbox">Christopher G Mathew</a>,<sup>33</sup> <a href="/pubmed/?term=McCann%20OT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960896309856" co-class="co-affbox">Owen T McCann</a>,<sup>24</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960880169392" co-class="co-affbox">Mark I McCarthy</a>,<sup>34</sup> <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960877562752" co-class="co-affbox">Colin N A Palmer</a>,<sup>35</sup> <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960992409344" co-class="co-affbox">Leena Peltonen</a>,<sup>24</sup> <a href="/pubmed/?term=Plomin%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960902207408" co-class="co-affbox">Robert Plomin</a>,<sup>36</sup> <a href="/pubmed/?term=Potter%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960994141568" co-class="co-affbox">Simon C Potter</a>,<sup>24</sup> <a href="/pubmed/?term=Rautanen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960884243920" co-class="co-affbox">Anna Rautanen</a>,<sup>24</sup> <a href="/pubmed/?term=Ravindrarajah%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960889897056" co-class="co-affbox">Radhi Ravindrarajah</a>,<sup>24</sup> <a href="/pubmed/?term=Ricketts%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960900945792" co-class="co-affbox">Michelle Ricketts</a>,<sup>24</sup> <a href="/pubmed/?term=Samani%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960889147200" co-class="co-affbox">Nilesh Samani</a>,<sup>37</sup> <a href="/pubmed/?term=Sawcer%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960878773920" co-class="co-affbox">Stephen J Sawcer</a>,<sup>38</sup> <a href="/pubmed/?term=Strange%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960894567360" co-class="co-affbox">Amy Strange</a>,<sup>2</sup> <a href="/pubmed/?term=Trembath%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960882654336" co-class="co-affbox">Richard C Trembath</a>,<sup>33</sup> <a href="/pubmed/?term=Viswanathan%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960894564800" co-class="co-affbox">Ananth C Viswanathan</a>,<sup>39,</sup><sup>40</sup> <a href="/pubmed/?term=Waller%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960902190064" co-class="co-affbox">Matthew Waller</a>,<sup>24</sup> <a href="/pubmed/?term=Weston%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960996215456" co-class="co-affbox">Paul Weston</a>,<sup>24</sup> <a href="/pubmed/?term=Whittaker%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960920662304" co-class="co-affbox">Pamela Whittaker</a>,<sup>24</sup> <a href="/pubmed/?term=Widaa%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960992931024" co-class="co-affbox">Sara Widaa</a>,<sup>24</sup> <a href="/pubmed/?term=Wood%20NW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960981484976" co-class="co-affbox">Nicholas W Wood</a>,<sup>41</sup> <a href="/pubmed/?term=McVean%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960894488032" co-class="co-affbox">Gilean McVean</a>,<sup>2</sup> <a href="/pubmed/?term=Reveille%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960878383456" co-class="co-affbox">John D Reveille</a>,<sup>42</sup> <a href="/pubmed/?term=Wordsworth%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960880972000" co-class="co-affbox">B Paul Wordsworth</a>,<sup>3</sup> <a href="/pubmed/?term=Brown%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960991224304" co-class="co-affbox">Matthew A Brown</a>,<sup>8,</sup><sup>44</sup> <a href="/pubmed/?term=Donnelly%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469" class="affpopup" co-rid="_co_idm139960889860704" co-class="co-affbox">Peter Donnelly</a>,<sup>2,</sup><sup>44</sup> The Australo-Anglo-American Spondyloarthritis Consortium (TASC),<sup>1</sup> and  the Wellcome Trust Case Control Consortium 2 (WTCCC2)<sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139960900967520"><h3 class="no_margin">David M Evans</h3><p><sup>1</sup>Medical Research Council (MRC) Centre for Causal Analyses in Translational Epidemiology, School of Social and Community Medicine, University of Bristol, Bristol, UK. </p><div>Find articles by <a href="/pubmed/?term=Evans%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">David M Evans</a></div></div><div id="_co_idm139960879311120"><h3 class="no_margin">Chris C A Spencer</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Spencer%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Chris C A Spencer</a></div></div><div id="_co_idm139960887393552"><h3 class="no_margin">Jennifer J Pointon</h3><p><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Pointon%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Jennifer J Pointon</a></div></div><div id="_co_idm139960891298464"><h3 class="no_margin">Zhan Su</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Su%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Zhan Su</a></div></div><div id="_co_idm139960888225872"><h3 class="no_margin">David Harvey</h3><p><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Harvey%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">David Harvey</a></div></div><div id="_co_idm139960877375680"><h3 class="no_margin">Grazyna Kochan</h3><p><sup>4</sup>Structural Genomics Consortium, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Kochan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Grazyna Kochan</a></div></div><div id="_co_idm139960899033680"><h3 class="no_margin">Udo Oppermann</h3><p><sup>4</sup>Structural Genomics Consortium, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Oppermann%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Udo Oppermann</a></div></div><div id="_co_idm139960888033280"><h3 class="no_margin">Alexander Dilthey</h3><p><sup>1</sup>Medical Research Council (MRC) Centre for Causal Analyses in Translational Epidemiology, School of Social and Community Medicine, University of Bristol, Bristol, UK. </p><div>Find articles by <a href="/pubmed/?term=Dilthey%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Alexander Dilthey</a></div></div><div id="_co_idm139960878661136"><h3 class="no_margin">Matti Pirinen</h3><p><sup>1</sup>Medical Research Council (MRC) Centre for Causal Analyses in Translational Epidemiology, School of Social and Community Medicine, University of Bristol, Bristol, UK. </p><div>Find articles by <a href="/pubmed/?term=Pirinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Matti Pirinen</a></div></div><div id="_co_idm139960883543168"><h3 class="no_margin">Millicent A Stone</h3><p><sup>5</sup>University of Bath, Bath, UK. </p><div>Find articles by <a href="/pubmed/?term=Stone%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Millicent A Stone</a></div></div><div id="_co_idm139960881686304"><h3 class="no_margin">Louise Appleton</h3><p><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Appleton%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Louise Appleton</a></div></div><div id="_co_idm139960898643664"><h3 class="no_margin">Loukas Moutsianas</h3><p><sup>6</sup>Department of Statistics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Moutsianas%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Loukas Moutsianas</a></div></div><div id="_co_idm139960885082592"><h3 class="no_margin">Stephen Leslie</h3><p><sup>7</sup>Department of Clinical Pharmacology, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Leslie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Stephen Leslie</a></div></div><div id="_co_idm139960895631264"><h3 class="no_margin">Tom Wordsworth</h3><p><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Wordsworth%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Tom Wordsworth</a></div></div><div id="_co_idm139960889633168"><h3 class="no_margin">Tony J Kenna</h3><p><sup>8</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Kenna%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Tony J Kenna</a></div></div><div id="_co_idm139960879505216"><h3 class="no_margin">Tugce Karaderi</h3><p><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Karaderi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Tugce Karaderi</a></div></div><div id="_co_idm139960898782144"><h3 class="no_margin">Gethin P Thomas</h3><p><sup>8</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Thomas%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Gethin P Thomas</a></div></div><div id="_co_idm139960993316816"><h3 class="no_margin">Michael M Ward</h3><p><sup>9</sup>National Institute of Arthritis and Musculoskeletal and Skin Diseases, US National Institutes of Health (NIH), Bethesda, Maryland, USA. </p><div>Find articles by <a href="/pubmed/?term=Ward%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Michael M Ward</a></div></div><div id="_co_idm139960992849920"><h3 class="no_margin">Michael H Weisman</h3><p><sup>10</sup>Department of Medicine/Rheumatology, Cedars-Sinai Medical Centre, Los Angeles, California, USA. </p><div>Find articles by <a href="/pubmed/?term=Weisman%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Michael H Weisman</a></div></div><div id="_co_idm139960992522560"><h3 class="no_margin">Claire Farrar</h3><p><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Farrar%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Claire Farrar</a></div></div><div id="_co_idm139961003733456"><h3 class="no_margin">Linda A Bradbury</h3><p><sup>8</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Bradbury%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Linda A Bradbury</a></div></div><div id="_co_idm139960981511424"><h3 class="no_margin">Patrick Danoy</h3><p><sup>8</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Danoy%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Patrick Danoy</a></div></div><div id="_co_idm139961003905280"><h3 class="no_margin">Robert D Inman</h3><p><sup>11</sup>University of Toronto, Toronto, Ontario, Canada. </p><div>Find articles by <a href="/pubmed/?term=Inman%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Robert D Inman</a></div></div><div id="_co_idm139960988554048"><h3 class="no_margin">Walter Maksymowych</h3><p><sup>12</sup>Department of Medicine, University of Alberta, Edmonton, Alberta, Canada. </p><div>Find articles by <a href="/pubmed/?term=Maksymowych%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Walter Maksymowych</a></div></div><div id="_co_idm139960995608672"><h3 class="no_margin">Dafna Gladman</h3><p><sup>11</sup>University of Toronto, Toronto, Ontario, Canada. </p><div>Find articles by <a href="/pubmed/?term=Gladman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Dafna Gladman</a></div></div><div id="_co_idm139960896209328"><h3 class="no_margin">Proton Rahman</h3><p><sup>13</sup>Memorial University, St. John’s, Newfoundland, Canada. </p><div>Find articles by <a href="/pubmed/?term=Rahman%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Proton Rahman</a></div></div><div id="_co_idm139960997368208"><h3 class="no_margin">Ann Morgan</h3><p><sup>14</sup>National Institute for Health Research (NIHR)–Leeds Musculoskeletal Biomedical Research Unit, University of Leeds, Leeds, UK. </p><div>Find articles by <a href="/pubmed/?term=Morgan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Ann Morgan</a></div></div><div id="_co_idm139961073311248"><h3 class="no_margin">Helena Marzo-Ortega</h3><p><sup>14</sup>National Institute for Health Research (NIHR)–Leeds Musculoskeletal Biomedical Research Unit, University of Leeds, Leeds, UK. </p><div>Find articles by <a href="/pubmed/?term=Marzo-Ortega%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Helena Marzo-Ortega</a></div></div><div id="_co_idm139960991313776"><h3 class="no_margin">Paul Bowness</h3><p><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Bowness%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Paul Bowness</a></div></div><div id="_co_idm139960902249616"><h3 class="no_margin">Karl Gaffney</h3><p><sup>15</sup>Department of Rheumatology, Norfolk &amp; Norwich University Hospital, Norfolk, UK. </p><div>Find articles by <a href="/pubmed/?term=Gaffney%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Karl Gaffney</a></div></div><div id="_co_idm139960991211680"><h3 class="no_margin">J S Hill Gaston</h3><p><sup>16</sup>Department of Medicine, University of Cambridge, Addenbrookes Hospital, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Gaston%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">J S Hill Gaston</a></div></div><div id="_co_idm139960992590800"><h3 class="no_margin">Malcolm Smith</h3><p><sup>17</sup>Repatriation General Hospital, Adelaide, Australia. </p><div>Find articles by <a href="/pubmed/?term=Smith%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Malcolm Smith</a></div></div><div id="_co_idm139960877356176"><h3 class="no_margin">Jacome Bruges-Armas</h3><p><sup>18</sup>Serviço Especializado De Epidemiologia E Biologia Molecular, Hospital de Santo Espírito, Angra do Heroísmo, Terceira, The Azores, Portugal. </p><p><sup>19</sup>Genetics and Arthritis Research Group, Institute for Molecular and Cell Biology (IBMC), University of Porto, Porto, Portugal. </p><div>Find articles by <a href="/pubmed/?term=Bruges-Armas%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Jacome Bruges-Armas</a></div></div><div id="_co_idm139960998509680"><h3 class="no_margin">Ana-Rita Couto</h3><p><sup>18</sup>Serviço Especializado De Epidemiologia E Biologia Molecular, Hospital de Santo Espírito, Angra do Heroísmo, Terceira, The Azores, Portugal. </p><div>Find articles by <a href="/pubmed/?term=Couto%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Ana-Rita Couto</a></div></div><div id="_co_idm139960888633408"><h3 class="no_margin">Rosa Sorrentino</h3><p><sup>20</sup>Department of Biology and Biotechnology ‘Charles Darwin’, Sapienza University of Rome, Rome, Italy. </p><div>Find articles by <a href="/pubmed/?term=Sorrentino%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Rosa Sorrentino</a></div></div><div id="_co_idm139960982630528"><h3 class="no_margin">Fabiana Paladini</h3><p><sup>20</sup>Department of Biology and Biotechnology ‘Charles Darwin’, Sapienza University of Rome, Rome, Italy. </p><div>Find articles by <a href="/pubmed/?term=Paladini%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Fabiana Paladini</a></div></div><div id="_co_idm139961002648560"><h3 class="no_margin">Manuel A Ferreira</h3><p><sup>21</sup>Queensland Institute of Medical Research, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Ferreira%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Manuel A Ferreira</a></div></div><div id="_co_idm139960902225040"><h3 class="no_margin">Huji Xu</h3><p><sup>22</sup>Department of Rheumatology and Immunology, Shanghai Changzheng Hospital, The Second Military Medical University Hospital, Shanghai, China. </p><div>Find articles by <a href="/pubmed/?term=Xu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Huji Xu</a></div></div><div id="_co_idm139960988100992"><h3 class="no_margin">Yu Liu</h3><p><sup>22</sup>Department of Rheumatology and Immunology, Shanghai Changzheng Hospital, The Second Military Medical University Hospital, Shanghai, China. </p><div>Find articles by <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Yu Liu</a></div></div><div id="_co_idm139960881170608"><h3 class="no_margin">Lei Jiang</h3><p><sup>22</sup>Department of Rheumatology and Immunology, Shanghai Changzheng Hospital, The Second Military Medical University Hospital, Shanghai, China. </p><div>Find articles by <a href="/pubmed/?term=Jiang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Lei Jiang</a></div></div><div id="_co_idm139960883700880"><h3 class="no_margin">Carlos Lopez-Larrea</h3><p><sup>23</sup>Department of Immunology, Hospital Universitario Central de Asturias, Oviedo, Spain. </p><div>Find articles by <a href="/pubmed/?term=Lopez-Larrea%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Carlos Lopez-Larrea</a></div></div><div id="_co_idm139960889635088"><h3 class="no_margin">Roberto Díaz-Peña</h3><p><sup>23</sup>Department of Immunology, Hospital Universitario Central de Asturias, Oviedo, Spain. </p><div>Find articles by <a href="/pubmed/?term=D%26%23x000ed%3Baz-Pe%26%23x000f1%3Ba%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Roberto Díaz-Peña</a></div></div><div id="_co_idm139960889093248"><h3 class="no_margin">Antonio López-Vázquez</h3><p><sup>23</sup>Department of Immunology, Hospital Universitario Central de Asturias, Oviedo, Spain. </p><div>Find articles by <a href="/pubmed/?term=L%26%23x000f3%3Bpez-V%26%23x000e1%3Bzquez%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Antonio López-Vázquez</a></div></div><div id="_co_idm139960994710352"><h3 class="no_margin">Tetyana Zayats</h3><p><sup>1</sup>Medical Research Council (MRC) Centre for Causal Analyses in Translational Epidemiology, School of Social and Community Medicine, University of Bristol, Bristol, UK. </p><div>Find articles by <a href="/pubmed/?term=Zayats%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Tetyana Zayats</a></div></div><div id="_co_idm139960896497184"><h3 class="no_margin">Gavin Band</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Band%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Gavin Band</a></div></div><div id="_co_idm139960884207728"><h3 class="no_margin">Céline Bellenguez</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Bellenguez%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Céline Bellenguez</a></div></div><div id="_co_idm139960901690720"><h3 class="no_margin">Hannah Blackburn</h3><p><sup>23</sup>Department of Immunology, Hospital Universitario Central de Asturias, Oviedo, Spain. </p><div>Find articles by <a href="/pubmed/?term=Blackburn%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Hannah Blackburn</a></div></div><div id="_co_idm139960898352608"><h3 class="no_margin">Jenefer M Blackwell</h3><p><sup>25</sup>Telethon Institute for Child Health Research, Centre for Child Health Research, University of Western Australia, Perth, Australia. </p><p><sup>26</sup>Genetics and Infection Laboratory, Cambridge Institute of Medical Research, Addenbrooke’s Hospital, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Blackwell%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Jenefer M Blackwell</a></div></div><div id="_co_idm139960991457296"><h3 class="no_margin">Elvira Bramon</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Bramon%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Elvira Bramon</a></div></div><div id="_co_idm139960992570576"><h3 class="no_margin">Suzannah J Bumpstead</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Bumpstead%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Suzannah J Bumpstead</a></div></div><div id="_co_idm139961076287936"><h3 class="no_margin">Juan P Casas</h3><p><sup>27</sup>Department of Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Casas%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Juan P Casas</a></div></div><div id="_co_idm139960998021232"><h3 class="no_margin">Aiden Corvin</h3><p><sup>28</sup>Neuropsychiatric Genetics Research Group, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland. </p><div>Find articles by <a href="/pubmed/?term=Corvin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Aiden Corvin</a></div></div><div id="_co_idm139960896091728"><h3 class="no_margin">Nicholas Craddock</h3><p><sup>29</sup>Department of Psychological Medicine, School of Medicine, Cardiff University, Cardiff, Wales. </p><div>Find articles by <a href="/pubmed/?term=Craddock%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Nicholas Craddock</a></div></div><div id="_co_idm139960892908384"><h3 class="no_margin">Panos Deloukas</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Panos Deloukas</a></div></div><div id="_co_idm139960897144208"><h3 class="no_margin">Serge Dronov</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Dronov%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Serge Dronov</a></div></div><div id="_co_idm139960883908528"><h3 class="no_margin">Audrey Duncanson</h3><p><sup>30</sup>Molecular and Physiological Sciences, The Wellcome Trust, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Duncanson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Audrey Duncanson</a></div></div><div id="_co_idm139960880789120"><h3 class="no_margin">Sarah Edkins</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Edkins%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Sarah Edkins</a></div></div><div id="_co_idm139960897427552"><h3 class="no_margin">Colin Freeman</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Freeman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Colin Freeman</a></div></div><div id="_co_idm139960887971872"><h3 class="no_margin">Matthew Gillman</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Gillman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Matthew Gillman</a></div></div><div id="_co_idm139960997354272"><h3 class="no_margin">Emma Gray</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Gray%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Emma Gray</a></div></div><div id="_co_idm139960882935104"><h3 class="no_margin">Rhian Gwilliam</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Gwilliam%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Rhian Gwilliam</a></div></div><div id="_co_idm139960891527120"><h3 class="no_margin">Naomi Hammond</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Hammond%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Naomi Hammond</a></div></div><div id="_co_idm139960985429040"><h3 class="no_margin">Sarah E Hunt</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Hunt%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Sarah E Hunt</a></div></div><div id="_co_idm139960891068048"><h3 class="no_margin">Janusz Jankowski</h3><p><sup>31</sup>Centre for Gastroenterology, Bart’s and the London School of Medicine and Dentistry, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Jankowski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Janusz Jankowski</a></div></div><div id="_co_idm139960898458912"><h3 class="no_margin">Alagurevathi Jayakumar</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Jayakumar%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Alagurevathi Jayakumar</a></div></div><div id="_co_idm139960883137808"><h3 class="no_margin">Cordelia Langford</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Langford%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Cordelia Langford</a></div></div><div id="_co_idm139960894135952"><h3 class="no_margin">Jennifer Liddle</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Liddle%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Jennifer Liddle</a></div></div><div id="_co_idm139960892313520"><h3 class="no_margin">Hugh S Markus</h3><p><sup>32</sup>Clinical Neurosciences, St. George’s University of London, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Markus%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Hugh S Markus</a></div></div><div id="_co_idm139960882066224"><h3 class="no_margin">Christopher G Mathew</h3><p><sup>33</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Mathew%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Christopher G Mathew</a></div></div><div id="_co_idm139960896309856"><h3 class="no_margin">Owen T McCann</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=McCann%20OT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Owen T McCann</a></div></div><div id="_co_idm139960880169392"><h3 class="no_margin">Mark I McCarthy</h3><p><sup>34</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, Churchill Hospital, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Mark I McCarthy</a></div></div><div id="_co_idm139960877562752"><h3 class="no_margin">Colin N A Palmer</h3><p><sup>35</sup>Biomedical Research Centre, Ninewells Hospital and Medical School, Dundee, UK. </p><div>Find articles by <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Colin N A Palmer</a></div></div><div id="_co_idm139960992409344"><h3 class="no_margin">Leena Peltonen</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Leena Peltonen</a></div></div><div id="_co_idm139960902207408"><h3 class="no_margin">Robert Plomin</h3><p><sup>36</sup>Social, Genetic and Developmental Psychiatry Centre, King’s College London Institute of Psychiatry, Denmark Hill, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Plomin%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Robert Plomin</a></div></div><div id="_co_idm139960994141568"><h3 class="no_margin">Simon C Potter</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Potter%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Simon C Potter</a></div></div><div id="_co_idm139960884243920"><h3 class="no_margin">Anna Rautanen</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Rautanen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Anna Rautanen</a></div></div><div id="_co_idm139960889897056"><h3 class="no_margin">Radhi Ravindrarajah</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Ravindrarajah%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Radhi Ravindrarajah</a></div></div><div id="_co_idm139960900945792"><h3 class="no_margin">Michelle Ricketts</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Ricketts%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Michelle Ricketts</a></div></div><div id="_co_idm139960889147200"><h3 class="no_margin">Nilesh Samani</h3><p><sup>37</sup>Department of Cardiovascular Science, University of Leicester, Glenfield General Hospital, Leicester, UK. </p><div>Find articles by <a href="/pubmed/?term=Samani%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Nilesh Samani</a></div></div><div id="_co_idm139960878773920"><h3 class="no_margin">Stephen J Sawcer</h3><p><sup>38</sup>University of Cambridge, Department of Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Sawcer%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Stephen J Sawcer</a></div></div><div id="_co_idm139960894567360"><h3 class="no_margin">Amy Strange</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Strange%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Amy Strange</a></div></div><div id="_co_idm139960882654336"><h3 class="no_margin">Richard C Trembath</h3><p><sup>33</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Trembath%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Richard C Trembath</a></div></div><div id="_co_idm139960894564800"><h3 class="no_margin">Ananth C Viswanathan</h3><p><sup>39</sup>Glaucoma Research Unit, Moorfields Eye Hospital NHS Foundation Trust, London, UK. </p><p><sup>40</sup>Department of Genetics, University College London Institute of Ophthalmology, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Viswanathan%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Ananth C Viswanathan</a></div></div><div id="_co_idm139960902190064"><h3 class="no_margin">Matthew Waller</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Waller%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Matthew Waller</a></div></div><div id="_co_idm139960996215456"><h3 class="no_margin">Paul Weston</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Weston%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Paul Weston</a></div></div><div id="_co_idm139960920662304"><h3 class="no_margin">Pamela Whittaker</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Whittaker%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Pamela Whittaker</a></div></div><div id="_co_idm139960992931024"><h3 class="no_margin">Sara Widaa</h3><p><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </p><div>Find articles by <a href="/pubmed/?term=Widaa%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Sara Widaa</a></div></div><div id="_co_idm139960981484976"><h3 class="no_margin">Nicholas W Wood</h3><p><sup>41</sup>Department of Molecular Neuroscience, Institute of Neurology, Queen Square, London, UK. </p><div>Find articles by <a href="/pubmed/?term=Wood%20NW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Nicholas W Wood</a></div></div><div id="_co_idm139960894488032"><h3 class="no_margin">Gilean McVean</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=McVean%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Gilean McVean</a></div></div><div id="_co_idm139960878383456"><h3 class="no_margin">John D Reveille</h3><p><sup>42</sup>Rheumatology and Clinical Immunogenetics, University of Texas Health Science Center at Houston, Houston, Texas, USA. </p><div>Find articles by <a href="/pubmed/?term=Reveille%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">John D Reveille</a></div></div><div id="_co_idm139960880972000"><h3 class="no_margin">B Paul Wordsworth</h3><p><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Wordsworth%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">B Paul Wordsworth</a></div></div><div id="_co_idm139960991224304"><h3 class="no_margin">Matthew A Brown</h3><p><sup>8</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </p><div>Find articles by <a href="/pubmed/?term=Brown%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Matthew A Brown</a></div></div><div id="_co_idm139960889860704"><h3 class="no_margin">Peter Donnelly</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </p><div>Find articles by <a href="/pubmed/?term=Donnelly%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21743469">Peter Donnelly</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139960885809568_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139960885809568_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139960885809568_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Medical Research Council (MRC) Centre for Causal Analyses in Translational Epidemiology, School of Social and Community Medicine, University of Bristol, Bristol, UK. </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>National Institute for Health Research Musculoskeletal Biomedical Research Unit, Nuffield Orthopaedic Centre, Headington, Oxford, UK. </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Structural Genomics Consortium, University of Oxford, Oxford, UK. </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>University of Bath, Bath, UK. </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Statistics, University of Oxford, Oxford, UK. </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Clinical Pharmacology, University of Oxford, Oxford, UK. </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, Brisbane, Australia. </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>National Institute of Arthritis and Musculoskeletal and Skin Diseases, US National Institutes of Health (NIH), Bethesda, Maryland, USA. </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Medicine/Rheumatology, Cedars-Sinai Medical Centre, Los Angeles, California, USA. </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>University of Toronto, Toronto, Ontario, Canada. </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Medicine, University of Alberta, Edmonton, Alberta, Canada. </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Memorial University, St. John’s, Newfoundland, Canada. </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>National Institute for Health Research (NIHR)–Leeds Musculoskeletal Biomedical Research Unit, University of Leeds, Leeds, UK. </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Rheumatology, Norfolk &amp; Norwich University Hospital, Norfolk, UK. </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Medicine, University of Cambridge, Addenbrookes Hospital, Cambridge, UK. </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Repatriation General Hospital, Adelaide, Australia. </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Serviço Especializado De Epidemiologia E Biologia Molecular, Hospital de Santo Espírito, Angra do Heroísmo, Terceira, The Azores, Portugal. </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Genetics and Arthritis Research Group, Institute for Molecular and Cell Biology (IBMC), University of Porto, Porto, Portugal. </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Department of Biology and Biotechnology ‘Charles Darwin’, Sapienza University of Rome, Rome, Italy. </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Queensland Institute of Medical Research, Brisbane, Australia. </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Rheumatology and Immunology, Shanghai Changzheng Hospital, The Second Military Medical University Hospital, Shanghai, China. </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Department of Immunology, Hospital Universitario Central de Asturias, Oviedo, Spain. </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Telethon Institute for Child Health Research, Centre for Child Health Research, University of Western Australia, Perth, Australia. </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Genetics and Infection Laboratory, Cambridge Institute of Medical Research, Addenbrooke’s Hospital, Cambridge, UK. </div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Department of Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, London, UK. </div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Neuropsychiatric Genetics Research Group, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland. </div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Department of Psychological Medicine, School of Medicine, Cardiff University, Cardiff, Wales. </div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Molecular and Physiological Sciences, The Wellcome Trust, London, UK. </div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Centre for Gastroenterology, Bart’s and the London School of Medicine and Dentistry, London, UK. </div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Clinical Neurosciences, St. George’s University of London, London, UK. </div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK. </div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, Churchill Hospital, Oxford, UK. </div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>Biomedical Research Centre, Ninewells Hospital and Medical School, Dundee, UK. </div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>Social, Genetic and Developmental Psychiatry Centre, King’s College London Institute of Psychiatry, Denmark Hill, London, UK. </div><div class="fm-affl" id="A37" lang="en"><sup>37</sup>Department of Cardiovascular Science, University of Leicester, Glenfield General Hospital, Leicester, UK. </div><div class="fm-affl" id="A38" lang="en"><sup>38</sup>University of Cambridge, Department of Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK. </div><div class="fm-affl" id="A39" lang="en"><sup>39</sup>Glaucoma Research Unit, Moorfields Eye Hospital NHS Foundation Trust, London, UK. </div><div class="fm-affl" id="A40" lang="en"><sup>40</sup>Department of Genetics, University College London Institute of Ophthalmology, London, UK. </div><div class="fm-affl" id="A41" lang="en"><sup>41</sup>Department of Molecular Neuroscience, Institute of Neurology, Queen Square, London, UK. </div><div class="fm-affl" id="A42" lang="en"><sup>42</sup>Rheumatology and Clinical Immunogenetics, University of Texas Health Science Center at Houston, Houston, Texas, USA. </div><div id="CR1"> Correspondence should be addressed to P. Donnelly (<a href="mailto:dev@null" data-email="ku.ca.xo.llew@yllennod.retep" class="oemail">ku.ca.xo.llew@yllennod.retep</a>) or M.A.B. (<a href="mailto:dev@null" data-email="ua.ude.qu@nworb.ttam" class="oemail">ua.ude.qu@nworb.ttam</a>). </div><div id="FN1"><sup>43</sup>These authors contributed equally to this work.</div><div id="FN2"><sup>44</sup>These authors jointly directed this work.</div><div id="FN3"><sup>1</sup>A full list of members is provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139960885809568_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21743469" target="pmc_ext" ref="reftype=publisher&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div><strong>This article has been corrected. </strong>See the correction in volume 43 on&nbsp;page&nbsp;919.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3640413/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental data.</strong></div><div><a href="/pmc/articles/PMC3640413/bin/NIHMS405164-supplement-Supplemental_data.pdf" data-ga-action="click_feat_suppl">NIHMS405164-supplement-Supplemental_data.pdf</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID:&nbsp;B9AE3C3C-D00B-452F-8DC9-527384EA6C73</div></div></dd></dl></div><div id="idm139960882944128" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139960882944128title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">Ankylosing spondylitis is a common form of inflammatory arthritis predominantly affecting the spine and pelvis that occurs in approximately 5 out of 1,000 adults of European descent. Here we report the identification of three variants in the <em>RUNX3, LTBRTNFRSF1A</em> and <em>IL12B</em> regions convincingly associated with ankylosing spondylitis (<em>P</em> &lt; 5 × 10<sup>−8</sup> in the combined discovery and replication datasets) and a further four loci at <em>PTGER4, TBKBP1, ANTXR2</em> and <em>CARD9</em> that show strong association across all our datasets (<em>P</em> &lt; 5 × 10<sup>−6</sup> overall, with support in each of the three datasets studied). We also show that polymorphisms of <em>ERAP1</em>, which encodes an endoplasmic reticulum aminopeptidase involved in peptide trimming before HLA class I presentation, only affect ankylosing spondylitis risk in HLA-B27–positive individuals. These findings provide strong evidence that HLA-B27 operates in ankylosing spondylitis through a mechanism involving aberrant processing of antigenic peptides.</p></div></div><div id="idm139961038416544" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Ankylosing spondylitis is the prototypic spondyloarthropathy, one of a group of conditions which also includes psoriatic arthritis, reactive arthritis and arthritis complicating inflammatory bowel disease. Ankylosing spondylitis is highly heritable (heritability &gt;90%)<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056010">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056020">2</a></sup> and familial (sibling recurrence risk ratio &gt;52)<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056014">3</a></sup>. Men are affected 2–3 times more frequently than women. The disease is strongly associated with HLA-B27; however, only 1–5% of HLA-B27–positive individuals develop ankylosing spondylitis, and there is strong epidemiologic evidence to suggest that other genes are involved<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056024">2</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056021">5</a></sup>. Spondyloarthropathy is characterized histopathologically by inflammation involving the attachments of tendons and ligaments to bone (entheses). This initial enthesitis is followed by bone formation, leading to fusion (ankylosis) of the affected joints. Effective treatments are available to suppress the inflammatory component of ankylosing spondylitis, but no treatment has yet been shown to improve its long-term outcome, particularly the loss of movement caused by bony ankylosis. In addition to its strong association with HLA-B27, ankylosing spondylitis has been convincingly associated with loci containing the genes <em>IL23R</em>, <em>ERAP1</em> and <em>KIF21B</em> and with the 2p15 and 21q22 intergenic regions<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056005">6</a>–<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056002">8</a></sup>.</p><p id="P6" class="p">In the current study, as part of the Wellcome Trust Case Control Consortium 2 (WTCCC2), we performed a genome-wide association study of 1,787 British and Australian affected individuals (cases) of European ancestry fulfilling the modified New York criteria for ankylosing spondylitis<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055987">9</a></sup> and 4,800 common controls, following quality control steps. We genotyped cases with the Illumina 660W-Quad microarray and genotyped controls with custom Illumina Human 1.2M-Duo chips. We combined these data with the findings of non-overlapping cases and controls from the Australo-Anglo-American Spondyloarthritis Consortium (TASC)<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055971">6</a></sup>, establishing a discovery set comprising 3,023 cases and 8,779 controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Online Methods</a>). We then performed a replication study in an independent cohort including 2,111 cases and 4,483 controls from Australia, Great Britain and The Spondyloarthritis Research Consortium of Canada (SPARCC).</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Primary association findings</h3><p id="P7" class="p p-first">We saw genome-wide significant association (<em>P</em> &lt; 5 × 10<sup>−8</sup> in the combined discovery and replication datasets) at SNPs close to or within previously established disease-associated loci, including <em>HLA-B</em>, <em>ERAP1</em>, <em>IL23R</em>, <em>KIF21B</em> and the gene deserts 2p15 and 21q22 (<a href="/pmc/articles/PMC3640413/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>), as well as at three new loci: <em>RUNX3</em>, <em>IL12B</em> and <em>LTBR-TNFRSF1A</em> (<a href="/pmc/articles/PMC3640413/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We also found strong evidence of association for SNPs in or near <em>ANTXR2</em> (<em>P</em> = 9.4 × 10<sup>−8</sup>), <em>CARD9</em> (<em>P</em> = 1.1 × 10<sup>−6</sup>), <em>PTGER4</em> (<em>P</em> = 2.6 × 10<sup>−7</sup>) and <em>TBKBP1</em> (<em>P</em> = 5.3 × 10<sup>−8</sup>), with support in all three datasets (<a href="/pmc/articles/PMC3640413/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 1</a>–<a href="#SD1" rid="SD1" class=" supplementary-material">3</a>). SNPs around two of these loci, <em>ANTXR2</em> and <em>CARD9</em>, have shown strong evidence of association in previous studies<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055983">6</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_344058593">10</a></sup>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Association evidence at loci previous identified as influencing ankylosing spondylitis risk</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960889486256" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="3" rowspan="1" valign="middle" align="right">WTCCC2 discovery sample<hr></th><th colspan="3" rowspan="1" valign="middle" align="right">TASC discovery sample<hr></th><th colspan="3" rowspan="1" valign="middle" align="right">Replication sample<hr></th><th rowspan="1" colspan="1" valign="middle" align="right">Combined</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="left">SNP ID</th><th rowspan="1" colspan="1" valign="middle" align="right">Position<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="left">Putative<br>gene</th><th rowspan="1" colspan="1" valign="middle" align="center">Risk<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="right">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="right">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="right">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="left">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="right">
<em>P</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">1p31</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11209026</td><td rowspan="1" colspan="1" valign="middle" align="right">67,478,546</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>IL23R</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="right">1.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.48</td><td rowspan="1" colspan="1" valign="middle" align="center">1.26–1.73</td><td rowspan="1" colspan="1" valign="middle" align="right">2.8 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.90</td><td rowspan="1" colspan="1" valign="middle" align="center">1.49–2.43</td><td rowspan="1" colspan="1" valign="middle" align="right">4.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.61</td><td rowspan="1" colspan="1" valign="middle" align="left">1.34–1.93</td><td rowspan="1" colspan="1" valign="middle" align="right">2.3 × 10<sup>−17</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1q32</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2297909</td><td rowspan="1" colspan="1" valign="middle" align="right">199,226,930</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>KIF21B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="right">0.0017</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14</td><td rowspan="1" colspan="1" valign="middle" align="center">1.05–1.24</td><td rowspan="1" colspan="1" valign="middle" align="right">0.00034</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09–1.35</td><td rowspan="1" colspan="1" valign="middle" align="right">1.8 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.25</td><td rowspan="1" colspan="1" valign="middle" align="left">1.15–1.36</td><td rowspan="1" colspan="1" valign="middle" align="right">5.2 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2p15</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10865331</td><td rowspan="1" colspan="1" valign="middle" align="right">62,404,976</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="right">2.9 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.32</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22–1.43</td><td rowspan="1" colspan="1" valign="middle" align="right">1.7 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.34</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22–1.47</td><td rowspan="1" colspan="1" valign="middle" align="right">2.5 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.36</td><td rowspan="1" colspan="1" valign="middle" align="left">1.26–1.47</td><td rowspan="1" colspan="1" valign="middle" align="right">6.5 × 10<sup>−34</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5q15</td><td rowspan="1" colspan="1" valign="middle" align="left">rs30187</td><td rowspan="1" colspan="1" valign="middle" align="right">96,150,086</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>ERAP1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="right">4.3 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.35</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24–1.46</td><td rowspan="1" colspan="1" valign="middle" align="right">7.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.31</td><td rowspan="1" colspan="1" valign="middle" align="center">1.19–1.44</td><td rowspan="1" colspan="1" valign="middle" align="right">1.1 × 10<strong>−9</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">1.28</td><td rowspan="1" colspan="1" valign="middle" align="left">1.18–1.39</td><td rowspan="1" colspan="1" valign="middle" align="right">1.8 × 10<sup>−27</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">6p21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4349859</td><td rowspan="1" colspan="1" valign="middle" align="right">31,473,766</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>HLA-B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="right">&lt;10<sup>−200</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">48.9</td><td rowspan="1" colspan="1" valign="middle" align="center">43.3–55.2</td><td rowspan="1" colspan="1" valign="middle" align="right">&lt;10<sup>−200</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">90.4</td><td rowspan="1" colspan="1" valign="middle" align="center">72.3–113</td><td rowspan="1" colspan="1" valign="middle" align="right">&lt;10<sup>−200</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">40.8</td><td rowspan="1" colspan="1" valign="middle" align="left">35.0–47.6</td><td rowspan="1" colspan="1" valign="middle" align="right">&lt;10<sup>−200</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">21q22</td><td rowspan="1" colspan="1" valign="middle" align="left">rs378108</td><td rowspan="1" colspan="1" valign="middle" align="right">39,391,390</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="right">1.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.23</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14–1.33</td><td rowspan="1" colspan="1" valign="middle" align="right">2.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.25</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14–1.38</td><td rowspan="1" colspan="1" valign="middle" align="right">0.059</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08</td><td rowspan="1" colspan="1" valign="middle" align="left">1.00–1.16</td><td rowspan="1" colspan="1" valign="middle" align="right">2.1 × 10<sup>−11</sup></td></tr></tbody></table></div><div id="largeobj_idm139960889486256" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3640413/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P51" class="p p-first-last">See text for references. HLA-B27 status is sometimes used as a diagnostic tool (especially in the TASC data), and this ascertainment effect means that estimates of its odds ratio (OR) may be biased upwards. Chr., chromosome.</p></div><div id="TFN2"><sup>a</sup>NCBI human genome build 36 coordinates.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Newly confirmed associations (upper tier) and those with strong and replicated evidence for association (lower tier)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960883638384" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="3" rowspan="1" valign="middle" align="left">WTCCC2 discovery sample<hr></th><th colspan="3" rowspan="1" valign="middle" align="right">TASC discovery sample<hr></th><th colspan="3" rowspan="1" valign="middle" align="right">Replication sample<hr></th><th rowspan="1" colspan="1" valign="middle" align="left">Combined</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="left">SNP ID</th><th rowspan="1" colspan="1" valign="middle" align="right">Position<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="left">Putative gene</th><th rowspan="1" colspan="1" valign="middle" align="center">Risk<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="right">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="right">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="left">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>P</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">1p36</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11249215</td><td rowspan="1" colspan="1" valign="middle" align="right">25,169,771</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>RUNX3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="left">2.1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.19</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10–1.28</td><td rowspan="1" colspan="1" valign="middle" align="right">0.00091</td><td rowspan="1" colspan="1" valign="middle" align="center">1.19</td><td rowspan="1" colspan="1" valign="middle" align="center">1.07–1.32</td><td rowspan="1" colspan="1" valign="middle" align="right">0.00029</td><td rowspan="1" colspan="1" valign="middle" align="left">1.15</td><td rowspan="1" colspan="1" valign="middle" align="center">1.07–1.24</td><td rowspan="1" colspan="1" valign="middle" align="left">9.2 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5q33</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6556416</td><td rowspan="1" colspan="1" valign="middle" align="right">158,751,323</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>IL12B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">1.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.18</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08–1.28</td><td rowspan="1" colspan="1" valign="middle" align="right">0.074</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11</td><td rowspan="1" colspan="1" valign="middle" align="center">0.99–1.23</td><td rowspan="1" colspan="1" valign="middle" align="right">0.00018</td><td rowspan="1" colspan="1" valign="middle" align="left">1.17</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08–1.27</td><td rowspan="1" colspan="1" valign="middle" align="left">1.9 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12p13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11616188</td><td rowspan="1" colspan="1" valign="middle" align="right">6,373,003</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LTBR-</em>
<br>
<em>TNFRSF1A</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="left">1.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.21</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11–1.31</td><td rowspan="1" colspan="1" valign="middle" align="right">6.2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.38</td><td rowspan="1" colspan="1" valign="middle" align="center">1.23–1.56</td><td rowspan="1" colspan="1" valign="middle" align="right">0.0036</td><td rowspan="1" colspan="1" valign="middle" align="left">1.12</td><td rowspan="1" colspan="1" valign="middle" align="center">1.04–1.21</td><td rowspan="1" colspan="1" valign="middle" align="left">4.1 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">4q21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4389526</td><td rowspan="1" colspan="1" valign="middle" align="right">81,165,499</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>ANTXR2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00054</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15</td><td rowspan="1" colspan="1" valign="middle" align="center">1.06–1.25</td><td rowspan="1" colspan="1" valign="middle" align="right">0.00027</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09–1.34</td><td rowspan="1" colspan="1" valign="middle" align="right">0.019</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1 0</td><td rowspan="1" colspan="1" valign="middle" align="center">1.02–1.19</td><td rowspan="1" colspan="1" valign="middle" align="left">9.4 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5p13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10440635</td><td rowspan="1" colspan="1" valign="middle" align="right">40,526,547</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>PTGER4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0027</td><td rowspan="1" colspan="1" valign="middle" align="center">1.13</td><td rowspan="1" colspan="1" valign="middle" align="center">1.04–1.22</td><td rowspan="1" colspan="1" valign="middle" align="right">0.00020</td><td rowspan="1" colspan="1" valign="middle" align="center">1.20</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09–1.32</td><td rowspan="1" colspan="1" valign="middle" align="right">0.017</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1 0</td><td rowspan="1" colspan="1" valign="middle" align="center">1.02–1.19</td><td rowspan="1" colspan="1" valign="middle" align="left">2.6 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">9q34</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10781500</td><td rowspan="1" colspan="1" valign="middle" align="right">138,389,159</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CARD9</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="left">2.0 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.18</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09–1.28</td><td rowspan="1" colspan="1" valign="middle" align="right">0.043</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00–1.22</td><td rowspan="1" colspan="1" valign="middle" align="right">0.041</td><td rowspan="1" colspan="1" valign="middle" align="left">1.08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00–1.17</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">17q21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs8070463</td><td rowspan="1" colspan="1" valign="middle" align="right">43,123,835</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>TBKBP1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0025</td><td rowspan="1" colspan="1" valign="middle" align="center">1.13</td><td rowspan="1" colspan="1" valign="middle" align="center">1.04–1.22</td><td rowspan="1" colspan="1" valign="middle" align="right">1.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.24</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12–1.37</td><td rowspan="1" colspan="1" valign="middle" align="right">0.015</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1 0</td><td rowspan="1" colspan="1" valign="middle" align="center">1.02–1.18</td><td rowspan="1" colspan="1" valign="middle" align="left">5.3 × 10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm139960883638384" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3640413/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="P53" class="p p-first-last">Top tier SNPs have a combined <em>P</em> &lt; 5 × 10<sup>−8</sup> and lower tier SNPs have combined <em>P</em> ≤ 5 × 10<sup>−6</sup>. Chr., chromosome.</p></div><div id="TFN4"><sup>a</sup>NCBI human genome build 36 coordinates.</div></div></div><p id="P8">To investigate the new associations more closely, we performed further statistical analysis and functional work on <em>RUNX3</em> and <em>PTGER4</em>. <em>RUNX3</em> encodes a transcription factor critically involved in CD8 lymphocyte differentiation<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055981">11</a></sup>. Expression of <em>RUNX3</em> is driven by IL-7 (<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055996">ref. 11</a></sup>); in the current study, we saw moderate association with SNPs in <em>IL7R</em> (rs13172042, <em>P</em> = 8.0 × 10<sup>−5</sup> and rs1993879, <em>P</em> = 9.0 × 10<sup>−5</sup>). In a previous genome-wide association study of peripheral blood cell subset counts, SNPs in <em>RUNX3</em> that were associated with ankylosing spondylitis in the current study also showed association with decreased CD8 lymphocyte counts (rs10903118, <em>P</em> = 0.0037, determining 0.42% of CD8 count variance and rs11249215, <em>P</em> = 0.0042, determining 0.41% of CD8 count variance)<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056008">12</a></sup>. Consistent with this observation, we observed that ankylosing spondylitis cases on no biological therapy have lower CD8 lymphocyte counts than age-matched controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>), suggesting that the <em>RUNX3</em> association with ankylosing spondylitis may operate through an effect on CD8 lymphocyte counts.</p><p id="P9" class="p p-last">PTGER4 is a component of the ‘mechanostat’ anabolic bone response to physical stress, including at the site of insertion of tendons and ligaments into bone (entheses), a site characteristically involved in ankylosing spondylitis. We therefore tested expression of <em>PTGER4</em> in synovial biopsy samples from spondyloarthritis cases compared with controls (either healthy or with non-inflammatory arthritis) and found increased expression in spondyloarthritis samples (1.9-fold change, <em>P</em> = 0.04, two-tailed test).</p></div><div id="S3" class="sec"><h3 id="S3title">Secondary signals at disease-associated loci</h3><p id="P10" class="p p-first">To look for regions where the pattern of association is not sufficiently characterized by the most significant SNP, we ran the program GENECLUSTER on the WTCCC2 data<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup>. Two loci, <em>ERAP1</em> and <em>IL23R</em>, showed strong evidence for a second SNP; the Bayes factor, which is the ratio of the probability of the data under a two mutation model relative to a one mutation model<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056016">14</a></sup>, was 12.3 at <em>ERAP1</em> and 28.8 at <em>IL23R</em>.</p><p id="P11" class="p p-last">At both loci, the pattern of association was consistent with a primary effect at the marginally most significant SNP (rs30187 at <em>ERAP1</em> and rs11209026 at <em>IL23R</em>) and a secondary effect at a nearby SNP (rs10050860 or rs17482078, two SNPs with <em>r</em><sup>2</sup> = 1, at <em>ERAP1</em> and rs11209032 at <em>IL23R</em>). Frequentist analysis conditioning on the primary SNP confirmed the secondary SNP as explaining a significant additional fraction of risk at both loci in both the discovery and replication data (<em>IL23R</em>, rs11209032, WTCCC2 <em>P</em> = 7.9 × 10<sup>−6</sup>, TASC <em>P</em> = 0.0018, replication <em>P</em> = 0.0018; <em>ERAP1</em>, rs10050860, WTCCC2 <em>P</em> = 0.0025, TASC <em>P</em> = 0.026, replication <em>P</em> = 0.00023; <a href="/pmc/articles/PMC3640413/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>). Around <em>IL23R</em>, the minor allele at rs11209026 (which is inferred to be non-ancestral) is strongly protective, whereas the minor allele of rs11209032 increases disease risk. In <em>ERAP1</em>, the major allele of rs30187 and the minor allele of rs10050860 are protective (and both are inferred to be non-ancestral). In combination, the protective effect was particularly strong; HLA-B27–positive individuals homozygous for protective variants at rs30187 and rs10050860 had approximately 3–4 times lower disease risk than HLA-B27–positive individuals with risk predisposing alleles at these loci (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p6">Evidence for secondary signals of association at <em>IL23R</em>, <em>ERAP1</em> and <em>HLA-B27</em></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960900865872" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="3" rowspan="1" valign="middle" align="right">WTCCC2 discovery sample<hr></th><th colspan="3" rowspan="1" valign="middle" align="right">TASC discovery sample<hr></th><th colspan="3" rowspan="1" valign="middle" align="left">Replication sample<hr></th><th rowspan="1" colspan="1" valign="middle" align="left">Combined</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="center">SNP ID</th><th rowspan="1" colspan="1" valign="middle" align="center">Position<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="left">Conditional<br>on SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Effect<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="right">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="right">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="left">
<em>P</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">1p31</td><td rowspan="1" colspan="1" valign="middle" align="center">rs11209032</td><td rowspan="1" colspan="1" valign="middle" align="center">67,512,680</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11209026</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="right">7.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.20</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11–1.30</td><td rowspan="1" colspan="1" valign="middle" align="right">0.0018</td><td rowspan="1" colspan="1" valign="middle" align="center">1.17</td><td rowspan="1" colspan="1" valign="middle" align="center">1.06–1.29</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0018</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14</td><td rowspan="1" colspan="1" valign="middle" align="center">1.05–1.23</td><td rowspan="1" colspan="1" valign="middle" align="left">5.1 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">5q15</td><td rowspan="1" colspan="1" valign="middle" align="center">rs10050860</td><td rowspan="1" colspan="1" valign="middle" align="center">96,147,966</td><td rowspan="1" colspan="1" valign="middle" align="left">rs30187</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="right">0.0025</td><td rowspan="1" colspan="1" valign="middle" align="center">1.17</td><td rowspan="1" colspan="1" valign="middle" align="center">1.06–1.30</td><td rowspan="1" colspan="1" valign="middle" align="right">0.026</td><td rowspan="1" colspan="1" valign="middle" align="center">1.16</td><td rowspan="1" colspan="1" valign="middle" align="center">1.02–1.33</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00023</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10–1.35</td><td rowspan="1" colspan="1" valign="middle" align="left">1.9 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">6p21</td><td rowspan="1" colspan="1" valign="middle" align="center">rs13202464</td><td rowspan="1" colspan="1" valign="middle" align="center">31,452,562</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4349859<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="right">3.7 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.32</td><td rowspan="1" colspan="1" valign="middle" align="center">0.21–0.47</td><td rowspan="1" colspan="1" valign="middle" align="right">2.1 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.15</td><td rowspan="1" colspan="1" valign="middle" align="center">0.09–0.23</td><td rowspan="1" colspan="1" valign="middle" align="left">2.0 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.34</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25–0.47</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1 × 10<sup>−29</sup></td></tr></tbody></table></div><div id="largeobj_idm139960900865872" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3640413/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><sup>a</sup>NCBI human genome build 36 coordinates.</div><div id="TFN6"><sup>b</sup>We assumed a dominant model for the risk allele at this SNP.</div></div></div></div><div id="S4" class="sec"><h3 id="S4title">Major histocompatibility complex (MHC) studies</h3><p id="P12" class="p p-first">In addition to our association analyses, we tested genotyped and imputed SNPs across the MHC for their ability to tag HLA-B27. We identified one SNP, rs4349859, that strongly tagged <em>HLA-B27</em> and had high sensitivity (98.0%) and specificity (99.0%) in 538 cases and 741 controls of European origin from Australia and Great Britain (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). This SNP also showed the strongest association with ankylosing spondylitis in our study. A second SNP, rs13202464, showed slightly higher sensitivity (98.7%) but slightly lower specificity (98.0%) for <em>HLA-B27</em> than rs4349859. Further studies will be required to determine the value of typing both SNPs over and above rs4349859 alone to impute HLA-B27 genotype. Controlling for rs4349859 and rs1320464, there were some residual signals in the MHC, which could warrant further investigation, but no other SNP across the MHC achieved <em>P</em> &lt; 4.0 × 10<sup>−6</sup> in the combined discovery and replication datasets.</p><p id="P13" class="p p-last">rs4349859 lies 41-kb centromeric of <em>HLA-B</em> and 5.4-kb telomeric of <em>MICA</em>, which neighbors <em>HLA-B</em>. <em>MICA</em> has previously been associated with ankylosing spondylitis<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055997">15</a>,<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056017">16</a></sup>; in order to distinguish whether our finding was caused by linkage disequilibrium with HLA-B27 or whether it indicated a potential disease-causative variant itself, we tested the ability of rs4349859 to tag HLA-B27 subtypes with differential association with ankylosing spondylitis. These studies confirmed that rs4349859 tags all the major European-ancestry ankylosing spondylitis–associated subtypes, HLA-B*2702, HLA-B*2705 and HLA-B*2708 (<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056006">ref. 17</a></sup>), but also tags HLA-B*2709, which is not associated with ankylosing spondylitis<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055998">18</a></sup>. However, rs4349859 does not effectively tag the African ankylosing spondylitis–associated subtype HLA-B*2703 nor the Asian HLA-B27 subtypes HLA-B*2704, HLA-B*2706 or HLA-B*2707 (<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055969">ref. 19</a></sup>), although it does tag HLA-B*2705 in Asian carriers (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Further, neither rs4349859 nor rs1320464 was more sensitive or specific than HLA-B27 for ankylosing spondylitis in cases and controls typed for each polymorphism in the WTCCC2-TASC dataset (data not shown).</p></div><div id="S5" class="sec"><h3 id="S5title">Gene-gene interaction studies</h3><p id="P14" class="p p-first">We tested loci known to be associated with ankylosing spondylitis, including those identified in the current study, for gene-gene interaction (defined as a departure from additivity on the log-odds scale). We observed strong evidence for gene-gene interaction between HLA-B27 (as tagged by rs4349859, with a dominant model<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup>) and the nonsynonymous SNP (nsSNP) rs30187 in <em>ERAP1</em> in the WTCCC2, TASC and replication datasets (WTCCC2 <em>P</em> = 0.02, TASC <em>P</em> = 0.014, replication <em>P</em> = 0.0019, combined <em>P</em> = 7.3 × 10<sup>−6</sup>; see also <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). The interaction takes a particularly simple form: in HLA-B27–positive cases and controls, <em>ERAP1</em> was associated with ankylosing spondylitis, whereas we saw no association with <em>ERAP1</em> in HLA-B27–negative cases (<a href="/pmc/articles/PMC3640413/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). In contrast, SNPs at other risk loci, including <em>IL23R</em>, <em>KIF21B</em>, <em>IL12B</em> and the intergenic regions 2p15 and 21q22, were associated with ankylosing spondylitis in both HLA-B27–positive and HLA-B27–negative disease (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3640413/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960893946928" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-405164-f0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-405164-f0001.jpg" src="/pmc/articles/PMC3640413/bin/nihms-405164-f0001.jpg"></div></a><div id="largeobj_idm139960893946928" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3640413/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3640413/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3640413/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Association findings for the <em>ERAP1</em> SNP rs30187 stratified by the HLA-B27 tag SNP rs4349859. rs4349859 allele A tags HLA-B27. Error bars, 95% confidence intervals. Note that the <em>ERAP1</em> risk allele T only increases risk in individuals carrying at least one copy of the HLA-B27 risk allele tag. The odds ratios for the genotype combinations were calculated using logistic regression in the R software package including covariates for ancestry (where appropriate). Genotypes with the low risk CC/GG genotype were set as the baseline, and the other genotype combinations were coded according to a series of dichotomous indicator variables. Odds ratios were derived by exponentiating the relevant coefficient from the logistic regression.</p></div></div></div><p id="P15" class="p p-last">Odds ratios and 95% confidence intervals for each of the possible genotype combinations are presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>. We tested the robustness of the HLA-B27–<em>ERAP1</em> interaction by also fitting the full (but less powerful) 4-degree-of-freedom interaction test (<a href="#SD1" rid="SD1" class=" supplementary-material">Online Methods</a>). We observed evidence for interaction between rs4349859 and rs30187 in the WTCCC2 (WTCCC2 χ<sub>4</sub><sup>2</sup> = 9.77, <em>P</em> = 0.04) and replication cohorts (replication χ<sup>2</sup> = 10.05 (4 degrees of freedom), <em>P</em> = 0.04), whereas the pattern of odds ratios in the TASC dataset indicated a trend in the same direction (TASC χ<sub>4</sub><sup>2</sup> = 7.03, <em>P</em> = 0.13). Combining the evidence across cohorts provided strong evidence for the existence of an interaction (combined χ<sub>4</sub><sup>2</sup> = 23.28, <em>P</em> = 0.0001), showing that our results are robust and are not likely to be an artifact.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Investigation of ERAP1 function</h3><p id="P16" class="p p-first">ERAP1 has two reported functions. First, it may act as a ‘molecular ruler’ within the endoplasmic reticulum, trimming peptides that have been partially processed by the proteosome (typically 9–17 amino acids in length) down to nine amino acids in length, the optimal length for HLA class I binding and presentation<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055986">21</a>,<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055977">22</a></sup>. Second, <em>in vitro</em> studies have suggested that ERAP1 may function as a ‘sheddase’, cleaving cytokine receptors off the cell surface, including IL-6R, IL-1R2 and TNFR<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056023">23</a>–<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055992">25</a></sup>. We tested the ability of ERAP1 to cleave TNFR and IL-6R from cell surfaces by measuring the appearance of these receptors in cell culture supernatants from single-cell suspensions prepared from <em>Erap1</em><sup>−/−</sup> and C57BL/6 control mouse spleens (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>). We observed no difference in the levels of these receptors over time, indicating that ERAP1 does not have a major influence on cytokine receptor trimming, at least in mice.</p><p id="P17" class="p p-last">In a preliminary <em>in vitro</em> examination of the effects of allelic variants on the function of ERAP1 in peptide trimming, we synthesized recombinant ERAP1 with the wild-type sequence or the ankylosing spondylitis–associated variants p.Lys528Arg (rs30187), p.Arg725Gln (rs17482078) or p.Asp575Asn (rs10050860) and tested their ability to trim an ERAP1 substrate<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055985">26</a></sup>. In the discovery dataset, the second SNP at <em>ERAP1</em>, rs10050860, was in complete linkage disequilibrium with rs17482078. Both the protective rs30187 and rs17482078 alleles had ~40% slower rates of substrate trimming than wild-type ERAP1 (<em>P</em> = 0.050, Wilcoxon rank sum test, one-tailed exact test), suggesting that both represent loss-of-function variants (<a href="/pmc/articles/PMC3640413/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>); the proteolytic rate of rs10050860 was similar to wild-type ERAP1.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3640413/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960996936336" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-405164-f0002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-405164-f0002.jpg" src="/pmc/articles/PMC3640413/bin/nihms-405164-f0002.jpg"></div></a><div id="largeobj_idm139960996936336" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3640413/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3640413/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3640413/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">The mean rate of trimming of N-terminal tryptophan (mol, substrate/mol, enzyme/sec) from 10-mer peptide WRVYEKCALK by wild-type ERAP1 and variants associated with ankylosing spondylitis (rs30187 (p.Lys528Arg), rs17482078 (p.Arg725Gln) and rs10050860 (p.Asp575Asn)). WT, wild type. Circles represent results for wild-type genotype samples, squares represent results for rs30187, upright triangles represent results for rs17482078 and inverted triangles refer to results for rs10050860.</p></div></div></div></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">DISCUSSION</h2><p id="P18" class="p p-first">This study increases the number of genetic loci convincingly associated with ankylosing spondylitis to nine (<em>IL23R</em>, <em>RUNX3</em>, <em>KIF21B</em>, 2p15, <em>IL12B</em>, <em>ERAP1</em>, <em>HLA-B</em>, <em>LTBR-TNFRSF1A</em> and 21q22), identifies a further four loci that show strong evidence of association and are likely to be involved in disease pathogenesis (<em>ANTXR2</em>, <em>PTGER4</em>, <em>CARD9</em> and <em>TBKBP1</em>) and greatly expands our understanding of the etiopathogenesis of the condition, including the likely mechanism by which HLA-B27 operates in the disease. These new genetic findings further implicate the involvement of the IL-23–IL-23R–IL-17 pathway in ankylosing spondylitis. Although further studies will be required to identify the true disease-associated variants, each associated locus contains candidate genes of strong biological interest in ankylosing spondylitis, which are discussed below. <em>IL12B</em> encodes IL12p40, which is a shared component of both IL-12 and IL-23. Association has previously been reported between SNPs in <em>IL23R</em> and ankylosing spondylitis<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055970">7</a></sup>. SNPs in or near <em>CARD9</em> and <em>PTGER4</em> (encoding prostaglandin E receptor 4, EP4 subtype) have also been associated with Crohn’s disease previously<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056003">27</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056018">28</a></sup>, and variants within <em>IL12B</em> and <em>IL23R</em> have been associated with both Crohn’s disease and psoriasis<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055975">29</a>–<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_482052563">31</a></sup>. PGE<sub>2</sub> acts through PTGER4 to induce the production of IL-23 and IL-17 and to promote expansion of T<sub>H</sub>17 lymphocyte counts; T<sub>H</sub>17 lymphocyte counts and IL-17 levels are both known to be elevated in ankylosing spondylitis<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055995">32</a></sup>. Consistent with this, we observed upregulation of <em>PTGER4</em> expression in spondyloarthritis synovium. CARD9 mediates signals from dectin-1 and -2, which are innate immunity receptors for β-glucan, a component of fungal and some bacterial cell walls. β-glucan induces PGE<sub>2</sub> production, which in turn promotes the expansion of T<sub>H</sub>17 cells<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055976">33</a></sup>. We and others have recently reported that SKG mice treated with β-glucan develop spondyloarthritis and Crohn’s disease<sup><a href="#R34" rid="R34" class=" bibr popnode">34</a></sup>; this mouse model is characterized by T<sub>H</sub>17 activation<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056011">35</a></sup>. These findings are thus consistent with a model of human ankylosing spondylitis in which ubiquitous pathogens carrying β-glucan induce disease by stimulating the IL-23–IL-23R–IL-17 pathway.</p><p id="P19">Despite considerable research since the discovery of the association of HLA-B27 with ankylosing spondylitis in the early 1970s, the mechanism explaining the involvement of HLA-B27 with the disease is unclear. Current theories can essentially be divided into those that favor canonical functions of HLA-B27 in peptide presentation (the ‘arthritogenic peptide’ theory) and those that invoke aberrant properties of HLA-B27 itself, including the propensity to form HLA-B27 homodimers and/or induce endoplasmic reticulum stress caused by slow and inefficient folding of nascent HLA-B27 molecules. A further possible hypothesis is that HLA-B27 acts as a tag for another, nearby, true susceptibility allele. Our findings that no other MHC SNP was more strongly associated with ankylosing spondylitis than rs4349859 or HLA-B27 indicate that this ‘linked gene’ theory is unlikely. The finding that rs4349859 does not tag HLA-B27 subtypes common in Asian populations but does tag HLA-B*2709, a European HLA-B27 subtype not associated with ankylosing spondylitis, indicates that rs4349859 is not disease causative in its own right. However, it does tag the ancestral HLA-B*2705 subtype, and other European HLA-B27 subtypes, with high sensitivity and specificity within the level of accuracy of HLA-B27 typing itself but at far lower cost and difficulty.</p><p id="P20">The interaction between HLA-B27 and <em>ERAP1</em> is also highly informative regarding potential mechanisms of disease in ankylosing spondylitis. The restriction of the association of <em>ERAP1</em> with ankylosing spondylitis to HLA-B27–positive cases is consistent with disease models in which aberrant trimming of peptides or presentation by ERAP1 and HLA-B27 are involved in the pathogenesis of HLA-B27–associated disease. HLA-B27–and ERAP1-negative disease is unlikely to be caused by a similar mechanism, and the fact that overexpression of TNF alone is sufficient to cause spondyloarthritis in mice<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055988">36</a></sup> suggests that overexpression or signaling by pro-inflammatory cytokines alone may be sufficient to cause ankylosing spondylitis. Although the findings of our studies of ERAP1 function using recombinant ERAP1 protein require much further experimental validation, they suggest that <em>ERAP1</em> influences disease risk in ankylosing spondylitis, with associated, non-ancestral alleles reducing the disease risk by a mechanism involving reduced peptide trimming and, likely as a consequence, altered peptide presentation by HLA-B27. This finding, and the strong protective effect of <em>ERAP1</em> variants in ankylosing spondylitis, suggest that inhibition of ERAP1 may be effective in ankylosing spondylitis, at least in HLA-B27–positive disease.</p><p id="P21">Modeling studies in ankylosing spondylitis<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055999">3</a></sup> and in many other common diseases<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055990">37</a>,<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055994">38</a></sup>, as well as findings in many plant and animal species, suggest that the best genetic models for common diseases involve gene-gene interaction. Very few well-replicated examples of gene-gene interaction in common polygenic diseases have been reported to date, making the finding of interaction between HLA-B27 and <em>ERAP1</em> in ankylosing spondylitis of particular interest. A similar observation was made in the overlapping WTCCC2 study of psoriasis<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_344058595">39</a></sup>, with gene-gene interaction observed between <em>ERAP1</em> and HLA-Cw6.</p><p id="P22">We present here the first convincing evidence of associations in HLA-B27–negative ankylosing spondylitis. HLA-B27–negative ankylosing spondylitis has very similar clinical presentation and disease course as HLA-B27–positive disease, with a similar natural history but with a later age of onset and lower incidence of uveitis<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055991">40</a>,<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055984">41</a></sup>. Few families have been reported with recurrence of ankylosing spondylitis in the absence of HLA-B27, no studies have examined the heritability of ankylosing spondylitis in its absence and no loci have previously been convincingly associated with ankylosing spondylitis in HLA-B27–negative cases, probably because only small studies have been performed addressing this. Our findings indicate both substantial overlap and differences in genetic susceptibility between HLA-B27–positive and –negative ankylosing spondylitis and suggest a shared etiopathogenesis.</p><p id="P23">This study provides further evidence of the involvement of TNF pathways in ankylosing spondylitis pathogenesis. TNF antagonists are highly effective in suppressing inflammation in ankylosing spondylitis, and mice overexpressing TNF develop spondyloarthritis<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055968">36</a></sup>. In the current study, we observed association with SNPs at chromosome 12p13 between <em>LTBR</em> (lymphotoxin beta receptor) and <em>TNFRSF1A</em> (tumor necrosis factor receptor 1 (TNFR1)) (rs11616188, WTCCC2 <em>P</em> = 1.2 × 10<sup>−5</sup>, TASC <em>P</em> = 6.2 × 10<sup>−8</sup>, replication <em>P</em> = 0.0036, combined <em>P</em> = 4.1 × 10<sup>−12</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). A mouse model with constitutive overexpression of TNF that develops inflammatory bowel disease and spondyloarthritis has been reported, in which the arthritis has been shown to be dependent on TNFR1 expression and in which TNFR1 expression in mesenchymal tissue alone is sufficient to permit disease<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056004">36</a></sup>. This is consistent with a role for TNFR1 in ankylosing spondylitis, but further studies will be required to determine if the genetic polymorphisms at this locus are associated with ankylosing spondylitis because of effects on <em>LTBR</em>, <em>TNFRSF1A</em> or both. We also observed association at chromosome 17q21 near <em>TBKBP1</em> (encoding TBK binding protein 1), a component of the TNF receptor signaling pathway (rs8070463, WTCCC2 <em>P</em> = 0.0025, TASC <em>P</em> = 1.6 × 10<sup>−5</sup>, replication <em>P</em> = 0.015, combined <em>P</em> = 5.3 × 10<sup>−8</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). <em>TRADD</em>, which has previously shown suggestive association<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055972">6</a>,<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056019">42</a></sup> and which also shows suggestive association with ankylosing spondylitis in this study encodes the TNF-receptor–associated death domain protein, also part of the TNF signaling pathway (rs9033, WTCCC2 <em>P</em> = 9.0 × 10<sup>−6</sup>, TASC <em>P</em> = 0.32, combined <em>P</em> = 4.9 × 10<sup>−5</sup>). Further research will be required to determine the precise mechanism underlying these associations.</p><p id="P24">The association of SNPs in and around <em>RUNX3</em> provides further support for the hypothesis that HLA-B27 is involved in ankylosing spondylitis by a mechanism involving peptide presentation. The canonical function of HLA class I molecules is to present antigen to CD8 lymphocytes. <em>RUNX3</em> is a key gene involved in CD8 lymphocyte differentiation, and we have shown here that ankylosing spondylitis cases have reduced CD8 lymphocyte counts and that the <em>RUNX3</em> SNPs associated with ankylosing spondylitis are also associated with reduced CD8 lymphocyte counts in healthy individuals. These findings are consistent with a key role for CD8 lymphocytes in ankylosing spondylitis pathogenesis, likely in response to antigenic peptides presented by HLA-B27.</p><p id="P25" class="p p-last">In conclusion, this study has identified further non-MHC genetic loci associated with ankylosing spondylitis. Our data strongly imply that HLA-B27 itself, rather than a nearby linked gene, is directly involved in ankylosing spondylitis susceptibility. We have also shown that HLA-B27–positive and –negative ankylosing spondylitis cases differ in association with <em>ERAP1</em>, which is involved in endoplasmic reticulum peptide processing before HLA class I assembly and peptide presentation. This represents one of the first convincingly replicated examples of gene-gene interaction in any common disease and indicates that the likely mechanism by which HLA-B27 induces ankylosing spondylitis involves aberrant presentation or handling of peptides.</p></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">ONLINE METHODS</h2><div id="S10" class="sec sec-first"><h3 id="S10title">Samples</h3><p id="P27" class="p p-first">For cases, ankylosing spondylitis was defined according to modified New York criteria<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056022">9</a></sup>. A total of 1,787 DNA samples from unrelated individuals of European ancestry were included in the GWAS discovery set, having passed genotyping control filters (see below and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>). The replication set was derived from individuals with ankylosing spondylitis recruited from the UK, Australia and Canada; 2,011 DNA samples passed quality control for replication. All case participants gave written, informed consent, and the study was approved by the relevant research ethics authorities at each participating center.</p><p id="P28" class="p p-last">For controls, a total of 4,800 individuals from the WTCCC2 common control set passed the quality control filters (see below) and were used in the discovery GWAS. This included 2,485 healthy blood donors from the UK Blood Service (UKBS) collection and 2,315 individuals from the 1958 Birth Cohort dataset (see below and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>). The number of replication control samples passing quality control totaled 4,483, comprising 2,831 from the People of the British Isles collection and 1,652 Canadian controls.</p></div><div id="S11" class="sec"><h3 id="S11title">GWAS genotyping</h3><p id="P29" class="p p-first-last">Following careful DNA quality control, including gender verification by genotyping in comparison with patient data manifests, samples were genotyped at the Wellcome Trust Sanger Institute on the Illumina Infinium platform. Samples from the case collection were genotyped on the Human660W-Quad (a custom chip designed by WTCCC2 and comprising Human550 and a set of ~6,000 common copy number variants from the Structural Variation Consortium<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056013">43</a></sup>). Samples from the control collections were genotyped on the Human1.2M-Duo (a WTCCC2 custom array comprising Human1M-Duo and the common copy number variant content described above). Bead intensity data was processed and normalized for each sample in BeadStudio; data for successfully genotyped samples was extracted and genotypes were called within collections using Illuminus<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055967">44</a></sup>.</p></div><div id="S12" class="sec"><h3 id="S12title">GWAS quality control</h3><p id="P30" class="p p-first">We identified and removed samples whose genome-wide patterns of diversity differed from those of the collection at large, interpreting them as likely to be caused by biases or artifacts. To do so, we used a Bayesian clustering approach to infer outlying individuals on the basis of call rate, heterozygosity, ancestry and average probe intensity. To obtain a set of putatively unrelated individuals, we used a Hidden Markov model to infer identify by descent along the genome and then iteratively removed individuals to obtain a set with pairwise identity by decent less than 5%. These methods are described elsewhere<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_344058594">39</a>,<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055982">45</a></sup>. Sample exclusions before imputation are outlined in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>.</p><p id="P31" class="p p-last">For SNPs, a measure of (Fisher) information for the allele frequency at each SNP was calculated using SNPTEST (see URLs). Autosomal SNPs were excluded if this information measure was below 0.975, if the minor allele frequency was less than 0.1%, if the SNP had &gt;2% missing data or if the Hardy-Weinberg <em>P</em> value was lower than 5 × 10<sup>−20</sup> in the case data or in either of the control collections.</p></div><div id="S13" class="sec"><h3 id="S13title">Imputation</h3><p id="P32" class="p p-first">For WTCCC2 data, genotypes were imputed using IMPUTE2 (<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055993">ref. 46</a></sup>), which adopts a two-stage approach using both haploid and diploid reference panels. For the haploid reference panel, we used HapMap2 and HapMap3 SNP data for the 120 non-related CEU trios (see URLs). For the diploid reference panel, we used 1958 Birth Cohorts and UK Blood Service control data, merging genotypes from the Illumina 1.2M Duo chip and the Affymetrix Genome Wide Human SNP array 6.0. Prior to analysis with IMPUTE2, we applied standard quality control filters akin to those described above. To further protect against potential errors misleading the imputation and phasing, we checked that each genotype conformed to local patterns of linkage disequilibrium in HapMap by using a leave-one-out imputation strategy. Specifically, we ran IMPUTE<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056009">46</a></sup> on each of the study samples in turn, both cases and controls, re-imputing known genotypes. Control individuals for which the imputed genotypes were more than 4.5% discordant with the original genotype were removed. The same rule was applied to case individuals with a discordance threshold of 6%. SNPs for which IMPUTE was confident of the imputation call but for which the genotyped data was discordant (and therefore indicative of genotyping error) were also removed if the difference between the measure of information and error rate was greater than 0.05. The final dataset comprised 1,787 cases and 4,800 controls.</p><p id="P33">After merging the imputed and genotyped data, we used the program SNPTEST to perform frequentist tests of association at each SNP under a missing data logistic regression model. Unless otherwise stated, we assumed a multiplicative model for allelic risk by encoding the genotypes at each SNP as a discrete explanatory variable with an indicator of case status as the binary response. <em>P</em> values were calculated using the score test implementation in SNPTEST. SNPs for which the (Fisher) information measure, as calculated by SNPTEST, was &lt;0.85 in the case collection or &lt;0.95 in either of the control groups were removed from the analysis. We also removed SNPs for which the equivalent observed (Fisher) information about the parameter describing the risk associated with the individual’s genotype was &lt;0.85. Both of these quality control steps exclude SNPs with high missingness rates or uncertain genotypes. Only SNPs with a minor allele frequency &gt;1% were used for subsequent analyses. Evidence for population stratification was minimal in the WTCCC2 discovery set (λ = 1.056), so covariates reflecting ancestry were not included in association models (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a>).</p><p id="P34" class="p p-last">For TASC data, findings from the discovery GWAS were combined with previously reported results from 1,236 non-overlapping cases and 3,979 controls from TASC<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055974">6</a></sup>. Data from the TASC discovery sets were imputed with Markov Chain Haplotyping software (MACH) using phased data from CEU individuals from release 22 of the HapMap project as the reference set of haplotypes<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056000">47</a></sup>. Only SNPs common to cases and controls were used for imputation. Only SNPs that could be imputed with relatively high confidence (<em>R</em><sup>2</sup> &gt; 0.3) and had a minor allele frequency &gt;1% were used in subsequent analyses. Association analysis of imputed SNPs was performed assuming an underlying additive model with the use of the software package MACH2DAT, which regresses affection status against expected dosage at each imputed locus<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055973">47</a></sup>. Analysis of genotyped SNPs was performed using logistic regression in PLINK<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056012">48</a></sup>. As the evidence of population stratification was appreciable in the TASC discovery set, the first four components from an EIGENSTRAT principal components analysis were included as covariates in the model, as has been described previously<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055979">6</a></sup>. After correction, the TASC data had an inflation factor of λ = 1.025 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a>).</p></div><div id="S14" class="sec"><h3 id="S14title">Meta-analysis</h3><p id="P35" class="p p-first-last">Prior to analysis, further checks were made to exclude any cryptic sample duplication within, or between, TASC and WTCCC2 cases and controls, leading to the exclusion of a further 13 samples. In total, the sample and SNP exclusions described above resulted in the association analysis of 3,023 cases and 8,779 controls at 2,223,620 autosomal SNPs. To combine the evidence of association across the TASC and WTCCC2 data, we conducted a fixed effect meta-analysis at SNPs present in both datasets. To do so, we averaged the estimated effect size parameter associated with genotype risk across the two datasets, weighted by the inverse of the variance in the estimates. <em>P</em> values were calculated assuming the square of the combined data z-score was distributed as χ<sup>2</sup> with 1 degree of freedom. As an additional quality control step, we compared the estimated effect size in each dataset to the meta-analysis estimate using Cochran’s <em>Q</em>, excluding SNPs with heterogeneity <em>P</em> &lt; 0.01. These analyses were conducted using the statistical package R<sup><a href="#R49" rid="R49" class=" bibr popnode">49</a></sup>.</p></div><div id="S15" class="sec"><h3 id="S15title">Replication genotyping and quality control</h3><p id="P36" class="p p-first-last">Genotyping was carried out at the Wellcome Trust Sanger Institute using the Sequenom iPLEX Gold assay. Individual samples were excluded from analysis if they had call rates &lt;90% or if the reported gender was discordant with sex-specific markers. We removed pairs of samples showing concordance indicative of being duplicates with individuals in the replication and/or discovery sets.</p></div><div id="S16" class="sec"><h3 id="S16title">Replication and combined analysis</h3><p id="P37" class="p p-first-last">Association in the replication cohort was performed using logistic regression assuming an additive model and including a dichotomous covariate reflecting Canadian ancestry. Analyses were also performed in the statistical package R. We report replication <em>P</em> values for the likelihood ratio test comparing the models with and without a SNP effect on risk, which is a two-tailed test on the direction of the effect.</p></div><div id="S17" class="sec"><h3 id="S17title">Interaction, conditional and haplotype analyses</h3><p id="P38" class="p p-first">Results from the WTCCC2-TASC meta-analysis and the replication data were combined using the fixed effect model described above. Frequentist conditional analysis was performed by including the primary SNP as a covariate in the logistic regression. We then tested whether the addition of a secondary SNP better explained risk at the locus using a likelihood ratio test with 1 degree of freedom. We took the same approach to testing for interaction (defined as a departure from additivity on the log-odds scale) by comparing a model with an additive term for each variant to one which also included a multiplicative interaction term: <em>y</em> = μ + <em>β</em><sub>1</sub><em>x</em><sub>1</sub> + <em>β</em><sub>2</sub><em>x</em><sub>2</sub> + <em>β</em><sub>12</sub><em>x</em><sub>1</sub>*<em>x</em><sub>2,</sub> where <em>y</em> is the log odds of disease, μ is an intercept, <em>x</em><sub>1</sub> and <em>x</em><sub>2</sub> are dummy variables coded so as to reflect underlying additive effects (0, 1 or 2) at the first and second SNPs and <em>β</em><sub>1</sub>, <em>β</em><sub>2</sub> and <em>β</em><sub>12</sub> are regression coefficients estimated from the data. In the case of rs4349859 (which tags HLA-B27), we fitted a dominant term for <em>x</em><sub>1</sub> rather than an additive term (<em>x</em><sub>1</sub> = 0 if the genotype at rs4349859 is GG, and <em>x</em><sub>1</sub> = 1 if the genotype at rs4349859 is AG or AA). Twice the difference in log likelihood between the full and base models is distributed as a χ<sup>2</sup> statistic with 1 degree of freedom.</p><p id="P39">Given the large difference in magnitude between main effects at rs4349859 and the other loci, we also tested the robustness of our findings by fitting a full two-locus model with additive and dominance terms and interactions between them: <em>y</em> = μ + <em>β</em><sub>a1</sub><em>x</em><sub>1</sub> + <em>β</em><sub>d1</sub><em>x</em><sub>1</sub> + <em>β</em><sub>a2</sub><em>x</em><sub>2</sub> + <em>β</em><sub>d2</sub><em>x</em><sub>2</sub> + <em>β</em><sub>aa</sub><em>x</em><sub>1</sub>*<em>x</em><sub>2</sub> + <em>β</em><sub>ad</sub><em>x</em><sub>1</sub>*<em>z</em><sub>2</sub> + <em>β</em><sub>da</sub><em>z</em><sub>1</sub>*<em>x</em><sub>2</sub> + <em>β</em><sub>dd</sub><em>z</em><sub>1</sub>*<em>z</em><sub>2</sub>, where <em>y</em> is the log odds of disease, μ is an intercept, <em>x</em><sub>1</sub> and <em>x</em><sub>2</sub> are dummy variables coded so as to reflect underlying additive effects (0, 1 or 2) at the first and second SNPs, <em>z</em><sub>1</sub> and <em>z</em><sub>2</sub> are dummy variables coded so as to pick up any dominance effects at the loci (0, 1 or 0), and <em>β</em><sub>a1</sub>, <em>β</em><sub>d1</sub>, <em>β</em><sub>a2</sub>, <em>β</em><sub>d2</sub>, <em>β</em><sub>aa</sub>, <em>β</em><sub>ad</sub>, <em>β</em><sub>da</sub> and <em>β</em><sub>dd</sub> are regression coefficients estimated from the data. Covariates were included in the model where appropriate to account for cohort and ancestry effects. The test for interaction was performed by comparing this full model to a model where only the main effects were fitted, with twice the difference in log likelihood between the models being distributed as a χ<sup>2</sup> statistic with 4 degrees of freedom.</p><p id="P40">As the pairs of SNPs identified at <em>HLA-B</em>, <em>ERAP1</em> and <em>IL23R</em> showed very little evidence of recombination (<em>D’</em> &gt; 0.98), we phased individual’s genotypes by parsimony, excluding the small number of samples (&lt;20 at each SNP) with genotypes consistent with rare recombination, genotyping error or repeat mutation. Haplotype risks were estimated from the replication data using a logistic regression model by including an additive term for each of the haplotypes.</p><p id="P41" class="p p-last">The program GENECLUSTER was used to look in the WTCCC2 data for regions where the pattern of association is not sufficiently characterized by the most significant SNP<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup>.</p></div><div id="S18" class="sec"><h3 id="S18title">Synovial tissue gene expression studies</h3><p id="P42" class="p p-first-last">We obtained 15 synovial biopsy tissue samples from the Synovial Tissue Bank at the Repatriation General Hospital in Adelaide, South Australia with the appropriate ethical approvals. The samples consisted of six seronegative spondyloarthropy, two ankylosing spondylitis, three osteoarthritis and four normal control biopsies. RNA was extracted from the formaldehyde-fixed paraffin-embedded tissue blocks using the Arcturus Paradise Plus Reagent System (Molecular Devices) according to the manufacturer’s protocol. We used 200 ng of RNA in the Illumina Whole-Genome DASL (complementary DNA-mediated annealing, selection, extension and ligation) Gene Expression Assay. Data was processed in GenomeStudio (Illumina) and analyzed using Lumi<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055989">50</a></sup> and BRB ArrayTools<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310055978">51</a></sup>. For analysis, ankylosing spondylitis and spondyloarthropy samples were grouped together and compared with a control group consisting of osteoarthritis (a non-inflammatory arthritis) and unaffected samples. Unsupervised clustering showed no differences between ankylosing spondylitis and spondyloarthropy or between osteoarthritis and normal samples, supporting the grouping of these samples for analysis. However, unsupervised clustering between the ankylosing spondylitis and spondyloarthropy and osteoarthritis and normal groups showed good delineation of the two groups, with only one sample from each group being misclassified. Differentially expressed genes were identified by unpaired <em>t</em>-tests with multivariate permutation correction.</p></div><div id="S19" class="sec"><h3 id="S19title">Cytokine receptor cleavage studies</h3><p id="P43" class="p p-first-last">Single-cell suspensions were prepared from spleens of <em>Erap1</em><sup>−/−</sup> and C57BL/6 control mice and plated in 48-well plates at 5 × 10<sup>6</sup> cells per well. Cells were stimulated with plate-bound anti-CD3 (10 μg/ml, Clone 145-2C11) and phorbol 12-myristate 13-acetate (PMA) (10 ng/ml) or left untreated. Levels of soluble TNF-R and IL-6R were quantified from cell culture supernatants obtained 24 h after stimulation TNF-R, and IL-6R levels were evaluated by ELISA according to the manufacturer’s protocol (R&amp;D Systems).</p></div><div id="S20" class="sec"><h3 id="S20title">ERAP1 peptidase activity studies</h3><p id="P44" class="p p-first-last">Full-length wild-type <em>ERAP1</em> was cloned into pFastBac as a C-terminal His10-tagged fusion protein using standard cloning techniques. ERAP1 substitution mutants p.Lys528Arg and p.Arg725Gln were generated by site-directed mutagenesis. Recombinant proteins were purified to homogeneity on affinity Ni-NTA, and purity was assessed by SDS-PAGE followed by Coomassie staining. Enzymatic activities were assessed as previously published<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310056001">26</a></sup>.</p></div><div id="S21" class="sec sec-last"><h3 id="S21title">CD8 lymphocyte counts in ankylosing spondylitis cases and controls</h3><p id="P45" class="p p-first-last">Peripheral blood was obtained from 16 individuals with active ankylosing spondylitis (erythrocyte sedimentation rate &gt; 25 and C-reactive protein &gt; 10) and 16 age-matched healthy controls. Peripheral blood mononuclear cells (PBMCs) were extracted using a standard density gradient centrifugation over Ficoll-Paque Plus (GE Healthcare). Extracted PBMCs were frozen in 90% FBS/10% DMSO until needed. Frozen PBMCs were thawed into RPMI + 20% FBS and washed once in RPMI + 10% FBS. PBMCs were rested in RPMI + 10% FBS for approximately 1 h at 37 °C, 5% CO<sub>2</sub> before further use. PBMCs were stained with CD3 ECD (UCHT1, Beckman Coulter) and CD8 APC AlexaFluor700 (B9.11, Beckman Coulter). Dead cells were excluded using a Live/Dead Fixable Dead Cell Stain kit (Invitrogen). Cells were acquired on a Gallios 10-color flow cytometer (Beckman Coulter) and staining was analyzed using Kaluza software (Beckman Coulter).</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemental data</h4><div class="sup-box half_rhythm" id="d36e2363"><a href="/pmc/articles/PMC3640413/bin/NIHMS405164-supplement-Supplemental_data.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.2M, pdf)</sup></div></div></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">We would like to thank all participating subjects with ankylosing spondylitis and healthy individuals who provided the DNA and clinical information necessary for this study. The Wellcome Trust Case Control Consortium 2 project is funded by the Wellcome Trust (083948/Z/07/Z). We also thank S. Bertrand, J. Bryant, S.L. Clark, J.S. Conquer, T. Dibling, J.C. Eldred, S. Gamble, C. Hind, A. Wilk, C.R. Stribling and S. Taylor of the Wellcome Trust Sanger Institute’s Sample and Genotyping Facilities for technical assistance. The TASC study was funded by the National Institute of Arthritis and Musculoskeletal and Skin Diseases (NIAMS) grants P01-052915 and R01-AR046208. Funding was also received from the University of Texas at Houston Clinical and Translational Science Awards grant UL1RR024188, Cedars-Sinai General Clinical Research Centre grant MO1-RR00425, Intramural Research Program, NIAMS/US National Institutes of Health and Rebecca Cooper Foundation (Australia). This study was funded, in part, by Arthritis Research UK (Grants 19536 and 18797), by the Wellcome Trust (grant number 076113) and by the Oxford Comprehensive Biomedical Research Centre ankylosing spondylitis chronic disease cohort (theme code: A91202). The Spondyloarthritis Research Consortium of Canada (SPARCC) was funded by a National Research Initiative Award from the Arthritis Society (Canada). G.P.T. was funded by a Lions Medical Research Foundation fellowship. M.A.B. is funded by a National Health and Medical Research Council (Australia) Principal Research Fellowship, and support for this study was received from a National Health and Medical Research Council (Australia) program grant (566938) and project grant (569829) and from the Australian Cancer Research Foundation and Rebecca Cooper Medical Research Foundation. P. Donnelly was supported in part by a Wolfson-Royal Society Merit Award. We are also very grateful for the invaluable support received from the National Ankylosing Spondylitis Society (UK) and Spondyloarthritis Association of America in case recruitment. Additional financial and technical support for subject recruitment was provided by the National Institute for Health Research (NIHR), Oxford Musculoskeletal Biomedical Research Unit and NIHR Thames Valley Comprehensive Local Research Network. We acknowledge use of the British 1958 Birth Cohort DNA collection, funded by the Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02, and we thank W. Bodmer and B. Winney for use of the People of the British Isles DNA collection, which was funded by the Wellcome Trust. We would like to thank A. Mathieu (Cagliari University) and S. Brown (Cedars-Sinai Hospital) for providing samples. The Structural Genomics Consortium is a registered charity (number 1097737) that receives funds from the Canadian Institutes for Health Research, the Canadian Foundation for Innovation, Genome Canada through the Ontario Genomics Institute, GlaxoSmithKline, Karolinska Institutet, the Knut and Alice Wallenberg Foundation, the Ontario Innovation Trust, the Ontario Ministry for Research and Innovation, Merck &amp; Co., Inc., the Novartis Research Foundation, the Swedish Agency for Innovation Systems, the Swedish Foundation for Strategic Research and the Wellcome Trust.</p></div></div><div id="idm139960882273152" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139960882273152title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="P46" class="p p-first-last"><strong>URLs.</strong> SNPTEST, <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a>; HapMap, <a href="http://hapmap.ncbi.nlm.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hapmap.ncbi.nlm.nih.gov/</a>.</p><p></p><p class="fn sec" id="FN5"></p><p id="P47" class="p p-first-last">Note: Supplementary information is available on the Nature Genetics website.</p><p></p><p class="fn sec" id="FN6"></p><p id="P48" class="p p-first-last"><strong>AUTHOR CONTRIBUTIONS</strong> M.A.B., L.A.B., C. Farrar, J.D.R., J.J.P., B.P.W., D.G., W.M. and P.R. oversaw cohort collection for the discovery and replication datasets. The WTCCC2 DNA, genotyping, data quality control and informatics group (S.J.B., S.D., S.E., E.G., C.L. and L.P.) executed GWAS sample handling, genotyping and quality control. The WTCCC2 data and analysis group (A.S., C.C.A.S., G.B., C.B., C. Freeman and P. Donnelly), D.M.E. and M.A.B. performed statistical analyses. M.A.B., D.M.E., C.C.A.S. and P. Donnelly contributed to writing the manuscript. The WTCCC2 management committee (J.M.B., E.B., M.A.B., J.P.C., A.C., P. Deloukas, P. Donnelly (chairperson), A. Duncanson, J.J., J.L., H.S.M., C.G.M., C.N.A.P., L.P., R.P., A.R., S.J.S., R.C.T., A.C.V. and N.W.W.) monitored the execution of the GWAS. D.H., G.K. and U.O. performed analyses of recombinant ERAP1 function. T.J.K. and G.P.T. performed gene expression, cell count and ERAP1 sheddase functional studies. Other authors contributed variously to sample collection and all other aspects of the study. All authors reviewed the final manuscript.</p><p></p><p class="fn sec" id="FN7"></p><p id="P49" class="p p-first-last"><strong>COMPETING FINANCIAL INTERESTS</strong> The authors declare competing financial interests: details accompany the full-text HTML version of the paper at <a href="http://www.nature.com/naturegenetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics/</a>.</p><p></p><p class="fn sec" id="FN8"></p><p id="P50" class="p p-first-last">Reprints and permissions information is available online at <a href="http://www.nature.com/reprints/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/reprints/index.html</a>.</p><p></p></div></div><div id="idm139960887630416" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139960887630416title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Pedersen OB, et al.  Ankylosing spondylitis in Danish and Norwegian twins: occurrence and the relative importance of genetic vs. environmental effectors in disease causation. <span><span class="ref-journal">Scand. J. Rheumatol. </span>2008;<span class="ref-vol">37</span>:120–126.</span> [<a href="/pubmed/18415769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Scand.+J.+Rheumatol&amp;title=Ankylosing+spondylitis+in+Danish+and+Norwegian+twins:+occurrence+and+the+relative+importance+of+genetic+vs.+environmental+effectors+in+disease+causation&amp;author=OB+Pedersen&amp;volume=37&amp;publication_year=2008&amp;pages=120-126&amp;pmid=18415769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Brown MA, et al.  Susceptibility to ankylosing spondylitis in twins: the role of genes, HLA, and the environment. <span><span class="ref-journal">Arthritis Rheum. </span>1997;<span class="ref-vol">40</span>:1823–1828.</span> [<a href="/pubmed/9336417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Susceptibility+to+ankylosing+spondylitis+in+twins:+the+role+of+genes,+HLA,+and+the+environment&amp;author=MA+Brown&amp;volume=40&amp;publication_year=1997&amp;pages=1823-1828&amp;pmid=9336417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Brown MA, Laval SH, Brophy S, Calin A. Recurrence risk modelling of the genetic susceptibility to ankylosing spondylitis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2000;<span class="ref-vol">59</span>:883–886.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1753017/">PMC free article</a>]</span> [<a href="/pubmed/11053066" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis&amp;title=Recurrence+risk+modelling+of+the+genetic+susceptibility+to+ankylosing+spondylitis&amp;author=MA+Brown&amp;author=SH+Laval&amp;author=S+Brophy&amp;author=A+Calin&amp;volume=59&amp;publication_year=2000&amp;pages=883-886&amp;pmid=11053066&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Calin A, Marder A, Becks E, Burns T. Genetic differences between B27 positive patients with ankylosing spondylitis and B27 positive healthy controls. <span><span class="ref-journal">Arthritis Rheum. </span>1983;<span class="ref-vol">26</span>:1460–1464.</span> [<a href="/pubmed/6606431" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Genetic+differences+between+B27+positive+patients+with+ankylosing+spondylitis+and+B27+positive+healthy+controls&amp;author=A+Calin&amp;author=A+Marder&amp;author=E+Becks&amp;author=T+Burns&amp;volume=26&amp;publication_year=1983&amp;pages=1460-1464&amp;pmid=6606431&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">van der Linden S, Valkenburg H, Cats A. The risk of developing ankylosing spondylitis in HLA-B27 positive individuals: a family and population study. <span><span class="ref-journal">Br. J. Rheumatol. </span>1983;<span class="ref-vol">22</span>:18–19.</span> [<a href="/pubmed/6606472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Rheumatol&amp;title=The+risk+of+developing+ankylosing+spondylitis+in+HLA-B27+positive+individuals:+a+family+and+population+study&amp;author=S+van+der+Linden&amp;author=H+Valkenburg&amp;author=A+Cats&amp;volume=22&amp;publication_year=1983&amp;pages=18-19&amp;pmid=6606472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Reveille JD, et al.  Genome-wide association study of ankylosing spondylitis identifies non-MHC susceptibility loci. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:123–127.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3224997/">PMC free article</a>]</span> [<a href="/pubmed/20062062" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+of+ankylosing+spondylitis+identifies+non-MHC+susceptibility+loci&amp;author=JD+Reveille&amp;volume=42&amp;publication_year=2010&amp;pages=123-127&amp;pmid=20062062&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Wellcome Trust Case Control Consortium et al.  Association scan of 14,500 nonsynonymous SNPs in four diseases identifies autoimmunity variants. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1329–1337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2680141/">PMC free article</a>]</span> [<a href="/pubmed/17952073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Association+scan+of+14,500+nonsynonymous+SNPs+in+four+diseases+identifies+autoimmunity+variants&amp;volume=39&amp;publication_year=2007&amp;pages=1329-1337&amp;pmid=17952073&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Danoy P, et al.  Association of variants at 1q32 and <em>STAT3</em> with ankylosing spondylitis suggests genetic overlap with Crohn’s disease. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1001195.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2996314/">PMC free article</a>]</span> [<a href="/pubmed/21152001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Association+of+variants+at+1q32+and+STAT3+with+ankylosing+spondylitis+suggests+genetic+overlap+with+Crohn’s+disease&amp;author=P+Danoy&amp;volume=6&amp;publication_year=2010&amp;pages=e1001195&amp;pmid=21152001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">van der Linden S, Valkenburg HA, Cats A. Evaluation of diagnostic criteria for ankylosing spondylitis. A proposal for modification of the New York criteria. <span><span class="ref-journal">Arthritis Rheum. </span>1984;<span class="ref-vol">27</span>:361–368.</span> [<a href="/pubmed/6231933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Evaluation+of+diagnostic+criteria+for+ankylosing+spondylitis.+A+proposal+for+modification+of+the+New+York+criteria&amp;author=S+van+der+Linden&amp;author=HA+Valkenburg&amp;author=A+Cats&amp;volume=27&amp;publication_year=1984&amp;pages=361-368&amp;pmid=6231933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Pointon JJ, et al.  Elucidating the chromosome 9 association with AS; CARD9 is a candidate gene. <span><span class="ref-journal">Genes and Immunity. </span>2011;<span class="ref-vol">12</span>:319–320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2933507/">PMC free article</a>]</span> [<a href="/pubmed/20463747" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+and+Immunity&amp;title=Elucidating+the+chromosome+9+association+with+AS;+CARD9+is+a+candidate+gene&amp;author=JJ+Pointon&amp;volume=12&amp;publication_year=2011&amp;pages=319-320&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Park JH, et al.  Signaling by intrathymic cytokines, not T cell antigen receptors, specifies CD8 lineage choice and promotes the differentiation of cytotoxic-lineage T cells. <span><span class="ref-journal">Nat. Immunol. </span>2010;<span class="ref-vol">11</span>:257–264.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3555225/">PMC free article</a>]</span> [<a href="/pubmed/20118929" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Immunol&amp;title=Signaling+by+intrathymic+cytokines,+not+T+cell+antigen+receptors,+specifies+CD8+lineage+choice+and+promotes+the+differentiation+of+cytotoxic-lineage+T+cells&amp;author=JH+Park&amp;volume=11&amp;publication_year=2010&amp;pages=257-264&amp;pmid=20118929&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Ferreira MA, et al.  Quantitative trait loci for CD4:CD8 lymphocyte ratio are associated with risk of type 1 diabetes and HIV-1 immune control. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2010;<span class="ref-vol">86</span>:88–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2801744/">PMC free article</a>]</span> [<a href="/pubmed/20045101" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Quantitative+trait+loci+for+CD4:CD8+lymphocyte+ratio+are+associated+with+risk+of+type+1+diabetes+and+HIV-1+immune+control&amp;author=MA+Ferreira&amp;volume=86&amp;publication_year=2010&amp;pages=88-92&amp;pmid=20045101&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Su Z, et al.  Bayesian method for detecting and characterizing allelic heterogeneity and boosting signals in genome-wide association studies. <span><span class="ref-journal">Stat. Sci. </span>2009;<span class="ref-vol">24</span>:430–450.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Sci&amp;title=Bayesian+method+for+detecting+and+characterizing+allelic+heterogeneity+and+boosting+signals+in+genome-wide+association+studies&amp;author=Z+Su&amp;volume=24&amp;publication_year=2009&amp;pages=430-450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Stephens M, Balding DJ. Bayesian statistical methods for genetic association studies. <span><span class="ref-journal">Nat. Rev. Genet. </span>2009;<span class="ref-vol">10</span>:681–690.</span> [<a href="/pubmed/19763151" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Bayesian+statistical+methods+for+genetic+association+studies&amp;author=M+Stephens&amp;author=DJ+Balding&amp;volume=10&amp;publication_year=2009&amp;pages=681-690&amp;pmid=19763151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Amroun H, et al.  Early-onset ankylosing spondylitis is associated with a functional MICA polymorphism. <span><span class="ref-journal">Hum. Immunol. </span>2005;<span class="ref-vol">66</span>:1057–1061.</span> [<a href="/pubmed/16386647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Immunol&amp;title=Early-onset+ankylosing+spondylitis+is+associated+with+a+functional+MICA+polymorphism&amp;author=H+Amroun&amp;volume=66&amp;publication_year=2005&amp;pages=1057-1061&amp;pmid=16386647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Yabuki K, et al.  Triplet repeat polymorphism in the <em>MICA</em> gene in HLA-B27 positive and negative Caucasian patients with ankylosing spondylitis. <span><span class="ref-journal">Hum. Immunol. </span>1999;<span class="ref-vol">60</span>:83–86.</span> [<a href="/pubmed/9952031" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Immunol&amp;title=Triplet+repeat+polymorphism+in+the+MICA+gene+in+HLA-B27+positive+and+negative+Caucasian+patients+with+ankylosing+spondylitis&amp;author=K+Yabuki&amp;volume=60&amp;publication_year=1999&amp;pages=83-86&amp;pmid=9952031&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Brown MA, et al.  HLA class I associations of ankylosing spondylitis in the white population in the United Kingdom. <span><span class="ref-journal">Ann. Rheum. Dis. </span>1996;<span class="ref-vol">55</span>:268–270.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1010149/">PMC free article</a>]</span> [<a href="/pubmed/8733445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis&amp;title=HLA+class+I+associations+of+ankylosing+spondylitis+in+the+white+population+in+the+United+Kingdom&amp;author=MA+Brown&amp;volume=55&amp;publication_year=1996&amp;pages=268-270&amp;pmid=8733445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">D’Amato M, et al.  Relevance of residue 116 of HLA-B27 in determining susceptibility to ankylosing spondylitis. <span><span class="ref-journal">Eur. J. Immunol. </span>1995;<span class="ref-vol">25</span>:3199–3201.</span> [<a href="/pubmed/7489765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Immunol&amp;title=Relevance+of+residue+116+of+HLA-B27+in+determining+susceptibility+to+ankylosing+spondylitis&amp;author=M+D’Amato&amp;volume=25&amp;publication_year=1995&amp;pages=3199-3201&amp;pmid=7489765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">López-Larrea C, et al.  HLA-B27 subtypes in Asian patients with ankylosing spondylitis. Evidence for new associations. <span><span class="ref-journal">Tissue Antigens. </span>1995;<span class="ref-vol">45</span>:169–176.</span> [<a href="/pubmed/7761976" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Tissue+Antigens&amp;title=HLA-B27+subtypes+in+Asian+patients+with+ankylosing+spondylitis.+Evidence+for+new+associations&amp;author=C+López-Larrea&amp;volume=45&amp;publication_year=1995&amp;pages=169-176&amp;pmid=7761976&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Brown MA, et al.  HLA Class I and II associations of ankylosing spondylitis. <span><span class="ref-journal">Arthritis Rheum. </span>2009;<span class="ref-vol">60</span>:S437.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=HLA+Class+I+and+II+associations+of+ankylosing+spondylitis&amp;author=MA+Brown&amp;volume=60&amp;publication_year=2009&amp;pages=S437&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Saric T, et al.  An IFN-gamma-induced aminopeptidase in the ER, ERAP1, trims precursors to MHC class I-presented peptides. <span><span class="ref-journal">Nat. Immunol. </span>2002;<span class="ref-vol">3</span>:1169–1176.</span> [<a href="/pubmed/12436109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Immunol&amp;title=An+IFN-gamma-induced+aminopeptidase+in+the+ER,+ERAP1,+trims+precursors+to+MHC+class+I-presented+peptides&amp;author=T+Saric&amp;volume=3&amp;publication_year=2002&amp;pages=1169-1176&amp;pmid=12436109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Chang SC, Momburg F, Bhutani N, Goldberg AL. The ER aminopeptidase, ERAP1, trims precursors to lengths of MHC class I peptides by a “molecular ruler” mechanism. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2005;<span class="ref-vol">102</span>:17107–17112.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1287962/">PMC free article</a>]</span> [<a href="/pubmed/16286653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=The+ER+aminopeptidase,+ERAP1,+trims+precursors+to+lengths+of+MHC+class+I+peptides+by+a+“molecular+ruler”+mechanism&amp;author=SC+Chang&amp;author=F+Momburg&amp;author=N+Bhutani&amp;author=AL+Goldberg&amp;volume=102&amp;publication_year=2005&amp;pages=17107-17112&amp;pmid=16286653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Cui X, Rouhani FN, Hawari F, Levine SJ. Shedding of the type II IL-1 decoy receptor requires a multifunctional aminopeptidase, aminopeptidase regulator of TNF receptor type 1 shedding. <span><span class="ref-journal">J. Immunol. </span>2003;<span class="ref-vol">171</span>:6814–6819.</span> [<a href="/pubmed/14662887" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol&amp;title=Shedding+of+the+type+II+IL-1+decoy+receptor+requires+a+multifunctional+aminopeptidase,+aminopeptidase+regulator+of+TNF+receptor+type+1+shedding&amp;author=X+Cui&amp;author=FN+Rouhani&amp;author=F+Hawari&amp;author=SJ+Levine&amp;volume=171&amp;publication_year=2003&amp;pages=6814-6819&amp;pmid=14662887&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Cui X, Rouhani FN, Hawari F, Levine SJ. An aminopeptidase, ARTS-1, is required for interleukin-6 receptor shedding. <span><span class="ref-journal">J. Biol. Chem. </span>2003;<span class="ref-vol">278</span>:28677–28685.</span> [<a href="/pubmed/12748171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=An+aminopeptidase,+ARTS-1,+is+required+for+interleukin-6+receptor+shedding&amp;author=X+Cui&amp;author=FN+Rouhani&amp;author=F+Hawari&amp;author=SJ+Levine&amp;volume=278&amp;publication_year=2003&amp;pages=28677-28685&amp;pmid=12748171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Cui X, et al.  Identification of ARTS-1 as a novel TNFR1-binding protein that promotes TNFR1 ectodomain shedding. <span><span class="ref-journal">J. Clin. Invest. </span>2002;<span class="ref-vol">110</span>:515–526.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC150410/">PMC free article</a>]</span> [<a href="/pubmed/12189246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Identification+of+ARTS-1+as+a+novel+TNFR1-binding+protein+that+promotes+TNFR1+ectodomain+shedding&amp;author=X+Cui&amp;volume=110&amp;publication_year=2002&amp;pages=515-526&amp;pmid=12189246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Evnouchidou I, Berardi MJ, Stratikos E. A continuous fluorigenic assay for the measurement of the activity of endoplasmic reticulum aminopeptidase 1: competition kinetics as a tool for enzyme specificity investigation. <span><span class="ref-journal">Anal. Biochem. </span>2009;<span class="ref-vol">395</span>:33–40.</span> [<a href="/pubmed/19638272" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal.+Biochem&amp;title=A+continuous+fluorigenic+assay+for+the+measurement+of+the+activity+of+endoplasmic+reticulum+aminopeptidase+1:+competition+kinetics+as+a+tool+for+enzyme+specificity+investigation&amp;author=I+Evnouchidou&amp;author=MJ+Berardi&amp;author=E+Stratikos&amp;volume=395&amp;publication_year=2009&amp;pages=33-40&amp;pmid=19638272&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Libioulle C, et al.  Novel Crohn disease locus identified by genome-wide association maps to a gene desert on 5p13.1 and modulates expression of <em>PTGER4</em>. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1853118/">PMC free article</a>]</span> [<a href="/pubmed/17447842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Novel+Crohn+disease+locus+identified+by+genome-wide+association+maps+to+a+gene+desert+on+5p13.1+and+modulates+expression+of+PTGER4&amp;author=C+Libioulle&amp;volume=3&amp;publication_year=2007&amp;pages=e58&amp;pmid=17447842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Cho JH. The genetics and immunopathogenesis of inflammatory bowel disease. <span><span class="ref-journal">Nat. Rev. Immunol. </span>2008;<span class="ref-vol">8</span>:458–466.</span> [<a href="/pubmed/18500230" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Immunol&amp;title=The+genetics+and+immunopathogenesis+of+inflammatory+bowel+disease&amp;author=JH+Cho&amp;volume=8&amp;publication_year=2008&amp;pages=458-466&amp;pmid=18500230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Zhang XJ, et al.  Psoriasis genome-wide association study identifies susceptibility variants within <em>LCE</em> gene cluster at 1q21. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:205–210.</span> [<a href="/pubmed/19169255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Psoriasis+genome-wide+association+study+identifies+susceptibility+variants+within+LCE+gene+cluster+at+1q21&amp;author=XJ+Zhang&amp;volume=41&amp;publication_year=2009&amp;pages=205-210&amp;pmid=19169255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Cargill M, et al.  A large-scale genetic association study confirms <em>IL12B</em> and leads to the identification of <em>IL23R</em> as psoriasis-risk genes. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:273–290.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785338/">PMC free article</a>]</span> [<a href="/pubmed/17236132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+large-scale+genetic+association+study+confirms+IL12B+and+leads+to+the+identification+of+IL23R+as+psoriasis-risk+genes&amp;author=M+Cargill&amp;volume=80&amp;publication_year=2007&amp;pages=273-290&amp;pmid=17236132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Duerr RH, et al.  A genome-wide association study identifies <em>IL23R</em> as an inflammatory bowel disease gene. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>:1461–1463.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4410764/">PMC free article</a>]</span> [<a href="/pubmed/17068223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+identifies+IL23R+as+an+inflammatory+bowel+disease+gene&amp;author=RH+Duerr&amp;volume=314&amp;publication_year=2006&amp;pages=1461-1463&amp;pmid=17068223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Shen H, Goodall JC, Hill Gaston JS. Frequency and phenotype of peripheral blood Th17 cells in ankylosing spondylitis and rheumatoid arthritis. <span><span class="ref-journal">Arthritis Rheum. </span>2009;<span class="ref-vol">60</span>:1647–1656.</span> [<a href="/pubmed/19479869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Frequency+and+phenotype+of+peripheral+blood+Th17+cells+in+ankylosing+spondylitis+and+rheumatoid+arthritis&amp;author=H+Shen&amp;author=JC+Goodall&amp;author=JS+Hill+Gaston&amp;volume=60&amp;publication_year=2009&amp;pages=1647-1656&amp;pmid=19479869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Gagliardi MC, et al.  Endogenous PGE2 promotes the induction of human Th17 responses by fungal β-glucan. <span><span class="ref-journal">J. Leukoc. Biol. </span>2010;<span class="ref-vol">88</span>:947–954.</span> [<a href="/pubmed/20807707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Leukoc.+Biol&amp;title=Endogenous+PGE2+promotes+the+induction+of+human+Th17+responses+by+fungal+β-glucan&amp;author=MC+Gagliardi&amp;volume=88&amp;publication_year=2010&amp;pages=947-954&amp;pmid=20807707&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Ruutu M, et al.  Fungal β-glucan triggers spondyloarthropathy and Crohn’s disease in SKG mice. <span><span class="ref-journal">Arthritis Rheum. </span>2010;<span class="ref-vol">S1</span>:1446.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Fungal+β-glucan+triggers+spondyloarthropathy+and+Crohn’s+disease+in+SKG+mice&amp;author=M+Ruutu&amp;volume=S1&amp;publication_year=2010&amp;pages=1446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Hirota K, et al.  T cell self-reactivity forms a cytokine milieu for spontaneous development of IL-17+ Th cells that cause autoimmune arthritis. <span><span class="ref-journal">J. Exp. Med. </span>2007;<span class="ref-vol">204</span>:41–47.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118414/">PMC free article</a>]</span> [<a href="/pubmed/17227914" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Med&amp;title=T+cell+self-reactivity+forms+a+cytokine+milieu+for+spontaneous+development+of+IL-17++Th+cells+that+cause+autoimmune+arthritis&amp;author=K+Hirota&amp;volume=204&amp;publication_year=2007&amp;pages=41-47&amp;pmid=17227914&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Armaka M, et al.  Mesenchymal cell targeting by TNF as a common pathogenic principle in chronic inflammatory joint and intestinal diseases. <span><span class="ref-journal">J. Exp. Med. </span>2008;<span class="ref-vol">205</span>:331–337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2271010/">PMC free article</a>]</span> [<a href="/pubmed/18250193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Med&amp;title=Mesenchymal+cell+targeting+by+TNF+as+a+common+pathogenic+principle+in+chronic+inflammatory+joint+and+intestinal+diseases&amp;author=M+Armaka&amp;volume=205&amp;publication_year=2008&amp;pages=331-337&amp;pmid=18250193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Risch N. Assessing the role of HLA-linked and unlinked determinants of disease. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1987;<span class="ref-vol">40</span>:1–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1684002/">PMC free article</a>]</span> [<a href="/pubmed/3468804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Assessing+the+role+of+HLA-linked+and+unlinked+determinants+of+disease&amp;author=N+Risch&amp;volume=40&amp;publication_year=1987&amp;pages=1-14&amp;pmid=3468804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Risch N. Linkage strategies for genetically complex traits. I. Multilocus models. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1990;<span class="ref-vol">46</span>:222–228.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1684987/">PMC free article</a>]</span> [<a href="/pubmed/2301392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Linkage+strategies+for+genetically+complex+traits.+I.+Multilocus+models&amp;author=N+Risch&amp;volume=46&amp;publication_year=1990&amp;pages=222-228&amp;pmid=2301392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Strange A, et al.  A genome-wide association study identifies new psoriasis susceptibility loci and an interaction between <em>HLA-C</em> and <em>ERAP1</em>. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:985–990.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3749730/">PMC free article</a>]</span> [<a href="/pubmed/20953190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+study+identifies+new+psoriasis+susceptibility+loci+and+an+interaction+between+HLA-C+and+ERAP1&amp;author=A+Strange&amp;volume=42&amp;publication_year=2010&amp;pages=985-990&amp;pmid=20953190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Jaakkola E, et al.  Finnish HLA studies confirm the increased risk conferred by HLA-B27 homozygosity in ankylosing spondylitis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2006;<span class="ref-vol">65</span>:775–780.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1798178/">PMC free article</a>]</span> [<a href="/pubmed/16249228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis&amp;title=Finnish+HLA+studies+confirm+the+increased+risk+conferred+by+HLA-B27+homozygosity+in+ankylosing+spondylitis&amp;author=E+Jaakkola&amp;volume=65&amp;publication_year=2006&amp;pages=775-780&amp;pmid=16249228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Rudwaleit M, et al.  Effectiveness, safety, and predictors of good clinical response in 1250 patients treated with adalimumab for active ankylosing spondylitis. <span><span class="ref-journal">J. Rheumatol. </span>2009;<span class="ref-vol">36</span>:801–808.</span> [<a href="/pubmed/19273449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Rheumatol&amp;title=Effectiveness,+safety,+and+predictors+of+good+clinical+response+in+1250+patients+treated+with+adalimumab+for+active+ankylosing+spondylitis&amp;author=M+Rudwaleit&amp;volume=36&amp;publication_year=2009&amp;pages=801-808&amp;pmid=19273449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Pointon JJ, et al.  The chromosome 16q region associated with ankylosing spondylitis includes the candidate gene tumour necrosis factor receptor type 1-associated death domain (<em>TRADD</em>) <span><span class="ref-journal">Ann. Rheum. Dis. </span>2010;<span class="ref-vol">69</span>:1243–1246.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2959172/">PMC free article</a>]</span> [<a href="/pubmed/19854717" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis&amp;title=The+chromosome+16q+region+associated+with+ankylosing+spondylitis+includes+the+candidate+gene+tumour+necrosis+factor+receptor+type+1-associated+death+domain+(TRADD)&amp;author=JJ+Pointon&amp;volume=69&amp;publication_year=2010&amp;pages=1243-1246&amp;pmid=19854717&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Conrad DF, et al.  Origins and functional impact of copy number variation in the human genome. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">464</span>:704–712.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3330748/">PMC free article</a>]</span> [<a href="/pubmed/19812545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Origins+and+functional+impact+of+copy+number+variation+in+the+human+genome&amp;author=DF+Conrad&amp;volume=464&amp;publication_year=2010&amp;pages=704-712&amp;pmid=19812545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Teo YY, et al.  A genotype calling algorithm for the Illumina BeadArray platform. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:2741–2746.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666488/">PMC free article</a>]</span> [<a href="/pubmed/17846035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+genotype+calling+algorithm+for+the+Illumina+BeadArray+platform&amp;author=YY+Teo&amp;volume=23&amp;publication_year=2007&amp;pages=2741-2746&amp;pmid=17846035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Spencer CC, et al.  Dissection of the genetics of Parkinson’s disease identifies an additional association 5′ of <em>SNCA</em> and multiple associated haplotypes at 17q21. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:345–353.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3005904/">PMC free article</a>]</span> [<a href="/pubmed/21044948" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Dissection+of+the+genetics+of+Parkinson’s+disease+identifies+an+additional+association+5′+of+SNCA+and+multiple+associated+haplotypes+at+17q21&amp;author=CC+Spencer&amp;volume=20&amp;publication_year=2011&amp;pages=345-353&amp;pmid=21044948&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu. Rev. Genomics Hum. Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genomics+Hum.+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">R Core Development Team  . <span class="ref-journal">R: A Language and Environment for Statistical Computing.</span> R Foundation for Statistical Computing; Vienna: 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+A+Language+and+Environment+for+Statistical+Computing&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Du P, Kibbe WA, Lin SM. lumi: a pipeline for processing Illumina microarray. <span><span class="ref-journal">Bioinformatics. </span>2008;<span class="ref-vol">24</span>:1547–1548.</span> [<a href="/pubmed/18467348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=lumi:+a+pipeline+for+processing+Illumina+microarray&amp;author=P+Du&amp;author=WA+Kibbe&amp;author=SM+Lin&amp;volume=24&amp;publication_year=2008&amp;pages=1547-1548&amp;pmid=18467348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Simon R, et al.  Analysis of gene expression data using BRB-array tools. <span><span class="ref-journal">Cancer Inform. </span>2007;<span class="ref-vol">3</span>:11–17.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2675854/">PMC free article</a>]</span> [<a href="/pubmed/19455231" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Inform&amp;title=Analysis+of+gene+expression+data+using+BRB-array+tools&amp;author=R+Simon&amp;volume=3&amp;publication_year=2007&amp;pages=11-17&amp;pmid=19455231&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3640413&amp;issue-id=222418&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139960882944128title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S7title">DISCUSSION</a></li><li><a href="#S9title">ONLINE METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S22title">ACKNOWLEDGMENTS</a></li><li><a href="#idm139960882273152title">Footnotes</a></li><li><a href="#idm139960887630416title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3640413/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3640413/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3640413/pdf/nihms-405164.pdf">PDF (367K)</a></li> | <li><a href="#" data-citationid="PMC3640413" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3640413%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3640413%2F&amp;text=Interaction%20between%20ERAP1%20and%20HLA-B27%20in%20ankylosing%20spondylitis%20implicates%20peptide%20handling%20in%20the%20mechanism%20for%20HLA-B27%20in%20disease%20susceptibility" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3640413%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21743469" data-aiid="3640413" data-aid="3640413" data-iid="222418" data-domainid="319" data-domain="nihpa" data-accid="PMC3640413" data-md5="95065706f03a5963f004f06e1d89e9f6"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:22:15-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D5F7DDD5298D100000000043C02D6&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3640413%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3640413/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>