<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3320515/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Integrative genomics identifies LMO1 as a neuroblastoma oncogene</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320515/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature"><meta name="citation_title" content="Integrative genomics identifies LMO1 as a neuroblastoma oncogene"><meta name="citation_authors" content="Kai Wang, Sharon J. Diskin, Haitao Zhang, Edward F. Attiyeh, Cynthia Winter, Cuiping Hou, Robert W. Schnepp, Maura Diamond, Kristopher Bosse, Patrick A. Mayes, Joseph Glessner, Cecilia Kim, Edward Frackelton, Maria Garris, Qun Wang, Wendy Glaberson, Rosetta Chiavacci, Le Nguyen, Jayanti Jagannathan, Norihisa Saeki, Hiroki Sasaki, Struan F. A. Grant, Achille Iolascon, Yael P. Mosse, Kristina A. Cole, Hongzhe Li, Marcella Devoto, Patrick W. McGrady, Wendy B. London, Mario Capasso, Nazneen Rahman, Hakon Hakonarson, John M. Maris"><meta name="citation_date" content="13 January 2011"><meta name="citation_issue" content="7329"><meta name="citation_volume" content="469"><meta name="citation_firstpage" content="216"><meta name="citation_doi" content="10.1038/nature09609"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3320515/?report=abstract"><meta name="citation_pmid" content="21124317"><meta name="DC.Title" content="Integrative genomics identifies LMO1 as a neuroblastoma oncogene"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Kai Wang"><meta name="DC.Contributor" content="Sharon J. Diskin"><meta name="DC.Contributor" content="Haitao Zhang"><meta name="DC.Contributor" content="Edward F. Attiyeh"><meta name="DC.Contributor" content="Cynthia Winter"><meta name="DC.Contributor" content="Cuiping Hou"><meta name="DC.Contributor" content="Robert W. Schnepp"><meta name="DC.Contributor" content="Maura Diamond"><meta name="DC.Contributor" content="Kristopher Bosse"><meta name="DC.Contributor" content="Patrick A. Mayes"><meta name="DC.Contributor" content="Joseph Glessner"><meta name="DC.Contributor" content="Cecilia Kim"><meta name="DC.Contributor" content="Edward Frackelton"><meta name="DC.Contributor" content="Maria Garris"><meta name="DC.Contributor" content="Qun Wang"><meta name="DC.Contributor" content="Wendy Glaberson"><meta name="DC.Contributor" content="Rosetta Chiavacci"><meta name="DC.Contributor" content="Le Nguyen"><meta name="DC.Contributor" content="Jayanti Jagannathan"><meta name="DC.Contributor" content="Norihisa Saeki"><meta name="DC.Contributor" content="Hiroki Sasaki"><meta name="DC.Contributor" content="Struan F. A. Grant"><meta name="DC.Contributor" content="Achille Iolascon"><meta name="DC.Contributor" content="Yael P. Mosse"><meta name="DC.Contributor" content="Kristina A. Cole"><meta name="DC.Contributor" content="Hongzhe Li"><meta name="DC.Contributor" content="Marcella Devoto"><meta name="DC.Contributor" content="Patrick W. McGrady"><meta name="DC.Contributor" content="Wendy B. London"><meta name="DC.Contributor" content="Mario Capasso"><meta name="DC.Contributor" content="Nazneen Rahman"><meta name="DC.Contributor" content="Hakon Hakonarson"><meta name="DC.Contributor" content="John M. Maris"><meta name="DC.Date" content="2011 Jan 13"><meta name="DC.Identifier" content="10.1038/nature09609"><meta name="DC.Language" content="en"><meta property="og:title" content="Integrative genomics identifies LMO1 as a neuroblastoma oncogene"><meta property="og:type" content="article"><meta property="og:description" content="Neuroblastoma is a childhood cancer of the sympathetic nervous system that accounts for approximately 10% of all paediatric oncology deaths[,] . To identify genetic risk factors for neuroblastoma, we performed a genome-wide association study (GWAS) on ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320515/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3320515/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3320515/pdf/nihms351119.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C5F51DD51D48100000000029A01AD.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3320515%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="gds" class="last">GEO DataSets</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3320515</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nature</a></span>. Author manuscript; available in PMC 2012 Apr 5.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21124317" target="pmc_ext" ref="reftype=publisher&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nature. 2011 Jan 13; 469(7329): 216–220. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Dec 1. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fnature09609" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/nature09609</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3320515</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS351119</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21124317">21124317</a></div></div></div></div><h1 class="content-title">Integrative genomics identifies <em>LMO1</em> as a neuroblastoma oncogene</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Wang%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426840863744" co-class="co-affbox">Kai Wang</a>,<sup>1,</sup><sup>†</sup><sup>*</sup> <a href="/pubmed/?term=Diskin%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140427006425952" co-class="co-affbox">Sharon J. Diskin</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Zhang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426834749824" co-class="co-affbox">Haitao Zhang</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Attiyeh%20EF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426843501328" co-class="co-affbox">Edward F. Attiyeh</a>,<sup>2</sup> <a href="/pubmed/?term=Winter%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426826091232" co-class="co-affbox">Cynthia Winter</a>,<sup>2</sup> <a href="/pubmed/?term=Hou%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140427044052288" co-class="co-affbox">Cuiping Hou</a>,<sup>1</sup> <a href="/pubmed/?term=Schnepp%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426816229184" co-class="co-affbox">Robert W. Schnepp</a>,<sup>2</sup> <a href="/pubmed/?term=Diamond%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426827524720" co-class="co-affbox">Maura Diamond</a>,<sup>2</sup> <a href="/pubmed/?term=Bosse%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426841676096" co-class="co-affbox">Kristopher Bosse</a>,<sup>2</sup> <a href="/pubmed/?term=Mayes%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426829653184" co-class="co-affbox">Patrick A. Mayes</a>,<sup>2</sup> <a href="/pubmed/?term=Glessner%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426840920896" co-class="co-affbox">Joseph Glessner</a>,<sup>1</sup> <a href="/pubmed/?term=Kim%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426844035696" co-class="co-affbox">Cecilia Kim</a>,<sup>1</sup> <a href="/pubmed/?term=Frackelton%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140427006681312" co-class="co-affbox">Edward Frackelton</a>,<sup>1</sup> <a href="/pubmed/?term=Garris%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426818226080" co-class="co-affbox">Maria Garris</a>,<sup>2</sup> <a href="/pubmed/?term=Wang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426842484544" co-class="co-affbox">Qun Wang</a>,<sup>2</sup> <a href="/pubmed/?term=Glaberson%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426820318208" co-class="co-affbox">Wendy Glaberson</a>,<sup>1</sup> <a href="/pubmed/?term=Chiavacci%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426833203312" co-class="co-affbox">Rosetta Chiavacci</a>,<sup>1</sup> <a href="/pubmed/?term=Nguyen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426828438352" co-class="co-affbox">Le Nguyen</a>,<sup>2,</sup><sup>3,</sup><sup>5</sup> <a href="/pubmed/?term=Jagannathan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426843909152" co-class="co-affbox">Jayanti Jagannathan</a>,<sup>2</sup> <a href="/pubmed/?term=Saeki%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426835683840" co-class="co-affbox">Norihisa Saeki</a>,<sup>4</sup> <a href="/pubmed/?term=Sasaki%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140427009779200" co-class="co-affbox">Hiroki Sasaki</a>,<sup>4</sup> <a href="/pubmed/?term=Grant%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426832394272" co-class="co-affbox">Struan F. A. Grant</a>,<sup>1,</sup><sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Iolascon%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426817297344" co-class="co-affbox">Achille Iolascon</a>,<sup>7,</sup><sup>11</sup> <a href="/pubmed/?term=Mosse%20YP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426832901232" co-class="co-affbox">Yael P. Mosse</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Cole%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140427009870736" co-class="co-affbox">Kristina A. Cole</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Li%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426815671056" co-class="co-affbox">Hongzhe Li</a>,<sup>3</sup> <a href="/pubmed/?term=Devoto%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426813183712" co-class="co-affbox">Marcella Devoto</a>,<sup>3,</sup><sup>5,</sup><sup>6,</sup><sup>8</sup> <a href="/pubmed/?term=McGrady%20PW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140427009560304" co-class="co-affbox">Patrick W. McGrady</a>,<sup>9</sup> <a href="/pubmed/?term=London%20WB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426841465424" co-class="co-affbox">Wendy B. London</a>,<sup>10</sup> <a href="/pubmed/?term=Capasso%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426843876944" co-class="co-affbox">Mario Capasso</a>,<sup>7,</sup><sup>11</sup> <a href="/pubmed/?term=Rahman%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426831014944" co-class="co-affbox">Nazneen Rahman</a>,<sup>12</sup> <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426837625424" co-class="co-affbox">Hakon Hakonarson</a>,<sup>1,</sup><sup>5,</sup><sup>6</sup> and  <a href="/pubmed/?term=Maris%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317" class="affpopup" co-rid="_co_idm140426822734512" co-class="co-affbox">John M. Maris</a><sup>2,</sup><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140426840863744"><h3 class="no_margin">Kai Wang</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Wang%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Kai Wang</a></div></div><div id="_co_idm140427006425952"><h3 class="no_margin">Sharon J. Diskin</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Diskin%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Sharon J. Diskin</a></div></div><div id="_co_idm140426834749824"><h3 class="no_margin">Haitao Zhang</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Zhang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Haitao Zhang</a></div></div><div id="_co_idm140426843501328"><h3 class="no_margin">Edward F. Attiyeh</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Attiyeh%20EF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Edward F. Attiyeh</a></div></div><div id="_co_idm140426826091232"><h3 class="no_margin">Cynthia Winter</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Winter%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Cynthia Winter</a></div></div><div id="_co_idm140427044052288"><h3 class="no_margin">Cuiping Hou</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Hou%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Cuiping Hou</a></div></div><div id="_co_idm140426816229184"><h3 class="no_margin">Robert W. Schnepp</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Schnepp%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Robert W. Schnepp</a></div></div><div id="_co_idm140426827524720"><h3 class="no_margin">Maura Diamond</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Diamond%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Maura Diamond</a></div></div><div id="_co_idm140426841676096"><h3 class="no_margin">Kristopher Bosse</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Bosse%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Kristopher Bosse</a></div></div><div id="_co_idm140426829653184"><h3 class="no_margin">Patrick A. Mayes</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Mayes%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Patrick A. Mayes</a></div></div><div id="_co_idm140426840920896"><h3 class="no_margin">Joseph Glessner</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Glessner%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Joseph Glessner</a></div></div><div id="_co_idm140426844035696"><h3 class="no_margin">Cecilia Kim</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Kim%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Cecilia Kim</a></div></div><div id="_co_idm140427006681312"><h3 class="no_margin">Edward Frackelton</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Frackelton%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Edward Frackelton</a></div></div><div id="_co_idm140426818226080"><h3 class="no_margin">Maria Garris</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Garris%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Maria Garris</a></div></div><div id="_co_idm140426842484544"><h3 class="no_margin">Qun Wang</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Wang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Qun Wang</a></div></div><div id="_co_idm140426820318208"><h3 class="no_margin">Wendy Glaberson</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Glaberson%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Wendy Glaberson</a></div></div><div id="_co_idm140426833203312"><h3 class="no_margin">Rosetta Chiavacci</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Chiavacci%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Rosetta Chiavacci</a></div></div><div id="_co_idm140426828438352"><h3 class="no_margin">Le Nguyen</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><p><sup>3</sup>Department of Biostatistics and Epidemiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><p><sup>5</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Nguyen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Le Nguyen</a></div></div><div id="_co_idm140426843909152"><h3 class="no_margin">Jayanti Jagannathan</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Jagannathan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Jayanti Jagannathan</a></div></div><div id="_co_idm140426835683840"><h3 class="no_margin">Norihisa Saeki</h3><p><sup>4</sup>Genetics Division, National Cancer Center Research Institute, Tokyo 104-0045, Japan</p><div>Find articles by <a href="/pubmed/?term=Saeki%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Norihisa Saeki</a></div></div><div id="_co_idm140427009779200"><h3 class="no_margin">Hiroki Sasaki</h3><p><sup>4</sup>Genetics Division, National Cancer Center Research Institute, Tokyo 104-0045, Japan</p><div>Find articles by <a href="/pubmed/?term=Sasaki%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Hiroki Sasaki</a></div></div><div id="_co_idm140426832394272"><h3 class="no_margin">Struan F. A. Grant</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><p><sup>5</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><p><sup>6</sup>Division of Human Genetics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Grant%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Struan F. A. Grant</a></div></div><div id="_co_idm140426817297344"><h3 class="no_margin">Achille Iolascon</h3><p><sup>7</sup>CEINGE Biotecnologie Avanzate, Naples 80145, Italy</p><p><sup>11</sup>Department of Biochemistry and Medical Biotechnology, University of Naples Federico II, Naples 80131 Italy</p><div>Find articles by <a href="/pubmed/?term=Iolascon%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Achille Iolascon</a></div></div><div id="_co_idm140426832901232"><h3 class="no_margin">Yael P. Mosse</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><p><sup>5</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Mosse%20YP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Yael P. Mosse</a></div></div><div id="_co_idm140427009870736"><h3 class="no_margin">Kristina A. Cole</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><p><sup>5</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Cole%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Kristina A. Cole</a></div></div><div id="_co_idm140426815671056"><h3 class="no_margin">Hongzhe Li</h3><p><sup>3</sup>Department of Biostatistics and Epidemiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Hongzhe Li</a></div></div><div id="_co_idm140426813183712"><h3 class="no_margin">Marcella Devoto</h3><p><sup>3</sup>Department of Biostatistics and Epidemiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><p><sup>5</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><p><sup>6</sup>Division of Human Genetics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania, 19104, USA</p><p><sup>8</sup>Department of Experimental Medicine, University La Sapienza, Rome 00185, Italy</p><div>Find articles by <a href="/pubmed/?term=Devoto%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Marcella Devoto</a></div></div><div id="_co_idm140427009560304"><h3 class="no_margin">Patrick W. McGrady</h3><p><sup>9</sup>Department of Statistics, University of Florida and Children’s Oncology Group, Gainesville, Florida, 32603 USA</p><div>Find articles by <a href="/pubmed/?term=McGrady%20PW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Patrick W. McGrady</a></div></div><div id="_co_idm140426841465424"><h3 class="no_margin">Wendy B. London</h3><p><sup>10</sup>Dana-Farber Children’s Hospital Cancer Center and Children’s Oncology Group, Boston, Massachusetts, 02115, USA</p><div>Find articles by <a href="/pubmed/?term=London%20WB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Wendy B. London</a></div></div><div id="_co_idm140426843876944"><h3 class="no_margin">Mario Capasso</h3><p><sup>7</sup>CEINGE Biotecnologie Avanzate, Naples 80145, Italy</p><p><sup>11</sup>Department of Biochemistry and Medical Biotechnology, University of Naples Federico II, Naples 80131 Italy</p><div>Find articles by <a href="/pubmed/?term=Capasso%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Mario Capasso</a></div></div><div id="_co_idm140426831014944"><h3 class="no_margin">Nazneen Rahman</h3><p><sup>12</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Rahman%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Nazneen Rahman</a></div></div><div id="_co_idm140426837625424"><h3 class="no_margin">Hakon Hakonarson</h3><p><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><p><sup>5</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><p><sup>6</sup>Division of Human Genetics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">Hakon Hakonarson</a></div></div><div id="_co_idm140426822734512"><h3 class="no_margin">John M. Maris</h3><p><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</p><p><sup>5</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Maris%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21124317">John M. Maris</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140426828964480_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140426828964480_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140426828964480_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Division of Oncology and Center for Childhood Cancer Research, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania 19104, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Biostatistics and Epidemiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Genetics Division, National Cancer Center Research Institute, Tokyo 104-0045, Japan</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, 19104, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Division of Human Genetics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania, 19104, USA</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>CEINGE Biotecnologie Avanzate, Naples 80145, Italy</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Experimental Medicine, University La Sapienza, Rome 00185, Italy</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Statistics, University of Florida and Children’s Oncology Group, Gainesville, Florida, 32603 USA</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Dana-Farber Children’s Hospital Cancer Center and Children’s Oncology Group, Boston, Massachusetts, 02115, USA</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Biochemistry and Medical Biotechnology, University of Naples Federico II, Naples 80131 Italy</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</div><div id="FN1">Correspondence and requests for materials should be addressed to H.H. (<a href="mailto:dev@null" data-email="ude.pohc@nosranokah" class="oemail">ude.pohc@nosranokah</a>) or J.M.M. (<a href="mailto:dev@null" data-email="ude.pohc@siram" class="oemail">ude.pohc@siram</a>)</div><div id="FN2"><sup>†</sup>Present address: Zilkha Neurogenetic Institute, Department of Psychiatry and Preventive Medicine, University of Southern California, Los Angeles, California 90089, USA.</div><div id="FN3"><sup>*</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140426828964480_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21124317" target="pmc_ext" ref="reftype=publisher&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nature</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3320515/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplement.</strong></div><div><a href="/pmc/articles/PMC3320515/bin/NIHMS351119-supplement-Supplement.pdf" data-ga-action="click_feat_suppl">NIHMS351119-supplement-Supplement.pdf</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID:&nbsp;BC5C9A91-1529-461A-9AF7-1219BE47BD4A</div></div></dd></dl></div><div id="idm140427012632144" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140427012632144title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Neuroblastoma is a childhood cancer of the sympathetic nervous system that accounts for approximately 10% of all paediatric oncology deaths<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707268">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707261">2</a></sup>. To identify genetic risk factors for neuroblastoma, we performed a genome-wide association study (GWAS) on 2,251 patients and 6,097 control subjects of European ancestry from four case series. Here we report a significant association within LIM domain only 1 (<em>LMO1</em>) at 11p15.4 (rs110419, combined <em>P</em> = 5.2 × 10<sup>−16</sup>, odds ratio of risk allele = 1.34 (95% confidence interval 1.25–1.44)). The signal was enriched in the subset of patients with the most aggressive form of the disease. <em>LMO1</em> encodes a cysteine-rich transcriptional regulator, and its paralogues (<em>LMO2</em>, <em>LMO3</em> and <em>LMO4</em>) have each been previously implicated in cancer. In parallel, we analysed genome-wide DNA copy number alterations in 701 primary tumours. We found that the <em>LMO1</em> locus was aberrant in 12.4% through a duplication event, and that this event was associated with more advanced disease (<em>P</em> &lt; 0.0001) and survival (<em>P</em> = 0.041). The germline single nucleotide polymorphism (SNP) risk alleles and somatic copy number gains were associated with increased <em>LMO1</em> expression in neuroblastoma cell lines and primary tumours, consistent with a gain-of-function role in tumorigenesis. Short hairpin RNA (shRNA)-mediated depletion of <em>LMO1</em> inhibited growth of neuroblastoma cells with high <em>LMO1</em> expression, whereas forced expression of <em>LMO1</em> in neuroblastoma cells with low <em>LMO1</em> expression enhanced proliferation. These data show that common polymorphisms at the <em>LMO1</em> locus are strongly associated with susceptibility to developing neuroblastoma, but also may influence the likelihood of further somatic alterations at this locus, leading to malignant progression.</p></div></div><div id="idm140426831331856" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Multiple somatically acquired chromosomal rearrangements, such as focal amplification of the <em>MYCN</em> oncogene or deletions at chromosome arms 1p or 11q, are each associated with an aggressive neuroblastoma phenotype<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707260">2</a></sup>. Although these somatically acquired genomic alterations are of clinical use as prognostic biomarkers, until recently little was known about the constitutional genetic events that initiate tumorigenesis. Highly penetrant gain-of-function mutations in the anaplastic lymphoma kinase (<em>ALK</em>) tyrosine kinase domain were recently identified as the major cause of familial neuroblastoma, and somatic mutations in this gene implicate it as a target for therapeutic intervention<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707259">3</a>–<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707279">6</a></sup>. In addition, a neuroblastoma GWAS identified common SNPs at 6p22 as being associated with susceptibility to aggressive neuroblastoma in sporadic cases<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707273">7</a></sup>; follow-up association analysis on the clinically relevant group of patients with an aggressive tumour phenotype indicated that common SNPs within <em>BARD1</em> also function as susceptibility variants<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707272">8</a></sup>. Finally, our GWAS has also identified a common copy number variation at 1q21.1 being highly associated with neuroblastoma and probably playing a role in early tumorigenesis through disruption of a novel neuroblastoma breakpoint family gene (<em>NBPF23</em>)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707276">9</a></sup>. Taken together, it has become clear that the embryonal cancer neuroblastoma is genetically heterogeneous, and initiation of sporadically occurring disease requires multiple interacting genetic factors, including both sequence and copy number variants.</p><p id="P3">To identify additional genetic risk factors, we expanded our previous GWAS and analysed 1,627 neuroblastoma patients accrued through the North American-based Children’s Oncology Group with 3,254 genetically matched control subjects of European ancestry (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). All subjects were genotyped using the Illumina HumanHap550 BeadChip with over 550,000 SNP markers; the genomic control inflation factor was 1.08 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). Clusters of SNPs from three genomic loci reached genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC3320515/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>), including two SNPs within <em>FLJ22536/FLJ44180</em> at the 6p22 locus (<em>P</em> values range from 2.46 × 10<sup>−14</sup> to 3.25 × 10<sup>−13</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>), nine SNPs within or nearby <em>BARD1</em> at the 2q35 locus (<em>P</em> values range from 3.05 × 10<sup>−13</sup> to 9.69 × 10<sup>−9</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>), each previously reported, and two SNPs within <em>LMO1</em> (LIM domain only 1), a newly identified neuroblastoma susceptibility locus at 11p15.4 (<em>P</em> values range from 5.12 × 10<sup>−10</sup> to 2.83 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC3320515/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC3320515/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a>). Closer examination of the <em>LMO1</em> locus identified a total of four SNPs that show strong association signals (<em>P</em> &lt; 1 × 10<sup>−4</sup>) with neuroblastoma (<a href="/pmc/articles/PMC3320515/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), which are in a moderate degree of linkage disequilibrium (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). We then examined each of the most significant SNPs from the 2q35, 6p22, 11p15.4 susceptibility loci and the 1q21.1 copy number variation. However, we did not find evidence for epistasis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 3 and 4</a>), indicating that these susceptibility loci increase disease risk independently.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3320515/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140426822816224" class="figure"><a href="/pmc/articles/PMC3320515/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3320515_nihms351119f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms351119f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3320515/bin/nihms351119f1.jpg"></a></div><div id="largeobj_idm140426822816224" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3320515/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3320515/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Discovery of <em>LMO1</em> at 11p15.4 as a neuroblastoma susceptibility locus</strong><p id="__p7"><strong>a</strong>, Manhattan plot of GWAS results from the discovery cases series, with the red horizontal line representing genome-wide significance threshold (<em>P</em> &lt; 5 × 10<sup>−8</sup>). <strong>b</strong>, Genomic position (National Center for Biotechnology Information build 36) of genotyped (triangles) and imputed (circles) SNPs. The <em>P</em> values are calculated by combining discovery and replication case series with whole-genome genotypes, and SNPs are coloured based on their correlations with rs110419 (purple diamond). Estimated recombination rates from the HapMap data are overlaid. <strong>c</strong>, Degree of linkage disequilibrium between SNPs (as <em>r</em><sup>2</sup> values) is represented by red colour intensity in the corresponding cells.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p10">Significantly associated SNPs at the <em>LMO1</em> locus on 11p15.4</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140427010924128" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="left">SNP<sup>†</sup></th><th rowspan="2" colspan="1" valign="middle" align="center">Risk/non-risk allele</th><th colspan="3" rowspan="1" valign="bottom" align="center">Discovery (HumanHap550)<sup>*</sup><hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">US replication (Human610)<sup>*</sup><hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">UK replication (TaqMan)<sup>*</sup><hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">Frequency of cases (<em>n</em> = 1,627)</th><th rowspan="1" colspan="1" valign="top" align="center">Frequency of controls (<em>n</em> = 3,254)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sup>‡</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Frequency of cases (<em>n</em> = 190)</th><th rowspan="1" colspan="1" valign="top" align="center">Frequency of controls (<em>n</em> = 1,507)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><sup>‡</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Frequency of cases (<em>n</em> = 253)</th><th rowspan="1" colspan="1" valign="top" align="center">Frequency of controls (<em>n</em> = 845)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sup>‡</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4758051</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.51</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="left">2.8 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">2.1 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.51</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="left">0.039</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10840002</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="left">6.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="left">0.61</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs110419</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">5.1 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.61</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">1.0 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="left">0.057</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs204938</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="left">1.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">0.058</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="left">0.032</td></tr></tbody></table></div><div id="largeobj_idm140427010924128" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3320515/table/T1/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140426828604944" class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="left">SNP<sup>†</sup></th><th rowspan="2" colspan="1" valign="middle" align="center">Risk/non-risk allele</th><th colspan="3" rowspan="1" valign="bottom" align="center">Italian replication (TaqMan)<sup>*</sup><hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Combined
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">Frequency of cases (<em>n</em> = 181)</th><th rowspan="1" colspan="1" valign="top" align="center">Frequency of controls (<em>n</em> = 491)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><sup>‡</sup></th><th rowspan="1" colspan="1" valign="top" align="left">CMH<sup>§</sup><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="center">CMH OR (95% confidence interval)<sup>||</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4758051</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="left">1.4 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.28 (1.19–1.37)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10840002</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">8.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 (1.12–1.30)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs110419</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="left">5.2 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.34 (1.25–1.44)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs204938</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">1.7 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.22 (1.13–1.31)</td></tr></tbody></table></div><div id="largeobj_idm140426828604944" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3320515/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>No deviations from Hardy–Weinberg equilibrium were observed (<em>P</em> &gt; 0.001) in all cohorts.</div><div id="TFN2"><sup>†</sup>SNP: <em>r</em><sup>2</sup> in controls between rs110419 and each of rs4758051, rs10840002 and rs204938 was 0.30, 0.17 and 0.29, respectively.</div><div id="TFN3"><sup>‡</sup><em>P</em> values were calculated by allelic test.</div><div id="TFN4"><sup>§</sup>CMH, Cochran–Mantel–Haenszel test.</div><div id="TFN5"><sup>||</sup>OR, odds ratio of risk allele.</div></div></div><p id="P4">To replicate our findings, we examined the association results from an independent case series of 190 patients from the Children’s Oncology Group and 1,507 control subjects, all of whom were genotyped on the Human610-Quad arrays. All four <em>LMO1</em> SNPs identified in the discovery effort showed the same direction of association in this replication cohort, with <em>P</em> values ranging from 1.01 × 10<sup>−5</sup> to 0.058. To seek additional evidence of replication, we performed quantitative PCR-based genotyping of these four SNPs in a third independent case series from UK, as well as the two most significant SNPs in a fourth independent case series from Italy. Combined analysis by the Cochran–Mantel–Haenszel method demonstrated that two of the four SNPs had <em>P</em> values that extend well beyond the genome-wide significance threshold (<a href="/pmc/articles/PMC3320515/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Additionally, using the two cohorts with whole-genome genotype data (discovery cohort and US replication cohort), we performed genotype imputation at 11p15.4 and identified six additional genome-wide significant markers, the most significant being rs110420 (<em>P</em> = 1.17 × 10<sup>−13</sup>), which is in complete linkage disequilibrium (<em>r</em><sup>2</sup> = 1 in HapMap CEU subjects (Utah residents with ancestry from northern and western Europe) with the genotyped marker rs110419 (<a href="/pmc/articles/PMC3320515/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1c</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>).</p><p id="P5">We next determined if the <em>LMO1</em> genotypes were associated with a particular clinical phenotype and/or patient survival. Similar to the association pattern observed for the 6p22 and 2q35 (<em>BARD1</em>) loci<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707266">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707277">8</a></sup>, the risk alleles of <em>LMO1</em> were significantly associated with metastatic disease (<em>P</em> = 0.0040), advanced age (greater than 1 year, <em>P</em> &lt; 0.0001) and a high-risk status by Children’s Oncology Group criteria for treatment stratification<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707262">2</a></sup> (<em>P</em> = 0.0010; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a>). Consistent with this observation, the rs110419 risk allele was associated with decreased event-free survival (<em>P</em> = 0.0085; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>) and overall survival (<em>P</em> = 0.0217; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>). Taken together, these data suggest that common germline variants at <em>LMO1</em> are associated not only with predisposition to develop neuroblastoma, but also with a predilection to develop the more aggressive form of the disease. They emphasize that <em>LMO1</em> genetic variations are associated with a particular neuroblastoma phenotype; however, this does not indicate that these variants have prognostic significance for an individual with neuroblastoma.</p><p id="P6">The <em>LMO1</em> gene encodes a cysteine-rich transcriptional regulator with two LIM zinc-binding domains that is mainly expressed in the nervous system<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707281">10</a></sup>. LMO1 belongs to a protein superfamily encoded by four genes, including <em>LMO1</em>, <em>LMO2</em>, <em>LMO3</em> and <em>LMO4</em>. Multiple lines of evidence, including chromosomal translocation events and mouse models, strongly implicate this gene family in the aetiology of human cancer<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707267">11</a>–<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707280">14</a></sup>. Most provocatively, retroviral insertion of the corrective gene for X-linked severe combined immunodeficiency into the <em>LMO2</em> locus resulted in T-cell leukaemias in several participants in gene therapy trials<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707282">15</a></sup>. <em>LMO4</em> represses the transcription of <em>BRCA1</em>, and dys-regulation of <em>LMO4</em> expression has been implicated in the breast car-cinogenesis<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707270">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707278">17</a></sup>. Finally, <em>LMO3</em> has been shown to act as an oncogene in neuroblastoma through the neuronal transcription factor <em>HEN2</em><sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707264">18</a></sup>. We therefore postulated that the common variants at the 11p15.4 locus discovered here may increase disease risk through a <em>cis</em>-acting effect on the regulation of expression or function of <em>LMO1</em>, but we cannot exclude the potential for <em>trans</em>-acting influences on loci distant from the discovered common variants.</p><p id="P7">We next examined tumour DNA genotyped on the Illumina SNP arrays for 701 neuroblastomas using a detection algorithm for copy number designed for tumour samples<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707275">19</a></sup>. We detected relative segmental gain (copy number changes at a given locus relative to whole-genome copy number changes) at <em>LMO1</em> in 87 out of 701 tumours (12.4%); this was particularly enriched in the high-risk group where the GWAS signal was most robust (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5a</a>). Most tumours with 11p gain showed a duplication of the entire chromosome p arm, but four tumours (approximately 5%) showed focal gain restricted to 11p15 including the <em>LMO1</em> locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig 5b</a>). These data demonstrate that <em>LMO1</em> is one of many genes showing somatic copy number gain on 11p, and here we used the GWAS data to prioritize it as a potential target of this somatically acquired chromosomal rearrangement.</p><p id="P8">We next examined whether somatic <em>LMO1</em> alterations were associated with neuroblastoma clinical phenotype and survival of patients (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>). Gain of <em>LMO1</em> was significantly more common in tumours from patients with metastatic disease (<em>P</em> &lt; 0.0001), advanced age (greater than 1 year, <em>P</em> &lt; 0.0001), unfavourable pathological grade (<em>P</em> = 0.0013) and Children’s Oncology Group high-risk classification (<em>P</em> &lt; 0.0001). Gain of 11p was rarely observed in the <em>MYCN</em> amplified cases (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>). Despite the strong association of 11p gain in cases without <em>MYCN</em> amplification, a known powerful adverse prognostic factor<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707274">1</a></sup>, <em>LMO1</em> gain was associated with decreased overall survival of patients (<em>P</em> = 0.041) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs 6 and 7</a>).</p><p id="P9">To investigate how the neuroblastoma-associated <em>LMO1</em> alleles may contribute to tumour initiation and/or clinical phenotype, we next genotyped a set of human neuroblastoma-derived cell lines with Illumina SNP arrays, and measured messenger RNA (mRNA) and protein expression levels on the subset of lines without copy number changes at 11p to avoid the influence of somatic DNA alterations on gene expression. Cell lines with diploid 11p status and harbouring homozygous risk alleles showed significantly higher <em>LMO1</em> mRNA and protein expression than those with homozygous non-risk alleles (<a href="/pmc/articles/PMC3320515/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 11</a>). This trend held in an expanded set of 25 neuroblastoma cell lines with variable 11p status (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 8</a>). To determine if this correlation existed in diagnostic tumour tissues, we next examined mRNA expression levels on a whole-genome Affymetrix expression microarray<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707271">20</a></sup> in a subset of 61 neuroblastoma primary tumours from patients whose blood samples and primary tumours had both been genotyped on the Illumina SNP arrays. Among these 61 tumours, 13 harboured somatic gain of 11p. Considering both somatic and germline genotypes in the same linear regression model, we detected an association between <em>LMO1</em> copy number gains and increased <em>LMO1</em> expression (<em>P</em> = 0.02; <a href="/pmc/articles/PMC3320515/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>), as well as an association between rs110419 risk alleles and increased <em>LMO1</em> expression (<em>P</em> = 0.022; <a href="/pmc/articles/PMC3320515/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). To refine the genotype-expression relationships further, we subsequently used quantitative PCR to measure <em>LMO1</em> expression in an additional set of 23 tumours without <em>LMO1</em> gain. We confirmed that the rs110419 risk allele is associated with <em>LMO1</em> expression (<em>P</em> = 0.01), independent of copy number changes (<a href="/pmc/articles/PMC3320515/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2c</span></a>). To determine whether a regulatory variant exists at a narrow promoter region of <em>LMO1</em>, we performed Sanger sequencing in 20 neuroblastoma cell lines but did not detect any potential causal variant (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 13</a>). Examination of the 1000 Genomes Project data identified over 300 SNPs within or surrounding <em>LMO1</em> that are in moderate to strong linkage disequilibrium (<em>D</em>′ &gt; 0.5) with rs110419 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 14</a>); however, fine mapping of this region through resequencing will be required to identify whether any are causal <em>cis</em>-regulatory variants. Subsequent experimentation will be required to determine if causal DNA variations directly impact <em>LMO1</em> expression, and if somatic copy-number gain indeed is targeting <em>LMO1</em> for further increased expression in tumour cells.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3320515/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140426819874496" class="figure"><a href="/pmc/articles/PMC3320515/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3320515_nihms351119f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms351119f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3320515/bin/nihms351119f2.jpg"></a></div><div id="largeobj_idm140426819874496" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3320515/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3320515/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong><em>LMO1</em> germline genotypes and somatic copy number gains are associated with mRNA and protein expression</strong><p id="__p8"><strong>a</strong>, <em>LMO1</em> mRNA and protein expression in nine human neuroblastoma-derived cell lines are highly correlated with rs110419 genotype. <strong>b</strong>, Microarray-based expression profiling on 61 primary tumours confirms that <em>LMO1</em> gene expression is associated with both <em>LMO1</em> gain (<em>P</em> = 0.02, <em>t</em>-test) and risk genotypes (<em>P</em> = 0.022, linear regression). <strong>c</strong>, Quantitative PCR-based expression profiling of an independent set of primary neuroblastomas without <em>LMO1</em> gain confirms the same association. Error bars, s.e.m.</p></div></div></div><p id="P10">As our germline and somatic genomic analyses implicated <em>LMO1</em> as a neuroblastoma oncogene, we next sought to determine the functional consequences of <em>LMO1</em> depletion or overexpression in a genotype- and expression-specific manner. First, after lentiviral-based shRNA infection of neuroblastoma cell lines, we were able to recover stable clones with 45–63% depletion of <em>LMO1</em> mRNA and protein (<a href="/pmc/articles/PMC3320515/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3e</span></a>). Cells with the homozygous neuroblastoma-associated genotype and high <em>LMO1</em> expression showed significantly decreased proliferation compared with mock-infected controls (<a href="/pmc/articles/PMC3320515/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3a, b</span></a>), whereas cells with homozygous non-risk alleles showed little phenotypic effect (<a href="/pmc/articles/PMC3320515/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3c, d</span></a>). Finally, to determine the cellular phenotypes of forced overexpression of <em>LMO1</em>, we stably overexpressed <em>LMO1</em> with approximately fourfold higher levels in the SK-N-BE2C cell line with low <em>de novo LMO1</em> expression, and detected significantly enhanced proliferation (<a href="/pmc/articles/PMC3320515/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3f</span></a>). Therefore it appears that inhibition of <em>LMO1</em> in cells expressing high levels of <em>LMO1</em> or activation of <em>LMO1</em> in cells with low levels of <em>LMO1</em> leads to pronounced phenotypes. Taken together, these data suggest that <em>LMO1</em> may function as an oncogene in a subset of human neuroblastomas.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3320515/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140426821121872" class="figure"><a href="/pmc/articles/PMC3320515/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3320515_nihms351119f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms351119f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3320515/bin/nihms351119f3.jpg"></a></div><div id="largeobj_idm140426821121872" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3320515/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3320515/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic manipulation of <em>LMO1</em> expression in neuroblastoma cell line models influences proliferative phenotype in an expression-specific manner</strong><p id="__p9"><strong>a</strong>–<strong>d</strong>, In cells with neuroblastoma risk alleles and higher <em>LMO1</em> expression levels, <em>LMO1</em> knockdown leads to inhibition of cellular proliferation. <strong>e</strong>, <em>LMO1</em> knockdown as measured by quantitative reverse-transcription PCR and western blot for experiments <strong>a–d. f</strong>, In SK-N-BE2C cells with non-risk alleles and low <em>LMO1</em> expression levels, forced overexpression of <em>LMO1</em> leads to enhanced cellular proliferation. Approximate fourfold overexpression of <em>LMO1</em> RNA and protein are shown. Error bars, s.e.m.</p></div></div></div><p id="P11" class="p">In conclusion, here we have identified germline sequence variants at the <em>LMO1</em> locus that are robustly associated with neuroblastoma. We have applied an integrative genomics approach to demonstrate that common genetic polymorphisms associated with cancer predisposition may also mark regions of the genome prone to somatic alterations influencing tumour progression. Our data suggest that GWAS studies can identify previously undiscovered oncogenic drivers of a malignant phenotype, especially when they occur in a region of the genome involved in large segmental rearrangements impacting hundreds of genes. In paediatric cancers such as neuroblastoma, the real translational potential of GWAS efforts may be in discovering therapeutic targets and predictive biomarkers of tumour aggressiveness.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS SUMMARY</h2><p id="P12" class="p p-first-last">All genome-wide SNP genotyping for the discovery cohorts was performed using the Illumina HumanHap550 BeadChip at the Center for Applied Genomics at the Children’s Hospital of Philadelphia. Multi-dimensional scaling was performed using PLINK version 1.06 on a subset of SNPs not in linkage disequilibrium to identify subjects of European ancestry, and all control subjects were genetically matched to patients. The first replication case series was genotyped by Illumina Human610 BeadChip, yet two additional replication case series were genotyped by TaqMan. Genotype imputation was performed by MACH (<a href="http://www.sph.umich.edu/csg/abecasis/MaCH/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MaCH/</a>) on discovery and replication case series with whole-genome genotypes. Alteration calls in tumour copy number were generated from data of SNP signal intensity by the OverUnder<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707269">19</a></sup>. Survival analyses used the methods of Kaplan and Meier, with standard errors following the methods of Peto <em>et al.</em><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707265">21</a></sup>. For gene expression profiling by Affymetrix U95Av2 microarrays, the expression measures for each probe set was extracted and normalized using robust multi-array average protocols from raw CEL files. Association tests on genotype and expression were performed on log-transformed expression values by linear regression or <em>t</em>-test. For quantitative PCR on <em>LMO1</em>, TaqMan probes were purchased from Applied Biosystems with assay identity Hs00231133_m1. Relative expression of the target gene was determined by normalization to <em>HPRT1</em> using a standard curve method with ten serial dilutions according to the manufacturer’s instructions. All quantitative PCR reactions were performed in triplicate with an ABI PrismTM 7900HT Sequence Detection System (Applied Biosystems). For the <em>LMO1</em> knockdown experiments, the lentiviral particles for shRNA knockdown were purchased from Santa Cruz, including copGFP Control Lentiviral Particles (catalogue number sc-108084) and LMO1 shRNA(h) Lentiviral Particles (catalogue number sc-38025-v). Pooled clones of SK-N-BE2C cells with <em>LMO1</em> overexpression were created through stable transfection of full-length <em>LMO1</em> complementary DNA in pCDNA3.1 as previously described<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256707263">22</a></sup>.</p></div><div id="S2" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplement</h4><div class="sup-box half_rhythm" id="N0xe935c20N0xf8d25e0"><a href="/pmc/articles/PMC3320515/bin/NIHMS351119-supplement-Supplement.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.2M, pdf)</sup></div></div></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title">Acknowledgments</h2><div class="sec"><p id="__p3">We acknowledge the Children’s Oncology Group for providing most blood and tumour specimens and clinical and outcome data (U10-CA98543 and U10-CA98413) from neuroblastoma patients. We thank G. P. Tonini for providing neuroblastoma DNA samples in the Italian replication cohort. This work was supported in part by National Institutes of Health grant R01-CA124709 (to J.M.M.), the Giulio D’Angio Endowed Chair (J.M.M.), the Alex’s Lemonade Stand Foundation (J.M.M.), the Evan Dunbar Foundation (J.M.M.), the Rally Foundation (J.M.M.), Andrew’s Army Foundation (J.M.M.), the Abramson Family Cancer Research Institute (J.M.M.), a Howard Hughes Medical Institute Research Training Fellowship (K.B.), a fellowship from Associazione Oncologia Pediatrica e Neuroblastoma (M.C.), a Research Development Award from the Cotswold Foundation (H.H.), UL1-RR024134-03 (H.H.) and an Institutional Development Award to the Center for Applied Genomics from the Children’s Hospital of Philadelphia (H.H.).</p></div></div><div id="idm140426839444832" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140426839444832title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="__p4" class="p p-first-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a> is linked to the online version of the paper at <a href="http://www.nature.com/nature" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.nature.com/nature</a>.</p><p></p><div><p id="__p5" class="p p-first-last"><strong>Author Contributions</strong> H.H. and J.M.M. conceived the study, guided interpretation of results and helped preparation of the manuscript. K.W., H.Z. and C.H. performed SNP association analysis. K.W., S.J.D., E.F.A. and J.J. performed gene expression and copy number analysis. C.W. and K.B. performed PCR validation of gene expression data. C.W., R.W.S., K.B., P.A.M., S.J.D. and K.A.C. performed and/or analysed shRNA transfection and <em>LMO1</em> overexpression experiments. N.S. and H.S. generated viral construct for human <em>LMO1</em> complementary DNA. M.C. and A.I. performed the replication study on the Italian case series, and N.R. performed the replication study on the UK case series. P.W.M. and W.B.L. performed outcome and clinical covariate analyses on the Children’s Oncology Group samples. C.H., C.K., E.F., M.G., W.G. and R.C. generated the genotyping data. L.N. and M.D. helped with data analysis. S.F.A.G., Y.P.M., H.L. and M.D. advised on data interpretation. K.W. drafted the manuscript; H.H., J.M.M. and other authors edited it.</p></div><p class="fn sec" id="FN6"></p><p id="__p6" class="p p-first-last"><strong>Author Information</strong> Microarray data are deposited in the GEO database under accession number <a id="__tag_256707283" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE3960" ref="/geo/query/acc.cgi?acc=GSE3960"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE3960","term_id":"3960"}}</span>GSE3960</a>. The genotypic and phenotypic information from this study is deposited in dbGaP (<a href="http://www.ncbi.nlm.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ncbi.nlm.gov/gap</a>) under accession number phs000124.v2.p1. Reprints and permissions information is available at <a href="http://www.nature.com/reprints" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.nature.com/reprints</a>. The authors declare no competing financial interests. Readers are welcome to comment on the online version of this article at <a href="http://www.nature.com/nature" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.nature.com/nature</a>.</p><p></p></div></div><div id="idm140426834603456" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140426834603456title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Maris JM. Recent advances in neuroblastoma. <span><span class="ref-journal">N Engl J Med. </span>2010;<span class="ref-vol">362</span>:2202–2211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3306838/">PMC free article</a>]</span> [<a href="/pubmed/20558371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Recent+advances+in+neuroblastoma&amp;author=JM+Maris&amp;volume=362&amp;publication_year=2010&amp;pages=2202-2211&amp;pmid=20558371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Maris JM, Hogarty MD, Bagatell R, Cohn SL. Neuroblastoma. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">369</span>:2106–2120.</span> [<a href="/pubmed/17586306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Neuroblastoma&amp;author=JM+Maris&amp;author=MD+Hogarty&amp;author=R+Bagatell&amp;author=SL+Cohn&amp;volume=369&amp;publication_year=2007&amp;pages=2106-2120&amp;pmid=17586306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Mosse YP, et al.  Identification of ALK as a major familial neuroblastoma predisposition gene. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:930–935.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672043/">PMC free article</a>]</span> [<a href="/pubmed/18724359" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Identification+of+ALK+as+a+major+familial+neuroblastoma+predisposition+gene&amp;author=YP+Mosse&amp;volume=455&amp;publication_year=2008&amp;pages=930-935&amp;pmid=18724359&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Janoueix-Lerosey I, et al.  Somatic and germline activating mutations of the ALK kinase receptor in neuroblastoma. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:967–970.</span> [<a href="/pubmed/18923523" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Somatic+and+germline+activating+mutations+of+the+ALK+kinase+receptor+in+neuroblastoma&amp;author=I+Janoueix-Lerosey&amp;volume=455&amp;publication_year=2008&amp;pages=967-970&amp;pmid=18923523&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Chen Y, et al.  Oncogenic mutations of ALK kinase in neuroblastoma. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:971–974.</span> [<a href="/pubmed/18923524" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Oncogenic+mutations+of+ALK+kinase+in+neuroblastoma&amp;author=Y+Chen&amp;volume=455&amp;publication_year=2008&amp;pages=971-974&amp;pmid=18923524&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">George RE, et al.  Activating mutations in ALK provide a therapeutic target in neuroblastoma. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:975–978.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2587486/">PMC free article</a>]</span> [<a href="/pubmed/18923525" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Activating+mutations+in+ALK+provide+a+therapeutic+target+in+neuroblastoma&amp;author=RE+George&amp;volume=455&amp;publication_year=2008&amp;pages=975-978&amp;pmid=18923525&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Maris JM, et al.  Chromosome 6p22 locus associated with clinically aggressive neuroblastoma. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:2585–2593.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2742373/">PMC free article</a>]</span> [<a href="/pubmed/18463370" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Chromosome+6p22+locus+associated+with+clinically+aggressive+neuroblastoma&amp;author=JM+Maris&amp;volume=358&amp;publication_year=2008&amp;pages=2585-2593&amp;pmid=18463370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Capasso M, et al.  Common variations in BARD1 influence susceptibility to high-risk neuroblastoma. <span><span class="ref-journal">Nature Genet. </span>2009;<span class="ref-vol">41</span>:718–723.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2753610/">PMC free article</a>]</span> [<a href="/pubmed/19412175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=Common+variations+in+BARD1+influence+susceptibility+to+high-risk+neuroblastoma&amp;author=M+Capasso&amp;volume=41&amp;publication_year=2009&amp;pages=718-723&amp;pmid=19412175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Diskin SJ, et al.  Copy number variation at 1q21.1 associated with neuroblastoma. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">459</span>:987–991.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2755253/">PMC free article</a>]</span> [<a href="/pubmed/19536264" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Copy+number+variation+at+1q21.1+associated+with+neuroblastoma&amp;author=SJ+Diskin&amp;volume=459&amp;publication_year=2009&amp;pages=987-991&amp;pmid=19536264&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Su AI, et al.  A gene atlas of the mouse and human protein-encoding transcriptomes. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2004;<span class="ref-vol">101</span>:6062–6067.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC395923/">PMC free article</a>]</span> [<a href="/pubmed/15075390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes&amp;author=AI+Su&amp;volume=101&amp;publication_year=2004&amp;pages=6062-6067&amp;pmid=15075390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Rabbitts TH. LMO T-cell translocation oncogenes typify genes activated by chromosomal translocations that alter transcription and developmental processes. <span><span class="ref-journal">Genes Dev. </span>1998;<span class="ref-vol">12</span>:2651–2657.</span> [<a href="/pubmed/9732263" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=LMO+T-cell+translocation+oncogenes+typify+genes+activated+by+chromosomal+translocations+that+alter+transcription+and+developmental+processes&amp;author=TH+Rabbitts&amp;volume=12&amp;publication_year=1998&amp;pages=2651-2657&amp;pmid=9732263&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Rabbitts TH, et al.  The effect of chromosomal translocations in acute leukemias: the LMO2 paradigm in transcription and development. <span><span class="ref-journal">Cancer Res. </span>1999;<span class="ref-vol">59</span>:1794s–1798s.</span> [<a href="/pubmed/10197599" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=The+effect+of+chromosomal+translocations+in+acute+leukemias:+the+LMO2+paradigm+in+transcription+and+development&amp;author=TH+Rabbitts&amp;volume=59&amp;publication_year=1999&amp;pages=1794s-1798s&amp;pmid=10197599&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Fisch P, et al.  T-cell acute lymphoblastic lymphoma induced in transgenic mice by the RBTN1 and RBTN2 LIM-domain genes. <span><span class="ref-journal">Oncogene. </span>1992;<span class="ref-vol">7</span>:2389–2397.</span> [<a href="/pubmed/1461647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=T-cell+acute+lymphoblastic+lymphoma+induced+in+transgenic+mice+by+the+RBTN1+and+RBTN2+LIM-domain+genes&amp;author=P+Fisch&amp;volume=7&amp;publication_year=1992&amp;pages=2389-2397&amp;pmid=1461647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Neale GA, Rehg JE, Goorha RM. Disruption of T-cell differentiation precedes T-cell tumor formation in LMO-2 (rhombotin-2) transgenic mice. <span><span class="ref-journal">Leukemia. </span>1997;<span class="ref-vol">11</span>(suppl 3):289–290.</span> [<a href="/pubmed/9209368" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Disruption+of+T-cell+differentiation+precedes+T-cell+tumor+formation+in+LMO-2+(rhombotin-2)+transgenic+mice&amp;author=GA+Neale&amp;author=JE+Rehg&amp;author=RM+Goorha&amp;volume=11&amp;issue=suppl+3&amp;publication_year=1997&amp;pages=289-290&amp;pmid=9209368&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Hacein-Bey-Abina S, et al.  LMO2-associated clonal T cell proliferation in two patients after gene therapy for SCID-X1. <span><span class="ref-journal">Science. </span>2003;<span class="ref-vol">302</span>:415–419.</span> [<a href="/pubmed/14564000" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=LMO2-associated+clonal+T+cell+proliferation+in+two+patients+after+gene+therapy+for+SCID-X1&amp;author=S+Hacein-Bey-Abina&amp;volume=302&amp;publication_year=2003&amp;pages=415-419&amp;pmid=14564000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Sum EY, et al.  The LIM domain protein LMO4 interacts with the cofactor CtIP and the tumor suppressor BRCA1 and inhibits BRCA1 activity. <span><span class="ref-journal">J Biol Chem. </span>2002;<span class="ref-vol">277</span>:7849–7856.</span> [<a href="/pubmed/11751867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+LIM+domain+protein+LMO4+interacts+with+the+cofactor+CtIP+and+the+tumor+suppressor+BRCA1+and+inhibits+BRCA1+activity&amp;author=EY+Sum&amp;volume=277&amp;publication_year=2002&amp;pages=7849-7856&amp;pmid=11751867&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Visvader JE, et al.  The LIM domain gene LMO4 inhibits differentiation of mammary epithelial cells <em>in vitro</em> and is overexpressed in breast cancer. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2001;<span class="ref-vol">98</span>:14452–14457.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC64702/">PMC free article</a>]</span> [<a href="/pubmed/11734645" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=The+LIM+domain+gene+LMO4+inhibits+differentiation+of+mammary+epithelial+cells+in+vitro+and+is+overexpressed+in+breast+cancer&amp;author=JE+Visvader&amp;volume=98&amp;publication_year=2001&amp;pages=14452-14457&amp;pmid=11734645&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Aoyama M, et al.  LMO3 interacts with neuronal transcription factor, HEN2, and acts as an oncogene in neuroblastoma. <span><span class="ref-journal">Cancer Res. </span>2005;<span class="ref-vol">65</span>:4587–4597.</span> [<a href="/pubmed/15930276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=LMO3+interacts+with+neuronal+transcription+factor,+HEN2,+and+acts+as+an+oncogene+in+neuroblastoma&amp;author=M+Aoyama&amp;volume=65&amp;publication_year=2005&amp;pages=4587-4597&amp;pmid=15930276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Attiyeh EF, et al.  Genomic copy number determination in cancer cells from single nucleotide polymorphism microarrays based on quantitative genotyping corrected for aneuploidy. <span><span class="ref-journal">Genome Res. </span>2009;<span class="ref-vol">19</span>:276–283.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2652209/">PMC free article</a>]</span> [<a href="/pubmed/19141597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Genomic+copy+number+determination+in+cancer+cells+from+single+nucleotide+polymorphism+microarrays+based+on+quantitative+genotyping+corrected+for+aneuploidy&amp;author=EF+Attiyeh&amp;volume=19&amp;publication_year=2009&amp;pages=276-283&amp;pmid=19141597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Wang Q, et al.  Integrative genomics identifies distinct molecular classes of neuroblastoma and shows that multiple genes are targeted by regional alterations in DNA copy number. <span><span class="ref-journal">Cancer Res. </span>2006;<span class="ref-vol">66</span>:6050–6062.</span> [<a href="/pubmed/16778177" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Integrative+genomics+identifies+distinct+molecular+classes+of+neuroblastoma+and+shows+that+multiple+genes+are+targeted+by+regional+alterations+in+DNA+copy+number&amp;author=Q+Wang&amp;volume=66&amp;publication_year=2006&amp;pages=6050-6062&amp;pmid=16778177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Peto R, et al.  Design and analysis of randomized clinical trials requiring prolonged observation of each patient. II. analysis and examples. <span><span class="ref-journal">Br J Cancer. </span>1977;<span class="ref-vol">35</span>:1–39.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2025310/">PMC free article</a>]</span> [<a href="/pubmed/831755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Design+and+analysis+of+randomized+clinical+trials+requiring+prolonged+observation+of+each+patient.+II.+analysis+and+examples&amp;author=R+Peto&amp;volume=35&amp;publication_year=1977&amp;pages=1-39&amp;pmid=831755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Saeki N, et al.  GASDERMIN, suppressed frequently in gastric cancer, is a target of LMO1 in TGF-beta-dependent apoptotic signalling. <span><span class="ref-journal">Oncogene. </span>2007;<span class="ref-vol">26</span>:6488–6498.</span> [<a href="/pubmed/17471240" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=GASDERMIN,+suppressed+frequently+in+gastric+cancer,+is+a+target+of+LMO1+in+TGF-beta-dependent+apoptotic+signalling&amp;author=N+Saeki&amp;volume=26&amp;publication_year=2007&amp;pages=6488-6498&amp;pmid=17471240&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3320515&amp;issue-id=208463&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140427012632144title">Abstract</a></li><li><a href="#S1title">METHODS SUMMARY</a></li><li><a href="#S2title">Supplementary Material</a></li><li><a href="#S3title">Acknowledgments</a></li><li><a href="#idm140426839444832title">Footnotes</a></li><li><a href="#idm140426834603456title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3320515/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3320515/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3320515/pdf/nihms351119.pdf">PDF (1.3M)</a></li> | <li><a href="#" data-citationid="PMC3320515" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3320515%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3320515%2F&amp;text=Integrative%20genomics%20identifies%20LMO1%20as%20a%20neuroblastoma%20oncogene" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3320515%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21124317" data-aiid="3320515" data-aid="3320515" data-iid="208463" data-domainid="319" data-domain="nihpa" data-accid="PMC3320515" data-md5="a2f7694bff9eba5f3d17b846cb1951fd"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:20:26-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C5F51DD51D48100000000029A01AD&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3320515%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3320515/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>