<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3832188/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies genes that may contribute to risk for developing heroin addiction</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3832188/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Psychiatric genetics"><meta name="citation_title" content="Genome-wide association study identifies genes that may contribute to risk for developing heroin addiction"><meta name="citation_authors" content="David A. Nielsen, Fei Ji, Vadim Yuferov, Ann Ho, Chunsheng He, Jurg Ott, Mary Jeanne Kreek"><meta name="citation_date" content="October 2010"><meta name="citation_issue" content="5"><meta name="citation_volume" content="20"><meta name="citation_doi" content="10.1097/YPG.0b013e32833a2106"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3832188/?report=abstract"><meta name="citation_pmid" content="20520587"><meta name="DC.Title" content="Genome-wide association study identifies genes that may contribute to risk for developing heroin addiction"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="David A. Nielsen"><meta name="DC.Contributor" content="Fei Ji"><meta name="DC.Contributor" content="Vadim Yuferov"><meta name="DC.Contributor" content="Ann Ho"><meta name="DC.Contributor" content="Chunsheng He"><meta name="DC.Contributor" content="Jurg Ott"><meta name="DC.Contributor" content="Mary Jeanne Kreek"><meta name="DC.Date" content="2010 Oct"><meta name="DC.Identifier" content="10.1097/YPG.0b013e32833a2106"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies genes that may contribute to risk for developing heroin addiction"><meta property="og:type" content="article"><meta property="og:description" content="We have used genome-wide association studies to identify variants that are associated with vulnerability to develop heroin addiction.DNA from 325 methadone stabilized, former severe heroin addicts and 250 control subjects were pooled by ethnicity (Caucasian ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3832188/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3832188/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3832188/pdf/nihms-203717.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE882E58DD5115810000000001E90155.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832188%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gquery" class="last">All Databases</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3832188</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Psychiatr Genet</a></span>. Author manuscript; available in PMC 2013 Nov 18.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20520587" target="pmc_ext" ref="reftype=publisher&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Psychiatr Genet. 2010 Oct; 20(5): 10.1097/YPG.0b013e32833a2106. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1097%2FYPG.0b013e32833a2106" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1097/YPG.0b013e32833a2106</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3832188</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS203717</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20520587">20520587</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies genes that may contribute to risk for developing heroin addiction</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Nielsen%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587" class="affpopup" co-rid="_co_idm140042274597488" co-class="co-affbox">David A. Nielsen</a>,<sup>a,</sup><sup>b</sup> <a href="/pubmed/?term=Ji%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587" class="affpopup" co-rid="_co_idm140042274595504" co-class="co-affbox">Fei Ji</a>,<sup>c</sup> <a href="/pubmed/?term=Yuferov%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587" class="affpopup" co-rid="_co_idm140042274593856" co-class="co-affbox">Vadim Yuferov</a>,<sup>a</sup> <a href="/pubmed/?term=Ho%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587" class="affpopup" co-rid="_co_idm140042271423952" co-class="co-affbox">Ann Ho</a>,<sup>a</sup> <a href="/pubmed/?term=He%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587" class="affpopup" co-rid="_co_idm140042271421968" co-class="co-affbox">Chunsheng He</a>,<sup>c</sup> <a href="/pubmed/?term=Ott%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587" class="affpopup" co-rid="_co_idm140042280020768" co-class="co-affbox">Jurg Ott</a>,<sup>c,</sup><sup>d</sup> and  <a href="/pubmed/?term=Kreek%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587" class="affpopup" co-rid="_co_idm140042280018048" co-class="co-affbox">Mary Jeanne Kreek</a><sup>a</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140042274597488"><h3 class="no_margin">David A. Nielsen</h3><p><sup>a</sup>Laboratory of the Biology of Addictive Diseases, The Rockefeller University, New York, NY </p><p><sup>b</sup>Menninger Department of Psychiatry and Behavioral Sciences, Baylor College of Medicine, and Michael E. DeBakey V.A. Medical Center, Houston, Texas, Houston, TX </p><div>Find articles by <a href="/pubmed/?term=Nielsen%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587">David A. Nielsen</a></div></div><div id="_co_idm140042274595504"><h3 class="no_margin">Fei Ji</h3><p><sup>c</sup>Laboratory of Statistical Genetics, The Rockefeller University, New York, NY, USA </p><div>Find articles by <a href="/pubmed/?term=Ji%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587">Fei Ji</a></div></div><div id="_co_idm140042274593856"><h3 class="no_margin">Vadim Yuferov</h3><p><sup>a</sup>Laboratory of the Biology of Addictive Diseases, The Rockefeller University, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Yuferov%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587">Vadim Yuferov</a></div></div><div id="_co_idm140042271423952"><h3 class="no_margin">Ann Ho</h3><p><sup>a</sup>Laboratory of the Biology of Addictive Diseases, The Rockefeller University, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Ho%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587">Ann Ho</a></div></div><div id="_co_idm140042271421968"><h3 class="no_margin">Chunsheng He</h3><p><sup>c</sup>Laboratory of Statistical Genetics, The Rockefeller University, New York, NY, USA </p><div>Find articles by <a href="/pubmed/?term=He%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587">Chunsheng He</a></div></div><div id="_co_idm140042280020768"><h3 class="no_margin">Jurg Ott</h3><p><sup>c</sup>Laboratory of Statistical Genetics, The Rockefeller University, New York, NY, USA </p><p><sup>d</sup>Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China </p><div>Find articles by <a href="/pubmed/?term=Ott%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587">Jurg Ott</a></div></div><div id="_co_idm140042280018048"><h3 class="no_margin">Mary Jeanne Kreek</h3><p><sup>a</sup>Laboratory of the Biology of Addictive Diseases, The Rockefeller University, New York, NY </p><div>Find articles by <a href="/pubmed/?term=Kreek%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20520587">Mary Jeanne Kreek</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140042279881808_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140042279881808_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140042279881808_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>a</sup>Laboratory of the Biology of Addictive Diseases, The Rockefeller University, New York, NY </div><div class="fm-affl" id="A2" lang="en"><sup>b</sup>Menninger Department of Psychiatry and Behavioral Sciences, Baylor College of Medicine, and Michael E. DeBakey V.A. Medical Center, Houston, Texas, Houston, TX </div><div class="fm-affl" id="A3" lang="en"><sup>c</sup>Laboratory of Statistical Genetics, The Rockefeller University, New York, NY, USA </div><div class="fm-affl" id="A4" lang="en"><sup>d</sup>Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China </div><div id="CR1">Correspondence and requests for reprints to: M. J. Kreek: Laboratory of the Biology of Addictive Diseases, The Rockefeller University, Box 171, 1230 York Avenue, New York, NY 10065, Tel: 212.327.8490, Fax: 212.327.8574, <a href="mailto:dev@null" data-email="ude.rellefekcor@keerk" class="oemail">ude.rellefekcor@keerk</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140042279881808_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20520587" target="pmc_ext" ref="reftype=publisher&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Psychiatr Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3832188/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC3832188/bin/NIHMS203717-supplement-1.doc" data-ga-action="click_feat_suppl">NIHMS203717-supplement-1.doc</a><span style="color:gray"> (3.1M)</span></div><div class="small guid">GUID:&nbsp;CA00232D-93D9-43D3-BAD6-A6B96F565FD2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC3832188/bin/NIHMS203717-supplement-2.doc" data-ga-action="click_feat_suppl">NIHMS203717-supplement-2.doc</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID:&nbsp;04A3D93E-B366-46B7-97F5-B6B2FCCECC24</div></div></dd></dl></div><div id="idm140042273559904" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140042273559904title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Objectives</h3><p id="P1" class="p p-first-last">We have used genome-wide association studies to identify variants that are associated with vulnerability to develop heroin addiction.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">DNA from 325 methadone stabilized, former severe heroin addicts and 250 control subjects were pooled by ethnicity (Caucasian and African American) and analyzed using the Affymetrix GeneChip Mapping 100K Set. Genome-wide association tests were conducted.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">The strongest association with vulnerability to develop heroin addiction, with experiment-wise significance (<em>P</em> = 0.035), was found in Caucasians with the variant rs10494334, a variant in an unannotated region of the genome (1q23.3). In African Americans, the variant most significantly associated with heroin addiction vulnerability was rs950302, found in the cytosolic dual specificity phosphatase 27 gene <em>DUSP27</em> (point-wise <em>P</em> = 0.0079). Furthermore, analysis of the top 500 variants with the most significant associations (point-wise <em>P</em> ≤ 0.0036) in Caucasians showed that three of these variants are clustered in the regulating synaptic membrane exocytosis protein 2 gene <em>RIMS2</em>. Of the top 500 variants in African Americans (point-wise <em>P</em> ≤ 0.0238), three variants are in the cardiomyopathy associated 3 gene <em>CMYA3</em>.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">This study identifies new genes and variants that may increase an individual’s vulnerability to develop heroin addiction.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Addiction, gene, heroin, microarray, polymorphism</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first">Factors that contribute to the development of heroin addiction, besides the drug of abuse itself, are both genetic and environmental. Epidemiological studies indicate that genetic factors, both those shared by addiction to several drugs of abuse and those specific for addiction to a given drug, contribute approximately 40-60% of the risk of developing heroin addiction (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583904">Kendler et al., 2003</a>, <a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583935">Tsuang et al., 1998</a>, <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583917">Tsuang et al., 1996</a>). One study found 38% genetic variance specific to the development of heroin addiction, a variance higher than that for marijuana, stimulants, sedatives, or psychedelics (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583899">Tsuang et al., 1998</a>). Genetic predisposition may explain why approximately one-third of those self-exposed to opiates become addicted (<a href="#R18" rid="R18" class=" bibr popnode">Kreek, 2002</a>).</p><p id="P6">Recently, using the Affymetrix 10K GeneChip, we found a genotype pattern of three unlinked alleles that was associated with developing heroin addiction and another pattern that was associated with protection from developing heroin addiction (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583905">Nielsen et al., 2008b</a>). That study suggested a role for five genes in the risk of developing heroin addiction including the gene coding for the μ opioid receptor.</p><p id="P7">An alternative method of genotyping large cohorts is to analyze pools consisting of multiple DNA samples (reviewed in (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583945">Sham et al., 2002</a>)), which was developed to reduce genotyping costs. Analysis of pooled samples still allows a comparison of allele frequencies between cases and controls. Pools are made from cases or control subjects, either as one pool containing the case samples and the other containing the controls, or as multiple pools of each category of subjects. This technique has been used successfully to find differences in allele frequencies using microarrays (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583930">Kirov et al., 2006</a>, <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583933">Liu et al., 2006</a>, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583937">Liu et al., 2005</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385129753">Johnson et al., 2006</a>, <a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583903">Uhl et al., 2007</a>, <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583898">Uhl et al., 2008</a>).</p><p id="P8" class="p p-last">Herein, we have used a pooled sample approach with the Affymetrix Gene Mapping 100K Set to identify variants and genes that may be associated with vulnerability to develop heroin addiction in a large cohort of volunteers ascertained in New York City. The subjects for this genome-wide association study were former severe heroin addicts or control subjects, with extensively defined phenotypic characterization.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Materials and methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Subjects and phenotyping</h3><p id="P9" class="p p-first">Two hundred and fifty control subjects and 325 former severe heroin addicts, who were consecutive volunteers in genetic studies in our laboratory at The Rockefeller University and who met the inclusion criteria defined below were included in this study. They were recruited from advertisements and from clinical resources in New York City. An informed consent approved by The Rockefeller University Hospital Institutional Review Board that gave specific consent for genetic studies was signed by all subjects. Based on the ethnic/cultural background of the subjects, their parents, grandparents, and great-grandparents, subjects were classified as Caucasian (n = 350) or African American (n = 225).</p><p id="P10" class="p p-last">The Addiction Severity Index (ASI) (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583912">McLellan et al., 1980</a>) was administered to and urine analyses were conducted for multiple drugs of abuse in all subjects. Former severe heroin addicts met Federal guidelines for methadone maintenance treatment (one year or more of daily multiple injections of heroin or other opiates) (<a href="#R30" rid="R30" class=" bibr popnode">Rettig and Yarmolinsky, 1995</a>). Control subjects had no history of alcohol or illicit drug use (illicit drug use or drinking to intoxication three or more times per week for 6 months or more), no use of cannabis (three or more times per week) for more than 4 years, and no current alcohol or illicit drug use (one or more instance of drinking to intoxication or any illicit drug use (except for possible cannabis use up to 12 days) in the last 30 days).</p></div><div id="S8" class="sec"><h3 id="S8title">Sample Pooling</h3><p id="P11" class="p p-first-last">Each pool was made in duplicate from the DNA of 25 subjects. Genomic DNA from each subject was diluted to approximately 10 μg/ml and the concentration determined using the Rediplate 96 PicoGreen assay (Invitrogen, Carlsbad CA). For each pool, 50 ng DNA of each subject was used, and the final pooled DNA was diluted to 300 μl with TE (0.1 mM EDTA, 10 mM Tris, pH 8.0). Each duplicate pool was made separately. There were six pools from Caucasian controls, eight pools from Caucasian former heroin addicts (cases), four pools from African American controls, and five pools from African American cases (<a href="/pmc/articles/PMC3832188/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p3">Pool composition</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140042270673168" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="right">Pools of<br>unique<br>subjects</th><th rowspan="1" colspan="1" valign="middle" align="right">Total<br>pools<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="left">Ethnicity</th><th rowspan="1" colspan="1" valign="middle" align="left">Category</th><th rowspan="1" colspan="1" valign="middle" align="center">No. of subjects</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="right">16</td><td rowspan="1" colspan="1" valign="middle" align="left">Caucasian</td><td rowspan="1" colspan="1" valign="middle" align="left">Severe former heroin addicts</td><td rowspan="1" colspan="1" valign="middle" align="center">200</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">12</td><td rowspan="1" colspan="1" valign="middle" align="left">Caucasian</td><td rowspan="1" colspan="1" valign="middle" align="left">Controls</td><td rowspan="1" colspan="1" valign="middle" align="center">150</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">5</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="left">African American</td><td rowspan="1" colspan="1" valign="middle" align="left">Severe former heroin addicts</td><td rowspan="1" colspan="1" valign="middle" align="center">125</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">4</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="left">African American</td><td rowspan="1" colspan="1" valign="middle" align="left">Control</td><td rowspan="1" colspan="1" valign="middle" align="center">100</td></tr><tr><td colspan="5" rowspan="1" valign="middle" align="right">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">Total: 23</td><td rowspan="1" colspan="1" valign="middle" align="right">46</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">575</td></tr></tbody></table></div><div id="largeobj_idm140042270673168" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832188/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Each pool was created separately (i.e., in duplicate) from DNA of 25 different subjects to yield a total of 46 pools.</div></div></div></div><div id="S9" class="sec"><h3 id="S9title">Microarrays</h3><p id="P12" class="p p-first-last">Two separate hybridizations were performed for each of the DNA pools using the Affymetrix 50K Hind 240 and the 50K Xba 240 arrays (Affymetrix, Santa Clara, CA). Five hundred ng DNA (in 120 μl) from each pool was ethanol precipitated, resuspended in 10 μl TE, and processed according to manufacturer’s protocol (Affymetrix). Processed arrays were scanned using an Affymetrix GeneChip Scanner 3000. Data was acquired using the GeneChip Operating System (GCOS) and analyzed using the GTYPE genotype Analysis Software (Affymetrix). Variant annotations are available from the NetAffx Analysis Center (<a href="http://www.affymetrix.com/analysis/index.affx" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.affymetrix.com/analysis/index.affx</a>).</p></div><div id="S10" class="sec"><h3 id="S10title">Data analysis</h3><p id="P13" class="p p-first">Separate analyses of Caucasians and African Americans were conducted to avoid errors due to population stratification. For each DNA microarray (one of two duplicate pools), a “background” value was calculated as the average fluorescence intensity from the 5% of cells with the lowest values; the “background” value was then subtracted from the intensity value of each cell. The background subtracted intensity values were then normalized by dividing by a “ceiling” value, which was the mean intensity of the 5% of cells with the highest values. The normalized A and B probe intensities were averaged from ten “perfect match” intensity values for each A and B probe. The methods are outlined in Johnson, <em>et al.</em> (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385129750">Johnson et al., 2006</a>). The ratio of averaged and normalized A probe intensity to the sum of averaged and normalized A plus B probe intensity was determined for each variant. The pooled A allele frequency was the average ratio from the duplicate arrays, and this value was used below.</p><p id="P14">The GeneChip Mapping 100K Set contains 116,204 variants. Since our pools contained both males and females, 2,361 X chromosomal variants could not be evaluated. No Y chromosomal variants are represented on the 100K Set. Hence, analyses were performed on autosomal variants only. In addition, the 100K Set contains 644 variants with no annotation. After exclusion due to low allele frequency (&lt; 0.03 within a single ethnic group), 113,135 variants in the Caucasians and 113,174 variants in the African Americans were evaluated further.</p><p id="P15">To test for differences in allele frequency between the cases and controls for each of the variants within each ethnic group, a two sample non-parametric t-test was conducted. Multivariate permutation (<a href="#R35" rid="R35" class=" bibr popnode">Simon et al., 2004</a>) was used to correct for multiple testing, and experiment-wise <em>P</em> values are reported. To perform permutation testing for the experiment-wise <em>P</em> value, the class labels were permuted and the <em>t</em> statistic values for each of the markers were recalculated. The maximum <em>t</em> statistic (corresponding to minimum <em>P</em> value) of all ~110,000 tests (one test for each marker) from this permutation was taken. This procedure was repeated for 3,003 permutations of the data in the Caucasian group. The value 3,003 was obtained by selecting 6 controls (or 8 cases) out of 14 pooled samples. The originally observed <em>t</em> statistic was compared to the distribution of <em>t</em> statistics composed of 3003 maximum <em>t</em> statistics to obtain the experiment-wise <em>P</em> value. For example, an experiment-wise <em>P</em> value of 0.035 means that 105 out of 3,003 permuted maximum <em>t</em> statistics were higher than or equal to our observed <em>t</em> statistic. The <em>P</em> value obtained by this method is called the multivariate <em>P</em> value.</p><p id="P16" class="p p-last">There is a high degree of correlation between many of the variants in a genome-wide association study due to linkage disequilibrium across the chromosomes. If we corrected for multiple testing using Bonferroni or False Discovery Rate (FDR), we would be discounting the correlation between the markers and over correcting our <em>P</em> value. Permutation testing allows us to maintain the correlation structure between the variants and calculate a global cut-off where any <em>P</em> values that are smaller than that observed value will have an experiment-wise significance. This approach allows the correlation among variants and is therefore less conservative than the Bonferroni approach. <em>P</em> values are reported for point-wise (nominal) significance and experiment-wise (corrected) significance.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Variant analysis</h3><p id="P17" class="p p-first">The finding of a significant association between a variant and a phenotype may be due several factors. The variant itself may modify function by altering the coding sequence of the gene, the stability of the resulting mRNA (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583941">Duan, Wainwright et al. 2003</a>), or the regulation of gene expression. Regulatory variants may be found far upstream of genes. For example, a number of variants have been identified upstream of the <em>SOX9</em> gene which are associated with the palatal lesion Pierre Robin sequence (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583896">Benko, Fantes et al. 2009</a>). One variant is located 1.44 million nucleotides upstream of <em>SOX9</em> and alters several predicted transcription binding sites. Other examples include two variants found upstream of the <em>SHH</em> gene. One is located one million nucleotides upstream of the <em>SHH</em> gene (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583942">Lettice, Heaney et al. 2003</a>) and was found to be associated with preaxial polydactyly, while the other is located 470,000 nucleotides upstream and was associated with holoprosencephaly (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583918">Jeong, Leskow et al. 2008</a>). Using 11,446 genes in a Bayesian hierarchical model, the Pritchard group found that 5% of the quantitative trait loci for gene expression (eQTLs) were located more than 20,000 nucleotides upstream of the transcription start sites (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583926">Veyrieras, Kudaravalli et al. 2008</a>).</p><p id="P18" class="p p-last">Significant associations may also be due to the variant being in linkage disequilibrium with a functional variant. While linkage disequilibrium (LD) decreases with increasing distance between markers, studies of some genes have shown that LD may be quite high past 100,000 nucleotides (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583921">Collins et al., 1999</a>, <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583902">Reich et al., 2001</a>). In this study, if an annotated gene was found within 100,000 nucleotides of a variant, the gene’s location relative to that variant is indicated. Mammalian conservation was determined using the “Vertebrate Multiz Alignment &amp; PhastCons Conservation (28 species)” and the “Evolutionary and Sequence Pattern Extraction through Reduced Representation” (ESPERR) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583931">King et al., 2005</a>) to evaluate predicted regulatory potential as implemented in the UCSC Genome Browser (March 2006 assembly, <a href="http://gemome.ucsc.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gemome.ucsc.edu/</a>). The “Transcription Element Search System” (TESS) was used to scan sequences for predicted transcription factor binding sites (<a href="#R32" rid="R32" class=" bibr popnode">Schug and Overton, 1977</a>).</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Results</h2><p id="P19" class="p p-first">In this study, we analyzed duplicate pooled DNA samples using the GeneChip Mapping 100K Set. Pools were formed by combining an equal amount of DNA from 25 subjects (<a href="/pmc/articles/PMC3832188/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). A total of 200 Caucasian former heroin addicts (cases), 150 Caucasian controls, 125 African American cases, and 100 African American controls were used to make the pools. More than 113,000 variants were examined in each ethnicity (see Data analysis).</p><div id="S13" class="sec"><h3 id="S13title">Association analyses</h3><p id="P20" class="p p-first">The ten variants with the smallest experiment-wise <em>P</em> values of association of allele frequency with heroin addiction in Caucasians and African Americans are listed in <a href="/pmc/articles/PMC3832188/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC3832188/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3,</span></span><span>3</span></a>, respectively. The variant rs10494334 (located at 1q23.3) had the smallest point-wise <em>P</em> value (<em>P</em> = 0.0003) for association of allele frequency with heroin addiction in the Caucasian group, and this association was significant experiment-wise when corrected for multiple testing (<em>P</em> = 0.035) (<a href="/pmc/articles/PMC3832188/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). An in-depth analysis of this intergenic variant and surrounding sequence revealed no annotated gene within 100,000 nucleotides or any indication of this variant altering known or predicted function. Variant rs2323218 was ranked second by ascending <em>P</em> value and is located 8,989 nucleotides downstream of the <em>T</em> gene, which codes for an embryonic nuclear transcription factor (point-wise <em>P</em> = 0.0003) (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583894">Edwards et al., 1996</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p4">Top 10 variants ranked by ascending experiment-wise <em>P</em> value based on analysis of the association of allele frequency with heroin addiction in the Caucasian group</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140042270802448" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="left">Rank<br>Order</th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="2" rowspan="1" valign="middle" align="center">Allele frequency, A<sup>a</sup><hr></th><th rowspan="2" colspan="1" valign="middle" align="center">Experiment-<br>wise <em>P</em><sup>b</sup></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="2" colspan="1" valign="middle" align="left">Variant<br>location<sup>c</sup></th><th rowspan="2" colspan="1" valign="middle" align="center">Distance<br>(bases)</th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Variant</th><th rowspan="1" colspan="1" valign="middle" align="center">Control</th><th rowspan="1" colspan="1" valign="middle" align="center">Case</th><th rowspan="1" colspan="1" valign="middle" align="left">Cytoband</th><th rowspan="1" colspan="1" valign="middle" align="left">Gene</th><th rowspan="1" colspan="1" valign="middle" align="left">Gene description</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10494334</td><td rowspan="1" colspan="1" valign="middle" align="center">0.18</td><td rowspan="1" colspan="1" valign="middle" align="center">0.23</td><td rowspan="1" colspan="1" valign="middle" align="center">0.035</td><td rowspan="1" colspan="1" valign="middle" align="left">1q23.3</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2323218</td><td rowspan="1" colspan="1" valign="middle" align="center">0.18</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25</td><td rowspan="1" colspan="1" valign="middle" align="center">0.221</td><td rowspan="1" colspan="1" valign="middle" align="left">6q27</td><td rowspan="1" colspan="1" valign="middle" align="left">down<sup>d</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">8,989</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>T</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">T, brachyury homolog</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7923687</td><td rowspan="1" colspan="1" valign="middle" align="center">0.83</td><td rowspan="1" colspan="1" valign="middle" align="center">0.79</td><td rowspan="1" colspan="1" valign="middle" align="center">0.426</td><td rowspan="1" colspan="1" valign="middle" align="left">10q25.1</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">4</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6121489</td><td rowspan="1" colspan="1" valign="middle" align="center">0.03</td><td rowspan="1" colspan="1" valign="middle" align="center">0.05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.523</td><td rowspan="1" colspan="1" valign="middle" align="left">20q13.33</td><td rowspan="1" colspan="1" valign="middle" align="left">intron<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CDH4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">cadherin 4, type 1, R-cadherin (retinal)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3849399</td><td rowspan="1" colspan="1" valign="middle" align="center">0.78</td><td rowspan="1" colspan="1" valign="middle" align="center">0.71</td><td rowspan="1" colspan="1" valign="middle" align="center">0.605</td><td rowspan="1" colspan="1" valign="middle" align="left">2p24.1</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">6</td><td rowspan="1" colspan="1" valign="middle" align="left">rs9271</td><td rowspan="1" colspan="1" valign="middle" align="center">0.75</td><td rowspan="1" colspan="1" valign="middle" align="center">0.68</td><td rowspan="1" colspan="1" valign="middle" align="center">0.620</td><td rowspan="1" colspan="1" valign="middle" align="left">17q25.3</td><td rowspan="1" colspan="1" valign="middle" align="left">3′UTR<sup>f</sup></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>WDR45L</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">DR45-like</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">7</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10518620</td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center"> 0.84</td><td rowspan="1" colspan="1" valign="middle" align="center">0.632</td><td rowspan="1" colspan="1" valign="middle" align="left">4q28.3</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3893249</td><td rowspan="1" colspan="1" valign="middle" align="center">0.79</td><td rowspan="1" colspan="1" valign="middle" align="center">0.72</td><td rowspan="1" colspan="1" valign="middle" align="center">0.640</td><td rowspan="1" colspan="1" valign="middle" align="left">2p24.1</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1711055</td><td rowspan="1" colspan="1" valign="middle" align="center">0.30</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25</td><td rowspan="1" colspan="1" valign="middle" align="center">0.736</td><td rowspan="1" colspan="1" valign="middle" align="left">15q22.1</td><td rowspan="1" colspan="1" valign="middle" align="left">intron</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>AQP9</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">aquaporin 9</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">10</td><td rowspan="1" colspan="1" valign="middle" align="left">rs724729</td><td rowspan="1" colspan="1" valign="middle" align="center">0.73</td><td rowspan="1" colspan="1" valign="middle" align="center">0.63</td><td rowspan="1" colspan="1" valign="middle" align="center">0.741</td><td rowspan="1" colspan="1" valign="middle" align="left">15q14</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr></tbody></table></div><div id="largeobj_idm140042270802448" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3832188/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>a</sup>A allele is defined by Affymetrix and can be found at the NetAffx Analysis Center (<a href="http://www.affymetrix.com/analysis/index.affx" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.affymetrix.com/analysis/index.affx</a>).</div><div id="TFN3"><sup>b</sup>Point-wise <em>P</em> value was 0.0003 for all the alleles listed. The <em>t</em> statistic of the observed data was calculated and then the case/control labels of each subject were permuted. For each permutation, a <em>t</em> statistic was obtained. For a sample size of 14 with 8 cases and 6 controls, there are 3,003 total possible permutations. The value 0.0003 was obtained when the observed <em>t</em> statistic is the smallest among 3,003 <em>t</em> statistics (1/3,003 = 0.0003). The originally observed <em>t</em> statistic was compared to the distribution of <em>t</em> statistics composed of 3,003 maximum <em>t</em> statistics to obtain the experiment-wise <em>P</em> value.</div><div id="TFN4"><sup>c</sup>Variant location is given when variant is found with 100,000 nucleotides of an annotated gene.</div><div id="TFN5"><sup>d</sup>down = distance the variant is located downstream of the polyadenylation site of the specified gene.</div><div id="TFN6"><sup>e</sup>intron = variant is located in the intron of specified gene.</div><div id="TFN7"><sup>f</sup>3′UTR = variant is located in the 3; untranslated region of the specified gene.</div><div id="TFN8"><p id="P44" class="p p-first-last">Data is from the NetAffx web site; verified and corrected using the USCS Genome Browser.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p5">Top 10 variants ranked by ascending experiment-wise <em>P</em> value based on analysis of the association of allele frequency with heroin addiction in the African American group</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140042272205296" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="left">Rank<br>Order</th><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="2" rowspan="1" valign="top" align="center">Allele frequency, A<hr></th><th rowspan="2" colspan="1" valign="top" align="left">Experiment<br>wise <em>P</em><sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="2" colspan="1" valign="top" align="left">Variant<br>location<sup>b</sup></th><th rowspan="2" colspan="1" valign="top" align="center">Distance<br>(bases)</th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Variant</th><th rowspan="1" colspan="1" valign="top" align="center">Control</th><th rowspan="1" colspan="1" valign="top" align="center">Case</th><th rowspan="1" colspan="1" valign="top" align="left">Cytoband</th><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">Gene description</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">rs950302</td><td rowspan="1" colspan="1" valign="top" align="center">0.58</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="left">0.214</td><td rowspan="1" colspan="1" valign="top" align="left">1q24.1</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left">
<em>DUSP27</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">dual specificity<br>phosphatase 27 (putative)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">rs990937</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="left">0.651</td><td rowspan="1" colspan="1" valign="top" align="left">9p22.3</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">rs10513523</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="left">0.873</td><td rowspan="1" colspan="1" valign="top" align="left">3q25.32</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">rs579533</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="left">0.976</td><td rowspan="1" colspan="1" valign="top" align="left">6q25.3</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">rs4075106</td><td rowspan="1" colspan="1" valign="top" align="center">0.78</td><td rowspan="1" colspan="1" valign="top" align="center">0.70</td><td rowspan="1" colspan="1" valign="top" align="left">0.984</td><td rowspan="1" colspan="1" valign="top" align="left">12p13.31</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CD163L1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">CD163 molecule-like 1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">rs2140014</td><td rowspan="1" colspan="1" valign="top" align="center">0.68</td><td rowspan="1" colspan="1" valign="top" align="center">0.82</td><td rowspan="1" colspan="1" valign="top" align="left">0.992</td><td rowspan="1" colspan="1" valign="top" align="left">8q21.13</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7</td><td rowspan="1" colspan="1" valign="top" align="left">rs4597125</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">12q21.2</td><td rowspan="1" colspan="1" valign="top" align="left">down</td><td rowspan="1" colspan="1" valign="top" align="center">10,655</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>PHLDA1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">pleckstrin homology-like<br>domain, family A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">rs1494066</td><td rowspan="1" colspan="1" valign="top" align="center">0.52</td><td rowspan="1" colspan="1" valign="top" align="center">0.63</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">5q15.33</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">rs10495722</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">2p24.1</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">rs9289317</td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">3q21.3</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left">
<em>MGLL</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">monoglyceride lipase isoform 1</td></tr></tbody></table></div><div id="largeobj_idm140042272205296" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3832188/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN9"><sup>a</sup>Point-wise <em>P</em> value was 0.0079 for all the alleles (refer to footnote <sup>b</sup> of <a href="/pmc/articles/PMC3832188/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>)</div></div></div><p id="P21" class="p p-last">In African Americans, the variant with the smallest <em>P</em> value for association of allele frequency with heroin addiction was variant rs950302 (point-wise <em>P</em> = 0.0079, <a href="/pmc/articles/PMC3832188/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). This variant is in the second intron of the dual specificity phosphatase 27 (putative) gene <em>DUSP27</em>. This gene encodes a recently identified member of the cytosolic dual specificity phosphatase family, which may be involved in energy metabolism (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583925">Friedberg et al., 2007</a>).</p></div><div id="S14" class="sec"><h3 id="S14title">Chromosomal regions with multiple variants</h3><p id="P22" class="p p-first-last">We searched for chromosomal regions that contained at least three variants associated with heroin addiction that were in close proximity with each other. Variants were sorted by ascending <em>P</em> value in each ethnic group (Caucasian and African American) and the 500 variants with the smallest <em>P</em> value (point-wise <em>P</em> ≤ 0.0037 in the Caucasians and <em>P</em> ≤ 0.0238 in the African Americans) were evaluated to determine if any set of at least three variants were within 100,000 nucleotides of each other (<a href="/pmc/articles/PMC3832188/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>, point-wise <em>P</em> ≤ 0.0020 for Caucasians and point-wise <em>P</em> ≤ 0.0079 for African Americans). In Caucasians, three variants from this list, spanning 14,000 nucleotides, are found in an intron of the regulating synaptic membrane exocytosis protein 2 gene <em>RIMS2</em>. The cardiomyopathy associated 3 gene <em>CMYA3</em> in African Americans has three variants in a single intron that spans 3,000 nucleotides; another set of three variants are located at chromosome 13q13.1, spanning a 9,000 nucleotide region with no known function.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p6">Sets of three or more variants within 100,000 nucleotides of each other that are in the top 500 most significant variant list</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140042347107104" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Variant</th><th colspan="2" rowspan="1" valign="middle" align="center">Allele frequency (A)<hr></th><th rowspan="2" colspan="1" valign="middle" align="center">Absolute<br>difference<br>in allele<br>frequency</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>P</em>
</th><th rowspan="2" colspan="1" valign="middle" align="left">Experiment-<br>wise <em>P</em></th><th rowspan="2" colspan="1" valign="middle" align="right">Rank of<br><em>P</em></th><th rowspan="2" colspan="1" valign="middle" align="left">Distance<br>to next<br>variant<br>(nucleotides)</th><th rowspan="1" colspan="1" valign="middle" align="center">Location</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">Control</th><th rowspan="1" colspan="1" valign="middle" align="center">Case</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th></tr></thead><tbody><tr><td colspan="9" rowspan="1" valign="middle" align="left">
<strong>A. Caucasian</strong>
</td></tr><tr><td colspan="9" rowspan="1" valign="middle" align="left">
<strong><em>RIMS2</em>: Regulating synaptic membrane exocytosis protein 2 (chromosome 8q22.3)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs2511571</td><td rowspan="1" colspan="1" valign="middle" align="center">0.59</td><td rowspan="1" colspan="1" valign="middle" align="center">0.66</td><td rowspan="1" colspan="1" valign="middle" align="center">0.07</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0007</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="right">146</td><td rowspan="1" colspan="1" valign="middle" align="left">2,031</td><td rowspan="1" colspan="1" valign="middle" align="center">intron</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs2511576</td><td rowspan="1" colspan="1" valign="middle" align="center">0.63</td><td rowspan="1" colspan="1" valign="middle" align="center">0.71</td><td rowspan="1" colspan="1" valign="middle" align="center">0.08</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0007</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="right">178</td><td rowspan="1" colspan="1" valign="middle" align="left">12,304</td><td rowspan="1" colspan="1" valign="middle" align="center">intron</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs1156813</td><td rowspan="1" colspan="1" valign="middle" align="center">0.33</td><td rowspan="1" colspan="1" valign="middle" align="center">0.26</td><td rowspan="1" colspan="1" valign="middle" align="center">0.07</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0003</td><td rowspan="1" colspan="1" valign="middle" align="left">0.98</td><td rowspan="1" colspan="1" valign="middle" align="right">34</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">intron</td></tr><tr><td colspan="9" rowspan="1" valign="middle" align="left">
<strong>B. African American i.</strong>
</td></tr><tr><td colspan="9" rowspan="1" valign="middle" align="left">
<strong>i. <em>CMYA3</em>: cardiomyopathy associated 3 (chromosome 2q24.3)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs1429931</td><td rowspan="1" colspan="1" valign="middle" align="center">0.73</td><td rowspan="1" colspan="1" valign="middle" align="center">0.79</td><td rowspan="1" colspan="1" valign="middle" align="center">0.07</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0238</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="right">364</td><td rowspan="1" colspan="1" valign="middle" align="left">428</td><td rowspan="1" colspan="1" valign="middle" align="center">intron</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs6706115</td><td rowspan="1" colspan="1" valign="middle" align="center">0.67</td><td rowspan="1" colspan="1" valign="middle" align="center">0.74</td><td rowspan="1" colspan="1" valign="middle" align="center">0.08</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0079</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="right">136</td><td rowspan="1" colspan="1" valign="middle" align="left">2,913</td><td rowspan="1" colspan="1" valign="middle" align="center">intron</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs10497320</td><td rowspan="1" colspan="1" valign="middle" align="center">0.20</td><td rowspan="1" colspan="1" valign="middle" align="center">0.17</td><td rowspan="1" colspan="1" valign="middle" align="center">0.03</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0079</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">intron</td></tr><tr><td colspan="9" rowspan="1" valign="middle" align="left">
<strong>ii. No gene within 100,000 nucleotides (chromosome 13q13.1)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs1937387</td><td rowspan="1" colspan="1" valign="middle" align="center">0.63</td><td rowspan="1" colspan="1" valign="middle" align="center">0.54</td><td rowspan="1" colspan="1" valign="middle" align="center">0.09</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0079</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="right">157</td><td rowspan="1" colspan="1" valign="middle" align="left">3,132</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs2876780</td><td rowspan="1" colspan="1" valign="middle" align="center">0.77</td><td rowspan="1" colspan="1" valign="middle" align="center">0.66</td><td rowspan="1" colspan="1" valign="middle" align="center">0.11</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0079</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="right">155</td><td rowspan="1" colspan="1" valign="middle" align="left">5,877</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> rs9318868</td><td rowspan="1" colspan="1" valign="middle" align="center">0.73</td><td rowspan="1" colspan="1" valign="middle" align="center">0.63</td><td rowspan="1" colspan="1" valign="middle" align="center">0.10</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0079</td><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140042347107104" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832188/table/T4/?report=objectonly">Open in a separate window</a></div></div></div><div id="S15" class="sec sec-last"><h3 id="S15title">Candidate gene association</h3><p id="P23" class="p p-first">In another approach, we examined 240 genes that may play a role in the vulnerability to develop addiction (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583909">Nielsen et al., 2008b</a>). On the 100K GeneChip, 2,081 variants were within 100,000 nucleotides of 167 of these genes (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Material, Table S1</a>). We also examined 153 other genes from a list of the Gershon laboratory (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583936">Hattori et al., 2005</a>). One thousand one hundred fifty-nine variants are within 100,000 nucleotides of 111 of these genes (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Material, Table S2</a>). Of these, ten variants with the smallest <em>P</em> values in each of the two ethnic groups examined (Caucasian and African American) are listed in <a href="/pmc/articles/PMC3832188/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>. In Caucasians, the variant with the smallest point-wise <em>P</em> value was in the second intron of the metabotropic glutamate receptor 8 gene <em>GRM8</em> (point-wise <em>P</em> = 0.0003). The variant with the second smallest <em>P</em> value was 44,000 nucleotides upstream of the neural cell adhesion molecule 1 <em>NCAM1</em> (point-wise <em>P</em> = 0.0003) (<a href="/pmc/articles/PMC3832188/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5A</span></a>). In the African American group, the variant with the smallest <em>P</em> value was in the gene encoding the cAMP-specific phosphodiesterase 4B <em>PDE4B</em> (point-wise <em>P</em> = 0.0079) and the variant with the second smallest <em>P</em> value was in the <em>N</em>-methyl D-aspartate 2A ionotropic glutamate receptor gene <em>GRIN2A</em> (point-wise <em>P</em> = 0.0079) (<a href="/pmc/articles/PMC3832188/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5B</span></a>). None of these variants were found to have experiment-wise significant association with heroin addiction.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p7">Genes from the hypothesis-based gene lists of the Kreek (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>) and Gershon (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S2</a>) containing the ten variants ranked by significance</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140042274872432" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Symbol</th><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">Gene<br>location<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Variant</th><th rowspan="1" colspan="1" valign="top" align="left">Variant<br>location</th><th rowspan="1" colspan="1" valign="top" align="left">Distance<br>from closet<br>annotated<br>gene<br>(nucleotides)</th><th rowspan="1" colspan="1" valign="top" align="center">Allele<br>associated<br>with heroin<br>addiction<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">T-statistic</th><th rowspan="1" colspan="1" valign="top" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="top" align="left">Rank<sup>c</sup></th></tr></thead><tbody><tr><td colspan="10" rowspan="1" valign="top" align="left">
<strong>A. Caucasian</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GRM8</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">glutamate receptor, metabotropic 8</td><td rowspan="1" colspan="1" valign="top" align="left">7q31.33</td><td rowspan="1" colspan="1" valign="top" align="left">rs6467108</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="left">5.56</td><td rowspan="1" colspan="1" valign="top" align="center">0.0003</td><td rowspan="1" colspan="1" valign="top" align="left">1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>NCAM1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">neural cell adhesion molecule 1</td><td rowspan="1" colspan="1" valign="top" align="left">11q23.1</td><td rowspan="1" colspan="1" valign="top" align="left">rs1245124</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="left">5.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.0003</td><td rowspan="1" colspan="1" valign="top" align="left">2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GABBR2</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">gamma-aminobutyric acid (GABA)<br>B receptor, 2</td><td rowspan="1" colspan="1" valign="top" align="left">9q22.33</td><td rowspan="1" colspan="1" valign="top" align="left">rs2779577</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="left">4.99</td><td rowspan="1" colspan="1" valign="top" align="center">0.0003</td><td rowspan="1" colspan="1" valign="top" align="left">3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GABRB2</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">gamma-aminobutyric acid (GABA)<br>A receptor, beta 2</td><td rowspan="1" colspan="1" valign="top" align="left">5q34</td><td rowspan="1" colspan="1" valign="top" align="left">rs10515827</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="left">4.89</td><td rowspan="1" colspan="1" valign="top" align="center">0.0003</td><td rowspan="1" colspan="1" valign="top" align="left">4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>KCNA1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">potassium voltage-gated channel,<br>shaker-related subfamily, member 1</td><td rowspan="1" colspan="1" valign="top" align="left">12p13.32</td><td rowspan="1" colspan="1" valign="top" align="left">rs10492065</td><td rowspan="1" colspan="1" valign="top" align="left">down</td><td rowspan="1" colspan="1" valign="top" align="left">21,439</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="left">6.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.0007</td><td rowspan="1" colspan="1" valign="top" align="left">5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GRIK4</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">glutamate receptor, ionotropic,<br>kainate 4</td><td rowspan="1" colspan="1" valign="top" align="left">11q23.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs10502240</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="left">5.88</td><td rowspan="1" colspan="1" valign="top" align="center">0.0007</td><td rowspan="1" colspan="1" valign="top" align="left">6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>KCND2</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">potassium voltage-gated channel,<br>Shal-related subfamily, member 2</td><td rowspan="1" colspan="1" valign="top" align="left">7q31.31</td><td rowspan="1" colspan="1" valign="top" align="left">rs917901</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="left">5.50</td><td rowspan="1" colspan="1" valign="top" align="center">0.0007</td><td rowspan="1" colspan="1" valign="top" align="left">7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GABRG3</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">gamma-aminobutyric acid (GABA)<br>A receptor, gamma 3</td><td rowspan="1" colspan="1" valign="top" align="left">15q12</td><td rowspan="1" colspan="1" valign="top" align="left">rs1378101</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="left">4.97</td><td rowspan="1" colspan="1" valign="top" align="center">0.0007</td><td rowspan="1" colspan="1" valign="top" align="left">8</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>HTR4</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">5-hydroxytryptamine (serotonin)<br>receptor 4</td><td rowspan="1" colspan="1" valign="top" align="left">5q33.1</td><td rowspan="1" colspan="1" valign="top" align="left">rs9325102</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="left">5.02</td><td rowspan="1" colspan="1" valign="top" align="center">0.0010</td><td rowspan="1" colspan="1" valign="top" align="left">9</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>OLIG2</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">oligodendrocyte lineage<br>transcription factor 2</td><td rowspan="1" colspan="1" valign="top" align="left">21q22.11</td><td rowspan="1" colspan="1" valign="top" align="left">rs2180</td><td rowspan="1" colspan="1" valign="top" align="left">up</td><td rowspan="1" colspan="1" valign="top" align="left">53,874</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="left">5.0</td><td rowspan="1" colspan="1" valign="top" align="center">0.0010</td><td rowspan="1" colspan="1" valign="top" align="left">10</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left">
<strong>B. African American</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>PDE4B</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">phosphodiesterase 4B, cAMP-specific</td><td rowspan="1" colspan="1" valign="top" align="left">1p31.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs10493398</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="left">7.20</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GRIN2A</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">glutamate receptor, ionotropic,<br>N-methyl D-aspartate 2A</td><td rowspan="1" colspan="1" valign="top" align="left">16p13.2</td><td rowspan="1" colspan="1" valign="top" align="left">rs10518152</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="left">5.73</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GRM5</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">glutamate receptor, metabotropic 5</td><td rowspan="1" colspan="1" valign="top" align="left">11q14.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs524874</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="left">5.51</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GLRA3</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">glycine receptor, alpha 3</td><td rowspan="1" colspan="1" valign="top" align="left">4q34.1</td><td rowspan="1" colspan="1" valign="top" align="left">rs1491402</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="left">4.58</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>SERPINA6</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">serpin peptidase inhibitor, clade A</td><td rowspan="1" colspan="1" valign="top" align="left">14q32.13</td><td rowspan="1" colspan="1" valign="top" align="left">rs8023023</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="left">4.47</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>PLA2G4F</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">phospholipase A2,<br>group IVD (cytosolic)</td><td rowspan="1" colspan="1" valign="top" align="left">15q15.1</td><td rowspan="1" colspan="1" valign="top" align="left">rs3825786</td><td rowspan="1" colspan="1" valign="top" align="left">down</td><td rowspan="1" colspan="1" valign="top" align="left">2,257</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="left">4.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>GRIA4</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">glutamate receptor, ionotropic,<br>AMPA 4</td><td rowspan="1" colspan="1" valign="top" align="left">11q22.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs502300</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="left">4.05</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>DTNBP1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">dystrobrevin binding protein 1</td><td rowspan="1" colspan="1" valign="top" align="left">6p22.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs9296978</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="left">3.84</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">8</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">rs2743868</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="left">3.80</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">9</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>TPH2</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">tryptophan hydroxylase 2</td><td rowspan="1" colspan="1" valign="top" align="left">12q21.1</td><td rowspan="1" colspan="1" valign="top" align="left">rs10506645</td><td rowspan="1" colspan="1" valign="top" align="left">intron</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="left">3.80</td><td rowspan="1" colspan="1" valign="top" align="center">0.0079</td><td rowspan="1" colspan="1" valign="top" align="left">10</td></tr></tbody></table></div><div id="largeobj_idm140042274872432" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3832188/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN10"><sup>a</sup>Gene location from UCSC Genome Browser, March 2006 assembly (NCBI Build 36.1).</div><div id="TFN11"><sup>b</sup>Allele more frequent in cases than controls</div><div id="TFN12"><sup>c</sup>Rank is based on ascending <em>P</em> value. If the point-wise non-parametric <em>P</em>s for several variants are the same, the variant with the larger T-statistic is ranked higher.</div></div></div><p id="P24" class="p p-last">All the variants in <a href="/pmc/articles/PMC3832188/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>, except three, are located within the annotated genes listed. We found the linkage disequilibrium, D’ and r<sup>2</sup>, between each of the three variants that were close to, but not within, the annotated gene and a second variant within the annotated gene using data from HapMap (release 27) (<a href="http://www.hapmap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.hapmap.org</a>). Between variant rs10492065 downstream of <em>KCNA1</em> and rs4766311 in <em>KCNA1</em> (26,423 nucleotides apart), D’ = 0.97 and r<sup>2</sup> = 0.027 in the Caucasian (CEPH) population, and between rs3825786 downstream of <em>PLA2G4F</em> and rs2280248 in <em>PLA2G4F</em> (2,660 nucleotides apart), D’ = 1 and r<sup>2</sup> = 0.149 in the Yoruba population. The linkage disequilibrium was low between rs2180 upstream of the <em>OLIGO2</em> gene and rs6517137 in <em>OLIGO2</em> (56,500 nucleotides apart) (D’ = 0.14, r<sup>2</sup> = 0.001) in the CEPH population). However, this variant may itself be functional.</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Discussion</h2><p id="P25" class="p p-first">We used a pooled DNA approach to identify single nucleotide polymorphisms associated with vulnerability to develop heroin addiction in Caucasians and African Americans. We found differences in the results between the ethnic groups studied. Other studies have found significant associations in some specific ethnicities, but not in others, most likely due to differences in allele frequencies and/or disease prevalence among ethnic groups [reviewed in (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583901">Cardon and Palmer, 2003</a>)].</p><p id="P26">In Caucasians, variant rs10494334, located on chromosome 1, had the smallest <em>P</em> values for association with vulnerability to develop heroin addiction, which was significant after correcting for multiple testing. This variant has no annotated genes within 100,000 nucleotides. However, there is a region of approximately 50 nucleotides, located 600 nucleotides from this variant, that is highly conserved. This region is rich in predicted transcription factor binding sites and could be a region controlling expression of many genes (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583897">Morley et al., 2004</a>).</p><p id="P27">The variant with the next smallest <em>P</em> value in the Caucasians was in the <em>T</em> gene that encodes a transcription factor, the T protein, required for posterior mesoderm differentiation and axial development (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583916">Edwards et al., 1996</a>). A variant in the <em>T</em> gene (TIVS<sub>7</sub>-2) has been reported to be associated with neural tube defects (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583895">Morrison et al., 1996</a>, <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583911">Shields et al., 2000</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583932">Jensen et al., 2004</a>).</p><p id="P28">Three variants in close proximity in the <em>RIMS2</em> gene were found in the 500 variants with the smallest <em>P</em> values in the Caucasian group. <em>RIMS2</em> codes for three isozymes, RIM2α, RIM2β, and RIM2γ, members of the RIM family of Rab3-interacting molecules (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583934">Wang and Sudhof, 2003</a>). RIMs are found in the active zone of presynaptic nerve terminals, the sites on the plasma membrane where synaptic vesicle exocytosis occurs (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583922">Wang et al., 2000</a>).</p><p id="P29">Several variants were found to be clustered in the cardiomyopathy associated 3 gene <em>CMYA3</em> in the African Americans. <em>CYMA3</em>, also known as <em>XIRP2</em>, the xin actin-binding repeat containing 2 gene, is primarily expressed in striated muscle. Another variant in <em>CMYA3</em>, rs3749004 (not in our study), has been reported to be associated with autism (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385129752">Faham et al., 2005</a>).</p><p id="P30">A finding of particular interest in this study corroborated our earlier study using the 10K GeneChip with Caucasians only (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583938">Nielsen et al., 2008b</a>). Different variants in the two studies implicate the metabotropic glutamate receptor 8 gene <em>GRM8</em> in the vulnerability to develop heroin addiction. In the earlier study, variant rs1034576, located 15,742 nucleotides downstream from <em>GRM8</em>, had the second smallest <em>P</em> value for the candidate genes. In our current study, this variant had the 339<sup>th</sup> smallest <em>P</em> value. In the Caucasians in our current study, variant rs6467108 (not on the 10K GeneChip) had the smallest <em>P</em> value of the candidate genes and is located in an intron of <em>GRM8</em>. Binding of glutamate to mGluR8 inhibits cAMP production and mGluR8 has been suggested to be a presynaptic receptor that modulates glutamate release (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583940">Scherer et al., 1997</a>). Glutamate neurotransmission in the ventral tegmental area and in the nucleus accumbens core were shown to be required for heroin seeking and reinforcement in rodents (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583913">Xi and Stein, 2002</a>, <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583908">LaLumiere and Kalivas, 2008</a>). Variants in <em>GRM8</em> were found to be associated with schizophrenia in Japanese (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583944">Takaki et al., 2004</a>).</p><p id="P31">We recently reported an association of a haplotype of tryptophan hydroxylase 2 gene <em>TPH2</em> with heroin addiction vulnerability in African Americans, as well as a significant association of a genotype pattern of <em>TPH1</em> and <em>TPH2</em> variants with heroin addiction in Hispanics (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583924">Nielsen et al., 2008a</a>). In the current study, we found that in African Americans, variant rs10506645 in <em>TPH2</em> was in the top ten of our list of candidate genes. This result supports our previous findings.</p><p id="P32">Other association studies of drug and alcohol addiction have used a similar DNA pooling approach. With a 1,494 variant chip, the Uhl group identified several variants that were associated with drug abuse vulnerability (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583939">Uhl et al., 2001</a>). They extended their study with pools of African- and European-American cohorts using the 10K GeneChip and found 38 “nominally reproducibly positive” variants associated with non-specific substance abuse (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583919">Liu et al., 2005</a>). Then, using the 100K GeneChip Set, they identified 51 “clustered positive” regions associated with alcohol dependence in European-Americans (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385129751">Johnson et al., 2006</a>). Based on these studies and a study that used 500K GeneChips, 89 genes were suggested to play a role in addiction vulnerability (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583920">Liu et al., 2006</a>). In another study that used both 100K and 500K GeneChips, 39 genes that were identified as “clustered positive” were found to be in association with methamphetamine dependence (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583906">Uhl et al., 2008</a>). Comparisons between studies are difficult to make as different platforms and dissimilar subject phenotyping were used.</p><p id="P33">Several linkage studies suggest the involvement of specific chromosomal regions in the vulnerability to develop heroin addiction (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583900">Gelernter et al., 2006</a>, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583915">Glatt et al., 2006</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583907">Glatt et al., 2008</a>). The initial linkage study by the Tsuang group using 192 Chinese families and 386 short tandem repeat (STR) markers found a region on chromosome 4 at D4S1644 with evidence for linkage with heroin dependence (point-wise <em>P</em> = 0.014) (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583929">Glatt et al., 2006</a>). In their follow-up study, which included the original 192 families with a total of 397 Chinese families and 385 STR markers, the linkage signal on chromosome 4 at D4S1644 with greater significance (point-wise <em>P</em> = 0.004) (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583910">Glatt et al., 2008</a>). Variant rs10518620, ranked seventh in <a href="/pmc/articles/PMC3832188/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> in our study and nominally associated with heroin addiction in the Caucasian group (point-wise <em>P</em> = 0.0003), is located between D4S1644 and the nearby marker D4S2394, which had nominal significance (point-wise <em>P</em> = 0.013) in the later Tsuang study. In the linkage study of Gelernter, which used 409 STR markers in 393 families, a linkage peak at D17S785 was found (LOD = 3.06, empirical <em>P</em> = 0.0002) (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583928">Gelernter et al., 2006</a>). Variant rs9271, which was ranked sixth in <a href="/pmc/articles/PMC3832188/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> in our current study and was found to be nominally associated with heroin addiction in the Caucasians (point-wise <em>P</em> = 0.0003), is located six million nucleotides from D17S785.</p><p id="P34">There are two recent linkage studies on opioid dependence using single nucleotide polymorphisms (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583923">Lachman et al., 2007</a>, <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583914">Yu et al., 2008</a>). The Gelernter group found eight variants with point-wise significance in an association study of opiate dependence (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583943">Yu et al., 2008</a>), and Lachman <em>et al.</em> found one region at chromosome 14q in their Hispanic group that was “suggestive” of genome-wide evidence for linkage (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341583927">Lachman et al., 2007</a>). We did not confirm these linkage findings in our study.</p><p id="P35" class="p p-last">Future association studies in other cohorts of well-defined ethnicity and carefully defined phenotypes will be required to replicate our findings, which have identified several new candidate genes that may be involved in vulnerability to develop heroin addiction. If confirmed, these findings could lead to new targets for strategies for prevention and the pharmacological treatment of heroin addiction.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d36e838"><a href="/pmc/articles/PMC3832188/bin/NIHMS203717-supplement-1.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.1M, doc)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d36e842"><a href="/pmc/articles/PMC3832188/bin/NIHMS203717-supplement-2.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.4M, doc)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgements</h2><div class="sec"><p id="__p1">We thank Dorothy Melia, R.N., Kathy Bell, R.N., Elizabeth Ducat, N.P., Lisa Borg, M.D., Brenda Ray, N.P., Pauline McHugh, M.D., James Schluger, M.D., and Heather Hofflich, D.O. for recruiting, screening, and assessment of study subjects. Connie Zhao, Ph.D. for the processing of the microarrays, and Sara Hamon for review and discussion of the statistical analyses. We are grateful to the late K. Steven LaForge for his role in the planning of these genetic studies.</p><p id="__p2">This work was supported in part by NIH-NIDA Grants K05-DA00049 (M.J.K.) and RO1-DA12848 (M.J.K.)<strong>,</strong> NIH-RR UL1RR024143 (B.C.), NIH-MH R01-44292 (J.O.), and NSFC grant 30730057 and 30700442 from the Chinese Government (J.O.).</p></div></div><div id="idm140042269354048" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140042269354048title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P36" class="p p-first-last">This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140042269352320" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140042269352320title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Benko S, Fantes JA, Amiel J, Kleinjan DJ, Thomas S, Ramsay J, et al.  Highly conserved non-coding elements on either side of SOX9 associated with Pierre Robin sequence. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:359–64.</span> [<a href="/pubmed/19234473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Highly+conserved+non-coding+elements+on+either+side+of+SOX9+associated+with+Pierre+Robin+sequence&amp;author=S+Benko&amp;author=JA+Fantes&amp;author=J+Amiel&amp;author=DJ+Kleinjan&amp;author=S+Thomas&amp;volume=41&amp;publication_year=2009&amp;pages=359-64&amp;pmid=19234473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Cardon LR, Palmer LJ. Population stratification and spurious allelic association. <span><span class="ref-journal">Lancet. </span>2003;<span class="ref-vol">361</span>:598–604.</span> [<a href="/pubmed/12598158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Population+stratification+and+spurious+allelic+association&amp;author=LR+Cardon&amp;author=LJ+Palmer&amp;volume=361&amp;publication_year=2003&amp;pages=598-604&amp;pmid=12598158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Collins A, Lonjou C, Morton NE. Genetic epidemiology of single-nucleotide polymorphisms. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>:15173–15177.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC24792/">PMC free article</a>]</span> [<a href="/pubmed/10611357" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Genetic+epidemiology+of+single-nucleotide+polymorphisms&amp;author=A+Collins&amp;author=C+Lonjou&amp;author=NE+Morton&amp;volume=96&amp;publication_year=1999&amp;pages=15173-15177&amp;pmid=10611357&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Duan J, Wainwright MS, Comeron JM, Saitou N, Sanders AR, Gelernter J, Gejman PV. Synonymous mutations in the human dopamine receptor D2 (DRD2) affect mRNA stability and synthesis of the receptor. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>:205–216.</span> [<a href="/pubmed/12554675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Synonymous+mutations+in+the+human+dopamine+receptor+D2+(DRD2)+affect+mRNA+stability+and+synthesis+of+the+receptor&amp;author=J+Duan&amp;author=MS+Wainwright&amp;author=JM+Comeron&amp;author=N+Saitou&amp;author=AR+Sanders&amp;volume=12&amp;publication_year=2003&amp;pages=205-216&amp;pmid=12554675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Edwards YH, Putt W, Lekoape KM, Stott D, Fox M, Hopkinson DA, et al.  The human homolog T of the mouse T(Brachyury) gene; gene structure, cDNA sequence, and assignment to chromosome 6q27. <span><span class="ref-journal">Genome Res. </span>1996;<span class="ref-vol">6</span>:226–33.</span> [<a href="/pubmed/8963900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=The+human+homolog+T+of+the+mouse+T(Brachyury)+gene;+gene+structure,+cDNA+sequence,+and+assignment+to+chromosome+6q27&amp;author=YH+Edwards&amp;author=W+Putt&amp;author=KM+Lekoape&amp;author=D+Stott&amp;author=M+Fox&amp;volume=6&amp;publication_year=1996&amp;pages=226-33&amp;pmid=8963900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Faham M, Zheng J, Moorhead M, Fakhrai-Rad H, Namsaraev E, Wong K, et al.  Multiplexed variation scanning for 1,000 amplicons in hundreds of patients using mismatch repair detection (MRD) on tag arrays. <span><span class="ref-journal">Proceedings of the National Academy of Sciences. </span>2005;<span class="ref-vol">102</span>:14717–14722.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1253580/">PMC free article</a>]</span> [<a href="/pubmed/16203980" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences&amp;title=Multiplexed+variation+scanning+for+1,000+amplicons+in+hundreds+of+patients+using+mismatch+repair+detection+(MRD)+on+tag+arrays&amp;author=M+Faham&amp;author=J+Zheng&amp;author=M+Moorhead&amp;author=H+Fakhrai-Rad&amp;author=E+Namsaraev&amp;volume=102&amp;publication_year=2005&amp;pages=14717-14722&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Friedberg I, Nika K, Tautz L, Saito K, Cerignoli F, Godzik A, et al.  Identification and characterization of DUSP27, a novel dual-specific protein phosphatase. <span><span class="ref-journal">FEBS Lett. </span>2007;<span class="ref-vol">581</span>:2527–2533.</span> [<a href="/pubmed/17498703" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=Identification+and+characterization+of+DUSP27,+a+novel+dual-specific+protein+phosphatase&amp;author=I+Friedberg&amp;author=K+Nika&amp;author=L+Tautz&amp;author=K+Saito&amp;author=F+Cerignoli&amp;volume=581&amp;publication_year=2007&amp;pages=2527-2533&amp;pmid=17498703&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Gelernter J, Panhuysen C, Wilcox M, Hesselbrock V, Rounsaville B, Poling J, et al.  Genomewide linkage scan for opioid dependence and related traits. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">78</span>:759–769.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1474044/">PMC free article</a>]</span> [<a href="/pubmed/16642432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genomewide+linkage+scan+for+opioid+dependence+and+related+traits&amp;author=J+Gelernter&amp;author=C+Panhuysen&amp;author=M+Wilcox&amp;author=V+Hesselbrock&amp;author=B+Rounsaville&amp;volume=78&amp;publication_year=2006&amp;pages=759-769&amp;pmid=16642432&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Glatt SJ, Lasky-Su JA, Zhu SC, Zhang R, Zhang B, Li J, et al.  Genome-wide linkage analysis of heroin dependence in Han Chinese: Results from Wave Two of a multi-stage study. <span><span class="ref-journal">Drug Alcohol Depend. </span>2008;<span class="ref-vol">98</span>:30–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2764288/">PMC free article</a>]</span> [<a href="/pubmed/18538955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Alcohol+Depend&amp;title=Genome-wide+linkage+analysis+of+heroin+dependence+in+Han+Chinese:+Results+from+Wave+Two+of+a+multi-stage+study&amp;author=SJ+Glatt&amp;author=JA+Lasky-Su&amp;author=SC+Zhu&amp;author=R+Zhang&amp;author=B+Zhang&amp;volume=98&amp;publication_year=2008&amp;pages=30-4&amp;pmid=18538955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Glatt SJ, Su JA, Zhu SC, Zhang R, Zhang B, Li J, et al.  Genome-wide linkage analysis of heroin dependence in Han Chinese: Results from wave one of a multi-stage study. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2006;<span class="ref-vol">141B</span>:648–652.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2562776/">PMC free article</a>]</span> [<a href="/pubmed/16856125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Genome-wide+linkage+analysis+of+heroin+dependence+in+Han+Chinese:+Results+from+wave+one+of+a+multi-stage+study&amp;author=SJ+Glatt&amp;author=JA+Su&amp;author=SC+Zhu&amp;author=R+Zhang&amp;author=B+Zhang&amp;volume=141B&amp;publication_year=2006&amp;pages=648-652&amp;pmid=16856125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Hattori E, Liu C, Zhu H, Gershon ES. Genetic tests of biologic systems in affective disorders. <span><span class="ref-journal">Mol Psychiatry. </span>2005;<span class="ref-vol">10</span>:719–740.</span> [<a href="/pubmed/15940293" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Genetic+tests+of+biologic+systems+in+affective+disorders&amp;author=E+Hattori&amp;author=C+Liu&amp;author=H+Zhu&amp;author=ES+Gershon&amp;volume=10&amp;publication_year=2005&amp;pages=719-740&amp;pmid=15940293&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Jensen LE, Barbaux S, Hoess K, Fraterman S, Whitehead AS, Mitchell LE. The human T locus and spina bifida risk. <span><span class="ref-journal">Hum Genet. </span>2004;<span class="ref-vol">115</span>:475–82.</span> [<a href="/pubmed/15449172" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=The+human+T+locus+and+spina+bifida+risk&amp;author=LE+Jensen&amp;author=S+Barbaux&amp;author=K+Hoess&amp;author=S+Fraterman&amp;author=AS+Whitehead&amp;volume=115&amp;publication_year=2004&amp;pages=475-82&amp;pmid=15449172&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Jeong Y, Leskow FC, El-Jaick K, Roessler E, Muenke M, Yocum A, et al.  Regulation of a remote Shh forebrain enhancer by the Six3 homeoprotein. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1348–53.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2648611/">PMC free article</a>]</span> [<a href="/pubmed/18836447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Regulation+of+a+remote+Shh+forebrain+enhancer+by+the+Six3+homeoprotein&amp;author=Y+Jeong&amp;author=FC+Leskow&amp;author=K+El-Jaick&amp;author=E+Roessler&amp;author=M+Muenke&amp;volume=40&amp;publication_year=2008&amp;pages=1348-53&amp;pmid=18836447&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Johnson C, Drgon T, Liu QR, Walther D, Edenberg H, Rice J, et al.  Pooled association genome scanning for alcohol dependence using 104,268 SNPs: Validation and use to identify alcoholism vulnerability loci in unrelated individuals from the collaborative study on the genetics of alcoholism. <span><span class="ref-journal">American Journal of Medical Genetics Part B: Neuropsychiatric Genetics. </span>2006;<span class="ref-vol">141B</span>:844–853.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3922200/">PMC free article</a>]</span> [<a href="/pubmed/16894614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Medical+Genetics+Part+B:+Neuropsychiatric+Genetics&amp;title=Pooled+association+genome+scanning+for+alcohol+dependence+using+104,268+SNPs:+Validation+and+use+to+identify+alcoholism+vulnerability+loci+in+unrelated+individuals+from+the+collaborative+study+on+the+genetics+of+alcoholism&amp;author=C+Johnson&amp;author=T+Drgon&amp;author=QR+Liu&amp;author=D+Walther&amp;author=H+Edenberg&amp;volume=141B&amp;publication_year=2006&amp;pages=844-853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Kendler KS, Jacobson KC, Prescott CA, Neale MC. Specificity of genetic and environmental risk factors for use and abuse/dependence of cannabis, cocaine, hallucinogens, sedatives, stimulants, and opiates in male twins. <span><span class="ref-journal">Am J Psychiatry. </span>2003;<span class="ref-vol">160</span>:687–695.</span> [<a href="/pubmed/12668357" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Specificity+of+genetic+and+environmental+risk+factors+for+use+and+abuse/dependence+of+cannabis,+cocaine,+hallucinogens,+sedatives,+stimulants,+and+opiates+in+male+twins&amp;author=KS+Kendler&amp;author=KC+Jacobson&amp;author=CA+Prescott&amp;author=MC+Neale&amp;volume=160&amp;publication_year=2003&amp;pages=687-695&amp;pmid=12668357&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">King DC, Taylor J, Elnitski L, Chiaromonte F, Miller W, Hardison RC. Evaluation of regulatory potential and conservation scores for detecting cis-regulatory modules in aligned mammalian genome sequences. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:1051–1060.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182217/">PMC free article</a>]</span> [<a href="/pubmed/16024817" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Evaluation+of+regulatory+potential+and+conservation+scores+for+detecting+cis-regulatory+modules+in+aligned+mammalian+genome+sequences&amp;author=DC+King&amp;author=J+Taylor&amp;author=L+Elnitski&amp;author=F+Chiaromonte&amp;author=W+Miller&amp;volume=15&amp;publication_year=2005&amp;pages=1051-1060&amp;pmid=16024817&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Kirov G, Nikolov I, Georgieva L, Moskvina V, Owen MJ, O’donovan M C. Pooled DNA genotyping on Affymetrix SNP genotyping arrays. <span><span class="ref-journal">BMC Genomics. </span>2006;<span class="ref-vol">7</span>:27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1382214/">PMC free article</a>]</span> [<a href="/pubmed/16480507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Pooled+DNA+genotyping+on+Affymetrix+SNP+genotyping+arrays&amp;author=G+Kirov&amp;author=I+Nikolov&amp;author=L+Georgieva&amp;author=V+Moskvina&amp;author=MJ+Owen&amp;volume=7&amp;publication_year=2006&amp;pages=27&amp;pmid=16480507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Kreek MJ.  Gene diversity in the endorphin system: SNPs, chips, and possible implications. In: Yudell M, Dasalle R, editors. <span class="ref-journal">The Genomic Revolution: Unveiling the Unity of Life.</span> Joseph Henry Press; Washington, DC: 2002.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Genomic+Revolution:+Unveiling+the+Unity+of+Life&amp;author=MJ+Kreek&amp;publication_year=2002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Lachman HM, Fann CS, Bartzis M, Evgrafov OV, Rosenthal RN, Nunes EV, et al.  Genomewide suggestive linkage of opioid dependence to chromosome 14q. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>:1327–1334.</span> [<a href="/pubmed/17409192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genomewide+suggestive+linkage+of+opioid+dependence+to+chromosome+14q&amp;author=HM+Lachman&amp;author=CS+Fann&amp;author=M+Bartzis&amp;author=OV+Evgrafov&amp;author=RN+Rosenthal&amp;volume=16&amp;publication_year=2007&amp;pages=1327-1334&amp;pmid=17409192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Lalumiere RT, Kalivas PW. Glutamate release in the nucleus accumbens core is necessary for heroin seeking. <span><span class="ref-journal">J Neurosci. </span>2008;<span class="ref-vol">28</span>:3170–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6670700/">PMC free article</a>]</span> [<a href="/pubmed/18354020" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Glutamate+release+in+the+nucleus+accumbens+core+is+necessary+for+heroin+seeking&amp;author=RT+Lalumiere&amp;author=PW+Kalivas&amp;volume=28&amp;publication_year=2008&amp;pages=3170-7&amp;pmid=18354020&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Lettice LA, Heaney SJ, Purdie LA, Li L, de Beer P, Oostra BA, et al.  A long-range Shh enhancer regulates expression in the developing limb and fin and is associated with preaxial polydactyly. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>:1725–35.</span> [<a href="/pubmed/12837695" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+long-range+Shh+enhancer+regulates+expression+in+the+developing+limb+and+fin+and+is+associated+with+preaxial+polydactyly&amp;author=LA+Lettice&amp;author=SJ+Heaney&amp;author=LA+Purdie&amp;author=L+Li&amp;author=P+de+Beer&amp;volume=12&amp;publication_year=2003&amp;pages=1725-35&amp;pmid=12837695&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Liu QR, Drgon T, Johnson C, Walther D, Hess J, Uhl GR. Addiction molecular genetics: 639,401 SNP whole genome association identifies many “cell adhesion” genes. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2006;<span class="ref-vol">141B</span>:918–925.</span> [<a href="/pubmed/17099884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Addiction+molecular+genetics:+639,401+SNP+whole+genome+association+identifies+many+“cell+adhesion”+genes&amp;author=QR+Liu&amp;author=T+Drgon&amp;author=C+Johnson&amp;author=D+Walther&amp;author=J+Hess&amp;volume=141B&amp;publication_year=2006&amp;pages=918-925&amp;pmid=17099884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Liu QR, Drgon T, Walther D, Johnson C, Poleskaya O, Hess J, et al.  Pooled association genome scanning: validation and use to identify addiction vulnerability loci in two samples. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2005;<span class="ref-vol">102</span>:11864–11869.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1183486/">PMC free article</a>]</span> [<a href="/pubmed/16091475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Pooled+association+genome+scanning:+validation+and+use+to+identify+addiction+vulnerability+loci+in+two+samples&amp;author=QR+Liu&amp;author=T+Drgon&amp;author=D+Walther&amp;author=C+Johnson&amp;author=O+Poleskaya&amp;volume=102&amp;publication_year=2005&amp;pages=11864-11869&amp;pmid=16091475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Mclellan AT, Luborsky L, Woody GE, O’brien CP. An improved diagnostic evaluation instrument for substance abuse patients. The Addiction Severity Index. <span><span class="ref-journal">J Nerv Ment Dis. </span>1980;<span class="ref-vol">168</span>:26–33.</span> [<a href="/pubmed/7351540" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Nerv+Ment+Dis&amp;title=An+improved+diagnostic+evaluation+instrument+for+substance+abuse+patients.+The+Addiction+Severity+Index&amp;author=AT+Mclellan&amp;author=L+Luborsky&amp;author=GE+Woody&amp;author=CP+O’brien&amp;volume=168&amp;publication_year=1980&amp;pages=26-33&amp;pmid=7351540&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Morley M, Molony CM, Weber TM, Devlin JL, Ewens KG, Spielman RS, et al.  Genetic analysis of genome-wide variation in human gene expression. <span><span class="ref-journal">Nature. </span>2004;<span class="ref-vol">430</span>:743–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2966974/">PMC free article</a>]</span> [<a href="/pubmed/15269782" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+analysis+of+genome-wide+variation+in+human+gene+expression&amp;author=M+Morley&amp;author=CM+Molony&amp;author=TM+Weber&amp;author=JL+Devlin&amp;author=KG+Ewens&amp;volume=430&amp;publication_year=2004&amp;pages=743-7&amp;pmid=15269782&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Morrison K, Papapetrou C, Attwood J, Hol F, Lynch SA, Sampath A, et al.  Genetic mapping of the human homologue (T) of mouse T(Brachyury) and a search for allele association between human T and spina bifida. <span><span class="ref-journal">Hum Mol Genet. </span>1996;<span class="ref-vol">5</span>:669–74.</span> [<a href="/pubmed/8733136" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genetic+mapping+of+the+human+homologue+(T)+of+mouse+T(Brachyury)+and+a+search+for+allele+association+between+human+T+and+spina+bifida&amp;author=K+Morrison&amp;author=C+Papapetrou&amp;author=J+Attwood&amp;author=F+Hol&amp;author=SA+Lynch&amp;volume=5&amp;publication_year=1996&amp;pages=669-74&amp;pmid=8733136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Nielsen DA, Barral S, Proudnikov D, Kellogg S, Ho A, Ott J, et al.  TPH2 and TPH1: Association of Variants and Interactions with Heroin Addiction. <span><span class="ref-journal">Behav Genet. </span>2008a;<span class="ref-vol">38</span>:133–50.</span> [<a href="/pubmed/18181017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behav+Genet&amp;title=TPH2+and+TPH1:+Association+of+Variants+and+Interactions+with+Heroin+Addiction&amp;author=DA+Nielsen&amp;author=S+Barral&amp;author=D+Proudnikov&amp;author=S+Kellogg&amp;author=A+Ho&amp;volume=38&amp;publication_year=2008a&amp;pages=133-50&amp;pmid=18181017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Nielsen DA, Ji F, Yuferov V, Ho A, Chen A, Levran O, et al.  Genotype patterns that contribute to increased risk for or protection from developing heroin addiction. <span><span class="ref-journal">Mol Psychiatry. </span>2008b;<span class="ref-vol">13</span>:417–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3810149/">PMC free article</a>]</span> [<a href="/pubmed/18195715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Genotype+patterns+that+contribute+to+increased+risk+for+or+protection+from+developing+heroin+addiction&amp;author=DA+Nielsen&amp;author=F+Ji&amp;author=V+Yuferov&amp;author=A+Ho&amp;author=A+Chen&amp;volume=13&amp;publication_year=2008b&amp;pages=417-28&amp;pmid=18195715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Reich DE, Cargill M, Bolk S, Ireland J, Sabeti PC, Richter DJ, et al.  Linkage disequilibrium in the human genome. <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">411</span>:199–204.</span> [<a href="/pubmed/11346797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Linkage+disequilibrium+in+the+human+genome&amp;author=DE+Reich&amp;author=M+Cargill&amp;author=S+Bolk&amp;author=J+Ireland&amp;author=PC+Sabeti&amp;volume=411&amp;publication_year=2001&amp;pages=199-204&amp;pmid=11346797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Rettig RA, Yarmolinsky A, editors. <span class="ref-journal">Federal Regulation of Methadone Treatment.</span> National Academy Press; Washington, DC: 1995.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Federal+Regulation+of+Methadone+Treatment&amp;publication_year=1995&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Scherer SW, Soder S, Duvoisin RM, Huizenga JJ, Tsui LC. The human metabotropic glutamate receptor 8 (GRM8) gene: a disproportionately large gene located at 7q31.3-q32.1. <span><span class="ref-journal">Genomics. </span>1997;<span class="ref-vol">44</span>:232–6.</span> [<a href="/pubmed/9299241" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=The+human+metabotropic+glutamate+receptor+8+(GRM8)+gene:+a+disproportionately+large+gene+located+at+7q31.3-q32.1&amp;author=SW+Scherer&amp;author=S+Soder&amp;author=RM+Duvoisin&amp;author=JJ+Huizenga&amp;author=LC+Tsui&amp;volume=44&amp;publication_year=1997&amp;pages=232-6&amp;pmid=9299241&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Schug J, Overton GC.  <span class="ref-journal">TESS: Transcription Element Search Software on the WWW.</span> Computational Biology and Informatics Laboratory, School of Medicine, University of Pennsylvania; Philadelphia: 1977.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=TESS:+Transcription+Element+Search+Software+on+the+WWW&amp;author=J+Schug&amp;author=GC+Overton&amp;publication_year=1977&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Sham P, Bader JS, Craig I, O’donovan M, Owen M. DNA Pooling: a tool for large-scale association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2002;<span class="ref-vol">3</span>:862–871.</span> [<a href="/pubmed/12415316" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=DNA+Pooling:+a+tool+for+large-scale+association+studies&amp;author=P+Sham&amp;author=JS+Bader&amp;author=I+Craig&amp;author=M+O’donovan&amp;author=M+Owen&amp;volume=3&amp;publication_year=2002&amp;pages=862-871&amp;pmid=12415316&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Shields DC, Ramsbottom D, Donoghue C, Pinjon E, Kirke PN, Molloy AM, et al.  Association between historically high frequencies of neural tube defects and the human T homologue of mouse T (Brachyury) <span><span class="ref-journal">Am J Med Genet. </span>2000;<span class="ref-vol">92</span>:206–11.</span> [<a href="/pubmed/10817656" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Association+between+historically+high+frequencies+of+neural+tube+defects+and+the+human+T+homologue+of+mouse+T+(Brachyury)&amp;author=DC+Shields&amp;author=D+Ramsbottom&amp;author=C+Donoghue&amp;author=E+Pinjon&amp;author=PN+Kirke&amp;volume=92&amp;publication_year=2000&amp;pages=206-11&amp;pmid=10817656&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Simon RM, Korn EL, Mcshane LM, Radmacher MD, Wright GW, Zhao Y. <span><span class="ref-journal">Design and Analysis of DNA Microarray Investigations Series: Statistics for Biology and Health. </span>2004</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Design+and+Analysis+of+DNA+Microarray+Investigations+Series:+Statistics+for+Biology+and+Health&amp;author=RM+Simon&amp;author=EL+Korn&amp;author=LM+Mcshane&amp;author=MD+Radmacher&amp;author=GW+Wright&amp;publication_year=2004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Takaki H, Kikuta R, Shibata H, Ninomiya H, Tashiro N, Fukumaki Y. Positive associations of polymorphisms in the metabotropic glutamate receptor type 8 gene (GRM8) with schizophrenia. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2004;<span class="ref-vol">128B</span>:6–14.</span> [<a href="/pubmed/15211621" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Positive+associations+of+polymorphisms+in+the+metabotropic+glutamate+receptor+type+8+gene+(GRM8)+with+schizophrenia&amp;author=H+Takaki&amp;author=R+Kikuta&amp;author=H+Shibata&amp;author=H+Ninomiya&amp;author=N+Tashiro&amp;volume=128B&amp;publication_year=2004&amp;pages=6-14&amp;pmid=15211621&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Tsuang MT, Lyons MJ, Eisen SA, Goldberg J, True W, Lin N, et al.  Genetic influences on DSM-III-R drug abuse and dependence: a study of 3,372 twin pairs. <span><span class="ref-journal">Am J Med Genet. </span>1996;<span class="ref-vol">67</span>:473–477.</span> [<a href="/pubmed/8886164" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Genetic+influences+on+DSM-III-R+drug+abuse+and+dependence:+a+study+of+3,372+twin+pairs&amp;author=MT+Tsuang&amp;author=MJ+Lyons&amp;author=SA+Eisen&amp;author=J+Goldberg&amp;author=W+True&amp;volume=67&amp;publication_year=1996&amp;pages=473-477&amp;pmid=8886164&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Tsuang MT, Lyons MJ, Meyer JM, Doyle T, Eisen SA, Goldberg J, et al.  Co-occurrence of abuse of different drugs in men: the role of drug-specific and shared vulnerabilities. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1998;<span class="ref-vol">55</span>:967–972.</span> [<a href="/pubmed/9819064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Co-occurrence+of+abuse+of+different+drugs+in+men:+the+role+of+drug-specific+and+shared+vulnerabilities&amp;author=MT+Tsuang&amp;author=MJ+Lyons&amp;author=JM+Meyer&amp;author=T+Doyle&amp;author=SA+Eisen&amp;volume=55&amp;publication_year=1998&amp;pages=967-972&amp;pmid=9819064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Uhl GR, Drgon T, Liu QR, Johnson C, Walther D, Komiyama T, et al.  Genome-wide association for methamphetamine dependence: convergent results from 2 samples. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2008;<span class="ref-vol">65</span>:345–55.</span> [<a href="/pubmed/18316681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Genome-wide+association+for+methamphetamine+dependence:+convergent+results+from+2+samples&amp;author=GR+Uhl&amp;author=T+Drgon&amp;author=QR+Liu&amp;author=C+Johnson&amp;author=D+Walther&amp;volume=65&amp;publication_year=2008&amp;pages=345-55&amp;pmid=18316681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Uhl GR, Liu QR, Drgon T, Johnson C, Walther D, Rose JE. Molecular genetics of nicotine dependence and abstinence: whole genome association using 520,000 SNPs. <span><span class="ref-journal">BMC Genet. </span>2007;<span class="ref-vol">8</span>:10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1853105/">PMC free article</a>]</span> [<a href="/pubmed/17407593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=Molecular+genetics+of+nicotine+dependence+and+abstinence:+whole+genome+association+using+520,000+SNPs&amp;author=GR+Uhl&amp;author=QR+Liu&amp;author=T+Drgon&amp;author=C+Johnson&amp;author=D+Walther&amp;volume=8&amp;publication_year=2007&amp;pages=10&amp;pmid=17407593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">Uhl GR, Liu QR, Walther D, Hess J, Naiman D. Polysubstance abuse-vulnerability genes: genome scans for association, using 1,004 subjects and 1,494 single-nucleotide polymorphisms. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">69</span>:1290–1300.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235541/">PMC free article</a>]</span> [<a href="/pubmed/11704927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Polysubstance+abuse-vulnerability+genes:+genome+scans+for+association,+using+1,004+subjects+and+1,494+single-nucleotide+polymorphisms&amp;author=GR+Uhl&amp;author=QR+Liu&amp;author=D+Walther&amp;author=J+Hess&amp;author=D+Naiman&amp;volume=69&amp;publication_year=2001&amp;pages=1290-1300&amp;pmid=11704927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Veyrieras JB, Kudaravalli S, Kim SY, Dermitzakis ET, Gilad Y, Stephens M, Pritchard JK. High-resolution mapping of expression-QTLs yields insight into human gene regulation. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000214.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2556086/">PMC free article</a>]</span> [<a href="/pubmed/18846210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=High-resolution+mapping+of+expression-QTLs+yields+insight+into+human+gene+regulation&amp;author=JB+Veyrieras&amp;author=S+Kudaravalli&amp;author=SY+Kim&amp;author=ET+Dermitzakis&amp;author=Y+Gilad&amp;volume=4&amp;publication_year=2008&amp;pages=e1000214&amp;pmid=18846210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Wang Y, Sudhof TC. Genomic definition of RIM proteins: evolutionary amplification of a family of synaptic regulatory proteins( small star, filled ) <span><span class="ref-journal">Genomics. </span>2003;<span class="ref-vol">81</span>:126–37.</span> [<a href="/pubmed/12620390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Genomic+definition+of+RIM+proteins:+evolutionary+amplification+of+a+family+of+synaptic+regulatory+proteins(+small+star,+filled+)&amp;author=Y+Wang&amp;author=TC+Sudhof&amp;volume=81&amp;publication_year=2003&amp;pages=126-37&amp;pmid=12620390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="element-citation">Wang Y, Sugita S, Sudhof TC. The RIM/NIM family of neuronal C2 domain proteins. Interactions with Rab3 and a new class of Src homology 3 domain proteins. <span><span class="ref-journal">J Biol Chem. </span>2000;<span class="ref-vol">275</span>:20033–44.</span> [<a href="/pubmed/10748113" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+RIM/NIM+family+of+neuronal+C2+domain+proteins.+Interactions+with+Rab3+and+a+new+class+of+Src+homology+3+domain+proteins&amp;author=Y+Wang&amp;author=S+Sugita&amp;author=TC+Sudhof&amp;volume=275&amp;publication_year=2000&amp;pages=20033-44&amp;pmid=10748113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R45"><span class="element-citation">Xi ZX, Stein EA. Blockade of ionotropic glutamatergic transmission in the ventral tegmental area reduces heroin reinforcement in rat. <span><span class="ref-journal">Psychopharmacology (Berl) </span>2002;<span class="ref-vol">164</span>:144–50.</span> [<a href="/pubmed/12404076" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychopharmacology+(Berl)&amp;title=Blockade+of+ionotropic+glutamatergic+transmission+in+the+ventral+tegmental+area+reduces+heroin+reinforcement+in+rat&amp;author=ZX+Xi&amp;author=EA+Stein&amp;volume=164&amp;publication_year=2002&amp;pages=144-50&amp;pmid=12404076&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R46"><span class="element-citation">Yu Y, Kranzler HR, Panhuysen C, Weiss RD, Poling J, Farrer LA, et al.  Substance dependence low-density whole genome association study in two distinct American populations. <span><span class="ref-journal">Hum Genet. </span>2008;<span class="ref-vol">123</span>:495–506.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3428017/">PMC free article</a>]</span> [<a href="/pubmed/18438686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Substance+dependence+low-density+whole+genome+association+study+in+two+distinct+American+populations&amp;author=Y+Yu&amp;author=HR+Kranzler&amp;author=C+Panhuysen&amp;author=RD+Weiss&amp;author=J+Poling&amp;volume=123&amp;publication_year=2008&amp;pages=495-506&amp;pmid=18438686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832188&amp;issue-id=229845&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140042273559904title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Materials and methods</a></li><li><a href="#S12title">Results</a></li><li><a href="#S16title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S17title">Acknowledgements</a></li><li><a href="#idm140042269354048title">Footnotes</a></li><li><a href="#idm140042269352320title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3832188/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3832188/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3832188/pdf/nihms-203717.pdf">PDF (66K)</a></li> | <li><a href="#" data-citationid="PMC3832188" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832188%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832188%2F&amp;text=Genome-wide%20association%20study%20identifies%20genes%20that%20may%20contribute%20to%20risk%20for%20developing%20heroin%20addiction" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832188%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20520587" data-aiid="3832188" data-aid="3832188" data-iid="229845" data-domainid="319" data-domain="nihpa" data-accid="PMC3832188" data-md5="50ba032dfd34d1ddc5048b092854458e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:24:04-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE882E58DD5115810000000001E90155&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832188%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3832188/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>