<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3601979/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of inherited genetic variations influencing prognosis in early onset breast cancer</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3601979/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Cancer research"><meta name="citation_title" content="Identification of inherited genetic variations influencing prognosis in early onset breast cancer"><meta name="citation_authors" content="Sajjad Rafiq, William Tapper, Andrew Collins, Sofia Khan, Ioannis Politopoulos, Sue Gerty, Carl Blomqvist, Fergus J Couch, Heli Nevanlinna, Jianjun Liu, Diana Eccles"><meta name="citation_date" content="15 March 2013"><meta name="citation_issue" content="6"><meta name="citation_volume" content="73"><meta name="citation_firstpage" content="1883"><meta name="citation_doi" content="10.1158/0008-5472.CAN-12-3377"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3601979/?report=abstract"><meta name="citation_pmid" content="23319801"><meta name="DC.Title" content="Identification of inherited genetic variations influencing prognosis in early onset breast cancer"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Sajjad Rafiq"><meta name="DC.Contributor" content="William Tapper"><meta name="DC.Contributor" content="Andrew Collins"><meta name="DC.Contributor" content="Sofia Khan"><meta name="DC.Contributor" content="Ioannis Politopoulos"><meta name="DC.Contributor" content="Sue Gerty"><meta name="DC.Contributor" content="Carl Blomqvist"><meta name="DC.Contributor" content="Fergus J Couch"><meta name="DC.Contributor" content="Heli Nevanlinna"><meta name="DC.Contributor" content="Jianjun Liu"><meta name="DC.Contributor" content="Diana Eccles"><meta name="DC.Date" content="2013 Mar 15"><meta name="DC.Identifier" content="10.1158/0008-5472.CAN-12-3377"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of inherited genetic variations influencing prognosis in early onset breast cancer"><meta property="og:type" content="article"><meta property="og:description" content="Genome Wide Association Studies (GWAs) have begun to investigate associations between inherited genetic variations and breast cancer prognosis. Here we report our findings from a GWAs conducted in 536 early onset breast cancer patients aged 40 or less ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3601979/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3601979/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3601979/pdf/emss-51101.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D0B8DDD54C7D10000000007E70541.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3601979%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC3601979</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Cancer Res</a></span>. Author manuscript; available in PMC 2013 Sep 15.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23319801" target="pmc_ext" ref="reftype=publisher&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Cancer Res. 2013 Mar 15; 73(6): 1883–1891. </span></a></div><span class="fm-vol-iss-date">Published online 2013 Jan 14. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1158%2F0008-5472.CAN-12-3377" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1158/0008-5472.CAN-12-3377</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3601979</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS51101</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23319801">23319801</a></div></div></div></div><h1 class="content-title">Identification of inherited genetic variations influencing prognosis in early onset breast cancer</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Rafiq%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376127026720" co-class="co-affbox">Sajjad Rafiq</a>,<sup>1</sup> <a href="/pubmed/?term=Tapper%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376121304176" co-class="co-affbox">William Tapper</a>,<sup>1</sup> <a href="/pubmed/?term=Collins%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376121302192" co-class="co-affbox">Andrew Collins</a>,<sup>1</sup> <a href="/pubmed/?term=Khan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376129467472" co-class="co-affbox">Sofia Khan</a>,<sup>4</sup> <a href="/pubmed/?term=Politopoulos%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376133726768" co-class="co-affbox">Ioannis Politopoulos</a>,<sup>1</sup> <a href="/pubmed/?term=Gerty%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376133724784" co-class="co-affbox">Sue Gerty</a>,<sup>2</sup> <a href="/pubmed/?term=Blomqvist%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376133220208" co-class="co-affbox">Carl Blomqvist</a>,<sup>5</sup> <a href="/pubmed/?term=Couch%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376133218224" co-class="co-affbox">Fergus J Couch</a>,<sup>3</sup> <a href="/pubmed/?term=Nevanlinna%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376129788416" co-class="co-affbox">Heli Nevanlinna</a>,<sup>4</sup> <a href="/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376128332112" co-class="co-affbox">Jianjun Liu</a>,<sup>6</sup> and  <a href="/pubmed/?term=Eccles%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801" class="affpopup" co-rid="_co_idm140376128330128" co-class="co-affbox">Diana Eccles</a><sup>7</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140376127026720"><h3 class="no_margin">Sajjad Rafiq</h3><p><sup>1</sup>Genetic Epidemiology and Bioinformatics Research Group, Human Genetics Research Division, University of Southampton, School of Medicine, Southampton General Hospital, Hants, UK </p><div>Find articles by <a href="/pubmed/?term=Rafiq%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Sajjad Rafiq</a></div></div><div id="_co_idm140376121304176"><h3 class="no_margin">William Tapper</h3><p><sup>1</sup>Genetic Epidemiology and Bioinformatics Research Group, Human Genetics Research Division, University of Southampton, School of Medicine, Southampton General Hospital, Hants, UK </p><div>Find articles by <a href="/pubmed/?term=Tapper%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">William Tapper</a></div></div><div id="_co_idm140376121302192"><h3 class="no_margin">Andrew Collins</h3><p><sup>1</sup>Genetic Epidemiology and Bioinformatics Research Group, Human Genetics Research Division, University of Southampton, School of Medicine, Southampton General Hospital, Hants, UK </p><div>Find articles by <a href="/pubmed/?term=Collins%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Andrew Collins</a></div></div><div id="_co_idm140376129467472"><h3 class="no_margin">Sofia Khan</h3><p><sup>4</sup>Department of Obstetrics and Gynaecology, University of Helsinki and Helsinki University Central Hospital, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Khan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Sofia Khan</a></div></div><div id="_co_idm140376133726768"><h3 class="no_margin">Ioannis Politopoulos</h3><p><sup>1</sup>Genetic Epidemiology and Bioinformatics Research Group, Human Genetics Research Division, University of Southampton, School of Medicine, Southampton General Hospital, Hants, UK </p><div>Find articles by <a href="/pubmed/?term=Politopoulos%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Ioannis Politopoulos</a></div></div><div id="_co_idm140376133724784"><h3 class="no_margin">Sue Gerty</h3><p><sup>2</sup>Clinical Trials Unit, Faculty of Medicine, University of Southampton, UK </p><div>Find articles by <a href="/pubmed/?term=Gerty%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Sue Gerty</a></div></div><div id="_co_idm140376133220208"><h3 class="no_margin">Carl Blomqvist</h3><p><sup>5</sup>Department of Oncology, Helsinki University Central Hospital, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Blomqvist%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Carl Blomqvist</a></div></div><div id="_co_idm140376133218224"><h3 class="no_margin">Fergus J Couch</h3><p><sup>3</sup>Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, Minnesota, USA. </p><div>Find articles by <a href="/pubmed/?term=Couch%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Fergus J Couch</a></div></div><div id="_co_idm140376129788416"><h3 class="no_margin">Heli Nevanlinna</h3><p><sup>4</sup>Department of Obstetrics and Gynaecology, University of Helsinki and Helsinki University Central Hospital, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Nevanlinna%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Heli Nevanlinna</a></div></div><div id="_co_idm140376128332112"><h3 class="no_margin">Jianjun Liu</h3><p><sup>6</sup>Human Genetics, Genome Institute of Singapore, 60 Biopolis St, Singapore </p><div>Find articles by <a href="/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Jianjun Liu</a></div></div><div id="_co_idm140376128330128"><h3 class="no_margin">Diana Eccles</h3><p><sup>7</sup>Cancer Sciences Division, University of Southampton, School of Medicine, Southampton General Hospital, Hants, UK </p><div>Find articles by <a href="/pubmed/?term=Eccles%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23319801">Diana Eccles</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140376134099472_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140376134099472_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140376134099472_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Genetic Epidemiology and Bioinformatics Research Group, Human Genetics Research Division, University of Southampton, School of Medicine, Southampton General Hospital, Hants, UK </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Clinical Trials Unit, Faculty of Medicine, University of Southampton, UK </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, Minnesota, USA. </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Obstetrics and Gynaecology, University of Helsinki and Helsinki University Central Hospital, Helsinki, Finland </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Oncology, Helsinki University Central Hospital, Helsinki, Finland </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Human Genetics, Genome Institute of Singapore, 60 Biopolis St, Singapore </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Cancer Sciences Division, University of Southampton, School of Medicine, Southampton General Hospital, Hants, UK </div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140376134099472_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23319801" target="pmc_ext" ref="reftype=publisher&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Cancer Res</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3601979/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="7" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-1.tif" data-ga-action="click_feat_suppl">NIHMS51101-supplement-1.tif</a><span style="color:gray"> (27K)</span></div><div class="small guid">GUID:&nbsp;9AABD93D-2E3A-4A86-9E97-CBF3FB99B643</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-2.tif" data-ga-action="click_feat_suppl">NIHMS51101-supplement-2.tif</a><span style="color:gray"> (93K)</span></div><div class="small guid">GUID:&nbsp;04B2A715-F6D1-41A1-A42E-3EC89C442698</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>3.</strong></div><div><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-3.tif" data-ga-action="click_feat_suppl">NIHMS51101-supplement-3.tif</a><span style="color:gray"> (111K)</span></div><div class="small guid">GUID:&nbsp;0C2D2335-9E85-4DCB-B2A7-8506CDC9A152</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>4.</strong></div><div><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-4.tif" data-ga-action="click_feat_suppl">NIHMS51101-supplement-4.tif</a><span style="color:gray"> (104K)</span></div><div class="small guid">GUID:&nbsp;E390EACB-CC45-4579-A3DC-784C7B6464D7</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>5.</strong></div><div><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-5.tif" data-ga-action="click_feat_suppl">NIHMS51101-supplement-5.tif</a><span style="color:gray"> (117K)</span></div><div class="small guid">GUID:&nbsp;ADBB8E0B-CDB0-41EE-AEF6-2381D86CDE10</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>6.</strong></div><div><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-6.doc" data-ga-action="click_feat_suppl">NIHMS51101-supplement-6.doc</a><span style="color:gray"> (18K)</span></div><div class="small guid">GUID:&nbsp;476B7AF3-462F-4D09-B91D-FECB04062D27</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 1.</strong></div><div><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-Table_1.pdf" data-ga-action="click_feat_suppl">NIHMS51101-supplement-Table_1.pdf</a><span style="color:gray"> (70K)</span></div><div class="small guid">GUID:&nbsp;E5F3230E-01FD-4070-997E-EC6A43AADECB</div></div></dd></dl></div><div id="idm140376133399040" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140376133399040title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Genome Wide Association Studies (GWAs) have begun to investigate associations between inherited genetic variations and breast cancer prognosis. Here we report our findings from a GWAs conducted in 536 early onset breast cancer patients aged 40 or less at diagnosis and with a mean follow-up period of 4.1 years (S.D=1.96). Patients were selected from the POSH (Prospective study of Outcomes in Sporadic versus Hereditary breast cancer). A Bonferroni correction for multiple testing determined that a p-value of 1.0 × 10<sup>−7</sup> was a statistically significant association signal. Following QC we identified 487496 SNPs for association tests in stage-1. In stage 2, 35 SNPs with the most significant associations were genotyped in 1516 independent cases from the same early onset cohort. In stage-2, 11 SNPs remained associated in the same direction (p{less than or equal to}0.05). Fixed effects meta-analysis models identified one SNP associated at close to genome wide level of significance 556 kb upstream of the <em>ARRDC3</em> locus HR=1.61 (1.33-1.96, p=9.5 × 10-7). Four further associations at or close to the <em>PBX1, RORα</em>, <em>NTN1</em> and <em>SYT6</em> loci also came close to genome wide significance levels (p=10-6). In the first ever GWAS for identification of SNPs associated with prognosis in early onset breast cancer patients we report a SNP upstream of the <em>ARRDC3</em> locus as potentially associated with prognosis (Median follow-up time for genotypes CC=4 years, CT=3 years and TT=2.7 years, Wilcoxon rank sum test CC vs. CT, p=4 × 10-4 and CT vs. TT, P=0.76). Four further loci might also be associated with prognosis</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Early onset, Breast cancer, Prognosis, Survival analysis and GWAs</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Breast cancer incidence increases with increasing age. Less than 5% of all breast cancer cases are diagnosed before 40 years of age and less than 20% before 50 years of age (<a href="#R1" rid="R1" class=" bibr popnode">1</a>). Treatments vary according to tumour stage and biological characteristics, age at diagnosis, menopausal status; co-morbidities are also important considerations in deciding what treatment to offer. Early age at breast cancer diagnosis is associated with a worse prognosis although the reasons for this are still imperfectly understood. Tumours in this age group are more likely to have adverse pathological features including a greater proportion of ER negative high grade tumours (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020815">2</a>). Despite accounting for these factors, outcomes remain worse for young onset patients, particularly those with ER positive cancers and this may reflect a poorer response to breast cancer treatments in younger patients (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020839">3</a>-<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020832">6</a>). A Swedish familial study demonstrated higher risk of mortality in affected first degree relatives of breast cancer patients which suggests a genetic component to prognosis following disease onset (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020835">7</a>). A smaller gap in age at diagnosis between sister-sister pairs compared with mother-daughter pairs in this familial study coupled with poorer prognosis in sister-sister pairs suggests that earlier disease onset in sister-sister pairs could be linked with a greater genetic component for prognosis. Rare high penetrance genetic predisposition genes like <em>BRCA1</em> are more frequently found to explain young onset breast cancer cases even in the absence of a family history (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020837">8</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020808">9</a>). In addition it is becoming clear that the growing number of common genetic variants which contribute to polygenic breast cancer risk, are associated often more strongly with susceptibility to a particular sub-type of breast cancer (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333665239">10</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020807">11</a>).</p><p id="P3">Common genetic variants may influence prognosis either by influencing the type of tumour that develops, the host response to tumour or the handling or metabolism of breast cancer directed therapies. Two recent studies developed from genome wide association experiments have failed to identify SNPs which are irrefutably associated with breast cancer prognosis in individuals of Caucasian ancestry (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333665241">12</a>). The median age at diagnosis in the patients recruited for these GWASs were 66 and 52 years hence these cohorts are largely comprised of later onset breast cancer patients. A more recent two stage GWAS in Chinese breast cancer patients identified two potential associations with breast cancer but the association effects in replication samples were much weaker than in the discovery set and would not satisfy stringent tests for multiple hypothesis correction (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020831">13</a>). Studies exploring the association of known risk SNPs with prognosis have hinted at a possible role for genetic variation in clinical outcomes (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020818">14</a>, <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020817">15</a>).</p><p id="P4" class="p p-last">Here we report a 2-stage Genome Wide Association Study to identify common genetic variants which are associated with breast cancer prognosis by using a discovery set of young onset patients that were enriched for rapid disease progression and long term breast cancer specific survival. We attempt replication in a larger sample of early onset breast cancer patients from the same cohort who were unselected for survival extremes. We also seek replication of the main findings from analysis in early onset patients in relatively later onset breast cancer cases from Helsinki.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Patients, Materials and Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Breast cancer patients</h3><p id="P5" class="p p-first-last">Early onset breast cancer cases were selected from the POSH (Prospective study of Outcomes in Sporadic versus Hereditary breast cancer) study, participants were diagnosed with invasive breast cancer and were aged 40 or younger at diagnosis. Recruitments to the POSH cohort were made between January 2000 and January 2008 from oncology clinics across the UK. The vast majority (98%) of patients recruited to the study presented symptomatically. The recruitment, data collection and follow up procedures for the POSH study participants are described in detail elsewhere (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020827">16</a>).</p></div><div id="S4" class="sec"><h3 id="S4title">Stage 1 discovery dataset</h3><p id="P6" class="p p-first-last">In stage 1, 574 participants from the POSH study were selected for the discovery phase of the analysis aimed at hypothesis generation (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020821">17</a>). To enrich the discovery set patients were selected from POSH in two different groups. The first group had ER, PR and HER-2 negative breast cancer (These triple negative patients have worse prognosis and they relapse early after diagnosis). This triple negative breast cancer group was also used to identify risk genes for Breast Cancer susceptibility in a previous study (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020823">11</a>). In the second group we specifically enriched the selection for patients with either very short duration of breast cancer specific survival (&lt;2 years, n=48) or relatively long duration of breast cancer specific survival (&gt;4 years, n=125) but no selection based on immunohistochemistry was made in this group. Breast cancer specific survival was used as the definitive end-point for the survival analysis. Enrichment of affected individuals in a genetic association design increases the efficiency of and power to detect genetic effects (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020826">18</a>). As all the study participants for this GWAs were derived from a single randomly sampled cohort of early onset patients, any overestimation of effect sizes in stage-1 was balanced out by meta-analysis with unenriched stage-2 samples. This approach is in keeping with a recent GWAS which identified five new breast cancer susceptibility loci by enriching cases by recruiting individuals with family history of breast cancer (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333665243">10</a>). There were no screen detected breast cancers amongst the POSH cases included in the discovery analysis, all had presented symptomatically. Amongst young onset breast cancer cases a higher than average proportion is likely to be <em>BRCA</em>1/2 gene carriers. Since <em>BRCA1</em> and <em>BRCA2</em> pathogenic germ-line mutations are not known to be independently associated with prognosis, <em>BRCA</em> status was not used in making the case selection for this study. The cohort has not yet been systematically tested for germ-line <em>BRCA</em>1/2 mutations but amongst the POSH study stage-1 participants, 27.4% (147 cases) had previously been analysed for <em>BRCA1</em> and <em>BRCA2</em> mutations, either as part of other research studies or because testing had been clinically indicated (strong family history). Of those tested 38 (25.8%) had been found to carry clearly pathogenic mutations in <em>BRCA1</em> or <em>BRCA2</em>.</p></div><div id="S5" class="sec"><h3 id="S5title">Stage 2 replication dataset 1</h3><p id="P7" class="p p-first-last">1516 additional young onset cases from the POSH study that had not been selected in the discovery set were genotyped for replication in stage 2. 22.4% of the stage-2 participants were tested for BRCA status. 21.7% of those tested for BRCA status were found to carry pathogenic <em>BRCA 1</em> and <em>BRCA 2</em> mutations.</p></div><div id="S6" class="sec"><h3 id="S6title">Breast Cancer Patients from Helsinki</h3><p id="P8" class="p p-first-last">The Helsinki samples were collected in Helsinki, Finland and are representative of Breast cancer cases at the recruitment centre during the collection period (1997-1998 and 2000). All the breast cancer cases included had histopathological and Survival data available. Detailed information on data collection and selection of participants has previously been published (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020806">19</a>). The mean age at diagnosis was 56.8 years.</p></div><div id="S7" class="sec"><h3 id="S7title">Genome wide genotyping</h3><p id="P9" class="p p-first">Genotyping for the 574 phase 1 breast cancer cases was conducted using the Illumina 660-Quad SNP array. Genotyping for the samples was conducted in two separate batches at two locations. 274 patients were genotyped at the Mayo Clinic, Rochester, Minnesota, USA and were selected because they were diagnosed with triple negative breast cancer (ER, PR and HER2 negative) (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020816">20</a>). 300 POSH patients were genotyped at the Genome Institute of Singapore, National University of Singapore; this group were selected based on either short duration of breast cancer specific survival (&lt;2 years) or for long duration of breast cancer specific suvrival (&gt;4 years).In order to ensure complete harmonisation of the genotype calling, the intensity data available from both these locations in form of *.idat files were combined and used to generate genotypes based on the algorithm available in the genotyping module of Illumina’s Genome Studio software. A GenCall threshold of 0.15 was selected and the HumanHap660 annotation file was used. Genotyping for the replication samples from Helsinki was conducted using the Illumina 550 platform as previously described (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020824">21</a>). The intensity data generated was loaded into Illumina’s Genome studio and genotypes were generated using a GenCall threshold of 0.15. HumanHap550-duo v3 annotation file was used.</p><p id="P10" class="p p-last">SNPs were excluded from analysis based on a minor allele frequency (MAF) cut-off of 0.01, genotyping call rate &lt;95% and Hardy-Weinberg equilibrium p-value&lt;0.0001. We used the pairwise Identity-by-state and multidimensional scaling, implemented in Plink (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020814">22</a>), to identify POSH and Helsinki participants whose genotypes did not concur with a European ancestry. 28 individuals were excluded based on a non-European ancestry or missing phenotype information in the POSH discovery analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>). Three individuals from the 300 samples genotyped at the site in Singapore were excluded from analysis because of call rates lower than 95%. No individuals from the 274 triple negative cohort genotyped at the Mayo clinic were excluded from analysis based on poor call rate. Genotyping accuracy for SNPs or SNP specific call rates were over 99% in samples genotyped at the Singapore and Mayo clinic sites.*.idat files available from the Helsinki participants were fed into Illumina’s Genome Studio software to call genotypes. No ethnic outliers were identified among the 805 Helsinki participants. 7 Helsinki participants were excluded because of SNP call rate (&lt;95%).</p></div><div id="S8" class="sec"><h3 id="S8title">Replication genotyping</h3><p id="P11" class="p p-first-last">Genotyping of the 35 best associated SNPs from stage-1 in the 1516 additional young onset cases from the POSH study was performed by KBioscience (<a href="#R23" rid="R23" class=" bibr popnode">23</a>). SNPs were genotyped using the KASPar chemistry, which is a competitive allele-specific PCR SNP genotyping system using FRET quencher cassette oligo (<a href="#R23" rid="R23" class=" bibr popnode">23</a>) .</p></div><div id="S9" class="sec"><h3 id="S9title">Statistical Analysis</h3><p id="P12" class="p p-first">To generate estimates of pairwise Identity by descent we performed genome wide linkage disequilibrium based pruning using the --indep-pairwise command in plink. SNP data were pruned after choosing an r<sup>2</sup> cut-off at 0.5. SNP pruning was initiated using a window of 50 SNPs. Pairwise LD was then calculated and one SNP within each SNP pair characterised by high LD (r<sup>2</sup>&gt;0.5) was excluded. This process was repeated while choosing smaller SNP windows of 5 SNPs at a time. Multi-dimensional scaling plots were then generated after generating clusters of related individuals based on pairwise IBS distances.</p><p id="P13">SNP quality control (QC) measures were implemented using Plink. Post-QC, transposed ped (tped) and tfam files were generated for further analysis. We used GenABEL (<a href="#R24" rid="R24" class=" bibr popnode">24</a>) in R.2.14.0 environment to perform survival analysis using Post-QC genome wide SNP data. Cox-proportional hazard models were implemented using the mlreg command in GenABEL . The mlreg command utilizes the survival package which is routinely used for survival analysis in R. ER status was the only covariate used in Cox models. Follow-up time was calculated as the difference between the date of diagnosis of breast cancer and the date of death due to breast cancer or the date of last follow-up if still alive or deceased from a non-breast cancer cause (breast cancer specific survival). The mean difference in time between age at diagnosis and age at registration was 0.78 years (SD=1.16 years).Kaplan Meier plots were generated using STATA v11.0 and IBM SPSS statistics 19. Mantel Haenszel Fixed effects meta-analysis was performed using the metan module in STATA v11.0 (<a href="#R25" rid="R25" class=" bibr popnode">25</a>). Genome wide meta-analysis was performed using MetABEL (<a href="#R24" rid="R24" class=" bibr popnode">24</a>).</p><p id="P14" class="p p-last">Imputation of the POSH GWAS data set was performed using MACH 1.0 (<a href="#R26" rid="R26" class=" bibr popnode">26</a>) based on SNP genotype and haplotype phase data specific for CEU population available from HapMap phase 2 project. Imputed genotypes were analysed using ProABEL (<a href="#R24" rid="R24" class=" bibr popnode">24</a>). A posterior probability of 0.9 was used to output imputed genotypes. QC- measures for imputation data included excluding SNPs based on a minor allele frequency (MAF) cut-off of 0.01, genotyping call rate &lt;95% and Hardy-Weinberg equilibrium p-value&lt;0.0001.</p></div><div id="S10" class="sec"><h3 id="S10title">Manhattan and Regional plots</h3><p id="P15" class="p p-first-last">Manhattan and QQ-plots were generated in R using the plot command. Regional plots were generated using LocusZoom (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020825">27</a>).</p></div><div id="S11" class="sec"><h3 id="S11title">Sample size calculations</h3><p id="P16" class="p p-first-last">Sample size calculations were performed in R.2.14.2 using survSNP package.</p></div><div id="S12" class="sec"><h3 id="S12title">Gene Expression variation by SNP</h3><p id="P17" class="p p-first-last">We used Genevar 3.2.0 to study variation in expression levels by SNP genotypes available from the MuTHER pilot project while using NCBI Build 36 /Ensembl 54 as <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020819">reference (28)</a>. Twin pairs were divided into two groups of unrelated individuals. Expression data from Lymphoblastoid cell lines are reported here.</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">Prediction of transcription factor binding site changes</h3><p id="P18" class="p p-first-last">The putative changes on transcription factor binding sites caused by the variants were predicted <em>in silico</em> with SNPInspector within Genomatix software suite v2.5 (Genomatix Software GmbH). SNPInspector analysis is based on MatInspector (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020836">29</a>).</p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Results</h2><p id="P19" class="p p-first">Clinical characteristics of stage-1 and stage-2 participants are summarised in <a href="/pmc/articles/PMC3601979/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Following QC we had SNP genotype data available for 487496 SNPs in stage-1. We had 79% power to detect a hazard ratio (HR) ≥1.50 when studying a SNP with minor allele frequency (maf≥0.10). In survival analysis models no associations were observed to survive a Bonferroni correction and reach a p-value≤10<sup>−7</sup>. Eight SNPs among the top 50 SNPs achieved a p-values &lt;7.0 × 10<sup>−6</sup>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">Characteristics of study participants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376115932096" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Study</th><th rowspan="1" colspan="1" valign="top" align="left">Number of<br>breast<br>cancer<br>deaths</th><th rowspan="1" colspan="1" valign="top" align="left">Total<br>number<br>of<br>Breast<br>cancer<br>patients</th><th rowspan="1" colspan="1" valign="top" align="left">Estrogen<br>Receptor<br>(ER) status-<br>Negative<br>(%)</th><th rowspan="1" colspan="1" valign="top" align="center">Average<br>age at<br>Diagnosis<br>(±SD)</th><th rowspan="1" colspan="1" valign="top" align="left">Follow-up<br>time in years<br>(±SD)</th><th rowspan="1" colspan="1" valign="top" align="left">N-stage</th><th rowspan="1" colspan="1" valign="top" align="left">M-stage</th><th rowspan="1" colspan="1" valign="top" align="left">HER2 status</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>POSH stage-</strong>
<br>
<strong>1</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">236</td><td rowspan="1" colspan="1" valign="top" align="left">536</td><td rowspan="1" colspan="1" valign="top" align="left">370 (69.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">35.7 (3.8)</td><td rowspan="1" colspan="1" valign="top" align="left">4.1 (2.0)</td><td rowspan="1" colspan="1" valign="top" align="left">N0-248<br>N1-262<br>NA-25<br>535</td><td rowspan="1" colspan="1" valign="top" align="left">M0-481<br>M1-50<br>NA-4<br>535</td><td rowspan="1" colspan="1" valign="top" align="left">Negative-369<br>Positive-92<br>NA-74<br>535</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>POSH stage-</strong>
<br>
<strong>2</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">468</td><td rowspan="1" colspan="1" valign="top" align="left">1518</td><td rowspan="1" colspan="1" valign="top" align="left">423 (27.4%)<br>NA-8</td><td rowspan="1" colspan="1" valign="top" align="center">35.7 (3.7)</td><td rowspan="1" colspan="1" valign="top" align="left">4.8 (1.4)</td><td rowspan="1" colspan="1" valign="top" align="left">N0-645<br>N1-838<br>NA-35</td><td rowspan="1" colspan="1" valign="top" align="left">M0-1470<br>M1-42<br>NA-6</td><td rowspan="1" colspan="1" valign="top" align="left">Negative-756<br>Positive-399<br>NA-363</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Helsinki non-</strong>
<br>
<strong>early onset</strong>
<br>
<strong>specific</strong>
<br>
<strong>participants</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">301</td><td rowspan="1" colspan="1" valign="top" align="left">805</td><td rowspan="1" colspan="1" valign="top" align="left">230 (30.0%)<br>NA-39</td><td rowspan="1" colspan="1" valign="top" align="center">56.8 (12.4)</td><td rowspan="1" colspan="1" valign="top" align="left">7.2 (2.9)</td><td rowspan="1" colspan="1" valign="top" align="left">N0-338<br>N1-446<br>NA-21</td><td rowspan="1" colspan="1" valign="top" align="left">M0-740<br>M1-57<br>NA-8</td><td rowspan="1" colspan="1" valign="top" align="left">Negative-402<br>Positive-86<br>NA-317</td></tr></tbody></table></div><div id="largeobj_idm140376115932096" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3601979/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P35" class="p p-first-last">NA=not available,</p></div><div id="TFN2"><p id="P36" class="p p-first-last">HER2= Human Epidermal Growth Factor Receptor 2, N-stage=metastasis to lymph node, M-stage=metastasis</p></div></div></div><p id="P20" class="p">41 of the remaining 42 SNPs achieved p-values at10<sup>−5</sup>. At the loci on which multiple SNPs were found to be strongly associated with survival we selected the lead SNP for follow-up in stage-2 along with any other SNP(s) from the same locus which were not in high LD with the lead SNP (r<sup>2</sup>&lt;0.6). Using this strategy we selected 35 of the best 50 associated SNPs (<a href="#SD7" rid="SD7" class=" supplementary-material">Supplementary Table 1</a>) for genotyping in the stage-2 validation samples. The qq-plot demonstrated deviation of observed log transformed values from the expected log transformed p-values for SNPs associated with p-values ranging from 10<sup>−4</sup> to 10<sup>−5</sup> (<a href="/pmc/articles/PMC3601979/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3601979/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376131326336" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is emss-51101-f0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is emss-51101-f0001.jpg" src="/pmc/articles/PMC3601979/bin/emss-51101-f0001.jpg"></div></a><div id="largeobj_idm140376131326336" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3601979/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3601979/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3601979/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">A quantile-quantile (qq plot) of log-transformed observed and expected P-values from the stage-1 analysis:</p></div></div></div><div id="S15" class="sec"><h3 id="S15title">Stage-2 Results</h3><p id="P21" class="p p-first-last">27 of the 35 SNPs included in the stage-2 genotyping were successfully genotyped and were available for analysis. 1 SNP had greater than 10% duplicate error rate and was excluded from replication analysis. We had 70% power to detect a HR (hazard ratio) ≥ 1.50 in stage 2 analysis. While testing for replication effects of the 27 SNPs, we found 11 SNPs at distinct loci which were associated with prognosis in the same direction as in the stage-1 analysis (<a href="/pmc/articles/PMC3601979/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Replication p-values for these 11 SNPs ranged from 0.05 to 0.005.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p7">Stage Wise association statistics for the eleven SNPs which were associated in stage-2 following discovery in stage-1 analysis:</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376134916352" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP (Minor Allele<br>Frequency)</th><th rowspan="1" colspan="1" valign="top" align="left">POSH stage-1<br>HR (95% CI), p-value</th><th rowspan="1" colspan="1" valign="top" align="left">POSH stage-2<br>replications HR (95%<br>CI), p-value</th><th rowspan="1" colspan="1" valign="top" align="left">Stage-1 and Stage-2 meta-analysis<br>results</th><th rowspan="1" colspan="1" valign="top" align="left">I<sup>2</sup> derived from<br>Cochran’s Q-statistic, p-<br>value for Q-statistic</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs421379 (0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">1.98 (1.46-2.70), p=1.2×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.42 (1.11-1.8), p=0.005</td><td rowspan="1" colspan="1" valign="top" align="left">1.61 (1.33-1.96), p=9.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">63.7%, p=0.10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3884558 (0.07)</td><td rowspan="1" colspan="1" valign="top" align="left">1.84 (1.40-2.41), p=1.3×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.29 (1.05-1.57), p=0.01</td><td rowspan="1" colspan="1" valign="top" align="left">1.46 (1.24-1.72), p=3.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">76.4%, p=0.04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs971398 (0.17)</td><td rowspan="1" colspan="1" valign="top" align="left">1.52 (1.23-1.88, p=1.2 × 10<sup>−4</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">1.24 (1.05-1.47, p=0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">1.34 (1.18-1.53, p=1.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">46.3%, p=0.17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7910841 (0.28)</td><td rowspan="1" colspan="1" valign="top" align="left">0.64 (0.51-0.80, p=8.2 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.83 (0.72-0.96, p=0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.77 (0.68-0.87), p=2.3 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">72.5%, p=0.06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12523819 (0.28)</td><td rowspan="1" colspan="1" valign="top" align="left">0.64 (0.51-0.80, p=1.1 × 10<sup>−4</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.86 (0.74-0.99), p=0.04</td><td rowspan="1" colspan="1" valign="top" align="left">0.77 (0.88-0.87), p=2.3 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">78.6%, p=0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3785982 (0.12)</td><td rowspan="1" colspan="1" valign="top" align="left">1.75 (1.36-2.24, p=1.3 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">1.24 (1.03-1.48, p=0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">1.40 (1.21-1.62), p=7.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">79.1%, p=0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2774307 (0.26)</td><td rowspan="1" colspan="1" valign="top" align="left">1.51 (1.24-1.85, p=4.3 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">1.21 (1.05-1.40, p=0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">1.30 (1.16-1.47), p=7.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">67.8%, p=0.08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10220397 (0.23)</td><td rowspan="1" colspan="1" valign="top" align="left">0.63 (0.50-0.79, p=8.6 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.85 (0.73-0.99, p=0.04)</td><td rowspan="1" colspan="1" valign="top" align="left">0.77 (0.68-0.88), p=8.3 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">78.1%, p=0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs303850 (0.42)</td><td rowspan="1" colspan="1" valign="top" align="left">1.48 (1.22-1.79, p=5.2 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">1.13 (1.00-1.29, p=0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">1.23 (1.10-1.36), p=1.5 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">81.1%, p=0.02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1387389 (0.36)</td><td rowspan="1" colspan="1" valign="top" align="left">1.47 (1.22-1.77, p=5.0 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">1.20 (1.05-1.37, p=0.007)</td><td rowspan="1" colspan="1" valign="top" align="left">1.28 (1.16-1.43), p=3.8 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">66.6%, p=0.08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1513848 (0.07)</td><td rowspan="1" colspan="1" valign="top" align="left">1.87 (1.41-2.46, p=1.0 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">1.25 (1.01-1.55, p=0.04</td><td rowspan="1" colspan="1" valign="top" align="left">1.45 (1.22-1.72), p=1.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">80.2%, p=0.02</td></tr></tbody></table></div><div id="largeobj_idm140376134916352" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3601979/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S16" class="sec"><h3 id="S16title">Stage-1 and Stage-2 meta-analysis</h3><p id="P22" class="p p-first-last">We included the eleven SNPs which remained associated in stage-2 based on consistent direction of effect, in Mantel-Haenszel fixed-effects meta-analysis models (<a href="/pmc/articles/PMC3601979/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table-2</span></a>). The strongest meta-analysis HR was observed at the rs421379 SNP which lies upstream of the <em>ARRDC3</em> gene (<a href="/pmc/articles/PMC3601979/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>, <a href="/pmc/articles/PMC3601979/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>) on the long arm of chromosome 5, HR=1.61 (1.33-1.96, p=9.5 × 10<sup>−</sup>
<sup>7</sup>). Adjusting for ER-status, N-stage and M-stage slightly reduced the strength of the overall association at this SNP in combined analysis across stage-1 and stage-2 (HR=1.55 (1.27-1.90, p=1.5 × 10<sup>−5</sup>)). The next best replication signal was observed in an intronic region of the <em>PBX1</em> (Pre-B-Cell Leukaemia transcription factor-1) gene. The replication p-value for this intronic SNP was second most significant after rs421379 in the two stage meta-analysis, and the overall association at this variant was close to being genome wide significant, HR=1.28 (1.16-1.43, p=3.8 × 10<sup>−6</sup>). Adjusting for ER-status, N-stage and M-stage did not affect the strength of the association at this variant (HR=1.26 (1.13-1.41, p=3.9 × 10<sup>−5</sup>)). The above two variants displayed the lowest levels of heterogeneity in meta-analysis. The association observed with a 5′UTR snp at the <em>RORα</em> locus (rs3884558) was also close to the threshold for genome wide significance HR=1.46 (1.24-1.72, p=3.9 × 10<sup>−6</sup>), although there was modest evidence of heterogeneity in Hazard ratios between stage-1 and stage-2 models (<a href="/pmc/articles/PMC3601979/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Two further associations rs3785982 in the <em>NTN1</em> gene (HR=1.40 (1.21-1.62, p=7.9 × 10<sup>−6</sup>) and rs2774307 in the <em>SYT6</em> gene (HR=1.30 (1.16-1.47, p=7.9 × 10<sup>−6</sup>)) also came close to genome wide significance. For the five SNPs associated at p≤10<sup>−6</sup>, we did not observe any eveidence of heterogeneity of effects on survival based on triple negative status of the POSH patients. The heterogeneity I<sup>2</sup>-statistic for these 5 SNPs ranged from 0-20.6%.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3601979/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376205924032" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is emss-51101-f0002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is emss-51101-f0002.jpg" src="/pmc/articles/PMC3601979/bin/emss-51101-f0002.jpg"></div></a><div id="largeobj_idm140376205924032" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3601979/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3601979/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3601979/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Regional plot of p-values arising from cox-proportional hazard models, 250 kb either side of the rs421379 variant in stage-1. P-values are from imputed and the genotyped SNPs.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3601979/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376121388656" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is emss-51101-f0003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is emss-51101-f0003.jpg" src="/pmc/articles/PMC3601979/bin/emss-51101-f0003.jpg"></div></a><div id="largeobj_idm140376121388656" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3601979/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC3601979/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3601979/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Kaplan Meier analysis plot depicting survival rates by rs421379 genotypes:</p></div></div></div></div><div id="S17" class="sec"><h3 id="S17title">Replication attempt in non-age-specific survival analysis</h3><p id="P23" class="p p-first-last">We had 87% power to detect a hazard ratio ≥1.50 when analysing SNPs with Maf≥0.10 in 874 patients available from the Helsinki study. We extracted genotypes from the GWS genotype data of the Helsinki samples for 11 of the 35 SNPs which were associated in the same direction in stage-2 as in stage-1 analysis. Helsinki patients belonged to a relatively higher age group at diagnosis when compared to POSH (Average at diagnosis=56.8, SD=12.4). We found that none of the 11 SNPs which were replicated as associated with prognosis in stage-2 were associated with the same outcome in patients with later onset from Helsinki.</p></div><div id="S18" class="sec"><h3 id="S18title">Association scan with imputed SNP data</h3><p id="P24" class="p p-first-last">We imputed SNP genotypes for 2.5 million SNPs based on HapMap Phase 2 data. Imputation analysis did not identify any additional variants which were more strongly associated than the ones we found as most strongly associated using real genotype data 250 kb either side of rs421379, rs3884558 (5′UTR-<em>RORα</em>), rs3785982 (<em>NTN1</em>), rs2774307 (<em>SYT6</em>) and rs1387389 (<em>PBX1</em>) (<a href="/pmc/articles/PMC3601979/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figures 2</a>-<a href="#SD5" rid="SD5" class=" supplementary-material">5</a>).</p></div><div id="S19" class="sec"><h3 id="S19title">Associations of the five best associations with clinical predictors</h3><p id="P25" class="p p-first-last">We assessed the associations of all SNPs which were associated at p-value≤10<sup>−6</sup> with clinical predictors of breast cancer prognosis. There were five SNPs which were associated at p-value≤10<sup>−6</sup>, none of these SNPs were associated with ER-status, N-stage or M-stage after performing a Bonferroni correction for number of tests performed (<a href="/pmc/articles/PMC3601979/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p8">Associations of SNPs associated at p≤10<sup>−6</sup> in the two stage meta-analysis with secondary traits linked to breast cancer mortality in stage 1 and stage 2 combined dataset:</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376118210928" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Secondary<br>trait</th><th rowspan="1" colspan="1" valign="top" align="left">rs421379</th><th rowspan="1" colspan="1" valign="top" align="left">rs3884558</th><th rowspan="1" colspan="1" valign="top" align="left">rs3785982</th><th rowspan="1" colspan="1" valign="top" align="left">rs2774307</th><th rowspan="1" colspan="1" valign="top" align="left">rs1387389</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">ER-status</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.13 (95% CI:0.86<br>to 1.47, p=0.38)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=0.93 (95% CI:0.75<br>to 1.16, p=0.52)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.14 (95% CI:0.94<br>to 1.39, p=0.54)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=0.96 (95% CI:0.83<br>to 1.11, p=0.59)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=0.97 (95% CI:0.85<br>to 1.10, p=0.29)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Nodal<br>Status</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.55 (95%CI:0.96-<br>2.49, p=0.07)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.70 (95%<br>CI:1.14-2.55, p=0.009)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.47 (95% CI:1.06-<br>2.05, p=0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.16<br>(95% CI:0.92-1.45,<br>p=0.20)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.20 (95%CI:0.98-<br>1.46, p=0.05)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">M-stage</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.22 (95%CI:0.88-<br>1.68, p=0.24)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.42 (95%<br>CI:1.07-1.87, p=0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.23 (95% CI:0.97-<br>1.56, p=0.08)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.04 (95%<br>CI:0.88-1.23, p=0.62)</td><td rowspan="1" colspan="1" valign="top" align="left">OR=1.07 (95%CI:0.92-<br>1.24, p=0.40)</td></tr></tbody></table></div><div id="largeobj_idm140376118210928" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3601979/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="P37" class="p p-first-last">ER-status= Estrogen receptor status, M-stage=Metastasis stage</p></div></div></div></div><div id="S20" class="sec sec-last"><h3 id="S20title">Gene Expression variation by SNP</h3><p id="P26" class="p p-first-last">We queried the Genevar 3.2.0 and SNP and CNV annotation database (scandb) to identify Cis or Trans eQTL effects resulting from rs421379. In 156 Lymphoblastoid cell lines sample collected from 78 twin pairs available via the Java based Genevar interface we did not observe an association of rs421379 with expression of <em>ARRDC3</em> gene (<a href="/pmc/articles/PMC3601979/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). In scandb we observed that rs421379 had trans eQTL effects on expression of <em>RAB34</em> (p=1× 10<sup>−5</sup>) and <em>ABCD1</em> (p=9 × 10<sup>−5</sup>), but neither of these associations were genome wide significant which could be a result of low sample size available with 30 HapMap-CEU trios available from scandb.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3601979/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376121385696" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is emss-51101-f0004.jpg" title="An external file that holds a picture, illustration, etc.
Object name is emss-51101-f0004.jpg" src="/pmc/articles/PMC3601979/bin/emss-51101-f0004.jpg"></div></a><div id="largeobj_idm140376121385696" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3601979/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a target="object" href="/pmc/articles/PMC3601979/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3601979/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p5">Variation in ARRDC3 expression levels with rs421379 variant. Results are reported from twins from the same pair who were separated by id in two samples named Twin1-L and Twin2-L, which were analysed independently. Rho is the spearman rank correlation (SRC) coefficient, p-value is for SRC and Pemp is the empirical p-value for SRC based on 10,000 permutations:</p></div></div></div></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Discussion</h2><p id="P27" class="p p-first">In this manuscript we have reported findings from the first genome wide association study of breast cancer prognosis in early onset breast cancer patients and enriched for poor survival and ER, PR and HER-2 negative cases in discovery stage. Recently two GWASs aimed at identifying risk alleles for poor prognosis were performed in unselected breast cancer patients of Chinese and European ancestries. Azzato et al (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333665240">12</a>) in their GWAS conducted in 4335 Caucasian breast cancer patients with mean age at diagnosis of 66 years (95% CI=44-83) did not replicate any of their main findings in a large relatively younger cohort of breast cancer patients (Mean age=51 (95% CI=23-69). Azzato et al (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333665242">12</a>) had taken forward 10 of their most significant findings forward for replication in the SEARCH study. On the contrary Shu et al (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020830">13</a>), attempted replication of their top 50 associations in their two stage GWAS in Chinese patients and identified two associations with p-values equal to 1.17 × 10<sup>−7</sup> (rs3784099, <em>RAD15L</em>) and 5.75 × 10<sup>−6</sup> (rs9934948). We did not find either of these two SNPs (rs3784099, p=0.61 and rs9934948, p=0.25) as associated with prognosis in the stage-1 data used for discovery in this GWAS nor in the Helsinki GWS data.</p><p id="P28">In our study we attempted replication of 50 of the strongest association signals from stage-1 by selecting the strongest associated SNPs at each of the new discovered loci while excluding any other SNPs which were in relative LD with the best associated SNP at the same locus r<sup>2</sup>≥0.6. Of the 35 SNPs which were selected from stage-1 for validation in stage-2, 27 SNPs were successfully genotyped in stage-2 and we found 11 of these SNPs to demonstrate nominal to strong replication signals (p-values range: 0.05-5 × 10<sup>−3</sup>). Such a high replication rate (40.7%) suggests low phenotypic heterogeneity in samples collected between stage-1 and stage-2. Large cohorts of young onset patients with comprehensive treatment and outcome data are uncommon given that less than 5% of all breast cancers are diagnosed before 40 years of age. We were able to enrich our stage-1 samples further with young onset triple negative patients and patients with very short duration of breast cancer specific survival. This allowed us increased statistical power to identify common genetic variants with modest effect sizes (OR≥1.50) in stage-1 data. Despite having a more enriched stage-1 dataset, we did not have sufficient power (&lt;80%) to detect association signals associated with HRs in range of 1.10 to 1.45 in our discovery samples. Future studies in larger early onset cohorts are therefore needed to identify true associations with lower effect sizes than HR≥1.50.</p><p id="P29">The strongest association signal in our study was observed 596 kb upstream of the <em>ARRDC3</em> gene. In HapMap we did not find any long range LD between rs421379 and any SNPs at or close to the <em>ARRDC3</em> locus. The <em>ARRDC3</em> gene is a member of the arrestin gene family and functions in a novel regulatory pathway that controls the cell surface adhesion molecule, b-4 integrin (ITGb4), a protein associated with aggressive tumour behaviour (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020811">30</a>). Furthermore deletion of the region of chromosome 5 containing the <em>ARRDC3</em> gene is observed more frequently in basal type breast cancer cancers (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020810">31</a>). Differential expression levels have also been associated with prognosis in prostate cancer patients (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020833">32</a>). The associated SNP rs421379 is located in the 5′ region of the <em>ARRDC3</em> gene and might affect a transcription binding site and <em>ARRDC3</em> gene expression, permitting development of a more aggressive, invasive tumour. The associated SNP rs421379 was predicted to disrupt a binding site for Myocyte-specific enhancer factor 2 (MEF2). This transcription factor family consists of four members (MEF2A-D) sharing the binding sequence and MEF2 could regulate <em>ARRDC3</em> gene expression. Previously, <em>MEF2C</em> has been found to be highly expressed in basal breast cancer along with Notch (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020820">33</a>). Later a strong co-expression of Notch1 and MEF2 paralogs has been observed in breast cancer tumour samples from patients with metastatic disease (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020813">34</a>).</p><p id="P30">We did not find a robust association signal between rs421379 and the probe representing variation in <em>ARRDC3</em> expression (<a href="/pmc/articles/PMC3601979/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). In the SNP and CNV annotation database (scandb) (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020834">35</a>), rs421379 is identified to have trans-effects on expression of <em>RAB34</em> (p=1× 10<sup>−5</sup>) and <em>ABCD1</em> (p=9 × 10<sup>−5</sup>) genes present on chromosomes 17 and X respectively. It should be noted that the association analysis in Genevar 3.2.0 and scandb analysis is based on 154 twins and the 30 HapMap CEU trios respectively as such the statistical power to detect modest effects on gene expression was not high and further given that the gene expression is quantified in lymphoblastoid cell lines these results are not reflective of potential cis-effect of rs421379 in breast cancer related cells.</p><p id="P31">The second strongest association we observed was at the <em>PBX1</em> locus (HR=1.28 (1.16-1.43), p=3.8 × 10<sup>−6</sup>). The protein coded by this gene drives ERα signalling and breast cancer progression through transcriptional programming (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020809">36</a>). There was no evidence of substantial heterogeneity in hazard ratios across stage-1 and stage-2 for the SNP associated at the <em>PBX1</em> locus (<a href="/pmc/articles/PMC3601979/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). 11 SNPs according to HapMap Phase 3 data are in high LD (r<sup>2</sup>&gt;0.8) with the <em>PBX1</em> SNP we found as strongly associated and all these SNPs were found to be intronic within the <em>PBX1</em> locus. We also observed strong suggestive evidence for an association at the <em>RORα</em> gene (HR=1.46 (1.24-1.72), p=3.9 × 10<sup>−6</sup>). It has recently been shown that RORα protein expression is reduced in breast cancer cells and also this lower expression is related to poorer prognosis in breast cancer patients (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020812">37</a>). There was some evidence of heterogeneity in hazard ratios between stage-1 and stage-2 patients for the rs3884558 which lies 78.3 kb upstream of the <em>RORα</em> gene (p=0.04). Although there were no SNPs in high LD (r<sup>2</sup>≥0.8) 2.4 kb beyond rs3884558, SNPs in moderately LD (r<sup>2</sup>=0.3) are located up to 95.7 kb away from rs3884558 and close to the <em>RORα</em> gene. The SNP rs3884558 was predicted to both disrupt and create multiple transcription factor binding sites. The binding site was predicted to be lost for transcription factors POU2F1, TGIF1, HMGA1/2 and CDX2 and new binding site was predicted to emerge for REV-ERBα, CREB1/2, HMGA1, VBP1 and E4F1. Interestingly REV-ERBα belongs to the same nuclear hormone receptor family as does RORα. Moreover these family members are known to cross-talk and REV-ERBα has been shown to suppress the transcriptional activity of RORα (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020829">38</a>).</p><p id="P32">The two other associations at <em>NTN1</em> and <em>SYT6</em> could also be real given <em>NTN1</em> expression is increased in breast cancer (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303020828">39</a>) and the replication signal at <em>SYT6</em> remained strong in post replication meta-analysis (HR=1.30 (1.16-1.47), p=7.9 × 10<sup>−6</sup>) with no strong evidence of heterogeneity in hazard ratios with the associated variant.</p><p id="P33" class="p p-last">Further studies at population level are needed to confirm the association of the 5 loci associated at p≤10<sup>−6</sup> that we have discovered from this two stage GWAS analysis. In future analyses we will study the most strongly associated SNPs from the current study by interrogating additional well characterised and early onset breast cancer cohorts. This will allow us to generate more accurate estimates of gene-survival associations and also allow the implementation of prediction statistics generated using gene score analysis. In addition, further studies are needed to establish beyond doubt the true validity of the remaining SNPs that replicated strongly but were not quite genome wide significant. Published results from biochemical analyses do suggest <em>PBX1, RORα</em> and <em>NTN1</em> are plausible candidate loci for an effect exerted by the host genotype in altering prognosis. Fine mapping and molecular studies are needed to establish the identity of the causal variant in the intragenic region, 596 kb upstream of <em>ARRDC3</em> gene. and provide insight into the mechanism of action. Much emphasis currently is on genotyping of somatic mutations in tumours to help refine prognosis and identify treatment targets but this is only a part of the information that influences prognosis. Selecting a well characterised poor prognosis group of patients with high breast cancer specific mortality has been a useful strategy to identifying SNPs that influence prognosis. The ultimate validation of the clinical utility for germ-line genetic variants that influence prognosis will come from genotyping in randomised adjuvant and neo-adjuvant treatment trials. With a clear understanding of the magnitude and mechanism of prognostic SNPs, such genotyping may in future be routinely used in cancer patients to help derive a more complete individualised risk assessment for early relapse and thereby guide treatment choices.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d35e802"><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-1.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(27K, tif)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d35e806"><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-2.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(93K, tif)</sup></div></div><div class="sec suppmat" id="SD3"><h4>3</h4><div class="sup-box half_rhythm" id="d35e810"><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-3.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(111K, tif)</sup></div></div><div class="sec suppmat" id="SD4"><h4>4</h4><div class="sup-box half_rhythm" id="d35e814"><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-4.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(104K, tif)</sup></div></div><div class="sec suppmat" id="SD5"><h4>5</h4><div class="sup-box half_rhythm" id="d35e818"><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-5.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(117K, tif)</sup></div></div><div class="sec suppmat" id="SD6"><h4>6</h4><div class="sup-box half_rhythm" id="d35e822"><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-6.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(18K, doc)</sup></div></div><div class="sec suppmat" id="SD7"><h4>Table 1</h4><div class="sup-box half_rhythm" id="d35e826"><a href="/pmc/articles/PMC3601979/bin/NIHMS51101-supplement-Table_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(70K, pdf)</sup></div></div></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Acknowledgments</h2><div class="sec"><p id="__p1">We would like to thank Nikki Graham (DNA bank) and the staff of the Southampton CRUK Centre Tissue Bank. The POSH study is supported by Breast Cancer Campaign grant number: 2010NovPR62. Funding for the POSH study was also provided by The Wessex Cancer Trust and Cancer Research UK (grant refs A7572, A11699, C22524). We wish to thank Drs. Kristiina Aittomäki, Kirsimari Aaltonen and Karl von Smitten and RN Irja Erkkilä for their help with the Helsinki data and samples. The Helsinki study was financially supported by the Helsinki University Central Hospital Research Fund, Academy of Finland (132473), the Finnish Cancer Society, The Nordic Cancer Union and the Sigrid Juselius Foundation. Genotyping at the National Institute of Singapore was financially supported by the Agency for Science, Technology and Research (A*STAR), Singapore.</p></div></div><div id="idm140376133262352" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140376133262352title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P34" class="p p-first-last"><strong>Conflict of Interests</strong>: The authors would like to disclose that no potential conflicts of interest exist</p><p></p></div></div><div id="idm140376133261104" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140376133261104title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation"><span></span>  <a href="http://info.cancerresearchuk.org/cancerstats/types/breast/incidence/#age" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://info.cancerresearchuk.org/cancerstats/types/breast/incidence/#age</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Gonzalez-Angulo AM, Broglio K, Kau SW, Eralp Y, Erlichman J, Valero V, et al.  Women age &lt; or = 35 years with primary breast carcinoma: disease features at presentation. <span><span class="ref-journal">Cancer. </span>2005;<span class="ref-vol">103</span>:2466–72.</span> [<a href="/pubmed/15852360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Women+age+<+or+=+35+years+with+primary+breast+carcinoma:+disease+features+at+presentation&amp;author=AM+Gonzalez-Angulo&amp;author=K+Broglio&amp;author=SW+Kau&amp;author=Y+Eralp&amp;author=J+Erlichman&amp;volume=103&amp;publication_year=2005&amp;pages=2466-72&amp;pmid=15852360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Walker RA, Lees E, Webb MB, Dearing SJ. Breast carcinomas occurring in young women (&lt; 35 years) are different. <span><span class="ref-journal">British journal of cancer. </span>1996;<span class="ref-vol">74</span>:1796–800.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2077218/">PMC free article</a>]</span> [<a href="/pubmed/8956795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=British+journal+of+cancer&amp;title=Breast+carcinomas+occurring+in+young+women+(<+35+years)+are+different&amp;author=RA+Walker&amp;author=E+Lees&amp;author=MB+Webb&amp;author=SJ+Dearing&amp;volume=74&amp;publication_year=1996&amp;pages=1796-800&amp;pmid=8956795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Anders CK, Hsu DS, Broadwater G, Acharya CR, Foekens JA, Zhang Y, et al.  Young age at diagnosis correlates with worse prognosis and defines a subset of breast cancers with shared patterns of gene expression. <span><span class="ref-journal">Journal of clinical oncology : official journal of the American Society of Clinical Oncology. </span>2008;<span class="ref-vol">26</span>:3324–30.</span> [<a href="/pubmed/18612148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+clinical+oncology+:+official+journal+of+the+American+Society+of+Clinical+Oncology&amp;title=Young+age+at+diagnosis+correlates+with+worse+prognosis+and+defines+a+subset+of+breast+cancers+with+shared+patterns+of+gene+expression&amp;author=CK+Anders&amp;author=DS+Hsu&amp;author=G+Broadwater&amp;author=CR+Acharya&amp;author=JA+Foekens&amp;volume=26&amp;publication_year=2008&amp;pages=3324-30&amp;pmid=18612148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Ahn SH, Son BH, Kim SW, Kim SI, Jeong J, Ko SS, et al.  Poor outcome of hormone receptor-positive breast cancer at very young age is due to tamoxifen resistance: nationwide survival data in Korea--a report from the Korean Breast Cancer Society. <span><span class="ref-journal">Journal of clinical oncology : official journal of the American Society of Clinical Oncology. </span>2007;<span class="ref-vol">25</span>:2360–8.</span> [<a href="/pubmed/17515570" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+clinical+oncology+:+official+journal+of+the+American+Society+of+Clinical+Oncology&amp;title=Poor+outcome+of+hormone+receptor-positive+breast+cancer+at+very+young+age+is+due+to+tamoxifen+resistance:+nationwide+survival+data+in+Korea--a+report+from+the+Korean+Breast+Cancer+Society&amp;author=SH+Ahn&amp;author=BH+Son&amp;author=SW+Kim&amp;author=SI+Kim&amp;author=J+Jeong&amp;volume=25&amp;publication_year=2007&amp;pages=2360-8&amp;pmid=17515570&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">El Saghir NS, Seoud M, Khalil MK, Charafeddine M, Salem ZK, Geara FB, et al.  Effects of young age at presentation on survival in breast cancer. <span><span class="ref-journal">BMC cancer. </span>2006;<span class="ref-vol">6</span>:194.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1555600/">PMC free article</a>]</span> [<a href="/pubmed/16857060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+cancer&amp;title=Effects+of+young+age+at+presentation+on+survival+in+breast+cancer&amp;author=NS+El+Saghir&amp;author=M+Seoud&amp;author=MK+Khalil&amp;author=M+Charafeddine&amp;author=ZK+Salem&amp;volume=6&amp;publication_year=2006&amp;pages=194&amp;pmid=16857060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Hartman M, Lindstrom L, Dickman PW, Adami HO, Hall P, Czene K. Is breast cancer prognosis inherited? <span><span class="ref-journal">Breast cancer research : BCR. </span>2007;<span class="ref-vol">9</span>:R39.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1929105/">PMC free article</a>]</span> [<a href="/pubmed/17598882" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+cancer+research+:+BCR&amp;title=Is+breast+cancer+prognosis+inherited?&amp;author=M+Hartman&amp;author=L+Lindstrom&amp;author=PW+Dickman&amp;author=HO+Adami&amp;author=P+Hall&amp;volume=9&amp;publication_year=2007&amp;pages=R39&amp;pmid=17598882&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Evans DG, Howell A, Ward D, Lalloo F, Jones JL, Eccles DM. Prevalence of BRCA1 and BRCA2 mutations in triple negative breast cancer. <span><span class="ref-journal">J Med Genet. </span>2011;<span class="ref-vol">48</span>:520–2.</span> [<a href="/pubmed/21653198" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Prevalence+of+BRCA1+and+BRCA2+mutations+in+triple+negative+breast+cancer&amp;author=DG+Evans&amp;author=A+Howell&amp;author=D+Ward&amp;author=F+Lalloo&amp;author=JL+Jones&amp;volume=48&amp;publication_year=2011&amp;pages=520-2&amp;pmid=21653198&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Robertson L, Hanson H, Seal S, Warren-Perry M, Hughes D, Howell I, et al.  BRCA1 testing should be offered to individuals with triple-negative breast cancer diagnosed below 50 years. <span><span class="ref-journal">British journal of cancer. </span>2012;<span class="ref-vol">106</span>:1234–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3304410/">PMC free article</a>]</span> [<a href="/pubmed/22333603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=British+journal+of+cancer&amp;title=BRCA1+testing+should+be+offered+to+individuals+with+triple-negative+breast+cancer+diagnosed+below+50+years&amp;author=L+Robertson&amp;author=H+Hanson&amp;author=S+Seal&amp;author=M+Warren-Perry&amp;author=D+Hughes&amp;volume=106&amp;publication_year=2012&amp;pages=1234-8&amp;pmid=22333603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Turnbull C, Ahmed S, Morrison J, Pernet D, Renwick A, Maranian M, et al.  Genome-wide association study identifies five new breast cancer susceptibility loci. <span><span class="ref-journal">Nature genetics. </span>2010;<span class="ref-vol">42</span>:504–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3632836/">PMC free article</a>]</span> [<a href="/pubmed/20453838" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Genome-wide+association+study+identifies+five+new+breast+cancer+susceptibility+loci&amp;author=C+Turnbull&amp;author=S+Ahmed&amp;author=J+Morrison&amp;author=D+Pernet&amp;author=A+Renwick&amp;volume=42&amp;publication_year=2010&amp;pages=504-7&amp;pmid=20453838&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Haiman CA, Chen GK, Vachon CM, Canzian F, Dunning A, Millikan RC, et al.  A common variant at the TERT-CLPTM1L locus is associated with estrogen receptor-negative breast cancer. <span><span class="ref-journal">Nature genetics. </span>2011;<span class="ref-vol">43</span>:1210–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3279120/">PMC free article</a>]</span> [<a href="/pubmed/22037553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=A+common+variant+at+the+TERT-CLPTM1L+locus+is+associated+with+estrogen+receptor-negative+breast+cancer&amp;author=CA+Haiman&amp;author=GK+Chen&amp;author=CM+Vachon&amp;author=F+Canzian&amp;author=A+Dunning&amp;volume=43&amp;publication_year=2011&amp;pages=1210-4&amp;pmid=22037553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Azzato EM, Pharoah PD, Harrington P, Easton DF, Greenberg D, Caporaso NE, et al.  A genome-wide association study of prognosis in breast cancer. <span><span class="ref-journal">Cancer epidemiology, biomarkers &amp; prevention : a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology. </span>2010;<span class="ref-vol">19</span>:1140–3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852476/">PMC free article</a>]</span> [<a href="/pubmed/20332263" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+epidemiology,+biomarkers+&amp;+prevention+:+a+publication+of+the+American+Association+for+Cancer+Research,+cosponsored+by+the+American+Society+of+Preventive+Oncology&amp;title=A+genome-wide+association+study+of+prognosis+in+breast+cancer&amp;author=EM+Azzato&amp;author=PD+Pharoah&amp;author=P+Harrington&amp;author=DF+Easton&amp;author=D+Greenberg&amp;volume=19&amp;publication_year=2010&amp;pages=1140-3&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Shu XO, Long J, Lu W, Li C, Chen WY, Delahanty R, et al.  Novel genetic markers of breast cancer survival identified by a genome-wide association study. <span><span class="ref-journal">Cancer research. </span>2012;<span class="ref-vol">72</span>:1182–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3294129/">PMC free article</a>]</span> [<a href="/pubmed/22232737" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+research&amp;title=Novel+genetic+markers+of+breast+cancer+survival+identified+by+a+genome-wide+association+study&amp;author=XO+Shu&amp;author=J+Long&amp;author=W+Lu&amp;author=C+Li&amp;author=WY+Chen&amp;volume=72&amp;publication_year=2012&amp;pages=1182-9&amp;pmid=22232737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Tapper W, Hammond V, Gerty S, Ennis S, Simmonds P, Collins A, et al.  The influence of genetic variation in 30 selected genes on the clinical characteristics of early onset breast cancer. <span><span class="ref-journal">Breast cancer research : BCR. </span>2008;<span class="ref-vol">10</span>:R108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2656905/">PMC free article</a>]</span> [<a href="/pubmed/19094228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+cancer+research+:+BCR&amp;title=The+influence+of+genetic+variation+in+30+selected+genes+on+the+clinical+characteristics+of+early+onset+breast+cancer&amp;author=W+Tapper&amp;author=V+Hammond&amp;author=S+Gerty&amp;author=S+Ennis&amp;author=P+Simmonds&amp;volume=10&amp;publication_year=2008&amp;pages=R108&amp;pmid=19094228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Fasching PA, Pharoah PD, Cox A, Nevanlinna H, Bojesen SE, Karn T, et al.  The role of genetic breast cancer susceptibility variants as prognostic factors. <span><span class="ref-journal">Hum Mol Genet. </span>2012;<span class="ref-vol">21</span>:3926–39.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3412377/">PMC free article</a>]</span> [<a href="/pubmed/22532573" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+role+of+genetic+breast+cancer+susceptibility+variants+as+prognostic+factors&amp;author=PA+Fasching&amp;author=PD+Pharoah&amp;author=A+Cox&amp;author=H+Nevanlinna&amp;author=SE+Bojesen&amp;volume=21&amp;publication_year=2012&amp;pages=3926-39&amp;pmid=22532573&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Eccles D, Gerty S, Simmonds P, Hammond V, Ennis S, Altman DG. Prospective study of Outcomes in Sporadic versus Hereditary breast cancer (POSH): study protocol. <span><span class="ref-journal">BMC cancer. </span>2007;<span class="ref-vol">7</span>:160.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995215/">PMC free article</a>]</span> [<a href="/pubmed/17697367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+cancer&amp;title=Prospective+study+of+Outcomes+in+Sporadic+versus+Hereditary+breast+cancer+(POSH):+study+protocol&amp;author=D+Eccles&amp;author=S+Gerty&amp;author=P+Simmonds&amp;author=V+Hammond&amp;author=S+Ennis&amp;volume=7&amp;publication_year=2007&amp;pages=160&amp;pmid=17697367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Eccles D, Gerty S, Simmonds P, Hammond V, Ennis S, Altman DG, et al.  Prospective study of Outcomes in Sporadic versus Hereditary breast cancer (POSH): study protocol. <span><span class="ref-journal">BMC cancer. </span>2007;<span class="ref-vol">7</span>:160.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995215/">PMC free article</a>]</span> [<a href="/pubmed/17697367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+cancer&amp;title=Prospective+study+of+Outcomes+in+Sporadic+versus+Hereditary+breast+cancer+(POSH):+study+protocol&amp;author=D+Eccles&amp;author=S+Gerty&amp;author=P+Simmonds&amp;author=V+Hammond&amp;author=S+Ennis&amp;volume=7&amp;publication_year=2007&amp;pages=160&amp;pmid=17697367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Wang K, Li WD, Zhang CK, Wang Z, Glessner JT, Grant SF, et al.  A genome-wide association study on obesity and obesity-related traits. <span><span class="ref-journal">PloS one. </span>2011;<span class="ref-vol">6</span>:e18939.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3084240/">PMC free article</a>]</span> [<a href="/pubmed/21552555" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+one&amp;title=A+genome-wide+association+study+on+obesity+and+obesity-related+traits&amp;author=K+Wang&amp;author=WD+Li&amp;author=CK+Zhang&amp;author=Z+Wang&amp;author=JT+Glessner&amp;volume=6&amp;publication_year=2011&amp;pages=e18939&amp;pmid=21552555&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Fagerholm R, Hofstetter B, Tommiska J, Aaltonen K, Vrtel R, Syrjakoski K, et al.  NAD(P)H:quinone oxidoreductase 1 NQO1*2 genotype (P187S) is a strong prognostic and predictive factor in breast cancer. <span><span class="ref-journal">Nature genetics. </span>2008;<span class="ref-vol">40</span>:844–53.</span> [<a href="/pubmed/18511948" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=NAD(P)H:quinone+oxidoreductase+1+NQO1*2+genotype+(P187S)+is+a+strong+prognostic+and+predictive+factor+in+breast+cancer&amp;author=R+Fagerholm&amp;author=B+Hofstetter&amp;author=J+Tommiska&amp;author=K+Aaltonen&amp;author=R+Vrtel&amp;volume=40&amp;publication_year=2008&amp;pages=844-53&amp;pmid=18511948&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Haiman CA, Chen GK, Vachon CM, Canzian F, Dunning A, Millikan RC, et al.  A common variant at the TERT-CLPTM1L locus is associated with estrogen receptor-negative breast cancer. <span><span class="ref-journal">Nature genetics. </span>2011;<span class="ref-vol">43</span>:1210–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3279120/">PMC free article</a>]</span> [<a href="/pubmed/22037553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=A+common+variant+at+the+TERT-CLPTM1L+locus+is+associated+with+estrogen+receptor-negative+breast+cancer&amp;author=CA+Haiman&amp;author=GK+Chen&amp;author=CM+Vachon&amp;author=F+Canzian&amp;author=A+Dunning&amp;volume=43&amp;publication_year=2011&amp;pages=1210-4&amp;pmid=22037553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Li J, Humphreys K, Heikkinen T, Aittomaki K, Blomqvist C, Pharoah PD, et al.  A combined analysis of genome-wide association studies in breast cancer. <span><span class="ref-journal">Breast Cancer Res Treat. </span>2011;<span class="ref-vol">126</span>:717–27.</span> [<a href="/pubmed/20872241" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+Cancer+Res+Treat&amp;title=A+combined+analysis+of+genome-wide+association+studies+in+breast+cancer&amp;author=J+Li&amp;author=K+Humphreys&amp;author=T+Heikkinen&amp;author=K+Aittomaki&amp;author=C+Blomqvist&amp;volume=126&amp;publication_year=2011&amp;pages=717-27&amp;pmid=20872241&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">American journal of human genetics. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation"><span></span>  <a href="http://www.kbioscience.co.uk/genotyping/genotyping_chemistry.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.kbioscience.co.uk/genotyping/genotyping_chemistry.html</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation"><span></span>  <a href="http://www.genabel.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genabel.org/</a></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Harri R, Bradburn M, Deeks J, Harbord R, Altman D, Sterne J. metan: fixed- and random-effects meta-analysis. <span><span class="ref-journal">The Stata Journal. </span>2008;<span class="ref-vol">8</span>:3–28.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Stata+Journal&amp;title=metan:+fixed-+and+random-effects+meta-analysis&amp;author=R+Harri&amp;author=M+Bradburn&amp;author=J+Deeks&amp;author=R+Harbord&amp;author=D+Altman&amp;volume=8&amp;publication_year=2008&amp;pages=3-28&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation"><span></span>  <a href="http://www.sph.umich.edu/csg/abecasis/MACH/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MACH/index.html</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-7&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Nica AC, Parts L, Glass D, Nisbet J, Barrett A, Sekowska M, et al.  The architecture of gene regulatory variation across multiple human tissues: the MuTHER study. <span><span class="ref-journal">PLoS genetics. </span>2011;<span class="ref-vol">7</span>:e1002003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3033383/">PMC free article</a>]</span> [<a href="/pubmed/21304890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=The+architecture+of+gene+regulatory+variation+across+multiple+human+tissues:+the+MuTHER+study&amp;author=AC+Nica&amp;author=L+Parts&amp;author=D+Glass&amp;author=J+Nisbet&amp;author=A+Barrett&amp;volume=7&amp;publication_year=2011&amp;pages=e1002003&amp;pmid=21304890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Cartharius K, Frech K, Grote K, Klocke B, Haltmeier M, Klingenhoff A, et al.  MatInspector and beyond: promoter analysis based on transcription factor binding sites. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:2933–42.</span> [<a href="/pubmed/15860560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MatInspector+and+beyond:+promoter+analysis+based+on+transcription+factor+binding+sites&amp;author=K+Cartharius&amp;author=K+Frech&amp;author=K+Grote&amp;author=B+Klocke&amp;author=M+Haltmeier&amp;volume=21&amp;publication_year=2005&amp;pages=2933-42&amp;pmid=15860560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Draheim KM, Chen HB, Tao Q, Moore N, Roche M, Lyle S. ARRDC3 suppresses breast cancer progression by negatively regulating integrin beta4. <span><span class="ref-journal">Oncogene. </span>2010;<span class="ref-vol">29</span>:5032–47.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997682/">PMC free article</a>]</span> [<a href="/pubmed/20603614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=ARRDC3+suppresses+breast+cancer+progression+by+negatively+regulating+integrin+beta4&amp;author=KM+Draheim&amp;author=HB+Chen&amp;author=Q+Tao&amp;author=N+Moore&amp;author=M+Roche&amp;volume=29&amp;publication_year=2010&amp;pages=5032-47&amp;pmid=20603614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Adelaide J, Finetti P, Bekhouche I, Repellini L, Geneix J, Sircoulomb F, et al.  Integrated profiling of basal and luminal breast cancers. <span><span class="ref-journal">Cancer research. </span>2007;<span class="ref-vol">67</span>:11565–75.</span> [<a href="/pubmed/18089785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+research&amp;title=Integrated+profiling+of+basal+and+luminal+breast+cancers&amp;author=J+Adelaide&amp;author=P+Finetti&amp;author=I+Bekhouche&amp;author=L+Repellini&amp;author=J+Geneix&amp;volume=67&amp;publication_year=2007&amp;pages=11565-75&amp;pmid=18089785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Huang CN, Huang SP, Pao JB, Chang TY, Lan YH, Lu TL, et al.  Genetic polymorphisms in androgen receptor-binding sites predict survival in prostate cancer patients receiving androgen-deprivation therapy. <span><span class="ref-journal">Ann Oncol. </span>2012;<span class="ref-vol">23</span>:707–13.</span> [<a href="/pubmed/21652578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Oncol&amp;title=Genetic+polymorphisms+in+androgen+receptor-binding+sites+predict+survival+in+prostate+cancer+patients+receiving+androgen-deprivation+therapy&amp;author=CN+Huang&amp;author=SP+Huang&amp;author=JB+Pao&amp;author=TY+Chang&amp;author=YH+Lan&amp;volume=23&amp;publication_year=2012&amp;pages=707-13&amp;pmid=21652578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Lim E, Wu D, Pal B, Bouras T, Asselin-Labat ML, Vaillant F, et al.  Transcriptome analyses of mouse and human mammary cell subpopulations reveal multiple conserved genes and pathways. <span><span class="ref-journal">Breast cancer research : BCR. </span>2010;<span class="ref-vol">12</span>:R21.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2879567/">PMC free article</a>]</span> [<a href="/pubmed/20346151" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+cancer+research+:+BCR&amp;title=Transcriptome+analyses+of+mouse+and+human+mammary+cell+subpopulations+reveal+multiple+conserved+genes+and+pathways&amp;author=E+Lim&amp;author=D+Wu&amp;author=B+Pal&amp;author=T+Bouras&amp;author=ML+Asselin-Labat&amp;volume=12&amp;publication_year=2010&amp;pages=R21&amp;pmid=20346151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Pallavi SK, Ho DM, Hicks C, Miele L, Artavanis-Tsakonas S. Notch and Mef2 synergize to promote proliferation and metastasis through JNK signal activation in Drosophila. <span><span class="ref-journal">EMBO J. </span>2012;<span class="ref-vol">31</span>:2895–907.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3395089/">PMC free article</a>]</span> [<a href="/pubmed/22580825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Notch+and+Mef2+synergize+to+promote+proliferation+and+metastasis+through+JNK+signal+activation+in+Drosophila&amp;author=SK+Pallavi&amp;author=DM+Ho&amp;author=C+Hicks&amp;author=L+Miele&amp;author=S+Artavanis-Tsakonas&amp;volume=31&amp;publication_year=2012&amp;pages=2895-907&amp;pmid=22580825&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Gamazon ER, Zhang W, Konkashbaev A, Duan S, Kistner EO, Nicolae DL, et al.  SCAN: SNP and copy number annotation. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:259–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852202/">PMC free article</a>]</span> [<a href="/pubmed/19933162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SCAN:+SNP+and+copy+number+annotation&amp;author=ER+Gamazon&amp;author=W+Zhang&amp;author=A+Konkashbaev&amp;author=S+Duan&amp;author=EO+Kistner&amp;volume=26&amp;publication_year=2010&amp;pages=259-62&amp;pmid=19933162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Magnani L, Ballantyne EB, Zhang X, Lupien M. PBX1 genomic pioneer function drives ERalpha signaling underlying progression in breast cancer. <span><span class="ref-journal">PLoS genetics. </span>2011;<span class="ref-vol">7</span>:e1002368.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3219601/">PMC free article</a>]</span> [<a href="/pubmed/22125492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=PBX1+genomic+pioneer+function+drives+ERalpha+signaling+underlying+progression+in+breast+cancer&amp;author=L+Magnani&amp;author=EB+Ballantyne&amp;author=X+Zhang&amp;author=M+Lupien&amp;volume=7&amp;publication_year=2011&amp;pages=e1002368&amp;pmid=22125492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Xiong G, Wang C, Evers BM, Zhou BP, Xu R. RORalpha suppresses breast tumor invasion by inducing SEMA3F expression. <span><span class="ref-journal">Cancer research. </span>2012;<span class="ref-vol">72</span>:1728–39.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3319846/">PMC free article</a>]</span> [<a href="/pubmed/22350413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+research&amp;title=RORalpha+suppresses+breast+tumor+invasion+by+inducing+SEMA3F+expression&amp;author=G+Xiong&amp;author=C+Wang&amp;author=BM+Evers&amp;author=BP+Zhou&amp;author=R+Xu&amp;volume=72&amp;publication_year=2012&amp;pages=1728-39&amp;pmid=22350413&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Forman BM, Chen J, Blumberg B, Kliewer SA, Henshaw R, Ong ES, et al.  Cross-talk among ROR alpha 1 and the Rev-erb family of orphan nuclear receptors. <span><span class="ref-journal">Mol Endocrinol. </span>1994;<span class="ref-vol">8</span>:1253–61.</span> [<a href="/pubmed/7838158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Endocrinol&amp;title=Cross-talk+among+ROR+alpha+1+and+the+Rev-erb+family+of+orphan+nuclear+receptors&amp;author=BM+Forman&amp;author=J+Chen&amp;author=B+Blumberg&amp;author=SA+Kliewer&amp;author=R+Henshaw&amp;volume=8&amp;publication_year=1994&amp;pages=1253-61&amp;pmid=7838158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Ramesh G, Berg A, Jayakumar C. Plasma netrin-1 is a diagnostic biomarker of human cancers. <span><span class="ref-journal">Biomarkers. </span>2011;<span class="ref-vol">16</span>:172–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3143477/">PMC free article</a>]</span> [<a href="/pubmed/21303223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biomarkers&amp;title=Plasma+netrin-1+is+a+diagnostic+biomarker+of+human+cancers&amp;author=G+Ramesh&amp;author=A+Berg&amp;author=C+Jayakumar&amp;volume=16&amp;publication_year=2011&amp;pages=172-80&amp;pmid=21303223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3601979&amp;issue-id=220813&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140376133399040title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Patients, Materials and Methods</a></li><li><a href="#S14title">Results</a></li><li><a href="#S21title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S22title">Acknowledgments</a></li><li><a href="#idm140376133262352title">Footnotes</a></li><li><a href="#idm140376133261104title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3601979/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3601979/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3601979/pdf/emss-51101.pdf">PDF (549K)</a></li> | <li><a href="#" data-citationid="PMC3601979" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3601979%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3601979%2F&amp;text=Identification%20of%20inherited%20genetic%20variations%20influencing%20prognosis%20in%20early%20onset%20breast%20cancer" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3601979%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23319801" data-aiid="3601979" data-aid="3601979" data-iid="220813" data-domainid="358" data-domain="wtpa" data-accid="PMC3601979" data-md5="af4982364cdca7761d7fe4c7e7124841"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:10:26-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D0B8DDD54C7D10000000007E70541&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3601979%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3601979/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>