<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3832790/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of 23 new prostate cancer susceptibility loci using the iCOGS custom genotyping array</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3832790/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Identification of 23 new prostate cancer susceptibility loci using the iCOGS custom genotyping array"><meta name="citation_authors" content="Rosalind A Eeles, Ali Amin Al Olama, Sara Benlloch, Edward J Saunders, Daniel A Leongamornlert, Malgorzata Tymrakiewicz, Maya Ghoussaini, Craig Luccarini, Joe Dennis, Sarah Jugurnauth-Little, Tokhir Dadaev, David E Neal, Freddie C Hamdy, Jenny L Donovan, Ken Muir, Graham G Giles, Gianluca Severi, Fredrik Wiklund, Henrik Gronberg, Christopher A Haiman, Fredrick Schumacher, Brian Henderson, Loic Le Marchand, Sara Lindstrom, Peter Kraft, David J Hunter, Susan Gapstur, Stephen J Chanock, Sonja I Berndt, Demetrius Albanes, Gerald Andriole, Johanna Schleutker, Maren Weischer, Federico Canzian, Elio Riboli, Tim J Key, Ruth Travis, Daniele Campa, Sue A Ingles, Esther M John, Richard B Hayes, Paul DP Pharoah, Nora Pashayan, Kay-Tee Khaw, Janet Stanford, Elaine A Ostrander, Lisa B Signorello, Stephen N Thibodeau, Dan Schaid, Christiane Maier, Walther Vogel, Adam S Kibel, Cezary Cybulski, Jan Lubinski,  Cannon-Albright, Hermann Brenner, Jong Y Park, Radka Kaneva, Jyotsna Batra, Amanda B Spurdle, Judith A Clements, Manuel R Teixeira, Ed Dicks, Andrew Lee, Alison Dunning, Caroline Baynes, Don Conroy, Melanie J Maranian, Shahana Ahmed, Koveela Govindasami, Michelle Guy, Rosemary A Wilkinson, Emma J Sawyer, Angela Morgan, David P Dearnaley, Alan Horwich, Robert A Huddart, Vincent S Khoo, Christopher C Parker, Nicholas J Van As, J Woodhouse, Alan Thompson, Tim Dudderidge, Chris Ogden, Colin Cooper, Artitaya Lophatananon, Angela Cox, Melissa Southey, John L Hopper, Dallas R English, Markus Aly, Jan Adolfsson, Jiangfeng Xu, Siqun Zheng, Meredith Yeager, Rudolf Kaaks, W Ryan Diver, Mia M Gaudet, Mariana Stern, Roman Corral, Amit D Joshi, Ahva Shahabi, Tiina Wahlfors, Teuvo J Tammela, Anssi Auvinen, Jarmo Virtamo, Peter Klarskov, Børge G Nordestgaard, Andreas Røder, Sune F Nielsen, Stig E Bojesen, Afshan Siddiq, Liesel FitzGerald, Suzanne Kolb, Erika Kwon, Danielle Karyadi, William J Blot, Wei Zheng, Qiuyin Cai, Shannon K McDonnell, Antje Rinckleb, Bettina Drake, Graham Colditz, Dominika Wokolorczyk, Robert A Stephenson, Craig Teerlink, Heiko Muller, Dietrich Rothenbacher, Thomas A Sellers, Hui-Yi Lin, Chavdar Slavov, Vanio Mitev, Felicity Lose, Srilakshmi Srinivasan, Sofia Maia, Paula Paulo, Ethan Lange, Kathleen A Cooney, Antonis Antoniou, Daniel Vincent, François Bacot,  Tessier, The COGS–Cancer Research UKGWAS–ELLIPSE(part of GAME-ON) Initiative, Australian Prostate Bioresource, UK Genetic Prostate Study Collaborators/British Association of Urological Surgeons’ Section of Oncology, UK ProtecT(Prostate testing for cancer and Treatment) Collaborators, PRACTICAL(Prostate Association Group to Investigate Cancer-Associated Alterations in the Genome) Consortium, Zsofia Kote-Jarai, Douglas F Easton"><meta name="citation_date" content="April 2013"><meta name="citation_issue" content="4"><meta name="citation_volume" content="45"><meta name="citation_doi" content="10.1038/ng.2560"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3832790/?report=abstract"><meta name="citation_pmid" content="23535732"><meta name="DC.Title" content="Identification of 23 new prostate cancer susceptibility loci using the iCOGS custom genotyping array"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Rosalind A Eeles"><meta name="DC.Contributor" content="Ali Amin Al Olama"><meta name="DC.Contributor" content="Sara Benlloch"><meta name="DC.Contributor" content="Edward J Saunders"><meta name="DC.Contributor" content="Daniel A Leongamornlert"><meta name="DC.Contributor" content="Malgorzata Tymrakiewicz"><meta name="DC.Contributor" content="Maya Ghoussaini"><meta name="DC.Contributor" content="Craig Luccarini"><meta name="DC.Contributor" content="Joe Dennis"><meta name="DC.Contributor" content="Sarah Jugurnauth-Little"><meta name="DC.Contributor" content="Tokhir Dadaev"><meta name="DC.Contributor" content="David E Neal"><meta name="DC.Contributor" content="Freddie C Hamdy"><meta name="DC.Contributor" content="Jenny L Donovan"><meta name="DC.Contributor" content="Ken Muir"><meta name="DC.Contributor" content="Graham G Giles"><meta name="DC.Contributor" content="Gianluca Severi"><meta name="DC.Contributor" content="Fredrik Wiklund"><meta name="DC.Contributor" content="Henrik Gronberg"><meta name="DC.Contributor" content="Christopher A Haiman"><meta name="DC.Contributor" content="Fredrick Schumacher"><meta name="DC.Contributor" content="Brian Henderson"><meta name="DC.Contributor" content="Loic Le Marchand"><meta name="DC.Contributor" content="Sara Lindstrom"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="David J Hunter"><meta name="DC.Contributor" content="Susan Gapstur"><meta name="DC.Contributor" content="Stephen J Chanock"><meta name="DC.Contributor" content="Sonja I Berndt"><meta name="DC.Contributor" content="Demetrius Albanes"><meta name="DC.Contributor" content="Gerald Andriole"><meta name="DC.Contributor" content="Johanna Schleutker"><meta name="DC.Contributor" content="Maren Weischer"><meta name="DC.Contributor" content="Federico Canzian"><meta name="DC.Contributor" content="Elio Riboli"><meta name="DC.Contributor" content="Tim J Key"><meta name="DC.Contributor" content="Ruth Travis"><meta name="DC.Contributor" content="Daniele Campa"><meta name="DC.Contributor" content="Sue A Ingles"><meta name="DC.Contributor" content="Esther M John"><meta name="DC.Contributor" content="Richard B Hayes"><meta name="DC.Contributor" content="Paul DP Pharoah"><meta name="DC.Contributor" content="Nora Pashayan"><meta name="DC.Contributor" content="Kay-Tee Khaw"><meta name="DC.Contributor" content="Janet Stanford"><meta name="DC.Contributor" content="Elaine A Ostrander"><meta name="DC.Contributor" content="Lisa B Signorello"><meta name="DC.Contributor" content="Stephen N Thibodeau"><meta name="DC.Contributor" content="Dan Schaid"><meta name="DC.Contributor" content="Christiane Maier"><meta name="DC.Contributor" content="Walther Vogel"><meta name="DC.Contributor" content="Adam S Kibel"><meta name="DC.Contributor" content="Cezary Cybulski"><meta name="DC.Contributor" content="Jan Lubinski"><meta name="DC.Contributor" content=" Cannon-Albright"><meta name="DC.Contributor" content="Hermann Brenner"><meta name="DC.Contributor" content="Jong Y Park"><meta name="DC.Contributor" content="Radka Kaneva"><meta name="DC.Contributor" content="Jyotsna Batra"><meta name="DC.Contributor" content="Amanda B Spurdle"><meta name="DC.Contributor" content="Judith A Clements"><meta name="DC.Contributor" content="Manuel R Teixeira"><meta name="DC.Contributor" content="Ed Dicks"><meta name="DC.Contributor" content="Andrew Lee"><meta name="DC.Contributor" content="Alison Dunning"><meta name="DC.Contributor" content="Caroline Baynes"><meta name="DC.Contributor" content="Don Conroy"><meta name="DC.Contributor" content="Melanie J Maranian"><meta name="DC.Contributor" content="Shahana Ahmed"><meta name="DC.Contributor" content="Koveela Govindasami"><meta name="DC.Contributor" content="Michelle Guy"><meta name="DC.Contributor" content="Rosemary A Wilkinson"><meta name="DC.Contributor" content="Emma J Sawyer"><meta name="DC.Contributor" content="Angela Morgan"><meta name="DC.Contributor" content="David P Dearnaley"><meta name="DC.Contributor" content="Alan Horwich"><meta name="DC.Contributor" content="Robert A Huddart"><meta name="DC.Contributor" content="Vincent S Khoo"><meta name="DC.Contributor" content="Christopher C Parker"><meta name="DC.Contributor" content="Nicholas J Van As"><meta name="DC.Contributor" content="J Woodhouse"><meta name="DC.Contributor" content="Alan Thompson"><meta name="DC.Contributor" content="Tim Dudderidge"><meta name="DC.Contributor" content="Chris Ogden"><meta name="DC.Contributor" content="Colin Cooper"><meta name="DC.Contributor" content="Artitaya Lophatananon"><meta name="DC.Contributor" content="Angela Cox"><meta name="DC.Contributor" content="Melissa Southey"><meta name="DC.Contributor" content="John L Hopper"><meta name="DC.Contributor" content="Dallas R English"><meta name="DC.Contributor" content="Markus Aly"><meta name="DC.Contributor" content="Jan Adolfsson"><meta name="DC.Contributor" content="Jiangfeng Xu"><meta name="DC.Contributor" content="Siqun Zheng"><meta name="DC.Contributor" content="Meredith Yeager"><meta name="DC.Contributor" content="Rudolf Kaaks"><meta name="DC.Contributor" content="W Ryan Diver"><meta name="DC.Contributor" content="Mia M Gaudet"><meta name="DC.Contributor" content="Mariana Stern"><meta name="DC.Contributor" content="Roman Corral"><meta name="DC.Contributor" content="Amit D Joshi"><meta name="DC.Contributor" content="Ahva Shahabi"><meta name="DC.Contributor" content="Tiina Wahlfors"><meta name="DC.Contributor" content="Teuvo J Tammela"><meta name="DC.Contributor" content="Anssi Auvinen"><meta name="DC.Contributor" content="Jarmo Virtamo"><meta name="DC.Contributor" content="Peter Klarskov"><meta name="DC.Contributor" content="Børge G Nordestgaard"><meta name="DC.Contributor" content="Andreas Røder"><meta name="DC.Contributor" content="Sune F Nielsen"><meta name="DC.Contributor" content="Stig E Bojesen"><meta name="DC.Contributor" content="Afshan Siddiq"><meta name="DC.Contributor" content="Liesel FitzGerald"><meta name="DC.Contributor" content="Suzanne Kolb"><meta name="DC.Contributor" content="Erika Kwon"><meta name="DC.Contributor" content="Danielle Karyadi"><meta name="DC.Contributor" content="William J Blot"><meta name="DC.Contributor" content="Wei Zheng"><meta name="DC.Contributor" content="Qiuyin Cai"><meta name="DC.Contributor" content="Shannon K McDonnell"><meta name="DC.Contributor" content="Antje Rinckleb"><meta name="DC.Contributor" content="Bettina Drake"><meta name="DC.Contributor" content="Graham Colditz"><meta name="DC.Contributor" content="Dominika Wokolorczyk"><meta name="DC.Contributor" content="Robert A Stephenson"><meta name="DC.Contributor" content="Craig Teerlink"><meta name="DC.Contributor" content="Heiko Muller"><meta name="DC.Contributor" content="Dietrich Rothenbacher"><meta name="DC.Contributor" content="Thomas A Sellers"><meta name="DC.Contributor" content="Hui-Yi Lin"><meta name="DC.Contributor" content="Chavdar Slavov"><meta name="DC.Contributor" content="Vanio Mitev"><meta name="DC.Contributor" content="Felicity Lose"><meta name="DC.Contributor" content="Srilakshmi Srinivasan"><meta name="DC.Contributor" content="Sofia Maia"><meta name="DC.Contributor" content="Paula Paulo"><meta name="DC.Contributor" content="Ethan Lange"><meta name="DC.Contributor" content="Kathleen A Cooney"><meta name="DC.Contributor" content="Antonis Antoniou"><meta name="DC.Contributor" content="Daniel Vincent"><meta name="DC.Contributor" content="François Bacot"><meta name="DC.Contributor" content=" Tessier"><meta name="DC.Contributor" content="The COGS–Cancer Research UKGWAS–ELLIPSE(part of GAME-ON) Initiative"><meta name="DC.Contributor" content="Australian Prostate Bioresource"><meta name="DC.Contributor" content="UK Genetic Prostate Study Collaborators/British Association of Urological Surgeons’ Section of Oncology"><meta name="DC.Contributor" content="UK ProtecT(Prostate testing for cancer and Treatment) Collaborators"><meta name="DC.Contributor" content="PRACTICAL(Prostate Association Group to Investigate Cancer-Associated Alterations in the Genome) Consortium"><meta name="DC.Contributor" content="Zsofia Kote-Jarai"><meta name="DC.Contributor" content="Douglas F Easton"><meta name="DC.Date" content="2013 Apr"><meta name="DC.Identifier" content="10.1038/ng.2560"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of 23 new prostate cancer susceptibility loci using the iCOGS custom genotyping array"><meta property="og:type" content="article"><meta property="og:description" content="Prostate cancer is the most frequently diagnosed cancer in males in developed countries. To identify common prostate cancer susceptibility alleles, we genotyped 211,155 SNPs on a custom Illumina array (iCOGS) in blood DNA from 25,074 prostate cancer cases ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3832790/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3832790/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3832790/pdf/nihms498644.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE884665DD5534410000000007920523.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832790%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3832790</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2013 Nov 19.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23535732" target="pmc_ext" ref="reftype=publisher&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2013 Apr; 45(4): 10.1038/ng.2560. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.2560" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.2560</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3832790</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS498644</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23535732">23535732</a></div></div></div></div><h1 class="content-title">Identification of 23 new prostate cancer susceptibility loci using the iCOGS custom genotyping array</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Eeles%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322460576" co-class="co-affbox">Rosalind A Eeles</a>,<sup>1,</sup><sup>2,</sup><sup>74</sup> <a href="/pubmed/?term=Olama%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822325323408" co-class="co-affbox">Ali Amin Al Olama</a>,<sup>3,</sup><sup>74</sup> <a href="/pubmed/?term=Benlloch%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322457440" co-class="co-affbox">Sara Benlloch</a>,<sup>3,</sup><sup>73</sup> <a href="/pubmed/?term=Saunders%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322455056" co-class="co-affbox">Edward J Saunders</a>,<sup>1,</sup><sup>73</sup> <a href="/pubmed/?term=Leongamornlert%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322452336" co-class="co-affbox">Daniel A Leongamornlert</a>,<sup>1,</sup><sup>73</sup> <a href="/pubmed/?term=Tymrakiewicz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320160016" co-class="co-affbox">Malgorzata Tymrakiewicz</a>,<sup>1,</sup><sup>73</sup> <a href="/pubmed/?term=Ghoussaini%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320157296" co-class="co-affbox">Maya Ghoussaini</a>,<sup>3,</sup><sup>73</sup> <a href="/pubmed/?term=Luccarini%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320154576" co-class="co-affbox">Craig Luccarini</a>,<sup>3,</sup><sup>73</sup> <a href="/pubmed/?term=Dennis%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320151856" co-class="co-affbox">Joe Dennis</a>,<sup>3,</sup><sup>73</sup> <a href="/pubmed/?term=Jugurnauth-Little%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320149136" co-class="co-affbox">Sarah Jugurnauth-Little</a>,<sup>1,</sup><sup>73</sup> <a href="/pubmed/?term=Dadaev%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324846688" co-class="co-affbox">Tokhir Dadaev</a>,<sup>1,</sup><sup>73</sup> <a href="/pubmed/?term=Neal%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324843968" co-class="co-affbox">David E Neal</a>,<sup>4,</sup><sup>5,</sup><sup>73</sup> <a href="/pubmed/?term=Hamdy%20FC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324840512" co-class="co-affbox">Freddie C Hamdy</a>,<sup>6,</sup><sup>7,</sup><sup>73</sup> <a href="/pubmed/?term=Donovan%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324837056" co-class="co-affbox">Jenny L Donovan</a>,<sup>8,</sup><sup>73</sup> <a href="/pubmed/?term=Muir%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324834336" co-class="co-affbox">Ken Muir</a>,<sup>9,</sup><sup>10,</sup><sup>73</sup> <a href="/pubmed/?term=Giles%20GG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320114576" co-class="co-affbox">Graham G Giles</a>,<sup>11,</sup><sup>12,</sup><sup>73</sup> <a href="/pubmed/?term=Severi%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320111120" co-class="co-affbox">Gianluca Severi</a>,<sup>11,</sup><sup>12,</sup><sup>73</sup> <a href="/pubmed/?term=Wiklund%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320107664" co-class="co-affbox">Fredrik Wiklund</a>,<sup>13,</sup><sup>73</sup> <a href="/pubmed/?term=Gronberg%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320104944" co-class="co-affbox">Henrik Gronberg</a>,<sup>13,</sup><sup>73</sup> <a href="/pubmed/?term=Haiman%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324271056" co-class="co-affbox">Christopher A Haiman</a>,<sup>14,</sup><sup>73</sup> <a href="/pubmed/?term=Schumacher%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324268336" co-class="co-affbox">Fredrick Schumacher</a>,<sup>14,</sup><sup>73</sup> <a href="/pubmed/?term=Henderson%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324265616" co-class="co-affbox">Brian Henderson</a>,<sup>14,</sup><sup>73</sup> <a href="/pubmed/?term=Le%20Marchand%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324262896" co-class="co-affbox">Loic Le Marchand</a>,<sup>15,</sup><sup>73</sup> <a href="/pubmed/?term=Lindstrom%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324260176" co-class="co-affbox">Sara Lindstrom</a>,<sup>16,</sup><sup>73</sup> <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322710256" co-class="co-affbox">Peter Kraft</a>,<sup>16,</sup><sup>73</sup> <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322707536" co-class="co-affbox">David J Hunter</a>,<sup>16,</sup><sup>73</sup> <a href="/pubmed/?term=Gapstur%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322704816" co-class="co-affbox">Susan Gapstur</a>,<sup>17,</sup><sup>73</sup> <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322702096" co-class="co-affbox">Stephen J Chanock</a>,<sup>18,</sup><sup>73</sup> <a href="/pubmed/?term=Berndt%20SI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322699376" co-class="co-affbox">Sonja I Berndt</a>,<sup>18,</sup><sup>73</sup> <a href="/pubmed/?term=Albanes%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322696656" co-class="co-affbox">Demetrius Albanes</a>,<sup>19,</sup><sup>73</sup> <a href="/pubmed/?term=Andriole%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321678576" co-class="co-affbox">Gerald Andriole</a>,<sup>20,</sup><sup>73</sup> <a href="/pubmed/?term=Schleutker%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321675856" co-class="co-affbox">Johanna Schleutker</a>,<sup>21,</sup><sup>22,</sup><sup>73</sup> <a href="/pubmed/?term=Weischer%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321672400" co-class="co-affbox">Maren Weischer</a>,<sup>23,</sup><sup>73</sup> <a href="/pubmed/?term=Canzian%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321669680" co-class="co-affbox">Federico Canzian</a>,<sup>24,</sup><sup>73</sup> <a href="/pubmed/?term=Riboli%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319143216" co-class="co-affbox">Elio Riboli</a>,<sup>25,</sup><sup>73</sup> <a href="/pubmed/?term=Key%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319140496" co-class="co-affbox">Tim J Key</a>,<sup>26,</sup><sup>73</sup> <a href="/pubmed/?term=Travis%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319137776" co-class="co-affbox">Ruth Travis</a>,<sup>26,</sup><sup>73</sup> <a href="/pubmed/?term=Campa%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319135056" co-class="co-affbox">Daniele Campa</a>,<sup>24,</sup><sup>73</sup> <a href="/pubmed/?term=Ingles%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319132336" co-class="co-affbox">Sue A Ingles</a>,<sup>14,</sup><sup>73</sup> <a href="/pubmed/?term=John%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319129616" co-class="co-affbox">Esther M John</a>,<sup>27,</sup><sup>28,</sup><sup>29,</sup><sup>73</sup> <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822328800400" co-class="co-affbox">Richard B Hayes</a>,<sup>30,</sup><sup>73</sup> <a href="/pubmed/?term=Pharoah%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822328797680" co-class="co-affbox">Paul DP Pharoah</a>,<sup>3,</sup><sup>73</sup> <a href="/pubmed/?term=Pashayan%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822328794960" co-class="co-affbox">Nora Pashayan</a>,<sup>3,</sup><sup>73</sup> <a href="/pubmed/?term=Khaw%20KT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822328792240" co-class="co-affbox">Kay-Tee Khaw</a>,<sup>31,</sup><sup>73</sup> <a href="/pubmed/?term=Stanford%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319302256" co-class="co-affbox">Janet Stanford</a>,<sup>32,</sup><sup>33,</sup><sup>73</sup> <a href="/pubmed/?term=Ostrander%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319298800" co-class="co-affbox">Elaine A Ostrander</a>,<sup>34,</sup><sup>73</sup> <a href="/pubmed/?term=Signorello%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319296080" co-class="co-affbox">Lisa B Signorello</a>,<sup>35,</sup><sup>36,</sup><sup>73</sup> <a href="/pubmed/?term=Thibodeau%20SN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319292624" co-class="co-affbox">Stephen N Thibodeau</a>,<sup>37,</sup><sup>73</sup> <a href="/pubmed/?term=Schaid%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319289904" co-class="co-affbox">Dan Schaid</a>,<sup>38,</sup><sup>73</sup> <a href="/pubmed/?term=Maier%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318464784" co-class="co-affbox">Christiane Maier</a>,<sup>39,</sup><sup>40,</sup><sup>73</sup> <a href="/pubmed/?term=Vogel%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318461328" co-class="co-affbox">Walther Vogel</a>,<sup>40,</sup><sup>73</sup> <a href="/pubmed/?term=Kibel%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318458608" co-class="co-affbox">Adam S Kibel</a>,<sup>41,</sup><sup>73</sup> <a href="/pubmed/?term=Cybulski%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318455888" co-class="co-affbox">Cezary Cybulski</a>,<sup>42,</sup><sup>73</sup> <a href="/pubmed/?term=Lubinski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318453168" co-class="co-affbox">Jan Lubinski</a>,<sup>42,</sup><sup>73</sup> <a href="/pubmed/?term=Cannon-Albright%20%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324063248" co-class="co-affbox"> Cannon-Albright</a>,<sup>43,</sup><sup>44,</sup><sup>73</sup> <a href="/pubmed/?term=Brenner%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324059920" co-class="co-affbox">Hermann Brenner</a>,<sup>45,</sup><sup>73</sup> <a href="/pubmed/?term=Park%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324057200" co-class="co-affbox">Jong Y Park</a>,<sup>46,</sup><sup>73</sup> <a href="/pubmed/?term=Kaneva%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324054480" co-class="co-affbox">Radka Kaneva</a>,<sup>47,</sup><sup>73</sup> <a href="/pubmed/?term=Batra%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822324051760" co-class="co-affbox">Jyotsna Batra</a>,<sup>48,</sup><sup>73</sup> <a href="/pubmed/?term=Spurdle%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321191408" co-class="co-affbox">Amanda B Spurdle</a>,<sup>49,</sup><sup>73</sup> <a href="/pubmed/?term=Clements%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321188688" co-class="co-affbox">Judith A Clements</a>,<sup>48,</sup><sup>73</sup> <a href="/pubmed/?term=Teixeira%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321185968" co-class="co-affbox">Manuel R Teixeira</a>,<sup>50,</sup><sup>51,</sup><sup>73</sup> <a href="/pubmed/?term=Dicks%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321182512" co-class="co-affbox">Ed Dicks</a>,<sup>3</sup> <a href="/pubmed/?term=Lee%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321180528" co-class="co-affbox">Andrew Lee</a>,<sup>3</sup> <a href="/pubmed/?term=Dunning%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321178544" co-class="co-affbox">Alison Dunning</a>,<sup>3</sup> <a href="/pubmed/?term=Baynes%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322887680" co-class="co-affbox">Caroline Baynes</a>,<sup>3</sup> <a href="/pubmed/?term=Conroy%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322885696" co-class="co-affbox">Don Conroy</a>,<sup>3</sup> <a href="/pubmed/?term=Maranian%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322883712" co-class="co-affbox">Melanie J Maranian</a>,<sup>3</sup> <a href="/pubmed/?term=Ahmed%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322881728" co-class="co-affbox">Shahana Ahmed</a>,<sup>3</sup> <a href="/pubmed/?term=Govindasami%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322879744" co-class="co-affbox">Koveela Govindasami</a>,<sup>1</sup> <a href="/pubmed/?term=Guy%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322877760" co-class="co-affbox">Michelle Guy</a>,<sup>1</sup> <a href="/pubmed/?term=Wilkinson%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322875776" co-class="co-affbox">Rosemary A Wilkinson</a>,<sup>1</sup> <a href="/pubmed/?term=Sawyer%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822325383472" co-class="co-affbox">Emma J Sawyer</a>,<sup>1</sup> <a href="/pubmed/?term=Morgan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822325381488" co-class="co-affbox">Angela Morgan</a>,<sup>1</sup> <a href="/pubmed/?term=Dearnaley%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822325379504" co-class="co-affbox">David P Dearnaley</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Horwich%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822325376784" co-class="co-affbox">Alan Horwich</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Huddart%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822325374064" co-class="co-affbox">Robert A Huddart</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Khoo%20VS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822325371344" co-class="co-affbox">Vincent S Khoo</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Parker%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318937808" co-class="co-affbox">Christopher C Parker</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Van%20As%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318935088" co-class="co-affbox">Nicholas J Van As</a>,<sup>2</sup> <a href="/pubmed/?term=Woodhouse%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318933104" co-class="co-affbox">J Woodhouse</a>,<sup>2</sup> <a href="/pubmed/?term=Thompson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318931120" co-class="co-affbox">Alan Thompson</a>,<sup>2</sup> <a href="/pubmed/?term=Dudderidge%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318929136" co-class="co-affbox">Tim Dudderidge</a>,<sup>2</sup> <a href="/pubmed/?term=Ogden%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318927152" co-class="co-affbox">Chris Ogden</a>,<sup>2</sup> <a href="/pubmed/?term=Cooper%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318925168" co-class="co-affbox">Colin Cooper</a>,<sup>1,</sup><sup>52</sup> <a href="/pubmed/?term=Lophatananon%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822323324016" co-class="co-affbox">Artitaya Lophatananon</a>,<sup>9</sup> <a href="/pubmed/?term=Cox%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822323322032" co-class="co-affbox">Angela Cox</a>,<sup>53</sup> <a href="/pubmed/?term=Southey%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822323320048" co-class="co-affbox">Melissa Southey</a>,<sup>54</sup> <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822323318064" co-class="co-affbox">John L Hopper</a>,<sup>12</sup> <a href="/pubmed/?term=English%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822323316080" co-class="co-affbox">Dallas R English</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Aly%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822323313360" co-class="co-affbox">Markus Aly</a>,<sup>13,</sup><sup>55</sup> <a href="/pubmed/?term=Adolfsson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320182464" co-class="co-affbox">Jan Adolfsson</a>,<sup>56</sup> <a href="/pubmed/?term=Xu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320180480" co-class="co-affbox">Jiangfeng Xu</a>,<sup>57</sup> <a href="/pubmed/?term=Zheng%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320178496" co-class="co-affbox">Siqun Zheng</a>,<sup>58</sup> <a href="/pubmed/?term=Yeager%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320176512" co-class="co-affbox">Meredith Yeager</a>,<sup>18,</sup><sup>59</sup> <a href="/pubmed/?term=Kaaks%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320173792" co-class="co-affbox">Rudolf Kaaks</a>,<sup>60</sup> <a href="/pubmed/?term=Diver%20WR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320171808" co-class="co-affbox">W Ryan Diver</a>,<sup>17</sup> <a href="/pubmed/?term=Gaudet%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822320169824" co-class="co-affbox">Mia M Gaudet</a>,<sup>17</sup> <a href="/pubmed/?term=Stern%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319456048" co-class="co-affbox">Mariana Stern</a>,<sup>14</sup> <a href="/pubmed/?term=Corral%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319454064" co-class="co-affbox">Roman Corral</a>,<sup>14</sup> <a href="/pubmed/?term=Joshi%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319452080" co-class="co-affbox">Amit D Joshi</a>,<sup>14</sup> <a href="/pubmed/?term=Shahabi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319450096" co-class="co-affbox">Ahva Shahabi</a>,<sup>14</sup> <a href="/pubmed/?term=Wahlfors%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319448112" co-class="co-affbox">Tiina Wahlfors</a>,<sup>21</sup> <a href="/pubmed/?term=Tammela%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319446128" co-class="co-affbox">Teuvo J Tammela</a>,<sup>61</sup> <a href="/pubmed/?term=Auvinen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319444144" co-class="co-affbox">Anssi Auvinen</a>,<sup>62</sup> <a href="/pubmed/?term=Virtamo%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319442160" co-class="co-affbox">Jarmo Virtamo</a>,<sup>63</sup> <a href="/pubmed/?term=Klarskov%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321163968" co-class="co-affbox">Peter Klarskov</a>,<sup>64</sup> <a href="/pubmed/?term=Nordestgaard%20BG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321161984" co-class="co-affbox">Børge G Nordestgaard</a>,<sup>23</sup> <a href="/pubmed/?term=R%26%23x000f8%3Bder%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321160000" co-class="co-affbox">Andreas Røder</a>,<sup>65</sup> <a href="/pubmed/?term=Nielsen%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321158016" co-class="co-affbox">Sune F Nielsen</a>,<sup>23</sup> <a href="/pubmed/?term=Bojesen%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321156032" co-class="co-affbox">Stig E Bojesen</a>,<sup>23</sup> <a href="/pubmed/?term=Siddiq%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321154048" co-class="co-affbox">Afshan Siddiq</a>,<sup>66</sup> <a href="/pubmed/?term=FitzGerald%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822321152064" co-class="co-affbox">Liesel FitzGerald</a>,<sup>32</sup> <a href="/pubmed/?term=Kolb%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322199184" co-class="co-affbox">Suzanne Kolb</a>,<sup>32</sup> <a href="/pubmed/?term=Kwon%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322197200" co-class="co-affbox">Erika Kwon</a>,<sup>34</sup> <a href="/pubmed/?term=Karyadi%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322195168" co-class="co-affbox">Danielle Karyadi</a>,<sup>34</sup> <a href="/pubmed/?term=Blot%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322193136" co-class="co-affbox">William J Blot</a>,<sup>35,</sup><sup>36</sup> <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322190320" co-class="co-affbox">Wei Zheng</a>,<sup>36</sup> <a href="/pubmed/?term=Cai%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322188288" co-class="co-affbox">Qiuyin Cai</a>,<sup>36</sup> <a href="/pubmed/?term=McDonnell%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322186256" co-class="co-affbox">Shannon K McDonnell</a>,<sup>37</sup> <a href="/pubmed/?term=Rinckleb%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318833664" co-class="co-affbox">Antje Rinckleb</a>,<sup>39,</sup><sup>40</sup> <a href="/pubmed/?term=Drake%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318830848" co-class="co-affbox">Bettina Drake</a>,<sup>20</sup> <a href="/pubmed/?term=Colditz%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318828816" co-class="co-affbox">Graham Colditz</a>,<sup>20</sup> <a href="/pubmed/?term=Wokolorczyk%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318826784" co-class="co-affbox">Dominika Wokolorczyk</a>,<sup>42</sup> <a href="/pubmed/?term=Stephenson%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318824752" co-class="co-affbox">Robert A Stephenson</a>,<sup>44,</sup><sup>67</sup> <a href="/pubmed/?term=Teerlink%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318821840" co-class="co-affbox">Craig Teerlink</a>,<sup>43</sup> <a href="/pubmed/?term=Muller%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822318819808" co-class="co-affbox">Heiko Muller</a>,<sup>45</sup> <a href="/pubmed/?term=Rothenbacher%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322325424" co-class="co-affbox">Dietrich Rothenbacher</a>,<sup>45</sup> <a href="/pubmed/?term=Sellers%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322323392" co-class="co-affbox">Thomas A Sellers</a>,<sup>46</sup> <a href="/pubmed/?term=Lin%20HY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322321360" co-class="co-affbox">Hui-Yi Lin</a>,<sup>46</sup> <a href="/pubmed/?term=Slavov%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322319328" co-class="co-affbox">Chavdar Slavov</a>,<sup>68</sup> <a href="/pubmed/?term=Mitev%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322317296" co-class="co-affbox">Vanio Mitev</a>,<sup>47</sup> <a href="/pubmed/?term=Lose%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322315264" co-class="co-affbox">Felicity Lose</a>,<sup>49</sup> <a href="/pubmed/?term=Srinivasan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322313232" co-class="co-affbox">Srilakshmi Srinivasan</a>,<sup>48</sup> <a href="/pubmed/?term=Maia%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822322311200" co-class="co-affbox">Sofia Maia</a>,<sup>50,</sup><sup>51</sup> <a href="/pubmed/?term=Paulo%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822329095584" co-class="co-affbox">Paula Paulo</a>,<sup>50,</sup><sup>51</sup> <a href="/pubmed/?term=Lange%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822329092768" co-class="co-affbox">Ethan Lange</a>,<sup>69</sup> <a href="/pubmed/?term=Cooney%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822329090736" co-class="co-affbox">Kathleen A Cooney</a>,<sup>70</sup> <a href="/pubmed/?term=Antoniou%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822329088704" co-class="co-affbox">Antonis Antoniou</a>,<sup>3</sup> <a href="/pubmed/?term=Vincent%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822329086672" co-class="co-affbox">Daniel Vincent</a>,<sup>71</sup> <a href="/pubmed/?term=Bacot%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822329084640" co-class="co-affbox">François Bacot</a>,<sup>71</sup> <a href="/pubmed/?term=Tessier%20%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319257328" co-class="co-affbox"> Tessier</a>,<sup>71</sup> The COGS–Cancer Research UKGWAS–ELLIPSE(part of GAME-ON) Initiative,<sup>72</sup> Australian Prostate Bioresource,<sup>72</sup> UK Genetic Prostate Study Collaborators/British Association of Urological Surgeons’ Section of Oncology,<sup>72</sup> UK ProtecT(Prostate testing for cancer and Treatment) Collaborators,<sup>72</sup> PRACTICAL(Prostate Association Group to Investigate Cancer-Associated Alterations in the Genome) Consortium,<sup>72</sup> <a href="/pubmed/?term=Kote-Jarai%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319248032" co-class="co-affbox">Zsofia Kote-Jarai</a>,<sup>1,</sup><sup>74</sup> and  <a href="/pubmed/?term=Easton%20DF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732" class="affpopup" co-rid="_co_idm139822319245216" co-class="co-affbox">Douglas F Easton</a><sup>3,</sup><sup>74</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139822322460576"><h3 class="no_margin">Rosalind A Eeles</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Eeles%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Rosalind A Eeles</a></div></div><div id="_co_idm139822325323408"><h3 class="no_margin">Ali Amin Al Olama</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Olama%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Ali Amin Al Olama</a></div></div><div id="_co_idm139822322457440"><h3 class="no_margin">Sara Benlloch</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Benlloch%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Sara Benlloch</a></div></div><div id="_co_idm139822322455056"><h3 class="no_margin">Edward J Saunders</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Saunders%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Edward J Saunders</a></div></div><div id="_co_idm139822322452336"><h3 class="no_margin">Daniel A Leongamornlert</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Leongamornlert%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Daniel A Leongamornlert</a></div></div><div id="_co_idm139822320160016"><h3 class="no_margin">Malgorzata Tymrakiewicz</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Tymrakiewicz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Malgorzata Tymrakiewicz</a></div></div><div id="_co_idm139822320157296"><h3 class="no_margin">Maya Ghoussaini</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Ghoussaini%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Maya Ghoussaini</a></div></div><div id="_co_idm139822320154576"><h3 class="no_margin">Craig Luccarini</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Luccarini%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Craig Luccarini</a></div></div><div id="_co_idm139822320151856"><h3 class="no_margin">Joe Dennis</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Dennis%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Joe Dennis</a></div></div><div id="_co_idm139822320149136"><h3 class="no_margin">Sarah Jugurnauth-Little</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Jugurnauth-Little%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Sarah Jugurnauth-Little</a></div></div><div id="_co_idm139822324846688"><h3 class="no_margin">Tokhir Dadaev</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Dadaev%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Tokhir Dadaev</a></div></div><div id="_co_idm139822324843968"><h3 class="no_margin">David E Neal</h3><p><sup>4</sup>Surgical Oncology (Uro-Oncology: S4), University of Cambridge, Addenbrooke’s Hospital, Cambridge, UK</p><p><sup>5</sup>Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Neal%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">David E Neal</a></div></div><div id="_co_idm139822324840512"><h3 class="no_margin">Freddie C Hamdy</h3><p><sup>6</sup>Nuffield Department of Surgical Sciences, University of Oxford, Oxford, UK</p><p><sup>7</sup>Faculty of Medical Science, University of Oxford, John Radcliffe Hospital, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Hamdy%20FC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Freddie C Hamdy</a></div></div><div id="_co_idm139822324837056"><h3 class="no_margin">Jenny L Donovan</h3><p><sup>8</sup>School of Social and Community Medicine, University of Bristol, Bristol, UK</p><div>Find articles by <a href="/pubmed/?term=Donovan%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Jenny L Donovan</a></div></div><div id="_co_idm139822324834336"><h3 class="no_margin">Ken Muir</h3><p><sup>9</sup>Warwick Medical School, University of Warwick, Coventry, UK</p><p><sup>10</sup>Institute of Population Health, University of Manchester, Manchester, UK</p><div>Find articles by <a href="/pubmed/?term=Muir%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Ken Muir</a></div></div><div id="_co_idm139822320114576"><h3 class="no_margin">Graham G Giles</h3><p><sup>11</sup>Cancer Epidemiology Centre, The Cancer Council Victoria, Carlton, Victoria, Australia</p><p><sup>12</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Giles%20GG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Graham G Giles</a></div></div><div id="_co_idm139822320111120"><h3 class="no_margin">Gianluca Severi</h3><p><sup>11</sup>Cancer Epidemiology Centre, The Cancer Council Victoria, Carlton, Victoria, Australia</p><p><sup>12</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Severi%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Gianluca Severi</a></div></div><div id="_co_idm139822320107664"><h3 class="no_margin">Fredrik Wiklund</h3><p><sup>13</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institute, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Wiklund%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Fredrik Wiklund</a></div></div><div id="_co_idm139822320104944"><h3 class="no_margin">Henrik Gronberg</h3><p><sup>13</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institute, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Gronberg%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Henrik Gronberg</a></div></div><div id="_co_idm139822324271056"><h3 class="no_margin">Christopher A Haiman</h3><p><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Haiman%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Christopher A Haiman</a></div></div><div id="_co_idm139822324268336"><h3 class="no_margin">Fredrick Schumacher</h3><p><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Schumacher%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Fredrick Schumacher</a></div></div><div id="_co_idm139822324265616"><h3 class="no_margin">Brian Henderson</h3><p><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Henderson%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Brian Henderson</a></div></div><div id="_co_idm139822324262896"><h3 class="no_margin">Loic Le Marchand</h3><p><sup>15</sup>University of Hawaii Cancer Center, Honolulu, Hawaii, USA</p><div>Find articles by <a href="/pubmed/?term=Le%20Marchand%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Loic Le Marchand</a></div></div><div id="_co_idm139822324260176"><h3 class="no_margin">Sara Lindstrom</h3><p><sup>16</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Pubic Health, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Lindstrom%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Sara Lindstrom</a></div></div><div id="_co_idm139822322710256"><h3 class="no_margin">Peter Kraft</h3><p><sup>16</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Pubic Health, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Peter Kraft</a></div></div><div id="_co_idm139822322707536"><h3 class="no_margin">David J Hunter</h3><p><sup>16</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Pubic Health, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">David J Hunter</a></div></div><div id="_co_idm139822322704816"><h3 class="no_margin">Susan Gapstur</h3><p><sup>17</sup>Epidemiology Research Program, American Cancer Society, Atlanta, Georgia, USA</p><div>Find articles by <a href="/pubmed/?term=Gapstur%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Susan Gapstur</a></div></div><div id="_co_idm139822322702096"><h3 class="no_margin">Stephen J Chanock</h3><p><sup>18</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, US National Institutes of Health (NIH), Bethesda, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Stephen J Chanock</a></div></div><div id="_co_idm139822322699376"><h3 class="no_margin">Sonja I Berndt</h3><p><sup>18</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, US National Institutes of Health (NIH), Bethesda, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Berndt%20SI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Sonja I Berndt</a></div></div><div id="_co_idm139822322696656"><h3 class="no_margin">Demetrius Albanes</h3><p><sup>19</sup>Nutritional Epidemiology Branch, National Cancer Institute, US NIH, Bethesda, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Albanes%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Demetrius Albanes</a></div></div><div id="_co_idm139822321678576"><h3 class="no_margin">Gerald Andriole</h3><p><sup>20</sup>Division of Urologic Surgery, Washington University School of Medicine, St. Louis, Missouri, USA</p><div>Find articles by <a href="/pubmed/?term=Andriole%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Gerald Andriole</a></div></div><div id="_co_idm139822321675856"><h3 class="no_margin">Johanna Schleutker</h3><p><sup>21</sup>Institute of Biomedical Technology (BioMediTech), University of Tampere and FimLab Laboratories, Tampere, Finland</p><p><sup>22</sup>Department of Medical Biochemistry and Genetics, University of Turku, Turku, Finland</p><div>Find articles by <a href="/pubmed/?term=Schleutker%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Johanna Schleutker</a></div></div><div id="_co_idm139822321672400"><h3 class="no_margin">Maren Weischer</h3><p><sup>23</sup>Department of Clinical Biochemistry, Herlev Hospital, Copenhagen University Hospital, Herlev, Denmark</p><div>Find articles by <a href="/pubmed/?term=Weischer%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Maren Weischer</a></div></div><div id="_co_idm139822321669680"><h3 class="no_margin">Federico Canzian</h3><p><sup>24</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Canzian%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Federico Canzian</a></div></div><div id="_co_idm139822319143216"><h3 class="no_margin">Elio Riboli</h3><p><sup>25</sup>Department of Epidemiology &amp; Biostatistics, School of Public Health, Imperial College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Riboli%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Elio Riboli</a></div></div><div id="_co_idm139822319140496"><h3 class="no_margin">Tim J Key</h3><p><sup>26</sup>Cancer Epidemiology Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Key%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Tim J Key</a></div></div><div id="_co_idm139822319137776"><h3 class="no_margin">Ruth Travis</h3><p><sup>26</sup>Cancer Epidemiology Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Travis%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Ruth Travis</a></div></div><div id="_co_idm139822319135056"><h3 class="no_margin">Daniele Campa</h3><p><sup>24</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Campa%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Daniele Campa</a></div></div><div id="_co_idm139822319132336"><h3 class="no_margin">Sue A Ingles</h3><p><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Ingles%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Sue A Ingles</a></div></div><div id="_co_idm139822319129616"><h3 class="no_margin">Esther M John</h3><p><sup>27</sup>Cancer Prevention Institute of California, Fremont, California, USA</p><p><sup>28</sup>Stanford Cancer Institute, Stanford University School of Medicine, Stanford, California, USA</p><p><sup>29</sup>Department of Health Research &amp; Policy, Division of Epidemiology, Stanford University School of Medicine, Stanford, California, USA</p><div>Find articles by <a href="/pubmed/?term=John%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Esther M John</a></div></div><div id="_co_idm139822328800400"><h3 class="no_margin">Richard B Hayes</h3><p><sup>30</sup>Department of Environmental Medicine, New York University (NYU) Langone Medical Center, Division of Epidemiology, NYU Cancer Institute, New York, New York, USA</p><div>Find articles by <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Richard B Hayes</a></div></div><div id="_co_idm139822328797680"><h3 class="no_margin">Paul DP Pharoah</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Pharoah%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Paul DP Pharoah</a></div></div><div id="_co_idm139822328794960"><h3 class="no_margin">Nora Pashayan</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Pashayan%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Nora Pashayan</a></div></div><div id="_co_idm139822328792240"><h3 class="no_margin">Kay-Tee Khaw</h3><p><sup>31</sup>Clinical Gerontology Unit, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Khaw%20KT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Kay-Tee Khaw</a></div></div><div id="_co_idm139822319302256"><h3 class="no_margin">Janet Stanford</h3><p><sup>32</sup>Fred Hutchinson Cancer Research Center, Division of Public Health Sciences, Seattle, Washington, USA</p><p><sup>33</sup>Department of Epidemiology, School of Public Health, University of Washington, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Stanford%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Janet Stanford</a></div></div><div id="_co_idm139822319298800"><h3 class="no_margin">Elaine A Ostrander</h3><p><sup>34</sup>National Human Genome Research Institute, US NIH, Bethesda, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Ostrander%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Elaine A Ostrander</a></div></div><div id="_co_idm139822319296080"><h3 class="no_margin">Lisa B Signorello</h3><p><sup>35</sup>International Epidemiology Institute, Rockville, Maryland, USA</p><p><sup>36</sup>Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Division of Epidemiology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Signorello%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Lisa B Signorello</a></div></div><div id="_co_idm139822319292624"><h3 class="no_margin">Stephen N Thibodeau</h3><p><sup>37</sup>Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Thibodeau%20SN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Stephen N Thibodeau</a></div></div><div id="_co_idm139822319289904"><h3 class="no_margin">Dan Schaid</h3><p><sup>38</sup>Division of Biomedical Statistics and Informatics, Department of Health Sciences Research, Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Schaid%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Dan Schaid</a></div></div><div id="_co_idm139822318464784"><h3 class="no_margin">Christiane Maier</h3><p><sup>39</sup>Department of Urology, University Hospital Ulm, Ulm, Germany</p><p><sup>40</sup>Institute of Human Genetics, University Hospital Ulm, Ulm, Germany</p><div>Find articles by <a href="/pubmed/?term=Maier%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Christiane Maier</a></div></div><div id="_co_idm139822318461328"><h3 class="no_margin">Walther Vogel</h3><p><sup>40</sup>Institute of Human Genetics, University Hospital Ulm, Ulm, Germany</p><div>Find articles by <a href="/pubmed/?term=Vogel%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Walther Vogel</a></div></div><div id="_co_idm139822318458608"><h3 class="no_margin">Adam S Kibel</h3><p><sup>41</sup>Division of Urologic Surgery, Brigham and Women’s Hospital, Dana-Farber Cancer Institute, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Kibel%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Adam S Kibel</a></div></div><div id="_co_idm139822318455888"><h3 class="no_margin">Cezary Cybulski</h3><p><sup>42</sup>Department of Genetics and Pathology, International Hereditary Cancer Center, Pomeranian Medical University, Szczecin, Poland</p><div>Find articles by <a href="/pubmed/?term=Cybulski%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Cezary Cybulski</a></div></div><div id="_co_idm139822318453168"><h3 class="no_margin">Jan Lubinski</h3><p><sup>42</sup>Department of Genetics and Pathology, International Hereditary Cancer Center, Pomeranian Medical University, Szczecin, Poland</p><div>Find articles by <a href="/pubmed/?term=Lubinski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Jan Lubinski</a></div></div><div id="_co_idm139822324063248"><h3 class="no_margin"> Cannon-Albright</h3><p><sup>43</sup>Division of Genetic Epidemiology, Department of Medicine, University of Utah School of Medicine, Salt Lake City, Utah, USA</p><p><sup>44</sup>George E. Wahlen Department of Veterans Affairs Medical Center, Salt Lake City, Utah, USA</p><div>Find articles by <a href="/pubmed/?term=Cannon-Albright%20%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732"> Cannon-Albright</a></div></div><div id="_co_idm139822324059920"><h3 class="no_margin">Hermann Brenner</h3><p><sup>45</sup>Division of Clinical Epidemiology and Aging Research, DKFZ, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Brenner%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Hermann Brenner</a></div></div><div id="_co_idm139822324057200"><h3 class="no_margin">Jong Y Park</h3><p><sup>46</sup>Division of Cancer Prevention and Control, H. Lee Moffitt Cancer Center, Tampa, Florida, USA</p><div>Find articles by <a href="/pubmed/?term=Park%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Jong Y Park</a></div></div><div id="_co_idm139822324054480"><h3 class="no_margin">Radka Kaneva</h3><p><sup>47</sup>Department of Medical Chemistry and Biochemistry, Molecular Medicine Center, Medical University of Sofia, Sofia, Bulgaria</p><div>Find articles by <a href="/pubmed/?term=Kaneva%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Radka Kaneva</a></div></div><div id="_co_idm139822324051760"><h3 class="no_margin">Jyotsna Batra</h3><p><sup>48</sup>Australian Prostate Cancer Research Centre, Queensland Institute of Health and Biomedical Innovation and School of Biomedical Science, Queensland University of Technology, Brisbane, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Batra%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Jyotsna Batra</a></div></div><div id="_co_idm139822321191408"><h3 class="no_margin">Amanda B Spurdle</h3><p><sup>49</sup>Molecular Cancer Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Spurdle%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Amanda B Spurdle</a></div></div><div id="_co_idm139822321188688"><h3 class="no_margin">Judith A Clements</h3><p><sup>48</sup>Australian Prostate Cancer Research Centre, Queensland Institute of Health and Biomedical Innovation and School of Biomedical Science, Queensland University of Technology, Brisbane, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Clements%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Judith A Clements</a></div></div><div id="_co_idm139822321185968"><h3 class="no_margin">Manuel R Teixeira</h3><p><sup>50</sup>Department of Genetics, Portuguese Oncology Institute, Porto, Portugal</p><p><sup>51</sup>Abel Salazar Biomedical Sciences Institute (ICBAS), Porto University, Porto, Portugal</p><div>Find articles by <a href="/pubmed/?term=Teixeira%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Manuel R Teixeira</a></div></div><div id="_co_idm139822321182512"><h3 class="no_margin">Ed Dicks</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Dicks%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Ed Dicks</a></div></div><div id="_co_idm139822321180528"><h3 class="no_margin">Andrew Lee</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Lee%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Andrew Lee</a></div></div><div id="_co_idm139822321178544"><h3 class="no_margin">Alison Dunning</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Dunning%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Alison Dunning</a></div></div><div id="_co_idm139822322887680"><h3 class="no_margin">Caroline Baynes</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Baynes%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Caroline Baynes</a></div></div><div id="_co_idm139822322885696"><h3 class="no_margin">Don Conroy</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Conroy%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Don Conroy</a></div></div><div id="_co_idm139822322883712"><h3 class="no_margin">Melanie J Maranian</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Maranian%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Melanie J Maranian</a></div></div><div id="_co_idm139822322881728"><h3 class="no_margin">Shahana Ahmed</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Ahmed%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Shahana Ahmed</a></div></div><div id="_co_idm139822322879744"><h3 class="no_margin">Koveela Govindasami</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Govindasami%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Koveela Govindasami</a></div></div><div id="_co_idm139822322877760"><h3 class="no_margin">Michelle Guy</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Guy%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Michelle Guy</a></div></div><div id="_co_idm139822322875776"><h3 class="no_margin">Rosemary A Wilkinson</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Wilkinson%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Rosemary A Wilkinson</a></div></div><div id="_co_idm139822325383472"><h3 class="no_margin">Emma J Sawyer</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Sawyer%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Emma J Sawyer</a></div></div><div id="_co_idm139822325381488"><h3 class="no_margin">Angela Morgan</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Morgan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Angela Morgan</a></div></div><div id="_co_idm139822325379504"><h3 class="no_margin">David P Dearnaley</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Dearnaley%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">David P Dearnaley</a></div></div><div id="_co_idm139822325376784"><h3 class="no_margin">Alan Horwich</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Horwich%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Alan Horwich</a></div></div><div id="_co_idm139822325374064"><h3 class="no_margin">Robert A Huddart</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Huddart%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Robert A Huddart</a></div></div><div id="_co_idm139822325371344"><h3 class="no_margin">Vincent S Khoo</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Khoo%20VS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Vincent S Khoo</a></div></div><div id="_co_idm139822318937808"><h3 class="no_margin">Christopher C Parker</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Parker%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Christopher C Parker</a></div></div><div id="_co_idm139822318935088"><h3 class="no_margin">Nicholas J Van As</h3><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Van%20As%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Nicholas J Van As</a></div></div><div id="_co_idm139822318933104"><h3 class="no_margin">J Woodhouse</h3><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Woodhouse%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">J Woodhouse</a></div></div><div id="_co_idm139822318931120"><h3 class="no_margin">Alan Thompson</h3><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Thompson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Alan Thompson</a></div></div><div id="_co_idm139822318929136"><h3 class="no_margin">Tim Dudderidge</h3><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Dudderidge%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Tim Dudderidge</a></div></div><div id="_co_idm139822318927152"><h3 class="no_margin">Chris Ogden</h3><p><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Ogden%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Chris Ogden</a></div></div><div id="_co_idm139822318925168"><h3 class="no_margin">Colin Cooper</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><p><sup>52</sup>School of Biological Sciences, University of East Anglia, Norwich, UK</p><div>Find articles by <a href="/pubmed/?term=Cooper%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Colin Cooper</a></div></div><div id="_co_idm139822323324016"><h3 class="no_margin">Artitaya Lophatananon</h3><p><sup>9</sup>Warwick Medical School, University of Warwick, Coventry, UK</p><div>Find articles by <a href="/pubmed/?term=Lophatananon%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Artitaya Lophatananon</a></div></div><div id="_co_idm139822323322032"><h3 class="no_margin">Angela Cox</h3><p><sup>53</sup>Cancer Research UK/Yorkshire Cancer Research Sheffield Cancer Research Centre, University of Sheffield, Sheffield, UK</p><div>Find articles by <a href="/pubmed/?term=Cox%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Angela Cox</a></div></div><div id="_co_idm139822323320048"><h3 class="no_margin">Melissa Southey</h3><p><sup>54</sup>Genetic Epidemiology Laboratory, Department of Pathology, The University of Melbourne, Parkville, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Southey%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Melissa Southey</a></div></div><div id="_co_idm139822323318064"><h3 class="no_margin">John L Hopper</h3><p><sup>12</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">John L Hopper</a></div></div><div id="_co_idm139822323316080"><h3 class="no_margin">Dallas R English</h3><p><sup>11</sup>Cancer Epidemiology Centre, The Cancer Council Victoria, Carlton, Victoria, Australia</p><p><sup>12</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=English%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Dallas R English</a></div></div><div id="_co_idm139822323313360"><h3 class="no_margin">Markus Aly</h3><p><sup>13</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institute, Stockholm, Sweden</p><p><sup>55</sup>Department of Clinical Sciences, Division of Urology, Danderyd Hospital, Karolinska Institute, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Aly%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Markus Aly</a></div></div><div id="_co_idm139822320182464"><h3 class="no_margin">Jan Adolfsson</h3><p><sup>56</sup>Regional Cancer Centre, Department of Clinical Science, Intervention and Technology (CLINTEC), Karolinska Institute, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Adolfsson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Jan Adolfsson</a></div></div><div id="_co_idm139822320180480"><h3 class="no_margin">Jiangfeng Xu</h3><p><sup>57</sup>Wake Forest University School of Medicine, Winston-Salem, North Carolina, USA</p><div>Find articles by <a href="/pubmed/?term=Xu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Jiangfeng Xu</a></div></div><div id="_co_idm139822320178496"><h3 class="no_margin">Siqun Zheng</h3><p><sup>58</sup>Genomics Genotyping Laboratory, Center for Cancer Genomics, Wake Forest University Health Sciences, Winston-Salem, North Carolina, USA</p><div>Find articles by <a href="/pubmed/?term=Zheng%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Siqun Zheng</a></div></div><div id="_co_idm139822320176512"><h3 class="no_margin">Meredith Yeager</h3><p><sup>18</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, US National Institutes of Health (NIH), Bethesda, Maryland, USA</p><p><sup>59</sup>Core Genotyping Facility, SAIC-Frederick, National Cancer Institute, US NIH, Gaithersburg, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Yeager%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Meredith Yeager</a></div></div><div id="_co_idm139822320173792"><h3 class="no_margin">Rudolf Kaaks</h3><p><sup>60</sup>Division of Cancer Epidemiology, DKFZ, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Kaaks%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Rudolf Kaaks</a></div></div><div id="_co_idm139822320171808"><h3 class="no_margin">W Ryan Diver</h3><p><sup>17</sup>Epidemiology Research Program, American Cancer Society, Atlanta, Georgia, USA</p><div>Find articles by <a href="/pubmed/?term=Diver%20WR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">W Ryan Diver</a></div></div><div id="_co_idm139822320169824"><h3 class="no_margin">Mia M Gaudet</h3><p><sup>17</sup>Epidemiology Research Program, American Cancer Society, Atlanta, Georgia, USA</p><div>Find articles by <a href="/pubmed/?term=Gaudet%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Mia M Gaudet</a></div></div><div id="_co_idm139822319456048"><h3 class="no_margin">Mariana Stern</h3><p><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Stern%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Mariana Stern</a></div></div><div id="_co_idm139822319454064"><h3 class="no_margin">Roman Corral</h3><p><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Corral%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Roman Corral</a></div></div><div id="_co_idm139822319452080"><h3 class="no_margin">Amit D Joshi</h3><p><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Joshi%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Amit D Joshi</a></div></div><div id="_co_idm139822319450096"><h3 class="no_margin">Ahva Shahabi</h3><p><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Shahabi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Ahva Shahabi</a></div></div><div id="_co_idm139822319448112"><h3 class="no_margin">Tiina Wahlfors</h3><p><sup>21</sup>Institute of Biomedical Technology (BioMediTech), University of Tampere and FimLab Laboratories, Tampere, Finland</p><div>Find articles by <a href="/pubmed/?term=Wahlfors%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Tiina Wahlfors</a></div></div><div id="_co_idm139822319446128"><h3 class="no_margin">Teuvo J Tammela</h3><p><sup>61</sup>Department of Urology, Tampere University Hospital and Medical School, University of Tampere, Tampere, Finland</p><div>Find articles by <a href="/pubmed/?term=Tammela%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Teuvo J Tammela</a></div></div><div id="_co_idm139822319444144"><h3 class="no_margin">Anssi Auvinen</h3><p><sup>62</sup>Department of Epidemiology, School of Health Sciences, University of Tampere, Tampere, Finland</p><div>Find articles by <a href="/pubmed/?term=Auvinen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Anssi Auvinen</a></div></div><div id="_co_idm139822319442160"><h3 class="no_margin">Jarmo Virtamo</h3><p><sup>63</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Virtamo%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Jarmo Virtamo</a></div></div><div id="_co_idm139822321163968"><h3 class="no_margin">Peter Klarskov</h3><p><sup>64</sup>Department of Urology, Herlev Hospital, Copenhagen University Hospital, Herlev, Denmark</p><div>Find articles by <a href="/pubmed/?term=Klarskov%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Peter Klarskov</a></div></div><div id="_co_idm139822321161984"><h3 class="no_margin">Børge G Nordestgaard</h3><p><sup>23</sup>Department of Clinical Biochemistry, Herlev Hospital, Copenhagen University Hospital, Herlev, Denmark</p><div>Find articles by <a href="/pubmed/?term=Nordestgaard%20BG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Børge G Nordestgaard</a></div></div><div id="_co_idm139822321160000"><h3 class="no_margin">Andreas Røder</h3><p><sup>65</sup>Department of Urology, Rigshospitalet, Copenhagen University Hospital, Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=R%26%23x000f8%3Bder%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Andreas Røder</a></div></div><div id="_co_idm139822321158016"><h3 class="no_margin">Sune F Nielsen</h3><p><sup>23</sup>Department of Clinical Biochemistry, Herlev Hospital, Copenhagen University Hospital, Herlev, Denmark</p><div>Find articles by <a href="/pubmed/?term=Nielsen%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Sune F Nielsen</a></div></div><div id="_co_idm139822321156032"><h3 class="no_margin">Stig E Bojesen</h3><p><sup>23</sup>Department of Clinical Biochemistry, Herlev Hospital, Copenhagen University Hospital, Herlev, Denmark</p><div>Find articles by <a href="/pubmed/?term=Bojesen%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Stig E Bojesen</a></div></div><div id="_co_idm139822321154048"><h3 class="no_margin">Afshan Siddiq</h3><p><sup>66</sup>Department of Genomics of Common Disease, School of Public Health, Imperial College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Siddiq%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Afshan Siddiq</a></div></div><div id="_co_idm139822321152064"><h3 class="no_margin">Liesel FitzGerald</h3><p><sup>32</sup>Fred Hutchinson Cancer Research Center, Division of Public Health Sciences, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=FitzGerald%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Liesel FitzGerald</a></div></div><div id="_co_idm139822322199184"><h3 class="no_margin">Suzanne Kolb</h3><p><sup>32</sup>Fred Hutchinson Cancer Research Center, Division of Public Health Sciences, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Kolb%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Suzanne Kolb</a></div></div><div id="_co_idm139822322197200"><h3 class="no_margin">Erika Kwon</h3><p><sup>34</sup>National Human Genome Research Institute, US NIH, Bethesda, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Kwon%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Erika Kwon</a></div></div><div id="_co_idm139822322195168"><h3 class="no_margin">Danielle Karyadi</h3><p><sup>34</sup>National Human Genome Research Institute, US NIH, Bethesda, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Karyadi%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Danielle Karyadi</a></div></div><div id="_co_idm139822322193136"><h3 class="no_margin">William J Blot</h3><p><sup>35</sup>International Epidemiology Institute, Rockville, Maryland, USA</p><p><sup>36</sup>Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Division of Epidemiology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Blot%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">William J Blot</a></div></div><div id="_co_idm139822322190320"><h3 class="no_margin">Wei Zheng</h3><p><sup>36</sup>Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Division of Epidemiology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Wei Zheng</a></div></div><div id="_co_idm139822322188288"><h3 class="no_margin">Qiuyin Cai</h3><p><sup>36</sup>Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Division of Epidemiology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Cai%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Qiuyin Cai</a></div></div><div id="_co_idm139822322186256"><h3 class="no_margin">Shannon K McDonnell</h3><p><sup>37</sup>Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=McDonnell%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Shannon K McDonnell</a></div></div><div id="_co_idm139822318833664"><h3 class="no_margin">Antje Rinckleb</h3><p><sup>39</sup>Department of Urology, University Hospital Ulm, Ulm, Germany</p><p><sup>40</sup>Institute of Human Genetics, University Hospital Ulm, Ulm, Germany</p><div>Find articles by <a href="/pubmed/?term=Rinckleb%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Antje Rinckleb</a></div></div><div id="_co_idm139822318830848"><h3 class="no_margin">Bettina Drake</h3><p><sup>20</sup>Division of Urologic Surgery, Washington University School of Medicine, St. Louis, Missouri, USA</p><div>Find articles by <a href="/pubmed/?term=Drake%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Bettina Drake</a></div></div><div id="_co_idm139822318828816"><h3 class="no_margin">Graham Colditz</h3><p><sup>20</sup>Division of Urologic Surgery, Washington University School of Medicine, St. Louis, Missouri, USA</p><div>Find articles by <a href="/pubmed/?term=Colditz%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Graham Colditz</a></div></div><div id="_co_idm139822318826784"><h3 class="no_margin">Dominika Wokolorczyk</h3><p><sup>42</sup>Department of Genetics and Pathology, International Hereditary Cancer Center, Pomeranian Medical University, Szczecin, Poland</p><div>Find articles by <a href="/pubmed/?term=Wokolorczyk%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Dominika Wokolorczyk</a></div></div><div id="_co_idm139822318824752"><h3 class="no_margin">Robert A Stephenson</h3><p><sup>44</sup>George E. Wahlen Department of Veterans Affairs Medical Center, Salt Lake City, Utah, USA</p><p><sup>67</sup>Huntsman Cancer Institute, Salt Lake City, Utah, USA</p><div>Find articles by <a href="/pubmed/?term=Stephenson%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Robert A Stephenson</a></div></div><div id="_co_idm139822318821840"><h3 class="no_margin">Craig Teerlink</h3><p><sup>43</sup>Division of Genetic Epidemiology, Department of Medicine, University of Utah School of Medicine, Salt Lake City, Utah, USA</p><div>Find articles by <a href="/pubmed/?term=Teerlink%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Craig Teerlink</a></div></div><div id="_co_idm139822318819808"><h3 class="no_margin">Heiko Muller</h3><p><sup>45</sup>Division of Clinical Epidemiology and Aging Research, DKFZ, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Muller%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Heiko Muller</a></div></div><div id="_co_idm139822322325424"><h3 class="no_margin">Dietrich Rothenbacher</h3><p><sup>45</sup>Division of Clinical Epidemiology and Aging Research, DKFZ, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Rothenbacher%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Dietrich Rothenbacher</a></div></div><div id="_co_idm139822322323392"><h3 class="no_margin">Thomas A Sellers</h3><p><sup>46</sup>Division of Cancer Prevention and Control, H. Lee Moffitt Cancer Center, Tampa, Florida, USA</p><div>Find articles by <a href="/pubmed/?term=Sellers%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Thomas A Sellers</a></div></div><div id="_co_idm139822322321360"><h3 class="no_margin">Hui-Yi Lin</h3><p><sup>46</sup>Division of Cancer Prevention and Control, H. Lee Moffitt Cancer Center, Tampa, Florida, USA</p><div>Find articles by <a href="/pubmed/?term=Lin%20HY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Hui-Yi Lin</a></div></div><div id="_co_idm139822322319328"><h3 class="no_margin">Chavdar Slavov</h3><p><sup>68</sup>Department of Urology, Alexandrovska University Hospital, Medical University Sofia, Sofia, Bulgaria</p><div>Find articles by <a href="/pubmed/?term=Slavov%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Chavdar Slavov</a></div></div><div id="_co_idm139822322317296"><h3 class="no_margin">Vanio Mitev</h3><p><sup>47</sup>Department of Medical Chemistry and Biochemistry, Molecular Medicine Center, Medical University of Sofia, Sofia, Bulgaria</p><div>Find articles by <a href="/pubmed/?term=Mitev%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Vanio Mitev</a></div></div><div id="_co_idm139822322315264"><h3 class="no_margin">Felicity Lose</h3><p><sup>49</sup>Molecular Cancer Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Lose%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Felicity Lose</a></div></div><div id="_co_idm139822322313232"><h3 class="no_margin">Srilakshmi Srinivasan</h3><p><sup>48</sup>Australian Prostate Cancer Research Centre, Queensland Institute of Health and Biomedical Innovation and School of Biomedical Science, Queensland University of Technology, Brisbane, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Srinivasan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Srilakshmi Srinivasan</a></div></div><div id="_co_idm139822322311200"><h3 class="no_margin">Sofia Maia</h3><p><sup>50</sup>Department of Genetics, Portuguese Oncology Institute, Porto, Portugal</p><p><sup>51</sup>Abel Salazar Biomedical Sciences Institute (ICBAS), Porto University, Porto, Portugal</p><div>Find articles by <a href="/pubmed/?term=Maia%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Sofia Maia</a></div></div><div id="_co_idm139822329095584"><h3 class="no_margin">Paula Paulo</h3><p><sup>50</sup>Department of Genetics, Portuguese Oncology Institute, Porto, Portugal</p><p><sup>51</sup>Abel Salazar Biomedical Sciences Institute (ICBAS), Porto University, Porto, Portugal</p><div>Find articles by <a href="/pubmed/?term=Paulo%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Paula Paulo</a></div></div><div id="_co_idm139822329092768"><h3 class="no_margin">Ethan Lange</h3><p><sup>69</sup>Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, USA</p><div>Find articles by <a href="/pubmed/?term=Lange%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Ethan Lange</a></div></div><div id="_co_idm139822329090736"><h3 class="no_margin">Kathleen A Cooney</h3><p><sup>70</sup>University of Michigan Comprehensive Cancer Center, Division of Hematology/Oncology, University of Michigan Medical School, Ann Arbor, Michigan, USA</p><div>Find articles by <a href="/pubmed/?term=Cooney%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Kathleen A Cooney</a></div></div><div id="_co_idm139822329088704"><h3 class="no_margin">Antonis Antoniou</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Antoniou%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Antonis Antoniou</a></div></div><div id="_co_idm139822329086672"><h3 class="no_margin">Daniel Vincent</h3><p><sup>71</sup>McGill University and Génome Québec Innovation Centre, Montreal, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Vincent%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Daniel Vincent</a></div></div><div id="_co_idm139822329084640"><h3 class="no_margin">François Bacot</h3><p><sup>71</sup>McGill University and Génome Québec Innovation Centre, Montreal, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Bacot%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">François Bacot</a></div></div><div id="_co_idm139822319257328"><h3 class="no_margin"> Tessier</h3><p><sup>71</sup>McGill University and Génome Québec Innovation Centre, Montreal, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Tessier%20%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732"> Tessier</a></div></div><div id="_co_idm139822319248032"><h3 class="no_margin">Zsofia Kote-Jarai</h3><p><sup>1</sup>The Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Kote-Jarai%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Zsofia Kote-Jarai</a></div></div><div id="_co_idm139822319245216"><h3 class="no_margin">Douglas F Easton</h3><p><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Easton%20DF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23535732">Douglas F Easton</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139822325294384_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139822325294384_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139822325294384_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>The Institute of Cancer Research, Sutton, UK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Royal Marsden National Health Service (NHS) Foundation Trust, London and Sutton, UK</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Centre for Cancer Genetic Epidemiology, Department of Oncology, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Surgical Oncology (Uro-Oncology: S4), University of Cambridge, Addenbrooke’s Hospital, Cambridge, UK</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre, Cambridge, UK</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Nuffield Department of Surgical Sciences, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Faculty of Medical Science, University of Oxford, John Radcliffe Hospital, Oxford, UK</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>School of Social and Community Medicine, University of Bristol, Bristol, UK</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Warwick Medical School, University of Warwick, Coventry, UK</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Institute of Population Health, University of Manchester, Manchester, UK</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Cancer Epidemiology Centre, The Cancer Council Victoria, Carlton, Victoria, Australia</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Melbourne, Victoria, Australia</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institute, Stockholm, Sweden</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California/Norris Comprehensive Cancer Center, Los Angeles, California, USA</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>University of Hawaii Cancer Center, Honolulu, Hawaii, USA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Pubic Health, Boston, Massachusetts, USA</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Epidemiology Research Program, American Cancer Society, Atlanta, Georgia, USA</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, US National Institutes of Health (NIH), Bethesda, Maryland, USA</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Nutritional Epidemiology Branch, National Cancer Institute, US NIH, Bethesda, Maryland, USA</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Division of Urologic Surgery, Washington University School of Medicine, St. Louis, Missouri, USA</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Institute of Biomedical Technology (BioMediTech), University of Tampere and FimLab Laboratories, Tampere, Finland</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Medical Biochemistry and Genetics, University of Turku, Turku, Finland</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Department of Clinical Biochemistry, Herlev Hospital, Copenhagen University Hospital, Herlev, Denmark</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg, Germany</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Department of Epidemiology &amp; Biostatistics, School of Public Health, Imperial College London, London, UK</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Cancer Epidemiology Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Cancer Prevention Institute of California, Fremont, California, USA</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Stanford Cancer Institute, Stanford University School of Medicine, Stanford, California, USA</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Department of Health Research &amp; Policy, Division of Epidemiology, Stanford University School of Medicine, Stanford, California, USA</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Department of Environmental Medicine, New York University (NYU) Langone Medical Center, Division of Epidemiology, NYU Cancer Institute, New York, New York, USA</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Clinical Gerontology Unit, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Fred Hutchinson Cancer Research Center, Division of Public Health Sciences, Seattle, Washington, USA</div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Department of Epidemiology, School of Public Health, University of Washington, Seattle, Washington, USA</div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>National Human Genome Research Institute, US NIH, Bethesda, Maryland, USA</div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>International Epidemiology Institute, Rockville, Maryland, USA</div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Division of Epidemiology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</div><div class="fm-affl" id="A37" lang="en"><sup>37</sup>Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, Minnesota, USA</div><div class="fm-affl" id="A38" lang="en"><sup>38</sup>Division of Biomedical Statistics and Informatics, Department of Health Sciences Research, Mayo Clinic, Rochester, Minnesota, USA</div><div class="fm-affl" id="A39" lang="en"><sup>39</sup>Department of Urology, University Hospital Ulm, Ulm, Germany</div><div class="fm-affl" id="A40" lang="en"><sup>40</sup>Institute of Human Genetics, University Hospital Ulm, Ulm, Germany</div><div class="fm-affl" id="A41" lang="en"><sup>41</sup>Division of Urologic Surgery, Brigham and Women’s Hospital, Dana-Farber Cancer Institute, Boston, Massachusetts, USA</div><div class="fm-affl" id="A42" lang="en"><sup>42</sup>Department of Genetics and Pathology, International Hereditary Cancer Center, Pomeranian Medical University, Szczecin, Poland</div><div class="fm-affl" id="A43" lang="en"><sup>43</sup>Division of Genetic Epidemiology, Department of Medicine, University of Utah School of Medicine, Salt Lake City, Utah, USA</div><div class="fm-affl" id="A44" lang="en"><sup>44</sup>George E. Wahlen Department of Veterans Affairs Medical Center, Salt Lake City, Utah, USA</div><div class="fm-affl" id="A45" lang="en"><sup>45</sup>Division of Clinical Epidemiology and Aging Research, DKFZ, Heidelberg, Germany</div><div class="fm-affl" id="A46" lang="en"><sup>46</sup>Division of Cancer Prevention and Control, H. Lee Moffitt Cancer Center, Tampa, Florida, USA</div><div class="fm-affl" id="A47" lang="en"><sup>47</sup>Department of Medical Chemistry and Biochemistry, Molecular Medicine Center, Medical University of Sofia, Sofia, Bulgaria</div><div class="fm-affl" id="A48" lang="en"><sup>48</sup>Australian Prostate Cancer Research Centre, Queensland Institute of Health and Biomedical Innovation and School of Biomedical Science, Queensland University of Technology, Brisbane, Queensland, Australia</div><div class="fm-affl" id="A49" lang="en"><sup>49</sup>Molecular Cancer Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Queensland, Australia</div><div class="fm-affl" id="A50" lang="en"><sup>50</sup>Department of Genetics, Portuguese Oncology Institute, Porto, Portugal</div><div class="fm-affl" id="A51" lang="en"><sup>51</sup>Abel Salazar Biomedical Sciences Institute (ICBAS), Porto University, Porto, Portugal</div><div class="fm-affl" id="A52" lang="en"><sup>52</sup>School of Biological Sciences, University of East Anglia, Norwich, UK</div><div class="fm-affl" id="A53" lang="en"><sup>53</sup>Cancer Research UK/Yorkshire Cancer Research Sheffield Cancer Research Centre, University of Sheffield, Sheffield, UK</div><div class="fm-affl" id="A54" lang="en"><sup>54</sup>Genetic Epidemiology Laboratory, Department of Pathology, The University of Melbourne, Parkville, Victoria, Australia</div><div class="fm-affl" id="A55" lang="en"><sup>55</sup>Department of Clinical Sciences, Division of Urology, Danderyd Hospital, Karolinska Institute, Stockholm, Sweden</div><div class="fm-affl" id="A56" lang="en"><sup>56</sup>Regional Cancer Centre, Department of Clinical Science, Intervention and Technology (CLINTEC), Karolinska Institute, Stockholm, Sweden</div><div class="fm-affl" id="A57" lang="en"><sup>57</sup>Wake Forest University School of Medicine, Winston-Salem, North Carolina, USA</div><div class="fm-affl" id="A58" lang="en"><sup>58</sup>Genomics Genotyping Laboratory, Center for Cancer Genomics, Wake Forest University Health Sciences, Winston-Salem, North Carolina, USA</div><div class="fm-affl" id="A59" lang="en"><sup>59</sup>Core Genotyping Facility, SAIC-Frederick, National Cancer Institute, US NIH, Gaithersburg, Maryland, USA</div><div class="fm-affl" id="A60" lang="en"><sup>60</sup>Division of Cancer Epidemiology, DKFZ, Heidelberg, Germany</div><div class="fm-affl" id="A61" lang="en"><sup>61</sup>Department of Urology, Tampere University Hospital and Medical School, University of Tampere, Tampere, Finland</div><div class="fm-affl" id="A62" lang="en"><sup>62</sup>Department of Epidemiology, School of Health Sciences, University of Tampere, Tampere, Finland</div><div class="fm-affl" id="A63" lang="en"><sup>63</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</div><div class="fm-affl" id="A64" lang="en"><sup>64</sup>Department of Urology, Herlev Hospital, Copenhagen University Hospital, Herlev, Denmark</div><div class="fm-affl" id="A65" lang="en"><sup>65</sup>Department of Urology, Rigshospitalet, Copenhagen University Hospital, Copenhagen, Denmark</div><div class="fm-affl" id="A66" lang="en"><sup>66</sup>Department of Genomics of Common Disease, School of Public Health, Imperial College London, London, UK</div><div class="fm-affl" id="A67" lang="en"><sup>67</sup>Huntsman Cancer Institute, Salt Lake City, Utah, USA</div><div class="fm-affl" id="A68" lang="en"><sup>68</sup>Department of Urology, Alexandrovska University Hospital, Medical University Sofia, Sofia, Bulgaria</div><div class="fm-affl" id="A69" lang="en"><sup>69</sup>Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, USA</div><div class="fm-affl" id="A70" lang="en"><sup>70</sup>University of Michigan Comprehensive Cancer Center, Division of Hematology/Oncology, University of Michigan Medical School, Ann Arbor, Michigan, USA</div><div class="fm-affl" id="A71" lang="en"><sup>71</sup>McGill University and Génome Québec Innovation Centre, Montreal, Quebec, Canada</div><div id="FN1">Correspondence should be addressed to R.E. (<a href="mailto:dev@null" data-email="ku.ca.rci@selee.dnilasor" class="oemail">ku.ca.rci@selee.dnilasor</a>)</div><div id="FN2"><sup>72</sup>A full list of members is provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</div><div id="FN3"><sup>73</sup>These authors contributed equally to this work.</div><div id="FN4"><sup>74</sup>These authors jointly directed this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139822325294384_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23535732" target="pmc_ext" ref="reftype=publisher&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3832790/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental Material.</strong></div><div><a href="/pmc/articles/PMC3832790/bin/NIHMS498644-supplement-Supplemental_Material.pdf" data-ga-action="click_feat_suppl">NIHMS498644-supplement-Supplemental_Material.pdf</a><span style="color:gray"> (12M)</span></div><div class="small guid">GUID:&nbsp;DE92F02E-89CB-45D1-9C3A-F15A06DA0602</div></div></dd></dl></div><div id="idm139822322370800" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139822322370800title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Prostate cancer is the most frequently diagnosed cancer in males in developed countries. To identify common prostate cancer susceptibility alleles, we genotyped 211,155 SNPs on a custom Illumina array (iCOGS) in blood DNA from 25,074 prostate cancer cases and 24,272 controls from the international PRACTICAL Consortium. Twenty-three new prostate cancer susceptibility loci were identified at genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>). More than 70 prostate cancer susceptibility loci, explaining ~30% of the familial risk for this disease, have now been identified. On the basis of combined risks conferred by the new and previously known risk loci, the top 1% of the risk distribution has a 4.7-fold higher risk than the average of the population being profiled. These results will facilitate population risk stratification for clinical studies.</p></div></div><div id="idm139822320626560" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Epidemiological studies provide strong evidence for genetic predisposition to prostate cancer. Most susceptibility loci identified thus far are common, low-penetrance variants found through genome-wide association studies (GWAS; reviewed in ref. <a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682818">1</a>). Fifty-four loci have been identified so far<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682804">1</a>–<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682813">6</a></sup>.</p><p id="P3">Because the risks associated with common susceptibility alleles are modest (per-allele odds ratios, ORs, ranging from 1.10–1.25), it is likely that other predisposition loci for prostate cancer have been missed by previous studies and that such loci should be detectable by studies with larger sample sizes<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534176427">7</a></sup>. Here, we report the findings from an extensive follow-up of GWAS conducted as part of a collaborative study with the Breast Cancer Association Consortium (BCAC), Ovarian Cancer Association Consortium (OCAC) and The Consortium of Investigators of Modifiers of <em>BRCA1/2</em> (CIMBA) as part of the COGS initiative.</p><p id="P4">We first conducted a meta-analysis of 4 GWAS conducted in populations of European ancestry that included 11,085 cases and 11,463 controls: UK/Australia, Cancer Genetic Markers of Susceptibility (CGEMS); Cancer of the Prostate in Sweden (CAPS) and the Breast and Prostate Cancer Cohort Consortium (BPC3). Genotype data from these GWAS were imputed using the HapMap 2 CEU panel (Utah residents of Northern and Western European ancestry) as a reference, and combined tests of association were then performed for ~2.6 million SNPs (Online Methods). From this meta-analysis, we selected 74,001 SNPs showing evidence of association with overall prostate cancer, aggressive prostate cancer or prostate cancer diagnosed at &lt;55 years of age (Online Methods). Specifically, we included all SNPs with significant association at <em>P</em> &lt; 0.01 for overall prostate cancer. These SNPs were genotyped as part of a custom array that included 211,155 SNPs (the iCOGS chip), 85,278 of which were specifically chosen for their potential relevance to prostate cancer (74,001 were from GWAS top hits as described, 13,739 were from fine mapping of known susceptibility regions at the time of the chip design and 1,398 were from candidate gene studies in key pathways (for example, hormone metabolism, HOX genes, the cell cycle and DNA repair; <a href="/pmc/articles/PMC3832790/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and Online Methods); some SNPs were in more than one category). The results of the GWAS component are presented here. The details of the iCOGS array can be found on the COGS website (see URLs).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3832790/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139822318677200" class="figure"><a href="/pmc/articles/PMC3832790/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3832790_nihms498644f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms498644f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3832790/bin/nihms498644f1.jpg"></a></div><div id="largeobj_idm139822318677200" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832790/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3832790/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div></div></div><p id="P5">The iCOGS array was used for the genotyping of 25,074 prostate cancer cases and 24,272 controls from 32 studies participating in the PRACTICAL Consortium (Online Methods). Of these, 39,337 samples of European ancestry and 1,192 of African-American or mixed African origin passed quality control and did not overlap with the GWAS sample sets. Only the results from samples of European ancestry are reported here (19,662 prostate cancer cases and 19,715 controls; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). Of the 201,598 SNPs that passed quality control, 72,157 were selected for replication of the combined GWAS (Online Methods).</p><p id="P6">Associations between SNP genotypes and prostate cancer were evaluated by logistic regression, adjusted for study and six principal components. Evidence for association was assessed using a 1-degree-of-freedom test for trend in risk by allele dose. When considering those SNPs not selected for association with prostate cancer, there was little evidence of inflation in the test statistics (λ= 1.136, equivalent to λ<sub>1,000</sub> = 1.008). There was, however, clear evidence of an excess of significant association for SNPs selected for replication of the prostate cancer GWAS (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>).</p><p id="P7">Results from the iCOGS replication stage were then combined with those from the GWAS to provide overall tests for association. After exclusion of SNPs in regions containing previously known loci associated with prostate cancer, 23 SNPs in 23 regions showed evidence of association in the combined GWAS and iCOGS replication stage analysis at <em>P</em> &lt; 5 × 10<sup>−8</sup> (<a href="/pmc/articles/PMC3832790/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="/pmc/articles/PMC3832790/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). There was no strong evidence for heterogeneity in the per-allele ORs between studies (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). All alleles are common (minor allele frequencies of 8–50%; <a href="/pmc/articles/PMC3832790/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) and conferred estimated per-allele ORs from 1.06–1.15. All but two of the autosomal SNPs associated with prostate cancer risk showed a pattern of association consistent with a log-additive model, as observed for most common cancer susceptibility alleles. For rs11902236 on chromosome 2, the estimated OR in the iCOGS replication stage for the heterozygote genotype was 1.04 (95% confidence interval (CI) = 0.99–1.08), which is smaller than expected under a log-additive model (<em>P</em> = 0.05), and, for rs7141529 on chromosome 14, the estimated OR in the iCOGS replication stage for the heterozygote genotype was 1.16 (95% CI = 1.10–1.21), which is greater than expected under a log-additive model (<em>P</em> = 0.004).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3832790/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139822318675024" class="figure"><a href="/pmc/articles/PMC3832790/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3832790_nihms498644f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms498644f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3832790/bin/nihms498644f2.jpg"></a></div><div id="largeobj_idm139822318675024" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832790/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3832790/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p10">Summary results for 23 SNPs identified as associated in European ancestry samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139822318667120" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="left">Marker</th><th rowspan="2" colspan="1" valign="top" align="left">Chr</th><th rowspan="2" colspan="1" valign="top" align="left">Position Allele</th><th rowspan="2" colspan="1" valign="top" align="left">MAF<sup>1</sup></th><th rowspan="2" colspan="1" valign="top" align="left">Per allele OR<sup>2</sup> (95%CI)</th><th rowspan="2" colspan="1" valign="top" align="left">Stage <sup>3</sup></th><th colspan="2" rowspan="1" valign="top" align="left">P-value</th><th rowspan="1" colspan="1" valign="top" align="left">Candidate genes</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Stage</th><th rowspan="1" colspan="1" valign="top" align="left">Combined</th><th rowspan="1" colspan="1" valign="top" align="left"></th></tr></thead><tbody><tr><td rowspan="2" colspan="1" valign="top" align="left">rs1218582</td><td rowspan="2" colspan="1" valign="top" align="left">1</td><td rowspan="2" colspan="1" valign="top" align="left">153100807 AG</td><td rowspan="2" colspan="1" valign="top" align="left">.45</td><td rowspan="2" colspan="1" valign="top" align="left">1.06 (1.03–1.09)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">6.0×10<sup>−5</sup></td><td rowspan="2" colspan="1" valign="top" align="left">1.95×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNN3</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">5.2×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs4245739</td><td rowspan="2" colspan="1" valign="top" align="left">1</td><td rowspan="2" colspan="1" valign="top" align="left">202785465 AC</td><td rowspan="2" colspan="1" valign="top" align="left">.25</td><td rowspan="2" colspan="1" valign="top" align="left">.91 (.88–.95)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">2.6×10<sup>−5</sup></td><td rowspan="2" colspan="1" valign="top" align="left">2.01×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>MDM4</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">1.7×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs11902236</td><td rowspan="2" colspan="1" valign="top" align="left">2</td><td rowspan="2" colspan="1" valign="top" align="left">10035319 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.27</td><td rowspan="2" colspan="1" valign="top" align="left">1.07 (1.03–1.10)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.8×10<sup>−5</sup></td><td rowspan="2" colspan="1" valign="top" align="left">2.84×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">1.1×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TAF1B:GRHL1</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs3771570</td><td rowspan="2" colspan="1" valign="top" align="left">2</td><td rowspan="2" colspan="1" valign="top" align="left">242031537 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.15</td><td rowspan="2" colspan="1" valign="top" align="left">1.12 (1.08–1.17)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">2.0×10<sup>−2</sup></td><td rowspan="2" colspan="1" valign="top" align="left">5.22×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">2.9×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>FARP2</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs7611694</td><td rowspan="2" colspan="1" valign="top" align="left">3</td><td rowspan="2" colspan="1" valign="top" align="left">114758314 AC</td><td rowspan="2" colspan="1" valign="top" align="left">.41</td><td rowspan="2" colspan="1" valign="top" align="left">.91 (.88–.93)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">7.1×10<sup>−4</sup></td><td rowspan="2" colspan="1" valign="top" align="left">3.80×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">6.3×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>SIDT1</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs1894292</td><td rowspan="2" colspan="1" valign="top" align="left">4</td><td rowspan="2" colspan="1" valign="top" align="left">74568022 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.48</td><td rowspan="2" colspan="1" valign="top" align="left">.91(.89–.94)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">9.8×10<sup>−5</sup></td><td rowspan="2" colspan="1" valign="top" align="left">5.02×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">9.9×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>AFM,RASSF6</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs6869841</td><td rowspan="2" colspan="1" valign="top" align="left">5</td><td rowspan="2" colspan="1" valign="top" align="left">172872032 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.21</td><td rowspan="2" colspan="1" valign="top" align="left">1.07 (1.04–1.11)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.2×10<sup>−4</sup></td><td rowspan="2" colspan="1" valign="top" align="left">4.63×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">7.8×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>FAM44B (BOD1)</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs3096702</td><td rowspan="2" colspan="1" valign="top" align="left">6</td><td rowspan="2" colspan="1" valign="top" align="left">32300309 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.40</td><td rowspan="2" colspan="1" valign="top" align="left">1.07 (1.04–1.10)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.0×10<sup>−4</sup></td><td rowspan="2" colspan="1" valign="top" align="left">4.78×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">1.1×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>NOTCH4</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs2273669</td><td rowspan="2" colspan="1" valign="top" align="left">6</td><td rowspan="2" colspan="1" valign="top" align="left">109391882 AG</td><td rowspan="2" colspan="1" valign="top" align="left">.15</td><td rowspan="2" colspan="1" valign="top" align="left">1.07 (1.03–1.11)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.6×10<sup>−7</sup></td><td rowspan="2" colspan="1" valign="top" align="left">7.91×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">1.0×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>ARMC2, SESN1</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs1933488</td><td rowspan="2" colspan="1" valign="top" align="left">6</td><td rowspan="2" colspan="1" valign="top" align="left">153482772 AG</td><td rowspan="2" colspan="1" valign="top" align="left">.41</td><td rowspan="2" colspan="1" valign="top" align="left">.89 (.87–.92)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.9×10<sup>−5</sup></td><td rowspan="2" colspan="1" valign="top" align="left">4.34×10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">2.6×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>RSG17</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs12155172</td><td rowspan="2" colspan="1" valign="top" align="left">7</td><td rowspan="2" colspan="1" valign="top" align="left">20961016 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.23</td><td rowspan="2" colspan="1" valign="top" align="left">1.11 (1.07–1.15)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">3.1×10<sup>−5</sup></td><td rowspan="2" colspan="1" valign="top" align="left">4.95×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">3.5×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>SP8</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs11135910</td><td rowspan="2" colspan="1" valign="top" align="left">8</td><td rowspan="2" colspan="1" valign="top" align="left">25948059 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.16</td><td rowspan="2" colspan="1" valign="top" align="left">1.11 (1.07–1.16)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">7.1×10<sup>−5</sup></td><td rowspan="2" colspan="1" valign="top" align="left">8.16×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">2.6×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>EBF2</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs3850699</td><td rowspan="2" colspan="1" valign="top" align="left">10</td><td rowspan="2" colspan="1" valign="top" align="left">104404211 AG</td><td rowspan="2" colspan="1" valign="top" align="left">.29</td><td rowspan="2" colspan="1" valign="top" align="left">.91 (.89–.94)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">3.5×10<sup>−3</sup></td><td rowspan="2" colspan="1" valign="top" align="left">4.87×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">2.6×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TRIM8</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs11568818</td><td rowspan="2" colspan="1" valign="top" align="left">11</td><td rowspan="2" colspan="1" valign="top" align="left">101906871 AG</td><td rowspan="2" colspan="1" valign="top" align="left">.44</td><td rowspan="2" colspan="1" valign="top" align="left">.91 (.88–.94)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.0×10<sup>−2</sup></td><td rowspan="2" colspan="1" valign="top" align="left">1.56×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">2.4×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>MMP7</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs1270884</td><td rowspan="2" colspan="1" valign="top" align="left">12</td><td rowspan="2" colspan="1" valign="top" align="left">113169954 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.49</td><td rowspan="2" colspan="1" valign="top" align="left">1.07 (1.04–1.10)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">4.1×10<sup>−7</sup></td><td rowspan="2" colspan="1" valign="top" align="left">6.75×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">1.2×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TBX5</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs8008270</td><td rowspan="2" colspan="1" valign="top" align="left">14</td><td rowspan="2" colspan="1" valign="top" align="left">52442080 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.18</td><td rowspan="2" colspan="1" valign="top" align="left">.89 (.86–.93)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">3.1×10<sup>−7</sup></td><td rowspan="2" colspan="1" valign="top" align="left">1.78×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">9.4×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>FERMT2</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs7141529</td><td rowspan="2" colspan="1" valign="top" align="left">14</td><td rowspan="2" colspan="1" valign="top" align="left">68196497 AG</td><td rowspan="2" colspan="1" valign="top" align="left">.50</td><td rowspan="2" colspan="1" valign="top" align="left">1.09 (1.06–1.12)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">3.3×10<sup>−3</sup></td><td rowspan="2" colspan="1" valign="top" align="left">2.77×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">1.3×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>RAD51L1</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs684232</td><td rowspan="2" colspan="1" valign="top" align="left">17</td><td rowspan="2" colspan="1" valign="top" align="left">565715 AG</td><td rowspan="2" colspan="1" valign="top" align="left">.36</td><td rowspan="2" colspan="1" valign="top" align="left">1.10 (1.07–1.14)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">4.7×10<sup>−6</sup></td><td rowspan="2" colspan="1" valign="top" align="left">5.17×10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">2.2×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>VPS53, FAM57A</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs11650494</td><td rowspan="2" colspan="1" valign="top" align="left">17</td><td rowspan="2" colspan="1" valign="top" align="left">44700185 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.08</td><td rowspan="2" colspan="1" valign="top" align="left">1.15 (1.09–1.22)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.5×10<sup>−3</sup></td><td rowspan="2" colspan="1" valign="top" align="left">1.97×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">3.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>HOXB13</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs7241993</td><td rowspan="2" colspan="1" valign="top" align="left">18</td><td rowspan="2" colspan="1" valign="top" align="left">74874961 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.30</td><td rowspan="2" colspan="1" valign="top" align="left">.92 (.89–.95)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.6×10<sup>−3</sup></td><td rowspan="2" colspan="1" valign="top" align="left">2.19×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">3.6×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>SALL3</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs2427345</td><td rowspan="2" colspan="1" valign="top" align="left">20</td><td rowspan="2" colspan="1" valign="top" align="left">60449006 GA</td><td rowspan="2" colspan="1" valign="top" align="left">.37</td><td rowspan="2" colspan="1" valign="top" align="left">.94 (.91–.97)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">4.4×10<sup>−4</sup></td><td rowspan="2" colspan="1" valign="top" align="left">3.64×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">2.1×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>GATAS, CABLES2</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs6062509</td><td rowspan="2" colspan="1" valign="top" align="left">20</td><td rowspan="2" colspan="1" valign="top" align="left">61833007 AC</td><td rowspan="2" colspan="1" valign="top" align="left">.30</td><td rowspan="2" colspan="1" valign="top" align="left">.89 (.86–.92)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.6×10<sup>−4</sup></td><td rowspan="2" colspan="1" valign="top" align="left">3.57×10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">4.1×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZGPAT</em></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="left">rs2405942</td><td rowspan="2" colspan="1" valign="top" align="left">X</td><td rowspan="2" colspan="1" valign="top" align="left">9774135 AG</td><td rowspan="2" colspan="1" valign="top" align="left">.21</td><td rowspan="2" colspan="1" valign="top" align="left">.88 (.83–.92)</td><td rowspan="1" colspan="1" valign="top" align="left">Combined GWAS</td><td rowspan="1" colspan="1" valign="top" align="left">1.4×10<sup>−4</sup></td><td rowspan="2" colspan="1" valign="top" align="left">2.37×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">iCOGS Replication Stage</td><td rowspan="1" colspan="1" valign="top" align="left">2.8×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>SHROOM2</em></td></tr></tbody></table></div><div id="largeobj_idm139822318667120" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3832790/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>1</sup>Allele frequency of the second allele in iCOGS replication stage.</div><div id="TFN2"><sup>2</sup>Per-allele OR in iCOGS replication stage for the second allele</div><div id="TFN3"><sup>3</sup>Combined GWAS: Stage 1&amp;2 UK, CGEMS, CAPS, BPC3</div></div></div><p id="P8">Aggressive disease was defined as that with Gleason score ≥ 8, prostate-specific antigen (PSA) &gt;100 ng/ml, disease stage of ‘distant’ (outside the pelvis) or death from prostate cancer. When aggressive disease was thus defined, three of the SNPs (rs3771570, rs2273669 and rs1270884) showed a significant difference in per-allele OR between aggressive and non-aggressive disease, in each case with a higher OR for non-aggressive disease and little or no association with aggressive disease (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). A similar pattern of association with respect to aggressive disease has been observed for SNPs in the <em>KLK3</em> region<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682806">8</a></sup>. The majority of SNPs, however, showed clear association when analysis was restricted to aggressive disease (for example, 13 SNPs showed significant associations at <em>P</em> &lt; 0.01 and 16 at <em>P</em> &lt; 0.05), and, for 22 of the 23 SNPs, the estimated ORs were in the same direction for aggressive and non-aggressive disease. Two SNPs, rs6869841 and rs1270884, were associated with PSA levels (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). Two of the SNPs showed a significantly higher OR in cases with a first- or second-degree relative with prostate cancer (rs3771570 and rs11135910; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Six SNPs showed a trend in OR with respect to age at diagnosis, with a higher OR at younger ages (rs3771570, rs7611694, rs6869841, rs3096702, rs684232 and rs7241993; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). This age effect has been seen previously for four prostate cancer susceptibility SNPs <sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682819">9</a></sup>.</p><p id="P9">We have also conducted an analysis of possible pathway enrichment for the previously reported susceptibility regions and those newly reported by extracting all genes overlapping a 500-kb or a 1-Mb window flanking each lead SNP (72 regions, 589 or 960 genes, respectively). GeneGo pathway enrichment analysis was used to identify any canonical pathways that were over-represented within this gene set. The most strongly associated pathways identified (false discovery rate &lt; 0.05) were cell adhesion and extracellular matrix (ECM) remodeling (<em>P</em> = 1.31 × 10<sup>−6</sup> to 3.6 × 10<sup>−9</sup>) and transcriptional regulation by the androgen receptor (<em>P</em> = 3.5 × 10<sup>−6</sup> to 3.5 × 10<sup>−8</sup>). WNT, FGF and IGF signaling also showed significant levels of enrichment (<em>P</em> = 1.69 × 10<sup>−4</sup> to 9.41 × 10<sup>−5</sup>).</p><p id="P10">The overall inflation in the test statistics for those SNPs selected for GWAS replication suggests that the number of susceptibility loci may be much larger. To address this possibility more formally, we identified 22,662 SNPs selected for replication of the prostate cancer GWAS that were uncorrelated (<em>r</em><sup>2</sup> &lt; 0.1 for any pair) and examined the directions of the estimated ORs in the iCOGS replication data set. The estimated effects were in the same direction as in the GWAS for 12,278 SNPs and in the opposite direction for 10,384 SNPs. On the basis of this analysis, 1,894 (95% CI = 1,600–2,188) selected SNPs reflect true associations with disease.</p><p id="P11">We have found 23 new loci associated with prostate cancer, 16 of which are associated with aggressive as well as non-aggressive disease, although none of the new loci are associated exclusively with the latter. This finding is, however, notable, as aggressive disease requires radical treatment, and, previously, the loci associated with prostate cancer were associated exclusively with non-aggressive disease, which is less likely to require clinical intervention.</p><p id="P12">All of the newly associated loci lie in linkage disequilibrium (LD) blocks that include plausible causative genes (<a href="/pmc/articles/PMC3832790/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3a–d</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>). LD regions vary greatly in the genome; here, we defined LD blocks as regions with SNPs with <em>r</em><sup>2</sup> &gt; 0.2 or took a 500-kb window around the lead SNPs. The list of genes in these 23 new susceptibility regions is given in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>. Fifteen of the 23 SNPs are either intronic (12 SNPs) or in the promoter region of a gene (3 SNPs). As described below, there are data in the literature that suggest that two of the newly associated SNPs impart direct functional effects that result in allele-specific alterations to the expression of the associated genes. This raises the possibility that these SNPs could themselves represent causative variants, although further fine-mapping studies and analysis of expression in primary prostate tissue would be needed to confirm this.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3832790/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139822318672848" class="figure"><a href="/pmc/articles/PMC3832790/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3832790_nihms498644f3a.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms498644f3a.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3832790/bin/nihms498644f3a.jpg"></a></div><div id="largeobj_idm139822318672848" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832790/figure/F3/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139822318671904" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3832790_nihms498644f3b.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms498644f3b.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3832790/bin/nihms498644f3b.jpg"></a></div><div id="largeobj_idm139822318671904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832790/figure/F3/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139822318670960" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3832790_nihms498644f3c.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms498644f3c.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3832790/bin/nihms498644f3c.jpg"></a></div><div id="largeobj_idm139822318670960" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832790/figure/F3/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139822318670016" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3832790_nihms498644f3d.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms498644f3d.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3832790/bin/nihms498644f3d.jpg"></a></div><div id="largeobj_idm139822318670016" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832790/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3832790/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div></div></div><p id="P13">Of the new loci identified in this study, SNP rs4245739 at 1q32 is situated in the 3′ UTR of the <em>MDM4</em> gene, 32 bp downstream of the stop codon. MDM4 is a negative regulator of TP53, thereby acting to inhibit cell cycle arrest and apoptosis, and is frequently overexpressed in a number of tumor types. rs4245739 is correlated with rs7556371 (<em>r</em><sup>2</sup> = 0.89), which showed some evidence of association with prostate cancer in a candidate gene study<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682825">10</a></sup>, and rs1380576 (<em>r</em><sup>2</sup> = 0.86), has previously been reported to be associated with prostate cancer aggressiveness in a case-only analysis of candidate SNPs in the TP53 pathway<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682836">11</a></sup>. rs4245739 has been shown to create an illegitimate binding site for miR-191 that results in the downregulation of <em>MDM4</em> expression<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682832">12</a></sup>; this is in agreement with our analysis using mirsnpscore<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682812">13</a></sup>, which predicted that the risk allele creates a binding site for miR-191, miR-887 and miR-3669. However, rs4245739 is also highly correlated with a number of other <em>MDM4</em> variants that overlap functional elements identified by the Encyclopedia of DNA Elements (ENCODE) Project<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682817">13</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682831">14</a></sup>. Other analyses using the iCOGS array have found that rs4245739 and correlated SNPs are associated with estrogen receptor (ER)-negative breast cancer<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682835">15</a></sup> and breast cancer in <em>BRCA1</em> mutation carriers<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682823">16</a></sup>. In addition, the risk allele (C) of rs4245739 has been associated with increased aggressiveness in individuals with ovarian cancer<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682809">17</a></sup>.</p><p id="P14">rs11568818 at 11q22 lies within a small LD region containing a single gene, <em>MMP7</em>, encoding a matrix metalloproteinase. Matrix metalloproteinases are implicated in metastasis, and elevated <em>MMP7</em> expression itself has been reported as a potential biomarker for metastatic prostate cancer and poor disease prognosis<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682808">18</a></sup>. This SNP is situated 181 bp upstream of the transcriptional start site in the promoter region, within an area of high sequence conservation that overlaps strong DNase hypersensitivity and transcription factor binding sites<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682827">13</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682821">14</a></sup>. rs11568818 itself has been established as a functional promoter variant, with the risk allele (A) having been shown to create a binding site for the FOXA2 transcription factor and result in higher <em>MMP7</em> expression<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682820">19</a></sup>. Increased expression of <em>MMP7</em> may represent a plausible mechanism responsible for the greater prostate cancer risk associated with this SNP; rs11568818 is correlated at <em>r</em><sup>2</sup> &gt; 0.5 with only four other variants and seems to be the most likely candidate for a causal variant.</p><p id="P15">rs7141529 at 14q24 lies within the last intron of the longest isoform of <em>RAD51B</em> (also known as <em>RAD51L1</em>). Members of the RAD51 family are involved in the repair of double-stranded DNA breaks by homologous recombination, and their loss is potentially oncogenic. A variant in <em>RAD51B</em> (rs999737) has previously been associated with breast cancer<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682830">20</a></sup>, and a second breast cancer susceptibility locus in intron 7 has also been identified in the iCOGS replication stage study<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682834">21</a></sup>. However, there is no correlation between rs7141529 and any of the breast cancer–associated SNPs.</p><p id="P16">rs11650494 is located at 17q21, a gene-dense locus that contains several genes that have been proposed as potential prostate cancer susceptibility or somatically altered genes, including <em>HOXB13</em>, <em>PRAC</em>, <em>SPOP</em> and <em>ZNF652</em>. rs11650494 is highly correlated with a number of other variants that overlap functional motifs identified in the ENCODE Project13,14. This signal appears to center around the <em>ZNF652</em> gene, with rs11650494 itself situated downstream of the gene within a long noncoding RNA (lincRNA) sequence. rs7210100 in intron 1 of <em>ZNF652</em> has previously been identified as a prostate cancer susceptibility gene in African-American men<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682822">22</a></sup>; however, this variant is rare among individuals of European ancestry, and the correlation between rs11650494 and rs7210100 is modest in the YRI (Yoruba from Ibadan, Nigeria) population (<em>r</em><sup>2</sup> = 0.22), suggesting that rs11650494 represents an independent or European-specific prostate cancer risk association. In addition, <em>ZNF652</em> has been reported to be highly expressed in the majority of prostate tumors and is associated with higher risk of relapse<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682814">23</a></sup>.</p><p id="P17">The <em>HOXB13</em>, <em>PRAC</em> and <em>SPOP</em> genes are all situated approximately 500 kb upstream or downstream of rs11650494; however, all are considered candidate prostate cancer genes, and, therefore, the possibility of a <em>trans</em>-regulatory element or locus control region associated with the rs11650494 association signal cannot be excluded. <em>HOXB13</em> is one of a cluster of homeobox domain–containing genes at this locus.</p><p id="P18">These genes are essential for vertebrate embryonic development, and <em>HOXB13</em> is important for normal prostate development and is a key regulator of the response to androgens<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682815">24</a></sup>. A rare variant in <em>HOXB13</em> (rs138213197; encoding a p.Gly84Glu alteration) has recently been shown to significantly increase prostate cancer risk, occurring in families with multiple cases of prostate cancer<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682816">25</a></sup>, and <em>HOXB13</em> expression levels have been proposed as a marker of prostate cancer<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682829">26</a></sup>. Analysis of 1,927 cases and 987 control samples from the CAPS study in which both rs11650494 and rs138213197 were genotyped showed that these SNPs are not correlated (<em>r</em><sup>2</sup> = 0.001) and that the OR for rs11650494 was not altered by adjustment for rs138213197 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>). <em>SPOP</em> encodes a protein that may modulate the transcriptional repression activities of death-associated protein 6 (encoded by <em>DAXX</em>), which interacts with histone deacetylase, core histones and other histone-associated proteins. <em>SPOP</em> is reported to be frequently mutated in prostate tumors, and it has been suggested that <em>SPOP</em> mutations may anchor a distinct genetic subtype of <em>ETS</em> fusion–negative prostate cancers<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682839">27</a></sup>.</p><p id="P19">In addition to the presence of plausible candidate genes, most of the 23 newly associated loci harbor several transcription factor binding sites within their LD regions.</p><p id="P20" class="p">With the identification of these new loci, 77 susceptibility loci for prostate cancer have now been identified. On the basis of an overall twofold familial relative risk for the first-degree relatives of prostate cancer cases and on the assumption that SNPs combine multiplicatively, the new loci reported here, together with those already known, explain approximately 30% of the familial risk of prostate cancer. Taking into consideration these SNPs and this risk model, the top 1% of men in the highest risk stratum have a 4.7-fold greater risk relative to the population average, and the top 10% of men have a 2.7-fold greater risk. For comparison, the former risk estimate is similar to that conferred by deleterious mutations in <em>BRCA2</em> (ref. <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682824">28</a>), and such mutation carriers are undergoing targeted screening in trials, for example, in the IMPACT (Identification of Men with a genetic predisposition to ProstAte Cancer: Targeted screening in men at higher genetic risk and controls) Study (see URLs). The SNP-based prostate cancer risk profile now available should therefore be able to distinguish men at a clinically meaningful level of risk. To evaluate the combined effect of the loci associated with prostate cancer risk, we included 68 of the known loci in a logistic regression (59 which were on iCOGS and 9 for which a surrogate with <em>r</em><sup>2</sup> &gt; 0.76 was available). The parameters from this model were used to generate polygenic risk scores (Online Methods). On the basis of these scores, the estimated risk for men in the top 1% of the risk distribution was 4.4-fold greater than the population average risk (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>), very close to the theoretical estimate predicted under a simple polygenic model (4.7-fold). Furthermore, under a polygenic genetic risk model<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385337506">29</a></sup>, an unaffected man aged 50 who has a father with prostate cancer diagnosed at 60 years of age would have a predicted lifetime risk of prostate cancer from his family history alone of just over 20%. However, if family history is taken into consideration along with the explicit effects of all known common prostate cancer susceptibility alleles, this predicted risk would rise to just over 60% if he were in the top 1% of the known polygenic risk score distribution (A. Antoniou, personal communication). Such differences in predicted risks will be important for facilitating risk stratification in targeted screening and prevention programs.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">URLs</h2><p id="P21" class="p p-first">COGs website, <a href="http://ec.europa.eu/research/health/medical-research/cancer/fp7-projects/cogs_en.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ec.europa.eu/research/health/medical-research/cancer/fp7-projects/cogs_en.html</a>;</p><p id="P22">IMPACT Study, <a href="http://www.impact-study.co.uk/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.impact-study.co.uk/</a>;</p><p id="P23">SNAP plots from the University of Michigan, <a href="http://csg.sph.umich.edu/locuszoom/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/locuszoom/</a>;</p><p id="P24">SNPTEST, <a href="https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html</a>;</p><p id="P25">MACH 1.0, <a href="http://www.sph.umich.edu/csg/abecasis/MACH/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MACH/</a>;</p><p id="P26">PRACTICAL, <a href="http://ccge.medschl.cam.ac.uk/consortia/practical/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ccge.medschl.cam.ac.uk/consortia/practical/index.html</a>;</p><p id="P27">GeneGo (now Thomson Reuters), <a href="http://thomsonreuters.com/products_services/science/systems-biology/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://thomsonreuters.com/products_services/science/systems-biology/</a>;</p><p id="P28">CGEMS Project, <a href="http://dceg.cancer.gov/research/how-we-study/genomic-studies/cgems-summary" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://dceg.cancer.gov/research/how-we-study/genomic-studies/cgems-summary</a>;</p><p id="P29">BPC3, <a href="http://epi.grants.cancer.gov/BPC3/cohorts.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://epi.grants.cancer.gov/BPC3/cohorts.html</a>;</p><p id="P30">CAPS, <a href="http://ki.se/ki/jsp/polopoly.jsp?d=13809&amp;a=29862&amp;l=en" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ki.se/ki/jsp/polopoly.jsp?d=13809&amp;a=29862&amp;l=en</a>;</p><p id="P31" class="p p-last">SCCS, <a href="http://www.southerncommunitystudy.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.southerncommunitystudy.org/</a>.</p></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title">ONLINE METHODS</h2><div id="S4" class="sec sec-first"><h3 id="S4title">GWAS analysis</h3><p id="P33" class="p p-first">Primary genotype data were obtained for three prostate cancer GWAS (CGEMS, UK/Australia stages 1 and 2, and CAPS). Standard quality control was performed on all scans; all individuals with low call rate (&lt;95%), extremely high or low heterozygosity (<em>P</em> &lt; 1 × 10<sup>−5</sup>) and non-European ancestry (&gt;15% non-European component by multidimensional scaling using the three HapMap 2 populations (European (CEU), Asian (CHB and JPT) and African (YRI)) as a reference) were excluded. SNPs with call rate &lt; 95%; call rate &lt; 99% and MAF &lt; 5%, or MAF &lt; 1% and SNPs whose genotype frequencies departed from Hardy-Weinberg equilibrium at <em>P</em> &lt; 1 × 10<sup>−6</sup> in controls or <em>P</em> &lt; 1 × 10<sup>−12</sup> in cases were excluded. For BPC3, quality control was performed as previously described<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682838">30</a></sup>. Genotypes in all four GWAS were imputed for ~2.6 million SNPs using the HapMap phase 2 CEU population as a reference. UK/Australia stages 1 and 2 and CGEMS were imputed using MACH 1.0 (see URLs) for auto-somal markers and IMPUTE v1 (ref. <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682826">31</a>) for chromosome X markers. Imputation for the BPC3 study used MACH 1.0. The CAPS study used IMPUTE v1. We included imputed data from a SNP in the combined analysis if the estimated correlation between the genotype scores and the true genotypes (<em>r</em>2) was &gt;0.3 (MACH) or if the quality information was &gt;0.3 (IMPUTE).</p><p id="P34" class="p p-last">For UK stages 1 and 2 and CGEMS, the imputed genotype probabilities were used to derive a 1-degree-of-freedom association score statistic and its corresponding variance for each SNP. The test statistic for UK/Australia stage 2 was stratified by population as previously described<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682805">32</a></sup>. In the BPC3 study, estimated β values and standard errors were calculated for each component study, including one principal component as a covariate to adjust for population structure using ProbABEL<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682803">33</a></sup>, and the results were combined to generate overall β values and standard errors using a fixed-effects meta-analysis. CAPS used SNPTEST (see URLs) to estimate β values and standard errors. We converted the results from all studies into test scores and variances and hence derived a combined χ<sup>2</sup> trend statistic for each SNP (equivalent to the Mantel extension test or as in a fixed-effects meta-analysis) in R. All studies were approved by the appropriate national ethics committees, and informed consent was obtained.</p></div><div id="S5" class="sec"><h3 id="S5title">SNP selection</h3><p id="P35" class="p p-first">SNPs were selected for the iCOGS array separately by each consortium. Each consortium was given a share of the array: nominally, 25% of the SNPs each for BCAC, PRACTICAL and OCAC and 17.5% for CIMBA; 7.5% were of general interest (COMMON area). In practice, the allocations were larger as a result of overlaps. In each consortium, the allocation was divided into three categories for GWAS replication, fine-mapping and candidate SNPs. The GWAS replication category consisted of a series of lists for each analysis (see the PRACTICAL website for a full description of the lists).</p><p id="P36">In general, we considered only SNPs with an Illumina design score of 0.8 or greater (some OCAC and CIMBA SNPs with lower design scores were included). Where possible, preference was given to SNPs previously genotyped by Illumina (design score = 1.1). For each category, we defined a series of ranked lists of SNPs. For the GWAS SNPs, these were merged in the following way to generate a single list. We selected SNPs in priority order from each list according to predefined weightings. When a SNP (or a surrogate) was selected on the basis of more than one list, the SNP counted toward the tally for each list. For each SNP, we preferentially accepted the SNP if it had a design score of 1.1 (meaning it had previously been genotyped on an Illumina platform). If this was not the case, we sought SNPs with <em>r</em><sup>2</sup> = 1 with the chosen SNP and selected the SNP with the best design score. If no such SNP was available, we selected SNPs with <em>r</em><sup>2</sup> &gt; 0.8 with the chosen SNP and selected the SNP with the best design score. We excluded SNPs that were in strong LD with a previously selected SNP (<em>r</em><sup>2</sup> &gt; 0.9). However, for SNPs that were highly significant in each list (<em>P</em> &lt; 0.00001), we required two surrogate SNPs. The candidate lists were merged in the same way, giving equal weight to lists from each study. The only differences were that (i) there was no provision for additional surrogates and (ii) SNPs were excluded if there was an existing surrogate at <em>r</em><sup>2</sup> = 1.</p><p id="P37">To merge the three categories, we first included all the selected fine-mapping SNPs and then included SNPs from the merged GWAS and candidate lists in priority order. COMMON SNPs were selected in a similar way.</p><p id="P38" class="p p-last">Finally, lists from each of the constituent consortia were merged, in priority order and in proportion to the allocated share of each consortium. SNPs selected by one consortium and subsequently selected by another counted toward both lists. The process continued until the maximum 240,000 attempted beadtypes had been reached. The final list comprised 220,123 SNPs. Of these, 211,155 were successfully manufactured on the array.</p></div><div id="S6" class="sec"><h3 id="S6title">iCOGS genotyping</h3><p id="P39" class="p p-first">Samples for the iCOGS replication stage were drawn from 32 studies participating in the PRACTICAL Consortium. The majority of studies were population-based or hospital-based case-control studies or were nested case-control studies, but some studies selected samples by age or oversampled for cases with a family history of disease; in the latter instance only, one case per family was genotyped (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). Studies were required to provide ~2% of samples in duplicate.</p><p id="P40" class="p p-last">Genotyping was conducted using a custom Illumina Infinium array (iCOGS) in seven centers, of which five were used for PRACTICAL samples. Genotypes were called using Illumina’s proprietary GenCall algorithm. Initial calling used a cluster file generated with 270 samples from HapMap 2. To generate the final calls, we first selected a subset of 3,018 individuals, including samples from each of the genotyping centers, each of the participating consortia and each major ancestry group. Only plates with consistently high call rates in the initial calling were used. We also included 380 samples of European, Asian or African ancestry genotyped as part of the HapMap Project and 1000 Genomes Project and 160 samples that were known positive controls for rare variants on the array. This subset was used to generate a cluster file that was then applied to call the genotypes for the remaining samples. We also investigated two other calling algorithms: Illumnus<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682810">34</a></sup> and GenoSNP<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682837">35</a></sup>. All three algorithms were &gt;99% concordant in their calling for 91% of the SNPs on the array. However, manual inspection of a sample of the discrepant SNPs indicated that the calls from GenCall were almost invariably superior (generally because Illumnus or GenoSNP attempted to call SNPs that clustered poorly). Therefore, only the genotypes called by GenCall have been used in the analyses reported here.</p></div><div id="S7" class="sec"><h3 id="S7title">Quality control</h3><p id="P41" class="p p-first">We excluded individuals for any of the following reasons: genotypically not male XY (XX or XXY); overall call rate &lt; 95%; low or high heterozygosity (<em>P</em> &lt; 1 × 10<sup>−6</sup>, separately for individuals of European and African-American ancestry); not concordant with previous genotyping within PRACTICAL; genotypes for the duplicate sample that appeared to be from a different individual; and cryptic duplicates where the phenotypic data indicated that the individuals were different. We searched for cryptic duplicates both within each study and between studies from the same country. For known and cryptic duplicates, the sample with the lower call rate was excluded. We attempted to identify first-degree relative pairs using identity-by-state estimates based on data from ~37,000 uncorrelated SNPs. For apparent first-degree relative pairs, we removed the control from a case-control pair, otherwise, the individual with the lower call rate. For all analyses presented here, we also excluded 6,766 individuals who were included in any of the GWAS to allow the GWAS and iCOGS replication stages to be combined.</p><p id="P42">Ancestry outliers were identified by multidimensional scaling, combining the iCOGS replication stage data with those from the three HapMap 2 populations, based on a subset of 37,000 uncorrelated markers that passed quality control (including ~1,000 selected as ancestry-informative markers). Most studies included individuals predominantly of single, European ancestry, and individuals with &gt;15% minority ancestry were excluded. One study (SCCS) primarily contained individuals of African-American ancestry, and two studies, FHCRC and MOFFITT, contained substantial fractions of individuals of both African-American and European ancestry. After exclusion of ancestry outliers, we used principal-components analysis to correct for inflation. Principal-components analyses were carried out separately for the European and African-American subgroups on the basis of a subset of 37,000 uncorrelated SNPs. We included the first six principal components as covariates in both the European and African-American subgroups. Addition of further principal components did not reduce inflation further. Only the European data are reported here.</p><p id="P43">We excluded SNPs with call rates of &lt;95%. We also excluded SNPs that deviated from Hardy-Weinberg equilibrium in controls at <em>P</em> &lt; 1 × 10<sup>−7</sup>, on the basis of a stratified 1-degree-of-freedom test in which the deviations were summed across strata<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682828">36</a></sup>. We also excluded SNPs for which the genotypes were discrepant in more than 2% of duplicate samples, across all COGS consortia. The final analyses were based on data from 201,598 SNPs.</p><p id="P44" class="p p-last">Genotype intensity cluster plots were examined manually (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>) for SNPs in each new region in which an association at genome-wide significance was obtained, and SNPs eliminated in the clustering were judged to be poor.</p></div><div id="S8" class="sec"><h3 id="S8title">Statistical analysis</h3><p id="P45" class="p p-first">For each SNP, we estimated a per-allele log(OR) and standard error by logistic regression, including study and principal components as covariates. Overall significance levels were obtained by combining the estimates from the combined GWAS and the iCOGS replication stage using a fixed-effects meta-analysis. Tests of homogeneity of the ORs across strata and populations were assessed using likelihood ratio tests. Modification of the ORs by disease aggressiveness and family history was assessed using a case-only analysis. Modification of the ORs by age was examined using a case-only analysis assessing the association between age and SNP genotype in the cases using polytomous regression. The associations between SNP genotypes and PSA levels were assessed using linear regression, after log transformation of PSA levels to correct for the positively skewed distribution of PSA levels (ng/ml). Analyses were performed in R, principally using GenABEL<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682811">37</a></sup>, SNPTEST, ProbABEL<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341682802">33</a></sup> and Stata.</p><p id="P46">The contribution of the known SNPs to the familial risk of prostate cancer, under a multiplicative model, was computed using the formula</p><p id="P47">
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><munder><mo>∑</mo><mi>k</mi></munder><mrow><mo stretchy="false">(</mo><mo>log</mo><msub><mi>λ</mi><mi>k</mi></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><mo>log</mo><mo stretchy="false">(</mo><msub><mi>λ</mi><mn>0</mn></msub><mo stretchy="false">)</mo></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M1" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="munder" id="MathJax-Span-3"><span style="display: inline-block; position: relative; width: 1.444em; height: 0px;"><span style="position: absolute; clip: rect(2.872em, 1001.39em, 4.582em, -1000em); top: -3.977em; left: 0em;"><span class="mo" id="MathJax-Span-4" style="font-family: MathJax_Size2; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.332em, 1000.36em, 4.24em, -1000em); top: -2.87em; left: 0.538em;"><span class="mi" id="MathJax-Span-5" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mrow" id="MathJax-Span-6" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main;">(</span><span class="mo" id="MathJax-Span-8" style="font-family: MathJax_Main;">log</span><span class="msub" id="MathJax-Span-9" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.026em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.56em, 4.144em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-10" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.583em;"><span class="mi" id="MathJax-Span-11" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-12" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-13" style="font-family: MathJax_Main; padding-left: 0.167em;">/</span><span class="mo" id="MathJax-Span-14" style="font-family: MathJax_Main; padding-left: 0.167em;">log</span><span class="mo" id="MathJax-Span-15" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-16"><span style="display: inline-block; position: relative; width: 1.012em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.56em, 4.144em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-17" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.583em;"><span class="mn" id="MathJax-Span-18" style="font-size: 70.7%; font-family: MathJax_Main;">0</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-19" style="font-family: MathJax_Main;">)</span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><munder><mo>∑</mo><mi>k</mi></munder><mrow><mo stretchy="false">(</mo><mo>log</mo><msub><mi>λ</mi><mi>k</mi></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><mo>log</mo><mo stretchy="false">(</mo><msub><mi>λ</mi><mn>0</mn></msub><mo stretchy="false">)</mo></math></script></div></div><p> where λ0 is the observed familial risk to first-degree relatives of prostate cancer cases, assumed to be 2, and λ<em>k</em> is the familial relative risk due to locus <em>k</em>, given by</p><p id="P48" class="p p-last">
</p><div class="disp-formula" id="FD2"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><msub><mi>λ</mi><mi>k</mi></msub><mo>=</mo><mfrac><mrow><msub><mi>p</mi><mi>k</mi></msub><msubsup><mi>r</mi><mi>k</mi><mn>2</mn></msubsup><mo>+</mo><msub><mi>q</mi><mi>k</mi></msub></mrow><msup><mrow><mo>(</mo><mrow><msub><mi>p</mi><mi>k</mi></msub><msub><mi>r</mi><mi>k</mi></msub><mo>+</mo><msub><mi>q</mi><mi>k</mi></msub></mrow><mo>)</mo></mrow><mn>2</mn></msup></mfrac></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style=""><nobr><span class="math" id="M2" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-21"><span class="msub" id="MathJax-Span-22"><span style="display: inline-block; position: relative; width: 1.026em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.56em, 4.144em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-23" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.583em;"><span class="mi" id="MathJax-Span-24" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-25" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-26" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 5.279em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.007em, 1003.95em, 4.466em, -1000em); top: -4.78em; left: 50%; margin-left: -1.976em;"><span class="mrow" id="MathJax-Span-27"><span class="msub" id="MathJax-Span-28"><span style="display: inline-block; position: relative; width: 0.946em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.5em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-29" style="font-family: MathJax_Math; font-style: italic;">p</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.503em;"><span class="mi" id="MathJax-Span-30" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-31"><span style="display: inline-block; position: relative; width: 0.894em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.43em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-32" style="font-family: MathJax_Math; font-style: italic;">r</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.351em, 1000.43em, 4.132em, -1000em); top: -4.322em; left: 0.451em;"><span class="mn" id="MathJax-Span-33" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.332em, 1000.44em, 4.14em, -1000em); top: -3.651em; left: 0.451em;"><span class="mi" id="MathJax-Span-34" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-35" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-36" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.889em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-37" style="font-family: MathJax_Math; font-style: italic;">q<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.014em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.446em;"><span class="mi" id="MathJax-Span-38" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(2.874em, 1005.16em, 4.382em, -1000em); top: -3.06em; left: 50%; margin-left: -2.579em;"><span class="msup" id="MathJax-Span-39"><span style="display: inline-block; position: relative; width: 5.159em; height: 0px;"><span style="position: absolute; clip: rect(3.072em, 1004.64em, 4.382em, -1000em); top: -3.977em; left: 0em;"><span class="mrow" id="MathJax-Span-40"><span class="mo" id="MathJax-Span-41" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-42"><span class="msub" id="MathJax-Span-43"><span style="display: inline-block; position: relative; width: 0.946em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.5em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-44" style="font-family: MathJax_Math; font-style: italic;">p</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.503em;"><span class="mi" id="MathJax-Span-45" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msub" id="MathJax-Span-46"><span style="display: inline-block; position: relative; width: 0.894em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.43em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-47" style="font-family: MathJax_Math; font-style: italic;">r</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.451em;"><span class="mi" id="MathJax-Span-48" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-49" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-50" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.889em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-51" style="font-family: MathJax_Math; font-style: italic;">q<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.014em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.446em;"><span class="mi" id="MathJax-Span-52" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-53" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -4.454em; left: 4.73em;"><span class="mn" id="MathJax-Span-54" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.865em, 1005.28em, 1.24em, -1000em); top: -1.305em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 5.279em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><msub><mi>λ</mi><mi>k</mi></msub><mo>=</mo><mfrac><mrow><msub><mi>p</mi><mi>k</mi></msub><msubsup><mi>r</mi><mi>k</mi><mn>2</mn></msubsup><mo>+</mo><msub><mi>q</mi><mi>k</mi></msub></mrow><msup><mrow><mo>(</mo><mrow><msub><mi>p</mi><mi>k</mi></msub><msub><mi>r</mi><mi>k</mi></msub><mo>+</mo><msub><mi>q</mi><mi>k</mi></msub></mrow><mo>)</mo></mrow><mn>2</mn></msup></mfrac></math></script></div></div><p> where <em>pk</em> is the frequency of the risk allele for locus <em>k</em>, <em>qk</em> = 1 − <em>pk</em>, and <em>rk</em> is the estimated per-allele OR.</p></div><div id="S9" class="sec"><h3 id="S9title">Inflation</h3><p id="P49" class="p p-first">We estimated the inflation for each analysis on the basis of the 45th percentile of the test statistic for SNPs not selected by PRACTICAL and not in the COMMON fine-mapping regions. The inflation was 1.136 for the subgroup of European ancestry and 1.001 for the subgroup of African-American ancestry. Inflation was converted to an equivalent inflation for a study with 1,000 cases and 1,000 controls (λ1,000) by adjusting for effective study size, namely</p><p id="P50" class="p p-last">
</p><div class="disp-formula" id="FD3"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" display="block" overflow="scroll"><msub><mi>λ</mi><mn>1000</mn></msub><mo>=</mo><mn>1</mn><mo>+</mo><mfrac><mrow><mn>500</mn><mo stretchy="false">(</mo><mi>λ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mi>k</mi></munder></mstyle><msup><mrow><mo>(</mo><mrow><mfrac><mn>1</mn><msub><mi>n</mi><mi>k</mi></msub></mfrac><mo>+</mo><mfrac><mn>1</mn><msub><mi>m</mi><mi>k</mi></msub></mfrac></mrow><mo>)</mo></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></mrow></mfrac></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style=""><nobr><span class="math" id="M3" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-56"><span class="msub" id="MathJax-Span-57"><span style="display: inline-block; position: relative; width: 2.072em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.56em, 4.144em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-58" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.583em;"><span class="mn" id="MathJax-Span-59" style="font-size: 70.7%; font-family: MathJax_Main;">1000</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-60" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-61" style="font-family: MathJax_Main; padding-left: 0.278em;">1</span><span class="mo" id="MathJax-Span-62" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mfrac" id="MathJax-Span-63" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 7.518em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.072em, 1004.49em, 4.382em, -1000em); top: -4.696em; left: 50%; margin-left: -2.292em;"><span class="mrow" id="MathJax-Span-64"><span class="mn" id="MathJax-Span-65" style="font-family: MathJax_Main;">500</span><span class="mo" id="MathJax-Span-66" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-67" style="font-family: MathJax_Math; font-style: italic;">λ</span><span class="mo" id="MathJax-Span-68" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="mn" id="MathJax-Span-69" style="font-family: MathJax_Main; padding-left: 0.222em;">1</span><span class="mo" id="MathJax-Span-70" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(2.475em, 1007.4em, 5.347em, -1000em); top: -2.661em; left: 50%; margin-left: -3.699em;"><span class="mrow" id="MathJax-Span-71"><span class="mstyle" id="MathJax-Span-72"><span class="mrow" id="MathJax-Span-73"><span class="munder" id="MathJax-Span-74"><span style="display: inline-block; position: relative; width: 1.444em; height: 0px;"><span style="position: absolute; clip: rect(2.872em, 1001.39em, 4.582em, -1000em); top: -3.977em; left: 0em;"><span class="mo" id="MathJax-Span-75" style="font-family: MathJax_Size2; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.332em, 1000.36em, 4.24em, -1000em); top: -2.87em; left: 0.538em;"><span class="mi" id="MathJax-Span-76" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span><span class="msup" id="MathJax-Span-77" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 5.787em; height: 0px;"><span style="position: absolute; clip: rect(2.673em, 1004.63em, 4.782em, -1000em); top: -3.977em; left: 0em;"><span class="mrow" id="MathJax-Span-78"><span class="mo" id="MathJax-Span-79" style="vertical-align: 0em;"><span style="font-family: MathJax_Size2;">(</span></span><span class="mrow" id="MathJax-Span-80"><span class="mfrac" id="MathJax-Span-81"><span style="display: inline-block; position: relative; width: 0.858em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.351em, 1000.3em, 4.132em, -1000em); top: -4.384em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-82" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.51em, 1000.74em, 4.244em, -1000em); top: -3.632em; left: 50%; margin-left: -0.369em;"><span class="msub" id="MathJax-Span-83"><span style="display: inline-block; position: relative; width: 0.738em; height: 0px;"><span style="position: absolute; clip: rect(3.51em, 1000.41em, 4.14em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-84" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.871em; left: 0.424em;"><span class="mi" id="MathJax-Span-85" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.865em, 1000.86em, 1.24em, -1000em); top: -1.305em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.858em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span><span class="mo" id="MathJax-Span-86" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mfrac" id="MathJax-Span-87" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.054em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.351em, 1000.3em, 4.132em, -1000em); top: -4.384em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-88" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.51em, 1000.93em, 4.244em, -1000em); top: -3.632em; left: 50%; margin-left: -0.467em;"><span class="msub" id="MathJax-Span-89"><span style="display: inline-block; position: relative; width: 0.934em; height: 0px;"><span style="position: absolute; clip: rect(3.51em, 1000.61em, 4.14em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-90" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.871em; left: 0.621em;"><span class="mi" id="MathJax-Span-91" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.865em, 1001.05em, 1.24em, -1000em); top: -1.305em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 1.054em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-92" style="vertical-align: 0em;"><span style="font-family: MathJax_Size2;">)</span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -4.854em; left: 4.808em;"><span class="mrow" id="MathJax-Span-93"><span class="mo" id="MathJax-Span-94" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-95" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.865em, 1007.52em, 1.24em, -1000em); top: -1.305em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 7.518em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" display="block" overflow="scroll"><msub><mi>λ</mi><mn>1000</mn></msub><mo>=</mo><mn>1</mn><mo>+</mo><mfrac><mrow><mn>500</mn><mo stretchy="false">(</mo><mi>λ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mi>k</mi></munder></mstyle><msup><mrow><mo>(</mo><mrow><mfrac><mn>1</mn><msub><mi>n</mi><mi>k</mi></msub></mfrac><mo>+</mo><mfrac><mn>1</mn><msub><mi>m</mi><mi>k</mi></msub></mfrac></mrow><mo>)</mo></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></mrow></mfrac></math></script></div></div><p> where <em>nk</em> and <em>mk</em> were the number of cases and controls, respectively, for study <em>k</em>.</p></div><div id="S10" class="sec"><h3 id="S10title">Estimation of the number of associated loci</h3><p id="P51" class="p p-first-last">To estimate the total number of newly associated loci selected for the iCOGS replication stage, we identified a set of 22,662 SNPs selected for replication of the GWAS and not selected for fine mapping to exclude previously known loci that were uncorrelated (<em>r</em><sup>2</sup> &lt; 0.1 for any pair). We then determined the number of loci for which the estimated effect sizes in the iCOGS replication were in the same direction as in the combined GWAS or in the opposite direction. Similar results were obtained using cutoffs of <em>r</em><sup>2</sup> &lt; 0.05 and <em>r</em><sup>2</sup> &lt; 0.2.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Pathway analysis</h3><p id="P52" class="p p-first-last">GeneGo pathway enrichment was used to determine whether any canonical pathway was significantly enriched with false discovery rate &lt; 0.05.</p></div></div><div id="S12" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemental Material</h4><div class="sup-box half_rhythm" id="d36e2600"><a href="/pmc/articles/PMC3832790/bin/NIHMS498644-supplement-Supplemental_Material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12M, pdf)</sup></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Acknowledgments</h2><div class="sec"><p id="__p4">Acknowledgments are detailed in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</p></div></div><div id="idm139822325551040" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139822325551040title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN5"></p><p id="__p5" class="p p-first-last"><em>Note:</em>
<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary information</a>
<em>is available in the online version of the paper.</em></p><p></p><div><p id="__p6" class="p p-first"><strong>AUTHOR CONTRIBUTIONS</strong></p>R.A.E. and D.F.E. designed the study. R.A.E. is principal investigator of PRACTICAL. D.F.E. is Scientific Director of the COGS initiative. Z.K.-J. is co-investigator of PRACTICAL. R.A.E., D.F.E., Z.K.-J. and A.A.A.O. wrote the manuscript; the following named coauthors commented on the manuscript. A.A.A.O. and D.F.E. performed the statistical analyses; S.B. collated the data set. J.D. managed the database. Z.K.-J., E.J. Saunders, D.A.L. and M.T. coordinated sample collation and quality control for iCOGS PRACTICAL genotyping. S.J.-L. carried out pathway analysis and constructed regional plots, and T. Dadaev, K.G., M. Guy, R.A.W., E.J. Sawyer and A.M. managed the UKGPCS database and manifests for genotyping. C.L., A.M.D., C.B., D. Conroy, M.J.M., S.A., E.D., A. Lee, D.C.T., F.B. and D.V. carried out iCOGS PRACTICAL genotyping and set quality control standards. M. Ghoussaini selected the iCOGS PRACTICAL SNPs for fine-scale mapping. K.M. and A. Lophatananon collected some of the UKGPCS samples and controls. F.C.H., D.E.N. and J.L.D. are joint principal investigators of ProtecT. B.E.H. and L.L.M. are principal investigators of MEC; C.A.H. and F.S. are co-investigators. S.I.B. and D.A. are principal investigators of the PLCO study; G.A. is the principal investigator for the St. Louis screening center for PLCO; and S.J.C. and M.Y. led the genotyping for PLCO. S.G., R.B.H. and W.R.D. provided samples for PLCO. D.J.H. directs and P. Kraft coordinates data collection and management/analysis for HPFS. M.W. is the principal investigator of CPCS1 and CPCS2. B.G.N., S.F.N. S.E.B., P. Klarskov and M.A.R. have collected samples and data, and contributed to genotyping in this study. J.L.S. is principal investigator of the Fred Hutchinson–based study; E.A.O. collaborated on the study; L.M.F. and S.K. coordinated data collation; and E.M.K. and D.M.K. coordinated the preparation of samples. L.C.-A. is principal investigator of the Utah study; C.T. is the analyst; and R.A.S. is the surgeon. S.L. is a co-investigator of the BPC3 Consortium. H.G. is principal investigator of the CAPS and STHM1 study; J.A., M.A., F.W., S.L.Z. and J.X. have contributed to sample collection, clinical data retrieval, analyses and molecular work. S.A.I. is principal investigator of the USC study, and E.M.J. is principal investigator of SFPCS; M.C. Stem and R.C. led the genotyping of both studies. A.D.J. and A. Shahabi were both involved in genotype data production for the USC and SFPCS studies. A.S.K. is principal investigator of WUGS. B.D. and G.C. collected and collated clinical data and performed sample selection. M.R.T. is the principal investigator of the IPO-Porto study; S.M. and P.P. collected familial and molecular data on cases. L.B.S. and W.J.B. are the principal investigators of SCCS; L.B.S., W.J.B., W.Z. and Q.C. were responsible for the original collection of the samples. W.Z. and Q.C. coordinated sample retrieval, DNA extraction and genotyping. L.B.S. oversaw the assembly of the phenotype data. J.B. and J.A.C. are principal investigators of the Queensland study with input from A.B.S., F.L. and S.S. coordinated the data collation. K.A.C. and E.L. provided imputed data for genotyping in carriers of the mutation encoding the p.Gly84Glu alteration in the <em>HOXB13</em> region. G.G.G., J.L.H., D.R.E. and G.S. are principal investigators of the Australian studies; M.C. Southey manages the molecular work. J.S. is principal investigator of the Tampere study; T.W. collected and collated clinical data and performed sample selection. T.L.J.T. coordinated sample collection. H.B. is principal investigator of the ESTHER study; D.R. and C.S. contributed to design and data collection; and H.M. is study coordinator. J.Y.P. is principal investigator of the Moffitt study; T.A.S. and H.-Y.L. are contributors to this study. R. Kaneva is principal investigator of the PCMUS study; C.S. provided the samples in the PCMUS study; V.M. oversaw the data collation. C.C. and J.L. are principal investigators of the Poland study; C.C. and D.W. collated the samples. C.M. and W.V. are principal investigators of the Ulm study; A.E.R. identified and collected clinical material, processed samples, undertook genotyping and/or collated data. E.R. is principal investigator of EPIC; F.C., R. Kaaks and D. Campa are investigators in Germany. T.J.K. is principal investigator of the EPIC-Oxford cohort and collected clinical material. R.C.T. collated data. K.-T.K. is principal investigator of the EPIC-Norfolk study. S.N.T. and D.S. are principal investigators of the Mayo Clinic study; S.K.M. coordinated data collation. M.M.G. provided samples for the ACS study. P.D.P.P. and N.P. provided samples for the East Anglia SEARCH study. C.S.C. gave advice about results and contributed to the manuscript. A.C.A. undertook risk prediction analysis for clinical application. D.P.D., A.H., R.A.H., V.S.K., C.C.P., N.J.V.A., C.J.W., A.T., T. Dudderidge, C.O., A.A., A.C., J.V. and A. Siddiq identified and collected clinical material. Other members of the UK Genetic Prostate Cancer Study Collaborators/British Association of Urological Surgeons’ Section of Oncology, the UK ProtecT Study Collaborators and the PRACTICAL Consortium (membership lists provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>) collected clinical samples, assisted in genotyping and provided data management. Members of the COGS–Cancer Research UK GWAS–ELLIPSE (part of GAME-ON) Initiatives, the Australian Prostate Cancer Bioresource, the UK Genetic Prostate Cancer Study Collaborators/British Association of Urological Surgeons’ Section of Oncology, the UK ProtecT Study Collaborators, the PRACTICAL Consortium and CSC collected clinical samples and/or assisted in genotyping and/or provided data management and/or discussion of the data.</div><p class="fn sec" id="FN7"> </p><p id="__p8" class="p p-first"><strong>COMPETING FINANCIAL INTERESTS</strong></p><p id="__p9" class="p p-last">The authors declare no competing financial interests.</p><p></p></div></div><div id="idm139822325536208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139822325536208title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Goh CL, et al.  Genetic variants associated with predisposition to prostate cancer and potential clinical implications. <span><span class="ref-journal">J Intern Med. </span>2012;<span class="ref-vol">271</span>:353–365.</span> [<a href="/pubmed/22308973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Intern+Med&amp;title=Genetic+variants+associated+with+predisposition+to+prostate+cancer+and+potential+clinical+implications&amp;author=CL+Goh&amp;volume=271&amp;publication_year=2012&amp;pages=353-365&amp;pmid=22308973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Akamatsu S, et al.  Common variants at 11q12, 10q26 and 3p11.2 are associated with prostate cancer susceptibility in Japanese. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:426–429.</span> [<a href="/pubmed/22366784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+11q12,+10q26+and+3p11.2+are+associated+with+prostate+cancer+susceptibility+in+Japanese&amp;author=S+Akamatsu&amp;volume=44&amp;publication_year=2012&amp;pages=426-429&amp;pmid=22366784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Gudmundsson J, et al.  Genome-wide association and replication studies identify four variants associated with prostate cancer susceptibility. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1122–1126.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3562712/">PMC free article</a>]</span> [<a href="/pubmed/19767754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+and+replication+studies+identify+four+variants+associated+with+prostate+cancer+susceptibility&amp;author=J+Gudmundsson&amp;volume=41&amp;publication_year=2009&amp;pages=1122-1126&amp;pmid=19767754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Xu J, et al.  Genome-wide association study in Chinese men identifies two new prostate cancer risk loci at 9q31.2 and 19q13.4. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:1231–1235.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4116636/">PMC free article</a>]</span> [<a href="/pubmed/23023329" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+in+Chinese+men+identifies+two+new+prostate+cancer+risk+loci+at+9q31.2+and+19q13.4&amp;author=J+Xu&amp;volume=44&amp;publication_year=2012&amp;pages=1231-1235&amp;pmid=23023329&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Amin Al Olama A, et al.  A meta-analysis of genome-wide association studies to identify prostate cancer susceptibility loci associated with aggressive and non-aggressive disease. <span><span class="ref-journal">Hum Mol Genet. </span>2013;<span class="ref-vol">22</span>:408–415.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3526158/">PMC free article</a>]</span> [<a href="/pubmed/23065704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+meta-analysis+of+genome-wide+association+studies+to+identify+prostate+cancer+susceptibility+loci+associated+with+aggressive+and+non-aggressive+disease&amp;author=A+Amin+Al+Olama&amp;volume=22&amp;publication_year=2013&amp;pages=408-415&amp;pmid=23065704&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Gudmundsson J, et al.  A study based on whole-genome sequencing yields a rare variant at 8q24 associated with prostate cancer. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:1326–1329.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3562711/">PMC free article</a>]</span> [<a href="/pubmed/23104005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+study+based+on+whole-genome+sequencing+yields+a+rare+variant+at+8q24+associated+with+prostate+cancer&amp;author=J+Gudmundsson&amp;volume=44&amp;publication_year=2012&amp;pages=1326-1329&amp;pmid=23104005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Park JH, et al.  Estimation of effect size distribution from genome-wide association studies and implications for future discoveries. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:570–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4615599/">PMC free article</a>]</span> [<a href="/pubmed/20562874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Estimation+of+effect+size+distribution+from+genome-wide+association+studies+and+implications+for+future+discoveries&amp;author=JH+Park&amp;volume=42&amp;publication_year=2010&amp;pages=570-575&amp;pmid=20562874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Kote-Jarai Z, et al.  Identification of a novel prostate cancer susceptibility variant in the <em>KLK3</em> gene transcript. <span><span class="ref-journal">Hum Genet. </span>2011;<span class="ref-vol">129</span>:687–694.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3092928/">PMC free article</a>]</span> [<a href="/pubmed/21465221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Identification+of+a+novel+prostate+cancer+susceptibility+variant+in+the+KLK3+gene+transcript&amp;author=Z+Kote-Jarai&amp;volume=129&amp;publication_year=2011&amp;pages=687-694&amp;pmid=21465221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Kote-Jarai Z, et al.  Multiple novel prostate cancer predisposition loci confirmed by an international study: the PRACTICAL Consortium. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2008;<span class="ref-vol">17</span>:2052–2061.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2776652/">PMC free article</a>]</span> [<a href="/pubmed/18708398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Multiple+novel+prostate+cancer+predisposition+loci+confirmed+by+an+international+study:+the+PRACTICAL+Consortium&amp;author=Z+Kote-Jarai&amp;volume=17&amp;publication_year=2008&amp;pages=2052-2061&amp;pmid=18708398&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Koutros S, et al.  Pooled analysis of phosphatidylinositol 3-kinase pathway variants and risk of prostate cancer. <span><span class="ref-journal">Cancer Res. </span>2010;<span class="ref-vol">70</span>:2389–2396.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2840184/">PMC free article</a>]</span> [<a href="/pubmed/20197460" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Pooled+analysis+of+phosphatidylinositol+3-kinase+pathway+variants+and+risk+of+prostate+cancer&amp;author=S+Koutros&amp;volume=70&amp;publication_year=2010&amp;pages=2389-2396&amp;pmid=20197460&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Sun T, et al.  Single-nucleotide polymorphisms in p53 pathway and aggressiveness of prostate cancer in a Caucasian population. <span><span class="ref-journal">Clin Cancer Res. </span>2010;<span class="ref-vol">16</span>:5244–5251.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2970725/">PMC free article</a>]</span> [<a href="/pubmed/20855462" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Cancer+Res&amp;title=Single-nucleotide+polymorphisms+in+p53+pathway+and+aggressiveness+of+prostate+cancer+in+a+Caucasian+population&amp;author=T+Sun&amp;volume=16&amp;publication_year=2010&amp;pages=5244-5251&amp;pmid=20855462&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Wynendaele J, et al.  An illegitimate microRNA target site within the 3′ UTR of <em>MDM4</em> affects ovarian cancer progression and chemosensitivity. <span><span class="ref-journal">Cancer Res. </span>2010;<span class="ref-vol">70</span>:9641–9649.</span> [<a href="/pubmed/21084273" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=An+illegitimate+microRNA+target+site+within+the+3′+UTR+of+MDM4+affects+ovarian+cancer+progression+and+chemosensitivity&amp;author=J+Wynendaele&amp;volume=70&amp;publication_year=2010&amp;pages=9641-9649&amp;pmid=21084273&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">ENCODE Project Consortium. A user’s guide to the encyclopedia of DNA elements (ENCODE) <span><span class="ref-journal">PLoS Biol. </span>2011;<span class="ref-vol">9</span>:e1001046.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079585/">PMC free article</a>]</span> [<a href="/pubmed/21526222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=A+user’s+guide+to+the+encyclopedia+of+DNA+elements+(ENCODE)&amp;volume=9&amp;publication_year=2011&amp;pages=e1001046&amp;pmid=21526222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:57–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span> [<a href="/pubmed/22955616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome&amp;volume=489&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Garcia-Closas M, et al.  Genome-wide association studies identify four ER negative–specific breast cancer risk loci. <span><span class="ref-journal">Nat Genet. </span>2013 Mar 27; doi: 10.1038/ng.2561.</span> published online. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3771695/">PMC free article</a>]</span> [<a href="/pubmed/23535733" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2561" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+studies+identify+four+ER+negative–specific+breast+cancer+risk+loci&amp;author=M+Garcia-Closas&amp;publication_year=2013&amp;doi=10.1038/ng.2561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Couch FJ, et al.  Genome-wide association study in <em>BRCA1</em> mutation carriers identifies novel loci associated with breast and ovarian cancer risk. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003212.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3609646/">PMC free article</a>]</span> [<a href="/pubmed/23544013" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+in+BRCA1+mutation+carriers+identifies+novel+loci+associated+with+breast+and+ovarian+cancer+risk&amp;author=FJ+Couch&amp;volume=9&amp;publication_year=2013&amp;pages=e1003212&amp;pmid=23544013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Volinia S, et al.  A microRNA expression signature of human solid tumors defines cancer gene targets. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2006;<span class="ref-vol">103</span>:2257–2261.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1413718/">PMC free article</a>]</span> [<a href="/pubmed/16461460" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=A+microRNA+expression+signature+of+human+solid+tumors+defines+cancer+gene+targets&amp;author=S+Volinia&amp;volume=103&amp;publication_year=2006&amp;pages=2257-2261&amp;pmid=16461460&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Szarvas T, et al.  Elevated serum matrix metalloproteinase 7 levels predict poor prognosis after radical prostatectomy. <span><span class="ref-journal">Int J Cancer. </span>2011;<span class="ref-vol">128</span>:1486–1492.</span> [<a href="/pubmed/20473942" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=Elevated+serum+matrix+metalloproteinase+7+levels+predict+poor+prognosis+after+radical+prostatectomy&amp;author=T+Szarvas&amp;volume=128&amp;publication_year=2011&amp;pages=1486-1492&amp;pmid=20473942&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Richards TJ, et al.  Allele-specific transactivation of matrix metalloproteinase 7 by FOXA2 and correlation with plasma levels in idiopathic pulmonary fibrosis. <span><span class="ref-journal">Am J Physiol Lung Cell Mol Physiol. </span>2012;<span class="ref-vol">302</span>:L746–L754.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3331579/">PMC free article</a>]</span> [<a href="/pubmed/22268124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Lung+Cell+Mol+Physiol&amp;title=Allele-specific+transactivation+of+matrix+metalloproteinase+7+by+FOXA2+and+correlation+with+plasma+levels+in+idiopathic+pulmonary+fibrosis&amp;author=TJ+Richards&amp;volume=302&amp;publication_year=2012&amp;pages=L746-L754&amp;pmid=22268124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Thomas G, et al.  A multistage genome-wide association study in breast cancer identifies two new risk alleles at 1p11.2 and 14q24.1 (<em>RAD51L1</em>) <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:579–584.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928646/">PMC free article</a>]</span> [<a href="/pubmed/19330030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+multistage+genome-wide+association+study+in+breast+cancer+identifies+two+new+risk+alleles+at+1p11.2+and+14q24.1+(RAD51L1)&amp;author=G+Thomas&amp;volume=41&amp;publication_year=2009&amp;pages=579-584&amp;pmid=19330030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Michailidou K, et al.  Large-scale genotyping identifies 41 new loci associated with breast cancer risk. <span><span class="ref-journal">Nat Genet. </span>2013 Mar 27; doi: 10.1038/ng.2563.</span> published online. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3771688/">PMC free article</a>]</span> [<a href="/pubmed/23535729" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2563" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Large-scale+genotyping+identifies+41+new+loci+associated+with+breast+cancer+risk&amp;author=K+Michailidou&amp;publication_year=2013&amp;doi=10.1038/ng.2563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Haiman CA, et al.  Genome-wide association study of prostate cancer in men of African ancestry identifies a susceptibility locus at 17q21. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:570–573.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3102788/">PMC free article</a>]</span> [<a href="/pubmed/21602798" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+prostate+cancer+in+men+of+African+ancestry+identifies+a+susceptibility+locus+at+17q21&amp;author=CA+Haiman&amp;volume=43&amp;publication_year=2011&amp;pages=570-573&amp;pmid=21602798&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Callen DF, et al.  Co-expression of the androgen receptor and the transcription factor ZNF652 is related to prostate cancer outcome. <span><span class="ref-journal">Oncol Rep. </span>2010;<span class="ref-vol">23</span>:1045–1052.</span> [<a href="/pubmed/20204290" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol+Rep&amp;title=Co-expression+of+the+androgen+receptor+and+the+transcription+factor+ZNF652+is+related+to+prostate+cancer+outcome&amp;author=DF+Callen&amp;volume=23&amp;publication_year=2010&amp;pages=1045-1052&amp;pmid=20204290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Norris JD, et al.  The homeodomain protein HOXB13 regulates the cellular response to androgens. <span><span class="ref-journal">Mol Cell. </span>2009;<span class="ref-vol">36</span>:405–416.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2788777/">PMC free article</a>]</span> [<a href="/pubmed/19917249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=The+homeodomain+protein+HOXB13+regulates+the+cellular+response+to+androgens&amp;author=JD+Norris&amp;volume=36&amp;publication_year=2009&amp;pages=405-416&amp;pmid=19917249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Ewing CM, et al.  Germline mutations in <em>HOXB13</em> and prostate-cancer risk. <span><span class="ref-journal">N Engl J Med. </span>2012;<span class="ref-vol">366</span>:141–149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3779870/">PMC free article</a>]</span> [<a href="/pubmed/22236224" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Germline+mutations+in+HOXB13+and+prostate-cancer+risk&amp;author=CM+Ewing&amp;volume=366&amp;publication_year=2012&amp;pages=141-149&amp;pmid=22236224&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Edwards S, et al.  Expression analysis onto microarrays of randomly selected cDNA clones highlights <em>HOXB13</em> as a marker of human prostate cancer. <span><span class="ref-journal">Br J Cancer. </span>2005;<span class="ref-vol">92</span>:376–381.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2361840/">PMC free article</a>]</span> [<a href="/pubmed/15583692" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Expression+analysis+onto+microarrays+of+randomly+selected+cDNA+clones+highlights+HOXB13+as+a+marker+of+human+prostate+cancer&amp;author=S+Edwards&amp;volume=92&amp;publication_year=2005&amp;pages=376-381&amp;pmid=15583692&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Barbieri CE, et al.  Exome sequencing identifies recurrent <em>SPOP</em>, <em>FOXA1</em> and <em>MED12</em> mutations in prostate cancer. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:685–689.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3673022/">PMC free article</a>]</span> [<a href="/pubmed/22610119" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Exome+sequencing+identifies+recurrent+SPOP,+FOXA1+and+MED12+mutations+in+prostate+cancer&amp;author=CE+Barbieri&amp;volume=44&amp;publication_year=2012&amp;pages=685-689&amp;pmid=22610119&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Breast Cancer Linkage Consortium. Cancer risks in <em>BRCA2</em> mutation carriers. The Breast Cancer Linkage Consortium. <span><span class="ref-journal">J Natl Cancer Inst. </span>1999;<span class="ref-vol">91</span>:1310–1316.</span> [<a href="/pubmed/10433620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Cancer+risks+in+BRCA2+mutation+carriers.+The+Breast+Cancer+Linkage+Consortium&amp;volume=91&amp;publication_year=1999&amp;pages=1310-1316&amp;pmid=10433620&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Macinnis RJ, et al.  A risk prediction algorithm based on family history and common genetic variants: application to prostate cancer with potential clinical impact. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:549–556.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3950816/">PMC free article</a>]</span> [<a href="/pubmed/21769933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=A+risk+prediction+algorithm+based+on+family+history+and+common+genetic+variants:+application+to+prostate+cancer+with+potential+clinical+impact&amp;author=RJ+Macinnis&amp;volume=35&amp;publication_year=2011&amp;pages=549-556&amp;pmid=21769933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Schumacher FR, et al.  Genome-wide association study identifies new prostate cancer susceptibility loci. <span><span class="ref-journal">Hum Mol Genet. </span>2011;<span class="ref-vol">20</span>:3867–3875.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3168287/">PMC free article</a>]</span> [<a href="/pubmed/21743057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+study+identifies+new+prostate+cancer+susceptibility+loci&amp;author=FR+Schumacher&amp;volume=20&amp;publication_year=2011&amp;pages=3867-3875&amp;pmid=21743057&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Marchini J, et al.  A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Eeles RA, et al.  Multiple newly identified loci associated with prostate cancer susceptibility. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:316–321.</span> [<a href="/pubmed/18264097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+newly+identified+loci+associated+with+prostate+cancer+susceptibility&amp;author=RA+Eeles&amp;volume=40&amp;publication_year=2008&amp;pages=316-321&amp;pmid=18264097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Aulchenko YS, Struchalin MV, van Duijn CM. ProbABEL package for genome-wide association analysis of imputed data. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:134.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2846909/">PMC free article</a>]</span> [<a href="/pubmed/20233392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=ProbABEL+package+for+genome-wide+association+analysis+of+imputed+data&amp;author=YS+Aulchenko&amp;author=MV+Struchalin&amp;author=CM+van+Duijn&amp;volume=11&amp;publication_year=2010&amp;pages=134&amp;pmid=20233392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Teo YY, et al.  A genotype calling algorithm for the Illumina BeadArray platform. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:2741–2746.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666488/">PMC free article</a>]</span> [<a href="/pubmed/17846035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+genotype+calling+algorithm+for+the+Illumina+BeadArray+platform&amp;author=YY+Teo&amp;volume=23&amp;publication_year=2007&amp;pages=2741-2746&amp;pmid=17846035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Giannoulatou E, et al.  GenoSNP: a variational Bayes within-sample SNP genotyping algorithm that does not require a reference population. <span><span class="ref-journal">Bioinformatics. </span>2008;<span class="ref-vol">24</span>:2209–2214.</span> [<a href="/pubmed/18653518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenoSNP:+a+variational+Bayes+within-sample+SNP+genotyping+algorithm+that+does+not+require+a+reference+population&amp;author=E+Giannoulatou&amp;volume=24&amp;publication_year=2008&amp;pages=2209-2214&amp;pmid=18653518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Haldane JB, Slater E. Assortative mating. <span><span class="ref-journal">Eugen Rev. </span>1946;<span class="ref-vol">38</span>:103.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2986343/">PMC free article</a>]</span> [<a href="/pubmed/20998131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eugen+Rev&amp;title=Assortative+mating&amp;author=JB+Haldane&amp;author=E+Slater&amp;volume=38&amp;publication_year=1946&amp;pages=103&amp;pmid=20998131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Aulchenko YS, Ripke S, Isaacs A, van Duijn CM. GenABEL: an R library for genome-wide association analysis. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1294–1296.</span> [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenABEL:+an+R+library+for+genome-wide+association+analysis&amp;author=YS+Aulchenko&amp;author=S+Ripke&amp;author=A+Isaacs&amp;author=CM+van+Duijn&amp;volume=23&amp;publication_year=2007&amp;pages=1294-1296&amp;pmid=17384015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832790&amp;issue-id=229905&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139822322370800title">Abstract</a></li><li><a href="#S1title">URLs</a></li><li><a href="#S3title">ONLINE METHODS</a></li><li><a href="#S12title">Supplementary Material</a></li><li><a href="#S13title">Acknowledgments</a></li><li><a href="#idm139822325551040title">Footnotes</a></li><li><a href="#idm139822325536208title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3832790/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3832790/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3832790/pdf/nihms498644.pdf">PDF (1.5M)</a></li> | <li><a href="#" data-citationid="PMC3832790" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832790%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832790%2F&amp;text=Identification%20of%2023%20new%20prostate%20cancer%20susceptibility%20loci%20using%20the%20iCOGS%20custom%20genotyping%20array" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832790%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23535732" data-aiid="3832790" data-aid="3832790" data-iid="229905" data-domainid="319" data-domain="nihpa" data-accid="PMC3832790" data-md5="a574bd820fb3893f4118cac0deb1e0e9"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:34:41-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE884665DD5534410000000007920523&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832790%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3832790/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Main, sans-serif;"></div></div></body></html>