<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3303194/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies five new schizophrenia loci</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3303194/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature Genetics"><meta name="citation_title" content="Genome-wide association study identifies five new schizophrenia loci"><meta name="citation_authors" content="The Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium"><meta name="citation_issue" content="10"><meta name="citation_volume" content="43"><meta name="citation_firstpage" content="969"><meta name="citation_doi" content="10.1038/ng.940"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3303194/?report=abstract"><meta name="citation_pmid" content="21926974"><meta name="DC.Title" content="Genome-wide association study identifies five new schizophrenia loci"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="The Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium"><meta name="DC.Identifier" content="10.1038/ng.940"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies five new schizophrenia loci"><meta property="og:type" content="article"><meta property="og:description" content="We examined the role of common genetic variation in schizophrenia in a genome-wide association study of substantial size: a stage 1 discovery sample of 21,856 individuals of European ancestry and a stage 2 replication sample of 29,839 independent subjects. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3303194/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3303194/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3303194/pdf/nihms360727.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D4174DD534CB10000000000500038.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3303194%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3303194</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2012 Oct 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21926974" target="pmc_ext" ref="reftype=publisher&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2011 Sep 18; 43(10): 969–976. </span></a></div><span class="fm-vol-iss-date">Published online 2011 Sep 18. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.940" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.940</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3303194</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS360727</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21926974">21926974</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies five new schizophrenia loci</h1><div class="half_rhythm"><div class="contrib-group fm-author">The Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium<sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140107430699536_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140107430699536_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140107430699536_ai" style="display:none"><div id="FN1"><sup>1</sup>A full list of authors and affiliations appears at the end of the paper.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140107430699536_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21926974" target="pmc_ext" ref="reftype=publisher&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3303194/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Note.</strong></div><div><a href="/pmc/articles/PMC3303194/bin/NIHMS360727-supplement-supplementary.pdf" data-ga-action="click_feat_suppl">NIHMS360727-supplement-supplementary.pdf</a><span style="color:gray"> (26M)</span></div><div class="small guid">GUID:&nbsp;5BFB2799-BAE0-43F1-A74D-858CA2CD406F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Online Methods.</strong></div><div><a href="/pmc/articles/PMC3303194/bin/NIHMS360727-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">NIHMS360727-supplement-supplement_1.pdf</a><span style="color:gray"> (108K)</span></div><div class="small guid">GUID:&nbsp;2CB38ABF-9E67-497B-99A3-52821A1D3A74</div></div></dd></dl></div><div id="idm140107433757056" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140107433757056title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">We examined the role of common genetic variation in schizophrenia in a genome-wide association study of substantial size: a stage 1 discovery sample of 21,856 individuals of European ancestry and a stage 2 replication sample of 29,839 independent subjects. The combined stage 1 and 2 analysis yielded genome-wide significant associations with schizophrenia for seven loci, five of which are new (1p21.3, 2q32.3, 8p23.2, 8q21.3 and 10q24.32-q24.33) and two of which have been previously implicated (6p21.32-p22.1 and 18q21.2). The strongest new finding (<em>P</em> = 1.6 × 10<sup>−11</sup>) was with rs1625579 within an intron of a putative primary transcript for <em>MIR137</em> (microRNA 137), a known regulator of neuronal development. Four other schizophrenia loci achieving genome-wide significance contain predicted targets of <em>MIR137</em>, suggesting <em>MIR137</em>-mediated dysregulation as a previously unknown etiologic mechanism in schizophrenia. In a joint analysis with a bipolar disorder sample (16,374 affected individuals and 14,044 controls), three loci reached genome-wide significance: <em>CACNA1C</em> (rs4765905, <em>P</em> = 7.0 × 10<sup>−9</sup>), <em>ANK3</em> (rs10994359, <em>P</em> = 2.5 × 10<sup>−8</sup>) and the <em>ITIH3-ITIH4</em> region (rs2239547, <em>P</em> = 7.8 × 10<sup>−9</sup>).</p></div></div><div id="S1" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">In stage 1, we conducted a mega-analysis combining genome-wide assocation study (GWAS) data from 17 separate studies (with a total of 9,394 cases and 12,462 controls; <a href="/pmc/articles/PMC3303194/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 1,2</a>). We imputed allelic dosages for 1,252,901 autosomal SNPs (<a href="/pmc/articles/PMC3303194/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>) using HapMap3 as the reference panel<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886357">1</a></sup>. We tested for association using logistic regression of imputed dosages with sample identifiers and three principal components as covariates to minimize inflation in significance testing caused by population stratification. The quantile-quantile plot (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>) deviated from the null distribution with a population stratification inflation factor of <em>λ</em> = 1.23. However, <em>λ</em><sub>1000</sub>, a metric that standardizes the degree of inflation by sample size, was only 1.02, similar to that observed in other GWAS meta-analyses<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886362">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886335">3</a></sup>. This deviation persisted despite comprehensive quality control and inclusion of up to 20 principal components (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). Thus, we interpret this deviation as indicative of a large number of weakly associated SNPs consistent with polygenic inheritance<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373102529">4</a></sup>. We also examined 298 ancestry-informative markers (AIMs) that reflect European-ancestry population substructure<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886349">5</a></sup>. Unadjusted analyses showed greater inflation in the test statistics than we saw for all markers (AIMs <em>λ</em> = 2.26 compared to all markers <em>λ</em> = 1.56). After inclusion of principal components, the distributions of the test statistics did not differ between AIMs (<em>λ</em> = 1.18) and all markers (<em>λ</em> = 1.23), a result inconsistent with population stratification explaining the residual deviation seen in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>. Moreover, the results of a meta-analysis using summary results generated using study specific principal components (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>) were highly correlated with those from the mega-analysis (Pearson correlation = 0.94, with a similar <em>λ</em> = 1.20; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). Of the ten SNPs in <a href="/pmc/articles/PMC3303194/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, four increased and six decreased in significance, suggesting that the most extreme values did not result from systematic inflation artifacts. Therefore, our primary analysis used unadjusted <em>P</em> values (nevertheless, see <a href="/pmc/articles/PMC3303194/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> for stage 1 <em>P</em> values adjusted for <em>λ</em> (ref. <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886358">6</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p143">Study design and samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140107422804288" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">Collection</th><th rowspan="2" colspan="1" valign="bottom" align="left">Country</th><th rowspan="2" colspan="1" valign="bottom" align="left">Platform</th><th colspan="4" rowspan="1" valign="bottom" align="center">Cases included by sex
<hr></th><th colspan="4" rowspan="1" valign="bottom" align="center">Controls included by sex
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="right">Male</th><th rowspan="1" colspan="1" valign="bottom" align="right">Female</th><th rowspan="1" colspan="1" valign="bottom" align="right">Unknown</th><th rowspan="1" colspan="1" valign="bottom" align="right">Total</th><th rowspan="1" colspan="1" valign="bottom" align="right">Male</th><th rowspan="1" colspan="1" valign="bottom" align="right">Female</th><th rowspan="1" colspan="1" valign="bottom" align="right">Unknown</th><th rowspan="1" colspan="1" valign="bottom" align="right">Total</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Cardiff UK</td><td rowspan="1" colspan="1" valign="top" align="left">UK</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 500K</td><td rowspan="1" colspan="1" valign="top" align="right">320</td><td rowspan="1" colspan="1" valign="top" align="right">152</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">472</td><td rowspan="1" colspan="1" valign="top" align="right">1,442</td><td rowspan="1" colspan="1" valign="top" align="right">1,492</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">2,934</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">CATIE</td><td rowspan="1" colspan="1" valign="top" align="left">United States</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 500K; Perlegen 164K</td><td rowspan="1" colspan="1" valign="top" align="right">308</td><td rowspan="1" colspan="1" valign="top" align="right">94</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">402</td><td rowspan="1" colspan="1" valign="top" align="right">161</td><td rowspan="1" colspan="1" valign="top" align="right">46</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">207</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISC-Aberdeen</td><td rowspan="1" colspan="1" valign="top" align="left">UK</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0</td><td rowspan="1" colspan="1" valign="top" align="right">536</td><td rowspan="1" colspan="1" valign="top" align="right">184</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">720</td><td rowspan="1" colspan="1" valign="top" align="right">447</td><td rowspan="1" colspan="1" valign="top" align="right">251</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">698</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISC-Cardiff</td><td rowspan="1" colspan="1" valign="top" align="left">Bulgaria</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">270</td><td rowspan="1" colspan="1" valign="top" align="right">257</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">527</td><td rowspan="1" colspan="1" valign="top" align="right">291</td><td rowspan="1" colspan="1" valign="top" align="right">318</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">609</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISC-Dublin</td><td rowspan="1" colspan="1" valign="top" align="left">Ireland</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">188</td><td rowspan="1" colspan="1" valign="top" align="right">82</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">270</td><td rowspan="1" colspan="1" valign="top" align="right">258</td><td rowspan="1" colspan="1" valign="top" align="right">602</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">860</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISC-Edinburgh</td><td rowspan="1" colspan="1" valign="top" align="left">UK</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">267</td><td rowspan="1" colspan="1" valign="top" align="right">101</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">368</td><td rowspan="1" colspan="1" valign="top" align="right">146</td><td rowspan="1" colspan="1" valign="top" align="right">138</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">284</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISC-London</td><td rowspan="1" colspan="1" valign="top" align="left">UK</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0; Affymetrix 500K</td><td rowspan="1" colspan="1" valign="top" align="right">369</td><td rowspan="1" colspan="1" valign="top" align="right">149</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">518</td><td rowspan="1" colspan="1" valign="top" align="right">207</td><td rowspan="1" colspan="1" valign="top" align="right">284</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">491</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISC-Portugal</td><td rowspan="1" colspan="1" valign="top" align="left">Portugal</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0</td><td rowspan="1" colspan="1" valign="top" align="right">213</td><td rowspan="1" colspan="1" valign="top" align="right">133</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">346</td><td rowspan="1" colspan="1" valign="top" align="right">80</td><td rowspan="1" colspan="1" valign="top" align="right">135</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">215</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISC-SW1</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0</td><td rowspan="1" colspan="1" valign="top" align="right">93</td><td rowspan="1" colspan="1" valign="top" align="right">75</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">168</td><td rowspan="1" colspan="1" valign="top" align="right">82</td><td rowspan="1" colspan="1" valign="top" align="right">85</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">167</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISC-SW2</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">231</td><td rowspan="1" colspan="1" valign="top" align="right">159</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">390</td><td rowspan="1" colspan="1" valign="top" align="right">116</td><td rowspan="1" colspan="1" valign="top" align="right">113</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">229</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MGS</td><td rowspan="1" colspan="1" valign="top" align="left">United States, Australia</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">1,863</td><td rowspan="1" colspan="1" valign="top" align="right">816</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">2,679</td><td rowspan="1" colspan="1" valign="top" align="right">1,140</td><td rowspan="1" colspan="1" valign="top" align="right">1,344</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">2,484</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Bonn</td><td rowspan="1" colspan="1" valign="top" align="left">Germany</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 550K</td><td rowspan="1" colspan="1" valign="top" align="right">238</td><td rowspan="1" colspan="1" valign="top" align="right">236</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">474</td><td rowspan="1" colspan="1" valign="top" align="right">664</td><td rowspan="1" colspan="1" valign="top" align="right">640</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">1,304</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Copenhagen</td><td rowspan="1" colspan="1" valign="top" align="left">Denmark</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina Human 610-Quad</td><td rowspan="1" colspan="1" valign="top" align="right">280</td><td rowspan="1" colspan="1" valign="top" align="right">202</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">482</td><td rowspan="1" colspan="1" valign="top" align="right">268</td><td rowspan="1" colspan="1" valign="top" align="right">189</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">457</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Munich</td><td rowspan="1" colspan="1" valign="top" align="left">Germany</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 300K</td><td rowspan="1" colspan="1" valign="top" align="right">279</td><td rowspan="1" colspan="1" valign="top" align="right">155</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">434</td><td rowspan="1" colspan="1" valign="top" align="right">167</td><td rowspan="1" colspan="1" valign="top" align="right">184</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">351</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-TOP3</td><td rowspan="1" colspan="1" valign="top" align="left">Norway</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">132</td><td rowspan="1" colspan="1" valign="top" align="right">116</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">248</td><td rowspan="1" colspan="1" valign="top" align="right">176</td><td rowspan="1" colspan="1" valign="top" align="right">175</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">351</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-UCLA</td><td rowspan="1" colspan="1" valign="top" align="left">The Netherlands</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 550K</td><td rowspan="1" colspan="1" valign="top" align="right">529</td><td rowspan="1" colspan="1" valign="top" align="right">175</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">704</td><td rowspan="1" colspan="1" valign="top" align="right">310</td><td rowspan="1" colspan="1" valign="top" align="right">321</td><td rowspan="1" colspan="1" valign="top" align="right">–</td><td rowspan="1" colspan="1" valign="top" align="right">631</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Zucker Hillside</td><td rowspan="1" colspan="1" valign="top" align="left">United States</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 500K</td><td rowspan="1" colspan="1" valign="top" align="right">128</td><td rowspan="1" colspan="1" valign="top" align="right">64</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">192</td><td rowspan="1" colspan="1" valign="top" align="right">92</td><td rowspan="1" colspan="1" valign="top" align="right">98</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">190</td></tr><tr><td colspan="2" rowspan="1" valign="top" align="left"><strong>Grand totals for the GWAs</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>6,244</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>3,150</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>0</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>9,394</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>6,047</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>6,415</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>–</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>12,462</strong></td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Multicenter Pedigree</td><td rowspan="1" colspan="1" valign="top" align="left">Europe, United States, Australia</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina Human 610-Quad</td><td rowspan="1" colspan="1" valign="top" align="right">n.a.</td><td rowspan="1" colspan="1" valign="top" align="right">n.a.</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">583</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Aarhus</td><td rowspan="1" colspan="1" valign="top" align="left">Denmark</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina Human 610-Quad</td><td rowspan="1" colspan="1" valign="top" align="right">477</td><td rowspan="1" colspan="1" valign="top" align="right">399</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">876</td><td rowspan="1" colspan="1" valign="top" align="right">477</td><td rowspan="1" colspan="1" valign="top" align="right">397</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">874</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Aarhus</td><td rowspan="1" colspan="1" valign="top" align="left">Denmark</td><td rowspan="1" colspan="1" valign="top" align="left">Centaurus</td><td rowspan="1" colspan="1" valign="top" align="right">114</td><td rowspan="1" colspan="1" valign="top" align="right">102</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">217</td><td rowspan="1" colspan="1" valign="top" align="right">176</td><td rowspan="1" colspan="1" valign="top" align="right">317</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">493</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Belgium</td><td rowspan="1" colspan="1" valign="top" align="left">Belgium</td><td rowspan="1" colspan="1" valign="top" align="left">Centaurus; Illumina 370K</td><td rowspan="1" colspan="1" valign="top" align="right">326</td><td rowspan="1" colspan="1" valign="top" align="right">184</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">510</td><td rowspan="1" colspan="1" valign="top" align="right">149</td><td rowspan="1" colspan="1" valign="top" align="right">192</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">341</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Copenhagen</td><td rowspan="1" colspan="1" valign="top" align="left">Denmark</td><td rowspan="1" colspan="1" valign="top" align="left">Centaurus</td><td rowspan="1" colspan="1" valign="top" align="right">264</td><td rowspan="1" colspan="1" valign="top" align="right">198</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">462</td><td rowspan="1" colspan="1" valign="top" align="right">499</td><td rowspan="1" colspan="1" valign="top" align="right">375</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">874</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Iceland</td><td rowspan="1" colspan="1" valign="top" align="left">Iceland</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 300K</td><td rowspan="1" colspan="1" valign="top" align="right">346</td><td rowspan="1" colspan="1" valign="top" align="right">185</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">531</td><td rowspan="1" colspan="1" valign="top" align="right">5,802</td><td rowspan="1" colspan="1" valign="top" align="right">5,813</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">11,615</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-England</td><td rowspan="1" colspan="1" valign="top" align="left">UK</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 300K</td><td rowspan="1" colspan="1" valign="top" align="right">71</td><td rowspan="1" colspan="1" valign="top" align="right">22</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">93</td><td rowspan="1" colspan="1" valign="top" align="right">48</td><td rowspan="1" colspan="1" valign="top" align="right">40</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">88</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Helsinki</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 300K</td><td rowspan="1" colspan="1" valign="top" align="right">112</td><td rowspan="1" colspan="1" valign="top" align="right">70</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">59</td><td rowspan="1" colspan="1" valign="top" align="right">122</td><td rowspan="1" colspan="1" valign="top" align="right">75</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">147</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Kuusamo</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 300K</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right">123</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right">50</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Hungary</td><td rowspan="1" colspan="1" valign="top" align="left">Hungary</td><td rowspan="1" colspan="1" valign="top" align="left">Centaurus</td><td rowspan="1" colspan="1" valign="top" align="right">105</td><td rowspan="1" colspan="1" valign="top" align="right">136</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">241</td><td rowspan="1" colspan="1" valign="top" align="right">89</td><td rowspan="1" colspan="1" valign="top" align="right">125</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">214</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Italy</td><td rowspan="1" colspan="1" valign="top" align="left">Italy</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 300K</td><td rowspan="1" colspan="1" valign="top" align="right">48</td><td rowspan="1" colspan="1" valign="top" align="right">36</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">84</td><td rowspan="1" colspan="1" valign="top" align="right">50</td><td rowspan="1" colspan="1" valign="top" align="right">39</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">89</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Munich</td><td rowspan="1" colspan="1" valign="top" align="left">Germany</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 300K</td><td rowspan="1" colspan="1" valign="top" align="right">280</td><td rowspan="1" colspan="1" valign="top" align="right">186</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">163</td><td rowspan="1" colspan="1" valign="top" align="right">887</td><td rowspan="1" colspan="1" valign="top" align="right">912</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">185</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Munich</td><td rowspan="1" colspan="1" valign="top" align="left">Germany</td><td rowspan="1" colspan="1" valign="top" align="left">Centaurus</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right">303</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right">1,614</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Russia</td><td rowspan="1" colspan="1" valign="top" align="left">Russia</td><td rowspan="1" colspan="1" valign="top" align="left">Centaurus</td><td rowspan="1" colspan="1" valign="top" align="right">132</td><td rowspan="1" colspan="1" valign="top" align="right">343</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">475</td><td rowspan="1" colspan="1" valign="top" align="right">178</td><td rowspan="1" colspan="1" valign="top" align="right">290</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">468</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SGENE-Sweden</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden</td><td rowspan="1" colspan="1" valign="top" align="left">Centaurus</td><td rowspan="1" colspan="1" valign="top" align="right">158</td><td rowspan="1" colspan="1" valign="top" align="right">94</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">252</td><td rowspan="1" colspan="1" valign="top" align="right">178</td><td rowspan="1" colspan="1" valign="top" align="right">109</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">287</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SW3</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">327</td><td rowspan="1" colspan="1" valign="top" align="right">212</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">539</td><td rowspan="1" colspan="1" valign="top" align="right">457</td><td rowspan="1" colspan="1" valign="top" align="right">448</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">905</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SW4</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">656</td><td rowspan="1" colspan="1" valign="top" align="right">407</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">1,063</td><td rowspan="1" colspan="1" valign="top" align="right">605</td><td rowspan="1" colspan="1" valign="top" align="right">568</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">1,173</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">UQ and ASRB</td><td rowspan="1" colspan="1" valign="top" align="left">Australia</td><td rowspan="1" colspan="1" valign="top" align="left">SequenomMassArray</td><td rowspan="1" colspan="1" valign="top" align="right">347</td><td rowspan="1" colspan="1" valign="top" align="right">190</td><td rowspan="1" colspan="1" valign="top" align="right">21</td><td rowspan="1" colspan="1" valign="top" align="right">558</td><td rowspan="1" colspan="1" valign="top" align="right">487</td><td rowspan="1" colspan="1" valign="top" align="right">455</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="right">957</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISGC and WTCCC2</td><td rowspan="1" colspan="1" valign="top" align="left">Ireland</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td><td rowspan="1" colspan="1" valign="top" align="right">968</td><td rowspan="1" colspan="1" valign="top" align="right">342</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">1,310</td><td rowspan="1" colspan="1" valign="top" align="right">245</td><td rowspan="1" colspan="1" valign="top" align="right">778</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">1,023</td></tr><tr><td colspan="2" rowspan="1" valign="top" align="left"><strong>Grand totals for the replication follow up</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>4,731</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>3,106</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>22</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>8,442</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>10,449</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>10,933</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>15</strong></td><td rowspan="1" colspan="1" valign="top" align="right"><strong>21,397</strong></td></tr></tbody></table></div><div id="largeobj_idm140107422804288" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3303194/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p144" class="p p-first-last">Stage 1 describes the 17 samples that provided full GWAS genotyping data, and stage 2 describes the 19 studies that provided results for the top SNPs identified in the combined analysis of stage 1 studies. Stage 2 replication SGENE-Belgium had four cases missing sex information. Stage 2 replication SGENE-Aarhus (focused genotyping sample) had one case missing sex information. Stage 2 replication-University of Queensland had 21 cases and 15 controls missing sex information. Sex information for the two stage 2 replication SGENE-Munich samples are combined. Sex information for the two stage 2 replication SGENE-Finnish (Helsinki and Kuusamo) samples are combined to enable that these two samples are located adjacent to each other in the table (rather than alphabetically). Multicenter Pedigree was a family sample, and so case sex counts are not applicable (n.a). SGENE, Schizophrenia Genetics Consortium; ISC, International Schizophrenia Consortium; TOP3, Thematic Organized Psychoses Research 3; UCLA, University of California at Los Angeles; SW1, Sweden 1; SW2, Sweden 2; WTCCC, Wellcome Trust case Control Consortium; for the Multicenter Pedigree study, the number of cases indicates the number of families; CATIE, Clinical Antipsychotic Trials of Intervention Effectiveness; MGS, Molecular Genetics of Schizophrenia; UQ, University of Queensland; ASRB, Australian Schizophrenia Research Bank; ISGC, Irish Schizophrenia Genomics Consortium.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p145">Top genome-wide association results for schizophrenia</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140107424867728" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">Chr.</th><th rowspan="1" colspan="1" valign="bottom" align="right">Mb</th><th rowspan="1" colspan="1" valign="bottom" align="center">Alleles</th><th rowspan="1" colspan="1" valign="bottom" align="center">Frequency</th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>P</em> (GC-adjusted <em>P</em> )</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Consistency of direction</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="right">Distance (kb)</th></tr></thead><tbody><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs1625579</td><td rowspan="3" colspan="1" valign="middle" align="left">1p21.3<sup>a</sup></td><td rowspan="3" colspan="1" valign="middle" align="right">98.3</td><td rowspan="3" colspan="1" valign="middle" align="center">TG</td><td rowspan="3" colspan="1" valign="middle" align="center">0.80</td><td rowspan="1" colspan="1" valign="top" align="right">5.72 × 10<sup>−7</sup> (6.52 × 10<sup>−6</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.14 (1.08–1.19)</td><td rowspan="3" colspan="1" valign="middle" align="center">+−++++−+</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>MIR137</em></td><td rowspan="3" colspan="1" valign="middle" align="right">Intragenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.65 × 10<sup>−6</sup></strong> (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.11 (1.07–1.16)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.59 × 10<sup>−11</sup></strong> (<strong>6.87 × 10<sup>−10</sup></strong>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.12 (1.09–1.16)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs17662626</td><td rowspan="3" colspan="1" valign="middle" align="left">2q32.3<sup>a</sup></td><td rowspan="3" colspan="1" valign="middle" align="right">193.7</td><td rowspan="3" colspan="1" valign="middle" align="center">AG</td><td rowspan="3" colspan="1" valign="middle" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="right">3.09 × 10<sup>−6</sup> (2.60 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.22 (1.13–1.30)</td><td rowspan="3" colspan="1" valign="middle" align="center">+ −+ +++</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>PCGEM1</em></td><td rowspan="3" colspan="1" valign="middle" align="right">343</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.70 × 10<sup>−3</sup></strong> (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.06–1.27)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>4.65 × 10<sup>−8</sup></strong> (1.25 × 10<sup>−6</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.13–1.26)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs2021722</td><td rowspan="3" colspan="1" valign="middle" align="left">6p21.3-p22.1</td><td rowspan="3" colspan="1" valign="middle" align="right">30.3</td><td rowspan="3" colspan="1" valign="middle" align="center">CT</td><td rowspan="3" colspan="1" valign="middle" align="center">0.78</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>4.30 × 10<sup>−11</sup> (2.76 × 10<sup>−9</sup>)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.18 (1.13–1.23)</td><td rowspan="3" colspan="1" valign="middle" align="center">+ ++ −++</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>TRIM26</em></td><td rowspan="3" colspan="1" valign="middle" align="right">Intragenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.55 × 10<sup>−3</sup></strong> (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.10 (1.03–1.17)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.18 × 10<sup>−12</sup> (2.88 × 10<sup>−10</sup></strong>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.15 (1.11–1.19)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs10503253</td><td rowspan="3" colspan="1" valign="middle" align="left">8p23.2<sup>a</sup></td><td rowspan="3" colspan="1" valign="middle" align="right">4.2</td><td rowspan="3" colspan="1" valign="middle" align="center">AC</td><td rowspan="3" colspan="1" valign="middle" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="right">3.84 × 10<sup>−7</sup> (4.71 × 10<sup>−6</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.14 (1.09–1.19)</td><td rowspan="3" colspan="1" valign="middle" align="center">+ ++ +−+</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>CSMD1</em></td><td rowspan="3" colspan="1" valign="middle" align="right">Intragenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>7.60 × 10<sup>−3</sup></strong> (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (1.01–1.14)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>4.14 × 10<sup>−8</sup></strong> (8.98 × 10<sup>−7</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.11 (1.07–1.15)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs7004633</td><td rowspan="3" colspan="1" valign="middle" align="left">8q21.3<sup>a</sup></td><td rowspan="3" colspan="1" valign="middle" align="right">89.8</td><td rowspan="3" colspan="1" valign="middle" align="center">GA</td><td rowspan="3" colspan="1" valign="middle" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.45 × 10<sup>−8</sup></strong> (3.22 × 10<sup>−7</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.11–1.21)</td><td rowspan="3" colspan="1" valign="middle" align="center">++−+++−+</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>MMP16</em></td><td rowspan="3" colspan="1" valign="middle" align="right">421</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>0.011</strong> (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.05 (1.01–1.10)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.75 × 10<sup>−8</sup></strong> (7.03 × 10<sup>−7</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.10 (1.07–1.14)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs7914558</td><td rowspan="3" colspan="1" valign="middle" align="left">10q24.32<sup>a</sup></td><td rowspan="3" colspan="1" valign="middle" align="right">104.8</td><td rowspan="3" colspan="1" valign="middle" align="center">GA</td><td rowspan="3" colspan="1" valign="middle" align="center">0.59</td><td rowspan="1" colspan="1" valign="top" align="right">1.58 × 10<sup>−7</sup> (2.27 × 10<sup>−6</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.11 (1.07–1.15)</td><td rowspan="3" colspan="1" valign="middle" align="center">+ ++ +++</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>CNNM2</em></td><td rowspan="3" colspan="1" valign="middle" align="right">Intragenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.07 × 10<sup>−3</sup></strong> (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (1.03–1.13)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.82 × 10<sup>−9</sup></strong> (<strong>3.11 × 10<sup>−8</sup></strong>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.10 (1.07–1.13)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs11191580</td><td rowspan="3" colspan="1" valign="middle" align="left">10q24.33<sup>a</sup></td><td rowspan="3" colspan="1" valign="middle" align="right">104.9</td><td rowspan="3" colspan="1" valign="middle" align="center">TC</td><td rowspan="3" colspan="1" valign="middle" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.23 × 10<sup>−8</sup></strong> (4.58 × 10<sup>−7</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.22 (1.15–1.29)</td><td rowspan="3" colspan="1" valign="middle" align="center">++++++++</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>NT5C2</em></td><td rowspan="3" colspan="1" valign="middle" align="right">Intragenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>5.09 × 10<sup>−3</sup></strong> (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.09 (1.02–1.16)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.11 × 10<sup>−8</sup></strong> (3.72 × 10<sup>−7</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.15 (1.10–1.20)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs548181</td><td rowspan="3" colspan="1" valign="middle" align="left">11q24.2</td><td rowspan="3" colspan="1" valign="middle" align="right">125.0</td><td rowspan="3" colspan="1" valign="middle" align="center">GA</td><td rowspan="3" colspan="1" valign="middle" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.91 × 10<sup>−8</sup></strong> (5.69 × 10<sup>−7</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.13–1.26)</td><td rowspan="3" colspan="1" valign="middle" align="center">+++++−+</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>STT3A</em></td><td rowspan="3" colspan="1" valign="middle" align="right">1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">0.068 (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.04 (0.98–1.11)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">8.87 × 10<sup>−7</sup> (1.74 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.11 (1.07–1.16)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs12966547</td><td rowspan="3" colspan="1" valign="middle" align="left">18q21.2</td><td rowspan="3" colspan="1" valign="middle" align="right">50.9</td><td rowspan="3" colspan="1" valign="middle" align="center">GA</td><td rowspan="3" colspan="1" valign="middle" align="center">0.58</td><td rowspan="1" colspan="1" valign="top" align="right">1.00 × 10<sup>−6</sup> (1.03 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.10 (1.06–1.14)</td><td rowspan="3" colspan="1" valign="middle" align="center">+−++++++</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>CCDC68</em></td><td rowspan="3" colspan="1" valign="middle" align="right">126</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.29 × 10<sup>−5</sup></strong> (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (1.04–1.12)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.60 × 10<sup>−10</sup> (5.99 × 10<sup>−9</sup></strong>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.09 (1.06–1.12)</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="left">rs17512836</td><td rowspan="3" colspan="1" valign="middle" align="left">18q21.2</td><td rowspan="3" colspan="1" valign="middle" align="right">51.3</td><td rowspan="3" colspan="1" valign="middle" align="center">CT</td><td rowspan="3" colspan="1" valign="middle" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.35 × 10<sup>−8</sup></strong> (4.78 × 10<sup>−7</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.40 (1.28–1.52)</td><td rowspan="3" colspan="1" valign="middle" align="center">−+++++++</td><td rowspan="3" colspan="1" valign="middle" align="left"><em>TCF4</em></td><td rowspan="3" colspan="1" valign="middle" align="right">Intragenic</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">0.085 (n.a.)</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (0.96–1.20)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">1.05 × 10<sup>−6</sup> (2.86 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 (1.14–1.31)</td></tr></tbody></table></div><div id="largeobj_idm140107424867728" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3303194/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="__p146" class="p p-first-last">The SNPs listed are those with a stage 1 <em>P</em> &lt; 5 × 10<sup>−8</sup> and/or a combined stage 1 and 2 <em>P</em> &lt; 5 × 10<sup>−8</sup>. These ten independent (<em>r</em><sup>2</sup> &lt; 0.2) SNPs represent eight physically distinct genomic loci, as there are two SNPs listed for two loci (10q24.32-q24.33 and 18q21.2). For the MHC region, only one SNP is listed for clarity. The eight susceptibility loci represent three previously reported and five new loci (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>). Stage 1 is the discovery GWAS mega-analysis. Stage 2 is the replication sample (single-tailed meta-analysis <em>P</em> values are weighted by 1/s.e.), and because the <em>P</em> values are single tailed, some 95% confidence intervals contain 1 (if 0.10 &lt; <em>P</em> &lt; 0.05). Combined values include stages 1 and 2 (two-tailed meta-analysis <em>P</em> values are weighted by 1/s.e.). For each SNP, <em>P</em> values and odds ratios are listed for stage 1 (top), stage 2 (middle) and combined stage 1 and 2 analysis (bottom) with the genomic control (GC)-adjusted values bracketed (n.a., not applicable for stage 2). Alleles are listed with the stage 1 risk allele first; the frequency (in stage 1 controls) and odds ratio (OR) refer to the stage 1 risk allele. Bolded <em>P</em> values indicate <em>P</em> &lt; 5 × 10<sup>−8</sup>, except for in the stage 2 data, where bolded values indicate <em>P</em> &lt; 0.05. The directions of association in eight replication samples are represented by + if the associations are in the same direction, − if they are in opposite directions and a blank space if the data are not available. Mb is the base position based on hg18. Cytogenetic bands are listed for each SNP, though because only one of multiple MHC SNPs are listed, a band range is given in that instance. The nearest gene (or microRNA) is listed, with the distance (kb) from the gene (or if the SNP is intragenic) noted. None of these SNPs showed a significant test for hetereogeneity among the samples. Chr., chromosome.</p></div><div id="TFN3"><sup>a</sup>New finding.</div></div></div><p id="P3">In stage 1 (<a href="/pmc/articles/PMC3303194/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 3 and 4</a>), 136 associations reached genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>)<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886361">7</a></sup>. The majority of these associations (<em>N</em> = 129) mapped to 5.5 Mb in the extended major histocompatibility complex (MHC, 6p21.32-p22.1), a region of high linkage disequilibrium (LD) previously implicated in schizophrenia in a subset of the samples used here<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373102531">4</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886345">8</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886350">9</a></sup>. The other stage 1 regions included new regions (10q24.33 and 8q21.3) and previously reported regions (18q21.2 at <em>TCF4</em> (encoding transcription factor 4) and 11q24.2 (ref. <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886352">8</a>)). The signal at 11q24.2 is ~0.85 Mb from <em>NRGN</em> (encoding neurogranin) and is uncorrelated with the previously associated variant near this gene<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886354">8</a></sup>.</p><p id="P4">In <a href="/pmc/articles/PMC3303194/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>, we denote regions of association by the most significant marker. Associated SNPs with <em>r</em><sup>2</sup> ≥ 0.2 in HapMap3 (CEU+TSI populations) were not considered independent. However, we noticed instances where multiple SNPs within 250 kb of each other yielded evidence for association (<em>P</em> &lt; 10<sup>−5</sup>) despite weak LD (<em>r</em><sup>2</sup> &lt; 0.2) between them. For regions with <em>P</em> &lt; 10<sup>−6</sup>, we performed a conditional analysis using as covariates the dosages of the strongest associated SNP, principal components 1–4 and 6 and study indicator. We observed multiple statistically independent signals at the MHC. Although a number of SNPs within the MHC were potentially independent per HapMap <em>r</em><sup>2</sup> values, only rs9272105 withstood formal conditional analysis, showing <em>P</em> = 1.8 × 10<sup>−6</sup> conditional on association to the best SNP, rs2021722 (stage 1 <em>P</em> = 4.3 × 10<sup>−11</sup>, inter-SNP distance = 2.4 Mb, <em>r</em><sup>2</sup> = 0.01 in HapMap). Excluding the MHC region, we identified six regions with at least one SNP associated at <em>P</em> &lt; 10<sup>−5</sup> and a second SNP with a conditionally independent <em>P</em> &lt; 10<sup>−3</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). We performed 100 simulations after permuting case-control status randomly within each study. In contrast to the six regions in the real dataset, we never observed more than a single region with co-localized statistically independent signals in any simulated genome-wide scan, indicating our observation is highly unlikely to have occurred by chance.</p><p id="P5">Noteworthy co-localizing independent signals occurred at three regions (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>): one region with a genome-wide significant association at 10q24.32-q24.33 (<a href="/pmc/articles/PMC3303194/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), a second region that nearly met this threshold at <em>MAD1L1</em> (encoding mitotic arrest deficient-like 1; rs10226475, <em>P</em> = 5.06 × 10<sup>−8</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>) and a third region at <em>CACNA1C</em> (encoding calcium channel, voltage-dependent, L type, α 1C subunit), the latter of which has previously been associated with bipolar disorder<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886366">10</a></sup> and other psychiatric phenotypes including schizophrenia<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886337">11</a></sup>. The conditionally independent signal at <em>CACNA1C</em> was more significant than any observation made in 100 permutations of the entire experiment (both conditional <em>P</em> &lt; 10<sup>−5</sup>) and supports <em>CACNA1C</em> in schizophrenia after genome-wide correction (<em>P</em> &lt; 0.01), even without considering these prior reports.</p><p id="P6">In stage 2, we evaluated in 29,839 independent subjects (8,442 cases and 21,397 controls) the most significant SNPs (<em>N</em> = 81) in each LD region where at least one SNP had surpassed <em>P</em> &lt; 2 × 10<sup>−5</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>) in the mega-analysis. Of 22 SNPs from the MHC, 5 surpassed the genome-wide significant threshold in stages 1 and 2 combined (minimum <em>P</em> = 2.2 × 10<sup>−12</sup> at rs2021722; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). Excluding the MHC region, a sign test for consistency between stages 1 and 2 was highly significant (<em>P</em> &lt; 10<sup>−6</sup>), with the same direction of effect as observed stage 1 also being observed in stage 2 for 49 of 59 SNPs. A Fisher’s combined test revealed the distribution of stage 2 <em>P</em> values was unlikely to have occurred by chance (<em>P</em> &lt; 10<sup>−15</sup>). We also performed a transmission analysis using the family based Multicenter Pedigree replication sample in conjunction with a GWAS of 622 parent-offspring schizophrenia trios from Bulgaria<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_444401684">12</a></sup>, and the stage 1 associated allele was over-transmitted to cases for 44 of the 59 SNPs (one-sided <em>P</em> = 1.0 × 10<sup>−4</sup>). Thus, the stage 2 replication results are highly consistent with the stage 1 discovery results.</p><p id="P7">In the combined dataset (stages 1 and 2), five new (1p21.3, 2q32.3, 8p23.2, 8q21.3 and 10q24.32-q24.33) and two previously reported (6p21.32-p22.1 and 18q21.2) loci met genome-wide significance (<a href="/pmc/articles/PMC3303194/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figs. 1</span></a>,<a href="/pmc/articles/PMC3303194/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,2,</span></span><span>2</span></a>, <a href="/pmc/articles/PMC3303194/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6,7</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>). After adjusting for <em>λ</em> (ref. <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886344">6</a>), four loci (1p21.3, 6p21.32-p22.1, 10q24.32-q24.33 and 18q21.2) remained significant at <em>P</em> ≤ 5 × 10<sup>−8</sup>. For the primary analyses (unadjusted for <em>λ</em>), the strongest new association was at 1p21.3 (rs1625579; <em>P</em> = 1.6 × 10<sup>−11</sup>), which is over 100 kb from any RefSeq protein-coding gene but is within intron 3 of <a id="__tag_776461640" class="tag_hotlink" href="/nuccore/AK094607" ref="/nuccore/AK094607"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK094607","term_id":"21753697"}}</span>AK094607</a>, which contains the primary transcript for <em>MIR137</em> (ref. <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886339">13</a>). The next best locus, 10q24.32 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>), has independent associations 130 kb apart at rs7914558 (<em>P</em> = 1.8 × 10<sup>−9</sup>) and rs11191580 (<em>P</em> = 1.1 × 10<sup>−8</sup>), implicating a 0.5-Mb region containing multiple genes (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>). The third best locus, rs7004633 (<em>P</em> = 2.8 × 10<sup>−8</sup>) on 8q21.3, is 400 kb from the nearest gene (<em>MMP16</em>, encoding matrix metallopeptidase 16). The fourth best locus, rs10503253 (<em>P</em> = 4.4 × 10<sup>−8</sup>) at 8p23.2, is in an intron of <em>CSMD1</em> (encoding CUB and Sushi multiple domains 1). Finally, rs17662626 (<em>P</em> = 4.7 × 10<sup>−8</sup>) at 2q32.3 is intergenic, mapping 300 kb from a non-coding RNA, <em>PCGEM1</em> (prostate-specific transcript 1)<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_334245847">14</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3303194/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140107431773264" class="figure"><a href="/pmc/articles/PMC3303194/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3303194_nihms360727f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms360727f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3303194/bin/nihms360727f1.jpg"></a></div><div id="largeobj_idm140107431773264" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3303194/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3303194/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p141">Manhattan plot for stages 1 and 2. Standard −log<sub>10</sub>
<em>P</em> plot of the study results. For the stage 1 results, 16 regions with one or more SNP achieving <em>P</em> &lt; 10<sup>−6</sup> are highlighted in color and labeled with the name of the nearest gene. SNPs selected for stage 2 replication are highlighted, with the resulting combined <em>P</em> value after replication (that is, after incorporation of stage 2 results) indicated by the large diamonds. Blue highlighting indicates SNPs that were less significantly associated after replication, and pink highlighting indicates SNPs that were more significantly associated after replication.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3303194/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140107472650112" class="figure"><a href="/pmc/articles/PMC3303194/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3303194_nihms360727f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms360727f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3303194/bin/nihms360727f2.jpg"></a></div><div id="largeobj_idm140107472650112" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3303194/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3303194/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p142">Regional association plots for five new schizophrenia loci. Regional <em>P</em> value plots for each of the five new schizophrenia loci: 1p21.3, 2q32.3, 8p23.2, 8q21.3 and 10q24.32-q24.33. Each plot shows the most associated SNP (key SNP) and its genomic region from the first column of <a href="/pmc/articles/PMC3303194/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>: stage 1 scan results for each SNP ± 200 kb to the key SNP are shown. On the <em>x</em> axis is the genomic position, and on the <em>y</em> axis is −log<sub>10</sub>
<em>P</em>. Larger SNP symbols indicate higher LD (based on HapMap 3 data) to the key SNP than smaller SNP symbols. Color coding (from red to blue) denotes LD information; see also the legend within the plot.</p></div></div></div><p id="P8"><em>MIR137</em> has been implicated in regulating adult neurogenesis<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886343">15</a>,<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886365">16</a></sup> and neuronal maturation<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886364">17</a></sup>, mechanisms through which variation at this locus could contribute to brain development abnormalities in schizophrenia. Of relevance, two independent schizophrenia imaging studies found <em>MIR137</em> to be one of three microRNAs with targets significantly enriched for association<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886338">18</a></sup>. In stage 1, SNPs in or near 301 high-confidence predicted <em>MIR137</em> targets (with a TargetScan<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886359">19</a></sup> probability of conserved targeting ≥0.9) were enriched for association compared with genes matched for size and marker density: 17 predicted <em>MIR137</em> targets (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>) had at least one SNP with <em>P</em> &lt; 10<sup>−4</sup>, which is more than twice as many as the control gene sets (<em>P</em> &lt; 0.01). Excluding the MHC and <em>MIR137</em>, of the nine loci with genome-wide significant support either in stage 1 or in the combined set (six loci, 2q32.3, 8p23.2, 8q21.3, 10q24.32-q24.33, 11q24.2 and 18q21.2; <a href="/pmc/articles/PMC3303194/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6,7</a>) or in a joint analysis with bipolar disorder (three genes, <em>CACNA1C</em>, <em>ANK3</em> and <em>ITIH3</em>-<em>ITIH4</em>, described below), four genes (<em>TCF4</em>, <em>CACNA1C</em>, <em>CSMD1</em> and <em>C10orf26</em>) have predicted <em>MIR137</em> target sites according to analyses using three different prediction programs (TargetScan<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886351">19</a></sup>, PicTar<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886333">20</a></sup> and miRanda<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886348">21</a></sup>). <em>In vitro</em> overexpression and locked nucleic acid–mediated knockdown of <em>MIR137</em> in neuronal cell line N2a leads to changes in expression levels of TCF4 protein, strongly supporting the prediction that <em>TCF4</em> is a target of <em>MIR137</em> (L.-H. Tsai, personal communication). Our observations suggest <em>MIR137</em>-mediated dysregulation as a new etiologic mechanism in schizophrenia.</p><p id="P9">The International Schizophrenia Consortium (ISC) reported evidence for a polygenic contribution to schizophrenia<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373102528">4</a></sup>. An independent family based study confirmed these results, greatly minimizing the possibility of population stratification artifact<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_444401683">12</a></sup>. We reevaluated the polygenic model, dividing stage 1 samples into independent training and testing sets (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). The training set had 15,429 subjects (over twice the size of the ISC training set), and the testing set consisted of 6,428 individuals independent of the ISC report. The proportion of variance (Nagelkerke’s <em>r</em><sup>2</sup>) explained in the testing set increased from 3% in the ISC to around 6% here (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a>). This estimate is much lower than the true total variation in liability that is tagged by all SNPs because SNP effects are estimated with error<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886336">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373102527">4</a>,<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886353">22</a>–<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886360">25</a></sup>. The polygenic model appears to explain a substantial fraction of the heritability of schizophrenia<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373102530">4</a></sup>, as has been shown for other complex traits<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886356">3</a>,<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886334">26</a>–<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886341">28</a></sup>. Some of these additional risk loci are likely contained near the most highly significant results of our stage 1 analysis. Supporting this hypothesis, of the top loci that did not reach genome-wide significance in the combined stage 1 and 2 analysis, a sign test (<em>P</em> &lt; 10<sup>−4</sup>) and a Fisher’s combined test (<em>P</em> &lt; 10<sup>−5</sup>) both showed an excess of same-direction allelic association (41 of 51 non-MHC SNPs) in the discovery and replication datasets.</p><p id="P10">Clinical, epidemiological and genetic findings suggest shared risk factors between bipolar disorder and schizophrenia<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373102526">29</a></sup>. In stage 1, three genes with strong support had prior genome-wide significant associations with bipolar disorder: <em>CACNA1C</em>, the region containing <em>ITIH3</em>-<em>ITIH4</em> (encoding inter-α (globulin) inhibitors H3 and H4) and <em>ANK3</em> (encoding ankyrin 3, node of Ranvier (ankyrin G))<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886331">10</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886329">11</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886355">30</a></sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a>). We performed a joint analysis with the Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium (PGC) for bipolar disorder applying identical analytical methods. After removing duplicate subjects, we analyzed 16,374 cases with schizophrenia, schizoaffective disorder or bipolar disorder and 14,044 controls. Support for shared susceptibility was strengthened (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 11</a>) at <em>CACNA1C</em> (rs4765905, <em>P</em> = 7.0 × 10<sup>−9</sup>), <em>ANK3</em> (rs10994359, <em>P</em> = 2.5 × 10<sup>−8</sup>) and the <em>ITIH3-ITIH4</em> region (rs2239547, <em>P</em> = 7.8 × 10<sup>−9</sup>), each of which reached genome-wide significance. A coding variant in <em>ITIH4</em> (p.Pro698Thr; rs4687657) is in perfect LD with the most associated SNP. Although we included all subjects from an earlier report<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253886346">10</a></sup>, the increased support found with additional independent cases (<em>N</em> = 11,987) and controls (<em>N</em> = 7,835) provides further evidence for shared risk effects of schizophrenia and bipolar disorder.</p><p id="P11">The risk variants implicated here confer small risks (odds ratios ~1.10), but the polygenic analysis shows many more susceptibility variants with effects for which our sample is underpowered (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>). At every stage where samples were added, we found an increase in the number of genome-wide significant loci and enhancement of signals at <em>CACNA1C, ANK3</em> and <em>ITIH3-ITIH4</em> when schizophrenia and bipolar disorder were jointly analyzed. Thus, gains in power offset any penalty for increased heterogeneity.</p><p id="P12" class="p">In summary, we report seven genome-wide significant schizophrenia associations (five of which are new) in a two-stage analysis of 51,695 individuals. We also report loci that confer susceptibility to both bipolar disorder and schizophrenia. The association near <em>MIR137</em>, associations in multiple predicted <em>MIR137</em> targets and the known role of <em>MIR137</em> in neuronal maturation and function together suggest an intriguing new insight into the pathogenesis of schizophrenia.</p><div id="S2" class="sec sec-last"><h3 id="S2title">URLs</h3><p id="P13" class="p p-first-last">PLINK, <a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink/</a>; Haploview, <a href="http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/haploview/haploview" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/haploview/haploview</a>.</p></div></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title">METHODS</h2><p id="P14" class="p p-first-last">Methods and any associated references are available in the online version of the paper at <a href="http://www.nature.com/naturegenetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics/</a>.</p></div><div id="S4" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Note</h4><div class="sup-box half_rhythm" id="N0x8d4eb60N0x9d01650"><a href="/pmc/articles/PMC3303194/bin/NIHMS360727-supplement-supplementary.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(26M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Online Methods</h4><div class="sup-box half_rhythm" id="N0x8d4eb60N0x9d01710"><a href="/pmc/articles/PMC3303194/bin/NIHMS360727-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(108K, pdf)</sup></div></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Acknowledgments</h2><div class="sec"><p id="__p2">We thank the study participants and the research staff at the many study sites. Over 40 US National Institutes of Health grants and similar numbers of government grants from other countries, along with substantial private and foundation support, enabled this work. We greatly appreciate the sustained efforts of T. Lehner (National Institute of Mental Health) on behalf of the Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium (PGC). Detailed acknowledgments, including grant support, are listed in the</p></div></div><div id="APP1" class="tsec sec headless whole_rhythm"><!--/article/back/app-group/app/--><p id="P15">Stephan Ripke<sup><sup><a href="#FN6" rid="FN6" class=" fn">1</a></sup></sup>, Alan R Sanders<sup><sup><a href="#FN7" rid="FN7" class=" fn">2</a></sup>,<sup><a href="#FN8" rid="FN8" class=" fn">3</a></sup></sup>, Kenneth S Kendler<sup><sup><a href="#FN9" rid="FN9" class=" fn">4</a></sup>–<sup><a href="#FN11" rid="FN11" class=" fn">6</a></sup></sup>, Douglas F Levinson<sup><sup><a href="#FN12" rid="FN12" class=" fn">7</a></sup></sup>, Pamela Sklar<sup><sup><a href="#FN6" rid="FN6" class=" fn">1</a></sup>,<sup><a href="#FN13" rid="FN13" class=" fn">8</a></sup></sup>, Peter A Holmans<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Dan-Yu Lin<sup><sup><a href="#FN16" rid="FN16" class=" fn">11</a></sup></sup>, Jubao Duan<sup><sup><a href="#FN7" rid="FN7" class=" fn">2</a></sup>,<sup><a href="#FN8" rid="FN8" class=" fn">3</a></sup></sup>, Roel A Ophoff<sup><sup><a href="#FN17" rid="FN17" class=" fn">12</a></sup>–<sup><a href="#FN20" rid="FN20" class=" fn">15</a></sup></sup>, Ole A Andreassen<sup><sup><a href="#FN21" rid="FN21" class=" fn">16</a></sup>,<sup><a href="#FN22" rid="FN22" class=" fn">17</a></sup></sup>, Edward Scolnick<sup><sup><a href="#FN23" rid="FN23" class=" fn">18</a></sup></sup>, Sven Cichon<sup><sup><a href="#FN24" rid="FN24" class=" fn">19</a></sup>–<sup><a href="#FN26" rid="FN26" class=" fn">21</a></sup></sup>, David St. Clair<sup><sup><a href="#FN27" rid="FN27" class=" fn">22</a></sup></sup>, Aiden Corvin<sup><sup><a href="#FN28" rid="FN28" class=" fn">23</a></sup></sup>, Hugh Gurling<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, Thomas Werge<sup><sup><a href="#FN30" rid="FN30" class=" fn">25</a></sup></sup>, Dan Rujescu<sup><sup><a href="#FN31" rid="FN31" class=" fn">26</a></sup></sup>, Douglas H R Blackwood<sup><sup><a href="#FN32" rid="FN32" class=" fn">27</a></sup></sup>, Carlos N Pato<sup><sup><a href="#FN33" rid="FN33" class=" fn">28</a></sup></sup>, Anil K Malhotra<sup><sup><a href="#FN34" rid="FN34" class=" fn">29</a></sup>–<sup><a href="#FN36" rid="FN36" class=" fn">31</a></sup></sup>, Shaun Purcell<sup><sup><a href="#FN23" rid="FN23" class=" fn">18</a></sup></sup>, Frank Dudbridge<sup><sup><a href="#FN37" rid="FN37" class=" fn">32</a></sup></sup>, Benjamin M Neale<sup><sup><a href="#FN23" rid="FN23" class=" fn">18</a></sup></sup>, Lizzy Rossin<sup><sup><a href="#FN6" rid="FN6" class=" fn">1</a></sup></sup>, Peter M Visscher<sup><sup><a href="#FN38" rid="FN38" class=" fn">33</a></sup></sup>, Danielle Posthuma<sup><sup><a href="#FN39" rid="FN39" class=" fn">34</a></sup>,<sup><a href="#FN40" rid="FN40" class=" fn">35</a></sup></sup>, Douglas M Ruderfer<sup><sup><a href="#FN6" rid="FN6" class=" fn">1</a></sup></sup>, Ayman Fanous<sup><sup><a href="#FN10" rid="FN10" class=" fn">5</a></sup>,<sup><a href="#FN41" rid="FN41" class=" fn">36</a></sup>,<sup><a href="#FN42" rid="FN42" class=" fn">37</a></sup></sup>, Hreinn Stefansson<sup><sup><a href="#FN43" rid="FN43" class=" fn">38</a></sup></sup>, Stacy Steinberg<sup><sup><a href="#FN43" rid="FN43" class=" fn">38</a></sup></sup>, Bryan J Mowry<sup><sup><a href="#FN44" rid="FN44" class=" fn">39</a></sup>,<sup><a href="#FN45" rid="FN45" class=" fn">40</a></sup></sup>, Vera Golimbet<sup><sup><a href="#FN46" rid="FN46" class=" fn">41</a></sup></sup>, Marc De Hert<sup><sup><a href="#FN47" rid="FN47" class=" fn">42</a></sup></sup>, Erik G Jönsson<sup><sup><a href="#FN48" rid="FN48" class=" fn">43</a></sup></sup>, István Bitter<sup><sup><a href="#FN49" rid="FN49" class=" fn">44</a></sup></sup>, Olli P H Pietiläinen<sup><sup><a href="#FN50" rid="FN50" class=" fn">45</a></sup>,<sup><a href="#FN51" rid="FN51" class=" fn">46</a></sup></sup>, David A Collier<sup><sup><a href="#FN52" rid="FN52" class=" fn">47</a></sup></sup>, Sarah Tosato<sup><sup><a href="#FN53" rid="FN53" class=" fn">48</a></sup></sup>, Ingrid Agartz<sup><sup><a href="#FN21" rid="FN21" class=" fn">16</a></sup>,<sup><a href="#FN54" rid="FN54" class=" fn">49</a></sup></sup>, Margot Albus<sup><sup><a href="#FN55" rid="FN55" class=" fn">50</a></sup></sup>, Madeline Alexander<sup><sup><a href="#FN12" rid="FN12" class=" fn">7</a></sup></sup>, Richard L Amdur<sup><sup><a href="#FN41" rid="FN41" class=" fn">36</a></sup>,<sup><a href="#FN42" rid="FN42" class=" fn">37</a></sup></sup>, Farooq Amin<sup><sup><a href="#FN56" rid="FN56" class=" fn">51</a></sup>,<sup><a href="#FN57" rid="FN57" class=" fn">52</a></sup></sup>, Nicholas Bass<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, Sarah E Bergen<sup><sup><a href="#FN6" rid="FN6" class=" fn">1</a></sup></sup>, Donald W Black<sup><sup><a href="#FN58" rid="FN58" class=" fn">53</a></sup></sup>, Anders D Børglum<sup><sup><a href="#FN59" rid="FN59" class=" fn">54</a></sup>,<sup><a href="#FN60" rid="FN60" class=" fn">55</a></sup></sup>, Matthew A Brown<sup><sup><a href="#FN61" rid="FN61" class=" fn">56</a></sup></sup>, Richard Bruggeman<sup><sup><a href="#FN62" rid="FN62" class=" fn">57</a></sup></sup>, Nancy G Buccola<sup><sup><a href="#FN63" rid="FN63" class=" fn">58</a></sup></sup>, William F Byerley<sup><sup><a href="#FN64" rid="FN64" class=" fn">59</a></sup>,<sup><a href="#FN65" rid="FN65" class=" fn">60</a></sup></sup>, Wiepke Cahn<sup><sup><a href="#FN66" rid="FN66" class=" fn">61</a></sup></sup>, Rita M Cantor<sup><sup><a href="#FN19" rid="FN19" class=" fn">14</a></sup>,<sup><a href="#FN20" rid="FN20" class=" fn">15</a></sup></sup>, Vaughan J Carr<sup><sup><a href="#FN67" rid="FN67" class=" fn">62</a></sup></sup>, Stanley V Catts<sup><sup><a href="#FN68" rid="FN68" class=" fn">63</a></sup></sup>, Khalid Choudhury<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, C Robert Cloninger<sup><sup><a href="#FN69" rid="FN69" class=" fn">64</a></sup></sup>, Paul Cormican<sup><sup><a href="#FN28" rid="FN28" class=" fn">23</a></sup></sup>, Nicholas Craddock<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Patrick A Danoy<sup><sup><a href="#FN61" rid="FN61" class=" fn">56</a></sup></sup>, Susmita Datta<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, Lieuwe de Haan<sup><sup><a href="#FN70" rid="FN70" class=" fn">65</a></sup></sup>, Ditte Demontis<sup><sup><a href="#FN59" rid="FN59" class=" fn">54</a></sup></sup>, Dimitris Dikeos<sup><sup><a href="#FN71" rid="FN71" class=" fn">66</a></sup></sup>, Srdjan Djurovic<sup><sup><a href="#FN21" rid="FN21" class=" fn">16</a></sup>,<sup><a href="#FN72" rid="FN72" class=" fn">67</a></sup></sup>, Peter Donnelly<sup><sup><a href="#FN73" rid="FN73" class=" fn">68</a></sup>,<sup><a href="#FN74" rid="FN74" class=" fn">69</a></sup></sup>, Gary Donohoe<sup><sup><a href="#FN28" rid="FN28" class=" fn">23</a></sup></sup>, Linh Duong<sup><sup><a href="#FN30" rid="FN30" class=" fn">25</a></sup></sup>, Sarah Dwyer<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Anders Fink-Jensen<sup><sup><a href="#FN75" rid="FN75" class=" fn">70</a></sup></sup>, Robert Freedman<sup><sup><a href="#FN76" rid="FN76" class=" fn">71</a></sup></sup>, Nelson B Freimer<sup><sup><a href="#FN19" rid="FN19" class=" fn">14</a></sup></sup>, Marion Friedl<sup><sup><a href="#FN31" rid="FN31" class=" fn">26</a></sup></sup>, Lyudmila Georgieva<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Ina Giegling<sup><sup><a href="#FN31" rid="FN31" class=" fn">26</a></sup></sup>, Michael Gill<sup><sup><a href="#FN28" rid="FN28" class=" fn">23</a></sup></sup>, Birte Glenthøj<sup><sup><a href="#FN77" rid="FN77" class=" fn">72</a></sup></sup>, Stephanie Godard<sup><sup><a href="#FN78" rid="FN78" class=" fn">73</a></sup></sup>, Marian Hamshere<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Mark Hansen<sup><sup><a href="#FN79" rid="FN79" class=" fn">74</a></sup></sup>, Thomas Hansen<sup><sup><a href="#FN30" rid="FN30" class=" fn">25</a></sup></sup>, Annette M Hartmann<sup><sup><a href="#FN31" rid="FN31" class=" fn">26</a></sup></sup>, Frans A Henskens<sup><sup><a href="#FN80" rid="FN80" class=" fn">75</a></sup></sup>, David M Hougaard<sup><sup><a href="#FN81" rid="FN81" class=" fn">76</a></sup></sup>, Christina M Hultman<sup><sup><a href="#FN82" rid="FN82" class=" fn">77</a></sup></sup>, Andrés Ingason<sup><sup><a href="#FN30" rid="FN30" class=" fn">25</a></sup></sup>, Assen V Jablensky<sup><sup><a href="#FN83" rid="FN83" class=" fn">78</a></sup></sup>, Klaus D Jakobsen<sup><sup><a href="#FN30" rid="FN30" class=" fn">25</a></sup></sup>, Maurice Jay<sup><sup><a href="#FN84" rid="FN84" class=" fn">79</a></sup>,<sup><a href="#FN137" rid="FN137" class=" fn">132</a></sup></sup>, Gesche Jürgens<sup><sup><a href="#FN85" rid="FN85" class=" fn">80</a></sup></sup>, René S Kahn<sup><sup><a href="#FN66" rid="FN66" class=" fn">61</a></sup></sup>, Matthew C Keller<sup><sup><a href="#FN86" rid="FN86" class=" fn">81</a></sup></sup>, Gunter Kenis<sup><sup><a href="#FN87" rid="FN87" class=" fn">82</a></sup></sup>, Elaine Kenny<sup><sup><a href="#FN28" rid="FN28" class=" fn">23</a></sup></sup>, Yunjung Kim<sup><sup><a href="#FN88" rid="FN88" class=" fn">83</a></sup></sup>, George K Kirov<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Heike Konnerth<sup><sup><a href="#FN31" rid="FN31" class=" fn">26</a></sup></sup>, Bettina Konte<sup><sup><a href="#FN31" rid="FN31" class=" fn">26</a></sup></sup>, Lydia Krabbendam<sup><sup><a href="#FN89" rid="FN89" class=" fn">84</a></sup></sup>, Robert Krasucki<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, Virginia K Lasseter<sup><sup><a href="#FN90" rid="FN90" class=" fn">85</a></sup>,<sup><a href="#FN137" rid="FN137" class=" fn">132</a></sup></sup>, Claudine Laurent<sup><sup><a href="#FN84" rid="FN84" class=" fn">79</a></sup></sup>, Jacob Lawrence<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, Todd Lencz<sup><sup><a href="#FN34" rid="FN34" class=" fn">29</a></sup>–<sup><a href="#FN36" rid="FN36" class=" fn">31</a></sup></sup>, F Bernard Lerer<sup><sup><a href="#FN91" rid="FN91" class=" fn">86</a></sup></sup>, Kung-Yee Liang<sup><sup><a href="#FN92" rid="FN92" class=" fn">87</a></sup></sup>, Paul Lichtenstein<sup><sup><a href="#FN82" rid="FN82" class=" fn">77</a></sup></sup>, Jeffrey A Lieberman<sup><sup><a href="#FN93" rid="FN93" class=" fn">88</a></sup></sup>, Don H Linszen<sup><sup><a href="#FN70" rid="FN70" class=" fn">65</a></sup></sup>, Jouko Lönnqvist<sup><sup><a href="#FN94" rid="FN94" class=" fn">89</a></sup></sup>, Carmel M Loughland<sup><sup><a href="#FN95" rid="FN95" class=" fn">90</a></sup></sup>, Alan W Maclean<sup><sup><a href="#FN32" rid="FN32" class=" fn">27</a></sup></sup>, Brion S Maher<sup><sup><a href="#FN9" rid="FN9" class=" fn">4</a></sup>–<sup><a href="#FN11" rid="FN11" class=" fn">6</a></sup></sup>, Wolfgang Maier<sup><sup><a href="#FN96" rid="FN96" class=" fn">91</a></sup></sup>, Jacques Mallet<sup><sup><a href="#FN97" rid="FN97" class=" fn">92</a></sup></sup>, Pat Malloy<sup><sup><a href="#FN32" rid="FN32" class=" fn">27</a></sup></sup>, Manuel Mattheisen<sup><sup><a href="#FN24" rid="FN24" class=" fn">19</a></sup>,<sup><a href="#FN26" rid="FN26" class=" fn">21</a></sup>,<sup><a href="#FN98" rid="FN98" class=" fn">93</a></sup></sup>, Morten Mattingsdal<sup><sup><a href="#FN21" rid="FN21" class=" fn">16</a></sup>,<sup><a href="#FN99" rid="FN99" class=" fn">94</a></sup></sup>, Kevin A McGhee<sup><sup><a href="#FN32" rid="FN32" class=" fn">27</a></sup></sup>, John J McGrath<sup><sup><a href="#FN44" rid="FN44" class=" fn">39</a></sup>,<sup><a href="#FN45" rid="FN45" class=" fn">40</a></sup></sup>, Andrew McIntosh<sup><sup><a href="#FN32" rid="FN32" class=" fn">27</a></sup></sup>, Duncan E McLean<sup><sup><a href="#FN100" rid="FN100" class=" fn">95</a></sup></sup>, Andrew McQuillin<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, Ingrid Melle<sup><sup><a href="#FN21" rid="FN21" class=" fn">16</a></sup>,<sup><a href="#FN22" rid="FN22" class=" fn">17</a></sup></sup>, Patricia T Michie<sup><sup><a href="#FN101" rid="FN101" class=" fn">96</a></sup>,<sup><a href="#FN102" rid="FN102" class=" fn">97</a></sup></sup>, Vihra Milanova<sup><sup><a href="#FN103" rid="FN103" class=" fn">98</a></sup></sup>, Derek W Morris<sup><sup><a href="#FN28" rid="FN28" class=" fn">23</a></sup></sup>, Ole Mors<sup><sup><a href="#FN60" rid="FN60" class=" fn">55</a></sup></sup>, Preben B Mortensen<sup><sup><a href="#FN104" rid="FN104" class=" fn">99</a></sup></sup>, Valentina Moskvina<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Pierandrea Muglia<sup><sup><a href="#FN105" rid="FN105" class=" fn">100</a></sup>,<sup><a href="#FN106" rid="FN106" class=" fn">101</a></sup></sup>, Inez Myin-Germeys<sup><sup><a href="#FN89" rid="FN89" class=" fn">84</a></sup></sup>, Deborah A Nertney<sup><sup><a href="#FN44" rid="FN44" class=" fn">39</a></sup>,<sup><a href="#FN45" rid="FN45" class=" fn">40</a></sup></sup>, Gerald Nestadt<sup><sup><a href="#FN90" rid="FN90" class=" fn">85</a></sup></sup>, Jimmi Nielsen<sup><sup><a href="#FN107" rid="FN107" class=" fn">102</a></sup></sup>, Ivan Nikolov<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Merete Nordentoft<sup><sup><a href="#FN108" rid="FN108" class=" fn">103</a></sup></sup>, Nadine Norton<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Markus M Nöthen<sup><sup><a href="#FN24" rid="FN24" class=" fn">19</a></sup>,<sup><a href="#FN26" rid="FN26" class=" fn">21</a></sup></sup>, Colm T O’Dushlaine<sup><sup><a href="#FN28" rid="FN28" class=" fn">23</a></sup></sup>, Ann Olincy<sup><sup><a href="#FN76" rid="FN76" class=" fn">71</a></sup></sup>, Line Olsen<sup><sup><a href="#FN30" rid="FN30" class=" fn">25</a></sup></sup>, F Anthony O’Neill<sup><sup><a href="#FN109" rid="FN109" class=" fn">104</a></sup></sup>, Torben F Ørntoft<sup><sup><a href="#FN110" rid="FN110" class=" fn">105</a></sup>,<sup><a href="#FN111" rid="FN111" class=" fn">106</a></sup></sup>, Michael J Owen<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Christos Pantelis<sup><sup><a href="#FN112" rid="FN112" class=" fn">107</a></sup></sup>, George Papadimitriou<sup><sup><a href="#FN71" rid="FN71" class=" fn">66</a></sup></sup>, Michele T Pato<sup><sup><a href="#FN33" rid="FN33" class=" fn">28</a></sup></sup>, Leena Peltonen<sup><sup><a href="#FN50" rid="FN50" class=" fn">45</a></sup>,<sup><a href="#FN51" rid="FN51" class=" fn">46</a></sup>,<sup><a href="#FN113" rid="FN113" class=" fn">108</a></sup>,<sup><a href="#FN137" rid="FN137" class=" fn">132</a></sup></sup>, Hannes Petursson<sup><sup><a href="#FN114" rid="FN114" class=" fn">109</a></sup></sup>, Ben Pickard<sup><sup><a href="#FN115" rid="FN115" class=" fn">110</a></sup></sup>, Jonathan Pimm<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, Ann E Pulver<sup><sup><a href="#FN90" rid="FN90" class=" fn">85</a></sup></sup>, Vinay Puri<sup><sup><a href="#FN29" rid="FN29" class=" fn">24</a></sup></sup>, Digby Quested<sup><sup><a href="#FN116" rid="FN116" class=" fn">111</a></sup></sup>, Emma M Quinn<sup><sup><a href="#FN28" rid="FN28" class=" fn">23</a></sup></sup>, Henrik B Rasmussen<sup><sup><a href="#FN30" rid="FN30" class=" fn">25</a></sup></sup>, János M Réthelyi<sup><sup><a href="#FN49" rid="FN49" class=" fn">44</a></sup></sup>, Robert Ribble<sup><sup><a href="#FN9" rid="FN9" class=" fn">4</a></sup>–<sup><a href="#FN11" rid="FN11" class=" fn">6</a></sup></sup>, Marcella Rietschel<sup><sup><a href="#FN96" rid="FN96" class=" fn">91</a></sup>,<sup><a href="#FN117" rid="FN117" class=" fn">112</a></sup></sup>, Brien P Riley<sup><sup><a href="#FN9" rid="FN9" class=" fn">4</a></sup>–<sup><a href="#FN11" rid="FN11" class=" fn">6</a></sup></sup>, Mirella Ruggeri<sup><sup><a href="#FN53" rid="FN53" class=" fn">48</a></sup></sup>, Ulrich Schall<sup><sup><a href="#FN102" rid="FN102" class=" fn">97</a></sup>,<sup><a href="#FN118" rid="FN118" class=" fn">113</a></sup></sup>, Thomas G Schulze<sup><sup><a href="#FN117" rid="FN117" class=" fn">112</a></sup>,<sup><a href="#FN119" rid="FN119" class=" fn">114</a></sup></sup>, Sibylle G Schwab<sup><sup><a href="#FN120" rid="FN120" class=" fn">115</a></sup>–<sup><a href="#FN122" rid="FN122" class=" fn">117</a></sup></sup>, Rodney J Scott<sup><sup><a href="#FN123" rid="FN123" class=" fn">118</a></sup></sup>, Jianxin Shi<sup><sup><a href="#FN124" rid="FN124" class=" fn">119</a></sup></sup>, Engilbert Sigurdsson<sup><sup><a href="#FN114" rid="FN114" class=" fn">109</a></sup>,<sup><a href="#FN125" rid="FN125" class=" fn">120</a></sup></sup>, Jeremy M Silverman<sup><sup><a href="#FN13" rid="FN13" class=" fn">8</a></sup>,<sup><a href="#FN126" rid="FN126" class=" fn">121</a></sup></sup>, Chris C A Spencer<sup><sup><a href="#FN73" rid="FN73" class=" fn">68</a></sup></sup>, Kari Stefansson<sup><sup><a href="#FN43" rid="FN43" class=" fn">38</a></sup></sup>, Amy Strange<sup><sup><a href="#FN73" rid="FN73" class=" fn">68</a></sup></sup>, Eric Strengman<sup><sup><a href="#FN17" rid="FN17" class=" fn">12</a></sup>,<sup><a href="#FN18" rid="FN18" class=" fn">13</a></sup></sup>, T Scott Stroup<sup><sup><a href="#FN93" rid="FN93" class=" fn">88</a></sup></sup>, Jaana Suvisaari<sup><sup><a href="#FN94" rid="FN94" class=" fn">89</a></sup></sup>, Lars Terenius<sup><sup><a href="#FN48" rid="FN48" class=" fn">43</a></sup></sup>, Srinivasa Thirumalai<sup><sup><a href="#FN127" rid="FN127" class=" fn">122</a></sup></sup>, Johan H Thygesen<sup><sup><a href="#FN30" rid="FN30" class=" fn">25</a></sup></sup>, Sally Timm<sup><sup><a href="#FN128" rid="FN128" class=" fn">123</a></sup></sup>, Draga Toncheva<sup><sup><a href="#FN129" rid="FN129" class=" fn">124</a></sup></sup>, Edwin van den Oord<sup><sup><a href="#FN130" rid="FN130" class=" fn">125</a></sup></sup>, Jim van Os<sup><sup><a href="#FN89" rid="FN89" class=" fn">84</a></sup></sup>, Ruud van Winkel<sup><sup><a href="#FN47" rid="FN47" class=" fn">42</a></sup>,<sup><a href="#FN87" rid="FN87" class=" fn">82</a></sup></sup>, Jan Veldink<sup><sup><a href="#FN131" rid="FN131" class=" fn">126</a></sup></sup>, Dermot Walsh<sup><sup><a href="#FN132" rid="FN132" class=" fn">127</a></sup></sup>, August G Wang<sup><sup><a href="#FN133" rid="FN133" class=" fn">128</a></sup></sup>, Durk Wiersma<sup><sup><a href="#FN62" rid="FN62" class=" fn">57</a></sup></sup>, Dieter B Wildenauer<sup><sup><a href="#FN120" rid="FN120" class=" fn">115</a></sup>,<sup><a href="#FN134" rid="FN134" class=" fn">129</a></sup></sup>, Hywel J Williams<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Nigel M Williams<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Brandon Wormley<sup><sup><a href="#FN9" rid="FN9" class=" fn">4</a></sup>–<sup><a href="#FN11" rid="FN11" class=" fn">6</a></sup></sup>, Stan Zammit<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Patrick F Sullivan<sup><sup><a href="#FN82" rid="FN82" class=" fn">77</a></sup>,<sup><a href="#FN88" rid="FN88" class=" fn">83</a></sup>,<sup><a href="#FN135" rid="FN135" class=" fn">130</a></sup>,<sup><a href="#FN136" rid="FN136" class=" fn">131</a></sup></sup>, Michael C O’Donovan<sup><sup><a href="#FN14" rid="FN14" class=" fn">9</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">10</a></sup></sup>, Mark J Daly<sup><sup><a href="#FN6" rid="FN6" class=" fn">1</a></sup></sup> &amp; Pablo V Gejman<sup><sup><a href="#FN7" rid="FN7" class=" fn">2</a></sup>,<sup><a href="#FN8" rid="FN8" class=" fn">3</a></sup></sup></p></div><div id="idm140107472427904" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140107472427904title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p3" class="p p-first-last">Note: <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary information</a> is available on the Nature Genetics website.</p><p></p><div><p id="__p4" class="p p-first"><strong>AUTHOR CONTRIBUTIONS</strong></p>The Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium (PGC): <strong>overall coordination</strong>: P.V.G. <strong>Coordination of statistical analyses:</strong> M.J.D. <strong>Coordination of phenotypic analyses:</strong> K.S.K. <strong>Statistical analyses:</strong> S.R., M.J.D., P.A.H., D.-Y.L., S.P., F.D., B.M.N., L.R., P.M.V., D.P., D.M.R. <strong>Manuscript preparation:</strong> P.V.G. (primary), M.J.D. (primary), A.R.S. (primary), S.R. (primary), M.C.O. (primary), K.S.K., D.F.L., P.S., P.A.H., P.F.S. (primary), D.-Y.L., J.D., R.A.O., O.A.A., E. Scolnick. <strong>Phenotypic analyses:</strong> K.S.K., A.F., A.C., R.L.A. <strong>Stage 1 GWAS sample 1–Cardiff, UK:</strong> M.C.O., N.C., P.A.H., M. Hamshere, H.J.W., V. Moskvina, S. Dwyer, L.G., S.Z., M.J.O. <strong>Stage 1 GWAS sample 2–Clinical Antipsychotic Trials of Intervention Effectiveness (CATIE):</strong> P.F.S., D.-Y.L., E.v.d.O., Y.K., T.S.S., J.A.L. <strong>Stage 1 GWAS sample 3–International Schizophrenia Consortium (ISC)–Aberdeen:</strong> D.St.C. <strong>Stage 1 GWAS sample 4–ISC–Cardiff:</strong> G.K.K., M.C.O., P.A.H., L.G., I.N., H.J.W., D.T., V. Milanova, M.J.O. <strong>Stage 1 GWAS sample 5–ISC–Dublin:</strong> D.W.M., C.T.O., E.K., E.M.Q., M.G., A.C. <strong>Stage 1 GWAS sample 6–ISC–Edinburgh:</strong> D.H.R.B., K.A.M., B.P., P. Malloy, A.W.M., A. McIntosh. <strong>Stage 1 GWAS sample 7–ISC–London:</strong> A. McQuillin, K.C., S. Datta, J.P., S. Thirumalai, V.P., R.K., J. Lawrence, D.Q., N.B., H.G. <strong>Stage 1 GWAS sample 8–ISC–Portugal:</strong> M.T.P., C.N.P., A.F. <strong>Stage 1 GWAS sample 9–ISC–SW1–Sweden, stage 1 GWAS sample 10–ISC–SW2–Sweden, stage 2 replication follow-up sample 16–SW3–Sweden, stage 2 replication follow-up sample 17–SW4–Sweden:</strong> C.M.H., P.L., S.E.B., S.P., E. Scolnick, P.S., P.F.S. <strong>Stage 1 GWAS sample 11–Molecular Genetics of Schizophrenia (MGS):</strong> J. Shi, D.F.L., J.D., A.R.S., M.C.K., B.J.M., A.O., F.A., C.R.C., J.M.S., N.G.B., W.FB., D.W.B., K.S.K., R.F., P.V.G. <strong>Stage 1 GWAS sample 12–Schizophrenia Genetics Consortium (SGENE)–Bonn:</strong> S.C., M. Rietschel, M.M.N., W.M., T.G.S., M. Mattheisen. <strong>Stage 1 GWAS sample 13–SGENE–Copenhagen, stage 2 replication follow-up sample 5–SGENE–Copenhagen:</strong> T.H., A.I., K.D.J., L.D., G.J., H.B.R., B.G., J.N., S. Timm, L.O., A.G.W., A.F.-J., J.H.T., T.W. <strong>Stage 1 GWAS sample 14–SGENE–Munich, stage 2 replication follow-up sample 12–SGENE–Munich, stage 2 replication follow-up sample 13–SGENE–Munich:</strong> I.G., A.M.H., H.K., M.F., B.K., P. Muglia, D.R. <strong>Stage 1 GWAS sample 15–SGENE–Thematic Organized Psychoses Research 3 (TOP3):</strong> S. Djurovic, M. Mattingsdal, I.A., I.M., O.A.A. <strong>Stage 1 GWAS sample 16–SGENE–UCLA:</strong> R.A.O., R.M.C., N.B.F., R.S.K., D.H.L., J.v.O., D. Wiersma, R.B., W.C., L.d.H., L.K., I.M.-G., E. Strengman. <strong>Stage 1 GWAS sample 17–Zucker Hillside:</strong> A.K.M., T.L. <strong>Stage 2 replication follow-up sample 1–multicenter pedigree:</strong> P.A.H., B.P.R., A.E.P., M.J.O., D.B.W., P.V.G., B.J.M., C.L., K.S.K., G.N., N.M.W., S.G.S., A.R.S., M. Hansen, D.A.N., J.M., B.W., V.K.L., M.C.O., J.D., M. Albus, M. Alexander, S.G., R.R., K.-Y.L., N.N., W.M., G.P., D. Walsh, M.J., F.A.O., F.B.L., D. Dikeos, J.M.S., D.F.L. <strong>Stage 2 replication follow-up sample 2–SGENE–Aarhus:</strong> A.D.B., D. Demontis, P.B.M., D.M.H., T.F.Ø., O.M. <strong>Stage 2 replication follow-up sample 3–SGENE–Aarhus:</strong> O.M., M.N., A.D.B. <strong>Stage 2 replication follow-up sample 4–SGENE–Belgium:</strong> R.v.W., G.K., M.D.H., J.V. <strong>Stage 2 replication follow-up sample 6–SGENE–Iceland:</strong> H.S., S.S., E. Sigurdsson, H.P., K.S. <strong>Stage 2 replication follow-up sample 7–SGENE–England:</strong> D.A.C. <strong>Stage 2 replication follow-up sample 8–SGENE–Helsinki, stage 2 replication follow-up sample 11–SGENE–Kuusamo:</strong> L.P., O.P.H.P., J. Suvisaari, J. Lönnqvist. <strong>Stage 2 replication follow-up sample 9–SGENE–Hungary:</strong> I.B., J.M.R. <strong>Stage 2 replication follow-up sample 10–SGENE–Italy:</strong> M. Ruggeri, S. Tosato. <strong>Stage 2 replication follow-up sample 14–SGENE–Russia:</strong> V.G. <strong>Stage 2 replication follow-up sample 15–SGENE–Sweden:</strong> E.G.J., I.A., L.T. <strong>Stage 2 replication follow-up sample 18–University of Queensland:</strong> B.J.M., M.A.B., P.A.D., J.J.M., D.E.M. <strong>Stage 2 replication follow-up sample 18–Australian Schizophrenia Research Bank:</strong> B.J.M., V.J.C., R.J.S., S.V.C., F.A.H., A.V.J., C.M.L., P.T.M., C.P., U.S. <strong>Stage 2 replication follow-up sample 19–Irish Schizophrenia Genomics Consortium (ISGC):</strong> A.C., D.W.M., P.C., B.S.M., C.T.O., G.D., F.A.O., M.G., K.S.K., B.P.R., ISGC (see the Acknowledgments in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a> for additional contributors not listed above). <strong>Stage 2 replication follow-up sample 19–Wellcome Trust Case Control Consortium 2 (WTCCC2):</strong> P.D. (Chair of Management Committee; Data and Analysis Group), C.C.A.S. (Data and Analysis Group; Publications Committee), A.S. (Data and Analysis Group), WTCCC2 (see Acknowledgments in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a> for additional contributors not listed above). All authors contributed to the current version of the paper.</div><p class="fn sec" id="FN4"> </p><p id="__p6" class="p p-first"><strong>COMPETING FINANCIAL INTERESTS</strong></p><p id="__p7" class="p p-last">The authors declare competing financial interests: details accompany the full-text HTML version of the paper at <a href="http://www.nature.com/naturegenetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics/</a>.</p><p></p><p class="fn sec" id="FN5"></p><p id="__p8" class="p p-first-last">Reprints and permissions information is available online at <a href="http://www.nature.com/reprints/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/reprints/index.html</a>.</p><p></p><p class="fn sec" id="FN6"><sup>1</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA.</p><p class="fn sec" id="FN7"><sup>2</sup>Department of Psychiatry and Behavioral Sciences, NorthShore University HealthSystem, Evanston, Illinois, USA.</p><p class="fn sec" id="FN8"><sup>3</sup>Department of Psychiatry and Behavioral Sciences, University of Chicago, Chicago, Illinois, USA.</p><p class="fn sec" id="FN9"><sup>4</sup>Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University School of Medicine, Richmond, Virginia, USA.</p><p class="fn sec" id="FN10"><sup>5</sup>Department of Psychiatry, Virginia Commonwealth University School of Medicine, Richmond, Virginia, USA.</p><p class="fn sec" id="FN11"><sup>6</sup>Department of Human and Molecular Genetics, Virginia Commonwealth University School of Medicine, Richmond, Virginia, USA.</p><p class="fn sec" id="FN12"><sup>7</sup>Department of Psychiatry and Behavioral Sciences, Stanford University, Stanford, California, USA.</p><p class="fn sec" id="FN13"><sup>8</sup>Department of Psychiatry, Mount Sinai School of Medicine, New York, New York, USA.</p><p class="fn sec" id="FN14"><sup>9</sup>Medical Research Council (MRC) Centre for Neuropsychiatric Genetics and Genomics, School of Medicine, Cardiff University, Cardiff, UK.</p><p class="fn sec" id="FN15"><sup>10</sup>Department of Psychological Medicine and Neurology, School of Medicine, Cardiff University, Cardiff, UK.</p><p class="fn sec" id="FN16"><sup>11</sup>Department of Biostatistics, University of North Carolina, Chapel Hill, North Carolina, USA.</p><p class="fn sec" id="FN17"><sup>12</sup>Department of Medical Genetics, University Medical Center Utrecht, Utrecht, The Netherlands.</p><p class="fn sec" id="FN18"><sup>13</sup>Rudolf Magnus Institute of Neuroscience, University Medical Center Utrecht, Utrecht, The Netherlands.</p><p class="fn sec" id="FN19"><sup>14</sup>University of California at Los Angeles (UCLA) Center for Neurobehavioral Genetics, University of California at Los Angeles, Los Angeles, California, USA.</p><p class="fn sec" id="FN20"><sup>15</sup>Department of Human Genetics, University of California at Los Angeles, Los Angeles, California, USA.</p><p class="fn sec" id="FN21"><sup>16</sup>Psychiatry Section, Institute of Clinical Medicine, University of Oslo, Oslo, Norway.</p><p class="fn sec" id="FN22"><sup>17</sup>Department of Psychiatry, Oslo University Hospital, Oslo, Norway.</p><p class="fn sec" id="FN23"><sup>18</sup>Broad Institute, Cambridge, Massachusetts, USA.</p><p class="fn sec" id="FN24"><sup>19</sup>Department of Genomics, Life and Brain Center, University of Bonn, Bonn, Germany.</p><p class="fn sec" id="FN25"><sup>20</sup>Institute of Neuroscience and Medicine (INM-1), Research Center Juelich, Juelich, Germany.</p><p class="fn sec" id="FN26"><sup>21</sup>Institute of Human Genetics, University of Bonn, Bonn, Germany.</p><p class="fn sec" id="FN27"><sup>22</sup>Institute of Medical Sciences, University of Aberdeen, Foresterhill, Aberdeen, UK.</p><p class="fn sec" id="FN28"><sup>23</sup>Neuropsychiatric Genetics Research Group, Trinity College Dublin, Dublin, Ireland.</p><p class="fn sec" id="FN29"><sup>24</sup>Molecular Psychiatry Laboratory, Research Department of Mental Health Sciences, University College London Medical School, Windeyer Institute of Medical Sciences, London, UK.</p><p class="fn sec" id="FN30"><sup>25</sup>Institute of Biological Psychiatry, Mental Health Center (MHC) Sct. Hans, Copenhagen University Hospital, Roskilde, Denmark.</p><p class="fn sec" id="FN31"><sup>26</sup>Molecular and Clinical Neurobiology, Department of Psychiatry, Ludwig-Maximilians-University, Munich, Germany.</p><p class="fn sec" id="FN32"><sup>27</sup>Division of Psychiatry, University of Edinburgh, Royal Edinburgh Hospital, Edinburgh, UK.</p><p class="fn sec" id="FN33"><sup>28</sup>Keck School of Medicine, University of Southern California, Los Angeles, California, USA.</p><p class="fn sec" id="FN34"><sup>29</sup>Department of Psychiatry, Division of Research, The Zucker Hillside Hospital Division of the North Shore-Long Island Jewish Health System, Glen Oaks, New York, USA.</p><p class="fn sec" id="FN35"><sup>30</sup>Center for Psychiatric Neuroscience, The Feinstein Institute for Medical Research, Manhasset, New York, USA.</p><p class="fn sec" id="FN36"><sup>31</sup>Department of Psychiatry and Behavioral Science, Albert Einstein College of Medicine of Yeshiva University, New York, New York, USA.</p><p class="fn sec" id="FN37"><sup>32</sup>Department of Non-Communicable Disease Epidemiology, London School of Hygiene and Tropical Medicine, London, UK.</p><p class="fn sec" id="FN38"><sup>33</sup>Queensland Statistical Genetics Laboratory, Queensland Institute of Medical Research, Brisbane, Queensland, Australia.</p><p class="fn sec" id="FN39"><sup>34</sup>Vrije Universiteit (VU), Center for Neurogenomics and Cognitive Research (CNCR), Department of Functional Genomics, Amsterdam, The Netherlands.</p><p class="fn sec" id="FN40"><sup>35</sup>VU Medical Centre, Department of Medical Genomics, Amsterdam, The Netherlands.</p><p class="fn sec" id="FN41"><sup>36</sup>Washington Veteran’s Affairs Medical Center, Washington, DC, USA.</p><p class="fn sec" id="FN42"><sup>37</sup>Department of Psychiatry, Georgetown University School of Medicine, Washington, DC, USA.</p><p class="fn sec" id="FN43"><sup>38</sup>deCODE Genetics, Reykjavik, Iceland.</p><p class="fn sec" id="FN44"><sup>39</sup>Queensland Brain Institute, University of Queensland, Brisbane, Queensland, Australia.</p><p class="fn sec" id="FN45"><sup>40</sup>Queensland Centre for Mental Health Research, University of Queensland, Brisbane, Queensland, Australia.</p><p class="fn sec" id="FN46"><sup>41</sup>Mental Health Research Center, Russian Academy of Medical Sciences, Moscow, Russia.</p><p class="fn sec" id="FN47"><sup>42</sup>University Psychiatric Centre, Catholic University Leuven, Kortenberg, Belgium.</p><p class="fn sec" id="FN48"><sup>43</sup>Department of Clinical Neuroscience, Human Brain Informatics (HUBIN) Project, Karolinska Institutet and Hospital, Stockholm, Sweden.</p><p class="fn sec" id="FN49"><sup>44</sup>Semmelweis University, Department of Psychiatry and Psychotherapy, Budapest, Hungary.</p><p class="fn sec" id="FN50"><sup>45</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland.</p><p class="fn sec" id="FN51"><sup>46</sup>Department of Medical Genetics, University of Helsinki, Helsinki, Finland.</p><p class="fn sec" id="FN52"><sup>47</sup>Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King’s College, London, UK.</p><p class="fn sec" id="FN53"><sup>48</sup>Section of Psychiatry and Clinical Psychology, University of Verona, Verona, Italy.</p><p class="fn sec" id="FN54"><sup>49</sup>Department of Research, Diakonhjemmet Hospital, Oslo, Norway.</p><p class="fn sec" id="FN55"><sup>50</sup>State Mental Hospital, Haar, Germany.</p><p class="fn sec" id="FN56"><sup>51</sup>Department of Psychiatry and Behavioral Sciences, Emory University, Atlanta, Georgia, USA.</p><p class="fn sec" id="FN57"><sup>52</sup>Department of Psychiatry and Behavioral Sciences, Atlanta Veterans Affairs Medical Center, Atlanta, Georgia, USA.</p><p class="fn sec" id="FN58"><sup>53</sup>Department of Psychiatry, University of Iowa Carver College of Medicine, Iowa City, Iowa, USA.</p><p class="fn sec" id="FN59"><sup>54</sup>Institute of Human Genetics, University of Aarhus, Aarhus, Denmark.</p><p class="fn sec" id="FN60"><sup>55</sup>Centre for Psychiatric Research, Aarhus University Hospital, Risskov, Denmark.</p><p class="fn sec" id="FN61"><sup>56</sup>University of Queensland Diamantina Institute, Princess Alexandra Hospital, University of Queensland, Brisbane, Queensland, Australia.</p><p class="fn sec" id="FN62"><sup>57</sup>University Medical Center Groningen, Department of Psychiatry, University of Groningen, Groningen, The Netherlands.</p><p class="fn sec" id="FN63"><sup>58</sup>School of Nursing, Louisiana State University Health Sciences Center, New Orleans, Louisiana, USA.</p><p class="fn sec" id="FN64"><sup>59</sup>Department of Psychiatry, University of California at San Francisco, San Francisco, California, USA.</p><p class="fn sec" id="FN65"><sup>60</sup>NCIRE (Northern California Institute for Research and Education), San Francisco, California, USA.</p><p class="fn sec" id="FN66"><sup>61</sup>Department of Psychiatry, Rudolf Magnus Institute of Neuroscience, University Medical Center Utrecht, Utrecht, The Netherlands.</p><p class="fn sec" id="FN67"><sup>62</sup>School of Psychiatry, University of New South Wales and Schizophrenia Research Institute, Sydney, New South Wales, Australia.</p><p class="fn sec" id="FN68"><sup>63</sup>Department of Psychiatry, University of Queensland, Royal Brisbane Hospital, Brisbane, Australia.</p><p class="fn sec" id="FN69"><sup>64</sup>Department of Psychiatry, Washington University, St. Louis, Missouri, USA.</p><p class="fn sec" id="FN70"><sup>65</sup>Academic Medical Centre, University of Amsterdam, Department of Psychiatry, Amsterdam, The Netherlands.</p><p class="fn sec" id="FN71"><sup>66</sup>Department of Psychiatry, University of Athens Medical School, Athens, Greece.</p><p class="fn sec" id="FN72"><sup>67</sup>Department of Medical Genetics, Oslo University Hospital, Oslo, Norway.</p><p class="fn sec" id="FN73"><sup>68</sup>Wellcome Trust Centre for Human Genetics, Oxford, UK.</p><p class="fn sec" id="FN74"><sup>69</sup>Department of Statistics, University of Oxford, Oxford, UK.</p><p class="fn sec" id="FN75"><sup>70</sup>Mental Health Center Copenhagen, Copenhagen University Hospital, Copenhagen, Denmark.</p><p class="fn sec" id="FN76"><sup>71</sup>Department of Psychiatry, University of Colorado Denver, Aurora, Colorado, USA.</p><p class="fn sec" id="FN77"><sup>72</sup>Center for Clinical Intervention and Neuropsychiatric Schizophrenia Research, Mental Health Center Glostrup, Copenhagen University Hospital, Glostrup, Denmark.</p><p class="fn sec" id="FN78"><sup>73</sup>INSERM, Institut de Myologie, Hôpital de la Pitié-Salpêtrière, Paris, France.</p><p class="fn sec" id="FN79"><sup>74</sup>Illumina, Inc., La Jolla, California, USA.</p><p class="fn sec" id="FN80"><sup>75</sup>School of Electrical Engineering and Computing Science, University of Newcastle, Newcastle, New South Wales, Australia.</p><p class="fn sec" id="FN81"><sup>76</sup>Section of Neonatal Screening and Hormones, Department of Clinical Chemistry and Immunology, The State Serum Institute, Copenhagen, Denmark.</p><p class="fn sec" id="FN82"><sup>77</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden.</p><p class="fn sec" id="FN83"><sup>78</sup>Centre for Clinical Research in Neuropsychiatry, School of Psychiatry and Clinical Neurosciences, The University of Western Australia, Perth, Western Australia, Australia.</p><p class="fn sec" id="FN84"><sup>79</sup>Department of Child and Adolescent Psychiatry, Pierre and Marie Curie Faculty of Medicine, Paris, France.</p><p class="fn sec" id="FN85"><sup>80</sup>Department of Clinical Pharmacology, Bispebjerg University Hospital, Copenhagen, Denmark.</p><p class="fn sec" id="FN86"><sup>81</sup>Department of Psychology, University of Colorado, Boulder, Boulder, Colorado, USA.</p><p class="fn sec" id="FN87"><sup>82</sup>Department of Psychiatry and Psychology, School of Mental Health and Neuroscience, European Graduate School of Neuroscience (EURON), South Limburg Mental Health Research and Teaching Network (SEARCH), Maastricht University Medical Centre, Maastricht, The Netherlands.</p><p class="fn sec" id="FN88"><sup>83</sup>Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, USA.</p><p class="fn sec" id="FN89"><sup>84</sup>Maastricht University Medical Centre, South Limburg Mental Health Research and Teaching Network, Maastricht, The Netherlands.</p><p class="fn sec" id="FN90"><sup>85</sup>Department of Psychiatry and Behavioral Sciences, Johns Hopkins University School of Medicine, Baltimore, Maryland, USA.</p><p class="fn sec" id="FN91"><sup>86</sup>Department of Psychiatry, Hadassah-Hebrew University Medical Center, Jerusalem, Israel.</p><p class="fn sec" id="FN92"><sup>87</sup>Department of Biostatistics, Johns Hopkins University Bloomberg School of Public Health, Baltimore, Maryland, USA.</p><p class="fn sec" id="FN93"><sup>88</sup>Department of Psychiatry, Columbia University, New York, New York, USA.</p><p class="fn sec" id="FN94"><sup>89</sup>Department of Mental Health and Substance Abuse Services, National Institute for Health and Welfare, Helsinki, Finland.</p><p class="fn sec" id="FN95"><sup>90</sup>Schizophrenia Research Institute, Sydney and Centre for Brain and Mental Health Research, University of Newcastle, Newcastle, New South Wales, Australia.</p><p class="fn sec" id="FN96"><sup>91</sup>Department of Psychiatry, University of Bonn, Bonn, Germany.</p><p class="fn sec" id="FN97"><sup>92</sup>Laboratoire de Génétique Moléculaire de la Neurotransmission et des Processus Neurodégénératifs, Centre National de la Recherche Scientifique, Hôpital de la Pitié Salpêtrière, Paris, France.</p><p class="fn sec" id="FN98"><sup>93</sup>Institute of Medical Biometry, Informatics and Epidemiology (IMBIE), University of Bonn, Bonn, Germany.</p><p class="fn sec" id="FN99"><sup>94</sup>Department of Research, Sørlandet Hospital, Kristiansand, Norway.</p><p class="fn sec" id="FN100"><sup>95</sup>Queensland Centre for Mental Health Research, The Park Centre for Mental Health, Wacol, Queensland, Australia.</p><p class="fn sec" id="FN101"><sup>96</sup>Functional NeuroImaging Laboratory, School of Psychology, University of Newcastle, Sydney, New South Wales, Australia.</p><p class="fn sec" id="FN102"><sup>97</sup>Schizophrenia Research Institute, Sydney, New South Wales, Australia.</p><p class="fn sec" id="FN103"><sup>98</sup>Department of Psychiatry, First Psychiatric Clinic, Alexander University Hospital, Sofia, Bulgaria.</p><p class="fn sec" id="FN104"><sup>99</sup>National Centre for Register-Based Research, University of Aarhus, Aarhus, Denmark.</p><p class="fn sec" id="FN105"><sup>100</sup>Department of Psychiatry, University of Toronto, Toronto, Canada.</p><p class="fn sec" id="FN106"><sup>101</sup>NeuroSearch A/S, Ballerup, Denmark.</p><p class="fn sec" id="FN107"><sup>102</sup>Unit for Psychiatric Research, Aalborg Psychiatric Hospital, Aalborg, Denmark.</p><p class="fn sec" id="FN108"><sup>103</sup>Psychiatric Centre Copenhagen, Copenhagen University Hospital, Copenhagen, Denmark.</p><p class="fn sec" id="FN109"><sup>104</sup>Department of Psychiatry, Queens University, Belfast, Ireland.</p><p class="fn sec" id="FN110"><sup>105</sup>ARoS Applied Biotechnology A/S, Skejby, Denmark.</p><p class="fn sec" id="FN111"><sup>106</sup>Department of Molecular Medicine, Aarhus University Hospital, Skejby, Denmark.</p><p class="fn sec" id="FN112"><sup>107</sup>Melbourne Neuropsychiatry Centre, Department of Psychiatry, The University of Melbourne and Melbourne Health, Melbourne, Victoria, Australia.</p><p class="fn sec" id="FN113"><sup>108</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="fn sec" id="FN114"><sup>109</sup>Department of Psychiatry, National University Hospital, Reykjavik, Iceland.</p><p class="fn sec" id="FN115"><sup>110</sup>Strathclyde Institute of Pharmacy and Biomedical Sciences, The John Arbuthnott Building, University of Strathclyde, Glasgow, UK.</p><p class="fn sec" id="FN116"><sup>111</sup>Department of Psychiatry, University of Oxford, Warneford Hospital, Headington, Oxford, UK.</p><p class="fn sec" id="FN117"><sup>112</sup>Department of Genetic Epidemiology in Psychiatry, Central Institute of Mental Health, University of Heidelberg, Mannheim, Germany.</p><p class="fn sec" id="FN118"><sup>113</sup>Priority Centre for Brain and Mental Health Research, University of Newcastle, Sydney, New South Wales, Australia.</p><p class="fn sec" id="FN119"><sup>114</sup>Department of Psychiatry and Psychotherapy, Georg-August-University, Göttingen, Germany.</p><p class="fn sec" id="FN120"><sup>115</sup>School of Psychiatry and Clinical Neurosciences, University of Western Australia, Perth, Western Australia, Australia.</p><p class="fn sec" id="FN121"><sup>116</sup>Department of Psychiatry, University of Erlangen-Nuremberg, Erlangen, Germany.</p><p class="fn sec" id="FN122"><sup>117</sup>Centre for Medical Research, Western Australian Institute for Medical Research, University of Western Australia, Perth, Western Australia, Australia.</p><p class="fn sec" id="FN123"><sup>118</sup>Centre for Information Based Medicine, University of Newcastle, Hunter Medical Research Institute, Newcastle and Schizophrenia Research Institute, Sydney, New South Wales, Australia.</p><p class="fn sec" id="FN124"><sup>119</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, Maryland, USA.</p><p class="fn sec" id="FN125"><sup>120</sup>Department of Psychiatry, University of Iceland, Reykjavik, Iceland.</p><p class="fn sec" id="FN126"><sup>121</sup>Department of Psychiatry, Veterans Affairs Medical Center, New York, New York, USA.</p><p class="fn sec" id="FN127"><sup>122</sup>West Berkshire National Health Service (NHS) Trust, Reading, UK.</p><p class="fn sec" id="FN128"><sup>123</sup>Mental Health Center Frederiksberg, Copenhagen University Hospital, Copenhagen, Denmark.</p><p class="fn sec" id="FN129"><sup>124</sup>Department of Medical Genetics, University Hospital Maichin Dom, Sofia, Bulgaria.</p><p class="fn sec" id="FN130"><sup>125</sup>Department of Pharmacy, Virginia Commonwealth University, Richmond, Virginia, USA.</p><p class="fn sec" id="FN131"><sup>126</sup>Rudolf Magnus Institute of Neuroscience, Department of Neurology, Universitair Medisch Centrum (UMC) Utrecht, Utrecht, The Netherlands.</p><p class="fn sec" id="FN132"><sup>127</sup>The Health Research Board, Dublin, Ireland.</p><p class="fn sec" id="FN133"><sup>128</sup>Mental Health Center Amager, Copenhagen University Hospital, Copenhagen, Denmark.</p><p class="fn sec" id="FN134"><sup>129</sup>Centre for Clinical Research in Neuropsychiatry, Graylands Hospital, Mt Claremont, Western Australia, Australia.</p><p class="fn sec" id="FN135"><sup>130</sup>Department of Psychiatry, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, USA.</p><p class="fn sec" id="FN136"><sup>131</sup>Department of Epidemiology, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, USA.</p><p class="fn sec" id="FN137"><sup>132</sup>Deceased. Correspondence should be addressed to P.V.G. (<a href="mailto:dev@null" data-email="moc.liamg@namjegp" class="oemail">moc.liamg@namjegp</a>).</p></div></div><div id="idm140107429548928" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140107429548928title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Altshuler DM, et al.  Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;author=DM+Altshuler&amp;volume=467&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Barrett JC, et al.  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn’s disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:955–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn’s+disease&amp;author=JC+Barrett&amp;volume=40&amp;publication_year=2008&amp;pages=955-962&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Lango Allen H, et al.  Hundreds of variants clustered in genomic loci and biological pathways affect human height. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:832–838.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955183/">PMC free article</a>]</span> [<a href="/pubmed/20881960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hundreds+of+variants+clustered+in+genomic+loci+and+biological+pathways+affect+human+height&amp;author=H+Lango+Allen&amp;volume=467&amp;publication_year=2010&amp;pages=832-838&amp;pmid=20881960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Purcell SM, et al.  Common polygenic variation contributes to risk of schizophrenia and bipolar disorder. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:748–752.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia+and+bipolar+disorder&amp;author=SM+Purcell&amp;volume=460&amp;publication_year=2009&amp;pages=748-752&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Price AL, et al.  Discerning the ancestry of European Americans in genetic association studies. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e236.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2211542/">PMC free article</a>]</span> [<a href="/pubmed/18208327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Discerning+the+ancestry+of+European+Americans+in+genetic+association+studies&amp;author=AL+Price&amp;volume=4&amp;publication_year=2008&amp;pages=e236&amp;pmid=18208327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Dudbridge F, Gusnanto A. Estimation of significance thresholds for genomewide association scans. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:227–234.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2573032/">PMC free article</a>]</span> [<a href="/pubmed/18300295" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+significance+thresholds+for+genomewide+association+scans&amp;author=F+Dudbridge&amp;author=A+Gusnanto&amp;volume=32&amp;publication_year=2008&amp;pages=227-234&amp;pmid=18300295&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Stefansson H, et al.  Common variants conferring risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:744–747.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3077530/">PMC free article</a>]</span> [<a href="/pubmed/19571808" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+variants+conferring+risk+of+schizophrenia&amp;author=H+Stefansson&amp;volume=460&amp;publication_year=2009&amp;pages=744-747&amp;pmid=19571808&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Shi J, et al.  Common variants on chromosome 6p22.1 are associated with schizophrenia. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:753–757.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775422/">PMC free article</a>]</span> [<a href="/pubmed/19571809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+variants+on+chromosome+6p22.1+are+associated+with+schizophrenia&amp;author=J+Shi&amp;volume=460&amp;publication_year=2009&amp;pages=753-757&amp;pmid=19571809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Ferreira MA, et al.  Collaborative genome-wide association analysis supports a role for <em>ANK3</em> and <em>CACNA1C</em> in bipolar disorder. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1056–1058.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2703780/">PMC free article</a>]</span> [<a href="/pubmed/18711365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Collaborative+genome-wide+association+analysis+supports+a+role+for+ANK3+and+CACNA1C+in+bipolar+disorder&amp;author=MA+Ferreira&amp;volume=40&amp;publication_year=2008&amp;pages=1056-1058&amp;pmid=18711365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Green EK, et al.  The bipolar disorder risk allele at <em>CACNA1C</em> also confers risk of recurrent major depression and of schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2010;<span class="ref-vol">15</span>:1016–1022.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3011210/">PMC free article</a>]</span> [<a href="/pubmed/19621016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=The+bipolar+disorder+risk+allele+at+CACNA1C+also+confers+risk+of+recurrent+major+depression+and+of+schizophrenia&amp;author=EK+Green&amp;volume=15&amp;publication_year=2010&amp;pages=1016-1022&amp;pmid=19621016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Ruderfer DM, et al.  A family-based study of common polygenic variation and risk of schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2011 April 12; doi: 10.1038/mp.2011.34.</span> published online. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4189087/">PMC free article</a>]</span> [<a href="/pubmed/21483432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fmp.2011.34" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=A+family-based+study+of+common+polygenic+variation+and+risk+of+schizophrenia&amp;author=DM+Ruderfer&amp;publication_year=2011&amp;doi=10.1038/mp.2011.34&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Bemis LT, et al.  MicroRNA-137 targets microphthalmia-associated transcription factor in melanoma cell lines. <span><span class="ref-journal">Cancer Res. </span>2008;<span class="ref-vol">68</span>:1362–1368.</span> [<a href="/pubmed/18316599" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=MicroRNA-137+targets+microphthalmia-associated+transcription+factor+in+melanoma+cell+lines&amp;author=LT+Bemis&amp;volume=68&amp;publication_year=2008&amp;pages=1362-1368&amp;pmid=18316599&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Srikantan V, et al.  <em>PCGEM1</em>, a prostate-specific gene, is overexpressed in prostate cancer. <span><span class="ref-journal">Proc Natl Acad SciUSA. </span>2000;<span class="ref-vol">97</span>:12216–12221.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC17321/">PMC free article</a>]</span> [<a href="/pubmed/11050243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+SciUSA&amp;title=PCGEM1,+a+prostate-specific+gene,+is+overexpressed+in+prostate+cancer&amp;author=V+Srikantan&amp;volume=97&amp;publication_year=2000&amp;pages=12216-12221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Szulwach KE, et al.  Cross talk between microRNA and epigenetic regulation in adult neurogenesis. <span><span class="ref-journal">J Cell Biol. </span>2010;<span class="ref-vol">189</span>:127–141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2854370/">PMC free article</a>]</span> [<a href="/pubmed/20368621" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=Cross+talk+between+microRNA+and+epigenetic+regulation+in+adult+neurogenesis&amp;author=KE+Szulwach&amp;volume=189&amp;publication_year=2010&amp;pages=127-141&amp;pmid=20368621&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Silber J, et al.  miR-124 and miR-137 inhibit proliferation of glioblastoma multiforme cells and induce differentiation of brain tumor stem cells. <span><span class="ref-journal">BMC Med. </span>2008;<span class="ref-vol">6</span>:14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2443372/">PMC free article</a>]</span> [<a href="/pubmed/18577219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med&amp;title=miR-124+and+miR-137+inhibit+proliferation+of+glioblastoma+multiforme+cells+and+induce+differentiation+of+brain+tumor+stem+cells&amp;author=J+Silber&amp;volume=6&amp;publication_year=2008&amp;pages=14&amp;pmid=18577219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Smrt RD, et al.  MicroRNA miR-137 regulates neuronal maturation by targeting ubiquitin ligase mind bomb-1. <span><span class="ref-journal">Stem Cells. </span>2010;<span class="ref-vol">28</span>:1060–1070.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3140955/">PMC free article</a>]</span> [<a href="/pubmed/20506192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stem+Cells&amp;title=MicroRNA+miR-137+regulates+neuronal+maturation+by+targeting+ubiquitin+ligase+mind+bomb-1&amp;author=RD+Smrt&amp;volume=28&amp;publication_year=2010&amp;pages=1060-1070&amp;pmid=20506192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Potkin SG, et al.  Identifying gene regulatory networks in schizophrenia. <span><span class="ref-journal">Neuroimage. </span>2010;<span class="ref-vol">53</span>:839–847.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3055795/">PMC free article</a>]</span> [<a href="/pubmed/20600988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Identifying+gene+regulatory+networks+in+schizophrenia&amp;author=SG+Potkin&amp;volume=53&amp;publication_year=2010&amp;pages=839-847&amp;pmid=20600988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. <span><span class="ref-journal">Cell. </span>2005;<span class="ref-vol">120</span>:15–20.</span> [<a href="/pubmed/15652477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Conserved+seed+pairing,+often+flanked+by+adenosines,+indicates+that+thousands+of+human+genes+are+microRNA+targets&amp;author=BP+Lewis&amp;author=CB+Burge&amp;author=DP+Bartel&amp;volume=120&amp;publication_year=2005&amp;pages=15-20&amp;pmid=15652477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Krek A, et al.  Combinatorial microRNA target predictions. <span><span class="ref-journal">Nat Genet. </span>2005;<span class="ref-vol">37</span>:495–500.</span> [<a href="/pubmed/15806104" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Combinatorial+microRNA+target+predictions&amp;author=A+Krek&amp;volume=37&amp;publication_year=2005&amp;pages=495-500&amp;pmid=15806104&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">John B, et al.  Human MicroRNA targets. <span><span class="ref-journal">PLoS Biol. </span>2004;<span class="ref-vol">2</span>:e363.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC521178/">PMC free article</a>]</span> [<a href="/pubmed/15502875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Human+MicroRNA+targets&amp;author=B+John&amp;volume=2&amp;publication_year=2004&amp;pages=e363&amp;pmid=15502875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Yang J, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;volume=42&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Yang J, et al.  Genomic inflation factors under polygenic inheritance. <span><span class="ref-journal">Eur J Hum Genet. </span>2011;<span class="ref-vol">19</span>:807–812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3137506/">PMC free article</a>]</span> [<a href="/pubmed/21407268" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Genomic+inflation+factors+under+polygenic+inheritance&amp;author=J+Yang&amp;volume=19&amp;publication_year=2011&amp;pages=807-812&amp;pmid=21407268&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Visscher PM, Yang J, Goddard ME. A commentary on ‘common SNPs explain a large proportion of the heritability for human height’ by Yang <em>et al</em>. (2010) <span><span class="ref-journal">Twin Res Hum Genet. </span>2010;<span class="ref-vol">13</span>:517–524.</span> [<a href="/pubmed/21142928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Hum+Genet&amp;title=A+commentary+on+‘common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height’+by+Yang+et+al.+(2010)&amp;author=PM+Visscher&amp;author=J+Yang&amp;author=ME+Goddard&amp;volume=13&amp;publication_year=2010&amp;pages=517-524&amp;pmid=21142928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Lee SH, Wray NR, Goddard ME, Visscher PM. Estimating missing heritability for disease from genome-wide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:294–305.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059431/">PMC free article</a>]</span> [<a href="/pubmed/21376301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Estimating+missing+heritability+for+disease+from+genome-wide+association+studies&amp;author=SH+Lee&amp;author=NR+Wray&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=294-305&amp;pmid=21376301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Speliotes EK, et al.  Association analyses of 249,796 individuals reveal 18 new loci associated with body mass index. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:937–948.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014648/">PMC free article</a>]</span> [<a href="/pubmed/20935630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+analyses+of+249,796+individuals+reveal+18+new+loci+associated+with+body+mass+index&amp;author=EK+Speliotes&amp;volume=42&amp;publication_year=2010&amp;pages=937-948&amp;pmid=20935630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Wei Z, et al.  From disease association to risk assessment: an optimistic view from genome-wide association studies on type 1 diabetes. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748686/">PMC free article</a>]</span> [<a href="/pubmed/19816555" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=From+disease+association+to+risk+assessment:+an+optimistic+view+from+genome-wide+association+studies+on+type+1+diabetes&amp;author=Z+Wei&amp;volume=5&amp;publication_year=2009&amp;pages=e1000678&amp;pmid=19816555&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Bush WS, et al.  Evidence for polygenic susceptibility to multiple sclerosis—the shape of things to come. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">86</span>:621–625.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850422/">PMC free article</a>]</span> [<a href="/pubmed/20362272" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Evidence+for+polygenic+susceptibility+to+multiple+sclerosis—the+shape+of+things+to+come&amp;author=WS+Bush&amp;volume=86&amp;publication_year=2010&amp;pages=621-625&amp;pmid=20362272&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Lichtenstein P, et al.  Common genetic determinants of schizophrenia and bipolar disorder in Swedish families: a population-based study. <span><span class="ref-journal">Lancet. </span>2009;<span class="ref-vol">373</span>:234–239.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3879718/">PMC free article</a>]</span> [<a href="/pubmed/19150704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Common+genetic+determinants+of+schizophrenia+and+bipolar+disorder+in+Swedish+families:+a+population-based+study&amp;author=P+Lichtenstein&amp;volume=373&amp;publication_year=2009&amp;pages=234-239&amp;pmid=19150704&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Scott LJ, et al.  Genome-wide association and meta-analysis of bipolar disorder in individuals of European ancestry. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2009;<span class="ref-vol">106</span>:7501–7506.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2678639/">PMC free article</a>]</span> [<a href="/pubmed/19416921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Genome-wide+association+and+meta-analysis+of+bipolar+disorder+in+individuals+of+European+ancestry&amp;author=LJ+Scott&amp;volume=106&amp;publication_year=2009&amp;pages=7501-7506&amp;pmid=19416921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3303194&amp;issue-id=206567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140107433757056title">Abstract</a></li><li><a href="#S3title">METHODS</a></li><li><a href="#S4title">Supplementary Material</a></li><li><a href="#S5title">Acknowledgments</a></li><li><a href="#idm140107472427904title">Footnotes</a></li><li><a href="#idm140107429548928title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3303194/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3303194/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3303194/pdf/nihms360727.pdf">PDF (1.4M)</a></li> | <li><a href="#" data-citationid="PMC3303194" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3303194%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3303194%2F&amp;text=Genome-wide%20association%20study%20identifies%20five%20new%20schizophrenia%20loci" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3303194%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21926974" data-aiid="3303194" data-aid="3303194" data-iid="206567" data-domainid="319" data-domain="nihpa" data-accid="PMC3303194" data-md5="98b717175f9b562f26d61bcece6bc150"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:45:06-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D4174DD534CB10000000000500038&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3303194%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3303194/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>