<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3056271/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A GENOME WIDE ASSOCIATION STUDY FOR DIABETIC NEPHROPATHY GENES IN AFRICAN AMERICANS</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3056271/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Kidney international"><meta name="citation_title" content="A GENOME WIDE ASSOCIATION STUDY FOR DIABETIC NEPHROPATHY GENES IN AFRICAN AMERICANS"><meta name="citation_authors" content="Caitrin W. McDonough, Nicholette D. Palmer, Pamela J. Hicks, Bong H. Roh, S. Sandy An, Jessica N. Cooke, Jessica M. Hester, Maria R. Wing, Meredith A. Bostrom, Megan E. Rudock, Joshua P. Lewis, Matthew E. Talbert, Rebecca A. Blevins, Lingyi Lu, Maggie C.Y. Ng, Michele M. Sale, Jasmin Divers, Carl D. Langefeld, Barry I. Freedman, Donald W. Bowden"><meta name="citation_date" content="March 2011"><meta name="citation_issue" content="5"><meta name="citation_volume" content="79"><meta name="citation_firstpage" content="563"><meta name="citation_doi" content="10.1038/ki.2010.467"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3056271/?report=abstract"><meta name="citation_pmid" content="21150874"><meta name="DC.Title" content="A GENOME WIDE ASSOCIATION STUDY FOR DIABETIC NEPHROPATHY GENES IN AFRICAN AMERICANS"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Caitrin W. McDonough"><meta name="DC.Contributor" content="Nicholette D. Palmer"><meta name="DC.Contributor" content="Pamela J. Hicks"><meta name="DC.Contributor" content="Bong H. Roh"><meta name="DC.Contributor" content="S. Sandy An"><meta name="DC.Contributor" content="Jessica N. Cooke"><meta name="DC.Contributor" content="Jessica M. Hester"><meta name="DC.Contributor" content="Maria R. Wing"><meta name="DC.Contributor" content="Meredith A. Bostrom"><meta name="DC.Contributor" content="Megan E. Rudock"><meta name="DC.Contributor" content="Joshua P. Lewis"><meta name="DC.Contributor" content="Matthew E. Talbert"><meta name="DC.Contributor" content="Rebecca A. Blevins"><meta name="DC.Contributor" content="Lingyi Lu"><meta name="DC.Contributor" content="Maggie C.Y. Ng"><meta name="DC.Contributor" content="Michele M. Sale"><meta name="DC.Contributor" content="Jasmin Divers"><meta name="DC.Contributor" content="Carl D. Langefeld"><meta name="DC.Contributor" content="Barry I. Freedman"><meta name="DC.Contributor" content="Donald W. Bowden"><meta name="DC.Date" content="2011 Mar"><meta name="DC.Identifier" content="10.1038/ki.2010.467"><meta name="DC.Language" content="en"><meta property="og:title" content="A GENOME WIDE ASSOCIATION STUDY FOR DIABETIC NEPHROPATHY GENES IN AFRICAN AMERICANS"><meta property="og:type" content="article"><meta property="og:description" content="A genome-wide association study was performed using the Affymetrix 6.0 chip to identify genes associated with diabetic nephropathy in African Americans. Association analysis was performed adjusting for admixture in 965 type 2 diabetic African American ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3056271/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3056271/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3056271/pdf/nihms271243.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A16B8DD520E5100000000011000B1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3056271%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3056271</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Kidney Int</a></span>. Author manuscript; available in PMC 2012 Mar 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21150874" target="pmc_ext" ref="reftype=publisher&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Kidney Int. 2011 Mar; 79(5): 563–572. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Dec 8. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fki.2010.467" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ki.2010.467</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3056271</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS271243</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21150874">21150874</a></div></div></div></div><h1 class="content-title">A GENOME WIDE ASSOCIATION STUDY FOR DIABETIC NEPHROPATHY GENES IN AFRICAN AMERICANS</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=McDonough%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173904188080" co-class="co-affbox">Caitrin W. McDonough</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Palmer%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173891503552" co-class="co-affbox">Nicholette D. Palmer</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Hicks%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173936587776" co-class="co-affbox">Pamela J. Hicks</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Roh%20BH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173937196576" co-class="co-affbox">Bong H. Roh</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=An%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173891341856" co-class="co-affbox">S. Sandy An</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Cooke%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173901864752" co-class="co-affbox">Jessica N. Cooke</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Hester%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173903493776" co-class="co-affbox">Jessica M. Hester</a>,<sup>2,</sup><sup>3,</sup><sup>5</sup> <a href="/pubmed/?term=Wing%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173936827776" co-class="co-affbox">Maria R. Wing</a>,<sup>2,</sup><sup>3,</sup><sup>5</sup> <a href="/pubmed/?term=Bostrom%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173902055920" co-class="co-affbox">Meredith A. Bostrom</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Rudock%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173897179456" co-class="co-affbox">Megan E. Rudock</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Lewis%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173897176496" co-class="co-affbox">Joshua P. Lewis</a>,<sup>2,</sup><sup>3,</sup><sup>5</sup> <a href="/pubmed/?term=Talbert%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173894639184" co-class="co-affbox">Matthew E. Talbert</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Blevins%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173975380512" co-class="co-affbox">Rebecca A. Blevins</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Lu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173901751376" co-class="co-affbox">Lingyi Lu</a>,<sup>6</sup> <a href="/pubmed/?term=Ng%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173937278960" co-class="co-affbox">Maggie C.Y. Ng</a>,<sup>2,</sup><sup>3,</sup><sup>7</sup> <a href="/pubmed/?term=Sale%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173893328304" co-class="co-affbox">Michele M. Sale</a>,<sup>9</sup> <a href="/pubmed/?term=Divers%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173893326080" co-class="co-affbox">Jasmin Divers</a>,<sup>6</sup> <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173936385136" co-class="co-affbox">Carl D. Langefeld</a>,<sup>6</sup> <a href="/pubmed/?term=Freedman%20BI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173936382912" co-class="co-affbox">Barry I. Freedman</a>,<sup>8</sup> and  <a href="/pubmed/?term=Bowden%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874" class="affpopup" co-rid="_co_idm140173895627344" co-class="co-affbox">Donald W. Bowden</a><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>8</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140173904188080"><h3 class="no_margin">Caitrin W. McDonough</h3><p><sup>1</sup>Program in Molecular Medicine and Translational Science, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=McDonough%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Caitrin W. McDonough</a></div></div><div id="_co_idm140173891503552"><h3 class="no_margin">Nicholette D. Palmer</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>4</sup>Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Palmer%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Nicholette D. Palmer</a></div></div><div id="_co_idm140173936587776"><h3 class="no_margin">Pamela J. Hicks</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>4</sup>Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Hicks%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Pamela J. Hicks</a></div></div><div id="_co_idm140173937196576"><h3 class="no_margin">Bong H. Roh</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>4</sup>Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Roh%20BH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Bong H. Roh</a></div></div><div id="_co_idm140173891341856"><h3 class="no_margin">S. Sandy An</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>4</sup>Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=An%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">S. Sandy An</a></div></div><div id="_co_idm140173901864752"><h3 class="no_margin">Jessica N. Cooke</h3><p><sup>1</sup>Program in Molecular Medicine and Translational Science, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Cooke%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Jessica N. Cooke</a></div></div><div id="_co_idm140173903493776"><h3 class="no_margin">Jessica M. Hester</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>5</sup>Program in Molecular Genetics and Genomics, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Hester%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Jessica M. Hester</a></div></div><div id="_co_idm140173936827776"><h3 class="no_margin">Maria R. Wing</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>5</sup>Program in Molecular Genetics and Genomics, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Wing%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Maria R. Wing</a></div></div><div id="_co_idm140173902055920"><h3 class="no_margin">Meredith A. Bostrom</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>4</sup>Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Bostrom%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Meredith A. Bostrom</a></div></div><div id="_co_idm140173897179456"><h3 class="no_margin">Megan E. Rudock</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Rudock%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Megan E. Rudock</a></div></div><div id="_co_idm140173897176496"><h3 class="no_margin">Joshua P. Lewis</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>5</sup>Program in Molecular Genetics and Genomics, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Lewis%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Joshua P. Lewis</a></div></div><div id="_co_idm140173894639184"><h3 class="no_margin">Matthew E. Talbert</h3><p><sup>1</sup>Program in Molecular Medicine and Translational Science, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Talbert%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Matthew E. Talbert</a></div></div><div id="_co_idm140173975380512"><h3 class="no_margin">Rebecca A. Blevins</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>4</sup>Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Blevins%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Rebecca A. Blevins</a></div></div><div id="_co_idm140173901751376"><h3 class="no_margin">Lingyi Lu</h3><p><sup>6</sup>Department of Biostatistical Sciences, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Lu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Lingyi Lu</a></div></div><div id="_co_idm140173937278960"><h3 class="no_margin">Maggie C.Y. Ng</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>7</sup>Department of Pediatrics, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Ng%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Maggie C.Y. Ng</a></div></div><div id="_co_idm140173893328304"><h3 class="no_margin">Michele M. Sale</h3><p><sup>9</sup>Center for Public Health Genomics, University of Virginia School of Medicine, Charlottesville, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Sale%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Michele M. Sale</a></div></div><div id="_co_idm140173893326080"><h3 class="no_margin">Jasmin Divers</h3><p><sup>6</sup>Department of Biostatistical Sciences, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Divers%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Jasmin Divers</a></div></div><div id="_co_idm140173936385136"><h3 class="no_margin">Carl D. Langefeld</h3><p><sup>6</sup>Department of Biostatistical Sciences, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Carl D. Langefeld</a></div></div><div id="_co_idm140173936382912"><h3 class="no_margin">Barry I. Freedman</h3><p><sup>8</sup>Department of Internal Medicine, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Freedman%20BI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Barry I. Freedman</a></div></div><div id="_co_idm140173895627344"><h3 class="no_margin">Donald W. Bowden</h3><p><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>4</sup>Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><p><sup>8</sup>Department of Internal Medicine, Wake Forest University School of Medicine, Winston-Salem, NC 27157</p><div>Find articles by <a href="/pubmed/?term=Bowden%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150874">Donald W. Bowden</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140173894405984_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140173894405984_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140173894405984_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Program in Molecular Medicine and Translational Science, Wake Forest University School of Medicine, Winston-Salem, NC 27157</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Center for Genomics and Personalized Medicine Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Center for Diabetes Research, Wake Forest University School of Medicine, Winston-Salem, NC 27157</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Biochemistry, Wake Forest University School of Medicine, Winston-Salem, NC 27157</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Program in Molecular Genetics and Genomics, Wake Forest University School of Medicine, Winston-Salem, NC 27157</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Biostatistical Sciences, Wake Forest University School of Medicine, Winston-Salem, NC 27157</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Pediatrics, Wake Forest University School of Medicine, Winston-Salem, NC 27157</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Internal Medicine, Wake Forest University School of Medicine, Winston-Salem, NC 27157</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Center for Public Health Genomics, University of Virginia School of Medicine, Charlottesville, VA, USA</div><div id="cor1">Address correspondence to: Donald W. Bowden, Ph.D., Department of Biochemistry, Wake Forest University School of Medicine, Medical Center Blvd., Winston-Salem, NC 27157, USA, Telephone: 336-713-7507 FAX: 336-713-7566, <a href="mailto:dev@null" data-email="ude.cmbufw@nedwobd" class="oemail">ude.cmbufw@nedwobd</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140173894405984_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21150874" target="pmc_ext" ref="reftype=publisher&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Kidney Int</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3056271/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="4" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp.</strong></div><div><a href="/pmc/articles/PMC3056271/bin/NIHMS271243-supplement-Supp.doc" data-ga-action="click_feat_suppl">NIHMS271243-supplement-Supp.doc</a><span style="color:gray"> (406K)</span></div><div class="small guid">GUID:&nbsp;48341673-EC1C-461C-8C2B-ADAE3C13FE8A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table 1.</strong></div><div><a href="/pmc/articles/PMC3056271/bin/NIHMS271243-supplement-Supp_Table_1.xls" data-ga-action="click_feat_suppl">NIHMS271243-supplement-Supp_Table_1.xls</a><span style="color:gray"> (342K)</span></div><div class="small guid">GUID:&nbsp;3C0B6AB0-528D-4E62-A848-E30FC0E2686D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table 3.</strong></div><div><a href="/pmc/articles/PMC3056271/bin/NIHMS271243-supplement-Supp_Table_3.xls" data-ga-action="click_feat_suppl">NIHMS271243-supplement-Supp_Table_3.xls</a><span style="color:gray"> (45K)</span></div><div class="small guid">GUID:&nbsp;54179601-1D6F-4AEF-8C90-F3F818EDE3F3</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table 5.</strong></div><div><a href="/pmc/articles/PMC3056271/bin/NIHMS271243-supplement-Supp_Table_5.xls" data-ga-action="click_feat_suppl">NIHMS271243-supplement-Supp_Table_5.xls</a><span style="color:gray"> (360K)</span></div><div class="small guid">GUID:&nbsp;B4976257-BBF9-43D3-A854-0D6138114184</div></div></dd></dl></div><div id="idm140173895412928" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140173895412928title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">A genome-wide association study was performed using the Affymetrix 6.0 chip to identify genes associated with diabetic nephropathy in African Americans. Association analysis was performed adjusting for admixture in 965 type 2 diabetic African American patients with end-stage renal disease (ESRD) and in 1029 African Americans without type 2 diabetes or kidney disease as controls. The top 724 single nucleotide polymorphisms (SNPs) with evidence of association to diabetic nephropathy were then genotyped in a replication sample of an additional 709 type 2 diabetes-ESRD patients and 690 controls. SNPs with evidence of association in both the original and replication studies were tested in additional African American cohorts consisting of 1246 patients with type 2 diabetes without kidney disease and 1216 with non-diabetic ESRD to differentiate candidate loci for type 2 diabetes-ESRD, type 2 diabetes, and/or all-cause ESRD. Twenty-five SNPs were significantly associated with type 2 diabetes-ESRD in the genome-wide association and initial replication. Although genome-wide significance with type 2 diabetes was not found for any of these 25 SNPs, several genes, including RPS12, LIMK2, and SFI1 are strong candidates for diabetic nephropathy. A combined analysis of all 2890 patients with ESRD showed significant association SNPs in LIMK2 and SFI1 suggesting that they also contribute to all-cause ESRD. Thus, our results suggest that multiple loci underlie susceptibility to kidney disease in African Americans with type 2 diabetes and some may also contribute to all-cause ESRD.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Diabetes-associated nephropathy is the most common cause of end-stage renal disease (ESRD) in the United States, accounting for 44.6% of incident cases (<a href="#R1" rid="R1" class=" bibr popnode">1</a>). African Americans (AA) have a disproportionately high risk for developing diabetic nephropathy (DN). Compared to Caucasian Americans (CA), AAs have a 3.7-fold greater incidence rate of developing ESRD and at least a 1.9-fold greater incidence rate compared to other racial and ethnic minorites in the United States (<a href="#R1" rid="R1" class=" bibr popnode">1</a>). Many studies have shown that there is a genetic component to ESRD as reviewed by Bowden, 2003 (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918237">2</a>). Familial aggregation of DN and diabetic ESRD has been demonstrated in CAs (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918228">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918286">5</a>), and AAs (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918259">6</a>). Clustering occurred in these families without significant differences in glycemic control (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918265">4</a>). However, marked racial and ethnic disparities in familial clustering exist. CAs with a close relative who has ESRD face a 2.7-fold increased risk of developing ESRD (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918243">5</a>); whereas AAs who have a close relative with ESRD have a 9-fold increased risk of developing ESRD (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918229">6</a>). This significant difference in rates of renal complications between CAs and AAs is observed after controlling for differences in socio-economic status (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918233">5</a>, <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918242">6</a>).</p><p id="P3" class="p p-last">Several studies have attempted to detect genetic variants influencing risk of DN and diabetic ESRD. The first genome-wide association study (GWAS) for DN was a low density (80K SNPs) gene-based study performed in a Japanese population (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918235">7</a>). This was followed by a study using a DNA pooling method investigating 115K SNPs in Pima Indians with DN (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918230">8</a>). More recently, a GWAS was conducted for type 1 diabetes-associated nephropathy in a CA population (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918250">9</a>), and multiple studies have assessed for association with chronic kidney disease (CKD), and glomerular filtration rate (GFR) in non-diabetic populations of European ancestry (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304663227">10</a>–<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_336923417">12</a>). There have been no prior reports of GWAS in AAs with T2DM-ESRD. Herein, we report the first GWAS investigating 832K SNPs for association with type 2 diabetes-associated ESRD (T2DM-ESRD) in AAs.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Results</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Clinical characteristics of the study samples</h3><p id="P4" class="p p-first-last">The clinical characteristics of the study samples used in the GWAS, replication, and trait discrimination phases are shown in <a href="/pmc/articles/PMC3056271/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The GWAS and replication populations are broadly similar. In both groups, the age at enrollment for the T2DM-ESRD subjects is older than for the control groups. However, the age at enrollment for the control groups in the GWAS and replication phases is older than the age of T2DM diagnosis in the T2DM-ESRD and T2DM subjects. All of the case groups with T2DM (T2DM-ESRD and T2DM) have a higher proportion of females; possibly reflecting the increased prevalence of T2DM among AA women (<a href="#R13" rid="R13" class=" bibr popnode">13</a>), participation bias, and survival. On average, all of the groups were overweight or obese at the time of enrollment. Subjects with ESRD lacking T2DM (non-T2DM ESRD subjects) had the lowest average body mass index (BMI; 27.0 kg/m², <a href="/pmc/articles/PMC3056271/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), and the T2DM subjects without nephropathy (T2DM) had the highest average BMI (33.5 kg/m², <a href="/pmc/articles/PMC3056271/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Clinical Characteristics of Study Samples</strong><p id="P40">Values are presented as trait mean and standard deviation.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140173894702464" class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="center"></th><th colspan="2" rowspan="1" valign="middle" align="center">GWAS</th><th colspan="2" rowspan="1" valign="middle" align="center">Replication</th><th colspan="2" rowspan="1" valign="middle" align="center">Trait Discrimination</th></tr><tr><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center">T2DM-ESRD</th><th rowspan="1" colspan="1" align="center">Control</th><th rowspan="1" colspan="1" align="center">T2DM-ESRD</th><th rowspan="1" colspan="1" align="center">Control</th><th rowspan="1" colspan="1" align="center">T2DM</th><th rowspan="1" colspan="1" align="center">non-T2DM ESRD</th></tr><tr><th rowspan="1" colspan="1" align="center"></th><th colspan="2" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="center"><hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center"><em>n</em></td><td rowspan="1" colspan="1" align="center">965</td><td rowspan="1" colspan="1" align="center">1029</td><td rowspan="1" colspan="1" align="center">709</td><td rowspan="1" colspan="1" align="center">690</td><td rowspan="1" colspan="1" align="center">1246</td><td rowspan="1" colspan="1" align="center">1216</td></tr><tr><td rowspan="1" colspan="1" align="center">Female (%)</td><td rowspan="1" colspan="1" align="center">61.20%</td><td rowspan="1" colspan="1" align="center">57.30%</td><td rowspan="1" colspan="1" align="center">55.70%</td><td rowspan="1" colspan="1" align="center">51.30%</td><td rowspan="1" colspan="1" align="center">64.00%</td><td rowspan="1" colspan="1" align="center">44.70%</td></tr><tr><td rowspan="1" colspan="1" align="center">Age at Enrollment (years)</td><td rowspan="1" colspan="1" align="center">61.6 ± 10.5</td><td rowspan="1" colspan="1" align="center">49.0 ± 11.9</td><td rowspan="1" colspan="1" align="center">60.2 ± 10.4</td><td rowspan="1" colspan="1" align="center">48.5 ± 12.8</td><td rowspan="1" colspan="1" align="center">57.2 ± 11.7</td><td rowspan="1" colspan="1" align="center">53.0 ± 14.5</td></tr><tr><td rowspan="1" colspan="1" align="center">Age at T2D diagnosis (years)</td><td rowspan="1" colspan="1" align="center">41.6 ± 12.4</td><td rowspan="1" colspan="1" align="center">―</td><td rowspan="1" colspan="1" align="center">39.4 ± 12.5</td><td rowspan="1" colspan="1" align="center">―</td><td rowspan="1" colspan="1" align="center">46.1 ± 12.6</td><td rowspan="1" colspan="1" align="center">―</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">Age at ESRD diagnosis<br>(years)</td><td rowspan="1" colspan="1" valign="bottom" align="center">58.0 ± 10.9</td><td rowspan="1" colspan="1" valign="bottom" align="center">―</td><td rowspan="1" colspan="1" valign="bottom" align="center">56.7 ± 10.9</td><td rowspan="1" colspan="1" valign="bottom" align="center">―</td><td rowspan="1" colspan="1" valign="bottom" align="center">―</td><td rowspan="1" colspan="1" valign="bottom" align="center">47.7 ± 15.5</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">T2D to ESRD duration<br>(years)</td><td rowspan="1" colspan="1" valign="bottom" align="center">16.2 ± 10.9</td><td rowspan="1" colspan="1" valign="bottom" align="center">―</td><td rowspan="1" colspan="1" valign="bottom" align="center">20.4 ± 10.5</td><td rowspan="1" colspan="1" valign="bottom" align="center">―</td><td rowspan="1" colspan="1" valign="bottom" align="center">―</td><td rowspan="1" colspan="1" valign="bottom" align="center">―</td></tr><tr><td rowspan="1" colspan="1" align="center">BMI (kg/m²)</td><td rowspan="1" colspan="1" align="center">29.7 ± 7.0</td><td rowspan="1" colspan="1" align="center">30.0 ± 7.0</td><td rowspan="1" colspan="1" align="center">29.8 ± 6.9</td><td rowspan="1" colspan="1" align="center">29.4 ± 7.6</td><td rowspan="1" colspan="1" align="center">33.5 ± 7.6</td><td rowspan="1" colspan="1" align="center">27.0 ± 7.0</td></tr></tbody></table></div><div id="largeobj_idm140173894702464" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3056271/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="S4" class="sec"><h3 id="S4title">GWAS</h3><p id="P5" class="p p-first-last">After the application of SNP and sample quality control metrics, 832,357 autosomal SNPs were analyzed in 965 AA T2DM-ESRD case subjects and 1,029 AA non-diabetic, non-nephropathy controls. A summary of the association results is shown in <a href="/pmc/articles/PMC3056271/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> and the corresponding quantile-quantile plot is shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 1</a>. The results shown are adjusted for admixture; however, the primary inferences remain the same adjusting for admixture, age, and gender. The top hit was rs5750250 located on chromosome 22 in the <em>MYH9</em> (non-muscle myosin heavy chain 9) gene (<em>P</em> = 3.00 ×10<sup>−7</sup>, <a href="/pmc/articles/PMC3056271/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). This gene has been previously associated with non-diabetic and diabetic forms of ESRD (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918249">14</a>–<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918262">17</a>). In total, there were 126 SNPs with <em>P</em> values &lt;1.0×10<sup>−4</sup> (<a href="/pmc/articles/PMC3056271/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). The flow of the study through the GWAS, replication, combined and trait discrimination phases is outlined in <a href="/pmc/articles/PMC3056271/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3056271/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140173900341888" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms271243f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms271243f1.jpg" src="/pmc/articles/PMC3056271/bin/nihms271243f1.jpg"></div></a><div id="largeobj_idm140173900341888" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3056271/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3056271/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3056271/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genome-Wide Association Study Results</strong><p id="P38">The top hit was rs5750250 located on chromosome 22 in the <em>MYH9</em> gene (<em>P</em> =3.00×10<sup>−7</sup>). Results are adjusted for admixture using PC1 as a covariate in the analysis. <em>P</em> values are shown under the additive model.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Study Design and Progression</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140173893817072" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="center">Step</th><th rowspan="1" colspan="1" align="center">Stage</th><th colspan="3" rowspan="1" valign="bottom" align="center">Sample<hr></th><th rowspan="1" colspan="1" align="center">Description</th></tr><tr><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center">Total</th><th rowspan="1" colspan="1" align="center">Cases</th><th rowspan="1" colspan="1" align="center">Controls</th><th rowspan="1" colspan="1" align="center"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">GWAS</td><td rowspan="1" colspan="1" valign="middle" align="center">1994</td><td rowspan="1" colspan="1" valign="middle" align="center">965 T2DM-ESRD Cases</td><td rowspan="1" colspan="1" valign="middle" align="center">1029 non-diabetic,<br>non-nephropathy Controls</td><td rowspan="1" colspan="1" valign="middle" align="center">GWAS scan with 832,357 SNPs</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">Replication</td><td rowspan="1" colspan="1" valign="middle" align="center">1399</td><td rowspan="1" colspan="1" valign="middle" align="center">709 T2DM-ESRD Cases</td><td rowspan="1" colspan="1" valign="middle" align="center">690 non-diabetic,<br>non-nephropathy Controls</td><td rowspan="1" colspan="1" valign="middle" align="center">Test 724 SNPs from<br>Step 1 for replication</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">Combined Analysis<br>(GWAS + Replication)</td><td rowspan="1" colspan="1" valign="middle" align="center">3393</td><td rowspan="1" colspan="1" valign="middle" align="center">1674 T2DM-ESRD Cases</td><td rowspan="1" colspan="1" valign="middle" align="center">1719 non-diabetic,<br>non-nephropathy Controls</td><td rowspan="1" colspan="1" valign="middle" align="center">Identify SNPs with the<br>strongest evidence of association<br>with T2DM-ESRD</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">Trait Discrimination:<br>Test for T2DM or T2DM-<br>ESRD</td><td rowspan="1" colspan="1" valign="middle" align="center">2920</td><td rowspan="1" colspan="1" valign="middle" align="center">1674 T2DM-ESRD Cases</td><td rowspan="1" colspan="1" valign="middle" align="center">1246 T2DM<br>(lacking nephropathy) Cases</td><td rowspan="1" colspan="1" valign="middle" align="center">Test 67 SNPs from Step 3 to<br>differentiate between<br>T2DM-ESRD &amp; T2DM loci</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">Test for non-T2DM ESRD</td><td rowspan="1" colspan="1" valign="middle" align="center">2935</td><td rowspan="1" colspan="1" valign="middle" align="center">1216 non-T2DM ESRD<br>Cases</td><td rowspan="1" colspan="1" valign="middle" align="center">1719 non-diabetic,<br>non-nephropathy Controls</td><td rowspan="1" colspan="1" valign="middle" align="center">Test 67 SNPs from Step 3 for<br>association with All-cause ESRD</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">Test for All-Cause ESRD</td><td rowspan="1" colspan="1" valign="middle" align="center">4609</td><td rowspan="1" colspan="1" valign="middle" align="center">2890 ESRD Cases<br>(1674 T2DM-ESRD and<br>1216 non-T2DM ESRD)</td><td rowspan="1" colspan="1" valign="middle" align="center">1719 non-diabetic,<br>non-nephropathy Controls</td><td rowspan="1" colspan="1" valign="middle" align="center">Test 67 SNPs from Step 3 for<br>association with All-cause ESRD<br>in the largest sample set</td></tr></tbody></table></div><div id="largeobj_idm140173893817072" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3056271/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S5" class="sec"><h3 id="S5title">Replication and Combined Analysis of T2DM-ESRD cases and non-diabetic, non-nephropathy controls</h3><p id="P6" class="p p-first">In an effort to replicate the GWAS results, 724 top scoring SNPs were genotyped in an independent sample of 709 AA T2DM-ESRD cases and 690 AA non-diabetic, non-nephropathy controls (Study Design, <a href="/pmc/articles/PMC3056271/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; Results, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table 1</a>). The 724 SNPs that were selected for testing in the replication sample were SNPs with the strongest <em>P</em> values for association and with high quality scores for genotyping, i.e. missing rate less than 0.02 (or missing rate less than 0.05 but missing rate between cases and controls not significantly different). In addition, <em>P</em> values for Hardy-Weinberg proportions for cases were &gt; 0.0001 and for controls &gt; 0.01. SNPs were prioritized on significance of the additive genetic model unless there was <em>a priori</em> evidence for follow-up, e.g. <em>MYH9</em>. In this replication analysis, 67 SNPs showed nominal evidence of replication: additive <em>P</em> value &lt;0.05 with association in the same direction (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table 1</a>). <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> summarizes the association results for 25 SNPs at 19 potential T2DM-ESRD loci. T2DM-ESRD loci were identified during the trait discrimination analyses as shown in <a href="/pmc/articles/PMC3056271/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and were based on the following criteria: 1) associated in the replication phase, 2) associated in the T2DM-ESRD v T2DM, non-nephropathy comparison (<a href="/pmc/articles/PMC3056271/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>), and 3) showed no or nominal association (<em>P</em> &gt;0.01) in the T2DM, non-nephropathy v Controls comparison (<a href="/pmc/articles/PMC3056271/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> shows <em>P</em> values for association in the GWAS, the replication sample, and the combined cohort: 1,674 T2DM-ESRD cases and 1,719 non-diabetic, non-nephropathy controls. No SNP reached genome-wide significance (<em>P</em> ≤ 5×10<sup>−8</sup>), <em>P</em> values ranged from 1.24×10<sup>−4</sup> to 7.04×10<sup>−7</sup>, <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, Combined analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Diabetic Nephropathy Candidate Loci</strong><p id="P41">GWAS, Replication, and Combined (GWAS+Replication) <em>P</em> values for diabetic nephropathy candidate loci across the genome.</p><p id="P42">MAF: Minor Allele Frequency. OR: Odds Ratio. 95% CI: 95% Confidence Interval.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140173901231232" class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><thead><tr><th colspan="5" rowspan="1" valign="bottom" align="center">Locus<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">GWAS<br>T2DM-ESRD (n=965)<br>Controls (n=1029)<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">Replication<br>T2DM-ESRD (n=709)<br>Controls (n=690)<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Combined<br>T2DM-ESRD (n=1674)<br>Controls (n=1719)<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Marker</th><th rowspan="1" colspan="1" valign="middle" align="center">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">Position</th><th rowspan="1" colspan="1" valign="middle" align="center">Nearest<br>Gene(s)</th><th rowspan="1" colspan="1" valign="middle" align="center">Minor<br>Allele<br>(Major<br>Allele)</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>T2DM-<br>ESRD</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>Control</th><th rowspan="1" colspan="1" valign="middle" align="center">Additive<br><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>T2DM-<br>ESRD</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>Control</th><th rowspan="1" colspan="1" valign="middle" align="center">Additive<br><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="center">Additive<br><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="center">OR<br>(95% CI)</th></tr><tr><th colspan="5" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="center"><hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">rs10888287</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">246,184,039</td><td rowspan="1" colspan="1" align="center">OR2L13</td><td rowspan="1" colspan="1" align="center">G(T)</td><td rowspan="1" colspan="1" align="center">0.439</td><td rowspan="1" colspan="1" align="center">0.503</td><td rowspan="1" colspan="1" align="center">5.03E-04</td><td rowspan="1" colspan="1" align="center">0.444</td><td rowspan="1" colspan="1" align="center">0.495</td><td rowspan="1" colspan="1" align="center">0.0482</td><td rowspan="1" colspan="1" align="center">5.97E-05</td><td rowspan="1" colspan="1" align="center">0.82 (0.75–0.90)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs4260465</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">164,231,046</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">G(C)</td><td rowspan="1" colspan="1" align="center">0.364</td><td rowspan="1" colspan="1" align="center">0.414</td><td rowspan="1" colspan="1" align="center">4.31E-04</td><td rowspan="1" colspan="1" align="center">0.381</td><td rowspan="1" colspan="1" align="center">0.421</td><td rowspan="1" colspan="1" align="center">0.0382</td><td rowspan="1" colspan="1" align="center">4.50E-05</td><td rowspan="1" colspan="1" align="center">0.81 (0.74–0.90)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11730446</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">96,382,267</td><td rowspan="1" colspan="1" align="center">UNC5C</td><td rowspan="1" colspan="1" align="center">G(A)</td><td rowspan="1" colspan="1" align="center">0.459</td><td rowspan="1" colspan="1" align="center">0.403</td><td rowspan="1" colspan="1" align="center">6.23E-04</td><td rowspan="1" colspan="1" align="center">0.459</td><td rowspan="1" colspan="1" align="center">0.424</td><td rowspan="1" colspan="1" align="center">0.0465</td><td rowspan="1" colspan="1" align="center">1.24E-04</td><td rowspan="1" colspan="1" align="center">1.21 (1.10–1.33)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs891382</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">147,371,790</td><td rowspan="1" colspan="1" align="center">SLC10A7/LSM6</td><td rowspan="1" colspan="1" align="center">G(A)</td><td rowspan="1" colspan="1" align="center">0.085</td><td rowspan="1" colspan="1" align="center">0.059</td><td rowspan="1" colspan="1" align="center">3.19E-04</td><td rowspan="1" colspan="1" align="center">0.081</td><td rowspan="1" colspan="1" align="center">0.065</td><td rowspan="1" colspan="1" align="center">0.0183</td><td rowspan="1" colspan="1" align="center">2.42E-05</td><td rowspan="1" colspan="1" align="center">1.49 (1.24–1.80)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7697691</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">182,853,951</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">G(A)</td><td rowspan="1" colspan="1" align="center">0.243</td><td rowspan="1" colspan="1" align="center">0.285</td><td rowspan="1" colspan="1" align="center">0.0016</td><td rowspan="1" colspan="1" align="center">0.243</td><td rowspan="1" colspan="1" align="center">0.285</td><td rowspan="1" colspan="1" align="center">0.0127</td><td rowspan="1" colspan="1" align="center">7.37E-05</td><td rowspan="1" colspan="1" align="center">0.80 (0.72–0.89)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs3822908</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">97,706,333</td><td rowspan="1" colspan="1" align="center">C6orf167</td><td rowspan="1" colspan="1" align="center">C(T)</td><td rowspan="1" colspan="1" align="center">0.125</td><td rowspan="1" colspan="1" align="center">0.162</td><td rowspan="1" colspan="1" align="center">3.63E-04</td><td rowspan="1" colspan="1" align="center">0.121</td><td rowspan="1" colspan="1" align="center">0.147</td><td rowspan="1" colspan="1" align="center">0.0462</td><td rowspan="1" colspan="1" align="center">4.52E-05</td><td rowspan="1" colspan="1" align="center">0.75 (0.65–0.86)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7760831</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">102,071,536</td><td rowspan="1" colspan="1" align="center">GRIK2</td><td rowspan="1" colspan="1" align="center">C(T)</td><td rowspan="1" colspan="1" align="center">0.176</td><td rowspan="1" colspan="1" align="center">0.222</td><td rowspan="1" colspan="1" align="center">2.55E-04</td><td rowspan="1" colspan="1" align="center">0.193</td><td rowspan="1" colspan="1" align="center">0.224</td><td rowspan="1" colspan="1" align="center">0.0455</td><td rowspan="1" colspan="1" align="center">3.48E-05</td><td rowspan="1" colspan="1" align="center">0.77 (0.69–0.87)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs208865</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">130,114,249</td><td rowspan="1" colspan="1" align="center">C6orf191/ARHGAP18</td><td rowspan="1" colspan="1" align="center">T(C)</td><td rowspan="1" colspan="1" align="center">0.118</td><td rowspan="1" colspan="1" align="center">0.082</td><td rowspan="1" colspan="1" align="center">2.14E-04</td><td rowspan="1" colspan="1" align="center">0.115</td><td rowspan="1" colspan="1" align="center">0.091</td><td rowspan="1" colspan="1" align="center">0.0364</td><td rowspan="1" colspan="1" align="center">2.90E-05</td><td rowspan="1" colspan="1" align="center">1.41 (1.20–1.66)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs9493454</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">133,186,322</td><td rowspan="1" colspan="1" align="center">RPS12</td><td rowspan="1" colspan="1" align="center">C(A)</td><td rowspan="1" colspan="1" align="center">0.439</td><td rowspan="1" colspan="1" align="center">0.376</td><td rowspan="1" colspan="1" align="center">4.91E-04</td><td rowspan="1" colspan="1" align="center">0.453</td><td rowspan="1" colspan="1" align="center">0.395</td><td rowspan="1" colspan="1" align="center">0.0089</td><td rowspan="1" colspan="1" align="center">1.72E-05</td><td rowspan="1" colspan="1" align="center">1.24 (1.13–1.37)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7769051</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">133,188,489</td><td rowspan="1" colspan="1" align="center">RPS12</td><td rowspan="1" colspan="1" align="center">A(C)</td><td rowspan="1" colspan="1" align="center">0.349</td><td rowspan="1" colspan="1" align="center">0.289</td><td rowspan="1" colspan="1" align="center">4.06E-04</td><td rowspan="1" colspan="1" align="center">0.362</td><td rowspan="1" colspan="1" align="center">0.3</td><td rowspan="1" colspan="1" align="center">0.0012</td><td rowspan="1" colspan="1" align="center">2.20E-06</td><td rowspan="1" colspan="1" align="center">1.28 (1.16–1.42)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs6930576</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">148,746,647</td><td rowspan="1" colspan="1" align="center">SASH1</td><td rowspan="1" colspan="1" align="center">A(G)</td><td rowspan="1" colspan="1" align="center">0.337</td><td rowspan="1" colspan="1" align="center">0.276</td><td rowspan="1" colspan="1" align="center">1.87E-05</td><td rowspan="1" colspan="1" align="center">0.319</td><td rowspan="1" colspan="1" align="center">0.274</td><td rowspan="1" colspan="1" align="center">0.0099</td><td rowspan="1" colspan="1" align="center">7.04E-07</td><td rowspan="1" colspan="1" align="center">1.31 (1.18–1.45)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs773506</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">93,015,292</td><td rowspan="1" colspan="1" align="center">AUH</td><td rowspan="1" colspan="1" align="center">A(G)</td><td rowspan="1" colspan="1" align="center">0.179</td><td rowspan="1" colspan="1" align="center">0.233</td><td rowspan="1" colspan="1" align="center">3.27E-04</td><td rowspan="1" colspan="1" align="center">0.174</td><td rowspan="1" colspan="1" align="center">0.223</td><td rowspan="1" colspan="1" align="center">0.0130</td><td rowspan="1" colspan="1" align="center">6.45E-06</td><td rowspan="1" colspan="1" align="center">0.76 (0.67–0.85)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11175885</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">64,400,897</td><td rowspan="1" colspan="1" align="center">C12orf66/TMEM5</td><td rowspan="1" colspan="1" align="center">G(A)</td><td rowspan="1" colspan="1" align="center">0.181</td><td rowspan="1" colspan="1" align="center">0.234</td><td rowspan="1" colspan="1" align="center">0.0013</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">0.249</td><td rowspan="1" colspan="1" align="center">0.0106</td><td rowspan="1" colspan="1" align="center">2.18E-05</td><td rowspan="1" colspan="1" align="center">0.77 (0.68–0.87)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2358944</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">64,403,825</td><td rowspan="1" colspan="1" align="center">MSRB3/HMGA2</td><td rowspan="1" colspan="1" align="center">A(G)</td><td rowspan="1" colspan="1" align="center">0.178</td><td rowspan="1" colspan="1" align="center">0.232</td><td rowspan="1" colspan="1" align="center">9.48E-04</td><td rowspan="1" colspan="1" align="center">0.188</td><td rowspan="1" colspan="1" align="center">0.256</td><td rowspan="1" colspan="1" align="center">0.0018</td><td rowspan="1" colspan="1" align="center">3.54E-06</td><td rowspan="1" colspan="1" align="center">0.75 (0.67–0.85)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2904532</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">65,582,311</td><td rowspan="1" colspan="1" align="center">GRIP1/CAND1</td><td rowspan="1" colspan="1" align="center">G(A)</td><td rowspan="1" colspan="1" align="center">0.141</td><td rowspan="1" colspan="1" align="center">0.183</td><td rowspan="1" colspan="1" align="center">8.17E-04</td><td rowspan="1" colspan="1" align="center">0.148</td><td rowspan="1" colspan="1" align="center">0.184</td><td rowspan="1" colspan="1" align="center">0.0172</td><td rowspan="1" colspan="1" align="center">4.87E-05</td><td rowspan="1" colspan="1" align="center">0.76 (0.67–0.87)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11176482</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">65,582,814</td><td rowspan="1" colspan="1" align="center">GRIP1/CAND1</td><td rowspan="1" colspan="1" align="center">C(T)</td><td rowspan="1" colspan="1" align="center">0.141</td><td rowspan="1" colspan="1" align="center">0.184</td><td rowspan="1" colspan="1" align="center">7.08E-04</td><td rowspan="1" colspan="1" align="center">0.147</td><td rowspan="1" colspan="1" align="center">0.182</td><td rowspan="1" colspan="1" align="center">0.0202</td><td rowspan="1" colspan="1" align="center">4.79E-05</td><td rowspan="1" colspan="1" align="center">0.76 (0.67–0.87)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs12302041</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">76,654,356</td><td rowspan="1" colspan="1" align="center">NAV3</td><td rowspan="1" colspan="1" align="center">A(G)</td><td rowspan="1" colspan="1" align="center">0.085</td><td rowspan="1" colspan="1" align="center">0.055</td><td rowspan="1" colspan="1" align="center">3.09E-04</td><td rowspan="1" colspan="1" align="center">0.088</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">0.0149</td><td rowspan="1" colspan="1" align="center">1.15E-05</td><td rowspan="1" colspan="1" align="center">1.54 (1.27–1.87)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs1978243</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">86,596,160</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">T(C)</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">0.274</td><td rowspan="1" colspan="1" align="center">7.53E-04</td><td rowspan="1" colspan="1" align="center">0.221</td><td rowspan="1" colspan="1" align="center">0.274</td><td rowspan="1" colspan="1" align="center">0.0105</td><td rowspan="1" colspan="1" align="center">1.61E-05</td><td rowspan="1" colspan="1" align="center">0.78 (0.70–0.87)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs6494387</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">61,132,579</td><td rowspan="1" colspan="1" align="center">TPM1</td><td rowspan="1" colspan="1" align="center">A(G)</td><td rowspan="1" colspan="1" align="center">0.367</td><td rowspan="1" colspan="1" align="center">0.311</td><td rowspan="1" colspan="1" align="center">2.32E-04</td><td rowspan="1" colspan="1" align="center">0.358</td><td rowspan="1" colspan="1" align="center">0.32</td><td rowspan="1" colspan="1" align="center">0.0209</td><td rowspan="1" colspan="1" align="center">1.80E-05</td><td rowspan="1" colspan="1" align="center">1.25 (1.13–1.39)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs1034589</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,909,233</td><td rowspan="1" colspan="1" align="center">RNF185</td><td rowspan="1" colspan="1" align="center">C(T)</td><td rowspan="1" colspan="1" align="center">0.036</td><td rowspan="1" colspan="1" align="center">0.062</td><td rowspan="1" colspan="1" align="center">0.0010</td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">0.065</td><td rowspan="1" colspan="1" align="center">0.0055</td><td rowspan="1" colspan="1" align="center">1.06E-05</td><td rowspan="1" colspan="1" align="center">0.59 (0.46–0.74)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2106294</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,975,759</td><td rowspan="1" colspan="1" align="center">LIMK2</td><td rowspan="1" colspan="1" align="center">C(T)</td><td rowspan="1" colspan="1" align="center">0.035</td><td rowspan="1" colspan="1" align="center">0.062</td><td rowspan="1" colspan="1" align="center">7.49E-04</td><td rowspan="1" colspan="1" align="center">0.031</td><td rowspan="1" colspan="1" align="center">0.064</td><td rowspan="1" colspan="1" align="center">0.0028</td><td rowspan="1" colspan="1" align="center">4.11E-06</td><td rowspan="1" colspan="1" align="center">0.57 (0.45–0.72)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs4820043</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,977,094</td><td rowspan="1" colspan="1" align="center">LIMK2</td><td rowspan="1" colspan="1" align="center">A(G)</td><td rowspan="1" colspan="1" align="center">0.036</td><td rowspan="1" colspan="1" align="center">0.062</td><td rowspan="1" colspan="1" align="center">0.0010</td><td rowspan="1" colspan="1" align="center">0.031</td><td rowspan="1" colspan="1" align="center">0.065</td><td rowspan="1" colspan="1" align="center">0.0026</td><td rowspan="1" colspan="1" align="center">5.07E-06</td><td rowspan="1" colspan="1" align="center">0.57 (0.45–0.73)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs5749286</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">30,230,359</td><td rowspan="1" colspan="1" align="center">SFI1</td><td rowspan="1" colspan="1" align="center">A(C)</td><td rowspan="1" colspan="1" align="center">0.038</td><td rowspan="1" colspan="1" align="center">0.065</td><td rowspan="1" colspan="1" align="center">0.0013</td><td rowspan="1" colspan="1" align="center">0.036</td><td rowspan="1" colspan="1" align="center">0.068</td><td rowspan="1" colspan="1" align="center">0.0030</td><td rowspan="1" colspan="1" align="center">9.79E-06</td><td rowspan="1" colspan="1" align="center">0.60 (0.47–0.75)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs16996381</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">34,841,225</td><td rowspan="1" colspan="1" align="center">APOL3</td><td rowspan="1" colspan="1" align="center">G(A)</td><td rowspan="1" colspan="1" align="center">0.336</td><td rowspan="1" colspan="1" align="center">0.391</td><td rowspan="1" colspan="1" align="center">0.0018</td><td rowspan="1" colspan="1" align="center">0.371</td><td rowspan="1" colspan="1" align="center">0.432</td><td rowspan="1" colspan="1" align="center">0.0191</td><td rowspan="1" colspan="1" align="center">1.06E-04</td><td rowspan="1" colspan="1" align="center">0.82 (0.75–0.91)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs735853</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">35,009,161</td><td rowspan="1" colspan="1" align="center">MYH9</td><td rowspan="1" colspan="1" align="center">G(C)</td><td rowspan="1" colspan="1" align="center">0.082</td><td rowspan="1" colspan="1" align="center">0.123</td><td rowspan="1" colspan="1" align="center">4.59E-04</td><td rowspan="1" colspan="1" align="center">0.079</td><td rowspan="1" colspan="1" align="center">0.118</td><td rowspan="1" colspan="1" align="center">0.0091</td><td rowspan="1" colspan="1" align="center">1.20E-05</td><td rowspan="1" colspan="1" align="center">0.69 (0.59–0.82)</td></tr></tbody></table></div><div id="largeobj_idm140173901231232" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3056271/table/T3/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Diabetic Nephropathy Candidate Loci: Trait Discrimination</strong><p id="P43">Comparison of T2DM-ESRD versus T2DM, non-nephropathy subjects and T2DM, non-nephropathy subjects versus non-diabetic normal controls in diabetic nephropathy candidate loci across the genome.</p><p id="P44">OR: Odds Ratio. 95% CI: 95% Confidence Interval</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140173900873712" class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="bottom" align="center">Locus<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">T2DM-ESRD (n=1674)<br>v<br>T2DM (n=1246)<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">T2DM (n=1246)<br>v<br>Controls (n=1719)<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Marker</th><th rowspan="1" colspan="1" valign="middle" align="center">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">Position</th><th rowspan="1" colspan="1" valign="middle" align="center">Nearest<br>Gene(s)</th><th rowspan="1" colspan="1" valign="middle" align="center">Additive<br><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">Additive<br><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th></tr><tr><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="center"><hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">rs10888287</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">246,184,039</td><td rowspan="1" colspan="1" align="center">OR2L13</td><td rowspan="1" colspan="1" align="center">0.0361</td><td rowspan="1" colspan="1" align="center">0.89 (0.80–0.99)</td><td rowspan="1" colspan="1" align="center">0.1487</td><td rowspan="1" colspan="1" align="center">0.93 (0.84–1.03)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs4260465</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">164,231,046</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">2.63E-05</td><td rowspan="1" colspan="1" align="center">0.80 (0.72–0.89)</td><td rowspan="1" colspan="1" align="center">0.5177</td><td rowspan="1" colspan="1" align="center">1.03 (0.93–1.15)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11730446</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">96,382,267</td><td rowspan="1" colspan="1" align="center">UNC5C</td><td rowspan="1" colspan="1" align="center">0.0068</td><td rowspan="1" colspan="1" align="center">1.16 (1.04–1.29)</td><td rowspan="1" colspan="1" align="center">0.3319</td><td rowspan="1" colspan="1" align="center">1.05 (0.95–1.17)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs891382</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">147,371,790</td><td rowspan="1" colspan="1" align="center">SLC10A7/LSM6</td><td rowspan="1" colspan="1" align="center">1.86E-04</td><td rowspan="1" colspan="1" align="center">1.49 (1.21–1.83)</td><td rowspan="1" colspan="1" align="center">0.8984</td><td rowspan="1" colspan="1" align="center">1.01 (0.82–1.25)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7697691</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">182,853,951</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">0.0046</td><td rowspan="1" colspan="1" align="center">0.84 (0.75–0.95)</td><td rowspan="1" colspan="1" align="center">0.3664</td><td rowspan="1" colspan="1" align="center">0.95 (0.84–1.07)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs3822908</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">97,706,333</td><td rowspan="1" colspan="1" align="center">C6orf167</td><td rowspan="1" colspan="1" align="center">0.0336</td><td rowspan="1" colspan="1" align="center">0.84 (0.72–0.99)</td><td rowspan="1" colspan="1" align="center">0.1979</td><td rowspan="1" colspan="1" align="center">0.91 (0.78–1.05)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7760831</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">102,071,536</td><td rowspan="1" colspan="1" align="center">GRIK2</td><td rowspan="1" colspan="1" align="center">0.0494</td><td rowspan="1" colspan="1" align="center">0.87 (0.76–1.00)</td><td rowspan="1" colspan="1" align="center">0.0587</td><td rowspan="1" colspan="1" align="center">0.88 (0.78–1.00)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs208865</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">130,114,249</td><td rowspan="1" colspan="1" align="center">C6orf191/ARHGAP18</td><td rowspan="1" colspan="1" align="center">0.0220</td><td rowspan="1" colspan="1" align="center">1.23 (1.03–1.46)</td><td rowspan="1" colspan="1" align="center">0.1259</td><td rowspan="1" colspan="1" align="center">1.15 (0.96–1.37)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs9493454</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">133,186,322</td><td rowspan="1" colspan="1" align="center">RPS12</td><td rowspan="1" colspan="1" align="center">8.79E-04</td><td rowspan="1" colspan="1" align="center">1.20 (1.08–1.34)</td><td rowspan="1" colspan="1" align="center">0.5079</td><td rowspan="1" colspan="1" align="center">1.04 (0.93–1.16)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7769051</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">133,188,489</td><td rowspan="1" colspan="1" align="center">RPS12</td><td rowspan="1" colspan="1" align="center">0.0036</td><td rowspan="1" colspan="1" align="center">1.18 (1.06–1.32)</td><td rowspan="1" colspan="1" align="center">0.1614</td><td rowspan="1" colspan="1" align="center">1.09 (0.97–1.22)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs6930576</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">148,746,647</td><td rowspan="1" colspan="1" align="center">SASH1</td><td rowspan="1" colspan="1" align="center">0.0053</td><td rowspan="1" colspan="1" align="center">1.18 (1.05–1.32)</td><td rowspan="1" colspan="1" align="center">0.0621</td><td rowspan="1" colspan="1" align="center">1.12 (0.99–1.25)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs773506</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">93,015,292</td><td rowspan="1" colspan="1" align="center">AUH</td><td rowspan="1" colspan="1" align="center">2.57E-04</td><td rowspan="1" colspan="1" align="center">0.78 (0.69–0.89)</td><td rowspan="1" colspan="1" align="center">0.7484</td><td rowspan="1" colspan="1" align="center">0.98 (0.87–1.11)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11175885</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">64,400,897</td><td rowspan="1" colspan="1" align="center">C12orf66/TMEM5</td><td rowspan="1" colspan="1" align="center">0.0226</td><td rowspan="1" colspan="1" align="center">0.86 (0.75–0.98)</td><td rowspan="1" colspan="1" align="center">0.1068</td><td rowspan="1" colspan="1" align="center">0.90 (0.79–1.02)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2358944</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">64,403,825</td><td rowspan="1" colspan="1" align="center">MSRB3/HMGA2</td><td rowspan="1" colspan="1" align="center">0.0047</td><td rowspan="1" colspan="1" align="center">0.83 (0.72–0.94)</td><td rowspan="1" colspan="1" align="center">0.1532</td><td rowspan="1" colspan="1" align="center">0.91 (0.80–1.03)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2904532</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">65,582,311</td><td rowspan="1" colspan="1" align="center">GRIP1/CAND1</td><td rowspan="1" colspan="1" align="center">0.0047</td><td rowspan="1" colspan="1" align="center">0.81 (0.71–0.94)</td><td rowspan="1" colspan="1" align="center">0.3862</td><td rowspan="1" colspan="1" align="center">0.94 (0.82–1.08)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11176482</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">65,582,814</td><td rowspan="1" colspan="1" align="center">GRIP1/CAND1</td><td rowspan="1" colspan="1" align="center">0.0070</td><td rowspan="1" colspan="1" align="center">0.82 (0.71–0.95)</td><td rowspan="1" colspan="1" align="center">0.3052</td><td rowspan="1" colspan="1" align="center">0.93 (0.81–1.07)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs12302041</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">76,654,356</td><td rowspan="1" colspan="1" align="center">NAV3</td><td rowspan="1" colspan="1" align="center">0.0226</td><td rowspan="1" colspan="1" align="center">1.27 (1.03–1.55)</td><td rowspan="1" colspan="1" align="center">0.0924</td><td rowspan="1" colspan="1" align="center">1.21 (0.97–1.50)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs1978243</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">86,596,160</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">0.0332</td><td rowspan="1" colspan="1" align="center">0.87 (0.77–0.99)</td><td rowspan="1" colspan="1" align="center">0.0743</td><td rowspan="1" colspan="1" align="center">0.90 (0.79–1.01)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs6494387</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">61,132,579</td><td rowspan="1" colspan="1" align="center">TPM1</td><td rowspan="1" colspan="1" align="center">0.0155</td><td rowspan="1" colspan="1" align="center">1.15 (1.03–1.29)</td><td rowspan="1" colspan="1" align="center">0.1683</td><td rowspan="1" colspan="1" align="center">1.08 (0.97–1.21)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs1034589</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,909,233</td><td rowspan="1" colspan="1" align="center">RNF185</td><td rowspan="1" colspan="1" align="center">0.0274</td><td rowspan="1" colspan="1" align="center">0.74 (0.57–0.97)</td><td rowspan="1" colspan="1" align="center">0.0438</td><td rowspan="1" colspan="1" align="center">0.79 (0.63–0.99)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2106294</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,975,759</td><td rowspan="1" colspan="1" align="center">LIMK2</td><td rowspan="1" colspan="1" align="center">0.0167</td><td rowspan="1" colspan="1" align="center">0.72 (0.55–0.94)</td><td rowspan="1" colspan="1" align="center">0.0411</td><td rowspan="1" colspan="1" align="center">0.79 (0.63–0.99)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs4820043</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,977,094</td><td rowspan="1" colspan="1" align="center">LIMK2</td><td rowspan="1" colspan="1" align="center">0.0162</td><td rowspan="1" colspan="1" align="center">0.72 (0.55–0.94)</td><td rowspan="1" colspan="1" align="center">0.0460</td><td rowspan="1" colspan="1" align="center">0.79 (0.63–1.00)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs5749286</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">30,230,359</td><td rowspan="1" colspan="1" align="center">SFI1</td><td rowspan="1" colspan="1" align="center">0.0274</td><td rowspan="1" colspan="1" align="center">0.75 (0.58–0.97)</td><td rowspan="1" colspan="1" align="center">0.0344</td><td rowspan="1" colspan="1" align="center">0.78 (0.62–0.98)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs16996381</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">34,841,225</td><td rowspan="1" colspan="1" align="center">APOL3</td><td rowspan="1" colspan="1" align="center">1.76E-04</td><td rowspan="1" colspan="1" align="center">0.82 (0.74–0.91)</td><td rowspan="1" colspan="1" align="center">0.8152</td><td rowspan="1" colspan="1" align="center">1.01 (0.91–1.12)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs735853</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">35,009,161</td><td rowspan="1" colspan="1" align="center">MYH9</td><td rowspan="1" colspan="1" align="center">8.57E-04</td><td rowspan="1" colspan="1" align="center">0.73 (0.61–0.88)</td><td rowspan="1" colspan="1" align="center">0.4515</td><td rowspan="1" colspan="1" align="center">0.94 (0.80–1.11)</td></tr></tbody></table></div><div id="largeobj_idm140173900873712" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3056271/table/T4/?report=objectonly">Open in a separate window</a></div></div><p id="P7" class="p p-last">The strongest association in the combined analysis (GWAS+replication) was with rs6930576 (<em>P</em> = 7.04×10<sup>−7</sup>, Odds Ratio (OR) (95% Confidence Interval (95% CI)) = 1.31 (1.18–1.45); <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2a</a>). SNP rs6930576 is located in intron 1 of the SAM and SH3 domain containing 1 (<em>SASH1</em>) gene. Six other hits were detected in four regions with combined <em>P</em> values less than 1.0×10<sup>−5</sup>. SNP rs7769051 is located 8kb upstream of the ribosomal S12 (<em>RPS12</em>) gene (<em>P</em> = 2.2×10<sup>−6</sup>, OR (95% CI) = 1.28 (1.16–1.42); <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2b</a>). SNP rs773506 (<em>P</em> = 6.45×10<sup>−6</sup>, OR (95% CI) = 0.76 (0.67–0.85); <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2c</a>) is located 600bp 3’ of <em>AUH</em> (AU RNA-binding protein/enoyl-Coenzyme A). Next, SNP rs2358944 (<em>P</em> = 3.54×10<sup>−6</sup>, OR (95% CI) = 0.75 (0.67–0.85); <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2d</a>) is 38kb from the non-coding RNA, ribosomal protein SA pseudogene 52 (<em>RPSAP52</em>). This SNP, rs2358944, is also located near two coding genes: <em>HMGA2</em>, high mobility group AT-hook 2, (101kb upstream of the variant) and <em>MSRB3</em>, methionine sulfoxide reductase B3, (257kb downstream of the marker). Finally, there were three hits located in the <em>LIMK2</em> (LIM domain kinase 2) – <em>SFI1</em> (Sfi1 homolog, spindle assembly associated (yeast)) region: rs2106294 and rs4820043 are both located in intron 2 of <em>LIMK2</em> (<em>P</em> = 4.11×10<sup>−6</sup> and 5.07×10<sup>−6</sup>, OR (95% CI) = 0.57 (0.45–0.72) and 0.57 (0.45–0.73) respectively, <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2e</a>), and rs5749286 is located in intron 1 of <em>SFI1</em> (<em>P</em> = 9.79×10<sup>−6</sup>, OR (95% CI) = 0.60 (0.47–0.75); <a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2e</a>). The <em>MYH9</em> SNP seen in the initial GWAS, rs5750250 did not replicate (<em>P</em> = 0.0594, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 2</a>) and had a combined <em>P</em> value = 1.66×10<sup>−7</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 2</a>).</p></div><div id="S6" class="sec"><h3 id="S6title">Discrimination between T2DM-ESRD, T2DM, and all-cause ESRD</h3><p id="P8" class="p p-first">The 67 SNPs that showed evidence of replication between T2DM-ESRD cases and non-nephropathy, non-diabetic controls were genotyped in an additional 1,246 AA T2DM, non-nephropathy cases and 1,216 non-T2DM ESRD cases to differentiate between T2DM-ESRD, T2DM, and all-cause ESRD loci (<a href="/pmc/articles/PMC3056271/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; Results shown in <a href="#SD3" rid="SD3" class=" supplementary-material">Supplemental Table 3</a>). Thus, T2DM-ESRD SNPs should have allele frequency differences when compared with cases with T2DM alone (lacking nephropathy). When the 25 T2DM-ESRD candidate SNPs were compared between T2DM-ESRD cases and T2DM subjects without nephropathy, there were significant differences between these two groups (<em>P</em> values ranging from 0.049 to 2.63×10<sup>−5</sup>, <a href="/pmc/articles/PMC3056271/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>), suggesting that these are T2DM-ESRD loci rather than T2DM loci. SNPs in <em>RPS12</em> (rs9493454) and <em>AUH</em> (rs7735506) were still strongly associated in this test (<em>P</em> = 8.79×10<sup>−4</sup> and 2.57×10<sup>−4</sup>, OR (95% CI) = 1.20 (1.08–1.34) and 0.78 (0.69–0.89), respectively, <a href="/pmc/articles/PMC3056271/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>).</p><p id="P9">As a second test, the allele frequencies of these 25 SNPs in T2DM, non-nephropathy subjects were compared with allele frequencies in the normal, non-diabetic, non-nephropathy controls (<a href="/pmc/articles/PMC3056271/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>; control data from the GWAS and replication control samples). True T2DM loci would be expected to show evidence of association in this analysis. While the four SNPs in the <em>LIMK2</em>-<em>SFI1</em> region (rs1034589, rs2106294, rs4820043, and rs5749286) showed nominal evidence of association with <em>P</em> values ranging from 0.034 to 0.046 (OR ranging from 0.78 to 0.79, <a href="/pmc/articles/PMC3056271/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>), these <em>P</em> values are orders of magnitude lower than comparing T2DM-ESRD to controls (<a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>; Combined analysis). This further supports these SNPs as T2DM-ESRD loci and not T2DM loci.</p><p id="P10">The 25 SNPs were also investigated for association with all-cause ESRD. In this analysis, cases consisted of 1,674 T2DM-ESRD subjects (from the GWAS and replication phases) and 1,216 non-diabetic ESRD subjects (predominantly ESRD attributed to hypertension and chronic glomerular diseases). These 2,890 cases with ESRD were compared to the combined set of non-diabetic, non-nephropathy controls (n=1,719, from the GWAS and replication phases). Five variants in two genomic regions achieved genome-wide significance in this analysis (<em>P</em> ≤ 5.0×10<sup>−8</sup>, <a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). The <em>LIMK2</em>-<em>SFI1</em> region contained four of these SNPs. First, rs1034589, located in ring finger protein 185 (<em>RNF185</em>), was associated with all-cause ESRD (<em>P</em> = 2.49×10<sup>−8</sup>, OR (95% CI) = 0.56 (0.46–0.69), <a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>, <a href="/pmc/articles/PMC3056271/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>). There were two SNPs that showed evidence of association in the <em>LIMK2</em> gene, rs2106294 and rs4820043 (<em>P</em> = 3.28×10<sup>−8</sup> and 3.04×10<sup>−8</sup>, OR (95% CI) = 0.56 (0.46–0.69) and 0.56 (0.46–0.69) respectively, <a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>, <a href="/pmc/articles/PMC3056271/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>). The last SNP that was significantly associated in this region was rs5749286, located in <em>SFI1</em> (<em>P</em> = 2.96×10<sup>−11</sup>, OR (95% CI) = 0.51 (0.41–0.62), <a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>, <a href="/pmc/articles/PMC3056271/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>). Finally, SNP rs735853, in <em>MYH9</em> was also significantly associated with all-cause ESRD (<em>P</em> = 4.77×10<sup>−10</sup>, OR (95% CI) = 0.63 (0.54–0.73), <a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>, <a href="/pmc/articles/PMC3056271/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2b</span></a>). All markers were also examined for association with non-diabetic ESRD compared to controls (<a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>; control data from the GWAS and replication control samples). Twelve markers showed association with non-diabetic ESRD (<em>P</em> values ranging from 0.0364 to 1.97×10<sup>−10</sup>, OR ranging from 0.38 to 0.86 and 1.15 to 1.31, <a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). However, all of these variants were more strongly associated in the test of all-cause ESRD (<a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>) and only four of these markers (rs1034589, rs5749286, rs16996381, and rs735853) showed stronger association with non-diabetic ESRD (<a href="/pmc/articles/PMC3056271/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>) than with diabetic nephropathy (<a href="/pmc/articles/PMC3056271/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, Combined analysis).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3056271/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140173899322944" class="figure"><a href="/pmc/articles/PMC3056271/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3056271_nihms271243f2a.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms271243f2a.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3056271/bin/nihms271243f2a.jpg"></a></div><div id="largeobj_idm140173899322944" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3056271/figure/F2/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140173899321968" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3056271_nihms271243f2b.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms271243f2b.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3056271/bin/nihms271243f2b.jpg"></a></div><div id="largeobj_idm140173899321968" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3056271/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3056271/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>All-cause ESRD candidate regions</strong><p id="P39">A) LIMK2-SFI1 gene region. B) MYH9 gene region. −log<sub>10</sub> P<sub>ADD</sub> values from the GWAS are plotted versus position (genome build 36). The large red diamond indicates the P<sub>ADD</sub> values from the GWAS of the marker(s) displayed. The large blue diamond and corresponding P value indicates the P<sub>ADD</sub> values from the combined analysis of the marker(s) displayed. LD based on the control samples is color-coded: red (r<sup>2</sup> to top SNP 0.8–1.0), orange (0.5–0.8), yellow (0.2–0.5), and white (&lt;0.2). Gene annotations were obtained from UCSC Genome Browser (RefSeq Genes, b36). Arrows represent direction of transcription.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Diabetic Nephropathy Candidate Loci: All-Cause ESRD</strong><p id="P45">Comparison of All-cause ESRD (1674 T2DM-ESRD cases and 1216 non-T2DM ESRD cases) with controls (combined from GWAS+Replicaiton), and non-T2DM ESRD cases with controls.</p><p id="P46">OR: Odds Ratio. 95% CI: 95% Confidence Interval. N/A: SNP rs11176482 failed to type in non-T2DM ESRD subjects.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140173894775376" class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="bottom" align="center">Locus<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">All-Cause ESRD (n=2890)<br>v<br>Controls (n=1719)<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">non-T2DM ESRD (n=1216)<br>v<br>Controls (n=1719)<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Marker</th><th rowspan="1" colspan="1" valign="middle" align="center">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">Position</th><th rowspan="1" colspan="1" valign="middle" align="center">Nearest<br>Gene(s)</th><th rowspan="1" colspan="1" valign="middle" align="center">Additive<br><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">Additive<br><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th></tr><tr><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="center"><hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">rs10888287</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">246,184,039</td><td rowspan="1" colspan="1" align="center">OR2L13</td><td rowspan="1" colspan="1" align="center">5.53E-05</td><td rowspan="1" colspan="1" align="center">0.84 (0.77–0.91)</td><td rowspan="1" colspan="1" align="center">0.0057</td><td rowspan="1" colspan="1" align="center">0.86 (0.78–0.96)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs4260465</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">164,231,046</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">0.0027</td><td rowspan="1" colspan="1" align="center">0.88 (0.81–0.96)</td><td rowspan="1" colspan="1" align="center">0.4548</td><td rowspan="1" colspan="1" align="center">0.96 (0.87–1.07)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11730446</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">96,382,267</td><td rowspan="1" colspan="1" align="center">UNC5C</td><td rowspan="1" colspan="1" align="center">8.25E-05</td><td rowspan="1" colspan="1" align="center">1.19 (1.09–1.30)</td><td rowspan="1" colspan="1" align="center">0.0085</td><td rowspan="1" colspan="1" align="center">1.15 (1.04–1.28)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs891382</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">147,371,790</td><td rowspan="1" colspan="1" align="center">SLC10A7/LSM6</td><td rowspan="1" colspan="1" align="center">2.56E-04</td><td rowspan="1" colspan="1" align="center">1.37 (1.16–1.63)</td><td rowspan="1" colspan="1" align="center">0.0859</td><td rowspan="1" colspan="1" align="center">1.20 (0.97–1.48)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7697691</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">182,853,951</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">0.0036</td><td rowspan="1" colspan="1" align="center">0.87 (0.79–0.95)</td><td rowspan="1" colspan="1" align="center">0.6059</td><td rowspan="1" colspan="1" align="center">0.97 (0.86–1.09)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs3822908</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">97,706,333</td><td rowspan="1" colspan="1" align="center">C6orf167</td><td rowspan="1" colspan="1" align="center">2.43E-04</td><td rowspan="1" colspan="1" align="center">0.80 (0.71–0.90)</td><td rowspan="1" colspan="1" align="center">0.0364</td><td rowspan="1" colspan="1" align="center">0.85 (0.73–0.99)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7760831</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">102,071,536</td><td rowspan="1" colspan="1" align="center">GRIK2</td><td rowspan="1" colspan="1" align="center">1.62E-06</td><td rowspan="1" colspan="1" align="center">0.77 (0.69–0.86)</td><td rowspan="1" colspan="1" align="center">1.43E-04</td><td rowspan="1" colspan="1" align="center">0.77 (0.68–0.88)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs208865</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">130,114,249</td><td rowspan="1" colspan="1" align="center">C6orf191/ARHGAP18</td><td rowspan="1" colspan="1" align="center">2.60E-05</td><td rowspan="1" colspan="1" align="center">1.37 (1.18–1.59)</td><td rowspan="1" colspan="1" align="center">0.0025</td><td rowspan="1" colspan="1" align="center">1.31 (1.10–1.56)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs9493454</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">133,186,322</td><td rowspan="1" colspan="1" align="center">RPS12</td><td rowspan="1" colspan="1" align="center">2.35E-05</td><td rowspan="1" colspan="1" align="center">1.21 (1.11–1.32)</td><td rowspan="1" colspan="1" align="center">0.2134</td><td rowspan="1" colspan="1" align="center">1.07 (0.96–1.19)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs7769051</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">133,188,489</td><td rowspan="1" colspan="1" align="center">RPS12</td><td rowspan="1" colspan="1" align="center">5.85E-06</td><td rowspan="1" colspan="1" align="center">1.24 (1.13–1.36)</td><td rowspan="1" colspan="1" align="center">0.1958</td><td rowspan="1" colspan="1" align="center">1.07 (0.96–1.19)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs6930576</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">148,746,647</td><td rowspan="1" colspan="1" align="center">SASH1</td><td rowspan="1" colspan="1" align="center">4.76E-07</td><td rowspan="1" colspan="1" align="center">1.27 (1.16–1.40)</td><td rowspan="1" colspan="1" align="center">4.80E-04</td><td rowspan="1" colspan="1" align="center">1.23 (1.09–1.38)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs773506</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">93,015,292</td><td rowspan="1" colspan="1" align="center">AUH</td><td rowspan="1" colspan="1" align="center">1.40E-04</td><td rowspan="1" colspan="1" align="center">0.82 (0.73–0.91)</td><td rowspan="1" colspan="1" align="center">0.0901</td><td rowspan="1" colspan="1" align="center">0.89 (0.79–1.02)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11175885</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">64,400,897</td><td rowspan="1" colspan="1" align="center">C12orf66/TMEM5</td><td rowspan="1" colspan="1" align="center">3.66E-04</td><td rowspan="1" colspan="1" align="center">0.83 (0.75–0.92)</td><td rowspan="1" colspan="1" align="center">0.1247</td><td rowspan="1" colspan="1" align="center">0.90 (0.80–1.03)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2358944</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">64,403,825</td><td rowspan="1" colspan="1" align="center">MSRB3/HMGA2</td><td rowspan="1" colspan="1" align="center">1.15E-04</td><td rowspan="1" colspan="1" align="center">0.82 (0.74–0.90)</td><td rowspan="1" colspan="1" align="center">0.1100</td><td rowspan="1" colspan="1" align="center">0.90 (0.79–1.02)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2904532</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">65,582,311</td><td rowspan="1" colspan="1" align="center">GRIP1/CAND1</td><td rowspan="1" colspan="1" align="center">4.13E-04</td><td rowspan="1" colspan="1" align="center">0.82 (0.73–0.91)</td><td rowspan="1" colspan="1" align="center">0.0902</td><td rowspan="1" colspan="1" align="center">0.89 (0.77–1.02)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11176482</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">65,582,814</td><td rowspan="1" colspan="1" align="center">GRIP1/CAND1</td><td rowspan="1" colspan="1" align="center">N/A</td><td rowspan="1" colspan="1" align="center">N/A</td><td rowspan="1" colspan="1" align="center">N/A</td><td rowspan="1" colspan="1" align="center">N/A</td></tr><tr><td rowspan="1" colspan="1" align="center">rs12302041</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">76,654,356</td><td rowspan="1" colspan="1" align="center">NAV3</td><td rowspan="1" colspan="1" align="center">1.87E-04</td><td rowspan="1" colspan="1" align="center">1.40 (1.17–1.67)</td><td rowspan="1" colspan="1" align="center">0.0895</td><td rowspan="1" colspan="1" align="center">1.21 (0.97–1.51)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs1978243</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">86,596,160</td><td rowspan="1" colspan="1" align="center">No gene</td><td rowspan="1" colspan="1" align="center">6.25E-04</td><td rowspan="1" colspan="1" align="center">0.84 (0.76–0.93)</td><td rowspan="1" colspan="1" align="center">0.2588</td><td rowspan="1" colspan="1" align="center">0.93 (0.83–1.05)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs6494387</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">61,132,579</td><td rowspan="1" colspan="1" align="center">TPM1</td><td rowspan="1" colspan="1" align="center">0.0017</td><td rowspan="1" colspan="1" align="center">1.16 (1.06–1.27)</td><td rowspan="1" colspan="1" align="center">0.4478</td><td rowspan="1" colspan="1" align="center">1.04 (0.93–1.17)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs1034589</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,909,233</td><td rowspan="1" colspan="1" align="center">RNF185</td><td rowspan="1" colspan="1" align="center">2.49E-08</td><td rowspan="1" colspan="1" align="center">0.56 (0.46–0.69)</td><td rowspan="1" colspan="1" align="center">2.17E-06</td><td rowspan="1" colspan="1" align="center">0.52 (0.40–0.68)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs2106294</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,975,759</td><td rowspan="1" colspan="1" align="center">LIMK2</td><td rowspan="1" colspan="1" align="center">3.28E-08</td><td rowspan="1" colspan="1" align="center">0.56 (0.46–0.69)</td><td rowspan="1" colspan="1" align="center">7.15E-06</td><td rowspan="1" colspan="1" align="center">0.54 (0.42–0.71)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs4820043</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">29,977,094</td><td rowspan="1" colspan="1" align="center">LIMK2</td><td rowspan="1" colspan="1" align="center">3.04E-08</td><td rowspan="1" colspan="1" align="center">0.56 (0.46–0.69)</td><td rowspan="1" colspan="1" align="center">5.77E-06</td><td rowspan="1" colspan="1" align="center">0.54 (0.41–0.70)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs5749286</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">30,230,359</td><td rowspan="1" colspan="1" align="center">SFI1</td><td rowspan="1" colspan="1" align="center">6.51E-11</td><td rowspan="1" colspan="1" align="center">0.51 (0.41–0.62)</td><td rowspan="1" colspan="1" align="center">1.97E-10</td><td rowspan="1" colspan="1" align="center">0.38 (0.28–0.51)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs16996381</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">34,841,225</td><td rowspan="1" colspan="1" align="center">APOL3</td><td rowspan="1" colspan="1" align="center">2.96E-07</td><td rowspan="1" colspan="1" align="center">0.80 (0.73–0.87)</td><td rowspan="1" colspan="1" align="center">7.20E-07</td><td rowspan="1" colspan="1" align="center">0.76 (0.69–0.85)</td></tr><tr><td rowspan="1" colspan="1" align="center">rs735853</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">35,009,161</td><td rowspan="1" colspan="1" align="center">MYH9</td><td rowspan="1" colspan="1" align="center">4.77E-10</td><td rowspan="1" colspan="1" align="center">0.63 (0.54–0.73)</td><td rowspan="1" colspan="1" align="center">7.76E-10</td><td rowspan="1" colspan="1" align="center">0.55 (0.45–0.66)</td></tr></tbody></table></div><div id="largeobj_idm140173894775376" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3056271/table/T5/?report=objectonly">Open in a separate window</a></div></div><p id="P11" class="p p-last">The four hits in the <em>LIMK2</em>-<em>SFI1</em> region are all highly correlated (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 4</a>). SNPs rs1034589 in <em>RNF185</em>, rs2106294 in <em>LIMK2</em>, and rs4820043 also in <em>LIMK2</em>, are all perfectly correlated with one another (r<sup>2</sup>=1.0, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 4</a>). In addition, these SNPs are all highly correlated with rs5749286 in <em>SFI1</em> (r<sup>2</sup>=0.85, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 4</a>). However, the <em>LIMK2</em>-<em>SFI1</em> region and the <em>MYH9</em> region are poorly correlated with r<sup>2</sup> values ranging from 0.0324 to 0.0335 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 4</a>).</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">Tests for Association of Previously Identified Nephropathy Genes</h3><p id="P12" class="p p-first-last">In prior studies (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918253">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918264">8</a>, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304663225">10</a>–<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_336923419">12</a>, <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918288">18</a>–<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918236">23</a>), multiple loci have been implicated in different ethnicities with either renal failure or renal function in both diabetics and non-diabetics. We have assessed association with many of these genes (<em>PVT1, FRMD3, CARS, ACACB, NEDD4L, SERPINB7, CNDP1, CNDP2, ELMO1, SHROOM3, UMOD, GATM-SPATA5L1, GCK2, ALMS1, DAB2, SLC34A1, VEGFA, SLC22A2, PRKAG2, STC1, ATXN2, DACH1, SLC7A9)</em> and T2DM-ESRD in our GWAS. For each locus the entire genic region, along with 10kb upstream and downstream of the gene, were inspected for association in the GWAS data to see if other variants in the same gene were associated with T2DM-ESRD in African Americans. All of these results are presented in <a href="#SD4" rid="SD4" class=" supplementary-material">Supplemental Table 5</a>. While there are some results that are nominally significant, none of the results meet a Bonferroni correction (<em>P</em> = 0.05/1909 total SNPs = 2.6 × 10<sup>−5</sup>) with the lowest <em>P</em> value for this analysis being 5.8 × 10<sup>−4</sup> in the <em>NEDD4L</em> gene.</p></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Discussion</h2><p id="P13" class="p p-first">We performed a high density genome-wide association study to investigate genetic susceptibility to T2DM-ESRD and all-cause ESRD in African Americans. Previous genome-wide association studies for diabetes associated nephropathy in Japanese (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918283">7</a>), Pima Indians (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918279">8</a>), and Caucasian Americans (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918231">9</a>) have identified several potential DN loci: <em>ELMO1</em> (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918244">7</a>), <em>PVT1</em> (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918274">8</a>), <em>FRMD3</em>, and <em>CARS</em> (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918276">9</a>). We have previously seen association with the <em>ELMO1</em> and <em>MYH9</em> genes and T2DM-ESRD in our African American population (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918241">17</a>, <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918266">20</a>). These studies in other ethnicities have used a study design of comparing cases with T2DM-ESRD and controls with T2DM and no nephropathy. While this study design might be appropriate for other ethnicities, it has serious limitations for African Americans. African Americans with T2DM and preserved renal function for 10 years or more (“hypernormal” controls) are perhaps the ideal control group, but are rare (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918270">24</a>). We estimate that less than 10% of African American T2DM-affecteds of 10 years duration have preserved renal function. We have chosen a study design that is flexible and support studies of T2DM and BMI in parallel with T2DM-ESRD. Importantly we have used both replication and several contrast groups to differentiate between T2DM-ESRD and T2DM. This system works quite well: the strongest T2DM associated gene in African Americans is <em>TCF7L2</em>, which is easily filtered out of our hits for T2DM-ESRD, but prominently associated with T2DM (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918234">25</a>, <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918294">26</a>).</p><p id="P14">We identified five novel gene regions with evidence of association with T2DM-ESRD in African Americans (<em>P</em> &lt; 1.0×10<sup>−5</sup>): <em>SASH1</em>, <em>RPS12</em>, <em>AUH</em>, <em>MSRB3</em>-<em>HMGA2</em>, and <em>LIMK2</em>-<em>SFI1</em>. One of these regions, <em>LIMK2</em>-<em>SFI1</em>, was also significantly associated (<em>P</em> &lt; 5.0×10<sup>−8</sup>) with all-cause ESRD. <em>MYH9</em>, which has been previously associated with non-diabetic nephropathy (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918256">14</a>–<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918263">16</a>), and diabetic nephropathy (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918292">17</a>), also showed significant association (<em>P</em> &lt; 5.0×10<sup>−8</sup>) with all-cause ESRD.</p><p id="P15"><em>SASH1</em>, located at 6q24.3, encodes the sterile α motif (SAM)- and SH3-domain containing 1 protein and is ubiquitously expressed. <em>SASH1</em> was originally identified as a candidate tumor suppressor gene, being downregulated in the majority (74%) of breast cancer tumors (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918285">27</a>), and has been shown to be downregulated in colon cancer (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918293">28</a>). SASH1 also functions as a signaling adapter, as a downstream target in the insulin-like growth factor (IGF1)/phosphatidylinositol 3-kinase (PI 3-kinase) signaling pathway (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918268">29</a>). SNP rs6930576, located within this gene, showed the strongest association with T2DM-ESRD in the combined analysis, suggesting this gene may also have a role in the progression of diabetic nephropathy.</p><p id="P16">Two other candidate regions are also worth mentioning. The ribosomal protein S12 gene <em>(RPS12</em>) is located on chromosome 6q23.2. The protein encoded by this gene is located in the cytoplasm and is a component of the 40S ribosomal subunit (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918238">30</a>). <em>RPS12</em> is also a potential target gene of MicroRNA-377 (miRNA-377), which has been consistently up-regulated in <em>in vitro</em> diabetic nephropathy models, as well as in <em>in vivo</em> diabetic nephropathy mouse models (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918255">31</a>). If <em>RPS12</em> is also up-regulated in the diabetic milieu, it may contribute to the progression of diabetic nephropathy.</p><p id="P17"><em>LIMK2</em>, LIM-kinase 2, is located on chromosome 22 at 22q12.2 (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918240">32</a>). There are three isoforms of <em>LIMK2</em> produced by alternative splicing: variants 1, 2a, and 2b (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918291">33</a>). <em>LIMK2</em> is one of two members of the LIM kinase family, and contains two LIM domains at the N terminus and a protein kinase domain at the C terminus (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918245">32</a>). LIMK2 is activated by Rho and Cdc42 via ROCK and PAK respectively; in turn, LIMK2 phosphorylates cofilin, eliminating cofilin’s ability to bind and depolymerize actin. This leads to an accumulation of actin filaments and aggregates. It is through this action that LIMK2 plays a role in the Rho- and Cdc42-induced reorganization of the actin cytoskeleton (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918287">34</a>–<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918248">36</a>). LIMK2 is widely expressed in all tissues (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918277">37</a>), showing moderate expression in the kidney (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918290">32</a>, <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918295">37</a>). Homozygous Limk2 gene-deficient mice (Limk2<sup>−</sup>/<sup>−</sup>), created using a Cre-mediated excision to disrupt all three LIMK2 isoforms, exhibited normal growth to adulthood when compared with wild-type and heterozygous mice (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918271">38</a>). However, Limk2<sup>−</sup>/<sup>−</sup> mice had moderate kidney abnormalities with dilated collecting tubules and slight decreases in the number of glomeruli, without changes in nephron function (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918247">38</a>). If variants in <em>LIMK2</em> lead to kidney abnormalities in humans, this may be detrimental in a diabetic environment, and could possibly lead to the progression of diabetic nephropathy. This would also support <em>LIMK2</em> as a potential all-cause ESRD candidate locus.</p><p id="P18">The other gene in the <em>LIMK2</em> region that showed evidence of association with T2DM-ESRD and all-cause ESRD was <em>SFI1</em>, Sfi1 homolog, spindle assembly associated (yeast). In yeast, SFI1 has been shown to be an essential and conserved component of centrosomes – known as spindle pole bodies in yeast (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918280">39</a>). Also, SFI1 was shown to be required for spindle pole body duplication and may be required for separation of the duplicated spindle pole bodies (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918281">40</a>).</p><p id="P19"><em>MYH9</em>, non-muscle myosin heavy chain 9, which showed evidence of association with T2DM-ESRD and all-cause ESRD in our study, has been previously associated with non-T2DM nephropathy (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918246">14</a>–<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918232">16</a>), and diabetic forms of nephropathy (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918269">17</a>). It should be noted that the results reported here are in reference to the minor (less frequent) allele, and are associated with protection. This remains consistent with previous associations where the reported risk allele corresponded to the major allele (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918278">14</a>–<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918273">17</a>). In the Yoruba (YRI) samples in HapMap the protective allele for rs735853 is observed in only 3% of chromosomes, while it is seen in over 57% of CEU chromosomes. For our African American sample, the allele frequencies are 0.082 for cases and 0.123 for controls. This result may reflect a selection for the admixed European allele, or more broadly, the European locus, that provides protection in African Americans. Whether this reflects that some cases in our study could be non-diabetic ESRD ((<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918260">17</a>): recently reestimated to be 13%), or reflects a role for this allele in T2DM-ESRD remains to be determined. Parenthetically, the <em>LIMK2-SFI1</em> gene region is located approximately 5 Mb away from the <em>MYH9</em> gene region. We evaluated linkage disequilibrium across chromosome 22 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 4</a>). The SNPs that showed evidence of association in the <em>LIMK2-SFI1</em> gene region were poorly correlated (r<sup>2</sup> ≤0.03) with the SNP that showed evidence of association in <em>MYH9</em>; suggesting that these are independent signals.</p><p id="P20">However, <em>MYH9</em> and <em>LIMK2</em> are both involved in the reorganization and maintenance of the actin cytoskeleton. <em>MYH9</em> encodes the nonmuscle myosin heavy chain IIA, which is highly conserved and similar to other nonmuscle myosins (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918252">41</a>, <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918251">42</a>). Myosin IIA is expressed in the podocytes and mesangial cells (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918261">41</a>). Within the podocyte, myosin IIA localizes to the foot process where it is involved in the movement of actin filaments in order to maintain cell structure (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918239">43</a>). Many studies have shown that mutations in <em>MYH9</em> are associated with kidney disease (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918267">14</a>–<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918289">17</a>). As LIMK2 is in the same pathway, it remains possible that mutations in <em>LIMK2</em> could produce kidney disease; thus, <em>LIMK2</em> is a strong candidate in T2DM-ESRD and all-cause ESRD.</p><p id="P21">We did two further assessments of loci in this study. First, we assessed several of the African American loci (13/25) to test whether they contributed to T2DM-ESRD in a sample of Caucasian American cases and controls. We observed no evidence for association in Caucasian Americans (data not shown), but a more comprehensive, better powered experiment might be more definitive. Conversely, we did not observe evidence of association with any of the diabetic nephropathy and chronic kidney disease candidate genes and T2DM-ESRD in African Americans in the current study. The GWAS here did not interrogate some of the specific disease associated SNPs such as <em>ACACB</em> SNP rs2268388 with which we previously reported evidence of association (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304663226">21</a>). The large number of SNPs evaluated here (over 1900) results in a more stringent threshold for significance. Since these variants were mainly identified in non-African derived racial/ethnic groups, they may play a limited role in the development of T2DM-ESRD in African Americans.</p><p id="P22">This study has similar limitations as other GWAS. While we have identified 25 variants that may influence T2DM-ESRD in 19 candidate regions, none of these regions have reached genome-wide significance (<em>P</em> &lt; 5.0×10<sup>−8</sup>) with the T2DM-ESRD phenotype. However, even though this study has a limited sample size for the GWAS discovery phase, power calculations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 6</a>) show that we have moderate statistical power to detect odds ratios of 1.3 (MAF=0.35), consistent with previously published effect sizes for DN (<em>FRMD3</em> and <em>CARS</em>, OR=1.45 and 1.36, respectively (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918257">9</a>)). In addition, the other phases of the study: replication, trait discrimination, and all-cause ESRD analysis have excellent power for detecting association given the goals of each step (e.g. nominal association in replication; <em>P</em> value ≤ 1×10<sup>−6</sup> for the all-cause ESRD analysis). There are almost certainly additional loci of moderate effect that have not been captured in this study and loci of weaker effect will require a significantly larger study. There are few other existing collections of appropriate samples in African Americans and we are one of the few centers with active recruiting efforts in this population. This will make the search for additional replication populations more difficult. In addition, it is difficult to ensure that all subjects clinically labeled as having T2DM-ESRD lack other renal lesions. This difficulty plagues all studies of diabetic nephropathy and we chose inclusion criteria similar to other large studies (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918258">44</a>).</p><p id="P23" class="p p-last">In conclusion, we performed a GWAS for T2DM-ESRD in African Americans. We then carried out a replication phase, as well as a trait discrimination phase in order to determine whether associations were with T2DM, T2DM-ESRD, and/or all-cause ESRD. Through these studies, we have discovered 25 potential variants in 19 genes associated with T2DM-ESRD including: <em>SASH1</em>, <em>RPS12</em>, <em>AUH</em>, <em>MSRB3</em>-<em>HMGA2</em>, and <em>LIMK2</em>-<em>SFI1</em>. These results require further replication in order to confirm their role in T2DM-ESRD susceptibility and clarify their role in all-cause ESRD in African Americans.</p></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Methods</h2><div id="S10" class="sec sec-first"><h3 id="S10title">Subjects</h3><div id="S11" class="sec sec-first"><p></p><h4 id="S11title" class="inline">Genome-wide association study (GWAS) samples and clinical characteristics </h4><p id="P24" class="p p-first-last">Recruitment and sample collection procedures were approved by the Institutional Review Board at Wake Forest University and informed consent was obtained from all study participants. Patients with T2DM were recruited from dialysis facilities. Individuals with a history of ketoacidosis, or who developed diabetes prior to the age of 25 years and received continuous insulin therapy since diagnosis were assumed to have type 1 diabetes and were excluded. T2DM was diagnosed in African-Americans who reported developing diabetes after the age of 25 years and who did not receive only insulin therapy since diagnosis. Cases had T2DM diagnosed at least 5 years before initiating renal replacement therapy, background or greater diabetic retinopathy, and/or ≥100 mg/dl proteinuria on urinalysis in the absence of other causes of nephropathy (T2DM-ESRD subjects). Unrelated African American controls without a current diagnosis of diabetes or renal disease were recruited from the community and internal medicine clinics (Control subjects). All cases and non-diabetic, non-nephropathy controls were born in North Carolina, South Carolina, Georgia, Tennessee, or Virginia. DNA extraction was performed using the PureGene system (Gentra Systems, Minneapolis, MN, USA).</p></div><div id="S12" class="sec"><p></p><h4 id="S12title" class="inline">Replication study samples and clinical characteristics </h4><p id="P25" class="p p-first-last">Informed consent was obtained from all study participants and recruitment and sample collection procedures were approved by the Institutional Review Board at Wake Forest University. African American T2DM-ESRD cases and non-diabetic, non-nephropathy controls were recruited using the same criteria as the case and control subjects that were used in the GWAS. DNA extraction was performed using the PureGene system.</p></div><div id="S13" class="sec sec-last"><p></p><h4 id="S13title" class="inline">T2DM, non-nephropathy and non-T2DM ESRD study samples and clinical characteristics </h4><p id="P26" class="p p-first-last">Sample collection procedures and recruitment was approved by the Wake Forest University Institutional Review Board. Informed consent was obtained from all study participants. Subjects with T2DM without evidence of nephropathy were recruited from medical clinics, churches, health fairs, and community resources (T2DM subjects). Patients who had ESRD attributed to hypertension or primary glomerular disease were recruited (non-T2DM ESRD subjects). Putative hypertension-associated ESRD was diagnosed in patients with high blood pressure preceding initiation of renal replacement therapy and low level proteinuria (≤30 mg/dL on urine dipstick, &lt;0.5 gram protein/24 hours on timed urine collection, or urine protein: creatinine ratio &lt;0.5 g/g) or in the absence of measures of proteinuria. Chronic glomerular disease-associated ESRD was diagnosed in non-diabetic subjects with renal biopsy evidence of primary glomerular disease (e.g. focal segmental glomerulosclerosis), proteinuria ≥ 0.5 grams/24 hours or ≥100 mg/dL on urinalysis. Individuals with diabetic nephropathy (as described above), cystic renal diseases, hereditary nephritis, or urologic causes of ESRD were excluded. Individuals were unrelated and self-described African American. All subjects were born in North Carolina, South Carolina, Georgia, Virginia, or Tennessee. The PureGene system was used for DNA extraction.</p></div></div><div id="S14" class="sec"><h3 id="S14title">Sample Preparation, Genotyping and Quality Control</h3><div id="S15" class="sec sec-first"><p></p><h4 id="S15title" class="inline">GWAS </h4><p id="P27" class="p p-first-last">Genotyping was performed at the Center for Inherited Disease Research (CIDR) using 1µg of genomic DNA (diluted in 1X TE buffer and at 50ng/µl) on the Affymetrix Genome-wide Human SNP array 6.0 (Affymetrix, Santa Clara, CA, USA). DNA from cases and controls were equally interleaved on 96-well master plates to ensure technical uniformity during sample processing. To confirm sample identity, a SNP barcode (96 SNPs) was generated prior to genotyping on the Affymetrix array and was confirmed on downstream released genotyping data. Genotypes were called using Birdseed version 2; Affymetrix Power Tools (APT) 1.10.0 (Affymetrix, Santa Clara, CA) by grouping samples by DNA plate to determine the genotype cluster boundaries. All autosomal SNPs (n=868,157) were included in analysis but were classified on data quality. The primary inference was drawn from SNPs which had less than 5% missing data, Hardy-Weinberg <em>P</em> values in cases greater than 0.0001 and in controls greater than 0.01, no significant difference in missing data rate between cases and controls and were polymorphic (n=832,357). The average sample call rate was 99.16% for all autosomal SNPs. Forty-six blind duplicates were included in genotyping and had a concordance rate of 99.59%. In addition, individuals whose gender call from X chromosome genotype data was discordant with the gender obtained from patient interviews were excluded from the analysis (<em>n</em>=1). Cryptic relatedness was estimated by pairwise identity-by-descent (IBD) analysis implemented in the PLINK analysis software package (<a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a>) (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918282">45</a>). Two duplicate samples were identified, and one sample in each duplicate pair was removed. In addition, 104 individuals from pairs of samples were identified as cryptic first degree relatives, these individuals were kept for the analysis. We also assessed heterozygosity by estimating the inbreeding coefficient, F, using PLINK. One subject had an F value &gt; 4 standard deviations from the mean, suggesting population substructure. This subject was removed. Our final dataset of consisted of 1994 individuals (965 T2DM-ESRD cases and 1,029 controls) in which we performed association analysis.</p></div><div id="S16" class="sec"><p></p><h4 id="S16title" class="inline">Replication </h4><p id="P28" class="p p-first-last">The replication population was recruited under identical ascertainment criteria to that of the GWAS. A total of 724 SNPs were genotyped using the iPLEX™ Sequenom MassARRAY platform (Sequenom, San Diego, CA, USA). Genotyping efficiency was greater than 90% and 45 blind duplicates were included to ensure genotyping accuracy.</p></div><div id="S17" class="sec sec-last"><p></p><h4 id="S17title" class="inline">Trait Discrimination </h4><p id="P29" class="p p-first-last">Sixty-seven SNPs were genotyped using the iPLEX™ Sequenom MassARRAY platform. Genotyping efficiency was greater than 90%. There were 50 blind duplicates included to ensure genotyping accuracy.</p></div></div><div id="S18" class="sec sec-last"><h3 id="S18title">Analysis</h3><div id="S19" class="sec sec-first"><p></p><h4 id="S19title" class="inline">GWAS </h4><p id="P30" class="p p-first-last">To address the effect of admixture in this African American dataset we performed a Principal Components Analysis (PCA) which utilized all high quality data from the GWAS and excluded regions of high LD and inversions. This approach is an iterative process whereby high quality autosomal SNPs (n=832,357) are used to calculate the top 50 principal components. Once calculated, the principal components are examined to determine if they are tied to a region of the genome. If so, those SNPs are excluded and the analysis is repeated. The first principal component (PC1) explained the largest proportion of variation at 22%. A direct comparison of the PCA with FRAPPE (Frequenist Estimation of individual ancestry proportion) analysis of 70 ancestry informative markers (AIMs) resulted in a high correlation between PC1 and the AIMs, r<sup>2</sup>=0.87. Other principal components were associated with regions of the genome, representing another unclassified source of variance, and did not further reduce the inflation factor. The mean (SD) African ancestry proportions in 965 T2DM-ESRD cases and 1,029 controls were 0.80±0.11 and 0.78±0.11, respectively, as estimated by FRAPPE analysis (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213918275">46</a>). To test for association with T2DM-ESRD, genotypic tests of association were performed on each SNP individually using SNPGWA version 4.0 (<a href="http://www.phs.wfubmc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.phs.wfubmc.edu</a>) (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_336923418">47</a>), an analytic package which includes the capability to perform association calculations adjusting for covariates. Genotypic association reported here is for analyses incorporating adjustment for PC1. The primary inference is based on the additive genetic model. The inflation factor was calculated from the observed mean chi-square statistic across high quality autosomal SNPs (n=832,357). Under the null hypothesis, if there is no inflation, the expected mean value is 1. In our population, the inflation factor calculated in the GWAS sample with 104 cryptic related individuals (n=1,994), adjusting for PC1 was 1.06. After 54 subjects were removed to eliminate the cryptic relations (n=1,940), the inflation factor in the GWAS, adjusting for PC1 was 1.04. With this minimal improvement in inflation, all subjects (n=1,994), were retained for analysis since the difference between the inflation factors is only 0.02.</p></div><div id="S20" class="sec"><p></p><h4 id="S20title" class="inline">Replication in T2DM-ESRD cases and non-diabetic, non-nephropathy controls </h4><p id="P31" class="p p-first-last">To account for admixture in the replication cohort, ancestral allele frequencies were estimated from the results of the 70 AIMs genotyped in 44 Yoruba Nigerians and 39 European Americans. Individual ancestral proportions were generated for each subject using FRAPPE, an EM algorithm, under a two-population model. The mean (SD) African ancestry proportions in 709 T2DM-ESRD cases and 690 controls were 0.80±0.12 and 0.76±0.13, respectively. Association analysis was performed as described for the GWAS.</p></div><div id="S21" class="sec"><p></p><h4 id="S21title" class="inline">Combined analysis of T2DM-ESRD cases and non-diabetic, non-nephropathy controls </h4><p id="P32" class="p p-first-last">The GWAS and replication cohorts were merged as one cohort for a combined analysis. Admixture in the combined cohort was accounted for by using ancestral allele frequencies as described for the replication analysis. Association analysis was performed as described for the GWAS and replication study.</p></div><div id="S22" class="sec sec-last"><p></p><h4 id="S22title" class="inline">Discrimination between T2DM-ESRD, T2DM, and all-cause ESRD </h4><p id="P33" class="p p-first-last">In order to determine association with T2DM-ESRD, T2DM, and/or all-cause ESRD additional association analyses were performed as described for the GWAS, replication, and combined analysis incorporating the 1,246 T2DM, non-nephropathy subjects and 1,216 non-T2DM ESRD subjects. Admixture was accounted for through ancestral allele frequencies as described in the replication and combined analysis. The mean (SD) African ancestry proportions in 1,246 T2DM, non-nephropathy cases and 1,216 non-T2DM ESRD cases were 0.76±0.13 and 0.79±0.12, respectively.</p></div></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp</h4><div class="sup-box half_rhythm" id="N0x15c52200N0x16abb0b0"><a href="/pmc/articles/PMC3056271/bin/NIHMS271243-supplement-Supp.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(406K, doc)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supp Table 1</h4><div class="sup-box half_rhythm" id="N0x15c52200N0x16abb170"><a href="/pmc/articles/PMC3056271/bin/NIHMS271243-supplement-Supp_Table_1.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(342K, xls)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Supp Table 3</h4><div class="sup-box half_rhythm" id="N0x15c52200N0x16abb230"><a href="/pmc/articles/PMC3056271/bin/NIHMS271243-supplement-Supp_Table_3.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(45K, xls)</sup></div></div><div class="sec suppmat" id="SD4"><h4>Supp Table 5</h4><div class="sup-box half_rhythm" id="N0x15c52200N0x16abb2f0"><a href="/pmc/articles/PMC3056271/bin/NIHMS271243-supplement-Supp_Table_5.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(360K, xls)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Acknowledgements</h2><div class="sec"><p id="P37">We wish to thank the patients, their relatives, and staff of the Southeastern Kidney Council, Inc./ESRD Network 6 for their participation. This work was supported by NIH grants R01 DK066358 (DWB), R01 DK053591 (DWB), R01 HL56266 (BIF), R01 DK070941 (BIF) and in part by the General Clinical Research Center of the Wake Forest University School of Medicine grant M01 RR07122.</p></div></div><div id="idm140173937346784" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140173937346784title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P34" class="p p-first"><strong>Disclosure</strong></p><p id="P35" class="p p-last">The authors report no interests to disclose.</p><p></p><p class="fn sec" id="FN2"></p><p id="P36" class="p p-first-last">This work was presented in abstract form at the 2009 Annual Meeting of the American Society of Human Genetics, Honolulu, HI, October 20 through 24; at the 2009 Annual Meeting of the American Society of Nephrology, San Diego, CA, October 29 through November 1; and at the 2010 Annual Meeting of the American Diabetes Association, Orlando, FL, June 25 through 29.</p><p></p></div></div><div id="idm140173901489520" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140173901489520title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">USRDS: U.S. Renal Data System 2009. <span class="ref-journal">Annual Data Report: Atlas of Chronic Kidney Disease and End-Stage Renal Disease in the United States.</span> Bethesda, MD: National Institutes of Health: National Institute of Diabetes and Digestive and Kidney Diseases; 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Annual+Data+Report:+Atlas+of+Chronic+Kidney+Disease+and+End-Stage+Renal+Disease+in+the+United+States&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Bowden DW. Genetics of kidney disease. <span><span class="ref-journal">Kidney Int Suppl. </span>2003;<span class="ref-vol">83</span>:S8–S12.</span> [<a href="/pubmed/12864867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int+Suppl&amp;title=Genetics+of+kidney+disease&amp;author=DW+Bowden&amp;volume=83&amp;publication_year=2003&amp;pages=S8-S12&amp;pmid=12864867&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Quinn M, Angelico MC, Warram JH, Krolewski AS. Familial factors determine the development of diabetic nephropathy in patients with IDDM. <span><span class="ref-journal">Diabetologia. </span>1996;<span class="ref-vol">39</span>:940–945.</span> [<a href="/pubmed/8858216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Familial+factors+determine+the+development+of+diabetic+nephropathy+in+patients+with+IDDM&amp;author=M+Quinn&amp;author=MC+Angelico&amp;author=JH+Warram&amp;author=AS+Krolewski&amp;volume=39&amp;publication_year=1996&amp;pages=940-945&amp;pmid=8858216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Seaquist ER, Goetz FC, Rich S, Barbosa J. Familial clustering of diabetic kidney disease. Evidence for genetic susceptibility to diabetic nephropathy. <span><span class="ref-journal">N Engl J Med. </span>1989;<span class="ref-vol">320</span>:1161–1165.</span> [<a href="/pubmed/2710189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Familial+clustering+of+diabetic+kidney+disease.+Evidence+for+genetic+susceptibility+to+diabetic+nephropathy&amp;author=ER+Seaquist&amp;author=FC+Goetz&amp;author=S+Rich&amp;author=J+Barbosa&amp;volume=320&amp;publication_year=1989&amp;pages=1161-1165&amp;pmid=2710189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Spray BJ, Atassi NG, Tuttle AB, Freedman BI. Familial risk, age at onset, and cause of end-stage renal disease in white Americans. <span><span class="ref-journal">J Am Soc Nephrol. </span>1995;<span class="ref-vol">5</span>:1806–1810.</span> [<a href="/pubmed/7787148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Familial+risk,+age+at+onset,+and+cause+of+end-stage+renal+disease+in+white+Americans&amp;author=BJ+Spray&amp;author=NG+Atassi&amp;author=AB+Tuttle&amp;author=BI+Freedman&amp;volume=5&amp;publication_year=1995&amp;pages=1806-1810&amp;pmid=7787148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Freedman BI, Tuttle AB, Spray BJ. Familial predisposition to nephropathy in African-Americans with non-insulin-dependent diabetes mellitus. <span><span class="ref-journal">Am J Kidney Dis. </span>1995;<span class="ref-vol">25</span>:710–713.</span> [<a href="/pubmed/7747724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Kidney+Dis&amp;title=Familial+predisposition+to+nephropathy+in+African-Americans+with+non-insulin-dependent+diabetes+mellitus&amp;author=BI+Freedman&amp;author=AB+Tuttle&amp;author=BJ+Spray&amp;volume=25&amp;publication_year=1995&amp;pages=710-713&amp;pmid=7747724&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Shimazaki A, Kawamura Y, Kanazawa A, Sekine A, et al.  Genetic variations in the gene encoding ELMO1 are associated with susceptibility to diabetic nephropathy. <span><span class="ref-journal">Diabetes. </span>2005;<span class="ref-vol">54</span>:1171–1178.</span> [<a href="/pubmed/15793258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Genetic+variations+in+the+gene+encoding+ELMO1+are+associated+with+susceptibility+to+diabetic+nephropathy&amp;author=A+Shimazaki&amp;author=Y+Kawamura&amp;author=A+Kanazawa&amp;author=A+Sekine&amp;volume=54&amp;publication_year=2005&amp;pages=1171-1178&amp;pmid=15793258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Hanson RL, Craig DW, Millis MP, Yeatts KA, et al.  Identification of PVT1 as a candidate gene for end-stage renal disease in type 2 diabetes using a pooling-based genome-wide single nucleotide polymorphism association study. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:975–983.</span> [<a href="/pubmed/17395743" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+PVT1+as+a+candidate+gene+for+end-stage+renal+disease+in+type+2+diabetes+using+a+pooling-based+genome-wide+single+nucleotide+polymorphism+association+study&amp;author=RL+Hanson&amp;author=DW+Craig&amp;author=MP+Millis&amp;author=KA+Yeatts&amp;volume=56&amp;publication_year=2007&amp;pages=975-983&amp;pmid=17395743&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Pezzolesi MG, Poznik GD, Mychaleckyj JC, Paterson AD, et al.  Genome-wide association scan for diabetic nephropathy susceptibility genes in type 1 diabetes. <span><span class="ref-journal">Diabetes. </span>2009;<span class="ref-vol">58</span>:1403–1410.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682673/">PMC free article</a>]</span> [<a href="/pubmed/19252134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Genome-wide+association+scan+for+diabetic+nephropathy+susceptibility+genes+in+type+1+diabetes&amp;author=MG+Pezzolesi&amp;author=GD+Poznik&amp;author=JC+Mychaleckyj&amp;author=AD+Paterson&amp;volume=58&amp;publication_year=2009&amp;pages=1403-1410&amp;pmid=19252134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Kottgen A, Glazer NL, Dehghan A, Hwang SJ, et al.  Multiple loci associated with indices of renal function and chronic kidney disease. <span><span class="ref-journal">Nat Genet. </span>2009</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039280/">PMC free article</a>]</span> [<a href="/pubmed/19430482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+loci+associated+with+indices+of+renal+function+and+chronic+kidney+disease&amp;author=A+Kottgen&amp;author=NL+Glazer&amp;author=A+Dehghan&amp;author=SJ+Hwang&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Kottgen A, Pattaro C, Boger CA, Fuchsberger C, et al.  New loci associated with kidney function and chronic kidney disease. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:376–384.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997674/">PMC free article</a>]</span> [<a href="/pubmed/20383146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+loci+associated+with+kidney+function+and+chronic+kidney+disease&amp;author=A+Kottgen&amp;author=C+Pattaro&amp;author=CA+Boger&amp;author=C+Fuchsberger&amp;volume=42&amp;publication_year=2010&amp;pages=376-384&amp;pmid=20383146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Chambers JC, Zhang W, Lord GM, van der Harst P, et al.  Genetic loci influencing kidney function and chronic kidney disease. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:373–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3748585/">PMC free article</a>]</span> [<a href="/pubmed/20383145" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+loci+influencing+kidney+function+and+chronic+kidney+disease&amp;author=JC+Chambers&amp;author=W+Zhang&amp;author=GM+Lord&amp;author=P+van+der+Harst&amp;volume=42&amp;publication_year=2010&amp;pages=373-375&amp;pmid=20383145&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">CDC: National Diabetes Fact Sheet: general information and national estimates on diabetes in the United States, 2007. <span class="ref-journal">U.S. Department of Health and Human Services.</span> Atlanta, GA: CDC; 2008.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=U.S.+Department+of+Health+and+Human+Services&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Kopp JB, Smith MW, Nelson GW, Johnson RC, et al.  MYH9 is a major-effect risk gene for focal segmental glomerulosclerosis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1175–1184.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2827354/">PMC free article</a>]</span> [<a href="/pubmed/18794856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=MYH9+is+a+major-effect+risk+gene+for+focal+segmental+glomerulosclerosis&amp;author=JB+Kopp&amp;author=MW+Smith&amp;author=GW+Nelson&amp;author=RC+Johnson&amp;volume=40&amp;publication_year=2008&amp;pages=1175-1184&amp;pmid=18794856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Kao WH, Klag MJ, Meoni LA, Reich D, et al.  MYH9 is associated with nondiabetic end-stage renal disease in African Americans. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1185–1192.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2614692/">PMC free article</a>]</span> [<a href="/pubmed/18794854" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=MYH9+is+associated+with+nondiabetic+end-stage+renal+disease+in+African+Americans&amp;author=WH+Kao&amp;author=MJ+Klag&amp;author=LA+Meoni&amp;author=D+Reich&amp;volume=40&amp;publication_year=2008&amp;pages=1185-1192&amp;pmid=18794854&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Freedman BI, Hicks PJ, Bostrom MA, Cunningham ME, et al.  Polymorphisms in the non-muscle myosin heavy chain 9 gene (MYH9) are strongly associated with end-stage renal disease historically attributed to hypertension in African Americans. <span><span class="ref-journal">Kidney Int. </span>2009;<span class="ref-vol">75</span>:736–745.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2698223/">PMC free article</a>]</span> [<a href="/pubmed/19177153" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Polymorphisms+in+the+non-muscle+myosin+heavy+chain+9+gene+(MYH9)+are+strongly+associated+with+end-stage+renal+disease+historically+attributed+to+hypertension+in+African+Americans&amp;author=BI+Freedman&amp;author=PJ+Hicks&amp;author=MA+Bostrom&amp;author=ME+Cunningham&amp;volume=75&amp;publication_year=2009&amp;pages=736-745&amp;pmid=19177153&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Freedman BI, Hicks PJ, Bostrom MA, Comeau ME, et al.  Non-muscle myosin heavy chain 9 gene MYH9 associations in African Americans with clinically diagnosed type 2 diabetes mellitus-associated ESRD. <span><span class="ref-journal">Nephrol Dial Transplant. </span>2009;<span class="ref-vol">24</span>:3366–3371.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2910323/">PMC free article</a>]</span> [<a href="/pubmed/19567477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=Non-muscle+myosin+heavy+chain+9+gene+MYH9+associations+in+African+Americans+with+clinically+diagnosed+type+2+diabetes+mellitus-associated+ESRD&amp;author=BI+Freedman&amp;author=PJ+Hicks&amp;author=MA+Bostrom&amp;author=ME+Comeau&amp;volume=24&amp;publication_year=2009&amp;pages=3366-3371&amp;pmid=19567477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Freedman BI, Hicks PJ, Sale MM, Pierson ED, et al.  A leucine repeat in the carnosinase gene CNDP1 is associated with diabetic end-stage renal disease in European Americans. <span><span class="ref-journal">Nephrol Dial Transplant. </span>2007;<span class="ref-vol">22</span>:1131–1135.</span> [<a href="/pubmed/17205963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=A+leucine+repeat+in+the+carnosinase+gene+CNDP1+is+associated+with+diabetic+end-stage+renal+disease+in+European+Americans&amp;author=BI+Freedman&amp;author=PJ+Hicks&amp;author=MM+Sale&amp;author=ED+Pierson&amp;volume=22&amp;publication_year=2007&amp;pages=1131-1135&amp;pmid=17205963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Janssen B, Hohenadel D, Brinkkoetter P, Peters V, et al.  Carnosine as a protective factor in diabetic nephropathy: association with a leucine repeat of the carnosinase gene CNDP1. <span><span class="ref-journal">Diabetes. </span>2005;<span class="ref-vol">54</span>:2320–2327.</span> [<a href="/pubmed/16046297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Carnosine+as+a+protective+factor+in+diabetic+nephropathy:+association+with+a+leucine+repeat+of+the+carnosinase+gene+CNDP1&amp;author=B+Janssen&amp;author=D+Hohenadel&amp;author=P+Brinkkoetter&amp;author=V+Peters&amp;volume=54&amp;publication_year=2005&amp;pages=2320-2327&amp;pmid=16046297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Leak TS, Perlegas PS, Smith SG, Keene KL, et al.  Variants in intron 13 of the ELMO1 gene are associated with diabetic nephropathy in African Americans. <span><span class="ref-journal">Ann Hum Genet. </span>2009;<span class="ref-vol">73</span>:152–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778056/">PMC free article</a>]</span> [<a href="/pubmed/19183347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Variants+in+intron+13+of+the+ELMO1+gene+are+associated+with+diabetic+nephropathy+in+African+Americans&amp;author=TS+Leak&amp;author=PS+Perlegas&amp;author=SG+Smith&amp;author=KL+Keene&amp;volume=73&amp;publication_year=2009&amp;pages=152-159&amp;pmid=19183347&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Maeda S, Kobayashi M, Araki S, Babazono T, et al.  A Single Nucleotide Polymorphism within the Acetyl-Coenzyme A Carboxylase Beta Gene Is Associated with Proteinuria in Patients with Type 2 Diabetes. <span><span class="ref-journal">Plos Genetics. </span>2010;<span class="ref-vol">6</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820513/">PMC free article</a>]</span> [<a href="/pubmed/20168990" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plos+Genetics&amp;title=A+Single+Nucleotide+Polymorphism+within+the+Acetyl-Coenzyme+A+Carboxylase+Beta+Gene+Is+Associated+with+Proteinuria+in+Patients+with+Type+2+Diabetes&amp;author=S+Maeda&amp;author=M+Kobayashi&amp;author=S+Araki&amp;author=T+Babazono&amp;volume=6&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">McDonough CW, Bostrom MA, Lu L, Hicks PJ, et al.  Genetic analysis of diabetic nephropathy on chromosome 18 in African Americans: linkage analysis and dense SNP mapping. <span><span class="ref-journal">Hum Genet. </span>2009;<span class="ref-vol">126</span>:805–817.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2937163/">PMC free article</a>]</span> [<a href="/pubmed/19690890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Genetic+analysis+of+diabetic+nephropathy+on+chromosome+18+in+African+Americans:+linkage+analysis+and+dense+SNP+mapping&amp;author=CW+McDonough&amp;author=MA+Bostrom&amp;author=L+Lu&amp;author=PJ+Hicks&amp;volume=126&amp;publication_year=2009&amp;pages=805-817&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">McDonough CW, Hicks PJ, Lu L, Langefeld CD, et al.  The influence of carnosinase gene polymorphisms on diabetic nephropathy risk in African-Americans. <span><span class="ref-journal">Hum Genet. </span>2009;<span class="ref-vol">126</span>:265–275.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2885788/">PMC free article</a>]</span> [<a href="/pubmed/19373489" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=The+influence+of+carnosinase+gene+polymorphisms+on+diabetic+nephropathy+risk+in+African-Americans&amp;author=CW+McDonough&amp;author=PJ+Hicks&amp;author=L+Lu&amp;author=CD+Langefeld&amp;volume=126&amp;publication_year=2009&amp;pages=265-275&amp;pmid=19373489&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Gitter J, Langefeld CD, Rich SS, Pedley CF, et al.  Prevalence of nephropathy in black patients with type 2 diabetes mellitus. <span><span class="ref-journal">Am J Nephrol. </span>2002;<span class="ref-vol">22</span>:35–41.</span> [<a href="/pubmed/11919401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Nephrol&amp;title=Prevalence+of+nephropathy+in+black+patients+with+type+2+diabetes+mellitus&amp;author=J+Gitter&amp;author=CD+Langefeld&amp;author=SS+Rich&amp;author=CF+Pedley&amp;volume=22&amp;publication_year=2002&amp;pages=35-41&amp;pmid=11919401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Lewis JP, Palmer ND, Hicks PJ, Sale MM, et al.  Association analysis in african americans of European-derived type 2 diabetes single nucleotide polymorphisms from whole-genome association studies. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:2220–2225.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2494685/">PMC free article</a>]</span> [<a href="/pubmed/18443202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Association+analysis+in+african+americans+of+European-derived+type+2+diabetes+single+nucleotide+polymorphisms+from+whole-genome+association+studies&amp;author=JP+Lewis&amp;author=ND+Palmer&amp;author=PJ+Hicks&amp;author=MM+Sale&amp;volume=57&amp;publication_year=2008&amp;pages=2220-2225&amp;pmid=18443202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Sale MM, Smith SG, Mychaleckyj JC, Keene KL, et al.  Variants of the transcription factor 7-like 2 (TCF7L2) gene are associated with type 2 diabetes in an African-American population enriched for nephropathy. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:2638–2642.</span> [<a href="/pubmed/17601994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Variants+of+the+transcription+factor+7-like+2+(TCF7L2)+gene+are+associated+with+type+2+diabetes+in+an+African-American+population+enriched+for+nephropathy&amp;author=MM+Sale&amp;author=SG+Smith&amp;author=JC+Mychaleckyj&amp;author=KL+Keene&amp;volume=56&amp;publication_year=2007&amp;pages=2638-2642&amp;pmid=17601994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Zeller C, Hinzmann B, Seitz S, Prokoph H, et al.  SASH1: a candidate tumor suppressor gene on chromosome 6q24.3 is downregulated in breast cancer. <span><span class="ref-journal">Oncogene. </span>2003;<span class="ref-vol">22</span>:2972–2983.</span> [<a href="/pubmed/12771949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=SASH1:+a+candidate+tumor+suppressor+gene+on+chromosome+6q24.3+is+downregulated+in+breast+cancer&amp;author=C+Zeller&amp;author=B+Hinzmann&amp;author=S+Seitz&amp;author=H+Prokoph&amp;volume=22&amp;publication_year=2003&amp;pages=2972-2983&amp;pmid=12771949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Rimkus C, Martini M, Friederichs J, Rosenberg R, et al.  Prognostic significance of downregulated expression of the candidate tumour suppressor gene SASH1 in colon cancer. <span><span class="ref-journal">Br J Cancer. </span>2006;<span class="ref-vol">95</span>:1419–1423.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2360597/">PMC free article</a>]</span> [<a href="/pubmed/17088907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Prognostic+significance+of+downregulated+expression+of+the+candidate+tumour+suppressor+gene+SASH1+in+colon+cancer&amp;author=C+Rimkus&amp;author=M+Martini&amp;author=J+Friederichs&amp;author=R+Rosenberg&amp;volume=95&amp;publication_year=2006&amp;pages=1419-1423&amp;pmid=17088907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Dubois F, Vandermoere F, Gernez A, Murphy J, et al.  Differential 14-3-3 affinity capture reveals new downstream targets of phosphatidylinositol 3-kinase signaling. <span><span class="ref-journal">Mol Cell Proteomics. </span>2009;<span class="ref-vol">8</span>:2487–2499.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2773716/">PMC free article</a>]</span> [<a href="/pubmed/19648646" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Proteomics&amp;title=Differential+14-3-3+affinity+capture+reveals+new+downstream+targets+of+phosphatidylinositol+3-kinase+signaling&amp;author=F+Dubois&amp;author=F+Vandermoere&amp;author=A+Gernez&amp;author=J+Murphy&amp;volume=8&amp;publication_year=2009&amp;pages=2487-2499&amp;pmid=19648646&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Yu Y, Ji H, Doudna JA, Leary JA. Mass spectrometric analysis of the human 40S ribosomal subunit: native and HCV IRES-bound complexes. <span><span class="ref-journal">Protein Sci. </span>2005;<span class="ref-vol">14</span>:1438–1446.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253395/">PMC free article</a>]</span> [<a href="/pubmed/15883184" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Mass+spectrometric+analysis+of+the+human+40S+ribosomal+subunit:+native+and+HCV+IRES-bound+complexes&amp;author=Y+Yu&amp;author=H+Ji&amp;author=JA+Doudna&amp;author=JA+Leary&amp;volume=14&amp;publication_year=2005&amp;pages=1438-1446&amp;pmid=15883184&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Wang Q, Wang Y, Minto AW, Wang J, et al.  MicroRNA-377 is up-regulated and can lead to increased fibronectin production in diabetic nephropathy. <span><span class="ref-journal">FASEB J. </span>2008;<span class="ref-vol">22</span>:4126–4135.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2614610/">PMC free article</a>]</span> [<a href="/pubmed/18716028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J&amp;title=MicroRNA-377+is+up-regulated+and+can+lead+to+increased+fibronectin+production+in+diabetic+nephropathy&amp;author=Q+Wang&amp;author=Y+Wang&amp;author=AW+Minto&amp;author=J+Wang&amp;volume=22&amp;publication_year=2008&amp;pages=4126-4135&amp;pmid=18716028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Okano I, Hiraoka J, Otera H, Nunoue K, et al.  Identification and characterization of a novel family of serine/threonine kinases containing two N-terminal LIM motifs. <span><span class="ref-journal">J Biol Chem. </span>1995;<span class="ref-vol">270</span>:31321–31330.</span> [<a href="/pubmed/8537403" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Identification+and+characterization+of+a+novel+family+of+serine/threonine+kinases+containing+two+N-terminal+LIM+motifs&amp;author=I+Okano&amp;author=J+Hiraoka&amp;author=H+Otera&amp;author=K+Nunoue&amp;volume=270&amp;publication_year=1995&amp;pages=31321-31330&amp;pmid=8537403&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Scott RW, Olson MF. LIM kinases: function, regulation and association with human disease. <span><span class="ref-journal">J Mol Med. </span>2007;<span class="ref-vol">85</span>:555–568.</span> [<a href="/pubmed/17294230" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Med&amp;title=LIM+kinases:+function,+regulation+and+association+with+human+disease&amp;author=RW+Scott&amp;author=MF+Olson&amp;volume=85&amp;publication_year=2007&amp;pages=555-568&amp;pmid=17294230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Sumi T, Matsumoto K, Takai Y, Nakamura T. Cofilin phosphorylation and actin cytoskeletal dynamics regulated by rho- and Cdc42-activated LIM-kinase 2. <span><span class="ref-journal">J Cell Biol. </span>1999;<span class="ref-vol">147</span>:1519–1532.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2174243/">PMC free article</a>]</span> [<a href="/pubmed/10613909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=Cofilin+phosphorylation+and+actin+cytoskeletal+dynamics+regulated+by+rho-+and+Cdc42-activated+LIM-kinase+2&amp;author=T+Sumi&amp;author=K+Matsumoto&amp;author=Y+Takai&amp;author=T+Nakamura&amp;volume=147&amp;publication_year=1999&amp;pages=1519-1532&amp;pmid=10613909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Edwards DC, Sanders LC, Bokoch GM, Gill GN. Activation of LIM-kinase by Pak1 couples Rac/Cdc42 GTPase signalling to actin cytoskeletal dynamics. <span><span class="ref-journal">Nat Cell Biol. </span>1999;<span class="ref-vol">1</span>:253–259.</span> [<a href="/pubmed/10559936" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=Activation+of+LIM-kinase+by+Pak1+couples+Rac/Cdc42+GTPase+signalling+to+actin+cytoskeletal+dynamics&amp;author=DC+Edwards&amp;author=LC+Sanders&amp;author=GM+Bokoch&amp;author=GN+Gill&amp;volume=1&amp;publication_year=1999&amp;pages=253-259&amp;pmid=10559936&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Ohashi K, Nagata K, Maekawa M, Ishizaki T, et al.  Rho-associated kinase ROCK activates LIM-kinase 1 by phosphorylation at threonine 508 within the activation loop. <span><span class="ref-journal">J Biol Chem. </span>2000;<span class="ref-vol">275</span>:3577–3582.</span> [<a href="/pubmed/10652353" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Rho-associated+kinase+ROCK+activates+LIM-kinase+1+by+phosphorylation+at+threonine+508+within+the+activation+loop&amp;author=K+Ohashi&amp;author=K+Nagata&amp;author=M+Maekawa&amp;author=T+Ishizaki&amp;volume=275&amp;publication_year=2000&amp;pages=3577-3582&amp;pmid=10652353&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Acevedo K, Moussi N, Li R, Soo P, et al.  LIM kinase 2 is widely expressed in all tissues. <span><span class="ref-journal">J Histochem Cytochem. </span>2006;<span class="ref-vol">54</span>:487–501.</span> [<a href="/pubmed/16399995" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Histochem+Cytochem&amp;title=LIM+kinase+2+is+widely+expressed+in+all+tissues&amp;author=K+Acevedo&amp;author=N+Moussi&amp;author=R+Li&amp;author=P+Soo&amp;volume=54&amp;publication_year=2006&amp;pages=487-501&amp;pmid=16399995&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Takahashi H, Koshimizu U, Miyazaki J, Nakamura T. Impaired spermatogenic ability of testicular germ cells in mice deficient in the LIM-kinase 2 gene. <span><span class="ref-journal">Dev Biol. </span>2002;<span class="ref-vol">241</span>:259–272.</span> [<a href="/pubmed/11784110" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Biol&amp;title=Impaired+spermatogenic+ability+of+testicular+germ+cells+in+mice+deficient+in+the+LIM-kinase+2+gene&amp;author=H+Takahashi&amp;author=U+Koshimizu&amp;author=J+Miyazaki&amp;author=T+Nakamura&amp;volume=241&amp;publication_year=2002&amp;pages=259-272&amp;pmid=11784110&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Kilmartin JV. Sfi1p has conserved centrin-binding sites and an essential function in budding yeast spindle pole body duplication. <span><span class="ref-journal">J Cell Biol. </span>2003;<span class="ref-vol">162</span>:1211–1221.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2173958/">PMC free article</a>]</span> [<a href="/pubmed/14504268" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=Sfi1p+has+conserved+centrin-binding+sites+and+an+essential+function+in+budding+yeast+spindle+pole+body+duplication&amp;author=JV+Kilmartin&amp;volume=162&amp;publication_year=2003&amp;pages=1211-1221&amp;pmid=14504268&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Anderson VE, Prudden J, Prochnik S, Giddings TH, Jr, et al.  Novel sfi1 alleles uncover additional functions for Sfi1p in bipolar spindle assembly and function. <span><span class="ref-journal">Mol Biol Cell. </span>2007;<span class="ref-vol">18</span>:2047–2056.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1877113/">PMC free article</a>]</span> [<a href="/pubmed/17392514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Cell&amp;title=Novel+sfi1+alleles+uncover+additional+functions+for+Sfi1p+in+bipolar+spindle+assembly+and+function&amp;author=VE+Anderson&amp;author=J+Prudden&amp;author=S+Prochnik&amp;author=TH+Giddings&amp;volume=18&amp;publication_year=2007&amp;pages=2047-2056&amp;pmid=17392514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Arrondel C, Vodovar N, Knebelmann B, Grunfeld JP, et al.  Expression of the nonmuscle myosin heavy chain IIA in the human kidney and screening for MYH9 mutations in Epstein and Fechtner syndromes. <span><span class="ref-journal">J Am Soc Nephrol. </span>2002;<span class="ref-vol">13</span>:65–74.</span> [<a href="/pubmed/11752022" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Expression+of+the+nonmuscle+myosin+heavy+chain+IIA+in+the+human+kidney+and+screening+for+MYH9+mutations+in+Epstein+and+Fechtner+syndromes&amp;author=C+Arrondel&amp;author=N+Vodovar&amp;author=B+Knebelmann&amp;author=JP+Grunfeld&amp;volume=13&amp;publication_year=2002&amp;pages=65-74&amp;pmid=11752022&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">D'Apolito M, Guarnieri V, Boncristiano M, Zelante L, et al.  Cloning of the murine non-muscle myosin heavy chain IIA gene ortholog of human MYH9 responsible for May-Hegglin, Sebastian, Fechtner, and Epstein syndromes. <span><span class="ref-journal">Gene. </span>2002;<span class="ref-vol">286</span>:215–222.</span> [<a href="/pubmed/11943476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Cloning+of+the+murine+non-muscle+myosin+heavy+chain+IIA+gene+ortholog+of+human+MYH9+responsible+for+May-Hegglin,+Sebastian,+Fechtner,+and+Epstein+syndromes&amp;author=M+D'Apolito&amp;author=V+Guarnieri&amp;author=M+Boncristiano&amp;author=L+Zelante&amp;volume=286&amp;publication_year=2002&amp;pages=215-222&amp;pmid=11943476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">Faul C, Asanuma K, Yanagida-Asanuma E, Kim K, et al.  Actin up: regulation of podocyte structure and function by components of the actin cytoskeleton. <span><span class="ref-journal">Trends Cell Biol. </span>2007;<span class="ref-vol">17</span>:428–437.</span> [<a href="/pubmed/17804239" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Cell+Biol&amp;title=Actin+up:+regulation+of+podocyte+structure+and+function+by+components+of+the+actin+cytoskeleton&amp;author=C+Faul&amp;author=K+Asanuma&amp;author=E+Yanagida-Asanuma&amp;author=K+Kim&amp;volume=17&amp;publication_year=2007&amp;pages=428-437&amp;pmid=17804239&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="citation">Knowler WC, Coresh J, Elston RC, Freedman BI, et al.  The Family Investigation of Nephropathy and Diabetes (FIND): design and methods. <span><span class="ref-journal">J Diabetes Complications. </span>2005;<span class="ref-vol">19</span>:1–9.</span> [<a href="/pubmed/15642484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Diabetes+Complications&amp;title=The+Family+Investigation+of+Nephropathy+and+Diabetes+(FIND):+design+and+methods&amp;author=WC+Knowler&amp;author=J+Coresh&amp;author=RC+Elston&amp;author=BI+Freedman&amp;volume=19&amp;publication_year=2005&amp;pages=1-9&amp;pmid=15642484&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="citation">Purcell S, Neale B, Todd-Brown K, Thomas L, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="citation">Tang H, Peng J, Wang P, Risch NJ. Estimation of individual admixture: analytical and study design considerations. <span><span class="ref-journal">Genet Epidemiol. </span>2005;<span class="ref-vol">28</span>:289–301.</span> [<a href="/pubmed/15712363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+individual+admixture:+analytical+and+study+design+considerations&amp;author=H+Tang&amp;author=J+Peng&amp;author=P+Wang&amp;author=NJ+Risch&amp;volume=28&amp;publication_year=2005&amp;pages=289-301&amp;pmid=15712363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="citation">Harley JB, Alarcon-Riquelme ME, Criswell LA, Jacob CO, et al.  Genome-wide association scan in women with systemic lupus erythematosus identifies susceptibility variants in ITGAM, PXK, KIAA1542 and other loci. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:204–210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3712260/">PMC free article</a>]</span> [<a href="/pubmed/18204446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+in+women+with+systemic+lupus+erythematosus+identifies+susceptibility+variants+in+ITGAM,+PXK,+KIAA1542+and+other+loci&amp;author=JB+Harley&amp;author=ME+Alarcon-Riquelme&amp;author=LA+Criswell&amp;author=CO+Jacob&amp;volume=40&amp;publication_year=2008&amp;pages=204-210&amp;pmid=18204446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3056271&amp;issue-id=195058&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140173895412928title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Results</a></li><li><a href="#S8title">Discussion</a></li><li><a href="#S9title">Methods</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S23title">Acknowledgements</a></li><li><a href="#idm140173937346784title">Footnotes</a></li><li><a href="#idm140173901489520title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3056271/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3056271/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3056271/pdf/nihms271243.pdf">PDF (623K)</a></li> | <li><a href="#" data-citationid="PMC3056271" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3056271%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3056271%2F&amp;text=A%20GENOME%20WIDE%20ASSOCIATION%20STUDY%20FOR%20DIABETIC%20NEPHROPATHY%20GENES%20IN%20AFRICAN%20AMERICANS" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3056271%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21150874" data-aiid="3056271" data-aid="3056271" data-iid="195058" data-domainid="319" data-domain="nihpa" data-accid="PMC3056271" data-md5="287122ece30ddc15cca87edb5668ceb3"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:25:24-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A16B8DD520E5100000000011000B1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3056271%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3056271/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>