<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3749730/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3749730/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1"><meta name="citation_authors" content="Amy Strange, Francesca Capon, Chris CA Spencer, Jo Knight, Michael E Weale, Michael H Allen, Anne Barton, Gavin Band, Céline Bellenguez, Judith GM Bergboer, Jenefer M Blackwell, Elvira Bramon, Suzannah J Bumpstead, Juan P Casas, Michael J Cork, Aiden Corvin, Panos Deloukas, Alexander Dilthey, Audrey Duncanson, Sarah Edkins, Xavier Estivill, Oliver Fitzgerald, Colin Freeman, Emiliano Giardina, Emma Gray, Angelika Hofer, Ulrike Hüffmeier, Sarah E Hunt, Alan D Irvine, Janusz Jankowski, Brian Kirby, Cordelia Langford, Jesús Lascorz, Joyce Leman, Stephen Leslie, Lotus Mallbris, Hugh S Markus, Christopher G Mathew, WH Irwin McLean, Ross McManus, Rotraut Mössner, Loukas Moutsianas, Åsa T Naluai, Frank O Nestle, Giuseppe Novelli, Alexandros Onoufriadis, Colin NA Palmer, Carlo Perricone, Matti Pirinen, Robert Plomin, Simon C Potter, Ramon M Pujol, Anna Rautanen, Eva Riveira-Munoz, Anthony W Ryan, Wolfgang Salmhofer, Lena Samuelsson, Stephen J Sawcer, Joost Schalkwijk, Catherine H Smith, Mona Ståhle, Zhan Su, Rachid Tazi-Ahnini, Heiko Traupe, Ananth C Viswanathan, Richard B Warren, Wolfgang Weger, Katarina Wolk, Nicholas Wood, Jane Worthington, Helen S Young, Patrick LJM Zeeuwen, Adrian Hayday, A David Burden, Christopher EM Griffiths, Juha Kere, André Reis, Gilean McVean, David M Evans, Matthew A Brown, Jonathan N Barker, Leena Peltonen, Peter Donnelly, Richard C Trembath"><meta name="citation_date" content="November 2010"><meta name="citation_issue" content="11"><meta name="citation_volume" content="42"><meta name="citation_firstpage" content="985"><meta name="citation_doi" content="10.1038/ng.694"><meta name="citation_pmid" content="20953190"><meta name="DC.Title" content="Genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Amy Strange"><meta name="DC.Contributor" content="Francesca Capon"><meta name="DC.Contributor" content="Chris CA Spencer"><meta name="DC.Contributor" content="Jo Knight"><meta name="DC.Contributor" content="Michael E Weale"><meta name="DC.Contributor" content="Michael H Allen"><meta name="DC.Contributor" content="Anne Barton"><meta name="DC.Contributor" content="Gavin Band"><meta name="DC.Contributor" content="Céline Bellenguez"><meta name="DC.Contributor" content="Judith GM Bergboer"><meta name="DC.Contributor" content="Jenefer M Blackwell"><meta name="DC.Contributor" content="Elvira Bramon"><meta name="DC.Contributor" content="Suzannah J Bumpstead"><meta name="DC.Contributor" content="Juan P Casas"><meta name="DC.Contributor" content="Michael J Cork"><meta name="DC.Contributor" content="Aiden Corvin"><meta name="DC.Contributor" content="Panos Deloukas"><meta name="DC.Contributor" content="Alexander Dilthey"><meta name="DC.Contributor" content="Audrey Duncanson"><meta name="DC.Contributor" content="Sarah Edkins"><meta name="DC.Contributor" content="Xavier Estivill"><meta name="DC.Contributor" content="Oliver Fitzgerald"><meta name="DC.Contributor" content="Colin Freeman"><meta name="DC.Contributor" content="Emiliano Giardina"><meta name="DC.Contributor" content="Emma Gray"><meta name="DC.Contributor" content="Angelika Hofer"><meta name="DC.Contributor" content="Ulrike Hüffmeier"><meta name="DC.Contributor" content="Sarah E Hunt"><meta name="DC.Contributor" content="Alan D Irvine"><meta name="DC.Contributor" content="Janusz Jankowski"><meta name="DC.Contributor" content="Brian Kirby"><meta name="DC.Contributor" content="Cordelia Langford"><meta name="DC.Contributor" content="Jesús Lascorz"><meta name="DC.Contributor" content="Joyce Leman"><meta name="DC.Contributor" content="Stephen Leslie"><meta name="DC.Contributor" content="Lotus Mallbris"><meta name="DC.Contributor" content="Hugh S Markus"><meta name="DC.Contributor" content="Christopher G Mathew"><meta name="DC.Contributor" content="WH Irwin McLean"><meta name="DC.Contributor" content="Ross McManus"><meta name="DC.Contributor" content="Rotraut Mössner"><meta name="DC.Contributor" content="Loukas Moutsianas"><meta name="DC.Contributor" content="Åsa T Naluai"><meta name="DC.Contributor" content="Frank O Nestle"><meta name="DC.Contributor" content="Giuseppe Novelli"><meta name="DC.Contributor" content="Alexandros Onoufriadis"><meta name="DC.Contributor" content="Colin NA Palmer"><meta name="DC.Contributor" content="Carlo Perricone"><meta name="DC.Contributor" content="Matti Pirinen"><meta name="DC.Contributor" content="Robert Plomin"><meta name="DC.Contributor" content="Simon C Potter"><meta name="DC.Contributor" content="Ramon M Pujol"><meta name="DC.Contributor" content="Anna Rautanen"><meta name="DC.Contributor" content="Eva Riveira-Munoz"><meta name="DC.Contributor" content="Anthony W Ryan"><meta name="DC.Contributor" content="Wolfgang Salmhofer"><meta name="DC.Contributor" content="Lena Samuelsson"><meta name="DC.Contributor" content="Stephen J Sawcer"><meta name="DC.Contributor" content="Joost Schalkwijk"><meta name="DC.Contributor" content="Catherine H Smith"><meta name="DC.Contributor" content="Mona Ståhle"><meta name="DC.Contributor" content="Zhan Su"><meta name="DC.Contributor" content="Rachid Tazi-Ahnini"><meta name="DC.Contributor" content="Heiko Traupe"><meta name="DC.Contributor" content="Ananth C Viswanathan"><meta name="DC.Contributor" content="Richard B Warren"><meta name="DC.Contributor" content="Wolfgang Weger"><meta name="DC.Contributor" content="Katarina Wolk"><meta name="DC.Contributor" content="Nicholas Wood"><meta name="DC.Contributor" content="Jane Worthington"><meta name="DC.Contributor" content="Helen S Young"><meta name="DC.Contributor" content="Patrick LJM Zeeuwen"><meta name="DC.Contributor" content="Adrian Hayday"><meta name="DC.Contributor" content="A David Burden"><meta name="DC.Contributor" content="Christopher EM Griffiths"><meta name="DC.Contributor" content="Juha Kere"><meta name="DC.Contributor" content="André Reis"><meta name="DC.Contributor" content="Gilean McVean"><meta name="DC.Contributor" content="David M Evans"><meta name="DC.Contributor" content="Matthew A Brown"><meta name="DC.Contributor" content="Jonathan N Barker"><meta name="DC.Contributor" content="Leena Peltonen"><meta name="DC.Contributor" content="Peter Donnelly"><meta name="DC.Contributor" content="Richard C Trembath"><meta name="DC.Date" content="2010 Nov"><meta name="DC.Identifier" content="10.1038/ng.694"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1"><meta property="og:type" content="article"><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3749730/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3749730/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3749730/pdf/emss-48359.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89A7B1DD51254100000000075A04D0.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3749730%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC3749730</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2013 Aug 22.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20953190" target="pmc_ext" ref="reftype=publisher&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2010 Nov; 42(11): 985–990. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Oct 17. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.694" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.694</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3749730</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS48359</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20953190">20953190</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies new psoriasis susceptibility loci and an interaction between <em>HLA-C</em> and <em>ERAP1</em></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Strange%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850986085552" co-class="co-affbox">Amy Strange</a>,<sup>§,</sup><sup>1</sup> <a href="/pubmed/?term=Capon%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850988397056" co-class="co-affbox">Francesca Capon</a>,<sup>§,</sup><sup>2</sup> <a href="/pubmed/?term=Spencer%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850986069728" co-class="co-affbox">Chris CA Spencer</a>,<sup>1</sup> <a href="/pubmed/?term=Knight%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850986067856" co-class="co-affbox">Jo Knight</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Weale%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850986065136" co-class="co-affbox">Michael E Weale</a>,<sup>2</sup> <a href="/pubmed/?term=Allen%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850986063152" co-class="co-affbox">Michael H Allen</a>,<sup>2</sup> <a href="/pubmed/?term=Barton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850986061168" co-class="co-affbox">Anne Barton</a>,<sup>4</sup> <a href="/pubmed/?term=Band%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850986059184" co-class="co-affbox">Gavin Band</a>,<sup>1</sup> <a href="/pubmed/?term=Bellenguez%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916716128" co-class="co-affbox">Céline Bellenguez</a>,<sup>1</sup> <a href="/pubmed/?term=Bergboer%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916714144" co-class="co-affbox">Judith GM Bergboer</a>,<sup>5</sup> <a href="/pubmed/?term=Blackwell%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916712160" co-class="co-affbox">Jenefer M Blackwell</a>,<sup>6</sup> <a href="/pubmed/?term=Bramon%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916710176" co-class="co-affbox">Elvira Bramon</a>,<sup>7</sup> <a href="/pubmed/?term=Bumpstead%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916708192" co-class="co-affbox">Suzannah J Bumpstead</a>,<sup>8</sup> <a href="/pubmed/?term=Casas%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916706208" co-class="co-affbox">Juan P Casas</a>,<sup>9</sup> <a href="/pubmed/?term=Cork%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916704224" co-class="co-affbox">Michael J Cork</a>,<sup>10</sup> <a href="/pubmed/?term=Corvin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916702240" co-class="co-affbox">Aiden Corvin</a>,<sup>11</sup> <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916700256" co-class="co-affbox">Panos Deloukas</a>,<sup>8</sup> <a href="/pubmed/?term=Dilthey%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916698272" co-class="co-affbox">Alexander Dilthey</a>,<sup>12</sup> <a href="/pubmed/?term=Duncanson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916696288" co-class="co-affbox">Audrey Duncanson</a>,<sup>13</sup> <a href="/pubmed/?term=Edkins%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916694304" co-class="co-affbox">Sarah Edkins</a>,<sup>8</sup> <a href="/pubmed/?term=Estivill%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916692320" co-class="co-affbox">Xavier Estivill</a>,<sup>14</sup> <a href="/pubmed/?term=Fitzgerald%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916690336" co-class="co-affbox">Oliver Fitzgerald</a>,<sup>15</sup> <a href="/pubmed/?term=Freeman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916688352" co-class="co-affbox">Colin Freeman</a>,<sup>1</sup> <a href="/pubmed/?term=Giardina%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916686368" co-class="co-affbox">Emiliano Giardina</a>,<sup>16</sup> <a href="/pubmed/?term=Gray%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916640192" co-class="co-affbox">Emma Gray</a>,<sup>8</sup> <a href="/pubmed/?term=Hofer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916638208" co-class="co-affbox">Angelika Hofer</a>,<sup>17</sup> <a href="/pubmed/?term=H%26%23x000fc%3Bffmeier%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916636224" co-class="co-affbox">Ulrike Hüffmeier</a>,<sup>18</sup> <a href="/pubmed/?term=Hunt%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916634240" co-class="co-affbox">Sarah E Hunt</a>,<sup>8</sup> <a href="/pubmed/?term=Irvine%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916632256" co-class="co-affbox">Alan D Irvine</a>,<sup>19</sup> <a href="/pubmed/?term=Jankowski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916630272" co-class="co-affbox">Janusz Jankowski</a>,<sup>20</sup> <a href="/pubmed/?term=Kirby%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916628288" co-class="co-affbox">Brian Kirby</a>,<sup>15</sup> <a href="/pubmed/?term=Langford%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916626304" co-class="co-affbox">Cordelia Langford</a>,<sup>8</sup> <a href="/pubmed/?term=Lascorz%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916624320" co-class="co-affbox">Jesús Lascorz</a>,<sup>21</sup> <a href="/pubmed/?term=Leman%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916622336" co-class="co-affbox">Joyce Leman</a>,<sup>22</sup> <a href="/pubmed/?term=Leslie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916620352" co-class="co-affbox">Stephen Leslie</a>,<sup>12</sup> <a href="/pubmed/?term=Mallbris%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916618368" co-class="co-affbox">Lotus Mallbris</a>,<sup>23</sup> <a href="/pubmed/?term=Markus%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916616384" co-class="co-affbox">Hugh S Markus</a>,<sup>24</sup> <a href="/pubmed/?term=Mathew%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916614400" co-class="co-affbox">Christopher G Mathew</a>,<sup>2</sup> <a href="/pubmed/?term=McLean%20WI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916612416" co-class="co-affbox">WH Irwin McLean</a>,<sup>25</sup> <a href="/pubmed/?term=McManus%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916610432" co-class="co-affbox">Ross McManus</a>,<sup>27</sup> <a href="/pubmed/?term=M%26%23x000f6%3Bssner%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916608448" co-class="co-affbox">Rotraut Mössner</a>,<sup>28</sup> <a href="/pubmed/?term=Moutsianas%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916606464" co-class="co-affbox">Loukas Moutsianas</a>,<sup>12</sup> <a href="/pubmed/?term=Naluai%20%26%23x000c5%3BT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916604480" co-class="co-affbox">Åsa T Naluai</a>,<sup>29</sup> <a href="/pubmed/?term=Nestle%20FO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916602496" co-class="co-affbox">Frank O Nestle</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Novelli%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916599776" co-class="co-affbox">Giuseppe Novelli</a>,<sup>16</sup> <a href="/pubmed/?term=Onoufriadis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916597792" co-class="co-affbox">Alexandros Onoufriadis</a>,<sup>2</sup> <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918371632" co-class="co-affbox">Colin NA Palmer</a>,<sup>30</sup> <a href="/pubmed/?term=Perricone%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918369648" co-class="co-affbox">Carlo Perricone</a>,<sup>31</sup> <a href="/pubmed/?term=Pirinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918367664" co-class="co-affbox">Matti Pirinen</a>,<sup>1</sup> <a href="/pubmed/?term=Plomin%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918365680" co-class="co-affbox">Robert Plomin</a>,<sup>32</sup> <a href="/pubmed/?term=Potter%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918363696" co-class="co-affbox">Simon C Potter</a>,<sup>8</sup> <a href="/pubmed/?term=Pujol%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918361712" co-class="co-affbox">Ramon M Pujol</a>,<sup>33</sup> <a href="/pubmed/?term=Rautanen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918359728" co-class="co-affbox">Anna Rautanen</a>,<sup>1</sup> <a href="/pubmed/?term=Riveira-Munoz%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918357744" co-class="co-affbox">Eva Riveira-Munoz</a>,<sup>14</sup> <a href="/pubmed/?term=Ryan%20AW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918355760" co-class="co-affbox">Anthony W Ryan</a>,<sup>26</sup> <a href="/pubmed/?term=Salmhofer%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918353776" co-class="co-affbox">Wolfgang Salmhofer</a>,<sup>17</sup> <a href="/pubmed/?term=Samuelsson%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918351792" co-class="co-affbox">Lena Samuelsson</a>,<sup>29</sup> <a href="/pubmed/?term=Sawcer%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918349808" co-class="co-affbox">Stephen J Sawcer</a>,<sup>34</sup> <a href="/pubmed/?term=Schalkwijk%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918347824" co-class="co-affbox">Joost Schalkwijk</a>,<sup>5</sup> <a href="/pubmed/?term=Smith%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918345840" co-class="co-affbox">Catherine H Smith</a>,<sup>2,</sup><sup>35</sup> <a href="/pubmed/?term=St%26%23x000e5%3Bhle%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918343120" co-class="co-affbox">Mona Ståhle</a>,<sup>23</sup> <a href="/pubmed/?term=Su%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918341136" co-class="co-affbox">Zhan Su</a>,<sup>1</sup> <a href="/pubmed/?term=Tazi-Ahnini%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918339152" co-class="co-affbox">Rachid Tazi-Ahnini</a>,<sup>10</sup> <a href="/pubmed/?term=Traupe%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918337168" co-class="co-affbox">Heiko Traupe</a>,<sup>36</sup> <a href="/pubmed/?term=Viswanathan%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918335184" co-class="co-affbox">Ananth C Viswanathan</a>,<sup>37,</sup><sup>38</sup> <a href="/pubmed/?term=Warren%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918332464" co-class="co-affbox">Richard B Warren</a>,<sup>39</sup> <a href="/pubmed/?term=Weger%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918330480" co-class="co-affbox">Wolfgang Weger</a>,<sup>17</sup> <a href="/pubmed/?term=Wolk%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918328496" co-class="co-affbox">Katarina Wolk</a>,<sup>23</sup> <a href="/pubmed/?term=Wood%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850918326512" co-class="co-affbox">Nicholas Wood</a>,<sup>40</sup> <a href="/pubmed/?term=Worthington%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916119712" co-class="co-affbox">Jane Worthington</a>,<sup>4</sup> <a href="/pubmed/?term=Young%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916117728" co-class="co-affbox">Helen S Young</a>,<sup>39</sup> <a href="/pubmed/?term=Zeeuwen%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916115744" co-class="co-affbox">Patrick LJM Zeeuwen</a>,<sup>5</sup> <a href="/pubmed/?term=Hayday%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916113760" co-class="co-affbox">Adrian Hayday</a>,<sup>34,</sup><sup>41,</sup><sup>42</sup> <a href="/pubmed/?term=Burden%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916110304" co-class="co-affbox">A David Burden</a>,<sup>22</sup> <a href="/pubmed/?term=Griffiths%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916108320" co-class="co-affbox">Christopher EM Griffiths</a>,<sup>39</sup> <a href="/pubmed/?term=Kere%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916106336" co-class="co-affbox">Juha Kere</a>,<sup>43,</sup><sup>44</sup> <a href="/pubmed/?term=Reis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916103616" co-class="co-affbox">André Reis</a>,<sup>18</sup> <a href="/pubmed/?term=McVean%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916101632" co-class="co-affbox">Gilean McVean</a>,<sup>12</sup> <a href="/pubmed/?term=Evans%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916099648" co-class="co-affbox">David M Evans</a>,<sup>45,</sup><sup>46</sup> <a href="/pubmed/?term=Brown%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916096928" co-class="co-affbox">Matthew A Brown</a>,<sup>46</sup> <a href="/pubmed/?term=Barker%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916094944" co-class="co-affbox">Jonathan N Barker</a>,<sup>2,</sup><sup>35</sup> <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916092224" co-class="co-affbox">Leena Peltonen</a>,<sup>8</sup> <a href="/pubmed/?term=Donnelly%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916090240" co-class="co-affbox">Peter Donnelly</a>,<sup>§,</sup><sup>1,</sup><sup>12</sup> and  <a href="/pubmed/?term=Trembath%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190" class="affpopup" co-rid="_co_idm139850916086784" co-class="co-affbox">Richard C Trembath</a><sup>§,</sup><sup>2,</sup><sup>3</sup>, Genetic Analysis of Psoriasis Consortium and the Wellcome Trust Case Control Consortium 2<sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139850986085552"><h3 class="no_margin">Amy Strange</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Strange%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Amy Strange</a></div></div><div id="_co_idm139850988397056"><h3 class="no_margin">Francesca Capon</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Capon%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Francesca Capon</a></div></div><div id="_co_idm139850986069728"><h3 class="no_margin">Chris CA Spencer</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Spencer%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Chris CA Spencer</a></div></div><div id="_co_idm139850986067856"><h3 class="no_margin">Jo Knight</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><p><sup>3</sup>National Institute for Health Research (NIHR), Biomedical Research Centre, Guy’s and St. Thomas’ NHS Foundation Trust and King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Knight%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Jo Knight</a></div></div><div id="_co_idm139850986065136"><h3 class="no_margin">Michael E Weale</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Weale%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Michael E Weale</a></div></div><div id="_co_idm139850986063152"><h3 class="no_margin">Michael H Allen</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Allen%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Michael H Allen</a></div></div><div id="_co_idm139850986061168"><h3 class="no_margin">Anne Barton</h3><p><sup>4</sup>Arthritis Research UK Epidemiology Unit, University of Manchester, Manchester Academic Health Science Centre, Manchester, UK </p><div>Find articles by <a href="/pubmed/?term=Barton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Anne Barton</a></div></div><div id="_co_idm139850986059184"><h3 class="no_margin">Gavin Band</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Band%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Gavin Band</a></div></div><div id="_co_idm139850916716128"><h3 class="no_margin">Céline Bellenguez</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Bellenguez%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Céline Bellenguez</a></div></div><div id="_co_idm139850916714144"><h3 class="no_margin">Judith GM Bergboer</h3><p><sup>5</sup>Department of Dermatology, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Bergboer%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Judith GM Bergboer</a></div></div><div id="_co_idm139850916712160"><h3 class="no_margin">Jenefer M Blackwell</h3><p><sup>6</sup>Genetics and Infection Laboratory, Cambridge Institute of Medical Research, Addenbrooke’s Hospital, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Blackwell%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Jenefer M Blackwell</a></div></div><div id="_co_idm139850916710176"><h3 class="no_margin">Elvira Bramon</h3><p><sup>7</sup>Division of Psychological Medicine and Psychiatry, Biomedical Research Centre for Mental Health at the Institute of Psychiatry, King’s College London and The South London and Maudsley NHS Foundation Trust, Denmark Hill, London, UK </p><div>Find articles by <a href="/pubmed/?term=Bramon%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Elvira Bramon</a></div></div><div id="_co_idm139850916708192"><h3 class="no_margin">Suzannah J Bumpstead</h3><p><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Bumpstead%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Suzannah J Bumpstead</a></div></div><div id="_co_idm139850916706208"><h3 class="no_margin">Juan P Casas</h3><p><sup>9</sup>Department of Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, London, UK </p><div>Find articles by <a href="/pubmed/?term=Casas%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Juan P Casas</a></div></div><div id="_co_idm139850916704224"><h3 class="no_margin">Michael J Cork</h3><p><sup>10</sup>Academic Unit of Dermatology Research, Department of Infection and Immunity, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="/pubmed/?term=Cork%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Michael J Cork</a></div></div><div id="_co_idm139850916702240"><h3 class="no_margin">Aiden Corvin</h3><p><sup>11</sup>Neuropsychiatric Genetics Research Group, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Corvin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Aiden Corvin</a></div></div><div id="_co_idm139850916700256"><h3 class="no_margin">Panos Deloukas</h3><p><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Panos Deloukas</a></div></div><div id="_co_idm139850916698272"><h3 class="no_margin">Alexander Dilthey</h3><p><sup>12</sup>Department of Statistics, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Dilthey%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Alexander Dilthey</a></div></div><div id="_co_idm139850916696288"><h3 class="no_margin">Audrey Duncanson</h3><p><sup>13</sup>Molecular and Physiological Sciences, The Wellcome Trust, London NW1 2BE, UK </p><div>Find articles by <a href="/pubmed/?term=Duncanson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Audrey Duncanson</a></div></div><div id="_co_idm139850916694304"><h3 class="no_margin">Sarah Edkins</h3><p><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Edkins%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Sarah Edkins</a></div></div><div id="_co_idm139850916692320"><h3 class="no_margin">Xavier Estivill</h3><p><sup>14</sup>Genes and Disease Programme, Centre for Genomic Regulation (CRG) and Public Health and Epidemiology Network Biomedical Research Center (CIBERESP), Barcelona, Spain </p><div>Find articles by <a href="/pubmed/?term=Estivill%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Xavier Estivill</a></div></div><div id="_co_idm139850916690336"><h3 class="no_margin">Oliver Fitzgerald</h3><p><sup>15</sup>St Vincent’s University Hospital, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Fitzgerald%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Oliver Fitzgerald</a></div></div><div id="_co_idm139850916688352"><h3 class="no_margin">Colin Freeman</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Freeman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Colin Freeman</a></div></div><div id="_co_idm139850916686368"><h3 class="no_margin">Emiliano Giardina</h3><p><sup>16</sup>Department of Biopathology, Centre of Excellence for Genomic risk Assessment in Multifactorial and Complex Diseases, School of Medicine, University of Rome Tor Vergata, Italy </p><div>Find articles by <a href="/pubmed/?term=Giardina%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Emiliano Giardina</a></div></div><div id="_co_idm139850916640192"><h3 class="no_margin">Emma Gray</h3><p><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Gray%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Emma Gray</a></div></div><div id="_co_idm139850916638208"><h3 class="no_margin">Angelika Hofer</h3><p><sup>17</sup>Department of Dermatology, Medical University of Graz, Graz, Austria </p><div>Find articles by <a href="/pubmed/?term=Hofer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Angelika Hofer</a></div></div><div id="_co_idm139850916636224"><h3 class="no_margin">Ulrike Hüffmeier</h3><p><sup>18</sup>Institute of Human Genetics, Uiversity of Erlangen-Nuremberg, Erlangen, Germany </p><div>Find articles by <a href="/pubmed/?term=H%26%23x000fc%3Bffmeier%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Ulrike Hüffmeier</a></div></div><div id="_co_idm139850916634240"><h3 class="no_margin">Sarah E Hunt</h3><p><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Hunt%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Sarah E Hunt</a></div></div><div id="_co_idm139850916632256"><h3 class="no_margin">Alan D Irvine</h3><p><sup>19</sup>Department of Clinical Medicine, Trinity College Dublin, Our Lady’s Children’s Hospital Crumlin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Irvine%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Alan D Irvine</a></div></div><div id="_co_idm139850916630272"><h3 class="no_margin">Janusz Jankowski</h3><p><sup>20</sup>Centre for Gastroenterology, Bart’s and the London School of Medicine and Dentistry, London, UK </p><div>Find articles by <a href="/pubmed/?term=Jankowski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Janusz Jankowski</a></div></div><div id="_co_idm139850916628288"><h3 class="no_margin">Brian Kirby</h3><p><sup>15</sup>St Vincent’s University Hospital, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Kirby%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Brian Kirby</a></div></div><div id="_co_idm139850916626304"><h3 class="no_margin">Cordelia Langford</h3><p><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Langford%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Cordelia Langford</a></div></div><div id="_co_idm139850916624320"><h3 class="no_margin">Jesús Lascorz</h3><p><sup>21</sup>Division of Molecular Genetic Epidemiology, German Cancer Research Center (DKFZ), Heidelberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Lascorz%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Jesús Lascorz</a></div></div><div id="_co_idm139850916622336"><h3 class="no_margin">Joyce Leman</h3><p><sup>22</sup>Department of Dermatology, Western Infirmary, Glasgow </p><div>Find articles by <a href="/pubmed/?term=Leman%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Joyce Leman</a></div></div><div id="_co_idm139850916620352"><h3 class="no_margin">Stephen Leslie</h3><p><sup>12</sup>Department of Statistics, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Leslie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Stephen Leslie</a></div></div><div id="_co_idm139850916618368"><h3 class="no_margin">Lotus Mallbris</h3><p><sup>23</sup>Dermatology Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Mallbris%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Lotus Mallbris</a></div></div><div id="_co_idm139850916616384"><h3 class="no_margin">Hugh S Markus</h3><p><sup>24</sup>Clinical Neurosciences, St George’s University of London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Markus%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Hugh S Markus</a></div></div><div id="_co_idm139850916614400"><h3 class="no_margin">Christopher G Mathew</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Mathew%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Christopher G Mathew</a></div></div><div id="_co_idm139850916612416"><h3 class="no_margin">WH Irwin McLean</h3><p><sup>25</sup>Epithelial Genetics Group, Division of Molecular Medicine, Colleges of Life Sciences and Medicine, Dentistry and Nursing, University of Dundee, Dundee, UK </p><div>Find articles by <a href="/pubmed/?term=McLean%20WI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">WH Irwin McLean</a></div></div><div id="_co_idm139850916610432"><h3 class="no_margin">Ross McManus</h3><p><sup>27</sup>Institute of Molecular Medicine, Trinity College Dublin, St. James’s Hospital, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=McManus%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Ross McManus</a></div></div><div id="_co_idm139850916608448"><h3 class="no_margin">Rotraut Mössner</h3><p><sup>28</sup>Department of Dermatology, University of Göttingen, Göttingen, Germany </p><div>Find articles by <a href="/pubmed/?term=M%26%23x000f6%3Bssner%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Rotraut Mössner</a></div></div><div id="_co_idm139850916606464"><h3 class="no_margin">Loukas Moutsianas</h3><p><sup>12</sup>Department of Statistics, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Moutsianas%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Loukas Moutsianas</a></div></div><div id="_co_idm139850916604480"><h3 class="no_margin">Åsa T Naluai</h3><p><sup>29</sup>Department of Medical and Clinical Genetics, Institute of Biomedicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden </p><div>Find articles by <a href="/pubmed/?term=Naluai%20%26%23x000c5%3BT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Åsa T Naluai</a></div></div><div id="_co_idm139850916602496"><h3 class="no_margin">Frank O Nestle</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><p><sup>3</sup>National Institute for Health Research (NIHR), Biomedical Research Centre, Guy’s and St. Thomas’ NHS Foundation Trust and King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Nestle%20FO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Frank O Nestle</a></div></div><div id="_co_idm139850916599776"><h3 class="no_margin">Giuseppe Novelli</h3><p><sup>16</sup>Department of Biopathology, Centre of Excellence for Genomic risk Assessment in Multifactorial and Complex Diseases, School of Medicine, University of Rome Tor Vergata, Italy </p><div>Find articles by <a href="/pubmed/?term=Novelli%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Giuseppe Novelli</a></div></div><div id="_co_idm139850916597792"><h3 class="no_margin">Alexandros Onoufriadis</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Onoufriadis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Alexandros Onoufriadis</a></div></div><div id="_co_idm139850918371632"><h3 class="no_margin">Colin NA Palmer</h3><p><sup>30</sup>Biomedical Research Institute, Ninewells Hospital and Medical School, Dundee, UK </p><div>Find articles by <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Colin NA Palmer</a></div></div><div id="_co_idm139850918369648"><h3 class="no_margin">Carlo Perricone</h3><p><sup>31</sup>La Sapienza University of Rome, Unit of Rheumatology, Department of Clinic and Medical Therapy, Rome, Italy. </p><div>Find articles by <a href="/pubmed/?term=Perricone%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Carlo Perricone</a></div></div><div id="_co_idm139850918367664"><h3 class="no_margin">Matti Pirinen</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Pirinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Matti Pirinen</a></div></div><div id="_co_idm139850918365680"><h3 class="no_margin">Robert Plomin</h3><p><sup>32</sup>Social, Genetic and Developmental Psychiatry Centre, King’s College London Institute of Psychiatry, Denmark Hill, London, UK </p><div>Find articles by <a href="/pubmed/?term=Plomin%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Robert Plomin</a></div></div><div id="_co_idm139850918363696"><h3 class="no_margin">Simon C Potter</h3><p><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Potter%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Simon C Potter</a></div></div><div id="_co_idm139850918361712"><h3 class="no_margin">Ramon M Pujol</h3><p><sup>33</sup>Dermatology Service, Hospital del Mar-IMAS, Barcelona, Spain </p><div>Find articles by <a href="/pubmed/?term=Pujol%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Ramon M Pujol</a></div></div><div id="_co_idm139850918359728"><h3 class="no_margin">Anna Rautanen</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Rautanen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Anna Rautanen</a></div></div><div id="_co_idm139850918357744"><h3 class="no_margin">Eva Riveira-Munoz</h3><p><sup>14</sup>Genes and Disease Programme, Centre for Genomic Regulation (CRG) and Public Health and Epidemiology Network Biomedical Research Center (CIBERESP), Barcelona, Spain </p><div>Find articles by <a href="/pubmed/?term=Riveira-Munoz%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Eva Riveira-Munoz</a></div></div><div id="_co_idm139850918355760"><h3 class="no_margin">Anthony W Ryan</h3><p><sup>26</sup>Department of Clinical Medicine, Trinity College Dublin, St. James’s Hospital, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Ryan%20AW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Anthony W Ryan</a></div></div><div id="_co_idm139850918353776"><h3 class="no_margin">Wolfgang Salmhofer</h3><p><sup>17</sup>Department of Dermatology, Medical University of Graz, Graz, Austria </p><div>Find articles by <a href="/pubmed/?term=Salmhofer%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Wolfgang Salmhofer</a></div></div><div id="_co_idm139850918351792"><h3 class="no_margin">Lena Samuelsson</h3><p><sup>29</sup>Department of Medical and Clinical Genetics, Institute of Biomedicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden </p><div>Find articles by <a href="/pubmed/?term=Samuelsson%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Lena Samuelsson</a></div></div><div id="_co_idm139850918349808"><h3 class="no_margin">Stephen J Sawcer</h3><p><sup>34</sup>University of Cambridge Department of Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Sawcer%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Stephen J Sawcer</a></div></div><div id="_co_idm139850918347824"><h3 class="no_margin">Joost Schalkwijk</h3><p><sup>5</sup>Department of Dermatology, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Schalkwijk%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Joost Schalkwijk</a></div></div><div id="_co_idm139850918345840"><h3 class="no_margin">Catherine H Smith</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><p><sup>35</sup>St John’s Institute of Dermatology, King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Smith%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Catherine H Smith</a></div></div><div id="_co_idm139850918343120"><h3 class="no_margin">Mona Ståhle</h3><p><sup>23</sup>Dermatology Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=St%26%23x000e5%3Bhle%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Mona Ståhle</a></div></div><div id="_co_idm139850918341136"><h3 class="no_margin">Zhan Su</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Su%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Zhan Su</a></div></div><div id="_co_idm139850918339152"><h3 class="no_margin">Rachid Tazi-Ahnini</h3><p><sup>10</sup>Academic Unit of Dermatology Research, Department of Infection and Immunity, The University of Sheffield, Sheffield, UK </p><div>Find articles by <a href="/pubmed/?term=Tazi-Ahnini%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Rachid Tazi-Ahnini</a></div></div><div id="_co_idm139850918337168"><h3 class="no_margin">Heiko Traupe</h3><p><sup>36</sup>Department of Dermatology, University of Münster, Münster, Germany </p><div>Find articles by <a href="/pubmed/?term=Traupe%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Heiko Traupe</a></div></div><div id="_co_idm139850918335184"><h3 class="no_margin">Ananth C Viswanathan</h3><p><sup>37</sup>Glaucoma Research Unit, Moorfields Eye Hospital NHS Foundation Trust, London, UK </p><p><sup>38</sup>Department of Genetics, University College London Institute of Ophthalmology, London, UK </p><div>Find articles by <a href="/pubmed/?term=Viswanathan%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Ananth C Viswanathan</a></div></div><div id="_co_idm139850918332464"><h3 class="no_margin">Richard B Warren</h3><p><sup>39</sup>Dermatological Sciences, Salford Royal NHS Foundation Trust, University of Manchester, Manchester Academic Health Science Centre, Manchester, UK </p><div>Find articles by <a href="/pubmed/?term=Warren%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Richard B Warren</a></div></div><div id="_co_idm139850918330480"><h3 class="no_margin">Wolfgang Weger</h3><p><sup>17</sup>Department of Dermatology, Medical University of Graz, Graz, Austria </p><div>Find articles by <a href="/pubmed/?term=Weger%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Wolfgang Weger</a></div></div><div id="_co_idm139850918328496"><h3 class="no_margin">Katarina Wolk</h3><p><sup>23</sup>Dermatology Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Wolk%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Katarina Wolk</a></div></div><div id="_co_idm139850918326512"><h3 class="no_margin">Nicholas Wood</h3><p><sup>40</sup>Department of Molecular Neuroscience, Institute of Neurology, Queen Square, London, UK </p><div>Find articles by <a href="/pubmed/?term=Wood%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Nicholas Wood</a></div></div><div id="_co_idm139850916119712"><h3 class="no_margin">Jane Worthington</h3><p><sup>4</sup>Arthritis Research UK Epidemiology Unit, University of Manchester, Manchester Academic Health Science Centre, Manchester, UK </p><div>Find articles by <a href="/pubmed/?term=Worthington%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Jane Worthington</a></div></div><div id="_co_idm139850916117728"><h3 class="no_margin">Helen S Young</h3><p><sup>39</sup>Dermatological Sciences, Salford Royal NHS Foundation Trust, University of Manchester, Manchester Academic Health Science Centre, Manchester, UK </p><div>Find articles by <a href="/pubmed/?term=Young%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Helen S Young</a></div></div><div id="_co_idm139850916115744"><h3 class="no_margin">Patrick LJM Zeeuwen</h3><p><sup>5</sup>Department of Dermatology, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Zeeuwen%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Patrick LJM Zeeuwen</a></div></div><div id="_co_idm139850916113760"><h3 class="no_margin">Adrian Hayday</h3><p><sup>34</sup>University of Cambridge Department of Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK </p><p><sup>41</sup>Division of Immunology, Infection and Inflammatory Disease; King’s College London, London, UK </p><p><sup>42</sup>Immuno Surveillance Laboratory, London Research Institute, London UK. </p><div>Find articles by <a href="/pubmed/?term=Hayday%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Adrian Hayday</a></div></div><div id="_co_idm139850916110304"><h3 class="no_margin">A David Burden</h3><p><sup>22</sup>Department of Dermatology, Western Infirmary, Glasgow </p><div>Find articles by <a href="/pubmed/?term=Burden%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">A David Burden</a></div></div><div id="_co_idm139850916108320"><h3 class="no_margin">Christopher EM Griffiths</h3><p><sup>39</sup>Dermatological Sciences, Salford Royal NHS Foundation Trust, University of Manchester, Manchester Academic Health Science Centre, Manchester, UK </p><div>Find articles by <a href="/pubmed/?term=Griffiths%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Christopher EM Griffiths</a></div></div><div id="_co_idm139850916106336"><h3 class="no_margin">Juha Kere</h3><p><sup>43</sup>Department of Biosciences and Nutrition, Karolinska Institutet </p><p><sup>44</sup>Science for Life Laboratory, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Kere%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Juha Kere</a></div></div><div id="_co_idm139850916103616"><h3 class="no_margin">André Reis</h3><p><sup>18</sup>Institute of Human Genetics, Uiversity of Erlangen-Nuremberg, Erlangen, Germany </p><div>Find articles by <a href="/pubmed/?term=Reis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">André Reis</a></div></div><div id="_co_idm139850916101632"><h3 class="no_margin">Gilean McVean</h3><p><sup>12</sup>Department of Statistics, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=McVean%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Gilean McVean</a></div></div><div id="_co_idm139850916099648"><h3 class="no_margin">David M Evans</h3><p><sup>45</sup>Folkhälsan Institute of Genetics, Helsinki, Finland </p><p><sup>46</sup>Department of Medical Genetics, University of Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Evans%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">David M Evans</a></div></div><div id="_co_idm139850916096928"><h3 class="no_margin">Matthew A Brown</h3><p><sup>46</sup>Department of Medical Genetics, University of Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Brown%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Matthew A Brown</a></div></div><div id="_co_idm139850916094944"><h3 class="no_margin">Jonathan N Barker</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><p><sup>35</sup>St John’s Institute of Dermatology, King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Barker%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Jonathan N Barker</a></div></div><div id="_co_idm139850916092224"><h3 class="no_margin">Leena Peltonen</h3><p><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Leena Peltonen</a></div></div><div id="_co_idm139850916090240"><h3 class="no_margin">Peter Donnelly</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </p><p><sup>12</sup>Department of Statistics, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Donnelly%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Peter Donnelly</a></div></div><div id="_co_idm139850916086784"><h3 class="no_margin">Richard C Trembath</h3><p><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </p><p><sup>3</sup>National Institute for Health Research (NIHR), Biomedical Research Centre, Guy’s and St. Thomas’ NHS Foundation Trust and King’s College London, London, UK </p><div>Find articles by <a href="/pubmed/?term=Trembath%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953190">Richard C Trembath</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139850988234400_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139850988234400_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139850988234400_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford, UK </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Division of Genetics and Molecular Medicine, King’s College London, London, UK </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>National Institute for Health Research (NIHR), Biomedical Research Centre, Guy’s and St. Thomas’ NHS Foundation Trust and King’s College London, London, UK </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Arthritis Research UK Epidemiology Unit, University of Manchester, Manchester Academic Health Science Centre, Manchester, UK </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Dermatology, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Genetics and Infection Laboratory, Cambridge Institute of Medical Research, Addenbrooke’s Hospital, Cambridge, UK </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Division of Psychological Medicine and Psychiatry, Biomedical Research Centre for Mental Health at the Institute of Psychiatry, King’s College London and The South London and Maudsley NHS Foundation Trust, Denmark Hill, London, UK </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, London, UK </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Academic Unit of Dermatology Research, Department of Infection and Immunity, The University of Sheffield, Sheffield, UK </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Neuropsychiatric Genetics Research Group, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Statistics, University of Oxford, Oxford, UK </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Molecular and Physiological Sciences, The Wellcome Trust, London NW1 2BE, UK </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Genes and Disease Programme, Centre for Genomic Regulation (CRG) and Public Health and Epidemiology Network Biomedical Research Center (CIBERESP), Barcelona, Spain </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>St Vincent’s University Hospital, Dublin, Ireland </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Biopathology, Centre of Excellence for Genomic risk Assessment in Multifactorial and Complex Diseases, School of Medicine, University of Rome Tor Vergata, Italy </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Dermatology, Medical University of Graz, Graz, Austria </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Institute of Human Genetics, Uiversity of Erlangen-Nuremberg, Erlangen, Germany </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Clinical Medicine, Trinity College Dublin, Our Lady’s Children’s Hospital Crumlin, Ireland </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Centre for Gastroenterology, Bart’s and the London School of Medicine and Dentistry, London, UK </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Division of Molecular Genetic Epidemiology, German Cancer Research Center (DKFZ), Heidelberg, Germany </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Dermatology, Western Infirmary, Glasgow </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Dermatology Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Clinical Neurosciences, St George’s University of London, London, UK </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Epithelial Genetics Group, Division of Molecular Medicine, Colleges of Life Sciences and Medicine, Dentistry and Nursing, University of Dundee, Dundee, UK </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Clinical Medicine, Trinity College Dublin, St. James’s Hospital, Dublin, Ireland </div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Institute of Molecular Medicine, Trinity College Dublin, St. James’s Hospital, Dublin, Ireland </div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Department of Dermatology, University of Göttingen, Göttingen, Germany </div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Department of Medical and Clinical Genetics, Institute of Biomedicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden </div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Biomedical Research Institute, Ninewells Hospital and Medical School, Dundee, UK </div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>La Sapienza University of Rome, Unit of Rheumatology, Department of Clinic and Medical Therapy, Rome, Italy. </div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Social, Genetic and Developmental Psychiatry Centre, King’s College London Institute of Psychiatry, Denmark Hill, London, UK </div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Dermatology Service, Hospital del Mar-IMAS, Barcelona, Spain </div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>University of Cambridge Department of Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK </div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>St John’s Institute of Dermatology, King’s College London, London, UK </div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>Department of Dermatology, University of Münster, Münster, Germany </div><div class="fm-affl" id="A37" lang="en"><sup>37</sup>Glaucoma Research Unit, Moorfields Eye Hospital NHS Foundation Trust, London, UK </div><div class="fm-affl" id="A38" lang="en"><sup>38</sup>Department of Genetics, University College London Institute of Ophthalmology, London, UK </div><div class="fm-affl" id="A39" lang="en"><sup>39</sup>Dermatological Sciences, Salford Royal NHS Foundation Trust, University of Manchester, Manchester Academic Health Science Centre, Manchester, UK </div><div class="fm-affl" id="A40" lang="en"><sup>40</sup>Department of Molecular Neuroscience, Institute of Neurology, Queen Square, London, UK </div><div class="fm-affl" id="A41" lang="en"><sup>41</sup>Division of Immunology, Infection and Inflammatory Disease; King’s College London, London, UK </div><div class="fm-affl" id="A42" lang="en"><sup>42</sup>Immuno Surveillance Laboratory, London Research Institute, London UK. </div><div class="fm-affl" id="A43" lang="en"><sup>43</sup>Department of Biosciences and Nutrition, Karolinska Institutet </div><div class="fm-affl" id="A44" lang="en"><sup>44</sup>Science for Life Laboratory, Stockholm, Sweden </div><div class="fm-affl" id="A45" lang="en"><sup>45</sup>Folkhälsan Institute of Genetics, Helsinki, Finland </div><div class="fm-affl" id="A46" lang="en"><sup>46</sup>Department of Medical Genetics, University of Helsinki, Finland </div><div class="fm-affl" id="A47" lang="en"><sup>47</sup>MRC Centre for Causal Analyses in Translational Epidemiology, Department of Social Medicine, University of Bristol, Bristol, UK </div><div class="fm-affl" id="A48" lang="en"><sup>48</sup>Diamantina Institute of Cancer, Immunology and Metabolic Medicine, Princess Alexandra Hospital, University of Queensland, Brisbane, Queensland, Australia. </div><div id="CR1">Correspondence should be addressed to P.D. (<a href="mailto:dev@null" data-email="ku.ca.xo.llew@yllennod.retep" class="oemail">ku.ca.xo.llew@yllennod.retep</a>) or R.C.T. (<a href="mailto:dev@null" data-email="ku.ca.lck@htabmert.drahcir" class="oemail">ku.ca.lck@htabmert.drahcir</a>)</div><div id="FN1"><sup>§</sup>These authors contributed equally to this work.</div><div id="FN2"><sup>*</sup>A full list of members of the Genetic Analysis of Psoriasis Consortium (GAP) and Wellcome Trust Case Control Consortium 2 (WTCCC2) consortia is provided in the <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Note online</a>.</div><div id="FN3"><strong>Author Contributions</strong> F.C., A.D.B., C.E.M.G., J.Kere., A. Reis, J.N.B and RCT oversaw cohort collection for both the discovery and the replication datasets. The WTCCC2 DNA, genotyping, data QC and informatics group executed GWAS sample handling, genotyping and quality control. The WTCCC2 data and analysis group, J.Knight, M.E.W and D.M.E. performed statistical analyses. A.Dilthey., S.L, L.Moutsianas and G.M. performed HLA imputation. A.S., F.C, C.C.A.S., J.Knight., M.E.W., A. Hayday, J.N.B., P. Donnelly and R.C.T. contributed to writing the manuscript. The WTCCC2 management committee (P.Donnelly (Chair)) monitored the execution of the GWAS. The GAP consortium contributed to sample collection. All authors reviewed the final manuscript.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139850988234400_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20953190" target="pmc_ext" ref="reftype=publisher&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3749730/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemantary Table 9.</strong></div><div><a href="/pmc/articles/PMC3749730/bin/NIHMS48359-supplement-Supplemantary_Table_9.xls" data-ga-action="click_feat_suppl">NIHMS48359-supplement-Supplemantary_Table_9.xls</a><span style="color:gray"> (37K)</span></div><div class="small guid">GUID:&nbsp;BEB54062-0F9E-4380-9131-F6807B243C17</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Text and Figures.</strong></div><div><a href="/pmc/articles/PMC3749730/bin/NIHMS48359-supplement-Supplementary_Text_and_Figures.pdf" data-ga-action="click_feat_suppl">NIHMS48359-supplement-Supplementary_Text_and_Figures.pdf</a><span style="color:gray"> (6.4M)</span></div><div class="small guid">GUID:&nbsp;2D1E95E0-5A8E-4181-84AB-764E09EE2C66</div></div></dd></dl><dl data-length="219" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">A list of the SNPs in the discovery scan exhibiting P &lt; 10<sup>−4</sup> are available in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>. Researchers can gain access to the data by applying to the data access committee (<a href="http://www.ebi.ac.uk/ega/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ebi.ac.uk/ega/</a>).</p></dd></dl></div><div id="idm139850919281696" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">To identify novel susceptibility loci for psoriasis, we undertook a genome-wide association study of 594,224 SNPs in 2,622 psoriatic individuals and 5,667 controls. We discovered associations at eight previously unreported regions of the genome. Seven harboured genes with recognised immune functions (<em>IL28RA</em>, <em>REL, IFIH1, ERAP1, TRAF3IP2</em>, <em>NFKBIA</em> and <em>TYK2</em>). These findings were replicated in 9,079 European samples with six loci generating a <em>P</em> &lt; 5 × 10<sup>−8</sup> and two resulting in <em>P</em> &lt; 5×10<sup>−7</sup> in the combined dataset. We also report compelling evidence for an interaction between the <em>HLA-C</em> and <em>ERAP1</em> loci (<em>P</em><sub>combined</sub> = 6.95×10<sup>−6</sup>). <em>ERAP1</em> plays an important role in MHC class I peptide processing. <em>ERAP1</em> disease variants only influenced psoriasis susceptibility in individuals carrying the <em>HLA-C</em> risk allele, a pattern also observed at the T cell regulator <em>ZAP70</em> (<em>P</em><sub>combined</sub> &lt; 5×10<sup>−7</sup>). Our findings implicate pathways that integrate epidermal barrier dysfunction with innate and adaptive immune dysregulation in psoriasis pathogenesis.</p><p id="P5">Psoriasis is a common, chronic, skin disease with a complex genetic and environmental etiology, characterised by epidermal hyper-proliferation, vascular remodelling and inflammation<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616250">1</a></sup>. Prior linkage and association studies have unambiguously identified a disease susceptibility locus of large effect (termed <em>PSORS1</em>), spanning ~ 300kb of the Major Histocompatibility Complex (MHC) class I region on chromosome 6p21<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616274">2</a></sup>. Large-scale resequencing has established <em>HLA-C</em> as the most likely <em>PSORS1</em> candidate gene<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616270">3</a></sup>.</p><p id="P6">Genome Wide Association Studies (GWAS) and analysis of candidate genomic regions have captured eight non-MHC loci which highlight three pathways of considerable biological relevance to psoriasis pathogenesis<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616252">4</a>-<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616259">9</a></sup>. These are NF-κB regulated signalling, mechanisms of T cell (particularly Th17) differentiation, and perturbation of the epidermal barrier. To identify additional risk variants we performed a GWAS as a component of the Wellcome Trust Case Control Consortium 2 (WTCCC2), powered by the inclusion of a substantially larger number of discovery samples, compared to previous psoriasis GWAS. We undertook replication of interesting findings in multiple cohorts drawn from across Europe.</p><p id="P7">Subjects for the GWAS discovery set were recruited from the UK and Ireland and were of self-reported European ancestry (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). Cases were genotyped on the Illumina Human660W-Quad and controls on Illumina custom Human1.2M-Duo (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Information</a>), with primary analysis performed on the overlapping set of SNPs. We performed stringent data quality control (see Methods), resulting in a GWAS data set comprising 2,178 individuals with psoriasis and 5,175 controls genotyped at 535,475 SNPs. Principal Components Analysis (PCA) of study data showed the first principal component acted to stratify individuals by population origin (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 1B</a>). We performed single SNP analysis using score tests under a logistic regression model which assumed multiplicative effects, including the first principal component as a covariate, and accounted for uncertainty in genotype calls as implemented in SNPTEST. After removal of known and replicated psoriasis association loci, the overdispersion factor<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616257">10</a></sup> of association test statistics was λ<sub>GC</sub> = 1.045 (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 1C</a>).</p><p id="P8">In <a href="/pmc/articles/PMC3749730/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> we show evidence of association in our GWAS data at all other previously reported psoriasis disease risk SNPs. Odds ratios (OR) outside the HLA region, range from 1.12 to 1.49 highlighting substantial power to detect susceptibility loci of small effect. As expected, SNPs within the chromosome 6p21 region showed strong evidence for association with psoriasis (top SNP rs10484554: P = 4 × 10<sup>−214</sup>, OR = 4.66, 95%CI: 4.23-5.13), which extended across a broad region surrounding the Class I MHC that includes the established risk locus at <em>HLA-C</em>. Comparison of disease models showed a strong departure from the multiplicative model (<em>P</em> = 1.95×10<sup>−08</sup>). Odds ratios for the heterozygote and homozygote are 5.41 (95%CI: 4.81-6.08) and 10.61 (95%CI: 7.86-14.33) respectively. While not exact, this is comparable to a dominant model, in keeping with previous observations<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616249">11</a></sup>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p8">GWAS signals from loci previously reported to be associated with psoriasis.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139850917984976" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Position<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Gene/<br>Locus</th><th rowspan="1" colspan="1" valign="middle" align="center">P value</th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">Reference</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">1p31</td><td rowspan="1" colspan="1" valign="middle" align="center">rs11209026</td><td rowspan="1" colspan="1" valign="middle" align="center">67478546</td><td rowspan="1" colspan="1" valign="middle" align="center">IL23R</td><td rowspan="1" colspan="1" valign="middle" align="center">7.13×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.49 (1.27-1.74)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R5" rid="R5" class=" bibr popnode">5</a></sup>]</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">1q21</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4112788</td><td rowspan="1" colspan="1" valign="middle" align="center">150817900</td><td rowspan="1" colspan="1" valign="middle" align="center">LCE3D</td><td rowspan="1" colspan="1" valign="middle" align="center">3.32×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.29 (1.19-1.40)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup>]</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">5q31</td><td rowspan="1" colspan="1" valign="middle" align="center">rs20541</td><td rowspan="1" colspan="1" valign="middle" align="center">132023863</td><td rowspan="1" colspan="1" valign="middle" align="center">IL13</td><td rowspan="1" colspan="1" valign="middle" align="center">2.32×10<sup>−02</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12 (1.02-1.24)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>]</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">5q33</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1024995<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">150456197</td><td rowspan="1" colspan="1" valign="middle" align="center">TNIP1</td><td rowspan="1" colspan="1" valign="middle" align="center">3.92×10<sup>−05</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.27 (1.14-1.44)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>]</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">5q33</td><td rowspan="1" colspan="1" valign="middle" align="center">rs3213094</td><td rowspan="1" colspan="1" valign="middle" align="center">158683347</td><td rowspan="1" colspan="1" valign="middle" align="center">IL12B</td><td rowspan="1" colspan="1" valign="middle" align="center">4.93×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.39 (1.26-1.53)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>]</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">6p21</td><td rowspan="1" colspan="1" valign="middle" align="center">rs10484554<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">31382534</td><td rowspan="1" colspan="1" valign="middle" align="center">HLA-C</td><td rowspan="1" colspan="1" valign="middle" align="center">4.06×10<sup>−214</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">4.66 (4.23-5.13)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>]</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">6q23</td><td rowspan="1" colspan="1" valign="middle" align="center">rs610604</td><td rowspan="1" colspan="1" valign="middle" align="center">138241110</td><td rowspan="1" colspan="1" valign="middle" align="center">TNFAIP3</td><td rowspan="1" colspan="1" valign="middle" align="center">6.54×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.13-1.32)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>]</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">12q13</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2066808</td><td rowspan="1" colspan="1" valign="middle" align="center">55024240</td><td rowspan="1" colspan="1" valign="middle" align="center">IL23A</td><td rowspan="1" colspan="1" valign="middle" align="center">2.49×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.49 (1.28-1.73)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup>]</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">20q13</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2235617<sup>d</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">47988384</td><td rowspan="1" colspan="1" valign="middle" align="center">ZNF313</td><td rowspan="1" colspan="1" valign="middle" align="center">1.65×10<sup>−06</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.20 (1.11-1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">[<sup><a href="#R4" rid="R4" class=" bibr popnode">4</a></sup>]</td></tr></tbody></table></div><div id="largeobj_idm139850917984976" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3749730/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P37" class="p p-first-last">SNPs reported in previous studies with robust GWAS and replication evidence. Odds ratios given for the previously reported risk allele.</p></div><div id="TFN2"><sup>a</sup>NCBI human genome build 36 coordinates.</div><div id="TFN3"><sup>b</sup><em>r</em><sup>2</sup> = 0.48 (0.49) with previously published SNP rs17728338<sup>8</sup>.</div><div id="TFN4"><sup>c</sup>At the HLA locus the results are presented for our top SNP</div><div id="TFN5"><sup>d</sup><em>r</em><sup>2</sup> = 1 with previously published SNP rs495337<sup>4</sup>. All <em>r</em><sup>2</sup> calculated from 58C and where different, the <em>r</em><sup>2</sup> from HapMap CEU is in brackets.</div></div></div><p id="P9">We identified 16 previously unreported loci for further investigation (<a href="/pmc/articles/PMC3749730/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>), including all regions with a <em>P</em> &lt; 10<sup>−7</sup> and further loci which yield a <em>P</em> ≤10<sup>−4</sup> that contain strong biological candidate genes<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616256">1</a></sup>. Seven independent replication datasets of psoriasis cases and controls were drawn from across Europe (10,060 individuals pre-QC). The most strongly associated SNP, or a surrogate, for each of the 16 previously unreported loci was genotyped using the Sequenom iPlex platform; see <a href="/pmc/articles/PMC3749730/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Tables 2</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">3</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3749730/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139850917999920" class="figure"><a href="/pmc/articles/PMC3749730/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3749730_emss-48359-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-48359-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3749730/bin/emss-48359-f0001.jpg"></a></div><div id="largeobj_idm139850917999920" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3749730/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3749730/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Plot of genome wide association results.</p><p id="__p3">Genome wide association results from 523,067 SNPs on chromosomes 1-22 and 12,408 SNPs on the X chromosome using the additive model in SNPTEST. The −log<sub>10</sub> (P) values are thresholded at 10<sup>−10</sup>. Regions in red are described in <a href="/pmc/articles/PMC3749730/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>. Regions which have been shown previously to be associated with psoriasis and which replicate in this study are highlighted in green, as described in <a href="/pmc/articles/PMC3749730/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p9">New loci identified in this study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139850917093136" class="xtable"><table frame="void" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="2" rowspan="1" valign="middle" align="center">RAF<sup>c</sup></th><th colspan="2" rowspan="1" valign="middle" align="center">Discovery sample</th><th colspan="2" rowspan="1" valign="middle" align="center">Replication sample</th><th rowspan="1" colspan="1" valign="middle" align="center">Combined</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">rsID</th><th rowspan="1" colspan="1" valign="middle" align="center">Position<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Gene of<br>interest (#)<sup>b</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Risk<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="center">cases</th><th rowspan="1" colspan="1" valign="middle" align="center">controls</th><th rowspan="1" colspan="1" valign="middle" align="center">P <sub>scan</sub></th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">P <sub>repl</sub></th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">P <sub>comb</sub></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">1p36</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4649203</td><td rowspan="1" colspan="1" valign="middle" align="center">24392507</td><td rowspan="1" colspan="1" valign="middle" align="center">IL28RA (2)</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.77</td><td rowspan="1" colspan="1" valign="middle" align="center">0.73</td><td rowspan="1" colspan="1" valign="middle" align="center">6.46×10<sup>−06</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.12-1.32)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.36×10<sup>−03</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.13 (1.05-1.22)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.89×10<sup>−08</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">2p16</td><td rowspan="1" colspan="1" valign="middle" align="center">rs702873</td><td rowspan="1" colspan="1" valign="middle" align="center">60935046</td><td rowspan="1" colspan="1" valign="middle" align="center">REL (5)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.62</td><td rowspan="1" colspan="1" valign="middle" align="center">0.56</td><td rowspan="1" colspan="1" valign="middle" align="center">1.32×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.23 (1.14-1.32)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.41×10<sup>−03</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12 (1.04-1.20)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.59×10<sup>−09</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">2q24</td><td rowspan="1" colspan="1" valign="middle" align="center">rs17716942</td><td rowspan="1" colspan="1" valign="middle" align="center">162968937</td><td rowspan="1" colspan="1" valign="middle" align="center">IFIH1 (5)</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.90</td><td rowspan="1" colspan="1" valign="middle" align="center">0.86</td><td rowspan="1" colspan="1" valign="middle" align="center">4.05×10<sup>−08</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.38 (1.23-1.54)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.82×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.29 (1.17-1.43)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.06×10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">3p24</td><td rowspan="1" colspan="1" valign="middle" align="center">rs6809854<sup>d</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">18759427</td><td rowspan="1" colspan="1" valign="middle" align="center">None</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.23</td><td rowspan="1" colspan="1" valign="middle" align="center">0.19</td><td rowspan="1" colspan="1" valign="middle" align="center">3.09×10<sup>−06</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.24 (1.14-1.36)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.92×10<sup>−03</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.14 (1.04-1.26)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12×10<sup>−07</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">5q15</td><td rowspan="1" colspan="1" valign="middle" align="center">rs27524</td><td rowspan="1" colspan="1" valign="middle" align="center">96127700</td><td rowspan="1" colspan="1" valign="middle" align="center">ERAP1 (2)</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.40</td><td rowspan="1" colspan="1" valign="middle" align="center">0.36</td><td rowspan="1" colspan="1" valign="middle" align="center">6.81×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.27 (1.18-1.38)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.96×10<sup>−04</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.13 (1.05-1.22)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.56×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">6q21</td><td rowspan="1" colspan="1" valign="middle" align="center">rs240993<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">111780407</td><td rowspan="1" colspan="1" valign="middle" align="center">TRAF3IP2 (4)</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.30</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25</td><td rowspan="1" colspan="1" valign="middle" align="center">8.71×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.36 (1.25-1.48)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.37×10<sup>−09</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.25 (1.16-1.34)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.29×10<sup>−20</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">rs458017<sup>e</sup><sup>f</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">111802784</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.09</td><td rowspan="1" colspan="1" valign="middle" align="center">0.06</td><td rowspan="1" colspan="1" valign="middle" align="center">8.11×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.62 (1.40-1.88)</td><td rowspan="1" colspan="1" valign="middle" align="center">9.59×10<sup>−08</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.37 (1.22-1.54)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.16×10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">14q13</td><td rowspan="1" colspan="1" valign="middle" align="center">rs8016947</td><td rowspan="1" colspan="1" valign="middle" align="center">34902417</td><td rowspan="1" colspan="1" valign="middle" align="center">NFKBIA (1)</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.61</td><td rowspan="1" colspan="1" valign="middle" align="center">0.57</td><td rowspan="1" colspan="1" valign="middle" align="center">4.26×10<sup>−06</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.19 (1.11-1.29)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.89×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.19 (1.11-1.27)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.52×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">19p13</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12720356<sup>g</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">10330975</td><td rowspan="1" colspan="1" valign="middle" align="center">TYK2 (9)</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.93</td><td rowspan="1" colspan="1" valign="middle" align="center">0.90</td><td rowspan="1" colspan="1" valign="middle" align="center">9.25×10<sup>−06</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.34 (1.18-1.52)</td><td rowspan="1" colspan="1" valign="middle" align="center">8.82×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.40 (1.23-1.61)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.04×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">rs280519<sup>g</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">10333933</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.51</td><td rowspan="1" colspan="1" valign="middle" align="center">0.47</td><td rowspan="1" colspan="1" valign="middle" align="center">8.51×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.20 (1.12-1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.93×10<sup>−04</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.13 (1.05-1.21)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.42×10<sup>−09</sup></td></tr></tbody></table></div><div id="largeobj_idm139850917093136" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3749730/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><p id="P42" class="p p-first-last">Evidence for association at loci reported in this study</p></div><div id="TFN7"><sup>a</sup>NCBI human genome build 36 coordinates</div><div id="TFN8"><sup>b</sup>The number in brackets shows the number of genes between recombination hotspots</div><div id="TFN9"><sup>c</sup>Risk Allele Frequency</div><div id="TFN10"><sup>d</sup>This SNP failed Sequenom genotyping so replication genotyping for this locus is at rs7428395 <em>r</em><sup>2</sup>=0.97 calculated from 58C (HapMap CEU <em>r</em><sup>2</sup>=1)</div><div id="TFN11"><sup>e</sup>Separate signals within the same locus, where the marginal P values and OR are reported. For further discussion and details of joint models, see text.</div><div id="TFN12"><sup>f</sup>This SNP failed Sequenom genotyping so replication genotyping for this locus is at rs465969 <em>r</em><sup>2</sup>=0.73 calculated from 58C (HapMap CEU <em>r</em><sup>2</sup>=0.82).</div><div id="TFN13"><sup>g</sup>Separate signals within the same locus, where the marginal P values and OR are reported. For further discussion and details of joint models, see text.</div></div></div><p id="P10">Eight regions showed strong evidence of association and independent replication (analysed using fixed-effects methods), six with <em>P</em> &lt; 5 × 10<sup>−8</sup> and two with <em>P</em> &lt; 5 × 10<sup>−7</sup> in the combined GWAS and replication dataset, with OR estimates in the replication sample ranging from 1.12 (chromosome 2p16) to 1.40 (chromosome 19p13). See <a href="/pmc/articles/PMC3749730/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, and <a href="/pmc/articles/PMC3749730/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> for the regional signal plots. Closer analysis (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary note</a>) of the original signal at the 2p16, 6q21, and 19p13 loci revealed multiple statistically independent SNP associations. Two of these were tested and confirmed in our replication study, see <a href="/pmc/articles/PMC3749730/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. Full details are given in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a>. Where there are signals at two SNPs within a locus it is difficult from our data to rule out the possibility that the pattern of association is generated by a single untyped SNP. Regardless of this, we note that under a multiplicative model, the effect of carrying an additional copy of each of the risk alleles at 2p16, 6q21, and 19p13 is 1.40 (95%CI: 1.25-1.56), 1.66 (95%CI: 1.44-1.92), and 1.52 (95%CI: 1.31-1.75) respectively. One important consequence of the potential allelic heterogeneity, is that the overall genetic effect at the locus is larger than that associated with the single SNP with the lowest <em>P</em> value.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3749730/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139850917993904" class="figure"><a href="/pmc/articles/PMC3749730/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3749730_emss-48359-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-48359-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3749730/bin/emss-48359-f0002.jpg"></a></div><div id="largeobj_idm139850917993904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3749730/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3749730/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p4">Regional association plots.</p><p id="__p5">The -log<sub>10</sub> (P) values for SNPs at eight new loci are shown on the upper part of each plot. SNPs are coloured based on their <em>r</em><sup>2</sup> with the labeled hit SNP which has the smallest P value in the region. <em>r</em><sup>2</sup> is calculated from the 58C control data. Where more than one SNP is labeled, there is evidence for multiple signals at the locus (see text). The bottom section of each plot shows the fine scale recombination rates estimated from HapMap individuals and genes are marked by horizontal blue lines. Genes within the recombination region of the hit SNP are labeled except for 19p13, where the genes are: GLP-1, FDX1L, RAVER1, ICAM3, TKY2, CDC37, PDE4A, KEAP1, S1PR5.</p></div></div></div><p id="P11">While most of the novel disease-associated intervals (as defined by recombination hotspots) in <a href="/pmc/articles/PMC3749730/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> span multiple transcripts, all but one contain at least one immune-related gene. These genes are strong biological candidates, and it is striking that they collectively cluster into a limited number of pathways with recognised roles in innate and adaptive immunity. For instance, <em>ERAP1</em> in the chromosome 5q15 associated region encodes an amino-peptidase regulating the quality of peptides bound to MHC class I molecules, such as <em>HLA-C</em><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616260">12</a></sup>. The SNP rs30187 within this interval is also significantly associated with ankylosing spondylitis<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616267">13</a></sup>. Given the established link between psoriasis and the development of spondyloarthropathy<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616263">14</a></sup>, the finding of an association at the <em>ERAP1</em> locus in both conditions is noteworthy, particularly as the coding SNP rs30187 also generated a very significant <em>P</em> value in our GWAS (<em>P</em> = 1.5 × 10<sup>−9</sup>, OR= 1.27, 95% CI: 1.13-1.31). Conditional analyses in our association data do not distinguish significantly between our top SNP rs27524, and the coding variant rs30187.</p><p id="P12">The chromosome 6q21 region spans four known genes (<a href="/pmc/articles/PMC3749730/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Of these, <em>TRAF3IP2</em> encodes Act1, an adaptor protein essential for IL17-dependent NF-κB activation and Th17-mediated inflammatory responses<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616245">15</a></sup>. Remarkably, genes participating in NF-κB and IL17-mediated signalling are also found in three further disease susceptibility regions identified by this study. <em>NFKBIA</em>, the only gene lying within the 14q13 associated region, encodes a key inhibitory protein that inactivates NF-κB dimers in the absence of inflammatory stimuli<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616262">16</a></sup>. Likewise, <em>REL</em>, which maps to the associated region on chromosome 2p16, encodes one of four subunits found in NF-κB dimers and has recently been implicated in risk for rheumatoid arthritis<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616271">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616276">17</a></sup>. Finally, the interval on chromosome 19p13 includes <em>TYK2</em>, which encodes a kinase promoting IL17 transcription via STAT3 phosphorylation<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616251">18</a></sup>. Common and low frequency variants of TYK2 confers risk in multiple autoimmune diseases, namely Type I diabetes and Systemic lupus erythematosus<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616268">19</a></sup>.</p><p id="P13">It is noteworthy that TYK2 also regulates type I and type III interferon (IFN) signaling<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616261">20</a></sup>, suggesting that innate IFN-activating anti-microbial response pathways may contribute to initiation or maintenance of disease progression. This would be consistent with our observation of disease-associated variants in the proximity of <em>IL28RA</em>, which encodes a type III IFN receptor subunit<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616278">21</a></sup>. Pathogenic involvement of IFN signaling pathways is also supported by the association signal detected on chromosome 2q24. In this case, the associated region encompasses the IFIH1, FAP and GCA genes together with part of KCNH7 as shown in <a href="/pmc/articles/PMC3749730/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>. Of these, IFIH1 encodes an innate receptor triggering type I IFN in response to microbial infection<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616279">22</a></sup> and both rare and common IFIH1 variants are associated with type I diabetes<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616246">23</a></sup>. The SNP rs1990760 (minor allele frequency 0.38) has a strong signal in our sample and type 1 diabetes but the correlation between this SNP and rs17716942 is low (<em>r<sup>2</sup></em> = 0.3). Further resequencing of IFIH1 will be required to determine the properties of rare variation in relation to psoriasis risk.</p><p id="P14">Given the coincidence of many signals within a small number of pathways, we looked for statistical interactions (see methods) between the top SNPs in each of the 17 known and confirmed novel regions in <a href="/pmc/articles/PMC3749730/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and <a href="/pmc/articles/PMC3749730/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2,</span></span><span>2</span></a>, fitting a dominant model at the <em>HLA-C</em> SNP rs10484554. One pair of SNPs, (rs10484554 (<em>HLA-C</em>) and rs27524 at <em>ERAP1</em>), generated strong evidence for interaction in the discovery, replication and combined data (Pdiscovery = 2.45×10<sup>−5</sup>, P<sub>replication</sub> = 0.027, P<sub>combined</sub> = 6.95×10<sup>−6</sup>) (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 5</a>). <a href="/pmc/articles/PMC3749730/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> shows that when stratified by genotypes at the two loci, the <em>ERAP1</em> SNP only has an effect in individuals carrying at least one copy of the risk allele at rs10484554.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3749730/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139850917987648" class="figure"><a href="/pmc/articles/PMC3749730/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3749730_emss-48359-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-48359-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3749730/bin/emss-48359-f0003.jpg"></a></div><div id="largeobj_idm139850917987648" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3749730/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3749730/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p6">Statistical interaction between <em>ERAP1</em> and <em>HLA-C</em> genotypes.</p><p id="__p7">Odds ratio estimates for <em>ERAP1</em> genotypes at rs27524 stratified by <em>HLA-C</em> genotypes at rs10484554. Odds ratios were estimated by fitting a logistic regression model which included the first principal component as a covariate and a parameter for each of the possible two-locus genotypes. Note that odds are measured relative to the most protective two-locus genotype (which by definition has an odds ratio of 1). Bars represent 95% confidence intervals calculated from the standard error of the estimates of the genotype parameters in logistic regression model.</p></div></div></div><p id="P15">Very few convincing examples of interactions between complex disease loci have been reported in humans<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616255">24</a>-<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616248">26</a></sup>, perhaps because the power to detect these is limited unless the causal SNPs or very good surrogates have been typed<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616269">27</a></sup>. The finding that variation at the ERAP1 locus is only associated with disease in individuals carrying the HLA-C risk allele is particularly interesting biologically, because of ERAP1’s role in class I peptide presentation. It is also noteworthy that the odds ratio for rs27524 is 1.43 (1.21-1.69) in the HLA-C positive subgroup, compared to the estimate of 1.27 (1.18-1.38) from the GWAS on the entire dataset (in the replication data these estimates were 1.23 (1.09-1.39) and 1.13 (1.05-1.22) respectively). Were this phenomenon to be more widespread, it would impact on the proportion of broad-sense heritability explained by GWAS findings.</p><p id="P16">Next, we looked systematically across the genome for other loci which might only have an effect on one or other of the <em>HLA-C</em> risk backgrounds. We divided individuals in our discovery study into two groups, depending on whether they carried zero copies (“<em>HLA-C</em> negative” individuals: 831 cases and 3,985 controls) or one/two copies (“<em>HLA-C</em> positive” individuals: 1,347 cases and 1,190 controls) of the risk allele at rs10484554, the index SNP for the <em>HLA-C</em> locus, and undertook separate GWAS in each group (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>
<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>).</p><p id="P17">Four other loci generated evidence for association in the GWAS of <em>HLA-C</em> positive individuals. One of these, SNP rs17695937, near <em>ZAP70</em>, also displayed a clear signal in the replication data, generating a <em>P</em><sub>combined</sub> = 2.37×10<sup>−7</sup>. Several of the other loci also warrant further investigation (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>). The identification of risk variation at <em>ZAP70</em> together with <em>ERAP1</em> in individuals with <em>HLA-C</em> risk alleles provides further genetic evidence that psoriasis may in part be caused by dysregulation of HLA-restricted CD8 T cells. ZAP70 is a tyrosine kinase that binds to the T cell receptor-associated chain, CD3-zeta (CD247) following the TCR engagement of MHC-peptide complexes, and is critical for setting the response threshold that in turn governs the T cell selection events which purge the T cell compartment of autoreactivity<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616265">28</a></sup>. Finally, although we followed-up several SNPs from the HLA-negative GWAS, none replicated (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>).</p><p id="P18">We undertook additional analyses of the relationship between the known classical HLA risk allele, Cw*0602 and typed SNPs. Based on molecular typing in a subset of 725 of the 58C individuals, the correlation between Cw*0602 genotypes (0, 1, or 2 copies of Cw*0602) and our top SNP, rs10484554, is <em>r</em><sup>2</sup>= 0.7. We imputed the <em>HLA-C</em> genotypes from our SNP data<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616273">29</a></sup> (see Methods). Association testing with these imputed genotypes showed a stronger signal for the classical allele than for any single SNP yet conditional analyses are not definitive in ruling out either variant as the primary source of the signal (see <a href="#SD2" rid="SD2" class=" supplementary-material">supplementary data</a>). We reexamined the <em>HLA-C/ERAP1</em> interaction in an attempt to ascertain whether it was driven by the top SNP (rs10484554) or classical allele at <em>HLA-C</em>, and likewise whether the top SNP (rs27524) or the previously reported coding SNP (rs30187) at <em>ERAP1</em> was more closely involved. Although evidence for interaction existed for all four pairs of markers (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 7</a>), the dominant model at the HLA Cw*0602 with an interaction at <em>ERAP1</em> SNP rs27524 best fits the GWAS data. Conditional analyses are not definitive in ruling out either variant at either site as the primary source of the interaction signal.</p><p id="P19" class="p">Collectively, the candidate genes identified in this study are relevant to the question of whether psoriasis is primarily an epidermal or immune dysfunction disorder. The present data provide evidence for an integrated model for the pathogenesis of psoriasis that combines skin barrier function (e.g. variants in <em>LCE3B/3C</em>); innate-immune pathogen sensing focused on NF-κB and IFN signalling (e.g. <em>NFKBIA, REL, TYK2, IFIH1, IL28RA</em>); and adaptive immunity involving CD8 T lymphocytes (<em>ERAP1</em> and <em>ZAP70</em>) and Th17 cell responses (e.g. <em>TRAF3IP3</em>, <em>TYK2</em>, <em>IL23R</em>). Fine mapping, functional characterisation and meta-analysis in additional psoriasis cohorts can now determine the detailed molecular consequences of genetic variation on these pathways. Finally, this study also provides one of the first compelling examples of an interaction between two GWAS loci, and the success of our separate GWAS in <em>HLA-C</em> positive individuals suggests that similar analyses might be valuable in other auto-immune diseases.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">ONLINE METHODS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Samples</h3><div id="S3" class="sec sec-first"><p></p><h4 id="S3title" class="inline">Cases </h4><p id="P20" class="p p-first-last">A total of 6,523 samples were assembled for this investigation of which 2,735 were processed for the GWAS discovery set and 3,788 for the replication set. For all samples, psoriasis was defined as psoriasis vulgaris (chronic plaque type psoriasis) the commonest manifestation of the disease. A total of 2,178 DNA samples from unrelated individuals of European ancestry, recruited from five centres in England, Scotland and Ireland, were included in the GWAS discovery set, having passed pre- and post-genotyping control filters (see below). Phenotypic data and blood samples were collected after research ethics approval was received at each site and patients had given written, informed consent. The replication set of DNA was derived from patients with psoriasis recruited in several European countries (Austria, Germany, Italy, Netherlands, Spain, Sweden, and UK) after research ethics approval and written informed consent was given. In total 3,174 DNA samples passed quality control for replication.</p></div><div id="S4" class="sec sec-last"><p></p><h4 id="S4title" class="inline">Controls </h4><p id="P21" class="p p-first">A total of 10,639 control individuals passed the quality control filters (see below). 5,175 individuals from the WTCCC2 common control set were used in the discovery GWAS. This included 2,501 healthy blood donors from the United Kingdom Blood Service (UKBS) collection and 2,674 individuals from the 1958 Birth Cohort (58C) dataset<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616275">27</a></sup>. 5,464 individuals were used in the replication dataset including 2,717 from the People of the British Isles (PoBI) collection. The remaining controls were taken from six other European populations selected to match the case individuals. These collections have been described elsewhere (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>).</p><p id="P22" class="p p-last">Details of DNA sample preparation and genotyping are given in the <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Note</a>.</p></div></div><div id="S5" class="sec"><h3 id="S5title">Quality Control (QC)</h3><p id="P23" class="p p-first">Full details of QC procedures are given in the <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Note</a>. In brief:</p><div id="S6" class="sec"><p></p><h4 id="S6title" class="inline">Samples </h4><p id="P24" class="p p-first-last">We used a recently developed Bayesian clustering approach to infer and exclude outlying individuals on the basis of call rate, heterozygosity, relatedness and ancestry<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616254">30</a></sup>, and for an additional artifact which led to spurious associations, where for some samples SNP intensities in one channel were systematically higher than in the other. We also excluded one of each pair of related individuals, as identified by a hidden Markov model.</p></div><div id="S7" class="sec sec-last"><p></p><h4 id="S7title" class="inline">SNPs </h4><p id="P25" class="p p-first">SNPs were excluded unless the (Fisher) information for the allele frequency was very close to unity, or if minor allele frequency was very low (&lt;0.01%), or for extreme departures from Hardy-Weinberg equilibrium (except in the MHC), or if they showed a strong plate effect. Cluster plots of SNPs showing putative associations were inspected manually (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 3</a>).</p><p id="P26" class="p p-last">The QC measures excluded 444 case and 492 control individuals, and 9.8% of SNPs. Of the 2,178 case samples which passed quality control, 345 have self-reported Irish ancestry (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 8</a>).</p></div></div><div id="S8" class="sec"><h3 id="S8title">Statistical methods</h3><p id="P27" class="p p-first">We performed Principal Component Analysis (PCA) on a subset of 205,842 post-QC SNPs (none from the MHC), selected so as to minimize the contribution from regions of extensive strong linkage disequilibrium and to ensure that only genome-wide effects are detected. Principal component scores were computed for the combined dataset of post-exclusion case and control samples. On inspection, the first principal component acts to differentiate UK from Irish individuals (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 1A</a>) suggesting that the major determinant of the variance in genome-wide patterns of diversity between sample individuals was due to differences in ancestry. To guard against possible artifacts due to population structure, we repeated the primary association analysis excluding the Irish cases. Reassuringly, this yielded comparable association signals after allowing for the reduction in sample size (data not shown). As no further principal components driven by genome-wide patterns exhibited significant differences between cases and controls we used only the first principal component to control for population stratification in subsequent analyses.</p><p id="P28">We used a logistic regression model with case/control status as the response variable, the first principal component as a covariate and the genotype at a particular SNP as the explanatory variable. Genome wide association tests were carried out at each SNP with uncertainty in genotype calls modeled using missing data likelihoods as implemented in SNPTEST. Unless otherwise stated, we assumed that the change in odds of case status due to each copy of the allele was multiplicative. To investigate patterns of association between SNPs, for example to look for interactions or secondary signals, we used the statistical package R<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup>.</p><p id="P29">It has become standard practice in GWAS to refer to the odds ratio associated with a particular allele or haplotype, which we estimate as e<sup>β</sup>, where β is the maximum likelihood estimate of the coefficient describing the effect of each predictor on the response in the assumed model. We note however that, as is true of this study and many others, where the controls are taken at random from the population without reference to disease status, β is actually the log of the <em>relative risk</em> and not the log of the odds ratio<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616277">32</a></sup>.</p><p id="P30">Imputation was performed using IMPUTE2<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616253">33</a></sup>, which adopts a two-stage approach using both a haploid reference panel and a diploid reference panel. We used two methods to investigate possible secondary signals within GWAS association regions, standard frequentist conditional analyses and the program GENECLUSTER<sup><a href="#R34" rid="R34" class=" bibr popnode">34</a></sup> which adopts a Bayesian approach to look for primary and secondary association signals at known and putative SNPs. See <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Note</a> for full details and for additional pre-imputation SNP QC.</p><p id="P31">We used standard “fixed-effect” meta-analysis techniques to analyse the replication data. Specifically, in each population, we fitted a logistic regression model for case/control status with no covariates at each SNP with a single parameter for the genetic effect (a multiplicative effect on the risk scale, additive on the log-odds scale). We then assumed a fixed underlying genetic effect, common to all populations, for the SNP, which we estimated by combining the effect estimates across populations, weighting by the precision of the estimate within each population. P-values, effect size estimates and confidence individuals for each population, and combined across populations are shown in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>, see also <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 4</a>. As is typical for GWAS replication studies which type a small number of SNPs, testing for possible substructure within populations is not possible. Our replication results involve averaging across separate results for seven populations. This provides robustness against false positives due to substructure in some populations, and it is encouraging that most or all replication populations typically exhibit evidence for the signal. We confirmed the power of this replication cohort by genotyping SNPs rs4085613 and rs4112788 which tag the known LCE3B/3C psoriasis risk locus<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616264">6</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616244">9</a></sup>, where we observed replication P values of 2.33 × 10<sup>−15</sup> and 4.44 × 10<sup>−16</sup>, respectively.</p><p id="P32" class="p p-last">To look for interactions between associated loci we considered all pairs of index SNPs from <a href="/pmc/articles/PMC3749730/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and <a href="/pmc/articles/PMC3749730/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2</span></span><span>2</span></a> and for each pair we compared two logistic regression models. In the first (null) model, a separate parameter for each SNP specifies the multiplicative increase in the odds of disease with each additional copy of the risk allele for that SNP. We compared this, via a likelihood ratio test, to a model for interaction in which there is a third parameter which modifies the effect associated with carrying risk alleles at both SNPs. There are many different ways to model both the marginal effects at each SNP and their joint effect<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616258">35</a></sup>, and so many approaches to search for interactions. The procedure we adopted is perhaps the simplest (multiplicative model marginally at each SNP and a single additional parameter for multiplicative interactions). Having identified the interaction between the <em>HLA-C</em> and <em>ERAP1</em> loci, we examined the nature of the effect, and observed that it takes the simple, and easily interpretable, form in which additional copies of the index SNP at <em>ERAP1</em> only affect disease risk in <em>HLA-C</em> positive individuals. To test for interactions between SNPs in the replication data we included a categorical covariate indicating population membership to the logistic regression model. We used this framework to compare two models, via a likelihood ratio test, which included or excluded a term for the interaction between the two SNPs.</p></div><div id="S9" class="sec"><h3 id="S9title">HLA Imputation</h3><p id="P33" class="p p-first-last">Imputation of classical HLA genotypes was performed using published statistical methodology<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616247">29</a></sup> and a reference panel of over 1,500 samples with classical HLA and SNP genotype data from the 1958 Birth Cohort and the International HapMap Project<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616272">36</a></sup>. When predicting whether an individual carries one or more copies of HLACW*0602, the primary risk allele for psoriasis, we estimate false positive rate to be 1% (no call threshold), and false negative rate &lt; 1%. In the replication study, four SNPs were typed in the HLA (rs10484554, rs3906272, rs12111032 and rs2243868), two of which (rs12111032 and rs2243868) form a near-perfect (false positive rate &lt; 1%) haplotype tag for CW*0602 (alleles A and G on the plus strand respectively). Imputations were performed by phasing the four SNPs with PHASE<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329616266">37</a></sup> and assuming the AG haplotype to be a perfect tag. From the independent validation study we estimate the false positive rate of this approach to be 9% (no call threshold), the decrease in accuracy resulting from uncertainty in phasing. As a check, we then applied the imputation approach we used in the replication data to the GWAS samples. Comparison of the imputations made using the four SNPs to those made from the full GWAS data show 1.4% of Cw*0602 presence/absence predictions to be discrepant.</p></div><div id="S10" class="sec sec-last"><h3 id="S10title">Data Availability</h3><p id="P34" class="p p-first-last">A list of the SNPs in the discovery scan exhibiting P &lt; 10<sup>−4</sup> are available in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>. Researchers can gain access to the data by applying to the data access committee (<a href="http://www.ebi.ac.uk/ega/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ebi.ac.uk/ega/</a>).</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemantary Table 9</h4><div class="sup-box half_rhythm" id="d36e1667"><a href="/pmc/articles/PMC3749730/bin/NIHMS48359-supplement-Supplemantary_Table_9.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(37K, xls)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supplementary Text and Figures</h4><div class="sup-box half_rhythm" id="d36e1671"><a href="/pmc/articles/PMC3749730/bin/NIHMS48359-supplement-Supplementary_Text_and_Figures.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(6.4M, pdf)</sup></div></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Acknowledgements</h2><div class="sec"><p id="__p1">The principal funding for this study was provided by the Wellcome Trust, as part of the Wellcome Trust Case Control Consortium 2 project (083948/Z/07/Z). We also thank S. Bertrand, J. Bryant, S.L. Clark, J.S. Conquer, T. Dibling, J.C. Eldred, S. Gamble, C. Hind, A. Wilk, C.R. Stribling and S. Taylor of the Wellcome Trust Sanger Institute’s Sample and Genotyping Facilities for technical assistance. We thank Dan Davison for making available his program “Shellfish” for calculating principal components in large genetic data sets. Case collections were supported by the Netherlands Organization for Health Research and Development (P.L.Z.); the Swedish Medical Research Council, Karolinska Institutet, Karolinska University Hospital, Psoriasis Foundation, AFA, and Welander Finsen Foundation (M.S.); the Association for the Defence of Psoriasis Patients (G.N.); Psoriasis Association and the Cecil King Memorial Foundation (M.J.C.); the Swedish Psoriasis Association (L.S.); the German Research Foundation (Tr 228/5-4 and Re 679/10-4) and The Interdisciplinary Centre for Clinical Research (IZKF B32/A8) of the University of Erlangen-Nuremberg (A.R.); the Spanish Ministry of Science and Innovation (grant SAF 2008-00357) and the “Generalitat de Catalunya” Departments of Health and Universities and Innovation (X.E.); the Genetic Repository in Ireland for Psoriasis and Psoriatic Arthritis (GRIPPsA), the Dublin Centre for Clinical Research (DCCR, funded by the Irish Health Research Board), The Wellcome Trust, and Science Foundation Ireland (R.McM.); National Institute for Health Research, Manchester Biomedical Research Centre (J.W., C.E.M.G., R.B.W., H.S.Y.); Arthritis Research UK (J.W., grant 17552). P.D. was supported in part by a Wolfson–Royal Society Merit Award and A.O. was supported by a PhD studentship from The Generation Trust. We also acknowledge support from the UK Medical Research Council (R.C.T., F.O.N., A.H., JNB grant G0601387), the Wellcome Trust (F.O.N., grant 078173/Z/05/Z) and the Department of Health via the National Institute for Health Research (NIHR) comprehensive Biomedical Research Centre awards to Guy’s &amp; St. Thomas’ NHS Foundation Trust in partnership with King’s College London (J.K, M.E.W, C.G.M, F.O.N, A.Hayday and J.N.B) and the NIHR award to Moorfields Eye Hospital NHS Foundation Trust and UCL Institute of Ophthalmology for a Specialist Biomedical Research Centre for Ophthalmology (A.C.V). We acknowledge use of the British 1958 Birth Cohort DNA collection, funded by the Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02, and of the UK National Blood Service controls funded by the Wellcome Trust. We thank W. Bodmer and B. Winney for use of the People of the British Isles DNA collection, which was funded by the Wellcome Trust.</p></div></div><div id="idm139850917475312" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139850917475312title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="P35" class="p p-first-last"><strong>URLs</strong>, SNPTEST, <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a>. WTCCC2, <a href="http://www.wtccc.org.uk/ccc2/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk/ccc2/</a>.</p><p></p><p class="fn sec" id="FN5"></p><p id="P36" class="p p-first-last"><strong>Competing financial interests</strong> The authors declare no competing financial interests</p><p></p></div></div><div id="idm139850917470912" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139850917470912title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Nestle FO, Kaplan DH, Barker J. Psoriasis. <span><span class="ref-journal">N Engl J Med. </span>2009;<span class="ref-vol">361</span>:496–509.</span> [<a href="/pubmed/19641206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Psoriasis&amp;author=FO+Nestle&amp;author=DH+Kaplan&amp;author=J+Barker&amp;volume=361&amp;publication_year=2009&amp;pages=496-509&amp;pmid=19641206&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Veal CD, et al.  Family-based analysis using a dense single-nucleotide polymorphism-based map defines genetic variation at PSORS1, the major psoriasis-susceptibility locus. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">71</span>:554–64.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379192/">PMC free article</a>]</span> [<a href="/pubmed/12148091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Family-based+analysis+using+a+dense+single-nucleotide+polymorphism-based+map+defines+genetic+variation+at+PSORS1,+the+major+psoriasis-susceptibility+locus&amp;author=CD+Veal&amp;volume=71&amp;publication_year=2002&amp;pages=554-64&amp;pmid=12148091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Nair RP, et al.  Sequence and haplotype analysis supports HLA-C as the psoriasis susceptibility 1 gene. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">78</span>:827–51.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1474031/">PMC free article</a>]</span> [<a href="/pubmed/16642438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Sequence+and+haplotype+analysis+supports+HLA-C+as+the+psoriasis+susceptibility+1+gene&amp;author=RP+Nair&amp;volume=78&amp;publication_year=2006&amp;pages=827-51&amp;pmid=16642438&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Capon F, et al.  Identification of ZNF313/RNF114 as a novel psoriasis susceptibility gene. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:1938–45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2900900/">PMC free article</a>]</span> [<a href="/pubmed/18364390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Identification+of+ZNF313/RNF114+as+a+novel+psoriasis+susceptibility+gene&amp;author=F+Capon&amp;volume=17&amp;publication_year=2008&amp;pages=1938-45&amp;pmid=18364390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Cargill M, et al.  A large-scale genetic association study confirms IL12B and leads to the identification of IL23R as psoriasis-risk genes. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:273–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785338/">PMC free article</a>]</span> [<a href="/pubmed/17236132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+large-scale+genetic+association+study+confirms+IL12B+and+leads+to+the+identification+of+IL23R+as+psoriasis-risk+genes&amp;author=M+Cargill&amp;volume=80&amp;publication_year=2007&amp;pages=273-90&amp;pmid=17236132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">de Cid R, et al.  Deletion of the late cornified envelope LCE3B and LCE3C genes as a susceptibility factor for psoriasis. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:211–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128734/">PMC free article</a>]</span> [<a href="/pubmed/19169253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Deletion+of+the+late+cornified+envelope+LCE3B+and+LCE3C+genes+as+a+susceptibility+factor+for+psoriasis&amp;author=R+de+Cid&amp;volume=41&amp;publication_year=2009&amp;pages=211-5&amp;pmid=19169253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Hollox EJ, et al.  Psoriasis is associated with increased beta-defensin genomic copy number. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:23–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2447885/">PMC free article</a>]</span> [<a href="/pubmed/18059266" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Psoriasis+is+associated+with+increased+beta-defensin+genomic+copy+number&amp;author=EJ+Hollox&amp;volume=40&amp;publication_year=2008&amp;pages=23-5&amp;pmid=18059266&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Nair RP, et al.  Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:199–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-kappaB+pathways&amp;author=RP+Nair&amp;volume=41&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Zhang XJ, et al.  Psoriasis genome-wide association study identifies susceptibility variants within LCE gene cluster at 1q21. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:205–10.</span> [<a href="/pubmed/19169255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Psoriasis+genome-wide+association+study+identifies+susceptibility+variants+within+LCE+gene+cluster+at+1q21&amp;author=XJ+Zhang&amp;volume=41&amp;publication_year=2009&amp;pages=205-10&amp;pmid=19169255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Jenisch S, et al.  Linkage analysis of human leukocyte antigen (HLA) markers in familial psoriasis: strong disequilibrium effects provide evidence for a major determinant in the HLA-B/-C region. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">63</span>:191–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377223/">PMC free article</a>]</span> [<a href="/pubmed/9634500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Linkage+analysis+of+human+leukocyte+antigen+(HLA)+markers+in+familial+psoriasis:+strong+disequilibrium+effects+provide+evidence+for+a+major+determinant+in+the+HLA-B/-C+region&amp;author=S+Jenisch&amp;volume=63&amp;publication_year=1998&amp;pages=191-9&amp;pmid=9634500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">York IA, et al.  The ER aminopeptidase ERAP1 enhances or limits antigen presentation by trimming epitopes to 8-9 residues. <span><span class="ref-journal">Nat Immunol. </span>2002;<span class="ref-vol">3</span>:1177–84.</span> [<a href="/pubmed/12436110" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=The+ER+aminopeptidase+ERAP1+enhances+or+limits+antigen+presentation+by+trimming+epitopes+to+8-9+residues&amp;author=IA+York&amp;volume=3&amp;publication_year=2002&amp;pages=1177-84&amp;pmid=12436110&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Burton PR, et al.  Association scan of 14,500 nonsynonymous SNPs in four diseases identifies autoimmunity variants. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1329–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2680141/">PMC free article</a>]</span> [<a href="/pubmed/17952073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+scan+of+14,500+nonsynonymous+SNPs+in+four+diseases+identifies+autoimmunity+variants&amp;author=PR+Burton&amp;volume=39&amp;publication_year=2007&amp;pages=1329-37&amp;pmid=17952073&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Khan MA. Update on spondyloarthropathies. <span><span class="ref-journal">Ann Intern Med. </span>2002;<span class="ref-vol">136</span>:896–907.</span> [<a href="/pubmed/12069564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Intern+Med&amp;title=Update+on+spondyloarthropathies&amp;author=MA+Khan&amp;volume=136&amp;publication_year=2002&amp;pages=896-907&amp;pmid=12069564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Qian Y, et al.  The adaptor Act1 is required for interleukin 17-dependent signaling associated with autoimmune and inflammatory disease. <span><span class="ref-journal">Nat Immunol. </span>2007;<span class="ref-vol">8</span>:247–56.</span> [<a href="/pubmed/17277779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=The+adaptor+Act1+is+required+for+interleukin+17-dependent+signaling+associated+with+autoimmune+and+inflammatory+disease&amp;author=Y+Qian&amp;volume=8&amp;publication_year=2007&amp;pages=247-56&amp;pmid=17277779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Hayden MS, Ghosh S. Shared principles in NF-kappaB signaling. <span><span class="ref-journal">Cell. </span>2008;<span class="ref-vol">132</span>:344–62.</span> [<a href="/pubmed/18267068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Shared+principles+in+NF-kappaB+signaling&amp;author=MS+Hayden&amp;author=S+Ghosh&amp;volume=132&amp;publication_year=2008&amp;pages=344-62&amp;pmid=18267068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Gregersen PK, et al.  REL, encoding a member of the NF-kappaB family of transcription factors, is a newly defined risk locus for rheumatoid arthritis. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:820–3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2705058/">PMC free article</a>]</span> [<a href="/pubmed/19503088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=REL,+encoding+a+member+of+the+NF-kappaB+family+of+transcription+factors,+is+a+newly+defined+risk+locus+for+rheumatoid+arthritis&amp;author=PK+Gregersen&amp;volume=41&amp;publication_year=2009&amp;pages=820-3&amp;pmid=19503088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Oyamada A, et al.  Tyrosine kinase 2 plays critical roles in the pathogenic CD4 T cell responses for the development of experimental autoimmune encephalomyelitis. <span><span class="ref-journal">J Immunol. </span>2009;<span class="ref-vol">183</span>:7539–46.</span> [<a href="/pubmed/19917699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Tyrosine+kinase+2+plays+critical+roles+in+the+pathogenic+CD4+T+cell+responses+for+the+development+of+experimental+autoimmune+encephalomyelitis&amp;author=A+Oyamada&amp;volume=183&amp;publication_year=2009&amp;pages=7539-46&amp;pmid=19917699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Wallace C, et al.  The imprinted DLK1-MEG3 gene region on chromosome 14q32.2 alters susceptibility to type 1 diabetes. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">42</span>:68–71.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820243/">PMC free article</a>]</span> [<a href="/pubmed/19966805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+imprinted+DLK1-MEG3+gene+region+on+chromosome+14q32.2+alters+susceptibility+to+type+1+diabetes&amp;author=C+Wallace&amp;volume=42&amp;publication_year=2009&amp;pages=68-71&amp;pmid=19966805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Zhou Z, et al.  Type III interferon (IFN) induces a type I IFN-like response in a restricted subset of cells through signaling pathways involving both the Jak-STAT pathway and the mitogen-activated protein kinases. <span><span class="ref-journal">J Virol. </span>2007;<span class="ref-vol">81</span>:7749–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1933366/">PMC free article</a>]</span> [<a href="/pubmed/17507495" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Virol&amp;title=Type+III+interferon+(IFN)+induces+a+type+I+IFN-like+response+in+a+restricted+subset+of+cells+through+signaling+pathways+involving+both+the+Jak-STAT+pathway+and+the+mitogen-activated+protein+kinases&amp;author=Z+Zhou&amp;volume=81&amp;publication_year=2007&amp;pages=7749-58&amp;pmid=17507495&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Kotenko SV, et al.  IFN-lambdas mediate antiviral protection through a distinct class II cytokine receptor complex. <span><span class="ref-journal">Nat Immunol. </span>2003;<span class="ref-vol">4</span>:69–77.</span> [<a href="/pubmed/12483210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=IFN-lambdas+mediate+antiviral+protection+through+a+distinct+class+II+cytokine+receptor+complex&amp;author=SV+Kotenko&amp;volume=4&amp;publication_year=2003&amp;pages=69-77&amp;pmid=12483210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Wilkins C, Gale M., Jr. Recognition of viruses by cytoplasmic sensors. <span><span class="ref-journal">Curr Opin Immunol. </span><span class="ref-vol">22</span>:41–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3172156/">PMC free article</a>]</span> [<a href="/pubmed/20061127" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Immunol&amp;title=Recognition+of+viruses+by+cytoplasmic+sensors&amp;author=C+Wilkins&amp;author=M+Gale&amp;volume=22&amp;pages=41-7&amp;pmid=20061127&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Nejentsev S, Walker N, Riches D, Egholm M, Todd JA. Rare variants of IFIH1, a gene implicated in antiviral responses, protect against type 1 diabetes. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">324</span>:387–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2707798/">PMC free article</a>]</span> [<a href="/pubmed/19264985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Rare+variants+of+IFIH1,+a+gene+implicated+in+antiviral+responses,+protect+against+type+1+diabetes&amp;author=S+Nejentsev&amp;author=N+Walker&amp;author=D+Riches&amp;author=M+Egholm&amp;author=JA+Todd&amp;volume=324&amp;publication_year=2009&amp;pages=387-9&amp;pmid=19264985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Lincoln MR, et al.  Epistasis among HLA-DRB1, HLA-DQA1, and HLA-DQB1 loci determines multiple sclerosis susceptibility. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009;<span class="ref-vol">106</span>:7542–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2678609/">PMC free article</a>]</span> [<a href="/pubmed/19380721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Epistasis+among+HLA-DRB1,+HLA-DQA1,+and+HLA-DQB1+loci+determines+multiple+sclerosis+susceptibility&amp;author=MR+Lincoln&amp;volume=106&amp;publication_year=2009&amp;pages=7542-7&amp;pmid=19380721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Orsmark-Pietras C, et al.  Biological and genetic interaction between tenascin C and neuropeptide S receptor 1 in allergic diseases. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:1673–82.</span> [<a href="/pubmed/18305139" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Biological+and+genetic+interaction+between+tenascin+C+and+neuropeptide+S+receptor+1+in+allergic+diseases&amp;author=C+Orsmark-Pietras&amp;volume=17&amp;publication_year=2008&amp;pages=1673-82&amp;pmid=18305139&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Smyth DJ, et al.  PTPN22 Trp620 explains the association of chromosome 1p13 with type 1 diabetes and shows a statistical interaction with HLA class II genotypes. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:1730–7.</span> [<a href="/pubmed/18305142" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=PTPN22+Trp620+explains+the+association+of+chromosome+1p13+with+type+1+diabetes+and+shows+a+statistical+interaction+with+HLA+class+II+genotypes&amp;author=DJ+Smyth&amp;volume=57&amp;publication_year=2008&amp;pages=1730-7&amp;pmid=18305142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">The Wellcome Trust Case-Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-78&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Picard C, et al.  Hypomorphic mutation of ZAP70 in human results in a late onset immunodeficiency and no autoimmunity. <span><span class="ref-journal">Eur J Immunol. </span>2009;<span class="ref-vol">39</span>:1966–76.</span> [<a href="/pubmed/19548248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Immunol&amp;title=Hypomorphic+mutation+of+ZAP70+in+human+results+in+a+late+onset+immunodeficiency+and+no+autoimmunity&amp;author=C+Picard&amp;volume=39&amp;publication_year=2009&amp;pages=1966-76&amp;pmid=19548248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Leslie S, Donnelly P, McVean G. A statistical method for predicting classical HLA alleles from SNP data. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">82</span>:48–56.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253983/">PMC free article</a>]</span> [<a href="/pubmed/18179884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+statistical+method+for+predicting+classical+HLA+alleles+from+SNP+data&amp;author=S+Leslie&amp;author=P+Donnelly&amp;author=G+McVean&amp;volume=82&amp;publication_year=2008&amp;pages=48-56&amp;pmid=18179884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Barrett JC, et al.  Genome-wide association study of ulcerative colitis identifies three new susceptibility loci, including the HNF4A region. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1330–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2812019/">PMC free article</a>]</span> [<a href="/pubmed/19915572" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+ulcerative+colitis+identifies+three+new+susceptibility+loci,+including+the+HNF4A+region&amp;author=JC+Barrett&amp;volume=41&amp;publication_year=2009&amp;pages=1330-4&amp;pmid=19915572&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">R Development Core Team  . <span class="ref-journal">R: A Language and Environment for Statistical Computing.</span> R Foundation for Statistical Computing; 2010.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+A+Language+and+Environment+for+Statistical+Computing&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Schouten EG, et al.  Risk ratio and rate ratio estimation in case-cohort designs: Hypertension and cardiovascular mortality. <span><span class="ref-journal">Statistics in Medicine. </span>1993;<span class="ref-vol">12</span>:1733–1745.</span> [<a href="/pubmed/8248665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Statistics+in+Medicine&amp;title=Risk+ratio+and+rate+ratio+estimation+in+case-cohort+designs:+Hypertension+and+cardiovascular+mortality&amp;author=EG+Schouten&amp;volume=12&amp;publication_year=1993&amp;pages=1733-1745&amp;pmid=8248665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Su Z, Cardin N, The Wellcome Trust Case Control Consortium. Donnelly P, Marchini J. A Bayesian Method for Detecting and Characterizing Allelic Heterogeneity and Boosting Signals in Genome-Wide Association Studies. <span><span class="ref-journal">Statistical Science. </span>2009;<span class="ref-vol">24</span>:430–450.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Statistical+Science&amp;title=A+Bayesian+Method+for+Detecting+and+Characterizing+Allelic+Heterogeneity+and+Boosting+Signals+in+Genome-Wide+Association+Studies&amp;author=Z+Su&amp;author=N+Cardin&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=24&amp;publication_year=2009&amp;pages=430-450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Marchini J, Donnelly P, Cardon LR. Genome-wide strategies for detecting multiple loci that influence complex diseases. <span><span class="ref-journal">Nat Genet. </span>2005;<span class="ref-vol">37</span>:413–7.</span> [<a href="/pubmed/15793588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+strategies+for+detecting+multiple+loci+that+influence+complex+diseases&amp;author=J+Marchini&amp;author=P+Donnelly&amp;author=LR+Cardon&amp;volume=37&amp;publication_year=2005&amp;pages=413-7&amp;pmid=15793588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">de Bakker PI, et al.  A high-resolution HLA and SNP haplotype map for disease association studies in the extended human MHC. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:1166–72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2670196/">PMC free article</a>]</span> [<a href="/pubmed/16998491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+high-resolution+HLA+and+SNP+haplotype+map+for+disease+association+studies+in+the+extended+human+MHC&amp;author=PI+de+Bakker&amp;volume=38&amp;publication_year=2006&amp;pages=1166-72&amp;pmid=16998491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">68</span>:978–89.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1275651/">PMC free article</a>]</span> [<a href="/pubmed/11254454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+new+statistical+method+for+haplotype+reconstruction+from+population+data&amp;author=M+Stephens&amp;author=NJ+Smith&amp;author=P+Donnelly&amp;volume=68&amp;publication_year=2001&amp;pages=978-89&amp;pmid=11254454&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3749730&amp;issue-id=226605&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#S1title">ONLINE METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S11title">Acknowledgements</a></li><li><a href="#idm139850917475312title">Footnotes</a></li><li><a href="#idm139850917470912title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3749730/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3749730/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3749730/pdf/emss-48359.pdf">PDF (1.1M)</a></li> | <li><a href="#" data-citationid="PMC3749730" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3749730%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3749730%2F&amp;text=Genome-wide%20association%20study%20identifies%20new%20psoriasis%20susceptibility%20loci%20and%20an%20interaction%20between%20HLA-C%20and%20ERAP1" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3749730%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20953190" data-aiid="3749730" data-aid="3749730" data-iid="226605" data-domainid="358" data-domain="wtpa" data-accid="PMC3749730" data-md5="ab329302b4ca6965d7451c0181fd1f99"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:05:46-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89A7B1DD51254100000000075A04D0&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3749730%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3749730/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>