<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3060316/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Association between ORMDL3, IL1RL1 and a deletion on chromosome 17q21 with asthma risk in Australia</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3060316/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="Association between ORMDL3, IL1RL1 and a deletion on chromosome 17q21 with asthma risk in Australia"><meta name="citation_authors" content="Manuel A R Ferreira, Allan F McRae, Sarah E Medland, Dale R Nyholt, Scott D Gordon, Margaret J Wright, Anjali K Henders, Pamela A Madden, Peter M Visscher, Naomi R Wray, Andrew C Heath, Grant W Montgomery, David L Duffy, Nicholas G Martin"><meta name="citation_date" content="April 2011"><meta name="citation_issue" content="4"><meta name="citation_volume" content="19"><meta name="citation_firstpage" content="458"><meta name="citation_doi" content="10.1038/ejhg.2010.191"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3060316/?report=abstract"><meta name="citation_pmid" content="21150878"><meta name="DC.Title" content="Association between ORMDL3, IL1RL1 and a deletion on chromosome 17q21 with asthma risk in Australia"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Manuel A R Ferreira"><meta name="DC.Contributor" content="Allan F McRae"><meta name="DC.Contributor" content="Sarah E Medland"><meta name="DC.Contributor" content="Dale R Nyholt"><meta name="DC.Contributor" content="Scott D Gordon"><meta name="DC.Contributor" content="Margaret J Wright"><meta name="DC.Contributor" content="Anjali K Henders"><meta name="DC.Contributor" content="Pamela A Madden"><meta name="DC.Contributor" content="Peter M Visscher"><meta name="DC.Contributor" content="Naomi R Wray"><meta name="DC.Contributor" content="Andrew C Heath"><meta name="DC.Contributor" content="Grant W Montgomery"><meta name="DC.Contributor" content="David L Duffy"><meta name="DC.Contributor" content="Nicholas G Martin"><meta name="DC.Date" content="2011 Apr"><meta name="DC.Identifier" content="10.1038/ejhg.2010.191"><meta name="DC.Language" content="en"><meta property="og:title" content="Association between ORMDL3, IL1RL1 and a deletion on chromosome 17q21 with asthma risk in Australia"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies followed by replication provide a powerful approach to map genetic risk factors for asthma. We sought to search for new variants associated with asthma and attempt to replicate the association with four loci reported previously ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3060316/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3060316/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3060316/pdf/ejhg2010191a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CA0D3DD51375100000000097A062A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3060316%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/195174/">v.19(4); 2011 Apr</a></li><li class="accid">PMC3060316</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2011 Apr; 19(4): 458–464. </span></div><div><span class="fm-vol-iss-date">Published online 2010 Dec 8. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2010.191" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2010.191</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3060316</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21150878">21150878</a></div></div></div></div><h1 class="content-title">Association between <em>ORMDL3</em>, <em>IL1RL1</em> and a deletion on chromosome 17q21 with asthma risk in Australia</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ferreira%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563763088496" co-class="co-affbox">Manuel A R Ferreira</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=McRae%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563757450528" co-class="co-affbox">Allan F McRae</a>,<sup>1</sup> <a href="/pubmed/?term=Medland%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563757464192" co-class="co-affbox">Sarah E Medland</a>,<sup>1</sup> <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563755694848" co-class="co-affbox">Dale R Nyholt</a>,<sup>1</sup> <a href="/pubmed/?term=Gordon%20SD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563753062832" co-class="co-affbox">Scott D Gordon</a>,<sup>1</sup> <a href="/pubmed/?term=Wright%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563760566160" co-class="co-affbox">Margaret J Wright</a>,<sup>1</sup> <a href="/pubmed/?term=Henders%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563760144128" co-class="co-affbox">Anjali K Henders</a>,<sup>1</sup> <a href="/pubmed/?term=Madden%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563755060896" co-class="co-affbox">Pamela A Madden</a>,<sup>2</sup> <a href="/pubmed/?term=Visscher%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563754225888" co-class="co-affbox">Peter M Visscher</a>,<sup>1</sup> <a href="/pubmed/?term=Wray%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563759834496" co-class="co-affbox">Naomi R Wray</a>,<sup>1</sup> <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563751752336" co-class="co-affbox">Andrew C Heath</a>,<sup>2</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563756943936" co-class="co-affbox">Grant W Montgomery</a>,<sup>1</sup> <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563748985392" co-class="co-affbox">David L Duffy</a>,<sup>1</sup> and  <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878" class="affpopup" co-rid="_co_idm140563752448912" co-class="co-affbox">Nicholas G Martin</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140563763088496"><h3 class="no_margin">Manuel A R Ferreira</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Ferreira%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Manuel A R Ferreira</a></div></div><div id="_co_idm140563757450528"><h3 class="no_margin">Allan F McRae</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=McRae%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Allan F McRae</a></div></div><div id="_co_idm140563757464192"><h3 class="no_margin">Sarah E Medland</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Medland%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Sarah E Medland</a></div></div><div id="_co_idm140563755694848"><h3 class="no_margin">Dale R Nyholt</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Dale R Nyholt</a></div></div><div id="_co_idm140563753062832"><h3 class="no_margin">Scott D Gordon</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Gordon%20SD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Scott D Gordon</a></div></div><div id="_co_idm140563760566160"><h3 class="no_margin">Margaret J Wright</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Wright%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Margaret J Wright</a></div></div><div id="_co_idm140563760144128"><h3 class="no_margin">Anjali K Henders</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Henders%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Anjali K Henders</a></div></div><div id="_co_idm140563755060896"><h3 class="no_margin">Pamela A Madden</h3><p><sup>2</sup>Washington University School of Medicine, St Louis, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Madden%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Pamela A Madden</a></div></div><div id="_co_idm140563754225888"><h3 class="no_margin">Peter M Visscher</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Visscher%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Peter M Visscher</a></div></div><div id="_co_idm140563759834496"><h3 class="no_margin">Naomi R Wray</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Wray%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Naomi R Wray</a></div></div><div id="_co_idm140563751752336"><h3 class="no_margin">Andrew C Heath</h3><p><sup>2</sup>Washington University School of Medicine, St Louis, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Andrew C Heath</a></div></div><div id="_co_idm140563756943936"><h3 class="no_margin">Grant W Montgomery</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Grant W Montgomery</a></div></div><div id="_co_idm140563748985392"><h3 class="no_margin">David L Duffy</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">David L Duffy</a></div></div><div id="_co_idm140563752448912"><h3 class="no_margin">Nicholas G Martin</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21150878">Nicholas G Martin</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140563835687712_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140563835687712_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140563835687712_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140563835687712_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Washington University School of Medicine, St Louis, MO, USA</div><div id="caf1"><sup>*</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, P.O. Royal Brisbane Hospital, Brisbane, QLD 4029, Australia. Tel: +61 7 3845 3552; Fax: +61 7 3362 0101; E-mail: <a href="mailto:dev@null" data-email="ua.ude.rmiq@arierref.leunam" class="oemail">ua.ude.rmiq@arierref.leunam</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140563835687712_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Aug 10; Revised 2010 Sep 28; Accepted 2010 Oct 5.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140563835687712_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2011 Macmillan Publishers Limited</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div><strong>This article has been corrected. </strong>See <a href="/pmc/articles/PMC3190252/">Eur J Hum Genet. 2011 October; 19(10): 1109</a>.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC3060316/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC3060316/bin/ejhg2010191x1.doc" data-ga-action="click_feat_suppl">ejhg2010191x1.doc</a><span style="color:gray"> (171K)</span></div><div class="small guid">GUID:&nbsp;77A84A2B-E712-4AFA-B742-31D30BA586B0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table.</strong></div><div><a href="/pmc/articles/PMC3060316/bin/ejhg2010191x2.doc" data-ga-action="click_feat_suppl">ejhg2010191x2.doc</a><span style="color:gray"> (194K)</span></div><div class="small guid">GUID:&nbsp;D59E0AA3-8D5F-4D8B-996F-69D3A3F3B433</div></div></dd></dl></div><div id="idm140563756007920" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563756007920title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Genome-wide association studies followed by replication provide a powerful approach to map genetic risk factors for asthma. We sought to search for new variants associated with asthma and attempt to replicate the association with four loci reported previously (<em>ORMDL3</em>, <em>PDE4D</em>, <em>DENND1B</em> and <em>IL1RL1)</em>. Genome-wide association analyses of individual single nucleotide polymorphisms (SNPs), rare copy number variants (CNVs) and overall CNV burden were carried out in 986 asthma cases and 1846 asthma-free controls from Australia. The most-associated locus in the SNP analysis was <em>ORMDL3</em> (rs6503525, <em>P</em>=4.8 × 10<sup>−7</sup>). Five other loci were associated with <em>P</em>&lt;10<sup>−5</sup>, most notably the chemokine CXC motif ligand 14 (<em>CXCL14</em>) gene (rs31263, <em>P</em>=7.8 × 10<sup>−6</sup>). We found no evidence for association with the specific risk variants reported recently for <em>PDE4D</em>, <em>DENND1B</em> and <em>ILR1L1</em>. However, a variant in <em>IL1RL1</em> that is in low linkage disequilibrium with that reported previously was associated with asthma risk after accounting for all variants tested (rs10197862, gene wide <em>P</em>=0.01). This association replicated convincingly in an independent cohort (<em>P</em>=2.4 × 10<sup>−4</sup>). A 300-kb deletion on chromosome 17q21 was associated with asthma risk, but this did not reach experiment-wide significance. Asthma cases and controls had comparable CNV rates, length and number of genes affected by deletions or duplications. In conclusion, we confirm the association between asthma risk and variants in <em>ORMDL3</em> and identify a novel risk variant in <em>IL1RL1.</em> Follow-up of the 17q21 deletion in larger cohorts is warranted.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">whole-genome, gene, atopy, heterogeneity, structural, <em>IKZF3</em></span></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Asthma currently affects 10% of the Australian population and is responsible for 1 million work days lost, 36 000 hospital admissions, 402 deaths and $606 million a year in direct costs.<sup><a href="#bib1" rid="bib1" class=" bibr popnode">1</a></sup> A complete understanding of the genetic and environmental risk factors for asthma is critical to design new treatments or prevention strategies that can reduce the disease burden. This is a challenging task, as asthma is a complex disease that involves the interplay between multiple physiological processes.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600152">2</a></sup></p><p id="__p3">Genome-wide association studies (GWAS) currently provide the most powerful approach to identify genetic risk factors for complex diseases. In the past few years, hundreds of variants with replicated associations have been identified for a range of complex diseases or traits using this approach.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600179">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304427998">4</a></sup> As hoped, several of these discoveries have identified aetiologic pathways not previously implicated in these traits, such as the autophagy pathway in Crohn's disease<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600159">5</a></sup> and the HLA-C locus in control of viral load in HIV infection.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600165">6</a></sup></p><p id="__p4">Seven GWAS of asthma have been published recently.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600144">7</a>, <a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600131">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600173">9</a>, <a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600171">10</a>, <a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600182">11</a>, <a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304427999">12</a>, <a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600153">13</a></sup> From these, three loci have emerged with reproducible effects on asthma risk in independent samples: the <em>GSDMB</em>/<em>ORMDL3</em> locus (henceforth referred to as <em>ORMDL3</em>) on chromosome 17q12-21;<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600156">7</a></sup> the <em>PDE4D</em> gene on chromosome 5q12<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600141">9</a></sup> and the <em>DENND1B</em> on chromosome 1q31.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600169">11</a></sup> Three GWAS of asthma biomarkers were also published recently.<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600137">14</a>, <a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600146">15</a>, <a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600151">16</a></sup> Quantitative trait loci were identified for the respective phenotype analysed in each study, but for only one of these there was also a convincing association with asthma, the <em>IL1RL1</em> gene on chromosome 2q12.<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600149">16</a></sup> A variant at this locus was a significant predictor of eosinophil levels while also reproducibly increasing asthma risk.</p><p id="__p5">Although single nucleotide polymorphisms (SNPs) are now routinely tested for association in GWAS for many common diseases, structural variants, a widespread class of genetic polymorphisms that range in size from only a few base pairs to whole chromosomal rearrangements,<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600167">17</a></sup> remain much less well studied. The Welcome Trust Case Control Consortium recently performed a GWAS of common copy number variants (CNVs) for eight common diseases, but failed to identify any new risk loci that had not been previously reported through the analysis of SNPs.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600135">18</a></sup> These results suggest that common CNVs are unlikely to contribute greatly to the genetic basis of common diseases, but leave open the possibility that rarer structural variants may have a greater impact on risk, as shown for obesity,<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_229344878">19</a></sup> autism<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600148">20</a></sup> and schizophrenia.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_378345784">21</a></sup> The only published genome-wide association analysis between CNV data and asthma identified a region on chromosome 7p14 that appeared to be associated with asthma risk, although closer inspection of the region could not rule out the possibility that this was a false-positive association as a result of a biological artefact.<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600160">22</a></sup></p><p id="__p6" class="p p-last">Therefore, the identified variants in <em>ORMDL3</em>, <em>PDE4D</em>, <em>DENND1B</em> and <em>IL1RL1</em> arguably represent the most convincing associations between genetic variants and asthma risk reported to date. In this study, we analysed whole-genome genotype data from 986 asthma cases and 1846 disease-free controls of European descent from the Australian population to (1) search for new sequence or structural variants with strong effects on asthma and (2) confirm the previously reported associations with <em>ORMDL3</em>, <em>PDE4D</em>, <em>DENND1B</em> and <em>IL1RL1</em>.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">MATERIALS AND METHODS</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Case and control samples</h3><p id="__p7" class="p p-first-last">The samples analysed herein are a subset of a larger cohort of 16 140 individuals ascertained in several waves from the Australian general population and genotyped recently as described in detail elsewhere.<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600150">23</a></sup> Of these, we selected for analysis 986 unrelated cases and 1846 controls based on the presence (cases) or absence (controls) of self-reported lifetime asthma, as reported in at least one lifestyle or health questionnaire completed as part of six epidemiological studies conducted at the Queensland Institute of Medical Research (QIMR) (Supplementary Table 1). Using SOLAR,<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600174">24</a></sup> we estimate that this phenotype is 61% heritable, based on a polygenic model fitted to 2054 individuals for whom phenotype information was also available for 3154 of their relatives (parents and/or siblings).</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Whole-genome genotyping and imputation of HapMap SNPs</h3><p id="__p8" class="p p-first-last">Genotype data were obtained with Illumina 317K, 370K or 610K arrays and stringent quality control (QC) procedures applied to ensure high data quality as described in detail previously<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600181">23</a></sup> and summarised in Supplementary Table 2. After QC, genotype data were available for 16 140 individuals and 561 815 SNPs. To increase whole-genome coverage, we inferred unmeasured HapMap SNPs using MACH (<a href="http://www.sph.umich.edu/csg/abecasis/MACH/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MACH/</a>) and the phased haplotype data from the CEU HapMap samples (phase I+II, release22, build 36). Following imputation, we restricted our analysis to 2.38 million SNPs that were genotyped or imputed with high confidence (<em>r</em><sup>2</sup>&gt;0.3) and had an MAF&gt;0.01, Hardy–Weinberg equilibrium test <em>P</em>-value&gt;10<sup>−6</sup> in controls and &lt;5% missing data in the selected sample of 2832 individuals with asthma information. Individuals were confirmed as unrelated through pair wise whole-genome identity-by-descent analysis.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Population substructure and whole-genome SNP association analyses</h3><p id="__p9" class="p p-first-last">All analyses were performed with PLINK.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600175">25</a></sup> To address whether subtle population substructure could have an impact on the association results, we performed a multidimensional scaling (MDS) analysis of identity-by-state (IBS) distances calculated between all pairs of individuals using a subset of SNPs in linkage equilibrium. For reference, we included the 11 HapMap III populations.<sup><a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600172">26</a></sup> To test whether cases and controls were well matched with respect to ancestry, we applied a permutation test for between-group IBS differences. A significant test would imply that cases and controls are unlikely to belong to the same population, which could lead to spurious association results. The main association analysis was performed using a standard allelic test. Age-of-onset information was only available for 20% of cases, and so was not considered for analysis. Lifetime smoking status, which was available for 96% of individuals (38% ever smokers), was not a significant predictor of asthma in this sample (<em>P</em>=0.82) and so was not included as a covariate.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Whole-genome CNV analysis</h3><p id="__p10" class="p p-first">CNV segments were identified for a subset of 759 individuals that participated in the longitudinal Brisbane Adolescent Twin Study (270 asthma cases and 489 controls; Supplementary Table 1) using the program QuantiSNP v1.1<sup><a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600176">27</a></sup> based on the SNP and CNV probes present in the Illumina 610K arrays. The default program settings were used in addition to a maximum copy number of four and GC correction. To minimise false-positive CNV calling, we restricted our analysis to large (&gt;100 kb and &lt;1 Mb) deletions or duplications called with high confidence (log Bayes' factor &gt;10 and with &gt;10 probes). We also dropped common CNVs (MAF&gt;0.05), as these were likely well tagged by SNPs.</p><p id="__p11">We performed two sets of association analyses between CNVs and asthma risk. First, we investigated whether the frequency of deletions or duplications at specific loci was associated with asthma status. This is equivalent to the genome-wide SNP analysis, except that a small number of rare structural variants are tested for association instead of a large number of common SNPs. Significance was estimated through the analysis of 10 000 permutations.</p><p id="__p12" class="p p-last">Second, we tested the hypothesis that the presence of multiple deletions or duplications across an individual's genome (ie CNV burden) may significantly increase the risk of developing asthma. Specifically, we addressed the following questions: (A) do asthma cases have on average more CNVs than controls? In those individuals having at least one CNV, do cases have on average (B) larger CNVs than controls or (C) more genes affected by CNVs than controls? For (A) and (B), the test applied was a logistic regression of disease status on the number of observed CNVs and CNV length (mean or total), respectively, while accounting for potential Illumina batch and plate effects. To determine whether more genes were affected by CNVs in cases than controls (C), we downloaded the genomic coordinates for 18 212 genes from the Mar 2006 UCSC browser (based on the longest isoform ±50 kb) and determined for each individual how many genes in total were intersected by a CNV (gene count). The test was then a logistic regression of disease status on gene count, while accounting for an individual's number of CNVs, mean CNV length, batch and plate effects. Significance was estimated from 10 000 permutations.</p></div><div id="__sec7" class="sec sec-last"><h3 id="__sec7title">Follow-up of top SNP and CNV associations</h3><p id="__p13" class="p p-first-last">We attempted to replicate the top SNP and CNV associations in an independent cohort of 604 individuals, including 391 doctor-diagnosed asthmatics (Supplementary Table 3). These individuals were ascertained from the same QIMR studies that contributed data to the GWAS (Supplementary Table 2) and were recently typed with Illumina 610K arrays as part of an additional, as yet unpublished wave of genotyping conducted in the first semester of 2010. All samples are of European descent and passed the standard QC filters described above for the GWAS. Data from this wave were available at this stage only for replication of specific associations.</p></div></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">RESULTS</h2><div id="__sec9" class="sec sec-first"><h3 id="__sec9title">Population substructure analyses</h3><p id="__p14" class="p p-first-last">We first performed an MDS analysis of IBS distances to identify subtle ancestry differences between samples that could potentially have an impact on the association results. As expected, most of the 2832 Australian samples analysed clustered closely with the CEU HapMap reference panel, indicating that most individuals have a homogeneous European ancestry (Supplementary Figure 1A). However, two groups of individuals clustered close to, but just outside this larger group of homogeneous European ancestry: one group clustered along the European–African ancestry axis, while the second group clustered along the European–Asian axis. The first group likely represents samples with a southern European ancestry (eg Maltese Australians), as indicated by the partial overlap with the HapMap samples from Tuscany in Italy (TSI) (Supplementary Figure 1B). The second group consists of samples with a predominant European ancestry that is partially mixed with Asian ancestry, as reflected by the small overlap with the HapMap Mexican ancestry samples from Los Angeles (MEX). Importantly, however, this subtle population substructure was found to be independent of case–control disease status (<em>P</em>=0.775; Supplementary Figure 1C), suggesting that it would not significantly bias whole-genome association results. Consistent with this observation, the genomic inflation factor for the main association analysis was 1.006 (Supplementary Figure 1D), thus confirming that population substructure or other technical artefacts had a minimal impact on the results.</p></div><div id="__sec10" class="sec"><h3 id="__sec10title">Whole-genome SNP association results</h3><p id="__p15" class="p p-first">Given that the 986 asthma cases and 1846 asthma-free controls were well matched with respect to ancestry, we then applied a standard allelic test of association between individual SNPs and lifetime disease status to search for new susceptibility loci for asthma. We acknowledge that given the modest sample size, power was only adequate (80%) at the genome-wide level (<em>α</em>=5 × 10<sup>−8</sup>) to detect loci with strong effects (eg an OR &gt;1.4, for an allele frequency of 40%). The loci most associated with asthma risk (<em>P</em>&lt;10<sup>−5</sup>) in this analysis are listed in <a href="/pmc/articles/PMC3060316/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>, with the full genome-wide results displayed in Supplementary Figure 2. No variant exceeded the widely used threshold for genome-wide significance of 5 × 10<sup>−8</sup>, but the most-associated SNP was located in the <em>ORMDL3</em> region (rs6503525, OR=1.33 for the C allele, <em>P</em>=4.8 × 10<sup>−7</sup>), a locus previously reported to have strong and reproducible effects on doctor-diagnosed asthma.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600140">7</a></sup> Multiple variants in this region were associated with disease risk in our analysis (<a href="/pmc/articles/PMC3060316/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1a</span></a>). The rs6503525 SNP is in linkage disequilibrium (LD, <em>r</em><sup>2</sup>=0.61) with the top <em>ORMDL3</em> variant (rs7216389) reported in the original study by Moffat <em>et al</em>;<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600147">7</a></sup> both predisposing alleles (rs6503525:C and rs7216389:T) are in phase. We found no evidence for additional independent risk variants in this region (not shown), consistent with Moffatt <em>et al.</em><sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600170">7</a></sup> These results thus confirm that a genetic variant in the <em>ORMDL3</em> locus significantly influences asthma risk in the Australian population.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3060316/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563756337808" class="figure"><a href="/pmc/articles/PMC3060316/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3060316_ejhg2010191f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg2010191f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3060316/bin/ejhg2010191f1.jpg"></a></div><div id="largeobj_idm140563756337808" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3060316/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3060316/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p38">Association plots for the four loci previously reported to associate with asthma risk (<strong>a</strong>–<strong>d</strong>). The most-associated SNP for each region is shown in blue, and the colour of the remaining markers reflects the linkage disequilibrium (<em>r</em><sup>2</sup>) with the top SNP in each panel (increasing red hue associated with increasing <em>r</em><sup>2</sup>). The most-associated SNP reported in the original study for each gene is shown in green. Imputed SNPs are represented by circles and genotyped SNPs by diamonds. The recombination rate (second <em>y</em> axis) is plotted in light blue and is based on the CEU HapMap population. Exons for each gene are represented by vertical bars.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Regions most associated with asthma (<em>P</em>&lt;10<sup>−5</sup>) in the genome-wide SNP analysis</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563747646848" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th colspan="2" charoff="50" rowspan="1" valign="top" align="center"><em>Allele frequency</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Chromosome, bp position</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SNP, reference allele</em><sup><a href="#t1-fn1" rid="t1-fn1" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Closest gene, distance kb</em><sup><a href="#t1-fn2" rid="t1-fn2" class=" fn">b</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Cases</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Controls</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Odds ratio</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P<em>-value</em></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">17, 35348700</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6503525, C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>ORMDL3</em>, 11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.50</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.43</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.33</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.8 × 10<sup>−7</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">6, 103075599</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs12206223, C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>GRIK2</em>, 450</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.54</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.0 × 10<sup>−6</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">7, 38792860</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs17700752, G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>VPS41</em><sup><em>#</em></sup>, 62</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.20</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.71</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.4 × 10<sup>−6</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">22, 34352607</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4622835, C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>LOC284912</em>, 378</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.53</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.46</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.29</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.6 × 10<sup>−6</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">10, 130181728</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7893667, A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>MKI67</em>, 367</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.29</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.31</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.1 × 10<sup>−6</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">5, 134996861</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs31263, C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>CXCL14</em>, 53</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.28</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.31</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7.8 × 10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140563747646848" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3060316/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1-fn1"><sup>a</sup>The SNPs listed were imputed with high confidence using MACH. With the exception of rs7893667, nearby genotyped SNPs provided comparable evidence for association (illustrated in <a href="/pmc/articles/PMC3060316/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1a</span></a> for the 17q12 region).</div><div id="t1-fn2"><sup>b</sup>Distance to first base pair of the first exon or last base pair of the last exon, whichever is smallest. A pound (#) indicates that the SNP is located within the gene.</div></div></div><p id="__p16" class="p p-last">Among the five remaining regions most associated with asthma (<a href="/pmc/articles/PMC3060316/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>), only for chromosome 5q31 there was an obvious candidate gene under the association peak, the chemokine CXC motif ligand 14 (<em>CXCL14</em>) gene, which is thought to be a potent chemoattractant and activator of dendritic cells.<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600145">28</a></sup></p></div><div id="__sec11" class="sec"><h3 id="__sec11title">Analysis of loci reported in previous GWAS of asthma</h3><p id="__p17" class="p p-first">Despite low power to detect risk loci for asthma with genome-wide significance, our whole-genome data set nonetheless provided an opportunity to attempt to replicate previously identified loci, given the less stringent threshold required for significance. We, therefore, investigated whether there was any evidence for association with SNPs located in the three remaining loci that have previously been reported to have reproducible effects on asthma: <em>PDE4D</em>, <em>DENND1B</em> and <em>IL1RL1</em>. For completeness, we also report results from these analyses for <em>ORMDL3</em>.</p><p id="__p18">We first focused on the individual variant with strongest association reported in the original study for each locus. We found consistent evidence for association with the leading literature SNP for <em>ORMDL3</em> (rs7216389, OR=1.25, <em>P</em>=6 × 10<sup>−5</sup>), but not for the three remaining loci, despite reasonable power (<a href="/pmc/articles/PMC3060316/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2a</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>SNP association results for <em>ORMDL3</em>, <em>PDE4D</em>, <em>DENND1B</em> and <em>IL1RL1</em></strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563751887008" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th colspan="2" charoff="50" rowspan="1" valign="top" align="center"><em>Allele frequency</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Region</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Gene</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Literature SNP, allele</em><sup><a href="#t2-fn1" rid="t2-fn1" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Reported OR</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Power,</em> %<sup><a href="#t2-fn2" rid="t2-fn2" class=" fn">b</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Cases</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Controls</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>OR</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>95</em>% <em>CI OR</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P<em>-value</em></th></tr></thead><tbody valign="top"><tr><td colspan="10" charoff="50" rowspan="1" valign="top" align="left"><em>(a) Most-associated variant reported in the original studies</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 17q12-21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>ORMDL3</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7216389, T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.45</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">100</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.53</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.47</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.25</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.12–1.40</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.0 × 10<sup>−5</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 5q12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>PDE4D</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1588265, G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.85</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">78</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.31</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.30</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.05</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.93–1.18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.4334</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 1q31</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>DENND1B</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1775456, G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.75</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">93</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.20</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.89–1.17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.7862</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 2q12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>IL1RL1</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1420101, T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.16</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">77</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.39</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.04</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.93–1.17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.4649</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td colspan="2" charoff="50" rowspan="1" valign="top" align="center"><em>Allele frequency</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td colspan="2" charoff="50" rowspan="1" valign="top" align="left">P<em>-value</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Region</em><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Gene</em><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Best SNP, allele</em><sup><a href="#t2-fn1" rid="t2-fn1" class=" fn">a</a></sup><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>N SNPs tested</em><sup><a href="#t2-fn3" rid="t2-fn3" class=" fn">c</a></sup><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>LD with literature SNP,</em> r<sup><a href="#t2-fn2" rid="t2-fn2" class=" fn">b</a></sup><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Cases</em><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Controls</em><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>OR</em><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Uncorrected</em><hr></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Corrected</em><sup><a href="#t2-fn4" rid="t2-fn4" class=" fn">d</a></sup><hr></td></tr><tr><td colspan="10" charoff="50" rowspan="1" valign="top" align="left"><em>(b) Most-associated variant in this study</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 17q12-21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>ORMDL3</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6503525,C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">232</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.61</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.50</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.43</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.33</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.8 × 10<sup>−7</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0002</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 5q12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>PDE4D</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs17725522,C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1503</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.00</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.06</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.68</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0007</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1948</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 1q31</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>DENND1B</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs12094462,G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">171</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.81</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0229</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.3238</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 2q12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>IL1RL1</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10197862,G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">182</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.09</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.16</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.75</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0004</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0122</td></tr></tbody></table></div><div id="largeobj_idm140563751887008" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3060316/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t2-fn1"><sup>a</sup>All four SNPs in (a) were present in the Illumina chips used and passed QC filters. The top SNPs listed in (b) were imputed with high confidence with MACH, with nearby genotyped SNPs providing comparable evidence for association (cf. <a href="/pmc/articles/PMC3060316/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>).</div><div id="t2-fn2"><sup>b</sup>Power to detect the initial association at <em>α</em>=0.05 in this study.</div><div id="t2-fn3"><sup>c</sup>Number of SNPs located in or within 50 kb of each gene. For <em>ORMDL3</em>, we considered all SNPs located between 50 kb downstream of <em>GSDMB</em> and 50 kb upstream of <em>GSDMA</em>.</div><div id="t2-fn4"><sup>d</sup>Significance of the best SNP after accounting for the number of and LD between all SNPs tested in the respective gene, estimated from 10 000 permutations.</div></div></div><p id="__p19">To address the possibility that allelic heterogeneity could explain the negative results for <em>PDE4D</em>, <em>DENND1B</em> and <em>IL1RL1</em>, we then extended the analysis to all variants available in our study for each of these loci (±50 kb). A variant in <em>IL1RL1</em> was found to be associated with asthma after accounting for the number of and LD between SNPs in the gene (<a href="/pmc/articles/PMC3060316/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2b</span></a>); in contrast, there was no overall evidence for association with <em>PDE4D</em> or <em>DENND1B</em> (<a href="/pmc/articles/PMC3060316/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2b</span></a> and <a href="/pmc/articles/PMC3060316/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>).</p><p id="__p20">The peak variant for <em>IL1RL1</em> was rs10197862 (OR=0.75 for the G allele, <em>P</em>=0.0004, corrected <em>P</em>=0.01), which is in low LD (<em>r</em><sup>2</sup>=0.09) with the rs1420101 variant reported by Gudbjartsson <em>et al.</em><sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600168">16</a></sup> Haplotype analysis of both variants indicates that the rs10197862:G protective allele is found exclusively on a haplotype that also carries the rs1420101:C protective allele (not shown). Although this GC haplotype could be a proxy for a single functional variant that explains both associations, this is unlikely given that with a high-risk allele frequency of 84%, the associated effect size for such variant would have to be high (OR of ∼1.8) to match the results reported for rs1420101 (OR=1.15) by Gudbjartsson <em>et al.</em><sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600158">16</a></sup> As rs10197862 was also in low LD (<em>r</em><sup>2</sup>&lt;0.1) with two nearby variants (rs2310173 and rs917997) reported recently to associate with other immune-related diseases,<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600177">29</a>, <a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_244389276">30</a></sup> our results suggest that rs10197862 represents or tags an independent functional variant in the 2q12 region with significant effects on asthma risk.</p><p id="__p21">Given that the <em>IL1RL1</em> and <em>ORMDL3</em> loci have also been reported to associate with celiac<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600154">29</a></sup> and Crohn's disease,<sup><a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600136">31</a></sup> respectively, we investigated whether there was any evidence for association between asthma and other confirmed loci for these diseases. Among these, no single locus was associated with asthma risk after accounting for the number of SNPs considered (Supplementary Tables 3 and 4).</p><p id="__p22">We extended our genome-wide search for asthma risk loci by considering CNV data that were obtained for 759 adolescents, including 270 doctor-diagnosed asthma cases and 489 controls.</p><p id="__p23">Given that common CNVs were likely tagged by SNPs tested in the genome-wide analysis,<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600161">18</a></sup> we focused on large (&gt;100 kb and &lt;1 Mb), uncommon (MAF &lt;0.05) deletions or duplications identified with high confidence. A total of 2681 such segments were detected, with a median of three CNVs per individual (mean=3.9, range 1–21); the median CNV size was 198 kb (mean 252 kb, range 100–998 kb). Of these, 1621 (61%) were deletions and 1060 were duplications (39%). These CNVs mapped to 244 (deletions) and 230 (duplications) non-overlapping regions of the genome, with a median region length of 259 kb (range 102 kb–4.0 Mb) and 281 kb (range 102 kb–3.6 Mb), respectively.</p><p id="__p24">We performed two sets of CNV association analyses. First, we tested whether the frequency of deletions or duplications at individual loci were associated with asthma status. After accounting for all the regions tested, no single locus was significantly associated with asthma; the two most-associated regions are shown in <a href="/pmc/articles/PMC3060316/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>, including a 102–775 kb deletion of chromosome 1q21 and a 129–432 kb deletion of chromosome 17q21. To visually validate the CNV calls made by QuantiSNPv1.1 for these two regions, we inspected the local log R ratio values for the individual samples identified by the software as carrying a CNV. Log R ratio patterns for the 1q21 region were ambiguous (Supplementary Figure 3) and so this region may represent a technical artefact. In contrast, the log R ratio patterns for all 38 individuals identified as carrying the chromosome 17q21 deletion are indeed consistent with a CNV in this region (Supplementary Figure 4), which is known to contain a common inversion in populations of European ancestry.<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600134">32</a></sup> The telomeric break point for this deletion was located in exon 13 of the <em>NSF</em> gene for 36 of the 38 individuals that carried this deletion, all of which belonged to the H2 haplotype as defined by the inversion-informative rs1800547 variant.<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600178">32</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Individual regions with strongest association between CNVs and asthma risk</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563800069312" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Chromosome</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1q21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">17q21</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">CNV type</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Deletion</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Deletion</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Average CNV span (range) (kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">283 (102–775)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">320 (129–432)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Range shared region (bp)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">146397003–146438291</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">42008822–42137359</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Length shared region (kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">129</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>N</em> cases with CNV/total</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8/270</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">20/270</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>N</em> controls with CNV/total</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2/489</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18/489</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">OR</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>P</em>-value</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0051</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0196</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Corrected <em>P</em>-value<sup><a href="#t3-fn1" rid="t3-fn1" class=" fn">a</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.4403</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9947</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Genes in region ±50 kb</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>NBPF1, NBPF10, NBPF12, NBPF14, PPIAL4A</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>ARL17, ARL17P1, ARL17A, ARL17B, NSF</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">SNP most correlated with CNV (<em>r</em><sup>2</sup>)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">rs7554500 (0.05)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">rs7225002 (0.30)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Visual validation of CNVs<sup><a href="#t3-fn2" rid="t3-fn2" class=" fn">b</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Ambiguous</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Yes</td></tr></tbody></table></div><div id="largeobj_idm140563800069312" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3060316/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t3-fn1"><sup>a</sup>Significance after correcting for all deletions or duplications tested across the genome, estimated from 10 000 permutations.</div><div id="t3-fn2"><sup>b</sup>Local log R ratio patterns for individuals identified as carrying a deletion were inspected to validate the respective CNV calls produced by QuantiSNPv1.1. Intensity patterns for the 17q21 but not for the 1q21 regions were consistent with the CNV calls made (Supplementary Figures 3 and 4).</div></div></div><p id="__p25" class="p p-last">Second, we tested the association between genome-wide CNV burden and asthma risk. We hypothesised that the cumulative effect of multiple deletions or duplications across an individual's genome could significantly increase the risk of developing asthma. After correcting for the number of hypotheses tested, we found no evidence that cases and controls had significant differences in the total number of CNVs, length of CNVs or in the number of genes affected by a CNV, either when considering deletions or duplications (<a href="/pmc/articles/PMC3060316/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Association results between genome-wide CNV burden and asthma risk in 270 asthma cases and 489 controls</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563749287904" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th colspan="4" charoff="50" rowspan="1" valign="top" align="center"><em>Deletions</em></th><th colspan="4" charoff="50" rowspan="1" valign="top" align="center"><em>Duplications</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Cases</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Controls</em></th><th colspan="2" charoff="50" rowspan="1" valign="top" align="center">P<em>-value</em><sup><a href="#t4-fn1" rid="t4-fn1" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Cases</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Controls</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">P<em>-value</em><sup><a href="#t4-fn1" rid="t4-fn1" class=" fn">a</a></sup></th><td rowspan="1" colspan="1">&nbsp;</td></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Uncorrected</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Corrected</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Uncorrected</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Corrected</em></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>N</em> CNVs detected</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">564</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1057</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">326</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">734</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">—</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>N</em> CNVs/individual</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.09</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.16</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8305</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9989</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.50</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0658</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2352</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Total CNV span (kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">693.5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">695.2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9371</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0000</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">476.7</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">572.2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0282</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0994</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Average CNV span (kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">223.3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">237.3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.4115</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8181</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">259.1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">269.1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.5835</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9623</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>N</em> genes spanned by CNVs</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.67</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7.31</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.4318</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8165</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.49</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.01</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0675</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2180</td></tr></tbody></table></div><div id="largeobj_idm140563749287904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3060316/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t4-fn1"><sup>a</sup>Two-sided <em>P</em>-value estimated from 10 000 permutations. Corrected <em>P</em>-values were obtained after adjusting for the overall number of hypotheses tested (<em>N</em>=8) through the analysis of the permuted data sets.</div></div></div></div><div id="__sec12" class="sec sec-last"><h3 id="__sec12title">Follow-up of top SNP and CNV associations</h3><p id="__p26" class="p p-first-last">Finally, we attempted to replicate the most-associated loci identified in the whole-genome SNP and CNV analyses, as well as the novel <em>IL1RL1</em> association, in an independent cohort of 391 doctor-diagnosed asthmatics and 213 controls (Supplementary Table 5). Power to nominally replicate these associations was relatively modest (30–60%), but we observed a convincing association between the <em>IL1RL1</em> variant rs10197862 and asthma (OR=0.58, <em>P</em>=1.2 × 10<sup>−5</sup>), which confirms the presence of a novel independent risk allele in this region (Supplementary Table 6). Also consistent with our initial observation, the 17q21 deletion was more frequent in asthma cases (5/391) than controls (1/213) (Supplementary Figure 5), but these effects were not significant (Supplementary Table 6). The different frequency estimates for this deletion between the discovery (5%) and replication (1%) panels suggests that its prevalence is underestimated in the latter (as all deletions were visually validated) and highlights the technical limitations of CNV detection using genotyping arrays, even when the same array (Illumina 610K) and CNV calling pipeline are used across projects. Further analyses in larger cohorts are thus warranted to confirm this new putative association.</p></div></div><div id="__sec13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">DISCUSSION</h2><p id="__p27" class="p p-first">Our analysis of whole-genome SNP data in 986 self-reported asthma cases and 1846 controls confirms that variants in <em>ORMDL3</em>, a locus previously reported to reproducibly associate with asthma in European and North American populations,<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600143">7</a></sup> influences disease risk in the Australian population. The most-associated variant in our analysis was rs6503525, which lies 39 kb upstream of <em>ORMDL3</em>. Although gene expression results point to <em>ORMDL3</em> as the most likely causal candidate at 17q12, we note that other nearby genes should not be neglected in future experimental work conducted under this broad region of high LD. Specifically, on a recent analysis of genetic determinants of peripheral blood lymphocyte cell levels,<sup><a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600133">33</a></sup> we found that a synonymous change (rs907092) in the Ikaros Family Zinc Finger 3 (<em>IKZF3</em>), which is located 57 kb downstream of <em>ORMDL3</em>, was a significant predictor of B cell levels in adolescents. The G allele that associated with increased numbers of B cells in that analysis had a predisposing effect for asthma in this study (OR=1.22, <em>P</em>=5.1 × 10<sup>−4</sup>). This variant is in partial LD with rs6503525 (<em>r</em><sup>2</sup>=0.52) and, although it may simply tag the same underlying causal variant, it may also represent an additional risk locus in this region. Further studies that explore this possibility are warranted.</p><p id="__p28">Our results also demonstrate that genetic variation in <em>IL1RL1</em> significantly influences asthma risk, but the specific variant identified in our study (rs10197862) is independent of that reported previously to influence eosinophil levels and asthma by Gudbjartsson <em>et al.</em><sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600164">16</a></sup> These results thus indicate that multiple independent variants in <em>IL1RL1</em> can contribute to asthma risk and further emphasise the need to apply gene-based association tests that can effectively account for such effects. Two additional variants near the neighbouring interleukin genes <em>IL18R1</em> and <em>IL1R2</em> have recently been reported to associate with celiac disease<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600157">29</a></sup> and ankylosing spondylitis,<sup><a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_244389277">30</a></sup> but these were in low LD with rs10197862. Together, these data indicate that the 2q12 region has an important and diverse function in the regulation of immune responses.</p><p id="__p29">PDE4D and DENND1B were recently identified as asthma susceptibility genes.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600132">9</a>, <a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600130">11</a></sup> Our analyses do not support these findings, either when focusing on the individual variants reported by the original studies or after extending to all the genetic variation tested in each gene in our study. Failure to replicate the reported associations with PDE4D and DENND1B could represent a false-negative finding in our study (eg limited power to detect an actual effect size that is smaller than that estimated in the original studies), but could also indicate that these genes represent false-positive discoveries. Further analyses in large cohorts are thus warranted to confirm both genes as true asthma susceptibility loci.</p><p id="__p30">We also show that CNVs with strong effects on asthma could not be detected with experiment-wide significance in a subset of 270 cases and 489 controls. However, it is noteworthy that the chromosome 17q21 deletion that associated with increase asthma risk in our analysis overlaps with the known 900 kb inversion<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_214600180">32</a></sup> that was recently reported to associate with corticosteroid response in asthma.<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_244389275">34</a></sup> We thus provide some further support for the association between structural variants in chromosome 17q21 and asthma, but further studies are required to carefully characterise this complex region of the genome. Last, results from our genome-wide CNV burden analysis suggest that asthma cases do not have increased numbers of uncommon large chromosomal deletions or duplications when compared with unaffected individuals. The number of genes affected by CNVs was also comparable between both groups. These results contrast with recent reports for psychiatric diseases, namely schizophrenia.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_378345783">21</a></sup> In our analysis, we observed significant confounding effects of technical variables, such as array batch effects. Ignoring these may result in false-positive results, particularly when cases and controls have been genotyped in different batches or at different times.</p><p id="__p31">There are a number of limitations that should be considered when interpreting our results. First, our definition of lifetime asthma status is somewhat loose, as it was based on self-reported information provided in surveys that often included comparable but not identical questions, and did not exclude co-morbid diseases such as COPD. However, we note that despite these differences, the resulting phenotype is heritable (61%) and allowed us to unambiguously confirm the association with <em>ORMDL3</em>, effectively a positive control for asthma. Second, the sample size analysed was small for a GWAS of a complex disease. These results will now be combined with other ongoing studies to provide a more powerful and comprehensive survey of asthma risk factors. Third, age-at-onset information was only available for a subset of cases and so it was not considered in our analysis. Nonetheless, we note that to date the only locus with confirmed age-at-onset effects is <em>ORMDL3</em>, a locus that we replicate in this study. Last, our analysis of rare structural variants relied heavily on the ability to identify such segments from SNP and CNV probe intensity patterns, an approach that can be influenced by a number of technical artefacts; the sample size used was also very modest. Thus, confirmation of our results by independent studies are warranted, particularly the putative association with the 17q21 deletion.</p><p id="__p32" class="p p-last">In conclusion, we report a convincing association between asthma risk and both novel and previously reported sequence variants in two loci, <em>IL1RL1</em> and <em>ORMDL1</em>; we also identify a 300 kb deletion on chromosome 17q21 with putative effects on disease risk. We found no evidence for association with <em>PDE4D</em> or <em>DENND1B</em>, and no indication that asthmatics have more or larger CNVs than non-asthmatics. These results may help direct future studies of asthma genetics.</p></div><div id="idm140563756603776" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563756603776title">Acknowledgments</h2><div class="sec"><p id="__p33">We thank the twins and their families for their participation; Dixie Statham, Ann Eldridge, Marlene Grace and Kerrie McAloney for sample collection; Lisa Bowdler and Steven Crooks for DNA processing; David Smyth, Harry Beeby and Daniel Park for IT support. Funding was provided by the Australian National Health and Medical Research Council (241944, 339462, 389927, 389875, 389891, 389892, 389938, 442915, 442981, 496739, 552485, 552498, 613627), the Australian Research Council (A7960034, A79906588, A79801419, DP0770096, DP0212016, DP0343921), the FP-5 GenomEUtwin Project (QLG2-CT-2002-01254) and the US National Institutes of Health (NIH Grants AA07728, AA07535, AA10248, AA11998, AA13320, AA13321, AA13326, AA14041, DA12854, MH66206). A portion of the genotyping on which this study was based (Illumina 370k scans on 4300 individuals) was carried out at the Center for Inherited Disease Research, Baltimore (CIDR) through an access award to our late colleague Dr Richard Todd (Psychiatry, Washington University School of Medicine, St Louis). SEM, DRN, MARF, GWM and PMV are supported by the NHMRC Fellowship Scheme.</p></div></div><div id="idm140563759851872" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563759851872title">Notes</h2><p id="__p35">The authors declare no conflict of interest.</p></div><div id="idm140563757159520" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563757159520title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140563795298208"></p><p id="__p34" class="p p-first-last">Supplementary Information accompanies the paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="idm140563795304272"><a href="/pmc/articles/PMC3060316/bin/ejhg2010191x1.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(171K, doc)</sup></div></div><div class="sec suppmat" id="xob2"><h4>Supplementary Table</h4><div class="sup-box half_rhythm" id="idm140563756188576"><a href="/pmc/articles/PMC3060316/bin/ejhg2010191x2.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(194K, doc)</sup></div></div></div><div id="idm140563760010064" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563760010064title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Australian Centre for Asthma Monitoring  . <span class="ref-journal">Asthma in Australia 2008.</span> Canberra: AIHW; 2008. AIHW Asthma Series No. 3. Cat. No. ACM 14. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Asthma+in+Australia+2008&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Lemanske RF, Jr, Busse WW. Asthma: clinical expression and molecular mechanisms. <span><span class="ref-journal">J Allergy Clin Immunol. </span>2010;<span class="ref-vol">125</span>:S95–102.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2853245/">PMC free article</a>]</span> [<a href="/pubmed/20176271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Asthma:+clinical+expression+and+molecular+mechanisms&amp;author=RF+Lemanske&amp;author=WW+Busse&amp;volume=125&amp;publication_year=2010&amp;pages=S95-102&amp;pmid=20176271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Altshuler D, Daly MJ, Lander ES. Genetic mapping in human disease. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">322</span>:881–888.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2694957/">PMC free article</a>]</span> [<a href="/pubmed/18988837" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+mapping+in+human+disease&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;author=ES+Lander&amp;volume=322&amp;publication_year=2008&amp;pages=881-888&amp;pmid=18988837&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Hindorff LA, Sethupathy P, Junkins HA, et al.  Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2009;<span class="ref-vol">106</span>:9362–9367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687147/">PMC free article</a>]</span> [<a href="/pubmed/19474294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Potential+etiologic+and+functional+implications+of+genome-wide+association+loci+for+human+diseases+and+traits&amp;author=LA+Hindorff&amp;author=P+Sethupathy&amp;author=HA+Junkins&amp;volume=106&amp;publication_year=2009&amp;pages=9362-9367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Rioux JD, Xavier RJ, Taylor KD, et al.  Genome-wide association study identifies new susceptibility loci for Crohn disease and implicates autophagy in disease pathogenesis. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:596–604.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2757939/">PMC free article</a>]</span> [<a href="/pubmed/17435756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+new+susceptibility+loci+for+Crohn+disease+and+implicates+autophagy+in+disease+pathogenesis&amp;author=JD+Rioux&amp;author=RJ+Xavier&amp;author=KD+Taylor&amp;volume=39&amp;publication_year=2007&amp;pages=596-604&amp;pmid=17435756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Fellay J, Shianna KV, Ge D, et al.  A whole-genome association study of major determinants for host control of HIV-1. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>:944–947.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1991296/">PMC free article</a>]</span> [<a href="/pubmed/17641165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+whole-genome+association+study+of+major+determinants+for+host+control+of+HIV-1&amp;author=J+Fellay&amp;author=KV+Shianna&amp;author=D+Ge&amp;volume=317&amp;publication_year=2007&amp;pages=944-947&amp;pmid=17641165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Moffatt MF, Kabesch M, Liang L, et al.  Genetic variants regulating ORMDL3 expression contribute to the risk of childhood asthma. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:470–473.</span> [<a href="/pubmed/17611496" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+variants+regulating+ORMDL3+expression+contribute+to+the+risk+of+childhood+asthma&amp;author=MF+Moffatt&amp;author=M+Kabesch&amp;author=L+Liang&amp;volume=448&amp;publication_year=2007&amp;pages=470-473&amp;pmid=17611496&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Choudhry S, Taub M, Mei R, et al.  Genome-wide screen for asthma in Puerto Ricans: evidence for association with 5q23 region. <span><span class="ref-journal">Hum Genet. </span>2008;<span class="ref-vol">123</span>:455–468.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2664533/">PMC free article</a>]</span> [<a href="/pubmed/18401594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Genome-wide+screen+for+asthma+in+Puerto+Ricans:+evidence+for+association+with+5q23+region&amp;author=S+Choudhry&amp;author=M+Taub&amp;author=R+Mei&amp;volume=123&amp;publication_year=2008&amp;pages=455-468&amp;pmid=18401594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">Himes BE, Hunninghake GM, Baurley JW, et al.  Genome-wide association analysis identifies PDE4D as an asthma-susceptibility gene. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:581–593.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681010/">PMC free article</a>]</span> [<a href="/pubmed/19426955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+association+analysis+identifies+PDE4D+as+an+asthma-susceptibility+gene&amp;author=BE+Himes&amp;author=GM+Hunninghake&amp;author=JW+Baurley&amp;volume=84&amp;publication_year=2009&amp;pages=581-593&amp;pmid=19426955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Hancock DB, Romieu I, Shi M, et al.  Genome-wide association study implicates chromosome 9q21.31 as a susceptibility locus for asthma in Mexican children. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000623.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722731/">PMC free article</a>]</span> [<a href="/pubmed/19714205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+implicates+chromosome+9q21.31+as+a+susceptibility+locus+for+asthma+in+Mexican+children&amp;author=DB+Hancock&amp;author=I+Romieu&amp;author=M+Shi&amp;volume=5&amp;publication_year=2009&amp;pages=e1000623&amp;pmid=19714205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Sleiman PM, Flory J, Imielinski M, et al.  Variants of DENND1B associated with asthma in children. <span><span class="ref-journal">N Engl J Med. </span>2010;<span class="ref-vol">362</span>:36–44.</span> [<a href="/pubmed/20032318" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Variants+of+DENND1B+associated+with+asthma+in+children&amp;author=PM+Sleiman&amp;author=J+Flory&amp;author=M+Imielinski&amp;volume=362&amp;publication_year=2010&amp;pages=36-44&amp;pmid=20032318&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Mathias RA, Grant AV, Rafaels N, et al.  A genome-wide association study on African-ancestry populations for asthma. <span><span class="ref-journal">J Allergy Clin Immunol. </span>2010;<span class="ref-vol">125</span>:336–346.e4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3606015/">PMC free article</a>]</span> [<a href="/pubmed/19910028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=A+genome-wide+association+study+on+African-ancestry+populations+for+asthma&amp;author=RA+Mathias&amp;author=AV+Grant&amp;author=N+Rafaels&amp;volume=125&amp;publication_year=2010&amp;pages=336-346.e4&amp;pmid=19910028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Li X, Howard TD, Zheng SL, et al.  Genome-wide association study of asthma identifies RAD50-IL13 and HLA-DR/DQ regions. <span><span class="ref-journal">J Allergy Clin Immunol. </span>2010;<span class="ref-vol">125</span>:328–335.e11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2824608/">PMC free article</a>]</span> [<a href="/pubmed/20159242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Genome-wide+association+study+of+asthma+identifies+RAD50-IL13+and+HLA-DR/DQ+regions&amp;author=X+Li&amp;author=TD+Howard&amp;author=SL+Zheng&amp;volume=125&amp;publication_year=2010&amp;pages=328-335.e11&amp;pmid=20159242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Ober C, Tan Z, Sun Y, et al.  Effect of variation in CHI3L1 on serum YKL-40 level, risk of asthma, and lung function. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:1682–1691.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2629486/">PMC free article</a>]</span> [<a href="/pubmed/18403759" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Effect+of+variation+in+CHI3L1+on+serum+YKL-40+level,+risk+of+asthma,+and+lung+function&amp;author=C+Ober&amp;author=Z+Tan&amp;author=Y+Sun&amp;volume=358&amp;publication_year=2008&amp;pages=1682-1691&amp;pmid=18403759&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Weidinger S, Gieger C, Rodriguez E, et al.  Genome-wide scan on total serum IgE levels identifies FCER1A as novel susceptibility locus. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000166.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2565692/">PMC free article</a>]</span> [<a href="/pubmed/18846228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+scan+on+total+serum+IgE+levels+identifies+FCER1A+as+novel+susceptibility+locus&amp;author=S+Weidinger&amp;author=C+Gieger&amp;author=E+Rodriguez&amp;volume=4&amp;publication_year=2008&amp;pages=e1000166&amp;pmid=18846228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib16"><span class="mixed-citation">Gudbjartsson DF, Bjornsdottir US, Halapi E, et al.  Sequence variants affecting eosinophil numbers associate with asthma and myocardial infarction. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:342–347.</span> [<a href="/pubmed/19198610" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Sequence+variants+affecting+eosinophil+numbers+associate+with+asthma+and+myocardial+infarction&amp;author=DF+Gudbjartsson&amp;author=US+Bjornsdottir&amp;author=E+Halapi&amp;volume=41&amp;publication_year=2009&amp;pages=342-347&amp;pmid=19198610&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Scherer SW, Lee C, Birney E, et al.  Challenges and standards in integrating surveys of structural variation. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:S7–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2698291/">PMC free article</a>]</span> [<a href="/pubmed/17597783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Challenges+and+standards+in+integrating+surveys+of+structural+variation&amp;author=SW+Scherer&amp;author=C+Lee&amp;author=E+Birney&amp;volume=39&amp;publication_year=2007&amp;pages=S7-15&amp;pmid=17597783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">The Wellcome Trust Case Control Consortium  Genome-wide association study of CNVs in 16 000 cases of eight common diseases and 3000 shared controls. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">464</span>:713–720.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2892339/">PMC free article</a>]</span> [<a href="/pubmed/20360734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+CNVs+in+16 000+cases+of+eight+common+diseases+and+3000+shared+controls&amp;volume=464&amp;publication_year=2010&amp;pages=713-720&amp;pmid=20360734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Bochukova EG, Huang N, Keogh J, et al.  Large, rare chromosomal deletions associated with severe early-onset obesity. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">463</span>:666–670.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3108883/">PMC free article</a>]</span> [<a href="/pubmed/19966786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Large,+rare+chromosomal+deletions+associated+with+severe+early-onset+obesity&amp;author=EG+Bochukova&amp;author=N+Huang&amp;author=J+Keogh&amp;volume=463&amp;publication_year=2010&amp;pages=666-670&amp;pmid=19966786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Weiss LA, Shen Y, Korn JM, et al.  Association between microdeletion and microduplication at 16p11.2 and autism. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:667–675.</span> [<a href="/pubmed/18184952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Association+between+microdeletion+and+microduplication+at+16p11.2+and+autism&amp;author=LA+Weiss&amp;author=Y+Shen&amp;author=JM+Korn&amp;volume=358&amp;publication_year=2008&amp;pages=667-675&amp;pmid=18184952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib21"><span class="mixed-citation">The International Schizophrenia Consortium  Rare chromosomal deletions and duplications increase risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:237–241.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912847/">PMC free article</a>]</span> [<a href="/pubmed/18668038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Rare+chromosomal+deletions+and+duplications+increase+risk+of+schizophrenia&amp;volume=455&amp;publication_year=2008&amp;pages=237-241&amp;pmid=18668038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib22"><span class="mixed-citation">Ionita-Laza I, Perry GH, Raby BA, et al.  On the analysis of copy-number variations in genome-wide association studies: a translation of the family-based association test. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:273–284.</span> [<a href="/pubmed/18228561" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=On+the+analysis+of+copy-number+variations+in+genome-wide+association+studies:+a+translation+of+the+family-based+association+test&amp;author=I+Ionita-Laza&amp;author=GH+Perry&amp;author=BA+Raby&amp;volume=32&amp;publication_year=2008&amp;pages=273-284&amp;pmid=18228561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib23"><span class="mixed-citation">Medland SE, Nyholt DR, Painter JN, et al.  Common variants in the trichohyalin gene are associated with straight hair in Europeans. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">85</span>:750–755.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775823/">PMC free article</a>]</span> [<a href="/pubmed/19896111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Common+variants+in+the+trichohyalin+gene+are+associated+with+straight+hair+in+Europeans&amp;author=SE+Medland&amp;author=DR+Nyholt&amp;author=JN+Painter&amp;volume=85&amp;publication_year=2009&amp;pages=750-755&amp;pmid=19896111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib24"><span class="mixed-citation">Blangero J, Almasy L. Multipoint oligogenic linkage analysis of quantitative traits. <span><span class="ref-journal">Genet Epidemiol. </span>1997;<span class="ref-vol">14</span>:959–964.</span> [<a href="/pubmed/9433607" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Multipoint+oligogenic+linkage+analysis+of+quantitative+traits&amp;author=J+Blangero&amp;author=L+Almasy&amp;volume=14&amp;publication_year=1997&amp;pages=959-964&amp;pmid=9433607&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib25"><span class="mixed-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib26"><span class="mixed-citation">Frazer KA, Ballinger DG, Cox DR, et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=KA+Frazer&amp;author=DG+Ballinger&amp;author=DR+Cox&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib27"><span class="mixed-citation">Colella S, Yau C, Taylor JM, et al.  QuantiSNP: an Objective Bayes Hidden-Markov Model to detect and accurately map copy number variation using SNP genotyping data. <span><span class="ref-journal">Nucleic Acids Res. </span>2007;<span class="ref-vol">35</span>:2013–2025.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1874617/">PMC free article</a>]</span> [<a href="/pubmed/17341461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=QuantiSNP:+an+Objective+Bayes+Hidden-Markov+Model+to+detect+and+accurately+map+copy+number+variation+using+SNP+genotyping+data&amp;author=S+Colella&amp;author=C+Yau&amp;author=JM+Taylor&amp;volume=35&amp;publication_year=2007&amp;pages=2013-2025&amp;pmid=17341461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib28"><span class="mixed-citation">Shurin GV, Ferris RL, Tourkova IL, et al.  Loss of new chemokine CXCL14 in tumor tissue is associated with low infiltration by dendritic cells (DC), while restoration of human CXCL14 expression in tumor cells causes attraction of DC both <em>in vitro</em> and <em>in vivo</em>. <span><span class="ref-journal">J Immunol. </span>2005;<span class="ref-vol">174</span>:5490–5498.</span> [<a href="/pubmed/15843547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Loss+of+new+chemokine+CXCL14+in+tumor+tissue+is+associated+with+low+infiltration+by+dendritic+cells+(DC),+while+restoration+of+human+CXCL14+expression+in+tumor+cells+causes+attraction+of+DC+both+in+vitro+and+in+vivo&amp;author=GV+Shurin&amp;author=RL+Ferris&amp;author=IL+Tourkova&amp;volume=174&amp;publication_year=2005&amp;pages=5490-5498&amp;pmid=15843547&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib29"><span class="mixed-citation">Dubois PC, Trynka G, Franke L, et al.  Multiple common variants for celiac disease influencing immune gene expression. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:295–302.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2847618/">PMC free article</a>]</span> [<a href="/pubmed/20190752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+common+variants+for+celiac+disease+influencing+immune+gene+expression&amp;author=PC+Dubois&amp;author=G+Trynka&amp;author=L+Franke&amp;volume=42&amp;publication_year=2010&amp;pages=295-302&amp;pmid=20190752&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib30"><span class="mixed-citation">Australo-Anglo-American Spondyloarthritis Consortium (TASC)  Reveille JD, Sims AM, Danoy P, et al.  Genome-wide association study of ankylosing spondylitis identifies non-MHC susceptibility loci. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:123–127.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3224997/">PMC free article</a>]</span> [<a href="/pubmed/20062062" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+ankylosing+spondylitis+identifies+non-MHC+susceptibility+loci&amp;author=JD+Reveille&amp;author=AM+Sims&amp;author=P+Danoy&amp;volume=42&amp;publication_year=2010&amp;pages=123-127&amp;pmid=20062062&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib31"><span class="mixed-citation">Barrett JC, Hansoul S, Nicolae DL, et al.  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn's disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:955–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn's+disease&amp;author=JC+Barrett&amp;author=S+Hansoul&amp;author=DL+Nicolae&amp;volume=40&amp;publication_year=2008&amp;pages=955-962&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib32"><span class="mixed-citation">Stefansson H, Helgason A, Thorleifsson G, et al.  A common inversion under selection in Europeans. <span><span class="ref-journal">Nat Genet. </span>2005;<span class="ref-vol">37</span>:129–137.</span> [<a href="/pubmed/15654335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+inversion+under+selection+in+Europeans&amp;author=H+Stefansson&amp;author=A+Helgason&amp;author=G+Thorleifsson&amp;volume=37&amp;publication_year=2005&amp;pages=129-137&amp;pmid=15654335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib33"><span class="mixed-citation">Ferreira MA, Mangino M, Brumme CJ, et al.  Quantitative trait loci for CD4: CD8 lymphocyte ratio are associated with risk of type 1 diabetes and HIV-1 immune control. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">86</span>:88–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2801744/">PMC free article</a>]</span> [<a href="/pubmed/20045101" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Quantitative+trait+loci+for+CD4:+CD8+lymphocyte+ratio+are+associated+with+risk+of+type+1+diabetes+and+HIV-1+immune+control&amp;author=MA+Ferreira&amp;author=M+Mangino&amp;author=CJ+Brumme&amp;volume=86&amp;publication_year=2010&amp;pages=88-92&amp;pmid=20045101&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib34"><span class="mixed-citation">Tantisira KG, Lazarus R, Litonjua AA, Klanderman B, Weiss ST. Chromosome 17: association of a large inversion polymorphism with corticosteroid response in asthma. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2008;<span class="ref-vol">18</span>:733–737.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3225071/">PMC free article</a>]</span> [<a href="/pubmed/18622266" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Chromosome+17:+association+of+a+large+inversion+polymorphism+with+corticosteroid+response+in+asthma&amp;author=KG+Tantisira&amp;author=R+Lazarus&amp;author=AA+Litonjua&amp;author=B+Klanderman&amp;author=ST+Weiss&amp;volume=18&amp;publication_year=2008&amp;pages=733-737&amp;pmid=18622266&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3060316&amp;issue-id=195174&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140563756007920title">Abstract</a></li><li><a href="#__sec1title">INTRODUCTION</a></li><li><a href="#__sec2title">MATERIALS AND METHODS</a></li><li><a href="#__sec8title">RESULTS</a></li><li><a href="#__sec13title">DISCUSSION</a></li><li><a href="#idm140563756603776title">Acknowledgments</a></li><li><a href="#idm140563759851872title">Notes</a></li><li><a href="#idm140563757159520title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm140563760010064title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3060316/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3060316/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3060316/pdf/ejhg2010191a.pdf">PDF (411K)</a></li> | <li><a href="#" data-citationid="PMC3060316" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3060316%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3060316%2F&amp;text=Association%20between%20ORMDL3%2C%20IL1RL1%20and%20a%20deletion%20on%20chromosome%2017q21%20with%20asthma%20risk%20in%20Australia" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3060316%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21150878" data-aiid="3060316" data-aid="3060316" data-iid="195174" data-domainid="1300" data-domain="ejhg" data-accid="PMC3060316" data-md5="ae40e5f1b23ba089710ee991bdef3533"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:26:26-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CA0D3DD51375100000000097A062A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3060316%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3060316/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>