<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3529467/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Association Study Implicates HLA-C*01: 02 as a Risk Factor at the Major Histocompatibility Complex Locus in Schizophrenia</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3529467/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Biological psychiatry"><meta name="citation_title" content="Genome-Wide Association Study Implicates HLA-C*01: 02 as a Risk Factor at the Major Histocompatibility Complex Locus in Schizophrenia"><meta name="citation_authors" content="Irish Schizophrenia Genomics Consortium and the Wellcome Trust Case Control Consortium 2"><meta name="citation_date" content="15 October 2012"><meta name="citation_issue" content="8"><meta name="citation_volume" content="72"><meta name="citation_firstpage" content="620"><meta name="citation_doi" content="10.1016/j.biopsych.2012.05.035"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3529467/?report=abstract"><meta name="citation_pmid" content="22883433"><meta name="DC.Title" content="Genome-Wide Association Study Implicates HLA-C*01: 02 as a Risk Factor at the Major Histocompatibility Complex Locus in Schizophrenia"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Irish Schizophrenia Genomics Consortium and the Wellcome Trust Case Control Consortium 2"><meta name="DC.Date" content="2012 Oct 15"><meta name="DC.Identifier" content="10.1016/j.biopsych.2012.05.035"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Association Study Implicates HLA-C*01: 02 as a Risk Factor at the Major Histocompatibility Complex Locus in Schizophrenia"><meta property="og:type" content="article"><meta property="og:description" content="We performed a genome-wide association study (GWAS) to identify common risk variants for schizophrenia.The discovery scan included 1606 patients and 1794 controls from Ireland, using 6,212,339 directly genotyped or imputed single nucleotide polymorphisms ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3529467/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3529467/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3529467/pdf/nihms428997.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88662DDD53E591000000000A4106C2.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3529467%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="mesh">MeSH</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3529467</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Biol Psychiatry</a></span>. Author manuscript; available in PMC 2012 Dec 24.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22883433" target="pmc_ext" ref="reftype=publisher&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Biol Psychiatry. 2012 Oct 15; 72(8): 620–628. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Aug 9. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.biopsych.2012.05.035" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.biopsych.2012.05.035</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3529467</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS428997</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22883433">22883433</a></div></div></div></div><h1 class="content-title">Genome-Wide Association Study Implicates HLA-C*01: 02 as a Risk Factor at the Major Histocompatibility Complex Locus in Schizophrenia</h1><div class="half_rhythm"><div class="contrib-group fm-author">Irish Schizophrenia Genomics Consortium and the Wellcome Trust Case Control Consortium 2</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140021746222608_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140021746222608_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140021746222608_ai" style="display:none"><div id="cor1">Address correspondence to Aiden Corvin, M.D., Ph.D., Trinity Centre for Health Sciences, Department of Psychiatry, St. James’s Hospital, Dublin 8, Ireland; <a href="mailto:dev@null" data-email="ei.dct@nivroca" class="oemail">ei.dct@nivroca</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140021746222608_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22883433" target="pmc_ext" ref="reftype=publisher&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Biol Psychiatry</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3529467/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>irish_SCH_GWAS_supp.</strong></div><div><a href="/pmc/articles/PMC3529467/bin/NIHMS428997-supplement-irish_SCH_GWAS_supp.pdf" data-ga-action="click_feat_suppl">NIHMS428997-supplement-irish_SCH_GWAS_supp.pdf</a><span style="color:gray"> (994K)</span></div><div class="small guid">GUID:&nbsp;12DD931A-39C5-4D04-93DF-F6E57B8215BF</div></div></dd></dl></div><div id="idm140021757167312" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140021757167312title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">We performed a genome-wide association study (GWAS) to identify common risk variants for schizophrenia.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">The discovery scan included 1606 patients and 1794 controls from Ireland, using 6,212,339 directly genotyped or imputed single nucleotide polymorphisms (SNPs). A subset of this sample (270 cases and 860 controls) was subsequently included in the Psychiatric GWAS Consortium-schizophrenia GWAS meta-analysis.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">One hundred eight SNPs were taken forward for replication in an independent sample of 13,195 cases and 31,021 control subjects. The most significant associations in discovery, corrected for genomic inflation, were (rs204999, <em>p</em> combined = 1.34 × 10<sup>−9</sup> and in combined samples (rs2523722 <em>p</em> combined = 2.88 × 10<sup>−16</sup>) mapped to the major histocompatibility complex (MHC) region. We imputed classical human leukocyte antigen (HLA) alleles at the locus; the most significant finding was with HLA-C*01:02. This association was distinct from the top SNP signal. The HLA alleles DRB1*03:01 and B*08:01 were protective, replicating a previous study.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">This study provides further support for involvement of MHC class I molecules in schizophrenia. We found evidence of association with previously reported risk alleles at the <em>TCF4</em>, <em>VRK2</em>, and <em>ZNF804A</em> loci.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text"><em>CACNA1I</em>, genetics, genome-wide association study, HLAC, major histocompatibility complex, polygene score, schizophrenia</span></div></div><div id="idm140021755143216" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Schizophrenia is a complex psychiatric disorder of substantial heritability (h<sup>2</sup>~.8) characterized by hallucinations, delusions, disordered thinking, and cognitive deficits (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024414">1</a>). The disorder poses a significant public health problem, as it affects approximately .8% of the population, causes considerable morbidity, and reduces average life expectancy by 20 to 25 years (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024463">2</a>). The onset of illness is typically in early adulthood, but despite current treatments, the evolution of symptoms, severity, and course of the disorder are variable.</p><p id="P6">Understanding the genetic component of schizophrenia has been a major focus in the field, with the hope that this will aid development of more effective diagnostics and therapeutics. Recently, a number of genome-wide association studies (GWAS) have made encouraging progress in identifying a spectrum of common and rare genetic risk variants that contribute to schizophrenia susceptibility (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395242925">3</a>–<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024464">11</a>). They have found evidence for a number of susceptibility loci, including the major histocompatibility complex (MHC) region on chromosome 6p21–6p22. Although the MHC has been implicated in multiple schizophrenia GWAS (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395242924">3</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024442">6</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024429">12</a>), the signals reported cover a broad region across the different studies from 27.2 megabase (Mb) to 32.7 Mb.</p><p id="P7" class="p">Here, we report a schizophrenia GWAS conducted as part of the Wellcome Trust Case Control Consortium 2 (WTCCC2) study of 15 complex disorders and traits. The discovery dataset was made up of an Irish cohort of 1606 affected and 1794 unaffected population control subjects. We performed a logistic regression case-control analysis (see Methods and Materials) and selected regions for follow-up with single nucleotide polymorphisms (SNPs) that surpassed a significance level of <em>p</em> &lt; 10<sup>−4</sup>. Replication data consisted of a large independent set of 13,195 cases and 31,021 control subjects. Imputations based on the 1000 Genomes (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024445">13</a>) reference panel yielded information at over six million SNPs after quality control (QC). In addition, we present evidence for association at loci that had met criteria for genome-wide association in previously reported schizophrenia and bipolar disorder GWAS. Finally, we imputed classical human leukocyte antigen (HLA) alleles and performed association analyses with these alleles to further define the MHC association seen in the SNP data.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Methods and Materials</h2><div id="S6" class="sec sec-first"><h3 id="S6title">Discovery Subjects</h3><div id="S7" class="sec sec-first"><p></p><h4 id="S7title" class="inline">Discovery Sample Cases </h4><p id="P8" class="p p-first-last">The case sample of 1908 individuals was recruited through community mental services and inpatient units in the Republic of Ireland and Northern Ireland following similar research protocols and with local ethics approval. All participants were interviewed using a structured clinical interview (Structured Clinical Interview for DSM-IV [<a href="#R14" rid="R14" class=" bibr popnode">14</a>]; Schedule for Affective Disorders and Schizophrenia [Lifetime Version] [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024434">15</a>]; or Schedule for Clinical Assessment in Neuropsychiatry [<a href="#R16" rid="R16" class=" bibr popnode">16</a>]). Diagnosis of a major psychotic disorder was made by the consensus lifetime best estimate method using DSM-IV criteria with all available information (interview, family or staff report, and chart review). All cases were over 18 years of age, were of Irish origin (being of Irish parents and having all four grandparents born in Ireland or the United Kingdom), and had been screened to exclude substance-induced psychotic disorder or psychosis due to a general medical condition. The final diagnostic determination was made by one of the four lead clinicians on the project (A.C., T.O.N., K.S.K., D.W.). The final analysis included samples from 1606 participants whose SNP data passed quality control filters, detailed in <a href="#SD1" rid="SD1" class=" supplementary-material">Table S1 in Supplement 1</a>, and met DSM-IV criteria for schizophrenia (<em>n</em> = 1418), schizoaffective disorder (<em>n</em> = 182), or schizophreniform disorder (<em>n</em> = 6).</p></div><div id="S8" class="sec sec-last"><p></p><h4 id="S8title" class="inline">Discovery Sample Control Subjects </h4><p id="P9" class="p p-first-last">Control subjects (<em>n</em> = 2048) were ascertained with written informed consent from the Irish GeneBank and represented blood donors from the Irish Blood Transfusion Service recruited in the Republic of Ireland. They met the same ethnicity criteria as cases but were not specifically screened for psychiatric illness. Individuals taking regularly prescribed medication are excluded from blood donation in Ireland and donors are not financially remunerated. As the lifetime prevalence of schizophrenia is relatively low (&lt;1%) and there was no obvious reason for individuals with schizophrenia to be overrepresented in the control subjects, the fraction of putative case individuals in the control collection was expected to be small. Following QC (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1 in Supplement 1</a>), the control sample included 1794 participants.</p></div></div><div id="S9" class="sec"><h3 id="S9title">Replication Subjects</h3><p id="P10" class="p p-first">The total replication sample included 13,195 cases and 31,021 control subjects. This was composed of nonoverlapping samples from the Psychiatric GWAS Consortium (PGC)-Schizophrenia consortium sample (9393 cases and 12,462 control subjects) and the SGENE+ consortium sample (1053 cases and 12,174 control subjects). It also included a WTCCC2 replication dataset (2749 cases from the United Kingdom and 6385 control subjects from the 1958 Birth Cohort and National Blood Service), genotyped on the Illumina Immunochip, which is referred to as the Immunochip replication sample.</p><div id="S10" class="sec"><p></p><h4 id="S10title" class="inline">Psychiatric GWAS Consortium Sample </h4><p id="P11" class="p p-first-last">The sample included individuals of European ancestry with a DSM-IV (or equivalent) diagnosis of schizophrenia or schizoaffective disorder. The full sample includes 9394 cases, which were collected with institutional review board approval at multiple centers in 17 individual cohorts representing 11 countries (Australia, Bulgaria, Denmark, Germany, Ireland, Netherlands, Norway, Portugal, Sweden, United Kingdom, and the United States of America). Control subjects consisted of 12,462 samples of European ancestry collected from the same countries. The PGC-Schizophrenia sample is described more fully elsewhere (<a href="#R9" rid="R9" class=" bibr popnode">9</a>). A small number of Irish cases (<em>n</em> = 270) and control subjects (<em>n</em> = 860) overlapped with our discovery sample and were excluded from the replication analysis for this project. Due to the consortium nature of the PGC, the composite populations are genotyped on different platforms and are imputed to give a common set of SNPs. The SNPs for which we attempted replication are therefore not genotyped in all of the individual populations and are either made up of a subset of genotyped populations, with the remaining imputed, or are imputed in all populations.</p></div><div id="S11" class="sec"><p></p><h4 id="S11title" class="inline">SGENE+ Sample </h4><p id="P12" class="p p-first-last">Details of the SGENE+ sample have been published elsewhere (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024438">12</a>). Part of the SGENE+ sample (identified in their paper as the SGENE-Bonn, Copenhagen, Munich, <em>TOP3</em>, and University of California, Los Angeles sites) had contributed to stage 1 of the PGC analysis and were excluded from the SGENE+ sample for replication purposes.</p></div><div id="S12" class="sec sec-last"><p></p><h4 id="S12title" class="inline">Immunochip Replication Sample </h4><p id="P13" class="p p-first-last">After quality control, this sample totaled 2749 cases collected in the United Kingdom and 6385 control subjects from 1958 Birth Cohort and National Blood Service cohorts. The control sample did not overlap with WTCCC1 control subjects included in the PGC sample. All cases were diagnosed with schizophrenia and were undergoing monitoring by the United Kingdom Clozaril Monitoring Service for clozapine therapy; full details on the control subjects are provided elsewhere (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024447">17</a>). Samples were genotyped on the Illumina Immunochip platform at the Sanger Institute, United Kingdom. The Immunochip SNP chip was designed jointly by a number of groups, including the Wellcome Trust Case Control Consortium 2, and included 2400 SNPs selected from the top hits reported in individual schizophrenia GWAS (including from this study) and from the PGC meta-analysis of schizophrenia GWAS data. For the study presented here, the Immunochip data were used as a shallow replication cohort, where data at less than 100 SNPs were analyzed.</p></div></div><div id="S13" class="sec"><h3 id="S13title">DNA Sample Preparation</h3><p id="P14" class="p p-first-last">For all participants in the discovery sample, DNA was extracted from whole blood using standard methods. Genomic DNA for all cases and control subjects was shipped to the Sanger Institute, United Kingdom, or the Broad Institute, United States. Similar procedures for DNA quality control were applied at both sites and are detailed elsewhere (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395242926">3</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024449">17</a>).</p></div><div id="S14" class="sec"><h3 id="S14title">Genotype Calling</h3><p id="P15" class="p p-first-last">All discovery samples were genotyped using the Affymetrix 6.0 platform totaling 893,634 autosomal SNPs, either at the Affymetrix (Santa Clara, California) or Broad Institute (Cambridge, Massachusetts) laboratories. A subset of samples (<em>n</em> = 37) were genotyped at both sites and successfully identified as duplicates as part of the QC procedure. The final sample included 1606 cases (1302 were genotyped at Affymetrix) and 1794 control subjects (1001 were genotyped at Affymetrix). For all samples passing laboratory quality control, raw intensities (from the .CEL files) were renormalized within collections using CelQuantileNorm (<a href="http://outmodedbonsai.sourceforge.net/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://outmodedbonsai.sourceforge.net/</a>). These normalized intensities were used to call genotypes with an updated version of the Chiamo software (<a href="https://mathgen.stats.ox.ac.uk/genetics_software/chiamo/chiamo.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/genetics_software/chiamo/chiamo.html</a>), adapted for Affymetrix 6.0 SNP data (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024416">17</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024461">18</a>).</p></div><div id="S15" class="sec"><h3 id="S15title">Quality Control</h3><p id="P16" class="p p-first">Full details of the quality control methods employed in the Wellcome Trust Case Control Consortium 2 have been published elsewhere (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024448">17</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024418">19</a>) and additional details are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods in Supplement 1</a>. After applying all of the quality control criteria, from a total of 3956 samples, 566 samples were excluded, leaving 1606 cases and 1794 control subjects for further analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1 in Supplement 1</a>). After quality control of the SNP data (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods in Supplement 1</a>), 696,951 directly assayed SNPs were available for analysis.</p><p id="P17" class="p p-last">The genomic inflation statistic (λ) was 1.10 (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1 in Supplement 1</a>). This is slightly higher than would be ideal for a genome-wide study (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024433">20</a>), and several methods were applied to try and understand any underlying structure in the data. Using a label corresponding to the genotyping center as a covariate gave a similar λ to just using the data that was genotyped at Affymetrix (λ = 1.095, λ = 1.09, respectively). Including the first two principal components as covariates also had no effect on reducing the value of λ (λ = 1.095). None of these methods substantially reduced the genomic inflation, suggesting there could be an underlying polygenic component to the inflation (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024422">21</a>), as previous studies have found for schizophrenia (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395242928">3</a>). Therefore, no covariates were used for the following analyses unless explicitly described. Genomic control was not applied to the results of the primary analysis, but for completeness, <a href="#SD1" rid="SD1" class=" supplementary-material">Table S4 in Supplement 1</a> shows <em>p</em> values after applying genomic control at SNPs that were taken through to replication. Visual inspection of the Manhattan plot (<a href="/pmc/articles/PMC3529467/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) suggested that the filtered data provide a good basis for association analyses.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3529467/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140021748452864" class="figure"><a href="/pmc/articles/PMC3529467/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3529467_nihms428997f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms428997f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3529467/bin/nihms428997f1.jpg"></a></div><div id="largeobj_idm140021748452864" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3529467/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3529467/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P56">Plot of genome-wide association results after fitting the multiplicative model in SNPTEST. Genome-wide association results in the discovery data at 696,957 directly genotyped single nucleotide polymorphisms, each represented as a point on the plot. Chromosomes are colored dark blue and light blue alternatively, as labeled on the x axis. The −log 10 (<em>p</em> value) is shown on the y axis. Loci identified in this study as being potentially associated with schizophrenia (<a href="/pmc/articles/PMC3529467/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) are highlighted in red and labeled by chromosomal region.</p></div></div></div></div><div id="S16" class="sec"><h3 id="S16title">Imputation</h3><div id="S17" class="sec sec-first"><p></p><h4 id="S17title" class="inline">SNP Imputation </h4><p id="P18" class="p p-first-last">Haplotype phasing and imputation was performed using IMPUTE2 (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312720154">22</a>). Only samples that had passed the quality control filtering were imputed and before imputation SNPs were subjected to additional quality control to remove any that could adversely affect the quality of the imputation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplement 1</a>). Imputation was performed using the HapMap2 and 3 and the 1000 Genomes reference panels; further details are in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplement 1</a>.</p></div><div id="S18" class="sec sec-last"><p></p><h4 id="S18title" class="inline">HLA Imputation </h4><p id="P19" class="p p-first-last">Human leukocyte antigen alleles were imputed using an updated version of HLA*IMP (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024428">23</a>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplement 1</a>).</p></div></div><div id="S19" class="sec sec-last"><h3 id="S19title">Statistical Methods</h3><p id="P20" class="p p-first">We report <em>p</em> values from 1-df Cochran-Armitage tests for trend as implemented in the software SNPTEST (<a href="https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html</a>). A parallel analysis using the Cochran-Armitage trend test in PLINK (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024431">24</a>) gave near identical results (data not shown). We also performed 2-df genotypic tests to verify that none of our associations showed significant deviation from a multiplicative model and two marker logistic regressions to test for interactions between associated markers. Effect size (odds ratio [OR]) estimates represent per-allele increase of risk in a multiplicative model.</p><div id="S20" class="sec sec-last"><p></p><h4 id="S20title" class="inline">Polyscore Analyses </h4><p id="P21" class="p p-first-last">We performed a direct replication experiment of the polyscore analysis performed by the International Schizophrenia Consortium (ISC) (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395242923">3</a>). The principle, described in that paper, is that multiple genetic variants of very small effect (OR &lt; 1.05) may have only a small chance of showing individual significance in a GWAS, but en masse they could contribute a substantial amount of risk. To be as close as possible to the original study, we chose the same 74,062 SNPs as in the original paper and used the same <em>p</em> value thresholds. Because of overlapping samples between this study and the International Schizophrenia Consortium, this analysis was restricted to a sample set of 1310 cases and 1023 control subjects.</p></div></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Results</h2><p id="P22" class="p p-first">The top association in this GWAS was in the MHC region (rs204999, <em>p</em> = 2.77 × 10<sup>−8</sup>) (<a href="/pmc/articles/PMC3529467/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figures 1</span></a> and <a href="/pmc/articles/PMC3529467/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2).</span></span><span>2</span></a>). This SNP is 63 kilobase upstream from, and in weak linkage disequilibrium (1000 Genomes CEU data <em>R</em><sup>2</sup> = .23) with, the signal reported by Stefansson <em>et al</em>. (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024426">12</a>) (rs3131296). After accounting for rs204999, rs3131296 does not add any further signal in our dataset (<em>p</em> = .816). The MHC was the only region in the discovery GWAS with SNPs that had <em>p</em> &lt; 10<sup>−7</sup>. Association analysis was also carried out on the X chromosome, but following quality control, no SNPs were significant at <em>p</em> &lt; 10<sup>−4</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplement 1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3529467/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140021752498512" class="figure"><a href="/pmc/articles/PMC3529467/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3529467_nihms428997f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms428997f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3529467/bin/nihms428997f2.jpg"></a></div><div id="largeobj_idm140021752498512" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3529467/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3529467/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="P57">Regional association plots of each of the loci described in <a href="/pmc/articles/PMC3529467/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. The −log 10 (<em>p</em> values) for the multiplicative model single nucleotide polymorphism (SNP) association in SNPTEST are shown on the upper part of the plot. Single nucleotide polymorphisms are colored based on their <em>R<sup>2</sup></em> with the labeled SNP, <em>R<sup>2</sup></em> calculated in the 58C control data. Where the SNP was chosen from the 1000 Genome imputation results, the plot includes the imputed SNPs. Circles represent the directly genotyped SNPs, and triangles represent the imputed SNPs. The bottom part of each plot shows the fine scale recombination rate estimated from individuals in the HapMap panel, and genes are marked by horizontal blue lines. Chr, chromosome; cM, centimorgan; Mb, megabase.</p></div></div></div><p id="P23">In total, we selected 79 regions for follow-up, where the most significant SNP had <em>p</em> &lt; 10<sup>−4</sup>, and in the case of directly genotyped SNPs, the cluster plots met quality control standards. One or more SNPs were chosen for replication in each region, depending on the local pattern of linkage disequilibrium and the availability of SNPs in the replication data (108 SNPs in total). Seven SNPs spanning the association signal from 30.3 Mb to 34.1 Mb in the MHC region were included in the replication phase of the study. Of the other regions, 56 were identified from the genotyped scan (66 SNPs), 10 from the HapMap imputation (11 SNPs), and 12 from the 1000 Genomes imputation (24 SNPs). Replication was carried out in silico using the PGC and SGENE+ samples and SNPs were genotyped directly in the Immunochip replication samples, giving a total of 13,195 cases and 31,021 control subjects. Data were not available for all the SNPs on all samples due to differences in imputation strategies and genotyping platforms (see Methods and Materials).</p><p id="P24">Of note, a small proportion of the Irish cases in the discovery sample (<em>n</em> = 270) and Irish control subjects (<em>n</em> = 860) were included in the PGC-schizophrenia GWAS meta-analysis. We included these samples in the discovery phase for two reasons. First, this analysis had additional genotype information provided by 1000 Genomes imputation (6,212,339 directly genotyped and imputed SNPs) and analysis of the X chromosome when compared with the PGC analysis (1,252,901 autosomal SNPs). Second, existing data suggest genetic heterogeneity (e.g., at the MHC) may be important in schizophrenia etiology, indicating a value in reporting the results of the complete Irish sample. We performed a sensitivity analysis excluding these overlapping cases and control subjects from our discovery GWAS scan. This reduced the number of regions meeting criteria for follow-up (from 79 to 48) but did not materially affect the study outcome. We provide data on the SNP findings for the full and restricted discovery datasets at all loci selected for follow-up in <a href="#SD1" rid="SD1" class=" supplementary-material">Table S4 in Supplement 1</a>. Expanding the sample to include additional cases with a broader psychotic disorder phenotype (<em>n</em> = 2049 cases, including mostly bipolar disorder cases) did not significantly alter the results.</p><p id="P25">Three of the seven SNPs tested in the MHC reached a <em>p</em> &lt; 5 × 10<sup>−8</sup> in a fixed effects meta-analysis across the four follow-up cohorts (<em>p</em> combined) (<a href="/pmc/articles/PMC3529467/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). These three SNPs showed evidence of association in two of the three replication datasets; no replication evidence (<em>p</em> &lt; .05) at the MHC was found in the SGENE+ data (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2 in Supplement 1</a>). These MHC SNPs were the only replication results to meet standard significance criteria (<em>p</em> &lt; .05) when correcting for the number of replication SNPs tested.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P58">Loci with Evidence for Association with Schizophrenia Identified in this Study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140021752495936" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="3" colspan="1" valign="bottom" align="left">Chromosome</td><td rowspan="3" colspan="1" valign="bottom" align="left">rsID</td><td rowspan="3" colspan="1" valign="bottom" align="right">Position<sup>a</sup></td><td rowspan="3" colspan="1" valign="bottom" align="left">Candidate<br>Locus</td><td colspan="5" rowspan="1" valign="bottom" align="center">Discovery</td><td rowspan="3" colspan="1" valign="bottom" align="left"><em>p</em> Value<br>SGENE+</td><td rowspan="3" colspan="1" valign="bottom" align="left"><em>p</em> Value PGC</td><td rowspan="3" colspan="1" valign="bottom" align="left"><em>p</em> Value<br>Immunochip</td><td rowspan="3" colspan="1" valign="bottom" align="left"><em>p</em> Value<br>Combined</td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="center">Risk<br>Allele</td><td rowspan="1" colspan="1" align="center">RAF<br>Cases</td><td rowspan="1" colspan="1" align="center">RAF<br>Control</td><td rowspan="1" colspan="1" align="center"><em>p</em> Value<br>Discovery</td><td rowspan="1" colspan="1" valign="bottom" align="center">OR (95% CI)</td></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">1p21</td><td rowspan="1" colspan="1" align="left">rs987913<sup>b</sup></td><td rowspan="1" colspan="1" align="right">95678503</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;—</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">.405</td><td rowspan="1" colspan="1" align="center">.357</td><td rowspan="1" colspan="1" align="center">2.92 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.24 (1.12–1.37)</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0176</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">1.48 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">3p14</td><td rowspan="1" colspan="1" align="left">rs862689<sup>a</sup></td><td rowspan="1" colspan="1" align="right">71744029</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;—</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">.412</td><td rowspan="1" colspan="1" align="center">.364</td><td rowspan="1" colspan="1" align="center">2.68 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.24 (1.12–1.37)</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0149</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.312<sup>e</sup></td><td rowspan="1" colspan="1" align="left">2.69 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">3p14</td><td rowspan="1" colspan="1" align="left">rs2684380<sup>b</sup></td><td rowspan="1" colspan="1" align="right">71745822</td><td rowspan="1" colspan="1" align="left"><em>FOXP1</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">.427</td><td rowspan="1" colspan="1" align="center">.383</td><td rowspan="1" colspan="1" align="center">1.91 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.20 (1.09–1.33)</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0332</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">6.05 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">3q13</td><td rowspan="1" colspan="1" align="left">rs7618341</td><td rowspan="1" colspan="1" align="right">119268497</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;—</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">.599</td><td rowspan="1" colspan="1" align="center">.552</td><td rowspan="1" colspan="1" align="center">1.17 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.20 (1.10–1.32)</td><td rowspan="1" colspan="1" align="left">.437 (−)</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0536</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0196</td><td rowspan="1" colspan="1" align="left">2.65 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">6p21</td><td rowspan="1" colspan="1" align="left">rs2523722</td><td rowspan="1" colspan="1" align="right">30273252</td><td rowspan="1" colspan="1" align="left"><em>MHC</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">.778</td><td rowspan="1" colspan="1" align="center">.737</td><td rowspan="1" colspan="1" align="center">8.71 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.25 (1.12–1.39)</td><td rowspan="1" colspan="1" align="left">.736</td><td rowspan="1" colspan="1" align="left">8.74 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="left">6.08 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="left">1.47 × 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">6p21</td><td rowspan="1" colspan="1" align="left">rs204999</td><td rowspan="1" colspan="1" align="right">32217957</td><td rowspan="1" colspan="1" align="left"><em>MHC</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">.695</td><td rowspan="1" colspan="1" align="center">.631</td><td rowspan="1" colspan="1" align="center">2.77 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.33 (1.20–1.47)</td><td rowspan="1" colspan="1" align="left">.621</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0442</td><td rowspan="1" colspan="1" align="left">1.93 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="left">5.37 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">6p21</td><td rowspan="1" colspan="1" align="left">rs7746922</td><td rowspan="1" colspan="1" align="right">32538953</td><td rowspan="1" colspan="1" align="left"><em>MHC</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">.765</td><td rowspan="1" colspan="1" align="center">.723</td><td rowspan="1" colspan="1" align="center">1.30 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.23 (1.10–1.36)</td><td rowspan="1" colspan="1" align="left">.197</td><td rowspan="1" colspan="1" align="left">2.82 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0314</td><td rowspan="1" colspan="1" align="left">4.48 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">7q31</td><td rowspan="1" colspan="1" align="left">rs4730488</td><td rowspan="1" colspan="1" align="right">110953401</td><td rowspan="1" colspan="1" align="left"><em>IMMP2L</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">.517</td><td rowspan="1" colspan="1" align="center">.466</td><td rowspan="1" colspan="1" align="center">2.85 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.22 (1.11–1.34)</td><td rowspan="1" colspan="1" align="left">.0718</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0287</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.152</td><td rowspan="1" colspan="1" align="left">2.09 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">15q15</td><td rowspan="1" colspan="1" align="left">rs12595425<sup>b</sup>,<sup>c</sup></td><td rowspan="1" colspan="1" align="right">38350279</td><td rowspan="1" colspan="1" align="left"><em>PAK6</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">.884</td><td rowspan="1" colspan="1" align="center">.852</td><td rowspan="1" colspan="1" align="center">8.26 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.44 (1.23–1.68)</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.0116</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">2.74 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">22q13</td><td rowspan="1" colspan="1" align="left">rs9611198<sup>b</sup></td><td rowspan="1" colspan="1" align="right">38285819</td><td rowspan="1" colspan="1" align="left"><em>CACNA1I</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">.508</td><td rowspan="1" colspan="1" align="center">.458</td><td rowspan="1" colspan="1" align="center">4.35 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.22 (1.11–1.34)</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">2.48 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NA</td><td rowspan="1" colspan="1" align="left">7.56 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">22q13</td><td rowspan="1" colspan="1" align="left">rs9607658<sup>d</sup></td><td rowspan="1" colspan="1" align="center">38287686</td><td rowspan="1" colspan="1" align="left"><em>CACNA1I</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">.513</td><td rowspan="1" colspan="1" align="center">.463</td><td rowspan="1" colspan="1" align="center">5.60 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.21 (1.10–1.33)</td><td rowspan="1" colspan="1" align="left">.275 (−)</td><td rowspan="1" colspan="1" align="left">7.19 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">6.33 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1.11 × 10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm140021752495936" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3529467/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P59" class="p p-first">The discovery, replication, and fixed effects meta-analysis results at SNPs for which there is evidence (one-sided <em>p</em> &lt; .05) in the majority of the replication cohorts where data are available. All <em>p</em> values shown are two-sided, and (−) denotes that the direction of effect is opposite to that in the discovery analysis.</p><p id="P60" class="p p-last">CI, confidence interval; NA, no data available; OR, odds ratio; PGC, Psychiatric GWAS Consortium; RAF, risk allele frequency; SNP, single nucleotide polymorphism.</p></div><div id="TFN2"><sup>a</sup>National Center for Biotechnology Information human genome build 36 coordinates.</div><div id="TFN3"><sup>b</sup>1000 Genomes imputed SNP.</div><div id="TFN4"><sup>c</sup>SNP (rs12595425) not present in the replication data so replication measured at rs3784396 (<em>R<sup>2</sup></em> = .99).</div><div id="TFN5"><sup>d</sup>HapMap imputed SNP.</div><div id="TFN6"><sup>e</sup>Replication was attempted at a tag SNP in the Immunochip replication, rs2594113 (<em>R<sup>2</sup></em> = .97).</div></div></div><p id="P26">Six other regions showed some evidence for association in the replication dataset (<a href="/pmc/articles/PMC3529467/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). In each case, the direction of effect was consistent in the majority of replication datasets, although the combined <em>p</em> values were &gt;5 × 10<sup>−8</sup> (<a href="/pmc/articles/PMC3529467/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="/pmc/articles/PMC3529467/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2 in Supplement 1</a>). These included two variants on chromosome 22, rs9611198 (<em>p</em> combined = 7.56 × 10<sup>−6</sup>) and rs9607658 (<em>p</em> combined = 1.11 × 10<sup>−5</sup>), at the calcium channel gene <em>CACNA1I</em>. This gene is of potential interest, as its protein product interacts with the product of <em>CACNA1C</em>, a susceptibility gene for both schizophrenia (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024415">25</a>) and bipolar disorder (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024458">26</a>). Other members of this calcium channel gene family have been implicated in both schizophrenia and bipolar disorder etiology (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024452">27</a>) and childhood seizure disorder (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024457">28</a>). We identified suggestive evidence for association at a locus that includes the gene <em>IMMP2L</em> (rs4730488) across three of the four datasets (<em>p</em> combined = 2.09 × 10<sup>−5</sup>). Disruption of this gene has been reported in sporadic and familial cases with Tourette syndrome (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024437">29</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024439">30</a>) and in individuals with autistic spectrum disorder (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024430">31</a>), and an <em>IMMP2L-DOCK4</em> deletion has been identified in a family with dyslexia (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024456">32</a>). Finally, we identified suggestive evidence for association with a SNP in the gene <em>PAK6</em> (rs12595425, <em>p</em> combined = 2.74 × 10<sup>−5</sup>), a member of the group B family of p21-activated kinase serine/threonine kinases. <em>PAK6</em> is highly expressed in the brain; disruption of this gene in mice is associated with deficits in learning and memory (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024443">33</a>) and upregulation of the gene may be part of the response to traumatic brain injury (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024451">34</a>).</p><p id="P27">In the schizophrenia GWAS literature to date, including the MHC region, SNPs at 13 independent loci have reached a significance threshold of <em>p</em> &lt; 5 × 10<sup>−8</sup>. Results at these SNPs in the WTCCC2 genome-wide scan are shown in <a href="/pmc/articles/PMC3529467/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. Results from our study at additional loci reported for schizophrenia or bipolar disorder, as collated in the National Human Genome Research Institute catalogue of GWAS studies (<a href="http://www.genome.gov/26525384" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/26525384</a> [searched January 2012]), are included in <a href="#SD1" rid="SD1" class=" supplementary-material">Table S2 in Supplement 1</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P66">Evidence in the Discovery Data at Loci Previously Reported to be Associated with Schizophrenia at <em>p</em> &lt; 5 × 10<sup>−8</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140021748009264" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Chr</td><td rowspan="1" colspan="1" valign="bottom" align="left">rsID</td><td rowspan="1" colspan="1" valign="bottom" align="right">Position<sup>a</sup></td><td rowspan="1" colspan="1" align="left">Locus of<br>Interest</td><td rowspan="1" colspan="1" align="center">Risk<br>Allele</td><td rowspan="1" colspan="1" align="left">Case<br>RAF</td><td rowspan="1" colspan="1" align="left">Control<br>RAF</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>p</em> Value<sup>b</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">OR</td><td rowspan="1" colspan="1" valign="bottom" align="right">(95% CI)</td><td rowspan="1" colspan="1" valign="bottom" align="right">Reference</td></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1</td><td rowspan="1" colspan="1" align="left">rs1625579<sup>c</sup></td><td rowspan="1" colspan="1" align="right">98275522</td><td rowspan="1" colspan="1" align="left"><em>MIR137</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">.83</td><td rowspan="1" colspan="1" align="left">.82</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.035</td><td rowspan="1" colspan="1" align="left">1.12</td><td rowspan="1" colspan="1" align="right">(.99–1.27)</td><td rowspan="1" colspan="1" align="right">(<a href="#R9" rid="R9" class=" bibr popnode">9</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1</td><td rowspan="1" colspan="1" align="left">rs10489202</td><td rowspan="1" colspan="1" align="right">166169703</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;—</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">.25</td><td rowspan="1" colspan="1" align="left">.23</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.027</td><td rowspan="1" colspan="1" align="left">1.11</td><td rowspan="1" colspan="1" align="right">(1.00–1.24)</td><td rowspan="1" colspan="1" align="right">(<a href="#R10" rid="R10" class=" bibr popnode">10</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;2</td><td rowspan="1" colspan="1" align="left">rs2312147<sup>d</sup></td><td rowspan="1" colspan="1" align="right">58076432</td><td rowspan="1" colspan="1" align="left"><em>VRK2</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">.64</td><td rowspan="1" colspan="1" align="left">.61</td><td rowspan="1" colspan="1" align="left">4.94 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1.14</td><td rowspan="1" colspan="1" align="right">(1.03–1.25)</td><td rowspan="1" colspan="1" align="right">(<a href="#R12" rid="R12" class=" bibr popnode">12</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;2</td><td rowspan="1" colspan="1" align="left">rs1344706<sup>e</sup></td><td rowspan="1" colspan="1" align="right">185486673</td><td rowspan="1" colspan="1" align="left"><em>ZNF804A</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">.65</td><td rowspan="1" colspan="1" align="left">.63</td><td rowspan="1" colspan="1" align="left">5.56 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1.17</td><td rowspan="1" colspan="1" align="right">(1.01–1.23)</td><td rowspan="1" colspan="1" align="right">(<a href="#R4" rid="R4" class=" bibr popnode">4</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;3</td><td rowspan="1" colspan="1" align="left">rs6782299<sup>e</sup></td><td rowspan="1" colspan="1" align="right">182033396</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;—</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">.74</td><td rowspan="1" colspan="1" align="left">.73</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.148</td><td rowspan="1" colspan="1" align="left">1.07</td><td rowspan="1" colspan="1" align="right">(.94–1.23)</td><td rowspan="1" colspan="1" align="right">(<a href="#R3" rid="R3" class=" bibr popnode">3</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;6</td><td rowspan="1" colspan="1" align="left">rs1635<sup>d</sup></td><td rowspan="1" colspan="1" align="right">28335583</td><td rowspan="1" colspan="1" align="left"><em>NKAPL</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">.97</td><td rowspan="1" colspan="1" align="left">.97</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.360</td><td rowspan="1" colspan="1" align="left">1.05</td><td rowspan="1" colspan="1" align="right">(.79–1.41)</td><td rowspan="1" colspan="1" align="right">(<a href="#R11" rid="R11" class=" bibr popnode">11</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;6</td><td rowspan="1" colspan="1" align="left">rs2142731</td><td rowspan="1" colspan="1" align="right">28358892</td><td rowspan="1" colspan="1" align="left"><em>PGBD1</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">.003</td><td rowspan="1" colspan="1" align="left">.002</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.099</td><td rowspan="1" colspan="1" align="left">1.92</td><td rowspan="1" colspan="1" align="right">(.71–5.18)</td><td rowspan="1" colspan="1" align="right">(<a href="#R11" rid="R11" class=" bibr popnode">11</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;6</td><td rowspan="1" colspan="1" align="left">rs3131296<sup>d</sup></td><td rowspan="1" colspan="1" align="right">32280971</td><td rowspan="1" colspan="1" align="left"><em>MHC</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">.85</td><td rowspan="1" colspan="1" align="left">.82</td><td rowspan="1" colspan="1" align="left">1.88 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="left">1.26</td><td rowspan="1" colspan="1" align="right">(1.11–1.44)</td><td rowspan="1" colspan="1" align="right">(<a href="#R12" rid="R12" class=" bibr popnode">12</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;6</td><td rowspan="1" colspan="1" align="left">rs9272219</td><td rowspan="1" colspan="1" align="right">32710247</td><td rowspan="1" colspan="1" align="left"><em>MHC</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">.77</td><td rowspan="1" colspan="1" align="left">.74</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.016</td><td rowspan="1" colspan="1" align="left">1.13</td><td rowspan="1" colspan="1" align="right">(1.01–1.26)</td><td rowspan="1" colspan="1" align="right">(<a href="#R6" rid="R6" class=" bibr popnode">6</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;8</td><td rowspan="1" colspan="1" align="left">rs10503253<sup>c</sup></td><td rowspan="1" colspan="1" align="right">4168252</td><td rowspan="1" colspan="1" align="left"><em>CSMD1</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">.22</td><td rowspan="1" colspan="1" align="left">.19</td><td rowspan="1" colspan="1" align="left">9.06 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1.15</td><td rowspan="1" colspan="1" align="right">(1.02 × 1.30)</td><td rowspan="1" colspan="1" align="right">(<a href="#R9" rid="R9" class=" bibr popnode">9</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;8</td><td rowspan="1" colspan="1" align="left">rs16887244</td><td rowspan="1" colspan="1" align="right">38150502</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;—</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">.77</td><td rowspan="1" colspan="1" align="left">.75</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.084</td><td rowspan="1" colspan="1" align="left">1.08</td><td rowspan="1" colspan="1" align="right">(.97–1.21)</td><td rowspan="1" colspan="1" align="right">(<a href="#R10" rid="R10" class=" bibr popnode">10</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;8</td><td rowspan="1" colspan="1" align="left">rs7004633<sup>c</sup></td><td rowspan="1" colspan="1" align="right">89829427</td><td rowspan="1" colspan="1" align="left"><em>MMP16</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">.18</td><td rowspan="1" colspan="1" align="left">.16</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.062</td><td rowspan="1" colspan="1" align="left">1.11</td><td rowspan="1" colspan="1" align="right">(.97–1.26)</td><td rowspan="1" colspan="1" align="right">(<a href="#R9" rid="R9" class=" bibr popnode">9</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left">rs11191580<sup>c</sup></td><td rowspan="1" colspan="1" align="right">104896201</td><td rowspan="1" colspan="1" align="left"><em>NT5C2</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">.93</td><td rowspan="1" colspan="1" align="left">.92</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.139</td><td rowspan="1" colspan="1" align="left">1.10</td><td rowspan="1" colspan="1" align="right">(.92–1.32)</td><td rowspan="1" colspan="1" align="right">(<a href="#R9" rid="R9" class=" bibr popnode">9</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">11</td><td rowspan="1" colspan="1" align="left">rs835784</td><td rowspan="1" colspan="1" align="right">44820394</td><td rowspan="1" colspan="1" align="left"><em>TSPAN18</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">.48</td><td rowspan="1" colspan="1" align="left">.47</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.256</td><td rowspan="1" colspan="1" align="left">1.03</td><td rowspan="1" colspan="1" align="right">(.94–1.14)</td><td rowspan="1" colspan="1" align="right">(<a href="#R11" rid="R11" class=" bibr popnode">11</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">11</td><td rowspan="1" colspan="1" align="left">rs12807809</td><td rowspan="1" colspan="1" align="right">124111495</td><td rowspan="1" colspan="1" align="left"><em>NRGN</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">.83</td><td rowspan="1" colspan="1" align="left">.82</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.144</td><td rowspan="1" colspan="1" align="left">1.07</td><td rowspan="1" colspan="1" align="right">(.94–1.22)</td><td rowspan="1" colspan="1" align="right">(<a href="#R12" rid="R12" class=" bibr popnode">12</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">18</td><td rowspan="1" colspan="1" align="left">rs17594526<sup>e</sup></td><td rowspan="1" colspan="1" align="right">51209236</td><td rowspan="1" colspan="1" align="left"><em>TCF4</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">.03</td><td rowspan="1" colspan="1" align="left">.02</td><td rowspan="1" colspan="1" align="left">1.05 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1.77</td><td rowspan="1" colspan="1" align="right">(1.22–2.54)</td><td rowspan="1" colspan="1" align="right">(<a href="#R3" rid="R3" class=" bibr popnode">3</a>)</td></tr><tr><td rowspan="1" colspan="1" align="left">18</td><td rowspan="1" colspan="1" align="left">rs9960767<sup>d</sup></td><td rowspan="1" colspan="1" align="right">51306000</td><td rowspan="1" colspan="1" align="left"><em>TCF4</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">.05</td><td rowspan="1" colspan="1" align="left">.04</td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;.080</td><td rowspan="1" colspan="1" align="left">1.18</td><td rowspan="1" colspan="1" align="right">(.94–1.49)</td><td rowspan="1" colspan="1" align="right">(<a href="#R12" rid="R12" class=" bibr popnode">12</a>)</td></tr></tbody></table></div><div id="largeobj_idm140021748009264" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3529467/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><p id="P67" class="p p-first-last">Chr, chromosome; CI, confidence interval; GWAS, genome-wide association studies; ISC, International Schizophrenia Consortium; OR, odds ratio; PGC, Psychiatric GWAS Consortium; RAF, risk allele frequency, SCZ, schizophrenia; SNP, single nucleotide polymorphism; WTCCC2, Wellcome Trust Case Control Consortium 2.</p></div><div id="TFN8"><sup>a</sup>National Center for Biotechnology Information human genome build 36 coordinates.</div><div id="TFN9"><sup>b</sup><em>p</em> value is one sided in the direction of risk in the original study; the direction of effect is the same as in the original study for all SNPs.</div><div id="TFN10"><sup>c</sup>The WTCCC2-SCZ data contributed to the PGC GWAS meta-analysis.</div><div id="TFN11"><sup>d</sup>SNP imputed in WTCCC2 data.</div><div id="TFN12"><sup>e</sup>There was some overlap between the WTCCC2-SCZ sample and the O’Donovan <em>et al.</em> (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024453">4</a>) and ISC studies. These cases and control subjects are excluded from the WTCCC2-SCZ sample for the purposes of the current analysis.</div></div></div><p id="P28">The strongest replication of currently known loci was found on chromosome 6p21 (SNP imputed, rs3131296, one-sided <em>p</em> = 1.88 × 10<sup>−4</sup>) at the MHC locus. A number of schizophrenia GWAS studies (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395242927">3</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024446">6</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024441">12</a>) have reported association at this locus; in our data, there was varying support for the SNPs previously reported (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S3 in Supplement 1</a>). Analysis of the MHC in our dataset is discussed more fully below. Outside the MHC, we found evidence of association at 6 of the 13 previously identified risk alleles, where the one-sided <em>p</em> &lt; .05 is in the same direction of effect as the original study. The three most significant findings mapped to the genes <em>TCF4</em>, <em>ZNF804A</em>, and <em>VRK2</em> (<a href="/pmc/articles/PMC3529467/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P29">The MHC is a highly polymorphic and complex region of the genome, containing many genes. To date, attempts to dissect the schizophrenia association signal at the MHC have been inconclusive (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395242929">3</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024436">6</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024413">12</a>). In this study, three MHC SNPs each reached a fixed effect <em>p</em> combined &lt; 5 × 10<sup>−8</sup> (rs204999, rs2523722, rs7746922). In the discovery cohort, conditioning on the most significantly associated SNP (rs204999) substantially reduced the evidence of association at the other replicated MHC SNPs (rs2523722 <em>p</em> = .071 and rs7746922 <em>p</em> = .250). However, rs204999 did not account for all evidence of SNP association at the locus. A secondary signal was seen at rs2499690 and rs3103431 (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S3 in Supplement 1</a>), although there was not strong replication evidence for this second signal.</p><p id="P30">Previous HLA analyses, including those reported from large collaborative GWAS efforts (representing most of our follow-up sample), have been equivocal, which likely reflects known heterogeneity at the locus and the contribution of multiple different populations to these studies. For this reason, we attempted to identify whether a specific classical allele increased risk in our sample and whether this could be replicated in a large, new sample collection from the United Kingdom (the Immunochip replication sample). To investigate whether any of the classical human leukocyte antigen alleles could be driving the SNP signal at the MHC, classical HLA alleles were imputed using an updated version of the HLA*IMP algorithm (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024455">23</a>,<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024435">35</a>). Human leukocyte antigen alleles were imputed to a four-digit accuracy at <em>HLA-A</em>, <em>HLA-B</em>, <em>HLA-C</em>, <em>HLA-DQA1</em>, <em>HLA-DQB1</em>, and <em>HLA-DRB1</em> (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods in Supplement 1</a>).</p><p id="P31">An association test for each allele was performed in the Irish sample (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S5 in Supplement 1</a>), and HLA-C*01:02 showed the strongest evidence of association with schizophrenia (<em>p</em> = 3.83 × 10<sup>−4</sup>), where presence of HLA-C*01:02 was associated with increased risk of schizophrenia (OR [95%CI] = 1.65 [1.25–2.18]). HLA-C*01:02 was found at a frequency of 4.1% in cases and 2.5% in control subjects and was imputed with high specificity and sensitivity (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods in Supplement 1</a>). This allele was also associated with risk of schizophrenia in the Immunochip replication data, where HLA imputation was available for 2690 cases and 6279 control subjects (<em>p</em> = .045, OR [95%CI] = 1.19 [1.01–1.41]).</p><p id="P32">When conditioning on HLA-C*01:02, the top SNP from the discovery scan, rs204999, was still the most significant SNP (<em>p</em> = 2.51 × 10<sup>−7</sup>), and similarly, when conditioning on rs204999, HLA-C*01:02 was still significant (<em>p</em> = 2.82 × 10<sup>−3</sup>). This suggests that both signals may be important in this dataset.</p><p id="P33">Stefansson <em>et al</em>. (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024417">12</a>) found that SNP haplotypes tagging DRB1*03 and HLA-B*08 were protective for schizophrenia. In this study, we replicated these findings at the imputed HLA alleles directly rather than at tagging SNP haplotypes: HLA-B*08:01 <em>p</em> = .024 and DRB1*03:01 <em>p</em> = .068 (one-sided <em>p</em> values shown). Both of these HLA alleles were imputed with high specificity and sensitivity (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods in Supplement 1</a>). DRB1*03:01 and B*08:01 were in high linkage disequilibrium (LD) (<em>R<sup>2</sup></em> = .61) and from these data it appears that these two HLA alleles were tagging the same signal.</p><p id="P34" class="p p-last">Taking the large set of autosomal score alleles (<em>n</em> = 74,062) from the ISC dataset, we investigated the aggregate risk scores for individuals in the WTCCC2 schizophrenia sample. We examined increasingly liberal <em>p</em> value thresholds (PT) from PT &lt; .01 to PT &lt; .7. We identified higher mean score in cases compared with control subjects in the WTCCC2 dataset. The ISC-derived score was highly associated with disease at PT &lt; .5, as has previously been reported in other schizophrenia datasets (PGC) (<a href="#R36" rid="R36" class=" bibr popnode">36</a>). In our sample, the most significant PT finding was at PT &lt; .3 (<em>p</em> &lt; 10<sup>−15</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4 in Supplement 1</a>).</p></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Discussion</h2><p id="P35" class="p p-first">We report a schizophrenia GWAS in a sample of 1606 cases and 1794 control subjects. We found strong support for association at three SNPs in the MHC, including the lead SNP in the discovery data rs204999. Multiple SNPs in the MHC region have been identified in previous schizophrenia GWAS (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395242930">3</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024419">6</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024460">12</a>). Conditional analysis of the associated SNPs suggests that in this data the signal is being driven by association with rs204999, although there is evidence of association with other SNPs at the locus. Previous work to pinpoint HLA alleles that could be responsible for the signal in the MHC have had varied success. Comparison of association with HLA alleles across studies is complicated by differences in the methodology used to call alleles, as well as differences in the populations being studied. In this study, the strongest HLA allele association was found with HLA-C*01:02, and this was separate from the top SNP signal. We also replicated association with two protective HLA alleles, HLA-B*08:01 and DRB1*03:01, which had previously been identified by Stefansson <em>et al</em>. (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024425">5</a>). Our analysis suggests that these alleles tag the same association signal.</p><p id="P36">There is growing awareness of the important role that MHC genes play in nervous system development and function (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024440">37</a>). Major histocompatibility complex class I molecules are expressed in the brain and function both presynaptically and postsynaptically in forming and regulating cortical connectivity across brain circuitry (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291024459">38</a>). Our study identifies risk and protective HLA alleles for schizophrenia, which may be particularly relevant because of autoimmune or infectious risk factors in this population.</p><p id="P37">Because of extensive linkage disequilibrium, localizing signals within the MHC is extremely challenging, as evident from <a href="/pmc/articles/PMC3529467/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>. The SNP rs204999 is in strong LD with seven other SNPs (<em>R<sup>2</sup></em> = .66 –.68; rs3130284, rs3134608, rs3130347, rs3134945, rs3096697, rs3134947, rs3132965), which span a region from 32.22 Mb to 32.26 Mb containing <em>PPT2</em>, <em>PRRT1</em>, <em>EGFL8</em>, <em>AGPAT1</em>, and <em>RNF5</em>. None of these genes feature in the pre-GWAS schizophrenia genetics literature and there is no strong in silico information available to suggest a functional role for rs204999 in the regulation of these genes or others. Single nucleotide polymorphism rs2523722 is at 30.27 Mb on chromosome 6 and is located in an intron of <em>TRIM26</em>. This variant is in complete LD (<em>R<sup>2</sup></em> = 1) with 11 other SNPs within a 10 kilobase region, all of which map to <em>TRIM26</em>. Nine of these SNPs are intronic, one (rs2523721) is a coding silent SNP, and one (rs971570) is located in the 5’ untranslated region. There is no prior evidence of association between <em>TRIM26</em> and schizophrenia and the gene is a member of a tripartite motif family of unknown function.</p><p id="P38" class="p p-last">We performed a polygene score analysis using the large set of autosomal score alleles identified in the previous ISC schizophrenia study. The proportion of variance explained (estimated by Nagelkerke’s <em>R<sup>2</sup></em>) in the original analysis was 3% to 4% and was 6% in this sample, adding support to the hypothesis that a substantial fraction of schizophrenia heritability is accounted for by a polygenic component. In this study, we attempted replication at 79 regions and identified nominal significant evidence of association (<em>p</em> &lt; .05) with the same risk allele at 17 regions; this is significantly more than expected by chance (<em>p</em> = 3.09 × 10<sup>−7</sup>). Larger studies are required to confirm which of these regions harbor schizophrenia susceptibility variants. We identified suggestive evidence of association at loci that could implicate three novel genes in schizophrenia susceptibility (<em>IMMP2L</em>, <em>PAK6</em>, and <em>CACNA1I</em>). Two of these loci were identified from imputation based on reference panels from the 1000 Genomes data, suggesting the inclusion of the imputation data was a useful addition to this study. Based on the experience for other common disorders, further investigation of these loci in a larger meta-analysis of GWAS datasets may be warranted.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>irish_SCH_GWAS_supp</h4><div class="sup-box half_rhythm" id="d35e860"><a href="/pmc/articles/PMC3529467/bin/NIHMS428997-supplement-irish_SCH_GWAS_supp.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(994K, pdf)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Acknowledgments</h2><div class="sec"><p id="P39">The authors sincerely thank all patients who contributed to this study and all staff who facilitated their involvement. Funding for this study was provided by the Wellcome Trust Case Control Consortium 2 project (085475/B/08/Z and 085475/Z/08/Z), the Wellcome Trust (072894/Z/03/Z, 090532/Z/09/Z and 075491/Z/04/B), NIMH grants (MH 41953 and <a id="__tag_788181541" class="tag_hotlink" href="/nuccore/MH083094" ref="/nuccore/MH083094"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH083094","term_id":"1365487447","term_text":"MH083094"}}</span>MH083094</a>) and Science Foundation Ireland (08/IN.1/B1916). P. Donnelly was supported in part by a Wolfson-Royal Society Merit Award. We also thank S. Bertrand, J. Bryant, S.L. Clark, J.S. Conquer, T. Dibling, J.C. Eldred, S. Gamble, C. Hind, A. Wilk, C.R. Stribling and S. Taylor of the Wellcome Trust Sanger Institute’s Sample and Genotyping Facilities for technical assistance. We thank S. Leslie for support with the HLA imputation analysis. We acknowledge use of the Irish GeneBank sample, the British 1958 Birth Cohort DNA collection funded by the Medical Research Council (grant G0000934) and the Wellcome Trust (grant 068545/Z/02) and of the UK National Blood Service controls funded by the Wellcome Trust.</p></div></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title">Appendix A</h2><!--/article/back/app-group/app/--><p id="P42">The authors of this paper are: Amy Strange<sup>1,*</sup>, Brien P Riley<sup>2,*</sup>, Chris C A Spencer<sup>1</sup>, Derek W Morris<sup>3</sup>, Matti Pirinen<sup>1</sup>, Colm T O’Dushlaine<sup>3</sup>, Zhan Su<sup>1</sup>, Brion S Maher<sup>2</sup>, Colin Freeman<sup>1</sup>, Paul Cormican<sup>3</sup>, Celine Bellenguez<sup>1</sup>, Elaine M Kenny<sup>3</sup>, Gavin Band<sup>1</sup>, Brandon Wormley<sup>2</sup>, Gary Donohoe<sup>3</sup>, Alexander Dilthey<sup>4</sup>, Loukas Moutsianas<sup>1,4</sup>, Emma Quinn<sup>3</sup>, Sarah Edkins<sup>5</sup>, Roisin Judge<sup>3</sup>, Kim Coleman<sup>3</sup>, Sarah Hunt<sup>5</sup>, Daniela Tropea<sup>3</sup>, Siobhan Roche<sup>6</sup>, Liz Cummings<sup>3</sup>, Eric Kelleher<sup>3</sup>, Patrick McKeon<sup>6</sup>, Ted Dinan<sup>7</sup>, Colm McDonald<sup>8</sup>, Kieran C Murphy<sup>9</sup>, Eadbhard O’Callaghan<sup>10,†</sup>, Francis A O’Neill<sup>11</sup>, John L Waddington<sup>12</sup>, Dermot Walsh<sup>13</sup>, Eleni Giannoulatou<sup>1</sup>, Cordelia Langford<sup>5</sup>, Panos Deloukas<sup>5</sup>, Emma Gray<sup>5</sup>, Serge Dronov<sup>5</sup>, Simon Potter<sup>5</sup>, Richard Pearson<sup>1</sup>, Damjan Vukcevic<sup>1</sup>, Avazeh Tashakkori-Ghanbaria<sup>5</sup>, Jenefer M Blackwell<sup>14,15</sup>, Elvira Bramon<sup>16</sup>, Matthew A Brown<sup>17</sup>, Juan P Casas<sup>18</sup>, Audrey Duncanson<sup>19</sup>, Janusz Jankowski<sup>20</sup>, Hugh S Markus<sup>21</sup>, Christopher G Mathew<sup>22</sup>, Colin NA Palmer<sup>23</sup>, Robert Plomin<sup>24</sup>, Anna Rautanen<sup>1</sup>, Stephen J Sawcer<sup>25</sup>, Richard C Trembath<sup>22</sup>, Ananth C Viswanathan<sup>26</sup>, Nicholas W Wood<sup>27</sup>, Jennifer Stone<sup>28</sup>, Ed Scolnick<sup>28,</sup>, Shaun Purcell<sup>28</sup>, Pamela Sklar<sup>28</sup>, SGENE+ Consortium, Stephan Ripke<sup>29</sup>, James Walters<sup>29</sup>, Michael J Owen<sup>29</sup>, Michael C O’Donovan<sup>29</sup>, Schizophrenia Working Group of the Psychiatric GWAS Consortium, Leena Peltonen<sup>5,†</sup>, Gil McVean<sup>4</sup>, Ken S Kendler<sup>2</sup>, Michael Gill<sup>3</sup>, Peter Donnelly<sup>1,^</sup>, Aiden Corvin<sup>3,^</sup></p><p id="P43">*Authors contributed equally to this work</p><p id="P44">^Authors jointly directed this work</p><p id="P45">†Deceased</p><p id="P46"><sup>1</sup>Wellcome Trust Centre for Human Genetics, Oxford, UK; <sup>2</sup>Depts of Psychiatry and Human Genetics, Virginia Institute of Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA; <sup>3</sup>Dept of Psychiatry &amp; Neuropsychiatric Genetics Group, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland; <sup>4</sup>Dept Statistics, University of Oxford, Oxford, UK; <sup>5</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK; <sup>6</sup>St Patrick’s University Hospital, James St., Dublin, Ireland; <sup>7</sup>Dept of Psychiatry, University College Cork, Cork, Ireland; <sup>8</sup>Dept of Psychiatry, National University of Ireland, Galway, Galway, Ireland; <sup>9</sup>Dept of Psychiatry, RCSI Education and Research Centre, Beaumont Hospital, Dublin, Ireland; <sup>10</sup>DETECT Early Intervention in Psychosis Services, Dun Laoghaire, Co. Dublin, Ireland; <sup>11</sup>Dept of Psychiatry, Queen’s University, Belfast, Northern Ireland, UK; <sup>2</sup>Dept Statistics, University of Oxford, Oxford, UK; <sup>12</sup>Molecular and Cellular Therapeutics, Royal College of Surgeons in Ireland, Dublin, Ireland; <sup>13</sup>Health Research Board, Dublin 2, Ireland; <sup>14</sup>Telethon Institute for Child Health Research, Centre for Child Health Research, University of Western Australia, Subiaco, Western Australia; <sup>15</sup>Cambridge Institute for Medical Research, University of Cambridge School of Clinical Medicine, Cambridge, UK; <sup>16</sup>Department of Psychosis Studies, NIHR Biomedical Research Centre for Mental Health at the Institute of Psychiatry, King’s College London and The South London and Maudsley NHS Foundation Trust, Denmark Hill, London, UK; <sup>17</sup>University of Queensland Diamantina Institute, Immunology and Metabolic Medicine, Princess Alexandra Hospital, University of Queensland, Brisbane, Queensland, Australia; <sup>18</sup>Dept Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, London and Dept Epidemiology and Public Health, University College London, UK; <sup>19</sup>Molecular and Physiological Sciences, The Wellcome Trust, London, UK; <sup>20</sup>Centre for Digestive Diseases, Queen Mary University of London, London, UK and Digestive Diseases Centre, Leicester Royal Infirmary, Leicester, UK and Department of Clinical Pharmacology, Old Road Campus, University of Oxford, Oxford, UK; <sup>21</sup>Clinical Neurosciences, St George’s University of London, London, UK; <sup>22</sup>King’s College London Dept Medical and Molecular Genetics, School of Medicine, Guy’s Hospital, London, UK; <sup>23</sup>Biomedical Research Centre, Ninewells Hospital and Medical School, Dundee, UK; <sup>24</sup>King’s College London Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, Denmark Hill, London, UK; <sup>25</sup>University of Cambridge Dept Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK; <sup>26</sup>NIHR Biomedical Research Centre for Ophthalmology, Moorfields Eye Hospital NHS Foundation Trust and UCL Institute of Ophthalmology, London, UK; <sup>27</sup>Dept Molecular Neuroscience, Institute of Neurology, Queen Square, London, UK; <sup>28</sup>Broad Institute and Center for Human Genetics Research of Massachusetts General Hospital, Boston, MA, USA; <sup>29</sup>MRC Centre for Neuropsychiatric Genetics and Genomics, School of Medicine, Cardiff University, Heath Park, Cardiff, UK.</p><p id="P47"><strong>Membership of Wellcome Trust Case Control Consortium 2:</strong></p><p id="P48"><span class="underline">Management Committee:</span> Peter Donnelly (Chair)<sup>1,2</sup>, Ines Barroso (Deputy Chair)<sup>3</sup>, Jenefer M Blackwell<sup>4, 5</sup>, Elvira Bramon<sup>6</sup>, Matthew A Brown<sup>7</sup>, Juan P Casas<sup>8</sup>, Aiden Corvin<sup>9</sup>, Panos Deloukas<sup>3</sup>, Audrey Duncanson<sup>10</sup>, Janusz Jankowski<sup>11</sup>, Hugh S Markus<sup>12</sup>, Christopher G Mathew<sup>13</sup>, Colin NA Palmer<sup>14</sup>, Robert Plomin<sup>15</sup>, Anna Rautanen<sup>1</sup>, Stephen J Sawcer<sup>16</sup>, Richard C Trembath<sup>13</sup>, Ananth C Viswanathan<sup>17</sup>, Nicholas W Wood<sup>18</sup></p><p id="P49"><span class="underline">Data and Analysis Group:</span> Chris C A Spencer<sup>1</sup>, Gavin Band<sup>1</sup>, Céline Bellenguez<sup>1</sup>, Alexander Dilthey<sup>2</sup>, Colin Freeman<sup>1</sup>, Garrett Hellenthal<sup>1</sup>, Eleni Giannoulatou<sup>1</sup>, Gil McVean<sup>2</sup>, Loukas Moutsianas<sup>2</sup>, Matti Pirinen<sup>1</sup>, Richard Pearson<sup>1</sup>, Amy Strange<sup>1</sup>, Zhan Su<sup>1</sup>, Avazeh Tashakkori-Ghanbaria<sup>3</sup>, Damjan Vukcevic<sup>1</sup>, Peter Donnelly<sup>1,2</sup></p><p id="P50"><span class="underline">DNA, Genotyping, Data QC and Informatics Group:</span> Cordelia Langford<sup>3</sup>, Sarah E Hunt<sup>3</sup>, Sarah Edkins<sup>3</sup>, Rhian Gwilliam<sup>3</sup>, Hannah Blackburn<sup>3</sup>, Suzannah J Bumpstead<sup>3</sup>, Serge Dronov<sup>3</sup>, Matthew Gillman<sup>3</sup>, Emma Gray<sup>3</sup>, Naomi Hammond<sup>3</sup>, Alagurevathi Jayakumar<sup>3</sup>, Owen T McCann<sup>3</sup>, Jennifer Liddle<sup>3</sup>, Simon C Potter<sup>3</sup>, Radhi Ravindrarajah<sup>3</sup>, Michelle Ricketts<sup>3</sup>, Matthew Waller<sup>3</sup>, Paul Weston<sup>3</sup>, Sara Widaa<sup>3</sup>, Pamela Whittaker<sup>3</sup>, Ines Barroso<sup>3</sup>, Panos Deloukas<sup>3</sup></p><p id="P51"><span class="underline">Publications Committee: Christopher G Mathew (Chair)</span><sup>13</sup>, Jenefer M Blackwell<sup>4,5</sup>, Matthew A Brown<sup>7</sup>, Aiden Corvin<sup>9</sup>, Chris C A Spencer<sup>1</sup></p><p id="P52"><strong>Membership of the Irish Schizophrenia Genomics Consortium:</strong></p><p id="P53">Brien P Riley<sup>18</sup>, Derek W Morris<sup>9</sup>, Colm T O’Dushlaine<sup>9</sup>, Brion S Maher<sup>18</sup>, Paul Cormican<sup>9</sup>, Elaine M Kenny<sup>9</sup>, Brandon Wormley<sup>18</sup>, Gary Donohoe<sup>9</sup>, Emma Quinn<sup>9</sup>, Roisin Judge<sup>9</sup>, Kim Coleman<sup>9</sup>, Daniela Tropea<sup>9</sup>, Siobhan Roche<sup>19</sup>, Liz Cummings<sup>9</sup>, Eric Kelleher<sup>9</sup>, Patrick McKeon<sup>19</sup>, Ted Dinan<sup>20</sup>, Colm McDonald<sup>21</sup>, Kieran C Murphy<sup>22</sup>, Eadbhard O’Callaghan<sup>23</sup>, Francis A O’Neill<sup>24</sup>, John L Waddington<sup>22</sup>, Dermot Walsh<sup>25</sup>, Jennifer Stone<sup>26</sup>, Ed Scolnick<sup>26,</sup>, Shaun Purcell<sup>26</sup>, Pamela Sklar<sup>26</sup>, SGENE+ Consortium, Stephan Ripke<sup>26</sup>, James Walters<sup>27</sup>, Michael J Owen<sup>27</sup>, Michael C O’Donovan<sup>27</sup>, Schizophrenia Working Group of the Psychiatric GWAS Consortium, Ken S Kendler<sup>18</sup>, Michael Gill<sup>9</sup>, Aiden Corvin<sup>9</sup></p><p id="P54">Membership lists of SGENE+ Consortium and Schizophrenia Working Group of the Psychiatric GWAS Consortium are provided in the Supplement.</p><p id="P55"><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK; <sup>2</sup> Dept Statistics, University of Oxford, Oxford, UK; <sup>3</sup> Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK; <sup>4</sup> Telethon Institute for Child Health Research, Centre for Child Health Research, University of Western Australia, Subiaco, Western Australia; <sup>5</sup> Cambridge Institute for Medical Research, University of Cambridge School of Clinical Medicine, Cambridge, UK; <sup>6</sup> Department of Psychosis Studies, NIHR Biomedical Research Centre for Mental Health at the Institute of Psychiatry, King’s College London and The South London and Maudsley NHS Foundation Trust, Denmark Hill, London, UK; <sup>7</sup> University of Queensland Diamantina Institute, Brisbane, Queensland, Australia; <sup>8</sup> Dept Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, London, and Dept Epidemiology and Public Health, University College London, UK; <sup>9</sup> Neuropsychiatric Genetics Research Group, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland; <sup>10</sup> Molecular and Physiological Sciences, The Wellcome Trust, London, UK; <sup>11</sup> Department of Oncology, Old Road Campus, University of Oxford, Oxford, UK, Digestive Diseases Centre, Leicester Royal Infirmary, Leicester, UK and Centre for Digestive Diseases, Queen Mary University of London, London, UK; <sup>12</sup> Clinical Neurosciences, St George’s University of London, London, UK; <sup>13</sup> King’s College London Dept Medical and Molecular Genetics, King’s Health Partners, Guy’s Hospital, London, UK; <sup>14</sup> Biomedical Research Centre, Ninewells Hospital and Medical School, Dundee, UK; <sup>15</sup> King’s College London Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, Denmark Hill, London, UK; <sup>16</sup> University of Cambridge Dept Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK; <sup>17</sup> NIHR Biomedical Research Centre for Ophthalmology, Moorfields Eye Hospital NHS Foundation Trust and UCL Institute of Ophthalmology, London, UK; <sup>18</sup> Dept Molecular Neuroscience, Institute of Neurology, Queen Square, London, UK, and Depts of Psychiatry and Human Genetics, Virginia Institute of Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA; <sup>19</sup> St Patrick’s University Hospital, James St., Dublin, Ireland; <sup>20</sup> Dept of Psychiatry, University College Cork, Cork, Ireland; <sup>21</sup> Dept of Psychiatry, National University of Ireland, Galway, Galway, Ireland; <sup>22</sup> Dept of Psychiatry, RCSI Education and Research Centre, Beaumont Hospital, Dublin, Ireland; <sup>23</sup> DETECT Early Intervention in Psychosis Services, Dun Laoghaire, Co. Dublin, Ireland; <sup>24</sup> Dept of Psychiatry, Queen’s University, Belfast, Northern Ireland, UK; <sup>25</sup> Health Research Board, Dublin, Ireland; <sup>26</sup> Broad Institute and Center for Human Genetics Research of Massachusetts General Hospital, Boston, MA, USA; <sup>27</sup> MRC Centre for Neuropsychiatric Genetics and Genomics, School of Medicine, Cardiff University, Heath Park, Cardiff, UK.</p></div><div id="idm140021747091056" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140021747091056title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"> </p><p id="P40" class="p p-first-last">The authors report no biomedical financial interests or potential conflicts of interest.</p><p></p><p class="fn sec" id="FN2"></p><p id="P41" class="p p-first-last">Supplementary material cited in this article is available online.</p><p></p></div></div><div id="idm140021754727984" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140021754727984title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Cardno AG, Gottesman II. Twin studies of schizophrenia: From bow-and-arrow concordances to star wars Mx and functional genomics. <span><span class="ref-journal">Am J Med Genet. </span>2000;<span class="ref-vol">97</span>:12–17.</span> [<a href="/pubmed/10813800" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Twin+studies+of+schizophrenia:+From+bow-and-arrow+concordances+to+star+wars+Mx+and+functional+genomics&amp;author=AG+Cardno&amp;author=II+Gottesman&amp;volume=97&amp;publication_year=2000&amp;pages=12-17&amp;pmid=10813800&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Tiihonen J, Lonnqvist J, Wahlbeck K, Klaukka T, Niskanen L, Tanskanen A, Haukka J. 11-year follow-up of mortality in patients with schizophrenia: A population-based cohort study (FIN11 study) <span><span class="ref-journal">Lancet. </span>2009;<span class="ref-vol">374</span>:620–627.</span> [<a href="/pubmed/19595447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=11-year+follow-up+of+mortality+in+patients+with+schizophrenia:+A+population-based+cohort+study+(FIN11+study)&amp;author=J+Tiihonen&amp;author=J+Lonnqvist&amp;author=K+Wahlbeck&amp;author=T+Klaukka&amp;author=L+Niskanen&amp;volume=374&amp;publication_year=2009&amp;pages=620-627&amp;pmid=19595447&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">International Schizophrenia Consortium. Purcell SM, Wray NR, Stone JL, Visscher PM, O’Donovan MC, et al.  Common polygenic variation contributes to risk of schizophrenia and bipolar disorder. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:748–752.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia+and+bipolar+disorder&amp;author=SM+Purcell&amp;author=NR+Wray&amp;author=JL+Stone&amp;author=PM+Visscher&amp;author=MC+O’Donovan&amp;volume=460&amp;publication_year=2009&amp;pages=748-752&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">O’Donovan MC, Craddock N, Norton N, Williams H, Peirce T, Moskvina V, et al.  Identification of loci associated with schizophrenia by genome-wide association and follow-up. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1053–1055.</span> [<a href="/pubmed/18677311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+loci+associated+with+schizophrenia+by+genome-wide+association+and+follow-up&amp;author=MC+O’Donovan&amp;author=N+Craddock&amp;author=N+Norton&amp;author=H+Williams&amp;author=T+Peirce&amp;volume=40&amp;publication_year=2008&amp;pages=1053-1055&amp;pmid=18677311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Stefansson H, Rujescu D, Cichon S, Pietilainen OP, Ingason A, Steinberg S, et al.  Large recurrent microdeletions associated with schizophrenia. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:232–236.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687075/">PMC free article</a>]</span> [<a href="/pubmed/18668039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Large+recurrent+microdeletions+associated+with+schizophrenia&amp;author=H+Stefansson&amp;author=D+Rujescu&amp;author=S+Cichon&amp;author=OP+Pietilainen&amp;author=A+Ingason&amp;volume=455&amp;publication_year=2008&amp;pages=232-236&amp;pmid=18668039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Shi J, Levinson DF, Duan J, Sanders AR, Zheng Y, Pe’er I, et al.  Common variants on chromosome 6p22.1 are associated with schizophrenia. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:753–757.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775422/">PMC free article</a>]</span> [<a href="/pubmed/19571809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+variants+on+chromosome+6p22.1+are+associated+with+schizophrenia&amp;author=J+Shi&amp;author=DF+Levinson&amp;author=J+Duan&amp;author=AR+Sanders&amp;author=Y+Zheng&amp;volume=460&amp;publication_year=2009&amp;pages=753-757&amp;pmid=19571809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Athanasiu L, Mattingsdal M, Kahler AK, Brown A, Gustafsson O, Agartz I, et al.  Gene variants associated with schizophrenia in a Norwegian genome-wide study are replicated in a large European cohort. <span><span class="ref-journal">J Psychiatr Res. </span>2010;<span class="ref-vol">44</span>:748–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3224994/">PMC free article</a>]</span> [<a href="/pubmed/20185149" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Psychiatr+Res&amp;title=Gene+variants+associated+with+schizophrenia+in+a+Norwegian+genome-wide+study+are+replicated+in+a+large+European+cohort&amp;author=L+Athanasiu&amp;author=M+Mattingsdal&amp;author=AK+Kahler&amp;author=A+Brown&amp;author=O+Gustafsson&amp;volume=44&amp;publication_year=2010&amp;pages=748-753&amp;pmid=20185149&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">International Schizophrenia Consortium. Rare chromosomal deletions and duplications increase risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:237–241.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912847/">PMC free article</a>]</span> [<a href="/pubmed/18668038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Rare+chromosomal+deletions+and+duplications+increase+risk+of+schizophrenia&amp;volume=455&amp;publication_year=2008&amp;pages=237-241&amp;pmid=18668038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Ripke S, Sanders AR, Kendler KS, Levinson DF, Sklar P, Holmans PA, et al.  Genome-wide association study identifies five new schizophrenia loci. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:969–976.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+five+new+schizophrenia+loci&amp;author=S+Ripke&amp;author=AR+Sanders&amp;author=KS+Kendler&amp;author=DF+Levinson&amp;author=P+Sklar&amp;volume=43&amp;publication_year=2011&amp;pages=969-976&amp;pmid=21926974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Shi Y, Li Z, Xu Q, Wang T, Li T, Shen J, et al.  Common variants on 8p12 and 1q24.2 confer risk of schizophrenia. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:1224–1227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3773910/">PMC free article</a>]</span> [<a href="/pubmed/22037555" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+on+8p12+and+1q24.2+confer+risk+of+schizophrenia&amp;author=Y+Shi&amp;author=Z+Li&amp;author=Q+Xu&amp;author=T+Wang&amp;author=T+Li&amp;volume=43&amp;publication_year=2011&amp;pages=1224-1227&amp;pmid=22037555&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Yue WH, Wang HF, Sun LD, Tang FL, Liu ZH, Zhang HX, et al.  Genome-wide association study identifies a susceptibility locus for schizophrenia in Han Chinese at 11p11.2. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:1228–1231.</span> [<a href="/pubmed/22037552" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+susceptibility+locus+for+schizophrenia+in+Han+Chinese+at+11p11.2&amp;author=WH+Yue&amp;author=HF+Wang&amp;author=LD+Sun&amp;author=FL+Tang&amp;author=ZH+Liu&amp;volume=43&amp;publication_year=2011&amp;pages=1228-1231&amp;pmid=22037552&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Stefansson H, Ophoff RA, Steinberg S, Andreassen OA, Cichon S, Rujescu D, et al.  Common variants conferring risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:744–747.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3077530/">PMC free article</a>]</span> [<a href="/pubmed/19571808" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+variants+conferring+risk+of+schizophrenia&amp;author=H+Stefansson&amp;author=RA+Ophoff&amp;author=S+Steinberg&amp;author=OA+Andreassen&amp;author=S+Cichon&amp;volume=460&amp;publication_year=2009&amp;pages=744-747&amp;pmid=19571808&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">The 1000 Genomes Project Consortium. A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">First MB, Spitzer RL, Robert L, Gibbon M, Williams JBW.  <span class="ref-journal">Structured Clinical Interview for DSM-IV-TR Axis I Disorders, Research Version, Patient Edition.</span> New York: Biometrics Research, New York State Psychiatric Institute; 2002.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Structured+Clinical+Interview+for+DSM-IV-TR+Axis+I+Disorders,+Research+Version,+Patient+Edition&amp;author=MB+First&amp;author=RL+Spitzer&amp;author=L+Robert&amp;author=M+Gibbon&amp;author=JBW+Williams&amp;publication_year=2002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Endicott J, Spitzer RL. A diagnostic interview: The schedule for affective disorders and schizophrenia. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1978;<span class="ref-vol">35</span>:837–844.</span> [<a href="/pubmed/678037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=A+diagnostic+interview:+The+schedule+for+affective+disorders+and+schizophrenia&amp;author=J+Endicott&amp;author=RL+Spitzer&amp;volume=35&amp;publication_year=1978&amp;pages=837-844&amp;pmid=678037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">World Health Organization. <span class="ref-journal">Schedules for Clinical Assessment in Neuropsychiatry (Version 1 0)</span> Geneva: WHO; 1992.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Schedules+for+Clinical+Assessment+in+Neuropsychiatry+(Version+1+0)&amp;publication_year=1992&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">UK IBD Genetics Consortium. Barrett JC, Lee JC, Lees CW, Prescott NJ, Anderson CA, et al.  Genome-wide association study of ulcerative colitis identifies three new susceptibility loci, including the HNF4A region. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1330–1334.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2812019/">PMC free article</a>]</span> [<a href="/pubmed/19915572" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+ulcerative+colitis+identifies+three+new+susceptibility+loci,+including+the+HNF4A+region&amp;author=JC+Barrett&amp;author=JC+Lee&amp;author=CW+Lees&amp;author=NJ+Prescott&amp;author=CA+Anderson&amp;volume=41&amp;publication_year=2009&amp;pages=1330-1334&amp;pmid=19915572&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Bellenguez C, Strange A, Freeman C, Donnelly P, Spencer CC Wellcome Trust Case Control Consortium. A robust clustering algorithm for identifying problematic samples in genome-wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:134–135.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3244763/">PMC free article</a>]</span> [<a href="/pubmed/22057162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+robust+clustering+algorithm+for+identifying+problematic+samples+in+genome-wide+association+studies&amp;author=C+Bellenguez&amp;author=A+Strange&amp;author=C+Freeman&amp;author=P+Donnelly&amp;author=CC+Spencer&amp;volume=28&amp;publication_year=2012&amp;pages=134-135&amp;pmid=22057162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Price AL, Zaitlen NA, Reich D, Patterson N. New approaches to population stratification in genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>:459–463.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2975875/">PMC free article</a>]</span> [<a href="/pubmed/20548291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=New+approaches+to+population+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NA+Zaitlen&amp;author=D+Reich&amp;author=N+Patterson&amp;volume=11&amp;publication_year=2010&amp;pages=459-463&amp;pmid=20548291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Yang J, Weedon MN, Purcell S, Lettre G, Estrada K, Willer CJ, et al.  Genomic inflation factors under polygenic inheritance. <span><span class="ref-journal">Eur J Hum Genet. </span>2011;<span class="ref-vol">19</span>:807–812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3137506/">PMC free article</a>]</span> [<a href="/pubmed/21407268" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Genomic+inflation+factors+under+polygenic+inheritance&amp;author=J+Yang&amp;author=MN+Weedon&amp;author=S+Purcell&amp;author=G+Lettre&amp;author=K+Estrada&amp;volume=19&amp;publication_year=2011&amp;pages=807-812&amp;pmid=21407268&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span></span> e1000529. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Dilthey AT, Moutsianas L, Leslie S, McVean G. HLA*IMP--an integrated framework for imputing classical HLA alleles from SNP genotypes. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:968–972.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3065693/">PMC free article</a>]</span> [<a href="/pubmed/21300701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=HLA*IMP--an+integrated+framework+for+imputing+classical+HLA+alleles+from+SNP+genotypes&amp;author=AT+Dilthey&amp;author=L+Moutsianas&amp;author=S+Leslie&amp;author=G+McVean&amp;volume=27&amp;publication_year=2011&amp;pages=968-972&amp;pmid=21300701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Green EK, Grozeva D, Jones I, Jones L, Kirov G, Caesar S, et al.  The bipolar disorder risk allele at CACNA1C also confers risk of recurrent major depression and of schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2010;<span class="ref-vol">15</span>:1016–1022.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3011210/">PMC free article</a>]</span> [<a href="/pubmed/19621016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=The+bipolar+disorder+risk+allele+at+CACNA1C+also+confers+risk+of+recurrent+major+depression+and+of+schizophrenia&amp;author=EK+Green&amp;author=D+Grozeva&amp;author=I+Jones&amp;author=L+Jones&amp;author=G+Kirov&amp;volume=15&amp;publication_year=2010&amp;pages=1016-1022&amp;pmid=19621016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Ferreira MA, O’Donovan MC, Meng YA, Jones IR, Ruderfer DM, Jones L, et al.  Collaborative genome-wide association analysis supports a role for ANK3 and CACNA1C in bipolar disorder. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1056–1058.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2703780/">PMC free article</a>]</span> [<a href="/pubmed/18711365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Collaborative+genome-wide+association+analysis+supports+a+role+for+ANK3+and+CACNA1C+in+bipolar+disorder&amp;author=MA+Ferreira&amp;author=MC+O’Donovan&amp;author=YA+Meng&amp;author=IR+Jones&amp;author=DM+Ruderfer&amp;volume=40&amp;publication_year=2008&amp;pages=1056-1058&amp;pmid=18711365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Curtis D, Vine AE, McQuillin A, Bass NJ, Pereira A, Kandaswamy R, et al.  Case-case genome-wide association analysis shows markers differentially associated with schizophrenia and bipolar disorder and implicates calcium channel genes. <span><span class="ref-journal">Psychiatr Genet. </span>2011;<span class="ref-vol">21</span>:1–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3024533/">PMC free article</a>]</span> [<a href="/pubmed/21057379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychiatr+Genet&amp;title=Case-case+genome-wide+association+analysis+shows+markers+differentially+associated+with+schizophrenia+and+bipolar+disorder+and+implicates+calcium+channel+genes&amp;author=D+Curtis&amp;author=AE+Vine&amp;author=A+McQuillin&amp;author=NJ+Bass&amp;author=A+Pereira&amp;volume=21&amp;publication_year=2011&amp;pages=1-4&amp;pmid=21057379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Chen Y, Lu J, Pan H, Zhang Y, Wu H, Xu K, et al.  Association between genetic variation of CACNA1H and childhood absence epilepsy. <span><span class="ref-journal">Ann Neurol. </span>2003;<span class="ref-vol">54</span>:239–243.</span> [<a href="/pubmed/12891677" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Association+between+genetic+variation+of+CACNA1H+and+childhood+absence+epilepsy&amp;author=Y+Chen&amp;author=J+Lu&amp;author=H+Pan&amp;author=Y+Zhang&amp;author=H+Wu&amp;volume=54&amp;publication_year=2003&amp;pages=239-243&amp;pmid=12891677&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Patel C, Cooper-Charles L, McMullan DJ, Walker JM, Davison V, Morton J. Translocation breakpoint at 7q31 associated with tics: Further evidence for IMMP2L as a candidate gene for Tourette syndrome. <span><span class="ref-journal">Eur J Hum Genet. </span>2011;<span class="ref-vol">19</span>:634–639.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3110039/">PMC free article</a>]</span> [<a href="/pubmed/21386874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Translocation+breakpoint+at+7q31+associated+with+tics:+Further+evidence+for+IMMP2L+as+a+candidate+gene+for+Tourette+syndrome&amp;author=C+Patel&amp;author=L+Cooper-Charles&amp;author=DJ+McMullan&amp;author=JM+Walker&amp;author=V+Davison&amp;volume=19&amp;publication_year=2011&amp;pages=634-639&amp;pmid=21386874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Petek E, Windpassinger C, Vincent JB, Cheung J, Boright AP, Scherer SW, et al.  Disruption of a novel gene (IMMP2L) by a breakpoint in 7q31 associated with Tourette syndrome. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">68</span>:848–858.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1275638/">PMC free article</a>]</span> [<a href="/pubmed/11254443" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Disruption+of+a+novel+gene+(IMMP2L)+by+a+breakpoint+in+7q31+associated+with+Tourette+syndrome&amp;author=E+Petek&amp;author=C+Windpassinger&amp;author=JB+Vincent&amp;author=J+Cheung&amp;author=AP+Boright&amp;volume=68&amp;publication_year=2001&amp;pages=848-858&amp;pmid=11254443&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Maestrini E, Pagnamenta AT, Lamb JA, Bacchelli E, Sykes NH, Sousa I, et al.  High-density SNP association study and copy number variation analysis of the AUTS1 and AUTS5 loci implicate the IMMP2L-DOCK4 gene region in autism susceptibility. <span><span class="ref-journal">Mol Psychiatry. </span>2010;<span class="ref-vol">15</span>:954–968.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2934739/">PMC free article</a>]</span> [<a href="/pubmed/19401682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=High-density+SNP+association+study+and+copy+number+variation+analysis+of+the+AUTS1+and+AUTS5+loci+implicate+the+IMMP2L-DOCK4+gene+region+in+autism+susceptibility&amp;author=E+Maestrini&amp;author=AT+Pagnamenta&amp;author=JA+Lamb&amp;author=E+Bacchelli&amp;author=NH+Sykes&amp;volume=15&amp;publication_year=2010&amp;pages=954-968&amp;pmid=19401682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Pagnamenta AT, Bacchelli E, de Jonge MV, Mirza G, Scerri TS, Minopoli F, et al.  Characterization of a family with rare deletions in CNTNAP5 and DOCK4 suggests novel risk loci for autism and dyslexia. <span><span class="ref-journal">Biol Psychiatry. </span>2010;<span class="ref-vol">68</span>:320–328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2941017/">PMC free article</a>]</span> [<a href="/pubmed/20346443" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Characterization+of+a+family+with+rare+deletions+in+CNTNAP5+and+DOCK4+suggests+novel+risk+loci+for+autism+and+dyslexia&amp;author=AT+Pagnamenta&amp;author=E+Bacchelli&amp;author=MV+de+Jonge&amp;author=G+Mirza&amp;author=TS+Scerri&amp;volume=68&amp;publication_year=2010&amp;pages=320-328&amp;pmid=20346443&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Nekrasova T, Jobes ML, Ting JH, Wagner GC, Minden A. Targeted disruption of the Pak5 and Pak6 genes in mice leads to deficits in learning and locomotion. <span><span class="ref-journal">Dev Biol. </span>2008;<span class="ref-vol">322</span>:95–108.</span> [<a href="/pubmed/18675265" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Biol&amp;title=Targeted+disruption+of+the+Pak5+and+Pak6+genes+in+mice+leads+to+deficits+in+learning+and+locomotion&amp;author=T+Nekrasova&amp;author=ML+Jobes&amp;author=JH+Ting&amp;author=GC+Wagner&amp;author=A+Minden&amp;volume=322&amp;publication_year=2008&amp;pages=95-108&amp;pmid=18675265&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Zhao W, Yang J, Shi W, Wu X, Shao B, Wu Q, et al.  Upregulation of p21-activated Kinase 6 in rat brain cortex after traumatic brain injury. <span><span class="ref-journal">J Mol Histol. </span>2011;<span class="ref-vol">42</span>:195–203.</span> [<a href="/pubmed/21541790" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Histol&amp;title=Upregulation+of+p21-activated+Kinase+6+in+rat+brain+cortex+after+traumatic+brain+injury&amp;author=W+Zhao&amp;author=J+Yang&amp;author=W+Shi&amp;author=X+Wu&amp;author=B+Shao&amp;volume=42&amp;publication_year=2011&amp;pages=195-203&amp;pmid=21541790&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Leslie S, Donnelly P, McVean G. A statistical method for predicting classical HLA alleles from SNP data. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">82</span>:48–56.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253983/">PMC free article</a>]</span> [<a href="/pubmed/18179884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+statistical+method+for+predicting+classical+HLA+alleles+from+SNP+data&amp;author=S+Leslie&amp;author=P+Donnelly&amp;author=G+McVean&amp;volume=82&amp;publication_year=2008&amp;pages=48-56&amp;pmid=18179884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Ripke S, Sanders AR, Kendler KS, Levinson DF, Sklar P, Holmans PA, et al.  Genome-wide association study identifies five new schizophrenia loci. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:969–976.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+five+new+schizophrenia+loci&amp;author=S+Ripke&amp;author=AR+Sanders&amp;author=KS+Kendler&amp;author=DF+Levinson&amp;author=P+Sklar&amp;volume=43&amp;publication_year=2011&amp;pages=969-976&amp;pmid=21926974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Shatz CJ. MHC class I: An unexpected role in neuronal plasticity. <span><span class="ref-journal">Neuron. </span>2009;<span class="ref-vol">64</span>:40–45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2773547/">PMC free article</a>]</span> [<a href="/pubmed/19840547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=MHC+class+I:+An+unexpected+role+in+neuronal+plasticity&amp;author=CJ+Shatz&amp;volume=64&amp;publication_year=2009&amp;pages=40-45&amp;pmid=19840547&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Needleman LA, Liu XB, El-Sabeawy F, Jones EG, McAllister AK. MHC class I molecules are present both pre- and postsynaptically in the visual cortex during postnatal development and in adulthood. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2011;<span class="ref-vol">107</span>:16999–17004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947898/">PMC free article</a>]</span> [<a href="/pubmed/20837535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=MHC+class+I+molecules+are+present+both+pre-+and+postsynaptically+in+the+visual+cortex+during+postnatal+development+and+in+adulthood&amp;author=LA+Needleman&amp;author=XB+Liu&amp;author=F+El-Sabeawy&amp;author=EG+Jones&amp;author=AK+McAllister&amp;volume=107&amp;publication_year=2011&amp;pages=16999-17004&amp;pmid=20837535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3529467&amp;issue-id=217545&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140021757167312title">Abstract</a></li><li><a href="#S5title">Methods and Materials</a></li><li><a href="#S21title">Results</a></li><li><a href="#S22title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S23title">Acknowledgments</a></li><li><a href="#APP1title">Appendix A</a></li><li><a href="#idm140021747091056title">Footnotes</a></li><li><a href="#idm140021754727984title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3529467/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3529467/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3529467/pdf/nihms428997.pdf">PDF (1.4M)</a></li> | <li><a href="#" data-citationid="PMC3529467" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3529467%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3529467%2F&amp;text=Genome-Wide%20Association%20Study%20Implicates%20HLA-C*01%3A%2002%20as%20a%20Risk%20Factor%20at%20the%20Major%20Histocompatibility%20Complex%20Locus%20in%20Schizophrenia" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3529467%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22883433" data-aiid="3529467" data-aid="3529467" data-iid="217545" data-domainid="319" data-domain="nihpa" data-accid="PMC3529467" data-md5="d835763672dc4ec73691c154a588d2f9"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:22:36-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88662DDD53E591000000000A4106C2&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3529467%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3529467/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>