<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3276283/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association study identifies a potential novel gene locus for keratoconus, one of the commonest causes for corneal transplantation in developed countries</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3276283/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="A genome-wide association study identifies a potential novel gene locus for keratoconus, one of the commonest causes for corneal transplantation in developed countries"><meta name="citation_authors" content="Xiaohui Li, Yelena Bykhovskaya, Talin Haritunians, David Siscovick, Anthony Aldave, Loretta Szczotka-Flynn, Sudha K. Iyengar, Jerome I. Rotter, Kent D. Taylor, Yaron S. Rabinowitz"><meta name="citation_date" content="15 January 2012"><meta name="citation_issue" content="2"><meta name="citation_volume" content="21"><meta name="citation_firstpage" content="421"><meta name="citation_doi" content="10.1093/hmg/ddr460"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3276283/?report=abstract"><meta name="citation_pmid" content="21979947"><meta name="DC.Title" content="A genome-wide association study identifies a potential novel gene locus for keratoconus, one of the commonest causes for corneal transplantation in developed countries"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Xiaohui Li"><meta name="DC.Contributor" content="Yelena Bykhovskaya"><meta name="DC.Contributor" content="Talin Haritunians"><meta name="DC.Contributor" content="David Siscovick"><meta name="DC.Contributor" content="Anthony Aldave"><meta name="DC.Contributor" content="Loretta Szczotka-Flynn"><meta name="DC.Contributor" content="Sudha K. Iyengar"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Kent D. Taylor"><meta name="DC.Contributor" content="Yaron S. Rabinowitz"><meta name="DC.Date" content="2012 Jan 15"><meta name="DC.Identifier" content="10.1093/hmg/ddr460"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association study identifies a potential novel gene locus for keratoconus, one of the commonest causes for corneal transplantation in developed countries"><meta property="og:type" content="article"><meta property="og:description" content="Keratoconus is a condition in which the cornea progressively thins over time, and is a major cause for cornea transplantation. To identify keratoconus susceptibility regions, we performed a comprehensive genome-wide association study (GWAS) using a discovery ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3276283/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3276283/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3276283/pdf/ddr460.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88B888DD532F3100000000024C0197.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3276283%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3276283</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2012 Jan 15; 21(2): 421–429. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Oct 6. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddr460" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddr460</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3276283</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21979947">21979947</a></div></div></div></div><h1 class="content-title">A genome-wide association study identifies a potential novel gene locus for keratoconus, one of the commonest causes for corneal transplantation in developed countries</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Li%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002469286016" co-class="co-affbox">Xiaohui Li</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Bykhovskaya%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002468447408" co-class="co-affbox">Yelena Bykhovskaya</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Haritunians%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002467870624" co-class="co-affbox">Talin Haritunians</a>,<sup>2</sup> <a href="/pubmed/?term=Siscovick%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002467868896" co-class="co-affbox">David Siscovick</a>,<sup>4</sup> <a href="/pubmed/?term=Aldave%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002467866656" co-class="co-affbox">Anthony Aldave</a>,<sup>5</sup> <a href="/pubmed/?term=Szczotka-Flynn%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002467864416" co-class="co-affbox">Loretta Szczotka-Flynn</a>,<sup>6</sup> <a href="/pubmed/?term=Iyengar%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002467862176" co-class="co-affbox">Sudha K. Iyengar</a>,<sup>6</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002467860000" co-class="co-affbox">Jerome I. Rotter</a>,<sup>2</sup> <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002467857824" co-class="co-affbox">Kent D. Taylor</a>,<sup>2</sup> and  <a href="/pubmed/?term=Rabinowitz%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947" class="affpopup" co-rid="_co_idm140002467855648" co-class="co-affbox">Yaron S. Rabinowitz</a><sup>1,</sup><sup>3,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140002469286016"><h3 class="no_margin">Xiaohui Li</h3><p><sup>1</sup>Cornea Genetic Eye Institute,</p><p><sup>2</sup>Medical Genetics Institute,  and</p><div>Find articles by <a href="/pubmed/?term=Li%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Xiaohui Li</a></div></div><div id="_co_idm140002468447408"><h3 class="no_margin">Yelena Bykhovskaya</h3><p><sup>1</sup>Cornea Genetic Eye Institute,</p><p><sup>3</sup>Regenerative Medicine Institute, Cedars-Sinai Medical Center, Los Angeles, CA, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Bykhovskaya%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Yelena Bykhovskaya</a></div></div><div id="_co_idm140002467870624"><h3 class="no_margin">Talin Haritunians</h3><p><sup>2</sup>Medical Genetics Institute,  and</p><div>Find articles by <a href="/pubmed/?term=Haritunians%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Talin Haritunians</a></div></div><div id="_co_idm140002467868896"><h3 class="no_margin">David Siscovick</h3><p><sup>4</sup>University of Washington, Seattle, WA, USA,</p><div>Find articles by <a href="/pubmed/?term=Siscovick%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">David Siscovick</a></div></div><div id="_co_idm140002467866656"><h3 class="no_margin">Anthony Aldave</h3><p><sup>5</sup>The Jules Stein Eye Institute, University of California Los Angeles, Los Angeles, CA, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Aldave%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Anthony Aldave</a></div></div><div id="_co_idm140002467864416"><h3 class="no_margin">Loretta Szczotka-Flynn</h3><p><sup>6</sup>Case Western Reserve University, Cleveland, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Szczotka-Flynn%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Loretta Szczotka-Flynn</a></div></div><div id="_co_idm140002467862176"><h3 class="no_margin">Sudha K. Iyengar</h3><p><sup>6</sup>Case Western Reserve University, Cleveland, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Iyengar%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Sudha K. Iyengar</a></div></div><div id="_co_idm140002467860000"><h3 class="no_margin">Jerome I. Rotter</h3><p><sup>2</sup>Medical Genetics Institute,  and</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Jerome I. Rotter</a></div></div><div id="_co_idm140002467857824"><h3 class="no_margin">Kent D. Taylor</h3><p><sup>2</sup>Medical Genetics Institute,  and</p><div>Find articles by <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Kent D. Taylor</a></div></div><div id="_co_idm140002467855648"><h3 class="no_margin">Yaron S. Rabinowitz</h3><p><sup>1</sup>Cornea Genetic Eye Institute,</p><p><sup>3</sup>Regenerative Medicine Institute, Cedars-Sinai Medical Center, Los Angeles, CA, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Rabinowitz%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21979947">Yaron S. Rabinowitz</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140002469818192_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140002469818192_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140002469818192_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140002469818192_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Cornea Genetic Eye Institute,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Medical Genetics Institute,  and</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Regenerative Medicine Institute, Cedars-Sinai Medical Center, Los Angeles, CA, USA,  and</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>University of Washington, Seattle, WA, USA,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>The Jules Stein Eye Institute, University of California Los Angeles, Los Angeles, CA, USA,  and</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Case Western Reserve University, Cleveland, OH, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Cornea Genetic Eye Institute, Cedars-Sinai Medical Center, 50 N La Cienega Bl., Suite 340, Beverly Hills, CA 90211, USA. Tel: +1 3104239640; Fax: +1 3102487485; Email: <a href="mailto:dev@null" data-email="gro.shsc@yztiwonibar" class="oemail">gro.shsc@yztiwonibar</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140002469818192_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Apr 29; Revised 2011 Sep 22; Accepted 2011 Oct 3.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140002469818192_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2011. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3276283/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3276283/bin/supp_21_2_421__index.html" data-ga-action="click_feat_suppl">supp_21_2_421__index.html</a><span style="color:gray"> (1.1K)</span></div><div class="small guid">GUID:&nbsp;08191ECA-322B-44EC-A46B-164D8353ED02</div>
<div><a href="/pmc/articles/PMC3276283/bin/supp_ddr460_ddr460supp.doc" data-ga-action="click_feat_suppl">supp_ddr460_ddr460supp.doc</a><span style="color:gray"> (3.4M)</span></div><div class="small guid">GUID:&nbsp;3B116A8B-0B25-4BB5-A803-4553F0EB2305</div>
</div></dd></dl></div><div id="idm140002466284512" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140002466284512title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Keratoconus is a condition in which the cornea progressively thins over time, and is a major cause for cornea transplantation. To identify keratoconus susceptibility regions, we performed a comprehensive genome-wide association study (GWAS) using a discovery and replication design. A discovery panel of 222 keratoconus Caucasian patients and 3324 Caucasian controls was genotyped using Illumina 370K beadchips. Further associated and fine-mapping single nucleotide polymorphisms (SNPs) (<em>n</em>= 4905) were genotyped in an independent replication case–control panel of 304 cases and 518 controls and a family panel of 307 subjects in 70 families. Logistic regression models implemented in PLINK were performed to test associations in case–control samples with and without principal component (PC) adjustments. Generalized estimation equation models accounting for familial correlations implemented in GWAF were used for association testing in families. No genome-wide associations were identified in the discovery GWAS panel. From the initial testing without adjustments for PCs, the top three SNPs located at 3p26 (rs6442925), 2q21.3 (rs4954218) and 19q13.3 (rs1428642) were identified with unadjusted <em>P</em>-values of 6.5 × 10<sup>−8</sup>, 2.4 × 10<sup>−7</sup> and 3.1 × 10<sup>−7</sup>, respectively. After adjustments for PCs, rs1428642 became the most significant through the genome with a <em>P</em>-value of 1.4 × 10<sup>−6</sup>, while rs6442925 and rs4954218 were less significant (<em>P</em>= 1.9 × 10<sup>−5</sup> and 2.6 × 10<sup>−4</sup>). SNP rs4954218 was confirmed in two independent replication panels with <em>P</em>-values of 0.004 and 0.009, respectively. Meta-analysis revealed a highest association at rs4954218 with adjusted <em>P</em>= 1.6 × 10<sup>−7</sup> (unadjusted <em>P</em>= 1.2 × 10<sup>−9</sup>). These findings suggest SNP rs4954218, located near the <em>RAB3GAP1</em> gene, previously reported to be associated with corneal malformation, is a potential susceptibility locus for keratoconus.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Keratoconus is a clinical term used to describe a condition in which the cornea assumes a conical shape as a result of non-inflammatory thinning and protrusion. It typically commences at puberty and progresses intermittently until the mid-thirties at which point in time it usually arrests. In ∼20% of cases, it progresses to the point of legal blindness and which can be treated only by corneal transplantation (<a href="#DDR460C1" rid="DDR460C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915831">1</a>–<a href="#DDR460C3" rid="DDR460C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915826">3</a>). It is a major cause of cornea transplantation in western developed countries (<a href="#DDR460C4" rid="DDR460C4" class=" bibr popnode">4</a>). It has an estimated prevalence of 1 in 2000 in the general population and may even be higher if new technologies are used for early diagnosis (<a href="#DDR460C5" rid="DDR460C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915814">5</a>). It can represent up to 5% of individuals who present in screening for refractive corneal surgery, such as LASIK eye surgery (<a href="#DDR460C6" rid="DDR460C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915806">6</a>). Despite intensive biochemical and genetic investigations, its underlying cause remains poorly understood (<a href="#DDR460C1" rid="DDR460C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915801">1</a>,<a href="#DDR460C7" rid="DDR460C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915818">7</a>–<a href="#DDR460C12" rid="DDR460C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915799">12</a>).</p><p id="__p3">Although the etiology of keratoconus is still unknown, genetic predisposition plays an important role as indicated by the association of keratoconus with genetic syndromes, segregation analyses, genetic epidemiological data and gene mapping studies (<a href="#DDR460C8" rid="DDR460C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_561220747">8</a>,<a href="#DDR460C9" rid="DDR460C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915837">9</a>,<a href="#DDR460C11" rid="DDR460C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915810">11</a>,<a href="#DDR460C12" rid="DDR460C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915795">12</a>). A number of gene loci/chromosomal regions for keratoconus have been mapped by linkage analysis (<a href="#DDR460C7" rid="DDR460C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915829">7</a>,<a href="#DDR460C10" rid="DDR460C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915813">10</a>,<a href="#DDR460C13" rid="DDR460C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915822">13</a>–<a href="#DDR460C17" rid="DDR460C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915823">17</a>); however, susceptibility genes have not been identified in these regions. Mutations in the visual system homeobox gene 1 (<em>VSX1</em>) have been identified through the targeted screening of this gene in keratoconus patients but with conflicting results (<a href="#DDR460C18" rid="DDR460C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915834">18</a>–<a href="#DDR460C21" rid="DDR460C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915833">21</a>).</p><p id="__p4">Results of gene mapping efforts imply the importance of genetic determinants, but the lack of consistent chromosomal loci among different studies suggests genetic heterogeneity and illustrates the complex nature of the genetic contribution to the development of keratoconus. In addition, the sample sizes for most studies were moderate, contributing to difficulty in replication. Moreover, it has been difficult to localize regions precisely using the linkage method. For example, using a four-generation pedigree and a fine-mapping approach, we could only localize a linkage region to a 20 Mb region on chromosome 5, which still contained hundreds of genes (<a href="#DDR460C16" rid="DDR460C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915828">16</a>). Although the candidate gene approach is useful, it is greatly limited by lack of information to adequately select keratoconus susceptibility genes. The recent developments in genotyping technology have made large-scale genetic studies feasible with hundreds of thousands of dense markers, such as genome-wide association studies (GWASs). This hypothesis-free high-throughput approach makes it possible to identify common genomic variations contributing to complex genetic diseases, such as glaucoma, macular degeneration, refractive error (<a href="#DDR460C22" rid="DDR460C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_561220748">22</a>–<a href="#DDR460C24" rid="DDR460C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915791">24</a>), ocular quantitative traits such as corneal thickness (<a href="#DDR460C25" rid="DDR460C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915792">25</a>–<a href="#DDR460C27" rid="DDR460C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915800">27</a>) and central retinal vessels (<a href="#DDR460C28" rid="DDR460C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915815">28</a>). However, thus far, no GWASs have been performed for keratoconus.</p><p id="__p5" class="p p-last">In this first GWAS study of keratoconus, we genotyped three independent case–control and family cohorts of Caucasian keratoconus patients. We identified in the discovery panel and confirmed in both replication panels a potential novel keratoconus susceptibility region located at the 2q21.3. This result suggests the involvement of the gene <em>RAB3GAP1</em> coding for the Rab3 GTPase-activating protein subunit 1, previously implicated in Warburg Micro Syndrome with corneal malformation. This comprehensive GWAS and replication performed in patients with keratoconus offers long-awaited insight into the genetic susceptibility to keratoconus, one of the common causes of corneal transplantation in the developed world (<a href="#DDR460C4" rid="DDR460C4" class=" bibr popnode">4</a>).</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">GWAS design: discovery and replication cohorts</h3><p id="__p6" class="p p-first">In this report, we have studied three independent cohorts of patients with keratoconus. The first case–control cohort used for the discovery phase of the GWAS consisted of 222 Caucasian keratoconus patients and 3324 Caucasian controls. The second case–control cohort of 304 Caucasian keratoconus patients and 518 Caucasian controls was genotyped in the regions identified in the discovery cohort for the purpose of their replication. We also analyzed a family sample of 307 individuals from 70 families consisting of 146 keratoconus patients and 161 unaffected family members.</p><p id="__p7">The gender distribution was similar between cases and controls in the discovery cohort (males: 56 versus 61%, <em>P</em>= 0.20). However, significantly more males were present in cases than in controls in the replication cohort (68 versus 52%, <em>P</em>&lt; 0.05).</p><p id="__p8" class="p p-last">Given the sample size, we have moderate power to identify genetic associations of keratoconus. Consequently, we sought to maximize the number of putative genomic regions for study by first applying mathematical methods without formal population structure correction in our primary analysis, then reducing the number of false positive associations and avoiding power loss by testing of multiple replication samples. However, we report association results both without and with population structure adjustments below in order to allow comparisons of the strength of our results with current GWASs.</p></div><div id="s2b" class="sec"><h3 id="s2btitle">Genetic association testing of the discovery cohort</h3><p id="__p9" class="p p-first-last">From the initial association testing without adjustments for population structure in the discovery cohort, the top three single nucleotide polymorphisms (SNPs), rs6442925 at 3p26, rs4954218 at 2q21.3 and rs1428642 at 19q13.3, were identified with unadjusted <em>P</em>-values of 6.5 × 10<sup>−8</sup>, 2.4 × 10<sup>−7</sup> and 3.1 × 10<sup>−7</sup>, respectively (Table&nbsp;<a href="/pmc/articles/PMC3276283/table/DDR460TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR460TB1" rid-ob="ob-DDR460TB1" co-legend-rid=""><span>1</span></a>, Fig.&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F1" rid-ob="ob-DDR460F1" co-legend-rid="lgnd_DDR460F1"><span>1</span></a>A). To control for population stratification, we performed PC analysis (PCA). After adjusting for sex and three significant PCs, the quantile–quantile (QQ) plot had a genomic control of 1.03 (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr460/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). The most significant SNP identified after adjustments was rs1428642 (ranked number 3 without adjustments) with a <em>P</em>-value of 1.4 × 10<sup>−6</sup>. Other top two ranked SNPs from the initial test, rs6442925 and rs4954218, became less significant with adjusted <em>P</em>-values of 1.9 × 10<sup>−5</sup> and 2.6 × 10<sup>−4</sup>, respectively (Table&nbsp;<a href="/pmc/articles/PMC3276283/table/DDR460TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR460TB1" rid-ob="ob-DDR460TB1" co-legend-rid=""><span>1</span></a>, Fig.&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F1" rid-ob="ob-DDR460F1" co-legend-rid="lgnd_DDR460F1"><span>1</span></a>B). No genomic variants reached genome-wide significance level of 5 × 10<sup>−8</sup> after adjustments. All SNPs with the suggestive genome-wide <em>P</em>-values &lt; 5 × 10<sup>−6</sup> before and after adjusting for population structure are shown in Table&nbsp;<a href="/pmc/articles/PMC3276283/table/DDR460TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR460TB1" rid-ob="ob-DDR460TB1" co-legend-rid=""><span>1</span></a>. PCA testing of the genotyping data of cases and controls in the discovery panel revealed a similar distribution between cases and controls (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr460/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>).

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR460TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p10">SNPs identified in the discovery panel with <em>P</em> &lt; 5 × 10<sup>−6</sup>, before and after adjustments for PCs, and genotyped in the confirmation cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140002466216528" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Closest gene</th><th rowspan="1" colspan="1" align="left">MA<sup>a</sup></th><th colspan="5" rowspan="1" align="left">GWAS<hr></th><th colspan="4" rowspan="1" align="left">Replication (case–control)</th><th colspan="4" rowspan="1" align="left">Replication (family)</th><th colspan="3" rowspan="1" align="left">Meta-analysis</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">F_A<sup>b</sup></th><th rowspan="1" colspan="1" align="left">F_U<sup>b</sup></th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">P_A<sup>c</sup></th><th rowspan="1" colspan="1" align="left">F_A</th><th rowspan="1" colspan="1" align="left">F_U</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">F_A</th><th rowspan="1" colspan="1" align="left">F_U</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">P_A<sup>‡</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs6442925</td><td rowspan="1" colspan="1">3p26</td><td rowspan="1" colspan="1">BHLHB2</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.85</td><td rowspan="1" colspan="1">6.5E−8</td><td rowspan="1" colspan="1">1.9E−5</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">1.05</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">1.22</td><td rowspan="1" colspan="1">0.38</td><td rowspan="1" colspan="1">1.56</td><td rowspan="1" colspan="1">2.0E−6</td><td rowspan="1" colspan="1">9.6E−5</td></tr><tr><td rowspan="1" colspan="1">rs4954218</td><td rowspan="1" colspan="1">2q21.3</td><td rowspan="1" colspan="1">RAB3GAP1</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">2.4E−7</td><td rowspan="1" colspan="1">2.6E−4</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.68</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1">1.2E−9</td><td rowspan="1" colspan="1">1.6E−7</td></tr><tr><td rowspan="1" colspan="1">rs1428642</td><td rowspan="1" colspan="1">19q13.3</td><td rowspan="1" colspan="1">BIRC8</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">0.59</td><td rowspan="1" colspan="1">3.1E−7</td><td rowspan="1" colspan="1">1.4E−6</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">0.82</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">0.014</td></tr><tr><td rowspan="1" colspan="1">rs1328083</td><td rowspan="1" colspan="1">13q33.3</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.76</td><td rowspan="1" colspan="1">8.7E−7</td><td rowspan="1" colspan="1">6.7E−4</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">1.04</td><td rowspan="1" colspan="1">0.78</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">1.08</td><td rowspan="1" colspan="1">0.63</td><td rowspan="1" colspan="1">1.31</td><td rowspan="1" colspan="1">3.9E−4</td><td rowspan="1" colspan="1">0.011</td></tr><tr><td rowspan="1" colspan="1">rs8111998</td><td rowspan="1" colspan="1">19p12</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">2.10</td><td rowspan="1" colspan="1">1.0E−6</td><td rowspan="1" colspan="1">0.0058</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">1.09</td><td rowspan="1" colspan="1">0.65</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">2.04</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">1.71</td><td rowspan="1" colspan="1">8.1E−7</td><td rowspan="1" colspan="1">4.9E−4</td></tr><tr><td rowspan="1" colspan="1">rs4839200</td><td rowspan="1" colspan="1">1p13.2</td><td rowspan="1" colspan="1">KCND3</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">1.79</td><td rowspan="1" colspan="1">1.1E−6</td><td rowspan="1" colspan="1">3.9E−4</td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1">rs12407427</td><td rowspan="1" colspan="1">1q44</td><td rowspan="1" colspan="1">KIF26B</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">1.79</td><td rowspan="1" colspan="1">1.8E−6</td><td rowspan="1" colspan="1">4.1E−5</td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1">rs6792542</td><td rowspan="1" colspan="1">3q26.2</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">1.63</td><td rowspan="1" colspan="1">1.8E−6</td><td rowspan="1" colspan="1">7.2E−4</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">1.04</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">1.31</td><td rowspan="1" colspan="1">8.9E−5</td><td rowspan="1" colspan="1">0.003</td></tr><tr><td rowspan="1" colspan="1">rs6430585</td><td rowspan="1" colspan="1">2q21.3</td><td rowspan="1" colspan="1">UBXD2</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">1.64</td><td rowspan="1" colspan="1">2.0E−6</td><td rowspan="1" colspan="1">0.033</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">1.23</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">1.22</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">1.39</td><td rowspan="1" colspan="1">1.2E−6</td><td rowspan="1" colspan="1">0.002</td></tr><tr><td rowspan="1" colspan="1">rs214884</td><td rowspan="1" colspan="1">7q31.1</td><td rowspan="1" colspan="1">IMMPL2</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">1.88</td><td rowspan="1" colspan="1">2.4E−6</td><td rowspan="1" colspan="1">0.0015</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">1.21</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">1.44</td><td rowspan="1" colspan="1">1.5E−4</td><td rowspan="1" colspan="1">0.007</td></tr><tr><td rowspan="1" colspan="1">rs757219</td><td rowspan="1" colspan="1">7q31.1</td><td rowspan="1" colspan="1">IMMPL2</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.70</td><td rowspan="1" colspan="1">6.0E−6</td><td rowspan="1" colspan="1">0.0064</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">1.33</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">1.36</td><td rowspan="1" colspan="1">5.8E−5</td><td rowspan="1" colspan="1">0.006</td></tr><tr><td rowspan="1" colspan="1">rs3749350</td><td rowspan="1" colspan="1">3p26.2</td><td rowspan="1" colspan="1">LRRN1</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">1.77</td><td rowspan="1" colspan="1">3.0E−6</td><td rowspan="1" colspan="1">2.4E−5</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.19</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.09</td><td rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">1.39</td><td rowspan="1" colspan="1">6.1E−5</td><td rowspan="1" colspan="1">1.4E−4</td></tr><tr><td rowspan="1" colspan="1">rs2659546</td><td rowspan="1" colspan="1">4q23</td><td rowspan="1" colspan="1">PPP3CA</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">2.16</td><td rowspan="1" colspan="1">3.3E−6</td><td rowspan="1" colspan="1">7.4E−4</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">1.39</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">1.36</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">1.63</td><td rowspan="1" colspan="1">3.5E−6</td><td rowspan="1" colspan="1">0.021</td></tr><tr><td rowspan="1" colspan="1">rs1328089</td><td rowspan="1" colspan="1">13q33.3</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">1.61</td><td rowspan="1" colspan="1">4.1E−6</td><td rowspan="1" colspan="1">8.8E−4</td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1">rs1978238</td><td rowspan="1" colspan="1">12p13.3</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">0.64</td><td rowspan="1" colspan="1">3.4E−5</td><td rowspan="1" colspan="1">3.4E−6</td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td><td rowspan="1" colspan="1">–<sup>d</sup></td></tr></tbody></table></div><div id="largeobj_idm140002466216528" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3276283/table/DDR460TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140002465939072"><p id="__p11" class="p p-first-last"><sup>a</sup>MA, minor allele.</p></div><div id="idm140002465938192"><p id="__p12" class="p p-first-last"><sup>b</sup>F_A, minor allele frequency in cases; F_U, minor allele frequency in controls.</p></div><div id="idm140002465937312"><p id="__p13" class="p p-first-last"><sup>c</sup>Adjustments for gender and PC variables.</p></div><div id="idm140002465936432"><p id="__p14" class="p p-first-last"><sup>d</sup>Genotyping failed in the replication panels.</p></div><div id="idm140002465935552"><p id="__p15" class="p p-first-last"><sup>‡</sup><em>P</em>-values using adjusted ORs for PCs.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR460F1" co-legend-rid="lgnd_DDR460F1"><a href="/pmc/articles/PMC3276283/figure/DDR460F1/" target="figure" rid-figpopup="DDR460F1" rid-ob="ob-DDR460F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140002465930176" class="figure"><a href="/pmc/articles/PMC3276283/figure/DDR460F1/" target="figure" rid-figpopup="DDR460F1" rid-ob="ob-DDR460F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3276283_ddr46001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr46001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3276283/bin/ddr46001.jpg"></a></div><div id="largeobj_idm140002465930176" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3276283/figure/DDR460F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR460F1"><div><a class="figpopup" href="/pmc/articles/PMC3276283/figure/DDR460F1/" target="figure" rid-figpopup="DDR460F1" rid-ob="ob-DDR460F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p16">Manhattan plots of GWAS of keratoconus. (<strong>A</strong>) Manhattan plot of unadjusted <em>P</em>-values; (<strong>B</strong>) Manhattan plot of <em>P</em>-values adjusted for PCs and gender: dots marked as black: top three SNPs identified without adjustments.</p></div></div></div></div><div id="s2c" class="sec"><h3 id="s2ctitle">Genetic association testing of replication cohorts</h3><p id="__p17" class="p p-first">Given the moderate power in the discovery cohort as indicated by lack of hits reaching the genome-wide significance level of 5 × 10<sup>−8</sup>, all SNPs identified with a less stringent significant threshold (10<sup>−4</sup>) were selected for genotyping and analysis in the replication cohorts. In addition, we included proxy SNPs in strong linkage disequilibrium with our primary associated SNPs and tag SNPs spanning associated regions to provide redundancy for replication and to perform additional fine-mapping of the regions. Minor allele frequencies were similar in the discovery and replication cohorts, thus confirming that both samples were well matched.</p><p id="__p18" class="p p-last">SNP rs4954218 located at the chromosomal region 2q21.3 was confirmed in both replication panels with <em>P</em>-values of 0.004 and 0.009 for case–controls and family panels, respectively, with consistent direction of effect in all three panels. Additional SNPs at 2q21.3 also showed significant association, with <em>P</em>-values ranging from 4.0 × 10<sup>−5</sup> to 0.05. Two regions at 19p12 and 7q31, associated in the discovery panel with unadjusted <em>P</em>-values of 1.0 × 10<sup>−6</sup> and 6.0 × 10<sup>−6</sup> (adjusted <em>P</em>= 0.0058 and 0.0064) (Table&nbsp;<a href="/pmc/articles/PMC3276283/table/DDR460TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR460TB1" rid-ob="ob-DDR460TB1" co-legend-rid=""><span>1</span></a>), were significant in one of the two replication panels. The region at 19p12 was associated in the family panel with a <em>P</em>-value of 0.003; the region at 7q31 in the case–control panel with the <em>P</em>-value of 0.03 (Table&nbsp;<a href="/pmc/articles/PMC3276283/table/DDR460TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR460TB1" rid-ob="ob-DDR460TB1" co-legend-rid=""><span>1</span></a>). Other top regions identified by GWAS were not associated with keratoconus in either confirmation panel. Although there was a difference in gender distribution between cases and controls, the <em>P</em>-values remained similar before and after adjustments for gender and other covariates (data not shown).</p></div><div id="s2d" class="sec"><h3 id="s2dtitle">Meta-analysis</h3><p id="__p19" class="p p-first-last">Meta-analysis of the combined genotyping data of discovery and two replication panels determined SNP rs4954218, located at the 2q21.3 genomic region, to be associated with keratoconus with PC-adjusted <em>P</em>-value of 1.6 × 10<sup>−7</sup> (unadjusted <em>P</em>-value of 1.2 × 10<sup>−9</sup>). The SNP rs4954218 was the top ranked SNP identified by meta-analysis both with and without PC adjustments; however, no SNPs reached genome-wide significance level after adjustments for PCs in the meta-analysis. The distributions of minor allele frequencies between cases and controls are similar among three data sets (0.14 versus 0.25 in GWAS; 0.17 versus 0.23 in the case–control replication panel; 0.16 versus 0.22 in the family-based replication panel).</p></div><div id="s2e" class="sec sec-last"><h3 id="s2etitle">Variation across 2q21.3 and keratoconus</h3><p id="__p20" class="p p-first-last">The most significant association signal identified not only in the discovery cohort, but also further confirmed in two replication cohorts, is located at SNP rs4954218 (Fig.&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2" co-legend-rid="lgnd_DDR460F2"><span>2</span></a>). It is located 6.4 kb 5′ to <em>RAB3GAP1</em>, which codes for the catalytic subunit of the RAB3 GTPase-activating protein, and, on the other strand, 21.2 kb 5′ to <em>YSK4,</em> coding for SPS1/STE20-related protein kinase. In addition, as shown on Figure&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2" co-legend-rid="lgnd_DDR460F2"><span>2</span></a>C, two additional SNPs, genotyped only in the confirmation panels, showed positive association in this region: SNP rs6730157 located ∼100 kb away in an intron of <em>RAB3GAP1</em> (<em>P</em>= 4.7 × 10<sup>−4</sup>), and SNP rs1561277 located ∼290 kb away in the intron of <em>ZRANB3,</em> coding for zinc finger, RAN-binding domain containing three genes (<em>P</em>= 2.7 × 10<sup>−5</sup>). In order to determine whether these signals represent independent associations, we performed conditional analysis by re-analyzing the replication keratoconus cohort after adjusting for rs4954218 genotypes in the association test. Figure&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2" co-legend-rid="lgnd_DDR460F2"><span>2</span></a> shows the regional plots at 2q21.3 before (Fig.&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2" co-legend-rid="lgnd_DDR460F2"><span>2</span></a>A, original regional plot; Fig.&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2" co-legend-rid="lgnd_DDR460F2"><span>2</span></a>B, regional plot with adjustments for PCs and Fig.&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2" co-legend-rid="lgnd_DDR460F2"><span>2</span></a>C, regional plot in replication samples) and after (Fig.&nbsp;2D) adjustment. After controlling for rs4954218 as a covariate, the associations for the other two SNPs were greatly diminished but remained significant (<em>P</em>= 0.004 and 0.02, respectively) as shown in Figure&nbsp;<a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2" co-legend-rid="lgnd_DDR460F2"><span>2</span></a>D. Because the signal of rs4954218 is weaker than the one from rs6730157 in the replication cohort where the conditional analysis is performed, there is a strong possibility that there is only one signal and that rs6730157 captures better the causal variant(s) than rs4954218. A final conclusion could be reached through additional fine-mapping and sequencing in the region, and likely will require additional subjects.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR460F2" co-legend-rid="lgnd_DDR460F2"><a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140002465896608" class="figure"><a href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3276283_ddr46002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr46002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3276283/bin/ddr46002.jpg"></a></div><div id="largeobj_idm140002465896608" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3276283/figure/DDR460F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR460F2"><div><a class="figpopup" href="/pmc/articles/PMC3276283/figure/DDR460F2/" target="figure" rid-figpopup="DDR460F2" rid-ob="ob-DDR460F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p21">Regional plots of rs4954218 at 2q21.3 (<strong>A</strong>) regional plot of GWAS; (<strong>B</strong>) regional plot of GWAS with PCs adjustments; (<strong>C</strong>) regional plot of replication; (<strong>D</strong>) regional plot of replication after adjustments for rs4954218.</p></div></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p22" class="p p-first">In this study, we identified a potential genetic susceptibility region located at 2q21.3 in the first GWAS of keratoconus and then confirmed this observation in two replication sample sets. Several candidate genes are located at this region. However, based on association results in the region and biological evidence, we propose <em>RAB3GAP1</em> as a gene for keratoconus. The most significantly associated SNP rs4954218 is located 5′ upstream and within a theoretical possibility of a regulatory effect to two genes: <em>RAB3GAP1</em> (6.4 kb away) and <em>YSK4</em> (21.2 kb away). The second highest association signal is located at SNP rs6730157 in the <em>RAB3GAP1</em> intron, and based on results of the conditional analysis, there is a strong possibility that both associated SNPs represent a single association signal linked to the causal variant(s) located in the <em>RAB3GAP1</em> gene. This gene encodes a catalytic subunit of Rab3 GTPase-activating protein (RAB3GAP), a direct activator of the Rab3 GTPase, a key regulator of calcium-mediated hormone and neurotransmitter exocytosis (<a href="#DDR460C29" rid="DDR460C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915825">29</a>,<a href="#DDR460C30" rid="DDR460C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915807">30</a>). Multiple mutations in <em>RAB3GAP1</em> have been identified in patients with Warburg Micro Syndrome, a rare, autosomal recessive syndrome characterized by a number of neurological and ocular developmental defects, including microcornea, congenital cataracts and optic atrophy (<a href="#DDR460C31" rid="DDR460C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915812">31</a>,<a href="#DDR460C32" rid="DDR460C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915817">32</a>). Congenital cataracts are also frequent in patients with Martsolf syndrome due to mutations in the non-catalytic subunit (RAB3GAP2) of RAB3GAP (<a href="#DDR460C33" rid="DDR460C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915794">33</a>). Eye abnormalities have been also identified in another Rab-related inherited disorder choroideremia, an inherited choroidal and retinal regeneration, due to mutations of Rab escort protein 1, a cofactor required for Rab protein function (<a href="#DDR460C34" rid="DDR460C34" class=" bibr popnode">34</a>).</p><p id="__p23">According to a published study, <em>RAB3GAP1</em> is ubiquitously expressed in a variety of human tissues; however, no eye tissue was tested (<a href="#DDR460C35" rid="DDR460C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915809">35</a>). BLAST search of <em>RAB3GAP1</em> transcript through EST database identified 15 ESTs from several independent cDNAs from a variety of eye tissues, including human normalized iris, trabecular meshwork, fetal lens, eyeball, retina, fetal eye; and mixed library of fetal lens, eye anterior segment, optic nerve, retina and choroid. No normal corneal ESTs matched to the <em>RAB3GAP1</em> transcript; however, most interestingly, a matching EST derived from an unamplified human keratoconus cornea library constructed by our group in collaboration with National Eye Institute (<a href="#DDR460C36" rid="DDR460C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915803">36</a>) was identified in a BLAST search. Search of the Bgee database (<a href="#DDR460C37" rid="DDR460C37" class=" bibr popnode">37</a>) of gene expression data derived from Affymetrix expression microarrays indicated that individual probes specific to the <em>RAB3GAP1</em> transcript are present in a mouse cornea in three independent microarray experiments, thus suggesting its presence in normal human corneas as well.</p><p id="__p24">Recurrent ocular abnormalities in disease phenotypes due to defects in <em>RAB3GAP1</em> provide clear indication that Rab3 proteins are required for both function and normal structural development of the eye. However, the precise mechanisms by which loss of RAB3GAP function could lead to developmental anomalies in the eye are not known. It is clear that in the human cornea, a number of biologically important pathways will be directly affected by a defect in the vesicular membrane transport due to the RAB3GAP defect.</p><p id="__p25">Based on its genomic location, biological function relevant to the function of the eye, involvement in ocular abnormalities and expression pattern, <em>RAB3GAP1</em> represents the most plausible candidate gene responsible for the keratoconus association signal. Extensive database searches and data mining did not identify any clear evidence to suggest preferential expression or function of <em>YSK4</em> and <em>ZRANB3</em> in the eye, as well as no ocular phenotypes linked to these genes' defects.</p><p id="__p26">Other potentially associated with keratoconus chromosomal regions identified in the discovery phase have not been formally confirmed. The 19p12 and 7q31 genomic regions were only significant in one of the two replication sample sets. The initial top hits at the 3p26 region (unadjusted) and at the 19q13.3 region (adjusted) were not significant in any replication panels.</p><p id="__p27">The keratoconus susceptibility region at 2q21.3 represents a potential novel locus which has not been described in previously published linkage studies of keratoconus families (cited in the Introduction). This comes as no surprise, as linkage and association methods are designed to identify rare and common disease-associated variants for complex multigenic disorders. This locus was not described among major genomic loci associated with population-based variation in central corneal thickness in recently published GWASs (<a href="#DDR460C25" rid="DDR460C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915798">25</a>–<a href="#DDR460C27" rid="DDR460C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915832">27</a>), indicating that other pathways may make a greater contribution to the keratoconus pathogenesis.</p><p id="__p28">Some limitations of our study should be noted. First is our approach to control for population structure and reduce the false positives. Keratoconus is a rare disease with population prevalence of 1/2000, which limits the identification and recruitment of new patients. Given the sample size and resources, we have moderate power to identify the associations. Although adjusting for population structure is an important tool to reduce the number of potentially inflated associations, we identified no SNPs with <em>P</em>-values at the 10<sup>−7</sup> level after adjustments, and only two SNPs with <em>P</em>-values below the 5 × 10<sup>−6</sup> level: rs1428642 (<em>P</em>= 1.4 × 10<sup>−6</sup>; ranked #3 without adjustment) and rs1978238 (<em>P</em>= 3.4 × 10<sup>−6</sup>). Meta <em>P</em>-value of the top SNP, rs1428642, was only 0.014, clearly indicating that it was not confirmed in either of the replication panels. In contrast, SNP rs4954218 in the 2q21.3 region with the unadjusted <em>P</em>-value of 2.4 × 10<sup>−7</sup> in the discovery panel was much less significant after PC adjustments (<em>P</em>= 2.6 × 10<sup>−4</sup>). However, it went through successful replication in two replication cohorts, with and without PC adjustments, and retained a high significance level with an adjusted meta <em>P</em>-value of 1.6 × 10<sup>−7</sup>, close to the GWAS significance level. In addition, the distributions of minor allele frequencies are similar in three data sets, which is further evidence that the SNP association is likely real. This suggests a possibility that PC adjustments used in the analysis of a relatively rare complex genetic disease, such as keratoconus, reduce the power for identifying potential disease-associated regions, as has been stated in the paper by Price <em>et al.</em> (<a href="#DDR460C38" rid="DDR460C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915811">38</a>). Thus, using the strategy we used in this paper, i.e. of following-up more possible candidate regions in the replication panels, appears to be a useful approach to reduce the false positive rate and still avoid the power loss. Notably in the past, some GWASs done in a limited number of patients have achieved major successes in identifying genetic associations, e.g. complement factor H in AMD (<a href="#DDR460C39" rid="DDR460C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915805">39</a>). These studies applied a design without population structure adjustments, which were not available at that time.</p><p id="__p29">Another issue is the failure of successful genotyping of several SNPs in the replication cohorts as we noted in the Table&nbsp;<a href="/pmc/articles/PMC3276283/table/DDR460TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR460TB1" rid-ob="ob-DDR460TB1" co-legend-rid=""><span>1</span></a>. Different from the technology for GWAS, we used customized Illumina iSelect panel to genotype the samples in replication cohorts. The total failure rate is ∼5% in our study (255/4905) which is much lower than the guarantee from Illumina (&lt;20%). Although the failure rate is not high, it could reduce the probabilities for identifying some true associations.</p><p id="__p30" class="p p-last">In summary, in this first comprehensive GWAS in patients with keratoconus, we identified a novel potential keratoconus locus at 2q21.3, containing a candidate gene <em>RAB3GAP1</em>. This novel genetic susceptibility locus should be further investigated in the future genetic research of keratoconus, the most common cause of corneal transplantation in the western developed world.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Subjects</h3><p id="__p31" class="p p-first">Clinically affected keratoconus patients were recruited at three major sites: the Cornea Genetic Eye Institute (<em>n</em>= 618) at Cedars-Sinai Medical Center, Los Angeles, CA, USA; the Jules Stein Eye Institute (<em>n</em>= 26) at UCLA, Los Angeles, CA, USA; and the University Hospitals Eye Institute (<em>n</em>= 46) at University Hospitals Case Medical Center and Case Western Reserve University, Cleveland, OH, USA. Institutional Review Board (IRB) approval was obtained at all clinic sites. Written informed consent was obtained from all subjects. The study was conducted in concordance with the provisions of the Declaration of Helsinki.</p><div id="s4a1" class="sec"><p></p><h4 id="s4a1title" class="inline">GWAS cohort </h4><p id="__p32" class="p p-first-last">Clinically affected Caucasian keratoconus cases (<em>n</em>= 240) were enrolled into the GWAS as a part of the longitudinal videokeratography and genetic study at the Cornea Genetic Eye Institute (<a href="#DDR460C12" rid="DDR460C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915830">12</a>). After removing samples with poor quality of genotyping, 222 samples were included in the analysis. Caucasian controls (<em>n</em>= 3324) were obtained from the Cardiovascular Health Study (CHS), a population-based cohort study of risk factors for cardiovascular disease and stroke in adults 65 years of age or older, recruited at four field centers. A total of 5201 predominantly Caucasian individuals were recruited in 1989–1990 from random samples of Medicare eligibility lists, followed by an additional 687 African-Americans recruited in 1992–1993 (total <em>n</em>= 5888). CHS was approved by the Institutional Review Board at each recruitment site, and subjects provided informed consent for the use of their genetic information. African-American CHS participants were excluded from the analysis due to insufficient number of ethnically matched cases (<a href="#DDR460C40" rid="DDR460C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915793">40</a>,<a href="#DDR460C41" rid="DDR460C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915821">41</a>).</p></div><div id="s4a2" class="sec"><p></p><h4 id="s4a2title" class="inline">Replication cohort </h4><p id="__p33" class="p p-first-last">An independent group of keratoconus cases and controls was recruited as a part of the collaborative replication study. A total of 232 independent keratoconus cases were recruited at the Cornea Genetic Eye Institute; 26 cases at the Jules Stein Eye Institute; and 46 cases at University Hospitals Eye Institute. This replication panel consisted of a total of 304 keratoconus cases. Five hundred and eighteen normal controls were also recruited at Cedars-Sinai Medical Center in Los Angeles.</p></div><div id="s4a3" class="sec sec-last"><p></p><h4 id="s4a3title" class="inline">Keratoconus families </h4><p id="__p34" class="p p-first-last">Family members of keratoconus cases diagnosed at the Cornea Genetic Eye Institute were recruited to perform family-based studies. A total of 307 individuals from 70 families consisting of 146 keratoconus patients and 161 unaffected family members were obtained for this study. One hundred and eighty-six individuals in 41 pedigrees were Caucasians, whereas 91 individuals in 20 pedigrees were identified as Hispanics.</p></div></div><div id="s4b" class="sec"><h3 id="s4btitle">Clinical diagnosis</h3><p id="__p35" class="p p-first">The diagnosis of keratoconus was performed by a cornea fellowship trained ophthalmologist or an experienced research optometrist (L.S.-F.) based on clinical examination and videokeratography pattern analysis. Clinical examination included slit-lamp biomicroscopy, cycloplegic retinoscopy and fundus evaluations. Slit-lamp biomicroscopy was used to identify stromal corneal thinning, Vogts' striae or a Fleischer ring. Retinoscopy examination was performed with a fully dilated pupil to determine the presence or absence of retro illumination signs of keratoconus, such as the oil droplet sign and scissoring of the red reflex 20 min after phenylephrine 2.5% and cyclopentolate 1% drops had been instilled in the eye. Videokeratography evaluation was performed on each eye using the Topographic Modeling System (Computed Anatomy, New York, NY, USA), Orbscan II (Bausch &amp; Lomb, Rochester, NY, USA), Oculus Pentacam (Oculus, Inc, Lynnwood, WA, USA) or Keratron (Optikon, Rome, Italy).</p><p id="__p36" class="p p-last">Subjects were considered having keratoconus if they had at least one clinical sign of keratoconus and a confirmatory videokeratography map with an asymmetric bowtie with skewed radial axis above and below the horizontal meridian (AB/SRAX) pattern (<a href="#DDR460C42" rid="DDR460C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915827">42</a>). Importantly, topography was screened for mimicking disease such as pellucid marginal degeneration which was excluded. Subjects that had bilateral keratoplasty for keratoconus were included if the surgical pathology report confirmed the presence of the disease.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Genotyping</h3><div id="s4c1" class="sec sec-first"><p></p><h4 id="s4c1title" class="inline">DNA extraction </h4><p id="__p37" class="p p-first-last">DNA was extracted from EBV transformed lymphoblastoid cell lines established from peripheral whole blood of each study participant using NucleoSpin Tissue kit (MACHEREY-NAGEL Inc., Bethlehem, PA, USA). Salt extraction protocol was used to tract DNA from buffy coats whenever cell lines were not available (<a href="#DDR460C43" rid="DDR460C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915797">43</a>).</p></div><div id="s4c2" class="sec"><p></p><h4 id="s4c2title" class="inline">Genome-wide association study </h4><p id="__p38" class="p p-first-last">All genotyping was performed at the genotyping laboratory of the General Clinical Research Center and the Medical Genetics Institute at Cedars-Sinai Medical Center. Discovery genotyping was performed using the Illumina Human CNV370 Quad whole genome genotyping beadchip, following the manufacturer's protocol (Illumina, San Diego, CA, USA) (<a href="#DDR460C44" rid="DDR460C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915820">44</a>,<a href="#DDR460C45" rid="DDR460C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915835">45</a>). The average genotyping rate for samples passing quality control was 99.77%. Samples with poor genotyping quality (<em>n</em>= 18) were removed from the analysis. Procedures for control of genotyping quality included genotyping rate per subject &gt;99%, narrow distribution (theta standard deviation) and wide separation between genotyping clusters (R/theta plots), minor allele frequency &gt;5%, Hardy–Weinberg test (SNPs with <em>P</em>-value &lt; 0.001 were excluded). SNPs were also excluded from analysis for significant differences in missing data between cases and controls (SNPs with <em>P</em>-value in missingness test &lt; 0.01 were excluded). Approximately 290K SNPs were used for the primary GWAS analysis. After the analyses, the cluster plots of the top SNPs identified by GWAS at the 10<sup>−6</sup> significance level were carefully reviewed to ensure the genotyping quality.</p></div><div id="s4c3" class="sec sec-last"><p></p><h4 id="s4c3title" class="inline">Replication </h4><p id="__p39" class="p p-first-last">Replication genotyping was performed using the custom iSelect Infinium technology (<a href="#DDR460C46" rid="DDR460C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915802">46</a>). Using the software Tagger as implemented in Haploview (<a href="#DDR460C47" rid="DDR460C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915836">47</a>,<a href="#DDR460C48" rid="DDR460C48" class=" bibr popnode">48</a>) and data from the CEPH population in the International HapMap project, release 2 (<a href="#DDR460C49" rid="DDR460C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915808">49</a>,<a href="#DDR460C50" rid="DDR460C50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915816">50</a>), we designed a panel of 4905 SNPs that were compatible with the Illumina Infinium technology, of which 4650 SNPs remained for analysis following clustering quality control. The average genotyping rate for samples genotyped on the iSelect platform and passing quality control was 99.98%. Genotyping concordance among 20 pairs of blind duplicates 100%.</p></div></div><div id="s4d" class="sec sec-last"><h3 id="s4dtitle">Statistical analyses</h3><div id="s4d1" class="sec sec-first"><p></p><h4 id="s4d1title" class="inline">GWAS and replication study </h4><p id="__p40" class="p p-first-last">Odds ratios (ORs) and their standard errors (SE) were calculated using genome-wide SNP data under logistic regression models using the PLINK program (v1.07) (<a href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</a>) (<a href="#DDR460C51" rid="DDR460C51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915824">51</a>). Genome-wide significance threshold of 5 × 10<sup>−8</sup> was applied to our GWAS analysis. PCA for population stratification was performed in the discovery GWAS panel using EIGENSTRAT (<a href="#DDR460C38" rid="DDR460C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915819">38</a>). Gender and three significant PC variables identified from PCA were used as covariates. Because of the large age difference between patients and controls in the discovery panel, age was not used as a covariate. Genomic control (λ<sub>GC</sub>) was calculated to assess the inflation factor of the test statistic. For the replication case–control samples, we additionally adjusted for the site in the analysis. In order to identify additional association signals at the 2q21.3 locus, testing for association was also performed after adjustment for SNP rs4954218 as an additional covariate. Association tests controlling for familial correlations by the Generalized Estimation Equation (GEE) model were performed in families using GWAF package (<a href="#DDR460C52" rid="DDR460C52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915804">52</a>). Regional association plots were generated by LocusZoom (<a href="http://cgs.sph.umich.edu/locuszoom/">http://cgs.sph.umich.edu/locuszoom/</a>) (<a href="#DDR460C53" rid="DDR460C53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_532915796">53</a>).</p></div><div id="s4d2" class="sec sec-last"><p></p><h4 id="s4d2title" class="inline">Meta-analysis </h4><p id="__p41" class="p p-first-last">Meta-analysis of associated SNPs obtained from the logistic regression models from discovery and replication cohorts was calculated using an inverse-variance weighting by PLINK.</p></div></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p42" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr460/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p43" class="p p-first-last">This work was supported by the National Eye Institute (<span>NEI</span>;
<span>09052</span>), the Eye Defect Research Foundation Inc. and in part by the Cedars-Sinai General Research Center Genotyping (<span>NCRR</span>;
<span>M01-RR00425</span>). The CHS data reported in this article was supported by the National Heart, Lung, and Blood Institute (<span>N01-HC-85079</span>; through <span>N01-HC-85086</span>;, <span>N01-HC-35129</span>;, <span>N01 HC-15103</span>;, <span>N01 HC-55222</span>;, <span>N01-HC-75150</span>;, <span>N01-HC-45133</span>;, <span>U01 HL080295</span>; and <span>R01 HL087652</span>), with additional contribution from the National Institute of Neurological Disorders and Stroke.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140002465788624"><a href="/pmc/articles/PMC3276283/bin/supp_21_2_421__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140002465785088" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140002465785088title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p44">This research was also supported in part by the Cedars-Sinai Board of Governors’ Chair in Medical Genetics. The authors thank the other investigators, the staff and the participants of the study for their valuable contributions. A full list of principal CHS investigators and institutions can be found at <a href="http://www.chs-nhlbi.org/pi.htm">http://www.chs-nhlbi.org/pi.htm</a>.</p><p id="__p45"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140002465782016" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140002465782016title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDR460C1">1. <span class="element-citation">Rabinowitz Y.S. Keratoconus. <span><span class="ref-journal">Surv. Ophthalmol. </span>1998;<span class="ref-vol">42</span>:297–319.</span> [<a href="/pubmed/9493273" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Surv.+Ophthalmol.&amp;title=Keratoconus&amp;author=Y.S.+Rabinowitz&amp;volume=42&amp;publication_year=1998&amp;pages=297-319&amp;pmid=9493273&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C2">2. <span class="element-citation">Crews M.J., Driebe W.T., Jr, Stern G.A. The clinical management of keratoconus: a 6 year retrospective study. <span><span class="ref-journal">CLAO J. </span>1994;<span class="ref-vol">20</span>:194–197.</span> [<a href="/pubmed/7955302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CLAO+J.&amp;title=The+clinical+management+of+keratoconus:+a+6+year+retrospective+study&amp;author=M.J.+Crews&amp;author=W.T.+Driebe&amp;author=G.A.+Stern&amp;volume=20&amp;publication_year=1994&amp;pages=194-197&amp;pmid=7955302&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C3">3. <span class="element-citation">Lass J.H., Lembach R.G., Park S.B., Hom D.L., Fritz M.E., Svilar G.M., Nuamah I.F., Reinhart W.J., Stocker E.G., Keates R.H., et al.  Clinical management of keratoconus. A multicenter analysis. <span><span class="ref-journal">Ophthalmology. </span>1990;<span class="ref-vol">97</span>:433–445.</span> [<a href="/pubmed/2183122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Clinical+management+of+keratoconus.+A+multicenter+analysis&amp;author=J.H.+Lass&amp;author=R.G.+Lembach&amp;author=S.B.+Park&amp;author=D.L.+Hom&amp;author=M.E.+Fritz&amp;volume=97&amp;publication_year=1990&amp;pages=433-445&amp;pmid=2183122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C4">4. <span class="element-citation"><span class="ref-journal">Eye Banking Statistical Report.</span> Washington, DC: Eye Bank Association of America; 2008.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Eye+Banking+Statistical+Report&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C5">5. <span class="element-citation">Kennedy R.H., Bourne W.M., Dyer J.A. A 48-year clinical and epidemiologic study of keratoconus. <span><span class="ref-journal">Am. J. Ophthalmol. </span>1986;<span class="ref-vol">101</span>:267–273.</span> [<a href="/pubmed/3513592" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Ophthalmol.&amp;title=A+48-year+clinical+and+epidemiologic+study+of+keratoconus&amp;author=R.H.+Kennedy&amp;author=W.M.+Bourne&amp;author=J.A.+Dyer&amp;volume=101&amp;publication_year=1986&amp;pages=267-273&amp;pmid=3513592&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C6">6. <span class="element-citation">Wilson S.E., Klyce S.D. Screening for corneal topographic abnormalities before refractive surgery. <span><span class="ref-journal">Ophthalmology. </span>1994;<span class="ref-vol">101</span>:147–252.</span> [<a href="/pubmed/8302548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Screening+for+corneal+topographic+abnormalities+before+refractive+surgery&amp;author=S.E.+Wilson&amp;author=S.D.+Klyce&amp;volume=101&amp;publication_year=1994&amp;pages=147-252&amp;pmid=8302548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C7">7. <span class="element-citation">Bisceglia L., De Bonis P., Pizzicoli C., Fischetti L., Laborante A., Di Perna M., Giuliani F., Delle Noci N., Buzzonetti L., Zelante L. Linkage analysis in keratoconus: replication of locus 5q21.2 and identification of other suggestive Loci. <span><span class="ref-journal">Invest. Ophthalmol. Vis. Sci. </span>2009;<span class="ref-vol">50</span>:1081–1086.</span> [<a href="/pubmed/18978346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci.&amp;title=Linkage+analysis+in+keratoconus:+replication+of+locus+5q21.2+and+identification+of+other+suggestive+Loci&amp;author=L.+Bisceglia&amp;author=P.+De+Bonis&amp;author=C.+Pizzicoli&amp;author=L.+Fischetti&amp;author=A.+Laborante&amp;volume=50&amp;publication_year=2009&amp;pages=1081-1086&amp;pmid=18978346&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C8">8. <span class="element-citation">Edwards M., McGhee C.N., Dean S. The genetics of keratoconus. <span><span class="ref-journal">Clin. Exp. Ophthalmol. </span>2001;<span class="ref-vol">29</span>:345–351.</span> [<a href="/pubmed/11778802" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Exp.+Ophthalmol.&amp;title=The+genetics+of+keratoconus&amp;author=M.+Edwards&amp;author=C.N.+McGhee&amp;author=S.+Dean&amp;volume=29&amp;publication_year=2001&amp;pages=345-351&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C9">9. <span class="element-citation">Forstot S.L., Goldstein J.H., Damiano R.E., Dukes D.K. Familial keratoconus. <span><span class="ref-journal">Am. J. Ophthalmol. </span>1988;<span class="ref-vol">105</span>:92–93.</span> [<a href="/pubmed/3337200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Ophthalmol.&amp;title=Familial+keratoconus&amp;author=S.L.+Forstot&amp;author=J.H.+Goldstein&amp;author=R.E.+Damiano&amp;author=D.K.+Dukes&amp;volume=105&amp;publication_year=1988&amp;pages=92-93&amp;pmid=3337200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C10">10. <span class="element-citation">Li X., Rabinowitz Y.S., Tang Y.G., Picornell Y., Taylor K.D., Hu M., Yang H. Two-stage genome-wide linkage scan in keratoconus sib pair families. <span><span class="ref-journal">Invest. Ophthalmol. Vis. Sci. </span>2006;<span class="ref-vol">47</span>:3791–3795.</span> [<a href="/pubmed/16936089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci.&amp;title=Two-stage+genome-wide+linkage+scan+in+keratoconus+sib+pair+families&amp;author=X.+Li&amp;author=Y.S.+Rabinowitz&amp;author=Y.G.+Tang&amp;author=Y.+Picornell&amp;author=K.D.+Taylor&amp;volume=47&amp;publication_year=2006&amp;pages=3791-3795&amp;pmid=16936089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C11">11. <span class="element-citation">Owens H., Gamble G. A profile of keratoconus in New Zealand. <span><span class="ref-journal">Cornea. </span>2003;<span class="ref-vol">22</span>:122–125.</span> [<a href="/pubmed/12605045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cornea&amp;title=A+profile+of+keratoconus+in+New+Zealand&amp;author=H.+Owens&amp;author=G.+Gamble&amp;volume=22&amp;publication_year=2003&amp;pages=122-125&amp;pmid=12605045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C12">12. <span class="element-citation">Wang Y., Rabinowitz Y.S., Rotter J.I., Yang H. Genetic epidemiological study of keratoconus: evidence for major gene determination. <span><span class="ref-journal">Am. J. Med. Genet. </span>2000;<span class="ref-vol">93</span>:403–409.</span> [<a href="/pubmed/10951465" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.&amp;title=Genetic+epidemiological+study+of+keratoconus:+evidence+for+major+gene+determination&amp;author=Y.+Wang&amp;author=Y.S.+Rabinowitz&amp;author=J.I.+Rotter&amp;author=H.+Yang&amp;volume=93&amp;publication_year=2000&amp;pages=403-409&amp;pmid=10951465&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C13">13. <span class="element-citation">Brancati F., Valente E.M., Sarkozy A., Feher J., Castori M., Del Duca P., Mingarelli R., Pizzuti A., Dallapiccola B. A locus for autosomal dominant keratoconus maps to human chromosome 3p14-q13. <span><span class="ref-journal">J. Med. Genet. </span>2004;<span class="ref-vol">41</span>:188–192.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1766922/">PMC free article</a>]</span> [<a href="/pubmed/14985379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet.&amp;title=A+locus+for+autosomal+dominant+keratoconus+maps+to+human+chromosome+3p14-q13&amp;author=F.+Brancati&amp;author=E.M.+Valente&amp;author=A.+Sarkozy&amp;author=J.+Feher&amp;author=M.+Castori&amp;volume=41&amp;publication_year=2004&amp;pages=188-192&amp;pmid=14985379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C14">14. <span class="element-citation">Fullerton J., Paprocki P., Foote S., Mackey D.A., Williamson R., Forrest S. Identity-by-descent approach to gene localisation in eight individuals affected by keratoconus from north-west Tasmania, Australia. <span><span class="ref-journal">Hum. Genet. </span>2002;<span class="ref-vol">110</span>:462–470.</span> [<a href="/pubmed/12073017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Identity-by-descent+approach+to+gene+localisation+in+eight+individuals+affected+by+keratoconus+from+north-west+Tasmania,+Australia&amp;author=J.+Fullerton&amp;author=P.+Paprocki&amp;author=S.+Foote&amp;author=D.A.+Mackey&amp;author=R.+Williamson&amp;volume=110&amp;publication_year=2002&amp;pages=462-470&amp;pmid=12073017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C15">15. <span class="element-citation">Hutchings H., Ginisty H., Le Gallo M., Levy D., Stoesser F., Rouland J.F., Arne J.L., Lalaux M.H., Calvas P., Roth M.P., et al.  Identification of a new locus for isolated familial keratoconus at 2p24. <span><span class="ref-journal">J. Med. Genet. </span>2005;<span class="ref-vol">42</span>:88–94.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1735904/">PMC free article</a>]</span> [<a href="/pubmed/15635082" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet.&amp;title=Identification+of+a+new+locus+for+isolated+familial+keratoconus+at+2p24&amp;author=H.+Hutchings&amp;author=H.+Ginisty&amp;author=M.+Le+Gallo&amp;author=D.+Levy&amp;author=F.+Stoesser&amp;volume=42&amp;publication_year=2005&amp;pages=88-94&amp;pmid=15635082&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C16">16. <span class="element-citation">Tang Y.G., Rabinowitz Y.S., Taylor K.D., Li X., Hu M., Picornell Y., Yang H. Genomewide linkage scan in a multigeneration Caucasian pedigree identifies a novel locus for keratoconus on chromosome 5q14.3-q21.1. <span><span class="ref-journal">Genet. Med. </span>2005;<span class="ref-vol">7</span>:397–405.</span> [<a href="/pubmed/16024971" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Med.&amp;title=Genomewide+linkage+scan+in+a+multigeneration+Caucasian+pedigree+identifies+a+novel+locus+for+keratoconus+on+chromosome+5q14.3-q21.1&amp;author=Y.G.+Tang&amp;author=Y.S.+Rabinowitz&amp;author=K.D.+Taylor&amp;author=X.+Li&amp;author=M.+Hu&amp;volume=7&amp;publication_year=2005&amp;pages=397-405&amp;pmid=16024971&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C17">17. <span class="element-citation">Tyynismaa H., Sistonen P., Tuupanen S., Tervo T., Dammert A., Latvala T., Alitalo T. A locus for autosomal dominant keratoconus: linkage to 16q22.3-q23.1 in Finnish families. <span><span class="ref-journal">Invest. Ophthalmol. Vis. Sci. </span>2002;<span class="ref-vol">43</span>:3160–3164.</span> [<a href="/pubmed/12356819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci.&amp;title=A+locus+for+autosomal+dominant+keratoconus:+linkage+to+16q22.3-q23.1+in+Finnish+families&amp;author=H.+Tyynismaa&amp;author=P.+Sistonen&amp;author=S.+Tuupanen&amp;author=T.+Tervo&amp;author=A.+Dammert&amp;volume=43&amp;publication_year=2002&amp;pages=3160-3164&amp;pmid=12356819&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C18">18. <span class="element-citation">Bisceglia L., Ciaschetti M., De Bonis P., Campo P.A., Pizzicoli C., Scala C., Grifa M., Ciavarella P., Delle Noci N., Vaira F., et al.  VSX1 mutational analysis in a series of Italian patients affected by keratoconus: detection of a novel mutation. <span><span class="ref-journal">Invest. Ophthalmol. Vis. Sci. </span>2005;<span class="ref-vol">46</span>:39–45.</span> [<a href="/pubmed/15623752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci.&amp;title=VSX1+mutational+analysis+in+a+series+of+Italian+patients+affected+by+keratoconus:+detection+of+a+novel+mutation&amp;author=L.+Bisceglia&amp;author=M.+Ciaschetti&amp;author=P.+De+Bonis&amp;author=P.A.+Campo&amp;author=C.+Pizzicoli&amp;volume=46&amp;publication_year=2005&amp;pages=39-45&amp;pmid=15623752&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C19">19. <span class="element-citation">Heon E., Greenberg A., Kopp K.K., Rootman D., Vincent A.L., Billingsley G., Priston M., Dorval K.M., Chow R.L., McInnes R.R., et al.  VSX1: a gene for posterior polymorphous dystrophy and keratoconus. <span><span class="ref-journal">Hum. Mol. Genet. </span>2002;<span class="ref-vol">11</span>:1029–1036.</span> [<a href="/pubmed/11978762" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=VSX1:+a+gene+for+posterior+polymorphous+dystrophy+and+keratoconus&amp;author=E.+Heon&amp;author=A.+Greenberg&amp;author=K.K.+Kopp&amp;author=D.+Rootman&amp;author=A.L.+Vincent&amp;volume=11&amp;publication_year=2002&amp;pages=1029-1036&amp;pmid=11978762&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C20">20. <span class="element-citation">Tang Y.G., Picornell Y., Su X., Li X., Yang H., Rabinowitz Y.S. Three VSX1 gene mutations, L159M, R166W, and H244R, are not associated with keratoconus. <span><span class="ref-journal">Cornea. </span>2008;<span class="ref-vol">27</span>:189–192.</span> [<a href="/pubmed/18216574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cornea&amp;title=Three+VSX1+gene+mutations,+L159M,+R166W,+and+H244R,+are+not+associated+with+keratoconus&amp;author=Y.G.+Tang&amp;author=Y.+Picornell&amp;author=X.+Su&amp;author=X.+Li&amp;author=H.+Yang&amp;volume=27&amp;publication_year=2008&amp;pages=189-192&amp;pmid=18216574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C21">21. <span class="element-citation">Tanwar M., Kumar M., Nayak B., Pathak D., Sharma N., Titiyal J.S., Dada R. VSX1 gene analysis in keratoconus. <span><span class="ref-journal">Mol. Vis. </span>2010;<span class="ref-vol">16</span>:2395–2401.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2994744/">PMC free article</a>]</span> [<a href="/pubmed/21139977" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Vis.&amp;title=VSX1+gene+analysis+in+keratoconus&amp;author=M.+Tanwar&amp;author=M.+Kumar&amp;author=B.+Nayak&amp;author=D.+Pathak&amp;author=N.+Sharma&amp;volume=16&amp;publication_year=2010&amp;pages=2395-2401&amp;pmid=21139977&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C22">22. <span class="element-citation">Kokotas H., Grigoriadou M., Petersen M.B. Review: age-related macular degeneration: genetic and clinical findings. <span><span class="ref-journal">Clin. Chem. Lab. Med. </span>2011;<span class="ref-vol">49</span>:610–616.</span> [<a href="/pubmed/21175380" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.+Lab.+Med.&amp;title=Review:+age-related+macular+degeneration:+genetic+and+clinical+findings&amp;author=H.+Kokotas&amp;author=M.+Grigoriadou&amp;author=M.B.+Petersen&amp;volume=49&amp;publication_year=2011&amp;pages=610-616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C23">23. <span class="element-citation">Fan B.J., Wiggs J.L. Glaucoma: genes, phenotypes, and new directions for therapy. <span><span class="ref-journal">J. Clin. Invest. </span>2010;<span class="ref-vol">120</span>:3064–3072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2929733/">PMC free article</a>]</span> [<a href="/pubmed/20811162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest.&amp;title=Glaucoma:+genes,+phenotypes,+and+new+directions+for+therapy&amp;author=B.J.+Fan&amp;author=J.L.+Wiggs&amp;volume=120&amp;publication_year=2010&amp;pages=3064-3072&amp;pmid=20811162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C24">24. <span class="element-citation">Young T.L., Metlapally R., Shay A.E. Complex trait genetics of refractive error. <span><span class="ref-journal">Arch. Ophthalmol. </span>2007;<span class="ref-vol">125</span>:38–48.</span> [<a href="/pubmed/17210850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Ophthalmol.&amp;title=Complex+trait+genetics+of+refractive+error&amp;author=T.L.+Young&amp;author=R.+Metlapally&amp;author=A.E.+Shay&amp;volume=125&amp;publication_year=2007&amp;pages=38-48&amp;pmid=17210850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C25">25. <span class="element-citation">Vitart V., Bencic G., Hayward C., Skunca Herman J., Huffman J., Campbell S., Bucan K., Navarro P., Gunjaca G., Marin J., et al.  New loci associated with central cornea thickness include COL5A1, AKAP13 and AVGR8. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:4304–4311.</span> [<a href="/pubmed/20719862" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=New+loci+associated+with+central+cornea+thickness+include+COL5A1,+AKAP13+and+AVGR8&amp;author=V.+Vitart&amp;author=G.+Bencic&amp;author=C.+Hayward&amp;author=J.+Skunca+Herman&amp;author=J.+Huffman&amp;volume=19&amp;publication_year=2010&amp;pages=4304-4311&amp;pmid=20719862&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C26">26. <span class="element-citation">Lu Y., Dimasi D.P., Hysi P.G., Hewitt A.W., Burdon K.P., Toh T., Ruddle J.B., Li Y.J., Mitchell P., Healey P.R., et al.  Common genetic variants near the Brittle Cornea Syndrome locus ZNF469 influence the blinding disease risk factor central corneal thickness. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000947.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2869325/">PMC free article</a>]</span> [<a href="/pubmed/20485516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Common+genetic+variants+near+the+Brittle+Cornea+Syndrome+locus+ZNF469+influence+the+blinding+disease+risk+factor+central+corneal+thickness&amp;author=Y.+Lu&amp;author=D.P.+Dimasi&amp;author=P.G.+Hysi&amp;author=A.W.+Hewitt&amp;author=K.P.+Burdon&amp;volume=6&amp;publication_year=2010&amp;pages=e1000947&amp;pmid=20485516&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C27">27. <span class="element-citation">Vithana E.N., Aung T., Khor C.C., Cornes B.K., Tay W.T., Sim X., Lavanya R., Wu R., Zheng Y., Hibberd M.L., et al.  Collagen-related genes influence the glaucoma risk factor, central corneal thickness. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">20</span>:649–658.</span> [<a href="/pubmed/21098505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Collagen-related+genes+influence+the+glaucoma+risk+factor,+central+corneal+thickness&amp;author=E.N.+Vithana&amp;author=T.+Aung&amp;author=C.C.+Khor&amp;author=B.K.+Cornes&amp;author=W.T.+Tay&amp;volume=20&amp;publication_year=2010&amp;pages=649-658&amp;pmid=21098505&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C28">28. <span class="element-citation">Ikram M.K., Sim X., Jensen R.A., Cotch M.F., Hewitt A.W., Ikram M.A., Wang J.J., Klein R., Klein B.E., Breteler M.M., et al.  Four novel loci (19q13, 6q24, 12q24, and 5q14) influence the microcirculation <em>in vivo</em>. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1001184.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2965750/">PMC free article</a>]</span> [<a href="/pubmed/21060863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Four+novel+loci+(19q13,+6q24,+12q24,+and+5q14)+influence+the+microcirculation+in+vivo&amp;author=M.K.+Ikram&amp;author=X.+Sim&amp;author=R.A.+Jensen&amp;author=M.F.+Cotch&amp;author=A.W.+Hewitt&amp;volume=6&amp;publication_year=2010&amp;pages=e1001184&amp;pmid=21060863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C29">29. <span class="element-citation">Darchen F., Goud B. Multiple aspects of Rab protein action in the secretory pathway: focus on Rab3 and Rab6. <span><span class="ref-journal">Biochimie. </span>2000;<span class="ref-vol">82</span>:375–384.</span> [<a href="/pubmed/10865125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochimie&amp;title=Multiple+aspects+of+Rab+protein+action+in+the+secretory+pathway:+focus+on+Rab3+and+Rab6&amp;author=F.+Darchen&amp;author=B.+Goud&amp;volume=82&amp;publication_year=2000&amp;pages=375-384&amp;pmid=10865125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C30">30. <span class="element-citation">Schluter O.M., Khvotchev M., Jahn R., Sudhof T.C. Localization versus function of Rab3 proteins. Evidence for a common regulatory role in controlling fusion. <span><span class="ref-journal">J. Biol. Chem. </span>2002;<span class="ref-vol">277</span>:40919–40929.</span> [<a href="/pubmed/12167638" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Localization+versus+function+of+Rab3+proteins.+Evidence+for+a+common+regulatory+role+in+controlling+fusion&amp;author=O.M.+Schluter&amp;author=M.+Khvotchev&amp;author=R.+Jahn&amp;author=T.C.+Sudhof&amp;volume=277&amp;publication_year=2002&amp;pages=40919-40929&amp;pmid=12167638&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C31">31. <span class="element-citation">Aligianis I.A., Johnson C.A., Gissen P., Chen D., Hampshire D., Hoffmann K., Maina E.N., Morgan N.V., Tee L., Morton J., et al.  Mutations of the catalytic subunit of RAB3GAP cause Warburg Micro syndrome. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:221–223.</span> [<a href="/pubmed/15696165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Mutations+of+the+catalytic+subunit+of+RAB3GAP+cause+Warburg+Micro+syndrome&amp;author=I.A.+Aligianis&amp;author=C.A.+Johnson&amp;author=P.+Gissen&amp;author=D.+Chen&amp;author=D.+Hampshire&amp;volume=37&amp;publication_year=2005&amp;pages=221-223&amp;pmid=15696165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C32">32. <span class="element-citation">Morris-Rosendahl D.J., Segel R., Born A.P., Conrad C., Loeys B., Brooks S.S., Muller L., Zeschnigk C., Botti C., Rabinowitz R., et al.  New RAB3GAP1 mutations in patients with Warburg Micro Syndrome from different ethnic backgrounds and a possible founder effect in the Danish. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2010;<span class="ref-vol">18</span>:1100–1106.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2987448/">PMC free article</a>]</span> [<a href="/pubmed/20512159" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=New+RAB3GAP1+mutations+in+patients+with+Warburg+Micro+Syndrome+from+different+ethnic+backgrounds+and+a+possible+founder+effect+in+the+Danish&amp;author=D.J.+Morris-Rosendahl&amp;author=R.+Segel&amp;author=A.P.+Born&amp;author=C.+Conrad&amp;author=B.+Loeys&amp;volume=18&amp;publication_year=2010&amp;pages=1100-1106&amp;pmid=20512159&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C33">33. <span class="element-citation">Aligianis I.A., Morgan N.V., Mione M., Johnson C.A., Rosser E., Hennekam R.C., Adams G., Trembath R.C., Pilz D.T., Stoodley N., et al.  Mutation in Rab3 GTPase-activating protein (RAB3GAP) noncatalytic subunit in a kindred with Martsolf syndrome. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">78</span>:702–707.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1424696/">PMC free article</a>]</span> [<a href="/pubmed/16532399" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Mutation+in+Rab3+GTPase-activating+protein+(RAB3GAP)+noncatalytic+subunit+in+a+kindred+with+Martsolf+syndrome&amp;author=I.A.+Aligianis&amp;author=N.V.+Morgan&amp;author=M.+Mione&amp;author=C.A.+Johnson&amp;author=E.+Rosser&amp;volume=78&amp;publication_year=2006&amp;pages=702-707&amp;pmid=16532399&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C34">34. <span class="element-citation">Cremers F., van de Pol D., Wieringa B., Collins F., Flintoff W., Siu V., Kaplan J., Frezal J., Ropers H.H. Molecular characterization of seven microdeletions and two X-autosome translocations in patients with choroideremia. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1989;<span class="ref-vol">45</span>:A182.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Molecular+characterization+of+seven+microdeletions+and+two+X-autosome+translocations+in+patients+with+choroideremia&amp;author=F.+Cremers&amp;author=D.+van+de+Pol&amp;author=B.+Wieringa&amp;author=F.+Collins&amp;author=W.+Flintoff&amp;volume=45&amp;publication_year=1989&amp;pages=A182&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C35">35. <span class="element-citation">Fukui K., Sasaki T., Imazumi K., Matsuura Y., Nakanishi H., Takai Y. Isolation and characterization of a GTPase activating protein specific for the Rab3 subfamily of small G proteins. <span><span class="ref-journal">J. Biol. Chem. </span>1997;<span class="ref-vol">272</span>:4655–4658.</span> [<a href="/pubmed/9030515" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Isolation+and+characterization+of+a+GTPase+activating+protein+specific+for+the+Rab3+subfamily+of+small+G+proteins&amp;author=K.+Fukui&amp;author=T.+Sasaki&amp;author=K.+Imazumi&amp;author=Y.+Matsuura&amp;author=H.+Nakanishi&amp;volume=272&amp;publication_year=1997&amp;pages=4655-4658&amp;pmid=9030515&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C36">36. <span class="element-citation">Rabinowitz Y.S., Dong L., Wistow G. Gene expression profile studies of human keratoconus cornea for NEIBank: a novel cornea-expressed gene and the absence of transcripts for aquaporin 5. <span><span class="ref-journal">Invest. Ophthalmol. Vis. Sci. </span>2005;<span class="ref-vol">46</span>:1239–1246.</span> [<a href="/pubmed/15790884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci.&amp;title=Gene+expression+profile+studies+of+human+keratoconus+cornea+for+NEIBank:+a+novel+cornea-expressed+gene+and+the+absence+of+transcripts+for+aquaporin+5&amp;author=Y.S.+Rabinowitz&amp;author=L.+Dong&amp;author=G.+Wistow&amp;volume=46&amp;publication_year=2005&amp;pages=1239-1246&amp;pmid=15790884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C37">37. <span class="element-citation">Bastian F., Parmentier G., Roux J., Moretti S., Laudet V., Robinson-Rechavi M. Bgee: Integrating and Comparing Heterogeneous Transcriptome Data Among Species.in DILS: Data Integration in Life Sciences. <span><span class="ref-journal">Lecture Notes in Computer Science. </span>2008;<span class="ref-vol">5109</span>:124–131.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lecture+Notes+in+Computer+Science&amp;title=Bgee:+Integrating+and+Comparing+Heterogeneous+Transcriptome+Data+Among+Species.in+DILS:+Data+Integration+in+Life+Sciences&amp;author=F.+Bastian&amp;author=G.+Parmentier&amp;author=J.+Roux&amp;author=S.+Moretti&amp;author=V.+Laudet&amp;volume=5109&amp;publication_year=2008&amp;pages=124-131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C38">38. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C39">39. <span class="element-citation">Klein R.J., Zeiss C., Chew E.Y., Tsai J.Y., Sackler R.S., Haynes C., Henning A.K., SanGiovanni J.P., Mane S.M., Mayne S.T., et al.  Complement factor H polymorphism in age-related macular degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:385–389.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1512523/">PMC free article</a>]</span> [<a href="/pubmed/15761122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+polymorphism+in+age-related+macular+degeneration&amp;author=R.J.+Klein&amp;author=C.+Zeiss&amp;author=E.Y.+Chew&amp;author=J.Y.+Tsai&amp;author=R.S.+Sackler&amp;volume=308&amp;publication_year=2005&amp;pages=385-389&amp;pmid=15761122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C40">40. <span class="element-citation">Fried L.P., Borhani N.O., Enright P., Furberg C.D., Gardin J.M., Kronmal R.A., Kuller L.H., Manolio T.A., Mittelmark M.B., Newman A., et al.  The Cardiovascular Health Study: design and rationale. <span><span class="ref-journal">Ann. Epidemiol. </span>1991;<span class="ref-vol">1</span>:263–276.</span> [<a href="/pubmed/1669507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Epidemiol.&amp;title=The+Cardiovascular+Health+Study:+design+and+rationale&amp;author=L.P.+Fried&amp;author=N.O.+Borhani&amp;author=P.+Enright&amp;author=C.D.+Furberg&amp;author=J.M.+Gardin&amp;volume=1&amp;publication_year=1991&amp;pages=263-276&amp;pmid=1669507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C41">41. <span class="element-citation">Psaty B.M., O'Donnell C.J., Gudnason V., Lunetta K.L., Folsom A.R., Rotter J.I., Uitterlinden A.G., Harris T.B., Witteman J.C., Boerwinkle E. Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium: design of prospective meta-analyses of genome-wide association studies from 5 cohorts. <span><span class="ref-journal">Circ. Cardiovasc. Genet. </span>2009;<span class="ref-vol">2</span>:73–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2875693/">PMC free article</a>]</span> [<a href="/pubmed/20031568" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet.&amp;title=Cohorts+for+Heart+and+Aging+Research+in+Genomic+Epidemiology+(CHARGE)+Consortium:+design+of+prospective+meta-analyses+of+genome-wide+association+studies+from+5+cohorts&amp;author=B.M.+Psaty&amp;author=C.J.+O'Donnell&amp;author=V.+Gudnason&amp;author=K.L.+Lunetta&amp;author=A.R.+Folsom&amp;volume=2&amp;publication_year=2009&amp;pages=73-80&amp;pmid=20031568&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C42">42. <span class="element-citation">Rabinowitz Y.S. Videokeratographic indices to aid in screening for keratoconus. <span><span class="ref-journal">J. Refract. Surg. </span>1995;<span class="ref-vol">11</span>:371–379.</span> [<a href="/pubmed/8528916" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Refract.+Surg.&amp;title=Videokeratographic+indices+to+aid+in+screening+for+keratoconus&amp;author=Y.S.+Rabinowitz&amp;volume=11&amp;publication_year=1995&amp;pages=371-379&amp;pmid=8528916&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C43">43. <span class="element-citation">Miller S.A., Dykes D.D., Polesky H.F. A simple salting out procedure for extracting DNA from human nucleated cells. <span><span class="ref-journal">Nucleic Acids Res. </span>1988;<span class="ref-vol">16</span>:1215.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC334765/">PMC free article</a>]</span> [<a href="/pubmed/3344216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=A+simple+salting+out+procedure+for+extracting+DNA+from+human+nucleated+cells&amp;author=S.A.+Miller&amp;author=D.D.+Dykes&amp;author=H.F.+Polesky&amp;volume=16&amp;publication_year=1988&amp;pages=1215&amp;pmid=3344216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C44">44. <span class="element-citation">Gunderson K.L., Steemers F.J., Lee G., Mendoza L.G., Chee M.S. A genome-wide scalable SNP genotyping assay using microarray technology. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:549–554.</span> [<a href="/pubmed/15838508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+scalable+SNP+genotyping+assay+using+microarray+technology&amp;author=K.L.+Gunderson&amp;author=F.J.+Steemers&amp;author=G.+Lee&amp;author=L.G.+Mendoza&amp;author=M.S.+Chee&amp;volume=37&amp;publication_year=2005&amp;pages=549-554&amp;pmid=15838508&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C45">45. <span class="element-citation">Gunderson K.L., Steemers F.J., Ren H., Ng P., Zhou L., Tsan C., Chang W., Bullis D., Musmacker J., King C., et al.  Whole-genome genotyping. <span><span class="ref-journal">Methods Enzymol. </span>2006;<span class="ref-vol">410</span>:359–376.</span> [<a href="/pubmed/16938560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol.&amp;title=Whole-genome+genotyping&amp;author=K.L.+Gunderson&amp;author=F.J.+Steemers&amp;author=H.+Ren&amp;author=P.+Ng&amp;author=L.+Zhou&amp;volume=410&amp;publication_year=2006&amp;pages=359-376&amp;pmid=16938560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C46">46. <span class="element-citation">Gunderson K.L., Kuhn K.M., Steemers F.J., Ng P., Murray S.S., Shen R. Whole-genome genotyping of haplotype tag single nucleotide polymorphisms. <span><span class="ref-journal">Pharmacogenomics. </span>2006;<span class="ref-vol">7</span>:641–648.</span> [<a href="/pubmed/16768648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Whole-genome+genotyping+of+haplotype+tag+single+nucleotide+polymorphisms&amp;author=K.L.+Gunderson&amp;author=K.M.+Kuhn&amp;author=F.J.+Steemers&amp;author=P.+Ng&amp;author=S.S.+Murray&amp;volume=7&amp;publication_year=2006&amp;pages=641-648&amp;pmid=16768648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C47">47. <span class="element-citation">Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=J.C.+Barrett&amp;author=B.+Fry&amp;author=J.+Maller&amp;author=M.J.+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C48">48. <span class="element-citation">De Bakker P.I. Tagger.  <a href="http://www.broad.mit.edu/mpg/tagger">http://www.broad.mit.edu/mpg/tagger</a> .</span></div><div class="ref-cit-blk half_rhythm" id="DDR460C49">49. <span class="element-citation">International HapMap Consortium. The International HapMap Project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+International+HapMap+Project&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C50">50. <span class="element-citation">Frazer K.A., Ballinger D.G., Cox D.R., Hinds D.A., Stuve L.L., Gibbs R.A., Belmont J.W., Boudreau A., Hardenbol P., Leal S.M., et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=K.A.+Frazer&amp;author=D.G.+Ballinger&amp;author=D.R.+Cox&amp;author=D.A.+Hinds&amp;author=L.L.+Stuve&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C51">51. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C52">52. <span class="element-citation">Chen M.H., Yang Q. GWAF: an R package for genome-wide association analyses with family data. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:580–581.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852219/">PMC free article</a>]</span> [<a href="/pubmed/20040588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GWAF:+an+R+package+for+genome-wide+association+analyses+with+family+data&amp;author=M.H.+Chen&amp;author=Q.+Yang&amp;volume=26&amp;publication_year=2010&amp;pages=580-581&amp;pmid=20040588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR460C53">53. <span class="element-citation">Pruim R.J., Welch R.P., Sanna S., Teslovich T.M., Chines P.S., Gliedt T.P., Boehnke M., Abecasis G.R., Willer C.J. LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=R.J.+Pruim&amp;author=R.P.+Welch&amp;author=S.+Sanna&amp;author=T.M.+Teslovich&amp;author=P.S.+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3276283&amp;issue-id=205406&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140002466284512title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140002465785088title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140002465782016title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3276283/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3276283/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3276283/pdf/ddr460.pdf">PDF (308K)</a></li> | <li><a href="#" data-citationid="PMC3276283" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3276283%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3276283%2F&amp;text=A%20genome-wide%20association%20study%20identifies%20a%20potential%20novel%20gene%20locus%20for%20keratoconus%2C%20one%20of%20the%20commonest%20causes%20for%20corneal%20transplantation%20in%20developed%20countries" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3276283%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21979947" data-aiid="3276283" data-aid="3276283" data-iid="205406" data-domainid="740" data-domain="hmg" data-accid="PMC3276283" data-md5="915ac8fe091d7e84b3554c06ea3a626d"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:50:52-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88B888DD532F3100000000024C0197&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3276283%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3276283/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>