<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3259012/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of body height in African Americans: the Women's Health Initiative SNP Health Association Resource (SHARe)</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3259012/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study of body height in African Americans: the Women's Health Initiative SNP Health Association Resource (SHARe)"><meta name="citation_authors" content="Cara L. Carty, Nicholas A. Johnson, Carolyn M. Hutter, Alexander P. Reiner, Ulrike Peters, Hua Tang, Charles Kooperberg"><meta name="citation_date" content="1 February 2012"><meta name="citation_issue" content="3"><meta name="citation_volume" content="21"><meta name="citation_firstpage" content="711"><meta name="citation_doi" content="10.1093/hmg/ddr489"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3259012/?report=abstract"><meta name="citation_pmid" content="22021425"><meta name="DC.Title" content="Genome-wide association study of body height in African Americans: the Women's Health Initiative SNP Health Association Resource (SHARe)"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Cara L. Carty"><meta name="DC.Contributor" content="Nicholas A. Johnson"><meta name="DC.Contributor" content="Carolyn M. Hutter"><meta name="DC.Contributor" content="Alexander P. Reiner"><meta name="DC.Contributor" content="Ulrike Peters"><meta name="DC.Contributor" content="Hua Tang"><meta name="DC.Contributor" content="Charles Kooperberg"><meta name="DC.Date" content="2012 Feb 1"><meta name="DC.Identifier" content="10.1093/hmg/ddr489"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of body height in African Americans: the Women's Health Initiative SNP Health Association Resource (SHARe)"><meta property="og:type" content="article"><meta property="og:description" content="Height is a complex trait under strong genetic influence. To date, numerous genetic loci have been associated with height in individuals of European ancestry. However, few large-scale discovery genome-wide association studies (GWAS) of height in minority ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3259012/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3259012/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3259012/pdf/ddr489.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE897849DD52ADE1000000000916063D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3259012%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="protein" class="last">Protein</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3259012</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2012 Feb 1; 21(3): 711–720. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Oct 21. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddr489" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddr489</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3259012</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22021425">22021425</a></div></div></div></div><h1 class="content-title">Genome-wide association study of body height in African Americans: the Women's Health Initiative SNP Health Association Resource (SHARe)</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Carty%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425" class="affpopup" co-rid="_co_idm140570579305456" co-class="co-affbox">Cara L. Carty</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Johnson%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425" class="affpopup" co-rid="_co_idm140570594725072" co-class="co-affbox">Nicholas A. Johnson</a>,<sup>2</sup> <a href="/pubmed/?term=Hutter%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425" class="affpopup" co-rid="_co_idm140570591771168" co-class="co-affbox">Carolyn M. Hutter</a>,<sup>1</sup> <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425" class="affpopup" co-rid="_co_idm140570595661248" co-class="co-affbox">Alexander P. Reiner</a>,<sup>1,</sup><sup>4</sup> <a href="/pubmed/?term=Peters%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425" class="affpopup" co-rid="_co_idm140570594073744" co-class="co-affbox">Ulrike Peters</a>,<sup>1,</sup><sup>4</sup> <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425" class="affpopup" co-rid="_co_idm140570591998848" co-class="co-affbox">Hua Tang</a>,<sup>3</sup> and  <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425" class="affpopup" co-rid="_co_idm140570579217248" co-class="co-affbox">Charles Kooperberg</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140570579305456"><h3 class="no_margin">Cara L. Carty</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA,</p><div>Find articles by <a href="/pubmed/?term=Carty%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425">Cara L. Carty</a></div></div><div id="_co_idm140570594725072"><h3 class="no_margin">Nicholas A. Johnson</h3><p><sup>2</sup>Department of Statistics,  and</p><div>Find articles by <a href="/pubmed/?term=Johnson%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425">Nicholas A. Johnson</a></div></div><div id="_co_idm140570591771168"><h3 class="no_margin">Carolyn M. Hutter</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA,</p><div>Find articles by <a href="/pubmed/?term=Hutter%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425">Carolyn M. Hutter</a></div></div><div id="_co_idm140570595661248"><h3 class="no_margin">Alexander P. Reiner</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA,</p><p><sup>4</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425">Alexander P. Reiner</a></div></div><div id="_co_idm140570594073744"><h3 class="no_margin">Ulrike Peters</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA,</p><p><sup>4</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Peters%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425">Ulrike Peters</a></div></div><div id="_co_idm140570591998848"><h3 class="no_margin">Hua Tang</h3><p><sup>3</sup>Department of Genetics, Stanford University, Stanford, CA 94305, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425">Hua Tang</a></div></div><div id="_co_idm140570579217248"><h3 class="no_margin">Charles Kooperberg</h3><p><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA,</p><div>Find articles by <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22021425">Charles Kooperberg</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140570584497776_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140570584497776_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140570584497776_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140570584497776_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Statistics,  and</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Genetics, Stanford University, Stanford, CA 94305, USA,  and</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed. Tel: +1 2066672873; Fax: +1 2066674142; Email: <a href="mailto:dev@null" data-email="gro.crchf@ytracc" class="oemail">gro.crchf@ytracc</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140570584497776_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Jul 8; Revised 2011 Sep 26; Accepted 2011 Oct 18.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140570584497776_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2011. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3259012/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supplementary Data
            </div>
            
            <div><a href="/pmc/articles/PMC3259012/bin/supp_21_3_711__index.html" data-ga-action="click_feat_suppl">supp_21_3_711__index.html</a><span style="color:gray"> (1.2K)</span></div><div class="small guid">GUID:&nbsp;1DA3C123-6E3D-4319-BF13-17533D9C67B1</div>
            <div><a href="/pmc/articles/PMC3259012/bin/supp_ddr489_ddr489supp.doc" data-ga-action="click_feat_suppl">supp_ddr489_ddr489supp.doc</a><span style="color:gray"> (489K)</span></div><div class="small guid">GUID:&nbsp;4E03035F-818F-45C7-8530-00067FA339EC</div>
         </div></dd></dl></div><div id="idm140570629090112" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140570629090112title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Height is a complex trait under strong genetic influence. To date, numerous genetic loci have been associated with height in individuals of European ancestry. However, few large-scale discovery genome-wide association studies (GWAS) of height in minority populations have been conducted and thus information about population-specific height regulation is limited. We conducted a GWA analysis of height in 8149 African-American (AA) women from the Women's Health Initiative. Genetic variants with <em>P</em>&lt; 5 × 10<sup>−5</sup> (<em>n</em> = 169) were followed up in a replication data set (<em>n</em> = 20 809) and meta-analyzed in a total of 28 958 AAs and African-descent individuals. Twelve single-nucleotide polymorphisms (SNPs) representing 7 independent loci were significantly associated with height at <em>P</em> &lt; 5 × 10<sup>−8</sup>. We identified novel SNPs in 17q23 (<em>TMEM100/PCTP</em>) and Xp22.3 (<em>ARSE</em>) reflecting population-specific regulation of height in AAs and replicated five loci previously reported in European-descent populations [4p15<em>/LCORL</em>, 11q13/<em>SERPINH1</em>, 12q14/<em>HMGA2</em>, 17q23/<em>MAP3K3</em> (mitogen-activated protein kinase3) and 18q21/<em>DYM</em>]. In addition, we performed an admixture mapping analysis of height which is both complementary and supportive to the GWA analysis and suggests potential associations between ancestry and height on chromosomes 4 (4q21), 15 (15q26) and 17 (17q23). Our findings provide insight into the genetic architecture of height and support the investigation of non-European-descent populations for identifying genetic factors associated with complex traits. Specifically, we identify new loci that may reflect population-specific regulation of height and report several known height loci that are important in determining height in African-descent populations.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first-last">Adult body height is a quantitative trait under strong genetic influence, with heritability estimates ranging from 0.69–0.98 (<a href="#DDR489C1" rid="DDR489C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544784">1</a>–<a href="#DDR489C3" rid="DDR489C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544769">3</a>). Yet, prior to 2007, candidate gene and genome-wide linkage studies had limited success in identifying genes consistently associated with height (<a href="#DDR489C4" rid="DDR489C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544808">4</a>). In recent years, genome-wide association studies (GWAS) conducted primarily in populations of European descent have identified new common genetic variants associated with height (<a href="#DDR489C5" rid="DDR489C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544755">5</a>–<a href="#DDR489C10" rid="DDR489C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544794">10</a>). A number of genes involved in skeletal growth and development, construction of extra-cellular matrix proteins as well as cell cycle regulation have been implicated (<a href="#DDR489C4" rid="DDR489C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544786">4</a>). A recent study of 183 727 European-descent individuals reported 180 loci influencing adult height (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544775">11</a>). Yet, genetic variation at these loci explains only ∼10% of the phenotypic variation in height, though the authors estimate that unidentified variants could account for up to 16% of the variation. Some of the remaining heritable contribution to height may be explained by small effects not detectable using current methods or by incomplete linkage disequilibrium (LD) between genotyped and causal single-nucleotide polymorphisms (SNPs) (<a href="#DDR489C12" rid="DDR489C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544796">12</a>). Race-specific variants may also contribute to height, and a better understanding of the genetic factors associated with height in minority populations could provide insight into the genetic architecture of height and other complex traits. However, to date, few GWAS of height have been conducted in non-European-descent populations with the exception of studies in Asian populations, including Chinese (<a href="#DDR489C13" rid="DDR489C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544759">13</a>), Japanese (<a href="#DDR489C14" rid="DDR489C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544760">14</a>), Korean (<a href="#DDR489C15" rid="DDR489C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544766">15</a>,<a href="#DDR489C16" rid="DDR489C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544819">16</a>) and Filipino (<a href="#DDR489C17" rid="DDR489C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544822">17</a>) individuals. One study of approximately 1930 African-descent individuals found 14 loci to be associated with height, though none reached genome-wide statistical significance (<a href="#DDR489C18" rid="DDR489C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544810">18</a>); a limitation of this study was its low power to detect variants of moderate effect. As an admixed population, AAs have a genetic profile reflecting the mixing of two ancestral, genetically distinct populations. Because this population admixture increases the genomic diversity, it has important implications for complex traits (<a href="#DDR489C19" rid="DDR489C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544806">19</a>,<a href="#DDR489C20" rid="DDR489C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544801">20</a>) and can be used to identify regions of the genome where African and European ancestral alleles are associated with height. Therefore, we conducted discovery GWA and admixture mapping analyses of height in AA women from the Women's Health Initiative (WHI) SNP Health Association Resource (SHARe). We then replicated our top GWA findings from WHI in a large data set of AA and African-descent individuals.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Discovery GWA results in WHI AAs</h3><p id="__p3" class="p p-first">The 8149 AA women included in the WHI SHARe height GWA had a mean (SD) age of 62 (<a href="#DDR489C7" rid="DDR489C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544802">7</a>) years. At baseline, ∼5 and 53% of the women self-reported having a doctor-diagnosis of osteoporosis and osteoarthritis, respectively. Body height ranged from 146.2 to 177.0 cm and was approximately normally distributed with a mean (SD) of 162.5 (6.2) cm. Height was negatively correlated with age in the age range of the WHI population (50–79 years) and hence analyses are adjusted for age. Although osteoarthritis was not associated with body height in the WHI population, osteoporosis was significantly associated with modest decreases in height, beta (SD) = −0.70 (0.33) cm, <em>P</em> = 0.03; women with osteoporosis were excluded in sensitivity analyses.</p><p id="__p4" class="p p-last">The WHI discovery GWA included 855 034 genotyped SNPs. The genomic control inflation factor <em>λ</em> for the association analyses was 1.087 after controlling for four eigenvectors, suggesting limited residual population substructure or cryptic relatedness (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr489/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). For polygenic traits such as height, elevated <em>λ</em> might be expected (<a href="#DDR489C21" rid="DDR489C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544768">21</a>). WHI SHARe GWA findings are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr489/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S2</a>. In sensitivity analyses, we attempted to replicate GWAS findings (14 SNPs which did not reach genome-wide significance) from one study of height in AAs (<a href="#DDR489C18" rid="DDR489C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544756">18</a>); none of these SNPs was significant at <em>P</em> = 0.05/14 in our WHI AA data set (data not shown).</p></div><div id="s2b" class="sec"><h3 id="s2btitle">Replication and combined meta-analysis of GWA findings</h3><p id="__p5" class="p p-first">The replication data set included up to 20 809 AAs and African-descent individuals. Adding WHI SHARe to these numbers, we had a total sample size of 28 958 individuals for the meta-analysis (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr489/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>), though it varied by SNP. A total of 169 SNPs with <em>P</em> &lt; 5 × 10<sup>−5</sup> from the WHI-SHARe GWA discovery phase were carried forward to the replication phase. Of these, 12 were significant in the combined WHI discovery + replication meta-analysis at <em>P</em> &lt; 5 × 10<sup>−8</sup> (Table&nbsp;<a href="/pmc/articles/PMC3259012/table/DDR489TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR489TB1" rid-ob="ob-DDR489TB1" co-legend-rid=""><span>1</span></a>). Of the 12 significant SNPs, 9 were in regions previously reported to be associated with height in largely European-descent populations (4p15, 11q13, 12q14, 17q23/<em>MAP3K3,</em> 17q24/<em>PRKCA</em> and 18q21) and the remaining 3 SNPs appear to reflect novel regions (17q23/<em>TMEM100-PCTP</em> and Xp22) in AAs.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR489TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p6">Significant findings from the replication meta-analysis in AAs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570592501264" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="top"><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="left">Locus</th><th rowspan="2" colspan="1" align="left">Position<sup>a</sup></th><th rowspan="2" colspan="1" align="left">Closest gene(s)</th><th rowspan="2" colspan="1" align="left">Allele</th><th rowspan="2" colspan="1" align="left">CEU AF<sup>b</sup></th><th rowspan="2" colspan="1" align="left">YRI AF<sup>b</sup></th><th colspan="4" rowspan="1" align="left">WHI<hr></th><th colspan="4" rowspan="1" align="left">Replication data set<hr></th><th rowspan="2" colspan="1" align="left">Meta-analysis <em>P</em>-value<sup>d</sup></th></tr><tr><th rowspan="1" colspan="1" align="left">AF</th><th rowspan="1" colspan="1" align="left"><em>n</em></th><th rowspan="1" colspan="1" align="left">Beta ± SE<sup>c</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>c</sup></th><th rowspan="1" colspan="1" align="left">AF</th><th rowspan="1" colspan="1" align="left"><em>n</em></th><th rowspan="1" colspan="1" align="left">Beta ± SE<sup>c</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>c</sup></th></tr></thead><tbody><tr><td colspan="16" rowspan="1">Regions not previously associated with height</td></tr><tr><td rowspan="1" colspan="1"> rs1549519<sup>e</sup></td><td rowspan="1" colspan="1">17q23</td><td rowspan="1" colspan="1">51 112 786</td><td rowspan="1" colspan="1"><em>TMEM100</em>/<em>PCTP</em></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">8046</td><td rowspan="1" colspan="1">−0.095 ± 0.020</td><td rowspan="1" colspan="1">2.19 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">20 413</td><td rowspan="1" colspan="1">−0.048 ± 0.012</td><td rowspan="1" colspan="1">7.99 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">5.80 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1"> rs1426745<sup>e</sup></td><td rowspan="1" colspan="1">17q23</td><td rowspan="1" colspan="1">51 131 494</td><td rowspan="1" colspan="1"><em>TMEM100</em>/<em>PCTP</em></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">0.76</td><td rowspan="1" colspan="1">8100</td><td rowspan="1" colspan="1">0.092 ± 0.019</td><td rowspan="1" colspan="1">2.09 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.78</td><td rowspan="1" colspan="1">20 347</td><td rowspan="1" colspan="1">0.041 ± 0.012</td><td rowspan="1" colspan="1">4.72 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">4.82 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1"> rs12393627</td><td rowspan="1" colspan="1">Xp22</td><td rowspan="1" colspan="1">2 895 723</td><td rowspan="1" colspan="1"><em>ARSE</em></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">&lt;0.01</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">8146</td><td rowspan="1" colspan="1">0.079 ± 0.016</td><td rowspan="1" colspan="1">1.46 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">7584</td><td rowspan="1" colspan="1">0.093 ± 0.019</td><td rowspan="1" colspan="1">6.35 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">4.96 × 10<sup>−12</sup></td></tr><tr><td colspan="16" rowspan="1">Regions previously associated with height</td></tr><tr><td rowspan="1" colspan="1"> rs925098</td><td rowspan="1" colspan="1">4p15</td><td rowspan="1" colspan="1">17 528 909</td><td rowspan="1" colspan="1"><em>LCORL</em></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">8101</td><td rowspan="1" colspan="1">0.084 ± 0.016</td><td rowspan="1" colspan="1">3.31 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">20 451</td><td rowspan="1" colspan="1">0.056 ± 0.010</td><td rowspan="1" colspan="1">5.55 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">3.05 × 10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1"> rs2320299</td><td rowspan="1" colspan="1">4p15</td><td rowspan="1" colspan="1">17 581 470</td><td rowspan="1" colspan="1"><em>LCORL</em></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">8140</td><td rowspan="1" colspan="1">0.076 ± 0.016</td><td rowspan="1" colspan="1">2.54 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">20 496</td><td rowspan="1" colspan="1">0.049 ± 0.010</td><td rowspan="1" colspan="1">2.61 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">8.50 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1"> rs2011603</td><td rowspan="1" colspan="1">4p15</td><td rowspan="1" colspan="1">17 634 582</td><td rowspan="1" colspan="1"><em>LCORL</em></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.64</td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">8147</td><td rowspan="1" colspan="1">−0.096 ± 0.016</td><td rowspan="1" colspan="1">6.51 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">20 495</td><td rowspan="1" colspan="1">−0.039 ± 0.010</td><td rowspan="1" colspan="1">1.12 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">2.71 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1"> rs606452</td><td rowspan="1" colspan="1">11q13</td><td rowspan="1" colspan="1">74 953 826</td><td rowspan="1" colspan="1"><em>SERPINH1</em></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.55</td><td rowspan="1" colspan="1">0.65</td><td rowspan="1" colspan="1">8081</td><td rowspan="1" colspan="1">−0.077 ± 0.017</td><td rowspan="1" colspan="1">3.47 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.64</td><td rowspan="1" colspan="1">20 474</td><td rowspan="1" colspan="1">−0.043 ± 0.010</td><td rowspan="1" colspan="1">1.96 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.56 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1"> rs7968682</td><td rowspan="1" colspan="1">12q14</td><td rowspan="1" colspan="1">64 658 147</td><td rowspan="1" colspan="1"><em>HMGA2</em></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">8144</td><td rowspan="1" colspan="1">−0.073 ± 0.016</td><td rowspan="1" colspan="1">5.49 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">20 484</td><td rowspan="1" colspan="1">−0.043 ± 0.009</td><td rowspan="1" colspan="1">4.68 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">4.22 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1"> rs11658329</td><td rowspan="1" colspan="1">17q23</td><td rowspan="1" colspan="1">59 116 763</td><td rowspan="1" colspan="1"><em>MAP3K3</em></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.72</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">8140</td><td rowspan="1" colspan="1">−0.072 ± 0.017</td><td rowspan="1" colspan="1">2.90 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">20 415</td><td rowspan="1" colspan="1">−0.049 ± 0.010</td><td rowspan="1" colspan="1">2.07 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">4.89 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1"> rs3889237</td><td rowspan="1" colspan="1">17q24</td><td rowspan="1" colspan="1">62 209 892</td><td rowspan="1" colspan="1"><em>PRKCA</em></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">&lt;0.01</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">8015</td><td rowspan="1" colspan="1">−0.096 ± 0.023</td><td rowspan="1" colspan="1">3.40 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">19 096</td><td rowspan="1" colspan="1">−0.056 ± 0.014</td><td rowspan="1" colspan="1">7.92 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">3.25 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1"> rs357897<sup>e</sup></td><td rowspan="1" colspan="1">18q21</td><td rowspan="1" colspan="1">44 838 182</td><td rowspan="1" colspan="1"><em>DYM</em><sup>f</sup></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">8139</td><td rowspan="1" colspan="1">0.071 ± 0.016</td><td rowspan="1" colspan="1">1.69 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">20 496</td><td rowspan="1" colspan="1">0.044 ± 0.010</td><td rowspan="1" colspan="1">5.96 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">1.09 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1"> rs1787200<sup>e</sup></td><td rowspan="1" colspan="1">18q21</td><td rowspan="1" colspan="1">44 841 652</td><td rowspan="1" colspan="1"><em>DYM</em><sup>f</sup></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">8146</td><td rowspan="1" colspan="1">0.074 ± 0.016</td><td rowspan="1" colspan="1">6.00 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">20 478</td><td rowspan="1" colspan="1">0.044 ± 0.010</td><td rowspan="1" colspan="1">8.04 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">7.37 × 10<sup>−10</sup></td></tr></tbody></table></div><div id="largeobj_idm140570592501264" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3259012/table/DDR489TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140570583076832"><p id="__p7" class="p p-first-last">Chr, chromosome; AF, coded allele frequency; SE, standard error.</p></div><div id="idm140570583076208"><p id="__p8" class="p p-first-last"><sup>a</sup>Chromosome position (Build 36).</p></div><div id="idm140570583075328"><p id="__p9" class="p p-first-last"><sup>b</sup>HapMap CEU and YRI allele frequencies.</p></div><div id="idm140570583074448"><p id="__p10" class="p p-first-last"><sup>c</sup>Beta coefficient representing the difference in mean height <em>Z</em>-score associated with each additional allele, and corresponding <em>P</em>-value.</p></div><div id="idm140570585155072"><p id="__p11" class="p p-first-last"><sup>d</sup>Meta-analysis of height <em>Z</em>-scores with inverse variance weighting.</p></div><div id="idm140570628324096"><p id="__p12" class="p p-first-last"><sup>e</sup>These SNPs are in high LD with each other in WHI AAs.</p></div><div id="idm140570628323216"><p id="__p13" class="p p-first-last"><sup>f</sup>Although the <em>DYM</em> region has been previously associated with height in European Americans, these SNPs are not in LD with the reported SNPs.</p></div></div></div><p id="__p14">Our top novel finding was rs12393627 (<em>P</em> = 5 × 10<sup>−12</sup>) located at Xp22, which has not been reported previously. The top two SNPs at 18q21 are located in the dymeclin (<em>DYM)</em> gene and are in high LD with each other in the WHI AA, <em>r</em><sup>2</sup> &gt; 0.95 (Fig.&nbsp;<a href="/pmc/articles/PMC3259012/figure/DDR489F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR489F1" rid-ob="ob-DDR489F1" co-legend-rid="lgnd_DDR489F1"><span>1</span></a>). Another <em>DYM</em> SNP, rs8099594, has been previously associated with height in European-descent populations (<a href="#DDR489C4" rid="DDR489C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544763">4</a>). In conditional analyses, adjustment for rs8099594 did not substantially change findings for either of our top <em>DYM</em> SNPs (Table&nbsp;<a href="/pmc/articles/PMC3259012/table/DDR489TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR489TB2" rid-ob="ob-DDR489TB2" co-legend-rid=""><span>2</span></a>). <em>DYM</em> is a large gene spanning almost 400 kb. Our <em>DYM</em> SNPs, rs357897 and rs1787200, are located &gt;370 kb from the previously reported European height-associated locus and are in low LD with it in the WHI AA population (<em>r</em><sup>2</sup> &lt; 0.3) as well as in HapMap CEU (<em>r</em><sup>2</sup> &lt; 0.2) and YRI (<em>r</em><sup>2</sup> &lt; 0.3). rs8099594 was not significantly associated with height in the WHI AA population, <em>P</em> = 0.208.

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR489TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p15">Conditional analysis for significant loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570628416112" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="top"><tr><th rowspan="1" colspan="1" align="left">Locus</th><th rowspan="1" colspan="1" align="left">Target SNP</th><th rowspan="1" colspan="1" align="left">Adjustment SNP</th><th rowspan="1" colspan="1" align="left"><em>n</em> (WHI only)</th><th rowspan="1" colspan="1" align="left">Target SNP marginal <em>P</em>-value<sup>a</sup></th><th rowspan="1" colspan="1" align="left">Target SNP conditional <em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">17q23/<em>TMEM100</em></td><td rowspan="1" colspan="1">rs1549519</td><td rowspan="1" colspan="1">rs11658329/<em>MAP3K3</em></td><td rowspan="1" colspan="1">8037</td><td rowspan="1" colspan="1">1.95 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">4.57 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">17q23/<em>TMEM100</em></td><td rowspan="1" colspan="1">rs1426745</td><td rowspan="1" colspan="1">rs11658329/<em>MAP3K3</em></td><td rowspan="1" colspan="1">8091</td><td rowspan="1" colspan="1">1.87 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">4.13 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">17q24/<em>PRKCA</em></td><td rowspan="1" colspan="1">rs3889237</td><td rowspan="1" colspan="1">rs11658329/<em>MAP3K3</em></td><td rowspan="1" colspan="1">8006</td><td rowspan="1" colspan="1">3.67 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.27 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1">18q21/<em>DYM</em></td><td rowspan="1" colspan="1">rs357897</td><td rowspan="1" colspan="1">rs8099594/<em>DYM</em></td><td rowspan="1" colspan="1">8137</td><td rowspan="1" colspan="1">1.85 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.83 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">18q21/<em>DYM</em></td><td rowspan="1" colspan="1">rs1787200</td><td rowspan="1" colspan="1">rs8099594/<em>DYM</em></td><td rowspan="1" colspan="1">8140</td><td rowspan="1" colspan="1">6.88 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">6.47 × 10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140570628416112" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3259012/table/DDR489TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140570631028864"><p id="__p16" class="p p-first-last"><sup>a</sup><em>P</em>-values may differ from those in Table&nbsp;<a href="/pmc/articles/PMC3259012/table/DDR489TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR489TB1" rid-ob="ob-DDR489TB1" co-legend-rid=""><span>1</span></a> because only individuals with complete data for both the target and adjustment SNPs were included in the models.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR489F1" co-legend-rid="lgnd_DDR489F1"><a href="/pmc/articles/PMC3259012/figure/DDR489F1/" target="figure" rid-figpopup="DDR489F1" rid-ob="ob-DDR489F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570631082352" class="figure"><a href="/pmc/articles/PMC3259012/figure/DDR489F1/" target="figure" rid-figpopup="DDR489F1" rid-ob="ob-DDR489F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3259012_ddr48901.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr48901.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3259012/bin/ddr48901.jpg"></a></div><div id="largeobj_idm140570631082352" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3259012/figure/DDR489F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR489F1"><div><a class="figpopup" href="/pmc/articles/PMC3259012/figure/DDR489F1/" target="figure" rid-figpopup="DDR489F1" rid-ob="ob-DDR489F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p17">Height association <em>P</em>-value and LD plot of the chromosome 18 <em>DYM</em> region <em>P</em>-values for WHI-only GWA results for SNPs within 150 kb of the top SNP from the replication, rs1787200, in the <em>DYM</em> region are shown. LD between rs1787200 and other SNPs in WHI is indicated using a gray scale, with black indicating the highest LD and white, the lowest LD. rs1787200 (black diamond) is in high LD with the other top <em>DYM</em> SNP from the replication meta-analysis, rs357897 (black circle). Top SNPs for the WHI-only analysis (rs7239949, rs833520 and rs181553) are also labeled.</p></div></div></div><p id="__p18">Within 17q23–24, we have evidence both confirming previously identified loci and for a novel finding. The 17q23 SNP with the lowest <em>P</em>-value = 5 × 10<sup>−10</sup>, rs11658329, is located in an intron of the <em>MAP3K3</em> gene. In CEU, it is in strong LD (<em>r</em><sup>2</sup> = 0.96) with rs2665838, an SNP in the chorionic somatomammotropin hormone 1 (<em>CSH1</em>)/growth hormone 1 (<em>GH1</em>) locus which has been associated with height in European-descent populations (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544792">11</a>). rs11658329 is also in strong LD (<em>r</em><sup>2</sup> &gt; 0.99) with rs8081612, an SNP in <em>MAP3K3</em> previously associated with height in European-descent individuals and confirmed in a multi-ethnic cohort (<a href="#DDR489C22" rid="DDR489C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544779">22</a>), which included small numbers of WHI AAs. We also identified two SNPs in the 17q23 locus, rs1549519 and rs1426745, which are located in the 3′ flanking region of the <em>TMEM100</em> gene and are in high LD with each other (<em>r</em><sup>2</sup> = 0.89) (Fig.&nbsp;<a href="/pmc/articles/PMC3259012/figure/DDR489F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR489F2" rid-ob="ob-DDR489F2" co-legend-rid="lgnd_DDR489F2"><span>2</span></a>). Additionally, we identified rs3889237 in the nearby 17q24 locus, which is located in an intron of the <em>PRKCA</em> gene. In a conditional analysis, adjustment of the two 17q23 <em>TMEM100</em> SNPs for the <em>MAP3K3</em> SNP did not substantially affect their significance (Table&nbsp;<a href="/pmc/articles/PMC3259012/table/DDR489TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR489TB2" rid-ob="ob-DDR489TB2" co-legend-rid=""><span>2</span></a>). However, in spite of the large distance between the <em>PRKCA</em> SNP rs3889237 and the <em>MAP3K3</em> SNP, results from the conditional analysis suggest that much of the rs3889237 association is explained by the <em>MAP3K3</em> SNP. For this reason, we classify the 17q24/<em>PRKCA</em> finding as a region previously associated with height in Table&nbsp;<a href="/pmc/articles/PMC3259012/table/DDR489TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR489TB1" rid-ob="ob-DDR489TB1" co-legend-rid=""><span>1</span></a>, whereas the <em>TMEM100</em> SNPs appear to be novel. In summary, we find evidence for two independent height-associated loci in chromosome 17q23–24 region: a locus near <em>TMEM100-PCTP</em> newly reported in AAs, and a second locus near the previously reported <em>CSH1-GH1</em> European height region that, in AAs, is characterized by extended LD and encompasses <em>MAP3K3</em> and <em>PRCKA</em>.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR489F2" co-legend-rid="lgnd_DDR489F2"><a href="/pmc/articles/PMC3259012/figure/DDR489F2/" target="figure" rid-figpopup="DDR489F2" rid-ob="ob-DDR489F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570593636720" class="figure"><a href="/pmc/articles/PMC3259012/figure/DDR489F2/" target="figure" rid-figpopup="DDR489F2" rid-ob="ob-DDR489F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3259012_ddr48902.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr48902.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3259012/bin/ddr48902.jpg"></a></div><div id="largeobj_idm140570593636720" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3259012/figure/DDR489F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR489F2"><div><a class="figpopup" href="/pmc/articles/PMC3259012/figure/DDR489F2/" target="figure" rid-figpopup="DDR489F2" rid-ob="ob-DDR489F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p19">Height association <em>P</em>-value and LD plot of the chromosome 17 <em>TMEM100-PCTP</em> region <em>P</em>-values for WHI-only GWA results for SNPs within 100 kb of the top hit in the <em>TMEM100-PCTP</em> region are shown. LD between rs1549519 and other SNPs in this region in the study population is indicated using a gray scale, with black indicating the highest LD and white, the lowest LD. rs1549519 (black diamond) is in high LD with the other top SNP from the replication meta-analysis, rs1426745.</p></div></div></div><p id="__p20">For the other known regions, variants at 4p15, 11q23 and 12q24 were in moderate-to-strong LD (<em>r</em><sup>2</sup> &gt; 0.7) in CEU with at least one SNP previously identified in European populations. The LD pattern for the 4p15 region is shown in Figure&nbsp;<a href="/pmc/articles/PMC3259012/figure/DDR489F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR489F3" rid-ob="ob-DDR489F3" co-legend-rid="lgnd_DDR489F3"><span>3</span></a>. The SNP with the lowest <em>P</em>-value in the replication, rs925098, is located in an intron of the ligand-dependent nuclear receptor co-repressor-like (<em>LCORL</em>) gene. It is in high LD, <em>r</em><sup>2</sup> &gt; 0.7, with the other top SNPs that we identified in this region, rs2011603 and rs2320299. rs925098 is not in LD (<em>r</em><sup>2</sup> ≤ 0.1) in HapMap CEU with the previously reported <em>LCORL</em> SNPs: rs6449353 (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544798">11</a>), rs6830062 (<a href="#DDR489C7" rid="DDR489C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544797">7</a>,<a href="#DDR489C10" rid="DDR489C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544818">10</a>), rs7678436 (<a href="#DDR489C14" rid="DDR489C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544776">14</a>) or rs16896068 (<a href="#DDR489C5" rid="DDR489C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544816">5</a>); however, it is in moderate-to-strong LD with rs6842303 (<a href="#DDR489C7" rid="DDR489C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544785">7</a>) (<em>r</em><sup>2</sup> = 0.76) and rs2724475 (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544773">11</a>) (<em>r</em><sup>2</sup> = 0.96), indicating that it is likely not a novel finding. Figure&nbsp;<a href="/pmc/articles/PMC3259012/figure/DDR489F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR489F3" rid-ob="ob-DDR489F3" co-legend-rid="lgnd_DDR489F3"><span>3</span></a> also shows LD and association data in the WHI data set for the three previously reported <em>LCORL</em> SNPs that were genotyped in WHI (rs6830062, rs6842303 and rs16896068).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR489F3" co-legend-rid="lgnd_DDR489F3"><a href="/pmc/articles/PMC3259012/figure/DDR489F3/" target="figure" rid-figpopup="DDR489F3" rid-ob="ob-DDR489F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570578830640" class="figure"><a href="/pmc/articles/PMC3259012/figure/DDR489F3/" target="figure" rid-figpopup="DDR489F3" rid-ob="ob-DDR489F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3259012_ddr48903.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr48903.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3259012/bin/ddr48903.jpg"></a></div><div id="largeobj_idm140570578830640" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3259012/figure/DDR489F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR489F3"><div><a class="figpopup" href="/pmc/articles/PMC3259012/figure/DDR489F3/" target="figure" rid-figpopup="DDR489F3" rid-ob="ob-DDR489F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p21">Height association <em>P</em>-value and LD plot of the chromosome 4 <em>LCORL</em> region <em>P</em>-values for WHI-only GWA results within 150 kb of the top SNP identified in the replication meta-analysis, rs925098, in the <em>LCORL</em> region are shown. rs925098 is labeled with a black diamond. LD between rs925098 and other SNPs in this region in the WHI study population is indicated using a gray scale, with black indicating the highest LD and white, the lowest LD. SNPs in the region with modestly significant associations with height and in high LD (<em>r</em><sup>2</sup> &gt; 0.6) with rs925098 include rs232099 and rs2011603. These data suggest that these three SNPs may not be independent. Other SNPs previously reported to be associated with height in European-descent populations (rs6842303, rs16896068 and rs6830062) are in lower LD with rs925098.</p></div></div></div><p id="__p22" class="p p-last">In an additional sensitivity analysis, we used publically available height GWAS results in approximately 130 000 European-descent individuals available from the GIANT consortium (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544758">11</a>) to look up our top 11 findings from the autosomes; X chromosome results were not available. Of the 11 autosomal SNPs, 8 SNPs from the 4p15, 11q13, 12q14, 17q23 and 18q21 regions also reached genome-wide significance in the European-descent GIANT population (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr489/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). However, the two novel 17q23 (<em>TMEM100/PCTP)</em> SNPs were not significant in GIANT (with <em>P</em>-values ranging from 0.14 to 0.18); nor was the 17q24 <em>PRKCA</em> SNP, rs3889237.</p></div><div id="s2c" class="sec sec-last"><h3 id="s2ctitle">Admixture mapping results in WHI AAs</h3><p id="__p23" class="p p-first-last">Admixture mapping identifies regions of the genome where the ancestry of a chromosomal region is associated with the trait of interest, after adjusting for the genome-wide average ancestry proportions. Results from the height admixture mapping analyses in WHI are shown in Figure&nbsp;<a href="/pmc/articles/PMC3259012/figure/DDR489F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR489F4" rid-ob="ob-DDR489F4" co-legend-rid="lgnd_DDR489F4"><span>4</span></a>. Low <em>P</em>-values meeting the genome-wide significance threshold in the admixture analysis (which is <em>P</em> &lt; 1 × 10<sup>−5</sup>) on chromosomes 4, 12, 15 and 17 suggest that these regions may harbor variants that both influence height and have large allele frequency differences between the European and African ancestral populations. The strongest signal with the lowest <em>P</em>-values (<em>P</em> = 1–2 × 10<sup>−6</sup>) was found on chromosome 4 between 80 and 82 Mb (4q21). Within this region, several SNPs show suggestive genotype associations and thus could explain the admixture mapping signal. For example, rs899900 is moderately associated with height in WHI, <em>P</em> = 6.7 × 10<sup>−7</sup>. In conditional analyses, adjustment for rs899900 explains the majority of the admixture mapping signal. Although this chromosome 4 region does not overlap with the WHI GWA findings which were centered at 4p15, we found some moderate height associations (5 × 10<sup>−5</sup>&lt; <em>P</em> &lt; 1 × 10<sup>−8</sup>) on either end of this admixture region (Fig.&nbsp;<a href="/pmc/articles/PMC3259012/figure/DDR489F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR489F5" rid-ob="ob-DDR489F5" co-legend-rid="lgnd_DDR489F5"><span>5</span></a>A). On chromosome 17, we saw evidence for ancestral associations with height in the region between 40 and 60 Mb; these results are consistent with our SNP associations in this region (Fig.&nbsp;<a href="/pmc/articles/PMC3259012/figure/DDR489F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR489F5" rid-ob="ob-DDR489F5" co-legend-rid="lgnd_DDR489F5"><span>5</span></a>B). In fact, the majority of this ancestry signal is explained by an SNP near the <em>TMEM100</em> gene, rs1549519. Results for chromosome 15 (15q26) were marginally significant, but again consistent for the SNP associations in this region (Fig.&nbsp;<a href="/pmc/articles/PMC3259012/figure/DDR489F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR489F5" rid-ob="ob-DDR489F5" co-legend-rid="lgnd_DDR489F5"><span>5</span></a>C). The modest peak on chromosome 12 is centered at the P-terminus close to the end of the chromosome where the local ancestry estimates tend to be noisier, and should be interpreted cautiously.

</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR489F4" co-legend-rid="lgnd_DDR489F4"><a href="/pmc/articles/PMC3259012/figure/DDR489F4/" target="figure" rid-figpopup="DDR489F4" rid-ob="ob-DDR489F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570590146016" class="figure"><a href="/pmc/articles/PMC3259012/figure/DDR489F4/" target="figure" rid-figpopup="DDR489F4" rid-ob="ob-DDR489F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3259012_ddr48904.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr48904.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3259012/bin/ddr48904.jpg"></a></div><div id="largeobj_idm140570590146016" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3259012/figure/DDR489F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR489F4"><div><a class="figpopup" href="/pmc/articles/PMC3259012/figure/DDR489F4/" target="figure" rid-figpopup="DDR489F4" rid-ob="ob-DDR489F4">Figure&nbsp;4.</a></div><!--caption a7--><div class="caption"><p id="__p24">WHI admixture mapping results. A plot of admixture mapping −log<sub>10</sub> <em>P</em>-value results in the WHI data set on chromosomes 1–22 and X is shown. Much like a Manhattan plot, <em>P</em>-values are plotted in order of SNP position on the chromosome. The tall peaks reaching <em>P</em>-values of &lt;1 × 10<sup>−5</sup> suggest that SNPs indicative of African ancestry on chromosomes 4, 12 and 17 are associated with height.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR489F5" co-legend-rid="lgnd_DDR489F5"><a href="/pmc/articles/PMC3259012/figure/DDR489F5/" target="figure" rid-figpopup="DDR489F5" rid-ob="ob-DDR489F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140570591374192" class="figure"><a href="/pmc/articles/PMC3259012/figure/DDR489F5/" target="figure" rid-figpopup="DDR489F5" rid-ob="ob-DDR489F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3259012_ddr48905.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr48905.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3259012/bin/ddr48905.jpg"></a></div><div id="largeobj_idm140570591374192" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3259012/figure/DDR489F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR489F5"><div><a class="figpopup" href="/pmc/articles/PMC3259012/figure/DDR489F5/" target="figure" rid-figpopup="DDR489F5" rid-ob="ob-DDR489F5">Figure&nbsp;5.</a></div><!--caption a7--><div class="caption"><p id="__p25">Comparison of GWA and admixture results in the WHI data set for chromosomal regions 4 (<strong>A</strong>), 17 (<strong>B</strong>) and 15 (<strong>C</strong>). In the WHI data set, GWA results for height are contrasted with height admixture mapping results for the same regions on the chromosome. Locations on the respective chromosome (in base pairs) are shown on the <em>X</em>-axes and −log<sub>10</sub> <em>P</em>-values are indicated on the <em>Y</em>-axes.</p></div></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p26" class="p p-first">In this discovery GWAS of height in AA women and replication population including a combined sample of more than 28 000 African-descent individuals, we identified new loci on chromosomes 17 (<em>TMEM100/PCTP</em>), 18 (<em>DYM</em>) and X (<em>ARSE</em>) significantly associated with height in AAs and replicated several height loci previously reported in European-descent individuals (4p15/<em>LCORL</em>, <em>11q13/SERPINH1</em>, 12q14/<em>HMGA2</em> and 17q23/<em>MAP3K3</em>). Using admixture mapping as a complementary approach to detect association signals in AAs, we identified a region on chromosome 4, independent of the <em>LCORL</em> locus, and regions on chromosomes 15 and 17, which potentially harbor population-specific variants that influence height, though these findings require further confirmation. Height is a trait that is influenced by many loci, some appear to be population specific, such as our GWA findings for chromosomes 17q23 and Xp22.3 and admixture findings for chromosome 4, whereas others are similar across European- and African-descent populations.</p><p id="__p27">Of the novel findings, we identified a significant SNP in the Xp22.3 locus, rs12393627, which is located ∼3 kb upstream of the <em>ARSE</em> or arylsulfatase E (chondrodysplasia punctata 1) gene, though other arylsulfatase genes (<em>ARSH</em>, <em>ARSD</em>, <em>ARSF</em>) are also found in this region. The arylsulfatase family of proteins direct composition of bone and cartilage matrix. Rare mutations in the <em>ARSE</em> gene have been associated with short stature, among other features (<a href="#DDR489C23" rid="DDR489C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544772">23</a>). Our SNP does not appear to be in high LD with any of the known functional variants in <em>ARSE</em>, but association data for the X chromosome are currently limited in the literature and in publically available databases, particularly for AAs. To our knowledge, we are the first to report the association between the Xp22 locus and common variation in body height using population data.</p><p id="__p28">The <em>DYM</em> (18q21) region has previously been associated with height, yet our significant <em>DYM</em> SNPs were in a different part of the gene and were in low LD with the previously identified SNPs in the HapMap CEU, HapMap YRI and the WHI AA populations, which suggests that our SNPs may represent a distinct height locus in <em>DYM</em> in AA or perhaps are more highly correlated with the AA causal variant in this region. In our sensitivity analyses looking up GIANT results, these SNPs reached GWAS significance in European-descent populations also, which implies that the signal in this region may not be population specific.</p><p id="__p29">The two SNPs that we identified at 17q23 (closest genes <em>TMEM100</em> and <em>PCTP</em>) appear to be independent of <em>MAP3K3</em> locus in WHI. Furthermore, they are located ∼1 Mb from the closest locus (<em>NOG</em>) associated with height in the large Lango Allen <em>et al.</em> study (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544788">11</a>). Given their distance from <em>NOG</em> and lack of LD with the <em>NOG</em> SNP in the WHI population or in HapMap CEU (<em>r</em><sup>2</sup> &lt; 0.01 in both populations), we believe that these SNPs are likely independent of the region identified by Lango Allen <em>et al</em>.</p><p id="__p30">Although we identified novel loci associated with height, we also replicated loci previously associated with height in studies of European-descent populations, such as <em>LCORL</em> (<a href="#DDR489C5" rid="DDR489C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544807">5</a>,<a href="#DDR489C7" rid="DDR489C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544790">7</a>,<a href="#DDR489C10" rid="DDR489C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544761">10</a>,<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544764">11</a>). The <em>LCORL</em> locus was also replicated in a height GWAS of Japanese individuals (<a href="#DDR489C14" rid="DDR489C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544821">14</a>). Although the reported SNPs from all studies differ from our top <em>LCORL</em> SNPs, three were genotyped in our study. We replicated these associations and furthermore identified <em>LCORL</em> SNPs more robustly associated with height in AAs. Similarly, our 11q13 SNP located in an intron of the <em>SERPINH1</em> gene and the 12q14 SNP which is downstream of <em>HMGA2</em> gene are each in high LD in CEU (<em>r</em><sup>2</sup> &gt; 0.93) with previously reported SNPs in these loci (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544820">11</a>,<a href="#DDR489C22" rid="DDR489C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544795">22</a>,<a href="#DDR489C24" rid="DDR489C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544799">24</a>). The 17q22–24 region is rich in genes, including the growth hormone gene cluster encompassing a distance of ∼65 kb (<a href="#DDR489C25" rid="DDR489C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544804">25</a>). Variation in the <em>GH1</em> gene has been associated with height in both population (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544767">11</a>,<a href="#DDR489C26" rid="DDR489C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544813">26</a>) and family (<a href="#DDR489C27" rid="DDR489C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544754">27</a>) studies. Our significant finding in the 17q24 locus, rs3889237, is located in an intron of the protein kinase C alpha (<em>PRKCA</em>) gene. <em>PRKCA</em> has been previously associated with BMI in females based on findings from a linkage study in Costa Rican families (<a href="#DDR489C28" rid="DDR489C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544765">28</a>); however, it has not previously been associated with height. Results from conditional analyses in WHI data suggest that the <em>PRKCA</em> SNP is not independent from the <em>MAP3K3</em> locus. Furthermore, this SNP was not significant in the European-descent GIANT consortium. This finding may reflect extended LD in this region as a result of admixture. Finally, our <em>MAP3K3</em> SNP rs11658329 is in high LD in CEU with rs2665838, an SNP in the <em>CSH1</em>/<em>GH1</em> locus previously associated with height in Lango Allen <em>et al</em>. (<a href="#DDR489C11" rid="DDR489C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544817">11</a>) and also with an <em>MAP3K3</em> SNP reported in Lanktree <em>et al.</em> (<a href="#DDR489C22" rid="DDR489C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544789">22</a>).</p><p id="__p31">Comparison of allele frequencies between HapMap CEU and YRI populations (Table&nbsp;<a href="/pmc/articles/PMC3259012/table/DDR489TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR489TB1" rid-ob="ob-DDR489TB1" co-legend-rid=""><span>1</span></a>) provides some clues as to why we were likely able to replicate previous findings and identify novel loci. For many of the previously reported SNPs, allele frequencies for CEU and YRI do not substantially differ from those in AA; we might expect associations to be consistent across races. Indeed, population allele frequencies were similar for the <em>LCORL</em> locus. In contrast, the allele for the Xp22 SNP associated with height in AA is essentially not present (or very rare in HapMap CEU), which demonstrates the advantage of studying a non-European population in which the allele frequency is higher and thus more likely to be identified through the GWAS approach. This X chromosome region did not pass the significance threshold in the admixture analyses, perhaps due to attenuated admixture associations as a result of its location at the P-terminus close to the end of the chromosome where the local ancestry estimates tend to be noisier. Again, focusing on allele frequency differences between the ancestral populations, admixture mapping provided findings at 17q23 consistent with the GWA and identified additional loci on chromosomes 4 (4q21) and 15 (15q26) associated with height. Although the lead SNPs driving these admixture signals at 4q21 and 15q26 were modestly associated with height in the WHI-GWAS, they were not significant in the replication. Although this may cast doubt on whether they are true findings, a full replication of these results would require an admixture replication data set, not just a GWAS replication.</p><p id="__p32">The discovery GWA was conducted in WHI postmenopausal women ranging in age from 50 to 79. Adult height decreases with age due to conditions such as osteoporosis and osteoarthritis (<a href="#DDR489C10" rid="DDR489C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544814">10</a>). Thus, height measurements in this older population may not reflect peak adult height (<a href="#DDR489C29" rid="DDR489C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544781">29</a>). Self-reported osteoarthritis was not associated with height in the WHI AAs, but osteoporosis, though less prevalent, was associated with height. Excluding women with osteoporosis in the WHI analyses did not have appreciable effects on results for the top 12 SNPs from the replication (data not shown). Although osteoarthritis was prevalent in the WHI population, and in spite of reasonable power, we did not replicate an osteoarthritis-associated locus (<em>GDF5-UQC</em>) on chromosome 20 reported to contribute to height variation in largely European-descent populations (<a href="#DDR489C8" rid="DDR489C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544783">8</a>); the <em>P</em>-value for rs6060369 was 0.07 in the WHI AA population. Importantly, we did not test rs143383 because it was not available in the WHI cohort; rs143383 was reported to be more strongly associated with height in AA perhaps due to LD differences at this locus.</p><p id="__p33">Our replication data set included several cohorts, varying in age, sex, geographic location and in some cases, disease status. The use of normalized <em>Z</em>-scores in the meta-analysis allowed us to compare results across the varied populations while accounting for differences in the distributions of height. However, there was some evidence of heterogeneity in the meta-analysis. This heterogeneity is likely a reflection of the relatively large and varying <em>Z</em>-score beta values between WHI and the replication data set which included both AA and African subjects. However, in general, coded allele frequencies were consistent between the WHI and the replication data set, suggesting that major allele frequency differences between the populations are not of concern. An additional limitation pertains to the complexities of studying an admixed population. Although the Affymetrix 6.0 chip provides dense SNP coverage including over 900 000 SNPs, a recent analysis of 76 genes found that only ∼45–55% of SNPs, depending on the method employed, were tagged (<em>r</em><sup>2</sup> &gt; 0.8) in Yoruban samples (<a href="#DDR489C30" rid="DDR489C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544778">30</a>). Coverage for CEU was better, but these results suggest that association analyses using the Affymetrix 6.0 chip in AA populations may miss potentially important variants (<a href="#DDR489C20" rid="DDR489C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544809">20</a>).</p><p id="__p34" class="p p-last">In spite of these limitations, our study was conducted in a large, well-characterized population with high-quality genetic data and measured body height. We are the first to report a discovery GWAS and admixture analysis of height in AA women. We pursued and replicated GWAS findings of interest using a large replication data set consisting of AAs and one small African cohort—identifying novel variants associated with height and generalizing findings from European-descent populations to AAs. Our synthesis of European-descent and AA results for chromosomes 4 and 18, in particular, may help to target future research in the <em>LCORL</em> and <em>DYM</em> genes. In addition, we describe the relationship between local ancestry and height—identifying potential associations between ancestry and height on chromosomes 4, 15 and 17. Overall, our findings provide insight into the genetic architecture of height and support investigation of non-European-descent populations for identifying genetic factors associated with complex traits.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Study population</h3><p id="__p35" class="p p-first-last">The WHI is a national health study focusing on strategies for preventing chronic diseases in postmenopausal women. A total of 161 808 women aged 50–79 years old were recruited from 40 clinical centers in the USA between 1993 and 1998 (<a href="#DDR489C31" rid="DDR489C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544805">31</a>). The WHI cohort consists of an observational study, two clinical trials of postmenopausal hormone therapy (estrogen alone and estrogen plus progestin), a clinical trial of calcium and vitamin D supplements and a dietary modification trial. Study recruitment and exclusion criteria have been described previously (<a href="#DDR489C31" rid="DDR489C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544771">31</a>). Study protocols and consent forms were approved by the institutional review boards at all participating institutions. The WHI SHARe minority cohort includes 8515 self-identified AA women from WHI who provided written informed consent for study participation and DNA analysis. Body height (cm) was measured at baseline using wall-mounted stadiometers. A total of 47 women did not have baseline height. We used height data collected at follow-up visits (all within 3 years) on 41 of these women; 6 women without any height data were excluded. Body height measurements were truncated for the 1st and 99th percentiles (<em>n</em> = 157). This truncation was requested by the Fred Hutchinson Cancer Research Center Institutional Review Board to preserve participant anonymity upon dbGAP submission; we retained these truncated values in the analyses. Research was approved by the Fred Hutchinson Cancer Research Center Institutional Review Board.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping</h3><p id="__p36" class="p p-first-last">Genetic data were obtained from genome-wide scans using the Genome-wide Human SNP Array 6.0 (Affymetrix, Santa Clara, CA, USA, <a href="http://www.affymetrix.com">www.affymetrix.com</a>) containing 909622 SNPs and following the manufacturer's protocol. Genotyping quality control included examination of concordance rates for blinded and un-blinded duplicates. Approximately 1% of SNPs failed genotyping and SNPs with call rates &lt;95% or concordance rates &lt;98%, minor allele frequency (MAF) &lt;1% were excluded. After these exclusions, 855034 SNPs on chromosomes 1–22 and X were available for association analysis. Genotyping failed in 99 samples. Additional participants were excluded based on low call rates &lt;95%, and sex or race/ethnicity discrepancy. First- or second-degree relatedness was assessed as described in Thornton and McPeek (<a href="#DDR489C32" rid="DDR489C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544803">32</a>). For the related individuals, the first- or second-degree relative with the highest call rate was retained and other family members were excluded. After applying all exclusions, a total of 8149 AA women with available height data were included in the analysis.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Population stratification correction</h3><p id="__p37" class="p p-first-last">A principal-component (PC) analysis of all samples was performed using EIGENSTRAT (<a href="#DDR489C33" rid="DDR489C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544800">33</a>). The first 10 PCs were calculated for each individual and evaluated for their contribution to ancestral variation. Because most of the ancestral variation was explained by the first four PCs, only these were included as covariates in the analyses.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Association analysis</h3><p id="__p38" class="p p-first">Linear regression models were used to assess associations between each SNP and baseline body height (continuous). Models were adjusted for age and ancestry PCs 1–4. Using additive genetic models, each SNP was coded as a count of the variant alleles (0, 1 or 2). Results for SNPs with minor allele frequencies ≥0.01 are reported. We inspected quantile–quantile (QQ) plots to determine whether the distribution of the observed SNP association <em>P</em>-values was consistent with the null distribution. We calculated lambda (<em>λ</em>), an indicator of over-dispersion due to potential population stratification by dividing the mean of the test statistics by the mean of the expected values from a <em>χ</em><sup>2</sup> distribution with 1 degree of freedom. Using <em>λ</em>, we investigated correction of <em>P</em>-values using genomic control to account for potential residual confounding by genetic ancestry (<a href="#DDR489C34" rid="DDR489C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544780">34</a>,<a href="#DDR489C35" rid="DDR489C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544812">35</a>). Analyses were conducted using the R software (version 2.11.1).</p><p id="__p39" class="p p-last">Plots were prepared in R using International HapMap3 data for Yoruba in Ibadan, Nigeria (YRI), and Utah residents with ancestry from northern and western Europe (CEU) (<a href="http://hapmap.ncbi.nlm.nih.gov/">http://hapmap.ncbi.nlm.nih.gov/</a>). HapMap population LD estimates were obtained from the SNP Annotation and Proxy Search (SNAP) software (<a href="#DDR489C36" rid="DDR489C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544762">36</a>). Power calculations were conducted assuming additive genetic models using Quanto v1.2 (<a href="#DDR489C37" rid="DDR489C37" class=" bibr popnode">37</a>,<a href="#DDR489C38" rid="DDR489C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544811">38</a>).</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Replication</h3><p id="__p40" class="p p-first-last">SNPs that achieved genome-wide significance of <em>P</em>&lt;5<em>×</em>10<sup>−8</sup> or that had highly suggestive <em>P</em>-values (5<em>×</em>10<sup>−8</sup>&lt;<em>P</em>&lt;5<em>×</em>10<sup>−5</sup>) in the WHI data were tested in an independent AA- and African-descent population of approximately 20000 men and women (<a href="#DDR489C39" rid="DDR489C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544787">39</a>). This replication data set includes data on AAs from the Candidate-Gene Association Resource (CARe) Consortium (<a href="#DDR489C40" rid="DDR489C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544782">40</a>) including Atherosclerosis Risk in Communities (ARIC), Coronary Artery Risk Development in young Adults (CARDIA), Cleveland Family Study (CFS), Jackson Heart Study (JHS) and the Multi-Ethnic Study of Atherosclerosis (MESA); the African Americans Breast Cancer Cohort (AABC) consisting of nine studies (in preparation), the African Americans Prostate Cancer Cohort (AAPC) consisting of 10 studies (<em>Nat. Genet.</em>, in press), the Maywood cohort and one African cohort recruited from Igbo-Ora and Ibadan in southwest Nigeria (<a href="#DDR489C18" rid="DDR489C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544774">18</a>). All participants were genotyped on the Affymetrix 6.0 platform; additional details about the height consortium are described in N'Diaye <em>et al</em>. (<a href="#DDR489C39" rid="DDR489C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544770">39</a>). Stratifying on sex and case–control status (if applicable), height was regressed on age and other appropriate covariates (e.g. recruitment center), after which the height residuals were normalized into height <em>Z</em>-scores. Height <em>Z</em>-scores from the different sub-data sets were combined and associations between genotyped or imputed SNPs and height <em>Z</em>-scores were tested using linear regression models with adjustment for ancestry. SNPs were coded assuming additive genetic models. Association results from all studies were combined in a meta-analysis using the inverse variance meta-analytic method as implemented in the software METAL (<a href="#DDR489C41" rid="DDR489C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544791">41</a>).</p></div><div id="s4f" class="sec"><h3 id="s4ftitle">Meta-analysis</h3><p id="__p41" class="p p-first">To facilitate meta-analyses using WHI SHARe results and results from the height replication data set, we re-analyzed the replication SNPs in WHI SHARe data using height <em>Z</em>-scores as the outcome. Briefly, we regressed age on height and normalized the height residuals. In linear regression models using these normalized residuals or height <em>Z</em>-scores as the outcome, we tested for associations with each SNP and included PCs for ancestry adjustment. We then combined these WHI <em>Z</em>-score results with the height replication data set <em>Z</em>-score results and conducted an inverse variance-weighted meta-analysis using METAL (<a href="#DDR489C41" rid="DDR489C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544757">41</a>).</p><p id="__p42" class="p p-last">Within WHI alone, we estimated our power to detect associations with a small 1.0 cm change in height to be ∼20% for SNPs with MAF = 0.05 and <em>α</em> = 5 × 10<sup>−8</sup>, increasing to &gt;77% for SNPs with MAF ≥ 0.10. Although our initial power to detect small differences in height associated with rare alleles was low, our power to detect small changes in height in the replication meta-analysis for rare alleles (MAF ∼0.02) was ∼50% increasing to &gt;99% for alleles with MAF ≥ 0.05.</p></div><div id="s4g" class="sec sec-last"><h3 id="s4gtitle">Admixture analysis</h3><p id="__p43" class="p p-first-last">For each AA individual in the sample, locus-specific ancestry was estimated using a higher order extension (up to order 15) of the Markov-hidden Markov model (<a href="#DDR489C42" rid="DDR489C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544815">42</a>). Briefly, this method uses a graphical model approach to adaptively capture local haplotype structure within each ancestral population, and thereby more accurately accounts for background LD (Johnson <em>et al.</em>, in preparation). In simulation studies, this method compares favorably with existing methods in terms of the accuracy and computational efficiency; the error rate was &lt;1.5%. In the current analysis, phased haplotype data from the HapMap3 CEU and YRI individuals were augmented as the reference panels. Local ancestry data were coded as 0 = two European alleles, 1 = one European/one African allele, 2 = two African alleles at each locus. Using the local ancestry estimates, we performed an admixture mapping analysis to detect variants that are present at different frequencies among the European and AA ancestral populations. In this analysis, we regressed height on locus-specific ancestry, adjusting for age and genome-wide ancestry proportions using the first four PCs (<a href="#DDR489C43" rid="DDR489C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544793">43</a>). Given the correlation between SNPs along the genome, the number of independent tests is reduced compared with the GWAS and hence significance level was assessed at the more liberal threshold of <em>P</em> &lt; 1 × 10<sup>−5</sup> (<a href="#DDR489C44" rid="DDR489C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247544777">44</a>).</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p44" class="p p-first"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr489/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p><p id="__p45" class="p p-last"><em>Conflict of Interest statement</em>. None declared.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p46" class="p p-first-last">This work was supported by the National Heart, Lung, and Blood Institute, National Institutes of Health, and the US Department of Health and Human Services through contracts <span>N01WH22110</span>;, <span>24152</span>;, <span>32100-2</span>;, <span>32105-6</span>;, <span>32108-9</span>;, <span>32111-13</span>;, <span>32115</span>;, <span>32118-32119</span>;, <span>32122</span>;, <span>42107-26</span>;, <span>42129-32</span>; and <span>44221</span>. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Funding information for the replication data set is provided in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr489/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supplementary Data:</strong>
            
            <div class="sup-box half_rhythm" id="idm140570592027232"><a href="/pmc/articles/PMC3259012/bin/supp_21_3_711__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140570586454928" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140570586454928title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p47">The authors thank the WHI investigators and staff for their dedication, and the study participants for making the program possible. A full listing of WHI investigators is available at <a href="http://www.whiscience.org/publications/WHI_investigators_shortlist.pdf">http://www.whiscience.org/publications/WHI_investigators_shortlist.pdf</a>.</p></div></div><div id="idm140570590723664" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140570590723664title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDR489C1">1. <span class="element-citation">Perola M., Sammalisto S., Hiekkalinna T., Martin N.G., Visscher P.M., Montgomery G.W., Benyamin B., Harris J.R., Boomsma D., Willemsen G., et al.  Combined genome scans for body stature in 6,602 European twins: evidence for common Caucasian loci. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1892350/">PMC free article</a>]</span> [<a href="/pubmed/17559308" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Combined+genome+scans+for+body+stature+in+6,602+European+twins:+evidence+for+common+Caucasian+loci&amp;author=M.+Perola&amp;author=S.+Sammalisto&amp;author=T.+Hiekkalinna&amp;author=N.G.+Martin&amp;author=P.M.+Visscher&amp;volume=3&amp;publication_year=2007&amp;pages=e97&amp;pmid=17559308&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C2">2. <span class="element-citation">Sale M.M., Freedman B.I., Hicks P.J., Williams A.H., Langefeld C.D., Gallagher C.J., Bowden D.W., Rich S.S. Loci contributing to adult height and body mass index in African American families ascertained for type 2 diabetes. <span><span class="ref-journal">Ann. Hum. Genet. </span>2005;<span class="ref-vol">69</span>:517–527.</span> [<a href="/pubmed/16138910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Genet&amp;title=Loci+contributing+to+adult+height+and+body+mass+index+in+African+American+families+ascertained+for+type+2+diabetes&amp;author=M.M.+Sale&amp;author=B.I.+Freedman&amp;author=P.J.+Hicks&amp;author=A.H.+Williams&amp;author=C.D.+Langefeld&amp;volume=69&amp;publication_year=2005&amp;pages=517-527&amp;pmid=16138910&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C3">3. <span class="element-citation">Hirschhorn J.N., Lindgren C.M., Daly M.J., Kirby A., Schaffner S.F., Burtt N.P., Altshuler D., Parker A., Rioux J.D., Platko J., et al.  Genomewide linkage analysis of stature in multiple populations reveals several regions with evidence of linkage to adult height. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2001;<span class="ref-vol">69</span>:106–116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226025/">PMC free article</a>]</span> [<a href="/pubmed/11410839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genomewide+linkage+analysis+of+stature+in+multiple+populations+reveals+several+regions+with+evidence+of+linkage+to+adult+height&amp;author=J.N.+Hirschhorn&amp;author=C.M.+Lindgren&amp;author=M.J.+Daly&amp;author=A.+Kirby&amp;author=S.F.+Schaffner&amp;volume=69&amp;publication_year=2001&amp;pages=106-116&amp;pmid=11410839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C4">4. <span class="element-citation">Weedon M.N., Frayling T.M. Reaching new heights: insights into the genetics of human stature. <span><span class="ref-journal">Trends Genet. </span>2008;<span class="ref-vol">24</span>:595–603.</span> [<a href="/pubmed/18950892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet.&amp;title=Reaching+new+heights:+insights+into+the+genetics+of+human+stature&amp;author=M.N.+Weedon&amp;author=T.M.+Frayling&amp;volume=24&amp;publication_year=2008&amp;pages=595-603&amp;pmid=18950892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C5">5. <span class="element-citation">Weedon M.N., Lango H., Lindgren C.M., Wallace C., Evans D.M., Mangino M., Freathy R.M., Perry J.R., Stevens S., Hall A.S., et al.  Genome-wide association analysis identifies 20 loci that influence adult height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:575–583.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681221/">PMC free article</a>]</span> [<a href="/pubmed/18391952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analysis+identifies+20+loci+that+influence+adult+height&amp;author=M.N.+Weedon&amp;author=H.+Lango&amp;author=C.M.+Lindgren&amp;author=C.+Wallace&amp;author=D.M.+Evans&amp;volume=40&amp;publication_year=2008&amp;pages=575-583&amp;pmid=18391952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C6">6. <span class="element-citation">Lettre G., Jackson A.U., Gieger C., Schumacher F.R., Berndt S.I., Sanna S., Eyheramendy S., Voight B.F., Butler J.L., Guiducci C., et al.  Identification of ten loci associated with height highlights new biological pathways in human growth. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:584–591.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687076/">PMC free article</a>]</span> [<a href="/pubmed/18391950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+ten+loci+associated+with+height+highlights+new+biological+pathways+in+human+growth&amp;author=G.+Lettre&amp;author=A.U.+Jackson&amp;author=C.+Gieger&amp;author=F.R.+Schumacher&amp;author=S.I.+Berndt&amp;volume=40&amp;publication_year=2008&amp;pages=584-591&amp;pmid=18391950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C7">7. <span class="element-citation">Gudbjartsson D.F., Walters G.B., Thorleifsson G., Stefansson H., Halldorsson B.V., Zusmanovich P., Sulem P., Thorlacius S., Gylfason A., Steinberg S., et al.  Many sequence variants affecting diversity of adult human height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:609–615.</span> [<a href="/pubmed/18391951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Many+sequence+variants+affecting+diversity+of+adult+human+height&amp;author=D.F.+Gudbjartsson&amp;author=G.B.+Walters&amp;author=G.+Thorleifsson&amp;author=H.+Stefansson&amp;author=B.V.+Halldorsson&amp;volume=40&amp;publication_year=2008&amp;pages=609-615&amp;pmid=18391951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C8">8. <span class="element-citation">Sanna S., Jackson A.U., Nagaraja R., Willer C.J., Chen W.M., Bonnycastle L.L., Shen H., Timpson N., Lettre G., Usala G., et al.  Common variants in the GDF5-UQCC region are associated with variation in human height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:198–203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914680/">PMC free article</a>]</span> [<a href="/pubmed/18193045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+in+the+GDF5-UQCC+region+are+associated+with+variation+in+human+height&amp;author=S.+Sanna&amp;author=A.U.+Jackson&amp;author=R.+Nagaraja&amp;author=C.J.+Willer&amp;author=W.M.+Chen&amp;volume=40&amp;publication_year=2008&amp;pages=198-203&amp;pmid=18193045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C9">9. <span class="element-citation">Estrada K., Krawczak M., Schreiber S., van Duijn K., Stolk L., van Meurs J.B., Liu F., Penninx B.W., Smit J.H., Vogelzangs N., et al.  A genome-wide association study of northwestern Europeans involves the C-type natriuretic peptide signaling pathway in the etiology of human height variation. <span><span class="ref-journal">Hum. Mol. Genet. </span>2009;<span class="ref-vol">18</span>:3516–3524.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2729669/">PMC free article</a>]</span> [<a href="/pubmed/19570815" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+genome-wide+association+study+of+northwestern+Europeans+involves+the+C-type+natriuretic+peptide+signaling+pathway+in+the+etiology+of+human+height+variation&amp;author=K.+Estrada&amp;author=M.+Krawczak&amp;author=S.+Schreiber&amp;author=K.+van+Duijn&amp;author=L.+Stolk&amp;volume=18&amp;publication_year=2009&amp;pages=3516-3524&amp;pmid=19570815&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C10">10. <span class="element-citation">Soranzo N., Rivadeneira F., Chinappen-Horsley U., Malkina I., Richards J.B., Hammond N., Stolk L., Nica A., Inouye M., Hofman A., et al.  Meta-analysis of genome-wide scans for human adult stature identifies novel loci and associations with measures of skeletal frame size. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2661236/">PMC free article</a>]</span> [<a href="/pubmed/19343178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Meta-analysis+of+genome-wide+scans+for+human+adult+stature+identifies+novel+loci+and+associations+with+measures+of+skeletal+frame+size&amp;author=N.+Soranzo&amp;author=F.+Rivadeneira&amp;author=U.+Chinappen-Horsley&amp;author=I.+Malkina&amp;author=J.B.+Richards&amp;volume=5&amp;publication_year=2009&amp;pages=e1000445&amp;pmid=19343178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C11">11. <span class="element-citation">Lango Allen H., Estrada K., Lettre G., Berndt S.I., Weedon M.N., Rivadeneira F., Willer C.J., Jackson A.U., Vedantam S., Raychaudhuri S., et al.  Hundreds of variants clustered in genomic loci and biological pathways affect human height. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:832–838.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955183/">PMC free article</a>]</span> [<a href="/pubmed/20881960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hundreds+of+variants+clustered+in+genomic+loci+and+biological+pathways+affect+human+height&amp;author=H.+Lango+Allen&amp;author=K.+Estrada&amp;author=G.+Lettre&amp;author=S.I.+Berndt&amp;author=M.N.+Weedon&amp;volume=467&amp;publication_year=2010&amp;pages=832-838&amp;pmid=20881960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C12">12. <span class="element-citation">Yang J., Benyamin B., McEvoy B.P., Gordon S., Henders A.K., Nyholt D.R., Madden P.A., Heath A.C., Martin N.G., Montgomery G.W., et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J.+Yang&amp;author=B.+Benyamin&amp;author=B.P.+McEvoy&amp;author=S.+Gordon&amp;author=A.K.+Henders&amp;volume=42&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C13">13. <span class="element-citation">Lei S.F., Yang T.L., Tan L.J., Chen X.D., Guo Y., Guo Y.F., Zhang L., Liu X.G., Yan H., Pan F., et al.  Genome-wide association scan for stature in Chinese: evidence for ethnic specific loci. <span><span class="ref-journal">Hum. Genet. </span>2009;<span class="ref-vol">125</span>:1–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2730511/">PMC free article</a>]</span> [<a href="/pubmed/19030899" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Genome-wide+association+scan+for+stature+in+Chinese:+evidence+for+ethnic+specific+loci&amp;author=S.F.+Lei&amp;author=T.L.+Yang&amp;author=L.J.+Tan&amp;author=X.D.+Chen&amp;author=Y.+Guo&amp;volume=125&amp;publication_year=2009&amp;pages=1-9&amp;pmid=19030899&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C14">14. <span class="element-citation">Okada Y., Kamatani Y., Takahashi A., Matsuda K., Hosono N., Ohmiya H., Daigo Y., Yamamoto K., Kubo M., Nakamura Y., et al.  A genome-wide association study in 19 633 Japanese subjects identified LHX3-QSOX2 and IGF1 as adult height loci. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:2303–2312.</span> [<a href="/pubmed/20189936" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+genome-wide+association+study+in+19+633+Japanese+subjects+identified+LHX3-QSOX2+and+IGF1+as+adult+height+loci&amp;author=Y.+Okada&amp;author=Y.+Kamatani&amp;author=A.+Takahashi&amp;author=K.+Matsuda&amp;author=N.+Hosono&amp;volume=19&amp;publication_year=2010&amp;pages=2303-2312&amp;pmid=20189936&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C15">15. <span class="element-citation">Cho Y.S., Go M.J., Kim Y.J., Heo J.Y., Oh J.H., Ban H.J., Yoon D., Lee M.H., Kim D.J., Park M., et al.  A large-scale genome-wide association study of Asian populations uncovers genetic factors influencing eight quantitative traits. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:527–534.</span> [<a href="/pubmed/19396169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+large-scale+genome-wide+association+study+of+Asian+populations+uncovers+genetic+factors+influencing+eight+quantitative+traits&amp;author=Y.S.+Cho&amp;author=M.J.+Go&amp;author=Y.J.+Kim&amp;author=J.Y.+Heo&amp;author=J.H.+Oh&amp;volume=41&amp;publication_year=2009&amp;pages=527-534&amp;pmid=19396169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C16">16. <span class="element-citation">Kim J.J., Lee H.I., Park T., Kim K., Lee J.E., Cho N.H., Shin C., Cho Y.S., Lee J.Y., Han B.G., et al.  Identification of 15 loci influencing height in a Korean population. <span><span class="ref-journal">J. Hum. Genet. </span>2010;<span class="ref-vol">55</span>:27–31.</span> [<a href="/pubmed/19893584" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet.&amp;title=Identification+of+15+loci+influencing+height+in+a+Korean+population&amp;author=J.J.+Kim&amp;author=H.I.+Lee&amp;author=T.+Park&amp;author=K.+Kim&amp;author=J.E.+Lee&amp;volume=55&amp;publication_year=2010&amp;pages=27-31&amp;pmid=19893584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C17">17. <span class="element-citation">Croteau-Chonka D.C., Marvelle A.F., Lange E.M., Lee N.R., Adair L.S., Lange L.A., Mohlke K.L. Genome-wide association study of anthropometric traits and evidence of interactions with age and study year in Filipino women. <span><span class="ref-journal">Obesity. </span>2011;<span class="ref-vol">19</span>:1019–1027.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3046220/">PMC free article</a>]</span> [<a href="/pubmed/20966902" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obesity&amp;title=Genome-wide+association+study+of+anthropometric+traits+and+evidence+of+interactions+with+age+and+study+year+in+Filipino+women&amp;author=D.C.+Croteau-Chonka&amp;author=A.F.+Marvelle&amp;author=E.M.+Lange&amp;author=N.R.+Lee&amp;author=L.S.+Adair&amp;volume=19&amp;publication_year=2011&amp;pages=1019-1027&amp;pmid=20966902&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C18">18. <span class="element-citation">Kang S.J., Chiang C.W., Palmer C.D., Tayo B.O., Lettre G., Butler J.L., Hackett R., Adeyemo A.A., Guiducci C., Berzins I., et al.  Genome-wide association of anthropometric traits in African- and African-derived populations. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:2725–2738.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2883343/">PMC free article</a>]</span> [<a href="/pubmed/20400458" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+of+anthropometric+traits+in+African-+and+African-derived+populations&amp;author=S.J.+Kang&amp;author=C.W.+Chiang&amp;author=C.D.+Palmer&amp;author=B.O.+Tayo&amp;author=G.+Lettre&amp;volume=19&amp;publication_year=2010&amp;pages=2725-2738&amp;pmid=20400458&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C19">19. <span class="element-citation">Patterson N., Petersen D.C., van der Ross R.E., Sudoyo H., Glashoff R.H., Marzuki S., Reich D., Hayes V.M. Genetic structure of a unique admixed population: implications for medical research. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:411–419.</span> [<a href="/pubmed/19892779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genetic+structure+of+a+unique+admixed+population:+implications+for+medical+research&amp;author=N.+Patterson&amp;author=D.C.+Petersen&amp;author=R.E.+van+der+Ross&amp;author=H.+Sudoyo&amp;author=R.H.+Glashoff&amp;volume=19&amp;publication_year=2010&amp;pages=411-419&amp;pmid=19892779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C20">20. <span class="element-citation">Cooper R.S., Tayo B., Zhu X. Genome-wide association studies: implications for multiethnic samples. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:R151–R155.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782359/">PMC free article</a>]</span> [<a href="/pubmed/18852204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+studies:+implications+for+multiethnic+samples&amp;author=R.S.+Cooper&amp;author=B.+Tayo&amp;author=X.+Zhu&amp;volume=17&amp;publication_year=2008&amp;pages=R151-R155&amp;pmid=18852204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C21">21. <span class="element-citation">Yang J., Weedon M.N., Purcell S., Lettre G., Estrada K., Willer C.J., Smith A.V., Ingelsson E., O'Connell J.R., Mangino M., et al.  Genomic inflation factors under polygenic inheritance. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2011;<span class="ref-vol">19</span>:807–812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3137506/">PMC free article</a>]</span> [<a href="/pubmed/21407268" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Genomic+inflation+factors+under+polygenic+inheritance&amp;author=J.+Yang&amp;author=M.N.+Weedon&amp;author=S.+Purcell&amp;author=G.+Lettre&amp;author=K.+Estrada&amp;volume=19&amp;publication_year=2011&amp;pages=807-812&amp;pmid=21407268&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C22">22. <span class="element-citation">Lanktree M.B., Guo Y., Murtaza M., Glessner J.T., Bailey S.D., Onland-Moret N.C., Lettre G., Ongen H., Rajagopalan R., Johnson T., et al.  Meta-analysis of dense genecentric association studies reveals common and uncommon variants associated with height. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:6–18.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014369/">PMC free article</a>]</span> [<a href="/pubmed/21194676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Meta-analysis+of+dense+genecentric+association+studies+reveals+common+and+uncommon+variants+associated+with+height&amp;author=M.B.+Lanktree&amp;author=Y.+Guo&amp;author=M.+Murtaza&amp;author=J.T.+Glessner&amp;author=S.D.+Bailey&amp;volume=88&amp;publication_year=2011&amp;pages=6-18&amp;pmid=21194676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C23">23. <span class="element-citation">Casarin A., Rusalen F., Doimo M., Trevisson E., Carraro S., Clementi M., Tenconi R., Baraldi E., Salviati L. X-linked brachytelephalangic chondrodysplasia punctata: a simple trait that is not so simple. <span><span class="ref-journal">Am. J. Med. Genet. A. </span>2009;<span class="ref-vol">149A</span>:2464–2468.</span> [<a href="/pubmed/19839041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+A&amp;title=X-linked+brachytelephalangic+chondrodysplasia+punctata:+a+simple+trait+that+is+not+so+simple&amp;author=A.+Casarin&amp;author=F.+Rusalen&amp;author=M.+Doimo&amp;author=E.+Trevisson&amp;author=S.+Carraro&amp;volume=149A&amp;publication_year=2009&amp;pages=2464-2468&amp;pmid=19839041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C24">24. <span class="element-citation">Weedon M.N., Lettre G., Freathy R.M., Lindgren C.M., Voight B.F., Perry J.R., Elliott K.S., Hackett R., Guiducci C., Shields B., et al.  A common variant of HMGA2 is associated with adult and childhood height in the general population. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1245–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3086278/">PMC free article</a>]</span> [<a href="/pubmed/17767157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+common+variant+of+HMGA2+is+associated+with+adult+and+childhood+height+in+the+general+population&amp;author=M.N.+Weedon&amp;author=G.+Lettre&amp;author=R.M.+Freathy&amp;author=C.M.+Lindgren&amp;author=B.F.+Voight&amp;volume=39&amp;publication_year=2007&amp;pages=1245-1250&amp;pmid=17767157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C25">25. <span class="element-citation">George D.L., Phillips J.A., III, Francke U., Seeburg P.H. The genes for growth hormone and chorionic somatomammotropin are on the long arm of human chromosome 17 in region q21 to qter. <span><span class="ref-journal">Hum. Genet. </span>1981;<span class="ref-vol">57</span>:138–141.</span> [<a href="/pubmed/6262212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=The+genes+for+growth+hormone+and+chorionic+somatomammotropin+are+on+the+long+arm+of+human+chromosome+17+in+region+q21+to+qter&amp;author=D.L.+George&amp;author=J.A.+Phillips&amp;author=U.+Francke&amp;author=P.H.+Seeburg&amp;volume=57&amp;publication_year=1981&amp;pages=138-141&amp;pmid=6262212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C26">26. <span class="element-citation">Millar D.S., Lewis M.D., Horan M., Newsway V., Easter T.E., Gregory J.W., Fryklund L., Norin M., Crowne E.C., Davies S.J., et al.  Novel mutations of the growth hormone 1 (GH1) gene disclosed by modulation of the clinical selection criteria for individuals with short stature. <span><span class="ref-journal">Hum. Mutat. </span>2003;<span class="ref-vol">21</span>:424–440.</span> [<a href="/pubmed/12655557" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=Novel+mutations+of+the+growth+hormone+1+(GH1)+gene+disclosed+by+modulation+of+the+clinical+selection+criteria+for+individuals+with+short+stature&amp;author=D.S.+Millar&amp;author=M.D.+Lewis&amp;author=M.+Horan&amp;author=V.+Newsway&amp;author=T.E.+Easter&amp;volume=21&amp;publication_year=2003&amp;pages=424-440&amp;pmid=12655557&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C27">27. <span class="element-citation">Wagner J.K., Eble A., Hindmarsh P.C., Mullis P.E. Prevalence of human GH-1 gene alterations in patients with isolated growth hormone deficiency. <span><span class="ref-journal">Pediatr. Res. </span>1998;<span class="ref-vol">43</span>:105–110.</span> [<a href="/pubmed/9432120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Res.&amp;title=Prevalence+of+human+GH-1+gene+alterations+in+patients+with+isolated+growth+hormone+deficiency&amp;author=J.K.+Wagner&amp;author=A.+Eble&amp;author=P.C.+Hindmarsh&amp;author=P.E.+Mullis&amp;volume=43&amp;publication_year=1998&amp;pages=105-110&amp;pmid=9432120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C28">28. <span class="element-citation">Murphy A., Tantisira K.G., Soto-Quiros M.E., Avila L., Klanderman B.J., Lake S., Weiss S.T., Celedon J.C. PRKCA: a positional candidate gene for body mass index and asthma. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">85</span>:87–96.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2706964/">PMC free article</a>]</span> [<a href="/pubmed/19576566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PRKCA:+a+positional+candidate+gene+for+body+mass+index+and+asthma&amp;author=A.+Murphy&amp;author=K.G.+Tantisira&amp;author=M.E.+Soto-Quiros&amp;author=L.+Avila&amp;author=B.J.+Klanderman&amp;volume=85&amp;publication_year=2009&amp;pages=87-96&amp;pmid=19576566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C29">29. <span class="element-citation">Sorkin J.D., Muller D.C., Andres R. Longitudinal change in height of men and women: implications for interpretation of the body mass index: the Baltimore Longitudinal Study of Aging. <span><span class="ref-journal">Am. J. Epidemiol. </span>1999;<span class="ref-vol">150</span>:969–977.</span> [<a href="/pubmed/10547143" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Longitudinal+change+in+height+of+men+and+women:+implications+for+interpretation+of+the+body+mass+index:+the+Baltimore+Longitudinal+Study+of+Aging&amp;author=J.D.+Sorkin&amp;author=D.C.+Muller&amp;author=R.+Andres&amp;volume=150&amp;publication_year=1999&amp;pages=969-977&amp;pmid=10547143&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C30">30. <span class="element-citation">Bhangale T.R., Rieder M.J., Nickerson D.A. Estimating coverage and power for genetic association studies using near-complete variation data. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:841–843.</span> [<a href="/pubmed/18568023" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Estimating+coverage+and+power+for+genetic+association+studies+using+near-complete+variation+data&amp;author=T.R.+Bhangale&amp;author=M.J.+Rieder&amp;author=D.A.+Nickerson&amp;volume=40&amp;publication_year=2008&amp;pages=841-843&amp;pmid=18568023&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C31">31. <span class="element-citation">The Women's Health Initiative Study Group. Design of the Women's Health Initiative clinical trial and observational study. <span><span class="ref-journal">Control Clin. Trials. </span>1998;<span class="ref-vol">19</span>:61–109.</span> [<a href="/pubmed/9492970" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Control+Clin.+Trials&amp;title=Design+of+the+Women's+Health+Initiative+clinical+trial+and+observational+study&amp;volume=19&amp;publication_year=1998&amp;pages=61-109&amp;pmid=9492970&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C32">32. <span class="element-citation">Thornton T., McPeek M.S. ROADTRIPS: case-control association testing with partially or completely unknown population and pedigree structure. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2010;<span class="ref-vol">86</span>:172–184.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820184/">PMC free article</a>]</span> [<a href="/pubmed/20137780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=ROADTRIPS:+case-control+association+testing+with+partially+or+completely+unknown+population+and+pedigree+structure&amp;author=T.+Thornton&amp;author=M.S.+McPeek&amp;volume=86&amp;publication_year=2010&amp;pages=172-184&amp;pmid=20137780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C33">33. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C34">34. <span class="element-citation">Devlin B., Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C35">35. <span class="element-citation">Bacanu S.A., Devlin B., Roeder K. Association studies for quantitative traits in structured populations. <span><span class="ref-journal">Genet. Epidemiol. </span>2002;<span class="ref-vol">22</span>:78–93.</span> [<a href="/pubmed/11754475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Association+studies+for+quantitative+traits+in+structured+populations&amp;author=S.A.+Bacanu&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=22&amp;publication_year=2002&amp;pages=78-93&amp;pmid=11754475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C36">36. <span class="element-citation">Johnson A.D., Handsaker R.E., Pulit S.L., Nizzari M.M., O'Donnell C.J., de Bakker P.I. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. <span><span class="ref-journal">Bioinformatics. </span>2008;<span class="ref-vol">24</span>:2938–2939.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNAP:+a+web-based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;author=A.D.+Johnson&amp;author=R.E.+Handsaker&amp;author=S.L.+Pulit&amp;author=M.M.+Nizzari&amp;author=C.J.+O'Donnell&amp;volume=24&amp;publication_year=2008&amp;pages=2938-2939&amp;pmid=18974171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C37">37. <span class="element-citation">Gauderman W.J., Morrison J.M. QUANTO 1.1: a computer program for power and sample size calculations for genetic-epidemiology studies. <span>2006</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Gauderman+W.J.+Morrison+J.M.+QUANTO+1.1:+a+computer+program+for+power+and+sample+size+calculations+for+genetic-epidemiology+studies+2006+" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C38">38. <span class="element-citation">Gauderman W.J. Sample size requirements for association studies of gene-gene interaction. <span><span class="ref-journal">Am. J. Epidemiol. </span>2002;<span class="ref-vol">155</span>:478–484.</span> [<a href="/pubmed/11867360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Sample+size+requirements+for+association+studies+of+gene-gene+interaction&amp;author=W.J.+Gauderman&amp;volume=155&amp;publication_year=2002&amp;pages=478-484&amp;pmid=11867360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C39">39. <span class="element-citation">N'Diaye A., Chen G.K., Palmer C.D., Ge B., Tayo B., Mathias R.A., Ding J., Nalls M.A., Adeyemo A., et al.  Identification, replication, and fine-mapping of loci associated with adult height in individuals of African ancestry. <span><span class="ref-journal">PLoS. Genet. </span>2011;<span class="ref-vol">7</span>:e1002298.</span>  <a href="http://dx.doi.org/10.1371/journal.pgen.1002298" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1371/journal.pgen.1002298</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3188544/">PMC free article</a>]</span> [<a href="/pubmed/21998595" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS.+Genet.&amp;title=Identification,+replication,+and+fine-mapping+of+loci+associated+with+adult+height+in+individuals+of+African+ancestry&amp;author=A.+N'Diaye&amp;author=G.K.+Chen&amp;author=C.D.+Palmer&amp;author=B.+Ge&amp;author=B.+Tayo&amp;volume=7&amp;publication_year=2011&amp;pages=e1002298&amp;pmid=21998595&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C40">40. <span class="element-citation">Lettre G., Palmer C.D., Young T., Ejebe K.G., Allayee H., Benjamin E.J., Bennett F., Bowden D.W., Chakravarti A., Dreisbach A., et al.  Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: The NHLBI CARe Project. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1001300.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037413/">PMC free article</a>]</span> [<a href="/pubmed/21347282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+study+of+coronary+heart+disease+and+its+risk+factors+in+8,090+African+Americans:+The+NHLBI+CARe+Project&amp;author=G.+Lettre&amp;author=C.D.+Palmer&amp;author=T.+Young&amp;author=K.G.+Ejebe&amp;author=H.+Allayee&amp;volume=7&amp;publication_year=2011&amp;pages=e1001300&amp;pmid=21347282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C41">41. <span class="element-citation">Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C42">42. <span class="element-citation">Tang H., Coram M., Wang P., Zhu X., Risch N. Reconstructing genetic ancestry blocks in admixed individuals. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">79</span>:1–12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1474129/">PMC free article</a>]</span> [<a href="/pubmed/16773560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Reconstructing+genetic+ancestry+blocks+in+admixed+individuals&amp;author=H.+Tang&amp;author=M.+Coram&amp;author=P.+Wang&amp;author=X.+Zhu&amp;author=N.+Risch&amp;volume=79&amp;publication_year=2006&amp;pages=1-12&amp;pmid=16773560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C43">43. <span class="element-citation">Basu A., Tang H., Arnett D., Gu C.C., Mosley T., Kardia S., Luke A., Tayo B., Cooper R., Zhu X., et al.  Admixture mapping of quantitative trait loci for BMI in African Americans: evidence for loci on chromosomes 3q, 5q, and 15q. <span><span class="ref-journal">Obesity. </span>2009;<span class="ref-vol">17</span>:1226–1231.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2929755/">PMC free article</a>]</span> [<a href="/pubmed/19584881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obesity&amp;title=Admixture+mapping+of+quantitative+trait+loci+for+BMI+in+African+Americans:+evidence+for+loci+on+chromosomes+3q,+5q,+and+15q&amp;author=A.+Basu&amp;author=H.+Tang&amp;author=D.+Arnett&amp;author=C.C.+Gu&amp;author=T.+Mosley&amp;volume=17&amp;publication_year=2009&amp;pages=1226-1231&amp;pmid=19584881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR489C44">44. <span class="element-citation">Tang H., Siegmund D.O., Johnson N.A., Romieu I., London S.J. Joint testing of genotype and ancestry association in admixed families. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:783–791.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3103820/">PMC free article</a>]</span> [<a href="/pubmed/21031451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Joint+testing+of+genotype+and+ancestry+association+in+admixed+families&amp;author=H.+Tang&amp;author=D.O.+Siegmund&amp;author=N.A.+Johnson&amp;author=I.+Romieu&amp;author=S.J.+London&amp;volume=34&amp;publication_year=2010&amp;pages=783-791&amp;pmid=21031451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3259012&amp;issue-id=204500&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140570629090112title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140570586454928title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140570590723664title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3259012/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3259012/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3259012/pdf/ddr489.pdf">PDF (318K)</a></li> | <li><a href="#" data-citationid="PMC3259012" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3259012%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3259012%2F&amp;text=Genome-wide%20association%20study%20of%20body%20height%20in%20African%20Americans%3A%20the%20Women's%20Health%20Initiative%20SNP%20Health%20Association%20Resource%20(SHARe)" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3259012%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22021425" data-aiid="3259012" data-aid="3259012" data-iid="204500" data-domainid="740" data-domain="hmg" data-accid="PMC3259012" data-md5="c5a88a48f53e761df9d4766e59bd2186"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:59:13-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE897849DD52ADE1000000000916063D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3259012%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3259012/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>