<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3959809/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association meta-analysis identifies a novel locus at 17q11.2 associated with sporadic amyotrophic lateral sclerosis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3959809/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="A genome-wide association meta-analysis identifies a novel locus at 17q11.2 associated with sporadic amyotrophic lateral sclerosis"><meta name="citation_authors" content="Isabella Fogh, Antonia Ratti, Cinzia Gellera, Kuang Lin, Cinzia Tiloca, Valentina Moskvina, Lucia Corrado, Gianni Sorarù, Cristina Cereda, Stefania Corti, Davide Gentilini, Daniela Calini, Barbara Castellotti, Letizia Mazzini, Giorgia Querin, Stella Gagliardi, Roberto Del Bo, Francesca L. Conforti, Gabriele Siciliano, Maurizio Inghilleri, Francesco Saccà, Paolo Bongioanni, Silvana Penco, Massimo Corbo, Sandro Sorbi, Massimiliano Filosto, Alessandra Ferlini, Anna M. Di Blasio, Stefano Signorini, Aleksey Shatunov, Ashley Jones, Pamela J. Shaw, Karen E. Morrison, Anne E. Farmer, Philip Van Damme, Wim Robberecht, Adriano Chiò, Bryan J. Traynor, Michael Sendtner, Judith Melki, Vincent Meininger, Orla Hardiman, Peter M. Andersen, Nigel P. Leigh, Jonathan D. Glass, Daniel Overste, Frank P. Diekstra, Jan H. Veldink, Michael A. van Es, Christopher E. Shaw, Michael E. Weale, Cathryn M. Lewis, Julie Williams, Robert H. Brown, John E. Landers, Nicola Ticozzi, Mauro Ceroni, Elena Pegoraro, Giacomo P. Comi, Sandra D'Alfonso, Leonard H. van den Berg, Franco Taroni, Ammar Al-Chalabi, John Powell, Vincenzo Silani, the SLAGEN Consortium and Collaborators, Vincenzo Brescia Morra, Alessandro Filla, Filosto Massimo, Angela Marsili, Pensato Viviana, Giorgia Puorro, Vincenzo La Bella, Giancarlo Logroscino, Maria Rosaria Monsurrò, Aldo Quattrone, Isabella Laura Simone, Kreshnik B. Ahmeti, Senda Ajroud-Driss, Jennifer Armstrong, Anne Birve, Hylke M. Blauw, Lucie Bruijn, Wenjie Chen, Mary C. Comeau, Simon Cronin, Gkazi Athina Soraya, Josh D. Grab, Ewout J. Groen, Jonathan L. Haines, Scott Heller, Jie Huang, Wu-Yen Hung, ITALSGEN Consortium, James M. Jaworski, Humaira Khan, Carl D. Langefeld, Miranda C. Marion, Russell L. McLaughlin, Jack W. Miller, Gabriele Mora, Margaret A. Pericak-Vance, Evadnie Rampersaud, Nailah Siddique, Teepu Siddique, Bradley N. Smith, Robert Sufit, Simon Topp, Caroline Vance, Paul van Vught, Yi Yang, J.G. Zheng"><meta name="citation_date" content="15 April 2014"><meta name="citation_issue" content="8"><meta name="citation_volume" content="23"><meta name="citation_firstpage" content="2220"><meta name="citation_doi" content="10.1093/hmg/ddt587"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3959809/?report=abstract"><meta name="citation_pmid" content="24256812"><meta name="DC.Title" content="A genome-wide association meta-analysis identifies a novel locus at 17q11.2 associated with sporadic amyotrophic lateral sclerosis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Isabella Fogh"><meta name="DC.Contributor" content="Antonia Ratti"><meta name="DC.Contributor" content="Cinzia Gellera"><meta name="DC.Contributor" content="Kuang Lin"><meta name="DC.Contributor" content="Cinzia Tiloca"><meta name="DC.Contributor" content="Valentina Moskvina"><meta name="DC.Contributor" content="Lucia Corrado"><meta name="DC.Contributor" content="Gianni Sorarù"><meta name="DC.Contributor" content="Cristina Cereda"><meta name="DC.Contributor" content="Stefania Corti"><meta name="DC.Contributor" content="Davide Gentilini"><meta name="DC.Contributor" content="Daniela Calini"><meta name="DC.Contributor" content="Barbara Castellotti"><meta name="DC.Contributor" content="Letizia Mazzini"><meta name="DC.Contributor" content="Giorgia Querin"><meta name="DC.Contributor" content="Stella Gagliardi"><meta name="DC.Contributor" content="Roberto Del Bo"><meta name="DC.Contributor" content="Francesca L. Conforti"><meta name="DC.Contributor" content="Gabriele Siciliano"><meta name="DC.Contributor" content="Maurizio Inghilleri"><meta name="DC.Contributor" content="Francesco Saccà"><meta name="DC.Contributor" content="Paolo Bongioanni"><meta name="DC.Contributor" content="Silvana Penco"><meta name="DC.Contributor" content="Massimo Corbo"><meta name="DC.Contributor" content="Sandro Sorbi"><meta name="DC.Contributor" content="Massimiliano Filosto"><meta name="DC.Contributor" content="Alessandra Ferlini"><meta name="DC.Contributor" content="Anna M. Di Blasio"><meta name="DC.Contributor" content="Stefano Signorini"><meta name="DC.Contributor" content="Aleksey Shatunov"><meta name="DC.Contributor" content="Ashley Jones"><meta name="DC.Contributor" content="Pamela J. Shaw"><meta name="DC.Contributor" content="Karen E. Morrison"><meta name="DC.Contributor" content="Anne E. Farmer"><meta name="DC.Contributor" content="Philip Van Damme"><meta name="DC.Contributor" content="Wim Robberecht"><meta name="DC.Contributor" content="Adriano Chiò"><meta name="DC.Contributor" content="Bryan J. Traynor"><meta name="DC.Contributor" content="Michael Sendtner"><meta name="DC.Contributor" content="Judith Melki"><meta name="DC.Contributor" content="Vincent Meininger"><meta name="DC.Contributor" content="Orla Hardiman"><meta name="DC.Contributor" content="Peter M. Andersen"><meta name="DC.Contributor" content="Nigel P. Leigh"><meta name="DC.Contributor" content="Jonathan D. Glass"><meta name="DC.Contributor" content="Daniel Overste"><meta name="DC.Contributor" content="Frank P. Diekstra"><meta name="DC.Contributor" content="Jan H. Veldink"><meta name="DC.Contributor" content="Michael A. van Es"><meta name="DC.Contributor" content="Christopher E. Shaw"><meta name="DC.Contributor" content="Michael E. Weale"><meta name="DC.Contributor" content="Cathryn M. Lewis"><meta name="DC.Contributor" content="Julie Williams"><meta name="DC.Contributor" content="Robert H. Brown"><meta name="DC.Contributor" content="John E. Landers"><meta name="DC.Contributor" content="Nicola Ticozzi"><meta name="DC.Contributor" content="Mauro Ceroni"><meta name="DC.Contributor" content="Elena Pegoraro"><meta name="DC.Contributor" content="Giacomo P. Comi"><meta name="DC.Contributor" content="Sandra D'Alfonso"><meta name="DC.Contributor" content="Leonard H. van den Berg"><meta name="DC.Contributor" content="Franco Taroni"><meta name="DC.Contributor" content="Ammar Al-Chalabi"><meta name="DC.Contributor" content="John Powell"><meta name="DC.Contributor" content="Vincenzo Silani"><meta name="DC.Contributor" content="the SLAGEN Consortium and Collaborators"><meta name="DC.Contributor" content="Vincenzo Brescia Morra"><meta name="DC.Contributor" content="Alessandro Filla"><meta name="DC.Contributor" content="Filosto Massimo"><meta name="DC.Contributor" content="Angela Marsili"><meta name="DC.Contributor" content="Pensato Viviana"><meta name="DC.Contributor" content="Giorgia Puorro"><meta name="DC.Contributor" content="Vincenzo La Bella"><meta name="DC.Contributor" content="Giancarlo Logroscino"><meta name="DC.Contributor" content="Maria Rosaria Monsurrò"><meta name="DC.Contributor" content="Aldo Quattrone"><meta name="DC.Contributor" content="Isabella Laura Simone"><meta name="DC.Contributor" content="Kreshnik B. Ahmeti"><meta name="DC.Contributor" content="Senda Ajroud-Driss"><meta name="DC.Contributor" content="Jennifer Armstrong"><meta name="DC.Contributor" content="Anne Birve"><meta name="DC.Contributor" content="Hylke M. Blauw"><meta name="DC.Contributor" content="Lucie Bruijn"><meta name="DC.Contributor" content="Wenjie Chen"><meta name="DC.Contributor" content="Mary C. Comeau"><meta name="DC.Contributor" content="Simon Cronin"><meta name="DC.Contributor" content="Gkazi Athina Soraya"><meta name="DC.Contributor" content="Josh D. Grab"><meta name="DC.Contributor" content="Ewout J. Groen"><meta name="DC.Contributor" content="Jonathan L. Haines"><meta name="DC.Contributor" content="Scott Heller"><meta name="DC.Contributor" content="Jie Huang"><meta name="DC.Contributor" content="Wu-Yen Hung"><meta name="DC.Contributor" content="ITALSGEN Consortium"><meta name="DC.Contributor" content="James M. Jaworski"><meta name="DC.Contributor" content="Humaira Khan"><meta name="DC.Contributor" content="Carl D. Langefeld"><meta name="DC.Contributor" content="Miranda C. Marion"><meta name="DC.Contributor" content="Russell L. McLaughlin"><meta name="DC.Contributor" content="Jack W. Miller"><meta name="DC.Contributor" content="Gabriele Mora"><meta name="DC.Contributor" content="Margaret A. Pericak-Vance"><meta name="DC.Contributor" content="Evadnie Rampersaud"><meta name="DC.Contributor" content="Nailah Siddique"><meta name="DC.Contributor" content="Teepu Siddique"><meta name="DC.Contributor" content="Bradley N. Smith"><meta name="DC.Contributor" content="Robert Sufit"><meta name="DC.Contributor" content="Simon Topp"><meta name="DC.Contributor" content="Caroline Vance"><meta name="DC.Contributor" content="Paul van Vught"><meta name="DC.Contributor" content="Yi Yang"><meta name="DC.Contributor" content="J.G. Zheng"><meta name="DC.Date" content="2014 Apr 15"><meta name="DC.Identifier" content="10.1093/hmg/ddt587"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association meta-analysis identifies a novel locus at 17q11.2 associated with sporadic amyotrophic lateral sclerosis"><meta property="og:type" content="article"><meta property="og:description" content="Identification of mutations at familial loci for amyotrophic lateral sclerosis (ALS) has provided novel insights into the aetiology of this rapidly progressing fatal neurodegenerative disease. However, genome-wide association studies (GWAS) of the more ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3959809/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3959809/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3959809/pdf/ddt587.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BB229DD561D71000000000723043E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3959809%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3959809</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2014 Apr 15; 23(8): 2220–2231. </span></div><div><span class="fm-vol-iss-date">Published online 2013 Nov 20. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddt587" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddt587</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3959809</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24256812">24256812</a></div></div></div></div><h1 class="content-title">A genome-wide association meta-analysis identifies a novel locus at 17q11.2 associated with sporadic amyotrophic lateral sclerosis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Fogh%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748205680" co-class="co-affbox">Isabella Fogh</a>,<sup>1,</sup><sup>4,</sup><sup>5,</sup><sup>*</sup> <a href="/pubmed/?term=Ratti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748202160" co-class="co-affbox">Antonia Ratti</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Gellera%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459750665456" co-class="co-affbox">Cinzia Gellera</a>,<sup>7</sup> <a href="/pubmed/?term=Lin%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459750663472" co-class="co-affbox">Kuang Lin</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Tiloca%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459750660752" co-class="co-affbox">Cinzia Tiloca</a>,<sup>4,</sup><sup>6,</sup><sup>†</sup> <a href="/pubmed/?term=Moskvina%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747450752" co-class="co-affbox">Valentina Moskvina</a>,<sup>8,</sup><sup>†</sup> <a href="/pubmed/?term=Corrado%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747448032" co-class="co-affbox">Lucia Corrado</a>,<sup>9</sup> <a href="/pubmed/?term=Sorar%26%23x000f9%3B%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747446048" co-class="co-affbox">Gianni Sorarù</a>,<sup>10</sup> <a href="/pubmed/?term=Cereda%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747444064" co-class="co-affbox">Cristina Cereda</a>,<sup>11</sup> <a href="/pubmed/?term=Corti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747442080" co-class="co-affbox">Stefania Corti</a>,<sup>5,</sup><sup>13</sup> <a href="/pubmed/?term=Gentilini%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747439360" co-class="co-affbox">Davide Gentilini</a>,<sup>14</sup> <a href="/pubmed/?term=Calini%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747437376" co-class="co-affbox">Daniela Calini</a>,<sup>4</sup> <a href="/pubmed/?term=Castellotti%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747435392" co-class="co-affbox">Barbara Castellotti</a>,<sup>7</sup> <a href="/pubmed/?term=Mazzini%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747433408" co-class="co-affbox">Letizia Mazzini</a>,<sup>9,</sup><sup>15</sup> <a href="/pubmed/?term=Querin%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459747430688" co-class="co-affbox">Giorgia Querin</a>,<sup>10</sup> <a href="/pubmed/?term=Gagliardi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749804928" co-class="co-affbox">Stella Gagliardi</a>,<sup>11</sup> <a href="/pubmed/?term=Del%20Bo%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749802944" co-class="co-affbox">Roberto Del Bo</a>,<sup>5,</sup><sup>13</sup> <a href="/pubmed/?term=Conforti%20FL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749800224" co-class="co-affbox">Francesca L. Conforti</a>,<sup>16</sup> <a href="/pubmed/?term=Siciliano%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749798240" co-class="co-affbox">Gabriele Siciliano</a>,<sup>17</sup> <a href="/pubmed/?term=Inghilleri%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749796256" co-class="co-affbox">Maurizio Inghilleri</a>,<sup>18</sup> <a href="/pubmed/?term=Sacc%26%23x000e0%3B%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749794272" co-class="co-affbox">Francesco Saccà</a>,<sup>19</sup> <a href="/pubmed/?term=Bongioanni%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749792288" co-class="co-affbox">Paolo Bongioanni</a>,<sup>20</sup> <a href="/pubmed/?term=Penco%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749790304" co-class="co-affbox">Silvana Penco</a>,<sup>21</sup> <a href="/pubmed/?term=Corbo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749788320" co-class="co-affbox">Massimo Corbo</a>,<sup>22</sup> <a href="/pubmed/?term=Sorbi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749786336" co-class="co-affbox">Sandro Sorbi</a>,<sup>23</sup> <a href="/pubmed/?term=Filosto%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749784352" co-class="co-affbox">Massimiliano Filosto</a>,<sup>24</sup> <a href="/pubmed/?term=Ferlini%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459749782368" co-class="co-affbox">Alessandra Ferlini</a>,<sup>25</sup> <a href="/pubmed/?term=Di%20Blasio%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746014944" co-class="co-affbox">Anna M. Di Blasio</a>,<sup>14</sup> <a href="/pubmed/?term=Signorini%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746012960" co-class="co-affbox">Stefano Signorini</a>,<sup>26</sup> <a href="/pubmed/?term=Shatunov%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746010976" co-class="co-affbox">Aleksey Shatunov</a>,<sup>2</sup> <a href="/pubmed/?term=Jones%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746008992" co-class="co-affbox">Ashley Jones</a>,<sup>2</sup> <a href="/pubmed/?term=Shaw%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746007008" co-class="co-affbox">Pamela J. Shaw</a>,<sup>27</sup> <a href="/pubmed/?term=Morrison%20KE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746005024" co-class="co-affbox">Karen E. Morrison</a>,<sup>28,</sup><sup>29</sup> <a href="/pubmed/?term=Farmer%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746002304" co-class="co-affbox">Anne E. Farmer</a>,<sup>3</sup> <a href="/pubmed/?term=Van%20Damme%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746000320" co-class="co-affbox">Philip Van Damme</a>,<sup>30,</sup><sup>31</sup> <a href="/pubmed/?term=Robberecht%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459745997600" co-class="co-affbox">Wim Robberecht</a>,<sup>30,</sup><sup>31</sup> <a href="/pubmed/?term=Chi%26%23x000f2%3B%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459745994880" co-class="co-affbox">Adriano Chiò</a>,<sup>32,</sup><sup>33</sup> <a href="/pubmed/?term=Traynor%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459745992160" co-class="co-affbox">Bryan J. Traynor</a>,<sup>34</sup> <a href="/pubmed/?term=Sendtner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748659776" co-class="co-affbox">Michael Sendtner</a>,<sup>35</sup> <a href="/pubmed/?term=Melki%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748657792" co-class="co-affbox">Judith Melki</a>,<sup>36</sup> <a href="/pubmed/?term=Meininger%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748655808" co-class="co-affbox">Vincent Meininger</a>,<sup>37</sup> <a href="/pubmed/?term=Hardiman%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748653824" co-class="co-affbox">Orla Hardiman</a>,<sup>38</sup> <a href="/pubmed/?term=Andersen%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748651840" co-class="co-affbox">Peter M. Andersen</a>,<sup>39</sup> <a href="/pubmed/?term=Leigh%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748649856" co-class="co-affbox">Nigel P. Leigh</a>,<sup>40</sup> <a href="/pubmed/?term=Glass%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748647872" co-class="co-affbox">Jonathan D. Glass</a>,<sup>41</sup> <a href="/pubmed/?term=Overste%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748645888" co-class="co-affbox">Daniel Overste</a>,<sup>42</sup> <a href="/pubmed/?term=Diekstra%20FP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748643904" co-class="co-affbox">Frank P. Diekstra</a>,<sup>42</sup> <a href="/pubmed/?term=Veldink%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748641920" co-class="co-affbox">Jan H. Veldink</a>,<sup>42</sup> <a href="/pubmed/?term=van%20Es%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748639936" co-class="co-affbox">Michael A. van Es</a>,<sup>42</sup> <a href="/pubmed/?term=Shaw%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459748637952" co-class="co-affbox">Christopher E. Shaw</a>,<sup>2</sup> <a href="/pubmed/?term=Weale%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746979488" co-class="co-affbox">Michael E. Weale</a>,<sup>3,</sup><sup>43</sup> <a href="/pubmed/?term=Lewis%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746976768" co-class="co-affbox">Cathryn M. Lewis</a>,<sup>3,</sup><sup>43</sup> <a href="/pubmed/?term=Williams%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746974048" co-class="co-affbox">Julie Williams</a>,<sup>8</sup> <a href="/pubmed/?term=Brown%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746972064" co-class="co-affbox">Robert H. Brown</a>,<sup>44</sup> <a href="/pubmed/?term=Landers%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746970080" co-class="co-affbox">John E. Landers</a>,<sup>44</sup> <a href="/pubmed/?term=Ticozzi%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746968096" co-class="co-affbox">Nicola Ticozzi</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Ceroni%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746965376" co-class="co-affbox">Mauro Ceroni</a>,<sup>12</sup> <a href="/pubmed/?term=Pegoraro%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746963392" co-class="co-affbox">Elena Pegoraro</a>,<sup>10</sup> <a href="/pubmed/?term=Comi%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746961408" co-class="co-affbox">Giacomo P. Comi</a>,<sup>5,</sup><sup>13</sup> <a href="/pubmed/?term=D'Alfonso%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746958688" co-class="co-affbox">Sandra D'Alfonso</a>,<sup>9</sup> <a href="/pubmed/?term=van%20den%20Berg%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746956704" co-class="co-affbox">Leonard H. van den Berg</a>,<sup>42</sup> <a href="/pubmed/?term=Taroni%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459746954720" co-class="co-affbox">Franco Taroni</a>,<sup>7</sup> <a href="/pubmed/?term=Al-Chalabi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459783363248" co-class="co-affbox">Ammar Al-Chalabi</a>,<sup>2</sup> <a href="/pubmed/?term=Powell%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459783361264" co-class="co-affbox">John Powell</a>,<sup>1,</sup><sup>‡</sup> <a href="/pubmed/?term=Silani%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812" class="affpopup" co-rid="_co_idm140459783358544" co-class="co-affbox">Vincenzo Silani</a>,<sup>4,</sup><sup>5,</sup><sup>‡</sup> and  the SLAGEN Consortium and Collaborators<sup>§</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140459748205680"><h3 class="no_margin">Isabella Fogh</h3><p><sup>1</sup>Department of Neuroscience,</p><p><sup>4</sup>Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, 20149 Milano, Italy</p><p><sup>5</sup>Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center,  and</p><div>Find articles by <a href="/pubmed/?term=Fogh%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Isabella Fogh</a></div></div><div id="_co_idm140459748202160"><h3 class="no_margin">Antonia Ratti</h3><p><sup>4</sup>Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, 20149 Milano, Italy</p><p><sup>5</sup>Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center,  and</p><div>Find articles by <a href="/pubmed/?term=Ratti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Antonia Ratti</a></div></div><div id="_co_idm140459750665456"><h3 class="no_margin">Cinzia Gellera</h3><p><sup>7</sup>Unit of Genetics of Neurodegenerative and Metabolic Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, 20133 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Gellera%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Cinzia Gellera</a></div></div><div id="_co_idm140459750663472"><h3 class="no_margin">Kuang Lin</h3><p><sup>1</sup>Department of Neuroscience,</p><div>Find articles by <a href="/pubmed/?term=Lin%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Kuang Lin</a></div></div><div id="_co_idm140459750660752"><h3 class="no_margin">Cinzia Tiloca</h3><p><sup>4</sup>Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, 20149 Milano, Italy</p><p><sup>6</sup>Doctoral School in Molecular Medicine, Università degli Studi di Milano, 20122 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Tiloca%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Cinzia Tiloca</a></div></div><div id="_co_idm140459747450752"><h3 class="no_margin">Valentina Moskvina</h3><p><sup>8</sup>Department of Psychological Medicine and Neurology, School of Medicine, MRC Centre for Neuropsychiatric Genetics and Genomics, Cardiff University, Cardiff CF14 4XN, UK</p><div>Find articles by <a href="/pubmed/?term=Moskvina%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Valentina Moskvina</a></div></div><div id="_co_idm140459747448032"><h3 class="no_margin">Lucia Corrado</h3><p><sup>9</sup>Department of Health Sciences, Interdisciplinary Research Center of Autoimmune Diseases, ‘A. Avogadro’ University, 28100 Novara, Italy</p><div>Find articles by <a href="/pubmed/?term=Corrado%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Lucia Corrado</a></div></div><div id="_co_idm140459747446048"><h3 class="no_margin">Gianni Sorarù</h3><p><sup>10</sup>Department of Neurosciences, University of Padova, 35128 Padova, Italy</p><div>Find articles by <a href="/pubmed/?term=Sorar%26%23x000f9%3B%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Gianni Sorarù</a></div></div><div id="_co_idm140459747444064"><h3 class="no_margin">Cristina Cereda</h3><p><sup>11</sup>Laboratory of Experimental Neurobiology,  and</p><div>Find articles by <a href="/pubmed/?term=Cereda%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Cristina Cereda</a></div></div><div id="_co_idm140459747442080"><h3 class="no_margin">Stefania Corti</h3><p><sup>5</sup>Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center,  and</p><p><sup>13</sup>Neurologic Unit, IRCCS Foundation Ca’ Granda Ospedale Maggiore Policlinico, 20122 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Corti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Stefania Corti</a></div></div><div id="_co_idm140459747439360"><h3 class="no_margin">Davide Gentilini</h3><p><sup>14</sup>Laboratory of Molecular Biology, IRCCS Istituto Auxologico Italiano, 20095 Cusano Milanino, Italy</p><div>Find articles by <a href="/pubmed/?term=Gentilini%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Davide Gentilini</a></div></div><div id="_co_idm140459747437376"><h3 class="no_margin">Daniela Calini</h3><p><sup>4</sup>Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, 20149 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Calini%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Daniela Calini</a></div></div><div id="_co_idm140459747435392"><h3 class="no_margin">Barbara Castellotti</h3><p><sup>7</sup>Unit of Genetics of Neurodegenerative and Metabolic Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, 20133 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Castellotti%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Barbara Castellotti</a></div></div><div id="_co_idm140459747433408"><h3 class="no_margin">Letizia Mazzini</h3><p><sup>9</sup>Department of Health Sciences, Interdisciplinary Research Center of Autoimmune Diseases, ‘A. Avogadro’ University, 28100 Novara, Italy</p><p><sup>15</sup>Department of Neurology, ALS Center, Maggiore della Carità University Hospital, 28100 Novara, Italy</p><div>Find articles by <a href="/pubmed/?term=Mazzini%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Letizia Mazzini</a></div></div><div id="_co_idm140459747430688"><h3 class="no_margin">Giorgia Querin</h3><p><sup>10</sup>Department of Neurosciences, University of Padova, 35128 Padova, Italy</p><div>Find articles by <a href="/pubmed/?term=Querin%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Giorgia Querin</a></div></div><div id="_co_idm140459749804928"><h3 class="no_margin">Stella Gagliardi</h3><p><sup>11</sup>Laboratory of Experimental Neurobiology,  and</p><div>Find articles by <a href="/pubmed/?term=Gagliardi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Stella Gagliardi</a></div></div><div id="_co_idm140459749802944"><h3 class="no_margin">Roberto Del Bo</h3><p><sup>5</sup>Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center,  and</p><p><sup>13</sup>Neurologic Unit, IRCCS Foundation Ca’ Granda Ospedale Maggiore Policlinico, 20122 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Del%20Bo%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Roberto Del Bo</a></div></div><div id="_co_idm140459749800224"><h3 class="no_margin">Francesca L. Conforti</h3><p><sup>16</sup>Institute of Neurological Sciences, National Research Council, 87050 Mangone, Italy</p><div>Find articles by <a href="/pubmed/?term=Conforti%20FL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Francesca L. Conforti</a></div></div><div id="_co_idm140459749798240"><h3 class="no_margin">Gabriele Siciliano</h3><p><sup>17</sup>Division of Neurology, Department of Clinical and Experimental Medicine, University of Pisa, Nuovo Ospedale S.Chiara-Cisanello, 56126 Pisa, Italy</p><div>Find articles by <a href="/pubmed/?term=Siciliano%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Gabriele Siciliano</a></div></div><div id="_co_idm140459749796256"><h3 class="no_margin">Maurizio Inghilleri</h3><p><sup>18</sup>Department of Neurology and Psychiatry, University of Rome ‘Sapienza’, 00185 Roma, Italy</p><div>Find articles by <a href="/pubmed/?term=Inghilleri%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Maurizio Inghilleri</a></div></div><div id="_co_idm140459749794272"><h3 class="no_margin">Francesco Saccà</h3><p><sup>19</sup>Department of Neurological Sciences, University Federico II, 80131 Napoli, Italy</p><div>Find articles by <a href="/pubmed/?term=Sacc%26%23x000e0%3B%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Francesco Saccà</a></div></div><div id="_co_idm140459749792288"><h3 class="no_margin">Paolo Bongioanni</h3><p><sup>20</sup>Neurorehabilitation Unit, Azienda Ospedaliero-Universitaria Pisana, 56126 Pisa, Italy</p><div>Find articles by <a href="/pubmed/?term=Bongioanni%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Paolo Bongioanni</a></div></div><div id="_co_idm140459749790304"><h3 class="no_margin">Silvana Penco</h3><p><sup>21</sup>Department of Laboratory Medicine, Medical Genetics Unit, Niguarda Ca’ Granda Hospital, 20162 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Penco%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Silvana Penco</a></div></div><div id="_co_idm140459749788320"><h3 class="no_margin">Massimo Corbo</h3><p><sup>22</sup>NeuroMuscular Omnicentre (NEMO), Fondazione Serena Onlus, Niguarda Ca’ Granda Hospital, 20162 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Corbo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Massimo Corbo</a></div></div><div id="_co_idm140459749786336"><h3 class="no_margin">Sandro Sorbi</h3><p><sup>23</sup>Department of Neurological and Psychiatric Sciences, University of Firenze, 50139 Firenze, Italy</p><div>Find articles by <a href="/pubmed/?term=Sorbi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Sandro Sorbi</a></div></div><div id="_co_idm140459749784352"><h3 class="no_margin">Massimiliano Filosto</h3><p><sup>24</sup>Clinical Neurology, Section for Neuromuscular Diseases and Neuropathies, University Hospital ‘Spedali Civili’, 25123 Brescia, Italy</p><div>Find articles by <a href="/pubmed/?term=Filosto%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Massimiliano Filosto</a></div></div><div id="_co_idm140459749782368"><h3 class="no_margin">Alessandra Ferlini</h3><p><sup>25</sup>Department of Medical Sciences, University of Ferrara, 44100 Ferrara, Italy</p><div>Find articles by <a href="/pubmed/?term=Ferlini%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Alessandra Ferlini</a></div></div><div id="_co_idm140459746014944"><h3 class="no_margin">Anna M. Di Blasio</h3><p><sup>14</sup>Laboratory of Molecular Biology, IRCCS Istituto Auxologico Italiano, 20095 Cusano Milanino, Italy</p><div>Find articles by <a href="/pubmed/?term=Di%20Blasio%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Anna M. Di Blasio</a></div></div><div id="_co_idm140459746012960"><h3 class="no_margin">Stefano Signorini</h3><p><sup>26</sup>Azienda Ospedaliera di Desio e Vimercate, 20871 Vimercate, Italy</p><div>Find articles by <a href="/pubmed/?term=Signorini%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Stefano Signorini</a></div></div><div id="_co_idm140459746010976"><h3 class="no_margin">Aleksey Shatunov</h3><p><sup>2</sup>Department of Clinical Neuroscience,  and</p><div>Find articles by <a href="/pubmed/?term=Shatunov%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Aleksey Shatunov</a></div></div><div id="_co_idm140459746008992"><h3 class="no_margin">Ashley Jones</h3><p><sup>2</sup>Department of Clinical Neuroscience,  and</p><div>Find articles by <a href="/pubmed/?term=Jones%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Ashley Jones</a></div></div><div id="_co_idm140459746007008"><h3 class="no_margin">Pamela J. Shaw</h3><p><sup>27</sup>Academic Neurology Unit, Department of Neuroscience, Faculty of Medicine, Dentistry and Health, University of Sheffield, Sheffield S10 2HQ, UK</p><div>Find articles by <a href="/pubmed/?term=Shaw%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Pamela J. Shaw</a></div></div><div id="_co_idm140459746005024"><h3 class="no_margin">Karen E. Morrison</h3><p><sup>28</sup>School of Clinical and Experimental Medicine, College of Medicine and Dentistry, University of Birmingham, and</p><p><sup>29</sup>Neurosciences Division, University Hospitals Birmingham NHS Foundation Trust, Birmingham B15 2TT, UK</p><div>Find articles by <a href="/pubmed/?term=Morrison%20KE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Karen E. Morrison</a></div></div><div id="_co_idm140459746002304"><h3 class="no_margin">Anne E. Farmer</h3><p><sup>3</sup>MRC Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King's College London, London SE5 8AF, UK</p><div>Find articles by <a href="/pubmed/?term=Farmer%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Anne E. Farmer</a></div></div><div id="_co_idm140459746000320"><h3 class="no_margin">Philip Van Damme</h3><p><sup>30</sup>Department of Neurology, University Hospital Leuven, Leuven, Belgium</p><p><sup>31</sup>Laboratory for Neurobiology, Vesalius Research Center, VIB, University of Leuven, 3000 Leuven, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20Damme%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Philip Van Damme</a></div></div><div id="_co_idm140459745997600"><h3 class="no_margin">Wim Robberecht</h3><p><sup>30</sup>Department of Neurology, University Hospital Leuven, Leuven, Belgium</p><p><sup>31</sup>Laboratory for Neurobiology, Vesalius Research Center, VIB, University of Leuven, 3000 Leuven, Belgium</p><div>Find articles by <a href="/pubmed/?term=Robberecht%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Wim Robberecht</a></div></div><div id="_co_idm140459745994880"><h3 class="no_margin">Adriano Chiò</h3><p><sup>32</sup>Department of Neuroscience, University of Torino, Torino, Italy</p><p><sup>33</sup>Azienda Ospedaliera Città della Salute e della Scienza, 10126 Torino, Italy</p><div>Find articles by <a href="/pubmed/?term=Chi%26%23x000f2%3B%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Adriano Chiò</a></div></div><div id="_co_idm140459745992160"><h3 class="no_margin">Bryan J. Traynor</h3><p><sup>34</sup>Neuromuscular Diseases Research Group, Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Traynor%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Bryan J. Traynor</a></div></div><div id="_co_idm140459748659776"><h3 class="no_margin">Michael Sendtner</h3><p><sup>35</sup>Institute for Clinical Neurobiology, University Hospital Wurzburg, 97078 Wurzburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Sendtner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Michael Sendtner</a></div></div><div id="_co_idm140459748657792"><h3 class="no_margin">Judith Melki</h3><p><sup>36</sup>INSERM UMR-788 and University of Paris 11, Bicetre Hospital, Paris, Cedex 94275, France</p><div>Find articles by <a href="/pubmed/?term=Melki%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Judith Melki</a></div></div><div id="_co_idm140459748655808"><h3 class="no_margin">Vincent Meininger</h3><p><sup>37</sup>Assistance Publique - Hôpitaux de Paris, Université Pierre et Marie Curie, Hôpital de la Salpêtrière, 75013 Paris, France</p><div>Find articles by <a href="/pubmed/?term=Meininger%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Vincent Meininger</a></div></div><div id="_co_idm140459748653824"><h3 class="no_margin">Orla Hardiman</h3><p><sup>38</sup>Academic Neurology Unit, Trinity Biomedical Sciences Institute, Trinity College Dublin, Dublin2, Ireland</p><div>Find articles by <a href="/pubmed/?term=Hardiman%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Orla Hardiman</a></div></div><div id="_co_idm140459748651840"><h3 class="no_margin">Peter M. Andersen</h3><p><sup>39</sup>Department of Clinical Neuroscience, Umeå University, SE-901 85 Umeå, Sweden</p><div>Find articles by <a href="/pubmed/?term=Andersen%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Peter M. Andersen</a></div></div><div id="_co_idm140459748649856"><h3 class="no_margin">Nigel P. Leigh</h3><p><sup>40</sup>Brighton and Sussex Medical School, Trafford Centre for Biomedical Research, University of Sussex, Falmer East Sussex BN1 9RY, UK</p><div>Find articles by <a href="/pubmed/?term=Leigh%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Nigel P. Leigh</a></div></div><div id="_co_idm140459748647872"><h3 class="no_margin">Jonathan D. Glass</h3><p><sup>41</sup>Department of Neurology, Emory University, Atlanta, GA 30322, USA</p><div>Find articles by <a href="/pubmed/?term=Glass%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Jonathan D. Glass</a></div></div><div id="_co_idm140459748645888"><h3 class="no_margin">Daniel Overste</h3><p><sup>42</sup>Department of Neurology, Rudolf Magnus Institute of Neuroscience, University Medical Centre Utrecht, 3508 GA Utrecht, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Overste%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Daniel Overste</a></div></div><div id="_co_idm140459748643904"><h3 class="no_margin">Frank P. Diekstra</h3><p><sup>42</sup>Department of Neurology, Rudolf Magnus Institute of Neuroscience, University Medical Centre Utrecht, 3508 GA Utrecht, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Diekstra%20FP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Frank P. Diekstra</a></div></div><div id="_co_idm140459748641920"><h3 class="no_margin">Jan H. Veldink</h3><p><sup>42</sup>Department of Neurology, Rudolf Magnus Institute of Neuroscience, University Medical Centre Utrecht, 3508 GA Utrecht, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Veldink%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Jan H. Veldink</a></div></div><div id="_co_idm140459748639936"><h3 class="no_margin">Michael A. van Es</h3><p><sup>42</sup>Department of Neurology, Rudolf Magnus Institute of Neuroscience, University Medical Centre Utrecht, 3508 GA Utrecht, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Es%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Michael A. van Es</a></div></div><div id="_co_idm140459748637952"><h3 class="no_margin">Christopher E. Shaw</h3><p><sup>2</sup>Department of Clinical Neuroscience,  and</p><div>Find articles by <a href="/pubmed/?term=Shaw%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Christopher E. Shaw</a></div></div><div id="_co_idm140459746979488"><h3 class="no_margin">Michael E. Weale</h3><p><sup>3</sup>MRC Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King's College London, London SE5 8AF, UK</p><p><sup>43</sup>Department of Medical and Molecular Genetics, King's College London, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Weale%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Michael E. Weale</a></div></div><div id="_co_idm140459746976768"><h3 class="no_margin">Cathryn M. Lewis</h3><p><sup>3</sup>MRC Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King's College London, London SE5 8AF, UK</p><p><sup>43</sup>Department of Medical and Molecular Genetics, King's College London, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Lewis%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Cathryn M. Lewis</a></div></div><div id="_co_idm140459746974048"><h3 class="no_margin">Julie Williams</h3><p><sup>8</sup>Department of Psychological Medicine and Neurology, School of Medicine, MRC Centre for Neuropsychiatric Genetics and Genomics, Cardiff University, Cardiff CF14 4XN, UK</p><div>Find articles by <a href="/pubmed/?term=Williams%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Julie Williams</a></div></div><div id="_co_idm140459746972064"><h3 class="no_margin">Robert H. Brown</h3><p><sup>44</sup>Department of Neurology, University of Massachusetts Medical School, Worcester, MA 01605, USA</p><div>Find articles by <a href="/pubmed/?term=Brown%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Robert H. Brown</a></div></div><div id="_co_idm140459746970080"><h3 class="no_margin">John E. Landers</h3><p><sup>44</sup>Department of Neurology, University of Massachusetts Medical School, Worcester, MA 01605, USA</p><div>Find articles by <a href="/pubmed/?term=Landers%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">John E. Landers</a></div></div><div id="_co_idm140459746968096"><h3 class="no_margin">Nicola Ticozzi</h3><p><sup>4</sup>Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, 20149 Milano, Italy</p><p><sup>5</sup>Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center,  and</p><div>Find articles by <a href="/pubmed/?term=Ticozzi%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Nicola Ticozzi</a></div></div><div id="_co_idm140459746965376"><h3 class="no_margin">Mauro Ceroni</h3><p><sup>12</sup>Division of General Neurology, IRCCS ‘C. Mondino’ National Institute of Neurology Foundation, 27100 Pavia, Italy</p><div>Find articles by <a href="/pubmed/?term=Ceroni%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Mauro Ceroni</a></div></div><div id="_co_idm140459746963392"><h3 class="no_margin">Elena Pegoraro</h3><p><sup>10</sup>Department of Neurosciences, University of Padova, 35128 Padova, Italy</p><div>Find articles by <a href="/pubmed/?term=Pegoraro%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Elena Pegoraro</a></div></div><div id="_co_idm140459746961408"><h3 class="no_margin">Giacomo P. Comi</h3><p><sup>5</sup>Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center,  and</p><p><sup>13</sup>Neurologic Unit, IRCCS Foundation Ca’ Granda Ospedale Maggiore Policlinico, 20122 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Comi%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Giacomo P. Comi</a></div></div><div id="_co_idm140459746958688"><h3 class="no_margin">Sandra D'Alfonso</h3><p><sup>9</sup>Department of Health Sciences, Interdisciplinary Research Center of Autoimmune Diseases, ‘A. Avogadro’ University, 28100 Novara, Italy</p><div>Find articles by <a href="/pubmed/?term=D'Alfonso%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Sandra D'Alfonso</a></div></div><div id="_co_idm140459746956704"><h3 class="no_margin">Leonard H. van den Berg</h3><p><sup>42</sup>Department of Neurology, Rudolf Magnus Institute of Neuroscience, University Medical Centre Utrecht, 3508 GA Utrecht, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20den%20Berg%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Leonard H. van den Berg</a></div></div><div id="_co_idm140459746954720"><h3 class="no_margin">Franco Taroni</h3><p><sup>7</sup>Unit of Genetics of Neurodegenerative and Metabolic Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, 20133 Milano, Italy</p><div>Find articles by <a href="/pubmed/?term=Taroni%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Franco Taroni</a></div></div><div id="_co_idm140459783363248"><h3 class="no_margin">Ammar Al-Chalabi</h3><p><sup>2</sup>Department of Clinical Neuroscience,  and</p><div>Find articles by <a href="/pubmed/?term=Al-Chalabi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Ammar Al-Chalabi</a></div></div><div id="_co_idm140459783361264"><h3 class="no_margin">John Powell</h3><p><sup>1</sup>Department of Neuroscience,</p><div>Find articles by <a href="/pubmed/?term=Powell%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">John Powell</a></div></div><div id="_co_idm140459783358544"><h3 class="no_margin">Vincenzo Silani</h3><p><sup>4</sup>Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, 20149 Milano, Italy</p><p><sup>5</sup>Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center,  and</p><div>Find articles by <a href="/pubmed/?term=Silani%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Vincenzo Silani</a></div></div></div></div><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Brescia%20Morra%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Vincenzo Brescia Morra</a>, <a href="/pubmed/?term=Filla%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Alessandro Filla</a>, <a href="/pubmed/?term=Massimo%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Filosto Massimo</a>, <a href="/pubmed/?term=Marsili%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Angela Marsili</a>, <a href="/pubmed/?term=Viviana%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Pensato Viviana</a>, <a href="/pubmed/?term=Puorro%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Giorgia Puorro</a>, <a href="/pubmed/?term=La%20Bella%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Vincenzo La Bella</a>, <a href="/pubmed/?term=Logroscino%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Giancarlo Logroscino</a>, <a href="/pubmed/?term=Monsurr%26%23x000f2%3B%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Maria Rosaria Monsurrò</a>, <a href="/pubmed/?term=Quattrone%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Aldo Quattrone</a>, <a href="/pubmed/?term=Simone%20IL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Isabella Laura Simone</a>, <a href="/pubmed/?term=Ahmeti%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Kreshnik B. Ahmeti</a>, <a href="/pubmed/?term=Ajroud-Driss%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Senda Ajroud-Driss</a>, <a href="/pubmed/?term=Armstrong%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Jennifer Armstrong</a>, <a href="/pubmed/?term=Birve%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Anne Birve</a>, <a href="/pubmed/?term=Blauw%20HM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Hylke M. Blauw</a>, <a href="/pubmed/?term=Bruijn%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Lucie Bruijn</a>, <a href="/pubmed/?term=Chen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Wenjie Chen</a>, <a href="/pubmed/?term=Comeau%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Mary C. Comeau</a>, <a href="/pubmed/?term=Cronin%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Simon Cronin</a>, <a href="/pubmed/?term=Soraya%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Gkazi Athina Soraya</a>, <a href="/pubmed/?term=Grab%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Josh D. Grab</a>, <a href="/pubmed/?term=Groen%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Ewout J. Groen</a>, <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Jonathan L. Haines</a>, <a href="/pubmed/?term=Heller%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Scott Heller</a>, <a href="/pubmed/?term=Huang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Jie Huang</a>, <a href="/pubmed/?term=Hung%20WY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Wu-Yen Hung</a>, ITALSGEN Consortium, <a href="/pubmed/?term=Jaworski%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">James M. Jaworski</a>, <a href="/pubmed/?term=Khan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Humaira Khan</a>, <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Carl D. Langefeld</a>, <a href="/pubmed/?term=Marion%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Miranda C. Marion</a>, <a href="/pubmed/?term=McLaughlin%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Russell L. McLaughlin</a>, <a href="/pubmed/?term=Miller%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Jack W. Miller</a>, <a href="/pubmed/?term=Mora%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Gabriele Mora</a>, <a href="/pubmed/?term=Pericak-Vance%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Margaret A. Pericak-Vance</a>, <a href="/pubmed/?term=Rampersaud%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Evadnie Rampersaud</a>, <a href="/pubmed/?term=Siddique%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Nailah Siddique</a>, <a href="/pubmed/?term=Siddique%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Teepu Siddique</a>, <a href="/pubmed/?term=Smith%20BN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Bradley N. Smith</a>, <a href="/pubmed/?term=Sufit%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Robert Sufit</a>, <a href="/pubmed/?term=Topp%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Simon Topp</a>, <a href="/pubmed/?term=Vance%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Caroline Vance</a>, <a href="/pubmed/?term=van%20Vught%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Paul van Vught</a>, <a href="/pubmed/?term=Yang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">Yi Yang</a>, and  <a href="/pubmed/?term=Zheng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24256812">J.G. Zheng</a><div class="fm-panel half_rhythm"><div class="fm-authors-info fm-panel half_rhythm"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Neuroscience,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Clinical Neuroscience,  and</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>MRC Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, King's College London, London SE5 8AF, UK</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, 20149 Milano, Italy</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center,  and</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Doctoral School in Molecular Medicine, Università degli Studi di Milano, 20122 Milano, Italy</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Unit of Genetics of Neurodegenerative and Metabolic Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, 20133 Milano, Italy</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Psychological Medicine and Neurology, School of Medicine, MRC Centre for Neuropsychiatric Genetics and Genomics, Cardiff University, Cardiff CF14 4XN, UK</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Department of Health Sciences, Interdisciplinary Research Center of Autoimmune Diseases, ‘A. Avogadro’ University, 28100 Novara, Italy</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Department of Neurosciences, University of Padova, 35128 Padova, Italy</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Laboratory of Experimental Neurobiology,  and</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Division of General Neurology, IRCCS ‘C. Mondino’ National Institute of Neurology Foundation, 27100 Pavia, Italy</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Neurologic Unit, IRCCS Foundation Ca’ Granda Ospedale Maggiore Policlinico, 20122 Milano, Italy</div><div class="fm-affl" id="af14" lang="en"><sup>14</sup>Laboratory of Molecular Biology, IRCCS Istituto Auxologico Italiano, 20095 Cusano Milanino, Italy</div><div class="fm-affl" id="af15" lang="en"><sup>15</sup>Department of Neurology, ALS Center, Maggiore della Carità University Hospital, 28100 Novara, Italy</div><div class="fm-affl" id="af16" lang="en"><sup>16</sup>Institute of Neurological Sciences, National Research Council, 87050 Mangone, Italy</div><div class="fm-affl" id="af17" lang="en"><sup>17</sup>Division of Neurology, Department of Clinical and Experimental Medicine, University of Pisa, Nuovo Ospedale S.Chiara-Cisanello, 56126 Pisa, Italy</div><div class="fm-affl" id="af18" lang="en"><sup>18</sup>Department of Neurology and Psychiatry, University of Rome ‘Sapienza’, 00185 Roma, Italy</div><div class="fm-affl" id="af19" lang="en"><sup>19</sup>Department of Neurological Sciences, University Federico II, 80131 Napoli, Italy</div><div class="fm-affl" id="af20" lang="en"><sup>20</sup>Neurorehabilitation Unit, Azienda Ospedaliero-Universitaria Pisana, 56126 Pisa, Italy</div><div class="fm-affl" id="af21" lang="en"><sup>21</sup>Department of Laboratory Medicine, Medical Genetics Unit, Niguarda Ca’ Granda Hospital, 20162 Milano, Italy</div><div class="fm-affl" id="af22" lang="en"><sup>22</sup>NeuroMuscular Omnicentre (NEMO), Fondazione Serena Onlus, Niguarda Ca’ Granda Hospital, 20162 Milano, Italy</div><div class="fm-affl" id="af23" lang="en"><sup>23</sup>Department of Neurological and Psychiatric Sciences, University of Firenze, 50139 Firenze, Italy</div><div class="fm-affl" id="af24" lang="en"><sup>24</sup>Clinical Neurology, Section for Neuromuscular Diseases and Neuropathies, University Hospital ‘Spedali Civili’, 25123 Brescia, Italy</div><div class="fm-affl" id="af25" lang="en"><sup>25</sup>Department of Medical Sciences, University of Ferrara, 44100 Ferrara, Italy</div><div class="fm-affl" id="af26" lang="en"><sup>26</sup>Azienda Ospedaliera di Desio e Vimercate, 20871 Vimercate, Italy</div><div class="fm-affl" id="af27" lang="en"><sup>27</sup>Academic Neurology Unit, Department of Neuroscience, Faculty of Medicine, Dentistry and Health, University of Sheffield, Sheffield S10 2HQ, UK</div><div class="fm-affl" id="af28" lang="en"><sup>28</sup>School of Clinical and Experimental Medicine, College of Medicine and Dentistry, University of Birmingham, and</div><div class="fm-affl" id="af29" lang="en"><sup>29</sup>Neurosciences Division, University Hospitals Birmingham NHS Foundation Trust, Birmingham B15 2TT, UK</div><div class="fm-affl" id="af30" lang="en"><sup>30</sup>Department of Neurology, University Hospital Leuven, Leuven, Belgium</div><div class="fm-affl" id="af31" lang="en"><sup>31</sup>Laboratory for Neurobiology, Vesalius Research Center, VIB, University of Leuven, 3000 Leuven, Belgium</div><div class="fm-affl" id="af32" lang="en"><sup>32</sup>Department of Neuroscience, University of Torino, Torino, Italy</div><div class="fm-affl" id="af33" lang="en"><sup>33</sup>Azienda Ospedaliera Città della Salute e della Scienza, 10126 Torino, Italy</div><div class="fm-affl" id="af34" lang="en"><sup>34</sup>Neuromuscular Diseases Research Group, Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA</div><div class="fm-affl" id="af35" lang="en"><sup>35</sup>Institute for Clinical Neurobiology, University Hospital Wurzburg, 97078 Wurzburg, Germany</div><div class="fm-affl" id="af36" lang="en"><sup>36</sup>INSERM UMR-788 and University of Paris 11, Bicetre Hospital, Paris, Cedex 94275, France</div><div class="fm-affl" id="af37" lang="en"><sup>37</sup>Assistance Publique - Hôpitaux de Paris, Université Pierre et Marie Curie, Hôpital de la Salpêtrière, 75013 Paris, France</div><div class="fm-affl" id="af38" lang="en"><sup>38</sup>Academic Neurology Unit, Trinity Biomedical Sciences Institute, Trinity College Dublin, Dublin2, Ireland</div><div class="fm-affl" id="af39" lang="en"><sup>39</sup>Department of Clinical Neuroscience, Umeå University, SE-901 85 Umeå, Sweden</div><div class="fm-affl" id="af40" lang="en"><sup>40</sup>Brighton and Sussex Medical School, Trafford Centre for Biomedical Research, University of Sussex, Falmer East Sussex BN1 9RY, UK</div><div class="fm-affl" id="af41" lang="en"><sup>41</sup>Department of Neurology, Emory University, Atlanta, GA 30322, USA</div><div class="fm-affl" id="af42" lang="en"><sup>42</sup>Department of Neurology, Rudolf Magnus Institute of Neuroscience, University Medical Centre Utrecht, 3508 GA Utrecht, The Netherlands</div><div class="fm-affl" id="af43" lang="en"><sup>43</sup>Department of Medical and Molecular Genetics, King's College London, London SE1 9RT, UK</div><div class="fm-affl" id="af44" lang="en"><sup>44</sup>Department of Neurology, University of Massachusetts Medical School, Worcester, MA 01605, USA</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm"></div></div></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140459747954960_ai idm140459748205808_ai idm140459783353728_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140459747954960_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140459747954960_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140459747954960_ai" style="display:none"><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Department of Neuroscience, Institute of Psychiatry, King's College London, De Crespigny Park, London SE5 8AF, UK. Tel: +44 207848 5244; Fax: +44 2077080017; Email: <a href="mailto:dev@null" data-email="ku.ca.lck@hgof.allebasi" class="oemail">ku.ca.lck@hgof.allebasi</a></div><div id="AN1"><sup>†</sup>These authors contributed equally to the study.</div><div id="AN2"><sup>‡</sup>These two senior authors contributed equally to the study.</div><div id="AN3"><sup>§</sup>See Appendix for Slagen Consortium members.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140459747954960_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2013 Jul 31; Revised 2013 Nov 8; Accepted 2013 Nov 15.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140459747954960_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © Crown copyright 2013.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3959809/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3959809/bin/supp_23_8_2220__index.html" data-ga-action="click_feat_suppl">supp_23_8_2220__index.html</a><span style="color:gray"> (1.0K)</span></div><div class="small guid">GUID:&nbsp;EFA03E7E-1FB4-4EDB-88B3-92B378692C80</div>
<div><a href="/pmc/articles/PMC3959809/bin/supp_ddt587_ddt587supp.pdf" data-ga-action="click_feat_suppl">supp_ddt587_ddt587supp.pdf</a><span style="color:gray"> (974K)</span></div><div class="small guid">GUID:&nbsp;4863F86C-62CD-4C5E-9813-7ABDCD938330</div>
</div></dd></dl><dl data-length="251" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">Data were accessed by formal request to the consortia members. The complete summary meta-analysis data (SNP, genomic position, odds ratio and <em>P</em>-value) are freely available at the ALSOD website (<a href="http://alsod.iop.kcl.ac.uk/">http://alsod.iop.kcl.ac.uk/</a>) in a searchable format (<a href="#DDT587C47" rid="DDT587C47" class=" bibr popnode tag_hotlink tag_tooltip">47</a>).</p></dd></dl></div><div id="idm140459749532176" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140459749532176title">Abstract</h2><!--article-meta--><div><p id="__p4" class="p p-first-last">Identification of mutations at familial loci for amyotrophic lateral sclerosis (ALS) has provided novel insights into the aetiology of this rapidly progressing fatal neurodegenerative disease. However, genome-wide association studies (GWAS) of the more common (∼90%) sporadic form have been less successful with the exception of the replicated locus at 9p21.2. To identify new loci associated with disease susceptibility, we have established the largest association study in ALS to date and undertaken a GWAS meta-analytical study combining 3959 newly genotyped Italian individuals (1982 cases and 1977 controls) collected by SLAGEN (Italian Consortium for the Genetics of ALS) together with samples from Netherlands, USA, UK, Sweden, Belgium, France, Ireland and Italy collected by ALSGEN (the International Consortium on Amyotrophic Lateral Sclerosis Genetics). We analysed a total of 13 225 individuals, 6100 cases and 7125 controls for almost 7 million single-nucleotide polymorphisms (SNPs). We identified a novel locus with genome-wide significance at 17q11.2 (rs34517613 with <em>P</em> = 1.11 × 10<sup>−8</sup>; OR 0.82) that was validated when combined with genotype data from a replication cohort (<em>P</em> = 8.62 × 10<sup>−9</sup>; OR 0.833) of 4656 individuals. Furthermore, we confirmed the previously reported association at 9p21.2 (rs3849943 with <em>P</em> = 7.69 × 10<sup>−9</sup>; OR 1.16). Finally, we estimated the contribution of common variation to heritability of sporadic ALS as ∼12% using a linear mixed model accounting for all SNPs. Our results provide an insight into the genetic structure of sporadic ALS, confirming that common variation contributes to risk and that sufficiently powered studies can identify novel susceptibility loci.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p5" class="p p-first-last">Amyotrophic lateral sclerosis (ALS) is a late-onset progressive neurodegenerative disorder mainly affecting motor neurones. ALS is the most common adult onset motor neurone disease with prevalence of 5 per 100 000 and with family history, age and male gender as the major risk factors (<a href="#DDT587C1" rid="DDT587C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552405">1</a>–<a href="#DDT587C4" rid="DDT587C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_809257547">4</a>). While familial ALS is well characterized with several causative genes identified to date, the genetic architecture of the more common sporadic form is poorly understood. Previous genome-wide association studies (GWAS) have identified several loci associated with ALS risk such as <em>DPP6</em>, <em>ITPR2</em>, <em>FGGY</em> and <em>UNC13a</em> (<a href="#DDT587C5" rid="DDT587C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552422">5</a>–<a href="#DDT587C11" rid="DDT587C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552404">11</a>) that have failed to be replicated in independent populations (<a href="#DDT587C12" rid="DDT587C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552399">12</a>–<a href="#DDT587C15" rid="DDT587C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552401">15</a>). One exception is a locus on chromosome 9p21 (<a href="#DDT587C15" rid="DDT587C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552419">15</a>) that has been reliably replicated (<a href="#DDT587C16" rid="DDT587C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552446">16</a>). At this locus, an expanded hexanucleotide repeat in the <em>C9orf72</em> gene has been recently identified as the causative mutation in a large proportion of familial (23–47%) as well as sporadic (∼5%) ALS cases (<a href="#DDT587C17" rid="DDT587C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552437">17</a>–<a href="#DDT587C20" rid="DDT587C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552442">20</a>). This finding indicates that well-powered studies can identify novel loci associated with ALS susceptibility. To discover further loci, we designed a large GWAS meta-analysis with sufficient power to detect risk alleles with small effect sizes as observed in other neurodegenerative and complex diseases. There is evidence for a strong genetic component in sporadic ALS with heritability estimated to be 0.61 (95% CI: 0.38–0.78) in a combined study of 171 ALS twin pairs collected by three independent studies from Britain and Sweden (<a href="#DDT587C21" rid="DDT587C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552416">21</a>), so we sought to estimate heritability derived from unrelated individuals and explained by common variation using GWAS data. Phenotype variation in complex traits is owing to interactions of genetic and environmental factors; thus, the quantification of the genetic variance is relevant in the study of multifactorial diseases. Heritability, the proportion of phenotype explained by genetic variance, is typically estimated in closely related individuals such as twin pairs, but this can inflate estimates as a consequence of epistatic interactions or shared environment (<a href="#DDT587C22" rid="DDT587C22" class=" bibr popnode">22</a>,<a href="#DDT587C23" rid="DDT587C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552396">23</a>). In contrast, heritability explained by associated SNPs identified in GWAS as passing-accepted thresholds of significance (typically <em>P</em> ≤ 5 × 10<sup>−8</sup>) explains only a small fraction of the genetic variation in most complex diseases. The difference between the phenotypic variance explained by GWAS results and those estimated in family studies is referred to as the ‘missing heritability problem’ and possibly is explained by incomplete linkage disequilibrium (LD) between genotyped SNPs and causal variants and/or by the presence of gene-by-gene or gene-by-environment interactions (<a href="#DDT587C22" rid="DDT587C22" class=" bibr popnode">22</a>,<a href="#DDT587C23" rid="DDT587C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552435">23</a>). We have estimated heritability of ALS considering all SNPs simultaneously regardless of their association with ALS phenotype and compared this with heritability from twin studies.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Association analyses</h3><p id="__p6" class="p p-first">We analysed genotype raw data from eight independent studies including 3959 newly genotyped Italian individuals (1982 cases and 1977 controls) and 11 611 individuals (5195 cases and 6416 controls) from previously published studies. Full description of the sample size included in this study is reported in Table&nbsp;<a href="/pmc/articles/PMC3959809/table/DDT587TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT587TB1" rid-ob="ob-DDT587TB1" co-legend-rid=""><span>1</span></a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT587TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p7">Sample size and genotype platforms</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140459783723008" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Source</th><th rowspan="1" colspan="1" align="left">Sample ancestry</th><th rowspan="1" colspan="1" align="left">Sample size (<em>N</em>)</th><th rowspan="1" colspan="1" align="left">Cases (<em>N</em>)</th><th rowspan="1" colspan="1" align="left">Control (<em>N</em>)</th><th rowspan="1" colspan="1" align="left">Genotyping Illumina arrays</th><th rowspan="1" colspan="1" align="left">SNPs before imputation</th><th rowspan="1" colspan="1" align="left">SNPs after imputation</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">SLAGEN Consortium</td><td rowspan="1" colspan="1">Italian</td><td rowspan="1" colspan="1">3959</td><td rowspan="1" colspan="1">1982</td><td rowspan="1" colspan="1">1977</td><td rowspan="1" colspan="1">660K</td><td rowspan="1" colspan="1">657366</td><td rowspan="1" colspan="1">8391895</td></tr><tr><td rowspan="1" colspan="1">UMC Utrecht</td><td rowspan="1" colspan="1">Dutch</td><td rowspan="1" colspan="1">911</td><td rowspan="1" colspan="1">461</td><td rowspan="1" colspan="1">450</td><td rowspan="1" colspan="1">317K</td><td rowspan="1" colspan="1">317503</td><td rowspan="1" colspan="1">8328981</td></tr><tr><td rowspan="1" colspan="1">Utrecht, Umeå, Leuven</td><td rowspan="1" colspan="1">Dutch, Swedish, Belgian</td><td rowspan="1" colspan="1">2806</td><td rowspan="1" colspan="1">1364</td><td rowspan="1" colspan="1">1442</td><td rowspan="1" colspan="1">370K</td><td rowspan="1" colspan="1">317503</td><td rowspan="1" colspan="1">8372645</td></tr><tr><td rowspan="1" colspan="1">MGH, King's College, Evry</td><td rowspan="1" colspan="1">Northern American, British, French</td><td rowspan="1" colspan="1">3916</td><td rowspan="1" colspan="1">1710</td><td rowspan="1" colspan="1">2206</td><td rowspan="1" colspan="1">370K</td><td rowspan="1" colspan="1">307790</td><td rowspan="1" colspan="1">8370626</td></tr><tr><td rowspan="1" colspan="1">Beaumont Hospital, Dublin</td><td rowspan="1" colspan="1">Irish</td><td rowspan="1" colspan="1">432</td><td rowspan="1" colspan="1">221</td><td rowspan="1" colspan="1">211</td><td rowspan="1" colspan="1">550K</td><td rowspan="1" colspan="1">561466</td><td rowspan="1" colspan="1">8268635</td></tr><tr><td rowspan="1" colspan="1">NIH publicly available, Coriell biobank</td><td rowspan="1" colspan="1">Northern American</td><td rowspan="1" colspan="1">547</td><td rowspan="1" colspan="1">276</td><td rowspan="1" colspan="1">271</td><td rowspan="1" colspan="1">550K</td><td rowspan="1" colspan="1">555352</td><td rowspan="1" colspan="1">8325506</td></tr><tr><td rowspan="1" colspan="1">MND DNA bank and RADIANT study</td><td rowspan="1" colspan="1">British</td><td rowspan="1" colspan="1">2252</td><td rowspan="1" colspan="1">663</td><td rowspan="1" colspan="1">1589</td><td rowspan="1" colspan="1">610K</td><td rowspan="1" colspan="1">471994</td><td rowspan="1" colspan="1">8372429</td></tr><tr><td rowspan="1" colspan="1">NIH publicly available</td><td rowspan="1" colspan="1">Italian</td><td rowspan="1" colspan="1">747</td><td rowspan="1" colspan="1">500</td><td rowspan="1" colspan="1">247</td><td rowspan="1" colspan="1">550K</td><td rowspan="1" colspan="1">500002</td><td rowspan="1" colspan="1">8312642</td></tr></tbody></table></div><div id="largeobj_idm140459783723008" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3959809/table/DDT587TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140459749190128"><p id="__p8" class="p p-first-last">Independent studies analysed in the combined international meta-analysis were genotyped on different type of Illumina platforms. Italian data were collected by the SLAGEN Consortium (Italy) and the Dutch, Swedish, French, Belgian, Irish and North American, and Italian raw data were collected by the International ALS-GWAS Consortium (ALSAGEN). British cases were collected by UK National MND Bank samples and British controls by the RADIANT study (DeCC, BACCs) and NIH publicly available data from Coriell biobank (<a href="http://www.coriell.org">www.coriell.org</a>).</p></div></div></div><p id="__p9">Raw data for each study were assessed for quality control (QC) separately following the same criteria (see <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a> for full discussion). Population structure of the individual cohorts was studied by means of principal components analysis (PCA) with EIGENSTRAT software (<a href="#DDT587C24" rid="DDT587C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552411">24</a>,<a href="#DDT587C25" rid="DDT587C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552436">25</a>), and outliers were identified by the projection of the first 10 principal components (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Methods</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1</a>).</p><p id="__p10">To achieve the maximal coverage, we imputed genome-wide filtered originally genotyped data for each study (see Materials and methods). Overall, the average number of inferred genotypes was 8 342 920 SNPs varying proportionally to the original genotyping platform. After filtering (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Methods</a>), the eight datasets included genotype data for almost 7 million SNPs in 13 225 individuals (6100 cases and 7125 controls) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). Cleaned imputed genotypes were tested for association with ALS separately using SNPTESTv2.4.0 (<a href="#DDT587C26" rid="DDT587C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552395">26</a>), and the logistic regression analyses were adjusted by the appropriate principal component axes estimated in the individual cohorts (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). After genomic inflation control, we combined the logistic regression analysis results of each study in a meta-analysis using the program METAL (<a href="#DDT587C27" rid="DDT587C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552438">27</a>). We adopted the ‘standard error’ analysis scheme, which combines effect size estimates (<em>β</em>-coefficients) across the studies weighted according to the inverse of the corresponding standard errors. As a result of the meta-analysis, we observed two loci reaching genome-wide threshold of significance, rs3849943 with <em>P</em> = 7.69×10<sup>−9</sup> (OR 1.16, 95% CI: 1.10–1.22; average posterior probability (APP) 0.999; minor allele frequency (MAF) cases 0.268, controls 0.238) on chromosome 9p21 and SNP rs34517613 with <em>P</em> = 1.11 × 10<sup>−8</sup> (OR 0.82, 95% CI: 0.76–0.87; APP 0.9279; MAF cases 0.108, controls 0.129) on chromosome 17q11.2.</p><p id="__p11">A third locus, rs1788776,&nbsp;at 18q11.2 was very close to genome-wide significance threshold (<em>P</em> = 7.67 × 10<sup>−8</sup>, OR 0.87, 95% CI: 0.76–0.87; APP 0.9716; MAF cases 0.392, controls 0.362) (Table&nbsp;<a href="/pmc/articles/PMC3959809/table/DDT587TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT587TB2" rid-ob="ob-DDT587TB2" co-legend-rid=""><span>2</span></a>, Figs.&nbsp;<a href="/pmc/articles/PMC3959809/figure/DDT587F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT587F1" rid-ob="ob-DDT587F1" co-legend-rid="lgnd_DDT587F1"><span>1</span></a> and <a href="/pmc/articles/PMC3959809/figure/DDT587F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT587F2" rid-ob="ob-DDT587F2" co-legend-rid="lgnd_DDT587F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2,</span></span><span>2</span></a>, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>). With the exception of the locus on 9p21, loci previously found to be associated with ALS risk (<a href="#DDT587C5" rid="DDT587C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552449">5</a>–<a href="#DDT587C11" rid="DDT587C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552425">11</a>) did not reach genome-wide significance in our meta-analysis (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>) although SNP rs12608932 (<em>UNC13A</em>) at 19p13.3 had not complete coverage across datasets as it failed the QC threshold in the British dataset (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>). Additionally, examination of these previously associated loci in the Italian cohort as an independent replication study only found evidence for association for the 9p21 locus, and for the <em>ITPR2</em> locus while significant, direction of effect is the opposite to that reported in the original study (<a href="#DDT587C8" rid="DDT587C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552451">8</a>) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig S3</a>). Only the <em>ITPR2</em> locus showed significant evidence for heterogeneity across the different cohorts of European ancestry analysed here (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>). Further larger studies are needed to confirm the role of these loci in ALS susceptibility.


</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT587TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p12">ALS-GWAS meta-analysis results</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140459750518480" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Chromosome</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">A1</th><th rowspan="1" colspan="1" align="left">A2</th><th rowspan="1" colspan="1" align="left">APP</th><th rowspan="1" colspan="1" align="left">Info</th><th rowspan="1" colspan="1" align="left">MAF_ALS</th><th rowspan="1" colspan="1" align="left">MAF_CONT</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Direction</th><th rowspan="1" colspan="1" align="left">Low 95% CI</th><th rowspan="1" colspan="1" align="left">Up 95% CI</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs3849943</td><td rowspan="1" colspan="1">27533382</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.999</td><td rowspan="1" colspan="1">0.268</td><td rowspan="1" colspan="1">0.238</td><td rowspan="1" colspan="1">1.166</td><td rowspan="1" colspan="1">7.69E−09</td><td rowspan="1" colspan="1">+−−+++++</td><td rowspan="1" colspan="1">1.107</td><td rowspan="1" colspan="1">1.228</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs3849941</td><td rowspan="1" colspan="1">27528948</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.993</td><td rowspan="1" colspan="1">0.986</td><td rowspan="1" colspan="1">0.266</td><td rowspan="1" colspan="1">0.236</td><td rowspan="1" colspan="1">1.166</td><td rowspan="1" colspan="1">9.54E−09</td><td rowspan="1" colspan="1">−++−−−−−</td><td rowspan="1" colspan="1">1.106</td><td rowspan="1" colspan="1">1.229</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs2477523</td><td rowspan="1" colspan="1">27526894</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.997</td><td rowspan="1" colspan="1">0.992</td><td rowspan="1" colspan="1">0.269</td><td rowspan="1" colspan="1">0.239</td><td rowspan="1" colspan="1">1.163</td><td rowspan="1" colspan="1">1.06E−08</td><td rowspan="1" colspan="1">−++−−−−−</td><td rowspan="1" colspan="1">1.104</td><td rowspan="1" colspan="1">1.225</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs2453555</td><td rowspan="1" colspan="1">27553868</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.995</td><td rowspan="1" colspan="1">0.988</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.239</td><td rowspan="1" colspan="1">1.163</td><td rowspan="1" colspan="1">1.07E−08</td><td rowspan="1" colspan="1">−++−−−−−</td><td rowspan="1" colspan="1">1.104</td><td rowspan="1" colspan="1">1.225</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs34517613</td><td rowspan="1" colspan="1">23634379</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.922</td><td rowspan="1" colspan="1">0.734</td><td rowspan="1" colspan="1">0.108</td><td rowspan="1" colspan="1">0.129</td><td rowspan="1" colspan="1">0.822</td><td rowspan="1" colspan="1">1.11E−08</td><td rowspan="1" colspan="1">++++++++</td><td rowspan="1" colspan="1">0.769</td><td rowspan="1" colspan="1">0.879</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs12937920</td><td rowspan="1" colspan="1">23614784</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.951</td><td rowspan="1" colspan="1">0.854</td><td rowspan="1" colspan="1">0.125</td><td rowspan="1" colspan="1">0.149</td><td rowspan="1" colspan="1">0.83</td><td rowspan="1" colspan="1">1.24E−07</td><td rowspan="1" colspan="1">++++++++</td><td rowspan="1" colspan="1">0.775</td><td rowspan="1" colspan="1">0.889</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs9909055</td><td rowspan="1" colspan="1">23620255</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.217</td><td rowspan="1" colspan="1">0.249</td><td rowspan="1" colspan="1">0.857</td><td rowspan="1" colspan="1">1.98E−07</td><td rowspan="1" colspan="1">++++++++</td><td rowspan="1" colspan="1">0.809</td><td rowspan="1" colspan="1">0.908</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs16964203</td><td rowspan="1" colspan="1">23606837</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.978</td><td rowspan="1" colspan="1">0.954</td><td rowspan="1" colspan="1">0.224</td><td rowspan="1" colspan="1">0.257</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">2.55E−07</td><td rowspan="1" colspan="1">−−−−−−−−</td><td rowspan="1" colspan="1">0.812</td><td rowspan="1" colspan="1">0.911</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs2292633</td><td rowspan="1" colspan="1">23599507</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.956</td><td rowspan="1" colspan="1">0.906</td><td rowspan="1" colspan="1">0.221</td><td rowspan="1" colspan="1">0.254</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">2.82E−07</td><td rowspan="1" colspan="1">++++++++</td><td rowspan="1" colspan="1">0.812</td><td rowspan="1" colspan="1">0.911</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs35714695</td><td rowspan="1" colspan="1">23743915</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.973</td><td rowspan="1" colspan="1">0.934</td><td rowspan="1" colspan="1">0.169</td><td rowspan="1" colspan="1">0.192</td><td rowspan="1" colspan="1">0.845</td><td rowspan="1" colspan="1">3.21E−07</td><td rowspan="1" colspan="1">++++++++</td><td rowspan="1" colspan="1">0.792</td><td rowspan="1" colspan="1">0.901</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs739439</td><td rowspan="1" colspan="1">23747949</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.994</td><td rowspan="1" colspan="1">0.984</td><td rowspan="1" colspan="1">0.169</td><td rowspan="1" colspan="1">0.193</td><td rowspan="1" colspan="1">0.842</td><td rowspan="1" colspan="1">3.97E−07</td><td rowspan="1" colspan="1">++++++++</td><td rowspan="1" colspan="1">0.788</td><td rowspan="1" colspan="1">0.900</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs9900311</td><td rowspan="1" colspan="1">23619534</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.997</td><td rowspan="1" colspan="1">0.992</td><td rowspan="1" colspan="1">0.219</td><td rowspan="1" colspan="1">0.252</td><td rowspan="1" colspan="1">0.858</td><td rowspan="1" colspan="1">4.58E−07</td><td rowspan="1" colspan="1">++++++++</td><td rowspan="1" colspan="1">0.808</td><td rowspan="1" colspan="1">0.911</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">rs1788776</td><td rowspan="1" colspan="1">19498035</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.971</td><td rowspan="1" colspan="1">0.947</td><td rowspan="1" colspan="1">0.392</td><td rowspan="1" colspan="1">0.362</td><td rowspan="1" colspan="1">0.872</td><td rowspan="1" colspan="1">7.67E−08</td><td rowspan="1" colspan="1">+++++−++</td><td rowspan="1" colspan="1">0.830</td><td rowspan="1" colspan="1">0.917</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">rs1629335</td><td rowspan="1" colspan="1">19475137</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.956</td><td rowspan="1" colspan="1">0.927</td><td rowspan="1" colspan="1">0.383</td><td rowspan="1" colspan="1">0.353</td><td rowspan="1" colspan="1">0.874</td><td rowspan="1" colspan="1">1.00E−07</td><td rowspan="1" colspan="1">−−−−−−−−</td><td rowspan="1" colspan="1">0.832</td><td rowspan="1" colspan="1">0.918</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">rs62093304</td><td rowspan="1" colspan="1">19489968</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.979</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">0.389</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">0.875</td><td rowspan="1" colspan="1">2.09E−07</td><td rowspan="1" colspan="1">−−−−−+−−</td><td rowspan="1" colspan="1">0.832</td><td rowspan="1" colspan="1">0.920</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">rs12957850</td><td rowspan="1" colspan="1">19487958</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.988</td><td rowspan="1" colspan="1">0.976</td><td rowspan="1" colspan="1">0.408</td><td rowspan="1" colspan="1">0.382</td><td rowspan="1" colspan="1">0.878</td><td rowspan="1" colspan="1">2.64E−07</td><td rowspan="1" colspan="1">+++++−++</td><td rowspan="1" colspan="1">0.836</td><td rowspan="1" colspan="1">0.923</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">rs1652373</td><td rowspan="1" colspan="1">19482645</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.409</td><td rowspan="1" colspan="1">0.382</td><td rowspan="1" colspan="1">0.877</td><td rowspan="1" colspan="1">2.66E−07</td><td rowspan="1" colspan="1">+++++−</td><td rowspan="1" colspan="1">0.834</td><td rowspan="1" colspan="1">0.922</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">rs12606211</td><td rowspan="1" colspan="1">19499478</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.979</td><td rowspan="1" colspan="1">0.961</td><td rowspan="1" colspan="1">0.408</td><td rowspan="1" colspan="1">0.381</td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1">3.29E−07</td><td rowspan="1" colspan="1">+++++−++</td><td rowspan="1" colspan="1">0.838</td><td rowspan="1" colspan="1">0.924</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">rs1711468</td><td rowspan="1" colspan="1">19498537</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.945</td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1">0.384</td><td rowspan="1" colspan="1">0.882</td><td rowspan="1" colspan="1">4.16E−07</td><td rowspan="1" colspan="1">+++++−++</td><td rowspan="1" colspan="1">0.840</td><td rowspan="1" colspan="1">0.926</td></tr></tbody></table></div><div id="largeobj_idm140459750518480" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3959809/table/DDT587TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140459745674128"><p id="__p13" class="p p-first-last">Summary of the most significant associated SNPs in the international ALS-GWAS meta-analysis from imputed genotypes. Meta-analysis (METAL) was performed weighting by the β-coefficient estimates and the inverse of their corresponding standard errors. Effect size of reference allele A1 is expressed with positive and negative symbols; symbols ‘+’ and ‘−’ indicate the direction of the effect size (beta values from regression), where a plus symbol means that increasing frequency of allele A1 is correlated with increasing trait values and vice versa.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT587F1" co-legend-rid="lgnd_DDT587F1"><a href="/pmc/articles/PMC3959809/figure/DDT587F1/" target="figure" rid-figpopup="DDT587F1" rid-ob="ob-DDT587F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140459746084736" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddt58701.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddt58701.jpg" src="/pmc/articles/PMC3959809/bin/ddt58701.jpg"></div></a><div id="largeobj_idm140459746084736" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3959809/figure/DDT587F1/" target="figure" rid-figpopup="DDT587F1" rid-ob="ob-DDT587F1"></a><a target="object" href="/pmc/articles/PMC3959809/figure/DDT587F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT587F1"><div><a class="figpopup" href="/pmc/articles/PMC3959809/figure/DDT587F1/" target="figure" rid-figpopup="DDT587F1" rid-ob="ob-DDT587F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p14">Manhattan plot of the international ALS meta-analysis. Scatter plot of chromosome position (<em>x</em>-axis) against <em>–log<sub>10</sub></em> GWAS meta-analysis <em>P</em>-values (<em>y</em>-axis) from imputed data. The threshold of genome-wide significance (<em>P</em> = 5 × 10<sup>−8</sup>) is indicated as a horizontal red line. At locus 9p21.2, 18 SNPs close to <em>Corf72</em> gene lie above the red line (most significant SNP is rs3849943 with <em>P</em> = 7.69 × 10<sup>−9</sup>). Locus 17q11.2 shows SNP rs34517613 (<em>P</em> = 1.11 × 10<sup>−8</sup>) to be significantly associated. SNP rs1788776 at 18q11.2 is very close to the threshold of significance with <em>P</em> = 7.67 × 10<sup>−8</sup>. Manhattan plot was produced using <em>ggplot2</em> package in R.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT587F2" co-legend-rid="lgnd_DDT587F2"><a href="/pmc/articles/PMC3959809/figure/DDT587F2/" target="figure" rid-figpopup="DDT587F2" rid-ob="ob-DDT587F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140459746074944" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddt58702.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddt58702.jpg" src="/pmc/articles/PMC3959809/bin/ddt58702.jpg"></div></a><div id="largeobj_idm140459746074944" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3959809/figure/DDT587F2/" target="figure" rid-figpopup="DDT587F2" rid-ob="ob-DDT587F2"></a><a target="object" href="/pmc/articles/PMC3959809/figure/DDT587F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT587F2"><div><a class="figpopup" href="/pmc/articles/PMC3959809/figure/DDT587F2/" target="figure" rid-figpopup="DDT587F2" rid-ob="ob-DDT587F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p15">Regional association plots of the associated loci. LD structure of the three regions associated with ALS in the international GWAS meta-analysis. For each plot, the <em>−log<sub>10</sub> P-</em>values (left <em>y</em>-axis) of SNPs are shown according to their chromosomal positions (<em>x</em>-axis); the genetic recombination rates are shown on the right <em>y</em>-axis. −<em>log<sub>10</sub> P</em>-values are shown for both genotyped and imputed SNPs distributed in a 0.8-megabase genomic region. The top SNP of each region is indicated as a diamond, and SNPs colour reflects LD correlation (<em>r<sup>2</sup></em>). (<strong>A</strong>) locus at 9p21, (<strong>B</strong>) 17q11.2 and (<strong>C</strong>) 18q11.2. LD plots were generated by <em>LocusZoom v1.1</em>.</p></div></div></div><p id="__p16">As inclusion of non-confounding covariates such as gender and age can significantly reduce power to identify truly associated variants particularly when the disease prevalence is low (&lt;1%), we have not included non-confounding covariates in logistic regression analyses presented here (<a href="#DDT587C28" rid="DDT587C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552400">28</a>). In confirmation of our approach, the most significant SNPs (<em>P</em> ≤ 1 × 10<sup>−5</sup>) mapping in the two candidate loci at 17q and 18q showed no gender-by-SNP interaction in multi regression analysis (see Materials and methods).</p><p id="__p17">For these loci, we searched for secondary signals by performing logistic regression conditioning upon the most associated SNPs under an additive model. There was no evidence of independent SNPs in any of these regions (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>). As the international meta-analysis included samples from Northern and Southern Europe, we tested the most associated SNPs for genetic heterogeneity quantifying the effect of the degree of variation between studies (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Methods</a>). We observed no significant heterogeneity in the distribution of allele frequencies between European populations for these SNPs (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>).</p><p id="__p18">As additional evidence, we tested the candidate loci at 17q and 18q in a novel independent replication cohort of 2074 cases and 2556 controls collected in Italy, Netherlands and Germany (see full description in the <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Methods</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S8</a>). For each locus, we selected two variants, the most associated in the GWAS meta-analysis (imputed data) and combined joint analysis (original genotype data) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S9</a>). We found suggestive evidence for association in the independent replica analysis for SNP rs34517613 at 17q (<em>P</em> = 0.055; OR 0.89, 95% CI: 1.00–0.78) and when combined in meta-analysis with the full GWAS data, the association identified in the discovery sample became more significant (<em>P</em><sub>combined</sub> = 8.62 × 10<sup>−9</sup>; OR 0.833, 95% CI: 0.891–0.778). We found no additional evidence for association at the 18q11.2 locus (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S10</a>).</p><p id="__p19">The locus at 17q11.2 is gene rich with a number of plausible candidates (Fig.&nbsp;<a href="/pmc/articles/PMC3959809/figure/DDT587F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT587F2" rid-ob="ob-DDT587F2" co-legend-rid="lgnd_DDT587F2"><span>2</span></a>). In an effort to refine associations at this locus and at 18q11.2, we searched for non-synonymous SNPs in LD (<em>r<sup>2</sup></em> ≥ 0.4) with our two lead signals to identify possible functional variants that explained the associations with ALS risk. No non-synonymous variants were found in LD at the 17q11.2 locus; however, three SNPs, rs739439 (<em>r<sup>2</sup></em> = 0.416) in the 3′ untranslated region, the intronic SNPs rs35714695 (<em>r<sup>2</sup></em> = 0.457) and rs34660379 (<em>r<sup>2</sup></em> = 0.412) within the <em>SARM1</em> (sterile alpha and TIR-containing motif 1) gene were in LD (Table&nbsp;<a href="/pmc/articles/PMC3959809/table/DDT587TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT587TB2" rid-ob="ob-DDT587TB2" co-legend-rid=""><span>2</span></a>, Fig.&nbsp;<a href="/pmc/articles/PMC3959809/figure/DDT587F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT587F2" rid-ob="ob-DDT587F2" co-legend-rid="lgnd_DDT587F2"><span>2</span></a>).</p><p id="__p20">At the suggestive 18q11.2 locus, the lead SNP rs1788776 was only in strong LD with SNPs in the <em>ANKRD29</em> gene (Fig.&nbsp;<a href="/pmc/articles/PMC3959809/figure/DDT587F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT587F2" rid-ob="ob-DDT587F2" co-legend-rid="lgnd_DDT587F2"><span>2</span></a>), an ankyrin repeat domain-containing protein of undetermined function, these included missense SNPs (rs12956232, <em>r<sup>2</sup></em> = 0.78; rs1788758, <em>r<sup>2</sup></em> = 0.78; rs12960692, <em>r<sup>2</sup></em> = 0.74; rs11662113, <em>r<sup>2</sup></em> = 0.53) (Table&nbsp;<a href="/pmc/articles/PMC3959809/table/DDT587TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT587TB2" rid-ob="ob-DDT587TB2" co-legend-rid=""><span>2</span></a>, Fig.&nbsp;<a href="/pmc/articles/PMC3959809/figure/DDT587F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT587F2" rid-ob="ob-DDT587F2" co-legend-rid="lgnd_DDT587F2"><span>2</span></a>).</p><p id="__p21" class="p p-last">Next, we looked for cis eQTL (expression quantitative trait loci) in LD with our lead SNPs. All SNPs in LD (<em>r<sup>2</sup></em> ≥ 0.2) with rs34517613 and rs1788776 were analysed for association with cis eQTL using several publicly available eQTL databases (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>), but none reached the <em>P</em>-value threshold for significance. This should not be considered conclusive as expression data from central nervous system tissues are still rather limited across eQTL databases and further studies should be carried out.</p></div><div id="s2b" class="sec"><h3 id="s2btitle">Contribution of <em>C9orf72</em> to chromosome 9p21 association signal</h3><p id="__p22" class="p p-first-last">As chromosome 9p21 was found to be strongly associated in our meta-analysis, we further investigated the contribution of ALS sporadic cases that also carry the expanded hexanucleotide repeat in the <em>C9orf72</em> gene to this association. Information about carriers of the pathologic expansion in <em>C9orf72</em> gene was available for 2287 of 6100 (37.3%) ALS cases from the Italian, Dutch and British studies included in the meta-analysis. The frequency of the expanded repeats carriers in the sporadic cases we have analysed progressively decreased as you proceed from Northern to Southern Europe ranging from 8 to 4.7%, respectively, in the British and Italian patients (see Materials and methods, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S10</a>). Our results confirm data previously reported (<a href="#DDT587C17" rid="DDT587C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552409">17</a>–<a href="#DDT587C20" rid="DDT587C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552394">20</a>). We performed a meta-analysis of this sample subset (2287 sporadic cases and 4162 controls) including and excluding cases (<em>n</em> = 144) with the expanded hexanucleotide repeat. Significant association of SNP rs3849943 decreased from <em>P</em> = 7.72 × 10<sup>−5</sup> to <em>P</em> = 0.052 when the carriers were excluded from the logistic regression analysis, confirming that the association of 9p21 locus was largely dependent on carriers of the <em>C9orf72</em> expansion (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S4</a>).</p></div><div id="s2c" class="sec sec-last"><h3 id="s2ctitle">Heritability estimation</h3><p id="__p23" class="p p-first">We used the Genome-wide Complex Trait Analysis (GCTA) method (<a href="#DDT587C29" rid="DDT587C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552412">29</a>) to quantify additive genetic variance (narrow sense heritability) explained by all common SNPs regardless of their association with disease. GCTA uses a linear mixed-effect model that includes a random effect to estimate the polygenic component of trait variation (<a href="#DDT587C29" rid="DDT587C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552447">29</a>). We performed GCTA analysis on 13 225 individuals (6100 ALS cases and 7125 controls) whose pair wise relationship was estimated to be &lt;0.05 (PI_HAT) and used a restricted maximum likelihood (REML) algorithm to estimate the variance explained by imputed SNPs data. Analyses were carried out separately in the eight cohorts using a more stringent quality control threshold for imputed SNPs (MAF &gt;0.01, posterior probability &gt;0.9 and information measure &gt;0.8) (see Materials and methods, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>) under the assumption of an ALS prevalence of 5 per 100 000. Heritability estimates were similar (∼11.5%) for all cohorts (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S12</a>) and were combined in meta-analysis modelled for random effects and weighted by sample size. The summary heritability was 0.119 (95% CI: 0.110–0.127) (Fig.&nbsp;<a href="/pmc/articles/PMC3959809/figure/DDT587F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT587F3" rid-ob="ob-DDT587F3" co-legend-rid="lgnd_DDT587F3"><span>3</span></a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT587F3" co-legend-rid="lgnd_DDT587F3"><a href="/pmc/articles/PMC3959809/figure/DDT587F3/" target="figure" rid-figpopup="DDT587F3" rid-ob="ob-DDT587F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140459746220000" class="figure"><a href="/pmc/articles/PMC3959809/figure/DDT587F3/" target="figure" rid-figpopup="DDT587F3" rid-ob="ob-DDT587F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3959809_ddt58703.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddt58703.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3959809/bin/ddt58703.jpg"></a></div><div id="largeobj_idm140459746220000" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3959809/figure/DDT587F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT587F3"><div><a class="figpopup" href="/pmc/articles/PMC3959809/figure/DDT587F3/" target="figure" rid-figpopup="DDT587F3" rid-ob="ob-DDT587F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p24">Heritability estimate across ALS-GWA studies. Heritability estimated in eight independent ALS cohorts. Forest plot shows heritability value and confidence interval (95%) calculated in each study at the reported prevalence of 5/100 000. Black boxes indicate the single studies, and box sizes are proportional to the sample (<em>N</em>). Grey bars specify lower and upper limit of 95% confidence interval.</p></div></div></div><p id="__p25" class="p p-last">Exclusion of the chromosome 9p21 locus did not affect heritability estimates. As ALS is a disease of aging and age is increasing in the human population, an increase in the prevalence of ALS can be predicted. Consequentially, the true prevalence of ALS may be higher than that used in our analysis. We have therefore calculated heritability over a range of prevalences, but for all prevalences, estimation of heritability attributable to common variation does not exceed 0.25 (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S5</a>).</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p26" class="p p-first">In this study, we present the largest GWAS meta-analysis to date in ALS. Overall, we analysed 13 225 individuals with sufficient power to capture allelic association with small effects and low MAF. The novel associated SNP rs34517613 at 17q11.2 was confirmed when genotype data from the independent replication cohort were combined in meta-analysis with full GWAS data. We have also replicated the 9p21 locus at genome-wide significance; this locus was also identified by linkage studies of familial ALS patients and by analogy it may be that there is a corresponding familial locus for the 17q11.2 locus identified here, though we are not aware of any reports.</p><p id="__p27">The lead SNP at this locus was in LD with three SNPs in the <em>SARM1</em> gene. Interestingly, <em>SARM1</em> orthologues, <em>dSarm</em> in Drosophila melanogaster and <em>Sarm1</em> in mice, have been recently found to play a direct role in an axonal self-destruction pathway, a mechanism known as Wallerian degeneration, which shares morphological similarities with the axon dying-back degeneration observed in ALS and in other neurodegenerative pathologies (<a href="#DDT587C30" rid="DDT587C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_625065553">30</a>,<a href="#DDT587C31" rid="DDT587C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552415">31</a>). As axonal degeneration is an early feature of ALS progression, a mouse model for familial ALS, <em>SOD1G93A</em>, was crossed with the <em>Wld<sup>S</sup></em> mouse, a spontaneous mutant with phenotype of prolonged survival of injured axons because of a chromosomal rearrangement that disrupts two genes, <em>Ube4b</em> and <em>Nmnat1</em>. In the <em>SOD1G93A</em>/<em>Wld<sup>S</sup>,</em> the progression of axonopathies was attenuated, suggesting that the Wallerian pathway could be involved in the axon loss observed in ALS (<a href="#DDT587C32" rid="DDT587C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552428">32</a>,<a href="#DDT587C33" rid="DDT587C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_625065552">33</a>).</p><p id="__p28">We estimated the heritability of ALS owing to common variation as ∼12% based on a prevalence of 5 per 100 000 (<a href="#DDT587C1" rid="DDT587C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552408">1</a>,<a href="#DDT587C2" rid="DDT587C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552414">2</a>), although the true prevalence of ALS may be higher as consequence of increasing age in the human population as discussed earlier. We have also only considered autosomal SNPs and additional variance may be encoded on the X chromosome. Twin studies estimate heritability owing to all genetic variation including rare monogenic forms of the disease, whereas we have estimated heritability as a consequence of common polymorphisms either directly typed or imputed. The difference between heritability estimated from twin studies and from analysis of common SNP polygenic variation suggests a substantial role for variation not captured by genome-wide association studies.</p><p id="__p29">Our estimate for the heritability of ALS explained by common variation is lower than that for other late-onset neurodegenerative diseases. Using similar approaches, heritability for late-onset Parkinson's disease was 0.31 (<a href="#DDT587C34" rid="DDT587C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552443">34</a>), 0.24 for late-onset Alzheimer's disease and 0.30 for multiple sclerosis (<a href="#DDT587C35" rid="DDT587C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552420">35</a>). These differences in heritability owing to common variants are reflected in the difficulty in identifying genome-wide significant loci for ALS compared with these diseases. For example, a two-stage GWA study of Parkinson's disease with a stage 1 sample size of ∼6000 individuals identified two strong association signals (<a href="#DDT587C36" rid="DDT587C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552450">36</a>), and a study of Alzheimer's disease with a similar-sized stage 1 sample identified two loci additional to the well-replicated <em>APOE</em> locus (<a href="#DDT587C37" rid="DDT587C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552434">37</a>). In comparison with these studies, our combined sample size is &gt;13 000 individuals, suggesting that the genetic architecture of ALS may be different from other more common neurodegenerative diseases. A recent study of ALS in a Han Chinese population identified two loci of genome-wide significance in a combined sample of ∼5 000 (<a href="#DDT587C38" rid="DDT587C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552444">38</a>); strikingly these loci were not replicated in populations of European ancestry, nor in the combined meta-analysis reported here. This suggests there may be considerable heterogeneity in ALS risk loci across different ethnicities; in support of this, the expanded repeat in the <em>C9orf72</em> gene has been reported to be infrequent in Asian populations (<a href="#DDT587C39" rid="DDT587C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_428134424">39</a>,<a href="#DDT587C40" rid="DDT587C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552410">40</a>).</p><p id="__p30" class="p p-last">In conclusion, we have identified a novel locus for sporadic ALS risk at 17q11.2, but not replicated the suggestive evidence for a second locus at 18q11.2 and confirmed the association at 9p21 with the expanded hexanucleotide repeat in the <em>C9orf72</em> gene. Furthermore, we provide evidence from our heritability estimates that further common variation affecting ALS risk remains to be detected by current GWAS platforms and with larger cohorts but that denser genome-wide assays and next generation sequencing technologies are required to detect rarer variation.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Participating individuals</h3><p id="__p31" class="p p-first">The discovery sample consisted of a novel Italian cohort collected by the Italian Consortium for the Genetics of ALS (SLAGEN) and seven independent published studies collected by ALSGEN, The International Consortium on Amyotrophic Lateral Sclerosis Genetics (<a href="#DDT587C5" rid="DDT587C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552413">5</a>–<a href="#DDT587C11" rid="DDT587C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552445">11</a>,<a href="#DDT587C15" rid="DDT587C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552407">15</a>,<a href="#DDT587C16" rid="DDT587C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552432">16</a>). All patients fulfilled the El Escorial revised criteria for ALS (<a href="#DDT587C41" rid="DDT587C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552403">41</a>). Written informed consent according to the Declaration of Helsinki was obtained from all patients and healthy subjects participating to this study. Local ethical committees for each participating institution approved these studies. All samples across the eight datasets were of European ancestry, and a full description of demographic details is reported in the <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>. All patients with family history of ALS or carrying Mendelian risk genes were excluded from analyses. Screening for the expanded hexanucleotide repeat in the <em>C9orf72</em> gene were performed subsequently; information of mutation carries was available for a subset of cases included in the Italian, Dutch and British cohorts (see <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a> for full description).</p><p id="__p32" class="p p-last">A replication sample of 4630 individuals (2074 cases and 2556 controls) was collected across Italy, The Netherlands and Germany using the same diagnostic criteria (see <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>).</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping procedures</h3><p id="__p33" class="p p-first-last">Samples from the individual cohorts were genotyped on different Illumina beadchips (Illumina, CA, USA) as shown in Table&nbsp;<a href="/pmc/articles/PMC3959809/table/DDT587TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT587TB1" rid-ob="ob-DDT587TB1" co-legend-rid=""><span>1</span></a>. Additional SNPs were genotyped in the replication phase by PCR-based KASP technique by the KBioscience (UK) facility for the Italian samples (622 cases and 971 controls) or by TaqMan_2013 PCR method for the Dutch (877 cases and 1226 controls) and German samples (575 cases and 359 controls) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>). Cross-validation between methodologies showed 100% concordance.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Statistical analysis</h3><div id="s4c1" class="sec sec-first"><p></p><h4 id="s4c1title" class="inline">Quality control and meta-analysis of imputed data </h4><p id="__p34" class="p p-first">Before imputation analysis QC of samples and markers was performed separately in the eight studies using the PLINK software package (<a href="#DDT587C42" rid="DDT587C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552418">42</a>). A detailed description of QC procedure is reported in the <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a>. Genotype data from British cases and controls were merged and analysed as single cohort as fully described in the <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a>.</p><p id="__p35">Ancestry differences between individuals within each cohort were detected by PCA. Principal component axes were generated by genotypes of a genome-wide subset of LD-independent SNPs using EIGENSTRAT software (<a href="#DDT587C24" rid="DDT587C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552402">24</a>,<a href="#DDT587C25" rid="DDT587C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552398">25</a>), and outliers identified by the first 10 principal components (PCs) were removed. The number of significant principal components was estimated by Tracy–Widom distribution using the program TW statistic (<a href="#DDT587C25" rid="DDT587C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552431">25</a>) and included as covariates in the logistic regression analyses of each study for population stratification control. Scatter plots of PC1 and PC2 showed no evidence of substructure between cases and controls within the single cohorts (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). As an example, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S5</a> shows the population structure of the novel Italian cohort by the projection of the first two PC axes.</p><p id="__p36">After QC, original genotype data of each study were tested for genomic inflation and lambda estimates resulted to be minimal (λ<sub>(gc)</sub> &lt; 1.02). Individual datasets were imputed genome wide separately using the IMPUTE.v2 program (<a href="#DDT587C43" rid="DDT587C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552448">43</a>,<a href="#DDT587C44" rid="DDT587C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552426">44</a>), which employs combined reference panels of known phased haplotypes provided by HapMap 3 (Feb 2009), 1000 Genomes Project (Mar 2010) (NCBI build 36 coordinates) and the study's sample genotypes (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>). In each cohort, imputed genotypes were tested for association with ALS status by logistic regression analysis (SNPTESTv2.4.0) including the specific PCs as confounder covariates to control population stratification. For each test, statistic spurious associations and genomic inflation were controlled by plotting the observed quantiles versus the expected ones in Q–Q plots and by factor λ<sub>(gc)</sub> estimate (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S7</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S3</a>) (R package software).</p><p id="__p37">SNP statistic data were filtered for uncertainty of inferred genotypes according to posterior probability (APP) of &gt;0.9 and statistical information of allele frequency (info) of &gt;0.4 (<a href="#DDT587C26" rid="DDT587C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552441">26</a>). The average number of SNPs filtered out in each dataset was 1.47 million (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). Finally, filtered SNP tables were combined in a meta-analysis and analysed with the program METAL (<a href="#DDT587C27" rid="DDT587C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552433">27</a>), applying the standard error scheme option that weights effect size estimates, or <em>β</em>-coefficients, using the inverse of the corresponding standard errors. The <em>z</em>-scores were calculated from the <em>P</em>-values of each study with the sign of the <em>z</em>-scores indicating the direction of allelic effect in that study (<em>z</em> &gt; 0 for odds ratio &gt;1). Meta-analysis was performed applying the fixed-effects model that does not account for study heterogeneity; therefore, heterogeneity was quantified by the Cochran's statistic (<em>Q</em>) and <em>I<sup>2</sup></em> (METAL). <em>P</em>-values were estimated by comparing the statistic with a <em>X<sup>2</sup></em> distribution (<em>N</em>—1&nbsp;‘degrees of freedom’, <em>N</em> = number of studies) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S5 and S7</a>) (<a href="#DDT587C45" rid="DDT587C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552424">45</a>,<a href="#DDT587C46" rid="DDT587C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552430">46</a>). Meta-analysis was repeated stratifying by gender and testing for the gender-by-SNP interaction as described in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>.</p><p id="__p38" class="p p-last">For the independent replication cohort, logistic regression analyses (SNPTESTv2.4.0) were performed separately in the Italian, Dutch and German samples and then combined in meta-analysis (METAL) applying standard error scheme as described earlier. Next, statistic tables of the three novel replication samples were combined with the discovery sample in a final meta-analysis (METAL).</p></div><div id="s4c2" class="sec sec-last"><p></p><h4 id="s4c2title" class="inline">Power calculation </h4><p id="__p39" class="p p-first-last">The final meta-analysis cohort had 99.99% power to detect allelic association with an odds ratio (OR) of 1.2 and MAF of 0.25 at genome-wide significance (<em>P</em> = 5 × 10<sup>−8</sup>). For low-frequency variants with MAF ranging from 0.01 to 0.04 and of small effect (OR &lt; 1.5), there was low power to capture variants with MAF 0.01, whereas for SNPs with MAF 0.02 power ranged from 60 to 98% for alleles with OR of 1.3 and 1.4, respectively (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S8</a>).</p></div></div><div id="s4d" class="sec"><h3 id="s4dtitle">Screening of expanded repeats in <em>C9orf72</em> gene</h3><p id="__p40" class="p p-first-last">Hexanucleotide repeat expansion data for the <em>C9orf72</em> gene were available for the Italian, Dutch and British cases (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S10</a>). As previously reported, carriers were defined as individuals with a range of the GGGGCC repeats of &gt;23 units (<a href="#DDT587C17" rid="DDT587C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552423">17</a>–<a href="#DDT587C20" rid="DDT587C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552417">20</a>). We performed conditional logistic regression analysis on a subset of 2287 screened cases and 4162 controls including and excluding pathologic expansions carriers (<em>n</em> = 144).</p></div><div id="s4e" class="sec sec-last"><h3 id="s4etitle">Estimation of the genetic variance tagged by all SNPs</h3><p id="__p41" class="p p-first-last">We used GCTA software (<a href="#DDT587C29" rid="DDT587C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552421">29</a>) to estimate the proportion of ALS phenotypic variance explained by autosomal SNPs distributed genome wide. Genetic relationships between pairs of individuals were calculated separately in the eight cohorts where all samples were previously tested for cryptic relatedness and excluded if the proportion of IBD (identical-by-descent) estimate was &gt;0.05 (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). We estimated the genetic relationship matrix (GRM) including imputed genotypes filtered by MAF &gt;0.01, posterior probability &gt;0.9 and information measure &gt;0.8. Filtered genotype data were submitted to GRM analysis, and GRMs output data were used in the REML analysis. In each dataset, REML analysis was carried out by fitting the specific principal components (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>) as covariates to control for possible population stratification. The variance estimate was transformed from the observed scale (V(1)/Vp) to a scale of liability (V(1)/Vp_L) by specifying a disease prevalence for ALS of 5 per 100 000 persons (<a href="#DDT587C1" rid="DDT587C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552406">1</a>,<a href="#DDT587C2" rid="DDT587C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552397">2</a>). The variance estimate of each cohort was combined by random effects meta-analysis weighted by sample size using the library <em>rmeta</em> in R toolset.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p42" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt587/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">DATA AVAILABILITY</h2><p id="__p43" class="p p-first-last">Data were accessed by formal request to the consortia members. The complete summary meta-analysis data (SNP, genomic position, odds ratio and <em>P</em>-value) are freely available at the ALSOD website (<a href="http://alsod.iop.kcl.ac.uk/">http://alsod.iop.kcl.ac.uk/</a>) in a searchable format (<a href="#DDT587C47" rid="DDT587C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_362552439">47</a>).</p></div><div id="s7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s7title">FUNDING</h2><p id="__p44" class="p p-first-last">The SLAGEN Consortium was supported by funds from the Ministero della Salute Italiano (GWA-SLA, Ricerca Finalizzata 2007 grant number <span>31</span>) and Fondazione Istituto Auxologico Italiano; I.F., C.G., B.C., J.F.P. and V.S. were supported by the Istituto Superiore di Sanità (grant number <span>526D/34</span>, 2006); I.F. received salary support from ‘Amici del Centro Dino Ferrari’; V.S. acknowledges support from the ‘Amici del Centro Dino Ferrari’. A.R., C.G., B.C., N.T. and V.S. were supported by the Ministero della Salute Italiano (ALS-FTD, <span>RF-2009-1473856</span>); G.Si. was supported by Monte dei Paschi di Siena Foundation (grant <span>39167</span>) and the Ministero della Salute Italiano (grant number <span>RF-INP-2007-627227</span>); A.C. was supported in part by Federazione Italiana Giuoco Calcio and Fondazione Vialli e Mauro per la Sclerosi Laterale Amiotrofica onlus, Ministero della Salute Italiano (grant number RF-PIE-2007-635695); AF was supported by Progetto Regione Emilia Romagna-Università ‘RARER’. The Italian replica control samples were obtained from the ‘Parkinson Institute Biobank’ (<a href="http://www.parkinsonbiobank.com">http://www.parkinsonbiobank.com</a>) of the Telethon Genetic Biobank Network (<a href="http://www.biobanknetwork.org">http://www.biobanknetwork.org</a>) supported by TELETHON Italy (grant number <span>GTB07001</span>) and by ‘Fondazione Grigioni per il Morbo di Parkinson’. ALSAGEN Consortium is supported by funds from the National Institutes for Health Research Biomedical Research Centre for Mental Health at the South London and Maudsley National Health Service Foundation Trust and Institute of Psychiatry, King's College London, the
University of Leuven (grant number <span>GOA/11/014</span>), the Interuniversity Attraction Poles (IUAP) program P7/16 of the Belgian Federal Science Policy Office and the 7th Framework Programme of the European Union (ADAMS project, grant numbers <span>HEALTH-F4-2009-242257</span>; and <span>FP7/2007-2013</span>;, grant agreements <span>259867</span>; and <span>278611</span>). P.V.D. holds a clinical investigatorship from FWO-Vlaanderen; W.R. is supported through the E. von Behring Chair for Neuromuscular and Neurodegenerative Disorders. The UK National DNA Bank for Motor Neuron Disease Research was funded by the Motor Neurone Disease Association (<span>grant 3/3</span>), the Wellcome Trust (grant number <span>070122/A/02/Z</span>) and the National Institute for Health Research Dementias and Neurodegenerative Diseases Research Network. A.A.-C. and J.F.P. receive salary support from the National Institute for Health Research Dementia Biomedical Research
Unit at South London and Maudsley National Health Service Foundation Trust and King's College London. The RADIANT depression and bipolar case control studies (DeCC, BACC) were genotyped under Medical Research Council (grant number <span>G0701420</span>). A.A.-C. thanks the Motor Neurone Disease Association of Great Britain and Northern Ireland, the ALS Association, the ALS Therapy Alliance and the Medical Research Council (grant number <span>G0600974</span>) for support. R.H.B. acknowledges support from the National Institute of Neurological Disorders and Stroke (grant number <span>5RO1-NS050557-05</span>) and the National Institute of Neurological Disorders and Stroke American Recovery and Reinvestment Act Award (grant number <span>RC2-NS070-342</span>), the Angel Fund, the ALS Association, P2ALS, Project ALS, the Pierre L. de Bourgknecht ALS Research Foundation and the ALS Therapy Alliance. J.E.L. acknowledges grant support from National Institute of Health/ National Institute of Neurological Disorders and Stroke (grant number <span>1R01NS073873</span>). The Dutch, Belgian and Swedish GWAS data generation was supported by the Prinses Beatrix Fonds, VSB fonds, H. Kersten and M. Kersten (Kersten Foundation), The Netherlands ALS Foundation, and J.R. van Dijk and the Adessium Foundation.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140459749898800"><a href="/pmc/articles/PMC3959809/bin/supp_23_8_2220__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140459749895168" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140459749895168title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p45">Genetic and statistical analysis was performed using a Linux cluster supported by the National Institute for Health Research Biomedical Research Centre for Mental Health at the South London and Maudsley National Health Service Foundation Trust and Institute of Psychiatry King's College London. G.So. acknowledges EuroBiobank for ALS DNA samples; K.M. acknowledges Motor Neurone Disease Association UK, Queen Elizabeth Hospital Birmingham Charity. M.S. acknowledges support through EU via the Euro-MOTOR consortium and German BMBF Motoneuron disease network.</p><p id="__p46"><em>Conflict of Interest statement</em><em>:</em> None declared.</p></div></div><div id="idm140459784275504" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140459784275504title">APPENDIX</h2><!--/article/back/app-group/app/--><div id="s8" class="sec sec-first"><div id="s8a" class="sec sec-first"><p></p><h4 id="s8atitle" class="inline">Slagen Consortium members </h4><p id="__p47" class="p p-first-last">Cereda Cristina (Laboratory of Experimental Neurobiology, IRCCS ‘C. Mondino’ National Institute of Neurology Foundation, Pavia, Italy), Comi Giacomo (Neurologic Unit, IRCCS Foundation Ca' Granda Ospedale Maggiore Policlinico, and Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center, Università degli Studi di Milano, Milano, Italy), D'Alfonso Sandra (Department of Health Sciences, Interdisciplinary Research Center of Autoimmune Diseases, ‘A. Avogadro’ University, Novara, Italy), Fogh Isabella (Department of Neuroscience, Institute of Psychiatry, King's College London, London, UK; and Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, Milano, Italy), Gellera Cinzia (Unit of Genetics of Neurodegenerative and Metabolic Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, Milano, Italy), Ratti Antonia (Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano; and Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center, Università degli Studi di Milano, Milano, Italy), Silani Vincenzo (Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano; and Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center, Università degli Studi di Milano, Milano, Italy), Sorarù Gianni (Department of Neurosciences, University of Padova, Padova, Italy).</p></div><div id="s8b" class="sec"><p></p><h4 id="s8btitle" class="inline">Slagen collaborators </h4><p id="__p48" class="p p-first-last">Bongioanni Paolo (Neurorehabilitation Unit, Azienda Ospedaliero-Universitaria Pisana, Pisa, Italy), Brescia Morra Vincenzo (Department of Neurological Sciences, University Federico II, Napoli, Italy), Calini Daniela (Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, Milan, Italy), Castellotti Barbara (Unit of Genetics of Neurodegenerative and Metabolic Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, Milano, Italy), Ceroni Mauro (Laboratory of Experimental Neurobiology, IRCCS ‘C. Mondino’ National Institute of Neurology Foundation, Pavia, Italy), Conforti Francesca Luisa (Institute of Neurological Sciences, National Research Council, Mangone, Italy), Corbo Massimo (NeuroMuscular Omnicentre-NEMO, Fondazione Serena Onlus, Niguarda Ca' Granda Hospital, Milano, Italy), Corrado Lucia (Department of Health Sciences, Interdisciplinary Research Center of Autoimmune Diseases, ‘A. Avogadro’ University, Novara, Italy), Corti Stefania (Neurologic Unit, IRCCS Foundation Ca' Granda Ospedale Maggiore Policlinico; and Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center, Università degli Studi di Milano, Milano, Italy), Del Bo Roberto (Neurologic Unit, IRCCS Foundation Ca' Granda Ospedale Maggiore Policlinico; and Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center, Università degli Studi di Milano, Milano, Italy), Ferlini Alessandra (Department of Medical Sciences, University of Ferrara, Ferrara, Italy), Filla Alessandro (Department of Neurological Sciences, University Federico II, Napoli, Italy), Filosto Massimo (Section for Neuromuscular Diseases and Neuropathies, Clinical Neurology, University Hospital ‘Spedali Civili’, Brescia, Italy), Gagliardi Stella (Laboratory of Experimental Neurobiology, IRCCS ‘C. Mondino’ National Institute of Neurology Foundation, Pavia, Italy), Inghilleri Maurizio (Department of Neurology and Psychiatry, University of Rome ‘Sapienza’, Roma, Italy), Marsili Angela (Department of Neurological Sciences, University Federico II, Napoli, Italy), Mazzini Letizia (Department of Health Sciences, Interdisciplinary Research Center of Autoimmune Diseases, ‘A. Avogadro’ University; and ALS Center Department of Neurology, ‘Maggiore della Carità’ University Hospital, Novara, Italy), Pegoraro Elena (Department of Neurosciences, University of Padova, Padova, Italy), Penco Silvana (Department of Laboratory Medicine, Medical Genetics Unit, Niguarda Ca' Granda Hospital, Milano, Italy), Pensato Viviana (Unit of Genetics of Neurodegenerative and Metabolic Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, Milano, Italy), Puorro Giorgia (Department of Neurological Sciences, University Federico II, Napoli, Italy), Querin Giorgia (Department of Neurosciences, University of Padova, Padova, Italy), Saccà Francesco (Department of Neurological Sciences, University Federico II, Napoli, Italy), Siciliano Gabriele (Division of Neurology, Department of Clinical and Experimental Medicine, University of Pisa, Nuovo Ospedale S.Chiara-Cisanello, Pisa, Italy), Sorbi Sandro (Department of Neurological and Psychiatric Sciences, University of Firenze, Firenze, Italy), Taroni Franco (Unit of Genetics of Neurodegenerative and Metabolic Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, Milano, Italy), Ticozzi Nicola (Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano; and Department of Pathophysiology and Tranplantation, ‘Dino Ferrari’ Center, Università degli Studi di Milano, Milano, Italy), Tiloca Cinzia (Department of Neurology and Laboratory of Neuroscience, IRCCS Istituto Auxologico Italiano, Milan, Italy).</p></div><div id="s8c" class="sec"><p></p><h4 id="s8ctitle" class="inline">Collaborators </h4><p id="__p49" class="p p-first-last">La Bella Vincenzo (ALS Clinical Research Center, Department of Experimental BioMedicine and Clinical Neurosciences, Bio.Ne.C, University of Palermo, Palermo, Italy), Logroscino Giancarlo (Department of Neurosciences and Sense Organs, University of Bari, Policlinico, Bari, Italy), Monsurrò Maria Rosaria (Second Division of Neurology, Second University of Naples, Naples, Italy), Quattrone Aldo (Institute of Neurology, University Magna Graecia, Catanzaro, Italy), Simone Isabella Laura (Department of Neurosciences and Sense Organs, University of Bari, Policlinico, Bari, Italy).</p></div><div id="s8d" class="sec sec-last"><p></p><h4 id="s8dtitle" class="inline">Alsgen consortium collaborators </h4><p id="__p50" class="p p-first-last">Ahmeti Kreshnik B. (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Ajroud-Driss Senda (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Al-Chalabi Ammar (MRC Centre for Neurodegeneration Research, Institute of Psychiatry, King's College London, London, UK), Andersen Peter M. (Institute of Pharmacology and Clinical Neuroscience, Section for Neurology, Umeå University, Umeå, Sweden), Armstrong Jennifer (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Birve Anne (Institute of Pharmacology and Clinical Neuroscience, Section for Neurology, Umeå University, Umeå, Sweden), Blauw Hylke M. (Rudolf Magnus Institute of Neuroscience, Department of Neurology, University Medical Centre Utrecht, Utrecht, The Netherlands), Brown Robert H. (Department of Neurology, University of Massachusetts Medical School, Worcester, MA, USA.), Bruijn Lucie (The ALS Association), Chen Wenjie (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Chiò Adriano (ALS Center, Department of Neuroscience, University of Turin, Torino, Italy), Comeau Mary C. (Wake Forest University Health Sciences, Medical Center Blvrd, Winston-Salem, NC, USA), Cronin Simon (Department of Neurology, Beaumont Hospital, Dublin, Ireland), Diekstra Frank P. (Rudolf Magnus Institute of Neuroscience, Department of Neurology, University Medical Centre Utrecht, Utrecht, The Netherlands), Soraya Gkazi Athina (King's Health Partners Centre for Neurodegeneration Research and Department of Clinical Neurosciences, Institute of Psychiatry Kings College London, London, UK), Glass Jonathan D. (Department of Neurology, Center for Neurodegenerative Disease, Emory University School of Medicine, Atlanta, GA, USA), Grab Josh D. (Wake Forest University Health Sciences, Medical Center Blvrd, Winston-Salem, NC, USA), Groen Ewout J. (Rudolf Magnus Institute of Neuroscience, Department of Neurology, University Medical Centre Utrecht, Utrecht, The Netherlands), Haines Jonathan L. (Vanderbilt University Medical Center, Nashville, TN, USA), Hardiman Orla (Dept Neurology, Beaumont Hospital and Trinity College Dublin, Dublin, Ireland), Heller Scott (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Huang Jie (Wellcome Trust Sanger Institute, Hinxton, UK), Hung Wu-Yen (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), ITALSGEN Consortium, Jaworski James M. (University of Miami, Miller School of Medicine, Biomedical Research Building, Miami, FL, USA), Jones Ashley (MRC Centre for Neurodegeneration Research, Institute of Psychiatry, King's College London, London, UK), Khan Humaira, Landers John E. (Department of Neurology, University of Massachusetts Medical School, Worcester, MA, USA.), Langefeld Carl D. (Wake Forest University Health Sciences, Medical Center Blvrd, Winston-Salem, NC, USA), Leigh Nigel P. (Brighton and Sussex medical School, Trafford Centre for Medical Research, Falmer, Brighton,UK), Marion Miranda C. (Wake Forest University Health Sciences, Medical Center Blvrd, Winston-Salem, NC, USA), McLaughlin Russell L. (Dept Population Genetics, Smurfit Institute, Trinity College Dublin, Dublin, Ireland), Meininger Vincent (Centre Référent Maladies Rares, APHP, UPMC, Hôpital de la Salpêtrière, Paris, France), Melki Judith (Institut National de la Santé et de la Recherche Médicale UMR-788, University of Paris 11, Biomedical Institute of Bicetre, Le Kremlin-Bicêtre, France), Miller Jack W. (King's Health Partners Centre for Neurodegeneration Research and Department of Clinical Neurosciences, Institute of Psychiatry, Kings College London, London, UK), Mora Gabriele (Department of Neurorehabilitation, Salvatore Maugeri Foundation, IRCCS, Scientific Institute of Milan, Milan, Italy), Pericak-Vance Margaret A. (University of Miami, Miller School of Medicine, Biomedical Research Building (BRB) #313, Miami, FL, USA), Rampersaud Evadnie (University of Miami, Miller School of Medicine, Biomedical Research Building (BRB) #313, Miami, FL, USA), Robberecht Wim (Neurology Department, KU Leuven, Belgium; and Vesalius Research Center, VIB, Leuven, Belgium), Shaw Christopher E. (PO43, Department of Clinical Neurosciences, Institute of Psychiatry, Kings College London, London, UK), Siddique Nailah (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Siddique Teepu (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Smith Bradley N. (King's Health Partners Centre for Neurodegeneration Research; PO43, Department of Clinical Neurosciences, Institute of Psychiatry Kings College London, London, UK), Sufit Robert (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Topp Simon (King's Health Partners Centre for Neurodegeneration Research; PO43, Department of Clinical Neurosciences, Institute of Psychiatry, Kings College London, London, UK), Traynor Bryan J. (Laboratory of Neurogenetics, National Institutes of Health, Bethesda, MD, USA), Vance Caroline (King's Health Partners Centre for Neurodegeneration Research, London, UK), van Damme Philip (Neurology Department, KU Leuven, Belgium; and Vesalius Research Center, VIB, Leuven, Belgium), van den Berg Leonard H. (Rudolf Magnus Institute of Neuroscience, Department of Neurology, University Medical Centre Utrecht, Utrecht, The Netherlands), van Es Michael (Rudolf Magnus Institute of Neuroscience, Department of Neurology, University Medical Centre Utrecht, Utrecht, The Netherlands), van Vught Paul (Rudolf Magnus Institute of Neuroscience, Department of Neurology, University Medical Centre Utrecht, Utrecht, The Netherlands), Veldink Jan H. (Rudolf Magnus Institute of Neuroscience, Department of Neurology, University Medical Centre Utrecht, Utrecht, The Netherlands), Yang Yi (Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA), Zheng J.G.(Northwestern University School of Medicine, Davee Department of Neurology, Division of Neuromuscular Medicine, Chicago, IL, USA).</p></div></div></div><div id="idm140459749892080" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140459749892080title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDT587C1">1. <span class="element-citation">Mitchell J.D., Borasio G.D. Amyotrophic lateral sclerosis. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">369</span>:2031–2041.</span> [<a href="/pubmed/17574095" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Amyotrophic+lateral+sclerosis&amp;author=J.D.+Mitchell&amp;author=G.D.+Borasio&amp;volume=369&amp;publication_year=2007&amp;pages=2031-2041&amp;pmid=17574095&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C2">2. <span class="element-citation">Beghi E., Logroscino G., Chiò A., Hardiman O., Mitchell D., Swingler R., Traynor B.J. EURALS Consortium. The epidemiology of ALS and the role of population-based registries. <span><span class="ref-journal">Biochim. Biophys. Acta. </span>2006;<span class="ref-vol">1762</span>:1150–1157.</span> [<a href="/pubmed/17071060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta&amp;title=The+epidemiology+of+ALS+and+the+role+of+population-based+registries&amp;author=E.+Beghi&amp;author=G.+Logroscino&amp;author=A.+Chiò&amp;author=O.+Hardiman&amp;author=D.+Mitchell&amp;volume=1762&amp;publication_year=2006&amp;pages=1150-1157&amp;pmid=17071060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C3">3. <span class="element-citation">Armon C. An evidence based medicine approach to the evaluation of the role of exogenous risk factors in sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Neuroepidemiology. </span>2003;<span class="ref-vol">22</span>:217–228.</span> [<a href="/pubmed/12792141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroepidemiology&amp;title=An+evidence+based+medicine+approach+to+the+evaluation+of+the+role+of+exogenous+risk+factors+in+sporadic+amyotrophic+lateral+sclerosis&amp;author=C.+Armon&amp;volume=22&amp;publication_year=2003&amp;pages=217-228&amp;pmid=12792141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C4">4. <span class="element-citation">Manjaly Z.R., Scott K.M., Abhinav K., Wijesekera L., Ganesalingam J., Goldstein L.H., Janssen A., Dougherty A., Willey E., Stanton B.R., et al.  The sex ratio in amyotrophic lateral sclerosis, A population based study. <span><span class="ref-journal">Amyotroph. Lateral. Scler. </span>2010;<span class="ref-vol">11</span>:439–442.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6485484/">PMC free article</a>]</span> [<a href="/pubmed/20225930" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Amyotroph.+Lateral.+Scler.&amp;title=The+sex+ratio+in+amyotrophic+lateral+sclerosis,+A+population+based+study&amp;author=Z.R.+Manjaly&amp;author=K.M.+Scott&amp;author=K.+Abhinav&amp;author=L.+Wijesekera&amp;author=J.+Ganesalingam&amp;volume=11&amp;publication_year=2010&amp;pages=439-442&amp;pmid=20225930&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C5">5. <span class="element-citation">van Es M.A., van Vught P.W., Blauw H.M., Franke L., Saris C.G., Van den Bosch L., de Jong S.W., de Jong V., Baas F., van't Slot R., et al.  Genetic variation in DPP6 is associated with susceptibility to amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">40</span>:29–31.</span> [<a href="/pubmed/18084291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+variation+in+DPP6+is+associated+with+susceptibility+to+amyotrophic+lateral+sclerosis&amp;author=M.A.+van+Es&amp;author=P.W.+van+Vught&amp;author=H.M.+Blauw&amp;author=L.+Franke&amp;author=C.G.+Saris&amp;volume=40&amp;publication_year=2007&amp;pages=29-31&amp;pmid=18084291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C6">6. <span class="element-citation">Cronin S., Berger S., Ding J., Schymick J.C., Washecka N., Hernandez D.G., Greenway M.J., Bradley D.G., Traynor B.J., Hardiman O. A genome-wide association study of sporadic ALS in a homogenous Irish population. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:768–774.</span> [<a href="/pubmed/18057069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+genome-wide+association+study+of+sporadic+ALS+in+a+homogenous+Irish+population&amp;author=S.+Cronin&amp;author=S.+Berger&amp;author=J.+Ding&amp;author=J.C.+Schymick&amp;author=N.+Washecka&amp;volume=17&amp;publication_year=2008&amp;pages=768-774&amp;pmid=18057069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C7">7. <span class="element-citation">Schymick J.C., Scholz S.W., Fung H.C., Britton A., Arepalli S., Gibbs J.R., Lombardo F., Matarin M., Kasperaviciute D., Hernandez D.G., et al.  Genome-wide genotyping in amyotrophic lateral sclerosis and neurologically normal controls, first stage analysis and public release of data. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:322–328.</span> [<a href="/pubmed/17362836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol.&amp;title=Genome-wide+genotyping+in+amyotrophic+lateral+sclerosis+and+neurologically+normal+controls,+first+stage+analysis+and+public+release+of+data&amp;author=J.C.+Schymick&amp;author=S.W.+Scholz&amp;author=H.C.+Fung&amp;author=A.+Britton&amp;author=S.+Arepalli&amp;volume=6&amp;publication_year=2007&amp;pages=322-328&amp;pmid=17362836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C8">8. <span class="element-citation">van Es M.A., Van Vught P.W., Blauw H.M., Franke L., Saris C.G., Andersen P.M., Van Den Bosch L., de Jong S.W., van’ t Slot R., Birve A., et al.  ITPR2 as a susceptibility gene in sporadic amyotrophic lateral sclerosis, a genome-wide association study. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:869–877.</span> [<a href="/pubmed/17827064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol.&amp;title=ITPR2+as+a+susceptibility+gene+in+sporadic+amyotrophic+lateral+sclerosis,+a+genome-wide+association+study&amp;author=M.A.+van+Es&amp;author=P.W.+Van+Vught&amp;author=H.M.+Blauw&amp;author=L.+Franke&amp;author=C.G.+Saris&amp;volume=6&amp;publication_year=2007&amp;pages=869-877&amp;pmid=17827064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C9">9. <span class="element-citation">Chiò A., Schymick J.C., Restagno G., Scholz S.W., Lombardo F., Lai S.L., Mora G., Fung H.C., Britton A., Arepalli S., et al.  A two-stage genome-wide association study of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Hum. Mol. Genet. </span>2009;<span class="ref-vol">18</span>:1524–1532.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2664150/">PMC free article</a>]</span> [<a href="/pubmed/19193627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+two-stage+genome-wide+association+study+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=A.+Chiò&amp;author=J.C.+Schymick&amp;author=G.+Restagno&amp;author=S.W.+Scholz&amp;author=F.+Lombardo&amp;volume=18&amp;publication_year=2009&amp;pages=1524-1532&amp;pmid=19193627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C10">10. <span class="element-citation">Dunckley T., Huentelman M.J., Craig D.W., Pearson J.V., Szelinger S., Joshipura K., Halperin R.F., Stamper C., Jensen K.R., Letizia D., et al.  Whole-genome analysis of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">N. Engl. J. Med. 8. </span>2007:775–788.</span> [<a href="/pubmed/17671248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.+8&amp;title=Whole-genome+analysis+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=T.+Dunckley&amp;author=M.J.+Huentelman&amp;author=D.W.+Craig&amp;author=J.V.+Pearson&amp;author=S.+Szelinger&amp;publication_year=2007&amp;pages=775-788&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C11">11. <span class="element-citation">van Es M.A., Veldink J.H., Saris C.G., Blauw H.M., van Vught P.W., Birve A., Lemmens R., Schelhaas H.J., Groen E.J., Huisman M.H., et al.  Genome-wide association study identifies 19p13.3 (UNC13A) and 9p21.2 as susceptibility loci for sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1083–1087.</span> [<a href="/pubmed/19734901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+19p13.3+(UNC13A)+and+9p21.2+as+susceptibility+loci+for+sporadic+amyotrophic+lateral+sclerosis&amp;author=M.A.+van+Es&amp;author=J.H.+Veldink&amp;author=C.G.+Saris&amp;author=H.M.+Blauw&amp;author=P.W.+van+Vught&amp;volume=41&amp;publication_year=2009&amp;pages=1083-1087&amp;pmid=19734901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C12">12. <span class="element-citation">Fogh I., D'Alfonso S., Gellera C., Ratti A., Cereda C., Penco S., Corrado L., Sorarù G., Castellotti B., Tiloca C., et al.  No association of DPP6 with amyotrophic lateral sclerosis in an Italian population. <span><span class="ref-journal">Neurobiol. Aging. </span>2009;<span class="ref-vol">32</span>:966–967.</span> [<a href="/pubmed/19525032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Aging&amp;title=No+association+of+DPP6+with+amyotrophic+lateral+sclerosis+in+an+Italian+population&amp;author=I.+Fogh&amp;author=S.+D'Alfonso&amp;author=C.+Gellera&amp;author=A.+Ratti&amp;author=C.+Cereda&amp;volume=32&amp;publication_year=2009&amp;pages=966-967&amp;pmid=19525032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C13">13. <span class="element-citation">Li X.G., Zhang J.H., Xie M.Q., Liu M.S., Li B.H., Zhao Y.H., Ren H.T., Cui L.Y. Association between DPP6 polymorphism and the risk of sporadic amyotrophic lateral sclerosis in Chinese patients. <span><span class="ref-journal">Chin. Med. J. (Engl) </span>2009;<span class="ref-vol">24</span>:2989–2992.</span> [<a href="/pubmed/20137488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chin.+Med.+J.+(Engl)&amp;title=Association+between+DPP6+polymorphism+and+the+risk+of+sporadic+amyotrophic+lateral+sclerosis+in+Chinese+patients&amp;author=X.G.+Li&amp;author=J.H.+Zhang&amp;author=M.Q.+Xie&amp;author=M.S.+Liu&amp;author=B.H.+Li&amp;volume=24&amp;publication_year=2009&amp;pages=2989-2992&amp;pmid=20137488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C14">14. <span class="element-citation">Fernández-Santiago R., Sharma M., Berg D., Illig T., Anneser J., Meyer T., Ludolph A., Gasser T., et al.  No evidence of association of FLJ10986 and ITPR2 with ALS in a large German cohort. <span><span class="ref-journal">Neurobiol. Aging. </span>2011;<span class="ref-vol">32</span>:551–554.</span> [<a href="/pubmed/19464757" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Aging&amp;title=No+evidence+of+association+of+FLJ10986+and+ITPR2+with+ALS+in+a+large+German+cohort&amp;author=R.+Fernández-Santiago&amp;author=M.+Sharma&amp;author=D.+Berg&amp;author=T.+Illig&amp;author=J.+Anneser&amp;volume=32&amp;publication_year=2011&amp;pages=551-554&amp;pmid=19464757&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C15">15. <span class="element-citation">Shatunov A., Mok K., Newhouse S., Weale M.E., Smith B., Vance C., Johnson L., Veldink J.H., van Es M.A., van den Berg L.H., et al.  Chromosome 9p21 in sporadic amyotrophic lateral sclerosis in the UK and seven other countries, a genome-wide association study. <span><span class="ref-journal">Lancet Neurol. </span>2010;<span class="ref-vol">9</span>:986–994.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3257853/">PMC free article</a>]</span> [<a href="/pubmed/20801717" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol.&amp;title=Chromosome+9p21+in+sporadic+amyotrophic+lateral+sclerosis+in+the+UK+and+seven+other+countries,+a+genome-wide+association+study&amp;author=A.+Shatunov&amp;author=K.+Mok&amp;author=S.+Newhouse&amp;author=M.E.+Weale&amp;author=B.+Smith&amp;volume=9&amp;publication_year=2010&amp;pages=986-994&amp;pmid=20801717&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C16">16. <span class="element-citation">Ahmeti K.B., Ajroud-Driss S., Al-Chalabi A., Andersen P.M., Armstrong J., Birve A., Blauw H.M., Brown R.H., Bruijn L., et al.  ALSGEN Consortium. Age of onset of amyotrophic lateral sclerosis is modulated by a locus on 1p34.1. <span><span class="ref-journal">Neurobiol. Aging. </span>2012;<span class="ref-vol">34</span>:357.e7–357.e19.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3839234/">PMC free article</a>]</span> [<a href="/pubmed/22959728" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Aging&amp;title=Age+of+onset+of+amyotrophic+lateral+sclerosis+is+modulated+by+a+locus+on+1p34.1&amp;author=K.B.+Ahmeti&amp;author=S.+Ajroud-Driss&amp;author=A.+Al-Chalabi&amp;author=P.M.+Andersen&amp;author=J.+Armstrong&amp;volume=34&amp;publication_year=2012&amp;pages=357.e7-357.e19&amp;pmid=22959728&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C17">17. <span class="element-citation">DeJesus-Hernandez M., Mackenzie I.R., Boeve B.F., Boxer A.L., Baker M., Rutherford N.J., Nicholson A.M., Finch N.A., Flynn H., Adamson J., et al.  Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS. <span><span class="ref-journal">Neuron. </span>2011;<span class="ref-vol">72</span>:245–256.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3202986/">PMC free article</a>]</span> [<a href="/pubmed/21944778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Expanded+GGGGCC+hexanucleotide+repeat+in+noncoding+region+of+C9ORF72+causes+chromosome+9p-linked+FTD+and+ALS&amp;author=M.+DeJesus-Hernandez&amp;author=I.R.+Mackenzie&amp;author=B.F.+Boeve&amp;author=A.L.+Boxer&amp;author=M.+Baker&amp;volume=72&amp;publication_year=2011&amp;pages=245-256&amp;pmid=21944778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C18">18. <span class="element-citation">Renton A.E., Majounie E., Waite A., Simón-Sánchez J., Rollinson S., Gibbs J.R., Schymick J.C., Laaksovirta H., van Swieten J.C., Myllykangas L., et al.  A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD. <span><span class="ref-journal">Neuron. </span>2011;<span class="ref-vol">72</span>:257–268.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3200438/">PMC free article</a>]</span> [<a href="/pubmed/21944779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=A+hexanucleotide+repeat+expansion+in+C9ORF72+is+the+cause+of+chromosome+9p21-linked+ALS-FTD&amp;author=A.E.+Renton&amp;author=E.+Majounie&amp;author=A.+Waite&amp;author=J.+Simón-Sánchez&amp;author=S.+Rollinson&amp;volume=72&amp;publication_year=2011&amp;pages=257-268&amp;pmid=21944779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C19">19. <span class="element-citation">Gijselinck I., Van Langenhove T., van der Zee J., Sleegers K., Philtjens S., Kleinberger G., Janssens J., Bettens K., Van Cauwenberghe C., Pereson S., et al.  A C9orf72 promoter repeat expansion in a Flanders-Belgian cohort with disorders of the frontotemporal lobar degeneration-amyotrophic lateral sclerosis spectrum: a gene identification study. <span><span class="ref-journal">Lancet Neurol. </span>2012;<span class="ref-vol">11</span>:54–65.</span> [<a href="/pubmed/22154785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol.&amp;title=A+C9orf72+promoter+repeat+expansion+in+a+Flanders-Belgian+cohort+with+disorders+of+the+frontotemporal+lobar+degeneration-amyotrophic+lateral+sclerosis+spectrum:+a+gene+identification+study&amp;author=I.+Gijselinck&amp;author=T.+Van+Langenhove&amp;author=J.+van+der+Zee&amp;author=K.+Sleegers&amp;author=S.+Philtjens&amp;volume=11&amp;publication_year=2012&amp;pages=54-65&amp;pmid=22154785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C20">20. <span class="element-citation">Ratti A., Corrado L., Castellotti B., Del Bo R., Fogh I., Cereda C., Tiloca C., D'Ascenzo C., Bagarotti A., Pensato V., et al.  C9ORF72 repeat expansion in a large Italian ALS cohort: evidence of a founder effect. <span><span class="ref-journal">Neurobiol. Aging. </span>2012;<span class="ref-vol">33</span>:2528.e7–2528.e14.</span> [<a href="/pubmed/22766072" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Aging&amp;title=C9ORF72+repeat+expansion+in+a+large+Italian+ALS+cohort:+evidence+of+a+founder+effect&amp;author=A.+Ratti&amp;author=L.+Corrado&amp;author=B.+Castellotti&amp;author=R.+Del+Bo&amp;author=I.+Fogh&amp;volume=33&amp;publication_year=2012&amp;pages=2528.e7-2528.e14&amp;pmid=22766072&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C21">21. <span class="element-citation">Al-Chalabi A., Fang F., Hanby M.F., Leigh P.N., Shaw C.E., Ye W., Rijsdijk F. An estimate of amyotrophic lateral sclerosis heritability using twin data. <span><span class="ref-journal">J. Neurol. Neurosurg. Psychiatry. </span>2010;<span class="ref-vol">81</span>:1324–1326.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2988617/">PMC free article</a>]</span> [<a href="/pubmed/20861059" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurol.+Neurosurg.+Psychiatry&amp;title=An+estimate+of+amyotrophic+lateral+sclerosis+heritability+using+twin+data&amp;author=A.+Al-Chalabi&amp;author=F.+Fang&amp;author=M.F.+Hanby&amp;author=P.N.+Leigh&amp;author=C.E.+Shaw&amp;volume=81&amp;publication_year=2010&amp;pages=1324-1326&amp;pmid=20861059&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C22">22. <span class="element-citation">Falconer D.S.  <span class="ref-journal">Introduction to quantitative genetics.</span> Longman Scientific &amp; Technical, Wiley; 1986.  Burnt Mill, Harlow, Essex, England New York. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Introduction+to+quantitative+genetics&amp;author=D.S.+Falconer&amp;publication_year=1986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C23">23. <span class="element-citation">Zaitlen N., Kraft P., Patterson N., Pasaniuc B., Bhatia G., Pollack S., Price A.L. Using extended genealogy to estimate components of heritability for 23 quantitative and dichotomous traits. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">5</span>:e1003520.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3667752/">PMC free article</a>]</span> [<a href="/pubmed/23737753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Using+extended+genealogy+to+estimate+components+of+heritability+for+23+quantitative+and+dichotomous+traits&amp;author=N.+Zaitlen&amp;author=P.+Kraft&amp;author=N.+Patterson&amp;author=B.+Pasaniuc&amp;author=G.+Bhatia&amp;volume=5&amp;publication_year=2013&amp;pages=e1003520&amp;pmid=23737753&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C24">24. <span class="element-citation">Patterson N., Price A.L., Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Population+structure+and+eigenanalysis&amp;author=N.+Patterson&amp;author=A.L.+Price&amp;author=D.+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C25">25. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C26">26. <span class="element-citation">Marchini J., Howie B., Myers S., McVean G., Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J.+Marchini&amp;author=B.+Howie&amp;author=S.+Myers&amp;author=G.+McVean&amp;author=P.+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C27">27. <span class="element-citation">Willer C.J., Li Y., Abecasis G.R. METAL, fast and efficient meta-analysis of genome wide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL,+fast+and+efficient+meta-analysis+of+genome+wide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C28">28. <span class="element-citation">Pirinen M., Donnelly P., Spencer C.C. Including known covariates can reduce power to detect genetic effects in case-control studies. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:848–851.</span> [<a href="/pubmed/22820511" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Including+known+covariates+can+reduce+power+to+detect+genetic+effects+in+case-control+studies&amp;author=M.+Pirinen&amp;author=P.+Donnelly&amp;author=C.C.+Spencer&amp;volume=44&amp;publication_year=2012&amp;pages=848-851&amp;pmid=22820511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C29">29. <span class="element-citation">Yang J., Lee S.H., Goddard M.E., Visscher P.M. GCTA, a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=GCTA,+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C30">30. <span class="element-citation">Osterloh J.M., Yang J., Rooney T.M., Fox A.N., Adalbert R., Powell E.H., Sheehan A.E., Avery M.A., Hackett R., Logan M.A., et al.  dSarm/Sarm1 is required for activation of an injury-induced axon death pathway. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">337</span>:481–484.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5225956/">PMC free article</a>]</span> [<a href="/pubmed/22678360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=dSarm/Sarm1+is+required+for+activation+of+an+injury-induced+axon+death+pathway&amp;author=J.M.+Osterloh&amp;author=J.+Yang&amp;author=T.M.+Rooney&amp;author=A.N.+Fox&amp;author=R.+Adalbert&amp;volume=337&amp;publication_year=2012&amp;pages=481-484&amp;pmid=22678360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C31">31. <span class="element-citation">Yu X.M., Luo L. dSarm-ing axon degeneration. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">337</span>:418–419.</span> [<a href="/pubmed/22837513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=dSarm-ing+axon+degeneration&amp;author=X.M.+Yu&amp;author=L.+Luo&amp;volume=337&amp;publication_year=2012&amp;pages=418-419&amp;pmid=22837513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C32">32. <span class="element-citation">Fischer L.R., Culver D.G., Davis A.A., Tennant P., Wang M., Coleman M., Asress S., Adalbert R., Alexander G.M., Glass J.D., et al.  Wld<sup>S</sup> gene modestly prolongs survival in the SOD1G93A. <span><span class="ref-journal">Neurobiol. Dis. </span>2005;<span class="ref-vol">19</span>:293–300.</span> [<a href="/pubmed/15837585" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Dis.&amp;title=WldS+gene+modestly+prolongs+survival+in+the+SOD1G93A&amp;author=L.R.+Fischer&amp;author=D.G.+Culver&amp;author=A.A.+Davis&amp;author=P.+Tennant&amp;author=M.+Wang&amp;volume=19&amp;publication_year=2005&amp;pages=293-300&amp;pmid=15837585&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C33">33. <span class="element-citation">Coleman M.P., Freeman M.R. Wallerian degeneration, wld(s) and nmnat. <span><span class="ref-journal">Annu. Rev. Neurosci. </span>2010;<span class="ref-vol">33</span>:245–267.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5223592/">PMC free article</a>]</span> [<a href="/pubmed/20345246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Neurosci.&amp;title=Wallerian+degeneration,+wld(s)+and+nmnat&amp;author=M.P.+Coleman&amp;author=M.R.+Freeman&amp;volume=33&amp;publication_year=2010&amp;pages=245-267&amp;pmid=20345246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C34">34. <span class="element-citation">Keller M.F., Saad M., Bras J., Bettella F., Nicolaou N., Simón-Sánchez J., Mittag F., Büchel F., Sharma M., Gibbs J.R., et al.  Using genome-wide complex trait analysis to quantify ‘missing heritability’ in Parkinson's disease. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:4996–5009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3576713/">PMC free article</a>]</span> [<a href="/pubmed/22892372" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Using+genome-wide+complex+trait+analysis+to+quantify+‘missing+heritability’+in+Parkinson's+disease&amp;author=M.F.+Keller&amp;author=M.+Saad&amp;author=J.+Bras&amp;author=F.+Bettella&amp;author=N.+Nicolaou&amp;volume=21&amp;publication_year=2012&amp;pages=4996-5009&amp;pmid=22892372&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C35">35. <span class="element-citation">Lee S.H., Harold D., Nyholt D.R. Goddard M.E., Zondervan K.T., Williams J., Montgomery G.W., Wray N.R., Visscher P.M., editors. ANZGene Consortium, International Endogene Consortium, Genetic and Environmental Risk for Alzheimer's disease Consortium. Estimation and partitioning of polygenic variation captured by common SNPs for Alzheimer's disease, multiple sclerosis and endometriosis. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:832–841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3554206/">PMC free article</a>]</span> [<a href="/pubmed/23193196" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Estimation+and+partitioning+of+polygenic+variation+captured+by+common+SNPs+for+Alzheimer's+disease,+multiple+sclerosis+and+endometriosis&amp;author=S.H.+Lee&amp;author=D.+Harold&amp;author=D.R.+Nyholt&amp;volume=22&amp;publication_year=2013&amp;pages=832-841&amp;pmid=23193196&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C36">36. <span class="element-citation">Simón-Sánchez J., Schulte C., Bras J.M., Sharma M., Gibbs J.R., Berg D., Paisan-Ruiz C., Lichtner P., Scholz S.W., Hernandez D.G., et al.  Genome-wide association study reveals genetic risk underlying Parkinson's disease. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1308–1312.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2787725/">PMC free article</a>]</span> [<a href="/pubmed/19915575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+reveals+genetic+risk+underlying+Parkinson's+disease&amp;author=J.+Simón-Sánchez&amp;author=C.+Schulte&amp;author=J.M.+Bras&amp;author=M.+Sharma&amp;author=J.R.+Gibbs&amp;volume=41&amp;publication_year=2009&amp;pages=1308-1312&amp;pmid=19915575&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C37">37. <span class="element-citation">Lambert J.C., Heath S., Even G., Campion D., Sleegers K., Hiltunen M., Combarros O., Zelenika D., Bullido M.J., Tavernier B., et al.  Genome-wide association study identifies variants at CLU and CR1 associated with Alzheimer's disease. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1094–1099.</span> [<a href="/pubmed/19734903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+variants+at+CLU+and+CR1+associated+with+Alzheimer's+disease&amp;author=J.C.+Lambert&amp;author=S.+Heath&amp;author=G.+Even&amp;author=D.+Campion&amp;author=K.+Sleegers&amp;volume=41&amp;publication_year=2009&amp;pages=1094-1099&amp;pmid=19734903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C38">38. <span class="element-citation">Deng M., Wei L., Zuo X., Tian Y., Xie F., Hu P., Zhu C., Yu F., Meng Y., Wang H., et al.  Genome-wide association analyses in Han Chinese identify two new susceptibility loci for amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:697–700.</span> [<a href="/pubmed/23624525" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analyses+in+Han+Chinese+identify+two+new+susceptibility+loci+for+amyotrophic+lateral+sclerosis&amp;author=M.+Deng&amp;author=L.+Wei&amp;author=X.+Zuo&amp;author=Y.+Tian&amp;author=F.+Xie&amp;volume=45&amp;publication_year=2013&amp;pages=697-700&amp;pmid=23624525&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C39">39. <span class="element-citation">Iida A., Takahashi A., Deng M., Zhang Y., Wang J., Atsuta N., Tanaka F., Kamei T., Sano M., Oshima S., et al.  Replication analysis of SNPs on 9p21.2 and 19p13.3 with amyotrophic lateral sclerosis in East Asians. <span><span class="ref-journal">Neurobiol. Aging. </span>2011;<span class="ref-vol">32</span>:713–754.</span> [<a href="/pubmed/21295378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Aging&amp;title=Replication+analysis+of+SNPs+on+9p21.2+and+19p13.3+with+amyotrophic+lateral+sclerosis+in+East+Asians&amp;author=A.+Iida&amp;author=A.+Takahashi&amp;author=M.+Deng&amp;author=Y.+Zhang&amp;author=J.+Wang&amp;volume=32&amp;publication_year=2011&amp;pages=713-754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C40">40. <span class="element-citation">Ogaki K., Li Y., Atsuta N., Tomiyama H., Funayama M., Watanabe H., Nakamura R., Yoshino H., Yato S., Tamura A., et al.  Analysis of <em>C9orf72</em> repeat expansion in 563 Japanese patients with amyotrophic lateral sclerosis. <span><span class="ref-journal">Neurobiol. Aging. </span>2012;<span class="ref-vol">33</span>:2527.e11–2527.e16.</span> [<a href="/pubmed/22727276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Aging&amp;title=Analysis+of+C9orf72+repeat+expansion+in+563+Japanese+patients+with+amyotrophic+lateral+sclerosis&amp;author=K.+Ogaki&amp;author=Y.+Li&amp;author=N.+Atsuta&amp;author=H.+Tomiyama&amp;author=M.+Funayama&amp;volume=33&amp;publication_year=2012&amp;pages=2527.e11-2527.e16&amp;pmid=22727276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C41">41. <span class="element-citation">Miller R.G., Munsat T.L., Swash M., Brooks B.R. Consensus guidelines for the design and implementation of clinical trials in ALS. World Federation of Neurology committee on Research. <span><span class="ref-journal">J. Neurol. Sci. </span>1999;<span class="ref-vol">169</span>:2–12.</span> [<a href="/pubmed/10540001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurol.+Sci.&amp;title=Consensus+guidelines+for+the+design+and+implementation+of+clinical+trials+in+ALS.+World+Federation+of+Neurology+committee+on+Research&amp;author=R.G.+Miller&amp;author=T.L.+Munsat&amp;author=M.+Swash&amp;author=B.R.+Brooks&amp;volume=169&amp;publication_year=1999&amp;pages=2-12&amp;pmid=10540001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C42">42. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK, a toolset for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK,+a+toolset+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C43">43. <span class="element-citation">Howie B.N., Donnelly P., Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=B.N.+Howie&amp;author=P.+Donnelly&amp;author=J.+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C44">44. <span class="element-citation">Marchini J., Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat. Rev. Genet. </span>2010;<span class="ref-vol">11</span>:499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J.+Marchini&amp;author=B.+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C45">45. <span class="element-citation">Higgins J.P., Thompson S.G., Deeks J.J., Altman D.G. Measuring inconsistency in meta-analyses. <span><span class="ref-journal">Br. Med. J. </span>2003;<span class="ref-vol">327</span>:557–560.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC192859/">PMC free article</a>]</span> [<a href="/pubmed/12958120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+Med.+J.&amp;title=Measuring+inconsistency+in+meta-analyses&amp;author=J.P.+Higgins&amp;author=S.G.+Thompson&amp;author=J.J.+Deeks&amp;author=D.G.+Altman&amp;volume=327&amp;publication_year=2003&amp;pages=557-560&amp;pmid=12958120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C46">46. <span class="element-citation">Ioannidis J.P., Patsopoulos N.A., Evangelou E. Heterogeneity in meta-analyses of genome-wide association investigations. <span><span class="ref-journal">PLoS One. </span>2007;<span class="ref-vol">2</span>:e841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950790/">PMC free article</a>]</span> [<a href="/pubmed/17786212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Heterogeneity+in+meta-analyses+of+genome-wide+association+investigations&amp;author=J.P.+Ioannidis&amp;author=N.A.+Patsopoulos&amp;author=E.+Evangelou&amp;volume=2&amp;publication_year=2007&amp;pages=e841&amp;pmid=17786212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT587C47">47. <span class="element-citation">Abel O., Powell J.F., Andersen P.M., Al-Chalabi A. ALSoD: a user-friendly online bioinformatics tool for amyotrophic lateral sclerosis genetics. <span><span class="ref-journal">Hum. Mutat. </span>2012;<span class="ref-vol">9</span>:1345–1351.</span> [<a href="/pubmed/22753137" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=ALSoD:+a+user-friendly+online+bioinformatics+tool+for+amyotrophic+lateral+sclerosis+genetics&amp;author=O.+Abel&amp;author=J.F.+Powell&amp;author=P.M.+Andersen&amp;author=A.+Al-Chalabi&amp;volume=9&amp;publication_year=2012&amp;pages=1345-1351&amp;pmid=22753137&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3959809&amp;issue-id=235296&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140459749532176title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">DATA AVAILABILITY</a></li><li><a href="#s7title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140459749895168title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140459784275504title">APPENDIX</a></li><li><a href="#idm140459749892080title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3959809/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3959809/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3959809/pdf/ddt587.pdf">PDF (426K)</a></li> | <li><a href="#" data-citationid="PMC3959809" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3959809%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3959809%2F&amp;text=A%20genome-wide%20association%20meta-analysis%20identifies%20a%20novel%20locus%20at%2017q11.2%20associated%20with%20sporadic%20amyotrophic%20lateral%20sclerosis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3959809%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24256812" data-aiid="3959809" data-aid="3959809" data-iid="235296" data-domainid="740" data-domain="hmg" data-accid="PMC3959809" data-md5="d433e778dfa51ec015c1607a42828bec"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T11:35:11-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BB229DD561D71000000000723043E&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3959809%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3959809/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>