<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3740040/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of preeclampsia detects novel maternal single nucleotide polymorphisms and copy-number variants in subsets of the Hyperglycemia and Adverse Pregnancy Outcome (HAPO) study cohort</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3740040/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Annals of human genetics"><meta name="citation_title" content="Genome-wide association study of preeclampsia detects novel maternal single nucleotide polymorphisms and copy-number variants in subsets of the Hyperglycemia and Adverse Pregnancy Outcome (HAPO) study cohort"><meta name="citation_authors" content="Linlu Zhao, Michael B. Bracken, Andrew T. DeWan"><meta name="citation_date" content="July 2013"><meta name="citation_issue" content="4"><meta name="citation_volume" content="77"><meta name="citation_firstpage" content="277"><meta name="citation_doi" content="10.1111/ahg.12021"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3740040/?report=abstract"><meta name="citation_pmid" content="23551011"><meta name="DC.Title" content="Genome-wide association study of preeclampsia detects novel maternal single nucleotide polymorphisms and copy-number variants in subsets of the Hyperglycemia and Adverse Pregnancy Outcome (HAPO) study cohort"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Linlu Zhao"><meta name="DC.Contributor" content="Michael B. Bracken"><meta name="DC.Contributor" content="Andrew T. DeWan"><meta name="DC.Date" content="2013 Jul"><meta name="DC.Identifier" content="10.1111/ahg.12021"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of preeclampsia detects novel maternal single nucleotide polymorphisms and copy-number variants in subsets of the Hyperglycemia and Adverse Pregnancy Outcome (HAPO) study cohort"><meta property="og:type" content="article"><meta property="og:description" content="A genome-wide association study was undertaken to identify maternal single nucleotide polymorphisms (SNPs) and copy-number variants (CNVs) associated with preeclampsia. Case-control analysis was performed on 1070 Afro-Caribbean (n=21 cases and 1049 controls) ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3740040/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3740040/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3740040/pdf/nihms448601.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C53F1DD55C0C10000000001930107.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3740040%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="mesh">MeSH</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3740040</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Ann Hum Genet</a></span>. Author manuscript; available in PMC 2014 Jul 30.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23551011" target="pmc_ext" ref="reftype=publisher&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Ann Hum Genet. 2013 Jul; 77(4): 277–287. </span></a></div><span class="fm-vol-iss-date">Published online 2013 Apr 2. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1111%2Fahg.12021" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1111/ahg.12021</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3740040</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS448601</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23551011">23551011</a></div></div></div></div><h1 class="content-title">Genome-wide association study of preeclampsia detects novel maternal single nucleotide polymorphisms and copy-number variants in subsets of the Hyperglycemia and Adverse Pregnancy Outcome (HAPO) study cohort</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Zhao%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23551011" class="affpopup" co-rid="_co_idm139807749914464" co-class="co-affbox">Linlu Zhao</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Bracken%20MB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23551011" class="affpopup" co-rid="_co_idm139807750306480" co-class="co-affbox">Michael B. Bracken</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> and  <a href="/pubmed/?term=DeWan%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23551011" class="affpopup" co-rid="_co_idm139807756752688" co-class="co-affbox">Andrew T. DeWan</a><sup>1,</sup><sup>2,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139807749914464"><h3 class="no_margin">Linlu Zhao</h3><p><sup>1</sup>Center for Perinatal, Pediatric and Environmental Epidemiology, Yale Schools of Public Health and Medicine, New Haven, Connecticut, USA</p><p><sup>2</sup>Department of Chronic Disease Epidemiology, Yale School of Public Health, New Haven, Connecticut, USA</p><div>Find articles by <a href="/pubmed/?term=Zhao%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23551011">Linlu Zhao</a></div></div><div id="_co_idm139807750306480"><h3 class="no_margin">Michael B. Bracken</h3><p><sup>1</sup>Center for Perinatal, Pediatric and Environmental Epidemiology, Yale Schools of Public Health and Medicine, New Haven, Connecticut, USA</p><p><sup>2</sup>Department of Chronic Disease Epidemiology, Yale School of Public Health, New Haven, Connecticut, USA</p><p><sup>3</sup>Department of Obstetrics, Gynecology, and Reproductive Sciences, Yale School of Medicine, New Haven, Connecticut, USA</p><div>Find articles by <a href="/pubmed/?term=Bracken%20MB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23551011">Michael B. Bracken</a></div></div><div id="_co_idm139807756752688"><h3 class="no_margin">Andrew T. DeWan</h3><p><sup>1</sup>Center for Perinatal, Pediatric and Environmental Epidemiology, Yale Schools of Public Health and Medicine, New Haven, Connecticut, USA</p><p><sup>2</sup>Department of Chronic Disease Epidemiology, Yale School of Public Health, New Haven, Connecticut, USA</p><div>Find articles by <a href="/pubmed/?term=DeWan%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23551011">Andrew T. DeWan</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139807759190304_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139807759190304_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139807759190304_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Center for Perinatal, Pediatric and Environmental Epidemiology, Yale Schools of Public Health and Medicine, New Haven, Connecticut, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Chronic Disease Epidemiology, Yale School of Public Health, New Haven, Connecticut, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Obstetrics, Gynecology, and Reproductive Sciences, Yale School of Medicine, New Haven, Connecticut, USA</div><div id="FN1"><sup>*</sup>Corresponding author: Andrew T. DeWan, Yale School of Public Health, Department of Chronic Disease Epidemiology, 60 College Street, Room 403, New Haven, CT 06520, USA; phone: 203-785-3528; fax: 203-785-6279; <a href="mailto:dev@null" data-email="ude.elay@nawed.werdna" class="oemail">ude.elay@nawed.werdna</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139807759190304_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23551011" target="pmc_ext" ref="reftype=publisher&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Ann Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3740040/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp F ig S1-S5: </strong><strong>Figure S1.</strong> EIGENSTRAT analysis of population stratification for Afro-Caribbean HAPO study subjects.<p><strong>Description:</strong> The top scatter plot shows the eighth and ninth eigenvectors of genotypic variance (PC8 vs. PC9) for Afro-Caribbean HAPO study subjects. The bottom scatter plot shows the ninth and tenth eigenvectors (PC9 vs. PC10). The ninth eigenvector is marginally significant (<em>p</em>=0.029).</p><p><strong>Figure S2.</strong> EIGENSTRAT analysis of population stratification for HAPO study subjects of European ancestry.</p><p><strong>Description:</strong> The top scatter plot shows the first two eigenvectors of genotypic variance (PC1 vs. PC2) for HAPO study subjects of European ancestry. The bottom scatter plot shows the second and third eigenvectors (PC2 vs. PC3). The first eigenvector is significant (<em>p</em>=0.004).</p><p><strong>Figure S3.</strong> LocusZoom plot of rs11617740 and ±500 kb flanking region SNP associations in Afro-Caribbean HAPO subjects</p><p><strong>Description:</strong> The main panel shows logistic regression <em>p</em>-values on the -log10 scale on the vertical axis and chromosomal position on the horizontal axis based on the NCBI36/hg18 genome assembly. The bottom panel shows positions of genes in the region. Colors of plotted SNP <em>p</em>-values represent linkage disequilibrium r<sup>2</sup> values of each SNP with the most significant SNP, rs11617740 (<em>p</em>=7.32×10<sup>−7</sup>), denoted by a diamond on the plot, calculated using HapMap 2 CEU data.</p><p><strong>Figure S4.</strong> LocusZoom plot of rs7322722 and ±500 kb flanking region SNP associations in European ancestry HAPO subjects</p><p><strong>Description:</strong> The main panel shows logistic regression <em>p</em>-values on the -log10 scale on the vertical axis and chromosomal position on the horizontal axis based on the NCBI36/hg18 genome assembly. The bottom panel shows positions of genes in the region. Colors of plotted SNP <em>p</em>-values represent linkage disequilibrium r<sup>2</sup> values of each SNP with the most significant SNP, rs7322722 (<em>p</em>=1.23×10<sup>−6</sup>), denoted by a diamond on the plot, calculated using HapMap 2 CEU data.</p><p><strong>Figure S5.</strong> LocusZoom plot of rs17412740 and ±500 kb flanking region SNP associations in Hispanic HAPO subjects</p><p><strong>Description:</strong> The main panel shows logistic regression <em>p</em>-values on the -log10 scale on the vertical axis and chromosomal position on the horizontal axis based on the NCBI36/hg18 genome assembly. The bottom panel shows positions of genes in the region. Colors of plotted SNP <em>p</em>-values represent linkage disequilibrium r<sup>2</sup> values of each SNP with the most significant SNP, rs17412740 (<em>p</em>=2.14×10<sup>−6</sup>), denoted by a diamond on the plot, calculated using HapMap 2 CEU data.</p></div><div><a href="/pmc/articles/PMC3740040/bin/NIHMS448601-supplement-Supp_F_ig_S1-S5.pdf" data-ga-action="click_feat_suppl">NIHMS448601-supplement-Supp_F_ig_S1-S5.pdf</a><span style="color:gray"> (225K)</span></div><div class="small guid">GUID:&nbsp;1EA58B45-15A2-4FDC-A669-CA21DAD53C99</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table 01: </strong>Table S1 Association of the top SNP candidates (logistic regression <em>p</em>-value &lt;10<sup>−5</sup>) with PE for each ethnic group of HAPO study subjects and Caucasian SOPHIA study subjects<p><strong>Description:</strong> Top SNP candidates associated with PE ranked by logistic regression <em>p</em>-values are presented in this table for Afro-Caribbean, European ancestry and Hispanic HAPO study subjects and Caucasian SOPHIA study subjects. The logistic regression models for the Afro-Caribbean and European ancestry HAPO subjects were adjusted with PC9 and PC1, respectively. Cochrane-Armitage trend tests were performed in the comparison groups to attempt replication of top SNP findings in the study group of interest. Ungenotyped SNPs in the comparison groups were imputed and tested (trend test) in PLINK.</p></div><div><a href="/pmc/articles/PMC3740040/bin/NIHMS448601-supplement-Supp_Table_01.docx" data-ga-action="click_feat_suppl">NIHMS448601-supplement-Supp_Table_01.docx</a><span style="color:gray"> (23K)</span></div><div class="small guid">GUID:&nbsp;3471A0A4-8C72-4D78-8755-65F320390CE4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table 02: </strong>Table S2 Comparison of call frequencies and differences in probe coverage for the top regions of copy-number deletion identified from the SOPHIA study with HAPO study controls of European ancestry<p><strong>Description:</strong> CNV frequency and microarray probe coverage differences between Caucasian female controls from the SOPHIA study (n=114 with an additional 770 population-based controls) and female controls of European descent from the HAPO study (n=1152). Affymetrix SNP 6.0 microarray was used for whole-genome genotyping in the SOPHIA study whereas Illumina Human610 microarray was used to genotype women of European ancestry in the HAPO study.</p></div><div><a href="/pmc/articles/PMC3740040/bin/NIHMS448601-supplement-Supp_Table_02.doc" data-ga-action="click_feat_suppl">NIHMS448601-supplement-Supp_Table_02.doc</a><span style="color:gray"> (49K)</span></div><div class="small guid">GUID:&nbsp;894CE96A-2CDC-48E0-A078-C0903C80B294</div></div></dd></dl></div><div id="idm139807744694592" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139807744694592title">Summary</h2><!--article-meta--><div><p id="P2" class="p p-first-last">A genome-wide association study was undertaken to identify maternal single nucleotide polymorphisms (SNPs) and copy-number variants (CNVs) associated with preeclampsia. Case-control analysis was performed on 1070 Afro-Caribbean (n=21 cases and 1049 controls) and 723 Hispanic (n=62 cases and 661 controls) mothers and 1257 mothers of European ancestry (n=50 cases and 1207 controls) from the Hyperglycemia and Adverse Pregnancy Outcome (HAPO) study. European ancestry subjects were genotyped on Illumina Human610-Quad and Afro-Caribbean and Hispanic subjects were genotyped on Illumina Human1M-Duo BeadChip microarrays. Genome-wide SNP data were analyzed using PLINK. CNVs were called using three detection algorithms (GNOSIS, PennCNV, and QuantiSNP), merged using CNVision, and then screened using stringent criteria. SNP and CNV findings were compared to those of the Study of Pregnancy Hypertension in Iowa (SOPHIA), an independent preeclampsia case-control dataset of Caucasian mothers (n=177 cases and 116 controls). A list of top SNPs were identified for each of the HAPO ethnic groups, but none reached Bonferroni-corrected significance. Novel candidate CNVs showing enrichment among preeclampsia cases were also identified in each of the three ethnic groups. Several variants were suggestively replicated in SOPHIA. The discovered SNPs and copy-number variable regions present interesting candidate genetic variants for preeclampsia that warrant further replication and investigation.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">copy-number variant, genome-wide association study, microarray analysis, preeclampsia, single nucleotide polymorphism</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P3" class="p p-first">Preeclampsia (PE) is a multi-system, hypertensive disorder of pregnancy that complicates 2–7% of pregnancies (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492153">Sibai, et al. 2005</a>). Characterized by new onset hypertension and proteinuria, PE is a major contributor to maternal morbidity and mortality worldwide. Maternal ethnic differences in PE rates exist: in one study, compared to Caucasians, Afro-American women had higher risk of PE (odds ratio=1.41, 95% confidence interval: 1.25–1.62), while Asian (OR=0.79, 95% CI: 0.72–0.88) and Hispanic (OR=0.90, 95% CI: 0.84–0.97) women had lower risk (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492154">Caughey et al. 2005</a>).</p><p id="P4">Although evidence from family-based studies suggests PE has a heritable component, its exact etiology and specific genetic contributions remain unknown (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492174">Arngrimsson et al. 1990</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492156">Lie et al. 1998</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492164">Mogren et al. 1999</a>, <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492169">Sutherland et al. 1981</a>). Susceptibility to PE is thought to be influenced by complex interactions between maternal and fetal genetic and environmental factors (<a href="#R23" rid="R23" class=" bibr popnode">Williamson, 2005</a>). <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492173">Cnattingius et al. (2004)</a> found that 35% of the variance in PE was attributable to maternal genetic effects, 20% to fetal genetic effects, 13% to the “couple effect,” and 32% to unmeasured environmental factors.</p><p id="P5">Many candidate gene studies have been conducted in attempt to elucidate genetic contributions to PE, but no single gene has been identified with consistent replication (<a href="#R23" rid="R23" class=" bibr popnode">Williamson, 2005</a>). The lack of success with the candidate gene approach may be due to inadequate statistical power and poor study design (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492161">Colhoun et al. 2003</a>). In addition, studies examining the genetic factors contributing to PE have focused on single nucleotide polymorphisms (SNPs), which explain only a fraction of overall heritability. An estimated 13% of the genome has been reported to be copy-number variable in one or more individuals, suggesting that copy-number variants (CNVs) may be substantial contributors to the “missing heritability” (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492158">Stankiewicz &amp; Lupski. 2010</a>). Prior genome-wide CNV analysis from the Study of Pregnancy Hypertension in Iowa (SOPHIA) conducted by the authors and colleagues discovered three rare deletions that may confer risk for PE, including a potentially functionally important copy-number deletion in <em>PSG11</em> (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492152">Zhao et al. 2012</a>).</p><p id="P6" class="p p-last">A genome-wide association (GWA) study using a case-control design was conducted to identify PE-associated SNPs and CNVs in Afro-Caribbean, European ancestry, and Hispanic ethnic groups using publicly available genotype data from the Hyperglycemia and Adverse Pregnancy Outcome (HAPO) study (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492170">HAPO Study Cooperative Research Group, 2002</a>). The present study was also compared to the SOPHIA study to attempt corroboration of findings.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">MATERIALS AND METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Study population</h3><p id="P7" class="p p-first">The HAPO study is an international, multi-center prospective cohort study of maternal glycemia and adverse pregnancy outcomes. Detailed recruitment methods have been described elsewhere (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492175">HAPO Study Cooperative Research Group, 2002</a>). Approximately 4750 Afro-Caribbean (Barbados), European ancestry (Brisbane and Newcastle, Australia; Toronto, Canada; and Belfast, United Kingdom), Hispanic (Bellflower, CA), and Thai (Bangkok, Thailand) mothers were genotyped.</p><p id="P8" class="p p-last">A case-control design was applied to each ethnic group in the present study with subjects drawn from the HAPO study with both phenotype and genotype information available (Afro-Caribbean, n=23 cases and 1075 controls; European ancestry, n=52 cases and 1250 controls; and Hispanic, n=62 cases and 661 controls). Genotype and phenotype information were acquired from the National Center for Biotechnology Information’s Genotypes and Phenotypes Database (NCBI dbGaP study accession: phs000096.v4.p1) (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492177">HAPO Study Cooperative Research Group, 2002</a>). Thai mothers were not further considered for analysis due to a very limited number of PE cases (n=9). Mothers presenting with PE were eligible as cases and normotensive mothers were eligible as controls. Subjects with chronic hypertension, gestational hypertension, or PE superimposed on chronic hypertension as well as those with unavailable details on hypertension status were excluded.</p></div><div id="S4" class="sec"><h3 id="S4title">Phenotype definition</h3><p id="P9" class="p p-first-last">PE was defined according to the International Society for the Study of Hypertension in Pregnancy guidelines (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492172">Brown. et al. 2001</a>) as having <em>de novo</em> hypertension (systolic blood pressure ≥140 mmHg and/or diastolic blood pressure ≥90 mmHg on two or more occasions at a minimum of six hours apart) and proteinuria (dipstick protein test value ≥1+ or a 24-hour urine collection with protein ≥300 mg) after 20 weeks of gestation.</p></div><div id="S5" class="sec"><h3 id="S5title">Whole-genome genotyping</h3><p id="P10" class="p p-first">Genomic DNA was isolated from peripheral blood samples using Autopure LS (Gentra Systems, Minneapolis, MN) (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492176">Urbanek. et al. 2012</a>). Genotyping was performed at the Broad Institute of MIT and Harvard using Illumina Human610-Quad (European ancestry) and Illumina Human1M-Duo (Afro-Caribbean and Hispanic) BeadChip microarray platforms (Illumina, San Diego, CA), interrogating over 601,000 and 1.1 million SNP and copy-number (non-polymorphic) probes, respectively.</p><p id="P11" class="p p-last">Sample quality was assessed by signal intensity using R and overall call rate using PLINK version 1.07 (<a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a>) (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492160">Purcell et al. 2007</a>). Subjects were excluded if low signal intensity was exhibited across all chromosomes. When examining the mean chromosomal probe intensities, these excluded samples formed an outlying low-intensity “tail” from the main cluster, a pattern consistent throughout the autosomes and X chromosome. No additional samples were excluded due to having overall call rate lower than 90% (default value in PLINK), with Afro-Caribbean, European ancestry, and Hispanic subjects having mean call rates of 99.9%, 95.4%, and 99.6% respectively. A total of 1070 Afro-Caribbean (n=21 cases and 1049 controls), 1257 European ancestry (n=50 cases and 1207 controls), and 723 Hispanic (n=62 cases and 661 controls) mothers met eligibility criteria for further analysis.</p></div><div id="S6" class="sec"><h3 id="S6title">SNP analysis</h3><p id="P12" class="p p-first">Genome-wide SNP analysis was performed using PLINK at default settings. A total of 1,145,510 SNPs were genotyped for both Afro-Caribbean and Hispanic study subjects on Illumina Human1M microarrays and 598,821 SNPs for European ancestry subjects on Illumina Human610 microarrays. SNPs with minor allele frequency (MAF) &lt;0.01 (Afro-Caribbean, n=160,642; European ancestry, n=55,303; and Hispanic, n=141,199), call rate &lt;95% (Afro-Caribbean, n=84,828; European ancestry, n=28,141; and Hispanic, n=54,248), or significant deviation from Hardy-Weinberg equilibrium at <em>p</em>-value ≤0.001 (Afro-Caribbean, n=4616; European ancestry, n=1529; and Hispanic, n=4390) were excluded, leaving 979,693 (Afro-Caribbean), 541,023 (European ancestry), and 964,533 (Hispanic) SNPs.</p><p id="P13" class="p p-last">Genetic quality control included assessments for both population stratification and cryptic relatedness. Population stratification was assessed using EIGENSTRAT version 3.0 (<a href="http://genepath.med.harvard.edu/~reich/Software.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genepath.med.harvard.edu/~reich/Software.htm</a>) by performing a principal components analysis (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492163">Price et al. 2006</a>). Cryptic relatedness between subjects in each ethnic group was examined by pairwise identity-by-descent (IBD) estimation in PLINK. Pairs showing pi-hat (estimated proportion of genomic variation shared IBD) &gt;0.2 were inspected and 39, 5 and 3 controls were excluded from downstream SNP and CNV analyses in Afro-Caribbean, European ancestry and Hispanic groups, respectively. Logistic regression analysis was performed to test the association between individual SNPs and PE with adjustment for significant principal components identified from the EIGENSTRAT analysis. Unadjusted genotype association testing was also performed using the Cochrane-Armitage trend test to facilitate comparisons of <em>p</em>-values of genotyped and ungenotyped SNPs among the HAPO study groups and the SOPHIA study. Ungenotyped SNPs in Afro-Caribbean subjects, subjects of European descent from HAPO and Caucasian subjects from SOPHIA, and Hispanic subjects were imputed and tested in PLINK using HapMap 3 female founders of African ancestry in Southwest USA (ASW), Northern and Western European ancestry from Utah (CEU), and Mexican ancestry in Los Angeles, CA (MEX), respectively (<a href="http://www.broadinstitute.org/~debakker/p3.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/~debakker/p3.html</a>). Bonferroni-corrected genome-wide significance thresholds were applied to each ethnic group to maintain an overall study α of 0.05. As Bonferroni correction is conservative, only tested markers were included in the calculation of the corrected significance thresholds; therefore, the significance thresholds used were 5.10×10<sup>−8</sup> (0.05/979,693) for Afro-Caribbean subjects, 9.24×10<sup>−8</sup> (0.05/541,023) for subjects of European ancestry, and 5.18×10<sup>−8</sup> (0.05/964,533) for Hispanic subjects. Mitochondrial SNPs were excluded from analysis.</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">CNV analysis</h3><p id="P14" class="p p-first">CNVision (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492166">Sanders et al. 2011</a>), a pipeline specifically designed for use with Illumina microarray data, was used to predict and analyze CNVs using three algorithms with two independent approaches: GNOSIS (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492151">Sanders et al. 2011</a>), PennCNV June 16, 2011 version (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492155">Wang et al. 2007</a>), and QuantiSNP version 1.1 (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492150">Colella et al. 2007</a>). GNOSIS is a distribution function algorithm that uses sliding windows to detect CNVs. In contrast, PennCNV and QuantiSNP both implement a hidden Markov model. However, all three detection algorithms utilize log R ratio (LRR; a measure of total signal intensity of probes) and B allele frequency (BAF; a measure of relative intensity ratio of allelic probes) information to detect CNVs. Raw data were batch-processed by ethnic group and detection algorithms were run under microarray-specific default CNVision pipeline settings for autosomal and X chromosomes.</p><p id="P15">Stringent sample and call quality control (QC) measures were undertaken in attempt to increase the accuracy of CNV prediction. Samples with log-transformed number of CNV calls greater than three standard deviations from the mean of the log-transformed calls per sample for at least one of the three algorithms were considered of low quality and removed from further analysis (Afro-Caribbean, n=50 controls; European ancestry, n=50 controls; Hispanic, n=2 cases and 13 controls). Related subjects identified by pairwise IBD were also removed (Afro-Caribbean, n=37 controls; European ancestry, n=5 controls; Hispanic, n=3 controls). Raw CNV calls spanning less than ten microarray probes were excluded. CNVision was then used to merge, analyze, and annotate the QC filtered outputs of GNOSIS, PennCNV, and QuantiSNP. A merged CNV call meeting one or more of the following screening criteria was excluded: 1) &lt;50% overlap between two algorithms and &lt;25% overlap among three algorithms; 2) &lt;10 consecutive microarray probes; or 3) contains conflicting calls (both deletion and amplification) across algorithms within the same genomic region in the given sample. Finally, the following selection criteria were applied to minimal regions of overlap of merged CNV calls across samples to generate a shortlist of candidate regions: 1) CNVs called in ≥3 cases and absence of calls in controls or, if present in controls, having positive ORs comparing PE cases versus normotensive controls and corresponding 95% CIs that excluded the null; 2) does not overlap centromeric or telomeric regions according to PennCNV definitions; and 3) contains ≥10 consecutive SNP or copy-number probes. <a href="/pmc/articles/PMC3740040/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> illustrates the boundaries of a minimal region of overlap of merged CNV calls across samples for a deletion enriched in HAPO subjects of European ancestry.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3740040/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139807747637872" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms448601f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms448601f1.jpg" src="/pmc/articles/PMC3740040/bin/nihms448601f1.jpg"></div></a><div id="largeobj_idm139807747637872" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3740040/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3740040/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3740040/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>UCSC Genome Browser plot of the copy-number deletion at chr9:1.490–1.503 Mb identified in HAPO subjects of European ancestry</strong><p id="__p4">The vertical black lines indicate the minimal region of overlap of merged calls across all subjects harboring the deletion. Each red horizontal bar represents the length and breakpoints of a merged deletion call detected in either PE cases or controls in HAPO subjects of European ancestry. Exact CNV breakpoints are unknown.</p></div></div></div><p id="P16">Reliability of CNVs detected <em>in silico</em> was assessed by visual inspection of LRR and BAF plots for a clear change in probe hybridization intensity and zygosity, respectively. Fisher’s exact tests were performed to quantify the significance of the difference in CNV frequency. Top SNPs and CNVs identified in the HAPO study were compared to the SOPHIA study, an independent dataset of primiparous Caucasian women, for replication of findings. The SOPHIA study genotyped 177 PE cases and 116 normotensive controls using the Affymetrix Genome-Wide Human SNP Array 6.0 (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492168">Zhao et al. 2012</a>). Bonferroni-corrected significance threshold in the SOPHIA study for the SNP analysis was defined as 7.1×10<sup>−8</sup>. SNPs and CNVs identified in HAPO reaching nominal significance in SOPHIA (<em>p</em>-value&lt;0.05) were considered to be suggestively replicated.</p><p id="P17" class="p p-last">Genomic positions are presented according to the NCBI36/hg18 human genome assembly.</p></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">RESULTS</h2><div id="S9" class="sec sec-first"><h3 id="S9title">SNP associations</h3><p id="P18" class="p p-first">SNP analysis was performed on 1031 Afro-Caribbean (n=21 cases and 1010 controls), 1252 European ancestry (n=50 cases and 1202 controls), and 720 Hispanic (n=62 cases and 658 controls) subjects. EIGENSTRAT analysis found evidence of population stratification in Afro-Caribbean and European ancestry study subjects; the ninth eigenvector was significant for Afro-Caribbean subjects (principal component 9 or PC9, <em>p</em>=0.029; see <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1</a> for scatter plots of PC8 vs. PC9 and PC9 vs. PC10) and the first eigenvector was significant for subjects of European descent (PC1, <em>p</em>=0.004; see <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a> for a scatter plot of PC1 vs. PC2). PC1 and PC9 were adjusted in the logistic regression models for European ancestry and Afro-Caribbean subjects, respectively. The top 10 principal components for Hispanic subjects were not significant.</p><p id="P19" class="p p-last">After Bonferroni correction for multiple testing, no SNPs were found to be significantly associated with PE in any ethnic group. The most significant SNP identified in Afro-Caribbean subjects adjusted for PC9 was rs11617740 in <em>FGF14</em> (OR=16.96, 95% CI: 5.53–51.97, <em>p</em>=7.32×10<sup>−7</sup>) and in European ancestry subjects adjusted for PC1 was rs7322722 in <em>MYCBP2</em> (OR=2.93, 95% CI: 1.90–4.52, <em>p</em>=1.23×10<sup>−6</sup>). For Hispanic subjects, the most significant SNP was rs17412740 located in an intergenic region near <em>LZTS1</em> with a <em>p</em>-value of 2.14×10<sup>−6</sup> and an OR of 6.08 (95% CI: 2.88–12.81). Top SNP candidates (logistic regression <em>p</em>-value &lt;10<sup>−5</sup>) for the three HAPO ethnic groups are presented in <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>. LocusZoom (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492171">Pruim= et al. 2010</a>) plots visualizing association results for the most significant SNP in each of the three HAPO ethnic groups and its ±500 kb flanking region SNPs are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Figures S3-S5</a>. The chromosome 13 region of European ancestry subjects contains a set of SNPs with small <em>p</em>-values (&lt;10<sup>−5</sup>) and relatively high degrees of association (r<sup>2</sup>&gt;0.8) to rs7322722 within and proximal to the <em>MYCBP2</em> gene (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S4</a>).</p></div><div id="S10" class="sec"><h3 id="S10title">CNV associations</h3><div id="S11" class="sec sec-first"><p></p><h4 id="S11title" class="inline">Afro-Caribbean subjects </h4><p id="P20" class="p p-first-last">Among Afro-Caribbean mothers, 23,962 putative CNVs (15,774 deletions and 8188 amplifications) spanning ten or more SNP or copy-number probes were detected among the 21 PE cases and 962 normotensive controls that passed sample QC. CNVs ranging in size from ~0.8 kb to 21.5 Mb (median=49.0 kb) were merged into 7739 regions of deletion and 4630 regions of amplification. Of these, nine minimal regions of overlap of recurrent CNVs met the selection criteria (see <a href="/pmc/articles/PMC3740040/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> for an annotated list of candidate CNVs among Afro-Caribbean mothers). Five of the candidate CNVs achieved nominal significance (<em>p</em>-value&lt;0.05).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">CNVs identified among Afro-Caribbean HAPO study subjects<sup>1</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139807756457616" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">CNV region<br>(chromosome:<br>start position-<br>stop position)</th><th rowspan="1" colspan="1" valign="bottom" align="left">CNV type</th><th rowspan="1" colspan="1" valign="bottom" align="left">Region<br>size (bp)</th><th rowspan="1" colspan="1" valign="bottom" align="left"># probes<br>within<br>region</th><th rowspan="1" colspan="1" valign="bottom" align="left"># case<br>CNV<br>calls<br>(n=21)</th><th rowspan="1" colspan="1" valign="bottom" align="left"># control<br>CNV<br>calls<br>(n=962)</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">F-exact<br><em>p</em>-value</th><th rowspan="1" colspan="1" valign="bottom" align="left"># genes<br>(exons)<br>in<br>region</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene in<br>region</th><th rowspan="1" colspan="1" valign="bottom" align="left">Location</th><th rowspan="1" colspan="1" valign="bottom" align="left">5′ gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">5′<br>distance<br>(kb)</th><th rowspan="1" colspan="1" valign="bottom" align="left">3′ gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">3′<br>distance<br>(kb)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">chr2:56583659-56599070</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">15412</td><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">15</td><td rowspan="1" colspan="1" valign="top" align="left">10.52 (2.80–39.56)</td><td rowspan="1" colspan="1" valign="top" align="left">0.006</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CCDC85A</em></td><td rowspan="1" colspan="1" valign="top" align="left">116.8</td><td rowspan="1" colspan="1" valign="top" align="left"><em>VRK2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1528.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">chr2:133001803-133025106</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">23304</td><td rowspan="1" colspan="1" valign="top" align="left">14</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">40</td><td rowspan="1" colspan="1" valign="top" align="left">5.42 (1.74–16.86)</td><td rowspan="1" colspan="1" valign="top" align="left">0.012</td><td rowspan="1" colspan="1" valign="top" align="left">1 (0)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GPR39</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td style="background-color:#D3D3D3" rowspan="1" colspan="1" valign="top" align="left">chr2:184807930-184822613</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">14684</td><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">39</td><td rowspan="1" colspan="1" valign="top" align="left">3.94 (1.11–13.96)</td><td rowspan="1" colspan="1" valign="top" align="left">0.056</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NUP35</em></td><td rowspan="1" colspan="1" valign="top" align="left">1073.3</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF804A</em></td><td rowspan="1" colspan="1" valign="top" align="left">348.7</td></tr><tr><td style="background-color:#D3D3D3" rowspan="1" colspan="1" valign="top" align="left">chr2:184822613-184842482</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">19870</td><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">38</td><td rowspan="1" colspan="1" valign="top" align="left">4.05 (1.14–14.35)</td><td rowspan="1" colspan="1" valign="top" align="left">0.053</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NUP35</em></td><td rowspan="1" colspan="1" valign="top" align="left">1088.0</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF804A</em></td><td rowspan="1" colspan="1" valign="top" align="left">328.9</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">chr4:69720442-69740862</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">20421</td><td rowspan="1" colspan="1" valign="top" align="left">15</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">31</td><td rowspan="1" colspan="1" valign="top" align="left">5.01 (1.40–17.89)</td><td rowspan="1" colspan="1" valign="top" align="left">0.033</td><td rowspan="1" colspan="1" valign="top" align="left">1 (4)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>UGT2B10</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">chr6:29976032-30001174</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">25143</td><td rowspan="1" colspan="1" valign="top" align="left">36</td><td rowspan="1" colspan="1" valign="top" align="left">7</td><td rowspan="1" colspan="1" valign="top" align="left">143</td><td rowspan="1" colspan="1" valign="top" align="left">2.86 (1.14–7.22)</td><td rowspan="1" colspan="1" valign="top" align="left">0.030</td><td rowspan="1" colspan="1" valign="top" align="left">2 (2)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HCG2P7, HCG4P6</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">chr6:31468234-31557306</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">89073</td><td rowspan="1" colspan="1" valign="top" align="left">206</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">39</td><td rowspan="1" colspan="1" valign="top" align="left">3.94 (1.11–13.96)</td><td rowspan="1" colspan="1" valign="top" align="left">0.056</td><td rowspan="1" colspan="1" valign="top" align="left">3 (9)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HCG26, HCP5, MICA</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">chr6:31557757-31561597</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">3841</td><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">43</td><td rowspan="1" colspan="1" valign="top" align="left">3.56 (1.01–12.56)</td><td rowspan="1" colspan="1" valign="top" align="left">0.071</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HCG26</em></td><td rowspan="1" colspan="1" valign="top" align="left">9.6</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MICB</em></td><td rowspan="1" colspan="1" valign="top" align="left">12.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">chr7:78652346-78681225</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">28880</td><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">20</td><td rowspan="1" colspan="1" valign="top" align="left">11.08 (3.42–35.92)</td><td rowspan="1" colspan="1" valign="top" align="left">0.001</td><td rowspan="1" colspan="1" valign="top" align="left">1 (0)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MAGI2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr></tbody></table></div><div id="largeobj_idm139807756457616" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3740040/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>1</sup>Shaded areas indicate contiguous merged CNV regions. Genomic positions are indicated according to the NCBI36/hg18 human genome assembly.</div><div id="TFN2"><p id="__p8" class="p p-first-last">Abbreviations: bp, base pair; CI, confidence interval; CNV, copy-number variant; F-exact, Fisher’s exact test; OR, odds ratio; PE, preeclampsia.</p></div></div></div></div><div id="S12" class="sec"><p></p><h4 id="S12title" class="inline">Subjects of European ancestry </h4><p id="P21" class="p p-first-last">Among mothers of European descent, 13,733 CNVs containing ten or more consecutive probes (8242 deletions and 5491 amplifications) were detected among the 50 PE cases and 1152 normotensive controls meeting sample QC. CNV calls of ~3.1 kb to 25.4 Mb (median=98.5 kb) were merged into 4493 regions of deletion and 2593 regions of amplification with a total of three genomic regions of minimal call overlap meeting the selection criteria (see <a href="/pmc/articles/PMC3740040/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> for an annotated list of candidate CNVs among mothers of European descent). Nominal significance was achieved for all three CNVs.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p9">CNVs identified among HAPO study subjects of European ancestry<sup>1</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139807760115152" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">CNV region (chromosome: start position-stop position)</th><th rowspan="1" colspan="1" valign="bottom" align="left">CNV type</th><th rowspan="1" colspan="1" valign="bottom" align="left">Region size (bp)</th><th rowspan="1" colspan="1" valign="bottom" align="left"># probes within region</th><th rowspan="1" colspan="1" valign="bottom" align="left"># case CNV calls (n=50)</th><th rowspan="1" colspan="1" valign="bottom" align="left"># control CNV calls (n=1152)</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">F-exact <em>p</em>-value</th><th rowspan="1" colspan="1" valign="bottom" align="left"># genes (exons) in region</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene in region</th><th rowspan="1" colspan="1" valign="bottom" align="left">Location</th><th rowspan="1" colspan="1" valign="bottom" align="left">5′ gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">5′ distance (kb)</th><th rowspan="1" colspan="1" valign="bottom" align="left">3′ gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">3′ distance (kb)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr3:152997280-153028731</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">31452</td><td rowspan="1" colspan="1" valign="top" align="left">20</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">17</td><td rowspan="1" colspan="1" valign="top" align="left">5.81 (1.88–17.94)</td><td rowspan="1" colspan="1" valign="top" align="left">0.009</td><td rowspan="1" colspan="1" valign="top" align="left">1 (5)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>AADAC</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr9:1489641-1503917</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">14277</td><td rowspan="1" colspan="1" valign="top" align="left">23</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">7</td><td rowspan="1" colspan="1" valign="top" align="left">10.44 (2.62–41.65)</td><td rowspan="1" colspan="1" valign="top" align="left">0.007</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DMRT2</em></td><td rowspan="1" colspan="1" valign="top" align="left">442.1</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SMARCA2</em></td><td rowspan="1" colspan="1" valign="top" align="left">501.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr12:31257563-31296067</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">38505</td><td rowspan="1" colspan="1" valign="top" align="left">17</td><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">72</td><td rowspan="1" colspan="1" valign="top" align="left">2.86 (1.29–6.31)</td><td rowspan="1" colspan="1" valign="top" align="left">0.015</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DDX11</em></td><td rowspan="1" colspan="1" valign="top" align="left">108.6</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FAM60A</em></td><td rowspan="1" colspan="1" valign="top" align="left">28.7</td></tr></tbody></table></div><div id="largeobj_idm139807760115152" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3740040/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>1</sup>Black box borders indicate contiguous merged CNV regions. Genomic positions are indicated according to the NCBI36/hg18 human genome assembly.</div><div id="TFN4"><p id="__p11" class="p p-first-last">Abbreviations: bp, base pair; CI, confidence interval; CNV, copy-number variant; F-exact, Fisher’s exact test; OR, odds ratio; PE, preeclampsia.</p></div></div></div></div><div id="S13" class="sec sec-last"><p></p><h4 id="S13title" class="inline">Hispanic subjects </h4><p id="P22" class="p p-first-last">A total of 24,046 filtered CNV calls spanning at least 10 probes (20,597 deletions and 3449 amplifications) were made among the eligible 60 PE case and 645 normotensive control Hispanic subjects. Genomic variants ranged in size from ~0.8 kb to 8.6 Mb (median=81.5 kb). CNV calls were merged and screened into 15,263 regions of deletion and 2183 regions of amplification. Fourteen of 15 minimal regions of call overlap enriched among cases that met the pre-specified selection criteria reached nominal significance (see <a href="/pmc/articles/PMC3740040/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> for an annotated list of candidate CNVs among Hispanic mothers).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p12">CNVs identified among Hispanic HAPO study subjects<sup>1</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139807753295232" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">CNV region<br>(chromosome:<br>start position-<br>stop position)</th><th rowspan="1" colspan="1" valign="bottom" align="left">CNV type</th><th rowspan="1" colspan="1" valign="bottom" align="left">Region<br>size (bp)</th><th rowspan="1" colspan="1" valign="bottom" align="left">#<br>probes<br>within<br>region</th><th rowspan="1" colspan="1" valign="bottom" align="left"># case<br>CNV<br>calls<br>(n=60)</th><th rowspan="1" colspan="1" valign="bottom" align="left"># control<br>CNV<br>calls<br>(n=645)</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">F-exact<br><em>p</em>-value</th><th rowspan="1" colspan="1" valign="bottom" align="left"># genes<br>(exons)<br>in<br>region</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene in<br>region</th><th rowspan="1" colspan="1" valign="bottom" align="left">Location</th><th rowspan="1" colspan="1" valign="bottom" align="left">5′ gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">5′<br>distance<br>(kb)</th><th rowspan="1" colspan="1" valign="bottom" align="left">3′ gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">3′<br>distance<br>(kb)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr1:12783336-12793914</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">10579</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">3.47 (1.10–11.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.049</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PRAMEF1</em></td><td rowspan="1" colspan="1" valign="top" align="left">4.5</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HNRNPCL1</em></td><td rowspan="1" colspan="1" valign="top" align="left">35.9</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr1:99983403-100017169</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">33767</td><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">11.26 (2.22–57.09)</td><td rowspan="1" colspan="1" valign="top" align="left">0.010</td><td rowspan="1" colspan="1" valign="top" align="left">1 (4)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FRRS1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr2:57832362-57931532</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">99171</td><td rowspan="1" colspan="1" valign="top" align="left">23</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">6.74 (1.57–28.92)</td><td rowspan="1" colspan="1" valign="top" align="left">0.024</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CCDC85A</em></td><td rowspan="1" colspan="1" valign="top" align="left">1365.5</td><td rowspan="1" colspan="1" valign="top" align="left"><em>VRK2</em></td><td rowspan="1" colspan="1" valign="top" align="left">195.7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr2:219032561-219097202</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">64642</td><td rowspan="1" colspan="1" valign="top" align="left">24</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">5.61 (1.37–23.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.034</td><td rowspan="1" colspan="1" valign="top" align="left">1 (13)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>USP37</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr3:83298713-83407928</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">109216</td><td rowspan="1" colspan="1" valign="top" align="left">20</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">4.19 (1.08–16.24)</td><td rowspan="1" colspan="1" valign="top" align="left">0.059</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GBE1</em></td><td rowspan="1" colspan="1" valign="top" align="left">1405.1</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CADM2</em></td><td rowspan="1" colspan="1" valign="top" align="left">2450.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr3:129874316-129910429</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">36114</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">7</td><td rowspan="1" colspan="1" valign="top" align="left">4.80 (1.21–19.06)</td><td rowspan="1" colspan="1" valign="top" align="left">0.046</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RPN1</em></td><td rowspan="1" colspan="1" valign="top" align="left">21.9</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RAB7A</em></td><td rowspan="1" colspan="1" valign="top" align="left">17.2</td></tr><tr><td style="background-color:#D3D3D3" rowspan="1" colspan="1" valign="top" align="left">Chr5:32144879-32175495</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">30617</td><td rowspan="1" colspan="1" valign="top" align="left">20</td><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">4.80 (1.63–14.11)</td><td rowspan="1" colspan="1" valign="top" align="left">0.011</td><td rowspan="1" colspan="1" valign="top" align="left">2 (3)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GOLPH3, PDZD2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td style="background-color:#D3D3D3" rowspan="1" colspan="1" valign="top" align="left">Chr5:32175495-32195274</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">19780</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">5.77 (1.91–17.49)</td><td rowspan="1" colspan="1" valign="top" align="left">0.006</td><td rowspan="1" colspan="1" valign="top" align="left">1 (1)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GOLPH3</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr6:29982364-30000415</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">18052</td><td rowspan="1" colspan="1" valign="top" align="left">23</td><td rowspan="1" colspan="1" valign="top" align="left">23</td><td rowspan="1" colspan="1" valign="top" align="left">162</td><td rowspan="1" colspan="1" valign="top" align="left">1.85 (1.07–3.21)</td><td rowspan="1" colspan="1" valign="top" align="left">0.032</td><td rowspan="1" colspan="1" valign="top" align="left">1 (1)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HCG4P6</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr7:75959305-75977389</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">18085</td><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">9.14 (2.39–35.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.004</td><td rowspan="1" colspan="1" valign="top" align="left">1 (7)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DTX2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td style="background-color:#D3D3D3" rowspan="1" colspan="1" valign="top" align="left">Chr7:76027543-76115070</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">87528</td><td rowspan="1" colspan="1" valign="top" align="left">23</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">5.69 (1.66–19.48)</td><td rowspan="1" colspan="1" valign="top" align="left">0.014</td><td rowspan="1" colspan="1" valign="top" align="left">1 (7)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>POMZP3</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr><tr><td style="background-color:#D3D3D3" rowspan="1" colspan="1" valign="top" align="left">Chr7:76115070-76157984</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">42915</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">5.05 (1.51–16.91)</td><td rowspan="1" colspan="1" valign="top" align="left">0.019</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>POMZP3</em></td><td rowspan="1" colspan="1" valign="top" align="left">20.5</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PMS2L11</em></td><td rowspan="1" colspan="1" valign="top" align="left">290.1</td></tr><tr><td style="background-color:#D3D3D3" rowspan="1" colspan="1" valign="top" align="left">Chr7:76157984-76291345</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">133362</td><td rowspan="1" colspan="1" valign="top" align="left">56</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">4.54 (1.38–14.93)</td><td rowspan="1" colspan="1" valign="top" align="left">0.025</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>POMZP3</em></td><td rowspan="1" colspan="1" valign="top" align="left">63.4</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PMS2L11</em></td><td rowspan="1" colspan="1" valign="top" align="left">156.7</td></tr><tr><td style="background-color:#D3D3D3" rowspan="1" colspan="1" valign="top" align="left">Chr7:76291345-76400451</td><td rowspan="1" colspan="1" valign="top" align="left">Amplification</td><td rowspan="1" colspan="1" valign="top" align="left">109107</td><td rowspan="1" colspan="1" valign="top" align="left">35</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">4.12 (1.27–13.35)</td><td rowspan="1" colspan="1" valign="top" align="left">0.032</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0)</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>POMZP3</em></td><td rowspan="1" colspan="1" valign="top" align="left">196.8</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PMS2L11</em></td><td rowspan="1" colspan="1" valign="top" align="left">47.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chr22:18274542- 18287303</td><td rowspan="1" colspan="1" valign="top" align="left">Deletion</td><td rowspan="1" colspan="1" valign="top" align="left">12762</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">5.61 (1.37–23.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.034</td><td rowspan="1" colspan="1" valign="top" align="left">1 (6)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TXNRD2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td><td rowspan="1" colspan="1" valign="top" align="left">In</td><td rowspan="1" colspan="1" valign="top" align="left">0.0</td></tr></tbody></table></div><div id="largeobj_idm139807753295232" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3740040/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><sup>1</sup>Shaded areas indicate contiguous merged CNV regions. Genomic positions are indicated according to the NCBI36/hg18 human genome assembly.</div><div id="TFN6"><p id="__p14" class="p p-first-last">Abbreviations: bp, base pair; CI, confidence interval; CNV, copy-number variant; F-exact, Fisher’s exact test; OR, odds ratio; PE, preeclampsia.</p></div></div></div></div></div><div id="S14" class="sec sec-last"><h3 id="S14title">SOPHIA study replication</h3><p id="P23" class="p p-first">Among the top SNPs identified in each of the ethnic groups of the HAPO study, none reached Bonferroni-corrected significance and only several were nominally significant (<em>p</em>&lt;0.05) in the comparison groups (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>). Despite a lack of replication due to the number of examined SNPs, it is interesting to note that two SNPs in HAPO subjects of European ancestry on chromosome 9 (rs7047693 and rs10988989) in an intronic region of the <em>INVS</em> gene yielded trend test <em>p</em>-values of &lt;0.05 in Caucasian SOPHIA subjects.</p><p id="P24" class="p p-last">When comparing the SOPHIA study to the HAPO CNV findings, similar patterns of calls were observed for the copy-number amplification in 12p11.21 in HAPO subjects of European ancestry and in two regions of amplification enriched in Hispanic cases at 5p13.3 and 7q11.23. The intergenic amplification in chromosome 12 was detected in 8/50 (16.0%) cases and 72/1152 (6.3%) controls of the HAPO study and in 19/169 (11.2%) cases and 6/114 (5.3%) controls of the SOPHIA study. Both chromosome 5 and 7 amplifications are exonic. The former region was detected in 5/60 (8.3%) cases and 10/645 (1.6%) controls of the HAPO study and in 6/169 (3.6%) cases and 2/114 (1.8%) controls of the SOPHIA study while the latter region was detected in 4/60 (6.7%) cases and 8/645 (1.2%) controls of the HAPO study and in 4/169 (2.4%) cases and 1/114 (0.9%) controls of the SOPHIA study. Other candidate CNVs identified in HAPO were not replicated.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">DISCUSSION</h2><p id="P25" class="p p-first">Numerous SNP candidates were identified for each of the HAPO ethnic groups, but none were significantly associated with PE after Bonferroni correction (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>). Because the number of PE cases for each ethnic group was relatively small, the present study had limited statistical power to detect a significant association for SNPs of small to moderate effect size. Post-hoc power calculations were performed in PS: Power and Sample Size Calculation version 3.0 (<a href="http://biostat.mc.vanderbilt.edu/wiki/Main/PowerSampleSize" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://biostat.mc.vanderbilt.edu/wiki/Main/PowerSampleSize</a>) using assumptions of minor allele frequency from dbSNP (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492165">Sherry et al. 2001</a>) and effect size based on logistic regression ORs for the top ranked SNP (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>) of each HAPO study group. This study had ~81.2%, 9.8% and 59.4% power to detect associations at the effect sizes reported for the top SNPs among Afro-Caribbean, European ancestry, and Hispanic subjects, respectively. Another limitation on statistical power is that the principal components analysis used to assess and correct population stratification does not explicitly model cryptic relatedness; association statistics that account for cryptic relatedness may achieve higher power due to improved weighting of data (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492159">Price et al. 2010</a>).</p><p id="P26">Genome-wide CNV analysis was successfully performed on 983 Afro-Caribbean (n=21 PE cases and 962 normotensive controls), 1202 European ancestry (n=50 cases and 1152 controls), and 705 Hispanic (n=60 cases and 645 controls) subjects. Multiple candidate regions of copy-number deletion or amplification were detected in each of the three ethnic groups, but none were shared among them (<a href="/pmc/articles/PMC3740040/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>–<a href="/pmc/articles/PMC3740040/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>3</span></a>). Although a majority of these candidate CNVs showed nominal significance with PE (<em>p</em>-value&lt;0.05), no region surpassed the threshold for significance accounting for multiple comparisons.</p><p id="P27">Array-based CNV detection methods are typically plagued by substantial false-positive and false-negative rates (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492157">Tsuang et al. 2010</a>). Until recently, studies commonly used a single algorithm detection approach. Due to the limitations of a single algorithm strategy, molecular techniques are necessary to confirm the presence of CNVs. However, HAPO study DNA samples were unavailable for laboratory validation of identified CNVs. Despite this, confidence can be placed in the accuracy of CNV call prediction as three algorithms (GNOSIS, PennCNV, and QuantiSNP) following two independent statistical approaches were employed with stringent algorithm overlap criteria. Moreover, the reliability of calls can be assessed visually using the LRR and BAF plots. For example, LRR drops to around −0.5 and BAF clusters around 0 or 1, but not near 0.5, for a heterozygous copy-number deletion call, as shown in <a href="/pmc/articles/PMC3740040/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>. Inspection of the plots for the candidate regions revealed signal intensity patterns consistent with their respective copy-number calls.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3740040/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139807749126592" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms448601f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms448601f2.jpg" src="/pmc/articles/PMC3740040/bin/nihms448601f2.jpg"></div></a><div id="largeobj_idm139807749126592" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3740040/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3740040/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3740040/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Representative LRR and BAF plots for a genomic region called as a heterozygous deletion</strong><p id="__p5">LRR and BAF values for each probe are represented as dots. The algorithm-detected deletion is encompassed between the two vertical grey bars (chr3:152.998–153.028 Mb). LRR values for the SNP and copy-number probes drop to the −0.5 region and BAF values for the SNP probes cluster around 0 or 1 within the deleted region (red dots). LRR values of the bordering copy-normal chromosomal regions bunch around zero and BAF values segregate to three clusters (blue dots).</p></div></div></div><p id="P28">Several novel candidate CNVs were identified in the HAPO ethnic groups under study. However, the present study was unable to replicate the top three CNVs identified from the SOPHIA study (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492162">Zhao et al. 2012</a>). The lack of replication may be attributable to microarray probe coverage differences in these genomic regions, an issue that is also relevant to the SNP analysis. The Illumina Human610 microarray used to genotype HAPO subjects of European descent only contained one probe in the 16p13.11 deleted region and two probes in the 19p13.31 deleted region, compared to ten and eight probes respectively in these genomic regions on the Affymetrix SNP 6.0 microarray used in SOPHIA. Contrarily, the three novel copy-number variable genomic regions identified among European ancestry subjects in HAPO (<a href="/pmc/articles/PMC3740040/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) have comparable probe coverage to SOPHIA. The lack of replication in the present study of the top SOPHIA CNVs may be further attributable to geographic differences between the two study populations. Mothers of European descent from the HAPO study were recruited from Australia, Canada, and the United Kingdom whereas Caucasian mothers from the SOPHIA study were recruited from Iowa. Comparing samples from different geographic populations may be problematic as some variants are specific to certain populations (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392492167">Redon et al. 2006</a>). In support of this view, CNV frequencies appear to differ between the control subjects of the two studies (see <a href="#SD3" rid="SD3" class=" supplementary-material">Table S2</a> for a comparison of the call frequencies in female controls and differences in probe coverage for the top candidate regions identified from SOPHIA with HAPO subjects of European ancestry). Therefore, comparability of the two studies for CNV findings is diminished due to microarray coverage differences and geographic variation of the study populations. At the time of writing, no other PE datasets were available to the authors as replication cohorts for array-based CNV detection. These issues, in addition to maintaining adequate statistical power, should be taken into consideration for future PE genetic studies.</p><p id="P29" class="p p-last">In this GWA study, a list of top SNP candidates was identified among Afro-Caribbean, European ancestry, and Hispanic HAPO study subjects, but none reached Bonferroni-corrected significance. However, the top SNP candidates reported in this study may be useful for future work in PE genetics as a resource for replication and meta-analysis. Genome-wide analysis of CNVs across the three ethnic groups identified several novel variants enriched in cases that were markedly less frequent in controls. The validity and functional significance of these candidate CNVs need further investigation. Larger replication studies are necessary for confirmation of the variants identified in this study.</p></div><div id="S16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp F ig S1-S5</h4><p id="P30"><strong>Figure S1.</strong> EIGENSTRAT analysis of population stratification for Afro-Caribbean HAPO study subjects.</p><p id="P31"><strong>Description:</strong> The top scatter plot shows the eighth and ninth eigenvectors of genotypic variance (PC8 vs. PC9) for Afro-Caribbean HAPO study subjects. The bottom scatter plot shows the ninth and tenth eigenvectors (PC9 vs. PC10). The ninth eigenvector is marginally significant (<em>p</em>=0.029).</p><p id="P32"><strong>Figure S2.</strong> EIGENSTRAT analysis of population stratification for HAPO study subjects of European ancestry.</p><p id="P33"><strong>Description:</strong> The top scatter plot shows the first two eigenvectors of genotypic variance (PC1 vs. PC2) for HAPO study subjects of European ancestry. The bottom scatter plot shows the second and third eigenvectors (PC2 vs. PC3). The first eigenvector is significant (<em>p</em>=0.004).</p><p id="P34"><strong>Figure S3.</strong> LocusZoom plot of rs11617740 and ±500 kb flanking region SNP associations in Afro-Caribbean HAPO subjects</p><p id="P35"><strong>Description:</strong> The main panel shows logistic regression <em>p</em>-values on the -log10 scale on the vertical axis and chromosomal position on the horizontal axis based on the NCBI36/hg18 genome assembly. The bottom panel shows positions of genes in the region. Colors of plotted SNP <em>p</em>-values represent linkage disequilibrium r<sup>2</sup> values of each SNP with the most significant SNP, rs11617740 (<em>p</em>=7.32×10<sup>−7</sup>), denoted by a diamond on the plot, calculated using HapMap 2 CEU data.</p><p id="P36"><strong>Figure S4.</strong> LocusZoom plot of rs7322722 and ±500 kb flanking region SNP associations in European ancestry HAPO subjects</p><p id="P37"><strong>Description:</strong> The main panel shows logistic regression <em>p</em>-values on the -log10 scale on the vertical axis and chromosomal position on the horizontal axis based on the NCBI36/hg18 genome assembly. The bottom panel shows positions of genes in the region. Colors of plotted SNP <em>p</em>-values represent linkage disequilibrium r<sup>2</sup> values of each SNP with the most significant SNP, rs7322722 (<em>p</em>=1.23×10<sup>−6</sup>), denoted by a diamond on the plot, calculated using HapMap 2 CEU data.</p><p id="P38"><strong>Figure S5.</strong> LocusZoom plot of rs17412740 and ±500 kb flanking region SNP associations in Hispanic HAPO subjects</p><p id="P39"><strong>Description:</strong> The main panel shows logistic regression <em>p</em>-values on the -log10 scale on the vertical axis and chromosomal position on the horizontal axis based on the NCBI36/hg18 genome assembly. The bottom panel shows positions of genes in the region. Colors of plotted SNP <em>p</em>-values represent linkage disequilibrium r<sup>2</sup> values of each SNP with the most significant SNP, rs17412740 (<em>p</em>=2.14×10<sup>−6</sup>), denoted by a diamond on the plot, calculated using HapMap 2 CEU data.</p><div class="sup-box half_rhythm" id="d36e581"><a href="/pmc/articles/PMC3740040/bin/NIHMS448601-supplement-Supp_F_ig_S1-S5.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(225K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supp Table 01</h4><!--caption a9--><strong>Table S1: </strong><p id="P40">Association of the top SNP candidates (logistic regression <em>p</em>-value &lt;10<sup>−5</sup>) with PE for each ethnic group of HAPO study subjects and Caucasian SOPHIA study subjects</p><p id="P41"><strong>Description:</strong> Top SNP candidates associated with PE ranked by logistic regression <em>p</em>-values are presented in this table for Afro-Caribbean, European ancestry and Hispanic HAPO study subjects and Caucasian SOPHIA study subjects. The logistic regression models for the Afro-Caribbean and European ancestry HAPO subjects were adjusted with PC9 and PC1, respectively. Cochrane-Armitage trend tests were performed in the comparison groups to attempt replication of top SNP findings in the study group of interest. Ungenotyped SNPs in the comparison groups were imputed and tested (trend test) in PLINK.</p><div class="sup-box half_rhythm" id="d36e603"><a href="/pmc/articles/PMC3740040/bin/NIHMS448601-supplement-Supp_Table_01.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(23K, docx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Supp Table 02</h4><!--caption a9--><strong>Table S2: </strong><p id="P42">Comparison of call frequencies and differences in probe coverage for the top regions of copy-number deletion identified from the SOPHIA study with HAPO study controls of European ancestry</p><p id="P43"><strong>Description:</strong> CNV frequency and microarray probe coverage differences between Caucasian female controls from the SOPHIA study (n=114 with an additional 770 population-based controls) and female controls of European descent from the HAPO study (n=1152). Affymetrix SNP 6.0 microarray was used for whole-genome genotyping in the SOPHIA study whereas Illumina Human610 microarray was used to genotype women of European ancestry in the HAPO study.</p><div class="sup-box half_rhythm" id="d36e616"><a href="/pmc/articles/PMC3740040/bin/NIHMS448601-supplement-Supp_Table_02.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(49K, doc)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p1">The authors would like to thank M. Geoffrey Hayes for providing additional methodological details of the HAPO study. Computational work was supported by the Yale University Biomedical High Performance Computing Center and the National Institutes of Health (RR19895 and RR029676-01). Funding support for the GWA mapping: Maternal Metabolism-Birth Weight Interactions study was provided through the NIH Genes, Environment and Health Initiative (GEI) (U01HG004415). The GWA mapping: Maternal Metabolism-Birth Weight Interactions study is one of the GWA studies funded as part of the Gene Environment Association Studies (GENEVA) under GEI. Assistance with phenotype harmonization and genotype cleaning, as well as with general study coordination, was provided by the GENEVA Coordinating Center (U01 <a id="__tag_786507900" class="tag_hotlink" href="/nuccore/HG004446" ref="/nuccore/HG004446"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG004446","term_id":"585865293","term_text":"HG004446"}}</span>HG004446</a>). Assistance with data cleaning was provided by the National Center for Biotechnology Information. Funding support for genotyping, which was performed at the Broad Institute of MIT and Harvard, was provided by the NIH GEI (U01 HG04424). The datasets used for the analyses described in this manuscript were obtained from dbGaP at <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/sites/entrez?db=gap</a> through dbGaP accession number phs000096.v4.p1.</p></div></div><div id="idm139807753846208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139807753846208title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"> </p><p id="__p2" class="p p-first"><strong>Conflict of interest</strong></p><p id="__p3" class="p p-last">The authors declare that they have no conflicts of interest.</p><p></p></div></div><div id="idm139807756827968" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139807756827968title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Arngrimsson R, Björnsson S, Geirsson RT, Björnsson H, Walker JJ, Snaedal G. Genetic and familial predisposition to eclampsia and pre-eclampsia in a defined population. <span><span class="ref-journal">Br J Obstet Gynaecol. </span>1990;<span class="ref-vol">97</span>:762–769.</span> [<a href="/pubmed/2242360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Obstet+Gynaecol&amp;title=Genetic+and+familial+predisposition+to+eclampsia+and+pre-eclampsia+in+a+defined+population&amp;author=R+Arngrimsson&amp;author=S+Björnsson&amp;author=RT+Geirsson&amp;author=H+Björnsson&amp;author=JJ+Walker&amp;volume=97&amp;publication_year=1990&amp;pages=762-769&amp;pmid=2242360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Brown MA, Lindheimer MD, de Swiet M, Van Assche A, Moutquin JM. The classification and diagnosis of the hypertensive disorders of pregnancy: statement from the International Society for the Study of Hypertension in Pregnancy (ISSHP) <span><span class="ref-journal">Hypertens Pregnancy. </span>2001;<span class="ref-vol">20</span>:IX–XIV.</span> [<a href="/pubmed/12044323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertens+Pregnancy&amp;title=The+classification+and+diagnosis+of+the+hypertensive+disorders+of+pregnancy:+statement+from+the+International+Society+for+the+Study+of+Hypertension+in+Pregnancy+(ISSHP)&amp;author=MA+Brown&amp;author=MD+Lindheimer&amp;author=M+de+Swiet&amp;author=A+Van+Assche&amp;author=JM+Moutquin&amp;volume=20&amp;publication_year=2001&amp;pages=IX-XIV&amp;pmid=12044323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Caughey AB, Stotland NE, Washington AE, Escobar GJ. Maternal ethnicity, paternal ethnicity, and parental ethnic discordance: predictors of preeclampsia. <span><span class="ref-journal">Obstet Gynecol. </span>2005;<span class="ref-vol">106</span>:156–161.</span> [<a href="/pubmed/15994632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obstet+Gynecol&amp;title=Maternal+ethnicity,+paternal+ethnicity,+and+parental+ethnic+discordance:+predictors+of+preeclampsia&amp;author=AB+Caughey&amp;author=NE+Stotland&amp;author=AE+Washington&amp;author=GJ+Escobar&amp;volume=106&amp;publication_year=2005&amp;pages=156-161&amp;pmid=15994632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Cnattingius S, Reilly M, Pawitan Y, Lichtenstein P. Maternal and fetal genetic factors account for most of familial aggregation of preeclampsia: a population-based Swedish cohort study. <span><span class="ref-journal">Am J Med Genet A. </span>2004;<span class="ref-vol">130A</span>:365–371.</span> [<a href="/pubmed/15384082" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Maternal+and+fetal+genetic+factors+account+for+most+of+familial+aggregation+of+preeclampsia:+a+population-based+Swedish+cohort+study&amp;author=S+Cnattingius&amp;author=M+Reilly&amp;author=Y+Pawitan&amp;author=P+Lichtenstein&amp;volume=130A&amp;publication_year=2004&amp;pages=365-371&amp;pmid=15384082&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Colella S, Yau C, Taylor JM, Mirza G, Butler H, Clouston P, Bassett AS, Seller A, Holmes CC, Ragoussis J. QuantiSNP: an Objective Bayes Hidden-Markov Model to detect and accurately map copy number variation using SNP genotyping data. <span><span class="ref-journal">Nucleic Acids Res. </span>2007;<span class="ref-vol">35</span>:2013–2025.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1874617/">PMC free article</a>]</span> [<a href="/pubmed/17341461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=QuantiSNP:+an+Objective+Bayes+Hidden-Markov+Model+to+detect+and+accurately+map+copy+number+variation+using+SNP+genotyping+data&amp;author=S+Colella&amp;author=C+Yau&amp;author=JM+Taylor&amp;author=G+Mirza&amp;author=H+Butler&amp;volume=35&amp;publication_year=2007&amp;pages=2013-2025&amp;pmid=17341461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Colhoun HM, McKeigue PM, Davey Smith G. Problems of reporting genetic associations with complex outcomes. <span><span class="ref-journal">Lancet. </span>2003;<span class="ref-vol">361</span>:865–872.</span> [<a href="/pubmed/12642066" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Problems+of+reporting+genetic+associations+with+complex+outcomes&amp;author=HM+Colhoun&amp;author=PM+McKeigue&amp;author=G+Davey+Smith&amp;volume=361&amp;publication_year=2003&amp;pages=865-872&amp;pmid=12642066&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">HAPO Study Cooperative Research Group. The Hyperglycemia and Adverse Pregnancy Outcome (HAPO) Study. <span><span class="ref-journal">Int J Gynaecol Obstet. </span>2002;<span class="ref-vol">78</span>:69–77.</span> [<a href="/pubmed/12113977" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Gynaecol+Obstet&amp;title=The+Hyperglycemia+and+Adverse+Pregnancy+Outcome+(HAPO)+Study&amp;volume=78&amp;publication_year=2002&amp;pages=69-77&amp;pmid=12113977&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Lie RT, Rasmussen S, Brunborg H, Gjessing HK, Lie-Nielsen E, Irgens LM. Fetal and maternal contributions to risk of pre-eclampsia: population based study. <span><span class="ref-journal">BMJ. </span>1998;<span class="ref-vol">316</span>:1343–1347.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC28531/">PMC free article</a>]</span> [<a href="/pubmed/9563982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Fetal+and+maternal+contributions+to+risk+of+pre-eclampsia:+population+based+study&amp;author=RT+Lie&amp;author=S+Rasmussen&amp;author=H+Brunborg&amp;author=HK+Gjessing&amp;author=E+Lie-Nielsen&amp;volume=316&amp;publication_year=1998&amp;pages=1343-1347&amp;pmid=9563982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Mogren I, Högberg U, Winkvist A, Stenlund H. Familial occurrence of preeclampsia. <span><span class="ref-journal">Epidemiology. </span>1999;<span class="ref-vol">10</span>:518–522.</span> [<a href="/pubmed/10468424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epidemiology&amp;title=Familial+occurrence+of+preeclampsia&amp;author=I+Mogren&amp;author=U+Högberg&amp;author=A+Winkvist&amp;author=H+Stenlund&amp;volume=10&amp;publication_year=1999&amp;pages=518-522&amp;pmid=10468424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Price AL, Zaitlen NA, Reich D, Patterson N. New approaches to population stratification in genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>:459–463.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2975875/">PMC free article</a>]</span> [<a href="/pubmed/20548291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=New+approaches+to+population+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NA+Zaitlen&amp;author=D+Reich&amp;author=N+Patterson&amp;volume=11&amp;publication_year=2010&amp;pages=459-463&amp;pmid=20548291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, Boehnke M, Abecasis GR, Willer CJ. LocusZoom: Regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+Regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, Andrews TD, Fiegler H, Shapero MH, Carson AR, Chen W, Cho EK, Dallaire S, Freeman JL, González JR, Gratacòs M, Huang J, Kalaitzopoulos D, Komura D, MacDonald JR, Marshall CR, Mei R, Montgomery L, Nishimura K, Okamura K, Shen F, Somerville MJ, Tchinda J, Valsesia A, Woodwark C, Yang F, Zhang J, Zerjal T, Zhang J, Armengol L, Conrad DF, Estivill X, Tyler-Smith C, Carter NP, Aburatani H, Lee C, Jones KW, Scherer SW, Hurles ME. Global variation in copy number in the human genome. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">444</span>:444–454.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669898/">PMC free article</a>]</span> [<a href="/pubmed/17122850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Global+variation+in+copy+number+in+the+human+genome&amp;author=R+Redon&amp;author=S+Ishikawa&amp;author=KR+Fitch&amp;author=L+Feuk&amp;author=GH+Perry&amp;volume=444&amp;publication_year=2006&amp;pages=444-454&amp;pmid=17122850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Sanders SJ, Ercan-Sencicek AG, Hus V, Luo R, Murtha MT, Moreno-De-Luca D, Chu SH, Moreau MP, Gupta AR, Thomson SA, Mason CE, Bilguvar K, Celestino-Soper PB, Choi M, Crawford EL, Davis L, Wright NR, Dhodapkar RM, DiCola M, DiLullo NM, Fernandez TV, Fielding-Singh V, Fishman DO, Frahm S, Garagaloyan R, Goh GS, Kammela S, Klei L, Lowe JK, Lund SC, McGrew AD, Meyer KA, Moffat WJ, Murdoch JD, O’Roak BJ, Ober GT, Pottenger RS, Raubeson MJ, Song Y, Wang Q, Yaspan BL, Yu TW, Yurkiewicz IR, Beaudet AL, Cantor RM, Curland M, Grice DE, Günel M, Lifton RP, Mane SM, Martin DM, Shaw CA, Sheldon M, Tischfield JA, Walsh CA, Morrow EM, Ledbetter DH, Fombonne E, Lord C, Martin CL, Brooks AI, Sutcliffe JS, Cook EH, Jr, Geschwind D, Roeder K, Devlin B, State MW. Multiple recurrent de novo CNVs, including duplications of the 7q11.23 Williams syndrome region, are strongly associated with autism. <span><span class="ref-journal">Neuron. </span>2011;<span class="ref-vol">70</span>:863–885.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3939065/">PMC free article</a>]</span> [<a href="/pubmed/21658581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Multiple+recurrent+de+novo+CNVs,+including+duplications+of+the+7q11.23+Williams+syndrome+region,+are+strongly+associated+with+autism&amp;author=SJ+Sanders&amp;author=AG+Ercan-Sencicek&amp;author=V+Hus&amp;author=R+Luo&amp;author=MT+Murtha&amp;volume=70&amp;publication_year=2011&amp;pages=863-885&amp;pmid=21658581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Sherry ST, Ward MH, Kholodov M, Baker J, Phan L, Smigielski EM, Sirotkin K. dbSNP: the NCBI database of genetic variation. <span><span class="ref-journal">Nucleic Acids Res. </span>2001;<span class="ref-vol">29</span>:308–311.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC29783/">PMC free article</a>]</span> [<a href="/pubmed/11125122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=dbSNP:+the+NCBI+database+of+genetic+variation&amp;author=ST+Sherry&amp;author=MH+Ward&amp;author=M+Kholodov&amp;author=J+Baker&amp;author=L+Phan&amp;volume=29&amp;publication_year=2001&amp;pages=308-311&amp;pmid=11125122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Sibai B, Dekker G, Kupferminc M. Pre-eclampsia. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">365</span>:785–799.</span> [<a href="/pubmed/15733721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Pre-eclampsia&amp;author=B+Sibai&amp;author=G+Dekker&amp;author=M+Kupferminc&amp;volume=365&amp;publication_year=2005&amp;pages=785-799&amp;pmid=15733721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Stankiewicz P, Lupski JR. Structural variation in the human genome and its role in disease. <span><span class="ref-journal">Annu Rev Med. </span>2010;<span class="ref-vol">61</span>:437–455.</span> [<a href="/pubmed/20059347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Med&amp;title=Structural+variation+in+the+human+genome+and+its+role+in+disease&amp;author=P+Stankiewicz&amp;author=JR+Lupski&amp;volume=61&amp;publication_year=2010&amp;pages=437-455&amp;pmid=20059347&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Sutherland A, Cooper DW, Howie PW, Liston WA, MacGillivray I. The indicence of severe pre-eclampsia amongst mothers and mothers-in-law of pre-eclamptics and controls. <span><span class="ref-journal">Br J Obstet Gynaecol. </span>1981;<span class="ref-vol">88</span>:785–791.</span> [<a href="/pubmed/7259998" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Obstet+Gynaecol&amp;title=The+indicence+of+severe+pre-eclampsia+amongst+mothers+and+mothers-in-law+of+pre-eclamptics+and+controls&amp;author=A+Sutherland&amp;author=DW+Cooper&amp;author=PW+Howie&amp;author=WA+Liston&amp;author=I+MacGillivray&amp;volume=88&amp;publication_year=1981&amp;pages=785-791&amp;pmid=7259998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Tsuang DW, Millard SP, Ely B, Chi P, Wang K, Raskind WH, Kim S, Brkanac Z, Yu CE. The effect of algorithms on copy number variant detection. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>:e14456.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3012691/">PMC free article</a>]</span> [<a href="/pubmed/21209939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=The+effect+of+algorithms+on+copy+number+variant+detection&amp;author=DW+Tsuang&amp;author=SP+Millard&amp;author=B+Ely&amp;author=P+Chi&amp;author=K+Wang&amp;volume=5&amp;publication_year=2010&amp;pages=e14456&amp;pmid=21209939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Urbanek M, Hayes MG, Lee H, Freathy RM, Lowe LP, Ackerman C, Jafari N, Dyer AR, Cox NJ, Dunger DB, Hattersley AT, Metzger BE, Lowe WL., Jr The Role of Inflammatory Pathway Genetic Variation on Maternal Metabolic Phenotypes during Pregnancy. <span><span class="ref-journal">PLoS One. </span>2012;<span class="ref-vol">7</span>:e32958.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3316547/">PMC free article</a>]</span> [<a href="/pubmed/22479352" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=The+Role+of+Inflammatory+Pathway+Genetic+Variation+on+Maternal+Metabolic+Phenotypes+during+Pregnancy&amp;author=M+Urbanek&amp;author=MG+Hayes&amp;author=H+Lee&amp;author=RM+Freathy&amp;author=LP+Lowe&amp;volume=7&amp;publication_year=2012&amp;pages=e32958&amp;pmid=22479352&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Wang K, Li M, Hadley D, Liu R, Glessner J, Grant SF, Hakonarson H, Bucan M. PennCNV: an integrated hidden Markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data. <span><span class="ref-journal">Genome Res. </span>2007;<span class="ref-vol">17</span>:1665–1674.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2045149/">PMC free article</a>]</span> [<a href="/pubmed/17921354" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=PennCNV:+an+integrated+hidden+Markov+model+designed+for+high-resolution+copy+number+variation+detection+in+whole-genome+SNP+genotyping+data&amp;author=K+Wang&amp;author=M+Li&amp;author=D+Hadley&amp;author=R+Liu&amp;author=J+Glessner&amp;volume=17&amp;publication_year=2007&amp;pages=1665-1674&amp;pmid=17921354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Williamson C. Molecular biology related to pre-eclampsia. <span><span class="ref-journal">Int Congr Ser. </span>2005;<span class="ref-vol">1279</span>:282–289.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+Congr+Ser&amp;title=Molecular+biology+related+to+pre-eclampsia&amp;author=C+Williamson&amp;volume=1279&amp;publication_year=2005&amp;pages=282-289&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Zhao L, Triche EW, Walsh KM, Bracken MB, Saftlas AF, Hoh J, Dewan AT. Genome-wide association study identifies a maternal copy-number deletion in PSG11 enriched among preeclampsia patients. <span><span class="ref-journal">BMC Pregnancy Childbirth. </span>2012;<span class="ref-vol">12</span>:61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3476390/">PMC free article</a>]</span> [<a href="/pubmed/22748001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Pregnancy+Childbirth&amp;title=Genome-wide+association+study+identifies+a+maternal+copy-number+deletion+in+PSG11+enriched+among+preeclampsia+patients&amp;author=L+Zhao&amp;author=EW+Triche&amp;author=KM+Walsh&amp;author=MB+Bracken&amp;author=AF+Saftlas&amp;volume=12&amp;publication_year=2012&amp;pages=61&amp;pmid=22748001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3740040&amp;issue-id=226215&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139807744694592title">Summary</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">MATERIALS AND METHODS</a></li><li><a href="#S8title">RESULTS</a></li><li><a href="#S15title">DISCUSSION</a></li><li><a href="#S16title">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#idm139807753846208title">Footnotes</a></li><li><a href="#idm139807756827968title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3740040/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3740040/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3740040/pdf/nihms448601.pdf">PDF (151K)</a></li> | <li><a href="#" data-citationid="PMC3740040" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3740040%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3740040%2F&amp;text=Genome-wide%20association%20study%20of%20preeclampsia%20detects%20novel%20maternal%20single%20nucleotide%20polymorphisms%20and%20copy-number%20variants%20in%20subsets%20of%20the%20Hyperglycemia%20and%20Adverse%20Pregnancy%20Outcome%20(HAPO)%20study%20cohort" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3740040%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23551011" data-aiid="3740040" data-aid="3740040" data-iid="226215" data-domainid="319" data-domain="nihpa" data-accid="PMC3740040" data-md5="1bcd7990ec56bfcd8b4668a4fc23dfad"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:38:50-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C53F1DD55C0C10000000001930107&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3740040%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3740040/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>