<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3335297/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A Genome Wide Association Study for Coronary Artery Disease Identifies a Novel Susceptibility Locus in the Major Histocompatibility Complex</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3335297/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Circulation. Cardiovascular Genetics"><meta name="citation_title" content="A Genome Wide Association Study for Coronary Artery Disease Identifies a Novel Susceptibility Locus in the Major Histocompatibility Complex"><meta name="citation_authors" content="Robert W. Davies, George A. Wells, Alexandre F.R. Stewart, Jeanette Erdmann, Svati H. Shah, Jane F. Ferguson, Alistair S. Hall, Sonia S. Anand, Mary S. Burnett, Stephen E. Epstein, Sonny Dandona, Li Chen, Janja Nahrstaedt, Christina Loley, Inke R. König, William E. Kraus, Christopher B. Granger, James C. Engert, Christian Hengstenberg, H.-Erich Wichmann, Stefan Schreiber, W. H. Wilson Tang, Stephen G. Ellis, Daniel J. Rader, Stanley L. Hazen, Muredach P. Reilly, Nilesh J. Samani, Heribert Schunkert, Robert Roberts, Ruth McPherson"><meta name="citation_date" content="1 April 2012"><meta name="citation_issue" content="2"><meta name="citation_volume" content="5"><meta name="citation_firstpage" content="217"><meta name="citation_doi" content="10.1161/CIRCGENETICS.111.961243"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3335297/?report=abstract"><meta name="citation_pmid" content="22319020"><meta name="DC.Title" content="A Genome Wide Association Study for Coronary Artery Disease Identifies a Novel Susceptibility Locus in the Major Histocompatibility Complex"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Robert W. Davies"><meta name="DC.Contributor" content="George A. Wells"><meta name="DC.Contributor" content="Alexandre F.R. Stewart"><meta name="DC.Contributor" content="Jeanette Erdmann"><meta name="DC.Contributor" content="Svati H. Shah"><meta name="DC.Contributor" content="Jane F. Ferguson"><meta name="DC.Contributor" content="Alistair S. Hall"><meta name="DC.Contributor" content="Sonia S. Anand"><meta name="DC.Contributor" content="Mary S. Burnett"><meta name="DC.Contributor" content="Stephen E. Epstein"><meta name="DC.Contributor" content="Sonny Dandona"><meta name="DC.Contributor" content="Li Chen"><meta name="DC.Contributor" content="Janja Nahrstaedt"><meta name="DC.Contributor" content="Christina Loley"><meta name="DC.Contributor" content="Inke R. König"><meta name="DC.Contributor" content="William E. Kraus"><meta name="DC.Contributor" content="Christopher B. Granger"><meta name="DC.Contributor" content="James C. Engert"><meta name="DC.Contributor" content="Christian Hengstenberg"><meta name="DC.Contributor" content="H.-Erich Wichmann"><meta name="DC.Contributor" content="Stefan Schreiber"><meta name="DC.Contributor" content="W. H. Wilson Tang"><meta name="DC.Contributor" content="Stephen G. Ellis"><meta name="DC.Contributor" content="Daniel J. Rader"><meta name="DC.Contributor" content="Stanley L. Hazen"><meta name="DC.Contributor" content="Muredach P. Reilly"><meta name="DC.Contributor" content="Nilesh J. Samani"><meta name="DC.Contributor" content="Heribert Schunkert"><meta name="DC.Contributor" content="Robert Roberts"><meta name="DC.Contributor" content="Ruth McPherson"><meta name="DC.Date" content="2012 Apr 1"><meta name="DC.Identifier" content="10.1161/CIRCGENETICS.111.961243"><meta name="DC.Language" content="en"><meta property="og:title" content="A Genome Wide Association Study for Coronary Artery Disease Identifies a Novel Susceptibility Locus in the Major Histocompatibility Complex"><meta property="og:type" content="article"><meta property="og:description" content="Recent genome-wide association studies (GWAS) have identified several novel loci that reproducibly associate with CAD and/or MI risk. However, known common CAD risk variants explain only 10% of the predicted genetic heritability of the disease, suggesting ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3335297/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3335297/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3335297/pdf/nihms362303.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8940E3DD53509100000000076E0506.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3335297%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="nuccore">Nucleotide</option><option value="clinvar" class="last">ClinVar</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3335297</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Circ Cardiovasc Genet</a></span>. Author manuscript; available in PMC 2012 Apr 24.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22319020" target="pmc_ext" ref="reftype=publisher&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Circ Cardiovasc Genet. 2012 Apr 1; 5(2): 217–225. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Feb 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1161%2FCIRCGENETICS.111.961243" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1161/CIRCGENETICS.111.961243</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3335297</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS362303</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span><a class="int-reflink" href="/pmc/articles/mid/UKMS47259/">UKMS47259</a></span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22319020">22319020</a></div></div></div></div><h1 class="content-title">A Genome Wide Association Study for Coronary Artery Disease Identifies a Novel Susceptibility Locus in the Major Histocompatibility Complex</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Davies%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435594275664" co-class="co-affbox">Robert W. Davies</a>, MSc,<sup>1</sup> <a href="/pubmed/?term=Wells%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435592593200" co-class="co-affbox">George A. Wells</a>, MSc, PhD,<sup>1</sup> <a href="/pubmed/?term=Stewart%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435594272192" co-class="co-affbox">Alexandre F.R. Stewart</a>, PhD,<sup>2</sup> <a href="/pubmed/?term=Erdmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435594269712" co-class="co-affbox">Jeanette Erdmann</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Shah%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435556425984" co-class="co-affbox">Svati H. Shah</a>, MD, MHS,<sup>4</sup> <a href="/pubmed/?term=Ferguson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435556423504" co-class="co-affbox">Jane F. Ferguson</a>, PhD,<sup>5</sup> <a href="/pubmed/?term=Hall%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435556421024" co-class="co-affbox">Alistair S. Hall</a>, MD, PhD,<sup>6</sup> <a href="/pubmed/?term=Anand%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435556418544" co-class="co-affbox">Sonia S. Anand</a>, MD, PhD,<sup>7</sup> <a href="/pubmed/?term=Burnett%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435552157632" co-class="co-affbox">Mary S. Burnett</a>, PHD,<sup>8</sup> <a href="/pubmed/?term=Epstein%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435552155152" co-class="co-affbox">Stephen E. Epstein</a>, MD,<sup>8</sup> <a href="/pubmed/?term=Dandona%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435552152672" co-class="co-affbox">Sonny Dandona</a>, MD,<sup>2</sup> <a href="/pubmed/?term=Chen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435552150192" co-class="co-affbox">Li Chen</a>, MSc,<sup>1</sup> <a href="/pubmed/?term=Nahrstaedt%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435554839216" co-class="co-affbox">Janja Nahrstaedt</a>, MSc,<sup>3,</sup><sup>9</sup> <a href="/pubmed/?term=Loley%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435554836000" co-class="co-affbox">Christina Loley</a>, MSc,<sup>3,</sup><sup>9</sup> <a href="/pubmed/?term=K%26%23x000f6%3Bnig%20IR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435554832784" co-class="co-affbox">Inke R. König</a>, PhD,<sup>9</sup> <a href="/pubmed/?term=Kraus%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435554713040" co-class="co-affbox">William E. Kraus</a>, MD,<sup>4</sup> <a href="/pubmed/?term=Granger%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435554710560" co-class="co-affbox">Christopher B. Granger</a>, MD,<sup>4</sup> <a href="/pubmed/?term=Engert%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435554708080" co-class="co-affbox">James C. Engert</a>, PhD,<sup>10</sup> <a href="/pubmed/?term=Hengstenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435554705600" co-class="co-affbox">Christian Hengstenberg</a>, MD,<sup>11</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435556332304" co-class="co-affbox">H.-Erich Wichmann</a>, MD, PhD,<sup>12</sup> <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435556329824" co-class="co-affbox">Stefan Schreiber</a>, MD,<sup>13</sup> <a href="/pubmed/?term=Wilson%20Tang%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435556327344" co-class="co-affbox">W. H. Wilson Tang</a>, MD,<sup>14</sup> <a href="/pubmed/?term=Ellis%20SG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435556324864" co-class="co-affbox">Stephen G. Ellis</a>, MD,<sup>14</sup> <a href="/pubmed/?term=Rader%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435557129792" co-class="co-affbox">Daniel J. Rader</a>, MD,<sup>15</sup> <a href="/pubmed/?term=Hazen%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435557127312" co-class="co-affbox">Stanley L. Hazen</a>, MD, PhD,<sup>14</sup> <a href="/pubmed/?term=Reilly%20MP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435557124832" co-class="co-affbox">Muredach P. Reilly</a>, MB,<sup>5</sup> <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435557122352" co-class="co-affbox">Nilesh J. Samani</a>, MD, FMedSci,<sup>16</sup> <a href="/pubmed/?term=Schunkert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435594190448" co-class="co-affbox">Heribert Schunkert</a>, MD,<sup>3</sup> <a href="/pubmed/?term=Roberts%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435594187968" co-class="co-affbox">Robert Roberts</a>, MD,<sup>2</sup> and  <a href="/pubmed/?term=McPherson%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020" class="affpopup" co-rid="_co_idm140435594185488" co-class="co-affbox">Ruth McPherson</a>, MD, PhD<sup>2,</sup><sup>17</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140435594275664"><h3 class="no_margin">Robert W. Davies</h3><p><sup>1</sup>Cardiovascular Rsrch Methods Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</p><div>Find articles by <a href="/pubmed/?term=Davies%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Robert W. Davies</a></div></div><div id="_co_idm140435592593200"><h3 class="no_margin">George A. Wells</h3><p><sup>1</sup>Cardiovascular Rsrch Methods Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</p><div>Find articles by <a href="/pubmed/?term=Wells%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">George A. Wells</a></div></div><div id="_co_idm140435594272192"><h3 class="no_margin">Alexandre F.R. Stewart</h3><p><sup>2</sup>John &amp; Jennifer Ruddy Canadian Cardiovascular Rsrch Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</p><div>Find articles by <a href="/pubmed/?term=Stewart%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Alexandre F.R. Stewart</a></div></div><div id="_co_idm140435594269712"><h3 class="no_margin">Jeanette Erdmann</h3><p><sup>3</sup>Medizinische Klinik II, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Erdmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Jeanette Erdmann</a></div></div><div id="_co_idm140435556425984"><h3 class="no_margin">Svati H. Shah</h3><p><sup>4</sup>Dept of Med, Duke Univ Med Ctr, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Shah%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Svati H. Shah</a></div></div><div id="_co_idm140435556423504"><h3 class="no_margin">Jane F. Ferguson</h3><p><sup>5</sup>Cardiovascular Inst, Univ of Pennsylvania, Philadelphia, PA</p><div>Find articles by <a href="/pubmed/?term=Ferguson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Jane F. Ferguson</a></div></div><div id="_co_idm140435556421024"><h3 class="no_margin">Alistair S. Hall</h3><p><sup>6</sup>Division of Cardiovascular &amp; Neuronal Remodeling, Multidisciplinary Cardiovascular Rsrch Ctr, Leeds Inst of Genetics, Health &amp; Therapeutics, Univ of Leeds, UK</p><div>Find articles by <a href="/pubmed/?term=Hall%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Alistair S. Hall</a></div></div><div id="_co_idm140435556418544"><h3 class="no_margin">Sonia S. Anand</h3><p><sup>7</sup>Population Health Rsrch Inst, McMaster Univ, Hamilton Health Sciences, Hamilton, ON, Canada</p><div>Find articles by <a href="/pubmed/?term=Anand%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Sonia S. Anand</a></div></div><div id="_co_idm140435552157632"><h3 class="no_margin">Mary S. Burnett</h3><p><sup>8</sup>Cardiovascular Rsrch Inst, Medstar Health Rsrch Inst, Washington Hospital Ctr, Washington, DC</p><div>Find articles by <a href="/pubmed/?term=Burnett%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Mary S. Burnett</a></div></div><div id="_co_idm140435552155152"><h3 class="no_margin">Stephen E. Epstein</h3><p><sup>8</sup>Cardiovascular Rsrch Inst, Medstar Health Rsrch Inst, Washington Hospital Ctr, Washington, DC</p><div>Find articles by <a href="/pubmed/?term=Epstein%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Stephen E. Epstein</a></div></div><div id="_co_idm140435552152672"><h3 class="no_margin">Sonny Dandona</h3><p><sup>2</sup>John &amp; Jennifer Ruddy Canadian Cardiovascular Rsrch Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</p><div>Find articles by <a href="/pubmed/?term=Dandona%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Sonny Dandona</a></div></div><div id="_co_idm140435552150192"><h3 class="no_margin">Li Chen</h3><p><sup>1</sup>Cardiovascular Rsrch Methods Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</p><div>Find articles by <a href="/pubmed/?term=Chen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Li Chen</a></div></div><div id="_co_idm140435554839216"><h3 class="no_margin">Janja Nahrstaedt</h3><p><sup>3</sup>Medizinische Klinik II, Lübeck, Germany</p><p><sup>9</sup>Inst für Medizinische Biometrie und Statistik, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Nahrstaedt%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Janja Nahrstaedt</a></div></div><div id="_co_idm140435554836000"><h3 class="no_margin">Christina Loley</h3><p><sup>3</sup>Medizinische Klinik II, Lübeck, Germany</p><p><sup>9</sup>Inst für Medizinische Biometrie und Statistik, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Loley%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Christina Loley</a></div></div><div id="_co_idm140435554832784"><h3 class="no_margin">Inke R. König</h3><p><sup>9</sup>Inst für Medizinische Biometrie und Statistik, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000f6%3Bnig%20IR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Inke R. König</a></div></div><div id="_co_idm140435554713040"><h3 class="no_margin">William E. Kraus</h3><p><sup>4</sup>Dept of Med, Duke Univ Med Ctr, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Kraus%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">William E. Kraus</a></div></div><div id="_co_idm140435554710560"><h3 class="no_margin">Christopher B. Granger</h3><p><sup>4</sup>Dept of Med, Duke Univ Med Ctr, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Granger%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Christopher B. Granger</a></div></div><div id="_co_idm140435554708080"><h3 class="no_margin">James C. Engert</h3><p><sup>10</sup>Depts of Med &amp; Human Genetics, McGill Univ, Montreal, Canada</p><div>Find articles by <a href="/pubmed/?term=Engert%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">James C. Engert</a></div></div><div id="_co_idm140435554705600"><h3 class="no_margin">Christian Hengstenberg</h3><p><sup>11</sup>Klinik und Poliklinik für Innere Medizin II, Univ Regensburg, Regensburg</p><div>Find articles by <a href="/pubmed/?term=Hengstenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Christian Hengstenberg</a></div></div><div id="_co_idm140435556332304"><h3 class="no_margin">H.-Erich Wichmann</h3><p><sup>12</sup>Inst of Epidemiology I, Helmholtz Zentrum München – German Rsrch Ctr for Environmental Health, Neuherberg, Klinikum Grosshadern, Munich, Inst of Med Info Science, Biometry &amp; Epidemiology, Ludwig-Maximilians-Univ München</p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">H.-Erich Wichmann</a></div></div><div id="_co_idm140435556329824"><h3 class="no_margin">Stefan Schreiber</h3><p><sup>13</sup>Institut für Klinische Molekularbiologie, Christian-Albrechts Univ, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Stefan Schreiber</a></div></div><div id="_co_idm140435556327344"><h3 class="no_margin">W. H. Wilson Tang</h3><p><sup>14</sup>Cleveland Clinic, Cleveland, OH</p><div>Find articles by <a href="/pubmed/?term=Wilson%20Tang%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">W. H. Wilson Tang</a></div></div><div id="_co_idm140435556324864"><h3 class="no_margin">Stephen G. Ellis</h3><p><sup>14</sup>Cleveland Clinic, Cleveland, OH</p><div>Find articles by <a href="/pubmed/?term=Ellis%20SG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Stephen G. Ellis</a></div></div><div id="_co_idm140435557129792"><h3 class="no_margin">Daniel J. Rader</h3><p><sup>15</sup>Inst for Translational Med &amp; Therapeutics, School of Med Univ of Pennsylvania, Philadelphia, PA</p><div>Find articles by <a href="/pubmed/?term=Rader%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Daniel J. Rader</a></div></div><div id="_co_idm140435557127312"><h3 class="no_margin">Stanley L. Hazen</h3><p><sup>14</sup>Cleveland Clinic, Cleveland, OH</p><div>Find articles by <a href="/pubmed/?term=Hazen%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Stanley L. Hazen</a></div></div><div id="_co_idm140435557124832"><h3 class="no_margin">Muredach P. Reilly</h3><p><sup>5</sup>Cardiovascular Inst, Univ of Pennsylvania, Philadelphia, PA</p><div>Find articles by <a href="/pubmed/?term=Reilly%20MP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Muredach P. Reilly</a></div></div><div id="_co_idm140435557122352"><h3 class="no_margin">Nilesh J. Samani</h3><p><sup>16</sup>Dept of Cardiovascular Sciences, Univ of Leicester, Clinical Sciences Wing, Glenfield Hospital; Leicester Nat Inst for Health Rsrch, Biomedical Rsrch Unit in Cardiovascular Disease, Leicester, UK</p><div>Find articles by <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Nilesh J. Samani</a></div></div><div id="_co_idm140435594190448"><h3 class="no_margin">Heribert Schunkert</h3><p><sup>3</sup>Medizinische Klinik II, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Schunkert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Heribert Schunkert</a></div></div><div id="_co_idm140435594187968"><h3 class="no_margin">Robert Roberts</h3><p><sup>2</sup>John &amp; Jennifer Ruddy Canadian Cardiovascular Rsrch Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</p><div>Find articles by <a href="/pubmed/?term=Roberts%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Robert Roberts</a></div></div><div id="_co_idm140435594185488"><h3 class="no_margin">Ruth McPherson</h3><p><sup>2</sup>John &amp; Jennifer Ruddy Canadian Cardiovascular Rsrch Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</p><p><sup>17</sup>Atherogenomics Lab, Univ of Ottawa Heart Inst, Ottawa, Canada</p><div>Find articles by <a href="/pubmed/?term=McPherson%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22319020">Ruth McPherson</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140435597745648_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140435597745648_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140435597745648_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Cardiovascular Rsrch Methods Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>John &amp; Jennifer Ruddy Canadian Cardiovascular Rsrch Ctr, Univ of Ottawa Heart Inst, Ottawa, Canada</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Medizinische Klinik II, Lübeck, Germany</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Dept of Med, Duke Univ Med Ctr, Durham, NC</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Cardiovascular Inst, Univ of Pennsylvania, Philadelphia, PA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Division of Cardiovascular &amp; Neuronal Remodeling, Multidisciplinary Cardiovascular Rsrch Ctr, Leeds Inst of Genetics, Health &amp; Therapeutics, Univ of Leeds, UK</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Population Health Rsrch Inst, McMaster Univ, Hamilton Health Sciences, Hamilton, ON, Canada</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Cardiovascular Rsrch Inst, Medstar Health Rsrch Inst, Washington Hospital Ctr, Washington, DC</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Inst für Medizinische Biometrie und Statistik, Lübeck, Germany</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Depts of Med &amp; Human Genetics, McGill Univ, Montreal, Canada</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Klinik und Poliklinik für Innere Medizin II, Univ Regensburg, Regensburg</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Inst of Epidemiology I, Helmholtz Zentrum München – German Rsrch Ctr for Environmental Health, Neuherberg, Klinikum Grosshadern, Munich, Inst of Med Info Science, Biometry &amp; Epidemiology, Ludwig-Maximilians-Univ München</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Institut für Klinische Molekularbiologie, Christian-Albrechts Univ, Kiel, Germany</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Cleveland Clinic, Cleveland, OH</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Inst for Translational Med &amp; Therapeutics, School of Med Univ of Pennsylvania, Philadelphia, PA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Dept of Cardiovascular Sciences, Univ of Leicester, Clinical Sciences Wing, Glenfield Hospital; Leicester Nat Inst for Health Rsrch, Biomedical Rsrch Unit in Cardiovascular Disease, Leicester, UK</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Atherogenomics Lab, Univ of Ottawa Heart Inst, Ottawa, Canada</div><div id="FN1">Address for correspondence: Robert W Davies, Cardiovascular Research Methods Centre, University of Ottawa Heart Institute, 40 Ruskin Street - H2267-A, Ottawa, Canada, K1Y 4W7, t: 613-798-5555x19124, f: 613-761-4412, <a href="mailto:dev@null" data-email="ku.ca.xo.llew@seivadwr" class="oemail">ku.ca.xo.llew@seivadwr</a>. Ruth McPherson, Atherogenomics Laboratory, University of Ottawa Heart Institute, 40 Ruskin St - H4203, Ottawa, Canada, K1Y 4W7, t: 613-761-5256, f: 613-761-4676, <a href="mailto:dev@null" data-email="ac.traehawatto@nosrehpcmr" class="oemail">ac.traehawatto@nosrehpcmr</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140435597745648_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22319020" target="pmc_ext" ref="reftype=publisher&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Circ Cardiovasc Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3335297/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental Data.</strong></div><div><a href="/pmc/articles/PMC3335297/bin/NIHMS362303-supplement-Supplemental_Data.pdf" data-ga-action="click_feat_suppl">NIHMS362303-supplement-Supplemental_Data.pdf</a><span style="color:gray"> (254K)</span></div><div class="small guid">GUID:&nbsp;1AAEA3F8-C5BD-428D-941F-04F0910FC0CA</div></div></dd></dl></div><div id="idm140435598787088" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140435598787088title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">Recent genome-wide association studies (GWAS) have identified several novel loci that reproducibly associate with CAD and/or MI risk. However, known common CAD risk variants explain only 10% of the predicted genetic heritability of the disease, suggesting that important genetic signals remain to be discovered.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods and Results</h3><p id="P2" class="p p-first-last">We performed a discovery meta-analysis of 5 GWASs involving 13,949 subjects (7123 cases, 6826 controls) imputed at approximately 5 million SNPs using pilot 1000 Genomes based haplotypes. Promising loci were followed up in an additional 5 studies with 11,032 subjects (5211 cases, 5821 controls). A novel CAD locus on chromosome 6p21.3 in the major histocompatibility complex (MHC) between <em>HCG27</em> and <em>HLA-C</em> was identified and achieved genome wide significance in the combined analysis (rs3869109; p<sub>discovery</sub>=3.3×10<sup>−7</sup>, p<sub>replication</sub>=5.3×10<sup>−4</sup> p<sub>combined</sub>=1.12×10<sup>−9</sup>). A sub-analysis combining discovery GWASs showed an attenuation of significance when stringent corrections for European population structure were employed (p=4.1×10-10 versus 3.2×10-7) suggesting the observed signal is partly confounded due to population stratification. This gene dense region plays an important role in inflammation, immunity and self cell recognition. To determine whether the underlying association was driven by MHC class I alleles, we statistically imputed common <em>HLA</em> alleles into the discovery subjects; however, no single common <em>HLA</em> type contributed significantly or fully explained the observed association.</p></div><div id="S3" class="sec sec-last"><h3 id="S3title">Conclusions</h3><p id="P3" class="p p-first-last">We have identified a novel locus in the MHC associated with CAD. MHC genes regulate inflammation and T cell responses that contribute importantly to the initiation and propagation of atherosclerosis. Further laboratory studies will be required to understand the biological basis of this association and identify the causative allele(s).</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">coronary artery disease, myocardial infarction, meta-analysis, genetics</span></div></div><div id="idm140435596708592" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">Coronary artery disease is a complex trait resulting from a combination of environmental and genetic factors. The age specific incidence of CAD is increased twofold in subjects with a family history of premature disease and the contribution of family history cannot be fully explained by known CAD risk factors.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450784">1</a></sup> The identification of genetic variants contributing to CAD may provide insight into novel biological pathways affecting atherosclerosis initiation or progression and ultimately assist in risk assessment. Several genome-wide association studies (GWAS) have identified novel loci that reproducibly associate with coronary artery disease (CAD) and/or myocardial infarction (MI) risk.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450786">2</a>–<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450769">13</a></sup> However, in aggregate, the confirmed associations explain a small proportion of known risk. In the recent report by the Coronary ARtery DIsease Genome wide Replication and Meta-analysis (CARDIoGRAM) consortium, it was estimated that 13 newly discovered and 10 previously identified loci associated with CAD account for only 10% of additive genetic heritability of the disease,<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450791">11</a></sup> which suggests that important CAD associated loci remain to be discovered.</p><p id="P5">Most GWASs have benefited from the use of imputation, a statistical technique which uses reference haplotypes from a source such as HapMap along with mathematical models to infer genotypes at single nucleotide polymorphisms (SNPs). Imputation is a reliable way to boost power in GWAS by facilitating meta-analyses of studies genotyped on different arrays and allowing for the interrogation of SNPs not directly genotyped on an array but indirectly tagged through combinations of SNPs.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450777">14</a>, <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450758">15</a></sup> However, the process of imputation is limited by both the number of reference haplotypes available and the number of SNPs present on the haplotypes. Previous GWAS for CAD have employed imputation using approximately 2.5 million SNPs from the HapMap Phase II populations. More recently, low pass whole genome sequencing by the 1000 Genomes project has allowed for reference haplotypes panels with approximately 10 million SNPs to become available. To what extent undiscovered risk variants for CAD are better tagged using these newer panels is largely unknown.</p><p id="P6">Furthermore, previous GWAS for CAD have focused solely on the consideration of a single model of inheritance, the additive model. However, recent evidence suggests that the risk imparted by CAD variants is often better modeled by a dominant or recessive mode of inheritance. In the recent CARDIoGRAM meta-analysis, of the 13 novel loci discovered, seven were better suited to either a recessive or dominant mode of inheritance, as opposed to an additive mode.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450783">11</a></sup> This suggests that considering non-additive discovery screens may be a promising method by which to boost power in GWASs.</p><p id="P7" class="p">The goal of the present study was to search for novel variants associated with CAD by performing a meta-analysis of 5 GWAS studies for CAD, making use of 1000 Genomes based imputation and considering additive, recessive and dominant modes of inheritance.</p></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Materials and Methods</h2><p id="P8" class="p p-first">Research ethics committee approval and individual informed consent was obtained for all subjects in each study cohort.</p><div id="S5" class="sec"><h3 id="S5title">Stage 1 Studies</h3><p id="P9" class="p p-first-last">The discovery analysis was comprised of the Ottawa Heart Genomics Study in collaboration with the Cleveland Clinic Gene Bank, (OHGS_A and OHGS_CCGB_B), the CAD GWAS from the Wellcome Trust Case Control Consortium (WTCCC), a subset of the INTERHEART case control study (ITH), and the Duke CATHGEN study (DUKE). Genotyping was performed using the Affymetrix 500K array (OHGS_A, WTCCC), 6.0 array (OHGS_CCGB_B, ITH) or Axiom array (DUKE). Further details on recruitment methodology and phenotypic criteria is given in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>, while brief summary phenotypic details are available in <a href="/pmc/articles/PMC3335297/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p8">Discovery population characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140435592588000" class="xtable"><table frame="void" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">N</th><th rowspan="1" colspan="1" valign="bottom" align="left">Age<sup>*</sup> (Mean± SD)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Male (%)</th><th rowspan="1" colspan="1" valign="bottom" align="left">MI (%)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Diabetics excluded</th><th rowspan="1" colspan="1" valign="bottom" align="left">Proportion new<sup>†</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">OHGS_A</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">921</td><td rowspan="1" colspan="1" valign="top" align="left">48.2±7.0</td><td rowspan="1" colspan="1" valign="top" align="left">78.1</td><td rowspan="1" colspan="1" valign="top" align="left">54.6</td><td rowspan="1" colspan="1" valign="top" align="left">Y</td><td rowspan="1" colspan="1" valign="top" align="left">6.3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">994</td><td rowspan="1" colspan="1" valign="top" align="left">74.9±4.9</td><td rowspan="1" colspan="1" valign="top" align="left">54.6</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">N</td><td rowspan="1" colspan="1" valign="top" align="left">3.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">OHGS_CCGB_B</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">2688</td><td rowspan="1" colspan="1" valign="top" align="left">49.8±7.7</td><td rowspan="1" colspan="1" valign="top" align="left">75.1</td><td rowspan="1" colspan="1" valign="top" align="left">59.8</td><td rowspan="1" colspan="1" valign="top" align="left">Y</td><td rowspan="1" colspan="1" valign="top" align="left">79.1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">1819</td><td rowspan="1" colspan="1" valign="top" align="left">74.8±5.4</td><td rowspan="1" colspan="1" valign="top" align="left">49</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">N</td><td rowspan="1" colspan="1" valign="top" align="left">76.3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DUKE</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">1200</td><td rowspan="1" colspan="1" valign="top" align="left">56.7±9.7</td><td rowspan="1" colspan="1" valign="top" align="left">69.4</td><td rowspan="1" colspan="1" valign="top" align="left">48</td><td rowspan="1" colspan="1" valign="top" align="left">Y</td><td rowspan="1" colspan="1" valign="top" align="left">100</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">648</td><td rowspan="1" colspan="1" valign="top" align="left">63.3±8.7</td><td rowspan="1" colspan="1" valign="top" align="left">42</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">Y</td><td rowspan="1" colspan="1" valign="top" align="left">100</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">WTCCC</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">1926</td><td rowspan="1" colspan="1" valign="top" align="left">49.8±7.7</td><td rowspan="1" colspan="1" valign="top" align="left">79.3</td><td rowspan="1" colspan="1" valign="top" align="left">71.5</td><td rowspan="1" colspan="1" valign="top" align="left">N</td><td rowspan="1" colspan="1" valign="top" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">2938</td><td rowspan="1" colspan="1" valign="top" align="left">N/A</td><td rowspan="1" colspan="1" valign="top" align="left">50</td><td rowspan="1" colspan="1" valign="top" align="left">N/A</td><td rowspan="1" colspan="1" valign="top" align="left">N</td><td rowspan="1" colspan="1" valign="top" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ITH</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">388</td><td rowspan="1" colspan="1" valign="top" align="left">63.3±10.6</td><td rowspan="1" colspan="1" valign="top" align="left">45.1</td><td rowspan="1" colspan="1" valign="top" align="left">100</td><td rowspan="1" colspan="1" valign="top" align="left">Y</td><td rowspan="1" colspan="1" valign="top" align="left">100</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">427</td><td rowspan="1" colspan="1" valign="top" align="left">64.3±10.3</td><td rowspan="1" colspan="1" valign="top" align="left">45.5</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">y</td><td rowspan="1" colspan="1" valign="top" align="left">100</td></tr></tbody></table></div><div id="largeobj_idm140435592588000" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3335297/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>Age refers to age at diagnosis (cases) and age at recruitment (controls)</div><div id="TFN2"><sup>†</sup>Proportion new refers to proportion which have not previously been used for a discovery genome wide association study</div></div></div></div><div id="S6" class="sec"><h3 id="S6title">Stage 1 Analysis</h3><p id="P10" class="p p-first">Removal of subjects of admixed or non-European ancestry was performed using the smartPCA program from EIGENSOFT v3.0.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450757">16</a></sup> Study subjects were processed with 270 HapMap2 subjects for PCA (90 CEU, 90 JPT+CHB, 90 YRI). In the resulting first two dimensions from PCA, k-means was used to ascertain the center of each of the CEU, JPT+CHB and YRI clusters, and the original two PC dimensions were projected onto these axes. Subjects were removed if they fell outside an oval whose major axes were 10 times the standard deviation of the CEU cluster along the two transformed axes. Results are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 1</a>.</p><p id="P11">Genotyping quality control (QC) was synchronized across studies wherever possible, leaving as few as 367,036 SNPs (OHGS_A) or as many as 648,636 SNPs (ITH) to inform imputation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Pre-imputation QC was set meeting each of a) Hardy-Weinberg Equilibrium (HWE) p- value &gt;1e-6; b) Call Rate (CR) &gt; 95% and Minor Allele Frequency (MAF) &gt;5% or CR&gt;99% and MAF&lt;5%.</p><p id="P12" class="p p-last">Imputation for each study was performed separately using IMPUTE2 using 112 available CEU haplotypes from the 1000 Genomes project (August 2009 release) as well as 298 haplotypes from a combined CEU/TSI reference panel with CEU subjects not used in the 1000 Genomes panel.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450764">14</a></sup> Post-imputation QC was set at meeting each of a) HWE p&gt;1e-6; b) CR &gt; 90%; c) INFO &gt;0.5. Post-imputation, the number of SNPs ranged from 4,101,281 SNPs (DUKE) to 5,501,436 (OHGS_CCGB_B). All analyses were adjusted for gender and the first two principal components of ancestry; age was not used due to non-overlapping case/control age distributions in several cohorts.</p></div><div id="S7" class="sec"><h3 id="S7title">Stage 1 Meta Analysis</h3><p id="P13" class="p p-first">A fixed-effects genomic-control inverse-variance meta-analysis using META (v1.2) was employed, where the standard error for each cohort was multiplied by the square root of the lambda as estimated by the genotyped SNPs for that inheritance model for that cohort.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450761">17</a></sup> Only SNPs with data from at least 2 studies and with an I<sup>2</sup> less than 50% were considered. For any locus where a SNP met p&lt;5×10<sup>−6</sup>, the SNP with the best p-value across each of the 3 models was taken as the lead SNP for the locus. A separate method of inheritance procedure was then performed to determine whether it showed additive, dominant or recessive inheritance, as outlined below</p><p id="P14" class="p p-last">SNPs were excluded for replication if they fell within 250 KBp of a locus previously reported to be associated with CAD. The exact number of SNPs for each study and model is provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 2</a>. Each SNP representing a locus was tested for replication using only its discovery determined inheritance model.</p></div><div id="S8" class="sec"><h3 id="S8title">Method of Inheritance Determination</h3><p id="P15" class="p p-first-last">Following a similar procedure to that used in CARDIoGRAM,<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450755">11</a></sup> inheritance was estimated for each top SNP as follows. Merging raw data for all discovery studies which passed QC at that SNP, a dominant and recessive model were fit, with the one giving the lower deviance chosen as the most likely method of inheritance. SNPs were then determined to be additive if the likelihood ratio test p-value for the model with both recessive and dominant effects had p&lt;0.05 compared to the recessive or dominant model alone.</p></div><div id="S9" class="sec"><h3 id="S9title">Stage 2 Studies</h3><p id="P16" class="p p-first">Confirmatory evidence was sought in five studies: German Myocardial Infarction Family Studies 1 and 2 (GerMIFS1, GerMIFS2), including KORA (Collaborative Health Research in the Region of Augsburg);<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450778">5</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450792">8</a></sup> PennCath;<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450756">4</a></sup> MedStar; and Ottawa Heart Genomics Study in collaboration with the Cleveland Clinic GeneBank <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary</a> (OHGS_CCGB_S). The latter cohort used controls from the WTCCC2<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450773">18</a></sup> not included in the WTCCC CAD analysis and cases from Ottawa and Cleveland for which genotypes and phenotypes became available after performing Stage 1 of the study. Genotyping was performed using the Affymetrix 500K (GerMIFS1) and 6.0 (GerMIFS2, PennCath, MedStar, OHGS_CCGB_S) arrays. Further descriptions of these cohorts are available in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>, while brief summary phenotypic details are available in <a href="/pmc/articles/PMC3335297/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p11">Replication population characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140435593865424" class="xtable"><table frame="void" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">N</th><th rowspan="1" colspan="1" valign="bottom" align="left">Age<sup>*</sup> (Mean± SD)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Male (%)</th><th rowspan="1" colspan="1" valign="bottom" align="left">MI (%)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Diabetics excluded</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">GerMIFS1</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">894</td><td rowspan="1" colspan="1" valign="top" align="left">50.2±7.8</td><td rowspan="1" colspan="1" valign="top" align="left">67.0</td><td rowspan="1" colspan="1" valign="top" align="left">100</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">1604</td><td rowspan="1" colspan="1" valign="top" align="left">62.6±10.0</td><td rowspan="1" colspan="1" valign="top" align="left">49.2</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GerMIFS2</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">1218</td><td rowspan="1" colspan="1" valign="top" align="left">51.4±11.9</td><td rowspan="1" colspan="1" valign="top" align="left">79.6</td><td rowspan="1" colspan="1" valign="top" align="left">100</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">1284</td><td rowspan="1" colspan="1" valign="top" align="left">51.2±11.9</td><td rowspan="1" colspan="1" valign="top" align="left">52.1</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">PennCath</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">933</td><td rowspan="1" colspan="1" valign="top" align="left">52.7±7.6</td><td rowspan="1" colspan="1" valign="top" align="left">76.3</td><td rowspan="1" colspan="1" valign="top" align="left">50.1</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">468</td><td rowspan="1" colspan="1" valign="top" align="left">61.7±9.6</td><td rowspan="1" colspan="1" valign="top" align="left">48.1</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MedStar</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">874</td><td rowspan="1" colspan="1" valign="top" align="left">48.9±6.4</td><td rowspan="1" colspan="1" valign="top" align="left">67</td><td rowspan="1" colspan="1" valign="top" align="left">48.1</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">447</td><td rowspan="1" colspan="1" valign="top" align="left">59.7±8.9</td><td rowspan="1" colspan="1" valign="top" align="left">45.4</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">OHGS_CCGB_S</td><td rowspan="1" colspan="1" valign="top" align="left">Case</td><td rowspan="1" colspan="1" valign="top" align="left">582</td><td rowspan="1" colspan="1" valign="top" align="left">54.4±8.6</td><td rowspan="1" colspan="1" valign="top" align="left">79.7</td><td rowspan="1" colspan="1" valign="top" align="left">N/A</td><td rowspan="1" colspan="1" valign="top" align="left">Y</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Control</td><td rowspan="1" colspan="1" valign="top" align="left">2728</td><td rowspan="1" colspan="1" valign="top" align="left">N/A</td><td rowspan="1" colspan="1" valign="top" align="left">52</td><td rowspan="1" colspan="1" valign="top" align="left">N/A</td><td rowspan="1" colspan="1" valign="top" align="left">N</td></tr></tbody></table></div><div id="largeobj_idm140435593865424" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3335297/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>*</sup>Age refers to age at diagnosis (cases) and age at recruitment (controls)</div></div></div><p id="P17" class="p p-last">We <em>a priori</em> defined SNPs to be significant following the second stage of analysis if they met both a Bonferroni adjusted alpha of 0.05 and if when meta-analyzed with the results from the first stage of the analysis, they achieved a genome-wide significant p-value threshold. Since three tests were employed for each SNP, and the 4,916,498 SNPs tested in this analysis correspond to no more than 548,447 semi-independent SNPs (pairwise removal of one SNP in a pair with r<sup>2</sup>&gt;0.8 using a sliding window along the genome), we defined a genome-wide significant p-value threshold at 0.05/(3×548,447)=3.0×10<sup>−8</sup>.</p></div><div id="S10" class="sec"><h3 id="S10title">Ancestry Plots</h3><p id="P18" class="p p-first-last">PCA plots which are superimposed with ancestry were made available based on subject’s self-reported grandparental ethnicity as recorded for a subset of OHGS cases. European ethnicity was divided broadly into Northern Europe (Britain, Denmark, Estonia, Iceland, Ireland, Latvia, Lithuania, Scandinavia, Scotland, Wales), Central Europe (Austria, Belgium, France, Germany, Luxembourg, Monaco, Netherlands, Switzerland), Southern Europe (Albania, Andorra, Bosnia, Croatia, Greece, Italy, Portugal, Serbia, Spain), Eastern European (Belarus, Bulgaria, Czech Republic, Hungary, Poland, Romania, Russia, Ukraine), French Canadian, Arabic, Ashkenazi Jewish, Other (specify) and Unknown. The unknown, other and mixed (&lt;3 grandparents from one ethnic region) were merged into one category.</p></div><div id="S11" class="sec"><h3 id="S11title">MHC Class I imputation</h3><p id="P19" class="p p-first">MHC Class I alleles (<em>HLA-A, HLA-B</em> and <em>HLA-C)</em> were imputed using HLA*IMP, a method similar to SNP imputation which infers <em>HLA</em> alleles using genotyped SNP data and a reference dataset with both genotyped SNPs and <em>HLA</em> alleles.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450788">19</a>, <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450766">20</a></sup> Imputation with HLA*IMP is performed using pre-defined sets of SNPs for each of several common commercial microarrays, including the Affymetrix 500K and 6.0, but not the Axiom array. For each of the discovery cohorts, SNPs surrounding the MHC were phased using IMPUTE2, with 1000 Genomes haplotypes as reference haplotypes. SNPs which were required by HLA*IMP but which failed QC and were not in the 1000 Genomes panel were substituted using their next best tagging variant, either genotyped or imputed. Imputation of <em>HLA</em> alleles for subjects genotyped with the Axiom array were generated using a combination of Axiom genotype data and imputed SNPs using the SNPs selected for the 6.0 array.</p><p id="P20" class="p p-last">Post-imputation, we used a posterior probability threshold of 0.7 to define successful <em>HLA</em> imputation of an allele, as described in the original HLA*IMP paper. Subjects were considered to have been successfully imputed if both alleles of the relevant <em>HLA</em> type met a posterior probability threshold of 0.7. As such, each Class I allele could be treated as a SNP with each subject having 0, 1 or 2 copies of the <em>HLA</em> allele. Quality control was set as being similar to SNPs with a post-imputation cutoff of HWE&gt;1×10<sup>−6</sup> and missing data rate &lt;10%. Alleles with a control frequency of less than 2% were removed, corresponding to leaving 88.4%, 81.8% and 93.8% of available genotypes for analysis for <em>HLA-A, HLA-B</em> and <em>HLA-C</em>, respectively. Posterior probability estimates suggested that the procedure was highly accurate, with most alleles achieving a median posterior estimate of &gt;0.98 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). The effect of an <em>HLA</em> allele <em>vis-à-vis</em> rs3869109 was assessed using a likelihood ratio test, by assessing a model with both rs3869109 and the relevant <em>HLA</em> allele against a model with one or the other. All analyses were performed on the entire discovery panel adjusted for gender and the first two principal components of ancestry.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">URLs</h3><p id="P21" class="p p-first-last">META v1.2 <a href="http://www.stats.ox.ac.uk/~jsliu/meta.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/~jsliu/meta.html</a>; IMPUTE v2.1.0 <a href="https://mathgen.stats.ox.ac.uk/impute/impute.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute.html</a>; SNPTEST v2.1.1 <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a>; GTOOL v0.6.1 <a href="http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html</a>; EIGENOFT v3.0 <a href="http://genepath.med.harvard.edu/~reich/Software.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genepath.med.harvard.edu/~reich/Software.htm</a>; LocusZOOM <a href="http://csg.sph.umich.locuszoom/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.locuszoom/</a>; R <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>; HLA*IMP <a href="https://oxfordhla.well.ox.ac.uk/hla/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://oxfordhla.well.ox.ac.uk/hla/</a></p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Results</h2><p id="P22" class="p p-first">A total of 13,949 subjects (7,123 cases and 6,826 controls) of European ethnicity from 5 CAD case control GWASs were included in the discovery meta-analysis (<a href="/pmc/articles/PMC3335297/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Approximately 5 million SNPs were imputed and tested for each of the three models, and inflation of test statistics was low (additive: 4,916,498 SNPs, λ=1.032; dominant: 4,889,746 SNPs, λ=1.036; recessive: 4,637,828 SNPs, λ&lt;1; see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). A total of 37 loci met the stage 1 significance threshold (p&lt;5×10<sup>−6</sup>), including 16 known CAD loci and 21 putative loci (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 2</a>). Of the lead SNPs carried forward for each of the 21 novel loci, 2 were identified by an additive model, 12 by a dominant model and 7 by a recessive model. Three of the lead SNPs were 1000 Genomes specific, with no genotyped or pre-1000 Genomes imputation reference panel with r<sup>2</sup>&gt;0.8.</p><p id="P23" class="p">Follow up was carried out in 5 additional CAD case control studies of European ethnicity with a total sample size of 11,032 (5,211 cases and 5,821 controls). We identified a single novel association with CAD at chromosome 6p21.3 in the major histocompatibility complex (MHC) which met our <em>a priori</em> significance requirements and reached a genome wide significant level in the combined analysis (discovery p=3.3×10<sup>−7</sup>, replication p= 5.3×10<sup>−4</sup> &lt; 0.05/21=2.3×10<sup>−3</sup>, global p=1.12×10<sup>−9</sup> &lt; 3.0×10<sup>−8</sup> (<a href="/pmc/articles/PMC3335297/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>); see <a href="/pmc/articles/PMC3335297/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> for more details of rs3869109 and <a href="/pmc/articles/PMC3335297/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> for the forest plot). The lead SNP, rs3869109, lies in an intergenic region between <em>HCG27</em> and <em>HLA-C.</em> The gene density of the surrounding region is quite high and the discovery signal spans a large region, as shown in <a href="/pmc/articles/PMC3335297/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>. The extended MHC, specifically the <em>HLA</em> genes, has been implicated in a wide variety of disorders, including autoimmune thyroid disease, multiple sclerosis, psoriasis, celiac disease, systemic lupus erythematosus and type 1 diabetes.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450779">22</a></sup> Results for SNPs which did not replicate are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 3</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3335297/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140435597662112" class="figure"><a href="/pmc/articles/PMC3335297/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3335297_nihms362303f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms362303f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3335297/bin/nihms362303f1.jpg"></a></div><div id="largeobj_idm140435597662112" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3335297/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3335297/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p4">Forest plot for association of rs3869109 with CAD</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3335297/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140435597657648" class="figure"><a href="/pmc/articles/PMC3335297/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3335297_nihms362303f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms362303f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3335297/bin/nihms362303f2.jpg"></a></div><div id="largeobj_idm140435597657648" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3335297/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3335297/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p5">Manhattan plot of rs3869109 and surrounding genes at 6p21.3. Local Manhattan plot generated using LocusZoom.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450768">21</a></sup> Shown is rs3869109 and flanking region. The final meta-analysis p-value is given for rs3869109, while for all other SNPs, the p-value indicated is from the discovery analysis.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p13">Association of rs3869109 with CAD</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140435553548784" class="xtable"><table frame="void" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Band</th><th rowspan="1" colspan="1" valign="top" align="left">rsID</th><th rowspan="1" colspan="1" valign="top" align="left">Nearby genes</th><th rowspan="1" colspan="1" valign="top" align="left">RAF<sup>*</sup> (RA)</th><th rowspan="1" colspan="1" valign="top" align="left">Discovery OR</th><th rowspan="1" colspan="1" valign="top" align="left">Discovery p</th><th rowspan="1" colspan="1" valign="top" align="left">Replication OR</th><th rowspan="1" colspan="1" valign="top" align="left">Replication p</th><th rowspan="1" colspan="1" valign="top" align="left">Global OR</th><th rowspan="1" colspan="1" valign="top" align="left">Global p</th><th rowspan="1" colspan="1" valign="top" align="left">Global I<sup>2</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">6p21.32</td><td rowspan="1" colspan="1" valign="top" align="left">rs3869109</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-C</em>, <em>HLA-B</em>, <em>HCG27</em></td><td rowspan="1" colspan="1" valign="top" align="left">0.55 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">1.16</td><td rowspan="1" colspan="1" valign="top" align="left">3.3×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.10</td><td rowspan="1" colspan="1" valign="top" align="left">5.28×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.14</td><td rowspan="1" colspan="1" valign="top" align="left">1.12×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0</td></tr></tbody></table></div><div id="largeobj_idm140435553548784" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3335297/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="__p14" class="p p-first-last">RAF = risk allele frequency, RA = risk allele, OR = odds ratio.</p></div><div id="TFN5"><sup>*</sup>RAF taken from OHGS_CCGB_B control cohort.</div></div></div><div id="S14" class="sec"><h3 id="S14title">Correlation with Ethnicity</h3><p id="P24" class="p p-first">Due to its role in immunological processes, the MHC, and the region surrounding <em>HLA-C</em> and <em>HLA-B</em> in particular, is under strong evolutionary pressure. Several studies attempting to determine ancestry-informative markers have noted that SNPs in the MHC are highly correlated with geographic background in European populations.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450772">23</a>, <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450770">24</a></sup> As susceptibility to CAD varies among different ethnic populations, we sought to confirm that the observed association was not a false positive driven by population stratification by combining all genotyped SNPs in discovery subjects into a single cohort and repeating PCA. Ethnicity was collected from a subset of subjects and superimposed. Results are illustrated in <a href="/pmc/articles/PMC3335297/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2</a>, and show strong agreement with past reports demonstrating that the primary ethnic difference between Europeans largely falls into a Northern versus Southern and Eastern versus Western gradient.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450787">25</a></sup>
<a href="/pmc/articles/PMC3335297/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3C</span></a> illustrates the result of predicting the fraction of rs3869109 dosage on case/control status and PC1 using linear regression, and demonstrates that there exists a slight difference among Europeans. We note that the allele frequency of the G allele of rs3869109 is 56.2 % in the CEU population (equivalent to a dosage of 1.12) and 65.2 % in the TSI population (dosage of 1.30), which closely matches these results.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3335297/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140435597653184" class="figure"><a href="/pmc/articles/PMC3335297/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3335297_nihms362303f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms362303f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3335297/bin/nihms362303f3.jpg"></a></div><div id="largeobj_idm140435597653184" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3335297/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3335297/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p6">The effect of ethnicity of rs3869109. A) A map of Europe with countries colored to reflect collected ethnic information B) First main principal component for PCA performed on SNPs genotyped in the entire discovery cohort, broadly corresponding to Northern versus Southern geographic background, with a subset colored according to ethnicity. Horizontal axis variation is uniform normal random distribution for ease of visual interpretation. See <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a> for the second main principal component and more details. C) Predicted frequency of rs3869109 in cases and controls with respect to first main principal component. This graph is aligned vertically with Panel C.</p></div></div></div><p id="P25" class="p p-last">After correcting for known within Europe ethnicity, the significance of the original result remained but was slightly attenuated, with a p-value of 4.1×10<sup>−10</sup> for the entire merged discovery cohort reduced to 6.2×10<sup>−8</sup> when adjusting for the first two PC’s. Further correcting for more subtle population structure by including the top 10 PC’s again slightly tapered the results, leading to a p-value 3.2×10<sup>−7</sup>, but nonetheless the result remained highly significant and this value was similar to the meta-analyzed value from the discovery analysis of 3.3×10<sup>−7</sup>. This indicates that the observed signal is only partly explained by confounding due to European population structure.</p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Class I MHC types</h3><p id="P26" class="p p-first-last">Association signals in intergenic regions near MHC class I genes may be due to differences in disease susceptibility risk among underlying <em>HLA</em> alleles. For example, a psoriasis association at rs10484554 was attributed to HLA-C*0602.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450753">19</a></sup> Given the proximity of rs3869109 to <em>HLA-B</em> and <em>HLA-C</em>, we imputed MHC class I alleles into the discovery subjects using HLA*IMP to determine whether the association at rs3869109 could be refined.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450752">19</a>, <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450785">20</a></sup> We were able to assign biallelic <em>HLA-A</em>, <em>HLA-B</em> and <em>HLA-C</em> genotypes to 93.3%, 93.1% and 95.4% of subjects, respectively, using the recommended posterior threshold cutoff of 0.7, which has been previously shown to predict 4 digit <em>HLA</em> alleles with &gt;96% accuracy. Not considering lower frequency <em>HLA</em> alleles (removal of &lt;2% frequency), no individual <em>HLA</em> allele appears to have a significantly greater effect than rs3869109; in fact, no common <em>HLA</em> allele significantly improved a model containing rs3869109 (min p=0.03), whereas rs3869109 added significantly to every <em>HLA</em> allele (max p=8×10<sup>−5</sup>) (<a href="/pmc/articles/PMC3335297/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). Furthermore, repeating the discovery meta-analysis correcting for rs3869109 did not indicate an association beyond rs3869109 (p&gt;1×10<sup>−5</sup>), suggesting no single underlying <em>HLA</em> allele accounted for the observed association.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3335297/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140435592590912" class="figure"><a href="/pmc/articles/PMC3335297/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3335297_nihms362303f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms362303f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3335297/bin/nihms362303f4.jpg"></a></div><div id="largeobj_idm140435592590912" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3335297/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3335297/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p7">Association of common MHC Class I alleles with CAD. Association of common (&gt;2% frequency) MHC Class I alleles with CAD in discovery population unadjusted (left) and adjusted (right) for rs3869109.</p></div></div></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Discussion</h2><p id="P27" class="p p-first">Although cholesterol accumulation in the arterial wall is a <em>sine qua non</em> for atherosclerosis, it is well accepted that atherosclerosis is driven by a chronic inflammatory process <sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450765">26</a></sup> in response to subendothelial lipoprotein retention<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450789">27</a>, <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450759">28</a></sup> and involving innate and adaptive immune responses.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450794">29</a></sup> Here, we have identified a novel CAD risk allele at 6p21.3 within the major histocompatibility locus. Relevant to the immunological origins of atherosclerosis<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450774">30</a></sup>, the observed association signal at rs3869109 spans a large region containing numerous genes in addition to <em>HLA-B</em> and <em>HLA-C</em>, many with known functions in immune mediated processes. Class II MHC molecules are expressed on antigen presenting cells including dendritic cells within the atherosclerotic plaque and present the peptide fragments of oxidized LDL to helper T cells, key participants in atherosclerosis. <sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450754">30</a></sup></p><p id="P28">Although this is the first robust GWAS signal for the MHC locus in CAD, smaller candidate gene studies have found associations with <em>MHC2TA</em> encoding the MHC class II transactivator<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450767">31</a></sup>, and genes involved in pro-inflammatory leukotriene synthesis.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450781">32</a></sup> Previous studies have also non-convincingly implicated <em>LTA</em> (lymphotoxin A), a MHC class III gene, in myocardial infarction. Ozaki <em>et al.</em> identified T26N (rs1041981) in <em>LTA</em> which lies ~350Kbp away from rs3869109.<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450793">33</a></sup> However, this association was not confirmed by other studies in European and Japanese populations.<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450763">34</a>, <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450771">35</a></sup> Of note, rs1041981 and rs3869109 are not closely linked as assessed using European (r<sup>2</sup>=0.01, D′=0.14) or Asian (r<sup>2</sup>=0.00, D′=0.01) 1000 Genomes data. Furthermore, rs1041981 is well imputed using Affymetrix 6.0 and Axiom data, and in this study shows no evidence with CAD (p=0.46).</p><p id="P29">We employed a number of novel strategies in an attempt to discover important SNP associations that may have been missed in previous large GWAS for CAD.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450776">11</a></sup> Specifically, this study marks the first attempt to use 1000 Genomes based imputation and non-additive discovery screens in a GWAS for CAD or MI. The non-additive discovery screen enabled us to identify 4 known loci which would have otherwise been missed, indicating that this technique may be an important screening tool. Although the 1000 Genomes based imputation interrogated twice as many SNPs as other recent analyses, including that of the CARDIoGRAM consortium<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450782">11</a></sup>, only three of the loci brought to replication were based on haplotypes not present in previous imputation reference sets. Thus, the utility of this approach will likely be contingent on the progressive availability of a larger number of reference haplotypes, so as to better enable imputation of lower frequency variants.</p><p id="P30">Of note, these two approaches do not appear to have contributed substantially to the novel association in the MHC locus, since rs3869109 was genotyped directly in most of the studies included in this analysis and was identified by an additive inheritance model. The fact that the MHC was identified here but not in previous studies likely reflects several factors. One can subdivide previous GWAS efforts into those which featured large scale discovery meta-analyses, most notably the recent CARDIoGRAM and C4D reports, and previous investigations. Studies published before CARDIoGRAM and C4D either featured smaller discovery populations than analyzed here, with a consequent reduction in power, or staged designs, which, although successful, can certainly miss SNPs. Consider for example the highly successful meta-analysis by the Myocardial Infarction Genomics Consortium (MIGC), which, despite identifying 5 new loci for CAD/MI, failed to identify several loci with rather large effect sizes, such as PPAP2B and ADAMTS7. In the C4D meta-analysis, it is worth noting that their European discovery population was smaller than analyzed here (8,424 versus 13,949), and while their combined South-East Asian and European discovery population was significantly larger than analyzed here, C4D was by intent a non-ethnically homogeneous population. If the association is driven by haplotypes of varying frequency among different populations, it is not surprising that it was not evident in a GWAS meta analysis including multi-ethnic populations.</p><p id="P31" class="p p-last">It is also important to note that the design of studies which made up the discovery meta-analysis in this study generally featured early-onset CAD cases with a family history of CAD whereas controls were recruited on the basis of normal angiograms and/or age greater than 70 years with no history or symptoms of vascular disease. This was a consequence of the decision to base the primary (discovery) analysis of this study on subjects for whom genotype level data was available in Ottawa. This was primarily intended to facilitate imputation and analysis in a timely and uniform fashion but also secondarily led to the aforementioned more rigorously defined CAD case/control phenotype. Consequently, replication samples were sought with similar phenotypic definitions at a sample size which would ensure replication of genuinely associated putative loci. Thus, in comparison to CARDIoGRAM, for which there exists substantial overlap in subjects considered here, a more uniform study description was used, which would be expected to decrease effect heterogeneity among studied studies. Most notably, CARDIoGRAM included approximately 60,000 out of 85,000, or two thirds of its subjects from population studies with incident disease and population (unscreened controls). Although CARDIoGRAM was highly successful, the efforts taken here in selecting more similar studies (both ethnically with North-Western Europeans and phenotypically with extreme case control designs) could be expected to decrease effect heterogeneity between studies and facilitate identification of new loci. Finally, it is worth noting that even in CARDIoGRAM, loci identified elsewhere, including LIPA locus<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450762">12</a>, <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450775">36</a></sup> and PDGFD<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450760">12</a></sup> did not reach significance.</p></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Conclusion</h2><p id="P32" class="p p-first-last">We have identified a novel association with CAD at 6p21.3 within the MHC locus. MHC genes play roles in both innate and adaptive immunity, regulating inflammation and T cell responses that contribute importantly to the initiation and propagation of atherosclerosis.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450790">29</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_258450780">30</a></sup> Further laboratory studies will be required to understand the biological basis of this association and identify the causative allele(s).</p></div><div id="S18" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemental Data</h4><div class="sup-box half_rhythm" id="d35e1045"><a href="/pmc/articles/PMC3335297/bin/NIHMS362303-supplement-Supplemental_Data.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(254K, pdf)</sup></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Acknowledgments</h2><div class="sec"><p id="__p1">We gratefully acknowledge the assistance of the staff of the University of Ottawa Information Technology Department, particularly Hana Pika, Pierre Lefebvre and Marc Charbonneau; the staff of the High Performance Computing Virtual Laboratory, particularly Dr. Peter Taillon; and the staff of the John and Jennifer Ruddy Canadian Cardiovascular Genetics Centre, particularly Heather Doelle and Melody Dallaire.</p><p id="__p2"><strong>Funding Sources:</strong> RWD is supported by a CANNeCTIN Biostatistics Fellowship. R.M. is supported by a Merck Frosst Canada/University of Ottawa Chair in Atherosclerosis Research. The Ottawa Heart Genomics Study is supported by the Canadian Institutes of Health Research #MOP82810, #MOP77682; the Canada Foundation for Innovation CFI #11966; the Heart and Stroke Foundation of Ontario #NA6001, #NA6650. <strong>OHGS_CCGB_B and OHGS_CCGB_S cases – CCGB:</strong> The Cleveland Clinic GeneBank is supported by National Institutes of Health grants P01 HL076491, P01 HL098055, R01 DK080732, R01 HL103866 and the Cleveland Clinic Clinical Research Unit of the Cleveland Clinic/Case Western Reserve University CTSA (1UL1RR024989). <strong>DUKE:</strong> WEK is supported by 1RC2-HL101612-01 and SHS is supported by R01-HL095987. <strong>ITH:</strong> A subset of INTERHEART cases and controls of European ancestry was used for this analysis. DNA samples were processed, prepared by the Population Health Research Institute and Clinical Trial Research Laboratory in Hamilton (Canada Foundation for Innovation CFI #88056), and genotyping was performed at the Ottawa Heart Institute. <strong>WTCCC-CAD and OHGS_CCGB_S controls (WTCCC2):</strong> N.J.S. holds a British Heart Foundation Chair of Cardiology and is supported by the Leicester NIHR Biomedical Research Unit in Cardiovascular Disease. This study makes use of data generated by the Wellcome Trust Case- Control Consortium. This study makes use of data generated by the Wellcome Trust Case- Control Consortium 2. A full list of the investigators who contributed to the generation of the data is available from <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk</a>. Funding for the project was provided by the Wellcome Trust under award 076113 and award 085475. <strong>GerMIFS1, GerMIFS2:</strong> Supported by the Deutsche Forschungsgemeinschaft and the German Federal Ministry of Education and Research (BMBF) in the context of the German National Genome Research Network (NGFN-2 and NGFN-plus), the FP6 and FP7 EU funded integrated projects Cardiogenics (LSHM-CT-2006-037593) and ENGAGE (201413), and the bi-national BMBF/ANR funded project CARDomics (01KU0908A). <strong>PennCath/MedStar:</strong> Supported by the Cardiovascular Institute of the University of Pennsylvania and a research grant from GlaxoSmithKline. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div></div><div id="idm140435596154208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140435596154208title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"> </p><p id="__p3" class="p p-first-last"><strong>Conflict of Interest Disclosures:</strong> Genotyping of PennCath and MedStar was supported by GlaxoSmithKline. There are no other disclosures to report.</p><p></p></div></div><div id="idm140435596152544" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140435596152544title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Lloyd-Jones DM, Nam BH, D’Agostino RB, Sr, Levy D, Murabito JM, Wang TJ, et al.  Parental cardiovascular disease as a risk factor for cardiovascular disease in middle-aged adults: a prospective study of parents and offspring. <span><span class="ref-journal">JAMA. </span>2004;<span class="ref-vol">291</span>:2204–11.</span> [<a href="/pubmed/15138242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Parental+cardiovascular+disease+as+a+risk+factor+for+cardiovascular+disease+in+middle-aged+adults:+a+prospective+study+of+parents+and+offspring&amp;author=DM+Lloyd-Jones&amp;author=BH+Nam&amp;author=RB+D’Agostino&amp;author=D+Levy&amp;author=JM+Murabito&amp;volume=291&amp;publication_year=2004&amp;pages=2204-11&amp;pmid=15138242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">McPherson R, Pertsemlidis A, Kavaslar N, Stewart A, Roberts R, Cox DR, et al.  A common allele on chromosome 9 associated with coronary heart disease. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1488–1491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2711874/">PMC free article</a>]</span> [<a href="/pubmed/17478681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+allele+on+chromosome+9+associated+with+coronary+heart+disease&amp;author=R+McPherson&amp;author=A+Pertsemlidis&amp;author=N+Kavaslar&amp;author=A+Stewart&amp;author=R+Roberts&amp;volume=316&amp;publication_year=2007&amp;pages=1488-1491&amp;pmid=17478681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Helgadottir A, Thorleifsson G, Manolescu A, Gretarsdottir S, Blondal T, Jonasdottir A, et al.  A common variant on chromosome 9p21 affects the risk of myocardial infarction. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1491–1493.</span> [<a href="/pubmed/17478679" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+on+chromosome+9p21+affects+the+risk+of+myocardial+infarction&amp;author=A+Helgadottir&amp;author=G+Thorleifsson&amp;author=A+Manolescu&amp;author=S+Gretarsdottir&amp;author=T+Blondal&amp;volume=316&amp;publication_year=2007&amp;pages=1491-1493&amp;pmid=17478679&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Reilly MP, Li M, He J, Ferguson JF, Stylianou IM, Mehta NN, et al.  Identification of ADAMTS7 as a novel locus for coronary atherosclerosis and association of ABO with myocardial infarction in the presence of coronary atherosclerosis: two genome-wide association studies. <span><span class="ref-journal">Lancet. </span>2011;<span class="ref-vol">377</span>:383–392.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3297116/">PMC free article</a>]</span> [<a href="/pubmed/21239051" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Identification+of+ADAMTS7+as+a+novel+locus+for+coronary+atherosclerosis+and+association+of+ABO+with+myocardial+infarction+in+the+presence+of+coronary+atherosclerosis:+two+genome-wide+association+studies&amp;author=MP+Reilly&amp;author=M+Li&amp;author=J+He&amp;author=JF+Ferguson&amp;author=IM+Stylianou&amp;volume=377&amp;publication_year=2011&amp;pages=383-392&amp;pmid=21239051&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Samani NJ, Erdmann J, Hall AS, Hengstenberg C, Mangino M, Mayer B, et al.  Genomewide association analysis of coronary artery disease. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:443–453.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719290/">PMC free article</a>]</span> [<a href="/pubmed/17634449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genomewide+association+analysis+of+coronary+artery+disease&amp;author=NJ+Samani&amp;author=J+Erdmann&amp;author=AS+Hall&amp;author=C+Hengstenberg&amp;author=M+Mangino&amp;volume=357&amp;publication_year=2007&amp;pages=443-453&amp;pmid=17634449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Kathiresan S, Voight BF, Purcell S, Musunuru K, Ardissino D, Mannucci PM, et al.  Genome-wide association of early-onset myocardial infarction with single nucleotide polymorphisms and copy number variants. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:334–341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681011/">PMC free article</a>]</span> [<a href="/pubmed/19198609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+of+early-onset+myocardial+infarction+with+single+nucleotide+polymorphisms+and+copy+number+variants&amp;author=S+Kathiresan&amp;author=BF+Voight&amp;author=S+Purcell&amp;author=K+Musunuru&amp;author=D+Ardissino&amp;volume=41&amp;publication_year=2009&amp;pages=334-341&amp;pmid=19198609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Erdmann J, Grosshennig A, Braund PS, Konig IR, Hengstenberg C, Hall AS, et al.  New susceptibility locus for coronary artery disease on chromosome 3q22.3. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:280– 282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695543/">PMC free article</a>]</span> [<a href="/pubmed/19198612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+susceptibility+locus+for+coronary+artery+disease+on+chromosome+3q22.3&amp;author=J+Erdmann&amp;author=A+Grosshennig&amp;author=PS+Braund&amp;author=IR+Konig&amp;author=C+Hengstenberg&amp;volume=41&amp;publication_year=2009&amp;pages=280-+282&amp;pmid=19198612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Tregouet DA, Konig IR, Erdmann J, Munteanu A, Braund PS, Hall AS, et al.  Genome-wide haplotype association study identifies the SLC22A3-LPAL2-LPA gene cluster as a risk locus for coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:283–285.</span> [<a href="/pubmed/19198611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+haplotype+association+study+identifies+the+SLC22A3-LPAL2-LPA+gene+cluster+as+a+risk+locus+for+coronary+artery+disease&amp;author=DA+Tregouet&amp;author=IR+Konig&amp;author=J+Erdmann&amp;author=A+Munteanu&amp;author=PS+Braund&amp;volume=41&amp;publication_year=2009&amp;pages=283-285&amp;pmid=19198611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Erdmann J, Willenborg C, Nahrstaedt J, Preuss M, Konig IR, Baumert J, et al.  Genome-wide association study identifies a new locus for coronary artery disease on chromosome 10p11.23. <span><span class="ref-journal">Eur Heart J. </span>2011;<span class="ref-vol">32</span>:158–168.</span> [<a href="/pubmed/21088011" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+Heart+J&amp;title=Genome-wide+association+study+identifies+a+new+locus+for+coronary+artery+disease+on+chromosome+10p11.23&amp;author=J+Erdmann&amp;author=C+Willenborg&amp;author=J+Nahrstaedt&amp;author=M+Preuss&amp;author=IR+Konig&amp;volume=32&amp;publication_year=2011&amp;pages=158-168&amp;pmid=21088011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Schunkert H, Konig IR, Kathiresan S, Reilly MP, Assimes TL, Holm H, et al.  Large-scale association analysis identifies 13 new susceptibility loci for coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:333–338.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3119261/">PMC free article</a>]</span> [<a href="/pubmed/21378990" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Large-scale+association+analysis+identifies+13+new+susceptibility+loci+for+coronary+artery+disease&amp;author=H+Schunkert&amp;author=IR+Konig&amp;author=S+Kathiresan&amp;author=MP+Reilly&amp;author=TL+Assimes&amp;volume=43&amp;publication_year=2011&amp;pages=333-338&amp;pmid=21378990&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Peden JF, Hopewell JC, Saleheen D, Chambers JC, Hager J, Soranzo N, et al.  A genome-wide association study in Europeans and South Asians identifies five new loci for coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:339–344.</span> [<a href="/pubmed/21378988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+in+Europeans+and+South+Asians+identifies+five+new+loci+for+coronary+artery+disease&amp;author=JF+Peden&amp;author=JC+Hopewell&amp;author=D+Saleheen&amp;author=JC+Chambers&amp;author=J+Hager&amp;volume=43&amp;publication_year=2011&amp;pages=339-344&amp;pmid=21378988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Wang F, Xu CQ, He Q, Cai JP, Li XC, Wang D, et al.  Genome-wide association identifies a susceptibility locus for coronary artery disease in the Chinese Han population. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:345–9.</span> [<a href="/pubmed/21378986" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+identifies+a+susceptibility+locus+for+coronary+artery+disease+in+the+Chinese+Han+population&amp;author=F+Wang&amp;author=CQ+Xu&amp;author=Q+He&amp;author=JP+Cai&amp;author=XC+Li&amp;volume=43&amp;publication_year=2011&amp;pages=345-9&amp;pmid=21378986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Barrett JC, Lee JC, Lees CW, Prescott NJ, Anderson CA, Phillips A, et al.  Genome-wide association study of ulcerative colitis identifies three new susceptibility loci, including the HNF4A region. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1330–1334.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2812019/">PMC free article</a>]</span> [<a href="/pubmed/19915572" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+ulcerative+colitis+identifies+three+new+susceptibility+loci,+including+the+HNF4A+region&amp;author=JC+Barrett&amp;author=JC+Lee&amp;author=CW+Lees&amp;author=NJ+Prescott&amp;author=CA+Anderson&amp;volume=41&amp;publication_year=2009&amp;pages=1330-1334&amp;pmid=19915572&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Dilthey A, Moutsianas L, Leslie S, McVean G. HLA*IMP - an integrated framework for imputing classical HLA alleles from SNP genotypes. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:968–972.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3065693/">PMC free article</a>]</span> [<a href="/pubmed/21300701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=HLA*IMP+-+an+integrated+framework+for+imputing+classical+HLA+alleles+from+SNP+genotypes&amp;author=A+Dilthey&amp;author=L+Moutsianas&amp;author=S+Leslie&amp;author=G+McVean&amp;volume=27&amp;publication_year=2011&amp;pages=968-972&amp;pmid=21300701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Leslie S, Donnelly P, McVean G. A statistical method for predicting classical HLA alleles from SNP data. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">82</span>:48–56.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253983/">PMC free article</a>]</span> [<a href="/pubmed/18179884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+statistical+method+for+predicting+classical+HLA+alleles+from+SNP+data&amp;author=S+Leslie&amp;author=P+Donnelly&amp;author=G+McVean&amp;volume=82&amp;publication_year=2008&amp;pages=48-56&amp;pmid=18179884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336– 2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-+2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">de Bakker PI, McVean G, Sabeti PC, Miretti MM, Green T, Marchini J, et al.  A high-resolution HLA and SNP haplotype map for disease association studies in the extended human MHC. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:1166–1172.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2670196/">PMC free article</a>]</span> [<a href="/pubmed/16998491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+high-resolution+HLA+and+SNP+haplotype+map+for+disease+association+studies+in+the+extended+human+MHC&amp;author=PI+de+Bakker&amp;author=G+McVean&amp;author=PC+Sabeti&amp;author=MM+Miretti&amp;author=T+Green&amp;volume=38&amp;publication_year=2006&amp;pages=1166-1172&amp;pmid=16998491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Paschou P, Drineas P, Lewis J, Nievergelt CM, Nickerson DA, Smith JD, et al.  Tracing sub-structure in the European American population with PCA-informative markers. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000114.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2537989/">PMC free article</a>]</span> [<a href="/pubmed/18797516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Tracing+sub-structure+in+the+European+American+population+with+PCA-informative+markers&amp;author=P+Paschou&amp;author=P+Drineas&amp;author=J+Lewis&amp;author=CM+Nievergelt&amp;author=DA+Nickerson&amp;volume=4&amp;publication_year=2008&amp;pages=e1000114&amp;pmid=18797516&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Price AL, Butler J, Patterson N, Capelli C, Pascali VL, Scarnicci F, et al.  Discerning the ancestry of European Americans in genetic association studies. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e236.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2211542/">PMC free article</a>]</span> [<a href="/pubmed/18208327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Discerning+the+ancestry+of+European+Americans+in+genetic+association+studies&amp;author=AL+Price&amp;author=J+Butler&amp;author=N+Patterson&amp;author=C+Capelli&amp;author=VL+Pascali&amp;volume=4&amp;publication_year=2008&amp;pages=e236&amp;pmid=18208327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Novembre J, Johnson T, Bryc K, Kutalik Z, Boyko AR, Auton A, et al.  Genes mirror geography within Europe. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">456</span>:98–101.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2735096/">PMC free article</a>]</span> [<a href="/pubmed/18758442" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genes+mirror+geography+within+Europe&amp;author=J+Novembre&amp;author=T+Johnson&amp;author=K+Bryc&amp;author=Z+Kutalik&amp;author=AR+Boyko&amp;volume=456&amp;publication_year=2008&amp;pages=98-101&amp;pmid=18758442&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Ross R. Atherosclerosis--an inflammatory disease. <span><span class="ref-journal">N Engl J Med. </span>1999;<span class="ref-vol">340</span>:115–126.</span> [<a href="/pubmed/9887164" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Atherosclerosis--an+inflammatory+disease&amp;author=R+Ross&amp;volume=340&amp;publication_year=1999&amp;pages=115-126&amp;pmid=9887164&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Duewell P, Kono H, Rayner KJ, Sirois CM, Vladimer G, Bauernfeind FG, et al.  NLRP3 inflammasomes are required for atherogenesis and activated by cholesterol crystals. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">464</span>:1357–1361.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2946640/">PMC free article</a>]</span> [<a href="/pubmed/20428172" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=NLRP3+inflammasomes+are+required+for+atherogenesis+and+activated+by+cholesterol+crystals&amp;author=P+Duewell&amp;author=H+Kono&amp;author=KJ+Rayner&amp;author=CM+Sirois&amp;author=G+Vladimer&amp;volume=464&amp;publication_year=2010&amp;pages=1357-1361&amp;pmid=20428172&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Tabas I, Williams KJ, Boren J. Subendothelial lipoprotein retention as the initiating process in atherosclerosis: update and therapeutic implications. <span><span class="ref-journal">Circulation. </span>2007;<span class="ref-vol">116</span>:1832–1844.</span> [<a href="/pubmed/17938300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Subendothelial+lipoprotein+retention+as+the+initiating+process+in+atherosclerosis:+update+and+therapeutic+implications&amp;author=I+Tabas&amp;author=KJ+Williams&amp;author=J+Boren&amp;volume=116&amp;publication_year=2007&amp;pages=1832-1844&amp;pmid=17938300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Hansson GK, Hermansson A. The immune system in atherosclerosis. <span><span class="ref-journal">Nat Immunol. </span>2011;<span class="ref-vol">12</span>:204–212.</span> [<a href="/pubmed/21321594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=The+immune+system+in+atherosclerosis&amp;author=GK+Hansson&amp;author=A+Hermansson&amp;volume=12&amp;publication_year=2011&amp;pages=204-212&amp;pmid=21321594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Lahoute C, Herbin O, Mallat Z, Tedgui A. Adaptive immunity in atherosclerosis: mechanisms and future therapeutic targets. <span><span class="ref-journal">Nat Rev Cardiol. </span>2011;<span class="ref-vol">8</span>:348–358.</span> [<a href="/pubmed/21502963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cardiol&amp;title=Adaptive+immunity+in+atherosclerosis:+mechanisms+and+future+therapeutic+targets&amp;author=C+Lahoute&amp;author=O+Herbin&amp;author=Z+Mallat&amp;author=A+Tedgui&amp;volume=8&amp;publication_year=2011&amp;pages=348-358&amp;pmid=21502963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Swanberg M, Lidman O, Padyukov L, Eriksson P, Akesson E, Jagodic M, et al.  MHC2TA is associated with differential MHC molecule expression and susceptibility to rheumatoid arthritis, multiple sclerosis and myocardial infarction. <span><span class="ref-journal">Nat Genet. </span>2005;<span class="ref-vol">37</span>:486–494.</span> [<a href="/pubmed/15821736" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=MHC2TA+is+associated+with+differential+MHC+molecule+expression+and+susceptibility+to+rheumatoid+arthritis,+multiple+sclerosis+and+myocardial+infarction&amp;author=M+Swanberg&amp;author=O+Lidman&amp;author=L+Padyukov&amp;author=P+Eriksson&amp;author=E+Akesson&amp;volume=37&amp;publication_year=2005&amp;pages=486-494&amp;pmid=15821736&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Dwyer JH, Allayee H, Dwyer KM, Fan J, Wu H, Mar R, et al.  Arachidonate 5-lipoxygenase promoter genotype, dietary arachidonic acid, and atherosclerosis. <span><span class="ref-journal">N Engl J Med. </span>2004;<span class="ref-vol">350</span>:29– 37.</span> [<a href="/pubmed/14702425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Arachidonate+5-lipoxygenase+promoter+genotype,+dietary+arachidonic+acid,+and+atherosclerosis&amp;author=JH+Dwyer&amp;author=H+Allayee&amp;author=KM+Dwyer&amp;author=J+Fan&amp;author=H+Wu&amp;volume=350&amp;publication_year=2004&amp;pages=29-+37&amp;pmid=14702425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Ozaki K, Ohnishi Y, Iida A, Sekine A, Yamada R, Tsunoda T, et al.  Functional SNPs in the lymphotoxin-alpha gene that are associated with susceptibility to myocardial infarction. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">32</span>:650–654.</span> [<a href="/pubmed/12426569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Functional+SNPs+in+the+lymphotoxin-alpha+gene+that+are+associated+with+susceptibility+to+myocardial+infarction&amp;author=K+Ozaki&amp;author=Y+Ohnishi&amp;author=A+Iida&amp;author=A+Sekine&amp;author=R+Yamada&amp;volume=32&amp;publication_year=2002&amp;pages=650-654&amp;pmid=12426569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Koch W, Hoppmann P, Michou E, Jung V, Pfeufer A, Mueller JC, et al.  Association of variants in the BAT1-NFKBIL1-LTA genomic region with protection against myocardial infarction in Europeans. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>:1821–1827.</span> [<a href="/pubmed/17517687" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Association+of+variants+in+the+BAT1-NFKBIL1-LTA+genomic+region+with+protection+against+myocardial+infarction+in+Europeans&amp;author=W+Koch&amp;author=P+Hoppmann&amp;author=E+Michou&amp;author=V+Jung&amp;author=A+Pfeufer&amp;volume=16&amp;publication_year=2007&amp;pages=1821-1827&amp;pmid=17517687&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Yamada A, Ichihara S, Murase Y, Kato T, Izawa H, Nagata K, et al.  Lack of association of polymorphisms of the lymphotoxin alpha gene with myocardial infarction in Japanese. <span><span class="ref-journal">J Mol Med. </span>2004;<span class="ref-vol">82</span>:477–483.</span> [<a href="/pubmed/15175864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Med&amp;title=Lack+of+association+of+polymorphisms+of+the+lymphotoxin+alpha+gene+with+myocardial+infarction+in+Japanese&amp;author=A+Yamada&amp;author=S+Ichihara&amp;author=Y+Murase&amp;author=T+Kato&amp;author=H+Izawa&amp;volume=82&amp;publication_year=2004&amp;pages=477-483&amp;pmid=15175864&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Wild PS, Zeller T, Schillert A, Szymczak S, Sinning CR, Deiseroth A, et al.  A Genome-wide Association Study Identifies LIPA as a Susceptibility Gene for Coronary Artery Disease. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2011;<span class="ref-vol">4</span>:403–412.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3157552/">PMC free article</a>]</span> [<a href="/pubmed/21606135" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=A+Genome-wide+Association+Study+Identifies+LIPA+as+a+Susceptibility+Gene+for+Coronary+Artery+Disease&amp;author=PS+Wild&amp;author=T+Zeller&amp;author=A+Schillert&amp;author=S+Szymczak&amp;author=CR+Sinning&amp;volume=4&amp;publication_year=2011&amp;pages=403-412&amp;pmid=21606135&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3335297&amp;issue-id=209258&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140435598787088title">Abstract</a></li><li><a href="#S4title">Materials and Methods</a></li><li><a href="#S13title">Results</a></li><li><a href="#S16title">Discussion</a></li><li><a href="#S17title">Conclusion</a></li><li><a href="#S18title">Supplementary Material</a></li><li><a href="#S19title">Acknowledgments</a></li><li><a href="#idm140435596154208title">Footnotes</a></li><li><a href="#idm140435596152544title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3335297/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3335297/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3335297/pdf/nihms362303.pdf">PDF (1.6M)</a></li> | <li><a href="#" data-citationid="PMC3335297" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3335297%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3335297%2F&amp;text=A%20Genome%20Wide%20Association%20Study%20for%20Coronary%20Artery%20Disease%20Identifies%20a%20Novel%20Susceptibility%20Locus%20in%20the%20Major%20Histocompatibility%20Complex" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3335297%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22319020" data-aiid="3335297" data-aid="3335297" data-iid="209258" data-domainid="319" data-domain="nihpa" data-accid="PMC3335297" data-md5="6a1a5afb133f14e8e5027f0ab5d0b8ce"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:30:36-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE8940E3DD53509100000000076E0506&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3335297%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3335297/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>