<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jid"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3080910/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Association Study Implicates PARD3B-based AIDS Restriction</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080910/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The Journal of Infectious Diseases"><meta name="citation_title" content="Genome-wide Association Study Implicates PARD3B-based AIDS Restriction"><meta name="citation_authors" content="Jennifer L. Troyer, George W. Nelson, James A. Lautenberger, Leslie Chinn, Carl McIntosh, Randall C. Johnson, Efe Sezgin, Bailey Kessing, Michael Malasky, Sher L. Hendrickson, Guan Li, Joan Pontius, Minzhong Tang, Ping An, Cheryl A. Winkler, Sophie Limou, Sigrid Le Clerc, Olivier Delaneau, Jean-François Zagury, Hanneke Schuitemaker, Daniëlle van Manen, Jay H. Bream, Edward D. Gomperts, Susan Buchbinder, James J. Goedert, Gregory D. Kirk, Stephen J. O'Brien"><meta name="citation_date" content="15 May 2011"><meta name="citation_issue" content="10"><meta name="citation_volume" content="203"><meta name="citation_firstpage" content="1491"><meta name="citation_doi" content="10.1093/infdis/jir046"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3080910/?report=abstract"><meta name="citation_pmid" content="21502085"><meta name="DC.Title" content="Genome-wide Association Study Implicates PARD3B-based AIDS Restriction"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Jennifer L. Troyer"><meta name="DC.Contributor" content="George W. Nelson"><meta name="DC.Contributor" content="James A. Lautenberger"><meta name="DC.Contributor" content="Leslie Chinn"><meta name="DC.Contributor" content="Carl McIntosh"><meta name="DC.Contributor" content="Randall C. Johnson"><meta name="DC.Contributor" content="Efe Sezgin"><meta name="DC.Contributor" content="Bailey Kessing"><meta name="DC.Contributor" content="Michael Malasky"><meta name="DC.Contributor" content="Sher L. Hendrickson"><meta name="DC.Contributor" content="Guan Li"><meta name="DC.Contributor" content="Joan Pontius"><meta name="DC.Contributor" content="Minzhong Tang"><meta name="DC.Contributor" content="Ping An"><meta name="DC.Contributor" content="Cheryl A. Winkler"><meta name="DC.Contributor" content="Sophie Limou"><meta name="DC.Contributor" content="Sigrid Le Clerc"><meta name="DC.Contributor" content="Olivier Delaneau"><meta name="DC.Contributor" content="Jean-François Zagury"><meta name="DC.Contributor" content="Hanneke Schuitemaker"><meta name="DC.Contributor" content="Daniëlle van Manen"><meta name="DC.Contributor" content="Jay H. Bream"><meta name="DC.Contributor" content="Edward D. Gomperts"><meta name="DC.Contributor" content="Susan Buchbinder"><meta name="DC.Contributor" content="James J. Goedert"><meta name="DC.Contributor" content="Gregory D. Kirk"><meta name="DC.Contributor" content="Stephen J. O'Brien"><meta name="DC.Date" content="2011 May 15"><meta name="DC.Identifier" content="10.1093/infdis/jir046"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Association Study Implicates PARD3B-based AIDS Restriction"><meta property="og:type" content="article"><meta property="og:description" content="Background. Host genetic variation influences human immunodeficiency virus (HIV) infection and progression to AIDS. Here we used clinically well-characterized subjects from 5 pretreatment HIV/AIDS cohorts for a genome-wide association study to ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080910/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jid.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3080910/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3080910/pdf/jir046.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89E03CDD522BC100000000051B034A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080910%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22 J Infect Dis%22[journal]" class="navlink">J Infect Dis</a></li><li class="accid">PMC3080910</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jid.png" alt="Logo of jid" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://academic.oup.com/jid/" target="pmc_ext" ref="reftype=publisher&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Infect Dis</a></span>. 2011 May 15; 203(10): 1491–1502. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Finfdis%2Fjir046" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/infdis/jir046</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3080910</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21502085">21502085</a></div></div></div></div><h1 class="content-title">Genome-wide Association Study Implicates <em>PARD3B</em>-based AIDS Restriction</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Troyer%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227888480848" co-class="co-affbox">Jennifer L. Troyer</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup> <a href="/pubmed/?term=Nelson%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227880897184" co-class="co-affbox">George W. Nelson</a>,<sup>1</sup> <a href="/pubmed/?term=Lautenberger%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227884932224" co-class="co-affbox">James A. Lautenberger</a>,<sup>2</sup> <a href="/pubmed/?term=Chinn%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227886508240" co-class="co-affbox">Leslie Chinn</a>,<sup>2</sup> <a href="/pubmed/?term=McIntosh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227886506016" co-class="co-affbox">Carl McIntosh</a>,<sup>2</sup> <a href="/pubmed/?term=Johnson%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227886758688" co-class="co-affbox">Randall C. Johnson</a>,<sup>1</sup> <a href="/pubmed/?term=Sezgin%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227887918400" co-class="co-affbox">Efe Sezgin</a>,<sup>2</sup> <a href="/pubmed/?term=Kessing%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227886459248" co-class="co-affbox">Bailey Kessing</a>,<sup>1</sup> <a href="/pubmed/?term=Malasky%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227889267504" co-class="co-affbox">Michael Malasky</a>,<sup>1</sup> <a href="/pubmed/?term=Hendrickson%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227889265392" co-class="co-affbox">Sher L. Hendrickson</a>,<sup>2</sup> <a href="/pubmed/?term=Li%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227879672640" co-class="co-affbox">Guan Li</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup> <a href="/pubmed/?term=Pontius%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227889819984" co-class="co-affbox">Joan Pontius</a>,<sup>1</sup> <a href="/pubmed/?term=Tang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227878311360" co-class="co-affbox">Minzhong Tang</a>,<sup>2</sup> <a href="/pubmed/?term=An%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227889408432" co-class="co-affbox">Ping An</a>,<sup>1</sup> <a href="/pubmed/?term=Winkler%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227889857408" co-class="co-affbox">Cheryl A. Winkler</a>,<sup>1</sup> <a href="/pubmed/?term=Limou%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227889855184" co-class="co-affbox">Sophie Limou</a>,<sup>3</sup> <a href="/pubmed/?term=Le%20Clerc%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227892310656" co-class="co-affbox">Sigrid Le Clerc</a>,<sup>3</sup> <a href="/pubmed/?term=Delaneau%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227892893168" co-class="co-affbox">Olivier Delaneau</a>,<sup>3</sup> <a href="/pubmed/?term=Zagury%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227890250336" co-class="co-affbox">Jean-François Zagury</a>,<sup>3</sup> <a href="/pubmed/?term=Schuitemaker%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227884438288" co-class="co-affbox">Hanneke Schuitemaker</a>,<sup>4</sup> <a href="/pubmed/?term=van%20Manen%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227884436176" co-class="co-affbox">Daniëlle van Manen</a>,<sup>4</sup> <a href="/pubmed/?term=Bream%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227884798640" co-class="co-affbox">Jay H. Bream</a>,<sup>5</sup> <a href="/pubmed/?term=Gomperts%20ED%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227889694848" co-class="co-affbox">Edward D. Gomperts</a>,<sup>6</sup> <a href="/pubmed/?term=Buchbinder%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227887394464" co-class="co-affbox">Susan Buchbinder</a>,<sup>7</sup> <a href="/pubmed/?term=Goedert%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227891396640" co-class="co-affbox">James J. Goedert</a>,<sup>8</sup> <a href="/pubmed/?term=Kirk%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227891394528" co-class="co-affbox">Gregory D. Kirk</a>,<sup>9</sup> and  <a href="/pubmed/?term=O'Brien%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085" class="affpopup" co-rid="_co_idm140227888907104" co-class="co-affbox">Stephen J. O'Brien</a><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140227888480848"><h3 class="no_margin">Jennifer L. Troyer</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=Troyer%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Jennifer L. Troyer</a></div></div><div id="_co_idm140227880897184"><h3 class="no_margin">George W. Nelson</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=Nelson%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">George W. Nelson</a></div></div><div id="_co_idm140227884932224"><h3 class="no_margin">James A. Lautenberger</h3><p><sup>2</sup>Laboratory of Genomic Diversity, National Cancer Institute-Frederick, Maryland</p><div>Find articles by <a href="/pubmed/?term=Lautenberger%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">James A. Lautenberger</a></div></div><div id="_co_idm140227886508240"><h3 class="no_margin">Leslie Chinn</h3><p><sup>2</sup>Laboratory of Genomic Diversity, National Cancer Institute-Frederick, Maryland</p><div>Find articles by <a href="/pubmed/?term=Chinn%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Leslie Chinn</a></div></div><div id="_co_idm140227886506016"><h3 class="no_margin">Carl McIntosh</h3><p><sup>2</sup>Laboratory of Genomic Diversity, National Cancer Institute-Frederick, Maryland</p><div>Find articles by <a href="/pubmed/?term=McIntosh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Carl McIntosh</a></div></div><div id="_co_idm140227886758688"><h3 class="no_margin">Randall C. Johnson</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=Johnson%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Randall C. Johnson</a></div></div><div id="_co_idm140227887918400"><h3 class="no_margin">Efe Sezgin</h3><p><sup>2</sup>Laboratory of Genomic Diversity, National Cancer Institute-Frederick, Maryland</p><div>Find articles by <a href="/pubmed/?term=Sezgin%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Efe Sezgin</a></div></div><div id="_co_idm140227886459248"><h3 class="no_margin">Bailey Kessing</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=Kessing%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Bailey Kessing</a></div></div><div id="_co_idm140227889267504"><h3 class="no_margin">Michael Malasky</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=Malasky%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Michael Malasky</a></div></div><div id="_co_idm140227889265392"><h3 class="no_margin">Sher L. Hendrickson</h3><p><sup>2</sup>Laboratory of Genomic Diversity, National Cancer Institute-Frederick, Maryland</p><div>Find articles by <a href="/pubmed/?term=Hendrickson%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Sher L. Hendrickson</a></div></div><div id="_co_idm140227879672640"><h3 class="no_margin">Guan Li</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=Li%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Guan Li</a></div></div><div id="_co_idm140227889819984"><h3 class="no_margin">Joan Pontius</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=Pontius%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Joan Pontius</a></div></div><div id="_co_idm140227878311360"><h3 class="no_margin">Minzhong Tang</h3><p><sup>2</sup>Laboratory of Genomic Diversity, National Cancer Institute-Frederick, Maryland</p><div>Find articles by <a href="/pubmed/?term=Tang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Minzhong Tang</a></div></div><div id="_co_idm140227889408432"><h3 class="no_margin">Ping An</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=An%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Ping An</a></div></div><div id="_co_idm140227889857408"><h3 class="no_margin">Cheryl A. Winkler</h3><p><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</p><div>Find articles by <a href="/pubmed/?term=Winkler%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Cheryl A. Winkler</a></div></div><div id="_co_idm140227889855184"><h3 class="no_margin">Sophie Limou</h3><p><sup>3</sup>Department of Molecular Microbiology and Immunology</p><div>Find articles by <a href="/pubmed/?term=Limou%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Sophie Limou</a></div></div><div id="_co_idm140227892310656"><h3 class="no_margin">Sigrid Le Clerc</h3><p><sup>3</sup>Department of Molecular Microbiology and Immunology</p><div>Find articles by <a href="/pubmed/?term=Le%20Clerc%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Sigrid Le Clerc</a></div></div><div id="_co_idm140227892893168"><h3 class="no_margin">Olivier Delaneau</h3><p><sup>3</sup>Department of Molecular Microbiology and Immunology</p><div>Find articles by <a href="/pubmed/?term=Delaneau%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Olivier Delaneau</a></div></div><div id="_co_idm140227890250336"><h3 class="no_margin">Jean-François Zagury</h3><p><sup>3</sup>Department of Molecular Microbiology and Immunology</p><div>Find articles by <a href="/pubmed/?term=Zagury%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Jean-François Zagury</a></div></div><div id="_co_idm140227884438288"><h3 class="no_margin">Hanneke Schuitemaker</h3><p><sup>4</sup>Department of Epidemiology, Johns Hopkins University, Bloomberg School of Public Health, Baltimore</p><div>Find articles by <a href="/pubmed/?term=Schuitemaker%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Hanneke Schuitemaker</a></div></div><div id="_co_idm140227884436176"><h3 class="no_margin">Daniëlle van Manen</h3><p><sup>4</sup>Department of Epidemiology, Johns Hopkins University, Bloomberg School of Public Health, Baltimore</p><div>Find articles by <a href="/pubmed/?term=van%20Manen%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Daniëlle van Manen</a></div></div><div id="_co_idm140227884798640"><h3 class="no_margin">Jay H. Bream</h3><p><sup>5</sup>Department of Infections and Immunoepidemiology Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Rockville, Maryland</p><div>Find articles by <a href="/pubmed/?term=Bream%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Jay H. Bream</a></div></div><div id="_co_idm140227889694848"><h3 class="no_margin">Edward D. Gomperts</h3><p><sup>6</sup>Department of Hematology, Children's Hospital Los Angeles</p><div>Find articles by <a href="/pubmed/?term=Gomperts%20ED%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Edward D. Gomperts</a></div></div><div id="_co_idm140227887394464"><h3 class="no_margin">Susan Buchbinder</h3><p><sup>7</sup>San Francisco Department of Public Health, HIV Research Section, California</p><div>Find articles by <a href="/pubmed/?term=Buchbinder%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Susan Buchbinder</a></div></div><div id="_co_idm140227891396640"><h3 class="no_margin">James J. Goedert</h3><p><sup>8</sup>Chaire de Bioinformatique, Conservatoire National des Arts et Métiers, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Goedert%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">James J. Goedert</a></div></div><div id="_co_idm140227891394528"><h3 class="no_margin">Gregory D. Kirk</h3><p><sup>9</sup>Department of Experimental Immunology, Sanquin Research, Landsteiner Laboratory and Center for Infection and Immunity Amsterdam (CINIMA), Academic Medical Center, University of Amsterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Kirk%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Gregory D. Kirk</a></div></div><div id="_co_idm140227888907104"><h3 class="no_margin">Stephen J. O'Brien</h3><p><sup>2</sup>Laboratory of Genomic Diversity, National Cancer Institute-Frederick, Maryland</p><div>Find articles by <a href="/pubmed/?term=O'Brien%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21502085">Stephen J. O'Brien</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140227884571968_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140227884571968_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140227884571968_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140227884571968_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Laboratory of Genomic Diversity, SAIC-Frederick, Inc.,</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Laboratory of Genomic Diversity, National Cancer Institute-Frederick, Maryland</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Department of Molecular Microbiology and Immunology</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Epidemiology, Johns Hopkins University, Bloomberg School of Public Health, Baltimore</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Infections and Immunoepidemiology Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Rockville, Maryland</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Hematology, Children's Hospital Los Angeles</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>San Francisco Department of Public Health, HIV Research Section, California</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Chaire de Bioinformatique, Conservatoire National des Arts et Métiers, Paris, France</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Department of Experimental Immunology, Sanquin Research, Landsteiner Laboratory and Center for Infection and Immunity Amsterdam (CINIMA), Academic Medical Center, University of Amsterdam, the Netherlands</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="idm140227881505904">Correspondence: Jennifer L. Troyer, MD, Laboratory of Genomic Diversity, Bldg 560, Rm 21-56, NCI-Frederick, Frederick, MD 21702 (<a href="mailto:dev@null" data-email="vog.hin.liam@jreyort" class="oemail">vog.hin.liam@jreyort</a>).</div><div id="idm140227881504880">Potential conflicts of interest: none reported.</div><div id="idm140227881504256">Presented in part: Conference on Retroviruses and Opportunistic Infections, San Francisco, California, 15–19 February 2010. Poster board #458.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140227884571968_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Sep 30; Accepted 2010 Dec 22.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140227884571968_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> Published by Oxford University Press on behalf of the Infectious Diseases Society of America 2011.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3080910/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140227925194480" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140227925194480title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first"><strong><em>Background.</em></strong> Host genetic variation influences human immunodeficiency virus (HIV) infection and progression to AIDS. Here we used clinically well-characterized subjects from 5 pretreatment HIV/AIDS cohorts for a genome-wide association study to identify gene associations with rate of AIDS progression.</p><p id="__p4"><strong><em>Methods.</em></strong>  European American HIV seroconverters <em>(n</em> = 755) were interrogated for single-nucleotide polymorphisms (SNPs) <em>(n</em> = 700,022) associated with progression to AIDS 1987 (Cox proportional hazards regression analysis, co-dominant model).</p><p id="__p5"><strong><em>Results.</em></strong>  Association with slower progression was observed for SNPs in the gene <em>PARD3B</em>. One of these, rs11884476, reached genome-wide significance (relative hazard = 0.3; <em>P</em> =3. 370 × 10<sup>−9</sup>) after statistical correction for 700,022 SNPs and contributes 4.52% of the overall variance in AIDS progression in this study. Nine of the top-ranked SNPs define a PARD3B haplotype that also displays significant association with progression to AIDS (hazard ratio, 0.3; <em>P</em> = 3.220 × 10<sup>−8</sup>). One of these SNPs, rs10185378, is a predicted exonic splicing enhancer; significant alteration in the expression profile of <em>PARD3B</em> splicing transcripts was observed in B cell lines with alternate rs10185378 genotypes. This SNP was typed in European cohorts of rapid progressors and was found to be protective for AIDS 1993 definition (odds ratio, 0.43, <em>P</em> = .025).</p><p id="__p6" class="p p-last"><strong><em>Conclusions</em>.</strong> These observations suggest a potential unsuspected pathway of host genetic influence on the dynamics of AIDS progression.</p></div></div><div id="idm140227876714784" class="tsec sec headless whole_rhythm"><p id="__p7" class="p p-first">AIDS has taken over 30 million lives in the 30 years since it first emerged as a cluster of rare infections and cancers among men who have sex with men (MSM) and recipients of human immunodeficiency virus (HIV)–contaminated blood products in the early 1980s. HIV has evaded most public health prevention campaigns, and efforts to develop a vaccine have been largely unsuccessful. Highly active antiretroviral therapy (HAART) has been developed, which delays progressive immune deficiency and improves survival. However, HAART does not clear HIV infection, can be difficult to tolerate, and occasionally elicits serious adverse reactions [<a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684932">1</a>]. Genetic association studies of candidate genes have revealed several common genetic variants that modulate AIDS progression, including <em>CCR5-Δ32</em>, <em>HLA</em>-homozygosity, <em>HLA-B27</em> and <em>-B57</em>, <em>IL10-5'A</em>, <em>MYH9</em>, and <em>KIR</em> variants[<a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684921">2</a>–<a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_356549127">6</a>]. One of these AIDS Restriction Genes (<em>CCR5-Δ32</em>) encouraged the pharmaceutical development of HIV entry inhibitors, a new class of anti-HIV drugs[<a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684922">7</a>–<a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684947">9</a>] and, in one case, was employed in a successful bone marrow transplantation to a patient with AIDS who remained clear of HIV-1 for &gt;2 years [<a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684935">10</a>].</p><p id="__p8">There are specific challenges to genome-wide association studies with susceptibility and resistance to infectious diseases generally and HIV/AIDS associations in particular. Because genetic variation is not causative, but rather modulates the host's response to an infectious agent, effects are expected to be moderate. In addition, operative protective or susceptible alleles may be rare in a population. Finally, few clinically well-defined pretreatment HIV/AIDS cohorts exist, and clinical outcomes and progression measures are varied. Therefore, power for any one test is limited. Nevertheless, a number of genome-wide association studies (GWAS) have been applied with some success to AIDS cohorts [<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684930">11</a>–<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684940">15</a>]. Genome-wide significant findings (<em>P</em> &lt; 5 × 10<sup>−8</sup>) have emphasized the importance of the human leukocyte antigen (HLA) region in viral load (VL) set point and progression. There have also been a few intriguing suggestions of new associations with AIDS progression that approach but do not achieve a strict Bonferroni correction for significance (with <em>P</em> values in the range of 10<sup>−5</sup> to 10<sup>−7</sup>), replicate in additional cohorts, and have as-yet-unexplored biological or functional links to AIDS. Most of these studies have been limited to small numbers of patients (ranging from &lt;50 to a few hundred) with extreme and well-defined categorical progression phenotypes (for instance, rapid and slow progressors [<a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684915">14</a>–<a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_323490287">18</a>]). To target as-yet-undiscovered associations, we have interrogated a different and larger set of SNPs (those present on the Affymetrix 6.0 genotyping array) and have taken advantage of the detailed clinical data from 6 prospective American AIDS cohorts, previously used for discovery of 32 candidate gene associations with HIV acquisition, AIDS progression, AIDS defining condition, or HAART efficacy [<a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684936">3</a>, <a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684945">19</a>]. Subjects (a total of 1526 European American [EA] and 437 African American [AA] study participants; <a href="http://jid.oxfordjournals.org/cgi/content/full/jir046/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">supplemental Table 1</a>) were genotyped for 934,968 SNP variants represented on the Affymetrix 6.0 genotyping array; after quality control (QC) filters that screen for SNP veracity and utility, 700,022 SNPs were analyzed. Here, we present association with progression to AIDS 1987 (Centers for Disease Control and Prevention [CDC] definition[<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_323490285">20</a>]), a complex but clinically relevant phenotype, in all of the successfully genotyped EA seroconverters (755 total subjects).</p><p id="__p11" class="p">After correction for multiple SNPs, we found a statistically significant signal for a single SNP within the <em>PARD3B</em> gene region on Chromosome 2. Additional linked SNPs in the region approached genome-wide significance (<em>P</em> values of 10<sup>−5</sup> to 10<sup>−8</sup>) and define a protective haplotype (odds ratio [OR], 0.3; <em>P</em> = 3.2 × 10<sup>−8</sup>). One of these SNPs was assessed for functional consequence by quantification of <em>PARD3B</em> transcripts (spliced and unspliced variants) in lymphoblastoid cell lines with alternative <em>PARD3B</em> SNP genotypes and was interrogated in additional cohorts for association with AIDS (1993 definition [<a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684919">21</a>]).</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">METHODS</h2><div id="__sec2" class="sec sec-first"><h3 id="__sec2title">Subjects</h3><p id="__p12" class="p p-first-last">EA and AA subjects from 6 cohorts—Multicenter AIDS Cohort Study (MACS, principal investigator [PI] J. Phair), Multicenter Hemophilia Cohort Study (MHCS, PI J. Goedert), D.C. Gays (DCG, PI J. Goedert), Hemophilia Growth and Development Study (HGDS, PI E. Gomperts), San Francisco City Clinic Study (SFCC, PI S. Buchbinder), and AIDS linked to Intravenous Experience (ALIVE, PI G. Kirk) —were selected for GWAS on the basis of the clinical relevance and availability of high-quality high–molecular weight DNA. Seroconverters, defined as subjects who entered their respective studies with a seronegative HIV status and had an interval of ≤3 years between their last negative and first positive HIV test result, were available from 5 cohorts (EA: 405 from MACS, 222 from MHCS, 40 from DCG, 75 from SFCC, and 13 from ALIVE; AA: 39 from MACS, 9 from MHCS, 2 from DCG, 6 from SFCC, and 226 from ALIVE); seroconversion date was set as the midpoint between last negative and first positive HIV test result.</p></div><div id="__sec3" class="sec"><h3 id="__sec3title">Genotyping</h3><p id="__p13" class="p p-first-last">DNA was extracted from lymphoblast cell lines[<a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684918">22</a>] and run on the Affymetrix genome-wide human SNP array 6.0 genotyping platform. DNA (325 ng/sample) was prepared for both Sty1 and Nsp1 restriction enzyme digestion for this assay, an increase of 125 ng over Affymetrix recommendations, which greatly increased our success rate. The remainder of the assay was performed as per manufacturer's instructions.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Sample QC</h3><p id="__p14" class="p p-first">First-pass QC of samples was performed in Genotyping Console 3.0.2. All samples that passed contrast QC (CQC) (CQC&gt;.4) were genotyped for 934,968 SNPs using Affymetrix Power Tools (apt-probeset-genotype 1.10.0). Samples were genotyped by batch (200–500 samples), and individuals with &lt;90% call rates were excluded from further analysis. Four controls were run a total of 39 times, and 145 duplicate samples and 8 Centre d'Etude du Polymorphisme Humain (CEPH) parent-offspring trios were run within and between plates and verified for concordance across runs (concordances ≥99%). Once acceptable genotyping standards were met, Affymetrix genotypes were compared with previously produced Taqman, Illumina, and/or Perlegen genotypes at a mean of 85 (range, 10–142) sites per sample. Samples with &lt;90% concordance with their previous genotypes were discarded. Affymetrix sex calls (based on Y chromosome markers, X chromosome heterozygosity, and the intensity of the sex chromosome invariant probes) were compared with clinical data files, and inconsistent individuals were discarded.</p><p id="__p15" class="p p-last">PLINK[<a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684913">23</a>] was used to assess cryptic identity and cryptic relatedness and to verify sex calls. Identical subjects were investigated and discarded (if clinical data did not match) or were reduced to a single genotype file (if clinical data, including birth date, matched). Related subjects were flagged.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">SNP QC</h3><p id="__p16" class="p p-first-last">Individuals that passed subject QC were re-clustered and interrogated for all 934,968 SNPs as a single batch. SNPs were then filtered on the basis of multiple criteria. Autosomal perfect-match SNPs supported by Affymetrix <em>(n</em> = 868,157) were evaluated for Mendelian inheritance in CEPH trios (389 dropped), SNP call rates of &gt;95% (39,220 dropped), Hardy Weinburg equilibrium (HWE; <em>P</em> &gt; .001; 6512 dropped), and minor allele frequency &gt;0.01 (121,403 dropped), resulting in a total of 700,022 SNPs for association analyses in EAs.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Population Structure</h3><p id="__p17" class="p p-first">Population structure was assessed using the Principal Components Analysis (PCA) module of <em>Eigensoft</em> software[<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684931">24</a>]. EA and AA subjects segregated with no discrepancies between clinical and genetic assignment. Within EA subjects, the top 10 most significant eigenvectors were calculated, and the top 2 were plotted (data not shown). The distribution of EA study subjects along the top 2 eigenvectors demonstrated a topology that mirrored that seen in several other studies of the European population structure[<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684954">25</a>], including a cline that probably reflects Northern European to Southern European descent and a cluster that most likely represents Ashkenazi Jews in our cohorts. AIDS outcomes were not significantly associated with population substructure. We also corrected for population structure with the genomic control method [<a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684927">26</a>]. The λ value was 1.01, and a λ correction did not alter the association results significantly. Both the PCA and genomic control analyses suggested that population stratification was not a confounding factor in our HIV progression analyses.</p><p id="__p18" class="p">All SNPs that contributed to population structure (<em>P</em> &lt; 10<sup>−7</sup>) were flagged in subsequent tests (displayed as an “X” in the Manhattan plot; <a href="/pmc/articles/PMC3080910/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a><em>B</em>). For all SNPs that indicated a potential association, the first 2 eigenvectors were used as covariates in the applicable tests, and <em>P</em> values were adjusted accordingly. None of the SNPs presented in this manuscript demonstrated significant contribution to population structure, and correction for structure did not alter the <em>P</em> values significantly.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3080910/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140227885039248" class="figure"><a href="/pmc/articles/PMC3080910/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3080910_infdisjir046f01_3c.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is infdisjir046f01_3c.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3080910/bin/infdisjir046f01_3c.jpg"></a></div><div id="largeobj_idm140227885039248" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3080910/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3080910/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p19"><em>A,</em> Quantile-quantile plot for the expected versus observed <em>P</em> values for this study. λ is the slope of the first 95% of the data. <em>B,</em> Manhattan plot showing the distribution of –log<sub>10</sub> (<em>P</em> values) for time to AIDS87 (co-dominant model) along the autosomes for 700,022 single-nucleotide polymorphisms (SNPs). Each chromosome is represented by a different color and numbered sequentially along the x-axis. SNPs are ordered by chromosomal position. Values for SNPs that contributed significantly to population sub-structure are designated with an X rather than a dot. The horizontal black line marks the level of genome-wide significance at the .01 significance level (<em>P</em> = 1.370 × 10<sup>−8</sup>).</p></div></div></div></div><div id="__sec7" class="sec"><h3 id="__sec7title">Association Analysis</h3><p id="__p20" class="p p-first">Cox model analysis was performed for all SNPs passing the QC filters, using an additive model taking the number of copies of the rare allele, 0, 1 or 2, as the explanatory variable. The analysis used PROC PHREG in Statistical Analysis Software (SAS) [<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684950">27</a>]; the significance was evaluated using the log likelihood test. We performed a strict Bonferroni correction for the 700,022 SNPs used in the GWAS[<a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684923">28</a>–<a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684908">30</a>. All SNPs identified as interesting by statistical analysis were examined visually for quality assurance. SNP rs11884476 was further investigated to calculate explained fraction (EF) [<a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684928">3</a>, <a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684917">31</a>, <a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684907">32</a>] using the SAS-based program, SUREV[<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684938">27</a>]. Haplotypes were inferred by implementation of an expectation-maximization algorithm.</p><p id="__p21" class="p p-last">Both rapid progressors (AIDS, by the CDC 1993 definition, within 3 years of seroconversion) and slow progressors (CD4+ cell count &gt;500 cells/mL and no clinical symptoms of AIDS at least 8 years after seroconversion) from the French GRIV cohort were typed using the Illumina HumanHap300 genotyping beadchips [<a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684939">14</a>, <a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684912">15</a>], as were rapid progressors and controls in the Amsterdam Cohort Study (ACS<a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684951">[33])</a>. 376 Dutch and 695 French general population seronegative controls were typed on the same platform. One SNP in the PARD3B region of significance was the same on both Illumina and Affymetrix platforms (rs10185378) and was tested as a candidate SNP in a categorical case-control analysis in a combined set of GRIV and ACS rapid progressors <em>(n</em> = 123) versus HIV-negative control subjects of similar ethnicity <em>(n</em> = 1071). Eigenstrat, implemented in Eigensoft, was used to identify outliers in the population who were subsequently removed from the analysis. The first 2 eigenvectors were used as covariates to correct for any remaining population structure. An attempt was made to impute the other 8 SNPs presented here using IMPUTE 2 [<a href="#bib34" rid="bib34" class=" bibr popnode">34</a>]. However, these SNPs could not be imputed with high confidence in all of the data; 8%–26% of the samples had &lt;90% confidence for genotype calls, with most SNPs missing ∼20% of the calls. Minor alleles were absent from most of the imputed data, so statistics could not be reliably performed for these SNPs.</p></div><div id="__sec8" class="sec sec-last"><h3 id="__sec8title">RNA Isolation and Quantification of <em>PARD3B</em> Expression</h3><p id="__p22" class="p p-first-last">RNA was extracted from lymphoblastoid cell lines of individuals with alternative <em>PARD3B</em> genotypes using the RNeasy Mini kit (Qiagen) and quantified using a NanoDrop spectrophotometer (NanoDrop Technologies). First-strand complementary DNA synthesis was performed using SuperScript III (Life Technologies), according to the manufacturer's instructions. Gene expression was measured by quantitative reverse-transcription polymerase chain reaction (PCR) on a LightCycler 480 (Roche Applied Science). Primers were designed to span the junction between <em>PARD3B</em> exons 19 and 20 (isoforms B and C) or exons 19 and 21 (isoform A) and included 1 forward primer (5′-AAACATGGTGGCCTGAGAGA-3′) and 2 reverse isoform-specific primers (isoform A: 5′-CGTAGGACGGCCACTTTC-3′, isoforms B and C: 5′-TGATGTTTTGCTCCAATCCTT-3′). The FAM-labeled probe used to detect both <em>PARD3B</em> isoforms was Universal ProbeLibrary probe 78 (Roche Applied Science). Quantification of <em>PARD3B</em> expression was performed in multiplex reactions with the human β-actin reference gene assay (Roche Applied Science). Each reaction contained 1× Light Cycler Master, 100 nM each probe, and 200 nM each primer, in a final volume of 10 μL. Reactions were performed in triplicate. PCR conditions were 5 min at 95°C, followed by 45 cycles of 10 s at 95°C, 20 s at 60°C, and 1 s at 72°C. <em>PARD3B</em> isoform expression was normalized to β-actin expression for each reaction, and samples were excluded if the coefficient of variation was &gt;.04 for the triplicate. The relative effect of genotype at rs10185378 on differential splicing was tested by linear regression using R, version 2.11.1, functions lm, and estimable (in library gmodels).</p></div></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">RESULTS</h2><p id="__p23" class="p p-first">We present data from 700,022 SNPS that passed QC filters (see Methods) in the 755 EA seroconverters from 5 HIV/AIDS natural history cohorts. Of 803 EA seroconverters from 5 cohorts, 24 failed Affymetrix QC or the initial call rate cutoff of 90%, 5 were excluded for inconsistencies with their clinical file, and 15 were excluded due to sample mix-up (cryptic identity or discordance with previous genotypes). Once these 759 subjects were re-clustered and recalled; all but 4 had call rates of &gt;95%, with an average call rate of 98.8%. When those 4 subjects and SNPS failing filters were removed from the analysis, leaving the 755 seroconverters and 700,022 SNPs considered here, average call rate increased to 99.3%.</p><p id="__p24">A test for progression to the 1987 definition of AIDS<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_323490286">[20]</a> using Cox model survival analysis revealed departure of multiple SNPs from expectation (<a href="/pmc/articles/PMC3080910/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a><em>A</em> and 1<em>B</em> ) including 1 SNP, rs11884476 on chromosome 2, position 206,026,838, that reached genome-wide significance (<em>P</em> = 3.370 × 10<sup>−9</sup>; Figure 1<em>B</em>). The minor allele was low frequency in our population (0.063) and protective, with a relative hazard of 0.3 (<a href="/pmc/articles/PMC3080910/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a><em>A</em>). When the study cohort was subdivided on the basis of risk group, MSM and hemophiliacs displayed parallel significant associations (for MSM, relative hazard (RH) = 0.43 [<em>P</em> = .0024]; for hemophiliacs, RH = 0.3 [<em>P</em> = .0012]; Figures <em>2B</em> and <em>2C</em>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC3080910/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140227892516112" class="figure"><a href="/pmc/articles/PMC3080910/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3080910_infdisjir046f02_3c.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is infdisjir046f02_3c.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3080910/bin/infdisjir046f02_3c.jpg"></a></div><div id="largeobj_idm140227892516112" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3080910/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC3080910/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p25"> Kaplan-Meier survival curve, demonstrating the co-dominant effect of rs11884476 genotype on time to AIDS87 for all 755 seroconverters (<em>A</em>), 479 men who have sex with men (MSM; <em>B</em>), and 221 hemophiliacs (<em>C</em>).</p></div></div></div><p id="__p26">The protective SNP is located near exon 20 of the <em>PARD3B</em> gene region, which is a homolog of a cell polarity gene identified in <em>Caenorhobditis elegans</em> that is associated with tight junctions in <em>C. elegans</em> and human cells<a href="#bib34" rid="bib34" class=" bibr popnode">[35]</a>. To quantify the influence of the <em>PARD3B</em> genotype on AIDS progression, we calculated the EF of progression to AIDS 1987[<a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684925">3</a>, <a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684920">31</a>, <a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684937">32</a>] for <em>PARD3B</em> rs11884476 along with 13 genetic factors previously identified as affecting AIDS progression (<a href="/pmc/articles/PMC3080910/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). When compared with the 13 previously described AIDS restriction genes re-assessed in this group of subjects, <em>PARD3B</em> factor rs11884476 had the largest EF, 4.52%. The overall EF for all 14 factors considered together was 15.6%.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p9">Explained Fraction of AIDS Restriction Genes (ARGs) for Progression to AIDS in 755 European American Seroconverter Subjects</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140227884734960" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Locus</td><td rowspan="1" colspan="1" align="center">Variant</td><td rowspan="1" colspan="1" align="center">Model</td><td rowspan="1" colspan="1" align="center">Frequency, %</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center">RH</td><td rowspan="1" colspan="1" align="center">EF (%)</td></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>CCR5</em></td><td rowspan="1" colspan="1">Δ32</td><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1" align="center">20.1</td><td rowspan="1" colspan="1" align="center">.038</td><td rowspan="1" colspan="1" align="center">0.744</td><td rowspan="1" colspan="1" align="center">0.56</td></tr><tr><td rowspan="1" colspan="1"><em>CCR2</em></td><td rowspan="1" colspan="1">64I</td><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1" align="center">18.0</td><td rowspan="1" colspan="1" align="center">.025</td><td rowspan="1" colspan="1" align="center">0.723</td><td rowspan="1" colspan="1" align="center">0.6</td></tr><tr><td rowspan="1" colspan="1"><em>SDF1</em></td><td rowspan="1" colspan="1">3'A</td><td rowspan="1" colspan="1">R</td><td rowspan="1" colspan="1" align="center">3.4</td><td rowspan="1" colspan="1" align="center">.061</td><td rowspan="1" colspan="1" align="center">0.488</td><td rowspan="1" colspan="1" align="center">0.5</td></tr><tr><td rowspan="1" colspan="1"><em>CCR5</em></td><td rowspan="1" colspan="1">+/P1/+ haplotype</td><td rowspan="1" colspan="1">R</td><td rowspan="1" colspan="1" align="center">12.4</td><td rowspan="1" colspan="1" align="center">.001</td><td rowspan="1" colspan="1" align="center">1.667</td><td rowspan="1" colspan="1" align="center">1.18</td></tr><tr><td rowspan="1" colspan="1"><em>IL10</em></td><td rowspan="1" colspan="1">5'A</td><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1" align="center">41.9</td><td rowspan="1" colspan="1" align="center">.170</td><td rowspan="1" colspan="1" align="center">1.165</td><td rowspan="1" colspan="1" align="center">0.2</td></tr><tr><td rowspan="1" colspan="1"><em>HLA B</em></td><td rowspan="1" colspan="1">B35Px</td><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1" align="center">8.4</td><td rowspan="1" colspan="1" align="center">&lt;.001</td><td rowspan="1" colspan="1" align="center">2.365</td><td rowspan="1" colspan="1" align="center">2.63</td></tr><tr><td rowspan="1" colspan="1"><em>HLA B</em></td><td rowspan="1" colspan="1">B27</td><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1" align="center">9.8</td><td rowspan="1" colspan="1" align="center">.002</td><td rowspan="1" colspan="1" align="center">0.513</td><td rowspan="1" colspan="1" align="center">1.65</td></tr><tr><td rowspan="1" colspan="1"><em>HLA B</em></td><td rowspan="1" colspan="1">B57</td><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1" align="center">8.5</td><td rowspan="1" colspan="1" align="center">.033</td><td rowspan="1" colspan="1" align="center">0.559</td><td rowspan="1" colspan="1" align="center">0.86</td></tr><tr><td rowspan="1" colspan="1"><em>HLA A, B, C</em></td><td rowspan="1" colspan="1">homozygosity at 1 gene</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1" align="center">22.6</td><td rowspan="1" colspan="1" align="center">.664</td><td rowspan="1" colspan="1" align="center">1.067</td><td rowspan="1" colspan="1" align="center">0.03</td></tr><tr><td rowspan="1" colspan="1"><em>HLA A, B, C</em></td><td rowspan="1" colspan="1">homozygosity at 2 or 3 genes</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1" align="center">6.5</td><td rowspan="1" colspan="1" align="center">.001</td><td rowspan="1" colspan="1" align="center">2.019</td><td rowspan="1" colspan="1" align="center">1.02</td></tr><tr><td rowspan="1" colspan="1"><em>RANTES</em></td><td rowspan="1" colspan="1">Haplotype 3 or 5</td><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1" align="center">5.2</td><td rowspan="1" colspan="1" align="center">.004</td><td rowspan="1" colspan="1" align="center">1.813</td><td rowspan="1" colspan="1" align="center">0.84</td></tr><tr><td rowspan="1" colspan="1"><em>KIR</em></td><td rowspan="1" colspan="1">3DS1</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">21.8</td><td rowspan="1" colspan="1" align="center">&gt;.990</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0</td></tr><tr><td rowspan="1" colspan="1"><em>KIR3DS1-HLA Bw480I</em></td><td rowspan="1" colspan="1">epistasis</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1" align="center">.013</td><td rowspan="1" colspan="1" align="center">0.623</td><td rowspan="1" colspan="1" align="center">1.01</td></tr><tr><td rowspan="1" colspan="1"><em>PARD3B</em></td><td rowspan="1" colspan="1">G at rs11884476</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">6.3</td><td rowspan="1" colspan="1" align="center">3.370 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.29</td><td rowspan="1" colspan="1" align="center">4.52</td></tr><tr><td rowspan="1" colspan="1">Combined ARGs</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">15.6</td></tr></tbody></table></div><div id="largeobj_idm140227884734960" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3080910/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140227878207120"><p id="__p10" class="p p-first-last"><strong>NOTE.</strong> RH, relative hazard.</p></div></div></div><p id="__p27">Eight additional SNPs clustered around the exon 20 region of <em>PARD3B</em> were among the top 10 hits, with <em>P</em> ≤ 1.9 × 10<sup>−06</sup> and a protective effect (<a href="/pmc/articles/PMC3080910/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>; Figures 1<em>B</em> and <em>3A</em>). Additionally, a total of 14 <em>PARD3B</em> SNPs fell within the lowest 1% of GWAS <em>P</em> values (data not shown). The 9 highest-ranking PARD3B SNPs, including the top 5 hits overall, were all included in a single protective haplotype within <em>PARD3B</em> (<em>P</em> = 3.2 × 10<sup>−8</sup>; hazard ratio [HR], 0.29; Figure <em>3A</em>). These signals did not replicate in a cohort of 282 AA seroconverters.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p28">Twenty Lowest <em>P</em> Values for Survival Analysis to AIDS-1987, Co-dominant Model</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140227882517184" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Rank</td><td rowspan="1" colspan="1" align="center">rs number</td><td rowspan="1" colspan="1" align="center">chr</td><td rowspan="1" colspan="1" align="center">Position</td><td rowspan="1" colspan="1" align="center">Gene</td><td rowspan="1" colspan="1" align="center">MAF</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center">HR<sup>b</sup></td></tr></thead><tbody><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs11884476</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">206,026,838</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0631</td><td rowspan="1" colspan="1" align="center">3.370E-09</td><td rowspan="1" colspan="1" align="center">0.301</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs11896375</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">206,015,793</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0642</td><td rowspan="1" colspan="1" align="center">4.174E-08</td><td rowspan="1" colspan="1" align="center">0.347</td></tr><tr><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">rs10185378</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">206,013,401</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0609</td><td rowspan="1" colspan="1" align="center">9.234E-08</td><td rowspan="1" colspan="1" align="center">0.349</td></tr><tr><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">rs10210768</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">205,998,621</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0689</td><td rowspan="1" colspan="1" align="center">1.585E-07</td><td rowspan="1" colspan="1" align="center">0.387</td></tr><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">rs11886997</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">206,039,636</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0663</td><td rowspan="1" colspan="1" align="center">4.496E-07</td><td rowspan="1" colspan="1" align="center">0.388</td></tr><tr><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs13410658</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">206,050,327</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0589</td><td rowspan="1" colspan="1" align="center">5.143E-07</td><td rowspan="1" colspan="1" align="center">0.362</td></tr><tr><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">rs6467710</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">136,854,359</td><td rowspan="1" colspan="1">DGKI</td><td rowspan="1" colspan="1" align="center">.0159</td><td rowspan="1" colspan="1" align="center">5.596E-07</td><td rowspan="1" colspan="1" align="center">4.302</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs880343</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">206,023,936</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0675</td><td rowspan="1" colspan="1" align="center">1.134E-06</td><td rowspan="1" colspan="1" align="center">0.416</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs11239930</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">145,027,188</td><td rowspan="1" colspan="1">NBPF14</td><td rowspan="1" colspan="1" align="center">.4184</td><td rowspan="1" colspan="1" align="center">1.556E-06</td><td rowspan="1" colspan="1" align="center">0.681</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs7565154</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">206,034,837</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0742</td><td rowspan="1" colspan="1" align="center">1.923E-06</td><td rowspan="1" colspan="1" align="center">0.443</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs7217319</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">38,924</td><td rowspan="1" colspan="1">RPH3AL</td><td rowspan="1" colspan="1" align="center">.3714</td><td rowspan="1" colspan="1" align="center">4.482E-06</td><td rowspan="1" colspan="1" align="center">0.677</td></tr><tr><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">rs6441975</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">46,403,194</td><td rowspan="1" colspan="1">CCRL2</td><td rowspan="1" colspan="1" align="center">.3138</td><td rowspan="1" colspan="1" align="center">4.643E-06</td><td rowspan="1" colspan="1" align="center">1.483</td></tr><tr><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">rs477687</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">134,495,599</td><td rowspan="1" colspan="1">H2AFY</td><td rowspan="1" colspan="1" align="center">.1807</td><td rowspan="1" colspan="1" align="center">5.744E-06</td><td rowspan="1" colspan="1" align="center">1.548</td></tr><tr><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">rs1015164</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">46,426,684</td><td rowspan="1" colspan="1">LTF</td><td rowspan="1" colspan="1" align="center">.3088</td><td rowspan="1" colspan="1" align="center">8.636E-06</td><td rowspan="1" colspan="1" align="center">1.463</td></tr><tr><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">rs10932110</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">206,050,138</td><td rowspan="1" colspan="1">PARD3B</td><td rowspan="1" colspan="1" align="center">.0622</td><td rowspan="1" colspan="1" align="center">8.868E-06</td><td rowspan="1" colspan="1" align="center">0.429</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs1553477</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">53,152,723</td><td rowspan="1" colspan="1">ARL15</td><td rowspan="1" colspan="1" align="center">.4651</td><td rowspan="1" colspan="1" align="center">1.335E-05</td><td rowspan="1" colspan="1" align="center">0.699</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs17112431</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">20,027,330</td><td rowspan="1" colspan="1">RNASE10</td><td rowspan="1" colspan="1" align="center">.2078</td><td rowspan="1" colspan="1" align="center">1.456E-05</td><td rowspan="1" colspan="1" align="center">0.644</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">rs17225585</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">66,882,025</td><td rowspan="1" colspan="1">SOX9</td><td rowspan="1" colspan="1" align="center">.1245</td><td rowspan="1" colspan="1" align="center">1.750E-05</td><td rowspan="1" colspan="1" align="center">0.568</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs7275915</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">34,753,498</td><td rowspan="1" colspan="1">KCNE1</td><td rowspan="1" colspan="1" align="center">.1851</td><td rowspan="1" colspan="1" align="center">1.979E-05</td><td rowspan="1" colspan="1" align="center">0.627</td></tr><tr><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs4397202</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">168,718,725</td><td rowspan="1" colspan="1">SMOC2</td><td rowspan="1" colspan="1" align="center">.0471</td><td rowspan="1" colspan="1" align="center">2.024E-05</td><td rowspan="1" colspan="1" align="center">2.092</td></tr></tbody></table></div><div id="largeobj_idm140227882517184" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3080910/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140227886591664"><p id="__p29" class="p p-first-last"><strong>NOTE.</strong> Rank of <em>P</em> values for 700,022 SNPs tested. chr, chromosome; MAF, minor allele frequency; HR, hazard ratio; Chromosome assignment, base-pair positions, and gene associations of single-nucleotide polymorphisms (SNPs) presented here are from Affymetrix SNP Annotation, version 13.1, which is based on National Center for Biotechnology Information human genome build 36.1.</p></div></div></div><p id="__p31">Only 1 SNP (rs10185378) in the protective linkage group was in the coding region: a missense (C to T) in exon 20 that results in a change from a Threonine to an Isoleucine. This SNP has also been genotyped in 2 well-defined cohorts of rapid progressors; from the GRIV cohort [<a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684916">14</a>, <a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684942">15</a>] and the Amsterdam Cohort Study <a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684924">[33]</a> <em>(n</em> = 84 and <em>n</em> = 39, respectively). We examined the effect of this SNP in an analysis of rapid progressors compared with uninfected controls from both cohorts and found that it had a significant protective influence (<em>P</em> = .025; odds ratio [OR], 0.47; co-dominant model), consistent with our findings (<a href="/pmc/articles/PMC3080910/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure 3</span></a>). In all cohorts (GRIV, ACS, and the discovery NCI - LGD combined cohorts), not a single rapid progressor was homozygous for the minor allele, and the overall frequency of this allele within rapid progressors was half that of the general population (data not shown). Although this amino acid change is not predicted to have a profound effect on protein function, this SNP is within a predicted splice site enhancer for exon 20 <a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684910">[36]</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC3080910/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140227878863200" class="figure"><a href="/pmc/articles/PMC3080910/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3080910_infdisjir046f03_3c.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is infdisjir046f03_3c.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3080910/bin/infdisjir046f03_3c.jpg"></a></div><div id="largeobj_idm140227878863200" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3080910/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC3080910/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure 3.</a></div><!--caption a7--><div class="caption"><p id="__p30">Analysis of PARD3B single-nucleotide polymorphisms (SNPs). <em>A</em>, Haplotype and LD structure of the SNPs for the PARD3B region. SNPs outlined in blue were typed on Affymetrix 6.0 for European American samples and those highlighted with yellow were typed on Illummina300 in the GRIV and Amsterdam Cohort Study (ACS) European cohorts. Only HapMap SNPs are included here; underlying LD structure is from HapMap. SNPs with an asterisk were among the top ranking SNPs in this study. All haplotypes with a frequency &gt;1% in the discovery cohorts are shown. SNPs that rank in the top 10 of the AIDS 93 progression analysis track a single haplotype, hap 4 (tracking SNPs are highlighted in bold), that is itself significant at the genome-wide level. Alleles from the top hit in this study (rs11884476) are shown in green; alleles from the ranking SNP typed on both platforms (rs10185378) are shown in blue. The three other SNPs in this region that are typed in the European cohorts (yellow) do not track the significant haplotype. <em>B</em>, Distribution of rs10185378 genotypes in the combined GRIV and ACS cohorts. Genotypes and allele frequencies for in control subjects (<em>n</em> = 1071) and case patients (rapid progressors; <em>n</em> = 231) CT (<em>light gray</em>), and TT (<em>black</em>) genotypes and T allele frequency (<em>dark gray</em>), where T is the minor allele with an overall frequency of .075, a frequency in controls of .078, and a frequency of .041 among rapid progressors.</p></div></div></div><p id="__p32" class="p p-last">To explore this potential functional basis of <em>PARD3B</em>-based association, we developed a real-time PCR assay to detect expression of alternative splice forms in lymphoblastoid cell lines (LCLs) with alternative <em>PARD3B</em> genotypes for the codon-altering SNP, rs10185378 [<a href="#bib37" rid="bib37" class=" bibr popnode">37</a>] (<a href="/pmc/articles/PMC3080910/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure 4</span></a>A). <em>PARD3B</em> produces 3 alternatively spliced reference sequences; 2 of these, isoforms B and C, have an exon (exon 20) that is absent in isoform A <a href="#bib38" rid="bib38" class=" bibr popnode">[38]</a> (Figure <em>4B</em>). When we compared RNA expression of these isoforms in LCLs bearing different <em>PARD3B</em> genotypes (CC, <em>n</em> = 12; CT, <em>n</em> = 8; TT, <em>n</em> = 9), we observed an increased average expression of isoform A and a decreased expression of isoforms B and C in cell lines homozygous (TT) for the minor (protective) allele (Figure <em>4B</em>). There is a significant shift in relative expression of the 2 splice forms (<em>P</em> = .006; co-dominant model; Figure <em>4C</em>). These data lend in vitro support to the in silico prediction that this enhancer influences <em>PARD3B</em> splicing <a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684944">[36]</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC3080910/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140227893363968" class="figure"><a href="/pmc/articles/PMC3080910/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3080910_infdisjir046f04_3c.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is infdisjir046f04_3c.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3080910/bin/infdisjir046f04_3c.jpg"></a></div><div id="largeobj_idm140227893363968" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3080910/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC3080910/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure 4.</a></div><!--caption a7--><div class="caption"><p id="__p36"><em>A</em>, Primer probe design for the expression of alternative splice forms. Introns are represented as thin blue lines, and exons are blue rectangles with the exon number designated in black. Probes are represented as red lines, and primers are blue with arrows. <em>B</em>, Expression of alternative <em>PARD3B</em> splice forms by rs10185378 genotype. The arithmetic means of <em>PARD3B</em> normalized transcript expression grouped by genotype (CC, <em>n</em> = 12; CT, <em>n</em> = 8; TT, <em>n</em> = 9) are shown. Error bars indicate the standard error of the mean. Differences between expression profiles of the alternate splice forms were consistent with the predicted effect for this SNP (also see <a href="http://jid.oxfordjournals.org/cgi/content/full/jir046/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental figure 5</a>). <em>C</em>, Relative difference between PARD3B expression of isoform A and isoforms B/C with 95% confidence intervals plotted by genotype (<em>P</em> = .006, co-dominant model).</p></div></div></div></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">DISCUSSION</h2><p id="__p33" class="p p-first">The advance from candidate gene studies to genome-wide association studies for AIDS, as for other diseases, has the notable advantage of being able to reveal previously unsuspected genetic influences. The GWAS results presented here implicate rare genetic variants in the <em>PARD3B</em> gene as conferring slower progression to clinical AIDS among HIV-infected EAs.</p><p id="__p34"><em>PARD3B</em> is a homolog of a <em>C. elegans</em> cell polarity determinant that localizes to tight junctions in human epithelial cells <a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684934">[35]</a>. It may be relevant that human <em>PARD3B</em> gene products interact with several members of the <em>SMAD</em> (mothers against decapentaplegic homolog) family. <em>SMAD</em> proteins are signal transducers and transcriptional modulators that mediate multiple signaling pathways and are involved in a range of biological activities, including cell growth, apoptosis, morphogenesis, development, and immune responses. Of particular interest are the interactions of <em>PARD3B</em> encoded proteins with <em>SMAD2</em>, <em>SMAD3</em>, <em>SMAD4</em>, and <em>SMAD7</em>, all of which interact directly with HIV-1 [<a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684952">39</a>–<a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684953">43</a>]. <em>SMAD3</em> and 4 also interact with TGFβ to regulate Tat-mediated transcription of HIV-1 LTR. Individually, <em>SMAD3</em> increases viral promoter activity, whereas <em>SMAD4</em> decreases it. <em>SMAD4</em> also suppresses activation by <em>SMAD3</em> <a href="#bib41" rid="bib41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684948">[41]</a>. These proteins have similar effects on Tat-mediated transcription of CCL2 (formerly MCP-1; <a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684941">[40]</a>). <em>SMAD2</em> and 7 are involved in HIV-1 glycoprotein 120-induced tubular epithelial cell apoptosis <a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684906">[43]</a>. SMAD7 helps prevent apoptosis, whereas <em>SMAD2</em> is involved in downstream signaling <a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684949">[43]</a>.</p><p id="__p35"><em>PARD3B</em> is a large gene (with 23 exons coding for 1205 amino acids and spanning 1.07 Mbp) located on chromosome 2. One of the exons (exon 20) is alternately spliced. Three splice variants of the <em>PARD3B</em> product have been characterized and shown to have differences in their protein binding properties <a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684926">[44]</a>. The functional significance of these alternative forms is unknown, but they may also have different expression profiles <a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684909">[44]</a>. A SNP in the cluster of association, rs10185378, which introduces a missense mutation in exon 20, also eliminates a predicted splice enhancer for this exon. We tested this same SNP for transcript production using a group of 29 lymphoblastoid cell lines of alternate genotypes and observed a significant increase in messenger RNA levels of the <em>PARD3B</em> isoform lacking exon 20 relative to forms with exon 20 with the rare allele (TT) at rs10185378 (<a href="/pmc/articles/PMC3080910/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure 4</span></a>). It is by no means clear that this is the causative polymorphism; additional studies into PARD3B function with respect to HIV-host interactions are indicated.</p><p id="__p37">As with any association result that may produce false-positive results, replication in independent cohorts is essential. Although results here are significant for this association, several factors make independent replication particularly important. The fact that the gene identified falls in an unexpected category demands additional evidence for the association to be validated. By the same token, if confirmed, the identification of <em>PARD3B</em> would have the potential to reveal an entirely new avenue of HIV cellular activity, with potential new targets for therapy, and could lead to a new understanding of the process of AIDS pathogenesis. The corroborative association of <em>PARD3B</em>-SNP rs10185378 in the European GRIV and ACS rapid progression cohorts provides an initial replication for this association (Figure <em>3B</em>). A EA seroconverter cohort with AIDS 1987 progression would be a more valuable, though unavailable, comparison, because the signal in our cohort clearly associates with this particular phenotype and is not evident when comparing rapid and slow progressors from our cohorts for CD4+ cell count &lt;200 cells/mL and does not achieve genome-wide significance for AIDS 1993 definition (<a href="/pmc/articles/PMC3080910/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figure 5</span></a><em>A</em>). Therefore, it is not surprising that our SNP of highest significance did not replicate for CD4+ cell count &lt;300 cells/mL or AIDS 1993 in a dichotomous analysis of the GRIV rapid versus slow progressors (data not shown). In addition, allele frequency and haplotype structure in African populations is quite different than in EA populations, which might account for the failure to replicate this signal in a small <em>(n</em> = 282) AA seroconverter replication cohort.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC3080910/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140227889733744" class="figure"><a href="/pmc/articles/PMC3080910/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3080910_infdisjir046f05_3c.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is infdisjir046f05_3c.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3080910/bin/infdisjir046f05_3c.jpg"></a></div><div id="largeobj_idm140227889733744" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3080910/figure/fig5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC3080910/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Figure 5.</a></div><!--caption a7--><div class="caption"><p id="__p38">Additional analysis of the GWAS data. <em>A</em>, Categorical analysis of rs11884476 for different clinical outcomes. Shown here are CD4+ cell count &lt;200 cells/mL and AIDS 1993 definition in &lt;8 years or ≥8 years and AIDS 1987 with a cutoff value of 10 years. B. Results of survival analysis (AIDS-1987) for the HLA region on chromosome 6. Negative log <em>P</em> values are graphed with respect to chromosomal position for MACS seroconvertors alone (<em>black</em>) or all European American seroconvertors (<em>red</em>). Categorical tests on several cohorts, including MACS samples overlapping those in this study, have identified single-nucleotide polymorphisms (SNPs) on chromosome 6 that are highly associated with viral load set point and, to a lesser extent, progression to CD4+ cell count &lt;400 cells/mL and rapid disease progression. The SNPs with robust association (rs9264942 and rs2395029) are indicated with dotted arrows; neither of these are typed or tracked on the Affymetrix 6.0 platform. No other SNPs in this region reached genome-wide significance for association with time to AIDS 1987. Fellay et al [<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684929">11</a>] observed differences in association signals when tests for viral load set point were compared with test for CD4+ cell count depletion; AIDS 1987 is a much more complex phenotype that includes a range of clinical conditions that may have complex and conflicting interactions with genetic variants within this region. Therefore, the lack of signal in this study does not conflict with or contradict previous reports.</p></div></div></div><p id="__p39" class="p p-last">Although this GWAS demonstrates the strength of this approach for detecting unsuspected AIDS restriction genes, it also illustrates some weaknesses. Although we observed significant unadjusted <em>P</em> values for many of the previously validated ARGs [<a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684946">3</a>], none achieved genome-wide significance in the present GWAS, nor did we detect additional recently reported associations from other HIV/AIDS progression GWAS[<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684933">11</a>–<a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217684943">14</a>] with this phenotype (Figure 5<em>B</em>). Deviations in SNP choice and region coverage from platform to platform account for some of the disparity between results from similar studies (as demonstrated by our difficulty in using Illumina HapMap300 SNPs to impute Affy 6.0 SNPs with high confidence in our <em>PARD3B</em> region of significance; see Methods), as do alternative choices in disease phenotype tests and genetic models. The AIDS progression end point phenotype examined here represents a proven, well-powered analysis for progression to clinical AIDS. However, it is a complex phenotype, and further exploration of PARD3B association with multiple sequelae is now clearly indicated. A more detailed exploration of genetic associations with all of the clinical data collected on these AIDS cohorts will ultimately allow a much broader view of multiple AIDS phenomena, albeit with statistical penalties for multiple test approaches (J. A. Lautenberger, J. L. Troyer, and S. J. O'Brien, unpublished data). As the interpretation of the full range of data necessarily involves subjective judgment, with a corresponding loss of statistical rigor, it seems useful to extract from the full dataset the statistically straightforward analysis presented here, which implicates a strong new AIDS progression genetic association.</p></div><div id="__sec11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title">Supplementary Data</h2><p id="__p40" class="p p-first-last"><a href="http://jid.oxfordjournals.org/cgi/content/full/jir046/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary data</a> are available at <a href="http://jid.oxfordjournals.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://jid.oxfordjournals.org</a> online.</p></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Funding</h2><p id="__p41" class="p p-first-last">The Intramural Research Program of the National Institutes of Health, National Cancer Institute, Center for Cancer Research; the National Cancer Institute, National Institutes of Health, under contract HHSN26120080001E. The Hemophilia Growth and Development Study is funded by the National Institutes of Health, National Institute of Child Health and Human Development, 1 R01 HD41224. Funding for genetic studies on the ACS was provided by the Netherlands Organization for Scientific Research (TOP, registration number 9120.6046).</p></div><div id="idm140227885268160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140227885268160title">Acknowledgments</h2><div class="sec"><p id="__p42">We gratefully acknowledge and assign collaborative credit investigators who have assembled the AIDS study cohorts: Multicenter AIDS Cohort Study (MACS, PI J. Bream), Multicenter Hemophilia Cohort Study (MHCS, PI J. Goedert), D.C. Gays (DCG, PI J. Goedert), Hemophilia Growth and Development Study (HGDS, PI E. Gomperts), San Francisco City Clinic Study (SFCC, PI S. Buchbinder) and AIDS linked to Intravenous Experience (ALIVE, PI G. Kirk). We also acknowledge the excellent work of Michelle Hall, Mary McNally, Lisa Maslan, David Wells, and Jamie Troxler, who worked tirelessly to produce the genotypes, and the LGD student interns who provided sample preparation and technical assistance. This study utilized the high-performance computational capabilities of the Biowulf Linux cluster at the National Institutes of Health, Bethesda, MD. The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products, or organizations imply endorsement by the US Government.</p></div></div><div id="idm140227885267536" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140227885267536title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Pereira CF, Paridaen JT. Anti-HIV drug development–an overview. <span><span class="ref-journal">Curr Pharm Des. </span>2004;<span class="ref-vol">10</span>:4005–4037.</span> [<a href="/pubmed/15579085" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Pharm+Des&amp;title=Anti-HIV+drug+development–an+overview&amp;author=CF+Pereira&amp;author=JT+Paridaen&amp;volume=10&amp;publication_year=2004&amp;pages=4005-4037&amp;pmid=15579085&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Carrington M, O'Brien SJ. The influence of HLA genotype on AIDS. <span><span class="ref-journal">Annu Rev Med. </span>2003;<span class="ref-vol">54</span>:535–551.</span> [<a href="/pubmed/12525683" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Med&amp;title=The+influence+of+HLA+genotype+on+AIDS&amp;author=M+Carrington&amp;author=SJ+O'Brien&amp;volume=54&amp;publication_year=2003&amp;pages=535-551&amp;pmid=12525683&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">O'Brien SJ, Nelson GW. Human genes that limit AIDS. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:565–574.</span> [<a href="/pubmed/15167933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Human+genes+that+limit+AIDS&amp;author=SJ+O'Brien&amp;author=GW+Nelson&amp;volume=36&amp;publication_year=2004&amp;pages=565-574&amp;pmid=15167933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Heeney JL, Dalgleish AG, Weiss RA. Origins of HIV and the evolution of resistance to AIDS. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">313</span>:462–466.</span> [<a href="/pubmed/16873637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Origins+of+HIV+and+the+evolution+of+resistance+to+AIDS&amp;author=JL+Heeney&amp;author=AG+Dalgleish&amp;author=RA+Weiss&amp;volume=313&amp;publication_year=2006&amp;pages=462-466&amp;pmid=16873637&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Kopp JB, Smith MW, Nelson GW, et al.  MYH9 is a major-effect risk gene for focal segmental glomerulosclerosis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1175–1184.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2827354/">PMC free article</a>]</span> [<a href="/pubmed/18794856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=MYH9+is+a+major-effect+risk+gene+for+focal+segmental+glomerulosclerosis&amp;author=JB+Kopp&amp;author=MW+Smith&amp;author=GW+Nelson&amp;volume=40&amp;publication_year=2008&amp;pages=1175-1184&amp;pmid=18794856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">An P, Winkler CA. Host genes associated with HIV/AIDS: advances in gene discovery. <span><span class="ref-journal">Trends Genet. </span>2010;<span class="ref-vol">26</span>:119–131.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3792714/">PMC free article</a>]</span> [<a href="/pubmed/20149939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=Host+genes+associated+with+HIV/AIDS:+advances+in+gene+discovery&amp;author=P+An&amp;author=CA+Winkler&amp;volume=26&amp;publication_year=2010&amp;pages=119-131&amp;pmid=20149939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">Manfredi R, Sabbatani S. A novel antiretroviral class (fusion inhibitors) in the management of HIV infection. Present features and future perspectives of enfuvirtide (T-20) <span><span class="ref-journal">Curr Med Chem. </span>2006;<span class="ref-vol">13</span>:2369–2384.</span> [<a href="/pubmed/16918361" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Med+Chem&amp;title=A+novel+antiretroviral+class+(fusion+inhibitors)+in+the+management+of+HIV+infection.+Present+features+and+future+perspectives+of+enfuvirtide+(T-20)&amp;author=R+Manfredi&amp;author=S+Sabbatani&amp;volume=13&amp;publication_year=2006&amp;pages=2369-2384&amp;pmid=16918361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">Aquaro S, D'Arrigo R, Svicher V, et al.  Specific mutations in HIV-1 gp41 are associated with immunological success in HIV-1-infected patients receiving enfuvirtide treatment. <span><span class="ref-journal">J Antimicrob Chemother. </span>2006;<span class="ref-vol">58</span>:714–22.</span> [<a href="/pubmed/16891628" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Antimicrob+Chemother&amp;title=Specific+mutations+in+HIV-1+gp41+are+associated+with+immunological+success+in+HIV-1-infected+patients+receiving+enfuvirtide+treatment&amp;author=S+Aquaro&amp;author=R+D'Arrigo&amp;author=V+Svicher&amp;volume=58&amp;publication_year=2006&amp;pages=714-22&amp;pmid=16891628&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Westby M, van der Ryst E. CCR5 antagonists: host-targeted antivirals for the treatment of HIV infection. <span><span class="ref-journal">Antivir Chem Chemother. </span>2005;<span class="ref-vol">16</span>:339–54.</span> [<a href="/pubmed/16329283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antivir+Chem+Chemother&amp;title=CCR5+antagonists:+host-targeted+antivirals+for+the+treatment+of+HIV+infection&amp;author=M+Westby&amp;author=E+van+der+Ryst&amp;volume=16&amp;publication_year=2005&amp;pages=339-54&amp;pmid=16329283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">Hutter G, Nowak D, Mossner M, et al.  Long-term control of HIV by CCR5 Delta32/Delta32 stem-cell transplantation. <span><span class="ref-journal">N Engl J Med. </span>2009;<span class="ref-vol">360</span>:692–8.</span> [<a href="/pubmed/19213682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Long-term+control+of+HIV+by+CCR5+Delta32/Delta32+stem-cell+transplantation&amp;author=G+Hutter&amp;author=D+Nowak&amp;author=M+Mossner&amp;volume=360&amp;publication_year=2009&amp;pages=692-8&amp;pmid=19213682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">Fellay J, Ge D, Shianna KV, et al.  Common genetic variation and the control of HIV-1 in humans. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000791.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2791220/">PMC free article</a>]</span> [<a href="/pubmed/20041166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Common+genetic+variation+and+the+control+of+HIV-1+in+humans&amp;author=J+Fellay&amp;author=D+Ge&amp;author=KV+Shianna&amp;volume=5&amp;publication_year=2009&amp;pages=e1000791&amp;pmid=20041166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Fellay J, Shianna KV, Ge D, et al.  A whole-genome association study of major determinants for host control of HIV-1. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>:944–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1991296/">PMC free article</a>]</span> [<a href="/pubmed/17641165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+whole-genome+association+study+of+major+determinants+for+host+control+of+HIV-1&amp;author=J+Fellay&amp;author=KV+Shianna&amp;author=D+Ge&amp;volume=317&amp;publication_year=2007&amp;pages=944-7&amp;pmid=17641165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Dalmasso C, Carpentier W, Meyer L, et al.  Distinct genetic loci control plasma HIV-RNA and cellular HIV-DNA levels in HIV-1 infection: the ANRS Genome Wide Association 01 study. <span><span class="ref-journal">LoS One. </span>2008;<span class="ref-vol">3</span>:e3907.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2603319/">PMC free article</a>]</span> [<a href="/pubmed/19107206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=LoS+One&amp;title=Distinct+genetic+loci+control+plasma+HIV-RNA+and+cellular+HIV-DNA+levels+in+HIV-1+infection:+the+ANRS+Genome+Wide+Association+01+study&amp;author=C+Dalmasso&amp;author=W+Carpentier&amp;author=L+Meyer&amp;volume=3&amp;publication_year=2008&amp;pages=e3907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Limou S, Le Clerc S, Coulonges C, et al.  Genomewide association study of an AIDS-nonprogression cohort emphasizes the role played by HLA genes (ANRS Genomewide Association Study 02) <span><span class="ref-journal">J Infect Dis. </span>2009;<span class="ref-vol">199</span>:419–26.</span> [<a href="/pubmed/19115949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Genomewide+association+study+of+an+AIDS-nonprogression+cohort+emphasizes+the+role+played+by+HLA+genes+(ANRS+Genomewide+Association+Study+02)&amp;author=S+Limou&amp;author=S+Le+Clerc&amp;author=C+Coulonges&amp;volume=199&amp;publication_year=2009&amp;pages=419-26&amp;pmid=19115949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Le Clerc S, Limou S, Coulonges C, et al.  Genomewide association study of a rapid progression cohort identifies new susceptibility alleles for AIDS (ANRS Genomewide Association Study 03) <span><span class="ref-journal">J Infect Dis. </span>2009;<span class="ref-vol">200</span>:1194–201.</span> [<a href="/pubmed/19754311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Genomewide+association+study+of+a+rapid+progression+cohort+identifies+new+susceptibility+alleles+for+AIDS+(ANRS+Genomewide+Association+Study+03)&amp;author=S+Le+Clerc&amp;author=S+Limou&amp;author=C+Coulonges&amp;volume=200&amp;publication_year=2009&amp;pages=1194-201&amp;pmid=19754311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Herbeck JT, Gottlieb GS, Winkler CA, et al.  Multistage genomewide association study identifies a locus at 1q41 associated with rate of HIV-1 disease progression to clinical AIDS. <span><span class="ref-journal">J Infect Dis. </span>2010;<span class="ref-vol">201</span>:618–26.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928718/">PMC free article</a>]</span> [<a href="/pubmed/20064070" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Multistage+genomewide+association+study+identifies+a+locus+at+1q41+associated+with+rate+of+HIV-1+disease+progression+to+clinical+AIDS&amp;author=JT+Herbeck&amp;author=GS+Gottlieb&amp;author=CA+Winkler&amp;volume=201&amp;publication_year=2010&amp;pages=618-26&amp;pmid=20064070&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Le Clerc S, Coulonges C, Delaneau O. Screening low frequency SNPS from genome wide association study reveals a new risk allele for progression to AIDS. <span><span class="ref-journal">J Acquir Immune Defic Syndr. </span>2010</span> Epub ahead of print, 23 November. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386792/">PMC free article</a>]</span> [<a href="/pubmed/21107268" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Acquir+Immune+Defic+Syndr.&amp;title=Screening+low+frequency+SNPS+from+genome+wide+association+study+reveals+a+new+risk+allele+for+progression+to+AIDS&amp;author=S+Le+Clerc&amp;author=C+Coulonges&amp;author=O+Delaneau&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Limou S, Coulonges C, Herbeck JT, et al.  Multiple-cohort genetic association study reveals CXCR6 as a new chemokine receptor involved in long-term nonprogression to AIDS. <span><span class="ref-journal">J Infect Dis. </span>2010;<span class="ref-vol">202</span>:908–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3601691/">PMC free article</a>]</span> [<a href="/pubmed/20704485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Multiple-cohort+genetic+association+study+reveals+CXCR6+as+a+new+chemokine+receptor+involved+in+long-term+nonprogression+to+AIDS&amp;author=S+Limou&amp;author=C+Coulonges&amp;author=JT+Herbeck&amp;volume=202&amp;publication_year=2010&amp;pages=908-15&amp;pmid=20704485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Hutcheson HB, Lautenberger JA, Nelson GW, et al.  Detecting AIDS restriction genes: from candidate genes to genome-wide association discovery. <span><span class="ref-journal">Vaccine. </span>2008;<span class="ref-vol">26</span>:2951–65.</span> [<a href="/pubmed/18325640" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Detecting+AIDS+restriction+genes:+from+candidate+genes+to+genome-wide+association+discovery&amp;author=HB+Hutcheson&amp;author=JA+Lautenberger&amp;author=GW+Nelson&amp;volume=26&amp;publication_year=2008&amp;pages=2951-65&amp;pmid=18325640&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Centers for Disease Control Revision of the CDC Surveillance Case Definition for Acquired Immunodeficiency Syndrome. <span><span class="ref-journal">MMWR Morb Mortal Wkly Rep. </span>1987;<span class="ref-vol">36</span></span> [<a href="/pubmed/3039334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MMWR+Morb+Mortal+Wkly+Rep&amp;title=Revision+of+the+CDC+Surveillance+Case+Definition+for+Acquired+Immunodeficiency+Syndrome&amp;author=+Centers+for+Disease+Control&amp;volume=36&amp;publication_year=1987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">From the Centers for Disease Control and Prevention. 1993 revised classification system for HIV infection and expanded surveillance case definition for AIDS among adolescents and adults. <span><span class="ref-journal">JAMA. </span>1993;<span class="ref-vol">269</span>:729–30.</span> [<a href="/pubmed/8093740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=1993+revised+classification+system+for+HIV+infection+and+expanded+surveillance+case+definition+for+AIDS+among+adolescents+and+adults&amp;volume=269&amp;publication_year=1993&amp;pages=729-30&amp;pmid=8093740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">Dean M, Carrington M, Winkler C, et al.  Genetic restriction of HIV-1 infection and progression to AIDS by a deletion allele of the CKR5 structural gene. Hemophilia Growth and Development Study, Multicenter AIDS Cohort Study, Multicenter Hemophilia Cohort Study, San Francisco City Cohort, ALIVE Study. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">273</span>:1856–62.</span> [<a href="/pubmed/8791590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+restriction+of+HIV-1+infection+and+progression+to+AIDS+by+a+deletion+allele+of+the+CKR5+structural+gene.+Hemophilia+Growth+and+Development+Study,+Multicenter+AIDS+Cohort+Study,+Multicenter+Hemophilia+Cohort+Study,+San+Francisco+City+Cohort,+ALIVE+Study&amp;author=M+Dean&amp;author=M+Carrington&amp;author=C+Winkler&amp;volume=273&amp;publication_year=1996&amp;pages=1856-62&amp;pmid=8791590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Price AL, Butler J, Patterson N, et al.  Discerning the ancestry of European Americans in genetic association studies. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e236.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2211542/">PMC free article</a>]</span> [<a href="/pubmed/18208327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Discerning+the+ancestry+of+European+Americans+in+genetic+association+studies&amp;author=AL+Price&amp;author=J+Butler&amp;author=N+Patterson&amp;volume=4&amp;publication_year=2008&amp;pages=e236&amp;pmid=18208327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation">Schaubel DE, Wei G. Fitting semiparametric additive hazards models using standard statistical software. <span><span class="ref-journal">Biom J. </span>2007;<span class="ref-vol">49</span>:719–30.</span> [<a href="/pubmed/17638295" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biom+J&amp;title=Fitting+semiparametric+additive+hazards+models+using+standard+statistical+software&amp;author=DE+Schaubel&amp;author=G+Wei&amp;volume=49&amp;publication_year=2007&amp;pages=719-30&amp;pmid=17638295&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation">Hoggart CJ, Whittaker JC, De Iorio M, Balding DJ. Simultaneous analysis of all SNPs in genome-wide and re-sequencing association studies. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2464715/">PMC free article</a>]</span> [<a href="/pubmed/18654633" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Simultaneous+analysis+of+all+SNPs+in+genome-wide+and+re-sequencing+association+studies&amp;author=CJ+Hoggart&amp;author=JC+Whittaker&amp;author=M+De+Iorio&amp;author=DJ+Balding&amp;volume=4&amp;publication_year=2008&amp;pages=e1000130&amp;pmid=18654633&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation">McCarthy MI, Abecasis GR, Cardon LR, et al.  Genome-wide association studies for complex traits: consensus, uncertainty and challenges. <span><span class="ref-journal">Nat Rev Genet. </span>2008;<span class="ref-vol">9</span>:356–69.</span> [<a href="/pubmed/18398418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+for+complex+traits:+consensus,+uncertainty+and+challenges&amp;author=MI+McCarthy&amp;author=GR+Abecasis&amp;author=LR+Cardon&amp;volume=9&amp;publication_year=2008&amp;pages=356-69&amp;pmid=18398418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation">Risch N, Merikangas K. The future of genetic studies of complex human diseases. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">273</span>:1516–7.</span> [<a href="/pubmed/8801636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+future+of+genetic+studies+of+complex+human+diseases&amp;author=N+Risch&amp;author=K+Merikangas&amp;volume=273&amp;publication_year=1996&amp;pages=1516-7&amp;pmid=8801636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation">Nelson GW, O'Brien SJ. Using mutual information to measure the impact of multiple genetic factors on AIDS. <span><span class="ref-journal">J Acquir Immune Defic Syndr. </span>2006;<span class="ref-vol">42</span>:347–54.</span> [<a href="/pubmed/16763524" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Acquir+Immune+Defic+Syndr&amp;title=Using+mutual+information+to+measure+the+impact+of+multiple+genetic+factors+on+AIDS&amp;author=GW+Nelson&amp;author=SJ+O'Brien&amp;volume=42&amp;publication_year=2006&amp;pages=347-54&amp;pmid=16763524&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation">Schemper M, Henderson R. Predictive accuracy and explained variation in Cox regression. <span><span class="ref-journal">Biometrics. </span>2000;<span class="ref-vol">56</span>:249–55.</span> [<a href="/pubmed/10783803" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Predictive+accuracy+and+explained+variation+in+Cox+regression&amp;author=M+Schemper&amp;author=R+Henderson&amp;volume=56&amp;publication_year=2000&amp;pages=249-55&amp;pmid=10783803&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation">van Manen D, Kootstra NA, Boeser-Nunnink B, Handulle MA, van't Wout AB, Schuitemaker H. Association of HLA-C and HCP5 gene regions with the clinical course of HIV-1 infection. <span><span class="ref-journal">AIDS. </span>2009;<span class="ref-vol">23</span>:19–28.</span> [<a href="/pubmed/19050382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=AIDS&amp;title=Association+of+HLA-C+and+HCP5+gene+regions+with+the+clinical+course+of+HIV-1+infection&amp;author=D+van+Manen&amp;author=NA+Kootstra&amp;author=B+Boeser-Nunnink&amp;author=MA+Handulle&amp;author=AB+van't+Wout&amp;volume=23&amp;publication_year=2009&amp;pages=19-28&amp;pmid=19050382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation">Howie BN, Donnelly P, Marchini J IMPUTE2.    <a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://mathgen.stats.ox.ac.uk/impute/impute_v2.html</a>. Accessed 17 February 2011. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Howie+BN+Donnelly+P+Marchini+J+IMPUTE2+http://mathgen.stats.ox.ac.uk/impute/impute_v2.html+.+Accessed+17+February+2011+" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation">Kohjima M, Noda Y, Takeya R, Saito N, Takeuchi K, Sumimoto H. PAR3beta, a novel homologue of the cell polarity protein PAR3, localizes to tight junctions. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2002;<span class="ref-vol">299</span>:641–6.</span> [<a href="/pubmed/12459187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=PAR3beta,+a+novel+homologue+of+the+cell+polarity+protein+PAR3,+localizes+to+tight+junctions&amp;author=M+Kohjima&amp;author=Y+Noda&amp;author=R+Takeya&amp;author=N+Saito&amp;author=K+Takeuchi&amp;volume=299&amp;publication_year=2002&amp;pages=641-6&amp;pmid=12459187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation">Yuan HY, Chiou JJ, Tseng WH, et al.  FASTSNP: an always up-to-date and extendable service for SNP function analysis and prioritization. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:W635–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1538865/">PMC free article</a>]</span> [<a href="/pubmed/16845089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=FASTSNP:+an+always+up-to-date+and+extendable+service+for+SNP+function+analysis+and+prioritization&amp;author=HY+Yuan&amp;author=JJ+Chiou&amp;author=WH+Tseng&amp;volume=34&amp;publication_year=2006&amp;pages=W635-41&amp;pmid=16845089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation">National Center for Biotechnology Information. PARD3B par-3 partitioning defective 3 homolog B (C. elegans) [ Homo sapiens].  <a href="http://www.ncbi.nlm.nih.gov/gene/117583" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gene/117583</a>. Accessed 17 February 2011  <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=National+Center+for+Biotechnology+Information.+PARD3B+par-3+partitioning+defective+3+homolog+B+(C.+elegans)+[+Homo+sapiens].++http://www.ncbi.nlm.nih.gov/gene/117583+.+Accessed+17+February+2011+" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation"> National Center for Biotechnology Information. PARD3B par-3 partitioning defective 3 homolog B (C. elegans) [Homo sapiens].  <a href="http://www.ncbi.nlm.nih.gov/gene/117583" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gene/117583</a>. Accessed 17 February 2011. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=National+Center+for+Biotechnology+Information.+PARD3B+par-3+partitioning+defective+3+homolog+B+(C.+elegans)+[Homo+sapiens].++http://www.ncbi.nlm.nih.gov/gene/117583+.+Accessed+17+February+2011+" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation">Fu W, Sanders-Beer BE, Katz KS, Maglott DR, Pruitt KD, Ptak RG. Human immunodeficiency virus type 1, human protein interaction database at NCBI. <span><span class="ref-journal">Nucleic Acids Res. </span>2009;<span class="ref-vol">37</span>:D417–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2686594/">PMC free article</a>]</span> [<a href="/pubmed/18927109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Human+immunodeficiency+virus+type+1,+human+protein+interaction+database+at+NCBI&amp;author=W+Fu&amp;author=BE+Sanders-Beer&amp;author=KS+Katz&amp;author=DR+Maglott&amp;author=KD+Pruitt&amp;volume=37&amp;publication_year=2009&amp;pages=D417-22&amp;pmid=18927109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation">Abraham S, Sawaya BE, Safak M, Batuman O, Khalili K, Amini S. Regulation of MCP-1 gene transcription by Smads and HIV-1 Tat in human glial cells. <span><span class="ref-journal">Virology. </span>2003;<span class="ref-vol">309</span>:196–202.</span> [<a href="/pubmed/12758167" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Regulation+of+MCP-1+gene+transcription+by+Smads+and+HIV-1+Tat+in+human+glial+cells&amp;author=S+Abraham&amp;author=BE+Sawaya&amp;author=M+Safak&amp;author=O+Batuman&amp;author=K+Khalili&amp;volume=309&amp;publication_year=2003&amp;pages=196-202&amp;pmid=12758167&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation">Abraham S, Sweet T, Sawaya BE, Rappaport J, Khalili K, Amini S. Cooperative interaction of C/EBP beta and Tat modulates MCP-1 gene transcription in astrocytes. <span><span class="ref-journal">J Neuroimmunol. </span>2005;<span class="ref-vol">160</span>:219–27.</span> [<a href="/pubmed/15710476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neuroimmunol&amp;title=Cooperative+interaction+of+C/EBP+beta+and+Tat+modulates+MCP-1+gene+transcription+in+astrocytes&amp;author=S+Abraham&amp;author=T+Sweet&amp;author=BE+Sawaya&amp;author=J+Rappaport&amp;author=K+Khalili&amp;volume=160&amp;publication_year=2005&amp;pages=219-27&amp;pmid=15710476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation">Coyle-Rink J, Sweet T, Abraham S, et al.  Interaction between TGFbeta signaling proteins and C/EBP controls basal and Tat-mediated transcription of HIV-1 LTR in astrocytes. <span><span class="ref-journal">Virology. </span>2002;<span class="ref-vol">299</span>:240–7.</span> [<a href="/pubmed/12202226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Interaction+between+TGFbeta+signaling+proteins+and+C/EBP+controls+basal+and+Tat-mediated+transcription+of+HIV-1+LTR+in+astrocytes&amp;author=J+Coyle-Rink&amp;author=T+Sweet&amp;author=S+Abraham&amp;volume=299&amp;publication_year=2002&amp;pages=240-7&amp;pmid=12202226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation">Kapasi AA, Fan S, Singhal PC. p300 modulates HIV-1 gp120-induced apoptosis in human proximal tubular cells: associated with alteration of TGF-beta and Smad signaling. <span><span class="ref-journal">Nephron Exp Nephrol. </span>2006;<span class="ref-vol">102</span>:e30–8.</span> [<a href="/pubmed/16179804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephron+Exp+Nephrol&amp;title=p300+modulates+HIV-1+gp120-induced+apoptosis+in+human+proximal+tubular+cells:+associated+with+alteration+of+TGF-beta+and+Smad+signaling&amp;author=AA+Kapasi&amp;author=S+Fan&amp;author=PC+Singhal&amp;volume=102&amp;publication_year=2006&amp;pages=e30-8&amp;pmid=16179804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation">Gao L, Macara IG, Joberty G. Multiple splice variants of Par3 and of a novel related gene, Par3L, produce proteins with different binding properties. <span><span class="ref-journal">Gene. </span>2002;<span class="ref-vol">294</span>:99–107.</span> [<a href="/pubmed/12234671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Multiple+splice+variants+of+Par3+and+of+a+novel+related+gene,+Par3L,+produce+proteins+with+different+binding+properties&amp;author=L+Gao&amp;author=IG+Macara&amp;author=G+Joberty&amp;volume=294&amp;publication_year=2002&amp;pages=99-107&amp;pmid=12234671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080910&amp;issue-id=196014&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140227925194480title">Abstract</a></li><li><a href="#__sec1title">METHODS</a></li><li><a href="#__sec9title">RESULTS</a></li><li><a href="#__sec10title">DISCUSSION</a></li><li><a href="#__sec11title">Supplementary Data</a></li><li><a href="#__sec12title">Funding</a></li><li><a href="#idm140227885268160title">Acknowledgments</a></li><li><a href="#idm140227885267536title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">The Journal of Infectious Diseases</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3080910/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3080910/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3080910/pdf/jir046.pdf">PDF (870K)</a></li> | <li><a href="#" data-citationid="PMC3080910" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080910%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080910%2F&amp;text=Genome-wide%20Association%20Study%20Implicates%20PARD3B-based%20AIDS%20Restriction" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080910%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21502085" data-aiid="3080910" data-aid="3080910" data-iid="196014" data-domainid="1410" data-domain="jid" data-accid="PMC3080910" data-md5="43bd7a3d5a25def70d6a6ed350ca283f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:57:21-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89E03CDD522BC100000000051B034A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080910%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jid&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3080910/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>