<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3651583/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common variants at 6p21.1 are associated with large artery atherosclerotic stroke</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3651583/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Common variants at 6p21.1 are associated with large artery atherosclerotic stroke"><meta name="citation_authors" content="Elizabeth G Holliday, Jane M Maguire, Tiffany-Jane Evans, Simon A Koblar, Jim Jannes, Jonathan W Sturm, Graeme J Hankey, Ross Baker, Jonathan Golledge, Mark W Parsons, Rainer Malik, Mark McEvoy, Erik Biros, Martin D Lewis, Lisa F Lincz, Roseanne Peel, Christopher Oldmeadow, Wayne Smith, Pablo Moscato, Simona Barlera, Steve Bevan, Joshua C Bis, Eric Boerwinkle, Giorgio B Boncoraglio, Thomas G Brott, Robert D Brown, Jr, Yu-Ching Cheng, John W Cole, Ioana Cotlarciuc, William J Devan, Myriam Fornage, Karen L Furie, Sólveig Grétarsdóttir, Andreas Gschwendtner, M Arfan Ikram, W T Longstreth, Jr, James F Meschia, Braxton D Mitchell, Thomas H Mosley, Michael A Nalls, Eugenio A Parati, Bruce M Psaty, Pankaj Sharma, Kari Stefansson, Gudmar Thorleifsson, Unnur Thorsteinsdottir, Matthew Traylor, Benjamin F J Verhaaren, Kerri L Wiggins, Bradford B Worrall, The Australian Stroke Genetics Collaborative, The International Stroke Genetics Consortium, The Wellcome Trust Case Control Consortium 2, Cathie Sudlow, Peter M Rothwell, Martin Farrall, Martin Dichgans, Jonathan Rosand, Hugh S Markus, Rodney J Scott, Christopher Levi, John Attia"><meta name="citation_date" content="October 2012"><meta name="citation_issue" content="10"><meta name="citation_volume" content="44"><meta name="citation_doi" content="10.1038/ng.2397"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3651583/?report=abstract"><meta name="citation_pmid" content="22941190"><meta name="DC.Title" content="Common variants at 6p21.1 are associated with large artery atherosclerotic stroke"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Elizabeth G Holliday"><meta name="DC.Contributor" content="Jane M Maguire"><meta name="DC.Contributor" content="Tiffany-Jane Evans"><meta name="DC.Contributor" content="Simon A Koblar"><meta name="DC.Contributor" content="Jim Jannes"><meta name="DC.Contributor" content="Jonathan W Sturm"><meta name="DC.Contributor" content="Graeme J Hankey"><meta name="DC.Contributor" content="Ross Baker"><meta name="DC.Contributor" content="Jonathan Golledge"><meta name="DC.Contributor" content="Mark W Parsons"><meta name="DC.Contributor" content="Rainer Malik"><meta name="DC.Contributor" content="Mark McEvoy"><meta name="DC.Contributor" content="Erik Biros"><meta name="DC.Contributor" content="Martin D Lewis"><meta name="DC.Contributor" content="Lisa F Lincz"><meta name="DC.Contributor" content="Roseanne Peel"><meta name="DC.Contributor" content="Christopher Oldmeadow"><meta name="DC.Contributor" content="Wayne Smith"><meta name="DC.Contributor" content="Pablo Moscato"><meta name="DC.Contributor" content="Simona Barlera"><meta name="DC.Contributor" content="Steve Bevan"><meta name="DC.Contributor" content="Joshua C Bis"><meta name="DC.Contributor" content="Eric Boerwinkle"><meta name="DC.Contributor" content="Giorgio B Boncoraglio"><meta name="DC.Contributor" content="Thomas G Brott"><meta name="DC.Contributor" content="Robert D Brown, Jr"><meta name="DC.Contributor" content="Yu-Ching Cheng"><meta name="DC.Contributor" content="John W Cole"><meta name="DC.Contributor" content="Ioana Cotlarciuc"><meta name="DC.Contributor" content="William J Devan"><meta name="DC.Contributor" content="Myriam Fornage"><meta name="DC.Contributor" content="Karen L Furie"><meta name="DC.Contributor" content="Sólveig Grétarsdóttir"><meta name="DC.Contributor" content="Andreas Gschwendtner"><meta name="DC.Contributor" content="M Arfan Ikram"><meta name="DC.Contributor" content="W T Longstreth, Jr"><meta name="DC.Contributor" content="James F Meschia"><meta name="DC.Contributor" content="Braxton D Mitchell"><meta name="DC.Contributor" content="Thomas H Mosley"><meta name="DC.Contributor" content="Michael A Nalls"><meta name="DC.Contributor" content="Eugenio A Parati"><meta name="DC.Contributor" content="Bruce M Psaty"><meta name="DC.Contributor" content="Pankaj Sharma"><meta name="DC.Contributor" content="Kari Stefansson"><meta name="DC.Contributor" content="Gudmar Thorleifsson"><meta name="DC.Contributor" content="Unnur Thorsteinsdottir"><meta name="DC.Contributor" content="Matthew Traylor"><meta name="DC.Contributor" content="Benjamin F J Verhaaren"><meta name="DC.Contributor" content="Kerri L Wiggins"><meta name="DC.Contributor" content="Bradford B Worrall"><meta name="DC.Contributor" content="The Australian Stroke Genetics Collaborative"><meta name="DC.Contributor" content="The International Stroke Genetics Consortium"><meta name="DC.Contributor" content="The Wellcome Trust Case Control Consortium 2"><meta name="DC.Contributor" content="Cathie Sudlow"><meta name="DC.Contributor" content="Peter M Rothwell"><meta name="DC.Contributor" content="Martin Farrall"><meta name="DC.Contributor" content="Martin Dichgans"><meta name="DC.Contributor" content="Jonathan Rosand"><meta name="DC.Contributor" content="Hugh S Markus"><meta name="DC.Contributor" content="Rodney J Scott"><meta name="DC.Contributor" content="Christopher Levi"><meta name="DC.Contributor" content="John Attia"><meta name="DC.Date" content="2012 Oct"><meta name="DC.Identifier" content="10.1038/ng.2397"><meta name="DC.Language" content="en"><meta property="og:title" content="Common variants at 6p21.1 are associated with large artery atherosclerotic stroke"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies (GWAS) have not consistently detected replicable genetic risk factors for ischemic stroke, potentially due to etiological heterogeneity of this trait. We performed GWAS of ischemic stroke and a major ischemic stroke subtype ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3651583/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3651583/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3651583/pdf/nihms457841.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B6576DD548DD100000000043F02AD.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3651583%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3651583</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2013 May 11.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22941190" target="pmc_ext" ref="reftype=publisher&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2012 Oct; 44(10): 10.1038/ng.2397. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Sep 2. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.2397" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.2397</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3651583</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS457841</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22941190">22941190</a></div></div></div></div><h1 class="content-title">Common variants at 6p21.1 are associated with large artery atherosclerotic stroke</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Holliday%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066448326192" co-class="co-affbox">Elizabeth G Holliday</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Maguire%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066457829552" co-class="co-affbox">Jane M Maguire</a>,<sup>3,</sup><sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Evans%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066457826656" co-class="co-affbox">Tiffany-Jane Evans</a>,<sup>2,</sup><sup>6</sup> <a href="/pubmed/?term=Koblar%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066458488128" co-class="co-affbox">Simon A Koblar</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Jannes%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066493386960" co-class="co-affbox">Jim Jannes</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Sturm%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066493384240" co-class="co-affbox">Jonathan W Sturm</a>,<sup>5,</sup><sup>9</sup> <a href="/pubmed/?term=Hankey%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066454686288" co-class="co-affbox">Graeme J Hankey</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Baker%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066527241376" co-class="co-affbox">Ross Baker</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Golledge%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066527238656" co-class="co-affbox">Jonathan Golledge</a>,<sup>14,</sup><sup>15</sup> <a href="/pubmed/?term=Parsons%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066453913552" co-class="co-affbox">Mark W Parsons</a>,<sup>4</sup> <a href="/pubmed/?term=Malik%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066453911568" co-class="co-affbox">Rainer Malik</a>,<sup>16</sup> <a href="/pubmed/?term=McEvoy%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066525933648" co-class="co-affbox">Mark McEvoy</a>,<sup>1,</sup><sup>9,</sup><sup>17</sup> <a href="/pubmed/?term=Biros%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066526222768" co-class="co-affbox">Erik Biros</a>,<sup>14</sup> <a href="/pubmed/?term=Lewis%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066526220784" co-class="co-affbox">Martin D Lewis</a>,<sup>7,</sup><sup>18</sup> <a href="/pubmed/?term=Lincz%20LF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066448667616" co-class="co-affbox">Lisa F Lincz</a>,<sup>4,</sup><sup>6,</sup><sup>19</sup> <a href="/pubmed/?term=Peel%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066457369056" co-class="co-affbox">Roseanne Peel</a>,<sup>1,</sup><sup>9,</sup><sup>17</sup> <a href="/pubmed/?term=Oldmeadow%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066445286704" co-class="co-affbox">Christopher Oldmeadow</a>,<sup>9,</sup><sup>20</sup> <a href="/pubmed/?term=Smith%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066445283984" co-class="co-affbox">Wayne Smith</a>,<sup>9,</sup><sup>17</sup> <a href="/pubmed/?term=Moscato%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066444252240" co-class="co-affbox">Pablo Moscato</a>,<sup>2,</sup><sup>21</sup> <a href="/pubmed/?term=Barlera%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066457824464" co-class="co-affbox">Simona Barlera</a>,<sup>22</sup> <a href="/pubmed/?term=Bevan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066457822480" co-class="co-affbox">Steve Bevan</a>,<sup>23</sup> <a href="/pubmed/?term=Bis%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066458705216" co-class="co-affbox">Joshua C Bis</a>,<sup>24</sup> <a href="/pubmed/?term=Boerwinkle%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066458703232" co-class="co-affbox">Eric Boerwinkle</a>,<sup>25,</sup><sup>26</sup> <a href="/pubmed/?term=Boncoraglio%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066526892208" co-class="co-affbox">Giorgio B Boncoraglio</a>,<sup>27</sup> <a href="/pubmed/?term=Brott%20TG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066526890224" co-class="co-affbox">Thomas G Brott</a>,<sup>28</sup> <a href="/pubmed/?term=Brown%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066458079152" co-class="co-affbox">Robert D Brown, Jr</a>,<sup>29</sup> <a href="/pubmed/?term=Cheng%20YC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066458076912" co-class="co-affbox">Yu-Ching Cheng</a>,<sup>30</sup> <a href="/pubmed/?term=Cole%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066526881536" co-class="co-affbox">John W Cole</a>,<sup>31,</sup><sup>32</sup> <a href="/pubmed/?term=Cotlarciuc%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066455320096" co-class="co-affbox">Ioana Cotlarciuc</a>,<sup>33</sup> <a href="/pubmed/?term=Devan%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066455318112" co-class="co-affbox">William J Devan</a>,<sup>34,</sup><sup>35,</sup><sup>36</sup> <a href="/pubmed/?term=Fornage%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066455068720" co-class="co-affbox">Myriam Fornage</a>,<sup>25,</sup><sup>26</sup> <a href="/pubmed/?term=Furie%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066453841008" co-class="co-affbox">Karen L Furie</a>,<sup>35,</sup><sup>36</sup> <a href="/pubmed/?term=Gr%26%23x000e9%3Btarsd%26%23x000f3%3Bttir%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066453838288" co-class="co-affbox">Sólveig Grétarsdóttir</a>,<sup>37</sup> <a href="/pubmed/?term=Gschwendtner%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066448178064" co-class="co-affbox">Andreas Gschwendtner</a>,<sup>16</sup> <a href="/pubmed/?term=Ikram%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066448176080" co-class="co-affbox">M Arfan Ikram</a>,<sup>38,</sup><sup>39,</sup><sup>40</sup> <a href="/pubmed/?term=Longstreth%20WT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066528596672" co-class="co-affbox">W T Longstreth, Jr</a>,<sup>41,</sup><sup>42,</sup><sup>43</sup> <a href="/pubmed/?term=Meschia%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066458210752" co-class="co-affbox">James F Meschia</a>,<sup>28</sup> <a href="/pubmed/?term=Mitchell%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066458208768" co-class="co-affbox">Braxton D Mitchell</a>,<sup>30</sup> <a href="/pubmed/?term=Mosley%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066457062528" co-class="co-affbox">Thomas H Mosley</a>,<sup>44</sup> <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066457060544" co-class="co-affbox">Michael A Nalls</a>,<sup>45</sup> <a href="/pubmed/?term=Parati%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066456307696" co-class="co-affbox">Eugenio A Parati</a>,<sup>27</sup> <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066456305712" co-class="co-affbox">Bruce M Psaty</a>,<sup>24,</sup><sup>41,</sup><sup>46,</sup><sup>47</sup> <a href="/pubmed/?term=Sharma%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066454385680" co-class="co-affbox">Pankaj Sharma</a>,<sup>33</sup> <a href="/pubmed/?term=Stefansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066445471184" co-class="co-affbox">Kari Stefansson</a>,<sup>37,</sup><sup>48</sup> <a href="/pubmed/?term=Thorleifsson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066452820592" co-class="co-affbox">Gudmar Thorleifsson</a>,<sup>37</sup> <a href="/pubmed/?term=Thorsteinsdottir%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066452818608" co-class="co-affbox">Unnur Thorsteinsdottir</a>,<sup>37,</sup><sup>48</sup> <a href="/pubmed/?term=Traylor%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066492191216" co-class="co-affbox">Matthew Traylor</a>,<sup>23</sup> <a href="/pubmed/?term=Verhaaren%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066492189232" co-class="co-affbox">Benjamin F J Verhaaren</a>,<sup>38,</sup><sup>40</sup> <a href="/pubmed/?term=Wiggins%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066455119312" co-class="co-affbox">Kerri L Wiggins</a>,<sup>24</sup> <a href="/pubmed/?term=Worrall%20BB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066455117328" co-class="co-affbox">Bradford B Worrall</a>,<sup>49</sup> The Australian Stroke Genetics Collaborative,<sup>50</sup> The International Stroke Genetics Consortium,<sup>50</sup> The Wellcome Trust Case Control Consortium 2,<sup>50</sup> <a href="/pubmed/?term=Sudlow%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066445171424" co-class="co-affbox">Cathie Sudlow</a>,<sup>51,</sup><sup>52</sup> <a href="/pubmed/?term=Rothwell%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066491819600" co-class="co-affbox">Peter M Rothwell</a>,<sup>53</sup> <a href="/pubmed/?term=Farrall%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066491817616" co-class="co-affbox">Martin Farrall</a>,<sup>54,</sup><sup>55</sup> <a href="/pubmed/?term=Dichgans%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066448011184" co-class="co-affbox">Martin Dichgans</a>,<sup>16</sup> <a href="/pubmed/?term=Rosand%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066448009200" co-class="co-affbox">Jonathan Rosand</a>,<sup>34,</sup><sup>35,</sup><sup>36</sup> <a href="/pubmed/?term=Markus%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066454408096" co-class="co-affbox">Hugh S Markus</a>,<sup>23</sup> <a href="/pubmed/?term=Scott%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066454498800" co-class="co-affbox">Rodney J Scott</a>,<sup>2,</sup><sup>6,</sup><sup>56,</sup><sup>57</sup> <a href="/pubmed/?term=Levi%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066454440864" co-class="co-affbox">Christopher Levi</a>,<sup>4,</sup><sup>57</sup> and  <a href="/pubmed/?term=Attia%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190" class="affpopup" co-rid="_co_idm140066454438144" co-class="co-affbox">John Attia</a><sup>1,</sup><sup>2,</sup><sup>57</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140066448326192"><h3 class="no_margin">Elizabeth G Holliday</h3><p><sup>1</sup>Centre for Clinical Epidemiology and Biostatistics, School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>2</sup>Centre for Bioinformatics, Biomarker Discovery and Information-Based Medicine, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Holliday%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Elizabeth G Holliday</a></div></div><div id="_co_idm140066457829552"><h3 class="no_margin">Jane M Maguire</h3><p><sup>3</sup>School of Nursing and Midwifery, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>4</sup>Centre for Brain and Mental Health Research, University of Newcastle and Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><p><sup>5</sup>Department of Neurosciences, Gosford Hospital, Central Coast Area Health, Gosford, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Maguire%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Jane M Maguire</a></div></div><div id="_co_idm140066457826656"><h3 class="no_margin">Tiffany-Jane Evans</h3><p><sup>2</sup>Centre for Bioinformatics, Biomarker Discovery and Information-Based Medicine, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><p><sup>6</sup>School of Biomedical Sciences and Pharmacy, University of Newcastle, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Evans%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Tiffany-Jane Evans</a></div></div><div id="_co_idm140066458488128"><h3 class="no_margin">Simon A Koblar</h3><p><sup>7</sup>Stroke Research Program, School of Medicine, University of Adelaide, Adelaide, South Australia, Australia</p><p><sup>8</sup>Stroke Unit, Department of Neurology, Queen Elizabeth Hospital, Adelaide, South Australia, Australia</p><div>Find articles by <a href="/pubmed/?term=Koblar%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Simon A Koblar</a></div></div><div id="_co_idm140066493386960"><h3 class="no_margin">Jim Jannes</h3><p><sup>7</sup>Stroke Research Program, School of Medicine, University of Adelaide, Adelaide, South Australia, Australia</p><p><sup>8</sup>Stroke Unit, Department of Neurology, Queen Elizabeth Hospital, Adelaide, South Australia, Australia</p><div>Find articles by <a href="/pubmed/?term=Jannes%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Jim Jannes</a></div></div><div id="_co_idm140066493384240"><h3 class="no_margin">Jonathan W Sturm</h3><p><sup>5</sup>Department of Neurosciences, Gosford Hospital, Central Coast Area Health, Gosford, New South Wales, Australia</p><p><sup>9</sup>School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Sturm%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Jonathan W Sturm</a></div></div><div id="_co_idm140066454686288"><h3 class="no_margin">Graeme J Hankey</h3><p><sup>10</sup>Department of Neurology, Royal Perth Hospital, Perth, Western Australia, Australia</p><p><sup>11</sup>School of Medicine and Pharmacology, University of Western Australia, Perth, Western Australia, Australia</p><div>Find articles by <a href="/pubmed/?term=Hankey%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Graeme J Hankey</a></div></div><div id="_co_idm140066527241376"><h3 class="no_margin">Ross Baker</h3><p><sup>12</sup>Department of Haematology, Royal Perth Hospital, Perth, Western Australia, Australia</p><p><sup>13</sup>Centre for Thrombosis and Haemophilia, Murdoch University, Perth, Western Australia, Australia</p><div>Find articles by <a href="/pubmed/?term=Baker%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Ross Baker</a></div></div><div id="_co_idm140066527238656"><h3 class="no_margin">Jonathan Golledge</h3><p><sup>14</sup>Vascular Biology Unit, School of Medicine and Dentistry, James Cook University, Townsville, Queensland, Australia</p><p><sup>15</sup>Department of Vascular Surgery, The Townsville Hospital, Townsville, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Golledge%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Jonathan Golledge</a></div></div><div id="_co_idm140066453913552"><h3 class="no_margin">Mark W Parsons</h3><p><sup>4</sup>Centre for Brain and Mental Health Research, University of Newcastle and Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Parsons%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Mark W Parsons</a></div></div><div id="_co_idm140066453911568"><h3 class="no_margin">Rainer Malik</h3><p><sup>16</sup>Institute for Stroke and Dementia Research (ISD), Medical Center, Klinikum der Universität München, Ludwig-Maximilians-University, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Malik%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Rainer Malik</a></div></div><div id="_co_idm140066525933648"><h3 class="no_margin">Mark McEvoy</h3><p><sup>1</sup>Centre for Clinical Epidemiology and Biostatistics, School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>9</sup>School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>17</sup>Public Health Research Program, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=McEvoy%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Mark McEvoy</a></div></div><div id="_co_idm140066526222768"><h3 class="no_margin">Erik Biros</h3><p><sup>14</sup>Vascular Biology Unit, School of Medicine and Dentistry, James Cook University, Townsville, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Biros%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Erik Biros</a></div></div><div id="_co_idm140066526220784"><h3 class="no_margin">Martin D Lewis</h3><p><sup>7</sup>Stroke Research Program, School of Medicine, University of Adelaide, Adelaide, South Australia, Australia</p><p><sup>18</sup>Discipline of Genetics, School of Molecular &amp; Biomedical Sciences, University of Adelaide, Adelaide, South Australia, Australia</p><div>Find articles by <a href="/pubmed/?term=Lewis%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Martin D Lewis</a></div></div><div id="_co_idm140066448667616"><h3 class="no_margin">Lisa F Lincz</h3><p><sup>4</sup>Centre for Brain and Mental Health Research, University of Newcastle and Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><p><sup>6</sup>School of Biomedical Sciences and Pharmacy, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>19</sup>Hunter Haematology Research Group, Calvary Mater Newcastle Hospital, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Lincz%20LF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Lisa F Lincz</a></div></div><div id="_co_idm140066457369056"><h3 class="no_margin">Roseanne Peel</h3><p><sup>1</sup>Centre for Clinical Epidemiology and Biostatistics, School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>9</sup>School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>17</sup>Public Health Research Program, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Peel%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Roseanne Peel</a></div></div><div id="_co_idm140066445286704"><h3 class="no_margin">Christopher Oldmeadow</h3><p><sup>9</sup>School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>20</sup>Clinical Research Design, IT and Statistical Support Unit, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Oldmeadow%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Christopher Oldmeadow</a></div></div><div id="_co_idm140066445283984"><h3 class="no_margin">Wayne Smith</h3><p><sup>9</sup>School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>17</sup>Public Health Research Program, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Smith%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Wayne Smith</a></div></div><div id="_co_idm140066444252240"><h3 class="no_margin">Pablo Moscato</h3><p><sup>2</sup>Centre for Bioinformatics, Biomarker Discovery and Information-Based Medicine, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><p><sup>21</sup>School of Electrical Engineering and Computer Science, University of Newcastle, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Moscato%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Pablo Moscato</a></div></div><div id="_co_idm140066457824464"><h3 class="no_margin">Simona Barlera</h3><p><sup>22</sup>Department of Cardiovascular Research, Istituto di Ricerche Farmacologiche Mario Negri, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Barlera%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Simona Barlera</a></div></div><div id="_co_idm140066457822480"><h3 class="no_margin">Steve Bevan</h3><p><sup>23</sup>Stroke and Dementia Research Centre, St. George’s University of London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Bevan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Steve Bevan</a></div></div><div id="_co_idm140066458705216"><h3 class="no_margin">Joshua C Bis</h3><p><sup>24</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Bis%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Joshua C Bis</a></div></div><div id="_co_idm140066458703232"><h3 class="no_margin">Eric Boerwinkle</h3><p><sup>25</sup>Institute of Molecular Medicine, University of Texas Health Science Center at Houston, Houston, Texas, USA</p><p><sup>26</sup>Human Genetics Center, University of Texas Health Science Center at Houston, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Boerwinkle%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Eric Boerwinkle</a></div></div><div id="_co_idm140066526892208"><h3 class="no_margin">Giorgio B Boncoraglio</h3><p><sup>27</sup>Department of Cerebrovascular Diseases, Fondazione Istituto di Ricovero e Cura a Carattere Scientifico (IRCCS) Istituto Neurologico Carlo Besta, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Boncoraglio%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Giorgio B Boncoraglio</a></div></div><div id="_co_idm140066526890224"><h3 class="no_margin">Thomas G Brott</h3><p><sup>28</sup>Department of Neurology, Mayo Clinic, Jacksonville, Florida, USA</p><div>Find articles by <a href="/pubmed/?term=Brott%20TG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Thomas G Brott</a></div></div><div id="_co_idm140066458079152"><h3 class="no_margin">Robert D Brown, Jr</h3><p><sup>29</sup>Department of Neurology, Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Brown%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Robert D Brown, Jr</a></div></div><div id="_co_idm140066458076912"><h3 class="no_margin">Yu-Ching Cheng</h3><p><sup>30</sup>Department of Medicine, University of Maryland, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Cheng%20YC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Yu-Ching Cheng</a></div></div><div id="_co_idm140066526881536"><h3 class="no_margin">John W Cole</h3><p><sup>31</sup>Baltimore Veterans Affairs Medical Center, Baltimore, Maryland, USA</p><p><sup>32</sup>School of Medicine, University of Maryland, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Cole%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">John W Cole</a></div></div><div id="_co_idm140066455320096"><h3 class="no_margin">Ioana Cotlarciuc</h3><p><sup>33</sup>Imperial College Cerebrovascular Research Unit (ICCRU), Imperial College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Cotlarciuc%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Ioana Cotlarciuc</a></div></div><div id="_co_idm140066455318112"><h3 class="no_margin">William J Devan</h3><p><sup>34</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, Massachusetts, USA</p><p><sup>35</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA</p><p><sup>36</sup>Department of Neurology, Harvard Medical School, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Devan%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">William J Devan</a></div></div><div id="_co_idm140066455068720"><h3 class="no_margin">Myriam Fornage</h3><p><sup>25</sup>Institute of Molecular Medicine, University of Texas Health Science Center at Houston, Houston, Texas, USA</p><p><sup>26</sup>Human Genetics Center, University of Texas Health Science Center at Houston, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Fornage%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Myriam Fornage</a></div></div><div id="_co_idm140066453841008"><h3 class="no_margin">Karen L Furie</h3><p><sup>35</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA</p><p><sup>36</sup>Department of Neurology, Harvard Medical School, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Furie%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Karen L Furie</a></div></div><div id="_co_idm140066453838288"><h3 class="no_margin">Sólveig Grétarsdóttir</h3><p><sup>37</sup>deCODE Genetics, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Gr%26%23x000e9%3Btarsd%26%23x000f3%3Bttir%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Sólveig Grétarsdóttir</a></div></div><div id="_co_idm140066448178064"><h3 class="no_margin">Andreas Gschwendtner</h3><p><sup>16</sup>Institute for Stroke and Dementia Research (ISD), Medical Center, Klinikum der Universität München, Ludwig-Maximilians-University, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Gschwendtner%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Andreas Gschwendtner</a></div></div><div id="_co_idm140066448176080"><h3 class="no_margin">M Arfan Ikram</h3><p><sup>38</sup>Department of Epidemiology, Erasmus Medical Center (MC)–University Medical Center, Rotterdam, The Netherlands</p><p><sup>39</sup>Department of Neurology, Erasmus MC–University Medical Center, Rotterdam, The Netherlands</p><p><sup>40</sup>Department of Radiology, Erasmus MC–University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Ikram%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">M Arfan Ikram</a></div></div><div id="_co_idm140066528596672"><h3 class="no_margin">W T Longstreth, Jr</h3><p><sup>41</sup>Department of Epidemiology, University of Washington, Seattle, Washington, USA</p><p><sup>42</sup>Department of Medicine, University of Washington, Seattle, Washington, USA</p><p><sup>43</sup>Department of Neurology, University of Washington, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Longstreth%20WT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">W T Longstreth, Jr</a></div></div><div id="_co_idm140066458210752"><h3 class="no_margin">James F Meschia</h3><p><sup>28</sup>Department of Neurology, Mayo Clinic, Jacksonville, Florida, USA</p><div>Find articles by <a href="/pubmed/?term=Meschia%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">James F Meschia</a></div></div><div id="_co_idm140066458208768"><h3 class="no_margin">Braxton D Mitchell</h3><p><sup>30</sup>Department of Medicine, University of Maryland, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Mitchell%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Braxton D Mitchell</a></div></div><div id="_co_idm140066457062528"><h3 class="no_margin">Thomas H Mosley</h3><p><sup>44</sup>Department of Medicine, University of Mississippi Medical Center, Jackson, Mississippi, USA</p><div>Find articles by <a href="/pubmed/?term=Mosley%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Thomas H Mosley</a></div></div><div id="_co_idm140066457060544"><h3 class="no_margin">Michael A Nalls</h3><p><sup>45</sup>Laboratory of Neurogenetics, National Institute on Aging, US National Institutes of Health, Bethesda, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Michael A Nalls</a></div></div><div id="_co_idm140066456307696"><h3 class="no_margin">Eugenio A Parati</h3><p><sup>27</sup>Department of Cerebrovascular Diseases, Fondazione Istituto di Ricovero e Cura a Carattere Scientifico (IRCCS) Istituto Neurologico Carlo Besta, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Parati%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Eugenio A Parati</a></div></div><div id="_co_idm140066456305712"><h3 class="no_margin">Bruce M Psaty</h3><p><sup>24</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, Washington, USA</p><p><sup>41</sup>Department of Epidemiology, University of Washington, Seattle, Washington, USA</p><p><sup>46</sup>Department of Health Services, University of Washington, Seattle, Washington, USA</p><p><sup>47</sup>Group Health Research Institute, Group Health, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Bruce M Psaty</a></div></div><div id="_co_idm140066454385680"><h3 class="no_margin">Pankaj Sharma</h3><p><sup>33</sup>Imperial College Cerebrovascular Research Unit (ICCRU), Imperial College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Sharma%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Pankaj Sharma</a></div></div><div id="_co_idm140066445471184"><h3 class="no_margin">Kari Stefansson</h3><p><sup>37</sup>deCODE Genetics, Reykjavik, Iceland</p><p><sup>48</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Stefansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Kari Stefansson</a></div></div><div id="_co_idm140066452820592"><h3 class="no_margin">Gudmar Thorleifsson</h3><p><sup>37</sup>deCODE Genetics, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Thorleifsson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Gudmar Thorleifsson</a></div></div><div id="_co_idm140066452818608"><h3 class="no_margin">Unnur Thorsteinsdottir</h3><p><sup>37</sup>deCODE Genetics, Reykjavik, Iceland</p><p><sup>48</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Thorsteinsdottir%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Unnur Thorsteinsdottir</a></div></div><div id="_co_idm140066492191216"><h3 class="no_margin">Matthew Traylor</h3><p><sup>23</sup>Stroke and Dementia Research Centre, St. George’s University of London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Traylor%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Matthew Traylor</a></div></div><div id="_co_idm140066492189232"><h3 class="no_margin">Benjamin F J Verhaaren</h3><p><sup>38</sup>Department of Epidemiology, Erasmus Medical Center (MC)–University Medical Center, Rotterdam, The Netherlands</p><p><sup>40</sup>Department of Radiology, Erasmus MC–University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Verhaaren%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Benjamin F J Verhaaren</a></div></div><div id="_co_idm140066455119312"><h3 class="no_margin">Kerri L Wiggins</h3><p><sup>24</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Wiggins%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Kerri L Wiggins</a></div></div><div id="_co_idm140066455117328"><h3 class="no_margin">Bradford B Worrall</h3><p><sup>49</sup>Department of Neurology, University of Virginia, Charlottesville, Virginia, USA</p><div>Find articles by <a href="/pubmed/?term=Worrall%20BB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Bradford B Worrall</a></div></div><div id="_co_idm140066445171424"><h3 class="no_margin">Cathie Sudlow</h3><p><sup>51</sup>Division of Clinical Neurosciences, University of Edinburgh, Edinburgh, UK</p><p><sup>52</sup>Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh, UK</p><div>Find articles by <a href="/pubmed/?term=Sudlow%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Cathie Sudlow</a></div></div><div id="_co_idm140066491819600"><h3 class="no_margin">Peter M Rothwell</h3><p><sup>53</sup>Department of Clinical Neurology, John Radcliffe Hospital, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Rothwell%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Peter M Rothwell</a></div></div><div id="_co_idm140066491817616"><h3 class="no_margin">Martin Farrall</h3><p><sup>54</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><p><sup>55</sup>Department of Cardiovascular Medicine, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Farrall%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Martin Farrall</a></div></div><div id="_co_idm140066448011184"><h3 class="no_margin">Martin Dichgans</h3><p><sup>16</sup>Institute for Stroke and Dementia Research (ISD), Medical Center, Klinikum der Universität München, Ludwig-Maximilians-University, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Dichgans%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Martin Dichgans</a></div></div><div id="_co_idm140066448009200"><h3 class="no_margin">Jonathan Rosand</h3><p><sup>34</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, Massachusetts, USA</p><p><sup>35</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA</p><p><sup>36</sup>Department of Neurology, Harvard Medical School, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Rosand%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Jonathan Rosand</a></div></div><div id="_co_idm140066454408096"><h3 class="no_margin">Hugh S Markus</h3><p><sup>23</sup>Stroke and Dementia Research Centre, St. George’s University of London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Markus%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Hugh S Markus</a></div></div><div id="_co_idm140066454498800"><h3 class="no_margin">Rodney J Scott</h3><p><sup>2</sup>Centre for Bioinformatics, Biomarker Discovery and Information-Based Medicine, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><p><sup>6</sup>School of Biomedical Sciences and Pharmacy, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>56</sup>Division of Genetics, Hunter Area Pathology Service, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Scott%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Rodney J Scott</a></div></div><div id="_co_idm140066454440864"><h3 class="no_margin">Christopher Levi</h3><p><sup>4</sup>Centre for Brain and Mental Health Research, University of Newcastle and Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Levi%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">Christopher Levi</a></div></div><div id="_co_idm140066454438144"><h3 class="no_margin">John Attia</h3><p><sup>1</sup>Centre for Clinical Epidemiology and Biostatistics, School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</p><p><sup>2</sup>Centre for Bioinformatics, Biomarker Discovery and Information-Based Medicine, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Attia%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22941190">John Attia</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140066455193744_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140066455193744_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140066455193744_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Centre for Clinical Epidemiology and Biostatistics, School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Centre for Bioinformatics, Biomarker Discovery and Information-Based Medicine, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>School of Nursing and Midwifery, University of Newcastle, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Centre for Brain and Mental Health Research, University of Newcastle and Hunter Medical Research Institute, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Neurosciences, Gosford Hospital, Central Coast Area Health, Gosford, New South Wales, Australia</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>School of Biomedical Sciences and Pharmacy, University of Newcastle, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Stroke Research Program, School of Medicine, University of Adelaide, Adelaide, South Australia, Australia</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Stroke Unit, Department of Neurology, Queen Elizabeth Hospital, Adelaide, South Australia, Australia</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>School of Medicine and Public Health, University of Newcastle, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Neurology, Royal Perth Hospital, Perth, Western Australia, Australia</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>School of Medicine and Pharmacology, University of Western Australia, Perth, Western Australia, Australia</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Haematology, Royal Perth Hospital, Perth, Western Australia, Australia</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Centre for Thrombosis and Haemophilia, Murdoch University, Perth, Western Australia, Australia</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Vascular Biology Unit, School of Medicine and Dentistry, James Cook University, Townsville, Queensland, Australia</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Vascular Surgery, The Townsville Hospital, Townsville, Queensland, Australia</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Institute for Stroke and Dementia Research (ISD), Medical Center, Klinikum der Universität München, Ludwig-Maximilians-University, Munich, Germany</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Public Health Research Program, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Discipline of Genetics, School of Molecular &amp; Biomedical Sciences, University of Adelaide, Adelaide, South Australia, Australia</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Hunter Haematology Research Group, Calvary Mater Newcastle Hospital, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Clinical Research Design, IT and Statistical Support Unit, Hunter Medical Research Institute, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>School of Electrical Engineering and Computer Science, University of Newcastle, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Cardiovascular Research, Istituto di Ricerche Farmacologiche Mario Negri, Milan, Italy</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Stroke and Dementia Research Centre, St. George’s University of London, London, UK</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, Washington, USA</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Institute of Molecular Medicine, University of Texas Health Science Center at Houston, Houston, Texas, USA</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Human Genetics Center, University of Texas Health Science Center at Houston, Houston, Texas, USA</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Department of Cerebrovascular Diseases, Fondazione Istituto di Ricovero e Cura a Carattere Scientifico (IRCCS) Istituto Neurologico Carlo Besta, Milan, Italy</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Department of Neurology, Mayo Clinic, Jacksonville, Florida, USA</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Department of Neurology, Mayo Clinic, Rochester, Minnesota, USA</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Department of Medicine, University of Maryland, Baltimore, Maryland, USA</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Baltimore Veterans Affairs Medical Center, Baltimore, Maryland, USA</div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>School of Medicine, University of Maryland, Baltimore, Maryland, USA</div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Imperial College Cerebrovascular Research Unit (ICCRU), Imperial College London, London, UK</div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, Massachusetts, USA</div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA</div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>Department of Neurology, Harvard Medical School, Boston, Massachusetts, USA</div><div class="fm-affl" id="A37" lang="en"><sup>37</sup>deCODE Genetics, Reykjavik, Iceland</div><div class="fm-affl" id="A38" lang="en"><sup>38</sup>Department of Epidemiology, Erasmus Medical Center (MC)–University Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="A39" lang="en"><sup>39</sup>Department of Neurology, Erasmus MC–University Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="A40" lang="en"><sup>40</sup>Department of Radiology, Erasmus MC–University Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="A41" lang="en"><sup>41</sup>Department of Epidemiology, University of Washington, Seattle, Washington, USA</div><div class="fm-affl" id="A42" lang="en"><sup>42</sup>Department of Medicine, University of Washington, Seattle, Washington, USA</div><div class="fm-affl" id="A43" lang="en"><sup>43</sup>Department of Neurology, University of Washington, Seattle, Washington, USA</div><div class="fm-affl" id="A44" lang="en"><sup>44</sup>Department of Medicine, University of Mississippi Medical Center, Jackson, Mississippi, USA</div><div class="fm-affl" id="A45" lang="en"><sup>45</sup>Laboratory of Neurogenetics, National Institute on Aging, US National Institutes of Health, Bethesda, Maryland, USA</div><div class="fm-affl" id="A46" lang="en"><sup>46</sup>Department of Health Services, University of Washington, Seattle, Washington, USA</div><div class="fm-affl" id="A47" lang="en"><sup>47</sup>Group Health Research Institute, Group Health, Seattle, Washington, USA</div><div class="fm-affl" id="A48" lang="en"><sup>48</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland</div><div class="fm-affl" id="A49" lang="en"><sup>49</sup>Department of Neurology, University of Virginia, Charlottesville, Virginia, USA</div><div class="fm-affl" id="A50" lang="en"><sup>51</sup>Division of Clinical Neurosciences, University of Edinburgh, Edinburgh, UK</div><div class="fm-affl" id="A51" lang="en"><sup>52</sup>Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh, UK</div><div class="fm-affl" id="A52" lang="en"><sup>53</sup>Department of Clinical Neurology, John Radcliffe Hospital, Oxford, UK</div><div class="fm-affl" id="A53" lang="en"><sup>54</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A54" lang="en"><sup>55</sup>Department of Cardiovascular Medicine, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A55" lang="en"><sup>56</sup>Division of Genetics, Hunter Area Pathology Service, Newcastle, New South Wales, Australia</div><div id="FN1">Correspondence should be addressed to E.G.H. (<a href="mailto:dev@null" data-email="ua.ude.eltsacwen@yadilloh.zil" class="oemail">ua.ude.eltsacwen@yadilloh.zil</a>)</div><div id="FN2"><sup>50</sup>A full list of members is provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary Note</a>.</div><div id="FN3"><sup>57</sup>These authors jointly directed this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140066455193744_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22941190" target="pmc_ext" ref="reftype=publisher&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3651583/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplementary data and figures.</strong></div><div><a href="/pmc/articles/PMC3651583/bin/NIHMS457841-supplement-supplementary_data_and_figures.pdf" data-ga-action="click_feat_suppl">NIHMS457841-supplement-supplementary_data_and_figures.pdf</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;3FBBCE04-7A49-4A8B-91B7-16CB32AE4416</div></div></dd></dl></div><div id="idm140066448380864" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140066448380864title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Genome-wide association studies (GWAS) have not consistently detected replicable genetic risk factors for ischemic stroke, potentially due to etiological heterogeneity of this trait. We performed GWAS of ischemic stroke and a major ischemic stroke subtype (large artery atherosclerosis, LAA) using 1,162 ischemic stroke cases (including 421 LAA cases) and 1,244 population controls from Australia. Evidence for a genetic influence on ischemic stroke risk was detected, but this influence was higher and more significant for the LAA subtype. We identified a new LAA susceptibility locus on chromosome 6p21.1 (rs556621: odds ratio (OR) = 1.62, <em>P</em> = 3.9 × 10<sup>−8</sup>) and replicated this association in 1,715 LAA cases and 52,695 population controls from 10 independent population cohorts (meta-analysis replication OR = 1.15, <em>P</em> = 3.9 × 10<sup>−4</sup>; discovery and replication combined OR = 1.21, <em>P</em> = 4.7 × 10<sup>−8</sup>). This study identifies a genetic risk locus for LAA and shows how analyzing etiological subtypes may better identify genetic risk alleles for ischemic stroke.</p></div></div><div id="idm140066457805984" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Stroke affects approximately 15 million persons worldwide each year<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup> and is a leading cause of death and adult acquired disability<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905482">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905500">3</a></sup>. The vast majority of strokes are ischemic, involving cerebral artery blockage by atherosclerotic plaque or embolus. Although clinical risk factors for ischemic stroke are well established<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905513">4</a></sup>, the genetic risk alleles are incompletely identified. Genetic influences on stroke risk are supported, however, by higher concordance among monozygotic than dizygotic twins<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905486">5</a></sup>, increased risk among family members of affected individuals<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905512">6</a></sup> and high heritability of intermediate predictors, including carotid intima-media thickness (IMT: <em>h</em><sup>2</sup> ≈ 30–60%)<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905521">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905510">8</a></sup> and white matter lesions (<em>h</em><sup>2</sup> ≈ 50–70%)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905520">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905497">10</a></sup>.</p><p id="P3">With the exception of the 4q25 locus associated with atrial fibrillation and ischemic stroke<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905492">11</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905493">12</a></sup>, the 9p21 region associated with coronary artery disease and ischemic stroke<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905487">13</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905480">14</a></sup> and a recently described 7p21.1 association with LAA<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905508">15</a></sup>, GWAS for ischemic stroke have identified few convincingly associated variants. Inability to replicate many reported associations may be attributable to phenotypic heterogeneity, a challenge that could be partly addressed by more complete subtyping of ischemic stroke etiology. At least three major ischemic stroke etiologi-cal types are commonly distinguished: (i) large artery atherosclerosis (LAA); (ii) cardioembolism (CE); and (iii) small vessel occlusion (SVO)<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905489">16</a></sup>. Genetic heterogeneity may contribute to this phenotypic diversity; a recent, well-powered GWAS of ischemic stroke detected heterogeneity of risk locus effects across stroke subtypes<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905495">15</a></sup>, and family studies have also identified differences in subtype heritability, owing perhaps to variable roles of heritable intermediate phenotypes, such as hypertension and large vessel atherosclerosis<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905494">17</a></sup>. The greatest familial risk has been associated with LAA, for which family history confers significant risk, even beyond the seventh decade of life<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905504">6</a></sup>.</p><p id="P4">We conducted a GWAS of ischemic stroke in an Australian sample of European ancestry involving 1,230 cases and 1,280 population controls. The causal subtype of ischemic stroke was classified using TOAST criteria<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905485">16</a></sup>. Demographic and clinical characteristics of the Australian Stroke Genetics Collaborative (ASGC) data set are summarized in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>.</p><p id="P5">After quality control filtering of genotype data, data on 551, 514 SNPs from 1,162 ischemic stroke cases and 1,244 controls were used for genotype imputation and genetic analysis. Before performing genome-wide association analyses, we assessed the genetic contribution to ischemic stroke and the LAA, CE and SVO subtypes using a recent method<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905488">18</a></sup> that estimates the proportion of phenotypic variance (<em>V<sub>g</sub></em>/<em>V<sub>p</sub></em>) attributable to variation in genotyped SNPs, where <em>V<sub>g</sub></em> is the component of phenotypic variance attributable to variation in genotyped SNPs and <em>V<sub>p</sub></em> is the total observed phenotypic variance. For ischemic stroke, the estimated genetic load was substantial (<em>V<sub>g</sub></em>/<em>V<sub>p</sub></em>
<sub>IS</sub> = 0.39), with SNPs explaining a significant proportion of phenotypic variation (<em>P</em> = 4.5 × 10<sup>−4</sup>). For cases classified in the LAA subtype, we observed a higher, more significant estimate of genetic load (<em>V<sub>g</sub></em>/<em>V<sub>p</sub></em>
<sub>LAA</sub> = 0.66; <em>P</em> = 5.6 × 10<sup>−5</sup>), consistent with previous reports of high familial risk for LAA<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905491">6</a></sup>. Evidence for genetic contribution was less significant for the CE and SVO subtypes (<em>V<sub>g</sub></em>/<em>V<sub>p</sub></em>
<sub>CE</sub> = 0.6, <em>P</em> = 0.0026 and <em>V<sub>g</sub></em>/<em>V<sub>p</sub></em>
<sub>SVO</sub> = 0.1, <em>P</em> = 0.33, respectively; <a href="/pmc/articles/PMC3651583/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p14">Proportion of case-control phenotypic variation explained by genome-wide SNP data for all ischemic stroke, LAA, SVO and CE</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140066525950864" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Phenotype</th><th rowspan="1" colspan="1" valign="top" align="right">Cases<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Controls<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" overflow="scroll"><msubsup><mi>σ</mi><mi>g</mi><mn>2</mn></msubsup><mo>/</mo><msubsup><mi>σ</mi><mi>p</mi><mn>2</mn></msubsup></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M2" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 144%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="msubsup" id="MathJax-Span-3"><span style="display: inline-block; position: relative; width: 1.051em; height: 0px;"><span style="position: absolute; clip: rect(3.418em, 1000.57em, 4.167em, -1000em); top: -4.002em; left: 0em;"><span class="mi" id="MathJax-Span-4" style="font-family: MathJax_Math; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.001em;"></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.378em, 1000.43em, 4.156em, -1000em); top: -4.347em; left: 0.622em;"><span class="mn" id="MathJax-Span-5" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.536em, 1000.41em, 4.301em, -1000em); top: -3.854em; left: 0.571em;"><span class="mi" id="MathJax-Span-6" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main;">/</span><span class="msubsup" id="MathJax-Span-8"><span style="display: inline-block; position: relative; width: 1.051em; height: 0px;"><span style="position: absolute; clip: rect(3.418em, 1000.57em, 4.167em, -1000em); top: -4.002em; left: 0em;"><span class="mi" id="MathJax-Span-9" style="font-family: MathJax_Math; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.001em;"></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.378em, 1000.43em, 4.156em, -1000em); top: -4.347em; left: 0.622em;"><span class="mn" id="MathJax-Span-10" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.536em, 1000.43em, 4.294em, -1000em); top: -3.854em; left: 0.571em;"><span class="mi" id="MathJax-Span-11" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">p</span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" overflow="scroll"><msubsup><mi>σ</mi><mi>g</mi><mn>2</mn></msubsup><mo>/</mo><msubsup><mi>σ</mi><mi>p</mi><mn>2</mn></msubsup></math></script> (s.e)<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="right">LRT<sup>c</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value<sup>d</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Ischemic stroke</td><td rowspan="1" colspan="1" valign="top" align="right">1,079</td><td rowspan="1" colspan="1" valign="top" align="center">1,172</td><td rowspan="1" colspan="1" valign="top" align="center">0.39 (0.15)</td><td rowspan="1" colspan="1" valign="top" align="right">11.04</td><td rowspan="1" colspan="1" valign="top" align="center">4.5 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LAA</td><td rowspan="1" colspan="1" valign="top" align="right">400</td><td rowspan="1" colspan="1" valign="top" align="center">1,172</td><td rowspan="1" colspan="1" valign="top" align="center">0.66 (0.21)</td><td rowspan="1" colspan="1" valign="top" align="right">14.94</td><td rowspan="1" colspan="1" valign="top" align="center">5.6 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SVO</td><td rowspan="1" colspan="1" valign="top" align="right">288</td><td rowspan="1" colspan="1" valign="top" align="center">1,172</td><td rowspan="1" colspan="1" valign="top" align="center">0.10 (0.24)</td><td rowspan="1" colspan="1" valign="top" align="right">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">3.3 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">CE</td><td rowspan="1" colspan="1" valign="top" align="right">226</td><td rowspan="1" colspan="1" valign="top" align="center">1,172</td><td rowspan="1" colspan="1" valign="top" align="center">0.60 (0.25)</td><td rowspan="1" colspan="1" valign="top" align="right">7.78</td><td rowspan="1" colspan="1" valign="top" align="center">2.6 × 10<sup>−3</sup></td></tr></tbody></table></div><div id="largeobj_idm140066525950864" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3651583/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p15" class="p p-first-last">Genetic relationships between individuals were estimated using 457,533 SNPs.</p></div><div id="TFN2"><sup>a</sup>Smaller sample sizes compared with the GWAS because of additional quality control filtering conducted before this analysis.</div><div id="TFN3"><sup>b</sup>Estimated proportion (standard error, s.e.) of variation in case-control status explained by all SNPs.</div><div id="TFN4"><sup>c</sup>Likelihood ratio test (LRT) statistic.</div><div id="TFN5"><sup>d</sup><em>P</em> values were calculated assuming that the LRT is distributed as a 50:50 mixture of a point mass at zero and 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-2-Frame" tabindex="0" style=""><nobr><span class="math" id="M3" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 144%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-13"><span class="msubsup" id="MathJax-Span-14"><span style="display: inline-block; position: relative; width: 1.055em; height: 0px;"><span style="position: absolute; clip: rect(3.407em, 1000.6em, 4.36em, -1000em); top: -4.002em; left: 0em;"><span class="mi" id="MathJax-Span-15" style="font-family: MathJax_Math; font-style: italic;">χ</span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.378em, 1000.43em, 4.156em, -1000em); top: -4.347em; left: 0.626em;"><span class="mn" id="MathJax-Span-16" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.378em, 1000.43em, 4.156em, -1000em); top: -3.696em; left: 0.626em;"><span class="mn" id="MathJax-Span-17" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup></math></script> under the null hypothesis.</div></div></div><p id="P6">We performed two primary GWAS in the Australian discovery sample, comparing (i) all ischemic stroke cases (<em>n</em> = 1,162) and (ii) LAA cases (<em>n</em> = 421) with population controls (<em>n</em> = 1,244). GWAS of the CE and SVO subtypes, which both had fewer cases and a less significant <em>V<sub>g</sub></em>/<em>V<sub>p</sub></em> estimate, were performed as supplementary analyses (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 1 and 2</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 2 and 3</a>). Genotypic effects were estimated using logistic regression models (1-degree-of-freedom additive trend tests) adjusted for age and sex. Results were compared with a prespecified significance threshold of 5 × 10<sup>−8</sup>, corresponding to Bonferroni adjustment for 1 × 10<sup>6</sup> independent tests. Quantile-quantile plots indicated excellent quality of the GWAS data and an absence of systematic bias caused by population substructure or other artifacts (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>).</p><p id="P7">Analyses of ischemic stroke detected the strongest signals at several SNPs within the <em>SLC5A4</em> gene on chromosome 22q12.3 (<a href="/pmc/articles/PMC3651583/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Peak association was detected at rs5998322 (<em><sub>P</sub></em><sub>trend</sub> = 3.91 × 10<sup>−7</sup>; OR = 1.97, 95% confidence interval (CI) = 1.51–2.57) within exon 11. A strong signal was also detected 4 Mb downstream of this peak at a number of SNPs located within and upstream of the <em>APOL2</em> gene (peak association at rs4479522: <em><sub>P</sub></em><sub>trend</sub> = 3.23 × 10<sup>−6</sup>; OR = 1.34, 95% CI = 1.18–1.51). Analysis of rs5998322 adjusted for allele dosage at rs4479522 produced similar results to the unadjusted analysis (<em><sub>P</sub></em><sub>trend</sub> = 4.47 × 10<sup>−7</sup>), suggesting independence of the two associated loci at 22q.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3651583/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140066492802848" class="figure"><a href="/pmc/articles/PMC3651583/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3651583_nihms457841f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms457841f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3651583/bin/nihms457841f1.jpg"></a></div><div id="largeobj_idm140066492802848" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3651583/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3651583/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p11">Genome-wide association results. (<strong>a</strong>,<strong>b</strong>) Data are shown for ischemic stroke (<strong>a</strong>) and LAA (<strong>b</strong>). The plots show −log<sub>10</sub>-transformed <em>P</em> values for genotyped and imputed SNPs with respect to their physical positions. The threshold for association at genome-wide significance (<em>P</em> = 5 × 10<sup>−8</sup>) is shown by the upper dashed line, and the lower dashed line corresponds to <em>P</em> = 1 × 10<sup>−5</sup>.</p></div></div></div><p id="P8">The GWAS of LAA detected two associated SNPs on chromosome 6p21.1 exceeding the prespecified threshold for genome-wide significance (<em>α</em> = 5 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC3651583/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figs. 1</span></a> and <a href="/pmc/articles/PMC3651583/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2).</span></span><span>2</span></a>). These variants, rs556621 (<em><sub>P</sub></em><sub>trend</sub> = 3.92 × 10<sup>−8</sup>; OR (A allele) = 1.62, 95% CI = 1.36–1.93) and rs556512 (<em><sub>P</sub></em><sub>trend</sub> = 4.25 × 10<sup>−8</sup>; OR (A allele) = 1.62, 95% CI = 1.36–1.93) were in perfect linkage disequilibrium (LD) in HapMap Phase 2 Utah residents of Northern and Western European ancestry (CEU) data (<em>r</em><sup>2</sup> = 1, <em>D</em>′ = 1; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>), with a minor (A) allele population frequency of 0.33. The rs556621 SNP was directly genotyped in our sample, whereas rs556512 was imputed with excellent reliability (imputation <em>r</em><sup>2</sup> = 0.99). Very similar effect sizes for rs556621 were estimated in logistic models further adjusted for the first ten ancestry principal components and several correlated clinical risk factors (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>), indicating a lack of confounding by population substructure or clinically related heritable traits. Consistent but attenuated association of the 6p21.1 variants was observed for the broad ischemic stroke phenotype, with peak association also detected at rs556621 (<em>P</em> = 5.6 × 10<sup>−5</sup>; OR (A allele) = 1.29, 95% CI = 1.14–1.47) (<a href="/pmc/articles/PMC3651583/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Supplementary analyses of CE and SVO subtypes revealed no association with rs556621 (<em>P</em> = 0.73 and 0.39, respectively; data not shown). In addition to the 6p21.1 locus, the LAA GWAS also detected clusters of suggestively associated SNPs (<em>P</em> &lt; 1 × 10<sup>−5</sup>) at 14q32.33 and the second 22q12.3 locus detected in the GWAS of ischemic stroke (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3651583/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140066492799088" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms457841f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms457841f2.jpg" src="/pmc/articles/PMC3651583/bin/nihms457841f2.jpg"></div></a><div id="largeobj_idm140066492799088" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3651583/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3651583/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3651583/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p12">Regional association results for the chromosome 6p21.1 locus showing association at genome-wide significance with LAA. The index associated SNP is labeled (rs556621: <em>P</em> = 3.9 × 10<sup>−8</sup>).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p20">Association of rs556621 with LAA and overall ischemic stroke in discovery, replication and combined cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140066449660304" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">Phenotype</th><th colspan="3" rowspan="1" valign="bottom" align="center">Discovery
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">Replication
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">Combined discovery and replication
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em>
<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">OR<sup>b</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>N</em><sub>cases</sub>; <em>N</em><sub>controls</sub></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>N</em><sub>cases</sub>; <em>N</em><sub>controls</sub></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>N</em><sub>cases</sub>; <em>N</em><sub>controls</sub></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">LAA</td><td rowspan="1" colspan="1" valign="top" align="center">3.9 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.62 (1.36–1.93)</td><td rowspan="1" colspan="1" valign="top" align="center">421; 1,244</td><td rowspan="1" colspan="1" valign="top" align="center">3.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.15 (1.06–1.24)</td><td rowspan="1" colspan="1" valign="top" align="center">1,715; 52,695</td><td rowspan="1" colspan="1" valign="top" align="center">4.7 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 (1.13–1.30)</td><td rowspan="1" colspan="1" valign="top" align="center">2,136; 53,939</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Ischemic stroke</td><td rowspan="1" colspan="1" valign="top" align="center">5.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.29 (1.14–1.47)</td><td rowspan="1" colspan="1" valign="top" align="center">1,162; 1,244</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">1.02 (0.98–1.06)</td><td rowspan="1" colspan="1" valign="top" align="center">9,552; 52,695</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">1.04 (1.00–1.08)</td><td rowspan="1" colspan="1" valign="top" align="center">10,714; 53,939</td></tr></tbody></table></div><div id="largeobj_idm140066449660304" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3651583/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><p id="__p21" class="p p-first-last">rs556621 is located at 6p21.1 (44,702,137 bp) according to chromosome and NCBI Human Genome Build 36.3 coordinates, and <em>CDC5L</em> and <em>SUPT3H</em> are the closest genes.</p></div><div id="TFN7"><p id="__p22" class="p p-first-last">The minor allele A has a risk allele frequency (RAF) in controls of 0.30.</p></div><div id="TFN8"><sup>a</sup><em>P</em> value from 1-degree-of-freedom trend test.</div><div id="TFN9"><sup>b</sup>Odds ratio with 95% confidence interval for the effect of each additional copy of the minor allele, assuming an additive effect on the log-odds scale.</div></div></div><p id="P9">In a subsequent LAA GWAS adjusted for rs556621 genotype, no SNP showed evidence of strong independent association with LAA (peak <em>P</em> = 5.6 × 10<sup>−6</sup> for rs11625862 at 14q32.33). Haplotype association tests across the 6p21.1 region also did not detect multi-marker haplotypes that were more strongly associated with LAA than the two index SNPs (data not shown).</p><p id="P10">The addition of rs556621 genotypes to a risk prediction model containing various clinical traits associated with LAA occurrence produced a small but significant increase in the area under the receiver operator characteristic (ROC) curve (ΔAUC = 0.01; <em>P</em> = 1.2 × 10<sup>−5</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>), although this ΔAUC estimate may be inflated by estimation in the discovery cohort. To further assess the internal validity of the association at rs556621, the sample was randomly partitioned into training and test groups containing two-thirds and one-third of the LAA cases and controls, respectively. Association with LAA was evaluated in the training set, with genotyped SNPs reaching <em>P</em> &lt; 1 × 10<sup>−4</sup> (<em>n</em> = 44) then assessed in the test set (the remaining third of the sample). The index SNP at 6p21.1 (rs556621) reached <em>P</em> = 5.69 × 10<sup>−5</sup> in the training set and was the only SNP associated with LAA in the independent test set after permutation- based adjustment for the testing of 44 non-independent SNPs (familywise adjusted <em>P</em> = 6.74 × 10<sup>−3</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>).</p><p id="P11">External validity of the observed association of rs556621 with LAA risk was assessed in a replication study involving 10 independent population cohorts contributing 1,715 LAA cases (1,323 European and 392 US) and 52,695 controls (39,509 European and 13,186 US) of confirmed European ancestry. Details of the individual cohorts are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a> and the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>. Association analyses for the index SNP at 6p21.1 (rs556621) were performed separately within each of the ten cohorts, with the results combined using fixed-effects, inverse variance–weighted meta-analysis. Because association evidence was assessed for a single SNP in the independent replication study, no multiple-testing adjustment was indicated, and the result was compared with a prespecified significance threshold of 0.05.</p><p id="P12">The replication study confirmed association of rs556621 with LAA (<em><sub>P</sub></em><sub>trend</sub> = 3.9 × 10<sup>−4</sup>; OR (A allele) = 1.15, 95% CI = 1.06–1.24), with no evidence of between-study heterogeneity (<em>P</em> = 0.50, <em>I</em><sup>2</sup> = 0.0%) (<a href="/pmc/articles/PMC3651583/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>, <a href="/pmc/articles/PMC3651583/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 11</a>). The estimated population- attributable risk for rs556621 in the replication study was ~5%. When the discovery and replication cohorts were combined, meta- analyses yielded <em><sub>P</sub></em><sub>trend</sub> = 4.7 × 10<sup>−8</sup> for the association (OR = 1.21, 95% CI = 1.13–1.30). However, the heterogeneity statistic for the combined analysis was moderately significant (<em>P</em> = 0.02, <em>I</em><sup>2</sup> = 43.4%), indicating some inflation of the effect size in the discovery cohort (winner’s curse). For this reason, the estimated effect in the independent replication study is likely a better estimate of the true population effect. Meta-analyses of rs556621 for overall ischemic stroke in the replication study showed no evidence for association, despite a greater than fivefold increase in case numbers (9,552 cases and 52,695 controls; <em><sub>P</sub></em><sub>trend</sub> = 0.29; OR (A allele) = 1.02, 95% CI = 0.98–1.06; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a>). These results support the existence of a common 6p21.1 risk variant of modest but genuine effect specific to the LAA stroke subtype. Neither this SNP nor SNPs in high LD with rs556621 have previously been reported to be associated with coronary heart disease risk.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3651583/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140066451785856" class="figure"><a href="/pmc/articles/PMC3651583/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3651583_nihms457841f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms457841f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3651583/bin/nihms457841f3.jpg"></a></div><div id="largeobj_idm140066451785856" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3651583/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3651583/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p13">Forest plot showing association of rs556621 with LAA across the ten replication cohorts. For each cohort, the square and horizontal line show the estimated OR and 95% CI, respectively, representing the effect of each additional copy of the risk (A) allele on the odds of disease. The size of the square is inversely proportional to the standard error of the estimated allelic effect. A fixed-effects, inverse variance–weighted meta-analysis was used to combine association evidence across cohorts. There was no evidence of effect size heterogeneity across the ten cohorts (<em>P</em> = 0.5).</p></div></div></div><p id="P13">The 6p21.1 SNPs are located in an intergenic region of moderate LD (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 7</a>), ~200 kb upstream of the <em>SUPT3H</em> gene (forward strand) and ~180 kb upstream of <em>CDC5L</em> (reverse strand). rs556621 and rs556512 both lie within a small length of genomic sequence that contains BCL3 and PBX3 transcription factor–binding motifs and enriched for enhancer- and/or promoter-associated marks of histone protein modification. The associated SNPs or other correlated variants may thus function in regulating gene expression via altered responsiveness of key transcription factor–binding sites<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905518">19</a></sup>. A number of predicted microRNAs (miRNAs) also lie in the vicinity of rs556621 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>), suggesting that variants in LD with rs556621 could also potentially regulate gene expression through alteration of regulatory miRNA sequences. Queries of four public expression quantitative trait locus (eQTL) databases did not identify rs556621 or proxy SNPs in high LD with rs556621 as <em>cis</em> eQTLs in the assayed tissue or cell types. Future targeted investigations in atherosclerotic neurovascular tissue may help to elucidate the mechanisms by which the associated SNPs influence LAA risk.</p><p id="P14">Suggestive association with both ischemic stroke and LAA was also detected for variants in a chromosome 22q12.3 region containing the <em>APOL1</em>-<em>APOL4</em> gene cluster. These primate-specific genes are implicated in lipid metabolism and vascular biology<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905481">20</a>,<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905516">21</a></sup>, where their expression is strongly induced by proinflammatory cytokines<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905517">22</a>–<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905496">24</a></sup>. <em>APOL2</em>, <em>APOL3</em> and <em>APOL4</em> are thought to encode intracellular proteins; <em>APOL2</em>, across which association evidence was strongest, is almost exclusively expressed in the brain, with reduced expression in the heart<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905509">23</a></sup>.</p><p id="P15" class="p">This is one of the first reported GWAS for large artery atherosclerosis, a major subtype of ischemic stroke. We report the identification of variants at 6p21.1 that associate with LAA risk in individuals of European ancestry. We also report a locus within the <em>APOL1</em>-<em>APOL4</em> gene cluster that is suggestively associated with both LAA and broad ischemic stroke. The potential pathological function of these variants and their contributions to stroke risk in non-European populations remain to be determined.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">ONLINE METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Study participants: the ASGC discovery sample</h3><p id="P17" class="p p-first">ASGC stroke cases comprised stroke patients of European ancestry who were admitted to four clinical centers across Australia (The Neurosciences Department at Gosford Hospital, Gosford; the Neurology Department at John Hunter Hospital, Newcastle; The Queen Elizabeth Hospital, Adelaide; and the Royal Perth Hospital, Perth) between 2003 and 2008. Stroke was defined by World Health Organization criteria as a sudden focal neurological deficit of vascular origin, lasting more than 24 h and confirmed by imaging, such as computerized tomography (CT) and/or magnetic resonance imaging (MRI) brain scan. Other investigative tests such as electrocardiogram, carotid doppler and trans-esophageal echocardio-gram were conducted to define ischemic stroke mechanism as clinically appropriate. Cases were excluded from participation if they were aged &lt;18 years, were diagnosed with hemorrhagic stroke or had transient ischemic attack rather than ischemic stroke or if they were unable to undergo baseline brain imaging. On the basis of these criteria, a total of 1,230 ischemic stroke cases were included in the current study. Ischemic stroke subtypes were assigned using TOAST criteria on the basis of clinical, imaging and risk factor data<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905511">16</a></sup>.</p><p id="P18">ASGC controls were participants in the Hunter Community Study (HCS), a population-based cohort of individuals aged 55–85 years, predominantly of European ancestry and residing in the Hunter Region in New South Wales, Australia. Detailed recruitment methods for the HCS have been previously described<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905483">25</a></sup>. Briefly, participants were randomly selected from the New South Wales State electoral roll and were contacted by mail between 2004 and 2007. Consenting participants completed five detailed self-report questionnaires and attended the HCS data collection center, at which time a series of clinical measures were obtained. A total of 1,280 HCS participants were genotyped for the current study.</p><p id="P19" class="p p-last">All study participants gave informed consent for participation in genetic studies. Approval for the individual studies was obtained from the relevant institutional ethics committees.</p></div><div id="S4" class="sec"><h3 id="S4title">Study participants: replication cohorts</h3><p id="P20" class="p p-first-last">Replication data were contributed by a total of 11 cohorts involved in the Metastroke and International Stroke Genetics Consortia (ISGC): the Atherosclerosis Risk in Communities Study (ARIC), the Bio-Repository of DNA in Stroke (BRAINS), deCODE Genetics, the Baltimore Genetics of Early Onset Stroke (GEOS) Study, the Heart and Vascular Health (HVH) Study, the Ischemic Stroke Genetics Study/Siblings With Ischemic Stroke Study (ISGS/SWISS), The Massachusetts General Hospital Genes Affecting Stroke Risk and Outcome Study (MGH-GASROS), the Milano stroke genetics study, the Rotterdam Study, the Wellcome Trust Case Control Consortium 2–Munich (WTCCC2-Munich) and the Wellcome Trust Case Control Consortium 2–UK (WTCCC2-UK). All replication cohorts defined ischemic stroke and the LAA, CE and SVO subtypes using clinical criteria consistent with those used for the ASGC discovery sample. Summary demographic data and clinical phenotyping details for these individual cohorts are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> and the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</p></div><div id="S5" class="sec"><h3 id="S5title">Genome-wide genotyping and quality control: ASGC discovery sample</h3><p id="P21" class="p p-first">ASGC cases and controls were genotyped using the Illumina HumanHap610-Quad array. Quality control excluded SNPs with genotype call rate of &lt;0.95, deviation from Hardy-Weinberg equilibrium (<em>P</em> &lt; 1 × 10<sup>−6</sup>) or minor allele frequency of &lt;0.01. At the sample level, quality control excluded individuals with (i) genotype call rate of &lt;95% (<em>n</em> = 4); (ii) genome-wide heterozygosity of &lt;23.3% or &gt;27.2% (<em>n</em> = 9); (iii) inadequate clinical data or inconsistent clinical and genotypic gender (<em>n</em> = 45); and (iv) an inferred first- or second-degree relative in the sample identified on the basis of pairwise allele sharing estimates (estimated genome proportion shared identical by descent (IBD); <em><span class="MathJax_Preview"><math><mover accent="true"><mi>π</mi><mo>̂</mo></mover></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-3-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-18" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-19"><span class="mover" id="MathJax-Span-20"><span style="display: inline-block; position: relative; width: 0.591em; height: 0px;"><span style="position: absolute; clip: rect(3.391em, 1000.57em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-21" style="font-family: MathJax_Math; font-style: italic;">π<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.078em, 1000.56em, 3.581em, -1000em); top: -3.986em; left: 0.035em;"><span class="mo" id="MathJax-Span-22" style=""><span style="font-family: MathJax_Size1;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math><mover accent="true"><mi>π</mi><mo>̂</mo></mover></math></script></em> &gt; 0.1875; <em>n</em> = 37). After these exclusions, Eigenstrat principal-components analysis (PCA) was performed, incorporating genotype data from Phase 3 HapMap populations (CEU, Han Chinese in Beijing, China (CHB), Japanese in Tokyo, Japan (JPT), Toscani in Italia (TSI) and Yoruba from Ibadan, Nigeria (YRI)). In eigenvector plots, the majority of ASGC samples clustered closely with European (CEU and TSI) reference populations. Eighteen samples (16 cases and 2 controls) showed prominent evidence of Asian ancestry and were removed. Principal-component and IBD analyses were performed using a pruned subset of quasi-independent SNPs (~130,000 SNPs) to avoid confounding by LD. After quality control, 1,162 cases and 1,244 controls were available for association analyses at 551,514 SNPs.</p><p id="P22" class="p p-last">Genotype imputation in the filtered sample was performed using MACH v1.0.16 on the basis of HapMap Phase 2 (release 24) phased haplotypes for samples of European ancestry (CEU). Subsequent quality control excluded imputed SNPs with minor allele frequency of &lt;0.01 or ratio of observed dosage variance to expected binomial variance (<em>r</em><sup>2</sup>) of &lt;0.3.</p></div><div id="S6" class="sec"><h3 id="S6title">Genotyping and quality control: replication cohorts</h3><p id="P23" class="p p-first-last">Each replication cohort performed genome-wide genotyping, quality control and imputation as part of its own primary study. The particular arrays and quality control filters used by the individual cohorts are described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>. Of the 11 cohorts, 6 directly genotyped rs556621, and 5 imputed allelic dosages for this SNP. To ensure the accuracy of results, imputed data were only included if the quality of imputation was high, defined as a ratio of observed to expected binomial dosage variance (<em>r</em><sup>2</sup>) of &gt;0.7. This resulted in the exclusion of one sample (HVH; <em>r</em><sup>2</sup> = 0.64). All other samples had <em>r</em><sup>2</sup> ≥0.95 for rs556621.</p></div><div id="S7" class="sec"><h3 id="S7title">Estimating the proportion of phenotypic variation attributable to genotyped SNPs</h3><p id="P24" class="p p-first-last">The proportion of case-control variation attributable to variation in genotyped SNPs was estimated in the discovery sample with GCTA software<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905499">18</a>,<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905484">26</a></sup>, which uses genome-wide SNP data to estimate additive genetic relationships (correlations) between essentially unrelated individuals, using a linear mixed model (LMM) to estimate the contribution of genotyped SNPs (and causal variants in LD with genotyped SNPs) to observed variation in case-control status. Before analysis, additional quality control of genotype data was performed to reduce bias in variance estimates from the accrued effects of small genotyping errors<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905503">27</a></sup>. We excluded SNPs with missingness of &gt;0.1% or Hardy-Weinberg equilibrium <em>P</em> value of &lt;1 × 10<sup>−4</sup> and individuals with &gt;0.1% missing genotype data or estimated relatedness of &gt;0.05 (approximately closer than second cousins)<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905490">27</a></sup>. After quality control, genotypes at 457,533 SNPs were available for estimating genetic effects for 1,079 ischemic stroke cases, 400 LAA cases, 288 SVO cases and 226 CE cases. Each case group was evaluated in a separate analysis using a common control sample of 1,172 individuals; all fitted LMMs were adjusted for age and sex. Heritability estimates shown in <a href="/pmc/articles/PMC3651583/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> relate to the observed (binary) risk scale and case-control proportions. We note that, although these estimates do not represent heritability in the conventional sense, the test statistics and their associated significance levels are invariant under adjustment for ascertainment bias or liability scale<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905498">28</a></sup>.</p></div><div id="S8" class="sec"><h3 id="S8title">Genome-wide association analyses in the Australian discovery cohort</h3><p id="P25" class="p p-first">Genome-wide association analyses were performed using 1-degree-of-freedom trend tests, assuming an additive effect of allele dosage. Parameters were estimated using logistic regression models adjusted for age and sex. Analyses were not adjusted for principal components of population ancestry, as observed genomic inflation factors in unadjusted models (<em>λ</em> = 1.031, <em>λ</em><sub>1,000</sub> = 1.026 for ischemic stroke; <em>λ</em> = 1.007, <em>λ</em><sub>1,000</sub> = 1.011 for LAA) indicated an absence of bias due to population stratification. Meta-analysis genomic control inflation factors (<em>λ</em>) were calculated as previously described, as were standardized values for a sample of 1,000 cases and 1,000 controls (<em>λ</em><sub>1,000</sub>)<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905501">29</a></sup>. Secondary analyses of peak regions were adjusted for ancestry principal components and clinical traits, including hypertension, hypercholesterolemia, diabetes mellitus, atrial fibrillation, myocardial infarction and smoking status, to investigate potential confounders of the observed genetic associations. Association tests were performed using maximum-likelihood estimated dosages for imputed SNPs and observed integer dosages for genotyped SNPs. Logistic models were fitted using mach2dat software, which calculates significance levels for estimated parameters using a likelihood-ratio test<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905519">30</a>,<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905506">31</a></sup>. The two secondary logistic analyses conditioned on rs4479522 and rs556621 genotypes were adjusted for age, sex and integer-valued dosage of the test allele at conditioned SNPs.</p><p id="P26" class="p p-last">Pairwise LD between SNPs was assessed and visualized using Haploview software<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905507">32</a></sup> on the basis of European (CEU) HapMap Phase 2 data. Haplotype analyses of the 6p21.1 region used genotyped data and maximum-likelihood genotypes for SNPs imputed with high reliability (<em>r</em><sup>2</sup> &gt; 0.7). Sliding window hap-lotypes incorporating from two to six adjacent SNPs were estimated and assessed for association with LAA case-control status using UNPHASED software<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905514">33</a></sup>. Regional association plots were constructed using LocusZoom software<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905502">34</a></sup>.</p></div><div id="S9" class="sec"><h3 id="S9title">Meta-analysis of rs556621 in replication cohorts</h3><p id="P27" class="p p-first">For rs556621, each replication sample performed logistic regression using a 1-degree-of-freedom trend test relating the presence of stroke (LAA or overall ischemic stroke) to allelic dosage, assuming an additive effect of the test allele. The test allele, estimated <em>β</em> coefficient, standard error and effective sample size were provided for the combined replication analysis. Fixed-effects, inverse variance–weighted meta-analyses of the ten replication cohorts providing high-quality data for rs556621 was performed using METAL software. Between-study heterogeneity was investigated using Cochran’s <em>Q</em> statistic with its associated <em>P</em> value and the <em>I</em><sup>2</sup> metric, representing the percentage of between-study heterogeneity exceeding the value expected by chance. Population-attributable risk (PAR%) was estimated for rs556621 using the formula</p><p id="P28" class="p p-last">
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><mtext>PAR</mtext><mo>%</mo><mo>=</mo><mfrac><mrow><mn>100</mn><mo>×</mo><mi>p</mi><mo stretchy="false">(</mo><mtext>OR</mtext><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mrow><mi>p</mi><mo stretchy="false">(</mo><mtext>OR</mtext><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mn>1</mn></mrow></mfrac></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0" style=""><nobr><span class="math" id="M1" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-24"><span class="mtext" id="MathJax-Span-25" style="font-family: MathJax_Main;">PAR</span><span class="mo" id="MathJax-Span-26" style="font-family: MathJax_Main;">%</span><span class="mo" id="MathJax-Span-27" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-28" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 7.359em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.072em, 1007.14em, 4.382em, -1000em); top: -4.696em; left: 50%; margin-left: -3.62em;"><span class="mrow" id="MathJax-Span-29"><span class="mn" id="MathJax-Span-30" style="font-family: MathJax_Main;">100</span><span class="mo" id="MathJax-Span-31" style="font-family: MathJax_Main; padding-left: 0.222em;">×</span><span class="mi" id="MathJax-Span-32" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">p</span><span class="mo" id="MathJax-Span-33" style="font-family: MathJax_Main;">(</span><span class="mtext" id="MathJax-Span-34" style="font-family: MathJax_Main;">OR</span><span class="mo" id="MathJax-Span-35" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="mn" id="MathJax-Span-36" style="font-family: MathJax_Main; padding-left: 0.222em;">1</span><span class="mo" id="MathJax-Span-37" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.072em, 1006.17em, 4.382em, -1000em); top: -3.258em; left: 50%; margin-left: -3.12em;"><span class="mrow" id="MathJax-Span-38"><span class="mi" id="MathJax-Span-39" style="font-family: MathJax_Math; font-style: italic;">p</span><span class="mo" id="MathJax-Span-40" style="font-family: MathJax_Main;">(</span><span class="mtext" id="MathJax-Span-41" style="font-family: MathJax_Main;">OR</span><span class="mo" id="MathJax-Span-42" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="mn" id="MathJax-Span-43" style="font-family: MathJax_Main; padding-left: 0.222em;">1</span><span class="mo" id="MathJax-Span-44" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-45" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mn" id="MathJax-Span-46" style="font-family: MathJax_Main; padding-left: 0.222em;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.865em, 1007.36em, 1.24em, -1000em); top: -1.305em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 7.359em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><mtext>PAR</mtext><mo>%</mo><mo>=</mo><mfrac><mrow><mn>100</mn><mo>×</mo><mi>p</mi><mo stretchy="false">(</mo><mtext>OR</mtext><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mrow><mi>p</mi><mo stretchy="false">(</mo><mtext>OR</mtext><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mn>1</mn></mrow></mfrac></math></script></div></div><p> where OR is the odds ratio estimated using independent replication data and <em>p</em> is the prevalence of the risk allele in controls<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905505">35</a></sup>.</p></div><div id="S10" class="sec"><h3 id="S10title">Predictive modeling using ROC curves</h3><p id="P29" class="p p-first-last">Predictive models incorporating clinical and genetic risk factors were evaluated for their ability to discriminate between case and control participants by calculating the area under the receiver operator characteristic (ROC) curve (AUC). ROC curves show the relationship between sensitivity (true positive rate) and 1-specificity (false negative rate) for all possible cut-points of a diagnostic test. For specified covariates, the ROC curve was fitted and the AUC calculated using Stata software<sup><a href="#R36" rid="R36" class=" bibr popnode">36</a></sup>, on the basis of parameter estimates from logistic regression models. Likelihood-ratio tests were used to assess the significance of changes in model fit.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">eQTL analyses</h3><p id="P30" class="p p-first-last">For the lead SNP at 6p21.1 (rs556621), proxy SNPs with <em>r</em><sup>2</sup> of &gt;0.8 were identified from HapMap CEU Phases 1 and 2 (release 22) and 3 data (release 2) using SNAP (v2.2). Four publicly available eQTL databases were searched to determine whether genotypes of the lead or proxy SNPs have been previously associated with gene expression in <em>cis</em> in a range of tissue and cell types. We defined potential <em>cis</em> eQTLs as candidate SNPs associated with expression of a gene transcript mapping to a genomic region within 1 Mb<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346905515">37</a></sup> at a nominal significance level of 1 × 10<sup>−3</sup>. The databases searched were (i) SCAN–SNP and CNV Annotation Database; (ii) the NCBI GTEx (Genotype-Tissue Expression) eQTL Browser; (iii) the Pritchard laboratory UChicago eQTL browser; and (iv) mRNA by SNP Browser v1.0.1. The tissue and cell types assessed in these databases include liver, brain, lymphoblastoid cell lines (LCLs), monocytes, fibroblasts and T cells.</p></div></div><div id="S12" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supplementary data and figures</h4><div class="sup-box half_rhythm" id="d36e1791"><a href="/pmc/articles/PMC3651583/bin/NIHMS457841-supplement-supplementary_data_and_figures.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.1M, pdf)</sup></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Acknowledgments</h2><div class="sec"><p id="__p3">A complete list of funding acknowledgments is included in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>. We are grateful to the participants with ischemic stroke and also to their families for participating in this study. Australian population control data were derived from the Hunter Community Study. We also thank the University of Newcastle for funding and the men and women of the Hunter region who participated in this study. This research was funded by grants from the Australian National Health and Medical Research Council (NHMRC; project grant 569257), the Australian National Heart Foundation (NHF; project grant G 04S 1623), the University of Newcastle, the Gladys M Brawn Fellowship scheme and the Vincent Fairfax Family Foundation in Australia. E.G.H. is supported by the Australian NHMRC Fellowship scheme. J.G. is supported by a Practitioner Fellowship from the NHMRC and a Senior Clinical Research Fellowship from the Australian Office of Health and Medical Research. The principal funding for the Wellcome Trust Case Control Consortium 2 (WTCCC2) ischemic stroke study was provided by the Wellcome Trust, as part of the WTCCC2 project (085475/B/ 08/Z, 085475/Z/08/Z and WT084724MA). This work was also supported by the European Community’s Sixth Framework Programme (LSHM-CT-2007-037273), the Wellcome Trust core award (090532/Z/09/Z) and AstraZeneca. M. Farrall is a member of the Oxford British Heart Foundation (BHF) Centre of Research Excellence. The Siblings with Ischemic Stroke Study (SWISS) and the Ischemic Stroke Genetics Study (ISGS) were funded by grants from the US National Institute of Neurological Disorders and Stroke. Additional funding was provided by the US National Institute of Neurological Disorders and Stroke (U01NS069208). The Rotterdam Study received principal funding for this report from the Netherlands Heart Foundation (grant 2009B102).</p></div></div><div id="idm140066445279888" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140066445279888title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="__p4" class="p p-first-last">Note: <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary information</a> is available in the online version of the paper.</p><p></p><p class="fn sec" id="FN5"></p><p id="__p5" class="p p-first-last"><strong>URLs.</strong> MACH, <a href="http://www.sph.umich.edu/csg/yli/mach/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/yli/mach/index.html</a>; Haploview, <a href="http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/haploview/haploview" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/haploview/haploview</a>; UNPHASED, <a href="http://unphased.sourceforge.net/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://unphased.sourceforge.net/</a>; LocusZoom, <a href="http://csg.sph.umich.edu/locuszoom/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/locuszoom/</a>; METAL, <a href="http://www.sph.umich.edu/csg/abecasis/Metal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Metal/</a>; SNAP, <a href="http://www.broadinstitute.org/mpg/snap/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/snap/</a>; SCAN–SNP and CNV Annotation Database, <a href="http://scan.bsd.uchicago.edu/newinterface/about.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://scan.bsd.uchicago.edu/newinterface/about.html</a>; NCBI GTEx (Genotype-Tissue Expression) eQTL Browser, <a href="http://www.ncbi.nlm.nih.gov/gtex/GTEX2/gtex.cgi" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gtex/GTEX2/gtex.cgi</a>; Pritchard laboratory UChicago eQTL browser, <a href="http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/</a>; mRNA by SNP Browser, <a href="http://www.sph.umich.edu/csg/liang/asthma/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/liang/asthma/</a>.</p><p></p><div><p id="__p6" class="p p-first"><strong>AUTHOR CONTRIBUTIONS</strong></p>S.A.K., J.W.S., L.F.L., P.M., R.J.S., C.L. and J.A. designed the study. E.G.H. performed statistical analyses in the discovery cohort, meta-analyses of replication data and wrote the first draft of the manuscript. T.-J.E. and R.J.S. coordinated genotyping of the discovery cohort. J.M.M., J.G., J.J., G.J.H., R.B., M.W.P., J.W.S., L.F.L., C.L., M.M., R.P., W.S. and J.A. performed phenotype collection and data management in the Australian sample. E. Biros, M.D.L. and C.O. performed bioinformatic analyses. Replication data were provided by S. Barlera, S. Bevan, J.C.B., E. Boerwinkle, G.B.B., T.G.B., R.D.B., Y.-C.C., J.W.C., I.C., W.J.D., M. Fornage, K.L.F., S.G., A.G., M.A.I., W.T.L., R.M., J.F.M., B.D.M., T.H.M., M.A.N., E.A.P., B.M.P., P.S., K.S., G.T., M.T., U.T., B.F.J.V., K.L.W., B.B.W., C.S., P.M.R., M. Farrall, M.D., J.R. and H.S.M. All authors critically reviewed the manuscript and gave advice on the contents of the paper.</div><p class="fn sec" id="FN7"> </p><p id="__p8" class="p p-first"><strong>COMPETING FINANCIAL INTERESTS</strong></p><p id="__p9" class="p p-last">The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN8"></p><p id="__p10" class="p p-first-last">Reprints and permissions information is available online at <a href="http://www.nature.com/reprints/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/reprints/index.html</a>.</p><p></p></div></div><div id="idm140066451340832" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140066451340832title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">World Health Organization. . <span class="ref-journal">Atlas of Heart Disease and Stroke.</span> World Health Organization; Geneva: 2004.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Atlas+of+Heart+Disease+and+Stroke&amp;publication_year=2004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Feigin VL, Lawes CM, Bennett DA, Barker-Collo SL, Parag V. Worldwide stroke incidence and early case fatality reported in 56 population-based studies: a systematic review. <span><span class="ref-journal">Lancet Neurol. </span>2009;<span class="ref-vol">8</span>:355–369.</span> [<a href="/pubmed/19233729" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Worldwide+stroke+incidence+and+early+case+fatality+reported+in+56+population-based+studies:+a+systematic+review&amp;author=VL+Feigin&amp;author=CM+Lawes&amp;author=DA+Bennett&amp;author=SL+Barker-Collo&amp;author=V+Parag&amp;volume=8&amp;publication_year=2009&amp;pages=355-369&amp;pmid=19233729&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Strong K, Mathers C, Bonita R. Preventing stroke: saving lives around the world. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:182–187.</span> [<a href="/pubmed/17239805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Preventing+stroke:+saving+lives+around+the+world&amp;author=K+Strong&amp;author=C+Mathers&amp;author=R+Bonita&amp;volume=6&amp;publication_year=2007&amp;pages=182-187&amp;pmid=17239805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">O’Donnell MJ, et al.  Risk factors for ischaemic and intracerebral haemorrhagic stroke in 22 countries (the INTERSTROKE study): a case-control study. <span><span class="ref-journal">Lancet. </span>2010;<span class="ref-vol">376</span>:112–123.</span> [<a href="/pubmed/20561675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Risk+factors+for+ischaemic+and+intracerebral+haemorrhagic+stroke+in+22+countries+(the+INTERSTROKE+study):+a+case-control+study&amp;author=MJ+O’Donnell&amp;volume=376&amp;publication_year=2010&amp;pages=112-123&amp;pmid=20561675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Flossmann E, Schulz UG, Rothwell PM. Systematic review of methods and results of studies of the genetic epidemiology of ischemic stroke. <span><span class="ref-journal">Stroke. </span>2004;<span class="ref-vol">35</span>:212–227.</span> [<a href="/pubmed/14684773" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Systematic+review+of+methods+and+results+of+studies+of+the+genetic+epidemiology+of+ischemic+stroke&amp;author=E+Flossmann&amp;author=UG+Schulz&amp;author=PM+Rothwell&amp;volume=35&amp;publication_year=2004&amp;pages=212-227&amp;pmid=14684773&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Jerrard-Dunne P, Cloud G, Hassan A, Markus HS. Evaluating the genetic component of ischemic stroke subtypes: a family history study. <span><span class="ref-journal">Stroke. </span>2003;<span class="ref-vol">34</span>:1364–1369.</span> [<a href="/pubmed/12714707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Evaluating+the+genetic+component+of+ischemic+stroke+subtypes:+a+family+history+study&amp;author=P+Jerrard-Dunne&amp;author=G+Cloud&amp;author=A+Hassan&amp;author=HS+Markus&amp;volume=34&amp;publication_year=2003&amp;pages=1364-1369&amp;pmid=12714707&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Fox CS, et al.  Genetic and environmental contributions to atherosclerosis phenotypes in men and women: heritability of carotid intima-media thickness in the Framingham Heart Study. <span><span class="ref-journal">Stroke. </span>2003;<span class="ref-vol">34</span>:397–401.</span> [<a href="/pubmed/12574549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Genetic+and+environmental+contributions+to+atherosclerosis+phenotypes+in+men+and+women:+heritability+of+carotid+intima-media+thickness+in+the+Framingham+Heart+Study&amp;author=CS+Fox&amp;volume=34&amp;publication_year=2003&amp;pages=397-401&amp;pmid=12574549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Moskau S, et al.  Heritability of carotid artery atherosclerotic lesions: an ultrasound study in 154 families. <span><span class="ref-journal">Stroke. </span>2005;<span class="ref-vol">36</span>:5–8.</span> [<a href="/pubmed/15569868" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Heritability+of+carotid+artery+atherosclerotic+lesions:+an+ultrasound+study+in+154+families&amp;author=S+Moskau&amp;volume=36&amp;publication_year=2005&amp;pages=5-8&amp;pmid=15569868&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Turner ST, et al.  Heritability of leukoaraiosis in hypertensive sibships. <span><span class="ref-journal">Hypertension. </span>2004;<span class="ref-vol">43</span>:483–487.</span> [<a href="/pubmed/14718359" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Heritability+of+leukoaraiosis+in+hypertensive+sibships&amp;author=ST+Turner&amp;volume=43&amp;publication_year=2004&amp;pages=483-487&amp;pmid=14718359&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Carmelli D, et al.  Evidence for genetic variance in white matter hyperintensity volume in normal elderly male twins. <span><span class="ref-journal">Stroke. </span>1998;<span class="ref-vol">29</span>:1177–1181.</span> [<a href="/pubmed/9626291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Evidence+for+genetic+variance+in+white+matter+hyperintensity+volume+in+normal+elderly+male+twins&amp;author=D+Carmelli&amp;volume=29&amp;publication_year=1998&amp;pages=1177-1181&amp;pmid=9626291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Kääb S, et al.  Large scale replication and meta-analysis of variants on chromosome 4q25 associated with atrial fibrillation. <span><span class="ref-journal">Eur Heart J. </span>2009;<span class="ref-vol">30</span>:813–819.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2663727/">PMC free article</a>]</span> [<a href="/pubmed/19141561" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+Heart+J&amp;title=Large+scale+replication+and+meta-analysis+of+variants+on+chromosome+4q25+associated+with+atrial+fibrillation&amp;author=S+Kääb&amp;volume=30&amp;publication_year=2009&amp;pages=813-819&amp;pmid=19141561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Gretarsdottir S, et al.  Risk variants for atrial fibrillation on chromosome 4q25 associate with ischemic stroke. <span><span class="ref-journal">Ann Neurol. </span>2008;<span class="ref-vol">64</span>:402–409.</span> [<a href="/pubmed/18991354" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Risk+variants+for+atrial+fibrillation+on+chromosome+4q25+associate+with+ischemic+stroke&amp;author=S+Gretarsdottir&amp;volume=64&amp;publication_year=2008&amp;pages=402-409&amp;pmid=18991354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Palomaki GE, Melillo S, Bradley LA. Association between 9p21 genomic markers and heart disease: a meta-analysis. <span><span class="ref-journal">J Am Med Assoc. </span>2010;<span class="ref-vol">303</span>:648–656.</span> [<a href="/pubmed/20159873" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Assoc&amp;title=Association+between+9p21+genomic+markers+and+heart+disease:+a+meta-analysis&amp;author=GE+Palomaki&amp;author=S+Melillo&amp;author=LA+Bradley&amp;volume=303&amp;publication_year=2010&amp;pages=648-656&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Smith JG, et al.  Common genetic variants on chromosome 9p21 confers risk of ischemic stroke: a large-scale genetic association study. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2009;<span class="ref-vol">2</span>:159–164.</span> [<a href="/pubmed/20031580" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Common+genetic+variants+on+chromosome+9p21+confers+risk+of+ischemic+stroke:+a+large-scale+genetic+association+study&amp;author=JG+Smith&amp;volume=2&amp;publication_year=2009&amp;pages=159-164&amp;pmid=20031580&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Bellenguez C, et al.  Genome-wide association study identifies a variant in <em>HDAC9</em> associated with large vessel ischemic stroke. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:328–333.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3303115/">PMC free article</a>]</span> [<a href="/pubmed/22306652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+variant+in+HDAC9+associated+with+large+vessel+ischemic+stroke&amp;author=C+Bellenguez&amp;volume=44&amp;publication_year=2012&amp;pages=328-333&amp;pmid=22306652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Adams HP, Jr, et al.  Classification of subtype of acute ischemic stroke. Definitions for use in a multicenter clinical trial. TOAST. Trial of Org 10172 in Acute Stroke Treatment. <span><span class="ref-journal">Stroke. </span>1993;<span class="ref-vol">24</span>:35–41.</span> [<a href="/pubmed/7678184" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Classification+of+subtype+of+acute+ischemic+stroke.+Definitions+for+use+in+a+multicenter+clinical+trial.+TOAST.+Trial+of+Org+10172+in+Acute+Stroke+Treatment&amp;author=HP+Adams&amp;volume=24&amp;publication_year=1993&amp;pages=35-41&amp;pmid=7678184&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Flossmann E, Schulz UG, Rothwell PM. Potential confounding by intermediate phenotypes in studies of the genetics of ischaemic stroke. <span><span class="ref-journal">Cerebrovasc Dis. </span>2005;<span class="ref-vol">19</span>:1–10.</span> [<a href="/pubmed/15528878" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cerebrovasc+Dis&amp;title=Potential+confounding+by+intermediate+phenotypes+in+studies+of+the+genetics+of+ischaemic+stroke&amp;author=E+Flossmann&amp;author=UG+Schulz&amp;author=PM+Rothwell&amp;volume=19&amp;publication_year=2005&amp;pages=1-10&amp;pmid=15528878&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Heintzman ND, et al.  Histone modifications at human enhancers reflect global cell-type-specific gene expression. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">459</span>:108–112.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2910248/">PMC free article</a>]</span> [<a href="/pubmed/19295514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Histone+modifications+at+human+enhancers+reflect+global+cell-type-specific+gene+expression&amp;author=ND+Heintzman&amp;volume=459&amp;publication_year=2009&amp;pages=108-112&amp;pmid=19295514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Page NM, Butlin DJ, Lomthaisong K, Lowry PJ. The human apolipoprotein L gene cluster: identification, classification, and sites of distribution. <span><span class="ref-journal">Genomics. </span>2001;<span class="ref-vol">74</span>:71–78.</span> [<a href="/pubmed/11374903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=The+human+apolipoprotein+L+gene+cluster:+identification,+classification,+and+sites+of+distribution&amp;author=NM+Page&amp;author=DJ+Butlin&amp;author=K+Lomthaisong&amp;author=PJ+Lowry&amp;volume=74&amp;publication_year=2001&amp;pages=71-78&amp;pmid=11374903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Duchateau PN, et al.  Plasma apolipoprotein L concentrations correlate with plasma triglycerides and cholesterol levels in normolipidemic, hyperlipidemic, and diabetic subjects. <span><span class="ref-journal">J Lipid Res. </span>2000;<span class="ref-vol">41</span>:1231–1236.</span> [<a href="/pubmed/10946010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Lipid+Res&amp;title=Plasma+apolipoprotein+L+concentrations+correlate+with+plasma+triglycerides+and+cholesterol+levels+in+normolipidemic,+hyperlipidemic,+and+diabetic+subjects&amp;author=PN+Duchateau&amp;volume=41&amp;publication_year=2000&amp;pages=1231-1236&amp;pmid=10946010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Horrevoets AJ, et al.  Vascular endothelial genes that are responsive to tumor necrosis factor-α <em>in vitro</em> are expressed in atherosclerotic lesions, including inhibitor of apoptosis protein-1, stannin, and two novel genes. <span><span class="ref-journal">Blood. </span>1999;<span class="ref-vol">93</span>:3418–3431.</span> [<a href="/pubmed/10233894" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Vascular+endothelial+genes+that+are+responsive+to+tumor+necrosis+factor-α+in+vitro+are+expressed+in+atherosclerotic+lesions,+including+inhibitor+of+apoptosis+protein-1,+stannin,+and+two+novel+genes&amp;author=AJ+Horrevoets&amp;volume=93&amp;publication_year=1999&amp;pages=3418-3431&amp;pmid=10233894&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Monajemi H, Fontijn RD, Pannekoek H, Horrevoets AJ. The apolipoprotein L gene cluster has emerged recently in evolution and is expressed in human vascular tissue. <span><span class="ref-journal">Genomics. </span>2002;<span class="ref-vol">79</span>:539–546.</span> [<a href="/pubmed/11944986" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=The+apolipoprotein+L+gene+cluster+has+emerged+recently+in+evolution+and+is+expressed+in+human+vascular+tissue&amp;author=H+Monajemi&amp;author=RD+Fontijn&amp;author=H+Pannekoek&amp;author=AJ+Horrevoets&amp;volume=79&amp;publication_year=2002&amp;pages=539-546&amp;pmid=11944986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Sana TR, Janatpour MJ, Sathe M, McEvoy LM, McClanahan TK. Microarray analysis of primary endothelial cells challenged with different inflammatory and immune cytokines. <span><span class="ref-journal">Cytokine. </span>2005;<span class="ref-vol">29</span>:256–269.</span> [<a href="/pubmed/15749026" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine&amp;title=Microarray+analysis+of+primary+endothelial+cells+challenged+with+different+inflammatory+and+immune+cytokines&amp;author=TR+Sana&amp;author=MJ+Janatpour&amp;author=M+Sathe&amp;author=LM+McEvoy&amp;author=TK+McClanahan&amp;volume=29&amp;publication_year=2005&amp;pages=256-269&amp;pmid=15749026&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">McEvoy M, et al.  Cohort profile: The Hunter Community Study. <span><span class="ref-journal">Int J Epidemiol. </span>2010;<span class="ref-vol">39</span>:1452–1463.</span> [<a href="/pubmed/20056765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Cohort+profile:+The+Hunter+Community+Study&amp;author=M+McEvoy&amp;volume=39&amp;publication_year=2010&amp;pages=1452-1463&amp;pmid=20056765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Yang J, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;volume=42&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Lee SH, Wray NR, Goddard ME, Visscher PM. Estimating missing heritability for disease from genome-wide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:294–305.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059431/">PMC free article</a>]</span> [<a href="/pubmed/21376301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Estimating+missing+heritability+for+disease+from+genome-wide+association+studies&amp;author=SH+Lee&amp;author=NR+Wray&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=294-305&amp;pmid=21376301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Painter JN, et al.  Genome-wide association study identifies a locus at 7p15.2 associated with endometriosis. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:51–54.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3019124/">PMC free article</a>]</span> [<a href="/pubmed/21151130" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+locus+at+7p15.2+associated+with+endometriosis&amp;author=JN+Painter&amp;volume=43&amp;publication_year=2011&amp;pages=51-54&amp;pmid=21151130&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">de Bakker PI, et al.  Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:R122–R128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=PI+de+Bakker&amp;volume=17&amp;publication_year=2008&amp;pages=R122-R128&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;author=P+Scheet&amp;author=GR+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Dudbridge F. Likelihood-based association analysis for nuclear families and unrelated subjects with missing genotype data. <span><span class="ref-journal">Hum Hered. </span>2008;<span class="ref-vol">66</span>:87–98.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2386559/">PMC free article</a>]</span> [<a href="/pubmed/18382088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Likelihood-based+association+analysis+for+nuclear+families+and+unrelated+subjects+with+missing+genotype+data&amp;author=F+Dudbridge&amp;volume=66&amp;publication_year=2008&amp;pages=87-98&amp;pmid=18382088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Pruim RJ, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Zheng SL, et al.  Cumulative association of five genetic variants with prostate cancer. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:910–919.</span> [<a href="/pubmed/18199855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Cumulative+association+of+five+genetic+variants+with+prostate+cancer&amp;author=SL+Zheng&amp;volume=358&amp;publication_year=2008&amp;pages=910-919&amp;pmid=18199855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">StataCorp. <span class="ref-journal">Statistical Software.</span> StataCorp LP; College Station, Texas: 2009. Stata: Release 11. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Statistical+Software&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Webster JA, et al.  Genetic control of human brain transcript expression in Alzheimer disease. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:445–458.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2667989/">PMC free article</a>]</span> [<a href="/pubmed/19361613" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+control+of+human+brain+transcript+expression+in+Alzheimer+disease&amp;author=JA+Webster&amp;volume=84&amp;publication_year=2009&amp;pages=445-458&amp;pmid=19361613&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3651583&amp;issue-id=222791&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140066448380864title">Abstract</a></li><li><a href="#S2title">ONLINE METHODS</a></li><li><a href="#S12title">Supplementary Material</a></li><li><a href="#S13title">Acknowledgments</a></li><li><a href="#idm140066445279888title">Footnotes</a></li><li><a href="#idm140066451340832title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3651583/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3651583/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3651583/pdf/nihms457841.pdf">PDF (1.0M)</a></li> | <li><a href="#" data-citationid="PMC3651583" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3651583%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3651583%2F&amp;text=Common%20variants%20at%206p21.1%20are%20associated%20with%20large%20artery%20atherosclerotic%20stroke" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3651583%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22941190" data-aiid="3651583" data-aid="3651583" data-iid="222791" data-domainid="319" data-domain="nihpa" data-accid="PMC3651583" data-md5="8434c83e99af2206da91742c37d8b79b"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:29:53-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B6576DD548DD100000000043F02AD&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3651583%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3651583/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size1, sans-serif;"></div></div></body></html>