<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3690971/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association studies identify several new loci associated with pigmentation traits and skin cancer risk in European Americans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3690971/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association studies identify several new loci associated with pigmentation traits and skin cancer risk in European Americans"><meta name="citation_authors" content="Mingfeng Zhang, Fengju Song, Liming Liang, Hongmei Nan, Jiangwen Zhang, Hongliang Liu, Li-E. Wang, Qingyi Wei, Jeffrey E. Lee, Christopher I. Amos, Peter Kraft, Abrar A. Qureshi, Jiali Han"><meta name="citation_date" content="15 July 2013"><meta name="citation_issue" content="14"><meta name="citation_volume" content="22"><meta name="citation_firstpage" content="2948"><meta name="citation_doi" content="10.1093/hmg/ddt142"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3690971/?report=abstract"><meta name="citation_pmid" content="23548203"><meta name="DC.Title" content="Genome-wide association studies identify several new loci associated with pigmentation traits and skin cancer risk in European Americans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Mingfeng Zhang"><meta name="DC.Contributor" content="Fengju Song"><meta name="DC.Contributor" content="Liming Liang"><meta name="DC.Contributor" content="Hongmei Nan"><meta name="DC.Contributor" content="Jiangwen Zhang"><meta name="DC.Contributor" content="Hongliang Liu"><meta name="DC.Contributor" content="Li-E. Wang"><meta name="DC.Contributor" content="Qingyi Wei"><meta name="DC.Contributor" content="Jeffrey E. Lee"><meta name="DC.Contributor" content="Christopher I. Amos"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="Abrar A. Qureshi"><meta name="DC.Contributor" content="Jiali Han"><meta name="DC.Date" content="2013 Jul 15"><meta name="DC.Identifier" content="10.1093/hmg/ddt142"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association studies identify several new loci associated with pigmentation traits and skin cancer risk in European Americans"><meta property="og:type" content="article"><meta property="og:description" content="Aiming to identify novel genetic loci for pigmentation and skin cancer, we conducted a series of genome-wide association studies on hair color, eye color, number of sunburns, tanning ability and number of non-melanoma skin cancers (NMSCs) among 10 183 ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3690971/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3690971/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3690971/pdf/ddt142.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CBA5FDD5558B100000000063603EA.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3690971%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3690971</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2013 Jul 15; 22(14): 2948–2959. </span></div><div><span class="fm-vol-iss-date">Published online 2013 Apr 1. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddt142" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddt142</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3690971</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23548203">23548203</a></div></div></div></div><h1 class="content-title">Genome-wide association studies identify several new loci associated with pigmentation traits and skin cancer risk in European Americans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Zhang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992320900144" co-class="co-affbox">Mingfeng Zhang</a>,<sup>1</sup> <a href="/pubmed/?term=Song%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992327002192" co-class="co-affbox">Fengju Song</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Liang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992320896752" co-class="co-affbox">Liming Liang</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Nan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992320894144" co-class="co-affbox">Hongmei Nan</a>,<sup>2,</sup><sup>6</sup> <a href="/pubmed/?term=Zhang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992320891424" co-class="co-affbox">Jiangwen Zhang</a>,<sup>7</sup> <a href="/pubmed/?term=Liu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992320889440" co-class="co-affbox">Hongliang Liu</a>,<sup>8</sup> <a href="/pubmed/?term=Wang%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992322585344" co-class="co-affbox">Li-E. Wang</a>,<sup>8</sup> <a href="/pubmed/?term=Wei%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992322583360" co-class="co-affbox">Qingyi Wei</a>,<sup>8</sup> <a href="/pubmed/?term=Lee%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992322581376" co-class="co-affbox">Jeffrey E. Lee</a>,<sup>9</sup> <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992322579392" co-class="co-affbox">Christopher I. Amos</a>,<sup>10</sup> <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992322577408" co-class="co-affbox">Peter Kraft</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Qureshi%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992322574688" co-class="co-affbox">Abrar A. Qureshi</a>,<sup>1,</sup><sup>2</sup> and  <a href="/pubmed/?term=Han%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203" class="affpopup" co-rid="_co_idm139992322571968" co-class="co-affbox">Jiali Han</a><sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139992320900144"><h3 class="no_margin">Mingfeng Zhang</h3><p><sup>1</sup>Clinical Research Program, Department of Dermatology and</p><div>Find articles by <a href="/pubmed/?term=Zhang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Mingfeng Zhang</a></div></div><div id="_co_idm139992327002192"><h3 class="no_margin">Fengju Song</h3><p><sup>1</sup>Clinical Research Program, Department of Dermatology and</p><p><sup>3</sup>Department of Epidemiology, Tianjin Medical University Cancer Institute and Hospital, Tianjin, China,</p><div>Find articles by <a href="/pubmed/?term=Song%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Fengju Song</a></div></div><div id="_co_idm139992320896752"><h3 class="no_margin">Liming Liang</h3><p><sup>4</sup>Department of Epidemiology and</p><p><sup>5</sup>Department of Biostatistics, Harvard School of Public Health, Boston, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Liang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Liming Liang</a></div></div><div id="_co_idm139992320894144"><h3 class="no_margin">Hongmei Nan</h3><p><sup>2</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA, USA,</p><p><sup>6</sup>Division of Cancer Epidemiology, Department of Epidemiology and Public Health, University of Maryland School of Medicine, Baltimore, MD, USA,</p><div>Find articles by <a href="/pubmed/?term=Nan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Hongmei Nan</a></div></div><div id="_co_idm139992320891424"><h3 class="no_margin">Jiangwen Zhang</h3><p><sup>7</sup>FAS Research Computing, Harvard University, Cambridge, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Zhang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Jiangwen Zhang</a></div></div><div id="_co_idm139992320889440"><h3 class="no_margin">Hongliang Liu</h3><p><sup>8</sup>Department of Epidemiology and</p><div>Find articles by <a href="/pubmed/?term=Liu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Hongliang Liu</a></div></div><div id="_co_idm139992322585344"><h3 class="no_margin">Li-E. Wang</h3><p><sup>8</sup>Department of Epidemiology and</p><div>Find articles by <a href="/pubmed/?term=Wang%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Li-E. Wang</a></div></div><div id="_co_idm139992322583360"><h3 class="no_margin">Qingyi Wei</h3><p><sup>8</sup>Department of Epidemiology and</p><div>Find articles by <a href="/pubmed/?term=Wei%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Qingyi Wei</a></div></div><div id="_co_idm139992322581376"><h3 class="no_margin">Jeffrey E. Lee</h3><p><sup>9</sup>Department of Surgical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Lee%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Jeffrey E. Lee</a></div></div><div id="_co_idm139992322579392"><h3 class="no_margin">Christopher I. Amos</h3><p><sup>10</sup>Department of Community and Family Medicine, Geisel School of Medicine, Dartmouth College, Hanover, NH, USA</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Christopher I. Amos</a></div></div><div id="_co_idm139992322577408"><h3 class="no_margin">Peter Kraft</h3><p><sup>4</sup>Department of Epidemiology and</p><p><sup>5</sup>Department of Biostatistics, Harvard School of Public Health, Boston, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Peter Kraft</a></div></div><div id="_co_idm139992322574688"><h3 class="no_margin">Abrar A. Qureshi</h3><p><sup>1</sup>Clinical Research Program, Department of Dermatology and</p><p><sup>2</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Qureshi%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Abrar A. Qureshi</a></div></div><div id="_co_idm139992322571968"><h3 class="no_margin">Jiali Han</h3><p><sup>1</sup>Clinical Research Program, Department of Dermatology and</p><p><sup>2</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA, USA,</p><p><sup>3</sup>Department of Epidemiology, Tianjin Medical University Cancer Institute and Hospital, Tianjin, China,</p><p><sup>4</sup>Department of Epidemiology and</p><div>Find articles by <a href="/pubmed/?term=Han%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23548203">Jiali Han</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139992324646656_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139992324646656_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139992324646656_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139992324646656_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Clinical Research Program, Department of Dermatology and</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA, USA,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Epidemiology, Tianjin Medical University Cancer Institute and Hospital, Tianjin, China,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Epidemiology and</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Biostatistics, Harvard School of Public Health, Boston, MA, USA,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Division of Cancer Epidemiology, Department of Epidemiology and Public Health, University of Maryland School of Medicine, Baltimore, MD, USA,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>FAS Research Computing, Harvard University, Cambridge, MA, USA,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Epidemiology and</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Department of Surgical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA,  and</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Department of Community and Family Medicine, Geisel School of Medicine, Dartmouth College, Hanover, NH, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: 181 Longwood Avenue, Boston, MA 02120, USA. Tel: +1 6175252098; Fax: +1 6175252088; Email: <a href="mailto:dev@null" data-email="ude.dravrah.gninnahc@nah.ilaij" class="oemail">ude.dravrah.gninnahc@nah.ilaij</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139992324646656_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Dec 29; Revised 2013 Mar 4; Accepted 2013 Mar 22.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139992324646656_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2013. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3690971/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3690971/bin/supp_22_14_2948__index.html" data-ga-action="click_feat_suppl">supp_22_14_2948__index.html</a><span style="color:gray"> (1.2K)</span></div><div class="small guid">GUID:&nbsp;DEE033C8-7188-4584-844A-9FE9B9498579</div>
<div><a href="/pmc/articles/PMC3690971/bin/supp_ddt142_ddt142supp_figs.pdf" data-ga-action="click_feat_suppl">supp_ddt142_ddt142supp_figs.pdf</a><span style="color:gray"> (415K)</span></div><div class="small guid">GUID:&nbsp;B155C679-5502-42D4-B411-697AD2773AFD</div>
<div><a href="/pmc/articles/PMC3690971/bin/supp_ddt142_ddt142supp.doc" data-ga-action="click_feat_suppl">supp_ddt142_ddt142supp.doc</a><span style="color:gray"> (92K)</span></div><div class="small guid">GUID:&nbsp;65380766-E8E4-4679-AC9B-43CDD14266E6</div>
<div><a href="/pmc/articles/PMC3690971/bin/supp_ddt142_ddt142supp_tables.xlsx" data-ga-action="click_feat_suppl">supp_ddt142_ddt142supp_tables.xlsx</a><span style="color:gray"> (24K)</span></div><div class="small guid">GUID:&nbsp;F61D1C69-91D0-46B0-A657-70507F72D8C3</div>
</div></dd></dl></div><div id="idm139992329178160" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139992329178160title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Aiming to identify novel genetic loci for pigmentation and skin cancer, we conducted a series of genome-wide association studies on hair color, eye color, number of sunburns, tanning ability and number of non-melanoma skin cancers (NMSCs) among 10 183 European Americans in the discovery stage and 4504 European Americans in the replication stage (for eye color, 3871 males in the discovery stage and 2496 males in the replication stage). We targeted novel chromosome regions besides the known ones for replication. As a result, we identified a new region downstream of the <em>EDNRB</em> gene on 13q22 associated with hair color and the strongest association was the single-nucleotide polymorphism (SNP) rs975739 (<em>P</em> = 2.4 × 10<sup>−14</sup>; <em>P</em> = 5.4 × 10<sup>−9</sup> in the discovery set and <em>P</em> = 1.2 × 10<sup>−6</sup> in the replication set). Using blue, intermediate (including green) and brown eye colors as co-dominant outcomes, we identified the SNP rs3002288 in <em>VASH2</em> on 1q32.3 associated with brown eye (<em>P</em> = 7.0 × 10<sup>−8</sup>; <em>P</em> = 5.3 × 10<sup>−5</sup> in the discovery set and <em>P</em> = 0.02 in the replication set). Additionally, we identified a significant interaction between the SNPs rs7173419 and rs12913832 in the <em>OCA2</em> gene region on brown eye color (<em>P</em>-value for interaction = 3.8 × 10<sup>−3</sup>). As for the number of NMSCs, we identified two independent SNPs on chr6 and one SNP on chromosome 14: rs12203592 in <em>IRF4</em> (<em>P</em> = 7.2 × 10<sup>−14</sup>; <em>P</em> = 1.8 × 10<sup>−8</sup> in the discovery set and <em>P</em> = 6.7 × 10<sup>−7</sup> in the replication set), rs12202284 between <em>IRF4</em> and <em>EXOC2</em> (<em>P</em> = 5.0 × 10<sup>−8</sup>; <em>P</em> = 6.6 × 10<sup>−7</sup> in the discovery set and <em>P</em> = 3.0 × 10<sup>−3</sup> in the replication set) and rs8015138 upstream of <em>GNG2</em> (<em>P</em> = 6.6 × 10<sup>−8</sup>; <em>P</em> = 5.3 × 10<sup>−7</sup> in the discovery set and <em>P</em> = 0.01 in the replication set).</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first-last">Natural hair color, eye color, skin color and sun sensitivity reflect the variation in human basal and transient pigmentation, which has been known to be highly heritable (<a href="#DDT142C1" rid="DDT142C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900027">1</a>,<a href="#DDT142C2" rid="DDT142C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900041">2</a>). More recently, dozens of common genetic variants have been identified to be associated with pigmentary phenotypes from the genome-wide association studies (GWASs) and were reviewed by Gerstenblith <em>et al.</em> (<a href="#DDT142C3" rid="DDT142C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900074">3</a>). These GWASs identified genetic variants in the genes already known to be associated with pigmentation, such as melanocortin 1 receptor (<em>MC1R</em>) (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900066">4</a>,<a href="#DDT142C5" rid="DDT142C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900032">5</a>), oculocutaneous albinism II (OCA2) (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900057">4</a>), solute carrier family 45, member 2 (SLC45A2) (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900075">4</a>), tyrosinase (<em>TYR</em>) (<a href="#DDT142C5" rid="DDT142C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900034">5</a>,<a href="#DDT142C6" rid="DDT142C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900014">6</a>), tyrosinase-related protein 1 (<em>TYRP1</em>) (<a href="#DDT142C6" rid="DDT142C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900067">6</a>), kit ligand (<em>KITLG</em>) (<a href="#DDT142C5" rid="DDT142C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900070">5</a>) and the agouti signaling protein (<em>ASIP</em>) (<a href="#DDT142C6" rid="DDT142C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900077">6</a>), and also identified novel potential pigmentation genes, such as solute carrier family 24, member 4 (<em>SLC24A4</em>) (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900063">4</a>,<a href="#DDT142C5" rid="DDT142C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900065">5</a>), two-pore segment channel 2 (<em>TPCN2</em>) (<a href="#DDT142C6" rid="DDT142C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900021">6</a>) and interferon regulatory factor 4 (<em>IRF4</em>) (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900026">4</a>). Of interest, some of the genetic determinants are shared across different pigmentation traits, whereas others are distinct, suggesting various mechanisms of pigmentation across different organs (<a href="#DDT142C3" rid="DDT142C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900028">3</a>). Europeans have the majority of variation in pigmentation characteristics compared with other populations, especially in hair and eye colors (<a href="#DDT142C5" rid="DDT142C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900023">5</a>). In this study, we conducted a series of GWASs on hair color, eye colors, tanning ability and the number of sunburns in European Americans, aiming to identify novel genetic loci for pigmentation and to understand the variability of complex pigmentation characteristics in the general population to a greater extent. In addition, given that human pigmentation traits are susceptibility factors for skin cancer (<a href="#DDT142C3" rid="DDT142C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900072">3</a>,<a href="#DDT142C7" rid="DDT142C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900045">7</a>), we further evaluated the newly identified pigmentation loci in association with the risk of melanoma and non-melanoma skin cancer (NMSC). We also conducted a GWAS on the number of NMSCs in this study to identify new loci for the risk of NMSCs (<a href="#DDT142C8" rid="DDT142C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900055">8</a>–<a href="#DDT142C10" rid="DDT142C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900046">10</a>).</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p3" class="p p-first">The samples used in each GWAS of the present study are summarized in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>. The quantile–quantile plots based on the meta-analysis did not demonstrate a systematic deviation from the expected distribution, with the overall genomic control inflation factors (<em>λ</em><sub>GC</sub>) of 0.99–1.04 (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). The Manhattan plot of each GWAS is presented in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S2</a>. We focused on the novel chromosome regions suggesting promising associations besides the previously known ones and selected the single-nucleotide polymorphisms (SNPs) with the strongest associations in these regions for replication. A total of 16 SNPs were finally selected for replication, with detailed information presented in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>. Details of the SNP selection for replication are presented in Materials and Methods.</p><p id="__p4">As a result, for the GWAS of natural hair color, the SNP rs975739, which ranked the top besides the known pigmentation SNPs in the discovery set (<em>P</em> = 5.4 × 10<sup>−9</sup> in the discovery set), revealed a nominal significance in both of the replication sets (<em>P</em> = 0.05 in the skin cancer set, <em>P</em> = 3.3 × 10<sup>−6</sup> in the RF_NHS set and <em>P</em> = 1.2 × 10<sup>−6</sup> in the combined replication set, Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB1" rid-ob="ob-DDT142TB1" co-legend-rid=""><span>1</span></a>). This association reached a <em>P</em>-value of 2.4 × 10<sup>−14</sup> after combining the discovery set and the replication set. The variant allele (G allele) of this SNP was associated with darker hair color with the regression parameter beta of 0.08 and a standard error (SE) of 0.01. Another tag-SNP rs1146927 in this region, which was in linkage disequilibrium (LD) with rs975739 (<em>r</em><sup>2</sup> = 0.84 in this study), was also replicated in the replication set (<em>P</em> = 2.9 × 10<sup>−8</sup> in the discovery set, <em>P</em> = 0.03 in the skin cancer replication set and <em>P</em> = 2.8 × 10<sup>−9</sup> in the combined set, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). Both of the SNPs (rs975739 and rs1146927) reached the GWAS significance level and revealed a similar association pattern (beta = 0.8 and SE = 0.01 for rs975739; beta = 0.7 and SE = 0.01 for rs1146927), suggesting the same signal represented by them. The association of the other tag-SNP rs58188699 in this region (LD <em>r</em><sup>2</sup> = 0.42 with rs975739 in this study) with natural hair color (<em>P</em> = 7.5 × 10<sup>−8</sup> in the discovery set) was not replicated in the replication set (<em>P</em> = 0.27 in the skin cancer replication set, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT142TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p5">Association of the SNP rs975739 downstream of the <em>EDNRB</em> gene with non-red hair colors<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992362074336" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Data set</th><th colspan="5" rowspan="1" align="left">rs975739 T&gt;G<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left">Blond hair color</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>b</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value for heterogeneity</th></tr></thead><tbody><tr><td colspan="6" rowspan="1">Discovery set</td></tr><tr><td rowspan="1" colspan="1"> BC_NHS (female)<sup>c</sup></td><td rowspan="1" colspan="1">1741</td><td rowspan="1" colspan="1">223</td><td rowspan="1" colspan="1">0.07 (0.02)</td><td rowspan="1" colspan="1">4.8E − 03</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> T2D_NHS (female)<sup>d</sup></td><td rowspan="1" colspan="1">2377</td><td rowspan="1" colspan="1">302</td><td rowspan="1" colspan="1">0.07 (0.02)</td><td rowspan="1" colspan="1">1.6E − 03</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> T2D_HPFS (male)<sup>e</sup></td><td rowspan="1" colspan="1">1529</td><td rowspan="1" colspan="1">165</td><td rowspan="1" colspan="1">0.10 (0.03)</td><td rowspan="1" colspan="1">8.0E − 04</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_NHS (female)<sup>f</sup></td><td rowspan="1" colspan="1">783</td><td rowspan="1" colspan="1">97</td><td rowspan="1" colspan="1">0.06 (0.04)</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_HPFS (male)<sup>g</sup></td><td rowspan="1" colspan="1">640</td><td rowspan="1" colspan="1">87</td><td rowspan="1" colspan="1">0.08 (0.04)</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> All (meta-analysis)</td><td rowspan="1" colspan="1">7070</td><td rowspan="1" colspan="1">874</td><td rowspan="1" colspan="1">0.07 (0.01)</td><td rowspan="1" colspan="1">5.4E − 09</td><td rowspan="1" colspan="1">0.92</td></tr><tr><td colspan="6" rowspan="1">Replication set</td></tr><tr><td rowspan="1" colspan="1"> Skin cancer (female and male)<sup>h</sup></td><td rowspan="1" colspan="1">1639</td><td rowspan="1" colspan="1">183</td><td rowspan="1" colspan="1">0.05 (0.03)</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> RF_NHS (female)<sup>i</sup></td><td rowspan="1" colspan="1">2516</td><td rowspan="1" colspan="1">314</td><td rowspan="1" colspan="1">0.10 (0.02)</td><td rowspan="1" colspan="1">3.3E − 06</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> All (meta-analysis)</td><td rowspan="1" colspan="1">4155</td><td rowspan="1" colspan="1">497</td><td rowspan="1" colspan="1">0.08 (0.02)</td><td rowspan="1" colspan="1">1.2E − 06</td><td rowspan="1" colspan="1">0.16</td></tr><tr><td rowspan="1" colspan="1">Combined set (meta-analysis)</td><td rowspan="1" colspan="1">11 225</td><td rowspan="1" colspan="1">1371</td><td rowspan="1" colspan="1">0.08 (0.01)</td><td rowspan="1" colspan="1">2.4E − 14</td><td rowspan="1" colspan="1">0.74</td></tr></tbody></table></div><div id="largeobj_idm139992362074336" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3690971/table/DDT142TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139992362011552"><p id="__p6" class="p p-first-last"><sup>a</sup>Non-red hair colors were coded as: 1, blond; 2, light brown; 3, dark brown; 4, black; beta &gt; 0 means association with darker hair color.</p></div><div id="idm139992362010512"><p id="__p7" class="p p-first-last"><sup>b</sup>Adjusted for the top four principal components of genetic variance in the discovery sets and adjusted for gender in the skin cancer replication set.</p></div><div id="idm139992362009472"><p id="__p8" class="p p-first-last"><sup>c</sup>Postmenopausal invasive breast cancer case–control study nested within the NHS.</p></div><div id="idm139992362008496"><p id="__p9" class="p p-first-last"><sup>d</sup>Type 2 diabetes case–control study nested within the NHS.</p></div><div id="idm139992362007536"><p id="__p10" class="p p-first-last"><sup>e</sup>Type 2 diabetes case–control study nested within the HPFS.</p></div><div id="idm139992362006576"><p id="__p11" class="p p-first-last"><sup>f</sup>Coronary heart disease case–control study nested within the NHS.</p></div><div id="idm139992362005600"><p id="__p12" class="p p-first-last"><sup>g</sup>Coronary heart disease case–control study nested within the HPFS.</p></div><div id="idm139992362004624"><p id="__p13" class="p p-first-last"><sup>h</sup>Skin cancer case–control study nested within the NHS and HPFS.</p></div><div id="idm139992362003664"><p id="__p14" class="p p-first-last"><sup>i</sup>Renal function study nested within the NHS.</p></div></div></div><p id="__p15">These novel hair-color-related SNPs identified in this study are located in the region downstream of the endothelin receptor type B (<em>EDNRB</em>) gene. We additionally evaluated the association of all the SNPs within 200 kb surrounding the gene region with hair color. The regional association plot in the discovery set is presented in Figure&nbsp;<a href="/pmc/articles/PMC3690971/figure/DDT142F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT142F1" rid-ob="ob-DDT142F1" co-legend-rid="lgnd_DDT142F1"><span>1</span></a>. After adjustment for the SNP rs975739, the associations of 1611 SNPs in this region were substantially attenuated, and none of them remained regionally significant (0.05/1611 = 3.1 × 10<sup>−5</sup>, the smallest <em>P</em> = 1.5 × 10<sup>−4</sup> after adjustment, Fig.&nbsp;<a href="/pmc/articles/PMC3690971/figure/DDT142F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT142F1" rid-ob="ob-DDT142F1" co-legend-rid="lgnd_DDT142F1"><span>1</span></a>). We also constructed the haplotypes based on the three tag-SNPs (rs975739, rs1146927 and rs58188699). When we used the most prevalent haplotype (the combination of the common allele of rs58188699 with the rare alleles of both rs975739 and rs1146927, 38.7% of the population) as a reference, the haplotypes with the common alleles of both rs975739 and rs1146927, together with either the rare allele (37.8% of the population) or the common allele (18.9% of the population) of rs58188699, were marginally associated with lighter hair color (<em>P</em> = 0.08, beta = 0.07, SE = 0.04 with the rare allele of rs58188699, and <em>P</em> = 0.07, beta = 0.05, SE = 0.03 with the common allele of rs58188699), suggesting that these associations were originally from the same signal as rs975739 and rs1146927. Additionally, all of the 117 SNPs with <em>P</em> &lt; 1 × 10<sup>−7</sup> in the discovery stage were in LD (<em>r</em><sup>2</sup> &gt; 0.6).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT142F1" co-legend-rid="lgnd_DDT142F1"><a href="/pmc/articles/PMC3690971/figure/DDT142F1/" target="figure" rid-figpopup="DDT142F1" rid-ob="ob-DDT142F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992361990544" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddt14201.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddt14201.jpg" src="/pmc/articles/PMC3690971/bin/ddt14201.jpg"></div></a><div id="largeobj_idm139992361990544" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3690971/figure/DDT142F1/" target="figure" rid-figpopup="DDT142F1" rid-ob="ob-DDT142F1"></a><a target="object" href="/pmc/articles/PMC3690971/figure/DDT142F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT142F1"><div><a class="figpopup" href="/pmc/articles/PMC3690971/figure/DDT142F1/" target="figure" rid-figpopup="DDT142F1" rid-ob="ob-DDT142F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p16">Regional plot of the <em>EDNRB</em> gene. Manhattan plots for the GWAS of hair color in the region within 200 kb surrounding <em>EDNRB</em> on chromosome 13, before and after adjustment by the SNP rs975739.</p></div></div></div><p id="__p17">The SNP rs975739 was not associated with eye colors (<em>P</em> = 0.78 for brown eye color, <em>P</em> = 0.33 for intermediate eye color, <em>P</em> = 0.46 for blue eye color), tanning ability (<em>P</em> = 0.72) or the number of sunburns (<em>P</em> = 0.67). Similar results were detected for the other two SNPs (rs1146927 and rs58188699). We also evaluated the association between the two significant SNPs and red hair color phenotype, and no significant association was found (<em>P</em> = 0.95 for rs975739 and <em>P</em> = 0.90 for rs1146927).</p><p id="__p18">We also replicated the association between the SNP rs12421680 in the neurotrimin (<em>NTM</em>) gene on chromosome 11 and the number of sunburns (<em>P</em> = 5.3 × 10<sup>−5</sup> in the discovery set and <em>P</em> = 0.02 in the replication set, Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB2" rid-ob="ob-DDT142TB2" co-legend-rid=""><span>2</span></a>). After combining the discovery and replication sets, <em>P</em>-value for the association of this SNP reached 5.6 × 10<sup>−6</sup>. The variant allele (A allele) was associated with an increased number of sunburns (beta = 0.41, SE = 0.09). Additionally, based on the transcript expression profiling data of 79 HapMap CEU cell lines (NCBI GEO database, accession <a id="__tag_320900079" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE7792" ref="/geo/query/acc.cgi?acc=GSE7792"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE7792","term_id":"7792"}}</span>GSE7792</a>) (<a href="#DDT142C11" rid="DDT142C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900006">11</a>), we found that the AA genotype was associated with a lower expression of <em>NTM</em> than the GG/GA genotypes (<em>P</em> &lt; 0.001, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>). There was no association between this SNP and eye colors (<em>P</em> = 0.54 for brown eye color, <em>P</em> = 0.69 for intermediate eye color, <em>P</em> = 0.80 for blue eye color), natural hair color (<em>P</em> = 0.55) or tanning ability (<em>P</em> = 0.49). Because the imputation quality of this SNP was not particularly high in four subsets in the discovery stage (imputation <em>r</em><sup>2</sup>: 0.61–0.62 for T2D_NHS, CHD_NHS, T2D_HPFS and CHD_HPFS), we re-genotyped this SNP in the subset of T2D_HPFS using TaqMan assay to validate the result. The association of this SNP with the number of sunburns was comparable by using the re-genotyped data (beta = 0.82, SE = 0.30, <em>P</em> = 0.01) and the imputed data (beta = 0.88, SE = 0.38, <em>P</em> = 0.02).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT142TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p19">Association of the SNP rs12421680 in the <em>NTM</em> gene with the number of sunburns<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992321029824" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Data set</th><th colspan="5" rowspan="1" align="left">rs12421680 G&gt;A<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left">Sunburns 10+</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>i</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value for heterogeneity</th></tr></thead><tbody><tr><td colspan="6" rowspan="1">Discovery set</td></tr><tr><td rowspan="1" colspan="1"> BC_NHS (female)<sup>b</sup></td><td rowspan="1" colspan="1">2153</td><td rowspan="1" colspan="1">852</td><td rowspan="1" colspan="1">0.37 (0.19)</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> T2D_NHS (female)<sup>c</sup></td><td rowspan="1" colspan="1">2915</td><td rowspan="1" colspan="1">1430</td><td rowspan="1" colspan="1">0.55 (0.22)</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> T2D_HPFS (male)<sup>d</sup></td><td rowspan="1" colspan="1">2170</td><td rowspan="1" colspan="1">1075</td><td rowspan="1" colspan="1">0.88 (0.38)</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_NHS (female)<sup>e</sup></td><td rowspan="1" colspan="1">1056</td><td rowspan="1" colspan="1">668</td><td rowspan="1" colspan="1">0.41 (0.35)</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_HPFS (male)<sup>f</sup></td><td rowspan="1" colspan="1">989</td><td rowspan="1" colspan="1">631</td><td rowspan="1" colspan="1">0.67 (0.56)</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> All (meta-analysis)</td><td rowspan="1" colspan="1">9283</td><td rowspan="1" colspan="1">4656</td><td rowspan="1" colspan="1">0.49 (0.12)</td><td rowspan="1" colspan="1">5.3E − 05</td><td rowspan="1" colspan="1">0.79</td></tr><tr><td colspan="6" rowspan="1">Replication set</td></tr><tr><td rowspan="1" colspan="1"> Skin cancer (female and male)<sup>g</sup></td><td rowspan="1" colspan="1">1655</td><td rowspan="1" colspan="1">558</td><td rowspan="1" colspan="1">0.44 (0.23)</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> RF_NHS (female)<sup>h</sup></td><td rowspan="1" colspan="1">2581</td><td rowspan="1" colspan="1">891</td><td rowspan="1" colspan="1">0.24 (0.17)</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> All (meta-analysis)</td><td rowspan="1" colspan="1">4236</td><td rowspan="1" colspan="1">1449</td><td rowspan="1" colspan="1">0.31 (0.14)</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.49</td></tr><tr><td rowspan="1" colspan="1">Combined set (meta-analysis)</td><td rowspan="1" colspan="1">13 519</td><td rowspan="1" colspan="1">6105</td><td rowspan="1" colspan="1">0.41 (0.09)</td><td rowspan="1" colspan="1">5.6E − 06</td><td rowspan="1" colspan="1">0.32</td></tr></tbody></table></div><div id="largeobj_idm139992321029824" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3690971/table/DDT142TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139992320967232"><p id="__p20" class="p p-first-last"><sup>a</sup>The number of sunburns was coded as a continuous variable; beta &gt; 0 means association with increased number of sunburns.</p></div><div id="idm139992320966160"><p id="__p21" class="p p-first-last"><sup>b</sup>Postmenopausal invasive breast cancer case–control study nested within the NHS.</p></div><div id="idm139992320965184"><p id="__p22" class="p p-first-last"><sup>c</sup>Type 2 diabetes case–control study nested within the NHS.</p></div><div id="idm139992320964304"><p id="__p23" class="p p-first-last"><sup>d</sup>Type 2 diabetes case–control study nested within the HPFS.</p></div><div id="idm139992320963344"><p id="__p24" class="p p-first-last"><sup>e</sup>Coronary heart disease case–control study nested within the NHS.</p></div><div id="idm139992320962320"><p id="__p25" class="p p-first-last"><sup>f</sup>Coronary heart disease case–control study nested within the HPFS.</p></div><div id="idm139992320961344"><p id="__p26" class="p p-first-last"><sup>g</sup>Skin cancer case–control study nested within the NHS and HPFS.</p></div><div id="idm139992320960384"><p id="__p27" class="p p-first-last"><sup>h</sup>Renal function study nested within the NHS.</p></div><div id="idm139992320959504"><p id="__p28" class="p p-first-last"><sup>i</sup>Adjusted for age and the top four principal components of genetic variance in the discovery sets and adjusted for age and gender in the skin cancer replication set.</p></div></div></div><p id="__p29">As for the GWASs on eye color, we replicated the association of the SNP rs3002288 in vasohibin 2 (<em>VASH2</em>) with brown eye color (<em>P</em> = 7.0 × 10<sup>−8</sup> in the combined set; <em>P</em> = 4.3 × 10<sup>−5</sup> in the discovery set and <em>P</em> = 3.3 × 10<sup>−5</sup> in the replication set), as well as the association of the SNP rs12520016 upstream of DNA polymerase sigma (<em>POLS</em>) with intermediate (including green) eye color (<em>P</em> = 9.1 × 10<sup>−7</sup> in the combined set; <em>P</em> = 4.7 × 10<sup>−5</sup> in the discovery set and <em>P</em> = 0.03 in the replication set, Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB3" rid-ob="ob-DDT142TB3" co-legend-rid=""><span>3</span></a>). The variant allele of rs3002288 (A allele) showed an inverse association with brown eye color (beta = −0.22, SE = 0.04), and the variant allele of rs12520016 (G allele) showed a positive association with intermediate eye color (beta = 0.42, SE = 0.08). The association between the SNP rs3002288 and brown eye color was approaching the GWAS significance level of 10<sup>−8</sup>. Neither of the two SNPs revealed an association with natural hair color (<em>P</em> = 0.37 for rs3002288 and <em>P</em> = 0.58 for rs12520016), tanning ability (<em>P</em> = 0.93 for rs3002288 and <em>P</em> = 0.71 for rs12520016) or the number of sunburns (<em>P</em> = 0.57 for rs3002288 and <em>P</em> = 0.91 for rs12520016).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT142TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p30">Association of the SNP rs3002288 in the <em>VASH2</em> gene with brown eye color and the SNP rs12520016 upstream of <em>POLS</em> gene with intermediate eye color<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992320941536" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Data set</th><th rowspan="2" colspan="1" align="left">Sample size</th><th colspan="4" rowspan="1" align="left">rs3002288 G&gt;A<hr></th><th colspan="4" rowspan="1" align="left">rs12520016 T&gt;G<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Brown eye color</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>f</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value for heterogeneity</th><th rowspan="1" colspan="1" align="left">Intermediate eye color</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>f</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value for heterogeneity</th></tr></thead><tbody><tr><td colspan="10" rowspan="1">Discovery set</td></tr><tr><td rowspan="1" colspan="1"> T2D_HPFS (male)<sup>b</sup></td><td rowspan="1" colspan="1">2292</td><td rowspan="1" colspan="1">717</td><td rowspan="1" colspan="1">−0.25 (0.07)</td><td rowspan="1" colspan="1">4.0E − 04</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">761</td><td rowspan="1" colspan="1">0.52 (0.14)</td><td rowspan="1" colspan="1">3.4E − 04</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_HPFS (male)<sup>c</sup></td><td rowspan="1" colspan="1">1064</td><td rowspan="1" colspan="1">316</td><td rowspan="1" colspan="1">−0.12 (0.11)</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">348</td><td rowspan="1" colspan="1">0.58 (0.19)</td><td rowspan="1" colspan="1">2.8E − 03</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> KS_HPFS (male)<sup>d</sup></td><td rowspan="1" colspan="1">515</td><td rowspan="1" colspan="1">165</td><td rowspan="1" colspan="1">−0.50 (0.15)</td><td rowspan="1" colspan="1">5.1E − 04</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">146</td><td rowspan="1" colspan="1">0.15 (0.31)</td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> All (meta-analysis)</td><td rowspan="1" colspan="1">3871</td><td rowspan="1" colspan="1">1033</td><td rowspan="1" colspan="1">−0.25 (0.06)</td><td rowspan="1" colspan="1">4.3E − 06</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">1109</td><td rowspan="1" colspan="1">0.49 (0.11)</td><td rowspan="1" colspan="1">4.7E − 06</td><td rowspan="1" colspan="1">0.48</td></tr><tr><td colspan="10" rowspan="1">Replication set</td></tr><tr><td rowspan="1" colspan="1"> PrCa_HPFS (male)<sup>e</sup></td><td rowspan="1" colspan="1">2496</td><td rowspan="1" colspan="1">760</td><td rowspan="1" colspan="1">−0.18 (0.06)</td><td rowspan="1" colspan="1">3.3E − 03</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">805</td><td rowspan="1" colspan="1">0.29 (0.14)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Combined set (meta-analysis)</td><td rowspan="1" colspan="1">6367</td><td rowspan="1" colspan="1">1793</td><td rowspan="1" colspan="1">−0.22 (0.04)</td><td rowspan="1" colspan="1">7.0E − 08</td><td rowspan="1" colspan="1">0.38</td><td rowspan="1" colspan="1">1914</td><td rowspan="1" colspan="1">0.42 (0.08)</td><td rowspan="1" colspan="1">9.1E − 07</td><td rowspan="1" colspan="1">0.25</td></tr></tbody></table></div><div id="largeobj_idm139992320941536" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3690971/table/DDT142TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139992326892784"><p id="__p31" class="p p-first-last"><sup>a</sup>Eye colors were coded as co-dominant outcomes (brown versus non-brown eye color and green versus non-green eye color).</p></div><div id="idm139992326891776"><p id="__p32" class="p p-first-last"><sup>b</sup>Type 2 diabetes case–control study nested within the HPFS.</p></div><div id="idm139992326890848"><p id="__p33" class="p p-first-last"><sup>c</sup>Coronary heart disease case–control study nested within the HPFS.</p></div><div id="idm139992326889872"><p id="__p34" class="p p-first-last"><sup>d</sup>Kidney stone case–control study nested within the HPFS.</p></div><div id="idm139992326888912"><p id="__p35" class="p p-first-last"><sup>e</sup>Prostate cancer case–control study nested within the HPFS.</p></div><div id="idm139992326887952"><p id="__p36" class="p p-first-last"><sup>f</sup>Adjusted for the top four principal components of genetic variance in the discovery sets.</p></div></div></div><p id="__p37">Of interest, we found that the SNP rs7173419 located in the intron region of the <em>OCA2</em> gene revealed a significant association with brown eye color (T allele, beta = 0.33, SE = 0.06; <em>P</em> = 1.3 × 10<sup>−7</sup> in the discovery set), whereas this association was inverted (T allele, beta = −0.48, SE = 0.08; <em>P</em> = 1.5 × 10<sup>−8</sup> in the discovery set) after adjustment by the top SNP in this region (rs12913832, A allele, beta = 2.82, SE = 0.10; <em>P</em> = 1.2 × 10<sup>−177</sup> in the discovery stage of brown eye color GWAS). The SNP rs12913832 is located upstream of <em>OCA2</em> (in the intron of <em>HERC2</em>) and known as a pigmentation SNP by controlling the expression of <em>OCA2</em> (<a href="#DDT142C12" rid="DDT142C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900071">12</a>,<a href="#DDT142C13" rid="DDT142C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900076">13</a>). We replicated this finding in the replication set (rs7173419, T allele; beta = 0.56, SE = 0.07, <em>P</em> = 4.1 × 10<sup>−15</sup> before adjustment by rs12913832; and beta = −0.21, SE = 0.10, <em>P</em> = 0.04 after adjustment by rs12913832). We further conducted a stratification analysis of the SNP rs12913832 by the genotypes of rs7173419 among the pooled population of the discovery set and the replication set. As shown in Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB4" rid-ob="ob-DDT142TB4" co-legend-rid=""><span>4</span></a>, the effect of the SNP rs12913832 on brown eye color decreased with more counts of the variant allele (T allele) of rs7173419, and the interaction of these two SNPs was significant (<em>P</em> = 3.8 × 10<sup>−3</sup>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT142TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p38">Interaction between the SNPs rs7173419 and rs12913832 on brown eye color<sup>a</sup> and melanoma risk</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992326871488" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">rs7173419</th><th colspan="2" rowspan="1" align="left">Eye color<hr></th><th colspan="3" rowspan="1" align="left">rs12913832 G&gt;A<sup>b</sup><hr></th><th rowspan="2" colspan="1" align="left"><em>P</em>-value for interaction</th></tr><tr><th rowspan="1" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left">Brown eye color, % (SD)</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SD</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>c</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">CC</td><td rowspan="1" colspan="1">3755</td><td rowspan="1" colspan="1">26.3 (0.7)</td><td rowspan="1" colspan="1">3.92</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">6.9E − 215</td><td rowspan="1" colspan="1">3.8E − 03</td></tr><tr><td rowspan="1" colspan="1">CT</td><td rowspan="1" colspan="1">2178</td><td rowspan="1" colspan="1">36.1 (1.0)</td><td rowspan="1" colspan="1">2.54</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">6.9E − 103</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">TT</td><td rowspan="1" colspan="1">337</td><td rowspan="1" colspan="1">45.4 (2.7)</td><td rowspan="1" colspan="1">2.18</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">6.6E − 17</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="2" colspan="1">rs7173419</td><td colspan="2" rowspan="1" align="left">Melanoma risk</td><td colspan="3" rowspan="1" align="left">rs12913832 G&gt;A<sup>b</sup></td><td rowspan="2" colspan="1"><em>P</em>-value for interaction</td></tr><tr><td rowspan="1" colspan="1">Cases</td><td rowspan="1" colspan="1">Controls</td><td rowspan="1" colspan="1">OR</td><td rowspan="1" colspan="1">95% CI</td><td rowspan="1" colspan="1"><em>P</em>-value<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">CC</td><td rowspan="1" colspan="1">1134</td><td rowspan="1" colspan="1">610</td><td rowspan="1" colspan="1">0.64</td><td rowspan="1" colspan="1">0.53–0.77</td><td rowspan="1" colspan="1">3.6E − 06</td><td rowspan="1" colspan="1">0.06</td></tr><tr><td rowspan="1" colspan="1">CT</td><td rowspan="1" colspan="1">592</td><td rowspan="1" colspan="1">363</td><td rowspan="1" colspan="1">0.76</td><td rowspan="1" colspan="1">0.61–0.94</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">TT</td><td rowspan="1" colspan="1">78</td><td rowspan="1" colspan="1">53</td><td rowspan="1" colspan="1">1.05</td><td rowspan="1" colspan="1">0.63–1.73</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm139992326871488" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3690971/table/DDT142TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139992323214368"><p id="__p39" class="p p-first-last"><sup>a</sup>Eye colors were coded as brown versus non-brown eye color.</p></div><div id="idm139992323213408"><p id="__p40" class="p p-first-last"><sup>b</sup>rs12913832 was coded in an additive model, with G as wild-type allele and A as variant allele.</p></div><div id="idm139992323212416"><p id="__p41" class="p p-first-last"><sup>c</sup>Adjusted for the top four principal components.</p></div><div id="idm139992323211488"><p id="__p42" class="p p-first-last"><sup>d</sup>Adjusted for age, gender and the top two principal components.</p></div></div></div><p id="__p43">As for the number of NMSCs, we identified two independent SNPs on chromosome 6 (LD <em>r</em><sup>2</sup> = 0.12): rs12203592 in <em>IRF4</em> with a <em>P</em>-value of 7.2 × 10<sup>−14</sup> in the combined set (T allele, beta = 0.21, SE = 0.03; <em>P</em> = 1.8 × 10<sup>−8</sup> in the discovery stage and <em>P</em> = 6.7 × 10<sup>−7</sup> in the replication stage) and rs12202284 between <em>IRF4</em> and exocyst complex component 2 (<em>EXOC2</em>) with a <em>P</em>-value of 5.0 × 10<sup>−8</sup> in the combined set (A allele, beta = 0.15, SE = 0.03; <em>P</em> = 6.6 × 10<sup>−7</sup> in the discovery stage and <em>P</em> = 3.0 × 10<sup>−3</sup> in the replication stage; Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB5/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB5" rid-ob="ob-DDT142TB5" co-legend-rid=""><span>5</span></a>). After mutual adjustment by each other, the association of each SNP with the number of NMSCs remained significant (<em>P</em> = 6.0 × 10<sup>−4</sup> for rs12203592 and <em>P</em> = 5.8 × 10<sup>−3</sup> for rs12202284), suggesting independent effects of these two SNPs. We previously reported an association of the SNP rs12203592 with hair color (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900013">4</a>) and tanning ability (<a href="#DDT142C14" rid="DDT142C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900078">14</a>). In this study, we additionally found a significant association of this SNP with the number of sunburns (T allele, beta = 1.13, SE = 0.24; <em>P</em> = 2.2 × 10<sup>−6</sup>; Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB6/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB6" rid-ob="ob-DDT142TB6" co-legend-rid=""><span>6</span></a>). We also identified a novel SNP rs8015138 upstream of guanine nucleotide-binding protein, gamma 2 (<em>GNG2</em>), on chromosome 14 associated with the number of NMSCs with a <em>P</em>-value of 6.6 × 10<sup>−8</sup> in the combined set (C allele, beta = −0.11, SE = 0.02; <em>P</em> = 5.3 × 10<sup>−7</sup> in the discovery stage and <em>P</em> = 0.01 in the replication stage; Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB5/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB5" rid-ob="ob-DDT142TB5" co-legend-rid=""><span>5</span></a>).

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT142TB5"><h3>Table&nbsp;5.</h3><!--caption a7--><div class="caption"><p id="__p44">Association of the SNP rs12203592 in the <em>IRF4</em> gene, rs12202284 between <em>IRF4</em> and EXOC2 and rs8015138 upstream of GNG2 with the number of NMSCs<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992323185040" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Data set</th><th rowspan="2" colspan="1" align="left">Sample size</th><th rowspan="2" colspan="1" align="left">NMSCs, 5+</th><th colspan="3" rowspan="1" align="left">rs12203592 C&gt;T<hr></th><th colspan="3" rowspan="1" align="left">rs12202284 C&gt;A<hr></th><th colspan="3" rowspan="1" align="left">rs8015138 A&gt;C<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>h</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value for heterogeneity</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>h</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value for heterogeneity</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>h</sup></th><th rowspan="1" colspan="1" align="left"><em>P</em>-value for heterogeneity</th></tr></thead><tbody><tr><td colspan="12" rowspan="1">Discovery set</td></tr><tr><td rowspan="1" colspan="1"> BC_NHS (female)<sup>b</sup></td><td rowspan="1" colspan="1">1620</td><td rowspan="1" colspan="1">45</td><td rowspan="1" colspan="1">0.20 (0.07)</td><td rowspan="1" colspan="1">0.0037</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.21 (0.08)</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.09 (0.05)</td><td rowspan="1" colspan="1">0.081</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> T2D_NHS (female)<sup>c</sup></td><td rowspan="1" colspan="1">2427</td><td rowspan="1" colspan="1">65</td><td rowspan="1" colspan="1">0.21 (0.08)</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.14 (0.07)</td><td rowspan="1" colspan="1">0.039</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.12 (0.04)</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> T2D_HPFS (male)<sup>d</sup></td><td rowspan="1" colspan="1">1308</td><td rowspan="1" colspan="1">83</td><td rowspan="1" colspan="1">0.24 (0.15)</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.17 (0.12)</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.07 (0.08)</td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_NHS (female)<sup>e</sup></td><td rowspan="1" colspan="1">760</td><td rowspan="1" colspan="1">23</td><td rowspan="1" colspan="1">0.21 (0.14)</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.36 (0.11)</td><td rowspan="1" colspan="1">0.0015</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.23 (0.08)</td><td rowspan="1" colspan="1">0.002</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_HPFS (male)<sup>f</sup></td><td rowspan="1" colspan="1">632</td><td rowspan="1" colspan="1">39</td><td rowspan="1" colspan="1">0.47 (0.25)</td><td rowspan="1" colspan="1">0.056</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.49 (0.21)</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.32 (0.13)</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> All (meta-analysis)</td><td rowspan="1" colspan="1">6747</td><td rowspan="1" colspan="1">255</td><td rowspan="1" colspan="1">0.22 (0.03)</td><td rowspan="1" colspan="1">1.8E − 8</td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">0.21 (0.04)</td><td rowspan="1" colspan="1">6.6E − 7</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">−0.14 (0.03)</td><td rowspan="1" colspan="1">5.3E − 7</td><td rowspan="1" colspan="1">0.26</td></tr><tr><td colspan="12" rowspan="1">Replication set</td></tr><tr><td rowspan="1" colspan="1"> Skin cancer (female and male)<sup>g</sup></td><td rowspan="1" colspan="1">2078</td><td rowspan="1" colspan="1">118</td><td rowspan="1" colspan="1">0.20 (0.04)</td><td rowspan="1" colspan="1">6.7E − 7</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.10 (0.03)</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.08 (0.03)</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Combined set (meta-analysis)</td><td rowspan="1" colspan="1">8825</td><td rowspan="1" colspan="1">373</td><td rowspan="1" colspan="1">0.21 (0.03)</td><td rowspan="1" colspan="1">7.2E − 14</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">0.15 (0.03)</td><td rowspan="1" colspan="1">5.0E − 8</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">−0.11 (0.02)</td><td rowspan="1" colspan="1">6.6E − 8</td><td rowspan="1" colspan="1">0.11</td></tr></tbody></table></div><div id="largeobj_idm139992323185040" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3690971/table/DDT142TB5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139992321381872"><p id="__p45" class="p p-first-last"><sup>a</sup>The number of NMSCs was coded as: 1-0, 2-1, 3-2-4, 4-5+; beta &gt; 0 means association with increased number of NMSCs.</p></div><div id="idm139992321380848"><p id="__p46" class="p p-first-last"><sup>b</sup>Postmenopausal invasive breast cancer case–control study nested within the NHS.</p></div><div id="idm139992321379872"><p id="__p47" class="p p-first-last"><sup>c</sup>Type 2 diabetes case–control study nested within the NHS.</p></div><div id="idm139992321378992"><p id="__p48" class="p p-first-last"><sup>d</sup>Type 2 diabetes case–control study nested within the HPFS.</p></div><div id="idm139992321378032"><p id="__p49" class="p p-first-last"><sup>e</sup>Coronary heart disease case–control study nested within the NHS.</p></div><div id="idm139992321377056"><p id="__p50" class="p p-first-last"><sup>f</sup>Coronary heart disease case–control study nested within the HPFS.</p></div><div id="idm139992321376080"><p id="__p51" class="p p-first-last"><sup>g</sup>Skin cancer case–control study nested within the NHS and HPFS.</p></div><div id="idm139992321375120"><p id="__p52" class="p p-first-last"><sup>h</sup>Adjusted for the top four principal components of genetic variance in the discovery sets and adjusted for gender in the skin cancer replication set.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT142TB6"><h3>Table&nbsp;6.</h3><!--caption a7--><div class="caption"><p id="__p53">Summary of pigmentary SNPs identified from the previous and the present GWASs and their associations with pigmentation traits, melanoma risk and NMSCs in this study<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992321371696" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="left">Gene</th><th rowspan="2" colspan="1" align="left">Chr.</th><th rowspan="2" colspan="1" align="left">Location</th><th rowspan="2" colspan="1" align="left">Ref.</th><th rowspan="2" colspan="1" align="left">A1</th><th colspan="3" rowspan="1" align="left">Hair color<hr></th><th colspan="3" rowspan="1" align="left">Tanning ability<hr></th><th colspan="3" rowspan="1" align="left">Number of sunburns<hr></th><th colspan="3" rowspan="1" align="left">Blue eye color<hr></th><th rowspan="2" colspan="1" align="left">&nbsp;</th><th rowspan="2" colspan="1" align="left">&nbsp;</th><th rowspan="2" colspan="1" align="left">&nbsp;</th></tr><tr><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs16891982<sup>b</sup></td><td rowspan="1" colspan="1"><em>SLC45A2</em></td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">33 987 450</td><td rowspan="1" colspan="1">G (major)</td><td rowspan="1" colspan="1">C (minor)</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">−0.42</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">5.4E − 03</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12203592</td><td rowspan="1" colspan="1"><em>IRF4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">341 321</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.0E − 28</td><td rowspan="1" colspan="1">−0.35</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">3.3E − 23</td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">2.2E − 06</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1408799</td><td rowspan="1" colspan="1"><em>TYRP1</em></td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">12 662 097</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">5.6E − 03</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">4.3E − 04</td><td rowspan="1" colspan="1">−0.17</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">−0.27</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">4.9E − 07</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs35264875<sup>c</sup></td><td rowspan="1" colspan="1"><em>TPCN2</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">68 602 975</td><td rowspan="1" colspan="1">T (minor)</td><td rowspan="1" colspan="1">A (major)</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">3.8E − 04</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs3829241</td><td rowspan="1" colspan="1"><em>TPCN2</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">68 611 939</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">2.1E − 03</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1126809<sup>c</sup></td><td rowspan="1" colspan="1"><em>TYR</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">88 657 609</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">5.0E − 21</td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">1.8E − 08</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1042602<sup>b</sup></td><td rowspan="1" colspan="1"><em>TYR</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">88 551 344</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">−0.08</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12821256</td><td rowspan="1" colspan="1"><em>KITLG</em></td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">87 852 466</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">6.8E − 19</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">−0.21</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">−0.18</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12896399</td><td rowspan="1" colspan="1"><em>SLC24A4</em></td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">91 843 416</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.16</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.5E − 36</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">4.1E − 11</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12913832</td><td rowspan="1" colspan="1"><em>HERC2</em></td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">26 039 213</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.4E − 167</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.4E − 22</td><td rowspan="1" colspan="1">−0.48</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">2.1E − 05</td><td rowspan="1" colspan="1">−2.84</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">1.2E − 158</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1805007</td><td rowspan="1" colspan="1"><em>MC1R</em></td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">88 513 618</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.16</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">2.9E − 09</td><td rowspan="1" colspan="1">−0.39</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.1E − 65</td><td rowspan="1" colspan="1">1.66</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">1.5E − 19</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1805008<sup>c</sup></td><td rowspan="1" colspan="1"><em>MC1R</em></td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">88 513 645</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.09</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">7.2E − 05</td><td rowspan="1" colspan="1">−0.16</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.3E − 13</td><td rowspan="1" colspan="1">0.71</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">4.8E − 05</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4911414</td><td rowspan="1" colspan="1"><em>ASIP Haplotype</em></td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">32 193 105</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.3E − 04</td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">3.8E − 09</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1015362</td><td rowspan="1" colspan="1"><em>ASIP Haplotype</em></td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">32 202 273</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs975739<sup>d</sup></td><td rowspan="1" colspan="1"><em>EDNRB</em></td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">77 279 147</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">5.4E − 09</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.72</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.67</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12421680<sup>d</sup></td><td rowspan="1" colspan="1"><em>NTM</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">130 856 178</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.55</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">5.3E − 05</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs3002288<sup>d</sup></td><td rowspan="1" colspan="1"><em>VASH2</em></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">211 193 188</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.57</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12520016<sup>d</sup></td><td rowspan="1" colspan="1"><em>POLS</em></td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">6 820 312</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.71</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">2.1E − 03</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td></tr><tr><td colspan="3" rowspan="1">Intermediate/green eye color</td><td colspan="3" rowspan="1">Brown eye color</td><td colspan="3" rowspan="1" align="left">Melanoma risk</td><td colspan="3" rowspan="1" align="left">Mole count</td><td colspan="3" rowspan="1" align="left">BCC risk</td><td colspan="3" rowspan="1" align="left">SCC risk</td><td colspan="3" rowspan="1" align="left">Number of NMSCs</td></tr><tr><td rowspan="1" colspan="1">Beta</td><td rowspan="1" colspan="1">SE</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">Beta</td><td rowspan="1" colspan="1">SE</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">Beta</td><td rowspan="1" colspan="1">SE</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">Beta</td><td rowspan="1" colspan="1">SE</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">Beta</td><td rowspan="1" colspan="1">SE</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">Beta</td><td rowspan="1" colspan="1">SE</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">Beta</td><td rowspan="1" colspan="1">SE</td><td rowspan="1" colspan="1"><em>P</em>-value</td></tr><tr><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1">−0.52</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">−0.49</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">−0.21</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">7.0E − 04</td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">9.5E − 09</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.8E − 08</td></tr><tr><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">4.8E − 03</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.68</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.70</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.71</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.15</td></tr><tr><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.70</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.12</td></tr><tr><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">6.0E − 03</td></tr><tr><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">−0.10</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.75</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">−0.08</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.87</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.16</td></tr><tr><td rowspan="1" colspan="1">−0.27</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">2.5E − 07</td><td rowspan="1" colspan="1">−0.08</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">−0.09</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1">−0.24</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">2.1E − 05</td><td rowspan="1" colspan="1">2.82</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">1.2E − 177</td><td rowspan="1" colspan="1">−0.35</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">1.6E − 07</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">−0.15</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">6.6E − 04</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">−0.08</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">−0.10</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">0.55</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">8.0E − 09</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">3.8E − 09</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">1.1E − 03</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">3.40E − 10</td></tr><tr><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">1.2E − 03</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.99</td></tr><tr><td rowspan="1" colspan="1">−0.09</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.72</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.11</td></tr><tr><td rowspan="1" colspan="1">−0.10</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.75</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.74</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.87</td></tr><tr><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.78</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">−0.09</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">9.0E − 03</td></tr><tr><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">−0.10</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.5</td></tr><tr><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−0.25</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">4.3E − 06</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.63</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.55</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.88</td></tr><tr><td rowspan="1" colspan="1">−0.49</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">4.7E − 06</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.47</td></tr></tbody></table></div><div id="largeobj_idm139992321371696" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3690971/table/DDT142TB6/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139992326431808"><p id="__p54" class="p p-first-last"><sup>a</sup>The SNP rs1805009 in the <em>MC1R</em> gene was not in 1000 Genome or HapMap imputed data.</p></div><div id="idm139992326430128"><p id="__p55" class="p p-first-last"><sup>b</sup>Not in 1000 Genome imputed data.</p></div><div id="idm139992326429104"><p id="__p56" class="p p-first-last"><sup>c</sup>Not in HapMap imputed data.</p></div><div id="idm139992326428064"><p id="__p57" class="p p-first-last"><sup>d</sup>Newly identified pigmentation SNPs in this present GWAS.</p></div></div></div><p id="__p58">We did not find novel susceptibility loci for tanning ability, but replicated the previous findings, including rs12203592 in <em>IRF4</em> (<em>P</em> = 3.3 × 10<sup>−23</sup>), rs1126809 in <em>TYR</em> (<em>P</em> = 5.0 × 10<sup>−21</sup>), rs12913832 in <em>HERC2</em> (<em>P</em> = 1.4 × 10<sup>−22</sup>) and rs1805007 (<em>P</em> = 1.1 × 10<sup>−65</sup>) and rs1805008 (<em>P</em> = 1.3 × 10<sup>−13</sup>) in <em>MC1R</em> (Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB6/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB6" rid-ob="ob-DDT142TB6" co-legend-rid=""><span>6</span></a>).</p><p id="__p59">Based on the knowledge that human pigmentation traits are host susceptibility factors for skin cancer (<a href="#DDT142C7" rid="DDT142C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900059">7</a>), we further evaluated the association between the pigmentation determining loci and melanoma risk using the melanoma GWAS of the MD Anderson Cancer Center with 1804 melanoma cases and 1026 controls. As shown in Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB7/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB7" rid-ob="ob-DDT142TB7" co-legend-rid=""><span>7</span></a>, the SNP rs12520016 associated with intermediate eye color was associated with decreased risk of melanoma (G allele; OR, 0.74; 95% CI, 0.58–0.95; <em>P</em> = 0.02). The SNP rs12421680 associated with increased numbers of sunburns and lower expression of the gene <em>NTM</em> was associated with increased risk of melanoma (A allele; OR 1.12; 95% CI 1.00–1.26; <em>P</em> = 0.05). Consistent with the interaction between the SNP rs7173419 and the SNP rs12913832 on brown eye color, the protective effect of the SNP rs12913832 on melanoma risk was most significant among the carriers of CC genotype of rs7173419 (OR 0.64; 95% CI 0.53–0.77; <em>P</em> = 3.6 × 10<sup>−6</sup>), compared with the carriers of CT genotype (OR 0.76; 95% CI 0.61–0.94; <em>P</em> = 0.01) and TT genotype (OR 1.05; 95% CI 0.63–1.73; <em>P</em> = 0.86). The interaction was borderline significant (<em>P</em>-value for interaction = 0.07, Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB4" rid-ob="ob-DDT142TB4" co-legend-rid=""><span>4</span></a>). Besides, the SNP rs12203592 associated with increased number of NMSCs was also associated with increased risk of melanoma (T allele; OR 1.18; 95% CI 1.02–1.34; <em>P</em> = 0.02, Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB7/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB7" rid-ob="ob-DDT142TB7" co-legend-rid=""><span>7</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT142TB7"><h3>Table&nbsp;7.</h3><!--caption a7--><div class="caption"><p id="__p60">Associations of the pigmentation SNPs identified in this study with melanoma risk<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992326402608" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Reference allele</th><th rowspan="1" colspan="1" align="left">Test allele</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>b</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs975739</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.93 (0.83–1.04)</td><td rowspan="1" colspan="1">0.25</td></tr><tr><td rowspan="1" colspan="1">rs12421680</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.14 (1.01–1.27)</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">rs3002288</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.02 (0.92–1.14)</td><td rowspan="1" colspan="1">0.63</td></tr><tr><td rowspan="1" colspan="1">rs12520016</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.75 (0.59–0.96)</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">rs7173419</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.88 (0.77–1.00)</td><td rowspan="1" colspan="1">0.06</td></tr><tr><td rowspan="1" colspan="1">rs12203592</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1.16 (1.01–1.34)</td><td rowspan="1" colspan="1">0.03</td></tr></tbody></table></div><div id="largeobj_idm139992326402608" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3690971/table/DDT142TB7/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139992326368416"><p id="__p61" class="p p-first-last"><sup>a</sup>The SNPs rs12202284 and rs8015138 were not available in the melanoma GWAS data.</p></div><div id="idm139992326367344"><p id="__p62" class="p p-first-last"><sup>b</sup>Adjusted for age, gender and the top two principal components.</p></div></div></div><p id="__p63" class="p p-last">In Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB6/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB6" rid-ob="ob-DDT142TB6" co-legend-rid=""><span>6</span></a>, we summarize the pigmentation SNPs identified in the previous GWASs (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900050">4</a>–<a href="#DDT142C6" rid="DDT142C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900044">6</a>) and the present study. We presented the associations of them with natural hair color, eye colors, tanning ability, the number of sunburns, mole counts, as well as the risk of melanoma, basal cell carcinoma (BCC), squamous cell carcinoma (SCC) and the number of NMSCs in our study population. In addition, we found that the associations of the SNPs newly identified in the present study remained similar after further adjusting for the pigmentation SNPs identified in the previous studies (rs16891982, rs12203592, rs1408799, rs35264875, rs3829241, rs1126809, rs1042602, rs12821256, rs12896399, rs12913832, rs1805007, rs1805008, rs4911414 and rs1015362, as listed in Table&nbsp;<a href="/pmc/articles/PMC3690971/table/DDT142TB6/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT142TB6" rid-ob="ob-DDT142TB6" co-legend-rid=""><span>6</span></a>), suggesting they are independent from the previously known pigmentation SNPs (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). We further screened the pair-wise interactions between these SNPs (including the ASIP haplotypes composed of rs4911414 and rs1015362) on pigmentation traits. None of them revealed significant interactions with each other on any pigmentation traits after the Bonferroni correction (<em>P</em> = 0.05/78 = 6.4 × 10<sup>−4</sup>).</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p64" class="p p-first">Previously, we reported the GWASs of hair color and tanning response among 2287 women with 528 173 SNPs genotyped by the Illumina HumanHap550 (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900038">4</a>,<a href="#DDT142C14" rid="DDT142C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900051">14</a>), in which we identified novel loci associated with hair color and tanning response in the gene regions of <em>IRF4</em> and <em>SLC24A4</em> in addition to the known ones, MC1R, OCA2 and SLC45A2 (<a href="#DDT142C4" rid="DDT142C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900020">4</a>). In the present study, we enlarged the sample size to 10 183 men and women and analyzed 8 221 074 autosomal SNPs imputed from 1000 Genomes Projects on hair color, tanning ability, the number of sunburns and the number of NMSCs. We additionally conducted a GWAS on eye color among 3871 males with 2 543 887 HapMap imputed autosomal SNPs. Using a multistage study design, we identified several novel genetic variants associated with pigmentary phenotypes and the risk of skin cancer. A novel region in the <em>EDNRB</em> gene was most significantly associated with natural hair color. The <em>EDNRB</em> gene encodes a G protein-coupled receptor that activates a phosphatidylinositol-calcium second messenger system. Studies suggest that mutations in <em>EDNRB</em> can cause Waardenburg syndrome, which is characterized by pigmentation abnormalities, including depigmented patches of the skin and hair, vivid blue eyes or heterochromia irides and sensorineural hearing loss (<a href="#DDT142C15" rid="DDT142C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900037">15</a>). The requirement for EDNRB signaling pathway during melanocyte development has been reported (<a href="#DDT142C16" rid="DDT142C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900019">16</a>,<a href="#DDT142C17" rid="DDT142C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900016">17</a>). However, the previous studies on genetic variants were limited to the nonsynonymous mutations (<a href="#DDT142C15" rid="DDT142C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900017">15</a>). In the present study, we identified a region downstream of this gene associated with natural human hair color, and the SNP rs975739 revealed the strongest association (<em>P</em> = 2.4 × 10<sup>−14</sup>). Our finding was consistent with a previous study presented at the 2010 meeting of the American Society of Human Genetics (<a href="#DDT142C18" rid="DDT142C18" class=" bibr popnode">18</a>), reporting an association between another <em>EDNRB</em> SNP rs1668619 (in complete LD with rs975739) and hair color with a <em>P</em>-value of 1 × 10<sup>−8</sup>. The SNP rs1668619 reached a <em>P</em>-value of 6.8 × 10<sup>−9</sup> in our hair color GWAS. We did not find significant association between variants in this region and melanoma risk, and the conclusion of the association between EDNRB and melanoma risk was still controversial (<a href="#DDT142C19" rid="DDT142C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900012">19</a>,<a href="#DDT142C20" rid="DDT142C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900030">20</a>). No association was identified for tanning ability or the number of sunburns either, suggesting the involvement of this gene in the basal pigmentation but not the response to sun exposure.</p><p id="__p65">In consistent with our previous findings on the association of the <em>IRF4</em> SNP rs12203592 with increased risk of all three types of skin cancer (<a href="#DDT142C21" rid="DDT142C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900042">21</a>), we further found that this SNP was associated with an increased number of NMSCs. We also confirmed the association of this SNP with melanoma risk among an independent population from our previous report in the melanoma GWAS of the MD Anderson Cancer Center. IRF4 regulates gene expression in response to interferon and other cytokines (<a href="#DDT142C22" rid="DDT142C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900048">22</a>) and is involved in tumor immunity (<a href="#DDT142C23" rid="DDT142C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900011">23</a>), which is plausible to play an important role in cancer development. The other SNP rs12202284 associated with the number of NMSCs was located between <em>IRF4</em> and <em>EXOC2</em>. We previously reported another SNP rs12210050 in this region associated with an increased risk of both BCC and SCC (<a href="#DDT142C24" rid="DDT142C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_801463661">24</a>). These two SNPs (rs12202284 and rs12210050) were in modest LD (<em>r</em><sup>2</sup> = 0.77) and revealed similar effects on the number of NMSCs in the present study (rs12202284, A allele, beta = 0.15, SE = 0.03; <em>P</em> = 5.0 × 10<sup>−8</sup>; rs12210050, T allele, beta = 0.19, SE = 0.04, <em>P</em> = 8.5 × 10<sup>−8</sup>), which may represent the same causal signal. Neither of the two SNPs was in LD with the SNP rs12203592 in this study (<em>r</em><sup>2</sup> = 0.10 and 0.12, respectively).</p><p id="__p66">An <em>NTM</em> polymorphism was associated with the number of sunburns in this study. NTM is a member of the IgLON family of immunoglobulin domain-containing glycosylphosphatidylinositol-anchored cell adhesion molecules (<a href="#DDT142C25" rid="DDT142C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900005">25</a>), which is involved in a variety of specific cell–cell interactions in the developing nervous system (<a href="#DDT142C26" rid="DDT142C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900064">26</a>). As for the skin, NTM was involved in the aging process as a marker of cell adhesion, and a down-regulation of NTM was detected during the replicative senescence of human dermal fibroblasts (<a href="#DDT142C21" rid="DDT142C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900008">21</a>). Although the studies of NTM have been mostly limited to the neural system, our finding of the association between an <em>NTM</em> polymorphism and the number of sunburns, along with the evidence from a previous study on skin (<a href="#DDT142C21" rid="DDT142C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900029">21</a>), suggested a promising role in cell adhesion that NTM might play in the skin besides the neural system. The lack of association between the <em>NTM</em> SNP with natural hair color and tanning ability further suggested a pigmentation-independent mechanism underlying sunburn reaction.</p><p id="__p67">The presence of melanin pigment within the iris is responsible for the visual impression of human eye colors (<a href="#DDT142C12" rid="DDT142C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900025">12</a>). The function of the two genes identified in eye color GWASs (<em>VASH2</em> and <em>POLS</em>) remained unknown in terms of pigmentation. VASH2 was known to play a role in the regulation of angiogenesis (<a href="#DDT142C27" rid="DDT142C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900018">27</a>), and POLS was involved in DNA synthesis (<a href="#DDT142C28" rid="DDT142C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900062">28</a>). Further studies on the two genes are required to validate the involvement in pigmentation process, especially in the eyes. The association of the <em>POLS</em> polymorphism and melanoma risk is upon further replication, and functional studies are warranted to investigate the biological mechanisms.</p><p id="__p68">The <em>GNG2</em> gene was identified as a novel locus conferring susceptibility to the number of NMSCs in this study. <em>GNG2</em> encodes one of the gamma subunits of a G-protein, which is involved in signaling mechanisms across membranes and has been known to play important roles in the development and progression of various cancers. The G-protein βγ subunits play an important role in melanoma cell migration by crosstalk with Epac in Ca<sup>2+</sup> signaling (<a href="#DDT142C29" rid="DDT142C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900068">29</a>). However, there was still lack of specific studies of GNG2 on NMSCs.</p><p id="__p69">One limitation of this study is that we used self-reported pigmentation information. However, information on hair color and eye color is relatively objective, and a high reliability of self-reported tanning ability and sunburn history has been demonstrated in the previous studies (<a href="#DDT142C30" rid="DDT142C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900022">30</a>,<a href="#DDT142C31" rid="DDT142C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900060">31</a>). The high education level and interest in the health of cohort members further allows high quality and valid information to be collected on self-administered forms, and the self-reported pigmentation phenotypes in our cohorts significantly predicted the risk of skin cancers (including melanoma, SCC and BCC) in the previous studies (<a href="#DDT142C7" rid="DDT142C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900003">7</a>,<a href="#DDT142C32" rid="DDT142C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352269226">32</a>,<a href="#DDT142C33" rid="DDT142C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900047">33</a>). Another limitation on the pigmentation information was that we had categorized information only for some continuous pigmentation traits, such as hair color, which may limit our power to reveal more new pigmentation genes.</p><p id="__p70" class="p p-last">In summary, we newly identified several genes and loci associated with human pigmentation traits and skin cancer risk by conducting a series of GWASs on hair color, eye colors, tanning ability, the number of sunburns and the number of NMSCs among individuals of European ancestry. The identification of these novel genetic determinants of pigmentary phenotypes and skin cancer may help further understand the complex mechanisms of human pigmentation and the genetic basis of skin cancer. The different variants identified for different traits may explain the highly variable pigmentation characteristics in European Americans. External replication is needed to extend our finding to other populations.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Study population</h3><div id="s4a1" class="sec sec-first-last"><p></p><h4 id="s4a1title" class="inline">Pigmentation GWASs </h4><p id="__p71" class="p p-first">In the discovery stage of the GWASs on hair color, tanning ability, the number of sunburns and the number of NMSCs, we combined data from five case–control studies nested within the Nurses' Health Study (NHS) and the Health Professionals Follow-up Study (HPFS): a postmenopausal invasive breast cancer case–control study nested within the NHS (BC_NHS, <em>n</em> = 2287), a type 2 diabetes case–control study nested within the NHS (T2D_NHS, <em>n</em> = 3116), a type 2 diabetes case–control study nested within the HPFS (T2D_HPFS, <em>n</em> = 2487), a coronary heart disease case–control study nested within the NHS (CHD_NHS, <em>n</em> = 1146) and a coronary heart disease case–control study nested within the HPFS (CHD_HPFS, <em>n</em> = 1147). We excluded the breast cancer cases in BC_NHS for the GWAS on the number of NMSCs. In the replication stage, a fast-track replication was conducted among 1782 healthy controls in the skin cancer case–control study nested within the NHS and HPFS (skin cancer). Two pigmentation SNPs successfully replicated in the skin cancer set were selected for a second replication among 2722 individuals from a renal function study nested within the NHS (RF_NHS). Information on eye color was available in the HPFS only, so we combined data from three GWAS sets within the HPFS, T2D_HPFS, CHD_HPFS and a kidney stone case–control study nested within the HPFS (KS_HPFS, <em>n</em> = 515), with a total of 3871 men in the discovery stage of eye color GWASs. Two novel SNPs with promising association were selected for replication among 2496 participants from the prostate cancer case–control study nested within the HPFS (PrCa_HPFS). Overlapped samples among the component data sets were excluded. Detailed descriptions of the study population were published previously (<a href="#DDT142C34" rid="DDT142C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900033">34</a>–<a href="#DDT142C38" rid="DDT142C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900054">38</a>). The study protocol was approved by the Institutional Review Board of Brigham and Women's Hospital and the Harvard School of Public Health.</p><p id="__p72" class="p p-last">Information on pigmentation traits was collected from prospective questionnaires in both the NHS and HPFS using similar wording, except for natural eye color, which was collected in the HPFS questionnaires only. We regressed ordinal coding for natural hair color (1 = blonde, 2 = light brown, 3 = dark brown, 4 = black; excluding red hair color) and tanning ability (1 = practically none, 2 = light tan, 3 = average tan, 4 = deep tan in the NHS; 1 = pain: burn/peel, 2 = burn then tan, 3 = tan without burn in the HPFS). For the hair color phenotype, we excluded the individuals with red hair because red hair color has been known as a distinct trait from other hair colors and is predominantly determined by the <em>MC1R</em> gene. A total of 289 (3.9%) individuals with red hair were excluded in the hair color GWAS (77 in BC_NHS, 41 in CHD_NHS, 103 in T2D_NHS, 19 in CHD_HPFS and 49 in T2D_HPFS). The number of lifetime severe sunburns was coded as a continuous variable, summarizing those on the face, limbs and back. Different eye colors have been hypothesized to be distinct phenotypes and we treated three categories of eye colors, blue, intermediate (including green) and brown eye colors, as co-dominant outcomes, comparing each with the other two, following a previous GWAS on eye color (<a href="#DDT142C5" rid="DDT142C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900024">5</a>).</p></div></div><div id="s4b" class="sec"><h3 id="s4btitle">NMSC case ascertainment</h3><p id="__p73" class="p p-first">For all types of skin cancers, participants reported new cases biennially. With their permission, medical records were obtained and reviewed by physicians to confirm their self-reported diagnosis. Medical records were not obtained for self-reported cases of BCC, but the validity of BCC self-reports was &gt;90% in validation studies in our cohorts in early years. Information on the cumulative number of NMSCs was collected through 2004 in the NHS and 2008 in the HPFS (<a href="#DDT142C39" rid="DDT142C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900039">39</a>,<a href="#DDT142C40" rid="DDT142C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900035">40</a>). The question asked was ‘How many squamous or basal cell carcinoma lesions have you ever had removed by surgery, cryotherapy or other means? (Include only new primary cancers. Exclude melanoma and benign lesions like moles or actinic keratoses.)’. The possible choices were never, 1, 2–4, 5–10 and 11+. To minimize the misclassification of the outcome, the eligible participants in this study were Caucasians who reported at least one pathologically confirmed diagnosis of SCC or self-reported BCC in the cohort follow-up from 1984 up to 2004. We coded the number of NMSCs as a continuous variable by using the median number in each category.</p><div id="s4b1" class="sec sec-last"><p></p><h4 id="s4b1title" class="inline">Melanoma risk association study </h4><p id="__p74" class="p p-first-last">We evaluated the association between the pigmentary SNPs identified in this study with melanoma risk using the melanoma GWAS data from MD Anderson Cancer Center with 1804 melanoma cases and 1026 controls. Details of the study population have previously been described (<a href="#DDT142C41" rid="DDT142C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900043">41</a>). Briefly, in this hospital-based case–control study, 1804 non-Hispanic patients with newly diagnosed, histologically confirmed, untreated melanoma were consecutively recruited at the University of Texas MD Anderson Cancer Center in Houston, TX, between April 1994 and April 2008. An additional 1026 cancer-free controls matched to the cases on age (±5 years), sex and ethnicity were recruited during the same period from among visitors to MD Anderson Cancer Center who were accompanying patients to outpatient clinics but neither seeking medical care nor biologically related to the patients included in this study. After informed consent was obtained, each participant was asked to complete a questionnaire to provide information about their demographic and known risk factors for CM. A one-time blood sample (30 ml) was drawn from each study participant. All subjects included in this analysis were US non-Hispanic European descendants. The research protocol was approved by the Institutional Review Board at MD Anderson Cancer Center.</p></div></div><div id="s4c" class="sec"><h3 id="s4ctitle">SNP selection for replication</h3><p id="__p75" class="p p-first-last">Four novel regions revealed the strongest associations with pigmentation traits in the discovery stage of this study besides the known pigmentation loci and were selected for replication: one region on chromosome 13 with hair color (smallest <em>P</em> = 5.4 × 10<sup>−9</sup>), one region on chromosome 11 with the number of sunburns (smallest <em>P</em> = 5.3 × 10<sup>−5</sup>), one region on chromosome 1 with brown eye color (smallest <em>P</em> = 4.2 × 10<sup>−6</sup>) and one region on chromosome 5 with intermediate (including green) eye color (smallest <em>P</em> = 3.1 × 10<sup>−6</sup>). For the region on chromosome 13 associated with natural hair color GWAS, we found 117 SNPs with <em>P</em> &lt; 1 × 10<sup>−7</sup> in two LD blocks in this region (chr13:77 259 946–77 370 845). We forced in the top SNP in the regional Manhattan plot (rs975739) and selected other two tag-SNPs (rs1146927 and rs58188699) for replication using the Tagger program in Haploview 4.2 (<em>r</em><sup>2</sup> &gt; 0.8) (<a href="#DDT142C42" rid="DDT142C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900010">42</a>,<a href="#DDT142C43" rid="DDT142C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900058">43</a>). A proxy SNP was selected for each of them to ensure the validity of genotyping. For each region associated with the number of sunburns, brown eye color and intermediate eye color, we selected the SNP with the strongest association as the representative. Besides, two novel regions revealed promising associations with the number of NMSCs in the discovery stage: one on chromosome 6 and the other on chromosome 14. We selected the five SNPs with the strongest associations as the representatives (two on chromosome 6 and three on chromosome 14). Additionally, the well-known pigmentation SNP rs12913832 in HECT and RLD domain containing E3 ubiquitin protein ligase 2 (<em>HERC2</em>) revealed the strongest association with brown eye color in our study and also a significant interaction with the SNP rs7173419 in <em>OCA2</em>. To validate this interaction, we further selected rs12913832 and rs7173419 for replication.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Genotyping, imputation and quality control</h3><p id="__p76" class="p p-first">We previously used either the Illumina/Affymetrix SNP chip for the genotyping of the component GWAS sets (<a href="#DDT142C34" rid="DDT142C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900049">34</a>–<a href="#DDT142C36" rid="DDT142C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900053">36</a>). We used Illumina HumanHap550 array to perform genotyping in the BC_NHS (<a href="#DDT142C34" rid="DDT142C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900015">34</a>), used Illumina 610Q array to perform genotyping in the KS_HPFS and used Affymetrix 6.0 array in the other four GWAS sets, including T2D_NHS, T2D_HPFS (<a href="#DDT142C35" rid="DDT142C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900061">35</a>), as well as CHD_NHS, CHD_HPFS (<a href="#DDT142C36" rid="DDT142C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900004">36</a>). For five of the GWAS sets (BC_NHS, T2D_NHS, T2D_HPFS, CHD_NHS and CHD_HPFS), we used the MACH program (<a href="#DDT142C44" rid="DDT142C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900002">44</a>) to impute 8 221 074 autosomal SNPs based on haplotypes from the 1000 Genomes Project (using the August 2009 release of the Sanger Institute SNP calls based on the pilot 1 project). For the KS_HPFS set, we used the same program to impute 2 543 887 autosomal SNPs based on haplotypes from the HapMap database phase II data build 35 (CEU) (<a href="#DDT142C45" rid="DDT142C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900007">45</a>). The HapMap imputed data of T2D_HPFS and CHD_HPFS were used for eye color GWASs to be combined with KS_HPFS. Samples from the six GWASs were imputed separately. We observed high imputation quality in each cohort's imputation. In particular, the numbers of SNPs with MACH <em>r</em><sup>2</sup> &gt; 0.4 were 7 438 552 (90.5%) for NHS_BC, 7 399 703 (90.0%) for NHS_T2D, 7 437 076 (90.5%) for NHS_CHD, 7 370 611 (89.7%) for HPFS_T2D and 7 417 449 (90.2%) for HPFS_CHD. Most of the imputed SNPs were common [for all five cohorts, &gt;98% SNPs have a minor allele frequency (MAF) &gt; 0.01 and &gt;79% SNPs have an MAF &gt; 0.05]. SNPs with imputation <em>r</em><sup>2</sup> &gt; 0.4 and an MAF &gt; 0.01 in each study were included in meta-analysis. Finally, 7 588 169 SNPs were analyzed for natural hair color, tanning ability, the number of sunburns and the number of NMSCs, and 2 469 762 SNPs were analyzed for eye colors.</p><p id="__p77">The promising SNPs selected from the discovery stage were genotyped in the replication sets using TaqMan/BioTrove assays at the Dana Farber/Harvard Cancer Center Polymorphism Detection Core. Laboratory personnel were blinded to the case–control status, and blinded quality control samples were inserted to validate genotyping procedures; concordance for the blinded samples was 100%. Primers, probes and conditions for genotyping assays are available upon request. All these SNPs were successfully genotyped with call rate &gt;95%, except for the SNP rs1146926 (a proxy SNP of rs1146927), which failed in genotyping.</p><p id="__p78" class="p p-last">As for the melanoma risk association study, the samples were previously genotyped by the Illumina Omni 1M Quad V1-0_B SNP chip (<a href="#DDT142C41" rid="DDT142C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900031">41</a>). An imputation was conducted based on the genotyped SNPs and haplotype information in the HapMap phase II data build 35 (CEU) (<a href="#DDT142C45" rid="DDT142C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900073">45</a>). For the SNPs used in this study, the SNPs rs12913832, rs12421680, rs12520016 and rs12203592 were directly genotyped, and the SNPs rs975739, rs3002288 and rs7173419 were imputed with imputation <em>r</em><sup>2</sup> of 1.00, 0.92 and 0.98, respectively.</p></div><div id="s4e" class="sec sec-last"><h3 id="s4etitle">Statistical analysis</h3><p id="__p79" class="p p-first-last">We used linear regression to test the association between minor allele counts and phenotypes of hair color, tanning ability, the number of sunburns and the number of NMSCs, and used logistic regression to test association separately for brown, intermediate and blue eye colors as co-dominant outcomes (comparing one with the other two). The imputed dosage data were used for the analysis. The model was adjusted for the first four principal components. Age was also adjusted for the GWASs of the number of sunburns and the number of NMSCs, but not for the other pigmentation traits, which were not likely to be confounded by age. We used ProbABEL for the GWAS analysis in this study. Associations in each component GWAS set were combined in an inverse-variance-weighted meta-analysis using the METAL software. We additionally adjusted for gender in the skin cancer replication set, which was mixed of males and females. The same meta-analysis was conducted to combine the component replication sets and to combine the discovery set and the replication set. We used logistic regression for the melanoma risk association study and adjusted for age, gender and the first two principal components. These principal components were calculated for all individuals on the basis of approximately 10 000 unlinked markers using the EIGENSTRAT software (<a href="#DDT142C46" rid="DDT142C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900036">46</a>). We used the algorithm developed by Kraft <em>et al.</em> (<a href="#DDT142C47" rid="DDT142C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900009">47</a>) to infer the haplotypes from the unphased genotypic data. The description of this algorithm was previously published, and this approach has been proved by simulation studies as well as subsequent applications in several other studies (<a href="#DDT142C47" rid="DDT142C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900052">47</a>–<a href="#DDT142C49" rid="DDT142C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320900040">49</a>).</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">WEB RESOURCES</h2><p id="__p80" class="p p-first">The URLs for data presented herein are as follows:</p><p id="__p81">1000 Genomes Project, <a href="http://www.1000genomes.org">http://www.1000genomes.org</a></p><p id="__p82">International HapMap Project, <a href="http://www.hapmap.org">http://www.hapmap.org</a></p><p id="__p83" class="p p-last">METAL software, <a href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</a></p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">SUPPLEMENTARY MATERIAL</h2><p id="__p84" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt142/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s7title">FUNDING</h2><p id="__p85" class="p p-first-last">This work is supported by National Institutes of Health grants <span>CA122838, CA87969, CA055075, CA49449, CA100264 and CA093459</span>.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm139992327111696"><a href="/pmc/articles/PMC3690971/bin/supp_22_14_2948__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>



</div></div><div id="idm139992327104000" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139992327104000title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p86">We thank Dr David Hunter, Dr Frank B. Hu, Dr Eric B. Rimm and Dr Gary Curhan for their work on the component GWAS sets. We thank Dr Mousheng Xu and Constance Chen for programming support, and thank Pati Soule and Dr Hardeep Ranu of the Dana Farber/Harvard Cancer Center High-Throughput Polymorphism Detection Core for sample handling and genotyping of the NHS and HPFS samples. We are grateful to Merck Research Laboratories for the funding of the GWAS of coronary heart disease. We thank Dr Jinyan Huang for improving the figure quality. We are also indebted to the participants in all of these studies. We thank the cancer registries in the following states for their help: Alabama, Arizona, Arkansas, California, Colorado, Connecticut, Delaware, Florida, Georgia, Idaho, Illinois, Indiana, Iowa, Kentucky, Louisiana, Maine, Maryland, Massachusetts, Michigan, Nebraska, New Hampshire, New Jersey, New York, North Carolina, North Dakota, Ohio, Oklahoma, Oregon, Pennsylvania, Rhode Island, South Carolina, Tennessee, Texas, Virginia, Washington and Wyoming.</p><p id="__p87"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm139992327100784" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139992327100784title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDT142C1">1. <span class="element-citation">Clark P., Stark A.E., Walsh R.J., Jardine R., Martin N.G. A twin study of skin reflectance. <span><span class="ref-journal">Ann. Hum. Biol. </span>1981;<span class="ref-vol">8</span>:529–541.</span>  <a href="http://dx.doi.org/10.1080/03014468100005371" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1080/03014468100005371</a>. [<a href="/pubmed/7199888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Biol.&amp;title=A+twin+study+of+skin+reflectance&amp;author=P.+Clark&amp;author=A.E.+Stark&amp;author=R.J.+Walsh&amp;author=R.+Jardine&amp;author=N.G.+Martin&amp;volume=8&amp;publication_year=1981&amp;pages=529-541&amp;pmid=7199888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C2">2. <span class="element-citation">Frisancho A.R., Wainwright R., Way A. Heritability and components of phenotypic expression in skin reflectance of Mestizos from the Peruvian lowlands. <span><span class="ref-journal">Am. J. Phys. Anthropol. </span>1981;<span class="ref-vol">55</span>:203–208.</span>  <a href="http://dx.doi.org/10.1002/ajpa.1330550207" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1002/ajpa.1330550207</a>. [<a href="/pubmed/7258341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Phys.+Anthropol.&amp;title=Heritability+and+components+of+phenotypic+expression+in+skin+reflectance+of+Mestizos+from+the+Peruvian+lowlands&amp;author=A.R.+Frisancho&amp;author=R.+Wainwright&amp;author=A.+Way&amp;volume=55&amp;publication_year=1981&amp;pages=203-208&amp;pmid=7258341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C3">3. <span class="element-citation">Gerstenblith M.R., Shi J., Landi M.T. Genome-wide association studies of pigmentation and skin cancer: a review and meta-analysis. <span><span class="ref-journal">Pigment Cell Melanoma Res. </span>2010;<span class="ref-vol">23</span>:587–606.</span>  <a href="http://dx.doi.org/10.1111/j.1755-148X.2010.00730.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.1755-148X.2010.00730.x</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3179913/">PMC free article</a>]</span> [<a href="/pubmed/20546537" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pigment+Cell+Melanoma+Res.&amp;title=Genome-wide+association+studies+of+pigmentation+and+skin+cancer:+a+review+and+meta-analysis&amp;author=M.R.+Gerstenblith&amp;author=J.+Shi&amp;author=M.T.+Landi&amp;volume=23&amp;publication_year=2010&amp;pages=587-606&amp;pmid=20546537&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C4">4. <span class="element-citation">Han J., Kraft P., Nan H., Guo Q., Chen C., Qureshi A., Hankinson S.E., Hu F.B., Duffy D.L., Zhao Z.Z., et al.  A genome-wide association study identifies novel alleles associated with hair color and skin pigmentation. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000074.</span>  <a href="http://dx.doi.org/10.1371/journal.pgen.1000074" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1371/journal.pgen.1000074</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2367449/">PMC free article</a>]</span> [<a href="/pubmed/18483556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+genome-wide+association+study+identifies+novel+alleles+associated+with+hair+color+and+skin+pigmentation&amp;author=J.+Han&amp;author=P.+Kraft&amp;author=H.+Nan&amp;author=Q.+Guo&amp;author=C.+Chen&amp;volume=4&amp;publication_year=2008&amp;pages=e1000074&amp;pmid=18483556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C5">5. <span class="element-citation">Sulem P., Gudbjartsson D.F., Stacey S.N., Helgason A., Rafnar T., Magnusson K.P., Manolescu A., Karason A., Palsson A., Thorleifsson G., et al.  Genetic determinants of hair, eye and skin pigmentation in Europeans. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1443–1452.</span>  <a href="http://dx.doi.org/10.1038/ng.2007.13" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.2007.13</a>. [<a href="/pubmed/17952075" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+determinants+of+hair,+eye+and+skin+pigmentation+in+Europeans&amp;author=P.+Sulem&amp;author=D.F.+Gudbjartsson&amp;author=S.N.+Stacey&amp;author=A.+Helgason&amp;author=T.+Rafnar&amp;volume=39&amp;publication_year=2007&amp;pages=1443-1452&amp;pmid=17952075&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C6">6. <span class="element-citation">Sulem P., Gudbjartsson D.F., Stacey S.N., Helgason A., Rafnar T., Jakobsdottir M., Steinberg S., Gudjonsson S.A., Palsson A., Thorleifsson G., et al.  Two newly identified genetic determinants of pigmentation in Europeans. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:835–837.</span>  <a href="http://dx.doi.org/10.1038/ng.160" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.160</a>. [<a href="/pubmed/18488028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Two+newly+identified+genetic+determinants+of+pigmentation+in+Europeans&amp;author=P.+Sulem&amp;author=D.F.+Gudbjartsson&amp;author=S.N.+Stacey&amp;author=A.+Helgason&amp;author=T.+Rafnar&amp;volume=40&amp;publication_year=2008&amp;pages=835-837&amp;pmid=18488028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C7">7. <span class="element-citation">Han J., Colditz G.A., Hunter D.J. Risk factors for skin cancers: a nested case-control study within the Nurses' Health Study. <span><span class="ref-journal">Int. J. Epidemiol. </span>2006;<span class="ref-vol">35</span>:1514–1521.</span>  <a href="http://dx.doi.org/10.1093/ije/dyl197" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/ije/dyl197</a>. [<a href="/pubmed/16943234" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol.&amp;title=Risk+factors+for+skin+cancers:+a+nested+case-control+study+within+the+Nurses'+Health+Study&amp;author=J.+Han&amp;author=G.A.+Colditz&amp;author=D.J.+Hunter&amp;volume=35&amp;publication_year=2006&amp;pages=1514-1521&amp;pmid=16943234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C8">8. <span class="element-citation">Stacey S.N., Gudbjartsson D.F., Sulem P., Bergthorsson J.T., Kumar R., Thorleifsson G., Sigurdsson A., Jakobsdottir M., Sigurgeirsson B., Benediktsdottir K.R., et al.  Common variants on 1p36 and 1q42 are associated with cutaneous basal cell carcinoma but not with melanoma or pigmentation traits. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1313–1318.</span>  <a href="http://dx.doi.org/10.1038/ng.234" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.234</a>. [<a href="/pubmed/18849993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+on+1p36+and+1q42+are+associated+with+cutaneous+basal+cell+carcinoma+but+not+with+melanoma+or+pigmentation+traits&amp;author=S.N.+Stacey&amp;author=D.F.+Gudbjartsson&amp;author=P.+Sulem&amp;author=J.T.+Bergthorsson&amp;author=R.+Kumar&amp;volume=40&amp;publication_year=2008&amp;pages=1313-1318&amp;pmid=18849993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C9">9. <span class="element-citation">Stacey S.N., Sulem P., Masson G., Gudjonsson S.A., Thorleifsson G., Jakobsdottir M., Sigurdsson A., Gudbjartsson D.F., Sigurgeirsson B., Benediktsdottir K.R., et al.  New common variants affecting susceptibility to basal cell carcinoma. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:909–914.</span>  <a href="http://dx.doi.org/10.1038/ng.412" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.412</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2973331/">PMC free article</a>]</span> [<a href="/pubmed/19578363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=New+common+variants+affecting+susceptibility+to+basal+cell+carcinoma&amp;author=S.N.+Stacey&amp;author=P.+Sulem&amp;author=G.+Masson&amp;author=S.A.+Gudjonsson&amp;author=G.+Thorleifsson&amp;volume=41&amp;publication_year=2009&amp;pages=909-914&amp;pmid=19578363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C10">10. <span class="element-citation">Nan H., Xu M., Kraft P., Qureshi A.A., Chen C., Guo Q., Hu F.B., Curhan G., Amos C.I., Wang L.E., et al.  Genome-wide association study identifies novel alleles associated with risk of cutaneous basal cell carcinoma and squamous cell carcinoma. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:3718–3724.</span>  <a href="http://dx.doi.org/10.1093/hmg/ddr287" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/hmg/ddr287</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3159556/">PMC free article</a>]</span> [<a href="/pubmed/21700618" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+study+identifies+novel+alleles+associated+with+risk+of+cutaneous+basal+cell+carcinoma+and+squamous+cell+carcinoma&amp;author=H.+Nan&amp;author=M.+Xu&amp;author=P.+Kraft&amp;author=A.A.+Qureshi&amp;author=C.+Chen&amp;volume=20&amp;publication_year=2011&amp;pages=3718-3724&amp;pmid=21700618&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C11">11. <span class="element-citation">Huang R.S., Duan S., Bleibel W.K., Kistner E.O., Zhang W., Clark T.A., Chen T.X., Schweitzer A.C., Blume J.E., Cox N.J., et al.  A genome-wide approach to identify genetic variants that contribute to etoposide-induced cytotoxicity. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2007;<span class="ref-vol">104</span>:9758–9763.</span>  <a href="http://dx.doi.org/10.1073/pnas.0703736104" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1073/pnas.0703736104</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1887589/">PMC free article</a>]</span> [<a href="/pubmed/17537913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=A+genome-wide+approach+to+identify+genetic+variants+that+contribute+to+etoposide-induced+cytotoxicity&amp;author=R.S.+Huang&amp;author=S.+Duan&amp;author=W.K.+Bleibel&amp;author=E.O.+Kistner&amp;author=W.+Zhang&amp;volume=104&amp;publication_year=2007&amp;pages=9758-9763&amp;pmid=17537913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C12">12. <span class="element-citation">Sturm R.A., Larsson M. Genetics of human iris colour and patterns. <span><span class="ref-journal">Pigment Cell Melanoma Res. </span>2009;<span class="ref-vol">22</span>:544–562.</span>  <a href="http://dx.doi.org/10.1111/j.1755-148X.2009.00606.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.1755-148X.2009.00606.x</a>. [<a href="/pubmed/19619260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pigment+Cell+Melanoma+Res.&amp;title=Genetics+of+human+iris+colour+and+patterns&amp;author=R.A.+Sturm&amp;author=M.+Larsson&amp;volume=22&amp;publication_year=2009&amp;pages=544-562&amp;pmid=19619260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C13">13. <span class="element-citation">Sturm R.A., Duffy D.L., Zhao Z.Z., Leite F.P., Stark M.S., Hayward N.K., Martin N.G., Montgomery G.W. A single SNP in an evolutionary conserved region within intron 86 of the HERC2 gene determines human blue-brown eye color. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:424–431.</span>  <a href="http://dx.doi.org/10.1016/j.ajhg.2007.11.005" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.ajhg.2007.11.005</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427173/">PMC free article</a>]</span> [<a href="/pubmed/18252222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+single+SNP+in+an+evolutionary+conserved+region+within+intron+86+of+the+HERC2+gene+determines+human+blue-brown+eye+color&amp;author=R.A.+Sturm&amp;author=D.L.+Duffy&amp;author=Z.Z.+Zhao&amp;author=F.P.+Leite&amp;author=M.S.+Stark&amp;volume=82&amp;publication_year=2008&amp;pages=424-431&amp;pmid=18252222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C14">14. <span class="element-citation">Nan H., Kraft P., Qureshi A.A., Guo Q., Chen C., Hankinson S.E., Hu F.B., Thomas G., Hoover R.N., Chanock S., et al.  Genome-wide association study of tanning phenotype in a population of European ancestry. <span><span class="ref-journal">J. Invest. Dermatol. </span>2009;<span class="ref-vol">129</span>:2250–2257.</span>  <a href="http://dx.doi.org/10.1038/jid.2009.62" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/jid.2009.62</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2758927/">PMC free article</a>]</span> [<a href="/pubmed/19340012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol.&amp;title=Genome-wide+association+study+of+tanning+phenotype+in+a+population+of+European+ancestry&amp;author=H.+Nan&amp;author=P.+Kraft&amp;author=A.A.+Qureshi&amp;author=Q.+Guo&amp;author=C.+Chen&amp;volume=129&amp;publication_year=2009&amp;pages=2250-2257&amp;pmid=19340012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C15">15. <span class="element-citation">Pingault V., Ente D., Dastot-Le Moal F., Goossens M., Marlin S., Bondurand N. Review and update of mutations causing Waardenburg syndrome. <span><span class="ref-journal">Hum. Mutat. </span><span class="ref-vol">31</span>:391–406.</span>  <a href="http://dx.doi.org/10.1002/humu.21211" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1002/humu.21211</a>. [<a href="/pubmed/20127975" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=Review+and+update+of+mutations+causing+Waardenburg+syndrome&amp;author=V.+Pingault&amp;author=D.+Ente&amp;author=F.+Dastot-Le+Moal&amp;author=M.+Goossens&amp;author=S.+Marlin&amp;volume=31&amp;pages=391-406&amp;pmid=20127975&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C16">16. <span class="element-citation">Hou L., Pavan W.J., Shin M.K., Arnheiter H. Cell-autonomous and cell non-autonomous signaling through endothelin receptor B during melanocyte development. <span><span class="ref-journal">Development. </span>2004;<span class="ref-vol">131</span>:3239–3247.</span>  <a href="http://dx.doi.org/10.1242/dev.01193" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1242/dev.01193</a>. [<a href="/pubmed/15201217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Cell-autonomous+and+cell+non-autonomous+signaling+through+endothelin+receptor+B+during+melanocyte+development&amp;author=L.+Hou&amp;author=W.J.+Pavan&amp;author=M.K.+Shin&amp;author=H.+Arnheiter&amp;volume=131&amp;publication_year=2004&amp;pages=3239-3247&amp;pmid=15201217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C17">17. <span class="element-citation">Stanchina L., Baral V., Robert F., Pingault V., Lemort N., Pachnis V., Goossens M., Bondurand N. Interactions between Sox10, Edn3 and Ednrb during enteric nervous system and melanocyte development. <span><span class="ref-journal">Dev. Biol. </span>2006;<span class="ref-vol">295</span>:232–249.</span>  <a href="http://dx.doi.org/10.1016/j.ydbio.2006.03.031" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.ydbio.2006.03.031</a>. [<a href="/pubmed/16650841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Biol.&amp;title=Interactions+between+Sox10,+Edn3+and+Ednrb+during+enteric+nervous+system+and+melanocyte+development&amp;author=L.+Stanchina&amp;author=V.+Baral&amp;author=F.+Robert&amp;author=V.+Pingault&amp;author=N.+Lemort&amp;volume=295&amp;publication_year=2006&amp;pages=232-249&amp;pmid=16650841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C18">18. <span class="element-citation">Hinds D.A., Macpherson J.M., Tung J., Naughton B., Kiefer A., Do C., Wojcicki A., Mountain J., Eriksson N. Genetics of infectious disease susceptibility and morphological traits in a participant driven cohort. Abstract presented at the 60th Annual Meeting of The American Society of Human Genetics; 2–6 November 2010; Washington, DC: 2010.  <a href="http://www.ashg.org/2010meeting/abstracts/fulltext/f22581.htm">http://www.ashg.org/2010meeting/abstracts/fulltext/f22581.htm</a> . <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Hinds+D.A.+Macpherson+J.M.+Tung+J.+Naughton+B.+Kiefer+A.+Do+C.+Wojcicki+A.+Mountain+J.+Eriksson+N.+Genetics+of+infectious+disease+susceptibility+and+morphological+traits+in+a+participant+driven+cohort+2010+Abstract+presented+at+the+60th+Annual+Meeting+of+The+American+Society+of+Human+Genetics+2–6+November+2010+Washington,+DC+http://www.ashg.org/2010meeting/abstracts/fulltext/f22581.htm+" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C19">19. <span class="element-citation">Fernandez L.P., Milne R.L., Pita G., Floristan U., Sendagorta E., Feito M., Aviles J.A., Martin-Gonzalez M., Lazaro P., Benitez J., et al.  Pigmentation-related genes and their implication in malignant melanoma susceptibility. <span><span class="ref-journal">Exp. Dermatol. </span>2009;<span class="ref-vol">18</span>:634–642.</span>  <a href="http://dx.doi.org/10.1111/j.1600-0625.2009.00846.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.1600-0625.2009.00846.x</a>. [<a href="/pubmed/19320733" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Dermatol.&amp;title=Pigmentation-related+genes+and+their+implication+in+malignant+melanoma+susceptibility&amp;author=L.P.+Fernandez&amp;author=R.L.+Milne&amp;author=G.+Pita&amp;author=U.+Floristan&amp;author=E.+Sendagorta&amp;volume=18&amp;publication_year=2009&amp;pages=634-642&amp;pmid=19320733&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C20">20. <span class="element-citation">Soufir N., Meziani R., Lacapere J.J., Bertrand G., Fumeron F., Bourillon A., Gerard B., Descamps V., Crickx B., Ollivaud L., et al.  Association between endothelin receptor B nonsynonymous variants and melanoma risk. <span><span class="ref-journal">J. Natl Cancer. Inst. </span>2005;<span class="ref-vol">97</span>:1297–1301.</span>  <a href="http://dx.doi.org/10.1093/jnci/dji253" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/jnci/dji253</a>. [<a href="/pubmed/16145050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer.+Inst.&amp;title=Association+between+endothelin+receptor+B+nonsynonymous+variants+and+melanoma+risk&amp;author=N.+Soufir&amp;author=R.+Meziani&amp;author=J.J.+Lacapere&amp;author=G.+Bertrand&amp;author=F.+Fumeron&amp;volume=97&amp;publication_year=2005&amp;pages=1297-1301&amp;pmid=16145050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C21">21. <span class="element-citation">Yoon I.K., Kim H.K., Kim Y.K., Song I.H., Kim W., Kim S., Baek S.H., Kim J.H., Kim J.R. Exploration of replicative senescence-associated genes in human dermal fibroblasts by cDNA microarray technology. <span><span class="ref-journal">Exp. Gerontol. </span>2004;<span class="ref-vol">39</span>:1369–1378.</span>  <a href="http://dx.doi.org/10.1016/j.exger.2004.07.002" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.exger.2004.07.002</a>. [<a href="/pubmed/15489060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Gerontol.&amp;title=Exploration+of+replicative+senescence-associated+genes+in+human+dermal+fibroblasts+by+cDNA+microarray+technology&amp;author=I.K.+Yoon&amp;author=H.K.+Kim&amp;author=Y.K.+Kim&amp;author=I.H.+Song&amp;author=W.+Kim&amp;volume=39&amp;publication_year=2004&amp;pages=1369-1378&amp;pmid=15489060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C22">22. <span class="element-citation">Lu R. Interferon regulatory factor 4 and 8 in B-cell development. <span><span class="ref-journal">Trends Immunol. </span>2008;<span class="ref-vol">29</span>:487–492.</span>  <a href="http://dx.doi.org/10.1016/j.it.2008.07.006" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.it.2008.07.006</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2823592/">PMC free article</a>]</span> [<a href="/pubmed/18775669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Immunol.&amp;title=Interferon+regulatory+factor+4+and+8+in+B-cell+development&amp;author=R.+Lu&amp;volume=29&amp;publication_year=2008&amp;pages=487-492&amp;pmid=18775669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C23">23. <span class="element-citation">Shaffer A.L., Emre N.C., Romesser P.B., Staudt L.M. IRF4: Immunity. Malignancy! Therapy? <span><span class="ref-journal">Clin. Cancer Res. </span>2009;<span class="ref-vol">15</span>:2954–2961.</span>  <a href="http://dx.doi.org/10.1158/1078-0432.CCR-08-1845" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1158/1078-0432.CCR-08-1845</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2790720/">PMC free article</a>]</span> [<a href="/pubmed/19383829" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=IRF4:+Immunity.+Malignancy!+Therapy?&amp;author=A.L.+Shaffer&amp;author=N.C.+Emre&amp;author=P.B.+Romesser&amp;author=L.M.+Staudt&amp;volume=15&amp;publication_year=2009&amp;pages=2954-2961&amp;pmid=19383829&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C24">24. <span class="element-citation">Struyk A.F., Canoll P.D., Wolfgang M.J., Rosen C.L., D'Eustachio P., Salzer J.L. Cloning of neurotrimin defines a new subfamily of differentially expressed neural cell adhesion molecules. <span><span class="ref-journal">J. Neurosci. </span>1995;<span class="ref-vol">15</span>:2141–2156.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6578143/">PMC free article</a>]</span> [<a href="/pubmed/7891157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci.&amp;title=Cloning+of+neurotrimin+defines+a+new+subfamily+of+differentially+expressed+neural+cell+adhesion+molecules&amp;author=A.F.+Struyk&amp;author=P.D.+Canoll&amp;author=M.J.+Wolfgang&amp;author=C.L.+Rosen&amp;author=P.+D'Eustachio&amp;volume=15&amp;publication_year=1995&amp;pages=2141-2156&amp;pmid=7891157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C25">25. <span class="element-citation">Sellar G.C., Watt K.P., Rabiasz G.J., Stronach E.A., Li L., Miller E.P., Massie C.E., Miller J., Contreras-Moreira B., Scott D., et al.  OPCML at 11q25 is epigenetically inactivated and has tumor-suppressor function in epithelial ovarian cancer. <span><span class="ref-journal">Nat. Genet. </span>2003;<span class="ref-vol">34</span>:337–343.</span>  <a href="http://dx.doi.org/10.1038/ng1183" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng1183</a>. [<a href="/pubmed/12819783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=OPCML+at+11q25+is+epigenetically+inactivated+and+has+tumor-suppressor+function+in+epithelial+ovarian+cancer&amp;author=G.C.+Sellar&amp;author=K.P.+Watt&amp;author=G.J.+Rabiasz&amp;author=E.A.+Stronach&amp;author=L.+Li&amp;volume=34&amp;publication_year=2003&amp;pages=337-343&amp;pmid=12819783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C26">26. <span class="element-citation">Liu J., Li G., Peng X., Liu B., Yin B., Tan X., Fan M., Fan W., Qiang B., Yuan J. The cloning and preliminarily functional analysis of the human neurotrimin gene. <span><span class="ref-journal">Sci. China C Life Sci. </span>2004;<span class="ref-vol">47</span>:158–164.</span>  <a href="http://dx.doi.org/10.1360/03yc0072" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1360/03yc0072</a>. [<a href="/pubmed/15379248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+China+C+Life+Sci.&amp;title=The+cloning+and+preliminarily+functional+analysis+of+the+human+neurotrimin+gene&amp;author=J.+Liu&amp;author=G.+Li&amp;author=X.+Peng&amp;author=B.+Liu&amp;author=B.+Yin&amp;volume=47&amp;publication_year=2004&amp;pages=158-164&amp;pmid=15379248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C27">27. <span class="element-citation">Kimura H., Miyashita H., Suzuki Y., Kobayashi M., Watanabe K., Sonoda H., Ohta H., Fujiwara T., Shimosegawa T., Sato Y. Distinctive localization and opposed roles of vasohibin-1 and vasohibin-2 in the regulation of angiogenesis. <span><span class="ref-journal">Blood. </span>2009;<span class="ref-vol">113</span>:4810–4818.</span>  <a href="http://dx.doi.org/10.1182/blood-2008-07-170316" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1182/blood-2008-07-170316</a>. [<a href="/pubmed/19204325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Distinctive+localization+and+opposed+roles+of+vasohibin-1+and+vasohibin-2+in+the+regulation+of+angiogenesis&amp;author=H.+Kimura&amp;author=H.+Miyashita&amp;author=Y.+Suzuki&amp;author=M.+Kobayashi&amp;author=K.+Watanabe&amp;volume=113&amp;publication_year=2009&amp;pages=4810-4818&amp;pmid=19204325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C28">28. <span class="element-citation">Wang Z., Castano I.B., De Las Penas A., Adams C., Christman M.F. Pol kappa: a DNA polymerase required for sister chromatid cohesion. <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">289</span>:774–779.</span>  <a href="http://dx.doi.org/10.1126/science.289.5480.774" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1126/science.289.5480.774</a>. [<a href="/pubmed/10926539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Pol+kappa:+a+DNA+polymerase+required+for+sister+chromatid+cohesion&amp;author=Z.+Wang&amp;author=I.B.+Castano&amp;author=A.+De+Las+Penas&amp;author=C.+Adams&amp;author=M.F.+Christman&amp;volume=289&amp;publication_year=2000&amp;pages=774-779&amp;pmid=10926539&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C29">29. <span class="element-citation">Baljinnyam E., Umemura M., De Lorenzo M.S., Xie L.H., Nowycky M., Iwatsubo M., Chen S., Goydos J.S., Iwatsubo K. Gβγ subunits inhibit Epac-induced melanoma cell migration. <span><span class="ref-journal">BMC Cancer. </span>2011;<span class="ref-vol">11</span>:256.</span>  <a href="http://dx.doi.org/10.1186/1471-2407-11-256" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1186/1471-2407-11-256</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3141774/">PMC free article</a>]</span> [<a href="/pubmed/21679469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Cancer&amp;title=Gβγ+subunits+inhibit+Epac-induced+melanoma+cell+migration&amp;author=E.+Baljinnyam&amp;author=M.+Umemura&amp;author=M.S.+De+Lorenzo&amp;author=L.H.+Xie&amp;author=M.+Nowycky&amp;volume=11&amp;publication_year=2011&amp;pages=256&amp;pmid=21679469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C30">30. <span class="element-citation">Westerdahl J., Anderson H., Olsson H., Ingvar C. Reproducibility of a self-administered questionnaire for assessment of melanoma risk. <span><span class="ref-journal">Int. J. Epidemiol. </span>1996;<span class="ref-vol">25</span>:245–251.</span>  <a href="http://dx.doi.org/10.1093/ije/25.2.245" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/ije/25.2.245</a>. [<a href="/pubmed/9119548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol.&amp;title=Reproducibility+of+a+self-administered+questionnaire+for+assessment+of+melanoma+risk&amp;author=J.+Westerdahl&amp;author=H.+Anderson&amp;author=H.+Olsson&amp;author=C.+Ingvar&amp;volume=25&amp;publication_year=1996&amp;pages=245-251&amp;pmid=9119548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C31">31. <span class="element-citation">Berwick M., Chen Y.T. Reliability of reported sunburn history in a case-control study of cutaneous malignant melanoma. <span><span class="ref-journal">Am. J. Epidemiol. </span>1995;<span class="ref-vol">141</span>:1033–1037.</span> [<a href="/pubmed/7771439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Reliability+of+reported+sunburn+history+in+a+case-control+study+of+cutaneous+malignant+melanoma&amp;author=M.+Berwick&amp;author=Y.T.+Chen&amp;volume=141&amp;publication_year=1995&amp;pages=1033-1037&amp;pmid=7771439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C32">32. <span class="element-citation">Qureshi A.A., Zhang M., Han J. Heterogeneity in host risk factors for incident melanoma and non-melanoma skin cancer in a cohort of US women. <span><span class="ref-journal">J. Epidemiol. </span>2011;<span class="ref-vol">21</span>:197–203.</span>  <a href="http://dx.doi.org/10.2188/jea.JE20100145" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.2188/jea.JE20100145</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3899409/">PMC free article</a>]</span> [<a href="/pubmed/21515942" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Epidemiol&amp;title=Heterogeneity+in+host+risk+factors+for+incident+melanoma+and+non-melanoma+skin+cancer+in+a+cohort+of+US+women&amp;author=A.A.+Qureshi&amp;author=M.+Zhang&amp;author=J.+Han&amp;volume=21&amp;publication_year=2011&amp;pages=197-203&amp;pmid=21515942&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C33">33. <span class="element-citation">Cho E., Rosner B.A., Feskanich D., Colditz G.A. Risk factors and individual probabilities of melanoma for whites. <span><span class="ref-journal">J. Clin. Oncol. </span>2005;<span class="ref-vol">23</span>:2669–2675.</span>  <a href="http://dx.doi.org/10.1200/JCO.2005.11.108" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1200/JCO.2005.11.108</a>. [<a href="/pubmed/15837981" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Risk+factors+and+individual+probabilities+of+melanoma+for+whites&amp;author=E.+Cho&amp;author=B.A.+Rosner&amp;author=D.+Feskanich&amp;author=G.A.+Colditz&amp;volume=23&amp;publication_year=2005&amp;pages=2669-2675&amp;pmid=15837981&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C34">34. <span class="element-citation">Hunter D.J., Kraft P., Jacobs K.B., Cox D.G., Yeager M., Hankinson S.E., Wacholder S., Wang Z., Welch R., Hutchinson A., et al.  A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span>  <a href="http://dx.doi.org/10.1038/ng2075" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng2075</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=D.J.+Hunter&amp;author=P.+Kraft&amp;author=K.B.+Jacobs&amp;author=D.G.+Cox&amp;author=M.+Yeager&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C35">35. <span class="element-citation">Qi L., Cornelis M.C., Kraft P., Stanya K.J., Linda Kao W.H., Pankow J.S., Dupuis J., Florez J.C., Fox C.S., Pare G., et al.  Genetic variants at 2q24 are associated with susceptibility to type 2 diabetes. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:2706–2715.</span>  <a href="http://dx.doi.org/10.1093/hmg/ddq156" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/hmg/ddq156</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2883345/">PMC free article</a>]</span> [<a href="/pubmed/20418489" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genetic+variants+at+2q24+are+associated+with+susceptibility+to+type+2+diabetes&amp;author=L.+Qi&amp;author=M.C.+Cornelis&amp;author=P.+Kraft&amp;author=K.J.+Stanya&amp;author=W.H.+Linda+Kao&amp;volume=19&amp;publication_year=2010&amp;pages=2706-2715&amp;pmid=20418489&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C36">36. <span class="element-citation">Rimm E.B., Giovannucci E.L., Willett W.C., Colditz G.A., Ascherio A., Rosner B., Stampfer M.J. Prospective study of alcohol consumption and risk of coronary disease in men. <span><span class="ref-journal">Lancet. </span>1991;<span class="ref-vol">338</span>:464–468.</span>  <a href="http://dx.doi.org/10.1016/0140-6736(91)90542-W" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/0140-6736(91)90542-W</a>. [<a href="/pubmed/1678444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Prospective+study+of+alcohol+consumption+and+risk+of+coronary+disease+in+men&amp;author=E.B.+Rimm&amp;author=E.L.+Giovannucci&amp;author=W.C.+Willett&amp;author=G.A.+Colditz&amp;author=A.+Ascherio&amp;volume=338&amp;publication_year=1991&amp;pages=464-468&amp;pmid=1678444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C37">37. <span class="element-citation">Taylor E.N., Stampfer M.J., Curhan G.C. Obesity, weight gain, and the risk of kidney stones. <span><span class="ref-journal">JAMA. </span>2005;<span class="ref-vol">293</span>:455–462.</span>  <a href="http://dx.doi.org/10.1001/jama.293.4.455" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1001/jama.293.4.455</a>. [<a href="/pubmed/15671430" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Obesity,+weight+gain,+and+the+risk+of+kidney+stones&amp;author=E.N.+Taylor&amp;author=M.J.+Stampfer&amp;author=G.C.+Curhan&amp;volume=293&amp;publication_year=2005&amp;pages=455-462&amp;pmid=15671430&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C38">38. <span class="element-citation">Forman J.P., Fisher N.D., Schopick E.L., Curhan G.C. Higher levels of albuminuria within the normal range predict incident hypertension. <span><span class="ref-journal">J. Am. Soc. Nephrol. </span>2008;<span class="ref-vol">19</span>:1983–1988.</span>  <a href="http://dx.doi.org/10.1681/ASN.2008010038" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1681/ASN.2008010038</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2551568/">PMC free article</a>]</span> [<a href="/pubmed/18579639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Soc.+Nephrol.&amp;title=Higher+levels+of+albuminuria+within+the+normal+range+predict+incident+hypertension&amp;author=J.P.+Forman&amp;author=N.D.+Fisher&amp;author=E.L.+Schopick&amp;author=G.C.+Curhan&amp;volume=19&amp;publication_year=2008&amp;pages=1983-1988&amp;pmid=18579639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C39">39. <span class="element-citation">Lin W., Qureshi A.A., Kraft P., Nan H., Guo Q., Hu F.B., Jensen M.K., Han J. ASIP genetic variants and the number of non-melanoma skin cancers. <span><span class="ref-journal">Cancer Causes Control. </span>2011;<span class="ref-vol">22</span>:495–501.</span>  <a href="http://dx.doi.org/10.1007/s10552-010-9724-1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1007/s10552-010-9724-1</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042546/">PMC free article</a>]</span> [<a href="/pubmed/21221757" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Causes+Control&amp;title=ASIP+genetic+variants+and+the+number+of+non-melanoma+skin+cancers&amp;author=W.+Lin&amp;author=A.A.+Qureshi&amp;author=P.+Kraft&amp;author=H.+Nan&amp;author=Q.+Guo&amp;volume=22&amp;publication_year=2011&amp;pages=495-501&amp;pmid=21221757&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C40">40. <span class="element-citation">Wei-Passanese E.X., Han J., Lin W., Li T., Laden F., Qureshi A.A. Geographical variation in residence and risk of multiple nonmelanoma skin cancers in US women and men. <span><span class="ref-journal">Photochem. Photobiol. </span>2012;<span class="ref-vol">88</span>:483–489.</span>  <a href="http://dx.doi.org/10.1111/j.1751-1097.2012.01077.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.1751-1097.2012.01077.x</a>. [<a href="/pubmed/22211791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Photochem.+Photobiol.&amp;title=Geographical+variation+in+residence+and+risk+of+multiple+nonmelanoma+skin+cancers+in+US+women+and+men&amp;author=E.X.+Wei-Passanese&amp;author=J.+Han&amp;author=W.+Lin&amp;author=T.+Li&amp;author=F.+Laden&amp;volume=88&amp;publication_year=2012&amp;pages=483-489&amp;pmid=22211791&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C41">41. <span class="element-citation">Amos C.I., Wang L.E., Lee J.E., Gershenwald J.E., Chen W.V., Fang S., Kosoy R., Zhang M., Qureshi A.A., Vattathil S., et al.  Genome-wide association study identifies novel loci predisposing to cutaneous melanoma. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:5012–5023.</span>  <a href="http://dx.doi.org/10.1093/hmg/ddr415" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/hmg/ddr415</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3298855/">PMC free article</a>]</span> [<a href="/pubmed/21926416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+study+identifies+novel+loci+predisposing+to+cutaneous+melanoma&amp;author=C.I.+Amos&amp;author=L.E.+Wang&amp;author=J.E.+Lee&amp;author=J.E.+Gershenwald&amp;author=W.V.+Chen&amp;volume=20&amp;publication_year=2011&amp;pages=5012-5023&amp;pmid=21926416&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C42">42. <span class="element-citation">Carlson C.S., Eberle M.A., Rieder M.J., Yi Q., Kruglyak L., Nickerson D.A. Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2004;<span class="ref-vol">74</span>:106–120.</span>  <a href="http://dx.doi.org/10.1086/381000" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1086/381000</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181897/">PMC free article</a>]</span> [<a href="/pubmed/14681826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Selecting+a+maximally+informative+set+of+single-nucleotide+polymorphisms+for+association+analyses+using+linkage+disequilibrium&amp;author=C.S.+Carlson&amp;author=M.A.+Eberle&amp;author=M.J.+Rieder&amp;author=Q.+Yi&amp;author=L.+Kruglyak&amp;volume=74&amp;publication_year=2004&amp;pages=106-120&amp;pmid=14681826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C43">43. <span class="element-citation">Stram D.O., Leigh Pearce C., Bretsky P., Freedman M., Hirschhorn J.N., Altshuler D., Kolonel L.N., Henderson B.E., Thomas D.C. Modeling and E-M estimation of haplotype-specific relative risks from genotype data for a case-control study of unrelated individuals. <span><span class="ref-journal">Hum. Hered. </span>2003;<span class="ref-vol">55</span>:179–190.</span>  <a href="http://dx.doi.org/10.1159/000073202" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1159/000073202</a>. [<a href="/pubmed/14566096" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered.&amp;title=Modeling+and+E-M+estimation+of+haplotype-specific+relative+risks+from+genotype+data+for+a+case-control+study+of+unrelated+individuals&amp;author=D.O.+Stram&amp;author=C.+Leigh+Pearce&amp;author=P.+Bretsky&amp;author=M.+Freedman&amp;author=J.N.+Hirschhorn&amp;volume=55&amp;publication_year=2003&amp;pages=179-190&amp;pmid=14566096&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C44">44. <span class="element-citation">Li Y., Willer C., Sanna S., Abecasis G. Genotype imputation. <span><span class="ref-journal">Ann. Rev. Genomics Hum. Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span>  <a href="http://dx.doi.org/10.1146/annurev.genom.9.081307.164242" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1146/annurev.genom.9.081307.164242</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rev.+Genomics+Hum.+Genet.&amp;title=Genotype+imputation&amp;author=Y.+Li&amp;author=C.+Willer&amp;author=S.+Sanna&amp;author=G.+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C45">45. <span class="element-citation">Marchini J., Donnelly P., Cardon L.R. Genome-wide strategies for detecting multiple loci that influence complex diseases. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:413–417.</span>  <a href="http://dx.doi.org/10.1038/ng1537" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng1537</a>. [<a href="/pubmed/15793588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+strategies+for+detecting+multiple+loci+that+influence+complex+diseases&amp;author=J.+Marchini&amp;author=P.+Donnelly&amp;author=L.R.+Cardon&amp;volume=37&amp;publication_year=2005&amp;pages=413-417&amp;pmid=15793588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C46">46. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span>  <a href="http://dx.doi.org/10.1038/ng1847" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng1847</a>. [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C47">47. <span class="element-citation">Kraft P., Cox D.G., Paynter R.A., Hunter D., De Vivo I. Accounting for haplotype uncertainty in matched association studies: a comparison of simple and flexible techniques. <span><span class="ref-journal">Genet. Epidemiol. </span>2005;<span class="ref-vol">28</span>:261–272.</span>  <a href="http://dx.doi.org/10.1002/gepi.20061" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1002/gepi.20061</a>. [<a href="/pubmed/15637718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Accounting+for+haplotype+uncertainty+in+matched+association+studies:+a+comparison+of+simple+and+flexible+techniques&amp;author=P.+Kraft&amp;author=D.G.+Cox&amp;author=R.A.+Paynter&amp;author=D.+Hunter&amp;author=I.+De+Vivo&amp;volume=28&amp;publication_year=2005&amp;pages=261-272&amp;pmid=15637718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C48">48. <span class="element-citation">Zhai R., Gong M.N., Zhou W., Thompson T.B., Kraft P., Su L., Christiani D.C. Genotypes and haplotypes of the VEGF gene are associated with higher mortality and lower VEGF plasma levels in patients with ARDS. <span><span class="ref-journal">Thorax. </span>2007;<span class="ref-vol">62</span>:718–722.</span>  <a href="http://dx.doi.org/10.1136/thx.2006.069393" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1136/thx.2006.069393</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2117284/">PMC free article</a>]</span> [<a href="/pubmed/17289863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thorax&amp;title=Genotypes+and+haplotypes+of+the+VEGF+gene+are+associated+with+higher+mortality+and+lower+VEGF+plasma+levels+in+patients+with+ARDS&amp;author=R.+Zhai&amp;author=M.N.+Gong&amp;author=W.+Zhou&amp;author=T.B.+Thompson&amp;author=P.+Kraft&amp;volume=62&amp;publication_year=2007&amp;pages=718-722&amp;pmid=17289863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT142C49">49. <span class="element-citation">Feigelson H.S., Cox D.G., Cann H.M., Wacholder S., Kaaks R., Henderson B.E., Albanes D., Altshuler D., Berglund G., Berrino F., et al.  Haplotype analysis of the HSD17B1 gene and risk of breast cancer: a comprehensive approach to multicenter analyses of prospective cohort studies. <span><span class="ref-journal">Cancer Res. </span>2006;<span class="ref-vol">66</span>:2468–2475.</span>  <a href="http://dx.doi.org/10.1158/0008-5472.CAN-05-3574" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1158/0008-5472.CAN-05-3574</a>. [<a href="/pubmed/16489054" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Haplotype+analysis+of+the+HSD17B1+gene+and+risk+of+breast+cancer:+a+comprehensive+approach+to+multicenter+analyses+of+prospective+cohort+studies&amp;author=H.S.+Feigelson&amp;author=D.G.+Cox&amp;author=H.M.+Cann&amp;author=S.+Wacholder&amp;author=R.+Kaaks&amp;volume=66&amp;publication_year=2006&amp;pages=2468-2475&amp;pmid=16489054&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3690971&amp;issue-id=224326&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139992329178160title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">WEB RESOURCES</a></li><li><a href="#s6title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s7title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm139992327104000title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm139992327100784title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3690971/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3690971/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3690971/pdf/ddt142.pdf">PDF (208K)</a></li> | <li><a href="#" data-citationid="PMC3690971" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3690971%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3690971%2F&amp;text=Genome-wide%20association%20studies%20identify%20several%20new%20loci%20associated%20with%20pigmentation%20traits%20and%20skin%20cancer%20risk%20in%20European%20Americans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3690971%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23548203" data-aiid="3690971" data-aid="3690971" data-iid="224326" data-domainid="740" data-domain="hmg" data-accid="PMC3690971" data-md5="b645b000d4015f58db26e04f177b640a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:37:53-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CBA5FDD5558B100000000063603EA&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3690971%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3690971/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>