<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3658166/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association analysis of red blood cell traits in African Americans: the COGENT Network</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3658166/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association analysis of red blood cell traits in African Americans: the COGENT Network"><meta name="citation_authors" content="Zhao Chen, Hua Tang, Rehan Qayyum, Ursula M. Schick, Michael A. Nalls, Robert Handsaker, Jin Li, Yingchang Lu, Lisa R. Yanek, Brendan Keating, Yan Meng, Frank J.A. van Rooij, Yukinori Okada, Michiaki Kubo, Laura Rasmussen-Torvik, Margaux F. Keller, Leslie Lange, Michele Evans, Erwin P. Bottinger, Michael D. Linderman, Douglas M. Ruderfer, Hakon Hakonarson, George Papanicolaou, Alan B. Zonderman, Omri Gottesman, BioBank Japan Project, CHARGE Consortium, Cynthia Thomson, Elad Ziv, Andrew B. Singleton, Ruth J.F. Loos, Patrick M.A. Sleiman, Santhi Ganesh, Steven McCarroll, Diane M. Becker, James G. Wilson, Guillaume Lettre, Alexander P. Reiner"><meta name="citation_date" content="15 June 2013"><meta name="citation_issue" content="12"><meta name="citation_volume" content="22"><meta name="citation_firstpage" content="2529"><meta name="citation_doi" content="10.1093/hmg/ddt087"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3658166/?report=abstract"><meta name="citation_pmid" content="23446634"><meta name="DC.Title" content="Genome-wide association analysis of red blood cell traits in African Americans: the COGENT Network"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Zhao Chen"><meta name="DC.Contributor" content="Hua Tang"><meta name="DC.Contributor" content="Rehan Qayyum"><meta name="DC.Contributor" content="Ursula M. Schick"><meta name="DC.Contributor" content="Michael A. Nalls"><meta name="DC.Contributor" content="Robert Handsaker"><meta name="DC.Contributor" content="Jin Li"><meta name="DC.Contributor" content="Yingchang Lu"><meta name="DC.Contributor" content="Lisa R. Yanek"><meta name="DC.Contributor" content="Brendan Keating"><meta name="DC.Contributor" content="Yan Meng"><meta name="DC.Contributor" content="Frank J.A. van Rooij"><meta name="DC.Contributor" content="Yukinori Okada"><meta name="DC.Contributor" content="Michiaki Kubo"><meta name="DC.Contributor" content="Laura Rasmussen-Torvik"><meta name="DC.Contributor" content="Margaux F. Keller"><meta name="DC.Contributor" content="Leslie Lange"><meta name="DC.Contributor" content="Michele Evans"><meta name="DC.Contributor" content="Erwin P. Bottinger"><meta name="DC.Contributor" content="Michael D. Linderman"><meta name="DC.Contributor" content="Douglas M. Ruderfer"><meta name="DC.Contributor" content="Hakon Hakonarson"><meta name="DC.Contributor" content="George Papanicolaou"><meta name="DC.Contributor" content="Alan B. Zonderman"><meta name="DC.Contributor" content="Omri Gottesman"><meta name="DC.Contributor" content="BioBank Japan Project, CHARGE Consortium"><meta name="DC.Contributor" content="Cynthia Thomson"><meta name="DC.Contributor" content="Elad Ziv"><meta name="DC.Contributor" content="Andrew B. Singleton"><meta name="DC.Contributor" content="Ruth J.F. Loos"><meta name="DC.Contributor" content="Patrick M.A. Sleiman"><meta name="DC.Contributor" content="Santhi Ganesh"><meta name="DC.Contributor" content="Steven McCarroll"><meta name="DC.Contributor" content="Diane M. Becker"><meta name="DC.Contributor" content="James G. Wilson"><meta name="DC.Contributor" content="Guillaume Lettre"><meta name="DC.Contributor" content="Alexander P. Reiner"><meta name="DC.Date" content="2013 Jun 15"><meta name="DC.Identifier" content="10.1093/hmg/ddt087"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association analysis of red blood cell traits in African Americans: the COGENT Network"><meta property="og:type" content="article"><meta property="og:description" content="Laboratory red blood cell (RBC) measurements are clinically important, heritable and differ among ethnic groups. To identify genetic variants that contribute to RBC phenotypes in African Americans (AAs), we conducted a genome-wide association study in ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3658166/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3658166/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3658166/pdf/ddt087.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89C91EDD5543B100000000046302D1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3658166%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3658166</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2013 Jun 15; 22(12): 2529–2538. </span></div><div><span class="fm-vol-iss-date">Published online 2013 Feb 26. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddt087" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddt087</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3658166</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23446634">23446634</a></div></div></div></div><h1 class="content-title">Genome-wide association analysis of red blood cell traits in African Americans: the COGENT Network</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Chen%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702662105328" co-class="co-affbox">Zhao Chen</a>,<sup>1</sup> <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702735855200" co-class="co-affbox">Hua Tang</a>,<sup>3</sup> <a href="/pubmed/?term=Qayyum%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702651830304" co-class="co-affbox">Rehan Qayyum</a>,<sup>4</sup> <a href="/pubmed/?term=Schick%20UM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702651828544" co-class="co-affbox">Ursula M. Schick</a>,<sup>5</sup> <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702651826560" co-class="co-affbox">Michael A. Nalls</a>,<sup>6</sup> <a href="/pubmed/?term=Handsaker%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702658107056" co-class="co-affbox">Robert Handsaker</a>,<sup>7</sup> <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702658105072" co-class="co-affbox">Jin Li</a>,<sup>8</sup> <a href="/pubmed/?term=Lu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702658103088" co-class="co-affbox">Yingchang Lu</a>,<sup>10</sup> <a href="/pubmed/?term=Yanek%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661747136" co-class="co-affbox">Lisa R. Yanek</a>,<sup>15</sup> <a href="/pubmed/?term=Keating%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661745152" co-class="co-affbox">Brendan Keating</a>,<sup>16</sup> <a href="/pubmed/?term=Meng%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661743168" co-class="co-affbox">Yan Meng</a>,<sup>18</sup> <a href="/pubmed/?term=van%20Rooij%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702660354960" co-class="co-affbox">Frank J.A. van Rooij</a>,<sup>19</sup> <a href="/pubmed/?term=Okada%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702660352976" co-class="co-affbox">Yukinori Okada</a>,<sup>20,</sup><sup>21,</sup><sup>22</sup> <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702660349520" co-class="co-affbox">Michiaki Kubo</a>,<sup>23</sup> <a href="/pubmed/?term=Rasmussen-Torvik%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661713040" co-class="co-affbox">Laura Rasmussen-Torvik</a>,<sup>24</sup> <a href="/pubmed/?term=Keller%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661711056" co-class="co-affbox">Margaux F. Keller</a>,<sup>25</sup> <a href="/pubmed/?term=Lange%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661709072" co-class="co-affbox">Leslie Lange</a>,<sup>26</sup> <a href="/pubmed/?term=Evans%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661045008" co-class="co-affbox">Michele Evans</a>,<sup>27</sup> <a href="/pubmed/?term=Bottinger%20EP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661043024" co-class="co-affbox">Erwin P. Bottinger</a>,<sup>11</sup> <a href="/pubmed/?term=Linderman%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702661041040" co-class="co-affbox">Michael D. Linderman</a>,<sup>12</sup> <a href="/pubmed/?term=Ruderfer%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702652142992" co-class="co-affbox">Douglas M. Ruderfer</a>,<sup>13</sup> <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702652141008" co-class="co-affbox">Hakon Hakonarson</a>,<sup>8,</sup><sup>9,</sup><sup>17</sup> <a href="/pubmed/?term=Papanicolaou%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702652137552" co-class="co-affbox">George Papanicolaou</a>,<sup>28</sup> <a href="/pubmed/?term=Zonderman%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702652562832" co-class="co-affbox">Alan B. Zonderman</a>,<sup>29</sup> <a href="/pubmed/?term=Gottesman%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702652560848" co-class="co-affbox">Omri Gottesman</a>,<sup>11</sup> BioBank Japan Project, CHARGE Consortium, <a href="/pubmed/?term=Thomson%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702652558240" co-class="co-affbox">Cynthia Thomson</a>,<sup>2</sup> <a href="/pubmed/?term=Ziv%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702653855968" co-class="co-affbox">Elad Ziv</a>,<sup>30</sup> <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702653853984" co-class="co-affbox">Andrew B. Singleton</a>,<sup>25</sup> <a href="/pubmed/?term=Loos%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702653852000" co-class="co-affbox">Ruth J.F. Loos</a>,<sup>14</sup> <a href="/pubmed/?term=Sleiman%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702734439952" co-class="co-affbox">Patrick M.A. Sleiman</a>,<sup>8,</sup><sup>9,</sup><sup>17</sup> <a href="/pubmed/?term=Ganesh%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702734436496" co-class="co-affbox">Santhi Ganesh</a>,<sup>31</sup> <a href="/pubmed/?term=McCarroll%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702662122592" co-class="co-affbox">Steven McCarroll</a>,<sup>32,</sup><sup>33</sup> <a href="/pubmed/?term=Becker%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702662119872" co-class="co-affbox">Diane M. Becker</a>,<sup>4</sup> <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702662117888" co-class="co-affbox">James G. Wilson</a>,<sup>34</sup> <a href="/pubmed/?term=Lettre%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702660616048" co-class="co-affbox">Guillaume Lettre</a>,<sup>35</sup> and  <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634" class="affpopup" co-rid="_co_idm139702660614064" co-class="co-affbox">Alexander P. Reiner</a><sup>5,</sup><sup>36,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139702662105328"><h3 class="no_margin">Zhao Chen</h3><p><sup>1</sup>Division of Epidemiology and Biostatistics, Mel and Enid Zuckerman College of Public Health,  and</p><div>Find articles by <a href="/pubmed/?term=Chen%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Zhao Chen</a></div></div><div id="_co_idm139702735855200"><h3 class="no_margin">Hua Tang</h3><p><sup>3</sup>Department of Statistics and Department of Genetics, Stanford University, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Hua Tang</a></div></div><div id="_co_idm139702651830304"><h3 class="no_margin">Rehan Qayyum</h3><p><sup>4</sup>GeneSTAR Research Program, Division of General Internal Medicine, Johns Hopkins School of Medicine, Baltimore, MD 21287, USA</p><div>Find articles by <a href="/pubmed/?term=Qayyum%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Rehan Qayyum</a></div></div><div id="_co_idm139702651828544"><h3 class="no_margin">Ursula M. Schick</h3><p><sup>5</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Schick%20UM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Ursula M. Schick</a></div></div><div id="_co_idm139702651826560"><h3 class="no_margin">Michael A. Nalls</h3><p><sup>6</sup>Molecular Genetics Section, Laboratory of Neurogenetics, National Institute on Aging, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Michael A. Nalls</a></div></div><div id="_co_idm139702658107056"><h3 class="no_margin">Robert Handsaker</h3><p><sup>7</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA 02141, USA</p><div>Find articles by <a href="/pubmed/?term=Handsaker%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Robert Handsaker</a></div></div><div id="_co_idm139702658105072"><h3 class="no_margin">Jin Li</h3><p><sup>8</sup>Center for Applied Genomics, Abramson Research Center,  and</p><div>Find articles by <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Jin Li</a></div></div><div id="_co_idm139702658103088"><h3 class="no_margin">Yingchang Lu</h3><p><sup>10</sup>The Charles Bronfman Institute for Personalized Medicine, The Genetics of Obesity and Related Metabolic Traits Program</p><div>Find articles by <a href="/pubmed/?term=Lu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Yingchang Lu</a></div></div><div id="_co_idm139702661747136"><h3 class="no_margin">Lisa R. Yanek</h3><p><sup>15</sup>Department of Medicine, The Johns Hopkins University School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Yanek%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Lisa R. Yanek</a></div></div><div id="_co_idm139702661745152"><h3 class="no_margin">Brendan Keating</h3><p><sup>16</sup>Department of Medicine,  and</p><div>Find articles by <a href="/pubmed/?term=Keating%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Brendan Keating</a></div></div><div id="_co_idm139702661743168"><h3 class="no_margin">Yan Meng</h3><p><sup>18</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Meng%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Yan Meng</a></div></div><div id="_co_idm139702660354960"><h3 class="no_margin">Frank J.A. van Rooij</h3><p><sup>19</sup>Department of Epidemiology, Erasmus MC, University Medical Center Rotterdam, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Rooij%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Frank J.A. van Rooij</a></div></div><div id="_co_idm139702660352976"><h3 class="no_margin">Yukinori Okada</h3><p><sup>20</sup>Division of Rheumatology, Immunology, and Allergy,  and</p><p><sup>21</sup>Division of Genetics, Brigham and Women's Hospital, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>22</sup>Medical and Population Genetics Program, Broad Institute, Cambridge, MA 02142, USA</p><div>Find articles by <a href="/pubmed/?term=Okada%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Yukinori Okada</a></div></div><div id="_co_idm139702660349520"><h3 class="no_margin">Michiaki Kubo</h3><p><sup>23</sup>Laboratory for Genotyping Development, CGM, RIKEN, Yokohama, Japan</p><div>Find articles by <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Michiaki Kubo</a></div></div><div id="_co_idm139702661713040"><h3 class="no_margin">Laura Rasmussen-Torvik</h3><p><sup>24</sup>Department of Preventive Medicine, Northwestern University Feinberg School of Medicine, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Rasmussen-Torvik%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Laura Rasmussen-Torvik</a></div></div><div id="_co_idm139702661711056"><h3 class="no_margin">Margaux F. Keller</h3><p><sup>25</sup>Laboratory of Neurogenetics,  and</p><div>Find articles by <a href="/pubmed/?term=Keller%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Margaux F. Keller</a></div></div><div id="_co_idm139702661709072"><h3 class="no_margin">Leslie Lange</h3><p><sup>26</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA</p><div>Find articles by <a href="/pubmed/?term=Lange%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Leslie Lange</a></div></div><div id="_co_idm139702661045008"><h3 class="no_margin">Michele Evans</h3><p><sup>27</sup>Health Disparities Research Section, Clinical Research Branch, National Institute on Aging, National Institutes of Health, Baltimore, MD 21225, USA</p><div>Find articles by <a href="/pubmed/?term=Evans%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Michele Evans</a></div></div><div id="_co_idm139702661043024"><h3 class="no_margin">Erwin P. Bottinger</h3><p><sup>11</sup>The Charles Bronfman Institute for Personalized Medicine</p><div>Find articles by <a href="/pubmed/?term=Bottinger%20EP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Erwin P. Bottinger</a></div></div><div id="_co_idm139702661041040"><h3 class="no_margin">Michael D. Linderman</h3><p><sup>12</sup>Department of Genetics and Genomic Sciences, Institute of Genomics and Multiscale Biology</p><div>Find articles by <a href="/pubmed/?term=Linderman%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Michael D. Linderman</a></div></div><div id="_co_idm139702652142992"><h3 class="no_margin">Douglas M. Ruderfer</h3><p><sup>13</sup>Division of Psychiatric Genomics, Department of Psychiatry,  and</p><div>Find articles by <a href="/pubmed/?term=Ruderfer%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Douglas M. Ruderfer</a></div></div><div id="_co_idm139702652141008"><h3 class="no_margin">Hakon Hakonarson</h3><p><sup>8</sup>Center for Applied Genomics, Abramson Research Center,  and</p><p><sup>9</sup>Division of Human Genetics, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA</p><p><sup>17</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, PA 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Hakon Hakonarson</a></div></div><div id="_co_idm139702652137552"><h3 class="no_margin">George Papanicolaou</h3><p><sup>28</sup>Division of Cardiovascular Sciences, National Heart, Lung, and Blood Institute (NHLBI), Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Papanicolaou%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">George Papanicolaou</a></div></div><div id="_co_idm139702652562832"><h3 class="no_margin">Alan B. Zonderman</h3><p><sup>29</sup>Laboratory of Personality and Cognition, National Institute on Aging, National Institutes of Health, Baltimore, MD 21224, USA</p><div>Find articles by <a href="/pubmed/?term=Zonderman%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Alan B. Zonderman</a></div></div><div id="_co_idm139702652560848"><h3 class="no_margin">Omri Gottesman</h3><p><sup>11</sup>The Charles Bronfman Institute for Personalized Medicine</p><div>Find articles by <a href="/pubmed/?term=Gottesman%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Omri Gottesman</a></div></div><div id="_co_idm139702652558240"><h3 class="no_margin">Cynthia Thomson</h3><p><sup>2</sup>Division of Nutrition, Mel and Enid Zuckerman College of Public Health, University of Arizona, Tucson, AZ 85724, USA</p><div>Find articles by <a href="/pubmed/?term=Thomson%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Cynthia Thomson</a></div></div><div id="_co_idm139702653855968"><h3 class="no_margin">Elad Ziv</h3><p><sup>30</sup>Department of Medicine, University of California, San Francisco, CA 94143, USA</p><div>Find articles by <a href="/pubmed/?term=Ziv%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Elad Ziv</a></div></div><div id="_co_idm139702653853984"><h3 class="no_margin">Andrew B. Singleton</h3><p><sup>25</sup>Laboratory of Neurogenetics,  and</p><div>Find articles by <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Andrew B. Singleton</a></div></div><div id="_co_idm139702653852000"><h3 class="no_margin">Ruth J.F. Loos</h3><p><sup>14</sup>The Charles Bronfman Institute for Personalized Medicine, Institute of Child Health and Development, The Genetics of Obesity and Related Metabolic Traits Program, Mount Sinai School of Medicine, New York, NY 10029, USA</p><div>Find articles by <a href="/pubmed/?term=Loos%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Ruth J.F. Loos</a></div></div><div id="_co_idm139702734439952"><h3 class="no_margin">Patrick M.A. Sleiman</h3><p><sup>8</sup>Center for Applied Genomics, Abramson Research Center,  and</p><p><sup>9</sup>Division of Human Genetics, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA</p><p><sup>17</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, PA 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Sleiman%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Patrick M.A. Sleiman</a></div></div><div id="_co_idm139702734436496"><h3 class="no_margin">Santhi Ganesh</h3><p><sup>31</sup>Division of Cardiology, University of Michigan Health System, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Ganesh%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Santhi Ganesh</a></div></div><div id="_co_idm139702662122592"><h3 class="no_margin">Steven McCarroll</h3><p><sup>32</sup>Department of Genetics, Harvard Medical School, Cambridge, MA, USA</p><p><sup>33</sup>Broad Institute of MIT and Harvard, Cambridge, MA, USA</p><div>Find articles by <a href="/pubmed/?term=McCarroll%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Steven McCarroll</a></div></div><div id="_co_idm139702662119872"><h3 class="no_margin">Diane M. Becker</h3><p><sup>4</sup>GeneSTAR Research Program, Division of General Internal Medicine, Johns Hopkins School of Medicine, Baltimore, MD 21287, USA</p><div>Find articles by <a href="/pubmed/?term=Becker%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Diane M. Becker</a></div></div><div id="_co_idm139702662117888"><h3 class="no_margin">James G. Wilson</h3><p><sup>34</sup>Department of Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS 39216, USA</p><div>Find articles by <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">James G. Wilson</a></div></div><div id="_co_idm139702660616048"><h3 class="no_margin">Guillaume Lettre</h3><p><sup>35</sup>Montreal Heart Institute, Montréal, Québec, Canada, H1T 1C8</p><div>Find articles by <a href="/pubmed/?term=Lettre%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Guillaume Lettre</a></div></div><div id="_co_idm139702660614064"><h3 class="no_margin">Alexander P. Reiner</h3><p><sup>5</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA 98195, USA</p><p><sup>36</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23446634">Alexander P. Reiner</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139702731562192_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139702731562192_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139702731562192_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139702731562192_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Division of Epidemiology and Biostatistics, Mel and Enid Zuckerman College of Public Health,  and</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Division of Nutrition, Mel and Enid Zuckerman College of Public Health, University of Arizona, Tucson, AZ 85724, USA</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Statistics and Department of Genetics, Stanford University, Stanford, CA 94305, USA</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>GeneSTAR Research Program, Division of General Internal Medicine, Johns Hopkins School of Medicine, Baltimore, MD 21287, USA</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA 98195, USA</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Molecular Genetics Section, Laboratory of Neurogenetics, National Institute on Aging, Bethesda, MD 20892, USA</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA 02141, USA</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Center for Applied Genomics, Abramson Research Center,  and</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Division of Human Genetics, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>The Charles Bronfman Institute for Personalized Medicine, The Genetics of Obesity and Related Metabolic Traits Program</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>The Charles Bronfman Institute for Personalized Medicine</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Department of Genetics and Genomic Sciences, Institute of Genomics and Multiscale Biology</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Division of Psychiatric Genomics, Department of Psychiatry,  and</div><div class="fm-affl" id="af14" lang="en"><sup>14</sup>The Charles Bronfman Institute for Personalized Medicine, Institute of Child Health and Development, The Genetics of Obesity and Related Metabolic Traits Program, Mount Sinai School of Medicine, New York, NY 10029, USA</div><div class="fm-affl" id="af15" lang="en"><sup>15</sup>Department of Medicine, The Johns Hopkins University School of Medicine, Baltimore, MD, USA</div><div class="fm-affl" id="af16" lang="en"><sup>16</sup>Department of Medicine,  and</div><div class="fm-affl" id="af17" lang="en"><sup>17</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, PA 19104, USA</div><div class="fm-affl" id="af18" lang="en"><sup>18</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA, USA</div><div class="fm-affl" id="af19" lang="en"><sup>19</sup>Department of Epidemiology, Erasmus MC, University Medical Center Rotterdam, Rotterdam, The Netherlands</div><div class="fm-affl" id="af20" lang="en"><sup>20</sup>Division of Rheumatology, Immunology, and Allergy,  and</div><div class="fm-affl" id="af21" lang="en"><sup>21</sup>Division of Genetics, Brigham and Women's Hospital, Harvard Medical School, Boston, MA 02115, USA</div><div class="fm-affl" id="af22" lang="en"><sup>22</sup>Medical and Population Genetics Program, Broad Institute, Cambridge, MA 02142, USA</div><div class="fm-affl" id="af23" lang="en"><sup>23</sup>Laboratory for Genotyping Development, CGM, RIKEN, Yokohama, Japan</div><div class="fm-affl" id="af24" lang="en"><sup>24</sup>Department of Preventive Medicine, Northwestern University Feinberg School of Medicine, Chicago, IL, USA</div><div class="fm-affl" id="af25" lang="en"><sup>25</sup>Laboratory of Neurogenetics,  and</div><div class="fm-affl" id="af26" lang="en"><sup>26</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA</div><div class="fm-affl" id="af27" lang="en"><sup>27</sup>Health Disparities Research Section, Clinical Research Branch, National Institute on Aging, National Institutes of Health, Baltimore, MD 21225, USA</div><div class="fm-affl" id="af28" lang="en"><sup>28</sup>Division of Cardiovascular Sciences, National Heart, Lung, and Blood Institute (NHLBI), Bethesda, MD, USA</div><div class="fm-affl" id="af29" lang="en"><sup>29</sup>Laboratory of Personality and Cognition, National Institute on Aging, National Institutes of Health, Baltimore, MD 21224, USA</div><div class="fm-affl" id="af30" lang="en"><sup>30</sup>Department of Medicine, University of California, San Francisco, CA 94143, USA</div><div class="fm-affl" id="af31" lang="en"><sup>31</sup>Division of Cardiology, University of Michigan Health System, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="af32" lang="en"><sup>32</sup>Department of Genetics, Harvard Medical School, Cambridge, MA, USA</div><div class="fm-affl" id="af33" lang="en"><sup>33</sup>Broad Institute of MIT and Harvard, Cambridge, MA, USA</div><div class="fm-affl" id="af34" lang="en"><sup>34</sup>Department of Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS 39216, USA</div><div class="fm-affl" id="af35" lang="en"><sup>35</sup>Montreal Heart Institute, Montréal, Québec, Canada, H1T 1C8</div><div class="fm-affl" id="af36" lang="en"><sup>36</sup>Department of Epidemiology, University of Washington, Seattle, WA 98195, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: 1100 N Fairview Ave N M3-A410, Seattle, WA, USA. Tel: +1 206 667 2710; Fax: +1 206 667 4142; Email: <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@renierpa" class="oemail">ude.notgnihsaw.u@renierpa</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139702731562192_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Dec 4; Revised 2013 Feb 9; Accepted 2013 Feb 18.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139702731562192_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2013. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3658166/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3658166/bin/supp_22_12_2529__index.html" data-ga-action="click_feat_suppl">supp_22_12_2529__index.html</a><span style="color:gray"> (1.0K)</span></div><div class="small guid">GUID:&nbsp;887D5E35-7DDF-4190-B3CB-693F6AE00B4C</div>
<div><a href="/pmc/articles/PMC3658166/bin/supp_ddt087_ddt087supp.doc" data-ga-action="click_feat_suppl">supp_ddt087_ddt087supp.doc</a><span style="color:gray"> (1.9M)</span></div><div class="small guid">GUID:&nbsp;E9DAB495-C7C9-4A3D-BD91-5E34FBF54D8C</div>
</div></dd></dl></div><div id="idm139702662739312" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139702662739312title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Laboratory red blood cell (RBC) measurements are clinically important, heritable and differ among ethnic groups. To identify genetic variants that contribute to RBC phenotypes in African Americans (AAs), we conducted a genome-wide association study in up to ∼16 500 AAs. The alpha-globin locus on chromosome 16pter [lead SNP rs13335629 in <em>ITFG3</em> gene; <em>P</em> &lt; 1E−13 for hemoglobin (Hgb), RBC count, mean corpuscular volume (MCV), MCH and MCHC] and the G6PD locus on Xq28 [lead SNP rs1050828; <em>P</em> &lt; 1E − 13 for Hgb, hematocrit (Hct), MCV, RBC count and red cell distribution width (RDW)] were each associated with multiple RBC traits. At the alpha-globin region, both the common African 3.7 kb deletion and common single nucleotide polymorphisms (SNPs) appear to contribute independently to RBC phenotypes among AAs. In the 2p21 region, we identified a novel variant of <em>PRKCE</em> distinctly associated with Hct in AAs. In a genome-wide admixture mapping scan, local European ancestry at the 6p22 region containing <em>HFE</em> and <em>LRRC16A</em> was associated with higher Hgb. <em>LRRC16A</em> has been previously associated with the platelet count and mean platelet volume in AAs, but not with Hgb. Finally, we extended to AAs the findings of association of erythrocyte traits with several loci previously reported in Europeans and/or Asians, including <em>CD164</em> and <em>HBS1L-MYB</em>. In summary, this large-scale genome-wide analysis in AAs has extended the importance of several RBC-associated genetic loci to AAs and identified allelic heterogeneity and pleiotropy at several previously known genetic loci associated with blood cell traits in AAs.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Laboratory red blood cell (RBC) measurements are important for the diagnosis and classification of various hematologic disorders. Some disorders of RBCs, such as sickle cell anemia and alpha thalassemia, are single-gene diseases with higher frequency among populations of African descent (<a href="#DDT087C1" rid="DDT087C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783486">1</a>,<a href="#DDT087C2" rid="DDT087C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783504">2</a>). Even among healthy individuals, African Americans (AAs) have lower hemoglobin (Hgb), hematocrit (Hct) and mean corpuscular volume (MCV) compared with other racial/ethnic groups across all ages (<a href="#DDT087C3" rid="DDT087C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783484">3</a>–<a href="#DDT087C5" rid="DDT087C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783482">5</a>).</p><p id="__p3">Heritability studies suggest that RBC traits are under significant genetic influence. Genome-wide association studies (GWASs) of RBC indices have been reported among European and Japanese populations (<a href="#DDT087C6" rid="DDT087C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783521">6</a>–<a href="#DDT087C8" rid="DDT087C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783515">8</a>), but to our knowledge have not yet been reported for AA. In a gene-centric association study from the CARe consortium, the common African glucose-6-phosphate dehydrogenase (<em>G6PD</em>) A-variant on chromosome X and another variant of the α-globin (<em>HBA2-HBA1</em>) locus were associated with multiple RBC traits in AAs (<a href="#DDT087C9" rid="DDT087C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783513">9</a>).</p><p id="__p4" class="p p-last">The genetic loci reported to date explain only a small fraction of heritability in RBC traits, highlighting the need for larger studies that include ethnic minorities and complementary analytic approaches (<a href="#DDT087C10" rid="DDT087C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783522">10</a>). Thus, we performed a GWA meta-analysis of RBC traits among AA participants from cohorts of the Continental Origins and Genetic Epidemiology Network (COGENT). As AAs are an admixed population, the resulting genomic architecture can be leveraged to identify regions where either African or European ancestral alleles are associated with traits such as Hgb which differ significantly between European and African populations. Therefore, we performed admixture mapping for the association between available RBC traits (Hgb, Hct, MCHC) and local ancestry.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Descriptive analysis</h3><p id="__p5" class="p p-first-last">Since not all RBC traits were available in every COGENT cohort, the numbers of individuals available for meta-analysis varied by each RBC trait (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). Only Hgb (<em>n</em> = 16 485) and Hct (<em>n</em> = 16 496) were available in all cohorts. MCHC (<em>n</em> = 12 152), MCV (<em>n</em> = 6438), RBC count (4818), MCH (<em>n</em> = 4066) and RDW (<em>n</em> = 3811) were available in subsets of participating cohorts. There were varying degrees of pairwise correlation between RBC traits (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). Pearson's correlation coefficients were highest (&gt;0.95) between Hgb and Hct, and between MCV and MCH and were lowest between the RDW and RBC count (0.03).</p></div><div id="s2b" class="sec"><h3 id="s2btitle">GWAS of RBC traits in COGENT AAs</h3><p id="__p6" class="p p-first">The GWA results for each RBC trait are summarized by Manhattan (Fig.&nbsp;<a href="/pmc/articles/PMC3658166/figure/DDT087F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT087F1" rid-ob="ob-DDT087F1" co-legend-rid="lgnd_DDT087F1"><span>1</span></a>) and quantile–quantile (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>) plots. The meta-analysis inflation factors were all near unity (0.998–1.005), suggesting that confounders and other technical artifacts were well-controlled. In total, seven independent genomic loci met the experiment-wide significance threshold (<em>P</em> &lt; 1 × 10<sup>−8</sup>) for one or more RBC traits (Table&nbsp;<a href="/pmc/articles/PMC3658166/table/DDT087TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT087TB1" rid-ob="ob-DDT087TB1" co-legend-rid=""><span>1</span></a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). Three loci (1p31.1, 13q31.2, 16p13.3centromeric) have not been previously associated with RBC traits, whereas four loci (2p21, 6q21, 16p31.3telomeric, and Xq28) have been associated with at least one such trait in populations of European, Japanese or African descent.

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT087TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p7">Results of genome-wide significant SNPs for RBC traits in COGENT AA</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139702660608112" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Trait</th><th rowspan="1" colspan="1" align="left">Chromo-some</th><th rowspan="1" colspan="1" align="left">Number of SNPs with <em>P</em> &lt; 5 × 10<sup>−8</sup></th><th rowspan="1" colspan="1" align="left">Top SNP in the region</th><th rowspan="1" colspan="1" align="left">Position Hg18</th><th rowspan="1" colspan="1" align="left">Candidate genes</th><th rowspan="1" colspan="1" align="left">Minor/major allele</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">Effect size (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">HCT</td><td rowspan="1" colspan="1">Xq28</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs762516</td><td rowspan="1" colspan="1">153 417 857</td><td rowspan="1" colspan="1"><em>G6PD, TKTL1, MECP2, MPP1</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.148</td><td rowspan="1" colspan="1">−0.452 (0.055)</td><td rowspan="1" colspan="1">2.17E − 16</td></tr><tr><td rowspan="1" colspan="1">HCT</td><td rowspan="1" colspan="1">2p21</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">rs13008603</td><td rowspan="1" colspan="1">46 209 352</td><td rowspan="1" colspan="1"><em>PRKCE</em></td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">0.163</td><td rowspan="1" colspan="1">−0.277 (0.047)</td><td rowspan="1" colspan="1">4.09E − 09</td></tr><tr><td rowspan="1" colspan="1">HGB</td><td rowspan="1" colspan="1">16p13.3</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs13335629</td><td rowspan="1" colspan="1">250 381</td><td rowspan="1" colspan="1"><em>ITFG3, LUCL7, NPRL3, POLR3K, RPL2B, MPG</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.120</td><td rowspan="1" colspan="1">−0.190 (0.019)</td><td rowspan="1" colspan="1">2.63E − 23</td></tr><tr><td rowspan="1" colspan="1">HGB</td><td rowspan="1" colspan="1">Xq28</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs762516</td><td rowspan="1" colspan="1">153 417 857</td><td rowspan="1" colspan="1"><em>G6PD, TKTL1, MECP2, MPP1</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.146</td><td rowspan="1" colspan="1">−0.1614 (0.0186)</td><td rowspan="1" colspan="1">3.73E − 18</td></tr><tr><td rowspan="1" colspan="1">MCHC</td><td rowspan="1" colspan="1">16p13.3</td><td rowspan="1" colspan="1">38</td><td rowspan="1" colspan="1">rs13335629</td><td rowspan="1" colspan="1">250 381</td><td rowspan="1" colspan="1"><em>ITFG3, LUCL7, NPRL3, POLR3K, RPL2B, MPG, NME4, DECR</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.117</td><td rowspan="1" colspan="1">−0.3298 (0.0227)</td><td rowspan="1" colspan="1">8.66E − 48</td></tr><tr><td rowspan="1" colspan="1">MCH</td><td rowspan="1" colspan="1">16p13.3</td><td rowspan="1" colspan="1">28</td><td rowspan="1" colspan="1">rs13339636</td><td rowspan="1" colspan="1">238 589</td><td rowspan="1" colspan="1"><em>ITFG3, LUCL7, NPRL3, POLR3K, RPL2B, MPG, NME4, DECR</em></td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.132</td><td rowspan="1" colspan="1">−0.6847 (0.0559)</td><td rowspan="1" colspan="1">1.87E − 34</td></tr><tr><td rowspan="1" colspan="1">MCH</td><td rowspan="1" colspan="1">13q31.2</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">rs9559892<sup>a</sup></td><td rowspan="1" colspan="1">88 166 665</td><td rowspan="1" colspan="1"><em>–</em></td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">0.253</td><td rowspan="1" colspan="1">−0.2589 (0.0444)</td><td rowspan="1" colspan="1">5.46E − 09</td></tr><tr><td rowspan="1" colspan="1">MCH</td><td rowspan="1" colspan="1">16p13.3</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs7192051<sup>a</sup></td><td rowspan="1" colspan="1">4 482 118</td><td rowspan="1" colspan="1"><em>HMOX2</em></td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">0.360</td><td rowspan="1" colspan="1">−0.2396 (0.0411)</td><td rowspan="1" colspan="1">5.70E − 09</td></tr><tr><td rowspan="1" colspan="1">MCH</td><td rowspan="1" colspan="1">6q21</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs9386791</td><td rowspan="1" colspan="1">109 715 190</td><td rowspan="1" colspan="1"><em>CD164</em></td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.416</td><td rowspan="1" colspan="1">−0.2294 (0.0401)</td><td rowspan="1" colspan="1">1.20E − 08</td></tr><tr><td rowspan="1" colspan="1">MCV</td><td rowspan="1" colspan="1">16p13.3</td><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">rs13335629</td><td rowspan="1" colspan="1">250 381</td><td rowspan="1" colspan="1"><em>ITFG3, LUCL7, NPRL3, POLR3K, RPL2B, MPG, NME4, DECR, RHOT2, LMF1, WDR90</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.134</td><td rowspan="1" colspan="1">−0.648 (0.0669)</td><td rowspan="1" colspan="1">3.61E − 22</td></tr><tr><td rowspan="1" colspan="1">MCV</td><td rowspan="1" colspan="1">Xq28</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs762516</td><td rowspan="1" colspan="1">153 417 857</td><td rowspan="1" colspan="1"><em>G6PD, FAM3A, F8, MPP1</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.137</td><td rowspan="1" colspan="1">1.5768 (0.2083)</td><td rowspan="1" colspan="1">3.76E − 14</td></tr><tr><td rowspan="1" colspan="1">MCV</td><td rowspan="1" colspan="1">16p13.3</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs7192051<sup>a</sup></td><td rowspan="1" colspan="1">4 482 118</td><td rowspan="1" colspan="1"><em>HMOX2</em></td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">0.363</td><td rowspan="1" colspan="1">−0.259 (0.0475)</td><td rowspan="1" colspan="1">4.83E − 08</td></tr><tr><td rowspan="1" colspan="1">RBC</td><td rowspan="1" colspan="1">16p13.3</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">rs13335629</td><td rowspan="1" colspan="1">250 381</td><td rowspan="1" colspan="1"><em>ITFG3, LUCL7</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.120</td><td rowspan="1" colspan="1">0.1699 (0.0169)</td><td rowspan="1" colspan="1">7.48E − 24</td></tr><tr><td rowspan="1" colspan="1">RBC</td><td rowspan="1" colspan="1">Xq28</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs1050828</td><td rowspan="1" colspan="1">153 417 411</td><td rowspan="1" colspan="1"><em>G6PD, F8, MPP1, MECP2, CTAG2</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.108</td><td rowspan="1" colspan="1">−0.1424 (0.0159)</td><td rowspan="1" colspan="1">4.00E − 19</td></tr><tr><td rowspan="1" colspan="1">RDW</td><td rowspan="1" colspan="1">Xq28</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs1050828</td><td rowspan="1" colspan="1">153 417 411</td><td rowspan="1" colspan="1"><em>G6PD</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.116</td><td rowspan="1" colspan="1">−0.0326 (0.0048)</td><td rowspan="1" colspan="1">1.70E − 11</td></tr><tr><td rowspan="1" colspan="1">RDW</td><td rowspan="1" colspan="1">1p31.1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs10493739<sup>a</sup></td><td rowspan="1" colspan="1">83 698 745</td><td rowspan="1" colspan="1"><em>–</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.334</td><td rowspan="1" colspan="1">0.0128 (0.0023)</td><td rowspan="1" colspan="1">3.02E − 08</td></tr></tbody></table></div><div id="largeobj_idm139702660608112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3658166/table/DDT087TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139702659282112"><p id="__p8" class="p p-first-last"><sup>a</sup>Novel loci.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT087F1" co-legend-rid="lgnd_DDT087F1"><a href="/pmc/articles/PMC3658166/figure/DDT087F1/" target="figure" rid-figpopup="DDT087F1" rid-ob="ob-DDT087F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139702660750624" class="figure"><a href="/pmc/articles/PMC3658166/figure/DDT087F1/" target="figure" rid-figpopup="DDT087F1" rid-ob="ob-DDT087F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3658166_ddt08701.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddt08701.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3658166/bin/ddt08701.jpg"></a></div><div id="largeobj_idm139702660750624" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3658166/figure/DDT087F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT087F1"><div><a class="figpopup" href="/pmc/articles/PMC3658166/figure/DDT087F1/" target="figure" rid-figpopup="DDT087F1" rid-ob="ob-DDT087F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p9">Manhattan plots of GWAS analysis for RBC traits (<strong>A</strong>) Hct; (<strong>B</strong>) Hgb; (<strong>C</strong>) MCHC; (<strong>D</strong>) MCH; (<strong>E</strong>) MCV; (<strong>F</strong>) RBC count and (<strong>G</strong>) RDW. The dashed horizontal red line indicates <em>P</em> = 1 × 10<sup>−8</sup>. The dashed horizontal blue line indicates <em>P</em> = 5 × 10<sup>−3</sup>.</p></div></div></div><div id="s2b1" class="sec"><p></p><h4 id="s2b1title" class="inline">Previously reported RBC loci </h4><p id="__p10" class="p p-first">The two top Xq28 single nucleotide polymorphisms (SNPs) (rs762516, rs1050828) for Hgb, Hct, MCV and RBC count are located in the <em>G6PD</em> gene. rs1050828 encodes the G6PD amino acid substitution Val68Met that results in the <em>G6PD</em> A<sup>−</sup> allele known to cause <em>G6PD</em> deficiency (MIM #305900). The G6PD A-variant has been previously associated with lower Hct, Hgb and RBC count, and with higher MCV in AAs (<a href="#DDT087C9" rid="DDT087C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783478">9</a>). Here, we additionally report that the G6PD A<sup>−</sup> allele is associated with the lower RDW. Given the extent of the association signal at Xq28, we repeated the Hgb and Hct association analyses in women from Women Health Initiative (WHI), the largest AA cohort (<em>n</em> = 8304) for Hgb and conditioning the lead SNPs, rs1050828. After adjusting for rs1050828, the strength of association with Hgb for the remaining SNPs on Xq28 was greatly attenuated and no longer significant (data not shown).</p><p id="__p11">The index SNP on 16p13, which encompasses the α-globin (<em>HBA2-HBA1</em>) locus, was rs13335629 within an intron of <em>ITFG3</em>. rs13335629 met the genome-wide significance threshold for association with lower Hgb, MCH, MCHC and MCV and also with a higher RBC count<em>.</em> The rs13335629 variant was also nominally associated with lower Hct (<em>β</em> = −0.215 ± 0.056; <em>P</em> = 1.33E−04) and higher RDW (<em>β</em> = 0.0053 ± 0.0021; <em>P</em> = 0.01). Lo <em>et al</em>. previously reported a common rs1211375 variant within the 16p13 region associated with lower Hgb, MCH and MCV in AAs, and that these associations were not present in Caucasians (<a href="#DDT087C9" rid="DDT087C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783483">9</a>). Our index SNP rs13335629 is in moderate linkage disequilibrium (LD) with rs1211375 (<em>r</em><sup>2</sup> = 0.33 in HapMap YRI).</p><p id="__p12">Three intronic variants of the protein kinase C (PKC)-epsilon gene <em>PRKCE</em> on 2p21 were associated with lower Hct. The index SNP rs13008603 was also nominally associated with a lower RBC count (<em>β</em> = −0.044 ± 0.013; <em>P</em> = 4.69E−04), but not with other RBC traits (<em>P</em> &gt; 0.05 for Hgb, MCV, MCH, MCHC, RDW). The three Hct-associated <em>PRKCE</em> variants are in strong LD (pairwise <em>r</em><sup>2</sup> &gt; 0.7). Another intronic variant of <em>PRKCE</em> (rs10495928) was previously associated with Hgb and Hct in Europeans (<a href="#DDT087C8" rid="DDT087C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783512">8</a>) and with the RBC count in Japanese (<a href="#DDT087C6" rid="DDT087C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783506">6</a>), but showed no evidence of association in AAs (<em>P</em> = 0.50 and 0.71 for Hct and Hgb, respectively). In European and African HapMap populations, there is no evidence of LD between rs10495928 and any of the three Hct-associated variants observed in COGENT AA. These results strongly suggest ethnicity-specific allelic heterogeneity for RBC traits at the <em>PRKCE</em> locus.</p><p id="__p13" class="p p-last">At 6q21, a haplotype comprised of 10 SNPs (lead SNP = rs9386791) was associated with a lower MCH, and nominally with a lower MCV (<em>P</em> = 1.09E−05), Hgb (<em>P</em> = 0.007), Hct (<em>P</em> = 0.03), MCHC (<em>P</em> = 0.02), RBC count (<em>P</em> = 0.01). These variants are located ∼50 kb upstream of <em>CD164</em>, which encodes a mucin-like molecule expressed by human CD34(+) hematopoietic progenitor cells that regulate erythropoiesis. Other variants of the <em>CD164</em> 5′ flanking region have been associated with RBC, MCH and MCV in Japanese (rs11966072) (<a href="#DDT087C6" rid="DDT087C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783508">6</a>) and with MCV in Europeans (rs9374080) (<a href="#DDT087C8" rid="DDT087C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783477">8</a>). In HapMap CEU, rs9374080 is in LD with our AA index SNP rs9386791 (<em>r</em><sup>2</sup> = 0.87).</p></div><div id="s2b2" class="sec sec-last"><p></p><h4 id="s2b2title" class="inline">Newly discovered RBC loci </h4><p id="__p14" class="p p-first">Of the three novel loci associated with RBC traits, rs10493739 at 1p31.1 (associated with RDW) and rs9559892 at 13q31.2 (associated with MCH) are both located in regions devoid of known genes. <em>TTLL7</em> is the closest gene to rs10493739 (400 kb away) and encodes a tubulin polyglutamylase, which modifies beta-tubulin (<a href="#DDT087C11" rid="DDT087C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783514">11</a>). There are no known genes within 500 kb on either side of rs9559892. The lead SNP at the third locus, rs7192051 is located within the second intron of the heme oxygenase-2 gene (<em>HMOX2</em>) and was associated with lower MCH and MCV. Heme oxygenase 2, the protein product of HMOX2, degrades heme and is important in erythropoiesis (<a href="#DDT087C12" rid="DDT087C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783500">12</a>). Although <em>HMOX2</em> is located ∼4 Mb centromic to the alpha-globin locus, it is not in LD with the previously identified 16p13 association signals (maximum <em>r</em><sup>2</sup> = 0.004 with rs13335629).</p><p id="__p15" class="p p-last">We attempted to validate two of our three novel RBC loci discovered in COGENT in two independent population-based samples: ∼7700 AA youths ages 8–21 years from CHOP and 2010 AA adults from the Mount Sinai eMERGE study. There was no evidence of replication of rs9559892 with MCH, nor of rs7192051 with MCV or MCH (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>) in the validation sample. It was not possible to pursue replication of rs10493739 in CHOP and eMERGE because this SNP was not genotyped and it could not be imputed in the available replication samples. In over 20 000 Europeans from the CHARGE consortium and 14 000 Japanese from RIKEN, there was no evidence of association of rs9559892 with MCH. Similarly, there was no evidence of association of rs7192051 with MCV or MCH in CHARGE Europeans (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>).</p></div></div><div id="s2c" class="sec"><h3 id="s2ctitle">Admixture mapping analysis of Hgb, Hct and MCHC traits in WHI AA</h3><p id="__p16" class="p p-first">As a complementary approach to identifying variants associated with RBC traits in AAs that occur at disparate frequencies in ancestral African versus European populations, we performed admixture mapping for Hgb, Hct and MCHC in WHI, the largest cohort comprising COGENT. For MCHC, there was one genome-wide significant association signal at the p-term of 16 containing the alpha-globin locus (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>). Local African ancestry in this region was associated with lower MCHC. The admixture association peak is at rs7203694 (<em>P</em> = 2.78e−06) located within <em>RAB40C</em>, and the genome-wide significant region spans 0–0.78 mb (build 36). There were no genome-wide significant admixture associations for Hct (data not shown). For Hgb, a 2 mb region on chromosome 6p22.2–6p22.1 (25.2–27.1 mb, build 36) reached genome-wide significance, with increased European ancestry associated with higher Hgb levels (Fig.&nbsp;<a href="/pmc/articles/PMC3658166/figure/DDT087F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT087F2" rid-ob="ob-DDT087F2" co-legend-rid="lgnd_DDT087F2"><span>2</span></a>A). The Hgb admixture signal appears to be comprised of two peaks (Fig.&nbsp;<a href="/pmc/articles/PMC3658166/figure/DDT087F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT087F2" rid-ob="ob-DDT087F2" co-legend-rid="lgnd_DDT087F2"><span>2</span></a>B). Underlying the centromeric peak is <em>HFE</em>, the hemochromatosis protein-coding gene, which regulates iron absorption by modulating the interaction of the transferrin receptor with transferrin. Two known <em>HFE</em> mutations C282Y (rs1800562) and H63D (rs1799945) cause hereditary hemochromatosis, an autosomal recessive iron storage disorder (<a href="#DDT087C13" rid="DDT087C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783493">13</a>). Among individuals of European descent, the frequencies of C282Y and H63D are 3.23 and 16.6 %, respectively. Both the mutations are essentially absent in the HapMap YRI populations, and therefore the frequency in AAs is low and is the result of European admixture. C282Y was directly typed in WHI SHARe and other COGENT cohorts (total <em>N</em> = 15 584); the genotype association test yielded an Hgb association <em>P</em> = 0.0003 in WHI alone and 4.3 × 10<sup>−6</sup> in COGENT overall (minor allele frequency = 0.015; <em>β</em> = 0.239 ± 0.052). H63D was not directly typed; however, it is tagged by rs129128 (<em>r</em><sup>2</sup> = 1.0 in CEU), which was associated with Hgb levels in WHI (<em>P</em> = 0.008) but not when all COGENT cohorts were analyzed together (<em>P</em> = 0.07).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT087F2" co-legend-rid="lgnd_DDT087F2"><a href="/pmc/articles/PMC3658166/figure/DDT087F2/" target="figure" rid-figpopup="DDT087F2" rid-ob="ob-DDT087F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139702629761312" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddt08702.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddt08702.jpg" src="/pmc/articles/PMC3658166/bin/ddt08702.jpg"></div></a><div id="largeobj_idm139702629761312" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3658166/figure/DDT087F2/" target="figure" rid-figpopup="DDT087F2" rid-ob="ob-DDT087F2"></a><a target="object" href="/pmc/articles/PMC3658166/figure/DDT087F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT087F2"><div><a class="figpopup" href="/pmc/articles/PMC3658166/figure/DDT087F2/" target="figure" rid-figpopup="DDT087F2" rid-ob="ob-DDT087F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p17">Admixture scan of Hgb concentration. (<strong>A</strong>) shows a genome-wide plot of −log(<em>P</em>-values) for local-ancestry association with Hgb. The dashed horizontal line indicates the experiment-wide admixture scan significance threshold of <em>P</em> &lt; 7 × 10<sup>−6</sup>. (<strong>B</strong>) indicates a zoom-in of the genome-wide significant region on chromosome 6, where there appears to be a broad, bimodal admixture peak. The region corresponding to the <em>HFE</em> gene is shown in blue.</p></div></div></div><p id="__p18" class="p p-last">After adjusting for genotypes at C282Y and the H63D proxy rs129128 in WHI, the admixture <em>P</em> value for chromosome 6p22 was attenuated, but remained significant (from 3.28 × 10<sup>−7</sup> to 1.12 × 10<sup>−4</sup>), suggesting the existence of additional variants in this region that contribute to inter-population differences in Hgb levels. Located within the telomeric peak of the admixture signal (Fig.&nbsp;<a href="/pmc/articles/PMC3658166/figure/DDT087F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT087F2" rid-ob="ob-DDT087F2" co-legend-rid="lgnd_DDT087F2"><span>2</span></a>B) are a number of additional variants that have Fst &gt; 0.3, including several near <em>LRRC16A</em>, which has been associated with both serum transferrin levels in whites and the platelet count in AAs from COGENT (<a href="#DDT087C14" rid="DDT087C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783481">14</a>). The most strongly associated <em>LRRC16A</em> variant rs9356970 is located ∼25 kb upstream of the 5′ flanking region (MAF = 0.09; <em>β</em> = 0.118 ± 0.028; <em>P</em> = 2.7 × 10<sup>−5</sup>). According to the HapMap, the minor allele is present in 30% of European chromosomes, but only 2.5% of YRI chromosomes. In a regression model simultaneously adjusting for <em>LRRC16A</em> rs9356970 in addition to <em>HFE</em> C282Y and rs129128, the association signal for local African ancestry at 6p22 was further attenuated, but remained nominally associated with lower Hgb (<em>β</em> = −0.058 ± 0.029; <em>P</em> = 0.045). Together, these results suggest that several European-derived alleles in the 6p22 region, including those of <em>HFE</em> and <em>LRRC16A</em>, may contribute to higher Hgb levels observed in populations of European descent compared with AAs.</p></div><div id="s2d" class="sec"><h3 id="s2dtitle">CNV analysis and assessment of allelic heterogeneity at 16p13 alpha-globin region</h3><p id="__p19" class="p p-first">Given the genetic complexity of the alpha-globin locus on chromosome 16p13, including the presence of a common 3.7 kb alpha-thalassemia deletion in AAs (<a href="#DDT087C15" rid="DDT087C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783501">15</a>), and the extent and magnitude of the observed GWAS signal for RBC traits at 16p13, we assessed structural variation at the 16p13 alpha-globin locus using data from 1000 genomes. First, we confirmed the presence of a common deletion (−α<sup>3.7</sup>) among African Americans (AAs) and West Africans that removes one alpha-globin gene copy (<em>HBA2</em>) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>). Using pooled sequence data from 16 samples (9 YRI, 5 LWK, 1 ASW, 1 CLM) that appear to be homozygous for −α<sup>3.7</sup> deletion, we further localized the breakpoints, which appear to be bounded by ∼300 bp of nearly identical sequence located within the 5′ flanking regions of <em>HBA1</em> and <em>HBA2</em> (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S4</a>). Second, we identified a rare deletion spanning <em>HBM</em> through <em>HBQ1</em>, in three Han Chinese individuals, and several other possible (but uncertain) rare copy number variations (CNVs) including one duplication (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S5</a>) and a very rare deletion that deletes a known regulatory element MCS-R1 (<a href="#DDT087C16" rid="DDT087C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783492">16</a>) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S6</a>).</p><p id="__p20" class="p p-last">Among all typed and imputed SNPs in the WHI dataset, the strongest correlation with alpha37 in 63 YRI samples from 1000 Genomes was observed in the region of <em>ITFG3</em>. This includes rs13335629 (r-squared = 0.6), which is also the top Hgb- and MCHC-associated SNP in COGENT AA. We repeated the association analyses in WHI (<em>n</em> = 8304) for Hgb and MCHC conditioning on rs13335629. The top SNP for Hgb in the conditional analysis was <em>POLR3K</em> rs798693 (<em>P</em> = 8.7E−06). For MCHC, rs2541612 in <em>NPRL3</em> remained genome-wide significantly associated with lower MCHC (<em>P</em> = 1.14E−09). When both <em>ITFG3</em> rs13335629 and <em>NPRL3</em> rs2541612 were included as covariates in the conditional analysis, the SNP most strongly associated with lower MCHC was <em>LUC7L</em> rs1211375 (<em>P</em> = 1.50E−06). Taken together, the 1000 Genomes CNV analyses and the results of conditional regression analyses for Hgb and MCHC suggest that while some of the red cell GWAS association signal may be due to the common African alpha37 deletion, there appears to be independent signals coming from other structural variants and/or SNP(s) in the region.</p></div><div id="s2e" class="sec"><h3 id="s2etitle">Cross-ethnic transferability of previously reported RBC to COGENT AA</h3><p id="__p21" class="p p-first-last">We assessed whether 72 SNPs previously associated with RBC traits in European or Japanese populations are associated with RBC traits in COGENT AA (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>). Using the conservative Bonferroni multiple comparison corrected significance threshold (<em>P</em> &lt; 0.0001), we validated four associations. In addition to the association of <em>HFE</em> rs1800562 with Hgb, these include <em>ITFG3</em> rs1122794 (previously associated with MCH in Europeans) with higher MCHC (<em>P</em> = 1.5 × 10<sup>−8</sup>), MCH (<em>P</em> = 7.2 × 10<sup>−6</sup>) and MCV (<em>P</em> = 7.1 × 10<sup>−5</sup>) in AAs; <em>ITFG3</em> rs7189020 (previously associated with MCV in Europeans) with higher MCH (<em>P</em> = 1.0 × 10<sup>−5</sup>) and MCV (<em>P</em> = 1.5 × 10<sup>−5</sup>) in AAs; and <em>HBS1L-MYB</em> rs7775698 (previously associated with HCT, MCH, MCHC, MCV and RBC count in Japanese) with a lower RBC count (<em>P</em> = 3.3 × 10<sup>−5</sup>) in AAs.</p></div><div id="s2f" class="sec sec-last"><h3 id="s2ftitle">RBC-associated genetic variants and anemia in AA women</h3><p id="__p22" class="p p-first-last">To identify genetic variants associated with anemia, defined dichotomously as Hgb &lt; 12 g/dl, we performed a GWA scan in 8304 AA women 50–79 years old from WHI. Two loci, Xq28 and 16p13, met the threshold of genome-wide significance. The G6PD rs1050828 A-variant was associated with a 1.49-fold (95% CI: 1.33–1.67) increased risk of anemia (<em>P</em> = 3.3 × 10<sup>−12</sup>). The index SNP at 16p13 (rs1088638) is located ∼20 kb 3′ to <em>POLR3K</em>, and was associated with a 1.42-fold (95% CI: 1.26–1.60) increased risk of anemia (<em>P</em> = 1.2 × 10<sup>−8</sup>). We also constructed a composite RBC genetic risk score (GRS) by summing genotyped or imputed allele dosage at the 15 SNPs associated with at least one RBC trait in AA through the GWA scan, admixture mapping scan, conditional analyses or cross-ethnic transferability analyses described above. The GRS ranged from 5 to 19, with a median of 12. When modeled as a quantitative trait, the GRS was strongly associated with anemia (<em>P</em> = 3.5 × 10<sup>−18</sup>), explaining 1.4% of the anemia phenotypic variance, or 2.2% of the variance in Hgb concentration. When WHI participants were grouped into four GRS categories, those in the highest GRS category had a 1.95-fold increased risk of anemia (95% CI: 1.56–2.42) compared with those in the lowest GRS category (<em>P</em> = 3.6 × 10<sup>−9</sup>).</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p23" class="p p-first">In this first reported GWAS meta-analysis of RBC traits in AAs, we report genome-wide associations for four loci (<em>G6PD</em> on Xq28, alpha-globin locus on 16pter, <em>PRKCE</em> on 2p21 and <em>CD164</em> on 6q21). We also validated the association in AAs of variants in genes such as <em>HFE</em> and <em>HBS1L-MYB</em>, which have previously been associated with RBC traits in other ethnicities. At the alpha-globin locus, there appears to be allelic heterogeneity (particularly for MCHC), with both copy number variants and SNPs having apparent independent effects. At <em>PRKCE</em>, the variants associated with lower Hct in our AA sample appear to be distinct from another set of <em>PRKCE</em> variants that have been associated with Hgb, Hct and RBC count in Europeans and Japanese.</p><p id="__p24">Hemizygous males and in some instances female carriers of the X-linked G6PD A- allele are predisposed to acute episodes of drug- or infection-induced hemolytic anemia. Under basal conditions, however, the G6PD A-allele is not generally thought to be associated with RBC abnormalities, and hemizygous G6PD A-individuals have been reported to have normal baseline red cell survival in the absence of oxidant stress (<a href="#DDT087C17" rid="DDT087C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783509">17</a>). Nonetheless, the association of low RDW with G6PD deficiency may be due to low grade hemolysis resulting in an increase in the MCV with rightward shift of the overall distribution of RBC volume without change in the shape of the distribution (<a href="#DDT087C18" rid="DDT087C18" class=" bibr popnode">18</a>). The G6PD A-variant is in LD with other nearby genetic variants that plausibly could influence Hgb or RBC morphology. <em>TKTL1</em> encodes a transketolase enzyme that links the pentose phosphate pathway with anaerobic glycolysis, which constitutes the two major metabolic pathways for glucose utilization in human erythrocytes. <em>MPP1</em> encodes the red cell membrane protein p55, a scaffolding protein that anchors the actin cytoskeleton to the plasma membrane by forming a ternary complex with protein 4.1R and glycophorin C (<a href="#DDT087C19" rid="DDT087C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783476">19</a>).</p><p id="__p25">Aside from genes involved in Hgb synthesis or metabolism, other genetic loci such as <em>CD164</em> and <em>PRKCE</em> may be associated with RBC traits through effects on erythropoiesis. CD164 (endolyn) is an adhesive receptor present on early hematopoietic progenitors and maturing erythroid cells that regulates the adhesion of CD34+ cells to bone marrow stroma and affects migration and proliferation of hematopoietic stem cells and progenitor cells (<a href="#DDT087C20" rid="DDT087C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783479">20</a>,<a href="#DDT087C21" rid="DDT087C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783488">21</a>). The upstream region harboring the RBC trait-associated variants contains an erythroleukemia cell line (K562)-specific cluster of histone modifications and ENCODE transcription factor ChIP-seq binding sites including those for GATA-2 and c-Jun. <em>PRKCE</em> encodes an isoform of PKC, PKC epsilon, which is expressed in hematopoietic progenitor cells in a lineage- and stage-specific manner and appears to influence erythroid and megakaryocytic progenitor proliferation and differentiation by modulating the response of hematopoietic precursors to a tumor necrosis factor-related apoptosis-inducing ligand (<a href="#DDT087C22" rid="DDT087C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783510">22</a>–<a href="#DDT087C24" rid="DDT087C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783511">24</a>).</p><p id="__p26">Though the finding was not validated in independent AA samples, one of our novel genome-wide significant associations in our discovery cohorts was the association of MCH and MCV with <em>HMOX2</em>, which encodes heme oxygenase-2, a constitutively expressed enzyme with a major role in heme catabolism. Heme induces expression of globin genes in erythrocyte progenitor cells and thus plays an important role in erythropoiesis (<a href="#DDT087C12" rid="DDT087C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783490">12</a>,<a href="#DDT087C25" rid="DDT087C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783485">25</a>,<a href="#DDT087C26" rid="DDT087C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783499">26</a>). The lead SNP in this region, rs7192051, is within 5 kb of predicted <em>HMOX2</em> regulatory elements such as transcription factor binding sites, DNase sites and histone modification sites (<a href="#DDT087C27" rid="DDT087C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783507">27</a>). Therefore, further study of this variant in larger, independent samples of AA may be warranted.</p><p id="__p27" class="p p-last">Our findings have potential clinical implications. Although previous studies have explored the role of common genetic variation in the regulation of these RBC phenotypes in populations of European and Asian descent (<a href="#DDT087C6" rid="DDT087C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783517">6</a>,<a href="#DDT087C7" rid="DDT087C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783518">7</a>,<a href="#DDT087C10" rid="DDT087C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783496">10</a>), no systematic genetic association studies of these traits have been reported in African-ancestry populations. This is particularly important, as there are marked differences in these RBC indices among ethnic groups, and anemia is more prevalent in populations of African descent (<a href="#DDT087C28" rid="DDT087C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783523">28</a>). While it appears that some of the phenotypic variations for RBC and other hematologic traits are controlled by genetic variation shared across ethnic groups (<a href="#DDT087C29" rid="DDT087C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783480">29</a>), other RBC loci are relatively unique to Africans. Rare variants, which are not well captured by GWASs, and undetected common variants of more modest effect may account for additional genetic variance. Discovery and validation of these and additional genetic variants associated with RBC traits in other ethnic populations are likely to uncover new mechanisms and pathways that affect hematopoiesis and RBC turnover, offering insights that may inform further research into red cell biology. Indeed, recent reports have shown that genetic loci uncovered through an unbiased genome-wide study in human populations, together with follow-up functional studies incorporating gene expression, bioinformatic analyses and insights from mouse models and gene knockdown experiments, can greatly contribute to our understanding of the biological mechanisms underlying RBC production (<a href="#DDT087C30" rid="DDT087C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783520">30</a>,<a href="#DDT087C31" rid="DDT087C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783487">31</a>).</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Primary subjects and data collection</h3><p id="__p28" class="p p-first">We performed GWA analysis of RBC traits in over 16 000 AAs from seven population-based cohorts that comprise the Continental Origins and Genetic Epidemiology Network (COGENT). The characteristics of each cohort were described in previous publications (<a href="#DDT087C14" rid="DDT087C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783519">14</a>,<a href="#DDT087C29" rid="DDT087C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783516">29</a>). Fasting blood samples were drawn and analyzed for RBC traits at designated clinical laboratories using an automated electronic cell counter. These counters directly measure Hgb concentration (in grams per deciliter), RBC count (in millions per microliter) and MCV, the average size of the RBC in femtoliters. Electronic cell counters calculate MCH, MCHC, Hct and RDW. Hct is the percentage of blood by volume that is occupied by RBC and is calculated by multiplying the RBC count in millions/microliter by the MCV in femtoliters. MCH is the average amount of Hgb inside an RBC expressed in picograms and is calculated by dividing the Hgb concentration by the RBC count in millions per microliter, then multiplying by 10. The MCHC is the average concentration of Hgb in RBCs and is calculated by dividing Hgb in grams per deciliter by Hct. The RDW is a measure of the variance in RBC size and is calculated by dividing the standard deviation of RBC volume by the MCV and multiplying by 100.</p><p id="__p29" class="p p-last">All participants self-reported their race/ethnicity. Additional clinical information was collected by self-report and clinical examination. Participants provided written informed consent as approved by local Human Subjects Committees. Study participants who were pregnant or had a diagnosis of cancer or AIDS at the time of blood count were excluded.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Replication subjects and data collection</h3><p id="__p30" class="p p-first-last">For validation of novel, genome-wide significant associations identified in the COGENT discovery sample, we performed association analyses in two independent population-based samples: ∼7700 AA youths ages 8–21 years from Children's Hospital of Philadelphia (CHOP) and 2010 AA adults from the Mount Sinai electronic Medical Records and Genomics (eMERGE) study. We also attempted to replicate novel loci in two other ethnic populations: 14 088 Japanese from RIKEN and up to 30 000 European Americans from CHARGE. Details of each validation cohort are provided under <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Genotyping and quality-control</h3><p id="__p31" class="p p-first-last">Genomic DNA was extracted from peripheral blood leukocytes and genotyping was performed on the Affymetrix 6.0 array or Illumina Omni or 1 M platforms within each cohort using methods described previously (<a href="#DDT087C14" rid="DDT087C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783489">14</a>,<a href="#DDT087C29" rid="DDT087C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783503">29</a>). DNA samples with a genome-wide genotyping success rate of &lt;90% or sex discordance were excluded, as were genetic ancestry outliers (identified by cluster analysis using principal components analysis or multi-dimensional scaling). SNPs with a genotyping success rate of &lt;95% or MAF &lt;1%, monomorphic SNPs and SNPs that map to several genomic locations were removed from the analyses. Participants and SNPs passing basic quality control thresholds were imputed to &gt;2.2 million autosomal SNPs based on HapMap2 haplotype data using a 1:1 mixture of Europeans (CEU) and Africans (YRI) as the reference panel. Details of the genotype imputation procedure have been described previously (<a href="#DDT087C14" rid="DDT087C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783495">14</a>,<a href="#DDT087C29" rid="DDT087C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783491">29</a>). Prior to discovery meta-analyses, SNPs were excluded if imputation quality metrics (equivalent to the squared correlation between proximal imputed and genotyped SNPs) were &lt;0.30.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Data analyses</h3><p id="__p32" class="p p-first">For all cohorts, GWA analysis was performed on the raw, untransformed RBC trait using linear regression adjusted for covariates, implemented in either PLINK v1.07 or MACH2QTL v1.08. In GeneSTAR, the family structure was accounted for in the association tests using linear mixed-effects models implemented in R (<a href="#DDT087C32" rid="DDT087C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783497">32</a>). For the 22 autosomes, analysis was performed using genotyped and imputed SNPs. For the X chromosome, only genotyped SNPs were analyzed due to the technical limitations of imputing X-linked SNPs. All analyses were performed under an additive genetic model using allelic dosage (genotyped or imputed) at each SNP, adjusted for age, age-squared, sex and clinic site (if applicable), 4–10 principal components.</p><p id="__p33">For each phenotype, meta-analysis was conducted using inverse-variance weighted fixed-effects models to combine <em>β</em> coefficients and standard errors from study-level regression results for each SNP, to derive pooled estimates. Study-level results were corrected for genomic inflation factors (<em>λ</em>) by multiplying the standard errors (SEs) of the regression coefficients by the square-root of the study-specific <em>λ</em>. Meta-analyses were implemented in the METAL software. Between-study heterogeneity of results was assessed by using Cochran’s <em>Q</em> statistic and the <em>I</em><sup>2</sup> inconsistency metric. A threshold of <em>α</em> = 1 × 10<sup>−8</sup> was used to declare genome-wide statistical significance. This statistical threshold accounts for the greater nucleotide diversity and lower LD in African descent populations combined with testing of multiple, correlated RBC traits (<a href="#DDT087C31" rid="DDT087C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783502">31</a>,<a href="#DDT087C33" rid="DDT087C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783494">33</a>). We carried out replication testing of ‘suggestive’ SNPs selected on the basis of a more liberal significance threshold in our primary AA discovery GWAS (<em>P</em> &lt; 5 × 10<sup>−8</sup>).</p><p id="__p34">To assess the potential existence of multiple, independent variants influencing a trait at the same locus (allelic heterogeneity), regression analyses were repeated in the largest sample (WHI, <em>n</em> = 8095), conditional on the most strongly associated (index) SNP in that region.</p><p id="__p35" class="p p-last">We also assessed the transferability to AAs of SNPs previously associated with RBC traits in populations of European or Japanese ancestry by assessing association with RBC traits in the COGENT discovery meta-analyses. For validation, we considered consistency of direction of effect, and assessed statistical significance using a simple Bonferroni adjustment for the total number of SNPs assessed, using a two-sided hypothesis test.</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Local ancestry estimation and admixture mapping in WHI</h3><p id="__p36" class="p p-first-last">For each AA individual in the WHI sample, locus-specific ancestry was estimated using an extension of the model described by Tang <em>et al</em>. (<a href="#DDT087C34" rid="DDT087C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783505">34</a>). We used phased haplotype data from HapMap3 CEU and YRI individuals as reference panels. An admixture mapping analysis was performed in WHI to test for association between Hgb levels and ancestry at each genomic location (local ancestry), while adjusting for the first 10 principal components, regions of recruitment, clinical trial, age and age-squared. The critical value for genome-wide significance level of admixture mapping is substantially lower than the genotype test due to the extensive correlation in local ancestry between adjacent markers that result from the recent admixture in AAs. We therefore adopted an empirically determined genome-wide significance threshold of <em>P</em> &lt; 7.1 × 10<sup>−6</sup>, which corresponds to a Bonferroni correction of ∼7000 independent tests (<a href="#DDT087C35" rid="DDT087C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312783498">35</a>).</p></div><div id="s4f" class="sec sec-last"><h3 id="s4ftitle">Copy number variation (CNV) analysis using 1000 genomes data</h3><p id="__p37" class="p p-first-last">We used the 1000 Genomes sequencing data to investigate CNVs at the chromosome 16 p31 alpha-globin locus, studying 946 African-ancestry samples at roughly 4× sequencing coverage. As a result of noise in depth-based genotyping at this locus (due to low-pass sequencing, high %GC and potential overlapping variants), some of our analyses were confined to the 76 YRI samples, which have higher sequence coverage in 1000 Genomes data and more complete genotyping (call rate 84 % at 95 % CI).</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p38" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt087/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p39" class="p p-first-last">Additional support for this work was provided by NIH (<span>R01 HL71862-06</span> and ARRA
<span>N000949304</span> to A.P.R.). Some of the results of this paper were obtained by using the program package S.A.G.E., which is supported by a US Public Health Service Resource Grant (<span>RR03655</span>) from the National Center for Research Resources. Additional support came from the National Cancer Institute (grant <span>R25CA094880</span> to U.M.S.).</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm139702659514928"><a href="/pmc/articles/PMC3658166/bin/supp_22_12_2529__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm139702661394384" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139702661394384title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p40">The authors wish to acknowledge the support of the National Heart, Lung and Blood Institute and the contributions of the involved research institutions, study investigators, field staff and study participants of Atherosclerosis Risk in Communities (ARIC), Coronary Artery Risk in Young Adults (CARDIA), Jackson Heart Study (JHS) and Broad Institute in creating the Candidate-gene Association Resource for biomedical research (CARe; <a href="http://public.nhlbi.nih.gov/GeneticsGeno​mics/home/care.aspx">http://public.nhlbi.nih.gov/GeneticsGeno​mics/home/care.aspx</a>).</p><p id="__p41">The authors also wish to thank the investigators, staff and participants of GeneSTAR, Health ABC, Healthy Aging in Neighborhoods of Diversity across the Life Span Study (HANDLS) and Women Health Initiative (WHI) for their important contributions. A listing of WHI investigators can be found at <a href="http://www.whiscience.org/publications/WHI_investigators_shortlist.pdf">http://www.whiscience.org/publications/WHI_investigators_shortlist.pdf</a>.</p><p id="__p42">We thank all the children who donated blood samples for genetic research purpose. The CHOP study was funded by the Institute Development Funds to the Center for Applied Genomics at the Children's Hospital of Philadelphia and an Adele S. and Daniel S. Kubert Estate gift to the Center for Applied Genomics.</p><p id="__p43">The Mount Sinai IPM Biobank Program is supported by The Andrea and Charles Bronfman Philanthropies.</p><p id="__p44">The authors acknowledge the essential role of the Cohorts for Heart and Aging Research in Genome Epidemiology (CHARGE) Consortium in development and support of this manuscript. CHARGE members include the Rotterdam Study (RS), Framingham Heart Study (FHS), Cardiovascular Health Study (CHS), the NHLBI's Atherosclerosis Risk in Communities (ARIC) Study and the NIA's Iceland Age, Gene/Environment Susceptibility (AGES) Study. The collaboration of studies such as the Health Aging and Body Composition Study (Health ABC), the Baltimore Longitudinal Study of Aging (BLSA), the Invecchiare in Chianti Study (InChianti), and the Heart and Vascular Health Study (HVH) also played a vital role.</p><p id="__p45">The following parent studies contributed study data, ancillary study data and DNA samples through the Broad Institute (N01-HC-65226) to create this genotype/phenotype data base for wide dissemination to the biomedical research community:</p><p id="__p46">Atherosclerosis Risk in Communities (ARIC): University of North Carolina at Chapel Hill (N01-HC-55015), Baylor Medical College (N01-HC-55016), University of Mississippi Medical Center (N01-HC-55021), University of Minnesota (N01-HC-55019), Johns Hopkins University (N01-HC-55020), University of Texas, Houston (N01-HC-55017), University of North Carolina (N01-HC-55018). Other NIH support contributing to the GWAS in ARIC are: R01HL087641, R01HL59367, R01HL86694, U01HG004402 and HHSN268200625226C.</p><p id="__p47">Coronary Artery Risk in Young Adults (CARDIA): University of Alabama at Birmingham (N01-HC-48047), University of Minnesota (N01-HC-48048), Northwestern University (N01-HC-48049), Kaiser Foundation Research Institute (N01-HC-48050), University of Alabama at Birmingham (N01-HC-95095), Tufts-New England Medical Center (N01-HC-45204), Wake Forest University (N01-HC-45205), Harbor-UCLA Research and Education Institute (N01-HC-05187), University of California, Irvine (N01-HC-45134, N01-HC-95100).</p><p id="__p48">Jackson Heart Study (JHS): Jackson State University (N01-HC-95170), University of Mississippi (N01-HC-95171), Tougaloo College (N01-HC-95172).</p><p id="__p49">Healthy Aging in Neighborhoods of Diversity across the Life Span Study (HANDLS): this research was supported by the Intramural Research Program of the NIH, National Institute on Aging and the National Center on Minority Health and Health Disparities (intramural project Z01-<a id="__tag_788190098" class="tag_hotlink" href="/nuccore/AG000513" ref="/nuccore/AG000513"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG000513","term_id":"55785907"}}</span>AG000513</a> and human subjects protocol 2009-149). Data analyses for the HANDLS study utilized the high-performance computational capabilities of the Biowulf Linux cluster at the National Institutes of Health, Bethesda, MD, USA (<a href="http://biowulf.nih.gov">http://biowulf.nih.gov</a>).</p><p id="__p50">Health ABC: this research was supported by NIA contracts N01AG62101, N01AG62103 and N01AG62106. The GWAS was funded by NIA grant 1R01AG032098-01A1 to Wake Forest University Health Sciences and genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract number HHSN268200782096C. This research was supported in part by the Intramural Research Program of the NIH, National Institute on Aging.</p><p id="__p51">GeneSTAR: this research was supported by the National Heart, Lung and Blood Institute (NHLBI) through the PROGENI (U01 HL72518) and STAMPEED (R01 HL087698-01) consortia. Additional support was provided by grants from the NIH/National Institute of Nursing Research (R01 NR08153), and the NIH/National Center for Research Resources (M01-RR000052) to the Johns Hopkins General Clinical Research Center.</p><p id="__p52">WHI: the WHI program is funded by the National Heart, Lung and Blood Institute, National Institutes of Health, US Department of Health and Human Services through contracts N01WH22110, 24152, 32100–2, 32105–6, 32108–9, 32111–13, 32115, 32118–32119, 32122, 42107–26, 42129–32 and 44221.</p><p id="__p53">AGES: the Age, Gene/Environment Susceptibility Reykjavik Study is funded by NIH contract N01-AG-12100, the NIA Intramural Research Program, Hjartavernd (the Icelandic Heart Association) and the Althingi (the Icelandic Parliament).</p><p id="__p54">Framingham: the National Heart, Lung and Blood Institute's Framingham Heart Study is a joint project of the National Institutes of Health and Boston University School of Medicine and was supported bythe National Heart, Lung, and Blood Institute's Framingham Heart Study (contract No. N01-HC-25195) and its contract with Affymetrix, Inc. for genotyping services (contract No. N02-HL-6-4278). Analyses reflect the efforts and resource development from the Framingham Heart Study investigators participating in the SNP Health Association Resource (SHARe) project. A portion of this research was conducted using the Linux Cluster for Genetic Analysis (LinGA-II) funded by the Robert Dawson Evans Endowment of the Department of Medicine at Boston University School of Medicine and Boston Medical Center.</p><p id="__p55">InChianti: the InChianti Study was supported as a “targeted project” (ICS 110.1RS97.71) by the Italian Ministry of Health, by the US National Institute on Aging (Contracts N01-AG-916413, N01-AG-821336, 263 MD 9164 13 and 263 MD 821336) and in part by the Intramural Research Program, National Institute on Aging, National Institutes of Health, USA.</p><p id="__p56">Rotterdam: Rotterdam Study GWAS database of the Rotterdam Study was funded through the Netherlands Organization of Scientific Research NWO (no. 175.010.2005.011, 911.03.012) and the Research Institute for Diseases in the Elderly (RIDE). This study was supported by the Netherlands Genomics Initiative (NGI)/NWO project number 050 060 810 (Netherlands Consortium for Healthy Ageing). We thank Dr Michael Moorhouse, Pascal Arp, Mila Jhamai, Marijn Verkerk and Sander Bervoets for their help in creating the genetic database. We thank the laboratory technicians Jeannette M Vergeer—Drop, Bernadette H M van Ast—Copier, Andy A L J van Oosterhout, Sue Ellen Mauricia, Andrea J M Vermeij—Verdoold, Els Halbmeijer—van der Plas, Debby M S Lont and Hasna Kariouh for their help in phenotype assessment. The Rotterdam Study is supported by the Erasmus Medical Center and Erasmus University, Rotterdam; the Netherlands organization for scientific research (NWO), the Netherlands Organization for the Health Research and Development (ZonMw), the Research Institute for Diseases in the Elderly (RIDE), the Netherlands Heart Foundation, the Ministry of Education, Culture and Science, the Ministry of Health, Welfare and Sports, the European Commission (DG XII) and the Municipality of Rotterdam.</p><p id="__p57">RIKEN: we would like to thank all the staff of the Laboratory for Statistical Analysis at RIKEN for their technical assistance. The BioBank Japan Project was supported by Ministry of Education, Culture, Sports, Science and Technology, Japan.</p><p id="__p58"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm139702660167728" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139702660167728title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDT087C1">1. <span class="element-citation">Camaschella C. Understanding iron homeostasis through genetic analysis of hemochromatosis and related disorders. <span><span class="ref-journal">Blood. </span>2005;<span class="ref-vol">106</span>:3710–3717.</span> [<a href="/pubmed/16030190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Understanding+iron+homeostasis+through+genetic+analysis+of+hemochromatosis+and+related+disorders&amp;author=C.+Camaschella&amp;volume=106&amp;publication_year=2005&amp;pages=3710-3717&amp;pmid=16030190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C2">2. <span class="element-citation">Melis M.A., Cau M., Congiu R., Sole G., Barella S., Cao A., Westerman M., Cazzola M., Galanello R. A mutation in the TMPRSS6 gene, encoding a transmembrane serine protease that suppresses hepcidin production, in familial iron deficiency anemia refractory to oral iron. <span><span class="ref-journal">Haematologica. </span>2008;<span class="ref-vol">93</span>:1473–1479.</span> [<a href="/pubmed/18603562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Haematologica&amp;title=A+mutation+in+the+TMPRSS6+gene,+encoding+a+transmembrane+serine+protease+that+suppresses+hepcidin+production,+in+familial+iron+deficiency+anemia+refractory+to+oral+iron&amp;author=M.A.+Melis&amp;author=M.+Cau&amp;author=R.+Congiu&amp;author=G.+Sole&amp;author=S.+Barella&amp;volume=93&amp;publication_year=2008&amp;pages=1473-1479&amp;pmid=18603562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C3">3. <span class="element-citation">Patel K.V., Longo D.L., Ershler W.B., Yu B., Semba R.D., Ferrucci L., Guralnik J.M. Haemoglobin concentration and the risk of death in older adults: differences by race/ethnicity in the NHANES III follow-up. <span><span class="ref-journal">Br. J. Haematol. </span>2009;<span class="ref-vol">145</span>:514–523.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2747286/">PMC free article</a>]</span> [<a href="/pubmed/19344387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Haematol.&amp;title=Haemoglobin+concentration+and+the+risk+of+death+in+older+adults:+differences+by+race/ethnicity+in+the+NHANES+III+follow-up&amp;author=K.V.+Patel&amp;author=D.L.+Longo&amp;author=W.B.+Ershler&amp;author=B.+Yu&amp;author=R.D.+Semba&amp;volume=145&amp;publication_year=2009&amp;pages=514-523&amp;pmid=19344387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C4">4. <span class="element-citation">Schechter G.P. Hemoglobin levels in African-Americans. <span><span class="ref-journal">Blood. </span>2006;<span class="ref-vol">107</span>:2208.</span> author reply 2208–2209. [<a href="/pubmed/16493011" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Hemoglobin+levels+in+African-Americans&amp;author=G.P.+Schechter&amp;volume=107&amp;publication_year=2006&amp;pages=2208&amp;pmid=16493011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C5">5. <span class="element-citation">Beutler E., Duparc S. Glucose-6-phosphate dehydrogenase deficiency and antimalarial drug development. <span><span class="ref-journal">Am. J. Trop. Med. Hyg. </span>2007;<span class="ref-vol">77</span>:779–789.</span> [<a href="/pubmed/17978087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Trop.+Med.+Hyg.&amp;title=Glucose-6-phosphate+dehydrogenase+deficiency+and+antimalarial+drug+development&amp;author=E.+Beutler&amp;author=S.+Duparc&amp;volume=77&amp;publication_year=2007&amp;pages=779-789&amp;pmid=17978087&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C6">6. <span class="element-citation">Kamatani Y., Matsuda K., Okada Y., Kubo M., Hosono N., Daigo Y., Nakamura Y., Kamatani N. Genome-wide association study of hematological and biochemical traits in a Japanese population. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:210–215.</span> [<a href="/pubmed/20139978" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+of+hematological+and+biochemical+traits+in+a+Japanese+population&amp;author=Y.+Kamatani&amp;author=K.+Matsuda&amp;author=Y.+Okada&amp;author=M.+Kubo&amp;author=N.+Hosono&amp;volume=42&amp;publication_year=2010&amp;pages=210-215&amp;pmid=20139978&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C7">7. <span class="element-citation">Soranzo N., Spector T.D., Mangino M., Kühnel B., Rendon A., Teumer A., Willenborg C., Wright B., Chen L., Li M., et al.  A genome-wide meta-analysis identifies 22 loci associated with eight hematological parameters in the HaemGen consortium. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1182–1190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3108459/">PMC free article</a>]</span> [<a href="/pubmed/19820697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=A+genome-wide+meta-analysis+identifies+22+loci+associated+with+eight+hematological+parameters+in+the+HaemGen+consortium&amp;author=N.+Soranzo&amp;author=T.D.+Spector&amp;author=M.+Mangino&amp;author=B.+Kühnel&amp;author=A.+Rendon&amp;volume=41&amp;publication_year=2009&amp;pages=1182-1190&amp;pmid=19820697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C8">8. <span class="element-citation">Ganesh S.K., Zakai N.A., van Rooij F.J., Soranzo N., Smith A.V., Nalls M.A., Chen M.H., Kottgen A., Glazer N.L., Dehghan A., et al.  Multiple loci influence erythrocyte phenotypes in the CHARGE Consortium. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1191–1198.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778265/">PMC free article</a>]</span> [<a href="/pubmed/19862010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Multiple+loci+influence+erythrocyte+phenotypes+in+the+CHARGE+Consortium&amp;author=S.K.+Ganesh&amp;author=N.A.+Zakai&amp;author=F.J.+van+Rooij&amp;author=N.+Soranzo&amp;author=A.V.+Smith&amp;volume=41&amp;publication_year=2009&amp;pages=1191-1198&amp;pmid=19862010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C9">9. <span class="element-citation">Lo K.S., Wilson J.G., Lange L.A., Folsom A.R., Galarneau G., Ganesh S.K., Grant S.F., Keating B.J., McCarroll S.A., Mohler E.R., III, et al.  Genetic association analysis highlights new loci that modulate hematological trait variation in Caucasians and African Americans. <span><span class="ref-journal">Hum. Genet. </span>2011;<span class="ref-vol">129</span>:307–317.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3442357/">PMC free article</a>]</span> [<a href="/pubmed/21153663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Genetic+association+analysis+highlights+new+loci+that+modulate+hematological+trait+variation+in+Caucasians+and+African+Americans&amp;author=K.S.+Lo&amp;author=J.G.+Wilson&amp;author=L.A.+Lange&amp;author=A.R.+Folsom&amp;author=G.+Galarneau&amp;volume=129&amp;publication_year=2011&amp;pages=307-317&amp;pmid=21153663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C10">10. <span class="element-citation">Kullo I.J., Ding K., Jouni H., Smith C.Y., Chute C.G. A genome-wide association study of red blood cell traits using the electronic medical record. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>:e13011.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2946914/">PMC free article</a>]</span> [<a href="/pubmed/20927387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=A+genome-wide+association+study+of+red+blood+cell+traits+using+the+electronic+medical+record&amp;author=I.J.+Kullo&amp;author=K.+Ding&amp;author=H.+Jouni&amp;author=C.Y.+Smith&amp;author=C.G.+Chute&amp;volume=5&amp;publication_year=2010&amp;pages=e13011&amp;pmid=20927387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C11">11. <span class="element-citation">Mukai M., Ikegami K., Sugiura Y., Takeshita K., Nakagawa A., Setou M. Recombinant mammalian tubulin polyglutamylase TTLL7 performs both initiation and elongation of polyglutamylation on beta-tubulin through a random sequential pathway. <span><span class="ref-journal">Biochemistry. </span>2009;<span class="ref-vol">48</span>:1084–1093.</span> [<a href="/pubmed/19152315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Recombinant+mammalian+tubulin+polyglutamylase+TTLL7+performs+both+initiation+and+elongation+of+polyglutamylation+on+beta-tubulin+through+a+random+sequential+pathway&amp;author=M.+Mukai&amp;author=K.+Ikegami&amp;author=Y.+Sugiura&amp;author=K.+Takeshita&amp;author=A.+Nakagawa&amp;volume=48&amp;publication_year=2009&amp;pages=1084-1093&amp;pmid=19152315&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C12">12. <span class="element-citation">Alves L.R., Costa E.S., Sorgine M.H., Nascimento-Silva M.C., Teodosio C., Barcena P., Castro-Faria-Neto H.C., Bozza P.T., Orfao A., Oliveira P.L., et al.  Heme-oxygenases during erythropoiesis in K562 and human bone marrow cells. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e21358.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3135583/">PMC free article</a>]</span> [<a href="/pubmed/21765894" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Heme-oxygenases+during+erythropoiesis+in+K562+and+human+bone+marrow+cells&amp;author=L.R.+Alves&amp;author=E.S.+Costa&amp;author=M.H.+Sorgine&amp;author=M.C.+Nascimento-Silva&amp;author=C.+Teodosio&amp;volume=6&amp;publication_year=2011&amp;pages=e21358&amp;pmid=21765894&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C13">13. <span class="element-citation">Feder J.N., Gnirke A., Thomas W., Tsuchihashi Z., Ruddy D.A., Basava A., Dormishian F., Domingo R., Jr., Ellis M.C., Fullan A., et al.  A novel MHC class I-like gene is mutated in patients with hereditary haemochromatosis. <span><span class="ref-journal">Nat. Genet. </span>1996;<span class="ref-vol">13</span>:399–408.</span> [<a href="/pubmed/8696333" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+novel+MHC+class+I-like+gene+is+mutated+in+patients+with+hereditary+haemochromatosis&amp;author=J.N.+Feder&amp;author=A.+Gnirke&amp;author=W.+Thomas&amp;author=Z.+Tsuchihashi&amp;author=D.A.+Ruddy&amp;volume=13&amp;publication_year=1996&amp;pages=399-408&amp;pmid=8696333&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C14">14. <span class="element-citation">Qayyum R., Snively B.M., Ziv E., Nalls M.A., Liu Y., Tang W., Yanek L.R., Lange L., Evans M.K., Ganesh S., et al.  A meta-analysis and genome-wide association study of platelet count and mean platelet volume in african americans. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3299192/">PMC free article</a>]</span> [<a href="/pubmed/22423221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+meta-analysis+and+genome-wide+association+study+of+platelet+count+and+mean+platelet+volume+in+african+americans&amp;author=R.+Qayyum&amp;author=B.M.+Snively&amp;author=E.+Ziv&amp;author=M.A.+Nalls&amp;author=Y.+Liu&amp;volume=8&amp;publication_year=2012&amp;pages=e1002491&amp;pmid=22423221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C15">15. <span class="element-citation">Beutler E., West C. Hematologic differences between African-Americans and whites: the roles of iron deficiency and alpha-thalassemia on hemoglobin levels and mean corpuscular volume. <span><span class="ref-journal">Blood. </span>2005;<span class="ref-vol">106</span>:740–745.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1895180/">PMC free article</a>]</span> [<a href="/pubmed/15790781" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Hematologic+differences+between+African-Americans+and+whites:+the+roles+of+iron+deficiency+and+alpha-thalassemia+on+hemoglobin+levels+and+mean+corpuscular+volume&amp;author=E.+Beutler&amp;author=C.+West&amp;volume=106&amp;publication_year=2005&amp;pages=740-745&amp;pmid=15790781&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C16">16. <span class="element-citation">Viprakasit V., Harteveld C.L., Ayyub H., Stanley J.S., Giordano P.C., Wood W.G., Higgs D.R. A novel deletion causing alpha thalassemia clarifies the importance of the major human alpha globin regulatory element. <span><span class="ref-journal">Blood. </span>2006;<span class="ref-vol">107</span>:3811–3812.</span> [<a href="/pubmed/16627766" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=A+novel+deletion+causing+alpha+thalassemia+clarifies+the+importance+of+the+major+human+alpha+globin+regulatory+element&amp;author=V.+Viprakasit&amp;author=C.L.+Harteveld&amp;author=H.+Ayyub&amp;author=J.S.+Stanley&amp;author=P.C.+Giordano&amp;volume=107&amp;publication_year=2006&amp;pages=3811-3812&amp;pmid=16627766&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C17">17. <span class="element-citation">Beutler E. G6PD deficiency. <span><span class="ref-journal">Blood. </span>1994;<span class="ref-vol">84</span>:3613–3636.</span> [<a href="/pubmed/7949118" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=G6PD+deficiency&amp;author=E.+Beutler&amp;volume=84&amp;publication_year=1994&amp;pages=3613-3636&amp;pmid=7949118&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C18">18. <span class="element-citation">Nakhaee A., Dabiri S., Noora M. Survey of the prevalence of glucose-6-phosphate dehydrogenase (G6PD) deficiency in admitted men for premarriage tests in Zahedan-Iran Reference Laboratory. <span><span class="ref-journal">Zahedan J. Res. Med. Sci. </span>2009;<span class="ref-vol">11</span>:0–0.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Zahedan+J.+Res.+Med.+Sci.&amp;title=Survey+of+the+prevalence+of+glucose-6-phosphate+dehydrogenase+(G6PD)+deficiency+in+admitted+men+for+premarriage+tests+in+Zahedan-Iran+Reference+Laboratory&amp;author=A.+Nakhaee&amp;author=S.+Dabiri&amp;author=M.+Noora&amp;volume=11&amp;publication_year=2009&amp;pages=0-0&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C19">19. <span class="element-citation">Chishti A.H. Function of p55 and its nonerythroid homologues. <span><span class="ref-journal">Curr. Opin. Hematol. </span>1998;<span class="ref-vol">5</span>:116–121.</span> [<a href="/pubmed/9570704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Hematol.&amp;title=Function+of+p55+and+its+nonerythroid+homologues&amp;author=A.H.+Chishti&amp;volume=5&amp;publication_year=1998&amp;pages=116-121&amp;pmid=9570704&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C20">20. <span class="element-citation">Watt S.M., Buhring H.J., Rappold I., Chan J.Y., Lee-Prudhoe J., Jones T., Zannettino A.C., Simmons P.J., Doyonnas R., Sheer D., et al.  CD164, a novel sialomucin on CD34(+) and erythroid subsets, is located on human chromosome 6q21. <span><span class="ref-journal">Blood. </span>1998;<span class="ref-vol">92</span>:849–866.</span> [<a href="/pubmed/9680353" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=CD164,+a+novel+sialomucin+on+CD34(+)+and+erythroid+subsets,+is+located+on+human+chromosome+6q21&amp;author=S.M.+Watt&amp;author=H.J.+Buhring&amp;author=I.+Rappold&amp;author=J.Y.+Chan&amp;author=J.+Lee-Prudhoe&amp;volume=92&amp;publication_year=1998&amp;pages=849-866&amp;pmid=9680353&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C21">21. <span class="element-citation">Forde S., Tye B.J., Newey S.E., Roubelakis M., Smythe J., McGuckin C.P., Pettengell R., Watt S.M. Endolyn (CD164) modulates the CXCL12-mediated migration of umbilical cord blood CD133+ cells. <span><span class="ref-journal">Blood. </span>2007;<span class="ref-vol">109</span>:1825–1833.</span> [<a href="/pubmed/17077324" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Endolyn+(CD164)+modulates+the+CXCL12-mediated+migration+of+umbilical+cord+blood+CD133++cells&amp;author=S.+Forde&amp;author=B.J.+Tye&amp;author=S.E.+Newey&amp;author=M.+Roubelakis&amp;author=J.+Smythe&amp;volume=109&amp;publication_year=2007&amp;pages=1825-1833&amp;pmid=17077324&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C22">22. <span class="element-citation">Klingmuller U., Wu H., Hsiao J.G., Toker A., Duckworth B.C., Cantley L.C., Lodish H.F. Identification of a novel pathway important for proliferation and differentiation of primary erythroid progenitors. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>1997;<span class="ref-vol">94</span>:3016–3021.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC20314/">PMC free article</a>]</span> [<a href="/pubmed/9096338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Identification+of+a+novel+pathway+important+for+proliferation+and+differentiation+of+primary+erythroid+progenitors&amp;author=U.+Klingmuller&amp;author=H.+Wu&amp;author=J.G.+Hsiao&amp;author=A.+Toker&amp;author=B.C.+Duckworth&amp;volume=94&amp;publication_year=1997&amp;pages=3016-3021&amp;pmid=9096338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C23">23. <span class="element-citation">Gobbi G., Mirandola P., Sponzilli I., Micheloni C., Malinverno C., Cocco L., Vitale M. Timing and expression level of protein kinase C epsilon regulate the megakaryocytic differentiation of human CD34 cells. <span><span class="ref-journal">Stem Cells. </span>2007;<span class="ref-vol">25</span>:2322–2329.</span> [<a href="/pubmed/17569788" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stem+Cells&amp;title=Timing+and+expression+level+of+protein+kinase+C+epsilon+regulate+the+megakaryocytic+differentiation+of+human+CD34+cells&amp;author=G.+Gobbi&amp;author=P.+Mirandola&amp;author=I.+Sponzilli&amp;author=C.+Micheloni&amp;author=C.+Malinverno&amp;volume=25&amp;publication_year=2007&amp;pages=2322-2329&amp;pmid=17569788&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C24">24. <span class="element-citation">Mirandola P., Gobbi G., Ponti C., Sponzilli I., Cocco L., Vitale M. PKC epsilon controls protection against TRAIL in erythroid progenitors. <span><span class="ref-journal">Blood. </span>2006;<span class="ref-vol">107</span>:508–513.</span> [<a href="/pubmed/16166586" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=PKC+epsilon+controls+protection+against+TRAIL+in+erythroid+progenitors&amp;author=P.+Mirandola&amp;author=G.+Gobbi&amp;author=C.+Ponti&amp;author=I.+Sponzilli&amp;author=L.+Cocco&amp;volume=107&amp;publication_year=2006&amp;pages=508-513&amp;pmid=16166586&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C25">25. <span class="element-citation">Kollia P., Noguchi C.T., Fibach E., Loukopoulos D., Schechter A.N. Modulation of globin gene expression in cultured erythroid precursors derived from normal individuals: transcriptional and posttranscriptional regulation by hemin. <span><span class="ref-journal">Proc. Assoc. Am. Phys. </span>1997;<span class="ref-vol">109</span>:420–428.</span> [<a href="/pubmed/9220539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Assoc.+Am.+Phys.&amp;title=Modulation+of+globin+gene+expression+in+cultured+erythroid+precursors+derived+from+normal+individuals:+transcriptional+and+posttranscriptional+regulation+by+hemin&amp;author=P.+Kollia&amp;author=C.T.+Noguchi&amp;author=E.+Fibach&amp;author=D.+Loukopoulos&amp;author=A.N.+Schechter&amp;volume=109&amp;publication_year=1997&amp;pages=420-428&amp;pmid=9220539&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C26">26. <span class="element-citation">Melefors O., Goossen B., Johansson H.E., Stripecke R., Gray N.K., Hentze M.W. Translational control of 5-aminolevulinate synthase mRNA by iron-responsive elements in erythroid cells. <span><span class="ref-journal">J. Biol. Chem. </span>1993;<span class="ref-vol">268</span>:5974–5978.</span> [<a href="/pubmed/8449958" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Translational+control+of+5-aminolevulinate+synthase+mRNA+by+iron-responsive+elements+in+erythroid+cells&amp;author=O.+Melefors&amp;author=B.+Goossen&amp;author=H.E.+Johansson&amp;author=R.+Stripecke&amp;author=N.K.+Gray&amp;volume=268&amp;publication_year=1993&amp;pages=5974-5978&amp;pmid=8449958&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C27">27. <span class="element-citation">Rosenbloom K.R., Dreszer T.R., Long J.C., Malladi V.S., Sloan C.A., Raney B.J., Cline M.S., Karolchik D., Barber G.P., Clawson H., et al.  <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">Vol. 40</span>:D912–D917.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245183/">PMC free article</a>]</span> [<a href="/pubmed/22075998" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;author=K.R.+Rosenbloom&amp;author=T.R.+Dreszer&amp;author=J.C.+Long&amp;author=V.S.+Malladi&amp;author=C.A.+Sloan&amp;volume=Vol.+40&amp;publication_year=2012&amp;pages=D912-D917&amp;pmid=22075998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C28">28. <span class="element-citation">Zakai N.A., McClure L.A., Prineas R., Howard G., McClellan W., Holmes C.E., Newsome B.B., Warnock D.G., Audhya P., Cushman M. Correlates of anemia in American blacks and whites: the REGARDS Renal Ancillary Study. <span><span class="ref-journal">Am. J. Epidemiol. </span>2009;<span class="ref-vol">169</span>:355–364.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720717/">PMC free article</a>]</span> [<a href="/pubmed/19066309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Correlates+of+anemia+in+American+blacks+and+whites:+the+REGARDS+Renal+Ancillary+Study&amp;author=N.A.+Zakai&amp;author=L.A.+McClure&amp;author=R.+Prineas&amp;author=G.+Howard&amp;author=W.+McClellan&amp;volume=169&amp;publication_year=2009&amp;pages=355-364&amp;pmid=19066309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C29">29. <span class="element-citation">Reiner A.P., Lettre G., Nalls M.A., Ganesh S.K., Mathias R., Austin M.A., Dean E., Arepalli S., Britton A., Chen Z., et al.  Genome-wide association study of white blood cell count in 16,388 African Americans: the continental origins and genetic epidemiology network (COGENT) <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128101/">PMC free article</a>]</span> [<a href="/pubmed/21738479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+study+of+white+blood+cell+count+in+16,388+African+Americans:+the+continental+origins+and+genetic+epidemiology+network+(COGENT)&amp;author=A.P.+Reiner&amp;author=G.+Lettre&amp;author=M.A.+Nalls&amp;author=S.K.+Ganesh&amp;author=R.+Mathias&amp;volume=7&amp;publication_year=2011&amp;pages=e1002108&amp;pmid=21738479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C30">30. <span class="element-citation">Sankaran V.G., Ludwig L.S., Sicinska E., Xu J., Bauer D.E., Eng J.C., Patterson H.C., Metcalf R.A., Natkunam Y., Orkin S.H., et al.  Cyclin D3 coordinates the cell cycle during differentiation to regulate erythrocyte size and number. <span><span class="ref-journal">Genes Dev. </span>2012;<span class="ref-vol">26</span>:2075–2087.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3444733/">PMC free article</a>]</span> [<a href="/pubmed/22929040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Cyclin+D3+coordinates+the+cell+cycle+during+differentiation+to+regulate+erythrocyte+size+and+number&amp;author=V.G.+Sankaran&amp;author=L.S.+Ludwig&amp;author=E.+Sicinska&amp;author=J.+Xu&amp;author=D.E.+Bauer&amp;volume=26&amp;publication_year=2012&amp;pages=2075-2087&amp;pmid=22929040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C31">31. <span class="element-citation">van der Harst P., Zhang W., Mateo Leach I., Rendon A., Verweij N., Sehmi J., Paul D.S., Elling U., Allayee H., Li X., et al.  Seventy-five genetic loci influencing the human red blood cell. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">492</span>:369–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3623669/">PMC free article</a>]</span> [<a href="/pubmed/23222517" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Seventy-five+genetic+loci+influencing+the+human+red+blood+cell&amp;author=P.+van+der+Harst&amp;author=W.+Zhang&amp;author=I.+Mateo+Leach&amp;author=A.+Rendon&amp;author=N.+Verweij&amp;volume=492&amp;publication_year=2012&amp;pages=369-375&amp;pmid=23222517&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C32">32. <span class="element-citation">Chen M.H., Yang Q. GWAF: an R package for genome-wide association analyses with family data. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:580–581.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852219/">PMC free article</a>]</span> [<a href="/pubmed/20040588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GWAF:+an+R+package+for+genome-wide+association+analyses+with+family+data&amp;author=M.H.+Chen&amp;author=Q.+Yang&amp;volume=26&amp;publication_year=2010&amp;pages=580-581&amp;pmid=20040588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C33">33. <span class="element-citation">Pe'er I., Yelensky R., Altshuler D., Daly M.J. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I.+Pe'er&amp;author=R.+Yelensky&amp;author=D.+Altshuler&amp;author=M.J.+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C34">34. <span class="element-citation">Tang H., Coram M., Wang P., Zhu X., Risch N. Reconstructing genetic ancestry blocks in admixed individuals. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">79</span>:1–12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1474129/">PMC free article</a>]</span> [<a href="/pubmed/16773560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Reconstructing+genetic+ancestry+blocks+in+admixed+individuals&amp;author=H.+Tang&amp;author=M.+Coram&amp;author=P.+Wang&amp;author=X.+Zhu&amp;author=N.+Risch&amp;volume=79&amp;publication_year=2006&amp;pages=1-12&amp;pmid=16773560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT087C35">35. <span class="element-citation">Tang H., Siegmund D.O., Johnson N.A., Romieu I., London S.J. Joint testing of genotype and ancestry association in admixed families. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:783–791.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3103820/">PMC free article</a>]</span> [<a href="/pubmed/21031451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Joint+testing+of+genotype+and+ancestry+association+in+admixed+families&amp;author=H.+Tang&amp;author=D.O.+Siegmund&amp;author=N.A.+Johnson&amp;author=I.+Romieu&amp;author=S.J.+London&amp;volume=34&amp;publication_year=2010&amp;pages=783-791&amp;pmid=21031451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3658166&amp;issue-id=223066&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139702662739312title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm139702661394384title">ACKNOWLEDGMENTS</a></li><li><a href="#idm139702660167728title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3658166/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3658166/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3658166/pdf/ddt087.pdf">PDF (314K)</a></li> | <li><a href="#" data-citationid="PMC3658166" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3658166%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3658166%2F&amp;text=Genome-wide%20association%20analysis%20of%20red%20blood%20cell%20traits%20in%20African%20Americans%3A%20the%20COGENT%20Network" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3658166%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23446634" data-aiid="3658166" data-aid="3658166" data-iid="223066" data-domainid="740" data-domain="hmg" data-accid="PMC3658166" data-md5="f78f08e61d4152d2802083fbb4408107"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:21:44-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89C91EDD5543B100000000046302D1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3658166%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3658166/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>