<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3576836/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Two-marker association tests yield new disease associations for coronary artery disease and hypertension</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3576836/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human genetics"><meta name="citation_title" content="Two-marker association tests yield new disease associations for coronary artery disease and hypertension"><meta name="citation_authors" content="Thomas P. Slavin, Tao Feng, Audrey Schnell, Xiaofeng Zhu, Robert C. Elston"><meta name="citation_date" content="December 2011"><meta name="citation_issue" content="6"><meta name="citation_volume" content="130"><meta name="citation_firstpage" content="725"><meta name="citation_doi" content="10.1007/s00439-011-1009-6"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3576836/?report=abstract"><meta name="citation_pmid" content="21626137"><meta name="DC.Title" content="Two-marker association tests yield new disease associations for coronary artery disease and hypertension"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Thomas P. Slavin"><meta name="DC.Contributor" content="Tao Feng"><meta name="DC.Contributor" content="Audrey Schnell"><meta name="DC.Contributor" content="Xiaofeng Zhu"><meta name="DC.Contributor" content="Robert C. Elston"><meta name="DC.Date" content="2011 Dec"><meta name="DC.Identifier" content="10.1007/s00439-011-1009-6"><meta name="DC.Language" content="en"><meta property="og:title" content="Two-marker association tests yield new disease associations for coronary artery disease and hypertension"><meta property="og:type" content="article"><meta property="og:description" content="It has been postulated that multiple-marker methods may have added ability, over single-marker methods, to detect genetic variants associated with disease. The Wellcome Trust Case Control Consortium (WTCCC) provided the first successful large genome-wide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3576836/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3576836/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3576836/pdf/nihms439840.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DAABBDD51FE2100000000086E055C.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3576836%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3576836</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Genet</a></span>. Author manuscript; available in PMC 2013 Feb 20.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21626137" target="pmc_ext" ref="reftype=publisher&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Hum Genet. 2011 Dec; 130(6): 725–733. </span></a></div><span class="fm-vol-iss-date">Published online 2011 May 28. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00439-011-1009-6" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00439-011-1009-6</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3576836</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS439840</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21626137">21626137</a></div></div></div></div><h1 class="content-title">Two-marker association tests yield new disease associations for coronary artery disease and hypertension</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Slavin%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137" class="affpopup" co-rid="_co_idm140310718759184" co-class="co-affbox">Thomas P. Slavin</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup> <a href="/pubmed/?term=Feng%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137" class="affpopup" co-rid="_co_idm140310766130752" co-class="co-affbox">Tao Feng</a>, <a href="/pubmed/?term=Schnell%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137" class="affpopup" co-rid="_co_idm140310766128736" co-class="co-affbox">Audrey Schnell</a>, <a href="/pubmed/?term=Zhu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137" class="affpopup" co-rid="_co_idm140310720025008" co-class="co-affbox">Xiaofeng Zhu</a>, and  <a href="/pubmed/?term=Elston%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137" class="affpopup" co-rid="_co_idm140310720289680" co-class="co-affbox">Robert C. Elston</a></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140310718759184"><h3 class="no_margin">Thomas P. Slavin</h3><p>Department of Genetics, Center for Human Genetics, University Hospitals of Cleveland and Case Western Reserve University, 10524 Euclid Avenue, Cleveland, OH, USA</p><p>Hawai’i Community Genetics, 1441 Kapi‘olani Blvd., Ste 1800, Honolulu, HI 96814, USA, <a href="mailto:dev@null" data-email="gro.inaloipak@nivals.samoht" class="oemail">gro.inaloipak@nivals.samoht</a></p><div>Find articles by <a href="/pubmed/?term=Slavin%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137">Thomas P. Slavin</a></div></div><div id="_co_idm140310766130752"><h3 class="no_margin">Tao Feng</h3><p>Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH 44106, USA</p><p>Department of Mathematics, Heilongjiang University, Harbin 150086, China</p><div>Find articles by <a href="/pubmed/?term=Feng%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137">Tao Feng</a></div></div><div id="_co_idm140310766128736"><h3 class="no_margin">Audrey Schnell</h3><p>Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH 44106, USA</p><div>Find articles by <a href="/pubmed/?term=Schnell%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137">Audrey Schnell</a></div></div><div id="_co_idm140310720025008"><h3 class="no_margin">Xiaofeng Zhu</h3><p>Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH 44106, USA</p><div>Find articles by <a href="/pubmed/?term=Zhu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137">Xiaofeng Zhu</a></div></div><div id="_co_idm140310720289680"><h3 class="no_margin">Robert C. Elston</h3><p>Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH 44106, USA</p><div>Find articles by <a href="/pubmed/?term=Elston%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21626137">Robert C. Elston</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140310732777904_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140310732777904_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140310732777904_ai" style="display:none"><div><span class="fm-affl">Thomas P. Slavin, </span><span class="fm-affl" id="A1" lang="en"> Department of Genetics, Center for Human Genetics, University Hospitals of Cleveland and Case Western Reserve University, 10524 Euclid Avenue, Cleveland, OH, USA;</span><span class="fm-affl" id="A2" lang="en"> Hawai’i Community Genetics, 1441 Kapi‘olani Blvd., Ste 1800, Honolulu, HI 96814, USA, <a href="mailto:dev@null" data-email="gro.inaloipak@nivals.samoht" class="oemail">gro.inaloipak@nivals.samoht</a>;</span></div><div><a href="#idm140310732777904aff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140310732777904_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21626137" target="pmc_ext" ref="reftype=publisher&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3576836/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>spuul3.</strong></div><div><a href="/pmc/articles/PMC3576836/bin/NIHMS439840-supplement-spuul3.doc" data-ga-action="click_feat_suppl">NIHMS439840-supplement-spuul3.doc</a><span style="color:gray"> (115K)</span></div><div class="small guid">GUID:&nbsp;9CF0D8BC-E69C-4BCA-8B2C-6AE01C782D6A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>suppl1.</strong></div><div><a href="/pmc/articles/PMC3576836/bin/NIHMS439840-supplement-suppl1.doc" data-ga-action="click_feat_suppl">NIHMS439840-supplement-suppl1.doc</a><span style="color:gray"> (87K)</span></div><div class="small guid">GUID:&nbsp;0D65E4C5-F7DD-4780-B090-55E740C1D385</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>suppl2.</strong></div><div><a href="/pmc/articles/PMC3576836/bin/NIHMS439840-supplement-suppl2.doc" data-ga-action="click_feat_suppl">NIHMS439840-supplement-suppl2.doc</a><span style="color:gray"> (49K)</span></div><div class="small guid">GUID:&nbsp;87A68646-1EAE-48A1-846B-18C55CD8A7EA</div></div></dd></dl></div><div id="idm140310726480672" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140310726480672title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">It has been postulated that multiple-marker methods may have added ability, over single-marker methods, to detect genetic variants associated with disease. The Wellcome Trust Case Control Consortium (WTCCC) provided the first successful large genome-wide association studies (GWAS) which included single-marker association analyses for seven common complex diseases. Of those signals detected, only one was associated with coronary artery disease (CAD), and none were identified for hypertension (HTN). Our objective was to find additional genetic associations and pathways for cardiovascular disease by examining the WTCCC data for variants associated with CAD and HTN using two-marker testing methods. We applied two-marker association testing to the WTCCC dataset, which includes ~2,000 affected individuals with each disorder, and a shared pool of ~3,000 controls, all genotyped using Affymetrix GeneChip 500 K arrays. For CAD, we detected single nucleotide polymorphisms (SNP) pairs in three genes showing genome-wide significance: <em>HFE2</em>, <em>STK32B</em>, and <em>DIPC2</em>. The most notable SNP pairs in a non-protein-coding region were at 9p21, a known major CAD-associated region. For HTN, we detected SNP pairs in five genes: <em>GPR39</em>, <em>XRCC4</em>, <em>MYO6</em>, <em>ZFAT</em>, and <em>MACROD2</em>. Four further associated SNP pair regions were at least 70 kb from any known gene. We have shown that novel, multiple-marker, statistical methods can be of use in finding variants in GWAS. We describe many new, associated variants for both CAD and HTN and describe their known genetic mechanisms.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">In spite of the recent success of genome-wide association studies (GWAS) in searching for common or rare variants of complex diseases, it has been observed that the genetic variants identified through GWAS have accounted for only a small proportion of the presumed genotypic variation and hence, many variants remain to be discovered (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898154">McCarthy et al. 2008</a>).</p><p id="P3">The SNP chips used in GWAS provide a good coverage of the human genome by genotyping hundreds of thousands of SNPs at a time. Although the coverage provided for common variants is good, it is still unclear exactly how many SNPs are in the genome. Therefore, it is more likely that the variants of interest will not be directly observed and it is important to develop the methods of searching for variants without typing the variant directly.</p><p id="P4">Two categories of tests have been applied to search for the disease association in GWAS: single-marker association tests, which examine a single SNP at a time and multiple-marker tests, which examine multiple SNPs simultaneously (<a href="/pmc/articles/PMC3576836/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figs. 1</span></a>, <a href="/pmc/articles/PMC3576836/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,2).</span></span><span>2</span></a>). The Wellcome Trust Case Control Consortium (WTCCC) provided the first successful large comprehensive GWAS, performing single-marker association data analysis for seven complex diseases: bipolar disease (BD), Crohn’s disease (CD), rheumatoid arthritis (RA), type 1 diabetes (T1D), type 2 diabetes (T2D), CAD, and HTN—with 2,000 cases for each disease and 3,000 shared common controls (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898172">Wellcome Trust Case Control Consortium 2007</a>). Twenty-four independent association signals had a genome-wide significance of <em>P</em> &lt;5 × 10<sup>−7</sup>, many of which have been replicated in the later independent studies (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898153">Wellcome Trust Case Control Consortium 2007</a>, <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898160">Zhu et al. 2010</a>, <a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898167">Browning and Browning 2008</a>). Of those signals detected in the original WTCCC, only one was associated with CAD, none were identified for HTN (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898174">Wellcome Trust Case Control Consortium 2007</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3576836/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140310715461920" class="figure"><a href="/pmc/articles/PMC3576836/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3576836_nihms439840f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms439840f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3576836/bin/nihms439840f1.jpg"></a></div><div id="largeobj_idm140310715461920" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3576836/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3576836/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="P37">Hypothetical SNP chip typing for single-marker method. <em>Straight lines</em> depict a pair of chromosomes. SNP positions are represented by the <em>diamonds</em>. <strong>a</strong> shows a normal tagged SNP (pointed at by <em>arrow</em>), <strong>b</strong> and <strong>c</strong> show the disease scenarios. Ideally, the disease variant of interest (<em>shaded diamond</em>) would be directly typed, as in <strong>b</strong>; however, as current platforms likely only type a small portion of possible SNPs, it is more likely that the disease causing SNP will not be tagged directly, as in <strong>c</strong>
</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3576836/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140310720545984" class="figure"><a href="/pmc/articles/PMC3576836/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3576836_nihms439840f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms439840f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3576836/bin/nihms439840f2.jpg"></a></div><div id="largeobj_idm140310720545984" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3576836/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3576836/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="P38">Hypothetical SNP chip typing using a two-marker method. Prior to typing, phased diplotypes for the AA, Aa, or aa genotypes and BB, Bb, or bb genotypes could include sixteen possibilities. However, only the four heterozygous combinations (<em>light gray shading</em>) are informative about the phase of the diplotype. The reduction in possible diplotypes when the flanking markers are both heterozygous makes the information gained more powerful. This concept is also depicted in the <em>bottom figure</em>, where <em>straight lines</em> depict a pair of chromosomes, <em>diamonds</em> indicate hypothetical SNP positions along 5 kb of DNA (<em>horizontal lines</em>), the <em>shaded diamond</em> indicates the disease SNP of interest, and <em>arrows</em> indicate the SNPs typed by the chip platform. Thus, when the markers are close enough together so that they are in linkage disequilibrium, two-marker association testing methods have increased ability to find non-typed variants: both SNPs near the disease causing variant combine to reach genome-wide significance </p></div></div></div><p id="P5" class="p p-last">One possible explanation for the limited results for CAD and HTN may have been due to lack of statistical power provided by single-marker analysis to detect the disease-susceptibility variants. The purpose of this study is to find additional genetic associations and pathways for cardiovascular disease by examining the WTCCC data for variants associated with CAD and HTN using two-marker testing methods as described by<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898142">Kim et al. (2010)</a>. We show that these novel statistical methods can be of use in finding variants of genome-wide significance for complex diseases. We describe multiple, new, disease associations for both CAD and HTN and describe the known genetic mechanisms that may be relevant.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Methods</h2><p id="P6" class="p p-first">Detailed description of the study samples can be found in the original WTCCC GWAS paper (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898139">Wellcome Trust Case Control Consortium 2007</a>). In brief, the WTCCC dataset includes seven major complex diseases: BD, CD, RA, T1D, T2D, CAD, and HTN; each has ~2,000 affected individuals and a shared pool of ~3,000 controls (which consist of a 1958 Birth Cohort and a recently recruited UK Blood Service sample). The majority of subjects were of European ancestry. All the individuals were genotyped using Affymetrix GeneChip 500K arrays. We downloaded the genotype data called with the algorithm CHIAMO only for the CAD and HTN disease cases and the shared controls from the WTCCC website.</p><div id="S3" class="sec"><h3 id="S3title">CAD phenotype description</h3><p id="P7" class="p p-first-last">CAD cases had a validated history of myocardial infarction, coronary artery bypass surgery, and/or percutaneous coronary angioplasty prior to their 66th birthday. Verification was required either from hospital records or from a primary care physician. More details about the sample collection and ascertainment can be found in the original WTCCC GWAS paper (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898169">Wellcome Trust Case Control Consortium 2007</a>).</p></div><div id="S4" class="sec"><h3 id="S4title">HT phenotype description</h3><p id="P8" class="p p-first-last">Cases of HTN had a history of HTN diagnosed before 60 years of age, with blood pressure (BP) levels of 150/100 mmHg (if based on a single reading), or if the mean of three readings was &gt;145/95 mmHg. Hypertensive individuals were excluded if they: self-reportedly consumed &gt;21 units of alcohol per week, had diabetes, had intrinsic renal disease, or had a history of secondary hypertension or other co-morbid condition. Cases were not screened for rare, known monogenic causes of HTN. Recruitment was focused on individuals with a body mass index (BMI) under 30 kg/m<sup>2</sup>. More details in the sample collection and ascertainment can be found in the original WTCCC GWAS paper (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898138">Wellcome Trust Case Control Consortium 2007</a>).</p></div><div id="S5" class="sec"><h3 id="S5title">Quality controls</h3><p id="P9" class="p p-first-last">The majority of subjects had at least one sibling also affected, whether by CAD or HTN, but only one subject from each family was included in the ~2,000 cases. Individuals were further dropped in the WTCCC study because of evidence of non-European ancestry or poor call rates (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898166">Wellcome Trust Case Control Consortium 2007</a>). We applied the following criteria to call SNPs: (1) CHIAMO probability &gt;0.99; (2) Hardy–Weinberg Equilibrium exact test <em>P</em> value &gt;5.7 × 10<sup>−7</sup> in controls; (3) allele frequency difference test based on 1 degree of freedom (<em>df</em>) trend test <em>P</em> value &gt;5.7 × 10<sup>−7</sup>, or genotype frequency difference based on 2 df general test &gt;5.7 × 10<sup>−7</sup>, between the two control groups (the 1958) Birth Cohort and the recently recruited UK Blood Service sample); (4) missing genotype proportions &lt;1% and (5) minor allele frequencies &gt;1%. We dropped SNPs with bad genotype calling, as suggested in the original WTCCC analysis (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898151">Wellcome Trust Case Control Consortium 2007</a>). This resulted in a total of 407,576 SNPs that were used in this genome-wide scan for CAD and 405,022 SNPs for HTN.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Association test methods</h3><p id="P10" class="p p-first">As the WTCCC used the Cochran–Armitage trend test (and not a logistic regression model) for single-marker analysis, we first performed a single-marker logistic regression test on the entire dataset as in<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898165">Kim et al. (2010)</a> on the CAD and HTN WTCCC data (<a href="#SD2" rid="SD2" class=" supplementary-material">supplemental material, Table 1</a>). The model is log (μ/<sub>1 − μ</sub>) = β<sub>o</sub> + β<sub>x</sub><em>X</em><sub>i</sub>, where μ/<sub>1 − μ</sub> is the odds of being affected, β<sub>o</sub> is the intercept, β<sub>x</sub> is the additive effect of SNP X on the logit scale and <em>X</em><sub>i</sub> is the value of the i-th person’s SNP coded as the number of minor alleles in person i’s genotype. In large enough samples, this test should be identical to the trend test used by the WTCCC, and indeed gave essentially similar results in this sample.</p><p id="P11">We then applied two-marker analysis with the 2 <em>df</em> and 3 <em>df</em> models as described in<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898168">Kim et al. (2010)</a>. Test 2–2 (2 <em>df</em>) jointly tests allelic contrasts for two neighboring SNPs (SNP X and SNP Y), and test 2–3 (3 <em>df</em>) additionally tests the product of the two additively coded SNPs (i.e. the term <em>X</em><sub>i</sub><em>Y</em><sub>i</sub>). Thus, the model for test 2–2 is log (μ/<sub>1 − μ</sub>) = β<sub>o</sub> + β<sub>x</sub><em>X</em><sub>i</sub> + β<sub>y</sub><em>Y</em><sub>i</sub> and that for test 2–3 is log (μ/<sub>1 − μ</sub>) = β<sub>o</sub> + β<sub>x</sub><em>X</em><sub>i</sub> + β<sub>y</sub><em>Y</em><sub>i</sub> + β<sub>xy</sub><em>X</em><sub>i</sub><em>Y</em><sub>i</sub>. Under this latter model, we can test the coefficient β<sub>xy</sub>, and hence determine whether including this product term significantly helps us to detect associations. Note that, since we only included consecutive adjacent SNP pairs [(SNP1, SNP2), (SNP2, SNP3), (SNP3, SNP4), etc.] in the model, in each case we performed slightly fewer tests in the two-marker analyses than in the single-marker analysis. We further included as covariates in the model the first ten principal components of all the SNPs, cases and controls combined, which can control for any population structure (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898175">Zhu et al. 2008</a>, <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898149">2002</a>).</p><p id="P12" class="p p-last">Using the SNP map annotations provided by Affymetrix 5.0 GeneChip as a reference (<a href="https://www.affymetrix.com/support/technical/annotationfilesmain.affx" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.affymetrix.com/support/technical/annotationfilesmain.affx</a>), we evaluated all SNPs with a <em>P</em> value &lt;5 × 10<sup>−7</sup>, the same as used by the WTCCC. As seen in the results, tests 2–2 and 2–3 are highly correlated; therefore, performing both tests does not double the number of independent tests performed. Furthermore, since the tests, whether test 2–2 or tests 2–3, are no longer independent, but positively correlated [e.g., the tests for (SNP1, SNP2), and (SNP2, SNP3) both include testing SNP2], the effective number of independent tests was smaller. If the SNP was in a known gene, the gene was evaluated through published cardiovascular literature for a possible pathogenic link. If SNP pairs were located between two neighboring genes, we mapped them to the nearest gene and also evaluated the locus for possible published cardiovascular pathogenic links.</p></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Results</h2><p id="P13" class="p p-first">Results for the two-marker association analysis for CAD and HTN using the 2 and 3 <em>df</em> models are shown in <a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and <a href="/pmc/articles/PMC3576836/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2.</span></span><span>2</span></a>. The 2 <em>df</em> model yielded the exact same loci with genome-wide significance as the 3 <em>df</em> model for both CAD and HTN, with the exception of one additional SNP pair locus for CAD (<a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, asterisk). The term representing the extra <em>df</em> in the 3 <em>df</em> model was calculated and statistically significant for only the CAD SNP pair on chromosome 10 (<a href="#SD3" rid="SD3" class=" supplementary-material">supplemental material, Table 2</a>) and therefore, we choose to display only the 2 <em>df</em> test Manhattan plot results for CAD and HTN (<a href="/pmc/articles/PMC3576836/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3576836/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140310731495360" class="figure"><a href="/pmc/articles/PMC3576836/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3576836_nihms439840f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms439840f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3576836/bin/nihms439840f3.jpg"></a></div><div id="largeobj_idm140310731495360" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3576836/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3576836/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="P39">Manhattan plots of 2 <em>df</em> test results for CAD and HTN. Two <em>df</em> Manhattan plots for CAD and HTN are as marked. Three <em>df</em> testing methods yielded the same loci with the exception of one additional chromosome 10 SNP pair for CAD marked by the <em>asterisk</em> in <a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a></p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P41">Two-marker logistic model association test results for coronary artery disease</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140310726422768" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Ch</th><th rowspan="1" colspan="1" valign="top" align="left">SNP1</th><th rowspan="1" colspan="1" valign="top" align="left">MAF1</th><th rowspan="1" colspan="1" valign="top" align="left">SNP2</th><th rowspan="1" colspan="1" valign="top" align="left">MAF2</th><th rowspan="1" colspan="1" valign="top" align="left">2 <em>df P</em> value</th><th rowspan="1" colspan="1" valign="top" align="left">3 <em>df P</em> value</th><th rowspan="1" colspan="1" valign="top" align="left">Hap</th><th rowspan="1" colspan="1" valign="top" align="left"><em>R</em><sup>2</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>D</em>′</th><th rowspan="1" colspan="1" valign="top" align="left">Pairs</th><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">CAD<br>Link?</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><em>1</em></td><td rowspan="1" colspan="1" align="left"><em>rs12091564</em></td><td rowspan="1" colspan="1" align="left"><em>0.037</em></td><td rowspan="1" colspan="1" align="left"><em>rs10218795</em></td><td rowspan="1" colspan="1" align="left"><em>0.034</em></td><td rowspan="1" colspan="1" align="left"><em>1.75 × 10<sup>−7</sup></em></td><td rowspan="1" colspan="1" align="left"><em>1.75 × 10<sup>−7</sup></em></td><td rowspan="1" colspan="1" align="left"><em>CC (R)</em></td><td rowspan="1" colspan="1" align="left"><em>0.92</em></td><td rowspan="1" colspan="1" align="left"><em>1</em></td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>1</em></td><td rowspan="1" colspan="1" align="left"><em>HFE2</em></td><td rowspan="1" colspan="1" align="left"><em>Yes</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">rs11924705</td><td rowspan="1" colspan="1" valign="top" align="left">0.423</td><td rowspan="1" colspan="1" valign="top" align="left">rs6789378</td><td rowspan="1" colspan="1" valign="top" align="left">0.416</td><td rowspan="1" colspan="1" valign="top" align="left">4.04 <em>×</em> 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.34 <em>×</em> 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">CA (<em>P</em>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.96</td><td rowspan="1" colspan="1" valign="top" align="left">0.98</td><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;1</td><td rowspan="1" colspan="1" valign="top" align="left">&gt;1 Mb from any<br>&nbsp;&nbsp;gene</td><td rowspan="1" colspan="1" valign="top" align="left">No</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>4</em></td><td rowspan="1" colspan="1" align="left"><em>rs7697839</em></td><td rowspan="1" colspan="1" align="left"><em>0.026</em></td><td rowspan="1" colspan="1" align="left"><em>rs7673097</em></td><td rowspan="1" colspan="1" align="left"><em>0.023</em></td><td rowspan="1" colspan="1" align="left"><em>2.19 × 10<sup>−11</sup></em></td><td rowspan="1" colspan="1" align="left"><em>2.19 × 10<sup>−11</sup></em></td><td rowspan="1" colspan="1" align="left"><em>GG (R)</em></td><td rowspan="1" colspan="1" align="left"><em>0.89</em></td><td rowspan="1" colspan="1" align="left"><em>1</em></td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>2</em></td><td rowspan="1" colspan="1" align="left"><em>STK32B</em></td><td rowspan="1" colspan="1" align="left"><em>No</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">rs1333048</td><td rowspan="1" colspan="1" valign="top" align="left">0.477</td><td rowspan="1" colspan="1" valign="top" align="left">rs1333049</td><td rowspan="1" colspan="1" valign="top" align="left">0.494</td><td rowspan="1" colspan="1" valign="top" align="left">7.08 <em>×</em> 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.55 <em>×</em> 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">GG (<em>P</em>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.92</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">18</td><td rowspan="1" colspan="1" valign="top" align="left">115 kb from<br>&nbsp;&nbsp;<em>CDKN2B/2A</em></td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>10</em></td><td rowspan="1" colspan="1" align="left"><em>rs2066314</em></td><td rowspan="1" colspan="1" align="left"><em>0.016</em></td><td rowspan="1" colspan="1" align="left"><em>rs2066315</em></td><td rowspan="1" colspan="1" align="left"><em>0.034</em></td><td rowspan="1" colspan="1" align="left"><em>&nbsp;&nbsp;8.6 × 10<sup>−2</sup><sup>*</sup></em></td><td rowspan="1" colspan="1" align="left"><em>1.18 × 10<sup>−8</sup></em></td><td rowspan="1" colspan="1" align="left"><em>GT (R)</em></td><td rowspan="1" colspan="1" align="left"><em>0.36</em></td><td rowspan="1" colspan="1" align="left"><em>0.87</em></td><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>1</em></td><td rowspan="1" colspan="1" align="left"><em>DIP2C</em></td><td rowspan="1" colspan="1" align="left"><em>No</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">rs1165668</td><td rowspan="1" colspan="1" valign="top" align="left">0.260</td><td rowspan="1" colspan="1" valign="top" align="left">rs1165669</td><td rowspan="1" colspan="1" valign="top" align="left">0.261</td><td rowspan="1" colspan="1" valign="top" align="left">3.05 <em>×</em> 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.32 <em>×</em> 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">GC (<em>R</em>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.98</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;1</td><td rowspan="1" colspan="1" valign="top" align="left">10 kb from<br>&nbsp;&nbsp;<em>HSP90B1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td></tr></tbody></table></div><div id="largeobj_idm140310726422768" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3576836/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P42" class="p p-first-last">Two and three degree of freedom (<em>df</em>) logistic model results for CAD. If multiple SNP pairs showed significant association with a <em>P</em> value &lt;5 × 10<sup>−7</sup>, only the lowest <em>P</em> value pair is shown. The <em>asterisk</em> for the SNP pair on chromosome (Ch) 10 denotes a SNP pair only reaching genome-wide significance with 3 <em>df</em> model testing. Mean allele frequency (MAF) refers to the MAF of SNP 1 and SNP 2, respectively. The associated risk (<em>R</em>) or protective (<em>P</em>) haplotypes (Hap) are given. Correlation (<em>R</em>
<sup>2</sup>) and linkage disequilibrium (<em>D′</em>) values for the individual SNP pairs are listed. The “<em>Pairs”</em> column displays the number of SNP pairs in the area with a <em>P</em> value &lt;5 × 10<sup>−7</sup>. Please refer to the text for specific information on the CAD linking literature. <em>Italics</em> indicate genes
</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P43">Two-marker logistic model association test results for hypertension</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140310702403056" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Ch</th><th rowspan="1" colspan="1" valign="top" align="left">SNP1</th><th rowspan="1" colspan="1" valign="top" align="left">MAF1</th><th rowspan="1" colspan="1" valign="top" align="left">SNP2</th><th rowspan="1" colspan="1" valign="top" align="left">MAF2</th><th rowspan="1" colspan="1" valign="top" align="left">2 <em>df P</em> value</th><th rowspan="1" colspan="1" valign="top" align="left">3 <em>df P</em> value</th><th rowspan="1" colspan="1" valign="top" align="left">Hap</th><th rowspan="1" colspan="1" valign="top" align="left"><em>R</em><sup>2</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>D</em>′</th><th rowspan="1" colspan="1" valign="top" align="left">Pairs</th><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">HTN<br>Link?</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">rs10496288</td><td rowspan="1" colspan="1" valign="top" align="left">0.104</td><td rowspan="1" colspan="1" valign="top" align="left">rs10496289</td><td rowspan="1" colspan="1" valign="top" align="left">0.098</td><td rowspan="1" colspan="1" valign="top" align="left">2.08 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.32 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">GC (<em>P</em>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.96</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">H: &gt; 1 Mb from any<br>&nbsp;&nbsp;gene</td><td rowspan="1" colspan="1" valign="top" align="left">No</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>2</em></td><td rowspan="1" colspan="1" align="left"><em>rs13420028</em></td><td rowspan="1" colspan="1" align="left"><em>0.200</em></td><td rowspan="1" colspan="1" align="left"><em>rs10188442</em></td><td rowspan="1" colspan="1" align="left"><em>0.204</em></td><td rowspan="1" colspan="1" align="left"><em>1.03 × 10<sup>−10</sup></em></td><td rowspan="1" colspan="1" align="left"><em>4.98 × 10<sup>−10</sup></em></td><td rowspan="1" colspan="1" align="left"><em>AA (P)</em></td><td rowspan="1" colspan="1" align="left"><em>0.97</em></td><td rowspan="1" colspan="1" align="left"><em>0.99</em></td><td rowspan="1" colspan="1" align="left"><em>1</em></td><td rowspan="1" colspan="1" align="left"><em>GPR39</em></td><td rowspan="1" colspan="1" align="left"><em>No</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">rs7735940</td><td rowspan="1" colspan="1" valign="top" align="left">0.240</td><td rowspan="1" colspan="1" valign="top" align="left">rs12522034</td><td rowspan="1" colspan="1" valign="top" align="left">0.250</td><td rowspan="1" colspan="1" valign="top" align="left">5.47 <em>×</em> 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.04 <em>×</em> 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="left">AC (<em>R</em>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.95</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">&gt;200 kb from<br>&nbsp;&nbsp;<em>RANBP3L</em></td><td rowspan="1" colspan="1" valign="top" align="left">No</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>5</em></td><td rowspan="1" colspan="1" align="left"><em>rs6452524</em></td><td rowspan="1" colspan="1" align="left"><em>0.442</em></td><td rowspan="1" colspan="1" align="left"><em>rs6887846</em></td><td rowspan="1" colspan="1" align="left"><em>0.448</em></td><td rowspan="1" colspan="1" align="left"><em>1.69 × 10<sup>−7</sup></em></td><td rowspan="1" colspan="1" align="left"><em>7.12 × 10<sup>−7</sup></em></td><td rowspan="1" colspan="1" align="left"><em>GA (P)</em></td><td rowspan="1" colspan="1" align="left"><em>0.98</em></td><td rowspan="1" colspan="1" align="left"><em>0.99</em></td><td rowspan="1" colspan="1" align="left"><em>1</em></td><td rowspan="1" colspan="1" align="left"><em>XRCC4</em></td><td rowspan="1" colspan="1" align="left"><em>No</em></td></tr><tr><td rowspan="1" colspan="1" align="left"><em>6</em></td><td rowspan="1" colspan="1" align="left"><em>rs3798440</em></td><td rowspan="1" colspan="1" align="left"><em>0.157</em></td><td rowspan="1" colspan="1" align="left"><em>rs9350602</em></td><td rowspan="1" colspan="1" align="left"><em>0.151</em></td><td rowspan="1" colspan="1" align="left"><em>3.38 × 10<sup>−10</sup></em></td><td rowspan="1" colspan="1" align="left"><em>7.43 × 10<sup>−10</sup></em></td><td rowspan="1" colspan="1" align="left"><em>AC (P)</em></td><td rowspan="1" colspan="1" align="left"><em>0.96</em></td><td rowspan="1" colspan="1" align="left"><em>1</em></td><td rowspan="1" colspan="1" align="left"><em>2</em></td><td rowspan="1" colspan="1" align="left"><em>MYO6</em></td><td rowspan="1" colspan="1" align="left"><em>No</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">rs2469997</td><td rowspan="1" colspan="1" valign="top" align="left">0.186</td><td rowspan="1" colspan="1" valign="top" align="left">rs6469823</td><td rowspan="1" colspan="1" valign="top" align="left">0.181</td><td rowspan="1" colspan="1" valign="top" align="left">3.25 <em>×</em> 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.22 <em>×</em> 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="left">GC (<em>R</em>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.97</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">70 kb from <em>NOV</em></td><td rowspan="1" colspan="1" valign="top" align="left">No<br>&nbsp;&nbsp;(?)</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>8</em></td><td rowspan="1" colspan="1" align="left"><em>rs7827545</em></td><td rowspan="1" colspan="1" align="left"><em>0.321</em></td><td rowspan="1" colspan="1" align="left"><em>rs1372662</em></td><td rowspan="1" colspan="1" align="left"><em>0.333</em></td><td rowspan="1" colspan="1" align="left"><em>1.65 × 10<sup>−44</sup></em></td><td rowspan="1" colspan="1" align="left"><em>4.11 × 10<sup>−44</sup></em></td><td rowspan="1" colspan="1" align="left"><em>CG (R)</em></td><td rowspan="1" colspan="1" align="left"><em>0.95</em></td><td rowspan="1" colspan="1" align="left"><em>1</em></td><td rowspan="1" colspan="1" align="left"><em>2</em></td><td rowspan="1" colspan="1" align="left"><em>ZFAT</em></td><td rowspan="1" colspan="1" align="left"><em>Yes</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">rs7960483</td><td rowspan="1" colspan="1" valign="top" align="left">0.331</td><td rowspan="1" colspan="1" valign="top" align="left">rs10785581</td><td rowspan="1" colspan="1" valign="top" align="left">0.338</td><td rowspan="1" colspan="1" valign="top" align="left">1.43 <em>×</em> 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.27 <em>×</em> 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">TC (<em>P</em>)</td><td rowspan="1" colspan="1" valign="top" align="left">0.97</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">~100 kb from<br>&nbsp;&nbsp;<em>AN06</em></td><td rowspan="1" colspan="1" valign="top" align="left">No</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>20</em></td><td rowspan="1" colspan="1" align="left"><em>rs200752</em></td><td rowspan="1" colspan="1" align="left"><em>0.115</em></td><td rowspan="1" colspan="1" align="left"><em>rs200759</em></td><td rowspan="1" colspan="1" align="left"><em>0.113</em></td><td rowspan="1" colspan="1" align="left"><em>6.65 × 10<sup>−09</sup></em></td><td rowspan="1" colspan="1" align="left"><em>3.34 × 10<sup>−08</sup></em></td><td rowspan="1" colspan="1" align="left"><em>TA (R)</em></td><td rowspan="1" colspan="1" align="left"><em>0.92</em></td><td rowspan="1" colspan="1" align="left"><em>0.96</em></td><td rowspan="1" colspan="1" align="left"><em>1</em></td><td rowspan="1" colspan="1" align="left"><em>MACROD2</em></td><td rowspan="1" colspan="1" align="left"><em>Yes</em></td></tr></tbody></table></div><div id="largeobj_idm140310702403056" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3576836/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="P44" class="p p-first-last">Two degree of freedom (<em>df</em>) logistic model results for HTN. If multiple SNP pairs showed significant association with a <em>P</em> value &lt;5 × 10<sup>−7</sup>, only the lowest <em>P</em> value pair is shown. Mean allele frequency (MAF) refers to the MAF of SNP 1 and SNP 2, respectively. The associated risk (<em>R</em>) or protective (<em>P</em>) haplotypes (Hap) are given. Correlation (<em>R</em>
<sup>2</sup>) and linkage disequilibrium (<em>D</em>′) values for the individual SNP pairs are listed by their respective chromosome (Ch). The “<em>Pairs</em>” column displays the number of SNP pairs in the area with a <em>P</em> value &lt;5 × 10<sup>−7</sup>. Please refer to the text for specific information on the HTN linking literature. <em>Italics</em> indicate genes. The (?) symbol is used for the SNP pair on chromosome (ch) 8, as <em>NOV</em>, which is 70 kb away from the associated SNP pair and has HTN linking literature</p></div></div></div><div id="S8" class="sec"><h3 id="S8title">Coronary artery disease results</h3><p id="P14" class="p p-first">Results of the single-marker logistic regression (test 1–1) indicated multiple SNPs for CAD in the 9p21 region. The 9p21 region was the same area associated with CAD in the original WTCCC GWAS paper (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898162">Wellcome Trust Case Control Consortium 2007</a>). This was to be expected as in large samples, the trend test is equivalent to logistic regression; however, we report here the <em>P</em> values after including the first ten principal components in the model, so there are some minor discrepancies when comparing with the <em>P</em> values reported by the WTCCC (<a href="#SD2" rid="SD2" class=" supplementary-material">supplemental material, Table 1</a>).</p><p id="P15">Using the two-marker methods, we detected SNP pairs in three genes, and three non-protein coding regions showing genome-wide significant association evidence for CAD (<a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). An additional SNP pair on chromosome 10 was found to attain genome-wide significance with the 3 <em>df</em> model, two-marker analysis (<a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, asterisk). If more than one SNP pair in a particular region showed significant association, only the lowest <em>P</em> value pair is shown. The three associated genes include: hemochromatosis type 2 (<em>HFE2</em>), serine/threonine kinase 32B (<em>STK32B</em>), and disco-interacting protein 2 homolog C (<em>DIP2C</em>).</p><p id="P16" class="p p-last">Of the other non-protein-coding regions shown in <a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, the chromosome 9p21 region was the only area reaching genome-wide significance in the original WTCCC study. Additionally, the associated non-protein-coding region on chromosome 12 is only 10 kb from heat shock protein 90 kDa beta (<em>HSP90B1</em>).</p></div><div id="S9" class="sec sec-last"><h3 id="S9title">Hypertension results</h3><p id="P17" class="p p-first">There was no SNP reaching genome-wide significance for HTN in the original WTCCC report. Similarly, no SNPs in our single-marker logistic regression (test 1–1), with or without inclusion of the principal components in the model to control for population stratification, had a <em>P</em> value &lt;5 × 10<sup>−7</sup> (<a href="#SD2" rid="SD2" class=" supplementary-material">supplemental material, Table 1</a>).</p><p id="P18" class="p p-last">Using the two-marker methods, we detected SNP pairs with significant association for HTN in five genes, and four non-protein-coding regions (<a href="/pmc/articles/PMC3576836/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). If more than one SNP pair showed significant association, only the lowest <em>P</em> value pair is shown. The five associated genes include: G protein-coupled receptor 39 (<em>GPR39</em>), X-ray repair, complementing defective, in Chinese Hamster 4 (<em>XRCC4</em>), myosin VI (<em>MYO6</em>), zinc finger and AT hook domain (<em>ZFAT</em>), and MACRO domain containing 2 (<em>MACROD2)</em>. The remaining four associated non-protein-coding regions were at least 70 kb from any known gene.</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Discussion</h2><p id="P19" class="p p-first">Since multiple forms of statistical testing methods are used in GWAS, it is important to use a model that can provide dependable, replicable results without inflation of significance. We have shown our experience using two, two-marker association tests on the WTCCC CAD and HTN data. We specifically chose tests 2–2 and 2–3, because they were previously predicted to provide reliable power for disease models in<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898150">Kim et al. (2010)</a>.</p><p id="P20">Our results show new SNPs associated with both CAD and HTN. Although some associations have been previously described, others may include new variants found by using two-marker methods. Our results did not find any overlapping SNP pairs between HTN and CAD.</p><p id="P21">For CAD, the study identified six separate genome regions with SNP pairs showing a strong association, three of which were located in genes: <em>HFE2, STK32B</em>, and <em>DIP2C. HFE2</em> is the cause of hemochromatosis type 2A, a form of juvenile hemochromatosis (JH), a disorder that can cause cardiomyopathy from iron overload (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898141">Rivard et al. 2003</a>). <em>HFE2</em> was also detected for CAD with haplotype analysis using roughly 30 SNPs at a time by<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898145">Zhu et al. (2010)</a>. As proper iron metabolism is necessary for good heart function, disruptions in this pathway may be deleterious to cardiac function. <em>HFE2</em> is also in a region associated with copy number variation (CNV), and CNV, as it relates to disease, is not well understood. Additionally, <em>HFE2</em> is expressed in the heart.</p><p id="P22">No cardiovascular link was determined for the following: <em>STK32B</em>, which belongs to the serine threonine protein kinase superfamily, and <em>DIP2C</em>, which has an unknown genetic function but moderate expression in most fetal and adult tissues (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898158">Nagase et al. 1999</a>). <em>DIP2C</em> is in a region associated with CNV.</p><p id="P23">The most notable SNP pairs that were in non-protein-coding regions include those on chromosome 9, near the coding sequences of two cyclin dependent kinase inhibitors, <em>CDKN2A</em> and <em>CDKN2B</em>. We found 20 SNP pairs with <em>P</em> values &lt;5 × 10<sup>−7</sup> in the area. This was also the only area of genome-wide significance in both the WTCCC and our single-marker logistic regression analysis. As the 9p21 locus has been replicated in multiple studies (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898170">Wellcome Trust Case Control Consortium 2007</a>; <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898171">Horne et al. 2008</a>; <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898147">Samani and Schunkert 2008</a>), there is little doubt that it is a major CAD locus.</p><p id="P24">The associated SNP pair on chromosome 12 is only 10 kb from <em>HSP90B1</em>. <em>HSP90B1</em> is a highly conserved molecular chaperone protein. <em>HSP90B1</em> has been shown to form a chaperone complex that interacts with p38, an important stress activated protein kinase involved in gene regulation, proliferation, differentiation, and cell death regulation in the cardiomyocyte (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898146">Ota et al. 2010</a>). It also has been shown to associate with nitric oxide synthase (NOS3) to aid rats in myocardial ischemia–reperfusion injury resistance (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898163">An et al. 2009</a>). The other reported associated SNP pair on chromosome 3 for CAD was far removed from any known gene.</p><p id="P25">For HTN, we detected SNP pairs in five genes and four non-protein-coding regions reaching genome-wide significance using the two-marker methods. The five associated genes are: <em>ZFAT</em>, <em>MACROD2</em>, <em>GPR39</em>, <em>XRCC4</em>, and <em>MYO6</em>. Of note, <em>ZFAT</em> conferred the lowest <em>P</em> value in our entire study. It is a newly described immune regulatory gene and has previously been linked to HTN (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898173">Zhu et al. 2010</a>), although the mechanism is unclear. Furthermore, the 8q22–23 locus has shown significant linkage (i.e., genomic location) evidence to essential hypertension (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898176">Ciullo et al. 2006</a>). <em>MACROD2</em>, a newly described gene with unclear function, was also found to be associated with HTN (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898136">Zhu et al. 2010</a>). In addition, it has been associated with extreme obesity (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898152">Cotsapas et al. 2009</a>).</p><p id="P26">Genes without literature evidence of cardiovascular involvement include <em>GPR39</em>, <em>XRCC4</em>, and <em>MYO6. GPR39</em> has an unknown protein function, but has been shown to have expression in human atrial tissue (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898164">Iglesias et al. 2007</a>); it is also in an area of CNV. <em>XRCC4</em> repairs double-strand DNA breaks through nonhomologous end-joining recombination (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898140">Mari et al. 2006</a>). It has no known link to HTN. <em>MYO6</em> encodes an actin-based molecular motor involved in intracellular vesicle and organelle transport. It also has no known link to HTN.<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898144">Avraham et al. (1995)</a> found that the <em>MYO6</em> gene is defective in certain types of deafness.</p><p id="P27">Of the remaining associated SNP pairs for HTN, two on chromosome 8 were 70 kb from <em>NOV</em>, a member of the CCN family of regulatory proteins. CCN regulatory proteins are extracellular matrix-associated proteins that play crucial roles in cardiovascular and skeletal development, injury repair, cancer, and fibrotic diseases (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898161">Chen and Lau. 2009</a>). The remaining four HTN associated SNP pairs were at least 100 kb from any known gene.</p><p id="P28">Typically, genome-wide association studies include a quantile–quantile (Q–Q) plot of the results, i.e., a plot of the observed results against what would be expected under the null hypothesis, in order to check the correctness of the found <em>P</em> values. Since our tests are correlated, we do not know the large-sample theoretical null distribution of our −log<sub>10</sub>
<em>P</em> values, so we estimated it from the permutation distribution. We permuted the cases and controls, repeated the analysis and ranked the −log<sub>10</sub>
<em>P</em> values. We similarly ranked the −log<sub>10</sub>
<em>P</em> values for the observed data and then formed the Q–Q plot by plotting the observed values against the permutation values (<a href="/pmc/articles/PMC3576836/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). The Q–Q plot appears as a straight diagonal line except at the tail, suggesting that our significant findings cannot be attributed to population structure or cryptic relatedness. When we plotted the observed −log<sub>10</sub>
<em>P</em> values against the −log<sub>10</sub>
<em>P</em> values of another independent permutation, the result was essentially the same (data not shown).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3576836/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140310714400336" class="figure"><a href="/pmc/articles/PMC3576836/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3576836_nihms439840f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms439840f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3576836/bin/nihms439840f4.jpg"></a></div><div id="largeobj_idm140310714400336" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3576836/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3576836/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><p id="P40">Q–Q plots of −log<sub>10</sub>
<em>P</em> for test 2–2: left, coronary artery disease; right, hypertension. See text for how the expected distribution was computed </p></div></div></div><p id="P29">However, one of our main concerns was that the correlations between some of the SNP pairs in <a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and <a href="/pmc/articles/PMC3576836/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2</span></span><span>2</span></a> are close to 1, suggesting that the pair should have also been found on single-marker analysis. We attribute the fact that these SNPs were not found on single-marker analysis, and yet significant by two-marker analysis, to the extra power two-marker analysis affords to provide haplotype information when the markers are both heterozygous and in high enough linkage disequilibrium. However, it is known that multicollinearity of the predictors can inflate significance levels in regression analysis. Therefore, to check our results, for each SNP pair given in <a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and <a href="/pmc/articles/PMC3576836/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2,</span></span><span>2</span></a>, we used the software PHASE, version 2.1 (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898148">Stephens et al. 2001</a>; <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898143">Stephens and Scheet 2005</a>) to infer the haplotypes and found that, with few exceptions, only three of the four possible haplotypes were present; in all but one of these exceptions the fourth haplotype was rare in both cases and controls. We excluded those rare haplotypes (the largest excluded total count being 12, five cases and seven controls), and then the smallest expected value (on the assumption of no association between haplotypes and disease status) was 11.47 haplotypes. Furthermore, in all but two of the fifteen SNP pair tables, the expected haplotype count was always &gt;15, so for these there is no reason to doubt the validity of the <em>P</em> value based on asymptotic assumptions. For the two tables for which the smallest expected value was &lt;15, we calculated <em>P</em> values by Fisher’s exact test, two-sided because the association could have been with risk or protection, then multiplied by four because the associated haplotype could have been any one of the four haplotypes. The <em>P</em> values for these extreme cases were within 1–2 orders of magnitude of those based on asymptotic assumptions; therefore, we believe the increased evidence for association to be valid and due to real haplotype effects rather than an artifact as a result of either multicollinearity or small sample numbers (actual haplotype counts and odds ratios are presented in <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental material, Table 3</a>).</p><p id="P30">Furthermore, for both CAD and HTN, the 3 <em>df</em> two-marker model, containing the addition of a cross-product term, yielded roughly the same SNP pairs with a <em>P</em> value of &lt;5 × 10<sup>−7</sup> as the 2 <em>df</em> purely additive model. The only exception is noted by the asterisk in <a href="/pmc/articles/PMC3576836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Both models gave similar <em>P</em> values, even though we were expecting the 3 <em>df</em> test to yield more significant association because of the additional cross-product term. In all but the above case, the extra term in the 3 <em>df</em> test proved to not be statistically significant (<a href="#SD3" rid="SD3" class=" supplementary-material">supplemental material, Table 2</a>). The reason for this appears to be that in the situation where two disease flanking heterozygote genotypes are called, only four diplotypes are possible instead of the prior 16 possible diplotypes (<a href="/pmc/articles/PMC3576836/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>), yielding two-locus phased genotype (diplotype) information. Therefore, the SNP pair becomes more informative and hence the association with disease would be less likely to be due to chance alone—regardless of any additional effect of the cross-product term. Whenever one of the SNPs in the pair is called as homozygous, the two-marker testing would be expected to perform as a single- marker analysis, though with slightly less power owing to the extra <em>df</em>.</p><p id="P31">Ideally, all information from GWAS should be replicated using a separate dataset. Replication is challenging, especially for rare variants, as the necessary large available datasets are frequently not available and, if available, may involve variable population demographics, quality control, and phenotypic disease descriptors. We reviewed some additional associations that some recent GWAS for CAD and HTN report that are not listed in this study but, since CAD and HTN are disorders with complex inheritance, the list of genes that could be involved with disease would be lengthy and vary significantly from population to population (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898155">Newton-Cheh et al. 2009</a>; <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898157">Reilly et al. 2011</a>; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898137">Levy et al. 2009</a>). Of the eight genes cited by this study, three have been previously found in GWAS; <em>HFE2</em>, <em>ZFAT</em>, and <em>MACROD2</em> (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898159">Zhu et al. 2010</a>). The only non-protein-coding locus with previous association was 9p21, which was the only area reaching genome-wide significance in the WTCCC (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_298898156">Wellcome Trust Case Control Consortium 2007</a>). </p><p id="P32">In conclusion, using two-marker methods, we found many interesting SNP pairs and associated genes not previously reported to be linked to CAD or HTN. Of note, CAD associated genes <em>HFE2</em>, <em>CDKN2A/CDKN2B</em>, and <em>HSP90B1</em>, and HTN associated genes <em>ZFAT</em>, <em>MACROD2</em> and <em>NOV</em>, have known cardiovascular literature and/or known genetic mechanisms that may be relevant to disease.</p><p id="P33" class="p p-last">We believe that multiple-marker association models may be better suited to detect variants because they are more informative of phased diplotypes than single-marker tests. Additional associated variants could also be found by using less stringent quality control filters. As many methods are available for multiple-marker association tests, further analysis using different techniques will need to be compared to determine the best fitting model for the detection of causal and protective variants for common, complex diseases such as HTN and CAD.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>spuul3</h4><div class="sup-box half_rhythm" id="d35e841"><a href="/pmc/articles/PMC3576836/bin/NIHMS439840-supplement-spuul3.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(115K, doc)</sup></div></div><div class="sec suppmat" id="SD2"><h4>suppl1</h4><div class="sup-box half_rhythm" id="d35e845"><a href="/pmc/articles/PMC3576836/bin/NIHMS439840-supplement-suppl1.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(87K, doc)</sup></div></div><div class="sec suppmat" id="SD3"><h4>suppl2</h4><div class="sup-box half_rhythm" id="d35e849"><a href="/pmc/articles/PMC3576836/bin/NIHMS439840-supplement-suppl2.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(49K, doc)</sup></div></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Acknowledgments</h2><div class="sec"><p id="P36">This study makes use of data generated by the Wellcome Trust Case Control Consortium. A full list of the investigators who contributed to the generation of the data is available from <a href="http://www.wtccc.org.uk/info/participants.shtml" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk/info/participants.shtml</a>. The work was supported by the National Institutes of Health, grant numbers: <a id="__tag_807492489" class="tag_hotlink" href="/nuccore/HL074166" ref="/nuccore/HL074166"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL074166","term_id":"1051637767","term_text":"HL074166"}}</span>HL074166</a> and <a id="__tag_807492488" class="tag_hotlink" href="/nuccore/HL086718" ref="/nuccore/HL086718"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL086718","term_id":"1051657127","term_text":"HL086718"}}</span>HL086718</a> from the National Heart, Lung, Blood Institute, <a id="__tag_807492490" class="tag_hotlink" href="/nuccore/HG003054" ref="/nuccore/HG003054"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG003054","term_id":"549035330","term_text":"HG003054"}}</span>HG003054</a> from the National Human Genome Research Institute, and, RR03655 from the National Center for Research Resources. Funding for the original WTCCC project was provided by the Wellcome Trust under award 076113.</p></div></div><div id="idm140310726495392" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140310726495392title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P34" class="p p-first-last"><strong>Electronic supplementary material</strong> The online version of this article (doi:10.1007/s00439-011-1009-6) contains supplementary material, which is available to authorized users.
</p><p></p><p class="fn sec" id="FN2"> </p><p id="P35" class="p p-first-last"><strong>Conflict of interest</strong> None.</p><p></p></div></div><div id="idm140310732777904aff-info" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140310732777904aff-infotitle">Contributor Information</h2><p><span class="fm-affl">Thomas P. Slavin, </span><span class="fm-affl" id="A1" lang="en"> Department of Genetics, Center for Human Genetics, University Hospitals of Cleveland and Case Western Reserve University, 10524 Euclid Avenue, Cleveland, OH, USA.</span><span class="fm-affl" id="A2" lang="en"> Hawai’i Community Genetics, 1441 Kapi‘olani Blvd., Ste 1800, Honolulu, HI 96814, USA, <a href="mailto:dev@null" data-email="gro.inaloipak@nivals.samoht" class="oemail">gro.inaloipak@nivals.samoht</a>.</span></p><p><span class="fm-affl">Tao Feng, </span><span class="fm-affl" id="A3" lang="en"> Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH 44106, USA.</span><span class="fm-affl" id="A4" lang="en"> Department of Mathematics, Heilongjiang University, Harbin 150086, China.</span></p><p><span class="fm-affl">Audrey Schnell, </span><span class="fm-affl" id="A5" lang="en"> Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH 44106, USA.</span></p><p><span class="fm-affl">Xiaofeng Zhu, </span><span class="fm-affl" id="A6" lang="en"> Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH 44106, USA.</span></p><p><span class="fm-affl">Robert C. Elston, </span><span class="fm-affl" id="A7" lang="en"> Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH 44106, USA.</span></p></div><div id="idm140310720112928" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140310720112928title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">An J, Du J, Wei N, Xu H, Pritchard KA, Jr, Shi Y. Role of tetrahydrobiopterin in resistance to myocardial ischemia in Brown Norway and Dahl S rats. <span><span class="ref-journal">Am J Physiol Heart Circ Physiol. </span>2009;<span class="ref-vol">297</span>:H1783–H1791.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2781371/">PMC free article</a>]</span> [<a href="/pubmed/19717731" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Heart+Circ+Physiol&amp;title=Role+of+tetrahydrobiopterin+in+resistance+to+myocardial+ischemia+in+Brown+Norway+and+Dahl+S+rats&amp;author=J+An&amp;author=J+Du&amp;author=N+Wei&amp;author=H+Xu&amp;author=KA+Pritchard&amp;volume=297&amp;publication_year=2009&amp;pages=H1783-H1791&amp;pmid=19717731&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Avraham KB, Hasson T, Steel KP, Kingsley DM, Russell LB, Mooseker MS, Copeland NG, Jenkins NA. The mouse Snell’s waltzer deafness gene encodes an unconventional myosin required for structural integrity of inner ear hair cells. <span><span class="ref-journal">Nat Genet. </span>1995;<span class="ref-vol">11</span>:369–375.</span> [<a href="/pubmed/7493015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+mouse+Snell’s+waltzer+deafness+gene+encodes+an+unconventional+myosin+required+for+structural+integrity+of+inner+ear+hair+cells&amp;author=KB+Avraham&amp;author=T+Hasson&amp;author=KP+Steel&amp;author=DM+Kingsley&amp;author=LB+Russell&amp;volume=11&amp;publication_year=1995&amp;pages=369-375&amp;pmid=7493015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Browning BL, Browning SR. Haplotypic analysis of Wellcome Trust Case Control Consortium data. <span><span class="ref-journal">Hum Genet. </span>2008;<span class="ref-vol">123</span>:273–280.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2384233/">PMC free article</a>]</span> [<a href="/pubmed/18224336" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Haplotypic+analysis+of+Wellcome+Trust+Case+Control+Consortium+data&amp;author=BL+Browning&amp;author=SR+Browning&amp;volume=123&amp;publication_year=2008&amp;pages=273-280&amp;pmid=18224336&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Chen CC, Lau LF. Functions and mechanisms of action of CCN matricellular proteins. <span><span class="ref-journal">Int J Biochem Cell Biol. </span>2009;<span class="ref-vol">41</span>:771–783.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668982/">PMC free article</a>]</span> [<a href="/pubmed/18775791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Biochem+Cell+Biol&amp;title=Functions+and+mechanisms+of+action+of+CCN+matricellular+proteins&amp;author=CC+Chen&amp;author=LF+Lau&amp;volume=41&amp;publication_year=2009&amp;pages=771-783&amp;pmid=18775791&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Ciullo M, Bellenguez C, Colonna V, Nutile T, Calabria A, Pacente R, Iovino G, Trimarco B, Bourgain C, Persico MG. New susceptibility locus for hypertension on chromosome 8q by efficient pedigree-breaking in an Italian isolate. <span><span class="ref-journal">Hum Mol Genet. </span>2006;<span class="ref-vol">15</span>:1735–1743.</span> [<a href="/pubmed/16611673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=New+susceptibility+locus+for+hypertension+on+chromosome+8q+by+efficient+pedigree-breaking+in+an+Italian+isolate&amp;author=M+Ciullo&amp;author=C+Bellenguez&amp;author=V+Colonna&amp;author=T+Nutile&amp;author=A+Calabria&amp;volume=15&amp;publication_year=2006&amp;pages=1735-1743&amp;pmid=16611673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Cotsapas C, Speliotes EK, Hatoum IJ, Greenawalt DM, Dobrin R, Lum PY, Suver C, Chudin E, Kemp D, Reitman M, Voight BF, Neale BM, Schadt EE, Hirschhorn JN, Kaplan LM, Daly MJ GIANT Consortium. Common body mass index-associated variants confer risk of extreme obesity. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:3502–3507.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2729668/">PMC free article</a>]</span> [<a href="/pubmed/19553259" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Common+body+mass+index-associated+variants+confer+risk+of+extreme+obesity&amp;author=C+Cotsapas&amp;author=EK+Speliotes&amp;author=IJ+Hatoum&amp;author=DM+Greenawalt&amp;author=R+Dobrin&amp;volume=18&amp;publication_year=2009&amp;pages=3502-3507&amp;pmid=19553259&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Horne BD, Carlquist JF, Muhlestein JB, Bair TL, Anderson JL. Association of variation in the chromosome 9p21 locus with myocardial infarction versus chronic coronary artery disease. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2008;<span class="ref-vol">1</span>:85–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745117/">PMC free article</a>]</span> [<a href="/pubmed/19956784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Association+of+variation+in+the+chromosome+9p21+locus+with+myocardial+infarction+versus+chronic+coronary+artery+disease&amp;author=BD+Horne&amp;author=JF+Carlquist&amp;author=JB+Muhlestein&amp;author=TL+Bair&amp;author=JL+Anderson&amp;volume=1&amp;publication_year=2008&amp;pages=85-92&amp;pmid=19956784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Iglesias MJ, Salgado A, Pineiro R, Rodino BK, Otero MF, Grigorian L, Gallego R, Dieguez C, Gualillo O, Gonzalez-Juanatey JR, Lago F. Lack of effect of the ghrelin gene-derived peptide obestatin on cardiomyocyte viability and metabolism. <span><span class="ref-journal">J Endocrinol Invest. </span>2007;<span class="ref-vol">30</span>:470–476.</span> [<a href="/pubmed/17646721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Endocrinol+Invest&amp;title=Lack+of+effect+of+the+ghrelin+gene-derived+peptide+obestatin+on+cardiomyocyte+viability+and+metabolism&amp;author=MJ+Iglesias&amp;author=A+Salgado&amp;author=R+Pineiro&amp;author=BK+Rodino&amp;author=MF+Otero&amp;volume=30&amp;publication_year=2007&amp;pages=470-476&amp;pmid=17646721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Kim S, Morris NJ, Won S, Elston RC. Single-marker and twomarker association tests for unphased case-control genotype data, with a power comparison. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>:67–77.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2796706/">PMC free article</a>]</span> [<a href="/pubmed/19557751" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Single-marker+and+twomarker+association+tests+for+unphased+case-control+genotype+data,+with+a+power+comparison&amp;author=S+Kim&amp;author=NJ+Morris&amp;author=S+Won&amp;author=RC+Elston&amp;volume=34&amp;publication_year=2010&amp;pages=67-77&amp;pmid=19557751&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Levy D, Ehret GB, Rice K, Verwoert GC, Launer LJ, Dehghan A, Glazer NL, Morrison AC, Johnson AD, Aspelund T, Aulchenko Y, Lumley T, Kottgen A, Vasan RS, Rivadeneira F, Eiriksdottir G, Guo X, Arking DE, Mitchell GF, Mattace-Raso FU, Smith AV, Taylor K, Scharpf RB, Hwang SJ, Sijbrands EJ, Bis J, Harris TB, Ganesh SK, O’Donnell CJ, Hofman A, Rotter JI, Coresh J, Benjamin EJ, Uitterlinden AG, Heiss G, Fox CS, Witteman JC, Boerwinkle E, Wang TJ, Gudnason V, Larson MG, Chakravarti A, Psaty BM, van Duijn CM. Genomewide association study of blood pressure and hypertension. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:677–687.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2998712/">PMC free article</a>]</span> [<a href="/pubmed/19430479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genomewide+association+study+of+blood+pressure+and+hypertension&amp;author=D+Levy&amp;author=GB+Ehret&amp;author=K+Rice&amp;author=GC+Verwoert&amp;author=LJ+Launer&amp;volume=41&amp;publication_year=2009&amp;pages=677-687&amp;pmid=19430479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Mari PO, Florea BI, Persengiev SP, Verkaik NS, Bruggenwirth HT, Modesti M, Giglia-Mari G, Bezstarosti K, Demmers JA, Luider TM, Houtsmuller AB, van Gent DC. Dynamic assembly of end-joining complexes requires interaction between Ku70/80 and XRCC4. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2006;<span class="ref-vol">103</span>:18597–18602.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1693708/">PMC free article</a>]</span> [<a href="/pubmed/17124166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Dynamic+assembly+of+end-joining+complexes+requires+interaction+between+Ku70/80+and+XRCC4&amp;author=PO+Mari&amp;author=BI+Florea&amp;author=SP+Persengiev&amp;author=NS+Verkaik&amp;author=HT+Bruggenwirth&amp;volume=103&amp;publication_year=2006&amp;pages=18597-18602&amp;pmid=17124166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">McCarthy MI, Abecasis GR, Cardon LR, Goldstein DB, Little J, Ioannidis JP, Hirschhorn JN. Genome-wide association studies for complex traits: consensus, uncertainty and challenges. <span><span class="ref-journal">Nat Rev Genet. </span>2008;<span class="ref-vol">9</span>:356–369.</span> [<a href="/pubmed/18398418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+for+complex+traits:+consensus,+uncertainty+and+challenges&amp;author=MI+McCarthy&amp;author=GR+Abecasis&amp;author=LR+Cardon&amp;author=DB+Goldstein&amp;author=J+Little&amp;volume=9&amp;publication_year=2008&amp;pages=356-369&amp;pmid=18398418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Nagase T, Ishikawa K, Suyama M, Kikuno R, Hirosawa M, Miyajima N, Tanaka A, Kotani H, Nomura N, Ohara O. Prediction of the coding sequences of unidentified human genes. XIII. The complete sequences of 100 new cDNA clones from brain which code for large proteins in vitro. <span><span class="ref-journal">DNA Res. </span>1999;<span class="ref-vol">6</span>:63–70.</span> [<a href="/pubmed/10231032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Res&amp;title=Prediction+of+the+coding+sequences+of+unidentified+human+genes.+XIII.+The+complete+sequences+of+100+new+cDNA+clones+from+brain+which+code+for+large+proteins+in+vitro&amp;author=T+Nagase&amp;author=K+Ishikawa&amp;author=M+Suyama&amp;author=R+Kikuno&amp;author=M+Hirosawa&amp;volume=6&amp;publication_year=1999&amp;pages=63-70&amp;pmid=10231032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Newton-Cheh C, Johnson T, Gateva V, Tobin MD, Bochud M, Coin L, Najjar SS, Zhao JH, Heath SC, Eyheramendy S, Papadakis K, Voight BF, Scott LJ, Zhang F, Farrall M, Tanaka T, Wallace C, Chambers JC, Khaw KT, Nilsson P, van der Harst P, Polidoro S, Grobbee DE, Onland-Moret NC, Bots ML, Wain LV, Elliott KS, Teumer A, Luan J, Lucas G, Kuusisto J, Burton PR, Hadley D, McArdle WL, Wellcome Trust Case Control Consortium, Brown M, Dominiczak A, Newhouse SJ, Samani NJ, Webster J, Zeggini E, Beckmann JS, Bergmann S, Lim N, Song K, Vollenweider P, Waeber G, Waterworth DM, Yuan X, Groop L, Orho-Melander M, Allione A, Di Gregorio A, Guarrera S, Panico S, Ricceri F, Romanazzi V, Sacerdote C, Vineis P, Barroso I, Sandhu MS, Luben RN, Crawford GJ, Jousilahti P, Perola M, Boehnke M, Bonnycastle LL, Collins FS, Jackson AU, Mohlke KL, Stringham HM, Valle TT, Willer CJ, Bergman RN, Morken MA, Doring A, Gieger C, Illig T, Meitinger T, Org E, Pfeufer A, Wichmann HE, Kathiresan S, Marrugat J, O’Donnell CJ, Schwartz SM, Siscovick DS, Subirana I, Freimer NB, Hartikainen AL, McCarthy MI, O’Reilly PF, Peltonen L, Pouta A, de Jong PE, Snieder H, van Gilst WH, Clarke R, Goel A, Hamsten A, Peden JF, Seedorf U, Syvanen AC, Tognoni G, Lakatta EG, Sanna S, Scheet P, Schlessinger D, Scuteri A, Dorr M, Ernst F, Felix SB, Homuth G, Lorbeer R, Reffelmann T, Rettig R, Volker U, Galan P, Gut IG, Hercberg S, Lathrop GM, Zelenika D, Deloukas P, Soranzo N, Williams FM, Zhai G, Salomaa V, Laakso M, Elosua R, Forouhi NG, Volzke H, Uiterwaal CS, van der Schouw YT, Numans ME, Matullo G, Navis G, Berglund G, Bingham SA, Kooner JS, Connell JM, Bandinelli S, Ferrucci L, Watkins H, Spector TD, Tuomilehto J, Altshuler D, Strachan DP, Laan M, Meneton P, Wareham NJ, Uda M, Jarvelin MR, Mooser V, Melander O, Loos RJ, Elliott P, Abecasis GR, Caulfield M, Munroe PB. Genome-wide association study identifies eight loci associated with blood pressure. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:666–676.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2891673/">PMC free article</a>]</span> [<a href="/pubmed/19430483" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+eight+loci+associated+with+blood+pressure&amp;author=C+Newton-Cheh&amp;author=T+Johnson&amp;author=V+Gateva&amp;author=MD+Tobin&amp;author=M+Bochud&amp;volume=41&amp;publication_year=2009&amp;pages=666-676&amp;pmid=19430483&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Ota A, Zhang J, Ping P, Han J, Wang Y. Specific regulation of noncanonical p38alpha activation by Hsp90-Cdc37 chaperone complex in cardiomyocyte. <span><span class="ref-journal">Circ Res. </span>2010;<span class="ref-vol">106</span>:1404–1412.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2891038/">PMC free article</a>]</span> [<a href="/pubmed/20299663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Specific+regulation+of+noncanonical+p38alpha+activation+by+Hsp90-Cdc37+chaperone+complex+in+cardiomyocyte&amp;author=A+Ota&amp;author=J+Zhang&amp;author=P+Ping&amp;author=J+Han&amp;author=Y+Wang&amp;volume=106&amp;publication_year=2010&amp;pages=1404-1412&amp;pmid=20299663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Reilly MP, Li M, He J, Ferguson JF, Stylianou IM, Mehta NN, Burnett MS, Devaney JM, Knouff CW, Thompson JR, Horne BD, Stewart AF, Assimes TL, Wild PS, Allayee H, Nitschke PL, Patel RS, Myocardial Infarction Genetics Consortium, Wellcome Trust Case Control Consortium, Martinelli N, Girelli D, Quyyumi AA, Anderson JL, Erdmann J, Hall AS, Schunkert H, Quertermous T, Blankenberg S, Hazen SL, Roberts R, Kathiresan S, Samani NJ, Epstein SE, Rader DJ, Qasim AN, DerOhannessian SL, Qu L, Cappola TP, Chen Z, Matthai W, Hakonarson HH, Wilensky R, Kent KM, Lindsay JM, Pichard AD, Satler L, Waksman R. Identification of ADAMTS7 as a novel locus for coronary atherosclerosis and association of ABO with myocardial infarction in the presence of coronary atherosclerosis: two genome-wide association studies. <span><span class="ref-journal">Lancet. </span>2011;<span class="ref-vol">377</span>:383–392.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3297116/">PMC free article</a>]</span> [<a href="/pubmed/21239051" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Identification+of+ADAMTS7+as+a+novel+locus+for+coronary+atherosclerosis+and+association+of+ABO+with+myocardial+infarction+in+the+presence+of+coronary+atherosclerosis:+two+genome-wide+association+studies&amp;author=MP+Reilly&amp;author=M+Li&amp;author=J+He&amp;author=JF+Ferguson&amp;author=IM+Stylianou&amp;volume=377&amp;publication_year=2011&amp;pages=383-392&amp;pmid=21239051&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Rivard SR, Lanzara C, Grimard D, Carella M, Simard H, Ficarella R, Simard R, D’Adamo AP, Ferec C, Camaschella C, Mura C, Roetto A, De Braekeleer M, Bechner L, Gasparini P. Juvenile hemochromatosis locus maps to chromosome 1q in a French Canadian population. <span><span class="ref-journal">Eur J Hum Genet. </span>2003;<span class="ref-vol">11</span>:585–589.</span> [<a href="/pubmed/12891378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Juvenile+hemochromatosis+locus+maps+to+chromosome+1q+in+a+French+Canadian+population&amp;author=SR+Rivard&amp;author=C+Lanzara&amp;author=D+Grimard&amp;author=M+Carella&amp;author=H+Simard&amp;volume=11&amp;publication_year=2003&amp;pages=585-589&amp;pmid=12891378&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Samani NJ, Schunkert H. Chromosome 9p21 and cardiovascular disease: the story unfolds. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2008;<span class="ref-vol">1</span>:81–84.</span> [<a href="/pubmed/20031549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Chromosome+9p21+and+cardiovascular+disease:+the+story+unfolds&amp;author=NJ+Samani&amp;author=H+Schunkert&amp;volume=1&amp;publication_year=2008&amp;pages=81-84&amp;pmid=20031549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Stephens M, Scheet P. Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation. <span><span class="ref-journal">Am J Hum Genet. </span>2005;<span class="ref-vol">76</span>:449–462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1196397/">PMC free article</a>]</span> [<a href="/pubmed/15700229" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Accounting+for+decay+of+linkage+disequilibrium+in+haplotype+inference+and+missing-data+imputation&amp;author=M+Stephens&amp;author=P+Scheet&amp;volume=76&amp;publication_year=2005&amp;pages=449-462&amp;pmid=15700229&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">68</span>:978–989.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1275651/">PMC free article</a>]</span> [<a href="/pubmed/11254454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+new+statistical+method+for+haplotype+reconstruction+from+population+data&amp;author=M+Stephens&amp;author=NJ+Smith&amp;author=P+Donnelly&amp;volume=68&amp;publication_year=2001&amp;pages=978-989&amp;pmid=11254454&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Zhu X, Zhang S, Zhao H, Cooper RS. Association mapping, using a mixture model for complex traits. <span><span class="ref-journal">Genet Epidemiol. </span>2002;<span class="ref-vol">23</span>:181–196.</span> [<a href="/pubmed/12214310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Association+mapping,+using+a+mixture+model+for+complex+traits&amp;author=X+Zhu&amp;author=S+Zhang&amp;author=H+Zhao&amp;author=RS+Cooper&amp;volume=23&amp;publication_year=2002&amp;pages=181-196&amp;pmid=12214310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Zhu X, Li S, Cooper RS, Elston RC. A unified association analysis approach for family and unrelated samples correcting for stratification. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">82</span>:352–365.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427300/">PMC free article</a>]</span> [<a href="/pubmed/18252216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+unified+association+analysis+approach+for+family+and+unrelated+samples+correcting+for+stratification&amp;author=X+Zhu&amp;author=S+Li&amp;author=RS+Cooper&amp;author=RC+Elston&amp;volume=82&amp;publication_year=2008&amp;pages=352-365&amp;pmid=18252216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Zhu X, Feng T, Li Y, Lu Q, Elston RC. Detecting rare variants for complex traits using family and unrelated data. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>:171–187.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2811752/">PMC free article</a>]</span> [<a href="/pubmed/19847924" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Detecting+rare+variants+for+complex+traits+using+family+and+unrelated+data&amp;author=X+Zhu&amp;author=T+Feng&amp;author=Y+Li&amp;author=Q+Lu&amp;author=RC+Elston&amp;volume=34&amp;publication_year=2010&amp;pages=171-187&amp;pmid=19847924&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3576836&amp;issue-id=219807&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140310726480672title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Methods</a></li><li><a href="#S7title">Results</a></li><li><a href="#S10title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S11title">Acknowledgments</a></li><li><a href="#idm140310726495392title">Footnotes</a></li><li><a href="#idm140310732777904aff-infotitle">Contributor Information</a></li><li><a href="#idm140310720112928title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3576836/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3576836/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3576836/pdf/nihms439840.pdf">PDF (1.9M)</a></li> | <li><a href="#" data-citationid="PMC3576836" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3576836%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3576836%2F&amp;text=Two-marker%20association%20tests%20yield%20new%20disease%20associations%20for%20coronary%20artery%20disease%20and%20hypertension" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3576836%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21626137" data-aiid="3576836" data-aid="3576836" data-iid="219807" data-domainid="319" data-domain="nihpa" data-accid="PMC3576836" data-md5="38eec96631b6a65104991c373aaa6ade"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:07:06-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DAABBDD51FE2100000000086E055C&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3576836%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3576836/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>