<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3428154/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of age-related macular degeneration identifies associated variants in the TNXB–FKBPL–NOTCH4 region of chromosome 6p21.3</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3428154/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study of age-related macular degeneration identifies associated variants in the TNXB–FKBPL–NOTCH4 region of chromosome 6p21.3"><meta name="citation_authors" content="Valentina Cipriani, Hin-Tak Leung, Vincent Plagnol, Catey Bunce, Jane C. Khan, Humma Shahid, Anthony T. Moore, Simon P. Harding, Paul N. Bishop, Caroline Hayward, Susan Campbell, Ana Maria Armbrecht, Baljean Dhillon, Ian J. Deary, Harry Campbell, Malcolm Dunlop, Anna F. Dominiczak, Samantha S. Mann, Sharon A. Jenkins, Andrew R. Webster, Alan C. Bird, Mark Lathrop, Diana Zelenika, Eric H. Souied, José-Alain Sahel, Thierry Léveillard, French AMD Investigators, Angela J. Cree, Jane Gibson, Sarah Ennis, Andrew J. Lotery, Alan F. Wright, David G. Clayton, John R.W. Yates"><meta name="citation_date" content="15 September 2012"><meta name="citation_issue" content="18"><meta name="citation_volume" content="21"><meta name="citation_firstpage" content="4138"><meta name="citation_doi" content="10.1093/hmg/dds225"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3428154/?report=abstract"><meta name="citation_pmid" content="22694956"><meta name="DC.Title" content="Genome-wide association study of age-related macular degeneration identifies associated variants in the TNXB–FKBPL–NOTCH4 region of chromosome 6p21.3"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Valentina Cipriani"><meta name="DC.Contributor" content="Hin-Tak Leung"><meta name="DC.Contributor" content="Vincent Plagnol"><meta name="DC.Contributor" content="Catey Bunce"><meta name="DC.Contributor" content="Jane C. Khan"><meta name="DC.Contributor" content="Humma Shahid"><meta name="DC.Contributor" content="Anthony T. Moore"><meta name="DC.Contributor" content="Simon P. Harding"><meta name="DC.Contributor" content="Paul N. Bishop"><meta name="DC.Contributor" content="Caroline Hayward"><meta name="DC.Contributor" content="Susan Campbell"><meta name="DC.Contributor" content="Ana Maria Armbrecht"><meta name="DC.Contributor" content="Baljean Dhillon"><meta name="DC.Contributor" content="Ian J. Deary"><meta name="DC.Contributor" content="Harry Campbell"><meta name="DC.Contributor" content="Malcolm Dunlop"><meta name="DC.Contributor" content="Anna F. Dominiczak"><meta name="DC.Contributor" content="Samantha S. Mann"><meta name="DC.Contributor" content="Sharon A. Jenkins"><meta name="DC.Contributor" content="Andrew R. Webster"><meta name="DC.Contributor" content="Alan C. Bird"><meta name="DC.Contributor" content="Mark Lathrop"><meta name="DC.Contributor" content="Diana Zelenika"><meta name="DC.Contributor" content="Eric H. Souied"><meta name="DC.Contributor" content="José-Alain Sahel"><meta name="DC.Contributor" content="Thierry Léveillard"><meta name="DC.Contributor" content="French AMD Investigators"><meta name="DC.Contributor" content="Angela J. Cree"><meta name="DC.Contributor" content="Jane Gibson"><meta name="DC.Contributor" content="Sarah Ennis"><meta name="DC.Contributor" content="Andrew J. Lotery"><meta name="DC.Contributor" content="Alan F. Wright"><meta name="DC.Contributor" content="David G. Clayton"><meta name="DC.Contributor" content="John R.W. Yates"><meta name="DC.Date" content="2012 Sep 15"><meta name="DC.Identifier" content="10.1093/hmg/dds225"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of age-related macular degeneration identifies associated variants in the TNXB–FKBPL–NOTCH4 region of chromosome 6p21.3"><meta property="og:type" content="article"><meta property="og:description" content="Age-related macular degeneration (AMD) is a leading cause of visual loss in Western populations. Susceptibility is influenced by age, environmental and genetic factors. Known genetic risk loci do not account for all the heritability. We therefore carried ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3428154/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3428154/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3428154/pdf/dds225.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C2FDADD541A010000000004C30330.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3428154%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3428154</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2012 Sep 15; 21(18): 4138–4150. </span></div><div><span class="fm-vol-iss-date">Published online 2012 Jun 13. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fdds225" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/dds225</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3428154</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22694956">22694956</a></div></div></div></div><h1 class="content-title">Genome-wide association study of age-related macular degeneration identifies associated variants in the <em>TNXB–FKBPL–NOTCH4</em> region of chromosome 6p21.3</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Cipriani%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193048644800" co-class="co-affbox">Valentina Cipriani</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Leung%20HT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051624160" co-class="co-affbox">Hin-Tak Leung</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Plagnol%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193054969472" co-class="co-affbox">Vincent Plagnol</a>,<sup>5</sup> <a href="/pubmed/?term=Bunce%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193054436528" co-class="co-affbox">Catey Bunce</a>,<sup>1,</sup><sup>2,</sup><sup>6</sup> <a href="/pubmed/?term=Khan%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051115264" co-class="co-affbox">Jane C. Khan</a>,<sup>3,</sup><sup>7</sup> <a href="/pubmed/?term=Shahid%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193043861904" co-class="co-affbox">Humma Shahid</a>,<sup>3,</sup><sup>8</sup> <a href="/pubmed/?term=Moore%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193055922368" co-class="co-affbox">Anthony T. Moore</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Harding%20SP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051583536" co-class="co-affbox">Simon P. Harding</a>,<sup>9</sup> <a href="/pubmed/?term=Bishop%20PN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051994656" co-class="co-affbox">Paul N. Bishop</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051465296" co-class="co-affbox">Caroline Hayward</a>,<sup>12</sup> <a href="/pubmed/?term=Campbell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051329264" co-class="co-affbox">Susan Campbell</a>,<sup>12</sup> <a href="/pubmed/?term=Armbrecht%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051210352" co-class="co-affbox">Ana Maria Armbrecht</a>,<sup>13</sup> <a href="/pubmed/?term=Dhillon%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051207808" co-class="co-affbox">Baljean Dhillon</a>,<sup>13</sup> <a href="/pubmed/?term=Deary%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051179824" co-class="co-affbox">Ian J. Deary</a>,<sup>14</sup> <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193092496800" co-class="co-affbox">Harry Campbell</a>,<sup>15,</sup><sup>16</sup> <a href="/pubmed/?term=Dunlop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193051413696" co-class="co-affbox">Malcolm Dunlop</a>,<sup>15</sup> <a href="/pubmed/?term=Dominiczak%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193041694592" co-class="co-affbox">Anna F. Dominiczak</a>,<sup>17</sup> <a href="/pubmed/?term=Mann%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193041692048" co-class="co-affbox">Samantha S. Mann</a>,<sup>1,</sup><sup>2,</sup><sup>18</sup> <a href="/pubmed/?term=Jenkins%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193091102528" co-class="co-affbox">Sharon A. Jenkins</a>,<sup>1,</sup><sup>2,</sup><sup>19</sup> <a href="/pubmed/?term=Webster%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193050354192" co-class="co-affbox">Andrew R. Webster</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Bird%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193052199632" co-class="co-affbox">Alan C. Bird</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193047387568" co-class="co-affbox">Mark Lathrop</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Zelenika%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193050870624" co-class="co-affbox">Diana Zelenika</a>,<sup>20</sup> <a href="/pubmed/?term=Souied%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193053673888" co-class="co-affbox">Eric H. Souied</a>,<sup>22</sup> <a href="/pubmed/?term=Sahel%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193053671344" co-class="co-affbox">José-Alain Sahel</a>,<sup>1,</sup><sup>23,</sup><sup>24</sup> <a href="/pubmed/?term=L%26%23x000e9%3Bveillard%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193044001040" co-class="co-affbox">Thierry Léveillard</a>,<sup>23</sup> French AMD Investigators,<sup>†</sup> <a href="/pubmed/?term=Cree%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193054479280" co-class="co-affbox">Angela J. Cree</a>,<sup>25</sup> <a href="/pubmed/?term=Gibson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193127731312" co-class="co-affbox">Jane Gibson</a>,<sup>26</sup> <a href="/pubmed/?term=Ennis%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193127728768" co-class="co-affbox">Sarah Ennis</a>,<sup>26</sup> <a href="/pubmed/?term=Lotery%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193056564096" co-class="co-affbox">Andrew J. Lotery</a>,<sup>25,</sup><sup>27</sup> <a href="/pubmed/?term=Wright%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193092296256" co-class="co-affbox">Alan F. Wright</a>,<sup>12</sup> <a href="/pubmed/?term=Clayton%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193054630192" co-class="co-affbox">David G. Clayton</a>,<sup>3</sup> and  <a href="/pubmed/?term=Yates%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956" class="affpopup" co-rid="_co_idm140193043694928" co-class="co-affbox">John R.W. Yates</a><sup>1,</sup><sup>2,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140193048644800"><h3 class="no_margin">Valentina Cipriani</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</p><div>Find articles by <a href="/pubmed/?term=Cipriani%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Valentina Cipriani</a></div></div><div id="_co_idm140193051624160"><h3 class="no_margin">Hin-Tak Leung</h3><p><sup>3</sup>Department of Medical Genetics,, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK,</p><p><sup>4</sup>Department of Ophthalmology and Visual Sciences, The Chinese University of Hong Kong, Hong Kong, China,</p><div>Find articles by <a href="/pubmed/?term=Leung%20HT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Hin-Tak Leung</a></div></div><div id="_co_idm140193054969472"><h3 class="no_margin">Vincent Plagnol</h3><p><sup>5</sup>UCL Genetics Institute, London WC1E 6BT, UK,</p><div>Find articles by <a href="/pubmed/?term=Plagnol%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Vincent Plagnol</a></div></div><div id="_co_idm140193054436528"><h3 class="no_margin">Catey Bunce</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</p><p><sup>6</sup>Department of Infectious and Tropical Diseases, London School of Hygiene &amp; Tropical Medicine, University of London, London WC1E 7HT, UK,</p><div>Find articles by <a href="/pubmed/?term=Bunce%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Catey Bunce</a></div></div><div id="_co_idm140193051115264"><h3 class="no_margin">Jane C. Khan</h3><p><sup>3</sup>Department of Medical Genetics,, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK,</p><p><sup>7</sup>Department of Ophthalmology, Royal Perth Hospital, Perth, Western Australia 6001, Australia,</p><div>Find articles by <a href="/pubmed/?term=Khan%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Jane C. Khan</a></div></div><div id="_co_idm140193043861904"><h3 class="no_margin">Humma Shahid</h3><p><sup>3</sup>Department of Medical Genetics,, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK,</p><p><sup>8</sup>Department of Ophthalmology, Addenbrooke's Hospital, Cambridge CB2 0QQ, UK,</p><div>Find articles by <a href="/pubmed/?term=Shahid%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Humma Shahid</a></div></div><div id="_co_idm140193055922368"><h3 class="no_margin">Anthony T. Moore</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</p><div>Find articles by <a href="/pubmed/?term=Moore%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Anthony T. Moore</a></div></div><div id="_co_idm140193051583536"><h3 class="no_margin">Simon P. Harding</h3><p><sup>9</sup>Department of Eye and Vision Sciences, Institute of Ageing and Chronic Disease, UCD Building, University of Liverpool, Liverpool L69 3GA, UK,</p><div>Find articles by <a href="/pubmed/?term=Harding%20SP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Simon P. Harding</a></div></div><div id="_co_idm140193051994656"><h3 class="no_margin">Paul N. Bishop</h3><p><sup>10</sup>School of Biomedicine, Faculty of Medical and Human Sciences, University of Manchester, Manchester M13 9PT, UK,</p><p><sup>11</sup>Centre for Advanced Discovery and Experimental Therapeutics, Manchester Academic Health Science Centre, Central Manchester Foundation Trust, Manchester M13 9PL, UK,</p><div>Find articles by <a href="/pubmed/?term=Bishop%20PN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Paul N. Bishop</a></div></div><div id="_co_idm140193051465296"><h3 class="no_margin">Caroline Hayward</h3><p><sup>12</sup>Medical Research Council Human Genetics Unit, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, UK,</p><div>Find articles by <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Caroline Hayward</a></div></div><div id="_co_idm140193051329264"><h3 class="no_margin">Susan Campbell</h3><p><sup>12</sup>Medical Research Council Human Genetics Unit, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, UK,</p><div>Find articles by <a href="/pubmed/?term=Campbell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Susan Campbell</a></div></div><div id="_co_idm140193051210352"><h3 class="no_margin">Ana Maria Armbrecht</h3><p><sup>13</sup>Department of Ophthalmology, University of Edinburgh and Princess Alexandra Eye Pavilion, Edinburgh EH3 9HA, UK,</p><div>Find articles by <a href="/pubmed/?term=Armbrecht%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Ana Maria Armbrecht</a></div></div><div id="_co_idm140193051207808"><h3 class="no_margin">Baljean Dhillon</h3><p><sup>13</sup>Department of Ophthalmology, University of Edinburgh and Princess Alexandra Eye Pavilion, Edinburgh EH3 9HA, UK,</p><div>Find articles by <a href="/pubmed/?term=Dhillon%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Baljean Dhillon</a></div></div><div id="_co_idm140193051179824"><h3 class="no_margin">Ian J. Deary</h3><p><sup>14</sup>Department of Psychology, Centre for Cognitive Ageing and Cognitive Epidemiology, University of Edinburgh, Edinburgh EH8 9JZ, UK,</p><div>Find articles by <a href="/pubmed/?term=Deary%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Ian J. Deary</a></div></div><div id="_co_idm140193092496800"><h3 class="no_margin">Harry Campbell</h3><p><sup>15</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK,</p><p><sup>16</sup>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, UK,</p><div>Find articles by <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Harry Campbell</a></div></div><div id="_co_idm140193051413696"><h3 class="no_margin">Malcolm Dunlop</h3><p><sup>15</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK,</p><div>Find articles by <a href="/pubmed/?term=Dunlop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Malcolm Dunlop</a></div></div><div id="_co_idm140193041694592"><h3 class="no_margin">Anna F. Dominiczak</h3><p><sup>17</sup>Institute of Cardiovascular and Medical Sciences, University of Glasgow, Glasgow G12 8QQ, UK,</p><div>Find articles by <a href="/pubmed/?term=Dominiczak%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Anna F. Dominiczak</a></div></div><div id="_co_idm140193041692048"><h3 class="no_margin">Samantha S. Mann</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</p><p><sup>18</sup>St Thomas' Hospital, London SE1 7EH, UK,</p><div>Find articles by <a href="/pubmed/?term=Mann%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Samantha S. Mann</a></div></div><div id="_co_idm140193091102528"><h3 class="no_margin">Sharon A. Jenkins</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</p><p><sup>19</sup>Department of Inherited Cardiovascular Disease, The Heart Hospital, London, W1G 8PH, UK,</p><div>Find articles by <a href="/pubmed/?term=Jenkins%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Sharon A. Jenkins</a></div></div><div id="_co_idm140193050354192"><h3 class="no_margin">Andrew R. Webster</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</p><div>Find articles by <a href="/pubmed/?term=Webster%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Andrew R. Webster</a></div></div><div id="_co_idm140193052199632"><h3 class="no_margin">Alan C. Bird</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</p><div>Find articles by <a href="/pubmed/?term=Bird%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Alan C. Bird</a></div></div><div id="_co_idm140193047387568"><h3 class="no_margin">Mark Lathrop</h3><p><sup>20</sup>Centre National de Génotypage, Institut Génomique, Commissariat à l'Énergie Atomique, Evry, France,</p><p><sup>21</sup>Fondation Jean Dausset-CEPH, Paris, France,</p><div>Find articles by <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Mark Lathrop</a></div></div><div id="_co_idm140193050870624"><h3 class="no_margin">Diana Zelenika</h3><p><sup>20</sup>Centre National de Génotypage, Institut Génomique, Commissariat à l'Énergie Atomique, Evry, France,</p><div>Find articles by <a href="/pubmed/?term=Zelenika%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Diana Zelenika</a></div></div><div id="_co_idm140193053673888"><h3 class="no_margin">Eric H. Souied</h3><p><sup>22</sup>Centre Hospitalier Intercommunal de Créteil, Créteil, France,</p><div>Find articles by <a href="/pubmed/?term=Souied%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Eric H. Souied</a></div></div><div id="_co_idm140193053671344"><h3 class="no_margin">José-Alain Sahel</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>23</sup>Department of Genetics, Institut de la Vision, UPMC Univ Paris 06 UMR_S 968/INSERM U968/CNRS-UMR 7210, Paris F-75012, France,</p><p><sup>24</sup>Clinical Investigation Center 503, Centre Hospitalier National d'Ophtalmologie des Quinze-Vingts, Paris F-75012, France,</p><div>Find articles by <a href="/pubmed/?term=Sahel%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">José-Alain Sahel</a></div></div><div id="_co_idm140193044001040"><h3 class="no_margin">Thierry Léveillard</h3><p><sup>23</sup>Department of Genetics, Institut de la Vision, UPMC Univ Paris 06 UMR_S 968/INSERM U968/CNRS-UMR 7210, Paris F-75012, France,</p><div>Find articles by <a href="/pubmed/?term=L%26%23x000e9%3Bveillard%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Thierry Léveillard</a></div></div><div id="_co_idm140193054479280"><h3 class="no_margin">Angela J. Cree</h3><p><sup>25</sup>Clinical Neurosciences Research Grouping, Clinical and Experimental Sciences,  and</p><div>Find articles by <a href="/pubmed/?term=Cree%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Angela J. Cree</a></div></div><div id="_co_idm140193127731312"><h3 class="no_margin">Jane Gibson</h3><p><sup>26</sup>Genetic Epidemiology and Genomic Informatics Group, Human Genetics, Faculty of Medicine, University of Southampton, Southampton SO16 6YD, UK,  and</p><div>Find articles by <a href="/pubmed/?term=Gibson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Jane Gibson</a></div></div><div id="_co_idm140193127728768"><h3 class="no_margin">Sarah Ennis</h3><p><sup>26</sup>Genetic Epidemiology and Genomic Informatics Group, Human Genetics, Faculty of Medicine, University of Southampton, Southampton SO16 6YD, UK,  and</p><div>Find articles by <a href="/pubmed/?term=Ennis%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Sarah Ennis</a></div></div><div id="_co_idm140193056564096"><h3 class="no_margin">Andrew J. Lotery</h3><p><sup>25</sup>Clinical Neurosciences Research Grouping, Clinical and Experimental Sciences,  and</p><p><sup>27</sup>Southampton Eye Unit, Southampton General Hospital, Southampton SO16 6YD, UK</p><div>Find articles by <a href="/pubmed/?term=Lotery%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Andrew J. Lotery</a></div></div><div id="_co_idm140193092296256"><h3 class="no_margin">Alan F. Wright</h3><p><sup>12</sup>Medical Research Council Human Genetics Unit, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, UK,</p><div>Find articles by <a href="/pubmed/?term=Wright%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">Alan F. Wright</a></div></div><div id="_co_idm140193054630192"><h3 class="no_margin">David G. Clayton</h3><p><sup>3</sup>Department of Medical Genetics,, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK,</p><div>Find articles by <a href="/pubmed/?term=Clayton%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">David G. Clayton</a></div></div><div id="_co_idm140193043694928"><h3 class="no_margin">John R.W. Yates</h3><p><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</p><p><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</p><p><sup>3</sup>Department of Medical Genetics,, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK,</p><div>Find articles by <a href="/pubmed/?term=Yates%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22694956">John R.W. Yates</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140193053350400_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140193053350400_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140193053350400_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140193053350400_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Institute of Ophthalmology, University College London, London EC1V 9EL, UK,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Moorfields Eye Hospital, London EC1V 2PD, UK,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Medical Genetics,, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Ophthalmology and Visual Sciences, The Chinese University of Hong Kong, Hong Kong, China,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>UCL Genetics Institute, London WC1E 6BT, UK,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Department of Infectious and Tropical Diseases, London School of Hygiene &amp; Tropical Medicine, University of London, London WC1E 7HT, UK,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Ophthalmology, Royal Perth Hospital, Perth, Western Australia 6001, Australia,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Ophthalmology, Addenbrooke's Hospital, Cambridge CB2 0QQ, UK,</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Department of Eye and Vision Sciences, Institute of Ageing and Chronic Disease, UCD Building, University of Liverpool, Liverpool L69 3GA, UK,</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>School of Biomedicine, Faculty of Medical and Human Sciences, University of Manchester, Manchester M13 9PT, UK,</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Centre for Advanced Discovery and Experimental Therapeutics, Manchester Academic Health Science Centre, Central Manchester Foundation Trust, Manchester M13 9PL, UK,</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Medical Research Council Human Genetics Unit, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, UK,</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Department of Ophthalmology, University of Edinburgh and Princess Alexandra Eye Pavilion, Edinburgh EH3 9HA, UK,</div><div class="fm-affl" id="af14" lang="en"><sup>14</sup>Department of Psychology, Centre for Cognitive Ageing and Cognitive Epidemiology, University of Edinburgh, Edinburgh EH8 9JZ, UK,</div><div class="fm-affl" id="af15" lang="en"><sup>15</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK,</div><div class="fm-affl" id="af16" lang="en"><sup>16</sup>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, UK,</div><div class="fm-affl" id="af17" lang="en"><sup>17</sup>Institute of Cardiovascular and Medical Sciences, University of Glasgow, Glasgow G12 8QQ, UK,</div><div class="fm-affl" id="af18" lang="en"><sup>18</sup>St Thomas' Hospital, London SE1 7EH, UK,</div><div class="fm-affl" id="af19" lang="en"><sup>19</sup>Department of Inherited Cardiovascular Disease, The Heart Hospital, London, W1G 8PH, UK,</div><div class="fm-affl" id="af20" lang="en"><sup>20</sup>Centre National de Génotypage, Institut Génomique, Commissariat à l'Énergie Atomique, Evry, France,</div><div class="fm-affl" id="af21" lang="en"><sup>21</sup>Fondation Jean Dausset-CEPH, Paris, France,</div><div class="fm-affl" id="af22" lang="en"><sup>22</sup>Centre Hospitalier Intercommunal de Créteil, Créteil, France,</div><div class="fm-affl" id="af23" lang="en"><sup>23</sup>Department of Genetics, Institut de la Vision, UPMC Univ Paris 06 UMR_S 968/INSERM U968/CNRS-UMR 7210, Paris F-75012, France,</div><div class="fm-affl" id="af24" lang="en"><sup>24</sup>Clinical Investigation Center 503, Centre Hospitalier National d'Ophtalmologie des Quinze-Vingts, Paris F-75012, France,</div><div class="fm-affl" id="af25" lang="en"><sup>25</sup>Clinical Neurosciences Research Grouping, Clinical and Experimental Sciences,  and</div><div class="fm-affl" id="af26" lang="en"><sup>26</sup>Genetic Epidemiology and Genomic Informatics Group, Human Genetics, Faculty of Medicine, University of Southampton, Southampton SO16 6YD, UK,  and</div><div class="fm-affl" id="af27" lang="en"><sup>27</sup>Southampton Eye Unit, Southampton General Hospital, Southampton SO16 6YD, UK</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Department of Ocular Biology and Therapeutics, UCL Institute of Ophthalmology, 11-43 Bath Street, London EC1V 9EL, UK. Tel: +44 2076086911; Fax: +44 2076086830; Email: <a href="mailto:dev@null" data-email="ku.ca.lcu@inairpic.v" class="oemail">ku.ca.lcu@inairpic.v</a></div><div id="AN1"><sup>†</sup>Members are listed in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140193053350400_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Feb 25; Revised 2012 May 23; Accepted 2012 Jun 5.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140193053350400_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2012. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3428154/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3428154/bin/supp_21_18_4138__index.html" data-ga-action="click_feat_suppl">supp_21_18_4138__index.html</a><span style="color:gray"> (1.1K)</span></div><div class="small guid">GUID:&nbsp;3D138BBF-A04A-470D-A13E-93B00E746CB0</div>
<div><a href="/pmc/articles/PMC3428154/bin/supp_dds225_dds225supp.doc" data-ga-action="click_feat_suppl">supp_dds225_dds225supp.doc</a><span style="color:gray"> (601K)</span></div><div class="small guid">GUID:&nbsp;A8697EF2-1F61-4251-9B41-8503700BC76E</div>
</div></dd></dl></div><div id="idm140193045995072" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140193045995072title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Age-related macular degeneration (AMD) is a leading cause of visual loss in Western populations. Susceptibility is influenced by age, environmental and genetic factors. Known genetic risk loci do not account for all the heritability. We therefore carried out a genome-wide association study of AMD in the UK population with 893 cases of advanced AMD and 2199 controls. This showed an association with the well-established AMD risk loci <em>ARMS2</em> (age-related maculopathy susceptibility 2)–<em>HTRA1</em> (HtrA serine peptidase 1) (<em>P =</em>2.7 × 10<sup>−72</sup>), <em>CFH</em> (complement factor H) (<em>P =</em>2.3 × 10<sup>−47</sup>), <em>C2</em> (complement component 2)–<em>CFB</em> (complement factor B) (<em>P =</em>5.2 × 10<sup>−9</sup>), <em>C3</em> (complement component 3) (<em>P =</em>2.2 × 10<sup>−3</sup>) and <em>CFI</em> (<em>P =</em>3.6 × 10<sup>−3</sup>) and with more recently reported risk loci at <em>VEGFA</em> (<em>P =</em>1.2 × 10<sup>−3</sup>) and <em>LIPC</em> (hepatic lipase) (<em>P =</em>0.04). Using a replication sample of 1411 advanced AMD cases and 1431 examined controls, we confirmed a novel association between AMD and single-nucleotide polymorphisms on chromosome 6p21.3 at <em>TNXB</em> (tenascin XB)–<em>FKBPL</em> (FK506 binding protein like) [rs12153855/rs9391734; discovery <em>P =</em>4.3 × 10<sup>−7</sup>, replication <em>P =</em>3.0 × 10<sup>−4</sup>, combined <em>P =</em>1.3 × 10<sup>−9</sup>, odds ratio (OR) = 1.4, 95% confidence interval (CI) = 1.3–1.6] and the neighbouring gene <em>NOTCH4</em> (Notch 4) (rs2071277; discovery <em>P =</em>3.2 × 10<sup>−8</sup>, replication <em>P =</em>3.8 × 10<sup>−5</sup>, combined <em>P =</em>2.0 × 10<sup>−11</sup>, OR = 1.3, 95% CI = 1.2–1.4). These associations remained significant in conditional analyses which included the adjacent <em>C2</em>–<em>CFB</em> locus. <em>TNXB</em>, <em>FKBPL</em> and <em>NOTCH4</em> are all plausible AMD susceptibility genes, but further research will be needed to identify the causal variants and determine whether any of these genes are involved in the pathogenesis of AMD.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first">Age-related macular degeneration (AMD) is a leading cause of visual loss in Western populations (<a href="#DDS225C1" rid="DDS225C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224667">1</a>,<a href="#DDS225C2" rid="DDS225C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224647">2</a>), reducing the quality of life of tens of millions of older people worldwide. It affects the macular region of the retina, which has a high density of photoreceptors for detailed central vision. Early in the disease, deposits called drusen form along Bruch's membrane, which separates the retinal pigment epithelium (RPE) from the underlying choroid (<a href="#DDS225C3" rid="DDS225C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224608">3</a>). The later stages of the disease are characterized by focal atrophy of the RPE and overlying photoreceptors (geographic atrophy, GA) and/or growth of new blood vessels from the choroid through Bruch's membrane into the RPE (choroidal neovascularization, CNV) (<a href="#DDS225C3" rid="DDS225C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224594">3</a>). Both of these processes can result in the loss of central vision.</p><p id="__p4" class="p p-last">Susceptibility to AMD is influenced by age, environmental and genetic factors (<a href="#DDS225C3" rid="DDS225C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224690">3</a>). Smoking is the most important environmental risk factor (<a href="#DDS225C4" rid="DDS225C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224646">4</a>). Striking progress has been made in understanding the genetics of AMD (<a href="#DDS225C5" rid="DDS225C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_445453680">5</a>,<a href="#DDS225C6" rid="DDS225C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291170345">6</a>). Common sequence variants in the complement pathway genes <em>CFH</em> (<a href="#DDS225C7" rid="DDS225C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224575">7</a>–<a href="#DDS225C10" rid="DDS225C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224664">10</a>), <em>C2</em> (complement component 2)–<em>CFB</em> (complement factor B) (<a href="#DDS225C11" rid="DDS225C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224617">11</a>,<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224619">12</a>) and <em>C3</em> (complement component 3) (<a href="#DDS225C13" rid="DDS225C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224630">13</a>,<a href="#DDS225C14" rid="DDS225C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224689">14</a>) are established risk factors and there is another risk locus in the vicinity of <em>CFI</em> (complement factor I) (<a href="#DDS225C15" rid="DDS225C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224570">15</a>,<a href="#DDS225C16" rid="DDS225C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224682">16</a>). This and other evidence points to the activation of the alternative complement pathway as an important component of the pathogenesis of AMD (<a href="#DDS225C17" rid="DDS225C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224679">17</a>,<a href="#DDS225C18" rid="DDS225C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224580">18</a>). Variants at the <em>ARMS2</em> (age-related maculopathy susceptibility 2)–<em>HTRA1</em> (HtrA serine peptidase 1) locus are strongly associated with AMD (<a href="#DDS225C19" rid="DDS225C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224656">19</a>,<a href="#DDS225C20" rid="DDS225C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224597">20</a>), but the mechanism is uncertain (<a href="#DDS225C21" rid="DDS225C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224600">21</a>). More recently reported risk loci at <em>LIPC</em> (hepatic lipase) (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224669">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224573">24</a>), <em>CETP</em> (cholesteryl ester transfer protein) (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224697">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224685">24</a>), <em>TIMP3</em> (TIMP metallopeptidase inhibitor 3)–<em>SYN3</em> (synapsin III) (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224654">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224662">24</a>) and <em>VEGFA</em> (vascular endothelial growth factor A) (<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224596">24</a>) implicate lipid metabolism, matrix homeostasis and control of angiogenesis as additional factors in the pathogenesis of AMD. The known genetic risk variants do not account for all the heritability of AMD. We have therefore carried out a genome-wide association study (GWAS) in the UK population to identify additional susceptibility loci.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p5" class="p p-first">We conducted a GWAS in the UK population obtaining genotypes for 893 cases and 2199 controls that passed quality control metrics (Materials and Methods, Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB1" rid-ob="ob-DDS225TB1" co-legend-rid=""><span>1</span></a>, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). Subjects were typed with either the Illumina 300k array (150 cases) or the Illumina 550k array (743 cases and 2199 controls). Quality control criteria (Materials and Methods) were not met for 24 125 single-nucleotide polymorphisms (SNPs) in the 300k array and 68 531 SNPs in the 550k array. A small number of additional SNPs were excluded after visual inspection of their cluster plots so that subsequent analyses were based on 286 135 and 488 867 genotyped SNPs from the 300k and 550k arrays, respectively. We imputed variants using the CEU HapMap II reference panel and combined results from the two arrays following imputation for a total of 2 272 849 SNPs (Materials and Methods). Unless otherwise specified, the results discussed here did not show evidence of heterogeneity between the two platforms. The genomic inflation factor (<em>λ</em>) was 1.014 using all the genome-wide results and 1.007 after excluding the test statistics observed at the previously reported <em>CFH</em>, <em>C2</em>–<em>CFB</em> and <em>ARMS2</em>–<em>HTRA1</em> loci; the quantile–quantile plots showed no widespread departure from the expected distribution of <em>P</em>-values (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figs S1</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">S2</a>). Therefore, no further correction for inflation was applied to the test statistics. Replication studies were conducted on a sample of up to 1411 advanced AMD cases and 1431 controls (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS225TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p6">Gender, age, phenotype and geographic origin of subjects in the discovery samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140193057308752" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Cases (<em>n</em> = 893)</th><th rowspan="1" colspan="1" align="left">Controls (<em>n</em> = 2199)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Female, <em>n</em> (%)</td><td rowspan="1" colspan="1">494 (55.3)</td><td rowspan="1" colspan="1">1117 (50.8)</td></tr><tr><td rowspan="1" colspan="1">Mean age ± SD, years</td><td rowspan="1" colspan="1">78.6 ± 7.5</td><td rowspan="1" colspan="1">44–45<sup>a</sup></td></tr><tr><td colspan="3" rowspan="1">Phenotype</td></tr><tr><td rowspan="1" colspan="1"> Unexamined</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">2199</td></tr><tr><td rowspan="1" colspan="1"> CNV</td><td rowspan="1" colspan="1">593</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> GA</td><td rowspan="1" colspan="1">182</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CNV and GA</td><td rowspan="1" colspan="1">118</td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="3" rowspan="1">Geographic origin</td></tr><tr><td rowspan="1" colspan="1"> L</td><td rowspan="1" colspan="1">80</td><td rowspan="1" colspan="1">156</td></tr><tr><td rowspan="1" colspan="1"> NMESES</td><td rowspan="1" colspan="1">548</td><td rowspan="1" colspan="1">1373</td></tr><tr><td rowspan="1" colspan="1"> NW</td><td rowspan="1" colspan="1">163</td><td rowspan="1" colspan="1">272</td></tr><tr><td rowspan="1" colspan="1"> S</td><td rowspan="1" colspan="1">102</td><td rowspan="1" colspan="1">398</td></tr></tbody></table></div><div id="largeobj_idm140193057308752" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3428154/table/DDS225TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140193053623968"><p id="__p7" class="p p-first-last">CNV, choroidal neovascularization; GA, geographic atrophy; L, London; NMESES, North Midlands, East, Southeast and South of England; NW, Northwest of England; S, Scotland.</p></div><div id="idm140193044712368"><p id="__p8" class="p p-first-last"><sup>a</sup>Controls were selected from the 58BC (<a href="#DDS225C46" rid="DDS225C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224610">46</a>,<a href="#DDS225C47" rid="DDS225C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224640">47</a>).</p></div></div></div><div id="s2a" class="sec"><h3 id="s2atitle">Previously reported definite and probable AMD-associated loci</h3><p id="__p9" class="p p-first">An association at genome-wide significance level (<em>P</em> &lt; 5 × 10<sup>−8</sup>) was observed at the well-established AMD susceptibility loci <em>ARMS2</em>–<em>HTRA1</em> (rs10490924/rs2284665, <em>P =</em>2.7 × 10<sup>−72</sup>), <em>CFH</em> (rs10801555, an almost perfect proxy for rs1061170, <em>P =</em>2.3 × 10<sup>−47</sup>) and <em>C2</em>–<em>CFB</em> (rs541862, a perfect proxy for rs641153, <em>P =</em>5.2 × 10<sup>−9</sup>). At the <em>CFH</em> locus, an analysis conditional on rs10801555 confirmed a second independent association with rs1329428 (<em>P =</em>1.1 × 10<sup>−10</sup>) (<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224648">12</a>,<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224612">22</a>,<a href="#DDS225C25" rid="DDS225C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224595">25</a>). For the reported independent signal at SNP rs9332739 in <em>C2</em> (<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224632">12</a>), we observed weak statistical support with <em>P =</em>0.02 and no association when conditioning on rs541862 (<em>P =</em>0.78). These results are presented in Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB2" rid-ob="ob-DDS225TB2" co-legend-rid=""><span>2</span></a> together with association signals at other previously reported definite or probable AMD risk loci.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS225TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p10">Association analysis in the discovery samples for previously reported definite or probable AMD susceptibility loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140193051014688" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Locus</th><th rowspan="2" colspan="1" align="left">Reported SNP/proxy SNP (<em>R</em><sup>2</sup>)</th><th rowspan="2" colspan="1" align="left">Reference</th><th rowspan="2" colspan="1" align="left">Chr</th><th rowspan="2" colspan="1" align="left">Position (bp)</th><th rowspan="2" colspan="1" align="left">Effect allele (EA)/other allele</th><th colspan="2" rowspan="1" align="left">EA frequency<hr></th><th rowspan="2" colspan="1" align="left"><em>P</em></th><th rowspan="2" colspan="1" align="left">OR</th><th rowspan="2" colspan="1" align="left">95% CI</th></tr><tr><th rowspan="1" colspan="1" align="left">Cases</th><th rowspan="1" colspan="1" align="left">Controls</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>ARMS2</em>–<em>HTRA1</em></td><td rowspan="1" colspan="1">rs10490924</td><td rowspan="1" colspan="1">(<a href="#DDS225C19" rid="DDS225C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224688">19</a>,<a href="#DDS225C20" rid="DDS225C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224683">20</a>)</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">124 204 440</td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">2.7 × 10<sup>−72</sup></td><td rowspan="1" colspan="1">3.00</td><td rowspan="1" colspan="1">2.64–3.40</td></tr><tr><td rowspan="1" colspan="1"><em>CFH</em></td><td rowspan="1" colspan="1">rs1061170/rs10801555 (0.97)</td><td rowspan="1" colspan="1">(<a href="#DDS225C10" rid="DDS225C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224618">10</a>)</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">194 926 880</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">2.3 × 10<sup>−47</sup></td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">0.38–0.48</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs1410996/rs1329428 (1.0)</td><td rowspan="1" colspan="1">(<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224684">12</a>)</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">194 969 430</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">3.1 × 10<sup>−43</sup></td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">0.35–0.45</td></tr><tr><td rowspan="1" colspan="1"><em>C2</em>–<em>CFB</em></td><td rowspan="1" colspan="1">rs641153/rs541862 (1.0)</td><td rowspan="1" colspan="1">(<a href="#DDS225C11" rid="DDS225C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224653">11</a>,<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224628">12</a>)</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 024 930</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.95</td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1">5.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">1.97</td><td rowspan="1" colspan="1">1.56–2.49</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs9332739</td><td rowspan="1" colspan="1">(<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224668">12</a>)</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 011 783</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.47</td><td rowspan="1" colspan="1">1.06–2.04</td></tr><tr><td rowspan="1" colspan="1"><em>VEGFA</em></td><td rowspan="1" colspan="1">rs4711751/rs943080 (1.0)</td><td rowspan="1" colspan="1">(<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224671">24</a>)</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">43 934 605</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">1.6 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">1.20</td><td rowspan="1" colspan="1">1.07–1.35</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs833069</td><td rowspan="1" colspan="1">(<a href="#DDS225C26" rid="DDS225C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224588">26</a>)</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">43 850 557</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">0.68</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">0.82–1.04</td></tr><tr><td rowspan="1" colspan="1"><em>C3</em></td><td rowspan="1" colspan="1">rs2230199</td><td rowspan="1" colspan="1">(<a href="#DDS225C13" rid="DDS225C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224620">13</a>,<a href="#DDS225C14" rid="DDS225C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224652">14</a>)</td><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">6 669 387</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1">0.76</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">2.2 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.69–0.92</td></tr><tr><td rowspan="1" colspan="1"><em>CCDC109B</em>–<em>
PLA2G12A</em>–<em>CFI</em></td><td rowspan="1" colspan="1">rs7690921</td><td rowspan="1" colspan="1">(<a href="#DDS225C23" rid="DDS225C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224663">23</a>)</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">110 787 610</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">3.6 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">1.19</td><td rowspan="1" colspan="1">1.06–1.34</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs10033900</td><td rowspan="1" colspan="1">(<a href="#DDS225C16" rid="DDS225C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224650">16</a>)</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">110 878 520</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.51</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">1.07</td><td rowspan="1" colspan="1">0.96–1.20</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2285714</td><td rowspan="1" colspan="1">(<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224598">22</a>)</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">110 858 259</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.89–1.11</td></tr><tr><td rowspan="1" colspan="1"><em>LIPC</em></td><td rowspan="1" colspan="1">rs493258</td><td rowspan="1" colspan="1">(<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224626">22</a>,<a href="#DDS225C23" rid="DDS225C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224607">23</a>)</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">56 475 172</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">0.79–0.99</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs10468017</td><td rowspan="1" colspan="1">(<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224637">24</a>)</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">56 465 804</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.80–1.03</td></tr><tr><td rowspan="1" colspan="1"><em>CETP</em></td><td rowspan="1" colspan="1">rs3764261</td><td rowspan="1" colspan="1">(<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224571">22</a>-<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224574">24</a>)</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">55 550 825</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">0.67</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">0.83–1.05</td></tr><tr><td rowspan="1" colspan="1"><em>SYN3</em>–<em>TIMP3</em></td><td rowspan="1" colspan="1">rs9621532</td><td rowspan="1" colspan="1">(<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224680">22</a>-<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224581">24</a>)</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">31 414 511</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.69–1.20</td></tr></tbody></table></div><div id="largeobj_idm140193051014688" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3428154/table/DDS225TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140193050091872"><p id="__p11" class="p p-first-last">Chr, chromosome; bp, base pair; Effect allele, allele for which the ORs and 95% CIs were calculated; <em>P</em>, <em>P</em>-value from the stratified analysis using the Mantel's extension of the 1 df Cochrane–Armitage trend test. Heterogeneity <em>P</em>-value for SNP rs10468017 = 0.03.</p></div></div></div><p id="__p12">At the <em>LIPC</em> locus, the reported SNP rs493258 (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224633">22</a>,<a href="#DDS225C23" rid="DDS225C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224569">23</a>) showed weak evidence of association (<em>P =</em>0.04) but in the same direction as previously reported [odds ratio (OR) = 0.89 and 95% confidence interval (CI) = 0.79–0.99 for allele T]. There was no evidence of an association with the reported functional variant rs10468017 in this gene (<em>R</em><sup>2</sup>= 0.42; <em>P =</em>0.11, OR = 0.91 and 95% CI = 0.80–1.03 for allele T) (<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224705">24</a>). We found an association with SNP rs943080 at the <em>VEGFA</em> locus (<em>P</em> = 1.6 × 10<sup>−3</sup>, a perfect proxy for the reported SNP rs4711751, OR = 1.20 and 95% CI = 1.07–1.35 for allele T) (<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224623">24</a>), but no association with the other reported SNP rs833069 (<em>P =</em>0.18, OR = 0.92 and 95% CI = 0.82–1.04) (<a href="#DDS225C26" rid="DDS225C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224629">26</a>). At the CFI locus, evidence of association was found with the previously reported SNP rs7690921 in <em>CCDC109B</em> (coiled-coil domain containing 109B) (<em>P</em> = 3.6 × 10<sup>−3</sup>, OR = 1.19 and 95% CI = 1.06–1.34 for allele T) (<a href="#DDS225C16" rid="DDS225C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224707">16</a>,<a href="#DDS225C23" rid="DDS225C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224699">23</a>), but not for rs10033900 (<em>P</em>= 0.22) (<a href="#DDS225C16" rid="DDS225C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224659">16</a>) or rs2285714 (<em>P</em>= 0.92) (<a href="#DDS225C16" rid="DDS225C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224641">16</a>,<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224609">22</a>). We did not find support for the previously reported association with variants at <em>CETP</em> (rs3764261, <em>P</em> = 0.26) (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224675">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224643">24</a>) or <em>SYN3</em>-<em>TIMP3</em> (rs9621532, <em>P</em> = 0.48) (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224703">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224706">24</a>).</p><p id="__p13" class="p p-last">At the <em>APOE</em> (apolipoprotein E) locus, the two SNPs that determine the E2/E3/E4 <em>APOE</em> alleles (rs429358 and rs7412) were not typed and could not be imputed. However, there was evidence of an association with rs2075650 (<em>P</em> = 3.1 × 10<sup>−6</sup>) in the same region, although with some degree of heterogeneity between the 300k and 550k arrays (<em>P</em> = 0.06). SNP rs2075650 lies within the <em>TOMM40</em> (translocase of outer mitochondrial membrane 40 homolog) gene and is in modest linkage disequilibrium (LD) with rs429358 (<em>R</em><sup>2</sup> = 0.2). We genotyped rs2075650 and <em>APOE</em> SNP rs429358 in our replication samples (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>) and confirmed the known protective effect of the C allele of rs429358 (<em>P</em> = 2.8 × 10<sup>−6</sup>; heterogeneity <em>P</em> = 0.04; OR = 0.68; 95% CI = 0.58–0.80) and the association with rs2075650 (<em>P</em> = 0.0037; OR = 0.78; 95% CI = 0.67–0.92 for allele G), although some heterogeneity across the follow-up samples was observed (<em>P</em> = 0.03) (Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB3" rid-ob="ob-DDS225TB3" co-legend-rid=""><span>3</span></a>). However, the association with rs2075650 became non-significant after conditioning on rs429358 (<em>P =</em>0.64).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS225TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p14">Association analysis for SNPs in <em>TNXB</em> and <em>NOTCH4</em> and other SNPs typed in the replication studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140193048536672" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="left">Locus</th><th rowspan="2" colspan="1" align="left">Chr</th><th rowspan="2" colspan="1" align="left">Position (bp)</th><th rowspan="2" colspan="1" align="left">Effect allele (EA)/other allele</th><th colspan="2" rowspan="1" align="left">Discovery samples (893 cases, 2199 controls)<hr></th><th colspan="2" rowspan="1" align="left">English replication samples (1267 cases, 1102 controls)<hr></th><th colspan="2" rowspan="1" align="left">Scottish replication samples (144 cases, 329 controls)<hr></th><th colspan="2" rowspan="1" align="left">Replication samples combined (1411 cases, 1431 controls)<hr></th><th colspan="3" rowspan="1" align="left">All samples combined (2304 cases, 3630 controls)<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">EA frequency (cases/controls)</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">EA frequency (cases/controls)</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">EA frequency (cases/controls)</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">Het <em>P</em></th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left">Het <em>P</em></th></tr></thead><tbody><tr><td colspan="16" rowspan="1">Newly identified SNPs</td></tr><tr><td rowspan="1" colspan="1"> rs12153855</td><td rowspan="1" colspan="1"><em>TNXB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 182 782</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.023/0.011</td><td rowspan="1" colspan="1">4.3 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.016/0.008</td><td rowspan="1" colspan="1">6.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.022/0.019</td><td rowspan="1" colspan="1">0.9266</td><td rowspan="1" colspan="1">3.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.0959</td><td rowspan="1" colspan="1">1.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">1.44 (1.28–1.63)</td><td rowspan="1" colspan="1">0.3892</td></tr><tr><td rowspan="1" colspan="1"> rs2071277</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 279 661</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.277/0.219</td><td rowspan="1" colspan="1">3.2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.271/0.222</td><td rowspan="1" colspan="1">2.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.222/0.212</td><td rowspan="1" colspan="1">0.5481</td><td rowspan="1" colspan="1">3.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.3004</td><td rowspan="1" colspan="1">2.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">1.30 (1.20–1.41)</td><td rowspan="1" colspan="1">0.1869</td></tr><tr><td colspan="16" rowspan="1">Other SNPs typed in the replication studies</td></tr><tr><td rowspan="1" colspan="1"> rs13095226</td><td rowspan="1" colspan="1"><em>COL8A1</em></td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">100 878 962</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.124/0.100</td><td rowspan="1" colspan="1">4.0 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">0.105/0.099</td><td rowspan="1" colspan="1">0.4807</td><td rowspan="1" colspan="1">0.128/0.113</td><td rowspan="1" colspan="1">0.5205</td><td rowspan="1" colspan="1">0.3690</td><td rowspan="1" colspan="1">0.7481</td><td rowspan="1" colspan="1">0.0059</td><td rowspan="1" colspan="1">1.19 (1.05–1.35)</td><td rowspan="1" colspan="1">0.5970</td></tr><tr><td rowspan="1" colspan="1"> rs17778253</td><td rowspan="1" colspan="1"><em>COL8A1</em></td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">100 917 792</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.113/0.084</td><td rowspan="1" colspan="1">4.3 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.095/0.087</td><td rowspan="1" colspan="1">0.3432</td><td rowspan="1" colspan="1">0.112/0.098</td><td rowspan="1" colspan="1">0.5501</td><td rowspan="1" colspan="1">0.2703</td><td rowspan="1" colspan="1">0.8408</td><td rowspan="1" colspan="1">0.0011</td><td rowspan="1" colspan="1">1.24 (1.09–1.42)</td><td rowspan="1" colspan="1">0.5490</td></tr><tr><td rowspan="1" colspan="1"> rs541862</td><td rowspan="1" colspan="1"><em>CFB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 024 930</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.003/0.010</td><td rowspan="1" colspan="1">5.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.005/0.012</td><td rowspan="1" colspan="1">9.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.003/0</td><td rowspan="1" colspan="1">0.0777</td><td rowspan="1" colspan="1">2.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.5955</td><td rowspan="1" colspan="1">8.8 × 10<sup>−17</sup></td><td rowspan="1" colspan="1">0.53 (0.46–0.62)</td><td rowspan="1" colspan="1">0.9112</td></tr><tr><td rowspan="1" colspan="1"> rs3132946</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 298 006</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.009/0.019</td><td rowspan="1" colspan="1">2.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">0.024/0.019</td><td rowspan="1" colspan="1">0.7380</td><td rowspan="1" colspan="1">0.028/0.022</td><td rowspan="1" colspan="1">0.9694</td><td rowspan="1" colspan="1">0.7691</td><td rowspan="1" colspan="1">0.8690</td><td rowspan="1" colspan="1">2.9 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">0.88 (0.79–0.99)</td><td rowspan="1" colspan="1">0.4089</td></tr><tr><td rowspan="1" colspan="1"> rs12231166</td><td rowspan="1" colspan="1"><em>GOLGA2P5</em>–<em>ACTR6</em></td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">20 332 348</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1">0.164/0.116</td><td rowspan="1" colspan="1">6.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.134/0.135</td><td rowspan="1" colspan="1">0.9312</td><td rowspan="1" colspan="1">0.135/0.127</td><td rowspan="1" colspan="1">0.7379</td><td rowspan="1" colspan="1">0.9687</td><td rowspan="1" colspan="1">0.7313</td><td rowspan="1" colspan="1">5.3 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">1.22 (1.09–1.37)</td><td rowspan="1" colspan="1">0.0026</td></tr><tr><td rowspan="1" colspan="1"> rs2075650</td><td rowspan="1" colspan="1"><em>TOMM40</em></td><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">50 087 459</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.105/0.149</td><td rowspan="1" colspan="1">3.1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.007/0.022</td><td rowspan="1" colspan="1">5.6 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.016/0.006</td><td rowspan="1" colspan="1">0.2288</td><td rowspan="1" colspan="1">0.0037</td><td rowspan="1" colspan="1">0.0261</td><td rowspan="1" colspan="1">8.4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.73 (0.65–0.82)</td><td rowspan="1" colspan="1">0.0148</td></tr><tr><td rowspan="1" colspan="1"> rs429358</td><td rowspan="1" colspan="1"><em>APOE</em></td><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">50 103 781</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.006/0.024</td><td rowspan="1" colspan="1">3.4 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.008/0.003</td><td rowspan="1" colspan="1">0.7469</td><td rowspan="1" colspan="1">2.8 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.0414</td><td rowspan="1" colspan="1">2.8 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.68 (0.58–0.80)</td><td rowspan="1" colspan="1">0.0414</td></tr></tbody></table></div><div id="largeobj_idm140193048536672" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3428154/table/DDS225TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140193044280224"><p id="__p15" class="p p-first-last">Chr, chromosome; bp, base pair; <em>P</em>, <em>P</em>-value from the stratified analysis using the Mantel's extension of the 1 df Cochrane–Armitage trend test; Het <em>P</em>, heterogeneity <em>P</em>-value. For the GOLGA2P5–ACTR6 locus, proxy SNP rs476497 (<em>R</em><sup>2</sup>=0.92) was followed up in the replication samples.</p></div></div></div></div><div id="s2b" class="sec"><h3 id="s2btitle">Novel loci</h3><p id="__p16" class="p p-first">Using a threshold of <em>P</em> &lt; 10<sup>−6</sup> and excluding previously known associations, we identified four novel associated SNPs. For one of these, rs12231166 (<em>P</em> = 6.2 × 10<sup>−7</sup>) at 12q23.1, we genotyped the proxy rs476497 (<em>R</em><sup>2</sup> = 0.92) in our replication samples (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>), but found no evidence of association (replication <em>P</em> = 0.97) (Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB3" rid-ob="ob-DDS225TB3" co-legend-rid=""><span>3</span></a>).</p><p id="__p17">The other novel SNPs were rs2071277 in <em>NOTCH4</em> (Notch 4) (<em>P</em> = 3.2 × 10<sup>−8</sup>), rs12153855 in <em>TNXB</em> (tenascin XB) (<em>P</em> = 4.3 × 10<sup>−7</sup>) and the imputed SNP rs9391734 (a perfect proxy for rs12153855) in <em>FKBPL</em> (FK506 binding protein like) (<em>P</em>= 4.3 × 10<sup>−7</sup>), all located at 6p21.3 and ∼250 kb from <em>C2</em>–<em>CFB</em> (Fig.&nbsp;<a href="/pmc/articles/PMC3428154/figure/DDS225F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDS225F1" rid-ob="ob-DDS225F1" co-legend-rid="lgnd_DDS225F1"><span>1</span></a>)<em>.</em> These SNPs are in low LD with rs541862 (<em>R</em><sup>2</sup>= 0.02 and <em>R</em><sup>2</sup>= 0.01 for rs2071277 and rs12153855/rs9391734, respectively), the SNP in <em>CFB</em> which has a well-established association with AMD (<a href="#DDS225C11" rid="DDS225C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224677">11</a>,<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224692">12</a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDS225F1" co-legend-rid="lgnd_DDS225F1"><a href="/pmc/articles/PMC3428154/figure/DDS225F1/" target="figure" rid-figpopup="DDS225F1" rid-ob="ob-DDS225F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140193046813392" class="figure"><a href="/pmc/articles/PMC3428154/figure/DDS225F1/" target="figure" rid-figpopup="DDS225F1" rid-ob="ob-DDS225F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3428154_dds22501.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is dds22501.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3428154/bin/dds22501.jpg"></a></div><div id="largeobj_idm140193046813392" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3428154/figure/DDS225F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDS225F1"><div><a class="figpopup" href="/pmc/articles/PMC3428154/figure/DDS225F1/" target="figure" rid-figpopup="DDS225F1" rid-ob="ob-DDS225F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p18">Regional plots of association on chromosome 6p21.3 in the discovery samples (<strong>A</strong>) for the established association signal at the <em>C2–CFB</em> locus (<strong>B</strong>) for the newly identified association in the <em>TNXB–FKBPL–NOTCH4</em> region. The most associated SNP in each region is denoted by a purple diamond and other SNPs by circles coloured to reflect their degree of LD with the most associated SNP (based on HapMap II).</p></div></div></div><p id="__p19">Although our discovery data showed no overall evidence of residual population structure (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>), the stratification by geographic region used in our association analysis might be unsatisfactory in genomic regions such as 6p21.3 which contain long-range haplotypes that can be sensitive to population structure. Therefore, we also performed an association analysis by including the first two principal components (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3A</a>) as covariates in the logistic regression model and repeated analogous analysis after excluding ethnic outliers (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3B</a>). We observed negligible effect of these adjustments on the <em>P</em>-values and the effect size estimates (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>).</p><p id="__p20">Conditioning for the effect of rs541862 (<em>P</em> = 5.2 × 10<sup>−9</sup>) did not eliminate evidence of association for several SNPs spanning the <em>TNXB</em>–<em>NOTCH4</em> locus (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>, Fig.&nbsp;<a href="/pmc/articles/PMC3428154/figure/DDS225F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDS225F1" rid-ob="ob-DDS225F1" co-legend-rid="lgnd_DDS225F1"><span>1</span></a>B). Together with rs541862 in <em>CFB</em>, we selected rs12153855 in <em>TNXB</em>for follow-up, which showed a conditioned <em>P</em> = 9.0 × 10<sup>−6</sup>, and rs2071277 and rs3132946 in <em>NOTCH4</em> for which we observed some residual evidence of association after controlling for both rs541862 and rs12153855 (<em>P</em> = 3.8 × 10<sup>−3</sup> and <em>P</em> = 3.5 × 10<sup>−3</sup>, respectively). Corresponding replication results are summarized in Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB3" rid-ob="ob-DDS225TB3" co-legend-rid=""><span>3</span></a>. As expected, the known protective association at the C allele of rs541862 was confirmed (<em>P</em> = 8.8 × 10<sup>−17</sup>; OR = 0.53; 95% CI = 0.46–0.62 when combining all samples). Association at both rs2071277 and rs12153855 was consistently corroborated across the replication samples (<em>P =</em>3.8 × 10<sup>−5</sup> and 3.0 × 10<sup>−4</sup>, respectively) and reached genome-wide significance when combining all samples (<em>P =</em>2.0 × 10<sup>−11</sup>; OR = 1.30; 95% CI = 1.20–1.41 for allele C of rs2071277; <em>P =</em>1.3 × 10<sup>−9</sup>; OR = 1.44; 95% CI = 1.28–1.63 for allele C of rs12153855). Repeating the conditional analysis, we found that both rs12153855 in <em>TNXB</em> and rs2071277 in <em>NOTCH4</em> conferred independent risk beyond the known signal at the <em>C2</em>–<em>CFB</em> locus as shown in Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB4" rid-ob="ob-DDS225TB4" co-legend-rid=""><span>4</span></a>. Either conditioning on rs541862 and rs12153855 or on rs541862 and rs2071277 did not eliminate the effect at the third SNP with a conditioned <em>P</em>-value of 9.0 × 10<sup>−4</sup> for rs2071277 and 2.5 × 10<sup>−4</sup> for rs12153855.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS225TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p21">Conditional association analysis in the extended C2/CFB region</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140193048018528" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Test SNP</th><th rowspan="2" colspan="1" align="left">Gene</th><th rowspan="2" colspan="1" align="left">Chr</th><th rowspan="2" colspan="1" align="left">Position (bp)</th><th rowspan="2" colspan="1" align="left">Conditioning SNP(s)</th><th colspan="2" rowspan="1" align="left">Discovery samples (893 cases, 2199 controls)<hr></th><th colspan="2" rowspan="1" align="left">Replication samples (1411 cases, 1431 controls)<hr></th><th colspan="2" rowspan="1" align="left">All samples combined (2304 cases, 3630 controls)<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Unconditioned <em>P</em></th><th rowspan="1" colspan="1" align="left">Conditioned <em>P</em></th><th rowspan="1" colspan="1" align="left">Unconditioned <em>P</em></th><th rowspan="1" colspan="1" align="left">Conditioned <em>P</em></th><th rowspan="1" colspan="1" align="left">Unconditioned <em>P</em></th><th rowspan="1" colspan="1" align="left">Conditioned <em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs541862</td><td rowspan="1" colspan="1"><em>CFB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 024 930</td><td rowspan="4" colspan="1">rs541862</td><td rowspan="1" colspan="1">4.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">8.8 × 10<sup>−17</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs12153855</td><td rowspan="1" colspan="1"><em>TNXB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 182 782</td><td rowspan="1" colspan="1">4.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">9.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">2.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.0024</td><td rowspan="1" colspan="1">1.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">1.6 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">rs2071277</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 279 661</td><td rowspan="1" colspan="1">3.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">2.3 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">3.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.0048</td><td rowspan="1" colspan="1">2.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">1.1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs3132946</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 298 006</td><td rowspan="1" colspan="1">2.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">4.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">2.5 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1">rs541862</td><td rowspan="1" colspan="1"><em>CFB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 024 930</td><td rowspan="4" colspan="1">rs541862, rs12153855</td><td rowspan="1" colspan="1">4.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">8.8 × 10<sup>−17</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs12153855</td><td rowspan="1" colspan="1"><em>TNXB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 182 782</td><td rowspan="1" colspan="1">4.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">1.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs2071277</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 279 661</td><td rowspan="1" colspan="1">3.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">3.8 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">3.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">2.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">9.1 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1">rs3132946</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 298 006</td><td rowspan="1" colspan="1">2.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">3.5 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">rs541862</td><td rowspan="1" colspan="1"><em>CFB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 024 930</td><td rowspan="4" colspan="1">rs541862, rs2071277</td><td rowspan="1" colspan="1">4.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">8.8 × 10<sup>−17</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs12153855</td><td rowspan="1" colspan="1"><em>TNXB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 182 782</td><td rowspan="1" colspan="1">4.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">2.0 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">2.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">1.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">2.5 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1">rs2071277</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 279 661</td><td rowspan="1" colspan="1">3.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">3.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs3132946</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 298 006</td><td rowspan="1" colspan="1">2.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">0.95</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.21</td></tr><tr><td rowspan="1" colspan="1">rs541862</td><td rowspan="1" colspan="1"><em>CFB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 024 930</td><td rowspan="4" colspan="1">rs541862, rs3132946</td><td rowspan="1" colspan="1">4.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">8.8 × 10<sup>−17</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs12153855</td><td rowspan="1" colspan="1"><em>TNXB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 182 782</td><td rowspan="1" colspan="1">4.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">7.7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">2.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">1.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">1.2 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs2071277</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 279 661</td><td rowspan="1" colspan="1">3.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">2.2 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">3.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">2.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">1.4 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1">rs3132946</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 298 006</td><td rowspan="1" colspan="1">2.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs541862</td><td rowspan="1" colspan="1"><em>CFB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 024 930</td><td rowspan="4" colspan="1">rs541862, rs12153855, rs3132946</td><td rowspan="1" colspan="1">4.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">8.8 × 10<sup>−17</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs12153855</td><td rowspan="1" colspan="1"><em>TNXB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 182 782</td><td rowspan="1" colspan="1">4.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">1.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs2071277</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 279 661</td><td rowspan="1" colspan="1">3.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">3.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">2.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.01</td></tr><tr><td rowspan="1" colspan="1">rs3132946</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 298 006</td><td rowspan="1" colspan="1">2.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs541862</td><td rowspan="1" colspan="1"><em>CFB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 024 930</td><td rowspan="4" colspan="1">rs541862, rs12153855, rs2071277</td><td rowspan="1" colspan="1">4.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.2 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">8.8 × 10<sup>−17</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs12153855</td><td rowspan="1" colspan="1"><em>TNXB</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 182 782</td><td rowspan="1" colspan="1">4.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">1.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs2071277</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 279 661</td><td rowspan="1" colspan="1">3.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">3.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">2.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">N/A</td></tr><tr><td rowspan="1" colspan="1">rs3132946</td><td rowspan="1" colspan="1"><em>NOTCH4</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 298 006</td><td rowspan="1" colspan="1">2.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.22</td></tr></tbody></table></div><div id="largeobj_idm140193048018528" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3428154/table/DDS225TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140193048720000"><p id="__p22" class="p p-first-last">Chr, chromosome; bp, base pair; <em>P</em>, <em>P</em>-value from the stepwise logistic regression using the Wald test on the beta coefficients. LD based on HapMap II: <em>R</em><sup>2</sup>(rs541862, rs12153855) = 0.01; <em>R</em><sup>2</sup>(rs541862, rs2071277) = 0.02; <em>R</em><sup>2</sup>(rs541862, rs3132946) = 0.02; <em>R</em><sup>2</sup>(rs12153855, rs2071277) = 0.14; <em>R</em><sup>2</sup>(rs12153855, rs3132946) = 0.02; <em>R</em><sup>2</sup>(rs2071277, rs3132946) = 0.17.</p></div></div></div><p id="__p23" class="p p-last">To further characterize the association in this region, we carried out a haplotype analysis of rs541862, rs12153855 and rs2071277. Four haplotypes at frequency &gt;1% accounted for 98.9% of the total haplotypes observed (Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB5/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB5" rid-ob="ob-DDS225TB5" co-legend-rid=""><span>5</span></a>). The most frequent haplotype TTT was used as reference to calculate ORs. Distinct levels of AMD risk with non-overlapping CIs were associated with these four haplotypes (<em>P =</em>9.1 × 10<sup>−23</sup>). The known protective haplotype (CTT) tagged by the C allele of rs541862 was present in 9.4% of controls and 5.1% of cases after combining all samples. The other two haplotypes TTC and TCC that do not harbour the protective C allele of rs541862 significantly differed from the baseline haplotype TTT with OR = 1.14 (95% CI = 1.05–1.25) and OR = 1.47 (95% CI = 1.29–1.67), respectively.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDS225TB5"><h3>Table&nbsp;5.</h3><!--caption a7--><div class="caption"><p id="__p24">Association analysis for the common haplotypes spanning <em>CFB</em>–<em>TNXB</em>–<em>NOTCH4</em></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140193048573008" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="3" rowspan="1" align="left">Haplotype<hr></th><th colspan="2" rowspan="1" align="left">Discovery samples (893 cases, 2199 controls)</th><th colspan="2" rowspan="1" align="left">Replication samples (1411 cases, 1431 controls)</th><th colspan="4" rowspan="1" align="left">All samples combined (2304 cases, 3630 controls)</th></tr><tr><th rowspan="2" colspan="1" align="left">rs541862 (CFB)</th><th rowspan="2" colspan="1" align="left">rs12153855 (TNXB)</th><th rowspan="2" colspan="1" align="left">rs2071277 (NOTCH4)</th><th rowspan="2" colspan="1" align="left">OR</th><th rowspan="2" colspan="1" align="left">95% CI</th><th rowspan="2" colspan="1" align="left">OR</th><th rowspan="2" colspan="1" align="left">95% CI</th><th colspan="2" rowspan="1" align="left">Haplotype frequency<hr></th><th rowspan="2" colspan="1" align="left">OR</th><th rowspan="2" colspan="1" align="left">95% CI</th></tr><tr><th rowspan="1" colspan="1" align="left">Cases</th><th rowspan="1" colspan="1" align="left">Controls</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.423</td><td rowspan="1" colspan="1">0.449</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.18</td><td rowspan="1" colspan="1">1.04–1.34</td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1">1.00–1.27</td><td rowspan="1" colspan="1">0.394</td><td rowspan="1" colspan="1">0.359</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">1.05–1.25</td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.62</td><td rowspan="1" colspan="1">1.35–1.94</td><td rowspan="1" colspan="1">1.35</td><td rowspan="1" colspan="1">1.12–1.62</td><td rowspan="1" colspan="1">0.132</td><td rowspan="1" colspan="1">0.098</td><td rowspan="1" colspan="1">1.47</td><td rowspan="1" colspan="1">1.29–1.67</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.42–0.71</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">0.46–0.73</td><td rowspan="1" colspan="1">0.051</td><td rowspan="1" colspan="1">0.094</td><td rowspan="1" colspan="1">0.56</td><td rowspan="1" colspan="1">0.48–0.67</td></tr></tbody></table></div><div id="largeobj_idm140193048573008" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3428154/table/DDS225TB5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140193048659232"><p id="__p25" class="p p-first-last">TTT, TTC, TCC and CTT account for 98.9% of the total haplotypes observed.</p></div></div></div></div><div id="s2c" class="sec"><h3 id="s2ctitle">Previously reported putative AMD-associated loci</h3><p id="__p26" class="p p-first-last">A literature search for studies reporting evidence of an association between a common SNP and AMD identified a total of 47 loci (Materials and Methods) in addition to the established and probable loci discussed earlier (Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB2" rid-ob="ob-DDS225TB2" co-legend-rid=""><span>2</span></a>). We reviewed our GWAS discovery data for the most associated SNP reported at each of these putative loci (or a proxy for that SNP). Association results for 45 of these SNPs were available as summarized in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>. Using a Bonferroni-corrected <em>P</em>-value threshold of 0.001 to account for multiple testing, only rs429608 in <em>SKIV2L</em> (superkiller viralicidic activity 2-like) was clearly significant (<em>P =</em>1.0 × 10<sup>−6</sup>). This SNP lies 13 kb from the established <em>C2</em>–<em>CFB</em> locus (<em>R</em><sup>2</sup> = 0.38) and had been reported to confer an independent protective effect (<a href="#DDS225C27" rid="DDS225C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224673">27</a>). In our conditional analysis of the extended <em>C2</em>–<em>CFB</em> region, the association signal at rs429608 disappeared after conditioning on rs550605 (<em>C2</em>), rs541862 (<em>CFB</em>) or rs438999 (<em>SKIV2L</em>) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>). SNP rs13095226 in <em>COL8A1</em> (collagen, type VIII, alpha 1) approached the Bonferroni-corrected level of significance (<em>P =</em>4.1 × 10<sup>−3</sup>), and rs17778253 in the same gene (<em>R</em><sup>2</sup> = 0.74) showed stronger evidence of association (<em>P =</em>4.3 × 10<sup>−4</sup>). However, replication studies for rs13095226 and rs17778253 showed no statistical support for association (<em>P =</em>0.37 and 0.27, respectively) (Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB3" rid-ob="ob-DDS225TB3" co-legend-rid=""><span>3</span></a>).</p></div><div id="s2d" class="sec sec-last"><h3 id="s2dtitle">Secondary analyses</h3><p id="__p27" class="p p-first-last">Genome-wide association analyses were repeated for the subgroups of subjects affected by CNV only and GA only. Only SNPs at the <em>CFH</em> and <em>ARMS2</em>–<em>HTRA1</em> loci showed association at a genome-wide level of significance in both subgroups. Evidence of association was observed at the newly identified SNPs rs12153855/rs9391734 in <em>TNXB</em>–<em>FKBPL</em> in both subgroups (<em>P</em> = 5.0 × 10<sup>−4</sup>, OR = 0.70, 95% CI = 0.58–0.86 and <em>P</em> = 5.0 × 10<sup>−6</sup>, OR = 0.51, 95% CI = 0.38–0.68 in the CNV only and GA only subgroup analyses, respectively). The evidence of association at SNP rs2071277 in <em>NOTCH4</em> was stronger in the CNV-only subgroup than in the GA-only subgroup (<em>P</em> = 3.5 × 10<sup>−6</sup>, OR = 0.73, 95% CI = 0.64–0.84 and <em>P</em> = 0.07, OR = 0.82, 95% CI = 0.66–1.01, respectively), but there was no statistically significant difference between the two subgroup analyses (<em>P</em> = 0.81).</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p28" class="p p-first">We have carried out the first GWAS of AMD in the UK population. As expected, we found association with the well-established susceptibility loci <em>ARMS2</em>–<em>HTRA1</em> (<a href="#DDS225C19" rid="DDS225C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224691">19</a>,<a href="#DDS225C20" rid="DDS225C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224657">20</a>), <em>CFH</em> (<a href="#DDS225C7" rid="DDS225C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224693">7</a>–<a href="#DDS225C10" rid="DDS225C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224579">10</a>), <em>CFB</em>–<em>C2</em> (<a href="#DDS225C11" rid="DDS225C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224638">11</a>,<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224674">12</a>), <em>C3</em> (<a href="#DDS225C13" rid="DDS225C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224639">13</a>,<a href="#DDS225C14" rid="DDS225C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224698">14</a>), <em>CFI</em> (<a href="#DDS225C15" rid="DDS225C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224704">15</a>,<a href="#DDS225C16" rid="DDS225C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224649">16</a>) and <em>APOE</em> (<a href="#DDS225C28" rid="DDS225C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224642">28</a>). There was also evidence of association for more recently reported risk loci at <em>VEGFA</em> (<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224695">24</a>) and <em>LIPC</em> (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224590">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224700">24</a>), but not <em>CETP</em> or <em>TIMP3</em>–<em>SYN3</em> (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224591">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224634">24</a>). Our discovery experiment had limited power to detect these weaker associations because of the sample size and the use of younger unexamined controls (<a href="#DDS225C29" rid="DDS225C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224686">29</a>).</p><p id="__p29">At the <em>CFI</em> locus, the strongest association was with an SNP in the neighbouring <em>CCDC109B</em> gene as reported by others (<a href="#DDS225C23" rid="DDS225C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224701">23</a>) rather than with the originally reported SNP rs10033900 closer to <em>CFI</em> (<a href="#DDS225C16" rid="DDS225C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224627">16</a>). This is unsurprising as we have previously reported lack of association with rs10033900 in two independent UK case–control samples (<a href="#DDS225C15" rid="DDS225C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224708">15</a>) and it was a subset of cases from these studies that was used in our GWAS. The other reported association at this locus with rs2285714 in <em>PLA2G12A</em> (phospholipase A2, group XIIA) (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224583">22</a>) showed no evidence of association in our discovery analysis. Further studies are needed to clarify the origin of the association signal in this region.</p><p id="__p30">The association at the <em>CFH</em> locus on chromosome 1q32 was extensive, as found in other GWAS (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224584">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224586">24</a>). It spanned a megabase of DNA encompassing many genes. Conditional analysis of the region identified two independent association signals, one characterized by SNP rs10801555 (an almost perfect proxy for rs10611760, Y402H) and the other by rs1329428 (a proxy for rs1410996). These two independent association signals have been reported previously (<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224696">12</a>,<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224666">22</a>,<a href="#DDS225C25" rid="DDS225C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224622">25</a>).</p><p id="__p31">On chromosome 10q26, there was a strong association spanning the <em>ARMS2</em> and <em>HTRA1</em> genes as found by others (<a href="#DDS225C22" rid="DDS225C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224678">22</a>–<a href="#DDS225C24" rid="DDS225C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224614">24</a>), but high LD prevented further localization of the association signal.</p><p id="__p32">On chromosome 6p21.3, there was a strong association with rs541862, a perfect proxy for a nonsynonymous change R32Q (rs641153) in <em>CFB</em> which has a well-established association with AMD, the minor allele being protective (<a href="#DDS225C11" rid="DDS225C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224645">11</a>,<a href="#DDS225C12" rid="DDS225C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224635">12</a>). This was one of several strongly associated SNPs spanning <em>C2</em>, <em>CFB</em>, <em>RDBP</em> (RD RNA-binding protein) and <em>SKIV2L</em> in a region of high LD (Fig.&nbsp;<a href="/pmc/articles/PMC3428154/figure/DDS225F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDS225F1" rid-ob="ob-DDS225F1" co-legend-rid="lgnd_DDS225F1"><span>1</span></a>A) which makes the dissection of the statistical association difficult. Support for rs641153 as the causal variant comes from functional studies showing that the 32Q variant of <em>CFB</em> has reduced activity compared with 32R because of weaker binding to <em>C3b</em> (<a href="#DDS225C30" rid="DDS225C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224676">30</a>). Another associated SNP from this region, rs438999 (R151Q) in the <em>SKIV2L</em>, has also been suggested as a causal variant (<a href="#DDS225C31" rid="DDS225C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224592">31</a>), and a protective effect of rs429608 in the same gene has been reported (<a href="#DDS225C27" rid="DDS225C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224582">27</a>). Our data showed evidence of association with both rs438999 and rs429608, but conditional analysis did not provide support for an association at <em>SKIV2L</em> independent of the signal from rs541862 in <em>CFB</em> (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>).</p><p id="__p33">On chromosome 6p21.3, we also found a strong association reaching genome-wide significance for SNP rs2071277 in the <em>NOTCH4</em> gene some 250 kb from <em>CFB</em>. Conditional analysis confirmed that this was independent of the <em>C2</em>–<em>CFB</em> signal represented by rs541862 and also provided support for a second independent association with rs12153855 in the intervening <em>TNXB</em> gene. Only four haplotypes defined by these three SNPs were observed (Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB5/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB5" rid-ob="ob-DDS225TB5" co-legend-rid=""><span>5</span></a>), including the protective haplotype CTT characterized by a C allele at rs541862 which is the well-established protective association reported for <em>C2</em>–<em>CFB</em>. The other three haplotypes had a T allele at rs541862. The commonest of these was TTT, which we designated as the reference haplotype. The next most common haplotype TTC was associated with an increased risk for AMD, giving an OR of 1.14 (95% CI 1.05–1.25); the fourth haplotype TCC was associated with higher risk, the OR being 1.47 (95% CI 1.29–1.67).</p><p id="__p34">Both rs12153855 in <em>TNXB</em> and rs2071277 in <em>NOTCH4</em> are intronic and unlikely to be causal variants. SNP rs12153855 has a perfect proxy rs9391734 in the 5′ untranslated region of the neighbouring gene <em>FKBPL</em>, which might have functional significance. So we have identified novel associated variants but do not have evidence to pinpoint the causal variants responsible for these associations, as is the case with other susceptibility loci for AMD and many other complex diseases. All three genes could play a role in the pathogenesis of AMD as we review in what follows. However, further research will be needed to determine whether or not this is the case. Other disease associations have been reported for this region of chromosome 6 and notably an association with dementia and variants near <em>NOTCH4</em> and <em>AGER</em> (advanced glycosylation end product-specific receptor) (<a href="#DDS225C32" rid="DDS225C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_291170344">32</a>).</p><p id="__p35"><em>TNXB</em> encodes the extracellular matrix protein tenascin-X, which is present in the macular Bruch's membrane/choroid complex (<a href="#DDS225C33" rid="DDS225C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224606">33</a>). Various functions have been attributed to tenascin-X, which could have relevance to AMD. It has been shown to modulate collagen deposition (<a href="#DDS225C34" rid="DDS225C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224585">34</a>). There is also evidence that it modulates the mechanical properties of collagen networks (<a href="#DDS225C35" rid="DDS225C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224605">35</a>) and is involved in the maturation and/or maintenance of collagen and elastin networks (<a href="#DDS225C36" rid="DDS225C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224615">36</a>); both collagen and elastin are present in Bruch's membrane. In addition, forms of tenascin-X have been shown to bind to <em>VEGFA</em> and <em>VEGFB</em> (vascular endothelial growth factor B) and enhance their pro-angiogenic activity (<a href="#DDS225C37" rid="DDS225C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224644">37</a>).</p><p id="__p36"><em>FKBPL</em> encodes the <em>FKBP</em>-like protein which belongs to the family of <em>FK506</em>-binding proteins and has a role in intracellular signalling (<a href="#DDS225C38" rid="DDS225C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224568">38</a>). Of particular interest in the context of AMD, <em>FKBPL</em> also appears to have a separate role in controlling angiogenesis as a secreted protein that inhibits endothelial cell migration, tubule formation and angiogenesis through a <em>CD44</em>-mediated pathway (<a href="#DDS225C39" rid="DDS225C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224616">39</a>).</p><p id="__p37"><em>NOTCH4</em> encodes one of the four cell surface receptors of the Notch signalling pathway which has a key role in cell differentiation decisions across many cell types and at different stages of cell lineage progression (<a href="#DDS225C40" rid="DDS225C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224621">40</a>). Notch signalling is important in embryonic development, the regulation of tissue homeostasis and the maintenance of stem cells. Notch signalling is particularly important in vascular development, not only in embryogenesis but also in pathological angiogenesis, prompting research on Notch pathway inhibitors as potential agents to disrupt tumour-related angiogenesis for cancer therapy (<a href="#DDS225C41" rid="DDS225C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_520565122">41</a>).</p><p id="__p38">Notch signalling plays an important role in the developing retinal vasculature by regulating the specification of endothelial cells into stalk and tip cells (<a href="#DDS225C42" rid="DDS225C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224658">42</a>). In studies of mice, <em>NOTCH4</em> is strongly expressed in retinal arterial endothelial cells (<a href="#DDS225C43" rid="DDS225C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224613">43</a>). Moreover, Notch signalling has been shown to influence pathological angiogenesis in a study of laser-induced CNV in rats (<a href="#DDS225C44" rid="DDS225C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224576">44</a>). Following laser photocoagulation, inhibition of Notch signalling by intravitreal or subcutaneous administration of the gamma secretase inhibitor <em>DAPT</em> resulted in a substantial increase in the CNV volume compared with controls; conversely, stimulation of Notch signalling by intravitreal administration of Jagged1 peptide, a Notch ligand, substantially reduced the CNV volume (<a href="#DDS225C44" rid="DDS225C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224611">44</a>). In the same report, evidence was presented that Notch signalling contributes to angiogenic homeostasis by providing a counterbalance to proangiogenic pathways such as that mediated by <em>VEGF</em> (<a href="#DDS225C44" rid="DDS225C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224681">44</a>). This led the authors to propose the Notch pathway as a potential therapeutic target for the treatment of CNV, which is the severest form of AMD (<a href="#DDS225C44" rid="DDS225C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224694">44</a>). This might complement the use of anti-<em>VEGF</em> antibodies, which is the mainstay of current treatment. Our finding that SNPs at the <em>NOTCH4</em> locus are associated with AMD provides support for the involvement of Notch signalling in AMD and points specifically to a role for the <em>NOTCH4</em> receptor. However, if this were mediated through an alteration in angiogenic homeostasis, we might expect the association to be specifically with CNV. Most of the association signal does indeed come from CNV because this is the condition affecting most of the cases in our study, but the subgroup analysis suggests that there is also an association in the minority of cases with GA. This raises the possibility that other functions of the Notch pathway may be more important, such as its role in the differentiation of immune cells, including macrophages, which, as recent evidence suggests, may play a key part in the pathogenesis of AMD (<a href="#DDS225C45" rid="DDS225C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224578">45</a>).</p><p id="__p39" class="p p-last">In summary, our UK GWAS has shown the expected association between AMD and known risk loci at <em>ARMS2</em>–<em>HTRA1</em>, <em>CFH</em>, <em>C2</em>–<em>CFB</em>, <em>C3</em>, <em>CFI</em>–<em>CCDC109B</em> and <em>APOE</em> and with more recently reported risk loci at <em>LIPC</em> and near <em>VEGFA</em>. In addition, we have found novel associations with variants in <em>TNXB</em>–<em>FKBPL</em> and the neighbouring gene <em>NOTCH4</em> on chromosome 6p21.3 which are independent of the associated SNPs at the nearby <em>C2</em>–<em>CFB</em> locus. <em>TNXB</em>, <em>FKBPL</em> and <em>NOTCH4</em> are all plausible AMD susceptibility genes, but further research will be needed to identify the causal variants and determine whether any of these genes are involved in the pathogenesis of AMD.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Cases and controls</h3><p id="__p40" class="p p-first">Cases for the discovery experiment were selected from existing Cambridge and Edinburgh AMD case–control sample collections (<a href="#DDS225C14" rid="DDS225C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224660">14</a>). All cases selected for the GWAS had at least one eye affected by CNV and/or GA. For cases genotyped using the 300k platform, preference was given to cases with early onset. Control genotyping data were obtained from the British 1958 Birth Cohort (58BC) (<a href="#DDS225C46" rid="DDS225C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224572">46</a>,<a href="#DDS225C47" rid="DDS225C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224601">47</a>). For cases genotyped using the 300k platform, controls matched by geographic area were selected from the 58BC subjects used in the Wellcome Trust Case Control Consortium study (<a href="#DDS225C48" rid="DDS225C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224655">48</a>). For cases genotyped using the 550k platform, controls matched by geographic area were selected from the 58BC subjects genotyped as controls for the Type 1 Diabetes Genetics Consortium (T1DGC) study (<a href="#DDS225C46" rid="DDS225C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224593">46</a>). The replication studies used cases and controls from the Cambridge and Edinburgh sample collections and from case–control collections from London and Southampton in the UK. All the studies used in this research followed the tenets of the Declaration of Helsinki and had appropriate ethical approval. All participants gave informed written consent.</p><div id="s4a1" class="sec"><p></p><h4 id="s4a1title" class="inline">Cambridge AMD study subjects </h4><p id="__p41" class="p p-first-last">Cases and controls were recruited from ophthalmic clinics in London, the South East of England and the North West of England between 2002 and 2006 (<a href="#DDS225C14" rid="DDS225C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224589">14</a>). Almost all the controls were the spouses or partners of index cases and the remainder were friends of cases. All subjects described themselves as ‘white’ rather than ‘other’ on a recruitment questionnaire. Subjects were examined by an ophthalmologist, and health, lifestyle and smoking data were collected. All subjects had colour, stereoscopic fundus photography of the macular region, and the images were graded at the Reading Centre, Moorfields Eye Hospital, London, using the International Classification of Age-related Maculopathy and Macular Degeneration (<a href="#DDS225C49" rid="DDS225C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224702">49</a>).</p></div><div id="s4a2" class="sec"><p></p><h4 id="s4a2title" class="inline">Edinburgh AMD study subjects </h4><p id="__p42" class="p p-first-last">The majority of cases and controls were recruited from ophthalmic clinics in Edinburgh, Dundee and Inverness between 2004 and 2006 (<a href="#DDS225C14" rid="DDS225C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224587">14</a>). Controls were predominantly recruited from cataract clinics in the same centres but some spouses were also included. Cases and controls were all examined by an ophthalmologist, and visual acuities, health, lifestyle and smoking data were collected. Members of the 1921 Lothian Birth Cohort (<a href="#DDS225C50" rid="DDS225C50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224602">50</a>) were examined by an ophthalmologist and those found to have AMD and a similar number of unaffected controls from the Cohort were also included in the study. All cases and controls had colour, stereoscopic fundus photography of the macular region and images were graded by an ophthalmologist working on the study, using the International Classification of Age-related Maculopathy and Macular Degeneration (<a href="#DDS225C49" rid="DDS225C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224604">49</a>); for validation, 100 cases and controls were independently graded at the Moorfields Reading Centre (kappa statistic = 0.84).</p></div><div id="s4a3" class="sec"><p></p><h4 id="s4a3title" class="inline">London AMD study subjects </h4><p id="__p43" class="p p-first-last">Cases were recruited from Moorfields Eye Hospital, London. Controls were recruited from spouses, partners or friends of cases, or were from local residential homes for the elderly within 8 km of the hospital. All subjects were of Caucasian descent. Cases were examined by an ophthalmologist and had colour, stereoscopic fundus photography of the macular region with grading of the photographs at the Reading Centre, Moorfields Eye Hospital, according to the International Classification of Age-related Maculopathy and Macular Degeneration (<a href="#DDS225C49" rid="DDS225C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224603">49</a>). Cases were excluded if they had retino-choroidal inflammatory disease, diabetic retinopathy, branch retinal vein or artery occlusion or any other cause of visual loss other than amblyopia. The ophthalmic examination included Snellen acuity, slit-lamp examination and biomicroscopic fundoscopy. Auto-fluorescence images were taken of the macula, and fluorescein angiography was performed when CNV was suspected. For patients presenting with visual dysfunction in the second eye, retrospective data were gathered from hospital records concerning previous acuities. Moreover, any colour images or fluorescein images relating to previous visual loss were located from the hospital archive. All images were digitized. Controls had the same evaluation as cases and were excluded if they had drusen &gt;63 µm or more extensive evidence of AMD. Each participant was interviewed specifically for the study, and a family history, smoking history and other medical history were taken.</p></div><div id="s4a4" class="sec sec-last"><p></p><h4 id="s4a4title" class="inline">Southampton subjects </h4><p id="__p44" class="p p-first-last">Cases were ascertained through the Southampton Eye Unit or research clinics in Guernsey, UK. Controls were spouses/partners of AMD cases, patients attending eye clinics with unrelated eye disease or their spouses/partners. All participants were white and &gt;55 years of age. All cases and controls were evaluated by an experienced retinal specialist and had dilated eye examination. Phenotyping was based on the Age-Related Eye Disease Study (AREDS) classification system (<a href="#DDS225C51" rid="DDS225C51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224651">51</a>), in cases taking into account the results of fundus photography and fluorescein angiography and in controls based on clinical findings alone.</p></div></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping</h3><p id="__p45" class="p p-first">Genotyping of cases for the discovery experiment used genomic DNA extracted from peripheral blood leucocytes. Cases were typed either at the British Heart Foundation Glasgow Cardiovascular Research Centre, using the Illumina Infinium HumanHap300 BeadChip, or at the Wellcome Trust Clinical Research Facility, Edinburgh, using the Illumina Infinium HumanHap550v3 BeadChip, according to the manufacturer's protocols. Controls from the 58BC had been previously genotyped using the Illumina Infinium HumanHap550v1 BeadChip (58BC controls) (<a href="#DDS225C48" rid="DDS225C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224672">48</a>) and the Illumina Infinium HumanHap550v3 BeadChip (T1DGC controls) (<a href="#DDS225C46" rid="DDS225C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224661">46</a>).</p><p id="__p46" class="p p-last">The replication studies used genomic DNA extracted from the peripheral blood. Genotyping was carried out by the Medical Research Council Human Genetics Unit (MRC HGU) in Edinburgh, the Centre National de Génotypage (CNG) in France or the commercial company KBioscience. At MRC HGU and CNG, genotyping was carried out using TaqMan SNP Genotyping Assays (Applied Biosystems, Foster City, CA, USA) according to the manufacturer's protocols. KBioscience (<a href="http://www.kbioscience.co.uk">http://www.kbioscience.co.uk</a>) used their own proprietary KASP SNP genotyping system, a competitive allele-specific PCR incorporating a fluorescent resonance energy transfer quencher cassette (see <a href="http://www.kbioscience.co.uk/reagents/KASP.html">http://www.kbioscience.co.uk/reagents/KASP.html</a>).</p></div><div id="s4c" class="sec sec-last"><h3 id="s4ctitle">Statistical analysis</h3><div id="s4c1" class="sec sec-first"><p></p><h4 id="s4c1title" class="inline">Genotype and sample quality control </h4><p id="__p47" class="p p-first-last">Genotypes for the Illumina arrays were called using the genotyping module made available in the Illumina GenomeStudio 2009.2 software that incorporates the GenTrain 2.0 clustering algorithm. Given the different platforms used in the 300k analysis, only the overlap set of genotyped SNPs was used in the analysis. SNPs were excluded if they had a minor allele frequency of &lt;0.05, or a departure from Hardy–Weinberg equilibrium at <em>P</em> &lt; 10<sup>−5</sup> in controls, or a call rate &lt;0.975 either in cases or in controls in the 300k and 550k data sets separately. Genotype cluster plots were visually inspected for each putative association and results were discarded if the clusters were not clearly separated. Samples were removed if they presented inconsistencies between reported sex and sex determined by homozygosity on the X chromosome, or an excessive genetic relatedness based on IBS metrics calculated for each pair of individuals, or a call rate &lt;0.975. To control for possible population stratification, cases were stratified according to their reported geographic origin in four strata (London; North Midlands, East, Southeast and South of England; Northwest of England; Scotland), and only controls from matching geographic regions were used.</p></div><div id="s4c2" class="sec"><p></p><h4 id="s4c2title" class="inline">Imputation </h4><p id="__p48" class="p p-first-last">Autosomal imputation was performed using the R package snpStats from the BioConductor Project (<a href="http://www.bioconductor.org">http://www.bioconductor.org</a>) and CEU HapMap II samples as the reference data set. The imputation method has been described previously (<a href="#DDS225C52" rid="DDS225C52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224687">52</a>). Briefly, for the 300k and the 550k arrays separately, HapMap SNPs were divided into those that passed quality control steps in the study (<em>X</em>) and those that were not typed or failed quality control (<em>Y</em>). Linear regression models were used to predict each SNP in <em>Y</em> from nearby SNPs in <em>X. R</em><sup>2</sup> was used as a measure of accuracy of the imputation.</p></div><div id="s4c3" class="sec"><p></p><h4 id="s4c3title" class="inline">Discovery and replication association analyses </h4><p id="__p49" class="p p-first-last">Single SNP association analyses were carried out using the R package snpStats (<a href="http://www.bioconductor.org">http://www.bioconductor.org</a>). For both discovery and replication analyses, single SNP association was evaluated using the Mantel extension of the 1 degree-of-freedom (df) trend test (<a href="#DDS225C53" rid="DDS225C53" class=" bibr popnode">53</a>), which is a stratified version of the Cochrane–Armitage test, to take into account the geographic stratification of the samples. Results from the 300k and 550k arrays were combined by pooling the respective single SNP 1 df score statistics, and corresponding combined <em>P</em>-values were obtained from a 1 df <em>χ</em><sup>2</sup> distribution. Presence of heterogeneity between the 300k and 550k association results was evaluated through a 1 df <em>χ</em><sup>2</sup> test. As an alternative procedure to control for potential population stratification, we performed principal component analysis to categorize samples in terms of their large-scale genetic ancestry, using the GCTA software (<a href="#DDS225C54" rid="DDS225C54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224631">54</a>), and HapMap III samples as reference, and repeated the association analysis including the first two principal components as covariates in the logistic regression model, using PLINK v1.07 (<a href="#DDS225C55" rid="DDS225C55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224636">55</a>). The presence of residual population structure was assessed by evaluating the genomic inflation factors (<em>λ</em>) (<a href="#DDS225C56" rid="DDS225C56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224665">56</a>) and examining the quantile–quantile plots of the genome-wide <em>P</em>-values before and after exclusion of the established AMD loci. SNPs showing association at <em>P</em>&lt; 10<sup>−6</sup> in regions not previously reported were selected for follow-up in the replication samples. Regional association results were plotted using the LocusZoom software (<a href="http://csg.sph.umich.edu/locuszoom">http://csg.sph.umich.edu/locuszoom</a>).</p></div><div id="s4c4" class="sec"><p></p><h4 id="s4c4title" class="inline">Conditional and haplotype-based association analyses </h4><p id="__p50" class="p p-first">To detect secondary independent signals within associated regions, we used stepwise logistic regression models stratified by array and geographic origin after controlling for the strongest associated genotyped variants at each region added to the models in a forward fashion (R package snpStats, <a href="http://www.bioconductor.org">http://www.bioconductor.org</a>).</p><p id="__p51" class="p p-last">Haplotype-based analysis was carried out in PLINK v1.07 (<a href="#DDS225C55" rid="DDS225C55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_270224577">55</a>). Array information and geographic origin were added as covariates in the analysis. Only haplotypes with overall frequency &gt;1% were considered. The ORs and the corresponding CIs were estimated using the commonest haplotype as reference.</p></div><div id="s4c5" class="sec sec-last"><p></p><h4 id="s4c5title" class="inline">Signal at previously reported SNPs </h4><p id="__p52" class="p p-first-last">A literature search was conducted to identify studies reporting an association between a common SNP and AMD. Reports of associations with established and probable AMD loci were excluded (Table&nbsp;<a href="/pmc/articles/PMC3428154/table/DDS225TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDS225TB2" rid-ob="ob-DDS225TB2" co-legend-rid=""><span>2</span></a>), as were associations with deletions, rare variants, mitochondrial variants or SNP haplotypes where no single SNP showed a significant association. This led to the identification of 47 previously reported putative AMD susceptibility loci (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>). For the SNP reported to show the strongest evidence of association at each of these loci, we checked the corresponding signal in our discovery samples. SNPs that passed the Bonferroni correction were chosen for follow-up in our replication samples.</p></div></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p53" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds225/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p54" class="p p-first-last">This work was supported by the Medical Research Council, UK (grant <span>G0000067</span> to J.R.W.Y., A.T.M., D.G.C., A.C.B.; separate grants to A.R.W., A.C.B., A.F.W.); the Macular Disease Society (grants to J.R.W.Y., A.T.M., A.J.L.&nbsp;and S.E.); the Guide Dogs for the Blind Association (<span>OR2006-02d</span> to A.T.M., J.R.W.Y., A.R.W., D.G.C., C.B.); Fight for Sight (Mercer Fund to A.R.W.); the Wellcome Trust (grants <span>061860</span>;, <span>091388</span>;, <span>061858</span>; and <span>091157</span> to D.G.C.); the Juvenile Diabetes Research Foundation (grants <span>4-2000-948</span>;, <span>9-2005-25</span>; and <span>9-2011-253</span> to D.G.C.); the Giles' Family Memorial Funds (H.-T.L.); the Macula Vision Research Foundation (A.F.W.); the Chief Scientist Office, Scotland (<span>CZB/4/79</span> to B.D., C.H., A.M.A., A.F.W.); The Brian Mercer Charitable Trust (A.J.L.); The British Council for the Prevention of Blindness (A.J.L.); The TFC Frost Charitable Trust (A.J.L.); The Gift of Sight (A.J.L.); Centre National de Génotypage (T.L.) and Inserm (Centre d'Investigation Clinique des Quinze-Vingts, J.-A.S.); the Department of Health's NIHR Biomedical Research Centre for Ophthalmology at Moorfields Eye Hospital and UCL Institute of Ophthalmology (A.T.M., A.R.W., J.R.W.Y., C.B.); and the Manchester NIHR Biomedical Research Centre (P.N.B.). The views expressed in the publication are those of the authors and not necessarily those of the Department of Health. We acknowledge the use of genotype data from the British 1958 Birth Cohort DNA collection, funded by the Medical Research Council (<span>G0000934</span>) and the Wellcome Trust (<span>068545/Z/02</span>). H.C. and M.D. have been supported by Cancer Research UK (<span>C3 48/A3758</span>; and <span>A8896, C48/A6361</span>); the Medical Research Council UK (<span>G0000657-53203</span>); the Scottish Executive Chief Scientist's Office (<span>K/OPR/2/2/D333</span>;, <span>CZB/4/449</span>); and a Centre Grant from CORE as part of the Digestive Cancer Campaign. The Lothian Birth Cohort collection was supported by the Biotechnology and Biological Sciences Research Council, a Royal Society–Wolfson Research Merit Award to I.J.D. and two awards from the Chief Scientist Office of the Scottish Government's Health Directorates (<span>ETM/55</span>; and <span>CZB/4/505</span>).</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140193051499056"><a href="/pmc/articles/PMC3428154/bin/supp_21_18_4138__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140193054816816" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140193054816816title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p55">We gratefully acknowledge help with patient recruitment from members of the Genetic Factors in AMD Study Group (P. Black, Z. Butt, V. Chong, C. Edelsten, A. Fitt, D.W. Flanagan, A. Glenn, C. Jakeman, C. Jones, R.J. Lamb, V. Moffatt, C.M. Moorman, R.J. Pushpanathan, E. Redmond, T. Rimmer and D.A. Thurlby) and the Scottish Macula Society Study Group (M. Gavin, F. Imrie, N. Lois, R. Murray, A. Purdie, A. Pyott, S. Roxburgh, C. Styles, M. Virdi and W. Wykes). We thank all the clinical and research staff who contributed to the recruitment and the evaluation of the subjects for the case–control collections used in this study. For the Southampton replication cohort, we acknowledge help with recruitment (M. Nelson and nursing staff from the Wellcome Trust Clinical Research Facility at Southampton General Hospital) and DNA extraction (H. Griffiths). We thank Tunde Peto and colleagues at the Reading Centre, Moorfields Eye Hospital, London, for grading fundus photographs. We are grateful to Lee Murphy and Angela Fawkes (Wellcome Trust Clinical Research Facility, Edinburgh) and Wai Kwong Lee (British Heart Foundation Glasgow Cardiovascular Research Centre) for carrying out the genotyping of the cases for the discovery experiment. We thank Neil Walker and colleagues (Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research) and Panos Deloukas and colleagues (The Sanger Institute, Cambridge) for their help and support. We thank Marcus Fruttiger, Christiana Ruhrberg, John Greenwood, Jim Bainbridge and Glen Jeffery for helpful discussions. We are most grateful to all the subjects who kindly participated in this research.</p><p id="__p56"><em>Conflict of Interest statement</em>. J.R.W.Y. declares a financial interest in a patent that Cambridge Enterprise, on behalf of Cambridge University, has filed in North America arising from the discovery that a genetic variant in the complement <em>C3</em> gene influences the risk of developing AMD. V.C., A.T.M. and J.R.W.Y. declare a financial interest in a patent application that UCL Business has made on behalf of University College London arising from the discoveries reported here.</p></div></div><div id="idm140193089751424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140193089751424title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDS225C1">1. <span class="element-citation">Bunce C., Xing W., Wormald R. Causes of blind and partial sight certifications in England and Wales: April 2007-March 2008. <span><span class="ref-journal">Eye (Lond.) </span>2010;<span class="ref-vol">24</span>:1692–1699.</span> [<a href="/pubmed/20847749" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eye+(Lond.)&amp;title=Causes+of+blind+and+partial+sight+certifications+in+England+and+Wales:+April+2007-March+2008&amp;author=C.+Bunce&amp;author=W.+Xing&amp;author=R.+Wormald&amp;volume=24&amp;publication_year=2010&amp;pages=1692-1699&amp;pmid=20847749&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C2">2. <span class="element-citation">Klein R., Chou C.F., Klein B.E.K., Zhang X., Meuer S.M., Saaddine J.B. Prevalence of age-related macular degeneration in the US population. <span><span class="ref-journal">Arch. Ophthalmol. </span>2011;<span class="ref-vol">129</span>:75–80.</span> [<a href="/pubmed/21220632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Ophthalmol.&amp;title=Prevalence+of+age-related+macular+degeneration+in+the+US+population&amp;author=R.+Klein&amp;author=C.F.+Chou&amp;author=B.E.K.+Klein&amp;author=X.+Zhang&amp;author=S.M.+Meuer&amp;volume=129&amp;publication_year=2011&amp;pages=75-80&amp;pmid=21220632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C3">3. <span class="element-citation">Jager R.D., Mieler W.F., Miller J.W. Age-related macular degeneration. <span><span class="ref-journal">N. Engl. J. Med. </span>2008;<span class="ref-vol">358</span>:2606–2617.</span> [<a href="/pubmed/18550876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Age-related+macular+degeneration&amp;author=R.D.+Jager&amp;author=W.F.+Mieler&amp;author=J.W.+Miller&amp;volume=358&amp;publication_year=2008&amp;pages=2606-2617&amp;pmid=18550876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C4">4. <span class="element-citation">Chakravarthy U., Wong T.Y., Fletcher A., Piault E., Evans C., Zlateva G., Buggage R., Pleil A., Mitchell P. Clinical risk factors for age-related macular degeneration: a systematic review and meta-analysis. <span><span class="ref-journal">BMC Ophthalmol. </span>2010;<span class="ref-vol">10</span>:31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3009619/">PMC free article</a>]</span> [<a href="/pubmed/21144031" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Ophthalmol.&amp;title=Clinical+risk+factors+for+age-related+macular+degeneration:+a+systematic+review+and+meta-analysis&amp;author=U.+Chakravarthy&amp;author=T.Y.+Wong&amp;author=A.+Fletcher&amp;author=E.+Piault&amp;author=C.+Evans&amp;volume=10&amp;publication_year=2010&amp;pages=31&amp;pmid=21144031&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C5">5. <span class="element-citation">Deangelis M.M., Silveira A.C., Carr E.A., Kim I.K. Genetics of age-related macular degeneration: current concepts, future directions. <span><span class="ref-journal">Semin. Ophthalmol. </span>2011;<span class="ref-vol">26</span>:77–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4242505/">PMC free article</a>]</span> [<a href="/pubmed/21609220" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin.+Ophthalmol.&amp;title=Genetics+of+age-related+macular+degeneration:+current+concepts,+future+directions&amp;author=M.M.+Deangelis&amp;author=A.C.+Silveira&amp;author=E.A.+Carr&amp;author=I.K.+Kim&amp;volume=26&amp;publication_year=2011&amp;pages=77-93&amp;pmid=21609220&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C6">6. <span class="element-citation">Swaroop A., Chew E.Y., Rickman C.B., Abecasis G.R. Unraveling a multifactorial late-onset disease: from genetic susceptibility to disease mechanisms for age-related macular degeneration. <span><span class="ref-journal">Annu. Rev. Genomics Hum. Genet. </span>2009;<span class="ref-vol">10</span>:19–43.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3469316/">PMC free article</a>]</span> [<a href="/pubmed/19405847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genomics+Hum.+Genet.&amp;title=Unraveling+a+multifactorial+late-onset+disease:+from+genetic+susceptibility+to+disease+mechanisms+for+age-related+macular+degeneration&amp;author=A.+Swaroop&amp;author=E.Y.+Chew&amp;author=C.B.+Rickman&amp;author=G.R.+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=19-43&amp;pmid=19405847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C7">7. <span class="element-citation">Edwards A.O., Ritter R.,, III, Abel K.J., Manning A., Panhuysen C., Farrer L.A. Complement factor H polymorphism and age-related macular degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:421–424.</span> [<a href="/pubmed/15761121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+polymorphism+and+age-related+macular+degeneration&amp;author=A.O.+Edwards&amp;author=R.,+Ritter&amp;author=K.J.+Abel&amp;author=A.+Manning&amp;author=C.+Panhuysen&amp;volume=308&amp;publication_year=2005&amp;pages=421-424&amp;pmid=15761121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C8">8. <span class="element-citation">Hageman G.S., Anderson D.H., Johnson L.V., Hancox L.S., Taiber A.J., Hardisty L.I., Hageman J.L., Stockman H.A., Borchardt J.D., Gehrs K.M., et al.  A common haplotype in the complement regulatory gene factor H (HF1/CFH) predisposes individuals to age-related macular degeneration. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2005;<span class="ref-vol">102</span>:7227–7232.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1088171/">PMC free article</a>]</span> [<a href="/pubmed/15870199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=A+common+haplotype+in+the+complement+regulatory+gene+factor+H+(HF1/CFH)+predisposes+individuals+to+age-related+macular+degeneration&amp;author=G.S.+Hageman&amp;author=D.H.+Anderson&amp;author=L.V.+Johnson&amp;author=L.S.+Hancox&amp;author=A.J.+Taiber&amp;volume=102&amp;publication_year=2005&amp;pages=7227-7232&amp;pmid=15870199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C9">9. <span class="element-citation">Haines J.L., Hauser M.A., Schmidt S., Scott W.K., Olson L.M., Gallins P., Spencer K.L., Kwan S.Y., Noureddine M., Gilbert J.R., et al.  Complement factor H variant increases the risk of age-related macular degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:419–421.</span> [<a href="/pubmed/15761120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+variant+increases+the+risk+of+age-related+macular+degeneration&amp;author=J.L.+Haines&amp;author=M.A.+Hauser&amp;author=S.+Schmidt&amp;author=W.K.+Scott&amp;author=L.M.+Olson&amp;volume=308&amp;publication_year=2005&amp;pages=419-421&amp;pmid=15761120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C10">10. <span class="element-citation">Klein R.J., Zeiss C., Chew E.Y., Tsai J.Y., Sackler R.S., Haynes C., Henning A.K., SanGiovanni J.P., Mane S.M., Mayne S.T., et al.  Complement factor H polymorphism in age-related macular degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:385–389.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1512523/">PMC free article</a>]</span> [<a href="/pubmed/15761122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+polymorphism+in+age-related+macular+degeneration&amp;author=R.J.+Klein&amp;author=C.+Zeiss&amp;author=E.Y.+Chew&amp;author=J.Y.+Tsai&amp;author=R.S.+Sackler&amp;volume=308&amp;publication_year=2005&amp;pages=385-389&amp;pmid=15761122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C11">11. <span class="element-citation">Gold B., Merriam J.E., Zernant J., Hancox L.S., Taiber A.J., Gehrs K., Cramer K., Neel J., Bergeron J., Barile G.R., et al.  Variation in factor B (BF) and complement component 2 (C2) genes is associated with age-related macular degeneration. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:458–462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2921703/">PMC free article</a>]</span> [<a href="/pubmed/16518403" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Variation+in+factor+B+(BF)+and+complement+component+2+(C2)+genes+is+associated+with+age-related+macular+degeneration&amp;author=B.+Gold&amp;author=J.E.+Merriam&amp;author=J.+Zernant&amp;author=L.S.+Hancox&amp;author=A.J.+Taiber&amp;volume=38&amp;publication_year=2006&amp;pages=458-462&amp;pmid=16518403&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C12">12. <span class="element-citation">Maller J., George S., Purcell S., Fagerness J., Altshuler D., Daly M.J., Seddon J.M. Common variation in three genes, including a noncoding variant in CFH, strongly influences risk of age-related macular degeneration. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:1055–1059.</span> [<a href="/pubmed/16936732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variation+in+three+genes,+including+a+noncoding+variant+in+CFH,+strongly+influences+risk+of+age-related+macular+degeneration&amp;author=J.+Maller&amp;author=S.+George&amp;author=S.+Purcell&amp;author=J.+Fagerness&amp;author=D.+Altshuler&amp;volume=38&amp;publication_year=2006&amp;pages=1055-1059&amp;pmid=16936732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C13">13. <span class="element-citation">Maller J.B., Fagerness J.A., Reynolds R.C., Neale B.M., Daly M.J., Seddon J.M. Variation in complement factor 3 is associated with risk of age-related macular degeneration. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1200–1201.</span> [<a href="/pubmed/17767156" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Variation+in+complement+factor+3+is+associated+with+risk+of+age-related+macular+degeneration&amp;author=J.B.+Maller&amp;author=J.A.+Fagerness&amp;author=R.C.+Reynolds&amp;author=B.M.+Neale&amp;author=M.J.+Daly&amp;volume=39&amp;publication_year=2007&amp;pages=1200-1201&amp;pmid=17767156&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C14">14. <span class="element-citation">Yates J.R., Sepp T., Matharu B.K., Khan J.C., Thurlby D.A., Shahid H., Clayton D.G., Hayward C., Morgan J., Wright A.F., et al.  Complement C3 variant and the risk of age-related macular degeneration. <span><span class="ref-journal">N. Engl. J. Med. </span>2007;<span class="ref-vol">357</span>:553–561.</span> [<a href="/pubmed/17634448" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Complement+C3+variant+and+the+risk+of+age-related+macular+degeneration&amp;author=J.R.+Yates&amp;author=T.+Sepp&amp;author=B.K.+Matharu&amp;author=J.C.+Khan&amp;author=D.A.+Thurlby&amp;volume=357&amp;publication_year=2007&amp;pages=553-561&amp;pmid=17634448&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C15">15. <span class="element-citation">Cipriani V., Matharu B.K., Khan J.C., Shahid H., Hayward C., Wright A.F., Armbrecht A.M., Dhillon B., Harding S.P., Bishop P.N., et al.  No evidence of association between complement factor I genetic variant rs10033900 and age-related macular degeneration. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2012;<span class="ref-vol">20</span>:1–2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3234500/">PMC free article</a>]</span> [<a href="/pubmed/21989362" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=No+evidence+of+association+between+complement+factor+I+genetic+variant+rs10033900+and+age-related+macular+degeneration&amp;author=V.+Cipriani&amp;author=B.K.+Matharu&amp;author=J.C.+Khan&amp;author=H.+Shahid&amp;author=C.+Hayward&amp;volume=20&amp;publication_year=2012&amp;pages=1-2&amp;pmid=21989362&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C16">16. <span class="element-citation">Fagerness J.A., Maller J.B., Neale B.M., Reynolds R.C., Daly M.J., Seddon J.M. Variation near complement factor I is associated with risk of advanced AMD. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2009;<span class="ref-vol">17</span>:100–104.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2985963/">PMC free article</a>]</span> [<a href="/pubmed/18685559" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Variation+near+complement+factor+I+is+associated+with+risk+of+advanced+AMD&amp;author=J.A.+Fagerness&amp;author=J.B.+Maller&amp;author=B.M.+Neale&amp;author=R.C.+Reynolds&amp;author=M.J.+Daly&amp;volume=17&amp;publication_year=2009&amp;pages=100-104&amp;pmid=18685559&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C17">17. <span class="element-citation">Bradley D.T., Zipfel P.F., Hughes A.E. Complement in age-related macular degeneration: a focus on function. <span><span class="ref-journal">Eye (Lond.) </span>2011;<span class="ref-vol">25</span>:683–693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3178140/">PMC free article</a>]</span> [<a href="/pubmed/21394116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eye+(Lond.)&amp;title=Complement+in+age-related+macular+degeneration:+a+focus+on+function&amp;author=D.T.+Bradley&amp;author=P.F.+Zipfel&amp;author=A.E.+Hughes&amp;volume=25&amp;publication_year=2011&amp;pages=683-693&amp;pmid=21394116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C18">18. <span class="element-citation">Khandhadia S., Cipriani V., Yates J.R.W., Lotery A.J. Age-related macular degeneration and the complement system. <span><span class="ref-journal">Immunobiology. </span>2011;<span class="ref-vol">217</span>:127–146.</span> [<a href="/pubmed/21868123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunobiology&amp;title=Age-related+macular+degeneration+and+the+complement+system&amp;author=S.+Khandhadia&amp;author=V.+Cipriani&amp;author=J.R.W.+Yates&amp;author=A.J.+Lotery&amp;volume=217&amp;publication_year=2011&amp;pages=127-146&amp;pmid=21868123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C19">19. <span class="element-citation">Jakobsdottir J., Conley Y.P., Weeks D.E., Mah T.S., Ferrell R.E., Gorin M.B. Susceptibility genes for age-related maculopathy on chromosome 10q26. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2005;<span class="ref-vol">77</span>:389–407.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226205/">PMC free article</a>]</span> [<a href="/pubmed/16080115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Susceptibility+genes+for+age-related+maculopathy+on+chromosome+10q26&amp;author=J.+Jakobsdottir&amp;author=Y.P.+Conley&amp;author=D.E.+Weeks&amp;author=T.S.+Mah&amp;author=R.E.+Ferrell&amp;volume=77&amp;publication_year=2005&amp;pages=389-407&amp;pmid=16080115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C20">20. <span class="element-citation">Rivera A., Fisher S.A., Fritsche L.G., Keilhauer C.N., Lichtner P., Meitinger T., Weber B.H. Hypothetical LOC387715 is a second major susceptibility gene for age-related macular degeneration, contributing independently of complement factor H to disease risk. <span><span class="ref-journal">Hum. Mol. Genet. </span>2005;<span class="ref-vol">14</span>:3227–3236.</span> [<a href="/pubmed/16174643" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Hypothetical+LOC387715+is+a+second+major+susceptibility+gene+for+age-related+macular+degeneration,+contributing+independently+of+complement+factor+H+to+disease+risk&amp;author=A.+Rivera&amp;author=S.A.+Fisher&amp;author=L.G.+Fritsche&amp;author=C.N.+Keilhauer&amp;author=P.+Lichtner&amp;volume=14&amp;publication_year=2005&amp;pages=3227-3236&amp;pmid=16174643&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C21">21. <span class="element-citation">Friedrich U., Myers C.A., Fritsche L.G., Milenkovich A., Wolf A., Corbo J.C., Weber B.H. Risk- and non-risk-associated variants at the 10q26 AMD locus influence ARMS2 mRNA expression but exclude pathogenic effects due to protein deficiency. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:1387–1399.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3049360/">PMC free article</a>]</span> [<a href="/pubmed/21252205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Risk-+and+non-risk-associated+variants+at+the+10q26+AMD+locus+influence+ARMS2+mRNA+expression+but+exclude+pathogenic+effects+due+to+protein+deficiency&amp;author=U.+Friedrich&amp;author=C.A.+Myers&amp;author=L.G.+Fritsche&amp;author=A.+Milenkovich&amp;author=A.+Wolf&amp;volume=20&amp;publication_year=2011&amp;pages=1387-1399&amp;pmid=21252205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C22">22. <span class="element-citation">Chen W., Stambolian D., Edwards A.O., Branham K.E., Othman M., Jakobsdottir J., Tosakulwong N., Pericak-Vance M.A., Campochiaro P.A., Klein M.L., et al.  Genetic variants near TIMP3 and high-density lipoprotein-associated loci influence susceptibility to age-related macular degeneration. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2010;<span class="ref-vol">107</span>:7401–7406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2867722/">PMC free article</a>]</span> [<a href="/pubmed/20385819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Genetic+variants+near+TIMP3+and+high-density+lipoprotein-associated+loci+influence+susceptibility+to+age-related+macular+degeneration&amp;author=W.+Chen&amp;author=D.+Stambolian&amp;author=A.O.+Edwards&amp;author=K.E.+Branham&amp;author=M.+Othman&amp;volume=107&amp;publication_year=2010&amp;pages=7401-7406&amp;pmid=20385819&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C23">23. <span class="element-citation">Neale B.M., Fagerness J., Reynolds R., Sobrin L., Parker M., Raychaudhuri S., Tan P.L., Oh E.C., Merriam J.E., Souied E., et al.  Genome-wide association study of advanced age-related macular degeneration identifies a role of the hepatic lipase gene (LIPC) <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2010;<span class="ref-vol">107</span>:7395–7400.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2867697/">PMC free article</a>]</span> [<a href="/pubmed/20385826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Genome-wide+association+study+of+advanced+age-related+macular+degeneration+identifies+a+role+of+the+hepatic+lipase+gene+(LIPC)&amp;author=B.M.+Neale&amp;author=J.+Fagerness&amp;author=R.+Reynolds&amp;author=L.+Sobrin&amp;author=M.+Parker&amp;volume=107&amp;publication_year=2010&amp;pages=7395-7400&amp;pmid=20385826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C24">24. <span class="element-citation">Yu Y., Bhangale T.R., Fagerness J., Ripke S., Thorleifsson G., Tan P.L., Souied E.H., Richardson A.J., Merriam J.E., Buitendijk G.H., et al.  Common variants near FRK/COL10A1 and VEGFA are associated with advanced age-related macular degeneration. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:3699–3709.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3159552/">PMC free article</a>]</span> [<a href="/pubmed/21665990" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Common+variants+near+FRK/COL10A1+and+VEGFA+are+associated+with+advanced+age-related+macular+degeneration&amp;author=Y.+Yu&amp;author=T.R.+Bhangale&amp;author=J.+Fagerness&amp;author=S.+Ripke&amp;author=G.+Thorleifsson&amp;volume=20&amp;publication_year=2011&amp;pages=3699-3709&amp;pmid=21665990&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C25">25. <span class="element-citation">Sivakumaran T.A., Igo R.P., Jr., Kidd J.M., Itsara A., Kopplin L.J., Chen W., Hagstrom S.A., Peachey N.S., Francis P.J., Klein M.L., et al.  A 32 kb critical region excluding Y402H in CFH mediates risk for age-related macular degeneration. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e25598.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3192039/">PMC free article</a>]</span> [<a href="/pubmed/22022419" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=A+32+kb+critical+region+excluding+Y402H+in+CFH+mediates+risk+for+age-related+macular+degeneration&amp;author=T.A.+Sivakumaran&amp;author=R.P.+Igo&amp;author=J.M.+Kidd&amp;author=A.+Itsara&amp;author=L.J.+Kopplin&amp;volume=6&amp;publication_year=2011&amp;pages=e25598&amp;pmid=22022419&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C26">26. <span class="element-citation">Galan A., Ferlin A., Caretti L., Buson G., Sato G., Frigo A.C., Foresta C. Association of age-related macular degeneration with polymorphisms in vascular endothelial growth factor and its receptor. <span><span class="ref-journal">Ophthalmology. </span>2010;<span class="ref-vol">117</span>:1769–1774.</span> [<a href="/pubmed/20471686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Association+of+age-related+macular+degeneration+with+polymorphisms+in+vascular+endothelial+growth+factor+and+its+receptor&amp;author=A.+Galan&amp;author=A.+Ferlin&amp;author=L.+Caretti&amp;author=G.+Buson&amp;author=G.+Sato&amp;volume=117&amp;publication_year=2010&amp;pages=1769-1774&amp;pmid=20471686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C27">27. <span class="element-citation">Kopplin L.J., Igo R.P., Jr., Wang Y., Sivakumaran T.A., Hagstrom S.A., Peachey N.S., Francis P.J., Klein M.L., SanGiovanni J.P., Chew E.Y., et al.  Genome-wide association identifies SKIV2L and MYRIP as protective factors for age-related macular degeneration. <span><span class="ref-journal">Genes Immun. </span>2010;<span class="ref-vol">11</span>:609–621.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3375062/">PMC free article</a>]</span> [<a href="/pubmed/20861866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun.&amp;title=Genome-wide+association+identifies+SKIV2L+and+MYRIP+as+protective+factors+for+age-related+macular+degeneration&amp;author=L.J.+Kopplin&amp;author=R.P.+Igo&amp;author=Y.+Wang&amp;author=T.A.+Sivakumaran&amp;author=S.A.+Hagstrom&amp;volume=11&amp;publication_year=2010&amp;pages=609-621&amp;pmid=20861866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C28">28. <span class="element-citation">McKay G.J., Patterson C.C., Chakravarthy U., Dasari S., Klaver C.C., Vingerling J.R., Ho L., de Jong P.T., Fletcher A.E., Young I.S., et al.  Evidence of association of APOE with age-related macular degeneration - a pooled analysis of 15 studies. <span><span class="ref-journal">Hum. Mutat. </span>2011;<span class="ref-vol">32</span>:1407–1416.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3217135/">PMC free article</a>]</span> [<a href="/pubmed/21882290" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=Evidence+of+association+of+APOE+with+age-related+macular+degeneration+-+a+pooled+analysis+of+15+studies&amp;author=G.J.+McKay&amp;author=C.C.+Patterson&amp;author=U.+Chakravarthy&amp;author=S.+Dasari&amp;author=C.C.+Klaver&amp;volume=32&amp;publication_year=2011&amp;pages=1407-1416&amp;pmid=21882290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C29">29. <span class="element-citation">McCarthy M.I., Abecasis G.R., Cardon L.R., Goldstein D.B., Little J., Ioannidis J.P., Hirschhorn J.N. Genome-wide association studies for complex traits: consensus, uncertainty and challenges. <span><span class="ref-journal">Nat. Rev. Genet. </span>2008;<span class="ref-vol">9</span>:356–369.</span> [<a href="/pubmed/18398418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genome-wide+association+studies+for+complex+traits:+consensus,+uncertainty+and+challenges&amp;author=M.I.+McCarthy&amp;author=G.R.+Abecasis&amp;author=L.R.+Cardon&amp;author=D.B.+Goldstein&amp;author=J.+Little&amp;volume=9&amp;publication_year=2008&amp;pages=356-369&amp;pmid=18398418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C30">30. <span class="element-citation">Montes T., Tortajada A., Morgan B.P., Rodriguez de Cordoba S., Harris C.L. Functional basis of protection against age-related macular degeneration conferred by a common polymorphism in complement factor B. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2009;<span class="ref-vol">106</span>:4366–4371.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2657369/">PMC free article</a>]</span> [<a href="/pubmed/19255449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Functional+basis+of+protection+against+age-related+macular+degeneration+conferred+by+a+common+polymorphism+in+complement+factor+B&amp;author=T.+Montes&amp;author=A.+Tortajada&amp;author=B.P.+Morgan&amp;author=S.+Rodriguez+de+Cordoba&amp;author=C.L.+Harris&amp;volume=106&amp;publication_year=2009&amp;pages=4366-4371&amp;pmid=19255449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C31">31. <span class="element-citation">McKay G.J., Silvestri G., Patterson C.C., Hogg R.E., Chakravarthy U., Hughes A.E. Further assessment of the complement component 2 and factor B region associated with age-related macular degeneration. <span><span class="ref-journal">Invest. Ophthalmol. Vis. Sci. </span>2009;<span class="ref-vol">50</span>:533–539.</span> [<a href="/pubmed/18806297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci.&amp;title=Further+assessment+of+the+complement+component+2+and+factor+B+region+associated+with+age-related+macular+degeneration&amp;author=G.J.+McKay&amp;author=G.+Silvestri&amp;author=C.C.+Patterson&amp;author=R.E.+Hogg&amp;author=U.+Chakravarthy&amp;volume=50&amp;publication_year=2009&amp;pages=533-539&amp;pmid=18806297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C32">32. <span class="element-citation">Bennet A.M., Reynolds C.A., Eriksson U.K., Hong M.G., Blennow K., Gatz M., Alexeyenko A., Pedersen N.L., Prince J.A. Genetic association of sequence variants near AGER/NOTCH4 and dementia. <span><span class="ref-journal">J. Alzheimers Dis. </span>2011;<span class="ref-vol">24</span>:475–484.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3477600/">PMC free article</a>]</span> [<a href="/pubmed/21297263" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Alzheimers+Dis.&amp;title=Genetic+association+of+sequence+variants+near+AGER/NOTCH4+and+dementia&amp;author=A.M.+Bennet&amp;author=C.A.+Reynolds&amp;author=U.K.+Eriksson&amp;author=M.G.+Hong&amp;author=K.+Blennow&amp;volume=24&amp;publication_year=2011&amp;pages=475-484&amp;pmid=21297263&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C33">33. <span class="element-citation">Yuan X., Gu X., Crabb J.S., Yue X., Shadrach K., Hollyfield J.G., Crabb J.W. Quantitative proteomics: comparison of the macular Bruch membrane/choroid complex from age-related macular degeneration and normal eyes. <span><span class="ref-journal">Mol. Cell. Proteomics. </span>2010;<span class="ref-vol">9</span>:1031–1046.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2877969/">PMC free article</a>]</span> [<a href="/pubmed/20177130" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteomics&amp;title=Quantitative+proteomics:+comparison+of+the+macular+Bruch+membrane/choroid+complex+from+age-related+macular+degeneration+and+normal+eyes&amp;author=X.+Yuan&amp;author=X.+Gu&amp;author=J.S.+Crabb&amp;author=X.+Yue&amp;author=K.+Shadrach&amp;volume=9&amp;publication_year=2010&amp;pages=1031-1046&amp;pmid=20177130&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C34">34. <span class="element-citation">Mao J.R., Taylor G., Dean W.B., Wagner D.R., Afzal V., Lotz J.C., Rubin E.M., Bristow J. Tenascin-X deficiency mimics Ehlers-Danlos syndrome in mice through alteration of collagen deposition. <span><span class="ref-journal">Nat. Genet. </span>2002;<span class="ref-vol">30</span>:421–425.</span> [<a href="/pubmed/11925569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Tenascin-X+deficiency+mimics+Ehlers-Danlos+syndrome+in+mice+through+alteration+of+collagen+deposition&amp;author=J.R.+Mao&amp;author=G.+Taylor&amp;author=W.B.+Dean&amp;author=D.R.+Wagner&amp;author=V.+Afzal&amp;volume=30&amp;publication_year=2002&amp;pages=421-425&amp;pmid=11925569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C35">35. <span class="element-citation">Margaron Y., Bostan L., Exposito J.Y., Malbouyres M., Trunfio-Sfarghiu A.M., Berthier Y., Lethias C. Tenascin-X increases the stiffness of collagen gels without affecting fibrillogenesis. <span><span class="ref-journal">Biophys. Chem. </span>2010;<span class="ref-vol">147</span>:87–91.</span> [<a href="/pubmed/20089348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.+Chem.&amp;title=Tenascin-X+increases+the+stiffness+of+collagen+gels+without+affecting+fibrillogenesis&amp;author=Y.+Margaron&amp;author=L.+Bostan&amp;author=J.Y.+Exposito&amp;author=M.+Malbouyres&amp;author=A.M.+Trunfio-Sfarghiu&amp;volume=147&amp;publication_year=2010&amp;pages=87-91&amp;pmid=20089348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C36">36. <span class="element-citation">Egging D.F., van Vlijmen I., Starcher B., Gijsen Y., Zweers M.C., Blankevoort L., Bristow J., Schalkwijk J. Dermal connective tissue development in mice: an essential role for tenascin-X. <span><span class="ref-journal">Cell Tissue Res. </span>2006;<span class="ref-vol">323</span>:465–474.</span> [<a href="/pubmed/16331473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Tissue+Res.&amp;title=Dermal+connective+tissue+development+in+mice:+an+essential+role+for+tenascin-X&amp;author=D.F.+Egging&amp;author=I.+van+Vlijmen&amp;author=B.+Starcher&amp;author=Y.+Gijsen&amp;author=M.C.+Zweers&amp;volume=323&amp;publication_year=2006&amp;pages=465-474&amp;pmid=16331473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C37">37. <span class="element-citation">Ikuta T., Ariga H., Matsumoto K.I. Effect of tenascin-X together with vascular endothelial growth factor A on cell proliferation in cultured embryonic hearts. <span><span class="ref-journal">Biol. Pharm. Bull. </span>2001;<span class="ref-vol">24</span>:1320–1323.</span> [<a href="/pubmed/11725972" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Pharm.+Bull.&amp;title=Effect+of+tenascin-X+together+with+vascular+endothelial+growth+factor+A+on+cell+proliferation+in+cultured+embryonic+hearts&amp;author=T.+Ikuta&amp;author=H.+Ariga&amp;author=K.I.+Matsumoto&amp;volume=24&amp;publication_year=2001&amp;pages=1320-1323&amp;pmid=11725972&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C38">38. <span class="element-citation">McKeen H.D., Brennan D.J., Hegarty S., Lanigan F., Jirstrom K., Byrne C., Yakkundi A., McCarthy H.O., Gallagher W.M., Robson T. The emerging role of FK506-binding proteins as cancer biomarkers: a focus on FKBPL. <span><span class="ref-journal">Biochem. Soc. Trans. </span>2011;<span class="ref-vol">39</span>:663–668.</span> [<a href="/pubmed/21428958" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Soc.+Trans.&amp;title=The+emerging+role+of+FK506-binding+proteins+as+cancer+biomarkers:+a+focus+on+FKBPL&amp;author=H.D.+McKeen&amp;author=D.J.+Brennan&amp;author=S.+Hegarty&amp;author=F.+Lanigan&amp;author=K.+Jirstrom&amp;volume=39&amp;publication_year=2011&amp;pages=663-668&amp;pmid=21428958&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C39">39. <span class="element-citation">Valentine A., O'Rourke M., Yakkundi A., Worthington J., Hookham M., Bicknell R., McCarthy H.O., McClelland K., McCallum L., Dyer H., et al.  FKBPL and peptide derivatives: novel biological agents that inhibit angiogenesis by a CD44-dependent mechanism. <span><span class="ref-journal">Clin. Cancer Res. </span>2011;<span class="ref-vol">17</span>:1044–1056.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059488/">PMC free article</a>]</span> [<a href="/pubmed/21364036" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=FKBPL+and+peptide+derivatives:+novel+biological+agents+that+inhibit+angiogenesis+by+a+CD44-dependent+mechanism&amp;author=A.+Valentine&amp;author=M.+O'Rourke&amp;author=A.+Yakkundi&amp;author=J.+Worthington&amp;author=M.+Hookham&amp;volume=17&amp;publication_year=2011&amp;pages=1044-1056&amp;pmid=21364036&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C40">40. <span class="element-citation">Andersson E.R., Sandberg R., Lendahl U. Notch signaling: simplicity in design, versatility in function. <span><span class="ref-journal">Development. </span>2011;<span class="ref-vol">138</span>:3593–3612.</span> [<a href="/pubmed/21828089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Notch+signaling:+simplicity+in+design,+versatility+in+function&amp;author=E.R.+Andersson&amp;author=R.+Sandberg&amp;author=U.+Lendahl&amp;volume=138&amp;publication_year=2011&amp;pages=3593-3612&amp;pmid=21828089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C41">41. <span class="element-citation">Kerbel R.S. Tumor angiogenesis. <span><span class="ref-journal">N. Engl J. Med. </span>2008;<span class="ref-vol">358</span>:2039–2049.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4542009/">PMC free article</a>]</span> [<a href="/pubmed/18463380" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl+J.+Med.&amp;title=Tumor+angiogenesis&amp;author=R.S.+Kerbel&amp;volume=358&amp;publication_year=2008&amp;pages=2039-2049&amp;pmid=18463380&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C42">42. <span class="element-citation">Hellstrom M., Phng L.K., Hofmann J.J., Wallgard E., Coultas L., Lindblom P., Alva J., Nilsson A.K., Karlsson L., Gaiano N., et al.  Dll4 signalling through Notch1 regulates formation of tip cells during angiogenesis. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:776–780.</span> [<a href="/pubmed/17259973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Dll4+signalling+through+Notch1+regulates+formation+of+tip+cells+during+angiogenesis&amp;author=M.+Hellstrom&amp;author=L.K.+Phng&amp;author=J.J.+Hofmann&amp;author=E.+Wallgard&amp;author=L.+Coultas&amp;volume=445&amp;publication_year=2007&amp;pages=776-780&amp;pmid=17259973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C43">43. <span class="element-citation">Claxton S., Fruttiger M. Periodic Delta-like 4 expression in developing retinal arteries. <span><span class="ref-journal">Gene Expr. Patterns. </span>2004;<span class="ref-vol">5</span>:123–127.</span> [<a href="/pubmed/15533827" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene+Expr.+Patterns&amp;title=Periodic+Delta-like+4+expression+in+developing+retinal+arteries&amp;author=S.+Claxton&amp;author=M.+Fruttiger&amp;volume=5&amp;publication_year=2004&amp;pages=123-127&amp;pmid=15533827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C44">44. <span class="element-citation">Ahmad I., Balasubramanian S., Del Debbio C.B., Parameswaran S., Katz A.R., Toris C., Fariss R.N. Regulation of ocular angiogenesis by Notch signaling: implications in neovascular age-related macular degeneration. <span><span class="ref-journal">Invest. Ophthalmol. Vis. Sci. </span>2011;<span class="ref-vol">52</span>:2868–2878.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3109005/">PMC free article</a>]</span> [<a href="/pubmed/21228388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci.&amp;title=Regulation+of+ocular+angiogenesis+by+Notch+signaling:+implications+in+neovascular+age-related+macular+degeneration&amp;author=I.+Ahmad&amp;author=S.+Balasubramanian&amp;author=C.B.+Del+Debbio&amp;author=S.+Parameswaran&amp;author=A.R.+Katz&amp;volume=52&amp;publication_year=2011&amp;pages=2868-2878&amp;pmid=21228388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C45">45. <span class="element-citation">Cherepanoff S., McMenamin P., Gillies M.C., Kettle E., Sarks S.H. Bruch's membrane and choroidal macrophages in early and advanced age-related macular degeneration. <span><span class="ref-journal">Br. J. Ophthalmol. </span>2010;<span class="ref-vol">94</span>:918–925.</span> [<a href="/pubmed/19965817" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Ophthalmol.&amp;title=Bruch's+membrane+and+choroidal+macrophages+in+early+and+advanced+age-related+macular+degeneration&amp;author=S.+Cherepanoff&amp;author=P.+McMenamin&amp;author=M.C.+Gillies&amp;author=E.+Kettle&amp;author=S.H.+Sarks&amp;volume=94&amp;publication_year=2010&amp;pages=918-925&amp;pmid=19965817&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C46">46. <span class="element-citation">Barrett J.C., Clayton D.G., Concannon P., Akolkar B., Cooper J.D., Erlich H.A., Julier C., Morahan G., Nerup J., Nierras C., et al.  Genome-wide association study and meta-analysis find that over 40 loci affect risk of type 1 diabetes. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:703–707.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2889014/">PMC free article</a>]</span> [<a href="/pubmed/19430480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+and+meta-analysis+find+that+over+40+loci+affect+risk+of+type+1+diabetes&amp;author=J.C.+Barrett&amp;author=D.G.+Clayton&amp;author=P.+Concannon&amp;author=B.+Akolkar&amp;author=J.D.+Cooper&amp;volume=41&amp;publication_year=2009&amp;pages=703-707&amp;pmid=19430480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C47">47. <span class="element-citation">Power C., Elliott J. Cohort profile: 1958 British Birth Cohort (National Child Development Study) <span><span class="ref-journal">Int. J. Epidemiol. </span>2006;<span class="ref-vol">35</span>:34–41.</span> [<a href="/pubmed/16155052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol.&amp;title=Cohort+profile:+1958+British+Birth+Cohort+(National+Child+Development+Study)&amp;author=C.+Power&amp;author=J.+Elliott&amp;volume=35&amp;publication_year=2006&amp;pages=34-41&amp;pmid=16155052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C48">48. <span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C49">49. <span class="element-citation">Bird A.C., Bressler N.M., Bressler S.B., Chisholm I.H., Coscas G., Davis M.D., de Jong P.T., Klaver C.C.W., Klein B.E.K., Klein R., et al.  An international classification and grading system for age-related maculopathy and age-related macular degeneration. The International ARM Epidemiological Study Group. <span><span class="ref-journal">Surv. Ophthalmol. </span>1995;<span class="ref-vol">39</span>:367–374.</span> [<a href="/pubmed/7604360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Surv.+Ophthalmol.&amp;title=An+international+classification+and+grading+system+for+age-related+maculopathy+and+age-related+macular+degeneration.+The+International+ARM+Epidemiological+Study+Group&amp;author=A.C.+Bird&amp;author=N.M.+Bressler&amp;author=S.B.+Bressler&amp;author=I.H.+Chisholm&amp;author=G.+Coscas&amp;volume=39&amp;publication_year=1995&amp;pages=367-374&amp;pmid=7604360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C50">50. <span class="element-citation">Deary I.J., Whiteman M.C., Starr J.M., Whalley L.J., Fox H.C. The impact of childhood intelligence on later life: following up the Scottish mental surveys of 1932 and 1947. <span><span class="ref-journal">J. Pers. Soc. Psychol. </span>2004;<span class="ref-vol">86</span>:130–147.</span> [<a href="/pubmed/14717632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Pers.+Soc.+Psychol.&amp;title=The+impact+of+childhood+intelligence+on+later+life:+following+up+the+Scottish+mental+surveys+of+1932+and+1947&amp;author=I.J.+Deary&amp;author=M.C.+Whiteman&amp;author=J.M.+Starr&amp;author=L.J.+Whalley&amp;author=H.C.+Fox&amp;volume=86&amp;publication_year=2004&amp;pages=130-147&amp;pmid=14717632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C51">51. <span class="element-citation">The Age-Related Eye Disease Study Research Group. The age-related eye disease study system for classifying age-related macular degeneration from stereoscopic color fundus photographs: the Age-Related Eye Disease Study report number 6. <span><span class="ref-journal">Am. J. Ophthalmol. </span>2001;<span class="ref-vol">132</span>:668–681.</span> [<a href="/pubmed/11704028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Ophthalmol.&amp;title=The+age-related+eye+disease+study+system+for+classifying+age-related+macular+degeneration+from+stereoscopic+color+fundus+photographs:+the+Age-Related+Eye+Disease+Study+report+number+6&amp;volume=132&amp;publication_year=2001&amp;pages=668-681&amp;pmid=11704028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C52">52. <span class="element-citation">Wallace C., Smyth D.J., Maisuria-Armer M., Walker N.M., Todd J.A., Clayton D.G. The imprinted DLK1-MEG3 gene region on chromosome 14q32.2 alters susceptibility to type 1 diabetes. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:68–71.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820243/">PMC free article</a>]</span> [<a href="/pubmed/19966805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=The+imprinted+DLK1-MEG3+gene+region+on+chromosome+14q32.2+alters+susceptibility+to+type+1+diabetes&amp;author=C.+Wallace&amp;author=D.J.+Smyth&amp;author=M.+Maisuria-Armer&amp;author=N.M.+Walker&amp;author=J.A.+Todd&amp;volume=42&amp;publication_year=2010&amp;pages=68-71&amp;pmid=19966805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C53">53. <span class="element-citation">Mantel N. Chi-square tests with one degree of freedom; extensions of the Mantel-Haenszel procedure. <span><span class="ref-journal">J. Am. Stat. Assoc. </span>1963;<span class="ref-vol">58</span>:690–700.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Stat.+Assoc.&amp;title=Chi-square+tests+with+one+degree+of+freedom;+extensions+of+the+Mantel-Haenszel+procedure&amp;author=N.+Mantel&amp;volume=58&amp;publication_year=1963&amp;pages=690-700&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C54">54. <span class="element-citation">Yang J., Lee S.H., Goddard M.E., Visscher P.M. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C55">55. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDS225C56">56. <span class="element-citation">Devlin B., Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3428154&amp;issue-id=213402&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140193045995072title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140193054816816title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140193089751424title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3428154/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3428154/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3428154/pdf/dds225.pdf">PDF (232K)</a></li> | <li><a href="#" data-citationid="PMC3428154" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3428154%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3428154%2F&amp;text=Genome-wide%20association%20study%20of%20age-related%20macular%20degeneration%20identifies%20associated%20variants%20in%20the%20TNXB%E2%80%93FKBPL%E2%80%93NOTCH4%20region%20of%20chromosome%206p21.3" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3428154%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22694956" data-aiid="3428154" data-aid="3428154" data-iid="213402" data-domainid="740" data-domain="hmg" data-accid="PMC3428154" data-md5="4362c2879ec15c6022b2d2df3fd43e28"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:06:39-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C2FDADD541A010000000004C30330&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3428154%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3428154/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>