<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3247076/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genomewide meta-analysis identifies novel multiple sclerosis susceptibility loci</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3247076/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Annals of neurology"><meta name="citation_title" content="Genomewide meta-analysis identifies novel multiple sclerosis susceptibility loci"><meta name="citation_authors" content="Nikolaos A. Patsopoulos, the BSP MS Genetics working group and the steering committees of studies evaluating IFNb-1b and a CCR1-antagonist, ANZgene Consortium, GeneMSA, International Multiple Sclerosis Genetics Consortium, Paul I.W. de Bakker"><meta name="citation_date" content="December 2011"><meta name="citation_issue" content="6"><meta name="citation_volume" content="70"><meta name="citation_firstpage" content="897"><meta name="citation_doi" content="10.1002/ana.22609"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3247076/?report=abstract"><meta name="citation_pmid" content="22190364"><meta name="DC.Title" content="Genomewide meta-analysis identifies novel multiple sclerosis susceptibility loci"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Nikolaos A. Patsopoulos"><meta name="DC.Contributor" content="the BSP MS Genetics working group and the steering committees of studies evaluating IFNb-1b and a CCR1-antagonist, ANZgene Consortium, GeneMSA, International Multiple Sclerosis Genetics Consortium"><meta name="DC.Contributor" content="Paul I.W. de Bakker"><meta name="DC.Date" content="2011 Dec"><meta name="DC.Identifier" content="10.1002/ana.22609"><meta name="DC.Language" content="en"><meta property="og:title" content="Genomewide meta-analysis identifies novel multiple sclerosis susceptibility loci"><meta property="og:type" content="article"><meta property="og:description" content="To perform a one-stage meta-analysis of genome-wide association studies (GWAS) of multiple sclerosis (MS) susceptibility and explore functional consequences of new susceptibility loci.We synthesized 7 MS GWAS. Each dataset was imputed using HapMap phase ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3247076/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3247076/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3247076/pdf/nihms318621.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89C5E4DD53B43100000000016E010D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3247076%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3247076</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Ann Neurol</a></span>. Author manuscript; available in PMC 2012 Dec 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22190364" target="pmc_ext" ref="reftype=publisher&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Ann Neurol. 2011 Dec; 70(6): 897–912. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fana.22609" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/ana.22609</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3247076</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS318621</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22190364">22190364</a></div></div></div></div><h1 class="content-title">Genomewide meta-analysis identifies novel multiple sclerosis susceptibility loci</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Patsopoulos%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22190364" class="affpopup" co-rid="_co_idm140337209879968" co-class="co-affbox">Nikolaos A. Patsopoulos</a>, MD, PhD,<sup>1,</sup><sup>2,</sup><sup>3</sup> the BSP MS Genetics working group and the steering committees of studies evaluating IFNb-1b and a CCR1-antagonist, ANZgene Consortium, GeneMSA, International Multiple Sclerosis Genetics Consortium, and  <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22190364" class="affpopup" co-rid="_co_idm140337129586064" co-class="co-affbox">Paul I.W. de Bakker</a>, PhD<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140337209879968"><h3 class="no_margin">Nikolaos A. Patsopoulos</h3><p><sup>1</sup>Program in Translational NeuroPsychiatric Genomics, Institute for the Neurosciences, Department of Neurology, Brigham &amp; Women’s Hospital, Boston, MA 02115, USA</p><p><sup>2</sup>Division of Genetics, Department of Medicine, Brigham &amp; Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, MA 02139, USA</p><div>Find articles by <a href="/pubmed/?term=Patsopoulos%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22190364">Nikolaos A. Patsopoulos</a></div></div><div id="_co_idm140337129586064"><h3 class="no_margin">Paul I.W. de Bakker</h3><p><sup>2</sup>Division of Genetics, Department of Medicine, Brigham &amp; Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, MA 02139, USA</p><p><sup>4</sup>Department of Medical Genetics, Division of Biomedical Genetics, University Medical Center, Utrecht, The Netherlands</p><p><sup>5</sup>Julius Center for Health Sciences and Primary Care, University Medical Center, Utrecht, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22190364">Paul I.W. de Bakker</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140337118434528_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140337118434528_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140337118434528_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Program in Translational NeuroPsychiatric Genomics, Institute for the Neurosciences, Department of Neurology, Brigham &amp; Women’s Hospital, Boston, MA 02115, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Division of Genetics, Department of Medicine, Brigham &amp; Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, MA 02139, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Medical Genetics, Division of Biomedical Genetics, University Medical Center, Utrecht, The Netherlands</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Julius Center for Health Sciences and Primary Care, University Medical Center, Utrecht, The Netherlands</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140337118434528_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22190364" target="pmc_ext" ref="reftype=publisher&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Ann Neurol</a></div><div><strong>This article has been corrected. </strong>See the correction in volume 73 on&nbsp;page&nbsp;561.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3247076/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table S1-S5 &amp; Fig S1-S4.</strong></div><div><a href="/pmc/articles/PMC3247076/bin/NIHMS318621-supplement-Supp_Table_S1-S5___Fig_S1-S4.doc" data-ga-action="click_feat_suppl">NIHMS318621-supplement-Supp_Table_S1-S5___Fig_S1-S4.doc</a><span style="color:gray"> (13M)</span></div><div class="small guid">GUID:&nbsp;A310E686-5611-4AFC-AB8B-906C048CE8DC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table S6.</strong></div><div><a href="/pmc/articles/PMC3247076/bin/NIHMS318621-supplement-Supp_Table_S6.xls" data-ga-action="click_feat_suppl">NIHMS318621-supplement-Supp_Table_S6.xls</a><span style="color:gray"> (44K)</span></div><div class="small guid">GUID:&nbsp;278AF8EA-E8A1-4214-BABF-3852D4345B48</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table S7.</strong></div><div><a href="/pmc/articles/PMC3247076/bin/NIHMS318621-supplement-Supp_Table_S7.xls" data-ga-action="click_feat_suppl">NIHMS318621-supplement-Supp_Table_S7.xls</a><span style="color:gray"> (315K)</span></div><div class="small guid">GUID:&nbsp;303AC966-9019-411D-A726-966BD9D69533</div></div></dd></dl></div><div id="idm140337117463856" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140337117463856title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Objective</h3><p id="P1" class="p p-first-last">To perform a one-stage meta-analysis of genome-wide association studies (GWAS) of multiple sclerosis (MS) susceptibility and explore functional consequences of new susceptibility loci.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">We synthesized 7 MS GWAS. Each dataset was imputed using HapMap phase II and a per-SNP meta-analysis was performed across the 7 datasets. We explored RNA expression data using a quantitative trait analysis in peripheral blood mononuclear cells (PBMCs) of 228 subjects with demyelinating disease.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">We meta-analyzed 2,529,394 unique SNPs in 5,545 cases and 12,153 controls. We identified three novel susceptibility alleles: rs170934<sup>T</sup> at 3p24.1 (OR=1.17, P = 1.6 × 10<sup>−8</sup>) near <em>EOMES</em>, rs2150702<sup>G</sup> in the second intron of <em>MLANA</em> on chromosome 9p24.1 (OR = 1.16, P = 3.3 × 10<sup>−8</sup>), and rs6718520<sup>A</sup> in an intergenic region on chromosome 2p21, with <em>THADA</em> as the nearest flanking gene (OR = 1.17, P = 3.4 × 10<sup>−8</sup>). The three new loci do not have a strong “<em>cis</em>” effect on RNA expression in PBMCs. Ten other susceptibility loci had a suggestive P&lt;1×10<sup>−6</sup>, some of which have evidence of association in other inflammatory diseases, i.e. <em>IL12B</em>, <em>TAGAP</em>, <em>PLEK</em>, and <em>ZMIZ1.</em></p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Interpretation</h3><p id="P4" class="p p-first-last">We have performed a meta-analysis of GWAS in MS that more than doubles the size of previous gene discovery efforts and highlights three novel MS susceptibility loci. These and additional loci with suggestive evidence of association are excellent candidates for further investigations to refine and validate their role in the genetic architecture of MS.</p></div></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">INTRODUCTION</h2><p id="P5" class="p p-first">Multiple sclerosis (MS) is thought to emerge when genetically susceptible individuals encounter environmental triggers that initiate an inflammatory reaction against self-antigens in the central nervous system (CNS); these events result in recurring episodes of inflammatory demyelination and, in many cases, a progressive neurodegenerative process.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890441">1</a></sup> The genetic architecture underlying susceptibility to MS is complex, and recent efforts have revealed over a dozen susceptibility loci of modest effect<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890479">2</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890433">5</a></sup> in addition to the major histocompatibility complex (MHC) that contains multiple independent susceptibility alleles in class I and class II loci. These discoveries were made possible by the emergence of genome-wide genotyping technologies and collaborative meta-analysis efforts to maximize statistical power.</p><p id="P6" class="p p-last">Here, we perform a one-stage meta-analysis, of most of the genome-wide single nucleotide polymorphism (SNP) data generated in the field of MS, to conduct the largest gene discovery effort to date for this disease.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">MATERIALS &amp; METHODS</h2><p id="P7" class="p p-first">The present study comprises seven data sets of non-overlapping case and control subjects of European descent. Five strata (IMSGC-US, IMSGC-UK, GeneMSA-US, GeneMSA-NL, and GeneMSA-CH) were taken from a previously published meta-analysis.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890438">3</a></sup> Details on these data sets can be found elsewhere.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890448">2</a>–<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890446">3</a></sup> The sixth stratum (BWH/TT) is based on data from our previous study<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890468">3</a></sup> enriched with additional 1453 MS cases and 2176 controls, with all samples genotyped on the Affymetrix GeneChip 6.0 platform. Finally, we have added another stratum (ANZ) from a recently described genome-wide study containing 1618 cases and 1988 controls.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890477">5</a></sup> Further information on these two strata and quality control applied can be found in the <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary material</a>. <a href="/pmc/articles/PMC3247076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> summarizes the subject collections that have been assembled for this meta-analysis. All subjects met either 1) a diagnosis of MS per McDonald criteria<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890445">6</a></sup> or 2) a diagnosis of clinically isolated demyelinating syndrome (CIS) in which individuals have had one episode of inflammatory demyelination and harbor two or more T2 hyperintense lesions in their brain or spinal cord. The majority of CIS subjects go on to have a second episode of inflammatory demyelination, which results in a diagnosis of MS. An earlier study did not find differences in the distribution of susceptibility alleles in CIS and MS subjects, suggesting that their genetic architecture is similar.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890432">7</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p78">Characteristics of the meta-analyzed datasets</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337124670064" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Dataset</th><th rowspan="1" colspan="1" valign="bottom" align="center">IMSGC UK<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">IMSGC US<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene MSA CH<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene MSA NL<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene MSA US<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">ANZ<sup>†</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">BWH/TT<sup>§</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left"><strong>Number of individuals (cases/controls)</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">453/2950</td><td rowspan="1" colspan="1" valign="middle" align="center">342/1679</td><td rowspan="1" colspan="1" valign="middle" align="center">253/208</td><td rowspan="1" colspan="1" valign="middle" align="center">230/232</td><td rowspan="1" colspan="1" valign="middle" align="center">486/431</td><td rowspan="1" colspan="1" valign="middle" align="center">1618/1988</td><td rowspan="1" colspan="1" valign="middle" align="center">2313/4857</td></tr><tr><td colspan="8" rowspan="1" valign="middle" align="left"><strong>Clinical characteristics (cases only)</strong></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> <strong>Female:Male Ratio</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">3.0:1:0</td><td rowspan="1" colspan="1" valign="middle" align="center">3.2:1.0</td><td rowspan="1" colspan="1" valign="middle" align="center">2.8:1.0</td><td rowspan="1" colspan="1" valign="middle" align="center">2.9:1.0</td><td rowspan="1" colspan="1" valign="middle" align="center">3.1:1.0</td><td rowspan="1" colspan="1" valign="middle" align="center">2.6:1</td><td rowspan="1" colspan="1" valign="middle" align="center">2.6:1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> <strong>Mean Disease Duration, years (range)</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">11 (0–40)</td><td rowspan="1" colspan="1" valign="middle" align="center">16 (&lt;1–36)</td><td rowspan="1" colspan="1" valign="middle" align="center">12 (&lt;1–58)</td><td rowspan="1" colspan="1" valign="middle" align="center">13 (&lt;1–39)</td><td rowspan="1" colspan="1" valign="middle" align="center">15 (1–59)</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td><td rowspan="1" colspan="1" valign="middle" align="center">14 (2–53)/5(0–33)<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> <strong>Mean age at onset, years (range)</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">27 (10–48)</td><td rowspan="1" colspan="1" valign="middle" align="center">29 (11–50)</td><td rowspan="1" colspan="1" valign="middle" align="center">33 (8–59)</td><td rowspan="1" colspan="1" valign="middle" align="center">33 (13–71)</td><td rowspan="1" colspan="1" valign="middle" align="center">33 (1–70)</td><td rowspan="1" colspan="1" valign="middle" align="center">34 (7–67)</td><td rowspan="1" colspan="1" valign="middle" align="center">33 (9–60)/30 (8–54) <sup>‡</sup></td></tr><tr><td colspan="8" rowspan="1" valign="middle" align="left"><strong>Genotypic and analysis characteristics</strong></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> <strong>Genotyping platform</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">Affy 500K</td><td rowspan="1" colspan="1" valign="middle" align="center">Affy 500K</td><td rowspan="1" colspan="1" valign="middle" align="center">Illumina 550</td><td rowspan="1" colspan="1" valign="middle" align="center">Illumina 550</td><td rowspan="1" colspan="1" valign="middle" align="center">Illumina 550</td><td rowspan="1" colspan="1" valign="middle" align="center">Cases: Illumina Hap370CNV<br>Controls: Illumina Infinium</td><td rowspan="1" colspan="1" valign="middle" align="center">Affy 6.0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> <strong>Analyzed individuals (cases/controls)</strong><sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">449/2928</td><td rowspan="1" colspan="1" valign="middle" align="center">341/1679</td><td rowspan="1" colspan="1" valign="middle" align="center">251/208</td><td rowspan="1" colspan="1" valign="middle" align="center">225/228</td><td rowspan="1" colspan="1" valign="middle" align="center">477/425</td><td rowspan="1" colspan="1" valign="middle" align="center">1616/1987</td><td rowspan="1" colspan="1" valign="middle" align="center">2186/4698</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"> <strong>Genomic inflation factor (lambda)</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">1.029</td><td rowspan="1" colspan="1" valign="middle" align="center">1.034</td><td rowspan="1" colspan="1" valign="middle" align="center">1.04</td><td rowspan="1" colspan="1" valign="middle" align="center">1.026</td><td rowspan="1" colspan="1" valign="middle" align="center">1.029</td><td rowspan="1" colspan="1" valign="middle" align="center">1.061</td><td rowspan="1" colspan="1" valign="middle" align="center">1.054</td></tr></tbody></table></div><div id="largeobj_idm140337124670064" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3247076/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>Analyzed in our previous meta-analysis <sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890462">2</a></sup></div><div id="TFN2"><sup>†</sup>Analyzed in the ANZgene Study<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343095587">4</a></sup></div><div id="TFN3"><sup>§</sup>Analyzed in part in our previous meta-analysis.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890467">3</a></sup> The dataset used in De Jager et al. has 860 subjects with MS from BWH and their 1720 matched healthy controls.</div><div id="TFN4"><sup>‡</sup>Values are for BWH cases and TT cases separately.</div><div id="TFN5"><p id="__p83" class="p p-first-last">NA: not available</p></div><div id="TFN6"><sup>a</sup>Analyzed cases and controls per dataset after Quality Control.</div></div></div><p id="P8">We used EIGENSOFT to remove outliers in terms of genetic ancestry and to calculate the top ten eigenvectors of the genotype data within each stratum.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890469">8</a></sup> The seven data sets were genotyped using different genotyping platforms. To maximize genome-wide coverage, we used the imputation algorithm implemented in MACH to yield 2.5 million SNPs across the genome in all data sets.<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup> Imputation based on linkage disequilibrium (LD) patterns observed in a representative European population sample in HapMap is a widely used approach to increase the power of GWAS and facilitate <em>in silico</em> meta-analysis.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890461">10</a></sup> After imputation, we excluded all SNPs with an imputation quality score less or equal to 0.10 or minor allele frequency (MAF) &lt;0.01 per stratum. For each stratum, we tested the imputed dosages for association to case-control status using logistic regression, including the ten first eigenvectors as covariates to correct for population stratification. For each SNP, the dosage corresponds to the (imputed) number of the coded allele in a given individual and varies from 0 to 2 on a continuous scale, thus incorporating information about the imputation uncertainty. Under a per-allele model, we calculated the odds ratio (OR), its corresponding standard error (SE) and p-value. To evaluate the robustness of the observed distribution of the test statistic, we inspected the quantile- quantile (Q-Q) plot and calculated the genomic inflation factor (λ<sub>GC</sub>).<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890431">11</a></sup> To correct for any residual, unexplained inflation of the test statistic, we corrected the SEs by multiplying them with the square root of the λ<sub>GC</sub>.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890442">10</a></sup> Finally, we performed the same analyses adjusting for sex and report these sensitivity analyses in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary material</a>.</p><p id="P9">Ensuring consistency in the strand orientation of the alleles across all strata, we meta-analyzed the ORs with the respective corrected SEs using inverse variance weighting under a fixed-effects model. We calculated the λ<sub>GC</sub> of the genome-wide association results to evaluate the robustness of the meta-analysis. Furthermore, in a secondary sensitivity analysis, we used a random effects model to meta-analyze, thus allowing for between-study heterogeneity. We used Cochran’s Q to test for the presence of statistical heterogeneity and I<sup>2</sup>, with respective 95% confidence intervals, to quantify inconsistency of effects across the different strata.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890459">12</a>, <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890486">13</a></sup></p><p id="P10">We performed linkage disequilibrium (LD) pruning (r<sup>2</sup> &gt; 0.5) among correlated SNPs to identify the most statistically significant SNP in regions of strong LD. For non-MHC loci, we performed conditional analysis using a forward stepwise logistic regression for the most statistically significant SNPs (P &lt; 10<sup>−6</sup>) within a 2-Mb distance from the best index SNP (with the lowest p-value) at a locus.</p><p id="P11">We pre-defined genome-wide significance for our meta-analysis at a P-value of &lt; 5×10<sup>−8</sup>. At this type I error rate, and under a fixed effect model, we have more than 80% power to detect an OR of 1.15 for a risk allele with 0.4 minor allele frequency. The Cochran’s Q test was considered to be statistically significant at P &lt; 0.10. For analyses we used the PLINK v1.07<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890450">14</a></sup> and R-2.11.</p><p id="P12">To leverage the rapidly growing list of susceptibility loci associated with inflammatory diseases, we tested all known SNP associations with Crohn’s disease (CD), ulcerative colitis (UC), celiac disease (CE), type 1 diabetes (T1D), rheumatoid arthritis (RA), systemic lupus erythematosus (SLE) and psoriasis (PS) for a role in MS. To identify these <em>bona fide</em> associations, we searched the online NHGRI catalog (<a href="http://www.genome.gov/26525384" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.genome.gov/26525384</a>) and PubMed for GWAs, meta-analyses of GWAS, or follow-up studies that reported a non-MHC SNP for these diseases with genome-wide significance (P &lt; 5×10<sup>−8</sup>). For each of these SNPs we tested for replication of an effect using different p-value thresholds (0.05, 10<sup>−3</sup>, and 10<sup>−4</sup>), allowing for heterogeneity in the direction of effect. As a comparison, we also list the genome-wide non-MHC SNPs associated with type 2 diabetes (T2D),<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890449">15</a></sup> height (HI),<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890478">16</a></sup> lipid traits (LI),<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890470">17</a></sup> and myocardial infraction (MI)<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890456">18</a>–<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890480">20</a></sup> as negative control diseases, because we do not expect these diseases to have an etiologic relationship with MS.</p><p id="P13">For all SNPs that reached a P &lt; 10<sup>−6</sup>, we also performed a meta-analysis under a recessive and dominant model. We used the posterior probabilities for each of the three genotypes (AA, AB, BB) from the imputations to calculate the corresponding dominant and recessive dosage in each individual for each SNP. With these dosages, we calculated the per-stratum ORs and corrected SEs, and meta-analyzed these to obtain the overall ORs and the corresponding p-values.</p><p id="P14" class="p p-last">For the newly identified susceptibility loci, we sought to test the hypothesis that the identified SNPs can affect expression levels of nearby genes (within 1 Mb upstream and downstream of the SNP). We collected RNA expression data with an Affymetrix U133 v2.0 array from peripheral blood mononuclear cells (PBMCs) of 228 subjects with Relapsing Remitting (RR) MS or CIS. These data were collected between July 2002 and October 2007, as part of the Comprehensive Longitudinal Investigation of MS at the Brigham and Women’s Hospital.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890455">21</a>,<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890458">22</a></sup> We regressed the observed gene expression on the SNP imputed dosages, adjusted for the treatment used. The probes that passed our quality check criteria (n=20,517) were used for the subsequent analyses. In an exploratory analysis, we performed an eQTL analysis of all of probes for each newly identified loci and organized the tail of the distribution of the results, i.e. probes that reached a nominal significance threshold (p &lt; 0.05), using the Ingenuity Pathway Analysis (IPA) software. Ingenuity maps probe IDs to its database and performs statistical computing to identify the most significant canonical pathways and networks overrepresented in a given gene list as compared with the whole list of genes in the Human Genome U133 Plus 2.0 array. The canonical pathway analysis tool identified the pathways from the IPA library of canonical pathways that were most significant to the dataset, based upon genes within the dataset that were associated with a canonical pathway in the Ingenuity Pathways Knowledge Base. In a similar way, the software leveraged the input gene expression data to provide networks. Specifically, molecules of interest, which interact with each other, and molecules in the Ingenuity Knowledge Base were identified as Network Eligible Molecules, which served as “seeds” for generating networks.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">RESULTS</h2><p id="P15" class="p p-first">Overall, 5,545 cases and 12,153 controls passed QC and were included in the meta- analysis (<a href="/pmc/articles/PMC3247076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), and 2,529,394 unique SNPs were analyzed in at least two strata. The genomic inflation factor (λ<sub>GC</sub>) for the seven strata ranged from 1.026 to 1.061 (<a href="/pmc/articles/PMC3247076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), suggesting that population stratification within the individual strata was limited. The genomic inflation factor of the genome-wide meta-analysis results was 1.051, indicating that the test statistic distribution is well calibrated and that the extent of residual bias (including unaccounted for stratification and technical artifact) is minimal.</p><div id="S8" class="sec"><h3 id="S8title">Genome-wide significant SNPs under a per-allele genetic model (P&lt;5×10<sup>−8</sup>)</h3><p id="P16" class="p p-first">Of 2,617 SNPs that reach genome-wide significance (P &lt; 5.0 × 10<sup>−8</sup>), 2,583 SNPs are located within the MHC on chromosome 6p21, where index SNP rs3129889 (OR = 2.97, P = 1.03×10<sup>−206</sup>) and rs9260489 (OR = 1.21, P = 1.16×10<sup>−11</sup>) tag the <em>HLA-DRB1</em> and <em>HLA-B</em> associations, respectively.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890440">3</a></sup> Outside of the MHC, we observed SNP associations at seven loci with genome-wide significance (<a href="/pmc/articles/PMC3247076/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>, <a href="/pmc/articles/PMC3247076/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> for the sex-adjusted analyses). Three of these loci have not been described previously as being associated with MS or other inflammatory diseases. First, rs170934 at locus 3p24.1 (P = 1.6 × 10<sup>−8</sup>) demonstrated an OR of 1.17 for the minor T allele, with little evidence for statistical heterogeneity (I<sup>2</sup> = 2%; P = 4.7 × 10<sup>−8</sup> under a random-effects model). <a href="/pmc/articles/PMC3247076/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a> shows the regional association plot for this SNP, which is located in an intergenic area between <em>CMC1</em>, a gene with no known function, and <em>EOMES</em>, a T-box gene family member and a paralog of <em>TBX21/TBET</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3247076/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337126356864" class="figure"><a href="/pmc/articles/PMC3247076/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3247076_nihms318621f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms318621f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3247076/bin/nihms318621f1.jpg"></a></div><div id="largeobj_idm140337126356864" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3247076/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3247076/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Manhattan plot for the meta-analysis genome-wide – log(p-values) (fixed effects)</strong><p id="__p75">X axis displays the 22 autosomal chromosomes and Y axis the –log(p-values) per SNP. The red line represents the genome-wide significance level (5×10<sup>−8</sup>)</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3247076/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337124612832" class="figure"><a href="/pmc/articles/PMC3247076/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3247076_nihms318621f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms318621f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3247076/bin/nihms318621f2.jpg"></a></div><div id="largeobj_idm140337124612832" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3247076/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3247076/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional association plots for the newly identified genome-wide significant loci and respective forest plots</strong><p id="__p76"><strong>(A, B)</strong> rs170934 in <em>EOMES</em>, <strong>(C, D)</strong> rs2150702 in <em>MLANA</em>, and <strong>(E, F)</strong> rs6718520 near <em>THADA</em>. Regional plots: The X axis plots 1 million basepairs around the most statistically significant (index) SNP, which is highlighted by a large red diamond. r<sup>2</sup> of a given SNP with the index SNP is illustrated with the intensity of the red color. The blue line represents the recombination rate. Each square represents one SNP. Forest Plots: The per-datasets’ weights are from the fixed effects meta-analysis. The p-value is for the Cohran’s Q test for statistical heterogeneity. INFO score is an imputation quality metric, corresponding to the ratio of observed vs. expected allele frequency. Values greater of 0.8 indicate high imputation quality. Genotyped SNPs have a value of 1. SNPs that were genotyped in a given dataset are marked with an asterisk.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p85">Meta-analysis results for SNPs with p-value &lt;10<sup>−6</sup> and known genes/loci.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337124079552" class="xtable"><table frame="void" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Minor Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Major Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">CHR</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF<sup>$</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Locus (±200Kbp)</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR</th><th rowspan="1" colspan="1" valign="bottom" align="center">p-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">p-value for statistical heterogeneity</th><th rowspan="1" colspan="1" valign="bottom" align="center">I<sup>2</sup> (95%CI)</th></tr></thead><tbody><tr><td colspan="11" rowspan="1" valign="top" align="center">Never reported as genome-wide significant</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs170934</td><td rowspan="1" colspan="1" valign="top" align="center">28054089</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center"><em>EOMES</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.17</td><td rowspan="1" colspan="1" valign="top" align="center">1.57E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.407</td><td rowspan="1" colspan="1" valign="top" align="center">2% (0–72%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2150702</td><td rowspan="1" colspan="1" valign="top" align="center">5883861</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MLANA</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">3.28E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.682</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6718520</td><td rowspan="1" colspan="1" valign="top" align="center">43179074</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center"><em>THADA</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.17</td><td rowspan="1" colspan="1" valign="top" align="center">3.42E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.157</td><td rowspan="1" colspan="1" valign="top" align="center">35% (0–73%)</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="11" rowspan="1" valign="top" align="center">Published genes/loci</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10492972</td><td rowspan="1" colspan="1" valign="top" align="center">10275699</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KIF1B</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.243</td><td rowspan="1" colspan="1" valign="top" align="center">0.605</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2300747</td><td rowspan="1" colspan="1" valign="top" align="center">116905738</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CD58</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.73</td><td rowspan="1" colspan="1" valign="top" align="center">6.46E-09</td><td rowspan="1" colspan="1" valign="top" align="center">0.715</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2760524</td><td rowspan="1" colspan="1" valign="top" align="center">190797171</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center"><em>RGS1</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">3.00E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">63% (16–84%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12122721</td><td rowspan="1" colspan="1" valign="top" align="center">199251103</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KIF21B</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">2.43E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.247</td><td rowspan="1" colspan="1" valign="top" align="center">24% (0–66%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9846534<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">107064440</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CBLB</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.97</td><td rowspan="1" colspan="1" valign="top" align="center">0.434</td><td rowspan="1" colspan="1" valign="top" align="center">0.327</td><td rowspan="1" colspan="1" valign="top" align="center">13% (0–75%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1132200</td><td rowspan="1" colspan="1" valign="top" align="center">120633526</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TMEM39A</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">3.67E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.442</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–58%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4680534</td><td rowspan="1" colspan="1" valign="top" align="center">161181639</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IL12A</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.12</td><td rowspan="1" colspan="1" valign="top" align="center">3.28E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.957</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1393122</td><td rowspan="1" colspan="1" valign="top" align="center">4778148</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.96</td><td rowspan="1" colspan="1" valign="top" align="center">0.287</td><td rowspan="1" colspan="1" valign="top" align="center">0.949</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6897932</td><td rowspan="1" colspan="1" valign="top" align="center">35910332</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IL7RA</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">2.26E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.304</td><td rowspan="1" colspan="1" valign="top" align="center">16% (0–60%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9260489</td><td rowspan="1" colspan="1" valign="top" align="center">30028311</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center"><em>HLA-B</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.21</td><td rowspan="1" colspan="1" valign="top" align="center">1.16E-11</td><td rowspan="1" colspan="1" valign="top" align="center">0.221</td><td rowspan="1" colspan="1" valign="top" align="center">27% (0–68%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3129889</td><td rowspan="1" colspan="1" valign="top" align="center">32521523</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center"><em>HLA-DRB1</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.97</td><td rowspan="1" colspan="1" valign="top" align="center">1.03E-206</td><td rowspan="1" colspan="1" valign="top" align="center">0.506</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12722489 <sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6142018</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IL2RA</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td><td rowspan="1" colspan="1" valign="top" align="center">3.66E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.818</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7089861 <sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6150332</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IL2RA</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td><td rowspan="1" colspan="1" valign="top" align="center">3.84E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.113</td><td rowspan="1" colspan="1" valign="top" align="center">42% (0–73%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs17824933</td><td rowspan="1" colspan="1" valign="top" align="center">60517188</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CD6</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.14</td><td rowspan="1" colspan="1" valign="top" align="center">3.38E-05</td><td rowspan="1" colspan="1" valign="top" align="center">4.70E-03</td><td rowspan="1" colspan="1" valign="top" align="center">68% (29–86%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1800693</td><td rowspan="1" colspan="1" valign="top" align="center">6310270</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TNFRSF1A</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.14</td><td rowspan="1" colspan="1" valign="top" align="center">1.41E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.349</td><td rowspan="1" colspan="1" valign="top" align="center">10% (0–74%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs703842</td><td rowspan="1" colspan="1" valign="top" align="center">56449006</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center"><em>METTL1</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">1.72E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.877</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1790100</td><td rowspan="1" colspan="1" valign="top" align="center">122222678</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MPHOSPH9</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">6.61E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.486</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs17445836</td><td rowspan="1" colspan="1" valign="top" align="center">84575164</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IRF8</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">5.30E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.211</td><td rowspan="1" colspan="1" valign="top" align="center">28% (0–69%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12708716</td><td rowspan="1" colspan="1" valign="top" align="center">11087374</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CLEC16A</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">1.08E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.509</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs744166<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="center">37767727</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center"><em>STAT3</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">6.35E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.327</td><td rowspan="1" colspan="1" valign="top" align="center">13% (0–75%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2293152<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="center">37735055</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center"><em>STAT3</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.82</td><td rowspan="1" colspan="1" valign="top" align="center">4.09E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.486</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs763361</td><td rowspan="1" colspan="1" valign="top" align="center">65682622</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CD226</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.045</td><td rowspan="1" colspan="1" valign="top" align="center">0.994</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6074022<sup>d</sup></td><td rowspan="1" colspan="1" valign="top" align="center">44173603</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CD40</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.15</td><td rowspan="1" colspan="1" valign="top" align="center">4.91E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.170</td><td rowspan="1" colspan="1" valign="top" align="center">34% (0–72%)</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="11" rowspan="1" valign="top" align="center">Suggestive (5×10<sup>−8</sup> &lt;p&lt;1×10<sup>−6</sup>)</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2546890 <sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="center">158692478</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IL12B</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.86</td><td rowspan="1" colspan="1" valign="top" align="center">7.95E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.509</td><td rowspan="1" colspan="1" valign="top" align="center">0%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs8070463</td><td rowspan="1" colspan="1" valign="top" align="center">43123835</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KPNB1/TBKBP1/TBX21</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">9.55E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.586</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10411936</td><td rowspan="1" colspan="1" valign="top" align="center">16409375</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center"><em>EPS15L1</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">2.04E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.881</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2681424</td><td rowspan="1" colspan="1" valign="top" align="center">123252212</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ILDR1/CD86</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">2.33E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.996</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7592330</td><td rowspan="1" colspan="1" valign="top" align="center">68500287</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PLEK/FBXO48/C2orf13</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">2.42E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.983</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1738074</td><td rowspan="1" colspan="1" valign="top" align="center">159385965</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TAGAP</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">3.724E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.238</td><td rowspan="1" colspan="1" valign="top" align="center">25% (0–67%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1250542</td><td rowspan="1" colspan="1" valign="top" align="center">80704676</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ZMIZ1</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.15</td><td rowspan="1" colspan="1" valign="top" align="center">3.97E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.370</td><td rowspan="1" colspan="1" valign="top" align="center">8% (0–73%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7191700</td><td rowspan="1" colspan="1" valign="top" align="center">11314304</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TNP2/PRM3/PRM2/PRM1/C16orf75</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">6.40E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.705</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10866713<sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="center">158851472</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IL12B</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.17</td><td rowspan="1" colspan="1" valign="top" align="center">6.57E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.658</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9596270</td><td rowspan="1" colspan="1" valign="top" align="center">49740441</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">13</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.74</td><td rowspan="1" colspan="1" valign="top" align="center">7.00E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.347</td><td rowspan="1" colspan="1" valign="top" align="center">11% (0–74%)</td></tr></tbody></table></div><div id="largeobj_idm140337124079552" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3247076/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><sup>$</sup>Weighted minor allele frequency across all datasets.</div><div id="TFN8"><sup>a</sup>The reported SNP in the original publication is rs9657904. These two are in perfect LD (r<sup>2</sup>=1).</div><div id="TFN9"><sup>b</sup>rs12722489 and rs7089861 are in week linkage disequilibrium (r<sup>2</sup>=0.128).</div><div id="TFN10"><sup>c</sup>rs744166 was reported in the original publication. rs744166 and rs2293152 are in week LD (r<sup>2</sup>=0. 0.128). These represent one effect.</div><div id="TFN11"><sup>d</sup>rs6074022 did not reached genome wide significance in the original publication (p=1.30E-07).</div><div id="TFN12"><sup>e</sup>rs2546890 and rs10866713 are in week LD (r<sup>2</sup>= 0.01). These represent two independent effects.</div></div></div><p id="P17">The second novel locus we have identified is tagged by rs2150702, a SNP in the second intron of the <em>MLANA</em> gene on chromosome 9p24.1, which is known as a melanoma antigen (<a href="/pmc/articles/PMC3247076/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2C, D</span></a>). The minor G allele of this SNP increases risk (OR = 1.16) with no evidence for statistical heterogeneity across the seven strata (P = 3.3 × 10<sup>−8</sup> under both fixed and random-effects models).</p><p id="P18">The third novel locus is tagged by rs6718520, which maps to an intergenic region on chromosome 2p21, with <em>THADA</em> as the nearest flanking genes at 132kbps distance (<a href="/pmc/articles/PMC3247076/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2E</span></a>). The minor A allele of this SNP has an OR of 1.17 (P = 3.4 × 10<sup>−8</sup>) with modest evidence for statistical heterogeneity (I<sup>2</sup> = 35%, P = 2.6 × 10<sup>−4</sup> under a random-effects model). This heterogeneity may come from the fact that the quality of this SNP’s imputation varies across the strata of the meta-analysis, that residual population substructure in some strata influences our analysis, or it may be true heterogeneity in the effect of the SNP in different groups of subjects. The stronger per-dataset effect was observed in a low imputation quality dataset (Gene MSA NL, <a href="/pmc/articles/PMC3247076/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2F</span></a>), so we explored the influence of the imputation’s quality on the statistical heterogeneity and the overall effect size (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 3</a>). Under the most conservative scenario of synthesizing only the high imputation quality studies (INFO&gt;0.9) the p-value was 8.44 × 10<sup>−6</sup>, although suffering a huge power loss due to the smaller sample size.</p><p id="P19" class="p p-last">The other four loci with genome-wide significant SNPs correspond to already known MS susceptibility loci. At the <em>IL2RA</em> gene, we observed two index SNPs (rs2104286 and rs7089861), with independent effects (r<sup>2</sup>=0.128 in HapMap-CEU) that correspond to the two previously described independent effects within the <em>IL2RA</em> locus.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890463">23</a></sup> Another SNP, rs1335532, tags the known association in the <em>CD58</em> locus (r<sup>2</sup>=0.87 with rs2300747, the previously described best marker).<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890483">21</a></sup> The fourth SNP, rs2293152, captures the recently described association of the <em>STAT3</em> locus with MS susceptibility.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890435">24</a></sup> Many of the subjects are shared between the present study and the original discoveries of these three loci, so the findings here do not constitute independent evidence for replication of these loci. In addition to the <em>CD58</em>, <em>IL2RA</em> and <em>STAT3</em> loci that reach genome-wide significance in the present meta-analysis, we have tabulated the association results for all MS susceptibility loci reported to date (<a href="/pmc/articles/PMC3247076/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We observe substantial evidence of association for all known loci with three exceptions (<em>KIF1B</em>,<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890444">25</a></sup> <em>CBLB</em>,<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343095588">4</a></sup> and <em>chr5p15.32</em><sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890484">26</a></sup>) that do not reach nominal statistical significance (<a href="/pmc/articles/PMC3247076/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p></div><div id="S9" class="sec"><h3 id="S9title">SNPs with suggestive evidence of association (p &lt; 10<sup>−6</sup>)</h3><p id="P20" class="p p-first-last">Ten additional SNPs present strongly suggestive evidence of association with MS (defined as p &lt; 10<sup>−6</sup>) (<a href="/pmc/articles/PMC3247076/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Two of these are found the <em>IL12B</em> locus, which is known to be associated with Crohn’s disease and psoriasis (<a href="/pmc/articles/PMC3247076/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary table 1</a>).<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890466">27</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890451">28</a></sup> Another 3 loci -- <em>TAGAP</em> (rs1738074), <em>ZMIZ1</em> (rs1250542), and <em>PLEK</em> (rs7592330) -- have been described previously as susceptibility loci for other inflammatory diseases (<a href="/pmc/articles/PMC3247076/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>),<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890452">29</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890474">30</a></sup> and <em>ZMIZ1</em> has been described previously as having suggestive evidence of association with MS.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890457">3</a></sup> Of the remaining loci, two contain genes that have long been described as being involved in the immune dysregulation seen in MS: <em>TBX21</em> and <em>CD86</em>. The rs8070463 SNP (OR = 0.87, p-value = 9.55×10<sup>−8</sup>) lies within 42kb of <em>TBX21,</em> which is also known as <em>TBET</em> and is a paralog of the <em>EOMES</em> gene found in the novel 3p24.1 susceptibility locus described above. <em>TBX21</em> is of great interest in T cell function because it is a master regulatory gene necessary for the differentiation of pathogenic Th1 lymphocytes that play an important role in murine inflammatory demyelination.<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890460">31</a>,<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890436">32</a></sup> Nonetheless, this locus contains another gene, <em>TBKBP1</em>, that is involved in <em>NFKB</em> signaling and could also be implicated in the effect of this locus; functional dissection will be required to differentiate the role of the different genes found in this locus. The second locus of immunological interest is tagged by rs2681424 (OR =1.16, p-value=2.33 × 10<sup>−7</sup>) and contains <em>CD86</em>, a costimulatory molecule that is the receptor for the CD28 and CTLA4 molecules and is an important component of the machinery regulating the activation of T cells <sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890482">33</a></sup>. These two genes are found in loci associated with susceptibility to rheumatoid arthritis (<em>CD28</em>) and T1D as well as thyroiditis (<em>CTLA4</em>).<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481353068">34</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p92">Meta-analysis results for previously published known (p&lt;5×10<sup>−8</sup>) non-MHC SNPs in inflammatory diseases that are nominally (&lt;0.05) significant in the current meta-analysis.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337206764976" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="3" colspan="1" valign="bottom" align="center">Position</th><th rowspan="3" colspan="1" valign="bottom" align="center">Allele</th><th rowspan="3" colspan="1" valign="bottom" align="center">CHR</th><th rowspan="3" colspan="1" valign="bottom" align="center">Gene/locus<sup>*</sup></th><th colspan="3" rowspan="2" valign="bottom" align="center">Inflammatory diseases</th><th colspan="8" rowspan="1" valign="bottom" align="center">Multiple Sclerosis</th></tr><tr><th rowspan="2" colspan="1" valign="bottom" align="center">Allele (other allele)</th><th colspan="2" rowspan="1" valign="bottom" align="center">Fixed effects</th><th colspan="2" rowspan="1" valign="bottom" align="center">Random effects</th><th rowspan="2" colspan="1" valign="bottom" align="center">p-value for statistical heterogeneity</th><th rowspan="2" colspan="1" valign="bottom" align="center">I<sup>2</sup> (95%CI)</th><th rowspan="2" colspan="1" valign="bottom" align="center">FREQ</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Disease</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR</th><th rowspan="1" colspan="1" valign="bottom" align="center">p-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR</th><th rowspan="1" colspan="1" valign="bottom" align="center">p-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR</th><th rowspan="1" colspan="1" valign="bottom" align="center">p-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3748816</td><td rowspan="1" colspan="1" valign="top" align="center">2516596</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">TNFRSF14, MMEL1</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">3.28E-09</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">2.81E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">2.81E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.977</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10903122</td><td rowspan="1" colspan="1" valign="top" align="center">25176153</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">RUNX3</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">1.73E-10</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">3.27E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.127</td><td rowspan="1" colspan="1" valign="top" align="center">0.038</td><td rowspan="1" colspan="1" valign="top" align="center">55% (0–81%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2201841</td><td rowspan="1" colspan="1" valign="top" align="center">67466780</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">IL23R</td><td rowspan="1" colspan="1" valign="top" align="center">UC</td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">1.30E-13</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">8% (0–73%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.69</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">PS</td><td rowspan="1" colspan="1" valign="top" align="center">1.35</td><td rowspan="1" colspan="1" valign="top" align="center">3E-08</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">8% (0–73%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2816316</td><td rowspan="1" colspan="1" valign="top" align="center">190803426</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">RGS1</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">0.8</td><td rowspan="1" colspan="1" valign="top" align="center">2.20E-17</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">9.03E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.85</td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="center">60% (9–83%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs296547</td><td rowspan="1" colspan="1" valign="top" align="center">199158750</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">4.11E-09</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">7.48E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">0.011</td><td rowspan="1" colspan="1" valign="top" align="center">0.135</td><td rowspan="1" colspan="1" valign="top" align="center">39% (0–74%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11584383</td><td rowspan="1" colspan="1" valign="top" align="center">199202479</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.18</td><td rowspan="1" colspan="1" valign="top" align="center">1.43E-11</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">4.63E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">6.12E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.201</td><td rowspan="1" colspan="1" valign="top" align="center">30% (0–70%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.78</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs13017599</td><td rowspan="1" colspan="1" valign="top" align="center">61017825</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">REL</td><td rowspan="1" colspan="1" valign="top" align="center">RA</td><td rowspan="1" colspan="1" valign="top" align="center">1.21</td><td rowspan="1" colspan="1" valign="top" align="center">2E-12</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.12</td><td rowspan="1" colspan="1" valign="top" align="center">6.66E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">4.69E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.191</td><td rowspan="1" colspan="1" valign="top" align="center">31% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs13003464</td><td rowspan="1" colspan="1" valign="top" align="center">61040323</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">REL</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">1.15</td><td rowspan="1" colspan="1" valign="top" align="center">3.71E-13</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.015</td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.015</td><td rowspan="1" colspan="1" valign="top" align="center">0.970</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">UC</td><td rowspan="1" colspan="1" valign="top" align="center">1.2</td><td rowspan="1" colspan="1" valign="top" align="center">7.4E-09</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.015</td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.015</td><td rowspan="1" colspan="1" valign="top" align="center">0.970</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs934734</td><td rowspan="1" colspan="1" valign="top" align="center">65449080</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">SPRED2</td><td rowspan="1" colspan="1" valign="top" align="center">RA</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">5.30E-10</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">4.5E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.2776</td><td rowspan="1" colspan="1" valign="top" align="center">3E-03</td><td rowspan="1" colspan="1" valign="top" align="center">69% (33–86%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs17035378</td><td rowspan="1" colspan="1" valign="top" align="center">68452449</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">PLEK</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">7.79E-09</td><td rowspan="1" colspan="1" valign="top" align="center">C (T)</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">3.52E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">3.79E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.266</td><td rowspan="1" colspan="1" valign="top" align="center">21% (0–65%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3828309</td><td rowspan="1" colspan="1" valign="top" align="center">233845139</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">ATG16L1</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.28</td><td rowspan="1" colspan="1" valign="top" align="center">2.36E-32</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.018</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.018</td><td rowspan="1" colspan="1" valign="top" align="center">0.981</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4957048</td><td rowspan="1" colspan="1" valign="top" align="center">636432</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">CEP72, TPPP</td><td rowspan="1" colspan="1" valign="top" align="center">UC</td><td rowspan="1" colspan="1" valign="top" align="center">1.3</td><td rowspan="1" colspan="1" valign="top" align="center">1.2E-09</td><td rowspan="1" colspan="1" valign="top" align="center">G (A)</td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.035</td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">0.035</td><td rowspan="1" colspan="1" valign="top" align="center">0.756</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.79</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4613763</td><td rowspan="1" colspan="1" valign="top" align="center">40428475</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">PTGER4</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.32</td><td rowspan="1" colspan="1" valign="top" align="center">6.82E-27</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">1.18</td><td rowspan="1" colspan="1" valign="top" align="center">1.09E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.18</td><td rowspan="1" colspan="1" valign="top" align="center">1.09E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.901</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6859219</td><td rowspan="1" colspan="1" valign="top" align="center">55474327</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">ANKRD55, IL6ST</td><td rowspan="1" colspan="1" valign="top" align="center">RA</td><td rowspan="1" colspan="1" valign="top" align="center">0.85</td><td rowspan="1" colspan="1" valign="top" align="center">9.60E-12</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">0.014</td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">0.057</td><td rowspan="1" colspan="1" valign="top" align="center">0.119</td><td rowspan="1" colspan="1" valign="top" align="center">41% (0–75%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2082412</td><td rowspan="1" colspan="1" valign="top" align="center">158650357</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">IL12B</td><td rowspan="1" colspan="1" valign="top" align="center">PS</td><td rowspan="1" colspan="1" valign="top" align="center">1.56</td><td rowspan="1" colspan="1" valign="top" align="center">2E-28</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.018</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.018</td><td rowspan="1" colspan="1" valign="top" align="center">0.467</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.80</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6887695</td><td rowspan="1" colspan="1" valign="top" align="center">158755213</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">PS</td><td rowspan="1" colspan="1" valign="top" align="center">0.7</td><td rowspan="1" colspan="1" valign="top" align="center">4.08E-10</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">2.02E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">2.02E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.906</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10806425</td><td rowspan="1" colspan="1" valign="top" align="center">90983323</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">BACH2</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">3.89E-10</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">3.90E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">3.90E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.954</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11755527</td><td rowspan="1" colspan="1" valign="top" align="center">91014942</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">BACH2</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">5E-12</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">2.69E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">3.33E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.404</td><td rowspan="1" colspan="1" valign="top" align="center">2% (0–72%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs802734</td><td rowspan="1" colspan="1" valign="top" align="center">128320481</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">PTPRK</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">1.17</td><td rowspan="1" colspan="1" valign="top" align="center">2.62E-14</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">2.01E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">2.01E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.883</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10499194</td><td rowspan="1" colspan="1" valign="top" align="center">138044320</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">TNFAIP3</td><td rowspan="1" colspan="1" valign="top" align="center">RA</td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td><td rowspan="1" colspan="1" valign="top" align="center">1.00E-09</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">3.58E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.358</td><td rowspan="1" colspan="1" valign="top" align="center">2.3E-03</td><td rowspan="1" colspan="1" valign="top" align="center">71% (36–87%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1738074</td><td rowspan="1" colspan="1" valign="top" align="center">159385955</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">TAGAP</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">2.94E-15</td><td rowspan="1" colspan="1" valign="top" align="center">T (C)</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">3.72E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">1.69E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.238</td><td rowspan="1" colspan="1" valign="top" align="center">25% (0–67%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">0.92</td><td rowspan="1" colspan="1" valign="top" align="center">7.59E-09</td><td rowspan="1" colspan="1" valign="top" align="center">T (C)</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">3.72E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">1.69E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.238</td><td rowspan="1" colspan="1" valign="top" align="center">25% (0–67%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs212389</td><td rowspan="1" colspan="1" valign="top" align="center">159409769</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">TAGAP</td><td rowspan="1" colspan="1" valign="top" align="center">RA</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">2.70E-09</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">2.17E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">5.15E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.229</td><td rowspan="1" colspan="1" valign="top" align="center">26% (0–68%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3093023</td><td rowspan="1" colspan="1" valign="top" align="center">167454270</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">CCR6</td><td rowspan="1" colspan="1" valign="top" align="center">RA</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">1.50E-11</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.021</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.021</td><td rowspan="1" colspan="1" valign="top" align="center">0.903</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10758669</td><td rowspan="1" colspan="1" valign="top" align="center">4971592</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">JAK2</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.12</td><td rowspan="1" colspan="1" valign="top" align="center">3.46E-09</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">4.21E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">4.21E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.546</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs706778</td><td rowspan="1" colspan="1" valign="top" align="center">6138945</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">IL2RA</td><td rowspan="1" colspan="1" valign="top" align="center">RA</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">1.40E-11</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">1.78E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">1.78E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.586</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs17582416</td><td rowspan="1" colspan="1" valign="top" align="center">35327646</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">1.79E-09</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.028</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.028</td><td rowspan="1" colspan="1" valign="top" align="center">0.601</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10995271</td><td rowspan="1" colspan="1" valign="top" align="center">64108482</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">ZNF365</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.25</td><td rowspan="1" colspan="1" valign="top" align="center">4.46E-20</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">0.065</td><td rowspan="1" colspan="1" valign="top" align="center">0.101</td><td rowspan="1" colspan="1" valign="top" align="center">43% (0–76%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1250552</td><td rowspan="1" colspan="1" valign="top" align="center">80728023</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">ZMIZ1</td><td rowspan="1" colspan="1" valign="top" align="center">Celiac</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">9.09E-10</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">5.43E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">5.43E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.765</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1250550</td><td rowspan="1" colspan="1" valign="top" align="center">80730313</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">ZMIZ1</td><td rowspan="1" colspan="1" valign="top" align="center">IBD (early onset)</td><td rowspan="1" colspan="1" valign="top" align="center">0.86</td><td rowspan="1" colspan="1" valign="top" align="center">6E-09</td><td rowspan="1" colspan="1" valign="top" align="center">A (C)</td><td rowspan="1" colspan="1" valign="top" align="center">1.12</td><td rowspan="1" colspan="1" valign="top" align="center">1.62E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.12</td><td rowspan="1" colspan="1" valign="top" align="center">1.62E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.749</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4763879</td><td rowspan="1" colspan="1" valign="top" align="center">9801421</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">CD69</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">1.9E-11</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">0.012</td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">0.012</td><td rowspan="1" colspan="1" valign="top" align="center">0.929</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs653178</td><td rowspan="1" colspan="1" valign="top" align="center">110492129</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">SH2B3</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">1.2</td><td rowspan="1" colspan="1" valign="top" align="center">7.15E-21</td><td rowspan="1" colspan="1" valign="top" align="center">C (T)</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">3.82E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">3.82E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.809</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs17696736</td><td rowspan="1" colspan="1" valign="top" align="center">110971191</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">C12orf30,SH2B3,LNK,TRAFD1,PTPN1</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">1.22</td><td rowspan="1" colspan="1" valign="top" align="center">2E-16</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">9.37E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">9.37E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.515</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4900384</td><td rowspan="1" colspan="1" valign="top" align="center">97568694</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">3.7E-09</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.039</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.039</td><td rowspan="1" colspan="1" valign="top" align="center">0.817</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3825932</td><td rowspan="1" colspan="1" valign="top" align="center">77022491</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">CTSH</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">3E-15</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.011</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.011</td><td rowspan="1" colspan="1" valign="top" align="center">0.525</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.67</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12708716</td><td rowspan="1" colspan="1" valign="top" align="center">11087364</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">CLEC16A</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">1.23</td><td rowspan="1" colspan="1" valign="top" align="center">3E-18</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">1.08E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td><td rowspan="1" colspan="1" valign="top" align="center">1.08E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.509</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12928822</td><td rowspan="1" colspan="1" valign="top" align="center">11311384</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">SOCS1</td><td rowspan="1" colspan="1" valign="top" align="center">CE</td><td rowspan="1" colspan="1" valign="top" align="center">0.86</td><td rowspan="1" colspan="1" valign="top" align="center">3.12E-08</td><td rowspan="1" colspan="1" valign="top" align="center">T (C)</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">8.66E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">8.66E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.513</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11860650</td><td rowspan="1" colspan="1" valign="top" align="center">31324197</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">ITGAM</td><td rowspan="1" colspan="1" valign="top" align="center">SLE</td><td rowspan="1" colspan="1" valign="top" align="center">1.43</td><td rowspan="1" colspan="1" valign="top" align="center">1.90E-20</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">2.72E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">2.72E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.535</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1728785</td><td rowspan="1" colspan="1" valign="top" align="center">67148721</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">CDH1</td><td rowspan="1" colspan="1" valign="top" align="center">UC</td><td rowspan="1" colspan="1" valign="top" align="center">1.17</td><td rowspan="1" colspan="1" valign="top" align="center">3E-08</td><td rowspan="1" colspan="1" valign="top" align="center">C (A)</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.021</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.021</td><td rowspan="1" colspan="1" valign="top" align="center">0.893</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.78</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2872507</td><td rowspan="1" colspan="1" valign="top" align="center">35294299</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">ORMDL3</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.12</td><td rowspan="1" colspan="1" valign="top" align="center">5.00E-09</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">5.53E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">5.53E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.464</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2305480</td><td rowspan="1" colspan="1" valign="top" align="center">35315712</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">ORMLD3 region</td><td rowspan="1" colspan="1" valign="top" align="center">UC</td><td rowspan="1" colspan="1" valign="top" align="center">1.14</td><td rowspan="1" colspan="1" valign="top" align="center">3.0E-08</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">5.91E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">9.46E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.404</td><td rowspan="1" colspan="1" valign="top" align="center">3% (0–72%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2290400</td><td rowspan="1" colspan="1" valign="top" align="center">35319756</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">ORMDL3</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">5.5E-13</td><td rowspan="1" colspan="1" valign="top" align="center">C (T)</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">5.54E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.017</td><td rowspan="1" colspan="1" valign="top" align="center">0.208</td><td rowspan="1" colspan="1" valign="top" align="center">29% (0–69%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs744166</td><td rowspan="1" colspan="1" valign="top" align="center">37767717</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">STAT3</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.18</td><td rowspan="1" colspan="1" valign="top" align="center">6.82E-12</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">6.35E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="center">6.14E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.327</td><td rowspan="1" colspan="1" valign="top" align="center">13% (0–75%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.57</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs763361</td><td rowspan="1" colspan="1" valign="top" align="center">65682612</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">CD226</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">1E-08</td><td rowspan="1" colspan="1" valign="top" align="center">T (C)</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.045</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.045</td><td rowspan="1" colspan="1" valign="top" align="center">0.994</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs425105</td><td rowspan="1" colspan="1" valign="top" align="center">51900311</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">T1D</td><td rowspan="1" colspan="1" valign="top" align="center">0.86</td><td rowspan="1" colspan="1" valign="top" align="center">2.7E-11</td><td rowspan="1" colspan="1" valign="top" align="center">T (C)</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.137</td><td rowspan="1" colspan="1" valign="top" align="center">0.150</td><td rowspan="1" colspan="1" valign="top" align="center">36% (0–73%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4810485</td><td rowspan="1" colspan="1" valign="top" align="center">44181364</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">CD40</td><td rowspan="1" colspan="1" valign="top" align="center">RA</td><td rowspan="1" colspan="1" valign="top" align="center">0.85</td><td rowspan="1" colspan="1" valign="top" align="center">8.20E-09</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.15</td><td rowspan="1" colspan="1" valign="top" align="center">4.43E-06</td><td rowspan="1" colspan="1" valign="top" align="center">1.16</td><td rowspan="1" colspan="1" valign="top" align="center">2.47E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.180</td><td rowspan="1" colspan="1" valign="top" align="center">32% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2836878</td><td rowspan="1" colspan="1" valign="top" align="center">39387394</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="center">PSMG1</td><td rowspan="1" colspan="1" valign="top" align="center">UC</td><td rowspan="1" colspan="1" valign="top" align="center">0.73</td><td rowspan="1" colspan="1" valign="top" align="center">1.4E-08</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">1.47E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">1.47E-03</td><td rowspan="1" colspan="1" valign="top" align="center">0.973</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs762421</td><td rowspan="1" colspan="1" valign="top" align="center">44439979</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="center">ICOSLG</td><td rowspan="1" colspan="1" valign="top" align="center">CD</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">1.41E-09</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.029</td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">0.029</td><td rowspan="1" colspan="1" valign="top" align="center">0.597</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–71%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs5771069</td><td rowspan="1" colspan="1" valign="top" align="center">48777597</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="center">IL17REL</td><td rowspan="1" colspan="1" valign="top" align="center">UC</td><td rowspan="1" colspan="1" valign="top" align="center">1.17</td><td rowspan="1" colspan="1" valign="top" align="center">4E-08</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">1.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">0.507</td><td rowspan="1" colspan="1" valign="top" align="center">0% (0–90%)</td><td rowspan="1" colspan="1" valign="top" align="center">0.54</td></tr></tbody></table></div><div id="largeobj_idm140337206764976" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3247076/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN13"><sup>*</sup>Gene/locus as reported in the original publications.</div><div id="TFN14"><p id="__p94" class="p p-first-last">CD: Crohn’s disease, UC: ulcerative colitis, CE: celiac disease, T1D: type 1 diabetes, RA: rheumatoid arthritis, SLE: systemic lupus erythematosus, PS: psoriasis, and IBD: inflammatory bowel disease. All SNPs had data from 7 strata, except rs5771069 that had data from only 2.</p></div></div></div></div><div id="S10" class="sec"><h3 id="S10title">Susceptibility allele overlap between MS and other inflammatory diseases</h3><p id="P21" class="p p-first-last">Motivated by recent progress in the identification of susceptibility loci of other inflammatory diseases, we sought to test the effect of all non-MHC SNPs described to date as <em>bona fide</em> inflammatory disease associated polymorphisms. Of these 145 SNPs, 48 are associated with MS risk at nominal significance, consistent with a shared genetic etiology of inflammatory diseases (<a href="/pmc/articles/PMC3247076/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>, <a href="/pmc/articles/PMC3247076/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a>). Of all the listed inflammatory diseases, SNPs identified in ulcerative colitis and celiac disease seem to replicate better in MS (60% and 45% at a nominal level, respectively. <a href="/pmc/articles/PMC3247076/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a>). Under more stringent statistical significance cut-offs (&lt; 10<sup>−3</sup> and &lt; 10<sup>−4</sup>) few SNPs replicated in our MS dataset, with celiac disease being the most replicated (34% at &lt;10<sup>−3</sup> and 15% at 10<sup>−4</sup>, <a href="/pmc/articles/PMC3247076/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a>). It is interesting to note that, for celiac disease (and most of the other inflammatory diseases) a minority of loci appears to demonstrate association with MS, but in the opposite direction such that a celiac disease risk allele is protective for MS. While such opposite effects have been noted before, we gain an appreciation that they are widespread and that they highlight the complexity of the shared architecture among the inflammatory disease. One example of this complexity is the <em>TAGAP</em> locus in which rs1738074 reaches a p-value &lt; 10<sup>−6</sup> in MS (<a href="/pmc/articles/PMC3247076/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>); this locus has attained genome-wide significance in celiac disease<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890487">35</a></sup> (opposite direction of effect relative to MS) and type 1 diabetes<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890485">30</a></sup> (same direction of effect as MS) (<a href="/pmc/articles/PMC3247076/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="/pmc/articles/PMC3247076/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3B</span></a>). Further, within the <em>TAGAP</em> locus, another SNP (rs212389, r<sup>2</sup>=0.268 with rs1738074) that is associated with susceptibility to RA<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481353069">34</a></sup> has the same direction of effect in MS (<a href="/pmc/articles/PMC3247076/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="/pmc/articles/PMC3247076/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3B</span></a>). Given the extended evidence of association of the <em>TAGAP</em> locus in many inflammatory diseases, it is a strong candidate MS susceptibility locus, although it has not reached a level of genome-wide significance in our study.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3247076/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140337118067072" class="figure"><a href="/pmc/articles/PMC3247076/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3247076_nihms318621f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms318621f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3247076/bin/nihms318621f3.jpg"></a></div><div id="largeobj_idm140337118067072" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3247076/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3247076/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Overlap of the genetic architecture of MS with that of other inflammatory diseases</strong><p id="__p77">(A) Percentage of non-MHC genome-wide significant (P&lt;5×10<sup>−8</sup>) SNPs of inflammatory diseases that are non-statistically significant (NS), or significant in the same direction (SD) or the opposite direction (OD) in the current MS meta-analysis. CE: celiac disease, CD: Crohn’s disease, UC: ulcerative colitis, IBD: inflammatory bowel diseases (CD+UC), PSO: psoriasis, RA: rheumatoid arthritis, SLE: systemic lupus erythematosus, T1D: type 1 diabetes, T2D: type 2 diabetes, HE: height, LI: lipids, MI: myocardial infraction. NS: non-statistically significant, OD: opposite direction of effects, SD: same direction of effects. <strong>(B)</strong> Regional association plot for the <em>TAGAP</em> gene. All SNPs report –lop(p-values) from the MS meta-analysis, besides the 3 ones indicated by the respective disease names. The p-values reported for these 3 SNPs come from the respective original publications. The T1D and CE SNP is rs1738074, whereas the RA one is rs212389.</p></div></div></div></div><div id="S11" class="sec"><h3 id="S11title">Analysis under dominant and recessive genetic models</h3><p id="P22" class="p p-first-last">For the SNP associations with P &lt; 10<sup>−6</sup>, we also explored dominant and recessive genetic models (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>) to assess whether these models fit our data better than our default additive model.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890481">36</a></sup> In several cases, association is enhanced when considering a different model, which could guide the design of replication studies. For example, at the 2p21 locus, the recessive model for the minor rs6718520<sup>A</sup> allele is slightly more significant (OR = 1.29, P = 2.9 × 10<sup>−8</sup>) than the additive model (OR = 1.17, P = 3.4×10<sup>−8</sup>). For two other SNPs, rs1335532 in <em>CD58</em> and rs2293152 in <em>STAT3</em>, the dominant model was more significant than the additive model. Finally, SNP rs9901869, which is found in an intergenic region on chromosome 17 near <em>NPEPPS</em> gene, reached genome-wide significance under the recessive model for the minor G allele (OR = 1.26, P = 3.7 × 10<sup>−8</sup> under both fixed and random-effects models). Thus, considering non-additive models for certain loci appears to be warranted and informs the potential mechanism of a variant’s effect.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Transcriptional and exploratory pathway analysis</h3><p id="P23" class="p p-first">To explore the functional consequences of our new MS susceptibility loci, we assessed whether each of the three index SNPs influenced the level of RNA expression from genes located in the vicinity of each SNP (<em>cis</em> expression quantitative trait locus (eQTL) analysis). We performed these analyses in a set of 228 subjects with demyelinating disease on which we have obtained a genome-wide RNA expression profile from PBMC. While a few suggestive <em>cis</em> associations (P&lt;0.05) with RNA expression were noted in the <em>MLANA</em> (rs2150702) locus, none of the three newly identified susceptibility alleles had statistically significant <em>cis</em> effects after correcting for the number of hypotheses tested in these data (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). This is not unusual as only 625 out of 1598 SNPs associated with a human trait were found to have a cis eQTL effect in a recent survey of validated loci.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890465">37</a></sup></p><p id="P24" class="p p-last">In an exploratory analysis, we implemented a pathway analysis approach by highlighting those pathways or networks in which the expression of multiple genes is influenced by a given SNP in a modest manner (P&lt;0.05). Of the three tested SNPs, rs2150702 in the <em>MLANA</em> locus has a number of pathways that are involved in immune system function. Specifically the best-scoring pathway is related to CD28 signaling, but T and B cell receptor signaling and the IL-2 signaling appear to be involved as well (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). We subsequently evaluated the networks generated for each of the new MS susceptibility loci. In the highest scoring network of the <em>EOMES</em> locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4A</a>), TNFα signaling appears to play a central role, connecting the locus to our previously described <em>TNFRSF1A</em> susceptibility one. The network also contains a decrease in <em>HLA-A</em> gene expression relative to the susceptibility allele, which is in agreement with the validation of at least one protective allele (<em>HLA-A*02</em>) in this gene.<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890443">38</a></sup> The highest scoring network for the <em>MLANA</em> locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4B</a>) contains <em>ERK</em> and connections with type 1 interferon responses previously implicated in MS with the <em>IRF8</em> locus. Reducing the activity of the ERK signaling pathway has been reported to ameliorate experimental inflammatory encephalomyelitis (EAE), a murine model systems that captures many features of MS. <sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890475">39</a></sup> The best network of the <em>THADA</em> locus includes <em>STAT3</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4A</a>), one of the other genes with genome-wide significance in our meta-analysis (<a href="/pmc/articles/PMC3247076/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The expression of <em>STAT3</em> RNA is enhanced relative to the risk allele rs6718520<sup>A</sup> which is consistent with reports that diminished expression of <em>STAT3</em> in humans blocks the development of Th17 cells that play an important role in mediating inflammatory demyelination in MS.<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890454">38</a></sup> Further, <em>STAT3</em> is part of a broader network that includes key regulators of cellular signaling such as <em>ERK</em>.</p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">DISCUSSION</h2><p id="P25" class="p p-first">We have performed a meta-analysis of GWAS in multiple sclerosis and have highlighted three novel loci that reach genome-wide significance. The <em>EOMES</em> and <em>MLANA</em> loci showed no evidence of statistical heterogeneity, while the third locus near <em>THADA</em> showed some degree of heterogeneity among the strata of data. In addition, we point out several loci as having suggestive evidence of being associated with MS, such as the locus containing the <em>TAGAP</em> gene (previously associated with celiac disease, type 1 diabetes, and rheumatoid arthritis) and a locus on chromosome 17 that includes the transcription factor <em>TBX21</em> which plays an important role in the immunopathogenesis of murine models of MS.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890464">32</a></sup></p><p id="P26">These results extend the list of loci associated with MS from earlier genome scans<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890472">2</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890434">5</a></sup> and confirm theoretical predictions that increasing sample sizes will lead to additional discoveries given the magnitude of effect seen for non-MHC MS susceptibility loci. However, &gt;10,000 subjects are needed to be fully powered to identify such common susceptibility alleles of modest effect.<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890437">41</a></sup> One element of our strategy that enhances the likelihood for gene discovery is imputation of genotypes at SNPs not sampled by genotyping arrays using reference maps such as HapMap that catalogue the correlation structure among SNPs in human populations. Imputation allows the integrated analysis of datasets generated on different platforms and extends the analysis from roughly 750,000 genotyped SNPs at the end of quality control pipelines to the 2.5 million SNPs that we considered in this meta-analysis. Critically, confidence in an imputed genotype varies depending on how well it correlates with genotyped SNPs, and, if any uncertainty is present, it is incorporated into the test statistic, down-weighting the importance of the association results at that SNP. Imputation can be a very valuable tool: for example, when a genotyped SNP is an imperfect surrogate marker for a causal variant, the genotyped SNP may not display very strong evidence of association, and such a result may be lost in a genome-wide analysis. By estimating the results of the association test with a causal variant that is not genotyped, imputation can provide more robust evidence of association with a locus than a genotyped marker that is poorly correlated with the causal variant. Thus, imputed SNPs enhance our ability to detect and localize the effect of a new susceptibility locus. <sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890439">41</a>, <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245890471">42</a></sup> However, such results, as with any result from genome scans, needs to be independently replicated before being considered a validated locus.</p><p id="P27">Several limitations are present in our study. First, not all samples were genotyped with the same microarray platform, which can incorporate heterogeneity due to differences in coverage or genotype quality despite genome-wide imputation of &gt;2 million SNPs on the HapMap reference dataset. Second, residual population substructure (even after excluding outliers and incorporating principal components into our analysis as covariates) may have inflated the test statistic. When implementing a very conservative correction for the modest level of genomic inflation seen in each stratum, we find that the <em>EOMES</em> locus remains significant at a genome-wide level, but that the other two new loci fluctuate over this threshold of significance. Thus, the <em>EOMES</em> locus is our most robust result, and we look forward to efforts to validate the role of these loci that we describe in additional collections of MS subjects.</p><p id="P28" class="p p-last">In conclusion, we report three new loci involved in the etiology of MS. Several additional loci are involved in inflammatory disease or immunologic function. Replication in large samples is now required to validate these loci as <em>bona fide</em> susceptibility loci in MS. Once validated, future fine-mapping studies across these loci are needed to provide a comprehensive picture of which variants have causal relationships with MS risk.</p></div><div id="S14" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp Table S1-S5 &amp; Fig S1-S4</h4><div class="sup-box half_rhythm" id="N0x205fae20N0x21548610"><a href="/pmc/articles/PMC3247076/bin/NIHMS318621-supplement-Supp_Table_S1-S5___Fig_S1-S4.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(13M, doc)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supp Table S6</h4><div class="sup-box half_rhythm" id="N0x205fae20N0x215486d0"><a href="/pmc/articles/PMC3247076/bin/NIHMS318621-supplement-Supp_Table_S6.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(44K, xls)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Supp Table S7</h4><div class="sup-box half_rhythm" id="N0x205fae20N0x21548790"><a href="/pmc/articles/PMC3247076/bin/NIHMS318621-supplement-Supp_Table_S7.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(315K, xls)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Acknowledgments</h2><div class="sec"><p id="__p1">P.L.D. is a Harry Weaver Neuroscience Scholar Award Recipient of the National MS Society (NMSS). This project was also supported in part by R01 NS067305 and RC2 NS070340. We thank the subjects with MS who have generously donated DNA samples for theses studies. We thank individuals with MS in Australia and New Zealand for supporting this research. We are grateful to J. Wright and C. Remediakis from Multiple Sclerosis Research Australia (MSRA) for expediting this research. This work was supported by MSRA, John T. Reid Charitable Trusts, Trish MS Research Foundation and the Australian Research Council, under the Linkage Projects Scheme (LP0776744). We thank the International MS Genetics Consortium for the use of genotype data. We thank the Myocardial Infarction Genetics Consortium (MIGen) study for the use of their genotype data as control data in our study. The MIGen study was funded by the U.S. National Institutes of Health and National Heart, Lung, and Blood Institute’s STAMPEED genomics research program and a grant from the National Center for Research Resources. We acknowledge use of genotype data from the British 1958 Birth Cohort DNA collection, funded by the Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02.</p></div></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title">Author Index</h2><!--/article/back/app-group/app/--><p id="P29">Nikolaos A. Patsopoulos<sup><sup><a href="#FN3" rid="FN3" class=" fn">1</a></sup>–<sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,</sup>, Federica Esposito<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>–<sup><a href="#FN8" rid="FN8" class=" fn">6</a></sup></sup>, Joachim Reischl<sup><sup><a href="#FN9" rid="FN9" class=" fn">7</a></sup>,<sup><a href="#FN10" rid="FN10" class=" fn">8</a></sup></sup>, Stephan Lehr<sup><sup><a href="#FN9" rid="FN9" class=" fn">7</a></sup>,<sup><a href="#FN10" rid="FN10" class=" fn">8</a></sup></sup>, David Bauer<sup><sup><a href="#FN9" rid="FN9" class=" fn">7</a></sup>,<sup><a href="#FN10" rid="FN10" class=" fn">8</a></sup></sup>, Jürgen Heubach<sup><sup><a href="#FN9" rid="FN9" class=" fn">7</a></sup>,<sup><a href="#FN10" rid="FN10" class=" fn">8</a></sup></sup>, Rupert Sandbrink<sup><sup><a href="#FN9" rid="FN9" class=" fn">7</a></sup>,<sup><a href="#FN10" rid="FN10" class=" fn">8</a></sup>,<sup><a href="#FN11" rid="FN11" class=" fn">9</a></sup></sup>, Christoph Pohl<sup><sup><a href="#FN9" rid="FN9" class=" fn">7</a></sup>,<sup><a href="#FN10" rid="FN10" class=" fn">8</a></sup>,<sup><a href="#FN12" rid="FN12" class=" fn">10</a></sup></sup>, Gilles Edan<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">12</a></sup>,<sup><a href="#FN15" rid="FN15" class=" fn">13</a></sup></sup>, Ludwig Kappos<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">12</a></sup>,<sup><a href="#FN16" rid="FN16" class=" fn">14</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup>,<sup><a href="#FN18" rid="FN18" class=" fn">16</a></sup></sup>, David Miller<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">12</a></sup>,<sup><a href="#FN19" rid="FN19" class=" fn">17</a></sup></sup>, Javier Montalbán<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">12</a></sup>,<sup><a href="#FN20" rid="FN20" class=" fn">18</a></sup></sup>, Chris H. Polman<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">12</a></sup>, <sup><a href="#FN18" rid="FN18" class=" fn">16</a></sup>, <sup><a href="#FN21" rid="FN21" class=" fn">19</a></sup></sup>, Mark S. Freedman<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">12</a></sup>,<sup><a href="#FN22" rid="FN22" class=" fn">20</a></sup></sup>, Hans-Peter Hartung<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">12</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup>,<sup><a href="#FN23" rid="FN23" class=" fn">21</a></sup>,<sup><a href="#FN24" rid="FN24" class=" fn">22</a></sup></sup>, Barry G.W. Arnason<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup>,<sup><a href="#FN25" rid="FN25" class=" fn">23</a></sup></sup>, Giancarlo Comi<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>-<sup><a href="#FN8" rid="FN8" class=" fn">6</a></sup>,<sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup></sup>, Stuart Cook<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup>,<sup><a href="#FN26" rid="FN26" class=" fn">24</a></sup></sup>, Massimo Filippi<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup>,<sup><a href="#FN24" rid="FN24" class=" fn">22</a></sup>,<sup><a href="#FN27" rid="FN27" class=" fn">25</a></sup></sup>, Douglas S. Goodin<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup>,<sup><a href="#FN28" rid="FN28" class=" fn">26</a></sup>,<sup><a href="#FN29" rid="FN29" class=" fn">27</a></sup></sup>, Douglas Jeffery<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup>,<sup><a href="#FN30" rid="FN30" class=" fn">28</a></sup></sup>, Paul O’Connor<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN17" rid="FN17" class=" fn">15</a></sup>,<sup><a href="#FN31" rid="FN31" class=" fn">29</a></sup></sup>, George C. Ebers<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN29" rid="FN29" class=" fn">27</a></sup>,<sup><a href="#FN32" rid="FN32" class=" fn">30</a></sup></sup>, Dawn Langdon<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN29" rid="FN29" class=" fn">27</a></sup>,<sup><a href="#FN33" rid="FN33" class=" fn">31</a></sup></sup>, Anthony T. Reder<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN29" rid="FN29" class=" fn">27</a></sup>,<sup><a href="#FN34" rid="FN34" class=" fn">32</a></sup></sup>, Anthony Traboulsee<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN29" rid="FN29" class=" fn">27</a></sup>,<sup><a href="#FN35" rid="FN35" class=" fn">33</a></sup></sup>, Frauke Zipp<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN24" rid="FN24" class=" fn">22</a></sup>,<sup><a href="#FN36" rid="FN36" class=" fn">34</a></sup></sup>, Jan Schimrigk<sup><sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN24" rid="FN24" class=" fn">22</a></sup>,<sup><a href="#FN38" rid="FN38" class=" fn">35</a></sup></sup>, Jan Hillert<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN13" rid="FN13" class=" fn">11</a></sup>,<sup><a href="#FN24" rid="FN24" class=" fn">22</a></sup>,<sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup></sup>, Melanie Bahlo<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN40" rid="FN40" class=" fn">37</a></sup></sup>, David R Booth<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN41" rid="FN41" class=" fn">38</a></sup></sup>, Simon Broadley<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN42" rid="FN42" class=" fn">39</a></sup>,<sup><a href="#FN43" rid="FN43" class=" fn">40</a></sup></sup>, Matthew A Brown<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN44" rid="FN44" class=" fn">41</a></sup>,<sup><a href="#FN45" rid="FN45" class=" fn">42</a></sup></sup>, Brian L Browning<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN46" rid="FN46" class=" fn">43</a></sup></sup>, Sharon R Browning<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN46" rid="FN46" class=" fn">43</a></sup></sup>, Helmut Butzkueven<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN47" rid="FN47" class=" fn">44</a></sup>,<sup><a href="#FN48" rid="FN48" class=" fn">45</a></sup></sup>, William M Carroll<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN49" rid="FN49" class=" fn">46</a></sup>,<sup><a href="#FN50" rid="FN50" class=" fn">47</a></sup></sup>, Caron Chapman<sup><sup><a href="#FN51" rid="FN51" class=" fn">48</a></sup></sup>, Simon J Foote<sup><sup><a href="#FN52" rid="FN52" class=" fn">49</a></sup></sup>, Lyn Griffiths<sup><sup><a href="#FN53" rid="FN53" class=" fn">50</a></sup></sup>, Allan G Kermode<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN49" rid="FN49" class=" fn">46</a></sup>,<sup><a href="#FN50" rid="FN50" class=" fn">47</a></sup></sup>, ,Trevor J Kilpatrick<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN47" rid="FN47" class=" fn">44</a></sup>,<sup><a href="#FN54" rid="FN54" class=" fn">51</a></sup>,<sup><a href="#FN55" rid="FN55" class=" fn">52</a></sup></sup>, Jeanette Lechner-Scott<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN56" rid="FN56" class=" fn">53</a></sup>,<sup><a href="#FN46" rid="FN46" class=" fn">43</a></sup></sup>, Mark Marriott<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN55" rid="FN55" class=" fn">52</a></sup></sup>, Deborah Mason<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN58" rid="FN58" class=" fn">55</a></sup></sup>, Pablo Moscato<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN57" rid="FN57" class=" fn">54</a></sup>,<sup><a href="#FN59" rid="FN59" class=" fn">56</a></sup></sup>, Robert N Heard<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN41" rid="FN41" class=" fn">38</a></sup></sup>, Michael P Pender<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN60" rid="FN60" class=" fn">57</a></sup>,<sup><a href="#FN61" rid="FN61" class=" fn">58</a></sup></sup>, Victoria M Perreau<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN54" rid="FN54" class=" fn">51</a></sup></sup>, Devindri Perera<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN52" rid="FN52" class=" fn">49</a></sup></sup>, Justin P Rubio<sup><sup><a href="#FN47" rid="FN47" class=" fn">44</a></sup></sup>, Rodney J Scott<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN56" rid="FN56" class=" fn">53</a></sup>,<sup><a href="#FN57" rid="FN57" class=" fn">54</a></sup>,<sup><a href="#FN59" rid="FN59" class=" fn">56</a></sup></sup>, Mark Slee<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN61" rid="FN61" class=" fn">58</a></sup></sup>, Jim Stankovich<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN47" rid="FN47" class=" fn">44</a></sup></sup>, Graeme J Stewart<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN41" rid="FN41" class=" fn">38</a></sup></sup>, Bruce V Taylor<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN47" rid="FN47" class=" fn">44</a></sup></sup>, Niall Tubridy<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN63" rid="FN63" class=" fn">60</a></sup></sup>, Ernest Willoughby<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN64" rid="FN64" class=" fn">61</a></sup></sup>, James Wiley<sup><sup><a href="#FN39" rid="FN39" class=" fn">36</a></sup>,<sup><a href="#FN65" rid="FN65" class=" fn">62</a></sup></sup>, Paul Matthews<sup><sup><a href="#FN18" rid="FN18" class=" fn">16</a></sup>,<sup><a href="#FN66" rid="FN66" class=" fn">63</a></sup></sup>, Filippo M. Boneschi<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>-<sup><a href="#FN8" rid="FN8" class=" fn">6</a></sup></sup>, Alastair Compston<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN67" rid="FN67" class=" fn">64</a></sup></sup>, Jonathan Haines<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN68" rid="FN68" class=" fn">65</a></sup></sup>, Stephen L Hauser<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN69" rid="FN69" class=" fn">66</a></sup></sup>, Jacob McCauley<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN70" rid="FN70" class=" fn">67</a></sup></sup>, Adrian Ivinson<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN71" rid="FN71" class=" fn">68</a></sup></sup>, Jorge R Oksenberg<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN69" rid="FN69" class=" fn">66</a></sup></sup>, Margaret Pericak-Vance <sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN70" rid="FN70" class=" fn">67</a></sup></sup>, Stephen J Sawcer<sup><sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN67" rid="FN67" class=" fn">64</a></sup></sup>, Philip L. De Jager<sup><sup><a href="#FN1" rid="FN1" class=" fn">#</a></sup><sup><a href="#FN2" rid="FN2" class=" fn">*</a>,</sup><sup><a href="#FN3" rid="FN3" class=" fn">1</a></sup>–<sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup></sup>, David A. Hafler<sup><a href="#FN2" rid="FN2" class=" fn">*</a></sup><sup><sup><a href="#FN5" rid="FN5" class=" fn">3</a></sup>,<sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN71" rid="FN71" class=" fn">68</a></sup></sup>, Paul I.W. de Bakker<sup><sup><a href="#FN4" rid="FN4" class=" fn">2</a></sup>–<sup><a href="#FN6" rid="FN6" class=" fn">4</a></sup>,<sup><a href="#FN72" rid="FN72" class=" fn">69</a></sup>,<sup><a href="#FN73" rid="FN73" class=" fn">70</a></sup></sup></p></div><div id="idm140337123136448" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140337123136448title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"><sup>#</sup>Corresponding author: Philip L. De Jager, MD PhD, Program in NeuroPsychiatric Genomics, Department of Neurology, Brigham &amp; Women’s Hospital, 77 Avenue Louis Pasteur, NRB168, Boston, MA 02115, Tel: 617 525 4529, Fax: 617 525 5722, <a href="mailto:dev@null" data-email="ude.dravrah.hwb.scir@regajedp" class="oemail">ude.dravrah.hwb.scir@regajedp</a></p><p class="fn sec" id="FN2"><sup>*</sup>contributed equally with PIW de Bakker</p><p class="fn sec" id="FN3"><sup>1</sup>Program in Translational NeuroPsychiatric Genomics, Neurosciences Institute, Departments of Neurology &amp; Psychiatry, Brigham &amp; Women’s Hospital, Boston, MA 02115, USA.</p><p class="fn sec" id="FN4"><sup>2</sup>Division of Genetics, Department of Medicine, Brigham &amp; Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA.</p><p class="fn sec" id="FN5"><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, MA 02139, USA.</p><p class="fn sec" id="FN6"><sup>4</sup>International Multiple Sclerosis Genetics Consortium (IMSGC)</p><p class="fn sec" id="FN7"><sup>5</sup>Department of Neurology, Scientific Institute San Raffaele, 20132 Milan, Italy</p><p class="fn sec" id="FN8"><sup>6</sup>Institute of Experimental Neurology, Scientific Institute San Raffaele, 20132 Milan, Italy</p><p class="fn sec" id="FN9"><sup>7</sup>The Bayer Schering Pharma Genomics (BSP-Gen) working group</p><p class="fn sec" id="FN10"><sup>8</sup>Bayer Schering Pharma AG, 13342, Berlin, Germany.</p><p class="fn sec" id="FN11"><sup>9</sup>Department of Neurology, Heinrich Heine University Düsseldorf, Germany</p><p class="fn sec" id="FN12"><sup>10</sup>Department of Neurology, University Hosptial Bonn, Germany</p><p class="fn sec" id="FN13"><sup>11</sup>steering committees of studies evaluating IFNb-1b and a CCR1-antagonist</p><p class="fn sec" id="FN14"><sup>12</sup>The BEtaferon®/BEtaseron® in Newly Emerging Multiple Sclerosis For Initial Treatment (BENEFIT) study</p><p class="fn sec" id="FN15"><sup>13</sup>Service Neurologie, C.H.U. Rennes Hôpital Pontchaillou/France.</p><p class="fn sec" id="FN16"><sup>14</sup>Neurologische Klinik und poliklinik, Universitaetsspital Basel/Switzerland.</p><p class="fn sec" id="FN17"><sup>15</sup>The Betaferon® Efficacy Yielding Outcomes of a New Dose (BEYOND) study</p><p class="fn sec" id="FN18"><sup>16</sup>The GeneMSA Consortium</p><p class="fn sec" id="FN19"><sup>17</sup>Institute of Neurology, The National Hospital, Queen Square, London/GB</p><p class="fn sec" id="FN20"><sup>18</sup>Unitat de Neuroimmunología Clínica Hospitals Vall dHebrón, Madrid/Spain</p><p class="fn sec" id="FN21"><sup>19</sup>Dept. Neurology Free University Hospital, Amsterdam/The Netherlands</p><p class="fn sec" id="FN22"><sup>20</sup>General Campus - Division of Neurology, The Ottawa Hospital, Ottawa/Canada</p><p class="fn sec" id="FN23"><sup>21</sup>Neurologische Klinik der Heinrich-Heine-Universität, Düsseldorf/Germany</p><p class="fn sec" id="FN24"><sup>22</sup>The CCR1-antagonist study</p><p class="fn sec" id="FN25"><sup>23</sup>Department of Neurology, Surgery Brain Research Institutes, Chicago/USA</p><p class="fn sec" id="FN26"><sup>24</sup>Department of Neurology and Neuroscience, UMDNJ-New Jersey Medical School, Newark, New Jersey/USA</p><p class="fn sec" id="FN27"><sup>25</sup>Neuroimaging Research Unit, University Hospital San Raffaele, Milano/Italy</p><p class="fn sec" id="FN28"><sup>26</sup>Department of Neurology, University of California San Francisco, San Francisco/USA</p><p class="fn sec" id="FN29"><sup>27</sup>The 16 year long term follow-up of the pivotal study on IFNB-1b in RRMS</p><p class="fn sec" id="FN30"><sup>28</sup>Wake Forest University, Department of Neurology, Winston-Salem/USA</p><p class="fn sec" id="FN31"><sup>29</sup>Division of Neurology, St. Michael’s Hospital, Toronto/Canada</p><p class="fn sec" id="FN32"><sup>30</sup>University Dept of Clinical Neurology, Oxford/UK</p><p class="fn sec" id="FN33"><sup>31</sup>Psychology Department, Royal Holloway, University of London/UK</p><p class="fn sec" id="FN34"><sup>32</sup>The University of Chicago, Department of Neurology, Chicago/USA</p><p class="fn sec" id="FN35"><sup>33</sup>UBC Hospital, Vancouver/Canada</p><p class="fn sec" id="FN36"><sup>34</sup>Cecilie-Vogt-Klinik, Charite – University Hospital Berlin, Germany</p><p class="fn sec" id="FN37"><sup>34</sup>University Hospital Bochum/Germany</p><p class="fn sec" id="FN38"><sup>35</sup>Karolinska Institutet at Huddinge University Hospital, Huddinge/Sweden</p><p class="fn sec" id="FN39"><sup>36</sup>ANZGene Consortium</p><p class="fn sec" id="FN40"><sup>37</sup>The Walter and Eliza Hall Institute of Medical Research, Parkville, VIC, AUS, 3050.</p><p class="fn sec" id="FN41"><sup>38</sup>The Westmead Millenium Institute, Westmead, NSW, AUS 2145.</p><p class="fn sec" id="FN42"><sup>39</sup>School of Medicine, Griffith University, QLD, AUS, 4222.</p><p class="fn sec" id="FN43"><sup>40</sup>Department of Neurology, Gold Coast Hospital, QLD, AUS 4215.</p><p class="fn sec" id="FN44"><sup>41</sup>Diamantina Institute of Cancer, Immunology and Metabolic Medicine, Princess Alexandra Hospital, University of Queensland, Brisbane, QLD, AUS 4102.</p><p class="fn sec" id="FN45"><sup>42</sup>Botnar Research Centre, Nuffield Department of Orthopaedic Surgery, University of Oxford, Oxford, OX3 7BN, UK.</p><p class="fn sec" id="FN46"><sup>43</sup>Department of Statistics, The University of Auckland, Auckland, NZ.</p><p class="fn sec" id="FN47"><sup>44</sup>The Howard Florey Institute, University of Melbourne, VIC, AUS 3010.</p><p class="fn sec" id="FN48"><sup>45</sup>Department of Medicine, University of Melbourne, VIC, AUS 3010.</p><p class="fn sec" id="FN49"><sup>46</sup>Sir Charles Gairdner Hospital, Nedlands, WA, AUS 6009.</p><p class="fn sec" id="FN50"><sup>47</sup>Australian Neuromuscular Research Institute, Nedlands WA, AUS 6009.</p><p class="fn sec" id="FN51"><sup>48</sup>Barwon Health, Geelong, VIC, AUS 3220.</p><p class="fn sec" id="FN52"><sup>49</sup>Menzies Research Institute, University of Tasmania, Hobart TAS 7001.</p><p class="fn sec" id="FN53"><sup>50</sup>Genomics Research Centre, Griffith University, QLD, AUS 4222.</p><p class="fn sec" id="FN54"><sup>51</sup>Centre for Neuroscience, University of Melbourne, VIC, AUS 3010.</p><p class="fn sec" id="FN55"><sup>52</sup>Royal Melbourne Hospital, Parkville, VIC, AUS 3050.</p><p class="fn sec" id="FN56"><sup>53</sup>John Hunter Hospital, Hunter New England Health Service, Newcastle NSW, AUS 2310.</p><p class="fn sec" id="FN57"><sup>54</sup>Hunter Medical Research Institute, Newcastle, NSW, AUS 2308.</p><p class="fn sec" id="FN58"><sup>55</sup>Canterbury District Health Board, Christchurch, NZ.</p><p class="fn sec" id="FN59"><sup>56</sup>Centre for Bioinformatics, Biomarker Discovery and Information-based Medicine, University of Newcastle, NSW, AUS 2308.</p><p class="fn sec" id="FN60"><sup>57</sup>School of Medicine, University of Queensland, QLD, AUS 4029.</p><p class="fn sec" id="FN61"><sup>58</sup>Department of Neurology, Royal Brisbane and Women’s Hospital, QLD, AUS 4029.</p><p class="fn sec" id="FN62"><sup>59</sup>School of Medicine, Department of Neurology, Flinders University of SouthAustralia, Bedford Park, Adelaide, SA, AUS 5042.</p><p class="fn sec" id="FN63"><sup>60</sup>Department of Neurology, St. Vincent’s University Hospital, Dublin, Republic of Ireland.</p><p class="fn sec" id="FN64"><sup>61</sup>Auckland District Health board, Auckland, NZ.</p><p class="fn sec" id="FN65"><sup>62</sup>Department of Medicine, Nepean Hospital, Penrith, NSW, AUS 2751.</p><p class="fn sec" id="FN66"><sup>63</sup>GSK Clinical Imaging Centre, Hammersmith Hospital and Department of Clinical Neurosciences, Imperial College, London.</p><p class="fn sec" id="FN67"><sup>64</sup>University of Cambridge, Department of Clinical Neuroscience, Addenbrooke’s Hospital, Hills Road, Cambridge, CB2 2QQ, UK</p><p class="fn sec" id="FN68"><sup>65</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, TN 37232-0700, USA</p><p class="fn sec" id="FN69"><sup>66</sup>Department of Neurology, University of California San Francisco, San Francisco, CA, 94143, USA</p><p class="fn sec" id="FN70"><sup>67</sup>John P. Hussman Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, FL 33136, USA</p><p class="fn sec" id="FN71"><sup>68</sup>Department of Neurology, Yale University School of Medicine, New Haven, CT 06520, USA.</p><p class="fn sec" id="FN72"><sup>69</sup>Department of Medical Genetics, Division of Biomedical Genetics, University Medical Center, Utrecht, The Netherlands</p><p class="fn sec" id="FN73"><sup>70</sup>Julius Center for Health Sciences and Primary Care, University Medical Center, Utrecht, The Netherlands</p></div></div><div id="idm140337116832848" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140337116832848title">Bibliography</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Hauser SL, Oksenberg JR. The neurobiology of multiple sclerosis: genes, inflammation, and neurodegeneration. <span><span class="ref-journal">Neuron. </span>2006;<span class="ref-vol">52</span>:61–76.</span> [<a href="/pubmed/17015227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=The+neurobiology+of+multiple+sclerosis:+genes,+inflammation,+and+neurodegeneration&amp;author=SL+Hauser&amp;author=JR+Oksenberg&amp;volume=52&amp;publication_year=2006&amp;pages=61-76&amp;pmid=17015227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Hafler DA, Compston A, et al.  International Multiple Sclerosis Genetics Consortium. Risk alleles for multiple sclerosis identified by a genomewide study. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:851–62.</span> [<a href="/pubmed/17660530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study&amp;author=DA+Hafler&amp;author=A+Compston&amp;volume=357&amp;publication_year=2007&amp;pages=851-62&amp;pmid=17660530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">De Jager PL, Jia X, Wang J, et al.  Meta-analysis of genome scans and replication identify CD6, IRF8 and TNFRSF1A as new multiple sclerosis susceptibility loci. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:776–782.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2757648/">PMC free article</a>]</span> [<a href="/pubmed/19525953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome+scans+and+replication+identify+CD6,+IRF8+and+TNFRSF1A+as+new+multiple+sclerosis+susceptibility+loci&amp;author=PL+De+Jager&amp;author=X+Jia&amp;author=J+Wang&amp;volume=41&amp;publication_year=2009&amp;pages=776-782&amp;pmid=19525953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Sanna S, Pitzalis M, Zoledziewska M, et al.  Variants within the immunoregulatory CBLB gene are associated with multiple sclerosis. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:495–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3786343/">PMC free article</a>]</span> [<a href="/pubmed/20453840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variants+within+the+immunoregulatory+CBLB+gene+are+associated+with+multiple+sclerosis&amp;author=S+Sanna&amp;author=M+Pitzalis&amp;author=M+Zoledziewska&amp;volume=42&amp;publication_year=2010&amp;pages=495-7&amp;pmid=20453840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">The Australia and New Zealand Multiple Sclerosis Genetics Consortium (ANZgene) Genome-wide association study identifies new multiple sclerosis susceptibility loci on chromosomes 12 and 20. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:824–828.</span> [<a href="/pubmed/19525955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+new+multiple+sclerosis+susceptibility+loci+on+chromosomes+12+and+20&amp;volume=41&amp;publication_year=2009&amp;pages=824-828&amp;pmid=19525955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">McDonald WI, Compston A, Edan G, et al.  Recommended diagnostic criteria for multiple sclerosis: guidelines from the International Panel on the diagnosis of multiple sclerosis. <span><span class="ref-journal">Ann Neurol. </span>2001;<span class="ref-vol">50</span>:121–127.</span> [<a href="/pubmed/11456302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Recommended+diagnostic+criteria+for+multiple+sclerosis:+guidelines+from+the+International+Panel+on+the+diagnosis+of+multiple+sclerosis&amp;author=WI+McDonald&amp;author=A+Compston&amp;author=G+Edan&amp;volume=50&amp;publication_year=2001&amp;pages=121-127&amp;pmid=11456302&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">De Jager PL, Chibnik LB, Cui J, et al.  Integration of genetic risk factors into a clinical algorithm for multiple sclerosis susceptibility: a weighted genetic risk score. <span><span class="ref-journal">Lancet Neurol. </span>2009;<span class="ref-vol">8</span>:1111–1119.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3099419/">PMC free article</a>]</span> [<a href="/pubmed/19879194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Integration+of+genetic+risk+factors+into+a+clinical+algorithm+for+multiple+sclerosis+susceptibility:+a+weighted+genetic+risk+score&amp;author=PL+De+Jager&amp;author=LB+Chibnik&amp;author=J+Cui&amp;volume=8&amp;publication_year=2009&amp;pages=1111-1119&amp;pmid=19879194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Li Y, Abecasis GR. Rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">American Journal of Human Genetics. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">de Bakker PI, Ferreira MA, Jia X, et al.  Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:R122–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=PI+de+Bakker&amp;author=MA+Ferreira&amp;author=X+Jia&amp;volume=17&amp;publication_year=2008&amp;pages=R122-8&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Higgins JP, Thompson SG. Quantifying heterogeneity in a meta-analysis. <span><span class="ref-journal">Stat Med. </span>2002;<span class="ref-vol">21</span>:1539–1558.</span> [<a href="/pubmed/12111919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Quantifying+heterogeneity+in+a+meta-analysis&amp;author=JP+Higgins&amp;author=SG+Thompson&amp;volume=21&amp;publication_year=2002&amp;pages=1539-1558&amp;pmid=12111919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Ioannidis JP, Patsopoulos NA, Evangelou E. Uncertainty in heterogeneity estimates in meta- analyses. <span><span class="ref-journal">BMJ. </span>2007;<span class="ref-vol">335</span>:914–916.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2048840/">PMC free article</a>]</span> [<a href="/pubmed/17974687" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Uncertainty+in+heterogeneity+estimates+in+meta-+analyses&amp;author=JP+Ioannidis&amp;author=NA+Patsopoulos&amp;author=E+Evangelou&amp;volume=335&amp;publication_year=2007&amp;pages=914-916&amp;pmid=17974687&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Voight BF, Scott LJ, Steinthorsdottir V, et al.  Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:579–589.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080658/">PMC free article</a>]</span> [<a href="/pubmed/20581827" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Twelve+type+2+diabetes+susceptibility+loci+identified+through+large-scale+association+analysis&amp;author=BF+Voight&amp;author=LJ+Scott&amp;author=V+Steinthorsdottir&amp;volume=42&amp;publication_year=2010&amp;pages=579-589&amp;pmid=20581827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Lango Allen H, Estrada K, Lettre G, et al.  Hundreds of variants clustered in genomic loci and biological pathways affect human height. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:832–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955183/">PMC free article</a>]</span> [<a href="/pubmed/20881960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hundreds+of+variants+clustered+in+genomic+loci+and+biological+pathways+affect+human+height&amp;author=H+Lango+Allen&amp;author=K+Estrada&amp;author=G+Lettre&amp;volume=467&amp;publication_year=2010&amp;pages=832-8&amp;pmid=20881960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Teslovich TM, Musunuru K, Smith AV, et al.  Biological, clinical and population relevance of 95 loci for blood lipids. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:707–713.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039276/">PMC free article</a>]</span> [<a href="/pubmed/20686565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological,+clinical+and+population+relevance+of+95+loci+for+blood+lipids&amp;author=TM+Teslovich&amp;author=K+Musunuru&amp;author=AV+Smith&amp;volume=466&amp;publication_year=2010&amp;pages=707-713&amp;pmid=20686565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Kathiresan S, Voight BF, Purcell S, et al.  Genome-wide association of early-onset myocardial infarction with single nucleotide polymorphisms and copy number variants. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:334–341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681011/">PMC free article</a>]</span> [<a href="/pubmed/19198609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+of+early-onset+myocardial+infarction+with+single+nucleotide+polymorphisms+and+copy+number+variants&amp;author=S+Kathiresan&amp;author=BF+Voight&amp;author=S+Purcell&amp;volume=41&amp;publication_year=2009&amp;pages=334-341&amp;pmid=19198609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Erdmann J, Grosshennig A, Braund PS, et al.  New susceptibility locus for coronary artery disease on chromosome 3q22.3. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:280–282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695543/">PMC free article</a>]</span> [<a href="/pubmed/19198612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+susceptibility+locus+for+coronary+artery+disease+on+chromosome+3q22.3&amp;author=J+Erdmann&amp;author=A+Grosshennig&amp;author=PS+Braund&amp;volume=41&amp;publication_year=2009&amp;pages=280-282&amp;pmid=19198612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Samani NJ, Erdmann J, Hall AS, et al.  Genomewide association analysis of coronary artery disease. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:443–453.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719290/">PMC free article</a>]</span> [<a href="/pubmed/17634449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genomewide+association+analysis+of+coronary+artery+disease&amp;author=NJ+Samani&amp;author=J+Erdmann&amp;author=AS+Hall&amp;volume=357&amp;publication_year=2007&amp;pages=443-453&amp;pmid=17634449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Gauthier SA, Mandel M, Guttmann CR, et al.  Predicting short-term disability in multiple sclerosis. <span><span class="ref-journal">Neurology. </span>2007;<span class="ref-vol">68</span>:2059–2065.</span> [<a href="/pubmed/17562826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Predicting+short-term+disability+in+multiple+sclerosis&amp;author=SA+Gauthier&amp;author=M+Mandel&amp;author=CR+Guttmann&amp;volume=68&amp;publication_year=2007&amp;pages=2059-2065&amp;pmid=17562826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">De Jager PL, Baecher-Allan C, Maier LM, et al.  The role of the CD58 locus in multiple sclerosis. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009;<span class="ref-vol">106</span>:5264–5269.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2664005/">PMC free article</a>]</span> [<a href="/pubmed/19237575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=The+role+of+the+CD58+locus+in+multiple+sclerosis&amp;author=PL+De+Jager&amp;author=C+Baecher-Allan&amp;author=LM+Maier&amp;volume=106&amp;publication_year=2009&amp;pages=5264-5269&amp;pmid=19237575&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Maier LM, Lowe CE, Cooper J, et al.  IL2RA genetic heterogeneity in multiple sclerosis and type 1 diabetes susceptibility and soluble interleukin-2 receptor production. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000322.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2602853/">PMC free article</a>]</span> [<a href="/pubmed/19119414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=IL2RA+genetic+heterogeneity+in+multiple+sclerosis+and+type+1+diabetes+susceptibility+and+soluble+interleukin-2+receptor+production&amp;author=LM+Maier&amp;author=CE+Lowe&amp;author=J+Cooper&amp;volume=5&amp;publication_year=2009&amp;pages=e1000322&amp;pmid=19119414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Jakkula E, Leppa V, Sulonen AM, et al.  Genome-wide association study in a high-risk isolate for multiple sclerosis reveals associated variants in STAT3 gene. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">86</span>:285–291.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820168/">PMC free article</a>]</span> [<a href="/pubmed/20159113" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+association+study+in+a+high-risk+isolate+for+multiple+sclerosis+reveals+associated+variants+in+STAT3+gene&amp;author=E+Jakkula&amp;author=V+Leppa&amp;author=AM+Sulonen&amp;volume=86&amp;publication_year=2010&amp;pages=285-291&amp;pmid=20159113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Aulchenko YS, Hoppenbrouwers IA, Ramagopalan SV, et al.  Genetic variation in the KIF1B locus influences susceptibility to multiple sclerosis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1402–1403.</span> [<a href="/pubmed/18997785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+the+KIF1B+locus+influences+susceptibility+to+multiple+sclerosis&amp;author=YS+Aulchenko&amp;author=IA+Hoppenbrouwers&amp;author=SV+Ramagopalan&amp;volume=40&amp;publication_year=2008&amp;pages=1402-1403&amp;pmid=18997785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">International Multiple Sclerosis Genetics Consortium (IMSGC) Comprehensive follow-up of the first genome-wide association study of multiple sclerosis identifies KIF21B and TMEM39A as susceptibility loci. <span><span class="ref-journal">Hum Mol Genet. </span>2010;<span class="ref-vol">19</span>:953–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2816610/">PMC free article</a>]</span> [<a href="/pubmed/20007504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Comprehensive+follow-up+of+the+first+genome-wide+association+study+of+multiple+sclerosis+identifies+KIF21B+and+TMEM39A+as+susceptibility+loci&amp;volume=19&amp;publication_year=2010&amp;pages=953-962&amp;pmid=20007504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Barrett JC, Hansoul S, Nicolae DL, et al.  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn’s disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:955–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn’s+disease&amp;author=JC+Barrett&amp;author=S+Hansoul&amp;author=DL+Nicolae&amp;volume=40&amp;publication_year=2008&amp;pages=955-962&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Nair RP, Duffin KC, Helms C, et al.  Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:199–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-kappaB+pathways&amp;author=RP+Nair&amp;author=KC+Duffin&amp;author=C+Helms&amp;volume=41&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Hunt KA, Zhernakova A, Turner G, et al.  Newly identified genetic risk variants for celiac disease related to the immune response. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:395–402.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2673512/">PMC free article</a>]</span> [<a href="/pubmed/18311140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Newly+identified+genetic+risk+variants+for+celiac+disease+related+to+the+immune+response&amp;author=KA+Hunt&amp;author=A+Zhernakova&amp;author=G+Turner&amp;volume=40&amp;publication_year=2008&amp;pages=395-402&amp;pmid=18311140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Smyth DJ, Plagnol V, Walker NM, et al.  Shared and distinct genetic variants in type 1 diabetes and celiac disease. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">359</span>:2767–2777.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2840835/">PMC free article</a>]</span> [<a href="/pubmed/19073967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Shared+and+distinct+genetic+variants+in+type+1+diabetes+and+celiac+disease&amp;author=DJ+Smyth&amp;author=V+Plagnol&amp;author=NM+Walker&amp;volume=359&amp;publication_year=2008&amp;pages=2767-2777&amp;pmid=19073967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Lovett-Racke AE, Rocchini AE, Choy J, et al.  Silencing T-bet defines a critical role in the differentiation of autoreactive T lymphocytes. <span><span class="ref-journal">Immunity. </span>2004;<span class="ref-vol">21</span>:719–731.</span> [<a href="/pubmed/15539157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Silencing+T-bet+defines+a+critical+role+in+the+differentiation+of+autoreactive+T+lymphocytes&amp;author=AE+Lovett-Racke&amp;author=AE+Rocchini&amp;author=J+Choy&amp;volume=21&amp;publication_year=2004&amp;pages=719-731&amp;pmid=15539157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Placek K, Coffre M, Maiella S, et al.  Genetic and epigenetic networks controlling T helper 1 cell differentiation. <span><span class="ref-journal">Immunology. </span>2009;<span class="ref-vol">127</span>:155–162.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2691780/">PMC free article</a>]</span> [<a href="/pubmed/19476511" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunology&amp;title=Genetic+and+epigenetic+networks+controlling+T+helper+1+cell+differentiation&amp;author=K+Placek&amp;author=M+Coffre&amp;author=S+Maiella&amp;volume=127&amp;publication_year=2009&amp;pages=155-162&amp;pmid=19476511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Nurieva RI, Liu X, Dong C. Yin-Yang of costimulation: crucial controls of immune tolerance and function. <span><span class="ref-journal">Immunol Rev. </span>2009;<span class="ref-vol">229</span>:88–100.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2875245/">PMC free article</a>]</span> [<a href="/pubmed/19426216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunol+Rev&amp;title=Yin-Yang+of+costimulation:+crucial+controls+of+immune+tolerance+and+function&amp;author=RI+Nurieva&amp;author=X+Liu&amp;author=C+Dong&amp;volume=229&amp;publication_year=2009&amp;pages=88-100&amp;pmid=19426216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Stahl EA, Raychaudhuri S, Remmers EF, et al.  Genome-wide association study meta- analysis identifies seven new rheumatoid arthritis risk loci. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:508–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4243840/">PMC free article</a>]</span> [<a href="/pubmed/20453842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+meta-+analysis+identifies+seven+new+rheumatoid+arthritis+risk+loci&amp;author=EA+Stahl&amp;author=S+Raychaudhuri&amp;author=EF+Remmers&amp;volume=42&amp;publication_year=2010&amp;pages=508-14&amp;pmid=20453842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Dubois PC, Trynka G, Franke L, et al.  Multiple common variants for celiac disease influencing immune gene expression. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:295–302.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2847618/">PMC free article</a>]</span> [<a href="/pubmed/20190752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+common+variants+for+celiac+disease+influencing+immune+gene+expression&amp;author=PC+Dubois&amp;author=G+Trynka&amp;author=L+Franke&amp;volume=42&amp;publication_year=2010&amp;pages=295-302&amp;pmid=20190752&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Pereira TV, Patsopoulos NA, Salanti G, et al.  Discovery properties of genome-wide association signals from cumulatively combined data sets. <span><span class="ref-journal">Am J Epidemiol. </span>2009;<span class="ref-vol">170</span>:1197–1206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2800267/">PMC free article</a>]</span> [<a href="/pubmed/19808636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Discovery+properties+of+genome-wide+association+signals+from+cumulatively+combined+data+sets&amp;author=TV+Pereira&amp;author=NA+Patsopoulos&amp;author=G+Salanti&amp;volume=170&amp;publication_year=2009&amp;pages=1197-1206&amp;pmid=19808636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Nicolae DL, Gamazon E, Zhang W, et al.  Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000888.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2848547/">PMC free article</a>]</span> [<a href="/pubmed/20369019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Trait-associated+SNPs+are+more+likely+to+be+eQTLs:+annotation+to+enhance+discovery+from+GWAS&amp;author=DL+Nicolae&amp;author=E+Gamazon&amp;author=W+Zhang&amp;volume=6&amp;publication_year=2010&amp;pages=e1000888&amp;pmid=20369019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Brynedal B, Duvefelt K, Jonasdottir G, et al.  HLA-A confers an HLA-DRB1 independent influence on the risk of multiple sclerosis. <span><span class="ref-journal">PLoS One. </span>2007;<span class="ref-vol">2</span>:e664.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1919434/">PMC free article</a>]</span> [<a href="/pubmed/17653284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=HLA-A+confers+an+HLA-DRB1+independent+influence+on+the+risk+of+multiple+sclerosis&amp;author=B+Brynedal&amp;author=K+Duvefelt&amp;author=G+Jonasdottir&amp;volume=2&amp;publication_year=2007&amp;pages=e664&amp;pmid=17653284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Brereton CF, Sutton CE, Lalor SJ, et al.  Inhibition of ERK MAPK suppresses IL-23- and IL-1-driven IL-17 production and attenuates autoimmune disease. <span><span class="ref-journal">J Immunol. </span>2009;<span class="ref-vol">183</span>:1715–1723.</span> [<a href="/pubmed/19570828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Inhibition+of+ERK+MAPK+suppresses+IL-23-+and+IL-1-driven+IL-17+production+and+attenuates+autoimmune+disease&amp;author=CF+Brereton&amp;author=CE+Sutton&amp;author=SJ+Lalor&amp;volume=183&amp;publication_year=2009&amp;pages=1715-1723&amp;pmid=19570828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">de Beaucoudrey L, Puel A, Filipe-Santos O, et al.  Mutations in STAT3 and IL12RB1 impair the development of human IL-17-producing T cells. <span><span class="ref-journal">J Exp Med. </span>2008;<span class="ref-vol">205</span>:1543–1550.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2442631/">PMC free article</a>]</span> [<a href="/pubmed/18591412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Mutations+in+STAT3+and+IL12RB1+impair+the+development+of+human+IL-17-producing+T+cells&amp;author=L+de+Beaucoudrey&amp;author=A+Puel&amp;author=O+Filipe-Santos&amp;volume=205&amp;publication_year=2008&amp;pages=1543-1550&amp;pmid=18591412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Marchini J, Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>:499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Li Y, Willer C, Sanna S, et al.  Genotype imputation. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3247076&amp;issue-id=203878&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140337117463856title">Abstract</a></li><li><a href="#S5title">INTRODUCTION</a></li><li><a href="#S6title">MATERIALS &amp; METHODS</a></li><li><a href="#S7title">RESULTS</a></li><li><a href="#S13title">DISCUSSION</a></li><li><a href="#S14title">Supplementary Material</a></li><li><a href="#S15title">Acknowledgments</a></li><li><a href="#APP1title">Author Index</a></li><li><a href="#idm140337123136448title">Footnotes</a></li><li><a href="#idm140337116832848title">Bibliography</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3247076/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3247076/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3247076/pdf/nihms318621.pdf">PDF (620K)</a></li> | <li><a href="#" data-citationid="PMC3247076" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3247076%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3247076%2F&amp;text=Genomewide%20meta-analysis%20identifies%20novel%20multiple%20sclerosis%20susceptibility%20loci" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3247076%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22190364" data-aiid="3247076" data-aid="3247076" data-iid="203878" data-domainid="319" data-domain="nihpa" data-accid="PMC3247076" data-md5="2430058acc72c769dff636057925c437"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:20:04-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89C5E4DD53B43100000000016E010D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3247076%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3247076/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>